<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="edgeR 4.0: powerful differential analysis of sequencing data with expanded functionality and improved support for small counts and larger datasets
Authors: Chen, Y.; Chen, L.; Lun, A. T. L.; Baldoni, P. L.; Smyth, G. K.
Score: 142.8, Published: 2024-01-24 DOI: 10.1101/2024.01.21.576131
edgeR is an R/Bioconductor software package for differential analyses of sequencing data in the form of read counts for genes or genomic features. Over the past 15 years, edgeR has been a popular choice for statistical analysis of data from sequencing technologies such as RNA-seq or ChIP-seq.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="edgeR 4.0: powerful differential analysis of sequencing data with expanded functionality and improved support for small counts and larger datasets
Authors: Chen, Y.; Chen, L.; Lun, A. T. L.; Baldoni, P. L.; Smyth, G. K.
Score: 142.8, Published: 2024-01-24 DOI: 10.1101/2024.01.21.576131
edgeR is an R/Bioconductor software package for differential analyses of sequencing data in the form of read counts for genes or genomic features. Over the past 15 years, edgeR has been a popular choice for statistical analysis of data from sequencing technologies such as RNA-seq or ChIP-seq." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-11T10:38:55+00:00" />
<meta property="article:modified_time" content="2024-02-11T10:38:55+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="edgeR 4.0: powerful differential analysis of sequencing data with expanded functionality and improved support for small counts and larger datasets
Authors: Chen, Y.; Chen, L.; Lun, A. T. L.; Baldoni, P. L.; Smyth, G. K.
Score: 142.8, Published: 2024-01-24 DOI: 10.1101/2024.01.21.576131
edgeR is an R/Bioconductor software package for differential analyses of sequencing data in the form of read counts for genes or genomic features. Over the past 15 years, edgeR has been a popular choice for statistical analysis of data from sequencing technologies such as RNA-seq or ChIP-seq."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "edgeR 4.0: powerful differential analysis of sequencing data with expanded functionality and improved support for small counts and larger datasets\nAuthors: Chen, Y.; Chen, L.; Lun, A. T. L.; Baldoni, P. L.; Smyth, G. K.\nScore: 142.8, Published: 2024-01-24 DOI: 10.1101/2024.01.21.576131\nedgeR is an R/Bioconductor software package for differential analyses of sequencing data in the form of read counts for genes or genomic features. Over the past 15 years, edgeR has been a popular choice for statistical analysis of data from sequencing technologies such as RNA-seq or ChIP-seq.",
  "keywords": [
    
  ],
  "articleBody": " edgeR 4.0: powerful differential analysis of sequencing data with expanded functionality and improved support for small counts and larger datasets\nAuthors: Chen, Y.; Chen, L.; Lun, A. T. L.; Baldoni, P. L.; Smyth, G. K.\nScore: 142.8, Published: 2024-01-24 DOI: 10.1101/2024.01.21.576131\nedgeR is an R/Bioconductor software package for differential analyses of sequencing data in the form of read counts for genes or genomic features. Over the past 15 years, edgeR has been a popular choice for statistical analysis of data from sequencing technologies such as RNA-seq or ChIP-seq. edgeR pioneered the use of the negative binomial distribution to model read count data with replicates and the use of generalized linear models to analyse complex experimental designs. edgeR implements empirical Bayes moderation methods to allow reliable inference when the number of replicates is small. This article announces edgeR version 4, which includes new developments across a range of application areas. Infrastructure improvements include support for fractional counts, implementation of model fitting in C++, and a new statistical treatment of the quasi-likelihood pipeline that improves accuracy for small counts. The revised package has new functionality for differential methylation analysis, differential transcript expression, differential transcript and exon usage, testing relative to a fold-change threshold and pathway analysis. This article reviews the statistical framework and computational implementation of edgeR, briefly summarizing all the existing features and functionalities but with special attention to new features and those that have not been described previously.\nMetagenomic estimation of dietary intake from human stool\nAuthors: Diener, C.; Gibbons, S. M.\nScore: 52.1, Published: 2024-02-06 DOI: 10.1101/2024.02.02.578701\nDietary intake is tightly coupled to gut microbiota composition, human metabolism, and to the incidence of virtually all major chronic diseases. Dietary and nutrient intake are usually quantified using dietary questionnaires, which tend to focus on broad food categories, suffer from self-reporting biases, and require strong compliance from study participants. Here, we present MEDI (Metagenomic Estimation of Dietary Intake): a method for quantifying dietary intake using food-derived DNA in stool metagenomes. We show that food items can be accurately detected in metagenomic shotgun sequencing data, even when present at low abundances (\u003e10 reads). Furthermore, we show how dietary intake, in terms of DNA abundance from specific organisms, can be converted into a detailed metabolic representation of nutrient intake. MEDI could identify the onset of solid food consumption in infants and it accurately predicted food questionnaire responses in an adult population. Additionally, we were able to identify specific dietary features associated with metabolic syndrome in a large clinical cohort, providing a proof-of-concept for detailed quantification of individual-specific dietary patterns without the need for questionnaires.\nSpotiflow: accurate and efficient spot detection for imaging-based spatial transcriptomics with stereographic flow regression\nAuthors: Dominguez Mantes, A.; Herrera, A.; Khven, I.; Schlaeppi, A.; Kyriacou, E.; Tsissios, G.; Aztekin, C.; Ligner, J.; La Manno, G.; Weigert, M.\nScore: 46.2, Published: 2024-02-05 DOI: 10.1101/2024.02.01.578426\nThe identification of spot-like structures in large and noisy microscopy images is an important task in many life science techniques, and it is essential to their quantitative performance. For example, imaging-based spatial transcriptomics (iST) methods rely critically on the accurate detection of millions of transcripts in low signal-to-noise ratio (SNR) images. While recent developments in computer vision have revolutionized many bioimage tasks, currently adopted spot detection approaches for iST still rely on classical signal processing methods that are fragile and require manually tuning. In this work we introduce Spotiflow, a deep-learning method that casts the spot-detection problem as a multiscale stereographic flow regression problem that yields subpixel-accurate localizations. Spotiflow is robust to different noise conditions and generalizes across different chemistries while being up to an order-of-magnitude more time and memory efficient than commonly used methods. We show the efficacy of Spotiflow by comprehensive quantitative comparisons against other methods on a variety of datasets and demonstrate the impact of its increased accuracy on the biological conclusions drawn from iST and live imaging experiments. Spotiflow is available as an easy-to-use Python library as well as a napari plugin at http://www.github.com/weigertlab/spotiflow.\nStaVia: Spatially and temporally aware cartography with higher order random walks for cell atlases\nAuthors: Stassen, S. V.; Kobashi, M.; Huang, Y.; Ho, J. W. K.; Tsia, K.\nScore: 71.7, Published: 2024-01-31 DOI: 10.1101/2024.01.29.577871\nSingle-cell atlases are critical for unraveling the cellular basis of health and disease, yet their sheer diversity and vast data landscape in time and space pose daunting computational challenges pertaining to the delineation of multiple simultaneously emerging lineages, the integration of spatial and temporal information, and the visualization of trajectories across large atlases with enough resolution to observe localized transitions. To tackle this intricacy, we introduce StaVia, a computational framework that synergizes multi-faceted single-cell data--spanning time-series data, spatial gene expression patterns, and directional trends from RNA velocity-- with higher-order random walks that leverage the memory of cells past states. StaVia fuses this method with a cartographic \"Atlas View\" that offers intuitive graph visualization, simultaneously capturing the nuanced details of cellular development at single-cell resolution as well as the broader connectivity of cell lineages, avoiding common pitfalls of merged distinct trajectories or missed transitional states seen in existing methods which are all memoryless. Notably, we demonstrate that StaVia unlocks new insights into placode development, radial glia pluripotency during neurulation, and the transitions pivotal to these processes in a large-scale Zebrafish developmental atlas. StaVia also allows spatially aware cartography that captures relationships between cell populations based on their spatial location as well as their gene expressions in a MERFISH dataset - underscoring its potential to dissect complex biological landscapes in both spatial and temporal contexts.\nSEraster: a rasterization preprocessing framework for scalable spatial omics data analysis\nAuthors: Aihara, G.; Clifton, K.; Chen, M.; Atta, L.; Miller, B. F.; Fan, J.\nScore: 30.7, Published: 2024-02-05 DOI: 10.1101/2024.02.01.578436\nMotivationSpatial omics data demand computational analysis but many analysis tools have computational resource requirements that increase with the number of cells analyzed. This presents scalability challenges as researchers use spatial omics technologies to profile millions of cells. ResultsTo enhance the scalability of spatial omics data analysis, we developed a rasterization preprocessing framework called SEraster that aggregates cellular information into spatial pixels. We apply SEraster to both real and simulated spatial omics data prior to spatial variable gene expression analysis to demonstrate that such preprocessing can reduce resource requirements while maintaining high performance. We further integrate SEraster with existing analysis tools to characterize cell-type spatial cooccurrence. Finally, we apply SEraster to enable analysis of a mouse pup spatial omics dataset with over a million cells to identify tissue-level and cell-type-specific spatially variable genes as well as cooccurring cell-types that recapitulate expected organ structures. Availability and implementationSource code is available on GitHub (https://github.com/JEFworks-Lab/SEraster) with additional tutorials at https://JEF.works/SEraster.\nBiaPy: A unified framework for versatile bioimage analysis with deep learning\nAuthors: Franco-Barranco, D.; Andres-San Roman, J. A.; Hidalgo-Cenalmor, I.; Backova, L.; Gonzalez-Marfil, A.; Caporal, C.; Chessel, A.; Gomez-Galvez, P.; Escudero, L. M.; Wei, D.; Munoz-Barrutia, A.; Arganda-Carreras, I.\nScore: 25.9, Published: 2024-02-05 DOI: 10.1101/2024.02.03.576026\nBiaPy, a unified open-source bioimage analysis library, offers a comprehensive suite of deep learning-powered workflows. Tailored for users of all levels, BiaPy features an intuitive interface, zero-code notebooks, and Docker integration. With support for 2D and 3D image data, it addresses existing gaps by providing multi-GPU capabilities, memory optimization, and compatibility with large datasets. As a collaborative and accessible solution, BiaPy aims to empower researchers by democratizing the use of sophisticated and efficient bioimage analysis workflows.\nGenerRNA: A generative pre-trained language model for de novo RNA design\nAuthors: Zhao, Y.; Oono, K.; Takizawa, H.; Kotera, M.\nScore: 22.4, Published: 2024-02-08 DOI: 10.1101/2024.02.01.578496\nThe design of RNA plays a crucial role in developing RNA vaccines, nucleic acid therapeutics, and innovative biotechnological tools. Nevertheless, existing techniques lack versatility across various tasks and frequently suffer from a deficiency of automated generation. Inspired by the remarkable success of Large Language Models (LLMs) in the realm of protein and molecule design, we present GenerRNA, the first large-scale pre-trained model for RNA generation, aiming to further automate RNA design. Our approach eliminates the need for secondary structure or other prior knowledge and is capable of de novo generation of RNA with stable secondary structures while ensuring its distinctiveness from existing sequences. This widens our exploration of RNA space, thereby enriching our understanding of RNA structures and functions. Moreover, GenerRNA is fine-tunable on smaller, more specialized datasets for particular subtasks. This flexibility and versatility enables the generation of RNAs with desired specific functionalities or properties. Upon fine-tuning GenerRNA, we successfully generated novel RNA sequences exhibiting high affinity for target proteins. GenerRNA is freely available at the following repository: https://github.com/pfnet-research/GenerRNA\nForseti: A mechanistic and predictive model of the splicing status of scRNA-seq reads\nAuthors: He, D.; Gao, Y.; Chan, S. S.; Quintana-Parrilla, N.; Patro, R.\nScore: 21.4, Published: 2024-02-05 DOI: 10.1101/2024.02.01.577813\nMotivationShort-read single-cell RNA-sequencing (scRNA-seq) has been used to study cellular heterogeneity, cellular fate, and transcriptional dynamics. Modeling splicing dynamics in scRNA-seq data is challenging, with inherent difficulty in even the seemingly straightforward task of elucidating the splicing status of the molecules from which sequenced fragments are drawn. This difficulty arises, in part, from the limited read length and positional biases, which substantially reduce the specificity of the sequenced fragments. As a result, the splicing status of many reads in scRNA-seq is ambiguous because of a lack of definitive evidence. We are therefore in need of methods that can recover the splicing status of ambiguous reads which, in turn, can lead to more accuracy and confidence in downstream analyses. ResultsWe develop Forseti, a predictive model to probabilistically assign a splicing status to scRNA-seq reads. Our model has two key components. First, we train a binding affinity model to assign a probability that a given transcriptomic site is used in fragment generation. Second, we fit a robust fragment length distribution model that generalizes well across datasets deriving from different species and tissue types. Forseti combines these two trained models to predict the splicing status of the molecule of origin of reads by scoring putative fragments that associate each alignment of sequenced reads with proximate potential priming sites. Using both simulated and experimental data, we show that our model can precisely predict the splicing status of reads and identify the true gene origin of multi-gene mapped reads. AvailabilityForseti and the code used for producing the results are available at https://github.com/COMBINE-lab/forseti under a BSD 3-clause license.\nMulti-genome synteny detection using minimizer graph mappings\nAuthors: Coombe, L.; Kazemi, P.; Wong, J.; Birol, I.; Warren, R. L.\nScore: 17.6, Published: 2024-02-09 DOI: 10.1101/2024.02.07.579356\nIn recent years, the landscape of reference-grade genome assemblies has seen substantial diversification. With such rich data, there is pressing demand for robust tools for scalable, multi-species comparative genomics analyses, including detecting genome synteny, which informs on the sequence conservation between genomes and contributes crucial insights into species evolution. Here, we introduce ntSynt, a scalable utility for computing large-scale multi-genome synteny blocks using a minimizer graph-based approach. Through extensive testing utilizing multiple ~3 Gbp genomes, we demonstrate how ntSynt produces synteny blocks with coverages between 79-100% in at most 2h using 34 GB of memory, even for genomes with appreciable (\u003e15%) sequence divergence. Compared to existing state-of-the-art methodologies, ntSynt offers enhanced flexibility to diverse input genome sequences and synteny block granularity. We expect the macrosyntenic genome analyses facilitated by ntSynt will have broad utility in generating critical evolutionary insights within and between species across the tree of life.\nLazyAF, a pipeline for accessible medium-scale in silico prediction of protein-protein interactions\nAuthors: McLean, T. C.\nScore: 104.3, Published: 2024-02-01 DOI: 10.1101/2024.01.29.577767\nArtificial intelligence has revolutionized the field of protein structure prediction. However, with more powerful and complex software being developed, it is accessibility and ease of use rather than capability that is quickly becoming a limiting factor to end users. Here, I present a Google Colaboratory-based pipeline, named LazyAF, which integrates the existing ColabFold BATCH to streamline the process of medium-scale protein-protein interaction prediction. I apply LazyAF to predict the interactome of the 76 proteins encoded on a broad-host-range multi-drug resistance plasmid RK2, demonstrating the ease and accessibility the pipeline provides. AvailabilityLazyAF is freely available at https://github.com/ThomasCMcLean/LazyAF\n",
  "wordCount" : "2007",
  "inLanguage": "en",
  "datePublished": "2024-02-11T10:38:55Z",
  "dateModified": "2024-02-11T10:38:55Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 11, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.21.576131">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.21.576131" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.21.576131">
        <p class="paperTitle">edgeR 4.0: powerful differential analysis of sequencing data with expanded functionality and improved support for small counts and larger datasets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.21.576131" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.21.576131" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, Y.; Chen, L.; Lun, A. T. L.; Baldoni, P. L.; Smyth, G. K.</p>
        <p class="info">Score: 142.8, Published: 2024-01-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.21.576131' target='https://doi.org/10.1101/2024.01.21.576131'> 10.1101/2024.01.21.576131</a></p>
        <p class="abstract">edgeR is an R/Bioconductor software package for differential analyses of sequencing data in the form of read counts for genes or genomic features. Over the past 15 years, edgeR has been a popular choice for statistical analysis of data from sequencing technologies such as RNA-seq or ChIP-seq. edgeR pioneered the use of the negative binomial distribution to model read count data with replicates and the use of generalized linear models to analyse complex experimental designs. edgeR implements empirical Bayes moderation methods to allow reliable inference when the number of replicates is small. This article announces edgeR version 4, which includes new developments across a range of application areas. Infrastructure improvements include support for fractional counts, implementation of model fitting in C&#43;&#43;, and a new statistical treatment of the quasi-likelihood pipeline that improves accuracy for small counts. The revised package has new functionality for differential methylation analysis, differential transcript expression, differential transcript and exon usage, testing relative to a fold-change threshold and pathway analysis. This article reviews the statistical framework and computational implementation of edgeR, briefly summarizing all the existing features and functionalities but with special attention to new features and those that have not been described previously.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.02.578701">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.02.578701" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.02.578701">
        <p class="paperTitle">Metagenomic estimation of dietary intake from human stool</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.02.578701" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.02.578701" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Diener, C.; Gibbons, S. M.</p>
        <p class="info">Score: 52.1, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.02.578701' target='https://doi.org/10.1101/2024.02.02.578701'> 10.1101/2024.02.02.578701</a></p>
        <p class="abstract">Dietary intake is tightly coupled to gut microbiota composition, human metabolism, and to the incidence of virtually all major chronic diseases. Dietary and nutrient intake are usually quantified using dietary questionnaires, which tend to focus on broad food categories, suffer from self-reporting biases, and require strong compliance from study participants. Here, we present MEDI (Metagenomic Estimation of Dietary Intake): a method for quantifying dietary intake using food-derived DNA in stool metagenomes. We show that food items can be accurately detected in metagenomic shotgun sequencing data, even when present at low abundances (&gt;10 reads). Furthermore, we show how dietary intake, in terms of DNA abundance from specific organisms, can be converted into a detailed metabolic representation of nutrient intake. MEDI could identify the onset of solid food consumption in infants and it accurately predicted food questionnaire responses in an adult population. Additionally, we were able to identify specific dietary features associated with metabolic syndrome in a large clinical cohort, providing a proof-of-concept for detailed quantification of individual-specific dietary patterns without the need for questionnaires.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578426">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578426" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578426">
        <p class="paperTitle">Spotiflow: accurate and efficient spot detection for imaging-based spatial transcriptomics with stereographic flow regression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578426" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578426" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dominguez Mantes, A.; Herrera, A.; Khven, I.; Schlaeppi, A.; Kyriacou, E.; Tsissios, G.; Aztekin, C.; Ligner, J.; La Manno, G.; Weigert, M.</p>
        <p class="info">Score: 46.2, Published: 2024-02-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578426' target='https://doi.org/10.1101/2024.02.01.578426'> 10.1101/2024.02.01.578426</a></p>
        <p class="abstract">The identification of spot-like structures in large and noisy microscopy images is an important task in many life science techniques, and it is essential to their quantitative performance. For example, imaging-based spatial transcriptomics (iST) methods rely critically on the accurate detection of millions of transcripts in low signal-to-noise ratio (SNR) images. While recent developments in computer vision have revolutionized many bioimage tasks, currently adopted spot detection approaches for iST still rely on classical signal processing methods that are fragile and require manually tuning. In this work we introduce Spotiflow, a deep-learning method that casts the spot-detection problem as a multiscale stereographic flow regression problem that yields subpixel-accurate localizations. Spotiflow is robust to different noise conditions and generalizes across different chemistries while being up to an order-of-magnitude more time and memory efficient than commonly used methods. We show the efficacy of Spotiflow by comprehensive quantitative comparisons against other methods on a variety of datasets and demonstrate the impact of its increased accuracy on the biological conclusions drawn from iST and live imaging experiments. Spotiflow is available as an easy-to-use Python library as well as a napari plugin at http://www.github.com/weigertlab/spotiflow.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577871">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577871" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577871">
        <p class="paperTitle">StaVia: Spatially and temporally aware cartography with higher order random walks for cell atlases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577871" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577871" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stassen, S. V.; Kobashi, M.; Huang, Y.; Ho, J. W. K.; Tsia, K.</p>
        <p class="info">Score: 71.7, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577871' target='https://doi.org/10.1101/2024.01.29.577871'> 10.1101/2024.01.29.577871</a></p>
        <p class="abstract">Single-cell atlases are critical for unraveling the cellular basis of health and disease, yet their sheer diversity and vast data landscape in time and space pose daunting computational challenges pertaining to the delineation of multiple simultaneously emerging lineages, the integration of spatial and temporal information, and the visualization of trajectories across large atlases with enough resolution to observe localized transitions. To tackle this intricacy, we introduce StaVia, a computational framework that synergizes multi-faceted single-cell data--spanning time-series data, spatial gene expression patterns, and directional trends from RNA velocity-- with higher-order random walks that leverage the memory of cells past states. StaVia fuses this method with a cartographic &#34;Atlas View&#34; that offers intuitive graph visualization, simultaneously capturing the nuanced details of cellular development at single-cell resolution as well as the broader connectivity of cell lineages, avoiding common pitfalls of merged distinct trajectories or missed transitional states seen in existing methods which are all memoryless. Notably, we demonstrate that StaVia unlocks new insights into placode development, radial glia pluripotency during neurulation, and the transitions pivotal to these processes in a large-scale Zebrafish developmental atlas. StaVia also allows spatially aware cartography that captures relationships between cell populations based on their spatial location as well as their gene expressions in a MERFISH dataset - underscoring its potential to dissect complex biological landscapes in both spatial and temporal contexts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578436">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578436" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578436">
        <p class="paperTitle">SEraster: a rasterization preprocessing framework for scalable spatial omics data analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578436" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578436" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aihara, G.; Clifton, K.; Chen, M.; Atta, L.; Miller, B. F.; Fan, J.</p>
        <p class="info">Score: 30.7, Published: 2024-02-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578436' target='https://doi.org/10.1101/2024.02.01.578436'> 10.1101/2024.02.01.578436</a></p>
        <p class="abstract">MotivationSpatial omics data demand computational analysis but many analysis tools have computational resource requirements that increase with the number of cells analyzed. This presents scalability challenges as researchers use spatial omics technologies to profile millions of cells.

ResultsTo enhance the scalability of spatial omics data analysis, we developed a rasterization preprocessing framework called SEraster that aggregates cellular information into spatial pixels. We apply SEraster to both real and simulated spatial omics data prior to spatial variable gene expression analysis to demonstrate that such preprocessing can reduce resource requirements while maintaining high performance. We further integrate SEraster with existing analysis tools to characterize cell-type spatial cooccurrence. Finally, we apply SEraster to enable analysis of a mouse pup spatial omics dataset with over a million cells to identify tissue-level and cell-type-specific spatially variable genes as well as cooccurring cell-types that recapitulate expected organ structures.

Availability and implementationSource code is available on GitHub (https://github.com/JEFworks-Lab/SEraster) with additional tutorials at https://JEF.works/SEraster.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.03.576026">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.03.576026" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.03.576026">
        <p class="paperTitle">BiaPy: A unified framework for versatile bioimage analysis with deep learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.03.576026" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.03.576026" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Franco-Barranco, D.; Andres-San Roman, J. A.; Hidalgo-Cenalmor, I.; Backova, L.; Gonzalez-Marfil, A.; Caporal, C.; Chessel, A.; Gomez-Galvez, P.; Escudero, L. M.; Wei, D.; Munoz-Barrutia, A.; Arganda-Carreras, I.</p>
        <p class="info">Score: 25.9, Published: 2024-02-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.03.576026' target='https://doi.org/10.1101/2024.02.03.576026'> 10.1101/2024.02.03.576026</a></p>
        <p class="abstract">BiaPy, a unified open-source bioimage analysis library, offers a comprehensive suite of deep learning-powered workflows. Tailored for users of all levels, BiaPy features an intuitive interface, zero-code notebooks, and Docker integration. With support for 2D and 3D image data, it addresses existing gaps by providing multi-GPU capabilities, memory optimization, and compatibility with large datasets. As a collaborative and accessible solution, BiaPy aims to empower researchers by democratizing the use of sophisticated and efficient bioimage analysis workflows.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578496">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578496" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578496">
        <p class="paperTitle">GenerRNA: A generative pre-trained language model for de novo RNA design</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578496" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578496" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhao, Y.; Oono, K.; Takizawa, H.; Kotera, M.</p>
        <p class="info">Score: 22.4, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578496' target='https://doi.org/10.1101/2024.02.01.578496'> 10.1101/2024.02.01.578496</a></p>
        <p class="abstract">The design of RNA plays a crucial role in developing RNA vaccines, nucleic acid therapeutics, and innovative biotechnological tools. Nevertheless, existing techniques lack versatility across various tasks and frequently suffer from a deficiency of automated generation. Inspired by the remarkable success of Large Language Models (LLMs) in the realm of protein and molecule design, we present GenerRNA, the first large-scale pre-trained model for RNA generation, aiming to further automate RNA design. Our approach eliminates the need for secondary structure or other prior knowledge and is capable of de novo generation of RNA with stable secondary structures while ensuring its distinctiveness from existing sequences. This widens our exploration of RNA space, thereby enriching our understanding of RNA structures and functions. Moreover, GenerRNA is fine-tunable on smaller, more specialized datasets for particular subtasks. This flexibility and versatility enables the generation of RNAs with desired specific functionalities or properties. Upon fine-tuning GenerRNA, we successfully generated novel RNA sequences exhibiting high affinity for target proteins. GenerRNA is freely available at the following repository: https://github.com/pfnet-research/GenerRNA</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.577813">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.577813" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.577813">
        <p class="paperTitle">Forseti: A mechanistic and predictive model of the splicing status of scRNA-seq reads</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.577813" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.577813" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: He, D.; Gao, Y.; Chan, S. S.; Quintana-Parrilla, N.; Patro, R.</p>
        <p class="info">Score: 21.4, Published: 2024-02-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.577813' target='https://doi.org/10.1101/2024.02.01.577813'> 10.1101/2024.02.01.577813</a></p>
        <p class="abstract">MotivationShort-read single-cell RNA-sequencing (scRNA-seq) has been used to study cellular heterogeneity, cellular fate, and transcriptional dynamics. Modeling splicing dynamics in scRNA-seq data is challenging, with inherent difficulty in even the seemingly straightforward task of elucidating the splicing status of the molecules from which sequenced fragments are drawn. This difficulty arises, in part, from the limited read length and positional biases, which substantially reduce the specificity of the sequenced fragments. As a result, the splicing status of many reads in scRNA-seq is ambiguous because of a lack of definitive evidence. We are therefore in need of methods that can recover the splicing status of ambiguous reads which, in turn, can lead to more accuracy and confidence in downstream analyses.

ResultsWe develop Forseti, a predictive model to probabilistically assign a splicing status to scRNA-seq reads. Our model has two key components. First, we train a binding affinity model to assign a probability that a given transcriptomic site is used in fragment generation. Second, we fit a robust fragment length distribution model that generalizes well across datasets deriving from different species and tissue types. Forseti combines these two trained models to predict the splicing status of the molecule of origin of reads by scoring putative fragments that associate each alignment of sequenced reads with proximate potential priming sites. Using both simulated and experimental data, we show that our model can precisely predict the splicing status of reads and identify the true gene origin of multi-gene mapped reads.

AvailabilityForseti and the code used for producing the results are available at https://github.com/COMBINE-lab/forseti under a BSD 3-clause license.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.07.579356">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.07.579356" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.07.579356">
        <p class="paperTitle">Multi-genome synteny detection using minimizer graph mappings</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.07.579356" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.07.579356" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Coombe, L.; Kazemi, P.; Wong, J.; Birol, I.; Warren, R. L.</p>
        <p class="info">Score: 17.6, Published: 2024-02-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.07.579356' target='https://doi.org/10.1101/2024.02.07.579356'> 10.1101/2024.02.07.579356</a></p>
        <p class="abstract">In recent years, the landscape of reference-grade genome assemblies has seen substantial diversification. With such rich data, there is pressing demand for robust tools for scalable, multi-species comparative genomics analyses, including detecting genome synteny, which informs on the sequence conservation between genomes and contributes crucial insights into species evolution. Here, we introduce ntSynt, a scalable utility for computing large-scale multi-genome synteny blocks using a minimizer graph-based approach. Through extensive testing utilizing multiple ~3 Gbp genomes, we demonstrate how ntSynt produces synteny blocks with coverages between 79-100% in at most 2h using 34 GB of memory, even for genomes with appreciable (&gt;15%) sequence divergence. Compared to existing state-of-the-art methodologies, ntSynt offers enhanced flexibility to diverse input genome sequences and synteny block granularity. We expect the macrosyntenic genome analyses facilitated by ntSynt will have broad utility in generating critical evolutionary insights within and between species across the tree of life.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577767">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577767" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577767">
        <p class="paperTitle">LazyAF, a pipeline for accessible medium-scale in silico prediction of protein-protein interactions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577767" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577767" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McLean, T. C.</p>
        <p class="info">Score: 104.3, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577767' target='https://doi.org/10.1101/2024.01.29.577767'> 10.1101/2024.01.29.577767</a></p>
        <p class="abstract">Artificial intelligence has revolutionized the field of protein structure prediction. However, with more powerful and complex software being developed, it is accessibility and ease of use rather than capability that is quickly becoming a limiting factor to end users. Here, I present a Google Colaboratory-based pipeline, named LazyAF, which integrates the existing ColabFold BATCH to streamline the process of medium-scale protein-protein interaction prediction. I apply LazyAF to predict the interactome of the 76 proteins encoded on a broad-host-range multi-drug resistance plasmid RK2, demonstrating the ease and accessibility the pipeline provides.

AvailabilityLazyAF is freely available at https://github.com/ThomasCMcLean/LazyAF</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
