<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="DrugGPT: A GPT-based Strategy for Designing Potential Ligands Targeting Specific Proteins
Authors: Li, Y.; Gao, C.; Song, X.; Wang, X.; Xu, Y.; Han, S.
Score: 35.7, Published: 2023-06-30 DOI: 10.1101/2023.06.29.543848
DrugGPT presents a ligand design strategy based on the autoregressive model, GPT, focusing on chemical space exploration and the discovery of ligands for specific proteins. Deep learning language models have shown significant potential in various domains including protein design and biomedical text analysis, providing strong support for the proposition of DrugGPT.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="DrugGPT: A GPT-based Strategy for Designing Potential Ligands Targeting Specific Proteins
Authors: Li, Y.; Gao, C.; Song, X.; Wang, X.; Xu, Y.; Han, S.
Score: 35.7, Published: 2023-06-30 DOI: 10.1101/2023.06.29.543848
DrugGPT presents a ligand design strategy based on the autoregressive model, GPT, focusing on chemical space exploration and the discovery of ligands for specific proteins. Deep learning language models have shown significant potential in various domains including protein design and biomedical text analysis, providing strong support for the proposition of DrugGPT." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-05T10:42:43+00:00" />
<meta property="article:modified_time" content="2023-07-05T10:42:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="DrugGPT: A GPT-based Strategy for Designing Potential Ligands Targeting Specific Proteins
Authors: Li, Y.; Gao, C.; Song, X.; Wang, X.; Xu, Y.; Han, S.
Score: 35.7, Published: 2023-06-30 DOI: 10.1101/2023.06.29.543848
DrugGPT presents a ligand design strategy based on the autoregressive model, GPT, focusing on chemical space exploration and the discovery of ligands for specific proteins. Deep learning language models have shown significant potential in various domains including protein design and biomedical text analysis, providing strong support for the proposition of DrugGPT."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "DrugGPT: A GPT-based Strategy for Designing Potential Ligands Targeting Specific Proteins\nAuthors: Li, Y.; Gao, C.; Song, X.; Wang, X.; Xu, Y.; Han, S.\nScore: 35.7, Published: 2023-06-30 DOI: 10.1101/2023.06.29.543848\nDrugGPT presents a ligand design strategy based on the autoregressive model, GPT, focusing on chemical space exploration and the discovery of ligands for specific proteins. Deep learning language models have shown significant potential in various domains including protein design and biomedical text analysis, providing strong support for the proposition of DrugGPT.",
  "keywords": [
    
  ],
  "articleBody": " DrugGPT: A GPT-based Strategy for Designing Potential Ligands Targeting Specific Proteins\nAuthors: Li, Y.; Gao, C.; Song, X.; Wang, X.; Xu, Y.; Han, S.\nScore: 35.7, Published: 2023-06-30 DOI: 10.1101/2023.06.29.543848\nDrugGPT presents a ligand design strategy based on the autoregressive model, GPT, focusing on chemical space exploration and the discovery of ligands for specific proteins. Deep learning language models have shown significant potential in various domains including protein design and biomedical text analysis, providing strong support for the proposition of DrugGPT. In this study, we employ the DrugGPT model to learn a substantial amount of protein-ligand binding data, aiming to discover novel molecules that can bind with specific proteins. This strategy not only significantly improves the efficiency of ligand design but also offers a swift and effective avenue for the drug development process, bringing new possibilities to the pharmaceutical domain. In our research, we particularly optimized and trained the GPT-2 model to better adapt to the requirements of drug design. Given the characteristics of proteins and ligands, we redesigned the tokenizer using the BPE algorithm, abandoned the original tokenizer, and trained the GPT-2 model from scratch. This improvement enables DrugGPT to more accurately capture and understand the structural information and chemical rules of drug molecules. It also enhances its comprehension of binding information between proteins and ligands, thereby generating potentially active drug candidate molecules. Theoretically, DrugGPT has significant advantages. During the model training process, DrugGPT aims to maximize the conditional probability and employs the back-propagation algorithm for training, making the training process more stable and avoiding the Mode Collapse problem that may occur in Generative Adversarial Networks in drug design. Furthermore, the design philosophy of DrugGPT endows it with strong generalization capabilities, giving it the potential to adapt to different tasks. In conclusion, DrugGPT provides a forward-thinking and practical new approach to ligand design. By optimizing the tokenizer and retraining the GPT-2 model, the ligand design process becomes more direct and efficient. This not only reflects the theoretical advantages of DrugGPT but also reveals its potential applications in the drug development process, thereby opening new perspectives and possibilities in the pharmaceutical field.\nFlexible Protein-Protein Docking with a Multi-Track Iterative Transformer\nAuthors: Chu, L.-S.; Ruffolo, J. A.; Harmalkar, A.; Gray, J. J.\nScore: 23.8, Published: 2023-07-01 DOI: 10.1101/2023.06.29.547134\nConventional protein-protein docking algorithms usually rely on heavy candidate sampling and re-ranking, but these steps are time-consuming and hinder applications that require high-throughput complex structure prediction, e.g., structure-based virtual screening. Existing deep learning methods for protein-protein docking, despite being much faster, suffer from low docking success rates. In addition, they simplify the problem to assume no conformational changes within any protein upon binding (rigid docking). This assumption precludes applications when binding-induced conformational changes play a role, such as allosteric inhibition or docking from uncertain unbound model structures. To address these limitations, we present GeoDock, a multi-track iterative transformer network to predict a docked structure from separate docking partners. Unlike deep learning models for protein structure prediction that input multiple sequence alignments (MSAs), GeoDock inputs just the sequences and structures of the docking partners, which suits the tasks when the individual structures are given. GeoDock is flexible at the protein residue level, allowing the prediction of conformational changes upon binding. For a benchmark set of rigid targets, GeoDock obtains a 41% success rate, outperforming all the other tested methods. For a more challenging benchmark set of flexible targets, GeoDock achieves a similar number of top-model successes as the traditional method ClusPro, but fewer than ReplicaDock2. GeoDock attains an average inference speed of under one second on a single GPU, enabling its application in large-scale structure screening. Although binding-induced conformational changes are still a challenge owing to limited training and evaluation data, our architecture sets up the foundation to capture this backbone flexibility. Code and a demonstration Jupyter notebook are available at https://github.com/Graylab/GeoDock.\nBGCFlow: Systematic pangenome workflow for the analysis of biosynthetic gene clusters across large genomic datasets\nAuthors: Nuhamunada, M.; Mohite, O. S.; Phaneuf, P. V.; Palsson, B.; Weber, T.\nScore: 39.5, Published: 2023-06-15 DOI: 10.1101/2023.06.14.545018\nGenome mining is revolutionizing natural products discovery efforts. The rapid increase in available genomes demands comprehensive computational platforms to effectively extract biosynthetic knowledge encoded across bacterial pangenomes. Here, we present BGCFlow, a novel systematic workflow integrating analytics for large-scale genome mining of bacterial pangenomes. BGCFlow incorporates several genome analytics and mining tools grouped into five common stages of analysis such as; i) data selection, ii) functional annotation, iii) phylogenetic analysis, iv) genome mining, and v) comparative analysis. Furthermore, BGCFlow provides easy configuration of different projects, parallel distribution, scheduled job monitoring, an interactive database to visualize tables, exploratory Jupyter notebooks, and customized reports. Here, we demonstrate the application of BGCFlow by investigating the phylogenetic distribution of various biosynthetic gene clusters detected across 42 genomes of the Saccharopolyspora genus, known to produce industrially important secondary/specialized metabolites. The BGCFlow-guided analysis predicted more accurate dereplication of BGCs and guided the targeted comparative analysis of selected RiPPs. The scalable, interoperable, adaptable, re-entrant, and reproducible nature of the BGCFlow will provide an effective novel way to extract the biosynthetic knowledge in the ever-growing genomic datasets of biotechnologically relevant bacterial species. BGCFlow is available for downloading at https://github.com/NBChub/bgcflow.\nColabDock: inverting AlphaFold structure prediction model for protein-protein docking with experimental restraints\nAuthors: Feng, S.; Chen, Z.; Zhang, C.; Xie, Y.; Ovchinnikov, S.; Gao, Y. Q.; Liu, S.\nScore: 22.0, Published: 2023-07-04 DOI: 10.1101/2023.07.04.547599\nPrediction of protein complex structures and interfaces potentially has wide applications and can benefit the study of biological mechanisms involving protein-protein interactions. However, the surface prediction accuracy of traditional docking methods and AlphaFold-Multimer is limited. Here we present ColabDock, a framework that makes use of ColabDesign, but reimplements it for the purpose of restrained complex conformation prediction. With a generation-prediction architecture and trained ranking model, ColabDock outperforms HADDOCK and ClusPro not only in complex structure predictions with simulated residue and surface restraints, but also in those assisted by NMR chemical shift perturbation as well as covalent labeling. It further assists antibody-antigen interface prediction with emulated interface scan restraints, which could be obtained by experiments such as Deep Mutation Scan. ColabDock provides a general approach to integrate sparse interface restraints of different experimental forms and sources into one optimization framework.\nFractional Hitting Sets for Efficient and Lightweight Genomic Data Sketching\nAuthors: Rouze, T.; Martayan, I.; MARCHET, C.; Limasset, A.\nScore: 14.9, Published: 2023-06-24 DOI: 10.1101/2023.06.21.545875\nThe exponential increase in publicly available sequencing data and genomic resources necessitates the development of highly efficient methods for data processing and analysis. Locality-sensitive hashing techniques have successfully transformed large datasets into smaller, more manageable sketches while maintaining comparability using metrics such as Jaccard and containment indices. However, fixed-size sketches encounter difficulties when applied to divergent datasets. Scalable sketching methods, such as Sourmash, provide valuable solutions but still lack resourceefficient, tailored indexing. Our objective is to create lighter sketches with comparable results while enhancing efficiency. We introduce the concept of Fractional Hitting Sets, a generalization of Universal Hitting Sets, which uniformly cover a specified fraction of the k-mer space. In theory and practice, we demonstrate the feasibility of achieving such coverage with simple but highly efficient schemes. By encoding the covered k-mers as super-k-mers, we provide a space-efficient exact representation that also enables optimized comparisons. Our novel tool, SuperSampler, implements this scheme, and experimental results with real bacterial collections closely match our theoretical findings. In comparison to Sourmash, SuperSampler achieves similar outcomes while utilizing an order of magnitude less space and memory and operating several times faster. This highlights the potential of our approach in addressing the challenges presented by the ever-expanding landscape of genomic data. SuperSampler is an open-source software and can be accessed at github.com/TimRouze/supersampler. The data required to reproduce the results presented in this manuscript is available at github.com/TimRouze/Expe_SPSP.\nImproved protein complex prediction with AlphaFold-multimer by denoising the MSA profile\nAuthors: Bryant, P.; Noe, F.\nScore: 14.5, Published: 2023-07-04 DOI: 10.1101/2023.07.04.547638\nStructure prediction of protein complexes has improved significantly with AlphaFold2 and AlphaFold-multimer (AFM), but only 60% of dimers are accurately predicted. A way to improve the predictions is to inject noise to generate more diverse predictions. However, thousands of predictions are needed to obtain a few that are accurate in difficult cases. Here, we learn a bias to the MSA representation that improves the predictions by performing gradient descent through the AFM network. We effectively denoise the MSA profile, similar to how a blurry image would be sharpened. We demonstrate the performance on seven difficult targets from CASP15 and increase the average MMscore to 0.76 compared to 0.63 with AFM. We evaluate the procedure on 334 protein complexes where AFM fails and demonstrate an increased success rate (MMscore\u003e0.75) of 8% on these hard targets. Our protocol, AFProfile, provides a way to direct predictions towards a defined target function guided by the MSA. We expect gradient descent over the MSA to be useful for different tasks, such as generating alternative conformations. AFProfile is freely available at: https://github.com/patrickbryant1/AFProfile\nJuggling offsets unlocks RNA-seq tools for fast scalable differential usage, aberrant splicing and expression analyses.\nAuthors: Segers, A.; Gilis, J.; Van Heetvelde, M.; De Baere, E.; Clement, L.\nScore: 14.0, Published: 2023-07-01 DOI: 10.1101/2023.06.29.547014\nRNA-sequencing (RNA-seq) is increasingly used to diagnose patients with rare diseases by prioritising genes with aberrant expression and/or splicing. State-of-the-art methods for detecting aberrant expression and splicing, however, are extremely slow. The latter, also discard much information because they only use junction reads to infer aberrant splicing. In this contribution, we show that replacing the offset for library size unlocks conventional bulk RNA-seq workflows for fast and scalable differential usage, aberrant splicing and expression analyses. Our method, saseR, is several orders of magnitude faster than the state-of-the-art methods and dramatically outperforms these in terms of sensitivity and specificity for aberrant splicing, while being on par with these inferring differential usage and aberrant expression. Finally, our framework is also very flexible and can be used for all applications that involve the analysis of proportions of short- or long RNA-seq read counts.\nOPUS-Fold3: a gradient-based protein all-atom folding and docking framework on TensorFlow\nAuthors: Xu, G.; Luo, Z.; Zhou, R.; Wang, Q.; Ma, J.\nScore: 14.5, Published: 2023-07-02 DOI: 10.1101/2022.08.31.506128\nFor refining and designing protein structures, it is essential to have an efficient protein folding and docking framework that generates a protein 3D structure based on given constraints. In this study, we introduce OPUS-Fold3 as a gradient-based, all-atom protein folding and docking framework, which accurately generates 3D protein structures in compliance with specified constraints, such as a potential function as long as it can be expressed as a function of positions of heavy atoms. Our tests show that, for example, OPUS-Fold3 achieves performance comparable to pyRosetta in backbone folding, and significantly better in side-chain modeling. Developed using Python and TensorFlow 2.4, OPUS-Fold3 is user-friendly for any source-code level modifications and can be seamlessly combined with other deep learning models, thus facilitating collaboration between the biology and AI communities. The source code of OPUS-Fold3 can be downloaded from http://github.com/OPUS-MaLab/opus_fold3. It is freely available for academic usage.\nScalable Integration of Multiomic Single Cell Data Using Generative Adversarial Networks\nAuthors: Giansanti, V.; Giannese, F.; Botrugno, O. A.; Gandolfi, G.; Balestrieri, C.; Antoniotti, M.; Tonon, G.; Cittaro, D.\nScore: 11.4, Published: 2023-06-29 DOI: 10.1101/2023.06.26.546547\nSingle cell profiling has become a common practice to investigate the complexity of tissues, organs and organisms. Recent technological advances are expanding our capabilities to profile various molecular layers beyond the transcriptome such as, but not limited to, the genome, the epigenome and the proteome. Depending on the experimental procedure, these data can be obtained from separate assays or from the very same cells. Despite development of computational methods for data integration is an active research field, most of the available strategies have been devised for the joint analysis of two modalities and cannot accommodate a high number of them. To solve this problem, we here propose a multiomic data integration framework based on Wasserstein Generative Adversarial Networks (MOWGAN) suitable for the analysis of paired or unpaired data with high number of modalities (\u003e2). At the core of our strategy is a single network trained on all modalities together, limiting the computational burden when many molecular layers are evaluated. Source code of our framework is available at https://github.com/vgiansanti/MOWGAN.\nNew genome scale network modeling and mining workflow for detecting metabolic changes induced by exposure to chemicals\nAuthors: Fresnais, L.; Perin, O.; Riu, A.; Grall, R.; Ott, A.; Fromenty, B.; Gallardo, J. C.; Stingl, M.; Frainay, C.; Jourdan, F.; Poupin, N.\nScore: 11.2, Published: 2023-07-02 DOI: 10.1101/2023.06.30.547200\nThe growing abundance of in vitro omics data, coupled with the necessity to reduce animal testing in the safety assessment of chemical compounds and even eliminate it in the evaluation of cosmetics, highlights the need for abundant computational methodologies. Data from omics technologies allow the exploration of a wide range of biological processes, therefore providing a better understanding of mechanisms of action (MoA) related to chemical exposure in biological systems. However, the analysis of these large datasets remains difficult due to the complexity of modulations spanning multiple biological processes. To address this, we propose a new computational workflow that combines knowledge on endogenous metabolism from a genome scale metabolic network (GSMN) and in vitro transcriptomics data with the aim of better identifying the metabolic MoA (mMoA) of chemicals. Our workflow proceeds in three main steps. The first step consists of building cell condition-specific models representing the metabolic impact of each exposure condition while taking into account the diversity of possible optimal solutions with a partial enumeration algorithm. In a second step, based on these enumerations, two conditions can be compared by extracting differentially activated reactions (DARs) between the two sets of enumerated possible condition-specific models. Finally, in the third step, DARs are grouped into clusters of functionally interconnected metabolic reactions using the distance-based clustering and subnetwork extraction method. The first part of the workflow was exemplified on eight molecules selected for their known human hepatotoxic outcomes associated with specific MoAs well described in the literature and for which we retrieved primary human hepatocytes (PHH) transcriptomic data in Open TG-GATEs. Then, we applied this new workflow to model and visualize associated mMoA for two of these eight molecules (amiodarone and valproic acid). Despite large disparities in transcriptomic effects for these two chemicals, i.e., two differentially expressed genes (DEGs) for amiodarone vs 5709 DEGs for valproic acid, our results well fitted evidence from the literature regarding known MoA. Beyond these confirmations, the workflow highlighted potential other unexplored mMoA.\n",
  "wordCount" : "2414",
  "inLanguage": "en",
  "datePublished": "2023-07-05T10:42:43Z",
  "dateModified": "2023-07-05T10:42:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on July 5, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.543848">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.543848" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.543848">
        <p class="paperTitle">DrugGPT: A GPT-based Strategy for Designing Potential Ligands Targeting Specific Proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.543848" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.543848" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, Y.; Gao, C.; Song, X.; Wang, X.; Xu, Y.; Han, S.</p>
        <p class="info">Score: 35.7, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.543848' target='https://doi.org/10.1101/2023.06.29.543848'> 10.1101/2023.06.29.543848</a></p>
        <p class="abstract">DrugGPT presents a ligand design strategy based on the autoregressive model, GPT, focusing on chemical space exploration and the discovery of ligands for specific proteins. Deep learning language models have shown significant potential in various domains including protein design and biomedical text analysis, providing strong support for the proposition of DrugGPT. In this study, we employ the DrugGPT model to learn a substantial amount of protein-ligand binding data, aiming to discover novel molecules that can bind with specific proteins. This strategy not only significantly improves the efficiency of ligand design but also offers a swift and effective avenue for the drug development process, bringing new possibilities to the pharmaceutical domain. In our research, we particularly optimized and trained the GPT-2 model to better adapt to the requirements of drug design. Given the characteristics of proteins and ligands, we redesigned the tokenizer using the BPE algorithm, abandoned the original tokenizer, and trained the GPT-2 model from scratch. This improvement enables DrugGPT to more accurately capture and understand the structural information and chemical rules of drug molecules. It also enhances its comprehension of binding information between proteins and ligands, thereby generating potentially active drug candidate molecules. Theoretically, DrugGPT has significant advantages. During the model training process, DrugGPT aims to maximize the conditional probability and employs the back-propagation algorithm for training, making the training process more stable and avoiding the Mode Collapse problem that may occur in Generative Adversarial Networks in drug design. Furthermore, the design philosophy of DrugGPT endows it with strong generalization capabilities, giving it the potential to adapt to different tasks. In conclusion, DrugGPT provides a forward-thinking and practical new approach to ligand design. By optimizing the tokenizer and retraining the GPT-2 model, the ligand design process becomes more direct and efficient. This not only reflects the theoretical advantages of DrugGPT but also reveals its potential applications in the drug development process, thereby opening new perspectives and possibilities in the pharmaceutical field.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547134">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547134" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547134">
        <p class="paperTitle">Flexible Protein-Protein Docking with a Multi-Track Iterative Transformer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547134" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547134" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chu, L.-S.; Ruffolo, J. A.; Harmalkar, A.; Gray, J. J.</p>
        <p class="info">Score: 23.8, Published: 2023-07-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547134' target='https://doi.org/10.1101/2023.06.29.547134'> 10.1101/2023.06.29.547134</a></p>
        <p class="abstract">Conventional protein-protein docking algorithms usually rely on heavy candidate sampling and re-ranking, but these steps are time-consuming and hinder applications that require high-throughput complex structure prediction, e.g., structure-based virtual screening. Existing deep learning methods for protein-protein docking, despite being much faster, suffer from low docking success rates. In addition, they simplify the problem to assume no conformational changes within any protein upon binding (rigid docking). This assumption precludes applications when binding-induced conformational changes play a role, such as allosteric inhibition or docking from uncertain unbound model structures. To address these limitations, we present GeoDock, a multi-track iterative transformer network to predict a docked structure from separate docking partners. Unlike deep learning models for protein structure prediction that input multiple sequence alignments (MSAs), GeoDock inputs just the sequences and structures of the docking partners, which suits the tasks when the individual structures are given. GeoDock is flexible at the protein residue level, allowing the prediction of conformational changes upon binding. For a benchmark set of rigid targets, GeoDock obtains a 41% success rate, outperforming all the other tested methods. For a more challenging benchmark set of flexible targets, GeoDock achieves a similar number of top-model successes as the traditional method ClusPro, but fewer than ReplicaDock2. GeoDock attains an average inference speed of under one second on a single GPU, enabling its application in large-scale structure screening. Although binding-induced conformational changes are still a challenge owing to limited training and evaluation data, our architecture sets up the foundation to capture this backbone flexibility. Code and a demonstration Jupyter notebook are available at https://github.com/Graylab/GeoDock.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.14.545018">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.14.545018" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.14.545018">
        <p class="paperTitle">BGCFlow: Systematic pangenome workflow for the analysis of biosynthetic gene clusters across large genomic datasets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.14.545018" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.14.545018" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nuhamunada, M.; Mohite, O. S.; Phaneuf, P. V.; Palsson, B.; Weber, T.</p>
        <p class="info">Score: 39.5, Published: 2023-06-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.14.545018' target='https://doi.org/10.1101/2023.06.14.545018'> 10.1101/2023.06.14.545018</a></p>
        <p class="abstract">Genome mining is revolutionizing natural products discovery efforts. The rapid increase in available genomes demands comprehensive computational platforms to effectively extract biosynthetic knowledge encoded across bacterial pangenomes. Here, we present BGCFlow, a novel systematic workflow integrating analytics for large-scale genome mining of bacterial pangenomes. BGCFlow incorporates several genome analytics and mining tools grouped into five common stages of analysis such as; i) data selection, ii) functional annotation, iii) phylogenetic analysis, iv) genome mining, and v) comparative analysis. Furthermore, BGCFlow provides easy configuration of different projects, parallel distribution, scheduled job monitoring, an interactive database to visualize tables, exploratory Jupyter notebooks, and customized reports. Here, we demonstrate the application of BGCFlow by investigating the phylogenetic distribution of various biosynthetic gene clusters detected across 42 genomes of the Saccharopolyspora genus, known to produce industrially important secondary/specialized metabolites. The BGCFlow-guided analysis predicted more accurate dereplication of BGCs and guided the targeted comparative analysis of selected RiPPs. The scalable, interoperable, adaptable, re-entrant, and reproducible nature of the BGCFlow will provide an effective novel way to extract the biosynthetic knowledge in the ever-growing genomic datasets of biotechnologically relevant bacterial species. BGCFlow is available for downloading at https://github.com/NBChub/bgcflow.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547599">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547599" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547599">
        <p class="paperTitle">ColabDock: inverting AlphaFold structure prediction model for protein-protein docking with experimental restraints</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547599" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547599" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Feng, S.; Chen, Z.; Zhang, C.; Xie, Y.; Ovchinnikov, S.; Gao, Y. Q.; Liu, S.</p>
        <p class="info">Score: 22.0, Published: 2023-07-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547599' target='https://doi.org/10.1101/2023.07.04.547599'> 10.1101/2023.07.04.547599</a></p>
        <p class="abstract">Prediction of protein complex structures and interfaces potentially has wide applications and can benefit the study of biological mechanisms involving protein-protein interactions. However, the surface prediction accuracy of traditional docking methods and AlphaFold-Multimer is limited. Here we present ColabDock, a framework that makes use of ColabDesign, but reimplements it for the purpose of restrained complex conformation prediction. With a generation-prediction architecture and trained ranking model, ColabDock outperforms HADDOCK and ClusPro not only in complex structure predictions with simulated residue and surface restraints, but also in those assisted by NMR chemical shift perturbation as well as covalent labeling. It further assists antibody-antigen interface prediction with emulated interface scan restraints, which could be obtained by experiments such as Deep Mutation Scan. ColabDock provides a general approach to integrate sparse interface restraints of different experimental forms and sources into one optimization framework.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545875">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545875" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545875">
        <p class="paperTitle">Fractional Hitting Sets for Efficient and Lightweight Genomic Data Sketching</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545875" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545875" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rouze, T.; Martayan, I.; MARCHET, C.; Limasset, A.</p>
        <p class="info">Score: 14.9, Published: 2023-06-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545875' target='https://doi.org/10.1101/2023.06.21.545875'> 10.1101/2023.06.21.545875</a></p>
        <p class="abstract">The exponential increase in publicly available sequencing data and genomic resources necessitates the development of highly efficient methods for data processing and analysis. Locality-sensitive hashing techniques have successfully transformed large datasets into smaller, more manageable sketches while maintaining comparability using metrics such as Jaccard and containment indices. However, fixed-size sketches encounter difficulties when applied to divergent datasets.

Scalable sketching methods, such as Sourmash, provide valuable solutions but still lack resourceefficient, tailored indexing. Our objective is to create lighter sketches with comparable results while enhancing efficiency. We introduce the concept of Fractional Hitting Sets, a generalization of Universal Hitting Sets, which uniformly cover a specified fraction of the k-mer space. In theory and practice, we demonstrate the feasibility of achieving such coverage with simple but highly efficient schemes.

By encoding the covered k-mers as super-k-mers, we provide a space-efficient exact representation that also enables optimized comparisons. Our novel tool, SuperSampler, implements this scheme, and experimental results with real bacterial collections closely match our theoretical findings.

In comparison to Sourmash, SuperSampler achieves similar outcomes while utilizing an order of magnitude less space and memory and operating several times faster. This highlights the potential of our approach in addressing the challenges presented by the ever-expanding landscape of genomic data.

SuperSampler is an open-source software and can be accessed at github.com/TimRouze/supersampler. The data required to reproduce the results presented in this manuscript is available at github.com/TimRouze/Expe_SPSP.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547638">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547638" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547638">
        <p class="paperTitle">Improved protein complex prediction with AlphaFold-multimer by denoising the MSA profile</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547638" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547638" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bryant, P.; Noe, F.</p>
        <p class="info">Score: 14.5, Published: 2023-07-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547638' target='https://doi.org/10.1101/2023.07.04.547638'> 10.1101/2023.07.04.547638</a></p>
        <p class="abstract">Structure prediction of protein complexes has improved significantly with AlphaFold2 and AlphaFold-multimer (AFM), but only 60% of dimers are accurately predicted. A way to improve the predictions is to inject noise to generate more diverse predictions. However, thousands of predictions are needed to obtain a few that are accurate in difficult cases. Here, we learn a bias to the MSA representation that improves the predictions by performing gradient descent through the AFM network. We effectively denoise the MSA profile, similar to how a blurry image would be sharpened. We demonstrate the performance on seven difficult targets from CASP15 and increase the average MMscore to 0.76 compared to 0.63 with AFM. We evaluate the procedure on 334 protein complexes where AFM fails and demonstrate an increased success rate (MMscore&gt;0.75) of 8% on these hard targets. Our protocol, AFProfile, provides a way to direct predictions towards a defined target function guided by the MSA. We expect gradient descent over the MSA to be useful for different tasks, such as generating alternative conformations. AFProfile is freely available at: https://github.com/patrickbryant1/AFProfile</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547014">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547014" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547014">
        <p class="paperTitle">Juggling offsets unlocks RNA-seq tools for fast scalable differential usage, aberrant splicing and expression analyses.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547014" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547014" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Segers, A.; Gilis, J.; Van Heetvelde, M.; De Baere, E.; Clement, L.</p>
        <p class="info">Score: 14.0, Published: 2023-07-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547014' target='https://doi.org/10.1101/2023.06.29.547014'> 10.1101/2023.06.29.547014</a></p>
        <p class="abstract">RNA-sequencing (RNA-seq) is increasingly used to diagnose patients with rare diseases by prioritising genes with aberrant expression and/or splicing. State-of-the-art methods for detecting aberrant expression and splicing, however, are extremely slow. The latter, also discard much information because they only use junction reads to infer aberrant splicing. In this contribution, we show that replacing the offset for library size unlocks conventional bulk RNA-seq workflows for fast and scalable differential usage, aberrant splicing and expression analyses. Our method, saseR, is several orders of magnitude faster than the state-of-the-art methods and dramatically outperforms these in terms of sensitivity and specificity for aberrant splicing, while being on par with these inferring differential usage and aberrant expression. Finally, our framework is also very flexible and can be used for all applications that involve the analysis of proportions of short- or long RNA-seq read counts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.08.31.506128">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.08.31.506128" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.08.31.506128">
        <p class="paperTitle">OPUS-Fold3: a gradient-based protein all-atom folding and docking framework on TensorFlow</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.08.31.506128" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.08.31.506128" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xu, G.; Luo, Z.; Zhou, R.; Wang, Q.; Ma, J.</p>
        <p class="info">Score: 14.5, Published: 2023-07-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.08.31.506128' target='https://doi.org/10.1101/2022.08.31.506128'> 10.1101/2022.08.31.506128</a></p>
        <p class="abstract">For refining and designing protein structures, it is essential to have an efficient protein folding and docking framework that generates a protein 3D structure based on given constraints. In this study, we introduce OPUS-Fold3 as a gradient-based, all-atom protein folding and docking framework, which accurately generates 3D protein structures in compliance with specified constraints, such as a potential function as long as it can be expressed as a function of positions of heavy atoms. Our tests show that, for example, OPUS-Fold3 achieves performance comparable to pyRosetta in backbone folding, and significantly better in side-chain modeling. Developed using Python and TensorFlow 2.4, OPUS-Fold3 is user-friendly for any source-code level modifications and can be seamlessly combined with other deep learning models, thus facilitating collaboration between the biology and AI communities. The source code of OPUS-Fold3 can be downloaded from http://github.com/OPUS-MaLab/opus_fold3. It is freely available for academic usage.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546547">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546547" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546547">
        <p class="paperTitle">Scalable Integration of Multiomic Single Cell Data Using Generative Adversarial Networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546547" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546547" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Giansanti, V.; Giannese, F.; Botrugno, O. A.; Gandolfi, G.; Balestrieri, C.; Antoniotti, M.; Tonon, G.; Cittaro, D.</p>
        <p class="info">Score: 11.4, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546547' target='https://doi.org/10.1101/2023.06.26.546547'> 10.1101/2023.06.26.546547</a></p>
        <p class="abstract">Single cell profiling has become a common practice to investigate the complexity of tissues, organs and organisms. Recent technological advances are expanding our capabilities to profile various molecular layers beyond the transcriptome such as, but not limited to, the genome, the epigenome and the proteome. Depending on the experimental procedure, these data can be obtained from separate assays or from the very same cells. Despite development of computational methods for data integration is an active research field, most of the available strategies have been devised for the joint analysis of two modalities and cannot accommodate a high number of them.

To solve this problem, we here propose a multiomic data integration framework based on Wasserstein Generative Adversarial Networks (MOWGAN) suitable for the analysis of paired or unpaired data with high number of modalities (&gt;2). At the core of our strategy is a single network trained on all modalities together, limiting the computational burden when many molecular layers are evaluated. Source code of our framework is available at https://github.com/vgiansanti/MOWGAN.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547200">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547200" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547200">
        <p class="paperTitle">New genome scale network modeling and mining workflow for detecting metabolic changes induced by exposure to chemicals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547200" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547200" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fresnais, L.; Perin, O.; Riu, A.; Grall, R.; Ott, A.; Fromenty, B.; Gallardo, J. C.; Stingl, M.; Frainay, C.; Jourdan, F.; Poupin, N.</p>
        <p class="info">Score: 11.2, Published: 2023-07-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547200' target='https://doi.org/10.1101/2023.06.30.547200'> 10.1101/2023.06.30.547200</a></p>
        <p class="abstract">The growing abundance of in vitro omics data, coupled with the necessity to reduce animal testing in the safety assessment of chemical compounds and even eliminate it in the evaluation of cosmetics, highlights the need for abundant computational methodologies. Data from omics technologies allow the exploration of a wide range of biological processes, therefore providing a better understanding of mechanisms of action (MoA) related to chemical exposure in biological systems. However, the analysis of these large datasets remains difficult due to the complexity of modulations spanning multiple biological processes. To address this, we propose a new computational workflow that combines knowledge on endogenous metabolism from a genome scale metabolic network (GSMN) and in vitro transcriptomics data with the aim of better identifying the metabolic MoA (mMoA) of chemicals. Our workflow proceeds in three main steps. The first step consists of building cell condition-specific models representing the metabolic impact of each exposure condition while taking into account the diversity of possible optimal solutions with a partial enumeration algorithm. In a second step, based on these enumerations, two conditions can be compared by extracting differentially activated reactions (DARs) between the two sets of enumerated possible condition-specific models. Finally, in the third step, DARs are grouped into clusters of functionally interconnected metabolic reactions using the distance-based clustering and subnetwork extraction method. The first part of the workflow was exemplified on eight molecules selected for their known human hepatotoxic outcomes associated with specific MoAs well described in the literature and for which we retrieved primary human hepatocytes (PHH) transcriptomic data in Open TG-GATEs. Then, we applied this new workflow to model and visualize associated mMoA for two of these eight molecules (amiodarone and valproic acid). Despite large disparities in transcriptomic effects for these two chemicals, i.e., two differentially expressed genes (DEGs) for amiodarone vs 5709 DEGs for valproic acid, our results well fitted evidence from the literature regarding known MoA. Beyond these confirmations, the workflow highlighted potential other unexplored mMoA.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
