<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="phytools 2.0: An updated R ecosystem for phylogenetic comparative methods (and other things)
Authors: Revell, L. J.
Score: 205.4, Published: 2023-08-03 DOI: 10.1101/2023.03.08.531791
Phylogenetic comparative methods comprise the general endeavor of using an estimated phylogenetic tree (or set of trees) to make secondary inferences: about trait evolution, diversification dynamics, biogeography, community ecology, and a wide range of other phenomena or processes. Over the past ten years or so, the phytools R package (Revell 2012) has grown to become an important research tools for phylogenetic comparative analysis.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="phytools 2.0: An updated R ecosystem for phylogenetic comparative methods (and other things)
Authors: Revell, L. J.
Score: 205.4, Published: 2023-08-03 DOI: 10.1101/2023.03.08.531791
Phylogenetic comparative methods comprise the general endeavor of using an estimated phylogenetic tree (or set of trees) to make secondary inferences: about trait evolution, diversification dynamics, biogeography, community ecology, and a wide range of other phenomena or processes. Over the past ten years or so, the phytools R package (Revell 2012) has grown to become an important research tools for phylogenetic comparative analysis." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-09T10:39:33+00:00" />
<meta property="article:modified_time" content="2023-08-09T10:39:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="phytools 2.0: An updated R ecosystem for phylogenetic comparative methods (and other things)
Authors: Revell, L. J.
Score: 205.4, Published: 2023-08-03 DOI: 10.1101/2023.03.08.531791
Phylogenetic comparative methods comprise the general endeavor of using an estimated phylogenetic tree (or set of trees) to make secondary inferences: about trait evolution, diversification dynamics, biogeography, community ecology, and a wide range of other phenomena or processes. Over the past ten years or so, the phytools R package (Revell 2012) has grown to become an important research tools for phylogenetic comparative analysis."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "phytools 2.0: An updated R ecosystem for phylogenetic comparative methods (and other things)\nAuthors: Revell, L. J.\nScore: 205.4, Published: 2023-08-03 DOI: 10.1101/2023.03.08.531791\nPhylogenetic comparative methods comprise the general endeavor of using an estimated phylogenetic tree (or set of trees) to make secondary inferences: about trait evolution, diversification dynamics, biogeography, community ecology, and a wide range of other phenomena or processes. Over the past ten years or so, the phytools R package (Revell 2012) has grown to become an important research tools for phylogenetic comparative analysis.",
  "keywords": [
    
  ],
  "articleBody": " phytools 2.0: An updated R ecosystem for phylogenetic comparative methods (and other things)\nAuthors: Revell, L. J.\nScore: 205.4, Published: 2023-08-03 DOI: 10.1101/2023.03.08.531791\nPhylogenetic comparative methods comprise the general endeavor of using an estimated phylogenetic tree (or set of trees) to make secondary inferences: about trait evolution, diversification dynamics, biogeography, community ecology, and a wide range of other phenomena or processes. Over the past ten years or so, the phytools R package (Revell 2012) has grown to become an important research tools for phylogenetic comparative analysis. phytools is a diverse contributed R library now consisting of hundreds of different functions covering a variety of methods and purposes in phylogenetic biology. As of the time of writing, phytools included functionality for fitting models of trait evolution, for reconstructing ancestral states, for studying diversification on trees, and for visualizing phylogenies, comparative data, and fitted models, as well numerous other tasks related to phylogenetic biology. Here, I describe some significant features of and recent updates to phytools, while also illustrating several popular workflows of the phytools computational software.\nFrom Proteins to Ligands: Decoding Deep Learning Methods for Binding Affinity Prediction\nAuthors: Gorantla, R.; Kubincova, A.; Weisse, A. Y.; Mey, A. S. J. S.\nScore: 41.4, Published: 2023-08-03 DOI: 10.1101/2023.08.01.551483\nAccurate in silico prediction of protein-ligand binding affinity is important in the early stages of drug discovery. Deep learning-based methods exist but have yet to overtake more conventional methods such as giga-docking largely due to their lack of generalisability. To improve generalizability we need to understand what these models learn from input protein and ligand data. We systematically investigated a sequence-based deep learning framework to assess the impact of protein and ligand encodings on predicting binding affinities for commonly used kinase data sets. The role of proteins is studied using convolutional neural network-based encodings obtained from sequences and graph neural network-based encodings enriched with structural information from contact maps. Ligand-based encodings are generated from graph-neural networks. We test different ligand perturbations by randomizing node and edge properties. For proteins we make use of 3 different protein contact generation methods (AlphaFold2, Pconsc4, and ESM-1b) and compare these with a random control. Our investigation shows that protein encodings do not substantially impact the binding predictions, with no statistically significant difference in binding affinity for KIBA in the investigated metrics (concordance index, Pearsons R Spearmans Rank, and RMSE). Significant differences are seen for ligand encodings with random ligands and random ligand node properties, suggesting a much bigger reliance on ligand data for the learning tasks. Using different ways to combine protein and ligand encodings, did not show a significant change in performance. TOC Graphic O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=111 SRC=\"FIGDIR/small/551483v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (25K): org.highwire.dtl.DTLVardef@117f37forg.highwire.dtl.DTLVardef@173427eorg.highwire.dtl.DTLVardef@126a39org.highwire.dtl.DTLVardef@fbcc6d_HPS_FORMAT_FIGEXP M_FIG C_FIG\nQuantifying Cell-State Densities in Single-Cell Phenotypic Landscapes using Mellon\nAuthors: Otto, D. J.; Jordan, C.; Dury, B.; Dien, C.; Setty, M.\nScore: 47.7, Published: 2023-07-11 DOI: 10.1101/2023.07.09.548272\nCell-state density characterizes the distribution of cells along phenotypic landscapes and is crucial for unraveling the mechanisms that drive cellular differentiation, regeneration, and disease. Here, we present Mellon, a novel computational algorithm for high-resolution estimation of cell-state densities from single-cell data. We demonstrate Mellons efficacy by dissecting the density landscape of various differentiating systems, revealing a consistent pattern of high-density regions corresponding to major cell types intertwined with low-density, rare transitory states. Utilizing hematopoietic stem cell fate specification to B-cells as a case study, we present evidence implicating enhancer priming and the activation of master regulators in the emergence of these transitory states. Mellon offers the flexibility to perform temporal interpolation of time-series data, providing a detailed view of cell-state dynamics during the inherently continuous developmental processes. Scalable and adaptable, Mellon facilitates density estimation across various single-cell data modalities, scaling linearly with the number of cells. Our work underscores the importance of cell-state density in understanding the differentiation processes, and the potential of Mellon to provide new insights into the regulatory mechanisms guiding cellular fate decisions.\nAAontology: An ontology of amino acid scales for interpretable machine learning\nAuthors: Breimann, S.; Kamp, F.; Steiner, H.; Frishman, D.\nScore: 17.0, Published: 2023-08-05 DOI: 10.1101/2023.08.03.551768\nAmino acid scales are crucial for protein prediction tasks, many of them being curated in the AAindex database. Despite various clustering attempts to organize them and to better understand their relationships, these approaches lack the fine-grained classification necessary for satisfactory interpretability in many protein prediction problems. To address this issue, we developed AAontology--a two-level classification for 586 amino acid scales (mainly from AAindex) together with an in-depth analysis of their relations--using bag-of-word-based classification, clustering, and manual refinement over multiple iterations. AAontology organizes physicochemical scales into 8 categories and 67 subcategories, enhancing the interpretability of scale-based machine learning methods in protein bioinformatics. Thereby it enables researchers to gain a deeper biological insight. We anticipate that AAontology will be a building block to link amino acid properties with protein function and dysfunctions as well as aid informed decision-making in mutation analysis or protein drug design.\nFragment ion intensity prediction improves the identification rate of non-tryptic peptides in TimsTOF\nAuthors: Adams, C.; Gabriel, W.; Laukens, K.; Wilhelm, M.; Bittremieux, W.; Boonen, K.\nScore: 18.5, Published: 2023-07-19 DOI: 10.1101/2023.07.17.549401\nImmunopeptidomics plays a crucial role in identifying targets for immunotherapy and vaccine development. Because the generation of immunopeptides from their parent proteins does not adhere to clear-cut rules, rather than being able to use known digestion patterns, every possible protein subsequence within HLA class-specific length restrictions needs to be considered. This leads to an inflation of the search space and results in lower spectrum annotation rates. Rescoring is a powerful enhancement of standard sequence database searching that boosts the spectrum annotation performance. In the field of immunopeptidomics low abundant peptides often occur, which is why the highly sensitive timsTOF instruments are increasingly gaining popularity. To improve rescoring for immunopeptides measured using timsTOF instruments, we trained a deep learning-based fragment ion intensity prediction model. Over 300,000 synthesized non-tryptic peptides from the ProteomeTools project were analyzed on a timsTOF-Pro to generate a dataset that was used to fine-tune an existing Prosit model. By applying our fragment ion intensity prediction model, we demonstrate up to 3-fold improvement in the identification of immunopeptides. Furthermore, our approach increased detection of immunopeptides even from low input samples.\nCellGO: A novel deep learning-based framework and webserver for cell type-specific gene function interpretation\nAuthors: Li, P.; Wei, J.; Zhu, Y.\nScore: 12.5, Published: 2023-08-05 DOI: 10.1101/2023.08.02.551654\nInterpreting the function of genes and gene sets identified from omics experiments remains a challenge, as current pathway analysis tools often fail to account for complex interactions across genes and pathways under specific tissues and cell types. We introduce CellGO, a tool for cell type-specific gene functional analysis. CellGO employs a deep learning model to simulate signaling propagation within a cell, enabling the development of a heuristic pathway activity measuring system to identify cell type-specific active pathways given a single gene or a gene set. It is featured with additional functions to uncover pathway communities and the most active genes within pathways to facilitate mechanistic interpretation. This study demonstrated that CellGO can effectively capture cell type-specific pathways even when working with mixed cell-type markers. CellGOs performance was benchmarked using gene knockout datasets, and its implementation effectively infers the cell type-specific pathogenesis of risk genes associated with neurodevelopmental and neurodegenerative disorders, suggesting its potential in understanding complex polygenic diseases. CellGO is accessible through a python package and a four-mode web interface for interactive usage with pretrained models on 71 single-cell datasets from human and mouse fetal and postnatal brains.\nHighly accurate and robust protein sequence design with CarbonDesign\nAuthors: Ren, M.; Yu, C.; Bu, D.; Zhang, H.\nScore: 11.8, Published: 2023-08-07 DOI: 10.1101/2023.08.07.552204\nProtein sequence design, the inverse problem of protein structure prediction, plays a crucial role in protein engineering. Although recent deep learning-based methods have shown promising advancements, achieving accurate and robust protein sequence design remains an ongoing challenge. Here, we present CarbonDesign, a new approach that draws inspiration from successful ingredients of AlphaFold for protein structure prediction and makes significant and novel developments tailored specifically for protein sequence design. At its core, CarbonDesign explores Inverseformer, a novel network architecture adapted from AlphaFold's Evoformer, to learn representations from backbone structures and an amortized Markov Random Fields model for sequence decoding. Moreover, we incorporate other essential AlphaFold concepts into CarbonDesign: an end-to-end network recycling technique to leverage evolutionary constraints in protein language models and a multi-task learning technique to generate side chain structures corresponding to the designed sequences. Through rigorous evaluations on independent testing data sets, including the CAMEO and recent CASP15 data sets, as well as the predicted structures from AlphaFold, we show that CarbonDesign outperforms other published methods, achieving high accuracy in sequence generation. Moreover, it exhibits superior performance on de novo backbone structures obtained from recent diffusion generative models such as RFdiffusion and FrameDiff, highlighting its potential for enhancing de novo protein design. Notably, CarbonDesign also supports zero-shot prediction of the functional effects of sequence variants, indicating its potential application in directed evolution-based design. In summary, our results illustrate CarbonDesign's accurate and robust performance in protein sequence design, making it a promising tool for applications in bioengineering.\nCan Protein Structure Prediction Methods Capture Alternative Conformations of Membrane Proteins?\nAuthors: Xie, T.; Huang, J.\nScore: 11.2, Published: 2023-08-07 DOI: 10.1101/2023.08.04.552045\nUnderstanding the conformational dynamics of proteins, such as the inward-facing (IF) and outward-facing (OF) transition observed in transporters, is vital for elucidating their functional mechanisms. Despite significant advances in protein structure prediction (PSP) over the past three decades, most efforts have been focused on single-state prediction, leaving multi-state or alternative conformation prediction (ACP) relatively unexplored. This discrepancy has led to the development of highly accurate PSP methods such as AlphaFold, yet their capabilities for ACP remain limited. To investigate the performance of current PSP methods in ACP, we curated a dataset, named IOMemP, consisting of 32 experimentally determined high-resolution IF and OF structures of 16 membrane proteins. We benchmarked 12 representative PSP methods, along with two recent multi-state methods based on AlphaFold, against this dataset. Our findings reveal an escalating bias towards one specific state in deep learning-based methods and a remarkably consistent preference for specific states across various PSP methods. We elucidated how coevolution information in MSAs influences the state preference. Moreover, we showed that AlphaFold, when excluding coevolution information, estimated similar energies between the experimental IF and OF conformations, indicating that the energy model learned by AlphaFold is not biased towards any particular state. Our IOMemP dataset and benchmark results are anticipated to advance the development of robust ACP methods.\nphuEGO: A network-based method to reconstruct active signalling pathways from phosphoproteomics datasets\nAuthors: Giudice, G.; Chen, H.; Petsalaki, E.\nScore: 10.7, Published: 2023-08-07 DOI: 10.1101/2023.08.07.552249\nSignalling networks are critical for virtually all cell functions. Our current knowledge of cell signalling has been summarised in signalling pathway databases, which, while useful, are highly biassed towards well-studied processes, and don't capture context specific network wiring or pathway cross-talk. Mass spectrometry-based phosphoproteomics data can provide a more unbiased view of active cell signalling processes in a given context, however, it suffers from low signal-to-noise ratio and poor reproducibility across experiments. Methods to extract active signalling signatures from such data struggle to produce unbiased and interpretable networks that can be used for hypothesis generation and designing downstream experiments. Here we present phuEGO, which combines three-layer network propagation with ego network decomposition to provide small networks comprising active functional signalling modules. PhuEGO boosts the signal-to-noise ratio from global phosphoproteomics datasets, enriches the resulting networks for functional phosphosites and allows the improved comparison and integration across datasets. We applied phuEGO to five phosphoproteomics data sets from cell lines collected upon infection with SARS CoV2. PhuEGO was better able to identify common active functions across datasets and to point to a subnetwork enriched for known COVID-19 targets. Overall, phuEGO provides a tool to the community for the improved functional interpretation of global phosphoproteomics datasets.\nCan a Sparse 29 x 29 Pixel Chaos Game Representation Predict Protein Binding Sites using Fine-Tuned State-of-the Art Deep Learning Semantic Segmentation Models?\nAuthors: Dick, K.; Green, J. R.\nScore: 10.3, Published: 2023-08-04 DOI: 10.1101/2023.08.04.410498\nNo. While our experiments ultimately failed, this work was motivated by the seemingly reasonable hypothesis that encoding protein sequences as a fractal-based image in combination with a binary mask identifying those pixels representative of the protein binding interface could effectively be used to fine-tune a semantic segmentation model. We were wrong. Despite the shortcomings of this work, a number of insights were drawn, inspiring discussion about how this fractal-based space may be exploited to generate effective protein binding site predictors in the future. Furthermore, these realizations promise to orient complimentary studies leveraging fractal-based representations, whether in the field of bioinformatics, or more broadly within disparate fields leveraging sequence-type data, such as Natural Language Processing. In a non-traditional way, this work presents the experimental design undertaken and interleaves various insights and limitations. It is the hope of this work that those interested in leveraging fractal-based representations and deep learning architectures as part of their work will benefit from the insights arising from this work.\n",
  "wordCount" : "2181",
  "inLanguage": "en",
  "datePublished": "2023-08-09T10:39:33Z",
  "dateModified": "2023-08-09T10:39:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on August 9, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.03.08.531791">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.03.08.531791" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.03.08.531791">
        <p class="paperTitle">phytools 2.0: An updated R ecosystem for phylogenetic comparative methods (and other things)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.03.08.531791" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.03.08.531791" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Revell, L. J.</p>
        <p class="info">Score: 205.4, Published: 2023-08-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.03.08.531791' target='https://doi.org/10.1101/2023.03.08.531791'> 10.1101/2023.03.08.531791</a></p>
        <p class="abstract">Phylogenetic comparative methods comprise the general endeavor of using an estimated phylogenetic tree (or set of trees) to make secondary inferences: about trait evolution, diversification dynamics, biogeography, community ecology, and a wide range of other phenomena or processes. Over the past ten years or so, the phytools R package (Revell 2012) has grown to become an important research tools for phylogenetic comparative analysis. phytools is a diverse contributed R library now consisting of hundreds of different functions covering a variety of methods and purposes in phylogenetic biology. As of the time of writing, phytools included functionality for fitting models of trait evolution, for reconstructing ancestral states, for studying diversification on trees, and for visualizing phylogenies, comparative data, and fitted models, as well numerous other tasks related to phylogenetic biology. Here, I describe some significant features of and recent updates to phytools, while also illustrating several popular workflows of the phytools computational software.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.551483">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.551483" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.551483">
        <p class="paperTitle">From Proteins to Ligands: Decoding Deep Learning Methods for Binding Affinity Prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.551483" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.551483" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gorantla, R.; Kubincova, A.; Weisse, A. Y.; Mey, A. S. J. S.</p>
        <p class="info">Score: 41.4, Published: 2023-08-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.551483' target='https://doi.org/10.1101/2023.08.01.551483'> 10.1101/2023.08.01.551483</a></p>
        <p class="abstract">Accurate in silico prediction of protein-ligand binding affinity is important in the early stages of drug discovery. Deep learning-based methods exist but have yet to overtake more conventional methods such as giga-docking largely due to their lack of generalisability. To improve generalizability we need to understand what these models learn from input protein and ligand data. We systematically investigated a sequence-based deep learning framework to assess the impact of protein and ligand encodings on predicting binding affinities for commonly used kinase data sets. The role of proteins is studied using convolutional neural network-based encodings obtained from sequences and graph neural network-based encodings enriched with structural information from contact maps. Ligand-based encodings are generated from graph-neural networks. We test different ligand perturbations by randomizing node and edge properties. For proteins we make use of 3 different protein contact generation methods (AlphaFold2, Pconsc4, and ESM-1b) and compare these with a random control. Our investigation shows that protein encodings do not substantially impact the binding predictions, with no statistically significant difference in binding affinity for KIBA in the investigated metrics (concordance index, Pearsons R Spearmans Rank, and RMSE). Significant differences are seen for ligand encodings with random ligands and random ligand node properties, suggesting a much bigger reliance on ligand data for the learning tasks. Using different ways to combine protein and ligand encodings, did not show a significant change in performance.

TOC Graphic

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=111 SRC=&#34;FIGDIR/small/551483v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (25K):
org.highwire.dtl.DTLVardef@117f37forg.highwire.dtl.DTLVardef@173427eorg.highwire.dtl.DTLVardef@126a39org.highwire.dtl.DTLVardef@fbcc6d_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.09.548272">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.09.548272" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.09.548272">
        <p class="paperTitle">Quantifying Cell-State Densities in Single-Cell Phenotypic Landscapes using Mellon</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.09.548272" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.09.548272" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Otto, D. J.; Jordan, C.; Dury, B.; Dien, C.; Setty, M.</p>
        <p class="info">Score: 47.7, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.09.548272' target='https://doi.org/10.1101/2023.07.09.548272'> 10.1101/2023.07.09.548272</a></p>
        <p class="abstract">Cell-state density characterizes the distribution of cells along phenotypic landscapes and is crucial for unraveling the mechanisms that drive cellular differentiation, regeneration, and disease. Here, we present Mellon, a novel computational algorithm for high-resolution estimation of cell-state densities from single-cell data. We demonstrate Mellons efficacy by dissecting the density landscape of various differentiating systems, revealing a consistent pattern of high-density regions corresponding to major cell types intertwined with low-density, rare transitory states. Utilizing hematopoietic stem cell fate specification to B-cells as a case study, we present evidence implicating enhancer priming and the activation of master regulators in the emergence of these transitory states. Mellon offers the flexibility to perform temporal interpolation of time-series data, providing a detailed view of cell-state dynamics during the inherently continuous developmental processes. Scalable and adaptable, Mellon facilitates density estimation across various single-cell data modalities, scaling linearly with the number of cells. Our work underscores the importance of cell-state density in understanding the differentiation processes, and the potential of Mellon to provide new insights into the regulatory mechanisms guiding cellular fate decisions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.03.551768">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.03.551768" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.03.551768">
        <p class="paperTitle">AAontology: An ontology of amino acid scales for interpretable machine learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.03.551768" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.03.551768" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Breimann, S.; Kamp, F.; Steiner, H.; Frishman, D.</p>
        <p class="info">Score: 17.0, Published: 2023-08-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.03.551768' target='https://doi.org/10.1101/2023.08.03.551768'> 10.1101/2023.08.03.551768</a></p>
        <p class="abstract">Amino acid scales are crucial for protein prediction tasks, many of them being curated in the AAindex database. Despite various clustering attempts to organize them and to better understand their relationships, these approaches lack the fine-grained classification necessary for satisfactory interpretability in many protein prediction problems.

To address this issue, we developed AAontology--a two-level classification for 586 amino acid scales (mainly from AAindex) together with an in-depth analysis of their relations--using bag-of-word-based classification, clustering, and manual refinement over multiple iterations. AAontology organizes physicochemical scales into 8 categories and 67 subcategories, enhancing the interpretability of scale-based machine learning methods in protein bioinformatics. Thereby it enables researchers to gain a deeper biological insight. We anticipate that AAontology will be a building block to link amino acid properties with protein function and dysfunctions as well as aid informed decision-making in mutation analysis or protein drug design.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549401">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549401" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549401">
        <p class="paperTitle">Fragment ion intensity prediction improves the identification rate of non-tryptic peptides in TimsTOF</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549401" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549401" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Adams, C.; Gabriel, W.; Laukens, K.; Wilhelm, M.; Bittremieux, W.; Boonen, K.</p>
        <p class="info">Score: 18.5, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549401' target='https://doi.org/10.1101/2023.07.17.549401'> 10.1101/2023.07.17.549401</a></p>
        <p class="abstract">Immunopeptidomics plays a crucial role in identifying targets for immunotherapy and vaccine development. Because the generation of immunopeptides from their parent proteins does not adhere to clear-cut rules, rather than being able to use known digestion patterns, every possible protein subsequence within HLA class-specific length restrictions needs to be considered. This leads to an inflation of the search space and results in lower spectrum annotation rates. Rescoring is a powerful enhancement of standard sequence database searching that boosts the spectrum annotation performance. In the field of immunopeptidomics low abundant peptides often occur, which is why the highly sensitive timsTOF instruments are increasingly gaining popularity. To improve rescoring for immunopeptides measured using timsTOF instruments, we trained a deep learning-based fragment ion intensity prediction model. Over 300,000 synthesized non-tryptic peptides from the ProteomeTools project were analyzed on a timsTOF-Pro to generate a dataset that was used to fine-tune an existing Prosit model. By applying our fragment ion intensity prediction model, we demonstrate up to 3-fold improvement in the identification of immunopeptides. Furthermore, our approach increased detection of immunopeptides even from low input samples.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.551654">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.551654" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.551654">
        <p class="paperTitle">CellGO: A novel deep learning-based framework and webserver for cell type-specific gene function interpretation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.551654" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.551654" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, P.; Wei, J.; Zhu, Y.</p>
        <p class="info">Score: 12.5, Published: 2023-08-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.551654' target='https://doi.org/10.1101/2023.08.02.551654'> 10.1101/2023.08.02.551654</a></p>
        <p class="abstract">Interpreting the function of genes and gene sets identified from omics experiments remains a challenge, as current pathway analysis tools often fail to account for complex interactions across genes and pathways under specific tissues and cell types. We introduce CellGO, a tool for cell type-specific gene functional analysis. CellGO employs a deep learning model to simulate signaling propagation within a cell, enabling the development of a heuristic pathway activity measuring system to identify cell type-specific active pathways given a single gene or a gene set. It is featured with additional functions to uncover pathway communities and the most active genes within pathways to facilitate mechanistic interpretation. This study demonstrated that CellGO can effectively capture cell type-specific pathways even when working with mixed cell-type markers. CellGOs performance was benchmarked using gene knockout datasets, and its implementation effectively infers the cell type-specific pathogenesis of risk genes associated with neurodevelopmental and neurodegenerative disorders, suggesting its potential in understanding complex polygenic diseases. CellGO is accessible through a python package and a four-mode web interface for interactive usage with pretrained models on 71 single-cell datasets from human and mouse fetal and postnatal brains.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552204">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552204" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552204">
        <p class="paperTitle">Highly accurate and robust protein sequence design with CarbonDesign</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552204" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552204" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ren, M.; Yu, C.; Bu, D.; Zhang, H.</p>
        <p class="info">Score: 11.8, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552204' target='https://doi.org/10.1101/2023.08.07.552204'> 10.1101/2023.08.07.552204</a></p>
        <p class="abstract">Protein sequence design, the inverse problem of protein structure prediction, plays a crucial role in protein engineering. Although recent deep learning-based methods have shown promising advancements, achieving accurate and robust protein sequence design remains an ongoing challenge. Here, we present CarbonDesign, a new approach that draws inspiration from successful ingredients of AlphaFold for protein structure prediction and makes significant and novel developments tailored specifically for protein sequence design. At its core, CarbonDesign explores Inverseformer, a novel network architecture adapted from AlphaFold&#39;s Evoformer, to learn representations from backbone structures and an amortized Markov Random Fields model for sequence decoding. Moreover, we incorporate other essential AlphaFold concepts into CarbonDesign: an end-to-end network recycling technique to leverage evolutionary constraints in protein language models and a multi-task learning technique to generate side chain structures corresponding to the designed sequences. Through rigorous evaluations on independent testing data sets, including the CAMEO and recent CASP15 data sets, as well as the predicted structures from AlphaFold, we show that CarbonDesign outperforms other published methods, achieving high accuracy in sequence generation. Moreover, it exhibits superior performance on de novo backbone structures obtained from recent diffusion generative models such as RFdiffusion and FrameDiff, highlighting its potential for enhancing de novo protein design. Notably, CarbonDesign also supports zero-shot prediction of the functional effects of sequence variants, indicating its potential application in directed evolution-based design. In summary, our results illustrate CarbonDesign&#39;s accurate and robust performance in protein sequence design, making it a promising tool for applications in bioengineering.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.552045">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.552045" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.552045">
        <p class="paperTitle">Can Protein Structure Prediction Methods Capture Alternative Conformations of Membrane Proteins?</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.552045" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.552045" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xie, T.; Huang, J.</p>
        <p class="info">Score: 11.2, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.552045' target='https://doi.org/10.1101/2023.08.04.552045'> 10.1101/2023.08.04.552045</a></p>
        <p class="abstract">Understanding the conformational dynamics of proteins, such as the inward-facing (IF) and outward-facing (OF) transition observed in transporters, is vital for elucidating their functional mechanisms. Despite significant advances in protein structure prediction (PSP) over the past three decades, most efforts have been focused on single-state prediction, leaving multi-state or alternative conformation prediction (ACP) relatively unexplored. This discrepancy has led to the development of highly accurate PSP methods such as AlphaFold, yet their capabilities for ACP remain limited. To investigate the performance of current PSP methods in ACP, we curated a dataset, named IOMemP, consisting of 32 experimentally determined high-resolution IF and OF structures of 16 membrane proteins. We benchmarked 12 representative PSP methods, along with two recent multi-state methods based on AlphaFold, against this dataset. Our findings reveal an escalating bias towards one specific state in deep learning-based methods and a remarkably consistent preference for specific states across various PSP methods. We elucidated how coevolution information in MSAs influences the state preference. Moreover, we showed that AlphaFold, when excluding coevolution information, estimated similar energies between the experimental IF and OF conformations, indicating that the energy model learned by AlphaFold is not biased towards any particular state. Our IOMemP dataset and benchmark results are anticipated to advance the development of robust ACP methods.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552249">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552249" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552249">
        <p class="paperTitle">phuEGO: A network-based method to reconstruct active signalling pathways from phosphoproteomics datasets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552249" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552249" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Giudice, G.; Chen, H.; Petsalaki, E.</p>
        <p class="info">Score: 10.7, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552249' target='https://doi.org/10.1101/2023.08.07.552249'> 10.1101/2023.08.07.552249</a></p>
        <p class="abstract">Signalling networks are critical for virtually all cell functions. Our current knowledge of cell signalling has been summarised in signalling pathway databases, which, while useful, are highly biassed towards well-studied processes, and don&#39;t capture context specific network wiring or pathway cross-talk. Mass spectrometry-based phosphoproteomics data can provide a more unbiased view of active cell signalling processes in a given context, however, it suffers from low signal-to-noise ratio and poor reproducibility across experiments. Methods to extract active signalling signatures from such data struggle to produce unbiased and interpretable networks that can be used for hypothesis generation and designing downstream experiments. Here we present phuEGO, which combines three-layer network propagation with ego network decomposition to provide small networks comprising active functional signalling modules. PhuEGO boosts the signal-to-noise ratio from global phosphoproteomics datasets, enriches the resulting networks for functional phosphosites and allows the improved comparison and integration across datasets. We applied phuEGO to five phosphoproteomics data sets from cell lines collected upon infection with SARS CoV2. PhuEGO was better able to identify common active functions across datasets and to point to a subnetwork enriched for known COVID-19 targets. Overall, phuEGO provides a tool to the community for the improved functional interpretation of global phosphoproteomics datasets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.410498">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.410498" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.410498">
        <p class="paperTitle">Can a Sparse 29 x 29 Pixel Chaos Game Representation Predict Protein Binding Sites using Fine-Tuned State-of-the Art Deep Learning Semantic Segmentation Models?</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.410498" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.410498" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dick, K.; Green, J. R.</p>
        <p class="info">Score: 10.3, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.410498' target='https://doi.org/10.1101/2023.08.04.410498'> 10.1101/2023.08.04.410498</a></p>
        <p class="abstract">No. While our experiments ultimately failed, this work was motivated by the seemingly reasonable hypothesis that encoding protein sequences as a fractal-based image in combination with a binary mask identifying those pixels representative of the protein binding interface could effectively be used to fine-tune a semantic segmentation model. We were wrong.

Despite the shortcomings of this work, a number of insights were drawn, inspiring discussion about how this fractal-based space may be exploited to generate effective protein binding site predictors in the future. Furthermore, these realizations promise to orient complimentary studies leveraging fractal-based representations, whether in the field of bioinformatics, or more broadly within disparate fields leveraging sequence-type data, such as Natural Language Processing.

In a non-traditional way, this work presents the experimental design undertaken and interleaves various insights and limitations. It is the hope of this work that those interested in leveraging fractal-based representations and deep learning architectures as part of their work will benefit from the insights arising from this work.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
