<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="DrugGPT: A GPT-based Strategy for Designing Potential Ligands Targeting Specific Proteins
Authors: Li, Y.; Gao, C.; Song, X.; Wang, X.; Xu, Y.; Han, S.
Score: 29.8, Published: 2023-06-30 DOI: 10.1101/2023.06.29.543848
DrugGPT presents a ligand design strategy based on the autoregressive model, GPT, focusing on chemical space exploration and the discovery of ligands for specific proteins. Deep learning language models have shown significant potential in various domains including protein design and biomedical text analysis, providing strong support for the proposition of DrugGPT.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="DrugGPT: A GPT-based Strategy for Designing Potential Ligands Targeting Specific Proteins
Authors: Li, Y.; Gao, C.; Song, X.; Wang, X.; Xu, Y.; Han, S.
Score: 29.8, Published: 2023-06-30 DOI: 10.1101/2023.06.29.543848
DrugGPT presents a ligand design strategy based on the autoregressive model, GPT, focusing on chemical space exploration and the discovery of ligands for specific proteins. Deep learning language models have shown significant potential in various domains including protein design and biomedical text analysis, providing strong support for the proposition of DrugGPT." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-02T10:39:32+00:00" />
<meta property="article:modified_time" content="2023-07-02T10:39:32+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="DrugGPT: A GPT-based Strategy for Designing Potential Ligands Targeting Specific Proteins
Authors: Li, Y.; Gao, C.; Song, X.; Wang, X.; Xu, Y.; Han, S.
Score: 29.8, Published: 2023-06-30 DOI: 10.1101/2023.06.29.543848
DrugGPT presents a ligand design strategy based on the autoregressive model, GPT, focusing on chemical space exploration and the discovery of ligands for specific proteins. Deep learning language models have shown significant potential in various domains including protein design and biomedical text analysis, providing strong support for the proposition of DrugGPT."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "DrugGPT: A GPT-based Strategy for Designing Potential Ligands Targeting Specific Proteins\nAuthors: Li, Y.; Gao, C.; Song, X.; Wang, X.; Xu, Y.; Han, S.\nScore: 29.8, Published: 2023-06-30 DOI: 10.1101/2023.06.29.543848\nDrugGPT presents a ligand design strategy based on the autoregressive model, GPT, focusing on chemical space exploration and the discovery of ligands for specific proteins. Deep learning language models have shown significant potential in various domains including protein design and biomedical text analysis, providing strong support for the proposition of DrugGPT.",
  "keywords": [
    
  ],
  "articleBody": " DrugGPT: A GPT-based Strategy for Designing Potential Ligands Targeting Specific Proteins\nAuthors: Li, Y.; Gao, C.; Song, X.; Wang, X.; Xu, Y.; Han, S.\nScore: 29.8, Published: 2023-06-30 DOI: 10.1101/2023.06.29.543848\nDrugGPT presents a ligand design strategy based on the autoregressive model, GPT, focusing on chemical space exploration and the discovery of ligands for specific proteins. Deep learning language models have shown significant potential in various domains including protein design and biomedical text analysis, providing strong support for the proposition of DrugGPT. In this study, we employ the DrugGPT model to learn a substantial amount of protein-ligand binding data, aiming to discover novel molecules that can bind with specific proteins. This strategy not only significantly improves the efficiency of ligand design but also offers a swift and effective avenue for the drug development process, bringing new possibilities to the pharmaceutical domain. In our research, we particularly optimized and trained the GPT-2 model to better adapt to the requirements of drug design. Given the characteristics of proteins and ligands, we redesigned the tokenizer using the BPE algorithm, abandoned the original tokenizer, and trained the GPT-2 model from scratch. This improvement enables DrugGPT to more accurately capture and understand the structural information and chemical rules of drug molecules. It also enhances its comprehension of binding information between proteins and ligands, thereby generating potentially active drug candidate molecules. Theoretically, DrugGPT has significant advantages. During the model training process, DrugGPT aims to maximize the conditional probability and employs the back-propagation algorithm for training, making the training process more stable and avoiding the Mode Collapse problem that may occur in Generative Adversarial Networks in drug design. Furthermore, the design philosophy of DrugGPT endows it with strong generalization capabilities, giving it the potential to adapt to different tasks. In conclusion, DrugGPT provides a forward-thinking and practical new approach to ligand design. By optimizing the tokenizer and retraining the GPT-2 model, the ligand design process becomes more direct and efficient. This not only reflects the theoretical advantages of DrugGPT but also reveals its potential applications in the drug development process, thereby opening new perspectives and possibilities in the pharmaceutical field.\nBGCFlow: Systematic pangenome workflow for the analysis of biosynthetic gene clusters across large genomic datasets\nAuthors: Nuhamunada, M.; Mohite, O. S.; Phaneuf, P. V.; Palsson, B.; Weber, T.\nScore: 39.2, Published: 2023-06-15 DOI: 10.1101/2023.06.14.545018\nGenome mining is revolutionizing natural products discovery efforts. The rapid increase in available genomes demands comprehensive computational platforms to effectively extract biosynthetic knowledge encoded across bacterial pangenomes. Here, we present BGCFlow, a novel systematic workflow integrating analytics for large-scale genome mining of bacterial pangenomes. BGCFlow incorporates several genome analytics and mining tools grouped into five common stages of analysis such as; i) data selection, ii) functional annotation, iii) phylogenetic analysis, iv) genome mining, and v) comparative analysis. Furthermore, BGCFlow provides easy configuration of different projects, parallel distribution, scheduled job monitoring, an interactive database to visualize tables, exploratory Jupyter notebooks, and customized reports. Here, we demonstrate the application of BGCFlow by investigating the phylogenetic distribution of various biosynthetic gene clusters detected across 42 genomes of the Saccharopolyspora genus, known to produce industrially important secondary/specialized metabolites. The BGCFlow-guided analysis predicted more accurate dereplication of BGCs and guided the targeted comparative analysis of selected RiPPs. The scalable, interoperable, adaptable, re-entrant, and reproducible nature of the BGCFlow will provide an effective novel way to extract the biosynthetic knowledge in the ever-growing genomic datasets of biotechnologically relevant bacterial species. BGCFlow is available for downloading at https://github.com/NBChub/bgcflow.\nCo-linear Chaining on Pangenome Graphs\nAuthors: Rajput, J.; Chandra, G.; Jain, C.\nScore: 21.1, Published: 2023-06-24 DOI: 10.1101/2023.06.21.545871\nPangenome reference graphs are useful in genomics because they compactly represent the genetic diversity within a species, a capability that linear references lack. However, efficiently aligning sequences to these graphs with complex topology and cycles can be challenging. The seed-chain-extend based alignment algorithms use co-linear chaining as a standard technique to identify a good cluster of exact seed matches that can be combined to form an alignment. Recent works show how the co-linear chaining problem can be efficiently solved for acyclic pangenome graphs by exploiting their small width [Makinen et al., TALG19] and how incorporating gap cost in the scoring function improves alignment accuracy [Chandra and Jain, RECOMB23]. However, it remains open on how to effectively generalize these techniques for general pangenome graphs which contain cycles. Here we present the first practical formulation and an exact algorithm for co-linear chaining on cyclic pangenome graphs. We rigorously prove the correctness and computational complexity of the proposed algorithm. We evaluate the empirical performance of our algorithm by aligning simulated long reads from the human genome to a cyclic pangenome graph constructed from 95 publicly available haplotype-resolved human genome assemblies. While the existing heuristic-based algorithms are faster, the proposed algorithm provides a significant advantage in terms of accuracy. Implementationhttps://github.com/at-cg/PanAligner\nQuantUMS: uncertainty minimisation enables confident quantification in proteomics\nAuthors: Kistner, F.; Grossmann, J. L.; Sinn, L. R.; Demichev, V.\nScore: 49.8, Published: 2023-06-24 DOI: 10.1101/2023.06.20.545604\nMass spectrometry-based proteomics has been rapidly gaining traction as a powerful analytical method both in basic research and translation. While the problem of error control in peptide and protein identification has been addressed extensively, the quality of the resulting quantities remains challenging to evaluate. Here we introduce QuantUMS (Quantification using an Uncertainty Minimising Solution), a machine learning-based method which minimises errors and eliminates bias in peptide and protein quantification by integrating multiple sources of quantitative information. In combination with data-independent acquisition proteomics, QuantUMS boosts accuracy and precision of quantities, as well as reports an uncertainty metric, enabling effective filtering of data for downstream analysis. The algorithm has linear complexity with respect to the number of mass spectrometry acquisitions in the experiment and is thus scalable to infinitely large proteomic experiments. For an easy implementation in a proteomics laboratory, we integrate QuantUMS in our automated DIA-NN software suite.\nBrainRBPedia: a resource for RNA-binding proteins relevant to neurodevelopmental disorders\nAuthors: Han, K.; Wainberg, M.; Calarco, J. A.; Smibert, C. A.; Lipshitz, H. D.; Lee, H. O.; Tripathy, S. J.\nScore: 23.8, Published: 2023-06-09 DOI: 10.1101/2023.06.07.542483\nRNA-binding proteins (RBPs) are crucial players in the post-transcriptional regulation of mRNA and play major roles in ensuring proper neuronal development and function. Deficits in RBP function have been implicated in a number of neurodevelopmental disorders including autism spectrum disorder (ASD) and intellectual disability (ID), yet we lack resources that integrate current knowledge of RBP function, tissue expression, and disease association in one place to aid in their experimental characterization. Here we introduce BrainRBPedia - a database of 1072 RBPs with both disease annotations for neurodevelopmental disorders and functional annotations relevant to these disorders, including loss-of-function intolerance and expression specificity to the brain, neurons, and neuronal development. Using these functional annotations, we develop a machine learning model to prioritize RBPs likely to be involved in ASD and ID. Our model indicates that RBPs with high loss-of-function intolerance and those upregulated during neuronal differentiation are disproportionately likely to contribute to ASD and ID etiology. In summary, BrainRBPedia comprises a unique resource for researchers interested in the experimental characterization of RBPs in relation to neurodevelopmental disorders and suggests functional signatures of RBPs likely to play a role in neurodevelopment.\nDeepRank-GNN-esm: A Graph Neural Network for Scoring Protein-Protein Models using Protein Language Model\nAuthors: Xu, X.; Bonvin, A. M.\nScore: 18.8, Published: 2023-06-24 DOI: 10.1101/2023.06.22.546080\nMotivationProtein-Protein interactions (PPIs) play critical roles in numerous cellular processes. By modelling the three-dimensional structures of the correspond protein complexes valuable insights can be obtained, providing, for example, starting points for drug and protein design. One challenge in the modelling process is however the identification of near-native models from the large pool of generated models. To this end we previously developed DeepRank-GNN, a graph neural network that integrates structural and sequence information to enable effective pattern learning at PPI interfaces. Its main features are related to the Position Specific Scoring Matrices (PSSM), which are computationally expensive to generate and significantly limit the algorithms usability. ResultsWe introduce here DeepRank-GNN-esm that includes as additional features protein language model embeddings from the EMS-2 model. We show that the ESM-2 embeddings can actually replace the PSSM features at no cost in-, or even better performance on two PPI-related tasks: scoring docking poses and detecting crystal artifacts. This new DeepRank version bypasses thus the need of generating PSSM, greatly improving the usability of the software and opening new application opportunities for systems for which PSSM profiles cannot be obtained or are irrelevant (e.g. antibody-antigen complexes). Availability and implementationDeepRank-GNN-esm is freely available from https://github.com/DeepRank/DeepRank-GNN-esm\nAccurate prediction of protein assembly structure by combining AlphaFold and symmetrical docking\nAuthors: Jeppesen, M.; Andre, I.\nScore: 16.7, Published: 2023-06-24 DOI: 10.1101/2023.06.22.546069\nAlphaFold can predict the structures of monomeric and multimeric proteins with high accuracy but has a limit on the number of chains and residues it can fold. Here we show that a combination of AlphaFold and all-atom symmetric docking simulations enables highly accurate prediction of the structure of complex symmetrical assemblies. We present a method to predict the structure of complexes with cubic - tetrahedral, octahedral and icosahedral - symmetry from sequence. Focusing on proteins where AlphaFold can make confident predictions on the subunit structure, 21 cubic systems were assembled with a median TM-score of 0.99 and a DockQ score of 0.71. 15 had TM-scores of above 0.8 and were categorized as high-quality according to DockQ. The resulting models are energetically optimized and can be used for detailed studies of intermolecular interactions in higher-order symmetrical assemblies. The results demonstrate how explicit treatment of structural symmetry can significantly expand the size and complexity of AlphaFold predictions.\nFractional Hitting Sets for Efficient and Lightweight Genomic Data Sketching\nAuthors: Rouze, T.; Martayan, I.; MARCHET, C.; Limasset, A.\nScore: 13.9, Published: 2023-06-24 DOI: 10.1101/2023.06.21.545875\nThe exponential increase in publicly available sequencing data and genomic resources necessitates the development of highly efficient methods for data processing and analysis. Locality-sensitive hashing techniques have successfully transformed large datasets into smaller, more manageable sketches while maintaining comparability using metrics such as Jaccard and containment indices. However, fixed-size sketches encounter difficulties when applied to divergent datasets. Scalable sketching methods, such as Sourmash, provide valuable solutions but still lack resourceefficient, tailored indexing. Our objective is to create lighter sketches with comparable results while enhancing efficiency. We introduce the concept of Fractional Hitting Sets, a generalization of Universal Hitting Sets, which uniformly cover a specified fraction of the k-mer space. In theory and practice, we demonstrate the feasibility of achieving such coverage with simple but highly efficient schemes. By encoding the covered k-mers as super-k-mers, we provide a space-efficient exact representation that also enables optimized comparisons. Our novel tool, SuperSampler, implements this scheme, and experimental results with real bacterial collections closely match our theoretical findings. In comparison to Sourmash, SuperSampler achieves similar outcomes while utilizing an order of magnitude less space and memory and operating several times faster. This highlights the potential of our approach in addressing the challenges presented by the ever-expanding landscape of genomic data. SuperSampler is an open-source software and can be accessed at github.com/TimRouze/supersampler. The data required to reproduce the results presented in this manuscript is available at github.com/TimRouze/Expe_SPSP.\nFine-tuning Protein Embeddings for GeneralizableAnnotation Propagation\nAuthors: Dickson, A. M.; Mofrad, M. R. K.\nScore: 11.9, Published: 2023-06-24 DOI: 10.1101/2023.06.22.546084\nA central goal of bioinformatics research is to understand proteins on a functional level, typically by extrapolating from experimental results with the protein sequence information. One strategy is to assume that proteins with similar sequences will also share function. This has the benefit of being interpretable; it gives a very clear idea of why a protein might have a particular function by comparing with the most similar reference example. However, direct machine learning classifiers now outperform pure sequence similarity methods in raw prediction ability. A hybrid method is to use pre-trained language models to create protein embeddings, and then indirectly predict protein function using their relative similarity. We find that fine-tuning an auxiliary objective on protein function indirectly improves these hy-brid methods, to the point that they are in some cases better than direct classifiers. Our empirical results demonstrate that interpretable protein comparison models can be developed using fine-tuning techniques, without cost, or even with some benefit, to overall performance. K-nearest neighbors (KNN) embedding-based models also offer free generalization to previously unknown classes, while continuing to outperform only pre-trained models, further demonstrating the potential of fine-tuned embeddings outside of direct classification. Significance StatementFor the purposes of biological understanding, or engineering, it is particularly important that protein annotation is done through interpretable means, such as comparison to existing proteins with known properties. Our work builds upon previous efforts to do so by embedding proteins into high-dimensional vectors with pre-trained protein language models. We demonstrate that further training of these models on protein function classification drastically improves their performance, while allowing for exactly the same similarity based classifications on both known and novel potential functions.\nScalable Integration of Multiomic Single Cell Data Using Generative Adversarial Networks\nAuthors: Giansanti, V.; Giannese, F.; Botrugno, O. A.; Gandolfi, G.; Balestrieri, C.; Antoniotti, M.; Tonon, G.; Cittaro, D.\nScore: 11.6, Published: 2023-06-29 DOI: 10.1101/2023.06.26.546547\nSingle cell profiling has become a common practice to investigate the complexity of tissues, organs and organisms. Recent technological advances are expanding our capabilities to profile various molecular layers beyond the transcriptome such as, but not limited to, the genome, the epigenome and the proteome. Depending on the experimental procedure, these data can be obtained from separate assays or from the very same cells. Despite development of computational methods for data integration is an active research field, most of the available strategies have been devised for the joint analysis of two modalities and cannot accommodate a high number of them. To solve this problem, we here propose a multiomic data integration framework based on Wasserstein Generative Adversarial Networks (MOWGAN) suitable for the analysis of paired or unpaired data with high number of modalities (\u003e2). At the core of our strategy is a single network trained on all modalities together, limiting the computational burden when many molecular layers are evaluated. Source code of our framework is available at https://github.com/vgiansanti/MOWGAN.\n",
  "wordCount" : "2350",
  "inLanguage": "en",
  "datePublished": "2023-07-02T10:39:32Z",
  "dateModified": "2023-07-02T10:39:32Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on July 2, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.543848">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.543848" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.543848">
        <p class="paperTitle">DrugGPT: A GPT-based Strategy for Designing Potential Ligands Targeting Specific Proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.543848" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.543848" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, Y.; Gao, C.; Song, X.; Wang, X.; Xu, Y.; Han, S.</p>
        <p class="info">Score: 29.8, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.543848' target='https://doi.org/10.1101/2023.06.29.543848'> 10.1101/2023.06.29.543848</a></p>
        <p class="abstract">DrugGPT presents a ligand design strategy based on the autoregressive model, GPT, focusing on chemical space exploration and the discovery of ligands for specific proteins. Deep learning language models have shown significant potential in various domains including protein design and biomedical text analysis, providing strong support for the proposition of DrugGPT. In this study, we employ the DrugGPT model to learn a substantial amount of protein-ligand binding data, aiming to discover novel molecules that can bind with specific proteins. This strategy not only significantly improves the efficiency of ligand design but also offers a swift and effective avenue for the drug development process, bringing new possibilities to the pharmaceutical domain. In our research, we particularly optimized and trained the GPT-2 model to better adapt to the requirements of drug design. Given the characteristics of proteins and ligands, we redesigned the tokenizer using the BPE algorithm, abandoned the original tokenizer, and trained the GPT-2 model from scratch. This improvement enables DrugGPT to more accurately capture and understand the structural information and chemical rules of drug molecules. It also enhances its comprehension of binding information between proteins and ligands, thereby generating potentially active drug candidate molecules. Theoretically, DrugGPT has significant advantages. During the model training process, DrugGPT aims to maximize the conditional probability and employs the back-propagation algorithm for training, making the training process more stable and avoiding the Mode Collapse problem that may occur in Generative Adversarial Networks in drug design. Furthermore, the design philosophy of DrugGPT endows it with strong generalization capabilities, giving it the potential to adapt to different tasks. In conclusion, DrugGPT provides a forward-thinking and practical new approach to ligand design. By optimizing the tokenizer and retraining the GPT-2 model, the ligand design process becomes more direct and efficient. This not only reflects the theoretical advantages of DrugGPT but also reveals its potential applications in the drug development process, thereby opening new perspectives and possibilities in the pharmaceutical field.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.14.545018">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.14.545018" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.14.545018">
        <p class="paperTitle">BGCFlow: Systematic pangenome workflow for the analysis of biosynthetic gene clusters across large genomic datasets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.14.545018" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.14.545018" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nuhamunada, M.; Mohite, O. S.; Phaneuf, P. V.; Palsson, B.; Weber, T.</p>
        <p class="info">Score: 39.2, Published: 2023-06-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.14.545018' target='https://doi.org/10.1101/2023.06.14.545018'> 10.1101/2023.06.14.545018</a></p>
        <p class="abstract">Genome mining is revolutionizing natural products discovery efforts. The rapid increase in available genomes demands comprehensive computational platforms to effectively extract biosynthetic knowledge encoded across bacterial pangenomes. Here, we present BGCFlow, a novel systematic workflow integrating analytics for large-scale genome mining of bacterial pangenomes. BGCFlow incorporates several genome analytics and mining tools grouped into five common stages of analysis such as; i) data selection, ii) functional annotation, iii) phylogenetic analysis, iv) genome mining, and v) comparative analysis. Furthermore, BGCFlow provides easy configuration of different projects, parallel distribution, scheduled job monitoring, an interactive database to visualize tables, exploratory Jupyter notebooks, and customized reports. Here, we demonstrate the application of BGCFlow by investigating the phylogenetic distribution of various biosynthetic gene clusters detected across 42 genomes of the Saccharopolyspora genus, known to produce industrially important secondary/specialized metabolites. The BGCFlow-guided analysis predicted more accurate dereplication of BGCs and guided the targeted comparative analysis of selected RiPPs. The scalable, interoperable, adaptable, re-entrant, and reproducible nature of the BGCFlow will provide an effective novel way to extract the biosynthetic knowledge in the ever-growing genomic datasets of biotechnologically relevant bacterial species. BGCFlow is available for downloading at https://github.com/NBChub/bgcflow.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545871">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545871" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545871">
        <p class="paperTitle">Co-linear Chaining on Pangenome Graphs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545871" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545871" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rajput, J.; Chandra, G.; Jain, C.</p>
        <p class="info">Score: 21.1, Published: 2023-06-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545871' target='https://doi.org/10.1101/2023.06.21.545871'> 10.1101/2023.06.21.545871</a></p>
        <p class="abstract">Pangenome reference graphs are useful in genomics because they compactly represent the genetic diversity within a species, a capability that linear references lack. However, efficiently aligning sequences to these graphs with complex topology and cycles can be challenging. The seed-chain-extend based alignment algorithms use co-linear chaining as a standard technique to identify a good cluster of exact seed matches that can be combined to form an alignment. Recent works show how the co-linear chaining problem can be efficiently solved for acyclic pangenome graphs by exploiting their small width [Makinen et al., TALG19] and how incorporating gap cost in the scoring function improves alignment accuracy [Chandra and Jain, RECOMB23]. However, it remains open on how to effectively generalize these techniques for general pangenome graphs which contain cycles. Here we present the first practical formulation and an exact algorithm for co-linear chaining on cyclic pangenome graphs. We rigorously prove the correctness and computational complexity of the proposed algorithm. We evaluate the empirical performance of our algorithm by aligning simulated long reads from the human genome to a cyclic pangenome graph constructed from 95 publicly available haplotype-resolved human genome assemblies. While the existing heuristic-based algorithms are faster, the proposed algorithm provides a significant advantage in terms of accuracy.

Implementationhttps://github.com/at-cg/PanAligner</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.20.545604">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.20.545604" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.20.545604">
        <p class="paperTitle">QuantUMS: uncertainty minimisation enables confident quantification in proteomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.20.545604" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.20.545604" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kistner, F.; Grossmann, J. L.; Sinn, L. R.; Demichev, V.</p>
        <p class="info">Score: 49.8, Published: 2023-06-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.20.545604' target='https://doi.org/10.1101/2023.06.20.545604'> 10.1101/2023.06.20.545604</a></p>
        <p class="abstract">Mass spectrometry-based proteomics has been rapidly gaining traction as a powerful analytical method both in basic research and translation. While the problem of error control in peptide and protein identification has been addressed extensively, the quality of the resulting quantities remains challenging to evaluate. Here we introduce QuantUMS (Quantification using an Uncertainty Minimising Solution), a machine learning-based method which minimises errors and eliminates bias in peptide and protein quantification by integrating multiple sources of quantitative information. In combination with data-independent acquisition proteomics, QuantUMS boosts accuracy and precision of quantities, as well as reports an uncertainty metric, enabling effective filtering of data for downstream analysis. The algorithm has linear complexity with respect to the number of mass spectrometry acquisitions in the experiment and is thus scalable to infinitely large proteomic experiments. For an easy implementation in a proteomics laboratory, we integrate QuantUMS in our automated DIA-NN software suite.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.07.542483">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.07.542483" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.07.542483">
        <p class="paperTitle">BrainRBPedia: a resource for RNA-binding proteins relevant to neurodevelopmental disorders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.07.542483" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.07.542483" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Han, K.; Wainberg, M.; Calarco, J. A.; Smibert, C. A.; Lipshitz, H. D.; Lee, H. O.; Tripathy, S. J.</p>
        <p class="info">Score: 23.8, Published: 2023-06-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.07.542483' target='https://doi.org/10.1101/2023.06.07.542483'> 10.1101/2023.06.07.542483</a></p>
        <p class="abstract">RNA-binding proteins (RBPs) are crucial players in the post-transcriptional regulation of mRNA and play major roles in ensuring proper neuronal development and function. Deficits in RBP function have been implicated in a number of neurodevelopmental disorders including autism spectrum disorder (ASD) and intellectual disability (ID), yet we lack resources that integrate current knowledge of RBP function, tissue expression, and disease association in one place to aid in their experimental characterization. Here we introduce BrainRBPedia - a database of 1072 RBPs with both disease annotations for neurodevelopmental disorders and functional annotations relevant to these disorders, including loss-of-function intolerance and expression specificity to the brain, neurons, and neuronal development. Using these functional annotations, we develop a machine learning model to prioritize RBPs likely to be involved in ASD and ID. Our model indicates that RBPs with high loss-of-function intolerance and those upregulated during neuronal differentiation are disproportionately likely to contribute to ASD and ID etiology. In summary, BrainRBPedia comprises a unique resource for researchers interested in the experimental characterization of RBPs in relation to neurodevelopmental disorders and suggests functional signatures of RBPs likely to play a role in neurodevelopment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.22.546080">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.22.546080" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.22.546080">
        <p class="paperTitle">DeepRank-GNN-esm: A Graph Neural Network for Scoring Protein-Protein Models using Protein Language Model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.22.546080" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.22.546080" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xu, X.; Bonvin, A. M.</p>
        <p class="info">Score: 18.8, Published: 2023-06-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.22.546080' target='https://doi.org/10.1101/2023.06.22.546080'> 10.1101/2023.06.22.546080</a></p>
        <p class="abstract">MotivationProtein-Protein interactions (PPIs) play critical roles in numerous cellular processes. By modelling the three-dimensional structures of the correspond protein complexes valuable insights can be obtained, providing, for example, starting points for drug and protein design. One challenge in the modelling process is however the identification of near-native models from the large pool of generated models. To this end we previously developed DeepRank-GNN, a graph neural network that integrates structural and sequence information to enable effective pattern learning at PPI interfaces. Its main features are related to the Position Specific Scoring Matrices (PSSM), which are computationally expensive to generate and significantly limit the algorithms usability.

ResultsWe introduce here DeepRank-GNN-esm that includes as additional features protein language model embeddings from the EMS-2 model. We show that the ESM-2 embeddings can actually replace the PSSM features at no cost in-, or even better performance on two PPI-related tasks: scoring docking poses and detecting crystal artifacts. This new DeepRank version bypasses thus the need of generating PSSM, greatly improving the usability of the software and opening new application opportunities for systems for which PSSM profiles cannot be obtained or are irrelevant (e.g. antibody-antigen complexes).

Availability and implementationDeepRank-GNN-esm is freely available from https://github.com/DeepRank/DeepRank-GNN-esm</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.22.546069">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.22.546069" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.22.546069">
        <p class="paperTitle">Accurate prediction of protein assembly structure by combining AlphaFold and symmetrical docking</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.22.546069" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.22.546069" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jeppesen, M.; Andre, I.</p>
        <p class="info">Score: 16.7, Published: 2023-06-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.22.546069' target='https://doi.org/10.1101/2023.06.22.546069'> 10.1101/2023.06.22.546069</a></p>
        <p class="abstract">AlphaFold can predict the structures of monomeric and multimeric proteins with high accuracy but has a limit on the number of chains and residues it can fold. Here we show that a combination of AlphaFold and all-atom symmetric docking simulations enables highly accurate prediction of the structure of complex symmetrical assemblies. We present a method to predict the structure of complexes with cubic - tetrahedral, octahedral and icosahedral - symmetry from sequence. Focusing on proteins where AlphaFold can make confident predictions on the subunit structure, 21 cubic systems were assembled with a median TM-score of 0.99 and a DockQ score of 0.71. 15 had TM-scores of above 0.8 and were categorized as high-quality according to DockQ. The resulting models are energetically optimized and can be used for detailed studies of intermolecular interactions in higher-order symmetrical assemblies. The results demonstrate how explicit treatment of structural symmetry can significantly expand the size and complexity of AlphaFold predictions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545875">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545875" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545875">
        <p class="paperTitle">Fractional Hitting Sets for Efficient and Lightweight Genomic Data Sketching</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545875" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545875" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rouze, T.; Martayan, I.; MARCHET, C.; Limasset, A.</p>
        <p class="info">Score: 13.9, Published: 2023-06-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545875' target='https://doi.org/10.1101/2023.06.21.545875'> 10.1101/2023.06.21.545875</a></p>
        <p class="abstract">The exponential increase in publicly available sequencing data and genomic resources necessitates the development of highly efficient methods for data processing and analysis. Locality-sensitive hashing techniques have successfully transformed large datasets into smaller, more manageable sketches while maintaining comparability using metrics such as Jaccard and containment indices. However, fixed-size sketches encounter difficulties when applied to divergent datasets.

Scalable sketching methods, such as Sourmash, provide valuable solutions but still lack resourceefficient, tailored indexing. Our objective is to create lighter sketches with comparable results while enhancing efficiency. We introduce the concept of Fractional Hitting Sets, a generalization of Universal Hitting Sets, which uniformly cover a specified fraction of the k-mer space. In theory and practice, we demonstrate the feasibility of achieving such coverage with simple but highly efficient schemes.

By encoding the covered k-mers as super-k-mers, we provide a space-efficient exact representation that also enables optimized comparisons. Our novel tool, SuperSampler, implements this scheme, and experimental results with real bacterial collections closely match our theoretical findings.

In comparison to Sourmash, SuperSampler achieves similar outcomes while utilizing an order of magnitude less space and memory and operating several times faster. This highlights the potential of our approach in addressing the challenges presented by the ever-expanding landscape of genomic data.

SuperSampler is an open-source software and can be accessed at github.com/TimRouze/supersampler. The data required to reproduce the results presented in this manuscript is available at github.com/TimRouze/Expe_SPSP.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.22.546084">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.22.546084" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.22.546084">
        <p class="paperTitle">Fine-tuning Protein Embeddings for GeneralizableAnnotation Propagation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.22.546084" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.22.546084" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dickson, A. M.; Mofrad, M. R. K.</p>
        <p class="info">Score: 11.9, Published: 2023-06-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.22.546084' target='https://doi.org/10.1101/2023.06.22.546084'> 10.1101/2023.06.22.546084</a></p>
        <p class="abstract">A central goal of bioinformatics research is to understand proteins on a functional level, typically by extrapolating from experimental results with the protein sequence information. One strategy is to assume that proteins with similar sequences will also share function. This has the benefit of being interpretable; it gives a very clear idea of why a protein might have a particular function by comparing with the most similar reference example. However, direct machine learning classifiers now outperform pure sequence similarity methods in raw prediction ability. A hybrid method is to use pre-trained language models to create protein embeddings, and then indirectly predict protein function using their relative similarity. We find that fine-tuning an auxiliary objective on protein function indirectly improves these hy-brid methods, to the point that they are in some cases better than direct classifiers. Our empirical results demonstrate that interpretable protein comparison models can be developed using fine-tuning techniques, without cost, or even with some benefit, to overall performance. K-nearest neighbors (KNN) embedding-based models also offer free generalization to previously unknown classes, while continuing to outperform only pre-trained models, further demonstrating the potential of fine-tuned embeddings outside of direct classification.

Significance StatementFor the purposes of biological understanding, or engineering, it is particularly important that protein annotation is done through interpretable means, such as comparison to existing proteins with known properties. Our work builds upon previous efforts to do so by embedding proteins into high-dimensional vectors with pre-trained protein language models. We demonstrate that further training of these models on protein function classification drastically improves their performance, while allowing for exactly the same similarity based classifications on both known and novel potential functions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546547">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546547" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546547">
        <p class="paperTitle">Scalable Integration of Multiomic Single Cell Data Using Generative Adversarial Networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546547" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546547" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Giansanti, V.; Giannese, F.; Botrugno, O. A.; Gandolfi, G.; Balestrieri, C.; Antoniotti, M.; Tonon, G.; Cittaro, D.</p>
        <p class="info">Score: 11.6, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546547' target='https://doi.org/10.1101/2023.06.26.546547'> 10.1101/2023.06.26.546547</a></p>
        <p class="abstract">Single cell profiling has become a common practice to investigate the complexity of tissues, organs and organisms. Recent technological advances are expanding our capabilities to profile various molecular layers beyond the transcriptome such as, but not limited to, the genome, the epigenome and the proteome. Depending on the experimental procedure, these data can be obtained from separate assays or from the very same cells. Despite development of computational methods for data integration is an active research field, most of the available strategies have been devised for the joint analysis of two modalities and cannot accommodate a high number of them. To solve this problem, we here propose a multiomic data integration framework based on Wasserstein Generative Adversarial Networks (MOWGAN) suitable for the analysis of paired or unpaired data with high number of modalities (&gt;2). At the core of our strategy is a single network trained on all modalities together, limiting the computational burden when many molecular layers are evaluated. Source code of our framework is available at https://github.com/vgiansanti/MOWGAN.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
