<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Benchmarking computational methods for single-cell chromatin data analysis
Authors: Luo, S.; Germain, P.-L.; Robinson, M. D.; von Meyenn, F.
Score: 22.2, Published: 2023-08-07 DOI: 10.1101/2023.08.04.552046
Single-cell chromatin accessibility assays, such as scATAC-seq, are increasingly employed in individual and joint multi-omic profiling of single cells. As the accumulation of scATAC-seq and multi-omics datasets continue, challenges in analyzing such sparse, noisy, and high-dimensional data become pressing. Specifically, one challenge relates to optimizing the processing of chromatin-level measurements and efficiently extracting information to discern cellular heterogeneity.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="Benchmarking computational methods for single-cell chromatin data analysis
Authors: Luo, S.; Germain, P.-L.; Robinson, M. D.; von Meyenn, F.
Score: 22.2, Published: 2023-08-07 DOI: 10.1101/2023.08.04.552046
Single-cell chromatin accessibility assays, such as scATAC-seq, are increasingly employed in individual and joint multi-omic profiling of single cells. As the accumulation of scATAC-seq and multi-omics datasets continue, challenges in analyzing such sparse, noisy, and high-dimensional data become pressing. Specifically, one challenge relates to optimizing the processing of chromatin-level measurements and efficiently extracting information to discern cellular heterogeneity." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-13T10:36:01+00:00" />
<meta property="article:modified_time" content="2023-08-13T10:36:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="Benchmarking computational methods for single-cell chromatin data analysis
Authors: Luo, S.; Germain, P.-L.; Robinson, M. D.; von Meyenn, F.
Score: 22.2, Published: 2023-08-07 DOI: 10.1101/2023.08.04.552046
Single-cell chromatin accessibility assays, such as scATAC-seq, are increasingly employed in individual and joint multi-omic profiling of single cells. As the accumulation of scATAC-seq and multi-omics datasets continue, challenges in analyzing such sparse, noisy, and high-dimensional data become pressing. Specifically, one challenge relates to optimizing the processing of chromatin-level measurements and efficiently extracting information to discern cellular heterogeneity."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "Benchmarking computational methods for single-cell chromatin data analysis\nAuthors: Luo, S.; Germain, P.-L.; Robinson, M. D.; von Meyenn, F.\nScore: 22.2, Published: 2023-08-07 DOI: 10.1101/2023.08.04.552046\nSingle-cell chromatin accessibility assays, such as scATAC-seq, are increasingly employed in individual and joint multi-omic profiling of single cells. As the accumulation of scATAC-seq and multi-omics datasets continue, challenges in analyzing such sparse, noisy, and high-dimensional data become pressing. Specifically, one challenge relates to optimizing the processing of chromatin-level measurements and efficiently extracting information to discern cellular heterogeneity.",
  "keywords": [
    
  ],
  "articleBody": " Benchmarking computational methods for single-cell chromatin data analysis\nAuthors: Luo, S.; Germain, P.-L.; Robinson, M. D.; von Meyenn, F.\nScore: 22.2, Published: 2023-08-07 DOI: 10.1101/2023.08.04.552046\nSingle-cell chromatin accessibility assays, such as scATAC-seq, are increasingly employed in individual and joint multi-omic profiling of single cells. As the accumulation of scATAC-seq and multi-omics datasets continue, challenges in analyzing such sparse, noisy, and high-dimensional data become pressing. Specifically, one challenge relates to optimizing the processing of chromatin-level measurements and efficiently extracting information to discern cellular heterogeneity. This is of critical importance, since the identification of cell types is a fundamental step in current single-cell data analysis practices. We benchmarked 8 feature engineering pipelines derived from 5 recent methods to assess their ability to discover and discriminate cell types. By using 10 metrics calculated at the cell embedding, shared nearest neighbor graph, or partition levels, we evaluated the performance of each method at different data processing stages. This comprehensive approach allowed us to thoroughly understand the strengths and weaknesses of each method and the influence of parameter selection. Our analysis provides guidelines for choosing analysis methods for different datasets. Overall, feature aggregation, SnapATAC, and SnapATAC2 outperform latent semantic indexing-based methods. For datasets with complex cell-type structures, SnapATAC and SnapATAC2 are preferred. With large datasets, SnapATAC2 and ArchR are most scalable.\nComprehensive Structural Variant Detection: From Mosaic to Population-Level\nAuthors: Smolka, M.; Paulin, L. F.; Grochowski, C. M.; Horner, D. W.; Mahmoud, M.; Behera, S.; Kalef-Ezra, E.; Gandhi, M.; Hong, K.; Pehlivan, D.; Scholz, S. W.; Carvalho, C. M.; Proukakis, C.; Sedlazeck, F. J.\nScore: 70.3, Published: 2023-08-09 DOI: 10.1101/2022.04.04.487055\nLong-read Structural Variation (SV) calling remains a challenging but highly accurate way to identify complex genomic alterations. Here, we present Sniffles2, which is faster and more accurate than state-of-the-art SV caller across different coverages, sequencing technologies, and SV types. Furthermore, Sniffles2 solves the problem of family- to population-level SV calling to produce fully genotyped VCF files by introducing a gVCF file concept. Across 11 probands, we accurately identified causative SVs around MECP2, including highly complex alleles with three overlapping SVs. Sniffles2 also enables the detection of mosaic SVs in bulk long-read data. As a result, we successfully identified multiple mosaic SVs across a multiple system atrophy patient brain. The identified SV showed a remarkable diversity within the cingulate cortex, impacting both genes involved in neuron function and repetitive elements. In summary, we demonstrate the utility and versatility of Sniffles2 to identify SVs from the mosaic to population levels.\nPDBe CCDUtils: an RDKit-based toolkit for handling and analysing small molecules in the Protein Data Bank\nAuthors: Kunnakkattu, I. R.; Choudhary, P.; Pravda, L.; Nadzirin, N.; Smart, O. S.; Yuan, Q.; Anyango, S.; Nair, S.; Varadi, M.; Velankar, S.\nScore: 18.4, Published: 2023-08-07 DOI: 10.1101/2023.08.04.552003\nWhile the Protein Data Bank (PDB) contains a wealth of structural information on ligands bound to macromolecules, their analysis can be challenging due to the large amount and diversity of data. Here, we present PDBe CCDUtils, a versatile toolkit for processing and analysing small molecules from the PDB in PDBx/mmCIF format. PDBe CCDUtils provides streamlined access to all the metadata for small molecules in the PDB and offers a set of convenient methods to compute various properties using RDKit, such as 2D depictions, 3D conformers, physicochemical properties, scaffolds, common fragments and cross-references to small molecule databases using UniChem. The toolkit also provides methods for identifying all the covalently attached chemical components in a macromolecular structure and calculating similarity among small molecules. By providing a broad range of functionality, PDBe CCDUtils caters to the needs of researchers in cheminformatics, structural biology, bioinformatics and computational chemistry.\nMetaCerberus: distributed highly parallelized scalable HMM-based implementation for robust functional annotation across the tree of life\nAuthors: Figueroa, J. L.; Dhungel, E.; Brouwer, C. R.; White, R. A.\nScore: 16.6, Published: 2023-08-12 DOI: 10.1101/2023.08.10.552700\nSummary: MetaCerberus is an exclusive HMM/HMMER-based tool that is massively parallel, on low memory, and provides rapid scalable annotation for functional gene inference across genomes to metacommunities. It provides robust enumeration of functional genes and pathways across many current public databases including KEGG (KO), COGs, CAZy, FOAM, and viral specific databases (i.e., VOGs and PHROGs). In a direct comparison, MetaCerberus was twice as fast as EggNOG-Mapper, and produced better annotation of viruses, phages, and archaeal viruses than DRAM, PROKKA, or InterProScan. MetaCerberus annotates more KOs across domains when compared to DRAM, with a 186x smaller database and a third less memory. MetaCerberus is fully integrated with differential statistical tools (i.e., DESeq2 and edgeR), pathway enrichment (GAGE R), and Pathview R for quantitative elucidation of metabolic pathways. MetaCerberus implements the key to unlocking the biosphere across the tree of life at scale. Availability and implementation: MetaCerberus is written in Python and distributed under a BSD-3 license. The source code of MetaCerberus is freely available at https://github.com/raw-lab/metacerberus. Written in python 3 for both Linux and Mac OS X. MetaCerberus can also be easily installed using mamba create -n metacerberus -c bioconda -c conda-forge metacerberus. Contact: Richard Allen White III, UNC Charlotte, rwhit101@charlotte.edu Supplementary information: Supplementary data are available online.\nphytools 2.0: An updated R ecosystem for phylogenetic comparative methods (and other things)\nAuthors: Revell, L. J.\nScore: 208.4, Published: 2023-08-03 DOI: 10.1101/2023.03.08.531791\nPhylogenetic comparative methods comprise the general endeavor of using an estimated phylogenetic tree (or set of trees) to make secondary inferences: about trait evolution, diversification dynamics, biogeography, community ecology, and a wide range of other phenomena or processes. Over the past ten years or so, the phytools R package (Revell 2012) has grown to become an important research tools for phylogenetic comparative analysis. phytools is a diverse contributed R library now consisting of hundreds of different functions covering a variety of methods and purposes in phylogenetic biology. As of the time of writing, phytools included functionality for fitting models of trait evolution, for reconstructing ancestral states, for studying diversification on trees, and for visualizing phylogenies, comparative data, and fitted models, as well numerous other tasks related to phylogenetic biology. Here, I describe some significant features of and recent updates to phytools, while also illustrating several popular workflows of the phytools computational software.\nIdentifying small-molecules binding sites in RNA conformational ensembles with SHAMAN\nAuthors: Panei, F. P.; Gkeka, P.; Bonomi, M.\nScore: 16.3, Published: 2023-08-09 DOI: 10.1101/2023.08.08.552403\nThe rational targeting of RNA with small molecules is hampered by our still limited understanding of RNA structural and dynamic properties. Most in silico tools for binding site identification rely on static structures and therefore cannot face the challenges posed by the dynamic nature of RNA molecules. Here we present SHAMAN, a computational technique to identify potential small-molecule binding sites in RNA structural ensembles. SHAMAN enables exploring the conformational landscape of RNA with atomistic molecular dynamics and at the same time identifying RNA pockets in an efficient way with the aid of probes and enhanced-sampling techniques. In our benchmark composed of large, structured riboswitches as well as small, flexible viral RNAs, SHAMAN successfully identified all the experimentally resolved pockets and ranked them among the most favorite probe hotspots. Overall, SHAMAN sets a solid foundation for future drug design efforts targeting RNA with small molecules, effectively addressing the long-standing challenges in the field.\nAAontology: An ontology of amino acid scales for interpretable machine learning\nAuthors: Breimann, S.; Kamp, F.; Steiner, H.; Frishman, D.\nScore: 16.6, Published: 2023-08-05 DOI: 10.1101/2023.08.03.551768\nAmino acid scales are crucial for protein prediction tasks, many of them being curated in the AAindex database. Despite various clustering attempts to organize them and to better understand their relationships, these approaches lack the fine-grained classification necessary for satisfactory interpretability in many protein prediction problems. To address this issue, we developed AAontology--a two-level classification for 586 amino acid scales (mainly from AAindex) together with an in-depth analysis of their relations--using bag-of-word-based classification, clustering, and manual refinement over multiple iterations. AAontology organizes physicochemical scales into 8 categories and 67 subcategories, enhancing the interpretability of scale-based machine learning methods in protein bioinformatics. Thereby it enables researchers to gain a deeper biological insight. We anticipate that AAontology will be a building block to link amino acid properties with protein function and dysfunctions as well as aid informed decision-making in mutation analysis or protein drug design.\nCellGO: A novel deep learning-based framework and webserver for cell type-specific gene function interpretation\nAuthors: Li, P.; Wei, J.; Zhu, Y.\nScore: 15.3, Published: 2023-08-05 DOI: 10.1101/2023.08.02.551654\nInterpreting the function of genes and gene sets identified from omics experiments remains a challenge, as current pathway analysis tools often fail to account for complex interactions across genes and pathways under specific tissues and cell types. We introduce CellGO, a tool for cell type-specific gene functional analysis. CellGO employs a deep learning model to simulate signaling propagation within a cell, enabling the development of a heuristic pathway activity measuring system to identify cell type-specific active pathways given a single gene or a gene set. It is featured with additional functions to uncover pathway communities and the most active genes within pathways to facilitate mechanistic interpretation. This study demonstrated that CellGO can effectively capture cell type-specific pathways even when working with mixed cell-type markers. CellGOs performance was benchmarked using gene knockout datasets, and its implementation effectively infers the cell type-specific pathogenesis of risk genes associated with neurodevelopmental and neurodegenerative disorders, suggesting its potential in understanding complex polygenic diseases. CellGO is accessible through a python package and a four-mode web interface for interactive usage with pretrained models on 71 single-cell datasets from human and mouse fetal and postnatal brains.\nHighly accurate and robust protein sequence design with CarbonDesign\nAuthors: Ren, M.; Yu, C.; Bu, D.; Zhang, H.\nScore: 12.8, Published: 2023-08-07 DOI: 10.1101/2023.08.07.552204\nProtein sequence design, the inverse problem of protein structure prediction, plays a crucial role in protein engineering. Although recent deep learning-based methods have shown promising advancements, achieving accurate and robust protein sequence design remains an ongoing challenge. Here, we present CarbonDesign, a new approach that draws inspiration from successful ingredients of AlphaFold for protein structure prediction and makes significant and novel developments tailored specifically for protein sequence design. At its core, CarbonDesign explores Inverseformer, a novel network architecture adapted from AlphaFolds Evoformer, to learn representations from backbone structures and an amortized Markov Random Fields model for sequence decoding. Moreover, we incorporate other essential AlphaFold concepts into CarbonDesign: an end-to-end network recycling technique to leverage evolutionary constraints in protein language models and a multi-task learning technique to generate side chain structures corresponding to the designed sequences. Through rigorous evaluations on independent testing data sets, including the CAMEO and recent CASP15 data sets, as well as the predicted structures from AlphaFold, we show that CarbonDesign outperforms other published methods, achieving high accuracy in sequence generation. Moreover, it exhibits superior performance on de novo backbone structures obtained from recent diffusion generative models such as RFdiffusion and FrameDiff, highlighting its potential for enhancing de novo protein design. Notably, CarbonDesign also supports zero-shot prediction of the functional effects of sequence variants, indicating its potential application in directed evolution-based design. In summary, our results illustrate CarbonDesigns accurate and robust performance in protein sequence design, making it a promising tool for applications in bioengineering.\nphuEGO: A network-based method to reconstruct active signalling pathways from phosphoproteomics datasets\nAuthors: Giudice, G.; Chen, H.; Petsalaki, E.\nScore: 12.6, Published: 2023-08-07 DOI: 10.1101/2023.08.07.552249\nSignalling networks are critical for virtually all cell functions. Our current knowledge of cell signalling has been summarised in signalling pathway databases, which, while useful, are highly biassed towards well-studied processes, and dont capture context specific network wiring or pathway cross-talk. Mass spectrometry-based phosphoproteomics data can provide a more unbiased view of active cell signalling processes in a given context, however, it suffers from low signal-to-noise ratio and poor reproducibility across experiments. Methods to extract active signalling signatures from such data struggle to produce unbiased and interpretable networks that can be used for hypothesis generation and designing downstream experiments. Here we present phuEGO, which combines three-layer network propagation with ego network decomposition to provide small networks comprising active functional signalling modules. PhuEGO boosts the signal-to-noise ratio from global phosphoproteomics datasets, enriches the resulting networks for functional phosphosites and allows the improved comparison and integration across datasets. We applied phuEGO to five phosphoproteomics data sets from cell lines collected upon infection with SARS CoV2. PhuEGO was better able to identify common active functions across datasets and to point to a subnetwork enriched for known COVID-19 targets. Overall, phuEGO provides a tool to the community for the improved functional interpretation of global phosphoproteomics datasets.\n",
  "wordCount" : "2076",
  "inLanguage": "en",
  "datePublished": "2023-08-13T10:36:01Z",
  "dateModified": "2023-08-13T10:36:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on August 13, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.552046">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.552046" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.552046">
        <p class="paperTitle">Benchmarking computational methods for single-cell chromatin data analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.552046" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.552046" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Luo, S.; Germain, P.-L.; Robinson, M. D.; von Meyenn, F.</p>
        <p class="info">Score: 22.2, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.552046' target='https://doi.org/10.1101/2023.08.04.552046'> 10.1101/2023.08.04.552046</a></p>
        <p class="abstract">Single-cell chromatin accessibility assays, such as scATAC-seq, are increasingly employed in individual and joint multi-omic profiling of single cells. As the accumulation of scATAC-seq and multi-omics datasets continue, challenges in analyzing such sparse, noisy, and high-dimensional data become pressing. Specifically, one challenge relates to optimizing the processing of chromatin-level measurements and efficiently extracting information to discern cellular heterogeneity. This is of critical importance, since the identification of cell types is a fundamental step in current single-cell data analysis practices.

We benchmarked 8 feature engineering pipelines derived from 5 recent methods to assess their ability to discover and discriminate cell types. By using 10 metrics calculated at the cell embedding, shared nearest neighbor graph, or partition levels, we evaluated the performance of each method at different data processing stages. This comprehensive approach allowed us to thoroughly understand the strengths and weaknesses of each method and the influence of parameter selection.

Our analysis provides guidelines for choosing analysis methods for different datasets. Overall, feature aggregation, SnapATAC, and SnapATAC2 outperform latent semantic indexing-based methods. For datasets with complex cell-type structures, SnapATAC and SnapATAC2 are preferred. With large datasets, SnapATAC2 and ArchR are most scalable.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.04.04.487055">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.04.04.487055" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.04.04.487055">
        <p class="paperTitle">Comprehensive Structural Variant Detection: From Mosaic to Population-Level</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.04.04.487055" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.04.04.487055" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smolka, M.; Paulin, L. F.; Grochowski, C. M.; Horner, D. W.; Mahmoud, M.; Behera, S.; Kalef-Ezra, E.; Gandhi, M.; Hong, K.; Pehlivan, D.; Scholz, S. W.; Carvalho, C. M.; Proukakis, C.; Sedlazeck, F. J.</p>
        <p class="info">Score: 70.3, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.04.04.487055' target='https://doi.org/10.1101/2022.04.04.487055'> 10.1101/2022.04.04.487055</a></p>
        <p class="abstract">Long-read Structural Variation (SV) calling remains a challenging but highly accurate way to identify complex genomic alterations. Here, we present Sniffles2, which is faster and more accurate than state-of-the-art SV caller across different coverages, sequencing technologies, and SV types. Furthermore, Sniffles2 solves the problem of family- to population-level SV calling to produce fully genotyped VCF files by introducing a gVCF file concept. Across 11 probands, we accurately identified causative SVs around MECP2, including highly complex alleles with three overlapping SVs. Sniffles2 also enables the detection of mosaic SVs in bulk long-read data. As a result, we successfully identified multiple mosaic SVs across a multiple system atrophy patient brain. The identified SV showed a remarkable diversity within the cingulate cortex, impacting both genes involved in neuron function and repetitive elements. In summary, we demonstrate the utility and versatility of Sniffles2 to identify SVs from the mosaic to population levels.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.552003">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.552003" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.552003">
        <p class="paperTitle">PDBe CCDUtils: an RDKit-based toolkit for handling and analysing small molecules in the Protein Data Bank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.552003" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.552003" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kunnakkattu, I. R.; Choudhary, P.; Pravda, L.; Nadzirin, N.; Smart, O. S.; Yuan, Q.; Anyango, S.; Nair, S.; Varadi, M.; Velankar, S.</p>
        <p class="info">Score: 18.4, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.552003' target='https://doi.org/10.1101/2023.08.04.552003'> 10.1101/2023.08.04.552003</a></p>
        <p class="abstract">While the Protein Data Bank (PDB) contains a wealth of structural information on ligands bound to macromolecules, their analysis can be challenging due to the large amount and diversity of data. Here, we present PDBe CCDUtils, a versatile toolkit for processing and analysing small molecules from the PDB in PDBx/mmCIF format.

PDBe CCDUtils provides streamlined access to all the metadata for small molecules in the PDB and offers a set of convenient methods to compute various properties using RDKit, such as 2D depictions, 3D conformers, physicochemical properties, scaffolds, common fragments and cross-references to small molecule databases using UniChem. The toolkit also provides methods for identifying all the covalently attached chemical components in a macromolecular structure and calculating similarity among small molecules. By providing a broad range of functionality, PDBe CCDUtils caters to the needs of researchers in cheminformatics, structural biology, bioinformatics and computational chemistry.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552700">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552700" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552700">
        <p class="paperTitle">MetaCerberus: distributed highly parallelized scalable HMM-based implementation for robust functional annotation across the tree of life</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552700" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552700" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Figueroa, J. L.; Dhungel, E.; Brouwer, C. R.; White, R. A.</p>
        <p class="info">Score: 16.6, Published: 2023-08-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552700' target='https://doi.org/10.1101/2023.08.10.552700'> 10.1101/2023.08.10.552700</a></p>
        <p class="abstract">Summary: MetaCerberus is an exclusive HMM/HMMER-based tool that is massively parallel, on low memory, and provides rapid scalable annotation for functional gene inference across genomes to metacommunities. It provides robust enumeration of functional genes and pathways across many current public databases including KEGG (KO), COGs, CAZy, FOAM, and viral specific databases (i.e., VOGs and PHROGs). In a direct comparison, MetaCerberus was twice as fast as EggNOG-Mapper, and produced better annotation of viruses, phages, and archaeal viruses than DRAM, PROKKA, or InterProScan. MetaCerberus annotates more KOs across domains when compared to DRAM, with a 186x smaller database and a third less memory. MetaCerberus is fully integrated with differential statistical tools (i.e., DESeq2 and edgeR), pathway enrichment (GAGE R), and Pathview R for quantitative elucidation of metabolic pathways. MetaCerberus implements the key to unlocking the biosphere across the tree of life at scale. Availability and implementation: MetaCerberus is written in Python and distributed under a BSD-3 license. The source code of MetaCerberus is freely available at https://github.com/raw-lab/metacerberus. Written in python 3 for both Linux and Mac OS X. MetaCerberus can also be easily installed using mamba create -n metacerberus -c bioconda -c conda-forge metacerberus. Contact: Richard Allen White III, UNC Charlotte, rwhit101@charlotte.edu Supplementary information: Supplementary data are available online.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.03.08.531791">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.03.08.531791" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.03.08.531791">
        <p class="paperTitle">phytools 2.0: An updated R ecosystem for phylogenetic comparative methods (and other things)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.03.08.531791" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.03.08.531791" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Revell, L. J.</p>
        <p class="info">Score: 208.4, Published: 2023-08-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.03.08.531791' target='https://doi.org/10.1101/2023.03.08.531791'> 10.1101/2023.03.08.531791</a></p>
        <p class="abstract">Phylogenetic comparative methods comprise the general endeavor of using an estimated phylogenetic tree (or set of trees) to make secondary inferences: about trait evolution, diversification dynamics, biogeography, community ecology, and a wide range of other phenomena or processes. Over the past ten years or so, the phytools R package (Revell 2012) has grown to become an important research tools for phylogenetic comparative analysis. phytools is a diverse contributed R library now consisting of hundreds of different functions covering a variety of methods and purposes in phylogenetic biology. As of the time of writing, phytools included functionality for fitting models of trait evolution, for reconstructing ancestral states, for studying diversification on trees, and for visualizing phylogenies, comparative data, and fitted models, as well numerous other tasks related to phylogenetic biology. Here, I describe some significant features of and recent updates to phytools, while also illustrating several popular workflows of the phytools computational software.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552403">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552403" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552403">
        <p class="paperTitle">Identifying small-molecules binding sites in RNA conformational ensembles with SHAMAN</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552403" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552403" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Panei, F. P.; Gkeka, P.; Bonomi, M.</p>
        <p class="info">Score: 16.3, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552403' target='https://doi.org/10.1101/2023.08.08.552403'> 10.1101/2023.08.08.552403</a></p>
        <p class="abstract">The rational targeting of RNA with small molecules is hampered by our still limited understanding of RNA structural and dynamic properties. Most in silico tools for binding site identification rely on static structures and therefore cannot face the challenges posed by the dynamic nature of RNA molecules. Here we present SHAMAN, a computational technique to identify potential small-molecule binding sites in RNA structural ensembles. SHAMAN enables exploring the conformational landscape of RNA with atomistic molecular dynamics and at the same time identifying RNA pockets in an efficient way with the aid of probes and enhanced-sampling techniques. In our benchmark composed of large, structured riboswitches as well as small, flexible viral RNAs, SHAMAN successfully identified all the experimentally resolved pockets and ranked them among the most favorite probe hotspots. Overall, SHAMAN sets a solid foundation for future drug design efforts targeting RNA with small molecules, effectively addressing the long-standing challenges in the field.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.03.551768">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.03.551768" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.03.551768">
        <p class="paperTitle">AAontology: An ontology of amino acid scales for interpretable machine learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.03.551768" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.03.551768" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Breimann, S.; Kamp, F.; Steiner, H.; Frishman, D.</p>
        <p class="info">Score: 16.6, Published: 2023-08-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.03.551768' target='https://doi.org/10.1101/2023.08.03.551768'> 10.1101/2023.08.03.551768</a></p>
        <p class="abstract">Amino acid scales are crucial for protein prediction tasks, many of them being curated in the AAindex database. Despite various clustering attempts to organize them and to better understand their relationships, these approaches lack the fine-grained classification necessary for satisfactory interpretability in many protein prediction problems.

To address this issue, we developed AAontology--a two-level classification for 586 amino acid scales (mainly from AAindex) together with an in-depth analysis of their relations--using bag-of-word-based classification, clustering, and manual refinement over multiple iterations. AAontology organizes physicochemical scales into 8 categories and 67 subcategories, enhancing the interpretability of scale-based machine learning methods in protein bioinformatics. Thereby it enables researchers to gain a deeper biological insight. We anticipate that AAontology will be a building block to link amino acid properties with protein function and dysfunctions as well as aid informed decision-making in mutation analysis or protein drug design.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.551654">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.551654" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.551654">
        <p class="paperTitle">CellGO: A novel deep learning-based framework and webserver for cell type-specific gene function interpretation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.551654" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.551654" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, P.; Wei, J.; Zhu, Y.</p>
        <p class="info">Score: 15.3, Published: 2023-08-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.551654' target='https://doi.org/10.1101/2023.08.02.551654'> 10.1101/2023.08.02.551654</a></p>
        <p class="abstract">Interpreting the function of genes and gene sets identified from omics experiments remains a challenge, as current pathway analysis tools often fail to account for complex interactions across genes and pathways under specific tissues and cell types. We introduce CellGO, a tool for cell type-specific gene functional analysis. CellGO employs a deep learning model to simulate signaling propagation within a cell, enabling the development of a heuristic pathway activity measuring system to identify cell type-specific active pathways given a single gene or a gene set. It is featured with additional functions to uncover pathway communities and the most active genes within pathways to facilitate mechanistic interpretation. This study demonstrated that CellGO can effectively capture cell type-specific pathways even when working with mixed cell-type markers. CellGOs performance was benchmarked using gene knockout datasets, and its implementation effectively infers the cell type-specific pathogenesis of risk genes associated with neurodevelopmental and neurodegenerative disorders, suggesting its potential in understanding complex polygenic diseases. CellGO is accessible through a python package and a four-mode web interface for interactive usage with pretrained models on 71 single-cell datasets from human and mouse fetal and postnatal brains.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552204">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552204" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552204">
        <p class="paperTitle">Highly accurate and robust protein sequence design with CarbonDesign</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552204" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552204" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ren, M.; Yu, C.; Bu, D.; Zhang, H.</p>
        <p class="info">Score: 12.8, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552204' target='https://doi.org/10.1101/2023.08.07.552204'> 10.1101/2023.08.07.552204</a></p>
        <p class="abstract">Protein sequence design, the inverse problem of protein structure prediction, plays a crucial role in protein engineering. Although recent deep learning-based methods have shown promising advancements, achieving accurate and robust protein sequence design remains an ongoing challenge. Here, we present CarbonDesign, a new approach that draws inspiration from successful ingredients of AlphaFold for protein structure prediction and makes significant and novel developments tailored specifically for protein sequence design. At its core, CarbonDesign explores Inverseformer, a novel network architecture adapted from AlphaFolds Evoformer, to learn representations from backbone structures and an amortized Markov Random Fields model for sequence decoding. Moreover, we incorporate other essential AlphaFold concepts into CarbonDesign: an end-to-end network recycling technique to leverage evolutionary constraints in protein language models and a multi-task learning technique to generate side chain structures corresponding to the designed sequences. Through rigorous evaluations on independent testing data sets, including the CAMEO and recent CASP15 data sets, as well as the predicted structures from AlphaFold, we show that CarbonDesign outperforms other published methods, achieving high accuracy in sequence generation. Moreover, it exhibits superior performance on de novo backbone structures obtained from recent diffusion generative models such as RFdiffusion and FrameDiff, highlighting its potential for enhancing de novo protein design. Notably, CarbonDesign also supports zero-shot prediction of the functional effects of sequence variants, indicating its potential application in directed evolution-based design. In summary, our results illustrate CarbonDesigns accurate and robust performance in protein sequence design, making it a promising tool for applications in bioengineering.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552249">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552249" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552249">
        <p class="paperTitle">phuEGO: A network-based method to reconstruct active signalling pathways from phosphoproteomics datasets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552249" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552249" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Giudice, G.; Chen, H.; Petsalaki, E.</p>
        <p class="info">Score: 12.6, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552249' target='https://doi.org/10.1101/2023.08.07.552249'> 10.1101/2023.08.07.552249</a></p>
        <p class="abstract">Signalling networks are critical for virtually all cell functions. Our current knowledge of cell signalling has been summarised in signalling pathway databases, which, while useful, are highly biassed towards well-studied processes, and dont capture context specific network wiring or pathway cross-talk. Mass spectrometry-based phosphoproteomics data can provide a more unbiased view of active cell signalling processes in a given context, however, it suffers from low signal-to-noise ratio and poor reproducibility across experiments. Methods to extract active signalling signatures from such data struggle to produce unbiased and interpretable networks that can be used for hypothesis generation and designing downstream experiments.

Here we present phuEGO, which combines three-layer network propagation with ego network decomposition to provide small networks comprising active functional signalling modules. PhuEGO boosts the signal-to-noise ratio from global phosphoproteomics datasets, enriches the resulting networks for functional phosphosites and allows the improved comparison and integration across datasets. We applied phuEGO to five phosphoproteomics data sets from cell lines collected upon infection with SARS CoV2. PhuEGO was better able to identify common active functions across datasets and to point to a subnetwork enriched for known COVID-19 targets. Overall, phuEGO provides a tool to the community for the improved functional interpretation of global phosphoproteomics datasets.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
