<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="simona: a Comprehensive R package for Semantic Similarity Analysis on Bio-Ontologies
Authors: Gu, Z.
Score: 30.4, Published: 2023-12-05 DOI: 10.1101/2023.12.03.569758
Bio-ontologies are keys in structuring complex biological information for effective data integration and knowledge representation. In this paper, we present simona, a novel R package for semantic similarity analysis on general bio-ontolgies. Simona implements infrastructures for ontology analysis by offering efficient data structures, fast ontology traversal methods, and elegant visualizations.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="simona: a Comprehensive R package for Semantic Similarity Analysis on Bio-Ontologies
Authors: Gu, Z.
Score: 30.4, Published: 2023-12-05 DOI: 10.1101/2023.12.03.569758
Bio-ontologies are keys in structuring complex biological information for effective data integration and knowledge representation. In this paper, we present simona, a novel R package for semantic similarity analysis on general bio-ontolgies. Simona implements infrastructures for ontology analysis by offering efficient data structures, fast ontology traversal methods, and elegant visualizations." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-10T10:37:48+00:00" />
<meta property="article:modified_time" content="2023-12-10T10:37:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="simona: a Comprehensive R package for Semantic Similarity Analysis on Bio-Ontologies
Authors: Gu, Z.
Score: 30.4, Published: 2023-12-05 DOI: 10.1101/2023.12.03.569758
Bio-ontologies are keys in structuring complex biological information for effective data integration and knowledge representation. In this paper, we present simona, a novel R package for semantic similarity analysis on general bio-ontolgies. Simona implements infrastructures for ontology analysis by offering efficient data structures, fast ontology traversal methods, and elegant visualizations."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "simona: a Comprehensive R package for Semantic Similarity Analysis on Bio-Ontologies\nAuthors: Gu, Z.\nScore: 30.4, Published: 2023-12-05 DOI: 10.1101/2023.12.03.569758\nBio-ontologies are keys in structuring complex biological information for effective data integration and knowledge representation. In this paper, we present simona, a novel R package for semantic similarity analysis on general bio-ontolgies. Simona implements infrastructures for ontology analysis by offering efficient data structures, fast ontology traversal methods, and elegant visualizations.",
  "keywords": [
    
  ],
  "articleBody": " simona: a Comprehensive R package for Semantic Similarity Analysis on Bio-Ontologies\nAuthors: Gu, Z.\nScore: 30.4, Published: 2023-12-05 DOI: 10.1101/2023.12.03.569758\nBio-ontologies are keys in structuring complex biological information for effective data integration and knowledge representation. In this paper, we present simona, a novel R package for semantic similarity analysis on general bio-ontolgies. Simona implements infrastructures for ontology analysis by offering efficient data structures, fast ontology traversal methods, and elegant visualizations. Moreover, it provides a robust toolbox supporting over 70 methods for semantic similarity analysis. With simona, we conduct a benchmark against current semantic similarity methods. The results demonstrate methods are clustered based on their mathematical methodologies, providing guidance for researchers in the selection of appropriate methods. Additionally, we explore annotation-based versus topology-based methods, revealing that semantic similarities solely based on ontology topology can efficiently reveal semantic similarity structures, facilitating analysis on less-studied organisms and other ontologies. Simona is freely available from https://bioconductor.org/packages/simona/.\nDistance-AF: Modifying Predicted Protein Structure Models by Alphafold2 with User-Specified Distance Constraints\nAuthors: Zhang, Y.; Zhang, Z.; Kagaya, Y.; Terashi, G.; Zhao, B.; Xiong, Y.; Kihara, D.\nScore: 22.9, Published: 2023-12-04 DOI: 10.1101/2023.12.01.569498\nThe three-dimensional structure of a protein plays a fundamental role in determining its function and has an essential impact on understanding biological processes. Despite significant progress in protein structure prediction, such as AlphaFold2, challenges remain on those hard targets that Alphafold2 does not often perform well due to the complex folding of protein and a large number of possible conformations. Here we present a modified version of the AlphaFold2, called Distance-AF, which aims to improve the performance of AlphaFold2 by including distance constraints as input information. Distance-AF uses AlphaFold2s predicted structure as a starting point and incorporates distance constraints between amino acids to adjust folding of the protein structure until it meets the constraints. Distance-AF can correct the domain orientation on challenging targets, leading to more accurate structures with a lower root mean square deviation (RMSD). The ability of Distance-AF is also useful in fitting protein structures into cryo-electron microscopy maps.\nQuality assessment of splice site annotation based on conservation across multiple species\nAuthors: Minkin, I.; Salzberg, S.\nScore: 18.4, Published: 2023-12-02 DOI: 10.1101/2023.12.01.569581\nAO_SCPLOWBSTRACTC_SCPLOWDespite many improvements over the years, the annotation of the human genome remains imperfect, and even the best annotations of the human reference genome sometimes contradict one another. Hence, refinement of the human genome annotation is an important challenge. The use of evolutionarily conserved sequences provides a strategy for addressing this problem, and the rapidly growing number of genomes from other species increases the power of an evolution-driven approach. Using the latest large-scale whole genome alignment data, we found that splice sites from protein-coding genes in the high-quality MANE annotation are consistently conserved across more than 400 species. We also studied splice sites from the RefSeq, GENCODE, and CHESS databases that are not present in MANE, from both protein-coding genes and lncRNAs. We trained a logistic regression classifier to distinguish between the conservation patterns exhibited by splice sites from MANE versus sites that were flanked by the standard GT-AG dinucleotides, but that were chosen randomly from a sequence not under selection. We found that up to 70% of splice sites from annotated protein-coding transcripts outside of MANE exhibit conservation patterns closer to random sequence as opposed to highly-conserved splice sites from MANE. Our study highlights potentially erroneous splice sites that might require further scrutiny.\nBiolearn, an open-source library for biomarkers of aging\nAuthors: Ying, K.; Paulson, S.; Perez-Guevara, M.; Emamifar, M.; Casas Martinez, M.; Kwon, D.; Poganik, J. R.; Moqri, M.; Gladyshev, V. N.\nScore: 13.7, Published: 2023-12-04 DOI: 10.1101/2023.12.02.569722\nIdentifying and validating biomarkers of aging is pivotal for understanding the aging process and testing longevity interventions. Despite the development of numerous aging biomarkers, their clinical validation remains elusive, largely due to the lack of cross-population validation, which is hampered by disparate biomarker designs and inconsistencies in dataset structures. To bridge this gap, we introduce Biolearn, an innovative open-source library dedicated to the implementation and application of aging biomarkers. Biolearn facilitates (1) harmonization of existing aging biomarkers, while presenting a structured framework for novel biomarkers in standardized formats; (2) unification of public datasets, ensuring coherent structuring and formatting, thus simplifying cross-population validation studies; and (3) provision of computational methodologies to assess any harmonized biomarker against unified datasets. By furnishing a community-driven platform, Biolearn significantly augments the development, assessment, and validation trajectories of aging biomarkers, paving the way toward more rigorous clinical validation and, ultimately, application in clinical trials targeting healthy longevity. The Biolearn package is open-source and freely available at https://Bio-Learn.github.io/\nBenchmarking computational methods to identify spatially variable genes and peaks\nAuthors: Li, Z.; Patel, Z. M.; Song, D.; Yan, G.; Li, J. J.; Pinello, L.\nScore: 13.0, Published: 2023-12-03 DOI: 10.1101/2023.12.02.569717\nSpatially resolved transcriptomics offers unprecedented insight by enabling the profiling of gene expression within the intact spatial context of cells, effectively adding a new and essential dimension to data interpretation. To efficiently detect spatial structure of interest, an essential step in analyzing such data involves identifying spatially variable genes. Despite researchers having developed several computational methods to accomplish this task, the lack of a comprehensive benchmark evaluating their performance remains a considerable gap in the field. Here, we present a systematic evaluation of 14 methods using 60 simulated datasets generated by four different simulation strategies, 12 real-world transcriptomics, and three spatial ATAC-seq datasets. We find that spatialDE2 consistently outperforms the other benchmarked methods, and Morans I achieves competitive performance in different experimental settings. Moreover, our results reveal that more specialized algorithms are needed to identify spatially variable peaks.\nLearning sequence, structure, and function representations of proteins with language models\nAuthors: Hamamsy, T.; Barot, M.; Morton, J. T.; Steinegger, M.; Bonneau, R.; Cho, K.\nScore: 34.7, Published: 2023-11-26 DOI: 10.1101/2023.11.26.568742\nThe sequence-structure-function relationships that ultimately generate the diversity of extant observed proteins is complex, as proteins bridge the gap between multiple informational and physical scales involved in nearly all cellular processes. One limitation of existing protein annotation databases such as UniProt is that less than 1% of proteins have experimentally verified functions, and computational methods are needed to fill in the missing information. Here, we demonstrate that a multi-aspect framework based on protein language models can learn sequence-structure-function representations of amino acid sequences, and can provide the foundation for sensitive sequence-structure-function aware protein sequence search and annotation. Based on this model, we introduce a multi-aspect information retrieval system for proteins, Protein-Vec, covering sequence, structure, and function aspects, that enables computational protein annotation and function prediction at tree-of-life scales.\nPROTEOME-SCALE RECOMBINANT STANDARDS AND A ROBUST HIGH-SPEED SEARCH ENGINE TO ADVANCE CROSS-LINKING MS-BASED INTERACTOMICS\nAuthors: Clasen, M. A.; Ruwolt, M.; Kurt, L. U.; Gozzo, F. C.; Wang, S.; Chen, T.; Carvalho, P. C.; Lima, D. B.; Liu, F.\nScore: 10.7, Published: 2023-12-02 DOI: 10.1101/2023.11.30.569448\nAO_SCPLOWBSTRACTC_SCPLOWAdvancing data analysis tools for proteome-wide cross-linking mass spectrometry (XL-MS) requires ground-truth standards that mimic biological complexity. Here, we develop wellcontrolled XL-MS standards comprising hundreds of recombinant proteins that are systematically mixed for cross-linking. We use one standard dataset to guide the development of Scout, a search engine for XL-MS with MS-cleavable cross-linkers. Using other, independent datasets from our standards as well as published datasets, we benchmark the performance of Scout and existing XL-MS software. This demonstrates that Scout offers the best combination of speed, sensitivity, and false-discovery rate control. These results illustrate how our large recombinant standards can support the development of XL-MS analysis tools and evaluation of XL-MS results.\nContextualized Networks Reveal Heterogeneous Transcriptomic Regulation in Tumors at Sample-Specific Resolution\nAuthors: Ellington, C. N.; Lengerich, B. J.; Watkins, T. B.; Yang, J.; Xiao, H.; Kellis, M.; Xing, E. P.\nScore: 10.5, Published: 2023-12-04 DOI: 10.1101/2023.12.01.569658\nCancers are shaped by somatic mutations, microenvironment, and patient background, each altering gene expression and regulation in complex ways, resulting in heterogeneous cellular states and dynamics. Inferring gene regulatory network (GRN) models from expression data can help characterize this regulation-driven heterogeneity, but network inference requires many statistical samples, traditionally limiting GRNs to cluster-level analyses that ignore intra-cluster heterogeneity. We propose to move beyond cluster-based analyses by using contextualized learning, a multi-task learning paradigm which allows us to infer sample-specific models using phenotypic, molecular, and environmental information pertinent to the model, encoded as the models \"context\" to be conditioned on. We unify three network model classes (Correlation, Markov, Neighborhood) and estimate context-specific GRNs for 7997 tumors across 25 tumor types, with each network contextualized by copy number and driver mutation profiles, tumor microenvironment, and patient demographics. Contextualized GRNs provide a structured view of expression dynamics at sample-specific resolution, which reveal co-expression modules in correlation networks (CNs), as well as cliques and independent regulatory elements in Markov Networks (MNs) and Neighborhood Regression Networks (NNs). Our generative modeling approach allows us to predict GRNs for unseen tumor types based on a pan-cancer model of how somatic mutations affect gene regulation. Finally, contextualized networks enable GRN-based precision oncology, explaining known biomarkers in terms of network-mediated effects, and leading to novel subtypings for thyroid, brain, and gastrointestinal tumors that improve survival prognosis.\nFrustraevo: A Web Server To Localize And Quantify The Conservation Of Local Energetic Frustration In Protein Families\nAuthors: Parra, R. G.; Freiberger, M. I.; Poley-Gil, M.; Fernandez-Martin, M.; Radusky, L. G.; Wolynes, P. G.; Ferreiro, D. U.; Valencia, A.\nScore: 10.5, Published: 2023-12-01 DOI: 10.1101/2023.11.29.569273\nAccording to the Principle of Minimal Frustration, folded proteins can have only a minimal number of strong energetic conflicts in their native states. However, not all interactions are energetically optimized for folding but some remain in energetic conflict, i.e. they are highly frustrated. This remaining local energetic frustration has been shown to be statistically correlated with distinct functional aspects such as protein-protein interaction sites, allosterism and catalysis. Fuelled by the recent breakthroughs in efficient protein structure prediction that have made available good quality models for most proteins, we have developed a strategy to calculate local energetic frustration within large protein families and quantify its conservation over evolutionary time. Based on this evolutionary information we can identify how stability and functional constraints have appeared at the common ancestor of the family and have been maintained over the course of evolution. Here, we present FrustraEvo, a web server tool to calculate and quantify the conservation of local energetic frustration in protein families. The webserver is freely available at URL: https://frustraevo.qb.fcen.uba.ar\nCytoSimplex: Visualizing Single-cell Fates and Transitions on a Simplex\nAuthors: Liu, J.; Wang, Y.; Li, C.; Gu, Y.; Ono, N.; Welch, J. D.\nScore: 9.2, Published: 2023-12-08 DOI: 10.1101/2023.12.07.570655\nCells differentiate to their final fates along unique trajectories, often involving multi-potent progenitors that can produce multiple terminally differentiated cell types. Recent developments in single-cell transcriptomic and epigenomic measurement provide tremendous opportunities for mapping these trajectories. The visualization of single-cell data often relies on dimension reduction methods such as UMAP to simplify high-dimensional single-cell data down into an understandable two-dimensional (2D) form. However, these visualization methods can be misleading and often do not effectively represent the direction of cell differentiation. To address these limitations, we developed a new approach that places each cell from a single-cell dataset within a simplex whose vertices correspond to terminally differentiated cell types. Our approach can quantify and visualize current cell fate commitment and future cell potential. We developed CytoSimplex, a standalone open-source package implemented in R and Python that provides simple and intuitive visualizations of cell differentiation in 2D ternary and three-dimensional (3D) quaternary plots. We believe that CytoSimplex can help researchers gain a better understanding of cell type transitions in specific tissues and characterize developmental processes.\n",
  "wordCount" : "1921",
  "inLanguage": "en",
  "datePublished": "2023-12-10T10:37:48Z",
  "dateModified": "2023-12-10T10:37:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 10, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.03.569758">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.03.569758" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.03.569758">
        <p class="paperTitle">simona: a Comprehensive R package for Semantic Similarity Analysis on Bio-Ontologies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.03.569758" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.03.569758" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gu, Z.</p>
        <p class="info">Score: 30.4, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.03.569758' target='https://doi.org/10.1101/2023.12.03.569758'> 10.1101/2023.12.03.569758</a></p>
        <p class="abstract">Bio-ontologies are keys in structuring complex biological information for effective data integration and knowledge representation. In this paper, we present simona, a novel R package for semantic similarity analysis on general bio-ontolgies. Simona implements infrastructures for ontology analysis by offering efficient data structures, fast ontology traversal methods, and elegant visualizations. Moreover, it provides a robust toolbox supporting over 70 methods for semantic similarity analysis. With simona, we conduct a benchmark against current semantic similarity methods. The results demonstrate methods are clustered based on their mathematical methodologies, providing guidance for researchers in the selection of appropriate methods. Additionally, we explore annotation-based versus topology-based methods, revealing that semantic similarities solely based on ontology topology can efficiently reveal semantic similarity structures, facilitating analysis on less-studied organisms and other ontologies. Simona is freely available from https://bioconductor.org/packages/simona/.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569498">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569498" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569498">
        <p class="paperTitle">Distance-AF: Modifying Predicted Protein Structure Models by Alphafold2 with User-Specified Distance Constraints</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569498" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569498" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, Y.; Zhang, Z.; Kagaya, Y.; Terashi, G.; Zhao, B.; Xiong, Y.; Kihara, D.</p>
        <p class="info">Score: 22.9, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569498' target='https://doi.org/10.1101/2023.12.01.569498'> 10.1101/2023.12.01.569498</a></p>
        <p class="abstract">The three-dimensional structure of a protein plays a fundamental role in determining its function and has an essential impact on understanding biological processes. Despite significant progress in protein structure prediction, such as AlphaFold2, challenges remain on those hard targets that Alphafold2 does not often perform well due to the complex folding of protein and a large number of possible conformations. Here we present a modified version of the AlphaFold2, called Distance-AF, which aims to improve the performance of AlphaFold2 by including distance constraints as input information. Distance-AF uses AlphaFold2s predicted structure as a starting point and incorporates distance constraints between amino acids to adjust folding of the protein structure until it meets the constraints. Distance-AF can correct the domain orientation on challenging targets, leading to more accurate structures with a lower root mean square deviation (RMSD). The ability of Distance-AF is also useful in fitting protein structures into cryo-electron microscopy maps.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569581">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569581" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569581">
        <p class="paperTitle">Quality assessment of splice site annotation based on conservation across multiple species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569581" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569581" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Minkin, I.; Salzberg, S.</p>
        <p class="info">Score: 18.4, Published: 2023-12-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569581' target='https://doi.org/10.1101/2023.12.01.569581'> 10.1101/2023.12.01.569581</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWDespite many improvements over the years, the annotation of the human genome remains imperfect, and even the best annotations of the human reference genome sometimes contradict one another. Hence, refinement of the human genome annotation is an important challenge. The use of evolutionarily conserved sequences provides a strategy for addressing this problem, and the rapidly growing number of genomes from other species increases the power of an evolution-driven approach. Using the latest large-scale whole genome alignment data, we found that splice sites from protein-coding genes in the high-quality MANE annotation are consistently conserved across more than 400 species. We also studied splice sites from the RefSeq, GENCODE, and CHESS databases that are not present in MANE, from both protein-coding genes and lncRNAs. We trained a logistic regression classifier to distinguish between the conservation patterns exhibited by splice sites from MANE versus sites that were flanked by the standard GT-AG dinucleotides, but that were chosen randomly from a sequence not under selection. We found that up to 70% of splice sites from annotated protein-coding transcripts outside of MANE exhibit conservation patterns closer to random sequence as opposed to highly-conserved splice sites from MANE. Our study highlights potentially erroneous splice sites that might require further scrutiny.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.02.569722">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.02.569722" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.02.569722">
        <p class="paperTitle">Biolearn, an open-source library for biomarkers of aging</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.02.569722" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.02.569722" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ying, K.; Paulson, S.; Perez-Guevara, M.; Emamifar, M.; Casas Martinez, M.; Kwon, D.; Poganik, J. R.; Moqri, M.; Gladyshev, V. N.</p>
        <p class="info">Score: 13.7, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.02.569722' target='https://doi.org/10.1101/2023.12.02.569722'> 10.1101/2023.12.02.569722</a></p>
        <p class="abstract">Identifying and validating biomarkers of aging is pivotal for understanding the aging process and testing longevity interventions. Despite the development of numerous aging biomarkers, their clinical validation remains elusive, largely due to the lack of cross-population validation, which is hampered by disparate biomarker designs and inconsistencies in dataset structures. To bridge this gap, we introduce Biolearn, an innovative open-source library dedicated to the implementation and application of aging biomarkers. Biolearn facilitates (1) harmonization of existing aging biomarkers, while presenting a structured framework for novel biomarkers in standardized formats; (2) unification of public datasets, ensuring coherent structuring and formatting, thus simplifying cross-population validation studies; and (3) provision of computational methodologies to assess any harmonized biomarker against unified datasets. By furnishing a community-driven platform, Biolearn significantly augments the development, assessment, and validation trajectories of aging biomarkers, paving the way toward more rigorous clinical validation and, ultimately, application in clinical trials targeting healthy longevity. The Biolearn package is open-source and freely available at https://Bio-Learn.github.io/</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.02.569717">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.02.569717" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.02.569717">
        <p class="paperTitle">Benchmarking computational methods to identify spatially variable genes and peaks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.02.569717" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.02.569717" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, Z.; Patel, Z. M.; Song, D.; Yan, G.; Li, J. J.; Pinello, L.</p>
        <p class="info">Score: 13.0, Published: 2023-12-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.02.569717' target='https://doi.org/10.1101/2023.12.02.569717'> 10.1101/2023.12.02.569717</a></p>
        <p class="abstract">Spatially resolved transcriptomics offers unprecedented insight by enabling the profiling of gene expression within the intact spatial context of cells, effectively adding a new and essential dimension to data interpretation. To efficiently detect spatial structure of interest, an essential step in analyzing such data involves identifying spatially variable genes. Despite researchers having developed several computational methods to accomplish this task, the lack of a comprehensive benchmark evaluating their performance remains a considerable gap in the field. Here, we present a systematic evaluation of 14 methods using 60 simulated datasets generated by four different simulation strategies, 12 real-world transcriptomics, and three spatial ATAC-seq datasets. We find that spatialDE2 consistently outperforms the other benchmarked methods, and Morans I achieves competitive performance in different experimental settings. Moreover, our results reveal that more specialized algorithms are needed to identify spatially variable peaks.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.26.568742">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.26.568742" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.26.568742">
        <p class="paperTitle">Learning sequence, structure, and function representations of proteins with language models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.26.568742" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.26.568742" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hamamsy, T.; Barot, M.; Morton, J. T.; Steinegger, M.; Bonneau, R.; Cho, K.</p>
        <p class="info">Score: 34.7, Published: 2023-11-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.26.568742' target='https://doi.org/10.1101/2023.11.26.568742'> 10.1101/2023.11.26.568742</a></p>
        <p class="abstract">The sequence-structure-function relationships that ultimately generate the diversity of extant observed proteins is complex, as proteins bridge the gap between multiple informational and physical scales involved in nearly all cellular processes. One limitation of existing protein annotation databases such as UniProt is that less than 1% of proteins have experimentally verified functions, and computational methods are needed to fill in the missing information. Here, we demonstrate that a multi-aspect framework based on protein language models can learn sequence-structure-function representations of amino acid sequences, and can provide the foundation for sensitive sequence-structure-function aware protein sequence search and annotation. Based on this model, we introduce a multi-aspect information retrieval system for proteins, Protein-Vec, covering sequence, structure, and function aspects, that enables computational protein annotation and function prediction at tree-of-life scales.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569448">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569448" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569448">
        <p class="paperTitle">PROTEOME-SCALE RECOMBINANT STANDARDS AND A ROBUST HIGH-SPEED SEARCH ENGINE TO ADVANCE CROSS-LINKING MS-BASED INTERACTOMICS</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569448" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569448" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Clasen, M. A.; Ruwolt, M.; Kurt, L. U.; Gozzo, F. C.; Wang, S.; Chen, T.; Carvalho, P. C.; Lima, D. B.; Liu, F.</p>
        <p class="info">Score: 10.7, Published: 2023-12-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569448' target='https://doi.org/10.1101/2023.11.30.569448'> 10.1101/2023.11.30.569448</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWAdvancing data analysis tools for proteome-wide cross-linking mass spectrometry (XL-MS) requires ground-truth standards that mimic biological complexity. Here, we develop wellcontrolled XL-MS standards comprising hundreds of recombinant proteins that are systematically mixed for cross-linking. We use one standard dataset to guide the development of Scout, a search engine for XL-MS with MS-cleavable cross-linkers. Using other, independent datasets from our standards as well as published datasets, we benchmark the performance of Scout and existing XL-MS software. This demonstrates that Scout offers the best combination of speed, sensitivity, and false-discovery rate control. These results illustrate how our large recombinant standards can support the development of XL-MS analysis tools and evaluation of XL-MS results.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569658">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569658" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569658">
        <p class="paperTitle">Contextualized Networks Reveal Heterogeneous Transcriptomic Regulation in Tumors at Sample-Specific Resolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569658" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569658" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ellington, C. N.; Lengerich, B. J.; Watkins, T. B.; Yang, J.; Xiao, H.; Kellis, M.; Xing, E. P.</p>
        <p class="info">Score: 10.5, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569658' target='https://doi.org/10.1101/2023.12.01.569658'> 10.1101/2023.12.01.569658</a></p>
        <p class="abstract">Cancers are shaped by somatic mutations, microenvironment, and patient background, each altering gene expression and regulation in complex ways, resulting in heterogeneous cellular states and dynamics. Inferring gene regulatory network (GRN) models from expression data can help characterize this regulation-driven heterogeneity, but network inference requires many statistical samples, traditionally limiting GRNs to cluster-level analyses that ignore intra-cluster heterogeneity. We propose to move beyond cluster-based analyses by using contextualized learning, a multi-task learning paradigm which allows us to infer sample-specific models using phenotypic, molecular, and environmental information pertinent to the model, encoded as the models &#34;context&#34; to be conditioned on. We unify three network model classes (Correlation, Markov, Neighborhood) and estimate context-specific GRNs for 7997 tumors across 25 tumor types, with each network contextualized by copy number and driver mutation profiles, tumor microenvironment, and patient demographics. Contextualized GRNs provide a structured view of expression dynamics at sample-specific resolution, which reveal co-expression modules in correlation networks (CNs), as well as cliques and independent regulatory elements in Markov Networks (MNs) and Neighborhood Regression Networks (NNs). Our generative modeling approach allows us to predict GRNs for unseen tumor types based on a pan-cancer model of how somatic mutations affect gene regulation. Finally, contextualized networks enable GRN-based precision oncology, explaining known biomarkers in terms of network-mediated effects, and leading to novel subtypings for thyroid, brain, and gastrointestinal tumors that improve survival prognosis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569273">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569273" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569273">
        <p class="paperTitle">Frustraevo: A Web Server To Localize And Quantify The Conservation Of Local Energetic Frustration In Protein Families</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569273" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569273" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Parra, R. G.; Freiberger, M. I.; Poley-Gil, M.; Fernandez-Martin, M.; Radusky, L. G.; Wolynes, P. G.; Ferreiro, D. U.; Valencia, A.</p>
        <p class="info">Score: 10.5, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569273' target='https://doi.org/10.1101/2023.11.29.569273'> 10.1101/2023.11.29.569273</a></p>
        <p class="abstract">According to the Principle of Minimal Frustration, folded proteins can have only a minimal number of strong energetic conflicts in their native states. However, not all interactions are energetically optimized for folding but some remain in energetic conflict, i.e. they are highly frustrated. This remaining local energetic frustration has been shown to be statistically correlated with distinct functional aspects such as protein-protein interaction sites, allosterism and catalysis. Fuelled by the recent breakthroughs in efficient protein structure prediction that have made available good quality models for most proteins, we have developed a strategy to calculate local energetic frustration within large protein families and quantify its conservation over evolutionary time. Based on this evolutionary information we can identify how stability and functional constraints have appeared at the common ancestor of the family and have been maintained over the course of evolution. Here, we present FrustraEvo, a web server tool to calculate and quantify the conservation of local energetic frustration in protein families.

The webserver is freely available at URL: https://frustraevo.qb.fcen.uba.ar</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570655">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570655" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570655">
        <p class="paperTitle">CytoSimplex: Visualizing Single-cell Fates and Transitions on a Simplex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570655" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570655" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, J.; Wang, Y.; Li, C.; Gu, Y.; Ono, N.; Welch, J. D.</p>
        <p class="info">Score: 9.2, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570655' target='https://doi.org/10.1101/2023.12.07.570655'> 10.1101/2023.12.07.570655</a></p>
        <p class="abstract">Cells differentiate to their final fates along unique trajectories, often involving multi-potent progenitors that can produce multiple terminally differentiated cell types. Recent developments in single-cell transcriptomic and epigenomic measurement provide tremendous opportunities for mapping these trajectories. The visualization of single-cell data often relies on dimension reduction methods such as UMAP to simplify high-dimensional single-cell data down into an understandable two-dimensional (2D) form. However, these visualization methods can be misleading and often do not effectively represent the direction of cell differentiation. To address these limitations, we developed a new approach that places each cell from a single-cell dataset within a simplex whose vertices correspond to terminally differentiated cell types. Our approach can quantify and visualize current cell fate commitment and future cell potential. We developed CytoSimplex, a standalone open-source package implemented in R and Python that provides simple and intuitive visualizations of cell differentiation in 2D ternary and three-dimensional (3D) quaternary plots. We believe that CytoSimplex can help researchers gain a better understanding of cell type transitions in specific tissues and characterize developmental processes.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
