<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="NanoPyx: super-fast bioimage analysis powered by adaptive machine learning
Authors: Saraiva, B. M.; Cunha, I. M.; Brito, A. D.; Follain, G.; Portela, R.; Haase, R.; Pereira, P. M.; Jacquemet, G.; Henriques, R.
Score: 56.6, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553080
To overcome the challenges posed by large and complex microscopy datasets, we have developed NanoPyx, an adaptive bioimage analysis framework designed for high-speed processing. At the core of NanoPyx is the Liquid Engine, an agent-based machine-learning system that predicts acceleration strategies for image analysis tasks.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="NanoPyx: super-fast bioimage analysis powered by adaptive machine learning
Authors: Saraiva, B. M.; Cunha, I. M.; Brito, A. D.; Follain, G.; Portela, R.; Haase, R.; Pereira, P. M.; Jacquemet, G.; Henriques, R.
Score: 56.6, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553080
To overcome the challenges posed by large and complex microscopy datasets, we have developed NanoPyx, an adaptive bioimage analysis framework designed for high-speed processing. At the core of NanoPyx is the Liquid Engine, an agent-based machine-learning system that predicts acceleration strategies for image analysis tasks." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-20T10:35:54+00:00" />
<meta property="article:modified_time" content="2023-08-20T10:35:54+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="NanoPyx: super-fast bioimage analysis powered by adaptive machine learning
Authors: Saraiva, B. M.; Cunha, I. M.; Brito, A. D.; Follain, G.; Portela, R.; Haase, R.; Pereira, P. M.; Jacquemet, G.; Henriques, R.
Score: 56.6, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553080
To overcome the challenges posed by large and complex microscopy datasets, we have developed NanoPyx, an adaptive bioimage analysis framework designed for high-speed processing. At the core of NanoPyx is the Liquid Engine, an agent-based machine-learning system that predicts acceleration strategies for image analysis tasks."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "NanoPyx: super-fast bioimage analysis powered by adaptive machine learning\nAuthors: Saraiva, B. M.; Cunha, I. M.; Brito, A. D.; Follain, G.; Portela, R.; Haase, R.; Pereira, P. M.; Jacquemet, G.; Henriques, R.\nScore: 56.6, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553080\nTo overcome the challenges posed by large and complex microscopy datasets, we have developed NanoPyx, an adaptive bioimage analysis framework designed for high-speed processing. At the core of NanoPyx is the Liquid Engine, an agent-based machine-learning system that predicts acceleration strategies for image analysis tasks.",
  "keywords": [
    
  ],
  "articleBody": " NanoPyx: super-fast bioimage analysis powered by adaptive machine learning\nAuthors: Saraiva, B. M.; Cunha, I. M.; Brito, A. D.; Follain, G.; Portela, R.; Haase, R.; Pereira, P. M.; Jacquemet, G.; Henriques, R.\nScore: 56.6, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553080\nTo overcome the challenges posed by large and complex microscopy datasets, we have developed NanoPyx, an adaptive bioimage analysis framework designed for high-speed processing. At the core of NanoPyx is the Liquid Engine, an agent-based machine-learning system that predicts acceleration strategies for image analysis tasks. Unlike traditional single-algorithm methods, the Liquid Engine generates multiple CPU and GPU code variations using a meta-programming system, creating a competitive environment where different algorithms are benchmarked against each other to achieve optimal performance under the user\"s computational environment. In initial experiments focusing on super-resolution analysis methods, the Liquid Engine demonstrated an over 10-fold computational speed improvement by accurately predicting the ideal scenarios to switch between algorithmic implementations. NanoPyx is accessible to users through a Python library, code-free Jupyter notebooks, and a napari plugin, making it suitable for individuals regardless of their coding proficiency. Furthermore, the optimisation principles embodied by the Liquid Engine have broader implications, extending their applicability to various high-performance computing fields.\nMetaCerberus: distributed highly parallelized scalable HMM-based implementation for robust functional annotation across the tree of life\nAuthors: Figueroa, J. L.; Dhungel, E.; Brouwer, C. R.; White, R. A.\nScore: 47.4, Published: 2023-08-12 DOI: 10.1101/2023.08.10.552700\nSummaryMetaCerberus is an exclusive HMM/HMMER-based tool that is massively parallel, on low memory, and provides rapid scalable annotation for functional gene inference across genomes to metacommunities. It provides robust enumeration of functional genes and pathways across many current public databases including KEGG (KO), COGs, CAZy, FOAM, and viral specific databases (i.e., VOGs and PHROGs). In a direct comparison, MetaCerberus was twice as fast as EggNOG-Mapper, and produced better annotation of viruses, phages, and archaeal viruses than DRAM, PROKKA, or InterProScan. MetaCerberus annotates more KOs across domains when compared to DRAM, with a 186x smaller database and a third less memory. MetaCerberus is fully integrated with differential statistical tools (i.e., DESeq2 and edgeR), pathway enrichment (GAGE R), and Pathview R for quantitative elucidation of metabolic pathways. MetaCerberus implements the key to unlocking the biosphere across the tree of life at scale. Availability and implementationMetaCerberus is written in Python and distributed under a BSD-3 license. The source code of MetaCerberus is freely available at https://github.com/raw-lab/metacerberus. Written in python 3 for both Linux and Mac OS X. MetaCerberus can also be easily installed using mamba create -n metacerberus -c bioconda -c conda-forge metacerberus\nAccurate human genome analysis with Element Avidity sequencing\nAuthors: Carroll, A.; Kolesnikov, A.; Cook, D. E.; Brambrink, L.; Wiseman, K. N.; Billings, S. M.; Kruglyak, S.; Lajoie, B. R.; Zhao, J.; Levy, S. E.; McLean, C. Y.; Shafin, K.; Nattestad, M.; Chang, P.-C.\nScore: 31.0, Published: 2023-08-14 DOI: 10.1101/2023.08.11.553043\nWe investigate the new sequencing technology Avidity from Element Biosciences. We show that Element whole genome sequencing achieves higher mapping and variant calling accuracy compared to Illumina sequencing at the same coverage, with larger differences at lower coverages (20x-30x). We quantify base error rates of Element reads, finding lower error rates, especially in homopolymer and tandem repeat regions. We use Elements ability to generate paired end sequencing with longer insert sizes than typical short-read sequencing. We show that longer insert sizes result in even higher accuracy, with long insert Element sequencing giving noticeably more accurate genome analyses at all coverages.\nDiPPI: A curated dataset for drug-like molecules in protein-protein interfaces\nAuthors: Cankara, F.; Senyuz, S.; Sayin, A. Z.; Gursoy, A.; Keskin, O.\nScore: 29.9, Published: 2023-08-14 DOI: 10.1101/2023.08.09.552637\nProteins interact through their interfaces, and dysfunction of protein-protein interactions (PPIs) has been associated with various diseases. Therefore, investigating the properties of the drug-modulated PPIs and interface-targeting drugs is critical. Here, we present a curated large dataset for drug-like molecules in protein interfaces. We further present DiPPI (Drugs in Protein-Protein Interfaces), a two-module website to facilitate the search for such molecules and their properties by exploiting our dataset in drug repurposing studies. In the interface module of the website, we extracted several properties of interfaces, such as amino acid properties, hotspots, evolutionary conservation of drug-binding amino acids, and post-translational modifications of these residues. On the drug-like molecule side, we curated a list of drug-like small molecules and FDA-approved drugs from various databases and extracted those that bind to the interfaces. We further clustered the drugs based on their molecular fingerprints to confine the search for an alternative drug to a smaller space. Drug properties, including Lipinskis rules and various molecular descriptors, are also calculated and made available on the website to guide the selection of drug molecules. Our dataset contains 534,203 interfaces for 98,632 proteins, of which 55,135 are detected to bind to a drug-like molecule. 2,214 drug-like molecules are deposited on our website, among which 335 are FDA-approved. DiPPI provides users with an easy-to-follow scheme for drug repurposing studies through its well-curated and clustered interface and drug data; and is freely available at http://interactome.ku.edu.tr:8501.\nDNA language models are powerful predictors of genome-wide variant effects\nAuthors: Benegas, G.; Batra, S. S.; Song, Y. S.\nScore: 170.0, Published: 2023-08-04 DOI: 10.1101/2022.08.22.504706\nThe expanding catalog of genome-wide association studies (GWAS) provides biological insights across a variety of species, but identifying the causal variants behind these associations remains a significant challenge. Experimental validation is both labor-intensive and costly, highlighting the need for accurate, scalable computational methods to predict the effects of genetic variants across the entire genome. Inspired by recent progress in natural language processing, unsupervised pre-training on large protein sequence databases has proven successful in extracting complex information related to proteins. These models showcase their ability to learn variant effects in coding regions using an unsupervised approach. Expanding on this idea, we here introduce the Genomic Pre-trained Network (GPN), a model designed to learn genome-wide variant effects through unsupervised pre-training on genomic DNA sequences. Our model also successfully learns gene structure and DNA motifs without any supervision. To demonstrate its utility, we train GPN on unaligned reference genomes of Arabidopsis thaliana and seven related species within the Brassicales order, and evaluate its ability to predict the functional impact of genetic variants in Arabidopsis thaliana by utilizing allele frequencies from the 1001 Genomes Project and a comprehensive database of GWAS. Notably, GPN outperforms predictors based on popular conservation scores such as phyloP and phastCons. Our predictions for Arabidopsis thaliana can be visualized as sequence logos in the UCSC Genome Browser (https://genome.ucsc.edu/s/gbenegas/gpn-arabidopsis). We provide code (https://github.com/songlab-cal/gpn) to train GPN for any given species using its DNA sequence alone, enabling unsupervised prediction of variant effects across the entire genome.\nProtein Structure Refinement via DeepTracer and AlphaFold2\nAuthors: Chen, J.; Zia, A.; Hou, J.; Wang, F.; Cao, R.; Si, D.\nScore: 25.3, Published: 2023-08-17 DOI: 10.1101/2023.08.16.553616\nUnderstanding the structures of proteins has numerous applications, such as vaccine development. It is a slow and labor-intensive task to manually build protein structures from experimental electron density maps, therefore, machine learning approaches have been proposed to automate this process. However, most of the experimental maps are not atomic resolution, so they are insufficient for computer vision-based machine learning methods to precisely determine the protein structure. On the other hand, methods that utilize evolutionary information from protein sequences to predict structures, like AlphaFold2, have recently achieved groundbreaking accuracy but often require manual effort to refine the results. We propose DeepTracer-Refine, an automated method to refine AlphaFold structures by aligning them to DeepTracer's predicted structure. We tested our method on 39 multi-domain proteins and we improved the average residue coverage from 78.2% to 90.0% and average lDDT score from 0.67 to 0.71. We also compared DeepTracer-Refine against another method, Phenix's AlphaFold refinement, to demonstrate that our method not only performs better when the initial AlphaFold model is less precise but also exceeds Phenix in run-time performance.\nSearch and Match across Spatial Omics Samples at Single-cell Resolution\nAuthors: Tang, Z.; Luo, S.; Zeng, H.; Huang, J.; Wu, M.; Wang, X.\nScore: 23.4, Published: 2023-08-15 DOI: 10.1101/2023.08.13.552987\nSpatial omics technologies characterize tissue molecular properties with spatial information, but integrating and comparing spatial data across different technologies and modalities is challenging. A comparative analysis tool that can search, match, and visualize both similarities and differences of molecular features in space across multiple samples is lacking. To address this, we introduce CAST (Cross-sample Alignment of SpaTial omics), a deep graph neural network (GNN)-based method enabling spatial-to-spatial searching and matching at the single-cell level. CAST aligns tissues based on intrinsic similarities of spatial molecular features and reconstructs spatially resolved single-cell multi-omic profiles. CAST enables spatially resolved differential analysis ({Delta}Analysis) to pinpoint and visualize disease-associated molecular pathways and cell-cell interactions, and single-cell relative translational efficiency (scRTE) profiling to reveal variations in translational control across cell types and regions. CAST serves as an integrative framework for seamless single-cell spatial data searching and matching across technologies, modalities, and disease conditions, analogous to BLAST in sequence alignment.\nEagleC Explorer: A desktop application for interactively detecting and visualizing SVs and enhancer hijacking on Hi-C contact maps\nAuthors: Fu, Y.; Wang, X.; Yue, F.\nScore: 23.2, Published: 2023-08-08 DOI: 10.1101/2023.08.07.552228\nIt has been shown that Hi-C can be used as a powerful tool to detect structural variations (SVs) and enhancer hijacking events. However, there has been no existing programs that can directly visualize and detect such events on a personal computer, which hinders the broad adaption of the technology for intuitive discovery in cancer studies. Here, we introduce the EagleC Explorer, a desktop software that is specifically designed for exploring Hi-C and other chromatin contact data in cancer genomes. EagleC Explorer has a set of unique features, including 1) conveniently visualizing global and local Hi-C data; 2) interactively detecting SVs on a Hi-C map for any user-selected region on screen within seconds, using a deep-learning model; 3) reconstructing local Hi-C map surrounding user-provided SVs and generating publication-quality figures; 4) detecting enhancer hijacking events for any user-suggested regions on screen. In addition, EagleC Explorer can also incorporate other genomic tracks such as RNA-Seq or ChIP-Seq to facilitate scientists for integrative data analysis and making novel discoveries.\nDeGenPrime provides robust primer design and optimization1 unlocking the biosphere\nAuthors: Fulghum, B. W.; Tanker, S.; White, R. A.\nScore: 17.3, Published: 2023-08-14 DOI: 10.1101/2023.08.11.553048\nPolymerase chain reaction (PCR) is the worlds most important molecular diagnostic with applications ranging from medicine to ecology. PCR can fail because of poor primer design. The nearest-neighbor thermodynamic properties, picking conserved regions, and filtration via penalty of oligonucleotides form the basis for good primer design. DeGenPrime is a console-based high quality PCR primer design tool that can utilize MSA formats and degenerate bases expanding target range for a single primer set. Our software utilizes thermodynamic properties, filtration metrics, penalty scoring, and conserved region finding of any proposed primer. It has degeneracy, repeated k-mers, relative GC content, and temperature range filters. Minimal penalty scoring is included according to secondary structure self-dimerization metrics, GC clamping, tri- and tetra-loop hairpins and internal repetition. We compared PrimerDesign-M, DegePrime, ConsensusPrimer, and DeGenPrime on acceptable primer yield. PrimerDesign-M, DegePrime, and ConsensusPrimer provided 0%, 11%, and 17% yield respectively for alternative iron nitrogenase (anfD) gene target. DeGenPrime successfully identified quality primers within the conserved regions of the T4-like phage major capsid protein (g23), conserved regions of molybdenum-based nitrogenase (nif), and its alternatives vanadium (vnf) and iron (anf) nitrogenase. DeGenPrime provides a universal and scalable primer design tool for the entire tree of life.\nOpenFold: Retraining AlphaFold2 yields new insights into its learning mechanisms and capacity for generalization\nAuthors: Ahdritz, G.; Bouatta, N.; Floristean, C.; Kadyan, S.; Xia, Q.; Gerecke, W.; O'Donnell, T. J.; Berenberg, D.; Fisk, I.; Zanichelli, N.; Zhang, B.; Nowaczynski, A.; Wang, B.; Stepniewska-Dziubinska, M. M.; Zhang, S.; Ojewole, A.; Guney, M. E.; Biderman, S.; Watkins, A. M.; Ra, S.; Lorenzo, P. R.; Nivon, L.; Weitzner, B.; Ban, Y.-E. A.; Sorger, P. K.; Mostaque, E.; Zhang, Z.; Bonneau, R.; AlQuraishi, M.\nScore: 581.9, Published: 2023-08-12 DOI: 10.1101/2022.11.20.517210\nAlphaFold2 revolutionized structural biology with the ability to predict protein structures with exceptionally high accuracy. Its implementation, however, lacks the code and data required to train new models. These are necessary to (i) tackle new tasks, like protein-ligand complex structure prediction, (ii) investigate the process by which the model learns, which remains poorly understood, and (iii) assess the models generalization capacity to unseen regions of fold space. Here we report OpenFold, a fast, memory-efficient, and trainable implementation of AlphaFold2. We train OpenFold from scratch, fully matching the accuracy of AlphaFold2. Having established parity, we assess OpenFolds capacity to generalize across fold space by retraining it using carefully designed datasets. We find that OpenFold is remarkably robust at generalizing despite extreme reductions in training set size and diversity, including near-complete elisions of classes of secondary structure elements. By analyzing intermediate structures produced by OpenFold during training, we also gain surprising insights into the manner in which the model learns to fold proteins, discovering that spatial dimensions are learned sequentially. Taken together, our studies demonstrate the power and utility of OpenFold, which we believe will prove to be a crucial new resource for the protein modeling community.\n",
  "wordCount" : "2184",
  "inLanguage": "en",
  "datePublished": "2023-08-20T10:35:54Z",
  "dateModified": "2023-08-20T10:35:54Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on August 20, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.13.553080">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.13.553080" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.13.553080">
        <p class="paperTitle">NanoPyx: super-fast bioimage analysis powered by adaptive machine learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.13.553080" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.13.553080" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Saraiva, B. M.; Cunha, I. M.; Brito, A. D.; Follain, G.; Portela, R.; Haase, R.; Pereira, P. M.; Jacquemet, G.; Henriques, R.</p>
        <p class="info">Score: 56.6, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.13.553080' target='https://doi.org/10.1101/2023.08.13.553080'> 10.1101/2023.08.13.553080</a></p>
        <p class="abstract">To overcome the challenges posed by large and complex microscopy datasets, we have developed NanoPyx, an adaptive bioimage analysis framework designed for high-speed processing. At the core of NanoPyx is the Liquid Engine, an agent-based machine-learning system that predicts acceleration strategies for image analysis tasks. Unlike traditional single-algorithm methods, the Liquid Engine generates multiple CPU and GPU code variations using a meta-programming system, creating a competitive environment where different algorithms are benchmarked against each other to achieve optimal performance under the user&#34;s computational environment. In initial experiments focusing on super-resolution analysis methods, the Liquid Engine demonstrated an over 10-fold computational speed improvement by accurately predicting the ideal scenarios to switch between algorithmic implementations. NanoPyx is accessible to users through a Python library, code-free Jupyter notebooks, and a napari plugin, making it suitable for individuals regardless of their coding proficiency. Furthermore, the optimisation principles embodied by the Liquid Engine have broader implications, extending their applicability to various high-performance computing fields.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552700">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552700" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552700">
        <p class="paperTitle">MetaCerberus: distributed highly parallelized scalable HMM-based implementation for robust functional annotation across the tree of life</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552700" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552700" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Figueroa, J. L.; Dhungel, E.; Brouwer, C. R.; White, R. A.</p>
        <p class="info">Score: 47.4, Published: 2023-08-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552700' target='https://doi.org/10.1101/2023.08.10.552700'> 10.1101/2023.08.10.552700</a></p>
        <p class="abstract">SummaryMetaCerberus is an exclusive HMM/HMMER-based tool that is massively parallel, on low memory, and provides rapid scalable annotation for functional gene inference across genomes to metacommunities. It provides robust enumeration of functional genes and pathways across many current public databases including KEGG (KO), COGs, CAZy, FOAM, and viral specific databases (i.e., VOGs and PHROGs). In a direct comparison, MetaCerberus was twice as fast as EggNOG-Mapper, and produced better annotation of viruses, phages, and archaeal viruses than DRAM, PROKKA, or InterProScan. MetaCerberus annotates more KOs across domains when compared to DRAM, with a 186x smaller database and a third less memory. MetaCerberus is fully integrated with differential statistical tools (i.e., DESeq2 and edgeR), pathway enrichment (GAGE R), and Pathview R for quantitative elucidation of metabolic pathways. MetaCerberus implements the key to unlocking the biosphere across the tree of life at scale.

Availability and implementationMetaCerberus is written in Python and distributed under a BSD-3 license. The source code of MetaCerberus is freely available at https://github.com/raw-lab/metacerberus. Written in python 3 for both Linux and Mac OS X. MetaCerberus can also be easily installed using mamba create -n metacerberus -c bioconda -c conda-forge metacerberus</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.553043">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.553043" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.553043">
        <p class="paperTitle">Accurate human genome analysis with Element Avidity sequencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.553043" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.553043" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carroll, A.; Kolesnikov, A.; Cook, D. E.; Brambrink, L.; Wiseman, K. N.; Billings, S. M.; Kruglyak, S.; Lajoie, B. R.; Zhao, J.; Levy, S. E.; McLean, C. Y.; Shafin, K.; Nattestad, M.; Chang, P.-C.</p>
        <p class="info">Score: 31.0, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.553043' target='https://doi.org/10.1101/2023.08.11.553043'> 10.1101/2023.08.11.553043</a></p>
        <p class="abstract">We investigate the new sequencing technology Avidity from Element Biosciences. We show that Element whole genome sequencing achieves higher mapping and variant calling accuracy compared to Illumina sequencing at the same coverage, with larger differences at lower coverages (20x-30x). We quantify base error rates of Element reads, finding lower error rates, especially in homopolymer and tandem repeat regions. We use Elements ability to generate paired end sequencing with longer insert sizes than typical short-read sequencing. We show that longer insert sizes result in even higher accuracy, with long insert Element sequencing giving noticeably more accurate genome analyses at all coverages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.552637">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.552637" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.552637">
        <p class="paperTitle">DiPPI: A curated dataset for drug-like molecules in protein-protein interfaces</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.552637" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.552637" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cankara, F.; Senyuz, S.; Sayin, A. Z.; Gursoy, A.; Keskin, O.</p>
        <p class="info">Score: 29.9, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.552637' target='https://doi.org/10.1101/2023.08.09.552637'> 10.1101/2023.08.09.552637</a></p>
        <p class="abstract">Proteins interact through their interfaces, and dysfunction of protein-protein interactions (PPIs) has been associated with various diseases. Therefore, investigating the properties of the drug-modulated PPIs and interface-targeting drugs is critical. Here, we present a curated large dataset for drug-like molecules in protein interfaces. We further present DiPPI (Drugs in Protein-Protein Interfaces), a two-module website to facilitate the search for such molecules and their properties by exploiting our dataset in drug repurposing studies. In the interface module of the website, we extracted several properties of interfaces, such as amino acid properties, hotspots, evolutionary conservation of drug-binding amino acids, and post-translational modifications of these residues. On the drug-like molecule side, we curated a list of drug-like small molecules and FDA-approved drugs from various databases and extracted those that bind to the interfaces. We further clustered the drugs based on their molecular fingerprints to confine the search for an alternative drug to a smaller space. Drug properties, including Lipinskis rules and various molecular descriptors, are also calculated and made available on the website to guide the selection of drug molecules. Our dataset contains 534,203 interfaces for 98,632 proteins, of which 55,135 are detected to bind to a drug-like molecule. 2,214 drug-like molecules are deposited on our website, among which 335 are FDA-approved. DiPPI provides users with an easy-to-follow scheme for drug repurposing studies through its well-curated and clustered interface and drug data; and is freely available at http://interactome.ku.edu.tr:8501.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.08.22.504706">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.08.22.504706" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.08.22.504706">
        <p class="paperTitle">DNA language models are powerful predictors of genome-wide variant effects</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.08.22.504706" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.08.22.504706" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Benegas, G.; Batra, S. S.; Song, Y. S.</p>
        <p class="info">Score: 170.0, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.08.22.504706' target='https://doi.org/10.1101/2022.08.22.504706'> 10.1101/2022.08.22.504706</a></p>
        <p class="abstract">The expanding catalog of genome-wide association studies (GWAS) provides biological insights across a variety of species, but identifying the causal variants behind these associations remains a significant challenge. Experimental validation is both labor-intensive and costly, highlighting the need for accurate, scalable computational methods to predict the effects of genetic variants across the entire genome. Inspired by recent progress in natural language processing, unsupervised pre-training on large protein sequence databases has proven successful in extracting complex information related to proteins. These models showcase their ability to learn variant effects in coding regions using an unsupervised approach. Expanding on this idea, we here introduce the Genomic Pre-trained Network (GPN), a model designed to learn genome-wide variant effects through unsupervised pre-training on genomic DNA sequences. Our model also successfully learns gene structure and DNA motifs without any supervision. To demonstrate its utility, we train GPN on unaligned reference genomes of Arabidopsis thaliana and seven related species within the Brassicales order, and evaluate its ability to predict the functional impact of genetic variants in Arabidopsis thaliana by utilizing allele frequencies from the 1001 Genomes Project and a comprehensive database of GWAS. Notably, GPN outperforms predictors based on popular conservation scores such as phyloP and phastCons. Our predictions for Arabidopsis thaliana can be visualized as sequence logos in the UCSC Genome Browser (https://genome.ucsc.edu/s/gbenegas/gpn-arabidopsis). We provide code (https://github.com/songlab-cal/gpn) to train GPN for any given species using its DNA sequence alone, enabling unsupervised prediction of variant effects across the entire genome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.553616">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.553616" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.553616">
        <p class="paperTitle">Protein Structure Refinement via DeepTracer and AlphaFold2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.553616" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.553616" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, J.; Zia, A.; Hou, J.; Wang, F.; Cao, R.; Si, D.</p>
        <p class="info">Score: 25.3, Published: 2023-08-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.553616' target='https://doi.org/10.1101/2023.08.16.553616'> 10.1101/2023.08.16.553616</a></p>
        <p class="abstract">Understanding the structures of proteins has numerous applications, such as vaccine development. It is a slow and labor-intensive task to manually build protein structures from experimental electron density maps, therefore, machine learning approaches have been proposed to automate this process. However, most of the experimental maps are not atomic resolution, so they are insufficient for computer vision-based machine learning methods to precisely determine the protein structure. On the other hand, methods that utilize evolutionary information from protein sequences to predict structures, like AlphaFold2, have recently achieved groundbreaking accuracy but often require manual effort to refine the results. We propose DeepTracer-Refine, an automated method to refine AlphaFold structures by aligning them to DeepTracer&#39;s predicted structure. We tested our method on 39 multi-domain proteins and we improved the average residue coverage from 78.2% to 90.0% and average lDDT score from 0.67 to 0.71. We also compared DeepTracer-Refine against another method, Phenix&#39;s AlphaFold refinement, to demonstrate that our method not only performs better when the initial AlphaFold model is less precise but also exceeds Phenix in run-time performance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.13.552987">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.13.552987" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.13.552987">
        <p class="paperTitle">Search and Match across Spatial Omics Samples at Single-cell Resolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.13.552987" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.13.552987" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tang, Z.; Luo, S.; Zeng, H.; Huang, J.; Wu, M.; Wang, X.</p>
        <p class="info">Score: 23.4, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.13.552987' target='https://doi.org/10.1101/2023.08.13.552987'> 10.1101/2023.08.13.552987</a></p>
        <p class="abstract">Spatial omics technologies characterize tissue molecular properties with spatial information, but integrating and comparing spatial data across different technologies and modalities is challenging. A comparative analysis tool that can search, match, and visualize both similarities and differences of molecular features in space across multiple samples is lacking. To address this, we introduce CAST (Cross-sample Alignment of SpaTial omics), a deep graph neural network (GNN)-based method enabling spatial-to-spatial searching and matching at the single-cell level. CAST aligns tissues based on intrinsic similarities of spatial molecular features and reconstructs spatially resolved single-cell multi-omic profiles. CAST enables spatially resolved differential analysis ({Delta}Analysis) to pinpoint and visualize disease-associated molecular pathways and cell-cell interactions, and single-cell relative translational efficiency (scRTE) profiling to reveal variations in translational control across cell types and regions. CAST serves as an integrative framework for seamless single-cell spatial data searching and matching across technologies, modalities, and disease conditions, analogous to BLAST in sequence alignment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552228">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552228" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552228">
        <p class="paperTitle">EagleC Explorer: A desktop application for interactively detecting and visualizing SVs and enhancer hijacking on Hi-C contact maps</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552228" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552228" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fu, Y.; Wang, X.; Yue, F.</p>
        <p class="info">Score: 23.2, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552228' target='https://doi.org/10.1101/2023.08.07.552228'> 10.1101/2023.08.07.552228</a></p>
        <p class="abstract">It has been shown that Hi-C can be used as a powerful tool to detect structural variations (SVs) and enhancer hijacking events. However, there has been no existing programs that can directly visualize and detect such events on a personal computer, which hinders the broad adaption of the technology for intuitive discovery in cancer studies. Here, we introduce the EagleC Explorer, a desktop software that is specifically designed for exploring Hi-C and other chromatin contact data in cancer genomes. EagleC Explorer has a set of unique features, including 1) conveniently visualizing global and local Hi-C data; 2) interactively detecting SVs on a Hi-C map for any user-selected region on screen within seconds, using a deep-learning model; 3) reconstructing local Hi-C map surrounding user-provided SVs and generating publication-quality figures; 4) detecting enhancer hijacking events for any user-suggested regions on screen. In addition, EagleC Explorer can also incorporate other genomic tracks such as RNA-Seq or ChIP-Seq to facilitate scientists for integrative data analysis and making novel discoveries.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.553048">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.553048" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.553048">
        <p class="paperTitle">DeGenPrime provides robust primer design and optimization1 unlocking the biosphere</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.553048" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.553048" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fulghum, B. W.; Tanker, S.; White, R. A.</p>
        <p class="info">Score: 17.3, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.553048' target='https://doi.org/10.1101/2023.08.11.553048'> 10.1101/2023.08.11.553048</a></p>
        <p class="abstract">Polymerase chain reaction (PCR) is the worlds most important molecular diagnostic with applications ranging from medicine to ecology. PCR can fail because of poor primer design. The nearest-neighbor thermodynamic properties, picking conserved regions, and filtration via penalty of oligonucleotides form the basis for good primer design.

DeGenPrime is a console-based high quality PCR primer design tool that can utilize MSA formats and degenerate bases expanding target range for a single primer set. Our software utilizes thermodynamic properties, filtration metrics, penalty scoring, and conserved region finding of any proposed primer. It has degeneracy, repeated k-mers, relative GC content, and temperature range filters. Minimal penalty scoring is included according to secondary structure self-dimerization metrics, GC clamping, tri- and tetra-loop hairpins and internal repetition.

We compared PrimerDesign-M, DegePrime, ConsensusPrimer, and DeGenPrime on acceptable primer yield. PrimerDesign-M, DegePrime, and ConsensusPrimer provided 0%, 11%, and 17% yield respectively for alternative iron nitrogenase (anfD) gene target. DeGenPrime successfully identified quality primers within the conserved regions of the T4-like phage major capsid protein (g23), conserved regions of molybdenum-based nitrogenase (nif), and its alternatives vanadium (vnf) and iron (anf) nitrogenase. DeGenPrime provides a universal and scalable primer design tool for the entire tree of life.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.11.20.517210">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.11.20.517210" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.11.20.517210">
        <p class="paperTitle">OpenFold: Retraining AlphaFold2 yields new insights into its learning mechanisms and capacity for generalization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.11.20.517210" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.11.20.517210" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ahdritz, G.; Bouatta, N.; Floristean, C.; Kadyan, S.; Xia, Q.; Gerecke, W.; O&#39;Donnell, T. J.; Berenberg, D.; Fisk, I.; Zanichelli, N.; Zhang, B.; Nowaczynski, A.; Wang, B.; Stepniewska-Dziubinska, M. M.; Zhang, S.; Ojewole, A.; Guney, M. E.; Biderman, S.; Watkins, A. M.; Ra, S.; Lorenzo, P. R.; Nivon, L.; Weitzner, B.; Ban, Y.-E. A.; Sorger, P. K.; Mostaque, E.; Zhang, Z.; Bonneau, R.; AlQuraishi, M.</p>
        <p class="info">Score: 581.9, Published: 2023-08-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.11.20.517210' target='https://doi.org/10.1101/2022.11.20.517210'> 10.1101/2022.11.20.517210</a></p>
        <p class="abstract">AlphaFold2 revolutionized structural biology with the ability to predict protein structures with exceptionally high accuracy. Its implementation, however, lacks the code and data required to train new models. These are necessary to (i) tackle new tasks, like protein-ligand complex structure prediction, (ii) investigate the process by which the model learns, which remains poorly understood, and (iii) assess the models generalization capacity to unseen regions of fold space. Here we report OpenFold, a fast, memory-efficient, and trainable implementation of AlphaFold2. We train OpenFold from scratch, fully matching the accuracy of AlphaFold2. Having established parity, we assess OpenFolds capacity to generalize across fold space by retraining it using carefully designed datasets. We find that OpenFold is remarkably robust at generalizing despite extreme reductions in training set size and diversity, including near-complete elisions of classes of secondary structure elements. By analyzing intermediate structures produced by OpenFold during training, we also gain surprising insights into the manner in which the model learns to fold proteins, discovering that spatial dimensions are learned sequentially. Taken together, our studies demonstrate the power and utility of OpenFold, which we believe will prove to be a crucial new resource for the protein modeling community.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
