<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://yorks0n.github.io/TRxiv2" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/TRxiv2/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="QuantUMS: uncertainty minimisation enables confident quantification in proteomics
Authors: Kistner, F.; Grossmann, J. L.; Sinn, L. R.; Demichev, V.
Score: 49.3, Published: 2023-06-24 DOI: 10.1101/2023.06.20.545604
Mass spectrometry-based proteomics has been rapidly gaining traction as a powerful analytical method both in basic research and translation. While the problem of error control in peptide and protein identification has been addressed extensively, the quality of the resulting quantities remains challenging to evaluate. Here we introduce QuantUMS (Quantification using an Uncertainty Minimising Solution), a machine learning-based method which minimises errors and eliminates bias in peptide and protein quantification by integrating multiple sources of quantitative information.">
<meta name="author" content="">
<link rel="canonical" href="https://yorks0n.github.io/TRxiv2/posts/bioinformatics/">
<link crossorigin="anonymous" href="/TRxiv2/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/TRxiv2/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://yorks0n.github.io/TRxiv2/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://yorks0n.github.io/TRxiv2/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://yorks0n.github.io/TRxiv2/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://yorks0n.github.io/TRxiv2/apple-touch-icon.png">
<link rel="mask-icon" href="https://yorks0n.github.io/TRxiv2/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="QuantUMS: uncertainty minimisation enables confident quantification in proteomics
Authors: Kistner, F.; Grossmann, J. L.; Sinn, L. R.; Demichev, V.
Score: 49.3, Published: 2023-06-24 DOI: 10.1101/2023.06.20.545604
Mass spectrometry-based proteomics has been rapidly gaining traction as a powerful analytical method both in basic research and translation. While the problem of error control in peptide and protein identification has been addressed extensively, the quality of the resulting quantities remains challenging to evaluate. Here we introduce QuantUMS (Quantification using an Uncertainty Minimising Solution), a machine learning-based method which minimises errors and eliminates bias in peptide and protein quantification by integrating multiple sources of quantitative information." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://yorks0n.github.io/TRxiv2/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-06-28T09:51:07+00:00" />
<meta property="article:modified_time" content="2023-06-28T09:51:07+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="QuantUMS: uncertainty minimisation enables confident quantification in proteomics
Authors: Kistner, F.; Grossmann, J. L.; Sinn, L. R.; Demichev, V.
Score: 49.3, Published: 2023-06-24 DOI: 10.1101/2023.06.20.545604
Mass spectrometry-based proteomics has been rapidly gaining traction as a powerful analytical method both in basic research and translation. While the problem of error control in peptide and protein identification has been addressed extensively, the quality of the resulting quantities remains challenging to evaluate. Here we introduce QuantUMS (Quantification using an Uncertainty Minimising Solution), a machine learning-based method which minimises errors and eliminates bias in peptide and protein quantification by integrating multiple sources of quantitative information."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://yorks0n.github.io/TRxiv2/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://yorks0n.github.io/TRxiv2/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "QuantUMS: uncertainty minimisation enables confident quantification in proteomics\nAuthors: Kistner, F.; Grossmann, J. L.; Sinn, L. R.; Demichev, V.\nScore: 49.3, Published: 2023-06-24 DOI: 10.1101/2023.06.20.545604\nMass spectrometry-based proteomics has been rapidly gaining traction as a powerful analytical method both in basic research and translation. While the problem of error control in peptide and protein identification has been addressed extensively, the quality of the resulting quantities remains challenging to evaluate. Here we introduce QuantUMS (Quantification using an Uncertainty Minimising Solution), a machine learning-based method which minimises errors and eliminates bias in peptide and protein quantification by integrating multiple sources of quantitative information.",
  "keywords": [
    
  ],
  "articleBody": " QuantUMS: uncertainty minimisation enables confident quantification in proteomics\nAuthors: Kistner, F.; Grossmann, J. L.; Sinn, L. R.; Demichev, V.\nScore: 49.3, Published: 2023-06-24 DOI: 10.1101/2023.06.20.545604\nMass spectrometry-based proteomics has been rapidly gaining traction as a powerful analytical method both in basic research and translation. While the problem of error control in peptide and protein identification has been addressed extensively, the quality of the resulting quantities remains challenging to evaluate. Here we introduce QuantUMS (Quantification using an Uncertainty Minimising Solution), a machine learning-based method which minimises errors and eliminates bias in peptide and protein quantification by integrating multiple sources of quantitative information. In combination with data-independent acquisition proteomics, QuantUMS boosts accuracy and precision of quantities, as well as reports an uncertainty metric, enabling effective filtering of data for downstream analysis. The algorithm has linear complexity with respect to the number of mass spectrometry acquisitions in the experiment and is thus scalable to infinitely large proteomic experiments. For an easy implementation in a proteomics laboratory, we integrate QuantUMS in our automated DIA-NN software suite.\nTooManyCellsInteractive: a visualization tool for dynamic exploration of single-cell data\nAuthors: Klamann, C.; Lau, C.; Schwartz, G. W.\nScore: 34.5, Published: 2023-06-18 DOI: 10.1101/2023.06.16.544954\nAs single-cell sequencing data sets grow in size, visualizations of large cellular populations become difficult to parse and require extensive processing to identify subpopulations of cells. Managing many of these charts is laborious for technical users and unintuitive for non-technical users. To address this issue, we developed TooManyCellsInteractive (TMCI), a browser-based JavaScript application for visualizing hierarchical cellular populations as an interactive radial tree. TMCI allows users to explore, filter, and manipulate hierarchical data structures through an intuitive interface while also enabling batch export of high-quality custom graphics. Here we describe the software architecture and illustrate how TMCI has identified unique survival pathways among drug-tolerant persister cells in a pan-cancer analysis. TMCI will help guide increasingly large data visualizations and facilitate multi-resolution data exploration in a user-friendly way.\nComprehensive annotation of 3'UTRs from primary cells and their quantification from scRNA-seq data\nAuthors: Fansler, M. M.; Mitschka, S.; Mayr, C.\nScore: 33.1, Published: 2023-06-22 DOI: 10.1101/2021.11.22.469635\nApproximately half of human genes generate mRNA isoforms that differ in their 3'UTRs while encoding the same protein. 3'UTR and mRNA length is determined by 3' end cleavage sites (CS). Here, we mapped and categorized mRNA 3' end CS in more than 200 primary human and mouse cell types, resulting in a 40% increase of CS annotations relative to the GENCODE database. We incorporated these annotations into a novel computational pipeline, called scUTRquant, for rapid, precise, and accurate quantification of gene and 3'UTR isoform expression from single-cell RNA sequencing (scRNA-seq) data. When applying scUTRquant to data from 474 cell types and 2,134 perturbations, we discovered extensive 3'UTR length changes across cell types that are as widespread and dynamically regulated as gene expression changes. Our data indicate that mRNA abundance and mRNA length are two independent axes of gene regulation that together determine the amount and spatial organization of protein synthesis.\nBrainRBPedia: a resource for RNA-binding proteins relevant to neurodevelopmental disorders\nAuthors: Han, K.; Wainberg, M.; Calarco, J. A.; Smibert, C. A.; Lipshitz, H. D.; Lee, H. O.; Tripathy, S. J.\nScore: 23.8, Published: 2023-06-09 DOI: 10.1101/2023.06.07.542483\nRNA-binding proteins (RBPs) are crucial players in the post-transcriptional regulation of mRNA and play major roles in ensuring proper neuronal development and function. Deficits in RBP function have been implicated in a number of neurodevelopmental disorders including autism spectrum disorder (ASD) and intellectual disability (ID), yet we lack resources that integrate current knowledge of RBP function, tissue expression, and disease association in one place to aid in their experimental characterization. Here we introduce BrainRBPedia - a database of 1072 RBPs with both disease annotations for neurodevelopmental disorders and functional annotations relevant to these disorders, including loss-of-function intolerance and expression specificity to the brain, neurons, and neuronal development. Using these functional annotations, we develop a machine learning model to prioritize RBPs likely to be involved in ASD and ID. Our model indicates that RBPs with high loss-of-function intolerance and those upregulated during neuronal differentiation are disproportionately likely to contribute to ASD and ID etiology. In summary, BrainRBPedia comprises a unique resource for researchers interested in the experimental characterization of RBPs in relation to neurodevelopmental disorders and suggests functional signatures of RBPs likely to play a role in neurodevelopment.\nDeepRank-GNN-esm: A Graph Neural Network for Scoring Protein-Protein Models using Protein Language Model\nAuthors: Xu, X.; Bonvin, A. M.\nScore: 18.3, Published: 2023-06-24 DOI: 10.1101/2023.06.22.546080\nProtein-Protein interactions (PPIs) play critical roles in numerous cellular processes. By modelling the three-dimensional structures of the correspond protein complexes valuable insights can be obtained, providing, for example, starting points for drug and protein design. One challenge in the modelling process is however the identification of near-native models from the large pool of generated models. To this end we previously developed DeepRank-GNN, a graph neural network that integrates structural and sequence information to enable effective pattern learning at PPI interfaces. Its main features are related to the Position Specific Scoring Matrices (PSSM), which are computationally expensive to generate and significantly limit the algorithm's usability. We introduce here DeepRank-GNN-esm that includes as additional features protein language model embeddings from the EMS-2 model. We show that the ESM-2 embeddings can actually replace the PSSM features at no cost in-, or even better performance on two PPI-related tasks: scoring docking poses and detecting crystal artifacts. This new DeepRank version bypasses thus the need of generating PSSM, greatly improving the usability of the software and opening new application opportunities for systems for which PSSM profiles cannot be obtained or are irrelevant (e.g. antibody-antigen complexes). DeepRank-GNN-esm is freely available from https://github.com/DeepRank/DeepRank-GNN-esm .\nAccurate prediction of protein assembly structure by combining AlphaFold and symmetrical docking\nAuthors: Jeppesen, M.; Andre, I.\nScore: 16.4, Published: 2023-06-24 DOI: 10.1101/2023.06.22.546069\nAlphaFold can predict the structures of monomeric and multimeric proteins with high accuracy but has a limit on the number of chains and residues it can fold. Here we show that a combination of AlphaFold and all-atom symmetric docking simulations enables highly accurate prediction of the structure of complex symmetrical assemblies. We present a method to predict the structure of complexes with cubic - tetrahedral, octahedral and icosahedral - symmetry from sequence. Focusing on proteins where AlphaFold can make confident predictions on the subunit structure, 21 cubic systems were assembled with a median TM-score of 0.99 and a DockQ score of 0.71. 15 had TM-scores of above 0.8 and were categorized as high-quality according to DockQ. The resulting models are energetically optimized and can be used for detailed studies of intermolecular interactions in higher-order symmetrical assemblies. The results demonstrate how explicit treatment of structural symmetry can significantly expand the size and complexity of AlphaFold predictions.\nCo-linear Chaining on Pangenome Graphs\nAuthors: Rajput, J.; Chandra, G.; Jain, C.\nScore: 15.9, Published: 2023-06-24 DOI: 10.1101/2023.06.21.545871\nPangenome reference graphs are useful in genomics because they compactly represent the genetic diversity within a species, a capability that linear references lack. However, efficiently aligning sequences to these graphs with complex topology and cycles can be challenging. The seed-chain-extend based alignment algorithms use co-linear chaining as a standard technique to identify a good cluster of exact seed matches that can be combined to form an alignment. Recent works show how the co-linear chaining problem can be efficiently solved for acyclic pangenome graphs by exploiting their small width [Makinen et al., TALG19] and how incorporating gap cost in the scoring function improves alignment accuracy [Chandra and Jain, RECOMB23]. However, it remains open on how to effectively generalize these techniques for general pangenome graphs which contain cycles. Here we present the first practical formulation and an exact algorithm for co-linear chaining on cyclic pangenome graphs. We rigorously prove the correctness and computational complexity of the proposed algorithm. We evaluate the empirical performance of our algorithm by aligning simulated long reads from the human genome to a cyclic pangenome graph constructed from 95 publicly available haplotype-resolved human genome assemblies. While the existing heuristic-based algorithms are faster, the proposed algorithm provides a significant advantage in terms of accuracy. Implementationhttps://github.com/at-cg/PanAligner\nInGene: Finding influential genes from embeddings of nonlinear dimension reduction techniques\nAuthors: Goswami, C.; Sengupta, D.\nScore: 14.2, Published: 2023-06-21 DOI: 10.1101/2023.06.19.545592\nWe introduce InGene, the first of its kind, fast and scalable non-linear, unsupervised method for analyzing single-cell RNA sequencing data (scRNA-seq). While non-linear dimensionality reduction techniques such as tSNE and UMAP are effective at visualizing cellular sub-populations in low-dimensional space, they do not identify the specific genes that influence the transformation. InGene addresses this issue by assigning an importance score to each expressed gene based on its contribution to the construction of the low-dimensional map. InGene can provide insight into the cellular heterogeneity of scRNA-seq data and accurately identify genes associated with cell-type populations or diseases, as demonstrated in our analysis of scRNA-seq datasets.\nSpatial and Temporal Analysis of SARS-CoV-2 Genome Evolutionary Patterns\nAuthors: Gupta, S.; Gupta, D.; Bhatnagar, S.\nScore: 12.7, Published: 2023-06-21 DOI: 10.1101/2023.06.21.545910\nThe spread of SARS-CoV-2 virus accompanied by availability of abundant sequence data publicly, provides a window for determining the spatio-temporal patterns of viral evolution in response to vaccination. In this study, SARS-CoV-2 genome sequences were collected from seven countries in the period January 2020-December 2022. The sequences were classified into three phases, namely: pre-vaccination, post-vaccination, and recent period. Comparison was performed between these phases based on parameters like mutation rates, selection pressure (dN/dS ratio), and transition to transversion ratios (Ti/Tv). Similar comparisons were performed among SARS-CoV-2 variants. Statistical significance was tested using Graphpad unpaired t-test. The comparative analysis showed an increase in the percent genomic mutation rates post-vaccination and in recent periods across different countries from the pre-vaccination phase. The dN/dS ratios showed positive selection that increased after vaccination, and the Ti/Tv ratios decreased after vaccination. C[-\u0026gt;]U and G[-\u0026gt;]U were the most frequent transitions and transversions in all the countries. However, U[-\u0026gt;]G was the most frequent transversion in recent period. The Omicron variant had the highest genomic mutation rates, while Delta showed the highest dN/dS ratio. Mutation rates were highest in NSP3, S, N and NSP12b before and increased further after vaccination. NSP4 showed the largest change in mutation rates after vaccination. N, ORF8, ORF3a and ORF10 were under highest positive selection before vaccination. They were overtaken by E, S and NSP1 in the after vaccination as well as recent sequences, with the largest change observed in NSP1. Protein-wise dN/dS ratio was also seen to vary across the different variants. ImportanceIrrespective of the different vaccine technologies used, geographical regions and host genetics, variations in the SARS-CoV-2 genome have maintained similar patterns worldwide. To the best of our knowledge, there exists no other large-scale study of the genomic and protein-wise mutation patterns during the time course of evolution in different countries. Analysing the SARS-CoV-2 evolution patterns in response to spatial, temporal, and biological signals is important for diagnostics, therapeutics, and pharmacovigilance of SARS-CoV-2.\nShinyMultiome.UiO: An interactive open-source framework utilizing Seurat Objects for visualizing single-cell Multiomes\nAuthors: Akshay, A.; Sharma, A.; Eskeland, R.\nScore: 11.4, Published: 2023-06-25 DOI: 10.1101/2023.06.20.545756\nMotivationSingle-cell genomics has been revolutionized by the advent of single-cell Multiome sequencing which allows for simultaneous profiling of chromatin accessibility and gene expression in individual nuclei. The analysis and interpretation of large and often complex scMultiome datasets requires in-depth knowledge of computational programming. ResultsWe present ShinyMultiome.UiO, a user-friendly, integrative, and open-source web-based tool that supports Seurat objects for visualization of single-cell Multiome. The ShinyMultiome.UiO facilitates interactive reporting and comprehensive characterization of cellular heterogeneity and regulatory landscapes. With ShinyMultiome.UiO, users can facilitate collaborative efforts for interpretation of single-cell Multiome data and make it available to the public. Availability and implementationhttps://github.com/EskelandLab/ShinyMultiomeUiO and a demo server set up with a PBMC reference dataset https://cancell.medisin.uio.no/ShinyMultiome.UiO/ ContactAnkush.Sharma@medisin.uio.no \u0026 Ragnhild.Eskeland@medisin.uio.no\n",
  "wordCount" : "1956",
  "inLanguage": "en",
  "datePublished": "2023-06-28T09:51:07Z",
  "dateModified": "2023-06-28T09:51:07Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://yorks0n.github.io/TRxiv2/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://yorks0n.github.io/TRxiv2/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://yorks0n.github.io/TRxiv2" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on June 28, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.20.545604">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.20.545604" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.20.545604">
        <p class="paperTitle">QuantUMS: uncertainty minimisation enables confident quantification in proteomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.20.545604" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.20.545604" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kistner, F.; Grossmann, J. L.; Sinn, L. R.; Demichev, V.</p>
        <p class="info">Score: 49.3, Published: 2023-06-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.20.545604' target='https://doi.org/10.1101/2023.06.20.545604'> 10.1101/2023.06.20.545604</a></p>
        <p class="abstract">Mass spectrometry-based proteomics has been rapidly gaining traction as a powerful analytical method both in basic research and translation. While the problem of error control in peptide and protein identification has been addressed extensively, the quality of the resulting quantities remains challenging to evaluate. Here we introduce QuantUMS (Quantification using an Uncertainty Minimising Solution), a machine learning-based method which minimises errors and eliminates bias in peptide and protein quantification by integrating multiple sources of quantitative information. In combination with data-independent acquisition proteomics, QuantUMS boosts accuracy and precision of quantities, as well as reports an uncertainty metric, enabling effective filtering of data for downstream analysis. The algorithm has linear complexity with respect to the number of mass spectrometry acquisitions in the experiment and is thus scalable to infinitely large proteomic experiments. For an easy implementation in a proteomics laboratory, we integrate QuantUMS in our automated DIA-NN software suite.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.16.544954">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.16.544954" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.16.544954">
        <p class="paperTitle">TooManyCellsInteractive: a visualization tool for dynamic exploration of single-cell data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.16.544954" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.16.544954" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Klamann, C.; Lau, C.; Schwartz, G. W.</p>
        <p class="info">Score: 34.5, Published: 2023-06-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.16.544954' target='https://doi.org/10.1101/2023.06.16.544954'> 10.1101/2023.06.16.544954</a></p>
        <p class="abstract">As single-cell sequencing data sets grow in size, visualizations of large cellular populations become difficult to parse and require extensive processing to identify subpopulations of cells. Managing many of these charts is laborious for technical users and unintuitive for non-technical users. To address this issue, we developed TooManyCellsInteractive (TMCI), a browser-based JavaScript application for visualizing hierarchical cellular populations as an interactive radial tree. TMCI allows users to explore, filter, and manipulate hierarchical data structures through an intuitive interface while also enabling batch export of high-quality custom graphics. Here we describe the software architecture and illustrate how TMCI has identified unique survival pathways among drug-tolerant persister cells in a pan-cancer analysis. TMCI will help guide increasingly large data visualizations and facilitate multi-resolution data exploration in a user-friendly way.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2021.11.22.469635">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2021.11.22.469635" aria-expanded="false" aria-controls="flush-collapse10.1101/2021.11.22.469635">
        <p class="paperTitle">Comprehensive annotation of 3&#39;UTRs from primary cells and their quantification from scRNA-seq data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2021.11.22.469635" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2021.11.22.469635" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fansler, M. M.; Mitschka, S.; Mayr, C.</p>
        <p class="info">Score: 33.1, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2021.11.22.469635' target='https://doi.org/10.1101/2021.11.22.469635'> 10.1101/2021.11.22.469635</a></p>
        <p class="abstract">Approximately half of human genes generate mRNA isoforms that differ in their 3&#39;UTRs while encoding the same protein. 3&#39;UTR and mRNA length is determined by 3&#39; end cleavage sites (CS). Here, we mapped and categorized mRNA 3&#39; end CS in more than 200 primary human and mouse cell types, resulting in a 40% increase of CS annotations relative to the GENCODE database. We incorporated these annotations into a novel computational pipeline, called scUTRquant, for rapid, precise, and accurate quantification of gene and 3&#39;UTR isoform expression from single-cell RNA sequencing (scRNA-seq) data. When applying scUTRquant to data from 474 cell types and 2,134 perturbations, we discovered extensive 3&#39;UTR length changes across cell types that are as widespread and dynamically regulated as gene expression changes. Our data indicate that mRNA abundance and mRNA length are two independent axes of gene regulation that together determine the amount and spatial organization of protein synthesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.07.542483">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.07.542483" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.07.542483">
        <p class="paperTitle">BrainRBPedia: a resource for RNA-binding proteins relevant to neurodevelopmental disorders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.07.542483" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.07.542483" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Han, K.; Wainberg, M.; Calarco, J. A.; Smibert, C. A.; Lipshitz, H. D.; Lee, H. O.; Tripathy, S. J.</p>
        <p class="info">Score: 23.8, Published: 2023-06-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.07.542483' target='https://doi.org/10.1101/2023.06.07.542483'> 10.1101/2023.06.07.542483</a></p>
        <p class="abstract">RNA-binding proteins (RBPs) are crucial players in the post-transcriptional regulation of mRNA and play major roles in ensuring proper neuronal development and function. Deficits in RBP function have been implicated in a number of neurodevelopmental disorders including autism spectrum disorder (ASD) and intellectual disability (ID), yet we lack resources that integrate current knowledge of RBP function, tissue expression, and disease association in one place to aid in their experimental characterization. Here we introduce BrainRBPedia - a database of 1072 RBPs with both disease annotations for neurodevelopmental disorders and functional annotations relevant to these disorders, including loss-of-function intolerance and expression specificity to the brain, neurons, and neuronal development. Using these functional annotations, we develop a machine learning model to prioritize RBPs likely to be involved in ASD and ID. Our model indicates that RBPs with high loss-of-function intolerance and those upregulated during neuronal differentiation are disproportionately likely to contribute to ASD and ID etiology. In summary, BrainRBPedia comprises a unique resource for researchers interested in the experimental characterization of RBPs in relation to neurodevelopmental disorders and suggests functional signatures of RBPs likely to play a role in neurodevelopment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.22.546080">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.22.546080" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.22.546080">
        <p class="paperTitle">DeepRank-GNN-esm: A Graph Neural Network for Scoring Protein-Protein Models using Protein Language Model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.22.546080" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.22.546080" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xu, X.; Bonvin, A. M.</p>
        <p class="info">Score: 18.3, Published: 2023-06-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.22.546080' target='https://doi.org/10.1101/2023.06.22.546080'> 10.1101/2023.06.22.546080</a></p>
        <p class="abstract">Protein-Protein interactions (PPIs) play critical roles in numerous cellular processes. By modelling the three-dimensional structures of the correspond protein complexes valuable insights can be obtained, providing, for example, starting points for drug and protein design. One challenge in the modelling process is however the identification of near-native models from the large pool of generated models. To this end we previously developed DeepRank-GNN, a graph neural network that integrates structural and sequence information to enable effective pattern learning at PPI interfaces. Its main features are related to the Position Specific Scoring Matrices (PSSM), which are computationally expensive to generate and significantly limit the algorithm&#39;s usability. We introduce here DeepRank-GNN-esm that includes as additional features protein language model embeddings from the EMS-2 model. We show that the ESM-2 embeddings can actually replace the PSSM features at no cost in-, or even better performance on two PPI-related tasks: scoring docking poses and detecting crystal artifacts. This new DeepRank version bypasses thus the need of generating PSSM, greatly improving the usability of the software and opening new application opportunities for systems for which PSSM profiles cannot be obtained or are irrelevant (e.g. antibody-antigen complexes). DeepRank-GNN-esm is freely available from https://github.com/DeepRank/DeepRank-GNN-esm .</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.22.546069">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.22.546069" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.22.546069">
        <p class="paperTitle">Accurate prediction of protein assembly structure by combining AlphaFold and symmetrical docking</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.22.546069" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.22.546069" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jeppesen, M.; Andre, I.</p>
        <p class="info">Score: 16.4, Published: 2023-06-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.22.546069' target='https://doi.org/10.1101/2023.06.22.546069'> 10.1101/2023.06.22.546069</a></p>
        <p class="abstract">AlphaFold can predict the structures of monomeric and multimeric proteins with high accuracy but has a limit on the number of chains and residues it can fold. Here we show that a combination of AlphaFold and all-atom symmetric docking simulations enables highly accurate prediction of the structure of complex symmetrical assemblies. We present a method to predict the structure of complexes with cubic - tetrahedral, octahedral and icosahedral - symmetry from sequence. Focusing on proteins where AlphaFold can make confident predictions on the subunit structure, 21 cubic systems were assembled with a median TM-score of 0.99 and a DockQ score of 0.71. 15 had TM-scores of above 0.8 and were categorized as high-quality according to DockQ. The resulting models are energetically optimized and can be used for detailed studies of intermolecular interactions in higher-order symmetrical assemblies. The results demonstrate how explicit treatment of structural symmetry can significantly expand the size and complexity of AlphaFold predictions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545871">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545871" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545871">
        <p class="paperTitle">Co-linear Chaining on Pangenome Graphs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545871" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545871" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rajput, J.; Chandra, G.; Jain, C.</p>
        <p class="info">Score: 15.9, Published: 2023-06-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545871' target='https://doi.org/10.1101/2023.06.21.545871'> 10.1101/2023.06.21.545871</a></p>
        <p class="abstract">Pangenome reference graphs are useful in genomics because they compactly represent the genetic diversity within a species, a capability that linear references lack. However, efficiently aligning sequences to these graphs with complex topology and cycles can be challenging. The seed-chain-extend based alignment algorithms use co-linear chaining as a standard technique to identify a good cluster of exact seed matches that can be combined to form an alignment. Recent works show how the co-linear chaining problem can be efficiently solved for acyclic pangenome graphs by exploiting their small width [Makinen et al., TALG19] and how incorporating gap cost in the scoring function improves alignment accuracy [Chandra and Jain, RECOMB23]. However, it remains open on how to effectively generalize these techniques for general pangenome graphs which contain cycles. Here we present the first practical formulation and an exact algorithm for co-linear chaining on cyclic pangenome graphs. We rigorously prove the correctness and computational complexity of the proposed algorithm. We evaluate the empirical performance of our algorithm by aligning simulated long reads from the human genome to a cyclic pangenome graph constructed from 95 publicly available haplotype-resolved human genome assemblies. While the existing heuristic-based algorithms are faster, the proposed algorithm provides a significant advantage in terms of accuracy.

Implementationhttps://github.com/at-cg/PanAligner</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.545592">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.545592" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.545592">
        <p class="paperTitle">InGene: Finding influential genes from embeddings of nonlinear dimension reduction techniques</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.545592" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.545592" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Goswami, C.; Sengupta, D.</p>
        <p class="info">Score: 14.2, Published: 2023-06-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.545592' target='https://doi.org/10.1101/2023.06.19.545592'> 10.1101/2023.06.19.545592</a></p>
        <p class="abstract">We introduce InGene, the first of its kind, fast and scalable non-linear, unsupervised method for analyzing single-cell RNA sequencing data (scRNA-seq). While non-linear dimensionality reduction techniques such as tSNE and UMAP are effective at visualizing cellular sub-populations in low-dimensional space, they do not identify the specific genes that influence the transformation. InGene addresses this issue by assigning an importance score to each expressed gene based on its contribution to the construction of the low-dimensional map. InGene can provide insight into the cellular heterogeneity of scRNA-seq data and accurately identify genes associated with cell-type populations or diseases, as demonstrated in our analysis of scRNA-seq datasets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545910">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545910" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545910">
        <p class="paperTitle">Spatial and Temporal Analysis of SARS-CoV-2 Genome Evolutionary Patterns</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545910" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545910" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gupta, S.; Gupta, D.; Bhatnagar, S.</p>
        <p class="info">Score: 12.7, Published: 2023-06-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545910' target='https://doi.org/10.1101/2023.06.21.545910'> 10.1101/2023.06.21.545910</a></p>
        <p class="abstract">The spread of SARS-CoV-2 virus accompanied by availability of abundant sequence data publicly, provides a window for determining the spatio-temporal patterns of viral evolution in response to vaccination. In this study, SARS-CoV-2 genome sequences were collected from seven countries in the period January 2020-December 2022. The sequences were classified into three phases, namely: pre-vaccination, post-vaccination, and recent period. Comparison was performed between these phases based on parameters like mutation rates, selection pressure (dN/dS ratio), and transition to transversion ratios (Ti/Tv). Similar comparisons were performed among SARS-CoV-2 variants. Statistical significance was tested using Graphpad unpaired t-test. The comparative analysis showed an increase in the percent genomic mutation rates post-vaccination and in recent periods across different countries from the pre-vaccination phase. The dN/dS ratios showed positive selection that increased after vaccination, and the Ti/Tv ratios decreased after vaccination. C[-&amp;gt;]U and G[-&amp;gt;]U were the most frequent transitions and transversions in all the countries. However, U[-&amp;gt;]G was the most frequent transversion in recent period. The Omicron variant had the highest genomic mutation rates, while Delta showed the highest dN/dS ratio. Mutation rates were highest in NSP3, S, N and NSP12b before and increased further after vaccination. NSP4 showed the largest change in mutation rates after vaccination. N, ORF8, ORF3a and ORF10 were under highest positive selection before vaccination. They were overtaken by E, S and NSP1 in the after vaccination as well as recent sequences, with the largest change observed in NSP1. Protein-wise dN/dS ratio was also seen to vary across the different variants.

ImportanceIrrespective of the different vaccine technologies used, geographical regions and host genetics, variations in the SARS-CoV-2 genome have maintained similar patterns worldwide. To the best of our knowledge, there exists no other large-scale study of the genomic and protein-wise mutation patterns during the time course of evolution in different countries. Analysing the SARS-CoV-2 evolution patterns in response to spatial, temporal, and biological signals is important for diagnostics, therapeutics, and pharmacovigilance of SARS-CoV-2.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.20.545756">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.20.545756" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.20.545756">
        <p class="paperTitle">ShinyMultiome.UiO: An interactive open-source framework utilizing Seurat Objects for visualizing single-cell Multiomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.20.545756" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.20.545756" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Akshay, A.; Sharma, A.; Eskeland, R.</p>
        <p class="info">Score: 11.4, Published: 2023-06-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.20.545756' target='https://doi.org/10.1101/2023.06.20.545756'> 10.1101/2023.06.20.545756</a></p>
        <p class="abstract">MotivationSingle-cell genomics has been revolutionized by the advent of single-cell Multiome sequencing which allows for simultaneous profiling of chromatin accessibility and gene expression in individual nuclei. The analysis and interpretation of large and often complex scMultiome datasets requires in-depth knowledge of computational programming.

ResultsWe present ShinyMultiome.UiO, a user-friendly, integrative, and open-source web-based tool that supports Seurat objects for visualization of single-cell Multiome. The ShinyMultiome.UiO facilitates interactive reporting and comprehensive characterization of cellular heterogeneity and regulatory landscapes. With ShinyMultiome.UiO, users can facilitate collaborative efforts for interpretation of single-cell Multiome data and make it available to the public.

Availability and implementationhttps://github.com/EskelandLab/ShinyMultiomeUiO and a demo server set up with a PBMC reference dataset https://cancell.medisin.uio.no/ShinyMultiome.UiO/

ContactAnkush.Sharma@medisin.uio.no &amp; Ragnhild.Eskeland@medisin.uio.no</p>
      </div>
    </div>
  </div>
</div>









<script src="/TRxiv2/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://yorks0n.github.io/TRxiv2">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
