<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Accurate human genome analysis with Element Avidity sequencing
Authors: Carroll, A.; Kolesnikov, A.; Cook, D. E.; Brambrink, L.; Wiseman, K. N.; Billings, S. M.; Kruglyak, S.; Lajoie, B. R.; Zhao, J.; Levy, S. E.; McLean, C. Y.; Shafin, K.; Nattestad, M.; Chang, P.-C.
Score: 543.5, Published: 2023-08-14 DOI: 10.1101/2023.08.11.553043
We investigate the new sequencing technology Avidity from Element Biosciences. We show that Element whole genome sequencing achieves higher mapping and variant calling accuracy compared to Illumina sequencing at the same coverage, with larger differences at lower coverages (20x-30x).">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="Accurate human genome analysis with Element Avidity sequencing
Authors: Carroll, A.; Kolesnikov, A.; Cook, D. E.; Brambrink, L.; Wiseman, K. N.; Billings, S. M.; Kruglyak, S.; Lajoie, B. R.; Zhao, J.; Levy, S. E.; McLean, C. Y.; Shafin, K.; Nattestad, M.; Chang, P.-C.
Score: 543.5, Published: 2023-08-14 DOI: 10.1101/2023.08.11.553043
We investigate the new sequencing technology Avidity from Element Biosciences. We show that Element whole genome sequencing achieves higher mapping and variant calling accuracy compared to Illumina sequencing at the same coverage, with larger differences at lower coverages (20x-30x)." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-10T10:36:49+00:00" />
<meta property="article:modified_time" content="2023-09-10T10:36:49+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="Accurate human genome analysis with Element Avidity sequencing
Authors: Carroll, A.; Kolesnikov, A.; Cook, D. E.; Brambrink, L.; Wiseman, K. N.; Billings, S. M.; Kruglyak, S.; Lajoie, B. R.; Zhao, J.; Levy, S. E.; McLean, C. Y.; Shafin, K.; Nattestad, M.; Chang, P.-C.
Score: 543.5, Published: 2023-08-14 DOI: 10.1101/2023.08.11.553043
We investigate the new sequencing technology Avidity from Element Biosciences. We show that Element whole genome sequencing achieves higher mapping and variant calling accuracy compared to Illumina sequencing at the same coverage, with larger differences at lower coverages (20x-30x)."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "Accurate human genome analysis with Element Avidity sequencing\nAuthors: Carroll, A.; Kolesnikov, A.; Cook, D. E.; Brambrink, L.; Wiseman, K. N.; Billings, S. M.; Kruglyak, S.; Lajoie, B. R.; Zhao, J.; Levy, S. E.; McLean, C. Y.; Shafin, K.; Nattestad, M.; Chang, P.-C.\nScore: 543.5, Published: 2023-08-14 DOI: 10.1101/2023.08.11.553043\nWe investigate the new sequencing technology Avidity from Element Biosciences. We show that Element whole genome sequencing achieves higher mapping and variant calling accuracy compared to Illumina sequencing at the same coverage, with larger differences at lower coverages (20x-30x).",
  "keywords": [
    
  ],
  "articleBody": " Accurate human genome analysis with Element Avidity sequencing\nAuthors: Carroll, A.; Kolesnikov, A.; Cook, D. E.; Brambrink, L.; Wiseman, K. N.; Billings, S. M.; Kruglyak, S.; Lajoie, B. R.; Zhao, J.; Levy, S. E.; McLean, C. Y.; Shafin, K.; Nattestad, M.; Chang, P.-C.\nScore: 543.5, Published: 2023-08-14 DOI: 10.1101/2023.08.11.553043\nWe investigate the new sequencing technology Avidity from Element Biosciences. We show that Element whole genome sequencing achieves higher mapping and variant calling accuracy compared to Illumina sequencing at the same coverage, with larger differences at lower coverages (20x-30x). We quantify base error rates of Element reads, finding lower error rates, especially in homopolymer and tandem repeat regions. We use Elements ability to generate paired end sequencing with longer insert sizes than typical short-read sequencing. We show that longer insert sizes result in even higher accuracy, with long insert Element sequencing giving noticeably more accurate genome analyses at all coverages.\nGene count normalization in single-cell imaging-based spatially resolved transcriptomics\nAuthors: Atta, L.; Clifton, K.; Anant, M.; Fan, J.\nScore: 35.2, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555624\nRecent advances in imaging-based spatially resolved transcriptomics (im-SRT) technologies now enable high-throughput profiling of targeted genes and their locations in fixed tissues. Normalization of gene expression data is often needed to account for technical factors that may confound underlying biological signals. Here, we investigate the potential impact of different gene count normalization methods with different targeted gene panels in the analysis and interpretation of im-SRT data. Using different simulated gene panels that overrepresent genes expressed in specific tissue anatomical regions or cell types, we find that normalization methods that use scaling factors derived from gene counts differentially impact normalized gene expression magnitudes in a region- or cell type-specific manner. We show that these normalization-induced effects may reduce the reliability of downstream differential gene expression and fold change analysis, introducing false positive and false negative results when compared to results obtained from gene panels that are more representative of the gene expression of the tissues component cell types. These effects are not observed without normalization or when scaling factors are not derived from gene counts, such as with cell volume normalization. Overall, we caution that the choice of normalization method and gene panel may impact the biological interpretation of the im-SRT data.\nOmix: A Multi-Omics Integration Pipeline\nAuthors: Schneegans, E.; Fancy, N.; Thomas, M.; Willumsen, N.; Matthews, P. M.; Jackson, J.\nScore: 35.2, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555486\nSummaryThe Omix pipeline offers an integration and analysis framework for multiomics intended to preprocess, analyse, and visualise multimodal data flexibly to address various research questions. From biomarker discovery and patient stratification to the investigation of complex biological processes, Omix empowers researchers to derive valuable insights from omics data. Using Alzheimers Disease (AD) bulk proteomics and transcriptomics datasets generated from two distinct regions derived from post-mortem brains, we demonstrate the utility of Omix in generating an integrated pseudo-temporal multi-omics profile of AD. Availability and ImplementationOmix is implemented as a software package in R. The code for the Omix package is available at https://github.com/eleonoreschneeg/Omix. Reference documentation and online tutorials are available at https://eleonore-schneeg.github.io/Omix. All code is open-source and available under the GNU General Public License v3.0 (GPL-3). Contacteleonore.schneegans17@imperial.ac.uk, johanna.jackson@imperial.ac.uk\nDeep Learning for Protein Peptide bindingPrediction: Incorporating Sequence, Structural andLanguage Model Features\nAuthors: Chandra, A.; Sharma, A.; Dehzangi, I.; Tsunoda, T.; Sattar, A.\nScore: 26.8, Published: 2023-09-06 DOI: 10.1101/2023.09.02.556055\nProtein-peptide interactions play a crucial role in various cellular processes and are implicated in abnormal cellular behaviors leading to diseases such as cancer. Therefore, understanding these interactions is vital for both functional genomics and drug discovery efforts. Despite a significant increase in the availability of protein-peptide complexes, experimental methods for studying these interactions remain laborious, time-consuming, and expensive. Computational methods offer a complementary approach but often fall short in terms of prediction accuracy. To address these challenges, we introduce PepCNN, a deep learning-based prediction model that incorporates structural and sequence-based information from primary protein sequences. By utilizing a combination of half-sphere exposure, position specific scoring matrices, and pre-trained transformer language model, PepCNN outperforms state-of-the-art methods in terms of specificity, precision, and AUC. The PepCNN software and datasets are publicly available at https://github.com/abelavit/PepCNN.git.\nAutometa 2: A versatile tool for recovering genomes from highly-complex metagenomic communities\nAuthors: Rees, E. R.; Uppal, S.; Clark, C.; Lail, A. J.; Waterworth, S. C.; Roesemann, S. D.; Wolf, K. A.; Kwan, J. C.\nScore: 20.3, Published: 2023-09-05 DOI: 10.1101/2023.09.01.555939\nIn 2019, we developed Autometa, an automated binning pipeline that is able to effectively recover metagenome-assembled genomes from complex environmental and non-model host-associated microbial communities. Autometa has gained widespread use in a variety of environments and has been applied in multiple research projects. However, the genome-binning workflow was at times overly complex and computationally demanding. As a consequence of Autometa's diverse application, non-technical and technical researchers alike have noted its burdensome installation and inefficient as well as error-prone processes. Moreover its taxon-binning and genome-binning behaviors have remained obscure. For these reasons we set out to improve its accessibility, efficiency and efficacy to further enable the research community during their exploration of Earth's environments. The highly augmented Autometa 2 release, which we present here, has vastly simplified installation, a graphical user interface and a refactored workflow for transparency and reproducibility. Furthermore, we conducted a parameter sweep on standardized community datasets to show that it is possible for Autometa to achieve better performance than any other binning pipeline, as judged by Adjusted Rand Index. Improvements in Autometa 2 enhance its accessibility for non-bioinformatic oriented researchers, scalability for large-scale and highly-complex samples and interpretation of recovered microbial communities.\nevopython: a Python package for feature-focused, comparative genomic data exploration\nAuthors: Mick, S. T.; Fiszbein, A.\nScore: 19.2, Published: 2023-09-06 DOI: 10.1101/2023.09.02.556042\nMotivationSoftware such as LiftOver allows for the intra- and inter-species conversion of genomic coordinates between genome assemblies, but this coordinate-centric workflow is naive to sequence alterations underling the conversion. For instance, it does not clarify the location of any insertions or deletions that might have occurred. ResultsTo facilitate inter-species, sequence-based analyses, we developed evopython, a simple, object-oriented Python package that enables the sequence-aware resolution of genomic coordinates directly from pairwise and multiple whole-genome alignment data. The output is a Python dictionary storing all information relevant to the alignment: the participating species names, the position of the alignment in each species genome assembly, and the aligned sequences. Availability and implementationThe source code and documentation are available at https://github.com/fiszbein-lab/evopython. Contactanafisz@bu.edu\nBRAKER3: Fully Automated Genome Annotation Using RNA-Seq and Protein Evidence with GeneMark-ETP, AUGUSTUS and TSEBRA\nAuthors: Gabriel, L.; Bruna, T.; Hoff, K. J.; Ebel, M.; Lomsadze, A.; Borodovsky, M.; Stanke, M.\nScore: 114.4, Published: 2023-09-02 DOI: 10.1101/2023.06.10.544449\nGene prediction remains an active area of bioinformatics research. Challenges are presented by large eukaryotic genomes and heterogeneous data situations. To meet the challenges, several streams of evidence must be integrated, from protein homology and transcriptome data, as well as information derived from the genome itself. The amount and significance of the available evidence from transcriptomes and proteomes vary from genome to genome, between genes and even along a single gene. User-friendly and accurate annotation pipelines that can cope with such data heterogeneity are needed. The previously developed annotation pipelines BRAKER1 and BRAKER2 use RNA-Seq or protein data, respectively, but not both. The recently released GeneMark-ETP integrates all three types of data and achieves much higher levels of accuracy. We here present the BRAKER3 pipeline that builds on GeneMark-ETP and AUGUSTUS and further improves accuracy using the TSEBRA combiner. BRAKER3 annotates protein-coding genes in eukaryotic genomes using both short-read RNA-Seq and a large protein database along with statistical models learned iteratively and specifically for the target genome. We benchmarked the new pipeline on 11 species under controlled conditions on the assumed relatedness of the target species to available proteomes. BRAKER3 outperformed BRAKER1 and BRAKER2, increasing the average transcript-level F1-score by[~] 20 percentage points, most pronounced for species with large and complex genomes. BRAKER3 also outperforms MAKER2 and Funannotate. For the first time, we provide a Singularity container for the BRAKER software to minimize installation obstacles. Overall, BRAKER3 is an accurate, easy-to-use tool for the annotation of eukaryotic genomes.\n3Dmapper: A Command Line Tool For BioBank-scale Mapping Of Variants To Protein Structures\nAuthors: Ruiz-Serra, V.; Valentini, S.; Madronero, S.; Valencia, A.; Porta-Pardo, E.\nScore: 14.7, Published: 2023-09-05 DOI: 10.1101/2023.09.01.555502\nThe interpretation of genomic data is crucial to understand the molecular mechanisms of biological processes. Protein structures play a vital role in facilitating this interpretation by providing functional context to genetic coding variants. However, mapping genes to proteins is a tedious and error-prone task due to inconsistencies in data formats. Over the past two decades, numerous tools and databases have been developed to automatically map annotated positions and variants to protein structures. However, most of these tools are web-based and not well-suited for large-scale genomic data analysis. To address this issue, we introduce 3Dmapper, a standalone command-line tool developed in Python and R. It systematically maps annotated protein positions and variants to protein structures, providing a solution that is both efficient and reliable. 3Dmapper is freely available on GitHub at https://github.com/vicruiser/3Dmapper.\nAssessing the performance of protein regression models\nAuthors: Michael, R.; Kaestel-Hansen, J.; Groth, P. M.; Bartels, S.; Salomon, J.; Tian, P.; Hatzakis, N. S.; Boomsma, W.\nScore: 22.5, Published: 2023-09-02 DOI: 10.1101/2023.06.18.545472\nTo optimize proteins for particular traits holds great promise for industrial and pharmaceutical purposes. Machine Learning is increasingly applied in this field to predict properties of proteins, thereby guiding the experimental optimization process. A natural question is: How much progress are we making with such predictions, and how important is the choice of regressor and representation? In this paper, we demonstrate that different assessment criteria for regressor performance can lead to dramatically different conclusions, depending on the choice of metric, and how one defines generalization. We highlight the fundamental issues of sample bias in typical regression scenarios and how this can lead to misleading conclusions about regressor performance. Finally, we make the case for the importance of calibrated uncertainty in this domain.\nCellsFromSpace: A versatile tool for spatial transcriptomic data analysis with reference-free deconvolution and guided cell type/activity annotation\nAuthors: Thuilliez, C.; Moquin-Beaudry, G.; Khneisser, p.; Marques Da Costa, M. E.; Karkar, S.; Boudhouche, H.; Drubay, D.; Audinot, B.; Geoerger, B.; Scoazec, J.-Y.; Gaspar, N.; Marchais, A.\nScore: 11.9, Published: 2023-09-04 DOI: 10.1101/2023.08.30.555558\nSpatial transcriptomics involves capturing the transcriptomic profiles of millions of cells within their spatial contexts, enabling the analysis of cell crosstalk in healthy and diseased organs. However, spatial transcriptomics also raises new computational challenges for analyzing multidimensional data associated with spatial coordinates. Depending on the technology used, the capture of transcript is either at a near single-cell level or at a subcellular level. This introduces corresponding challenges, such as inferring cell composition in regions covering multiple cells or segmenting z-stacked cells from images to assign transcripts to specific cells. In this context, we introduce a novel framework called CellsFromSpace. This framework allows users to analyze various commercially available technologies without relying on a single-cell reference dataset. Based on the independent component analysis, CellsFromSpace decomposes spatial transcriptomic data into components that represent distinct cell types or activities. The framework provides a user-friendly graphical interface that enables non-bioinformaticians to perform a full analysis and to annotate the components based on marker genes and spatial distributions. The direct annotation of components, allows users to identify and isolate cell populations in the latent space, even when they overlap. Then, they can focus their studies on specific populations or analyze the proximity between different populations. Additionally, CellsFromSpace offers the capability to reduce noise or artifacts by component selection and supports analyses on multiple datasets simultaneously. Here, we demonstrate the efficiency of CellsFromSpace to successfully identifies spatially distributed cells as well as rare diffuse cells on datasets from the Visium, Slide-seq, MERSCOPE, and COSMX technologies. Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=190 SRC=\"FIGDIR/small/555558v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (67K): org.highwire.dtl.DTLVardef@64603borg.highwire.dtl.DTLVardef@1e9e283org.highwire.dtl.DTLVardef@16470aorg.highwire.dtl.DTLVardef@91e4a5_HPS_FORMAT_FIGEXP M_FIG C_FIG\n",
  "wordCount" : "1979",
  "inLanguage": "en",
  "datePublished": "2023-09-10T10:36:49Z",
  "dateModified": "2023-09-10T10:36:49Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on September 10, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.553043">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.553043" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.553043">
        <p class="paperTitle">Accurate human genome analysis with Element Avidity sequencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.553043" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.553043" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carroll, A.; Kolesnikov, A.; Cook, D. E.; Brambrink, L.; Wiseman, K. N.; Billings, S. M.; Kruglyak, S.; Lajoie, B. R.; Zhao, J.; Levy, S. E.; McLean, C. Y.; Shafin, K.; Nattestad, M.; Chang, P.-C.</p>
        <p class="info">Score: 543.5, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.553043' target='https://doi.org/10.1101/2023.08.11.553043'> 10.1101/2023.08.11.553043</a></p>
        <p class="abstract">We investigate the new sequencing technology Avidity from Element Biosciences. We show that Element whole genome sequencing achieves higher mapping and variant calling accuracy compared to Illumina sequencing at the same coverage, with larger differences at lower coverages (20x-30x). We quantify base error rates of Element reads, finding lower error rates, especially in homopolymer and tandem repeat regions. We use Elements ability to generate paired end sequencing with longer insert sizes than typical short-read sequencing. We show that longer insert sizes result in even higher accuracy, with long insert Element sequencing giving noticeably more accurate genome analyses at all coverages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555624">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555624" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555624">
        <p class="paperTitle">Gene count normalization in single-cell imaging-based spatially resolved transcriptomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555624" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555624" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Atta, L.; Clifton, K.; Anant, M.; Fan, J.</p>
        <p class="info">Score: 35.2, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555624' target='https://doi.org/10.1101/2023.08.30.555624'> 10.1101/2023.08.30.555624</a></p>
        <p class="abstract">Recent advances in imaging-based spatially resolved transcriptomics (im-SRT) technologies now enable high-throughput profiling of targeted genes and their locations in fixed tissues. Normalization of gene expression data is often needed to account for technical factors that may confound underlying biological signals. Here, we investigate the potential impact of different gene count normalization methods with different targeted gene panels in the analysis and interpretation of im-SRT data. Using different simulated gene panels that overrepresent genes expressed in specific tissue anatomical regions or cell types, we find that normalization methods that use scaling factors derived from gene counts differentially impact normalized gene expression magnitudes in a region- or cell type-specific manner. We show that these normalization-induced effects may reduce the reliability of downstream differential gene expression and fold change analysis, introducing false positive and false negative results when compared to results obtained from gene panels that are more representative of the gene expression of the tissues component cell types. These effects are not observed without normalization or when scaling factors are not derived from gene counts, such as with cell volume normalization. Overall, we caution that the choice of normalization method and gene panel may impact the biological interpretation of the im-SRT data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555486">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555486" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555486">
        <p class="paperTitle">Omix: A Multi-Omics Integration Pipeline</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555486" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555486" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schneegans, E.; Fancy, N.; Thomas, M.; Willumsen, N.; Matthews, P. M.; Jackson, J.</p>
        <p class="info">Score: 35.2, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555486' target='https://doi.org/10.1101/2023.08.30.555486'> 10.1101/2023.08.30.555486</a></p>
        <p class="abstract">SummaryThe Omix pipeline offers an integration and analysis framework for multiomics intended to preprocess, analyse, and visualise multimodal data flexibly to address various research questions. From biomarker discovery and patient stratification to the investigation of complex biological processes, Omix empowers researchers to derive valuable insights from omics data. Using Alzheimers Disease (AD) bulk proteomics and transcriptomics datasets generated from two distinct regions derived from post-mortem brains, we demonstrate the utility of Omix in generating an integrated pseudo-temporal multi-omics profile of AD.

Availability and ImplementationOmix is implemented as a software package in R. The code for the Omix package is available at https://github.com/eleonoreschneeg/Omix. Reference documentation and online tutorials are available at https://eleonore-schneeg.github.io/Omix. All code is open-source and available under the GNU General Public License v3.0 (GPL-3).

Contacteleonore.schneegans17@imperial.ac.uk, johanna.jackson@imperial.ac.uk</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.02.556055">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.02.556055" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.02.556055">
        <p class="paperTitle">Deep Learning for Protein Peptide bindingPrediction: Incorporating Sequence, Structural andLanguage Model Features</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.02.556055" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.02.556055" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chandra, A.; Sharma, A.; Dehzangi, I.; Tsunoda, T.; Sattar, A.</p>
        <p class="info">Score: 26.8, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.02.556055' target='https://doi.org/10.1101/2023.09.02.556055'> 10.1101/2023.09.02.556055</a></p>
        <p class="abstract">Protein-peptide interactions play a crucial role in various cellular processes and are implicated in abnormal cellular behaviors leading to diseases such as cancer. Therefore, understanding these interactions is vital for both functional genomics and drug discovery efforts. Despite a significant increase in the availability of protein-peptide complexes, experimental methods for studying these interactions remain laborious, time-consuming, and expensive. Computational methods offer a complementary approach but often fall short in terms of prediction accuracy. To address these challenges, we introduce PepCNN, a deep learning-based prediction model that incorporates structural and sequence-based information from primary protein sequences. By utilizing a combination of half-sphere exposure, position specific scoring matrices, and pre-trained transformer language model, PepCNN outperforms state-of-the-art methods in terms of specificity, precision, and AUC. The PepCNN software and datasets are publicly available at https://github.com/abelavit/PepCNN.git.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555939">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555939" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555939">
        <p class="paperTitle">Autometa 2: A versatile tool for recovering genomes from highly-complex metagenomic communities</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555939" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555939" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rees, E. R.; Uppal, S.; Clark, C.; Lail, A. J.; Waterworth, S. C.; Roesemann, S. D.; Wolf, K. A.; Kwan, J. C.</p>
        <p class="info">Score: 20.3, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555939' target='https://doi.org/10.1101/2023.09.01.555939'> 10.1101/2023.09.01.555939</a></p>
        <p class="abstract">In 2019, we developed Autometa, an automated binning pipeline that is able to effectively recover metagenome-assembled genomes from complex environmental and non-model host-associated microbial communities. Autometa has gained widespread use in a variety of environments and has been applied in multiple research projects. However, the genome-binning workflow was at times overly complex and computationally demanding. As a consequence of Autometa&#39;s diverse application, non-technical and technical researchers alike have noted its burdensome installation and inefficient as well as error-prone processes. Moreover its taxon-binning and genome-binning behaviors have remained obscure. For these reasons we set out to improve its accessibility, efficiency and efficacy to further enable the research community during their exploration of Earth&#39;s environments. The highly augmented Autometa 2 release, which we present here, has vastly simplified installation, a graphical user interface and a refactored workflow for transparency and reproducibility. Furthermore, we conducted a parameter sweep on standardized community datasets to show that it is possible for Autometa to achieve better performance than any other binning pipeline, as judged by Adjusted Rand Index. Improvements in Autometa 2 enhance its accessibility for non-bioinformatic oriented researchers, scalability for large-scale and highly-complex samples and interpretation of recovered microbial communities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.02.556042">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.02.556042" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.02.556042">
        <p class="paperTitle">evopython: a Python package for feature-focused, comparative genomic data exploration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.02.556042" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.02.556042" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mick, S. T.; Fiszbein, A.</p>
        <p class="info">Score: 19.2, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.02.556042' target='https://doi.org/10.1101/2023.09.02.556042'> 10.1101/2023.09.02.556042</a></p>
        <p class="abstract">MotivationSoftware such as LiftOver allows for the intra- and inter-species conversion of genomic coordinates between genome assemblies, but this coordinate-centric workflow is naive to sequence alterations underling the conversion. For instance, it does not clarify the location of any insertions or deletions that might have occurred.

ResultsTo facilitate inter-species, sequence-based analyses, we developed evopython, a simple, object-oriented Python package that enables the sequence-aware resolution of genomic coordinates directly from pairwise and multiple whole-genome alignment data. The output is a Python dictionary storing all information relevant to the alignment: the participating species names, the position of the alignment in each species genome assembly, and the aligned sequences.

Availability and implementationThe source code and documentation are available at https://github.com/fiszbein-lab/evopython.

Contactanafisz@bu.edu</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.10.544449">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.10.544449" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.10.544449">
        <p class="paperTitle">BRAKER3: Fully Automated Genome Annotation Using RNA-Seq and Protein Evidence with GeneMark-ETP, AUGUSTUS and TSEBRA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.10.544449" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.10.544449" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gabriel, L.; Bruna, T.; Hoff, K. J.; Ebel, M.; Lomsadze, A.; Borodovsky, M.; Stanke, M.</p>
        <p class="info">Score: 114.4, Published: 2023-09-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.10.544449' target='https://doi.org/10.1101/2023.06.10.544449'> 10.1101/2023.06.10.544449</a></p>
        <p class="abstract">Gene prediction remains an active area of bioinformatics research. Challenges are presented by large eukaryotic genomes and heterogeneous data situations. To meet the challenges, several streams of evidence must be integrated, from protein homology and transcriptome data, as well as information derived from the genome itself. The amount and significance of the available evidence from transcriptomes and proteomes vary from genome to genome, between genes and even along a single gene. User-friendly and accurate annotation pipelines that can cope with such data heterogeneity are needed. The previously developed annotation pipelines BRAKER1 and BRAKER2 use RNA-Seq or protein data, respectively, but not both. The recently released GeneMark-ETP integrates all three types of data and achieves much higher levels of accuracy.

We here present the BRAKER3 pipeline that builds on GeneMark-ETP and AUGUSTUS and further improves accuracy using the TSEBRA combiner. BRAKER3 annotates protein-coding genes in eukaryotic genomes using both short-read RNA-Seq and a large protein database along with statistical models learned iteratively and specifically for the target genome. We benchmarked the new pipeline on 11 species under controlled conditions on the assumed relatedness of the target species to available proteomes. BRAKER3 outperformed BRAKER1 and BRAKER2, increasing the average transcript-level F1-score by[~] 20 percentage points, most pronounced for species with large and complex genomes. BRAKER3 also outperforms MAKER2 and Funannotate. For the first time, we provide a Singularity container for the BRAKER software to minimize installation obstacles. Overall, BRAKER3 is an accurate, easy-to-use tool for the annotation of eukaryotic genomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555502">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555502" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555502">
        <p class="paperTitle">3Dmapper: A Command Line Tool For BioBank-scale Mapping Of Variants To Protein Structures</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555502" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555502" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ruiz-Serra, V.; Valentini, S.; Madronero, S.; Valencia, A.; Porta-Pardo, E.</p>
        <p class="info">Score: 14.7, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555502' target='https://doi.org/10.1101/2023.09.01.555502'> 10.1101/2023.09.01.555502</a></p>
        <p class="abstract">The interpretation of genomic data is crucial to understand the molecular mechanisms of biological processes. Protein structures play a vital role in facilitating this interpretation by providing functional context to genetic coding variants. However, mapping genes to proteins is a tedious and error-prone task due to inconsistencies in data formats. Over the past two decades, numerous tools and databases have been developed to automatically map annotated positions and variants to protein structures. However, most of these tools are web-based and not well-suited for large-scale genomic data analysis. To address this issue, we introduce 3Dmapper, a standalone command-line tool developed in Python and R. It systematically maps annotated protein positions and variants to protein structures, providing a solution that is both efficient and reliable. 3Dmapper is freely available on GitHub at https://github.com/vicruiser/3Dmapper.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.18.545472">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.18.545472" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.18.545472">
        <p class="paperTitle">Assessing the performance of protein regression models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.18.545472" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.18.545472" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Michael, R.; Kaestel-Hansen, J.; Groth, P. M.; Bartels, S.; Salomon, J.; Tian, P.; Hatzakis, N. S.; Boomsma, W.</p>
        <p class="info">Score: 22.5, Published: 2023-09-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.18.545472' target='https://doi.org/10.1101/2023.06.18.545472'> 10.1101/2023.06.18.545472</a></p>
        <p class="abstract">To optimize proteins for particular traits holds great promise for industrial and pharmaceutical purposes. Machine Learning is increasingly applied in this field to predict properties of proteins, thereby guiding the experimental optimization process. A natural question is: How much progress are we making with such predictions, and how important is the choice of regressor and representation? In this paper, we demonstrate that different assessment criteria for regressor performance can lead to dramatically different conclusions, depending on the choice of metric, and how one defines generalization. We highlight the fundamental issues of sample bias in typical regression scenarios and how this can lead to misleading conclusions about regressor performance. Finally, we make the case for the importance of calibrated uncertainty in this domain.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555558">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555558" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555558">
        <p class="paperTitle">CellsFromSpace: A versatile tool for spatial transcriptomic data analysis with reference-free deconvolution and guided cell type/activity annotation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555558" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555558" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Thuilliez, C.; Moquin-Beaudry, G.; Khneisser, p.; Marques Da Costa, M. E.; Karkar, S.; Boudhouche, H.; Drubay, D.; Audinot, B.; Geoerger, B.; Scoazec, J.-Y.; Gaspar, N.; Marchais, A.</p>
        <p class="info">Score: 11.9, Published: 2023-09-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555558' target='https://doi.org/10.1101/2023.08.30.555558'> 10.1101/2023.08.30.555558</a></p>
        <p class="abstract">Spatial transcriptomics involves capturing the transcriptomic profiles of millions of cells within their spatial contexts, enabling the analysis of cell crosstalk in healthy and diseased organs. However, spatial transcriptomics also raises new computational challenges for analyzing multidimensional data associated with spatial coordinates.

Depending on the technology used, the capture of transcript is either at a near single-cell level or at a subcellular level. This introduces corresponding challenges, such as inferring cell composition in regions covering multiple cells or segmenting z-stacked cells from images to assign transcripts to specific cells.

In this context, we introduce a novel framework called CellsFromSpace. This framework allows users to analyze various commercially available technologies without relying on a single-cell reference dataset. Based on the independent component analysis, CellsFromSpace decomposes spatial transcriptomic data into components that represent distinct cell types or activities.

The framework provides a user-friendly graphical interface that enables non-bioinformaticians to perform a full analysis and to annotate the components based on marker genes and spatial distributions. The direct annotation of components, allows users to identify and isolate cell populations in the latent space, even when they overlap. Then, they can focus their studies on specific populations or analyze the proximity between different populations. Additionally, CellsFromSpace offers the capability to reduce noise or artifacts by component selection and supports analyses on multiple datasets simultaneously.

Here, we demonstrate the efficiency of CellsFromSpace to successfully identifies spatially distributed cells as well as rare diffuse cells on datasets from the Visium, Slide-seq, MERSCOPE, and COSMX technologies.

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=190 SRC=&#34;FIGDIR/small/555558v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (67K):
org.highwire.dtl.DTLVardef@64603borg.highwire.dtl.DTLVardef@1e9e283org.highwire.dtl.DTLVardef@16470aorg.highwire.dtl.DTLVardef@91e4a5_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
