<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Rendering protein mutation movies with MutAmore
Authors: Weissenow, K.; Rost, B.
Score: 39.6, Published: 2023-09-17 DOI: 10.1101/2023.09.15.557870
We introduce MutAmore, a bioinformatics tool that renders individual photos of, e.g., protein three-dimensional (3D) structures for protein mutations into an easily understandable movie format. MutAmore provides a streamlined pipeline casting single amino-acid variations (SAVs) into a dynamic 3D protein mutation movie which provides a qualitative perspective on the mutational landscape of a protein.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="Rendering protein mutation movies with MutAmore
Authors: Weissenow, K.; Rost, B.
Score: 39.6, Published: 2023-09-17 DOI: 10.1101/2023.09.15.557870
We introduce MutAmore, a bioinformatics tool that renders individual photos of, e.g., protein three-dimensional (3D) structures for protein mutations into an easily understandable movie format. MutAmore provides a streamlined pipeline casting single amino-acid variations (SAVs) into a dynamic 3D protein mutation movie which provides a qualitative perspective on the mutational landscape of a protein." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-24T10:37:06+00:00" />
<meta property="article:modified_time" content="2023-09-24T10:37:06+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="Rendering protein mutation movies with MutAmore
Authors: Weissenow, K.; Rost, B.
Score: 39.6, Published: 2023-09-17 DOI: 10.1101/2023.09.15.557870
We introduce MutAmore, a bioinformatics tool that renders individual photos of, e.g., protein three-dimensional (3D) structures for protein mutations into an easily understandable movie format. MutAmore provides a streamlined pipeline casting single amino-acid variations (SAVs) into a dynamic 3D protein mutation movie which provides a qualitative perspective on the mutational landscape of a protein."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "Rendering protein mutation movies with MutAmore\nAuthors: Weissenow, K.; Rost, B.\nScore: 39.6, Published: 2023-09-17 DOI: 10.1101/2023.09.15.557870\nWe introduce MutAmore, a bioinformatics tool that renders individual photos of, e.g., protein three-dimensional (3D) structures for protein mutations into an easily understandable movie format. MutAmore provides a streamlined pipeline casting single amino-acid variations (SAVs) into a dynamic 3D protein mutation movie which provides a qualitative perspective on the mutational landscape of a protein.",
  "keywords": [
    
  ],
  "articleBody": " Rendering protein mutation movies with MutAmore\nAuthors: Weissenow, K.; Rost, B.\nScore: 39.6, Published: 2023-09-17 DOI: 10.1101/2023.09.15.557870\nWe introduce MutAmore, a bioinformatics tool that renders individual photos of, e.g., protein three-dimensional (3D) structures for protein mutations into an easily understandable movie format. MutAmore provides a streamlined pipeline casting single amino-acid variations (SAVs) into a dynamic 3D protein mutation movie which provides a qualitative perspective on the mutational landscape of a protein. By default, the tool first generates all possible variants of the protein sequence reachable through a SAVs (L*19 for proteins with L residues). Next, it predicts the structural conformation for all L*19 variants using state-of-the-art models. Finally, it visualizes the mutation matrix, and produces a color-coded 3D animation. MutAmore samples alternative protein configurations to study the dynamical space accessible from SAVs in the post-AlphaFold2 era of structural biology.\nGraffiTE: a Unified Framework to Analyze Transposable Element Insertion Polymorphisms using Genome-graphs\nAuthors: Groza, C.; Chen, X.; Wheeler, T. J.; Bourque, G.; Goubert, C.\nScore: 48.4, Published: 2023-09-15 DOI: 10.1101/2023.09.11.557209\nTransposable Elements (TEs) are abundant and mobile repetitive DNA sequences evolving within and across their hosts genomes. Active TEs cause insertion polymorphism and contribute to genomic diversity. Here, we present GraffiTE, a flexible and comprehensive pipeline for detecting and genotyping polymorphic mobile elements (pMEs). By integrating state-of-the-art SV detection algorithms and graph-genome frameworks, GraffiTE enables the accurate identification of pMEs from genomic assemblies and long-read as well as the precise genotyping of these variants using short-or long-read data. Performance evaluations using simulated and benchmark datasets demonstrate high precision and recall rates. Notably, we demonstrate the versatility of GraffiTE by analyzing the human reference pangenome, 30 Drosophila melanogaster genomes, and multiple cultivars of the emerging crop model Cannabis sativa, where pMEs are undocumented. These analyses reveal the landscapes of pMEs and their frequency variations across individuals, strains, and cultivars. GraffiTE provides a user-friendly interface, allowing non-expert users to perform comprehensive pME analyses, including in models with limited TE prior knowledge. The pipelines extensible design and compatibility with various sequencing technologies make it a valuable integrative framework for studying TE dynamics and their impact on genome evolution. GraffiTE is freely available at https://github.com/cgroza/GraffiTE.\nNuFold: A Novel Tertiary RNA Structure Prediction Method Using Deep Learning with Flexible Nucleobase Center Representation\nAuthors: Kagaya, Y.; Zhang, Z.; Ibtehaz, N.; Wang, X.; Nakamura, T.; Huang, D.; Kihara, D.\nScore: 25.1, Published: 2023-09-22 DOI: 10.1101/2023.09.20.558715\nRNA is not only playing a core role in the central dogma as mRNA between DNA and protein, but also many non-coding RNAs have been discovered to have unique and diverse biological functions. As genome sequences become increasingly available and our knowledge of RNA sequences grows, the study of RNA's structure and function has become more demanding. However, experimental determination of three-dimensional RNA structures is both costly and time-consuming, resulting in a substantial disparity between RNA sequence data and structural insights. In response to this challenge, we propose a novel computational approach that harnesses state-of-the-art deep learning architecture NuFold to accurately predict RNA tertiary structures. This approach aims to offer a cost-effective and efficient means of bridging the gap between RNA sequence information and structural comprehension. NuFold implements a nucleobase center representation, which allows it to reproduce all possible nucleotide conformations accurately.\nAssessing the impact of transcriptomics data analysis pipelines on downstream functional enrichment results\nAuthors: Paton, V.; Gabor, A.; Ramirez Flores, R. O.; Badia-i-Mompel, P.; Tanevski, J.; Garrido-Rodriguez, M.; Saez-Rodriguez, J.\nScore: 24.9, Published: 2023-09-16 DOI: 10.1101/2023.09.13.557538\nTranscriptomics, and in particular RNA-Seq, has become a widely used approach to assess the molecular state of biological systems. To facilitate its analysis, many tools have been developed for different steps, such as filtering lowly expressed genes, normalisation, differential expression, and enrichment. While numerous studies have examined the impact of method choices on differential expression results, little attention has been paid to their effects on further downstream functional analysis using enrichment of gene sets, such as pathways, which typically provides the basis for interpretation and follow-up experiments. To address this gap, we introduce FLOP (FunctionaL Omics Processing), a comprehensive nextflow-based workflow that combines various methods for preprocessing and downstream enrichment analysis, allowing users to perform end-to-end analyses of count level transcriptomic data. We illustrate FLOP capabilities on diverse datasets comprising samples from end-stage heart failure patients and cancer cell lines in both basal and drug-perturbed states. We found that the correlation between gene set enrichment analysis results can vary significantly for alternative pipelines. Additionally, we observed that not filtering the data had the highest impact on the correlation between pipelines in the gene set space, especially in settings with limited statistical power. Overall, our results underscore the impact of carefully evaluating the consequences of the choice of preprocessing methods on downstream enrichment analyses. We envision FLOP as a valuable tool to measure the robustness of functional analyses, ultimately leading to more reliable and conclusive biological findings. Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=45 SRC=\"FIGDIR/small/557538v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (14K): org.highwire.dtl.DTLVardef@1b35ee7org.highwire.dtl.DTLVardef@11bd02aorg.highwire.dtl.DTLVardef@ff7b18org.highwire.dtl.DTLVardef@ea8d2c_HPS_FORMAT_FIGEXP M_FIG C_FIG\nPangenome databases provide superior host removal and mycobacteria classification from clinical metagenomic data\nAuthors: Hall, M. B.; Coin, L. J. M.\nScore: 22.9, Published: 2023-09-19 DOI: 10.1101/2023.09.18.558339\nBackgroundCulture-free real-time sequencing of clinical metagenomic samples promises both rapid pathogen detection and antimicrobial resistance profiling. However, this approach introduces the risk of patient DNA leakage. To mitigate this risk, we need near-comprehensive removal of human DNA sequence at the point of sequencing, typically involving use of resource-constrained devices. Existing benchmarks have largely focused on use of standardised databases and largely ignored the computational requirements of depletion pipelines as well as the impact of human genome diversity. ResultsWe benchmarked host removal pipelines on simulated Illumina and Nanopore metagenomic samples. We found that construction of a custom kraken database containing diverse human genomes results in the best balance of accuracy and computational resource usage. In addition, we benchmarked pipelines using kraken and minimap2 for taxonomic classification of Mycobacterium reads using standard and custom databases. With a database representative of the Mycobacterium genus, both tools obtained near-perfect precision and recall for classification of Mycobacterium tuberculosis. Computational efficiency of these custom databases was again superior to most standard approaches, allowing them to be executed on a laptop device. ConclusionsNanopore sequencing and a custom kraken human database with a diversity of genomes leads to superior host read removal from simulated metagenomic samples while being executable on a laptop. In addition, constructing a taxon-specific database provides excellent taxonomic read assignment while keeping runtime and memory low. We make all customised databases and pipelines freely available.\nMeasuring, visualizing and diagnosing reference bias with biastools\nAuthors: Lin, M.-J.; Iyer, S.; Chen, N.-C.; Langmead, B.\nScore: 21.2, Published: 2023-09-16 DOI: 10.1101/2023.09.13.557552\nA goal of recent alignment methods is to reduce reference bias, which occurs when reads containing non-reference alleles fail to align to their true point of origin. However, there is a lack of methods for systematically measuring, categorizing, and diagnosing reference bias. We present biastools, which analyzes and categorizes instances of reference bias. Biastools has different sets of functionality tailored to different scenarios, i.e. (a) when the donor genome is well-characterized and input reads are simulated, (b) when the donor is well-characterized and reads are real, and (c) when the donor is not well-characterized and reads are real. When possible, biastools divides instances of reference bias into categories according to their cause: bias due to loss, flux, or local misalignment. Biastoolss scan mode detects large-scale mapping artifacts due to structural variation and flaws in the reference representation. Our findings confirm that including more variants in a graph genome alignment method results in fewer reference biases. We also find that end-to-end alignment modes are effective in reducing bias at insertions and deletions, compared to local aligners that allow soft clipping. Finally, we use biastools to characterize the ways in which using the new telomere-to-telomere human reference can improve bias at a large scale. In short, biastools is a tool uniquely focused on reference bias, making it a valuable resource as the field continues to develop new aligners and pangenome representations to reduce bias.\nTargeted decontamination of sequencing data with CLEAN\nAuthors: Lataretu, M.; Krautwurst, S.; Viehweger, A.; Brandt, C.; Hölzer, M.\nScore: 19.8, Published: 2023-09-16 DOI: 10.1101/2023.08.05.552089\nBackgroundMany biological and medical questions are answered based on the analysis of sequence data. However, we can find contaminations, artificial spike-ins, and overrepresented rRNA sequences in various read collections and assemblies; complicating data analysis and making interpretation difficult. In particular, spike-ins used as controls, such as those known from Illumina (PhiX phage) or Nanopore data (DNA CS lambda phage, yeast enolase ENO2), are often not considered as contaminants and also not appropriately removed during bioinformatics analyses. FindingsTo address this, we developed CLEAN, a pipeline to remove unwanted sequence data from both long and short read sequencing techniques from a wide range of use cases. While focusing on Illumina and Nanopore data and removing of their technology-specific control sequences, the pipeline can also be used for everyday tasks, such as host decontamination of metagenomic reads and assemblies, or the removal of rRNA from RNA-Seq data. The results are the purified sequences and the sequences identified as contaminated with statistics summarized in an HTML report. ConclusionsThe decontaminated output files can be used directly in subsequent analyses, resulting in faster computations and improved results. Although decontamination is a task that seems mundane, many contaminants are routinely overlooked, cleaned by steps that are not fully reproducible or difficult to trace by the user. CLEAN will facilitate reproducible, platform-independent data analysis in genomics and transcriptomics and is freely available at https://github.com/hoelzer/clean under a BSD3 license.\nDeep learning predictions of TCR-epitope interactions reveal epitope-specific chains in dual alpha T cells\nAuthors: Croce, G.; Bobisse, S.; Moreno, D. L.; Schmidt, J.; Guillame, P.; Harari, A.; Gfeller, D.\nScore: 17.4, Published: 2023-09-16 DOI: 10.1101/2023.09.13.557561\nT cells have the ability to eliminate infected and cancer cells and play an essential role in cancer immunotherapy. T-cell activation is elicited by the binding of the T-cell receptor (TCR) to epitopes displayed on MHC molecules, and the TCR specificity is determined by the sequence of its and {beta} chains. Here, we collected and curated a dataset of 17,715 {beta}TCRs interacting with dozens of class I and class II epitopes. We used this curated data to develop MixTCRpred, a deep learning TCR-epitope interaction predictor. MixTCRpred accurately predicts TCRs recognizing several viral and cancer epitopes. MixTCRpred further provides a useful quality control tool for multiplexed single-cell TCR sequencing assays of epitope-specific T cells and pinpoints a substantial fraction of putative contaminants in public databases. Analysis of epitope-specific dual T cells demonstrates that MixTCRpred can identify chains mediating epitope recognition. Applying MixTCRpred to TCR repertoires from COVID-19 patients reveals enrichment of clonotypes predicted to bind an immunodominant SARS-CoV-2 epitope. Overall, MixTCRpred provides a robust tool to predict TCRs interacting with specific epitopes and interpret TCR-sequencing data from both bulk and epitope-specific T cells.\nThe tidyomics ecosystem: Enhancing omic data analyses\nAuthors: Hutchison, W. J.; Keyes, T. J.; Crowell, H. L.; Soneson, C.; Mu, W.; Park, J.-E.; Davis, E. S.; Nahid, A. A.; Tang, M.; Yuan, V.; Axisa, P.-P.; Kitt, J. W.; Poon, C.-L.; Sato, N.; Gottardo, R.; Morgan, M.; Lee, S.; Lawrence, M.; Hicks, S. C.; Nolan, G. P.; Davis, K. L.; Papenfuss, A. T.; Love, M. I.; Mangiola, S.\nScore: 132.7, Published: 2023-09-13 DOI: 10.1101/2023.09.10.557072\nThe exponential growth of omic data presents challenges in data manipulation, analysis, and integration. Addressing these challenges, Bioconductor offers an extensive data analysis platform and community, while R tidy programming offers a standard data organisation and manipulation that has revolutionised data science. Bioconductor and tidy R have mostly remained independent; bridging these two ecosystems would streamline omic analysis, ease learning and encourage cross-disciplinary collaborations. Here, we introduce the tidyomics ecosystem--a suite of interoperable software that brings the vast tidy software ecosystem to omic data analysis.\nPDB ProtVista: A reusable and open-source sequence feature viewer\nAuthors: Deshpande, M.; Varadi, M.; Paysan-Lafosse, T.; Nair, S.; Piovesan, D.; Mir, S.; Gutmanas, A.; Tosatto, S. C. E.; Velankar, S.\nScore: 28.7, Published: 2023-09-20 DOI: 10.1101/2022.07.22.500790\nThe PDB ProtVista is a reusable and customisable sequence feature viewer that provides intuitive and detailed 2D visualisation of residue-level annotations while supporting interactive communication with 3D viewers. The Protein Data Bank in Europe (PDBe) team develops and maintains PDB ProtVista. Several public web services use it to display structural and functional annotations such as macromolecular interaction interfaces, intrinsic disorder predictions, sequence variants, and sequence conservation. The PDB ProtVista is freely available from https://github.com/PDBeurope/protvista-pdb. We provide extensive documentation and step-by-step user guides on integrating PDB ProtVista with existing web applications and 3D molecular viewers. We also offer examples of displaying the users custom data and functional annotations for PDB and UniProt entries powered by a rich set of PDBe API endpoints.\n",
  "wordCount" : "2126",
  "inLanguage": "en",
  "datePublished": "2023-09-24T10:37:06Z",
  "dateModified": "2023-09-24T10:37:06Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on September 24, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.15.557870">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.15.557870" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.15.557870">
        <p class="paperTitle">Rendering protein mutation movies with MutAmore</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.15.557870" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.15.557870" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Weissenow, K.; Rost, B.</p>
        <p class="info">Score: 39.6, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.15.557870' target='https://doi.org/10.1101/2023.09.15.557870'> 10.1101/2023.09.15.557870</a></p>
        <p class="abstract">We introduce MutAmore, a bioinformatics tool that renders individual photos of, e.g., protein three-dimensional (3D) structures for protein mutations into an easily understandable movie format. MutAmore provides a streamlined pipeline casting single amino-acid variations (SAVs) into a dynamic 3D protein mutation movie which provides a qualitative perspective on the mutational landscape of a protein. By default, the tool first generates all possible variants of the protein sequence reachable through a SAVs (L*19 for proteins with L residues). Next, it predicts the structural conformation for all L*19 variants using state-of-the-art models. Finally, it visualizes the mutation matrix, and produces a color-coded 3D animation. MutAmore samples alternative protein configurations to study the dynamical space accessible from SAVs in the post-AlphaFold2 era of structural biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557209">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557209" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557209">
        <p class="paperTitle">GraffiTE: a Unified Framework to Analyze Transposable Element Insertion Polymorphisms using Genome-graphs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557209" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557209" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Groza, C.; Chen, X.; Wheeler, T. J.; Bourque, G.; Goubert, C.</p>
        <p class="info">Score: 48.4, Published: 2023-09-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557209' target='https://doi.org/10.1101/2023.09.11.557209'> 10.1101/2023.09.11.557209</a></p>
        <p class="abstract">Transposable Elements (TEs) are abundant and mobile repetitive DNA sequences evolving within and across their hosts genomes. Active TEs cause insertion polymorphism and contribute to genomic diversity. Here, we present GraffiTE, a flexible and comprehensive pipeline for detecting and genotyping polymorphic mobile elements (pMEs). By integrating state-of-the-art SV detection algorithms and graph-genome frameworks, GraffiTE enables the accurate identification of pMEs from genomic assemblies and long-read as well as the precise genotyping of these variants using short-or long-read data. Performance evaluations using simulated and benchmark datasets demonstrate high precision and recall rates. Notably, we demonstrate the versatility of GraffiTE by analyzing the human reference pangenome, 30 Drosophila melanogaster genomes, and multiple cultivars of the emerging crop model Cannabis sativa, where pMEs are undocumented. These analyses reveal the landscapes of pMEs and their frequency variations across individuals, strains, and cultivars. GraffiTE provides a user-friendly interface, allowing non-expert users to perform comprehensive pME analyses, including in models with limited TE prior knowledge. The pipelines extensible design and compatibility with various sequencing technologies make it a valuable integrative framework for studying TE dynamics and their impact on genome evolution. GraffiTE is freely available at https://github.com/cgroza/GraffiTE.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.558715">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.558715" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.558715">
        <p class="paperTitle">NuFold: A Novel Tertiary RNA Structure Prediction Method Using Deep Learning with Flexible Nucleobase Center Representation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.558715" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.558715" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kagaya, Y.; Zhang, Z.; Ibtehaz, N.; Wang, X.; Nakamura, T.; Huang, D.; Kihara, D.</p>
        <p class="info">Score: 25.1, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.558715' target='https://doi.org/10.1101/2023.09.20.558715'> 10.1101/2023.09.20.558715</a></p>
        <p class="abstract">RNA is not only playing a core role in the central dogma as mRNA between DNA and protein, but also many non-coding RNAs have been discovered to have unique and diverse biological functions. As genome sequences become increasingly available and our knowledge of RNA sequences grows, the study of RNA&#39;s structure and function has become more demanding. However, experimental determination of three-dimensional RNA structures is both costly and time-consuming, resulting in a substantial disparity between RNA sequence data and structural insights. In response to this challenge, we propose a novel computational approach that harnesses state-of-the-art deep learning architecture NuFold to accurately predict RNA tertiary structures. This approach aims to offer a cost-effective and efficient means of bridging the gap between RNA sequence information and structural comprehension. NuFold implements a nucleobase center representation, which allows it to reproduce all possible nucleotide conformations accurately.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.13.557538">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.13.557538" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.13.557538">
        <p class="paperTitle">Assessing the impact of transcriptomics data analysis pipelines on downstream functional enrichment results</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.13.557538" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.13.557538" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Paton, V.; Gabor, A.; Ramirez Flores, R. O.; Badia-i-Mompel, P.; Tanevski, J.; Garrido-Rodriguez, M.; Saez-Rodriguez, J.</p>
        <p class="info">Score: 24.9, Published: 2023-09-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.13.557538' target='https://doi.org/10.1101/2023.09.13.557538'> 10.1101/2023.09.13.557538</a></p>
        <p class="abstract">Transcriptomics, and in particular RNA-Seq, has become a widely used approach to assess the molecular state of biological systems. To facilitate its analysis, many tools have been developed for different steps, such as filtering lowly expressed genes, normalisation, differential expression, and enrichment. While numerous studies have examined the impact of method choices on differential expression results, little attention has been paid to their effects on further downstream functional analysis using enrichment of gene sets, such as pathways, which typically provides the basis for interpretation and follow-up experiments. To address this gap, we introduce FLOP (FunctionaL Omics Processing), a comprehensive nextflow-based workflow that combines various methods for preprocessing and downstream enrichment analysis, allowing users to perform end-to-end analyses of count level transcriptomic data. We illustrate FLOP capabilities on diverse datasets comprising samples from end-stage heart failure patients and cancer cell lines in both basal and drug-perturbed states. We found that the correlation between gene set enrichment analysis results can vary significantly for alternative pipelines. Additionally, we observed that not filtering the data had the highest impact on the correlation between pipelines in the gene set space, especially in settings with limited statistical power. Overall, our results underscore the impact of carefully evaluating the consequences of the choice of preprocessing methods on downstream enrichment analyses. We envision FLOP as a valuable tool to measure the robustness of functional analyses, ultimately leading to more reliable and conclusive biological findings.

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=45 SRC=&#34;FIGDIR/small/557538v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (14K):
org.highwire.dtl.DTLVardef@1b35ee7org.highwire.dtl.DTLVardef@11bd02aorg.highwire.dtl.DTLVardef@ff7b18org.highwire.dtl.DTLVardef@ea8d2c_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558339">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558339" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558339">
        <p class="paperTitle">Pangenome databases provide superior host removal and mycobacteria classification from clinical metagenomic data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558339" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558339" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hall, M. B.; Coin, L. J. M.</p>
        <p class="info">Score: 22.9, Published: 2023-09-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558339' target='https://doi.org/10.1101/2023.09.18.558339'> 10.1101/2023.09.18.558339</a></p>
        <p class="abstract">BackgroundCulture-free real-time sequencing of clinical metagenomic samples promises both rapid pathogen detection and antimicrobial resistance profiling. However, this approach introduces the risk of patient DNA leakage. To mitigate this risk, we need near-comprehensive removal of human DNA sequence at the point of sequencing, typically involving use of resource-constrained devices. Existing benchmarks have largely focused on use of standardised databases and largely ignored the computational requirements of depletion pipelines as well as the impact of human genome diversity.

ResultsWe benchmarked host removal pipelines on simulated Illumina and Nanopore metagenomic samples. We found that construction of a custom kraken database containing diverse human genomes results in the best balance of accuracy and computational resource usage. In addition, we benchmarked pipelines using kraken and minimap2 for taxonomic classification of Mycobacterium reads using standard and custom databases. With a database representative of the Mycobacterium genus, both tools obtained near-perfect precision and recall for classification of Mycobacterium tuberculosis. Computational efficiency of these custom databases was again superior to most standard approaches, allowing them to be executed on a laptop device.

ConclusionsNanopore sequencing and a custom kraken human database with a diversity of genomes leads to superior host read removal from simulated metagenomic samples while being executable on a laptop. In addition, constructing a taxon-specific database provides excellent taxonomic read assignment while keeping runtime and memory low. We make all customised databases and pipelines freely available.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.13.557552">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.13.557552" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.13.557552">
        <p class="paperTitle">Measuring, visualizing and diagnosing reference bias with biastools</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.13.557552" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.13.557552" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lin, M.-J.; Iyer, S.; Chen, N.-C.; Langmead, B.</p>
        <p class="info">Score: 21.2, Published: 2023-09-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.13.557552' target='https://doi.org/10.1101/2023.09.13.557552'> 10.1101/2023.09.13.557552</a></p>
        <p class="abstract">A goal of recent alignment methods is to reduce reference bias, which occurs when reads containing non-reference alleles fail to align to their true point of origin. However, there is a lack of methods for systematically measuring, categorizing, and diagnosing reference bias. We present biastools, which analyzes and categorizes instances of reference bias. Biastools has different sets of functionality tailored to different scenarios, i.e. (a) when the donor genome is well-characterized and input reads are simulated, (b) when the donor is well-characterized and reads are real, and (c) when the donor is not well-characterized and reads are real. When possible, biastools divides instances of reference bias into categories according to their cause: bias due to loss, flux, or local misalignment. Biastoolss scan mode detects large-scale mapping artifacts due to structural variation and flaws in the reference representation. Our findings confirm that including more variants in a graph genome alignment method results in fewer reference biases. We also find that end-to-end alignment modes are effective in reducing bias at insertions and deletions, compared to local aligners that allow soft clipping. Finally, we use biastools to characterize the ways in which using the new telomere-to-telomere human reference can improve bias at a large scale. In short, biastools is a tool uniquely focused on reference bias, making it a valuable resource as the field continues to develop new aligners and pangenome representations to reduce bias.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.05.552089">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.05.552089" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.05.552089">
        <p class="paperTitle">Targeted decontamination of sequencing data with CLEAN</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.05.552089" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.05.552089" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lataretu, M.; Krautwurst, S.; Viehweger, A.; Brandt, C.; Hölzer, M.</p>
        <p class="info">Score: 19.8, Published: 2023-09-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.05.552089' target='https://doi.org/10.1101/2023.08.05.552089'> 10.1101/2023.08.05.552089</a></p>
        <p class="abstract">BackgroundMany biological and medical questions are answered based on the analysis of sequence data. However, we can find contaminations, artificial spike-ins, and overrepresented rRNA sequences in various read collections and assemblies; complicating data analysis and making interpretation difficult. In particular, spike-ins used as controls, such as those known from Illumina (PhiX phage) or Nanopore data (DNA CS lambda phage, yeast enolase ENO2), are often not considered as contaminants and also not appropriately removed during bioinformatics analyses.

FindingsTo address this, we developed CLEAN, a pipeline to remove unwanted sequence data from both long and short read sequencing techniques from a wide range of use cases. While focusing on Illumina and Nanopore data and removing of their technology-specific control sequences, the pipeline can also be used for everyday tasks, such as host decontamination of metagenomic reads and assemblies, or the removal of rRNA from RNA-Seq data. The results are the purified sequences and the sequences identified as contaminated with statistics summarized in an HTML report.

ConclusionsThe decontaminated output files can be used directly in subsequent analyses, resulting in faster computations and improved results. Although decontamination is a task that seems mundane, many contaminants are routinely overlooked, cleaned by steps that are not fully reproducible or difficult to trace by the user. CLEAN will facilitate reproducible, platform-independent data analysis in genomics and transcriptomics and is freely available at https://github.com/hoelzer/clean under a BSD3 license.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.13.557561">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.13.557561" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.13.557561">
        <p class="paperTitle">Deep learning predictions of TCR-epitope interactions reveal epitope-specific chains in dual alpha T cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.13.557561" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.13.557561" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Croce, G.; Bobisse, S.; Moreno, D. L.; Schmidt, J.; Guillame, P.; Harari, A.; Gfeller, D.</p>
        <p class="info">Score: 17.4, Published: 2023-09-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.13.557561' target='https://doi.org/10.1101/2023.09.13.557561'> 10.1101/2023.09.13.557561</a></p>
        <p class="abstract">T cells have the ability to eliminate infected and cancer cells and play an essential role in cancer immunotherapy. T-cell activation is elicited by the binding of the T-cell receptor (TCR) to epitopes displayed on MHC molecules, and the TCR specificity is determined by the sequence of its  and {beta} chains. Here, we collected and curated a dataset of 17,715 {beta}TCRs interacting with dozens of class I and class II epitopes. We used this curated data to develop MixTCRpred, a deep learning TCR-epitope interaction predictor. MixTCRpred accurately predicts TCRs recognizing several viral and cancer epitopes. MixTCRpred further provides a useful quality control tool for multiplexed single-cell TCR sequencing assays of epitope-specific T cells and pinpoints a substantial fraction of putative contaminants in public databases. Analysis of epitope-specific dual  T cells demonstrates that MixTCRpred can identify  chains mediating epitope recognition. Applying MixTCRpred to TCR repertoires from COVID-19 patients reveals enrichment of clonotypes predicted to bind an immunodominant SARS-CoV-2 epitope. Overall, MixTCRpred provides a robust tool to predict TCRs interacting with specific epitopes and interpret TCR-sequencing data from both bulk and epitope-specific T cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.10.557072">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.10.557072" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.10.557072">
        <p class="paperTitle">The tidyomics ecosystem: Enhancing omic data analyses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.10.557072" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.10.557072" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hutchison, W. J.; Keyes, T. J.; Crowell, H. L.; Soneson, C.; Mu, W.; Park, J.-E.; Davis, E. S.; Nahid, A. A.; Tang, M.; Yuan, V.; Axisa, P.-P.; Kitt, J. W.; Poon, C.-L.; Sato, N.; Gottardo, R.; Morgan, M.; Lee, S.; Lawrence, M.; Hicks, S. C.; Nolan, G. P.; Davis, K. L.; Papenfuss, A. T.; Love, M. I.; Mangiola, S.</p>
        <p class="info">Score: 132.7, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.10.557072' target='https://doi.org/10.1101/2023.09.10.557072'> 10.1101/2023.09.10.557072</a></p>
        <p class="abstract">The exponential growth of omic data presents challenges in data manipulation, analysis, and integration. Addressing these challenges, Bioconductor offers an extensive data analysis platform and community, while R tidy programming offers a standard data organisation and manipulation that has revolutionised data science. Bioconductor and tidy R have mostly remained independent; bridging these two ecosystems would streamline omic analysis, ease learning and encourage cross-disciplinary collaborations. Here, we introduce the tidyomics ecosystem--a suite of interoperable software that brings the vast tidy software ecosystem to omic data analysis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.07.22.500790">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.07.22.500790" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.07.22.500790">
        <p class="paperTitle">PDB ProtVista: A reusable and open-source sequence feature viewer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.07.22.500790" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.07.22.500790" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Deshpande, M.; Varadi, M.; Paysan-Lafosse, T.; Nair, S.; Piovesan, D.; Mir, S.; Gutmanas, A.; Tosatto, S. C. E.; Velankar, S.</p>
        <p class="info">Score: 28.7, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.07.22.500790' target='https://doi.org/10.1101/2022.07.22.500790'> 10.1101/2022.07.22.500790</a></p>
        <p class="abstract">The PDB ProtVista is a reusable and customisable sequence feature viewer that provides intuitive and detailed 2D visualisation of residue-level annotations while supporting interactive communication with 3D viewers. The Protein Data Bank in Europe (PDBe) team develops and maintains PDB ProtVista. Several public web services use it to display structural and functional annotations such as macromolecular interaction interfaces, intrinsic disorder predictions, sequence variants, and sequence conservation. The PDB ProtVista is freely available from https://github.com/PDBeurope/protvista-pdb. We provide extensive documentation and step-by-step user guides on integrating PDB ProtVista with existing web applications and 3D molecular viewers. We also offer examples of displaying the users custom data and functional annotations for PDB and UniProt entries powered by a rich set of PDBe API endpoints.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
