<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Context-Dependent Design of Induced-fit Enzymes using Deep Learning Generates Well Expressed, Thermally Stable and Active Enzymes
Authors: Zimmerman, L.; Alon, N.; Levin, I.; Koganitsky, A.; Shpigel, N.; Brestel, C.; Lapidoth, G. D.
Score: 299.3, Published: 2023-08-02 DOI: 10.1101/2023.07.27.550799
The potential of engineered enzymes in practical applications is often constrained by limitations in their expression levels, thermal stability, and the diversity and magnitude of catalytic activities. De-novo enzyme design, though exciting, is challenged by the complex nature of enzymatic catalysis.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="Context-Dependent Design of Induced-fit Enzymes using Deep Learning Generates Well Expressed, Thermally Stable and Active Enzymes
Authors: Zimmerman, L.; Alon, N.; Levin, I.; Koganitsky, A.; Shpigel, N.; Brestel, C.; Lapidoth, G. D.
Score: 299.3, Published: 2023-08-02 DOI: 10.1101/2023.07.27.550799
The potential of engineered enzymes in practical applications is often constrained by limitations in their expression levels, thermal stability, and the diversity and magnitude of catalytic activities. De-novo enzyme design, though exciting, is challenged by the complex nature of enzymatic catalysis." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-27T10:37:02+00:00" />
<meta property="article:modified_time" content="2023-08-27T10:37:02+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="Context-Dependent Design of Induced-fit Enzymes using Deep Learning Generates Well Expressed, Thermally Stable and Active Enzymes
Authors: Zimmerman, L.; Alon, N.; Levin, I.; Koganitsky, A.; Shpigel, N.; Brestel, C.; Lapidoth, G. D.
Score: 299.3, Published: 2023-08-02 DOI: 10.1101/2023.07.27.550799
The potential of engineered enzymes in practical applications is often constrained by limitations in their expression levels, thermal stability, and the diversity and magnitude of catalytic activities. De-novo enzyme design, though exciting, is challenged by the complex nature of enzymatic catalysis."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "Context-Dependent Design of Induced-fit Enzymes using Deep Learning Generates Well Expressed, Thermally Stable and Active Enzymes\nAuthors: Zimmerman, L.; Alon, N.; Levin, I.; Koganitsky, A.; Shpigel, N.; Brestel, C.; Lapidoth, G. D.\nScore: 299.3, Published: 2023-08-02 DOI: 10.1101/2023.07.27.550799\nThe potential of engineered enzymes in practical applications is often constrained by limitations in their expression levels, thermal stability, and the diversity and magnitude of catalytic activities. De-novo enzyme design, though exciting, is challenged by the complex nature of enzymatic catalysis.",
  "keywords": [
    
  ],
  "articleBody": " Context-Dependent Design of Induced-fit Enzymes using Deep Learning Generates Well Expressed, Thermally Stable and Active Enzymes\nAuthors: Zimmerman, L.; Alon, N.; Levin, I.; Koganitsky, A.; Shpigel, N.; Brestel, C.; Lapidoth, G. D.\nScore: 299.3, Published: 2023-08-02 DOI: 10.1101/2023.07.27.550799\nThe potential of engineered enzymes in practical applications is often constrained by limitations in their expression levels, thermal stability, and the diversity and magnitude of catalytic activities. De-novo enzyme design, though exciting, is challenged by the complex nature of enzymatic catalysis. An alternative promising approach involves expanding the capabilities of existing natural enzymes to enable functionality across new substrates and operational parameters. To this end we introduce CoSaNN (Conformation Sampling using Neural Network), a novel strategy for enzyme design that utilizes advances in deep learning for structure prediction and sequence optimization. By controlling enzyme conformations, we can expand the chemical space beyond the reach of simple mutagenesis. CoSaNN uses a context-dependent approach that accurately generates novel enzyme designs by considering non-linear relationships in both sequence and structure space. Additionally, we have further developed SolvIT, a graph neural network trained to predict protein solubility in E.Coli, as an additional optimization layer for producing highly expressed enzymes. Through this approach, we have engineered novel enzymes exhibiting superior expression levels, with 54% of our designs expressed in E.Coli, and increased thermal stability with more than 30% of our designs having a higher Tm than the template enzyme. Furthermore, our research underscores the transformative potential of AI in protein design, adeptly capturing high order interactions and preserving allosteric mechanisms in extensively modified enzymes. These advancements pave the way for the creation of diverse, functional, and robust enzymes, thereby opening new avenues for targeted biotechnological applications.\nVoyager: exploratory single-cell genomics data analysis with geospatial statistics\nAuthors: Moses, L.; Einarsson, P. H.; Jackson, K. C.; Luebbert, L.; Booeshaghi, A. S.; Antonsson, S. E.; Bray, N.; Melsted, P.; Pachter, L.\nScore: 128.6, Published: 2023-08-20 DOI: 10.1101/2023.07.20.549945\nExploratory spatial data analysis (ESDA) can be a powerful approach to understanding single-cell genomics datasets, but it is not yet part of standard data analysis workflows. In particular, geospatial analyses, which have been developed and refined for decades, have yet to be fully adapted and applied to spatial single-cell analysis. We introduce the Voyager platform, which systematically brings the geospatial ESDA tradition to (spatial) -omics, with local, bivariate, and multivariate spatial methods not yet commonly applied to spatial -omics, united by a uniform user interface. Using Voyager, we showcase biological insights that can be derived with its methods, such as biologically relevant negative spatial autocorrelation. Underlying Voyager is the SpatialFeatureExperiment data structure, which combines Simple Feature with SingleCellExperiment and AnnData to represent and operate on geometries bundled with gene expression data. Voyager has comprehensive tutorials demonstrating ESDA built on GitHub Actions to ensure reproducibility and scalability, using data from popular commercial technologies. Voyager is implemented in both R/Bioconductor and Python/PyPI, and features compatibility tests to ensure that both implementations return consistent results.\nDesign of Cyclic Peptides Targeting Protein-Protein Interactions using AlphaFold\nAuthors: Kosugi, T.; Ohue, M.\nScore: 35.7, Published: 2023-08-21 DOI: 10.1101/2023.08.20.554056\nMore than 930,000 protein-protein interactions (PPIs) have been identified in recent years, but their physicochemical properties differ from conventional drug targets, complicating the use of conventional small molecules as modalities. Cyclic peptides are a promising modality for targeting protein-protein interactions (PPIs), but it is difficult to predict the structure of a target protein-cyclic peptide complex or to design a cyclic peptide sequence that binds to the target protein using computational methods. Recently, AlphaFold with a cyclic offset has enabled predicting the structure of cyclic peptides, thereby enabling de novo cyclic peptide designs. We developed a cyclic peptide complex offset to enable the structural prediction of target proteins and cyclic peptide complexes and found AlphaFold2 with a cyclic peptide complex offset can predict structures with high accuracy. We also applied the cyclic peptide complex offset to the binder hallucination protocol of AfDesign, a de novo protein design method using AlphaFold, and we could design a high predicted local-distance difference test and lower separated binding energy per unit interface area than the native MDM2/p53 structure. Furthermore, the method was applied to 12 other protein-peptide complexes and one protein-protein complex. Our approach shows that it is possible to design putative cyclic peptide sequences targeting PPI.\nSegment Anything for Microscopy\nAuthors: Archit, A.; Nair, S.; Khalid, N.; Hilt, P.; Rajashekar, V.; Freitag, M.; Gupta, S.; Dengel, A.; Ahmed, S.; Pape, C.\nScore: 34.0, Published: 2023-08-26 DOI: 10.1101/2023.08.21.554208\nWe present Segment Anything for Microscopy, a tool for interactive and automatic segmentation and tracking of objects in multi-dimensional microscopy data. Our method is based on Segment Anything, a vision foundation model for image segmentation. We extend it by training specialized models for microscopy data that significantly improve segmentation quality for a wide range of imaging conditions. We also implement annotation tools for interactive (volumetric) segmentation and tracking, that speed up data annotation significantly compared to established tools. Our work constitutes the first application of vision foundation models to microscopy, laying the groundwork for solving image analysis problems in these domains with a small set of powerful deep learning architectures.\nA fast comparative genome browser for diverse bacteria and archaea\nAuthors: Price, M. N.; Arkin, A. P.\nScore: 27.3, Published: 2023-08-24 DOI: 10.1101/2023.08.23.554478\nGenome sequencing has revealed an incredible diversity of bacteria and archaea, but there are no fast and convenient tools for browsing across these genomes. It is cumbersome to view the prevalence of homologs for a protein of interest, or the gene neighborhoods of those homologs, across the diversity of the prokaryotes. We developed a web-based tool, fast.genomics, that uses two strategies to support fast browsing across the diversity of prokaryotes. First, the database of genomes is split up. The main database contains one representative from each of the 6,377 genera that have a high-quality genome, and additional databases for each taxonomic order contain up to 10 representatives of each species. Second, homologs of proteins of interest are identified quickly by using accelerated searches, usually in a few seconds. Once homologs are identified, fast.genomics can quickly show their prevalence across taxa, view their neighboring genes, or compare the prevalence of two different proteins. Fast.genomics is available at https://fast.genomics.lbl.gov.\nAnalyzing microbial evolution through gene and genome phylogenies\nAuthors: Teichman, S.; Lee, M. D.; Willis, A. D.\nScore: 33.5, Published: 2023-08-16 DOI: 10.1101/2023.08.15.553440\nAO_SCPLOWBSTRACTC_SCPLOWMicrobiome scientists critically need modern tools to explore and analyze microbial evolution. Often this involves studying the evolution of microbial genomes as a whole. However, different genes in a single genome can be subject to different evolutionary pressures, which can result in distinct gene-level evolutionary histories. To address this challenge, we propose to treat estimated gene-level phylogenies as data objects, and present an interactive method for the analysis of a collection of gene phylogenies. We use a local linear approximation of phylogenetic tree space to visualize estimated gene trees as points in low-dimensional Euclidean space, and address important practical limitations of existing related approaches, allowing an intuitive visualization of complex data objects. We demonstrate the utility of our proposed approach through microbial data analyses, including by identifying outlying gene histories in strains of Prevotella, and by contrasting Streptococcus phylogenies estimated using different gene sets. Our method is available as an open-source R package, and assists with estimating, visualizing and interacting with a collection of bacterial gene phylogenies. dimension reduction, microbiome, non-Euclidean, statistical genetics, visualization\nSelf-supervised learning for characterising histomorphological diversity and spatial RNA expression prediction across 23 human tissue types\nAuthors: Cisternino, F.; Ometto, S.; Chatterjee, S.; Giacopuzzi, E.; Levine, A. P.; Glastonbury, C. A.\nScore: 21.2, Published: 2023-08-23 DOI: 10.1101/2023.08.22.554251\nAs vast histological archives are digitised, there is a pressing need to be able to associate specific tissue substructures and incident pathology to disease outcomes without arduous annotation. Such automation provides an opportunity to learn fundamental biology about how tissue structure and function varies in a population. Recently, self-supervised learning has proven competitive to supervised machine learning approaches in classification, segmentation and representation learning. Here, we leverage self-supervised learning to generate histology feature representations using 1.7M images across 23 healthy tissues in 838 donors from GTEx. Using these representations, we demonstrate we can automatically segment tissues into their constituent tissue substructures and pathology proportions, and surpass the performance of conventionally used pre-trained models. We observe striking population variability in canonical tissue substructures, highlight examples of missing pathological diagnoses, incorrect assignment of target tissue and cross-tissue contamination. We demonstrate that this variability in tissue composition leads to a likely overestimation of eQTL tissue sharing and drives dramatic differential gene expression changes. We use derived tissue substructures to detect 284 tissue substructures and pathology specific eQTLs. As our derived histology representations are rich morphological descriptors of the underlying tissue, we introduce a multiple instance learning model that can predict and spatially localise individual RNA expression levels directly from histology to specific substructures and pathological features. We validate our RNA spatial predictions with matched ground truth immunohistochemistry (IHC) for several well characterised marker genes, recapitulating their known spatial specificity. Finally, we derive a gene expression spatial enrichment metric, allowing us to detect genes specifically expressed within sites of pathology (e.g. arterial calcification). Together, these results demonstrate the power of self-supervised machine learning when applied to vast histological datasets to allow researchers to pose and answer questions about tissue pathology, its spatial organisation and the interplay between morphological tissue variability and gene expression.\nA multimodal Transformer Network for protein-small molecule interactions enhances drug-target affinity and enzyme-substrate predictions\nAuthors: Kroll, A.; Ranjan, S.; Lercher, M. J.\nScore: 18.6, Published: 2023-08-22 DOI: 10.1101/2023.08.21.554147\nThe activities of most enzymes and drugs rely on interactions between proteins and small molecules. Accurate predictions of these interactions could accelerate pharmaceutical and biotechnological research massively. Machine learning models designed for this task are currently limited by the lack of information exchange between the protein and the small molecule during the creation of the required numerical representations. Here, we introduce ProSmith, a machine learning framework that employs a multimodal Transformer Network to simultaneously process protein amino acid sequences and small molecule strings in the same input. This approach facilitates the exchange of all relevant information between the two molecule types during the calculation of their numerical representations, allowing the model to account for their structural and functional interactions. Our final model combines gradient boosting predictions based on the resulting multimodal Transformer Network with independent predictions based on separate deep learning representations of the proteins and small molecules. The corresponding predictions outperform all previous models for predicting protein-small molecule interactions across three diverse tasks: predicting Michaelis constants KM; inferring potential substrates for enzymes; and predicting protein-drug affinities. The Python code provided can be used to easily implement and improve machine learning predictions involving arbitrary protein-small molecule interactions.\nClairS: a deep-learning method for long-read somatic small variant calling\nAuthors: Zheng, Z.; Su, J.; Chen, L.; Lee, Y.-L.; Lam, T.-W.; Luo, R.\nScore: 15.5, Published: 2023-08-21 DOI: 10.1101/2023.08.17.553778\nIdentifying somatic variants in tumor samples is a crucial task, which is often performed using statistical methods and heuristic filters applied to short-read data. However, with the increasing demand for long-read somatic variant calling, existing methods have fallen short. To address this gap, we present ClairS, the first deep-learning-based, long-read somatic small variant caller. ClairS was trained on massive synthetic somatic variants with diverse coverages and variant allele frequencies (VAF), enabling it to accurately detect a wide range of somatic variants from paired tumor and normal samples. We evaluated ClairS using the latest Nanopore Q20+ HCC1395-HCC1395BL dataset. With 50-fold/25-fold tumor/normal, ClairS achieved a 93.01%/86.86% precision/recall rate for Single Nucleotide Variation (SNVs), and 66.54%/66.89% for somatic insertions and deletions (Indels). Applying ClairS to short-read datasets from multiple sources showed comparable or better performance than Strelka2 and Mutect2. Our findings suggest that improved read phasing enabled by long-read sequencing is key to accurate long-read SNV calling, especially for variants with low VAF. Through experiments across various coverage, purity, and contamination settings, we demonstrated that ClairS is a reliable somatic variant caller. ClairS is open-source at https://github.com/HKU-BAL/ClairS.\nProtein language model embedded geometric graphs power inter-protein contact prediction\nAuthors: Si, Y.; Yan, C.\nScore: 23.9, Published: 2023-08-24 DOI: 10.1101/2023.01.07.523121\nAccurate prediction of contacting residue pairs between interacting proteins is very useful for structural characterization of protein-protein interactions (PPIs). Although significant improvement has been made in inter-protein contact prediction recently, there is still large room for improving the prediction accuracy. Here we present a new deep learning method referred to as PLMGraph-Inter for inter-protein contact prediction. Specifically, we employ rotationally and translationally invariant geometric graphs obtained from structures of interacting proteins to integrate multiple protein language models, which are successively transformed by graph encoders formed by geometric vector perceptrons and residual networks formed by dimensional hybrid residual blocks to predict inter-protein contacts. Extensive evaluation on multiple test sets illustrates that PLMGraph-Inter outperforms five top inter-protein contact prediction methods, including DeepHomo, GLINTER, CDPred, DeepHomo2 and DRN-1D2D_Inter by large margins. In addition, we also show that the prediction of PLMGraph-Inter can complement the result of AlphaFold-Multimer. Finally, we show leveraging the contacts predicted by PLMGraph-Inter as constraints for protein-protein docking can dramatically improve its performance for protein complex structure prediction.\n",
  "wordCount" : "2192",
  "inLanguage": "en",
  "datePublished": "2023-08-27T10:37:02Z",
  "dateModified": "2023-08-27T10:37:02Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on August 27, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550799">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550799" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550799">
        <p class="paperTitle">Context-Dependent Design of Induced-fit Enzymes using Deep Learning Generates Well Expressed, Thermally Stable and Active Enzymes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550799" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550799" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zimmerman, L.; Alon, N.; Levin, I.; Koganitsky, A.; Shpigel, N.; Brestel, C.; Lapidoth, G. D.</p>
        <p class="info">Score: 299.3, Published: 2023-08-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550799' target='https://doi.org/10.1101/2023.07.27.550799'> 10.1101/2023.07.27.550799</a></p>
        <p class="abstract">The potential of engineered enzymes in practical applications is often constrained by limitations in their expression levels, thermal stability, and the diversity and magnitude of catalytic activities. De-novo enzyme design, though exciting, is challenged by the complex nature of enzymatic catalysis. An alternative promising approach involves expanding the capabilities of existing natural enzymes to enable functionality across new substrates and operational parameters. To this end we introduce CoSaNN (Conformation Sampling using Neural Network), a novel strategy for enzyme design that utilizes advances in deep learning for structure prediction and sequence optimization. By controlling enzyme conformations, we can expand the chemical space beyond the reach of simple mutagenesis. CoSaNN uses a context-dependent approach that accurately generates novel enzyme designs by considering non-linear relationships in both sequence and structure space. Additionally, we have further developed SolvIT, a graph neural network trained to predict protein solubility in E.Coli, as an additional optimization layer for producing highly expressed enzymes. Through this approach, we have engineered novel enzymes exhibiting superior expression levels, with 54% of our designs expressed in E.Coli, and increased thermal stability with more than 30% of our designs having a higher Tm than the template enzyme. Furthermore, our research underscores the transformative potential of AI in protein design, adeptly capturing high order interactions and preserving allosteric mechanisms in extensively modified enzymes. These advancements pave the way for the creation of diverse, functional, and robust enzymes, thereby opening new avenues for targeted biotechnological applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549945">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549945" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549945">
        <p class="paperTitle">Voyager: exploratory single-cell genomics data analysis with geospatial statistics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549945" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549945" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moses, L.; Einarsson, P. H.; Jackson, K. C.; Luebbert, L.; Booeshaghi, A. S.; Antonsson, S. E.; Bray, N.; Melsted, P.; Pachter, L.</p>
        <p class="info">Score: 128.6, Published: 2023-08-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549945' target='https://doi.org/10.1101/2023.07.20.549945'> 10.1101/2023.07.20.549945</a></p>
        <p class="abstract">Exploratory spatial data analysis (ESDA) can be a powerful approach to understanding single-cell genomics datasets, but it is not yet part of standard data analysis workflows. In particular, geospatial analyses, which have been developed and refined for decades, have yet to be fully adapted and applied to spatial single-cell analysis. We introduce the Voyager platform, which systematically brings the geospatial ESDA tradition to (spatial) -omics, with local, bivariate, and multivariate spatial methods not yet commonly applied to spatial -omics, united by a uniform user interface. Using Voyager, we showcase biological insights that can be derived with its methods, such as biologically relevant negative spatial autocorrelation. Underlying Voyager is the SpatialFeatureExperiment data structure, which combines Simple Feature with SingleCellExperiment and AnnData to represent and operate on geometries bundled with gene expression data. Voyager has comprehensive tutorials demonstrating ESDA built on GitHub Actions to ensure reproducibility and scalability, using data from popular commercial technologies. Voyager is implemented in both R/Bioconductor and Python/PyPI, and features compatibility tests to ensure that both implementations return consistent results.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.20.554056">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.20.554056" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.20.554056">
        <p class="paperTitle">Design of Cyclic Peptides Targeting Protein-Protein Interactions using AlphaFold</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.20.554056" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.20.554056" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kosugi, T.; Ohue, M.</p>
        <p class="info">Score: 35.7, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.20.554056' target='https://doi.org/10.1101/2023.08.20.554056'> 10.1101/2023.08.20.554056</a></p>
        <p class="abstract">More than 930,000 protein-protein interactions (PPIs) have been identified in recent years, but their physicochemical properties differ from conventional drug targets, complicating the use of conventional small molecules as modalities. Cyclic peptides are a promising modality for targeting protein-protein interactions (PPIs), but it is difficult to predict the structure of a target protein-cyclic peptide complex or to design a cyclic peptide sequence that binds to the target protein using computational methods. Recently, AlphaFold with a cyclic offset has enabled predicting the structure of cyclic peptides, thereby enabling de novo cyclic peptide designs. We developed a cyclic peptide complex offset to enable the structural prediction of target proteins and cyclic peptide complexes and found AlphaFold2 with a cyclic peptide complex offset can predict structures with high accuracy. We also applied the cyclic peptide complex offset to the binder hallucination protocol of AfDesign, a de novo protein design method using AlphaFold, and we could design a high predicted local-distance difference test and lower separated binding energy per unit interface area than the native MDM2/p53 structure. Furthermore, the method was applied to 12 other protein-peptide complexes and one protein-protein complex. Our approach shows that it is possible to design putative cyclic peptide sequences targeting PPI.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554208">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554208" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554208">
        <p class="paperTitle">Segment Anything for Microscopy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554208" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554208" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Archit, A.; Nair, S.; Khalid, N.; Hilt, P.; Rajashekar, V.; Freitag, M.; Gupta, S.; Dengel, A.; Ahmed, S.; Pape, C.</p>
        <p class="info">Score: 34.0, Published: 2023-08-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554208' target='https://doi.org/10.1101/2023.08.21.554208'> 10.1101/2023.08.21.554208</a></p>
        <p class="abstract">We present Segment Anything for Microscopy, a tool for interactive and automatic segmentation and tracking of objects in multi-dimensional microscopy data. Our method is based on Segment Anything, a vision foundation model for image segmentation. We extend it by training specialized models for microscopy data that significantly improve segmentation quality for a wide range of imaging conditions. We also implement annotation tools for interactive (volumetric) segmentation and tracking, that speed up data annotation significantly compared to established tools. Our work constitutes the first application of vision foundation models to microscopy, laying the groundwork for solving image analysis problems in these domains with a small set of powerful deep learning architectures.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554478">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554478" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554478">
        <p class="paperTitle">A fast comparative genome browser for diverse bacteria and archaea</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554478" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554478" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Price, M. N.; Arkin, A. P.</p>
        <p class="info">Score: 27.3, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554478' target='https://doi.org/10.1101/2023.08.23.554478'> 10.1101/2023.08.23.554478</a></p>
        <p class="abstract">Genome sequencing has revealed an incredible diversity of bacteria and archaea, but there are no fast and convenient tools for browsing across these genomes. It is cumbersome to view the prevalence of homologs for a protein of interest, or the gene neighborhoods of those homologs, across the diversity of the prokaryotes. We developed a web-based tool, fast.genomics, that uses two strategies to support fast browsing across the diversity of prokaryotes. First, the database of genomes is split up. The main database contains one representative from each of the 6,377 genera that have a high-quality genome, and additional databases for each taxonomic order contain up to 10 representatives of each species. Second, homologs of proteins of interest are identified quickly by using accelerated searches, usually in a few seconds. Once homologs are identified, fast.genomics can quickly show their prevalence across taxa, view their neighboring genes, or compare the prevalence of two different proteins. Fast.genomics is available at https://fast.genomics.lbl.gov.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.15.553440">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.15.553440" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.15.553440">
        <p class="paperTitle">Analyzing microbial evolution through gene and genome phylogenies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.15.553440" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.15.553440" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Teichman, S.; Lee, M. D.; Willis, A. D.</p>
        <p class="info">Score: 33.5, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.15.553440' target='https://doi.org/10.1101/2023.08.15.553440'> 10.1101/2023.08.15.553440</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWMicrobiome scientists critically need modern tools to explore and analyze microbial evolution. Often this involves studying the evolution of microbial genomes as a whole. However, different genes in a single genome can be subject to different evolutionary pressures, which can result in distinct gene-level evolutionary histories. To address this challenge, we propose to treat estimated gene-level phylogenies as data objects, and present an interactive method for the analysis of a collection of gene phylogenies. We use a local linear approximation of phylogenetic tree space to visualize estimated gene trees as points in low-dimensional Euclidean space, and address important practical limitations of existing related approaches, allowing an intuitive visualization of complex data objects. We demonstrate the utility of our proposed approach through microbial data analyses, including by identifying outlying gene histories in strains of Prevotella, and by contrasting Streptococcus phylogenies estimated using different gene sets. Our method is available as an open-source R package, and assists with estimating, visualizing and interacting with a collection of bacterial gene phylogenies. dimension reduction, microbiome, non-Euclidean, statistical genetics, visualization</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.554251">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.554251" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.554251">
        <p class="paperTitle">Self-supervised learning for characterising histomorphological diversity and spatial RNA expression prediction across 23 human tissue types</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.554251" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.554251" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cisternino, F.; Ometto, S.; Chatterjee, S.; Giacopuzzi, E.; Levine, A. P.; Glastonbury, C. A.</p>
        <p class="info">Score: 21.2, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.554251' target='https://doi.org/10.1101/2023.08.22.554251'> 10.1101/2023.08.22.554251</a></p>
        <p class="abstract">As vast histological archives are digitised, there is a pressing need to be able to associate specific tissue substructures and incident pathology to disease outcomes without arduous annotation. Such automation provides an opportunity to learn fundamental biology about how tissue structure and function varies in a population. Recently, self-supervised learning has proven competitive to supervised machine learning approaches in classification, segmentation and representation learning. Here, we leverage self-supervised learning to generate histology feature representations using 1.7M images across 23 healthy tissues in 838 donors from GTEx. Using these representations, we demonstrate we can automatically segment tissues into their constituent tissue substructures and pathology proportions, and surpass the performance of conventionally used pre-trained models. We observe striking population variability in canonical tissue substructures, highlight examples of missing pathological diagnoses, incorrect assignment of target tissue and cross-tissue contamination. We demonstrate that this variability in tissue composition leads to a likely overestimation of eQTL tissue sharing and drives dramatic differential gene expression changes. We use derived tissue substructures to detect 284 tissue substructures and pathology specific eQTLs. As our derived histology representations are rich morphological descriptors of the underlying tissue, we introduce a multiple instance learning model that can predict and spatially localise individual RNA expression levels directly from histology to specific substructures and pathological features. We validate our RNA spatial predictions with matched ground truth immunohistochemistry (IHC) for several well characterised marker genes, recapitulating their known spatial specificity. Finally, we derive a gene expression spatial enrichment metric, allowing us to detect genes specifically expressed within sites of pathology (e.g. arterial calcification). Together, these results demonstrate the power of self-supervised machine learning when applied to vast histological datasets to allow researchers to pose and answer questions about tissue pathology, its spatial organisation and the interplay between morphological tissue variability and gene expression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554147">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554147" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554147">
        <p class="paperTitle">A multimodal Transformer Network for protein-small molecule interactions enhances drug-target affinity and enzyme-substrate predictions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554147" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554147" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kroll, A.; Ranjan, S.; Lercher, M. J.</p>
        <p class="info">Score: 18.6, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554147' target='https://doi.org/10.1101/2023.08.21.554147'> 10.1101/2023.08.21.554147</a></p>
        <p class="abstract">The activities of most enzymes and drugs rely on interactions between proteins and small molecules. Accurate predictions of these interactions could accelerate pharmaceutical and biotechnological research massively. Machine learning models designed for this task are currently limited by the lack of information exchange between the protein and the small molecule during the creation of the required numerical representations. Here, we introduce ProSmith, a machine learning framework that employs a multimodal Transformer Network to simultaneously process protein amino acid sequences and small molecule strings in the same input. This approach facilitates the exchange of all relevant information between the two molecule types during the calculation of their numerical representations, allowing the model to account for their structural and functional interactions. Our final model combines gradient boosting predictions based on the resulting multimodal Transformer Network with independent predictions based on separate deep learning representations of the proteins and small molecules. The corresponding predictions outperform all previous models for predicting protein-small molecule interactions across three diverse tasks: predicting Michaelis constants KM; inferring potential substrates for enzymes; and predicting protein-drug affinities. The Python code provided can be used to easily implement and improve machine learning predictions involving arbitrary protein-small molecule interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.17.553778">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.17.553778" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.17.553778">
        <p class="paperTitle">ClairS: a deep-learning method for long-read somatic small variant calling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.17.553778" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.17.553778" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zheng, Z.; Su, J.; Chen, L.; Lee, Y.-L.; Lam, T.-W.; Luo, R.</p>
        <p class="info">Score: 15.5, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.17.553778' target='https://doi.org/10.1101/2023.08.17.553778'> 10.1101/2023.08.17.553778</a></p>
        <p class="abstract">Identifying somatic variants in tumor samples is a crucial task, which is often performed using statistical methods and heuristic filters applied to short-read data. However, with the increasing demand for long-read somatic variant calling, existing methods have fallen short. To address this gap, we present ClairS, the first deep-learning-based, long-read somatic small variant caller. ClairS was trained on massive synthetic somatic variants with diverse coverages and variant allele frequencies (VAF), enabling it to accurately detect a wide range of somatic variants from paired tumor and normal samples. We evaluated ClairS using the latest Nanopore Q20&#43; HCC1395-HCC1395BL dataset. With 50-fold/25-fold tumor/normal, ClairS achieved a 93.01%/86.86% precision/recall rate for Single Nucleotide Variation (SNVs), and 66.54%/66.89% for somatic insertions and deletions (Indels). Applying ClairS to short-read datasets from multiple sources showed comparable or better performance than Strelka2 and Mutect2. Our findings suggest that improved read phasing enabled by long-read sequencing is key to accurate long-read SNV calling, especially for variants with low VAF. Through experiments across various coverage, purity, and contamination settings, we demonstrated that ClairS is a reliable somatic variant caller. ClairS is open-source at https://github.com/HKU-BAL/ClairS.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.01.07.523121">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.01.07.523121" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.01.07.523121">
        <p class="paperTitle">Protein language model embedded geometric graphs power inter-protein contact prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.01.07.523121" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.01.07.523121" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Si, Y.; Yan, C.</p>
        <p class="info">Score: 23.9, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.01.07.523121' target='https://doi.org/10.1101/2023.01.07.523121'> 10.1101/2023.01.07.523121</a></p>
        <p class="abstract">Accurate prediction of contacting residue pairs between interacting proteins is very useful for structural characterization of protein-protein interactions (PPIs). Although significant improvement has been made in inter-protein contact prediction recently, there is still large room for improving the prediction accuracy. Here we present a new deep learning method referred to as PLMGraph-Inter for inter-protein contact prediction. Specifically, we employ rotationally and translationally invariant geometric graphs obtained from structures of interacting proteins to integrate multiple protein language models, which are successively transformed by graph encoders formed by geometric vector perceptrons and residual networks formed by dimensional hybrid residual blocks to predict inter-protein contacts. Extensive evaluation on multiple test sets illustrates that PLMGraph-Inter outperforms five top inter-protein contact prediction methods, including DeepHomo, GLINTER, CDPred, DeepHomo2 and DRN-1D2D_Inter by large margins. In addition, we also show that the prediction of PLMGraph-Inter can complement the result of AlphaFold-Multimer. Finally, we show leveraging the contacts predicted by PLMGraph-Inter as constraints for protein-protein docking can dramatically improve its performance for protein complex structure prediction.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
