<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Deep dynamical modelling of developmental trajectories with temporal transcriptomics
Authors: Maizels, R. J.; Snell, D. M.; Briscoe, J.
Score: 58.7, Published: 2023-07-07 DOI: 10.1101/2023.07.06.547989
Developmental cell fate decisions are dynamic processes driven by the complex behaviour of gene regulatory networks. A challenge in studying these processes using single-cell genomics is that the data provides only a static snapshot with no detail of dynamics. Metabolic labelling and splicing can provide time-resolved information, but current methods have limitations.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="Deep dynamical modelling of developmental trajectories with temporal transcriptomics
Authors: Maizels, R. J.; Snell, D. M.; Briscoe, J.
Score: 58.7, Published: 2023-07-07 DOI: 10.1101/2023.07.06.547989
Developmental cell fate decisions are dynamic processes driven by the complex behaviour of gene regulatory networks. A challenge in studying these processes using single-cell genomics is that the data provides only a static snapshot with no detail of dynamics. Metabolic labelling and splicing can provide time-resolved information, but current methods have limitations." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-12T10:41:10+00:00" />
<meta property="article:modified_time" content="2023-07-12T10:41:10+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="Deep dynamical modelling of developmental trajectories with temporal transcriptomics
Authors: Maizels, R. J.; Snell, D. M.; Briscoe, J.
Score: 58.7, Published: 2023-07-07 DOI: 10.1101/2023.07.06.547989
Developmental cell fate decisions are dynamic processes driven by the complex behaviour of gene regulatory networks. A challenge in studying these processes using single-cell genomics is that the data provides only a static snapshot with no detail of dynamics. Metabolic labelling and splicing can provide time-resolved information, but current methods have limitations."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "Deep dynamical modelling of developmental trajectories with temporal transcriptomics\nAuthors: Maizels, R. J.; Snell, D. M.; Briscoe, J.\nScore: 58.7, Published: 2023-07-07 DOI: 10.1101/2023.07.06.547989\nDevelopmental cell fate decisions are dynamic processes driven by the complex behaviour of gene regulatory networks. A challenge in studying these processes using single-cell genomics is that the data provides only a static snapshot with no detail of dynamics. Metabolic labelling and splicing can provide time-resolved information, but current methods have limitations.",
  "keywords": [
    
  ],
  "articleBody": " Deep dynamical modelling of developmental trajectories with temporal transcriptomics\nAuthors: Maizels, R. J.; Snell, D. M.; Briscoe, J.\nScore: 58.7, Published: 2023-07-07 DOI: 10.1101/2023.07.06.547989\nDevelopmental cell fate decisions are dynamic processes driven by the complex behaviour of gene regulatory networks. A challenge in studying these processes using single-cell genomics is that the data provides only a static snapshot with no detail of dynamics. Metabolic labelling and splicing can provide time-resolved information, but current methods have limitations. Here, we present experimental and computational methods that overcome these limitations to allow dynamical modelling of gene expression from single-cell data. We developed sci-FATE2, an optimised metabolic labelling method that substantially increases data quality, and profiled approximately 45,000 embryonic stem cells differentiating into multiple neural tube identities. To recover dynamics, we developed velvet, a deep learning framework that extends beyond instantaneous velocity estimation by modelling gene expression dynamics through a neural stochastic differential equation system within a variational autoencoder. Velvet outperforms current velocity tools across quantitative benchmarks, and predicts trajectory distributions that accurately recapitulate underlying dataset distributions while conserving known biology. Velvet trajectory distributions capture dynamical aspects such as decision boundaries between alternative fates and correlative gene regulatory structure. Using velvet to provide a dynamical description of in vitro neural patterning, we highlight a process of sequential decision making and fate-specific patterns of developmental signalling. Together, these experimental and computational methods recast single-cell analyses from descriptions of observed data distributions to models of the dynamics that generated them, providing a new framework for investigating developmental gene regulation and cell fate decisions.\nScalable, accessible, and reproducible reference genome assembly and evaluation in Galaxy\nAuthors: Lariviere, D.; Abueg, L.; Brajuka, N.; Gallardo-Alba, C.; Gruning, B.; Ko, B. J.; Ostrovsky, A.; Palmada-Flores, M.; Pickett, B. D.; Rabbani, K.; Balacco, J. R.; Chaisson, M.; Cheng, H.; Collins, J.; Denisova, A.; Fedrigo, O.; Gallo, G. R.; Giani, A. M.; Gooder, G. M.; Jain, N.; Johnson, C.; Kim, H.; Lee, C.; Marques-Bonet, T.; O'Toole, B.; Rhie, A.; Secomandi, S.; Sozzoni, M.; Tilley, T.; Uliano-Silva, M.; van den Beek, M.; Waterhouse, R. M.; Phillippy, A. M.; Jarvis, E. D.; Schatz, M.; Nekrutenko, A.; Formenti, G.\nScore: 46.9, Published: 2023-06-30 DOI: 10.1101/2023.06.28.546576\nImprovements in genome sequencing and assembly are enabling high-quality reference genomes for all species. However, the assembly process is still laborious, computationally and technically demanding, lacks standards for reproducibility, and is not readily scalable. Here we present the latest Vertebrate Genomes Project assembly pipeline and demonstrate that it delivers high-quality reference genomes at scale across a set of vertebrate species arising over the last [~]500 million years. The pipeline is versatile and combines PacBio HiFi long-reads and Hi-C-based haplotype phasing in a new graph-based paradigm. Standardized quality control is performed automatically to troubleshoot assembly issues and assess biological complexities. We make the pipeline freely accessible through Galaxy, accommodating researchers even without local computational resources and enhanced reproducibility by democratizing the training and assembly process. We demonstrate the flexibility and reliability of the pipeline by assembling reference genomes for 51 vertebrate species from major taxonomic groups (fish, amphibians, reptiles, birds, and mammals).\nColabDock: inverting AlphaFold structure prediction model for protein-protein docking with experimental restraints\nAuthors: Feng, S.; Chen, Z.; Zhang, C.; Xie, Y.; Ovchinnikov, S.; Gao, Y. Q.; Liu, S.\nScore: 47.5, Published: 2023-07-04 DOI: 10.1101/2023.07.04.547599\nPrediction of protein complex structures and interfaces potentially has wide applications and can benefit the study of biological mechanisms involving protein-protein interactions. However, the surface prediction accuracy of traditional docking methods and AlphaFold-Multimer is limited. Here we present ColabDock, a framework that makes use of ColabDesign, but reimplements it for the purpose of restrained complex conformation prediction. With a generation-prediction architecture and trained ranking model, ColabDock outperforms HADDOCK and ClusPro not only in complex structure predictions with simulated residue and surface restraints, but also in those assisted by NMR chemical shift perturbation as well as covalent labeling. It further assists antibody-antigen interface prediction with emulated interface scan restraints, which could be obtained by experiments such as Deep Mutation Scan. ColabDock provides a general approach to integrate sparse interface restraints of different experimental forms and sources into one optimization framework.\nHostile: accurate host decontamination of microbial sequences\nAuthors: Constantinides, B.; Crook, D. W.\nScore: 36.0, Published: 2023-07-05 DOI: 10.1101/2023.07.04.547735\nSummaryMicrobial sequences generated from clinical samples are often contaminated with human host sequences that must be removed for ethical and legal reasons. Care must be taken to excise host sequences without inadvertently removing target microbial sequences to the detriment of downstream analyses such as variant calling and de novo assembly. To facilitate accurate host decontamination of both short and long sequencing reads, we developed Hostile, a tool capable of rapid host read removal using laptop specification hardware. We demonstrate that our approach removes at least 99.868% of real human reads and retains at least 99.997% of simulated bacterial reads. Use of a masked reference genome further increases bacterial read retention (\u003e=99.997%) with negligible (\u003c0.001%) reduction in human read removal performance. Compared with an existing tool, Hostile removed up to 11x more human reads and up to 11x fewer microbial reads while taking less time for typical workloads. Availability and implementationHostile is implemented as an MIT licensed Python package available at https://github.com/bede/hostile\nEfficient High-Quality Metagenome Assembly from Long Accurate Reads using Minimizer-space de Bruijn Graphs\nAuthors: Benoit, G.; Raguideau, S.; James, R.; Phillippy, A. M.; Chikhi, R.; Quince, C.\nScore: 34.3, Published: 2023-07-08 DOI: 10.1101/2023.07.07.548136\nWe introduce a novel metagenomics assembler for high-accuracy long reads. Our approach, implemented as metaMDBG, combines highly efficient de Bruijn graph assembly in minimizer space, with both a multi-k' approach for dealing with variations in genome coverage depth and an abundance-based filtering strategy for simplifying strain complexity. The resulting algorithm is more efficient than the state-of-the-art but with better assembly results. MetaMDBG was 1.5 to 12 times faster than competing assemblers and requires between one-tenth and one-thirtieth of the memory across a range of data sets. We obtained up to twice as many high-quality circularised prokaryotic metagenome assembled genomes (MAGs) on the most complex communities, and a better recovery of viruses and plasmids. MetaMDBG performs particularly well for abundant organisms whilst being robust to the presence of strain diversity. The result is that for the first time it is possible to efficiently reconstruct the majority of complex communities by abundance as near-complete MAGs.\nSemi-supervised integration of single-cell transcriptomics data\nAuthors: Andreatta, M.; Herault, L.; Gueguen, P.; Gfeller, D.; Berenstein, A. J.; Carmona, S. J.\nScore: 33.5, Published: 2023-07-07 DOI: 10.1101/2023.07.07.548105\nSingle-cell sequencing technologies offer unprecedented opportunities to characterize the complexity of biological samples with high resolution. At the same time, variations in sample processing and experimental protocols introduce technical variability - or \"batch effects\" - in the molecular readouts, hindering comparative analyses across samples and individuals. Although batch effect correction methods are routinely applied in single-cell omics analyses, data integration often leads to overcorrection, resulting in the loss of true biological variability. In this study, we present STACAS v2, a semi-supervised scRNA-seq data integration method that leverages prior knowledge in the form of cell type annotations to preserve biological variance. Through an open and reproducible benchmarking pipeline, we show that semi-supervised STACAS outperforms popular unsupervised methods such as Harmony, FastMNN, Seurat v4, scVI, and Scanorama, as well as supervised methods such as scANVI and scGen. Notably, STACAS is robust to incomplete and imprecise cell type annotations, which are commonly encountered in real-life integration tasks. Highlighting its scalability, we successfully applied semi-supervised STACAS to construct a high-resolution map of tumor-infiltrating CD8 T cells encompassing over 500,000 cells from 265 individuals. Based on our findings, we argue that the incorporation of prior cell type information should be a common practice in single-cell data integration, and we provide a flexible framework for semi-supervised batch effect correction. STACAS seamlessly integrates with Seurat pipelines and can be run with one command: Run.STACAS(seurat.list, cell.labels).\nImproved protein complex prediction with AlphaFold-multimer by denoising the MSA profile\nAuthors: Bryant, P.; Noe, F.\nScore: 29.2, Published: 2023-07-04 DOI: 10.1101/2023.07.04.547638\nStructure prediction of protein complexes has improved significantly with AlphaFold2 and AlphaFold-multimer (AFM), but only 60% of dimers are accurately predicted. A way to improve the predictions is to inject noise to generate more diverse predictions. However, thousands of predictions are needed to obtain a few that are accurate in difficult cases. Here, we learn a bias to the MSA representation that improves the predictions by performing gradient descent through the AFM network. We effectively denoise the MSA profile, similar to how a blurry image would be sharpened. We demonstrate the performance on seven difficult targets from CASP15 and increase the average MMscore to 0.76 compared to 0.63 with AFM. We evaluate the procedure on 334 protein complexes where AFM fails and demonstrate an increased success rate (MMscore\u003e0.75) of 8% on these hard targets. Our protocol, AFProfile, provides a way to direct predictions towards a defined target function guided by the MSA. We expect gradient descent over the MSA to be useful for different tasks, such as generating alternative conformations. AFProfile is freely available at: https://github.com/patrickbryant1/AFProfile\nPetascale Homology Search for Structure Prediction\nAuthors: Lee, S.; Kim, G.; Levy Karin, E.; Mirdita, M.; Park, S.; Chikhi, R.; Babaian, A.; Kryshtafovych, A.; Steinegger, M.\nScore: 28.4, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548308\nThe recent CASP15 competition highlighted the critical role of multiple sequence alignments (MSAs) in protein structure prediction, as demonstrated by the success of the top AlphaFold2-based prediction methods. To push the boundaries of MSA utilization, we conducted a petabase-scale search of the Sequence Read Archive (SRA), resulting in gigabytes of aligned homologs for CASP15 targets. These were merged with default MSAs produced by ColabFold-search and provided to ColabFold-predict. By using SRA data, we achieved highly accurate predictions (GDT_TS \u003e 70) for 66% of the non-easy targets, whereas using ColabFold-search default MSAs scored highly in only 52%. Next, we tested the effect of deep homology search and ColabFold's advanced features, such as more recycles, on prediction accuracy. While SRA homologs were most significant for improving ColabFold's CASP15 ranking from 11th to 3rd place, other strategies contributed too. We analyze these in the context of existing strategies to improve prediction.\nA pre-trained large generative model for translating single-cell transcriptome to proteome\nAuthors: Liu, L.; Li, W.; Wong, K.-c.; Yang, F.; Yao, J.\nScore: 26.8, Published: 2023-07-09 DOI: 10.1101/2023.07.04.547619\nProteins are crucial for life, and measuring their abundance at the single-cell level can facilitate a high-resolution understanding of biological mechanisms in cellular processes and disease progression. However, current single-cell proteomic technologies face challenges such as limited coverage, throughput, and sensitivity, as well as batch effects, high costs, and stringent experimental operations. Drawing inspiration from the translation procedure of both natural language processing (NLP) and the genetic central dogma, we propose a pre-trained, large generative model named scTranslator (single-cell translator). scTranslator is align-free and capable of generating multi-omics data by inferring the missing single-cell proteome based on the transcriptome. Systematic benchmarking confirms the accuracy, stability, and flexibility of scTranslator across various quantification techniques, cell types, and conditions. Furthermore, scTranslator has demonstrated its superiority in assisting various downstream analyses and applications, including gene/protein interaction inference, gene pseudo-knockout, cell clustering, batch correction, and cell origin recognition on pan-cancer data.\nModelling protein complexes with crosslinking mass spectrometry and deep learning\nAuthors: Stahl, K.; Brock, O.; Rappsilber, J.\nScore: 34.3, Published: 2023-06-28 DOI: 10.1101/2023.06.07.544059\nScarcity of structural and evolutionary information on protein complexes poses a challenge to deep learning-based structure modelling. We integrated experimental distance restraints obtained by crosslinking mass spectrometry (MS) into AlphaFold-Multimer, by extending AlphaLink to protein complexes. Integrating crosslinking MS data substantially improves modelling performance on challenging targets, by helping to identify interfaces, focusing sampling, and improving model selection. This extends to single crosslinks from whole-cell crosslinking MS, suggesting the possibility of whole-cell structural investigations driven by experimental data.\n",
  "wordCount" : "1959",
  "inLanguage": "en",
  "datePublished": "2023-07-12T10:41:10Z",
  "dateModified": "2023-07-12T10:41:10Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on July 12, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.547989">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.547989" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.547989">
        <p class="paperTitle">Deep dynamical modelling of developmental trajectories with temporal transcriptomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.547989" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.547989" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Maizels, R. J.; Snell, D. M.; Briscoe, J.</p>
        <p class="info">Score: 58.7, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.547989' target='https://doi.org/10.1101/2023.07.06.547989'> 10.1101/2023.07.06.547989</a></p>
        <p class="abstract">Developmental cell fate decisions are dynamic processes driven by the complex behaviour of gene regulatory networks. A challenge in studying these processes using single-cell genomics is that the data provides only a static snapshot with no detail of dynamics. Metabolic labelling and splicing can provide time-resolved information, but current methods have limitations. Here, we present experimental and computational methods that overcome these limitations to allow dynamical modelling of gene expression from single-cell data. We developed sci-FATE2, an optimised metabolic labelling method that substantially increases data quality, and profiled approximately 45,000 embryonic stem cells differentiating into multiple neural tube identities. To recover dynamics, we developed velvet, a deep learning framework that extends beyond instantaneous velocity estimation by modelling gene expression dynamics through a neural stochastic differential equation system within a variational autoencoder. Velvet outperforms current velocity tools across quantitative benchmarks, and predicts trajectory distributions that accurately recapitulate underlying dataset distributions while conserving known biology. Velvet trajectory distributions capture dynamical aspects such as decision boundaries between alternative fates and correlative gene regulatory structure. Using velvet to provide a dynamical description of in vitro neural patterning, we highlight a process of sequential decision making and fate-specific patterns of developmental signalling. Together, these experimental and computational methods recast single-cell analyses from descriptions of observed data distributions to models of the dynamics that generated them, providing a new framework for investigating developmental gene regulation and cell fate decisions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546576">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546576" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546576">
        <p class="paperTitle">Scalable, accessible, and reproducible reference genome assembly and evaluation in Galaxy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546576" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546576" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lariviere, D.; Abueg, L.; Brajuka, N.; Gallardo-Alba, C.; Gruning, B.; Ko, B. J.; Ostrovsky, A.; Palmada-Flores, M.; Pickett, B. D.; Rabbani, K.; Balacco, J. R.; Chaisson, M.; Cheng, H.; Collins, J.; Denisova, A.; Fedrigo, O.; Gallo, G. R.; Giani, A. M.; Gooder, G. M.; Jain, N.; Johnson, C.; Kim, H.; Lee, C.; Marques-Bonet, T.; O&#39;Toole, B.; Rhie, A.; Secomandi, S.; Sozzoni, M.; Tilley, T.; Uliano-Silva, M.; van den Beek, M.; Waterhouse, R. M.; Phillippy, A. M.; Jarvis, E. D.; Schatz, M.; Nekrutenko, A.; Formenti, G.</p>
        <p class="info">Score: 46.9, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546576' target='https://doi.org/10.1101/2023.06.28.546576'> 10.1101/2023.06.28.546576</a></p>
        <p class="abstract">Improvements in genome sequencing and assembly are enabling high-quality reference genomes for all species. However, the assembly process is still laborious, computationally and technically demanding, lacks standards for reproducibility, and is not readily scalable. Here we present the latest Vertebrate Genomes Project assembly pipeline and demonstrate that it delivers high-quality reference genomes at scale across a set of vertebrate species arising over the last [~]500 million years. The pipeline is versatile and combines PacBio HiFi long-reads and Hi-C-based haplotype phasing in a new graph-based paradigm. Standardized quality control is performed automatically to troubleshoot assembly issues and assess biological complexities. We make the pipeline freely accessible through Galaxy, accommodating researchers even without local computational resources and enhanced reproducibility by democratizing the training and assembly process. We demonstrate the flexibility and reliability of the pipeline by assembling reference genomes for 51 vertebrate species from major taxonomic groups (fish, amphibians, reptiles, birds, and mammals).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547599">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547599" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547599">
        <p class="paperTitle">ColabDock: inverting AlphaFold structure prediction model for protein-protein docking with experimental restraints</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547599" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547599" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Feng, S.; Chen, Z.; Zhang, C.; Xie, Y.; Ovchinnikov, S.; Gao, Y. Q.; Liu, S.</p>
        <p class="info">Score: 47.5, Published: 2023-07-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547599' target='https://doi.org/10.1101/2023.07.04.547599'> 10.1101/2023.07.04.547599</a></p>
        <p class="abstract">Prediction of protein complex structures and interfaces potentially has wide applications and can benefit the study of biological mechanisms involving protein-protein interactions. However, the surface prediction accuracy of traditional docking methods and AlphaFold-Multimer is limited. Here we present ColabDock, a framework that makes use of ColabDesign, but reimplements it for the purpose of restrained complex conformation prediction. With a generation-prediction architecture and trained ranking model, ColabDock outperforms HADDOCK and ClusPro not only in complex structure predictions with simulated residue and surface restraints, but also in those assisted by NMR chemical shift perturbation as well as covalent labeling. It further assists antibody-antigen interface prediction with emulated interface scan restraints, which could be obtained by experiments such as Deep Mutation Scan. ColabDock provides a general approach to integrate sparse interface restraints of different experimental forms and sources into one optimization framework.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547735">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547735" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547735">
        <p class="paperTitle">Hostile: accurate host decontamination of microbial sequences</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547735" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547735" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Constantinides, B.; Crook, D. W.</p>
        <p class="info">Score: 36.0, Published: 2023-07-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547735' target='https://doi.org/10.1101/2023.07.04.547735'> 10.1101/2023.07.04.547735</a></p>
        <p class="abstract">SummaryMicrobial sequences generated from clinical samples are often contaminated with human host sequences that must be removed for ethical and legal reasons. Care must be taken to excise host sequences without inadvertently removing target microbial sequences to the detriment of downstream analyses such as variant calling and de novo assembly. To facilitate accurate host decontamination of both short and long sequencing reads, we developed Hostile, a tool capable of rapid host read removal using laptop specification hardware. We demonstrate that our approach removes at least 99.868% of real human reads and retains at least 99.997% of simulated bacterial reads. Use of a masked reference genome further increases bacterial read retention (&gt;=99.997%) with negligible (&lt;0.001%) reduction in human read removal performance. Compared with an existing tool, Hostile removed up to 11x more human reads and up to 11x fewer microbial reads while taking less time for typical workloads.

Availability and implementationHostile is implemented as an MIT licensed Python package available at https://github.com/bede/hostile</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548136">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548136" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548136">
        <p class="paperTitle">Efficient High-Quality Metagenome Assembly from Long Accurate Reads using Minimizer-space de Bruijn Graphs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548136" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548136" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Benoit, G.; Raguideau, S.; James, R.; Phillippy, A. M.; Chikhi, R.; Quince, C.</p>
        <p class="info">Score: 34.3, Published: 2023-07-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548136' target='https://doi.org/10.1101/2023.07.07.548136'> 10.1101/2023.07.07.548136</a></p>
        <p class="abstract">We introduce a novel metagenomics assembler for high-accuracy long reads. Our approach, implemented as metaMDBG, combines highly efficient de Bruijn graph assembly in minimizer space, with both a multi-k&#39; approach for dealing with variations in genome coverage depth and an abundance-based filtering strategy for simplifying strain complexity. The resulting algorithm is more efficient than the state-of-the-art but with better assembly results. MetaMDBG was 1.5 to 12 times faster than competing assemblers and requires between one-tenth and one-thirtieth of the memory across a range of data sets. We obtained up to twice as many high-quality circularised prokaryotic metagenome assembled genomes (MAGs) on the most complex communities, and a better recovery of viruses and plasmids. MetaMDBG performs particularly well for abundant organisms whilst being robust to the presence of strain diversity. The result is that for the first time it is possible to efficiently reconstruct the majority of complex communities by abundance as near-complete MAGs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548105">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548105" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548105">
        <p class="paperTitle">Semi-supervised integration of single-cell transcriptomics data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548105" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548105" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Andreatta, M.; Herault, L.; Gueguen, P.; Gfeller, D.; Berenstein, A. J.; Carmona, S. J.</p>
        <p class="info">Score: 33.5, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548105' target='https://doi.org/10.1101/2023.07.07.548105'> 10.1101/2023.07.07.548105</a></p>
        <p class="abstract">Single-cell sequencing technologies offer unprecedented opportunities to characterize the complexity of biological samples with high resolution. At the same time, variations in sample processing and experimental protocols introduce technical variability - or &#34;batch effects&#34; - in the molecular readouts, hindering comparative analyses across samples and individuals. Although batch effect correction methods are routinely applied in single-cell omics analyses, data integration often leads to overcorrection, resulting in the loss of true biological variability. In this study, we present STACAS v2, a semi-supervised scRNA-seq data integration method that leverages prior knowledge in the form of cell type annotations to preserve biological variance. Through an open and reproducible benchmarking pipeline, we show that semi-supervised STACAS outperforms popular unsupervised methods such as Harmony, FastMNN, Seurat v4, scVI, and Scanorama, as well as supervised methods such as scANVI and scGen. Notably, STACAS is robust to incomplete and imprecise cell type annotations, which are commonly encountered in real-life integration tasks. Highlighting its scalability, we successfully applied semi-supervised STACAS to construct a high-resolution map of tumor-infiltrating CD8 T cells encompassing over 500,000 cells from 265 individuals. Based on our findings, we argue that the incorporation of prior cell type information should be a common practice in single-cell data integration, and we provide a flexible framework for semi-supervised batch effect correction. STACAS seamlessly integrates with Seurat pipelines and can be run with one command: Run.STACAS(seurat.list, cell.labels).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547638">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547638" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547638">
        <p class="paperTitle">Improved protein complex prediction with AlphaFold-multimer by denoising the MSA profile</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547638" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547638" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bryant, P.; Noe, F.</p>
        <p class="info">Score: 29.2, Published: 2023-07-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547638' target='https://doi.org/10.1101/2023.07.04.547638'> 10.1101/2023.07.04.547638</a></p>
        <p class="abstract">Structure prediction of protein complexes has improved significantly with AlphaFold2 and AlphaFold-multimer (AFM), but only 60% of dimers are accurately predicted. A way to improve the predictions is to inject noise to generate more diverse predictions. However, thousands of predictions are needed to obtain a few that are accurate in difficult cases. Here, we learn a bias to the MSA representation that improves the predictions by performing gradient descent through the AFM network. We effectively denoise the MSA profile, similar to how a blurry image would be sharpened. We demonstrate the performance on seven difficult targets from CASP15 and increase the average MMscore to 0.76 compared to 0.63 with AFM. We evaluate the procedure on 334 protein complexes where AFM fails and demonstrate an increased success rate (MMscore&gt;0.75) of 8% on these hard targets. Our protocol, AFProfile, provides a way to direct predictions towards a defined target function guided by the MSA. We expect gradient descent over the MSA to be useful for different tasks, such as generating alternative conformations. AFProfile is freely available at: https://github.com/patrickbryant1/AFProfile</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548308">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548308" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548308">
        <p class="paperTitle">Petascale Homology Search for Structure Prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548308" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548308" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, S.; Kim, G.; Levy Karin, E.; Mirdita, M.; Park, S.; Chikhi, R.; Babaian, A.; Kryshtafovych, A.; Steinegger, M.</p>
        <p class="info">Score: 28.4, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548308' target='https://doi.org/10.1101/2023.07.10.548308'> 10.1101/2023.07.10.548308</a></p>
        <p class="abstract">The recent CASP15 competition highlighted the critical role of multiple sequence alignments (MSAs) in protein structure prediction, as demonstrated by the success of the top AlphaFold2-based prediction methods. To push the boundaries of MSA utilization, we conducted a petabase-scale search of the Sequence Read Archive (SRA), resulting in gigabytes of aligned homologs for CASP15 targets. These were merged with default MSAs produced by ColabFold-search and provided to ColabFold-predict. By using SRA data, we achieved highly accurate predictions (GDT_TS &gt; 70) for 66% of the non-easy targets, whereas using ColabFold-search default MSAs scored highly in only 52%. Next, we tested the effect of deep homology search and ColabFold&#39;s advanced features, such as more recycles, on prediction accuracy. While SRA homologs were most significant for improving ColabFold&#39;s CASP15 ranking from 11th to 3rd place, other strategies contributed too. We analyze these in the context of existing strategies to improve prediction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547619">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547619" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547619">
        <p class="paperTitle">A pre-trained large generative model for translating single-cell transcriptome to proteome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547619" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547619" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, L.; Li, W.; Wong, K.-c.; Yang, F.; Yao, J.</p>
        <p class="info">Score: 26.8, Published: 2023-07-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547619' target='https://doi.org/10.1101/2023.07.04.547619'> 10.1101/2023.07.04.547619</a></p>
        <p class="abstract">Proteins are crucial for life, and measuring their abundance at the single-cell level can facilitate a high-resolution understanding of biological mechanisms in cellular processes and disease progression. However, current single-cell proteomic technologies face challenges such as limited coverage, throughput, and sensitivity, as well as batch effects, high costs, and stringent experimental operations. Drawing inspiration from the translation procedure of both natural language processing (NLP) and the genetic central dogma, we propose a pre-trained, large generative model named scTranslator (single-cell translator). scTranslator is align-free and capable of generating multi-omics data by inferring the missing single-cell proteome based on the transcriptome. Systematic benchmarking confirms the accuracy, stability, and flexibility of scTranslator across various quantification techniques, cell types, and conditions. Furthermore, scTranslator has demonstrated its superiority in assisting various downstream analyses and applications, including gene/protein interaction inference, gene pseudo-knockout, cell clustering, batch correction, and cell origin recognition on pan-cancer data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.07.544059">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.07.544059" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.07.544059">
        <p class="paperTitle">Modelling protein complexes with crosslinking mass spectrometry and deep learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.07.544059" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.07.544059" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stahl, K.; Brock, O.; Rappsilber, J.</p>
        <p class="info">Score: 34.3, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.07.544059' target='https://doi.org/10.1101/2023.06.07.544059'> 10.1101/2023.06.07.544059</a></p>
        <p class="abstract">Scarcity of structural and evolutionary information on protein complexes poses a challenge to deep learning-based structure modelling. We integrated experimental distance restraints obtained by crosslinking mass spectrometry (MS) into AlphaFold-Multimer, by extending AlphaLink to protein complexes. Integrating crosslinking MS data substantially improves modelling performance on challenging targets, by helping to identify interfaces, focusing sampling, and improving model selection. This extends to single crosslinks from whole-cell crosslinking MS, suggesting the possibility of whole-cell structural investigations driven by experimental data.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
