<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="The Scalable Variant Call Representation: Enabling Genetic Analysis Beyond One Million Genomes
Authors: Poterba, T.; Vittal, C.; King, D.; Goldstein, D.; Goldstein, J.; Schultz, P.; Karczewski, K.; Seed, C.; Neale, B.
Score: 17.8, Published: 2024-01-10 DOI: 10.1101/2024.01.09.574205
The Variant Call Format (VCF) is widely used in genome sequencing but scales poorly. For instance, we estimate a 150,000 genome VCF would occupy 900 TiB, making it both costly and complicated to produce and analyze.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="The Scalable Variant Call Representation: Enabling Genetic Analysis Beyond One Million Genomes
Authors: Poterba, T.; Vittal, C.; King, D.; Goldstein, D.; Goldstein, J.; Schultz, P.; Karczewski, K.; Seed, C.; Neale, B.
Score: 17.8, Published: 2024-01-10 DOI: 10.1101/2024.01.09.574205
The Variant Call Format (VCF) is widely used in genome sequencing but scales poorly. For instance, we estimate a 150,000 genome VCF would occupy 900 TiB, making it both costly and complicated to produce and analyze." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-17T10:40:22+00:00" />
<meta property="article:modified_time" content="2024-01-17T10:40:22+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="The Scalable Variant Call Representation: Enabling Genetic Analysis Beyond One Million Genomes
Authors: Poterba, T.; Vittal, C.; King, D.; Goldstein, D.; Goldstein, J.; Schultz, P.; Karczewski, K.; Seed, C.; Neale, B.
Score: 17.8, Published: 2024-01-10 DOI: 10.1101/2024.01.09.574205
The Variant Call Format (VCF) is widely used in genome sequencing but scales poorly. For instance, we estimate a 150,000 genome VCF would occupy 900 TiB, making it both costly and complicated to produce and analyze."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "The Scalable Variant Call Representation: Enabling Genetic Analysis Beyond One Million Genomes\nAuthors: Poterba, T.; Vittal, C.; King, D.; Goldstein, D.; Goldstein, J.; Schultz, P.; Karczewski, K.; Seed, C.; Neale, B.\nScore: 17.8, Published: 2024-01-10 DOI: 10.1101/2024.01.09.574205\nThe Variant Call Format (VCF) is widely used in genome sequencing but scales poorly. For instance, we estimate a 150,000 genome VCF would occupy 900 TiB, making it both costly and complicated to produce and analyze.",
  "keywords": [
    
  ],
  "articleBody": " The Scalable Variant Call Representation: Enabling Genetic Analysis Beyond One Million Genomes\nAuthors: Poterba, T.; Vittal, C.; King, D.; Goldstein, D.; Goldstein, J.; Schultz, P.; Karczewski, K.; Seed, C.; Neale, B.\nScore: 17.8, Published: 2024-01-10 DOI: 10.1101/2024.01.09.574205\nThe Variant Call Format (VCF) is widely used in genome sequencing but scales poorly. For instance, we estimate a 150,000 genome VCF would occupy 900 TiB, making it both costly and complicated to produce and analyze. The issue stems from VCFs requirement to densely represent both reference-genotypes and allele-indexed arrays. These requirements lead to unnecessary data duplication and, ultimately, very large files. To address these challenges, we introduce the Scalable Variant Call Representation (SVCR). This representation reduces file sizes by ensuring they scale linearly with samples. SVCR achieves this by adopting reference blocks from the Genomic Variant Call Format (GVCF) and employing local allele indices. SVCR is also lossless and mergeable, allowing for N+1 and N+K incremental joint-calling. We present two implementations of SVCR: SVCR-VCF, which encodes SVCR in VCF format, and VDS, which uses Hails native format. Our experiments confirm the linear scalability of SVCR-VCF and VDS, in contrast to the super-linear growth seen with standard VCF files. We also discuss the VDS Combiner, a scalable, open-source tool for producing a VDS from GVCFs and unique features of VDS which enable rapid data analysis. SVCR, and VDS in particular, ensure the scientific community can generate, analyze, and disseminate genetics datasets with millions of samples.\nscDirect: key transcription factor identification for directing cell state transitions based on single-cell multi-omics data\nAuthors: Li, C.; Chen, S.; Chen, Y.; Bian, H.; Hao, M.; Wei, L.; Zhang, X.\nScore: 17.4, Published: 2024-01-09 DOI: 10.1101/2024.01.08.574757\nCell state transitions are complicated processes that occur in various life activities. Understanding and artificially manipulating them have been longstanding challenges. Substantial experiments reveal that the transitions could be directed by several key transcription factors (TFs), and some computational methods have been developed to alleviate the burden of biological experiments on identifying key TFs. However, most existing methods employ data with resolution on the cell population, instead of individual cells, which will influence the identification quality due to cell heterogeneity. Besides, they require collecting abundant samples for candidate cell states and are not available for unknown cell states. As for the alternative single-cell analysis methods, they generally concentrate on differentially expressed genes between cell states but can hardly identify key TFs responsible for directing cells state transition. Here we present scDirect, a computational framework to identify key TFs based on single-cell multi-omics data. scDirect models the TF identification task as a linear inverse problem, and solve it with gene regulatory networks enhanced by a graph attention network. Through a benchmarking on a single-cell human embryonic stem cell atlas, we systematically demonstrate the robustness and superiority of scDirect against other alternative single-cell analysis methods on TF identification. With application on various single-cell datasets, scDirect exhibits high capability in identifying key TFs in cell differentiation, somatic cell conversion, and cell reprogramming. Furthermore, scDirect can quantitatively identify experimentally validated reprogramming TF combinations, which indicates the potential of scDirect to guide and assist the experimental design in cellular engineering. We envision that scDirect could utilize rapidly increasing single-cell datasets to identify key TFs for directing cell state transitions, and become an effective tool to facilitate regenerative medicine and therapeutic discovery.\nGenopyc: a python library for investigating the genomic basis of complex diseases\nAuthors: Gualdi, F.; Oliva, B.; Pinero, J.\nScore: 15.2, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575316\nMotivationUnderstanding the genetic basis of complex diseases is a paramount challenge in modern genomics. However, current tools often lack the versatility to efficiently analyze the intricate relation-ships between genetic variations and disease outcomes. To address this, we introduce Genopyc, a novel Python library designed for comprehensive investigation of the genetics underlying complex dis-eases. Genopyc offers an extensive suite of functions for heterogeneous data mining and visualization, enabling researchers to delve into and integrate biological information from large-scale genomic da-tasets with ease. ResultsIn this study, we present the Genopyc library through application to real-world genome wide association studies variants. Using Genopyc to investigate variants associated to intervertebral disc degeneration (IDD) enabled a deeper understanding of the potential dysregulated pathways involved in the disease, which can be explored and visualized by exploiting the functionalities featured in the package. Genopyc emerges as a powerful asset for researchers, fostering advancements in the un-derstanding of complex diseases and thus paving the way for more targeted therapeutic interventions. Availability: Genopyc is available at pip (https://pypi.org/project/genopyc/) and the source code of Genopyc is available at https://github.com/freh-g/genopyc Contactfrancesco.gualdi01@estudiant.upf.edu Supplementary informationsupplementary data are available at Bioinformatics online.\nGenerative artificial intelligence performs rudimentary structural biology modelling\nAuthors: Ille, A. M.; Markosian, C.; Burley, S. K.; Mathews, M. B.; Pasqualini, R.; Arap, W.\nScore: 13.0, Published: 2024-01-13 DOI: 10.1101/2024.01.10.575113\nGenerative artificial intelligence (AI) is being increasingly used by researchers in numerous fields of study, including the biological sciences. To date, the most commonly used tool grounded in this technology has been Chat Generative Pre-trained Transformer (ChatGPT). While ChatGPT is typically applied for natural language text generation, other application modes include coding and mathematical problem-solving. We have recently reported the ability of ChatGPT to interpret the central dogma of molecular biology and the genetic code. Here we explored how ChatGPT performs rudimentary structural biology modelling in order to gain evaluative insight. We prompted ChatGPT to model 3D structures for the 20 standard amino acids as well as an -helical polypeptide chain, with the latter involving incorporation of the Wolfram plugin for advanced mathematical computation. For amino acid modelling, distances and angles between atoms of the generated structures in most cases approximated to around experimentally-determined values. For -helix modelling, the generated structures were comparable to that of an experimentally-determined -helical structure. However, both amino acid and -helix modelling were sporadically error-prone and increased molecular complexity was not well tolerated. Despite current limitations, our findings show the capability of generative AI to perform basic structural biology modelling with atomic-scale accuracy. These results provide a precedent for the potential use of generative AI in structural biology as this technology continues to advance.\nGenome-aware annotation of CRISPR guides validates targets in variant cell lines and enhances discovery in screens\nAuthors: Lam, S.; Thomas, J. C.; Jackson, S. P.\nScore: 11.6, Published: 2024-01-14 DOI: 10.1101/2024.01.14.575203\nPooled CRISPR-Cas9 genetic knockout screens are powerful high-throughput tools for identifying chemo-genetic, synthetic-lethal and synthetic-viability interactions and are used as a key step towards identifying disease-modifying knockout candidates and informing drug design and therapeutic regimens. CRISPR guide libraries are commercially available for purchase and have been widely applied in different cell lines. However, discrepancies between the genomes used to design CRISPR libraries and the genomes of the cells subjected to CRISPR screens lead to loss of signal or introduction of bias towards the most conserved genes. Here, we present an algorithm, EXOme-guided Reannotation of nuCleotIde SEquences (Exorcise), which uses sequence search and CRISPR target annotation to adapt existing CRISPR libraries to user-defined genomes and exomes. Applying Exorcise on 55 commercially available CRISPR-spCas9 knockout libraries for human and mouse, we found that all libraries have mis-annotations, and that design strategy affects off-target effects and targeting accuracy relative to a standard reference sequence. In simulations on synthetic data, we modelled common mis-annotations in CRISPR libraries and found that they adversely affected recovery of the ground truth for all genes except for those with the strongest signals. Finally, we reanalysed DepMap and DDRcs CRISPR screens with Exorcise annotations and found that strong hits were retained, and lower-confidence hits were strengthened. Use of Exorcise on DepMap with exomes inferred from transcriptomic expression data demonstrated that cell-line-aware reannotation is possible without whole-genome sequencing. Taken together, our results show that Exorcise is a powerful reannotation tool that focuses existing CRISPR libraries towards the cell line genome under investigation and allows post-hoc reanalysis of completed CRISPR screens. Exorcise is open-source software licenced under a Creative Commons Zero Universal 1.0 licence and is available at .\nWhen the pen is mightier than the sword: semi-automatic 2 and 3D image labelling\nAuthors: Hollandi, R.; Bauer, D.; Diosdi, A.; Schrettner, B.; Toth, T.; Hirling, D.; Hollandi, G.; Harmati, M.; Molnar, J.; Horvath, P.\nScore: 10.6, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575658\nData is the driving engine of learning-based algorithms, the creation of which fundamentally determines the performance, accuracy, generalizability and quality of any model or method trained on it. When only skilled or trained personnel can create reliable annotations, assisted software solutions are desirable to reduce the time and effort the expert must spend on labelling. Herein is proposed an automated annotation helper software package in napari that offers multiple methods to assist the annotator in creating object-based labels on 2D or 3D images.\nMATES: A Deep Learning-Based Model for Locus-specific Quantification of Transposable Elements in Single Cell\nAuthors: Wang, R.; Zheng, Y.; Zhang, Z.; Zhu, X.; Wu, T. P.; Ding, J.\nScore: 9.8, Published: 2024-01-11 DOI: 10.1101/2024.01.09.574909\nTransposable elements (TEs) are crucial for genetic diversity and gene regulation. Current single-cell quantification methods often align multi-mapping reads to either best-mapped or random-mapped locations and categorize them at sub-family levels, overlooking the biological necessity for accurate, locus-specific TE quantification. Moreover, these existing methods are primarily designed for and focused on transcriptomics data, which restricts their adaptability to single-cell data of other modalities. To address these challenges, here we introduce MATES, a novel deep-learning approach that accurately allocates multi-mapping reads to specific loci of TEs, utilizing context from adjacent read alignments flanking the TE locus. When applied to diverse single-cell omics datasets, MATES shows improved performance over existing methods, enhancing the accuracy of TE quantification and aiding in the identification of marker TEs for identified cell populations. This development enables exploring single-cell heterogeneity and gene regulation through the lens of TEs, offering a transformative tool for the single-cell genomics community.\nBenchmarking computational tools for de novo motif discovery\nAuthors: Simonetti, L.; Ivarsson, Y.; Davey, N. E.\nScore: 8.8, Published: 2024-01-12 DOI: 10.1101/2024.01.12.574168\nSTRUCTURED ABSTRACTO_ST_ABSMotivationC_ST_ABSHigh-throughput experimental methods aimed at uncovering domain-peptide interactions have created numerous use cases for bioinformatic tools capable of detecting short linear motifs (SLiMs). Over the past twenty years, countless motif discovery tools have been introduced. However, the evaluation of these tools generally focus on a single tool and use artificially generated test-sets that fail to capture the authentic context of motifs within proteomic data. Consequently, these evaluations fall short in addressing real-world use cases. ResultsIn the current work, five motif discovery tools and seven general sequence alignment tools were benchmarked on their capacity to find SLiMs in datasets of peptides of varying complexity built from curated SLiM instances from the Eukaryotic Linear Motif database. We explored the effect of dataset size, peptide lengths, background noise levels and motif complexity on the motif discovery of these tools. The main metric used to compare all tools outputs was the percentage of correctly aligned SLiM containing peptides. Two motif discovery tools (DEME and SLiMFinder) and a sequence alignment tool (OPAL) outperformed the rest of the tools when benchmarked with this metric, averaging over 70% correctly aligned motif-containing peptides. The performance of the motif discovery tools and OPAL were not affected by the sizes of the test-sets, while increasing peptide length and noise levels lowered the performances of all tools. For N-/C-terminal motifs all tools performed well, while for those motifs that had low motif complexity only DEME and SLiMFinder returned correctly aligned motifs for [~]50% or more of the datasets. Availabilityhttps://doi.org/10.5281/zenodo.10467208.\nReformer: Deep learning model for characterizing protein-RNA interactions from sequence at single-base resolution\nAuthors: Shen, X.; Li, X.\nScore: 8.5, Published: 2024-01-15 DOI: 10.1101/2024.01.14.575540\nProtein-RNA interactions play an essential role in the regulation of transcription, translation, and metabolism of cellular RNA. Here, we develop Reformer, a deep learning model that predicts protein-RNA binding affinity purely from sequence. We developed Reformer with 155 RNA binding protein (RBP) targets from 3 cell lines. Reformer achieved high prediction accuracy at single-base resolution when tasking with inferring protein- and cell-type-specific binding affinity. We conducted electrophoretic mobility shift assays to validate high-impact RNA regulation mutations predicted by Reformer. In addition, Reformer learned to capture protein binding motifs that cannot be discovered by eCLIP-seq experiments. Furthermore, we demonstrated that motif signatures related to RNA processing functions are encoded within Reformer. In conclusion, Reformer will facilitate interpretation of the regulation mechanisms underlying RNA processing.\nPathFinder: a novel graph transformer model to infer multi-cell intra- and inter-cellular signaling pathways and communications\nAuthors: Feng, J.; Province, M.; Li, G.; Payne, P. R.; Chen, Y.; Li, F.\nScore: 7.5, Published: 2024-01-15 DOI: 10.1101/2024.01.13.575534\nRecently, large-scale scRNA-seq datasets have been generated to understand the complex and poorly understood signaling mechanisms within microenvironment of Alzheimer Disease (AD), which are critical for identifying novel therapeutic targets and precision medicine. Though a set of targets have been identified, however, it remains a challenging to infer the core intra- and inter-multi-cell signaling communication networks using the scRNA-seq data, considering the complex and highly interactive background signaling network. Herein, we introduced a novel graph transformer model, PathFinder, to infer multi-cell intra- and inter-cellular signaling pathways and signaling communications among multi-cell types. Compared with existing models, the novel and unique design of PathFinder is based on the divide-and-conquer strategy, which divides the complex signaling networks into signaling paths, and then score and rank them using a novel graph transformer architecture to infer the intra- and inter-cell signaling communications. We evaluated PathFinder using scRNA-seq data of APOE4-genotype specific AD mice models and identified novel APOE4 altered intra- and inter-cell interaction networks among neurons, astrocytes, and microglia. PathFinder is a general signaling network inference model and can be applied to other omics data-driven signaling network inference.\n",
  "wordCount" : "2271",
  "inLanguage": "en",
  "datePublished": "2024-01-17T10:40:22Z",
  "dateModified": "2024-01-17T10:40:22Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 17, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.574205">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.574205" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.574205">
        <p class="paperTitle">The Scalable Variant Call Representation: Enabling Genetic Analysis Beyond One Million Genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.574205" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.574205" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Poterba, T.; Vittal, C.; King, D.; Goldstein, D.; Goldstein, J.; Schultz, P.; Karczewski, K.; Seed, C.; Neale, B.</p>
        <p class="info">Score: 17.8, Published: 2024-01-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.574205' target='https://doi.org/10.1101/2024.01.09.574205'> 10.1101/2024.01.09.574205</a></p>
        <p class="abstract">The Variant Call Format (VCF) is widely used in genome sequencing but scales poorly. For instance, we estimate a 150,000 genome VCF would occupy 900 TiB, making it both costly and complicated to produce and analyze. The issue stems from VCFs requirement to densely represent both reference-genotypes and allele-indexed arrays. These requirements lead to unnecessary data duplication and, ultimately, very large files.

To address these challenges, we introduce the Scalable Variant Call Representation (SVCR). This representation reduces file sizes by ensuring they scale linearly with samples. SVCR achieves this by adopting reference blocks from the Genomic Variant Call Format (GVCF) and employing local allele indices. SVCR is also lossless and mergeable, allowing for N&#43;1 and N&#43;K incremental joint-calling.

We present two implementations of SVCR: SVCR-VCF, which encodes SVCR in VCF format, and VDS, which uses Hails native format. Our experiments confirm the linear scalability of SVCR-VCF and VDS, in contrast to the super-linear growth seen with standard VCF files. We also discuss the VDS Combiner, a scalable, open-source tool for producing a VDS from GVCFs and unique features of VDS which enable rapid data analysis. SVCR, and VDS in particular, ensure the scientific community can generate, analyze, and disseminate genetics datasets with millions of samples.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.574757">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.574757" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.574757">
        <p class="paperTitle">scDirect: key transcription factor identification for directing cell state transitions based on single-cell multi-omics data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.574757" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.574757" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, C.; Chen, S.; Chen, Y.; Bian, H.; Hao, M.; Wei, L.; Zhang, X.</p>
        <p class="info">Score: 17.4, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.574757' target='https://doi.org/10.1101/2024.01.08.574757'> 10.1101/2024.01.08.574757</a></p>
        <p class="abstract">Cell state transitions are complicated processes that occur in various life activities. Understanding and artificially manipulating them have been longstanding challenges. Substantial experiments reveal that the transitions could be directed by several key transcription factors (TFs), and some computational methods have been developed to alleviate the burden of biological experiments on identifying key TFs. However, most existing methods employ data with resolution on the cell population, instead of individual cells, which will influence the identification quality due to cell heterogeneity. Besides, they require collecting abundant samples for candidate cell states and are not available for unknown cell states. As for the alternative single-cell analysis methods, they generally concentrate on differentially expressed genes between cell states but can hardly identify key TFs responsible for directing cells state transition. Here we present scDirect, a computational framework to identify key TFs based on single-cell multi-omics data. scDirect models the TF identification task as a linear inverse problem, and solve it with gene regulatory networks enhanced by a graph attention network. Through a benchmarking on a single-cell human embryonic stem cell atlas, we systematically demonstrate the robustness and superiority of scDirect against other alternative single-cell analysis methods on TF identification. With application on various single-cell datasets, scDirect exhibits high capability in identifying key TFs in cell differentiation, somatic cell conversion, and cell reprogramming. Furthermore, scDirect can quantitatively identify experimentally validated reprogramming TF combinations, which indicates the potential of scDirect to guide and assist the experimental design in cellular engineering. We envision that scDirect could utilize rapidly increasing single-cell datasets to identify key TFs for directing cell state transitions, and become an effective tool to facilitate regenerative medicine and therapeutic discovery.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575316">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575316" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575316">
        <p class="paperTitle">Genopyc: a python library for investigating the genomic basis of complex diseases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575316" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575316" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gualdi, F.; Oliva, B.; Pinero, J.</p>
        <p class="info">Score: 15.2, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575316' target='https://doi.org/10.1101/2024.01.11.575316'> 10.1101/2024.01.11.575316</a></p>
        <p class="abstract">MotivationUnderstanding the genetic basis of complex diseases is a paramount challenge in modern genomics. However, current tools often lack the versatility to efficiently analyze the intricate relation-ships between genetic variations and disease outcomes. To address this, we introduce Genopyc, a novel Python library designed for comprehensive investigation of the genetics underlying complex dis-eases. Genopyc offers an extensive suite of functions for heterogeneous data mining and visualization, enabling researchers to delve into and integrate biological information from large-scale genomic da-tasets with ease.

ResultsIn this study, we present the Genopyc library through application to real-world genome wide association studies variants. Using Genopyc to investigate variants associated to intervertebral disc degeneration (IDD) enabled a deeper understanding of the potential dysregulated pathways involved in the disease, which can be explored and visualized by exploiting the functionalities featured in the package. Genopyc emerges as a powerful asset for researchers, fostering advancements in the un-derstanding of complex diseases and thus paving the way for more targeted therapeutic interventions. Availability: Genopyc is available at pip (https://pypi.org/project/genopyc/) and the source code of Genopyc is available at https://github.com/freh-g/genopyc

Contactfrancesco.gualdi01@estudiant.upf.edu

Supplementary informationsupplementary data are available at Bioinformatics online.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.575113">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.575113" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.575113">
        <p class="paperTitle">Generative artificial intelligence performs rudimentary structural biology modelling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.575113" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.575113" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ille, A. M.; Markosian, C.; Burley, S. K.; Mathews, M. B.; Pasqualini, R.; Arap, W.</p>
        <p class="info">Score: 13.0, Published: 2024-01-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.575113' target='https://doi.org/10.1101/2024.01.10.575113'> 10.1101/2024.01.10.575113</a></p>
        <p class="abstract">Generative artificial intelligence (AI) is being increasingly used by researchers in numerous fields of study, including the biological sciences. To date, the most commonly used tool grounded in this technology has been Chat Generative Pre-trained Transformer (ChatGPT). While ChatGPT is typically applied for natural language text generation, other application modes include coding and mathematical problem-solving. We have recently reported the ability of ChatGPT to interpret the central dogma of molecular biology and the genetic code. Here we explored how ChatGPT performs rudimentary structural biology modelling in order to gain evaluative insight. We prompted ChatGPT to model 3D structures for the 20 standard amino acids as well as an -helical polypeptide chain, with the latter involving incorporation of the Wolfram plugin for advanced mathematical computation. For amino acid modelling, distances and angles between atoms of the generated structures in most cases approximated to around experimentally-determined values. For -helix modelling, the generated structures were comparable to that of an experimentally-determined -helical structure. However, both amino acid and -helix modelling were sporadically error-prone and increased molecular complexity was not well tolerated. Despite current limitations, our findings show the capability of generative AI to perform basic structural biology modelling with atomic-scale accuracy. These results provide a precedent for the potential use of generative AI in structural biology as this technology continues to advance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.14.575203">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.14.575203" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.14.575203">
        <p class="paperTitle">Genome-aware annotation of CRISPR guides validates targets in variant cell lines and enhances discovery in screens</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.14.575203" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.14.575203" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lam, S.; Thomas, J. C.; Jackson, S. P.</p>
        <p class="info">Score: 11.6, Published: 2024-01-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.14.575203' target='https://doi.org/10.1101/2024.01.14.575203'> 10.1101/2024.01.14.575203</a></p>
        <p class="abstract">Pooled CRISPR-Cas9 genetic knockout screens are powerful high-throughput tools for identifying chemo-genetic, synthetic-lethal and synthetic-viability interactions and are used as a key step towards identifying disease-modifying knockout candidates and informing drug design and therapeutic regimens. CRISPR guide libraries are commercially available for purchase and have been widely applied in different cell lines. However, discrepancies between the genomes used to design CRISPR libraries and the genomes of the cells subjected to CRISPR screens lead to loss of signal or introduction of bias towards the most conserved genes. Here, we present an algorithm, EXOme-guided Reannotation of nuCleotIde SEquences (Exorcise), which uses sequence search and CRISPR target annotation to adapt existing CRISPR libraries to user-defined genomes and exomes. Applying Exorcise on 55 commercially available CRISPR-spCas9 knockout libraries for human and mouse, we found that all libraries have mis-annotations, and that design strategy affects off-target effects and targeting accuracy relative to a standard reference sequence. In simulations on synthetic data, we modelled common mis-annotations in CRISPR libraries and found that they adversely affected recovery of the ground truth for all genes except for those with the strongest signals. Finally, we reanalysed DepMap and DDRcs CRISPR screens with Exorcise annotations and found that strong hits were retained, and lower-confidence hits were strengthened. Use of Exorcise on DepMap with exomes inferred from transcriptomic expression data demonstrated that cell-line-aware reannotation is possible without whole-genome sequencing. Taken together, our results show that Exorcise is a powerful reannotation tool that focuses existing CRISPR libraries towards the cell line genome under investigation and allows post-hoc reanalysis of completed CRISPR screens. Exorcise is open-source software licenced under a Creative Commons Zero Universal 1.0 licence and is available at &lt;https://github.com/SimonLammmm/exorcise&gt;.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575658">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575658" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575658">
        <p class="paperTitle">When the pen is mightier than the sword: semi-automatic 2 and 3D image labelling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575658" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575658" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hollandi, R.; Bauer, D.; Diosdi, A.; Schrettner, B.; Toth, T.; Hirling, D.; Hollandi, G.; Harmati, M.; Molnar, J.; Horvath, P.</p>
        <p class="info">Score: 10.6, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575658' target='https://doi.org/10.1101/2024.01.15.575658'> 10.1101/2024.01.15.575658</a></p>
        <p class="abstract">Data is the driving engine of learning-based algorithms, the creation of which fundamentally determines the performance, accuracy, generalizability and quality of any model or method trained on it. When only skilled or trained personnel can create reliable annotations, assisted software solutions are desirable to reduce the time and effort the expert must spend on labelling. Herein is proposed an automated annotation helper software package in napari that offers multiple methods to assist the annotator in creating object-based labels on 2D or 3D images.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.574909">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.574909" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.574909">
        <p class="paperTitle">MATES: A Deep Learning-Based Model for Locus-specific Quantification of Transposable Elements in Single Cell</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.574909" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.574909" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, R.; Zheng, Y.; Zhang, Z.; Zhu, X.; Wu, T. P.; Ding, J.</p>
        <p class="info">Score: 9.8, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.574909' target='https://doi.org/10.1101/2024.01.09.574909'> 10.1101/2024.01.09.574909</a></p>
        <p class="abstract">Transposable elements (TEs) are crucial for genetic diversity and gene regulation. Current single-cell quantification methods often align multi-mapping reads to either  best-mapped or  random-mapped locations and categorize them at sub-family levels, overlooking the biological necessity for accurate, locus-specific TE quantification. Moreover, these existing methods are primarily designed for and focused on transcriptomics data, which restricts their adaptability to single-cell data of other modalities. To address these challenges, here we introduce MATES, a novel deep-learning approach that accurately allocates multi-mapping reads to specific loci of TEs, utilizing context from adjacent read alignments flanking the TE locus. When applied to diverse single-cell omics datasets, MATES shows improved performance over existing methods, enhancing the accuracy of TE quantification and aiding in the identification of marker TEs for identified cell populations. This development enables exploring single-cell heterogeneity and gene regulation through the lens of TEs, offering a transformative tool for the single-cell genomics community.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.574168">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.574168" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.574168">
        <p class="paperTitle">Benchmarking computational tools for de novo motif discovery</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.574168" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.574168" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Simonetti, L.; Ivarsson, Y.; Davey, N. E.</p>
        <p class="info">Score: 8.8, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.574168' target='https://doi.org/10.1101/2024.01.12.574168'> 10.1101/2024.01.12.574168</a></p>
        <p class="abstract">STRUCTURED ABSTRACTO_ST_ABSMotivationC_ST_ABSHigh-throughput experimental methods aimed at uncovering domain-peptide interactions have created numerous use cases for bioinformatic tools capable of detecting short linear motifs (SLiMs). Over the past twenty years, countless motif discovery tools have been introduced. However, the evaluation of these tools generally focus on a single tool and use artificially generated test-sets that fail to capture the authentic context of motifs within proteomic data. Consequently, these evaluations fall short in addressing real-world use cases.

ResultsIn the current work, five motif discovery tools and seven general sequence alignment tools were benchmarked on their capacity to find SLiMs in datasets of peptides of varying complexity built from curated SLiM instances from the Eukaryotic Linear Motif database. We explored the effect of dataset size, peptide lengths, background noise levels and motif complexity on the motif discovery of these tools. The main metric used to compare all tools outputs was the percentage of correctly aligned SLiM containing peptides. Two motif discovery tools (DEME and SLiMFinder) and a sequence alignment tool (OPAL) outperformed the rest of the tools when benchmarked with this metric, averaging over 70% correctly aligned motif-containing peptides. The performance of the motif discovery tools and OPAL were not affected by the sizes of the test-sets, while increasing peptide length and noise levels lowered the performances of all tools. For N-/C-terminal motifs all tools performed well, while for those motifs that had low motif complexity only DEME and SLiMFinder returned correctly aligned motifs for [~]50% or more of the datasets.

Availabilityhttps://doi.org/10.5281/zenodo.10467208.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.14.575540">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.14.575540" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.14.575540">
        <p class="paperTitle">Reformer: Deep learning model for characterizing protein-RNA interactions from sequence at single-base resolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.14.575540" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.14.575540" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shen, X.; Li, X.</p>
        <p class="info">Score: 8.5, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.14.575540' target='https://doi.org/10.1101/2024.01.14.575540'> 10.1101/2024.01.14.575540</a></p>
        <p class="abstract">Protein-RNA interactions play an essential role in the regulation of transcription, translation, and metabolism of cellular RNA. Here, we develop Reformer, a deep learning model that predicts protein-RNA binding affinity purely from sequence. We developed Reformer with 155 RNA binding protein (RBP) targets from 3 cell lines. Reformer achieved high prediction accuracy at single-base resolution when tasking with inferring protein- and cell-type-specific binding affinity. We conducted electrophoretic mobility shift assays to validate high-impact RNA regulation mutations predicted by Reformer. In addition, Reformer learned to capture protein binding motifs that cannot be discovered by eCLIP-seq experiments. Furthermore, we demonstrated that motif signatures related to RNA processing functions are encoded within Reformer. In conclusion, Reformer will facilitate interpretation of the regulation mechanisms underlying RNA processing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.13.575534">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.13.575534" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.13.575534">
        <p class="paperTitle">PathFinder: a novel graph transformer model to infer multi-cell intra- and inter-cellular signaling pathways and communications</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.13.575534" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.13.575534" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Feng, J.; Province, M.; Li, G.; Payne, P. R.; Chen, Y.; Li, F.</p>
        <p class="info">Score: 7.5, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.13.575534' target='https://doi.org/10.1101/2024.01.13.575534'> 10.1101/2024.01.13.575534</a></p>
        <p class="abstract">Recently, large-scale scRNA-seq datasets have been generated to understand the complex and poorly understood signaling mechanisms within microenvironment of Alzheimer Disease (AD), which are critical for identifying novel therapeutic targets and precision medicine. Though a set of targets have been identified, however, it remains a challenging to infer the core intra- and inter-multi-cell signaling communication networks using the scRNA-seq data, considering the complex and highly interactive background signaling network. Herein, we introduced a novel graph transformer model, PathFinder, to infer multi-cell intra- and inter-cellular signaling pathways and signaling communications among multi-cell types. Compared with existing models, the novel and unique design of PathFinder is based on the divide-and-conquer strategy, which divides the complex signaling networks into signaling paths, and then score and rank them using a novel graph transformer architecture to infer the intra- and inter-cell signaling communications. We evaluated PathFinder using scRNA-seq data of APOE4-genotype specific AD mice models and identified novel APOE4 altered intra- and inter-cell interaction networks among neurons, astrocytes, and microglia. PathFinder is a general signaling network inference model and can be applied to other omics data-driven signaling network inference.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
