<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A new paradigm for biological sequence retrieval inspired by natural language processing and database research
Authors: Rousseau, A.-J.; Lemal, S.; Korovin, Y.; Triantopoulos, G.; Brands, I.; Biemans, M.; Van Hyfte, D.; Valkenborg, D.
Score: 37.7, Published: 2023-11-09 DOI: 10.1101/2023.11.07.565984
Nearly-exponential growth and heterogeneity of biological sequence data make the task of biological sequence retrieval from databases more important and challenging than ever. In this manuscript, we present a novel search algorithm involving an indexing scheme based on patterns discovered by natural language processing, i.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="A new paradigm for biological sequence retrieval inspired by natural language processing and database research
Authors: Rousseau, A.-J.; Lemal, S.; Korovin, Y.; Triantopoulos, G.; Brands, I.; Biemans, M.; Van Hyfte, D.; Valkenborg, D.
Score: 37.7, Published: 2023-11-09 DOI: 10.1101/2023.11.07.565984
Nearly-exponential growth and heterogeneity of biological sequence data make the task of biological sequence retrieval from databases more important and challenging than ever. In this manuscript, we present a novel search algorithm involving an indexing scheme based on patterns discovered by natural language processing, i." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-15T10:40:26+00:00" />
<meta property="article:modified_time" content="2023-11-15T10:40:26+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="A new paradigm for biological sequence retrieval inspired by natural language processing and database research
Authors: Rousseau, A.-J.; Lemal, S.; Korovin, Y.; Triantopoulos, G.; Brands, I.; Biemans, M.; Van Hyfte, D.; Valkenborg, D.
Score: 37.7, Published: 2023-11-09 DOI: 10.1101/2023.11.07.565984
Nearly-exponential growth and heterogeneity of biological sequence data make the task of biological sequence retrieval from databases more important and challenging than ever. In this manuscript, we present a novel search algorithm involving an indexing scheme based on patterns discovered by natural language processing, i."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "A new paradigm for biological sequence retrieval inspired by natural language processing and database research\nAuthors: Rousseau, A.-J.; Lemal, S.; Korovin, Y.; Triantopoulos, G.; Brands, I.; Biemans, M.; Van Hyfte, D.; Valkenborg, D.\nScore: 37.7, Published: 2023-11-09 DOI: 10.1101/2023.11.07.565984\nNearly-exponential growth and heterogeneity of biological sequence data make the task of biological sequence retrieval from databases more important and challenging than ever. In this manuscript, we present a novel search algorithm involving an indexing scheme based on patterns discovered by natural language processing, i.",
  "keywords": [
    
  ],
  "articleBody": " A new paradigm for biological sequence retrieval inspired by natural language processing and database research\nAuthors: Rousseau, A.-J.; Lemal, S.; Korovin, Y.; Triantopoulos, G.; Brands, I.; Biemans, M.; Van Hyfte, D.; Valkenborg, D.\nScore: 37.7, Published: 2023-11-09 DOI: 10.1101/2023.11.07.565984\nNearly-exponential growth and heterogeneity of biological sequence data make the task of biological sequence retrieval from databases more important and challenging than ever. In this manuscript, we present a novel search algorithm involving an indexing scheme based on patterns discovered by natural language processing, i.e., short strings of nucleotides or amino acids, akin to standard k-mers, but mined from cumulative cross-species omic data repositories. More specifically, we benchmark the quality of the sequence retrieval process by comparing to BLASTP, a heuristic algorithm for the alignment of genomics or protein sequence data. The main argumentation is that to retrieve biological similar sequences it is not needed to mimic the alignment procedures as it is performed by BLAST. Our results suggests that the HYFT-indexing and searching is a good alternative and a static, alignment-free method to retrieve homologous sequence down to 50% sequence identity.\nTelomere-to-telomere assembly by preserving contained reads\nAuthors: Kamath, S. S.; Bindra, M.; Pal, D.; Jain, C.\nScore: 26.6, Published: 2023-11-11 DOI: 10.1101/2023.11.07.565066\nAutomated telomere-to-telomere (T2T) de novo assembly of diploid and polyploid genomes remains a formidable task. The string graph (Myers, JCB 1995) is the standard representation of graphs in overlap-based assembly algorithms. The string graph formulation employs graph simplification heuristics, which drastically reduce the count of vertices and edges in a read-overlap graph. One of these heuristics in-volves removing the reads contained in longer reads. However, this procedure is not guaranteed to be safe. In practice, it occasionally introduces breaks in the graph by removing all reads that cover one or more genome intervals. What factors contribute to such coverage gap events remains poorly understood. In this work, we mathematically derived the probability of a coverage gap near germline and somatic heterozygous variant loci. Our analysis shows that (i) a coverage gap event is two orders of magnitude more likely for Oxford Nanopore reads than PacBio HiFi reads due to differences in their read-length distributions, and (ii) increasing the sequencing depth reduces the probability at an exponential rate. Next, we addressed the weakness of the string graph formulation by developing the RAFT assembly algorithm. RAFT addresses the issue of contained reads by fragmenting reads and producing a more uniform read-length distribution. The algorithm retains spanned repeats in the reads during the fragmentation. We empirically demonstrate that RAFT drastically reduces the number of coverage gaps. Using Nanopore Duplex and PacBio HiFi datasets of the HG002 human genome, we achieved a twofold increase in the assembly NG50 and the number of haplotype-resolved T2T contigs compared to Hifiasm.\nOn the limits of 16S rRNA gene-based metagenome prediction and functional profiling\nAuthors: Matchado, M. S.; Ruhlemann, M. C.; Reimeiter, S.; Kacprowski, T.; Frost, F.; Haller, D.; Baumbach, J.; List, M.\nScore: 21.1, Published: 2023-11-08 DOI: 10.1101/2023.11.07.564315\nMolecular profiling techniques such as metagenomics, metatranscriptomics or metabolomics offer important insights into the functional diversity of the microbiome. In contrast, 16S rRNA gene sequencing, a widespread and cost-effective technique to measure microbial diversity, only allows for indirect estimation of microbial function. To mitigate this, tools such as PICRUSt2, Tax4Fun2, PanFP and MetGEM infer functional profiles from 16S rRNA gene sequencing data using different algorithms. Prior studies have cast doubts on the quality of these predictions, motivating us to systematically evaluate these tools using matched 16S rRNA gene sequencing, metagenomic datasets, and simulated data. Our contribution is threefold: (i) using simulated data, we investigate if technical biases could explain the discordance between inferred and expected results; (ii) considering human cohorts for type 2 diabetes, colorectal cancer and obesity, we test if health-related differential abundance measures of functional categories are concordant between 16S rRNA gene-inferred and metagenome-derived profiles and; (iii) since 16S rRNA gene copy number is an important confounder in functional profiles inference, we investigate if a customised copy number normalisation with the rrnDB database could improve the results. Our results show that 16S rRNA gene-based functional inference tools generally do not have the necessary sensitivity to delineate health-related functional changes in the microbiome and should thus be used with care. Furthermore, we outline important differences in the individual tools tested and offer recommendations for tool selection.\nBlood transcriptomics analysis offers insights into variant-specific immune response to SARS-CoV-2\nAuthors: Hoffmann, M.; Willruth, L.-L.; Dietrich, A.; Lee, H. K.; Knabl, L.; Trummer, N.; Baumbach, J.; Furth, P. A.; Hennighausen, L.; List, M.\nScore: 16.0, Published: 2023-11-06 DOI: 10.1101/2023.11.03.564190\nBulk RNA sequencing (RNA-seq) of blood is typically used for gene expression analysis in biomedical research but is still rarely used in clinical practice. In this study, we argue that RNA-seq should be considered a routine diagnostic tool, as it offers not only insights into aberrant gene expression and splicing but also delivers additional readouts on immune cell type composition as well as B-cell and T-cell receptor (BCR/TCR) repertoires. We demonstrate that RNA-seq offers vital insights into a patients immune status via integrative analysis of RNA-seq data from patients infected with various SARS-CoV-2 variants (in total 240 samples with up to 200 million reads sequencing depth). We compare the results of computational cell-type deconvolution methods (e.g., MCP-counter, xCell, EPIC, quanTIseq) to complete blood count data, the current gold standard in clinical practice. We observe varying levels of lymphocyte depletion and significant differences in neutrophil levels between SARS-CoV-2 variants. Additionally, we identify B and T cell receptor (BCR/TCR) sequences using the tools MiXCR and TRUST4 to show that - combined with sequence alignments and pBLAST - they could be used to classify a patients disease. Finally, we investigated the sequencing depth required for such analyses and concluded that 10 million reads per sample is sufficient. In conclusion, our study reveals that computational cell-type deconvolution and BCR/TCR methods using bulk RNA-seq analyses can supplement missing CBC data and offer insights into immune responses, disease severity, and pathogen-specific immunity, all achievable with a sequencing depth of 10 million reads per sample. Key PointsO_LIComputational deconvolution of transcriptomes can estimate immune cell abundances in SARS-CoV-2 patients, supplementing missing CBC data. C_LIO_LI10 million RNA sequencing reads per sample suffice for analyzing immune responses and disease severity, including BCR/TCR identification. C_LI\nStructure prediction of protein-ligand complexes from sequence information with Umol\nAuthors: Bryant, P.; Kelkar, A.; Guljas, A.; Clementi, C.; Noe, F.\nScore: 98.2, Published: 2023-11-05 DOI: 10.1101/2023.11.03.565471\nProtein-ligand docking is an established tool in drug discovery and development to narrow down potential therapeutics for experimental testing. However, a high-quality protein structure is required and often the protein is treated as fully or partially rigid. Here we develop an AI system that can predict the fully flexible all-atom structure of protein-ligand complexes directly, given a multiple sequence alignment representation of the protein and a SMILES string representing the ligand. At a high accuracy threshold, unseen protein-ligand complexes can be predicted more accurately than for RoseTTAFold-AA, and at medium accuracy even classical docking methods that use known protein structures as input are surpassed. The high accuracy presented here suggests that the goal of AI-based drug discovery is one step closer, but there is still a way to go to fully grasp the complexity of protein-ligand interactions. Umol is available at: https://github.com/patrickbryant1/Umol\nNavigating the manifold of single-cell gene coexpression to discover interpretable gene programs\nAuthors: Ozbay, S.; Parekh, A.; Singh, R.\nScore: 15.0, Published: 2023-11-11 DOI: 10.1101/2023.11.09.566448\nDuring development and differentiation, transcriptional regulation in the cell often occurs at the level of gene programs (i.e., sets of co-varying genes), rather than isolated genes. It is therefore crucial to identify differential program expression over time, or across case-vs-control samples. However, this has remained difficult: gene programs are inferred by analyzing gene coexpression, and mathematical operations on the latter are nontrivial. Gene coexpression is quantified as a symmetric positive-definite matrix, on which even basic quantities such as arithmetic differences are neither mathematically sound nor biologically interpretable. Here we exploit the structure of the Riemannian manifold of gene coexpression matrices to propose a novel abstraction of gene coexpression that is mathematically well-founded while being computationally tractable and statistically rigorous. Importantly, it also captures biological similarity better than standard coexpression. This conceptual advance enables us to introduce Sceodesic, an algorithm that invokes the log-Euclidean metric from differential geometry to quantify coexpression patterns specific to each cell state, and organizes them into a study-wide panel of interpretable gene programs. Applied to nine single-cell RNA-seq datasets, Sceodesic outperforms existing methods in early detection of cell fate commitment by leveraging differential expression of gene programs, and is also effective in discovering disease-linked programs in multi-sample studies. By respecting the manifold of gene coexpression matrices, Sceodesic resolves a longstanding challenge in relating biological variability to statistical analyses of single-cell RNA-seq data and enables the discovery of gene programs driving differentiation and disease. Software availabilityhttps://singhlab.net/Sceodesic\nPredmoter - Cross-species prediction of plant promoter and enhancer regions\nAuthors: Kindel, F.; Triesch, S.; Schlueter, U.; Randarevitch, L. A.; Reichel-Deland, V.; Weber, A.; Denton, A. K.\nScore: 44.6, Published: 2023-11-05 DOI: 10.1101/2023.11.03.565452\nMotivationThe identification of cis-regulatory elements (CREs) is crucial for the analysis of gene regulatory networks in plants. Several next generation sequencing (NGS)-based methods were developed to identify CREs. However, these methods can be time-consuming and costly. They also involve creating sequencing libraries for the entire genome. Since many research efforts only focus on specific genomic loci, this presents a considerable expenditure. Computational prediction of the outputs of specialized NGS methods to analyze CREs, like Assay for Transposase Accessible Chromatin using sequencing (ATAC-seq), would significantly cut costs and time investment. Yet, no such method is available to date. ResultsWe present Predmoter, a deep neural network able to predict base-wise ATAC-seq and histone Chromatin immunoprecipitation DNA-sequencing (ChIP-seq) read coverage for plant genomes. Predmoter uses only the DNA sequence as input. We evaluated our model on two plant genomes, the genome of the dicot Arabidopsis thaliana and of the monocot Oryza sativa. We trained our models on 10 species with publicly available ATAC-seq data and 15 species with ChIP-seq data. Our best models showed accurate predictions in peak positions and the overall pattern of peaks for ATAC- and Histone H3 trimethylated at lysine 4 (H3K4me3) ChIP-seq. Annotating putatively accessible chromatin regions provides valuable input for the identification of CREs. In conjunction with other in silico data, such as predicted binding affinities for transcription factors (TFs), this can significantly narrow down the search space to a manageable number of experimentally verifiable DNA-protein interaction pairs. Availability and ImplementationThe source code for Predmoter is available at: https://github.com/weberlab-hhu/Predmoter along with documentation for installation and usage. Predmoter uses a single-command inference, Predmoter.py, for both training and prediction. Predmoter takes a fasta file as input and outputs an h5 file and optionally bigWig and bedGraph files. HighlightPredmoter will help identifying CREs and so gaining further insight into gene regulatory networks in plants.\nCellChat for systematic analysis of cell-cell communication from single-cell and spatially resolved transcriptomics\nAuthors: Jin, S.; Plikus, M. V.; Nie, Q.\nScore: 57.6, Published: 2023-11-05 DOI: 10.1101/2023.11.05.565674\nRecent advances in single-cell sequencing technologies offer an opportunity to explore cell-cell communication in tissues systematically and with reduced bias. A key challenge is the integration between known molecular interactions and measurements into a framework to identify and analyze complex cell-cell communication networks. Previously, we developed a computational tool, named CellChat that infers and analyzes cell-cell communication networks from single-cell RNA-sequencing (scRNA-seq) data within an easily interpretable framework. CellChat quantifies the signaling communication probability between two cell groups using a simplified mass action-based model, which incorporates the core interaction between ligands and receptors with multi-subunit structure along with modulation by cofactors. CellChat v2 is an updated version that includes direct incorporation of spatial locations of cells, if available, to infer spatially proximal cell-cell communication, additional comparison functionalities, expanded database of ligand-receptor pairs along with rich annotations, and an Interactive CellChat Explorer. Here we provide a step-by-step protocol for using CellChat v2 that can be used for both scRNA-seq and spatially resolved transcriptomic data, including inference and analysis of cell-cell communication from one dataset and identification of altered signaling across different datasets. The key steps of applying CellChat v2 to spatially resolved transcriptomics are described in detail. The R implementation of CellChat v2 toolkit and tutorials with the graphic outputs are available at https://github.com/jinworks/CellChat. This protocol typically takes around 20 minutes, and no specialized prior bioinformatics training is required to complete the task.\nMiRGraph: A transformer-based feature learning approach to identify miRNA-target interactions\nAuthors: Liu, P.; Liu, Y.; Luo, J.; Li, Y.\nScore: 13.2, Published: 2023-11-06 DOI: 10.1101/2023.11.04.565620\nMicroRNAs (miRNAs) play a crucial role in the regulation of gene expression by targeting specific mRNAs, and they can function as both tumor suppressors and oncogenes depending on the specific miRNA and its target genes. Detecting miRNA-target interactions (MTIs) is critical for unraveling the complex mechanisms of gene regulation, the potential therapeutic targets and diagnostic markers. There are currently few studies on MTIs prediction that simultaneously perform feature learning on heterogeneous graph network and sequence information. Thus, we present a transformer-based multiview feature learning method, named MiRGraph, capable of modeling both heterogeneous network and sequence features to improve the prediction performance of MTIs. To alleviate the problem of partial missing sequence information when using k-mer encoding or candidate target sites (CTSs), we utilize the mature miRNA sequence and the complete 3UTR sequence of the target mRNAs to encode seqeunce features. And a transformer-based CNN (TransCNN) module is designed for miRNAs and genes respectively to extract their personalized sequence features. Then a heterogeneous graph transformer (HGT) module is adopted to learn the network features through extracting the relational and structural information in a heterogeneous graph consisting of miRNA-miRNA, gene-gene and miRNA-target interactions. Finally, we utilize a multilayer perceptron (MLP) to map the learned features of miRNAs and genes into a same space, and a bilinear function to calculate the prediction scores of MTIs. Comparisons with other existing MTIs prediction methods illustrate the superiority of MiRGraph under multiple standard criteria metrics. The comparing results of distinguishing the high functional MTIs and case studies on identifying the novel MTIs showcase the real-world applications of MiRGraph in biological research. AvailabilityMiRGraph is implemented in python and available at https://github.com/Liangyushi/MiRGraph.\nPre-processing of paleogenomes: Mitigating reference bias and postmortem damage in ancient genome data\nAuthors: Koptekin, D.; Yapar, E.; Vural, K. B.; Saglıcan, E.; Altınısık, N. E.; Malaspinas, A.-S.; Alkan, C.; Somel, M.\nScore: 10.9, Published: 2023-11-13 DOI: 10.1101/2023.11.11.566695\nAncient DNA analysis is subject to various technical challenges, including bias towards the reference allele (\"reference bias\"), postmortem damage (PMD) that confounds real variants, and limited coverage. Here, we conduct a systematic comparison of alternative approaches against reference bias and against PMD. To reduce reference bias, we either (a) mask variable sites before alignment or (b) align the data to a graph genome representing all variable sites. Compared to alignment to the linear reference genome, both masking and graph alignment effectively remove allelic bias when using simulated or real ancient human genome data, but only if sequencing data is available in FASTQ or unfiltered BAM format. Reference bias remains indelible in quality-filtered BAM files and in 1240K-capture data. We next study three approaches to overcome postmortem damage: (a) trimming, (b) rescaling base qualities, and (c) a new algorithm we present here, bamRefine, which masks only PMD-vulnerable polymorphic sites. We find that bamRefine is optimal in increasing the number of genotyped loci up to 20% compared to trimming and in improving accuracy compared to rescaling. We propose graph alignment coupled with bamRefine to minimise data loss and bias. We also urge the paleogenomics community to publish FASTQ files.\n",
  "wordCount" : "2607",
  "inLanguage": "en",
  "datePublished": "2023-11-15T10:40:26Z",
  "dateModified": "2023-11-15T10:40:26Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 15, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.565984">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.565984" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.565984">
        <p class="paperTitle">A new paradigm for biological sequence retrieval inspired by natural language processing and database research</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.565984" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.565984" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rousseau, A.-J.; Lemal, S.; Korovin, Y.; Triantopoulos, G.; Brands, I.; Biemans, M.; Van Hyfte, D.; Valkenborg, D.</p>
        <p class="info">Score: 37.7, Published: 2023-11-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.565984' target='https://doi.org/10.1101/2023.11.07.565984'> 10.1101/2023.11.07.565984</a></p>
        <p class="abstract">Nearly-exponential growth and heterogeneity of biological sequence data make the task of biological sequence retrieval from databases more important and challenging than ever. In this manuscript, we present a novel search algorithm involving an indexing scheme based on patterns discovered by natural language processing, i.e., short strings of nucleotides or amino acids, akin to standard k-mers, but mined from cumulative cross-species omic data repositories. More specifically, we benchmark the quality of the sequence retrieval process by comparing to BLASTP, a heuristic algorithm for the alignment of genomics or protein sequence data. The main argumentation is that to retrieve biological similar sequences it is not needed to mimic the alignment procedures as it is performed by BLAST. Our results suggests that the HYFT-indexing and searching is a good alternative and a static, alignment-free method to retrieve homologous sequence down to 50% sequence identity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.565066">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.565066" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.565066">
        <p class="paperTitle">Telomere-to-telomere assembly by preserving contained reads</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.565066" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.565066" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kamath, S. S.; Bindra, M.; Pal, D.; Jain, C.</p>
        <p class="info">Score: 26.6, Published: 2023-11-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.565066' target='https://doi.org/10.1101/2023.11.07.565066'> 10.1101/2023.11.07.565066</a></p>
        <p class="abstract">Automated telomere-to-telomere (T2T) de novo assembly of diploid and polyploid genomes remains a formidable task. The string graph (Myers, JCB 1995) is the standard representation of graphs in overlap-based assembly algorithms. The string graph formulation employs graph simplification heuristics, which drastically reduce the count of vertices and edges in a read-overlap graph. One of these heuristics in-volves removing the reads contained in longer reads. However, this procedure is not guaranteed to be safe. In practice, it occasionally introduces breaks in the graph by removing all reads that cover one or more genome intervals. What factors contribute to such coverage gap events remains poorly understood. In this work, we mathematically derived the probability of a coverage gap near germline and somatic heterozygous variant loci. Our analysis shows that (i) a coverage gap event is two orders of magnitude more likely for Oxford Nanopore reads than PacBio HiFi reads due to differences in their read-length distributions, and (ii) increasing the sequencing depth reduces the probability at an exponential rate. Next, we addressed the weakness of the string graph formulation by developing the RAFT assembly algorithm. RAFT addresses the issue of contained reads by fragmenting reads and producing a more uniform read-length distribution. The algorithm retains spanned repeats in the reads during the fragmentation. We empirically demonstrate that RAFT drastically reduces the number of coverage gaps. Using Nanopore Duplex and PacBio HiFi datasets of the HG002 human genome, we achieved a twofold increase in the assembly NG50 and the number of haplotype-resolved T2T contigs compared to Hifiasm.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.564315">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.564315" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.564315">
        <p class="paperTitle">On the limits of 16S rRNA gene-based metagenome prediction and functional profiling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.564315" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.564315" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Matchado, M. S.; Ruhlemann, M. C.; Reimeiter, S.; Kacprowski, T.; Frost, F.; Haller, D.; Baumbach, J.; List, M.</p>
        <p class="info">Score: 21.1, Published: 2023-11-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.564315' target='https://doi.org/10.1101/2023.11.07.564315'> 10.1101/2023.11.07.564315</a></p>
        <p class="abstract">Molecular profiling techniques such as metagenomics, metatranscriptomics or metabolomics offer important insights into the functional diversity of the microbiome. In contrast, 16S rRNA gene sequencing, a widespread and cost-effective technique to measure microbial diversity, only allows for indirect estimation of microbial function. To mitigate this, tools such as PICRUSt2, Tax4Fun2, PanFP and MetGEM infer functional profiles from 16S rRNA gene sequencing data using different algorithms. Prior studies have cast doubts on the quality of these predictions, motivating us to systematically evaluate these tools using matched 16S rRNA gene sequencing, metagenomic datasets, and simulated data. Our contribution is threefold: (i) using simulated data, we investigate if technical biases could explain the discordance between inferred and expected results; (ii) considering human cohorts for type 2 diabetes, colorectal cancer and obesity, we test if health-related differential abundance measures of functional categories are concordant between 16S rRNA gene-inferred and metagenome-derived profiles and; (iii) since 16S rRNA gene copy number is an important confounder in functional profiles inference, we investigate if a customised copy number normalisation with the rrnDB database could improve the results. Our results show that 16S rRNA gene-based functional inference tools generally do not have the necessary sensitivity to delineate health-related functional changes in the microbiome and should thus be used with care. Furthermore, we outline important differences in the individual tools tested and offer recommendations for tool selection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.03.564190">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.03.564190" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.03.564190">
        <p class="paperTitle">Blood transcriptomics analysis offers insights into variant-specific immune response to SARS-CoV-2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.03.564190" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.03.564190" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hoffmann, M.; Willruth, L.-L.; Dietrich, A.; Lee, H. K.; Knabl, L.; Trummer, N.; Baumbach, J.; Furth, P. A.; Hennighausen, L.; List, M.</p>
        <p class="info">Score: 16.0, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.03.564190' target='https://doi.org/10.1101/2023.11.03.564190'> 10.1101/2023.11.03.564190</a></p>
        <p class="abstract">Bulk RNA sequencing (RNA-seq) of blood is typically used for gene expression analysis in biomedical research but is still rarely used in clinical practice. In this study, we argue that RNA-seq should be considered a routine diagnostic tool, as it offers not only insights into aberrant gene expression and splicing but also delivers additional readouts on immune cell type composition as well as B-cell and T-cell receptor (BCR/TCR) repertoires. We demonstrate that RNA-seq offers vital insights into a patients immune status via integrative analysis of RNA-seq data from patients infected with various SARS-CoV-2 variants (in total 240 samples with up to 200 million reads sequencing depth). We compare the results of computational cell-type deconvolution methods (e.g., MCP-counter, xCell, EPIC, quanTIseq) to complete blood count data, the current gold standard in clinical practice. We observe varying levels of lymphocyte depletion and significant differences in neutrophil levels between SARS-CoV-2 variants. Additionally, we identify B and T cell receptor (BCR/TCR) sequences using the tools MiXCR and TRUST4 to show that - combined with sequence alignments and pBLAST - they could be used to classify a patients disease. Finally, we investigated the sequencing depth required for such analyses and concluded that 10 million reads per sample is sufficient. In conclusion, our study reveals that computational cell-type deconvolution and BCR/TCR methods using bulk RNA-seq analyses can supplement missing CBC data and offer insights into immune responses, disease severity, and pathogen-specific immunity, all achievable with a sequencing depth of 10 million reads per sample.

Key PointsO_LIComputational deconvolution of transcriptomes can estimate immune cell abundances in SARS-CoV-2 patients, supplementing missing CBC data.
C_LIO_LI10 million RNA sequencing reads per sample suffice for analyzing immune responses and disease severity, including BCR/TCR identification.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.03.565471">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.03.565471" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.03.565471">
        <p class="paperTitle">Structure prediction of protein-ligand complexes from sequence information with Umol</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.03.565471" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.03.565471" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bryant, P.; Kelkar, A.; Guljas, A.; Clementi, C.; Noe, F.</p>
        <p class="info">Score: 98.2, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.03.565471' target='https://doi.org/10.1101/2023.11.03.565471'> 10.1101/2023.11.03.565471</a></p>
        <p class="abstract">Protein-ligand docking is an established tool in drug discovery and development to narrow down potential therapeutics for experimental testing. However, a high-quality protein structure is required and often the protein is treated as fully or partially rigid. Here we develop an AI system that can predict the fully flexible all-atom structure of protein-ligand complexes directly, given a multiple sequence alignment representation of the protein and a SMILES string representing the ligand. At a high accuracy threshold, unseen protein-ligand complexes can be predicted more accurately than for RoseTTAFold-AA, and at medium accuracy even classical docking methods that use known protein structures as input are surpassed. The high accuracy presented here suggests that the goal of AI-based drug discovery is one step closer, but there is still a way to go to fully grasp the complexity of protein-ligand interactions. Umol is available at: https://github.com/patrickbryant1/Umol</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.566448">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.566448" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.566448">
        <p class="paperTitle">Navigating the manifold of single-cell gene coexpression to discover interpretable gene programs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.566448" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.566448" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ozbay, S.; Parekh, A.; Singh, R.</p>
        <p class="info">Score: 15.0, Published: 2023-11-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.566448' target='https://doi.org/10.1101/2023.11.09.566448'> 10.1101/2023.11.09.566448</a></p>
        <p class="abstract">During development and differentiation, transcriptional regulation in the cell often occurs at the level of gene programs (i.e., sets of co-varying genes), rather than isolated genes. It is therefore crucial to identify differential program expression over time, or across case-vs-control samples. However, this has remained difficult: gene programs are inferred by analyzing gene coexpression, and mathematical operations on the latter are nontrivial. Gene coexpression is quantified as a symmetric positive-definite matrix, on which even basic quantities such as arithmetic differences are neither mathematically sound nor biologically interpretable. Here we exploit the structure of the Riemannian manifold of gene coexpression matrices to propose a novel abstraction of gene coexpression that is mathematically well-founded while being computationally tractable and statistically rigorous. Importantly, it also captures biological similarity better than standard coexpression. This conceptual advance enables us to introduce Sceodesic, an algorithm that invokes the log-Euclidean metric from differential geometry to quantify coexpression patterns specific to each cell state, and organizes them into a study-wide panel of interpretable gene programs. Applied to nine single-cell RNA-seq datasets, Sceodesic outperforms existing methods in early detection of cell fate commitment by leveraging differential expression of gene programs, and is also effective in discovering disease-linked programs in multi-sample studies. By respecting the manifold of gene coexpression matrices, Sceodesic resolves a longstanding challenge in relating biological variability to statistical analyses of single-cell RNA-seq data and enables the discovery of gene programs driving differentiation and disease.

Software availabilityhttps://singhlab.net/Sceodesic</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.03.565452">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.03.565452" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.03.565452">
        <p class="paperTitle">Predmoter - Cross-species prediction of plant promoter and enhancer regions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.03.565452" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.03.565452" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kindel, F.; Triesch, S.; Schlueter, U.; Randarevitch, L. A.; Reichel-Deland, V.; Weber, A.; Denton, A. K.</p>
        <p class="info">Score: 44.6, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.03.565452' target='https://doi.org/10.1101/2023.11.03.565452'> 10.1101/2023.11.03.565452</a></p>
        <p class="abstract">MotivationThe identification of cis-regulatory elements (CREs) is crucial for the analysis of gene regulatory networks in plants. Several next generation sequencing (NGS)-based methods were developed to identify CREs. However, these methods can be time-consuming and costly. They also involve creating sequencing libraries for the entire genome. Since many research efforts only focus on specific genomic loci, this presents a considerable expenditure. Computational prediction of the outputs of specialized NGS methods to analyze CREs, like Assay for Transposase Accessible Chromatin using sequencing (ATAC-seq), would significantly cut costs and time investment. Yet, no such method is available to date.

ResultsWe present Predmoter, a deep neural network able to predict base-wise ATAC-seq and histone Chromatin immunoprecipitation DNA-sequencing (ChIP-seq) read coverage for plant genomes. Predmoter uses only the DNA sequence as input. We evaluated our model on two plant genomes, the genome of the dicot Arabidopsis thaliana and of the monocot Oryza sativa. We trained our models on 10 species with publicly available ATAC-seq data and 15 species with ChIP-seq data. Our best models showed accurate predictions in peak positions and the overall pattern of peaks for ATAC- and Histone H3 trimethylated at lysine 4 (H3K4me3) ChIP-seq. Annotating putatively accessible chromatin regions provides valuable input for the identification of CREs. In conjunction with other in silico data, such as predicted binding affinities for transcription factors (TFs), this can significantly narrow down the search space to a manageable number of experimentally verifiable DNA-protein interaction pairs.

Availability and ImplementationThe source code for Predmoter is available at: https://github.com/weberlab-hhu/Predmoter along with documentation for installation and usage. Predmoter uses a single-command inference, Predmoter.py, for both training and prediction. Predmoter takes a fasta file as input and outputs an h5 file and optionally bigWig and bedGraph files.

HighlightPredmoter will help identifying CREs and so gaining further insight into gene regulatory networks in plants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.05.565674">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.05.565674" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.05.565674">
        <p class="paperTitle">CellChat for systematic analysis of cell-cell communication from single-cell and spatially resolved transcriptomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.05.565674" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.05.565674" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jin, S.; Plikus, M. V.; Nie, Q.</p>
        <p class="info">Score: 57.6, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.05.565674' target='https://doi.org/10.1101/2023.11.05.565674'> 10.1101/2023.11.05.565674</a></p>
        <p class="abstract">Recent advances in single-cell sequencing technologies offer an opportunity to explore cell-cell communication in tissues systematically and with reduced bias. A key challenge is the integration between known molecular interactions and measurements into a framework to identify and analyze complex cell-cell communication networks. Previously, we developed a computational tool, named CellChat that infers and analyzes cell-cell communication networks from single-cell RNA-sequencing (scRNA-seq) data within an easily interpretable framework. CellChat quantifies the signaling communication probability between two cell groups using a simplified mass action-based model, which incorporates the core interaction between ligands and receptors with multi-subunit structure along with modulation by cofactors. CellChat v2 is an updated version that includes direct incorporation of spatial locations of cells, if available, to infer spatially proximal cell-cell communication, additional comparison functionalities, expanded database of ligand-receptor pairs along with rich annotations, and an Interactive CellChat Explorer. Here we provide a step-by-step protocol for using CellChat v2 that can be used for both scRNA-seq and spatially resolved transcriptomic data, including inference and analysis of cell-cell communication from one dataset and identification of altered signaling across different datasets. The key steps of applying CellChat v2 to spatially resolved transcriptomics are described in detail. The R implementation of CellChat v2 toolkit and tutorials with the graphic outputs are available at https://github.com/jinworks/CellChat. This protocol typically takes around 20 minutes, and no specialized prior bioinformatics training is required to complete the task.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.04.565620">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.04.565620" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.04.565620">
        <p class="paperTitle">MiRGraph: A transformer-based feature learning approach to identify miRNA-target interactions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.04.565620" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.04.565620" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, P.; Liu, Y.; Luo, J.; Li, Y.</p>
        <p class="info">Score: 13.2, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.04.565620' target='https://doi.org/10.1101/2023.11.04.565620'> 10.1101/2023.11.04.565620</a></p>
        <p class="abstract">MicroRNAs (miRNAs) play a crucial role in the regulation of gene expression by targeting specific mRNAs, and they can function as both tumor suppressors and oncogenes depending on the specific miRNA and its target genes. Detecting miRNA-target interactions (MTIs) is critical for unraveling the complex mechanisms of gene regulation, the potential therapeutic targets and diagnostic markers. There are currently few studies on MTIs prediction that simultaneously perform feature learning on heterogeneous graph network and sequence information. Thus, we present a transformer-based multiview feature learning method, named MiRGraph, capable of modeling both heterogeneous network and sequence features to improve the prediction performance of MTIs. To alleviate the problem of partial missing sequence information when using k-mer encoding or candidate target sites (CTSs), we utilize the mature miRNA sequence and the complete 3UTR sequence of the target mRNAs to encode seqeunce features. And a transformer-based CNN (TransCNN) module is designed for miRNAs and genes respectively to extract their personalized sequence features. Then a heterogeneous graph transformer (HGT) module is adopted to learn the network features through extracting the relational and structural information in a heterogeneous graph consisting of miRNA-miRNA, gene-gene and miRNA-target interactions. Finally, we utilize a multilayer perceptron (MLP) to map the learned features of miRNAs and genes into a same space, and a bilinear function to calculate the prediction scores of MTIs. Comparisons with other existing MTIs prediction methods illustrate the superiority of MiRGraph under multiple standard criteria metrics. The comparing results of distinguishing the high functional MTIs and case studies on identifying the novel MTIs showcase the real-world applications of MiRGraph in biological research.

AvailabilityMiRGraph is implemented in python and available at https://github.com/Liangyushi/MiRGraph.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.11.566695">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.11.566695" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.11.566695">
        <p class="paperTitle">Pre-processing of paleogenomes: Mitigating reference bias and postmortem damage in ancient genome data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.11.566695" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.11.566695" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Koptekin, D.; Yapar, E.; Vural, K. B.; Saglıcan, E.; Altınısık, N. E.; Malaspinas, A.-S.; Alkan, C.; Somel, M.</p>
        <p class="info">Score: 10.9, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.11.566695' target='https://doi.org/10.1101/2023.11.11.566695'> 10.1101/2023.11.11.566695</a></p>
        <p class="abstract">Ancient DNA analysis is subject to various technical challenges, including bias towards the reference allele (&#34;reference bias&#34;), postmortem damage (PMD) that confounds real variants, and limited coverage. Here, we conduct a systematic comparison of alternative approaches against reference bias and against PMD. To reduce reference bias, we either (a) mask variable sites before alignment or (b) align the data to a graph genome representing all variable sites. Compared to alignment to the linear reference genome, both masking and graph alignment effectively remove allelic bias when using simulated or real ancient human genome data, but only if sequencing data is available in FASTQ or unfiltered BAM format. Reference bias remains indelible in quality-filtered BAM files and in 1240K-capture data. We next study three approaches to overcome postmortem damage: (a) trimming, (b) rescaling base qualities, and (c) a new algorithm we present here, bamRefine, which masks only PMD-vulnerable polymorphic sites. We find that bamRefine is optimal in increasing the number of genotyped loci up to 20% compared to trimming and in improving accuracy compared to rescaling. We propose graph alignment coupled with bamRefine to minimise data loss and bias. We also urge the paleogenomics community to publish FASTQ files.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
