<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Assessing the limits of zero-shot foundation models in single-cell biology
Authors: Kedzierska, K. Z.; Crawford, L.; Amini, A. P.; Lu, A. X.
Score: 70.1, Published: 2023-10-17 DOI: 10.1101/2023.10.16.561085
The advent and success of foundation models such as GPT has sparked growing interest in their application to single-cell biology. Models like Geneformer and scGPT have emerged with the promise of serving as versatile tools for this specialized field. However, the efficacy of these models, particularly in zero-shot settings where models are not fine-tuned but used without any further training, remains an open question, especially as practical constraints require useful models to function in settings that preclude fine-tuning (e.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="Assessing the limits of zero-shot foundation models in single-cell biology
Authors: Kedzierska, K. Z.; Crawford, L.; Amini, A. P.; Lu, A. X.
Score: 70.1, Published: 2023-10-17 DOI: 10.1101/2023.10.16.561085
The advent and success of foundation models such as GPT has sparked growing interest in their application to single-cell biology. Models like Geneformer and scGPT have emerged with the promise of serving as versatile tools for this specialized field. However, the efficacy of these models, particularly in zero-shot settings where models are not fine-tuned but used without any further training, remains an open question, especially as practical constraints require useful models to function in settings that preclude fine-tuning (e." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-22T10:37:23+00:00" />
<meta property="article:modified_time" content="2023-10-22T10:37:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="Assessing the limits of zero-shot foundation models in single-cell biology
Authors: Kedzierska, K. Z.; Crawford, L.; Amini, A. P.; Lu, A. X.
Score: 70.1, Published: 2023-10-17 DOI: 10.1101/2023.10.16.561085
The advent and success of foundation models such as GPT has sparked growing interest in their application to single-cell biology. Models like Geneformer and scGPT have emerged with the promise of serving as versatile tools for this specialized field. However, the efficacy of these models, particularly in zero-shot settings where models are not fine-tuned but used without any further training, remains an open question, especially as practical constraints require useful models to function in settings that preclude fine-tuning (e."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "Assessing the limits of zero-shot foundation models in single-cell biology\nAuthors: Kedzierska, K. Z.; Crawford, L.; Amini, A. P.; Lu, A. X.\nScore: 70.1, Published: 2023-10-17 DOI: 10.1101/2023.10.16.561085\nThe advent and success of foundation models such as GPT has sparked growing interest in their application to single-cell biology. Models like Geneformer and scGPT have emerged with the promise of serving as versatile tools for this specialized field. However, the efficacy of these models, particularly in zero-shot settings where models are not fine-tuned but used without any further training, remains an open question, especially as practical constraints require useful models to function in settings that preclude fine-tuning (e.",
  "keywords": [
    
  ],
  "articleBody": " Assessing the limits of zero-shot foundation models in single-cell biology\nAuthors: Kedzierska, K. Z.; Crawford, L.; Amini, A. P.; Lu, A. X.\nScore: 70.1, Published: 2023-10-17 DOI: 10.1101/2023.10.16.561085\nThe advent and success of foundation models such as GPT has sparked growing interest in their application to single-cell biology. Models like Geneformer and scGPT have emerged with the promise of serving as versatile tools for this specialized field. However, the efficacy of these models, particularly in zero-shot settings where models are not fine-tuned but used without any further training, remains an open question, especially as practical constraints require useful models to function in settings that preclude fine-tuning (e.g., discovery settings where labels are not fully known). This paper presents a rigorous evaluation of the zero-shot performance of these proposed single-cell foundation models. We assess their utility in tasks such as cell type clustering and batch effect correction, and evaluate the generality of their pretraining objectives. Our results indicate that both Geneformer and scGPT exhibit limited reliability in zero-shot settings and often underperform compared to simpler methods. These findings serve as a cautionary note for the deployment of proposed single-cell foundation models and highlight the need for more focused research to realize their potential.2\nAF2BIND: Predicting ligand-binding sites using the pair representation of AlphaFold2\nAuthors: Gazizov, A.; Lian, A.; Goverde, C. A.; Ovchinnikov, S.; Polizzi, N. F.\nScore: 45.5, Published: 2023-10-18 DOI: 10.1101/2023.10.15.562410\nPredicting ligand-binding sites, particularly in the absence of previously resolved homologous structures, presents a significant challenge in structural biology. Here, we leverage the internal pairwise representation of AlphaFold2 (AF2) to train a model, AF2BIND, to accurately predict small-molecule-binding residues given only a target protein. AF2BIND uses 20 \"bait\" amino acids to optimally extract the binding signal in the absence of a small-molecule ligand. We find that the AF2 pair representation outperforms other neural-network representations for binding-site prediction. Moreover, unique combinations of the 20 bait amino acids are correlated with chemical properties of the ligand.\nGPN-MSA: an alignment-based DNA language model for genome-wide variant effect prediction\nAuthors: Benegas, G.; Albors, C.; Aw, A. J.; Ye, C.; Song, Y. S.\nScore: 65.6, Published: 2023-10-11 DOI: 10.1101/2023.10.10.561776\nWhereas protein language models have demonstrated remarkable efficacy in predicting the effects of missense variants, DNA counterparts have not yet achieved a similar competitive edge for genome-wide variant effect predictions, especially in complex genomes such as that of humans. To address this challenge, we here introduce GPN-MSA, a novel framework for DNA language models that leverages whole-genome sequence alignments across multiple species and takes only a few hours to train. Across several benchmarks on clinical databases (ClinVar, COSMIC, and OMIM) and population genomic data (gnomAD), our model for the human genome achieves outstanding performance on deleteriousness prediction for both coding and non-coding variants.\nA 5' UTR Language Model for Decoding Untranslated Regions of mRNA and Function Predictions\nAuthors: Chu, Y.; Yu, D.; Li, Y.; Huang, K.; Shen, Y.; Cong, L.; Zhang, J.; Wang, M.\nScore: 32.4, Published: 2023-10-14 DOI: 10.1101/2023.10.11.561938\nThe 5 UTR, a regulatory region at the beginning of an mRNA molecule, plays a crucial role in regulating the translation process and impacts the protein expression level. Language models have showcased their effectiveness in decoding the functions of protein and genome sequences. Here, we introduced a language model for 5 UTR, which we refer to as the UTR-LM. The UTR-LM is pre-trained on endogenous 5 UTRs from multiple species and is further augmented with supervised information including secondary structure and minimum free energy. We fine-tuned the UTR-LM in a variety of downstream tasks. The model outperformed the best-known benchmark by up to 42% for predicting the Mean Ribosome Loading, and by up to 60% for predicting the Translation Efficiency and the mRNA Expression Level. The model also applies to identifying unannotated Internal Ribosome Entry Sites within the untranslated region and improves the AUPR from 0.37 to 0.52 compared to the best baseline. Further, we designed a library of 211 novel 5 UTRs with high predicted values of translation efficiency and evaluated them via a wet-lab assay. Experiment results confirmed that our top designs achieved a 32.5% increase in protein production level relative to well-established 5 UTR optimized for therapeutics.\nMapping the topography of spatial gene expression with interpretable deep learning\nAuthors: Chitra, U.; Arnold, B. J.; Sarkar, H.; Ma, C.; Lopez-Darwin, S.; Sanno, K.; Raphael, B. J.\nScore: 35.1, Published: 2023-10-13 DOI: 10.1101/2023.10.10.561757\nSpatially resolved transcriptomics technologies provide high-throughput measurements of gene expression in a tissue slice, but the sparsity of this data complicates the analysis of spatial gene expression patterns such as gene expression gradients. We address these issues by deriving a topographic map of a tissue slice--analogous to a map of elevation in a landscape--using a novel quantity called the isodepth. Contours of constant isodepth enclose spatial domains with distinct cell type composition, while gradients of the isodepth indicate spatial directions of maximum change in gene expression. We develop GASTON, an unsupervised and interpretable deep learning algorithm that simultaneously learns the isodepth, spatial gene expression gradients, and piecewise linear functions of the isodepth that model both continuous gradients and discontinuous spatial variation in the expression of individual genes. We validate GASTON by showing that it accurately identifies spatial domains and marker genes across several biological systems. In SRT data from the brain, GASTON reveals gradients of neuronal differentiation and firing, and in SRT data from a tumor sample, GASTON infers gradients of metabolic activity and epithelial-mesenchymal transition (EMT)-related gene expression in the tumor microenvironment.\nGENEPT: A SIMPLE BUT HARD-TO-BEAT FOUNDATION MODEL FOR GENES AND CELLS BUILT FROM CHATGPT\nAuthors: Chen, Y. T.; Zou, J.\nScore: 25.3, Published: 2023-10-19 DOI: 10.1101/2023.10.16.562533\nThere has been significant recent progress in leveraging large-scale gene expression data to develop foundation models for single-cell transcriptomes such as Geneformer [1], scGPT [2], and scBERT [3]. These models infer gene functions and interrelations from the gene expression profiles of millions of cells, which requires extensive data curation and resource-intensive training. Here, we explore a much simpler alternative by leveraging ChatGPT embeddings of genes based on literature. Our proposal, GenePT, uses NCBI text descriptions of individual genes with GPT-3.5 to generate gene embeddings. From there, GenePT generates single-cell embeddings in two ways: (i) by averaging the gene embeddings, weighted by each gene's expression level; or (ii) by creating a sentence embedding for each cell, using gene names ordered by the expression level. Without the need for dataset curation and additional pretraining, GenePT is efficient and easy to use. On many downstream tasks used to evaluate recent single-cell foundation models --- e.g., classifying gene properties and cell types --- GenePT achieves comparable, and often better, performance than Geneformer and other methods. GenePT demonstrates that large language model embedding of literature is a simple and effective path for biological foundation models.\nBTR: A Bioinformatics Tool Recommendation System\nAuthors: Green, R.; Qu, X.; Liu, J.; Yu, T.\nScore: 23.8, Published: 2023-10-17 DOI: 10.1101/2023.10.13.562252\nMotivationThe rapid expansion of Bioinformatics research has resulted in a vast array of computational tools utilized in the development of scientific analysis pipelines. However, constructing these pipelines is a laborious and intricate task, one which demands extensive domain knowledge and careful consideration at all stages. As the Bioinformatics landscape continues to evolve, researchers, both novice and expert, may find themselves overwhelmed when working in unfamiliar fields. Consequently, this may result in the selection of unsuitable or suboptimal tools during workflow development. ResultsIn this paper, we propose the Bioinformatics Tool Recommendation system (BTR), an innovative deep learning model designed to recommend the most suitable tools for a given workflow-in-progress. BTR utilizes recent advances in graph neural network technology and introduces a novel approach, representing the entire workflow as a graph to capture essential context and structural information. Additionally, natural language processing techniques are integrated to enhance the quality of tool recommendations by analyzing associated tool descriptions. Experiments demonstrate that BTR outperforms the existing Galaxy tool recommendation system, highlighting its potential to greatly facilitate scientific workflow construction. Availability and implementationThe Python source code is available at https://github.com/ryangreenj/bioinformatics_tool_recommendation\nAccurate model and ensemble refinement using cryo-electron microscopy maps and Bayesian inference\nAuthors: Hoff, S. E.; Thomasen, F. E.; Lindorff-Larsen, K.; Bonomi, M.\nScore: 22.3, Published: 2023-10-19 DOI: 10.1101/2023.10.18.562710\nConverting cryo-electron microscopy (cryo-EM) data into high-quality structural models is a challenging problem of outstanding importance. Current refinement methods often generate unbalanced models in which physico-chemical quality is sacrificed for excellent fit to the data. Furthermore, these techniques struggle to represent the conformational heterogeneity averaged out in low-resolution regions of density maps. Here we introduce EMMIVox, a Bayesian inference approach to determine single-structure models as well as structural ensembles from cryo-EM maps. EMMIVox automatically balances experimental information with accurate physico-chemical models of the system and the surrounding environment, including waters, lipids, and ions. Explicit treatment of data correlation and noise as well as inference of accurate B-factors enable determination of structural models and ensembles with both excellent fit to the data and high stereochemical quality, thus outperforming state-of-the-art refinement techniques. EMMIVox represents a flexible approach to determine high-quality structural models that will contribute to advancing our understanding of the molecular mechanisms underlying biological functions.\nA deep learning model embedded framework to distinguish DNA and RNA mutations directly from RNA-seq\nAuthors: Fu, Z.-C.; Gao, B.-Q.; Nan, F.; Ma, X.-K.; Yang, L.\nScore: 20.7, Published: 2023-10-20 DOI: 10.1101/2023.10.17.562625\nWe develop a stepwise computational framework, called DEMINING, to directly detect expressed DNA and RNA mutations in RNA deep sequencing data. DEMINING incorporates a deep learning model named DeepDDR, which facilitates the separation of expressed DNA mutations from RNA mutations after RNA-seq read mapping and pileup. When applied in RNA-seq of acute myeloid leukemia patients, DEMINING uncovered previously-underappreciated DNA and RNA mutations, some associated with the upregulated expression of host genes or the production of neoantigens. Finally, we demonstrate that DEMINING could precisely classify DNA and RNA mutations in RNA-seq data from non-primate species through the utilization of transfer learning.\nUncover spatially informed shared variations for single-cell spatial transcriptomics with STew\nAuthors: Guo, N.; Vargas, J.; Fritz, D.; Krishna, R.; Zhang, F.\nScore: 30.6, Published: 2023-10-13 DOI: 10.1101/2023.10.10.561789\nMotivationThe recent spatial transcriptomics (ST) technologies have enabled characterization of gene expression patterns and spatial information, advancing our understanding of cell lineages within diseased tissues. Several analytical approaches have been proposed for ST data, but effectively utilizing spatial information to unveil the shared variation with gene expression remains a challenge. ResultsWe introduce STew, a Spatial Transcriptomic multi-viEW representative learning method, to jointly analyze spatial information and gene expression in a scalable manner, followed by a data-driven statistical framework to measure the goodness of model fit. Through benchmarking using Human DLPFC data with true manual annotations, STew achieved superior performance in both clustering accuracy and continuity of identified spatial domains compared with other methods. STew is also robust to generate consistent results insensitive to model parameters, including sparsity constraints. We next applied STew to various ST data acquired from 10x Visium and Slide-seqV2, encompassing samples from both mouse and human brain, which revealed spatially informed cell type clusters. We further identified a pro-inflammatory fibroblast spatial niche using ST data from psoriatic skins. Hence, STew is a generalized method to identify both spatially informed clusters and disease-relevant niches in complex tissues. AvailabilitySource code and the R software tool STew are available from github.com/fanzhanglab/STew. Contactfan.3.zhang@cuanschutz.edu Supplementary informationSupplementary data are provided.\n",
  "wordCount" : "1884",
  "inLanguage": "en",
  "datePublished": "2023-10-22T10:37:23Z",
  "dateModified": "2023-10-22T10:37:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on October 22, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.16.561085">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.16.561085" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.16.561085">
        <p class="paperTitle">Assessing the limits of zero-shot foundation models in single-cell biology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.16.561085" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.16.561085" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kedzierska, K. Z.; Crawford, L.; Amini, A. P.; Lu, A. X.</p>
        <p class="info">Score: 70.1, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.16.561085' target='https://doi.org/10.1101/2023.10.16.561085'> 10.1101/2023.10.16.561085</a></p>
        <p class="abstract">The advent and success of foundation models such as GPT has sparked growing interest in their application to single-cell biology. Models like Geneformer and scGPT have emerged with the promise of serving as versatile tools for this specialized field. However, the efficacy of these models, particularly in zero-shot settings where models are not fine-tuned but used without any further training, remains an open question, especially as practical constraints require useful models to function in settings that preclude fine-tuning (e.g., discovery settings where labels are not fully known). This paper presents a rigorous evaluation of the zero-shot performance of these proposed single-cell foundation models. We assess their utility in tasks such as cell type clustering and batch effect correction, and evaluate the generality of their pretraining objectives. Our results indicate that both Geneformer and scGPT exhibit limited reliability in zero-shot settings and often underperform compared to simpler methods. These findings serve as a cautionary note for the deployment of proposed single-cell foundation models and highlight the need for more focused research to realize their potential.2</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.15.562410">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.15.562410" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.15.562410">
        <p class="paperTitle">AF2BIND: Predicting ligand-binding sites using the pair representation of AlphaFold2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.15.562410" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.15.562410" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gazizov, A.; Lian, A.; Goverde, C. A.; Ovchinnikov, S.; Polizzi, N. F.</p>
        <p class="info">Score: 45.5, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.15.562410' target='https://doi.org/10.1101/2023.10.15.562410'> 10.1101/2023.10.15.562410</a></p>
        <p class="abstract">Predicting ligand-binding sites, particularly in the absence of previously resolved homologous structures, presents a significant challenge in structural biology. Here, we leverage the internal pairwise representation of AlphaFold2 (AF2) to train a model, AF2BIND, to accurately predict small-molecule-binding residues given only a target protein. AF2BIND uses 20 &#34;bait&#34; amino acids to optimally extract the binding signal in the absence of a small-molecule ligand. We find that the AF2 pair representation outperforms other neural-network representations for binding-site prediction. Moreover, unique combinations of the 20 bait amino acids are correlated with chemical properties of the ligand.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561776">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561776" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561776">
        <p class="paperTitle">GPN-MSA: an alignment-based DNA language model for genome-wide variant effect prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561776" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561776" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Benegas, G.; Albors, C.; Aw, A. J.; Ye, C.; Song, Y. S.</p>
        <p class="info">Score: 65.6, Published: 2023-10-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561776' target='https://doi.org/10.1101/2023.10.10.561776'> 10.1101/2023.10.10.561776</a></p>
        <p class="abstract">Whereas protein language models have demonstrated remarkable efficacy in predicting the effects of missense variants, DNA counterparts have not yet achieved a similar competitive edge for genome-wide variant effect predictions, especially in complex genomes such as that of humans. To address this challenge, we here introduce GPN-MSA, a novel framework for DNA language models that leverages whole-genome sequence alignments across multiple species and takes only a few hours to train. Across several benchmarks on clinical databases (ClinVar, COSMIC, and OMIM) and population genomic data (gnomAD), our model for the human genome achieves outstanding performance on deleteriousness prediction for both coding and non-coding variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.11.561938">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.11.561938" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.11.561938">
        <p class="paperTitle">A 5&#39; UTR Language Model for Decoding Untranslated Regions of mRNA and Function Predictions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.11.561938" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.11.561938" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chu, Y.; Yu, D.; Li, Y.; Huang, K.; Shen, Y.; Cong, L.; Zhang, J.; Wang, M.</p>
        <p class="info">Score: 32.4, Published: 2023-10-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.11.561938' target='https://doi.org/10.1101/2023.10.11.561938'> 10.1101/2023.10.11.561938</a></p>
        <p class="abstract">The 5 UTR, a regulatory region at the beginning of an mRNA molecule, plays a crucial role in regulating the translation process and impacts the protein expression level. Language models have showcased their effectiveness in decoding the functions of protein and genome sequences. Here, we introduced a language model for 5 UTR, which we refer to as the UTR-LM. The UTR-LM is pre-trained on endogenous 5 UTRs from multiple species and is further augmented with supervised information including secondary structure and minimum free energy. We fine-tuned the UTR-LM in a variety of downstream tasks. The model outperformed the best-known benchmark by up to 42% for predicting the Mean Ribosome Loading, and by up to 60% for predicting the Translation Efficiency and the mRNA Expression Level. The model also applies to identifying unannotated Internal Ribosome Entry Sites within the untranslated region and improves the AUPR from 0.37 to 0.52 compared to the best baseline. Further, we designed a library of 211 novel 5 UTRs with high predicted values of translation efficiency and evaluated them via a wet-lab assay. Experiment results confirmed that our top designs achieved a 32.5% increase in protein production level relative to well-established 5 UTR optimized for therapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561757">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561757" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561757">
        <p class="paperTitle">Mapping the topography of spatial gene expression with interpretable deep learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561757" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561757" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chitra, U.; Arnold, B. J.; Sarkar, H.; Ma, C.; Lopez-Darwin, S.; Sanno, K.; Raphael, B. J.</p>
        <p class="info">Score: 35.1, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561757' target='https://doi.org/10.1101/2023.10.10.561757'> 10.1101/2023.10.10.561757</a></p>
        <p class="abstract">Spatially resolved transcriptomics technologies provide high-throughput measurements of gene expression in a tissue slice, but the sparsity of this data complicates the analysis of spatial gene expression patterns such as gene expression gradients. We address these issues by deriving a topographic map of a tissue slice--analogous to a map of elevation in a landscape--using a novel quantity called the isodepth. Contours of constant isodepth enclose spatial domains with distinct cell type composition, while gradients of the isodepth indicate spatial directions of maximum change in gene expression. We develop GASTON, an unsupervised and interpretable deep learning algorithm that simultaneously learns the isodepth, spatial gene expression gradients, and piecewise linear functions of the isodepth that model both continuous gradients and discontinuous spatial variation in the expression of individual genes. We validate GASTON by showing that it accurately identifies spatial domains and marker genes across several biological systems. In SRT data from the brain, GASTON reveals gradients of neuronal differentiation and firing, and in SRT data from a tumor sample, GASTON infers gradients of metabolic activity and epithelial-mesenchymal transition (EMT)-related gene expression in the tumor microenvironment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.16.562533">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.16.562533" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.16.562533">
        <p class="paperTitle">GENEPT: A SIMPLE BUT HARD-TO-BEAT FOUNDATION MODEL FOR GENES AND CELLS BUILT FROM CHATGPT</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.16.562533" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.16.562533" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, Y. T.; Zou, J.</p>
        <p class="info">Score: 25.3, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.16.562533' target='https://doi.org/10.1101/2023.10.16.562533'> 10.1101/2023.10.16.562533</a></p>
        <p class="abstract">There has been significant recent progress in leveraging large-scale gene expression data to develop foundation models for single-cell transcriptomes such as Geneformer [1], scGPT [2], and scBERT [3]. These models infer gene functions and interrelations from the gene expression profiles of millions of cells, which requires extensive data curation and resource-intensive training. Here, we explore a much simpler alternative by leveraging ChatGPT embeddings of genes based on literature. Our proposal, GenePT, uses NCBI text descriptions of individual genes with GPT-3.5 to generate gene embeddings. From there, GenePT generates single-cell embeddings in two ways: (i) by averaging the gene embeddings, weighted by each gene&#39;s expression level; or (ii) by creating a sentence embedding for each cell, using gene names ordered by the expression level. Without the need for dataset curation and additional pretraining, GenePT is efficient and easy to use. On many downstream tasks used to evaluate recent single-cell foundation models --- e.g., classifying gene properties and cell types --- GenePT achieves comparable, and often better, performance than Geneformer and other methods. GenePT demonstrates that large language model embedding of literature is a simple and effective path for biological foundation models.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562252">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562252" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562252">
        <p class="paperTitle">BTR: A Bioinformatics Tool Recommendation System</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562252" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562252" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Green, R.; Qu, X.; Liu, J.; Yu, T.</p>
        <p class="info">Score: 23.8, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562252' target='https://doi.org/10.1101/2023.10.13.562252'> 10.1101/2023.10.13.562252</a></p>
        <p class="abstract">MotivationThe rapid expansion of Bioinformatics research has resulted in a vast array of computational tools utilized in the development of scientific analysis pipelines. However, constructing these pipelines is a laborious and intricate task, one which demands extensive domain knowledge and careful consideration at all stages. As the Bioinformatics landscape continues to evolve, researchers, both novice and expert, may find themselves overwhelmed when working in unfamiliar fields. Consequently, this may result in the selection of unsuitable or suboptimal tools during workflow development.

ResultsIn this paper, we propose the Bioinformatics Tool Recommendation system (BTR), an innovative deep learning model designed to recommend the most suitable tools for a given workflow-in-progress. BTR utilizes recent advances in graph neural network technology and introduces a novel approach, representing the entire workflow as a graph to capture essential context and structural information. Additionally, natural language processing techniques are integrated to enhance the quality of tool recommendations by analyzing associated tool descriptions. Experiments demonstrate that BTR outperforms the existing Galaxy tool recommendation system, highlighting its potential to greatly facilitate scientific workflow construction.

Availability and implementationThe Python source code is available at https://github.com/ryangreenj/bioinformatics_tool_recommendation</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.18.562710">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.18.562710" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.18.562710">
        <p class="paperTitle">Accurate model and ensemble refinement using cryo-electron microscopy maps and Bayesian inference</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.18.562710" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.18.562710" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hoff, S. E.; Thomasen, F. E.; Lindorff-Larsen, K.; Bonomi, M.</p>
        <p class="info">Score: 22.3, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.18.562710' target='https://doi.org/10.1101/2023.10.18.562710'> 10.1101/2023.10.18.562710</a></p>
        <p class="abstract">Converting cryo-electron microscopy (cryo-EM) data into high-quality structural models is a challenging problem of outstanding importance. Current refinement methods often generate unbalanced models in which physico-chemical quality is sacrificed for excellent fit to the data. Furthermore, these techniques struggle to represent the conformational heterogeneity averaged out in low-resolution regions of density maps. Here we introduce EMMIVox, a Bayesian inference approach to determine single-structure models as well as structural ensembles from cryo-EM maps. EMMIVox automatically balances experimental information with accurate physico-chemical models of the system and the surrounding environment, including waters, lipids, and ions. Explicit treatment of data correlation and noise as well as inference of accurate B-factors enable determination of structural models and ensembles with both excellent fit to the data and high stereochemical quality, thus outperforming state-of-the-art refinement techniques. EMMIVox represents a flexible approach to determine high-quality structural models that will contribute to advancing our understanding of the molecular mechanisms underlying biological functions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562625">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562625" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562625">
        <p class="paperTitle">A deep learning model embedded framework to distinguish DNA and RNA mutations directly from RNA-seq</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562625" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562625" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fu, Z.-C.; Gao, B.-Q.; Nan, F.; Ma, X.-K.; Yang, L.</p>
        <p class="info">Score: 20.7, Published: 2023-10-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562625' target='https://doi.org/10.1101/2023.10.17.562625'> 10.1101/2023.10.17.562625</a></p>
        <p class="abstract">We develop a stepwise computational framework, called DEMINING, to directly detect expressed DNA and RNA mutations in RNA deep sequencing data. DEMINING incorporates a deep learning model named DeepDDR, which facilitates the separation of expressed DNA mutations from RNA mutations after RNA-seq read mapping and pileup. When applied in RNA-seq of acute myeloid leukemia patients, DEMINING uncovered previously-underappreciated DNA and RNA mutations, some associated with the upregulated expression of host genes or the production of neoantigens. Finally, we demonstrate that DEMINING could precisely classify DNA and RNA mutations in RNA-seq data from non-primate species through the utilization of transfer learning.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561789">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561789" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561789">
        <p class="paperTitle">Uncover spatially informed shared variations for single-cell spatial transcriptomics with STew</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561789" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561789" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guo, N.; Vargas, J.; Fritz, D.; Krishna, R.; Zhang, F.</p>
        <p class="info">Score: 30.6, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561789' target='https://doi.org/10.1101/2023.10.10.561789'> 10.1101/2023.10.10.561789</a></p>
        <p class="abstract">MotivationThe recent spatial transcriptomics (ST) technologies have enabled characterization of gene expression patterns and spatial information, advancing our understanding of cell lineages within diseased tissues. Several analytical approaches have been proposed for ST data, but effectively utilizing spatial information to unveil the shared variation with gene expression remains a challenge.

ResultsWe introduce STew, a Spatial Transcriptomic multi-viEW representative learning method, to jointly analyze spatial information and gene expression in a scalable manner, followed by a data-driven statistical framework to measure the goodness of model fit. Through benchmarking using Human DLPFC data with true manual annotations, STew achieved superior performance in both clustering accuracy and continuity of identified spatial domains compared with other methods. STew is also robust to generate consistent results insensitive to model parameters, including sparsity constraints. We next applied STew to various ST data acquired from 10x Visium and Slide-seqV2, encompassing samples from both mouse and human brain, which revealed spatially informed cell type clusters. We further identified a pro-inflammatory fibroblast spatial niche using ST data from psoriatic skins. Hence, STew is a generalized method to identify both spatially informed clusters and disease-relevant niches in complex tissues.

AvailabilitySource code and the R software tool STew are available from github.com/fanzhanglab/STew.

Contactfan.3.zhang@cuanschutz.edu

Supplementary informationSupplementary data are provided.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
