<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="edgeR 4.0: powerful differential analysis of sequencing data with expanded functionality and improved support for small counts and larger datasets
Authors: Chen, Y.; Chen, L.; Lun, A. T. L.; Baldoni, P. L.; Smyth, G. K.
Score: 79.5, Published: 2024-01-24 DOI: 10.1101/2024.01.21.576131
edgeR is an R/Bioconductor software package for differential analyses of sequencing data in the form of read counts for genes or genomic features. Over the past 15 years, edgeR has been a popular choice for statistical analysis of data from sequencing technologies such as RNA-seq or ChIP-seq.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="edgeR 4.0: powerful differential analysis of sequencing data with expanded functionality and improved support for small counts and larger datasets
Authors: Chen, Y.; Chen, L.; Lun, A. T. L.; Baldoni, P. L.; Smyth, G. K.
Score: 79.5, Published: 2024-01-24 DOI: 10.1101/2024.01.21.576131
edgeR is an R/Bioconductor software package for differential analyses of sequencing data in the form of read counts for genes or genomic features. Over the past 15 years, edgeR has been a popular choice for statistical analysis of data from sequencing technologies such as RNA-seq or ChIP-seq." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-31T10:38:28+00:00" />
<meta property="article:modified_time" content="2024-01-31T10:38:28+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="edgeR 4.0: powerful differential analysis of sequencing data with expanded functionality and improved support for small counts and larger datasets
Authors: Chen, Y.; Chen, L.; Lun, A. T. L.; Baldoni, P. L.; Smyth, G. K.
Score: 79.5, Published: 2024-01-24 DOI: 10.1101/2024.01.21.576131
edgeR is an R/Bioconductor software package for differential analyses of sequencing data in the form of read counts for genes or genomic features. Over the past 15 years, edgeR has been a popular choice for statistical analysis of data from sequencing technologies such as RNA-seq or ChIP-seq."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "edgeR 4.0: powerful differential analysis of sequencing data with expanded functionality and improved support for small counts and larger datasets\nAuthors: Chen, Y.; Chen, L.; Lun, A. T. L.; Baldoni, P. L.; Smyth, G. K.\nScore: 79.5, Published: 2024-01-24 DOI: 10.1101/2024.01.21.576131\nedgeR is an R/Bioconductor software package for differential analyses of sequencing data in the form of read counts for genes or genomic features. Over the past 15 years, edgeR has been a popular choice for statistical analysis of data from sequencing technologies such as RNA-seq or ChIP-seq.",
  "keywords": [
    
  ],
  "articleBody": " edgeR 4.0: powerful differential analysis of sequencing data with expanded functionality and improved support for small counts and larger datasets\nAuthors: Chen, Y.; Chen, L.; Lun, A. T. L.; Baldoni, P. L.; Smyth, G. K.\nScore: 79.5, Published: 2024-01-24 DOI: 10.1101/2024.01.21.576131\nedgeR is an R/Bioconductor software package for differential analyses of sequencing data in the form of read counts for genes or genomic features. Over the past 15 years, edgeR has been a popular choice for statistical analysis of data from sequencing technologies such as RNA-seq or ChIP-seq. edgeR pioneered the use of the negative binomial distribution to model read count data with replicates and the use of generalized linear models to analyse complex experimental designs. edgeR implements empirical Bayes moderation methods to allow reliable inference when the number of replicates is small. This article announces edgeR version 4, which includes new developments across a range of application areas. Infrastructure improvements include support for fractional counts, implementation of model fitting in C++, and a new statistical treatment of the quasi-likelihood pipeline that improves accuracy for small counts. The revised package has new functionality for differential methylation analysis, differential transcript expression, differential transcript and exon usage, testing relative to a fold-change threshold and pathway analysis. This article reviews the statistical framework and computational implementation of edgeR, briefly summarizing all the existing features and functionalities but with special attention to new features and those that have not been described previously.\nPhylogenetic tree statistics: a systematic overview using the new R package 'treestats'\nAuthors: Janzen, T.; Etienne, R. S.\nScore: 73.9, Published: 2024-01-29 DOI: 10.1101/2024.01.24.576848\nPhylogenetic trees are believed to contain a wealth of information on diversification processes. Comparing phylogenetic trees is not straightforward due to their high dimensionality. Researchers have therefore defined a wide range of one-dimensional summary statistics. However, it remains unexplored to what extent these summary statistics cover the same underlying information and what summary statistics best explain observed variation across phylogenies. Furthermore, a large subset of available summary statistics focusses on measuring the topological features of a phylogenetic tree, but are often only explored at the extreme edge cases of the fully balanced or unbalanced tree and not for trees of intermediate balance. Here, we introduce a new R package that provides speed optimized code to compute 54 summary statistics. We study correlations between summary statistics on empirical trees and on trees simulated using several diversification models. Furthermore, we introduce an algorithm to create intermediately balanced trees in a well-defined manner, in order to explore variation in summary statistics across a balance gradient. We find that almost all summary statistics are correlated with tree size, and it is difficult if not impossible to correct for tree size, unless the tree generating model is known. Furthermore, we find that across empirical and simulated trees, at least two large clusters of correlated summary statistics can be found, where statistics group together based on information used (topology or branching times). However, the finer grained correlation structure appears to depend strongly on either the taxonomic group studied (in empirical studies) or the diversification model (in simulation studies). Nevertheless, we can identify multiple groups of summary statistics that are strongly and consistently correlated, indicating that these statistics measure the same underlying property of a tree. Lastly, we find that almost all topological summary statistics vary non-linearly and sometimes even non-monotonically with our intuitive balance gradient. Therefore, in order to avoid introducing biases and missing underlying information, we advocate for selecting as many summary statistics as possible in phylogenetic analyses. With the introduction of the treestats package, which provides fast and reliable calculations, such an approach is now routinely possible.\nCHOIR improves significance-based detection of cell types and states from single-cell data\nAuthors: Petersen, C.; Mucke, L.; Corces, M. R.\nScore: 63.1, Published: 2024-01-23 DOI: 10.1101/2024.01.18.576317\nClustering is a critical step in the analysis of single-cell data, as it enables the discovery and characterization of putative cell types and states. However, most popular clustering tools do not subject clustering results to statistical inference testing, leading to risks of overclustering or underclustering data and often resulting in ineffective identification of cell types with widely differing prevalence. To address these challenges, we present CHOIR (clustering hierarchy optimization by iterative random forests), which applies a framework of random forest classifiers and permutation tests across a hierarchical clustering tree to statistically determine which clusters represent distinct populations. We demonstrate the enhanced performance of CHOIR through extensive benchmarking against 14 existing clustering methods across 100 simulated and 4 real single-cell RNA-seq, ATAC-seq, spatial transcriptomic, and multi-omic datasets. CHOIR can be applied to any single-cell data type and provides a flexible, scalable, and robust solution to the important challenge of identifying biologically relevant cell groupings within heterogeneous single-cell data.\nBeav: A bacterial genome and mobile element annotation pipeline\nAuthors: Jung, J. M.; Rahman, A.; Schiffer, A. M.; Weisberg, A. J.\nScore: 28.4, Published: 2024-01-26 DOI: 10.1101/2024.01.25.577299\nComprehensive and accurate genome annotation is crucial for inferring the predicted functions of an organism. Numerous tools exist to annotate genes, gene clusters, mobile genetic elements, and other diverse features. However, these tools and pipelines can be difficult to install and run, be specialized for a particular element or feature, or lack annotations for larger elements that provide important genomic context. Integrating results across analyses is also important for understanding gene function. To address these challenges, we present the Beav annotation pipeline. Beav is a command-line tool that automates the annotation of bacterial genome sequences, mobile genetic elements, molecular systems and gene clusters, key regulatory features, and other elements. Beav uses existing tools in addition to custom models, scripts, and databases to annotate diverse elements, systems, and sequence features. Custom databases for plant-associated microbes are incorporated to improve annotation of key virulence and symbiosis genes in agriculturally important pathogens and mutualists. Beav includes an optional Agrobacterium-specific pipeline that identifies and classifies oncogenic plasmids and annotates plasmid-specific features. Following the completion of all analyses, annotations are consolidated to produce a single comprehensive output. Finally, Beav generates publication-quality genome and plasmid maps. Beav is on Bioconda and is available for download at https://github.com/weisberglab/beav.\nPolarity-JaM: An image analysis toolbox for cell polarity, junction and morphology quantification\nAuthors: Giese, W.; Albrecht, J. P.; Oppenheim, O.; Akmeric, E. B.; Kraxner, J.; Schmidt, D.; Harrington, K. I.; Gerhardt, H.\nScore: 25.5, Published: 2024-01-29 DOI: 10.1101/2024.01.24.577027\nCell polarity involves the asymmetric distribution of cellular components such as signaling molecules and organelles within a cell, asymmetries of a cell's shape as well as contacts with neighbouring cells. Gradients and mechanical forces often act as global cues that bias cell polarity and orientation, and polarity is coordinated by communication between adjacent cells. Advances in fluorescence microscopy combined with deep learning algorithms for image segmentation open up a wealth of possibilities to understand cell polarity behaviour in health and disease. We have therefore developed the open-source package Polarity-JaM, which offers versatile methods for performing reproducible exploratory image analysis. Multi-channel single cell segmentation is performed using a flexible and user-friendly interface to state-of-the-art deep learning algorithms. Interpretable single-cell features are automatically extracted, including cell and organelle orientation, cell-cell contact morphology, signaling molecule gradients, as well as collective orientation, tissue-wide size and shape variation. Circular statistics of cell polarity, including polarity indices, confidence intervals, and circular correlation analysis, can be computed using our web application. We have developed data graphs for comprehensive visualisation of key statistical measures and suggest the use of an adapted polarity index when the expected polarisation direction or the direction of a global cue is known a priori. The focus of our analysis is on fluorescence image data from endothelial cells (ECs) and their polarisation behaviour. ECs line the inside of blood vessels and are essential for vessel formation and repair, as well as for various cardiovascular diseases, cancer, and inflammation. However, the general architecture of the software will allow it to be applied to other cell types and image modalities. The Polarity-JaM package integrates these analyses in a reproducible manner with a level of documentation that allows the user to analyse image data accurately and efficiently. The software suite is documented at https://polarityjam.readthedocs.io and the app runs online at www.polarityjam.com.\nREADv2: Advanced and user-friendly detection of biological relatedness in archaeogenomics\nAuthors: Alacamlı, E.; Naidoo, T.; Aktürk, S.; Güler, M. N.; Mapelli, I.; Vural, K. B.; Somel, M.; Malmström, H.; Günther, T.\nScore: 26.7, Published: 2024-01-23 DOI: 10.1101/2024.01.23.576660\nThe possibility to obtain genome-wide ancient DNA data from multiple individuals has facilitated an unprecedented perspective into prehistoric societies. Studying biological relatedness in these groups requires tailored approaches for analyzing ancient DNA due to its low coverage, post-mortem damage, and potential ascertainment bias. Here we present READv2 (Relatedness Estimation from Ancient DNA version 2), an improved Python 3 re-implementation of the most widely used tool for this purpose. While providing increased portability and making the software future-proof, we are also able to show that READv2 (a) is orders of magnitude faster than its predecessor; (b) has increased power to detect pairs of relatives using optimized default parameters; and, when the number of overlapping SNPs is sufficient, (c) can differentiate between full-siblings and parent-offspring, and (d) can classify pairs of third-degree relatedness. We further use READv2 to analyze a large empirical dataset that has previously needed two separate tools to reconstruct complex pedigrees. We show that READv2 yields results and precision similar to the combined approach but is faster and simpler to run. READv2 will become a valuable part of the archaeogenomic toolkit in providing an efficient and user-friendly classification of biological relatedness from pseudohaploid ancient DNA data.\nDeepGOMeta: Predicting functions for microbes}\nAuthors: Hoehndorf, R.; Kulmanov, M.; Tawfiq, R.; Niu, K.\nScore: 12.9, Published: 2024-01-30 DOI: 10.1101/2024.01.28.577602\nAnalyzing microbial samples remains computationally challenging due to their diversity and complexity. The lack of robust de novo protein function prediction methods exacerbates the difficulty in deriving functional insights from these samples. Traditional prediction methods, dependent on homology and sequence similarity, often fail to predict functions for novel proteins and proteins without known homologs. Moreover, most of these methods have been trained on largely eukaryotic data, and have not been evaluated or applied to microbial datasets. This research introduces DeepGOMeta, a deep learning model designed for protein function prediction, as Gene Ontology (GO) terms, trained on a dataset relevant to microbes. The model is validated using novel evaluation strategies and applied to diverse microbial datasets. Data and code are available at https://github.com/bio-ontology-research-group/deepgometa\nPeptidoform analysis of IP-MS data allows detection of differentially present bait proteoforms\nAuthors: Sdelci, S.; Sdelci, S.; Kourtis, S.; Cianferoni, D.; Serrano, L.\nScore: 12.6, Published: 2024-01-26 DOI: 10.1101/2024.01.23.576810\nWhile it is recognised that protein functions are determined by their proteoform state, such as mutations and post-translational modifications, methods to determine their differential abundance between conditions are limited. Here, we present a novel workflow for classical immunoprecipitation coupled to mass spectrometry (IP-MS) data that focuses on identifying differential peptidoforms of the bait protein between conditions, providing additional information about protein function.\nGenotype sampling for deep-learning assisted experimental mapping of fitness landscapes\nAuthors: Wagner, A.\nScore: 13.3, Published: 2024-01-23 DOI: 10.1101/2024.01.18.576262\nMotivationExperimental characterization of fitness landscapes, which map genotypes onto fitness, is important for both evolutionary biology and protein engineering. It faces a fundamental obstacle in the astronomical number of genotypes whose fitness needs to be measured for any one protein. Deep learning may help to predict the fitness of many genotypes from a smaller neural network training sample of genotypes with experimentally measured fitness. Here I use a recently published experimentally mapped fitness landscape of more than 260,000 protein genotypes to ask how such sampling is best performed. ResultsI show that multilayer perceptrons, recurrent neural networks (RNNs), convolutional networks, and transformers, can explain more than 90 percent of fitness variance in the data. In addition, 90 percent of this performance is reached with a training sample comprising merely {approx}103 sequences. Generalization to unseen test data is best when training data is sampled randomly and uniformly, or sampled to minimize the number of synonymous sequences. In contrast, sampling to maximize sequence diversity or codon usage bias reduces performance substantially. These observations hold for more than one network architecture. Simple sampling strategies may perform best when training deep learning neural networks to map fitness landscapes from experimental data.\nGenome-wide profiling of genetic variation at tandem repeat from long reads\nAuthors: Ziaei Jam, H.; Zook, J. M.; Javadzadeh, S.; Park, J.; Sehgal, A.; Gymrek, M.\nScore: 12.2, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576266\nTandem repeats are frequent across the human genome, and variation in repeat length has been linked to a variety of traits. Recent improvements in long read sequencing technologies have the potential to greatly improve TR analysis, especially for long or complex repeats. Here we introduce LongTR, which accurately genotypes tandem repeats from high fidelity long reads available from both PacBio and Oxford Nanopore Technologies. LongTR is freely available at https://github.com/gymrek-lab/longtr.\n",
  "wordCount" : "2142",
  "inLanguage": "en",
  "datePublished": "2024-01-31T10:38:28Z",
  "dateModified": "2024-01-31T10:38:28Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 31, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.21.576131">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.21.576131" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.21.576131">
        <p class="paperTitle">edgeR 4.0: powerful differential analysis of sequencing data with expanded functionality and improved support for small counts and larger datasets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.21.576131" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.21.576131" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, Y.; Chen, L.; Lun, A. T. L.; Baldoni, P. L.; Smyth, G. K.</p>
        <p class="info">Score: 79.5, Published: 2024-01-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.21.576131' target='https://doi.org/10.1101/2024.01.21.576131'> 10.1101/2024.01.21.576131</a></p>
        <p class="abstract">edgeR is an R/Bioconductor software package for differential analyses of sequencing data in the form of read counts for genes or genomic features. Over the past 15 years, edgeR has been a popular choice for statistical analysis of data from sequencing technologies such as RNA-seq or ChIP-seq. edgeR pioneered the use of the negative binomial distribution to model read count data with replicates and the use of generalized linear models to analyse complex experimental designs. edgeR implements empirical Bayes moderation methods to allow reliable inference when the number of replicates is small. This article announces edgeR version 4, which includes new developments across a range of application areas. Infrastructure improvements include support for fractional counts, implementation of model fitting in C&#43;&#43;, and a new statistical treatment of the quasi-likelihood pipeline that improves accuracy for small counts. The revised package has new functionality for differential methylation analysis, differential transcript expression, differential transcript and exon usage, testing relative to a fold-change threshold and pathway analysis. This article reviews the statistical framework and computational implementation of edgeR, briefly summarizing all the existing features and functionalities but with special attention to new features and those that have not been described previously.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.576848">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.576848" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.576848">
        <p class="paperTitle">Phylogenetic tree statistics: a systematic overview using the new R package &#39;treestats&#39;</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.576848" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.576848" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Janzen, T.; Etienne, R. S.</p>
        <p class="info">Score: 73.9, Published: 2024-01-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.576848' target='https://doi.org/10.1101/2024.01.24.576848'> 10.1101/2024.01.24.576848</a></p>
        <p class="abstract">Phylogenetic trees are believed to contain a wealth of information on diversification processes. Comparing phylogenetic trees is not straightforward due to their high dimensionality. Researchers have therefore defined a wide range of one-dimensional summary statistics. However, it remains unexplored to what extent these summary statistics cover the same underlying information and what summary statistics best explain observed variation across phylogenies. Furthermore, a large subset of available summary statistics focusses on measuring the topological features of a phylogenetic tree, but are often only explored at the extreme edge cases of the fully balanced or unbalanced tree and not for trees of intermediate balance. Here, we introduce a new R package that provides speed optimized code to compute 54 summary statistics. We study correlations between summary statistics on empirical trees and on trees simulated using several diversification models. Furthermore, we introduce an algorithm to create intermediately balanced trees in a well-defined manner, in order to explore variation in summary statistics across a balance gradient. We find that almost all summary statistics are correlated with tree size, and it is difficult if not impossible to correct for tree size, unless the tree generating model is known. Furthermore, we find that across empirical and simulated trees, at least two large clusters of correlated summary statistics can be found, where statistics group together based on information used (topology or branching times). However, the finer grained correlation structure appears to depend strongly on either the taxonomic group studied (in empirical studies) or the diversification model (in simulation studies). Nevertheless, we can identify multiple groups of summary statistics that are strongly and consistently correlated, indicating that these statistics measure the same underlying property of a tree. Lastly, we find that almost all topological summary statistics vary non-linearly and sometimes even non-monotonically with our intuitive balance gradient. Therefore, in order to avoid introducing biases and missing underlying information, we advocate for selecting as many summary statistics as possible in phylogenetic analyses. With the introduction of the treestats package, which provides fast and reliable calculations, such an approach is now routinely possible.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576317">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576317" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576317">
        <p class="paperTitle">CHOIR improves significance-based detection of cell types and states from single-cell data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576317" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576317" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Petersen, C.; Mucke, L.; Corces, M. R.</p>
        <p class="info">Score: 63.1, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576317' target='https://doi.org/10.1101/2024.01.18.576317'> 10.1101/2024.01.18.576317</a></p>
        <p class="abstract">Clustering is a critical step in the analysis of single-cell data, as it enables the discovery and characterization of putative cell types and states. However, most popular clustering tools do not subject clustering results to statistical inference testing, leading to risks of overclustering or underclustering data and often resulting in ineffective identification of cell types with widely differing prevalence. To address these challenges, we present CHOIR (clustering hierarchy optimization by iterative random forests), which applies a framework of random forest classifiers and permutation tests across a hierarchical clustering tree to statistically determine which clusters represent distinct populations. We demonstrate the enhanced performance of CHOIR through extensive benchmarking against 14 existing clustering methods across 100 simulated and 4 real single-cell RNA-seq, ATAC-seq, spatial transcriptomic, and multi-omic datasets. CHOIR can be applied to any single-cell data type and provides a flexible, scalable, and robust solution to the important challenge of identifying biologically relevant cell groupings within heterogeneous single-cell data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.25.577299">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.25.577299" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.25.577299">
        <p class="paperTitle">Beav: A bacterial genome and mobile element annotation pipeline</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.25.577299" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.25.577299" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jung, J. M.; Rahman, A.; Schiffer, A. M.; Weisberg, A. J.</p>
        <p class="info">Score: 28.4, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.25.577299' target='https://doi.org/10.1101/2024.01.25.577299'> 10.1101/2024.01.25.577299</a></p>
        <p class="abstract">Comprehensive and accurate genome annotation is crucial for inferring the predicted functions of an organism. Numerous tools exist to annotate genes, gene clusters, mobile genetic elements, and other diverse features. However, these tools and pipelines can be difficult to install and run, be specialized for a particular element or feature, or lack annotations for larger elements that provide important genomic context. Integrating results across analyses is also important for understanding gene function. To address these challenges, we present the Beav annotation pipeline. Beav is a command-line tool that automates the annotation of bacterial genome sequences, mobile genetic elements, molecular systems and gene clusters, key regulatory features, and other elements. Beav uses existing tools in addition to custom models, scripts, and databases to annotate diverse elements, systems, and sequence features. Custom databases for plant-associated microbes are incorporated to improve annotation of key virulence and symbiosis genes in agriculturally important pathogens and mutualists. Beav includes an optional Agrobacterium-specific pipeline that identifies and classifies oncogenic plasmids and annotates plasmid-specific features. Following the completion of all analyses, annotations are consolidated to produce a single comprehensive output. Finally, Beav generates publication-quality genome and plasmid maps. Beav is on Bioconda and is available for download at https://github.com/weisberglab/beav.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.577027">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.577027" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.577027">
        <p class="paperTitle">Polarity-JaM: An image analysis toolbox for cell polarity, junction and morphology quantification</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.577027" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.577027" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Giese, W.; Albrecht, J. P.; Oppenheim, O.; Akmeric, E. B.; Kraxner, J.; Schmidt, D.; Harrington, K. I.; Gerhardt, H.</p>
        <p class="info">Score: 25.5, Published: 2024-01-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.577027' target='https://doi.org/10.1101/2024.01.24.577027'> 10.1101/2024.01.24.577027</a></p>
        <p class="abstract">Cell polarity involves the asymmetric distribution of cellular components such as signaling molecules and organelles within a cell, asymmetries of a cell&#39;s shape as well as contacts with neighbouring cells. Gradients and mechanical forces often act as global cues that bias cell polarity and orientation, and polarity is coordinated by communication between adjacent cells. Advances in fluorescence microscopy combined with deep learning algorithms for image segmentation open up a wealth of possibilities to understand cell polarity behaviour in health and disease. We have therefore developed the open-source package Polarity-JaM, which offers versatile methods for performing reproducible exploratory image analysis. Multi-channel single cell segmentation is performed using a flexible and user-friendly interface to state-of-the-art deep learning algorithms. Interpretable single-cell features are automatically extracted, including cell and organelle orientation, cell-cell contact morphology, signaling molecule gradients, as well as collective orientation, tissue-wide size and shape variation. Circular statistics of cell polarity, including polarity indices, confidence intervals, and circular correlation analysis, can be computed using our web application. We have developed data graphs for comprehensive visualisation of key statistical measures and suggest the use of an adapted polarity index when the expected polarisation direction or the direction of a global cue is known a priori. The focus of our analysis is on fluorescence image data from endothelial cells (ECs) and their polarisation behaviour. ECs line the inside of blood vessels and are essential for vessel formation and repair, as well as for various cardiovascular diseases, cancer, and inflammation. However, the general architecture of the software will allow it to be applied to other cell types and image modalities. The Polarity-JaM package integrates these analyses in a reproducible manner with a level of documentation that allows the user to analyse image data accurately and efficiently. The software suite is documented at https://polarityjam.readthedocs.io and the app runs online at www.polarityjam.com.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.23.576660">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.23.576660" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.23.576660">
        <p class="paperTitle">READv2: Advanced and user-friendly detection of biological relatedness in archaeogenomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.23.576660" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.23.576660" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alacamlı, E.; Naidoo, T.; Aktürk, S.; Güler, M. N.; Mapelli, I.; Vural, K. B.; Somel, M.; Malmström, H.; Günther, T.</p>
        <p class="info">Score: 26.7, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.23.576660' target='https://doi.org/10.1101/2024.01.23.576660'> 10.1101/2024.01.23.576660</a></p>
        <p class="abstract">The possibility to obtain genome-wide ancient DNA data from multiple individuals has facilitated an unprecedented perspective into prehistoric societies. Studying biological relatedness in these groups requires tailored approaches for analyzing ancient DNA due to its low coverage, post-mortem damage, and potential ascertainment bias. Here we present READv2 (Relatedness Estimation from Ancient DNA version 2), an improved Python 3 re-implementation of the most widely used tool for this purpose. While providing increased portability and making the software future-proof, we are also able to show that READv2 (a) is orders of magnitude faster than its predecessor; (b) has increased power to detect pairs of relatives using optimized default parameters; and, when the number of overlapping SNPs is sufficient, (c) can differentiate between full-siblings and parent-offspring, and (d) can classify pairs of third-degree relatedness. We further use READv2 to analyze a large empirical dataset that has previously needed two separate tools to reconstruct complex pedigrees. We show that READv2 yields results and precision similar to the combined approach but is faster and simpler to run. READv2 will become a valuable part of the archaeogenomic toolkit in providing an efficient and user-friendly classification of biological relatedness from pseudohaploid ancient DNA data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.28.577602">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.28.577602" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.28.577602">
        <p class="paperTitle">DeepGOMeta: Predicting functions for microbes}</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.28.577602" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.28.577602" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hoehndorf, R.; Kulmanov, M.; Tawfiq, R.; Niu, K.</p>
        <p class="info">Score: 12.9, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.28.577602' target='https://doi.org/10.1101/2024.01.28.577602'> 10.1101/2024.01.28.577602</a></p>
        <p class="abstract">Analyzing microbial samples remains computationally challenging due to their diversity and complexity. The lack of robust de novo protein function prediction methods exacerbates the difficulty in deriving functional insights from these samples. Traditional prediction methods, dependent on homology and sequence similarity, often fail to predict functions for novel proteins and proteins without known homologs. Moreover, most of these methods have been trained on largely eukaryotic data, and have not been evaluated or applied to microbial datasets. This research introduces DeepGOMeta, a deep learning model designed for protein function prediction, as Gene Ontology (GO) terms, trained on a dataset relevant to microbes. The model is validated using novel evaluation strategies and applied to diverse microbial datasets. Data and code are available at https://github.com/bio-ontology-research-group/deepgometa</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.23.576810">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.23.576810" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.23.576810">
        <p class="paperTitle">Peptidoform analysis of IP-MS data allows detection of differentially present bait proteoforms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.23.576810" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.23.576810" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sdelci, S.; Sdelci, S.; Kourtis, S.; Cianferoni, D.; Serrano, L.</p>
        <p class="info">Score: 12.6, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.23.576810' target='https://doi.org/10.1101/2024.01.23.576810'> 10.1101/2024.01.23.576810</a></p>
        <p class="abstract">While it is recognised that protein functions are determined by their proteoform state, such as mutations and post-translational modifications, methods to determine their differential abundance between conditions are limited. Here, we present a novel workflow for classical immunoprecipitation coupled to mass spectrometry (IP-MS) data that focuses on identifying differential peptidoforms of the bait protein between conditions, providing additional information about protein function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576262">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576262" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576262">
        <p class="paperTitle">Genotype sampling for deep-learning assisted experimental mapping of fitness landscapes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576262" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576262" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wagner, A.</p>
        <p class="info">Score: 13.3, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576262' target='https://doi.org/10.1101/2024.01.18.576262'> 10.1101/2024.01.18.576262</a></p>
        <p class="abstract">MotivationExperimental characterization of fitness landscapes, which map genotypes onto fitness, is important for both evolutionary biology and protein engineering. It faces a fundamental obstacle in the astronomical number of genotypes whose fitness needs to be measured for any one protein. Deep learning may help to predict the fitness of many genotypes from a smaller neural network training sample of genotypes with experimentally measured fitness. Here I use a recently published experimentally mapped fitness landscape of more than 260,000 protein genotypes to ask how such sampling is best performed.

ResultsI show that multilayer perceptrons, recurrent neural networks (RNNs), convolutional networks, and transformers, can explain more than 90 percent of fitness variance in the data. In addition, 90 percent of this performance is reached with a training sample comprising merely {approx}103 sequences. Generalization to unseen test data is best when training data is sampled randomly and uniformly, or sampled to minimize the number of synonymous sequences. In contrast, sampling to maximize sequence diversity or codon usage bias reduces performance substantially. These observations hold for more than one network architecture. Simple sampling strategies may perform best when training deep learning neural networks to map fitness landscapes from experimental data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576266">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576266" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576266">
        <p class="paperTitle">Genome-wide profiling of genetic variation at tandem repeat from long reads</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576266" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576266" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ziaei Jam, H.; Zook, J. M.; Javadzadeh, S.; Park, J.; Sehgal, A.; Gymrek, M.</p>
        <p class="info">Score: 12.2, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576266' target='https://doi.org/10.1101/2024.01.20.576266'> 10.1101/2024.01.20.576266</a></p>
        <p class="abstract">Tandem repeats are frequent across the human genome, and variation in repeat length has been linked to a variety of traits. Recent improvements in long read sequencing technologies have the potential to greatly improve TR analysis, especially for long or complex repeats. Here we introduce LongTR, which accurately genotypes tandem repeats from high fidelity long reads available from both PacBio and Oxford Nanopore Technologies. LongTR is freely available at https://github.com/gymrek-lab/longtr.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
