<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Eukfinder: a pipeline to retrieve microbial eukaryote genomes from metagenomic sequencing data.
Authors: Zhao, D.; Salas-Leiva, D. E.; Williams, S. K.; Dunn, K. A.; Roger, A. J.
Score: 39.2, Published: 2023-12-28 DOI: 10.1101/2023.12.28.573569
Whole-genome shotgun (WGS) metagenomic sequencing of microbial communities allows us to discover the functions, physiologies, and evolutionary histories of microbial prokaryote and eukaryote members of diverse ecosystems. Despite their importance, metagenomic studies of microbial eukaryotes lag behind those of prokaryotes, due to the difficulty in identifying and assembling high-quality eukaryotic genomes from WGS data.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="Eukfinder: a pipeline to retrieve microbial eukaryote genomes from metagenomic sequencing data.
Authors: Zhao, D.; Salas-Leiva, D. E.; Williams, S. K.; Dunn, K. A.; Roger, A. J.
Score: 39.2, Published: 2023-12-28 DOI: 10.1101/2023.12.28.573569
Whole-genome shotgun (WGS) metagenomic sequencing of microbial communities allows us to discover the functions, physiologies, and evolutionary histories of microbial prokaryote and eukaryote members of diverse ecosystems. Despite their importance, metagenomic studies of microbial eukaryotes lag behind those of prokaryotes, due to the difficulty in identifying and assembling high-quality eukaryotic genomes from WGS data." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-31T10:38:19+00:00" />
<meta property="article:modified_time" content="2023-12-31T10:38:19+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="Eukfinder: a pipeline to retrieve microbial eukaryote genomes from metagenomic sequencing data.
Authors: Zhao, D.; Salas-Leiva, D. E.; Williams, S. K.; Dunn, K. A.; Roger, A. J.
Score: 39.2, Published: 2023-12-28 DOI: 10.1101/2023.12.28.573569
Whole-genome shotgun (WGS) metagenomic sequencing of microbial communities allows us to discover the functions, physiologies, and evolutionary histories of microbial prokaryote and eukaryote members of diverse ecosystems. Despite their importance, metagenomic studies of microbial eukaryotes lag behind those of prokaryotes, due to the difficulty in identifying and assembling high-quality eukaryotic genomes from WGS data."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "Eukfinder: a pipeline to retrieve microbial eukaryote genomes from metagenomic sequencing data.\nAuthors: Zhao, D.; Salas-Leiva, D. E.; Williams, S. K.; Dunn, K. A.; Roger, A. J.\nScore: 39.2, Published: 2023-12-28 DOI: 10.1101/2023.12.28.573569\nWhole-genome shotgun (WGS) metagenomic sequencing of microbial communities allows us to discover the functions, physiologies, and evolutionary histories of microbial prokaryote and eukaryote members of diverse ecosystems. Despite their importance, metagenomic studies of microbial eukaryotes lag behind those of prokaryotes, due to the difficulty in identifying and assembling high-quality eukaryotic genomes from WGS data.",
  "keywords": [
    
  ],
  "articleBody": " Eukfinder: a pipeline to retrieve microbial eukaryote genomes from metagenomic sequencing data.\nAuthors: Zhao, D.; Salas-Leiva, D. E.; Williams, S. K.; Dunn, K. A.; Roger, A. J.\nScore: 39.2, Published: 2023-12-28 DOI: 10.1101/2023.12.28.573569\nWhole-genome shotgun (WGS) metagenomic sequencing of microbial communities allows us to discover the functions, physiologies, and evolutionary histories of microbial prokaryote and eukaryote members of diverse ecosystems. Despite their importance, metagenomic studies of microbial eukaryotes lag behind those of prokaryotes, due to the difficulty in identifying and assembling high-quality eukaryotic genomes from WGS data. To address this problem, we have developed Eukfinder, a bioinformatics pipeline that recovers and assembles nuclear and mitochondrial genomes of eukaryotic microbes from WGS metagenomics data. As part of its workflow, it utilizes two specialized databases to classify reads based on taxonomy which can be customized to the dataset or environment of interest. We applied Eukfinder to human gut microbiome WGS metagenomic sequencing data to recover genomes from the protistan parasite Blastocystis sp., a highly prevalent colonizer of the gastrointestinal tract of humans and animals. We tested Eukfinder using both a series of simulated gut microbiome datasets, which included varying numbers of Blastocystis reads combined with bacterial reads and by using real metagenomic gut samples containing Blastocystis. We compared the results of Eukfinder with other published workflows. With sufficient reads, Eukfinder efficiently assembles high-quality near-complete nuclear and mitochondrial genomes from diverse Blastocystis subtypes from metagenomic data without the aid of a reference genome. Furthermore, with sufficient depth of sequence sampling, Eukfinder outperforms similar tools used to recover eukaryotic genomes from metagenomic data. Eukfinder will be a useful tool for reference-independent and cultivation-free study of eukaryotic microbial genomes from environmental metagenomic sequencing samples.\nInvestigating the impact of whole genome duplication on transposable element evolution in ray-finned fishes\nAuthors: Mallik, R.; Wcisel, D. J.; Yoder, J. A.; Near, T. J.; Dornburg, A.\nScore: 27.4, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572151\nTransposable elements (TEs) can make up more than 50% of any given vertebrates genome, with substantial variability in TE composition among lineages. TE variation is often linked to changes in gene regulation, genome size, and speciation. However, the role that genome duplication events have played in generating abrupt shifts in the composition of the mobilome over macroevolutionary timescales remains unclear. We investigated the degree to which the teleost genome duplication (TGD) shaped the diversification trajectory of the ray-finned fish mobilome. We integrate a new high coverage genome of Polypterus bichir with data from over 100 publicly available actinopterygian genomes to assess the macroevolutionary implications of genome duplication events on TE evolution. Our results provide no evidence for a substantial shift in mobilome composition following the TGD event. Instead, the diversity of the actinopterygian mobilome appears to have been shaped by a history of lineage specific shifts in composition that are not correlated with commonly evoked drivers of diversification such as body size, water column usage, or latitude. Collectively, these results provide a new perspective on the early diversification of the actinopterygian mobilome and suggest that historic ploidy events may not necessarily catalyze bursts of TE diversification and innovation.\nBioLLMBench: A Comprehensive Benchmarking of Large Language Models in Bioinformatics\nAuthors: Sarwal, V.; Munteanu, V.; Suhodolschi, T.; Ciorba, D.; Eskin, E.; Wang, W.; Mangul, S.\nScore: 42.2, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572483\nLarge Language Models (LLMs) have shown great promise in their knowledge integration and problem-solving capabilities, but their ability to assist in bioinformatics research has not been systematically evaluated. To bridge this gap, we present BioLLMBench, a novel benchmarking framework coupled with a scoring metric scheme for comprehensively evaluating LLMs in solving bioinformatics tasks. Through BioLLMBench, we conducted a thorough evaluation of 2,160 experimental runs of the three most widely used models, GPT-4, Bard and LLaMA, focusing on 36 distinct tasks within the field of bioinformatics. The tasks come from six key areas of emphasis within bioinformatics that directly relate to the daily challenges and tasks faced by individuals within the field. These areas are domain expertise, mathematical problem-solving, coding proficiency, data visualization, summarizing research papers, and developing machine learning models. The tasks also span across varying levels of complexity, ranging from fundamental concepts to expert-level challenges. Each key area was evaluated using seven specifically designed task metrics, which were then used to conduct an overall evaluation of the LLMs response. To enhance our understanding of model responses under varying conditions, we implemented a Contextual Response Variability Analysis. Our results reveal a diverse spectrum of model performance, with GPT-4 leading in all tasks except mathematical problem solving. GPT4 was able to achieve an overall proficiency score of 91.3% in domain knowledge tasks, while Bard excelled in mathematical problem-solving with a 97.5% success rate. While GPT-4 outperformed in machine learning model development tasks with an average accuracy of 65.32%, both Bard and LLaMA were unable to generate executable end-to-end code. All models faced considerable challenges in research paper summarization, with none of them exceeding a 40% score in our evaluation using the Recall-Oriented Understudy for Gisting Evaluation (ROUGE) score, highlighting a significant area for future improvement. We observed an increase in model performance variance when using a new chatting window compared to using the same chat, although the average scores between the two contextual environments remained similar. Lastly, we discuss various limitations of these models and acknowledge the risks associated with their potential misuse.\nSOAPy: a Python package to dissect spatial architecture, dynamics and communication\nAuthors: Wang, H.; Li, J.; Jing, S.; Lin, P.; Li, Y.; Zhang, H.; Chen, Y.; Wang, Z.; Li, H.\nScore: 25.9, Published: 2023-12-22 DOI: 10.1101/2023.12.21.572725\nAdvances in spatial omics technologies have brought opportunities to dissect tissue microenvironment, while also posing more requirements and challenges for computational methods. Here we developed a package SOAPy to systematically dissect spatial architecture, dynamics and communication from spatial omics data. Specifically, it provides analysis methods for multiple spatial-related tasks, including spatial domain, spatial expression tendency, spatiotemporal expression pattern, cellular co-localization, multi-cellular niches, and ligand-receptor-mediated and spatial-constrained cell communication. Applying SOAPy on different spatial omics technologies and diverse biological fields has demonstrated its power on elucidation of biological questions about tumors, embryonic development, and normal physiological structures. Overall, SOAPy is a universal tool for spatial omics analysis, providing a foundation for continued investigation of the microenvironment.\nSopa: a technology-invariant pipeline for analyses of image-based spatial-omics\nAuthors: Blampey, Q.; Mulder, K.; Gardet, M.; Dutertre, C.-A.; Andre, F.; Ginhoux, F.; Cournede, P.-H.\nScore: 21.7, Published: 2023-12-23 DOI: 10.1101/2023.12.22.571863\nSpatial-omics data allow in-depth analysis of tissue architectures, opening new opportunities for biological discovery. In particular, imaging techniques offer single-cell resolutions, providing essential insights into cellular organizations and dynamics. Yet, the complexity of such data presents analytical challenges and demands substantial computing resources. Moreover, the proliferation of diverse spatial-omics technologies, such as Xenium, MERSCOPE, CosMX in spatial-transcriptomics, and MACSima and PhenoCycler in multiplex imaging, hinders the generality of existing tools. We introduce Sopa (https://github.com/gustaveroussy/sopa), a technology-invariant, memory-efficient pipeline with a unified visualizer for all image-based spatial omics. Built upon the universal SpatialData framework, Sopa optimizes tasks like segmentation, transcript/channel aggregation, annotation, and geometric/spatial analysis. Its output includes user-friendly web reports and visualizer files, as well as comprehensive data files for in-depth analysis. Overall, Sopa represents a significant step toward unifying spatial data analysis, enabling a more comprehensive understanding of cellular interactions and tissue organization in biological systems.\nHyLight: Strain aware assembly of low coverage metagenomes\nAuthors: Kang, X.; Zhang, W.; Luo, X.; Schonnhuth, A.\nScore: 17.7, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572963\nDifferent strains of identical species can vary substantially in terms of their spectrum of biomedically relevant phenotypes. Reconstructing the genomes of a microbial community at strain level poses major methodical challenges, because relative frequencies of individual strains match the rate of sequencing errors, which hampers the identification of their characteristic genetic variants. While next-generation sequencing (NGS) reads are too short to span complex repetitive regions, the considerably longer third-generation sequencing (TGS) reads are affected by larger sequencing error rates or are just significantly more expensive. Suppressing TGS coverage to limit costs implies losses in terms of the accuracy of the assemblies. Therefore, existing approaches have remained fragmentary: all approaches presented so far agree on losses in strain awareness, accuracy, possibly excessive costs of the assemblies, or combinations thereof. We present HyLight as, to the best of our knowledge, the first metagenome assembly approach that is not affected by any of the aforementioned drawbacks. In experiments, we demonstrate that HyLight assemblies are strain-aware, contiguous, contain little errors, and because operating on low coverage TGS data of the cheap kind, come at drastically reduced costs. HyLight implements hybrid assembly, which exploits the complementarity of TGS and NGS data. For unifying the two types of data, HyLight utilizes strain resolved overlap graphs (OG), which support the accurate reconstruction of the individual members of microbial communities at strain level: HyLight outperforms existing approaches in terms of strain identity preserving sequence by on average 25.53% (across all experiments / data sets: first quartile: 21.53%, median: 26.81%, third quartile: 31.98%), achieving near-complete strain awareness on many data sets. In summary, HyLight appears to implement the first protocol that delivers assemblies that are strain-aware, contiguous and accurate in combination.\nPredicting the Structural Impact of Human Alternative Splicing\nAuthors: Song, Y.; ZHANG, C.; Omenn, G. S.; O'Meara, M. J.; Welch, J. D.\nScore: 31.2, Published: 2023-12-24 DOI: 10.1101/2023.12.21.572928\nProtein structure prediction with neural networks is a powerful new method for linking protein sequence, structure, and function, but structures have generally been predicted for only a single isoform of each gene, neglecting splice variants. To investigate the structural implications of alternative splicing, we used AlphaFold2 to predict the structures of more than 11,000 human isoforms. We employed multiple metrics to identify splicing-induced structural alterations, including template matching score, secondary structure composition, surface charge distribution, radius of gyration, accessibility of post-translational modification sites, and structure-based function prediction. We identified examples of how alternative splicing induced clear changes in each of these properties. Structural similarity between isoforms largely correlated with degree of sequence identity, but we identified a subset of isoforms with low structural similarity despite high sequence similarity. Exon skipping and alternative last exons tended to increase the surface charge and radius of gyration. Splicing also buried or exposed numerous post-translational modification sites, most notably among the isoforms of BAX. Functional prediction nominated numerous functional differences among isoforms of the same gene, with loss of function compared to the reference predominating. Finally, we used single-cell RNA-seq data from the Tabula Sapiens to determine the cell types in which each structure is expressed. Our work represents an important resource for studying the structure and function of splice isoforms across the cell types of the human body.\nSingle-cell and spatial multiomic inference of gene regulatory networks using SCRIPro\nAuthors: Chang, Z.; Xu, Y.; Dong, X.; Gao, Y.; Wang, C.\nScore: 16.0, Published: 2023-12-23 DOI: 10.1101/2023.12.21.572934\nThe accurate reconstruction of gene regulation networks (GRNs) from sparse and noisy single-cell or spatial multi-omics data remains a challenge. Here, we present SCRIPro, a comprehensive computational framework that robustly infers GRNs for both single-cell and spatial multi-omics data. SCRIPro first addresses sample sparseness by a density clustering approach. SCRIPro assesses transcriptional regulator (TR) importance through chromatin reconstruction and in silico deletion, referencing 1,292 human and 994 mouse TRs. It combines TR-target importance scores with expression levels for precise GRN reconstruction. Finally, we benchmarked SCRIPro on diverse datasets, it outperforms existing motif-based methods and accurately reconstructs cell type-specific, stage-specific, and region-specific GRNs.\nA catalogue of small proteins from the global microbiome\nAuthors: Duan, Y.; Santos Junior, C. D.; Schmidt, T. S. B.; Fullam, A.; de Almeida, B. L. S.; Zhu, C.; Michael, K.; Zhao, X.-M.; Bork, P.; Coelho, L. P.\nScore: 14.8, Published: 2023-12-28 DOI: 10.1101/2023.12.27.573469\nSmall open reading frames (smORFs) shorter than 100 codons are widespread and perform essential roles in microorganisms, where they encode proteins active in several cell functions, including signal pathways, stress response, and antibacterial activities. However, the ecology, distribution and role of small proteins in the global microbiome remain unknown. Here, we constructed a global microbial smORFs catalogue (GMSC) derived from 63,410 publicly available metagenomes across 75 distinct habitats and 87,920 high-quality isolate genomes. GMSC contains 965 million non-redundant smORFs with comprehensive annotations. We found that archaea harbor more small proteins proportionally than bacteria. We moreover provide a tool called GMSC-mapper to identify and annotate small proteins from microbial (meta)genomes. Overall, this publicly-available resource demonstrates the immense and underexplored diversity of small proteins.\nTying the Knot: Unraveling the Intricacies of the Coronavirus Frameshift Pseudoknot\nAuthors: Trinity, L.; Stege, U.; Jabbari, H.\nScore: 13.0, Published: 2023-12-28 DOI: 10.1101/2023.12.28.573501\nUnderstanding and targeting functional RNA structures towards treatment of coronavirus infection can help us to prepare for novel variants of SARS-CoV-2 (the virus causing COVID-19), and any other coronaviruses that could emerge via human-to-human transmission or potential zoonotic (inter-species) events. Leveraging the fact that all coronaviruses use a mechanism known as -1 programmed ribosomal frameshifting (-1 PRF) to replicate, we apply algorithms to predict the most energetically favourable secondary structures (each nucleotide involved in at most one pairing) that may be involved in regulating the -1 PRF event in coronaviruses, especially SARS-CoV-2. We compute previously unknown most stable structure predictions for the frameshift site of coronaviruses via hierarchical folding, a biologically motivated framework where initial non-crossing structure folds first, followed by subsequent, possibly crossing (pseudoknotted), structures. Using mutual information from 181 coronavirus sequences, in conjunction with the algorithm KnotAli, we compute secondary structure predictions for the frameshift site of different coronaviruses. We then utilize the Shapify algorithm to obtain most stable SARS-CoV-2 secondary structure predictions guided by frameshift sequence-specific and genome-wide experimental data. We build on our previous secondary structure investigation of the singular SARS-CoV-2 68 nt frameshift element sequence, by using Shapify to obtain predictions for 132 extended sequences and including covariation information. Previous investigations have not applied hierarchical folding to extended length SARS-CoV-2 frameshift sequences. By doing so, we simulate the effects of ribosome interaction with the frameshift site, providing insight to biological function. We contribute in-depth discussion to contextualize secondary structure dual-graph motifs for SARS-CoV-2, highlighting the energetic stability of the previously identified 3_8 motif alongside the known dominant 3_3 and 3_6 (native-type) -1 PRF structures. Integrating experimental data within minimum free energy (MFE) hierarchical folding algorithms provides novel structure predictions to distill the relationship between RNA structure and function. In particular, fully categorizing most stable secondary structure predictions via hierarchical folding supports our identification of motif transitions and critical site targets for future therapeutic research.\n",
  "wordCount" : "2409",
  "inLanguage": "en",
  "datePublished": "2023-12-31T10:38:19Z",
  "dateModified": "2023-12-31T10:38:19Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 31, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.573569">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.573569" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.573569">
        <p class="paperTitle">Eukfinder: a pipeline to retrieve microbial eukaryote genomes from metagenomic sequencing data.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.573569" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.573569" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhao, D.; Salas-Leiva, D. E.; Williams, S. K.; Dunn, K. A.; Roger, A. J.</p>
        <p class="info">Score: 39.2, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.573569' target='https://doi.org/10.1101/2023.12.28.573569'> 10.1101/2023.12.28.573569</a></p>
        <p class="abstract">Whole-genome shotgun (WGS) metagenomic sequencing of microbial communities allows us to discover the functions, physiologies, and evolutionary histories of microbial prokaryote and eukaryote members of diverse ecosystems. Despite their importance, metagenomic studies of microbial eukaryotes lag behind those of prokaryotes, due to the difficulty in identifying and assembling high-quality eukaryotic genomes from WGS data. To address this problem, we have developed Eukfinder, a bioinformatics pipeline that recovers and assembles nuclear and mitochondrial genomes of eukaryotic microbes from WGS metagenomics data. As part of its workflow, it utilizes two specialized databases to classify reads based on taxonomy which can be customized to the dataset or environment of interest. We applied Eukfinder to human gut microbiome WGS metagenomic sequencing data to recover genomes from the protistan parasite Blastocystis sp., a highly prevalent colonizer of the gastrointestinal tract of humans and animals. We tested Eukfinder using both a series of simulated gut microbiome datasets, which included varying numbers of Blastocystis reads combined with bacterial reads and by using real metagenomic gut samples containing Blastocystis. We compared the results of Eukfinder with other published workflows. With sufficient reads, Eukfinder efficiently assembles high-quality near-complete nuclear and mitochondrial genomes from diverse Blastocystis subtypes from metagenomic data without the aid of a reference genome. Furthermore, with sufficient depth of sequence sampling, Eukfinder outperforms similar tools used to recover eukaryotic genomes from metagenomic data. Eukfinder will be a useful tool for reference-independent and cultivation-free study of eukaryotic microbial genomes from environmental metagenomic sequencing samples.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.572151">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.572151" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.572151">
        <p class="paperTitle">Investigating the impact of whole genome duplication on transposable element evolution in ray-finned fishes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.572151" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.572151" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mallik, R.; Wcisel, D. J.; Yoder, J. A.; Near, T. J.; Dornburg, A.</p>
        <p class="info">Score: 27.4, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.572151' target='https://doi.org/10.1101/2023.12.22.572151'> 10.1101/2023.12.22.572151</a></p>
        <p class="abstract">Transposable elements (TEs) can make up more than 50% of any given vertebrates genome, with substantial variability in TE composition among lineages. TE variation is often linked to changes in gene regulation, genome size, and speciation. However, the role that genome duplication events have played in generating abrupt shifts in the composition of the mobilome over macroevolutionary timescales remains unclear. We investigated the degree to which the teleost genome duplication (TGD) shaped the diversification trajectory of the ray-finned fish mobilome. We integrate a new high coverage genome of Polypterus bichir with data from over 100 publicly available actinopterygian genomes to assess the macroevolutionary implications of genome duplication events on TE evolution. Our results provide no evidence for a substantial shift in mobilome composition following the TGD event. Instead, the diversity of the actinopterygian mobilome appears to have been shaped by a history of lineage specific shifts in composition that are not correlated with commonly evoked drivers of diversification such as body size, water column usage, or latitude. Collectively, these results provide a new perspective on the early diversification of the actinopterygian mobilome and suggest that historic ploidy events may not necessarily catalyze bursts of TE diversification and innovation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572483">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572483" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572483">
        <p class="paperTitle">BioLLMBench: A Comprehensive Benchmarking of Large Language Models in Bioinformatics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572483" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572483" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sarwal, V.; Munteanu, V.; Suhodolschi, T.; Ciorba, D.; Eskin, E.; Wang, W.; Mangul, S.</p>
        <p class="info">Score: 42.2, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572483' target='https://doi.org/10.1101/2023.12.19.572483'> 10.1101/2023.12.19.572483</a></p>
        <p class="abstract">Large Language Models (LLMs) have shown great promise in their knowledge integration and problem-solving capabilities, but their ability to assist in bioinformatics research has not been systematically evaluated. To bridge this gap, we present BioLLMBench, a novel benchmarking framework coupled with a scoring metric scheme for comprehensively evaluating LLMs in solving bioinformatics tasks. Through BioLLMBench, we conducted a thorough evaluation of 2,160 experimental runs of the three most widely used models, GPT-4, Bard and LLaMA, focusing on 36 distinct tasks within the field of bioinformatics. The tasks come from six key areas of emphasis within bioinformatics that directly relate to the daily challenges and tasks faced by individuals within the field. These areas are domain expertise, mathematical problem-solving, coding proficiency, data visualization, summarizing research papers, and developing machine learning models. The tasks also span across varying levels of complexity, ranging from fundamental concepts to expert-level challenges. Each key area was evaluated using seven specifically designed task metrics, which were then used to conduct an overall evaluation of the LLMs response. To enhance our understanding of model responses under varying conditions, we implemented a Contextual Response Variability Analysis. Our results reveal a diverse spectrum of model performance, with GPT-4 leading in all tasks except mathematical problem solving. GPT4 was able to achieve an overall proficiency score of 91.3% in domain knowledge tasks, while Bard excelled in mathematical problem-solving with a 97.5% success rate. While GPT-4 outperformed in machine learning model development tasks with an average accuracy of 65.32%, both Bard and LLaMA were unable to generate executable end-to-end code. All models faced considerable challenges in research paper summarization, with none of them exceeding a 40% score in our evaluation using the Recall-Oriented Understudy for Gisting Evaluation (ROUGE) score, highlighting a significant area for future improvement. We observed an increase in model performance variance when using a new chatting window compared to using the same chat, although the average scores between the two contextual environments remained similar. Lastly, we discuss various limitations of these models and acknowledge the risks associated with their potential misuse.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572725">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572725" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572725">
        <p class="paperTitle">SOAPy: a Python package to dissect spatial architecture, dynamics and communication</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572725" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572725" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, H.; Li, J.; Jing, S.; Lin, P.; Li, Y.; Zhang, H.; Chen, Y.; Wang, Z.; Li, H.</p>
        <p class="info">Score: 25.9, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572725' target='https://doi.org/10.1101/2023.12.21.572725'> 10.1101/2023.12.21.572725</a></p>
        <p class="abstract">Advances in spatial omics technologies have brought opportunities to dissect tissue microenvironment, while also posing more requirements and challenges for computational methods. Here we developed a package SOAPy to systematically dissect spatial architecture, dynamics and communication from spatial omics data. Specifically, it provides analysis methods for multiple spatial-related tasks, including spatial domain, spatial expression tendency, spatiotemporal expression pattern, cellular co-localization, multi-cellular niches, and ligand-receptor-mediated and spatial-constrained cell communication. Applying SOAPy on different spatial omics technologies and diverse biological fields has demonstrated its power on elucidation of biological questions about tumors, embryonic development, and normal physiological structures. Overall, SOAPy is a universal tool for spatial omics analysis, providing a foundation for continued investigation of the microenvironment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.571863">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.571863" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.571863">
        <p class="paperTitle">Sopa: a technology-invariant pipeline for analyses of image-based spatial-omics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.571863" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.571863" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Blampey, Q.; Mulder, K.; Gardet, M.; Dutertre, C.-A.; Andre, F.; Ginhoux, F.; Cournede, P.-H.</p>
        <p class="info">Score: 21.7, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.571863' target='https://doi.org/10.1101/2023.12.22.571863'> 10.1101/2023.12.22.571863</a></p>
        <p class="abstract">Spatial-omics data allow in-depth analysis of tissue architectures, opening new opportunities for biological discovery. In particular, imaging techniques offer single-cell resolutions, providing essential insights into cellular organizations and dynamics. Yet, the complexity of such data presents analytical challenges and demands substantial computing resources. Moreover, the proliferation of diverse spatial-omics technologies, such as Xenium, MERSCOPE, CosMX in spatial-transcriptomics, and MACSima and PhenoCycler in multiplex imaging, hinders the generality of existing tools. We introduce Sopa (https://github.com/gustaveroussy/sopa), a technology-invariant, memory-efficient pipeline with a unified visualizer for all image-based spatial omics. Built upon the universal SpatialData framework, Sopa optimizes tasks like segmentation, transcript/channel aggregation, annotation, and geometric/spatial analysis. Its output includes user-friendly web reports and visualizer files, as well as comprehensive data files for in-depth analysis. Overall, Sopa represents a significant step toward unifying spatial data analysis, enabling a more comprehensive understanding of cellular interactions and tissue organization in biological systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.572963">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.572963" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.572963">
        <p class="paperTitle">HyLight: Strain aware assembly of low coverage metagenomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.572963" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.572963" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kang, X.; Zhang, W.; Luo, X.; Schonnhuth, A.</p>
        <p class="info">Score: 17.7, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.572963' target='https://doi.org/10.1101/2023.12.22.572963'> 10.1101/2023.12.22.572963</a></p>
        <p class="abstract">Different strains of identical species can vary substantially in terms of their spectrum of biomedically relevant phenotypes. Reconstructing the genomes of a microbial community at strain level poses major methodical challenges, because relative frequencies of individual strains match the rate of sequencing errors, which hampers the identification of their characteristic genetic variants. While next-generation sequencing (NGS) reads are too short to span complex repetitive regions, the considerably longer third-generation sequencing (TGS) reads are affected by larger sequencing error rates or are just significantly more expensive. Suppressing TGS coverage to limit costs implies losses in terms of the accuracy of the assemblies. Therefore, existing approaches have remained fragmentary: all approaches presented so far agree on losses in strain awareness, accuracy, possibly excessive costs of the assemblies, or combinations thereof.

We present HyLight as, to the best of our knowledge, the first metagenome assembly approach that is not affected by any of the aforementioned drawbacks. In experiments, we demonstrate that HyLight assemblies are strain-aware, contiguous, contain little errors, and because operating on low coverage TGS data of the cheap kind, come at drastically reduced costs. HyLight implements hybrid assembly, which exploits the complementarity of TGS and NGS data. For unifying the two types of data, HyLight utilizes strain resolved overlap graphs (OG), which support the accurate reconstruction of the individual members of microbial communities at strain level: HyLight outperforms existing approaches in terms of strain identity preserving sequence by on average 25.53% (across all experiments / data sets: first quartile: 21.53%, median: 26.81%, third quartile: 31.98%), achieving near-complete strain awareness on many data sets. In summary, HyLight appears to implement the first protocol that delivers assemblies that are strain-aware, contiguous and accurate in combination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572928">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572928" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572928">
        <p class="paperTitle">Predicting the Structural Impact of Human Alternative Splicing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572928" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572928" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Song, Y.; ZHANG, C.; Omenn, G. S.; O&#39;Meara, M. J.; Welch, J. D.</p>
        <p class="info">Score: 31.2, Published: 2023-12-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572928' target='https://doi.org/10.1101/2023.12.21.572928'> 10.1101/2023.12.21.572928</a></p>
        <p class="abstract">Protein structure prediction with neural networks is a powerful new method for linking protein sequence, structure, and function, but structures have generally been predicted for only a single isoform of each gene, neglecting splice variants. To investigate the structural implications of alternative splicing, we used AlphaFold2 to predict the structures of more than 11,000 human isoforms. We employed multiple metrics to identify splicing-induced structural alterations, including template matching score, secondary structure composition, surface charge distribution, radius of gyration, accessibility of post-translational modification sites, and structure-based function prediction. We identified examples of how alternative splicing induced clear changes in each of these properties. Structural similarity between isoforms largely correlated with degree of sequence identity, but we identified a subset of isoforms with low structural similarity despite high sequence similarity. Exon skipping and alternative last exons tended to increase the surface charge and radius of gyration. Splicing also buried or exposed numerous post-translational modification sites, most notably among the isoforms of BAX. Functional prediction nominated numerous functional differences among isoforms of the same gene, with loss of function compared to the reference predominating. Finally, we used single-cell RNA-seq data from the Tabula Sapiens to determine the cell types in which each structure is expressed. Our work represents an important resource for studying the structure and function of splice isoforms across the cell types of the human body.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572934">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572934" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572934">
        <p class="paperTitle">Single-cell and spatial multiomic inference of gene regulatory networks using SCRIPro</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572934" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572934" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chang, Z.; Xu, Y.; Dong, X.; Gao, Y.; Wang, C.</p>
        <p class="info">Score: 16.0, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572934' target='https://doi.org/10.1101/2023.12.21.572934'> 10.1101/2023.12.21.572934</a></p>
        <p class="abstract">The accurate reconstruction of gene regulation networks (GRNs) from sparse and noisy single-cell or spatial multi-omics data remains a challenge. Here, we present SCRIPro, a comprehensive computational framework that robustly infers GRNs for both single-cell and spatial multi-omics data. SCRIPro first addresses sample sparseness by a density clustering approach. SCRIPro assesses transcriptional regulator (TR) importance through chromatin reconstruction and in silico deletion, referencing 1,292 human and 994 mouse TRs. It combines TR-target importance scores with expression levels for precise GRN reconstruction. Finally, we benchmarked SCRIPro on diverse datasets, it outperforms existing motif-based methods and accurately reconstructs cell type-specific, stage-specific, and region-specific GRNs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.27.573469">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.27.573469" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.27.573469">
        <p class="paperTitle">A catalogue of small proteins from the global microbiome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.27.573469" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.27.573469" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Duan, Y.; Santos Junior, C. D.; Schmidt, T. S. B.; Fullam, A.; de Almeida, B. L. S.; Zhu, C.; Michael, K.; Zhao, X.-M.; Bork, P.; Coelho, L. P.</p>
        <p class="info">Score: 14.8, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.27.573469' target='https://doi.org/10.1101/2023.12.27.573469'> 10.1101/2023.12.27.573469</a></p>
        <p class="abstract">Small open reading frames (smORFs) shorter than 100 codons are widespread and perform essential roles in microorganisms, where they encode proteins active in several cell functions, including signal pathways, stress response, and antibacterial activities. However, the ecology, distribution and role of small proteins in the global microbiome remain unknown. Here, we constructed a global microbial smORFs catalogue (GMSC) derived from 63,410 publicly available metagenomes across 75 distinct habitats and 87,920 high-quality isolate genomes. GMSC contains 965 million non-redundant smORFs with comprehensive annotations. We found that archaea harbor more small proteins proportionally than bacteria. We moreover provide a tool called GMSC-mapper to identify and annotate small proteins from microbial (meta)genomes. Overall, this publicly-available resource demonstrates the immense and underexplored diversity of small proteins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.573501">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.573501" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.573501">
        <p class="paperTitle">Tying the Knot: Unraveling the Intricacies of the Coronavirus Frameshift Pseudoknot</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.573501" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.573501" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Trinity, L.; Stege, U.; Jabbari, H.</p>
        <p class="info">Score: 13.0, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.573501' target='https://doi.org/10.1101/2023.12.28.573501'> 10.1101/2023.12.28.573501</a></p>
        <p class="abstract">Understanding and targeting functional RNA structures towards treatment of coronavirus infection can help us to prepare for novel variants of SARS-CoV-2 (the virus causing COVID-19), and any other coronaviruses that could emerge via human-to-human transmission or potential zoonotic (inter-species) events. Leveraging the fact that all coronaviruses use a mechanism known as -1 programmed ribosomal frameshifting (-1 PRF) to replicate, we apply algorithms to predict the most energetically favourable secondary structures (each nucleotide involved in at most one pairing) that may be involved in regulating the -1 PRF event in coronaviruses, especially SARS-CoV-2. We compute previously unknown most stable structure predictions for the frameshift site of coronaviruses via hierarchical folding, a biologically motivated framework where initial non-crossing structure folds first, followed by subsequent, possibly crossing (pseudoknotted), structures. Using mutual information from 181 coronavirus sequences, in conjunction with the algorithm KnotAli, we compute secondary structure predictions for the frameshift site of different coronaviruses. We then utilize the Shapify algorithm to obtain most stable SARS-CoV-2 secondary structure predictions guided by frameshift sequence-specific and genome-wide experimental data. We build on our previous secondary structure investigation of the singular SARS-CoV-2 68 nt frameshift element sequence, by using Shapify to obtain predictions for 132 extended sequences and including covariation information. Previous investigations have not applied hierarchical folding to extended length SARS-CoV-2 frameshift sequences. By doing so, we simulate the effects of ribosome interaction with the frameshift site, providing insight to biological function. We contribute in-depth discussion to contextualize secondary structure dual-graph motifs for SARS-CoV-2, highlighting the energetic stability of the previously identified 3_8 motif alongside the known dominant 3_3 and 3_6 (native-type) -1 PRF structures. Integrating experimental data within minimum free energy (MFE) hierarchical folding algorithms provides novel structure predictions to distill the relationship between RNA structure and function. In particular, fully categorizing most stable secondary structure predictions via hierarchical folding supports our identification of motif transitions and critical site targets for future therapeutic research.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
