<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Eukfinder: a pipeline to retrieve microbial eukaryote genomes from metagenomic sequencing data.
Authors: Zhao, D.; Salas-Leiva, D. E.; Williams, S. K.; Dunn, K. A.; Roger, A. J.
Score: 44.3, Published: 2023-12-28 DOI: 10.1101/2023.12.28.573569
Whole-genome shotgun (WGS) metagenomic sequencing of microbial communities allows us to discover the functions, physiologies, and evolutionary histories of microbial prokaryote and eukaryote members of diverse ecosystems. Despite their importance, metagenomic studies of microbial eukaryotes lag behind those of prokaryotes, due to the difficulty in identifying and assembling high-quality eukaryotic genomes from WGS data.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="Eukfinder: a pipeline to retrieve microbial eukaryote genomes from metagenomic sequencing data.
Authors: Zhao, D.; Salas-Leiva, D. E.; Williams, S. K.; Dunn, K. A.; Roger, A. J.
Score: 44.3, Published: 2023-12-28 DOI: 10.1101/2023.12.28.573569
Whole-genome shotgun (WGS) metagenomic sequencing of microbial communities allows us to discover the functions, physiologies, and evolutionary histories of microbial prokaryote and eukaryote members of diverse ecosystems. Despite their importance, metagenomic studies of microbial eukaryotes lag behind those of prokaryotes, due to the difficulty in identifying and assembling high-quality eukaryotic genomes from WGS data." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-03T10:39:43+00:00" />
<meta property="article:modified_time" content="2024-01-03T10:39:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="Eukfinder: a pipeline to retrieve microbial eukaryote genomes from metagenomic sequencing data.
Authors: Zhao, D.; Salas-Leiva, D. E.; Williams, S. K.; Dunn, K. A.; Roger, A. J.
Score: 44.3, Published: 2023-12-28 DOI: 10.1101/2023.12.28.573569
Whole-genome shotgun (WGS) metagenomic sequencing of microbial communities allows us to discover the functions, physiologies, and evolutionary histories of microbial prokaryote and eukaryote members of diverse ecosystems. Despite their importance, metagenomic studies of microbial eukaryotes lag behind those of prokaryotes, due to the difficulty in identifying and assembling high-quality eukaryotic genomes from WGS data."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "Eukfinder: a pipeline to retrieve microbial eukaryote genomes from metagenomic sequencing data.\nAuthors: Zhao, D.; Salas-Leiva, D. E.; Williams, S. K.; Dunn, K. A.; Roger, A. J.\nScore: 44.3, Published: 2023-12-28 DOI: 10.1101/2023.12.28.573569\nWhole-genome shotgun (WGS) metagenomic sequencing of microbial communities allows us to discover the functions, physiologies, and evolutionary histories of microbial prokaryote and eukaryote members of diverse ecosystems. Despite their importance, metagenomic studies of microbial eukaryotes lag behind those of prokaryotes, due to the difficulty in identifying and assembling high-quality eukaryotic genomes from WGS data.",
  "keywords": [
    
  ],
  "articleBody": " Eukfinder: a pipeline to retrieve microbial eukaryote genomes from metagenomic sequencing data.\nAuthors: Zhao, D.; Salas-Leiva, D. E.; Williams, S. K.; Dunn, K. A.; Roger, A. J.\nScore: 44.3, Published: 2023-12-28 DOI: 10.1101/2023.12.28.573569\nWhole-genome shotgun (WGS) metagenomic sequencing of microbial communities allows us to discover the functions, physiologies, and evolutionary histories of microbial prokaryote and eukaryote members of diverse ecosystems. Despite their importance, metagenomic studies of microbial eukaryotes lag behind those of prokaryotes, due to the difficulty in identifying and assembling high-quality eukaryotic genomes from WGS data. To address this problem, we have developed Eukfinder, a bioinformatics pipeline that recovers and assembles nuclear and mitochondrial genomes of eukaryotic microbes from WGS metagenomics data. As part of its workflow, it utilizes two specialized databases to classify reads based on taxonomy which can be customized to the dataset or environment of interest. We applied Eukfinder to human gut microbiome WGS metagenomic sequencing data to recover genomes from the protistan parasite Blastocystis sp., a highly prevalent colonizer of the gastrointestinal tract of humans and animals. We tested Eukfinder using both a series of simulated gut microbiome datasets, which included varying numbers of Blastocystis reads combined with bacterial reads and by using real metagenomic gut samples containing Blastocystis. We compared the results of Eukfinder with other published workflows. With sufficient reads, Eukfinder efficiently assembles high-quality near-complete nuclear and mitochondrial genomes from diverse Blastocystis subtypes from metagenomic data without the aid of a reference genome. Furthermore, with sufficient depth of sequence sampling, Eukfinder outperforms similar tools used to recover eukaryotic genomes from metagenomic data. Eukfinder will be a useful tool for reference-independent and cultivation-free study of eukaryotic microbial genomes from environmental metagenomic sequencing samples. IMPORTANCERapid advancements in next-generation sequencing technologies have made whole-genome shotgun (WGS) metagenomic sequencing an efficient method for de novo reconstruction of microbial genomes from samples taken from different environments. So far, thousands of new prokaryotic genomes have been characterized from strains or species that were unknown to science. However, the relatively large size and complexity of protistan genomes has, until recently, precluded the use of the WGS metagenomic approach to sample microbial eukaryotic diversity. The bioinformatics pipeline we developed, Eukfinder, can recover eukaryotic microbial genomes from environmental WGS metagenomic samples. By retrieving high-quality protistan genomes from diverse metagenomic samples, we can increase numbers of reference genomes available to aid future metagenomic investigations into the functions, physiologies, and evolutionary histories of eukaryotic microbes in the gut microbiome and a variety of other ecosystems.\nSopa: a technology-invariant pipeline for analyses of image-based spatial-omics\nAuthors: Blampey, Q.; Mulder, K.; Gardet, M.; Dutertre, C.-A.; Andre, F.; Ginhoux, F.; Cournede, P.-H.\nScore: 22.2, Published: 2023-12-23 DOI: 10.1101/2023.12.22.571863\nSpatial-omics data allow in-depth analysis of tissue architectures, opening new opportunities for biological discovery. In particular, imaging techniques offer single-cell resolutions, providing essential insights into cellular organizations and dynamics. Yet, the complexity of such data presents analytical challenges and demands substantial computing resources. Moreover, the proliferation of diverse spatial-omics technologies, such as Xenium, MERSCOPE, CosMX in spatial-transcriptomics, and MACSima and PhenoCycler in multiplex imaging, hinders the generality of existing tools. We introduce Sopa (https://github.com/gustaveroussy/sopa), a technology-invariant, memory-efficient pipeline with a unified visualizer for all image-based spatial omics. Built upon the universal SpatialData framework, Sopa optimizes tasks like segmentation, transcript/channel aggregation, annotation, and geometric/spatial analysis. Its output includes user-friendly web reports and visualizer files, as well as comprehensive data files for in-depth analysis. Overall, Sopa represents a significant step toward unifying spatial data analysis, enabling a more comprehensive understanding of cellular interactions and tissue organization in biological systems.\nOptimal distance metrics for single-cell RNA-seq populations\nAuthors: Ji, Y.; Green, T.; Peidli, S.; Bahrami, M.; Liu, M.; Zappia, L.; Hrovatin, K.; Sander, C.; Theis, F.\nScore: 17.1, Published: 2023-12-27 DOI: 10.1101/2023.12.26.572833\nIn single-cell data workflows and modeling, distance metrics are commonly used in loss functions, model evaluation, and subpopulation analysis. However, these metrics behave differently depending on the source of variation, conditions and subpopulations in single-cell expression profiles due to data sparsity and high dimensionality. Thus, the metrics used for downstream tasks in this domain should be carefully selected. We establish a set of benchmarks with three evaluation measures, capturing desirable facets of absolute and relative distance behavior. Based on seven datasets using perturbation as ground truth, we evaluated 16 distance metrics applied to scRNA-seq data and demonstrated their application to three use cases. We find that linear metrics such as mean squared error (MSE) performed best across our three evaluation criteria. Therefore, we recommend the use of MSE for comparing single-cell RNA-seq populations and evaluating gene expression prediction models.\nA catalogue of small proteins from the global microbiome\nAuthors: Duan, Y.; Santos Junior, C. D.; Schmidt, T. S. B.; Fullam, A.; de Almeida, B. L. S.; Zhu, C.; Michael, K.; Zhao, X.-M.; Bork, P.; Coelho, L. P.\nScore: 16.9, Published: 2023-12-28 DOI: 10.1101/2023.12.27.573469\nSmall open reading frames (smORFs) shorter than 100 codons are widespread and perform essential roles in microorganisms, where they encode proteins active in several cell functions, including signal pathways, stress response, and antibacterial activities. However, the ecology, distribution and role of small proteins in the global microbiome remain unknown. Here, we constructed a global microbial smORFs catalogue (GMSC) derived from 63,410 publicly available metagenomes across 75 distinct habitats and 87,920 high-quality isolate genomes. GMSC contains 965 million non-redundant smORFs with comprehensive annotations. We found that archaea harbor more small proteins proportionally than bacteria. We moreover provide a tool called GMSC-mapper to identify and annotate small proteins from microbial (meta)genomes. Overall, this publicly-available resource demonstrates the immense and underexplored diversity of small proteins.\nTying the Knot: Unraveling the Intricacies of the Coronavirus Frameshift Pseudoknot\nAuthors: Trinity, L.; Stege, U.; Jabbari, H.\nScore: 13.6, Published: 2023-12-28 DOI: 10.1101/2023.12.28.573501\nUnderstanding and targeting functional RNA structures towards treatment of coronavirus infection can help us to prepare for novel variants of SARS-CoV-2 (the virus causing COVID-19), and any other coronaviruses that could emerge via human-to-human transmission or potential zoonotic (inter-species) events. Leveraging the fact that all coronaviruses use a mechanism known as -1 programmed ribosomal frameshifting (-1 PRF) to replicate, we apply algorithms to predict the most energetically favourable secondary structures (each nucleotide involved in at most one pairing) that may be involved in regulating the -1 PRF event in coronaviruses, especially SARS-CoV-2. We compute previously unknown most stable structure predictions for the frameshift site of coronaviruses via hierarchical folding, a biologically motivated framework where initial non-crossing structure folds first, followed by subsequent, possibly crossing (pseudoknotted), structures. Using mutual information from 181 coronavirus sequences, in conjunction with the algorithm KnotAli, we compute secondary structure predictions for the frameshift site of different coronaviruses. We then utilize the Shapify algorithm to obtain most stable SARS-CoV-2 secondary structure predictions guided by frameshift sequence-specific and genome-wide experimental data. We build on our previous secondary structure investigation of the singular SARS-CoV-2 68 nt frameshift element sequence, by using Shapify to obtain predictions for 132 extended sequences and including covariation information. Previous investigations have not applied hierarchical folding to extended length SARS-CoV-2 frameshift sequences. By doing so, we simulate the effects of ribosome interaction with the frameshift site, providing insight to biological function. We contribute in-depth discussion to contextualize secondary structure dual-graph motifs for SARS-CoV-2, highlighting the energetic stability of the previously identified 3 8 motif alongside the known dominant 3 3 and 3 6 (native-type) -1 PRF structures. Integrating experimental data within minimum free energy (MFE) hierarchical folding algorithms provides novel structure predictions to distill the relationship between RNA structure and function. In particular, fully categorizing most stable secondary structure predictions via hierarchical folding supports our identification of motif transitions and critical site targets for future therapeutic research. Author summaryFinding evolutionary connections between coronaviruses frameshift element RNA structures is a worthwhile goal in contributing to treatment development for afflicted human and animal populations. Predicting the most energetically favourable RNA secondary structures, and how they may form via the hierarchical folding hypothesis, is an efficient use of computational resources to shed light on RNA structure-function. We used the KnotAli algorithm to obtain mutual information from 181 coronaviruses frameshift RNA sequences. Guided by this evolutionary information, we computed secondary structure predictions to allow comparison of marked similarities and subtle differences between SARS-CoV-2 and other coronaviruses frameshift element RNA structures. In addition, we applied the Shapify algorithm to predict secondary structures for extended SARS-CoV-2 frameshift element sequences informed by SHAPE reactivity data. Here we critically expand the known landscape of most stable -1 PRF secondary structure conformations, isolating the location of key secondary structure motif transitions that can improve site targeting of viral therapeutics. Our application of hierarchical folding algorithms contributes novel predictions of functional RNA structures, enhancing discussion of how secondary structures unfold or refold to regulate frameshifting in coronaviruses.\nMultimodal analysis unveils tumor microenvironment heterogeneity linked to immune activity and evasion\nAuthors: Lapuente-Santana, O.; Sturm, G.; Kant, J.; Ausserhofer, M.; Zackl, C.; Zopoglou, M.; McGranahan, N.; Rieder, D.; Trajanoski, Z.; de Miranda, N. F. d. C. C.; Eduati, F.; Finotello, F.\nScore: 12.3, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572033\nThe cellular and molecular heterogeneity of tumors is a major obstacle to cancer immunotherapy. Here, we use a systems biology approach to derive a signature of the main sources of heterogeneity in the tumor microenvironment (TME) from lung cancer transcriptomic data. We demonstrate that this signature, which we called iHet, is conserved in different cancers and associated with antitumor immunity. Through the analysis of single-cell and spatial transcriptomics data, we trace back the cellular origin of the variability that explains the iHet signature. Finally, we demonstrate that iHet has predictive value for cancer immunotherapy, which can be further improved by disentangling three major determinants of anticancer immune responses: activity of immune cells, immune infiltration or exclusion, and cancer-cell foreignness. This work shows how transcriptomics data can be integrated to derive a holistic representation of the phenotypic heterogeneity of the TME, and ultimately to determine its unfolding and fate during immunotherapy with immune checkpoint blockers.\nGAPS: Geometric Attention-based Networks for Peptide Binding Sites Identification by the Transfer Learning Approach\nAuthors: Zhu, C.; Zhang, C.; Shang, T.; Zhang, C.; Zhai, S.; Su, Z.; Duan, H.\nScore: 7.3, Published: 2023-12-26 DOI: 10.1101/2023.12.26.573336\nThe identification of protein-peptide binding sites significantly advances our understanding of their interaction. Recent advancements in deep learning have profoundly transformed the prediction of protein-peptide binding sites. In this work, we describe the Geometric Attention-based networks for Peptide binding Sites identification (GAPS). The GAPS constructs atom representations using geometric feature engineering and employs various attention mechanisms to update pertinent biological features. In addition, the transfer learning strategy is implemented for leveraging the pre-trained protein-protein binding sites information to enhance training of the protein-peptide binding sites recognition, taking into account the similarity of proteins and peptides. Consequently, GAPS demonstrates state-of-the-art (SOTA) performance in this task. Our model also exhibits exceptional performance across several expanded experiments including predicting the apo protein-peptide, the protein-cyclic peptide, and the predicted protein-peptide binding sites. Overall, the GAPS is a powerful, versatile, stable method suitable for diverse binding site predictions.\nADMET-AI: A machine learning ADMET platform for evaluation of large-scale chemical libraries\nAuthors: Swanson, K.; Walther, P.; Leitz, J.; Mukherjee, S.; Wu, J. C.; Shivnaraine, R. V.; Zou, J.\nScore: 6.8, Published: 2023-12-28 DOI: 10.1101/2023.12.28.573531\nSummaryThe emergence of large chemical repositories and combinatorial chemical spaces, coupled with high-throughput docking and generative AI, have greatly expanded the chemical diversity of small molecules for drug discovery. Selecting compounds for experimental validation requires filtering these molecules based on favourable druglike properties, such as Absorption, Distribution, Metabolism, Excretion, and Toxicity (ADMET). We developed ADMET-AI, a machine learning platform that provides fast and accurate ADMET predictions both as a website and as a Python package. ADMET-AI has the highest average rank on the TDC ADMET Benchmark Group leaderboard, and it is currently the fastest web-based ADMET predictor, with a 45% reduction in time compared to the next fastest ADMET web server. ADMET-AI can also be run locally with predictions for one million molecules taking just 3.1 hours. Availability and ImplementationThe ADMET-AI platform is freely available both as a web server at admet.ai.greenstonebio.com and as an open-source Python package for local batch prediction at github.com/swansonk14/admet_ai (also archived on Zenodo at doi.org/10.5281/zenodo.10372930). All data and models are archived on Zenodo at doi.org/10.5281/zenodo.10372418.\nRepeatOBserver: tandem repeat visualization and centromere detection\nAuthors: Elphinstone, C.; Elphinstone, R.; Todesco, M.; Rieseberg, L.\nScore: 6.6, Published: 2023-12-30 DOI: 10.1101/2023.12.30.573697\nTandem repeats can play an important role in centromere structure, subtelomeric regions, DNA methylation, recombination, and the regulation of gene activity. There is a growing need for bioinformatics tools that can visualize and explore chromosome-scale repeats. Here we present RepeatOBserver, a new tool for visualizing tandem repeats and clustered transposable elements and for identifying potential natural centromere locations, using a Fourier transform of DNA walks: https://github.com/celphin/RepeatOBserverV1. RepeatOBserver can identify a broad range of repeats (3-20, 000bp long) in genome assemblies without any a priori knowledge of repeat sequences or the need for optimizing parameters. RepeatOBserver allows for easy visualization of the positions of both perfect and imperfect repeating sequences across each chromosome. We use RepeatOBserver to compare DNA walks, repeat patterns and centromere positions across genome assemblies in a wide range of well-studied species (e.g., human, mouse-ear cress), crops, and non-model organisms (e.g., fern, yew). Analyzing 107 chromosomes with known centromere positions, we find that centromeres consistently occur in regions that have the least diversity in repeat types (i.e. one or a few repeated sequences are present in very high numbers). Taking advantage of this information, we use a genomic Shannon diversity index to predict centromere locations in several other chromosome-scale genome assemblies. The Fourier spectra produced by RepeatOBserver can help visualize historic centromere positions, potential neocentromeres, retrotransposon clusters and gene copy variation. Identification of patterns of split and inverted tandem repeats at inversion boundaries suggests that at least some chromosomal inversions or misassemblies can be predicted with RepeatOBserver. RepeatOBserver is therefore a flexible tool for comprehensive characterization of tandem repeat patterns that can be used to visualize and identify a variety of regions of interest in genome assemblies.\nA comprehensive alignment-filtering methodology improves phylogeny particularly by filtering overly divergent segments\nAuthors: Zhang, Q.; Qin, X.; Lu, Y.; Li, P.; Huang, X.\nScore: 6.2, Published: 2023-12-26 DOI: 10.1101/2023.12.26.573321\nO_LIAlignment problems may be complicated and diverse and the performance of the existing alignment-filtering tools, particularly their potential effect on phylogenetic reconstruction, have remained debated or limitedly explored. C_LIO_LIIn the present study, we developed a new R package named alignmentFilter to treat the diverse alignment problems, especially masking ambiguously-aligned or overly divergent segments using a newly devised grouping-regrouping algorithms acting on sequence divergence in each of sliding windows throughout alignment. Then we tested and compared the power and accuracy of the prevalent alignment-filtering tools, particularly the effect on phylogeny based on Angiospermae genome-scale data and simulated data. C_LIO_LIThe results indicate the alignment-filtering methods alone may affect the phylogeny decisively, producing (strongly-supported) phylogenetic conflicts. In most cases, alignmentFilter most efficiently improves phylogeny, yielding more congruent phylogeny regardless of phylogeny-reconstruction methods, producing phylogeny more concordant to the priori tree regarding to both topology and branch length in simulation, and most efficiently decreasing the root-to-tip length heterogeneity that is implied to be negatively correlated with topological accuracy. C_LIO_LIThe concepts and algorithms of alignmentFilter are quite distinct from existing alignment-filtering methods. It is not susceptible to specific types of alignment errors that usually baffle other methods and can reach the optimal solution frequently with lowest computation complexity. The biological or statistical sense of the key optional stringency-controlling parameter is more straightforward and the setting and outcome may be more adjustable or predictable. A better and more comprehensive solution of alignment problems in this study will benefit tremendous downstream analyses sourced from alignment and phylogeny. C_LI\n",
  "wordCount" : "2617",
  "inLanguage": "en",
  "datePublished": "2024-01-03T10:39:43Z",
  "dateModified": "2024-01-03T10:39:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 3, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.573569">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.573569" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.573569">
        <p class="paperTitle">Eukfinder: a pipeline to retrieve microbial eukaryote genomes from metagenomic sequencing data.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.573569" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.573569" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhao, D.; Salas-Leiva, D. E.; Williams, S. K.; Dunn, K. A.; Roger, A. J.</p>
        <p class="info">Score: 44.3, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.573569' target='https://doi.org/10.1101/2023.12.28.573569'> 10.1101/2023.12.28.573569</a></p>
        <p class="abstract">Whole-genome shotgun (WGS) metagenomic sequencing of microbial communities allows us to discover the functions, physiologies, and evolutionary histories of microbial prokaryote and eukaryote members of diverse ecosystems. Despite their importance, metagenomic studies of microbial eukaryotes lag behind those of prokaryotes, due to the difficulty in identifying and assembling high-quality eukaryotic genomes from WGS data. To address this problem, we have developed Eukfinder, a bioinformatics pipeline that recovers and assembles nuclear and mitochondrial genomes of eukaryotic microbes from WGS metagenomics data. As part of its workflow, it utilizes two specialized databases to classify reads based on taxonomy which can be customized to the dataset or environment of interest. We applied Eukfinder to human gut microbiome WGS metagenomic sequencing data to recover genomes from the protistan parasite Blastocystis sp., a highly prevalent colonizer of the gastrointestinal tract of humans and animals. We tested Eukfinder using both a series of simulated gut microbiome datasets, which included varying numbers of Blastocystis reads combined with bacterial reads and by using real metagenomic gut samples containing Blastocystis. We compared the results of Eukfinder with other published workflows. With sufficient reads, Eukfinder efficiently assembles high-quality near-complete nuclear and mitochondrial genomes from diverse Blastocystis subtypes from metagenomic data without the aid of a reference genome. Furthermore, with sufficient depth of sequence sampling, Eukfinder outperforms similar tools used to recover eukaryotic genomes from metagenomic data. Eukfinder will be a useful tool for reference-independent and cultivation-free study of eukaryotic microbial genomes from environmental metagenomic sequencing samples.

IMPORTANCERapid advancements in next-generation sequencing technologies have made whole-genome shotgun (WGS) metagenomic sequencing an efficient method for de novo reconstruction of microbial genomes from samples taken from different environments. So far, thousands of new prokaryotic genomes have been characterized from strains or species that were unknown to science. However, the relatively large size and complexity of protistan genomes has, until recently, precluded the use of the WGS metagenomic approach to sample microbial eukaryotic diversity. The bioinformatics pipeline we developed, Eukfinder, can recover eukaryotic microbial genomes from environmental WGS metagenomic samples. By retrieving high-quality protistan genomes from diverse metagenomic samples, we can increase numbers of reference genomes available to aid future metagenomic investigations into the functions, physiologies, and evolutionary histories of eukaryotic microbes in the gut microbiome and a variety of other ecosystems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.571863">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.571863" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.571863">
        <p class="paperTitle">Sopa: a technology-invariant pipeline for analyses of image-based spatial-omics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.571863" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.571863" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Blampey, Q.; Mulder, K.; Gardet, M.; Dutertre, C.-A.; Andre, F.; Ginhoux, F.; Cournede, P.-H.</p>
        <p class="info">Score: 22.2, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.571863' target='https://doi.org/10.1101/2023.12.22.571863'> 10.1101/2023.12.22.571863</a></p>
        <p class="abstract">Spatial-omics data allow in-depth analysis of tissue architectures, opening new opportunities for biological discovery. In particular, imaging techniques offer single-cell resolutions, providing essential insights into cellular organizations and dynamics. Yet, the complexity of such data presents analytical challenges and demands substantial computing resources. Moreover, the proliferation of diverse spatial-omics technologies, such as Xenium, MERSCOPE, CosMX in spatial-transcriptomics, and MACSima and PhenoCycler in multiplex imaging, hinders the generality of existing tools. We introduce Sopa (https://github.com/gustaveroussy/sopa), a technology-invariant, memory-efficient pipeline with a unified visualizer for all image-based spatial omics. Built upon the universal SpatialData framework, Sopa optimizes tasks like segmentation, transcript/channel aggregation, annotation, and geometric/spatial analysis. Its output includes user-friendly web reports and visualizer files, as well as comprehensive data files for in-depth analysis. Overall, Sopa represents a significant step toward unifying spatial data analysis, enabling a more comprehensive understanding of cellular interactions and tissue organization in biological systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.26.572833">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.26.572833" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.26.572833">
        <p class="paperTitle">Optimal distance metrics for single-cell RNA-seq populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.26.572833" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.26.572833" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ji, Y.; Green, T.; Peidli, S.; Bahrami, M.; Liu, M.; Zappia, L.; Hrovatin, K.; Sander, C.; Theis, F.</p>
        <p class="info">Score: 17.1, Published: 2023-12-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.26.572833' target='https://doi.org/10.1101/2023.12.26.572833'> 10.1101/2023.12.26.572833</a></p>
        <p class="abstract">In single-cell data workflows and modeling, distance metrics are commonly used in loss functions, model evaluation, and subpopulation analysis. However, these metrics behave differently depending on the source of variation, conditions and subpopulations in single-cell expression profiles due to data sparsity and high dimensionality. Thus, the metrics used for downstream tasks in this domain should be carefully selected. We establish a set of benchmarks with three evaluation measures, capturing desirable facets of absolute and relative distance behavior. Based on seven datasets using perturbation as ground truth, we evaluated 16 distance metrics applied to scRNA-seq data and demonstrated their application to three use cases. We find that linear metrics such as mean squared error (MSE) performed best across our three evaluation criteria. Therefore, we recommend the use of MSE for comparing single-cell RNA-seq populations and evaluating gene expression prediction models.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.27.573469">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.27.573469" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.27.573469">
        <p class="paperTitle">A catalogue of small proteins from the global microbiome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.27.573469" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.27.573469" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Duan, Y.; Santos Junior, C. D.; Schmidt, T. S. B.; Fullam, A.; de Almeida, B. L. S.; Zhu, C.; Michael, K.; Zhao, X.-M.; Bork, P.; Coelho, L. P.</p>
        <p class="info">Score: 16.9, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.27.573469' target='https://doi.org/10.1101/2023.12.27.573469'> 10.1101/2023.12.27.573469</a></p>
        <p class="abstract">Small open reading frames (smORFs) shorter than 100 codons are widespread and perform essential roles in microorganisms, where they encode proteins active in several cell functions, including signal pathways, stress response, and antibacterial activities. However, the ecology, distribution and role of small proteins in the global microbiome remain unknown. Here, we constructed a global microbial smORFs catalogue (GMSC) derived from 63,410 publicly available metagenomes across 75 distinct habitats and 87,920 high-quality isolate genomes. GMSC contains 965 million non-redundant smORFs with comprehensive annotations. We found that archaea harbor more small proteins proportionally than bacteria. We moreover provide a tool called GMSC-mapper to identify and annotate small proteins from microbial (meta)genomes. Overall, this publicly-available resource demonstrates the immense and underexplored diversity of small proteins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.573501">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.573501" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.573501">
        <p class="paperTitle">Tying the Knot: Unraveling the Intricacies of the Coronavirus Frameshift Pseudoknot</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.573501" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.573501" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Trinity, L.; Stege, U.; Jabbari, H.</p>
        <p class="info">Score: 13.6, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.573501' target='https://doi.org/10.1101/2023.12.28.573501'> 10.1101/2023.12.28.573501</a></p>
        <p class="abstract">Understanding and targeting functional RNA structures towards treatment of coronavirus infection can help us to prepare for novel variants of SARS-CoV-2 (the virus causing COVID-19), and any other coronaviruses that could emerge via human-to-human transmission or potential zoonotic (inter-species) events. Leveraging the fact that all coronaviruses use a mechanism known as -1 programmed ribosomal frameshifting (-1 PRF) to replicate, we apply algorithms to predict the most energetically favourable secondary structures (each nucleotide involved in at most one pairing) that may be involved in regulating the -1 PRF event in coronaviruses, especially SARS-CoV-2. We compute previously unknown most stable structure predictions for the frameshift site of coronaviruses via hierarchical folding, a biologically motivated framework where initial non-crossing structure folds first, followed by subsequent, possibly crossing (pseudoknotted), structures. Using mutual information from 181 coronavirus sequences, in conjunction with the algorithm KnotAli, we compute secondary structure predictions for the frameshift site of different coronaviruses. We then utilize the Shapify algorithm to obtain most stable SARS-CoV-2 secondary structure predictions guided by frameshift sequence-specific and genome-wide experimental data. We build on our previous secondary structure investigation of the singular SARS-CoV-2 68 nt frameshift element sequence, by using Shapify to obtain predictions for 132 extended sequences and including covariation information. Previous investigations have not applied hierarchical folding to extended length SARS-CoV-2 frameshift sequences. By doing so, we simulate the effects of ribosome interaction with the frameshift site, providing insight to biological function. We contribute in-depth discussion to contextualize secondary structure dual-graph motifs for SARS-CoV-2, highlighting the energetic stability of the previously identified 3 8 motif alongside the known dominant 3 3 and 3 6 (native-type) -1 PRF structures. Integrating experimental data within minimum free energy (MFE) hierarchical folding algorithms provides novel structure predictions to distill the relationship between RNA structure and function. In particular, fully categorizing most stable secondary structure predictions via hierarchical folding supports our identification of motif transitions and critical site targets for future therapeutic research.

Author summaryFinding evolutionary connections between coronaviruses frameshift element RNA structures is a worthwhile goal in contributing to treatment development for afflicted human and animal populations. Predicting the most energetically favourable RNA secondary structures, and how they may form via the hierarchical folding hypothesis, is an efficient use of computational resources to shed light on RNA structure-function.

We used the KnotAli algorithm to obtain mutual information from 181 coronaviruses frameshift RNA sequences. Guided by this evolutionary information, we computed secondary structure predictions to allow comparison of marked similarities and subtle differences between SARS-CoV-2 and other coronaviruses frameshift element RNA structures. In addition, we applied the Shapify algorithm to predict secondary structures for extended SARS-CoV-2 frameshift element sequences informed by SHAPE reactivity data. Here we critically expand the known landscape of most stable -1 PRF secondary structure conformations, isolating the location of key secondary structure motif transitions that can improve site targeting of viral therapeutics. Our application of hierarchical folding algorithms contributes novel predictions of functional RNA structures, enhancing discussion of how secondary structures unfold or refold to regulate frameshifting in coronaviruses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572033">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572033" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572033">
        <p class="paperTitle">Multimodal analysis unveils tumor microenvironment heterogeneity linked to immune activity and evasion</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572033" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572033" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lapuente-Santana, O.; Sturm, G.; Kant, J.; Ausserhofer, M.; Zackl, C.; Zopoglou, M.; McGranahan, N.; Rieder, D.; Trajanoski, Z.; de Miranda, N. F. d. C. C.; Eduati, F.; Finotello, F.</p>
        <p class="info">Score: 12.3, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572033' target='https://doi.org/10.1101/2023.12.20.572033'> 10.1101/2023.12.20.572033</a></p>
        <p class="abstract">The cellular and molecular heterogeneity of tumors is a major obstacle to cancer immunotherapy. Here, we use a systems biology approach to derive a signature of the main sources of heterogeneity in the tumor microenvironment (TME) from lung cancer transcriptomic data. We demonstrate that this signature, which we called iHet, is conserved in different cancers and associated with antitumor immunity. Through the analysis of single-cell and spatial transcriptomics data, we trace back the cellular origin of the variability that explains the iHet signature. Finally, we demonstrate that iHet has predictive value for cancer immunotherapy, which can be further improved by disentangling three major determinants of anticancer immune responses: activity of immune cells, immune infiltration or exclusion, and cancer-cell foreignness. This work shows how transcriptomics data can be integrated to derive a holistic representation of the phenotypic heterogeneity of the TME, and ultimately to determine its unfolding and fate during immunotherapy with immune checkpoint blockers.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.26.573336">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.26.573336" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.26.573336">
        <p class="paperTitle">GAPS: Geometric Attention-based Networks for Peptide Binding Sites Identification by the Transfer Learning Approach</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.26.573336" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.26.573336" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhu, C.; Zhang, C.; Shang, T.; Zhang, C.; Zhai, S.; Su, Z.; Duan, H.</p>
        <p class="info">Score: 7.3, Published: 2023-12-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.26.573336' target='https://doi.org/10.1101/2023.12.26.573336'> 10.1101/2023.12.26.573336</a></p>
        <p class="abstract">The identification of protein-peptide binding sites significantly advances our understanding of their interaction. Recent advancements in deep learning have profoundly transformed the prediction of protein-peptide binding sites. In this work, we describe the Geometric Attention-based networks for Peptide binding Sites identification (GAPS). The GAPS constructs atom representations using geometric feature engineering and employs various attention mechanisms to update pertinent biological features. In addition, the transfer learning strategy is implemented for leveraging the pre-trained protein-protein binding sites information to enhance training of the protein-peptide binding sites recognition, taking into account the similarity of proteins and peptides. Consequently, GAPS demonstrates state-of-the-art (SOTA) performance in this task. Our model also exhibits exceptional performance across several expanded experiments including predicting the apo protein-peptide, the protein-cyclic peptide, and the predicted protein-peptide binding sites. Overall, the GAPS is a powerful, versatile, stable method suitable for diverse binding site predictions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.573531">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.573531" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.573531">
        <p class="paperTitle">ADMET-AI: A machine learning ADMET platform for evaluation of large-scale chemical libraries</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.573531" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.573531" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Swanson, K.; Walther, P.; Leitz, J.; Mukherjee, S.; Wu, J. C.; Shivnaraine, R. V.; Zou, J.</p>
        <p class="info">Score: 6.8, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.573531' target='https://doi.org/10.1101/2023.12.28.573531'> 10.1101/2023.12.28.573531</a></p>
        <p class="abstract">SummaryThe emergence of large chemical repositories and combinatorial chemical spaces, coupled with high-throughput docking and generative AI, have greatly expanded the chemical diversity of small molecules for drug discovery. Selecting compounds for experimental validation requires filtering these molecules based on favourable druglike properties, such as Absorption, Distribution, Metabolism, Excretion, and Toxicity (ADMET). We developed ADMET-AI, a machine learning platform that provides fast and accurate ADMET predictions both as a website and as a Python package. ADMET-AI has the highest average rank on the TDC ADMET Benchmark Group leaderboard, and it is currently the fastest web-based ADMET predictor, with a 45% reduction in time compared to the next fastest ADMET web server. ADMET-AI can also be run locally with predictions for one million molecules taking just 3.1 hours.

Availability and ImplementationThe ADMET-AI platform is freely available both as a web server at admet.ai.greenstonebio.com and as an open-source Python package for local batch prediction at github.com/swansonk14/admet_ai (also archived on Zenodo at doi.org/10.5281/zenodo.10372930). All data and models are archived on Zenodo at doi.org/10.5281/zenodo.10372418.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.30.573697">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.30.573697" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.30.573697">
        <p class="paperTitle">RepeatOBserver: tandem repeat visualization and centromere detection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.30.573697" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.30.573697" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Elphinstone, C.; Elphinstone, R.; Todesco, M.; Rieseberg, L.</p>
        <p class="info">Score: 6.6, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.30.573697' target='https://doi.org/10.1101/2023.12.30.573697'> 10.1101/2023.12.30.573697</a></p>
        <p class="abstract">Tandem repeats can play an important role in centromere structure, subtelomeric regions, DNA methylation, recombination, and the regulation of gene activity. There is a growing need for bioinformatics tools that can visualize and explore chromosome-scale repeats. Here we present RepeatOBserver, a new tool for visualizing tandem repeats and clustered transposable elements and for identifying potential natural centromere locations, using a Fourier transform of DNA walks: https://github.com/celphin/RepeatOBserverV1. RepeatOBserver can identify a broad range of repeats (3-20, 000bp long) in genome assemblies without any a priori knowledge of repeat sequences or the need for optimizing parameters. RepeatOBserver allows for easy visualization of the positions of both perfect and imperfect repeating sequences across each chromosome. We use RepeatOBserver to compare DNA walks, repeat patterns and centromere positions across genome assemblies in a wide range of well-studied species (e.g., human, mouse-ear cress), crops, and non-model organisms (e.g., fern, yew). Analyzing 107 chromosomes with known centromere positions, we find that centromeres consistently occur in regions that have the least diversity in repeat types (i.e. one or a few repeated sequences are present in very high numbers). Taking advantage of this information, we use a genomic Shannon diversity index to predict centromere locations in several other chromosome-scale genome assemblies. The Fourier spectra produced by RepeatOBserver can help visualize historic centromere positions, potential neocentromeres, retrotransposon clusters and gene copy variation. Identification of patterns of split and inverted tandem repeats at inversion boundaries suggests that at least some chromosomal inversions or misassemblies can be predicted with RepeatOBserver. RepeatOBserver is therefore a flexible tool for comprehensive characterization of tandem repeat patterns that can be used to visualize and identify a variety of regions of interest in genome assemblies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.26.573321">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.26.573321" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.26.573321">
        <p class="paperTitle">A comprehensive alignment-filtering methodology improves phylogeny particularly by filtering overly divergent segments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.26.573321" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.26.573321" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, Q.; Qin, X.; Lu, Y.; Li, P.; Huang, X.</p>
        <p class="info">Score: 6.2, Published: 2023-12-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.26.573321' target='https://doi.org/10.1101/2023.12.26.573321'> 10.1101/2023.12.26.573321</a></p>
        <p class="abstract">O_LIAlignment problems may be complicated and diverse and the performance of the existing alignment-filtering tools, particularly their potential effect on phylogenetic reconstruction, have remained debated or limitedly explored.
C_LIO_LIIn the present study, we developed a new R package named alignmentFilter to treat the diverse alignment problems, especially masking ambiguously-aligned or overly divergent segments using a newly devised grouping-regrouping algorithms acting on sequence divergence in each of sliding windows throughout alignment. Then we tested and compared the power and accuracy of the prevalent alignment-filtering tools, particularly the effect on phylogeny based on Angiospermae genome-scale data and simulated data.
C_LIO_LIThe results indicate the alignment-filtering methods alone may affect the phylogeny decisively, producing (strongly-supported) phylogenetic conflicts. In most cases, alignmentFilter most efficiently improves phylogeny, yielding more congruent phylogeny regardless of phylogeny-reconstruction methods, producing phylogeny more concordant to the priori tree regarding to both topology and branch length in simulation, and most efficiently decreasing the root-to-tip length heterogeneity that is implied to be negatively correlated with topological accuracy.
C_LIO_LIThe concepts and algorithms of alignmentFilter are quite distinct from existing alignment-filtering methods. It is not susceptible to specific types of alignment errors that usually baffle other methods and can reach the optimal solution frequently with lowest computation complexity. The biological or statistical sense of the key optional stringency-controlling parameter is more straightforward and the setting and outcome may be more adjustable or predictable. A better and more comprehensive solution of alignment problems in this study will benefit tremendous downstream analyses sourced from alignment and phylogeny.
C_LI</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
