<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="SaProt: Protein Language Modeling with Structure-aware Vocabulary
Authors: Su, J.; Han, C.; Zhou, Y.; Shan, J.; Zhou, X.; Yuan, F.
Score: 42.4, Published: 2023-10-02 DOI: 10.1101/2023.10.01.560349
Large-scale protein language models (PLMs), such as the ESM family, have achieved remarkable performance in various downstream tasks related to protein structure and function by undergoing unsupervised training on residue sequences. They have become essential tools for researchers and practitioners in biology. However, a limitation of vanilla PLMs is their lack of explicit consideration for protein structure information, which suggests the potential for further improvement.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="SaProt: Protein Language Modeling with Structure-aware Vocabulary
Authors: Su, J.; Han, C.; Zhou, Y.; Shan, J.; Zhou, X.; Yuan, F.
Score: 42.4, Published: 2023-10-02 DOI: 10.1101/2023.10.01.560349
Large-scale protein language models (PLMs), such as the ESM family, have achieved remarkable performance in various downstream tasks related to protein structure and function by undergoing unsupervised training on residue sequences. They have become essential tools for researchers and practitioners in biology. However, a limitation of vanilla PLMs is their lack of explicit consideration for protein structure information, which suggests the potential for further improvement." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-08T10:37:16+00:00" />
<meta property="article:modified_time" content="2023-10-08T10:37:16+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="SaProt: Protein Language Modeling with Structure-aware Vocabulary
Authors: Su, J.; Han, C.; Zhou, Y.; Shan, J.; Zhou, X.; Yuan, F.
Score: 42.4, Published: 2023-10-02 DOI: 10.1101/2023.10.01.560349
Large-scale protein language models (PLMs), such as the ESM family, have achieved remarkable performance in various downstream tasks related to protein structure and function by undergoing unsupervised training on residue sequences. They have become essential tools for researchers and practitioners in biology. However, a limitation of vanilla PLMs is their lack of explicit consideration for protein structure information, which suggests the potential for further improvement."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "SaProt: Protein Language Modeling with Structure-aware Vocabulary\nAuthors: Su, J.; Han, C.; Zhou, Y.; Shan, J.; Zhou, X.; Yuan, F.\nScore: 42.4, Published: 2023-10-02 DOI: 10.1101/2023.10.01.560349\nLarge-scale protein language models (PLMs), such as the ESM family, have achieved remarkable performance in various downstream tasks related to protein structure and function by undergoing unsupervised training on residue sequences. They have become essential tools for researchers and practitioners in biology. However, a limitation of vanilla PLMs is their lack of explicit consideration for protein structure information, which suggests the potential for further improvement.",
  "keywords": [
    
  ],
  "articleBody": " SaProt: Protein Language Modeling with Structure-aware Vocabulary\nAuthors: Su, J.; Han, C.; Zhou, Y.; Shan, J.; Zhou, X.; Yuan, F.\nScore: 42.4, Published: 2023-10-02 DOI: 10.1101/2023.10.01.560349\nLarge-scale protein language models (PLMs), such as the ESM family, have achieved remarkable performance in various downstream tasks related to protein structure and function by undergoing unsupervised training on residue sequences. They have become essential tools for researchers and practitioners in biology. However, a limitation of vanilla PLMs is their lack of explicit consideration for protein structure information, which suggests the potential for further improvement. Motivated by this, we introduce the concept of a \"structure-aware vocabulary\" that integrates residue tokens with structure tokens. The structure tokens are derived by encoding the 3D structure of proteins using Foldseek. We then propose SaProt, a large-scale general-purpose PLM trained on an extensive dataset comprising approximately 40 million protein sequences and structures. Through extensive evaluation, our SaProt model surpasses well-established and renowned baselines across 10 significant downstream tasks, demonstrating its exceptional capacity and broad applicability. We have made the code1, pretrained model, and all relevant materials available at https://github.com/westlake-repl/SaProt.\nPROTOSPACEJAM: AN OPEN-SOURCE, CUSTOMIZABLE AND WEB-ACCESSIBLE DESIGN PLATFORM FOR CRISPR/CAS9 INSERTIONAL KNOCK-IN\nAuthors: Peng, D.; Vangipuram, M.; Wong, J.; Leonetti, M.\nScore: 33.3, Published: 2023-10-04 DOI: 10.1101/2023.10.04.560793\nCRISPR/Cas9-mediated knock-in of DNA sequences enables precise genome engineering for research and therapeutic applications. However, designing effective guide RNAs (gRNAs) and homology-directed repair (HDR) donors remains a bottleneck. Here, we present protoSpaceJAM, an open-source algorithm to automate and optimize gRNA and HDR donor design for CRISPR/Cas9 insertional knock-in experiments. protoSpaceJAM utilizes biological rules to rank gRNAs based on specificity, distance to insertion site, and position relative to regulatory regions. protoSpaceJAM can introduce recoding mutations (silent mutations and mutations in non-coding sequences) in HDR donors to prevent re-cutting and increase knock-in efficiency. Users can customize parameters and design double-stranded or single-stranded donors. We validated protoSpaceJAM's design rules by demonstrating increased knock-in efficiency with recoding mutations and optimal strand selection for single-stranded donors. An additional module enables the design of genotyping primers for next-generation sequencing of edited alleles. Overall, protoSpaceJAM streamlines and optimizes CRISPR knock-in experimental design in a flexible and modular manner to benefit diverse research and therapeutic applications. protoSpaceJAM is available open-source as an interactive web tool at protospacejam.czbiohub.org or as a standalone Python package at github.com/czbiohub-sf/protoSpaceJAM.\nThe structural landscape of the immunoglobulin fold by large-scale de novo design\nAuthors: Roel-Touris, J.; Carcelen, L.; Marcos, E.\nScore: 24.9, Published: 2023-10-04 DOI: 10.1101/2023.10.03.560637\nDe novo designing immunoglobulin-like frameworks that allow for functional loop diversification shows great potential for crafting antibody-like scaffolds with fully customizable structures and functions. In this work, we combined de novo parametric design with deep-learning methods for protein structure prediction and design to explore the structural landscape of 7-stranded immunoglobulin domains. After screening folding of nearly 4 million designs, we have assembled a structurally diverse library of ~50,000 immunoglobulin domains with high-confidence AlphaFold2 predictions and structures diverging from naturally occurring ones. The designed dataset enabled us to identify structural requirements for the correct folding of immunoglobulin domains, shed light on {beta}-sheet-{beta}-sheet rotational preferences and how these are linked to functional properties. Our approach eliminates the need for preset loop conformations and opens the route to large-scale de novo design of immunoglobulin-like frameworks.\nSimulation-based Benchmarking of Ancient Haplotype Inference for Detecting Population Structure\nAuthors: Medina-Tretmanis, J.; Jay, F.; Avila-Arcos, M. C.; Huerta-Sanchez, E.\nScore: 19.9, Published: 2023-10-03 DOI: 10.1101/2023.09.28.560049\nPaleogenomic data has informed us about the movements, growth, and relationships of ancient populations. It has also given us context for medically relevant adaptations that appear in present-day humans due to introgression from other hominids, and it continues to help us characterize the evolutionary history of humans. However, ancient DNA (aDNA) presents several practical challenges as various factors such as deamination, high fragmentation, environmental contamination of aDNA, and low amounts of recoverable endogenous DNA, make aDNA recovery and analysis more difficult than modern DNA. Most studies with aDNA leverage only SNP data, and only a few studies have made inferences on human demographic history based on haplotype data, possibly because haplotype estimation (or phasing) has not yet been systematically evaluated in the context of aDNA. Here, we evaluate how the unique challenges of aDNA can impact phasing quality. We also develop a software tool that simulates aDNA taking into account the features of aDNA as well as the evolutionary history of the population. We measured phasing error as a function of aDNA quality and demographic history, and found that low phasing error is achievable even for very ancient individuals ([~] 400 generations in the past) as long as contamination and read depth are adequate. Our results show that population splits or bottleneck events occurring between the reference and phased populations affect phasing quality, with bottlenecks resulting in the highest average error rates. Finally, we found that using estimated haplotypes, even if not completely accurate, is superior to using the simulated genotype data when reconstructing changes in population structure after population splits between present-day and ancient populations. AvailabilityAll software used for simulation and analysis is available at github.com/Jazpy/Paleogenomic-Datasim\nAn RNA foundation model enables discovery of disease mechanisms and candidate therapeutics\nAuthors: Celaj, A.; Gao, A. J.; Lau, T. T. Y.; Holgersen, E. M.; Lo, A.; Lodaya, V.; Cole, C. B.; Denroche, R. E.; Spickett, C.; Wagih, O.; Pinheiro, P. O.; Vora, P.; Mohammadi-Shemirani, P.; Chan, S.; Nussbaum, Z.; Zhang, X.; Zhu, H.; Ramamurthy, E.; Kanuparthi, B.; Iacocca, M.; Ly, D.; Kron, K.; Verby, M.; Cheung-Ong, K.; Shalev, Z.; Vaz, B.; Bhargava, S.; Yusuf, F.; Samuel, S.; Alibai, S.; Baghestani, Z.; He, X.; Krastel, K.; Oladapo, O.; Mohan, A.; Shanavas, A.; Bugno, M.; Bogojeski, J.; Schmitges, F.; Kim, C.; Grant, S.; Jayaraman, R.; Masud, T.; Deshwar, A.; Gandhi, S.; Frey, B. J.\nScore: 82.4, Published: 2023-09-26 DOI: 10.1101/2023.09.20.558508\nAccurately modeling and predicting RNA biology has been a long-standing challenge, bearing significant clinical ramifications for variant interpretation and the formulation of tailored therapeutics. We describe a foundation model for RNA biology, \"BigRNA\", which was trained on thousands of genome-matched datasets to predict tissue-specific RNA expression, splicing, microRNA sites, and RNA binding protein specificity from DNA sequence. Unlike approaches that are restricted to missense variants, BigRNA can identify pathogenic non-coding variant effects across diverse mechanisms, including polyadenylation, exon skipping and intron retention. BigRNA accurately predicted the effects of steric blocking oligonucleotides (SBOs) on increasing the expression of 4 out of 4 genes, and on splicing for 18 out of 18 exons across 14 genes, including those involved in Wilson disease and spinal muscular atrophy. We anticipate that BigRNA and foundation models like it will have widespread applications in the field of personalized RNA therapeutics.\nGut Microbiome Wellness Index 2 for Enhanced Health Status Prediction from Gut Microbiome Taxonomic Profiles\nAuthors: Chang, D.; Gupta, V. K.; Hur, B.; Cobo-Lopez, S.; Cunningham, K. Y.; Han, N.; Lee, I.; Kronzer, V. L.; Teigen, L.; Karnatovskaia, L.; Longbrake, E. E.; Davis, J. M.; Nelson, H.; Sung, J.\nScore: 10.8, Published: 2023-10-02 DOI: 10.1101/2023.09.30.560294\nRecent advancements in human gut microbiome research have revealed its crucial role in shaping innovative predictive healthcare applications. We introduce Gut Microbiome Wellness Index 2 (GMWI2), an advanced iteration of our original GMWI prototype, designed as a robust, disease-agnostic health status indicator based on gut microbiome taxonomic profiles. Our analysis involved pooling existing 8069 stool shotgun metagenome data across a global demographic landscape to effectively capture biological signals linking gut taxonomies to health. GMWI2 achieves a cross-validation balanced accuracy of 80% in distinguishing healthy (no disease) from non-healthy (diseased) individuals and surpasses 90% accuracy for samples with higher confidence (i.e., outside the \"reject option\"). The enhanced classification accuracy of GMWI2 outperforms both the original GMWI model and traditional species-level -diversity indices, suggesting a more reliable tool for differentiating between healthy and non-healthy phenotypes using gut microbiome data. Furthermore, by reevaluating and reinterpreting previously published data, GMWI2 provides fresh insights into the established understanding of how diet, antibiotic exposure, and fecal microbiota transplantation influence gut health. Looking ahead, GMWI2 represents a timely pivotal tool for evaluating health based on an individuals unique gut microbial composition, paving the way for the early screening of adverse gut health shifts. GMWI2 is offered as an open-source command-line tool, ensuring it is both accessible to and adaptable for researchers interested in the translational applications of human gut microbiome science.\nSegzoo: a turnkey system that summarizes genome annotations\nAuthors: Mendez, M.; Liu, Y.; Asenjo Ponce de Leon, M.; Hoffman, M. M.\nScore: 10.1, Published: 2023-10-05 DOI: 10.1101/2023.10.03.559369\nSegmentation and automated genome annotation (SAGA) techniques, such as Segway and ChromHMM, assign labels to every part of the genome, identifying similar patterns across multiple genomic input signals. Inferring biological meaning in these patterns remains challenging. Doing so requires a time-consuming process of manually downloading reference data, running multiple analysis methods, and interpreting many individual results. To simplify these tasks, we developed the turnkey system Segzoo. As input, Segzoo only requires a genome annotation file in browser extensible data (BED) format. It automatically downloads the rest of the data required for comparisons. Segzoo performs analyses using these data and summarizes results in a single visualization. Availability and Implementation: Source code for Python [\u0026ge;]3.7 on Linux freely available for download at https://github.com/hoffmangroup/segzoo under the GNU General Public License (GPL) version 2. Segzoo is also available in the Bioconda package segzoo: https://anaconda.org/bioconda/segzoo.\nBATTER: Accurate Prediction of Rho-dependent and Rho-independent Transcription Terminators in Metagenomes\nAuthors: Jin, Y.; Ma, H.; Xu, Z. Z.; Lu, Z. J.\nScore: 9.3, Published: 2023-10-03 DOI: 10.1101/2023.10.02.560326\nTranscription terminators mark the 3' ends of both coding and noncoding transcripts in bacteria and play crucial roles in gene regulations (such as controlling the stoichiometry of gene expression and conditionally switching off gene expression by inducing premature termination). Recently developed experimental 3' end mapping techniques greatly improved the current understanding of bacteria transcription termination, but these methods cannot detect transcripts that are unexpressed in the limited experimental conditions and cannot utilize the vast amount of information embedded in the rapidly growing metagenome data. Computational approaches can relieve these problems, but the development of such in-silico methods lags behind the experimental techniques. Previous computational tools are limited to predicting rho-independent terminators (RITs) and are primarily optimized for a few model species. The prediction of rho-dependent terminators (RDTs) which lack obvious consensus sequence patterns, and terminators in diverse non-model bacteria species still presents significant challenges. To address these challenges, we introduce BATTER (BActeria Transcript Three prime End Recognizer), a computational tool for predicting both RITs and RDTs in diverse bacteria species that allows metagenome-scale scanning. We developed a data augmentation pipeline by leveraging available high throughput 3' end mapping data in 17 bacteria species, and a large collection of 42,905 species-level representative bacteria genomes. Taking advantage context sensitive natural language processing techniques, we trained a BERT-CRF model, using both local features and context information for tagging terminators in genomic sequences. Systematic evaluations demonstrated our model's superiority: at a false positive rate of 0.1/kilobase, BATTER achieves a sensitivity of 0.924 for predicting E. coli RDTs; and a sensitivity of 0.756 for predicting terminators on term-seq dataset of oral microbiome, outperforming the best existing tool by 0.153. Based on BATTER's predictions, we systematically analyzed the clade-specific properties of bacteria terminators. The practical utility of BATTER was exemplified through two case studies: identifying functional transcripts from metatranscriptome data and discovering candidate noncoding RNAs related to antimicrobial resistance. As far as we know, BATTER is the first tool simultaneously predicting RITs and RDTs in diverse bacteria species. BATTER is available at https://github.com/lulab/BATTER.\nStableMate: a new statistical method to select stable predictors in omics data\nAuthors: Deng, Y.; Mao, J.; Choi, J.; Le Cao, K.-A.\nScore: 10.1, Published: 2023-09-28 DOI: 10.1101/2023.09.26.559658\nInferring reproducible relationships between biological variables remains a challenge in the statistical analysis of omics data. For example, methods that identify statistical associations may lack interpretability or reproducibility. The situation can be greatly improved, however, by introducing the measure of stability into the association, where small perturbations in the data do not affect the association. We developed this concept into a new statistical framework called StableMate. Given data observed in different environments or conditions, such as experimental batches or disease states, StableMate identifies predictors which are environment-agnostic or specific in predicting the response using stabilised regression. StableMate is a flexible framework that can be applied to a wide range of biological data types and questions. We applied StableMate to 1) RNA-seq data of breast cancer to discover genes and gene modules that consistently predict estrogen receptor expression across disease conditions, 2) metagenomics data to identify fecal microbial species that show persistent association with colon cancer across studies from different countries and 3) single cell RNA-seq data of glioblastoma to discern signature genes associated with development of microglia to a pro-tumour phenotype regardless of cell location in the core. StableMate is an innovative, adaptable and efficient variable selection framework that achieves a comprehensive characterisation of a biological system for a wide range of biological data types for regression and classification analyses.\nRanking of cell clusters in a single-cell RNA-sequencing analysis framework using prior knowledge\nAuthors: Oulas, A.; Savva, K.; Karathanasis, N.; Spyrou, G. M.\nScore: 8.7, Published: 2023-10-03 DOI: 10.1101/2023.10.02.560416\nPrioritization or ranking of different cell types in a scRNA-Seq framework can be performed in a variety of ways, some of these include: i) obtaining an indication of the proportion of cell types between the different conditions under study, ii) counting the number of differentially expressed genes (DEGs) between cell types and conditions in the experiment or, iii) prioritizing cell types based on prior knowledge about the conditions under study (i.e., a specific disease). These methods have drawbacks and limitations thus novel methods for improving cell ranking are required. Here we present a novel methodology that exploits prior knowledge in combination with expert-user information to accentuate cell types from a scRNA-seq analysis that yield the most biologically meaningful results. Prior knowledge is incorporated in a standardized, structured manner, whereby a checklist is attained by querying MalaCards human disease database with a disease of interest. The checklist is comprised of pathways and drugs and, optionally, drug mode of actions (MOAs), associated with the disease. The user is prompted to \"edit\" this checklist by removing or adding terms (in the form of keywords) from the list of predefined terms. Our methodology has substantial advantages to more traditional cell ranking techniques and provides an informative complementary methodology that utilizes prior knowledge in a rapid and automated manner, that has previously not been attempted by other studies. The current methodology is also implemented as an R package entitled Single Cell Ranking Analysis Toolkit (scRANK) and is available for download and installation via GitHub (https://#hub.com/aoulas/scRANK)\n",
  "wordCount" : "2446",
  "inLanguage": "en",
  "datePublished": "2023-10-08T10:37:16Z",
  "dateModified": "2023-10-08T10:37:16Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on October 8, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.01.560349">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.01.560349" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.01.560349">
        <p class="paperTitle">SaProt: Protein Language Modeling with Structure-aware Vocabulary</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.01.560349" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.01.560349" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Su, J.; Han, C.; Zhou, Y.; Shan, J.; Zhou, X.; Yuan, F.</p>
        <p class="info">Score: 42.4, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.01.560349' target='https://doi.org/10.1101/2023.10.01.560349'> 10.1101/2023.10.01.560349</a></p>
        <p class="abstract">Large-scale protein language models (PLMs), such as the ESM family, have achieved remarkable performance in various downstream tasks related to protein structure and function by undergoing unsupervised training on residue sequences. They have become essential tools for researchers and practitioners in biology. However, a limitation of vanilla PLMs is their lack of explicit consideration for protein structure information, which suggests the potential for further improvement. Motivated by this, we introduce the concept of a &#34;structure-aware vocabulary&#34; that integrates residue tokens with structure tokens. The structure tokens are derived by encoding the 3D structure of proteins using Foldseek. We then propose SaProt, a large-scale general-purpose PLM trained on an extensive dataset comprising approximately 40 million protein sequences and structures. Through extensive evaluation, our SaProt model surpasses well-established and renowned baselines across 10 significant downstream tasks, demonstrating its exceptional capacity and broad applicability. We have made the code1, pretrained model, and all relevant materials available at https://github.com/westlake-repl/SaProt.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560793">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560793" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560793">
        <p class="paperTitle">PROTOSPACEJAM: AN OPEN-SOURCE, CUSTOMIZABLE AND WEB-ACCESSIBLE DESIGN PLATFORM FOR CRISPR/CAS9 INSERTIONAL KNOCK-IN</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560793" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560793" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peng, D.; Vangipuram, M.; Wong, J.; Leonetti, M.</p>
        <p class="info">Score: 33.3, Published: 2023-10-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560793' target='https://doi.org/10.1101/2023.10.04.560793'> 10.1101/2023.10.04.560793</a></p>
        <p class="abstract">CRISPR/Cas9-mediated knock-in of DNA sequences enables precise genome engineering for research and therapeutic applications. However, designing effective guide RNAs (gRNAs) and homology-directed repair (HDR) donors remains a bottleneck. Here, we present protoSpaceJAM, an open-source algorithm to automate and optimize gRNA and HDR donor design for CRISPR/Cas9 insertional knock-in experiments. protoSpaceJAM utilizes biological rules to rank gRNAs based on specificity, distance to insertion site, and position relative to regulatory regions. protoSpaceJAM can introduce recoding mutations (silent mutations and mutations in non-coding sequences) in HDR donors to prevent re-cutting and increase knock-in efficiency. Users can customize parameters and design double-stranded or single-stranded donors. We validated protoSpaceJAM&#39;s design rules by demonstrating increased knock-in efficiency with recoding mutations and optimal strand selection for single-stranded donors. An additional module enables the design of genotyping primers for next-generation sequencing of edited alleles. Overall, protoSpaceJAM streamlines and optimizes CRISPR knock-in experimental design in a flexible and modular manner to benefit diverse research and therapeutic applications. protoSpaceJAM is available open-source as an interactive web tool at protospacejam.czbiohub.org or as a standalone Python package at github.com/czbiohub-sf/protoSpaceJAM.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.560637">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.560637" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.560637">
        <p class="paperTitle">The structural landscape of the immunoglobulin fold by large-scale de novo design</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.560637" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.560637" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Roel-Touris, J.; Carcelen, L.; Marcos, E.</p>
        <p class="info">Score: 24.9, Published: 2023-10-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.560637' target='https://doi.org/10.1101/2023.10.03.560637'> 10.1101/2023.10.03.560637</a></p>
        <p class="abstract">De novo designing immunoglobulin-like frameworks that allow for functional loop diversification shows great potential for crafting antibody-like scaffolds with fully customizable structures and functions. In this work, we combined de novo parametric design with deep-learning methods for protein structure prediction and design to explore the structural landscape of 7-stranded immunoglobulin domains. After screening folding of nearly 4 million designs, we have assembled a structurally diverse library of ~50,000 immunoglobulin domains with high-confidence AlphaFold2 predictions and structures diverging from naturally occurring ones. The designed dataset enabled us to identify structural requirements for the correct folding of immunoglobulin domains, shed light on {beta}-sheet-{beta}-sheet rotational preferences and how these are linked to functional properties. Our approach eliminates the need for preset loop conformations and opens the route to large-scale de novo design of immunoglobulin-like frameworks.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.28.560049">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.28.560049" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.28.560049">
        <p class="paperTitle">Simulation-based Benchmarking of Ancient Haplotype Inference for Detecting Population Structure</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.28.560049" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.28.560049" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Medina-Tretmanis, J.; Jay, F.; Avila-Arcos, M. C.; Huerta-Sanchez, E.</p>
        <p class="info">Score: 19.9, Published: 2023-10-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.28.560049' target='https://doi.org/10.1101/2023.09.28.560049'> 10.1101/2023.09.28.560049</a></p>
        <p class="abstract">Paleogenomic data has informed us about the movements, growth, and relationships of ancient populations. It has also given us context for medically relevant adaptations that appear in present-day humans due to introgression from other hominids, and it continues to help us characterize the evolutionary history of humans. However, ancient DNA (aDNA) presents several practical challenges as various factors such as deamination, high fragmentation, environmental contamination of aDNA, and low amounts of recoverable endogenous DNA, make aDNA recovery and analysis more difficult than modern DNA. Most studies with aDNA leverage only SNP data, and only a few studies have made inferences on human demographic history based on haplotype data, possibly because haplotype estimation (or phasing) has not yet been systematically evaluated in the context of aDNA. Here, we evaluate how the unique challenges of aDNA can impact phasing quality. We also develop a software tool that simulates aDNA taking into account the features of aDNA as well as the evolutionary history of the population. We measured phasing error as a function of aDNA quality and demographic history, and found that low phasing error is achievable even for very ancient individuals ([~] 400 generations in the past) as long as contamination and read depth are adequate. Our results show that population splits or bottleneck events occurring between the reference and phased populations affect phasing quality, with bottlenecks resulting in the highest average error rates. Finally, we found that using estimated haplotypes, even if not completely accurate, is superior to using the simulated genotype data when reconstructing changes in population structure after population splits between present-day and ancient populations.

AvailabilityAll software used for simulation and analysis is available at github.com/Jazpy/Paleogenomic-Datasim</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.558508">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.558508" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.558508">
        <p class="paperTitle">An RNA foundation model enables discovery of disease mechanisms and candidate therapeutics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.558508" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.558508" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Celaj, A.; Gao, A. J.; Lau, T. T. Y.; Holgersen, E. M.; Lo, A.; Lodaya, V.; Cole, C. B.; Denroche, R. E.; Spickett, C.; Wagih, O.; Pinheiro, P. O.; Vora, P.; Mohammadi-Shemirani, P.; Chan, S.; Nussbaum, Z.; Zhang, X.; Zhu, H.; Ramamurthy, E.; Kanuparthi, B.; Iacocca, M.; Ly, D.; Kron, K.; Verby, M.; Cheung-Ong, K.; Shalev, Z.; Vaz, B.; Bhargava, S.; Yusuf, F.; Samuel, S.; Alibai, S.; Baghestani, Z.; He, X.; Krastel, K.; Oladapo, O.; Mohan, A.; Shanavas, A.; Bugno, M.; Bogojeski, J.; Schmitges, F.; Kim, C.; Grant, S.; Jayaraman, R.; Masud, T.; Deshwar, A.; Gandhi, S.; Frey, B. J.</p>
        <p class="info">Score: 82.4, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.558508' target='https://doi.org/10.1101/2023.09.20.558508'> 10.1101/2023.09.20.558508</a></p>
        <p class="abstract">Accurately modeling and predicting RNA biology has been a long-standing challenge, bearing significant clinical ramifications for variant interpretation and the formulation of tailored therapeutics. We describe a foundation model for RNA biology, &#34;BigRNA&#34;, which was trained on thousands of genome-matched datasets to predict tissue-specific RNA expression, splicing, microRNA sites, and RNA binding protein specificity from DNA sequence. Unlike approaches that are restricted to missense variants, BigRNA can identify pathogenic non-coding variant effects across diverse mechanisms, including polyadenylation, exon skipping and intron retention. BigRNA accurately predicted the effects of steric blocking oligonucleotides (SBOs) on increasing the expression of 4 out of 4 genes, and on splicing for 18 out of 18 exons across 14 genes, including those involved in Wilson disease and spinal muscular atrophy. We anticipate that BigRNA and foundation models like it will have widespread applications in the field of personalized RNA therapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.30.560294">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.30.560294" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.30.560294">
        <p class="paperTitle">Gut Microbiome Wellness Index 2 for Enhanced Health Status Prediction from Gut Microbiome Taxonomic Profiles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.30.560294" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.30.560294" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chang, D.; Gupta, V. K.; Hur, B.; Cobo-Lopez, S.; Cunningham, K. Y.; Han, N.; Lee, I.; Kronzer, V. L.; Teigen, L.; Karnatovskaia, L.; Longbrake, E. E.; Davis, J. M.; Nelson, H.; Sung, J.</p>
        <p class="info">Score: 10.8, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.30.560294' target='https://doi.org/10.1101/2023.09.30.560294'> 10.1101/2023.09.30.560294</a></p>
        <p class="abstract">Recent advancements in human gut microbiome research have revealed its crucial role in shaping innovative predictive healthcare applications. We introduce Gut Microbiome Wellness Index 2 (GMWI2), an advanced iteration of our original GMWI prototype, designed as a robust, disease-agnostic health status indicator based on gut microbiome taxonomic profiles. Our analysis involved pooling existing 8069 stool shotgun metagenome data across a global demographic landscape to effectively capture biological signals linking gut taxonomies to health. GMWI2 achieves a cross-validation balanced accuracy of 80% in distinguishing healthy (no disease) from non-healthy (diseased) individuals and surpasses 90% accuracy for samples with higher confidence (i.e., outside the &#34;reject option&#34;). The enhanced classification accuracy of GMWI2 outperforms both the original GMWI model and traditional species-level -diversity indices, suggesting a more reliable tool for differentiating between healthy and non-healthy phenotypes using gut microbiome data. Furthermore, by reevaluating and reinterpreting previously published data, GMWI2 provides fresh insights into the established understanding of how diet, antibiotic exposure, and fecal microbiota transplantation influence gut health. Looking ahead, GMWI2 represents a timely pivotal tool for evaluating health based on an individuals unique gut microbial composition, paving the way for the early screening of adverse gut health shifts. GMWI2 is offered as an open-source command-line tool, ensuring it is both accessible to and adaptable for researchers interested in the translational applications of human gut microbiome science.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.559369">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.559369" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.559369">
        <p class="paperTitle">Segzoo: a turnkey system that summarizes genome annotations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.559369" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.559369" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mendez, M.; Liu, Y.; Asenjo Ponce de Leon, M.; Hoffman, M. M.</p>
        <p class="info">Score: 10.1, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.559369' target='https://doi.org/10.1101/2023.10.03.559369'> 10.1101/2023.10.03.559369</a></p>
        <p class="abstract">Segmentation and automated genome annotation (SAGA) techniques, such as Segway and ChromHMM, assign labels to every part of the genome, identifying similar patterns across multiple genomic input signals. Inferring biological meaning in these patterns remains challenging. Doing so requires a time-consuming process of manually downloading reference data, running multiple analysis methods, and interpreting many individual results. To simplify these tasks, we developed the turnkey system Segzoo. As input, Segzoo only requires a genome annotation file in browser extensible data (BED) format. It automatically downloads the rest of the data required for comparisons. Segzoo performs analyses using these data and summarizes results in a single visualization. Availability and Implementation: Source code for Python [&amp;ge;]3.7 on Linux freely available for download at https://github.com/hoffmangroup/segzoo under the GNU General Public License (GPL) version 2. Segzoo is also available in the Bioconda package segzoo: https://anaconda.org/bioconda/segzoo.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560326">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560326" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560326">
        <p class="paperTitle">BATTER: Accurate Prediction of Rho-dependent and Rho-independent Transcription Terminators in Metagenomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560326" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560326" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jin, Y.; Ma, H.; Xu, Z. Z.; Lu, Z. J.</p>
        <p class="info">Score: 9.3, Published: 2023-10-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560326' target='https://doi.org/10.1101/2023.10.02.560326'> 10.1101/2023.10.02.560326</a></p>
        <p class="abstract">Transcription terminators mark the 3&#39; ends of both coding and noncoding transcripts in bacteria and play crucial roles in gene regulations (such as controlling the stoichiometry of gene expression and conditionally switching off gene expression by inducing premature termination). Recently developed experimental 3&#39; end mapping techniques greatly improved the current understanding of bacteria transcription termination, but these methods cannot detect transcripts that are unexpressed in the limited experimental conditions and cannot utilize the vast amount of information embedded in the rapidly growing metagenome data. Computational approaches can relieve these problems, but the development of such in-silico methods lags behind the experimental techniques. Previous computational tools are limited to predicting rho-independent terminators (RITs) and are primarily optimized for a few model species. The prediction of rho-dependent terminators (RDTs) which lack obvious consensus sequence patterns, and terminators in diverse non-model bacteria species still presents significant challenges. To address these challenges, we introduce BATTER (BActeria Transcript Three prime End Recognizer), a computational tool for predicting both RITs and RDTs in diverse bacteria species that allows metagenome-scale scanning. We developed a data augmentation pipeline by leveraging available high throughput 3&#39; end mapping data in 17 bacteria species, and a large collection of 42,905 species-level representative bacteria genomes. Taking advantage context sensitive natural language processing techniques, we trained a BERT-CRF model, using both local features and context information for tagging terminators in genomic sequences. Systematic evaluations demonstrated our model&#39;s superiority: at a false positive rate of 0.1/kilobase, BATTER achieves a sensitivity of 0.924 for predicting E. coli RDTs; and a sensitivity of 0.756 for predicting terminators on term-seq dataset of oral microbiome, outperforming the best existing tool by 0.153. Based on BATTER&#39;s predictions, we systematically analyzed the clade-specific properties of bacteria terminators. The practical utility of BATTER was exemplified through two case studies: identifying functional transcripts from metatranscriptome data and discovering candidate noncoding RNAs related to antimicrobial resistance. As far as we know, BATTER is the first tool simultaneously predicting RITs and RDTs in diverse bacteria species. BATTER is available at https://github.com/lulab/BATTER.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559658">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559658" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559658">
        <p class="paperTitle">StableMate: a new statistical method to select stable predictors in omics data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559658" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559658" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Deng, Y.; Mao, J.; Choi, J.; Le Cao, K.-A.</p>
        <p class="info">Score: 10.1, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559658' target='https://doi.org/10.1101/2023.09.26.559658'> 10.1101/2023.09.26.559658</a></p>
        <p class="abstract">Inferring reproducible relationships between biological variables remains a challenge in the statistical analysis of omics data. For example, methods that identify statistical associations may lack interpretability or reproducibility. The situation can be greatly improved, however, by introducing the measure of stability into the association, where small perturbations in the data do not affect the association. We developed this concept into a new statistical framework called StableMate. Given data observed in different environments or conditions, such as experimental batches or disease states, StableMate identifies predictors which are environment-agnostic or specific in predicting the response using stabilised regression.

StableMate is a flexible framework that can be applied to a wide range of biological data types and questions. We applied StableMate to 1) RNA-seq data of breast cancer to discover genes and gene modules that consistently predict estrogen receptor expression across disease conditions, 2) metagenomics data to identify fecal microbial species that show persistent association with colon cancer across studies from different countries and 3) single cell RNA-seq data of glioblastoma to discern signature genes associated with development of microglia to a pro-tumour phenotype regardless of cell location in the core.

StableMate is an innovative, adaptable and efficient variable selection framework that achieves a comprehensive characterisation of a biological system for a wide range of biological data types for regression and classification analyses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560416">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560416" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560416">
        <p class="paperTitle">Ranking of cell clusters in a single-cell RNA-sequencing analysis framework using prior knowledge</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560416" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560416" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Oulas, A.; Savva, K.; Karathanasis, N.; Spyrou, G. M.</p>
        <p class="info">Score: 8.7, Published: 2023-10-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560416' target='https://doi.org/10.1101/2023.10.02.560416'> 10.1101/2023.10.02.560416</a></p>
        <p class="abstract">Prioritization or ranking of different cell types in a scRNA-Seq framework can be performed in a variety of ways, some of these include: i) obtaining an indication of the proportion of cell types between the different conditions under study, ii) counting the number of differentially expressed genes (DEGs) between cell types and conditions in the experiment or, iii) prioritizing cell types based on prior knowledge about the conditions under study (i.e., a specific disease). These methods have drawbacks and limitations thus novel methods for improving cell ranking are required. Here we present a novel methodology that exploits prior knowledge in combination with expert-user information to accentuate cell types from a scRNA-seq analysis that yield the most biologically meaningful results. Prior knowledge is incorporated in a standardized, structured manner, whereby a checklist is attained by querying MalaCards human disease database with a disease of interest. The checklist is comprised of pathways and drugs and, optionally, drug mode of actions (MOAs), associated with the disease. The user is prompted to &#34;edit&#34; this checklist by removing or adding terms (in the form of keywords) from the list of predefined terms. Our methodology has substantial advantages to more traditional cell ranking techniques and provides an informative complementary methodology that utilizes prior knowledge in a rapid and automated manner, that has previously not been attempted by other studies. The current methodology is also implemented as an R package entitled Single Cell Ranking Analysis Toolkit (scRANK) and is available for download and installation via GitHub (https://#hub.com/aoulas/scRANK)</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
