<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="LazyAF, a pipeline for accessible medium-scale in silico prediction of protein-protein interactions
Authors: McLean, T. C.
Score: 94.7, Published: 2024-02-01 DOI: 10.1101/2024.01.29.577767
Artificial intelligence has revolutionized the field of protein structure prediction. However, with more powerful and complex software being developed, it is accessibility and ease of use rather than capability that is quickly becoming a limiting factor to end users. Here, I present a Google Colaboratory-based pipeline, named LazyAF, which integrates the existing ColabFold BATCH to streamline the process of medium-scale protein-protein interaction prediction.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="LazyAF, a pipeline for accessible medium-scale in silico prediction of protein-protein interactions
Authors: McLean, T. C.
Score: 94.7, Published: 2024-02-01 DOI: 10.1101/2024.01.29.577767
Artificial intelligence has revolutionized the field of protein structure prediction. However, with more powerful and complex software being developed, it is accessibility and ease of use rather than capability that is quickly becoming a limiting factor to end users. Here, I present a Google Colaboratory-based pipeline, named LazyAF, which integrates the existing ColabFold BATCH to streamline the process of medium-scale protein-protein interaction prediction." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-04T10:38:39+00:00" />
<meta property="article:modified_time" content="2024-02-04T10:38:39+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="LazyAF, a pipeline for accessible medium-scale in silico prediction of protein-protein interactions
Authors: McLean, T. C.
Score: 94.7, Published: 2024-02-01 DOI: 10.1101/2024.01.29.577767
Artificial intelligence has revolutionized the field of protein structure prediction. However, with more powerful and complex software being developed, it is accessibility and ease of use rather than capability that is quickly becoming a limiting factor to end users. Here, I present a Google Colaboratory-based pipeline, named LazyAF, which integrates the existing ColabFold BATCH to streamline the process of medium-scale protein-protein interaction prediction."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "LazyAF, a pipeline for accessible medium-scale in silico prediction of protein-protein interactions\nAuthors: McLean, T. C.\nScore: 94.7, Published: 2024-02-01 DOI: 10.1101/2024.01.29.577767\nArtificial intelligence has revolutionized the field of protein structure prediction. However, with more powerful and complex software being developed, it is accessibility and ease of use rather than capability that is quickly becoming a limiting factor to end users. Here, I present a Google Colaboratory-based pipeline, named LazyAF, which integrates the existing ColabFold BATCH to streamline the process of medium-scale protein-protein interaction prediction.",
  "keywords": [
    
  ],
  "articleBody": " LazyAF, a pipeline for accessible medium-scale in silico prediction of protein-protein interactions\nAuthors: McLean, T. C.\nScore: 94.7, Published: 2024-02-01 DOI: 10.1101/2024.01.29.577767\nArtificial intelligence has revolutionized the field of protein structure prediction. However, with more powerful and complex software being developed, it is accessibility and ease of use rather than capability that is quickly becoming a limiting factor to end users. Here, I present a Google Colaboratory-based pipeline, named LazyAF, which integrates the existing ColabFold BATCH to streamline the process of medium-scale protein-protein interaction prediction. I apply LazyAF to predict the interactome of the 76 proteins encoded on a broad-host-range multi-drug resistance plasmid RK2, demonstrating the ease and accessibility the pipeline provides.\nPhylogenetic tree statistics: a systematic overview using the new R package 'treestats'\nAuthors: Janzen, T.; Etienne, R. S.\nScore: 86.2, Published: 2024-01-29 DOI: 10.1101/2024.01.24.576848\nPhylogenetic trees are believed to contain a wealth of information on diversification processes. Comparing phylogenetic trees is not straightforward due to their high dimensionality. Researchers have therefore defined a wide range of one-dimensional summary statistics. However, it remains unexplored to what extent these summary statistics cover the same underlying information and what summary statistics best explain observed variation across phylogenies. Furthermore, a large subset of available summary statistics focusses on measuring the topological features of a phylogenetic tree, but are often only explored at the extreme edge cases of the fully balanced or unbalanced tree and not for trees of intermediate balance. Here, we introduce a new R package that provides speed optimized code to compute 54 summary statistics. We study correlations between summary statistics on empirical trees and on trees simulated using several diversification models. Furthermore, we introduce an algorithm to create intermediately balanced trees in a well-defined manner, in order to explore variation in summary statistics across a balance gradient. We find that almost all summary statistics are correlated with tree size, and it is difficult if not impossible to correct for tree size, unless the tree generating model is known. Furthermore, we find that across empirical and simulated trees, at least two large clusters of correlated summary statistics can be found, where statistics group together based on information used (topology or branching times). However, the finer grained correlation structure appears to depend strongly on either the taxonomic group studied (in empirical studies) or the diversification model (in simulation studies). Nevertheless, we can identify multiple groups of summary statistics that are strongly and consistently correlated, indicating that these statistics measure the same underlying property of a tree. Lastly, we find that almost all topological summary statistics vary non-linearly and sometimes even non-monotonically with our intuitive balance gradient. Therefore, in order to avoid introducing biases and missing underlying information, we advocate for selecting as many summary statistics as possible in phylogenetic analyses. With the introduction of the treestats package, which provides fast and reliable calculations, such an approach is now routinely possible.\nProtein language models learn evolutionary statistics of interacting sequence motifs\nAuthors: Zhang, Z.; Wayment-Steele, H. K.; Brixi, G.; Wang, H.; Dal Peraro, M.; Kern, D.; Ovchinnikov, S.\nScore: 82.6, Published: 2024-01-31 DOI: 10.1101/2024.01.30.577970\nProtein language models (pLMs) have emerged as potent tools for predicting and designing protein structure and function, and the degree to which these models fundamentally understand the inherent biophysics of protein structure stands as an open question. Motivated by a discovery that pLM-based structure predictors erroneously predict non-physical structures for protein isoforms, we investigated the nature of sequence context needed for contact predictions in the pLM ESM-2. We demonstrate by use of a \"categorical Jacobian\" calculation that ESM-2 stores statistics of coevolving residues, analogously to simpler modelling approaches like Markov Random Fields and Multivariate Gaussian models. We further investigated how ESM-2 \"stores\" information needed to predict contacts by comparing sequence masking strategies, and found that providing local windows of sequence information allowed ESM-2 to best recover predicted contacts. This suggests that pLMs predict contacts by storing motifs of pairwise contacts. Our investigation highlights the limitations of current pLMs and underscores the importance of understanding the underlying mechanisms of these models.\nAccurate quantification of single-cell and single-nucleus RNA-seq transcripts using distinguishing flanking k-mers\nAuthors: Eldjarn Hjörleifsson, K.; Sullivan, D. K.; Swarna, N. P.; Holley, G.; Melsted, P.; Pachter, L.\nScore: 85.5, Published: 2024-01-26 DOI: 10.1101/2022.12.02.518832\nIn single-cell and single-nucleus RNA sequencing, the coexistence of nascent (unprocessed) and mature (processed) mRNA poses challenges in accurate read mapping and the interpretation of count matrices. The traditional transcriptome reference, defining the region of interest in bulk RNA-seq, restricts its focus to mature mRNA transcripts. This restriction leads to two problems: reads originating outside of the region of interest are prone to mismapping within this region, and additionally, such external reads cannot be matched to specific transcript targets. Expanding the region of interest to encompass both nascent and mature mRNA transcript targets provides a more comprehensive framework for RNA-seq analysis. Here, we introduce the concept of distinguishing flanking k-mers (DFKs) to improve mapping of sequencing reads. We have developed an algorithm to identify DFKs, which serve as a sophisticated background filter, enhancing the accuracy of mRNA quantification. This dual strategy of an expanded region of interest coupled with the use of DFKs enhances the precision in quantifying both mature and nascent mRNA molecules, as well as in delineating reads of ambiguous status.\nStaVia: Spatially and temporally aware cartography with higher order random walks for cell atlases\nAuthors: Stassen, S. V.; Kobashi, M.; Huang, Y.; Ho, J. W. K.; Tsia, K.\nScore: 42.7, Published: 2024-01-31 DOI: 10.1101/2024.01.29.577871\nSingle-cell atlases are critical for unraveling the cellular basis of health and disease, yet their sheer diversity and vast data landscape in time and space pose daunting computational challenges pertaining to the delineation of multiple simultaneously emerging lineages, the integration of spatial and temporal information, and the visualization of trajectories across large atlases with enough resolution to observe localized transitions. To tackle this intricacy, we introduce StaVia, a computational framework that synergizes multi-faceted single-cell data--spanning time-series data, spatial gene expression patterns, and directional trends from RNA velocity-- with higher-order random walks that leverage the memory of cells past states. StaVia fuses this method with a cartographic \"Atlas View\" that offers intuitive graph visualization, simultaneously capturing the nuanced details of cellular development at single-cell resolution as well as the broader connectivity of cell lineages, avoiding common pitfalls of merged distinct trajectories or missed transitional states seen in existing methods which are all memoryless. Notably, we demonstrate that StaVia unlocks new insights into placode development, radial glia pluripotency during neurulation, and the transitions pivotal to these processes in a large-scale Zebrafish developmental atlas. StaVia also allows spatially aware cartography that captures relationships between cell populations based on their spatial location as well as their gene expressions in a MERFISH dataset - underscoring its potential to dissect complex biological landscapes in both spatial and temporal contexts.\nPolarity-JaM: An image analysis toolbox for cell polarity, junction and morphology quantification\nAuthors: Giese, W.; Albrecht, J. P.; Oppenheim, O.; Akmeric, E. B.; Kraxner, J.; Schmidt, D.; Harrington, K. I.; Gerhardt, H.\nScore: 30.6, Published: 2024-01-29 DOI: 10.1101/2024.01.24.577027\nCell polarity involves the asymmetric distribution of cellular components such as signaling molecules and organelles within a cell, asymmetries of a cells shape as well as contacts with neighbouring cells. Gradients and mechanical forces often act as global cues that bias cell polarity and orientation, and polarity is coordinated by communication between adjacent cells. Advances in fluorescence microscopy combined with deep learning algorithms for image segmentation open up a wealth of possibilities to understand cell polarity behaviour in health and disease. We have therefore developed the open-source package Polarity-JaM, which offers versatile methods for performing reproducible exploratory image analysis. Multi-channel single cell segmentation is performed using a flexible and user-friendly interface to state-of-the-art deep learning algorithms. Interpretable single-cell features are automatically extracted, including cell and organelle orientation, cell-cell contact morphology, signaling molecule gradients, as well as collective orientation, tissue-wide size and shape variation. Circular statistics of cell polarity, including polarity indices, confidence intervals, and circular correlation analysis, can be computed using our web application. We have developed data graphs for comprehensive visualisation of key statistical measures and suggest the use of an adapted polarity index when the expected polarisation direction or the direction of a global cue is known a priori. The focus of our analysis is on fluorescence image data from endothelial cells (ECs) and their polarisation behaviour. ECs line the inside of blood vessels and are essential for vessel formation and repair, as well as for various cardiovascular diseases, cancer, and inflammation. However, the general architecture of the software will allow it to be applied to other cell types and image modalities. The Polarity-JaM package integrates these analyses in a reproducible manner with a level of documentation that allows the user to analyse image data accurately and efficiently. The software suite is documented at https://polarityjam.readthedocs.io and the app runs online at www.polarityjam.com.\nOrthology inference at scale with FastOMA\nAuthors: Majidian, S.; Nevers, Y.; Yazdizadeh Kharrazi, A.; Warwick Vesztrocy, A.; Pascarelli, S.; Moi, D.; Glover, N.; Altenhoff, A. M.; Dessimoz, C.\nScore: 25.5, Published: 2024-01-31 DOI: 10.1101/2024.01.29.577392\nThe surge in genome data, with ongoing efforts aiming to sequence 1.5M eukaryotes in a decade, could revolutionise genomics, revealing the origins, evolution, and genetic innovations of biological processes. Yet, traditional genomics methods scale poorly with such large datasets. Addressing this, \"FastOMA\" provides linear scalability, enabling the processing of thousands of eukaryotic genomes within a day. FastOMA maintains the high accuracy and resolution of the well-established OMA approach in benchmarks. FastOMA is available at https://github.com/DessimozLab/FastOMA/.\nBeav: A bacterial genome and mobile element annotation pipeline\nAuthors: Jung, J. M.; Rahman, A.; Schiffer, A. M.; Weisberg, A. J.\nScore: 29.3, Published: 2024-01-26 DOI: 10.1101/2024.01.25.577299\nComprehensive and accurate genome annotation is crucial for inferring the predicted functions of an organism. Numerous tools exist to annotate genes, gene clusters, mobile genetic elements, and other diverse features. However, these tools and pipelines can be difficult to install and run, be specialized for a particular element or feature, or lack annotations for larger elements that provide important genomic context. Integrating results across analyses is also important for understanding gene function. To address these challenges, we present the Beav annotation pipeline. Beav is a command-line tool that automates the annotation of bacterial genome sequences, mobile genetic elements, molecular systems and gene clusters, key regulatory features, and other elements. Beav uses existing tools in addition to custom models, scripts, and databases to annotate diverse elements, systems, and sequence features. Custom databases for plant-associated microbes are incorporated to improve annotation of key virulence and symbiosis genes in agriculturally important pathogens and mutualists. Beav includes an optional Agrobacterium-specific pipeline that identifies and classifies oncogenic plasmids and annotates plasmid-specific features. Following the completion of all analyses, annotations are consolidated to produce a single comprehensive output. Finally, Beav generates publication-quality genome and plasmid maps. Beav is on Bioconda and is available for download at https://github.com/weisberglab/beav.\nDeepGOMeta: Predicting functions for microbes\nAuthors: Tawfiq, R.; Niu, K.; Hoehndorf, R.; Kulmanov, M.\nScore: 21.2, Published: 2024-01-31 DOI: 10.1101/2024.01.28.577602\nAnalyzing microbial samples remains computationally challenging due to their diversity and complexity. The lack of robust de novo protein function prediction methods exacerbates the difficulty in deriving functional insights from these samples. Traditional prediction methods, dependent on homology and sequence similarity, often fail to predict functions for novel proteins and proteins without known homologs. Moreover, most of these methods have been trained on largely eukaryotic data, and have not been evaluated or applied to microbial datasets. This research introduces DeepGOMeta, a deep learning model designed for protein function prediction, as Gene Ontology (GO) terms, trained on a dataset relevant to microbes. The model is validated using novel evaluation strategies and applied to diverse microbial datasets. Data and code are available at https://github.com/bio-ontology-research-group/deepgometa\nRapid proteome-wide prediction of lipid-interacting proteins through ligand-guided structural genomics\nAuthors: Chou, J. C.-C.; Decosto, C. M.; Chatterjee, P.; Dassama, L. M. K.\nScore: 19.8, Published: 2024-01-30 DOI: 10.1101/2024.01.26.577452\nLipids are primary metabolites that play essential roles in multiple cellular pathways. Alterations in lipid metabolism and transport are associated with infectious diseases and cancers. As such, proteins involved in lipid synthesis, trafficking, and modification, are targets for therapeutic intervention. The ability to rapidly detect these proteins can accelerate their biochemical and structural characterization. However, it remains challenging to identify lipid binding motifs in proteins due to a lack of conservation at the amino acids level. Therefore, new bioinformatic tools that can detect conserved features in lipid binding sites are necessary. Here, we present Structure-based Lipid-interacting Pocket Predictor (SLiPP), a structural bioinformatics algorithm that uses machine learning to detect protein cavities capable of binding to lipids in experimental and AlphaFold-predicted protein structures. SLiPP, which can be used at proteome-wide scales, predicts lipid binding pockets with an accuracy of 96.8% and a F1 score of 86.9%. Our analyses revealed that the algorithm relies on hydrophobicity-related features to distinguish lipid binding pockets from those that bind to other ligands. Use of the algorithm to detect lipid binding proteins in the proteomes of various bacteria, yeast, and human have produced hits annotated or verified as lipid binding proteins, and many other uncharacterized proteins whose functions are not discernable from sequence alone. Because of its ability to identify novel lipid binding proteins, SLiPP can spur the discovery of new lipid metabolic and trafficking pathways that can be targeted for therapeutic development.\n",
  "wordCount" : "2223",
  "inLanguage": "en",
  "datePublished": "2024-02-04T10:38:39Z",
  "dateModified": "2024-02-04T10:38:39Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 4, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577767">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577767" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577767">
        <p class="paperTitle">LazyAF, a pipeline for accessible medium-scale in silico prediction of protein-protein interactions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577767" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577767" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McLean, T. C.</p>
        <p class="info">Score: 94.7, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577767' target='https://doi.org/10.1101/2024.01.29.577767'> 10.1101/2024.01.29.577767</a></p>
        <p class="abstract">Artificial intelligence has revolutionized the field of protein structure prediction. However, with more powerful and complex software being developed, it is accessibility and ease of use rather than capability that is quickly becoming a limiting factor to end users. Here, I present a Google Colaboratory-based pipeline, named LazyAF, which integrates the existing ColabFold BATCH to streamline the process of medium-scale protein-protein interaction prediction. I apply LazyAF to predict the interactome of the 76 proteins encoded on a broad-host-range multi-drug resistance plasmid RK2, demonstrating the ease and accessibility the pipeline provides.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.576848">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.576848" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.576848">
        <p class="paperTitle">Phylogenetic tree statistics: a systematic overview using the new R package &#39;treestats&#39;</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.576848" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.576848" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Janzen, T.; Etienne, R. S.</p>
        <p class="info">Score: 86.2, Published: 2024-01-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.576848' target='https://doi.org/10.1101/2024.01.24.576848'> 10.1101/2024.01.24.576848</a></p>
        <p class="abstract">Phylogenetic trees are believed to contain a wealth of information on diversification processes. Comparing phylogenetic trees is not straightforward due to their high dimensionality. Researchers have therefore defined a wide range of one-dimensional summary statistics. However, it remains unexplored to what extent these summary statistics cover the same underlying information and what summary statistics best explain observed variation across phylogenies. Furthermore, a large subset of available summary statistics focusses on measuring the topological features of a phylogenetic tree, but are often only explored at the extreme edge cases of the fully balanced or unbalanced tree and not for trees of intermediate balance.

Here, we introduce a new R package that provides speed optimized code to compute 54 summary statistics. We study correlations between summary statistics on empirical trees and on trees simulated using several diversification models. Furthermore, we introduce an algorithm to create intermediately balanced trees in a well-defined manner, in order to explore variation in summary statistics across a balance gradient.

We find that almost all summary statistics are correlated with tree size, and it is difficult if not impossible to correct for tree size, unless the tree generating model is known. Furthermore, we find that across empirical and simulated trees, at least two large clusters of correlated summary statistics can be found, where statistics group together based on information used (topology or branching times). However, the finer grained correlation structure appears to depend strongly on either the taxonomic group studied (in empirical studies) or the diversification model (in simulation studies). Nevertheless, we can identify multiple groups of summary statistics that are strongly and consistently correlated, indicating that these statistics measure the same underlying property of a tree. Lastly, we find that almost all topological summary statistics vary non-linearly and sometimes even non-monotonically with our intuitive balance gradient.

Therefore, in order to avoid introducing biases and missing underlying information, we advocate for selecting as many summary statistics as possible in phylogenetic analyses. With the introduction of the treestats package, which provides fast and reliable calculations, such an approach is now routinely possible.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.30.577970">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.30.577970" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.30.577970">
        <p class="paperTitle">Protein language models learn evolutionary statistics of interacting sequence motifs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.30.577970" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.30.577970" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, Z.; Wayment-Steele, H. K.; Brixi, G.; Wang, H.; Dal Peraro, M.; Kern, D.; Ovchinnikov, S.</p>
        <p class="info">Score: 82.6, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.30.577970' target='https://doi.org/10.1101/2024.01.30.577970'> 10.1101/2024.01.30.577970</a></p>
        <p class="abstract">Protein language models (pLMs) have emerged as potent tools for predicting and designing protein structure and function, and the degree to which these models fundamentally understand the inherent biophysics of protein structure stands as an open question. Motivated by a discovery that pLM-based structure predictors erroneously predict non-physical structures for protein isoforms, we investigated the nature of sequence context needed for contact predictions in the pLM ESM-2. We demonstrate by use of a &#34;categorical Jacobian&#34; calculation that ESM-2 stores statistics of coevolving residues, analogously to simpler modelling approaches like Markov Random Fields and Multivariate Gaussian models. We further investigated how ESM-2 &#34;stores&#34; information needed to predict contacts by comparing sequence masking strategies, and found that providing local windows of sequence information allowed ESM-2 to best recover predicted contacts. This suggests that pLMs predict contacts by storing motifs of pairwise contacts. Our investigation highlights the limitations of current pLMs and underscores the importance of understanding the underlying mechanisms of these models.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.12.02.518832">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.12.02.518832" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.12.02.518832">
        <p class="paperTitle">Accurate quantification of single-cell and single-nucleus RNA-seq transcripts using distinguishing flanking k-mers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.12.02.518832" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.12.02.518832" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eldjarn Hjörleifsson, K.; Sullivan, D. K.; Swarna, N. P.; Holley, G.; Melsted, P.; Pachter, L.</p>
        <p class="info">Score: 85.5, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.12.02.518832' target='https://doi.org/10.1101/2022.12.02.518832'> 10.1101/2022.12.02.518832</a></p>
        <p class="abstract">In single-cell and single-nucleus RNA sequencing, the coexistence of nascent (unprocessed) and mature (processed) mRNA poses challenges in accurate read mapping and the interpretation of count matrices. The traditional transcriptome reference, defining the  region of interest in bulk RNA-seq, restricts its focus to mature mRNA transcripts. This restriction leads to two problems: reads originating outside of the  region of interest are prone to mismapping within this region, and additionally, such external reads cannot be matched to specific transcript targets. Expanding the  region of interest to encompass both nascent and mature mRNA transcript targets provides a more comprehensive framework for RNA-seq analysis. Here, we introduce the concept of distinguishing flanking k-mers (DFKs) to improve mapping of sequencing reads. We have developed an algorithm to identify DFKs, which serve as a sophisticated  background filter, enhancing the accuracy of mRNA quantification. This dual strategy of an expanded region of interest coupled with the use of DFKs enhances the precision in quantifying both mature and nascent mRNA molecules, as well as in delineating reads of ambiguous status.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577871">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577871" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577871">
        <p class="paperTitle">StaVia: Spatially and temporally aware cartography with higher order random walks for cell atlases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577871" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577871" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stassen, S. V.; Kobashi, M.; Huang, Y.; Ho, J. W. K.; Tsia, K.</p>
        <p class="info">Score: 42.7, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577871' target='https://doi.org/10.1101/2024.01.29.577871'> 10.1101/2024.01.29.577871</a></p>
        <p class="abstract">Single-cell atlases are critical for unraveling the cellular basis of health and disease, yet their sheer diversity and vast data landscape in time and space pose daunting computational challenges pertaining to the delineation of multiple simultaneously emerging lineages, the integration of spatial and temporal information, and the visualization of trajectories across large atlases with enough resolution to observe localized transitions. To tackle this intricacy, we introduce StaVia, a computational framework that synergizes multi-faceted single-cell data--spanning time-series data, spatial gene expression patterns, and directional trends from RNA velocity-- with higher-order random walks that leverage the memory of cells past states. StaVia fuses this method with a cartographic &#34;Atlas View&#34; that offers intuitive graph visualization, simultaneously capturing the nuanced details of cellular development at single-cell resolution as well as the broader connectivity of cell lineages, avoiding common pitfalls of merged distinct trajectories or missed transitional states seen in existing methods which are all memoryless. Notably, we demonstrate that StaVia unlocks new insights into placode development, radial glia pluripotency during neurulation, and the transitions pivotal to these processes in a large-scale Zebrafish developmental atlas. StaVia also allows spatially aware cartography that captures relationships between cell populations based on their spatial location as well as their gene expressions in a MERFISH dataset - underscoring its potential to dissect complex biological landscapes in both spatial and temporal contexts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.577027">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.577027" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.577027">
        <p class="paperTitle">Polarity-JaM: An image analysis toolbox for cell polarity, junction and morphology quantification</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.577027" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.577027" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Giese, W.; Albrecht, J. P.; Oppenheim, O.; Akmeric, E. B.; Kraxner, J.; Schmidt, D.; Harrington, K. I.; Gerhardt, H.</p>
        <p class="info">Score: 30.6, Published: 2024-01-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.577027' target='https://doi.org/10.1101/2024.01.24.577027'> 10.1101/2024.01.24.577027</a></p>
        <p class="abstract">Cell polarity involves the asymmetric distribution of cellular components such as signaling molecules and organelles within a cell, asymmetries of a cells shape as well as contacts with neighbouring cells. Gradients and mechanical forces often act as global cues that bias cell polarity and orientation, and polarity is coordinated by communication between adjacent cells.

Advances in fluorescence microscopy combined with deep learning algorithms for image segmentation open up a wealth of possibilities to understand cell polarity behaviour in health and disease. We have therefore developed the open-source package Polarity-JaM, which offers versatile methods for performing reproducible exploratory image analysis. Multi-channel single cell segmentation is performed using a flexible and user-friendly interface to state-of-the-art deep learning algorithms. Interpretable single-cell features are automatically extracted, including cell and organelle orientation, cell-cell contact morphology, signaling molecule gradients, as well as collective orientation, tissue-wide size and shape variation. Circular statistics of cell polarity, including polarity indices, confidence intervals, and circular correlation analysis, can be computed using our web application. We have developed data graphs for comprehensive visualisation of key statistical measures and suggest the use of an adapted polarity index when the expected polarisation direction or the direction of a global cue is known a priori.

The focus of our analysis is on fluorescence image data from endothelial cells (ECs) and their polarisation behaviour. ECs line the inside of blood vessels and are essential for vessel formation and repair, as well as for various cardiovascular diseases, cancer, and inflammation. However, the general architecture of the software will allow it to be applied to other cell types and image modalities. The Polarity-JaM package integrates these analyses in a reproducible manner with a level of documentation that allows the user to analyse image data accurately and efficiently. The software suite is documented at https://polarityjam.readthedocs.io and the app runs online at www.polarityjam.com.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577392">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577392" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577392">
        <p class="paperTitle">Orthology inference at scale with FastOMA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577392" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577392" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Majidian, S.; Nevers, Y.; Yazdizadeh Kharrazi, A.; Warwick Vesztrocy, A.; Pascarelli, S.; Moi, D.; Glover, N.; Altenhoff, A. M.; Dessimoz, C.</p>
        <p class="info">Score: 25.5, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577392' target='https://doi.org/10.1101/2024.01.29.577392'> 10.1101/2024.01.29.577392</a></p>
        <p class="abstract">The surge in genome data, with ongoing efforts aiming to sequence 1.5M eukaryotes in a decade, could revolutionise genomics, revealing the origins, evolution, and genetic innovations of biological processes. Yet, traditional genomics methods scale poorly with such large datasets. Addressing this, &#34;FastOMA&#34; provides linear scalability, enabling the processing of thousands of eukaryotic genomes within a day. FastOMA maintains the high accuracy and resolution of the well-established OMA approach in benchmarks. FastOMA is available at https://github.com/DessimozLab/FastOMA/.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.25.577299">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.25.577299" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.25.577299">
        <p class="paperTitle">Beav: A bacterial genome and mobile element annotation pipeline</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.25.577299" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.25.577299" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jung, J. M.; Rahman, A.; Schiffer, A. M.; Weisberg, A. J.</p>
        <p class="info">Score: 29.3, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.25.577299' target='https://doi.org/10.1101/2024.01.25.577299'> 10.1101/2024.01.25.577299</a></p>
        <p class="abstract">Comprehensive and accurate genome annotation is crucial for inferring the predicted functions of an organism. Numerous tools exist to annotate genes, gene clusters, mobile genetic elements, and other diverse features. However, these tools and pipelines can be difficult to install and run, be specialized for a particular element or feature, or lack annotations for larger elements that provide important genomic context. Integrating results across analyses is also important for understanding gene function. To address these challenges, we present the Beav annotation pipeline. Beav is a command-line tool that automates the annotation of bacterial genome sequences, mobile genetic elements, molecular systems and gene clusters, key regulatory features, and other elements. Beav uses existing tools in addition to custom models, scripts, and databases to annotate diverse elements, systems, and sequence features. Custom databases for plant-associated microbes are incorporated to improve annotation of key virulence and symbiosis genes in agriculturally important pathogens and mutualists. Beav includes an optional Agrobacterium-specific pipeline that identifies and classifies oncogenic plasmids and annotates plasmid-specific features. Following the completion of all analyses, annotations are consolidated to produce a single comprehensive output. Finally, Beav generates publication-quality genome and plasmid maps. Beav is on Bioconda and is available for download at https://github.com/weisberglab/beav.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.28.577602">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.28.577602" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.28.577602">
        <p class="paperTitle">DeepGOMeta: Predicting functions for microbes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.28.577602" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.28.577602" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tawfiq, R.; Niu, K.; Hoehndorf, R.; Kulmanov, M.</p>
        <p class="info">Score: 21.2, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.28.577602' target='https://doi.org/10.1101/2024.01.28.577602'> 10.1101/2024.01.28.577602</a></p>
        <p class="abstract">Analyzing microbial samples remains computationally challenging due to their diversity and complexity. The lack of robust de novo protein function prediction methods exacerbates the difficulty in deriving functional insights from these samples. Traditional prediction methods, dependent on homology and sequence similarity, often fail to predict functions for novel proteins and proteins without known homologs. Moreover, most of these methods have been trained on largely eukaryotic data, and have not been evaluated or applied to microbial datasets. This research introduces DeepGOMeta, a deep learning model designed for protein function prediction, as Gene Ontology (GO) terms, trained on a dataset relevant to microbes. The model is validated using novel evaluation strategies and applied to diverse microbial datasets. Data and code are available at https://github.com/bio-ontology-research-group/deepgometa</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.26.577452">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.26.577452" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.26.577452">
        <p class="paperTitle">Rapid proteome-wide prediction of lipid-interacting proteins through ligand-guided structural genomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.26.577452" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.26.577452" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chou, J. C.-C.; Decosto, C. M.; Chatterjee, P.; Dassama, L. M. K.</p>
        <p class="info">Score: 19.8, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.26.577452' target='https://doi.org/10.1101/2024.01.26.577452'> 10.1101/2024.01.26.577452</a></p>
        <p class="abstract">Lipids are primary metabolites that play essential roles in multiple cellular pathways. Alterations in lipid metabolism and transport are associated with infectious diseases and cancers. As such, proteins involved in lipid synthesis, trafficking, and modification, are targets for therapeutic intervention. The ability to rapidly detect these proteins can accelerate their biochemical and structural characterization. However, it remains challenging to identify lipid binding motifs in proteins due to a lack of conservation at the amino acids level. Therefore, new bioinformatic tools that can detect conserved features in lipid binding sites are necessary. Here, we present Structure-based Lipid-interacting Pocket Predictor (SLiPP), a structural bioinformatics algorithm that uses machine learning to detect protein cavities capable of binding to lipids in experimental and AlphaFold-predicted protein structures. SLiPP, which can be used at proteome-wide scales, predicts lipid binding pockets with an accuracy of 96.8% and a F1 score of 86.9%. Our analyses revealed that the algorithm relies on hydrophobicity-related features to distinguish lipid binding pockets from those that bind to other ligands. Use of the algorithm to detect lipid binding proteins in the proteomes of various bacteria, yeast, and human have produced hits annotated or verified as lipid binding proteins, and many other uncharacterized proteins whose functions are not discernable from sequence alone. Because of its ability to identify novel lipid binding proteins, SLiPP can spur the discovery of new lipid metabolic and trafficking pathways that can be targeted for therapeutic development.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
