<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Colabfold predicts alternative protein structures from single sequences, coevolution unnecessary for AF-cluster
Authors: Porter, L. L.; Chakravarty, D.; Schafer, J. W.; Chen, E. A.
Score: 31.2, Published: 2023-11-25 DOI: 10.1101/2023.11.21.567977
Though typically associated with a single folded state, globular proteins are dynamic and often assume alternative or transient structures important for their functions1,2. Wayment-Steele, et al. steered ColabFold3 to predict alternative structures of several proteins using a method they call AF-cluster4.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="Colabfold predicts alternative protein structures from single sequences, coevolution unnecessary for AF-cluster
Authors: Porter, L. L.; Chakravarty, D.; Schafer, J. W.; Chen, E. A.
Score: 31.2, Published: 2023-11-25 DOI: 10.1101/2023.11.21.567977
Though typically associated with a single folded state, globular proteins are dynamic and often assume alternative or transient structures important for their functions1,2. Wayment-Steele, et al. steered ColabFold3 to predict alternative structures of several proteins using a method they call AF-cluster4." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-29T10:40:14+00:00" />
<meta property="article:modified_time" content="2023-11-29T10:40:14+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="Colabfold predicts alternative protein structures from single sequences, coevolution unnecessary for AF-cluster
Authors: Porter, L. L.; Chakravarty, D.; Schafer, J. W.; Chen, E. A.
Score: 31.2, Published: 2023-11-25 DOI: 10.1101/2023.11.21.567977
Though typically associated with a single folded state, globular proteins are dynamic and often assume alternative or transient structures important for their functions1,2. Wayment-Steele, et al. steered ColabFold3 to predict alternative structures of several proteins using a method they call AF-cluster4."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "Colabfold predicts alternative protein structures from single sequences, coevolution unnecessary for AF-cluster\nAuthors: Porter, L. L.; Chakravarty, D.; Schafer, J. W.; Chen, E. A.\nScore: 31.2, Published: 2023-11-25 DOI: 10.1101/2023.11.21.567977\nThough typically associated with a single folded state, globular proteins are dynamic and often assume alternative or transient structures important for their functions1,2. Wayment-Steele, et al. steered ColabFold3 to predict alternative structures of several proteins using a method they call AF-cluster4.",
  "keywords": [
    
  ],
  "articleBody": " Colabfold predicts alternative protein structures from single sequences, coevolution unnecessary for AF-cluster\nAuthors: Porter, L. L.; Chakravarty, D.; Schafer, J. W.; Chen, E. A.\nScore: 31.2, Published: 2023-11-25 DOI: 10.1101/2023.11.21.567977\nThough typically associated with a single folded state, globular proteins are dynamic and often assume alternative or transient structures important for their functions1,2. Wayment-Steele, et al. steered ColabFold3 to predict alternative structures of several proteins using a method they call AF-cluster4. They propose that AF-cluster \"enables ColabFold to sample alternate states of known metamorphic proteins with high confidence\" by first clustering multiple sequence alignments (MSAs) in a way that \"deconvolves\" coevolutionary information specific to different conformations and then using these clusters as input for ColabFold. Contrary to this Coevolution Assumption, clustered MSAs are not needed to make these predictions. Rather, these alternative structures can be predicted from single sequences and/or sequence similarity, indicating that coevolutionary information is unnecessary for predictive success and may not be used at all. These results suggest that AF-clusters predictive scope is likely limited to sequences with distinct-yet-homologous structures within ColabFolds training set.\nBARtab \u0026 bartools: an integrated Nextflow pipeline and R package for the analysis of synthetic cellular barcodes in the genome and transcriptome\nAuthors: Vassiliadis, D.; Dawson, M. A.; Holze, H.; Fennell, K. A.; Talarmain, L.; Lam, E. Y.\nScore: 28.3, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568179\nCellular barcoding using heritable synthetic barcodes coupled to high throughput sequencing is a powerful technique for the accurate tracing of clonal lineages in a wide variety of biological contexts. Recent studies have integrated cellular barcoding with a single-cell transcriptomics readout, extending the capabilities of these lineage tracing methods to the single-cell level. However there remains a lack of scalable and standardised open-source tools to pre-process and visualise both population-level and single-cell level cellular barcoding datasets. To address these limitations, we developed BARtab, a portable and scalable Nextflow pipeline that automates upstream barcode extraction, quality control, filtering and enumeration from high throughput sequencing data; and bartools, an open-source R package that streamlines the analysis and visualisation of population and single-cell level cellular barcoding datasets. BARtab contains additional methods for the extraction and annotation of transcribed barcodes from single-cell RNA-seq and spatial transcriptomics experiments, thus extending this analytical toolbox to also support novel expressed cellular barcoding methodologies. We showcase the integrated BARtab and bartools workflow through comparison with previously published toolsets and via the analysis of exemplar bulk, single-cell, and spatial transcriptomics cellular barcoding datasets.\nNanopore and Illumina Sequencing Reveal Different Viral Populations from Human Gut Samples\nAuthors: Cook, R.; Telatin, A.; Hsieh, S.-Y.; Newberry, F.; Tariq, M. A.; Baker, D. J.; Carding, S. R.; Adriaenssens, E. M.\nScore: 26.4, Published: 2023-11-24 DOI: 10.1101/2023.11.24.568560\nThe advent of viral metagenomics, or viromics, has improved our knowledge and understanding of global viral diversity. High-throughput sequencing technologies enable explorations of the ecological roles, contributions to host metabolism, and the influence of viruses in various environments including the human gut microbiome. However, the bacterial metagenomic studies frequently have the advantage. The adoption of advanced technologies like long-read sequencing has the potential to be transformative in refining viromics and metagenomics. Here, we examined the effectiveness of long-read and hybrid sequencing by comparing Illumina short-read and Oxford Nanopore Technology (ONT) long-read sequencing technologies and different assembly strategies on recovering viral genomes from human faecal samples. Our findings showed that if a single sequencing technology is to be chosen for virome analysis, Illumina was preferable due to its superior ability to recover fully resolved viral genomes and minimise erroneous genomes. While ONT assemblies were effective in recovering viral diversity, the challenges related to input requirements and the necessity for amplification made it less ideal as a standalone solution. However, using a combined, hybrid approach enabled a more authentic representation of viral diversity to be obtained within samples. Impact StatementViral metagenomics, or viromics, has revolutionised our understanding of global viral diversity however long-read and hybrid approaches are not yet widespread in this field. Here, we compared the performance of Illumina short-read and Nanopore long-read assembly approaches for recovering fully resolved viral genomes from human faecal samples. We highlight Illuminas short-read sequencing for recovering fully resolved viral genomes, while acknowledging Oxford Nanopore Technologys long-read sequencing for capturing broader viral diversity. However, a hybrid approach, utilising both technologies, may mitigate the limitations of one technology alone. Data SummaryAll reads used in this study are available on European Nucleotide Archive (ENA) within the project PRJEB47625.\nLearning sequence, structure, and function representations of proteins with language models\nAuthors: Hamamsy, T.; Barot, M.; Morton, J. T.; Steinegger, M.; Bonneau, R.; Cho, K.\nScore: 25.1, Published: 2023-11-26 DOI: 10.1101/2023.11.26.568742\nThe sequence-structure-function relationships that ultimately generate the diversity of extant observed proteins is complex, as proteins bridge the gap between multiple informational and physical scales involved in nearly all cellular processes. One limitation of existing protein annotation databases such as UniProt is that less than 1% of proteins have experimentally verified functions, and computational methods are needed to fill in the missing information. Here, we demonstrate that a multi-aspect framework based on protein language models can learn sequence-structure-function representations of amino acid sequences, and can provide the foundation for sensitive sequence-structure-function aware protein sequence search and annotation. Based on this model, we introduce a multi-aspect information retrieval system for proteins, Protein-Vec, covering sequence, structure, and function aspects, that enables computational protein annotation and function prediction at tree-of-life scales.\nPolygraph: A Software Framework for the Systematic Assessment of Synthetic Regulatory DNA Elements\nAuthors: Gupta, A.; Lal, A.; Gunsalus, L. M.; Biancalani, T.; Eraslan, G.\nScore: 19.8, Published: 2023-11-27 DOI: 10.1101/2023.11.27.568764\nThe design of regulatory elements is pivotal in numerous therapeutic interventions, including gene and cell therapy, wherein the typical objective is to engineer DNA sequences exhibiting specific attributes like cell-type specificity and elevated expression levels. However, the systematic assessment of these constructed DNA sequences remains challenging due to the absence of robust metrics and an integrated software framework. Here, we introduce Polygraph, a Python framework for evaluating synthetic DNA sequences. Polygraph provides a variety of features to streamline the synthesis and scrutiny of regulatory elements, incorporating features like a diversity index, motif and k-mer composition, similarity to endogenous regulatory sequences, and screening with predictive and foundational models. Consequently, Polygraph stands as the first instrument for the assessment of synthetic regulatory sequences, enabling expedited progress in therapeutic applications and improving our understanding of gene regulatory mechanisms.\nTaxometer: Improving taxonomic classification of metagenomics contigs\nAuthors: Kutuzova, S.; Nielsen, M.; Piera Lindez, P.; Nybo Nissen, J.; Rasmussen, S.\nScore: 18.8, Published: 2023-11-23 DOI: 10.1101/2023.11.23.568413\nFor taxonomy based classification of metagenomics assembled contigs, current methods use sequence similarity to identify their most likely taxonomy. However, in the related field of metagenomics binning contigs are routinely clustered using information from both the contig sequences and their abundance. We introduce Taxometer, a neural network based method that improves the annotations and estimates the quality of any taxonomic classifier by combining contig abundance profiles and tetra-nucleotide frequencies. When applied to five short-read CAMI2 datasets, it increased the average share of correct species-level contig annotations of the MMSeqs2 tool from 66.6% to 86.2% and reduced the share of wrong species-level annotations in the CAMI2 Rhizosphere dataset two-fold on average for Metabuli, Centrifuge, and Kraken2. Finally, we applied Taxometer to two complex long-read metagenomics data sets for benchmarking taxonomic classifiers. Taxometer is available as open-source software and can enhance any taxonomic annotation of metagenomic contigs.\nDL4MicEverywhere: Deep learning for microscopy made flexible, shareable, and reproducible\nAuthors: Hidalgo-Cenalmor, I.; Pylvänäinen, J. W.; Ferreira, M. G.; Russell, C. T.; Arganda-Carreras, I.; AI4Life Consortium, ; Jacquemet, G.; Henriques, R.; Gomez-de-Mariscal, E.\nScore: 49.9, Published: 2023-11-19 DOI: 10.1101/2023.11.19.567606\nDeep learning has revolutionised the analysis of extensive microscopy datasets, yet challenges persist in the widespread adoption of these techniques. Many lack access to training data, computing resources, and expertise to develop complex models. We introduce DL4MicEverywhere, advancing our previous ZeroCostDL4Mic platform, to make deep learning more accessible. DL4MicEverywhere uniquely allows flexible training and deployment across diverse computational environments by encapsulating methods in interactive Jupyter notebooks within Docker containers -a standalone virtualisation of required packages and code to reproduce a computational environment-. This enhances reproducibility and convenience. The platform includes twice as many techniques as originally provided by ZeroCostDL4Mic and enables community contributions via automated build pipelines. DL4MicEverywhere empowers participatory innovation and aims to democratise deep learning for bioimage analysis.\nMetagenome profiling and containment estimation through abundance-corrected k-mer sketching with sylph\nAuthors: Shaw, J.; Yu, Y. W.\nScore: 37.2, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567879\nProfiling metagenomes against databases allows for the detection and quantification of microbes, even at low abundances where assembly is not possible. We introduce sylph (https://github.com/bluenote-1577/sylph), a metagenome profiler that estimates metagenome-genome average nucleotide identity (ANI) through zero-inflated Poisson k-mer statistics, enabling ANI-based taxa detection. Sylph is the most accurate method on the CAMI2 marine dataset, and compared to Kraken2 for multi-sample profiling, sylph takes 10x less CPU time and uses 30x less memory. Sylphs ANI estimates provide an orthogonal signal to abundance, enabling an ANI-based metagenome-wide association study for Parkinsons disease against 289,323 genomes, confirming known butyrate-PD associations at the strain level. Sylph takes \u003c 1 minute and 16 GB of RAM to profile against 85,205 prokaryotic and 2,917,521 viral genomes, detecting 30x more viral sequences in the human gut compared to RefSeq. Sylph offers precise, efficient profiling with accurate ANI estimation for even low-coverage genomes.\nSequence-based drug design using transformers\nAuthors: Zhang, S.; Huo, D.; Horne, R. I.; Qi, Y.; Ojeda, S. P.; Yan, A.; Vendruscolo, M.\nScore: 14.0, Published: 2023-11-27 DOI: 10.1101/2023.11.27.568880\nProtein-ligand interactions play central roles in biological processes and are of key importance in drug design. Deep learning-based approaches are emerging as cost-effective alternatives to high-throughput experimental methods for the screening of large libraries of ligands. Here, to predict the binding affinity between proteins and small molecules, we introduce Ligand-Transformer, a deep learning framework based on the AlphaFold2 transformer architecture. We applied Ligand-Transformer to screen inhibitors targeting the mutant EGFRLTC kinase, identifying compounds with low nanomolar potency. We then used this approach to predict the conformational population shifts induced by ABL kinase inhibitors. To show the applicability of Ligand-Transformer to disordered proteins, we explored the binding of small molecules to the Alzheimer's Abeta peptide, identifying compounds that delayed its aggregation. Overall, Ligand-Transformer illustrates the potential of transformers in accurately predicting the interactions of small molecules with both ordered and disordered proteins, thus uncovering molecular mechanisms and facilitating the initial steps in drug discovery.\nTowards the accurate modelling of antibody-antigen complexes from sequence using machine learning and information-driven docking\nAuthors: Giulini, M.; Schneider, C.; Cutting, D.; Desai, N.; Deane, C.; Bonvin, A. M. J. J.\nScore: 12.5, Published: 2023-11-17 DOI: 10.1101/2023.11.17.567543\nAntibody-antigen complex modelling is an important step in computational workflows for therapeutic antibody design. While experimentally determined structures of both antibody and the cognate antigen are often not available, recent advances in machine learning-driven protein modelling have enabled accurate prediction of both antibody and antigen structures. Here, we analyse the ability of protein-protein docking tools to use machine learning generated input structures for information-driven docking. We find that HADDOCK can generate accurate models of antibodyantigen complexes using an ensemble of antibody structures generated by machine learning tools and AlphaFold2 predicted antigen structures. Targeted docking using knowledge of the complementary determining regions on the antibody and some information about the targeted epitope allows the generation of high quality models of the complex with reduced sampling, resulting in a computationally cheap protocol that outperforms the ZDOCK baseline. The data set used to benchmark the docking protocols in this study is available at github.com/haddocking/ai-antibodies. The docking models will be deposited at data.sbgrid.org/labs/32/ upon acceptance.\n",
  "wordCount" : "1936",
  "inLanguage": "en",
  "datePublished": "2023-11-29T10:40:14Z",
  "dateModified": "2023-11-29T10:40:14Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 29, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.567977">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.567977" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.567977">
        <p class="paperTitle">Colabfold predicts alternative protein structures from single sequences, coevolution unnecessary for AF-cluster</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.567977" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.567977" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Porter, L. L.; Chakravarty, D.; Schafer, J. W.; Chen, E. A.</p>
        <p class="info">Score: 31.2, Published: 2023-11-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.567977' target='https://doi.org/10.1101/2023.11.21.567977'> 10.1101/2023.11.21.567977</a></p>
        <p class="abstract">Though typically associated with a single folded state, globular proteins are dynamic and often assume alternative or transient structures important for their functions1,2. Wayment-Steele, et al. steered ColabFold3 to predict alternative structures of several proteins using a method they call AF-cluster4. They propose that AF-cluster &#34;enables ColabFold to sample alternate states of known metamorphic proteins with high confidence&#34; by first clustering multiple sequence alignments (MSAs) in a way that &#34;deconvolves&#34; coevolutionary information specific to different conformations and then using these clusters as input for ColabFold. Contrary to this Coevolution Assumption, clustered MSAs are not needed to make these predictions. Rather, these alternative structures can be predicted from single sequences and/or sequence similarity, indicating that coevolutionary information is unnecessary for predictive success and may not be used at all. These results suggest that AF-clusters predictive scope is likely limited to sequences with distinct-yet-homologous structures within ColabFolds training set.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568179">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568179" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568179">
        <p class="paperTitle">BARtab &amp; bartools: an integrated Nextflow pipeline and R package for the analysis of synthetic cellular barcodes in the genome and transcriptome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568179" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568179" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vassiliadis, D.; Dawson, M. A.; Holze, H.; Fennell, K. A.; Talarmain, L.; Lam, E. Y.</p>
        <p class="info">Score: 28.3, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568179' target='https://doi.org/10.1101/2023.11.21.568179'> 10.1101/2023.11.21.568179</a></p>
        <p class="abstract">Cellular barcoding using heritable synthetic barcodes coupled to high throughput sequencing is a powerful technique for the accurate tracing of clonal lineages in a wide variety of biological contexts. Recent studies have integrated cellular barcoding with a single-cell transcriptomics readout, extending the capabilities of these lineage tracing methods to the single-cell level. However there remains a lack of scalable and standardised open-source tools to pre-process and visualise both population-level and single-cell level cellular barcoding datasets. To address these limitations, we developed BARtab, a portable and scalable Nextflow pipeline that automates upstream barcode extraction, quality control, filtering and enumeration from high throughput sequencing data; and bartools, an open-source R package that streamlines the analysis and visualisation of population and single-cell level cellular barcoding datasets. BARtab contains additional methods for the extraction and annotation of transcribed barcodes from single-cell RNA-seq and spatial transcriptomics experiments, thus extending this analytical toolbox to also support novel expressed cellular barcoding methodologies. We showcase the integrated BARtab and bartools workflow through comparison with previously published toolsets and via the analysis of exemplar bulk, single-cell, and spatial transcriptomics cellular barcoding datasets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.24.568560">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.24.568560" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.24.568560">
        <p class="paperTitle">Nanopore and Illumina Sequencing Reveal Different Viral Populations from Human Gut Samples</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.24.568560" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.24.568560" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cook, R.; Telatin, A.; Hsieh, S.-Y.; Newberry, F.; Tariq, M. A.; Baker, D. J.; Carding, S. R.; Adriaenssens, E. M.</p>
        <p class="info">Score: 26.4, Published: 2023-11-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.24.568560' target='https://doi.org/10.1101/2023.11.24.568560'> 10.1101/2023.11.24.568560</a></p>
        <p class="abstract">The advent of viral metagenomics, or viromics, has improved our knowledge and understanding of global viral diversity. High-throughput sequencing technologies enable explorations of the ecological roles, contributions to host metabolism, and the influence of viruses in various environments including the human gut microbiome. However, the bacterial metagenomic studies frequently have the advantage. The adoption of advanced technologies like long-read sequencing has the potential to be transformative in refining viromics and metagenomics.

Here, we examined the effectiveness of long-read and hybrid sequencing by comparing Illumina short-read and Oxford Nanopore Technology (ONT) long-read sequencing technologies and different assembly strategies on recovering viral genomes from human faecal samples.

Our findings showed that if a single sequencing technology is to be chosen for virome analysis, Illumina was preferable due to its superior ability to recover fully resolved viral genomes and minimise erroneous genomes. While ONT assemblies were effective in recovering viral diversity, the challenges related to input requirements and the necessity for amplification made it less ideal as a standalone solution. However, using a combined, hybrid approach enabled a more authentic representation of viral diversity to be obtained within samples.

Impact StatementViral metagenomics, or viromics, has revolutionised our understanding of global viral diversity however long-read and hybrid approaches are not yet widespread in this field. Here, we compared the performance of Illumina short-read and Nanopore long-read assembly approaches for recovering fully resolved viral genomes from human faecal samples. We highlight Illuminas short-read sequencing for recovering fully resolved viral genomes, while acknowledging Oxford Nanopore Technologys long-read sequencing for capturing broader viral diversity. However, a hybrid approach, utilising both technologies, may mitigate the limitations of one technology alone.

Data SummaryAll reads used in this study are available on European Nucleotide Archive (ENA) within the project PRJEB47625.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.26.568742">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.26.568742" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.26.568742">
        <p class="paperTitle">Learning sequence, structure, and function representations of proteins with language models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.26.568742" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.26.568742" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hamamsy, T.; Barot, M.; Morton, J. T.; Steinegger, M.; Bonneau, R.; Cho, K.</p>
        <p class="info">Score: 25.1, Published: 2023-11-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.26.568742' target='https://doi.org/10.1101/2023.11.26.568742'> 10.1101/2023.11.26.568742</a></p>
        <p class="abstract">The sequence-structure-function relationships that ultimately generate the diversity of extant observed proteins is complex, as proteins bridge the gap between multiple informational and physical scales involved in nearly all cellular processes. One limitation of existing protein annotation databases such as UniProt is that less than 1% of proteins have experimentally verified functions, and computational methods are needed to fill in the missing information. Here, we demonstrate that a multi-aspect framework based on protein language models can learn sequence-structure-function representations of amino acid sequences, and can provide the foundation for sensitive sequence-structure-function aware protein sequence search and annotation. Based on this model, we introduce a multi-aspect information retrieval system for proteins, Protein-Vec, covering sequence, structure, and function aspects, that enables computational protein annotation and function prediction at tree-of-life scales.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.568764">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.568764" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.568764">
        <p class="paperTitle">Polygraph: A Software Framework for the Systematic Assessment of Synthetic Regulatory DNA Elements</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.568764" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.568764" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gupta, A.; Lal, A.; Gunsalus, L. M.; Biancalani, T.; Eraslan, G.</p>
        <p class="info">Score: 19.8, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.568764' target='https://doi.org/10.1101/2023.11.27.568764'> 10.1101/2023.11.27.568764</a></p>
        <p class="abstract">The design of regulatory elements is pivotal in numerous therapeutic interventions, including gene and cell therapy, wherein the typical objective is to engineer DNA sequences exhibiting specific attributes like cell-type specificity and elevated expression levels. However, the systematic assessment of these constructed DNA sequences remains challenging due to the absence of robust metrics and an integrated software framework. Here, we introduce Polygraph, a Python framework for evaluating synthetic DNA sequences. Polygraph provides a variety of features to streamline the synthesis and scrutiny of regulatory elements, incorporating features like a diversity index, motif and k-mer composition, similarity to endogenous regulatory sequences, and screening with predictive and foundational models. Consequently, Polygraph stands as the first instrument for the assessment of synthetic regulatory sequences, enabling expedited progress in therapeutic applications and improving our understanding of gene regulatory mechanisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.568413">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.568413" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.568413">
        <p class="paperTitle">Taxometer: Improving taxonomic classification of metagenomics contigs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.568413" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.568413" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kutuzova, S.; Nielsen, M.; Piera Lindez, P.; Nybo Nissen, J.; Rasmussen, S.</p>
        <p class="info">Score: 18.8, Published: 2023-11-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.568413' target='https://doi.org/10.1101/2023.11.23.568413'> 10.1101/2023.11.23.568413</a></p>
        <p class="abstract">For taxonomy based classification of metagenomics assembled contigs, current methods use sequence similarity to identify their most likely taxonomy. However, in the related field of metagenomics binning contigs are routinely clustered using information from both the contig sequences and their abundance. We introduce Taxometer, a neural network based method that improves the annotations and estimates the quality of any taxonomic classifier by combining contig abundance profiles and tetra-nucleotide frequencies. When applied to five short-read CAMI2 datasets, it increased the average share of correct species-level contig annotations of the MMSeqs2 tool from 66.6% to 86.2% and reduced the share of wrong species-level annotations in the CAMI2 Rhizosphere dataset two-fold on average for Metabuli, Centrifuge, and Kraken2. Finally, we applied Taxometer to two complex long-read metagenomics data sets for benchmarking taxonomic classifiers. Taxometer is available as open-source software and can enhance any taxonomic annotation of metagenomic contigs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.19.567606">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.19.567606" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.19.567606">
        <p class="paperTitle">DL4MicEverywhere: Deep learning for microscopy made flexible, shareable, and reproducible</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.19.567606" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.19.567606" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hidalgo-Cenalmor, I.; Pylvänäinen, J. W.; Ferreira, M. G.; Russell, C. T.; Arganda-Carreras, I.; AI4Life Consortium,  ; Jacquemet, G.; Henriques, R.; Gomez-de-Mariscal, E.</p>
        <p class="info">Score: 49.9, Published: 2023-11-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.19.567606' target='https://doi.org/10.1101/2023.11.19.567606'> 10.1101/2023.11.19.567606</a></p>
        <p class="abstract">Deep learning has revolutionised the analysis of extensive microscopy datasets, yet challenges persist in the widespread adoption of these techniques. Many lack access to training data, computing resources, and expertise to develop complex models. We introduce DL4MicEverywhere, advancing our previous ZeroCostDL4Mic platform, to make deep learning more accessible. DL4MicEverywhere uniquely allows flexible training and deployment across diverse computational environments by encapsulating methods in interactive Jupyter notebooks within Docker containers -a standalone virtualisation of required packages and code to reproduce a computational environment-. This enhances reproducibility and convenience. The platform includes twice as many techniques as originally provided by ZeroCostDL4Mic and enables community contributions via automated build pipelines. DL4MicEverywhere empowers participatory innovation and aims to democratise deep learning for bioimage analysis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567879">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567879" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567879">
        <p class="paperTitle">Metagenome profiling and containment estimation through abundance-corrected k-mer sketching with sylph</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567879" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567879" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shaw, J.; Yu, Y. W.</p>
        <p class="info">Score: 37.2, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567879' target='https://doi.org/10.1101/2023.11.20.567879'> 10.1101/2023.11.20.567879</a></p>
        <p class="abstract">Profiling metagenomes against databases allows for the detection and quantification of microbes, even at low abundances where assembly is not possible. We introduce sylph (https://github.com/bluenote-1577/sylph), a metagenome profiler that estimates metagenome-genome average nucleotide identity (ANI) through zero-inflated Poisson k-mer statistics, enabling ANI-based taxa detection. Sylph is the most accurate method on the CAMI2 marine dataset, and compared to Kraken2 for multi-sample profiling, sylph takes 10x less CPU time and uses 30x less memory. Sylphs ANI estimates provide an orthogonal signal to abundance, enabling an ANI-based metagenome-wide association study for Parkinsons disease against 289,323 genomes, confirming known butyrate-PD associations at the strain level. Sylph takes &lt; 1 minute and 16 GB of RAM to profile against 85,205 prokaryotic and 2,917,521 viral genomes, detecting 30x more viral sequences in the human gut compared to RefSeq. Sylph offers precise, efficient profiling with accurate ANI estimation for even low-coverage genomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.568880">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.568880" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.568880">
        <p class="paperTitle">Sequence-based drug design using transformers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.568880" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.568880" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, S.; Huo, D.; Horne, R. I.; Qi, Y.; Ojeda, S. P.; Yan, A.; Vendruscolo, M.</p>
        <p class="info">Score: 14.0, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.568880' target='https://doi.org/10.1101/2023.11.27.568880'> 10.1101/2023.11.27.568880</a></p>
        <p class="abstract">Protein-ligand interactions play central roles in biological processes and are of key importance in drug design. Deep learning-based approaches are emerging as cost-effective alternatives to high-throughput experimental methods for the screening of large libraries of ligands. Here, to predict the binding affinity between proteins and small molecules, we introduce Ligand-Transformer, a deep learning framework based on the AlphaFold2 transformer architecture. We applied Ligand-Transformer to screen inhibitors targeting the mutant EGFRLTC kinase, identifying compounds with low nanomolar potency. We then used this approach to predict the conformational population shifts induced by ABL kinase inhibitors. To show the applicability of Ligand-Transformer to disordered proteins, we explored the binding of small molecules to the Alzheimer&#39;s Abeta peptide, identifying compounds that delayed its aggregation. Overall, Ligand-Transformer illustrates the potential of transformers in accurately predicting the interactions of small molecules with both ordered and disordered proteins, thus uncovering molecular mechanisms and facilitating the initial steps in drug discovery.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.17.567543">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.17.567543" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.17.567543">
        <p class="paperTitle">Towards the accurate modelling of antibody-antigen complexes from sequence using machine learning and information-driven docking</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.17.567543" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.17.567543" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Giulini, M.; Schneider, C.; Cutting, D.; Desai, N.; Deane, C.; Bonvin, A. M. J. J.</p>
        <p class="info">Score: 12.5, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.17.567543' target='https://doi.org/10.1101/2023.11.17.567543'> 10.1101/2023.11.17.567543</a></p>
        <p class="abstract">Antibody-antigen complex modelling is an important step in computational workflows for therapeutic antibody design. While experimentally determined structures of both antibody and the cognate antigen are often not available, recent advances in machine learning-driven protein modelling have enabled accurate prediction of both antibody and antigen structures. Here, we analyse the ability of protein-protein docking tools to use machine learning generated input structures for information-driven docking. We find that HADDOCK can generate accurate models of antibodyantigen complexes using an ensemble of antibody structures generated by machine learning tools and AlphaFold2 predicted antigen structures. Targeted docking using knowledge of the complementary determining regions on the antibody and some information about the targeted epitope allows the generation of high quality models of the complex with reduced sampling, resulting in a computationally cheap protocol that outperforms the ZDOCK baseline. The data set used to benchmark the docking protocols in this study is available at github.com/haddocking/ai-antibodies. The docking models will be deposited at data.sbgrid.org/labs/32/ upon acceptance.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
