<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="simona: a Comprehensive R package for Semantic Similarity Analysis on Bio-Ontologies
Authors: Gu, Z.
Score: 29.8, Published: 2023-12-05 DOI: 10.1101/2023.12.03.569758
Bio-ontologies are keys in structuring complex biological information for effective data integration and knowledge representation. In this paper, we present simona, a novel R package for semantic similarity analysis on general bio-ontolgies. Simona implements infrastructures for ontology analysis by offering efficient data structures, fast ontology traversal methods, and elegant visualizations.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="simona: a Comprehensive R package for Semantic Similarity Analysis on Bio-Ontologies
Authors: Gu, Z.
Score: 29.8, Published: 2023-12-05 DOI: 10.1101/2023.12.03.569758
Bio-ontologies are keys in structuring complex biological information for effective data integration and knowledge representation. In this paper, we present simona, a novel R package for semantic similarity analysis on general bio-ontolgies. Simona implements infrastructures for ontology analysis by offering efficient data structures, fast ontology traversal methods, and elegant visualizations." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-13T10:39:57+00:00" />
<meta property="article:modified_time" content="2023-12-13T10:39:57+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="simona: a Comprehensive R package for Semantic Similarity Analysis on Bio-Ontologies
Authors: Gu, Z.
Score: 29.8, Published: 2023-12-05 DOI: 10.1101/2023.12.03.569758
Bio-ontologies are keys in structuring complex biological information for effective data integration and knowledge representation. In this paper, we present simona, a novel R package for semantic similarity analysis on general bio-ontolgies. Simona implements infrastructures for ontology analysis by offering efficient data structures, fast ontology traversal methods, and elegant visualizations."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "simona: a Comprehensive R package for Semantic Similarity Analysis on Bio-Ontologies\nAuthors: Gu, Z.\nScore: 29.8, Published: 2023-12-05 DOI: 10.1101/2023.12.03.569758\nBio-ontologies are keys in structuring complex biological information for effective data integration and knowledge representation. In this paper, we present simona, a novel R package for semantic similarity analysis on general bio-ontolgies. Simona implements infrastructures for ontology analysis by offering efficient data structures, fast ontology traversal methods, and elegant visualizations.",
  "keywords": [
    
  ],
  "articleBody": " simona: a Comprehensive R package for Semantic Similarity Analysis on Bio-Ontologies\nAuthors: Gu, Z.\nScore: 29.8, Published: 2023-12-05 DOI: 10.1101/2023.12.03.569758\nBio-ontologies are keys in structuring complex biological information for effective data integration and knowledge representation. In this paper, we present simona, a novel R package for semantic similarity analysis on general bio-ontolgies. Simona implements infrastructures for ontology analysis by offering efficient data structures, fast ontology traversal methods, and elegant visualizations. Moreover, it provides a robust toolbox supporting over 70 methods for semantic similarity analysis. With simona, we conduct a benchmark against current semantic similarity methods. The results demonstrate methods are clustered based on their mathematical methodologies, providing guidance for researchers in the selection of appropriate methods. Additionally, we explore annotation-based versus topology-based methods, revealing that semantic similarities solely based on ontology topology can efficiently reveal semantic similarity structures, facilitating analysis on less-studied organisms and other ontologies. Simona is freely available from https://bioconductor.org/packages/simona/.\nReconstructing Spatial Transcriptomics at the Single-cell Resolution with BayesDeep\nAuthors: Jiang, X.; Dong, L.; Wang, S.; Wen, Z.; Chen, M.; Xu, L.; Xiao, G.; Li, Q.\nScore: 18.2, Published: 2023-12-08 DOI: 10.1101/2023.12.07.570715\nSpatially resolved transcriptomics (SRT) techniques have revolutionized the characterization of molecular profiles while preserving spatial and morphological context. However, most next-generation sequencing-based SRT techniques are limited to measuring gene expression in a confined array of spots, capturing only a fraction of the spatial domain. Typically, these spots encompass gene expression from a few to hundreds of cells, underscoring a critical need for more detailed, single-cell resolution SRT data to enhance our understanding of biological functions within the tissue context. Addressing this challenge, we introduce BayesDeep, a novel Bayesian hierarchical model that leverages cellular morphological data from histology images, commonly paired with SRT data, to reconstruct SRT data at the single-cell resolution. BayesDeep effectively model count data from SRT studies via a negative binomial regression model. This model incorporates explanatory variables such as cell types and nuclei-shape information for each cell extracted from the paired histology image. A feature selection scheme is integrated to examine the association between the morphological and molecular profiles, thereby improving the model robustness. We applied BayesDeep to two real SRT datasets, successfully demonstrating its capability to reconstruct SRT data at the single-cell resolution. This advancement not only yields new biological insights but also significantly enhances various downstream analyses, such as pseudotime and cell-cell communication.\nBiolearn, an open-source library for biomarkers of aging\nAuthors: Ying, K.; Paulson, S.; Perez-Guevara, M.; Emamifar, M.; Casas Martinez, M.; Kwon, D.; Poganik, J. R.; Moqri, M.; Gladyshev, V. N.\nScore: 18.2, Published: 2023-12-10 DOI: 10.1101/2023.12.02.569722\nIdentifying and validating biomarkers of aging is pivotal for understanding the aging process and testing longevity interventions. Despite the development of numerous aging biomarkers, their clinical validation remains elusive, largely due to the lack of cross-population validation, which is hampered by disparate biomarker designs and inconsistencies in dataset structures. To bridge this gap, we introduce Biolearn, an innovative open-source library dedicated to the implementation and application of aging biomarkers. Biolearn facilitates (1) harmonization of existing aging biomarkers, while presenting a structured framework for novel biomarkers in standardized formats; (2) unification of public datasets, ensuring coherent structuring and formatting, thus simplifying cross-population validation studies; and (3) provision of computational methodologies to assess any harmonized biomarker against unified datasets. By furnishing a community-driven platform, Biolearn significantly augments the development, assessment, and validation trajectories of aging biomarkers, paving the way toward more rigorous clinical validation and, ultimately, application in clinical trials targeting healthy longevity. The Biolearn package is open-source and freely available at https://Bio-Learn.github.io/\nCytoSimplex: Visualizing Single-cell Fates and Transitions on a Simplex\nAuthors: Liu, J.; Wang, Y.; Li, C.; Gu, Y.; Ono, N.; Welch, J. D.\nScore: 15.5, Published: 2023-12-08 DOI: 10.1101/2023.12.07.570655\nSummaryCells differentiate to their final fates along unique trajectories, often involving multi-potent progenitors that can produce multiple terminally differentiated cell types. Recent developments in single-cell transcriptomic and epigenomic measurement provide tremendous opportunities for mapping these trajectories. The visualization of single-cell data often relies on dimension reduction methods such as UMAP to simplify high-dimensional single-cell data down into an understandable two-dimensional (2D) form. However, these visualization methods can be misleading and often do not effectively represent the direction of cell differentiation. To address these limitations, we developed a new approach that places each cell from a single-cell dataset within a simplex whose vertices correspond to terminally differentiated cell types. Our approach can quantify and visualize current cell fate commitment and future cell potential. We developed CytoSimplex, a standalone open-source package implemented in R and Python that provides simple and intuitive visualizations of cell differentiation in 2D ternary and three-dimensional (3D) quaternary plots. We believe that CytoSimplex can help researchers gain a better understanding of cell type transitions in specific tissues and characterize developmental processes. Availability and implementationThe R version of CytoSimplex is available on Github at https://github.com/welch-lab/CytoSimplex. The Python version of CytoSimplex is available on Github at https://github.com/welch-lab/pyCytoSimplex.\nscDiffEq: drift-diffusion modeling of single-cell dynamics with neural stochastic differential equations\nAuthors: Vinyard, M. E.; Rasmussen, A.; Li, R.; Getz, G.; Pinello, L.\nScore: 12.6, Published: 2023-12-08 DOI: 10.1101/2023.12.06.570508\nSingle-cell sequencing measurements facilitate the reconstruction of dynamic biology by capturing snapshots of the molecular profiles of individual cells. Cell fate decisions in development and disease are orchestrated through an intricate balance of deterministic and stochastic regulatory events. Drift-diffusion equations are effective in modeling single-cell dynamics from high-dimensional single-cell measurements. While existing solutions describe the deterministic dynamics associated with the drift term of these equations at the level of cell state, the diffusion is modeled as a constant across cell states. To fully understand the dynamic regulatory logic in development and disease, models explicitly attuned to the balance between deterministic and stochastic biology are required. Addressing these limitations, we introduce scDiffEq, a generative framework for learning neural stochastic differential equations that approximate the deterministic and stochastic dynamics in biology. Using lineage-traced single-cell data, we demonstrate that scDiffEq offers improved reconstruction of held-out cell states and prediction of cell fate from multipotent progenitors during hematopoiesis. By imparting in silico perturbations to multipotent progenitor cells, we find that scDiffEq accurately recapitulates the dynamics of CRISPR-perturbed hematopoiesis. Using scDiffEq, we simulate high-resolution developmental cell trajectories, modeling their drift and diffusion, enabling us to study their time-dependent gene-level dynamics.\nDisDock: A Deep Learning Method for Metal Ion-Protein Redocking\nAuthors: Lin, M.; Li, K.; Zhang, Y.; Pan, F.; Wu, W.; Zhang, J.\nScore: 10.8, Published: 2023-12-08 DOI: 10.1101/2023.12.07.570531\nThe structures of metalloproteins are essential for comprehending their functions and interactions. The breakthrough of AlphaFold has made it possible to predict protein structures with experimental accuracy. However, the type of metal ion that a metalloprotein binds and the binding structure are still not readily available, even with the predicted protein structure. In this study, we present DisDock, a physics-driven deep learning method for predicting protein-metal docking. DisDock takes distogram of randomly initialized protein-ligand configuration as input and outputs the distogram of the predicted binding complex. It combines the U-net architecture with self-attention modules to enhance model performance. Taking inspiration from the physical principle that atoms in closer proximity display a stronger mutual attraction, this predictor capitalizes on geometric information to uncover latent characteristics indicative of atom interactions. To train our model, we employ a high-quality metalloprotein dataset sourced from the Mother of All Databases (MOAD). Experimental results demonstrate that our approach outperforms other existing methods in prediction accuracy for various types of metal ions.\nPrediction of DNA i-Motifs Via Machine Learning\nAuthors: Yang, B.; Guneri, D.; Yu, H.; Wright, E.; Chen, W.; Waller, Z. A. E.; DING, Y.\nScore: 9.5, Published: 2023-12-12 DOI: 10.1101/2023.12.11.571121\ni-Motifs (iMs), are secondary structures formed in cytosine-rich DNA sequences and are involved in multiple functions in the genome. Although putative iM forming sequences are widely distributed in the human genome, the folding status and strength of putative iMs vary dramatically. Much previous research on iM has focused on assessing the iM folding properties using biophysical experiments. However, there are no dedicated computational tools for predicting the folding status and strength of iM structures. Here, we introduce a machine learning pipeline, iM-Seeker, to predict both folding status and structural stability of DNA iMs. The programme iM-Seeker incorporates a Balanced Random Forest classifier trained on genome-wide iMab antibody-based CUT\u0026Tag sequencing data to predict the folding status and an Extreme Gradient Boosting regressor to estimate the folding strength according to both literature biophysical data and our in-house biophysical experiments. iM-Seeker predicts DNA iM folding status with a classification accuracy of 81% and estimates the folding strength with coefficient of determination (R2) of 0.642 on the test set. Model interpretation confirms that the nucleotide composition of the C-rich sequence significantly affects iM stability, with a positive correlation with sequences containing cytosine and thymine and a negative correlation with guanine and adenine.\nGeneExt: a gene model extension tool for enhanced single-cell RNA-seq analysis\nAuthors: Zolotarov, G.; Grau-Bove, X.; Sebe-Pedros, A.\nScore: 9.4, Published: 2023-12-07 DOI: 10.1101/2023.12.05.570120\nIncomplete gene models can negatively impact single-cell gene expression quantification. This is particularly true in non-model species where often 3 ends are inaccurately annotated, while most single-cell transcriptomics methods only capturing the 3 transcript region. As a result, many genes are incorrectly quantified or not detected at all. Here, we introduce a tool that leverages 3-end single-cell transcriptomics data to refine gene annotations. We exemplify its usage and its impact on the gene expression quantification of seven non-model organism single-cell atlases. By extending and homogenizing gene annotations, our tool will help improve biological interpretation and cross-species comparisons of cell type expression atlases.\nIdentification and quantification of transposable element transcripts using Long-Read RNA-seq in Drosophila germline tissues\nAuthors: Rebollo, R.; Gerenton, P.; Cumunel, E.; Mary, A.; Sabot, F.; Burlet, N.; Gillet, B.; Hughes, S.; Siqueira Oliveira, D.; Goubert, C.; Fablet, M.; Vieira, C.; Lacroix, V.\nScore: 15.9, Published: 2023-12-07 DOI: 10.1101/2023.05.27.542554\nTransposable elements (TEs) are repeated DNA sequences potentially able to move throughout the genome. In addition to their inherent mutagenic effects, TEs can disrupt nearby genes by donating their intrinsic regulatory sequences, for instance, promoting the ectopic expression of a cellular gene. TE transcription is therefore not only necessary for TE transposition per se but can also be associated with TE-gene fusion transcripts, and in some cases, be the product of pervasive transcription. Hence, correctly determining the transcription state of a TE copy is essential to apprehend the impact of the TE in the host genome. Methods to identify and quantify TE transcription have mostly relied on short RNA-seq reads to estimate TE expression at the family level while using specific algorithms to discriminate copy-specific transcription. However, assigning short reads to their correct genomic location, and genomic feature is not trivial. Here we retrieved full-length cDNA (TeloPrime, Lexogen) of Drosophila melanogaster gonads and sequenced them using Oxford Nanopore Technologies. We show that long-read RNA-seq can be used to identify and quantify transcribed TEs at the copy level. In particular, TE insertions overlapping annotated genes are better estimated using long reads than short reads. Nevertheless, long TE transcripts (\u003e 4.5 kb) are not well captured. Most expressed TE insertions correspond to copies that have lost their ability to transpose, and within a family, only a few copies are indeed expressed. Long-read sequencing also allowed the identification of spliced transcripts for around 105 TE copies. Overall, this first comparison of TEs between testes and ovaries uncovers differences in their transcriptional landscape, at the subclass and insertion level.\nDeconV: Probabilistic Cell Type Deconvolution from Bulk RNA-sequencing Data\nAuthors: Gynter, A.; Meistermann, D.; Lahdesmaki, H.; Kilpinen, H.\nScore: 7.0, Published: 2023-12-08 DOI: 10.1101/2023.12.07.570524\nBulk RNA-Seq remains a widely adopted technique to profile gene expression, primarily due to the persistent challenges associated with achieving single-cell resolution. However, a key challenge is accurately estimating the proportions of different cell types within these bulk samples. To address this issue, we introduce DeconV, a probabilistic framework for cell-type deconvolution that uses scRNA-Seq data as a reference. This approach aims to mitigate some of the limitations in existing methods by incorporating statistical frameworks developed for scRNA-Seq, thereby simplifying issues related to reference preprocessing such as normalization and marker gene selection. We benchmarked DeconV against established methods, including MuSiC, CIBERSORTx, and Scaden. Our results show that DeconV performs comparably in terms of accuracy to the best-performing method, Scaden, but provides additional interpretability by offering confidence intervals for its predictions. Furthermore, the modular design of DeconV allows for the investigation of discrepancies between bulk-sequenced samples and artificially generated pseudo-bulk samples.\n",
  "wordCount" : "2073",
  "inLanguage": "en",
  "datePublished": "2023-12-13T10:39:57Z",
  "dateModified": "2023-12-13T10:39:57Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 13, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.03.569758">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.03.569758" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.03.569758">
        <p class="paperTitle">simona: a Comprehensive R package for Semantic Similarity Analysis on Bio-Ontologies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.03.569758" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.03.569758" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gu, Z.</p>
        <p class="info">Score: 29.8, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.03.569758' target='https://doi.org/10.1101/2023.12.03.569758'> 10.1101/2023.12.03.569758</a></p>
        <p class="abstract">Bio-ontologies are keys in structuring complex biological information for effective data integration and knowledge representation. In this paper, we present simona, a novel R package for semantic similarity analysis on general bio-ontolgies. Simona implements infrastructures for ontology analysis by offering efficient data structures, fast ontology traversal methods, and elegant visualizations. Moreover, it provides a robust toolbox supporting over 70 methods for semantic similarity analysis. With simona, we conduct a benchmark against current semantic similarity methods. The results demonstrate methods are clustered based on their mathematical methodologies, providing guidance for researchers in the selection of appropriate methods. Additionally, we explore annotation-based versus topology-based methods, revealing that semantic similarities solely based on ontology topology can efficiently reveal semantic similarity structures, facilitating analysis on less-studied organisms and other ontologies. Simona is freely available from https://bioconductor.org/packages/simona/.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570715">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570715" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570715">
        <p class="paperTitle">Reconstructing Spatial Transcriptomics at the Single-cell Resolution with BayesDeep</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570715" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570715" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jiang, X.; Dong, L.; Wang, S.; Wen, Z.; Chen, M.; Xu, L.; Xiao, G.; Li, Q.</p>
        <p class="info">Score: 18.2, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570715' target='https://doi.org/10.1101/2023.12.07.570715'> 10.1101/2023.12.07.570715</a></p>
        <p class="abstract">Spatially resolved transcriptomics (SRT) techniques have revolutionized the characterization of molecular profiles while preserving spatial and morphological context. However, most next-generation sequencing-based SRT techniques are limited to measuring gene expression in a confined array of spots, capturing only a fraction of the spatial domain. Typically, these spots encompass gene expression from a few to hundreds of cells, underscoring a critical need for more detailed, single-cell resolution SRT data to enhance our understanding of biological functions within the tissue context. Addressing this challenge, we introduce BayesDeep, a novel Bayesian hierarchical model that leverages cellular morphological data from histology images, commonly paired with SRT data, to reconstruct SRT data at the single-cell resolution. BayesDeep effectively model count data from SRT studies via a negative binomial regression model. This model incorporates explanatory variables such as cell types and nuclei-shape information for each cell extracted from the paired histology image. A feature selection scheme is integrated to examine the association between the morphological and molecular profiles, thereby improving the model robustness. We applied BayesDeep to two real SRT datasets, successfully demonstrating its capability to reconstruct SRT data at the single-cell resolution. This advancement not only yields new biological insights but also significantly enhances various downstream analyses, such as pseudotime and cell-cell communication.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.02.569722">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.02.569722" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.02.569722">
        <p class="paperTitle">Biolearn, an open-source library for biomarkers of aging</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.02.569722" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.02.569722" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ying, K.; Paulson, S.; Perez-Guevara, M.; Emamifar, M.; Casas Martinez, M.; Kwon, D.; Poganik, J. R.; Moqri, M.; Gladyshev, V. N.</p>
        <p class="info">Score: 18.2, Published: 2023-12-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.02.569722' target='https://doi.org/10.1101/2023.12.02.569722'> 10.1101/2023.12.02.569722</a></p>
        <p class="abstract">Identifying and validating biomarkers of aging is pivotal for understanding the aging process and testing longevity interventions. Despite the development of numerous aging biomarkers, their clinical validation remains elusive, largely due to the lack of cross-population validation, which is hampered by disparate biomarker designs and inconsistencies in dataset structures. To bridge this gap, we introduce Biolearn, an innovative open-source library dedicated to the implementation and application of aging biomarkers. Biolearn facilitates (1) harmonization of existing aging biomarkers, while presenting a structured framework for novel biomarkers in standardized formats; (2) unification of public datasets, ensuring coherent structuring and formatting, thus simplifying cross-population validation studies; and (3) provision of computational methodologies to assess any harmonized biomarker against unified datasets. By furnishing a community-driven platform, Biolearn significantly augments the development, assessment, and validation trajectories of aging biomarkers, paving the way toward more rigorous clinical validation and, ultimately, application in clinical trials targeting healthy longevity. The Biolearn package is open-source and freely available at https://Bio-Learn.github.io/</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570655">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570655" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570655">
        <p class="paperTitle">CytoSimplex: Visualizing Single-cell Fates and Transitions on a Simplex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570655" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570655" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, J.; Wang, Y.; Li, C.; Gu, Y.; Ono, N.; Welch, J. D.</p>
        <p class="info">Score: 15.5, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570655' target='https://doi.org/10.1101/2023.12.07.570655'> 10.1101/2023.12.07.570655</a></p>
        <p class="abstract">SummaryCells differentiate to their final fates along unique trajectories, often involving multi-potent progenitors that can produce multiple terminally differentiated cell types. Recent developments in single-cell transcriptomic and epigenomic measurement provide tremendous opportunities for mapping these trajectories. The visualization of single-cell data often relies on dimension reduction methods such as UMAP to simplify high-dimensional single-cell data down into an understandable two-dimensional (2D) form. However, these visualization methods can be misleading and often do not effectively represent the direction of cell differentiation. To address these limitations, we developed a new approach that places each cell from a single-cell dataset within a simplex whose vertices correspond to terminally differentiated cell types. Our approach can quantify and visualize current cell fate commitment and future cell potential. We developed CytoSimplex, a standalone open-source package implemented in R and Python that provides simple and intuitive visualizations of cell differentiation in 2D ternary and three-dimensional (3D) quaternary plots. We believe that CytoSimplex can help researchers gain a better understanding of cell type transitions in specific tissues and characterize developmental processes.

Availability and implementationThe R version of CytoSimplex is available on Github at https://github.com/welch-lab/CytoSimplex. The Python version of CytoSimplex is available on Github at https://github.com/welch-lab/pyCytoSimplex.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.06.570508">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.06.570508" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.06.570508">
        <p class="paperTitle">scDiffEq: drift-diffusion modeling of single-cell dynamics with neural stochastic differential equations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.06.570508" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.06.570508" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vinyard, M. E.; Rasmussen, A.; Li, R.; Getz, G.; Pinello, L.</p>
        <p class="info">Score: 12.6, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.06.570508' target='https://doi.org/10.1101/2023.12.06.570508'> 10.1101/2023.12.06.570508</a></p>
        <p class="abstract">Single-cell sequencing measurements facilitate the reconstruction of dynamic biology by capturing snapshots of the molecular profiles of individual cells. Cell fate decisions in development and disease are orchestrated through an intricate balance of deterministic and stochastic regulatory events. Drift-diffusion equations are effective in modeling single-cell dynamics from high-dimensional single-cell measurements. While existing solutions describe the deterministic dynamics associated with the drift term of these equations at the level of cell state, the diffusion is modeled as a constant across cell states. To fully understand the dynamic regulatory logic in development and disease, models explicitly attuned to the balance between deterministic and stochastic biology are required. Addressing these limitations, we introduce scDiffEq, a generative framework for learning neural stochastic differential equations that approximate the deterministic and stochastic dynamics in biology. Using lineage-traced single-cell data, we demonstrate that scDiffEq offers improved reconstruction of held-out cell states and prediction of cell fate from multipotent progenitors during hematopoiesis. By imparting in silico perturbations to multipotent progenitor cells, we find that scDiffEq accurately recapitulates the dynamics of CRISPR-perturbed hematopoiesis. Using scDiffEq, we simulate high-resolution developmental cell trajectories, modeling their drift and diffusion, enabling us to study their time-dependent gene-level dynamics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570531">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570531" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570531">
        <p class="paperTitle">DisDock: A Deep Learning Method for Metal Ion-Protein Redocking</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570531" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570531" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lin, M.; Li, K.; Zhang, Y.; Pan, F.; Wu, W.; Zhang, J.</p>
        <p class="info">Score: 10.8, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570531' target='https://doi.org/10.1101/2023.12.07.570531'> 10.1101/2023.12.07.570531</a></p>
        <p class="abstract">The structures of metalloproteins are essential for comprehending their functions and interactions. The breakthrough of AlphaFold has made it possible to predict protein structures with experimental accuracy. However, the type of metal ion that a metalloprotein binds and the binding structure are still not readily available, even with the predicted protein structure. In this study, we present DisDock, a physics-driven deep learning method for predicting protein-metal docking. DisDock takes distogram of randomly initialized protein-ligand configuration as input and outputs the distogram of the predicted binding complex. It combines the U-net architecture with self-attention modules to enhance model performance. Taking inspiration from the physical principle that atoms in closer proximity display a stronger mutual attraction, this predictor capitalizes on geometric information to uncover latent characteristics indicative of atom interactions. To train our model, we employ a high-quality metalloprotein dataset sourced from the Mother of All Databases (MOAD). Experimental results demonstrate that our approach outperforms other existing methods in prediction accuracy for various types of metal ions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.11.571121">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.11.571121" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.11.571121">
        <p class="paperTitle">Prediction of DNA i-Motifs Via Machine Learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.11.571121" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.11.571121" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yang, B.; Guneri, D.; Yu, H.; Wright, E.; Chen, W.; Waller, Z. A. E.; DING, Y.</p>
        <p class="info">Score: 9.5, Published: 2023-12-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.11.571121' target='https://doi.org/10.1101/2023.12.11.571121'> 10.1101/2023.12.11.571121</a></p>
        <p class="abstract">i-Motifs (iMs), are secondary structures formed in cytosine-rich DNA sequences and are involved in multiple functions in the genome. Although putative iM forming sequences are widely distributed in the human genome, the folding status and strength of putative iMs vary dramatically. Much previous research on iM has focused on assessing the iM folding properties using biophysical experiments. However, there are no dedicated computational tools for predicting the folding status and strength of iM structures. Here, we introduce a machine learning pipeline, iM-Seeker, to predict both folding status and structural stability of DNA iMs. The programme iM-Seeker incorporates a Balanced Random Forest classifier trained on genome-wide iMab antibody-based CUT&amp;Tag sequencing data to predict the folding status and an Extreme Gradient Boosting regressor to estimate the folding strength according to both literature biophysical data and our in-house biophysical experiments. iM-Seeker predicts DNA iM folding status with a classification accuracy of 81% and estimates the folding strength with coefficient of determination (R2) of 0.642 on the test set. Model interpretation confirms that the nucleotide composition of the C-rich sequence significantly affects iM stability, with a positive correlation with sequences containing cytosine and thymine and a negative correlation with guanine and adenine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.570120">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.570120" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.570120">
        <p class="paperTitle">GeneExt: a gene model extension tool for enhanced single-cell RNA-seq analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.570120" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.570120" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zolotarov, G.; Grau-Bove, X.; Sebe-Pedros, A.</p>
        <p class="info">Score: 9.4, Published: 2023-12-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.570120' target='https://doi.org/10.1101/2023.12.05.570120'> 10.1101/2023.12.05.570120</a></p>
        <p class="abstract">Incomplete gene models can negatively impact single-cell gene expression quantification. This is particularly true in non-model species where often 3 ends are inaccurately annotated, while most single-cell transcriptomics methods only capturing the 3 transcript region. As a result, many genes are incorrectly quantified or not detected at all. Here, we introduce a tool that leverages 3-end single-cell transcriptomics data to refine gene annotations. We exemplify its usage and its impact on the gene expression quantification of seven non-model organism single-cell atlases. By extending and homogenizing gene annotations, our tool will help improve biological interpretation and cross-species comparisons of cell type expression atlases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.05.27.542554">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.05.27.542554" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.05.27.542554">
        <p class="paperTitle">Identification and quantification of transposable element transcripts using Long-Read RNA-seq in Drosophila germline tissues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.05.27.542554" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.05.27.542554" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rebollo, R.; Gerenton, P.; Cumunel, E.; Mary, A.; Sabot, F.; Burlet, N.; Gillet, B.; Hughes, S.; Siqueira Oliveira, D.; Goubert, C.; Fablet, M.; Vieira, C.; Lacroix, V.</p>
        <p class="info">Score: 15.9, Published: 2023-12-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.05.27.542554' target='https://doi.org/10.1101/2023.05.27.542554'> 10.1101/2023.05.27.542554</a></p>
        <p class="abstract">Transposable elements (TEs) are repeated DNA sequences potentially able to move throughout the genome. In addition to their inherent mutagenic effects, TEs can disrupt nearby genes by donating their intrinsic regulatory sequences, for instance, promoting the ectopic expression of a cellular gene. TE transcription is therefore not only necessary for TE transposition per se but can also be associated with TE-gene fusion transcripts, and in some cases, be the product of pervasive transcription. Hence, correctly determining the transcription state of a TE copy is essential to apprehend the impact of the TE in the host genome. Methods to identify and quantify TE transcription have mostly relied on short RNA-seq reads to estimate TE expression at the family level while using specific algorithms to discriminate copy-specific transcription. However, assigning short reads to their correct genomic location, and genomic feature is not trivial. Here we retrieved full-length cDNA (TeloPrime, Lexogen) of Drosophila melanogaster gonads and sequenced them using Oxford Nanopore Technologies. We show that long-read RNA-seq can be used to identify and quantify transcribed TEs at the copy level. In particular, TE insertions overlapping annotated genes are better estimated using long reads than short reads. Nevertheless, long TE transcripts (&gt; 4.5 kb) are not well captured. Most expressed TE insertions correspond to copies that have lost their ability to transpose, and within a family, only a few copies are indeed expressed. Long-read sequencing also allowed the identification of spliced transcripts for around 105 TE copies. Overall, this first comparison of TEs between testes and ovaries uncovers differences in their transcriptional landscape, at the subclass and insertion level.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570524">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570524" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570524">
        <p class="paperTitle">DeconV: Probabilistic Cell Type Deconvolution from Bulk RNA-sequencing Data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570524" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570524" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gynter, A.; Meistermann, D.; Lahdesmaki, H.; Kilpinen, H.</p>
        <p class="info">Score: 7.0, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570524' target='https://doi.org/10.1101/2023.12.07.570524'> 10.1101/2023.12.07.570524</a></p>
        <p class="abstract">Bulk RNA-Seq remains a widely adopted technique to profile gene expression, primarily due to the persistent challenges associated with achieving single-cell resolution. However, a key challenge is accurately estimating the proportions of different cell types within these bulk samples. To address this issue, we introduce DeconV, a probabilistic framework for cell-type deconvolution that uses scRNA-Seq data as a reference. This approach aims to mitigate some of the limitations in existing methods by incorporating statistical frameworks developed for scRNA-Seq, thereby simplifying issues related to reference preprocessing such as normalization and marker gene selection. We benchmarked DeconV against established methods, including MuSiC, CIBERSORTx, and Scaden. Our results show that DeconV performs comparably in terms of accuracy to the best-performing method, Scaden, but provides additional interpretability by offering confidence intervals for its predictions. Furthermore, the modular design of DeconV allows for the investigation of discrepancies between bulk-sequenced samples and artificially generated pseudo-bulk samples.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
