<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="ColabDock: inverting AlphaFold structure prediction model for protein-protein docking with experimental restraints
Authors: Feng, S.; Chen, Z.; Zhang, C.; Xie, Y.; Ovchinnikov, S.; Gao, Y. Q.; Liu, S.
Score: 46.6, Published: 2023-07-04 DOI: 10.1101/2023.07.04.547599
Prediction of protein complex structures and interfaces potentially has wide applications and can benefit the study of biological mechanisms involving protein-protein interactions. However, the surface prediction accuracy of traditional docking methods and AlphaFold-Multimer is limited. Here we present ColabDock, a framework that makes use of ColabDesign, but reimplements it for the purpose of restrained complex conformation prediction.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="ColabDock: inverting AlphaFold structure prediction model for protein-protein docking with experimental restraints
Authors: Feng, S.; Chen, Z.; Zhang, C.; Xie, Y.; Ovchinnikov, S.; Gao, Y. Q.; Liu, S.
Score: 46.6, Published: 2023-07-04 DOI: 10.1101/2023.07.04.547599
Prediction of protein complex structures and interfaces potentially has wide applications and can benefit the study of biological mechanisms involving protein-protein interactions. However, the surface prediction accuracy of traditional docking methods and AlphaFold-Multimer is limited. Here we present ColabDock, a framework that makes use of ColabDesign, but reimplements it for the purpose of restrained complex conformation prediction." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-09T10:40:17+00:00" />
<meta property="article:modified_time" content="2023-07-09T10:40:17+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="ColabDock: inverting AlphaFold structure prediction model for protein-protein docking with experimental restraints
Authors: Feng, S.; Chen, Z.; Zhang, C.; Xie, Y.; Ovchinnikov, S.; Gao, Y. Q.; Liu, S.
Score: 46.6, Published: 2023-07-04 DOI: 10.1101/2023.07.04.547599
Prediction of protein complex structures and interfaces potentially has wide applications and can benefit the study of biological mechanisms involving protein-protein interactions. However, the surface prediction accuracy of traditional docking methods and AlphaFold-Multimer is limited. Here we present ColabDock, a framework that makes use of ColabDesign, but reimplements it for the purpose of restrained complex conformation prediction."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "ColabDock: inverting AlphaFold structure prediction model for protein-protein docking with experimental restraints\nAuthors: Feng, S.; Chen, Z.; Zhang, C.; Xie, Y.; Ovchinnikov, S.; Gao, Y. Q.; Liu, S.\nScore: 46.6, Published: 2023-07-04 DOI: 10.1101/2023.07.04.547599\nPrediction of protein complex structures and interfaces potentially has wide applications and can benefit the study of biological mechanisms involving protein-protein interactions. However, the surface prediction accuracy of traditional docking methods and AlphaFold-Multimer is limited. Here we present ColabDock, a framework that makes use of ColabDesign, but reimplements it for the purpose of restrained complex conformation prediction.",
  "keywords": [
    
  ],
  "articleBody": " ColabDock: inverting AlphaFold structure prediction model for protein-protein docking with experimental restraints\nAuthors: Feng, S.; Chen, Z.; Zhang, C.; Xie, Y.; Ovchinnikov, S.; Gao, Y. Q.; Liu, S.\nScore: 46.6, Published: 2023-07-04 DOI: 10.1101/2023.07.04.547599\nPrediction of protein complex structures and interfaces potentially has wide applications and can benefit the study of biological mechanisms involving protein-protein interactions. However, the surface prediction accuracy of traditional docking methods and AlphaFold-Multimer is limited. Here we present ColabDock, a framework that makes use of ColabDesign, but reimplements it for the purpose of restrained complex conformation prediction. With a generation-prediction architecture and trained ranking model, ColabDock outperforms HADDOCK and ClusPro not only in complex structure predictions with simulated residue and surface restraints, but also in those assisted by NMR chemical shift perturbation as well as covalent labeling. It further assists antibody-antigen interface prediction with emulated interface scan restraints, which could be obtained by experiments such as Deep Mutation Scan. ColabDock provides a general approach to integrate sparse interface restraints of different experimental forms and sources into one optimization framework.\nScalable, accessible, and reproducible reference genome assembly and evaluation in Galaxy\nAuthors: Lariviere, D.; Abueg, L.; Brajuka, N.; Gallardo-Alba, C.; Gruning, B.; Ko, B. J.; Ostrovsky, A.; Palmada-Flores, M.; Pickett, B. D.; Rabbani, K.; Balacco, J. R.; Chaisson, M.; Cheng, H.; Collins, J.; Denisova, A.; Fedrigo, O.; Gallo, G. R.; Giani, A. M.; Gooder, G. M.; Jain, N.; Johnson, C.; Kim, H.; Lee, C.; Marques-Bonet, T.; O'Toole, B.; Rhie, A.; Secomandi, S.; Sozzoni, M.; Tilley, T.; Uliano-Silva, M.; van den Beek, M.; Waterhouse, R. M.; Phillippy, A. M.; Jarvis, E. D.; Schatz, M.; Nekrutenko, A.; Formenti, G.\nScore: 43.9, Published: 2023-06-30 DOI: 10.1101/2023.06.28.546576\nImprovements in genome sequencing and assembly are enabling high-quality reference genomes for all species. However, the assembly process is still laborious, computationally and technically demanding, lacks standards for reproducibility, and is not readily scalable. Here we present the latest Vertebrate Genomes Project assembly pipeline and demonstrate that it delivers high-quality reference genomes at scale across a set of vertebrate species arising over the last [~]500 million years. The pipeline is versatile and combines PacBio HiFi long-reads and Hi-C-based haplotype phasing in a new graph-based paradigm. Standardized quality control is performed automatically to troubleshoot assembly issues and assess biological complexities. We make the pipeline freely accessible through Galaxy, accommodating researchers even without local computational resources and enhanced reproducibility by democratizing the training and assembly process. We demonstrate the flexibility and reliability of the pipeline by assembling reference genomes for 51 vertebrate species from major taxonomic groups (fish, amphibians, reptiles, birds, and mammals).\nHostile: accurate host decontamination of microbial sequences\nAuthors: Constantinides, B.; Crook, D. W.\nScore: 34.2, Published: 2023-07-05 DOI: 10.1101/2023.07.04.547735\nMicrobial sequences generated from clinical samples are often contaminated with human host sequences that must be removed for ethical and legal reasons. Care must be taken to excise host sequences without inadvertently removing target microbial sequences to the detriment of downstream analyses such as variant calling and de novo assembly. To facilitate accurate host decontamination of both short and long sequencing reads, we developed Hostile, a tool capable of rapid host read removal using laptop specification hardware. We demonstrate that our approach removes at least 99.868% of real human reads and retains at least 99.997% of simulated bacterial reads. Use of a masked reference genome further increases bacterial read retention (\u003e=99.997%) with negligible (\u003c0.001%) reduction in human read removal performance. Compared with an existing tool, Hostile removed up to 11x more human reads and up to 11x fewer microbial reads while taking less time for typical workloads. Hostile is implemented as an MIT licensed Python package available at https://github.com/bede/hostile\nImproved protein complex prediction with AlphaFold-multimer by denoising the MSA profile\nAuthors: Bryant, P.; Noe, F.\nScore: 28.9, Published: 2023-07-04 DOI: 10.1101/2023.07.04.547638\nStructure prediction of protein complexes has improved significantly with AlphaFold2 and AlphaFold-multimer (AFM), but only 60% of dimers are accurately predicted. A way to improve the predictions is to inject noise to generate more diverse predictions. However, thousands of predictions are needed to obtain a few that are accurate in difficult cases. Here, we learn a bias to the MSA representation that improves the predictions by performing gradient descent through the AFM network. We effectively denoise the MSA profile, similar to how a blurry image would be sharpened. We demonstrate the performance on seven difficult targets from CASP15 and increase the average MMscore to 0.76 compared to 0.63 with AFM. We evaluate the procedure on 334 protein complexes where AFM fails and demonstrate an increased success rate (MMscore\u003e0.75) of 8% on these hard targets. Our protocol, AFProfile, provides a way to direct predictions towards a defined target function guided by the MSA. We expect gradient descent over the MSA to be useful for different tasks, such as generating alternative conformations. AFProfile is freely available at: https://github.com/patrickbryant1/AFProfile\nA pre-trained large language model for translating single-cell transcriptome to proteome\nAuthors: Liu, L.; Li, W.; Wong, K.-c.; Yang, F.; Yao, J.\nScore: 25.7, Published: 2023-07-04 DOI: 10.1101/2023.07.04.547619\nDespite the recent advancements in single-cell proteome technology, it still has limitation on throughput, proteome depth and batch effect, and the cost is still high. Inspired by the translation procedure of both natural language processing (NLP) and the genetic central dogma, we propose a pre-trained large language model named scTranslator (single-cell translator), which is align-free and generates absent single-cell proteome by inferring from the transcriptome. scTranslator achieves a general knowledge of RNA-protein interactions by being pre-trained on substantial amounts of bulk and single-cell data. Systematic benchmarking confirms the accuracy, stability, and flexibility of scTranslator across various quantification techniques, cell types, and conditions. Furthermore, we apply scTranslator to various downstream analyses and applications, including interaction inference, gene pseudo-knockout, cell clustering, and cell origin recognition of pan-cancer data.\nFlexible Protein-Protein Docking with a Multi-Track Iterative Transformer\nAuthors: Chu, L.-S.; Ruffolo, J. A.; Harmalkar, A.; Gray, J. J.\nScore: 24.8, Published: 2023-07-01 DOI: 10.1101/2023.06.29.547134\nConventional protein-protein docking algorithms usually rely on heavy candidate sampling and re-ranking, but these steps are time-consuming and hinder applications that require high-throughput complex structure prediction, e.g., structure-based virtual screening. Existing deep learning methods for protein-protein docking, despite being much faster, suffer from low docking success rates. In addition, they simplify the problem to assume no conformational changes within any protein upon binding (rigid docking). This assumption precludes applications when binding-induced conformational changes play a role, such as allosteric inhibition or docking from uncertain unbound model structures. To address these limitations, we present GeoDock, a multi-track iterative transformer network to predict a docked structure from separate docking partners. Unlike deep learning models for protein structure prediction that input multiple sequence alignments (MSAs), GeoDock inputs just the sequences and structures of the docking partners, which suits the tasks when the individual structures are given. GeoDock is flexible at the protein residue level, allowing the prediction of conformational changes upon binding. For a benchmark set of rigid targets, GeoDock obtains a 41% success rate, outperforming all the other tested methods. For a more challenging benchmark set of flexible targets, GeoDock achieves a similar number of top-model successes as the traditional method ClusPro [1], but fewer than ReplicaDock2 [2]. GeoDock attains an average inference speed of under one second on a single GPU, enabling its application in large-scale structure screening. Although binding-induced conformational changes are still a challenge owing to limited training and evaluation data, our architecture sets up the foundation to capture this backbone flexibility. Code and a demonstration Jupyter notebook are available at https://github.com/Graylab/GeoDock.\nModelling protein complexes with crosslinking mass spectrometry and deep learning\nAuthors: Stahl, K.; Brock, O.; Rappsilber, J.\nScore: 34.3, Published: 2023-06-28 DOI: 10.1101/2023.06.07.544059\nScarcity of structural and evolutionary information on protein complexes poses a challenge to deep learning-based structure modelling. We integrated experimental distance restraints obtained by crosslinking mass spectrometry (MS) into AlphaFold-Multimer, by extending AlphaLink to protein complexes. Integrating crosslinking MS data substantially improves modelling performance on challenging targets, by helping to identify interfaces, focusing sampling, and improving model selection. This extends to single crosslinks from whole-cell crosslinking MS, suggesting the possibility of whole-cell structural investigations driven by experimental data.\nJuggling offsets unlocks RNA-seq tools for fast scalable differential usage, aberrant splicing and expression analyses.\nAuthors: Segers, A.; Gilis, J.; Van Heetvelde, M.; De Baere, E.; Clement, L.\nScore: 17.5, Published: 2023-07-01 DOI: 10.1101/2023.06.29.547014\nRNA-sequencing (RNA-seq) is increasingly used to diagnose patients with rare diseases by prioritising genes with aberrant expression and/or splicing. State-of-the-art methods for detecting aberrant expression and splicing, however, are extremely slow. The latter, also discard much information because they only use junction reads to infer aberrant splicing. In this contribution, we show that replacing the offset for library size unlocks conventional bulk RNA-seq workflows for fast and scalable differential usage, aberrant splicing and expression analyses. Our method, saseR, is several orders of magnitude faster than the state-of-the-art methods and dramatically outperforms these in terms of sensitivity and specificity for aberrant splicing, while being on par with these inferring differential usage and aberrant expression. Finally, our framework is also very flexible and can be used for all applications that involve the analysis of proportions of short- or long RNA-seq read counts.\nDeep dynamical modelling of developmental trajectories with temporal transcriptomics\nAuthors: Maizels, R. J.; Snell, D. M.; Briscoe, J.\nScore: 12.1, Published: 2023-07-07 DOI: 10.1101/2023.07.06.547989\nDevelopmental cell fate decisions are dynamic processes driven by the complex behaviour of gene regulatory networks. A challenge in studying these processes using single-cell genomics is that the data provides only a static snapshot with no detail of dynamics. Metabolic labelling and splicing can provide time-resolved information, but current methods have limitations. Here, we present experimental and computational methods that overcome these limitations to allow dynamical modelling of gene expression from single-cell data. We developed sci-FATE2, an optimised metabolic labelling method that substantially increases data quality, and profiled approximately 45,000 embryonic stem cells differentiating into multiple neural tube identities. To recover dynamics, we developed velvet, a deep learning framework that extends beyond instantaneous velocity estimation by modelling gene expression dynamics through a neural stochastic differential equation system within a variational autoencoder. Velvet outperforms current velocity tools across quantitative benchmarks, and predicts trajectory distributions that accurately recapitulate underlying dataset distributions while conserving known biology. Velvet trajectory distributions capture dynamical aspects such as decision boundaries between alternative fates and correlative gene regulatory structure. Using velvet to provide a dynamical description of in vitro neural patterning, we highlight a process of sequential decision making and fate-specific patterns of developmental signalling. Together, these experimental and computational methods recast single-cell analyses from descriptions of observed data distributions to models of the dynamics that generated them, providing a new framework for investigating developmental gene regulation and cell fate decisions.\nDrugGPT: A GPT-based Strategy for Designing Potential Ligands Targeting Specific Proteins\nAuthors: Li, Y.; Gao, C.; Song, X.; Wang, X.; Xu, Y.; Han, S.\nScore: 35.2, Published: 2023-06-30 DOI: 10.1101/2023.06.29.543848\nDrugGPT presents a ligand design strategy based on the autoregressive model, GPT, focusing on chemical space exploration and the discovery of ligands for specific proteins. Deep learning language models have shown significant potential in various domains including protein design and biomedical text analysis, providing strong support for the proposition of DrugGPT. In this study, we employ the DrugGPT model to learn a substantial amount of protein-ligand binding data, aiming to discover novel molecules that can bind with specific proteins. This strategy not only significantly improves the efficiency of ligand design but also offers a swift and effective avenue for the drug development process, bringing new possibilities to the pharmaceutical domain. In our research, we particularly optimized and trained the GPT-2 model to better adapt to the requirements of drug design. Given the characteristics of proteins and ligands, we redesigned the tokenizer using the BPE algorithm, abandoned the original tokenizer, and trained the GPT-2 model from scratch. This improvement enables DrugGPT to more accurately capture and understand the structural information and chemical rules of drug molecules. It also enhances its comprehension of binding information between proteins and ligands, thereby generating potentially active drug candidate molecules. Theoretically, DrugGPT has significant advantages. During the model training process, DrugGPT aims to maximize the conditional probability and employs the back-propagation algorithm for training, making the training process more stable and avoiding the Mode Collapse problem that may occur in Generative Adversarial Networks in drug design. Furthermore, the design philosophy of DrugGPT endows it with strong generalization capabilities, giving it the potential to adapt to different tasks. In conclusion, DrugGPT provides a forward-thinking and practical new approach to ligand design. By optimizing the tokenizer and retraining the GPT-2 model, the ligand design process becomes more direct and efficient. This not only reflects the theoretical advantages of DrugGPT but also reveals its potential applications in the drug development process, thereby opening new perspectives and possibilities in the pharmaceutical field.\n",
  "wordCount" : "2129",
  "inLanguage": "en",
  "datePublished": "2023-07-09T10:40:17Z",
  "dateModified": "2023-07-09T10:40:17Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on July 9, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547599">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547599" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547599">
        <p class="paperTitle">ColabDock: inverting AlphaFold structure prediction model for protein-protein docking with experimental restraints</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547599" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547599" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Feng, S.; Chen, Z.; Zhang, C.; Xie, Y.; Ovchinnikov, S.; Gao, Y. Q.; Liu, S.</p>
        <p class="info">Score: 46.6, Published: 2023-07-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547599' target='https://doi.org/10.1101/2023.07.04.547599'> 10.1101/2023.07.04.547599</a></p>
        <p class="abstract">Prediction of protein complex structures and interfaces potentially has wide applications and can benefit the study of biological mechanisms involving protein-protein interactions. However, the surface prediction accuracy of traditional docking methods and AlphaFold-Multimer is limited. Here we present ColabDock, a framework that makes use of ColabDesign, but reimplements it for the purpose of restrained complex conformation prediction. With a generation-prediction architecture and trained ranking model, ColabDock outperforms HADDOCK and ClusPro not only in complex structure predictions with simulated residue and surface restraints, but also in those assisted by NMR chemical shift perturbation as well as covalent labeling. It further assists antibody-antigen interface prediction with emulated interface scan restraints, which could be obtained by experiments such as Deep Mutation Scan. ColabDock provides a general approach to integrate sparse interface restraints of different experimental forms and sources into one optimization framework.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546576">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546576" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546576">
        <p class="paperTitle">Scalable, accessible, and reproducible reference genome assembly and evaluation in Galaxy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546576" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546576" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lariviere, D.; Abueg, L.; Brajuka, N.; Gallardo-Alba, C.; Gruning, B.; Ko, B. J.; Ostrovsky, A.; Palmada-Flores, M.; Pickett, B. D.; Rabbani, K.; Balacco, J. R.; Chaisson, M.; Cheng, H.; Collins, J.; Denisova, A.; Fedrigo, O.; Gallo, G. R.; Giani, A. M.; Gooder, G. M.; Jain, N.; Johnson, C.; Kim, H.; Lee, C.; Marques-Bonet, T.; O&#39;Toole, B.; Rhie, A.; Secomandi, S.; Sozzoni, M.; Tilley, T.; Uliano-Silva, M.; van den Beek, M.; Waterhouse, R. M.; Phillippy, A. M.; Jarvis, E. D.; Schatz, M.; Nekrutenko, A.; Formenti, G.</p>
        <p class="info">Score: 43.9, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546576' target='https://doi.org/10.1101/2023.06.28.546576'> 10.1101/2023.06.28.546576</a></p>
        <p class="abstract">Improvements in genome sequencing and assembly are enabling high-quality reference genomes for all species. However, the assembly process is still laborious, computationally and technically demanding, lacks standards for reproducibility, and is not readily scalable. Here we present the latest Vertebrate Genomes Project assembly pipeline and demonstrate that it delivers high-quality reference genomes at scale across a set of vertebrate species arising over the last [~]500 million years. The pipeline is versatile and combines PacBio HiFi long-reads and Hi-C-based haplotype phasing in a new graph-based paradigm. Standardized quality control is performed automatically to troubleshoot assembly issues and assess biological complexities. We make the pipeline freely accessible through Galaxy, accommodating researchers even without local computational resources and enhanced reproducibility by democratizing the training and assembly process. We demonstrate the flexibility and reliability of the pipeline by assembling reference genomes for 51 vertebrate species from major taxonomic groups (fish, amphibians, reptiles, birds, and mammals).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547735">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547735" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547735">
        <p class="paperTitle">Hostile: accurate host decontamination of microbial sequences</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547735" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547735" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Constantinides, B.; Crook, D. W.</p>
        <p class="info">Score: 34.2, Published: 2023-07-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547735' target='https://doi.org/10.1101/2023.07.04.547735'> 10.1101/2023.07.04.547735</a></p>
        <p class="abstract">Microbial sequences generated from clinical samples are often contaminated with human host sequences that must be removed for ethical and legal reasons. Care must be taken to excise host sequences without inadvertently removing target microbial sequences to the detriment of downstream analyses such as variant calling and de novo assembly. To facilitate accurate host decontamination of both short and long sequencing reads, we developed Hostile, a tool capable of rapid host read removal using laptop specification hardware. We demonstrate that our approach removes at least 99.868% of real human reads and retains at least 99.997% of simulated bacterial reads. Use of a masked reference genome further increases bacterial read retention (&gt;=99.997%) with negligible (&lt;0.001%) reduction in human read removal performance. Compared with an existing tool, Hostile removed up to 11x more human reads and up to 11x fewer microbial reads while taking less time for typical workloads. Hostile is implemented as an MIT licensed Python package available at https://github.com/bede/hostile</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547638">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547638" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547638">
        <p class="paperTitle">Improved protein complex prediction with AlphaFold-multimer by denoising the MSA profile</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547638" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547638" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bryant, P.; Noe, F.</p>
        <p class="info">Score: 28.9, Published: 2023-07-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547638' target='https://doi.org/10.1101/2023.07.04.547638'> 10.1101/2023.07.04.547638</a></p>
        <p class="abstract">Structure prediction of protein complexes has improved significantly with AlphaFold2 and AlphaFold-multimer (AFM), but only 60% of dimers are accurately predicted. A way to improve the predictions is to inject noise to generate more diverse predictions. However, thousands of predictions are needed to obtain a few that are accurate in difficult cases. Here, we learn a bias to the MSA representation that improves the predictions by performing gradient descent through the AFM network. We effectively denoise the MSA profile, similar to how a blurry image would be sharpened. We demonstrate the performance on seven difficult targets from CASP15 and increase the average MMscore to 0.76 compared to 0.63 with AFM. We evaluate the procedure on 334 protein complexes where AFM fails and demonstrate an increased success rate (MMscore&gt;0.75) of 8% on these hard targets. Our protocol, AFProfile, provides a way to direct predictions towards a defined target function guided by the MSA. We expect gradient descent over the MSA to be useful for different tasks, such as generating alternative conformations. AFProfile is freely available at: https://github.com/patrickbryant1/AFProfile</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547619">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547619" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547619">
        <p class="paperTitle">A pre-trained large language model for translating single-cell transcriptome to proteome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547619" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547619" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, L.; Li, W.; Wong, K.-c.; Yang, F.; Yao, J.</p>
        <p class="info">Score: 25.7, Published: 2023-07-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547619' target='https://doi.org/10.1101/2023.07.04.547619'> 10.1101/2023.07.04.547619</a></p>
        <p class="abstract">Despite the recent advancements in single-cell proteome technology, it still has limitation on throughput, proteome depth and batch effect, and the cost is still high. Inspired by the translation procedure of both natural language processing (NLP) and the genetic central dogma, we propose a pre-trained large language model named scTranslator (single-cell translator), which is align-free and generates absent single-cell proteome by inferring from the transcriptome. scTranslator achieves a general knowledge of RNA-protein interactions by being pre-trained on substantial amounts of bulk and single-cell data. Systematic benchmarking confirms the accuracy, stability, and flexibility of scTranslator across various quantification techniques, cell types, and conditions. Furthermore, we apply scTranslator to various downstream analyses and applications, including interaction inference, gene pseudo-knockout, cell clustering, and cell origin recognition of pan-cancer data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547134">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547134" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547134">
        <p class="paperTitle">Flexible Protein-Protein Docking with a Multi-Track Iterative Transformer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547134" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547134" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chu, L.-S.; Ruffolo, J. A.; Harmalkar, A.; Gray, J. J.</p>
        <p class="info">Score: 24.8, Published: 2023-07-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547134' target='https://doi.org/10.1101/2023.06.29.547134'> 10.1101/2023.06.29.547134</a></p>
        <p class="abstract">Conventional protein-protein docking algorithms usually rely on heavy candidate sampling and re-ranking, but these steps are time-consuming and hinder applications that require high-throughput complex structure prediction, e.g., structure-based virtual screening. Existing deep learning methods for protein-protein docking, despite being much faster, suffer from low docking success rates. In addition, they simplify the problem to assume no conformational changes within any protein upon binding (rigid docking). This assumption precludes applications when binding-induced conformational changes play a role, such as allosteric inhibition or docking from uncertain unbound model structures. To address these limitations, we present GeoDock, a multi-track iterative transformer network to predict a docked structure from separate docking partners. Unlike deep learning models for protein structure prediction that input multiple sequence alignments (MSAs), GeoDock inputs just the sequences and structures of the docking partners, which suits the tasks when the individual structures are given. GeoDock is flexible at the protein residue level, allowing the prediction of conformational changes upon binding. For a benchmark set of rigid targets, GeoDock obtains a 41% success rate, outperforming all the other tested methods. For a more challenging benchmark set of flexible targets, GeoDock achieves a similar number of top-model successes as the traditional method ClusPro [1], but fewer than ReplicaDock2 [2]. GeoDock attains an average inference speed of under one second on a single GPU, enabling its application in large-scale structure screening. Although binding-induced conformational changes are still a challenge owing to limited training and evaluation data, our architecture sets up the foundation to capture this backbone flexibility. Code and a demonstration Jupyter notebook are available at https://github.com/Graylab/GeoDock.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.07.544059">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.07.544059" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.07.544059">
        <p class="paperTitle">Modelling protein complexes with crosslinking mass spectrometry and deep learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.07.544059" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.07.544059" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stahl, K.; Brock, O.; Rappsilber, J.</p>
        <p class="info">Score: 34.3, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.07.544059' target='https://doi.org/10.1101/2023.06.07.544059'> 10.1101/2023.06.07.544059</a></p>
        <p class="abstract">Scarcity of structural and evolutionary information on protein complexes poses a challenge to deep learning-based structure modelling. We integrated experimental distance restraints obtained by crosslinking mass spectrometry (MS) into AlphaFold-Multimer, by extending AlphaLink to protein complexes. Integrating crosslinking MS data substantially improves modelling performance on challenging targets, by helping to identify interfaces, focusing sampling, and improving model selection. This extends to single crosslinks from whole-cell crosslinking MS, suggesting the possibility of whole-cell structural investigations driven by experimental data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547014">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547014" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547014">
        <p class="paperTitle">Juggling offsets unlocks RNA-seq tools for fast scalable differential usage, aberrant splicing and expression analyses.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547014" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547014" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Segers, A.; Gilis, J.; Van Heetvelde, M.; De Baere, E.; Clement, L.</p>
        <p class="info">Score: 17.5, Published: 2023-07-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547014' target='https://doi.org/10.1101/2023.06.29.547014'> 10.1101/2023.06.29.547014</a></p>
        <p class="abstract">RNA-sequencing (RNA-seq) is increasingly used to diagnose patients with rare diseases by prioritising genes with aberrant expression and/or splicing. State-of-the-art methods for detecting aberrant expression and splicing, however, are extremely slow. The latter, also discard much information because they only use junction reads to infer aberrant splicing. In this contribution, we show that replacing the offset for library size unlocks conventional bulk RNA-seq workflows for fast and scalable differential usage, aberrant splicing and expression analyses. Our method, saseR, is several orders of magnitude faster than the state-of-the-art methods and dramatically outperforms these in terms of sensitivity and specificity for aberrant splicing, while being on par with these inferring differential usage and aberrant expression. Finally, our framework is also very flexible and can be used for all applications that involve the analysis of proportions of short- or long RNA-seq read counts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.547989">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.547989" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.547989">
        <p class="paperTitle">Deep dynamical modelling of developmental trajectories with temporal transcriptomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.547989" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.547989" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Maizels, R. J.; Snell, D. M.; Briscoe, J.</p>
        <p class="info">Score: 12.1, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.547989' target='https://doi.org/10.1101/2023.07.06.547989'> 10.1101/2023.07.06.547989</a></p>
        <p class="abstract">Developmental cell fate decisions are dynamic processes driven by the complex behaviour of gene regulatory networks. A challenge in studying these processes using single-cell genomics is that the data provides only a static snapshot with no detail of dynamics. Metabolic labelling and splicing can provide time-resolved information, but current methods have limitations. Here, we present experimental and computational methods that overcome these limitations to allow dynamical modelling of gene expression from single-cell data. We developed sci-FATE2, an optimised metabolic labelling method that substantially increases data quality, and profiled approximately 45,000 embryonic stem cells differentiating into multiple neural tube identities. To recover dynamics, we developed velvet, a deep learning framework that extends beyond instantaneous velocity estimation by modelling gene expression dynamics through a neural stochastic differential equation system within a variational autoencoder. Velvet outperforms current velocity tools across quantitative benchmarks, and predicts trajectory distributions that accurately recapitulate underlying dataset distributions while conserving known biology. Velvet trajectory distributions capture dynamical aspects such as decision boundaries between alternative fates and correlative gene regulatory structure. Using velvet to provide a dynamical description of in vitro neural patterning, we highlight a process of sequential decision making and fate-specific patterns of developmental signalling. Together, these experimental and computational methods recast single-cell analyses from descriptions of observed data distributions to models of the dynamics that generated them, providing a new framework for investigating developmental gene regulation and cell fate decisions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.543848">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.543848" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.543848">
        <p class="paperTitle">DrugGPT: A GPT-based Strategy for Designing Potential Ligands Targeting Specific Proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.543848" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.543848" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, Y.; Gao, C.; Song, X.; Wang, X.; Xu, Y.; Han, S.</p>
        <p class="info">Score: 35.2, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.543848' target='https://doi.org/10.1101/2023.06.29.543848'> 10.1101/2023.06.29.543848</a></p>
        <p class="abstract">DrugGPT presents a ligand design strategy based on the autoregressive model, GPT, focusing on chemical space exploration and the discovery of ligands for specific proteins. Deep learning language models have shown significant potential in various domains including protein design and biomedical text analysis, providing strong support for the proposition of DrugGPT. In this study, we employ the DrugGPT model to learn a substantial amount of protein-ligand binding data, aiming to discover novel molecules that can bind with specific proteins. This strategy not only significantly improves the efficiency of ligand design but also offers a swift and effective avenue for the drug development process, bringing new possibilities to the pharmaceutical domain.

In our research, we particularly optimized and trained the GPT-2 model to better adapt to the requirements of drug design. Given the characteristics of proteins and ligands, we redesigned the tokenizer using the BPE algorithm, abandoned the original tokenizer, and trained the GPT-2 model from scratch. This improvement enables DrugGPT to more accurately capture and understand the structural information and chemical rules of drug molecules. It also enhances its comprehension of binding information between proteins and ligands, thereby generating potentially active drug candidate molecules.

Theoretically, DrugGPT has significant advantages. During the model training process, DrugGPT aims to maximize the conditional probability and employs the back-propagation algorithm for training, making the training process more stable and avoiding the Mode Collapse problem that may occur in Generative Adversarial Networks in drug design. Furthermore, the design philosophy of DrugGPT endows it with strong generalization capabilities, giving it the potential to adapt to different tasks.

In conclusion, DrugGPT provides a forward-thinking and practical new approach to ligand design. By optimizing the tokenizer and retraining the GPT-2 model, the ligand design process becomes more direct and efficient. This not only reflects the theoretical advantages of DrugGPT but also reveals its potential applications in the drug development process, thereby opening new perspectives and possibilities in the pharmaceutical field.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
