<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="When the pen is mightier than the sword: semi-automatic 2 and 3D image labelling
Authors: Hollandi, R.; Bauer, D.; Diosdi, A.; Schrettner, B.; Toth, T.; Hirling, D.; Hollandi, G.; Harmati, M.; Molnar, J.; Horvath, P.
Score: 38.4, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575658
Data is the driving engine of learning-based algorithms, the creation of which fundamentally determines the performance, accuracy, generalizability and quality of any model or method trained on it. When only skilled or trained personnel can create reliable annotations, assisted software solutions are desirable to reduce the time and effort the expert must spend on labelling.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="When the pen is mightier than the sword: semi-automatic 2 and 3D image labelling
Authors: Hollandi, R.; Bauer, D.; Diosdi, A.; Schrettner, B.; Toth, T.; Hirling, D.; Hollandi, G.; Harmati, M.; Molnar, J.; Horvath, P.
Score: 38.4, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575658
Data is the driving engine of learning-based algorithms, the creation of which fundamentally determines the performance, accuracy, generalizability and quality of any model or method trained on it. When only skilled or trained personnel can create reliable annotations, assisted software solutions are desirable to reduce the time and effort the expert must spend on labelling." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-24T10:40:04+00:00" />
<meta property="article:modified_time" content="2024-01-24T10:40:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="When the pen is mightier than the sword: semi-automatic 2 and 3D image labelling
Authors: Hollandi, R.; Bauer, D.; Diosdi, A.; Schrettner, B.; Toth, T.; Hirling, D.; Hollandi, G.; Harmati, M.; Molnar, J.; Horvath, P.
Score: 38.4, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575658
Data is the driving engine of learning-based algorithms, the creation of which fundamentally determines the performance, accuracy, generalizability and quality of any model or method trained on it. When only skilled or trained personnel can create reliable annotations, assisted software solutions are desirable to reduce the time and effort the expert must spend on labelling."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "When the pen is mightier than the sword: semi-automatic 2 and 3D image labelling\nAuthors: Hollandi, R.; Bauer, D.; Diosdi, A.; Schrettner, B.; Toth, T.; Hirling, D.; Hollandi, G.; Harmati, M.; Molnar, J.; Horvath, P.\nScore: 38.4, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575658\nData is the driving engine of learning-based algorithms, the creation of which fundamentally determines the performance, accuracy, generalizability and quality of any model or method trained on it. When only skilled or trained personnel can create reliable annotations, assisted software solutions are desirable to reduce the time and effort the expert must spend on labelling.",
  "keywords": [
    
  ],
  "articleBody": " When the pen is mightier than the sword: semi-automatic 2 and 3D image labelling\nAuthors: Hollandi, R.; Bauer, D.; Diosdi, A.; Schrettner, B.; Toth, T.; Hirling, D.; Hollandi, G.; Harmati, M.; Molnar, J.; Horvath, P.\nScore: 38.4, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575658\nData is the driving engine of learning-based algorithms, the creation of which fundamentally determines the performance, accuracy, generalizability and quality of any model or method trained on it. When only skilled or trained personnel can create reliable annotations, assisted software solutions are desirable to reduce the time and effort the expert must spend on labelling. Herein is proposed an automated annotation helper software package in napari that offers multiple methods to assist the annotator in creating object-based labels on 2D or 3D images.\nCellTracksColab--A platform for compiling, analyzing, and exploring tracking data\nAuthors: Jacquemet, G.\nScore: 68.9, Published: 2024-01-22 DOI: 10.1101/2023.10.20.563252\nIn life sciences, tracking objects from movies enables researchers to quantify the behavior of single particles, organelles, bacteria, cells, and even whole animals. While numerous tools now allow automated tracking from video, a significant challenge persists in compiling, analyzing, and exploring the large datasets generated by these approaches. Here, we introduce CellTracksColab, an online platform tailored to simplify the exploration and analysis of tracking data by leveraging the free, cloud-based computational resources of Google Colab. CellTracksColab facilitates the amalgamation and analysis of results across multiple fields of view, conditions, and repeats, ensuring a holistic dataset overview. CellTracksColab also harnesses the power of high-dimensional data reduction and clustering, enabling researchers to identify distinct behavioral patterns and trends without bias. Finally, CellTracksColab also includes specialized analysis modules enabling spatial analyses (clustering, proximity to specific regions of interest). CellTracksColab is available for the broader scientific community here.\nnf-core/airrflow: an adaptive immune receptor repertoire analysis workflow employing the Immcantation framework\nAuthors: Gabernet, G.; Marquez, S.; Bjornson, R.; Peltzer, A.; Meng, H.; Aron, E.; Lee, N. Y.; Jensen, C.; Ladd, D.; Hanssen, F.; Heumos, S.; nf-core community, ; Yaari, G.; Kowarik, M. C.; Nahnsen, S.; Kleinstein, S. H.\nScore: 21.2, Published: 2024-01-20 DOI: 10.1101/2024.01.18.576147\nMotivationAdaptive Immune Receptor Repertoire sequencing (AIRR-seq) is a valuable experimental tool to study the immune state in health and following immune challenges such as infectious diseases, (auto)immune diseases, and cancer. Several tools have been developed to reconstruct B cell and T cell receptor sequences from AIRR-seq data and infer B and T cell clonal relationships. However, currently available tools offer limited parallelization across samples, scalability or portability to high-performance computing infrastructures. ResultsTo address this need, we developed nf-core/airrflow, an end-to-end bulk and single-cell AIRR-seq processing workflow which integrates the Immcantation Framework following BCR and TCR sequencing data analysis best practices. The Immcantation Framework is a comprehensive toolset, which allows the processing of bulk and single-cell AIRR-seq data from raw read processing to clonal inference. nf-core/airrflow is written in Nextflow and is part of the nf-core project, which collects community contributed and curated Nextflow workflows for a wide variety of analysis tasks. We assessed the performance of nf-core/airrflow on simulated sequencing data and show example results with real datasets. To demonstrate the applicability of nf-core/airrflow to the high-throughput processing of large AIRR-seq datasets, we validated and extended previously reported findings of convergent antibody responses to SARS-CoV-2 by analyzing 97 COVID-19 infected individuals and 99 healthy controls retrieved from public databases. Availability and implementationnf-core/airrflow is available free of charge, under the MIT license on GitHub (https://github.com/nf-core/airrflow). Documentation and example results are available at https://nf-co.re/airrflow. Visual abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=107 SRC=\"FIGDIR/small/576147v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (24K): org.highwire.dtl.DTLVardef@12d5857org.highwire.dtl.DTLVardef@2eba6org.highwire.dtl.DTLVardef@1bce158org.highwire.dtl.DTLVardef@16e8775_HPS_FORMAT_FIGEXP M_FIG C_FIG\nREADv2: Advanced and user-friendly detection of biological relatedness in archaeogenomics\nAuthors: Alacamlı, E.; Naidoo, T.; Aktürk, S.; Güler, M. N.; Mapelli, I.; Vural, K. B.; Somel, M.; Malmström, H.; Günther, T.\nScore: 18.8, Published: 2024-01-23 DOI: 10.1101/2024.01.23.576660\nThe possibility to obtain genome-wide ancient DNA data from multiple individuals has facilitated an unprecedented perspective into prehistoric societies. Studying biological relatedness in these groups requires tailored approaches for analyzing ancient DNA due to its low coverage, post-mortem damage, and potential ascertainment bias. Here we present READv2 (Relatedness Estimation from Ancient DNA version 2), an improved Python 3 re-implementation of the most widely used tool for this purpose. While providing increased portability and making the software future-proof, we are also able to show that READv2 (a) is orders of magnitude faster than its predecessor; (b) has increased power to detect pairs of relatives using optimized default parameters; and, when the number of overlapping SNPs is sufficient, (c) can differentiate between full-siblings and parent-offspring, and (d) can classify pairs of third-degree relatedness. We further use READv2 to analyze a large empirical dataset that has previously needed two separate tools to reconstruct complex pedigrees. We show that READv2 yields results and precision similar to the combined approach but is faster and simpler to run. READv2 will become a valuable part of the archaeogenomic toolkit in providing an efficient and user-friendly classification of biological relatedness from pseudohaploid ancient DNA data.\nAnnoView enables large-scale analysis, comparison, and visualization of microbial gene neighborhoods\nAuthors: Wei, X.; Tan, H.; Lobb, B.; Zhen, W.; Wu, Z.; Parks, D. H.; Neufeld, J. D.; Moreno-Hagelsieb, G.; Doxey, A. C.\nScore: 16.4, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575735\nThe analysis and comparison of gene neighborhoods is a powerful approach for exploring microbial genome structure, function, and evolution. Although numerous tools exist for genome visualization and comparison, genome exploration across large genomic databases or user-generated datasets remains a challenge. Here, we introduce AnnoView, a web server designed for interactive exploration of gene neighborhoods across the bacterial and archaeal tree of life. Our server offers users the ability to identify, compare, and visualize gene neighborhoods of interest from 30,238 bacterial genomes and 1,672 archaeal genomes, through integration with the comprehensive GTDB and AnnoTree databases. Identified gene neighborhoods can be visualized using pre-computed functional annotations from different sources such as KEGG, Pfam, and TIGRFAM, or clustered based on similarity. Alternatively, users can upload and explore their own custom genomic datasets in GBK, GFF, or CSV format, or use AnnoView as a genome browser for relatively small genomes (e.g., viruses and plasmids). Ultimately, we anticipate that AnnoView will catalyze biological discovery by enabling user-friendly search, comparison, and visualization of genomic data. AnnoView is available at http://annoview.uwaterloo.ca\nBSAlign: a library for nucleotide sequence alignment\nAuthors: Shao, H.; Ruan, J.\nScore: 14.9, Published: 2024-01-17 DOI: 10.1101/2024.01.15.575791\nIncreasing the accuracy of the nucleotide sequence alignment is an essential issue in genomics research. Although classic dynamic-programming algorithms (e.g., Smith-Waterman and Needleman-Wunsch) guarantee to produce the optimal result, their time complexity hinders the application of large-scale sequence alignment. Many optimization efforts that aim to accelerate the alignment process generally come from three perspectives: re-designing data structures (e.g., diagonal or striped Single Instruction Multiple Data (SIMD) implementations), increasing the number of parallelisms in SIMD operations (e.g., difference recurrence relation), or reducing searching space (e.g., banded dynamic programming). However, no methods combine all these three aspects to build an ultra-fast algorithm. We have developed a Banded Striped Aligner(library) named BSAlign that delivers accurate alignment results at an ultra-fast speed by knitting a series of novel methods together to take advantage of all of the aforementioned three perspectives with highlights such as active F-loop in striped vectorization and striped move in banded dynamic programming. We applied our new acceleration design on both regular and edit-distance pairwise alignment. BSAlign achieved 2-fold speed-up than other SIMD based implementations for regular pairwise alignment, and 1.5 to 4-fold speedup in edit distance based implementations for long reads. BSAlign is implemented in C programing language and is available at https://github.com/ruanjue/bsalign.\nReformer: Deep learning model for characterizing protein-RNA interactions from sequence at single-base resolution\nAuthors: Shen, X.; Li, X.\nScore: 16.3, Published: 2024-01-15 DOI: 10.1101/2024.01.14.575540\nProtein-RNA interactions play an essential role in the regulation of transcription, translation, and metabolism of cellular RNA. Here, we develop Reformer, a deep learning model that predicts protein-RNA binding affinity purely from sequence. We developed Reformer with 155 RNA binding protein (RBP) targets from 3 cell lines. Reformer achieved high prediction accuracy at single-base resolution when tasking with inferring protein- and cell-type-specific binding affinity. We conducted electrophoretic mobility shift assays to validate high-impact RNA regulation mutations predicted by Reformer. In addition, Reformer learned to capture protein binding motifs that cannot be discovered by eCLIP-seq experiments. Furthermore, we demonstrated that motif signatures related to RNA processing functions are encoded within Reformer. In conclusion, Reformer will facilitate interpretation of the regulation mechanisms underlying RNA processing.\nFateNet: an integration of dynamical systems and deep learning for cell fate prediction\nAuthors: Sadria, M.; Bury, T.\nScore: 13.0, Published: 2024-01-20 DOI: 10.1101/2024.01.16.575913\nUnderstanding cellular decision-making, particularly its timing and impact on the biological system such as tissue health and function, is a fundamental challenge in biology and medicine. Existing methods for inferring fate decisions and cellular state dynamics from single-cell RNA sequencing data lack precision regarding decision points and broader tissue implications. Addressing this gap, we present FateNet, a computational approach integrating dynamical systems theory and deep learning to probe the cell decision-making process using scRNA-seq data. By leveraging information about normal forms and scaling behavior near tipping pointscommon to many dynamical systems, FateNet accurately predicts cell decision occurrence and offers qualitative insights into the new state of the biological system. Also, through in-silico perturbation experiments, FateNet identifies key genes and pathways governing the differentiation process in hematopoiesis. Validated using different scRNA-seq data, FateNet emerges as a user-friendly and valuable tool for predicting critical points in biological processes, providing insights into complex trajectories.\nEvoAug-TF: Extending evolution-inspired data augmentations for genomic deep learning to TensorFlow\nAuthors: Koo, P.; Yu, Y.; Muthukumar, S.\nScore: 12.3, Published: 2024-01-18 DOI: 10.1101/2024.01.17.575961\nDeep neural networks (DNNs) have been widely applied to predict the molecular functions of regulatory regions in the non-coding genome. DNNs are data hungry and thus require many training examples to fit data well. However, functional genomics experiments typically generate limited amounts of data, constrained by the activity levels of the molecular function under study inside the cell. Recently, EvoAug was introduced to train a genomic DNN with evolution-inspired augmentations. EvoAug-trained DNNs have demonstrated improved generalization and interpretability with attribution analysis. However, EvoAug only supports PyTorch-based models, which limits its applications to a broad class of genomic DNNs based in TensorFlow. Here, we extend EvoAugs functionality to TensorFlow in a new package we call EvoAug-TF. Through a systematic benchmark, we find that EvoAug-TF yields comparable performance with the original EvoAug package. AvailabilityEvoAug-TF is freely available for users and is distributed under an open-source MIT license. Researchers can access the open-source code on GitHub (https://github.com/p-koo/evoaug-tf). The pre-compiled package is provided via PyPI (https://pypi.org/project/evoaug-tf) with in-depth documentation on ReadTheDocs (https://evoaug-tf.readthedocs.io). The scripts for reproducing the results are available at (https://github.com/p-koo/evoaug-tf_analysis).\nPathFinder: a novel graph transformer model to infer multi-cell intra- and inter-cellular signaling pathways and communications\nAuthors: Feng, J.; Province, M.; Li, G.; Payne, P. R.; Chen, Y.; Li, F.\nScore: 14.0, Published: 2024-01-15 DOI: 10.1101/2024.01.13.575534\nRecently, large-scale scRNA-seq datasets have been generated to understand the complex and poorly understood signaling mechanisms within microenvironment of Alzheimers Disease (AD), which are critical for identifying novel therapeutic targets and precision medicine. Though a set of targets have been identified, however, it remains a challenging to infer the core intra- and inter-multi-cell signaling communication networks using the scRNA-seq data, considering the complex and highly interactive background signaling network. Herein, we introduced a novel graph transformer model, PathFinder, to infer multi-cell intra- and inter-cellular signaling pathways and signaling communications among multi-cell types. Compared with existing models, the novel and unique design of PathFinder is based on the divide-and-conquer strategy, which divides the complex signaling networks into signaling paths, and then score and rank them using a novel graph transformer architecture to infer the intra- and inter-cell signaling communications. We evaluated PathFinder using scRNA-seq data of APOE4-genotype specific AD mice models and identified novel APOE4 altered intra- and inter-cell interaction networks among neurons, astrocytes, and microglia. PathFinder is a general signaling network inference model and can be applied to other omics data-driven signaling network inference.\n",
  "wordCount" : "2001",
  "inLanguage": "en",
  "datePublished": "2024-01-24T10:40:04Z",
  "dateModified": "2024-01-24T10:40:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 24, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575658">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575658" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575658">
        <p class="paperTitle">When the pen is mightier than the sword: semi-automatic 2 and 3D image labelling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575658" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575658" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hollandi, R.; Bauer, D.; Diosdi, A.; Schrettner, B.; Toth, T.; Hirling, D.; Hollandi, G.; Harmati, M.; Molnar, J.; Horvath, P.</p>
        <p class="info">Score: 38.4, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575658' target='https://doi.org/10.1101/2024.01.15.575658'> 10.1101/2024.01.15.575658</a></p>
        <p class="abstract">Data is the driving engine of learning-based algorithms, the creation of which fundamentally determines the performance, accuracy, generalizability and quality of any model or method trained on it. When only skilled or trained personnel can create reliable annotations, assisted software solutions are desirable to reduce the time and effort the expert must spend on labelling. Herein is proposed an automated annotation helper software package in napari that offers multiple methods to assist the annotator in creating object-based labels on 2D or 3D images.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.563252">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.563252" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.563252">
        <p class="paperTitle">CellTracksColab--A platform for compiling, analyzing, and exploring tracking data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.563252" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.563252" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jacquemet, G.</p>
        <p class="info">Score: 68.9, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.563252' target='https://doi.org/10.1101/2023.10.20.563252'> 10.1101/2023.10.20.563252</a></p>
        <p class="abstract">In life sciences, tracking objects from movies enables researchers to quantify the behavior of single particles, organelles, bacteria, cells, and even whole animals. While numerous tools now allow automated tracking from video, a significant challenge persists in compiling, analyzing, and exploring the large datasets generated by these approaches. Here, we introduce CellTracksColab, an online platform tailored to simplify the exploration and analysis of tracking data by leveraging the free, cloud-based computational resources of Google Colab. CellTracksColab facilitates the amalgamation and analysis of results across multiple fields of view, conditions, and repeats, ensuring a holistic dataset overview. CellTracksColab also harnesses the power of high-dimensional data reduction and clustering, enabling researchers to identify distinct behavioral patterns and trends without bias. Finally, CellTracksColab also includes specialized analysis modules enabling spatial analyses (clustering, proximity to specific regions of interest). CellTracksColab is available for the broader scientific community here.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576147">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576147" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576147">
        <p class="paperTitle">nf-core/airrflow: an adaptive immune receptor repertoire analysis workflow employing the Immcantation framework</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576147" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576147" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gabernet, G.; Marquez, S.; Bjornson, R.; Peltzer, A.; Meng, H.; Aron, E.; Lee, N. Y.; Jensen, C.; Ladd, D.; Hanssen, F.; Heumos, S.; nf-core community,  ; Yaari, G.; Kowarik, M. C.; Nahnsen, S.; Kleinstein, S. H.</p>
        <p class="info">Score: 21.2, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576147' target='https://doi.org/10.1101/2024.01.18.576147'> 10.1101/2024.01.18.576147</a></p>
        <p class="abstract">MotivationAdaptive Immune Receptor Repertoire sequencing (AIRR-seq) is a valuable experimental tool to study the immune state in health and following immune challenges such as infectious diseases, (auto)immune diseases, and cancer. Several tools have been developed to reconstruct B cell and T cell receptor sequences from AIRR-seq data and infer B and T cell clonal relationships. However, currently available tools offer limited parallelization across samples, scalability or portability to high-performance computing infrastructures.

ResultsTo address this need, we developed nf-core/airrflow, an end-to-end bulk and single-cell AIRR-seq processing workflow which integrates the Immcantation Framework following BCR and TCR sequencing data analysis best practices. The Immcantation Framework is a comprehensive toolset, which allows the processing of bulk and single-cell AIRR-seq data from raw read processing to clonal inference. nf-core/airrflow is written in Nextflow and is part of the nf-core project, which collects community contributed and curated Nextflow workflows for a wide variety of analysis tasks. We assessed the performance of nf-core/airrflow on simulated sequencing data and show example results with real datasets. To demonstrate the applicability of nf-core/airrflow to the high-throughput processing of large AIRR-seq datasets, we validated and extended previously reported findings of convergent antibody responses to SARS-CoV-2 by analyzing 97 COVID-19 infected individuals and 99 healthy controls retrieved from public databases.

Availability and implementationnf-core/airrflow is available free of charge, under the MIT license on GitHub (https://github.com/nf-core/airrflow). Documentation and example results are available at https://nf-co.re/airrflow.

Visual abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=107 SRC=&#34;FIGDIR/small/576147v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (24K):
org.highwire.dtl.DTLVardef@12d5857org.highwire.dtl.DTLVardef@2eba6org.highwire.dtl.DTLVardef@1bce158org.highwire.dtl.DTLVardef@16e8775_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.23.576660">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.23.576660" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.23.576660">
        <p class="paperTitle">READv2: Advanced and user-friendly detection of biological relatedness in archaeogenomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.23.576660" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.23.576660" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alacamlı, E.; Naidoo, T.; Aktürk, S.; Güler, M. N.; Mapelli, I.; Vural, K. B.; Somel, M.; Malmström, H.; Günther, T.</p>
        <p class="info">Score: 18.8, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.23.576660' target='https://doi.org/10.1101/2024.01.23.576660'> 10.1101/2024.01.23.576660</a></p>
        <p class="abstract">The possibility to obtain genome-wide ancient DNA data from multiple individuals has facilitated an unprecedented perspective into prehistoric societies. Studying biological relatedness in these groups requires tailored approaches for analyzing ancient DNA due to its low coverage, post-mortem damage, and potential ascertainment bias. Here we present READv2 (Relatedness Estimation from Ancient DNA version 2), an improved Python 3 re-implementation of the most widely used tool for this purpose. While providing increased portability and making the software future-proof, we are also able to show that READv2 (a) is orders of magnitude faster than its predecessor; (b) has increased power to detect pairs of relatives using optimized default parameters; and, when the number of overlapping SNPs is sufficient, (c) can differentiate between full-siblings and parent-offspring, and (d) can classify pairs of third-degree relatedness. We further use READv2 to analyze a large empirical dataset that has previously needed two separate tools to reconstruct complex pedigrees. We show that READv2 yields results and precision similar to the combined approach but is faster and simpler to run. READv2 will become a valuable part of the archaeogenomic toolkit in providing an efficient and user-friendly classification of biological relatedness from pseudohaploid ancient DNA data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575735">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575735" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575735">
        <p class="paperTitle">AnnoView enables large-scale analysis, comparison, and visualization of microbial gene neighborhoods</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575735" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575735" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wei, X.; Tan, H.; Lobb, B.; Zhen, W.; Wu, Z.; Parks, D. H.; Neufeld, J. D.; Moreno-Hagelsieb, G.; Doxey, A. C.</p>
        <p class="info">Score: 16.4, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575735' target='https://doi.org/10.1101/2024.01.15.575735'> 10.1101/2024.01.15.575735</a></p>
        <p class="abstract">The analysis and comparison of gene neighborhoods is a powerful approach for exploring microbial genome structure, function, and evolution. Although numerous tools exist for genome visualization and comparison, genome exploration across large genomic databases or user-generated datasets remains a challenge. Here, we introduce AnnoView, a web server designed for interactive exploration of gene neighborhoods across the bacterial and archaeal tree of life. Our server offers users the ability to identify, compare, and visualize gene neighborhoods of interest from 30,238 bacterial genomes and 1,672 archaeal genomes, through integration with the comprehensive GTDB and AnnoTree databases. Identified gene neighborhoods can be visualized using pre-computed functional annotations from different sources such as KEGG, Pfam, and TIGRFAM, or clustered based on similarity. Alternatively, users can upload and explore their own custom genomic datasets in GBK, GFF, or CSV format, or use AnnoView as a genome browser for relatively small genomes (e.g., viruses and plasmids). Ultimately, we anticipate that AnnoView will catalyze biological discovery by enabling user-friendly search, comparison, and visualization of genomic data. AnnoView is available at http://annoview.uwaterloo.ca</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575791">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575791" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575791">
        <p class="paperTitle">BSAlign: a library for nucleotide sequence alignment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575791" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575791" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shao, H.; Ruan, J.</p>
        <p class="info">Score: 14.9, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575791' target='https://doi.org/10.1101/2024.01.15.575791'> 10.1101/2024.01.15.575791</a></p>
        <p class="abstract">Increasing the accuracy of the nucleotide sequence alignment is an essential issue in genomics research. Although classic dynamic-programming algorithms (e.g., Smith-Waterman and Needleman-Wunsch) guarantee to produce the optimal result, their time complexity hinders the application of large-scale sequence alignment. Many optimization efforts that aim to accelerate the alignment process generally come from three perspectives: re-designing data structures (e.g., diagonal or striped Single Instruction Multiple Data (SIMD) implementations), increasing the number of parallelisms in SIMD operations (e.g., difference recurrence relation), or reducing searching space (e.g., banded dynamic programming). However, no methods combine all these three aspects to build an ultra-fast algorithm. We have developed a Banded Striped Aligner(library) named BSAlign that delivers accurate alignment results at an ultra-fast speed by knitting a series of novel methods together to take advantage of all of the aforementioned three perspectives with highlights such as active F-loop in striped vectorization and striped move in banded dynamic programming. We applied our new acceleration design on both regular and edit-distance pairwise alignment. BSAlign achieved 2-fold speed-up than other SIMD based implementations for regular pairwise alignment, and 1.5 to 4-fold speedup in edit distance based implementations for long reads. BSAlign is implemented in C programing language and is available at https://github.com/ruanjue/bsalign.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.14.575540">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.14.575540" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.14.575540">
        <p class="paperTitle">Reformer: Deep learning model for characterizing protein-RNA interactions from sequence at single-base resolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.14.575540" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.14.575540" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shen, X.; Li, X.</p>
        <p class="info">Score: 16.3, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.14.575540' target='https://doi.org/10.1101/2024.01.14.575540'> 10.1101/2024.01.14.575540</a></p>
        <p class="abstract">Protein-RNA interactions play an essential role in the regulation of transcription, translation, and metabolism of cellular RNA. Here, we develop Reformer, a deep learning model that predicts protein-RNA binding affinity purely from sequence. We developed Reformer with 155 RNA binding protein (RBP) targets from 3 cell lines. Reformer achieved high prediction accuracy at single-base resolution when tasking with inferring protein- and cell-type-specific binding affinity. We conducted electrophoretic mobility shift assays to validate high-impact RNA regulation mutations predicted by Reformer. In addition, Reformer learned to capture protein binding motifs that cannot be discovered by eCLIP-seq experiments. Furthermore, we demonstrated that motif signatures related to RNA processing functions are encoded within Reformer. In conclusion, Reformer will facilitate interpretation of the regulation mechanisms underlying RNA processing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.16.575913">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.16.575913" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.16.575913">
        <p class="paperTitle">FateNet: an integration of dynamical systems and deep learning for cell fate prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.16.575913" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.16.575913" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sadria, M.; Bury, T.</p>
        <p class="info">Score: 13.0, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.16.575913' target='https://doi.org/10.1101/2024.01.16.575913'> 10.1101/2024.01.16.575913</a></p>
        <p class="abstract">Understanding cellular decision-making, particularly its timing and impact on the biological system such as tissue health and function, is a fundamental challenge in biology and medicine. Existing methods for inferring fate decisions and cellular state dynamics from single-cell RNA sequencing data lack precision regarding decision points and broader tissue implications. Addressing this gap, we present FateNet, a computational approach integrating dynamical systems theory and deep learning to probe the cell decision-making process using scRNA-seq data. By leveraging information about normal forms and scaling behavior near tipping pointscommon to many dynamical systems, FateNet accurately predicts cell decision occurrence and offers qualitative insights into the new state of the biological system. Also, through in-silico perturbation experiments, FateNet identifies key genes and pathways governing the differentiation process in hematopoiesis. Validated using different scRNA-seq data, FateNet emerges as a user-friendly and valuable tool for predicting critical points in biological processes, providing insights into complex trajectories.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.575961">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.575961" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.575961">
        <p class="paperTitle">EvoAug-TF: Extending evolution-inspired data augmentations for genomic deep learning to TensorFlow</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.575961" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.575961" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Koo, P.; Yu, Y.; Muthukumar, S.</p>
        <p class="info">Score: 12.3, Published: 2024-01-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.575961' target='https://doi.org/10.1101/2024.01.17.575961'> 10.1101/2024.01.17.575961</a></p>
        <p class="abstract">Deep neural networks (DNNs) have been widely applied to predict the molecular functions of regulatory regions in the non-coding genome. DNNs are data hungry and thus require many training examples to fit data well. However, functional genomics experiments typically generate limited amounts of data, constrained by the activity levels of the molecular function under study inside the cell. Recently, EvoAug was introduced to train a genomic DNN with evolution-inspired augmentations. EvoAug-trained DNNs have demonstrated improved generalization and interpretability with attribution analysis. However, EvoAug only supports PyTorch-based models, which limits its applications to a broad class of genomic DNNs based in TensorFlow. Here, we extend EvoAugs functionality to TensorFlow in a new package we call EvoAug-TF. Through a systematic benchmark, we find that EvoAug-TF yields comparable performance with the original EvoAug package.

AvailabilityEvoAug-TF is freely available for users and is distributed under an open-source MIT license. Researchers can access the open-source code on GitHub (https://github.com/p-koo/evoaug-tf). The pre-compiled package is provided via PyPI (https://pypi.org/project/evoaug-tf) with in-depth documentation on ReadTheDocs (https://evoaug-tf.readthedocs.io). The scripts for reproducing the results are available at (https://github.com/p-koo/evoaug-tf_analysis).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.13.575534">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.13.575534" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.13.575534">
        <p class="paperTitle">PathFinder: a novel graph transformer model to infer multi-cell intra- and inter-cellular signaling pathways and communications</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.13.575534" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.13.575534" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Feng, J.; Province, M.; Li, G.; Payne, P. R.; Chen, Y.; Li, F.</p>
        <p class="info">Score: 14.0, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.13.575534' target='https://doi.org/10.1101/2024.01.13.575534'> 10.1101/2024.01.13.575534</a></p>
        <p class="abstract">Recently, large-scale scRNA-seq datasets have been generated to understand the complex and poorly understood signaling mechanisms within microenvironment of Alzheimers Disease (AD), which are critical for identifying novel therapeutic targets and precision medicine. Though a set of targets have been identified, however, it remains a challenging to infer the core intra- and inter-multi-cell signaling communication networks using the scRNA-seq data, considering the complex and highly interactive background signaling network. Herein, we introduced a novel graph transformer model, PathFinder, to infer multi-cell intra- and inter-cellular signaling pathways and signaling communications among multi-cell types. Compared with existing models, the novel and unique design of PathFinder is based on the divide-and-conquer strategy, which divides the complex signaling networks into signaling paths, and then score and rank them using a novel graph transformer architecture to infer the intra- and inter-cell signaling communications. We evaluated PathFinder using scRNA-seq data of APOE4-genotype specific AD mice models and identified novel APOE4 altered intra- and inter-cell interaction networks among neurons, astrocytes, and microglia. PathFinder is a general signaling network inference model and can be applied to other omics data-driven signaling network inference.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
