<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Polygraph: A Software Framework for the Systematic Assessment of Synthetic Regulatory DNA Elements
Authors: Gupta, A.; Lal, A.; Gunsalus, L. M.; Biancalani, T.; Eraslan, G.
Score: 31.9, Published: 2023-11-27 DOI: 10.1101/2023.11.27.568764
The design of regulatory elements is pivotal in numerous therapeutic interventions, including gene and cell therapy, wherein the typical objective is to engineer DNA sequences exhibiting specific attributes like cell-type specificity and elevated expression levels. However, the systematic assessment of these constructed DNA sequences remains challenging due to the absence of robust metrics and an integrated software framework.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="Polygraph: A Software Framework for the Systematic Assessment of Synthetic Regulatory DNA Elements
Authors: Gupta, A.; Lal, A.; Gunsalus, L. M.; Biancalani, T.; Eraslan, G.
Score: 31.9, Published: 2023-11-27 DOI: 10.1101/2023.11.27.568764
The design of regulatory elements is pivotal in numerous therapeutic interventions, including gene and cell therapy, wherein the typical objective is to engineer DNA sequences exhibiting specific attributes like cell-type specificity and elevated expression levels. However, the systematic assessment of these constructed DNA sequences remains challenging due to the absence of robust metrics and an integrated software framework." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-06T10:40:04+00:00" />
<meta property="article:modified_time" content="2023-12-06T10:40:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="Polygraph: A Software Framework for the Systematic Assessment of Synthetic Regulatory DNA Elements
Authors: Gupta, A.; Lal, A.; Gunsalus, L. M.; Biancalani, T.; Eraslan, G.
Score: 31.9, Published: 2023-11-27 DOI: 10.1101/2023.11.27.568764
The design of regulatory elements is pivotal in numerous therapeutic interventions, including gene and cell therapy, wherein the typical objective is to engineer DNA sequences exhibiting specific attributes like cell-type specificity and elevated expression levels. However, the systematic assessment of these constructed DNA sequences remains challenging due to the absence of robust metrics and an integrated software framework."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "Polygraph: A Software Framework for the Systematic Assessment of Synthetic Regulatory DNA Elements\nAuthors: Gupta, A.; Lal, A.; Gunsalus, L. M.; Biancalani, T.; Eraslan, G.\nScore: 31.9, Published: 2023-11-27 DOI: 10.1101/2023.11.27.568764\nThe design of regulatory elements is pivotal in numerous therapeutic interventions, including gene and cell therapy, wherein the typical objective is to engineer DNA sequences exhibiting specific attributes like cell-type specificity and elevated expression levels. However, the systematic assessment of these constructed DNA sequences remains challenging due to the absence of robust metrics and an integrated software framework.",
  "keywords": [
    
  ],
  "articleBody": " Polygraph: A Software Framework for the Systematic Assessment of Synthetic Regulatory DNA Elements\nAuthors: Gupta, A.; Lal, A.; Gunsalus, L. M.; Biancalani, T.; Eraslan, G.\nScore: 31.9, Published: 2023-11-27 DOI: 10.1101/2023.11.27.568764\nThe design of regulatory elements is pivotal in numerous therapeutic interventions, including gene and cell therapy, wherein the typical objective is to engineer DNA sequences exhibiting specific attributes like cell-type specificity and elevated expression levels. However, the systematic assessment of these constructed DNA sequences remains challenging due to the absence of robust metrics and an integrated software framework. Here, we introduce Polygraph, a Python framework for evaluating synthetic DNA sequences. Polygraph provides a variety of features to streamline the synthesis and scrutiny of regulatory elements, incorporating features like a diversity index, motif and k-mer composition, similarity to endogenous regulatory sequences, and screening with predictive and foundational models. Consequently, Polygraph stands as the first instrument for the assessment of synthetic regulatory sequences, enabling expedited progress in therapeutic applications and improving our understanding of gene regulatory mechanisms.\nGiotto Suite: a multi-scale and technology-agnostic spatial multi-omics analysis ecosystem\nAuthors: Chen, J. G.; Chavez-Fuentes, J. C.; O'Brien, M.; Xu, J.; Ruiz, E.; Wang, W.; Amin, I.; Sarfraz, I.; Guckhool, P.; Sistig, A.; Yuan, G.-C.; Dries, R.\nScore: 26.7, Published: 2023-11-27 DOI: 10.1101/2023.11.26.568752\nEmerging spatial omics technologies continue to advance the molecular mapping of tissue architecture and the investigation of gene regulation and cellular crosstalk, which in turn provide new mechanistic insights into a wide range of biological processes and diseases. Such technologies provide an increasingly large amount of information content at multiple spatial scales. However, representing and harmonizing diverse spatial datasets efficiently, including combining multiple modalities or spatial scales in a scalable and flexible manner, remains a substantial challenge. Here, we present Giotto Suite, a suite of open-source software packages that underlies a fully modular and integrated spatial data analysis toolbox. At its core, Giotto Suite is centered around an innovative and technology-agnostic data framework embedded in the R software environment, which allows the representation and integration of virtually any type of spatial omics data at any spatial resolution. In addition, Giotto Suite provides both scalable and extensible end-to-end solutions for data analysis, integration, and visualization. Giotto Suite integrates molecular, morphology, spatial, and annotated feature information to create a responsive and flexible workflow for multi-scale, multi-omic data analyses, as demonstrated here by applications to several state-of-the-art spatial technologies. Furthermore, Giotto Suite builds upon interoperable interfaces and data structures that bridge the established fields of genomics and spatial data science, thereby enabling independent developers to create custom-engineered pipelines. As such, Giotto Suite creates an immersive ecosystem for spatial multi-omic data analysis.\nStreamlining remote nanopore data access with slow5curl\nAuthors: Wong, B.; Ferguson, J. M.; Gamaarachchi, H.; Deveson, I. W.\nScore: 19.6, Published: 2023-11-30 DOI: 10.1101/2023.11.28.569128\nAs adoption of nanopore sequencing technology continues to advance, the need to maintain large volumes of raw current signal data for reanalysis with updated algorithms is a growing challenge. Here we introduce slow5curl, a software package designed to streamline nanopore data sharing, accessibility and reanalysis. Slow5curl allows a user to fetch a specified read or group of reads from a raw nanopore dataset stored on a remote server, such as a public data repository, without downloading the entire file. Slow5curl uses an index to quickly fetch specific reads from a large dataset in SLOW5/BLOW5 format and highly parallelised data access requests to maximise download speeds. Using all public nanopore data from the Human Pangenome Reference Consortium (\u003e22 TB), we demonstrate how slow5curl can be used to quickly fetch and reanalyse signal reads corresponding to a set of target genes from each individual in large cohort dataset (n = 91), minimising the time, egress costs, and local storage requirements for their reanalysis. We provide slow5curl as a free, open-source package that will reduce frictions in data sharing for the nanopore community: https://github.com/BonsonW/slow5curl\nDistance-AF: Modifying Predicted Protein Structure Models by Alphafold2 with User-Specified Distance Constraints\nAuthors: Zhang, Y.; Zhang, Z.; Kagaya, Y.; Terashi, G.; Zhao, B.; Xiong, Y.; Kihara, D.\nScore: 18.4, Published: 2023-12-04 DOI: 10.1101/2023.12.01.569498\nThe three-dimensional structure of a protein plays a fundamental role in determining its function and has an essential impact on understanding biological processes. Despite significant progress in protein structure prediction, such as AlphaFold2, challenges remain on those hard targets that Alphafold2 does not often perform well due to the complex folding of protein and a large number of possible conformations. Here we present a modified version of the AlphaFold2, called Distance-AF, which aims to improve the performance of AlphaFold2 by including distance constraints as input information. Distance-AF uses AlphaFold2's predicted structure as a starting point and incorporates distance constraints between amino acids to adjust folding of the protein structure until it meets the constraints. Distance-AF can correct the domain orientation on challenging targets, leading to more accurate structures with a lower root mean square deviation (RMSD). The ability of Distance-AF is also useful in fitting protein structures into cryo-electron microscopy maps.\nQuality assessment of splice site annotation based on conservation across multiple species\nAuthors: Minkin, I.; Salzberg, S.\nScore: 16.2, Published: 2023-12-02 DOI: 10.1101/2023.12.01.569581\nAO_SCPLOWBSTRACTC_SCPLOWDespite many improvements over the years, the annotation of the human genome remains imperfect, and even the best annotations of the human reference genome sometimes contradict one another. Hence, refinement of the human genome annotation is an important challenge. The use of evolutionarily conserved sequences provides a strategy for addressing this problem, and the rapidly growing number of genomes from other species increases the power of an evolution-driven approach. Using the latest large-scale whole genome alignment data, we found that splice sites from protein-coding genes in the high-quality MANE annotation are consistently conserved across more than 400 species. We also studied splice sites from the RefSeq, GENCODE, and CHESS databases that are not present in MANE, from both protein-coding genes and lncRNAs. We trained a logistic regression classifier to distinguish between the conservation patterns exhibited by splice sites from MANE versus sites that were flanked by the standard GT-AG dinucleotides, but that were chosen randomly from a sequence not under selection. We found that up to 70% of splice sites from annotated protein-coding transcripts outside of MANE exhibit conservation patterns closer to random sequence as opposed to highly-conserved splice sites from MANE. Our study highlights potentially erroneous splice sites that might require further scrutiny.\nLearning sequence, structure, and function representations of proteins with language models\nAuthors: Hamamsy, T.; Barot, M.; Morton, J. T.; Steinegger, M.; Bonneau, R.; Cho, K.\nScore: 34.3, Published: 2023-11-26 DOI: 10.1101/2023.11.26.568742\nThe sequence-structure-function relationships that ultimately generate the diversity of extant observed proteins is complex, as proteins bridge the gap between multiple informational and physical scales involved in nearly all cellular processes. One limitation of existing protein annotation databases such as UniProt is that less than 1% of proteins have experimentally verified functions, and computational methods are needed to fill in the missing information. Here, we demonstrate that a multi-aspect framework based on protein language models can learn sequence-structure-function representations of amino acid sequences, and can provide the foundation for sensitive sequence-structure-function aware protein sequence search and annotation. Based on this model, we introduce a multi-aspect information retrieval system for proteins, Protein-Vec, covering sequence, structure, and function aspects, that enables computational protein annotation and function prediction at tree-of-life scales.\npyaging: a Python-based compendium of GPU-optimized aging clocks\nAuthors: de Lima Camillo, L. P.\nScore: 12.0, Published: 2023-11-29 DOI: 10.1101/2023.11.28.569069\nMotivationAging is intricately linked to diseases and mortality and is reflected in molecular changes across various tissues. The development and refinement of biomarkers of aging, healthspan, and lifespan using machine learning models, known as aging clocks, leverage epigenetic and other molecular signatures. Despite advancements, as noted by the Biomarkers of Aging Consortium, the field grapples with challenges, notably the lack of robust software tools for integrating and comparing these diverse models. ResultsI introduce pyaging, a comprehensive Python package, designed to bridge the gap in aging research software tools. pyaging integrates over 30 aging clocks, with plans to expand to more than 100, covering a range of molecular data types including DNA methylation, transcriptomics, histone mark ChIP-Seq, and ATAC-Seq. The package features a variety of model types, from linear and principal component models to neural networks and automatic relevance determination models. Utilizing a PyTorch-based backend for GPU acceleration, pyaging ensures rapid inference even with large datasets and complex models. The package supports multi-species analysis, currently including humans, various mammals, and C. elegans. Availability and Implementationpyaging is accessible at https://github.com/rsinghlab/pyaging. The package is structured to facilitate ease of use and integration into existing research workflows, supporting the flexible anndata data format. Supplementary InformationSupplementary materials, including detailed documentation and usage examples, are available online at the pyaging documentation site (https://pyaging.readthedocs.io/en/latest/index.html).\nBenchmarking computational methods to identify spatially variable genes and peaks\nAuthors: Li, Z.; Patel, Z. M.; Song, D.; Yan, G.; Li, J. J.; Pinello, L.\nScore: 12.0, Published: 2023-12-03 DOI: 10.1101/2023.12.02.569717\nSpatially resolved transcriptomics offers unprecedented insight by enabling the profiling of gene expression within the intact spatial context of cells, effectively adding a new and essential dimension to data interpretation. To efficiently detect spatial structure of interest, an essential step in analyzing such data involves identifying spatially variable genes. Despite researchers having developed several computational methods to accomplish this task, the lack of a comprehensive benchmark evaluating their performance remains a considerable gap in the field. Here, we present a systematic evaluation of 14 methods using 60 simulated datasets generated by four different simulation strategies, 12 real-world transcriptomics, and three spatial ATAC-seq datasets. We find that spatialDE2 consistently outperforms the other benchmarked methods, and Morans I achieves competitive performance in different experimental settings. Moreover, our results reveal that more specialized algorithms are needed to identify spatially variable peaks.\nHyGAnno: Hybrid graph neural network-based cell type annotation for single-cell ATAC sequencing data\nAuthors: Zhang, W.; Cui, Y.; Liu, B.; Loza Lopez, M.; Park, S.-J.; Nakai, K.\nScore: 11.8, Published: 2023-12-03 DOI: 10.1101/2023.11.29.569114\nReliable cell type annotations are crucial for investigating cellular heterogeneity in single-cell omics data. Although various computational approaches have been proposed for single-cell RNA sequencing (scRNA-seq) annotation, high-quality cell labels are still lacking in single-cell ATAC sequencing (scATAC-seq) data, because of extreme sparsity and inconsistent chromatin accessibility between datasets. This calls for novel cell type annotation methods in scATAC-seq, to better explore cell type-specific gene regulatory mechanisms and provide a complementary epigenomic layer to scRNA-seq data. Here, we present a novel automated cell annotation method that transfers cell type information from a well-labeled scRNA-seq reference to an unlabeled scATAC-seq target, via a parallel graph neural network, in a semi-supervised manner. Unlike existing methods that utilize only gene expression or gene activity features, HyGAnno integrates genome-wide accessibility peak features to facilitate the training process. In addition, HyGAnno reconstructs a reference-target cell graph that can be used to detect cells with low prediction reliability, according to their specific graph connectivity patterns. HyGAnno was tested using large datasets and demonstrated the advantages of accurate cell annotation, interpretable cell embedding, robustness to noisy reference data, and adaptability to tumor tissues.\nBenchmarking active learning protocols for ligand binding affinity prediction\nAuthors: Gorantla, R.; Kubincova, A.; Suutari, B.; Cossins, B. P.; Mey, A.\nScore: 15.4, Published: 2023-11-24 DOI: 10.1101/2023.11.24.568570\nActive learning (AL) has become a powerful tool in computational drug discovery, enabling the identification of top binders from vast molecular libraries with reduced costs for relative binding free energy calculations and experiments. To design a robust AL protocol, it is important to understand the influence of AL parameters, as well as the features of the datasets on the outcomes. We use four affinity datasets for different targets (TYK2, USP7, D2R, Mpro) to systematically evaluate the performance of machine learning models (Gaussian Process model, Chemprop), sample selection protocols, as well as the batch size based on metrics describing the overall predictive power of the model (R2, Spearman rank, RMSE) as well as the accurate identification of top 2% / 5% binders (Recall, F1 score). Both models have a comparable Recall of top binders on large datasets, but the GP models surpass Chemprop when training data is sparse. A larger initial batch size, especially on diverse datasets, increased the Recall of both models as well as overall correlation metrics. However, for subsequent cycles, smaller batch sizes of 20 or 30 compounds proved to be desirable. Furthermore, the presence of Gaussian noise to the data, up to a certain threshold, still allowed the model to identify clusters with top-scoring compounds. However, excessive noise (\u003c1{sigma}) did impact the models predictive and exploitative capabilities. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=111 SRC=\"FIGDIR/small/568570v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (28K): org.highwire.dtl.DTLVardef@cefb11org.highwire.dtl.DTLVardef@c522bcorg.highwire.dtl.DTLVardef@6b8f18org.highwire.dtl.DTLVardef@17f7706_HPS_FORMAT_FIGEXP M_FIG O_FLOATNOTOC GraphicC_FLOATNO C_FIG\n",
  "wordCount" : "2100",
  "inLanguage": "en",
  "datePublished": "2023-12-06T10:40:04Z",
  "dateModified": "2023-12-06T10:40:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 6, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.568764">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.568764" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.568764">
        <p class="paperTitle">Polygraph: A Software Framework for the Systematic Assessment of Synthetic Regulatory DNA Elements</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.568764" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.568764" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gupta, A.; Lal, A.; Gunsalus, L. M.; Biancalani, T.; Eraslan, G.</p>
        <p class="info">Score: 31.9, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.568764' target='https://doi.org/10.1101/2023.11.27.568764'> 10.1101/2023.11.27.568764</a></p>
        <p class="abstract">The design of regulatory elements is pivotal in numerous therapeutic interventions, including gene and cell therapy, wherein the typical objective is to engineer DNA sequences exhibiting specific attributes like cell-type specificity and elevated expression levels. However, the systematic assessment of these constructed DNA sequences remains challenging due to the absence of robust metrics and an integrated software framework. Here, we introduce Polygraph, a Python framework for evaluating synthetic DNA sequences. Polygraph provides a variety of features to streamline the synthesis and scrutiny of regulatory elements, incorporating features like a diversity index, motif and k-mer composition, similarity to endogenous regulatory sequences, and screening with predictive and foundational models. Consequently, Polygraph stands as the first instrument for the assessment of synthetic regulatory sequences, enabling expedited progress in therapeutic applications and improving our understanding of gene regulatory mechanisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.26.568752">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.26.568752" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.26.568752">
        <p class="paperTitle">Giotto Suite: a multi-scale and technology-agnostic spatial multi-omics analysis ecosystem</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.26.568752" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.26.568752" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, J. G.; Chavez-Fuentes, J. C.; O&#39;Brien, M.; Xu, J.; Ruiz, E.; Wang, W.; Amin, I.; Sarfraz, I.; Guckhool, P.; Sistig, A.; Yuan, G.-C.; Dries, R.</p>
        <p class="info">Score: 26.7, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.26.568752' target='https://doi.org/10.1101/2023.11.26.568752'> 10.1101/2023.11.26.568752</a></p>
        <p class="abstract">Emerging spatial omics technologies continue to advance the molecular mapping of tissue architecture and the investigation of gene regulation and cellular crosstalk, which in turn provide new mechanistic insights into a wide range of biological processes and diseases. Such technologies provide an increasingly large amount of information content at multiple spatial scales. However, representing and harmonizing diverse spatial datasets efficiently, including combining multiple modalities or spatial scales in a scalable and flexible manner, remains a substantial challenge. Here, we present Giotto Suite, a suite of open-source software packages that underlies a fully modular and integrated spatial data analysis toolbox. At its core, Giotto Suite is centered around an innovative and technology-agnostic data framework embedded in the R software environment, which allows the representation and integration of virtually any type of spatial omics data at any spatial resolution. In addition, Giotto Suite provides both scalable and extensible end-to-end solutions for data analysis, integration, and visualization. Giotto Suite integrates molecular, morphology, spatial, and annotated feature information to create a responsive and flexible workflow for multi-scale, multi-omic data analyses, as demonstrated here by applications to several state-of-the-art spatial technologies. Furthermore, Giotto Suite builds upon interoperable interfaces and data structures that bridge the established fields of genomics and spatial data science, thereby enabling independent developers to create custom-engineered pipelines. As such, Giotto Suite creates an immersive ecosystem for spatial multi-omic data analysis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.569128">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.569128" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.569128">
        <p class="paperTitle">Streamlining remote nanopore data access with slow5curl</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.569128" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.569128" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wong, B.; Ferguson, J. M.; Gamaarachchi, H.; Deveson, I. W.</p>
        <p class="info">Score: 19.6, Published: 2023-11-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.569128' target='https://doi.org/10.1101/2023.11.28.569128'> 10.1101/2023.11.28.569128</a></p>
        <p class="abstract">As adoption of nanopore sequencing technology continues to advance, the need to maintain large volumes of raw current signal data for reanalysis with updated algorithms is a growing challenge. Here we introduce slow5curl, a software package designed to streamline nanopore data sharing, accessibility and reanalysis. Slow5curl allows a user to fetch a specified read or group of reads from a raw nanopore dataset stored on a remote server, such as a public data repository, without downloading the entire file. Slow5curl uses an index to quickly fetch specific reads from a large dataset in SLOW5/BLOW5 format and highly parallelised data access requests to maximise download speeds. Using all public nanopore data from the Human Pangenome Reference Consortium (&gt;22 TB), we demonstrate how slow5curl can be used to quickly fetch and reanalyse signal reads corresponding to a set of target genes from each individual in large cohort dataset (n = 91), minimising the time, egress costs, and local storage requirements for their reanalysis. We provide slow5curl as a free, open-source package that will reduce frictions in data sharing for the nanopore community: https://github.com/BonsonW/slow5curl</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569498">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569498" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569498">
        <p class="paperTitle">Distance-AF: Modifying Predicted Protein Structure Models by Alphafold2 with User-Specified Distance Constraints</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569498" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569498" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, Y.; Zhang, Z.; Kagaya, Y.; Terashi, G.; Zhao, B.; Xiong, Y.; Kihara, D.</p>
        <p class="info">Score: 18.4, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569498' target='https://doi.org/10.1101/2023.12.01.569498'> 10.1101/2023.12.01.569498</a></p>
        <p class="abstract">The three-dimensional structure of a protein plays a fundamental role in determining its function and has an essential impact on understanding biological processes. Despite significant progress in protein structure prediction, such as AlphaFold2, challenges remain on those hard targets that Alphafold2 does not often perform well due to the complex folding of protein and a large number of possible conformations. Here we present a modified version of the AlphaFold2, called Distance-AF, which aims to improve the performance of AlphaFold2 by including distance constraints as input information. Distance-AF uses AlphaFold2&#39;s predicted structure as a starting point and incorporates distance constraints between amino acids to adjust folding of the protein structure until it meets the constraints. Distance-AF can correct the domain orientation on challenging targets, leading to more accurate structures with a lower root mean square deviation (RMSD). The ability of Distance-AF is also useful in fitting protein structures into cryo-electron microscopy maps.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569581">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569581" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569581">
        <p class="paperTitle">Quality assessment of splice site annotation based on conservation across multiple species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569581" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569581" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Minkin, I.; Salzberg, S.</p>
        <p class="info">Score: 16.2, Published: 2023-12-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569581' target='https://doi.org/10.1101/2023.12.01.569581'> 10.1101/2023.12.01.569581</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWDespite many improvements over the years, the annotation of the human genome remains imperfect, and even the best annotations of the human reference genome sometimes contradict one another. Hence, refinement of the human genome annotation is an important challenge. The use of evolutionarily conserved sequences provides a strategy for addressing this problem, and the rapidly growing number of genomes from other species increases the power of an evolution-driven approach. Using the latest large-scale whole genome alignment data, we found that splice sites from protein-coding genes in the high-quality MANE annotation are consistently conserved across more than 400 species. We also studied splice sites from the RefSeq, GENCODE, and CHESS databases that are not present in MANE, from both protein-coding genes and lncRNAs. We trained a logistic regression classifier to distinguish between the conservation patterns exhibited by splice sites from MANE versus sites that were flanked by the standard GT-AG dinucleotides, but that were chosen randomly from a sequence not under selection. We found that up to 70% of splice sites from annotated protein-coding transcripts outside of MANE exhibit conservation patterns closer to random sequence as opposed to highly-conserved splice sites from MANE. Our study highlights potentially erroneous splice sites that might require further scrutiny.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.26.568742">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.26.568742" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.26.568742">
        <p class="paperTitle">Learning sequence, structure, and function representations of proteins with language models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.26.568742" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.26.568742" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hamamsy, T.; Barot, M.; Morton, J. T.; Steinegger, M.; Bonneau, R.; Cho, K.</p>
        <p class="info">Score: 34.3, Published: 2023-11-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.26.568742' target='https://doi.org/10.1101/2023.11.26.568742'> 10.1101/2023.11.26.568742</a></p>
        <p class="abstract">The sequence-structure-function relationships that ultimately generate the diversity of extant observed proteins is complex, as proteins bridge the gap between multiple informational and physical scales involved in nearly all cellular processes. One limitation of existing protein annotation databases such as UniProt is that less than 1% of proteins have experimentally verified functions, and computational methods are needed to fill in the missing information. Here, we demonstrate that a multi-aspect framework based on protein language models can learn sequence-structure-function representations of amino acid sequences, and can provide the foundation for sensitive sequence-structure-function aware protein sequence search and annotation. Based on this model, we introduce a multi-aspect information retrieval system for proteins, Protein-Vec, covering sequence, structure, and function aspects, that enables computational protein annotation and function prediction at tree-of-life scales.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.569069">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.569069" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.569069">
        <p class="paperTitle">pyaging: a Python-based compendium of GPU-optimized aging clocks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.569069" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.569069" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: de Lima Camillo, L. P.</p>
        <p class="info">Score: 12.0, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.569069' target='https://doi.org/10.1101/2023.11.28.569069'> 10.1101/2023.11.28.569069</a></p>
        <p class="abstract">MotivationAging is intricately linked to diseases and mortality and is reflected in molecular changes across various tissues. The development and refinement of biomarkers of aging, healthspan, and lifespan using machine learning models, known as aging clocks, leverage epigenetic and other molecular signatures. Despite advancements, as noted by the Biomarkers of Aging Consortium, the field grapples with challenges, notably the lack of robust software tools for integrating and comparing these diverse models.

ResultsI introduce pyaging, a comprehensive Python package, designed to bridge the gap in aging research software tools. pyaging integrates over 30 aging clocks, with plans to expand to more than 100, covering a range of molecular data types including DNA methylation, transcriptomics, histone mark ChIP-Seq, and ATAC-Seq. The package features a variety of model types, from linear and principal component models to neural networks and automatic relevance determination models. Utilizing a PyTorch-based backend for GPU acceleration, pyaging ensures rapid inference even with large datasets and complex models. The package supports multi-species analysis, currently including humans, various mammals, and C. elegans.

Availability and Implementationpyaging is accessible at https://github.com/rsinghlab/pyaging. The package is structured to facilitate ease of use and integration into existing research workflows, supporting the flexible anndata data format.

Supplementary InformationSupplementary materials, including detailed documentation and usage examples, are available online at the pyaging documentation site (https://pyaging.readthedocs.io/en/latest/index.html).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.02.569717">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.02.569717" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.02.569717">
        <p class="paperTitle">Benchmarking computational methods to identify spatially variable genes and peaks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.02.569717" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.02.569717" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, Z.; Patel, Z. M.; Song, D.; Yan, G.; Li, J. J.; Pinello, L.</p>
        <p class="info">Score: 12.0, Published: 2023-12-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.02.569717' target='https://doi.org/10.1101/2023.12.02.569717'> 10.1101/2023.12.02.569717</a></p>
        <p class="abstract">Spatially resolved transcriptomics offers unprecedented insight by enabling the profiling of gene expression within the intact spatial context of cells, effectively adding a new and essential dimension to data interpretation. To efficiently detect spatial structure of interest, an essential step in analyzing such data involves identifying spatially variable genes. Despite researchers having developed several computational methods to accomplish this task, the lack of a comprehensive benchmark evaluating their performance remains a considerable gap in the field. Here, we present a systematic evaluation of 14 methods using 60 simulated datasets generated by four different simulation strategies, 12 real-world transcriptomics, and three spatial ATAC-seq datasets. We find that spatialDE2 consistently outperforms the other benchmarked methods, and Morans I achieves competitive performance in different experimental settings. Moreover, our results reveal that more specialized algorithms are needed to identify spatially variable peaks.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569114">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569114" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569114">
        <p class="paperTitle">HyGAnno: Hybrid graph neural network-based cell type annotation for single-cell ATAC sequencing data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569114" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569114" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, W.; Cui, Y.; Liu, B.; Loza Lopez, M.; Park, S.-J.; Nakai, K.</p>
        <p class="info">Score: 11.8, Published: 2023-12-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569114' target='https://doi.org/10.1101/2023.11.29.569114'> 10.1101/2023.11.29.569114</a></p>
        <p class="abstract">Reliable cell type annotations are crucial for investigating cellular heterogeneity in single-cell omics data. Although various computational approaches have been proposed for single-cell RNA sequencing (scRNA-seq) annotation, high-quality cell labels are still lacking in single-cell ATAC sequencing (scATAC-seq) data, because of extreme sparsity and inconsistent chromatin accessibility between datasets. This calls for novel cell type annotation methods in scATAC-seq, to better explore cell type-specific gene regulatory mechanisms and provide a complementary epigenomic layer to scRNA-seq data. Here, we present a novel automated cell annotation method that transfers cell type information from a well-labeled scRNA-seq reference to an unlabeled scATAC-seq target, via a parallel graph neural network, in a semi-supervised manner. Unlike existing methods that utilize only gene expression or gene activity features, HyGAnno integrates genome-wide accessibility peak features to facilitate the training process. In addition, HyGAnno reconstructs a reference-target cell graph that can be used to detect cells with low prediction reliability, according to their specific graph connectivity patterns. HyGAnno was tested using large datasets and demonstrated the advantages of accurate cell annotation, interpretable cell embedding, robustness to noisy reference data, and adaptability to tumor tissues.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.24.568570">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.24.568570" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.24.568570">
        <p class="paperTitle">Benchmarking active learning protocols for ligand binding affinity prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.24.568570" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.24.568570" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gorantla, R.; Kubincova, A.; Suutari, B.; Cossins, B. P.; Mey, A.</p>
        <p class="info">Score: 15.4, Published: 2023-11-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.24.568570' target='https://doi.org/10.1101/2023.11.24.568570'> 10.1101/2023.11.24.568570</a></p>
        <p class="abstract">Active learning (AL) has become a powerful tool in computational drug discovery, enabling the identification of top binders from vast molecular libraries with reduced costs for relative binding free energy calculations and experiments. To design a robust AL protocol, it is important to understand the influence of AL parameters, as well as the features of the datasets on the outcomes. We use four affinity datasets for different targets (TYK2, USP7, D2R, Mpro) to systematically evaluate the performance of machine learning models (Gaussian Process model, Chemprop), sample selection protocols, as well as the batch size based on metrics describing the overall predictive power of the model (R2, Spearman rank, RMSE) as well as the accurate identification of top 2% / 5% binders (Recall, F1 score). Both models have a comparable Recall of top binders on large datasets, but the GP models surpass Chemprop when training data is sparse. A larger initial batch size, especially on diverse datasets, increased the Recall of both models as well as overall correlation metrics. However, for subsequent cycles, smaller batch sizes of 20 or 30 compounds proved to be desirable. Furthermore, the presence of Gaussian noise to the data, up to a certain threshold, still allowed the model to identify clusters with top-scoring compounds. However, excessive noise (&lt;1{sigma}) did impact the models predictive and exploitative capabilities.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=111 SRC=&#34;FIGDIR/small/568570v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (28K):
org.highwire.dtl.DTLVardef@cefb11org.highwire.dtl.DTLVardef@c522bcorg.highwire.dtl.DTLVardef@6b8f18org.highwire.dtl.DTLVardef@17f7706_HPS_FORMAT_FIGEXP  M_FIG O_FLOATNOTOC GraphicC_FLOATNO C_FIG</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
