<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="GPN-MSA: an alignment-based DNA language model for genome-wide variant effect prediction
Authors: Benegas, G.; Albors, C.; Aw, A. J.; Ye, C.; Song, Y. S.
Score: 62.3, Published: 2023-10-11 DOI: 10.1101/2023.10.10.561776
Whereas protein language models have demonstrated remarkable efficacy in predicting the effects of missense variants, DNA counterparts have not yet achieved a similar competitive edge for genome-wide variant effect predictions, especially in complex genomes such as that of humans. To address this challenge, we here introduce GPN-MSA, a novel framework for DNA language models that leverages whole-genome sequence alignments across multiple species and takes only a few hours to train.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="GPN-MSA: an alignment-based DNA language model for genome-wide variant effect prediction
Authors: Benegas, G.; Albors, C.; Aw, A. J.; Ye, C.; Song, Y. S.
Score: 62.3, Published: 2023-10-11 DOI: 10.1101/2023.10.10.561776
Whereas protein language models have demonstrated remarkable efficacy in predicting the effects of missense variants, DNA counterparts have not yet achieved a similar competitive edge for genome-wide variant effect predictions, especially in complex genomes such as that of humans. To address this challenge, we here introduce GPN-MSA, a novel framework for DNA language models that leverages whole-genome sequence alignments across multiple species and takes only a few hours to train." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-18T10:38:41+00:00" />
<meta property="article:modified_time" content="2023-10-18T10:38:41+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="GPN-MSA: an alignment-based DNA language model for genome-wide variant effect prediction
Authors: Benegas, G.; Albors, C.; Aw, A. J.; Ye, C.; Song, Y. S.
Score: 62.3, Published: 2023-10-11 DOI: 10.1101/2023.10.10.561776
Whereas protein language models have demonstrated remarkable efficacy in predicting the effects of missense variants, DNA counterparts have not yet achieved a similar competitive edge for genome-wide variant effect predictions, especially in complex genomes such as that of humans. To address this challenge, we here introduce GPN-MSA, a novel framework for DNA language models that leverages whole-genome sequence alignments across multiple species and takes only a few hours to train."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "GPN-MSA: an alignment-based DNA language model for genome-wide variant effect prediction\nAuthors: Benegas, G.; Albors, C.; Aw, A. J.; Ye, C.; Song, Y. S.\nScore: 62.3, Published: 2023-10-11 DOI: 10.1101/2023.10.10.561776\nWhereas protein language models have demonstrated remarkable efficacy in predicting the effects of missense variants, DNA counterparts have not yet achieved a similar competitive edge for genome-wide variant effect predictions, especially in complex genomes such as that of humans. To address this challenge, we here introduce GPN-MSA, a novel framework for DNA language models that leverages whole-genome sequence alignments across multiple species and takes only a few hours to train.",
  "keywords": [
    
  ],
  "articleBody": " GPN-MSA: an alignment-based DNA language model for genome-wide variant effect prediction\nAuthors: Benegas, G.; Albors, C.; Aw, A. J.; Ye, C.; Song, Y. S.\nScore: 62.3, Published: 2023-10-11 DOI: 10.1101/2023.10.10.561776\nWhereas protein language models have demonstrated remarkable efficacy in predicting the effects of missense variants, DNA counterparts have not yet achieved a similar competitive edge for genome-wide variant effect predictions, especially in complex genomes such as that of humans. To address this challenge, we here introduce GPN-MSA, a novel framework for DNA language models that leverages whole-genome sequence alignments across multiple species and takes only a few hours to train. Across several benchmarks on clinical databases (ClinVar, COSMIC, and OMIM) and population genomic data (gnomAD), our model for the human genome achieves outstanding performance on deleteriousness prediction for both coding and non-coding variants.\nMapping the topography of spatial gene expression with interpretable deep learning\nAuthors: Chitra, U.; Arnold, B. J.; Sarkar, H.; Ma, C.; Lopez-Darwin, S.; Sanno, K.; Raphael, B. J.\nScore: 33.7, Published: 2023-10-13 DOI: 10.1101/2023.10.10.561757\nSpatially resolved transcriptomics technologies provide high-throughput measurements of gene expression in a tissue slice, but the sparsity of this data complicates the analysis of spatial gene expression patterns such as gene expression gradients. We address these issues by deriving a topographic map of a tissue slice-analogous to a map of elevation in a landscape-using a novel quantity called the isodepth. Contours of constant isodepth enclose spatial domains with distinct cell type composition, while gradients of the isodepth indicate spatial directions of maximum change in gene expression. We develop GASTON, an unsupervised and interpretable deep learning algorithm that simultaneously learns the isodepth, spatial gene expression gradients, and piecewise linear functions of the isodepth that model both continuous gradients and discontinuous spatial variation in the expression of individual genes. We validate GASTON by showing that it accurately identifies spatial domains and marker genes across several biological systems. In SRT data from the brain, GASTON reveals gradients of neuronal differentiation and firing, and in SRT data from a tumor sample, GASTON infers gradients of metabolic activity and epithelial-mesenchymal transition (EMT)-related gene expression in the tumor microenvironment\nA 5' UTR Language Model for Decoding Untranslated Regions of mRNA and Function Predictions\nAuthors: Chu, Y.; Yu, D.; Li, Y.; Huang, K.; Shen, Y.; Cong, L.; Zhang, J.; Wang, M.\nScore: 32.7, Published: 2023-10-14 DOI: 10.1101/2023.10.11.561938\nThe 5' UTR, a regulatory region at the beginning of an mRNA molecule, plays a crucial role in regulating the translation process and impacts the protein expression level. Language models have showcased their effectiveness in decoding the functions of protein and genome sequences. Here, we introduced a language model for 5' UTR, which we refer to as the UTR-LM. The UTR-LM is pre-trained on endogenous 5' UTRs from multiple species and is further augmented with supervised information including secondary structure and minimum free energy. We fine-tuned the UTR-LM in a variety of downstream tasks. The model outperformed the best-known benchmark by up to 42% for predicting the Mean Ribosome Loading, and by up to 60% for predicting the Translation Efficiency and the mRNA Expression Level. The model also applies to identifying unannotated Internal Ribosome Entry Sites within the untranslated region and improves the AUPR from 0.37 to 0.52 compared to the best baseline. Further, we designed a library of 211 novel 5' UTRs with high predicted values of translation efficiency and evaluated them via a wet-lab assay. Experiment results confirmed that our top designs achieved a 32.5% increase in protein production level relative to well-established 5' UTR optimized for therapeutics.\nUncover spatially informed shared variations for single-cell spatial transcriptomics with STew\nAuthors: Guo, N.; Vargas, J.; Fritz, D.; Krishna, R.; Zhang, F.\nScore: 30.3, Published: 2023-10-13 DOI: 10.1101/2023.10.10.561789\nMotivation: The recent spatial transcriptomics (ST) technologies have enabled characterization of gene expression patterns and spatial information, advancing our understanding of cell lineages within diseased tissues. Several analytical approaches have been proposed for ST data, but effectively utilizing spatial information to unveil the shared variation with gene expression remains a challenge. Results: We introduce STew, a Spatial Transcriptomic multi-viEW representative learning method, to jointly analyze spatial information and gene expression in a scalable manner, followed by a data-driven statistical framework to measure the goodness of model fit. Through benchmarking using Human DLPFC data with true manual annotations, STew achieved superior performance in both clustering accuracy and continuity of identified spatial domains compared with other methods. STew is also robust to generate consistent results insensitive to model parameters, including sparsity constraints. We next applied STew to various ST data acquired from 10x Visium and Slide-seqV2, encompassing samples from both mouse and human brain, which revealed spatially informed cell type clusters. We further identified a pro-inflammatory fibroblast spatial niche using ST data from psoriatic skins. Hence, STew is a generalized method to identify both spatially informed clusters and disease-relevant niches in complex tissues. Availability: Source code and the R software tool STew are available from github.com/fanzhanglab/STew.\nOn the relationship between protist metabarcoding and protist metagenome-assembled genomes\nAuthors: Zavadska, D.; Henry, N.; Auladell, A.; Berney, C.; Richter, D. J.\nScore: 29.9, Published: 2023-10-10 DOI: 10.1101/2023.10.09.561583\nAO_SCPLOWBSTRACTC_SCPLOWThe two most commonly used approaches to study the composition of environmental protist communities are marker gene metabarcoding and whole genome analysis through metagenomics. Raw metabarcoding data are usually processed into Operational Taxonomic Units (OTUs) or amplicon sequence variants (ASVs) through clustering or denoising approaches, respectively. Analogous approaches have been developed to assemble metagenomic sequence reads into metagenome-assembled genomes (MAGs). Understanding the correspondence between the data produced by these two approaches can help to integrate information between the datasets and to explain how metabarcoding OTUs and MAGs are related with the underlying biological entities they are hypothesised to represent. Due to the nature of their construction, MAGs do not contain the most commonly used barcoding loci, meaning that sequence homology approaches cannot be used to match OTUs and MAGs. We made an attempt to match V9 metabarcoding OTUs from the 18S rRNA gene (V9 OTUs) and MAGs from the Tara Oceans expedition (2009-2013) based on the correspondence of their relative abundances across the same set of samples. We evaluated the performance of several methods for detecting correspondence between features in these two compositional datasets and developed a series of controls to filter artefacts of data structure and processing. After selecting the best-performing correspondence metrics, ranking the V9 OTU/MAG matches by their proportionality/correlation coefficients and applying a set of selection criteria, we identified candidate matches between V9 OTUs and MAGs. In a subset of cases, V9 OTUs and MAGs could be successfully matched with one another with a one-to-one correspondence, implying that they likely represent the same underlying biological entity. More generally, matches we observed could be classified into 4 scenarios: Scenario I - one V9 OTU matches more than one MAG; Scenario II - more than one V9 OTU matches more than one MAG; Scenario III - more than one V9 OTU matches one MAG; Scenario IV - one V9 OTU matches one MAG. These diverse scenarios for V9 OTU-MAG matches illustrate the complex nature of the OTU/MAG relationship. Notably, we found some instances in which different OTU-MAG matches from the same taxonomic group were not classified in the same scenario, with all four scenarios possible even within the same taxonomic group, illustrating that factors beyond taxonomic lineage influence the relationship between OTUs and MAGs. Overall, each scenario produces a different interpretation of V9 OTUs, MAGs and how they compare in terms of the genomic and ecological diversity that they represent.\nScaling cross-tissue single-cell annotation models\nAuthors: Fischer, F.; Fischer, D. S.; Biederstedt, E.; Villani, A.-C.; Theis, F. J.\nScore: 23.2, Published: 2023-10-10 DOI: 10.1101/2023.10.07.561331\nIdentifying cellular identities (both novel and well-studied) is one of the key use cases in single-cell transcriptomics. While supervised machine learning has been leveraged to automate cell annotation predictions for some time, there has been relatively little progress both in scaling neural networks to large data sets and in constructing models that generalize well across diverse tissues and biological contexts up to whole organisms. Here, we propose scTab, an automated, feature-attention-based cell type prediction model specific to tabular data, and train it using a novel data augmentation scheme across a large corpus of single-cell RNA-seq observations (22.2 million human cells in total). In addition, scTab leverages deep ensembles for uncertainty quantification. Moreover, we account for ontological relationships between labels in the model evaluation to accommodate for differences in annotation granularity across datasets. On this large-scale corpus, we show that cross-tissue annotation requires nonlinear models and that the performance of scTab scales in terms of training dataset size as well as model size - demonstrating the advantage of scTab over current state-of-the-art linear models in this context. Additionally, we show that the proposed data augmentation schema improves model generalization. In summary, we introduce a de novo cell type prediction model for single-cell RNA-seq data that can be trained across a large-scale collection of curated datasets from a diverse selection of human tissues and demonstrate the benefits of using deep learning methods in this paradigm. Our codebase, training data, and model checkpoints are publicly available at https://github.com/theislab/scTab to further enable rigorous benchmarks of foundation models for single-cell RNA-seq data.\nComputational design of soluble analogues of integral membrane protein structures\nAuthors: Goverde, C. A.; Pacesa, M.; Dornfeld, L. J.; Goldbach, N.; Georgeon, S.; Rosset, S.; Dauparas, J.; Shellhaas, C.; Kozlov, S.; Baker, D.; Ovchinnikov, S.; Correia, B.\nScore: 67.2, Published: 2023-10-12 DOI: 10.1101/2023.05.09.540044\nDe novo design of complex protein folds using solely computational means remains a significant challenge. Here, we use a robust deep learning pipeline to design complex folds and soluble analogues of integral membrane proteins. Unique membrane topologies, such as the GPCRs, are not found in the soluble proteome and we demonstrate that their structural features can be recapitulated in solution. Biophysical analyses reveal high thermal stability of the designs and experimental structures show remarkable design accuracy. Transplantation of native structural motifs demonstrates their potential for further functionalization as novel small molecule binders and new approaches to drug discovery. In summary, our workflow enables the design of complex protein topologies with high experimental success rates and low sequence similarity to natural proteins, leading to a de facto expansion of the soluble fold space.\nCUDASW++4.0: Ultra-fast GPU-based Smith-Waterman Protein Sequence Database Search\nAuthors: Schmidt, B.; Kallenborn, F.; Chacon, A.; Hundt, C.\nScore: 15.4, Published: 2023-10-12 DOI: 10.1101/2023.10.09.561526\nBackgroundThe maximal sensitivity for local pairwise alignment makes the Smith-Waterman algorithm a popular choice for protein sequence database search. However, its quadratic time complexity makes it compute-intensive. Unfortunately, current state-of-the-art software tools are not able to leverage the massively parallel processing capabilities of modern GPUs with close-to-peak performance. This motivates the need for more efficient implementations. ResultsCUDASW++4.0 is a fast software tool for scanning protein sequence databases with the Smith-Waterman algorithm on CUDA-enabled GPUs. Our approach achieves high efficiency for dynamic programming-based alignment computation by minimizing memory accesses and instructions. We provide both efficient matrix tiling, and sequence database partitioning schemes, and exploit next generation floating point arithmetic and novel DPX instructions. This leads to close-to-peak performance on modern GPU generations (Ampere, Ada, Hopper) with throughput rates of up to 1.94 TCUPS, 5.01 TCUPS, 5.71 TCUPS on an A100, L40S, and H100, respectively. Evaluation on the Swiss-Prot, UniRef50, and TrEMBL databases shows that CUDASW++4.0 gains over an order-of-magnitude performance improvements over previous GPU-based approaches (CUDASW++3.0, ADEPT). In addition, our algorithm demonstrates significant speedups over top-performing CPU-based tools (BLASTP, SWIPE, SWIMM2.0), can exploit multi-GPU nodes with linear scaling, and features an impressive energy efficiency of up to 15.7 GCUPS/Watt. ConclusionCUDASW++4.0 changes the standing of GPUs in protein sequence database search with Smith-Waterman alignment by providing close-to-peak performance on modern GPUs. It is freely available at https://github.com/asbschmidt/CUDASW4.\nILIAD: A suite of automated Snakemake workflows for processing genomic data for downstream applications\nAuthors: Herrick, N.; Walsh, S.\nScore: 15.1, Published: 2023-10-16 DOI: 10.1101/2023.10.11.561910\nBackground: Processing raw genomic data for downstream applications such as imputation, association studies, and modeling requires numerous third-party bioinformatics software tools. It is highly time-consuming and resource-intensive with computational demands and storage limitations that pose significant challenges that increase cost. The use of software tools independent of one another, in a disjointed stepwise fashion, increases the difficulty and sets forth higher error rates because of fragmented job executions in alignment, variant calling, and/or build conversion complications. As sequencing data availability grows, the ability of biologists to process it using stable, automated, and reproducible workflows is paramount as it significantly reduces the time to generate clean and reliable data. Results: The Iliad suite of genomic data workflows was developed to provide users with seamless file transitions from raw genomic data to a quality-controlled variant call format (VCF) file for downstream applications. Iliad benefits from the efficiency of the Snakemake best practices framework coupled with Singularity and Docker containers for repeatability, portability, and ease of installation. This feat is accomplished from the onset with download acquisitions of any raw data type (FASTQ, CRAM, IDAT) straight through to the generation of a clean merged data file that can combine any user-preferred datasets using robust programs such as BWA, Samtools, and BCFtools. Users can customize and direct their workflow with one straightforward configuration file. Iliad is compatible with Linux, MacOS, and Windows platforms and scalable from a local machine to a high-performance computing cluster. Conclusion: Iliad offers automated workflows with optimized time and resource management that are comparable to other workflows available but generates analysis-ready VCF files from the most common datatypes using a single command. The storage footprint challenge of genomic data is overcome by utilizing temporary intermediate files before the final VCF is generated. This file is ready for use in imputation, genome-wide association study (GWAS) pipelines, high-throughput population genetics studies, select gene candidate studies, and more. Iliad was developed to be portable, compatible, scalable, robust, and repeatable with a simplistic setup, so biologists who are less familiar with programming can manage their own big data with this open-source suite of workflows.\nTEclass2: Classification of transposable elements using Transformers\nAuthors: Bickmann, L.; Rodriguez, M.; Jiang, X.; Makalowski, W.\nScore: 10.1, Published: 2023-10-16 DOI: 10.1101/2023.10.13.562246\nMotivation: Transposable elements (TEs) are interspersed repetitive sequences that are major constituents of most eukaryotic genomes and are crucial for genome evolution. Despite the existence of multiple tools for their classification and annotation, none of them can achieve completely reliable results making it a challenge for genomic studies. In this work, we introduce TEclass2, a new software that uses a deep learning approach based upon a linear Transformer architecture with a k-mer tokenizer and further adaptations to handle DNA sequences. This software has an easy configuration that allows training models on new datasets and the classification of TE models providing multiple metrics for a reliable evaluation of the results. Results: This work shows a successful adaptation of deep learning with Transformers for the classification of TE models from consensus sequences, and these results lay a foundation for novel methodologies in bioinformatics. We provide a tool for the training of models and the classification of consensus sequences from TE models on custom data and a web page interface with a pre-trained dataset based on curated and non-curated TE libraries allowing a fast and simple classifica-tion of TEs. Availability: https://bioinformatics.uni-muenster.de/tools/teclass2/index.pl\n",
  "wordCount" : "2537",
  "inLanguage": "en",
  "datePublished": "2023-10-18T10:38:41Z",
  "dateModified": "2023-10-18T10:38:41Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on October 18, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561776">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561776" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561776">
        <p class="paperTitle">GPN-MSA: an alignment-based DNA language model for genome-wide variant effect prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561776" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561776" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Benegas, G.; Albors, C.; Aw, A. J.; Ye, C.; Song, Y. S.</p>
        <p class="info">Score: 62.3, Published: 2023-10-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561776' target='https://doi.org/10.1101/2023.10.10.561776'> 10.1101/2023.10.10.561776</a></p>
        <p class="abstract">Whereas protein language models have demonstrated remarkable efficacy in predicting the effects of missense variants, DNA counterparts have not yet achieved a similar competitive edge for genome-wide variant effect predictions, especially in complex genomes such as that of humans. To address this challenge, we here introduce GPN-MSA, a novel framework for DNA language models that leverages whole-genome sequence alignments across multiple species and takes only a few hours to train. Across several benchmarks on clinical databases (ClinVar, COSMIC, and OMIM) and population genomic data (gnomAD), our model for the human genome achieves outstanding performance on deleteriousness prediction for both coding and non-coding variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561757">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561757" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561757">
        <p class="paperTitle">Mapping the topography of spatial gene expression with interpretable deep learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561757" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561757" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chitra, U.; Arnold, B. J.; Sarkar, H.; Ma, C.; Lopez-Darwin, S.; Sanno, K.; Raphael, B. J.</p>
        <p class="info">Score: 33.7, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561757' target='https://doi.org/10.1101/2023.10.10.561757'> 10.1101/2023.10.10.561757</a></p>
        <p class="abstract">Spatially resolved transcriptomics technologies provide high-throughput measurements of gene expression in a tissue slice, but the sparsity of this data complicates the analysis of spatial gene expression patterns such as gene expression gradients. We address these issues by deriving a topographic map of a tissue slice-analogous to a map of elevation in a landscape-using a novel quantity called the isodepth. Contours of constant isodepth enclose spatial domains with distinct cell type composition, while gradients of the isodepth indicate spatial directions of maximum change in gene expression. We develop GASTON, an unsupervised and interpretable deep learning algorithm that simultaneously learns the isodepth, spatial gene expression gradients, and piecewise linear functions of the isodepth that model both continuous gradients and discontinuous spatial variation in the expression of individual genes. We validate GASTON by showing that it accurately identifies spatial domains and marker genes across several biological systems. In SRT data from the brain, GASTON reveals gradients of neuronal differentiation and firing, and in SRT data from a tumor sample, GASTON infers gradients of metabolic activity and epithelial-mesenchymal transition (EMT)-related gene expression in the tumor microenvironment</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.11.561938">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.11.561938" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.11.561938">
        <p class="paperTitle">A 5&#39; UTR Language Model for Decoding Untranslated Regions of mRNA and Function Predictions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.11.561938" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.11.561938" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chu, Y.; Yu, D.; Li, Y.; Huang, K.; Shen, Y.; Cong, L.; Zhang, J.; Wang, M.</p>
        <p class="info">Score: 32.7, Published: 2023-10-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.11.561938' target='https://doi.org/10.1101/2023.10.11.561938'> 10.1101/2023.10.11.561938</a></p>
        <p class="abstract">The 5&#39; UTR, a regulatory region at the beginning of an mRNA molecule, plays a crucial role in regulating the translation process and impacts the protein expression level. Language models have showcased their effectiveness in decoding the functions of protein and genome sequences. Here, we introduced a language model for 5&#39; UTR, which we refer to as the UTR-LM. The UTR-LM is pre-trained on endogenous 5&#39; UTRs from multiple species and is further augmented with supervised information including secondary structure and minimum free energy. We fine-tuned the UTR-LM in a variety of downstream tasks. The model outperformed the best-known benchmark by up to 42% for predicting the Mean Ribosome Loading, and by up to 60% for predicting the Translation Efficiency and the mRNA Expression Level. The model also applies to identifying unannotated Internal Ribosome Entry Sites within the untranslated region and improves the AUPR from 0.37 to 0.52 compared to the best baseline. Further, we designed a library of 211 novel 5&#39; UTRs with high predicted values of translation efficiency and evaluated them via a wet-lab assay. Experiment results confirmed that our top designs achieved a 32.5% increase in protein production level relative to well-established 5&#39; UTR optimized for therapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561789">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561789" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561789">
        <p class="paperTitle">Uncover spatially informed shared variations for single-cell spatial transcriptomics with STew</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561789" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561789" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guo, N.; Vargas, J.; Fritz, D.; Krishna, R.; Zhang, F.</p>
        <p class="info">Score: 30.3, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561789' target='https://doi.org/10.1101/2023.10.10.561789'> 10.1101/2023.10.10.561789</a></p>
        <p class="abstract">Motivation: The recent spatial transcriptomics (ST) technologies have enabled characterization of gene expression patterns and spatial information, advancing our understanding of cell lineages within diseased tissues. Several analytical approaches have been proposed for ST data, but effectively utilizing spatial information to unveil the shared variation with gene expression remains a challenge. Results: We introduce STew, a Spatial Transcriptomic multi-viEW representative learning method, to jointly analyze spatial information and gene expression in a scalable manner, followed by a data-driven statistical framework to measure the goodness of model fit. Through benchmarking using Human DLPFC data with true manual annotations, STew achieved superior performance in both clustering accuracy and continuity of identified spatial domains compared with other methods. STew is also robust to generate consistent results insensitive to model parameters, including sparsity constraints. We next applied STew to various ST data acquired from 10x Visium and Slide-seqV2, encompassing samples from both mouse and human brain, which revealed spatially informed cell type clusters. We further identified a pro-inflammatory fibroblast spatial niche using ST data from psoriatic skins. Hence, STew is a generalized method to identify both spatially informed clusters and disease-relevant niches in complex tissues. Availability: Source code and the R software tool STew are available from github.com/fanzhanglab/STew.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561583">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561583" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561583">
        <p class="paperTitle">On the relationship between protist metabarcoding and protist metagenome-assembled genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561583" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561583" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zavadska, D.; Henry, N.; Auladell, A.; Berney, C.; Richter, D. J.</p>
        <p class="info">Score: 29.9, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561583' target='https://doi.org/10.1101/2023.10.09.561583'> 10.1101/2023.10.09.561583</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWThe two most commonly used approaches to study the composition of environmental protist communities are marker gene metabarcoding and whole genome analysis through metagenomics. Raw metabarcoding data are usually processed into Operational Taxonomic Units (OTUs) or amplicon sequence variants (ASVs) through clustering or denoising approaches, respectively. Analogous approaches have been developed to assemble metagenomic sequence reads into metagenome-assembled genomes (MAGs). Understanding the correspondence between the data produced by these two approaches can help to integrate information between the datasets and to explain how metabarcoding OTUs and MAGs are related with the underlying biological entities they are hypothesised to represent. Due to the nature of their construction, MAGs do not contain the most commonly used barcoding loci, meaning that sequence homology approaches cannot be used to match OTUs and MAGs.

We made an attempt to match V9 metabarcoding OTUs from the 18S rRNA gene (V9 OTUs) and MAGs from the Tara Oceans expedition (2009-2013) based on the correspondence of their relative abundances across the same set of samples. We evaluated the performance of several methods for detecting correspondence between features in these two compositional datasets and developed a series of controls to filter artefacts of data structure and processing. After selecting the best-performing correspondence metrics, ranking the V9 OTU/MAG matches by their proportionality/correlation coefficients and applying a set of selection criteria, we identified candidate matches between V9 OTUs and MAGs.

In a subset of cases, V9 OTUs and MAGs could be successfully matched with one another with a one-to-one correspondence, implying that they likely represent the same underlying biological entity. More generally, matches we observed could be classified into 4 scenarios: Scenario I - one V9 OTU matches more than one MAG; Scenario II - more than one V9 OTU matches more than one MAG; Scenario III - more than one V9 OTU matches one MAG; Scenario IV - one V9 OTU matches one MAG.

These diverse scenarios for V9 OTU-MAG matches illustrate the complex nature of the OTU/MAG relationship. Notably, we found some instances in which different OTU-MAG matches from the same taxonomic group were not classified in the same scenario, with all four scenarios possible even within the same taxonomic group, illustrating that factors beyond taxonomic lineage influence the relationship between OTUs and MAGs. Overall, each scenario produces a different interpretation of V9 OTUs, MAGs and how they compare in terms of the genomic and ecological diversity that they represent.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.07.561331">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.07.561331" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.07.561331">
        <p class="paperTitle">Scaling cross-tissue single-cell annotation models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.07.561331" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.07.561331" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fischer, F.; Fischer, D. S.; Biederstedt, E.; Villani, A.-C.; Theis, F. J.</p>
        <p class="info">Score: 23.2, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.07.561331' target='https://doi.org/10.1101/2023.10.07.561331'> 10.1101/2023.10.07.561331</a></p>
        <p class="abstract">Identifying cellular identities (both novel and well-studied) is one of the key use cases in single-cell transcriptomics. While supervised machine learning has been leveraged to automate cell annotation predictions for some time, there has been relatively little progress both in scaling neural networks to large data sets and in constructing models that generalize well across diverse tissues and biological contexts up to whole organisms. Here, we propose scTab, an automated, feature-attention-based cell type prediction model specific to tabular data, and train it using a novel data augmentation scheme across a large corpus of single-cell RNA-seq observations (22.2 million human cells in total). In addition, scTab leverages deep ensembles for uncertainty quantification. Moreover, we account for ontological relationships between labels in the model evaluation to accommodate for differences in annotation granularity across datasets. On this large-scale corpus, we show that cross-tissue annotation requires nonlinear models and that the performance of scTab scales in terms of training dataset size as well as model size - demonstrating the advantage of scTab over current state-of-the-art linear models in this context. Additionally, we show that the proposed data augmentation schema improves model generalization. In summary, we introduce a de novo cell type prediction model for single-cell RNA-seq data that can be trained across a large-scale collection of curated datasets from a diverse selection of human tissues and demonstrate the benefits of using deep learning methods in this paradigm. Our codebase, training data, and model checkpoints are publicly available at https://github.com/theislab/scTab to further enable rigorous benchmarks of foundation models for single-cell RNA-seq data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.05.09.540044">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.05.09.540044" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.05.09.540044">
        <p class="paperTitle">Computational design of soluble analogues of integral membrane protein structures</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.05.09.540044" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.05.09.540044" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Goverde, C. A.; Pacesa, M.; Dornfeld, L. J.; Goldbach, N.; Georgeon, S.; Rosset, S.; Dauparas, J.; Shellhaas, C.; Kozlov, S.; Baker, D.; Ovchinnikov, S.; Correia, B.</p>
        <p class="info">Score: 67.2, Published: 2023-10-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.05.09.540044' target='https://doi.org/10.1101/2023.05.09.540044'> 10.1101/2023.05.09.540044</a></p>
        <p class="abstract">De novo design of complex protein folds using solely computational means remains a significant challenge. Here, we use a robust deep learning pipeline to design complex folds and soluble analogues of integral membrane proteins. Unique membrane topologies, such as the GPCRs, are not found in the soluble proteome and we demonstrate that their structural features can be recapitulated in solution. Biophysical analyses reveal high thermal stability of the designs and experimental structures show remarkable design accuracy. Transplantation of native structural motifs demonstrates their potential for further functionalization as novel small molecule binders and new approaches to drug discovery. In summary, our workflow enables the design of complex protein topologies with high experimental success rates and low sequence similarity to natural proteins, leading to a de facto expansion of the soluble fold space.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561526">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561526" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561526">
        <p class="paperTitle">CUDASW&#43;&#43;4.0: Ultra-fast GPU-based Smith-Waterman Protein Sequence Database Search</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561526" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561526" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schmidt, B.; Kallenborn, F.; Chacon, A.; Hundt, C.</p>
        <p class="info">Score: 15.4, Published: 2023-10-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561526' target='https://doi.org/10.1101/2023.10.09.561526'> 10.1101/2023.10.09.561526</a></p>
        <p class="abstract">BackgroundThe maximal sensitivity for local pairwise alignment makes the Smith-Waterman algorithm a popular choice for protein sequence database search. However, its quadratic time complexity makes it compute-intensive. Unfortunately, current state-of-the-art software tools are not able to leverage the massively parallel processing capabilities of modern GPUs with close-to-peak performance. This motivates the need for more efficient implementations.

ResultsCUDASW&#43;&#43;4.0 is a fast software tool for scanning protein sequence databases with the Smith-Waterman algorithm on CUDA-enabled GPUs. Our approach achieves high efficiency for dynamic programming-based alignment computation by minimizing memory accesses and instructions. We provide both efficient matrix tiling, and sequence database partitioning schemes, and exploit next generation floating point arithmetic and novel DPX instructions. This leads to close-to-peak performance on modern GPU generations (Ampere, Ada, Hopper) with throughput rates of up to 1.94 TCUPS, 5.01 TCUPS, 5.71 TCUPS on an A100, L40S, and H100, respectively. Evaluation on the Swiss-Prot, UniRef50, and TrEMBL databases shows that CUDASW&#43;&#43;4.0 gains over an order-of-magnitude performance improvements over previous GPU-based approaches (CUDASW&#43;&#43;3.0, ADEPT). In addition, our algorithm demonstrates significant speedups over top-performing CPU-based tools (BLASTP, SWIPE, SWIMM2.0), can exploit multi-GPU nodes with linear scaling, and features an impressive energy efficiency of up to 15.7 GCUPS/Watt.

ConclusionCUDASW&#43;&#43;4.0 changes the standing of GPUs in protein sequence database search with Smith-Waterman alignment by providing close-to-peak performance on modern GPUs. It is freely available at https://github.com/asbschmidt/CUDASW4.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.11.561910">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.11.561910" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.11.561910">
        <p class="paperTitle">ILIAD: A suite of automated Snakemake workflows for processing genomic data for downstream applications</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.11.561910" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.11.561910" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Herrick, N.; Walsh, S.</p>
        <p class="info">Score: 15.1, Published: 2023-10-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.11.561910' target='https://doi.org/10.1101/2023.10.11.561910'> 10.1101/2023.10.11.561910</a></p>
        <p class="abstract">Background: Processing raw genomic data for downstream applications such as imputation, association studies, and modeling requires numerous third-party bioinformatics software tools. It is highly time-consuming and resource-intensive with computational demands and storage limitations that pose significant challenges that increase cost. The use of software tools independent of one another, in a disjointed stepwise fashion, increases the difficulty and sets forth higher error rates because of fragmented job executions in alignment, variant calling, and/or build conversion complications. As sequencing data availability grows, the ability of biologists to process it using stable, automated, and reproducible workflows is paramount as it significantly reduces the time to generate clean and reliable data. Results: The Iliad suite of genomic data workflows was developed to provide users with seamless file transitions from raw genomic data to a quality-controlled variant call format (VCF) file for downstream applications. Iliad benefits from the efficiency of the Snakemake best practices framework coupled with Singularity and Docker containers for repeatability, portability, and ease of installation. This feat is accomplished from the onset with download acquisitions of any raw data type (FASTQ, CRAM, IDAT) straight through to the generation of a clean merged data file that can combine any user-preferred datasets using robust programs such as BWA, Samtools, and BCFtools. Users can customize and direct their workflow with one straightforward configuration file. Iliad is compatible with Linux, MacOS, and Windows platforms and scalable from a local machine to a high-performance computing cluster. Conclusion: Iliad offers automated workflows with optimized time and resource management that are comparable to other workflows available but generates analysis-ready VCF files from the most common datatypes using a single command. The storage footprint challenge of genomic data is overcome by utilizing temporary intermediate files before the final VCF is generated. This file is ready for use in imputation, genome-wide association study (GWAS) pipelines, high-throughput population genetics studies, select gene candidate studies, and more. Iliad was developed to be portable, compatible, scalable, robust, and repeatable with a simplistic setup, so biologists who are less familiar with programming can manage their own big data with this open-source suite of workflows.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562246">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562246" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562246">
        <p class="paperTitle">TEclass2: Classification of transposable elements using Transformers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562246" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562246" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bickmann, L.; Rodriguez, M.; Jiang, X.; Makalowski, W.</p>
        <p class="info">Score: 10.1, Published: 2023-10-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562246' target='https://doi.org/10.1101/2023.10.13.562246'> 10.1101/2023.10.13.562246</a></p>
        <p class="abstract">Motivation: Transposable elements (TEs) are interspersed repetitive sequences that are major constituents of most eukaryotic genomes and are crucial for genome evolution. Despite the existence of multiple tools for their classification and annotation, none of them can achieve completely reliable results making it a challenge for genomic studies. In this work, we introduce TEclass2, a new software that uses a deep learning approach based upon a linear Transformer architecture with a k-mer tokenizer and further adaptations to handle DNA sequences. This software has an easy configuration that allows training models on new datasets and the classification of TE models providing multiple metrics for a reliable evaluation of the results. Results: This work shows a successful adaptation of deep learning with Transformers for the classification of TE models from consensus sequences, and these results lay a foundation for novel methodologies in bioinformatics. We provide a tool for the training of models and the classification of consensus sequences from TE models on custom data and a web page interface with a pre-trained dataset based on curated and non-curated TE libraries allowing a fast and simple classifica-tion of TEs. Availability: https://bioinformatics.uni-muenster.de/tools/teclass2/index.pl</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
