<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Exploring &#34;dark matter&#34; protein folds using deep learning
Authors: Harteveld, Z.; Van Hall-Beauvais, A.; Morozova, I.; Southern, J.; Goverde, C. A.; Georgeon, S.; Rosset, S.; Loukas, A.; Vandergheynst, P.; Bronstein, M.; Correia, B.
Score: 45.3, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555621
De novo protein design aims to explore uncharted sequence- and structure areas to generate novel proteins that have not been sampled by evolution. One of the main challenges in de novo design involves crafting designable structural templates that can guide the sequence search towards adopting the target structures.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="Exploring &#34;dark matter&#34; protein folds using deep learning
Authors: Harteveld, Z.; Van Hall-Beauvais, A.; Morozova, I.; Southern, J.; Goverde, C. A.; Georgeon, S.; Rosset, S.; Loukas, A.; Vandergheynst, P.; Bronstein, M.; Correia, B.
Score: 45.3, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555621
De novo protein design aims to explore uncharted sequence- and structure areas to generate novel proteins that have not been sampled by evolution. One of the main challenges in de novo design involves crafting designable structural templates that can guide the sequence search towards adopting the target structures." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-06T10:39:07+00:00" />
<meta property="article:modified_time" content="2023-09-06T10:39:07+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="Exploring &#34;dark matter&#34; protein folds using deep learning
Authors: Harteveld, Z.; Van Hall-Beauvais, A.; Morozova, I.; Southern, J.; Goverde, C. A.; Georgeon, S.; Rosset, S.; Loukas, A.; Vandergheynst, P.; Bronstein, M.; Correia, B.
Score: 45.3, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555621
De novo protein design aims to explore uncharted sequence- and structure areas to generate novel proteins that have not been sampled by evolution. One of the main challenges in de novo design involves crafting designable structural templates that can guide the sequence search towards adopting the target structures."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "Exploring \u0026#34;dark matter\u0026#34; protein folds using deep learning\nAuthors: Harteveld, Z.; Van Hall-Beauvais, A.; Morozova, I.; Southern, J.; Goverde, C. A.; Georgeon, S.; Rosset, S.; Loukas, A.; Vandergheynst, P.; Bronstein, M.; Correia, B.\nScore: 45.3, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555621\nDe novo protein design aims to explore uncharted sequence- and structure areas to generate novel proteins that have not been sampled by evolution. One of the main challenges in de novo design involves crafting designable structural templates that can guide the sequence search towards adopting the target structures.",
  "keywords": [
    
  ],
  "articleBody": " Exploring \"dark matter\" protein folds using deep learning\nAuthors: Harteveld, Z.; Van Hall-Beauvais, A.; Morozova, I.; Southern, J.; Goverde, C. A.; Georgeon, S.; Rosset, S.; Loukas, A.; Vandergheynst, P.; Bronstein, M.; Correia, B.\nScore: 45.3, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555621\nDe novo protein design aims to explore uncharted sequence- and structure areas to generate novel proteins that have not been sampled by evolution. One of the main challenges in de novo design involves crafting designable structural templates that can guide the sequence search towards adopting the target structures. Here, we present an approach to learn patterns of protein structure based on a convolutional variational autoencoder, dubbed Genesis. We coupled Genesis with trRosetta to design sequences for a set of protein folds and found that Genesis is capable of reconstructing native-like distance- and angle distributions for five native folds and three novel, so-called dark-matter folds as a demonstration of generalizability. We used a high-throughput assay to characterize protease resistance of the designs, obtaining encouraging success rates for folded proteins and further biochemically characterized folded designs. The Genesis framework enables the exploration of the protein sequence and fold space within minutes and is not bound to specific protein topologies. Our approach addresses the backbone designability problem, showing that structural patterns in proteins can be efficiently learned by small neural networks and could ultimately contribute to the de novo design of proteins with new functions.\nOmix: A Multi-Omics Integration Pipeline\nAuthors: Schneegans, E.; Fancy, N.; Thomas, M.; Willumsen, N.; Matthews, P. M.; Jackson, J.\nScore: 35.8, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555486\nSummary: The Omix pipeline offers an integration and analysis framework for multi-omics intended to preprocess, analyse, and visualise multimodal data flexibly to address various research questions. From biomarker discovery and patient stratification to the investigation of complex biological processes, Omix empowers researchers to derive valuable insights from omics data. Using Alzheimer's Disease (AD) bulk proteomics and transcriptomics datasets generated from two distinct regions derived from post-mortem brains, we demonstrate the utility of Omix in generating an integrated pseudo-temporal multi-omics profile of AD. Availability and Implementation: Omix is implemented as a software package in R. The code for the Omix package is available at https://github.com/eleonore- schneeg/Omix. Reference documentation and online tutorials are available at https://eleonore-schneeg.github.io/Omix. All code is open-source and available under the GNU General Public License v3.0 (GPL-3).\nFlow: a web platform and open database to analyse, store, curate and share bioinformatics data at scale\nAuthors: Capitanchik, C.; Ireland, S.; Harston, A.; Cheshire, C.; Jones, D. M.; Lee, F. C. Y.; Ruiz de los Mozos, I.; Iosub, I. A.; Kuret, K.; Faraway, R.; Wilkins, O. G.; Arora, R.; Hallegger, M.; Modic, M.; Chakrabarti, A. M.; Luscombe, N. M.; Ule, J.\nScore: 32.5, Published: 2023-08-29 DOI: 10.1101/2023.08.22.544179\nEver-increasing volumes of sequencing data offer potential for large-scale meta-analyses to address significant biological questions. However, challenges such as insufficient data processing information, data quality concerns, and issues related to accessibility and curation often present obstacles. Additionally, most experimental biologists lack the time and expertise needed to independently analyse, manage and share their own data. To overcome these hurdles, we present Flow, a web-based platform that links bioinformatic analysis and database solutions with a user-friendly interface and web API. Flow currently accommodates a range of genomics methods and further DSL2-compliant Nextflow pipelines can be added via a simple JSON schema file. Deployable on local systems or cloud services, an instance is freely accessible to academic researchers at https://flow.bio.\nHuman Ageing Genomic Resources: updates on key databases in ageing research\nAuthors: de Magalhaes, J. P.; Abidi, Z.; Arantes dos Santos, G.; Avelar, R. A.; Barardo, D.; Chatsirisupachai, K.; Clark, P.; de Souza, E.; Johnson, E. J.; Lopes, I.; Novoa, G.; Senez, L.; Talay, A.; Thornton, D.; To, P. K. P.\nScore: 18.6, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555622\nAgeing is a complex and multifactorial process. For two decades, the Human Ageing Genomic Resources (HAGR) have aided researchers in the study of various aspects of ageing and its manipulation. Here we present the key features and recent enhancements of these resources, focusing on its six main databases. One database, GenAge, focuses on genes related to ageing, featuring 307 genes linked to human ageing and 2205 genes associated with longevity and ageing in model organisms. AnAge focuses on ageing, longevity, and life-history across animal species, containing data on 4645 species. DrugAge includes information about 1097 longevity drugs and compounds in model organisms such as mice, rats, flies, worms, and yeast. GenDR provides a list of 214 genes associated with the life-extending benefits of dietary restriction in model organisms. CellAge contains a catalogue of 866 genes associated with cellular senescence. The LongevityMap serves as a repository for genetic variants associated with human longevity, encompassing 3144 variants pertaining to 884 genes. Additionally, HAGR provides various tools as well as gene expression signatures of ageing, dietary restriction, and replicative senescence based on meta-analyses. Our databases are integrated, regularly updated, and manually curated by experts. HAGR is freely available online (https://genomics.senescence.info/).\nGene count normalization in single-cell imaging-based spatially resolved transcriptomics\nAuthors: Atta, L.; Clifton, K.; Anant, M.; Fan, J.\nScore: 17.0, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555624\nRecent advances in imaging-based spatially resolved transcriptomics (im-SRT) technologies now enable high-throughput profiling of targeted genes and their locations in fixed tissues. Normalization of gene expression data is often needed to account for technical factors that may confound underlying biological signals. Here, we investigate the potential impact of different gene count normalization methods with different targeted gene panels in the analysis and interpretation of im-SRT data. Using different simulated gene panels that overrepresent genes expressed in specific tissue anatomical regions or cell types, we find that normalization methods that use scaling factors derived from gene counts differentially impact normalized gene expression magnitudes in a region- or cell type-specific manner. We show that these normalization-induced effects may reduce the reliability of downstream differential gene expression and fold change analysis, introducing false positive and false negative results when compared to results obtained from gene panels that are more representative of the gene expression of the tissues component cell types. These effects are not observed without normalization or when scaling factors are not derived from gene counts, such as with cell volume normalization. Overall, we caution that the choice of normalization method and gene panel may impact the biological interpretation of the im-SRT data.\nSandy: A user-friendly and versatile NGS simulator to facilitate sequencing assay design and optimization\nAuthors: Miller, T. L. A.; Conceicao, H. B.; Mercuri, R. L.; Santos, F. R. C.; Barreiro, R.; Buzzo, J. L.; Rego, F. O.; Guardia, G. D. A.; Galante, P. A. F.\nScore: 22.2, Published: 2023-08-28 DOI: 10.1101/2023.08.25.554791\nNext-generation sequencing (NGS) is currently the gold standard technique for large-scale genome and transcriptome studies. However, the downstream processing of NGS data is a critical bottleneck that requires difficult decisions regarding data analysis methods and parameters. Simulated or synthetic NGS datasets are practical and cost-effective alternatives for overcoming these difficulties. Simulated NGS datasets have known true values and provide a standardized scenario for driving the development of data analysis methodologies and tuning cut-off values. Although tools for simulating NGS data are available, they have limitations in terms of their overall usability and documentation. Here, we present Sandy, an open-source simulator that generates synthetic reads that mimic DNA or RNA next-generation sequencing on the Illumina, Oxford Nanopore, and Pacific Bioscience platforms. Sandy is designed to be user-friendly, computationally efficient, and capable of simulating data resembling a wide range of features of real NGS assays, including sequencing quality, genomic variations, and gene expression profiles per tissue. To demonstrate Sandys versatility, we used it to address two critical questions in designing an NGS assay: (i) How many reads should be sequenced to ensure unbiased analysis of gene expression in an RNA sequencing run? (ii) What is the lowest genome coverage required to identify most (90%) of the single nucleotide variants and structural variations in whole-genome sequencing? In summary, Sandy is an ideal tool for assessing and validating pipelines for processing, optimizing results, and defining the costs of NGS assays. Sandy runs on Linux, MacOS, and Microsoft Windows and can provide feasible results, even on personal computers. Availability: Sandy is freely available at https://galantelab.github.io/sandy.\nDe novo peptide sequencing with InstaNovo: Accurate, database-free peptide identification for large scale proteomics experiments\nAuthors: Eloff, K.; Kalogeropoulos, K.; Morell, O.; Mabona, A.; Berg Jespersen, J.; Williams, W.; van Beljouw, S.; Skwark, M.; Hougaard Laustsen, A.; Brouns, S. J. J.; Ljungars, A.; Schoof, E. M.; Van Goey, J.; auf dem Keller, U.; Beguir, K.; Lopez Carranza, N.; Jenkins, T. P.\nScore: 16.2, Published: 2023-08-31 DOI: 10.1101/2023.08.30.555055\nBottom-up mass spectrometry-based proteomics is challenged by the task of identifying the peptide that generates a tandem mass spectrum. Traditional methods that rely on known peptide sequence databases are limited and may not be applicable in certain contexts. De novo peptide sequencing, which assigns peptide sequences to the spectra without prior information, is valuable for various biological applications; yet, due to a lack of accuracy, it remains challenging to apply this approach in many situations. Here, we introduce InstaNovo, a transformer neural network with the ability to translate fragment ion peaks into the sequence of amino acids that make up the studied peptide(s). The model was trained on 28 million labelled spectra matched to 742k human peptides from the ProteomeTools project. We demonstrate that InstaNovo outperforms current state-of-the-art methods on benchmark datasets and showcase its utility in several applications. Building upon human intuition, we also introduce InstaNovo+, a multinomial diffusion model that further improves performance by iterative refinement of predicted sequences. Using these models, we could de novo sequence antibodybased therapeutics with unprecedented coverage, discover novel peptides, and detect unreported organisms in different datasets, thereby expanding the scope and detection rate of proteomics searches. Finally, we could experimentally validate tryptic and non-tryptic peptides with targeted proteomics, demonstrating the fidelity of our predictions. Our models unlock a plethora of opportunities across different scientific domains, such as direct protein sequencing, immunopeptidomics, and exploration of the dark proteome. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=130 SRC=\"FIGDIR/small/555055v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (38K): org.highwire.dtl.DTLVardef@1e7bfaborg.highwire.dtl.DTLVardef@1d89454org.highwire.dtl.DTLVardef@1f21ad2org.highwire.dtl.DTLVardef@1bedf53_HPS_FORMAT_FIGEXP M_FIG C_FIG\nHigh-accuracy mapping of human and viral direct physical protein-protein interactions using the novel computational system AlphaFold-pairs\nAuthors: Poitras, C.; Lamontagne, F.; Grandvaux, N.; Song, H.; Pinard, M.; Coulombe, B.\nScore: 15.9, Published: 2023-08-30 DOI: 10.1101/2023.08.29.555151\nProtein-protein interactions are central, highly flexible components of regulatory mechanisms in all living cells. Over the years, diverse methods have been developed to map protein-protein interactions. These methods have revealed the organization of protein complexes and networks in numerous cells and conditions. However, these methods are also time consuming, costly and sensitive to various experimental artifacts. To avoid these caveats, we have taken advantage of the AlphaFold-Multimer software, which succeeded in predicting the structure of many protein complexes. We designed a relatively simple algorithm based on assessing the physical proximity of a test protein with other AlphaFold structures. Using this method, named AlphaFold-pairs, we have successfully defined the probability of a protein-protein interaction forming. AlphaFold-pairs was validated using well-defined protein-protein interactions found in the literature and specialized databases. All pairwise interactions forming within the 12-subunit transcription machinery RNA Polymerase II, according to available structures, have been identified. Out of 66 possible interactions (excluding homodimers), 19 specific interactions have been found, and an additional previously unknown interaction has been unveiled. The SARS-CoV-2 surface glycoprotein Spike (or S) was confirmed to interact with high preference with the human ACE2 receptor when compared to other human receptors. Notably, two additional receptors, INSR and FLT4, were found to interact with S. For the first time, we have successfully identified protein-protein interactions that are likely to form within the reassortant Eurasian avian-like (EA) H1N1 swine G4 genotype Influenza A virus, which poses a potential zoonotic threat. Testing G4 proteins against human transcription factors and molecular chaperones (a total of 100 proteins) revealed strong specific interactions between the G4 HA and HSP90B1, the G4 NS and the PAQosome subunit RPAP3, as well as the G4 PA and the POLR2A subunit. We predict that AlphaFold-pairs will revolutionize the study of protein-protein interactions in a large number of healthy and diseased systems in the years to come.\nBRAKER3: Fully Automated Genome Annotation Using RNA-Seq and Protein Evidence with GeneMark-ETP, AUGUSTUS and TSEBRA\nAuthors: Gabriel, L.; Bruna, T.; Hoff, K. J.; Ebel, M.; Lomsadze, A.; Borodovsky, M.; Stanke, M.\nScore: 112.8, Published: 2023-09-02 DOI: 10.1101/2023.06.10.544449\nGene prediction remains an active area of bioinformatics research. Challenges are presented by large eukaryotic genomes and heterogeneous data situations. To meet the challenges, several streams of evidence must be integrated, from protein homology and transcriptome data, as well as information derived from the genome itself. The amount and significance of the available evidence from transcriptomes and proteomes vary from genome to genome, between genes and even along a single gene. User-friendly and accurate annotation pipelines that can cope with such data heterogeneity are needed. The previously developed annotation pipelines BRAKER1 and BRAKER2 use RNA-Seq or protein data, respectively, but not both. The recently released GeneMark-ETP integrates all three types of data and achieves much higher levels of accuracy. We here present the BRAKER3 pipeline that builds on GeneMark-ETP and AUGUSTUS and further improves accuracy using the TSEBRA combiner. BRAKER3 annotates protein-coding genes in eukaryotic genomes using both short-read RNA-Seq and a large protein database along with statistical models learned iteratively and specifically for the target genome. We benchmarked the new pipeline on 11 species under controlled conditions on the assumed relatedness of the target species to available proteomes. BRAKER3 outperformed BRAKER1 and BRAKER2, increasing the average transcript-level F1-score by ~20 percentage points, most pronounced for species with large and complex genomes. BRAKER3 also outperforms MAKER2 and Funannotate. For the first time, we provide a Singularity container for the BRAKER software to minimize installation obstacles. Overall, BRAKER3 is an accurate, easy-to-use tool for the annotation of eukaryotic genomes.\nAutoHiC: a deep-learning method for automatic and accurate chromosome-level genome assembly\nAuthors: jiang, z.; peng, z.; luo, y.; bie, l.; wang, y.\nScore: 13.9, Published: 2023-08-28 DOI: 10.1101/2023.08.27.555031\nAn accurate genome at the chromosome level is the key to unraveling the mysteries of gene function and unlocking the mechanisms of disease. Irrespective of the sequencing methodology adopted, Hi-C aided scaffolding serves as a principal avenue for generating genome assemblies at the chromosomal level. However, the results of such scaffolding are often flawed and require extensive manual refinement. In this paper, we introduce AutoHiC, an innovative deep learning-based tool designed to identify and rectify genome assembly errors. Diverging from conventional approaches, AutoHiC harnesses the power of high-dimensional Hi-C data to enhance genome continuity and accuracy through a fully automated workflow and iterative error correction mechanism. AutoHiC was trained on Hi-C data from more than 300 species (approximately five hundred thousand interaction maps) in DNA Zoo and NCBI. Its confusion matrix results show that the average error detection accuracy is over 90%, and the area under the precision-recall curve is close to 1, making it a powerful error detection capability. The benchmarking results demonstrate AutoHiCs ability to substantially enhance genome continuity and significantly reduce error rates, providing a more reliable foundation for genomics research. Furthermore, AutoHiC generates comprehensive result reports, offering users insights into the assembly process and outcomes. In summary, AutoHiC represents a breakthrough in automated error detection and correction for genome assembly, effectively promoting more accurate and comprehensive genome assemblies.\n",
  "wordCount" : "2555",
  "inLanguage": "en",
  "datePublished": "2023-09-06T10:39:07Z",
  "dateModified": "2023-09-06T10:39:07Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on September 6, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555621">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555621" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555621">
        <p class="paperTitle">Exploring &#34;dark matter&#34; protein folds using deep learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555621" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555621" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Harteveld, Z.; Van Hall-Beauvais, A.; Morozova, I.; Southern, J.; Goverde, C. A.; Georgeon, S.; Rosset, S.; Loukas, A.; Vandergheynst, P.; Bronstein, M.; Correia, B.</p>
        <p class="info">Score: 45.3, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555621' target='https://doi.org/10.1101/2023.08.30.555621'> 10.1101/2023.08.30.555621</a></p>
        <p class="abstract">De novo protein design aims to explore uncharted sequence- and structure areas to generate novel proteins that have not been sampled by evolution. One of the main challenges in de novo design involves crafting designable structural templates that can guide the sequence search towards adopting the target structures. Here, we present an approach to learn patterns of protein structure based on a convolutional variational autoencoder, dubbed Genesis. We coupled Genesis with trRosetta to design sequences for a set of protein folds and found that Genesis is capable of reconstructing native-like distance- and angle distributions for five native folds and three novel, so-called dark-matter folds as a demonstration of generalizability. We used a high-throughput assay to characterize protease resistance of the designs, obtaining encouraging success rates for folded proteins and further biochemically characterized folded designs. The Genesis framework enables the exploration of the protein sequence and fold space within minutes and is not bound to specific protein topologies. Our approach addresses the backbone designability problem, showing that structural patterns in proteins can be efficiently learned by small neural networks and could ultimately contribute to the de novo design of proteins with new functions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555486">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555486" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555486">
        <p class="paperTitle">Omix: A Multi-Omics Integration Pipeline</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555486" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555486" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schneegans, E.; Fancy, N.; Thomas, M.; Willumsen, N.; Matthews, P. M.; Jackson, J.</p>
        <p class="info">Score: 35.8, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555486' target='https://doi.org/10.1101/2023.08.30.555486'> 10.1101/2023.08.30.555486</a></p>
        <p class="abstract">Summary: The Omix pipeline offers an integration and analysis framework for multi-omics intended to preprocess, analyse, and visualise multimodal data flexibly to address various research questions. From biomarker discovery and patient stratification to the investigation of complex biological processes, Omix empowers researchers to derive valuable insights from omics data. Using Alzheimer&#39;s Disease (AD) bulk proteomics and transcriptomics datasets generated from two distinct regions derived from post-mortem brains, we demonstrate the utility of Omix in generating an integrated pseudo-temporal multi-omics profile of AD. Availability and Implementation: Omix is implemented as a software package in R. The code for the Omix package is available at https://github.com/eleonore- schneeg/Omix. Reference documentation and online tutorials are available at https://eleonore-schneeg.github.io/Omix. All code is open-source and available under the GNU General Public License v3.0 (GPL-3).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.544179">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.544179" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.544179">
        <p class="paperTitle">Flow: a web platform and open database to analyse, store, curate and share bioinformatics data at scale</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.544179" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.544179" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Capitanchik, C.; Ireland, S.; Harston, A.; Cheshire, C.; Jones, D. M.; Lee, F. C. Y.; Ruiz de los Mozos, I.; Iosub, I. A.; Kuret, K.; Faraway, R.; Wilkins, O. G.; Arora, R.; Hallegger, M.; Modic, M.; Chakrabarti, A. M.; Luscombe, N. M.; Ule, J.</p>
        <p class="info">Score: 32.5, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.544179' target='https://doi.org/10.1101/2023.08.22.544179'> 10.1101/2023.08.22.544179</a></p>
        <p class="abstract">Ever-increasing volumes of sequencing data offer potential for large-scale meta-analyses to address significant biological questions. However, challenges such as insufficient data processing information, data quality concerns, and issues related to accessibility and curation often present obstacles. Additionally, most experimental biologists lack the time and expertise needed to independently analyse, manage and share their own data. To overcome these hurdles, we present Flow, a web-based platform that links bioinformatic analysis and database solutions with a user-friendly interface and web API. Flow currently accommodates a range of genomics methods and further DSL2-compliant Nextflow pipelines can be added via a simple JSON schema file. Deployable on local systems or cloud services, an instance is freely accessible to academic researchers at https://flow.bio.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555622">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555622" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555622">
        <p class="paperTitle">Human Ageing Genomic Resources: updates on key databases in ageing research</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555622" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555622" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: de Magalhaes, J. P.; Abidi, Z.; Arantes dos Santos, G.; Avelar, R. A.; Barardo, D.; Chatsirisupachai, K.; Clark, P.; de Souza, E.; Johnson, E. J.; Lopes, I.; Novoa, G.; Senez, L.; Talay, A.; Thornton, D.; To, P. K. P.</p>
        <p class="info">Score: 18.6, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555622' target='https://doi.org/10.1101/2023.08.30.555622'> 10.1101/2023.08.30.555622</a></p>
        <p class="abstract">Ageing is a complex and multifactorial process. For two decades, the Human Ageing Genomic Resources (HAGR) have aided researchers in the study of various aspects of ageing and its manipulation. Here we present the key features and recent enhancements of these resources, focusing on its six main databases. One database, GenAge, focuses on genes related to ageing, featuring 307 genes linked to human ageing and 2205 genes associated with longevity and ageing in model organisms. AnAge focuses on ageing, longevity, and life-history across animal species, containing data on 4645 species. DrugAge includes information about 1097 longevity drugs and compounds in model organisms such as mice, rats, flies, worms, and yeast. GenDR provides a list of 214 genes associated with the life-extending benefits of dietary restriction in model organisms. CellAge contains a catalogue of 866 genes associated with cellular senescence. The LongevityMap serves as a repository for genetic variants associated with human longevity, encompassing 3144 variants pertaining to 884 genes. Additionally, HAGR provides various tools as well as gene expression signatures of ageing, dietary restriction, and replicative senescence based on meta-analyses. Our databases are integrated, regularly updated, and manually curated by experts. HAGR is freely available online (https://genomics.senescence.info/).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555624">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555624" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555624">
        <p class="paperTitle">Gene count normalization in single-cell imaging-based spatially resolved transcriptomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555624" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555624" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Atta, L.; Clifton, K.; Anant, M.; Fan, J.</p>
        <p class="info">Score: 17.0, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555624' target='https://doi.org/10.1101/2023.08.30.555624'> 10.1101/2023.08.30.555624</a></p>
        <p class="abstract">Recent advances in imaging-based spatially resolved transcriptomics (im-SRT) technologies now enable high-throughput profiling of targeted genes and their locations in fixed tissues. Normalization of gene expression data is often needed to account for technical factors that may confound underlying biological signals. Here, we investigate the potential impact of different gene count normalization methods with different targeted gene panels in the analysis and interpretation of im-SRT data. Using different simulated gene panels that overrepresent genes expressed in specific tissue anatomical regions or cell types, we find that normalization methods that use scaling factors derived from gene counts differentially impact normalized gene expression magnitudes in a region- or cell type-specific manner. We show that these normalization-induced effects may reduce the reliability of downstream differential gene expression and fold change analysis, introducing false positive and false negative results when compared to results obtained from gene panels that are more representative of the gene expression of the tissues component cell types. These effects are not observed without normalization or when scaling factors are not derived from gene counts, such as with cell volume normalization. Overall, we caution that the choice of normalization method and gene panel may impact the biological interpretation of the im-SRT data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.554791">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.554791" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.554791">
        <p class="paperTitle">Sandy: A user-friendly and versatile NGS simulator to facilitate sequencing assay design and optimization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.554791" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.554791" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Miller, T. L. A.; Conceicao, H. B.; Mercuri, R. L.; Santos, F. R. C.; Barreiro, R.; Buzzo, J. L.; Rego, F. O.; Guardia, G. D. A.; Galante, P. A. F.</p>
        <p class="info">Score: 22.2, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.554791' target='https://doi.org/10.1101/2023.08.25.554791'> 10.1101/2023.08.25.554791</a></p>
        <p class="abstract">Next-generation sequencing (NGS) is currently the gold standard technique for large-scale genome and transcriptome studies. However, the downstream processing of NGS data is a critical bottleneck that requires difficult decisions regarding data analysis methods and parameters. Simulated or synthetic NGS datasets are practical and cost-effective alternatives for overcoming these difficulties. Simulated NGS datasets have known true values and provide a standardized scenario for driving the development of data analysis methodologies and tuning cut-off values. Although tools for simulating NGS data are available, they have limitations in terms of their overall usability and documentation. Here, we present Sandy, an open-source simulator that generates synthetic reads that mimic DNA or RNA next-generation sequencing on the Illumina, Oxford Nanopore, and Pacific Bioscience platforms. Sandy is designed to be user-friendly, computationally efficient, and capable of simulating data resembling a wide range of features of real NGS assays, including sequencing quality, genomic variations, and gene expression profiles per tissue. To demonstrate Sandys versatility, we used it to address two critical questions in designing an NGS assay: (i) How many reads should be sequenced to ensure unbiased analysis of gene expression in an RNA sequencing run? (ii) What is the lowest genome coverage required to identify most (90%) of the single nucleotide variants and structural variations in whole-genome sequencing? In summary, Sandy is an ideal tool for assessing and validating pipelines for processing, optimizing results, and defining the costs of NGS assays. Sandy runs on Linux, MacOS, and Microsoft Windows and can provide feasible results, even on personal computers. Availability: Sandy is freely available at https://galantelab.github.io/sandy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555055">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555055" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555055">
        <p class="paperTitle">De novo peptide sequencing with InstaNovo: Accurate, database-free peptide identification for large scale proteomics experiments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555055" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555055" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eloff, K.; Kalogeropoulos, K.; Morell, O.; Mabona, A.; Berg Jespersen, J.; Williams, W.; van Beljouw, S.; Skwark, M.; Hougaard Laustsen, A.; Brouns, S. J. J.; Ljungars, A.; Schoof, E. M.; Van Goey, J.; auf dem Keller, U.; Beguir, K.; Lopez Carranza, N.; Jenkins, T. P.</p>
        <p class="info">Score: 16.2, Published: 2023-08-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555055' target='https://doi.org/10.1101/2023.08.30.555055'> 10.1101/2023.08.30.555055</a></p>
        <p class="abstract">Bottom-up mass spectrometry-based proteomics is challenged by the task of identifying the peptide that generates a tandem mass spectrum. Traditional methods that rely on known peptide sequence databases are limited and may not be applicable in certain contexts. De novo peptide sequencing, which assigns peptide sequences to the spectra without prior information, is valuable for various biological applications; yet, due to a lack of accuracy, it remains challenging to apply this approach in many situations. Here, we introduce InstaNovo, a transformer neural network with the ability to translate fragment ion peaks into the sequence of amino acids that make up the studied peptide(s). The model was trained on 28 million labelled spectra matched to 742k human peptides from the ProteomeTools project. We demonstrate that InstaNovo outperforms current state-of-the-art methods on benchmark datasets and showcase its utility in several applications. Building upon human intuition, we also introduce InstaNovo&#43;, a multinomial diffusion model that further improves performance by iterative refinement of predicted sequences. Using these models, we could de novo sequence antibodybased therapeutics with unprecedented coverage, discover novel peptides, and detect unreported organisms in different datasets, thereby expanding the scope and detection rate of proteomics searches. Finally, we could experimentally validate tryptic and non-tryptic peptides with targeted proteomics, demonstrating the fidelity of our predictions. Our models unlock a plethora of opportunities across different scientific domains, such as direct protein sequencing, immunopeptidomics, and exploration of the dark proteome.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=130 SRC=&#34;FIGDIR/small/555055v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (38K):
org.highwire.dtl.DTLVardef@1e7bfaborg.highwire.dtl.DTLVardef@1d89454org.highwire.dtl.DTLVardef@1f21ad2org.highwire.dtl.DTLVardef@1bedf53_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.555151">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.555151" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.555151">
        <p class="paperTitle">High-accuracy mapping of human and viral direct physical protein-protein interactions using the novel computational system AlphaFold-pairs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.555151" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.555151" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Poitras, C.; Lamontagne, F.; Grandvaux, N.; Song, H.; Pinard, M.; Coulombe, B.</p>
        <p class="info">Score: 15.9, Published: 2023-08-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.555151' target='https://doi.org/10.1101/2023.08.29.555151'> 10.1101/2023.08.29.555151</a></p>
        <p class="abstract">Protein-protein interactions are central, highly flexible components of regulatory mechanisms in all living cells. Over the years, diverse methods have been developed to map protein-protein interactions. These methods have revealed the organization of protein complexes and networks in numerous cells and conditions. However, these methods are also time consuming, costly and sensitive to various experimental artifacts. To avoid these caveats, we have taken advantage of the AlphaFold-Multimer software, which succeeded in predicting the structure of many protein complexes. We designed a relatively simple algorithm based on assessing the physical proximity of a test protein with other AlphaFold structures. Using this method, named AlphaFold-pairs, we have successfully defined the probability of a protein-protein interaction forming. AlphaFold-pairs was validated using well-defined protein-protein interactions found in the literature and specialized databases. All pairwise interactions forming within the 12-subunit transcription machinery RNA Polymerase II, according to available structures, have been identified. Out of 66 possible interactions (excluding homodimers), 19 specific interactions have been found, and an additional previously unknown interaction has been unveiled. The SARS-CoV-2 surface glycoprotein Spike (or S) was confirmed to interact with high preference with the human ACE2 receptor when compared to other human receptors. Notably, two additional receptors, INSR and FLT4, were found to interact with S. For the first time, we have successfully identified protein-protein interactions that are likely to form within the reassortant Eurasian avian-like (EA) H1N1 swine G4 genotype Influenza A virus, which poses a potential zoonotic threat. Testing G4 proteins against human transcription factors and molecular chaperones (a total of 100 proteins) revealed strong specific interactions between the G4 HA and HSP90B1, the G4 NS and the PAQosome subunit RPAP3, as well as the G4 PA and the POLR2A subunit. We predict that AlphaFold-pairs will revolutionize the study of protein-protein interactions in a large number of healthy and diseased systems in the years to come.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.10.544449">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.10.544449" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.10.544449">
        <p class="paperTitle">BRAKER3: Fully Automated Genome Annotation Using RNA-Seq and Protein Evidence with GeneMark-ETP, AUGUSTUS and TSEBRA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.10.544449" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.10.544449" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gabriel, L.; Bruna, T.; Hoff, K. J.; Ebel, M.; Lomsadze, A.; Borodovsky, M.; Stanke, M.</p>
        <p class="info">Score: 112.8, Published: 2023-09-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.10.544449' target='https://doi.org/10.1101/2023.06.10.544449'> 10.1101/2023.06.10.544449</a></p>
        <p class="abstract">Gene prediction remains an active area of bioinformatics research. Challenges are presented by large eukaryotic genomes and heterogeneous data situations. To meet the challenges, several streams of evidence must be integrated, from protein homology and transcriptome data, as well as information derived from the genome itself. The amount and significance of the available evidence from transcriptomes and proteomes vary from genome to genome, between genes and even along a single gene. User-friendly and accurate annotation pipelines that can cope with such data heterogeneity are needed. The previously developed annotation pipelines BRAKER1 and BRAKER2 use RNA-Seq or protein data, respectively, but not both. The recently released GeneMark-ETP integrates all three types of data and achieves much higher levels of accuracy. We here present the BRAKER3 pipeline that builds on GeneMark-ETP and AUGUSTUS and further improves accuracy using the TSEBRA combiner. BRAKER3 annotates protein-coding genes in eukaryotic genomes using both short-read RNA-Seq and a large protein database along with statistical models learned iteratively and specifically for the target genome. We benchmarked the new pipeline on 11 species under controlled conditions on the assumed relatedness of the target species to available proteomes. BRAKER3 outperformed BRAKER1 and BRAKER2, increasing the average transcript-level F1-score by ~20 percentage points, most pronounced for species with large and complex genomes. BRAKER3 also outperforms MAKER2 and Funannotate. For the first time, we provide a Singularity container for the BRAKER software to minimize installation obstacles. Overall, BRAKER3 is an accurate, easy-to-use tool for the annotation of eukaryotic genomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.27.555031">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.27.555031" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.27.555031">
        <p class="paperTitle">AutoHiC: a deep-learning method for automatic and accurate chromosome-level genome assembly</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.27.555031" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.27.555031" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: jiang, z.; peng, z.; luo, y.; bie, l.; wang, y.</p>
        <p class="info">Score: 13.9, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.27.555031' target='https://doi.org/10.1101/2023.08.27.555031'> 10.1101/2023.08.27.555031</a></p>
        <p class="abstract">An accurate genome at the chromosome level is the key to unraveling the mysteries of gene function and unlocking the mechanisms of disease. Irrespective of the sequencing methodology adopted, Hi-C aided scaffolding serves as a principal avenue for generating genome assemblies at the chromosomal level. However, the results of such scaffolding are often flawed and require extensive manual refinement. In this paper, we introduce AutoHiC, an innovative deep learning-based tool designed to identify and rectify genome assembly errors. Diverging from conventional approaches, AutoHiC harnesses the power of high-dimensional Hi-C data to enhance genome continuity and accuracy through a fully automated workflow and iterative error correction mechanism. AutoHiC was trained on Hi-C data from more than 300 species (approximately five hundred thousand interaction maps) in DNA Zoo and NCBI. Its confusion matrix results show that the average error detection accuracy is over 90%, and the area under the precision-recall curve is close to 1, making it a powerful error detection capability. The benchmarking results demonstrate AutoHiCs ability to substantially enhance genome continuity and significantly reduce error rates, providing a more reliable foundation for genomics research. Furthermore, AutoHiC generates comprehensive result reports, offering users insights into the assembly process and outcomes. In summary, AutoHiC represents a breakthrough in automated error detection and correction for genome assembly, effectively promoting more accurate and comprehensive genome assemblies.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
