<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Deep generative model deciphers derailed trajectories in acute myeloid leukemia
Authors: Nazaret, A.; Fan, J. L.; Lavallee, V.-P.; Cornish, A. E.; Kiseliovas, V.; Masilionis, I.; Chun, J.; Bowman, R. L.; Eisman, S. E.; Wang, J.; Shi, L.; Levine, R. L.; Mazutis, L.; Blei, D.; Pe&#39;er, D.; Azizi, E.
Score: 56.3, Published: 2023-11-15 DOI: 10.1101/2023.11.11.566719
Single-cell genomics has the potential to map cell states and their dynamics in an unbiased way in response to perturbations like disease.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="Deep generative model deciphers derailed trajectories in acute myeloid leukemia
Authors: Nazaret, A.; Fan, J. L.; Lavallee, V.-P.; Cornish, A. E.; Kiseliovas, V.; Masilionis, I.; Chun, J.; Bowman, R. L.; Eisman, S. E.; Wang, J.; Shi, L.; Levine, R. L.; Mazutis, L.; Blei, D.; Pe&#39;er, D.; Azizi, E.
Score: 56.3, Published: 2023-11-15 DOI: 10.1101/2023.11.11.566719
Single-cell genomics has the potential to map cell states and their dynamics in an unbiased way in response to perturbations like disease." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-19T10:38:34+00:00" />
<meta property="article:modified_time" content="2023-11-19T10:38:34+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="Deep generative model deciphers derailed trajectories in acute myeloid leukemia
Authors: Nazaret, A.; Fan, J. L.; Lavallee, V.-P.; Cornish, A. E.; Kiseliovas, V.; Masilionis, I.; Chun, J.; Bowman, R. L.; Eisman, S. E.; Wang, J.; Shi, L.; Levine, R. L.; Mazutis, L.; Blei, D.; Pe&#39;er, D.; Azizi, E.
Score: 56.3, Published: 2023-11-15 DOI: 10.1101/2023.11.11.566719
Single-cell genomics has the potential to map cell states and their dynamics in an unbiased way in response to perturbations like disease."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "Deep generative model deciphers derailed trajectories in acute myeloid leukemia\nAuthors: Nazaret, A.; Fan, J. L.; Lavallee, V.-P.; Cornish, A. E.; Kiseliovas, V.; Masilionis, I.; Chun, J.; Bowman, R. L.; Eisman, S. E.; Wang, J.; Shi, L.; Levine, R. L.; Mazutis, L.; Blei, D.; Pe\u0026#39;er, D.; Azizi, E.\nScore: 56.3, Published: 2023-11-15 DOI: 10.1101/2023.11.11.566719\nSingle-cell genomics has the potential to map cell states and their dynamics in an unbiased way in response to perturbations like disease.",
  "keywords": [
    
  ],
  "articleBody": " Deep generative model deciphers derailed trajectories in acute myeloid leukemia\nAuthors: Nazaret, A.; Fan, J. L.; Lavallee, V.-P.; Cornish, A. E.; Kiseliovas, V.; Masilionis, I.; Chun, J.; Bowman, R. L.; Eisman, S. E.; Wang, J.; Shi, L.; Levine, R. L.; Mazutis, L.; Blei, D.; Pe'er, D.; Azizi, E.\nScore: 56.3, Published: 2023-11-15 DOI: 10.1101/2023.11.11.566719\nSingle-cell genomics has the potential to map cell states and their dynamics in an unbiased way in response to perturbations like disease. However, elucidating the cell-state transitions from healthy to disease requires analyzing data from perturbed samples jointly with unperturbed reference samples. Existing methods for integrating and jointly visualizing single-cell datasets from distinct contexts tend to remove key biological differences or do not correctly harmonize shared mechanisms. We present Decipher, a model that combines variational autoencoders with deep exponential families to reconstruct derailed trajectories (https://github.com/azizilab/decipher). Decipher jointly represents normal and perturbed single-cell RNA-seq datasets, revealing shared and disrupted dynamics. It further introduces a novel approach to visualize data, without the need for methods such as UMAP or TSNE. We demonstrate Decipher on data from acute myeloid leukemia patient bone marrow specimens, showing that it successfully characterizes the divergence from normal hematopoiesis and identifies transcriptional programs that become disrupted in each patient when they acquire NPM1 driver mutations.\nA new paradigm for biological sequence retrieval inspired by natural language processing and database research\nAuthors: Rousseau, A.-J.; Lemal, S.; Korovin, Y.; Triantopoulos, G.; Brands, I.; Biemans, M.; Van Hyfte, D.; Valkenborg, D.\nScore: 39.4, Published: 2023-11-09 DOI: 10.1101/2023.11.07.565984\nNearly-exponential growth and heterogeneity of biological sequence data make the task of biological sequence retrieval from databases more important and challenging than ever. In this manuscript, we present a novel search algorithm involving an indexing scheme based on patterns discovered by natural language processing, i.e., short strings of nucleotides or amino acids, akin to standard k-mers, but mined from cumulative cross-species omic data repositories. More specifically, we benchmark the quality of the sequence retrieval process by comparing to BLASTP, a heuristic algorithm for the alignment of genomics or protein sequence data. The main argumentation is that to retrieve biological similar sequences it is not needed to mimic the alignment procedures as it is performed by BLAST. Our results suggests that the HYFT-indexing and searching is a good alternative and a static, alignment-free method to retrieve homologous sequence down to 50% sequence identity.\nTelomere-to-telomere assembly by preserving contained reads\nAuthors: Kamath, S. S.; Bindra, M.; Pal, D.; Jain, C.\nScore: 27.1, Published: 2023-11-11 DOI: 10.1101/2023.11.07.565066\nAutomated telomere-to-telomere (T2T) de novo assembly of diploid and polyploid genomes remains a formidable task. The string graph (Myers, JCB 1995) is the standard representation of graphs in overlap-based assembly algorithms. The string graph formulation employs graph simplification heuristics, which drastically reduce the count of vertices and edges in a read-overlap graph. One of these heuristics in-volves removing the reads contained in longer reads. However, this procedure is not guaranteed to be safe. In practice, it occasionally introduces breaks in the graph by removing all reads that cover one or more genome intervals. What factors contribute to such coverage gap events remains poorly understood. In this work, we mathematically derived the probability of a coverage gap near germline and somatic heterozygous variant loci. Our analysis shows that (i) a coverage gap event is two orders of magnitude more likely for Oxford Nanopore reads than PacBio HiFi reads due to differences in their read-length distributions, and (ii) increasing the sequencing depth reduces the probability at an exponential rate. Next, we addressed the weakness of the string graph formulation by developing the RAFT assembly algorithm. RAFT addresses the issue of contained reads by fragmenting reads and producing a more uniform read-length distribution. The algorithm retains spanned repeats in the reads during the fragmentation. We empirically demonstrate that RAFT drastically reduces the number of coverage gaps. Using Nanopore Duplex and PacBio HiFi datasets of the HG002 human genome, we achieved a twofold increase in the assembly NG50 and the number of haplotype-resolved T2T contigs compared to Hifiasm.\nCentrifuger: lossless compression of microbial genomes for efficient and accurate metagenomic sequence classification\nAuthors: Song, L.; Langmead, B.\nScore: 18.1, Published: 2023-11-17 DOI: 10.1101/2023.11.15.567129\nCentrifuger is an efficient taxonomic classification method that compares sequencing reads against a microbial genome database. In Centrifuger, the Burrows-Wheeler transformed genome sequences are losslessly compressed using a novel scheme called run-block compression. Run-block compression achieves sublinear space complexity and is effective at compressing diverse microbial databases like RefSeq while supporting fast rank queries. Combining this compression method with other strategies for compacting the Ferragina-Manzini (FM) index, Centrifuger reduces the memory footprint by half compared to other FM-index-based approaches. Furthermore, the lossless compression and the unconstrained match length help Centrifuger achieve greater accuracy than competing methods at lower taxonomic levels.\nDemocratizing Protein Language Models with Parameter-Efficient Fine-Tuning\nAuthors: Sledzieski, S.; Kshirsagar, M.; Baek, M.; Berger, B.; Dodhia, R.; Lavista Ferres, J.\nScore: 18.4, Published: 2023-11-10 DOI: 10.1101/2023.11.09.566187\nProteomics has been revolutionized by large pre-trained protein language models, which learn unsupervised representations from large corpora of sequences. The parameters of these models are then fine-tuned in a supervised setting to tailor the model to a specific downstream task. However, as model size increases, the computational and memory footprint of fine-tuning becomes a barrier for many research groups. In the field of natural language processing, which has seen a similar explosion in the size of models, these challenges have been addressed by methods for parameter-efficient fine-tuning (PEFT). In this work, we newly bring parameter-efficient fine-tuning methods to proteomics. Using the parameter-efficient method LoRA, we train new models for two important proteomic tasks: predicting protein-protein interactions (PPI) and predicting the symmetry of homooligomers. We show that for homooligomer symmetry prediction, these approaches achieve performance competitive with traditional fine-tuning while requiring reduced memory and using three orders of magnitude fewer parameters. On the PPI prediction task, we surprisingly find that PEFT models actually outperform traditional fine-tuning while using two orders of magnitude fewer parameters. Here, we go even further to show that freezing the parameters of the language model and training only a classification head also outperforms fine-tuning, using five orders of magnitude fewer parameters, and that both of these models outperform state-of-the-art PPI prediction methods with substantially reduced compute. We also demonstrate that PEFT is robust to variations in training hyper-parameters, and elucidate where best practices for PEFT in proteomics differ from in natural language processing. Thus, we provide a blueprint to democratize the power of protein language model tuning to groups which have limited computational resources.\nBenchmarking multi-omics integration algorithms across single-cell RNA and ATAC data\nAuthors: Xiao, C.; Chen, Y.; Wei, L.; Zhang, X.\nScore: 15.5, Published: 2023-11-17 DOI: 10.1101/2023.11.15.564963\nRecent advancements in single-cell sequencing technologies have generated extensive omics data in various modalities and revolutionized cell research, especially in the single cell RNA and ATAC data. The joint analysis across scRNA-seq data and scATAC-seq data has paved the way to comprehending the cellular heterogeneity and complex cellular regulatory networks. Multi-omics integration is gaining attention as an important step in joint analysis, and the number of computational tools in this field is growing rapidly. In this paper, we benchmarked 12 multi-omics integration methods on three integration tasks via qualitative visualization and quantitative metrics, considering six main aspects that matter in multi-omics data analysis. Overall, we found that different methods have their own advantages on different aspects, while some methods outperformed other methods in most aspects. We therefore provided guidelines for selecting appropriate methods for specific scenarios and tasks to help obtain meaningful insights from multi-omics data integration.\nNavigating the manifold of single-cell gene coexpression to discover interpretable gene programs\nAuthors: Ozbay, S.; Parekh, A.; Singh, R.\nScore: 15.0, Published: 2023-11-11 DOI: 10.1101/2023.11.09.566448\nDuring development and differentiation, transcriptional regulation in the cell often occurs at the level of gene programs (i.e., sets of co-varying genes), rather than isolated genes. It is therefore crucial to identify differential program expression over time, or across case-vs-control samples. However, this has remained difficult: gene programs are inferred by analyzing gene coexpression, and mathematical operations on the latter are nontrivial. Gene coexpression is quantified as a symmetric positive-definite matrix, on which even basic quantities such as arithmetic differences are neither mathematically sound nor biologically interpretable. Here we exploit the structure of the Riemannian manifold of gene coexpression matrices to propose a novel abstraction of gene coexpression that is mathematically well-founded while being computationally tractable and statistically rigorous. Importantly, it also captures biological similarity better than standard coexpression. This conceptual advance enables us to introduce Sceodesic, an algorithm that invokes the log-Euclidean metric from differential geometry to quantify coexpression patterns specific to each cell state, and organizes them into a study-wide panel of interpretable gene programs. Applied to nine single-cell RNA-seq datasets, Sceodesic outperforms existing methods in early detection of cell fate commitment by leveraging differential expression of gene programs, and is also effective in discovering disease-linked programs in multi-sample studies. By respecting the manifold of gene coexpression matrices, Sceodesic resolves a longstanding challenge in relating biological variability to statistical analyses of single-cell RNA-seq data and enables the discovery of gene programs driving differentiation and disease. Software availabilityhttps://singhlab.net/Sceodesic\nRobust parametric UMAP for the analysis of single-cell data\nAuthors: Xu, B.; Zhang, G.\nScore: 13.0, Published: 2023-11-16 DOI: 10.1101/2023.11.14.567092\nThe increasing throughput of single-cell technologies and the pace of data generation are enhancing the resolution at which we observe cell state transitions. The characterization and visualization of these transitions rely on the construction of a low dimensional embedding, which is usually done via non-parametric methods such as t-SNE or UMAP. However, existing approaches become more and more inefficient as the size of the data gets larger and larger. Here, we test the viability of using parametric methods for the fact that they can be trained with a small subset of the data and be applied to future data when needed. We observed that the recently developed parametric version of UMAP is generalizable and robust to dropout. Additionally, to certify the robustness of the model, we use the theoretical upper and lower bounds of the mapped coordinates in the UMAP space to regularize the training process.\nPre-processing of paleogenomes: Mitigating reference bias and postmortem damage in ancient genome data\nAuthors: Koptekin, D.; Yapar, E.; Vural, K. B.; Saglıcan, E.; Altınısık, N. E.; Malaspinas, A.-S.; Alkan, C.; Somel, M.\nScore: 11.9, Published: 2023-11-13 DOI: 10.1101/2023.11.11.566695\nAncient DNA analysis is subject to various technical challenges, including bias towards the reference allele (\"reference bias\"), postmortem damage (PMD) that confounds real variants, and limited coverage. Here, we conduct a systematic comparison of alternative approaches against reference bias and against PMD. To reduce reference bias, we either (a) mask variable sites before alignment or (b) align the data to a graph genome representing all variable sites. Compared to alignment to the linear reference genome, both masking and graph alignment effectively remove allelic bias when using simulated or real ancient human genome data, but only if sequencing data is available in FASTQ or unfiltered BAM format. Reference bias remains indelible in quality-filtered BAM files and in 1240K-capture data. We next study three approaches to overcome postmortem damage: (a) trimming, (b) rescaling base qualities, and (c) a new algorithm we present here, bamRefine, which masks only PMD-vulnerable polymorphic sites. We find that bamRefine is optimal in increasing the number of genotyped loci up to 20% compared to trimming and in improving accuracy compared to rescaling. We propose graph alignment coupled with bamRefine to minimise data loss and bias. We also urge the paleogenomics community to publish FASTQ files.\nFLASHQuant: a fast algorithm for proteoform quantification in top-down proteomics\nAuthors: Kim, J.; Jeong, K.; Kaulich, P. T.; Winkels, K.; Tholey, A.; Kohlbacher, O.\nScore: 10.1, Published: 2023-11-11 DOI: 10.1101/2023.11.08.566181\nLiquid chromatography-mass spectrometry (LC-MS) based top-down proteomics (TDP) is an essential method for the analysis of intact proteoforms. The accurate quantification of individual proteoforms is a crucial step in identifying proteome-wide alterations in different biological conditions. Label-free quantification (LFQ) is the most common method for proteoform quantification as it requires no additional costly labeling. In TDP, due to frequent co-elution and complex signal structures, overlapping signals deriving from multiple proteoforms complicate accurate quantification. Here, we introduce FLASHQuant for MS1-level LFQ analysis in TDP, which is capable of automatically resolving and quantifying co-eluting proteoforms. FLASHQuant performs highly accurate and reproducible quantification in short runtimes of just a few minutes per LC-MS run. To validate the proteoforms reported by FLASHQuant, we evaluated them with identified proteoforms confirmed by tandem mass spectrometry, which showed high match rates. FLASHQuant is publicly available as platform-independent open-source software at https://openms.org/flashquant/.\n",
  "wordCount" : "2097",
  "inLanguage": "en",
  "datePublished": "2023-11-19T10:38:34Z",
  "dateModified": "2023-11-19T10:38:34Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 19, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.11.566719">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.11.566719" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.11.566719">
        <p class="paperTitle">Deep generative model deciphers derailed trajectories in acute myeloid leukemia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.11.566719" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.11.566719" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nazaret, A.; Fan, J. L.; Lavallee, V.-P.; Cornish, A. E.; Kiseliovas, V.; Masilionis, I.; Chun, J.; Bowman, R. L.; Eisman, S. E.; Wang, J.; Shi, L.; Levine, R. L.; Mazutis, L.; Blei, D.; Pe&#39;er, D.; Azizi, E.</p>
        <p class="info">Score: 56.3, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.11.566719' target='https://doi.org/10.1101/2023.11.11.566719'> 10.1101/2023.11.11.566719</a></p>
        <p class="abstract">Single-cell genomics has the potential to map cell states and their dynamics in an unbiased way in response to perturbations like disease. However, elucidating the cell-state transitions from healthy to disease requires analyzing data from perturbed samples jointly with unperturbed reference samples. Existing methods for integrating and jointly visualizing single-cell datasets from distinct contexts tend to remove key biological differences or do not correctly harmonize shared mechanisms. We present Decipher, a model that combines variational autoencoders with deep exponential families to reconstruct derailed trajectories (https://github.com/azizilab/decipher). Decipher jointly represents normal and perturbed single-cell RNA-seq datasets, revealing shared and disrupted dynamics. It further introduces a novel approach to visualize data, without the need for methods such as UMAP or TSNE. We demonstrate Decipher on data from acute myeloid leukemia patient bone marrow specimens, showing that it successfully characterizes the divergence from normal hematopoiesis and identifies transcriptional programs that become disrupted in each patient when they acquire NPM1 driver mutations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.565984">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.565984" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.565984">
        <p class="paperTitle">A new paradigm for biological sequence retrieval inspired by natural language processing and database research</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.565984" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.565984" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rousseau, A.-J.; Lemal, S.; Korovin, Y.; Triantopoulos, G.; Brands, I.; Biemans, M.; Van Hyfte, D.; Valkenborg, D.</p>
        <p class="info">Score: 39.4, Published: 2023-11-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.565984' target='https://doi.org/10.1101/2023.11.07.565984'> 10.1101/2023.11.07.565984</a></p>
        <p class="abstract">Nearly-exponential growth and heterogeneity of biological sequence data make the task of biological sequence retrieval from databases more important and challenging than ever. In this manuscript, we present a novel search algorithm involving an indexing scheme based on patterns discovered by natural language processing, i.e., short strings of nucleotides or amino acids, akin to standard k-mers, but mined from cumulative cross-species omic data repositories. More specifically, we benchmark the quality of the sequence retrieval process by comparing to BLASTP, a heuristic algorithm for the alignment of genomics or protein sequence data. The main argumentation is that to retrieve biological similar sequences it is not needed to mimic the alignment procedures as it is performed by BLAST. Our results suggests that the HYFT-indexing and searching is a good alternative and a static, alignment-free method to retrieve homologous sequence down to 50% sequence identity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.565066">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.565066" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.565066">
        <p class="paperTitle">Telomere-to-telomere assembly by preserving contained reads</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.565066" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.565066" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kamath, S. S.; Bindra, M.; Pal, D.; Jain, C.</p>
        <p class="info">Score: 27.1, Published: 2023-11-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.565066' target='https://doi.org/10.1101/2023.11.07.565066'> 10.1101/2023.11.07.565066</a></p>
        <p class="abstract">Automated telomere-to-telomere (T2T) de novo assembly of diploid and polyploid genomes remains a formidable task. The string graph (Myers, JCB 1995) is the standard representation of graphs in overlap-based assembly algorithms. The string graph formulation employs graph simplification heuristics, which drastically reduce the count of vertices and edges in a read-overlap graph. One of these heuristics in-volves removing the reads contained in longer reads. However, this procedure is not guaranteed to be safe. In practice, it occasionally introduces breaks in the graph by removing all reads that cover one or more genome intervals. What factors contribute to such coverage gap events remains poorly understood. In this work, we mathematically derived the probability of a coverage gap near germline and somatic heterozygous variant loci. Our analysis shows that (i) a coverage gap event is two orders of magnitude more likely for Oxford Nanopore reads than PacBio HiFi reads due to differences in their read-length distributions, and (ii) increasing the sequencing depth reduces the probability at an exponential rate. Next, we addressed the weakness of the string graph formulation by developing the RAFT assembly algorithm. RAFT addresses the issue of contained reads by fragmenting reads and producing a more uniform read-length distribution. The algorithm retains spanned repeats in the reads during the fragmentation. We empirically demonstrate that RAFT drastically reduces the number of coverage gaps. Using Nanopore Duplex and PacBio HiFi datasets of the HG002 human genome, we achieved a twofold increase in the assembly NG50 and the number of haplotype-resolved T2T contigs compared to Hifiasm.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.567129">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.567129" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.567129">
        <p class="paperTitle">Centrifuger: lossless compression of microbial genomes for efficient and accurate metagenomic sequence classification</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.567129" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.567129" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Song, L.; Langmead, B.</p>
        <p class="info">Score: 18.1, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.567129' target='https://doi.org/10.1101/2023.11.15.567129'> 10.1101/2023.11.15.567129</a></p>
        <p class="abstract">Centrifuger is an efficient taxonomic classification method that compares sequencing reads against a microbial genome database. In Centrifuger, the Burrows-Wheeler transformed genome sequences are losslessly compressed using a novel scheme called run-block compression. Run-block compression achieves sublinear space complexity and is effective at compressing diverse microbial databases like RefSeq while supporting fast rank queries. Combining this compression method with other strategies for compacting the Ferragina-Manzini (FM) index, Centrifuger reduces the memory footprint by half compared to other FM-index-based approaches. Furthermore, the lossless compression and the unconstrained match length help Centrifuger achieve greater accuracy than competing methods at lower taxonomic levels.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.566187">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.566187" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.566187">
        <p class="paperTitle">Democratizing Protein Language Models with Parameter-Efficient Fine-Tuning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.566187" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.566187" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sledzieski, S.; Kshirsagar, M.; Baek, M.; Berger, B.; Dodhia, R.; Lavista Ferres, J.</p>
        <p class="info">Score: 18.4, Published: 2023-11-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.566187' target='https://doi.org/10.1101/2023.11.09.566187'> 10.1101/2023.11.09.566187</a></p>
        <p class="abstract">Proteomics has been revolutionized by large pre-trained protein language models, which learn unsupervised representations from large corpora of sequences. The parameters of these models are then fine-tuned in a supervised setting to tailor the model to a specific downstream task. However, as model size increases, the computational and memory footprint of fine-tuning becomes a barrier for many research groups. In the field of natural language processing, which has seen a similar explosion in the size of models, these challenges have been addressed by methods for parameter-efficient fine-tuning (PEFT). In this work, we newly bring parameter-efficient fine-tuning methods to proteomics. Using the parameter-efficient method LoRA, we train new models for two important proteomic tasks: predicting protein-protein interactions (PPI) and predicting the symmetry of homooligomers. We show that for homooligomer symmetry prediction, these approaches achieve performance competitive with traditional fine-tuning while requiring reduced memory and using three orders of magnitude fewer parameters. On the PPI prediction task, we surprisingly find that PEFT models actually outperform traditional fine-tuning while using two orders of magnitude fewer parameters. Here, we go even further to show that freezing the parameters of the language model and training only a classification head also outperforms fine-tuning, using five orders of magnitude fewer parameters, and that both of these models outperform state-of-the-art PPI prediction methods with substantially reduced compute. We also demonstrate that PEFT is robust to variations in training hyper-parameters, and elucidate where best practices for PEFT in proteomics differ from in natural language processing. Thus, we provide a blueprint to democratize the power of protein language model tuning to groups which have limited computational resources.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.564963">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.564963" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.564963">
        <p class="paperTitle">Benchmarking multi-omics integration algorithms across single-cell RNA and ATAC data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.564963" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.564963" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xiao, C.; Chen, Y.; Wei, L.; Zhang, X.</p>
        <p class="info">Score: 15.5, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.564963' target='https://doi.org/10.1101/2023.11.15.564963'> 10.1101/2023.11.15.564963</a></p>
        <p class="abstract">Recent advancements in single-cell sequencing technologies have generated extensive omics data in various modalities and revolutionized cell research, especially in the single cell RNA and ATAC data. The joint analysis across scRNA-seq data and scATAC-seq data has paved the way to comprehending the cellular heterogeneity and complex cellular regulatory networks. Multi-omics integration is gaining attention as an important step in joint analysis, and the number of computational tools in this field is growing rapidly. In this paper, we benchmarked 12 multi-omics integration methods on three integration tasks via qualitative visualization and quantitative metrics, considering six main aspects that matter in multi-omics data analysis. Overall, we found that different methods have their own advantages on different aspects, while some methods outperformed other methods in most aspects. We therefore provided guidelines for selecting appropriate methods for specific scenarios and tasks to help obtain meaningful insights from multi-omics data integration.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.566448">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.566448" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.566448">
        <p class="paperTitle">Navigating the manifold of single-cell gene coexpression to discover interpretable gene programs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.566448" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.566448" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ozbay, S.; Parekh, A.; Singh, R.</p>
        <p class="info">Score: 15.0, Published: 2023-11-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.566448' target='https://doi.org/10.1101/2023.11.09.566448'> 10.1101/2023.11.09.566448</a></p>
        <p class="abstract">During development and differentiation, transcriptional regulation in the cell often occurs at the level of gene programs (i.e., sets of co-varying genes), rather than isolated genes. It is therefore crucial to identify differential program expression over time, or across case-vs-control samples. However, this has remained difficult: gene programs are inferred by analyzing gene coexpression, and mathematical operations on the latter are nontrivial. Gene coexpression is quantified as a symmetric positive-definite matrix, on which even basic quantities such as arithmetic differences are neither mathematically sound nor biologically interpretable. Here we exploit the structure of the Riemannian manifold of gene coexpression matrices to propose a novel abstraction of gene coexpression that is mathematically well-founded while being computationally tractable and statistically rigorous. Importantly, it also captures biological similarity better than standard coexpression. This conceptual advance enables us to introduce Sceodesic, an algorithm that invokes the log-Euclidean metric from differential geometry to quantify coexpression patterns specific to each cell state, and organizes them into a study-wide panel of interpretable gene programs. Applied to nine single-cell RNA-seq datasets, Sceodesic outperforms existing methods in early detection of cell fate commitment by leveraging differential expression of gene programs, and is also effective in discovering disease-linked programs in multi-sample studies. By respecting the manifold of gene coexpression matrices, Sceodesic resolves a longstanding challenge in relating biological variability to statistical analyses of single-cell RNA-seq data and enables the discovery of gene programs driving differentiation and disease.

Software availabilityhttps://singhlab.net/Sceodesic</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.14.567092">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.14.567092" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.14.567092">
        <p class="paperTitle">Robust parametric UMAP for the analysis of single-cell data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.14.567092" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.14.567092" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xu, B.; Zhang, G.</p>
        <p class="info">Score: 13.0, Published: 2023-11-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.14.567092' target='https://doi.org/10.1101/2023.11.14.567092'> 10.1101/2023.11.14.567092</a></p>
        <p class="abstract">The increasing throughput of single-cell technologies and the pace of data generation are enhancing the resolution at which we observe cell state transitions. The characterization and visualization of these transitions rely on the construction of a low dimensional embedding, which is usually done via non-parametric methods such as t-SNE or UMAP. However, existing approaches become more and more inefficient as the size of the data gets larger and larger. Here, we test the viability of using parametric methods for the fact that they can be trained with a small subset of the data and be applied to future data when needed. We observed that the recently developed parametric version of UMAP is generalizable and robust to dropout. Additionally, to certify the robustness of the model, we use the theoretical upper and lower bounds of the mapped coordinates in the UMAP space to regularize the training process.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.11.566695">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.11.566695" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.11.566695">
        <p class="paperTitle">Pre-processing of paleogenomes: Mitigating reference bias and postmortem damage in ancient genome data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.11.566695" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.11.566695" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Koptekin, D.; Yapar, E.; Vural, K. B.; Saglıcan, E.; Altınısık, N. E.; Malaspinas, A.-S.; Alkan, C.; Somel, M.</p>
        <p class="info">Score: 11.9, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.11.566695' target='https://doi.org/10.1101/2023.11.11.566695'> 10.1101/2023.11.11.566695</a></p>
        <p class="abstract">Ancient DNA analysis is subject to various technical challenges, including bias towards the reference allele (&#34;reference bias&#34;), postmortem damage (PMD) that confounds real variants, and limited coverage. Here, we conduct a systematic comparison of alternative approaches against reference bias and against PMD. To reduce reference bias, we either (a) mask variable sites before alignment or (b) align the data to a graph genome representing all variable sites. Compared to alignment to the linear reference genome, both masking and graph alignment effectively remove allelic bias when using simulated or real ancient human genome data, but only if sequencing data is available in FASTQ or unfiltered BAM format. Reference bias remains indelible in quality-filtered BAM files and in 1240K-capture data. We next study three approaches to overcome postmortem damage: (a) trimming, (b) rescaling base qualities, and (c) a new algorithm we present here, bamRefine, which masks only PMD-vulnerable polymorphic sites. We find that bamRefine is optimal in increasing the number of genotyped loci up to 20% compared to trimming and in improving accuracy compared to rescaling. We propose graph alignment coupled with bamRefine to minimise data loss and bias. We also urge the paleogenomics community to publish FASTQ files.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.08.566181">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.08.566181" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.08.566181">
        <p class="paperTitle">FLASHQuant: a fast algorithm for proteoform quantification in top-down proteomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.08.566181" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.08.566181" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kim, J.; Jeong, K.; Kaulich, P. T.; Winkels, K.; Tholey, A.; Kohlbacher, O.</p>
        <p class="info">Score: 10.1, Published: 2023-11-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.08.566181' target='https://doi.org/10.1101/2023.11.08.566181'> 10.1101/2023.11.08.566181</a></p>
        <p class="abstract">Liquid chromatography-mass spectrometry (LC-MS) based top-down proteomics (TDP) is an essential method for the analysis of intact proteoforms. The accurate quantification of individual proteoforms is a crucial step in identifying proteome-wide alterations in different biological conditions. Label-free quantification (LFQ) is the most common method for proteoform quantification as it requires no additional costly labeling. In TDP, due to frequent co-elution and complex signal structures, overlapping signals deriving from multiple proteoforms complicate accurate quantification. Here, we introduce FLASHQuant for MS1-level LFQ analysis in TDP, which is capable of automatically resolving and quantifying co-eluting proteoforms. FLASHQuant performs highly accurate and reproducible quantification in short runtimes of just a few minutes per LC-MS run. To validate the proteoforms reported by FLASHQuant, we evaluated them with identified proteoforms confirmed by tandem mass spectrometry, which showed high match rates. FLASHQuant is publicly available as platform-independent open-source software at https://openms.org/flashquant/.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
