<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Structural phylogenetics unravels the evolutionary diversification of communication systems in gram-positive bacteria and their viruses
Authors: Moi, D.; Bernard, C.; Steinegger, M.; Nevers, Y.; Langleib, M.; Dessimoz, C.
Score: 210.6, Published: 2023-09-23 DOI: 10.1101/2023.09.19.558401
Recent advances in AI-based protein structure modeling have yielded remarkable progress in predicting protein structures. Since structures are constrained by their biological function, their geometry tends to evolve more slowly than the underlying amino acids sequences.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="Structural phylogenetics unravels the evolutionary diversification of communication systems in gram-positive bacteria and their viruses
Authors: Moi, D.; Bernard, C.; Steinegger, M.; Nevers, Y.; Langleib, M.; Dessimoz, C.
Score: 210.6, Published: 2023-09-23 DOI: 10.1101/2023.09.19.558401
Recent advances in AI-based protein structure modeling have yielded remarkable progress in predicting protein structures. Since structures are constrained by their biological function, their geometry tends to evolve more slowly than the underlying amino acids sequences." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-01T10:37:47+00:00" />
<meta property="article:modified_time" content="2023-10-01T10:37:47+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="Structural phylogenetics unravels the evolutionary diversification of communication systems in gram-positive bacteria and their viruses
Authors: Moi, D.; Bernard, C.; Steinegger, M.; Nevers, Y.; Langleib, M.; Dessimoz, C.
Score: 210.6, Published: 2023-09-23 DOI: 10.1101/2023.09.19.558401
Recent advances in AI-based protein structure modeling have yielded remarkable progress in predicting protein structures. Since structures are constrained by their biological function, their geometry tends to evolve more slowly than the underlying amino acids sequences."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "Structural phylogenetics unravels the evolutionary diversification of communication systems in gram-positive bacteria and their viruses\nAuthors: Moi, D.; Bernard, C.; Steinegger, M.; Nevers, Y.; Langleib, M.; Dessimoz, C.\nScore: 210.6, Published: 2023-09-23 DOI: 10.1101/2023.09.19.558401\nRecent advances in AI-based protein structure modeling have yielded remarkable progress in predicting protein structures. Since structures are constrained by their biological function, their geometry tends to evolve more slowly than the underlying amino acids sequences.",
  "keywords": [
    
  ],
  "articleBody": " Structural phylogenetics unravels the evolutionary diversification of communication systems in gram-positive bacteria and their viruses\nAuthors: Moi, D.; Bernard, C.; Steinegger, M.; Nevers, Y.; Langleib, M.; Dessimoz, C.\nScore: 210.6, Published: 2023-09-23 DOI: 10.1101/2023.09.19.558401\nRecent advances in AI-based protein structure modeling have yielded remarkable progress in predicting protein structures. Since structures are constrained by their biological function, their geometry tends to evolve more slowly than the underlying amino acids sequences. This feature of structures could in principle be used to reconstruct phylogenetic trees over longer evolutionary timescales than sequence-based approaches, but until now a reliable structure-based tree building method has been elusive. Here, we demonstrate that the use of structure-based phylogenies can outperform sequence-based ones not only for distantly related proteins but also, remarkably, for more closely related ones. This is achieved by inferring trees from protein structures using a local structural alphabet, an approach robust to conformational changes that confound traditional structural distance measures. As an illustration, we used structures to decipher the evolutionary diversification of a particularly challenging family: the fast-evolving RRNPPA quorum sensing receptors enabling gram-positive bacteria, plasmids and bacteriophages to communicate and coordinate key behaviors such as sporulation, virulence, antibiotic resistance, conjugation or phage lysis/lysogeny decision. The advent of high-accuracy structural phylogenetics enables myriad of applications across biology, such as uncovering deeper evolutionary relationships, elucidating unknown protein functions, or refining the design of bioengineered molecules.\nPhylociraptor - A unified computational framework for reproducible phylogenomic inference\nAuthors: Resl, P.; Hahn, C.\nScore: 105.4, Published: 2023-09-23 DOI: 10.1101/2023.09.22.558970\nMulti-stage phylogenomic analyses require complex computational environments. A unified framework for software- and parameter space exploration, data organisation and -deposition is currently lacking. We present phylociraptor (https://github.com/reslp/phylociraptor), a pipeline facilitating fast, reliable, portable (containerized), and reproducible phylogenomics. We demonstrate phylociraptors utility and uncover topological discrepancies associated with software and parameter choices in large-scale datasets (31-101 species, 241-2894 genes) addressing enigmatic questions in fungal and vertebrate evolution.\nAn RNA foundation model enables discovery of disease mechanisms and candidate therapeutics\nAuthors: Celaj, A.; Gao, A. J.; Lau, T. T. Y.; Holgersen, E. M.; Lo, A.; Lodaya, V.; Cole, C. B.; Denroche, R. E.; Spickett, C.; Wagih, O.; Pinheiro, P. O.; Vora, P.; Mohammadi-Shemirani, P.; Chan, S.; Nussbaum, Z.; Zhang, X.; Zhu, H.; Ramamurthy, E.; Kanuparthi, B.; Iacocca, M.; Ly, D.; Kron, K.; Verby, M.; Cheung-Ong, K.; Shalev, Z.; Vaz, B.; Bhargava, S.; Yusuf, F.; Samuel, S.; Alibai, S.; Baghestani, Z.; He, X.; Krastel, K.; Oladapo, O.; Mohan, A.; Shanavas, A.; Bugno, M.; Bogojeski, J.; Schmitges, F.; Kim, C.; Grant, S.; Jayaraman, R.; Masud, T.; Deshwar, A.; Gandhi, S.; Frey, B. J.\nScore: 73.2, Published: 2023-09-26 DOI: 10.1101/2023.09.20.558508\nAccurately modeling and predicting RNA biology has been a long-standing challenge, bearing significant clinical ramifications for variant interpretation and the formulation of tailored therapeutics. We describe a foundation model for RNA biology, \"BigRNA\", which was trained on thousands of genome-matched datasets to predict tissue-specific RNA expression, splicing, microRNA sites, and RNA binding protein specificity from DNA sequence. Unlike approaches that are restricted to missense variants, BigRNA can identify pathogenic non-coding variant effects across diverse mechanisms, including polyadenylation, exon skipping and intron retention. BigRNA accurately predicted the effects of steric blocking oligonucleotides (SBOs) on increasing the expression of 4 out of 4 genes, and on splicing for 18 out of 18 exons across 14 genes, including those involved in Wilson disease and spinal muscular atrophy. We anticipate that BigRNA and foundation models like it will have widespread applications in the field of personalized RNA therapeutics.\nGIA: A genome interval arithmetic toolkit for high performance interval set operations\nAuthors: Teyssier, N.; Kampmann, M.; Goodarzi, H.\nScore: 51.9, Published: 2023-09-23 DOI: 10.1101/2023.09.20.558707\nMotivationThis study addresses the pressing need for efficient interval techniques in processing vast genomic datasets, such as those generated by ChIP-seq, RNA-seq, and whole-genome sequencing. Intervals are critical in characterizing biological features, necessitating streamlined interval manipulation for meaningful insights. Existing tools often struggle with memory and runtime requirements when managing extensive genomic region arithmetic. ResultsThe study introduces GIA (Genomic Interval Arithmetic) and BEDRS, a novel command-line tool and a rust library that significantly enhance the performance of genomic interval analysis. GIA outperforms existing tools like BEDOPS, BEDTools, and GenomicRanges by a factor of 2x to 20x across a range of operations. These advances enable researchers to perform genomic interval operations more efficiently, drastically reducing computational time and resource requirements in high-throughput genomic sequencing analysis. Availability and Implementation- https://github.com/noamteyssier/gia - https://github.com/noamteyssier/bedrs\nGET: a foundation model of transcription across human cell types\nAuthors: Fu, X.; Mo, S.; Shao, A.; Laurent, A.; Buendia, A.; Ferrando, A. A.; Ciccia, A.; Lan, Y.; Palomero, T.; Owens, D. M.; Xing, E. P.; Rabadan, R.\nScore: 48.9, Published: 2023-09-24 DOI: 10.1101/2023.09.24.559168\nTranscriptional regulation, involving the complex interplay between regulatory sequences and proteins, directs all biological processes. Computational models of transcriptions lack generalizability to accurately extrapolate in unseen cell types and conditions. Here, we introduce GET, an interpretable foundation model, designed to uncover regulatory grammars across 213 human fetal and adult cell types. Relying exclusively on chromatin accessibility data and sequence information, GET achieves experimental-level accuracy in predicting gene expression even in previously unseen cell types. GET showcases remarkable adaptability across new sequencing platforms and assays, enabling regulatory inference across a broad range of cell types and conditions, and uncovering universal and cell type specific transcription factor interaction networks. We evaluated its performance on prediction of regulatory activity, inference of regulatory elements and regulators, and identification of physical interactions between transcription factors. Specifically, we show GET outperforms current models in predicting lentivirus-based massive parallel reporter assay readout with reduced input data. In Fetal erythroblast, we identify distal (\u003e1Mbp) regulatory regions that were missed by previous models. In B cell, we identified a lymphocyte-specific transcription factor-transcription factor interaction that explains the functional significance of a lymphoma-risk predisposing germline mutation. In sum, we provide a generalizable and accurate model for transcription together with catalogs of gene regulation and transcription factor interactions, all with cell type specificity.\nTreeProfiler: A command-line tool for computing and visualizing phylogenetic profiles against large trees\nAuthors: Deng, Z.; Hernandez-Plaza, A.; Huerta-Cepas, J.\nScore: 38.9, Published: 2023-09-23 DOI: 10.1101/2023.09.21.558621\nPhylogenetic profiling is a common bioinformatics approach used to study the distribution of traits across species or, more broadly, along the topology of a phylogenetic tree. Originally proposed as a technique for establishing functional relationships between various genes through their correlated presence/absence patterns, phylogenetic profiling is currently considered a powerful exploration tool for investigating the evolutionary distribution of nearly any trait across both gene or species phylogenies. However, computing and visualizing phylogenetic profiles based on large trees is a laborious task. First, traits need to be mapped against tree nodes, which usually requires the implementation of ad hoc scripts or substantial manual work. Second, tree mapping is often limited to leaf nodes, neglecting the usefulness of profiling traits against internal tree nodes. Lastly, graphical exploration of large phylogenetic tree-based profiles is challenging, as plotting usually requires complex manual adjustments and most visualization tools cannot scale to the size and complexity of current datasets. To address these limitations, we have developed TreeProfiler, a command line tool designed to facilitate both the automated annotation of large trees using a wide range of data sources, and the visual exploration of these annotations as phylogenetic profiles. TreeProfiler enables seamless annotation of massive trees and data tables, including built-in support for functional data, taxonomic information, multiple sequence alignments and protein domain architectures. Importantly, TreeProfiler expands its annotation capabilities beyond tree leaves, enabling the annotation of internal nodes with summarized or consensus information under each branch. Additionally, TreeProfiler offers a predefined set of graphical layouts that are based on the ETE Toolkit v4.0 library. TreeProfiler offers unique capabilities that extend and facilitate the use of phylogenetic profiling techniques for a broad landscape of scenarios and for a wide community of researchers interested in evolutionary analysis. To demonstrate its value, we showcase TreeProfiler by automatically profiling multiple data sources across a large bacterial phylogeny including 45,555 species, and by exploring functional and taxonomic annotations over a large gene family tree. TreeProfiler is available at https://github.com/compgenomicslab/TreeProfiler.\nThe Topological Properties of the Protein Universe\nAuthors: Madsen, C. D.; Barbensi, A.; Zhang, S. Y.; Ham, L.; David, A.; Pires, D. E. V.; Stumpf, M. P. H.\nScore: 33.7, Published: 2023-09-26 DOI: 10.1101/2023.09.25.559443\nDeep learning methods have revolutionized our ability to predict protein structures, allowing us a glimpse into the entire protein universe. As a result, our understanding of how protein structure drives function is now lagging behind our ability to determine and predict protein structure. Here, we describe how topology, the branch of mathematics concerned with qualitative properties of spatial structures, provides a lens through which we can identify fundamental organizing features across the known protein universe. We identify topological determinants that capture global features of the protein universe, such as domain architecture and binding sites. Additionally, our analysis also identified highly specific properties, so-called topological generators, that can be used to provide deeper insights into protein structure-function and evolutionary relationships. We used our approach to determine structural, functional and disease consequences of mutations, explain differences in properties of proteins in mesophiles and thermophiles, and the likely structural and functional consequences of polymorphisms in a protein. Overall, we present a practical methodology for mapping the topology of the known protein universe at scale.\nWhat makes the effect of protein mutations difficult to predict?\nAuthors: van der Flier, F. J.; Estell, D.; Pricelius, S.; Dankmeyer, L.; van Stigt Thans, S.; Mulder, H.; Otsuka, R.; Goedegebuur, F.; Lammerts, L.; Staphorst, D.; van Dijk, A. D. J.; de Ridder, D.; Redestig, H.\nScore: 26.5, Published: 2023-09-25 DOI: 10.1101/2023.09.25.559319\nAO_SCPLOWBSTRACTC_SCPLOWProtein engineering increasingly relies on machine learning models to computationally pre-screen variants to identify those that meet the target requirements. Although machine learning approaches have proven effective, their performance on prospective screening data has room for improvement. Prediction accuracy can vary greatly from one variant to the next. So far, it is unclear what characterizes variants that are associated with large model error. We designed and generated a dataset that can be stratified based on four structural characteristics (buriedness, number of contact residues, proximity to the active site and presence of secondary structure), to answer this question. We found that variants with multiple mutations that are buried, closely connected with other residues or close to the active site, which we call challenging mutations, are harder to model than their counterparts (i.e. exposed, loosely connected, far from the active site). This effect emerges only for variants with multiple challenging mutations, since single mutations at these sites were not harder to model. Our findings indicate that variants with challenging mutations are appropriate benchmarking targets for assessing model quality and that stratified dataset design can be leveraged to highlight areas of improvement for machine learning guided protein engineering.\nProteinFlow: a Python Library to Pre-Process Protein Structure Data for Deep Learning Applications\nAuthors: Kozlova, E.; Valentin, A.; Khadhraoui, A.; Nakhaee-Zadeh Gutierrez, D.\nScore: 23.2, Published: 2023-09-26 DOI: 10.1101/2023.09.25.559346\nOver the past few years, deep learning tools for protein design have made significant advances in the field of bioengineering, opening up new opportunities for drug discovery, disease prevention or industrial biotechnology. However, despite the growing interest and excitement surrounding these tools, progress in the field is hindered by a lack of standardized datasets for benchmarking. Most models are trained on data from the Protein Data Bank (PDB), the largest repository of experimentally determined biological macromolecular structures. But filtering and processing this data involves many hyperparameter choices that are often not harmonized across the research community. Moreover, the task of splitting protein data into training and validation subsets with minimal data leakage is not trivial and often overlooked. Here we present ProteinFlow, a computational pipeline to pre-process protein sequence and structural data for deep learning applications. The pipeline is fully configurable and allows the extraction of all levels of protein organization (primary to quaternary), allowing end-users to cater the dataset for a multitude of downstream tasks, such as protein sequence design, protein folding modeling or protein-protein interaction prediction. In addition, we curate a feature-rich benchmarking dataset based on the latest annual release of the PDB and a selection of preprocessing parameters that are widely used across the research community. We showcase its utility by benchmarking a state-of-the-art (SOTA) deep learning model for protein sequence design. The open source code is packaged as a python library and can be accessed on https://github.com/adaptyvbio/ProteinFlow.\nSeqVerify: A quality-assurance pipeline for whole-genome sequencing data\nAuthors: Pierson Smela, M. D.; Pepe, V.; Church, G. M.\nScore: 22.6, Published: 2023-09-28 DOI: 10.1101/2023.09.27.559766\nOver the last decade, advances in genome editing and pluripotent stem cell (PSC) culture have let researchers generate edited PSC lines to study a wide variety of biological questions. However, abnormalities in cell lines can arise during PSC culture or due to undesired editing outcomes. These can include aneuploidy, on-target and off-target editing errors, and microbial contamination. Any of these abnormalities can invalidate experiments, so detecting them is crucial. The ongoing decline of next-generation sequencing prices has made whole genome sequencing (WGS) an effective quality control option, since WGS can detect any abnormality involving changes to DNA sequences or presence of unwanted sequences. However, until now, this approach has suffered from a lack of easily usable data analysis software. Here, we present SeqVerify, a computational pipeline designed to take raw WGS data and a list of intended edits, and verify that the edits are present and that there are no abnormalities. We anticipate that SeqVerify will be a useful tool for researchers generating edited PSCs, and more broadly, for cell line quality control in general.\n",
  "wordCount" : "2265",
  "inLanguage": "en",
  "datePublished": "2023-10-01T10:37:47Z",
  "dateModified": "2023-10-01T10:37:47Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on October 1, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.19.558401">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.19.558401" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.19.558401">
        <p class="paperTitle">Structural phylogenetics unravels the evolutionary diversification of communication systems in gram-positive bacteria and their viruses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.19.558401" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.19.558401" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moi, D.; Bernard, C.; Steinegger, M.; Nevers, Y.; Langleib, M.; Dessimoz, C.</p>
        <p class="info">Score: 210.6, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.19.558401' target='https://doi.org/10.1101/2023.09.19.558401'> 10.1101/2023.09.19.558401</a></p>
        <p class="abstract">Recent advances in AI-based protein structure modeling have yielded remarkable progress in predicting protein structures. Since structures are constrained by their biological function, their geometry tends to evolve more slowly than the underlying amino acids sequences. This feature of structures could in principle be used to reconstruct phylogenetic trees over longer evolutionary timescales than sequence-based approaches, but until now a reliable structure-based tree building method has been elusive. Here, we demonstrate that the use of structure-based phylogenies can outperform sequence-based ones not only for distantly related proteins but also, remarkably, for more closely related ones. This is achieved by inferring trees from protein structures using a local structural alphabet, an approach robust to conformational changes that confound traditional structural distance measures. As an illustration, we used structures to decipher the evolutionary diversification of a particularly challenging family: the fast-evolving RRNPPA quorum sensing receptors enabling gram-positive bacteria, plasmids and bacteriophages to communicate and coordinate key behaviors such as sporulation, virulence, antibiotic resistance, conjugation or phage lysis/lysogeny decision. The advent of high-accuracy structural phylogenetics enables myriad of applications across biology, such as uncovering deeper evolutionary relationships, elucidating unknown protein functions, or refining the design of bioengineered molecules.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.558970">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.558970" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.558970">
        <p class="paperTitle">Phylociraptor - A unified computational framework for reproducible phylogenomic inference</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.558970" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.558970" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Resl, P.; Hahn, C.</p>
        <p class="info">Score: 105.4, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.558970' target='https://doi.org/10.1101/2023.09.22.558970'> 10.1101/2023.09.22.558970</a></p>
        <p class="abstract">Multi-stage phylogenomic analyses require complex computational environments. A unified framework for software- and parameter space exploration, data organisation and -deposition is currently lacking. We present phylociraptor (https://github.com/reslp/phylociraptor), a pipeline facilitating fast, reliable, portable (containerized), and reproducible phylogenomics. We demonstrate phylociraptors utility and uncover topological discrepancies associated with software and parameter choices in large-scale datasets (31-101 species, 241-2894 genes) addressing enigmatic questions in fungal and vertebrate evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.558508">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.558508" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.558508">
        <p class="paperTitle">An RNA foundation model enables discovery of disease mechanisms and candidate therapeutics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.558508" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.558508" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Celaj, A.; Gao, A. J.; Lau, T. T. Y.; Holgersen, E. M.; Lo, A.; Lodaya, V.; Cole, C. B.; Denroche, R. E.; Spickett, C.; Wagih, O.; Pinheiro, P. O.; Vora, P.; Mohammadi-Shemirani, P.; Chan, S.; Nussbaum, Z.; Zhang, X.; Zhu, H.; Ramamurthy, E.; Kanuparthi, B.; Iacocca, M.; Ly, D.; Kron, K.; Verby, M.; Cheung-Ong, K.; Shalev, Z.; Vaz, B.; Bhargava, S.; Yusuf, F.; Samuel, S.; Alibai, S.; Baghestani, Z.; He, X.; Krastel, K.; Oladapo, O.; Mohan, A.; Shanavas, A.; Bugno, M.; Bogojeski, J.; Schmitges, F.; Kim, C.; Grant, S.; Jayaraman, R.; Masud, T.; Deshwar, A.; Gandhi, S.; Frey, B. J.</p>
        <p class="info">Score: 73.2, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.558508' target='https://doi.org/10.1101/2023.09.20.558508'> 10.1101/2023.09.20.558508</a></p>
        <p class="abstract">Accurately modeling and predicting RNA biology has been a long-standing challenge, bearing significant clinical ramifications for variant interpretation and the formulation of tailored therapeutics. We describe a foundation model for RNA biology, &#34;BigRNA&#34;, which was trained on thousands of genome-matched datasets to predict tissue-specific RNA expression, splicing, microRNA sites, and RNA binding protein specificity from DNA sequence. Unlike approaches that are restricted to missense variants, BigRNA can identify pathogenic non-coding variant effects across diverse mechanisms, including polyadenylation, exon skipping and intron retention. BigRNA accurately predicted the effects of steric blocking oligonucleotides (SBOs) on increasing the expression of 4 out of 4 genes, and on splicing for 18 out of 18 exons across 14 genes, including those involved in Wilson disease and spinal muscular atrophy. We anticipate that BigRNA and foundation models like it will have widespread applications in the field of personalized RNA therapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.558707">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.558707" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.558707">
        <p class="paperTitle">GIA: A genome interval arithmetic toolkit for high performance interval set operations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.558707" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.558707" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Teyssier, N.; Kampmann, M.; Goodarzi, H.</p>
        <p class="info">Score: 51.9, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.558707' target='https://doi.org/10.1101/2023.09.20.558707'> 10.1101/2023.09.20.558707</a></p>
        <p class="abstract">MotivationThis study addresses the pressing need for efficient interval techniques in processing vast genomic datasets, such as those generated by ChIP-seq, RNA-seq, and whole-genome sequencing. Intervals are critical in characterizing biological features, necessitating streamlined interval manipulation for meaningful insights. Existing tools often struggle with memory and runtime requirements when managing extensive genomic region arithmetic.

ResultsThe study introduces GIA (Genomic Interval Arithmetic) and BEDRS, a novel command-line tool and a rust library that significantly enhance the performance of genomic interval analysis. GIA outperforms existing tools like BEDOPS, BEDTools, and GenomicRanges by a factor of 2x to 20x across a range of operations. These advances enable researchers to perform genomic interval operations more efficiently, drastically reducing computational time and resource requirements in high-throughput genomic sequencing analysis.

Availability and Implementation- https://github.com/noamteyssier/gia
- https://github.com/noamteyssier/bedrs</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.24.559168">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.24.559168" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.24.559168">
        <p class="paperTitle">GET: a foundation model of transcription across human cell types</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.24.559168" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.24.559168" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fu, X.; Mo, S.; Shao, A.; Laurent, A.; Buendia, A.; Ferrando, A. A.; Ciccia, A.; Lan, Y.; Palomero, T.; Owens, D. M.; Xing, E. P.; Rabadan, R.</p>
        <p class="info">Score: 48.9, Published: 2023-09-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.24.559168' target='https://doi.org/10.1101/2023.09.24.559168'> 10.1101/2023.09.24.559168</a></p>
        <p class="abstract">Transcriptional regulation, involving the complex interplay between regulatory sequences and proteins, directs all biological processes. Computational models of transcriptions lack generalizability to accurately extrapolate in unseen cell types and conditions. Here, we introduce GET, an interpretable foundation model, designed to uncover regulatory grammars across 213 human fetal and adult cell types. Relying exclusively on chromatin accessibility data and sequence information, GET achieves experimental-level accuracy in predicting gene expression even in previously unseen cell types. GET showcases remarkable adaptability across new sequencing platforms and assays, enabling regulatory inference across a broad range of cell types and conditions, and uncovering universal and cell type specific transcription factor interaction networks. We evaluated its performance on prediction of regulatory activity, inference of regulatory elements and regulators, and identification of physical interactions between transcription factors. Specifically, we show GET outperforms current models in predicting lentivirus-based massive parallel reporter assay readout with reduced input data. In Fetal erythroblast, we identify distal (&gt;1Mbp) regulatory regions that were missed by previous models. In B cell, we identified a lymphocyte-specific transcription factor-transcription factor interaction that explains the functional significance of a lymphoma-risk predisposing germline mutation. In sum, we provide a generalizable and accurate model for transcription together with catalogs of gene regulation and transcription factor interactions, all with cell type specificity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558621">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558621" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558621">
        <p class="paperTitle">TreeProfiler: A command-line tool for computing and visualizing phylogenetic profiles against large trees</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558621" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558621" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Deng, Z.; Hernandez-Plaza, A.; Huerta-Cepas, J.</p>
        <p class="info">Score: 38.9, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558621' target='https://doi.org/10.1101/2023.09.21.558621'> 10.1101/2023.09.21.558621</a></p>
        <p class="abstract">Phylogenetic profiling is a common bioinformatics approach used to study the distribution of traits across species or, more broadly, along the topology of a phylogenetic tree. Originally proposed as a technique for establishing functional relationships between various genes through their correlated presence/absence patterns, phylogenetic profiling is currently considered a powerful exploration tool for investigating the evolutionary distribution of nearly any trait across both gene or species phylogenies. However, computing and visualizing phylogenetic profiles based on large trees is a laborious task. First, traits need to be mapped against tree nodes, which usually requires the implementation of ad hoc scripts or substantial manual work. Second, tree mapping is often limited to leaf nodes, neglecting the usefulness of profiling traits against internal tree nodes. Lastly, graphical exploration of large phylogenetic tree-based profiles is challenging, as plotting usually requires complex manual adjustments and most visualization tools cannot scale to the size and complexity of current datasets. To address these limitations, we have developed TreeProfiler, a command line tool designed to facilitate both the automated annotation of large trees using a wide range of data sources, and the visual exploration of these annotations as phylogenetic profiles. TreeProfiler enables seamless annotation of massive trees and data tables, including built-in support for functional data, taxonomic information, multiple sequence alignments and protein domain architectures. Importantly, TreeProfiler expands its annotation capabilities beyond tree leaves, enabling the annotation of internal nodes with summarized or consensus information under each branch. Additionally, TreeProfiler offers a predefined set of graphical layouts that are based on the ETE Toolkit v4.0 library. TreeProfiler offers unique capabilities that extend and facilitate the use of phylogenetic profiling techniques for a broad landscape of scenarios and for a wide community of researchers interested in evolutionary analysis. To demonstrate its value, we showcase TreeProfiler by automatically profiling multiple data sources across a large bacterial phylogeny including 45,555 species, and by exploring functional and taxonomic annotations over a large gene family tree. TreeProfiler is available at https://github.com/compgenomicslab/TreeProfiler.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559443">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559443" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559443">
        <p class="paperTitle">The Topological Properties of the Protein Universe</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559443" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559443" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Madsen, C. D.; Barbensi, A.; Zhang, S. Y.; Ham, L.; David, A.; Pires, D. E. V.; Stumpf, M. P. H.</p>
        <p class="info">Score: 33.7, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559443' target='https://doi.org/10.1101/2023.09.25.559443'> 10.1101/2023.09.25.559443</a></p>
        <p class="abstract">Deep learning methods have revolutionized our ability to predict protein structures, allowing us a glimpse into the entire protein universe. As a result, our understanding of how protein structure drives function is now lagging behind our ability to determine and predict protein structure. Here, we describe how topology, the branch of mathematics concerned with qualitative properties of spatial structures, provides a lens through which we can identify fundamental organizing features across the known protein universe. We identify topological determinants that capture global features of the protein universe, such as domain architecture and binding sites. Additionally, our analysis also identified highly specific properties, so-called topological generators, that can be used to provide deeper insights into protein structure-function and evolutionary relationships. We used our approach to determine structural, functional and disease consequences of mutations, explain differences in properties of proteins in mesophiles and thermophiles, and the likely structural and functional consequences of polymorphisms in a protein. Overall, we present a practical methodology for mapping the topology of the known protein universe at scale.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559319">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559319" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559319">
        <p class="paperTitle">What makes the effect of protein mutations difficult to predict?</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559319" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559319" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: van der Flier, F. J.; Estell, D.; Pricelius, S.; Dankmeyer, L.; van Stigt Thans, S.; Mulder, H.; Otsuka, R.; Goedegebuur, F.; Lammerts, L.; Staphorst, D.; van Dijk, A. D. J.; de Ridder, D.; Redestig, H.</p>
        <p class="info">Score: 26.5, Published: 2023-09-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559319' target='https://doi.org/10.1101/2023.09.25.559319'> 10.1101/2023.09.25.559319</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWProtein engineering increasingly relies on machine learning models to computationally pre-screen variants to identify those that meet the target requirements. Although machine learning approaches have proven effective, their performance on prospective screening data has room for improvement. Prediction accuracy can vary greatly from one variant to the next. So far, it is unclear what characterizes variants that are associated with large model error. We designed and generated a dataset that can be stratified based on four structural characteristics (buriedness, number of contact residues, proximity to the active site and presence of secondary structure), to answer this question. We found that variants with multiple mutations that are buried, closely connected with other residues or close to the active site, which we call challenging mutations, are harder to model than their counterparts (i.e. exposed, loosely connected, far from the active site). This effect emerges only for variants with multiple challenging mutations, since single mutations at these sites were not harder to model. Our findings indicate that variants with challenging mutations are appropriate benchmarking targets for assessing model quality and that stratified dataset design can be leveraged to highlight areas of improvement for machine learning guided protein engineering.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559346">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559346" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559346">
        <p class="paperTitle">ProteinFlow: a Python Library to Pre-Process Protein Structure Data for Deep Learning Applications</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559346" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559346" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kozlova, E.; Valentin, A.; Khadhraoui, A.; Nakhaee-Zadeh Gutierrez, D.</p>
        <p class="info">Score: 23.2, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559346' target='https://doi.org/10.1101/2023.09.25.559346'> 10.1101/2023.09.25.559346</a></p>
        <p class="abstract">Over the past few years, deep learning tools for protein design have made significant advances in the field of bioengineering, opening up new opportunities for drug discovery, disease prevention or industrial biotechnology. However, despite the growing interest and excitement surrounding these tools, progress in the field is hindered by a lack of standardized datasets for benchmarking. Most models are trained on data from the Protein Data Bank (PDB), the largest repository of experimentally determined biological macromolecular structures. But filtering and processing this data involves many hyperparameter choices that are often not harmonized across the research community. Moreover, the task of splitting protein data into training and validation subsets with minimal data leakage is not trivial and often overlooked. Here we present ProteinFlow, a computational pipeline to pre-process protein sequence and structural data for deep learning applications. The pipeline is fully configurable and allows the extraction of all levels of protein organization (primary to quaternary), allowing end-users to cater the dataset for a multitude of downstream tasks, such as protein sequence design, protein folding modeling or protein-protein interaction prediction. In addition, we curate a feature-rich benchmarking dataset based on the latest annual release of the PDB and a selection of preprocessing parameters that are widely used across the research community. We showcase its utility by benchmarking a state-of-the-art (SOTA) deep learning model for protein sequence design. The open source code is packaged as a python library and can be accessed on https://github.com/adaptyvbio/ProteinFlow.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.559766">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.559766" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.559766">
        <p class="paperTitle">SeqVerify: A quality-assurance pipeline for whole-genome sequencing data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.559766" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.559766" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pierson Smela, M. D.; Pepe, V.; Church, G. M.</p>
        <p class="info">Score: 22.6, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.559766' target='https://doi.org/10.1101/2023.09.27.559766'> 10.1101/2023.09.27.559766</a></p>
        <p class="abstract">Over the last decade, advances in genome editing and pluripotent stem cell (PSC) culture have let researchers generate edited PSC lines to study a wide variety of biological questions. However, abnormalities in cell lines can arise during PSC culture or due to undesired editing outcomes. These can include aneuploidy, on-target and off-target editing errors, and microbial contamination. Any of these abnormalities can invalidate experiments, so detecting them is crucial. The ongoing decline of next-generation sequencing prices has made whole genome sequencing (WGS) an effective quality control option, since WGS can detect any abnormality involving changes to DNA sequences or presence of unwanted sequences. However, until now, this approach has suffered from a lack of easily usable data analysis software. Here, we present SeqVerify, a computational pipeline designed to take raw WGS data and a list of intended edits, and verify that the edits are present and that there are no abnormalities. We anticipate that SeqVerify will be a useful tool for researchers generating edited PSCs, and more broadly, for cell line quality control in general.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
