<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Petascale Homology Search for Structure Prediction
Authors: Lee, S.; Kim, G.; Levy Karin, E.; Mirdita, M.; Park, S.; Chikhi, R.; Babaian, A.; Kryshtafovych, A.; Steinegger, M.
Score: 70.8, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548308
The recent CASP15 competition highlighted the critical role of multiple sequence alignments (MSAs) in protein structure prediction, as demonstrated by the success of the top AlphaFold2-based prediction methods. To push the boundaries of MSA utilization, we conducted a petabase-scale search of the Sequence Read Archive (SRA), resulting in gigabytes of aligned homologs for CASP15 targets.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="Petascale Homology Search for Structure Prediction
Authors: Lee, S.; Kim, G.; Levy Karin, E.; Mirdita, M.; Park, S.; Chikhi, R.; Babaian, A.; Kryshtafovych, A.; Steinegger, M.
Score: 70.8, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548308
The recent CASP15 competition highlighted the critical role of multiple sequence alignments (MSAs) in protein structure prediction, as demonstrated by the success of the top AlphaFold2-based prediction methods. To push the boundaries of MSA utilization, we conducted a petabase-scale search of the Sequence Read Archive (SRA), resulting in gigabytes of aligned homologs for CASP15 targets." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-16T10:39:40+00:00" />
<meta property="article:modified_time" content="2023-07-16T10:39:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="Petascale Homology Search for Structure Prediction
Authors: Lee, S.; Kim, G.; Levy Karin, E.; Mirdita, M.; Park, S.; Chikhi, R.; Babaian, A.; Kryshtafovych, A.; Steinegger, M.
Score: 70.8, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548308
The recent CASP15 competition highlighted the critical role of multiple sequence alignments (MSAs) in protein structure prediction, as demonstrated by the success of the top AlphaFold2-based prediction methods. To push the boundaries of MSA utilization, we conducted a petabase-scale search of the Sequence Read Archive (SRA), resulting in gigabytes of aligned homologs for CASP15 targets."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "Petascale Homology Search for Structure Prediction\nAuthors: Lee, S.; Kim, G.; Levy Karin, E.; Mirdita, M.; Park, S.; Chikhi, R.; Babaian, A.; Kryshtafovych, A.; Steinegger, M.\nScore: 70.8, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548308\nThe recent CASP15 competition highlighted the critical role of multiple sequence alignments (MSAs) in protein structure prediction, as demonstrated by the success of the top AlphaFold2-based prediction methods. To push the boundaries of MSA utilization, we conducted a petabase-scale search of the Sequence Read Archive (SRA), resulting in gigabytes of aligned homologs for CASP15 targets.",
  "keywords": [
    
  ],
  "articleBody": " Petascale Homology Search for Structure Prediction\nAuthors: Lee, S.; Kim, G.; Levy Karin, E.; Mirdita, M.; Park, S.; Chikhi, R.; Babaian, A.; Kryshtafovych, A.; Steinegger, M.\nScore: 70.8, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548308\nThe recent CASP15 competition highlighted the critical role of multiple sequence alignments (MSAs) in protein structure prediction, as demonstrated by the success of the top AlphaFold2-based prediction methods. To push the boundaries of MSA utilization, we conducted a petabase-scale search of the Sequence Read Archive (SRA), resulting in gigabytes of aligned homologs for CASP15 targets. These were merged with default MSAs produced by ColabFold-search and provided to ColabFold-predict. By using SRA data, we achieved highly accurate predictions (GDT_TS \u003e 70) for 66% of the non-easy targets, whereas using ColabFold-search default MSAs scored highly in only 52%. Next, we tested the effect of deep homology search and ColabFolds advanced features, such as more recycles, on prediction accuracy. While SRA homologs were most significant for improving ColabFolds CASP15 ranking from 11th to 3rd place, other strategies contributed too. We analyze these in the context of existing strategies to improve prediction.\nEfficient High-Quality Metagenome Assembly from Long Accurate Reads using Minimizer-space de Bruijn Graphs\nAuthors: Benoit, G.; Raguideau, S.; James, R.; Phillippy, A. M.; Chikhi, R.; Quince, C.\nScore: 55.7, Published: 2023-07-08 DOI: 10.1101/2023.07.07.548136\nWe introduce a novel metagenomics assembler for high-accuracy long reads. Our approach, implemented as metaMDBG, combines highly efficient de Bruijn graph assembly in minimizer space, with both a multi-k' approach for dealing with variations in genome coverage depth and an abundance-based filtering strategy for simplifying strain complexity. The resulting algorithm is more efficient than the state-of-the-art but with better assembly results. metaMDBG was 1.5 to 12 times faster than competing assemblers and requires between one-tenth and one-thirtieth of the memory across a range of data sets. We obtained up to twice as many high-quality circularised prokaryotic metagenome assembled genomes (MAGs) on the most complex communities, and a better recovery of viruses and plasmids. metaMDBG performs particularly well for abundant organisms whilst being robust to the presence of strain diversity. The result is that for the first time it is possible to efficiently reconstruct the majority of complex communities by abundance as nearcomplete MAGs.\nDeep dynamical modelling of developmental trajectories with temporal transcriptomics\nAuthors: Maizels, R. J.; Snell, D. M.; Briscoe, J.\nScore: 59.5, Published: 2023-07-07 DOI: 10.1101/2023.07.06.547989\nDevelopmental cell fate decisions are dynamic processes driven by the complex behaviour of gene regulatory networks. A challenge in studying these processes using single-cell genomics is that the data provides only a static snapshot with no detail of dynamics. Metabolic labelling and splicing can provide time-resolved information, but current methods have limitations. Here, we present experimental and computational methods that overcome these limitations to allow dynamical modelling of gene expression from single-cell data. We developed sci-FATE2, an optimised metabolic labelling method that substantially increases data quality, and profiled approximately 45,000 embryonic stem cells differentiating into multiple neural tube identities. To recover dynamics, we developed velvet, a deep learning framework that extends beyond instantaneous velocity estimation by modelling gene expression dynamics through a neural stochastic differential equation system within a variational autoencoder. Velvet outperforms current velocity tools across quantitative benchmarks, and predicts trajectory distributions that accurately recapitulate underlying dataset distributions while conserving known biology. Velvet trajectory distributions capture dynamical aspects such as decision boundaries between alternative fates and correlative gene regulatory structure. Using velvet to provide a dynamical description of in vitro neural patterning, we highlight a process of sequential decision making and fate-specific patterns of developmental signalling. Together, these experimental and computational methods recast single-cell analyses from descriptions of observed data distributions to models of the dynamics that generated them, providing a new framework for investigating developmental gene regulation and cell fate decisions.\nSemi-supervised integration of single-cell transcriptomics data\nAuthors: Andreatta, M.; Herault, L.; Gueguen, P.; Gfeller, D.; Berenstein, A. J.; Carmona, S. J.\nScore: 34.9, Published: 2023-07-07 DOI: 10.1101/2023.07.07.548105\nSingle-cell sequencing technologies offer unprecedented opportunities to characterize the complexity of biological samples with high resolution. At the same time, variations in sample processing and experimental protocols introduce technical variability - or \"batch effects\" - in the molecular readouts, hindering comparative analyses across samples and individuals. Although batch effect correction methods are routinely applied in single-cell omics analyses, data integration often leads to overcorrection, resulting in the loss of true biological variability. In this study, we present STACAS v2, a semi-supervised scRNA-seq data integration method that leverages prior knowledge in the form of cell type annotations to preserve biological variance. Through an open and reproducible benchmarking pipeline, we show that semi-supervised STACAS outperforms popular unsupervised methods such as Harmony, FastMNN, Seurat v4, scVI, and Scanorama, as well as supervised methods such as scANVI and scGen. Notably, STACAS is robust to incomplete and imprecise cell type annotations, which are commonly encountered in real-life integration tasks. Highlighting its scalability, we successfully applied semi-supervised STACAS to construct a high-resolution map of tumor-infiltrating CD8 T cells encompassing over 500,000 cells from 265 individuals. Based on our findings, we argue that the incorporation of prior cell type information should be a common practice in single-cell data integration, and we provide a flexible framework for semi-supervised batch effect correction. STACAS seamlessly integrates with Seurat pipelines and can be run with one command: Run.STACAS(seurat.list, cell.labels).\nIdentifying Protein Conformational States in the PDB and Comparison to AlphaFold2 Predictions\nAuthors: Ellaway, J. I. J.; Anyango, S.; Nair, S.; Zaki, H. A.; Nadzirin, N.; Powell, H. R.; Gutmanas, A.; Varadi, M.; Velankar, S.\nScore: 28.3, Published: 2023-07-14 DOI: 10.1101/2023.07.13.545008\nProteins, as molecular machines, are necessarily dynamic macromolecules that carry out essential cellular functions. Recognising their stable conformations is important for understanding the molecular mechanisms of disease. While AI-based computational methods have enabled protein structure prediction, the prediction of protein dynamics remains a challenge. Here, we present a deterministic pipeline that clusters experimentally determined protein structures to comprehensively recognise conformational states across the Protein Data Bank. Our approach clusters protein chains based on a GLObal CONformation (GLOCON) difference score, which is computed from pairwise C-alpha distances. By superposing the clustered structures, differences and similarities in conformational states can be observed. Additionally, we offer users the ability to superpose predicted models from the AlphaFold Database to the clusters of PDB structures. This clustering pipeline significantly advances researchers' ability to explore the conformational landscape within the PDB. All clustered and superposed models can be viewed in Mol* on the PDBe Knowledge Base website, or accessed in as raw annotations via our GraphAPI and FTP server. The clustering package is made available as an open-source Python3 package under the Apache-2.0 license.\nQuantifying Cell-State Densities in Single-Cell Phenotypic Landscapes using Mellon\nAuthors: Otto, D. J.; Jordan, C.; Dury, B.; Dien, C.; Setty, M.\nScore: 26.9, Published: 2023-07-11 DOI: 10.1101/2023.07.09.548272\nCell-state density characterizes the distribution of cells along phenotypic landscapes and is crucial for unraveling the mechanisms that drive cellular differentiation, regeneration, and disease. Here, we present Mellon, a novel computational algorithm for high-resolution estimation of cell-state densities from single-cell data. We demonstrate Mellons efficacy by dissecting the density landscape of various differentiating systems, revealing a consistent pattern of high-density regions corresponding to major cell types intertwined with low-density, rare transitory states. Utilizing hematopoietic stem cell fate specification to B-cells as a case study, we present evidence implicating enhancer priming and the activation of master regulators in the emergence of these transitory states. Mellon offers the flexibility to perform temporal interpolation of time-series data, providing a detailed view of cell-state dynamics during the inherently continuous developmental processes. Scalable and adaptable, Mellon facilitates density estimation across various single-cell data modalities, scaling linearly with the number of cells. Our work underscores the importance of cell-state density in understanding the differentiation processes, and the potential of Mellon to provide new insights into the regulatory mechanisms guiding cellular fate decisions.\nARCTIC-3D: Automatic Retrieval and ClusTering of Interfaces in Complexes from 3D structural information\nAuthors: Giulini, M.; Honorato, R. V.; Rivera, J. L.; Bonvin, A. M. J. J.\nScore: 26.7, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548477\nThe formation of a stable complex between proteins lies at the core of a wide variety of biological processes and has been the focus of countless experiments. The huge amount of information contained in the protein structural interactome in the Protein Data Bank can now be used to characterise and classify the existing biological interfaces. We here introduce ARCTIC-3D, a fast and user-friendly data mining and clustering software to retrieve data and rationalise the interface information associated with the protein input data. We demonstrate its use by various examples ranging from showing the increased interaction complexity of eukaryotic proteins, 20% of which on average have more than 3 different interfaces compared to only 10% for prokaryotes, to associating different functions to different interfaces. In the context of modelling biomolecular assemblies, we introduce the concept of \"recognition entropy\", related to the number of possible interfaces of the components of a protein-protein complex, which we demonstrate to correlate with the modelling difficulty. The identified interface clusters can also be used to generate various combinations of interface-specific restraints for integrative modelling. The ARCTIC-3D software is freely available at https://github.com/haddocking/arctic3d and can be accessed as a web-service at https://wenmr.science.uu.nl/arctic-3d\nSimulations of sequence evolution: how (un)realistic they really are and why\nAuthors: Trost, J.; Haag, J.; Hoehler, D.; Nesterenko, L.; Jacob, L.; Stamatakis, A.; Boussau, B.\nScore: 24.6, Published: 2023-07-12 DOI: 10.1101/2023.07.11.548509\nMotivationSimulating sequence evolution plays an important role in the development and evaluation of phylogenetic inference tools. Naturally, the simulated data needs to be as realistic as possible to be indicative of the performance of the developed tools on empirical data. Over the years, numerous phylogenetic sequence simulators, employing various models of evolution, have been published with the goal to simulate such empirical-like data. In this study, we simulated DNA and protein Multiple Sequence Alignments (MSAs) under increasingly complex models of evolution with and without insertion/deletion (indel) events using a state-of-the-art sequence simulator. We assessed their realism by quantifying how well supervised learning methods are able to predict whether a given MSA is simulated or empirical. ResultsOur results show that we can distinguish between empirical and simulated MSAs with high accuracy using two distinct and independently developed classification approaches across all tested models of sequence evolution. Our findings suggest that the current state-of-the-art models fail to accurately replicate the process of evolution. Data and Code AvailabilityAll simulated and empirical MSAs, as well as all analysis results, are available at https://cme.h-its.org/exelixis/material/simulation_study.tar.gz. All scripts required to reproduce our results are available at https://github.com/tschuelia/SimulationStudy and https://github.com/JohannaTrost/seqsharp. Contactjulia.haag@h-its.org\nTutorial: guidelines for manual cell type annotation of single-cell multi-omics datasets using interactive software\nAuthors: Kim, Y. J.; Tarashansky, A.; Liang, K.; Urisko, M.; Dorman, L.; Borja, M.; Neff, N.; Pisco, A. O.; Granados, A.\nScore: 22.1, Published: 2023-07-12 DOI: 10.1101/2023.07.11.548639\nAssigning cell identity to clusters of single cells is an essential step towards extracting biological insights from many genomics datasets. Although annotation workflows for datasets built with a single modality are well established, limitations exist in annotating cell types in datasets with multiple modalities due to the need for a framework to exploit them jointly. While, in principle, different modalities could convey complementary information about cell identity, it is unclear to what extent they can be combined to improve the accuracy and resolution of cell type annotations. Here, we present a conceptual framework to examine and jointly interrogate distinct modalities to identify cell types. We integrated our framework into a series of vignettes, using immune cells as a well-studied example, and demonstrate cell type annotation workflows ranging from using single-cell RNA-seq datasets alone, to using multiple modalities such as single-cell Multiome (RNA and chromatin accessibility), CITE-seq (RNA and surface proteins). In some cases, one or other single modality is superior to the other for identification of specific cell types, in others combining the two modalities improves resolution and the ability to identify finer subpopulations. Finally, we use interactive software from CZ CELLxGENE community tools to visualize and integrate histological and spatial transcriptomic data.\nIndexing and searching petabyte-scale nucleotide resources\nAuthors: Shiryev, S. A.; Agarwala, R.\nScore: 17.6, Published: 2023-07-09 DOI: 10.1101/2023.07.09.547343\nSearching vast and rapidly growing sets of nucleotide content in data resources, such as runs in Sequence Read Archive and assemblies for whole genome shotgun sequencing projects in GenBank, is currently impractical in any reasonable amount of time or resources available to most researchers. We present Pebblescout, a tool that navigates such content by providing indexing and search capabilities. Indexing uses dense sampling of the sequences in the resource. Search finds subjects that have short sequence matches to a user query with well-defined guarantees. Reported subjects are ranked using a score that considers the informativeness of the matches. Six databases that index over 3.5 petabases were created and used to illustrate the functionality of Pebblescout. Here we show that Pebblescout provides new research opportunities and a data-driven way for finding relevant subsets of large nucleotide resources for analysis, some of which are missed when relying only on sample metadata or tools using pre-defined reference sequences. For two computationally intensive published studies, we show that Pebblescout rejects a significant number of runs analyzed without changing the conclusions of these studies and finds additional relevant runs. A pilot web service for interactively searching the six databases is freely available at https://pebblescout.ncbi.nlm.nih.gov/\n",
  "wordCount" : "2214",
  "inLanguage": "en",
  "datePublished": "2023-07-16T10:39:40Z",
  "dateModified": "2023-07-16T10:39:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on July 16, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548308">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548308" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548308">
        <p class="paperTitle">Petascale Homology Search for Structure Prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548308" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548308" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, S.; Kim, G.; Levy Karin, E.; Mirdita, M.; Park, S.; Chikhi, R.; Babaian, A.; Kryshtafovych, A.; Steinegger, M.</p>
        <p class="info">Score: 70.8, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548308' target='https://doi.org/10.1101/2023.07.10.548308'> 10.1101/2023.07.10.548308</a></p>
        <p class="abstract">The recent CASP15 competition highlighted the critical role of multiple sequence alignments (MSAs) in protein structure prediction, as demonstrated by the success of the top AlphaFold2-based prediction methods. To push the boundaries of MSA utilization, we conducted a petabase-scale search of the Sequence Read Archive (SRA), resulting in gigabytes of aligned homologs for CASP15 targets. These were merged with default MSAs produced by ColabFold-search and provided to ColabFold-predict. By using SRA data, we achieved highly accurate predictions (GDT_TS &gt; 70) for 66% of the non-easy targets, whereas using ColabFold-search default MSAs scored highly in only 52%. Next, we tested the effect of deep homology search and ColabFolds advanced features, such as more recycles, on prediction accuracy. While SRA homologs were most significant for improving ColabFolds CASP15 ranking from 11th to 3rd place, other strategies contributed too. We analyze these in the context of existing strategies to improve prediction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548136">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548136" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548136">
        <p class="paperTitle">Efficient High-Quality Metagenome Assembly from Long Accurate Reads using Minimizer-space de Bruijn Graphs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548136" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548136" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Benoit, G.; Raguideau, S.; James, R.; Phillippy, A. M.; Chikhi, R.; Quince, C.</p>
        <p class="info">Score: 55.7, Published: 2023-07-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548136' target='https://doi.org/10.1101/2023.07.07.548136'> 10.1101/2023.07.07.548136</a></p>
        <p class="abstract">We introduce a novel metagenomics assembler for high-accuracy long reads. Our approach, implemented as metaMDBG, combines highly efficient de Bruijn graph assembly in minimizer space, with both a multi-k&#39; approach for dealing with variations in genome coverage depth and an abundance-based filtering strategy for simplifying strain complexity. The resulting algorithm is more efficient than the state-of-the-art but with better assembly results. metaMDBG was 1.5 to 12 times faster than competing assemblers and requires between one-tenth and one-thirtieth of the memory across a range of data sets. We obtained up to twice as many high-quality circularised prokaryotic metagenome assembled genomes (MAGs) on the most complex communities, and a better recovery of viruses and plasmids. metaMDBG performs particularly well for abundant organisms whilst being robust to the presence of strain diversity. The result is that for the first time it is possible to efficiently reconstruct the majority of complex communities by abundance as nearcomplete MAGs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.547989">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.547989" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.547989">
        <p class="paperTitle">Deep dynamical modelling of developmental trajectories with temporal transcriptomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.547989" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.547989" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Maizels, R. J.; Snell, D. M.; Briscoe, J.</p>
        <p class="info">Score: 59.5, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.547989' target='https://doi.org/10.1101/2023.07.06.547989'> 10.1101/2023.07.06.547989</a></p>
        <p class="abstract">Developmental cell fate decisions are dynamic processes driven by the complex behaviour of gene regulatory networks. A challenge in studying these processes using single-cell genomics is that the data provides only a static snapshot with no detail of dynamics. Metabolic labelling and splicing can provide time-resolved information, but current methods have limitations. Here, we present experimental and computational methods that overcome these limitations to allow dynamical modelling of gene expression from single-cell data. We developed sci-FATE2, an optimised metabolic labelling method that substantially increases data quality, and profiled approximately 45,000 embryonic stem cells differentiating into multiple neural tube identities. To recover dynamics, we developed velvet, a deep learning framework that extends beyond instantaneous velocity estimation by modelling gene expression dynamics through a neural stochastic differential equation system within a variational autoencoder. Velvet outperforms current velocity tools across quantitative benchmarks, and predicts trajectory distributions that accurately recapitulate underlying dataset distributions while conserving known biology. Velvet trajectory distributions capture dynamical aspects such as decision boundaries between alternative fates and correlative gene regulatory structure. Using velvet to provide a dynamical description of in vitro neural patterning, we highlight a process of sequential decision making and fate-specific patterns of developmental signalling. Together, these experimental and computational methods recast single-cell analyses from descriptions of observed data distributions to models of the dynamics that generated them, providing a new framework for investigating developmental gene regulation and cell fate decisions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548105">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548105" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548105">
        <p class="paperTitle">Semi-supervised integration of single-cell transcriptomics data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548105" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548105" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Andreatta, M.; Herault, L.; Gueguen, P.; Gfeller, D.; Berenstein, A. J.; Carmona, S. J.</p>
        <p class="info">Score: 34.9, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548105' target='https://doi.org/10.1101/2023.07.07.548105'> 10.1101/2023.07.07.548105</a></p>
        <p class="abstract">Single-cell sequencing technologies offer unprecedented opportunities to characterize the complexity of biological samples with high resolution. At the same time, variations in sample processing and experimental protocols introduce technical variability - or &#34;batch effects&#34; - in the molecular readouts, hindering comparative analyses across samples and individuals. Although batch effect correction methods are routinely applied in single-cell omics analyses, data integration often leads to overcorrection, resulting in the loss of true biological variability. In this study, we present STACAS v2, a semi-supervised scRNA-seq data integration method that leverages prior knowledge in the form of cell type annotations to preserve biological variance. Through an open and reproducible benchmarking pipeline, we show that semi-supervised STACAS outperforms popular unsupervised methods such as Harmony, FastMNN, Seurat v4, scVI, and Scanorama, as well as supervised methods such as scANVI and scGen. Notably, STACAS is robust to incomplete and imprecise cell type annotations, which are commonly encountered in real-life integration tasks. Highlighting its scalability, we successfully applied semi-supervised STACAS to construct a high-resolution map of tumor-infiltrating CD8 T cells encompassing over 500,000 cells from 265 individuals. Based on our findings, we argue that the incorporation of prior cell type information should be a common practice in single-cell data integration, and we provide a flexible framework for semi-supervised batch effect correction. STACAS seamlessly integrates with Seurat pipelines and can be run with one command: Run.STACAS(seurat.list, cell.labels).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.545008">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.545008" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.545008">
        <p class="paperTitle">Identifying Protein Conformational States in the PDB and Comparison to AlphaFold2 Predictions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.545008" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.545008" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ellaway, J. I. J.; Anyango, S.; Nair, S.; Zaki, H. A.; Nadzirin, N.; Powell, H. R.; Gutmanas, A.; Varadi, M.; Velankar, S.</p>
        <p class="info">Score: 28.3, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.545008' target='https://doi.org/10.1101/2023.07.13.545008'> 10.1101/2023.07.13.545008</a></p>
        <p class="abstract">Proteins, as molecular machines, are necessarily dynamic macromolecules that carry out essential cellular functions. Recognising their stable conformations is important for understanding the molecular mechanisms of disease. While AI-based computational methods have enabled protein structure prediction, the prediction of protein dynamics remains a challenge. Here, we present a deterministic pipeline that clusters experimentally determined protein structures to comprehensively recognise conformational states across the Protein Data Bank. Our approach clusters protein chains based on a GLObal CONformation (GLOCON) difference score, which is computed from pairwise C-alpha distances. By superposing the clustered structures, differences and similarities in conformational states can be observed. Additionally, we offer users the ability to superpose predicted models from the AlphaFold Database to the clusters of PDB structures. This clustering pipeline significantly advances researchers&#39; ability to explore the conformational landscape within the PDB. All clustered and superposed models can be viewed in Mol* on the PDBe Knowledge Base website, or accessed in as raw annotations via our GraphAPI and FTP server. The clustering package is made available as an open-source Python3 package under the Apache-2.0 license.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.09.548272">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.09.548272" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.09.548272">
        <p class="paperTitle">Quantifying Cell-State Densities in Single-Cell Phenotypic Landscapes using Mellon</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.09.548272" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.09.548272" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Otto, D. J.; Jordan, C.; Dury, B.; Dien, C.; Setty, M.</p>
        <p class="info">Score: 26.9, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.09.548272' target='https://doi.org/10.1101/2023.07.09.548272'> 10.1101/2023.07.09.548272</a></p>
        <p class="abstract">Cell-state density characterizes the distribution of cells along phenotypic landscapes and is crucial for unraveling the mechanisms that drive cellular differentiation, regeneration, and disease. Here, we present Mellon, a novel computational algorithm for high-resolution estimation of cell-state densities from single-cell data. We demonstrate Mellons efficacy by dissecting the density landscape of various differentiating systems, revealing a consistent pattern of high-density regions corresponding to major cell types intertwined with low-density, rare transitory states. Utilizing hematopoietic stem cell fate specification to B-cells as a case study, we present evidence implicating enhancer priming and the activation of master regulators in the emergence of these transitory states. Mellon offers the flexibility to perform temporal interpolation of time-series data, providing a detailed view of cell-state dynamics during the inherently continuous developmental processes. Scalable and adaptable, Mellon facilitates density estimation across various single-cell data modalities, scaling linearly with the number of cells. Our work underscores the importance of cell-state density in understanding the differentiation processes, and the potential of Mellon to provide new insights into the regulatory mechanisms guiding cellular fate decisions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548477">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548477" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548477">
        <p class="paperTitle">ARCTIC-3D: Automatic Retrieval and ClusTering of Interfaces in Complexes from 3D structural information</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548477" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548477" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Giulini, M.; Honorato, R. V.; Rivera, J. L.; Bonvin, A. M. J. J.</p>
        <p class="info">Score: 26.7, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548477' target='https://doi.org/10.1101/2023.07.10.548477'> 10.1101/2023.07.10.548477</a></p>
        <p class="abstract">The formation of a stable complex between proteins lies at the core of a wide variety of biological processes and has been the focus of countless experiments. The huge amount of information contained in the protein structural interactome in the Protein Data Bank can now be used to characterise and classify the existing biological interfaces. We here introduce ARCTIC-3D, a fast and user-friendly data mining and clustering software to retrieve data and rationalise the interface information associated with the protein input data. We demonstrate its use by various examples ranging from showing the increased interaction complexity of eukaryotic proteins, 20% of which on average have more than 3 different interfaces compared to only 10% for prokaryotes, to associating different functions to different interfaces. In the context of modelling biomolecular assemblies, we introduce the concept of &#34;recognition entropy&#34;, related to the number of possible interfaces of the components of a protein-protein complex, which we demonstrate to correlate with the modelling difficulty. The identified interface clusters can also be used to generate various combinations of interface-specific restraints for integrative modelling. The ARCTIC-3D software is freely available at https://github.com/haddocking/arctic3d and can be accessed as a web-service at https://wenmr.science.uu.nl/arctic-3d</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.548509">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.548509" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.548509">
        <p class="paperTitle">Simulations of sequence evolution: how (un)realistic they really are and why</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.548509" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.548509" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Trost, J.; Haag, J.; Hoehler, D.; Nesterenko, L.; Jacob, L.; Stamatakis, A.; Boussau, B.</p>
        <p class="info">Score: 24.6, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.548509' target='https://doi.org/10.1101/2023.07.11.548509'> 10.1101/2023.07.11.548509</a></p>
        <p class="abstract">MotivationSimulating sequence evolution plays an important role in the development and evaluation of phylogenetic inference tools. Naturally, the simulated data needs to be as realistic as possible to be indicative of the performance of the developed tools on empirical data. Over the years, numerous phylogenetic sequence simulators, employing various models of evolution, have been published with the goal to simulate such empirical-like data. In this study, we simulated DNA and protein Multiple Sequence Alignments (MSAs) under increasingly complex models of evolution with and without insertion/deletion (indel) events using a state-of-the-art sequence simulator. We assessed their realism by quantifying how well supervised learning methods are able to predict whether a given MSA is simulated or empirical.

ResultsOur results show that we can distinguish between empirical and simulated MSAs with high accuracy using two distinct and independently developed classification approaches across all tested models of sequence evolution. Our findings suggest that the current state-of-the-art models fail to accurately replicate the process of evolution.

Data and Code AvailabilityAll simulated and empirical MSAs, as well as all analysis results, are available at https://cme.h-its.org/exelixis/material/simulation_study.tar.gz. All scripts required to reproduce our results are available at https://github.com/tschuelia/SimulationStudy and https://github.com/JohannaTrost/seqsharp.

Contactjulia.haag@h-its.org</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.548639">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.548639" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.548639">
        <p class="paperTitle">Tutorial: guidelines for manual cell type annotation of single-cell multi-omics datasets using interactive software</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.548639" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.548639" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kim, Y. J.; Tarashansky, A.; Liang, K.; Urisko, M.; Dorman, L.; Borja, M.; Neff, N.; Pisco, A. O.; Granados, A.</p>
        <p class="info">Score: 22.1, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.548639' target='https://doi.org/10.1101/2023.07.11.548639'> 10.1101/2023.07.11.548639</a></p>
        <p class="abstract">Assigning cell identity to clusters of single cells is an essential step towards extracting biological insights from many genomics datasets. Although annotation workflows for datasets built with a single modality are well established, limitations exist in annotating cell types in datasets with multiple modalities due to the need for a framework to exploit them jointly. While, in principle, different modalities could convey complementary information about cell identity, it is unclear to what extent they can be combined to improve the accuracy and resolution of cell type annotations. Here, we present a conceptual framework to examine and jointly interrogate distinct modalities to identify cell types. We integrated our framework into a series of vignettes, using immune cells as a well-studied example, and demonstrate cell type annotation workflows ranging from using single-cell RNA-seq datasets alone, to using multiple modalities such as single-cell Multiome (RNA and chromatin accessibility), CITE-seq (RNA and surface proteins). In some cases, one or other single modality is superior to the other for identification of specific cell types, in others combining the two modalities improves resolution and the ability to identify finer subpopulations. Finally, we use interactive software from CZ CELLxGENE community tools to visualize and integrate histological and spatial transcriptomic data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.09.547343">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.09.547343" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.09.547343">
        <p class="paperTitle">Indexing and searching petabyte-scale nucleotide resources</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.09.547343" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.09.547343" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shiryev, S. A.; Agarwala, R.</p>
        <p class="info">Score: 17.6, Published: 2023-07-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.09.547343' target='https://doi.org/10.1101/2023.07.09.547343'> 10.1101/2023.07.09.547343</a></p>
        <p class="abstract">Searching vast and rapidly growing sets of nucleotide content in data resources, such as runs in Sequence Read Archive and assemblies for whole genome shotgun sequencing projects in GenBank, is currently impractical in any reasonable amount of time or resources available to most researchers. We present Pebblescout, a tool that navigates such content by providing indexing and search capabilities. Indexing uses dense sampling of the sequences in the resource. Search finds subjects that have short sequence matches to a user query with well-defined guarantees. Reported subjects are ranked using a score that considers the informativeness of the matches. Six databases that index over 3.5 petabases were created and used to illustrate the functionality of Pebblescout. Here we show that Pebblescout provides new research opportunities and a data-driven way for finding relevant subsets of large nucleotide resources for analysis, some of which are missed when relying only on sample metadata or tools using pre-defined reference sequences. For two computationally intensive published studies, we show that Pebblescout rejects a significant number of runs analyzed without changing the conclusions of these studies and finds additional relevant runs. A pilot web service for interactively searching the six databases is freely available at https://pebblescout.ncbi.nlm.nih.gov/</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
