<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="LazyAF, a pipeline for accessible medium-scale in silico prediction of protein-protein interactions
Authors: McLean, T. C.
Score: 102.6, Published: 2024-02-01 DOI: 10.1101/2024.01.29.577767
Artificial intelligence has revolutionized the field of protein structure prediction. However, with more powerful and complex software being developed, it is accessibility and ease of use rather than capability that is quickly becoming a limiting factor to end users. Here, I present a Google Colaboratory-based pipeline, named LazyAF, which integrates the existing ColabFold BATCH to streamline the process of medium-scale protein-protein interaction prediction.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="LazyAF, a pipeline for accessible medium-scale in silico prediction of protein-protein interactions
Authors: McLean, T. C.
Score: 102.6, Published: 2024-02-01 DOI: 10.1101/2024.01.29.577767
Artificial intelligence has revolutionized the field of protein structure prediction. However, with more powerful and complex software being developed, it is accessibility and ease of use rather than capability that is quickly becoming a limiting factor to end users. Here, I present a Google Colaboratory-based pipeline, named LazyAF, which integrates the existing ColabFold BATCH to streamline the process of medium-scale protein-protein interaction prediction." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-07T10:40:15+00:00" />
<meta property="article:modified_time" content="2024-02-07T10:40:15+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="LazyAF, a pipeline for accessible medium-scale in silico prediction of protein-protein interactions
Authors: McLean, T. C.
Score: 102.6, Published: 2024-02-01 DOI: 10.1101/2024.01.29.577767
Artificial intelligence has revolutionized the field of protein structure prediction. However, with more powerful and complex software being developed, it is accessibility and ease of use rather than capability that is quickly becoming a limiting factor to end users. Here, I present a Google Colaboratory-based pipeline, named LazyAF, which integrates the existing ColabFold BATCH to streamline the process of medium-scale protein-protein interaction prediction."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "LazyAF, a pipeline for accessible medium-scale in silico prediction of protein-protein interactions\nAuthors: McLean, T. C.\nScore: 102.6, Published: 2024-02-01 DOI: 10.1101/2024.01.29.577767\nArtificial intelligence has revolutionized the field of protein structure prediction. However, with more powerful and complex software being developed, it is accessibility and ease of use rather than capability that is quickly becoming a limiting factor to end users. Here, I present a Google Colaboratory-based pipeline, named LazyAF, which integrates the existing ColabFold BATCH to streamline the process of medium-scale protein-protein interaction prediction.",
  "keywords": [
    
  ],
  "articleBody": " LazyAF, a pipeline for accessible medium-scale in silico prediction of protein-protein interactions\nAuthors: McLean, T. C.\nScore: 102.6, Published: 2024-02-01 DOI: 10.1101/2024.01.29.577767\nArtificial intelligence has revolutionized the field of protein structure prediction. However, with more powerful and complex software being developed, it is accessibility and ease of use rather than capability that is quickly becoming a limiting factor to end users. Here, I present a Google Colaboratory-based pipeline, named LazyAF, which integrates the existing ColabFold BATCH to streamline the process of medium-scale protein-protein interaction prediction. I apply LazyAF to predict the interactome of the 76 proteins encoded on a broad-host-range multi-drug resistance plasmid RK2, demonstrating the ease and accessibility the pipeline provides. AvailabilityLazyAF is freely available at https://github.com/ThomasCMcLean/LazyAF\nProtein language models learn evolutionary statistics of interacting sequence motifs\nAuthors: Zhang, Z.; Wayment-Steele, H. K.; Brixi, G.; Wang, H.; Dal Peraro, M.; Kern, D.; Ovchinnikov, S.\nScore: 84.0, Published: 2024-01-31 DOI: 10.1101/2024.01.30.577970\nProtein language models (pLMs) have emerged as potent tools for predicting and designing protein structure and function, and the degree to which these models fundamentally understand the inherent biophysics of protein structure stands as an open question. Motivated by a discovery that pLM-based structure predictors erroneously predict nonphysical structures for protein isoforms, we investigated the nature of sequence context needed for contact predictions in the pLM ESM-2. We demonstrate by use of a \"categorical Jacobian\" calculation that ESM-2 stores statistics of coevolving residues, analogously to simpler modelling approaches like Markov Random Fields and Multivariate Gaussian models. We further investigated how ESM-2 \"stores\" information needed to predict contacts by comparing sequence masking strategies, and found that providing local windows of sequence information allowed ESM-2 to best recover predicted contacts. This suggests that pLMs predict contacts by storing motifs of pairwise contacts. Our investigation highlights the limitations of current pLMs and underscores the importance of understanding the underlying mechanisms of these models. Significance StatementProtein language models (pLMs) have exhibited remarkable capabilities in protein structure prediction and design. However, the extent to which they comprehend the intrinsic biophysics of protein structures remains uncertain. We present a suite of analyses that dissect how the flagship pLM ESM-2 predicts structure. Motivated by a consistent error of protein isoforms predicted as structured fragments, we developed a completely unsupervised method to uniformly evaluate any protein language model that allows for us to compare coevolutionary statistics to older linear models. We further identified t hat E SM-2 a ppears to have a precise context size that is needed to predict inter-residue contacts. Our study highlights the current limitations of pLMs and contributes to a deeper understanding of their underlying mechanisms, paving the way for more reliable protein structure predictions.\nStaVia: Spatially and temporally aware cartography with higher order random walks for cell atlases\nAuthors: Stassen, S. V.; Kobashi, M.; Huang, Y.; Ho, J. W. K.; Tsia, K.\nScore: 64.1, Published: 2024-01-31 DOI: 10.1101/2024.01.29.577871\nSingle-cell atlases are critical for unraveling the cellular basis of health and disease, yet their sheer diversity and vast data landscape in time and space pose daunting computational challenges pertaining to the delineation of multiple simultaneously emerging lineages, the integration of spatial and temporal information, and the visualization of trajectories across large atlases with enough resolution to observe localized transitions. To tackle this intricacy, we introduce StaVia, a computational framework that synergizes multi-faceted single-cell data--spanning time-series data, spatial gene expression patterns, and directional trends from RNA velocity-- with higher-order random walks that leverage the memory of cells past states. StaVia fuses this method with a cartographic \"Atlas View\" that offers intuitive graph visualization, simultaneously capturing the nuanced details of cellular development at single-cell resolution as well as the broader connectivity of cell lineages, avoiding common pitfalls of merged distinct trajectories or missed transitional states seen in existing methods which are all memoryless. Notably, we demonstrate that StaVia unlocks new insights into placode development, radial glia pluripotency during neurulation, and the transitions pivotal to these processes in a large-scale Zebrafish developmental atlas. StaVia also allows spatially aware cartography that captures relationships between cell populations based on their spatial location as well as their gene expressions in a MERFISH dataset - underscoring its potential to dissect complex biological landscapes in both spatial and temporal contexts.\nAccurate quantification of single-cell and single-nucleus RNA-seq transcripts using distinguishing flanking k-mers\nAuthors: Eldjarn Hjörleifsson, K.; Sullivan, D. K.; Swarna, N. P.; Holley, G.; Melsted, P.; Pachter, L.\nScore: 89.2, Published: 2024-01-26 DOI: 10.1101/2022.12.02.518832\nIn single-cell and single-nucleus RNA sequencing, the coexistence of nascent (unprocessed) and mature (processed) mRNA poses challenges in accurate read mapping and the interpretation of count matrices. The traditional transcriptome reference, defining the region of interest in bulk RNA-seq, restricts its focus to mature mRNA transcripts. This restriction leads to two problems: reads originating outside of the region of interest are prone to mismapping within this region, and additionally, such external reads cannot be matched to specific transcript targets. Expanding the region of interest to encompass both nascent and mature mRNA transcript targets provides a more comprehensive framework for RNA-seq analysis. Here, we introduce the concept of distinguishing flanking k-mers (DFKs) to improve mapping of sequencing reads. We have developed an algorithm to identify DFKs, which serve as a sophisticated background filter, enhancing the accuracy of mRNA quantification. This dual strategy of an expanded region of interest coupled with the use of DFKs enhances the precision in quantifying both mature and nascent mRNA molecules, as well as in delineating reads of ambiguous status.\nPhylogenetic tree statistics: a systematic overview using the new R package 'treestats'\nAuthors: Janzen, T.; Etienne, R. S.\nScore: 85.8, Published: 2024-01-29 DOI: 10.1101/2024.01.24.576848\nPhylogenetic trees are believed to contain a wealth of information on diversification processes. Comparing phylogenetic trees is not straightforward due to their high dimensionality. Researchers have therefore defined a wide range of one-dimensional summary statistics. However, it remains unexplored to what extent these summary statistics cover the same underlying information and what summary statistics best explain observed variation across phylogenies. Furthermore, a large subset of available summary statistics focusses on measuring the topological features of a phylogenetic tree, but are often only explored at the extreme edge cases of the fully balanced or unbalanced tree and not for trees of intermediate balance. Here, we introduce a new R package that provides speed optimized code to compute 54 summary statistics. We study correlations between summary statistics on empirical trees and on trees simulated using several diversification models. Furthermore, we introduce an algorithm to create intermediately balanced trees in a well-defined manner, in order to explore variation in summary statistics across a balance gradient. We find that almost all summary statistics are correlated with tree size, and it is difficult if not impossible to correct for tree size, unless the tree generating model is known. Furthermore, we find that across empirical and simulated trees, at least two large clusters of correlated summary statistics can be found, where statistics group together based on information used (topology or branching times). However, the finer grained correlation structure appears to depend strongly on either the taxonomic group studied (in empirical studies) or the diversification model (in simulation studies). Nevertheless, we can identify multiple groups of summary statistics that are strongly and consistently correlated, indicating that these statistics measure the same underlying property of a tree. Lastly, we find that almost all topological summary statistics vary non-linearly and sometimes even non-monotonically with our intuitive balance gradient. Therefore, in order to avoid introducing biases and missing underlying information, we advocate for selecting as many summary statistics as possible in phylogenetic analyses. With the introduction of the treestats package, which provides fast and reliable calculations, such an approach is now routinely possible.\nMetagenomic estimation of dietary intake from human stool\nAuthors: Diener, C.; Gibbons, S. M.\nScore: 33.7, Published: 2024-02-06 DOI: 10.1101/2024.02.02.578701\nDietary intake is tightly coupled to gut microbiota composition, human metabolism, and to the incidence of virtually all major chronic diseases. Dietary and nutrient intake are usually quantified using dietary questionnaires, which tend to focus on broad food categories, suffer from self-reporting biases, and require strong compliance from study participants. Here, we present MEDI (Metagenomic Estimation of Dietary Intake): a method for quantifying dietary intake using food-derived DNA in stool metagenomes. We show that food items can be accurately detected in metagenomic shotgun sequencing data, even when present at low abundances (\u003e10 reads). Furthermore, we show how dietary intake, in terms of DNA abundance from specific organisms, can be converted into a detailed metabolic representation of nutrient intake. MEDI could identify the onset of solid food consumption in infants and it accurately predicted food questionnaire responses in an adult population. Additionally, we were able to identify specific dietary features associated with metabolic syndrome in a large clinical cohort, providing a proof-of-concept for detailed quantification of individual-specific dietary patterns without the need for questionnaires.\nSpotiflow: accurate and efficient spot detection for imaging-based spatial transcriptomics with stereographic flow regression\nAuthors: Dominguez Mantes, A.; Herrera, A.; Khven, I.; Schlaeppi, A.; Kyriacou, E.; Tsissios, G.; Aztekin, C.; Ligner, J.; La Manno, G.; Weigert, M.\nScore: 26.8, Published: 2024-02-05 DOI: 10.1101/2024.02.01.578426\nThe identification of spot-like structures in large and noisy microscopy images is an important task in many life science techniques, and it is essential to their quantitative performance. For example, imaging-based spatial transcriptomics (iST) methods rely critically on the accurate detection of millions of transcripts in low signal-to-noise ratio (SNR) images. While recent developments in computer vision have revolutionized many bioimage tasks, currently adopted spot detection approaches for iST still rely on classical signal processing methods that are fragile and require manually tuning. In this work we introduce Spotiflow, a deep-learning method that casts the spot-detection problem as a multiscale stereographic flow regression problem that yields subpixel-accurate localizations. Spotiflow is robust to different noise conditions and generalizes across different chemistries while being up to an order-of-magnitude more time and memory efficient than commonly used methods. We show the efficacy of Spotiflow by comprehensive quantitative comparisons against other methods on a variety of datasets and demonstrate the impact of its increased accuracy on the biological conclusions drawn from iST and live imaging experiments. Spotiflow is available as an easy-to-use Python library as well as a napari plugin at http://www.github.com/weigertlab/spotiflow\nOrthology inference at scale with FastOMA\nAuthors: Majidian, S.; Nevers, Y.; Yazdizadeh Kharrazi, A.; Warwick Vesztrocy, A.; Pascarelli, S.; Moi, D.; Glover, N.; Altenhoff, A. M.; Dessimoz, C.\nScore: 25.5, Published: 2024-01-31 DOI: 10.1101/2024.01.29.577392\nThe surge in genome data, with ongoing efforts aiming to sequence 1.5M eukaryotes in a decade, could revolutionise genomics, revealing the origins, evolution, and genetic innovations of biological processes. Yet, traditional genomics methods scale poorly with such large datasets. Addressing this, \"FastOMA\" provides linear scalability, enabling the processing of thousands of eukaryotic genomes within a day. FastOMA maintains the high accuracy and resolution of the well-established OMA approach in benchmarks. FastOMA is available at https://github.com/DessimozLab/FastOMA/.\nSEraster: a rasterization preprocessing framework for scalable spatial omics data analysis\nAuthors: Aihara, G.; Clifton, K.; Chen, M.; Atta, L.; Miller, B. F.; Fan, J.\nScore: 25.1, Published: 2024-02-05 DOI: 10.1101/2024.02.01.578436\nMotivation Spatial omics data demand computational analysis but many analysis tools have computational resource requirements that increase with the number of cells analyzed. This presents scalability challenges as researchers use spatial omics technologies to profile millions of cells. Results To enhance the scalability of spatial omics data analysis, we developed a rasterization preprocessing framework called SEraster that aggregates cellular information into spatial pixels. We apply SEraster to both real and simulated spatial omics data prior to spatial variable gene expression analysis to demonstrate that such preprocessing can reduce resource requirements while maintaining high performance. We further integrate SEraster with existing analysis tools to characterize cell-type spatial cooccurrence. Finally, we apply SEraster to enable analysis of a mouse pup spatial omics dataset with over a million cells to identify tissue-level and cell-type-specific spatially variable genes as well as cooccurring cell-types that recapitulate expected organ structures. Availability and implementation Source code is available on GitHub (https://github.com/JEFworks-Lab/SEraster) with additional tutorials at https://JEF.works/SEraster.\nConway-Bromage-Lyndon (CBL): an exact, dynamic representation of k-mer sets\nAuthors: Martayan, I.; Cazaux, B.; Limasset, A.; Marchet, C.\nScore: 21.0, Published: 2024-01-31 DOI: 10.1101/2024.01.29.577700\nIn this paper, we introduce the Conway-Bromage-Lyndon (CBL) structure, a compressed, dynamic and exact method for representing k-mer sets. Originating from Conway and Bromages concept, CBL innovatively employs the smallest cyclic rotations of k-mers, akin to Lyndon words, to leverage lexico-graphic redundancies. In order to support dynamic operations and set operations, we propose a dynamic bit vector structure that draws a parallel with Elias-Fanos scheme. This structure is encapsulated in a Rust library, demonstrating a balanced blend of construction efficiency, cache locality, and compression. Our findings suggest that CBL outperforms existing k-mer set methods, particularly in dynamic scenarios. Unique to this work, CBL stands out as the only known exact k-mer structure offering in-place set operations. Its different combined abilities position it as a flexible Swiss knife structure for k-mer set management. Availability: https://github.com/imartayan/CBL\n",
  "wordCount" : "2154",
  "inLanguage": "en",
  "datePublished": "2024-02-07T10:40:15Z",
  "dateModified": "2024-02-07T10:40:15Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 7, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577767">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577767" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577767">
        <p class="paperTitle">LazyAF, a pipeline for accessible medium-scale in silico prediction of protein-protein interactions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577767" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577767" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McLean, T. C.</p>
        <p class="info">Score: 102.6, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577767' target='https://doi.org/10.1101/2024.01.29.577767'> 10.1101/2024.01.29.577767</a></p>
        <p class="abstract">Artificial intelligence has revolutionized the field of protein structure prediction. However, with more powerful and complex software being developed, it is accessibility and ease of use rather than capability that is quickly becoming a limiting factor to end users. Here, I present a Google Colaboratory-based pipeline, named LazyAF, which integrates the existing ColabFold BATCH to streamline the process of medium-scale protein-protein interaction prediction. I apply LazyAF to predict the interactome of the 76 proteins encoded on a broad-host-range multi-drug resistance plasmid RK2, demonstrating the ease and accessibility the pipeline provides.

AvailabilityLazyAF is freely available at https://github.com/ThomasCMcLean/LazyAF</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.30.577970">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.30.577970" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.30.577970">
        <p class="paperTitle">Protein language models learn evolutionary statistics of interacting sequence motifs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.30.577970" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.30.577970" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, Z.; Wayment-Steele, H. K.; Brixi, G.; Wang, H.; Dal Peraro, M.; Kern, D.; Ovchinnikov, S.</p>
        <p class="info">Score: 84.0, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.30.577970' target='https://doi.org/10.1101/2024.01.30.577970'> 10.1101/2024.01.30.577970</a></p>
        <p class="abstract">Protein language models (pLMs) have emerged as potent tools for predicting and designing protein structure and function, and the degree to which these models fundamentally understand the inherent biophysics of protein structure stands as an open question. Motivated by a discovery that pLM-based structure predictors erroneously predict nonphysical structures for protein isoforms, we investigated the nature of sequence context needed for contact predictions in the pLM ESM-2. We demonstrate by use of a &#34;categorical Jacobian&#34; calculation that ESM-2 stores statistics of coevolving residues, analogously to simpler modelling approaches like Markov Random Fields and Multivariate Gaussian models. We further investigated how ESM-2 &#34;stores&#34; information needed to predict contacts by comparing sequence masking strategies, and found that providing local windows of sequence information allowed ESM-2 to best recover predicted contacts. This suggests that pLMs predict contacts by storing motifs of pairwise contacts. Our investigation highlights the limitations of current pLMs and underscores the importance of understanding the underlying mechanisms of these models.

Significance StatementProtein language models (pLMs) have exhibited remarkable capabilities in protein structure prediction and design. However, the extent to which they comprehend the intrinsic biophysics of protein structures remains uncertain. We present a suite of analyses that dissect how the flagship pLM ESM-2 predicts structure. Motivated by a consistent error of protein isoforms predicted as structured fragments, we developed a completely unsupervised method to uniformly evaluate any protein language model that allows for us to compare coevolutionary statistics to older linear models. We further identified t hat E SM-2 a ppears to have a precise context size that is needed to predict inter-residue contacts. Our study highlights the current limitations of pLMs and contributes to a deeper understanding of their underlying mechanisms, paving the way for more reliable protein structure predictions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577871">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577871" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577871">
        <p class="paperTitle">StaVia: Spatially and temporally aware cartography with higher order random walks for cell atlases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577871" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577871" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stassen, S. V.; Kobashi, M.; Huang, Y.; Ho, J. W. K.; Tsia, K.</p>
        <p class="info">Score: 64.1, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577871' target='https://doi.org/10.1101/2024.01.29.577871'> 10.1101/2024.01.29.577871</a></p>
        <p class="abstract">Single-cell atlases are critical for unraveling the cellular basis of health and disease, yet their sheer diversity and vast data landscape in time and space pose daunting computational challenges pertaining to the delineation of multiple simultaneously emerging lineages, the integration of spatial and temporal information, and the visualization of trajectories across large atlases with enough resolution to observe localized transitions. To tackle this intricacy, we introduce StaVia, a computational framework that synergizes multi-faceted single-cell data--spanning time-series data, spatial gene expression patterns, and directional trends from RNA velocity-- with higher-order random walks that leverage the memory of cells past states. StaVia fuses this method with a cartographic &#34;Atlas View&#34; that offers intuitive graph visualization, simultaneously capturing the nuanced details of cellular development at single-cell resolution as well as the broader connectivity of cell lineages, avoiding common pitfalls of merged distinct trajectories or missed transitional states seen in existing methods which are all memoryless. Notably, we demonstrate that StaVia unlocks new insights into placode development, radial glia pluripotency during neurulation, and the transitions pivotal to these processes in a large-scale Zebrafish developmental atlas. StaVia also allows spatially aware cartography that captures relationships between cell populations based on their spatial location as well as their gene expressions in a MERFISH dataset - underscoring its potential to dissect complex biological landscapes in both spatial and temporal contexts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.12.02.518832">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.12.02.518832" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.12.02.518832">
        <p class="paperTitle">Accurate quantification of single-cell and single-nucleus RNA-seq transcripts using distinguishing flanking k-mers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.12.02.518832" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.12.02.518832" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eldjarn Hjörleifsson, K.; Sullivan, D. K.; Swarna, N. P.; Holley, G.; Melsted, P.; Pachter, L.</p>
        <p class="info">Score: 89.2, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.12.02.518832' target='https://doi.org/10.1101/2022.12.02.518832'> 10.1101/2022.12.02.518832</a></p>
        <p class="abstract">In single-cell and single-nucleus RNA sequencing, the coexistence of nascent (unprocessed) and mature (processed) mRNA poses challenges in accurate read mapping and the interpretation of count matrices. The traditional transcriptome reference, defining the  region of interest in bulk RNA-seq, restricts its focus to mature mRNA transcripts. This restriction leads to two problems: reads originating outside of the  region of interest are prone to mismapping within this region, and additionally, such external reads cannot be matched to specific transcript targets. Expanding the  region of interest to encompass both nascent and mature mRNA transcript targets provides a more comprehensive framework for RNA-seq analysis. Here, we introduce the concept of distinguishing flanking k-mers (DFKs) to improve mapping of sequencing reads. We have developed an algorithm to identify DFKs, which serve as a sophisticated  background filter, enhancing the accuracy of mRNA quantification. This dual strategy of an expanded region of interest coupled with the use of DFKs enhances the precision in quantifying both mature and nascent mRNA molecules, as well as in delineating reads of ambiguous status.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.576848">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.576848" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.576848">
        <p class="paperTitle">Phylogenetic tree statistics: a systematic overview using the new R package &#39;treestats&#39;</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.576848" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.576848" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Janzen, T.; Etienne, R. S.</p>
        <p class="info">Score: 85.8, Published: 2024-01-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.576848' target='https://doi.org/10.1101/2024.01.24.576848'> 10.1101/2024.01.24.576848</a></p>
        <p class="abstract">Phylogenetic trees are believed to contain a wealth of information on diversification processes. Comparing phylogenetic trees is not straightforward due to their high dimensionality. Researchers have therefore defined a wide range of one-dimensional summary statistics. However, it remains unexplored to what extent these summary statistics cover the same underlying information and what summary statistics best explain observed variation across phylogenies. Furthermore, a large subset of available summary statistics focusses on measuring the topological features of a phylogenetic tree, but are often only explored at the extreme edge cases of the fully balanced or unbalanced tree and not for trees of intermediate balance.

Here, we introduce a new R package that provides speed optimized code to compute 54 summary statistics. We study correlations between summary statistics on empirical trees and on trees simulated using several diversification models. Furthermore, we introduce an algorithm to create intermediately balanced trees in a well-defined manner, in order to explore variation in summary statistics across a balance gradient.

We find that almost all summary statistics are correlated with tree size, and it is difficult if not impossible to correct for tree size, unless the tree generating model is known. Furthermore, we find that across empirical and simulated trees, at least two large clusters of correlated summary statistics can be found, where statistics group together based on information used (topology or branching times). However, the finer grained correlation structure appears to depend strongly on either the taxonomic group studied (in empirical studies) or the diversification model (in simulation studies). Nevertheless, we can identify multiple groups of summary statistics that are strongly and consistently correlated, indicating that these statistics measure the same underlying property of a tree. Lastly, we find that almost all topological summary statistics vary non-linearly and sometimes even non-monotonically with our intuitive balance gradient.

Therefore, in order to avoid introducing biases and missing underlying information, we advocate for selecting as many summary statistics as possible in phylogenetic analyses. With the introduction of the treestats package, which provides fast and reliable calculations, such an approach is now routinely possible.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.02.578701">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.02.578701" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.02.578701">
        <p class="paperTitle">Metagenomic estimation of dietary intake from human stool</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.02.578701" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.02.578701" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Diener, C.; Gibbons, S. M.</p>
        <p class="info">Score: 33.7, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.02.578701' target='https://doi.org/10.1101/2024.02.02.578701'> 10.1101/2024.02.02.578701</a></p>
        <p class="abstract">Dietary intake is tightly coupled to gut microbiota composition, human metabolism, and to the incidence of virtually all major chronic diseases. Dietary and nutrient intake are usually quantified using dietary questionnaires, which tend to focus on broad food categories, suffer from self-reporting biases, and require strong compliance from study participants. Here, we present MEDI (Metagenomic Estimation of Dietary Intake): a method for quantifying dietary intake using food-derived DNA in stool metagenomes. We show that food items can be accurately detected in metagenomic shotgun sequencing data, even when present at low abundances (&gt;10 reads). Furthermore, we show how dietary intake, in terms of DNA abundance from specific organisms, can be converted into a detailed metabolic representation of nutrient intake. MEDI could identify the onset of solid food consumption in infants and it accurately predicted food questionnaire responses in an adult population. Additionally, we were able to identify specific dietary features associated with metabolic syndrome in a large clinical cohort, providing a proof-of-concept for detailed quantification of individual-specific dietary patterns without the need for questionnaires.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578426">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578426" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578426">
        <p class="paperTitle">Spotiflow: accurate and efficient spot detection for imaging-based spatial transcriptomics with stereographic flow regression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578426" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578426" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dominguez Mantes, A.; Herrera, A.; Khven, I.; Schlaeppi, A.; Kyriacou, E.; Tsissios, G.; Aztekin, C.; Ligner, J.; La Manno, G.; Weigert, M.</p>
        <p class="info">Score: 26.8, Published: 2024-02-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578426' target='https://doi.org/10.1101/2024.02.01.578426'> 10.1101/2024.02.01.578426</a></p>
        <p class="abstract">The identification of spot-like structures in large and noisy microscopy images is an important task in many life science techniques, and it is essential to their quantitative performance. For example, imaging-based spatial transcriptomics (iST) methods rely critically on the accurate detection of millions of transcripts in low signal-to-noise ratio (SNR) images. While recent developments in computer vision have revolutionized many bioimage tasks, currently adopted spot detection approaches for iST still rely on classical signal processing methods that are fragile and require manually tuning. In this work we introduce Spotiflow, a deep-learning method that casts the spot-detection problem as a multiscale stereographic flow regression problem that yields subpixel-accurate localizations. Spotiflow is robust to different noise conditions and generalizes across different chemistries while being up to an order-of-magnitude more time and memory efficient than commonly used methods. We show the efficacy of Spotiflow by comprehensive quantitative comparisons against other methods on a variety of datasets and demonstrate the impact of its increased accuracy on the biological conclusions drawn from iST and live imaging experiments. Spotiflow is available as an easy-to-use Python library as well as a napari plugin at http://www.github.com/weigertlab/spotiflow</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577392">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577392" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577392">
        <p class="paperTitle">Orthology inference at scale with FastOMA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577392" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577392" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Majidian, S.; Nevers, Y.; Yazdizadeh Kharrazi, A.; Warwick Vesztrocy, A.; Pascarelli, S.; Moi, D.; Glover, N.; Altenhoff, A. M.; Dessimoz, C.</p>
        <p class="info">Score: 25.5, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577392' target='https://doi.org/10.1101/2024.01.29.577392'> 10.1101/2024.01.29.577392</a></p>
        <p class="abstract">The surge in genome data, with ongoing efforts aiming to sequence 1.5M eukaryotes in a decade, could revolutionise genomics, revealing the origins, evolution, and genetic innovations of biological processes. Yet, traditional genomics methods scale poorly with such large datasets. Addressing this, &#34;FastOMA&#34; provides linear scalability, enabling the processing of thousands of eukaryotic genomes within a day. FastOMA maintains the high accuracy and resolution of the well-established OMA approach in benchmarks. FastOMA is available at https://github.com/DessimozLab/FastOMA/.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578436">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578436" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578436">
        <p class="paperTitle">SEraster: a rasterization preprocessing framework for scalable spatial omics data analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578436" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578436" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aihara, G.; Clifton, K.; Chen, M.; Atta, L.; Miller, B. F.; Fan, J.</p>
        <p class="info">Score: 25.1, Published: 2024-02-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578436' target='https://doi.org/10.1101/2024.02.01.578436'> 10.1101/2024.02.01.578436</a></p>
        <p class="abstract">Motivation Spatial omics data demand computational analysis but many analysis tools have computational resource requirements that increase with the number of cells analyzed. This presents scalability challenges as researchers use spatial omics technologies to profile millions of cells. Results To enhance the scalability of spatial omics data analysis, we developed a rasterization preprocessing framework called SEraster that aggregates cellular information into spatial pixels. We apply SEraster to both real and simulated spatial omics data prior to spatial variable gene expression analysis to demonstrate that such preprocessing can reduce resource requirements while maintaining high performance. We further integrate SEraster with existing analysis tools to characterize cell-type spatial cooccurrence. Finally, we apply SEraster to enable analysis of a mouse pup spatial omics dataset with over a million cells to identify tissue-level and cell-type-specific spatially variable genes as well as cooccurring cell-types that recapitulate expected organ structures. Availability and implementation Source code is available on GitHub (https://github.com/JEFworks-Lab/SEraster) with additional tutorials at https://JEF.works/SEraster.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577700">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577700" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577700">
        <p class="paperTitle">Conway-Bromage-Lyndon (CBL): an exact, dynamic representation of k-mer sets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577700" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577700" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martayan, I.; Cazaux, B.; Limasset, A.; Marchet, C.</p>
        <p class="info">Score: 21.0, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577700' target='https://doi.org/10.1101/2024.01.29.577700'> 10.1101/2024.01.29.577700</a></p>
        <p class="abstract">In this paper, we introduce the Conway-Bromage-Lyndon (CBL) structure, a compressed, dynamic and exact method for representing k-mer sets. Originating from Conway and Bromages concept, CBL innovatively employs the smallest cyclic rotations of k-mers, akin to Lyndon words, to leverage lexico-graphic redundancies. In order to support dynamic operations and set operations, we propose a dynamic bit vector structure that draws a parallel with Elias-Fanos scheme. This structure is encapsulated in a Rust library, demonstrating a balanced blend of construction efficiency, cache locality, and compression. Our findings suggest that CBL outperforms existing k-mer set methods, particularly in dynamic scenarios. Unique to this work, CBL stands out as the only known exact k-mer structure offering in-place set operations. Its different combined abilities position it as a flexible Swiss knife structure for k-mer set management. Availability: https://github.com/imartayan/CBL</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
