<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="NanoPyx: super-fast bioimage analysis powered by adaptive machine learning
Authors: Saraiva, B. M.; Cunha, I. M.; Brito, A. D.; Follain, G.; Portela, R.; Haase, R.; Pereira, P. M.; Jacquemet, G.; Henriques, R.
Score: 49.7, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553080
To overcome the challenges posed by large and complex microscopy datasets, we have developed NanoPyx, an adaptive bioimage analysis framework designed for high-speed processing. At the core of NanoPyx is the Liquid Engine, an agent-based machine-learning system that predicts acceleration strategies for image analysis tasks.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="NanoPyx: super-fast bioimage analysis powered by adaptive machine learning
Authors: Saraiva, B. M.; Cunha, I. M.; Brito, A. D.; Follain, G.; Portela, R.; Haase, R.; Pereira, P. M.; Jacquemet, G.; Henriques, R.
Score: 49.7, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553080
To overcome the challenges posed by large and complex microscopy datasets, we have developed NanoPyx, an adaptive bioimage analysis framework designed for high-speed processing. At the core of NanoPyx is the Liquid Engine, an agent-based machine-learning system that predicts acceleration strategies for image analysis tasks." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-16T10:37:20+00:00" />
<meta property="article:modified_time" content="2023-08-16T10:37:20+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="NanoPyx: super-fast bioimage analysis powered by adaptive machine learning
Authors: Saraiva, B. M.; Cunha, I. M.; Brito, A. D.; Follain, G.; Portela, R.; Haase, R.; Pereira, P. M.; Jacquemet, G.; Henriques, R.
Score: 49.7, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553080
To overcome the challenges posed by large and complex microscopy datasets, we have developed NanoPyx, an adaptive bioimage analysis framework designed for high-speed processing. At the core of NanoPyx is the Liquid Engine, an agent-based machine-learning system that predicts acceleration strategies for image analysis tasks."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "NanoPyx: super-fast bioimage analysis powered by adaptive machine learning\nAuthors: Saraiva, B. M.; Cunha, I. M.; Brito, A. D.; Follain, G.; Portela, R.; Haase, R.; Pereira, P. M.; Jacquemet, G.; Henriques, R.\nScore: 49.7, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553080\nTo overcome the challenges posed by large and complex microscopy datasets, we have developed NanoPyx, an adaptive bioimage analysis framework designed for high-speed processing. At the core of NanoPyx is the Liquid Engine, an agent-based machine-learning system that predicts acceleration strategies for image analysis tasks.",
  "keywords": [
    
  ],
  "articleBody": " NanoPyx: super-fast bioimage analysis powered by adaptive machine learning\nAuthors: Saraiva, B. M.; Cunha, I. M.; Brito, A. D.; Follain, G.; Portela, R.; Haase, R.; Pereira, P. M.; Jacquemet, G.; Henriques, R.\nScore: 49.7, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553080\nTo overcome the challenges posed by large and complex microscopy datasets, we have developed NanoPyx, an adaptive bioimage analysis framework designed for high-speed processing. At the core of NanoPyx is the Liquid Engine, an agent-based machine-learning system that predicts acceleration strategies for image analysis tasks. Unlike traditional single-algorithm methods, the Liquid Engine generates multiple CPU and GPU code variations using a meta-programming system, creating a competitive environment where different algorithms are benchmarked against each other to achieve optimal performance under the user's computational environment. In initial experiments focusing on super-resolution analysis methods, the Liquid Engine demonstrated an over 10-fold computational speed improvement by accurately predicting the ideal scenarios to switch between algorithmic implementations. NanoPyx is accessible to users through a Python library, code-free Jupyter notebooks, and a napari plugin, making it suitable for individuals regardless of their coding proficiency. Furthermore, the optimisation principles embodied by the Liquid Engine have broader implications, extending their applicability to various high-performance computing fields.\nMetaCerberus: distributed highly parallelized scalable HMM-based implementation for robust functional annotation across the tree of life\nAuthors: Figueroa, J. L.; Dhungel, E.; Brouwer, C. R.; White, R. A.\nScore: 47.4, Published: 2023-08-12 DOI: 10.1101/2023.08.10.552700\nSummaryMetaCerberus is an exclusive HMM/HMMER-based tool that is massively parallel, on low memory, and provides rapid scalable annotation for functional gene inference across genomes to metacommunities. It provides robust enumeration of functional genes and pathways across many current public databases including KEGG (KO), COGs, CAZy, FOAM, and viral specific databases (i.e., VOGs and PHROGs). In a direct comparison, MetaCerberus was twice as fast as EggNOG-Mapper, and produced better annotation of viruses, phages, and archaeal viruses than DRAM, PROKKA, or InterProScan. MetaCerberus annotates more KOs across domains when compared to DRAM, with a 186x smaller database and a third less memory. MetaCerberus is fully integrated with differential statistical tools (i.e., DESeq2 and edgeR), pathway enrichment (GAGE R), and Pathview R for quantitative elucidation of metabolic pathways. MetaCerberus implements the key to unlocking the biosphere across the tree of life at scale. Availability and implementationMetaCerberus is written in Python and distributed under a BSD-3 license. The source code of MetaCerberus is freely available at https://github.com/raw-lab/metacerberus. Written in python 3 for both Linux and Mac OS X. MetaCerberus can also be easily installed using mamba create -n metacerberus -c bioconda -c conda-forge metacerberus\nDiPPI: A curated dataset for drug-like molecules in protein-protein interfaces\nAuthors: Cankara, F.; Senyuz, S.; Sayin, A. Z.; Gursoy, A.; Keskin, O.\nScore: 25.6, Published: 2023-08-14 DOI: 10.1101/2023.08.09.552637\nProteins interact through their interfaces, and dysfunction of protein-protein interactions (PPIs) has been associated with various diseases. Therefore, investigating the properties of the drug-modulated PPIs and interface-targeting drugs is critical. Here, we present a curated large dataset for drug-like molecules in protein interfaces. We further present DiPPI (Drugs in Protein-Protein Interfaces), a two-module website to facilitate the search for such molecules and their properties by exploiting our dataset in drug repurposing studies. In the interface module of the website, we extracted several properties of interfaces, such as amino acid properties, hotspots, evolutionary conservation of drug-binding amino acids, and post-translational modifications of these residues. On the drug-like molecule side, we curated a list of drug-like small molecules and FDA-approved drugs from various databases and extracted those that bind to the interfaces. We further clustered the drugs based on their molecular fingerprints to confine the search for an alternative drug to a smaller space. Drug properties, including Lipinski rules and various molecular descriptors, are also calculated and made available on the website to guide the selection of drug molecules. Our dataset contains 534,203 interfaces for 98,632 proteins, of which 55,135 are detected to bind to a drug-like molecule. 2,214 drug-like molecules are deposited on our website, among which 335 are FDA-approved. DiPPI provides users with an easy-to-follow scheme for drug repurposing studies through its well-curated and clustered interface and drug data; and is freely available at http://interactome.ku.edu.tr:8501.\nAccurate human genome analysis with Element Avidity sequencing\nAuthors: Carroll, A.; Kolesnikov, A.; Cook, D. E.; Brambrink, L.; Wiseman, K. N.; Billings, S. M.; Kruglyak, S.; Lajoie, B. R.; Zhao, J.; Levy, S. E.; McLean, C. Y.; Shafin, K.; Nattestad, M.; Chang, P.-C.\nScore: 24.9, Published: 2023-08-14 DOI: 10.1101/2023.08.11.553043\nWe investigate the new sequencing technology Avidity from Element Biosciences. We show that Element whole genome sequencing achieves higher mapping and variant calling accuracy compared to Illumina sequencing at the same coverage, with larger differences at lower coverages (20x to 30x). We quantify base error rates of Element reads, finding lower error rates, especially in homopolymer and tandem repeat regions. We use the Element ability to generate paired end sequencing with longer insert sizes than typical short read sequencing. We show that longer insert sizes result in even higher accuracy, with long insert Element sequencing giving noticeably more accurate genome analyses at all coverages.\nComprehensive Structural Variant Detection: From Mosaic to Population-Level\nAuthors: Smolka, M.; Paulin, L. F.; Grochowski, C. M.; Horner, D. W.; Mahmoud, M.; Behera, S.; Kalef-Ezra, E.; Gandhi, M.; Hong, K.; Pehlivan, D.; Scholz, S. W.; Carvalho, C. M.; Proukakis, C.; Sedlazeck, F. J.\nScore: 73.8, Published: 2023-08-09 DOI: 10.1101/2022.04.04.487055\nLong-read Structural Variation (SV) calling remains a challenging but highly accurate way to identify complex genomic alterations. Here, we present Sniffles2, which is faster and more accurate than state-of-the-art SV caller across different coverages, sequencing technologies, and SV types. Furthermore, Sniffles2 solves the problem of family- to population-level SV calling to produce fully genotyped VCF files by introducing a gVCF file concept. Across 11 probands, we accurately identified causative SVs around MECP2, including highly complex alleles with three overlapping SVs. Sniffles2 also enables the detection of mosaic SVs in bulk long-read data. As a result, we successfully identified multiple mosaic SVs across a multiple system atrophy patient brain. The identified SV showed a remarkable diversity within the cingulate cortex, impacting both genes involved in neuron function and repetitive elements. In summary, we demonstrate the utility and versatility of Sniffles2 to identify SVs from the mosaic to population levels.\nBenchmarking computational methods for single-cell chromatin data analysis\nAuthors: Luo, S.; Germain, P.-L.; Robinson, M. D.; von Meyenn, F.\nScore: 22.2, Published: 2023-08-07 DOI: 10.1101/2023.08.04.552046\nSingle-cell chromatin accessibility assays, such as scATAC-seq, are increasingly employed in individual and joint multi-omic profiling of single cells. As the accumulation of scATAC-seq and multi-omics datasets continue, challenges in analyzing such sparse, noisy, and high-dimensional data become pressing. Specifically, one challenge relates to optimizing the processing of chromatin-level measurements and efficiently extracting information to discern cellular heterogeneity. This is of critical importance, since the identification of cell types is a fundamental step in current single-cell data analysis practices. We benchmarked 8 feature engineering pipelines derived from 5 recent methods to assess their ability to discover and discriminate cell types. By using 10 metrics calculated at the cell embedding, shared nearest neighbor graph, or partition levels, we evaluated the performance of each method at different data processing stages. This comprehensive approach allowed us to thoroughly understand the strengths and weaknesses of each method and the influence of parameter selection. Our analysis provides guidelines for choosing analysis methods for different datasets. Overall, feature aggregation, SnapATAC, and SnapATAC2 outperform latent semantic indexing-based methods. For datasets with complex cell-type structures, SnapATAC and SnapATAC2 are preferred. With large datasets, SnapATAC2 and ArchR are most scalable.\nEagleC Explorer: A desktop application for interactively detecting and visualizing SVs and enhancer hijacking on Hi-C contact maps\nAuthors: Fu, Y.; Wang, X.; Yue, F.\nScore: 19.6, Published: 2023-08-08 DOI: 10.1101/2023.08.07.552228\nIt has been shown that Hi-C can be used as a powerful tool to detect structural variations (SVs) and enhancer hijacking events. However, there has been no existing programs that can directly visualize and detect such events on a personal computer, which hinders the broad adaption of the technology for intuitive discovery in cancer studies. Here, we introduce the EagleC Explorer, a desktop software that is specifically designed for exploring Hi-C and other chromatin contact data in cancer genomes. EagleC Explorer has a set of unique features, including 1) conveniently visualizing global and local Hi-C data; 2) interactively detecting SVs on a Hi-C map for any user-selected region on screen within seconds, using a deep-learning model; 3) reconstructing local Hi-C map surrounding user-provided SVs and generating publication-quality figures; 4) detecting enhancer hijacking events for any user-suggested regions on screen. In addition, EagleC Explorer can also incorporate other genomic tracks such as RNA-Seq or ChIP-Seq to facilitate scientists for integrative data analysis and making novel discoveries.\nPDBe CCDUtils: an RDKit-based toolkit for handling and analysing small molecules in the Protein Data Bank\nAuthors: Kunnakkattu, I. R.; Choudhary, P.; Pravda, L.; Nadzirin, N.; Smart, O. S.; Yuan, Q.; Anyango, S.; Nair, S.; Varadi, M.; Velankar, S.\nScore: 18.4, Published: 2023-08-07 DOI: 10.1101/2023.08.04.552003\nWhile the Protein Data Bank (PDB) contains a wealth of structural information on ligands bound to macromolecules, their analysis can be challenging due to the large amount and diversity of data. Here, we present PDBe CCDUtils, a versatile toolkit for processing and analysing small molecules from the PDB in PDBx/mmCIF format. PDBe CCDUtils provides streamlined access to all the metadata for small molecules in the PDB and offers a set of convenient methods to compute various properties using RDKit, such as 2D depictions, 3D conformers, physicochemical properties, scaffolds, common fragments and cross-references to small molecule databases using UniChem. The toolkit also provides methods for identifying all the covalently attached chemical components in a macromolecular structure and calculating similarity among small molecules. By providing a broad range of functionality, PDBe CCDUtils caters to the needs of researchers in cheminformatics, structural biology, bioinformatics and computational chemistry.\nOpenFold: Retraining AlphaFold2 yields new insights into its learning mechanisms and capacity for generalization\nAuthors: Ahdritz, G.; Bouatta, N.; Floristean, C.; Kadyan, S.; Xia, Q.; Gerecke, W.; O'Donnell, T. J.; Berenberg, D.; Fisk, I.; Zanichelli, N.; Zhang, B.; Nowaczynski, A.; Wang, B.; Stepniewska-Dziubinska, M. M.; Zhang, S.; Ojewole, A.; Guney, M. E.; Biderman, S.; Watkins, A. M.; Ra, S.; Lorenzo, P. R.; Nivon, L.; Weitzner, B.; Ban, Y.-E. A.; Sorger, P. K.; Mostaque, E.; Zhang, Z.; Bonneau, R.; AlQuraishi, M.\nScore: 580.7, Published: 2023-08-12 DOI: 10.1101/2022.11.20.517210\nAlphaFold2 revolutionized structural biology with the ability to predict protein structures with exceptionally high accuracy. Its implementation, however, lacks the code and data required to train new models. These are necessary to (i) tackle new tasks, like protein-ligand complex structure prediction, (ii) investigate the process by which the model learns, which remains poorly understood, and (iii) assess the model's generalization capacity to unseen regions of fold space. Here we report OpenFold, a fast, memory-efficient, and trainable implementation of AlphaFold2. We train OpenFold from scratch, fully matching the accuracy of AlphaFold2. Having established parity, we assess OpenFold's capacity to generalize across fold space by retraining it using carefully designed datasets. We find that OpenFold is remarkably robust at generalizing despite extreme reductions in training set size and diversity, including near-complete elisions of classes of secondary structure elements. By analyzing intermediate structures produced by OpenFold during training, we also gain surprising insights into the manner in which the model learns to fold proteins, discovering that spatial dimensions are learned sequentially. Taken together, our studies demonstrate the power and utility of OpenFold, which we believe will prove to be a crucial new resource for the protein modeling community.\nIdentifying small-molecules binding sites in RNA conformational ensembles with SHAMAN\nAuthors: Panei, F. P.; Gkeka, P.; Bonomi, M.\nScore: 16.0, Published: 2023-08-09 DOI: 10.1101/2023.08.08.552403\nThe rational targeting of RNA with small molecules is hampered by our still limited understanding of RNA structural and dynamic properties. Most in silico tools for binding site identification rely on static structures and therefore cannot face the challenges posed by the dynamic nature of RNA molecules. Here we present SHAMAN, a computational technique to identify potential small-molecule binding sites in RNA structural ensembles. SHAMAN enables exploring the conformational landscape of RNA with atomistic molecular dynamics and at the same time identifying RNA pockets in an efficient way with the aid of probes and enhanced-sampling techniques. In our benchmark composed of large, structured riboswitches as well as small, flexible viral RNAs, SHAMAN successfully identified all the experimentally resolved pockets and ranked them among the most favorite probe hotspots. Overall, SHAMAN sets a solid foundation for future drug design efforts targeting RNA with small molecules, effectively addressing the long-standing challenges in the field.\n",
  "wordCount" : "2092",
  "inLanguage": "en",
  "datePublished": "2023-08-16T10:37:20Z",
  "dateModified": "2023-08-16T10:37:20Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on August 16, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.13.553080">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.13.553080" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.13.553080">
        <p class="paperTitle">NanoPyx: super-fast bioimage analysis powered by adaptive machine learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.13.553080" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.13.553080" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Saraiva, B. M.; Cunha, I. M.; Brito, A. D.; Follain, G.; Portela, R.; Haase, R.; Pereira, P. M.; Jacquemet, G.; Henriques, R.</p>
        <p class="info">Score: 49.7, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.13.553080' target='https://doi.org/10.1101/2023.08.13.553080'> 10.1101/2023.08.13.553080</a></p>
        <p class="abstract">To overcome the challenges posed by large and complex microscopy datasets, we have developed NanoPyx, an adaptive bioimage analysis framework designed for high-speed processing. At the core of NanoPyx is the Liquid Engine, an agent-based machine-learning system that predicts acceleration strategies for image analysis tasks. Unlike traditional single-algorithm methods, the Liquid Engine generates multiple CPU and GPU code variations using a meta-programming system, creating a competitive environment where different algorithms are benchmarked against each other to achieve optimal performance under the user&#39;s computational environment. In initial experiments focusing on super-resolution analysis methods, the Liquid Engine demonstrated an over 10-fold computational speed improvement by accurately predicting the ideal scenarios to switch between algorithmic implementations. NanoPyx is accessible to users through a Python library, code-free Jupyter notebooks, and a napari plugin, making it suitable for individuals regardless of their coding proficiency. Furthermore, the optimisation principles embodied by the Liquid Engine have broader implications, extending their applicability to various high-performance computing fields.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552700">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552700" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552700">
        <p class="paperTitle">MetaCerberus: distributed highly parallelized scalable HMM-based implementation for robust functional annotation across the tree of life</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552700" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552700" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Figueroa, J. L.; Dhungel, E.; Brouwer, C. R.; White, R. A.</p>
        <p class="info">Score: 47.4, Published: 2023-08-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552700' target='https://doi.org/10.1101/2023.08.10.552700'> 10.1101/2023.08.10.552700</a></p>
        <p class="abstract">SummaryMetaCerberus is an exclusive HMM/HMMER-based tool that is massively parallel, on low memory, and provides rapid scalable annotation for functional gene inference across genomes to metacommunities. It provides robust enumeration of functional genes and pathways across many current public databases including KEGG (KO), COGs, CAZy, FOAM, and viral specific databases (i.e., VOGs and PHROGs). In a direct comparison, MetaCerberus was twice as fast as EggNOG-Mapper, and produced better annotation of viruses, phages, and archaeal viruses than DRAM, PROKKA, or InterProScan. MetaCerberus annotates more KOs across domains when compared to DRAM, with a 186x smaller database and a third less memory. MetaCerberus is fully integrated with differential statistical tools (i.e., DESeq2 and edgeR), pathway enrichment (GAGE R), and Pathview R for quantitative elucidation of metabolic pathways. MetaCerberus implements the key to unlocking the biosphere across the tree of life at scale.

Availability and implementationMetaCerberus is written in Python and distributed under a BSD-3 license. The source code of MetaCerberus is freely available at https://github.com/raw-lab/metacerberus. Written in python 3 for both Linux and Mac OS X. MetaCerberus can also be easily installed using mamba create -n metacerberus -c bioconda -c conda-forge metacerberus</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.552637">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.552637" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.552637">
        <p class="paperTitle">DiPPI: A curated dataset for drug-like molecules in protein-protein interfaces</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.552637" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.552637" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cankara, F.; Senyuz, S.; Sayin, A. Z.; Gursoy, A.; Keskin, O.</p>
        <p class="info">Score: 25.6, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.552637' target='https://doi.org/10.1101/2023.08.09.552637'> 10.1101/2023.08.09.552637</a></p>
        <p class="abstract">Proteins interact through their interfaces, and dysfunction of protein-protein interactions (PPIs) has been associated with various diseases. Therefore, investigating the properties of the drug-modulated PPIs and interface-targeting drugs is critical. Here, we present a curated large dataset for drug-like molecules in protein interfaces. We further present DiPPI (Drugs in Protein-Protein Interfaces), a two-module website to facilitate the search for such molecules and their properties by exploiting our dataset in drug repurposing studies. In the interface module of the website, we extracted several properties of interfaces, such as amino acid properties, hotspots, evolutionary conservation of drug-binding amino acids, and post-translational modifications of these residues. On the drug-like molecule side, we curated a list of drug-like small molecules and FDA-approved drugs from various databases and extracted those that bind to the interfaces. We further clustered the drugs based on their molecular fingerprints to confine the search for an alternative drug to a smaller space. Drug properties, including Lipinski rules and various molecular descriptors, are also calculated and made available on the website to guide the selection of drug molecules. Our dataset contains 534,203 interfaces for 98,632 proteins, of which 55,135 are detected to bind to a drug-like molecule. 2,214 drug-like molecules are deposited on our website, among which 335 are FDA-approved. DiPPI provides users with an easy-to-follow scheme for drug repurposing studies through its well-curated and clustered interface and drug data; and is freely available at http://interactome.ku.edu.tr:8501.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.553043">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.553043" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.553043">
        <p class="paperTitle">Accurate human genome analysis with Element Avidity sequencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.553043" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.553043" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carroll, A.; Kolesnikov, A.; Cook, D. E.; Brambrink, L.; Wiseman, K. N.; Billings, S. M.; Kruglyak, S.; Lajoie, B. R.; Zhao, J.; Levy, S. E.; McLean, C. Y.; Shafin, K.; Nattestad, M.; Chang, P.-C.</p>
        <p class="info">Score: 24.9, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.553043' target='https://doi.org/10.1101/2023.08.11.553043'> 10.1101/2023.08.11.553043</a></p>
        <p class="abstract">We investigate the new sequencing technology Avidity from Element Biosciences. We show that Element whole genome sequencing achieves higher mapping and variant calling accuracy compared to Illumina sequencing at the same coverage, with larger differences at lower coverages (20x to 30x). We quantify base error rates of Element reads, finding lower error rates, especially in homopolymer and tandem repeat regions. We use the Element ability to generate paired end sequencing with longer insert sizes than typical short read sequencing. We show that longer insert sizes result in even higher accuracy, with long insert Element sequencing giving noticeably more accurate genome analyses at all coverages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.04.04.487055">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.04.04.487055" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.04.04.487055">
        <p class="paperTitle">Comprehensive Structural Variant Detection: From Mosaic to Population-Level</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.04.04.487055" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.04.04.487055" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smolka, M.; Paulin, L. F.; Grochowski, C. M.; Horner, D. W.; Mahmoud, M.; Behera, S.; Kalef-Ezra, E.; Gandhi, M.; Hong, K.; Pehlivan, D.; Scholz, S. W.; Carvalho, C. M.; Proukakis, C.; Sedlazeck, F. J.</p>
        <p class="info">Score: 73.8, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.04.04.487055' target='https://doi.org/10.1101/2022.04.04.487055'> 10.1101/2022.04.04.487055</a></p>
        <p class="abstract">Long-read Structural Variation (SV) calling remains a challenging but highly accurate way to identify complex genomic alterations. Here, we present Sniffles2, which is faster and more accurate than state-of-the-art SV caller across different coverages, sequencing technologies, and SV types. Furthermore, Sniffles2 solves the problem of family- to population-level SV calling to produce fully genotyped VCF files by introducing a gVCF file concept. Across 11 probands, we accurately identified causative SVs around MECP2, including highly complex alleles with three overlapping SVs. Sniffles2 also enables the detection of mosaic SVs in bulk long-read data. As a result, we successfully identified multiple mosaic SVs across a multiple system atrophy patient brain. The identified SV showed a remarkable diversity within the cingulate cortex, impacting both genes involved in neuron function and repetitive elements. In summary, we demonstrate the utility and versatility of Sniffles2 to identify SVs from the mosaic to population levels.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.552046">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.552046" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.552046">
        <p class="paperTitle">Benchmarking computational methods for single-cell chromatin data analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.552046" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.552046" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Luo, S.; Germain, P.-L.; Robinson, M. D.; von Meyenn, F.</p>
        <p class="info">Score: 22.2, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.552046' target='https://doi.org/10.1101/2023.08.04.552046'> 10.1101/2023.08.04.552046</a></p>
        <p class="abstract">Single-cell chromatin accessibility assays, such as scATAC-seq, are increasingly employed in individual and joint multi-omic profiling of single cells. As the accumulation of scATAC-seq and multi-omics datasets continue, challenges in analyzing such sparse, noisy, and high-dimensional data become pressing. Specifically, one challenge relates to optimizing the processing of chromatin-level measurements and efficiently extracting information to discern cellular heterogeneity. This is of critical importance, since the identification of cell types is a fundamental step in current single-cell data analysis practices.

We benchmarked 8 feature engineering pipelines derived from 5 recent methods to assess their ability to discover and discriminate cell types. By using 10 metrics calculated at the cell embedding, shared nearest neighbor graph, or partition levels, we evaluated the performance of each method at different data processing stages. This comprehensive approach allowed us to thoroughly understand the strengths and weaknesses of each method and the influence of parameter selection.

Our analysis provides guidelines for choosing analysis methods for different datasets. Overall, feature aggregation, SnapATAC, and SnapATAC2 outperform latent semantic indexing-based methods. For datasets with complex cell-type structures, SnapATAC and SnapATAC2 are preferred. With large datasets, SnapATAC2 and ArchR are most scalable.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552228">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552228" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552228">
        <p class="paperTitle">EagleC Explorer: A desktop application for interactively detecting and visualizing SVs and enhancer hijacking on Hi-C contact maps</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552228" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552228" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fu, Y.; Wang, X.; Yue, F.</p>
        <p class="info">Score: 19.6, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552228' target='https://doi.org/10.1101/2023.08.07.552228'> 10.1101/2023.08.07.552228</a></p>
        <p class="abstract">It has been shown that Hi-C can be used as a powerful tool to detect structural variations (SVs) and enhancer hijacking events. However, there has been no existing programs that can directly visualize and detect such events on a personal computer, which hinders the broad adaption of the technology for intuitive discovery in cancer studies. Here, we introduce the EagleC Explorer, a desktop software that is specifically designed for exploring Hi-C and other chromatin contact data in cancer genomes. EagleC Explorer has a set of unique features, including 1) conveniently visualizing global and local Hi-C data; 2) interactively detecting SVs on a Hi-C map for any user-selected region on screen within seconds, using a deep-learning model; 3) reconstructing local Hi-C map surrounding user-provided SVs and generating publication-quality figures; 4) detecting enhancer hijacking events for any user-suggested regions on screen. In addition, EagleC Explorer can also incorporate other genomic tracks such as RNA-Seq or ChIP-Seq to facilitate scientists for integrative data analysis and making novel discoveries.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.552003">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.552003" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.552003">
        <p class="paperTitle">PDBe CCDUtils: an RDKit-based toolkit for handling and analysing small molecules in the Protein Data Bank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.552003" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.552003" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kunnakkattu, I. R.; Choudhary, P.; Pravda, L.; Nadzirin, N.; Smart, O. S.; Yuan, Q.; Anyango, S.; Nair, S.; Varadi, M.; Velankar, S.</p>
        <p class="info">Score: 18.4, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.552003' target='https://doi.org/10.1101/2023.08.04.552003'> 10.1101/2023.08.04.552003</a></p>
        <p class="abstract">While the Protein Data Bank (PDB) contains a wealth of structural information on ligands bound to macromolecules, their analysis can be challenging due to the large amount and diversity of data. Here, we present PDBe CCDUtils, a versatile toolkit for processing and analysing small molecules from the PDB in PDBx/mmCIF format.

PDBe CCDUtils provides streamlined access to all the metadata for small molecules in the PDB and offers a set of convenient methods to compute various properties using RDKit, such as 2D depictions, 3D conformers, physicochemical properties, scaffolds, common fragments and cross-references to small molecule databases using UniChem. The toolkit also provides methods for identifying all the covalently attached chemical components in a macromolecular structure and calculating similarity among small molecules. By providing a broad range of functionality, PDBe CCDUtils caters to the needs of researchers in cheminformatics, structural biology, bioinformatics and computational chemistry.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.11.20.517210">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.11.20.517210" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.11.20.517210">
        <p class="paperTitle">OpenFold: Retraining AlphaFold2 yields new insights into its learning mechanisms and capacity for generalization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.11.20.517210" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.11.20.517210" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ahdritz, G.; Bouatta, N.; Floristean, C.; Kadyan, S.; Xia, Q.; Gerecke, W.; O&#39;Donnell, T. J.; Berenberg, D.; Fisk, I.; Zanichelli, N.; Zhang, B.; Nowaczynski, A.; Wang, B.; Stepniewska-Dziubinska, M. M.; Zhang, S.; Ojewole, A.; Guney, M. E.; Biderman, S.; Watkins, A. M.; Ra, S.; Lorenzo, P. R.; Nivon, L.; Weitzner, B.; Ban, Y.-E. A.; Sorger, P. K.; Mostaque, E.; Zhang, Z.; Bonneau, R.; AlQuraishi, M.</p>
        <p class="info">Score: 580.7, Published: 2023-08-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.11.20.517210' target='https://doi.org/10.1101/2022.11.20.517210'> 10.1101/2022.11.20.517210</a></p>
        <p class="abstract">AlphaFold2 revolutionized structural biology with the ability to predict protein structures with exceptionally high accuracy. Its implementation, however, lacks the code and data required to train new models. These are necessary to (i) tackle new tasks, like protein-ligand complex structure prediction, (ii) investigate the process by which the model learns, which remains poorly understood, and (iii) assess the model&#39;s generalization capacity to unseen regions of fold space. Here we report OpenFold, a fast, memory-efficient, and trainable implementation of AlphaFold2. We train OpenFold from scratch, fully matching the accuracy of AlphaFold2. Having established parity, we assess OpenFold&#39;s capacity to generalize across fold space by retraining it using carefully designed datasets. We find that OpenFold is remarkably robust at generalizing despite extreme reductions in training set size and diversity, including near-complete elisions of classes of secondary structure elements. By analyzing intermediate structures produced by OpenFold during training, we also gain surprising insights into the manner in which the model learns to fold proteins, discovering that spatial dimensions are learned sequentially. Taken together, our studies demonstrate the power and utility of OpenFold, which we believe will prove to be a crucial new resource for the protein modeling community.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552403">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552403" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552403">
        <p class="paperTitle">Identifying small-molecules binding sites in RNA conformational ensembles with SHAMAN</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552403" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552403" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Panei, F. P.; Gkeka, P.; Bonomi, M.</p>
        <p class="info">Score: 16.0, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552403' target='https://doi.org/10.1101/2023.08.08.552403'> 10.1101/2023.08.08.552403</a></p>
        <p class="abstract">The rational targeting of RNA with small molecules is hampered by our still limited understanding of RNA structural and dynamic properties. Most in silico tools for binding site identification rely on static structures and therefore cannot face the challenges posed by the dynamic nature of RNA molecules. Here we present SHAMAN, a computational technique to identify potential small-molecule binding sites in RNA structural ensembles. SHAMAN enables exploring the conformational landscape of RNA with atomistic molecular dynamics and at the same time identifying RNA pockets in an efficient way with the aid of probes and enhanced-sampling techniques. In our benchmark composed of large, structured riboswitches as well as small, flexible viral RNAs, SHAMAN successfully identified all the experimentally resolved pockets and ranked them among the most favorite probe hotspots. Overall, SHAMAN sets a solid foundation for future drug design efforts targeting RNA with small molecules, effectively addressing the long-standing challenges in the field.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
