<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="The tidyomics ecosystem: Enhancing omic data analyses
Authors: Hutchison, W. J.; Keyes, T. J.; Crowell, H. L.; Soneson, C.; Mu, W.; Park, J.-E.; Davis, E. S.; Nahid, A. A.; Tang, M.; Yuan, V.; Axisa, P.-P.; Kitt, J. W.; Poon, C.-L.; Sato, N.; Gottardo, R.; Morgan, M.; Lee, S.; Lawrence, M.; Hicks, S. C.; Nolan, G. P.; Davis, K. L.; Papenfuss, A. T.; Love, M. I.; Mangiola, S.
Score: 118.1, Published: 2023-09-13 DOI: 10.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="The tidyomics ecosystem: Enhancing omic data analyses
Authors: Hutchison, W. J.; Keyes, T. J.; Crowell, H. L.; Soneson, C.; Mu, W.; Park, J.-E.; Davis, E. S.; Nahid, A. A.; Tang, M.; Yuan, V.; Axisa, P.-P.; Kitt, J. W.; Poon, C.-L.; Sato, N.; Gottardo, R.; Morgan, M.; Lee, S.; Lawrence, M.; Hicks, S. C.; Nolan, G. P.; Davis, K. L.; Papenfuss, A. T.; Love, M. I.; Mangiola, S.
Score: 118.1, Published: 2023-09-13 DOI: 10." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-17T10:36:50+00:00" />
<meta property="article:modified_time" content="2023-09-17T10:36:50+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="The tidyomics ecosystem: Enhancing omic data analyses
Authors: Hutchison, W. J.; Keyes, T. J.; Crowell, H. L.; Soneson, C.; Mu, W.; Park, J.-E.; Davis, E. S.; Nahid, A. A.; Tang, M.; Yuan, V.; Axisa, P.-P.; Kitt, J. W.; Poon, C.-L.; Sato, N.; Gottardo, R.; Morgan, M.; Lee, S.; Lawrence, M.; Hicks, S. C.; Nolan, G. P.; Davis, K. L.; Papenfuss, A. T.; Love, M. I.; Mangiola, S.
Score: 118.1, Published: 2023-09-13 DOI: 10."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "The tidyomics ecosystem: Enhancing omic data analyses\nAuthors: Hutchison, W. J.; Keyes, T. J.; Crowell, H. L.; Soneson, C.; Mu, W.; Park, J.-E.; Davis, E. S.; Nahid, A. A.; Tang, M.; Yuan, V.; Axisa, P.-P.; Kitt, J. W.; Poon, C.-L.; Sato, N.; Gottardo, R.; Morgan, M.; Lee, S.; Lawrence, M.; Hicks, S. C.; Nolan, G. P.; Davis, K. L.; Papenfuss, A. T.; Love, M. I.; Mangiola, S.\nScore: 118.1, Published: 2023-09-13 DOI: 10.",
  "keywords": [
    
  ],
  "articleBody": " The tidyomics ecosystem: Enhancing omic data analyses\nAuthors: Hutchison, W. J.; Keyes, T. J.; Crowell, H. L.; Soneson, C.; Mu, W.; Park, J.-E.; Davis, E. S.; Nahid, A. A.; Tang, M.; Yuan, V.; Axisa, P.-P.; Kitt, J. W.; Poon, C.-L.; Sato, N.; Gottardo, R.; Morgan, M.; Lee, S.; Lawrence, M.; Hicks, S. C.; Nolan, G. P.; Davis, K. L.; Papenfuss, A. T.; Love, M. I.; Mangiola, S.\nScore: 118.1, Published: 2023-09-13 DOI: 10.1101/2023.09.10.557072\nThe exponential growth of omic data presents challenges in data manipulation, analysis, and integration. Addressing these challenges, Bioconductor offers an extensive data analysis platform and community, while R tidy programming offers a standard data organisation and manipulation that has revolutionised data science. Bioconductor and tidy R have mostly remained independent; bridging these two ecosystems would streamline omic analysis, ease learning and encourage cross-disciplinary collaborations. Here, we introduce the tidyomics ecosystem--a suite of interoperable software that brings the vast tidy software ecosystem to omic data analysis.\nCell2Sentence: Teaching Large Language Models the Language of Biology\nAuthors: Levine, D.; Rizvi, S. A.; Levy, S.; Pallikkavaliyaveetil MohammedSheriff, N.; Wu, R.; Zhang, Z.; Fonseca, A.; Chen, X.; Ghadermarzi, S.; Dhodapkar, R. M.; van Dijk, D.\nScore: 71.2, Published: 2023-09-14 DOI: 10.1101/2023.09.11.557287\nLarge language models like GPT have shown impressive performance on natural language tasks. Here, we present a novel method to directly adapt these pretrained models to a biological context, specifically single-cell transcriptomics, by representing gene expression data as text. Our Cell2Sentence approach converts each cell's gene expression profile into a sequence of gene names ordered by expression level. We show that these gene sequences, which we term \"cell sentences\", can be used to fine-tune causal language models like GPT-2. Critically, we find that natural language pretraining boosts model performance on cell sentence tasks. When fine-tuned on cell sentences, GPT-2 generates biologically valid cells when prompted with a cell type. Conversely, it can also accurately predict cell type labels when prompted with cell sentences. This demonstrates that language models fine-tuned using Cell2Sentence can gain a biological understanding of single-cell data, while retaining their ability to generate text. Our approach provides a simple, adaptable framework to combine natural language and transcriptomics using existing models and libraries. Our code is available at: https://github.com/vandijklab/cell2sentence-ft.\nCodonBERT: Large Language Models for mRNA Design and Optimization\nAuthors: Li, S.; Moayedpour, S.; Li, R.; Bailey, M.; Riahi, S.; Miladi, M.; Miner, J.; Zheng, D.; Wang, J.; Balsubramani, A.; Tran, K.; Zacharia, M.; Wu, M.; Gu, X.; Clinton, R.; Asquith, C.; Skalesk, J.; Boeglin, L.; Chivukula, S.; Dias, A.; Montoya, F. U.; Agarwal, V.; Bar-Joseph, Z.; Dr. Jager, S.\nScore: 61.0, Published: 2023-09-12 DOI: 10.1101/2023.09.09.556981\nAO_SCPLOWBSTRACTC_SCPLOWmRNA based vaccines and therapeutics are gaining popularity and usage across a wide range of conditions. One of the critical issues when designing such mRNAs is sequence optimization. Even small proteins or peptides can be encoded by an enormously large number of mRNAs. The actual mRNA sequence can have a large impact on several properties including expression, stability, immunogenicity, and more. To enable the selection of an optimal sequence, we developed CodonBERT, a large language model (LLM) for mRNAs. Unlike prior models, CodonBERT uses codons as inputs which enables it to learn better representations. CodonBERT was trained using more than 10 million mRNA sequences from a diverse set of organisms. The resulting model captures important biological concepts. CodonBERT can also be extended to perform prediction tasks for various mRNA properties. CodonBERT outperforms previous mRNA prediction methods including on a new flu vaccine dataset.\nEncoding the Space of Protein-protein Binding Interfaces by Artificial Intelligence\nAuthors: Su, Z.; Dhusia, K.; Wu, Y.\nScore: 27.1, Published: 2023-09-10 DOI: 10.1101/2023.09.08.556812\nThe physical interactions between proteins are largely determined by the structural properties at their binding interfaces. It was found that the binding interfaces in distinctive protein complexes are highly similar. The structural properties underlying different binding interfaces could be further captured by artificial intelligence. In order to test this hypothesis, we broke protein-protein binding interfaces into pairs of interacting fragments. We employed a generative model to encode these interface fragment pairs in a low-dimensional latent space. After training, new conformations of interface fragment pairs were generated. We found that, by only using a small number of interface fragment pairs that were generated by artificial intelligence, we were able to guide the assembly of protein complexes into their native conformations. These results demonstrate that the conformational space of fragment pairs at protein-protein binding interfaces is highly degenerate. Our study illustrates how artificial intelligence can be used to understand and characterize protein-protein binding interfaces. The method will be potentially useful to search for the conformation of unknown protein-protein interactions. This result demonstrated that the structural space of protein-protein interactions is highly degenerate under the representation of interface fragment pairs. Features in this degenerate space can be well characterized by artificial intelligence. In summary, our machine learning method will be potentially useful to search for and predict the conformations of unknown protein-protein interactions.\nGraffiTE: a Unified Framework to Analyze Transposable Element Insertion Polymorphisms using Genome-graphs\nAuthors: Groza, C.; Chen, X.; Wheeler, T. J.; Bourque, G.; Goubert, C.\nScore: 26.9, Published: 2023-09-15 DOI: 10.1101/2023.09.11.557209\nTransposable Elements (TEs) are abundant and mobile repetitive DNA sequences evolving within and across their hosts' genomes. Active TEs cause insertion polymorphism and contribute to genomic diversity. Here, we present GraffiTE, a flexible and comprehensive pipeline for detecting and genotyping polymorphic mobile elements (pMEs). By integrating state-of-the-art SV detection algorithms and graph-genome frameworks, GraffiTE enables the accurate identification of pMEs from genomic assemblies and long-read as well as the precise genotyping of these variants using short- or long-read data. Performance evaluations using simulated and benchmark datasets demonstrate high precision and recall rates. Notably, we demonstrate the versatility of GraffiTE by analyzing the human reference pangenome, 30 Drosophila melanogaster genomes, and multiple cultivars of the emerging crop model Cannabis sativa, where pMEs are undocumented. These analyses reveal the landscapes of pMEs and their frequency variations across individuals, strains, and cultivars. GraffiTE provides a user-friendly interface, allowing non-expert users to perform comprehensive pME analyses, including in models with limited TE prior knowledge. The pipeline's extensible design and compatibility with various sequencing technologies make it a valuable integrative framework for studying TE dynamics and their impact on genome evolution. GraffiTE is freely available at https://github.com/cgroza/GraffiTE.\nBayesian phylogenetics on globally emerging SARS-CoV-2 variant BA.2.86 suggest global distribution and rapid evolution\nAuthors: Rothstein, A. P.; Qiu, X.; Robison, K.; Collins, S.; Muir, G.; Lu, B.; Plocik, A.; Simen, B.; Philipson, C.\nScore: 26.5, Published: 2023-09-11 DOI: 10.1101/2023.09.08.556912\nUsing bioinformatic pipelines and Bayseian phylogenetic analyses, we characterized a SARS-CoV-2 variant designated by the World Health Organization as a variant under monitoring in August 2023. Here we analyze the genomes of this SARS-CoV-2 variant, BA.2.86, deposited into GISAID within the two weeks of its emergence (2023-08-14 first submission to 2023-08-31), including the first BA.2.86 genome reported from a traveler originating from Japan. We present bioinformatics methods using publicly available tools to help analysts identify the lineage-defining 12 nucleotide insertion (S:Ins16MPLF), which is often masked by most bioinformatics pipelines. We also applied maximum-likelihood and Bayesian phylogenetics to demonstrate the high mutational rate of the tree branch leading to the emergence of BA.2.86, hinting at possible origins, and predict that BA.2.86 emerged around May 2023 and spread globally rapidly. Taken together, these results provide a framework for more rigorous bioinformatics approaches for teams performing genomic surveillance on viral respiratory pathogens.\nLocal read haplotagging enables accurate long-read small variant calling\nAuthors: Kolesnikov, A.; Cook, D. E.; Nattestad, M.; Ashley, E. A.; Gorzynski, J.; Goenka, S. D.; Jain, M.; McNulty, B.; Miga, K. H.; Paten, B.; Chang, P.-C.; Carroll, A.; Shafin, K.\nScore: 19.4, Published: 2023-09-12 DOI: 10.1101/2023.09.07.556731\nLong-read sequencing technology has enabled variant detection in difficult-to-map regions of the genome and enabled rapid genetic diagnosis in clinical settings. Rapidly evolving third-generation sequencing platforms like Pacific Biosciences (PacBio) and Oxford nanopore technologies (ONT) are introducing newer platforms and data types. It has been demonstrated that variant calling methods based on deep neural networks can use local haplotyping information with long-reads to improve the genotyping accuracy. However, using local haplotype information creates an overhead as variant calling needs to be performed multiple times which ultimately makes it difficult to extend to new data types and platforms as they get introduced. In this work, we have developed a local haplotype approximate method that enables state-of-the-art variant calling performance with multiple sequencing platforms including PacBio Revio system, ONT R10.4 simplex and duplex data. This addition of local haplotype approximation makes DeepVariant a universal variant calling solution for long-read sequencing platforms.\nEvaluating the Utilities of Large Language Models in Single-cell Data Analysis\nAuthors: Liu, T.; Li, K.; Wang, Y.; Li, H.; Zhao, H.\nScore: 20.3, Published: 2023-09-10 DOI: 10.1101/2023.09.08.555192\nLarge Language Models (LLMs) or foundation models have made significant strides in both industrial and scientific domains. In this paper, we evaluate the performance of LLMs in single-cell sequencing data analysis through comprehensive experiments across eight downstream tasks pertinent to single-cell data. By comparing seven different single-cell LLMs with task-specific methods, we found that single-cell LLMs may not consistently excel in all tasks than task-specific methods. However, the emergent abilities and the successful applications of cross-species/cross-modality transfer learning of LLMs are promising. In addition, we present a systematic evaluation of the effects of hyper-parameters, initial settings, and stability for training single-cell LLMs based on a proposed scEval framework, and provide guidelines for pre-training and fine-tuning. Our work summarizes the current state of single-cell LLMs, and points to their constraints and avenues for future investigation.\nAutomated Bioinformatics Analysis via AutoBA\nAuthors: Zhou, J.; Zhang, B.; Chen, X.; Li, H.; Xu, X.; Chen, S.; Gao, X.\nScore: 12.9, Published: 2023-09-12 DOI: 10.1101/2023.09.08.556814\nWith the fast-growing and evolving omics data, the demand for streamlined and adaptable tools to handle the analysis continues to grow. In response to this need, we introduce Auto Bioinformatics Analysis (AutoBA), an autonomous AI agent based on a large language model designed explicitly for conventional omics data analysis. AutoBA simplifies the analytical process by requiring minimal user input while delivering detailed step-by-step plans for various bioinformatics tasks. Through rigorous validation by expert bioinformaticians, AutoBAs robustness and adaptability are affirmed across a diverse range of omics analysis cases, including whole genome sequencing (WGS), RNA sequencing (RNA-seq), single-cell RNA-seq, ChIP-seq, and spatial transcriptomics. AutoBAs unique capacity to self-design analysis processes based on input data variations further underscores its versatility. Compared with online bioinformatic services, AutoBA deploys the analysis locally, preserving data privacy. Moreover, different from the predefined pipeline, AutoBA has adaptability in sync with emerging bioinformatics tools. Overall, AutoBA represents a convenient tool, offering robustness and adaptability for complex omics data analysis.\nAllo: Accurate allocation of multi-mapped reads enables regulatory element analysis at repeats\nAuthors: Morrissey, A.; Shi, J.; James, D. Q.; Mahony, S.\nScore: 12.1, Published: 2023-09-15 DOI: 10.1101/2023.09.12.556916\nTransposable elements (TEs) and other repetitive regions have been shown to contain gene regulatory elements, including transcription factor binding sites. Unfortunately, regulatory elements harbored by repeats have proven difficult to characterize using short-read sequencing assays such as ChIP-seq or ATAC-seq. Most regulatory genomics analysis pipelines discard \"multi-mapped\" reads that align equally well to multiple genomic locations. Since multi-mapped reads arise predominantly from repeats, current analysis pipelines fail to detect a substantial portion of regulatory events that occur in repetitive regions. To address this shortcoming, we developed Allo, a new approach to allocate multi-mapped reads in an efficient, accurate, and user-friendly manner. Allo combines probabilistic mapping of multi-mapped reads with a convolutional neural network that recognizes the read distribution features of potential peaks, offering enhanced accuracy in multi-mapping read assignment. Allo also provides read-level output in the form of a corrected alignment file, making it compatible with existing regulatory genomics analysis pipelines and downstream peak-finders. In a demonstration application on CTCF ChIP-seq data, we show that Allo results in the discovery of thousands of new CTCF peaks. Many of these peaks contain the expected cognate motif and/or serve as TAD boundaries. We additionally apply Allo to a diverse collection of ENCODE ChIP-seq datasets, resulting in multiple previously unidentified interactions between transcription factors and repetitive element families. Finally, we show that Allo may be particularly effective in identifying ChIP-seq peaks in younger TEs, which hold evolutionary significance due to their emergence during human evolution from primates.\n",
  "wordCount" : "2049",
  "inLanguage": "en",
  "datePublished": "2023-09-17T10:36:50Z",
  "dateModified": "2023-09-17T10:36:50Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on September 17, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.10.557072">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.10.557072" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.10.557072">
        <p class="paperTitle">The tidyomics ecosystem: Enhancing omic data analyses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.10.557072" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.10.557072" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hutchison, W. J.; Keyes, T. J.; Crowell, H. L.; Soneson, C.; Mu, W.; Park, J.-E.; Davis, E. S.; Nahid, A. A.; Tang, M.; Yuan, V.; Axisa, P.-P.; Kitt, J. W.; Poon, C.-L.; Sato, N.; Gottardo, R.; Morgan, M.; Lee, S.; Lawrence, M.; Hicks, S. C.; Nolan, G. P.; Davis, K. L.; Papenfuss, A. T.; Love, M. I.; Mangiola, S.</p>
        <p class="info">Score: 118.1, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.10.557072' target='https://doi.org/10.1101/2023.09.10.557072'> 10.1101/2023.09.10.557072</a></p>
        <p class="abstract">The exponential growth of omic data presents challenges in data manipulation, analysis, and integration. Addressing these challenges, Bioconductor offers an extensive data analysis platform and community, while R tidy programming offers a standard data organisation and manipulation that has revolutionised data science. Bioconductor and tidy R have mostly remained independent; bridging these two ecosystems would streamline omic analysis, ease learning and encourage cross-disciplinary collaborations. Here, we introduce the tidyomics ecosystem--a suite of interoperable software that brings the vast tidy software ecosystem to omic data analysis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557287">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557287" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557287">
        <p class="paperTitle">Cell2Sentence: Teaching Large Language Models the Language of Biology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557287" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557287" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Levine, D.; Rizvi, S. A.; Levy, S.; Pallikkavaliyaveetil MohammedSheriff, N.; Wu, R.; Zhang, Z.; Fonseca, A.; Chen, X.; Ghadermarzi, S.; Dhodapkar, R. M.; van Dijk, D.</p>
        <p class="info">Score: 71.2, Published: 2023-09-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557287' target='https://doi.org/10.1101/2023.09.11.557287'> 10.1101/2023.09.11.557287</a></p>
        <p class="abstract">Large language models like GPT have shown impressive performance on natural language tasks. Here, we present a novel method to directly adapt these pretrained models to a biological context, specifically single-cell transcriptomics, by representing gene expression data as text. Our Cell2Sentence approach converts each cell&#39;s gene expression profile into a sequence of gene names ordered by expression level. We show that these gene sequences, which we term &#34;cell sentences&#34;, can be used to fine-tune causal language models like GPT-2. Critically, we find that natural language pretraining boosts model performance on cell sentence tasks. When fine-tuned on cell sentences, GPT-2 generates biologically valid cells when prompted with a cell type. Conversely, it can also accurately predict cell type labels when prompted with cell sentences. This demonstrates that language models fine-tuned using Cell2Sentence can gain a biological understanding of single-cell data, while retaining their ability to generate text. Our approach provides a simple, adaptable framework to combine natural language and transcriptomics using existing models and libraries. Our code is available at: https://github.com/vandijklab/cell2sentence-ft.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.09.556981">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.09.556981" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.09.556981">
        <p class="paperTitle">CodonBERT: Large Language Models for mRNA Design and Optimization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.09.556981" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.09.556981" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, S.; Moayedpour, S.; Li, R.; Bailey, M.; Riahi, S.; Miladi, M.; Miner, J.; Zheng, D.; Wang, J.; Balsubramani, A.; Tran, K.; Zacharia, M.; Wu, M.; Gu, X.; Clinton, R.; Asquith, C.; Skalesk, J.; Boeglin, L.; Chivukula, S.; Dias, A.; Montoya, F. U.; Agarwal, V.; Bar-Joseph, Z.; Dr. Jager, S.</p>
        <p class="info">Score: 61.0, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.09.556981' target='https://doi.org/10.1101/2023.09.09.556981'> 10.1101/2023.09.09.556981</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWmRNA based vaccines and therapeutics are gaining popularity and usage across a wide range of conditions. One of the critical issues when designing such mRNAs is sequence optimization. Even small proteins or peptides can be encoded by an enormously large number of mRNAs. The actual mRNA sequence can have a large impact on several properties including expression, stability, immunogenicity, and more. To enable the selection of an optimal sequence, we developed CodonBERT, a large language model (LLM) for mRNAs. Unlike prior models, CodonBERT uses codons as inputs which enables it to learn better representations. CodonBERT was trained using more than 10 million mRNA sequences from a diverse set of organisms. The resulting model captures important biological concepts. CodonBERT can also be extended to perform prediction tasks for various mRNA properties. CodonBERT outperforms previous mRNA prediction methods including on a new flu vaccine dataset.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556812">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556812" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556812">
        <p class="paperTitle">Encoding the Space of Protein-protein Binding Interfaces by Artificial Intelligence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556812" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556812" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Su, Z.; Dhusia, K.; Wu, Y.</p>
        <p class="info">Score: 27.1, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556812' target='https://doi.org/10.1101/2023.09.08.556812'> 10.1101/2023.09.08.556812</a></p>
        <p class="abstract">The physical interactions between proteins are largely determined by the structural properties at their binding interfaces. It was found that the binding interfaces in distinctive protein complexes are highly similar. The structural properties underlying different binding interfaces could be further captured by artificial intelligence. In order to test this hypothesis, we broke protein-protein binding interfaces into pairs of interacting fragments. We employed a generative model to encode these interface fragment pairs in a low-dimensional latent space. After training, new conformations of interface fragment pairs were generated. We found that, by only using a small number of interface fragment pairs that were generated by artificial intelligence, we were able to guide the assembly of protein complexes into their native conformations. These results demonstrate that the conformational space of fragment pairs at protein-protein binding interfaces is highly degenerate. Our study illustrates how artificial intelligence can be used to understand and characterize protein-protein binding interfaces. The method will be potentially useful to search for the conformation of unknown protein-protein interactions. This result demonstrated that the structural space of protein-protein interactions is highly degenerate under the representation of interface fragment pairs. Features in this degenerate space can be well characterized by artificial intelligence. In summary, our machine learning method will be potentially useful to search for and predict the conformations of unknown protein-protein interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557209">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557209" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557209">
        <p class="paperTitle">GraffiTE: a Unified Framework to Analyze Transposable Element Insertion Polymorphisms using Genome-graphs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557209" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557209" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Groza, C.; Chen, X.; Wheeler, T. J.; Bourque, G.; Goubert, C.</p>
        <p class="info">Score: 26.9, Published: 2023-09-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557209' target='https://doi.org/10.1101/2023.09.11.557209'> 10.1101/2023.09.11.557209</a></p>
        <p class="abstract">Transposable Elements (TEs) are abundant and mobile repetitive DNA sequences evolving within and across their hosts&#39; genomes. Active TEs cause insertion polymorphism and contribute to genomic diversity. Here, we present GraffiTE, a flexible and comprehensive pipeline for detecting and genotyping polymorphic mobile elements (pMEs). By integrating state-of-the-art SV detection algorithms and graph-genome frameworks, GraffiTE enables the accurate identification of pMEs from genomic assemblies and long-read as well as the precise genotyping of these variants using short- or long-read data. Performance evaluations using simulated and benchmark datasets demonstrate high precision and recall rates. Notably, we demonstrate the versatility of GraffiTE by analyzing the human reference pangenome, 30 Drosophila melanogaster genomes, and multiple cultivars of the emerging crop model Cannabis sativa, where pMEs are undocumented. These analyses reveal the landscapes of pMEs and their frequency variations across individuals, strains, and cultivars. GraffiTE provides a user-friendly interface, allowing non-expert users to perform comprehensive pME analyses, including in models with limited TE prior knowledge. The pipeline&#39;s extensible design and compatibility with various sequencing technologies make it a valuable integrative framework for studying TE dynamics and their impact on genome evolution. GraffiTE is freely available at https://github.com/cgroza/GraffiTE.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556912">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556912" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556912">
        <p class="paperTitle">Bayesian phylogenetics on globally emerging SARS-CoV-2 variant BA.2.86 suggest global distribution and rapid evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556912" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556912" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rothstein, A. P.; Qiu, X.; Robison, K.; Collins, S.; Muir, G.; Lu, B.; Plocik, A.; Simen, B.; Philipson, C.</p>
        <p class="info">Score: 26.5, Published: 2023-09-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556912' target='https://doi.org/10.1101/2023.09.08.556912'> 10.1101/2023.09.08.556912</a></p>
        <p class="abstract">Using bioinformatic pipelines and Bayseian phylogenetic analyses, we characterized a SARS-CoV-2 variant designated by the World Health Organization as a variant under monitoring in August 2023. Here we analyze the genomes of this SARS-CoV-2 variant, BA.2.86, deposited into GISAID within the two weeks of its emergence (2023-08-14 first submission to 2023-08-31), including the first BA.2.86 genome reported from a traveler originating from Japan. We present bioinformatics methods using publicly available tools to help analysts identify the lineage-defining 12 nucleotide insertion (S:Ins16MPLF), which is often masked by most bioinformatics pipelines. We also applied maximum-likelihood and Bayesian phylogenetics to demonstrate the high mutational rate of the tree branch leading to the emergence of BA.2.86, hinting at possible origins, and predict that BA.2.86 emerged around May 2023 and spread globally rapidly. Taken together, these results provide a framework for more rigorous bioinformatics approaches for teams performing genomic surveillance on viral respiratory pathogens.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556731">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556731" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556731">
        <p class="paperTitle">Local read haplotagging enables accurate long-read small variant calling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556731" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556731" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kolesnikov, A.; Cook, D. E.; Nattestad, M.; Ashley, E. A.; Gorzynski, J.; Goenka, S. D.; Jain, M.; McNulty, B.; Miga, K. H.; Paten, B.; Chang, P.-C.; Carroll, A.; Shafin, K.</p>
        <p class="info">Score: 19.4, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556731' target='https://doi.org/10.1101/2023.09.07.556731'> 10.1101/2023.09.07.556731</a></p>
        <p class="abstract">Long-read sequencing technology has enabled variant detection in difficult-to-map regions of the genome and enabled rapid genetic diagnosis in clinical settings. Rapidly evolving third-generation sequencing platforms like Pacific Biosciences (PacBio) and Oxford nanopore technologies (ONT) are introducing newer platforms and data types. It has been demonstrated that variant calling methods based on deep neural networks can use local haplotyping information with long-reads to improve the genotyping accuracy. However, using local haplotype information creates an overhead as variant calling needs to be performed multiple times which ultimately makes it difficult to extend to new data types and platforms as they get introduced. In this work, we have developed a local haplotype approximate method that enables state-of-the-art variant calling performance with multiple sequencing platforms including PacBio Revio system, ONT R10.4 simplex and duplex data. This addition of local haplotype approximation makes DeepVariant a universal variant calling solution for long-read sequencing platforms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.555192">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.555192" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.555192">
        <p class="paperTitle">Evaluating the Utilities of Large Language Models in Single-cell Data Analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.555192" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.555192" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, T.; Li, K.; Wang, Y.; Li, H.; Zhao, H.</p>
        <p class="info">Score: 20.3, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.555192' target='https://doi.org/10.1101/2023.09.08.555192'> 10.1101/2023.09.08.555192</a></p>
        <p class="abstract">Large Language Models (LLMs) or foundation models have made significant strides in both industrial and scientific domains. In this paper, we evaluate the performance of LLMs in single-cell sequencing data analysis through comprehensive experiments across eight downstream tasks pertinent to single-cell data. By comparing seven different single-cell LLMs with task-specific methods, we found that single-cell LLMs may not consistently excel in all tasks than task-specific methods. However, the emergent abilities and the successful applications of cross-species/cross-modality transfer learning of LLMs are promising. In addition, we present a systematic evaluation of the effects of hyper-parameters, initial settings, and stability for training single-cell LLMs based on a proposed scEval framework, and provide guidelines for pre-training and fine-tuning. Our work summarizes the current state of single-cell LLMs, and points to their constraints and avenues for future investigation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556814">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556814" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556814">
        <p class="paperTitle">Automated Bioinformatics Analysis via AutoBA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556814" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556814" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhou, J.; Zhang, B.; Chen, X.; Li, H.; Xu, X.; Chen, S.; Gao, X.</p>
        <p class="info">Score: 12.9, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556814' target='https://doi.org/10.1101/2023.09.08.556814'> 10.1101/2023.09.08.556814</a></p>
        <p class="abstract">With the fast-growing and evolving omics data, the demand for streamlined and adaptable tools to handle the analysis continues to grow. In response to this need, we introduce Auto Bioinformatics Analysis (AutoBA), an autonomous AI agent based on a large language model designed explicitly for conventional omics data analysis. AutoBA simplifies the analytical process by requiring minimal user input while delivering detailed step-by-step plans for various bioinformatics tasks. Through rigorous validation by expert bioinformaticians, AutoBAs robustness and adaptability are affirmed across a diverse range of omics analysis cases, including whole genome sequencing (WGS), RNA sequencing (RNA-seq), single-cell RNA-seq, ChIP-seq, and spatial transcriptomics. AutoBAs unique capacity to self-design analysis processes based on input data variations further underscores its versatility. Compared with online bioinformatic services, AutoBA deploys the analysis locally, preserving data privacy. Moreover, different from the predefined pipeline, AutoBA has adaptability in sync with emerging bioinformatics tools. Overall, AutoBA represents a convenient tool, offering robustness and adaptability for complex omics data analysis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.556916">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.556916" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.556916">
        <p class="paperTitle">Allo: Accurate allocation of multi-mapped reads enables regulatory element analysis at repeats</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.556916" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.556916" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Morrissey, A.; Shi, J.; James, D. Q.; Mahony, S.</p>
        <p class="info">Score: 12.1, Published: 2023-09-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.556916' target='https://doi.org/10.1101/2023.09.12.556916'> 10.1101/2023.09.12.556916</a></p>
        <p class="abstract">Transposable elements (TEs) and other repetitive regions have been shown to contain gene regulatory elements, including transcription factor binding sites. Unfortunately, regulatory elements harbored by repeats have proven difficult to characterize using short-read sequencing assays such as ChIP-seq or ATAC-seq. Most regulatory genomics analysis pipelines discard &#34;multi-mapped&#34; reads that align equally well to multiple genomic locations. Since multi-mapped reads arise predominantly from repeats, current analysis pipelines fail to detect a substantial portion of regulatory events that occur in repetitive regions. To address this shortcoming, we developed Allo, a new approach to allocate multi-mapped reads in an efficient, accurate, and user-friendly manner. Allo combines probabilistic mapping of multi-mapped reads with a convolutional neural network that recognizes the read distribution features of potential peaks, offering enhanced accuracy in multi-mapping read assignment. Allo also provides read-level output in the form of a corrected alignment file, making it compatible with existing regulatory genomics analysis pipelines and downstream peak-finders. In a demonstration application on CTCF ChIP-seq data, we show that Allo results in the discovery of thousands of new CTCF peaks. Many of these peaks contain the expected cognate motif and/or serve as TAD boundaries. We additionally apply Allo to a diverse collection of ENCODE ChIP-seq datasets, resulting in multiple previously unidentified interactions between transcription factors and repetitive element families. Finally, we show that Allo may be particularly effective in identifying ChIP-seq peaks in younger TEs, which hold evolutionary significance due to their emergence during human evolution from primates.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
