<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Accurate human genome analysis with Element Avidity sequencing
Authors: Carroll, A.; Kolesnikov, A.; Cook, D. E.; Brambrink, L.; Wiseman, K. N.; Billings, S. M.; Kruglyak, S.; Lajoie, B. R.; Zhao, J.; Levy, S. E.; McLean, C. Y.; Shafin, K.; Nattestad, M.; Chang, P.-C.
Score: 543.5, Published: 2023-08-14 DOI: 10.1101/2023.08.11.553043
We investigate the new sequencing technology Avidity from Element Biosciences. We show that Element whole genome sequencing achieves higher mapping and variant calling accuracy compared to Illumina sequencing at the same coverage, with larger differences at lower coverages (20x-30x).">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="Accurate human genome analysis with Element Avidity sequencing
Authors: Carroll, A.; Kolesnikov, A.; Cook, D. E.; Brambrink, L.; Wiseman, K. N.; Billings, S. M.; Kruglyak, S.; Lajoie, B. R.; Zhao, J.; Levy, S. E.; McLean, C. Y.; Shafin, K.; Nattestad, M.; Chang, P.-C.
Score: 543.5, Published: 2023-08-14 DOI: 10.1101/2023.08.11.553043
We investigate the new sequencing technology Avidity from Element Biosciences. We show that Element whole genome sequencing achieves higher mapping and variant calling accuracy compared to Illumina sequencing at the same coverage, with larger differences at lower coverages (20x-30x)." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-13T10:39:03+00:00" />
<meta property="article:modified_time" content="2023-09-13T10:39:03+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="Accurate human genome analysis with Element Avidity sequencing
Authors: Carroll, A.; Kolesnikov, A.; Cook, D. E.; Brambrink, L.; Wiseman, K. N.; Billings, S. M.; Kruglyak, S.; Lajoie, B. R.; Zhao, J.; Levy, S. E.; McLean, C. Y.; Shafin, K.; Nattestad, M.; Chang, P.-C.
Score: 543.5, Published: 2023-08-14 DOI: 10.1101/2023.08.11.553043
We investigate the new sequencing technology Avidity from Element Biosciences. We show that Element whole genome sequencing achieves higher mapping and variant calling accuracy compared to Illumina sequencing at the same coverage, with larger differences at lower coverages (20x-30x)."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "Accurate human genome analysis with Element Avidity sequencing\nAuthors: Carroll, A.; Kolesnikov, A.; Cook, D. E.; Brambrink, L.; Wiseman, K. N.; Billings, S. M.; Kruglyak, S.; Lajoie, B. R.; Zhao, J.; Levy, S. E.; McLean, C. Y.; Shafin, K.; Nattestad, M.; Chang, P.-C.\nScore: 543.5, Published: 2023-08-14 DOI: 10.1101/2023.08.11.553043\nWe investigate the new sequencing technology Avidity from Element Biosciences. We show that Element whole genome sequencing achieves higher mapping and variant calling accuracy compared to Illumina sequencing at the same coverage, with larger differences at lower coverages (20x-30x).",
  "keywords": [
    
  ],
  "articleBody": " Accurate human genome analysis with Element Avidity sequencing\nAuthors: Carroll, A.; Kolesnikov, A.; Cook, D. E.; Brambrink, L.; Wiseman, K. N.; Billings, S. M.; Kruglyak, S.; Lajoie, B. R.; Zhao, J.; Levy, S. E.; McLean, C. Y.; Shafin, K.; Nattestad, M.; Chang, P.-C.\nScore: 543.5, Published: 2023-08-14 DOI: 10.1101/2023.08.11.553043\nWe investigate the new sequencing technology Avidity from Element Biosciences. We show that Element whole genome sequencing achieves higher mapping and variant calling accuracy compared to Illumina sequencing at the same coverage, with larger differences at lower coverages (20x-30x). We quantify base error rates of Element reads, finding lower error rates, especially in homopolymer and tandem repeat regions. We use Elements ability to generate paired end sequencing with longer insert sizes than typical short-read sequencing. We show that longer insert sizes result in even higher accuracy, with long insert Element sequencing giving noticeably more accurate genome analyses at all coverages.\nGene count normalization in single-cell imaging-based spatially resolved transcriptomics\nAuthors: Atta, L.; Clifton, K.; Anant, M.; Fan, J.\nScore: 38.1, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555624\nRecent advances in imaging-based spatially resolved transcriptomics (im-SRT) technologies now enable high-throughput profiling of targeted genes and their locations in fixed tissues. Normalization of gene expression data is often needed to account for technical factors that may confound underlying biological signals. Here, we investigate the potential impact of different gene count normalization methods with different targeted gene panels in the analysis and interpretation of im-SRT data. Using different simulated gene panels that overrepresent genes expressed in specific tissue anatomical regions or cell types, we find that normalization methods that use scaling factors derived from gene counts differentially impact normalized gene expression magnitudes in a region- or cell type-specific manner. We show that these normalization-induced effects may reduce the reliability of downstream differential gene expression and fold change analysis, introducing false positive and false negative results when compared to results obtained from gene panels that are more representative of the gene expression of the tissues component cell types. These effects are not observed without normalization or when scaling factors are not derived from gene counts, such as with cell volume normalization. Overall, we caution that the choice of normalization method and gene panel may impact the biological interpretation of the im-SRT data.\nDeep Learning for Protein Peptide bindingPrediction: Incorporating Sequence, Structural andLanguage Model Features\nAuthors: Chandra, A.; Sharma, A.; Dehzangi, I.; Tsunoda, T.; Sattar, A.\nScore: 28.8, Published: 2023-09-06 DOI: 10.1101/2023.09.02.556055\nProtein-peptide interactions play a crucial role in various cellular processes and are implicated in abnormal cellular behaviors leading to diseases such as cancer. Therefore, understanding these interactions is vital for both functional genomics and drug discovery efforts. Despite a significant increase in the availability of protein-peptide complexes, experimental methods for studying these interactions remain laborious, time-consuming, and expensive. Computational methods offer a complementary approach but often fall short in terms of prediction accuracy. To address these challenges, we introduce PepCNN, a deep learning-based prediction model that incorporates structural and sequence-based information from primary protein sequences. By utilizing a combination of half-sphere exposure, position specific scoring matrices, and pre-trained transformer language model, PepCNN outperforms state-of-the-art methods in terms of specificity, precision, and AUC. The PepCNN software and datasets are publicly available at https://github.com/abelavit/PepCNN.git.\nEncoding the Space of Protein-protein Binding Interfaces by Artificial Intelligence\nAuthors: Su, Z.; Dhusia, K.; Wu, Y.\nScore: 26.3, Published: 2023-09-10 DOI: 10.1101/2023.09.08.556812\nThe physical interactions between proteins are largely determined by the structural properties at their binding interfaces. It was found that the binding interfaces in distinctive protein complexes are highly similar. The structural properties underlying different binding interfaces could be further captured by artificial intelligence. In order to test this hypothesis, we broke protein-protein binding interfaces into pairs of interacting fragments. We employed a generative model to encode these interface fragment pairs in a low-dimensional latent space. After training, new conformations of interface fragment pairs were generated. We found that, by only using a small number of interface fragment pairs that were generated by artificial intelligence, we were able to guide the assembly of protein complexes into their native conformations. These results demonstrate that the conformational space of fragment pairs at protein-protein binding interfaces is highly degenerate. Our study illustrates how artificial intelligence can be used to understand and characterize protein-protein binding interfaces. The method will be potentially useful to search for the conformation of unknown protein-protein interactions. This result demonstrated that the structural space of protein-protein interactions is highly degenerate under the representation of interface fragment pairs. Features in this degenerate space can be well characterized by artificial intelligence. In summary, our machine learning method will be potentially useful to search for and predict the conformations of unknown protein-protein interactions.\nBayesian phylogenetics on globally emerging SARS-CoV-2 variant BA.2.86 suggest global distribution and rapid evolution\nAuthors: Rothstein, A. P.; Qiu, X.; Robison, K.; Collins, S.; Muir, G.; Lu, B.; Plocik, A.; Simen, B.; Philipson, C.\nScore: 23.3, Published: 2023-09-11 DOI: 10.1101/2023.09.08.556912\nUsing bioinformatic pipelines and Bayseian phylogenetic analyses, we characterized a SARS-CoV-2 variant designated by the World Health Organization as a variant under monitoring in August 2023. Here we analyze the genomes of this SARS-CoV-2 variant, BA.2.86, deposited into GISAID within the two weeks of its emergence (2023-08-14 first submission to 2023-08-31), including the first BA.2.86 genome reported from a traveler originating from Japan. We present bioinformatics methods using publicly available tools to help analysts identify the lineage-defining 12 nucleotide insertion (S:Ins16MPLF), which is often masked by most bioinformatics pipelines. We also applied maximum-likelihood and Bayesian phylogenetics to demonstrate the high mutational rate of the tree branch leading to the emergence of BA.2.86, hinting at possible origins, and predict that BA.2.86 emerged around May 2023 and spread globally rapidly. Taken together, these results provide a framework for more rigorous bioinformatics approaches for teams performing genomic surveillance on viral respiratory pathogens.\nAutometa 2: A versatile tool for recovering genomes from highly-complex metagenomic communities\nAuthors: Rees, E. R.; Uppal, S.; Clark, C.; Lail, A. J.; Waterworth, S. C.; Roesemann, S. D.; Wolf, K. A.; Kwan, J. C.\nScore: 20.3, Published: 2023-09-05 DOI: 10.1101/2023.09.01.555939\nIn 2019, we developed Autometa, an automated binning pipeline that is able to effectively recover metagenome-assembled genomes from complex environmental and non-model host-associated microbial communities. Autometa has gained widespread use in a variety of environments and has been applied in multiple research projects. However, the genome-binning workflow was at times overly complex and computationally demanding. As a consequence of Autometas diverse application, non-technical and technical researchers alike have noted its burdensome installation and inefficient as well as error-prone processes. Moreover its taxon-binning and genome-binning behaviors have remained obscure. For these reasons we set out to improve its accessibility, efficiency and efficacy to further enable the research community during their exploration of Earths environments. The highly augmented Autometa 2 release, which we present here, has vastly simplified installation, a graphical user interface and a refactored workflow for transparency and reproducibility. Furthermore, we conducted a parameter sweep on standardized community datasets to show that it is possible for Autometa to achieve better performance than any other binning pipeline, as judged by Adjusted Rand Index. Improvements in Autometa 2 enhance its accessibility for non-bioinformatic oriented researchers, scalability for large-scale and highly-complex samples and interpretation of recovered microbial communities. Graphical abstractAutometa: An automated taxon binning and genome binning workflow for single sample resolution of metagenomic communities.\nevopython: a Python package for feature-focused, comparative genomic data exploration\nAuthors: Mick, S. T.; Fiszbein, A.\nScore: 18.7, Published: 2023-09-06 DOI: 10.1101/2023.09.02.556042\nMotivationSoftware such as LiftOver allows for the intra- and inter-species conversion of genomic coordinates between genome assemblies, but this coordinate-centric workflow is naive to sequence alterations underling the conversion. For instance, it does not clarify the location of any insertions or deletions that might have occurred. ResultsTo facilitate inter-species, sequence-based analyses, we developed evopython, a simple, object-oriented Python package that enables the sequence-aware resolution of genomic coordinates directly from pairwise and multiple whole-genome alignment data. The output is a Python dictionary storing all information relevant to the alignment: the participating species names, the position of the alignment in each species genome assembly, and the aligned sequences. Availability and implementationThe source code and documentation are available at https://github.com/fiszbein-lab/evopython. Contactanafisz@bu.edu\nEvaluating the Utilities of Large Language Models in Single-cell Data Analysis\nAuthors: Liu, T.; Li, K.; Wang, Y.; Li, H.; Zhao, H.\nScore: 17.8, Published: 2023-09-10 DOI: 10.1101/2023.09.08.555192\nLarge Language Models (LLMs) or foundation models have made significant strides in both industrial and scientific domains. In this paper, we evaluate the performance of LLMs in single-cell sequencing data analysis through comprehensive experiments across eight downstream tasks pertinent to single-cell data. By comparing seven different single-cell LLMs with task-specific methods, we found that single-cell LLMs may not consistently excel in all tasks than task-specific methods. However, the emergent abilities and the successful applications of cross-species/cross-modality transfer learning of LLMs are promising. In addition, we present a systematic evaluation of the effects of hyper-parameters, initial settings, and stability for training single-cell LLMs based on a proposed scEval framework, and provide guidelines for pre-training and fine-tuning. Our work summarizes the current state of single-cell LLMs, and points to their constraints and avenues for future investigation.\nCodonBERT: Large Language Models for mRNA Design and Optimization\nAuthors: Li, S.; Moayedpour, S.; Li, R.; Bailey, M.; Riahi, S.; Miladi, M.; Miner, J.; Zheng, D.; Wang, J.; Balsubramani, A.; Tran, K.; Zacharia, M.; Wu, M.; Gu, X.; Clinton, R.; Asquith, C.; Skalesk, J.; Boeglin, L.; Chivukula, S.; Dias, A.; Montoya, F. U.; Agarwal, V.; Bar-Joseph, Z.; Dr. Jager, S.\nScore: 17.0, Published: 2023-09-12 DOI: 10.1101/2023.09.09.556981\nmRNA based vaccines and therapeutics are gaining popularity and usage across a wide range of conditions. One of the critical issues when designing such mRNAs is sequence optimization. Even small proteins or peptides can be encoded by an enormously large number of mRNAs. The actual mRNA sequence can have a large impact on several properties including expression, stability, immunogenicity, and more. To enable the selection of an optimal sequence, we developed CodonBERT, a large language model (LLM) for mRNAs. Unlike prior models, CodonBERT uses codons as inputs which enables it to learn better representations. CodonBERT was trained using more than 10 million mRNA sequences from a diverse set of organisms. The resulting model captures important biological concepts. CodonBERT can also be extended to perform prediction tasks for various mRNA properties. CodonBERT outperforms previous mRNA prediction methods including on a new flu vaccine dataset.\nLocal read haplotagging enables accurate long-read small variant calling\nAuthors: Kolesnikov, A.; Cook, D. E.; Nattestad, M.; Ashley, E. A.; Gorzynski, J.; Goenka, S. D.; Jain, M.; McNulty, B.; Miga, K. H.; Paten, B.; Chang, P.-C.; Carroll, A.; Shafin, K.\nScore: 12.4, Published: 2023-09-12 DOI: 10.1101/2023.09.07.556731\nLong-read sequencing technology has enabled variant detection in difficult-to-map regions of the genome and enabled rapid genetic diagnosis in clinical settings. Rapidly evolving third-generation sequencing platforms like Pacific Biosciences (PacBio) and Oxford nanopore technologies (ONT) are introducing newer platforms and data types. It has been demonstrated that variant calling methods based on deep neural networks can use local haplotyping information with long-reads to improve the genotyping accuracy. However, using local haplotype information creates an overhead as variant calling needs to be performed multiple times which ultimately makes it difficult to extend to new data types and platforms as they get introduced. In this work, we have developed a local haplotype approximate method that enables state-of-the-art variant calling performance with multiple sequencing platforms including PacBio Revio system, ONT R10.4 simplex and duplex data. This addition of local haplotype approximation makes DeepVariant a universal variant calling solution for long-read sequencing platforms.\n",
  "wordCount" : "1926",
  "inLanguage": "en",
  "datePublished": "2023-09-13T10:39:03Z",
  "dateModified": "2023-09-13T10:39:03Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta"><span>updated on September 13, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.553043">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.553043" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.553043">
        <p class="paperTitle">Accurate human genome analysis with Element Avidity sequencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.553043" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.553043" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carroll, A.; Kolesnikov, A.; Cook, D. E.; Brambrink, L.; Wiseman, K. N.; Billings, S. M.; Kruglyak, S.; Lajoie, B. R.; Zhao, J.; Levy, S. E.; McLean, C. Y.; Shafin, K.; Nattestad, M.; Chang, P.-C.</p>
        <p class="info">Score: 543.5, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.553043' target='https://doi.org/10.1101/2023.08.11.553043'> 10.1101/2023.08.11.553043</a></p>
        <p class="abstract">We investigate the new sequencing technology Avidity from Element Biosciences. We show that Element whole genome sequencing achieves higher mapping and variant calling accuracy compared to Illumina sequencing at the same coverage, with larger differences at lower coverages (20x-30x). We quantify base error rates of Element reads, finding lower error rates, especially in homopolymer and tandem repeat regions. We use Elements ability to generate paired end sequencing with longer insert sizes than typical short-read sequencing. We show that longer insert sizes result in even higher accuracy, with long insert Element sequencing giving noticeably more accurate genome analyses at all coverages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555624">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555624" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555624">
        <p class="paperTitle">Gene count normalization in single-cell imaging-based spatially resolved transcriptomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555624" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555624" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Atta, L.; Clifton, K.; Anant, M.; Fan, J.</p>
        <p class="info">Score: 38.1, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555624' target='https://doi.org/10.1101/2023.08.30.555624'> 10.1101/2023.08.30.555624</a></p>
        <p class="abstract">Recent advances in imaging-based spatially resolved transcriptomics (im-SRT) technologies now enable high-throughput profiling of targeted genes and their locations in fixed tissues. Normalization of gene expression data is often needed to account for technical factors that may confound underlying biological signals. Here, we investigate the potential impact of different gene count normalization methods with different targeted gene panels in the analysis and interpretation of im-SRT data. Using different simulated gene panels that overrepresent genes expressed in specific tissue anatomical regions or cell types, we find that normalization methods that use scaling factors derived from gene counts differentially impact normalized gene expression magnitudes in a region- or cell type-specific manner. We show that these normalization-induced effects may reduce the reliability of downstream differential gene expression and fold change analysis, introducing false positive and false negative results when compared to results obtained from gene panels that are more representative of the gene expression of the tissues component cell types. These effects are not observed without normalization or when scaling factors are not derived from gene counts, such as with cell volume normalization. Overall, we caution that the choice of normalization method and gene panel may impact the biological interpretation of the im-SRT data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.02.556055">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.02.556055" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.02.556055">
        <p class="paperTitle">Deep Learning for Protein Peptide bindingPrediction: Incorporating Sequence, Structural andLanguage Model Features</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.02.556055" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.02.556055" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chandra, A.; Sharma, A.; Dehzangi, I.; Tsunoda, T.; Sattar, A.</p>
        <p class="info">Score: 28.8, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.02.556055' target='https://doi.org/10.1101/2023.09.02.556055'> 10.1101/2023.09.02.556055</a></p>
        <p class="abstract">Protein-peptide interactions play a crucial role in various cellular processes and are implicated in abnormal cellular behaviors leading to diseases such as cancer. Therefore, understanding these interactions is vital for both functional genomics and drug discovery efforts. Despite a significant increase in the availability of protein-peptide complexes, experimental methods for studying these interactions remain laborious, time-consuming, and expensive. Computational methods offer a complementary approach but often fall short in terms of prediction accuracy. To address these challenges, we introduce PepCNN, a deep learning-based prediction model that incorporates structural and sequence-based information from primary protein sequences. By utilizing a combination of half-sphere exposure, position specific scoring matrices, and pre-trained transformer language model, PepCNN outperforms state-of-the-art methods in terms of specificity, precision, and AUC. The PepCNN software and datasets are publicly available at https://github.com/abelavit/PepCNN.git.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556812">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556812" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556812">
        <p class="paperTitle">Encoding the Space of Protein-protein Binding Interfaces by Artificial Intelligence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556812" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556812" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Su, Z.; Dhusia, K.; Wu, Y.</p>
        <p class="info">Score: 26.3, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556812' target='https://doi.org/10.1101/2023.09.08.556812'> 10.1101/2023.09.08.556812</a></p>
        <p class="abstract">The physical interactions between proteins are largely determined by the structural properties at their binding interfaces. It was found that the binding interfaces in distinctive protein complexes are highly similar. The structural properties underlying different binding interfaces could be further captured by artificial intelligence. In order to test this hypothesis, we broke protein-protein binding interfaces into pairs of interacting fragments. We employed a generative model to encode these interface fragment pairs in a low-dimensional latent space. After training, new conformations of interface fragment pairs were generated. We found that, by only using a small number of interface fragment pairs that were generated by artificial intelligence, we were able to guide the assembly of protein complexes into their native conformations. These results demonstrate that the conformational space of fragment pairs at protein-protein binding interfaces is highly degenerate. Our study illustrates how artificial intelligence can be used to understand and characterize protein-protein binding interfaces. The method will be potentially useful to search for the conformation of unknown protein-protein interactions. This result demonstrated that the structural space of protein-protein interactions is highly degenerate under the representation of interface fragment pairs. Features in this degenerate space can be well characterized by artificial intelligence. In summary, our machine learning method will be potentially useful to search for and predict the conformations of unknown protein-protein interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556912">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556912" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556912">
        <p class="paperTitle">Bayesian phylogenetics on globally emerging SARS-CoV-2 variant BA.2.86 suggest global distribution and rapid evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556912" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556912" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rothstein, A. P.; Qiu, X.; Robison, K.; Collins, S.; Muir, G.; Lu, B.; Plocik, A.; Simen, B.; Philipson, C.</p>
        <p class="info">Score: 23.3, Published: 2023-09-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556912' target='https://doi.org/10.1101/2023.09.08.556912'> 10.1101/2023.09.08.556912</a></p>
        <p class="abstract">Using bioinformatic pipelines and Bayseian phylogenetic analyses, we characterized a SARS-CoV-2 variant designated by the World Health Organization as a variant under monitoring in August 2023. Here we analyze the genomes of this SARS-CoV-2 variant, BA.2.86, deposited into GISAID within the two weeks of its emergence (2023-08-14 first submission to 2023-08-31), including the first BA.2.86 genome reported from a traveler originating from Japan. We present bioinformatics methods using publicly available tools to help analysts identify the lineage-defining 12 nucleotide insertion (S:Ins16MPLF), which is often masked by most bioinformatics pipelines. We also applied maximum-likelihood and Bayesian phylogenetics to demonstrate the high mutational rate of the tree branch leading to the emergence of BA.2.86, hinting at possible origins, and predict that BA.2.86 emerged around May 2023 and spread globally rapidly. Taken together, these results provide a framework for more rigorous bioinformatics approaches for teams performing genomic surveillance on viral respiratory pathogens.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555939">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555939" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555939">
        <p class="paperTitle">Autometa 2: A versatile tool for recovering genomes from highly-complex metagenomic communities</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555939" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555939" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rees, E. R.; Uppal, S.; Clark, C.; Lail, A. J.; Waterworth, S. C.; Roesemann, S. D.; Wolf, K. A.; Kwan, J. C.</p>
        <p class="info">Score: 20.3, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555939' target='https://doi.org/10.1101/2023.09.01.555939'> 10.1101/2023.09.01.555939</a></p>
        <p class="abstract">In 2019, we developed Autometa, an automated binning pipeline that is able to effectively recover metagenome-assembled genomes from complex environmental and non-model host-associated microbial communities. Autometa has gained widespread use in a variety of environments and has been applied in multiple research projects. However, the genome-binning workflow was at times overly complex and computationally demanding. As a consequence of Autometas diverse application, non-technical and technical researchers alike have noted its burdensome installation and inefficient as well as error-prone processes. Moreover its taxon-binning and genome-binning behaviors have remained obscure. For these reasons we set out to improve its accessibility, efficiency and efficacy to further enable the research community during their exploration of Earths environments. The highly augmented Autometa 2 release, which we present here, has vastly simplified installation, a graphical user interface and a refactored workflow for transparency and reproducibility. Furthermore, we conducted a parameter sweep on standardized community datasets to show that it is possible for Autometa to achieve better performance than any other binning pipeline, as judged by Adjusted Rand Index. Improvements in Autometa 2 enhance its accessibility for non-bioinformatic oriented researchers, scalability for large-scale and highly-complex samples and interpretation of recovered microbial communities.

Graphical abstractAutometa: An automated taxon binning and genome binning workflow for single sample resolution of metagenomic communities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.02.556042">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.02.556042" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.02.556042">
        <p class="paperTitle">evopython: a Python package for feature-focused, comparative genomic data exploration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.02.556042" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.02.556042" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mick, S. T.; Fiszbein, A.</p>
        <p class="info">Score: 18.7, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.02.556042' target='https://doi.org/10.1101/2023.09.02.556042'> 10.1101/2023.09.02.556042</a></p>
        <p class="abstract">MotivationSoftware such as LiftOver allows for the intra- and inter-species conversion of genomic coordinates between genome assemblies, but this coordinate-centric workflow is naive to sequence alterations underling the conversion. For instance, it does not clarify the location of any insertions or deletions that might have occurred.

ResultsTo facilitate inter-species, sequence-based analyses, we developed evopython, a simple, object-oriented Python package that enables the sequence-aware resolution of genomic coordinates directly from pairwise and multiple whole-genome alignment data. The output is a Python dictionary storing all information relevant to the alignment: the participating species names, the position of the alignment in each species genome assembly, and the aligned sequences.

Availability and implementationThe source code and documentation are available at https://github.com/fiszbein-lab/evopython.

Contactanafisz@bu.edu</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.555192">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.555192" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.555192">
        <p class="paperTitle">Evaluating the Utilities of Large Language Models in Single-cell Data Analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.555192" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.555192" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, T.; Li, K.; Wang, Y.; Li, H.; Zhao, H.</p>
        <p class="info">Score: 17.8, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.555192' target='https://doi.org/10.1101/2023.09.08.555192'> 10.1101/2023.09.08.555192</a></p>
        <p class="abstract">Large Language Models (LLMs) or foundation models have made significant strides in both industrial and scientific domains. In this paper, we evaluate the performance of LLMs in single-cell sequencing data analysis through comprehensive experiments across eight downstream tasks pertinent to single-cell data. By comparing seven different single-cell LLMs with task-specific methods, we found that single-cell LLMs may not consistently excel in all tasks than task-specific methods. However, the emergent abilities and the successful applications of cross-species/cross-modality transfer learning of LLMs are promising. In addition, we present a systematic evaluation of the effects of hyper-parameters, initial settings, and stability for training single-cell LLMs based on a proposed scEval framework, and provide guidelines for pre-training and fine-tuning. Our work summarizes the current state of single-cell LLMs, and points to their constraints and avenues for future investigation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.09.556981">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.09.556981" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.09.556981">
        <p class="paperTitle">CodonBERT: Large Language Models for mRNA Design and Optimization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.09.556981" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.09.556981" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, S.; Moayedpour, S.; Li, R.; Bailey, M.; Riahi, S.; Miladi, M.; Miner, J.; Zheng, D.; Wang, J.; Balsubramani, A.; Tran, K.; Zacharia, M.; Wu, M.; Gu, X.; Clinton, R.; Asquith, C.; Skalesk, J.; Boeglin, L.; Chivukula, S.; Dias, A.; Montoya, F. U.; Agarwal, V.; Bar-Joseph, Z.; Dr. Jager, S.</p>
        <p class="info">Score: 17.0, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.09.556981' target='https://doi.org/10.1101/2023.09.09.556981'> 10.1101/2023.09.09.556981</a></p>
        <p class="abstract">mRNA based vaccines and therapeutics are gaining popularity and usage across a wide range of conditions. One of the critical issues when designing such mRNAs is sequence optimization. Even small proteins or peptides can be encoded by an enormously large number of mRNAs. The actual mRNA sequence can have a large impact on several properties including expression, stability, immunogenicity, and more. To enable the selection of an optimal sequence, we developed CodonBERT, a large language model (LLM) for mRNAs. Unlike prior models, CodonBERT uses codons as inputs which enables it to learn better representations. CodonBERT was trained using more than 10 million mRNA sequences from a diverse set of organisms. The resulting model captures important biological concepts. CodonBERT can also be extended to perform prediction tasks for various mRNA properties. CodonBERT outperforms previous mRNA prediction methods including on a new flu vaccine dataset.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556731">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556731" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556731">
        <p class="paperTitle">Local read haplotagging enables accurate long-read small variant calling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556731" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556731" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kolesnikov, A.; Cook, D. E.; Nattestad, M.; Ashley, E. A.; Gorzynski, J.; Goenka, S. D.; Jain, M.; McNulty, B.; Miga, K. H.; Paten, B.; Chang, P.-C.; Carroll, A.; Shafin, K.</p>
        <p class="info">Score: 12.4, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556731' target='https://doi.org/10.1101/2023.09.07.556731'> 10.1101/2023.09.07.556731</a></p>
        <p class="abstract">Long-read sequencing technology has enabled variant detection in difficult-to-map regions of the genome and enabled rapid genetic diagnosis in clinical settings. Rapidly evolving third-generation sequencing platforms like Pacific Biosciences (PacBio) and Oxford nanopore technologies (ONT) are introducing newer platforms and data types. It has been demonstrated that variant calling methods based on deep neural networks can use local haplotyping information with long-reads to improve the genotyping accuracy. However, using local haplotype information creates an overhead as variant calling needs to be performed multiple times which ultimately makes it difficult to extend to new data types and platforms as they get introduced. In this work, we have developed a local haplotype approximate method that enables state-of-the-art variant calling performance with multiple sequencing platforms including PacBio Revio system, ONT R10.4 simplex and duplex data. This addition of local haplotype approximation makes DeepVariant a universal variant calling solution for long-read sequencing platforms.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
