<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>bioinformatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>bioinformatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Polygraph: A Software Framework for the Systematic Assessment of Synthetic Regulatory DNA Elements
Authors: Gupta, A.; Lal, A.; Gunsalus, L. M.; Biancalani, T.; Eraslan, G.
Score: 30.9, Published: 2023-11-27 DOI: 10.1101/2023.11.27.568764
The design of regulatory elements is pivotal in numerous therapeutic interventions, including gene and cell therapy, wherein the typical objective is to engineer DNA sequences exhibiting specific attributes like cell-type specificity and elevated expression levels. However, the systematic assessment of these constructed DNA sequences remains challenging due to the absence of robust metrics and an integrated software framework.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/bioinformatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="bioinformatics" />
<meta property="og:description" content="Polygraph: A Software Framework for the Systematic Assessment of Synthetic Regulatory DNA Elements
Authors: Gupta, A.; Lal, A.; Gunsalus, L. M.; Biancalani, T.; Eraslan, G.
Score: 30.9, Published: 2023-11-27 DOI: 10.1101/2023.11.27.568764
The design of regulatory elements is pivotal in numerous therapeutic interventions, including gene and cell therapy, wherein the typical objective is to engineer DNA sequences exhibiting specific attributes like cell-type specificity and elevated expression levels. However, the systematic assessment of these constructed DNA sequences remains challenging due to the absence of robust metrics and an integrated software framework." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/bioinformatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-03T10:38:01+00:00" />
<meta property="article:modified_time" content="2023-12-03T10:38:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="bioinformatics"/>
<meta name="twitter:description" content="Polygraph: A Software Framework for the Systematic Assessment of Synthetic Regulatory DNA Elements
Authors: Gupta, A.; Lal, A.; Gunsalus, L. M.; Biancalani, T.; Eraslan, G.
Score: 30.9, Published: 2023-11-27 DOI: 10.1101/2023.11.27.568764
The design of regulatory elements is pivotal in numerous therapeutic interventions, including gene and cell therapy, wherein the typical objective is to engineer DNA sequences exhibiting specific attributes like cell-type specificity and elevated expression levels. However, the systematic assessment of these constructed DNA sequences remains challenging due to the absence of robust metrics and an integrated software framework."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "bioinformatics",
      "item": "https://trxiv.yorks0n.com/posts/bioinformatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "bioinformatics",
  "name": "bioinformatics",
  "description": "Polygraph: A Software Framework for the Systematic Assessment of Synthetic Regulatory DNA Elements\nAuthors: Gupta, A.; Lal, A.; Gunsalus, L. M.; Biancalani, T.; Eraslan, G.\nScore: 30.9, Published: 2023-11-27 DOI: 10.1101/2023.11.27.568764\nThe design of regulatory elements is pivotal in numerous therapeutic interventions, including gene and cell therapy, wherein the typical objective is to engineer DNA sequences exhibiting specific attributes like cell-type specificity and elevated expression levels. However, the systematic assessment of these constructed DNA sequences remains challenging due to the absence of robust metrics and an integrated software framework.",
  "keywords": [
    
  ],
  "articleBody": " Polygraph: A Software Framework for the Systematic Assessment of Synthetic Regulatory DNA Elements\nAuthors: Gupta, A.; Lal, A.; Gunsalus, L. M.; Biancalani, T.; Eraslan, G.\nScore: 30.9, Published: 2023-11-27 DOI: 10.1101/2023.11.27.568764\nThe design of regulatory elements is pivotal in numerous therapeutic interventions, including gene and cell therapy, wherein the typical objective is to engineer DNA sequences exhibiting specific attributes like cell-type specificity and elevated expression levels. However, the systematic assessment of these constructed DNA sequences remains challenging due to the absence of robust metrics and an integrated software framework. Here, we introduce Polygraph, a Python framework for evaluating synthetic DNA sequences. Polygraph provides a variety of features to streamline the synthesis and scrutiny of regulatory elements, incorporating features like a diversity index, motif and k-mer composition, similarity to endogenous regulatory sequences, and screening with predictive and foundational models. Consequently, Polygraph stands as the first instrument for the assessment of synthetic regulatory sequences, enabling expedited progress in therapeutic applications and improving our understanding of gene regulatory mechanisms.\nLearning sequence, structure, and function representations of proteins with language models\nAuthors: Hamamsy, T.; Barot, M.; Morton, J. T.; Steinegger, M.; Bonneau, R.; Cho, K.\nScore: 26.2, Published: 2023-11-26 DOI: 10.1101/2023.11.26.568742\nThe sequence-structure-function relationships that ultimately generate the diversity of extant observed proteins is complex, as proteins bridge the gap between multiple informational and physical scales involved in nearly all cellular processes. One limitation of existing protein annotation databases such as UniProt is that less than 1% of proteins have experimentally verified functions, and computational methods are needed to fill in the missing information. Here, we demonstrate that a multi-aspect framework based on protein language models can learn sequence-structure-function representations of amino acid sequences, and can provide the foundation for sensitive sequence-structure-function aware protein sequence search and annotation. Based on this model, we introduce a multi-aspect information retrieval system for proteins, Protein-Vec, covering sequence, structure, and function aspects, that enables computational protein annotation and function prediction at tree-of-life scales.\nNanopore and Illumina Sequencing Reveal Different Viral Populations from Human Gut Samples\nAuthors: Cook, R.; Telatin, A.; Hsieh, S.-Y.; Newberry, F.; Tariq, M. A.; Baker, D. J.; Carding, S. R.; Adriaenssens, E. M.\nScore: 30.9, Published: 2023-11-24 DOI: 10.1101/2023.11.24.568560\nThe advent of viral metagenomics, or viromics, has improved our knowledge and understanding of global viral diversity. High-throughput sequencing technologies enable explorations of the ecological roles, contributions to host metabolism, and the influence of viruses in various environments including the human gut microbiome. However, the bacterial metagenomic studies frequently have the advantage. The adoption of advanced technologies like long-read sequencing has the potential to be transformative in refining viromics and metagenomics. Here, we examined the effectiveness of long-read and hybrid sequencing by comparing Illumina short-read and Oxford Nanopore Technology (ONT) long-read sequencing technologies and different assembly strategies on recovering viral genomes from human faecal samples. Our findings showed that if a single sequencing technology is to be chosen for virome analysis, Illumina was preferable due to its superior ability to recover fully resolved viral genomes and minimise erroneous genomes. While ONT assemblies were effective in recovering viral diversity, the challenges related to input requirements and the necessity for amplification made it less ideal as a standalone solution. However, using a combined, hybrid approach enabled a more authentic representation of viral diversity to be obtained within samples. Impact StatementViral metagenomics, or viromics, has revolutionised our understanding of global viral diversity however long-read and hybrid approaches are not yet widespread in this field. Here, we compared the performance of Illumina short-read and Nanopore long-read assembly approaches for recovering fully resolved viral genomes from human faecal samples. We highlight Illuminas short-read sequencing for recovering fully resolved viral genomes, while acknowledging Oxford Nanopore Technologys long-read sequencing for capturing broader viral diversity. However, a hybrid approach, utilising both technologies, may mitigate the limitations of one technology alone. Data SummaryAll reads used in this study are available on European Nucleotide Archive (ENA) within the project PRJEB47625.\nColabfold predicts alternative protein structures from single sequences, coevolution unnecessary for AF-cluster\nAuthors: Porter, L. L.; Chakravarty, D.; Schafer, J. W.; Chen, E. A.\nScore: 31.6, Published: 2023-11-25 DOI: 10.1101/2023.11.21.567977\nThough typically associated with a single folded state, globular proteins are dynamic and often assume alternative or transient structures important for their functions1,2. Wayment-Steele, et al. steered ColabFold3 to predict alternative structures of several proteins using a method they call AF-cluster4. They propose that AF-cluster \"enables ColabFold to sample alternate states of known metamorphic proteins with high confidence\" by first clustering multiple sequence alignments (MSAs) in a way that \"deconvolves\" coevolutionary information specific to different conformations and then using these clusters as input for ColabFold. Contrary to this Coevolution Assumption, clustered MSAs are not needed to make these predictions. Rather, these alternative structures can be predicted from single sequences and/or sequence similarity, indicating that coevolutionary information is unnecessary for predictive success and may not be used at all. These results suggest that AF-clusters predictive scope is likely limited to sequences with distinct-yet-homologous structures within ColabFolds training set.\nGiotto Suite: a multi-scale and technology-agnostic spatial multi-omics analysis ecosystem\nAuthors: Chen, J. G.; Chavez-Fuentes, J. C.; O'Brien, M.; Xu, J.; Ruiz, E.; Wang, W.; Amin, I.; Sarfraz, I.; Guckhool, P.; Sistig, A.; Yuan, G.-C.; Dries, R.\nScore: 23.2, Published: 2023-11-27 DOI: 10.1101/2023.11.26.568752\nEmerging spatial omics technologies continue to advance the molecular mapping of tissue architecture and the investigation of gene regulation and cellular crosstalk, which in turn provide new mechanistic insights into a wide range of biological processes and diseases. Such technologies provide an increasingly large amount of information content at multiple spatial scales. However, representing and harmonizing diverse spatial datasets efficiently, including combining multiple modalities or spatial scales in a scalable and flexible manner, remains a substantial challenge. Here, we present Giotto Suite, a suite of open-source software packages that underlies a fully modular and integrated spatial data analysis toolbox. At its core, Giotto Suite is centered around an innovative and technology-agnostic data framework embedded in the R software environment, which allows the representation and integration of virtually any type of spatial omics data at any spatial resolution. In addition, Giotto Suite provides both scalable and extensible end-to-end solutions for data analysis, integration, and visualization. Giotto Suite integrates molecular, morphology, spatial, and annotated feature information to create a responsive and flexible workflow for multi-scale, multi-omic data analyses, as demonstrated here by applications to several state-of-the-art spatial technologies. Furthermore, Giotto Suite builds upon interoperable interfaces and data structures that bridge the established fields of genomics and spatial data science, thereby enabling independent developers to create custom-engineered pipelines. As such, Giotto Suite creates an immersive ecosystem for spatial multi-omic data analysis.\nStreamlining remote nanopore data access with slow5curl\nAuthors: Wong, B.; Ferguson, J. M.; Gamaarachchi, H.; Deveson, I. W.\nScore: 17.5, Published: 2023-11-30 DOI: 10.1101/2023.11.28.569128\nAs adoption of nanopore sequencing technology continues to advance, the need to maintain large volumes of raw current signal data for reanalysis with updated algorithms is a growing challenge. Here we introduce slow5curl, a software package designed to streamline nanopore data sharing, accessibility and reanalysis. Slow5curl allows a user to fetch a specified read or group of reads from a raw nanopore dataset stored on a remote server, such as a public data repository, without downloading the entire file. Slow5curl uses an index to quickly fetch specific reads from a large dataset in SLOW5/BLOW5 format and highly parallelised data access requests to maximise download speeds. Using all public nanopore data from the Human Pangenome Reference Consortium (\u003e22 TB), we demonstrate how slow5curl can be used to quickly fetch and reanalyse signal reads corresponding to a set of target genes from each individual in large cohort dataset (n = 91), minimising the time, egress costs, and local storage requirements for their reanalysis. We provide slow5curl as a free, open-source package that will reduce frictions in data sharing for the nanopore community: https://github.com/BonsonW/slow5curl\nSequence-based drug design using transformers\nAuthors: Zhang, S.; Huo, D.; Horne, R. I.; Qi, Y.; Ojeda, S. P.; Yan, A.; Vendruscolo, M.\nScore: 13.6, Published: 2023-11-27 DOI: 10.1101/2023.11.27.568880\nProtein-ligand interactions play central roles in biological processes and are of key importance in drug design. Deep learning-based approaches are emerging as cost-effective alternatives to high-throughput experimental methods for the screening of large libraries of ligands. Here, to predict the binding affinity between proteins and small molecules, we introduce Ligand-Transformer, a deep learning framework based on the AlphaFold2 transformer architecture. We applied Ligand-Transformer to screen inhibitors targeting the mutant EGFRLTC kinase, identifying compounds with low nanomolar potency. We then used this approach to predict the conformational population shifts induced by ABL kinase inhibitors. To show the applicability of Ligand-Transformer to disordered proteins, we explored the binding of small molecules to the Alzheimers A{beta} peptide, identifying compounds that delayed its aggregation. Overall, Ligand-Transformer illustrates the potential of transformers in accurately predicting the interactions of small molecules with both ordered and disordered proteins, thus uncovering molecular mechanisms and facilitating the initial steps in drug discovery.\nBenchmarking active learning protocols for ligand binding affinity prediction\nAuthors: Gorantla, R.; Kubincova, A.; Suutari, B.; Cossins, B. P.; Mey, A.\nScore: 15.7, Published: 2023-11-24 DOI: 10.1101/2023.11.24.568570\nActive learning (AL) has become a powerful tool in computational drug discovery, enabling the identification of top binders from vast molecular libraries with reduced costs for relative binding free energy calculations and experiments. To design a robust AL protocol, it is important to understand the influence of AL parameters, as well as the features of the datasets on the outcomes. We use four affinity datasets for different targets (TYK2, USP7, D2R, Mpro) to systematically evaluate the performance of machine learning models (Gaussian Process model, Chemprop), sample selection protocols, as well as the batch size based on metrics describing the overall predictive power of the model (R2, Spearman rank, RMSE) as well as the accurate identification of top 2% / 5% binders (Recall, F1 score). Both models have a comparable Recall of top binders on large datasets, but the GP models surpass Chemprop when training data is sparse. A larger initial batch size, especially on diverse datasets, increased the Recall of both models as well as overall correlation metrics. However, for subsequent cycles, smaller batch sizes of 20 or 30 compounds proved to be desirable. Furthermore, the presence of Gaussian noise to the data, up to a certain threshold, still allowed the model to identify clusters with top-scoring compounds. However, excessive noise (\u003c1{sigma}) did impact the models predictive and exploitative capabilities. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=111 SRC=\"FIGDIR/small/568570v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (28K): org.highwire.dtl.DTLVardef@cefb11org.highwire.dtl.DTLVardef@c522bcorg.highwire.dtl.DTLVardef@6b8f18org.highwire.dtl.DTLVardef@17f7706_HPS_FORMAT_FIGEXP M_FIG O_FLOATNOTOC GraphicC_FLOATNO C_FIG\npyaging: a Python-based compendium of GPU-optimized aging clocks\nAuthors: de Lima Camillo, L. P.\nScore: 12.0, Published: 2023-11-29 DOI: 10.1101/2023.11.28.569069\nMotivationAging is intricately linked to diseases and mortality and is reflected in molecular changes across various tissues. The development and refinement of biomarkers of aging, healthspan, and lifespan using machine learning models, known as aging clocks, leverage epigenetic and other molecular signatures. Despite advancements, as noted by the Biomarkers of Aging Consortium, the field grapples with challenges, notably the lack of robust software tools for integrating and comparing these diverse models. ResultsI introduce pyaging, a comprehensive Python package, designed to bridge the gap in aging research software tools. pyaging integrates over 30 aging clocks, with plans to expand to more than 100, covering a range of molecular data types including DNA methylation, transcriptomics, histone mark ChIP-Seq, and ATAC-Seq. The package features a variety of model types, from linear and principal component models to neural networks and automatic relevance determination models. Utilizing a PyTorch-based backend for GPU acceleration, pyaging ensures rapid inference even with large datasets and complex models. The package supports multi-species analysis, currently including humans, various mammals, and C. elegans. Availability and Implementationpyaging is accessible at https://github.com/rsinghlab/pyaging. The package is structured to facilitate ease of use and integration into existing research workflows, supporting the flexible anndata data format. Supplementary InformationSupplementary materials, including detailed documentation and usage examples, are available online at the pyaging documentation site (https://pyaging.readthedocs.io/en/latest/index.html).\nAssessing the limits of zero-shot foundation models in single-cell biology\nAuthors: Kedzierska, K. Z.; Crawford, L.; Amini, A. P.; Lu, A. X.\nScore: 88.7, Published: 2023-11-05 DOI: 10.1101/2023.10.16.561085\nThe advent and success of foundation models such as GPT has sparked growing interest in their application to single-cell biology. Models like Geneformer and scGPT have emerged with the promise of serving as versatile tools for this specialized field. However, the efficacy of these models, particularly in zero-shot settings where models are not fine-tuned but used without any further training, remains an open question, especially as practical constraints require useful models to function in settings that preclude fine-tuning (e.g., discovery settings where labels are not fully known). This paper presents a rigorous evaluation of the zero-shot performance of these proposed single-cell foundation models. We assess their utility in tasks such as cell type clustering and batch effect correction, and evaluate the generality of their pretraining objectives. Our results indicate that both Geneformer and scGPT exhibit limited reliability in zero-shot settings and often underperform compared to simpler methods. These findings serve as a cautionary note for the deployment of proposed single-cell foundation models and highlight the need for more focused research to realize their potential.2\n",
  "wordCount" : "2190",
  "inLanguage": "en",
  "datePublished": "2023-12-03T10:38:01Z",
  "dateModified": "2023-12-03T10:38:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/bioinformatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      bioinformatics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 3, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.568764">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.568764" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.568764">
        <p class="paperTitle">Polygraph: A Software Framework for the Systematic Assessment of Synthetic Regulatory DNA Elements</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.568764" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.568764" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gupta, A.; Lal, A.; Gunsalus, L. M.; Biancalani, T.; Eraslan, G.</p>
        <p class="info">Score: 30.9, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.568764' target='https://doi.org/10.1101/2023.11.27.568764'> 10.1101/2023.11.27.568764</a></p>
        <p class="abstract">The design of regulatory elements is pivotal in numerous therapeutic interventions, including gene and cell therapy, wherein the typical objective is to engineer DNA sequences exhibiting specific attributes like cell-type specificity and elevated expression levels. However, the systematic assessment of these constructed DNA sequences remains challenging due to the absence of robust metrics and an integrated software framework. Here, we introduce Polygraph, a Python framework for evaluating synthetic DNA sequences. Polygraph provides a variety of features to streamline the synthesis and scrutiny of regulatory elements, incorporating features like a diversity index, motif and k-mer composition, similarity to endogenous regulatory sequences, and screening with predictive and foundational models. Consequently, Polygraph stands as the first instrument for the assessment of synthetic regulatory sequences, enabling expedited progress in therapeutic applications and improving our understanding of gene regulatory mechanisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.26.568742">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.26.568742" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.26.568742">
        <p class="paperTitle">Learning sequence, structure, and function representations of proteins with language models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.26.568742" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.26.568742" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hamamsy, T.; Barot, M.; Morton, J. T.; Steinegger, M.; Bonneau, R.; Cho, K.</p>
        <p class="info">Score: 26.2, Published: 2023-11-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.26.568742' target='https://doi.org/10.1101/2023.11.26.568742'> 10.1101/2023.11.26.568742</a></p>
        <p class="abstract">The sequence-structure-function relationships that ultimately generate the diversity of extant observed proteins is complex, as proteins bridge the gap between multiple informational and physical scales involved in nearly all cellular processes. One limitation of existing protein annotation databases such as UniProt is that less than 1% of proteins have experimentally verified functions, and computational methods are needed to fill in the missing information. Here, we demonstrate that a multi-aspect framework based on protein language models can learn sequence-structure-function representations of amino acid sequences, and can provide the foundation for sensitive sequence-structure-function aware protein sequence search and annotation. Based on this model, we introduce a multi-aspect information retrieval system for proteins, Protein-Vec, covering sequence, structure, and function aspects, that enables computational protein annotation and function prediction at tree-of-life scales.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.24.568560">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.24.568560" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.24.568560">
        <p class="paperTitle">Nanopore and Illumina Sequencing Reveal Different Viral Populations from Human Gut Samples</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.24.568560" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.24.568560" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cook, R.; Telatin, A.; Hsieh, S.-Y.; Newberry, F.; Tariq, M. A.; Baker, D. J.; Carding, S. R.; Adriaenssens, E. M.</p>
        <p class="info">Score: 30.9, Published: 2023-11-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.24.568560' target='https://doi.org/10.1101/2023.11.24.568560'> 10.1101/2023.11.24.568560</a></p>
        <p class="abstract">The advent of viral metagenomics, or viromics, has improved our knowledge and understanding of global viral diversity. High-throughput sequencing technologies enable explorations of the ecological roles, contributions to host metabolism, and the influence of viruses in various environments including the human gut microbiome. However, the bacterial metagenomic studies frequently have the advantage. The adoption of advanced technologies like long-read sequencing has the potential to be transformative in refining viromics and metagenomics.

Here, we examined the effectiveness of long-read and hybrid sequencing by comparing Illumina short-read and Oxford Nanopore Technology (ONT) long-read sequencing technologies and different assembly strategies on recovering viral genomes from human faecal samples.

Our findings showed that if a single sequencing technology is to be chosen for virome analysis, Illumina was preferable due to its superior ability to recover fully resolved viral genomes and minimise erroneous genomes. While ONT assemblies were effective in recovering viral diversity, the challenges related to input requirements and the necessity for amplification made it less ideal as a standalone solution. However, using a combined, hybrid approach enabled a more authentic representation of viral diversity to be obtained within samples.

Impact StatementViral metagenomics, or viromics, has revolutionised our understanding of global viral diversity however long-read and hybrid approaches are not yet widespread in this field. Here, we compared the performance of Illumina short-read and Nanopore long-read assembly approaches for recovering fully resolved viral genomes from human faecal samples. We highlight Illuminas short-read sequencing for recovering fully resolved viral genomes, while acknowledging Oxford Nanopore Technologys long-read sequencing for capturing broader viral diversity. However, a hybrid approach, utilising both technologies, may mitigate the limitations of one technology alone.

Data SummaryAll reads used in this study are available on European Nucleotide Archive (ENA) within the project PRJEB47625.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.567977">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.567977" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.567977">
        <p class="paperTitle">Colabfold predicts alternative protein structures from single sequences, coevolution unnecessary for AF-cluster</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.567977" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.567977" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Porter, L. L.; Chakravarty, D.; Schafer, J. W.; Chen, E. A.</p>
        <p class="info">Score: 31.6, Published: 2023-11-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.567977' target='https://doi.org/10.1101/2023.11.21.567977'> 10.1101/2023.11.21.567977</a></p>
        <p class="abstract">Though typically associated with a single folded state, globular proteins are dynamic and often assume alternative or transient structures important for their functions1,2. Wayment-Steele, et al. steered ColabFold3 to predict alternative structures of several proteins using a method they call AF-cluster4. They propose that AF-cluster &#34;enables ColabFold to sample alternate states of known metamorphic proteins with high confidence&#34; by first clustering multiple sequence alignments (MSAs) in a way that &#34;deconvolves&#34; coevolutionary information specific to different conformations and then using these clusters as input for ColabFold. Contrary to this Coevolution Assumption, clustered MSAs are not needed to make these predictions. Rather, these alternative structures can be predicted from single sequences and/or sequence similarity, indicating that coevolutionary information is unnecessary for predictive success and may not be used at all. These results suggest that AF-clusters predictive scope is likely limited to sequences with distinct-yet-homologous structures within ColabFolds training set.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.26.568752">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.26.568752" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.26.568752">
        <p class="paperTitle">Giotto Suite: a multi-scale and technology-agnostic spatial multi-omics analysis ecosystem</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.26.568752" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.26.568752" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, J. G.; Chavez-Fuentes, J. C.; O&#39;Brien, M.; Xu, J.; Ruiz, E.; Wang, W.; Amin, I.; Sarfraz, I.; Guckhool, P.; Sistig, A.; Yuan, G.-C.; Dries, R.</p>
        <p class="info">Score: 23.2, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.26.568752' target='https://doi.org/10.1101/2023.11.26.568752'> 10.1101/2023.11.26.568752</a></p>
        <p class="abstract">Emerging spatial omics technologies continue to advance the molecular mapping of tissue architecture and the investigation of gene regulation and cellular crosstalk, which in turn provide new mechanistic insights into a wide range of biological processes and diseases. Such technologies provide an increasingly large amount of information content at multiple spatial scales. However, representing and harmonizing diverse spatial datasets efficiently, including combining multiple modalities or spatial scales in a scalable and flexible manner, remains a substantial challenge. Here, we present Giotto Suite, a suite of open-source software packages that underlies a fully modular and integrated spatial data analysis toolbox. At its core, Giotto Suite is centered around an innovative and technology-agnostic data framework embedded in the R software environment, which allows the representation and integration of virtually any type of spatial omics data at any spatial resolution. In addition, Giotto Suite provides both scalable and extensible end-to-end solutions for data analysis, integration, and visualization. Giotto Suite integrates molecular, morphology, spatial, and annotated feature information to create a responsive and flexible workflow for multi-scale, multi-omic data analyses, as demonstrated here by applications to several state-of-the-art spatial technologies. Furthermore, Giotto Suite builds upon interoperable interfaces and data structures that bridge the established fields of genomics and spatial data science, thereby enabling independent developers to create custom-engineered pipelines. As such, Giotto Suite creates an immersive ecosystem for spatial multi-omic data analysis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.569128">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.569128" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.569128">
        <p class="paperTitle">Streamlining remote nanopore data access with slow5curl</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.569128" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.569128" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wong, B.; Ferguson, J. M.; Gamaarachchi, H.; Deveson, I. W.</p>
        <p class="info">Score: 17.5, Published: 2023-11-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.569128' target='https://doi.org/10.1101/2023.11.28.569128'> 10.1101/2023.11.28.569128</a></p>
        <p class="abstract">As adoption of nanopore sequencing technology continues to advance, the need to maintain large volumes of raw current signal data for reanalysis with updated algorithms is a growing challenge. Here we introduce slow5curl, a software package designed to streamline nanopore data sharing, accessibility and reanalysis. Slow5curl allows a user to fetch a specified read or group of reads from a raw nanopore dataset stored on a remote server, such as a public data repository, without downloading the entire file. Slow5curl uses an index to quickly fetch specific reads from a large dataset in SLOW5/BLOW5 format and highly parallelised data access requests to maximise download speeds. Using all public nanopore data from the Human Pangenome Reference Consortium (&gt;22 TB), we demonstrate how slow5curl can be used to quickly fetch and reanalyse signal reads corresponding to a set of target genes from each individual in large cohort dataset (n = 91), minimising the time, egress costs, and local storage requirements for their reanalysis. We provide slow5curl as a free, open-source package that will reduce frictions in data sharing for the nanopore community: https://github.com/BonsonW/slow5curl</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.568880">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.568880" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.568880">
        <p class="paperTitle">Sequence-based drug design using transformers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.568880" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.568880" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, S.; Huo, D.; Horne, R. I.; Qi, Y.; Ojeda, S. P.; Yan, A.; Vendruscolo, M.</p>
        <p class="info">Score: 13.6, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.568880' target='https://doi.org/10.1101/2023.11.27.568880'> 10.1101/2023.11.27.568880</a></p>
        <p class="abstract">Protein-ligand interactions play central roles in biological processes and are of key importance in drug design. Deep learning-based approaches are emerging as cost-effective alternatives to high-throughput experimental methods for the screening of large libraries of ligands. Here, to predict the binding affinity between proteins and small molecules, we introduce Ligand-Transformer, a deep learning framework based on the AlphaFold2 transformer architecture. We applied Ligand-Transformer to screen inhibitors targeting the mutant EGFRLTC kinase, identifying compounds with low nanomolar potency. We then used this approach to predict the conformational population shifts induced by ABL kinase inhibitors. To show the applicability of Ligand-Transformer to disordered proteins, we explored the binding of small molecules to the Alzheimers A{beta} peptide, identifying compounds that delayed its aggregation. Overall, Ligand-Transformer illustrates the potential of transformers in accurately predicting the interactions of small molecules with both ordered and disordered proteins, thus uncovering molecular mechanisms and facilitating the initial steps in drug discovery.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.24.568570">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.24.568570" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.24.568570">
        <p class="paperTitle">Benchmarking active learning protocols for ligand binding affinity prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.24.568570" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.24.568570" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gorantla, R.; Kubincova, A.; Suutari, B.; Cossins, B. P.; Mey, A.</p>
        <p class="info">Score: 15.7, Published: 2023-11-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.24.568570' target='https://doi.org/10.1101/2023.11.24.568570'> 10.1101/2023.11.24.568570</a></p>
        <p class="abstract">Active learning (AL) has become a powerful tool in computational drug discovery, enabling the identification of top binders from vast molecular libraries with reduced costs for relative binding free energy calculations and experiments. To design a robust AL protocol, it is important to understand the influence of AL parameters, as well as the features of the datasets on the outcomes. We use four affinity datasets for different targets (TYK2, USP7, D2R, Mpro) to systematically evaluate the performance of machine learning models (Gaussian Process model, Chemprop), sample selection protocols, as well as the batch size based on metrics describing the overall predictive power of the model (R2, Spearman rank, RMSE) as well as the accurate identification of top 2% / 5% binders (Recall, F1 score). Both models have a comparable Recall of top binders on large datasets, but the GP models surpass Chemprop when training data is sparse. A larger initial batch size, especially on diverse datasets, increased the Recall of both models as well as overall correlation metrics. However, for subsequent cycles, smaller batch sizes of 20 or 30 compounds proved to be desirable. Furthermore, the presence of Gaussian noise to the data, up to a certain threshold, still allowed the model to identify clusters with top-scoring compounds. However, excessive noise (&lt;1{sigma}) did impact the models predictive and exploitative capabilities.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=111 SRC=&#34;FIGDIR/small/568570v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (28K):
org.highwire.dtl.DTLVardef@cefb11org.highwire.dtl.DTLVardef@c522bcorg.highwire.dtl.DTLVardef@6b8f18org.highwire.dtl.DTLVardef@17f7706_HPS_FORMAT_FIGEXP  M_FIG O_FLOATNOTOC GraphicC_FLOATNO C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.569069">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.569069" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.569069">
        <p class="paperTitle">pyaging: a Python-based compendium of GPU-optimized aging clocks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.569069" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.569069" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: de Lima Camillo, L. P.</p>
        <p class="info">Score: 12.0, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.569069' target='https://doi.org/10.1101/2023.11.28.569069'> 10.1101/2023.11.28.569069</a></p>
        <p class="abstract">MotivationAging is intricately linked to diseases and mortality and is reflected in molecular changes across various tissues. The development and refinement of biomarkers of aging, healthspan, and lifespan using machine learning models, known as aging clocks, leverage epigenetic and other molecular signatures. Despite advancements, as noted by the Biomarkers of Aging Consortium, the field grapples with challenges, notably the lack of robust software tools for integrating and comparing these diverse models.

ResultsI introduce pyaging, a comprehensive Python package, designed to bridge the gap in aging research software tools. pyaging integrates over 30 aging clocks, with plans to expand to more than 100, covering a range of molecular data types including DNA methylation, transcriptomics, histone mark ChIP-Seq, and ATAC-Seq. The package features a variety of model types, from linear and principal component models to neural networks and automatic relevance determination models. Utilizing a PyTorch-based backend for GPU acceleration, pyaging ensures rapid inference even with large datasets and complex models. The package supports multi-species analysis, currently including humans, various mammals, and C. elegans.

Availability and Implementationpyaging is accessible at https://github.com/rsinghlab/pyaging. The package is structured to facilitate ease of use and integration into existing research workflows, supporting the flexible anndata data format.

Supplementary InformationSupplementary materials, including detailed documentation and usage examples, are available online at the pyaging documentation site (https://pyaging.readthedocs.io/en/latest/index.html).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.16.561085">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.16.561085" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.16.561085">
        <p class="paperTitle">Assessing the limits of zero-shot foundation models in single-cell biology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.16.561085" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.16.561085" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kedzierska, K. Z.; Crawford, L.; Amini, A. P.; Lu, A. X.</p>
        <p class="info">Score: 88.7, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.16.561085' target='https://doi.org/10.1101/2023.10.16.561085'> 10.1101/2023.10.16.561085</a></p>
        <p class="abstract">The advent and success of foundation models such as GPT has sparked growing interest in their application to single-cell biology. Models like Geneformer and scGPT have emerged with the promise of serving as versatile tools for this specialized field. However, the efficacy of these models, particularly in zero-shot settings where models are not fine-tuned but used without any further training, remains an open question, especially as practical constraints require useful models to function in settings that preclude fine-tuning (e.g., discovery settings where labels are not fully known). This paper presents a rigorous evaluation of the zero-shot performance of these proposed single-cell foundation models. We assess their utility in tasks such as cell type clustering and batch effect correction, and evaluate the generality of their pretraining objectives. Our results indicate that both Geneformer and scGPT exhibit limited reliability in zero-shot settings and often underperform compared to simpler methods. These findings serve as a cautionary note for the deployment of proposed single-cell foundation models and highlight the need for more focused research to realize their potential.2</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
