<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="AlphaFold2 can predict single-mutation effects
Authors: Mcbride, J. M.; Reinharz, V.; Polev, K.; Grzybowski, B. A.; Tlusty, T.
Score: 108.4, Published: 2023-09-03 DOI: 10.1101/2022.04.14.488301
AlphaFold2 (AF) is a promising tool, but is it accurate enough to predict single mutation effects? Here, we report that the localized structural deformation between protein pairs differing by only 1-3 mutations - as measured by the effective strain - is correlated across 3,901 experimental and AF-predicted structures.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="AlphaFold2 can predict single-mutation effects
Authors: Mcbride, J. M.; Reinharz, V.; Polev, K.; Grzybowski, B. A.; Tlusty, T.
Score: 108.4, Published: 2023-09-03 DOI: 10.1101/2022.04.14.488301
AlphaFold2 (AF) is a promising tool, but is it accurate enough to predict single mutation effects? Here, we report that the localized structural deformation between protein pairs differing by only 1-3 mutations - as measured by the effective strain - is correlated across 3,901 experimental and AF-predicted structures." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-06T10:39:07+00:00" />
<meta property="article:modified_time" content="2023-09-06T10:39:07+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="AlphaFold2 can predict single-mutation effects
Authors: Mcbride, J. M.; Reinharz, V.; Polev, K.; Grzybowski, B. A.; Tlusty, T.
Score: 108.4, Published: 2023-09-03 DOI: 10.1101/2022.04.14.488301
AlphaFold2 (AF) is a promising tool, but is it accurate enough to predict single mutation effects? Here, we report that the localized structural deformation between protein pairs differing by only 1-3 mutations - as measured by the effective strain - is correlated across 3,901 experimental and AF-predicted structures."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "AlphaFold2 can predict single-mutation effects\nAuthors: Mcbride, J. M.; Reinharz, V.; Polev, K.; Grzybowski, B. A.; Tlusty, T.\nScore: 108.4, Published: 2023-09-03 DOI: 10.1101/2022.04.14.488301\nAlphaFold2 (AF) is a promising tool, but is it accurate enough to predict single mutation effects? Here, we report that the localized structural deformation between protein pairs differing by only 1-3 mutations - as measured by the effective strain - is correlated across 3,901 experimental and AF-predicted structures.",
  "keywords": [
    
  ],
  "articleBody": " AlphaFold2 can predict single-mutation effects\nAuthors: Mcbride, J. M.; Reinharz, V.; Polev, K.; Grzybowski, B. A.; Tlusty, T.\nScore: 108.4, Published: 2023-09-03 DOI: 10.1101/2022.04.14.488301\nAlphaFold2 (AF) is a promising tool, but is it accurate enough to predict single mutation effects? Here, we report that the localized structural deformation between protein pairs differing by only 1-3 mutations - as measured by the effective strain - is correlated across 3,901 experimental and AF-predicted structures. Furthermore, analysis of [~]11,000 proteins shows that the local structural change correlates with various phenotypic changes. These findings suggest that AF can predict the range and magnitude of single-mutation effects on average, and we propose a method to improve the precision of AF predictions and to indicate when predictions are unreliable.\nThe resting state of the human T-cell receptor\nAuthors: Notti, R. Q.; Yi, F.; Heissel, S.; Molina, H.; Klebanoff, C. A.; Walz, T.\nScore: 35.5, Published: 2023-08-22 DOI: 10.1101/2023.08.22.554360\nThe T-cell receptor (TCR) is central to the ligand-dependent activation of T lymphocytes and as such orchestrates both adaptive and pathologic immune processes1. However, major questions remain regarding the structure and function of the human TCR2-4. Here, we present cryogenic electron microscopy structures for the unliganded human TCR in a native-like lipid bilayer, revealing three related conformations that are distinct from previously reported structures of receptors in detergent. These new \"closed and compacted\" conformations afford insights into the interactions between the TCR and the membrane, including conserved surface patches that make extensive outer leaflet contact, and suggest novel conformational regulation by glycans. We show that the closed/compacted conformations, not the extended one previously reported in detergent5-8, represent the unliganded resting state for the TCR in vivo, underscoring the importance of structural interrogation of membrane proteins in native-like environments. We use conformation-locking disulfide mutants to show that juxtamembrane linker extension is necessary for ligand-dependent TCR activation, demonstrating that TCR-intrinsic conformational change is necessary for TCR activation and opening numerous avenues for immunoreceptor engineering.\nSCP Viz - A universal graphical user interface for single protein analysis in single cell proteomics datasets\nAuthors: Warshanna, A.; Orsburn, B.\nScore: 12.8, Published: 2023-08-29 DOI: 10.1101/2023.08.29.555397\nSingle cell proteomics (SCP) requires the analysis of dozens to thousands of single human cells to draw biological conclusions. However, assessing of the abundance of single proteins in output data presents a considerable challenge, and no simple universal solutions currently exist. To address this, we developed SCP Viz, a statistical package with a graphical user interface that can handle small and large scale SCP output from any instrument or data processing software. In this software, the abundance of individual proteins can be plotted in a variety of ways, using either unadjusted or normalized outputs. These outputs can also be transformed or imputed within the software. SCP Viz offers a variety of plotting options which can help identify significantly altered proteins between groups, both before and after quantitative transformations. Upon the discovery of subpopulations of single cells, users can easily regroup the cells of interest using straightforward text-based filters. When used in this way, SCP Viz allows users to visualize proteomic heterogeneity at the level of individual proteins, cells, or identified subcellular populations. SCP Viz is compatible with output files from MaxQuant, FragPipe, SpectroNaut, and Proteome Discoverer, and should work equally well with other formats. SCP Viz is publicly available at https://github.com/orsburn/SCPViz. For demonstrations, users can download our test data from GitHub and use an online version that accepts user input for analysis at https://orsburnlab.shinyapps.io/SCPViz/. Abstract graphic O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=167 SRC=\"FIGDIR/small/555397v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (27K): org.highwire.dtl.DTLVardef@76a171org.highwire.dtl.DTLVardef@6122c7org.highwire.dtl.DTLVardef@1324d94org.highwire.dtl.DTLVardef@1833e85_HPS_FORMAT_FIGEXP M_FIG C_FIG\nProtein Coronas on Functionalized Nanoparticles Enable Quantitative and Precise Large-Scale Deep Plasma Proteomics\nAuthors: Huang, T.; Wang, J.; Stukalov, A.; Donovan, M. K. R.; Ferdosi, S.; Williamson, L.; Just, S.; Castro, G.; Cantrell, L. S.; Elgierari, E.; Benz, R. W.; Huang, Y.; Motamedchaboki, K.; Hakimi, A.; Arrey, T.; Damoc, E.; Kreimer, S.; Farokhzad, O. C.; Batzoglou, S.; Siddiqui, A.; Van Eyk, J. E.; Hornburg, D.\nScore: 11.9, Published: 2023-08-29 DOI: 10.1101/2023.08.28.555225\nBackgroundThe wide dynamic range of circulating proteins coupled with the diversity of proteoforms present in plasma has historically impeded comprehensive and quantitative characterization of the plasma proteome at scale. Automated nanoparticle (NP) protein corona-based proteomics workflows can efficiently compress the dynamic range of protein abundances into a mass spectrometry (MS)-accessible detection range. This enhances the depth and scalability of quantitative MS-based methods, which can elucidate the molecular mechanisms of biological processes, discover new protein biomarkers, and improve comprehensiveness of MS-based diagnostics. MethodsInvestigating multi-species spike-in experiments and a cohort, we investigated fold-change accuracy, linearity, precision, and statistical power for the using the Proteograph Product Suite, a deep plasma proteomics workflow, in conjunction with multiple MS instruments. ResultsWe show that NP-based workflows enable accurate identification (false discovery rate of 1%) of more than 6,000 proteins from plasma (Orbitrap Astral) and, compared to a gold standard neat plasma workflow that is limited to the detection of hundreds of plasma proteins, facilitate quantification of more proteins with accurate fold-changes, high linearity, and precision. Furthermore, we demonstrate high statistical power for the discovery of biomarkers in small- and large-scale cohorts. ConclusionsThe automated NP workflow enables high-throughput, deep, and quantitative plasma proteomics investigation with sufficient power to discover new biomarker signatures with a peptide level resolution.\nRosetta Energy Analysis of AlphaFold2 models: Point Mutations and Conformational Ensembles\nAuthors: Stein, R. A.; Mchaourab, H. S.\nScore: 9.6, Published: 2023-09-05 DOI: 10.1101/2023.09.05.556364\nAlphaFold2's ability to accurately predict protein structures from a multiple sequence alignment (MSA) has raised many questions about the utility of the models generated in downstream structural analysis. Two outstanding questions are the prediction of the consequences of point mutations and the completeness of the landscape of protein conformational ensembles. We previously developed a method, SPEACH_AF, to obtain alternate conformations by introducing residue substitutions across the MSA and not just within the input sequence. Here, we compared the structural and energetic consequences of having the mutation(s) in the input sequence versus in the whole MSA (SPEACH_AF). Both methods yielded models different from the wild-type sequence, with more robust changes when the mutation(s) were in the whole MSA. To evaluate models of conformational diversity, we used SPEACH_AF and a new MSA subsampling method, AF_cluster, combined with model relaxation in Rosetta. We find that the energetics of the conformations generated by AlphaFold2 correspond to those seen in experimental crystal structures and explored by standard molecular dynamic methods. Combined, the results support the fact that AlphaFold2 can predict structural changes due to point mutations and has learned information about protein structural energetics that are encoded in the MSA.\nPigment Binding in The Light-Dependent Protochlorophyllide Oxidoreductase\nAuthors: Pesara, P.; Szafran, K.; Nguyen, H. C.; Sirohiwal, A.; Pantazis, D. A.; Gabruk, M.\nScore: 9.2, Published: 2023-08-29 DOI: 10.1101/2023.08.28.555081\nThe Light-Dependent Protochlorophyllide Oxidoreductase (LPOR) is a key enzyme in chlorophyll biosynthesis and its photocatalytic mechanism has long intrigued researchers. However, the lack of structural data for the active complex has impeded understanding of its reaction mechanism. A recent high-resolution structure of enzyme in the active conformation has established a robust foundation for validating hypotheses concerning pigment binding, residue involvement, and consequently, the reaction mechanism. Surprisingly, this new structure challenges previously proposed mechanisms, especially concerning the orientation of the bound protochlorophyllide (Pchlide) pigment. In this study, we employ molecular dynamics and hybrid quantum-mechanics/molecular-mechanics (QM/MM) simulations along with site-directed mutagenesis to compare two Pchlide binding modes: one aligned with previous proposals (mode A), and the other consistent with the recent experimental data (mode B). Binding energy calculations reveal thermodynamic instability of binding mode A due to nonspecific interactions, while mode B exhibits distinct stabilizing interactions yielding favorable binding. QM/MM-based local energy decomposition analysis unravels a complex interaction network that reinforces pigment stabilization in this conformation. Notably, interactions involving Tyr177, His319, and the carboxyl group at C131 influence the pigments excited state energy and potentially contributing to the substrate specificity of the enzyme. Our results uniformly favor binding mode B as represented in the new cryo-EM structure, over the previously assumed mode A. These findings challenge established interpretations and underscore the need for a comprehensive re-evaluation of the reaction mechanism of LPOR that correctly considers pigment interactions and substrate orientation in the binding pocket. Significance StatementA crucial step in the biosynthesis of the all-important photosynthetic pigment chlorophyll is the reduction of a double C=C bond in its precursor protochlorophyllide (PChlide). This is catalyzed by the Light-Dependent Protochlorophyllide Oxidoreductase (LPOR) via an extremely rare example of a biological photocatalytic reaction. Understanding of the LPOR mechanism has been hindered by limited insight into the structure of its active complex. A recent high-resolution LPOR cryo-EM structure substantiates pigment binding, residue interactions, and the reaction mechanism, but contrasts markedly with all previous assumptions regarding the binding mode of the substrate PChlide. Using molecular dynamics simulations, quantum-mechanics/molecular-mechanics calculations, and mutagenesis, we compare and evaluate the two possible Pchlide binding modes, the one assumed previously (mode A) and the one supported by recent data (mode B). Our findings conclusively favor mode B, challenging prior assumptions and pointing toward novel mechanistic possibilities for this unique photocatalytic reaction.\nA ubiquitin-specific, proximity-based labeling approach for the identification of ubiquitin ligase substrates\nAuthors: Bhogaraju, S.; Behrends, C.; Mukhopadhyay, U.; Levantovsky, S.; Stein, F.; Gharbi, S.\nScore: 8.1, Published: 2023-09-04 DOI: 10.1101/2023.09.04.556194\nUbiquitination of proteins is central to protein homeostasis and other cellular processes including DNA repair, vesicular transport, cell-division etc. The process of ubiquitination is conserved from yeast to humans and is carried out by the sequential action of three enzymes: E1, E2 and E3. There are an estimated \u003e600 E3 ligases in humans that execute ubiquitination of specific target proteins in a spatio-temporal manner to elicit desired signaling effects. Here, we developed a simple, proximity-based labeling method to selectively biotinylate substrates of a given ubiquitin ligase. Our method exploits the proximity and the relative orientation of the E3-ligase catalytic domain with respect to ubiquitin observed in the enzymatic intermediate-state structures of E3-E2~Ub. By fusing the biotin ligase BirA and an Avi tag variant to the candidate E3 ligase and ubiquitin, respectively, we were able to specifically enrich the bona fide substrates and potential new substrates of the ligase using a one-step Streptavidin pulldown under denaturing conditions. As proof-of-principle, we applied our method, which we named Ub-POD, to the RING E3 ligase Rad18. Rad18 ubiquitinates DNA-sliding clamp PCNA upon UV-induced DNA damage. We identified PCNA and several other critical players in the DNA damage repair pathway in a single Ub-POD experiment. We further discovered an unintended but important application of Ub-POD, where we were able to pin down the cellular localization of Rad18-mediated ubiquitination to the damaged DNA nuclear puncta through Streptavidin immunofluorescence. We also applied our method to TRAF6, another RING ubiquitin ligase involved in TNF signaling, successfully identifying known and potentially new substrates. Finally, we adapted our method to the U-box-type E3 ubiquitin ligase CHIP to demonstrate that we can identify substrates of two major classes of mammalian ubiquitin ligases. We anticipate that our method and principle could be widely adapted to all classes of ubiquitin ligases to identify substrates and localize the cellular site(s) of ubiquitination.\nSLP2 coordinates MICOS assembly and cristae morphogenesis via MIC13 and YME1L\nAuthors: Naha, R.; Strohm, R.; Urbach, J.; Wittig, I.; Reichert, A. S.; Kondadi, A. K.; Anand, R.\nScore: 7.6, Published: 2023-09-04 DOI: 10.1101/2023.09.04.556207\nThe MICOS complex subunit MIC13 is essential for mitochondrial cristae organization. Mutations in MIC13 cause severe mitochondrial hepato-encephalopathy displaying defective cristae morphology and loss of the MIC10-subcomplex. Here we identified SLP2 as a novel interacting partner of MIC13 and decipher a critical role of SLP2 for MICOS assembly at distinct steps. SLP2 provides a large interaction hub for MICOS subunits and loss of SLP2 imparted YME1L-mediated proteolysis of MIC26 and drastic alterations in cristae morphology. We further identified a MIC13-specific role in stabilizing the MIC10-subcomplex via a MIC13-YME1L axis. SLP2 together with the stabilized MIC10-subcomplex promotes efficient assembly of the MIC60-subcomplex forming the MICOS-MIB complex. Consistently, super-resolution nanoscopy showed a dispersed distribution of the MIC60 in cells lacking SLP2 and MIC13. Our study reveals converging and interdependent assembly pathways for the MIC10- and MIC60-subcomplexes which are controlled in two ways, the MIC13-YME1L and the SLP2-YME1L axes, revealing mechanistic insights of these factors in cristae morphogenesis. These results will be helpful in understanding the human pathophysiology linked to mutations in MIC13 or its interaction partners.\nAutomating data analysis for hydrogen/deuterium exchange mass spectrometry using data-independent acquisition methodology\nAuthors: Filandr, F.; Sarpe, V.; Raval, S.; Crowder, D. A.; Khan, M. F.; Douglas, P.; Coales, S.; Viner, R.; Syed, A.; Tainer, J. A.; Lees-MIller, S. P.; Schriemer, D. C.\nScore: 9.2, Published: 2023-08-26 DOI: 10.1101/2023.08.25.554852\nWe developed a hydrogen/deuterium exchange workflow coupled to tandem mass spectrometry (HX-MS2) that supports the acquisition of peptide fragment ions alongside their peptide precursors. The approach enables true auto-validation of HX data by mining a rich set of deuterated fragments, generated by collisional-induced dissociation (CID), to simultaneously confirm the peptide ID and authenticate MS1-based deuteration calculations. The high redundancy provided by the fragments supports a confidence assessment of deuterium calculations using a combinatorial strategy. The approach requires data-independent acquisition (DIA) methods that are available on most MS platforms, making the switch to HX-MS2 straightforward. Importantly, we find that HX-DIA enables a proteomics-grade approach and wide-spread applications. Considerable time is saved through auto-validation and complex samples can now be characterized and at higher throughput. We illustrate these advantages in a drug binding analysis of the ultra-large protein kinase DNA-PKcs, isolated directly from mammalian cells.\nDiscovery of 2-amide-3-methylester thiophenes inhibiting SARS-CoV-2 ADP-ribosyl hydrolysing macrodomain and coronavirus replication\nAuthors: Wazir, S.; Parviainen, T. A. O.; Maksimainen, M. M.; Duong, M. T. H.; Pfannenstiel, J. J.; Cluff, D.; Sowa, S. T.; Galera-Prat, A.; Ferraris, D. V.; Fehr, A.; Heiskanen, J. P.; LehtioÌˆ, L.\nScore: 7.0, Published: 2023-08-29 DOI: 10.1101/2023.08.28.555062\nThe COVID-19 pandemic caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) virus has made it clear that further development of antiviral therapies will be needed to combat additional SARS-CoV-2 variants or novel CoVs. Here, we describe small molecule inhibitors for SARS-CoV-2 Mac1, which counters ADP-ribosylation mediated innate immune responses. The compounds inhibiting Mac1 were discovered through high-throughput screening (HTS) using a protein FRET-based competition assay and the best hit compound had an IC50 of 14 {micro}M. Three validated HTS hits have the same 2-amide-3-methylester thiophene scaffold and the scaffold was selected for structure-activity relationship (SAR) studies through commercial and synthesized analogs. We studied the compound binding mode in detail using X-ray crystallography and this allowed us to focus on specific features of the compound and design analogs. Compound 27 (MDOLL-0229) had an IC50 of 2.1 {micro}M and was generally selective for CoV Mac1 proteins after profiling for activity against a panel of viral and human ADP-ribose binding proteins. The improved potency allowed testing of its effect on virus replication and indeed, 27 inhibited replication of a mouse hepatitis virus, a prototype CoV. Compound 27 is the first Mac1 targeted small molecule demonstrated to inhibit coronavirus replication in a cell model. This, together with its well-defined binding mode, makes 27 a good candidate for further hit/lead-optimization efforts.\n",
  "wordCount" : "2515",
  "inLanguage": "en",
  "datePublished": "2023-09-06T10:39:07Z",
  "dateModified": "2023-09-06T10:39:07Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta"><span>updated on September 6, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.04.14.488301">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.04.14.488301" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.04.14.488301">
        <p class="paperTitle">AlphaFold2 can predict single-mutation effects</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.04.14.488301" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.04.14.488301" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mcbride, J. M.; Reinharz, V.; Polev, K.; Grzybowski, B. A.; Tlusty, T.</p>
        <p class="info">Score: 108.4, Published: 2023-09-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.04.14.488301' target='https://doi.org/10.1101/2022.04.14.488301'> 10.1101/2022.04.14.488301</a></p>
        <p class="abstract">AlphaFold2 (AF) is a promising tool, but is it accurate enough to predict single mutation effects? Here, we report that the localized structural deformation between protein pairs differing by only 1-3 mutations - as measured by the effective strain - is correlated across 3,901 experimental and AF-predicted structures. Furthermore, analysis of [~]11,000 proteins shows that the local structural change correlates with various phenotypic changes. These findings suggest that AF can predict the range and magnitude of single-mutation effects on average, and we propose a method to improve the precision of AF predictions and to indicate when predictions are unreliable.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.554360">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.554360" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.554360">
        <p class="paperTitle">The resting state of the human T-cell receptor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.554360" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.554360" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Notti, R. Q.; Yi, F.; Heissel, S.; Molina, H.; Klebanoff, C. A.; Walz, T.</p>
        <p class="info">Score: 35.5, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.554360' target='https://doi.org/10.1101/2023.08.22.554360'> 10.1101/2023.08.22.554360</a></p>
        <p class="abstract">The T-cell receptor (TCR) is central to the ligand-dependent activation of T lymphocytes and as such orchestrates both adaptive and pathologic immune processes1. However, major questions remain regarding the structure and function of the human TCR2-4. Here, we present cryogenic electron microscopy structures for the unliganded human TCR in a native-like lipid bilayer, revealing three related conformations that are distinct from previously reported structures of receptors in detergent. These new &#34;closed and compacted&#34; conformations afford insights into the interactions between the TCR and the membrane, including conserved surface patches that make extensive outer leaflet contact, and suggest novel conformational regulation by glycans. We show that the closed/compacted conformations, not the extended one previously reported in detergent5-8, represent the unliganded resting state for the TCR in vivo, underscoring the importance of structural interrogation of membrane proteins in native-like environments. We use conformation-locking disulfide mutants to show that juxtamembrane linker extension is necessary for ligand-dependent TCR activation, demonstrating that TCR-intrinsic conformational change is necessary for TCR activation and opening numerous avenues for immunoreceptor engineering.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.555397">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.555397" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.555397">
        <p class="paperTitle">SCP Viz - A universal graphical user interface for single protein analysis in single cell proteomics datasets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.555397" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.555397" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Warshanna, A.; Orsburn, B.</p>
        <p class="info">Score: 12.8, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.555397' target='https://doi.org/10.1101/2023.08.29.555397'> 10.1101/2023.08.29.555397</a></p>
        <p class="abstract">Single cell proteomics (SCP) requires the analysis of dozens to thousands of single human cells to draw biological conclusions. However, assessing of the abundance of single proteins in output data presents a considerable challenge, and no simple universal solutions currently exist. To address this, we developed SCP Viz, a statistical package with a graphical user interface that can handle small and large scale SCP output from any instrument or data processing software. In this software, the abundance of individual proteins can be plotted in a variety of ways, using either unadjusted or normalized outputs. These outputs can also be transformed or imputed within the software. SCP Viz offers a variety of plotting options which can help identify significantly altered proteins between groups, both before and after quantitative transformations. Upon the discovery of subpopulations of single cells, users can easily regroup the cells of interest using straightforward text-based filters. When used in this way, SCP Viz allows users to visualize proteomic heterogeneity at the level of individual proteins, cells, or identified subcellular populations. SCP Viz is compatible with output files from MaxQuant, FragPipe, SpectroNaut, and Proteome Discoverer, and should work equally well with other formats. SCP Viz is publicly available at https://github.com/orsburn/SCPViz. For demonstrations, users can download our test data from GitHub and use an online version that accepts user input for analysis at https://orsburnlab.shinyapps.io/SCPViz/.

Abstract graphic

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=167 SRC=&#34;FIGDIR/small/555397v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (27K):
org.highwire.dtl.DTLVardef@76a171org.highwire.dtl.DTLVardef@6122c7org.highwire.dtl.DTLVardef@1324d94org.highwire.dtl.DTLVardef@1833e85_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555225">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555225" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555225">
        <p class="paperTitle">Protein Coronas on Functionalized Nanoparticles Enable Quantitative and Precise Large-Scale Deep Plasma Proteomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555225" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555225" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Huang, T.; Wang, J.; Stukalov, A.; Donovan, M. K. R.; Ferdosi, S.; Williamson, L.; Just, S.; Castro, G.; Cantrell, L. S.; Elgierari, E.; Benz, R. W.; Huang, Y.; Motamedchaboki, K.; Hakimi, A.; Arrey, T.; Damoc, E.; Kreimer, S.; Farokhzad, O. C.; Batzoglou, S.; Siddiqui, A.; Van Eyk, J. E.; Hornburg, D.</p>
        <p class="info">Score: 11.9, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555225' target='https://doi.org/10.1101/2023.08.28.555225'> 10.1101/2023.08.28.555225</a></p>
        <p class="abstract">BackgroundThe wide dynamic range of circulating proteins coupled with the diversity of proteoforms present in plasma has historically impeded comprehensive and quantitative characterization of the plasma proteome at scale. Automated nanoparticle (NP) protein corona-based proteomics workflows can efficiently compress the dynamic range of protein abundances into a mass spectrometry (MS)-accessible detection range. This enhances the depth and scalability of quantitative MS-based methods, which can elucidate the molecular mechanisms of biological processes, discover new protein biomarkers, and improve comprehensiveness of MS-based diagnostics.

MethodsInvestigating multi-species spike-in experiments and a cohort, we investigated fold-change accuracy, linearity, precision, and statistical power for the using the Proteograph Product Suite, a deep plasma proteomics workflow, in conjunction with multiple MS instruments.

ResultsWe show that NP-based workflows enable accurate identification (false discovery rate of 1%) of more than 6,000 proteins from plasma (Orbitrap Astral) and, compared to a gold standard neat plasma workflow that is limited to the detection of hundreds of plasma proteins, facilitate quantification of more proteins with accurate fold-changes, high linearity, and precision. Furthermore, we demonstrate high statistical power for the discovery of biomarkers in small- and large-scale cohorts.

ConclusionsThe automated NP workflow enables high-throughput, deep, and quantitative plasma proteomics investigation with sufficient power to discover new biomarker signatures with a peptide level resolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.05.556364">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.05.556364" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.05.556364">
        <p class="paperTitle">Rosetta Energy Analysis of AlphaFold2 models: Point Mutations and Conformational Ensembles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.05.556364" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.05.556364" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stein, R. A.; Mchaourab, H. S.</p>
        <p class="info">Score: 9.6, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.05.556364' target='https://doi.org/10.1101/2023.09.05.556364'> 10.1101/2023.09.05.556364</a></p>
        <p class="abstract">AlphaFold2&#39;s ability to accurately predict protein structures from a multiple sequence alignment (MSA) has raised many questions about the utility of the models generated in downstream structural analysis. Two outstanding questions are the prediction of the consequences of point mutations and the completeness of the landscape of protein conformational ensembles. We previously developed a method, SPEACH_AF, to obtain alternate conformations by introducing residue substitutions across the MSA and not just within the input sequence. Here, we compared the structural and energetic consequences of having the mutation(s) in the input sequence versus in the whole MSA (SPEACH_AF). Both methods yielded models different from the wild-type sequence, with more robust changes when the mutation(s) were in the whole MSA. To evaluate models of conformational diversity, we used SPEACH_AF and a new MSA subsampling method, AF_cluster, combined with model relaxation in Rosetta. We find that the energetics of the conformations generated by AlphaFold2 correspond to those seen in experimental crystal structures and explored by standard molecular dynamic methods. Combined, the results support the fact that AlphaFold2 can predict structural changes due to point mutations and has learned information about protein structural energetics that are encoded in the MSA.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555081">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555081" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555081">
        <p class="paperTitle">Pigment Binding in The Light-Dependent Protochlorophyllide Oxidoreductase</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555081" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555081" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pesara, P.; Szafran, K.; Nguyen, H. C.; Sirohiwal, A.; Pantazis, D. A.; Gabruk, M.</p>
        <p class="info">Score: 9.2, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555081' target='https://doi.org/10.1101/2023.08.28.555081'> 10.1101/2023.08.28.555081</a></p>
        <p class="abstract">The Light-Dependent Protochlorophyllide Oxidoreductase (LPOR) is a key enzyme in chlorophyll biosynthesis and its photocatalytic mechanism has long intrigued researchers. However, the lack of structural data for the active complex has impeded understanding of its reaction mechanism. A recent high-resolution structure of enzyme in the active conformation has established a robust foundation for validating hypotheses concerning pigment binding, residue involvement, and consequently, the reaction mechanism. Surprisingly, this new structure challenges previously proposed mechanisms, especially concerning the orientation of the bound protochlorophyllide (Pchlide) pigment. In this study, we employ molecular dynamics and hybrid quantum-mechanics/molecular-mechanics (QM/MM) simulations along with site-directed mutagenesis to compare two Pchlide binding modes: one aligned with previous proposals (mode A), and the other consistent with the recent experimental data (mode B). Binding energy calculations reveal thermodynamic instability of binding mode A due to nonspecific interactions, while mode B exhibits distinct stabilizing interactions yielding favorable binding. QM/MM-based local energy decomposition analysis unravels a complex interaction network that reinforces pigment stabilization in this conformation. Notably, interactions involving Tyr177, His319, and the carboxyl group at C131 influence the pigments excited state energy and potentially contributing to the substrate specificity of the enzyme. Our results uniformly favor binding mode B as represented in the new cryo-EM structure, over the previously assumed mode A. These findings challenge established interpretations and underscore the need for a comprehensive re-evaluation of the reaction mechanism of LPOR that correctly considers pigment interactions and substrate orientation in the binding pocket.

Significance StatementA crucial step in the biosynthesis of the all-important photosynthetic pigment chlorophyll is the reduction of a double C=C bond in its precursor protochlorophyllide (PChlide). This is catalyzed by the Light-Dependent Protochlorophyllide Oxidoreductase (LPOR) via an extremely rare example of a biological photocatalytic reaction. Understanding of the LPOR mechanism has been hindered by limited insight into the structure of its active complex. A recent high-resolution LPOR cryo-EM structure substantiates pigment binding, residue interactions, and the reaction mechanism, but contrasts markedly with all previous assumptions regarding the binding mode of the substrate PChlide. Using molecular dynamics simulations, quantum-mechanics/molecular-mechanics calculations, and mutagenesis, we compare and evaluate the two possible Pchlide binding modes, the one assumed previously (mode A) and the one supported by recent data (mode B). Our findings conclusively favor mode B, challenging prior assumptions and pointing toward novel mechanistic possibilities for this unique photocatalytic reaction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.04.556194">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.04.556194" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.04.556194">
        <p class="paperTitle">A ubiquitin-specific, proximity-based labeling approach for the identification of ubiquitin ligase substrates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.04.556194" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.04.556194" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bhogaraju, S.; Behrends, C.; Mukhopadhyay, U.; Levantovsky, S.; Stein, F.; Gharbi, S.</p>
        <p class="info">Score: 8.1, Published: 2023-09-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.04.556194' target='https://doi.org/10.1101/2023.09.04.556194'> 10.1101/2023.09.04.556194</a></p>
        <p class="abstract">Ubiquitination of proteins is central to protein homeostasis and other cellular processes including DNA repair, vesicular transport, cell-division etc. The process of ubiquitination is conserved from yeast to humans and is carried out by the sequential action of three enzymes: E1, E2 and E3. There are an estimated &gt;600 E3 ligases in humans that execute ubiquitination of specific target proteins in a spatio-temporal manner to elicit desired signaling effects. Here, we developed a simple, proximity-based labeling method to selectively biotinylate substrates of a given ubiquitin ligase. Our method exploits the proximity and the relative orientation of the E3-ligase catalytic domain with respect to ubiquitin observed in the enzymatic intermediate-state structures of E3-E2~Ub. By fusing the biotin ligase BirA and an Avi tag variant to the candidate E3 ligase and ubiquitin, respectively, we were able to specifically enrich the bona fide substrates and potential new substrates of the ligase using a one-step Streptavidin pulldown under denaturing conditions. As proof-of-principle, we applied our method, which we named Ub-POD, to the RING E3 ligase Rad18. Rad18 ubiquitinates DNA-sliding clamp PCNA upon UV-induced DNA damage. We identified PCNA and several other critical players in the DNA damage repair pathway in a single Ub-POD experiment. We further discovered an unintended but important application of Ub-POD, where we were able to pin down the cellular localization of Rad18-mediated ubiquitination to the damaged DNA nuclear puncta through Streptavidin immunofluorescence. We also applied our method to TRAF6, another RING ubiquitin ligase involved in TNF signaling, successfully identifying known and potentially new substrates. Finally, we adapted our method to the U-box-type E3 ubiquitin ligase CHIP to demonstrate that we can identify substrates of two major classes of mammalian ubiquitin ligases. We anticipate that our method and principle could be widely adapted to all classes of ubiquitin ligases to identify substrates and localize the cellular site(s) of ubiquitination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.04.556207">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.04.556207" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.04.556207">
        <p class="paperTitle">SLP2 coordinates MICOS assembly and cristae morphogenesis via MIC13 and YME1L</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.04.556207" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.04.556207" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Naha, R.; Strohm, R.; Urbach, J.; Wittig, I.; Reichert, A. S.; Kondadi, A. K.; Anand, R.</p>
        <p class="info">Score: 7.6, Published: 2023-09-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.04.556207' target='https://doi.org/10.1101/2023.09.04.556207'> 10.1101/2023.09.04.556207</a></p>
        <p class="abstract">The MICOS complex subunit MIC13 is essential for mitochondrial cristae organization. Mutations in MIC13 cause severe mitochondrial hepato-encephalopathy displaying defective cristae morphology and loss of the MIC10-subcomplex. Here we identified SLP2 as a novel interacting partner of MIC13 and decipher a critical role of SLP2 for MICOS assembly at distinct steps. SLP2 provides a large interaction hub for MICOS subunits and loss of SLP2 imparted YME1L-mediated proteolysis of MIC26 and drastic alterations in cristae morphology. We further identified a MIC13-specific role in stabilizing the MIC10-subcomplex via a MIC13-YME1L axis. SLP2 together with the stabilized MIC10-subcomplex promotes efficient assembly of the MIC60-subcomplex forming the MICOS-MIB complex. Consistently, super-resolution nanoscopy showed a dispersed distribution of the MIC60 in cells lacking SLP2 and MIC13. Our study reveals converging and interdependent assembly pathways for the MIC10- and MIC60-subcomplexes which are controlled in two ways, the MIC13-YME1L and the SLP2-YME1L axes, revealing mechanistic insights of these factors in cristae morphogenesis. These results will be helpful in understanding the human pathophysiology linked to mutations in MIC13 or its interaction partners.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.554852">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.554852" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.554852">
        <p class="paperTitle">Automating data analysis for hydrogen/deuterium exchange mass spectrometry using data-independent acquisition methodology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.554852" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.554852" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Filandr, F.; Sarpe, V.; Raval, S.; Crowder, D. A.; Khan, M. F.; Douglas, P.; Coales, S.; Viner, R.; Syed, A.; Tainer, J. A.; Lees-MIller, S. P.; Schriemer, D. C.</p>
        <p class="info">Score: 9.2, Published: 2023-08-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.554852' target='https://doi.org/10.1101/2023.08.25.554852'> 10.1101/2023.08.25.554852</a></p>
        <p class="abstract">We developed a hydrogen/deuterium exchange workflow coupled to tandem mass spectrometry (HX-MS2) that supports the acquisition of peptide fragment ions alongside their peptide precursors. The approach enables true auto-validation of HX data by mining a rich set of deuterated fragments, generated by collisional-induced dissociation (CID), to simultaneously confirm the peptide ID and authenticate MS1-based deuteration calculations. The high redundancy provided by the fragments supports a confidence assessment of deuterium calculations using a combinatorial strategy. The approach requires data-independent acquisition (DIA) methods that are available on most MS platforms, making the switch to HX-MS2 straightforward. Importantly, we find that HX-DIA enables a proteomics-grade approach and wide-spread applications. Considerable time is saved through auto-validation and complex samples can now be characterized and at higher throughput. We illustrate these advantages in a drug binding analysis of the ultra-large protein kinase DNA-PKcs, isolated directly from mammalian cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555062">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555062" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555062">
        <p class="paperTitle">Discovery of 2-amide-3-methylester thiophenes inhibiting SARS-CoV-2 ADP-ribosyl hydrolysing macrodomain and coronavirus replication</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555062" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555062" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wazir, S.; Parviainen, T. A. O.; Maksimainen, M. M.; Duong, M. T. H.; Pfannenstiel, J. J.; Cluff, D.; Sowa, S. T.; Galera-Prat, A.; Ferraris, D. V.; Fehr, A.; Heiskanen, J. P.; LehtioÌˆ, L.</p>
        <p class="info">Score: 7.0, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555062' target='https://doi.org/10.1101/2023.08.28.555062'> 10.1101/2023.08.28.555062</a></p>
        <p class="abstract">The COVID-19 pandemic caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) virus has made it clear that further development of antiviral therapies will be needed to combat additional SARS-CoV-2 variants or novel CoVs. Here, we describe small molecule inhibitors for SARS-CoV-2 Mac1, which counters ADP-ribosylation mediated innate immune responses. The compounds inhibiting Mac1 were discovered through high-throughput screening (HTS) using a protein FRET-based competition assay and the best hit compound had an IC50 of 14 {micro}M. Three validated HTS hits have the same 2-amide-3-methylester thiophene scaffold and the scaffold was selected for structure-activity relationship (SAR) studies through commercial and synthesized analogs. We studied the compound binding mode in detail using X-ray crystallography and this allowed us to focus on specific features of the compound and design analogs. Compound 27 (MDOLL-0229) had an IC50 of 2.1 {micro}M and was generally selective for CoV Mac1 proteins after profiling for activity against a panel of viral and human ADP-ribose binding proteins. The improved potency allowed testing of its effect on virus replication and indeed, 27 inhibited replication of a mouse hepatitis virus, a prototype CoV. Compound 27 is the first Mac1 targeted small molecule demonstrated to inhibit coronavirus replication in a cell model. This, together with its well-defined binding mode, makes 27 a good candidate for further hit/lead-optimization efforts.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
