<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Mechanism of degrader-targeted protein ubiquitinability
Authors: Crowe, C.; Nakasone, M.; Chandler, S.; Tatham, M. H.; Makukhin, N.; Hay, R. T.; Ciulli, A.
Score: 57.7, Published: 2024-02-05 DOI: 10.1101/2024.02.05.578957
Small molecule degraders of disease-driving proteins offer a clinically proven modality with enhanced therapeutic efficacy and the potential to tackle previously undrugged targets. Thermodynamically stable and kinetically long-lived degrader-mediated ternary complexes can drive faster, more profound and durable target degradation, however the mechanistic features by which they impact on target ubiquitination remain elusive.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Mechanism of degrader-targeted protein ubiquitinability
Authors: Crowe, C.; Nakasone, M.; Chandler, S.; Tatham, M. H.; Makukhin, N.; Hay, R. T.; Ciulli, A.
Score: 57.7, Published: 2024-02-05 DOI: 10.1101/2024.02.05.578957
Small molecule degraders of disease-driving proteins offer a clinically proven modality with enhanced therapeutic efficacy and the potential to tackle previously undrugged targets. Thermodynamically stable and kinetically long-lived degrader-mediated ternary complexes can drive faster, more profound and durable target degradation, however the mechanistic features by which they impact on target ubiquitination remain elusive." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-11T10:38:55+00:00" />
<meta property="article:modified_time" content="2024-02-11T10:38:55+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Mechanism of degrader-targeted protein ubiquitinability
Authors: Crowe, C.; Nakasone, M.; Chandler, S.; Tatham, M. H.; Makukhin, N.; Hay, R. T.; Ciulli, A.
Score: 57.7, Published: 2024-02-05 DOI: 10.1101/2024.02.05.578957
Small molecule degraders of disease-driving proteins offer a clinically proven modality with enhanced therapeutic efficacy and the potential to tackle previously undrugged targets. Thermodynamically stable and kinetically long-lived degrader-mediated ternary complexes can drive faster, more profound and durable target degradation, however the mechanistic features by which they impact on target ubiquitination remain elusive."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Mechanism of degrader-targeted protein ubiquitinability\nAuthors: Crowe, C.; Nakasone, M.; Chandler, S.; Tatham, M. H.; Makukhin, N.; Hay, R. T.; Ciulli, A.\nScore: 57.7, Published: 2024-02-05 DOI: 10.1101/2024.02.05.578957\nSmall molecule degraders of disease-driving proteins offer a clinically proven modality with enhanced therapeutic efficacy and the potential to tackle previously undrugged targets. Thermodynamically stable and kinetically long-lived degrader-mediated ternary complexes can drive faster, more profound and durable target degradation, however the mechanistic features by which they impact on target ubiquitination remain elusive.",
  "keywords": [
    
  ],
  "articleBody": " Mechanism of degrader-targeted protein ubiquitinability\nAuthors: Crowe, C.; Nakasone, M.; Chandler, S.; Tatham, M. H.; Makukhin, N.; Hay, R. T.; Ciulli, A.\nScore: 57.7, Published: 2024-02-05 DOI: 10.1101/2024.02.05.578957\nSmall molecule degraders of disease-driving proteins offer a clinically proven modality with enhanced therapeutic efficacy and the potential to tackle previously undrugged targets. Thermodynamically stable and kinetically long-lived degrader-mediated ternary complexes can drive faster, more profound and durable target degradation, however the mechanistic features by which they impact on target ubiquitination remain elusive. Here, we solve cryo-EM structures of the VHL Cullin 2 RING E3 ligase complexed with degrader MZ1, target protein Brd4BD2 and primed for catalysis with its cognate E2-ubiquitin bound. We find that Brd4BD2 adopts a favourable orientation towards the E2 active site. In vitro ubiquitination coupled with mass spectrometry illuminates a patch of ubiquitinable lysines on one face of Brd4BD2, with Lys456 showing optimal distance and geometry for nucleophilic attack. Our results demonstrate the proficiency of MZ1 in directing the substrate towards catalysis, explains the favourability of Brd4BD2 for ubiquitination, and reveals the flexibility of the enzyme in capturing sub-optimal lysines. We propose a model for ubiquitinability of degrader-recruited targets that provides a mechanistic blueprint for further rational drug design and optimization. One-Sentence SummaryStructural assembly a PROTAC-mediated complex of whole Cullin RING E3 ligase with bound target and E2-ubiquitin reveals structural and mechanistic insights of specificity for target protein ubiquitination.\nPersonalized Molecular Signatures of Insulin Resistance and Type 2 Diabetes\nAuthors: Larsen, J.; Stocks, B.; Henderson, J.; Andersson, D.; Backdahl, J.; Eriksson-Hogling, D.; Stidsen, J.; Sakamoto, K.; Hojlund, K.; Ryden, M.; Zierath, J. R.; Krook, A.; Deshmukh, A. S.\nScore: 20.2, Published: 2024-02-07 DOI: 10.1101/2024.02.06.578994\nHighlightsO_LIAdvanced proteomics analysis reveals personalized signatures of insulin resistance C_LIO_LIFasting muscle proteome and phosphoproteome predicts whole-body insulin sensitivity C_LIO_LIInsulin-stimulated phosphoproteome reveals selective insulin resistance signatures C_LIO_LIPhosphoproteome and proteome atlas explains sex-specific muscle metabolism C_LI O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=143 SRC=\"FIGDIR/small/578994v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (35K): org.highwire.dtl.DTLVardef@11237feorg.highwire.dtl.DTLVardef@11a0b66org.highwire.dtl.DTLVardef@e26998org.highwire.dtl.DTLVardef@8289ff_HPS_FORMAT_FIGEXP M_FIG O_FLOATNOGraphical AbstractC_FLOATNO C_FIG Insulin resistance is a hallmark of type 2 diabetes, which is a highly heterogeneous disease with diverse pathology. Understanding the molecular signatures of insulin resistance and its association with individual phenotypic traits is crucial for advancing precision medicine in type 2 diabetes. Utilizing cutting-edge proteomics technology, we mapped the proteome and phosphoproteome of skeletal muscle from \u003e120 men and women with normal glucose tolerance or type 2 diabetes, with varying degrees of insulin sensitivity. Leveraging deep in vivo phenotyping, we reveal that fasting proteome and phosphoproteome signatures strongly predict insulin sensitivity. Furthermore, the insulin-stimulated phosphoproteome revealed both dysregulated and preserved signaling nodes - even in individuals with severe insulin resistance. While substantial sex-specific differences in the proteome and phosphoproteome were identified, molecular signatures of insulin resistance remained largely similar between men and women. These findings underscore the need for precision medicine approaches in type 2 diabetes care, acknowledging disease heterogeneity.\nLarge-scale characterization of drug mechanism of action using proteome-wide thermal shift assays\nAuthors: Van Vranken, J. G.; Li, J.; Mintseris, J.; Gadzuk-Shea, M.; Gygi, S. P.; Schweppe, D.\nScore: 41.2, Published: 2024-02-10 DOI: 10.1101/2024.01.26.577428\nIn response to an ever-increasing demand of new small molecules therapeutics, numerous chemical and genetic tools have been developed to interrogate compound mechanism of action. Owing to its ability to characterize compound-dependent changes in thermal stability, the proteome-wide thermal shift assay has emerged as a powerful tool in this arsenal. The most recent iterations have drastically improved the overall efficiency of these assays, providing an opportunity to screen compounds at a previously unprecedented rate. Taking advantage of this advance, we quantified 1.498 million thermal stability measurements in response to multiple classes of therapeutic and tool compounds (96 compounds in living cells and 70 compounds in lysates). When interrogating the dataset as a whole, approximately 80% of compounds (with quantifiable targets) caused a significant change in the thermal stability of an annotated target. There was also a wealth of evidence portending off-target engagement despite the extensive use of the compounds in the laboratory and/or clinic. Finally, the combined application of cell- and lysate-based assays, aided in the classification of primary (direct ligand binding) and secondary (indirect) changes in thermal stability. Overall, this study highlights the value of these assays in the drug development process by affording an unbiased and reliable assessment of compound mechanism of action.\nConformational plasticity of a BiP-GRP94 chaperone complex\nAuthors: Brenner, J. C.; Zirden, L. C.; Almeida-Hernandez, Y.; Kaschani, F.; Kaiser, M.; Sanchez-Garcia, E.; Poepsel, S.; Hellerschmied, D.\nScore: 11.8, Published: 2024-02-02 DOI: 10.1101/2024.02.01.578445\nHsp70/Hsp90-chaperones and their regulatory co-chaperones are critical for maintaining protein homeostasis. GRP94, the sole Hsp90-chaperone in the secretory pathway of mammalian cells, is essential for the maturation of important secretory and transmembrane proteins. Without the requirement of co-chaperones, the Hsp70-protein BiP controls regulatory conformational changes of GRP94 - the structural basis of which has remained elusive. Here, we biochemically and structurally characterize the formation of a BiP-GRP94 chaperone complex and its transition to a conformation expected to support the loading of substrate proteins from BiP onto GRP94. BiP initially binds to the open GRP94 dimer via an interaction interface that is conserved among Hsp70/90 paralogs. Subsequently, binding of a second BiP protein stabilizes a semi-closed GRP94 dimer, thereby advancing the chaperone cycle. Our findings highlight a fundamental mechanism of direct Hsp70/90 cooperation, independent of co-chaperones.\nMolecular basis of ligand promiscuity, structural mimicry, and atypical dimerization in the chemokine receptors\nAuthors: Saha, S.; Sano, F. K.; Sharma, S.; Ganguly, M.; Saha, S.; Akasaka, H.; Kobayashi, T. A.; Zaidi, N.; Mishra, S.; Dalal, A.; Mohapatra, S.; Yadav, M.; Itoh, Y.; Leurs, R.; Chevigne, A.; Banerjee, R.; Shihoya, W.; Nureki, O.; SHUKLA, A. K.\nScore: 26.3, Published: 2024-02-02 DOI: 10.1101/2024.02.01.578380\nSelectivity of natural agonists for their cognate receptors is one of the hallmarks of the members of GPCR family, and it is crucial for the specificity of downstream signal-transduction. However, this selectivity often breaks down in the chemokine receptor subfamily, wherein a high degree of promiscuity is observed with one receptor recognizing multiple chemokines and one chemokine binding to multiple receptors. The molecular determinants of such a striking promiscuity for natural ligands in the chemokine-chemokine receptor system remain mostly elusive and represent an important knowledge gap in our current understanding. Here, we carry out a comprehensive transducer-coupling analysis, testing all known C-X-C chemokines on every C-X-C type chemokine receptor, to generate a global fingerprint of the selectivity and promiscuity encoded within this system. Taking lead from our finding, we determined cryo-EM structures of the most promiscuous receptor, CXCR2, in complex with every interacting chemokine, and deciphered the conserved molecular signatures and distinct binding modalities. While most chemokines position themselves on the receptor as a dimer, CXCL6 exhibits a monomeric binding pose induced by a previously unanticipated reorientation of its carboxyl-terminal -helix, leading to disruption of the dimer interface. Surprisingly, one of the chemokines, CXCL5, induces a ligand-swapped dimer of CXCR2, the first of its kind observed in class A GPCRs, wherein each protomer of the ligand engages its own receptor without any discernible receptor-receptor interface. These unique observations provide a possible structural mechanism for inherent functional specialization encoded in chemokines despite their convergence to a common receptor. Furthermore, we also determined cryo-EM structures of CXCR3 in complex with G-protein-biased and {beta}-arrestin-biased small molecule agonists that elucidate distinct allosteric modulations in the receptor driving their divergent transducer-coupling bias. Guided by structural analysis and experimental validation, we discover that in contrast to previously held notion, small molecule agonists of CXCR3 display robust agonism at CXCR7, an intrinsically biased, {beta}-arrestin-coupled receptor, making them first-in-class dual agonists for chemokine receptors with exclusive {beta}arr-bias at CXCR7. Taken together, our study provides molecular insights into ligand promiscuity and signaling bias at the chemokine receptors, and also demonstrates a proof of principle that naturally encoded structural mimicry can be recapitulated using synthetic pharmacophores with potential implications for developing novel therapeutics.\nConstraints on the Emergence of RNA through Non-Templated Primer Extension with Mixtures of Potentially Prebiotic Nucleotides\nAuthors: Jia, X.; Zhang, S. J.; Zhou, L.; Szostak, J. W.\nScore: 9.2, Published: 2024-01-22 DOI: 10.1101/2024.01.21.576316\nThe emergence of RNA on the early Earth is likely to have been influenced by a series of chemical and physical processes that acted to filter out various alternative nucleic acids. For example, UV photostability is thought to have favored the survival of the canonical nucleotides. In a recent proposal for the prebiotic synthesis of the building blocks of RNA, ribonucleotides share a common pathway with arabino- and threo-nucleotides. We have therefore investigated non-templated primer extension with 2-aminoimidazole-activated forms of these alternative nucleotides to see if the synthesis of the first oligonucleotides might have been biased in favor of RNA. We show that non-templated primer extension occurs predominantly through 5'-5' imidazolium bridged dinucleotides, echoing the mechanism of template-directed primer extension. Ribo- and arabino-nucleotides exhibited comparable rates and yields of non-templated primer extension, whereas threo-nucleotides showed lower reactivity. Competition experiments with mixtures of nucleotides confirmed the bias against the incorporation of threo-nucleotides into oligonucleotides. This bias, coupled with selective prebiotic synthesis and templated copying favoring ribonucleotides, provides a plausible model for the exclusion of threo-nucleotides from primordial oligonucleotides. In contrast, the exclusion of arabino-nucleotides may have resulted primarily from biases in synthesis and in template-directed primer extension.\nFunctional Analysis of the Zinc Finger Modules of the S. cerevisiae Splicing Factor Luc7\nAuthors: Carrocci, T. J.; DeMario, S. M.; He, K.; Zeps, N. J.; Harkner, C.; CHANFREAU, G. F.; Hoskins, A.\nScore: 6.0, Published: 2024-02-04 DOI: 10.1101/2024.02.04.578419\nIdentification of splice sites is a critical step in pre-mRNA splicing since definition of the exon/intron boundaries controls what nucleotides are incorporated into mature mRNAs. The intron boundary with the upstream exon is initially identified through interactions with the U1 snRNP. This involves both base pairing between the U1 snRNA and the pre-mRNA as well as snRNP proteins interacting with the 5 splice site/snRNA duplex. In yeast, this duplex is buttressed by two conserved protein factors, Yhc1 and Luc7. Luc7 has three human paralogs (LUC7L, LUC7L2, and LUC7L3) which play roles in alternative splicing. What domains of these paralogs promote splicing at particular sites is not yet clear. Here, we humanized the zinc finger domains of the yeast Luc7 protein in order to understand their roles in splice site selection using reporter assays, transcriptome analysis, and genetic interactions. While we were unable to determine a function for the first zinc finger domain, humanization of the second zinc finger domain to mirror that found in LUC7L or LUC7L2 resulted in altered usage of nonconsensus 5 splice sites. In contrast, the corresponding zinc finger domain of LUC7L3 could not support yeast viability. Further, humanization of Luc7 can suppress mutation of the ATPase Prp28, which is involved in U1 release and exchange for U6 at the 5 splice site. Our work reveals a role for the second zinc finger of Luc7 in splice site selection and suggests that different zinc finger domains may have different ATPase requirements for release by Prp28.\nStructural basis for the inhibition of PRC2 by active transcription histone posttranslational modifications\nAuthors: Cookis, T.; Lydecker, A.; Sauer, P. V.; Kasinath, V.; Nogales, E.\nScore: 5.0, Published: 2024-02-10 DOI: 10.1101/2024.02.09.579730\nPolycomb repressive complex 2 (PRC2) is an epigenetic regulator essential for embryonic development and maintenance of cell identity that trimethylates histone H3 at lysine 27 (H3K27me3) leading to gene silencing. PRC2 is regulated by association with protein cofactors and crosstalk with histone posttranslational modifications. Trimethylated histone H3 K4 (H3K4me3) and K36 (H3K36me3) localize to sites of active transcription where H3K27me3 is absent and inhibit PRC2 activity through unknown mechanisms. Using cryo-electron microscopy we reveal that histone H3 tails modified with H3K36me3 engage poorly with the PRC2 active site and preclude its effective interaction with chromatin, while the H3K4me3 modification binds to the allosteric site in the EED subunit, acting as an antagonist that competes with allosteric activators required for the spreading of the H3K27me3 repressive mark. Thus, the location along the H3 tail of the H3K4me3 and H3K36me3 modifications allow them to target two essential requirements for efficient trimethylation of histone H3K27. We further show that the JARID2 cofactor modulates PRC2 activity in the presence of these histone modifications.\nCharacterization of ribosome stalling and no-go mRNA decay stimulated by the Fragile X protein, FMRP\nAuthors: Scarpitti, M. R.; Pastore, B.; Tang, W.; Kearse, M. G.\nScore: 4.6, Published: 2024-02-04 DOI: 10.1101/2024.02.02.577121\nLoss of functional fragile X mental retardation protein (FMRP) causes fragile X syndrome (FXS) and is the leading monogenic cause of autism spectrum disorders and intellectual disability. FMRP is most notably a translational repressor and is thought to inhibit translation elongation by stalling ribosomes as FMRP-bound polyribosomes from brain tissue are resistant to puromycin and nuclease treatment. Here, we present data showing that the C-terminal non-canonical RNA-binding domain of FMRP is essential and sufficient to induce puromycin-resistant mRNA*ribosome complexes. Given that stalled ribosomes can stimulate ribosome collisions and no-go mRNA decay (NGD), we tested the ability of FMRP to drive NGD of its target transcripts in neuroblastoma cells. Indeed, FMRP and ribosomal proteins, but not PABPC1, were enriched in isolated nuclease-resistant disomes compared to controls. Using siRNA knockdown and RNA-seq, we identified 16 putative FMRP-mediated NGD substrates, many of which encode proteins involved in neuronal development and function. Increased mRNA stability of the putative substrates was also observed when either FMRP was depleted or NGD was prevented via RNAi. Taken together, these data support that FMRP stalls ribosomes and can stimulate NGD of a select set of transcripts in cells, revealing an unappreciated role of FMRP that would be misregulated in FXS.\nLung Mycobacterium tuberculosis infection perturbs metabolic pathways in non-pulmonary tissues\nAuthors: Pahwa, F.; Chaudhary, S.; Dayal, A.; Nanda, R.\nScore: 3.9, Published: 2024-02-09 DOI: 10.1101/2024.02.09.579656\nMycobacterium tuberculosis (Mtb), through aerosol, reaches the lungs to cause pulmonary tuberculosis (TB); however, it may also affect the metabolism of other tissues in age-specific ways. In this study, female C57BL/6 mice (2 and 5 months old; M) were infected with a low aerosol dose (100-200 cfu) of Mtb H37Rv to monitor tissue mycobacterial load and multi-tissue metabolite profiling using gas chromatography and mass spectrometry (GC-MS). 5M C57BL/6 mice showed separate tissue metabolic phenotype with significantly higher lung aspartic acid, fecal oxalic acid and tryptophan levels with lower liver lysine and aspartic acid and fecal phenylalanine levels (log2FC: 5M/2M\u003e +/-1.0, p\u003c0.1) compared to 2M young controls. Upon Mtb infection, the lung mycobacterial load of 2M and 5M mice were similar till 6 weeks post-infection. However, significantly higher lung phosphoric acid, malonic acid and lower mannose levels (log2FC: Mtb infected/healthy\u003e +/-1.0, p\u003c0.1) were observed in Mtb-infected 5M C57BL/6 mice. Meanwhile, Mtb-infected 2M mice showed higher liver xylose and lower lysine levels. The thigh muscles of Mtb-infected 2M and 5M mice showed increased malic acid and oxalic acid and decreased glycine, serine, and glycerol levels. Fecal aspartic acid level was higher in Mtb-infected 5M mice, while a decreased abundance of fecal lysine was observed in Mtb-infected 2M mice. Overall, this study demonstrates a deregulated tissue-specific amino acid metabolism in Mtb-infected mice groups of different age groups, which might be targeted for managing TB infection-related adverse effects.\n",
  "wordCount" : "2516",
  "inLanguage": "en",
  "datePublished": "2024-02-11T10:38:55Z",
  "dateModified": "2024-02-11T10:38:55Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 11, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.05.578957">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.05.578957" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.05.578957">
        <p class="paperTitle">Mechanism of degrader-targeted protein ubiquitinability</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.05.578957" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.05.578957" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Crowe, C.; Nakasone, M.; Chandler, S.; Tatham, M. H.; Makukhin, N.; Hay, R. T.; Ciulli, A.</p>
        <p class="info">Score: 57.7, Published: 2024-02-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.05.578957' target='https://doi.org/10.1101/2024.02.05.578957'> 10.1101/2024.02.05.578957</a></p>
        <p class="abstract">Small molecule degraders of disease-driving proteins offer a clinically proven modality with enhanced therapeutic efficacy and the potential to tackle previously undrugged targets. Thermodynamically stable and kinetically long-lived degrader-mediated ternary complexes can drive faster, more profound and durable target degradation, however the mechanistic features by which they impact on target ubiquitination remain elusive. Here, we solve cryo-EM structures of the VHL Cullin 2 RING E3 ligase complexed with degrader MZ1, target protein Brd4BD2 and primed for catalysis with its cognate E2-ubiquitin bound. We find that Brd4BD2 adopts a favourable orientation towards the E2 active site. In vitro ubiquitination coupled with mass spectrometry illuminates a patch of ubiquitinable lysines on one face of Brd4BD2, with Lys456 showing optimal distance and geometry for nucleophilic attack. Our results demonstrate the proficiency of MZ1 in directing the substrate towards catalysis, explains the favourability of Brd4BD2 for ubiquitination, and reveals the flexibility of the enzyme in capturing sub-optimal lysines. We propose a model for ubiquitinability of degrader-recruited targets that provides a mechanistic blueprint for further rational drug design and optimization.

One-Sentence SummaryStructural assembly a PROTAC-mediated complex of whole Cullin RING E3 ligase with bound target and E2-ubiquitin reveals structural and mechanistic insights of specificity for target protein ubiquitination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.578994">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.578994" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.578994">
        <p class="paperTitle">Personalized Molecular Signatures of Insulin Resistance and Type 2 Diabetes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.578994" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.578994" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Larsen, J.; Stocks, B.; Henderson, J.; Andersson, D.; Backdahl, J.; Eriksson-Hogling, D.; Stidsen, J.; Sakamoto, K.; Hojlund, K.; Ryden, M.; Zierath, J. R.; Krook, A.; Deshmukh, A. S.</p>
        <p class="info">Score: 20.2, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.578994' target='https://doi.org/10.1101/2024.02.06.578994'> 10.1101/2024.02.06.578994</a></p>
        <p class="abstract">HighlightsO_LIAdvanced proteomics analysis reveals personalized signatures of insulin resistance
C_LIO_LIFasting muscle proteome and phosphoproteome predicts whole-body insulin sensitivity
C_LIO_LIInsulin-stimulated phosphoproteome reveals selective insulin resistance signatures
C_LIO_LIPhosphoproteome and proteome atlas explains sex-specific muscle metabolism
C_LI



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=143 SRC=&#34;FIGDIR/small/578994v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (35K):
org.highwire.dtl.DTLVardef@11237feorg.highwire.dtl.DTLVardef@11a0b66org.highwire.dtl.DTLVardef@e26998org.highwire.dtl.DTLVardef@8289ff_HPS_FORMAT_FIGEXP  M_FIG O_FLOATNOGraphical AbstractC_FLOATNO C_FIG Insulin resistance is a hallmark of type 2 diabetes, which is a highly heterogeneous disease with diverse pathology. Understanding the molecular signatures of insulin resistance and its association with individual phenotypic traits is crucial for advancing precision medicine in type 2 diabetes. Utilizing cutting-edge proteomics technology, we mapped the proteome and phosphoproteome of skeletal muscle from &gt;120 men and women with normal glucose tolerance or type 2 diabetes, with varying degrees of insulin sensitivity. Leveraging deep in vivo phenotyping, we reveal that fasting proteome and phosphoproteome signatures strongly predict insulin sensitivity. Furthermore, the insulin-stimulated phosphoproteome revealed both dysregulated and preserved signaling nodes - even in individuals with severe insulin resistance. While substantial sex-specific differences in the proteome and phosphoproteome were identified, molecular signatures of insulin resistance remained largely similar between men and women. These findings underscore the need for precision medicine approaches in type 2 diabetes care, acknowledging disease heterogeneity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.26.577428">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.26.577428" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.26.577428">
        <p class="paperTitle">Large-scale characterization of drug mechanism of action using proteome-wide thermal shift assays</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.26.577428" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.26.577428" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Van Vranken, J. G.; Li, J.; Mintseris, J.; Gadzuk-Shea, M.; Gygi, S. P.; Schweppe, D.</p>
        <p class="info">Score: 41.2, Published: 2024-02-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.26.577428' target='https://doi.org/10.1101/2024.01.26.577428'> 10.1101/2024.01.26.577428</a></p>
        <p class="abstract">In response to an ever-increasing demand of new small molecules therapeutics, numerous chemical and genetic tools have been developed to interrogate compound mechanism of action. Owing to its ability to characterize compound-dependent changes in thermal stability, the proteome-wide thermal shift assay has emerged as a powerful tool in this arsenal. The most recent iterations have drastically improved the overall efficiency of these assays, providing an opportunity to screen compounds at a previously unprecedented rate. Taking advantage of this advance, we quantified 1.498 million thermal stability measurements in response to multiple classes of therapeutic and tool compounds (96 compounds in living cells and 70 compounds in lysates). When interrogating the dataset as a whole, approximately 80% of compounds (with quantifiable targets) caused a significant change in the thermal stability of an annotated target. There was also a wealth of evidence portending off-target engagement despite the extensive use of the compounds in the laboratory and/or clinic. Finally, the combined application of cell- and lysate-based assays, aided in the classification of primary (direct ligand binding) and secondary (indirect) changes in thermal stability. Overall, this study highlights the value of these assays in the drug development process by affording an unbiased and reliable assessment of compound mechanism of action.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578445">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578445" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578445">
        <p class="paperTitle">Conformational plasticity of a BiP-GRP94 chaperone complex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578445" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578445" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brenner, J. C.; Zirden, L. C.; Almeida-Hernandez, Y.; Kaschani, F.; Kaiser, M.; Sanchez-Garcia, E.; Poepsel, S.; Hellerschmied, D.</p>
        <p class="info">Score: 11.8, Published: 2024-02-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578445' target='https://doi.org/10.1101/2024.02.01.578445'> 10.1101/2024.02.01.578445</a></p>
        <p class="abstract">Hsp70/Hsp90-chaperones and their regulatory co-chaperones are critical for maintaining protein homeostasis. GRP94, the sole Hsp90-chaperone in the secretory pathway of mammalian cells, is essential for the maturation of important secretory and transmembrane proteins. Without the requirement of co-chaperones, the Hsp70-protein BiP controls regulatory conformational changes of GRP94 - the structural basis of which has remained elusive. Here, we biochemically and structurally characterize the formation of a BiP-GRP94 chaperone complex and its transition to a conformation expected to support the loading of substrate proteins from BiP onto GRP94. BiP initially binds to the open GRP94 dimer via an interaction interface that is conserved among Hsp70/90 paralogs. Subsequently, binding of a second BiP protein stabilizes a semi-closed GRP94 dimer, thereby advancing the chaperone cycle. Our findings highlight a fundamental mechanism of direct Hsp70/90 cooperation, independent of co-chaperones.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578380">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578380" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578380">
        <p class="paperTitle">Molecular basis of ligand promiscuity, structural mimicry, and atypical dimerization in the chemokine receptors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578380" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578380" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Saha, S.; Sano, F. K.; Sharma, S.; Ganguly, M.; Saha, S.; Akasaka, H.; Kobayashi, T. A.; Zaidi, N.; Mishra, S.; Dalal, A.; Mohapatra, S.; Yadav, M.; Itoh, Y.; Leurs, R.; Chevigne, A.; Banerjee, R.; Shihoya, W.; Nureki, O.; SHUKLA, A. K.</p>
        <p class="info">Score: 26.3, Published: 2024-02-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578380' target='https://doi.org/10.1101/2024.02.01.578380'> 10.1101/2024.02.01.578380</a></p>
        <p class="abstract">Selectivity of natural agonists for their cognate receptors is one of the hallmarks of the members of GPCR family, and it is crucial for the specificity of downstream signal-transduction. However, this selectivity often breaks down in the chemokine receptor subfamily, wherein a high degree of promiscuity is observed with one receptor recognizing multiple chemokines and one chemokine binding to multiple receptors. The molecular determinants of such a striking promiscuity for natural ligands in the chemokine-chemokine receptor system remain mostly elusive and represent an important knowledge gap in our current understanding. Here, we carry out a comprehensive transducer-coupling analysis, testing all known C-X-C chemokines on every C-X-C type chemokine receptor, to generate a global fingerprint of the selectivity and promiscuity encoded within this system. Taking lead from our finding, we determined cryo-EM structures of the most promiscuous receptor, CXCR2, in complex with every interacting chemokine, and deciphered the conserved molecular signatures and distinct binding modalities. While most chemokines position themselves on the receptor as a dimer, CXCL6 exhibits a monomeric binding pose induced by a previously unanticipated reorientation of its carboxyl-terminal -helix, leading to disruption of the dimer interface. Surprisingly, one of the chemokines, CXCL5, induces a ligand-swapped dimer of CXCR2, the first of its kind observed in class A GPCRs, wherein each protomer of the ligand engages its own receptor without any discernible receptor-receptor interface. These unique observations provide a possible structural mechanism for inherent functional specialization encoded in chemokines despite their convergence to a common receptor. Furthermore, we also determined cryo-EM structures of CXCR3 in complex with G-protein-biased and {beta}-arrestin-biased small molecule agonists that elucidate distinct allosteric modulations in the receptor driving their divergent transducer-coupling bias. Guided by structural analysis and experimental validation, we discover that in contrast to previously held notion, small molecule agonists of CXCR3 display robust agonism at CXCR7, an intrinsically biased, {beta}-arrestin-coupled receptor, making them first-in-class dual agonists for chemokine receptors with exclusive {beta}arr-bias at CXCR7. Taken together, our study provides molecular insights into ligand promiscuity and signaling bias at the chemokine receptors, and also demonstrates a proof of principle that naturally encoded structural mimicry can be recapitulated using synthetic pharmacophores with potential implications for developing novel therapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.21.576316">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.21.576316" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.21.576316">
        <p class="paperTitle">Constraints on the Emergence of RNA through Non-Templated Primer Extension with Mixtures of Potentially Prebiotic Nucleotides</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.21.576316" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.21.576316" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jia, X.; Zhang, S. J.; Zhou, L.; Szostak, J. W.</p>
        <p class="info">Score: 9.2, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.21.576316' target='https://doi.org/10.1101/2024.01.21.576316'> 10.1101/2024.01.21.576316</a></p>
        <p class="abstract">The emergence of RNA on the early Earth is likely to have been influenced by a series of chemical and physical processes that acted to filter out various alternative nucleic acids. For example, UV photostability is thought to have favored the survival of the canonical nucleotides. In a recent proposal for the prebiotic synthesis of the building blocks of RNA, ribonucleotides share a common pathway with arabino- and threo-nucleotides. We have therefore investigated non-templated primer extension with 2-aminoimidazole-activated forms of these alternative nucleotides to see if the synthesis of the first oligonucleotides might have been biased in favor of RNA. We show that non-templated primer extension occurs predominantly through 5&#39;-5&#39; imidazolium bridged dinucleotides, echoing the mechanism of template-directed primer extension. Ribo- and arabino-nucleotides exhibited comparable rates and yields of non-templated primer extension, whereas threo-nucleotides showed lower reactivity. Competition experiments with mixtures of nucleotides confirmed the bias against the incorporation of threo-nucleotides into oligonucleotides. This bias, coupled with selective prebiotic synthesis and templated copying favoring ribonucleotides, provides a plausible model for the exclusion of threo-nucleotides from primordial oligonucleotides. In contrast, the exclusion of arabino-nucleotides may have resulted primarily from biases in synthesis and in template-directed primer extension.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.04.578419">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.04.578419" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.04.578419">
        <p class="paperTitle">Functional Analysis of the Zinc Finger Modules of the S. cerevisiae Splicing Factor Luc7</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.04.578419" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.04.578419" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carrocci, T. J.; DeMario, S. M.; He, K.; Zeps, N. J.; Harkner, C.; CHANFREAU, G. F.; Hoskins, A.</p>
        <p class="info">Score: 6.0, Published: 2024-02-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.04.578419' target='https://doi.org/10.1101/2024.02.04.578419'> 10.1101/2024.02.04.578419</a></p>
        <p class="abstract">Identification of splice sites is a critical step in pre-mRNA splicing since definition of the exon/intron boundaries controls what nucleotides are incorporated into mature mRNAs. The intron boundary with the upstream exon is initially identified through interactions with the U1 snRNP. This involves both base pairing between the U1 snRNA and the pre-mRNA as well as snRNP proteins interacting with the 5 splice site/snRNA duplex. In yeast, this duplex is buttressed by two conserved protein factors, Yhc1 and Luc7. Luc7 has three human paralogs (LUC7L, LUC7L2, and LUC7L3) which play roles in alternative splicing. What domains of these paralogs promote splicing at particular sites is not yet clear. Here, we humanized the zinc finger domains of the yeast Luc7 protein in order to understand their roles in splice site selection using reporter assays, transcriptome analysis, and genetic interactions. While we were unable to determine a function for the first zinc finger domain, humanization of the second zinc finger domain to mirror that found in LUC7L or LUC7L2 resulted in altered usage of nonconsensus 5 splice sites. In contrast, the corresponding zinc finger domain of LUC7L3 could not support yeast viability. Further, humanization of Luc7 can suppress mutation of the ATPase Prp28, which is involved in U1 release and exchange for U6 at the 5 splice site. Our work reveals a role for the second zinc finger of Luc7 in splice site selection and suggests that different zinc finger domains may have different ATPase requirements for release by Prp28.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579730">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579730" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579730">
        <p class="paperTitle">Structural basis for the inhibition of PRC2 by active transcription histone posttranslational modifications</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579730" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579730" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cookis, T.; Lydecker, A.; Sauer, P. V.; Kasinath, V.; Nogales, E.</p>
        <p class="info">Score: 5.0, Published: 2024-02-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579730' target='https://doi.org/10.1101/2024.02.09.579730'> 10.1101/2024.02.09.579730</a></p>
        <p class="abstract">Polycomb repressive complex 2 (PRC2) is an epigenetic regulator essential for embryonic development and maintenance of cell identity that trimethylates histone H3 at lysine 27 (H3K27me3) leading to gene silencing. PRC2 is regulated by association with protein cofactors and crosstalk with histone posttranslational modifications. Trimethylated histone H3 K4 (H3K4me3) and K36 (H3K36me3) localize to sites of active transcription where H3K27me3 is absent and inhibit PRC2 activity through unknown mechanisms. Using cryo-electron microscopy we reveal that histone H3 tails modified with H3K36me3 engage poorly with the PRC2 active site and preclude its effective interaction with chromatin, while the H3K4me3 modification binds to the allosteric site in the EED subunit, acting as an antagonist that competes with allosteric activators required for the spreading of the H3K27me3 repressive mark. Thus, the location along the H3 tail of the H3K4me3 and H3K36me3 modifications allow them to target two essential requirements for efficient trimethylation of histone H3K27. We further show that the JARID2 cofactor modulates PRC2 activity in the presence of these histone modifications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.02.577121">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.02.577121" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.02.577121">
        <p class="paperTitle">Characterization of ribosome stalling and no-go mRNA decay stimulated by the Fragile X protein, FMRP</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.02.577121" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.02.577121" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Scarpitti, M. R.; Pastore, B.; Tang, W.; Kearse, M. G.</p>
        <p class="info">Score: 4.6, Published: 2024-02-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.02.577121' target='https://doi.org/10.1101/2024.02.02.577121'> 10.1101/2024.02.02.577121</a></p>
        <p class="abstract">Loss of functional fragile X mental retardation protein (FMRP) causes fragile X syndrome (FXS) and is the leading monogenic cause of autism spectrum disorders and intellectual disability. FMRP is most notably a translational repressor and is thought to inhibit translation elongation by stalling ribosomes as FMRP-bound polyribosomes from brain tissue are resistant to puromycin and nuclease treatment. Here, we present data showing that the C-terminal non-canonical RNA-binding domain of FMRP is essential and sufficient to induce puromycin-resistant mRNA*ribosome complexes. Given that stalled ribosomes can stimulate ribosome collisions and no-go mRNA decay (NGD), we tested the ability of FMRP to drive NGD of its target transcripts in neuroblastoma cells. Indeed, FMRP and ribosomal proteins, but not PABPC1, were enriched in isolated nuclease-resistant disomes compared to controls. Using siRNA knockdown and RNA-seq, we identified 16 putative FMRP-mediated NGD substrates, many of which encode proteins involved in neuronal development and function. Increased mRNA stability of the putative substrates was also observed when either FMRP was depleted or NGD was prevented via RNAi. Taken together, these data support that FMRP stalls ribosomes and can stimulate NGD of a select set of transcripts in cells, revealing an unappreciated role of FMRP that would be misregulated in FXS.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579656">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579656" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579656">
        <p class="paperTitle">Lung Mycobacterium tuberculosis infection perturbs metabolic pathways in non-pulmonary tissues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579656" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579656" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pahwa, F.; Chaudhary, S.; Dayal, A.; Nanda, R.</p>
        <p class="info">Score: 3.9, Published: 2024-02-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579656' target='https://doi.org/10.1101/2024.02.09.579656'> 10.1101/2024.02.09.579656</a></p>
        <p class="abstract">Mycobacterium tuberculosis (Mtb), through aerosol, reaches the lungs to cause pulmonary tuberculosis (TB); however, it may also affect the metabolism of other tissues in age-specific ways. In this study, female C57BL/6 mice (2 and 5 months old; M) were infected with a low aerosol dose (100-200 cfu) of Mtb H37Rv to monitor tissue mycobacterial load and multi-tissue metabolite profiling using gas chromatography and mass spectrometry (GC-MS). 5M C57BL/6 mice showed separate tissue metabolic phenotype with significantly higher lung aspartic acid, fecal oxalic acid and tryptophan levels with lower liver lysine and aspartic acid and fecal phenylalanine levels (log2FC: 5M/2M&gt; &#43;/-1.0, p&lt;0.1) compared to 2M young controls. Upon Mtb infection, the lung mycobacterial load of 2M and 5M mice were similar till 6 weeks post-infection. However, significantly higher lung phosphoric acid, malonic acid and lower mannose levels (log2FC: Mtb infected/healthy&gt; &#43;/-1.0, p&lt;0.1) were observed in Mtb-infected 5M C57BL/6 mice. Meanwhile, Mtb-infected 2M mice showed higher liver xylose and lower lysine levels. The thigh muscles of Mtb-infected 2M and 5M mice showed increased malic acid and oxalic acid and decreased glycine, serine, and glycerol levels. Fecal aspartic acid level was higher in Mtb-infected 5M mice, while a decreased abundance of fecal lysine was observed in Mtb-infected 2M mice. Overall, this study demonstrates a deregulated tissue-specific amino acid metabolism in Mtb-infected mice groups of different age groups, which might be targeted for managing TB infection-related adverse effects.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
