<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Highly specific intracellular ubiquitination of a small molecule
Authors: Li, W.; Garcia-Rivera, E. M.; Mitchell, D. C.; Chick, J. M.; Maetani, M.; Knapp, J. M.; Matthews, G. M.; Shirasaki, R.; de Matos Simoes, R.; Viswanathan, V.; Pulice, J. L.; Rees, M. G.; Roth, J. A.; Gygi, S. P.; Mitsiades, C. S.; Kadoch, C.; Schreiber, S. L.; Ostrem, J. M. L.
Score: 68.5, Published: 2024-01-28 DOI: 10.1101/2024.01.26.577493
Ubiquitin is a small, highly conserved protein that acts as a posttranslational modification in eukaryotes.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Highly specific intracellular ubiquitination of a small molecule
Authors: Li, W.; Garcia-Rivera, E. M.; Mitchell, D. C.; Chick, J. M.; Maetani, M.; Knapp, J. M.; Matthews, G. M.; Shirasaki, R.; de Matos Simoes, R.; Viswanathan, V.; Pulice, J. L.; Rees, M. G.; Roth, J. A.; Gygi, S. P.; Mitsiades, C. S.; Kadoch, C.; Schreiber, S. L.; Ostrem, J. M. L.
Score: 68.5, Published: 2024-01-28 DOI: 10.1101/2024.01.26.577493
Ubiquitin is a small, highly conserved protein that acts as a posttranslational modification in eukaryotes." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-31T10:38:28+00:00" />
<meta property="article:modified_time" content="2024-01-31T10:38:28+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Highly specific intracellular ubiquitination of a small molecule
Authors: Li, W.; Garcia-Rivera, E. M.; Mitchell, D. C.; Chick, J. M.; Maetani, M.; Knapp, J. M.; Matthews, G. M.; Shirasaki, R.; de Matos Simoes, R.; Viswanathan, V.; Pulice, J. L.; Rees, M. G.; Roth, J. A.; Gygi, S. P.; Mitsiades, C. S.; Kadoch, C.; Schreiber, S. L.; Ostrem, J. M. L.
Score: 68.5, Published: 2024-01-28 DOI: 10.1101/2024.01.26.577493
Ubiquitin is a small, highly conserved protein that acts as a posttranslational modification in eukaryotes."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Highly specific intracellular ubiquitination of a small molecule\nAuthors: Li, W.; Garcia-Rivera, E. M.; Mitchell, D. C.; Chick, J. M.; Maetani, M.; Knapp, J. M.; Matthews, G. M.; Shirasaki, R.; de Matos Simoes, R.; Viswanathan, V.; Pulice, J. L.; Rees, M. G.; Roth, J. A.; Gygi, S. P.; Mitsiades, C. S.; Kadoch, C.; Schreiber, S. L.; Ostrem, J. M. L.\nScore: 68.5, Published: 2024-01-28 DOI: 10.1101/2024.01.26.577493\nUbiquitin is a small, highly conserved protein that acts as a posttranslational modification in eukaryotes.",
  "keywords": [
    
  ],
  "articleBody": " Highly specific intracellular ubiquitination of a small molecule\nAuthors: Li, W.; Garcia-Rivera, E. M.; Mitchell, D. C.; Chick, J. M.; Maetani, M.; Knapp, J. M.; Matthews, G. M.; Shirasaki, R.; de Matos Simoes, R.; Viswanathan, V.; Pulice, J. L.; Rees, M. G.; Roth, J. A.; Gygi, S. P.; Mitsiades, C. S.; Kadoch, C.; Schreiber, S. L.; Ostrem, J. M. L.\nScore: 68.5, Published: 2024-01-28 DOI: 10.1101/2024.01.26.577493\nUbiquitin is a small, highly conserved protein that acts as a posttranslational modification in eukaryotes. Ubiquitination of proteins frequently serves as a degradation signal, marking them for disposal by the proteasome. Here, we report a novel small molecule from a diversity-oriented synthesis library, BRD1732, that is directly ubiquitinated in cells, resulting in dramatic accumulation of inactive ubiquitin monomers and polyubiquitin chains causing broad inhibition of the ubiquitin-proteasome system. Ubiquitination of BRD1732 and its associated cytotoxicity are stereospecific and dependent upon two homologous E3 ubiquitin ligases, RNF19A and RNF19B. Our finding opens the possibility for indirect ubiquitination of a target through a ubiquitinated bifunctional small molecule, and more broadly raises the potential for posttranslational modification in trans.\nLarge-scale characterization of drug mechanism of action using proteome-wide thermal shift assays\nAuthors: Van Vranken, J. G.; Li, J.; Mintseris, J.; Gadzuk-Shea, M.; Gygi, S. P.; Schweppe, D.\nScore: 30.4, Published: 2024-01-27 DOI: 10.1101/2024.01.26.577428\nIn response to an ever-increasing demand of new small molecules therapeutics, numerous chemical and genetic tools have been developed to interrogate compound mechanism of action. Owing to its ability to characterize compound-dependent changes in thermal stability, the proteome-wide thermal shift assay has emerged as a powerful tool in this arsenal. The most recent iterations have drastically improved the overall efficiency of these assays, providing an opportunity to screen compounds at a previously unprecedented rate. Taking advantage of this advance, we quantified 1.498 million thermal stability measurements in response to multiple classes of therapeutic and tool compounds (96 compounds in living cells and 70 compounds in lysates). When interrogating the dataset as a whole, approximately 80% of compounds (with quantifiable targets) caused a significant change in the thermal stability of an annotated target. There was also a wealth of evidence portending off-target engagement despite the extensive use of the compounds in the laboratory and/or clinic. Finally, the combined application of cell- and lysate-based assays, aided in the classification of primary (direct ligand binding) and secondary (indirect) changes in thermal stability. Overall, this study highlights the value of these assays in the drug development process by affording an unbiased and reliable assessment of compound mechanism of action.\nACKR3 Proximity Labeling Identifies Novel G protein- and Î²-arrestin-independent GPCR Interacting Proteins\nAuthors: Hicks, C.; Gardner, J.; Eiger, D. S.; Camarda, N.; Pham, U.; Dhar, S.; Rodriguez, H.; Chundi, A.; Rajagopal, S.\nScore: 17.9, Published: 2024-01-28 DOI: 10.1101/2024.01.27.577545\nThe canonical paradigm of GPCR signaling recognizes G proteins and {beta}-arrestins as the two primary transducers that promote GPCR signaling. Recent evidence suggests the atypical chemokine receptor 3 (ACKR3) does not couple to G proteins, and {beta}-arrestins are dispensable for some of its functions. Here, we employed proximity labeling to identify proteins that interact with ACKR3 in cells devoid of {beta}-arrestin. We identified proteins involved in the endocytic machinery and evaluated a subset of proteins conserved across several GPCR-based proximity labeling experiments. We discovered that the bone morphogenic protein 2-inducible kinase (BMP2K) interacts with many different GPCRs with varying dependency on {beta}-arrestin. Together, our work highlights the existence of modulators that can act independently of G proteins and {beta}-arrestins to regulate GPCR signaling and provides important evidence for other targets that may regulate GPCR signaling.\nMechanism of chaperone coordination during cotranslational protein folding in bacteria\nAuthors: Roeselova, A.; Maslen, S. L.; Shivakumaraswamy, S.; Pellowe, G. A.; Howell, S.; Joshi, D.; Redmond, J.; Kjaer, S.; Skehel, M.; Balchin, D.\nScore: 16.2, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576655\nProtein folding is assisted by molecular chaperones that bind nascent polypeptides during mRNA translation. Several structurally-distinct classes of chaperone promote de novo folding, suggesting that their activities are coordinated at the ribosome. We used biochemical reconstitution and structural proteomics to explore the molecular basis for cotranslational chaperone action in bacteria. We found that chaperone binding is disfavoured close to the ribosome, allowing folding to precede chaperone recruitment. Trigger factor subsequently recognises compact folding intermediates exposing extensive non-native surface and dictates DnaJ access to nascent chains. DnaJ uses a large surface to bind structurally diverse intermediates, and recruits DnaK to sequence-diverse solvent-accessible sites. Neither Trigger factor, DnaJ nor DnaK destabilize cotranslational folding intermediates. Instead, the chaperones collaborate to create a protected space for protein maturation that extends well beyond the ribosome exit tunnel. Our findings show how the chaperone network selects and modulates cotranslational folding intermediates.\nVisualizing chaperone-mediated multistep assembly of the human 20S proteasome\nAuthors: Adolf, F.; Du, J.; Goodall, E. A.; Walsh, R. M.; rawson, S.; von Gronaua, S.; Harper, J. W.; Hanna, J.; Schulman, B. A.\nScore: 14.5, Published: 2024-01-28 DOI: 10.1101/2024.01.27.577538\nDedicated assembly factors orchestrate stepwise production of many molecular machines, including the 28-subunit proteasome core particle (CP) that mediates protein degradation. Here, we report cryo-EM reconstructions of seven recombinant human subcomplexes that visualize all five chaperones and the three active site propeptides across a wide swath of the assembly pathway. Comparison of these chaperone-bound intermediates and a matching mature CP reveals molecular mechanisms determining the order of successive subunit additions, and how proteasome subcomplexes and assembly factors structurally adapt upon progressive subunit incorporation to stabilize intermediates, facilitate the formation of subsequent intermediates, and ultimately rearrange to coordinate proteolytic activation with gated access to active sites. The structural findings reported here explain many previous biochemical and genetic observations. This work establishes a methodologic approach for structural analysis of multiprotein complex assembly intermediates, illuminates specific functions of assembly factors, and reveals conceptual principles underlying human proteasome biogenesis.\nAutomated single-cell proteomics providing sufficient proteome depth to study complex biology beyond cell type classifications\nAuthors: Ctortecka, C.; Clark, N. M.; Boyle, B.; Seth, A.; Mani, D. R.; Udeshi, N.; Carr, S. A.\nScore: 23.1, Published: 2024-01-22 DOI: 10.1101/2024.01.20.576369\nMass spectrometry (MS)-based single-cell proteomics (SCP) has gained massive attention as a viable complement to other single cell approaches. The rapid technological and computational advances in the field have pushed the boundaries of sensitivity and throughput. However, reproducible quantification of thousands of proteins within a single cell at reasonable proteome depth to characterize biological phenomena remains a challenge. To address some of those limitations we present a combination of fully automated single cell sample preparation utilizing a dedicated chip within the picolitre dispensing robot, the cellenONE. The proteoCHIP EVO 96 can be directly interfaced with the Evosep One chromatographic system for in-line desalting and highly reproducible separation with a throughput of 80 samples per day. This, in combination with the Bruker timsTOF MS instruments, demonstrates double the identifications without manual sample handling. Moreover, relative to standard high-performance liquid chromatography, the Evosep One separation provides further 2-fold improvement in protein identifications. The implementation of the newest generation timsTOF Ultra with our proteoCHIP EVO 96-based sample preparation workflow reproducibly identifies up to 4,000 proteins per single HEK-293T without a carrier or match-between runs. Our current SCP depth spans over 4 orders of magnitude and identifies over 50 biologically relevant ubiquitin ligases. We complement our highly reproducible single-cell proteomics workflow to profile hundreds of lipopolysaccharide (LPS)-perturbed THP-1 cells and identified key regulatory proteins involved in interleukin and interferon signaling. This study demonstrates that the proteoCHIP EVO 96-based SCP sample preparation with the timsTOF Ultra provides sufficient proteome depth to study complex biology beyond cell-type classifications.\nDegradome analysis to identify direct protein substrates of small-molecule degraders\nAuthors: Jochem, M.; Schrempf, A.; Wagner, L.-M.; Cisneros, J.; Ng, A.; Winter, G. E.; Krijgsveld, J.\nScore: 10.2, Published: 2024-01-28 DOI: 10.1101/2024.01.28.577572\nTargeted protein degradation (TPD) has emerged as a powerful strategy to selectively eliminate cellular proteins using small-molecule degraders, offering therapeutic promise for targeting proteins that are otherwise undruggable. However, a remaining challenge is to unambiguously identify primary TPD targets that are distinct from secondary downstream effects in the proteome. Here we introduce an approach that combines stable isotope labeling and click-chemistry for selective quantification of protein degradation by mass spectrometry, excluding confounding effects of altered transcription and translation induced by target depletion. We show that the approach efficiently operates at the time scale of TPD (hours) and we demonstrate its utility by analyzing the Cyclin K degraders dCeMM2 and dCeMM4, which induce widespread transcriptional downregulation, and the GSPT1 degrader CC-885, an inhibitor of protein translation. Additionally, we apply it to characterize compound 1, a previously uncharacterized degrader, and identify the zinc-finger protein FIZ1 as a degraded target.\nIn-Depth Mapping of DNA-PKcs Signaling Uncovers Conserved Features of Its Kinase Specificity\nAuthors: Marshall, S.; Navarro, M.; Ascencao, C.; Smolka, M. B.\nScore: 8.9, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576037\nDNA-PKcs is a DNA damage sensor kinase with established roles in DNA double-strand break repair via non-homologous end joining. Recent studies have revealed additional roles of DNA-PKcs in the regulation of transcription, translation and DNA replication. However, the substrates through which DNA-PKcs regulates these processes remain largely undefined. Here we utilized quantitative phosphoproteomics to generate a high coverage map of DNA-PKcs signaling in response to ionizing radiation and mapped its interplay with the ATM kinase. Beyond the detection of the canonical S/T-Q phosphorylation motif, we uncovered a non-canonical mode of DNA-PKcs signaling targeting S/T-{psi}-D/E motifs. Cross-species analysis in mouse pre-B and human HCT116 cell lines revealed splicing factors and transcriptional regulators phosphorylated at this novel motif, several of which contain SAP domains. These findings expand the list of DNA-PKcs and ATM substrates and establish a novel preferential phosphorylation motif for DNA-PKcs that connects it to proteins involved in nucleotide processes and interactions.\nKinases in motion: impact of protein and small molecule interactions on kinase conformations\nAuthors: Kugler, V.; Schwaighofer, S.; Feichtner, A.; Enzler, F.; Fleischmann, J.; Strich, S.; Schwarz, S.; Wilson, R.; Tschaikner, P.; Troppmair, J.; Sexl, V.; Meier, P.; Kaserer, T.; Stefan, E.\nScore: 10.2, Published: 2024-01-18 DOI: 10.1101/2024.01.11.575270\nProtein kinases act as central molecular switches in the control of cellular functions. Alterations in the regulation and function of protein kinases may provoke diseases including cancer. In this study we investigate the conformational states of such disease-associated kinases using the high sensitivity of the Kinase Conformation (KinCon)-reporter system. We first tracked BRAF-kinase activity conformation changes upon melanoma drug binding. Second, we also use the KinCon reporter technology to examine the impact of regulatory protein interactions on LKB1-kinase tumor suppressor functions. Third, we explore the conformational dynamics of RIP-kinases in response to TNF-pathway activation and small molecule interactions. Finally, we show that CDK4/6 interactions with regulatory proteins alter conformations which remain unaffected in the presence of clinically applied inhibitors. Apart from its predictive value, the KinCon technology helps identify cellular factors that impact drug efficacies. The understanding of the dynamics of full-length protein kinases when interacting with small molecule inhibitors or regulatory proteins is crucial for designing more effective therapeutic strategies.\nDirect interrogation of context-dependent GPCR activity with a universal biosensor platform\nAuthors: Janicot, R.; Maziarz, M.; Park, J.-C.; Luebbers, A.; Green, E.; Zhao, J.; Philibert, C. E.; Zhang, H.; Layne, M. D.; Wu, J. C.; Garcia-Marcos, M.\nScore: 29.0, Published: 2024-01-02 DOI: 10.1101/2024.01.02.573921\nG protein-coupled receptors (GPCRs) are the largest family of druggable proteins in the human genome, but progress in understanding and targeting them is hindered by the lack of tools to reliably measure their nuanced behavior in physiologically-relevant contexts. Here, we developed a collection of compact ONE vector G-protein Optical (ONE-GO) biosensor constructs as a scalable platform that can be conveniently deployed to measure G-protein activation by virtually any GPCR with high fidelity even when expressed endogenously in primary cells. By characterizing dozens of GPCRs across many cell types like primary cardiovascular cells or neurons, we revealed new insights into the molecular basis for G-protein coupling selectivity of GPCRs, pharmacogenomic profiles of anti-psychotics on naturally-occurring GPCR variants, and G-protein subtype signaling bias by endogenous GPCRs depending on cell type or upon inducing disease-like states. In summary, this open-source platform makes the direct interrogation of context-dependent GPCR activity broadly accessible.\n",
  "wordCount" : "2012",
  "inLanguage": "en",
  "datePublished": "2024-01-31T10:38:28Z",
  "dateModified": "2024-01-31T10:38:28Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 31, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.26.577493">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.26.577493" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.26.577493">
        <p class="paperTitle">Highly specific intracellular ubiquitination of a small molecule</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.26.577493" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.26.577493" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, W.; Garcia-Rivera, E. M.; Mitchell, D. C.; Chick, J. M.; Maetani, M.; Knapp, J. M.; Matthews, G. M.; Shirasaki, R.; de Matos Simoes, R.; Viswanathan, V.; Pulice, J. L.; Rees, M. G.; Roth, J. A.; Gygi, S. P.; Mitsiades, C. S.; Kadoch, C.; Schreiber, S. L.; Ostrem, J. M. L.</p>
        <p class="info">Score: 68.5, Published: 2024-01-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.26.577493' target='https://doi.org/10.1101/2024.01.26.577493'> 10.1101/2024.01.26.577493</a></p>
        <p class="abstract">Ubiquitin is a small, highly conserved protein that acts as a posttranslational modification in eukaryotes. Ubiquitination of proteins frequently serves as a degradation signal, marking them for disposal by the proteasome. Here, we report a novel small molecule from a diversity-oriented synthesis library, BRD1732, that is directly ubiquitinated in cells, resulting in dramatic accumulation of inactive ubiquitin monomers and polyubiquitin chains causing broad inhibition of the ubiquitin-proteasome system. Ubiquitination of BRD1732 and its associated cytotoxicity are stereospecific and dependent upon two homologous E3 ubiquitin ligases, RNF19A and RNF19B. Our finding opens the possibility for indirect ubiquitination of a target through a ubiquitinated bifunctional small molecule, and more broadly raises the potential for posttranslational modification in trans.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.26.577428">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.26.577428" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.26.577428">
        <p class="paperTitle">Large-scale characterization of drug mechanism of action using proteome-wide thermal shift assays</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.26.577428" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.26.577428" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Van Vranken, J. G.; Li, J.; Mintseris, J.; Gadzuk-Shea, M.; Gygi, S. P.; Schweppe, D.</p>
        <p class="info">Score: 30.4, Published: 2024-01-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.26.577428' target='https://doi.org/10.1101/2024.01.26.577428'> 10.1101/2024.01.26.577428</a></p>
        <p class="abstract">In response to an ever-increasing demand of new small molecules therapeutics, numerous chemical and genetic tools have been developed to interrogate compound mechanism of action. Owing to its ability to characterize compound-dependent changes in thermal stability, the proteome-wide thermal shift assay has emerged as a powerful tool in this arsenal. The most recent iterations have drastically improved the overall efficiency of these assays, providing an opportunity to screen compounds at a previously unprecedented rate. Taking advantage of this advance, we quantified 1.498 million thermal stability measurements in response to multiple classes of therapeutic and tool compounds (96 compounds in living cells and 70 compounds in lysates). When interrogating the dataset as a whole, approximately 80% of compounds (with quantifiable targets) caused a significant change in the thermal stability of an annotated target. There was also a wealth of evidence portending off-target engagement despite the extensive use of the compounds in the laboratory and/or clinic. Finally, the combined application of cell- and lysate-based assays, aided in the classification of primary (direct ligand binding) and secondary (indirect) changes in thermal stability. Overall, this study highlights the value of these assays in the drug development process by affording an unbiased and reliable assessment of compound mechanism of action.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.27.577545">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.27.577545" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.27.577545">
        <p class="paperTitle">ACKR3 Proximity Labeling Identifies Novel G protein- and Î²-arrestin-independent GPCR Interacting Proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.27.577545" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.27.577545" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hicks, C.; Gardner, J.; Eiger, D. S.; Camarda, N.; Pham, U.; Dhar, S.; Rodriguez, H.; Chundi, A.; Rajagopal, S.</p>
        <p class="info">Score: 17.9, Published: 2024-01-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.27.577545' target='https://doi.org/10.1101/2024.01.27.577545'> 10.1101/2024.01.27.577545</a></p>
        <p class="abstract">The canonical paradigm of GPCR signaling recognizes G proteins and {beta}-arrestins as the two primary transducers that promote GPCR signaling. Recent evidence suggests the atypical chemokine receptor 3 (ACKR3) does not couple to G proteins, and {beta}-arrestins are dispensable for some of its functions. Here, we employed proximity labeling to identify proteins that interact with ACKR3 in cells devoid of {beta}-arrestin. We identified proteins involved in the endocytic machinery and evaluated a subset of proteins conserved across several GPCR-based proximity labeling experiments. We discovered that the bone morphogenic protein 2-inducible kinase (BMP2K) interacts with many different GPCRs with varying dependency on {beta}-arrestin. Together, our work highlights the existence of modulators that can act independently of G proteins and {beta}-arrestins to regulate GPCR signaling and provides important evidence for other targets that may regulate GPCR signaling.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.576655">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.576655" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.576655">
        <p class="paperTitle">Mechanism of chaperone coordination during cotranslational protein folding in bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.576655" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.576655" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Roeselova, A.; Maslen, S. L.; Shivakumaraswamy, S.; Pellowe, G. A.; Howell, S.; Joshi, D.; Redmond, J.; Kjaer, S.; Skehel, M.; Balchin, D.</p>
        <p class="info">Score: 16.2, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.576655' target='https://doi.org/10.1101/2024.01.22.576655'> 10.1101/2024.01.22.576655</a></p>
        <p class="abstract">Protein folding is assisted by molecular chaperones that bind nascent polypeptides during mRNA translation. Several structurally-distinct classes of chaperone promote de novo folding, suggesting that their activities are coordinated at the ribosome. We used biochemical reconstitution and structural proteomics to explore the molecular basis for cotranslational chaperone action in bacteria. We found that chaperone binding is disfavoured close to the ribosome, allowing folding to precede chaperone recruitment. Trigger factor subsequently recognises compact folding intermediates exposing extensive non-native surface and dictates DnaJ access to nascent chains. DnaJ uses a large surface to bind structurally diverse intermediates, and recruits DnaK to sequence-diverse solvent-accessible sites. Neither Trigger factor, DnaJ nor DnaK destabilize cotranslational folding intermediates. Instead, the chaperones collaborate to create a protected space for protein maturation that extends well beyond the ribosome exit tunnel. Our findings show how the chaperone network selects and modulates cotranslational folding intermediates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.27.577538">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.27.577538" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.27.577538">
        <p class="paperTitle">Visualizing chaperone-mediated multistep assembly of the human 20S proteasome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.27.577538" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.27.577538" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Adolf, F.; Du, J.; Goodall, E. A.; Walsh, R. M.; rawson, S.; von Gronaua, S.; Harper, J. W.; Hanna, J.; Schulman, B. A.</p>
        <p class="info">Score: 14.5, Published: 2024-01-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.27.577538' target='https://doi.org/10.1101/2024.01.27.577538'> 10.1101/2024.01.27.577538</a></p>
        <p class="abstract">Dedicated assembly factors orchestrate stepwise production of many molecular machines, including the 28-subunit proteasome core particle (CP) that mediates protein degradation. Here, we report cryo-EM reconstructions of seven recombinant human subcomplexes that visualize all five chaperones and the three active site propeptides across a wide swath of the assembly pathway. Comparison of these chaperone-bound intermediates and a matching mature CP reveals molecular mechanisms determining the order of successive subunit additions, and how proteasome subcomplexes and assembly factors structurally adapt upon progressive subunit incorporation to stabilize intermediates, facilitate the formation of subsequent intermediates, and ultimately rearrange to coordinate proteolytic activation with gated access to active sites. The structural findings reported here explain many previous biochemical and genetic observations. This work establishes a methodologic approach for structural analysis of multiprotein complex assembly intermediates, illuminates specific functions of assembly factors, and reveals conceptual principles underlying human proteasome biogenesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576369">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576369" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576369">
        <p class="paperTitle">Automated single-cell proteomics providing sufficient proteome depth to study complex biology beyond cell type classifications</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576369" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576369" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ctortecka, C.; Clark, N. M.; Boyle, B.; Seth, A.; Mani, D. R.; Udeshi, N.; Carr, S. A.</p>
        <p class="info">Score: 23.1, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576369' target='https://doi.org/10.1101/2024.01.20.576369'> 10.1101/2024.01.20.576369</a></p>
        <p class="abstract">Mass spectrometry (MS)-based single-cell proteomics (SCP) has gained massive attention as a viable complement to other single cell approaches. The rapid technological and computational advances in the field have pushed the boundaries of sensitivity and throughput. However, reproducible quantification of thousands of proteins within a single cell at reasonable proteome depth to characterize biological phenomena remains a challenge. To address some of those limitations we present a combination of fully automated single cell sample preparation utilizing a dedicated chip within the picolitre dispensing robot, the cellenONE. The proteoCHIP EVO 96 can be directly interfaced with the Evosep One chromatographic system for in-line desalting and highly reproducible separation with a throughput of 80 samples per day. This, in combination with the Bruker timsTOF MS instruments, demonstrates double the identifications without manual sample handling. Moreover, relative to standard high-performance liquid chromatography, the Evosep One separation provides further 2-fold improvement in protein identifications. The implementation of the newest generation timsTOF Ultra with our proteoCHIP EVO 96-based sample preparation workflow reproducibly identifies up to 4,000 proteins per single HEK-293T without a carrier or match-between runs. Our current SCP depth spans over 4 orders of magnitude and identifies over 50 biologically relevant ubiquitin ligases. We complement our highly reproducible single-cell proteomics workflow to profile hundreds of lipopolysaccharide (LPS)-perturbed THP-1 cells and identified key regulatory proteins involved in interleukin and interferon signaling. This study demonstrates that the proteoCHIP EVO 96-based SCP sample preparation with the timsTOF Ultra provides sufficient proteome depth to study complex biology beyond cell-type classifications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.28.577572">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.28.577572" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.28.577572">
        <p class="paperTitle">Degradome analysis to identify direct protein substrates of small-molecule degraders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.28.577572" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.28.577572" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jochem, M.; Schrempf, A.; Wagner, L.-M.; Cisneros, J.; Ng, A.; Winter, G. E.; Krijgsveld, J.</p>
        <p class="info">Score: 10.2, Published: 2024-01-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.28.577572' target='https://doi.org/10.1101/2024.01.28.577572'> 10.1101/2024.01.28.577572</a></p>
        <p class="abstract">Targeted protein degradation (TPD) has emerged as a powerful strategy to selectively eliminate cellular proteins using small-molecule degraders, offering therapeutic promise for targeting proteins that are otherwise undruggable. However, a remaining challenge is to unambiguously identify primary TPD targets that are distinct from secondary downstream effects in the proteome. Here we introduce an approach that combines stable isotope labeling and click-chemistry for selective quantification of protein degradation by mass spectrometry, excluding confounding effects of altered transcription and translation induced by target depletion. We show that the approach efficiently operates at the time scale of TPD (hours) and we demonstrate its utility by analyzing the Cyclin K degraders dCeMM2 and dCeMM4, which induce widespread transcriptional downregulation, and the GSPT1 degrader CC-885, an inhibitor of protein translation. Additionally, we apply it to characterize compound 1, a previously uncharacterized degrader, and identify the zinc-finger protein FIZ1 as a degraded target.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.576037">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.576037" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.576037">
        <p class="paperTitle">In-Depth Mapping of DNA-PKcs Signaling Uncovers Conserved Features of Its Kinase Specificity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.576037" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.576037" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Marshall, S.; Navarro, M.; Ascencao, C.; Smolka, M. B.</p>
        <p class="info">Score: 8.9, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.576037' target='https://doi.org/10.1101/2024.01.17.576037'> 10.1101/2024.01.17.576037</a></p>
        <p class="abstract">DNA-PKcs is a DNA damage sensor kinase with established roles in DNA double-strand break repair via non-homologous end joining. Recent studies have revealed additional roles of DNA-PKcs in the regulation of transcription, translation and DNA replication. However, the substrates through which DNA-PKcs regulates these processes remain largely undefined. Here we utilized quantitative phosphoproteomics to generate a high coverage map of DNA-PKcs signaling in response to ionizing radiation and mapped its interplay with the ATM kinase. Beyond the detection of the canonical S/T-Q phosphorylation motif, we uncovered a non-canonical mode of DNA-PKcs signaling targeting S/T-{psi}-D/E motifs. Cross-species analysis in mouse pre-B and human HCT116 cell lines revealed splicing factors and transcriptional regulators phosphorylated at this novel motif, several of which contain SAP domains. These findings expand the list of DNA-PKcs and ATM substrates and establish a novel preferential phosphorylation motif for DNA-PKcs that connects it to proteins involved in nucleotide processes and interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575270">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575270" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575270">
        <p class="paperTitle">Kinases in motion: impact of protein and small molecule interactions on kinase conformations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575270" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575270" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kugler, V.; Schwaighofer, S.; Feichtner, A.; Enzler, F.; Fleischmann, J.; Strich, S.; Schwarz, S.; Wilson, R.; Tschaikner, P.; Troppmair, J.; Sexl, V.; Meier, P.; Kaserer, T.; Stefan, E.</p>
        <p class="info">Score: 10.2, Published: 2024-01-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575270' target='https://doi.org/10.1101/2024.01.11.575270'> 10.1101/2024.01.11.575270</a></p>
        <p class="abstract">Protein kinases act as central molecular switches in the control of cellular functions. Alterations in the regulation and function of protein kinases may provoke diseases including cancer. In this study we investigate the conformational states of such disease-associated kinases using the high sensitivity of the Kinase Conformation (KinCon)-reporter system. We first tracked BRAF-kinase activity conformation changes upon melanoma drug binding. Second, we also use the KinCon reporter technology to examine the impact of regulatory protein interactions on LKB1-kinase tumor suppressor functions. Third, we explore the conformational dynamics of RIP-kinases in response to TNF-pathway activation and small molecule interactions. Finally, we show that CDK4/6 interactions with regulatory proteins alter conformations which remain unaffected in the presence of clinically applied inhibitors. Apart from its predictive value, the KinCon technology helps identify cellular factors that impact drug efficacies. The understanding of the dynamics of full-length protein kinases when interacting with small molecule inhibitors or regulatory proteins is crucial for designing more effective therapeutic strategies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573921">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573921" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573921">
        <p class="paperTitle">Direct interrogation of context-dependent GPCR activity with a universal biosensor platform</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573921" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573921" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Janicot, R.; Maziarz, M.; Park, J.-C.; Luebbers, A.; Green, E.; Zhao, J.; Philibert, C. E.; Zhang, H.; Layne, M. D.; Wu, J. C.; Garcia-Marcos, M.</p>
        <p class="info">Score: 29.0, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573921' target='https://doi.org/10.1101/2024.01.02.573921'> 10.1101/2024.01.02.573921</a></p>
        <p class="abstract">G protein-coupled receptors (GPCRs) are the largest family of druggable proteins in the human genome, but progress in understanding and targeting them is hindered by the lack of tools to reliably measure their nuanced behavior in physiologically-relevant contexts. Here, we developed a collection of compact ONE vector G-protein Optical (ONE-GO) biosensor constructs as a scalable platform that can be conveniently deployed to measure G-protein activation by virtually any GPCR with high fidelity even when expressed endogenously in primary cells. By characterizing dozens of GPCRs across many cell types like primary cardiovascular cells or neurons, we revealed new insights into the molecular basis for G-protein coupling selectivity of GPCRs, pharmacogenomic profiles of anti-psychotics on naturally-occurring GPCR variants, and G-protein subtype signaling bias by endogenous GPCRs depending on cell type or upon inducing disease-like states. In summary, this open-source platform makes the direct interrogation of context-dependent GPCR activity broadly accessible.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
