<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A double-ring of human RAD52 remodels replication forks restricting fork reversal
Authors: Honda, M.; Razzaghi, M.; Gaur, P.; Malacaria, E.; Di Biagi, L.; Aiello, F. A.; Paintsil, E. A.; Stanfield, A.; Deppe, B. J.; Gakhar, L.; Schnicker, N. J.; Spies, M. A.; Pichierri, P.; Spies, M.
Score: 23.2, Published: 2023-11-14 DOI: 10.1101/2023.11.14.566657
Human RAD521,2 is a multifunctional DNA repair protein involved in several cellular events that support genome stability including protection of stalled DNA replication forks from excessive degradation3-7.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="A double-ring of human RAD52 remodels replication forks restricting fork reversal
Authors: Honda, M.; Razzaghi, M.; Gaur, P.; Malacaria, E.; Di Biagi, L.; Aiello, F. A.; Paintsil, E. A.; Stanfield, A.; Deppe, B. J.; Gakhar, L.; Schnicker, N. J.; Spies, M. A.; Pichierri, P.; Spies, M.
Score: 23.2, Published: 2023-11-14 DOI: 10.1101/2023.11.14.566657
Human RAD521,2 is a multifunctional DNA repair protein involved in several cellular events that support genome stability including protection of stalled DNA replication forks from excessive degradation3-7." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-19T10:38:34+00:00" />
<meta property="article:modified_time" content="2023-11-19T10:38:34+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="A double-ring of human RAD52 remodels replication forks restricting fork reversal
Authors: Honda, M.; Razzaghi, M.; Gaur, P.; Malacaria, E.; Di Biagi, L.; Aiello, F. A.; Paintsil, E. A.; Stanfield, A.; Deppe, B. J.; Gakhar, L.; Schnicker, N. J.; Spies, M. A.; Pichierri, P.; Spies, M.
Score: 23.2, Published: 2023-11-14 DOI: 10.1101/2023.11.14.566657
Human RAD521,2 is a multifunctional DNA repair protein involved in several cellular events that support genome stability including protection of stalled DNA replication forks from excessive degradation3-7."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "A double-ring of human RAD52 remodels replication forks restricting fork reversal\nAuthors: Honda, M.; Razzaghi, M.; Gaur, P.; Malacaria, E.; Di Biagi, L.; Aiello, F. A.; Paintsil, E. A.; Stanfield, A.; Deppe, B. J.; Gakhar, L.; Schnicker, N. J.; Spies, M. A.; Pichierri, P.; Spies, M.\nScore: 23.2, Published: 2023-11-14 DOI: 10.1101/2023.11.14.566657\nHuman RAD521,2 is a multifunctional DNA repair protein involved in several cellular events that support genome stability including protection of stalled DNA replication forks from excessive degradation3-7.",
  "keywords": [
    
  ],
  "articleBody": " A double-ring of human RAD52 remodels replication forks restricting fork reversal\nAuthors: Honda, M.; Razzaghi, M.; Gaur, P.; Malacaria, E.; Di Biagi, L.; Aiello, F. A.; Paintsil, E. A.; Stanfield, A.; Deppe, B. J.; Gakhar, L.; Schnicker, N. J.; Spies, M. A.; Pichierri, P.; Spies, M.\nScore: 23.2, Published: 2023-11-14 DOI: 10.1101/2023.11.14.566657\nHuman RAD521,2 is a multifunctional DNA repair protein involved in several cellular events that support genome stability including protection of stalled DNA replication forks from excessive degradation3-7. In its gatekeeper role, RAD52 binds to and stabilizes stalled replication forks during replication stress protecting them from reversal by SMARCAL15. The structural and molecular mechanism of the RAD52-mediated fork protection remains elusive. Here, using P1 nuclease sensitivity, biochemical and single-molecule analyses we show that RAD52 dynamically remodels replication forks through its strand exchange activity. The presence of the ssDNA binding protein RPA at the fork modulates the kinetics of the strand exchange without impeding the reaction outcome. Mass photometry and single-particle cryo-electron microscopy show that the replication fork promotes a unique nucleoprotein structure containing head-to-head arrangement of two undecameric RAD52 rings with an extended positively charged surface that accommodates all three arms of the replication fork. We propose that the formation and continuity of this surface is important for the strand exchange reaction and for competition with SMARCAL1.\nInsulin and Exercise-induced Phosphoproteomics of Human Skeletal Muscle Identify REPS1 as a New Regulator of Muscle Glucose Uptake\nAuthors: Larsen, J. K.; Lindqvist, C. B.; Jessen, S.; Garcia-Urena, M.; Ehrlich, A.; Schlabs, F.; Quesada, J. P.; Schmalbruch, J.; Small, L.; Thomassen, M.; Lemminger, A. K.; Eibye, K.; Gonzalez-Franquesa, A.; Stidsen, J. v.; hoejlund, k.; Kilpelainen, T.; Bangsbo, J.; Treebak, J. T.; Hostrup, M.; Deshmukh, A. S.\nScore: 13.8, Published: 2023-11-11 DOI: 10.1101/2023.11.10.566644\nSkeletal muscle regulates glucose uptake in response to insulin and exercise which is critical for maintaining metabolic health. We conducted a comprehensive phosphoproteomic analysis of skeletal muscle from healthy people in response to an acute bout of exercise or insulin stimulation by a hyperinsulinemic euglycemic clamp. Our analysis revealed 233 phosphosites regulated by both exercise and insulin of which most phosphosites were regulated in opposite directions. However, 71 phosphosites on 55 proteins displayed regulation in the same direction, indicating a potential convergence of signaling pathways. We identified the vesicle-associated protein, REPS1, to be phosphorylated at Ser709 in response to both insulin and exercise. REPS1 protein level and Ser709 phosphorylation were closely related to insulin-stimulated glucose uptake in skeletal muscle and required for maximal insulin-stimulated glucose uptake. Furthermore, we observed that insulin triggered phosphorylation of REPS1 Ser709 via P90S6 kinase (RSK) and is impaired in mice and humans with insulin resistance. Collectively, REPS1 is a convergence point for insulin and exercise signaling and a promising therapeutic target in insulin resistance.\nFrom hot to cold: dissecting lipidome adaptation in Mycoplasma mycoides and the Minimal Cell JCVI-Syn3B\nAuthors: Safronova, N.; Junghans, L.; Saenz, J. P.\nScore: 11.6, Published: 2023-11-13 DOI: 10.1101/2023.11.10.566608\nCell membranes insulate and mediate interactions between life and its environment, with lipids determining their properties and functions. However, the intricacies of how cells adjust their lipidome compositions to tune membrane properties remain relatively undefined. The complexity of most model organisms has made it challenging to characterize lipidomic adaptation. An ideal model system would be a relatively simple organism with a single membrane that can adapt to environmental changes, particularly temperature, which is known to affect membrane properties. To this end, we used quantitative shotgun lipidomics to analyze temperature adaptation in Mycoplasma mycoides and its minimal synthetic counterpart, JCVI-Syn3B. Comparing with lipidomes from eukaryotes and bacteria, we observed a universal logarithmic distribution of lipid abundances. Additionally, the extent of lipid remodeling needed for temperature adaptation appears relatively constrained, irrespective of lipidomic or organismal complexity. Through lipid features analysis, we demonstrate head group-specific acyl chain remodeling as characteristic of temperature-induced lipidome adaptation and its deficiency in Syn3B is associated with impaired homeoviscous adaptation. Temporal analysis uncovers a two-stage cold adaptation process: swift cholesterol and cardiolipin shifts followed by gradual acyl chain modifications. This work provides an in-depth analysis of lipidome adaptation in minimal cells, laying a foundation to probe the fundamental design principles of living membranes.\nConserved 5-methyluridine tRNA modification modulates ribosome translocation\nAuthors: Jones, J. D.; Franco, M. K.; Tardu, M.; Smith, T. J.; Snyder, L. R.; Eyler, D. E.; Polikanov, Y. S.; Kennedy, R. T.; Niederer, R. O.; Koutmou, K. S.\nScore: 6.0, Published: 2023-11-12 DOI: 10.1101/2023.11.12.566704\nWhile the centrality of post-transcriptional modifications to RNA biology has long been acknowledged, the function of the vast majority of modified sites remains to be discovered. Illustrative of this, there is not yet a discrete biological role assigned for one the most highly conserved modifications, 5-methyluridine at position 54 in tRNAs (m5U54). Here, we uncover contributions of m5U54 to both tRNA maturation and protein synthesis. Our mass spectrometry analyses demonstrate that cells lacking the enzyme that installs m5U in the T-loop (TrmA in E. coli, Trm2 in S. cerevisiae) exhibit altered tRNA modifications patterns. Furthermore, m5U54 deficient tRNAs are desensitized to small molecules that prevent translocation in vitro. This finding is consistent with our observations that, relative to wild-type cells, trm2{Delta} cell growth and transcriptome-wide gene expression are less perturbed by translocation inhibitors. Together our data suggest a model in which m5U54 acts as an important modulator of tRNA maturation and translocation of the ribosome during protein synthesis.\nConformational diversity in class C GPCR positive allosteric modulation\nAuthors: Cannone, G.; Berto, L.; Malhaire, F.; Ferguson, G.; Fouillen, A.; Balor, S.; Font-Ingles, J.; Llebaria, A.; GOUDET, C.; Kotecha, A.; Kutti, V. R.; Lebon, G.\nScore: 5.7, Published: 2023-11-08 DOI: 10.1101/2023.11.07.565819\nThe metabotropic glutamate receptors (mGlus) are class C G protein coupled receptors (GPCR) that form obligate dimers activated by the major excitatory neurotransmitter L-glutamate1,2. The architecture of mGlu receptor comprises an extracellular Venus-Fly Trap domain (VFT) connected to a transmembrane domain (7TM) through a Cysteine-Rich Domain (CRD). The binding of L-glutamate in the VFTs and subsequent conformational change results in the signal being transmitted to the 7TM inducing G-protein binding and activation 3-6. The mGlu receptors signal transduction can be allosterically potentiated by positive allosteric modulators (PAMs) binding to the 7TMs, which are of therapeutic interest in various neurological disorders7-9. Here, we report the cryoEM structures of metabotropic glutamate receptor 5 (mGlu5) purified with three chemically and pharmacologically distinct PAMs. We find that PAMs modulate the receptor equilibrium through their different binding modes, revealing how their interactions in the 7TMs impact the mGlu5 receptor conformational landscape and function. In addition, we identified a PAM-free but agonist-bound intermediate state that is stabilised by interactions mediated by intracellular loop 2. The activation of mGlu5 receptor is a multi-step sequential process in which the binding of the PAMs in the 7TM modulates the equilibrium towards the active state.\nSingle-Molecule Direct RNA Sequencing Reveals the Shaping of Epitranscriptome Across Multiple Species\nAuthors: Luo, G.-z.; Xie, Y.-y.; Zhong, Z.-d.; Chen, H.-x.\nScore: 4.7, Published: 2023-11-16 DOI: 10.1101/2023.11.16.567334\nThe significance of RNA modification in gene regulation has been widely recognized. To understand the transcriptome-wide landscape and its underlying mechanisms, prevailing mapping strategies have been developed. However, these short-reads based techniques are primarily focused at the gene level, overlooking the nature of RNA as multiple copies within one cell. Third-generation sequencing (TGS) platforms provide direct RNA sequencing at the resolution of individual RNA molecules, offering the promise of detecting RNA modifications and RNA processing events simultaneously. In this study, we introduce SingleMod, a deep learning model tailored for the precise mapping of m6A modifications within single RNA molecules using nanopore direct RNA sequencing (DRS). We systematically dissect the transcriptome-wide m6A profiles in single- molecule and single-base resolution, refining our understanding of the genomics of m6A and revealing an additive mode through which m6A shapes the epitranscriptome. Through comparative analyses across diverse species, we identify three distinct m6A distribution patterns and unveil an exclusion-inclusion deposition mode that governs m6A biogenesis. Furthermore, we introduce a unified quantitative model that delineates this dual-mode in various species. This study pioneers single-molecule m6A landscape exploration across multiple species, deepening our understanding of m6A, including its genomics, biogenesis, mechanisms, and biological implications.\nA liquid-to-solid phase transition of biomolecular condensates drives in vivo formation of yeast amyloids and prions\nAuthors: Alieva, M. K.; Nikishina, S. B.; Kireev, I. I.; Golyshev, S. A.; Tyurin-Kuzmin, P. A.; Ivanova, L. V.; Alexandrov, A. I.; Kushnirov, V. V.; Dergalev, A. A.\nScore: 3.6, Published: 2023-11-18 DOI: 10.1101/2023.11.17.566816\nLiquid-liquid phase separation (LLPS) and liquid-solid phase transition (LSPT) of amyloidogenic proteins are now being intensively studied as a potentially widespread mechanism of pathological amyloids formation. However, the possibility and importance of such a mechanism in living systems is still questionable. Here, we investigated the possibility of such LSPT for a series of yeast prion proteins-based constructs overproduced in yeast cells lacking any pre-existing amyloid template. By combining fluorescence and electron microscopy with biochemical and genetic approaches, we have shown that three such constructs (containing the prion domains (PDs) of either Sup35, Rnq1 or Mot3 proteins) form amyloid fibrils via the intermediate stage of liquid-like condensates, that age over time into the more solid-like hydrogels and amyloid bodies. In turn, LSPT of these constructs triggers prion conversion of the corresponding wild-type protein. Two other constructs studied (Ure2- and Sap30-based) are unable to phase separate in vivo and their amyloidogenesis is therefore strongly suppressed. Using PrK-resistant amyloid core mapping, we showed that Sup35PD amyloids formed via LSPT have a different molecular architecture compared to those formed via amyloid cross-seeding. Finally, we showed that physiological LLPS of wild-type Sup35 protein can increase its prion conversion in yeast.\nDevelopment of LRRK2 designed ankyrin-repeat proteins\nAuthors: Dederer, V.; Sanz Murillo, M.; Karasmanis, E. P.; Hatch, K. S.; Chatterjee, D.; Preuss, F.; Abdul Azeez, K. R.; Nguyen, L. V.; Galicia, C.; Dreier, B.; Pluckthun, A.; Versees, W.; Mathea, S.; Leschziner, A. E.; Reck-Peterson, S. L.; Knapp, S.\nScore: 3.2, Published: 2023-11-15 DOI: 10.1101/2023.11.14.567123\nLeucine rich repeat kinase 2 (LRRK2) is a large multidomain protein containing two catalytic domains, a kinase and a GTPase, as well protein interactions domains, including a WD40 domain. The association of increased LRRK2 kinase activity with both the familial and sporadic forms of Parkinsons disease (PD) has led to intense interest in determining its cellular function. However, small molecule probes that can bind to LRRK2 and report on or affect its activity are needed. Here, we identified a series of high-affinity LRRK2-binding designed ankyrin-repeat proteins (DARPins). One of these DARPins (E11) bound to the LRRK2 WD40 domain with high affinity. LRRK2 bound to DARPin E11 showed improved behavior on cryo-EM grids, resulting in higher resolution LRRK2 structures. DARPin E11 did not affect the catalytic activity of a truncated form of LRRK2 in vitro but decreased the phosphorylation of Rab8A, a LRRK2 substrate, in cells. We also found that DARPin E11 disrupts the formation of microtubule-associated LRRK2 filaments in cells, which are known to require WD40-based dimerization. Thus, DARPin E11 is a new tool to explore the function and dysfunction of LRRK2 and guide the development of LRRK2 kinase inhibitors that target the WD40 domain instead of the kinase.\nDevelopment of supramolecular anticoagulants with on-demand reversibility\nAuthors: Dockerill, M.; Ford, D. J.; Angerani, S.; Alwis, I.; Ripoll-Rozada, J.; Smythe, R. E.; Liu, J. S. T.; Barbosa Pereira, P. J.; Jackson, S. P.; Payne, R. J.; Winssinger, N.\nScore: 3.1, Published: 2023-11-12 DOI: 10.1101/2023.11.12.566735\nDrugs are administered at a dosing schedule set by their therapeutic index and termination of action is achieved by clearance and metabolism of the drug (hours to days for small molecules, weeks to months for biologics). In some cases, it is important to achieve a fast reversal of the drugs action to overcome dangerous side effects or in response to unforeseen events. A case in point is for anticoagulant drugs. Here we report a general strategy to achieve on-demand reversibility by leveraging supramolecular assembly of drug fragments and showcase the approach with thrombin-inhibiting anticoagulants. In our supramolecular drug design, the action of the drug is reinforced by a dynamic hybridisation of peptide nucleic acids (PNAs) between drug fragments. We show that this design enables the generation of very potent bivalent direct thrombin inhibitors (Ki 74 pM) and this inhibition can be reversed through the use of a PNA antidote. We demonstrate that these supramolecular inhibitors exhibit potent anticoagulant activity in vitro and in vivo and that this activity can also be reversed on demand.\nHigh-throughput screening assay for PARP-HPF1 interaction inhibitors to affect DNA damage repair\nAuthors: Dhakar, S. S.; Galera-Prat, A.; Lehtiö, L.\nScore: 2.4, Published: 2023-11-14 DOI: 10.1101/2023.11.14.566986\nADP-ribosyltransferases PARP1 and PARP2 play a major role in DNA repair mechanism by detecting the DNA damage and inducing poly-ADP-ribosylation dependent chromatin relaxation and recruitment of repair proteins. Catalytic PARP inhibitors are used as anticancer drugs especially in the case of tumors arising from sensitizing mutations. Recently, a study showed that Histone PARylation Factor (HPF1) forms a joint active site with PARP1/2. The interaction of HPF1 with PARP1/2 alters the automodification site from Aspartate, Glutamate to Serine, which has been shown to be a key ADP-ribosylation event in the context of DNA damage. Therefore disruption of PARP1/2-HPF1 interaction could be an alternative strategy for drug development to block the PARP1/2 activity. In this study, we describe a FRET based high-throughput screening assay to screen inhibitor libraries against PARP-HPF1 interaction. We optimized the conditions for FRET signal and verified the interaction by competing the FRET pair in multiple ways. The assay is robust and easy to automate. Validatory screening showed the robust performance of the assay, and we discovered two compounds, Dimethylacrylshikonin and Alkannin, with {micro}M inhibition potency against PARP1/2-HPF1 interaction. The assay will facilitate the discovery of inhibitors against HPF1-PARP1/2 complex and to develop potentially new effective anticancer agents.\n",
  "wordCount" : "2294",
  "inLanguage": "en",
  "datePublished": "2023-11-19T10:38:34Z",
  "dateModified": "2023-11-19T10:38:34Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 19, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.14.566657">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.14.566657" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.14.566657">
        <p class="paperTitle">A double-ring of human RAD52 remodels replication forks restricting fork reversal</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.14.566657" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.14.566657" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Honda, M.; Razzaghi, M.; Gaur, P.; Malacaria, E.; Di Biagi, L.; Aiello, F. A.; Paintsil, E. A.; Stanfield, A.; Deppe, B. J.; Gakhar, L.; Schnicker, N. J.; Spies, M. A.; Pichierri, P.; Spies, M.</p>
        <p class="info">Score: 23.2, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.14.566657' target='https://doi.org/10.1101/2023.11.14.566657'> 10.1101/2023.11.14.566657</a></p>
        <p class="abstract">Human RAD521,2 is a multifunctional DNA repair protein involved in several cellular events that support genome stability including protection of stalled DNA replication forks from excessive degradation3-7. In its gatekeeper role, RAD52 binds to and stabilizes stalled replication forks during replication stress protecting them from reversal by SMARCAL15. The structural and molecular mechanism of the RAD52-mediated fork protection remains elusive. Here, using P1 nuclease sensitivity, biochemical and single-molecule analyses we show that RAD52 dynamically remodels replication forks through its strand exchange activity. The presence of the ssDNA binding protein RPA at the fork modulates the kinetics of the strand exchange without impeding the reaction outcome. Mass photometry and single-particle cryo-electron microscopy show that the replication fork promotes a unique nucleoprotein structure containing head-to-head arrangement of two undecameric RAD52 rings with an extended positively charged surface that accommodates all three arms of the replication fork. We propose that the formation and continuity of this surface is important for the strand exchange reaction and for competition with SMARCAL1.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.566644">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.566644" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.566644">
        <p class="paperTitle">Insulin and Exercise-induced Phosphoproteomics of Human Skeletal Muscle Identify REPS1 as a New Regulator of Muscle Glucose Uptake</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.566644" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.566644" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Larsen, J. K.; Lindqvist, C. B.; Jessen, S.; Garcia-Urena, M.; Ehrlich, A.; Schlabs, F.; Quesada, J. P.; Schmalbruch, J.; Small, L.; Thomassen, M.; Lemminger, A. K.; Eibye, K.; Gonzalez-Franquesa, A.; Stidsen, J. v.; hoejlund, k.; Kilpelainen, T.; Bangsbo, J.; Treebak, J. T.; Hostrup, M.; Deshmukh, A. S.</p>
        <p class="info">Score: 13.8, Published: 2023-11-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.566644' target='https://doi.org/10.1101/2023.11.10.566644'> 10.1101/2023.11.10.566644</a></p>
        <p class="abstract">Skeletal muscle regulates glucose uptake in response to insulin and exercise which is critical for maintaining metabolic health. We conducted a comprehensive phosphoproteomic analysis of skeletal muscle from healthy people in response to an acute bout of exercise or insulin stimulation by a hyperinsulinemic euglycemic clamp. Our analysis revealed 233 phosphosites regulated by both exercise and insulin of which most phosphosites were regulated in opposite directions. However, 71 phosphosites on 55 proteins displayed regulation in the same direction, indicating a potential convergence of signaling pathways. We identified the vesicle-associated protein, REPS1, to be phosphorylated at Ser709 in response to both insulin and exercise. REPS1 protein level and Ser709 phosphorylation were closely related to insulin-stimulated glucose uptake in skeletal muscle and required for maximal insulin-stimulated glucose uptake. Furthermore, we observed that insulin triggered phosphorylation of REPS1 Ser709 via P90S6 kinase (RSK) and is impaired in mice and humans with insulin resistance. Collectively, REPS1 is a convergence point for insulin and exercise signaling and a promising therapeutic target in insulin resistance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.566608">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.566608" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.566608">
        <p class="paperTitle">From hot to cold: dissecting lipidome adaptation in Mycoplasma mycoides and the Minimal Cell JCVI-Syn3B</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.566608" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.566608" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Safronova, N.; Junghans, L.; Saenz, J. P.</p>
        <p class="info">Score: 11.6, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.566608' target='https://doi.org/10.1101/2023.11.10.566608'> 10.1101/2023.11.10.566608</a></p>
        <p class="abstract">Cell membranes insulate and mediate interactions between life and its environment, with lipids determining their properties and functions. However, the intricacies of how cells adjust their lipidome compositions to tune membrane properties remain relatively undefined. The complexity of most model organisms has made it challenging to characterize lipidomic adaptation. An ideal model system would be a relatively simple organism with a single membrane that can adapt to environmental changes, particularly temperature, which is known to affect membrane properties. To this end, we used quantitative shotgun lipidomics to analyze temperature adaptation in Mycoplasma mycoides and its minimal synthetic counterpart, JCVI-Syn3B. Comparing with lipidomes from eukaryotes and bacteria, we observed a universal logarithmic distribution of lipid abundances. Additionally, the extent of lipid remodeling needed for temperature adaptation appears relatively constrained, irrespective of lipidomic or organismal complexity. Through lipid features analysis, we demonstrate head group-specific acyl chain remodeling as characteristic of temperature-induced lipidome adaptation and its deficiency in Syn3B is associated with impaired homeoviscous adaptation. Temporal analysis uncovers a two-stage cold adaptation process: swift cholesterol and cardiolipin shifts followed by gradual acyl chain modifications. This work provides an in-depth analysis of lipidome adaptation in minimal cells, laying a foundation to probe the fundamental design principles of living membranes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.12.566704">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.12.566704" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.12.566704">
        <p class="paperTitle">Conserved 5-methyluridine tRNA modification modulates ribosome translocation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.12.566704" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.12.566704" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jones, J. D.; Franco, M. K.; Tardu, M.; Smith, T. J.; Snyder, L. R.; Eyler, D. E.; Polikanov, Y. S.; Kennedy, R. T.; Niederer, R. O.; Koutmou, K. S.</p>
        <p class="info">Score: 6.0, Published: 2023-11-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.12.566704' target='https://doi.org/10.1101/2023.11.12.566704'> 10.1101/2023.11.12.566704</a></p>
        <p class="abstract">While the centrality of post-transcriptional modifications to RNA biology has long been acknowledged, the function of the vast majority of modified sites remains to be discovered. Illustrative of this, there is not yet a discrete biological role assigned for one the most highly conserved modifications, 5-methyluridine at position 54 in tRNAs (m5U54). Here, we uncover contributions of m5U54 to both tRNA maturation and protein synthesis. Our mass spectrometry analyses demonstrate that cells lacking the enzyme that installs m5U in the T-loop (TrmA in E. coli, Trm2 in S. cerevisiae) exhibit altered tRNA modifications patterns. Furthermore, m5U54 deficient tRNAs are desensitized to small molecules that prevent translocation in vitro. This finding is consistent with our observations that, relative to wild-type cells, trm2{Delta} cell growth and transcriptome-wide gene expression are less perturbed by translocation inhibitors. Together our data suggest a model in which m5U54 acts as an important modulator of tRNA maturation and translocation of the ribosome during protein synthesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.565819">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.565819" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.565819">
        <p class="paperTitle">Conformational diversity in class C GPCR positive allosteric modulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.565819" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.565819" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cannone, G.; Berto, L.; Malhaire, F.; Ferguson, G.; Fouillen, A.; Balor, S.; Font-Ingles, J.; Llebaria, A.; GOUDET, C.; Kotecha, A.; Kutti, V. R.; Lebon, G.</p>
        <p class="info">Score: 5.7, Published: 2023-11-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.565819' target='https://doi.org/10.1101/2023.11.07.565819'> 10.1101/2023.11.07.565819</a></p>
        <p class="abstract">The metabotropic glutamate receptors (mGlus) are class C G protein coupled receptors (GPCR) that form obligate dimers activated by the major excitatory neurotransmitter L-glutamate1,2. The architecture of mGlu receptor comprises an extracellular Venus-Fly Trap domain (VFT) connected to a transmembrane domain (7TM) through a Cysteine-Rich Domain (CRD). The binding of L-glutamate in the VFTs and subsequent conformational change results in the signal being transmitted to the 7TM inducing G-protein binding and activation 3-6. The mGlu receptors signal transduction can be allosterically potentiated by positive allosteric modulators (PAMs) binding to the 7TMs, which are of therapeutic interest in various neurological disorders7-9. Here, we report the cryoEM structures of metabotropic glutamate receptor 5 (mGlu5) purified with three chemically and pharmacologically distinct PAMs. We find that PAMs modulate the receptor equilibrium through their different binding modes, revealing how their interactions in the 7TMs impact the mGlu5 receptor conformational landscape and function. In addition, we identified a PAM-free but agonist-bound intermediate state that is stabilised by interactions mediated by intracellular loop 2. The activation of mGlu5 receptor is a multi-step sequential process in which the binding of the PAMs in the 7TM modulates the equilibrium towards the active state.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.567334">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.567334" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.567334">
        <p class="paperTitle">Single-Molecule Direct RNA Sequencing Reveals the Shaping of Epitranscriptome Across Multiple Species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.567334" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.567334" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Luo, G.-z.; Xie, Y.-y.; Zhong, Z.-d.; Chen, H.-x.</p>
        <p class="info">Score: 4.7, Published: 2023-11-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.567334' target='https://doi.org/10.1101/2023.11.16.567334'> 10.1101/2023.11.16.567334</a></p>
        <p class="abstract">The significance of RNA modification in gene regulation has been widely recognized. To understand the transcriptome-wide landscape and its underlying mechanisms, prevailing mapping strategies have been developed. However, these short-reads based techniques are primarily focused at the gene level, overlooking the nature of RNA as multiple copies within one cell. Third-generation sequencing (TGS) platforms provide direct RNA sequencing at the resolution of individual RNA molecules, offering the promise of detecting RNA modifications and RNA processing events simultaneously. In this study, we introduce SingleMod, a deep learning model tailored for the precise mapping of m6A modifications within single RNA molecules using nanopore direct RNA sequencing (DRS). We systematically dissect the transcriptome-wide m6A profiles in single- molecule and single-base resolution, refining our understanding of the genomics of m6A and revealing an additive mode through which m6A shapes the epitranscriptome. Through comparative analyses across diverse species, we identify three distinct m6A distribution patterns and unveil an exclusion-inclusion deposition mode that governs m6A biogenesis. Furthermore, we introduce a unified quantitative model that delineates this dual-mode in various species. This study pioneers single-molecule m6A landscape exploration across multiple species, deepening our understanding of m6A, including its genomics, biogenesis, mechanisms, and biological implications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.17.566816">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.17.566816" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.17.566816">
        <p class="paperTitle">A liquid-to-solid phase transition of biomolecular condensates drives in vivo formation of yeast amyloids and prions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.17.566816" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.17.566816" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alieva, M. K.; Nikishina, S. B.; Kireev, I. I.; Golyshev, S. A.; Tyurin-Kuzmin, P. A.; Ivanova, L. V.; Alexandrov, A. I.; Kushnirov, V. V.; Dergalev, A. A.</p>
        <p class="info">Score: 3.6, Published: 2023-11-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.17.566816' target='https://doi.org/10.1101/2023.11.17.566816'> 10.1101/2023.11.17.566816</a></p>
        <p class="abstract">Liquid-liquid phase separation (LLPS) and liquid-solid phase transition (LSPT) of amyloidogenic proteins are now being intensively studied as a potentially widespread mechanism of pathological amyloids formation. However, the possibility and importance of such a mechanism in living systems is still questionable. Here, we investigated the possibility of such LSPT for a series of yeast prion proteins-based constructs overproduced in yeast cells lacking any pre-existing amyloid template. By combining fluorescence and electron microscopy with biochemical and genetic approaches, we have shown that three such constructs (containing the prion domains (PDs) of either Sup35, Rnq1 or Mot3 proteins) form amyloid fibrils via the intermediate stage of liquid-like condensates, that age over time into the more solid-like hydrogels and amyloid bodies. In turn, LSPT of these constructs triggers prion conversion of the corresponding wild-type protein. Two other constructs studied (Ure2- and Sap30-based) are unable to phase separate in vivo and their amyloidogenesis is therefore strongly suppressed. Using PrK-resistant amyloid core mapping, we showed that Sup35PD amyloids formed via LSPT have a different molecular architecture compared to those formed via amyloid cross-seeding. Finally, we showed that physiological LLPS of wild-type Sup35 protein can increase its prion conversion in yeast.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.14.567123">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.14.567123" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.14.567123">
        <p class="paperTitle">Development of LRRK2 designed ankyrin-repeat proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.14.567123" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.14.567123" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dederer, V.; Sanz Murillo, M.; Karasmanis, E. P.; Hatch, K. S.; Chatterjee, D.; Preuss, F.; Abdul Azeez, K. R.; Nguyen, L. V.; Galicia, C.; Dreier, B.; Pluckthun, A.; Versees, W.; Mathea, S.; Leschziner, A. E.; Reck-Peterson, S. L.; Knapp, S.</p>
        <p class="info">Score: 3.2, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.14.567123' target='https://doi.org/10.1101/2023.11.14.567123'> 10.1101/2023.11.14.567123</a></p>
        <p class="abstract">Leucine rich repeat kinase 2 (LRRK2) is a large multidomain protein containing two catalytic domains, a kinase and a GTPase, as well protein interactions domains, including a WD40 domain. The association of increased LRRK2 kinase activity with both the familial and sporadic forms of Parkinsons disease (PD) has led to intense interest in determining its cellular function. However, small molecule probes that can bind to LRRK2 and report on or affect its activity are needed. Here, we identified a series of high-affinity LRRK2-binding designed ankyrin-repeat proteins (DARPins). One of these DARPins (E11) bound to the LRRK2 WD40 domain with high affinity. LRRK2 bound to DARPin E11 showed improved behavior on cryo-EM grids, resulting in higher resolution LRRK2 structures. DARPin E11 did not affect the catalytic activity of a truncated form of LRRK2 in vitro but decreased the phosphorylation of Rab8A, a LRRK2 substrate, in cells. We also found that DARPin E11 disrupts the formation of microtubule-associated LRRK2 filaments in cells, which are known to require WD40-based dimerization. Thus, DARPin E11 is a new tool to explore the function and dysfunction of LRRK2 and guide the development of LRRK2 kinase inhibitors that target the WD40 domain instead of the kinase.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.12.566735">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.12.566735" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.12.566735">
        <p class="paperTitle">Development of supramolecular anticoagulants with on-demand reversibility</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.12.566735" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.12.566735" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dockerill, M.; Ford, D. J.; Angerani, S.; Alwis, I.; Ripoll-Rozada, J.; Smythe, R. E.; Liu, J. S. T.; Barbosa Pereira, P. J.; Jackson, S. P.; Payne, R. J.; Winssinger, N.</p>
        <p class="info">Score: 3.1, Published: 2023-11-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.12.566735' target='https://doi.org/10.1101/2023.11.12.566735'> 10.1101/2023.11.12.566735</a></p>
        <p class="abstract">Drugs are administered at a dosing schedule set by their therapeutic index and termination of action is achieved by clearance and metabolism of the drug (hours to days for small molecules, weeks to months for biologics). In some cases, it is important to achieve a fast reversal of the drugs action to overcome dangerous side effects or in response to unforeseen events. A case in point is for anticoagulant drugs. Here we report a general strategy to achieve on-demand reversibility by leveraging supramolecular assembly of drug fragments and showcase the approach with thrombin-inhibiting anticoagulants. In our supramolecular drug design, the action of the drug is reinforced by a dynamic hybridisation of peptide nucleic acids (PNAs) between drug fragments. We show that this design enables the generation of very potent bivalent direct thrombin inhibitors (Ki 74 pM) and this inhibition can be reversed through the use of a PNA antidote. We demonstrate that these supramolecular inhibitors exhibit potent anticoagulant activity in vitro and in vivo and that this activity can also be reversed on demand.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.14.566986">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.14.566986" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.14.566986">
        <p class="paperTitle">High-throughput screening assay for PARP-HPF1 interaction inhibitors to affect DNA damage repair</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.14.566986" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.14.566986" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dhakar, S. S.; Galera-Prat, A.; Lehtiö, L.</p>
        <p class="info">Score: 2.4, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.14.566986' target='https://doi.org/10.1101/2023.11.14.566986'> 10.1101/2023.11.14.566986</a></p>
        <p class="abstract">ADP-ribosyltransferases PARP1 and PARP2 play a major role in DNA repair mechanism by detecting the DNA damage and inducing poly-ADP-ribosylation dependent chromatin relaxation and recruitment of repair proteins. Catalytic PARP inhibitors are used as anticancer drugs especially in the case of tumors arising from sensitizing mutations. Recently, a study showed that Histone PARylation Factor (HPF1) forms a joint active site with PARP1/2. The interaction of HPF1 with PARP1/2 alters the automodification site from Aspartate, Glutamate to Serine, which has been shown to be a key ADP-ribosylation event in the context of DNA damage. Therefore disruption of PARP1/2-HPF1 interaction could be an alternative strategy for drug development to block the PARP1/2 activity. In this study, we describe a FRET based high-throughput screening assay to screen inhibitor libraries against PARP-HPF1 interaction. We optimized the conditions for FRET signal and verified the interaction by competing the FRET pair in multiple ways. The assay is robust and easy to automate. Validatory screening showed the robust performance of the assay, and we discovered two compounds, Dimethylacrylshikonin and Alkannin, with {micro}M inhibition potency against PARP1/2-HPF1 interaction. The assay will facilitate the discovery of inhibitors against HPF1-PARP1/2 complex and to develop potentially new effective anticancer agents.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
