<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Protein Ensemble Generation through VariationalAutoencoder Latent Space Sampling
Authors: Mansoor, S.; Baek, M.; Park, H.; Lee, G. R.; Baker, D.
Score: 28.9, Published: 2023-08-01 DOI: 10.1101/2023.08.01.551540
Mapping the ensemble of protein conformations that contribute to function and can be targeted by small molecule drugs remains an outstanding challenge. Here we explore the use of soft-introspective variational autoencoders for reducing the challenge of dimensionality in the protein structure ensemble generation problem.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Protein Ensemble Generation through VariationalAutoencoder Latent Space Sampling
Authors: Mansoor, S.; Baek, M.; Park, H.; Lee, G. R.; Baker, D.
Score: 28.9, Published: 2023-08-01 DOI: 10.1101/2023.08.01.551540
Mapping the ensemble of protein conformations that contribute to function and can be targeted by small molecule drugs remains an outstanding challenge. Here we explore the use of soft-introspective variational autoencoders for reducing the challenge of dimensionality in the protein structure ensemble generation problem." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-09T10:39:33+00:00" />
<meta property="article:modified_time" content="2023-08-09T10:39:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Protein Ensemble Generation through VariationalAutoencoder Latent Space Sampling
Authors: Mansoor, S.; Baek, M.; Park, H.; Lee, G. R.; Baker, D.
Score: 28.9, Published: 2023-08-01 DOI: 10.1101/2023.08.01.551540
Mapping the ensemble of protein conformations that contribute to function and can be targeted by small molecule drugs remains an outstanding challenge. Here we explore the use of soft-introspective variational autoencoders for reducing the challenge of dimensionality in the protein structure ensemble generation problem."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Protein Ensemble Generation through VariationalAutoencoder Latent Space Sampling\nAuthors: Mansoor, S.; Baek, M.; Park, H.; Lee, G. R.; Baker, D.\nScore: 28.9, Published: 2023-08-01 DOI: 10.1101/2023.08.01.551540\nMapping the ensemble of protein conformations that contribute to function and can be targeted by small molecule drugs remains an outstanding challenge. Here we explore the use of soft-introspective variational autoencoders for reducing the challenge of dimensionality in the protein structure ensemble generation problem.",
  "keywords": [
    
  ],
  "articleBody": " Protein Ensemble Generation through VariationalAutoencoder Latent Space Sampling\nAuthors: Mansoor, S.; Baek, M.; Park, H.; Lee, G. R.; Baker, D.\nScore: 28.9, Published: 2023-08-01 DOI: 10.1101/2023.08.01.551540\nMapping the ensemble of protein conformations that contribute to function and can be targeted by small molecule drugs remains an outstanding challenge. Here we explore the use of soft-introspective variational autoencoders for reducing the challenge of dimensionality in the protein structure ensemble generation problem. We convert high-dimensional protein structural data into a continuous, low-dimensional representation, carry out search in this space guided by a structure quality metric, then use RoseTTAFold to generate 3D structures. We use this approach to generate ensembles for the cancer relevant protein K-Ras, training the VAE on a subset of the available K-Ras crystal structures and MD simulation snapshots, and assessing the extent of sampling close to crystal structures withheld from training. We find that our latent space sampling procedure rapidly generates ensembles with high structural quality and is able to sample within 1 angstrom of held out crystal structures, with a consistency higher than MD simulation or AlphaFold2 prediction. The sampled structures sufficiently recapitulate the cryptic pockets in the held-out K-Ras structures to allow for small molecule docking.\nRapid and automated design of two-component protein nanomaterials using ProteinMPNN\nAuthors: de Haas, R. J.; Brunette, N.; Goodson, A.; Dauparas, J.; Yi, S. Y.; Yang, E. C.; Dowling, Q.; Nguyen, H.; Kang, A.; Bera, A. K.; Sankaran, B.; de Vries, R.; Baker, D.; King, N. P.\nScore: 23.6, Published: 2023-08-04 DOI: 10.1101/2023.08.04.551935\nThe design of novel protein-protein interfaces using physics-based design methods such as Rosetta requires substantial computational resources and manual refinement by expert structural biologists. A new generation of deep learning methods promises to simplify protein-protein interface design and enable its application to a wide variety of problems by researchers from various scientific disciplines. Here we test the ability of a deep learning method for protein sequence design, ProteinMPNN, to design two-component tetrahedral protein nanomaterials and benchmark its performance against Rosetta. ProteinMPNN had a similar success rate to Rosetta, yielding 13 new experimentally confirmed assemblies, but required orders of magnitude less computation and no manual refinement. The interfaces designed by ProteinMPNN were substantially more polar than those designed by Rosetta, which facilitated in vitro assembly of the designed nanomaterials from independently purified components. Crystal structures of several of the assemblies confirmed the accuracy of the design method at high resolution. Our results showcase the potential of deep learning-based methods to unlock the widespread application of designed protein-protein interfaces and self-assembling protein nanomaterials in biotechnology.\nIdentification of factors that prevent ribosome stalling during early elongation\nAuthors: Hong, H.-R.; Prince, C. R.; Wu, L.; Feaga, H. A.\nScore: 21.2, Published: 2023-08-04 DOI: 10.1101/2023.08.04.552005\nProtein synthesis is catalyzed by the ribosome and a host of highly conserved elongation factors. Most elongation factors that are conserved in all domains of life are essential, such as EF-Tu (e/aEF1A) and EF-G (e/aEF2). In contrast, the universally conserved elongation factor P (EF-P/eIF5A) is essential in eukaryotes but is dispensable in most bacteria. EF-P prevents ribosome stalling at difficult-to translate sequences, especially polyprolines. Since efp deletion phenotypes range from modest to lethal in different bacterial species, we hypothesized that some bacteria encode an uncharacterized elongation factor with compensatory functions. To identify this factor, we used Tn-seq to screen for genes that are essential in the absence of EF-P in Bacillus subtilis. This screen identified YfmR, a member of the ABCF family of ATPases, as a translation factor that is essential when efp is deleted. We find that depleting YfmR from {Delta}efp cells decreases actively translating ribosomes and increases free ribosomal subunits bound to initiator tRNA, suggesting that ribosomes stall in early elongation. We also find that deleting efp from the spore-forming pathogen Bacillus anthracis causes a severe survival defect in liquid culture and in the presence of macrophages. B. anthracis {Delta}efp also does not produce detectable spores. We find that heterologous expression of B. subtilis YfmR in B. anthracis {Delta}efp cells partially rescues the severe growth and sporulation defects of this mutant. Our results therefore identify YfmR as an important translation factor that becomes essential in the absence of EF-P, and suggest that the essential function of YfmR and EF-P is to assist translation during early elongation. SignificanceTranslation is one of the most ancient and energetically demanding process that occurs in the cell. Ribosomes constitute more than 60% of cellular mass in actively growing cells, and ribosomes are a major target of antimicrobials and chemotherapeutics. Here, we report evidence that both EF-P and the uncharacterized protein YfmR are important for early translation elongation in bacteria, and that this activity is essential. YfmR is a member of the ABCF family of ATPases whose role in translation is only beginning to be understood. Given the broad distribution of ABCFs from bacteria to fungi, we expect our results to have implications for understanding the crucial first steps of translation elongation in diverse organisms.\nMesoscale DNA Features Impact APOBEC3A and APOBEC3B Deaminase Activity and Shape Tumor Mutational Landscapes\nAuthors: Sanchez, A.; Ortega, P.; Sakhtemani, R.; Manjunath, L.; Oh, S.; Bournique, E.; Becker, A.; Kim, K.; Durfee, C.; Temiz, N. A.; Chen, X.; Harris, R.; Lawrence, M.; Buisson, R.\nScore: 11.9, Published: 2023-08-02 DOI: 10.1101/2023.08.02.551499\nAntiviral DNA cytosine deaminases APOBEC3A and APOBEC3B are major sources of mutations in cancer by catalyzing cytosine-to-uracil deamination. APOBEC3A preferentially targets singlestranded DNAs, with a noted affinity for DNA regions that adopt stem-loop secondary structures. However, the detailed substrate preferences of APOBEC3A and APOBEC3B have been fully established, and the specific influence of the DNA sequence on APOBEC3A APOBEC3B deaminase activity remains to be investigated. Here, we find that APOBEC3B selectively targets DNA stem-loop structures, and they are distinct from those subjected deamination by APOBEC3A. We develop Oligo-seq, a novel in vitro sequencing-based to identify specific sequence contexts promoting APOBEC3A and APOBEC3B activity. Through this approach, we demonstrate that APOBEC3A an APOBEC3B deaminase activity is strongly regulated by specific sequences surrounding the targeted cytosine. Moreover, we identify structural features of APOBEC3B and APOBEC3A responsible for their substrate preferences. Importantly, we determine that APOBEC3B-induced mutations in hairpin-forming sequences within tumor genomes differ from the DNA stem-loop sequences mutated by APOBEC3A. Together, our study provides evidence that APOBEC3A and APOBEC3B can generate mutation landscapes in cancer genomes, driven by their unique substrate selectivity.\nAcetic acid is a superior acidifier for sub-nanogram and single cell proteomic studies\nAuthors: Orsburn, B. C.\nScore: 11.1, Published: 2023-08-02 DOI: 10.1101/2023.08.01.551522\nA recent study demonstrated a substantial signal increase when employing a 0.5% acetic acid buffer additive instead of the traditional 0.1% formic acid used in shotgun proteomics. In this study I compare these two buffers for a dilution series of tryptic digests down to 20 picograms peptide on column on a TIMSTOF single cell proteome (SCP) system. I observe a comparable relative level of signal increase as previously reported, which translates to improvements in proteome coverage at every peptide load assessed. The relative increase in peptide identifications is more apparent at lower concentrations with a striking 1.8-fold more peptides identified at 20 pg peptide load, resulting in over 2,000 protein groups identified in 30 minutes on this system. These results translate well to isolated single human cancer cells allowing over 1,000 protein groups to be identified in single human cells processed using a simple one step method in standard 96-well plates. All vendor raw and processed data has been made publicly available at www.massive.ucsd.edu and can be accessed as MSV000092563. Abstract graphic O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=87 SRC=\"FIGDIR/small/551522v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (38K): org.highwire.dtl.DTLVardef@1f7eb2aorg.highwire.dtl.DTLVardef@17047b9org.highwire.dtl.DTLVardef@74e64corg.highwire.dtl.DTLVardef@bbafd1_HPS_FORMAT_FIGEXP M_FIG C_FIG\nMolecular insights into the activation of Mre11-Rad50 endonuclease activity by Sae2/CtIP\nAuthors: Nicolas, Y.; Bret, H.; Cannavo, E.; Cejka, P.; Borde, V.; Guerois, R.\nScore: 10.4, Published: 2023-08-05 DOI: 10.1101/2023.08.05.552090\nIn S. cerevisiae, the Mre11-Rad50-Xrs2 (MRX)-Sae2 nuclease activity is required for the resection of DNA breaks with secondary structures or protein blocks, while in humans, the MRE11-RAD50-NBS1 (MRN) homologue with CtIP is needed to initiate DNA end resection of all breaks. Phosphorylated Sae2/CtIP stimulates the endonuclease activity of MRX/N. Structural insights into the activation of the Mre11 nuclease are available only for organisms lacking Sae2/CtIP, so little is known how Sae2/CtIP activates the nuclease ensemble. Here, we uncover the mechanism of Mre11 activation by Sae2 using a combination of AlphaFold2 structural modeling, biochemical and genetic assays. We show that Sae2 stabilizes the Mre11 nuclease in a conformation poised to cleave substrate DNA. Several designs of compensatory mutations establish how Sae2 activates MRX in vitro and in vivo, validating the structural model. Last, our study uncovers how human CtIP, despite considerable sequence divergence, employs a similar mechanism to activate MRN.\nMicroED as a powerful tool for structure determination of macrocyclic drug compounds directly from their powder formulations\nAuthors: Danelius, E.; Bu, G.; Wieske, L.; Gonen, T.\nScore: 9.2, Published: 2023-08-01 DOI: 10.1101/2023.07.31.551405\nMacrocycles are important drug leads with many advantages including the ability to target flat and featureless binding sites as well as act as molecular chameleons and thereby reach intracellular targets. However, due to their complex structures and inherent flexibility, macrocycles are difficult to study structurally and there are limited structural data available. Herein, we use the cryo-EM method MicroED to determine the novel atomic structures of several macrocycles which have previously resisted structural determination. We show that structures of similar complexity can now be obtained rapidly from nanograms of material, and that different conformations of flexible compounds can be derived from the same experiment. These results will have impact on contemporary drug discovery as well as natural product exploration.\nStrategic engineering unlocks in vitro type II polyketide biosynthesis\nAuthors: Li, K. K.; Cho, Y. I.; Tran, M. A.; Wiedemann, C.; Koweek, R. S.; Hoang, N. K.; Hamrick, G.; Bowen, M.; Kokona, B.; Beld, J.; Hellmich, U.; Charkoudian, L. K.\nScore: 8.8, Published: 2023-08-02 DOI: 10.1101/2023.08.02.551649\nMicrobial type II polyketides serve as powerful medicinally relevant agents. These molecules are biosynthesized by polyketide synthases (PKSs) comprised of a core ketosynthase-chain length factor (KS-CLF) and phosphopantetheinylated acyl carrier protein (holo-ACP). While engineering type II PKSs holds potential to unlock sustainable access to diverse bioactive molecules, the inability to obtain cognate type II KS-CLFs and holo-ACPs for in vitro studies represents a longstanding barrier. Herein, we share how the sequence and structural analysis of the Gloeocapsa sp. PCC 7428 ACP allowed us to tune to a requisite weak yet specific interaction with a phosphopantetheinyl transferase to afford the holo-ACP. This, coupled with our ability to heterologously express the cognate KS-CLF in high quantities, unlocked access to polyketide products via in vitro multienzyme assembly. We hope this work inspires future studies of type II PKSs that have previously evaded heterologous expression or have yet to be explored. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=90 SRC=\"FIGDIR/small/551649v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (25K): org.highwire.dtl.DTLVardef@125f7acorg.highwire.dtl.DTLVardef@1a765ccorg.highwire.dtl.DTLVardef@10c57cdorg.highwire.dtl.DTLVardef@1588db6_HPS_FORMAT_FIGEXP M_FIG C_FIG\nDe novo Protein Sequence Design Based on Deep Learning and Validation on CalB Hydrolase\nAuthors: Mu, J.; Li, Z.; Zhang, B.; Zhang, Q.; Iqbal, J.; Wadood, A.; Wei, T.; Feng, Y.; Chen, H.\nScore: 6.2, Published: 2023-08-01 DOI: 10.1101/2023.08.01.551444\nProtein design is central to nearly all protein engineering problems, as it can enable the creation of proteins with new biological function, such as improving the catalytic efficiency of enzymes. As one of the key tasks of protein design, fixed-backbone protein sequence design aims to design novel sequence that would fold into a given protein backbone structure. However, current sequence design methods have limitations in terms of low sequence diversity and experimental validation of designed protein function, which cannot meet the needs of functional protein design. We firstly constructed Graphormer-based Protein Design (GPD) model that directly applies Transformer to graph-based representation of 3D protein structure, and added Gaussian noise and sequence random mask to node features to improve the sequence recovery and diversity. Additionally, functional filtering based on the structure folding, solubility, and function were performed to improve the success rate in experiments. The process of \"sequence design-functional filtering -functional experiment\" was carried out for CalB hydrolase. The experimental results showed that the specify activity of designed protein improved 1.7 times than CalB wild type. This design and filtering platform will be a valuable tool for generating industrial enzymes and protein drugs with specific functions.\nProbing Bioactive Chemical Space to Discover RNA-Targeted Small Molecules\nAuthors: Wicks, S. L.; Morgan, B. S.; Wilson, A. W.; Hargrove, A. E.\nScore: 7.0, Published: 2023-07-31 DOI: 10.1101/2023.07.31.551350\nSmall molecules have become increasingly recognized as invaluable tools to study RNA structure and function and to develop RNA-targeted therapeutics. To rationally design RNA-targeting ligands, a comprehensive understanding and explicit testing of small molecule properties that govern molecular recognition is crucial. To date, most studies have primarily evaluated properties of small molecules that bind RNA in vitro, with little to no assessment of properties that are distinct to selective and bioactive RNA-targeted ligands. Therefore, we curated an RNA-focused library, termed the Duke RNA-Targeted Library (DRTL), that was biased towards the physicochemical and structural properties of biologically active and non-ribosomal RNA-targeted small molecules. The DRTL represents one of the largest academic RNA-focused small molecule libraries curated to date with more than 800 small molecules. These ligands were selected using computational approaches that measure similarity to known bioactive RNA ligands and that diversify the molecules within this space. We evaluated DRTL binding in vitro to a panel of four RNAs using two optimized fluorescent indicator displacement assays, and we successfully identified multiple small molecule hits, including several novel scaffolds for RNA. The DRTL has and will continue to provide insights into biologically relevant RNA chemical space, such as the identification of additional RNA-privileged scaffolds and validation of RNA-privileged molecular features. Future DRTL screening will focus on expanding both the targets and assays used, and we welcome collaboration from the scientific community. We envision that the DRTL will be a valuable resource for the discovery of RNA-targeted chemical probes and therapeutic leads.\n",
  "wordCount" : "2310",
  "inLanguage": "en",
  "datePublished": "2023-08-09T10:39:33Z",
  "dateModified": "2023-08-09T10:39:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta"><span>updated on August 9, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.551540">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.551540" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.551540">
        <p class="paperTitle">Protein Ensemble Generation through VariationalAutoencoder Latent Space Sampling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.551540" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.551540" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mansoor, S.; Baek, M.; Park, H.; Lee, G. R.; Baker, D.</p>
        <p class="info">Score: 28.9, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.551540' target='https://doi.org/10.1101/2023.08.01.551540'> 10.1101/2023.08.01.551540</a></p>
        <p class="abstract">Mapping the ensemble of protein conformations that contribute to function and can be targeted by small molecule drugs remains an outstanding challenge. Here we explore the use of soft-introspective variational autoencoders for reducing the challenge of dimensionality in the protein structure ensemble generation problem. We convert high-dimensional protein structural data into a continuous, low-dimensional representation, carry out search in this space guided by a structure quality metric, then use RoseTTAFold to generate 3D structures. We use this approach to generate ensembles for the cancer relevant protein K-Ras, training the VAE on a subset of the available K-Ras crystal structures and MD simulation snapshots, and assessing the extent of sampling close to crystal structures withheld from training. We find that our latent space sampling procedure rapidly generates ensembles with high structural quality and is able to sample within 1 angstrom of held out crystal structures, with a consistency higher than MD simulation or AlphaFold2 prediction. The sampled structures sufficiently recapitulate the cryptic pockets in the held-out K-Ras structures to allow for small molecule docking.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.551935">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.551935" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.551935">
        <p class="paperTitle">Rapid and automated design of two-component protein nanomaterials using ProteinMPNN</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.551935" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.551935" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: de Haas, R. J.; Brunette, N.; Goodson, A.; Dauparas, J.; Yi, S. Y.; Yang, E. C.; Dowling, Q.; Nguyen, H.; Kang, A.; Bera, A. K.; Sankaran, B.; de Vries, R.; Baker, D.; King, N. P.</p>
        <p class="info">Score: 23.6, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.551935' target='https://doi.org/10.1101/2023.08.04.551935'> 10.1101/2023.08.04.551935</a></p>
        <p class="abstract">The design of novel protein-protein interfaces using physics-based design methods such as Rosetta requires substantial computational resources and manual refinement by expert structural biologists. A new generation of deep learning methods promises to simplify protein-protein interface design and enable its application to a wide variety of problems by researchers from various scientific disciplines. Here we test the ability of a deep learning method for protein sequence design, ProteinMPNN, to design two-component tetrahedral protein nanomaterials and benchmark its performance against Rosetta. ProteinMPNN had a similar success rate to Rosetta, yielding 13 new experimentally confirmed assemblies, but required orders of magnitude less computation and no manual refinement. The interfaces designed by ProteinMPNN were substantially more polar than those designed by Rosetta, which facilitated in vitro assembly of the designed nanomaterials from independently purified components. Crystal structures of several of the assemblies confirmed the accuracy of the design method at high resolution. Our results showcase the potential of deep learning-based methods to unlock the widespread application of designed protein-protein interfaces and self-assembling protein nanomaterials in biotechnology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.552005">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.552005" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.552005">
        <p class="paperTitle">Identification of factors that prevent ribosome stalling during early elongation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.552005" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.552005" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hong, H.-R.; Prince, C. R.; Wu, L.; Feaga, H. A.</p>
        <p class="info">Score: 21.2, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.552005' target='https://doi.org/10.1101/2023.08.04.552005'> 10.1101/2023.08.04.552005</a></p>
        <p class="abstract">Protein synthesis is catalyzed by the ribosome and a host of highly conserved elongation factors. Most elongation factors that are conserved in all domains of life are essential, such as EF-Tu (e/aEF1A) and EF-G (e/aEF2). In contrast, the universally conserved elongation factor P (EF-P/eIF5A) is essential in eukaryotes but is dispensable in most bacteria. EF-P prevents ribosome stalling at difficult-to translate sequences, especially polyprolines. Since efp deletion phenotypes range from modest to lethal in different bacterial species, we hypothesized that some bacteria encode an uncharacterized elongation factor with compensatory functions. To identify this factor, we used Tn-seq to screen for genes that are essential in the absence of EF-P in Bacillus subtilis. This screen identified YfmR, a member of the ABCF family of ATPases, as a translation factor that is essential when efp is deleted. We find that depleting YfmR from {Delta}efp cells decreases actively translating ribosomes and increases free ribosomal subunits bound to initiator tRNA, suggesting that ribosomes stall in early elongation. We also find that deleting efp from the spore-forming pathogen Bacillus anthracis causes a severe survival defect in liquid culture and in the presence of macrophages. B. anthracis {Delta}efp also does not produce detectable spores. We find that heterologous expression of B. subtilis YfmR in B. anthracis {Delta}efp cells partially rescues the severe growth and sporulation defects of this mutant. Our results therefore identify YfmR as an important translation factor that becomes essential in the absence of EF-P, and suggest that the essential function of YfmR and EF-P is to assist translation during early elongation.

SignificanceTranslation is one of the most ancient and energetically demanding process that occurs in the cell. Ribosomes constitute more than 60% of cellular mass in actively growing cells, and ribosomes are a major target of antimicrobials and chemotherapeutics. Here, we report evidence that both EF-P and the uncharacterized protein YfmR are important for early translation elongation in bacteria, and that this activity is essential. YfmR is a member of the ABCF family of ATPases whose role in translation is only beginning to be understood. Given the broad distribution of ABCFs from bacteria to fungi, we expect our results to have implications for understanding the crucial first steps of translation elongation in diverse organisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.551499">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.551499" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.551499">
        <p class="paperTitle">Mesoscale DNA Features Impact APOBEC3A and APOBEC3B Deaminase Activity and Shape Tumor Mutational Landscapes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.551499" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.551499" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sanchez, A.; Ortega, P.; Sakhtemani, R.; Manjunath, L.; Oh, S.; Bournique, E.; Becker, A.; Kim, K.; Durfee, C.; Temiz, N. A.; Chen, X.; Harris, R.; Lawrence, M.; Buisson, R.</p>
        <p class="info">Score: 11.9, Published: 2023-08-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.551499' target='https://doi.org/10.1101/2023.08.02.551499'> 10.1101/2023.08.02.551499</a></p>
        <p class="abstract">Antiviral DNA cytosine deaminases APOBEC3A and APOBEC3B are major sources of mutations in cancer by catalyzing cytosine-to-uracil deamination. APOBEC3A preferentially targets singlestranded DNAs, with a noted affinity for DNA regions that adopt stem-loop secondary structures. However, the detailed substrate preferences of APOBEC3A and APOBEC3B have been fully established, and the specific influence of the DNA sequence on APOBEC3A APOBEC3B deaminase activity remains to be investigated. Here, we find that APOBEC3B selectively targets DNA stem-loop structures, and they are distinct from those subjected deamination by APOBEC3A. We develop Oligo-seq, a novel in vitro sequencing-based to identify specific sequence contexts promoting APOBEC3A and APOBEC3B activity. Through this approach, we demonstrate that APOBEC3A an APOBEC3B deaminase activity is strongly regulated by specific sequences surrounding the targeted cytosine. Moreover, we identify structural features of APOBEC3B and APOBEC3A responsible for their substrate preferences. Importantly, we determine that APOBEC3B-induced mutations in hairpin-forming sequences within tumor genomes differ from the DNA stem-loop sequences mutated by APOBEC3A. Together, our study provides evidence that APOBEC3A and APOBEC3B can generate mutation landscapes in cancer genomes, driven by their unique substrate selectivity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.551522">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.551522" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.551522">
        <p class="paperTitle">Acetic acid is a superior acidifier for sub-nanogram and single cell proteomic studies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.551522" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.551522" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Orsburn, B. C.</p>
        <p class="info">Score: 11.1, Published: 2023-08-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.551522' target='https://doi.org/10.1101/2023.08.01.551522'> 10.1101/2023.08.01.551522</a></p>
        <p class="abstract">A recent study demonstrated a substantial signal increase when employing a 0.5% acetic acid buffer additive instead of the traditional 0.1% formic acid used in shotgun proteomics. In this study I compare these two buffers for a dilution series of tryptic digests down to 20 picograms peptide on column on a TIMSTOF single cell proteome (SCP) system. I observe a comparable relative level of signal increase as previously reported, which translates to improvements in proteome coverage at every peptide load assessed. The relative increase in peptide identifications is more apparent at lower concentrations with a striking 1.8-fold more peptides identified at 20 pg peptide load, resulting in over 2,000 protein groups identified in 30 minutes on this system. These results translate well to isolated single human cancer cells allowing over 1,000 protein groups to be identified in single human cells processed using a simple one step method in standard 96-well plates.

All vendor raw and processed data has been made publicly available at www.massive.ucsd.edu and can be accessed as MSV000092563.

Abstract graphic

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=87 SRC=&#34;FIGDIR/small/551522v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (38K):
org.highwire.dtl.DTLVardef@1f7eb2aorg.highwire.dtl.DTLVardef@17047b9org.highwire.dtl.DTLVardef@74e64corg.highwire.dtl.DTLVardef@bbafd1_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.05.552090">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.05.552090" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.05.552090">
        <p class="paperTitle">Molecular insights into the activation of Mre11-Rad50 endonuclease activity by Sae2/CtIP</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.05.552090" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.05.552090" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nicolas, Y.; Bret, H.; Cannavo, E.; Cejka, P.; Borde, V.; Guerois, R.</p>
        <p class="info">Score: 10.4, Published: 2023-08-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.05.552090' target='https://doi.org/10.1101/2023.08.05.552090'> 10.1101/2023.08.05.552090</a></p>
        <p class="abstract">In S. cerevisiae, the Mre11-Rad50-Xrs2 (MRX)-Sae2 nuclease activity is required for the resection of DNA breaks with secondary structures or protein blocks, while in humans, the MRE11-RAD50-NBS1 (MRN) homologue with CtIP is needed to initiate DNA end resection of all breaks. Phosphorylated Sae2/CtIP stimulates the endonuclease activity of MRX/N. Structural insights into the activation of the Mre11 nuclease are available only for organisms lacking Sae2/CtIP, so little is known how Sae2/CtIP activates the nuclease ensemble. Here, we uncover the mechanism of Mre11 activation by Sae2 using a combination of AlphaFold2 structural modeling, biochemical and genetic assays. We show that Sae2 stabilizes the Mre11 nuclease in a conformation poised to cleave substrate DNA. Several designs of compensatory mutations establish how Sae2 activates MRX in vitro and in vivo, validating the structural model. Last, our study uncovers how human CtIP, despite considerable sequence divergence, employs a similar mechanism to activate MRN.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.551405">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.551405" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.551405">
        <p class="paperTitle">MicroED as a powerful tool for structure determination of macrocyclic drug compounds directly from their powder formulations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.551405" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.551405" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Danelius, E.; Bu, G.; Wieske, L.; Gonen, T.</p>
        <p class="info">Score: 9.2, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.551405' target='https://doi.org/10.1101/2023.07.31.551405'> 10.1101/2023.07.31.551405</a></p>
        <p class="abstract">Macrocycles are important drug leads with many advantages including the ability to target flat and featureless binding sites as well as act as molecular chameleons and thereby reach intracellular targets. However, due to their complex structures and inherent flexibility, macrocycles are difficult to study structurally and there are limited structural data available. Herein, we use the cryo-EM method MicroED to determine the novel atomic structures of several macrocycles which have previously resisted structural determination. We show that structures of similar complexity can now be obtained rapidly from nanograms of material, and that different conformations of flexible compounds can be derived from the same experiment. These results will have impact on contemporary drug discovery as well as natural product exploration.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.551649">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.551649" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.551649">
        <p class="paperTitle">Strategic engineering unlocks in vitro type II polyketide biosynthesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.551649" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.551649" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, K. K.; Cho, Y. I.; Tran, M. A.; Wiedemann, C.; Koweek, R. S.; Hoang, N. K.; Hamrick, G.; Bowen, M.; Kokona, B.; Beld, J.; Hellmich, U.; Charkoudian, L. K.</p>
        <p class="info">Score: 8.8, Published: 2023-08-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.551649' target='https://doi.org/10.1101/2023.08.02.551649'> 10.1101/2023.08.02.551649</a></p>
        <p class="abstract">Microbial type II polyketides serve as powerful medicinally relevant agents. These molecules are biosynthesized by polyketide synthases (PKSs) comprised of a core ketosynthase-chain length factor (KS-CLF) and phosphopantetheinylated acyl carrier protein (holo-ACP). While engineering type II PKSs holds potential to unlock sustainable access to diverse bioactive molecules, the inability to obtain cognate type II KS-CLFs and holo-ACPs for in vitro studies represents a longstanding barrier. Herein, we share how the sequence and structural analysis of the Gloeocapsa sp. PCC 7428 ACP allowed us to tune to a requisite weak yet specific interaction with a phosphopantetheinyl transferase to afford the holo-ACP. This, coupled with our ability to heterologously express the cognate KS-CLF in high quantities, unlocked access to polyketide products via in vitro multienzyme assembly. We hope this work inspires future studies of type II PKSs that have previously evaded heterologous expression or have yet to be explored.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=90 SRC=&#34;FIGDIR/small/551649v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (25K):
org.highwire.dtl.DTLVardef@125f7acorg.highwire.dtl.DTLVardef@1a765ccorg.highwire.dtl.DTLVardef@10c57cdorg.highwire.dtl.DTLVardef@1588db6_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.551444">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.551444" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.551444">
        <p class="paperTitle">De novo Protein Sequence Design Based on Deep Learning and Validation on CalB Hydrolase</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.551444" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.551444" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mu, J.; Li, Z.; Zhang, B.; Zhang, Q.; Iqbal, J.; Wadood, A.; Wei, T.; Feng, Y.; Chen, H.</p>
        <p class="info">Score: 6.2, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.551444' target='https://doi.org/10.1101/2023.08.01.551444'> 10.1101/2023.08.01.551444</a></p>
        <p class="abstract">Protein design is central to nearly all protein engineering problems, as it can enable the creation of proteins with new biological function, such as improving the catalytic efficiency of enzymes. As one of the key tasks of protein design, fixed-backbone protein sequence design aims to design novel sequence that would fold into a given protein backbone structure. However, current sequence design methods have limitations in terms of low sequence diversity and experimental validation of designed protein function, which cannot meet the needs of functional protein design. We firstly constructed Graphormer-based Protein Design (GPD) model that directly applies Transformer to graph-based representation of 3D protein structure, and added Gaussian noise and sequence random mask to node features to improve the sequence recovery and diversity. Additionally, functional filtering based on the structure folding, solubility, and function were performed to improve the success rate in experiments. The process of &#34;sequence design-functional filtering -functional experiment&#34; was carried out for CalB hydrolase. The experimental results showed that the specify activity of designed protein improved 1.7 times than CalB wild type. This design and filtering platform will be a valuable tool for generating industrial enzymes and protein drugs with specific functions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.551350">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.551350" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.551350">
        <p class="paperTitle">Probing Bioactive Chemical Space to Discover RNA-Targeted Small Molecules</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.551350" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.551350" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wicks, S. L.; Morgan, B. S.; Wilson, A. W.; Hargrove, A. E.</p>
        <p class="info">Score: 7.0, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.551350' target='https://doi.org/10.1101/2023.07.31.551350'> 10.1101/2023.07.31.551350</a></p>
        <p class="abstract">Small molecules have become increasingly recognized as invaluable tools to study RNA structure and function and to develop RNA-targeted therapeutics. To rationally design RNA-targeting ligands, a comprehensive understanding and explicit testing of small molecule properties that govern molecular recognition is crucial. To date, most studies have primarily evaluated properties of small molecules that bind RNA in vitro, with little to no assessment of properties that are distinct to selective and bioactive RNA-targeted ligands. Therefore, we curated an RNA-focused library, termed the Duke RNA-Targeted Library (DRTL), that was biased towards the physicochemical and structural properties of biologically active and non-ribosomal RNA-targeted small molecules. The DRTL represents one of the largest academic RNA-focused small molecule libraries curated to date with more than 800 small molecules. These ligands were selected using computational approaches that measure similarity to known bioactive RNA ligands and that diversify the molecules within this space. We evaluated DRTL binding in vitro to a panel of four RNAs using two optimized fluorescent indicator displacement assays, and we successfully identified multiple small molecule hits, including several novel scaffolds for RNA. The DRTL has and will continue to provide insights into biologically relevant RNA chemical space, such as the identification of additional RNA-privileged scaffolds and validation of RNA-privileged molecular features. Future DRTL screening will focus on expanding both the targets and assays used, and we welcome collaboration from the scientific community. We envision that the DRTL will be a valuable resource for the discovery of RNA-targeted chemical probes and therapeutic leads.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
