<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="De novo design of allosterically switchable protein assemblies
Authors: Pillai, A.; Idris, A.; Philomin, A.; Weidle, C.; Skotheim, R.; Leung, P. J. Y.; Broerman, A.; Demakis, C.; Borst, A. J.; Praetorius, F.; Baker, D.
Score: 32.4, Published: 2023-11-02 DOI: 10.1101/2023.11.01.565167
Allosteric modulation of protein function, wherein the binding of an effector to a protein triggers conformational changes at distant functional sites, plays a central role in the control of metabolism and cell signaling.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="De novo design of allosterically switchable protein assemblies
Authors: Pillai, A.; Idris, A.; Philomin, A.; Weidle, C.; Skotheim, R.; Leung, P. J. Y.; Broerman, A.; Demakis, C.; Borst, A. J.; Praetorius, F.; Baker, D.
Score: 32.4, Published: 2023-11-02 DOI: 10.1101/2023.11.01.565167
Allosteric modulation of protein function, wherein the binding of an effector to a protein triggers conformational changes at distant functional sites, plays a central role in the control of metabolism and cell signaling." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-05T10:37:37+00:00" />
<meta property="article:modified_time" content="2023-11-05T10:37:37+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="De novo design of allosterically switchable protein assemblies
Authors: Pillai, A.; Idris, A.; Philomin, A.; Weidle, C.; Skotheim, R.; Leung, P. J. Y.; Broerman, A.; Demakis, C.; Borst, A. J.; Praetorius, F.; Baker, D.
Score: 32.4, Published: 2023-11-02 DOI: 10.1101/2023.11.01.565167
Allosteric modulation of protein function, wherein the binding of an effector to a protein triggers conformational changes at distant functional sites, plays a central role in the control of metabolism and cell signaling."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "De novo design of allosterically switchable protein assemblies\nAuthors: Pillai, A.; Idris, A.; Philomin, A.; Weidle, C.; Skotheim, R.; Leung, P. J. Y.; Broerman, A.; Demakis, C.; Borst, A. J.; Praetorius, F.; Baker, D.\nScore: 32.4, Published: 2023-11-02 DOI: 10.1101/2023.11.01.565167\nAllosteric modulation of protein function, wherein the binding of an effector to a protein triggers conformational changes at distant functional sites, plays a central role in the control of metabolism and cell signaling.",
  "keywords": [
    
  ],
  "articleBody": " De novo design of allosterically switchable protein assemblies\nAuthors: Pillai, A.; Idris, A.; Philomin, A.; Weidle, C.; Skotheim, R.; Leung, P. J. Y.; Broerman, A.; Demakis, C.; Borst, A. J.; Praetorius, F.; Baker, D.\nScore: 32.4, Published: 2023-11-02 DOI: 10.1101/2023.11.01.565167\nAllosteric modulation of protein function, wherein the binding of an effector to a protein triggers conformational changes at distant functional sites, plays a central role in the control of metabolism and cell signaling. There has been considerable interest in designing allosteric systems, both to gain insight into the mechanisms underlying such \"action at a distance\" modulation and to create synthetic proteins whose functions can be regulated by effectors. However, emulating the subtle conformational changes distributed across many residues, characteristic of natural allosteric proteins, is a significant challenge. Here, inspired by the classic Monod-Changeux-Wyman model of cooperativity, we investigate the de novo design of allostery through rigid-body coupling of designed effector-switchable hinge modules to protein interfaces that direct the formation of alternative oligomeric states. We find that this approach can be used to generate a wide variety of allosterically switchable systems, including cyclic rings that incorporate or eject subunits in response to effector binding and dihedral cages that undergo effector-induced disassembly. Size-exclusion chromatography, mass photometry, and electron microscopy reveal that these designed allosteric protein assemblies closely resemble the design models in both the presence and absence of effectors and can have ligand-binding cooperativity comparable to classic natural systems such as hemoglobin. Our results indicate that allostery can arise from global coupling of the energetics of protein substructures without optimized sidechain-sidechain allosteric communication pathways and provide a roadmap for generating allosterically triggerable delivery systems, protein nanomachines, and cellular feedback control circuitry.\nPervasive aggregation and depletion of host and viral proteins in response to cysteine-reactive electrophilic compounds\nAuthors: Julio, A. R.; Shikwana, F.; Truong, C.; Burton, N. R.; Dominguez, E.; Turmon, A.; Cao, J.; Backus, K.\nScore: 27.0, Published: 2023-11-01 DOI: 10.1101/2023.10.30.564067\nProtein homeostasis is tightly regulated, with damaged or misfolded proteins quickly eliminated by the proteasome and autophagosome pathways. By co-opting these processes, targeted protein degradation technologies enable pharmacological manipulation of protein abundance. Recently, cysteine-reactive molecules have been added to the degrader toolbox, which offer the benefit of unlocking the therapeutic potential of undruggable protein targets. The proteome-wide impact of these molecules remains to be fully understood and given the general reactivity of many classes of cysteine-reactive electrophiles, on- and off-target effects are likely. Using chemical proteomics, we identified a cysteine-reactive small molecule degrader of the SARS-CoV-2 non-structural protein 14 (nsp14), which effects degradation through direct modification of cysteines in both nsp14 and in host chaperones together with activation of global cell stress response pathways. We find that cysteine-reactive electrophiles increase global protein ubiquitylation, trigger proteasome activation, and result in widespread aggregation and depletion of host proteins, including components of the nuclear pore complex. Formation of stress granules was also found to be a remarkably ubiquitous cellular response to nearly all cysteine-reactive compounds and degraders. Collectively, our study sheds light on complexities of covalent target protein degradation and highlights untapped opportunities in manipulating and characterizing proteostasis processes via deciphering the cysteine-centric regulation of stress response pathways.\nCovalent Degrader of the Oncogenic Transcription Factor β-Catenin\nAuthors: Gowans, F. A.; Forte, N.; Hatcher, J.; Huang, O. W.; Wang, Y.; Poblano, B. E. A.; Wertz, I. E.; Nomura, D. K.\nScore: 18.0, Published: 2023-11-02 DOI: 10.1101/2023.10.31.565018\n{beta}-catenin (CTNNB1) is an oncogenic transcription factor that is important in cell-cell adhesion and transcription of cell proliferation and survival genes that drives the pathogenesis of many different types of cancers. However, direct pharmacological targeting of CTNNB1 has remained challenging deeming this transcription factor as undruggable. Here, we have performed a screen with a library of cysteine-reactive covalent ligands to identify a monovalent degrader EN83 that depletes CTNNB1 in a ubiquitin proteasome dependent manner. We show that EN83 directly and covalently targets CTNNB1 through targeting four distinct cysteines within the armadillo repeat domain (C439, C466, C520, and C619) leading to a destabilization of CTNNB1. Using covalent chemoproteomic approaches, we show that EN83 directly engages CTNNB1 in cells with a moderate degree of selectivity. We further demonstrate that direct covalent targeting of three of these four cysteines (C466, C520, and C619) in cells contributes to CTNNB1 degradation in cells. We also demonstrate that EN83 can be further optimized to yield more potent CTNNB1 binders and degraders. Our results show that chemoproteomic approaches can be used to covalently target and degrade challenging transcription factors like CTNNB1 through a destabilization-mediated degradation.\nGeneralized Biomolecular Modeling and Design with RoseTTAFold All-Atom\nAuthors: Krishna, R.; Wang, J.; Ahern, W.; Sturmfels, P.; Venkatesh, P.; Kalvet, I.; Lee, G. R.; Morey-Burrows, F. S.; Anishchenko, I.; Humphreys, I. R.; McHugh, R.; Vafeados, D.; Li, X.; Sutherland, G. A.; Hitchcock, A.; Hunter, C. N.; Baek, M.; DiMaio, F.; Baker, D.\nScore: 345.6, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561603\nAlthough AlphaFold2 (AF2) and RoseTTAFold (RF) have transformed structural biology by enabling high-accuracy protein structure modeling, they are unable to model covalent modifications or interactions with small molecules and other non-protein molecules that can play key roles in biological function. Here, we describe RoseTTAFold All-Atom (RFAA), a deep network capable of modeling full biological assemblies containing proteins, nucleic acids, small molecules, metals, and covalent modifications given the sequences of the polymers and the atomic bonded geometry of the small molecules and covalent modifications. Following training on structures of full biological assemblies in the Protein Data Bank (PDB), RFAA has comparable protein structure prediction accuracy to AF2, excellent performance in CAMEO for flexible backbone small molecule docking, and reasonable prediction accuracy for protein covalent modifications and assemblies of proteins with multiple nucleic acid chains and small molecules which, to our knowledge, no existing method can model simultaneously. By fine-tuning on diffusive denoising tasks, we develop RFdiffusion All-Atom (RFdiffusionAA), which generates binding pockets by directly building protein structures around small molecules and other non-protein molecules. Starting from random distributions of amino acid residues surrounding target small molecules, we design and experimentally validate proteins that bind the cardiac disease therapeutic digoxigenin, the enzymatic cofactor heme, and optically active bilin molecules with potential for expanding the range of wavelengths captured by photosynthesis. We anticipate that RFAA and RFdiffusionAA will be widely useful for modeling and designing complex biomolecular systems.\nReconstitution of cytolinker-mediated crosstalk between actin and vimentin\nAuthors: Isturiz Petitjean, I.; Tran, Q. D.; Goutou, A.; Kabir, Z.; Wiche, G.; Leduc, C.; Koenderink, G. H.\nScore: 9.3, Published: 2023-10-28 DOI: 10.1101/2023.10.27.564417\nCell shape and motility are determined by the cytoskeleton, an interpenetrating network of actin filaments, microtubules, and intermediate filaments. The biophysical properties of each filament type individually have been studied extensively by cell-free reconstitution. By contrast, the interactions between the three cytoskeletal networks are relatively unexplored. They are coupled via crosslinkers of the plakin family such as plectin. These are challenging proteins for reconstitution because of their giant size and multidomain structure. Here we engineer a recombinant actin-vimentin crosslinker protein called ACTIF that provides a minimal model system for plectin, recapitulating its modular design with actin-binding and intermediate filament-binding domains separated by a coiled-coil linker for dimerisation. We show by fluorescence and electron microscopy that ACTIF has a high binding affinity for vimentin and actin and creates mixed actin-vimentin bundles. Rheology measurements show that ACTIF-mediated crosslinking strongly stiffens actin-vimentin composites. Finally, we demonstrate the modularity of this approach by creating an ACTIF variant with the intermediate filament binding domain of Adenomatous Polyposis Coli. Our protein engineering approach provides a new cell-free system for the biophysical characterization of intermediate filament-binding crosslinkers and for understanding the mechanical synergy between actin and vimentin in mesenchymal cells. HIGHLIGHTSO_LIEngineering of a recombinant actin-vimentin crosslinker called ACTIF with the plectin intermediate filament binding domain (IFBD), calponin homology domains that mediate actin binding, and a coiled-coil linker. C_LIO_LIACTIF crosslinks F-actin and vimentin and mediates their co-localization in vitro. C_LIO_LIACTIF has a binding affinity for vimentin that is about 500 times higher than for F-actin. C_LIO_LIACTIF forms composite bundles of F-actin and vimentin filaments. C_LIO_LIComposite F-actin/vimentin networks stiffen upon crosslinking with ACTIF. C_LIO_LIThe design of actin-vimentin crosslinker is modular, as other IFBDs like APCn2 can also be used. C_LI\nMulti-scale photocatalytic proximity labeling reveals cell surface neighbors on and between cells\nAuthors: Lin, Z.; Schaefer, K.; Lui, I.; Yao, Z.; Fossati, A. A.; Swaney, D. L.; Palar, A.; Sali, A.; Wells, J. A.\nScore: 8.1, Published: 2023-10-29 DOI: 10.1101/2023.10.28.564055\nThe cell membrane proteome is the primary biohub for cell communication, yet we are only beginning to understand the dynamic protein neighborhoods that form on the cell surface and between cells. Proximity labeling proteomics (PLP) strategies using chemically reactive probes are powerful approaches to yield snapshots of protein neighborhoods but are currently limited to one single resolution based on the probe labeling radius. Here, we describe a multi-scale PLP method with tunable resolution using a commercially available histological dye, Eosin Y, which upon visible light illumination, activates three different photo-probes with labeling radii ranging from [~]100 to 3000 [A]. We applied this platform to profile neighborhoods of the oncogenic epidermal growth factor receptor (EGFR) and orthogonally validated \u003e20 neighbors using immuno-assays and AlphaFold-Multimer prediction that generated plausible binary interaction models. We further profiled the protein neighborhoods of cell-cell synapses induced by bi-specific T-cell engagers (BiTEs) and chimeric antigen receptor (CAR)T cells at longer length scales. This integrated multi-scale PLP platform maps local and distal protein networks on cell surfaces and between cells. We believe this information will aid in the systematic construction of the cell surface interactome and reveal new opportunities for immunotherapeutics.\nDetermining the defining lengths between mature microRNAs/small interfering RNAs and tinyRNAs\nAuthors: Sim, G.; Kehling, A. C.; Park, M. S.; Divoky, C.; Zhang, H.; Malhotra, N.; Secor, J.; Nakanishi, K.\nScore: 7.8, Published: 2023-10-28 DOI: 10.1101/2023.10.27.564437\nMicroRNAs (miRNAs) and small interfering RNAs (siRNAs) are loaded into Argonaute (AGO) proteins, forming RNA-induced silencing complexes (RISCs). The assembly process establishes the seed, central, 3' supplementary, and tail regions across the loaded guide, enabling the RISC to recognize and cleave target RNAs. This guide segmentation is caused by anchoring the 3' end at the AGO PAZ domain, but the minimum guide length required for the conformation remains to be studied because there was no method by which to do so. Using a 3'[-\u0026gt;]5' exonuclease ISG20, we determined the lengths of AGO-associated miR-20a and let-7a with 3' ends that no longer reach the PAZ domain. Unexpectedly, miR-20a and let-7a needed different lengths, 19 and 20 nt, respectively, to maintain their RISC conformation. This difference can be explained by the low affinity of the PAZ domain for the adenosine at g19 of let-7a, suggesting that the tail-region sequence slightly alters the minimum guide length. We also present that 17-nt guides are sufficiently short enough to function as tinyRNAs (tyRNAs) whose 3 ends are not anchored at the PAZ domain. Since tyRNAs do not have the prerequisite anchoring for the standardized guide segmentation, they would recognize targets differently from miRNAs and siRNAs.\nA Toxoplasma gondii O-glycosyltransferase that modulates bradyzoite cyst wall rigidity is structurally and functionally distinct from host homologues\nAuthors: Kumar, P.; Tomita, T.; Gerken, T. A.; Ballard, C. J.; Lee, Y. S.; Weiss, L. M.; Samara, N. L.\nScore: 8.0, Published: 2023-10-26 DOI: 10.1101/2023.10.26.564164\nInfection with the Apicomplexan protozoan Toxoplasma gondii can be life-threatening in immunocompromised hosts. Transmission frequently occurs through the oral ingestion of T. gondii bradyzoite cysts, which transition to tachyzoites, disseminate, and then form cysts containing bradyzoites in the central nervous system, resulting in latent infection. There are currently no effective treatments to cure latent infection. Bradyzoites are encapsulated by a cyst wall that is critical for immune evasion, survival, and transmission. Cyst wall rigidity is influenced by the O-glycosylation of the mucin domain of the cyst wall protein CST1 by mucin-type O-glycosyltransferases (GalNAc-Ts). Here, we report the first structures of a protozoan GalNAc-T, T.gondii-GalNAc-T3 in the apo state and in complex with glycopeptide substrates. The structures reveal features that are strictly conserved in Apicomplexan homologues of T.gondii-GalNAc-T3, including a unique 2nd metal binding site that is coupled to substrate binding and enzymatic activity in vitro and cyst wall O-glycosylation in T. gondii. Additional structural features illustrate the divergence of GalNAc-Ts from parasite to host and highlight multiple druggable sites in T.gondii-GalNAc-T3 and its homologues in Apicomplexa that are responsible for a wide range of parasitic diseases.\nDesign of efficient artificial enzymes using crystallographically-enhanced conformational sampling\nAuthors: Rakotoharisoa, R. V.; Seifinoferest, B.; Zarifi, N.; Miller, J. D. M.; Rodriguez, J. M.; Thompson, M. C.; Chica, R. A.\nScore: 5.4, Published: 2023-11-02 DOI: 10.1101/2023.11.01.564846\nThe ability to create efficient artificial enzymes for any chemical reaction is of great interest. Here, we describe a computational design method for increasing catalytic efficiency of de novo enzymes to a level comparable to their natural counterparts without relying on directed evolution. Using structural ensembles generated from dynamics-based refinement against X-ray diffraction data collected from crystals of Kemp eliminases HG3 (kcat/KM 125 M-1 s-1) and KE70 (kcat/KM 57 M-1 s-1), we design from each enzyme [\u0026le;]10 sequences predicted to catalyze this reaction more efficiently. The most active designs display kcat/KM values improved by 100-250-fold, comparable to mutants obtained after screening thousands of variants in multiple rounds of directed evolution. Crystal structures show excellent agreement with computational models. Our work shows how computational design can generate efficient artificial enzymes by exploiting the true conformational ensemble to more effectively stabilize the transition state.\nCo-transcriptional capping using an RNA capping enzyme-T7 RNA polymerase fusion protein\nAuthors: Chan, S. H.; Tirole, L.; Kneller, D. W.; Kelley, T. M.; Dai, N.; Robb, G. B.\nScore: 5.3, Published: 2023-10-28 DOI: 10.1101/2023.10.28.564488\nmRNA vaccines and therapeutics are highly effective and can be developed and manufactured with a relatively short lead time. Here we report an all-enzyme platform to generate capped synthetic RNA in a one-step process based on an RNA capping enzyme-T7 RNA polymerase fusion protein. Under standard in vitro transcription reaction conditions, the fusion protein, in conjunction with an RNA cap 2'-O-methyltransferase, can generate synthetic mRNA with up to 95% of Cap-1 incorporation, greatly simplifying mRNA manufacturing workflows.\n",
  "wordCount" : "2343",
  "inLanguage": "en",
  "datePublished": "2023-11-05T10:37:37Z",
  "dateModified": "2023-11-05T10:37:37Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 5, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565167">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565167" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565167">
        <p class="paperTitle">De novo design of allosterically switchable protein assemblies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565167" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565167" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pillai, A.; Idris, A.; Philomin, A.; Weidle, C.; Skotheim, R.; Leung, P. J. Y.; Broerman, A.; Demakis, C.; Borst, A. J.; Praetorius, F.; Baker, D.</p>
        <p class="info">Score: 32.4, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565167' target='https://doi.org/10.1101/2023.11.01.565167'> 10.1101/2023.11.01.565167</a></p>
        <p class="abstract">Allosteric modulation of protein function, wherein the binding of an effector to a protein triggers conformational changes at distant functional sites, plays a central role in the control of metabolism and cell signaling. There has been considerable interest in designing allosteric systems, both to gain insight into the mechanisms underlying such &#34;action at a distance&#34; modulation and to create synthetic proteins whose functions can be regulated by effectors. However, emulating the subtle conformational changes distributed across many residues, characteristic of natural allosteric proteins, is a significant challenge. Here, inspired by the classic Monod-Changeux-Wyman model of cooperativity, we investigate the de novo design of allostery through rigid-body coupling of designed effector-switchable hinge modules to protein interfaces that direct the formation of alternative oligomeric states. We find that this approach can be used to generate a wide variety of allosterically switchable systems, including cyclic rings that incorporate or eject subunits in response to effector binding and dihedral cages that undergo effector-induced disassembly. Size-exclusion chromatography, mass photometry, and electron microscopy reveal that these designed allosteric protein assemblies closely resemble the design models in both the presence and absence of effectors and can have ligand-binding cooperativity comparable to classic natural systems such as hemoglobin. Our results indicate that allostery can arise from global coupling of the energetics of protein substructures without optimized sidechain-sidechain allosteric communication pathways and provide a roadmap for generating allosterically triggerable delivery systems, protein nanomachines, and cellular feedback control circuitry.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564067">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564067" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564067">
        <p class="paperTitle">Pervasive aggregation and depletion of host and viral proteins in response to cysteine-reactive electrophilic compounds</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564067" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564067" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Julio, A. R.; Shikwana, F.; Truong, C.; Burton, N. R.; Dominguez, E.; Turmon, A.; Cao, J.; Backus, K.</p>
        <p class="info">Score: 27.0, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564067' target='https://doi.org/10.1101/2023.10.30.564067'> 10.1101/2023.10.30.564067</a></p>
        <p class="abstract">Protein homeostasis is tightly regulated, with damaged or misfolded proteins quickly eliminated by the proteasome and autophagosome pathways. By co-opting these processes, targeted protein degradation technologies enable pharmacological manipulation of protein abundance. Recently, cysteine-reactive molecules have been added to the degrader toolbox, which offer the benefit of unlocking the therapeutic potential of undruggable protein targets. The proteome-wide impact of these molecules remains to be fully understood and given the general reactivity of many classes of cysteine-reactive electrophiles, on- and off-target effects are likely. Using chemical proteomics, we identified a cysteine-reactive small molecule degrader of the SARS-CoV-2 non-structural protein 14 (nsp14), which effects degradation through direct modification of cysteines in both nsp14 and in host chaperones together with activation of global cell stress response pathways. We find that cysteine-reactive electrophiles increase global protein ubiquitylation, trigger proteasome activation, and result in widespread aggregation and depletion of host proteins, including components of the nuclear pore complex. Formation of stress granules was also found to be a remarkably ubiquitous cellular response to nearly all cysteine-reactive compounds and degraders. Collectively, our study sheds light on complexities of covalent target protein degradation and highlights untapped opportunities in manipulating and characterizing proteostasis processes via deciphering the cysteine-centric regulation of stress response pathways.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.565018">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.565018" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.565018">
        <p class="paperTitle">Covalent Degrader of the Oncogenic Transcription Factor β-Catenin</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.565018" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.565018" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gowans, F. A.; Forte, N.; Hatcher, J.; Huang, O. W.; Wang, Y.; Poblano, B. E. A.; Wertz, I. E.; Nomura, D. K.</p>
        <p class="info">Score: 18.0, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.565018' target='https://doi.org/10.1101/2023.10.31.565018'> 10.1101/2023.10.31.565018</a></p>
        <p class="abstract">{beta}-catenin (CTNNB1) is an oncogenic transcription factor that is important in cell-cell adhesion and transcription of cell proliferation and survival genes that drives the pathogenesis of many different types of cancers. However, direct pharmacological targeting of CTNNB1 has remained challenging deeming this transcription factor as undruggable. Here, we have performed a screen with a library of cysteine-reactive covalent ligands to identify a monovalent degrader EN83 that depletes CTNNB1 in a ubiquitin proteasome dependent manner. We show that EN83 directly and covalently targets CTNNB1 through targeting four distinct cysteines within the armadillo repeat domain (C439, C466, C520, and C619) leading to a destabilization of CTNNB1. Using covalent chemoproteomic approaches, we show that EN83 directly engages CTNNB1 in cells with a moderate degree of selectivity. We further demonstrate that direct covalent targeting of three of these four cysteines (C466, C520, and C619) in cells contributes to CTNNB1 degradation in cells. We also demonstrate that EN83 can be further optimized to yield more potent CTNNB1 binders and degraders. Our results show that chemoproteomic approaches can be used to covalently target and degrade challenging transcription factors like CTNNB1 through a destabilization-mediated degradation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561603">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561603" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561603">
        <p class="paperTitle">Generalized Biomolecular Modeling and Design with RoseTTAFold All-Atom</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561603" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561603" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krishna, R.; Wang, J.; Ahern, W.; Sturmfels, P.; Venkatesh, P.; Kalvet, I.; Lee, G. R.; Morey-Burrows, F. S.; Anishchenko, I.; Humphreys, I. R.; McHugh, R.; Vafeados, D.; Li, X.; Sutherland, G. A.; Hitchcock, A.; Hunter, C. N.; Baek, M.; DiMaio, F.; Baker, D.</p>
        <p class="info">Score: 345.6, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561603' target='https://doi.org/10.1101/2023.10.09.561603'> 10.1101/2023.10.09.561603</a></p>
        <p class="abstract">Although AlphaFold2 (AF2) and RoseTTAFold (RF) have transformed structural biology by enabling high-accuracy protein structure modeling, they are unable to model covalent modifications or interactions with small molecules and other non-protein molecules that can play key roles in biological function. Here, we describe RoseTTAFold All-Atom (RFAA), a deep network capable of modeling full biological assemblies containing proteins, nucleic acids, small molecules, metals, and covalent modifications given the sequences of the polymers and the atomic bonded geometry of the small molecules and covalent modifications. Following training on structures of full biological assemblies in the Protein Data Bank (PDB), RFAA has comparable protein structure prediction accuracy to AF2, excellent performance in CAMEO for flexible backbone small molecule docking, and reasonable prediction accuracy for protein covalent modifications and assemblies of proteins with multiple nucleic acid chains and small molecules which, to our knowledge, no existing method can model simultaneously. By fine-tuning on diffusive denoising tasks, we develop RFdiffusion All-Atom (RFdiffusionAA), which generates binding pockets by directly building protein structures around small molecules and other non-protein molecules. Starting from random distributions of amino acid residues surrounding target small molecules, we design and experimentally validate proteins that bind the cardiac disease therapeutic digoxigenin, the enzymatic cofactor heme, and optically active bilin molecules with potential for expanding the range of wavelengths captured by photosynthesis. We anticipate that RFAA and RFdiffusionAA will be widely useful for modeling and designing complex biomolecular systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564417">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564417" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564417">
        <p class="paperTitle">Reconstitution of cytolinker-mediated crosstalk between actin and vimentin</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564417" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564417" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Isturiz Petitjean, I.; Tran, Q. D.; Goutou, A.; Kabir, Z.; Wiche, G.; Leduc, C.; Koenderink, G. H.</p>
        <p class="info">Score: 9.3, Published: 2023-10-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564417' target='https://doi.org/10.1101/2023.10.27.564417'> 10.1101/2023.10.27.564417</a></p>
        <p class="abstract">Cell shape and motility are determined by the cytoskeleton, an interpenetrating network of actin filaments, microtubules, and intermediate filaments. The biophysical properties of each filament type individually have been studied extensively by cell-free reconstitution. By contrast, the interactions between the three cytoskeletal networks are relatively unexplored. They are coupled via crosslinkers of the plakin family such as plectin. These are challenging proteins for reconstitution because of their giant size and multidomain structure. Here we engineer a recombinant actin-vimentin crosslinker protein called  ACTIF that provides a minimal model system for plectin, recapitulating its modular design with actin-binding and intermediate filament-binding domains separated by a coiled-coil linker for dimerisation. We show by fluorescence and electron microscopy that ACTIF has a high binding affinity for vimentin and actin and creates mixed actin-vimentin bundles. Rheology measurements show that ACTIF-mediated crosslinking strongly stiffens actin-vimentin composites. Finally, we demonstrate the modularity of this approach by creating an ACTIF variant with the intermediate filament binding domain of Adenomatous Polyposis Coli. Our protein engineering approach provides a new cell-free system for the biophysical characterization of intermediate filament-binding crosslinkers and for understanding the mechanical synergy between actin and vimentin in mesenchymal cells.

HIGHLIGHTSO_LIEngineering of a recombinant actin-vimentin crosslinker called ACTIF with the plectin intermediate filament binding domain (IFBD), calponin homology domains that mediate actin binding, and a coiled-coil linker.
C_LIO_LIACTIF crosslinks F-actin and vimentin and mediates their co-localization in vitro.
C_LIO_LIACTIF has a binding affinity for vimentin that is about 500 times higher than for F-actin.
C_LIO_LIACTIF forms composite bundles of F-actin and vimentin filaments.
C_LIO_LIComposite F-actin/vimentin networks stiffen upon crosslinking with ACTIF.
C_LIO_LIThe design of actin-vimentin crosslinker is modular, as other IFBDs like APCn2 can also be used.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.28.564055">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.28.564055" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.28.564055">
        <p class="paperTitle">Multi-scale photocatalytic proximity labeling reveals cell surface neighbors on and between cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.28.564055" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.28.564055" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lin, Z.; Schaefer, K.; Lui, I.; Yao, Z.; Fossati, A. A.; Swaney, D. L.; Palar, A.; Sali, A.; Wells, J. A.</p>
        <p class="info">Score: 8.1, Published: 2023-10-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.28.564055' target='https://doi.org/10.1101/2023.10.28.564055'> 10.1101/2023.10.28.564055</a></p>
        <p class="abstract">The cell membrane proteome is the primary biohub for cell communication, yet we are only beginning to understand the dynamic protein neighborhoods that form on the cell surface and between cells. Proximity labeling proteomics (PLP) strategies using chemically reactive probes are powerful approaches to yield snapshots of protein neighborhoods but are currently limited to one single resolution based on the probe labeling radius. Here, we describe a multi-scale PLP method with tunable resolution using a commercially available histological dye, Eosin Y, which upon visible light illumination, activates three different photo-probes with labeling radii ranging from [~]100 to 3000 [A]. We applied this platform to profile neighborhoods of the oncogenic epidermal growth factor receptor (EGFR) and orthogonally validated &gt;20 neighbors using immuno-assays and AlphaFold-Multimer prediction that generated plausible binary interaction models. We further profiled the protein neighborhoods of cell-cell synapses induced by bi-specific T-cell engagers (BiTEs) and chimeric antigen receptor (CAR)T cells at longer length scales. This integrated multi-scale PLP platform maps local and distal protein networks on cell surfaces and between cells. We believe this information will aid in the systematic construction of the cell surface interactome and reveal new opportunities for immunotherapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564437">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564437" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564437">
        <p class="paperTitle">Determining the defining lengths between mature microRNAs/small interfering RNAs and tinyRNAs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564437" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564437" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sim, G.; Kehling, A. C.; Park, M. S.; Divoky, C.; Zhang, H.; Malhotra, N.; Secor, J.; Nakanishi, K.</p>
        <p class="info">Score: 7.8, Published: 2023-10-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564437' target='https://doi.org/10.1101/2023.10.27.564437'> 10.1101/2023.10.27.564437</a></p>
        <p class="abstract">MicroRNAs (miRNAs) and small interfering RNAs (siRNAs) are loaded into Argonaute (AGO) proteins, forming RNA-induced silencing complexes (RISCs). The assembly process establishes the seed, central, 3&#39; supplementary, and tail regions across the loaded guide, enabling the RISC to recognize and cleave target RNAs. This guide segmentation is caused by anchoring the 3&#39; end at the AGO PAZ domain, but the minimum guide length required for the conformation remains to be studied because there was no method by which to do so. Using a 3&#39;[-&amp;gt;]5&#39; exonuclease ISG20, we determined the lengths of AGO-associated miR-20a and let-7a with 3&#39; ends that no longer reach the PAZ domain. Unexpectedly, miR-20a and let-7a needed different lengths, 19 and 20 nt, respectively, to maintain their RISC conformation. This difference can be explained by the low affinity of the PAZ domain for the adenosine at g19 of let-7a, suggesting that the tail-region sequence slightly alters the minimum guide length. We also present that 17-nt guides are sufficiently short enough to function as tinyRNAs (tyRNAs) whose 3 ends are not anchored at the PAZ domain. Since tyRNAs do not have the prerequisite anchoring for the standardized guide segmentation, they would recognize targets differently from miRNAs and siRNAs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564164">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564164" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564164">
        <p class="paperTitle">A Toxoplasma gondii O-glycosyltransferase that modulates bradyzoite cyst wall rigidity is structurally and functionally distinct from host homologues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564164" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564164" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kumar, P.; Tomita, T.; Gerken, T. A.; Ballard, C. J.; Lee, Y. S.; Weiss, L. M.; Samara, N. L.</p>
        <p class="info">Score: 8.0, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564164' target='https://doi.org/10.1101/2023.10.26.564164'> 10.1101/2023.10.26.564164</a></p>
        <p class="abstract">Infection with the Apicomplexan protozoan Toxoplasma gondii can be life-threatening in immunocompromised hosts. Transmission frequently occurs through the oral ingestion of T. gondii bradyzoite cysts, which transition to tachyzoites, disseminate, and then form cysts containing bradyzoites in the central nervous system, resulting in latent infection. There are currently no effective treatments to cure latent infection. Bradyzoites are encapsulated by a cyst wall that is critical for immune evasion, survival, and transmission. Cyst wall rigidity is influenced by the O-glycosylation of the mucin domain of the cyst wall protein CST1 by mucin-type O-glycosyltransferases (GalNAc-Ts). Here, we report the first structures of a protozoan GalNAc-T, T.gondii-GalNAc-T3 in the apo state and in complex with glycopeptide substrates. The structures reveal features that are strictly conserved in Apicomplexan homologues of T.gondii-GalNAc-T3, including a unique 2nd metal binding site that is coupled to substrate binding and enzymatic activity in vitro and cyst wall O-glycosylation in T. gondii. Additional structural features illustrate the divergence of GalNAc-Ts from parasite to host and highlight multiple druggable sites in T.gondii-GalNAc-T3 and its homologues in Apicomplexa that are responsible for a wide range of parasitic diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.564846">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.564846" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.564846">
        <p class="paperTitle">Design of efficient artificial enzymes using crystallographically-enhanced conformational sampling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.564846" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.564846" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rakotoharisoa, R. V.; Seifinoferest, B.; Zarifi, N.; Miller, J. D. M.; Rodriguez, J. M.; Thompson, M. C.; Chica, R. A.</p>
        <p class="info">Score: 5.4, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.564846' target='https://doi.org/10.1101/2023.11.01.564846'> 10.1101/2023.11.01.564846</a></p>
        <p class="abstract">The ability to create efficient artificial enzymes for any chemical reaction is of great interest. Here, we describe a computational design method for increasing catalytic efficiency of de novo enzymes to a level comparable to their natural counterparts without relying on directed evolution. Using structural ensembles generated from dynamics-based refinement against X-ray diffraction data collected from crystals of Kemp eliminases HG3 (kcat/KM 125 M-1 s-1) and KE70 (kcat/KM 57 M-1 s-1), we design from each enzyme [&amp;le;]10 sequences predicted to catalyze this reaction more efficiently. The most active designs display kcat/KM values improved by 100-250-fold, comparable to mutants obtained after screening thousands of variants in multiple rounds of directed evolution. Crystal structures show excellent agreement with computational models. Our work shows how computational design can generate efficient artificial enzymes by exploiting the true conformational ensemble to more effectively stabilize the transition state.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.28.564488">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.28.564488" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.28.564488">
        <p class="paperTitle">Co-transcriptional capping using an RNA capping enzyme-T7 RNA polymerase fusion protein</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.28.564488" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.28.564488" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chan, S. H.; Tirole, L.; Kneller, D. W.; Kelley, T. M.; Dai, N.; Robb, G. B.</p>
        <p class="info">Score: 5.3, Published: 2023-10-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.28.564488' target='https://doi.org/10.1101/2023.10.28.564488'> 10.1101/2023.10.28.564488</a></p>
        <p class="abstract">mRNA vaccines and therapeutics are highly effective and can be developed and manufactured with a relatively short lead time. Here we report an all-enzyme platform to generate capped synthetic RNA in a one-step process based on an RNA capping enzyme-T7 RNA polymerase fusion protein. Under standard in vitro transcription reaction conditions, the fusion protein, in conjunction with an RNA cap 2&#39;-O-methyltransferase, can generate synthetic mRNA with up to 95% of Cap-1 incorporation, greatly simplifying mRNA manufacturing workflows.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
