<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Human coronavirus HKU1 recognition of the TMPRSS2 host receptor
Authors: McCallum, M.; Park, Y.-J.; Stewart, C.; Sprouse, K. R.; Brown, J.; Tortorici, M. A.; Gibson, C.; Wong, E.; Ieven, M.; Telenti, A.; Veesler, D.
Score: 32.1, Published: 2024-01-09 DOI: 10.1101/2024.01.09.574565
The human coronavirus HKU1 spike (S) glycoprotein engages host cell surface sialoglycans and transmembrane protease serine 2 (TMPRSS2) to initiate infection. The molecular basis of HKU1 binding to TMPRSS2 and determinants of host receptor tropism remain elusive.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Human coronavirus HKU1 recognition of the TMPRSS2 host receptor
Authors: McCallum, M.; Park, Y.-J.; Stewart, C.; Sprouse, K. R.; Brown, J.; Tortorici, M. A.; Gibson, C.; Wong, E.; Ieven, M.; Telenti, A.; Veesler, D.
Score: 32.1, Published: 2024-01-09 DOI: 10.1101/2024.01.09.574565
The human coronavirus HKU1 spike (S) glycoprotein engages host cell surface sialoglycans and transmembrane protease serine 2 (TMPRSS2) to initiate infection. The molecular basis of HKU1 binding to TMPRSS2 and determinants of host receptor tropism remain elusive." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-14T10:38:40+00:00" />
<meta property="article:modified_time" content="2024-01-14T10:38:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Human coronavirus HKU1 recognition of the TMPRSS2 host receptor
Authors: McCallum, M.; Park, Y.-J.; Stewart, C.; Sprouse, K. R.; Brown, J.; Tortorici, M. A.; Gibson, C.; Wong, E.; Ieven, M.; Telenti, A.; Veesler, D.
Score: 32.1, Published: 2024-01-09 DOI: 10.1101/2024.01.09.574565
The human coronavirus HKU1 spike (S) glycoprotein engages host cell surface sialoglycans and transmembrane protease serine 2 (TMPRSS2) to initiate infection. The molecular basis of HKU1 binding to TMPRSS2 and determinants of host receptor tropism remain elusive."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Human coronavirus HKU1 recognition of the TMPRSS2 host receptor\nAuthors: McCallum, M.; Park, Y.-J.; Stewart, C.; Sprouse, K. R.; Brown, J.; Tortorici, M. A.; Gibson, C.; Wong, E.; Ieven, M.; Telenti, A.; Veesler, D.\nScore: 32.1, Published: 2024-01-09 DOI: 10.1101/2024.01.09.574565\nThe human coronavirus HKU1 spike (S) glycoprotein engages host cell surface sialoglycans and transmembrane protease serine 2 (TMPRSS2) to initiate infection. The molecular basis of HKU1 binding to TMPRSS2 and determinants of host receptor tropism remain elusive.",
  "keywords": [
    
  ],
  "articleBody": " Human coronavirus HKU1 recognition of the TMPRSS2 host receptor\nAuthors: McCallum, M.; Park, Y.-J.; Stewart, C.; Sprouse, K. R.; Brown, J.; Tortorici, M. A.; Gibson, C.; Wong, E.; Ieven, M.; Telenti, A.; Veesler, D.\nScore: 32.1, Published: 2024-01-09 DOI: 10.1101/2024.01.09.574565\nThe human coronavirus HKU1 spike (S) glycoprotein engages host cell surface sialoglycans and transmembrane protease serine 2 (TMPRSS2) to initiate infection. The molecular basis of HKU1 binding to TMPRSS2 and determinants of host receptor tropism remain elusive. Here, we designed an active human TMPRSS2 construct enabling high-yield recombinant production in human cells of this key therapeutic target. We determined a cryo-electron microscopy structure of the HKU1 RBD bound to human TMPRSS2 providing a blueprint of the interactions supporting viral entry and explaining the specificity for TMPRSS2 among human type 2 transmembrane serine proteases. We found that human, rat, hamster and camel TMPRSS2 promote HKU1 S-mediated entry into cells and identified key residues governing host receptor usage. Our data show that serum antibodies targeting the HKU1 RBD TMPRSS2 binding-site are key for neutralization and that HKU1 uses conformational masking and glycan shielding to balance immune evasion and receptor engagement.\nQuinone extraction drives atmospheric carbon monoxide oxidation in bacteria\nAuthors: Kropp, A.; Gillett, D. L.; Venugopal, H.; Gonzalvez, M. A.; Lingford, J. P.; Barlow, C. K.; Zhang, J.; Greening, C.; Grinter, R.\nScore: 30.9, Published: 2024-01-09 DOI: 10.1101/2024.01.09.574806\nDiverse bacteria and archaea use atmospheric carbon monoxide (CO) as an energy source during long-term survival. This process enhances the biodiversity of soil and marine ecosystems globally and removes 250 million tonnes of a toxic, climate-relevant pollutant from the atmosphere each year. Bacteria use [MoCu]-carbon monoxide dehydrogenases (Mo-CODH) to convert CO to carbon dioxide, then transfer the liberated high-energy electrons to the aerobic respiratory chain. However, given no high-affinity Mo-CODH has been purified, it is unknown how these enzymes oxidise CO at low concentrations and interact with the respiratory chain. Here we resolve these knowledge gaps by analysing Mo-CODH (CoxSML) and its hypothetical partner CoxG from Mycobacterium smegmatis. Kinetic and electrochemical analyses show purified Mo-CODH is a highly active high-affinity enzyme (Km = 139 nM, kcat = 54.2 s-1). Based on its 1.85 [A] resolution cryoEM structure, Mo-CODH forms a CoxSML homodimer similar to characterised low-affinity homologs, but has distinct active site coordination and narrower gas channels that may modulate affinity. We provide structural, biochemical, and genetic evidence that Mo-CODH transfers CO-derived electrons to the aerobic respiratory chain via the membrane-bound menaquinone-binding protein CoxG. Consistently, CoxG is required for CO-driven respiration, extracts menaquinone from mycobacterial membranes, and binds quinones in a hydrophobic pocket. Finally, we show that Mo-CODH and CoxG genetically and structurally associate in diverse bacteria and archaea. These findings reveal the basis of a biogeochemically and ecologically important process, while demonstrating that the newly discovered process of long-range quinone transport is a general mechanism of energy conservation, which convergently evolved on multiple occasions.\nACAD10 and ACAD11 enable mammalian 4-hydroxy acid lipid catabolism\nAuthors: Rashan, E. H.; Bartlett, A. K.; Khana, D. B.; Zhang, J.; Jain, R.; Smith, A. J.; Baker, Z. N.; Cook, T.; Caldwell, A.; Chevalier, A. R.; Pfleger, B. F.; Yuan, P.; Amador-Noguez, D.; Simcox, J. A.; Pagliarini, D. J.\nScore: 17.8, Published: 2024-01-09 DOI: 10.1101/2024.01.09.574893\nFatty acid {beta}-oxidation (FAO) is a central catabolic pathway with broad implications for organismal health. However, various fatty acids are largely incompatible with standard FAO machinery until they are modified by other enzymes. Included among these are the 4-hydroxy acids (4-HAs)--fatty acids hydroxylated at the 4 ({gamma}) position--which can be provided from dietary intake, lipid peroxidation, and certain drugs of abuse. Here, we reveal that two atypical and poorly characterized acyl-CoA dehydrogenases (ACADs), ACAD10 and ACAD11, drive 4-HA catabolism in mice. Unlike other ACADs, ACAD10 and ACAD11 feature kinase domains N-terminal to their ACAD domains that phosphorylate the 4-OH position as a requisite step in the conversion of 4-hydroxyacyl-CoAs into 2-enoyl-CoAs--conventional FAO intermediates. Our ACAD11 cryo-EM structure and molecular modeling reveal a unique binding pocket capable of accommodating this phosphorylated intermediate. We further show that ACAD10 is mitochondrial and necessary for catabolizing shorter-chain 4-HAs, whereas ACAD11 is peroxisomal and enables longer-chain 4-HA catabolism. Mice lacking ACAD11 accumulate 4-HAs in their plasma while comparable 3- and 5-hydroxy acids remain unchanged. Collectively, this work defines ACAD10 and ACAD11 as the primary gatekeepers of mammalian 4-HA catabolism and sets the stage for broader investigations into the ramifications of aberrant 4-HA metabolism in human health and disease.\nSwinging lever mechanism of myosin directly demonstrated by time-resolved cryoEM\nAuthors: Klebl, D. P.; McMillan, S. N.; Risi, C.; Forgacs, E.; Virok, B.; Atherton, J.; Stofella, M.; Winkelmann, D.; Sobott, F.; Galkin, V.; Knight, P. J.; Muench, S. P.; Scarff, C.; White, H. D.\nScore: 17.4, Published: 2024-01-06 DOI: 10.1101/2024.01.05.574365\nMyosins are essential for producing force and movement in cells through their interactions with F-actin. Generation of movement is proposed to occur through structural changes within the myosin motor domain, fuelled by ATP hydrolysis, that are amplified by a lever swing1, transitioning myosin from a primed (pre-powerstroke) state to a post-powerstroke state. However, the initial, primed actomyosin state, proposed to form prior to lever swing, has never been observed. Nor has the mechanism by which actin catalyses myosin ATPase activity been resolved. To address this, we performed time-resolved cryoEM of a myosin-5 mutant having slow hydrolysis product release. Primed actomyosin was captured 10 ms after mixing primed myosin with F-actin, whereas post-powerstroke actomyosin predominated at 120 ms, with no abundant intermediate structures. The structures were solved to 4.4[A] and 4.2[A] global resolution respectively. The primed motor binds to actin through its lower 50 kDa subdomain, with the actin-binding cleft open and Pi release prohibited. N-terminal actin interactions with myosin promote rotation of the upper 50 kDa subdomain, which closes the actin-binding cleft, and enables Pi release. Formation of upper 50 kDa subdomain interactions with actin creates the strong-binding interface required for effective force production. The myosin-5 lever swings through an angle of 93{degrees}, predominantly along the actin axis, with little twisting, to produce the post-powerstroke state. The magnitude of the lever swing matches the typical step length of myosin-5 walking along actin. These time-resolved structures directly demonstrate the swinging lever mechanism, ending decades of conjecture on how myosin produces force and movement.\nSingle cell proteomics by mass spectrometry reveals deep epigenetic insight and new targets of a class specific histone deacetylase inhibitor\nAuthors: Orsburn, B.\nScore: 16.5, Published: 2024-01-06 DOI: 10.1101/2024.01.05.574437\nEpigenetic programming has been shown to play a role in nearly every human system and disease where anyone has thought to look. However, the levels of heterogeneity at which epigenetic or epiproteomic modifications occur at single cell resolution across a population remains elusive. While recent advances in sequencing technology have allowed between 1 and 3 histone post-translational modifications to be analyzed in each single cell, over twenty separate chemical PTMs are known to exist, allowing thousands of possible combinations. Single cell proteomics by mass spectrometry (SCP) is an emerging technology in which hundreds or thousands of proteins can be directly quantified in typical human cells. As the proteins detected and quantified by SCP are heavily biased toward proteins of highest abundance, chromatin proteins are an attractive target for analysis. To this end, I applied SCP to the analysis of cancer cells treated with mocetinostat, a class specific histone deacetylase inhibitor. I find that 16 PTMs can be confidently identified and localized with high site specificity in single cells. In addition, the high abundance of histone proteins allows higher throughput methods to be utilized for SCP than ever described. While quantitative accuracy suffers when analyzing more than 700 cells per day, 9 histone proteins can be measured in single cells analyzed at even 3,500 cells per day, a throughput 10-fold greater than any previous report. In addition, the unbiased global approach utilized herein identifies a previously uncharacterized response to this drug through the S100-A8/S100-A9 protein complex partners. This response is observed in nearly every cell of the over 1,000 analyzed in this study, regardless of the relative throughput of the method utilized. While limitations exist in the methods described herein, current technologies can easily improve upon the results presented here to allow comprehensive analysis of histone PTMs to be performed in any mass spectrometry lab. All raw and processed data described in this study has been made publicly available through the ProteomeXchange/MASSIVE repository system as MSV000093434 Abstract graphic O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=150 SRC=\"FIGDIR/small/574437v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (43K): org.highwire.dtl.DTLVardef@154d2cdorg.highwire.dtl.DTLVardef@1db999eorg.highwire.dtl.DTLVardef@428122org.highwire.dtl.DTLVardef@2be88b_HPS_FORMAT_FIGEXP M_FIG C_FIG\nMicroED structure of the C11 cysteine protease Clostripain\nAuthors: Ruma, Y. N.; Bu, G.; Gonen, T.\nScore: 14.9, Published: 2024-01-05 DOI: 10.1101/2024.01.04.574240\nClostripain secreted from Clostridium histolyticum is the founding member of the C11 family of Clan CD cysteine peptidases, which is an important group of peptidases secreted by numerous bacteria. Clostripain is an arginine specific endopeptidase. Because of its efficacy as a cysteine peptidase, it is widely used in laboratory settings. Despite its importance the structure of clostripain remains unsolved. Here we describe the first structure of an active form of C. histolyticum Clostripain determined at 3.6 [A] resolution using microcrystal electron diffraction (MicroED). The structure was determined from a single nanocrystal after focused ion beam milling. The structure of Clostripain shows a typical Clan CD /{beta}/ sandwich architecture and the Cys231/His176 catalytic dyad in the active site. It has a large electronegative substrate binding pocket showing its ability to accommodate large and diverse substrates. A loop in the heavy chain formed between residues 452 to 457 is potentially important for substrate binding. In conclusion, this result demonstrates the importance of MicroED to determine the unknown structure of macromolecules such as Clostripain, which can be further used as a platform to study substrate binding and design of potential inhibitors against this class of peptidases.\nIdentification of extracellular membrane protein ENPP3 as a major cGAMP hydrolase, cementing cGAMP's role as an immunotransmitter\nAuthors: Mardjuki, R.; Wang, S.; Carozza, J.; Abhiraman, G.; Lyu, X.; Li, L.\nScore: 9.9, Published: 2024-01-13 DOI: 10.1101/2024.01.12.575449\ncGAMP is a second messenger that is synthesized in the cytosol upon detection of cytosolic dsDNA and passed between cells to facilitate downstream immune signaling. ENPP1, an extracellular enzyme, was the only metazoan cGAMP hydrolase known to regulate cGAMP levels to dampen anti-cancer immunity. Here, we uncover ENPP3 as the second and only other metazoan cGAMP hydrolase under homeostatic conditions. ENPP3 has a tissue expression pattern distinct from that of ENPP1 and accounts for all remaining cGAMP hydrolysis activity in mice lacking ENPP1. Importantly, we also show that as with ENPP1, selectively abolishing ENPP3's cGAMP hydrolase activity results in diminished cancer growth and metastasis of certain tumor types. Both ENPP1 and ENPP3 are extracellular enzymes, suggesting the dominant role that extracellular cGAMP must play as a mediator of cell-cell innate immune communication. Our work clearly shows that ENPP1 and ENPP3 non-redundantly dampen extracellular cGAMP-STING signaling, pointing to ENPP3 as a new target for cancer immunotherapy.\nStructural basis of archaeal RNA polymerase transcription elongation and Spt4/5 recruitment\nAuthors: Tarau, D.; Grunberger, F.; Pilsl, M.; Reichelt, R.; Heiss, F.; Konig, S.; Urlaub, H.; Hausner, W.; Engel, C.; Grohmann, D.\nScore: 8.9, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574147\nArchaeal transcription is carried out by a multi-subunit RNA polymerase (RNAP) that is highly homologous in structure and function to eukaryotic RNAP II. Among the set of basal transcription factors, only Spt5 is found in all domains of life but Spt5 has been shaped during evolution, which is also reflected in the heterodimerization of Spt5 with Spt4 in Archaea and Eukaryotes. To unravel the mechanistic basis of Spt4/5 function in Archaea, we performed structure-function analyses using the archaeal transcriptional machinery of Pyrococcus furiosus (Pfu). We report single-particle cryo-electron microscopy reconstructions of apo RNAP and the archaeal elongation complex (EC) in the absence and presence of Spt4/5. Surprisingly, Pfu Spt4/5 also binds the RNAP in the absence of nucleic acids in a distinct super-contracted conformation. We show that the RNAP clamp/stalk module exhibits conformational flexibility in the apo state of RNAP and that the enzyme contracts upon EC formation or Spt4/5 engagement. We furthermore identified a contact of the Spt5-NGN domain with the DNA duplex that stabilizes the upstream boundary of the transcription bubble and impacts Spt4/5 activity in vitro. This study, therefore, provides the structural basis for Spt4/5 function in archaeal transcription and reveals a potential role beyond the well-described support of elongation.\nA broadly generalizable stabilization strategy for sarbecovirus fusion machinery vaccines\nAuthors: Lee, J.; Stewart, C.; Schaefer, A.; Leaf, E. M.; Park, Y.-J.; Asarnow, D.; Powers, J. M.; Treichel, C.; Corti, D.; Baric, R.; King, N. P.; Veesler, D.\nScore: 77.7, Published: 2023-12-20 DOI: 10.1101/2023.12.12.571160\nContinuous evolution of SARS-CoV-2 alters the antigenicity of the immunodominant spike (S) receptor-binding domain and N-terminal domain, undermining the efficacy of vaccines and monoclonal antibody therapies. To overcome this challenge, we set out to develop a vaccine focusing antibody responses on the highly conserved but metastable S2 subunit, which folds as a spring-loaded fusion machinery. Here, we describe a protein design strategy enabling prefusion-stabilization of the SARS-CoV-2 S2 subunit and high yield recombinant expression of trimers with native structure and antigenicity. We demonstrate that our design strategy is broadly generalizable to all sarbecoviruses, as exemplified with the SARS-CoV-1 (clade 1a) and PRD-0038 (clade 3) S2 fusion machineries. Immunization of mice with a prefusion-stabilized SARS-CoV-2 S2 trimer vaccine elicits broadly reactive sarbecovirus antibody responses and neutralizing antibody titers of comparable magnitude against Wuhan-Hu-1 and the immune evasive XBB.1.5 variant. Vaccinated mice were protected from weight loss and disease upon challenge with SARS-CoV-2 XBB.1.5, providing proof-of-principle for fusion machinery sarbecovirus vaccines motivating future development.\nSubstrate recognition mechanism of the endoplasmic reticulum-associated ubiquitin ligase Doa10\nAuthors: Wu, K.; Itskanov, S.; Lynch, D. L.; Chen, Y.; Turner, A.; Gumbart, J.; Park, E.\nScore: 7.0, Published: 2024-01-09 DOI: 10.1101/2024.01.09.574907\nDoa10 (MARCH6 in metazoans) is a large polytopic membrane-embedded E3 ubiquitin ligase in the endoplasmic reticulum (ER) that plays an important role in quality control of cytosolic and ER proteins. Although Doa10 is highly conserved across eukaryotes, it is not understood how Doa10 recognizes its substrates. Here, we defined the substrate recognition mechanism of Doa10 by structural and functional analyses on Saccharomyces cerevisiae Doa10 and its well-defined degron Deg1. Cryo-EM analysis shows that Doa10 has unusual architecture with a large lipid-filled central cavity, and its conserved middle domain forms an additional water-filled lateral tunnel open to the cytosol. Our biochemical data and molecular dynamics simulations suggest that the entrance of the substrates degron peptide into the lateral tunnel is required for efficient polyubiquitination. The N- and C-terminal membrane domains of Doa10 seem to form fence-like features to restrict polyubiquitination to those proteins that can access the central cavity and lateral tunnel.\n",
  "wordCount" : "2447",
  "inLanguage": "en",
  "datePublished": "2024-01-14T10:38:40Z",
  "dateModified": "2024-01-14T10:38:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 14, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.574565">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.574565" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.574565">
        <p class="paperTitle">Human coronavirus HKU1 recognition of the TMPRSS2 host receptor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.574565" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.574565" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McCallum, M.; Park, Y.-J.; Stewart, C.; Sprouse, K. R.; Brown, J.; Tortorici, M. A.; Gibson, C.; Wong, E.; Ieven, M.; Telenti, A.; Veesler, D.</p>
        <p class="info">Score: 32.1, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.574565' target='https://doi.org/10.1101/2024.01.09.574565'> 10.1101/2024.01.09.574565</a></p>
        <p class="abstract">The human coronavirus HKU1 spike (S) glycoprotein engages host cell surface sialoglycans and transmembrane protease serine 2 (TMPRSS2) to initiate infection. The molecular basis of HKU1 binding to TMPRSS2 and determinants of host receptor tropism remain elusive. Here, we designed an active human TMPRSS2 construct enabling high-yield recombinant production in human cells of this key therapeutic target. We determined a cryo-electron microscopy structure of the HKU1 RBD bound to human TMPRSS2 providing a blueprint of the interactions supporting viral entry and explaining the specificity for TMPRSS2 among human type 2 transmembrane serine proteases. We found that human, rat, hamster and camel TMPRSS2 promote HKU1 S-mediated entry into cells and identified key residues governing host receptor usage. Our data show that serum antibodies targeting the HKU1 RBD TMPRSS2 binding-site are key for neutralization and that HKU1 uses conformational masking and glycan shielding to balance immune evasion and receptor engagement.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.574806">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.574806" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.574806">
        <p class="paperTitle">Quinone extraction drives atmospheric carbon monoxide oxidation in bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.574806" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.574806" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kropp, A.; Gillett, D. L.; Venugopal, H.; Gonzalvez, M. A.; Lingford, J. P.; Barlow, C. K.; Zhang, J.; Greening, C.; Grinter, R.</p>
        <p class="info">Score: 30.9, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.574806' target='https://doi.org/10.1101/2024.01.09.574806'> 10.1101/2024.01.09.574806</a></p>
        <p class="abstract">Diverse bacteria and archaea use atmospheric carbon monoxide (CO) as an energy source during long-term survival. This process enhances the biodiversity of soil and marine ecosystems globally and removes 250 million tonnes of a toxic, climate-relevant pollutant from the atmosphere each year. Bacteria use [MoCu]-carbon monoxide dehydrogenases (Mo-CODH) to convert CO to carbon dioxide, then transfer the liberated high-energy electrons to the aerobic respiratory chain. However, given no high-affinity Mo-CODH has been purified, it is unknown how these enzymes oxidise CO at low concentrations and interact with the respiratory chain. Here we resolve these knowledge gaps by analysing Mo-CODH (CoxSML) and its hypothetical partner CoxG from Mycobacterium smegmatis. Kinetic and electrochemical analyses show purified Mo-CODH is a highly active high-affinity enzyme (Km = 139 nM, kcat = 54.2 s-1). Based on its 1.85 [A] resolution cryoEM structure, Mo-CODH forms a CoxSML homodimer similar to characterised low-affinity homologs, but has distinct active site coordination and narrower gas channels that may modulate affinity. We provide structural, biochemical, and genetic evidence that Mo-CODH transfers CO-derived electrons to the aerobic respiratory chain via the membrane-bound menaquinone-binding protein CoxG. Consistently, CoxG is required for CO-driven respiration, extracts menaquinone from mycobacterial membranes, and binds quinones in a hydrophobic pocket. Finally, we show that Mo-CODH and CoxG genetically and structurally associate in diverse bacteria and archaea. These findings reveal the basis of a biogeochemically and ecologically important process, while demonstrating that the newly discovered process of long-range quinone transport is a general mechanism of energy conservation, which convergently evolved on multiple occasions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.574893">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.574893" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.574893">
        <p class="paperTitle">ACAD10 and ACAD11 enable mammalian 4-hydroxy acid lipid catabolism</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.574893" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.574893" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rashan, E. H.; Bartlett, A. K.; Khana, D. B.; Zhang, J.; Jain, R.; Smith, A. J.; Baker, Z. N.; Cook, T.; Caldwell, A.; Chevalier, A. R.; Pfleger, B. F.; Yuan, P.; Amador-Noguez, D.; Simcox, J. A.; Pagliarini, D. J.</p>
        <p class="info">Score: 17.8, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.574893' target='https://doi.org/10.1101/2024.01.09.574893'> 10.1101/2024.01.09.574893</a></p>
        <p class="abstract">Fatty acid {beta}-oxidation (FAO) is a central catabolic pathway with broad implications for organismal health. However, various fatty acids are largely incompatible with standard FAO machinery until they are modified by other enzymes. Included among these are the 4-hydroxy acids (4-HAs)--fatty acids hydroxylated at the 4 ({gamma}) position--which can be provided from dietary intake, lipid peroxidation, and certain drugs of abuse. Here, we reveal that two atypical and poorly characterized acyl-CoA dehydrogenases (ACADs), ACAD10 and ACAD11, drive 4-HA catabolism in mice. Unlike other ACADs, ACAD10 and ACAD11 feature kinase domains N-terminal to their ACAD domains that phosphorylate the 4-OH position as a requisite step in the conversion of 4-hydroxyacyl-CoAs into 2-enoyl-CoAs--conventional FAO intermediates. Our ACAD11 cryo-EM structure and molecular modeling reveal a unique binding pocket capable of accommodating this phosphorylated intermediate. We further show that ACAD10 is mitochondrial and necessary for catabolizing shorter-chain 4-HAs, whereas ACAD11 is peroxisomal and enables longer-chain 4-HA catabolism. Mice lacking ACAD11 accumulate 4-HAs in their plasma while comparable 3- and 5-hydroxy acids remain unchanged. Collectively, this work defines ACAD10 and ACAD11 as the primary gatekeepers of mammalian 4-HA catabolism and sets the stage for broader investigations into the ramifications of aberrant 4-HA metabolism in human health and disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574365">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574365" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574365">
        <p class="paperTitle">Swinging lever mechanism of myosin directly demonstrated by time-resolved cryoEM</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574365" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574365" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Klebl, D. P.; McMillan, S. N.; Risi, C.; Forgacs, E.; Virok, B.; Atherton, J.; Stofella, M.; Winkelmann, D.; Sobott, F.; Galkin, V.; Knight, P. J.; Muench, S. P.; Scarff, C.; White, H. D.</p>
        <p class="info">Score: 17.4, Published: 2024-01-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574365' target='https://doi.org/10.1101/2024.01.05.574365'> 10.1101/2024.01.05.574365</a></p>
        <p class="abstract">Myosins are essential for producing force and movement in cells through their interactions with F-actin. Generation of movement is proposed to occur through structural changes within the myosin motor domain, fuelled by ATP hydrolysis, that are amplified by a lever swing1, transitioning myosin from a primed (pre-powerstroke) state to a post-powerstroke state. However, the initial, primed actomyosin state, proposed to form prior to lever swing, has never been observed. Nor has the mechanism by which actin catalyses myosin ATPase activity been resolved. To address this, we performed time-resolved cryoEM of a myosin-5 mutant having slow hydrolysis product release. Primed actomyosin was captured 10 ms after mixing primed myosin with F-actin, whereas post-powerstroke actomyosin predominated at 120 ms, with no abundant intermediate structures. The structures were solved to 4.4[A] and 4.2[A] global resolution respectively. The primed motor binds to actin through its lower 50 kDa subdomain, with the actin-binding cleft open and Pi release prohibited. N-terminal actin interactions with myosin promote rotation of the upper 50 kDa subdomain, which closes the actin-binding cleft, and enables Pi release. Formation of upper 50 kDa subdomain interactions with actin creates the strong-binding interface required for effective force production. The myosin-5 lever swings through an angle of 93{degrees}, predominantly along the actin axis, with little twisting, to produce the post-powerstroke state. The magnitude of the lever swing matches the typical step length of myosin-5 walking along actin. These time-resolved structures directly demonstrate the swinging lever mechanism, ending decades of conjecture on how myosin produces force and movement.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574437">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574437" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574437">
        <p class="paperTitle">Single cell proteomics by mass spectrometry reveals deep epigenetic insight and new targets of a class specific histone deacetylase inhibitor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574437" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574437" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Orsburn, B.</p>
        <p class="info">Score: 16.5, Published: 2024-01-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574437' target='https://doi.org/10.1101/2024.01.05.574437'> 10.1101/2024.01.05.574437</a></p>
        <p class="abstract">Epigenetic programming has been shown to play a role in nearly every human system and disease where anyone has thought to look. However, the levels of heterogeneity at which epigenetic or epiproteomic modifications occur at single cell resolution across a population remains elusive. While recent advances in sequencing technology have allowed between 1 and 3 histone post-translational modifications to be analyzed in each single cell, over twenty separate chemical PTMs are known to exist, allowing thousands of possible combinations. Single cell proteomics by mass spectrometry (SCP) is an emerging technology in which hundreds or thousands of proteins can be directly quantified in typical human cells. As the proteins detected and quantified by SCP are heavily biased toward proteins of highest abundance, chromatin proteins are an attractive target for analysis. To this end, I applied SCP to the analysis of cancer cells treated with mocetinostat, a class specific histone deacetylase inhibitor. I find that 16 PTMs can be confidently identified and localized with high site specificity in single cells. In addition, the high abundance of histone proteins allows higher throughput methods to be utilized for SCP than ever described. While quantitative accuracy suffers when analyzing more than 700 cells per day, 9 histone proteins can be measured in single cells analyzed at even 3,500 cells per day, a throughput 10-fold greater than any previous report. In addition, the unbiased global approach utilized herein identifies a previously uncharacterized response to this drug through the S100-A8/S100-A9 protein complex partners. This response is observed in nearly every cell of the over 1,000 analyzed in this study, regardless of the relative throughput of the method utilized. While limitations exist in the methods described herein, current technologies can easily improve upon the results presented here to allow comprehensive analysis of histone PTMs to be performed in any mass spectrometry lab. All raw and processed data described in this study has been made publicly available through the ProteomeXchange/MASSIVE repository system as MSV000093434

Abstract graphic

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=150 SRC=&#34;FIGDIR/small/574437v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (43K):
org.highwire.dtl.DTLVardef@154d2cdorg.highwire.dtl.DTLVardef@1db999eorg.highwire.dtl.DTLVardef@428122org.highwire.dtl.DTLVardef@2be88b_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.574240">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.574240" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.574240">
        <p class="paperTitle">MicroED structure of the C11 cysteine protease Clostripain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.574240" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.574240" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ruma, Y. N.; Bu, G.; Gonen, T.</p>
        <p class="info">Score: 14.9, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.574240' target='https://doi.org/10.1101/2024.01.04.574240'> 10.1101/2024.01.04.574240</a></p>
        <p class="abstract">Clostripain secreted from Clostridium histolyticum is the founding member of the C11 family of Clan CD cysteine peptidases, which is an important group of peptidases secreted by numerous bacteria. Clostripain is an arginine specific endopeptidase. Because of its efficacy as a cysteine peptidase, it is widely used in laboratory settings. Despite its importance the structure of clostripain remains unsolved. Here we describe the first structure of an active form of C. histolyticum Clostripain determined at 3.6 [A] resolution using microcrystal electron diffraction (MicroED). The structure was determined from a single nanocrystal after focused ion beam milling. The structure of Clostripain shows a typical Clan CD /{beta}/ sandwich architecture and the Cys231/His176 catalytic dyad in the active site. It has a large electronegative substrate binding pocket showing its ability to accommodate large and diverse substrates. A loop in the heavy chain formed between residues 452 to 457 is potentially important for substrate binding. In conclusion, this result demonstrates the importance of MicroED to determine the unknown structure of macromolecules such as Clostripain, which can be further used as a platform to study substrate binding and design of potential inhibitors against this class of peptidases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.575449">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.575449" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.575449">
        <p class="paperTitle">Identification of extracellular membrane protein ENPP3 as a major cGAMP hydrolase, cementing cGAMP&#39;s role as an immunotransmitter</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.575449" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.575449" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mardjuki, R.; Wang, S.; Carozza, J.; Abhiraman, G.; Lyu, X.; Li, L.</p>
        <p class="info">Score: 9.9, Published: 2024-01-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.575449' target='https://doi.org/10.1101/2024.01.12.575449'> 10.1101/2024.01.12.575449</a></p>
        <p class="abstract">cGAMP is a second messenger that is synthesized in the cytosol upon detection of cytosolic dsDNA and passed between cells to facilitate downstream immune signaling. ENPP1, an extracellular enzyme, was the only metazoan cGAMP hydrolase known to regulate cGAMP levels to dampen anti-cancer immunity. Here, we uncover ENPP3 as the second and only other metazoan cGAMP hydrolase under homeostatic conditions. ENPP3 has a tissue expression pattern distinct from that of ENPP1 and accounts for all remaining cGAMP hydrolysis activity in mice lacking ENPP1. Importantly, we also show that as with ENPP1, selectively abolishing ENPP3&#39;s cGAMP hydrolase activity results in diminished cancer growth and metastasis of certain tumor types. Both ENPP1 and ENPP3 are extracellular enzymes, suggesting the dominant role that extracellular cGAMP must play as a mediator of cell-cell innate immune communication. Our work clearly shows that ENPP1 and ENPP3 non-redundantly dampen extracellular cGAMP-STING signaling, pointing to ENPP3 as a new target for cancer immunotherapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574147">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574147" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574147">
        <p class="paperTitle">Structural basis of archaeal RNA polymerase transcription elongation and Spt4/5 recruitment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574147" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574147" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tarau, D.; Grunberger, F.; Pilsl, M.; Reichelt, R.; Heiss, F.; Konig, S.; Urlaub, H.; Hausner, W.; Engel, C.; Grohmann, D.</p>
        <p class="info">Score: 8.9, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574147' target='https://doi.org/10.1101/2024.01.05.574147'> 10.1101/2024.01.05.574147</a></p>
        <p class="abstract">Archaeal transcription is carried out by a multi-subunit RNA polymerase (RNAP) that is highly homologous in structure and function to eukaryotic RNAP II. Among the set of basal transcription factors, only Spt5 is found in all domains of life but Spt5 has been shaped during evolution, which is also reflected in the heterodimerization of Spt5 with Spt4 in Archaea and Eukaryotes. To unravel the mechanistic basis of Spt4/5 function in Archaea, we performed structure-function analyses using the archaeal transcriptional machinery of Pyrococcus furiosus (Pfu). We report single-particle cryo-electron microscopy reconstructions of apo RNAP and the archaeal elongation complex (EC) in the absence and presence of Spt4/5. Surprisingly, Pfu Spt4/5 also binds the RNAP in the absence of nucleic acids in a distinct super-contracted conformation. We show that the RNAP clamp/stalk module exhibits conformational flexibility in the apo state of RNAP and that the enzyme contracts upon EC formation or Spt4/5 engagement. We furthermore identified a contact of the Spt5-NGN domain with the DNA duplex that stabilizes the upstream boundary of the transcription bubble and impacts Spt4/5 activity in vitro. This study, therefore, provides the structural basis for Spt4/5 function in archaeal transcription and reveals a potential role beyond the well-described support of elongation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.12.571160">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.12.571160" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.12.571160">
        <p class="paperTitle">A broadly generalizable stabilization strategy for sarbecovirus fusion machinery vaccines</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.12.571160" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.12.571160" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, J.; Stewart, C.; Schaefer, A.; Leaf, E. M.; Park, Y.-J.; Asarnow, D.; Powers, J. M.; Treichel, C.; Corti, D.; Baric, R.; King, N. P.; Veesler, D.</p>
        <p class="info">Score: 77.7, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.12.571160' target='https://doi.org/10.1101/2023.12.12.571160'> 10.1101/2023.12.12.571160</a></p>
        <p class="abstract">Continuous evolution of SARS-CoV-2 alters the antigenicity of the immunodominant spike (S) receptor-binding domain and N-terminal domain, undermining the efficacy of vaccines and monoclonal antibody therapies. To overcome this challenge, we set out to develop a vaccine focusing antibody responses on the highly conserved but metastable S2 subunit, which folds as a spring-loaded fusion machinery. Here, we describe a protein design strategy enabling prefusion-stabilization of the SARS-CoV-2 S2 subunit and high yield recombinant expression of trimers with native structure and antigenicity. We demonstrate that our design strategy is broadly generalizable to all sarbecoviruses, as exemplified with the SARS-CoV-1 (clade 1a) and PRD-0038 (clade 3) S2 fusion machineries. Immunization of mice with a prefusion-stabilized SARS-CoV-2 S2 trimer vaccine elicits broadly reactive sarbecovirus antibody responses and neutralizing antibody titers of comparable magnitude against Wuhan-Hu-1 and the immune evasive XBB.1.5 variant. Vaccinated mice were protected from weight loss and disease upon challenge with SARS-CoV-2 XBB.1.5, providing proof-of-principle for fusion machinery sarbecovirus vaccines motivating future development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.574907">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.574907" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.574907">
        <p class="paperTitle">Substrate recognition mechanism of the endoplasmic reticulum-associated ubiquitin ligase Doa10</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.574907" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.574907" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wu, K.; Itskanov, S.; Lynch, D. L.; Chen, Y.; Turner, A.; Gumbart, J.; Park, E.</p>
        <p class="info">Score: 7.0, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.574907' target='https://doi.org/10.1101/2024.01.09.574907'> 10.1101/2024.01.09.574907</a></p>
        <p class="abstract">Doa10 (MARCH6 in metazoans) is a large polytopic membrane-embedded E3 ubiquitin ligase in the endoplasmic reticulum (ER) that plays an important role in quality control of cytosolic and ER proteins. Although Doa10 is highly conserved across eukaryotes, it is not understood how Doa10 recognizes its substrates. Here, we defined the substrate recognition mechanism of Doa10 by structural and functional analyses on Saccharomyces cerevisiae Doa10 and its well-defined degron Deg1. Cryo-EM analysis shows that Doa10 has unusual architecture with a large lipid-filled central cavity, and its conserved middle domain forms an additional water-filled lateral tunnel open to the cytosol. Our biochemical data and molecular dynamics simulations suggest that the entrance of the substrates degron peptide into the lateral tunnel is required for efficient polyubiquitination. The N- and C-terminal membrane domains of Doa10 seem to form fence-like features to restrict polyubiquitination to those proteins that can access the central cavity and lateral tunnel.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
