<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="De novo design of diverse small molecule binders and sensors using Shape Complementary Pseudocycles
Authors: An, L.; Said, M.; Tran, L.; Majumder, S.; Goreshnik, I.; Lee, G. R.; Juergens, D.; Dauparas, J.; Anishchenko, I.; Coventry, B.; Bera, A. K.; Kang, A.; Levine, P. M.; Alvarez, V.; Pillai, A.; Norn, C.; Feldman, D.; Zorine, D.; Hicks, D. R.; Li, X.; Sanchez, M. G.; Vafeados, D. K.; Salveson, P. J.; Vorobieva, A.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="De novo design of diverse small molecule binders and sensors using Shape Complementary Pseudocycles
Authors: An, L.; Said, M.; Tran, L.; Majumder, S.; Goreshnik, I.; Lee, G. R.; Juergens, D.; Dauparas, J.; Anishchenko, I.; Coventry, B.; Bera, A. K.; Kang, A.; Levine, P. M.; Alvarez, V.; Pillai, A.; Norn, C.; Feldman, D.; Zorine, D.; Hicks, D. R.; Li, X.; Sanchez, M. G.; Vafeados, D. K.; Salveson, P. J.; Vorobieva, A." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-27T10:39:30+00:00" />
<meta property="article:modified_time" content="2023-12-27T10:39:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="De novo design of diverse small molecule binders and sensors using Shape Complementary Pseudocycles
Authors: An, L.; Said, M.; Tran, L.; Majumder, S.; Goreshnik, I.; Lee, G. R.; Juergens, D.; Dauparas, J.; Anishchenko, I.; Coventry, B.; Bera, A. K.; Kang, A.; Levine, P. M.; Alvarez, V.; Pillai, A.; Norn, C.; Feldman, D.; Zorine, D.; Hicks, D. R.; Li, X.; Sanchez, M. G.; Vafeados, D. K.; Salveson, P. J.; Vorobieva, A."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "De novo design of diverse small molecule binders and sensors using Shape Complementary Pseudocycles\nAuthors: An, L.; Said, M.; Tran, L.; Majumder, S.; Goreshnik, I.; Lee, G. R.; Juergens, D.; Dauparas, J.; Anishchenko, I.; Coventry, B.; Bera, A. K.; Kang, A.; Levine, P. M.; Alvarez, V.; Pillai, A.; Norn, C.; Feldman, D.; Zorine, D.; Hicks, D. R.; Li, X.; Sanchez, M. G.; Vafeados, D. K.; Salveson, P. J.; Vorobieva, A.",
  "keywords": [
    
  ],
  "articleBody": " De novo design of diverse small molecule binders and sensors using Shape Complementary Pseudocycles\nAuthors: An, L.; Said, M.; Tran, L.; Majumder, S.; Goreshnik, I.; Lee, G. R.; Juergens, D.; Dauparas, J.; Anishchenko, I.; Coventry, B.; Bera, A. K.; Kang, A.; Levine, P. M.; Alvarez, V.; Pillai, A.; Norn, C.; Feldman, D.; Zorine, D.; Hicks, D. R.; Li, X.; Sanchez, M. G.; Vafeados, D. K.; Salveson, P. J.; Vorobieva, A. A.; Baker, D.\nScore: 41.8, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572602\nA general method for designing proteins to bind and sense any small molecule of interest would be widely useful. Due to the small number of atoms to interact with, binding to small molecules with high affinity requires highly shape complementary pockets, and transducing binding events into signals is challenging. Here we describe an integrated deep learning and energy based approach for designing high shape complementarity binders to small molecules that are poised for downstream sensing applications. We employ deep learning generated psuedocycles with repeating structural units surrounding central pockets; depending on the geometry of the structural unit and repeat number, these pockets span wide ranges of sizes and shapes. For a small molecule target of interest, we extensively sample high shape complementarity pseudocycles to generate large numbers of customized potential binding pockets; the ligand binding poses and the interacting interfaces are then optimized for high affinity binding. We computationally design binders to four diverse molecules, including for the first time polar flexible molecules such as methotrexate and thyroxine, which are expressed at high levels and have nanomolar affinities straight out of the computer. Co-crystal structures are nearly identical to the design models. Taking advantage of the modular repeating structure of pseudocycles and central location of the binding pockets, we constructed low noise nanopore sensors and chemically induced dimerization systems by splitting the binders into domains which assemble into the original pseudocycle pocket upon target molecule addition. One Sentence SummaryWe use a pseuodocycle-based shape complementarity optimizing approach to design nanomolar binders to diverse ligands, including the flexible and polar methotrexate and thyroxine, that can be directly converted into ligand-gated nanopores and chemically induced dimerization systems.\nXylosyltransferase Bump-and-hole Engineering to Chemically Manipulate Proteoglycans in Mammalian Cells\nAuthors: Li, Z.; Di Vagno, L.; Ni Cheallaigh, A.; Sammon, D.; Briggs, D. C.; Chung, N.; Chang, V.; Mahoney, K. E.; Cioce, A.; Murphy, L. D.; Chen, Y.-H.; Narimatsu, Y.; Miller, R. L.; Willems, L. I.; Malaker, S. A.; Miller, G. J.; Hohenester, E.; Schumann, B.\nScore: 25.6, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572522\nMammalian cells orchestrate signalling through interaction events on their surfaces. Proteoglycans are an intricate part of these interactions, carrying large glycosaminoglycan polysaccharides that recruit signalling molecules. Despite their importance in development, cancer and neurobiology, a relatively small number of proteoglycans have been identified. In addition to the complexity of glycan extension, biosynthetic redundancy in the first protein glycosylation step by two xylosyltransferase isoenzymes XT1 and XT2 complicates annotation of proteoglycans. Here, we develop a chemical strategy that allows profiling of cellular proteoglycans. By employing a tactic termed bump-and-hole engineering, we engineer the two isoenzymes XT1 and XT2 to specifically transfer a chemically modified xylose analogue to target proteins. The chemical modification contains a bioorthogonal tag, allowing the ability to visualise and profile target proteins modified by both transferases in mammalian cells as well as pinpointing glycosylation sites by mass spectrometry. Engineered XT enzymes permit a view into proteoglycan biology that is orthogonal to conventional techniques in biochemistry.\nAtomic context-conditioned protein sequence design using LigandMPNN\nAuthors: Dauparas, J.; Lee, G. R.; Pecoraro, R.; An, L.; Anishchenko, I.; Glasscock, C.; Baker, D.\nScore: 23.2, Published: 2023-12-23 DOI: 10.1101/2023.12.22.573103\nProtein sequence design in the context of small molecules, nucleotides, and metals is critical to enzyme and small molecule binder and sensor design, but current state-of-the-art deep learning-based sequence design methods are unable to model non-protein atoms and molecules. Here, we describe a deep learning-based protein sequence design method called LigandMPNN that explicitly models all non-protein components of biomolecular systems. LigandMPNN significantly outperforms Rosetta and ProteinMPNN on native backbone sequence recovery for residues interacting with small molecules (63.3% vs. 50.4% \u0026 50.5%), nucleotides (50.5% vs. 35.2% \u0026 34.0%), and metals (77.5% vs. 36.0% \u0026 40.6%). LigandMPNN generates not only sequences but also sidechain conformations to allow detailed evaluation of binding interactions. Experimental characterization demonstrates that LigandMPNN can generate small molecule and DNA-binding proteins with high affinity and specificity.\nStructural analysis of the dynamic ribosome-translocon complex\nAuthors: Lewis, A. J. O.; Hegde, R. S.\nScore: 20.4, Published: 2023-12-22 DOI: 10.1101/2023.12.22.572959\nThe protein translocon at the endoplasmic reticulum comprises the Sec61 translocation channel and numerous accessory factors that collectively facilitate the biogenesis of secretory and membrane proteins. Here, we leveraged recent advances in cryo-EM and structure prediction to derive insights into several novel configurations of the ribosome-translocon complex. We show how a transmembrane domain (TMD) in a looped configuration passes through the Sec61 lateral gate during membrane insertion; how a nascent chain can bind and constrain the conformation of ribosomal protein uL22; and how the translocon-associated protein (TRAP) complex can adjust its position during different stages of protein biogenesis. Most unexpectedly, we find that a large proportion of translocon complexes contains RAMP4 intercalated into Sec61's lateral gate, widening Sec61's central pore and contributing to its hydrophilic interior. These structures lead to mechanistic hypotheses for translocon function and highlight a remarkably plastic machinery whose conformations and composition adjust dynamically to its diverse range of substrates.\nStructure of the minor spliceosomal U11 snRNP\nAuthors: Zhao, J.; Peter, D.; Brandina, I.; Liu, X.; Galej, W. P.\nScore: 17.1, Published: 2023-12-22 DOI: 10.1101/2023.12.22.573053\nThe minor spliceosome catalyses the excision of U12-dependent introns from pre-mRNAs. These introns are rare, but their removal is critical for cell viability. We obtained a cryo-EM reconstruction of the 13-subunit U11 snRNP complex, revealing structures of U11 snRNA and five minor spliceosome-specific factors. U11 snRNP appears strikingly different from the equivalent major spliceosome U1 snRNP. SNRNP25 and SNRNP35 form a dimer, which specifically recognises U11 snRNA. PDCD7 forms extended helices, which bridge SNRNP25 and SNRNP48, located at the distal ends of the particle. SNRNP48 forms multiple interfaces with U11 snRNP and, together with ZMAT5, are positioned near the 5'-end of the U11 snRNA and likely stabilise the binding of the incoming 5'-SS. Our structure provides mechanistic insights into U12-dependent intron recognition and the evolution of the splicing machinery.\nThree-step mechanism of promoter escape by RNA polymerase II\nAuthors: Zhan, Y.; Grabbe, F.; Oberbeckmann, E.; Dienemann, C.; Cramer, P.\nScore: 16.4, Published: 2023-12-22 DOI: 10.1101/2023.12.22.572998\nThe transition from transcription initiation to elongation is highly regulated in human cells but remains incompletely understood at the structural level. In particular, it is unclear how interactions between RNA polymerase (Pol) II and initiation factors are broken to enable promoter escape. Here we reconstitute Pol II promoter escape in vitro, determine high-resolution structures of five transition intermediates, and show that promoter escape occurs in three major steps. First, the growing RNA transcript displaces the B-reader element of the initiation factor TFIIB without evicting TFIIB. Second, rewinding of the upstream edge of the growing DNA bubble evicts TFIIA, TFIIB and TBP and repositions parts of TFIIE and TFIIF. Third, binding of DSIF and NELF evicts TFIIE and TFIIH, establishing the paused elongation complex. This three-step model for promoter escape fills a gap in our understanding of the initiation-elongation transition of Pol II transcription.\nMUT-7 exoribonuclease activity and localisation are mediated by an ancient domain\nAuthors: Busetto, V.; Pshanichnaya, L.; Lichtenberger, R.; Hann, S.; Ketting, R. F.; Falk, S.\nScore: 11.5, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572533\nThe MUT-7 family of 3-5 exoribonucleases is evolutionarily conserved across the animal kingdom and plays essential roles in small RNA production in the germline. Most MUT-7 homologs carry a C-terminal domain of unknown function named MUT7-C appended to the exoribonuclease domain. Our analysis shows that the MUT7-C is evolutionary ancient, as a minimal version of the domain exists as an individual protein in prokaryotes. In animals, MUT7-C has acquired an insertion that diverged during evolution, expanding its functions. C. elegans MUT-7 contains a specific insertion within MUT7-C, which allows binding to MUT-8 and, consequently, MUT-7 recruitment to germ granules. In addition, in C. elegans and human MUT-7, the MUT7-C domain contributes to RNA binding and is thereby crucial for nuclease activity. This RNA-binding function most likely represents the ancestral function of the MUT7-C domain. Overall, this study sheds light on MUT7-C and assigns two functions to this previously uncharacterised domain.\nStructure and methyl-lysine binding selectivity of the HUSH complex subunit MPP8\nAuthors: Nikolopoulos, N.; Oda, S.-i.; Prigozhin, D. M.; Modis, Y.\nScore: 9.5, Published: 2023-12-21 DOI: 10.1101/2023.12.21.572340\nThe Human Silencing Hub (HUSH) guards the genome from the pathogenic effects of retroelement expression. Composed of MPP8, TASOR, and Periphilin-1, HUSH recognizes actively transcribed retrotransposed sequences by the presence of long (\u003e1.5-kb) nascent transcripts without introns. HUSH recruits effectors that alter chromatin structure, degrade transcripts, and deposit transcriptionally repressive epigenetic marks. Structural and biochemical data for the three HUSH subunits are sparse. Here, we report the crystal structure of the C-terminal domain (CTD) of MPP8 necessary for HUSH activity. The MPP8 CTD consists of five ankyrin repeats followed by a domain with structural homology to the PINIT domains of Siz/PIAS-family SUMO E3 ligases. AlphaFold-Multimer modeling predicts that the MPP8 CTD forms extended interaction interfaces with a SPOC domain and a domain with a novel fold, DomI, in TASOR. The chromodomain of MPP8, known to bind the repressive mark H3K9me3, binds with similar or higher affinity to H3K9-like sequences in the mammalian H3K9 methyltransferase subunits SETDB1, ATF7IP, G9a, and GLP. Hence, MPP8 promotes heterochromatinization by recruiting H3K9 methyltransferases. Our work identifies novel structural elements in MPP8 that are required for HUSH complex assembly and silencing, thereby fulfilling vital functions in controlling the transcription of retrotransposons of endogenous and viral origin. Key PointsO_LIStructure of MPP8 CTD reveals ankyrin repeats and PINIT-like domain required for HUSH activity C_LIO_LIAlphaFold predicts a large interaction interface between MPP8 CTD and TASOR SPOC and DomI domains C_LIO_LIThe MPP8 chromodomain binds trimethyl-lysines in ATF7IP and GLP with higher affinity than H3K9me3 C_LI\nA new GUV-based assay to reconstitute membrane tethering in vitro demonstrates the vesicle tethering ability of the selective autophagy scaffold Atg11\nAuthors: Andhare, D.; Ragusa, M. J.\nScore: 9.5, Published: 2023-12-19 DOI: 10.1101/2023.12.19.572332\nMembrane tethering is essential for the generation of organelle contact sites and to anchor incoming vesicles to their target membranes before vesicle fusion. During autophagy in yeast, tethering of 30 nm vesicles to cargo is one of the first steps in the generation of the isolation membrane that engulfs the cargo to be degraded. While membrane tethering is critical for cellular function, many of the current biochemical techniques to assay for membrane tethering rely on indirect readouts and are limited in their ability to monitor protein localization at sites of tethering. As such, we developed a fluorescence-microscopy-based GUV liposome tethering assay (GLT) to directly visualize membrane tethering and monitor protein localization at tethering sites simultaneously. We initially used GLT with engineered membrane tethers to demonstrate the ease of use, versatility and sensitivity of the assay. We also demonstrated that the selective autophagy scaffolding protein Atg11 can bind, and tether negatively charged membranes but is unable to bind GUVs mimicking the charge of the outer mitochondrial membrane. Atg11 instead requires the selective autophagy receptor Atg32 to be recruited to mitochondrial membranes. Lastly, we demonstrate that Atg11 bound to Atg32 on GUVs can tether negatively charged vesicles to GUVs. Collectively, our results reconstitute one of the first steps during the initiation of mitochondrial autophagy and highlight the versatility of GLT to study a range of membrane tethering events biochemically.\nStructure of an insect gustatory receptor\nAuthors: Frank, H. M.; Walujkar, S.; Walsh, R. M.; Laursen, W. J.; Theobald, D. L.; Garrity, P. A.; Gaudet, R.\nScore: 8.0, Published: 2023-12-19 DOI: 10.1101/2023.12.19.572336\nGustatory Receptors (GRs) are critical for insect chemosensation and are potential targets for controlling pests and disease vectors. However, GR structures have not been experimentally determined. We present structures of Bombyx mori Gr9 (BmGr9), a fructose-gated cation channel, in agonist-free and fructose-bound states. BmGr9 forms a tetramer similar to distantly related insect Olfactory Receptors (ORs). Upon fructose binding, BmGr9s ion channel gate opens through helix S7b movements. In contrast to ORs, BmGR9s ligand-binding pocket, shaped by a kinked helix S4 and a shorter extracellular S3-S4 loop, is larger and solvent accessible in both agonist-free and fructose-bound states. Also unlike ORs, fructose binding by BmGr9 involves helix S5 and a binding pocket lined with aromatic and polar residues. Structure-based sequence alignments reveal distinct patterns of ligand-binding pocket residue conservation in GR subfamilies associated with distinct ligand classes. These data provide insight into the molecular basis of GR ligand specificity and function.\n",
  "wordCount" : "2121",
  "inLanguage": "en",
  "datePublished": "2023-12-27T10:39:30Z",
  "dateModified": "2023-12-27T10:39:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 27, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572602">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572602" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572602">
        <p class="paperTitle">De novo design of diverse small molecule binders and sensors using Shape Complementary Pseudocycles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572602" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572602" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: An, L.; Said, M.; Tran, L.; Majumder, S.; Goreshnik, I.; Lee, G. R.; Juergens, D.; Dauparas, J.; Anishchenko, I.; Coventry, B.; Bera, A. K.; Kang, A.; Levine, P. M.; Alvarez, V.; Pillai, A.; Norn, C.; Feldman, D.; Zorine, D.; Hicks, D. R.; Li, X.; Sanchez, M. G.; Vafeados, D. K.; Salveson, P. J.; Vorobieva, A. A.; Baker, D.</p>
        <p class="info">Score: 41.8, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572602' target='https://doi.org/10.1101/2023.12.20.572602'> 10.1101/2023.12.20.572602</a></p>
        <p class="abstract">A general method for designing proteins to bind and sense any small molecule of interest would be widely useful. Due to the small number of atoms to interact with, binding to small molecules with high affinity requires highly shape complementary pockets, and transducing binding events into signals is challenging. Here we describe an integrated deep learning and energy based approach for designing high shape complementarity binders to small molecules that are poised for downstream sensing applications. We employ deep learning generated psuedocycles with repeating structural units surrounding central pockets; depending on the geometry of the structural unit and repeat number, these pockets span wide ranges of sizes and shapes. For a small molecule target of interest, we extensively sample high shape complementarity pseudocycles to generate large numbers of customized potential binding pockets; the ligand binding poses and the interacting interfaces are then optimized for high affinity binding. We computationally design binders to four diverse molecules, including for the first time polar flexible molecules such as methotrexate and thyroxine, which are expressed at high levels and have nanomolar affinities straight out of the computer. Co-crystal structures are nearly identical to the design models. Taking advantage of the modular repeating structure of pseudocycles and central location of the binding pockets, we constructed low noise nanopore sensors and chemically induced dimerization systems by splitting the binders into domains which assemble into the original pseudocycle pocket upon target molecule addition.

One Sentence SummaryWe use a pseuodocycle-based shape complementarity optimizing approach to design nanomolar binders to diverse ligands, including the flexible and polar methotrexate and thyroxine, that can be directly converted into ligand-gated nanopores and chemically induced dimerization systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572522">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572522" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572522">
        <p class="paperTitle">Xylosyltransferase Bump-and-hole Engineering to Chemically Manipulate Proteoglycans in Mammalian Cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572522" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572522" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, Z.; Di Vagno, L.; Ni Cheallaigh, A.; Sammon, D.; Briggs, D. C.; Chung, N.; Chang, V.; Mahoney, K. E.; Cioce, A.; Murphy, L. D.; Chen, Y.-H.; Narimatsu, Y.; Miller, R. L.; Willems, L. I.; Malaker, S. A.; Miller, G. J.; Hohenester, E.; Schumann, B.</p>
        <p class="info">Score: 25.6, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572522' target='https://doi.org/10.1101/2023.12.20.572522'> 10.1101/2023.12.20.572522</a></p>
        <p class="abstract">Mammalian cells orchestrate signalling through interaction events on their surfaces. Proteoglycans are an intricate part of these interactions, carrying large glycosaminoglycan polysaccharides that recruit signalling molecules. Despite their importance in development, cancer and neurobiology, a relatively small number of proteoglycans have been identified. In addition to the complexity of glycan extension, biosynthetic redundancy in the first protein glycosylation step by two xylosyltransferase isoenzymes XT1 and XT2 complicates annotation of proteoglycans. Here, we develop a chemical strategy that allows profiling of cellular proteoglycans. By employing a tactic termed bump-and-hole engineering, we engineer the two isoenzymes XT1 and XT2 to specifically transfer a chemically modified xylose analogue to target proteins. The chemical modification contains a bioorthogonal tag, allowing the ability to visualise and profile target proteins modified by both transferases in mammalian cells as well as pinpointing glycosylation sites by mass spectrometry. Engineered XT enzymes permit a view into proteoglycan biology that is orthogonal to conventional techniques in biochemistry.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.573103">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.573103" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.573103">
        <p class="paperTitle">Atomic context-conditioned protein sequence design using LigandMPNN</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.573103" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.573103" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dauparas, J.; Lee, G. R.; Pecoraro, R.; An, L.; Anishchenko, I.; Glasscock, C.; Baker, D.</p>
        <p class="info">Score: 23.2, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.573103' target='https://doi.org/10.1101/2023.12.22.573103'> 10.1101/2023.12.22.573103</a></p>
        <p class="abstract">Protein sequence design in the context of small molecules, nucleotides, and metals is critical to enzyme and small molecule binder and sensor design, but current state-of-the-art deep learning-based sequence design methods are unable to model non-protein atoms and molecules. Here, we describe a deep learning-based protein sequence design method called LigandMPNN that explicitly models all non-protein components of biomolecular systems. LigandMPNN significantly outperforms Rosetta and ProteinMPNN on native backbone sequence recovery for residues interacting with small molecules (63.3% vs. 50.4% &amp; 50.5%), nucleotides (50.5% vs. 35.2% &amp; 34.0%), and metals (77.5% vs. 36.0% &amp; 40.6%). LigandMPNN generates not only sequences but also sidechain conformations to allow detailed evaluation of binding interactions. Experimental characterization demonstrates that LigandMPNN can generate small molecule and DNA-binding proteins with high affinity and specificity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.572959">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.572959" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.572959">
        <p class="paperTitle">Structural analysis of the dynamic ribosome-translocon complex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.572959" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.572959" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lewis, A. J. O.; Hegde, R. S.</p>
        <p class="info">Score: 20.4, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.572959' target='https://doi.org/10.1101/2023.12.22.572959'> 10.1101/2023.12.22.572959</a></p>
        <p class="abstract">The protein translocon at the endoplasmic reticulum comprises the Sec61 translocation channel and numerous accessory factors that collectively facilitate the biogenesis of secretory and membrane proteins. Here, we leveraged recent advances in cryo-EM and structure prediction to derive insights into several novel configurations of the ribosome-translocon complex. We show how a transmembrane domain (TMD) in a looped configuration passes through the Sec61 lateral gate during membrane insertion; how a nascent chain can bind and constrain the conformation of ribosomal protein uL22; and how the translocon-associated protein (TRAP) complex can adjust its position during different stages of protein biogenesis. Most unexpectedly, we find that a large proportion of translocon complexes contains RAMP4 intercalated into Sec61&#39;s lateral gate, widening Sec61&#39;s central pore and contributing to its hydrophilic interior. These structures lead to mechanistic hypotheses for translocon function and highlight a remarkably plastic machinery whose conformations and composition adjust dynamically to its diverse range of substrates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.573053">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.573053" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.573053">
        <p class="paperTitle">Structure of the minor spliceosomal U11 snRNP</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.573053" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.573053" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhao, J.; Peter, D.; Brandina, I.; Liu, X.; Galej, W. P.</p>
        <p class="info">Score: 17.1, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.573053' target='https://doi.org/10.1101/2023.12.22.573053'> 10.1101/2023.12.22.573053</a></p>
        <p class="abstract">The minor spliceosome catalyses the excision of U12-dependent introns from pre-mRNAs. These introns are rare, but their removal is critical for cell viability. We obtained a cryo-EM reconstruction of the 13-subunit U11 snRNP complex, revealing structures of U11 snRNA and five minor spliceosome-specific factors. U11 snRNP appears strikingly different from the equivalent major spliceosome U1 snRNP. SNRNP25 and SNRNP35 form a dimer, which specifically recognises U11 snRNA. PDCD7 forms extended helices, which bridge SNRNP25 and SNRNP48, located at the distal ends of the particle. SNRNP48 forms multiple interfaces with U11 snRNP and, together with ZMAT5, are positioned near the 5&#39;-end of the U11 snRNA and likely stabilise the binding of the incoming 5&#39;-SS. Our structure provides mechanistic insights into U12-dependent intron recognition and the evolution of the splicing machinery.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.572998">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.572998" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.572998">
        <p class="paperTitle">Three-step mechanism of promoter escape by RNA polymerase II</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.572998" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.572998" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhan, Y.; Grabbe, F.; Oberbeckmann, E.; Dienemann, C.; Cramer, P.</p>
        <p class="info">Score: 16.4, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.572998' target='https://doi.org/10.1101/2023.12.22.572998'> 10.1101/2023.12.22.572998</a></p>
        <p class="abstract">The transition from transcription initiation to elongation is highly regulated in human cells but remains incompletely understood at the structural level. In particular, it is unclear how interactions between RNA polymerase (Pol) II and initiation factors are broken to enable promoter escape. Here we reconstitute Pol II promoter escape in vitro, determine high-resolution structures of five transition intermediates, and show that promoter escape occurs in three major steps. First, the growing RNA transcript displaces the B-reader element of the initiation factor TFIIB without evicting TFIIB. Second, rewinding of the upstream edge of the growing DNA bubble evicts TFIIA, TFIIB and TBP and repositions parts of TFIIE and TFIIF. Third, binding of DSIF and NELF evicts TFIIE and TFIIH, establishing the paused elongation complex. This three-step model for promoter escape fills a gap in our understanding of the initiation-elongation transition of Pol II transcription.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572533">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572533" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572533">
        <p class="paperTitle">MUT-7 exoribonuclease activity and localisation are mediated by an ancient domain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572533" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572533" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Busetto, V.; Pshanichnaya, L.; Lichtenberger, R.; Hann, S.; Ketting, R. F.; Falk, S.</p>
        <p class="info">Score: 11.5, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572533' target='https://doi.org/10.1101/2023.12.20.572533'> 10.1101/2023.12.20.572533</a></p>
        <p class="abstract">The MUT-7 family of 3-5 exoribonucleases is evolutionarily conserved across the animal kingdom and plays essential roles in small RNA production in the germline. Most MUT-7 homologs carry a C-terminal domain of unknown function named MUT7-C appended to the exoribonuclease domain. Our analysis shows that the MUT7-C is evolutionary ancient, as a minimal version of the domain exists as an individual protein in prokaryotes. In animals, MUT7-C has acquired an insertion that diverged during evolution, expanding its functions. C. elegans MUT-7 contains a specific insertion within MUT7-C, which allows binding to MUT-8 and, consequently, MUT-7 recruitment to germ granules. In addition, in C. elegans and human MUT-7, the MUT7-C domain contributes to RNA binding and is thereby crucial for nuclease activity. This RNA-binding function most likely represents the ancestral function of the MUT7-C domain. Overall, this study sheds light on MUT7-C and assigns two functions to this previously uncharacterised domain.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572340">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572340" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572340">
        <p class="paperTitle">Structure and methyl-lysine binding selectivity of the HUSH complex subunit MPP8</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572340" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572340" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nikolopoulos, N.; Oda, S.-i.; Prigozhin, D. M.; Modis, Y.</p>
        <p class="info">Score: 9.5, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572340' target='https://doi.org/10.1101/2023.12.21.572340'> 10.1101/2023.12.21.572340</a></p>
        <p class="abstract">The Human Silencing Hub (HUSH) guards the genome from the pathogenic effects of retroelement expression. Composed of MPP8, TASOR, and Periphilin-1, HUSH recognizes actively transcribed retrotransposed sequences by the presence of long (&gt;1.5-kb) nascent transcripts without introns. HUSH recruits effectors that alter chromatin structure, degrade transcripts, and deposit transcriptionally repressive epigenetic marks. Structural and biochemical data for the three HUSH subunits are sparse. Here, we report the crystal structure of the C-terminal domain (CTD) of MPP8 necessary for HUSH activity. The MPP8 CTD consists of five ankyrin repeats followed by a domain with structural homology to the PINIT domains of Siz/PIAS-family SUMO E3 ligases. AlphaFold-Multimer modeling predicts that the MPP8 CTD forms extended interaction interfaces with a SPOC domain and a domain with a novel fold, DomI, in TASOR. The chromodomain of MPP8, known to bind the repressive mark H3K9me3, binds with similar or higher affinity to H3K9-like sequences in the mammalian H3K9 methyltransferase subunits SETDB1, ATF7IP, G9a, and GLP. Hence, MPP8 promotes heterochromatinization by recruiting H3K9 methyltransferases. Our work identifies novel structural elements in MPP8 that are required for HUSH complex assembly and silencing, thereby fulfilling vital functions in controlling the transcription of retrotransposons of endogenous and viral origin.

Key PointsO_LIStructure of MPP8 CTD reveals ankyrin repeats and PINIT-like domain required for HUSH activity
C_LIO_LIAlphaFold predicts a large interaction interface between MPP8 CTD and TASOR SPOC and DomI domains
C_LIO_LIThe MPP8 chromodomain binds trimethyl-lysines in ATF7IP and GLP with higher affinity than H3K9me3
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572332">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572332" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572332">
        <p class="paperTitle">A new GUV-based assay to reconstitute membrane tethering in vitro demonstrates the vesicle tethering ability of the selective autophagy scaffold Atg11</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572332" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572332" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Andhare, D.; Ragusa, M. J.</p>
        <p class="info">Score: 9.5, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572332' target='https://doi.org/10.1101/2023.12.19.572332'> 10.1101/2023.12.19.572332</a></p>
        <p class="abstract">Membrane tethering is essential for the generation of organelle contact sites and to anchor incoming vesicles to their target membranes before vesicle fusion. During autophagy in yeast, tethering of 30 nm vesicles to cargo is one of the first steps in the generation of the isolation membrane that engulfs the cargo to be degraded. While membrane tethering is critical for cellular function, many of the current biochemical techniques to assay for membrane tethering rely on indirect readouts and are limited in their ability to monitor protein localization at sites of tethering. As such, we developed a fluorescence-microscopy-based GUV liposome tethering assay (GLT) to directly visualize membrane tethering and monitor protein localization at tethering sites simultaneously. We initially used GLT with engineered membrane tethers to demonstrate the ease of use, versatility and sensitivity of the assay. We also demonstrated that the selective autophagy scaffolding protein Atg11 can bind, and tether negatively charged membranes but is unable to bind GUVs mimicking the charge of the outer mitochondrial membrane. Atg11 instead requires the selective autophagy receptor Atg32 to be recruited to mitochondrial membranes. Lastly, we demonstrate that Atg11 bound to Atg32 on GUVs can tether negatively charged vesicles to GUVs. Collectively, our results reconstitute one of the first steps during the initiation of mitochondrial autophagy and highlight the versatility of GLT to study a range of membrane tethering events biochemically.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572336">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572336" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572336">
        <p class="paperTitle">Structure of an insect gustatory receptor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572336" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572336" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Frank, H. M.; Walujkar, S.; Walsh, R. M.; Laursen, W. J.; Theobald, D. L.; Garrity, P. A.; Gaudet, R.</p>
        <p class="info">Score: 8.0, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572336' target='https://doi.org/10.1101/2023.12.19.572336'> 10.1101/2023.12.19.572336</a></p>
        <p class="abstract">Gustatory Receptors (GRs) are critical for insect chemosensation and are potential targets for controlling pests and disease vectors. However, GR structures have not been experimentally determined. We present structures of Bombyx mori Gr9 (BmGr9), a fructose-gated cation channel, in agonist-free and fructose-bound states. BmGr9 forms a tetramer similar to distantly related insect Olfactory Receptors (ORs). Upon fructose binding, BmGr9s ion channel gate opens through helix S7b movements. In contrast to ORs, BmGR9s ligand-binding pocket, shaped by a kinked helix S4 and a shorter extracellular S3-S4 loop, is larger and solvent accessible in both agonist-free and fructose-bound states. Also unlike ORs, fructose binding by BmGr9 involves helix S5 and a binding pocket lined with aromatic and polar residues. Structure-based sequence alignments reveal distinct patterns of ligand-binding pocket residue conservation in GR subfamilies associated with distinct ligand classes. These data provide insight into the molecular basis of GR ligand specificity and function.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
