<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Structure of the human Duffy antigen receptor
Authors: Saha, S.; Khanppnavar, B.; Maharana, J.; Kim, H.; Carino, C. M. C.; Daly, C.; Houston, S.; Kumari, P.; Yadav, P. N.; Plouffe, B.; Inoue, A.; Chung, K. Y.; Banerjee, R.; Korkhov, V. M.; Shukla, A. K.
Score: 28.9, Published: 2023-07-10 DOI: 10.1101/2023.07.09.548245
The Duffy antigen receptor, also known as FY glycoprotein or CD234, is a seven transmembrane protein expressed primarily at the surface of red blood cells, which displays promiscuous binding to multiple chemokines.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Structure of the human Duffy antigen receptor
Authors: Saha, S.; Khanppnavar, B.; Maharana, J.; Kim, H.; Carino, C. M. C.; Daly, C.; Houston, S.; Kumari, P.; Yadav, P. N.; Plouffe, B.; Inoue, A.; Chung, K. Y.; Banerjee, R.; Korkhov, V. M.; Shukla, A. K.
Score: 28.9, Published: 2023-07-10 DOI: 10.1101/2023.07.09.548245
The Duffy antigen receptor, also known as FY glycoprotein or CD234, is a seven transmembrane protein expressed primarily at the surface of red blood cells, which displays promiscuous binding to multiple chemokines." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-16T10:39:40+00:00" />
<meta property="article:modified_time" content="2023-07-16T10:39:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Structure of the human Duffy antigen receptor
Authors: Saha, S.; Khanppnavar, B.; Maharana, J.; Kim, H.; Carino, C. M. C.; Daly, C.; Houston, S.; Kumari, P.; Yadav, P. N.; Plouffe, B.; Inoue, A.; Chung, K. Y.; Banerjee, R.; Korkhov, V. M.; Shukla, A. K.
Score: 28.9, Published: 2023-07-10 DOI: 10.1101/2023.07.09.548245
The Duffy antigen receptor, also known as FY glycoprotein or CD234, is a seven transmembrane protein expressed primarily at the surface of red blood cells, which displays promiscuous binding to multiple chemokines."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Structure of the human Duffy antigen receptor\nAuthors: Saha, S.; Khanppnavar, B.; Maharana, J.; Kim, H.; Carino, C. M. C.; Daly, C.; Houston, S.; Kumari, P.; Yadav, P. N.; Plouffe, B.; Inoue, A.; Chung, K. Y.; Banerjee, R.; Korkhov, V. M.; Shukla, A. K.\nScore: 28.9, Published: 2023-07-10 DOI: 10.1101/2023.07.09.548245\nThe Duffy antigen receptor, also known as FY glycoprotein or CD234, is a seven transmembrane protein expressed primarily at the surface of red blood cells, which displays promiscuous binding to multiple chemokines.",
  "keywords": [
    
  ],
  "articleBody": " Structure of the human Duffy antigen receptor\nAuthors: Saha, S.; Khanppnavar, B.; Maharana, J.; Kim, H.; Carino, C. M. C.; Daly, C.; Houston, S.; Kumari, P.; Yadav, P. N.; Plouffe, B.; Inoue, A.; Chung, K. Y.; Banerjee, R.; Korkhov, V. M.; Shukla, A. K.\nScore: 28.9, Published: 2023-07-10 DOI: 10.1101/2023.07.09.548245\nThe Duffy antigen receptor, also known as FY glycoprotein or CD234, is a seven transmembrane protein expressed primarily at the surface of red blood cells, which displays promiscuous binding to multiple chemokines. Not only does it serve as the basis of the Duffy blood group system but it also acts as the primary attachment site for malarial parasite Plasmodium vivax on erythrocytes and as one of the nucleating receptors for the pore forming toxins secreted by Staphylococcus aureus. Despite a predicted 7TM architecture and efficient binding to a spectrum of chemokines, it fails to exhibit canonical second messenger response such as calcium release, likely due to a lack of G protein coupling. Unlike prototypical GPCRs and {beta}-arrestin-biased atypical chemokine receptors, the Duffy antigen receptor also appears to lack {beta}-arrestin binding, making it an enigmatic 7TM chemokine receptor. In order to decipher the molecular mechanism of this intriguing functional divergence exhibited by the Duffy antigen receptor, we have determined its cryo-EM structure in complex with a C-C type chemokine, CCL7. The structure reveals a relatively superficial binding mode of CCL7, with the N-terminus of the receptor serving as the key interaction interface, and a partially formed orthosteric binding pocket lacking the second site for chemokine recognition compared to prototypical chemokine receptors. The structural framework allows us to employ HDX-MS approach to uncover ligand-induced structural changes in the receptor and draw important insights into the promiscuous nature of chemokine binding. Interestingly, we also observe a dramatic shortening of TM5 and 6 on the intracellular side, compared to prototypical GPCRs, which precludes the coupling of canonical signal-transducers namely G proteins, GRKs and {beta}-arrestins, as demonstrated through extensive cellular assays. Taken together, our study uncovers a previously unknown structural mechanism that imparts unique functional divergence on the 7TM fold encoded in the Duffy antigen receptor while maintaining its scavenging function and should facilitate the designing of novel therapeutics targeting this receptor.\nEngineering and Evolution of the Complete Reductive Glycine Pathway in Saccharomyces cerevisiae for Formate and CO2 Assimilation\nAuthors: Bysani, V. R.; Alam, A. S. M.; Bar-Even, A.; Machens, F.\nScore: 17.0, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548313\nUsing captured CO2 and C1-feedstocks like formate and methanol derived from electrochemical activation of CO2 are key solutions for transforming industrial processes towards a circular carbon economy. Engineering formate and CO2-based growth in the biotechnologically relevant yeast Saccharomyces cerevisiae could boost the emergence of a formate-mediated circular bio-economy. This study adopts a growth-coupled selection scheme for modular implementation of the Reductive Glycine Pathway (RGP) and subsequent Adaptive Laboratory Evolution (ALE) to enable formate and CO2 assimilation for biomass formation in yeast. We first constructed a serine biosensor strain and then implemented the serine synthesis module of the RGP into yeast, establishing glycine and serine synthesis from formate and CO2. ALE improved the RGP-dependent growth by 8-fold. 13C-labeling experiments reveal glycine, serine, and pyruvate synthesis via the RGP, demonstrating the complete pathway activity. Further, we reestablished formate and CO2-dependent growth in non-evolved biosensor strains via reverse-engineering a mutation in GDH1 identified from ALE. This mutation led to significantly more 13C-formate assimilation than in WT without any selection or overexpression of the RGP. Overall, we demonstrated the activity of the complete RGP, showing evidence for carbon transfer from formate to pyruvate coupled with CO2 assimilation.\nDiversity and modularity of tyrosine-accepting tRNA-like structures\nAuthors: Sherlock, M. E.; Langeberg, C. J.; Kieft, J. S.\nScore: 12.4, Published: 2023-07-10 DOI: 10.1101/2023.07.08.548219\nCertain positive-sense single-stranded RNA viruses contain elements at their 3 termini that structurally mimic tRNAs. These tRNA-like structures (TLSs) are classified based on which amino acid is covalently added to the 3 end by host aminoacyl-tRNA synthetase. Recently, a cryoEM reconstruction of a representative tyrosine-accepting tRNA-like structure (TLSTyr) from brome mosaic virus (BMV) revealed a unique mode of recognition of the viral anticodon-mimicking domain by tyrosyl-tRNA synthetase. Some viruses in the hordeivirus genus of Virgaviridae are also selectively aminoacylated with tyrosine, yet these TLS RNAs have a different architecture in the 5 domain that comprises the atypical anticodon loop mimic. Herein, we present bioinformatic and biochemical data supporting a distinct secondary structure for the 5' domain of the hordeivirus TLSTyr compared to those in Bromoviridae. Despite forming a different secondary structure, the 5' domain is necessary to achieve robust in vitro aminoacylation. Furthermore, a chimeric RNA containing the 5' domain from the BMV TLSTyr and the 3' domain from a hordeivirus TLSTyr are aminoacylated, illustrating modularity in these structured RNA elements. We propose that the structurally distinct 5' domain of the hordeivirus TLSTyrs performs the same role in mimicking the anticodon loop as its counterpart in the BMV TLSTyr. Finally, these structurally and phylogenetically divergent types of TLSTyr provide insight into the evolutionary connections between all classes of viral tRNA-like structures.\nα-carboxysome size is controlled by the disordered scaffold protein CsoS2\nAuthors: Oltrogge, L. M.; Chen, A. W.; Chaijarasphong, T.; Turnsek, J. B.; Savage, D. F.\nScore: 8.7, Published: 2023-07-08 DOI: 10.1101/2023.07.07.548173\nCarboxysomes are protein microcompartments that function in the bacterial CO2 concentrating mechanism (CCM) to facilitate CO2 assimilation. To do so, carboxysomes assemble from thousands of constituent proteins into an icosahedral shell which encapsulates the enzymes rubisco and carbonic anhydrase to form structures typically \u003e100 nm and \u003e300 megadaltons. Although many of the protein interactions driving the assembly process have been determined, it remains unknown how size and composition are precisely controlled. Here we show that the size of -carboxysomes is controlled by the disordered scaffolding protein CsoS2. CsoS2 contains two classes of related peptide repeats which bind to the shell in a distinct fashion, and our data indicate that size is controlled by the relative number of these interactions. We propose an energetic and structural model wherein the two repeat classes bind at the junction of shell hexamers but differ in their preferences for the shell contact angles, and thus the local curvature. Thus, in contrast to other models driven by cargo or shell alone, our data show that CsoS2 and shell proteins interact to collectively achieve the large size and monodispersity of -carboxysomes.\nAlgal kainoid synthases exhibit substrate-dependent hydroxylation and cyclization activities\nAuthors: Hopiavuori, A. R.; McKinnie, S. M. K.\nScore: 8.3, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548461\nFeII/-ketoglutarate-dependent dioxygenases (Fe/KG) are a large enzyme family that functionalize C-H bonds on diverse organic substrates. Although Fe/KG homologs catalyze an array of chemically useful reactions, hydroxylation typically predominates. Microalgal DabC uniquely forms a novel C-C bond to construct the bioactive pyrrolidine ring in domoic acid biosynthesis. However, this kainoid synthase exclusively performs a stereospecific hydroxylation reaction on its cis substrate regioisomer. Mechanistic and kinetic analyses with native and alternative substrates identified a 20-fold rate increase in DabC radical cyclization over {beta}-hydroxylation, with no observable 1,5-hydrogen atom transfer. Moreover, this dual activity was conserved among macroalgal RadC1 and KabC homologs and provided insight into substrate recognition and reactivity trends. Investigation of this substrate-dependent chemistry improves our understanding of Fe/KG enzymes and their biocatalytic application.\nDistinct phases of cellular signaling revealed by time-resolved protein synthesis\nAuthors: Lee, G.; Muir, T. W.\nScore: 7.3, Published: 2023-07-10 DOI: 10.1101/2023.07.10.548208\nThe post-translational regulation of protein function is involved in most cellular processes. As such, synthetic biology tools that operate at this level provide opportunities for manipulating cellular states. Here, we deploy a proximity-triggered protein trans-splicing technology to enable the time-resolved synthesis of target proteins from pre-made parts. The modularity of the strategy allows for the addition or removal of various control elements as a function of the splicing reaction, in the process permitting the cellular location and/or activity state of starting materials and products to be differentiated. The approach is applied to a diverse set of proteins, including the kinase oncofusions BCR/ABL and DNAJB1/PRKACA where dynamic cellular phosphorylation events are dissected, revealing distinct phases of signaling and identifying molecular players connecting the oncofusion to cancer transformation as novel therapeutic targets of cancer cells. We envision that the tools and control strategies developed herein will allow the activity of both naturally occurring and designer proteins to be harnessed for basic and applied research.\nBifunctional small molecules that induce nuclearlocalization and targeted transcriptional regulation\nAuthors: Gibson, W. J.; Sadagopan, A.; Shoba, V. M.; Choudhary, A.; Meyerson, M. L.; Schreiber, S. L.\nScore: 38.0, Published: 2023-07-07 DOI: 10.1101/2023.07.07.548101\nThe aberrant localization of proteins in cells is a key factor in the development of various diseases, including cancer and neurodegenerative disease. To better understand and potentially manipulate protein localization for therapeutic purposes, we engineered bifunctional compounds that bind to proteins in separate cellular compartments. We show these compounds induce nuclear import of cytosolic cargoes, using nuclear-localized BRD4 as a \"carrier\" for co-import and nuclear trapping of cytosolic proteins. We use this system to calculate kinetic constants for passive diffusion across the nuclear pore and demonstrate single-cell heterogeneity in response to these bifunctional molecules, with cells requiring high carrier to cargo expression for complete import. We also observe incorporation of cargoes into BRD4-containing condensates. Proteins shown to be substrates for nuclear transport include oncogenic mutant nucleophosmin (NPM1c) and mutant PI3K catalytic subunit alpha (PIK3CAE545K), suggesting potential applications to cancer treatment. In addition, we demonstrate that chemical-induced localization of BRD4 to cytosolic-localized DNA-binding proteins, namely, IRF1 with a nuclear export signal, induces target gene expression. These results suggest that induced localization of proteins with bifunctional molecules enables the rewiring of cell circuitry with significant implications for disease therapy.\nA bespoke analytical workflow for the confident identification of sulfopeptides and their discrimination from phosphopeptides\nAuthors: Daly, L. A.; Byrne, D. P.; Perkins, S.; Brownridge, P. J.; McDonnell, E.; Jones, A. R.; Eyers, P. A.; Eyers, C. E.\nScore: 6.7, Published: 2023-07-15 DOI: 10.1101/2023.07.15.549150\nProtein tyrosine sulfation (sY) is a post-translational modification (PTM) catalysed by Golgi-resident Tyrosyl Protein Sul-foTransferases (TPSTs). Information on protein tyrosine sulfation is currently limited to ~50 human proteins with only a handful having verified sites of sulfation. The contribution of this chemical moiety for the regulation of biological processes, both inside and outside the cell, remains poorly defined, in large part due to analytical limitations. Mass spectrometry-based proteomics is the method of choice for PTM analysis, but has yet to be applied for the systematic investigation and large-scale analysis of biomolecular sulfation (constituting the sulfome), primarily due to issues associated with discrimination of sY- from phosphotyrosine (pY)-containing peptides. In this study, we developed a mass spectrometry (MS)-based workflow centred on the characterization of sY-peptides, incorporating optimised Zr4+-IMAC and TiO2 enrichment strategies. Extensive characterization of a panel of sY- and pY-peptides using an array of MS fragmentation regimes (CID, HCD, EThcC, ETciD, UVPD) highlights differences in the ability to generate site-determining product ions, which can be exploited to differentiate sulfated peptides from nominally isobaric phosphopeptides based on precursor ion neutral loss at low collision energy. Application of our analytical workflow to a HEK-293 cell extracellular secretome facilitated identification of 21 new sulfotyrosine-containing proteins, several of which we validate enzymatically using in vitro sulfation assays. This study demonstrates the applicability of our strategy for confident, high-throughput, sulfomics studies, and reveals new sY interplay between enzymes relevant to both protein and glycan sulfation.\nDecrypting the languages of allostery in membrane-bound K-Ras4B using four complementary in silico approaches\nAuthors: Castelli, M.; Marchetti, F.; Serapian, S. A.; Colombo, G.\nScore: 6.3, Published: 2023-07-14 DOI: 10.1101/2023.07.14.549022\nProteins are evolved molecular machines whose diverse biochemical functions are (also) dynamically regulated by allostery, through which even faraway protein residues can conformationally communicate. Allostery can express itself in different ways, akin to different \"languages\": allosteric control pathways predominating in an unperturbed protein are superseded by others as soon as a perturbation arises--e.g, mutation--that alters its function (pathologically or not). Accurately modeling these often-unintuitive phenomena could therefore help explain functional changes in a specific protein whenever they are unclear. Unbiased molecular dynamics (MD) simulations are a possibility; however, since allostery can operate at longer timescales than those accessible by MD, simulations require integration with a reliable method able to, e.g., detect regions of incipient allosteric change, or likely perturbation pathways. Several such (valid) methods exist, but are typically applied singularly, only yielding a partial (\"monolinguistic\") picture of allostery: we argue their joint application could significantly enrich this picture. To prove this, we perform unbiased MD simulations (~100 s) of the widely studied, allosterically active oncotarget K-Ras4B, solvated and embedded in a phospholipid membrane, proceeding to decrypt its allostery using four showcase \"languages\": Distance Fluctuation analysis and the Shortest Path Map capture allosteric communication hotspots at equilibrium; Anisotropic Thermal Diffusion and Dynamic Non-Equilibrium MD assess them once the GTP that oncogenically activates K-Ras4B is, respectively, either superheated or hydrolyzed. \"Languages\" provide a uniquely articulate, mutually coherent, experimentally consistent picture of allostery in K-Ras4B. At equilibrium, pathways stretch from the membrane-embedded hypervariable region all the way to the active site, touching known flexible allosteric \"switches\" and proposed pockets. Upon GTP cleavage/perturbation, known to inactivate K-Ras4B, allosteric signals most reverberate on switches and interfaces that recruit effector proteins. Our work highlights the benefits of integrating different allostery detection methods, even in conjunction with unbiased MD.\nThe pathogenic T42A mutation in SHP2 rewires interaction specificity and enhances signaling\nAuthors: van Vlimmeren, A. E.; Voleti, R.; Chartier, C. A.; Jiang, Z.; Karandur, D.; Shah, N. H.\nScore: 6.3, Published: 2023-07-10 DOI: 10.1101/2023.07.10.548257\nMutations in the tyrosine phosphatase SHP2 are associated with various human diseases. Most of these mutations increase the basal catalytic activity of SHP2 by disrupting auto-inhibitory interactions between its phosphatase domain and N-terminal SH2 (phosphotyrosine recognition) domain. By contrast, several other disease-associated mutations located in the ligand-binding pockets of the N- or C-terminal SH2 domains do not increase basal activity and likely exert their pathogenicity through alternative mechanisms. We lack a molecular understanding of how these SH2 mutations impact SHP2 structure, activity, and signaling. Here, we characterize five SHP2 SH2 domain ligand-binding pocket mutants through a combination of high-throughput biochemical screens, biophysical and biochemical measurements, molecular dynamics simulations, and cellular assays. We show that, while several of these mutants alter binding affinity to phosphorylation sites, the T42A mutation in the N-SH2 domain is unique in that it also alters ligand-binding specificity. The functional consequence of this altered specificity is that the T42A mutant has biased sensitivity toward a subset of activating ligands. Our study highlights an example of a nuanced mechanism of action for a disease-associated mutation, characterized by a change in protein-protein interaction specificity that alters enzyme activation.\n",
  "wordCount" : "2435",
  "inLanguage": "en",
  "datePublished": "2023-07-16T10:39:40Z",
  "dateModified": "2023-07-16T10:39:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta"><span>updated on July 16, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.09.548245">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.09.548245" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.09.548245">
        <p class="paperTitle">Structure of the human Duffy antigen receptor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.09.548245" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.09.548245" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Saha, S.; Khanppnavar, B.; Maharana, J.; Kim, H.; Carino, C. M. C.; Daly, C.; Houston, S.; Kumari, P.; Yadav, P. N.; Plouffe, B.; Inoue, A.; Chung, K. Y.; Banerjee, R.; Korkhov, V. M.; Shukla, A. K.</p>
        <p class="info">Score: 28.9, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.09.548245' target='https://doi.org/10.1101/2023.07.09.548245'> 10.1101/2023.07.09.548245</a></p>
        <p class="abstract">The Duffy antigen receptor, also known as FY glycoprotein or CD234, is a seven transmembrane protein expressed primarily at the surface of red blood cells, which displays promiscuous binding to multiple chemokines. Not only does it serve as the basis of the Duffy blood group system but it also acts as the primary attachment site for malarial parasite Plasmodium vivax on erythrocytes and as one of the nucleating receptors for the pore forming toxins secreted by Staphylococcus aureus. Despite a predicted 7TM architecture and efficient binding to a spectrum of chemokines, it fails to exhibit canonical second messenger response such as calcium release, likely due to a lack of G protein coupling. Unlike prototypical GPCRs and {beta}-arrestin-biased atypical chemokine receptors, the Duffy antigen receptor also appears to lack {beta}-arrestin binding, making it an enigmatic 7TM chemokine receptor. In order to decipher the molecular mechanism of this intriguing functional divergence exhibited by the Duffy antigen receptor, we have determined its cryo-EM structure in complex with a C-C type chemokine, CCL7. The structure reveals a relatively superficial binding mode of CCL7, with the N-terminus of the receptor serving as the key interaction interface, and a partially formed orthosteric binding pocket lacking the second site for chemokine recognition compared to prototypical chemokine receptors. The structural framework allows us to employ HDX-MS approach to uncover ligand-induced structural changes in the receptor and draw important insights into the promiscuous nature of chemokine binding. Interestingly, we also observe a dramatic shortening of TM5 and 6 on the intracellular side, compared to prototypical GPCRs, which precludes the coupling of canonical signal-transducers namely G proteins, GRKs and {beta}-arrestins, as demonstrated through extensive cellular assays. Taken together, our study uncovers a previously unknown structural mechanism that imparts unique functional divergence on the 7TM fold encoded in the Duffy antigen receptor while maintaining its scavenging function and should facilitate the designing of novel therapeutics targeting this receptor.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548313">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548313" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548313">
        <p class="paperTitle">Engineering and Evolution of the Complete Reductive Glycine Pathway in Saccharomyces cerevisiae for Formate and CO2 Assimilation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548313" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548313" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bysani, V. R.; Alam, A. S. M.; Bar-Even, A.; Machens, F.</p>
        <p class="info">Score: 17.0, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548313' target='https://doi.org/10.1101/2023.07.10.548313'> 10.1101/2023.07.10.548313</a></p>
        <p class="abstract">Using captured CO2 and C1-feedstocks like formate and methanol derived from electrochemical activation of CO2 are key solutions for transforming industrial processes towards a circular carbon economy. Engineering formate and CO2-based growth in the biotechnologically relevant yeast Saccharomyces cerevisiae could boost the emergence of a formate-mediated circular bio-economy. This study adopts a growth-coupled selection scheme for modular implementation of the Reductive Glycine Pathway (RGP) and subsequent Adaptive Laboratory Evolution (ALE) to enable formate and CO2 assimilation for biomass formation in yeast. We first constructed a serine biosensor strain and then implemented the serine synthesis module of the RGP into yeast, establishing glycine and serine synthesis from formate and CO2. ALE improved the RGP-dependent growth by 8-fold. 13C-labeling experiments reveal glycine, serine, and pyruvate synthesis via the RGP, demonstrating the complete pathway activity. Further, we reestablished formate and CO2-dependent growth in non-evolved biosensor strains via reverse-engineering a mutation in GDH1 identified from ALE. This mutation led to significantly more 13C-formate assimilation than in WT without any selection or overexpression of the RGP. Overall, we demonstrated the activity of the complete RGP, showing evidence for carbon transfer from formate to pyruvate coupled with CO2 assimilation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.08.548219">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.08.548219" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.08.548219">
        <p class="paperTitle">Diversity and modularity of tyrosine-accepting tRNA-like structures</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.08.548219" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.08.548219" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sherlock, M. E.; Langeberg, C. J.; Kieft, J. S.</p>
        <p class="info">Score: 12.4, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.08.548219' target='https://doi.org/10.1101/2023.07.08.548219'> 10.1101/2023.07.08.548219</a></p>
        <p class="abstract">Certain positive-sense single-stranded RNA viruses contain elements at their 3 termini that structurally mimic tRNAs. These tRNA-like structures (TLSs) are classified based on which amino acid is covalently added to the 3 end by host aminoacyl-tRNA synthetase. Recently, a cryoEM reconstruction of a representative tyrosine-accepting tRNA-like structure (TLSTyr) from brome mosaic virus (BMV) revealed a unique mode of recognition of the viral anticodon-mimicking domain by tyrosyl-tRNA synthetase. Some viruses in the hordeivirus genus of Virgaviridae are also selectively aminoacylated with tyrosine, yet these TLS RNAs have a different architecture in the 5 domain that comprises the atypical anticodon loop mimic. Herein, we present bioinformatic and biochemical data supporting a distinct secondary structure for the 5&#39; domain of the hordeivirus TLSTyr compared to those in Bromoviridae. Despite forming a different secondary structure, the 5&#39; domain is necessary to achieve robust in vitro aminoacylation. Furthermore, a chimeric RNA containing the 5&#39; domain from the BMV TLSTyr and the 3&#39; domain from a hordeivirus TLSTyr are aminoacylated, illustrating modularity in these structured RNA elements. We propose that the structurally distinct 5&#39; domain of the hordeivirus TLSTyrs performs the same role in mimicking the anticodon loop as its counterpart in the BMV TLSTyr. Finally, these structurally and phylogenetically divergent types of TLSTyr provide insight into the evolutionary connections between all classes of viral tRNA-like structures.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548173">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548173" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548173">
        <p class="paperTitle">α-carboxysome size is controlled by the disordered scaffold protein CsoS2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548173" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548173" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Oltrogge, L. M.; Chen, A. W.; Chaijarasphong, T.; Turnsek, J. B.; Savage, D. F.</p>
        <p class="info">Score: 8.7, Published: 2023-07-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548173' target='https://doi.org/10.1101/2023.07.07.548173'> 10.1101/2023.07.07.548173</a></p>
        <p class="abstract">Carboxysomes are protein microcompartments that function in the bacterial CO2 concentrating mechanism (CCM) to facilitate CO2 assimilation. To do so, carboxysomes assemble from thousands of constituent proteins into an icosahedral shell which encapsulates the enzymes rubisco and carbonic anhydrase to form structures typically &gt;100 nm and &gt;300 megadaltons. Although many of the protein interactions driving the assembly process have been determined, it remains unknown how size and composition are precisely controlled. Here we show that the size of -carboxysomes is controlled by the disordered scaffolding protein CsoS2. CsoS2 contains two classes of related peptide repeats which bind to the shell in a distinct fashion, and our data indicate that size is controlled by the relative number of these interactions. We propose an energetic and structural model wherein the two repeat classes bind at the junction of shell hexamers but differ in their preferences for the shell contact angles, and thus the local curvature. Thus, in contrast to other models driven by cargo or shell alone, our data show that CsoS2 and shell proteins interact to collectively achieve the large size and monodispersity of -carboxysomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548461">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548461" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548461">
        <p class="paperTitle">Algal kainoid synthases exhibit substrate-dependent hydroxylation and cyclization activities</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548461" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548461" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hopiavuori, A. R.; McKinnie, S. M. K.</p>
        <p class="info">Score: 8.3, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548461' target='https://doi.org/10.1101/2023.07.10.548461'> 10.1101/2023.07.10.548461</a></p>
        <p class="abstract">FeII/-ketoglutarate-dependent dioxygenases (Fe/KG) are a large enzyme family that functionalize C-H bonds on diverse organic substrates. Although Fe/KG homologs catalyze an array of chemically useful reactions, hydroxylation typically predominates. Microalgal DabC uniquely forms a novel C-C bond to construct the bioactive pyrrolidine ring in domoic acid biosynthesis. However, this kainoid synthase exclusively performs a stereospecific hydroxylation reaction on its cis substrate regioisomer. Mechanistic and kinetic analyses with native and alternative substrates identified a 20-fold rate increase in DabC radical cyclization over {beta}-hydroxylation, with no observable 1,5-hydrogen atom transfer. Moreover, this dual activity was conserved among macroalgal RadC1 and KabC homologs and provided insight into substrate recognition and reactivity trends. Investigation of this substrate-dependent chemistry improves our understanding of Fe/KG enzymes and their biocatalytic application.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548208">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548208" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548208">
        <p class="paperTitle">Distinct phases of cellular signaling revealed by time-resolved protein synthesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548208" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548208" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, G.; Muir, T. W.</p>
        <p class="info">Score: 7.3, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548208' target='https://doi.org/10.1101/2023.07.10.548208'> 10.1101/2023.07.10.548208</a></p>
        <p class="abstract">The post-translational regulation of protein function is involved in most cellular processes. As such, synthetic biology tools that operate at this level provide opportunities for manipulating cellular states. Here, we deploy a proximity-triggered protein trans-splicing technology to enable the time-resolved synthesis of target proteins from pre-made parts. The modularity of the strategy allows for the addition or removal of various control elements as a function of the splicing reaction, in the process permitting the cellular location and/or activity state of starting materials and products to be differentiated. The approach is applied to a diverse set of proteins, including the kinase oncofusions BCR/ABL and DNAJB1/PRKACA where dynamic cellular phosphorylation events are dissected, revealing distinct phases of signaling and identifying molecular players connecting the oncofusion to cancer transformation as novel therapeutic targets of cancer cells. We envision that the tools and control strategies developed herein will allow the activity of both naturally occurring and designer proteins to be harnessed for basic and applied research.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548101">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548101" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548101">
        <p class="paperTitle">Bifunctional small molecules that induce nuclearlocalization and targeted transcriptional regulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548101" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548101" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gibson, W. J.; Sadagopan, A.; Shoba, V. M.; Choudhary, A.; Meyerson, M. L.; Schreiber, S. L.</p>
        <p class="info">Score: 38.0, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548101' target='https://doi.org/10.1101/2023.07.07.548101'> 10.1101/2023.07.07.548101</a></p>
        <p class="abstract">The aberrant localization of proteins in cells is a key factor in the development of various diseases, including cancer and neurodegenerative disease. To better understand and potentially manipulate protein localization for therapeutic purposes, we engineered bifunctional compounds that bind to proteins in separate cellular compartments. We show these compounds induce nuclear import of cytosolic cargoes, using nuclear-localized BRD4 as a &#34;carrier&#34; for co-import and nuclear trapping of cytosolic proteins. We use this system to calculate kinetic constants for passive diffusion across the nuclear pore and demonstrate single-cell heterogeneity in response to these bifunctional molecules, with cells requiring high carrier to cargo expression for complete import. We also observe incorporation of cargoes into BRD4-containing condensates. Proteins shown to be substrates for nuclear transport include oncogenic mutant nucleophosmin (NPM1c) and mutant PI3K catalytic subunit alpha (PIK3CAE545K), suggesting potential applications to cancer treatment. In addition, we demonstrate that chemical-induced localization of BRD4 to cytosolic-localized DNA-binding proteins, namely, IRF1 with a nuclear export signal, induces target gene expression. These results suggest that induced localization of proteins with bifunctional molecules enables the rewiring of cell circuitry with significant implications for disease therapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.15.549150">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.15.549150" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.15.549150">
        <p class="paperTitle">A bespoke analytical workflow for the confident identification of sulfopeptides and their discrimination from phosphopeptides</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.15.549150" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.15.549150" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Daly, L. A.; Byrne, D. P.; Perkins, S.; Brownridge, P. J.; McDonnell, E.; Jones, A. R.; Eyers, P. A.; Eyers, C. E.</p>
        <p class="info">Score: 6.7, Published: 2023-07-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.15.549150' target='https://doi.org/10.1101/2023.07.15.549150'> 10.1101/2023.07.15.549150</a></p>
        <p class="abstract">Protein tyrosine sulfation (sY) is a post-translational modification (PTM) catalysed by Golgi-resident Tyrosyl Protein Sul-foTransferases (TPSTs). Information on protein tyrosine sulfation is currently limited to ~50 human proteins with only a handful having verified sites of sulfation. The contribution of this chemical moiety for the regulation of biological processes, both inside and outside the cell, remains poorly defined, in large part due to analytical limitations. Mass spectrometry-based proteomics is the method of choice for PTM analysis, but has yet to be applied for the systematic investigation and large-scale analysis of biomolecular sulfation (constituting the sulfome), primarily due to issues associated with discrimination of sY- from phosphotyrosine (pY)-containing peptides. In this study, we developed a mass spectrometry (MS)-based workflow centred on the characterization of sY-peptides, incorporating optimised Zr4&#43;-IMAC and TiO2 enrichment strategies. Extensive characterization of a panel of sY- and pY-peptides using an array of MS fragmentation regimes (CID, HCD, EThcC, ETciD, UVPD) highlights differences in the ability to generate site-determining product ions, which can be exploited to differentiate sulfated peptides from nominally isobaric phosphopeptides based on precursor ion neutral loss at low collision energy. Application of our analytical workflow to a HEK-293 cell extracellular secretome facilitated identification of 21 new sulfotyrosine-containing proteins, several of which we validate enzymatically using in vitro sulfation assays. This study demonstrates the applicability of our strategy for confident, high-throughput, sulfomics studies, and reveals new sY interplay between enzymes relevant to both protein and glycan sulfation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.549022">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.549022" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.549022">
        <p class="paperTitle">Decrypting the languages of allostery in membrane-bound K-Ras4B using four complementary in silico approaches</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.549022" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.549022" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Castelli, M.; Marchetti, F.; Serapian, S. A.; Colombo, G.</p>
        <p class="info">Score: 6.3, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.549022' target='https://doi.org/10.1101/2023.07.14.549022'> 10.1101/2023.07.14.549022</a></p>
        <p class="abstract">Proteins are evolved molecular machines whose diverse biochemical functions are (also) dynamically regulated by allostery, through which even faraway protein residues can conformationally communicate. Allostery can express itself in different ways, akin to different &#34;languages&#34;: allosteric control pathways predominating in an unperturbed protein are superseded by others as soon as a perturbation arises--e.g, mutation--that alters its function (pathologically or not). Accurately modeling these often-unintuitive phenomena could therefore help explain functional changes in a specific protein whenever they are unclear. Unbiased molecular dynamics (MD) simulations are a possibility; however, since allostery can operate at longer timescales than those accessible by MD, simulations require integration with a reliable method able to, e.g., detect regions of incipient allosteric change, or likely perturbation pathways. Several such (valid) methods exist, but are typically applied singularly, only yielding a partial (&#34;monolinguistic&#34;) picture of allostery: we argue their joint application could significantly enrich this picture. To prove this, we perform unbiased MD simulations (~100 s) of the widely studied, allosterically active oncotarget K-Ras4B, solvated and embedded in a phospholipid membrane, proceeding to decrypt its allostery using four showcase &#34;languages&#34;: Distance Fluctuation analysis and the Shortest Path Map capture allosteric communication hotspots at equilibrium; Anisotropic Thermal Diffusion and Dynamic Non-Equilibrium MD assess them once the GTP that oncogenically activates K-Ras4B is, respectively, either superheated or hydrolyzed. &#34;Languages&#34; provide a uniquely articulate, mutually coherent, experimentally consistent picture of allostery in K-Ras4B. At equilibrium, pathways stretch from the membrane-embedded hypervariable region all the way to the active site, touching known flexible allosteric &#34;switches&#34; and proposed pockets. Upon GTP cleavage/perturbation, known to inactivate K-Ras4B, allosteric signals most reverberate on switches and interfaces that recruit effector proteins. Our work highlights the benefits of integrating different allostery detection methods, even in conjunction with unbiased MD.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548257">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548257" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548257">
        <p class="paperTitle">The pathogenic T42A mutation in SHP2 rewires interaction specificity and enhances signaling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548257" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548257" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: van Vlimmeren, A. E.; Voleti, R.; Chartier, C. A.; Jiang, Z.; Karandur, D.; Shah, N. H.</p>
        <p class="info">Score: 6.3, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548257' target='https://doi.org/10.1101/2023.07.10.548257'> 10.1101/2023.07.10.548257</a></p>
        <p class="abstract">Mutations in the tyrosine phosphatase SHP2 are associated with various human diseases. Most of these mutations increase the basal catalytic activity of SHP2 by disrupting auto-inhibitory interactions between its phosphatase domain and N-terminal SH2 (phosphotyrosine recognition) domain. By contrast, several other disease-associated mutations located in the ligand-binding pockets of the N- or C-terminal SH2 domains do not increase basal activity and likely exert their pathogenicity through alternative mechanisms. We lack a molecular understanding of how these SH2 mutations impact SHP2 structure, activity, and signaling. Here, we characterize five SHP2 SH2 domain ligand-binding pocket mutants through a combination of high-throughput biochemical screens, biophysical and biochemical measurements, molecular dynamics simulations, and cellular assays. We show that, while several of these mutants alter binding affinity to phosphorylation sites, the T42A mutation in the N-SH2 domain is unique in that it also alters ligand-binding specificity. The functional consequence of this altered specificity is that the T42A mutant has biased sensitivity toward a subset of activating ligands. Our study highlights an example of a nuanced mechanism of action for a disease-associated mutation, characterized by a change in protein-protein interaction specificity that alters enzyme activation.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
