<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="AlphaFold2 can predict single-mutation effects
Authors: Mcbride, J. M.; Reinharz, V.; Polev, K.; Grzybowski, B. A.; Tlusty, T.
Score: 123.9, Published: 2023-09-03 DOI: 10.1101/2022.04.14.488301
AlphaFold2 (AF) is a promising tool, but is it accurate enough to predict single mutation effects? Here, we report that the localized structural deformation between protein pairs differing by only 1-3 mutations - as measured by the effective strain - is correlated across 3,901 experimental and AF-predicted structures.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="AlphaFold2 can predict single-mutation effects
Authors: Mcbride, J. M.; Reinharz, V.; Polev, K.; Grzybowski, B. A.; Tlusty, T.
Score: 123.9, Published: 2023-09-03 DOI: 10.1101/2022.04.14.488301
AlphaFold2 (AF) is a promising tool, but is it accurate enough to predict single mutation effects? Here, we report that the localized structural deformation between protein pairs differing by only 1-3 mutations - as measured by the effective strain - is correlated across 3,901 experimental and AF-predicted structures." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-10T10:36:49+00:00" />
<meta property="article:modified_time" content="2023-09-10T10:36:49+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="AlphaFold2 can predict single-mutation effects
Authors: Mcbride, J. M.; Reinharz, V.; Polev, K.; Grzybowski, B. A.; Tlusty, T.
Score: 123.9, Published: 2023-09-03 DOI: 10.1101/2022.04.14.488301
AlphaFold2 (AF) is a promising tool, but is it accurate enough to predict single mutation effects? Here, we report that the localized structural deformation between protein pairs differing by only 1-3 mutations - as measured by the effective strain - is correlated across 3,901 experimental and AF-predicted structures."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "AlphaFold2 can predict single-mutation effects\nAuthors: Mcbride, J. M.; Reinharz, V.; Polev, K.; Grzybowski, B. A.; Tlusty, T.\nScore: 123.9, Published: 2023-09-03 DOI: 10.1101/2022.04.14.488301\nAlphaFold2 (AF) is a promising tool, but is it accurate enough to predict single mutation effects? Here, we report that the localized structural deformation between protein pairs differing by only 1-3 mutations - as measured by the effective strain - is correlated across 3,901 experimental and AF-predicted structures.",
  "keywords": [
    
  ],
  "articleBody": " AlphaFold2 can predict single-mutation effects\nAuthors: Mcbride, J. M.; Reinharz, V.; Polev, K.; Grzybowski, B. A.; Tlusty, T.\nScore: 123.9, Published: 2023-09-03 DOI: 10.1101/2022.04.14.488301\nAlphaFold2 (AF) is a promising tool, but is it accurate enough to predict single mutation effects? Here, we report that the localized structural deformation between protein pairs differing by only 1-3 mutations - as measured by the effective strain - is correlated across 3,901 experimental and AF-predicted structures. Furthermore, analysis of [~]11,000 proteins shows that the local structural change correlates with various phenotypic changes. These findings suggest that AF can predict the range and magnitude of single-mutation effects on average, and we propose a method to improve precision of AF predictions and to indicate when predictions are unreliable.\nRosetta Energy Analysis of AlphaFold2 models: Point Mutations and Conformational Ensembles\nAuthors: Stein, R. A.; Mchaourab, H. S.\nScore: 24.1, Published: 2023-09-05 DOI: 10.1101/2023.09.05.556364\nAlphaFold2s ability to accurately predict protein structures from a multiple sequence alignment (MSA) has raised many questions about the utility of the models generated in downstream structural analysis. Two outstanding questions are the prediction of the consequences of point mutations and the completeness of the landscape of protein conformational ensembles. We previously developed a method, SPEACH_AF, to obtain alternate conformations by introducing residue substitutions across the MSA and not just within the input sequence. Here, we compared the structural and energetic consequences of having the mutation(s) in the input sequence versus in the whole MSA (SPEACH_AF). Both methods yielded models different from the wild-type sequence, with more robust changes when the mutation(s) were in the whole MSA. To evaluate models of conformational diversity, we used SPEACH_AF and a new MSA subsampling method, AF_cluster, combined with model relaxation in Rosetta. We find that the energetics of the conformations generated by AlphaFold2 correspond to those seen in experimental crystal structures and explored by standard molecular dynamic methods. Combined, the results support the fact that AlphaFold2 can predict structural changes due to point mutations and has learned information about protein structural energetics that are encoded in the MSA.\nA ubiquitin-specific, proximity-based labeling approach for the identification of ubiquitin ligase substrates\nAuthors: Bhogaraju, S.; Behrends, C.; Mukhopadhyay, U.; Levantovsky, S.; Stein, F.; Gharbi, S.\nScore: 20.4, Published: 2023-09-04 DOI: 10.1101/2023.09.04.556194\nUbiquitination of proteins is central to protein homeostasis and other cellular processes including DNA repair, vesicular transport, cell-division etc. The process of ubiquitination is conserved from yeast to humans and is carried out by the sequential action of three enzymes: E1, E2 and E3. There are an estimated \u003e600 E3 ligases in humans that execute ubiquitination of specific target proteins in a spatio-temporal manner to elicit desired signaling effects. Here, we developed a simple, proximity-based labeling method to selectively biotinylate substrates of a given ubiquitin ligase. Our method exploits the proximity and the relative orientation of the E3-ligase catalytic domain with respect to ubiquitin observed in the enzymatic intermediate-state structures of E3-E2[~]Ub. By fusing the biotin ligase BirA and an Avi tag variant to the candidate E3 ligase and ubiquitin, respectively, we were able to specifically enrich the bona fide substrates and potential new substrates of the ligase using a one-step Streptavidin pulldown under denaturing conditions. As proof-of-principle, we applied our method, which we named Ub-POD, to the RING E3 ligase Rad18. Rad18 ubiquitinates DNA-sliding clamp PCNA upon UV-induced DNA damage. We identified PCNA and several other critical players in the DNA damage repair pathway in a single Ub-POD experiment. We further discovered an unintended but important application of Ub-POD, where we were able to pin down the cellular localization of Rad18-mediated ubiquitination to the damaged DNA nuclear puncta through Streptavidin immunofluorescence. We also applied our method to TRAF6, another RING ubiquitin ligase involved in TNF signaling, successfully identifying known and potentially new substrates. Finally, we adapted our method to the U-box-type E3 ubiquitin ligase CHIP to demonstrate that we can identify substrates of two major classes of mammalian ubiquitin ligases. We anticipate that our method and principle could be widely adapted to all classes of ubiquitin ligases to identify substrates and localize the cellular site(s) of ubiquitination.\nStructural Mechanisms for VMAT2 inhibition by tetrabenazine\nAuthors: Dalton, M. P.; Cheng, M. H.; Bahar, I.; Coleman, J. A.\nScore: 12.4, Published: 2023-09-05 DOI: 10.1101/2023.09.05.556211\nThe vesicular monoamine transporter 2 (VMAT2) is a proton-dependent antiporter responsible for loading monoamine neurotransmitters into synaptic vesicles. Dysregulation of VMAT2 can lead to several neuropsychiatric disorders including Parkinsons disease and schizophrenia. Furthermore, drugs such as amphetamine and MDMA are known to act on VMAT2, exemplifying its role in the mechanisms of actions for drugs of abuse. Despite VMAT2s importance, there remains a critical lack of mechanistic understanding, largely driven by a lack of structural information. Here we report a 3.3 [A] resolution cryo-EM structure of VMAT2 complexed with TBZ, a non-competitive inhibitor used in the treatment of Huntingtons chorea. We find TBZ interacts with residues in a central binding site, locking VMAT2 in an occluded conformation and providing a mechanistic basis for non-competitive inhibition. We further identify residues critical for intracellular and luminal gating, including a cluster of hydrophobic residues which are involved in a luminal gating strategy. Our structure also highlights three distinct polar networks that may determine VMAT2 conformational change and play a role in proton transduction. The structure elucidates mechanisms of VMAT2 inhibition and transport, providing insights into VMAT2 architecture, function, and the design of small-molecule therapeutics.\nThe structure of a C. neoformans polysaccharide motif recognized by protective antibodies: a combined NMR and MD study\nAuthors: Hargett, A. A.; Azurmendi, H. F.; Crawford, C. J.; Wear, M. P.; Oscarson, S.; Casadevall, A. A.; Freedberg, D. I.\nScore: 7.3, Published: 2023-09-06 DOI: 10.1101/2023.09.06.556507\nCryptococcus neoformans is a fungal pathogen responsible for cryptococcosis and cryptococcal meningitis. The C. neoformans capsular polysaccharide and shed exopolysaccharide functions both as a key virulence factor and to protect the fungal cell from phagocytosis. Currently, a glycoconjugate of these polysaccharides is being explored as a vaccine to protect against C. neoformans infection. In this combined NMR and MD study, experimentally determined NOEs and J-couplings support a structure of the synthetic decasaccharide, GXM10-Ac3, obtained by MD. GXM10-Ac3 was designed as an extension of glucuronoxylomannan (GXM) polysaccharide motif (M2) which is common in the clinically predominant serotype A strains and is recognized by protective forms of GXM-specific monoclonal antibodies. The M2 motif is characterized by a 6-residue -mannan backbone repeating unit, consisting of a triad of -(1[-\u0026gt;]3)-mannoses, modified by {beta}-(1[-\u0026gt;]2)-xyloses on the first two mannoses and a {beta}-(1[-\u0026gt;]2)-glucuronic acid on the third mannose. The combined NMR and MD analyses reveal that GXM10-Ac3 adopts an extended structure, with xylose/glucuronic acid branches alternating sides along the -mannan backbone. O-acetyl esters also alternate sides and are grouped in pairs. MD analysis of a twelve M2-repeating unit polymer supports the notion that the GXM10-Ac3 structure is uniformly represented throughout the polysaccharide. This experimentally consistent GXM model displays high flexibility while maintaining a structural identity, yielding new insights to further explore intermolecular interactions between polysaccharides, interactions with anti-GXM mAbs, and the cryptococcal polysaccharide architecture.\nDefining RNA oligonucleotides that reverse deleterious phase transitions of RNA-binding proteins with prion-like domains\nAuthors: Guo, L.; Mann, J. R.; Mauna, J. C.; Copley, K. E.; Wang, H.; Rubien, J. D.; Odeh, H. M.; Lin, J.; Lee, B. L.; Ganser, L.; Robinson, E.; Kim, K. M.; Murthy, A. C.; Paul, T.; Portz, B.; Gleixner, A. M.; Diaz, Z.; Carey, J. L.; Smirnov, A.; Padilla, G.; Lavorando, E.; Espy, C.; Shang, Y.; Huang, E. J.; Chesi, A.; Fawzi, N. L.; Myong, S.; Donnelly, C. J.; Shorter, J.\nScore: 6.9, Published: 2023-09-04 DOI: 10.1101/2023.09.04.555754\nRNA-binding proteins with prion-like domains, such as FUS and TDP-43, condense into functional liquids, which can transform into pathological fibrils that underpin fatal neurodegenerative disorders, including amyotrophic lateral sclerosis (ALS)/frontotemporal dementia (FTD). Here, we define short RNAs (24-48 nucleotides) that prevent FUS fibrillization by promoting liquid phases, and distinct short RNAs that prevent and, remarkably, reverse FUS condensation and fibrillization. These activities require interactions with multiple RNA-binding domains of FUS and are encoded by RNA sequence, length, and structure. Importantly, we define a short RNA that dissolves aberrant cytoplasmic FUS condensates, restores nuclear FUS, and mitigates FUS proteotoxicity in optogenetic models and human motor neurons. Another short RNA dissolves aberrant cytoplasmic TDP-43 condensates, restores nuclear TDP-43, and mitigates TDP-43 proteotoxicity. Since short RNAs can be effectively delivered to the human brain, these oligonucleotides could have therapeutic utility for ALS/FTD and related disorders.\nSLP2 coordinates MICOS assembly and cristae morphogenesis via MIC13 and YME1L\nAuthors: Naha, R.; Strohm, R.; Urbach, J.; Wittig, I.; Reichert, A. S.; Kondadi, A. K.; Anand, R.\nScore: 6.8, Published: 2023-09-04 DOI: 10.1101/2023.09.04.556207\nThe MICOS complex subunit MIC13 is essential for mitochondrial cristae organization. Mutations in MIC13 cause severe mitochondrial hepato-encephalopathy displaying defective cristae morphology and loss of the MIC10-subcomplex. Here we identified SLP2 as a novel interacting partner of MIC13 and decipher a critical role of SLP2 for MICOS assembly at distinct steps. SLP2 provides a large interaction hub for MICOS subunits and loss of SLP2 imparted YME1L-mediated proteolysis of MIC26 and drastic alterations in cristae morphology. We further identified a MIC13-specific role in stabilizing the MIC10-subcomplex via a MIC13-YME1L axis. SLP2 together with the stabilized MIC10-subcomplex promotes efficient assembly of the MIC60-subcomplex forming the MICOS-MIB complex. Consistently, super-resolution nanoscopy showed a dispersed distribution of the MIC60 in cells lacking SLP2 and MIC13. Our study reveals converging and interdependent assembly pathways for the MIC10- and MIC60-subcomplexes which are controlled in two ways, the MIC13-YME1L and the SLP2-YME1L axes, revealing mechanistic insights of these factors in cristae morphogenesis. These results will be helpful in understanding the human pathophysiology linked to mutations in MIC13 or its interaction partners.\nApolipoprotein-E transforms intracellular Amyloid-β oligomers to a more toxic state\nAuthors: Dey, A.; Verma, A.; Bhaskar, U.; Sarkar, B.; Kallianpur, M.; Visvakarma, V.; Das, A. K.; Garai, K.; Mukherjee, O.; Ishii, K.; Tahara, T.; Maiti, S.\nScore: 5.9, Published: 2023-09-03 DOI: 10.1101/2023.09.03.556061\nIt is poorly understood why ApoE variants are major genetic risk factors in Alzheimers disease (AD), which is associated with the aggregation of amyloid beta (A{beta}). Here we directly image specific changes in small A{beta} oligomers in rat brain cells that correlate with the cellular ApoE content. An inhibitor of A{beta}-ApoE interaction suppresses this change and concomitantly reduces A{beta} toxicity in a dose-dependent manner. Single-molecule techniques show changes both in the conformation and the stoichiometry of the oligomers. hiPSC-derived neural stem cells from Alzheimers patients also show similar changes. Interaction with ApoE therefore changes the oligomeric state, membrane affinity, and toxicity of A{beta} oligomers, and can be directly read out in live cells. Our findings suggest a rapid and quantitative assay for AD drug discovery. One-sentence summaryApoE causes specific toxicogenic modifications of A{beta} oligomers, and these changes can be directly imaged in live cells.\nComparing peptide identifications by FAIMS versus quadrupole gas-phase fractionation\nAuthors: Faivre, D. A.; McGann, C. D.; Merrihew, G. E.; Schweppe, D. K.; MacCoss, M. J.\nScore: 4.7, Published: 2023-09-03 DOI: 10.1101/2023.09.01.552989\nHigh-field asymmetric waveform ion mobility spectrometry (FAIMS) coupled to liquid chromatography-mass spectrometry (LC-MS) has been shown to increase peptide and protein detections compared to LC-MS/MS alone. However, FAIMS has not been compared to other methods of gas-phase fractionation, such as quadrupole gas-phase fractionation, which could increase our understanding of the mechanisms of improvement. The goal of this work was to assess whether FAIMS improves peptide identifications because 1) gas-phase fractionation enables the analysis of less abundant signals by excluding more abundant precursors from filling the ion trap, 2) the use of FAIMS reduces co-isolation of peptides during the MS/MS process resulting in a reduction of chimeric spectra, or 3) a combination of both. To investigate these hypotheses, pooled human brain tissue samples were measured in triplicate using FAIMS gas-phase fractionation, quadrupole gas-phase fractionation, or no gas-phase fractionation on two Thermo Eclipse Tribrid Mass Spectrometers. On both instruments, our data confirmed prior observations that FAIMS increased the number of peptides identified. We further demonstrated that the main benefit of FAIMS is due to the reduced co-isolation of persistent peptide precursor ions, which results in a decrease in chimeric spectra.\nFRET-based sensor for measuring adenine nucleotide binding to AMPK\nAuthors: Abi Nahed, R.; Pelosse, M.; Aulicino, F.; Cottaz, F.; Berger, I.; Schlattner, U.\nScore: 3.8, Published: 2023-09-06 DOI: 10.1101/2023.09.05.553069\nAMP-activated protein kinase (AMPK) has evolved to detect a critical increase in cellular AMP/ATP and ADP/ATP concentration ratios as a signal for limiting energy supply. Such energy stress then leads to AMPK activation and downstream events that maintain cellular energy homeostasis. AMPK activation by AMP, ADP or pharmacological activators involves a conformational switch within the AMPK heterotrimeric complex. We have engineered an AMPK-based sensor, AMPfret, which translates the activating conformational switch into a fluorescence signal, based on increased fluorescence resonance energy transfer (FRET) between donor and acceptor fluorophores. Here we describe how this sensor can be used to analyze direct AMPK activation by small molecules in vitro using a fluorimeter, or to estimate changes in the energy state of cells using standard fluorescence or confocal microscopy.\n",
  "wordCount" : "2171",
  "inLanguage": "en",
  "datePublished": "2023-09-10T10:36:49Z",
  "dateModified": "2023-09-10T10:36:49Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta"><span>updated on September 10, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.04.14.488301">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.04.14.488301" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.04.14.488301">
        <p class="paperTitle">AlphaFold2 can predict single-mutation effects</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.04.14.488301" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.04.14.488301" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mcbride, J. M.; Reinharz, V.; Polev, K.; Grzybowski, B. A.; Tlusty, T.</p>
        <p class="info">Score: 123.9, Published: 2023-09-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.04.14.488301' target='https://doi.org/10.1101/2022.04.14.488301'> 10.1101/2022.04.14.488301</a></p>
        <p class="abstract">AlphaFold2 (AF) is a promising tool, but is it accurate enough to predict single mutation effects? Here, we report that the localized structural deformation between protein pairs differing by only 1-3 mutations - as measured by the effective strain - is correlated across 3,901 experimental and AF-predicted structures. Furthermore, analysis of [~]11,000 proteins shows that the local structural change correlates with various phenotypic changes. These findings suggest that AF can predict the range and magnitude of single-mutation effects on average, and we propose a method to improve precision of AF predictions and to indicate when predictions are unreliable.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.05.556364">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.05.556364" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.05.556364">
        <p class="paperTitle">Rosetta Energy Analysis of AlphaFold2 models: Point Mutations and Conformational Ensembles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.05.556364" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.05.556364" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stein, R. A.; Mchaourab, H. S.</p>
        <p class="info">Score: 24.1, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.05.556364' target='https://doi.org/10.1101/2023.09.05.556364'> 10.1101/2023.09.05.556364</a></p>
        <p class="abstract">AlphaFold2s ability to accurately predict protein structures from a multiple sequence alignment (MSA) has raised many questions about the utility of the models generated in downstream structural analysis. Two outstanding questions are the prediction of the consequences of point mutations and the completeness of the landscape of protein conformational ensembles. We previously developed a method, SPEACH_AF, to obtain alternate conformations by introducing residue substitutions across the MSA and not just within the input sequence. Here, we compared the structural and energetic consequences of having the mutation(s) in the input sequence versus in the whole MSA (SPEACH_AF). Both methods yielded models different from the wild-type sequence, with more robust changes when the mutation(s) were in the whole MSA. To evaluate models of conformational diversity, we used SPEACH_AF and a new MSA subsampling method, AF_cluster, combined with model relaxation in Rosetta. We find that the energetics of the conformations generated by AlphaFold2 correspond to those seen in experimental crystal structures and explored by standard molecular dynamic methods. Combined, the results support the fact that AlphaFold2 can predict structural changes due to point mutations and has learned information about protein structural energetics that are encoded in the MSA.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.04.556194">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.04.556194" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.04.556194">
        <p class="paperTitle">A ubiquitin-specific, proximity-based labeling approach for the identification of ubiquitin ligase substrates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.04.556194" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.04.556194" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bhogaraju, S.; Behrends, C.; Mukhopadhyay, U.; Levantovsky, S.; Stein, F.; Gharbi, S.</p>
        <p class="info">Score: 20.4, Published: 2023-09-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.04.556194' target='https://doi.org/10.1101/2023.09.04.556194'> 10.1101/2023.09.04.556194</a></p>
        <p class="abstract">Ubiquitination of proteins is central to protein homeostasis and other cellular processes including DNA repair, vesicular transport, cell-division etc. The process of ubiquitination is conserved from yeast to humans and is carried out by the sequential action of three enzymes: E1, E2 and E3. There are an estimated &gt;600 E3 ligases in humans that execute ubiquitination of specific target proteins in a spatio-temporal manner to elicit desired signaling effects. Here, we developed a simple, proximity-based labeling method to selectively biotinylate substrates of a given ubiquitin ligase. Our method exploits the proximity and the relative orientation of the E3-ligase catalytic domain with respect to ubiquitin observed in the enzymatic intermediate-state structures of E3-E2[~]Ub. By fusing the biotin ligase BirA and an Avi tag variant to the candidate E3 ligase and ubiquitin, respectively, we were able to specifically enrich the bona fide substrates and potential new substrates of the ligase using a one-step Streptavidin pulldown under denaturing conditions. As proof-of-principle, we applied our method, which we named Ub-POD, to the RING E3 ligase Rad18. Rad18 ubiquitinates DNA-sliding clamp PCNA upon UV-induced DNA damage. We identified PCNA and several other critical players in the DNA damage repair pathway in a single Ub-POD experiment. We further discovered an unintended but important application of Ub-POD, where we were able to pin down the cellular localization of Rad18-mediated ubiquitination to the damaged DNA nuclear puncta through Streptavidin immunofluorescence. We also applied our method to TRAF6, another RING ubiquitin ligase involved in TNF signaling, successfully identifying known and potentially new substrates. Finally, we adapted our method to the U-box-type E3 ubiquitin ligase CHIP to demonstrate that we can identify substrates of two major classes of mammalian ubiquitin ligases. We anticipate that our method and principle could be widely adapted to all classes of ubiquitin ligases to identify substrates and localize the cellular site(s) of ubiquitination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.05.556211">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.05.556211" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.05.556211">
        <p class="paperTitle">Structural Mechanisms for VMAT2 inhibition by tetrabenazine</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.05.556211" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.05.556211" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dalton, M. P.; Cheng, M. H.; Bahar, I.; Coleman, J. A.</p>
        <p class="info">Score: 12.4, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.05.556211' target='https://doi.org/10.1101/2023.09.05.556211'> 10.1101/2023.09.05.556211</a></p>
        <p class="abstract">The vesicular monoamine transporter 2 (VMAT2) is a proton-dependent antiporter responsible for loading monoamine neurotransmitters into synaptic vesicles. Dysregulation of VMAT2 can lead to several neuropsychiatric disorders including Parkinsons disease and schizophrenia.

Furthermore, drugs such as amphetamine and MDMA are known to act on VMAT2, exemplifying its role in the mechanisms of actions for drugs of abuse. Despite VMAT2s importance, there remains a critical lack of mechanistic understanding, largely driven by a lack of structural information. Here we report a 3.3 [A] resolution cryo-EM structure of VMAT2 complexed with TBZ, a non-competitive inhibitor used in the treatment of Huntingtons chorea. We find TBZ interacts with residues in a central binding site, locking VMAT2 in an occluded conformation and providing a mechanistic basis for non-competitive inhibition. We further identify residues critical for intracellular and luminal gating, including a cluster of hydrophobic residues which are involved in a luminal gating strategy. Our structure also highlights three distinct polar networks that may determine VMAT2 conformational change and play a role in proton transduction. The structure elucidates mechanisms of VMAT2 inhibition and transport, providing insights into VMAT2 architecture, function, and the design of small-molecule therapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.556507">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.556507" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.556507">
        <p class="paperTitle">The structure of a C. neoformans polysaccharide motif recognized by protective antibodies: a combined NMR and MD study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.556507" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.556507" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hargett, A. A.; Azurmendi, H. F.; Crawford, C. J.; Wear, M. P.; Oscarson, S.; Casadevall, A. A.; Freedberg, D. I.</p>
        <p class="info">Score: 7.3, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.556507' target='https://doi.org/10.1101/2023.09.06.556507'> 10.1101/2023.09.06.556507</a></p>
        <p class="abstract">Cryptococcus neoformans is a fungal pathogen responsible for cryptococcosis and cryptococcal meningitis. The C. neoformans capsular polysaccharide and shed exopolysaccharide functions both as a key virulence factor and to protect the fungal cell from phagocytosis. Currently, a glycoconjugate of these polysaccharides is being explored as a vaccine to protect against C. neoformans infection. In this combined NMR and MD study, experimentally determined NOEs and J-couplings support a structure of the synthetic decasaccharide, GXM10-Ac3, obtained by MD. GXM10-Ac3 was designed as an extension of glucuronoxylomannan (GXM) polysaccharide motif (M2) which is common in the clinically predominant serotype A strains and is recognized by protective forms of GXM-specific monoclonal antibodies. The M2 motif is characterized by a 6-residue -mannan backbone repeating unit, consisting of a triad of -(1[-&amp;gt;]3)-mannoses, modified by {beta}-(1[-&amp;gt;]2)-xyloses on the first two mannoses and a {beta}-(1[-&amp;gt;]2)-glucuronic acid on the third mannose. The combined NMR and MD analyses reveal that GXM10-Ac3 adopts an extended structure, with xylose/glucuronic acid branches alternating sides along the -mannan backbone. O-acetyl esters also alternate sides and are grouped in pairs. MD analysis of a twelve M2-repeating unit polymer supports the notion that the GXM10-Ac3 structure is uniformly represented throughout the polysaccharide. This experimentally consistent GXM model displays high flexibility while maintaining a structural identity, yielding new insights to further explore intermolecular interactions between polysaccharides, interactions with anti-GXM mAbs, and the cryptococcal polysaccharide architecture.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.04.555754">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.04.555754" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.04.555754">
        <p class="paperTitle">Defining RNA oligonucleotides that reverse deleterious phase transitions of RNA-binding proteins with prion-like domains</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.04.555754" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.04.555754" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guo, L.; Mann, J. R.; Mauna, J. C.; Copley, K. E.; Wang, H.; Rubien, J. D.; Odeh, H. M.; Lin, J.; Lee, B. L.; Ganser, L.; Robinson, E.; Kim, K. M.; Murthy, A. C.; Paul, T.; Portz, B.; Gleixner, A. M.; Diaz, Z.; Carey, J. L.; Smirnov, A.; Padilla, G.; Lavorando, E.; Espy, C.; Shang, Y.; Huang, E. J.; Chesi, A.; Fawzi, N. L.; Myong, S.; Donnelly, C. J.; Shorter, J.</p>
        <p class="info">Score: 6.9, Published: 2023-09-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.04.555754' target='https://doi.org/10.1101/2023.09.04.555754'> 10.1101/2023.09.04.555754</a></p>
        <p class="abstract">RNA-binding proteins with prion-like domains, such as FUS and TDP-43, condense into functional liquids, which can transform into pathological fibrils that underpin fatal neurodegenerative disorders, including amyotrophic lateral sclerosis (ALS)/frontotemporal dementia (FTD). Here, we define short RNAs (24-48 nucleotides) that prevent FUS fibrillization by promoting liquid phases, and distinct short RNAs that prevent and, remarkably, reverse FUS condensation and fibrillization. These activities require interactions with multiple RNA-binding domains of FUS and are encoded by RNA sequence, length, and structure. Importantly, we define a short RNA that dissolves aberrant cytoplasmic FUS condensates, restores nuclear FUS, and mitigates FUS proteotoxicity in optogenetic models and human motor neurons. Another short RNA dissolves aberrant cytoplasmic TDP-43 condensates, restores nuclear TDP-43, and mitigates TDP-43 proteotoxicity. Since short RNAs can be effectively delivered to the human brain, these oligonucleotides could have therapeutic utility for ALS/FTD and related disorders.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.04.556207">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.04.556207" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.04.556207">
        <p class="paperTitle">SLP2 coordinates MICOS assembly and cristae morphogenesis via MIC13 and YME1L</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.04.556207" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.04.556207" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Naha, R.; Strohm, R.; Urbach, J.; Wittig, I.; Reichert, A. S.; Kondadi, A. K.; Anand, R.</p>
        <p class="info">Score: 6.8, Published: 2023-09-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.04.556207' target='https://doi.org/10.1101/2023.09.04.556207'> 10.1101/2023.09.04.556207</a></p>
        <p class="abstract">The MICOS complex subunit MIC13 is essential for mitochondrial cristae organization. Mutations in MIC13 cause severe mitochondrial hepato-encephalopathy displaying defective cristae morphology and loss of the MIC10-subcomplex. Here we identified SLP2 as a novel interacting partner of MIC13 and decipher a critical role of SLP2 for MICOS assembly at distinct steps. SLP2 provides a large interaction hub for MICOS subunits and loss of SLP2 imparted YME1L-mediated proteolysis of MIC26 and drastic alterations in cristae morphology. We further identified a MIC13-specific role in stabilizing the MIC10-subcomplex via a MIC13-YME1L axis. SLP2 together with the stabilized MIC10-subcomplex promotes efficient assembly of the MIC60-subcomplex forming the MICOS-MIB complex. Consistently, super-resolution nanoscopy showed a dispersed distribution of the MIC60 in cells lacking SLP2 and MIC13. Our study reveals converging and interdependent assembly pathways for the MIC10- and MIC60-subcomplexes which are controlled in two ways, the MIC13-YME1L and the SLP2-YME1L axes, revealing mechanistic insights of these factors in cristae morphogenesis. These results will be helpful in understanding the human pathophysiology linked to mutations in MIC13 or its interaction partners.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.03.556061">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.03.556061" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.03.556061">
        <p class="paperTitle">Apolipoprotein-E transforms intracellular Amyloid-β oligomers to a more toxic state</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.03.556061" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.03.556061" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dey, A.; Verma, A.; Bhaskar, U.; Sarkar, B.; Kallianpur, M.; Visvakarma, V.; Das, A. K.; Garai, K.; Mukherjee, O.; Ishii, K.; Tahara, T.; Maiti, S.</p>
        <p class="info">Score: 5.9, Published: 2023-09-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.03.556061' target='https://doi.org/10.1101/2023.09.03.556061'> 10.1101/2023.09.03.556061</a></p>
        <p class="abstract">It is poorly understood why ApoE variants are major genetic risk factors in Alzheimers disease (AD), which is associated with the aggregation of amyloid beta (A{beta}). Here we directly image specific changes in small A{beta} oligomers in rat brain cells that correlate with the cellular ApoE content. An inhibitor of A{beta}-ApoE interaction suppresses this change and concomitantly reduces A{beta} toxicity in a dose-dependent manner. Single-molecule techniques show changes both in the conformation and the stoichiometry of the oligomers. hiPSC-derived neural stem cells from Alzheimers patients also show similar changes. Interaction with ApoE therefore changes the oligomeric state, membrane affinity, and toxicity of A{beta} oligomers, and can be directly read out in live cells. Our findings suggest a rapid and quantitative assay for AD drug discovery.

One-sentence summaryApoE causes specific toxicogenic modifications of A{beta} oligomers, and these changes can be directly imaged in live cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.552989">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.552989" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.552989">
        <p class="paperTitle">Comparing peptide identifications by FAIMS versus quadrupole gas-phase fractionation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.552989" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.552989" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Faivre, D. A.; McGann, C. D.; Merrihew, G. E.; Schweppe, D. K.; MacCoss, M. J.</p>
        <p class="info">Score: 4.7, Published: 2023-09-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.552989' target='https://doi.org/10.1101/2023.09.01.552989'> 10.1101/2023.09.01.552989</a></p>
        <p class="abstract">High-field asymmetric waveform ion mobility spectrometry (FAIMS) coupled to liquid chromatography-mass spectrometry (LC-MS) has been shown to increase peptide and protein detections compared to LC-MS/MS alone. However, FAIMS has not been compared to other methods of gas-phase fractionation, such as quadrupole gas-phase fractionation, which could increase our understanding of the mechanisms of improvement. The goal of this work was to assess whether FAIMS improves peptide identifications because 1) gas-phase fractionation enables the analysis of less abundant signals by excluding more abundant precursors from filling the ion trap, 2) the use of FAIMS reduces co-isolation of peptides during the MS/MS process resulting in a reduction of chimeric spectra, or 3) a combination of both. To investigate these hypotheses, pooled human brain tissue samples were measured in triplicate using FAIMS gas-phase fractionation, quadrupole gas-phase fractionation, or no gas-phase fractionation on two Thermo Eclipse Tribrid Mass Spectrometers. On both instruments, our data confirmed prior observations that FAIMS increased the number of peptides identified. We further demonstrated that the main benefit of FAIMS is due to the reduced co-isolation of persistent peptide precursor ions, which results in a decrease in chimeric spectra.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.05.553069">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.05.553069" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.05.553069">
        <p class="paperTitle">FRET-based sensor for measuring adenine nucleotide binding to AMPK</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.05.553069" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.05.553069" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Abi Nahed, R.; Pelosse, M.; Aulicino, F.; Cottaz, F.; Berger, I.; Schlattner, U.</p>
        <p class="info">Score: 3.8, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.05.553069' target='https://doi.org/10.1101/2023.09.05.553069'> 10.1101/2023.09.05.553069</a></p>
        <p class="abstract">AMP-activated protein kinase (AMPK) has evolved to detect a critical increase in cellular AMP/ATP and ADP/ATP concentration ratios as a signal for limiting energy supply. Such energy stress then leads to AMPK activation and downstream events that maintain cellular energy homeostasis. AMPK activation by AMP, ADP or pharmacological activators involves a conformational switch within the AMPK heterotrimeric complex. We have engineered an AMPK-based sensor, AMPfret, which translates the activating conformational switch into a fluorescence signal, based on increased fluorescence resonance energy transfer (FRET) between donor and acceptor fluorophores. Here we describe how this sensor can be used to analyze direct AMPK activation by small molecules in vitro using a fluorimeter, or to estimate changes in the energy state of cells using standard fluorescence or confocal microscopy.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
