<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="CRISPR-Cas12a exhibits metal-dependent specificity switching
Authors: Nguyen, G. T.; Schelling, M. A.; Buscher, K. A.; Sritharan, A.; Sashital, D. G.
Score: 40.9, Published: 2023-11-29 DOI: 10.1101/2023.11.29.569287
Cas12a is the immune effector of type V-A CRISPR-Cas systems and has been co-opted for genome editing and other biotechnology tools. The specificity of Cas12a has been the subject of extensive investigation both in vitro and in genome editing experiments. However, in vitro studies have often been performed at high magnesium ion concentrations that are inconsistent with the free Mg2&#43; concentrations that would be present in cells.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="CRISPR-Cas12a exhibits metal-dependent specificity switching
Authors: Nguyen, G. T.; Schelling, M. A.; Buscher, K. A.; Sritharan, A.; Sashital, D. G.
Score: 40.9, Published: 2023-11-29 DOI: 10.1101/2023.11.29.569287
Cas12a is the immune effector of type V-A CRISPR-Cas systems and has been co-opted for genome editing and other biotechnology tools. The specificity of Cas12a has been the subject of extensive investigation both in vitro and in genome editing experiments. However, in vitro studies have often been performed at high magnesium ion concentrations that are inconsistent with the free Mg2&#43; concentrations that would be present in cells." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-06T10:40:04+00:00" />
<meta property="article:modified_time" content="2023-12-06T10:40:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="CRISPR-Cas12a exhibits metal-dependent specificity switching
Authors: Nguyen, G. T.; Schelling, M. A.; Buscher, K. A.; Sritharan, A.; Sashital, D. G.
Score: 40.9, Published: 2023-11-29 DOI: 10.1101/2023.11.29.569287
Cas12a is the immune effector of type V-A CRISPR-Cas systems and has been co-opted for genome editing and other biotechnology tools. The specificity of Cas12a has been the subject of extensive investigation both in vitro and in genome editing experiments. However, in vitro studies have often been performed at high magnesium ion concentrations that are inconsistent with the free Mg2&#43; concentrations that would be present in cells."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "CRISPR-Cas12a exhibits metal-dependent specificity switching\nAuthors: Nguyen, G. T.; Schelling, M. A.; Buscher, K. A.; Sritharan, A.; Sashital, D. G.\nScore: 40.9, Published: 2023-11-29 DOI: 10.1101/2023.11.29.569287\nCas12a is the immune effector of type V-A CRISPR-Cas systems and has been co-opted for genome editing and other biotechnology tools. The specificity of Cas12a has been the subject of extensive investigation both in vitro and in genome editing experiments. However, in vitro studies have often been performed at high magnesium ion concentrations that are inconsistent with the free Mg2\u0026#43; concentrations that would be present in cells.",
  "keywords": [
    
  ],
  "articleBody": " CRISPR-Cas12a exhibits metal-dependent specificity switching\nAuthors: Nguyen, G. T.; Schelling, M. A.; Buscher, K. A.; Sritharan, A.; Sashital, D. G.\nScore: 40.9, Published: 2023-11-29 DOI: 10.1101/2023.11.29.569287\nCas12a is the immune effector of type V-A CRISPR-Cas systems and has been co-opted for genome editing and other biotechnology tools. The specificity of Cas12a has been the subject of extensive investigation both in vitro and in genome editing experiments. However, in vitro studies have often been performed at high magnesium ion concentrations that are inconsistent with the free Mg2+ concentrations that would be present in cells. By profiling the specificity of Cas12a orthologs at a range of Mg2+ concentrations, we find that Cas12a switches its specificity depending on metal ion concentration. Lowering Mg2+ concentration decreases cleavage defects caused by seed mismatches, while increasing the defects caused by PAM-distal mismatches. We show that Cas12a can unwind and bind seed mutant targets more rapidly at low Mg2+ concentrations, resulting in faster cleavage. In contrast, PAM-distal mismatches cause substantial defects in cleavage following formation of the Cas12a-target complex at low Mg2+ concentrations. We observe differences in Cas12a specificity switching between three orthologs that results in variations in the routes of phage escape from Cas12a-mediated immunity. Overall, our results reveal the importance of physiological metal ion conditions on the specificity of Cas effectors that are used in different cellular environments.\nA targeted metabolomic method to detect epigenetically relevant metabolites\nAuthors: Miro-Blanch, J.; Junza, A.; Capellades, J.; Balvay, A.; Moudet, C.; Kovatcheva, M.; Raineri, S.; Rabot, S.; Mellor, J.; Serrano, M.; Yanes, O.\nScore: 28.6, Published: 2023-11-30 DOI: 10.1101/2023.11.30.569455\nMetabolites play a central role in the chemical crosstalk between metabolism and epigenetic marks. Epigenetically relevant metabolites are substrates, products and cofactors that can act as activators or inhibitors of epigenetic enzymes, which control gene expression by adding or removing chemical marks in the DNA, RNA and histones. Diet composition, and biosynthetic pathways encoded in the gut microbiome and the host genome are the main sources of these metabolites for mammals. Despite the increasing interest in the study of the microbiota-nutrient metabolism-host epigenetic axis to understand health and disease, there is a lack of a sensitive and easy analytical method to detect epigenetically relevant metabolites simultaneously. Here, we show an straightforward biphasic extraction where the organic phase is directly analyzed by GC-EI MS to detect short-chain fatty acids and formate without chemical derivatization, and the aqueous phase is analyzed by HILIC coupled to ESI-MS/MS, which together can cover \u003e30 epigenetically relevant metabolites in biological samples such as liver, plasma or feces. In addition, we propose a stable isotope tracing method based on multiple-reaction monitoring (MRM) transitions by LC-QqQ MS to understand how 13C-labeled glucose or glutamine are used to build SAM and acetyl-CoA, the main methyl and acetyl group donors in epigenetic modifications, respectively. We anticipate that our methods will complement epigenomic and proteomic analyses adding another layer of molecular information towards mechanistic insights. HighlightsO_LIHost and microbiota metabolites link metabolism with epigenetic regulation. C_LIO_LIChemical structure diversity in epigenetically relevant metabolites challenges its analysis with a single method. C_LIO_LIA biphasic extraction with no chemical derivatization is able to recover SCFAs and other epigenetically relevant metabolites. C_LIO_LIA novel isotope trace experiment approach allows isotopomer resolution using MS2 data. C_LI\nStructure of the γ-tubulin ring complex-capped microtubule\nAuthors: Aher, A.; Urnavicius, L.; Xue, A.; Neselu, K.; Kapoor, T.\nScore: 38.5, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567916\nMicrotubules are composed of /{beta}-tubulin dimers positioned head-to-tail to form protofilaments that associate laterally in varying numbers. It is not known how cellular microtubules assemble with the canonical 13-protofilament architecture, resulting in micrometer-scale /{beta}-tubulin tracks for intracellular transport that align with, rather than spiral along, the filaments long-axis. We report that the human [~]2.3MDa {gamma}-tubulin ring complex ({gamma}-TuRC), an essential regulator of microtubule formation that contains 14 {gamma}-tubulins, selectively nucleates 13-protofilament microtubules. Cryo-EM reconstructions of {gamma}-TuRC-capped microtubule minus-ends reveal the extensive intra- and inter-domain motions of {gamma}-TuRC subunits that accommodate its actin-containing luminal bridge and establish lateral and longitudinal interactions between {gamma}- and -tubulins. Our structures reveal how free {gamma}-TuRC, an inefficient nucleation template due to its splayed conformation, transforms into a stable cap that blocks addition or loss of /{beta}-tubulins from minus-ends and sets the lattice architecture of cellular microtubules. One Sentence SummaryStructural insights into how the {gamma}-tubulin ring complex nucleates and caps a 13-protofilament microtubule.\nStructural basis of the mechanism and inhibition of a human ceramide synthase\nAuthors: Pascoa, T. C.; Pike, A. C. W.; Tautermann, C. S.; Chi, G.; Traub, M.; Quigley, A.; Chalk, R.; Stefanic, S.; Thamm, S.; Pautsch, A.; Carpenter, E. P.; Schnapp, G.; Sauer, D. B.\nScore: 17.8, Published: 2023-12-04 DOI: 10.1101/2023.12.02.569723\nCeramides are bioactive sphingolipids that play pivotal roles in regulating cellular metabolism. Ceramides and dihydroceramides are synthesized by a family of six ceramide synthase enzymes (CerS), each with distinct specificity for the acyl-CoA substrate. Importantly, the acyl chain length plays a key role in determining the physiological function of ceramides, as well as their role in metabolic disease. Ceramide with an acyl chain length of 16 carbons (C16 ceramide) has been implicated in obesity, insulin resistance and liver disease, and the C16 ceramide-synthesizing CerS6 is regarded as an attractive drug target for obesity-associated disease. Despite their importance, the molecular mechanism underlying ceramide synthesis by CerS enzymes remains poorly understood. Here, we report cryo-electron microscopy structures of human CerS6, capturing covalent intermediate and product-bound states. These structures, together with biochemical characterization using intact protein and small molecule mass spectrometry, reveal that CerS catalysis proceeds via a ping-pong reaction mechanism involving a covalent acyl-enzyme intermediate. Notably, the product-bound structure was obtained upon reaction with the mycotoxin fumonisin B1, providing new insights into its inhibition of CerS. These results provide a framework for understanding the mechanisms of CerS function, selectivity, and inhibition, and open new directions for future drug discovery targeting the ceramide and sphingolipid pathways.\nCO2 Reduction by the Iron Nitrogenase Competes with N2 Fixation Under Physiological Conditions\nAuthors: Oehlmann, N. N.; Schmidt, F. V.; Herzog, M.; Goldman, A. L.; Rebelein, J. G.\nScore: 17.6, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569367\nNitrogenases are the only known enzymes that reduce molecular nitrogen (N2) to ammonia. Recent findings have demonstrated that nitrogenases also reduce the greenhouse gas carbon dioxide (CO2), suggesting CO2 to be a competitor of N2. Intriguingly, nitrogenase isoforms (i.e., molybdenum (Mo), vanadium and iron (Fe) nitrogenase) differ significantly in their ability to reduce CO2, but the mechanisms underlying these differences remain elusive. Here, we study the competing reduction of CO2 and N2 by the two nitrogenases of Rhodobacter capsulatus, the Mo and Fe nitrogenase. Analyzing their full CO2 reduction product spectrum in vitro, we find the Fe nitrogenase almost three-fold more efficient in CO2 reduction than the Mo isoform. Furthermore, the in vitro competition experiments reveal the Fe nitrogenase to be profoundly less selective for the reduction of N2 than the Mo nitrogenase. We observe the same effects in vivo, where adding CO2 drastically increases the doubling times of diazotrophically grown R. capsulatus strains that rely on the Fe nitrogenase. The Fe nitrogenase-dependent R. capsulatus strains reduce CO2 to methane under physiological conditions, highlighting the potential of the Fe nitrogenase for the biotechnological conversion of CO2 into value-added compounds. Furthermore, both products are secreted into the surrounding, potentially influencing the composition of microbial communities in Mo-deficient environments.\nBioorthogonal metabolic labeling of the virulence factor phenolic glycolipid in mycobacteria\nAuthors: Guzman, L. E.; Cambier, C.; Cheng, T.-Y.; Naqvi, K. F.; Shiloh, M. U.; Moody, D. B.; Bertozzi, C.\nScore: 13.1, Published: 2023-11-29 DOI: 10.1101/2023.11.28.569059\nSurface lipids on pathogenic mycobacteria modulate infection outcomes by regulating host immune responses. Phenolic glycolipid (PGL) is a host-modulating surface lipid that varies among clinical Mycobacterium tuberculosis strains. PGL is also found in Mycobacterium marinum where it promotes infection of zebrafish through effects on the innate immune system. Given the important role this lipid plays in the host-pathogen relationship, tools for profiling its abundance, spatial distribution, and dynamics are needed. Here we report a strategy for imaging PGL in live mycobacteria using bioorthogonal metabolic labeling. We functionalized the PGL precursor p-hydroxybenzoic acid (pHB) with an azide group (3-azido pHB). When fed to mycobacteria, 3-azido pHB was incorporated into the cell surface, which could then be visualized via bioorthogonal conjugation of a fluorescent probe. We confirmed that 3-azido pHB incorporates into PGL using mass spectrometry methods and demonstrated selectivity for PGL-producing Mycobacterium marinum and Mycobacterium tuberculosis strains. Finally, we applied this metabolic labeling strategy to study the dynamics of PGL within the mycobacterial membrane. This new tool enables visualization of PGL which may facilitate studies of mycobacterial pathogenesis.\nStructural basis of anticancer drug recognition and amino acid transport by LAT1\nAuthors: Lee, Y.; Jin, C.; Ohgaki, R.; Xu, M.; Ogasawara, S.; Warshamanage, R.; Yamashita, K.; Murshudov, G.; Nureki, O.; Murata, T.; Kanai, Y.\nScore: 8.8, Published: 2023-12-03 DOI: 10.1101/2023.12.03.567112\nLAT1 (SLC7A5) transports large neutral amino acids and their derivatives across the plasma membrane and plays pivotal roles in cancer cell proliferation, immune response and drug delivery across the blood-brain barrier. Despite recent advances in structural understanding of LAT1, how it discriminates substrates and inhibitors including the clinically relevant anticancer drugs remains elusive. Here we report six structures of LAT1, captured in three different conformations and bound with diverse bioactive ligands, elucidating its substrate transport and inhibitory mechanisms. JPH203, also known as nanvuranlat or KYT-0353 and currently in clinical trials as an anticancer drug, binds to the wide-open substrate-binding pocket of LAT1. It adopts a U-shaped conformer, with its amino-phenylbenzoxazol moiety pushing against transmembrane helix 3 (TM3), bending TM10 and arresting the transporter in the outward-facing conformation. In contrast, the physiological substrate L-Phe does not exhibit such inhibitory interactions, whereas melphalan, a slow substrate, poses steric hindrance in the pocket, explaining its inhibitory activity. Unexpectedly, the \"classical\" system L inhibitor BCH induces an occluded state, a key structural intermediate required for substrate transport. Trans stimulation assays show that BCH facilitates transporter turnover and is therefore a transportable substrate. These findings provide a structural framework for the intricate mechanisms of substrate recognition and inhibition of LAT1, paving the way for developing more specific and effective drugs against it.\nStructural basis for inhibition of βFXIIa by garadacimab\nAuthors: Drulyte, I.; Ghai, R.; Ow, S. Y.; Kapp, E.; Quek, A.; Panousis, C.; Wilson, M. J.; Nash, A. D.; Pelzing, M.\nScore: 8.0, Published: 2023-12-04 DOI: 10.1101/2023.12.03.569829\nFXIIa is the principal initiator of the plasma contact system and can activate both procoagulant and proinflammatory pathways. Its activity is important in the pathophysiology of Hereditary Angioedema (HAE). Here, we describe a high-resolution cryo-EM structure of the beta-chain from FXIIa ({beta}FXIIa) complexed with the Fab fragment of garadacimab. Garadacimab binds to {beta}FXIIa through an unusually long CDR-H3 that inserts into the S1 pocket in a non-canonical way. This atypical structural mechanism is likely the primary contributor in the inhibition of activated FXII proteolytic activity in HAE. Garadacimab Fab-{beta}FXIIa structure also reveals critical determinants of high-affinity binding of garadacimab to activated FXII. Structural analysis with other bonafide FXIIa inhibitors, such as benzamidine and C1-INH, reveals a conserved mechanism of {beta}FXIIa inhibition, a novel finding of this study. In summary, garadacimab Fab-{beta}FXIIa structure provides crucial insights into its mechanism of action and delineates primary and auxiliary paratopes/epitopes. This work reaffirms the benefits of cryo-EM as a primary tool to study antigen-antibody complexes in near native state, particularly where other methods have fallen short.\nCyclase-associated protein is a pro-formin anti-capping processive depolymerase of actin barbed and pointed ends\nAuthors: Towsif, E. M.; Shekhar, S.\nScore: 5.7, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569482\nCellular actin networks display distinct assembly and disassembly dynamics resulting from multicomponent reactions occurring primarily at the two ends and the sides of actin filaments [1-3]. While barbed ends are considered the hotspot of actin assembly [4], disassembly is thought to primarily occur via reactions on filament sides and pointed ends [3, 5-11]. Cyclase-associated protein (CAP) has emerged as the main protagonist of actin disassembly and remodeling - it collaborates with cofilin to increase pointed-end depolymerization by 300-fold [6, 7], promotes filament \"coalescence\" in presence of Abp1 [12], and accelerates nucleotide exchange to regenerate monomers for new rounds of assembly [13-15]. CAP has also been reported to enhance cofilin-mediated severing [16, 17], but these claims have since been challenged [7]. Using microfluidics-assisted three-color single-molecule imaging, we now reveal that CAP also has important functions at filament barbed ends. We reveal that CAP is a processive barbed-end depolymerase capable of tracking both ends of the filament. Each CAP binding event leads to removal of about 5,175 and 620 subunits from the barbed and pointed ends respectively. We find that the WH2 domain is essential, and the CARP domain is dispensable for barbed-end depolymerization. We show that CAP co-localizes with barbed-end bound formin and capping protein, in the process increasing residence time of formin by 10-fold and promoting dissociation of CP by 4-fold. Our barbed-end observations combined with previously reported activities of CAP at pointed ends and sides, firmly establish CAP as a key player in actin dynamics.\nDifferences in clamp loader mechanism between bacteria and eukaryotes\nAuthors: Landeck, J. T.; Pajak, J.; Norman, E. K.; Sedivy, E. L.; Kelch, B. A.\nScore: 5.6, Published: 2023-11-30 DOI: 10.1101/2023.11.30.569468\nClamp loaders are pentameric ATPases that place circular sliding clamps onto DNA, where they function in DNA replication and genome integrity. The central activity of a clamp loader is the opening of the ring-shaped sliding clamp, and the subsequent binding to primer-template (p/t)-junctions. The general architecture of clamp loaders is conserved across all life, suggesting that their mechanism is retained. Recent structural studies of the eukaryotic clamp loader Replication Factor C (RFC) revealed that it functions using a crab-claw mechanism, where clamp opening is coupled to a massive conformational change in the loader. Here we investigate the clamp loading mechanism of the E. coli clamp loader at high resolution using cryo-electron microscopy (cryo-EM). We find that the E. coli clamp loader opens the clamp using a crab-claw motion at a single pivot point, whereas the eukaryotic RFC loader uses motions distributed across the complex. Furthermore, we find clamp opening occurs in multiple steps, starting with a partly open state with a spiral conformation, and proceeding to a wide open clamp in a surprising planar geometry. Finally, our structures in the presence of p/t-junctions illustrate how clamp closes around p/t-junctions and how the clamp loader initiates release from the loaded clamp. Our results reveal mechanistic distinctions in a macromolecular machine that is conserved across all domains of life.\n",
  "wordCount" : "2412",
  "inLanguage": "en",
  "datePublished": "2023-12-06T10:40:04Z",
  "dateModified": "2023-12-06T10:40:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 6, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569287">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569287" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569287">
        <p class="paperTitle">CRISPR-Cas12a exhibits metal-dependent specificity switching</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569287" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569287" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nguyen, G. T.; Schelling, M. A.; Buscher, K. A.; Sritharan, A.; Sashital, D. G.</p>
        <p class="info">Score: 40.9, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569287' target='https://doi.org/10.1101/2023.11.29.569287'> 10.1101/2023.11.29.569287</a></p>
        <p class="abstract">Cas12a is the immune effector of type V-A CRISPR-Cas systems and has been co-opted for genome editing and other biotechnology tools. The specificity of Cas12a has been the subject of extensive investigation both in vitro and in genome editing experiments. However, in vitro studies have often been performed at high magnesium ion concentrations that are inconsistent with the free Mg2&#43; concentrations that would be present in cells. By profiling the specificity of Cas12a orthologs at a range of Mg2&#43; concentrations, we find that Cas12a switches its specificity depending on metal ion concentration. Lowering Mg2&#43; concentration decreases cleavage defects caused by seed mismatches, while increasing the defects caused by PAM-distal mismatches. We show that Cas12a can unwind and bind seed mutant targets more rapidly at low Mg2&#43; concentrations, resulting in faster cleavage. In contrast, PAM-distal mismatches cause substantial defects in cleavage following formation of the Cas12a-target complex at low Mg2&#43; concentrations. We observe differences in Cas12a specificity switching between three orthologs that results in variations in the routes of phage escape from Cas12a-mediated immunity. Overall, our results reveal the importance of physiological metal ion conditions on the specificity of Cas effectors that are used in different cellular environments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569455">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569455" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569455">
        <p class="paperTitle">A targeted metabolomic method to detect epigenetically relevant metabolites</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569455" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569455" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Miro-Blanch, J.; Junza, A.; Capellades, J.; Balvay, A.; Moudet, C.; Kovatcheva, M.; Raineri, S.; Rabot, S.; Mellor, J.; Serrano, M.; Yanes, O.</p>
        <p class="info">Score: 28.6, Published: 2023-11-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569455' target='https://doi.org/10.1101/2023.11.30.569455'> 10.1101/2023.11.30.569455</a></p>
        <p class="abstract">Metabolites play a central role in the chemical crosstalk between metabolism and epigenetic marks. Epigenetically relevant metabolites are substrates, products and cofactors that can act as activators or inhibitors of epigenetic enzymes, which control gene expression by adding or removing chemical marks in the DNA, RNA and histones. Diet composition, and biosynthetic pathways encoded in the gut microbiome and the host genome are the main sources of these metabolites for mammals. Despite the increasing interest in the study of the  microbiota-nutrient metabolism-host epigenetic axis to understand health and disease, there is a lack of a sensitive and easy analytical method to detect epigenetically relevant metabolites simultaneously. Here, we show an straightforward biphasic extraction where the organic phase is directly analyzed by GC-EI MS to detect short-chain fatty acids and formate without chemical derivatization, and the aqueous phase is analyzed by HILIC coupled to ESI-MS/MS, which together can cover &gt;30 epigenetically relevant metabolites in biological samples such as liver, plasma or feces. In addition, we propose a stable isotope tracing method based on multiple-reaction monitoring (MRM) transitions by LC-QqQ MS to understand how 13C-labeled glucose or glutamine are used to build SAM and acetyl-CoA, the main methyl and acetyl group donors in epigenetic modifications, respectively. We anticipate that our methods will complement epigenomic and proteomic analyses adding another layer of molecular information towards mechanistic insights.

HighlightsO_LIHost and microbiota metabolites link metabolism with epigenetic regulation.
C_LIO_LIChemical structure diversity in epigenetically relevant metabolites challenges its analysis with a single method.
C_LIO_LIA biphasic extraction with no chemical derivatization is able to recover SCFAs and other epigenetically relevant metabolites.
C_LIO_LIA novel isotope trace experiment approach allows isotopomer resolution using MS2 data.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567916">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567916" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567916">
        <p class="paperTitle">Structure of the γ-tubulin ring complex-capped microtubule</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567916" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567916" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aher, A.; Urnavicius, L.; Xue, A.; Neselu, K.; Kapoor, T.</p>
        <p class="info">Score: 38.5, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567916' target='https://doi.org/10.1101/2023.11.20.567916'> 10.1101/2023.11.20.567916</a></p>
        <p class="abstract">Microtubules are composed of /{beta}-tubulin dimers positioned head-to-tail to form protofilaments that associate laterally in varying numbers. It is not known how cellular microtubules assemble with the canonical 13-protofilament architecture, resulting in micrometer-scale /{beta}-tubulin tracks for intracellular transport that align with, rather than spiral along, the filaments long-axis. We report that the human [~]2.3MDa {gamma}-tubulin ring complex ({gamma}-TuRC), an essential regulator of microtubule formation that contains 14 {gamma}-tubulins, selectively nucleates 13-protofilament microtubules. Cryo-EM reconstructions of {gamma}-TuRC-capped microtubule minus-ends reveal the extensive intra- and inter-domain motions of {gamma}-TuRC subunits that accommodate its actin-containing luminal bridge and establish lateral and longitudinal interactions between {gamma}- and -tubulins. Our structures reveal how free {gamma}-TuRC, an inefficient nucleation template due to its splayed conformation, transforms into a stable cap that blocks addition or loss of /{beta}-tubulins from minus-ends and sets the lattice architecture of cellular microtubules.

One Sentence SummaryStructural insights into how the {gamma}-tubulin ring complex nucleates and caps a 13-protofilament microtubule.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.02.569723">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.02.569723" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.02.569723">
        <p class="paperTitle">Structural basis of the mechanism and inhibition of a human ceramide synthase</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.02.569723" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.02.569723" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pascoa, T. C.; Pike, A. C. W.; Tautermann, C. S.; Chi, G.; Traub, M.; Quigley, A.; Chalk, R.; Stefanic, S.; Thamm, S.; Pautsch, A.; Carpenter, E. P.; Schnapp, G.; Sauer, D. B.</p>
        <p class="info">Score: 17.8, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.02.569723' target='https://doi.org/10.1101/2023.12.02.569723'> 10.1101/2023.12.02.569723</a></p>
        <p class="abstract">Ceramides are bioactive sphingolipids that play pivotal roles in regulating cellular metabolism. Ceramides and dihydroceramides are synthesized by a family of six ceramide synthase enzymes (CerS), each with distinct specificity for the acyl-CoA substrate. Importantly, the acyl chain length plays a key role in determining the physiological function of ceramides, as well as their role in metabolic disease. Ceramide with an acyl chain length of 16 carbons (C16 ceramide) has been implicated in obesity, insulin resistance and liver disease, and the C16 ceramide-synthesizing CerS6 is regarded as an attractive drug target for obesity-associated disease. Despite their importance, the molecular mechanism underlying ceramide synthesis by CerS enzymes remains poorly understood. Here, we report cryo-electron microscopy structures of human CerS6, capturing covalent intermediate and product-bound states. These structures, together with biochemical characterization using intact protein and small molecule mass spectrometry, reveal that CerS catalysis proceeds via a ping-pong reaction mechanism involving a covalent acyl-enzyme intermediate. Notably, the product-bound structure was obtained upon reaction with the mycotoxin fumonisin B1, providing new insights into its inhibition of CerS. These results provide a framework for understanding the mechanisms of CerS function, selectivity, and inhibition, and open new directions for future drug discovery targeting the ceramide and sphingolipid pathways.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569367">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569367" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569367">
        <p class="paperTitle">CO2 Reduction by the Iron Nitrogenase Competes with N2 Fixation Under Physiological Conditions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569367" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569367" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Oehlmann, N. N.; Schmidt, F. V.; Herzog, M.; Goldman, A. L.; Rebelein, J. G.</p>
        <p class="info">Score: 17.6, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569367' target='https://doi.org/10.1101/2023.11.30.569367'> 10.1101/2023.11.30.569367</a></p>
        <p class="abstract">Nitrogenases are the only known enzymes that reduce molecular nitrogen (N2) to ammonia. Recent findings have demonstrated that nitrogenases also reduce the greenhouse gas carbon dioxide (CO2), suggesting CO2 to be a competitor of N2. Intriguingly, nitrogenase isoforms (i.e., molybdenum (Mo), vanadium and iron (Fe) nitrogenase) differ significantly in their ability to reduce CO2, but the mechanisms underlying these differences remain elusive. Here, we study the competing reduction of CO2 and N2 by the two nitrogenases of Rhodobacter capsulatus, the Mo and Fe nitrogenase. Analyzing their full CO2 reduction product spectrum in vitro, we find the Fe nitrogenase almost three-fold more efficient in CO2 reduction than the Mo isoform. Furthermore, the in vitro competition experiments reveal the Fe nitrogenase to be profoundly less selective for the reduction of N2 than the Mo nitrogenase. We observe the same effects in vivo, where adding CO2 drastically increases the doubling times of diazotrophically grown R. capsulatus strains that rely on the Fe nitrogenase. The Fe nitrogenase-dependent R. capsulatus strains reduce CO2 to methane under physiological conditions, highlighting the potential of the Fe nitrogenase for the biotechnological conversion of CO2 into value-added compounds. Furthermore, both products are secreted into the surrounding, potentially influencing the composition of microbial communities in Mo-deficient environments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.569059">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.569059" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.569059">
        <p class="paperTitle">Bioorthogonal metabolic labeling of the virulence factor phenolic glycolipid in mycobacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.569059" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.569059" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guzman, L. E.; Cambier, C.; Cheng, T.-Y.; Naqvi, K. F.; Shiloh, M. U.; Moody, D. B.; Bertozzi, C.</p>
        <p class="info">Score: 13.1, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.569059' target='https://doi.org/10.1101/2023.11.28.569059'> 10.1101/2023.11.28.569059</a></p>
        <p class="abstract">Surface lipids on pathogenic mycobacteria modulate infection outcomes by regulating host immune responses. Phenolic glycolipid (PGL) is a host-modulating surface lipid that varies among clinical Mycobacterium tuberculosis strains. PGL is also found in Mycobacterium marinum where it promotes infection of zebrafish through effects on the innate immune system. Given the important role this lipid plays in the host-pathogen relationship, tools for profiling its abundance, spatial distribution, and dynamics are needed. Here we report a strategy for imaging PGL in live mycobacteria using bioorthogonal metabolic labeling. We functionalized the PGL precursor p-hydroxybenzoic acid (pHB) with an azide group (3-azido pHB). When fed to mycobacteria, 3-azido pHB was incorporated into the cell surface, which could then be visualized via bioorthogonal conjugation of a fluorescent probe. We confirmed that 3-azido pHB incorporates into PGL using mass spectrometry methods and demonstrated selectivity for PGL-producing Mycobacterium marinum and Mycobacterium tuberculosis strains. Finally, we applied this metabolic labeling strategy to study the dynamics of PGL within the mycobacterial membrane. This new tool enables visualization of PGL which may facilitate studies of mycobacterial pathogenesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.03.567112">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.03.567112" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.03.567112">
        <p class="paperTitle">Structural basis of anticancer drug recognition and amino acid transport by LAT1</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.03.567112" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.03.567112" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, Y.; Jin, C.; Ohgaki, R.; Xu, M.; Ogasawara, S.; Warshamanage, R.; Yamashita, K.; Murshudov, G.; Nureki, O.; Murata, T.; Kanai, Y.</p>
        <p class="info">Score: 8.8, Published: 2023-12-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.03.567112' target='https://doi.org/10.1101/2023.12.03.567112'> 10.1101/2023.12.03.567112</a></p>
        <p class="abstract">LAT1 (SLC7A5) transports large neutral amino acids and their derivatives across the plasma membrane and plays pivotal roles in cancer cell proliferation, immune response and drug delivery across the blood-brain barrier. Despite recent advances in structural understanding of LAT1, how it discriminates substrates and inhibitors including the clinically relevant anticancer drugs remains elusive. Here we report six structures of LAT1, captured in three different conformations and bound with diverse bioactive ligands, elucidating its substrate transport and inhibitory mechanisms. JPH203, also known as nanvuranlat or KYT-0353 and currently in clinical trials as an anticancer drug, binds to the wide-open substrate-binding pocket of LAT1. It adopts a U-shaped conformer, with its amino-phenylbenzoxazol moiety pushing against transmembrane helix 3 (TM3), bending TM10 and arresting the transporter in the outward-facing conformation. In contrast, the physiological substrate L-Phe does not exhibit such inhibitory interactions, whereas melphalan, a slow substrate, poses steric hindrance in the pocket, explaining its inhibitory activity. Unexpectedly, the &#34;classical&#34; system L inhibitor BCH induces an occluded state, a key structural intermediate required for substrate transport. Trans stimulation assays show that BCH facilitates transporter turnover and is therefore a transportable substrate. These findings provide a structural framework for the intricate mechanisms of substrate recognition and inhibition of LAT1, paving the way for developing more specific and effective drugs against it.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.03.569829">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.03.569829" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.03.569829">
        <p class="paperTitle">Structural basis for inhibition of βFXIIa by garadacimab</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.03.569829" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.03.569829" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Drulyte, I.; Ghai, R.; Ow, S. Y.; Kapp, E.; Quek, A.; Panousis, C.; Wilson, M. J.; Nash, A. D.; Pelzing, M.</p>
        <p class="info">Score: 8.0, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.03.569829' target='https://doi.org/10.1101/2023.12.03.569829'> 10.1101/2023.12.03.569829</a></p>
        <p class="abstract">FXIIa is the principal initiator of the plasma contact system and can activate both procoagulant and proinflammatory pathways. Its activity is important in the pathophysiology of Hereditary Angioedema (HAE). Here, we describe a high-resolution cryo-EM structure of the beta-chain from FXIIa ({beta}FXIIa) complexed with the Fab fragment of garadacimab. Garadacimab binds to {beta}FXIIa through an unusually long CDR-H3 that inserts into the S1 pocket in a non-canonical way. This atypical structural mechanism is likely the primary contributor in the inhibition of activated FXII proteolytic activity in HAE. Garadacimab Fab-{beta}FXIIa structure also reveals critical determinants of high-affinity binding of garadacimab to activated FXII. Structural analysis with other bonafide FXIIa inhibitors, such as benzamidine and C1-INH, reveals a conserved mechanism of {beta}FXIIa inhibition, a novel finding of this study. In summary, garadacimab Fab-{beta}FXIIa structure provides crucial insights into its mechanism of action and delineates primary and auxiliary paratopes/epitopes. This work reaffirms the benefits of cryo-EM as a primary tool to study antigen-antibody complexes in near native state, particularly where other methods have fallen short.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569482">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569482" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569482">
        <p class="paperTitle">Cyclase-associated protein is a pro-formin anti-capping processive depolymerase of actin barbed and pointed ends</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569482" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569482" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Towsif, E. M.; Shekhar, S.</p>
        <p class="info">Score: 5.7, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569482' target='https://doi.org/10.1101/2023.11.30.569482'> 10.1101/2023.11.30.569482</a></p>
        <p class="abstract">Cellular actin networks display distinct assembly and disassembly dynamics resulting from multicomponent reactions occurring primarily at the two ends and the sides of actin filaments [1-3]. While barbed ends are considered the hotspot of actin assembly [4], disassembly is thought to primarily occur via reactions on filament sides and pointed ends [3, 5-11]. Cyclase-associated protein (CAP) has emerged as the main protagonist of actin disassembly and remodeling - it collaborates with cofilin to increase pointed-end depolymerization by 300-fold [6, 7], promotes filament &#34;coalescence&#34; in presence of Abp1 [12], and accelerates nucleotide exchange to regenerate monomers for new rounds of assembly [13-15]. CAP has also been reported to enhance cofilin-mediated severing [16, 17], but these claims have since been challenged [7]. Using microfluidics-assisted three-color single-molecule imaging, we now reveal that CAP also has important functions at filament barbed ends. We reveal that CAP is a processive barbed-end depolymerase capable of tracking both ends of the filament. Each CAP binding event leads to removal of about 5,175 and 620 subunits from the barbed and pointed ends respectively. We find that the WH2 domain is essential, and the CARP domain is dispensable for barbed-end depolymerization. We show that CAP co-localizes with barbed-end bound formin and capping protein, in the process increasing residence time of formin by 10-fold and promoting dissociation of CP by 4-fold. Our barbed-end observations combined with previously reported activities of CAP at pointed ends and sides, firmly establish CAP as a key player in actin dynamics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569468">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569468" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569468">
        <p class="paperTitle">Differences in clamp loader mechanism between bacteria and eukaryotes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569468" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569468" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Landeck, J. T.; Pajak, J.; Norman, E. K.; Sedivy, E. L.; Kelch, B. A.</p>
        <p class="info">Score: 5.6, Published: 2023-11-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569468' target='https://doi.org/10.1101/2023.11.30.569468'> 10.1101/2023.11.30.569468</a></p>
        <p class="abstract">Clamp loaders are pentameric ATPases that place circular sliding clamps onto DNA, where they function in DNA replication and genome integrity. The central activity of a clamp loader is the opening of the ring-shaped sliding clamp, and the subsequent binding to primer-template (p/t)-junctions. The general architecture of clamp loaders is conserved across all life, suggesting that their mechanism is retained. Recent structural studies of the eukaryotic clamp loader Replication Factor C (RFC) revealed that it functions using a crab-claw mechanism, where clamp opening is coupled to a massive conformational change in the loader. Here we investigate the clamp loading mechanism of the E. coli clamp loader at high resolution using cryo-electron microscopy (cryo-EM). We find that the E. coli clamp loader opens the clamp using a crab-claw motion at a single pivot point, whereas the eukaryotic RFC loader uses motions distributed across the complex. Furthermore, we find clamp opening occurs in multiple steps, starting with a partly open state with a spiral conformation, and proceeding to a wide open clamp in a surprising planar geometry. Finally, our structures in the presence of p/t-junctions illustrate how clamp closes around p/t-junctions and how the clamp loader initiates release from the loaded clamp. Our results reveal mechanistic distinctions in a macromolecular machine that is conserved across all domains of life.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
