<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Bifunctional small molecules that induce nuclearlocalization and targeted transcriptional regulation
Authors: Gibson, W. J.; Sadagopan, A.; Shoba, V. M.; Choudhary, A.; Meyerson, M. L.; Schreiber, S. L.
Score: 33.8, Published: 2023-07-07 DOI: 10.1101/2023.07.07.548101
The aberrant localization of proteins in cells is a key factor in the development of various diseases, including cancer and neurodegenerative disease. To better understand and potentially manipulate protein localization for therapeutic purposes, we engineered bifunctional compounds that bind to proteins in separate cellular compartments.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Bifunctional small molecules that induce nuclearlocalization and targeted transcriptional regulation
Authors: Gibson, W. J.; Sadagopan, A.; Shoba, V. M.; Choudhary, A.; Meyerson, M. L.; Schreiber, S. L.
Score: 33.8, Published: 2023-07-07 DOI: 10.1101/2023.07.07.548101
The aberrant localization of proteins in cells is a key factor in the development of various diseases, including cancer and neurodegenerative disease. To better understand and potentially manipulate protein localization for therapeutic purposes, we engineered bifunctional compounds that bind to proteins in separate cellular compartments." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-09T10:40:17+00:00" />
<meta property="article:modified_time" content="2023-07-09T10:40:17+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Bifunctional small molecules that induce nuclearlocalization and targeted transcriptional regulation
Authors: Gibson, W. J.; Sadagopan, A.; Shoba, V. M.; Choudhary, A.; Meyerson, M. L.; Schreiber, S. L.
Score: 33.8, Published: 2023-07-07 DOI: 10.1101/2023.07.07.548101
The aberrant localization of proteins in cells is a key factor in the development of various diseases, including cancer and neurodegenerative disease. To better understand and potentially manipulate protein localization for therapeutic purposes, we engineered bifunctional compounds that bind to proteins in separate cellular compartments."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Bifunctional small molecules that induce nuclearlocalization and targeted transcriptional regulation\nAuthors: Gibson, W. J.; Sadagopan, A.; Shoba, V. M.; Choudhary, A.; Meyerson, M. L.; Schreiber, S. L.\nScore: 33.8, Published: 2023-07-07 DOI: 10.1101/2023.07.07.548101\nThe aberrant localization of proteins in cells is a key factor in the development of various diseases, including cancer and neurodegenerative disease. To better understand and potentially manipulate protein localization for therapeutic purposes, we engineered bifunctional compounds that bind to proteins in separate cellular compartments.",
  "keywords": [
    
  ],
  "articleBody": " Bifunctional small molecules that induce nuclearlocalization and targeted transcriptional regulation\nAuthors: Gibson, W. J.; Sadagopan, A.; Shoba, V. M.; Choudhary, A.; Meyerson, M. L.; Schreiber, S. L.\nScore: 33.8, Published: 2023-07-07 DOI: 10.1101/2023.07.07.548101\nThe aberrant localization of proteins in cells is a key factor in the development of various diseases, including cancer and neurodegenerative disease. To better understand and potentially manipulate protein localization for therapeutic purposes, we engineered bifunctional compounds that bind to proteins in separate cellular compartments. We show these compounds induce nuclear import of cytosolic cargoes, using nuclear-localized BRD4 as a \"carrier\" for co-import and nuclear trapping of cytosolic proteins. We use this system to calculate kinetic constants for passive diffusion across the nuclear pore and demonstrate single-cell heterogeneity in response to these bifunctional molecules, with cells requiring high carrier to cargo expression for complete import. We also observe incorporation of cargoes into BRD4-containing condensates. Proteins shown to be substrates for nuclear transport include oncogenic mutant nucleophosmin (NPM1c) and mutant PI3K catalytic subunit alpha (PIK3CA E545K), suggesting potential applications to cancer treatment. In addition, we demonstrate that chemical-induced localization of BRD4 to cytosolic-localized DNA-binding proteins, namely, IRF1 with a nuclear export signal, induces target gene expression. These results suggest that induced localization of proteins with bifunctional molecules enables the rewiring of cell circuitry with significant implications for disease therapy.\nMissing-link fold reveals the evolutionary pathway between RNA polymerase and ribosomal proteins\nAuthors: Yagi, S.; Tagami, S.\nScore: 16.7, Published: 2023-07-05 DOI: 10.1101/2023.07.05.547881\nNumerous molecular machines are required to drive the central dogma of molecular biology. However, the means by which such numerous proteins emerged in the early evolutionary stage of life remains enigmatic. Many of them possess small {beta}-barrel folds with different topologies, represented by DPBB conserved in DNA and RNA polymerases, and RIFT, OB, and SH3 in ribosomal proteins. Here, we discovered that the previously reconstructed ancient DPBB sequence could also adopt a novel {beta}-barrel fold named DZBB, which shares similarities with RIFT and OB. Indeed, DZBB could be transformed into them through simple engineering experiments. Furthermore, the OB designs could be converted into SH3 by circular-permutation. These results indicate that these {beta}-barrels diversified quickly from a common ancestor at the beginning of the central dogma evolution.\nStructure-guided engineering of biased-agonism in the human niacin receptor via single amino acid substitution\nAuthors: Yadav, M.; Sarma, P.; Ganguly, M.; Mishra, S.; Maharana, J.; Zaidi, N.; Dalal, A.; Singh, V.; Saha, S.; Mahajan, G.; Sharma, S.; Chami, M.; Banerjee, R.; Shukla, A. K.\nScore: 16.4, Published: 2023-07-03 DOI: 10.1101/2023.07.03.547505\nThe Hydroxycarboxylic acid receptor 2 (HCA2), also known as the niacin receptor or GPR109A, is a prototypical G protein-coupled receptor that plays a central role in the inhibition of lipolytic and atherogenic activities in our body. Interestingly, GPR109A activation also results in vasodilation that is linked to the side-effect of flushing associated with dyslipidemia drugs such as niacin. This receptor continues to be a key target for developing novel pharmacophores and lead compounds as potential therapeutics in dyslipidemia with minimized flushing response, however, the lack of structural insights into agonist-binding and receptor activation has limited the efforts. Here, we present five different cryo-EM structures of the GPR109A-G-protein complexes with the receptor bound to dyslipidemia drugs, niacin or acipimox, non-flushing agonists, MK6892 or GSK256073, and recently approved psoriasis drug, monomethyl fumarate (MMF). These structures allow us to visualize the binding mechanism of agonists with a conserved molecular interaction network, and elucidate the previously lacking molecular basis of receptor activation and transducer-coupling. Importantly, cellular pharmacology experiments, guided by the structural framework determined here, elucidate pathway-selective biased signaling elicited by the non-flushing agonists. Finally, taking lead from the structural insights, we successfully engineered receptor mutants via single amino acid substitutions that either fail to elicit agonist-induced transducer-coupling or exhibits G-protein signaling bias. Taken together, our study provides previously lacking structural framework to understand the agonist-binding and activation of GPR109A, and opens up the possibilities of structure-guided novel drug discovery targeting this therapeutically important receptor.\nMicroED structure of the human vasopressin 1B receptor\nAuthors: Shiriaeva, A.; Martynowycz, M. W.; Nicolas, W. J.; Cherezov, V.; Gonen, T.\nScore: 14.2, Published: 2023-07-06 DOI: 10.1101/2023.07.05.547888\nThe small size and flexibility of G protein-coupled receptors (GPCRs) have long posed a significant challenge to determining their structures for research and therapeutic applications. Single particle cryogenic electron microscopy (cryoEM) is often out of reach due to the small size of the receptor without a signaling partner. Crystallization of GPCRs in lipidic cubic phase (LCP) often results in crystals that may be too small and difficult to analyze using X-ray microcrystallography at synchrotron sources or even serial femtosecond crystallography at X-ray free electron lasers. Here, we determine the previously unknown structure of the human vasopressin 1B receptor (V1BR) using microcrystal electron diffraction (MicroED). To achieve this, we grew V1BR microcrystals in LCP and transferred the material directly onto electron microscopy grids. The protein was labeled with a fluorescent dye prior to crystallization to locate the microcrystals using cryogenic fluorescence microscopy, and then the surrounding material was removed using a plasma-focused ion beam to thin the sample to a thickness amenable to MicroED. MicroED data from 14 crystalline lamellae were used to determine the 3.2 A structure of the receptor in the crystallographic space group P 1. These results demonstrate the use of MicroED to determine previously unknown GPCR structures that, despite significant effort, were not tractable by other methods.\nMolecular insights into atypical modes of beta-arrestin interaction with seven transmembrane receptors\nAuthors: Maharana, J.; Sano, F. K.; Sarma, P.; Yadav, M. K.; Duan, L.; Stepniewski, T. M.; Chaturvedi, M.; Ranjan, A.; Singh, V.; Saha, S.; Mahajan, G.; Chami, M.; Shihoya, W.; Selent, J.; Chung, K. Y.; Banerjee, R.; Nureki, O.; Shukla, A. K.\nScore: 14.0, Published: 2023-07-05 DOI: 10.1101/2023.07.05.547776\nBeta-arrestins are multifunctional proteins that are critically involved in regulating spatio-temporal aspects of GPCR signaling. The interaction of beta-arrestins with GPCRs is typically conceptualized in terms of receptor activation and phosphorylation primarily in the carboxyl-terminus. Interestingly however, there are several GPCRs that harbor majority of phosphorylation sites in their 3rd intracellular loop (ICL3) instead of carboxyl-terminus but still robustly engage beta-arrestins. Moreover, there are several 7TMRs that are now characterized as intrinsically-biased, beta-arrestin-coupled receptors (ACRs) due to lack of functional G-protein-coupling but robust beta-arrestin binding leading to functional outcomes. The molecular basis of beta-arrestin interaction and activation upon binding to these types of 7TMRs is currently elusive, and it represents a major knowledge gap in our current understanding of this signaling system. Here, we present seven cryo-EM structures of beta-arrestins in basal state, activated by the muscarinic M2 receptor (M2R) through its ICL3, and a beta-arrestin-coupled receptor known as decoy D6 receptor (D6R). These structural snapshots combined with biochemical, cellular, and biophysical experiments including HDX-MS and MD simulation provide novel insights into the ability of beta-arrestins to preferentially select specific phosphorylation patterns in the receptors, and also illuminate the structural diversity in 7TMR-beta-arrestin interaction. Surprisingly, we also observe that the carboxyl-terminus of beta-arrestin2 but not beta-arrestin1 undergoes structural transition from a beta-strand to alpha-helix upon activation by D6R, which may preclude the core-interaction with the activated receptor. Taken together, our study elucidates previously unappreciated aspects of 7TMR-beta-arrestin interaction, and provides important mechanistic clues about how the two isoforms of beta-arrestins can recognize and regulate a large repertoire of GPCRs.\nAn engineered biosensor enables dynamic aspartate measurements in living cells\nAuthors: Davidsen, K.; Marvin, J. S.; Aggarwal, A.; Brown, T. A.; Sullivan, L. B.\nScore: 22.7, Published: 2023-06-27 DOI: 10.1101/2023.06.27.546775\nIntracellular levels of the amino acid aspartate are responsive to changes in metabolism in mammalian cells and can correspondingly alter cell function, highlighting the need for robust tools to measure aspartate abundance. However, comprehensive understanding of aspartate metabolism has been limited by the throughput, cost, and static nature of the mass spectrometry based measurements that are typically employed to measure aspartate levels. To address these issues, we have developed a GFP-based sensor of aspartate (jAspSnFR3), where the fluorescence intensity corresponds to aspartate concentration. As a purified protein, the sensor has a 20-fold increase in fluorescence upon aspartate saturation, with dose dependent fluorescence changes covering a physiologically relevant aspartate concentration range and no significant off target binding. Expressed in mammalian cell lines, sensor intensity correlated with aspartate levels measured by mass spectrometry and could resolve temporal changes in intracellular aspartate from genetic, pharmacological, and nutritional manipulations. These data demonstrate the utility of jAspSnFR3 and highlight the opportunities it provides for temporally resolved and high throughput applications of variables that affect aspartate levels.\nReverse engineering the anti-MUC1 hybridoma antibody 139H2 by mass spectrometry-based de novo sequencing\nAuthors: Peng, W.; Giesbers, K. C. A. P.; Siborova, M.; Beugelink, J. W.; Pronker, M. F.; Schulte, D.; Hilkens, J.; Janssen, B. J. C.; Strijbis, K.; Snijder, J.\nScore: 9.0, Published: 2023-07-05 DOI: 10.1101/2023.07.05.547778\nMucin 1 (MUC1) is a transmembrane mucin expressed at the apical surface of epithelial cells at different mucosal surfaces including breast and intestine. In the gastrointestinal tract, MUC1 has a barrier function against bacterial invasion, but can also serve as an entry receptor for pathogenic Salmonella bacteria. Moreover, MUC1 is well known for its aberrant expression and glycosylation in adenocarcinomas The MUC1 extracellular domain contains a variable number of tandem repeats (VNTR) of 20 amino acids, which are heavily O-linked glycosylated. Monoclonal antibodies against the MUC1 VNTR can be powerful tools because of their multiplicity of binding and possible applications in the diagnosis and treatment of MUC1-expressing cancers. One such antibody is the hybridoma mouse monoclonal 139H2, which is also widely used as a research tool to study non-cancer MUC1. Here we report direct mass spectrometry-based sequencing of hybridoma-derived 139H2 IgG, which enabled reverse engineering of a recombinant 139H2. The performance of the reverse engineered 139H2 IgG and its Fab fragment were validated by comparison to the hybridoma-derived product in Western blot and immunofluorescence microscopy. The reverse engineering of 139H2 allowed us to characterize binding to the VNTR peptide epitope by surface plasmon resonance (SPR) and solve the crystal structure of the 139H2 Fab fragment in complex with the MUC1 VNTR peptide. These analyses reveal the molecular basis for 139H2 binding specificity to MUC1 and its tolerance to O-glycosylation of the VNTR. The available sequence of 139H2 will allow further development of MUC1-related diagnostics, targeting and treatment strategies.\nStructural characterization of the Sel1-like repeat protein LceB from Legionella pneumophila\nAuthors: Penner, T. V.; Lorente Cobo, N.; Patel, D. T.; Patel, D. H.; Savchenko, A.; Brassinga, A. K. C.; Prehna, G.\nScore: 4.3, Published: 2023-07-03 DOI: 10.1101/2023.07.03.547437\nLegionella are freshwater Gram-negative bacteria that in their normal environment infect protozoa. However, this adaptation also allows Legionella to infect human alveolar macrophages and cause pneumonia. Central to Legionella pathogenesis are more than 330 secreted effectors, of which there are 9 core effectors that are conserved in all pathogenic species. Despite their importance, the biochemical function of several core effectors remains unclear. To address this, we have taken a structural approach to characterize the core effector of unknown function LceB, or Lpg1356, from Legionella pneumophila. Here we solve an X-ray crystal structure of LceB using an AlphaFold model for molecular replacement. The experimental structure shows that LceB adopts a Sel1-like repeat fold as predicted. However, the crystal structure captured multiple conformations of LceB all of which differed from the AlphaFold model. Comparison of the predicted model and the experimental models suggests that LceB is highly flexible in solution. Additionally, molecular analysis of LceB using its close structural homologues reveals sequence and structural motifs of known biochemical function. Specifically, LceB harbors a repeated KAAEQG motif that both stabilizes the Sel1-like repeat fold and is known to participate in protein-protein interactions with eukaryotic host proteins. We also observe that LceB forms several higher-order oligomers in solution. Overall, our results have revealed that LceB has conformational flexibility, self-associates, and contains a molecular surface for binding a target host-cell protein. Additionally, our data provides structural insights into the Sel1-like repeat family of proteins that remain poorly studied.\nStructure and Biosynthesis of Hectoramide B, a Linear Depsipeptide from the Marine Cyanobacterium Moorena producens JHB Discovered via Co-culture with Candida albicans\nAuthors: Ngo, T.-E.; Ecker, A. K.; Guild, A.; Remmel, A.; Boudreau, P. B.; Alexander, K. L.; Naman, C. B.; Glukhov, E.; Avalon, N. E.; Shende, V. V.; Gerwick, L.; Gerwick, W. H.\nScore: 3.8, Published: 2023-07-06 DOI: 10.1101/2023.07.06.547815\nThe tropical marine cyanobacterium Moorena producens JHB is a prolific source of secondary metabolites with potential biomedical utility. Previous studies of this strain led to the discovery of several novel compounds such as the hectochlorins and jamaicamides; however, bioinformatic analyses of its genome suggested that there were many more cryptic biosynthetic gene clusters yet to be characterized. To potentially stimulate the production of novel compounds from this strain, it was co-cultured with Candida albicans. From this experiment, we observed the increased production of a new compound that we characterize here as hectoramide B. Bioinformatic analysis of the M. producens JHB genome enabled the identification of a putative biosynthetic gene cluster responsible for hectoramide B biosynthesis. This work demonstrates that co-culture competition experiments can be a valuable method to facilitate the discovery of novel natural products from cyanobacteria.\nHuR-miRNA complex Activates RAS GTPase RalA to Facilitate Endosome Targeting and Extracellular Export of miRNAs\nAuthors: Ghosh, S.; Choudhury, S. H.; Mukherjee, K.; Bhattacharyya, S. N.\nScore: 3.7, Published: 2023-06-23 DOI: 10.1101/2023.06.22.546187\nExtracellular vesicles-mediated exchange of miRNA cargos between diverse types of mammalian cells is a major mechanism of controlling cellular miRNA levels and activity and thus to regulate expression of miRNA-target genes in both donor and recipient cells. Despite tremendous excitement related to extracellular vesicles-associated miRNAs as biomarkers or having therapeutic potential, the mechanism of selective packaging of miRNAs into endosomes and multivesicular bodies for subsequent extracellular export is a poorly studied area due to lack of assay system to study such processes in vitro. We have developed an in vitro assay with endosomes isolated from mammalian macrophage cells to follow miRNA packaging into endocytic organelles. The synthetic miRNAs, used in the assay, get imported inside the isolated endosomes during the in vitro reaction and become protected from RNase in a time and concentration dependent manner. The selective miRNA accumulation inside endosomes requires both ATP and GTP hydrolysis and the miRNA binding protein HuR. The HuR-miRNA complex binds and stimulates the endosomal RalA GTPase to facilitate the import of miRNAs into endosomes and their subsequent export as part of the extracellular vesicles. The endosomal targeting of miRNAs is also very much dependent on endosome maturation process that is controlled by Rab5 protein and ATP. Graphical Abstract{whitebullet} miRNAs get imported to endosomes in vitro and are become protected to RNase {whitebullet}Endosomal miRNA import is sequence specific and requires miRNA exporter protein HuR {whitebullet}HuR-miRNA complex activates RalA GTPases to complete miRNA import. {whitebullet}Rab5 protein and ATP hydrolysis is required for endosome maturation and miRNA import O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=149 SRC=\"FIGDIR/small/546187v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (31K): org.highwire.dtl.DTLVardef@217e52org.highwire.dtl.DTLVardef@ec6190org.highwire.dtl.DTLVardef@f972f5org.highwire.dtl.DTLVardef@ea67a4_HPS_FORMAT_FIGEXP M_FIG C_FIG\n",
  "wordCount" : "2489",
  "inLanguage": "en",
  "datePublished": "2023-07-09T10:40:17Z",
  "dateModified": "2023-07-09T10:40:17Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta"><span>updated on July 9, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548101">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548101" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548101">
        <p class="paperTitle">Bifunctional small molecules that induce nuclearlocalization and targeted transcriptional regulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548101" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548101" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gibson, W. J.; Sadagopan, A.; Shoba, V. M.; Choudhary, A.; Meyerson, M. L.; Schreiber, S. L.</p>
        <p class="info">Score: 33.8, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548101' target='https://doi.org/10.1101/2023.07.07.548101'> 10.1101/2023.07.07.548101</a></p>
        <p class="abstract">The aberrant localization of proteins in cells is a key factor in the development of various diseases, including cancer and neurodegenerative disease. To better understand and potentially manipulate protein localization for therapeutic purposes, we engineered bifunctional compounds that bind to proteins in separate cellular compartments. We show these compounds induce nuclear import of cytosolic cargoes, using nuclear-localized BRD4 as a &#34;carrier&#34; for co-import and nuclear trapping of cytosolic proteins. We use this system to calculate kinetic constants for passive diffusion across the nuclear pore and demonstrate single-cell heterogeneity in response to these bifunctional molecules, with cells requiring high carrier to cargo expression for complete import. We also observe incorporation of cargoes into BRD4-containing condensates. Proteins shown to be substrates for nuclear transport include oncogenic mutant nucleophosmin (NPM1c) and mutant PI3K catalytic subunit alpha (PIK3CA E545K), suggesting potential applications to cancer treatment. In addition, we demonstrate that chemical-induced localization of BRD4 to cytosolic-localized DNA-binding proteins, namely, IRF1 with a nuclear export signal, induces target gene expression. These results suggest that induced localization of proteins with bifunctional molecules enables the rewiring of cell circuitry with significant implications for disease therapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.547881">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.547881" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.547881">
        <p class="paperTitle">Missing-link fold reveals the evolutionary pathway between RNA polymerase and ribosomal proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.547881" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.547881" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yagi, S.; Tagami, S.</p>
        <p class="info">Score: 16.7, Published: 2023-07-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.547881' target='https://doi.org/10.1101/2023.07.05.547881'> 10.1101/2023.07.05.547881</a></p>
        <p class="abstract">Numerous molecular machines are required to drive the central dogma of molecular biology. However, the means by which such numerous proteins emerged in the early evolutionary stage of life remains enigmatic. Many of them possess small {beta}-barrel folds with different topologies, represented by DPBB conserved in DNA and RNA polymerases, and RIFT, OB, and SH3 in ribosomal proteins. Here, we discovered that the previously reconstructed ancient DPBB sequence could also adopt a novel {beta}-barrel fold named DZBB, which shares similarities with RIFT and OB. Indeed, DZBB could be transformed into them through simple engineering experiments. Furthermore, the OB designs could be converted into SH3 by circular-permutation. These results indicate that these {beta}-barrels diversified quickly from a common ancestor at the beginning of the central dogma evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547505">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547505" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547505">
        <p class="paperTitle">Structure-guided engineering of biased-agonism in the human niacin receptor via single amino acid substitution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547505" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547505" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yadav, M.; Sarma, P.; Ganguly, M.; Mishra, S.; Maharana, J.; Zaidi, N.; Dalal, A.; Singh, V.; Saha, S.; Mahajan, G.; Sharma, S.; Chami, M.; Banerjee, R.; Shukla, A. K.</p>
        <p class="info">Score: 16.4, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547505' target='https://doi.org/10.1101/2023.07.03.547505'> 10.1101/2023.07.03.547505</a></p>
        <p class="abstract">The Hydroxycarboxylic acid receptor 2 (HCA2), also known as the niacin receptor or GPR109A, is a prototypical G protein-coupled receptor that plays a central role in the inhibition of lipolytic and atherogenic activities in our body. Interestingly, GPR109A activation also results in vasodilation that is linked to the side-effect of flushing associated with dyslipidemia drugs such as niacin. This receptor continues to be a key target for developing novel pharmacophores and lead compounds as potential therapeutics in dyslipidemia with minimized flushing response, however, the lack of structural insights into agonist-binding and receptor activation has limited the efforts. Here, we present five different cryo-EM structures of the GPR109A-G-protein complexes with the receptor bound to dyslipidemia drugs, niacin or acipimox, non-flushing agonists, MK6892 or GSK256073, and recently approved psoriasis drug, monomethyl fumarate (MMF). These structures allow us to visualize the binding mechanism of agonists with a conserved molecular interaction network, and elucidate the previously lacking molecular basis of receptor activation and transducer-coupling. Importantly, cellular pharmacology experiments, guided by the structural framework determined here, elucidate pathway-selective biased signaling elicited by the non-flushing agonists. Finally, taking lead from the structural insights, we successfully engineered receptor mutants via single amino acid substitutions that either fail to elicit agonist-induced transducer-coupling or exhibits G-protein signaling bias. Taken together, our study provides previously lacking structural framework to understand the agonist-binding and activation of GPR109A, and opens up the possibilities of structure-guided novel drug discovery targeting this therapeutically important receptor.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.547888">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.547888" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.547888">
        <p class="paperTitle">MicroED structure of the human vasopressin 1B receptor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.547888" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.547888" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shiriaeva, A.; Martynowycz, M. W.; Nicolas, W. J.; Cherezov, V.; Gonen, T.</p>
        <p class="info">Score: 14.2, Published: 2023-07-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.547888' target='https://doi.org/10.1101/2023.07.05.547888'> 10.1101/2023.07.05.547888</a></p>
        <p class="abstract">The small size and flexibility of G protein-coupled receptors (GPCRs) have long posed a significant challenge to determining their structures for research and therapeutic applications. Single particle cryogenic electron microscopy (cryoEM) is often out of reach due to the small size of the receptor without a signaling partner. Crystallization of GPCRs in lipidic cubic phase (LCP) often results in crystals that may be too small and difficult to analyze using X-ray microcrystallography at synchrotron sources or even serial femtosecond crystallography at X-ray free electron lasers. Here, we determine the previously unknown structure of the human vasopressin 1B receptor (V1BR) using microcrystal electron diffraction (MicroED). To achieve this, we grew V1BR microcrystals in LCP and transferred the material directly onto electron microscopy grids. The protein was labeled with a fluorescent dye prior to crystallization to locate the microcrystals using cryogenic fluorescence microscopy, and then the surrounding material was removed using a plasma-focused ion beam to thin the sample to a thickness amenable to MicroED. MicroED data from 14 crystalline lamellae were used to determine the 3.2 A structure of the receptor in the crystallographic space group P 1. These results demonstrate the use of MicroED to determine previously unknown GPCR structures that, despite significant effort, were not tractable by other methods.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.547776">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.547776" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.547776">
        <p class="paperTitle">Molecular insights into atypical modes of beta-arrestin interaction with seven transmembrane receptors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.547776" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.547776" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Maharana, J.; Sano, F. K.; Sarma, P.; Yadav, M. K.; Duan, L.; Stepniewski, T. M.; Chaturvedi, M.; Ranjan, A.; Singh, V.; Saha, S.; Mahajan, G.; Chami, M.; Shihoya, W.; Selent, J.; Chung, K. Y.; Banerjee, R.; Nureki, O.; Shukla, A. K.</p>
        <p class="info">Score: 14.0, Published: 2023-07-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.547776' target='https://doi.org/10.1101/2023.07.05.547776'> 10.1101/2023.07.05.547776</a></p>
        <p class="abstract">Beta-arrestins are multifunctional proteins that are critically involved in regulating spatio-temporal aspects of GPCR signaling. The interaction of beta-arrestins with GPCRs is typically conceptualized in terms of receptor activation and phosphorylation primarily in the carboxyl-terminus. Interestingly however, there are several GPCRs that harbor majority of phosphorylation sites in their 3rd intracellular loop (ICL3) instead of carboxyl-terminus but still robustly engage beta-arrestins. Moreover, there are several 7TMRs that are now characterized as intrinsically-biased, beta-arrestin-coupled receptors (ACRs) due to lack of functional G-protein-coupling but robust beta-arrestin binding leading to functional outcomes. The molecular basis of beta-arrestin interaction and activation upon binding to these types of 7TMRs is currently elusive, and it represents a major knowledge gap in our current understanding of this signaling system. Here, we present seven cryo-EM structures of beta-arrestins in basal state, activated by the muscarinic M2 receptor (M2R) through its ICL3, and a beta-arrestin-coupled receptor known as decoy D6 receptor (D6R). These structural snapshots combined with biochemical, cellular, and biophysical experiments including HDX-MS and MD simulation provide novel insights into the ability of beta-arrestins to preferentially select specific phosphorylation patterns in the receptors, and also illuminate the structural diversity in 7TMR-beta-arrestin interaction. Surprisingly, we also observe that the carboxyl-terminus of beta-arrestin2 but not beta-arrestin1 undergoes structural transition from a beta-strand to alpha-helix upon activation by D6R, which may preclude the core-interaction with the activated receptor. Taken together, our study elucidates previously unappreciated aspects of 7TMR-beta-arrestin interaction, and provides important mechanistic clues about how the two isoforms of beta-arrestins can recognize and regulate a large repertoire of GPCRs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546775">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546775" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546775">
        <p class="paperTitle">An engineered biosensor enables dynamic aspartate measurements in living cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546775" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546775" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Davidsen, K.; Marvin, J. S.; Aggarwal, A.; Brown, T. A.; Sullivan, L. B.</p>
        <p class="info">Score: 22.7, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546775' target='https://doi.org/10.1101/2023.06.27.546775'> 10.1101/2023.06.27.546775</a></p>
        <p class="abstract">Intracellular levels of the amino acid aspartate are responsive to changes in metabolism in mammalian cells and can correspondingly alter cell function, highlighting the need for robust tools to measure aspartate abundance. However, comprehensive understanding of aspartate metabolism has been limited by the throughput, cost, and static nature of the mass spectrometry based measurements that are typically employed to measure aspartate levels. To address these issues, we have developed a GFP-based sensor of aspartate (jAspSnFR3), where the fluorescence intensity corresponds to aspartate concentration. As a purified protein, the sensor has a 20-fold increase in fluorescence upon aspartate saturation, with dose dependent fluorescence changes covering a physiologically relevant aspartate concentration range and no significant off target binding. Expressed in mammalian cell lines, sensor intensity correlated with aspartate levels measured by mass spectrometry and could resolve temporal changes in intracellular aspartate from genetic, pharmacological, and nutritional manipulations. These data demonstrate the utility of jAspSnFR3 and highlight the opportunities it provides for temporally resolved and high throughput applications of variables that affect aspartate levels.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.547778">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.547778" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.547778">
        <p class="paperTitle">Reverse engineering the anti-MUC1 hybridoma antibody 139H2 by mass spectrometry-based de novo sequencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.547778" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.547778" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peng, W.; Giesbers, K. C. A. P.; Siborova, M.; Beugelink, J. W.; Pronker, M. F.; Schulte, D.; Hilkens, J.; Janssen, B. J. C.; Strijbis, K.; Snijder, J.</p>
        <p class="info">Score: 9.0, Published: 2023-07-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.547778' target='https://doi.org/10.1101/2023.07.05.547778'> 10.1101/2023.07.05.547778</a></p>
        <p class="abstract">Mucin 1 (MUC1) is a transmembrane mucin expressed at the apical surface of epithelial cells at different mucosal surfaces including breast and intestine. In the gastrointestinal tract, MUC1 has a barrier function against bacterial invasion, but can also serve as an entry receptor for pathogenic Salmonella bacteria. Moreover, MUC1 is well known for its aberrant expression and glycosylation in adenocarcinomas The MUC1 extracellular domain contains a variable number of tandem repeats (VNTR) of 20 amino acids, which are heavily O-linked glycosylated. Monoclonal antibodies against the MUC1 VNTR can be powerful tools because of their multiplicity of binding and possible applications in the diagnosis and treatment of MUC1-expressing cancers. One such antibody is the hybridoma mouse monoclonal 139H2, which is also widely used as a research tool to study non-cancer MUC1. Here we report direct mass spectrometry-based sequencing of hybridoma-derived 139H2 IgG, which enabled reverse engineering of a recombinant 139H2. The performance of the reverse engineered 139H2 IgG and its Fab fragment were validated by comparison to the hybridoma-derived product in Western blot and immunofluorescence microscopy. The reverse engineering of 139H2 allowed us to characterize binding to the VNTR peptide epitope by surface plasmon resonance (SPR) and solve the crystal structure of the 139H2 Fab fragment in complex with the MUC1 VNTR peptide. These analyses reveal the molecular basis for 139H2 binding specificity to MUC1 and its tolerance to O-glycosylation of the VNTR. The available sequence of 139H2 will allow further development of MUC1-related diagnostics, targeting and treatment strategies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547437">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547437" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547437">
        <p class="paperTitle">Structural characterization of the Sel1-like repeat protein LceB from Legionella pneumophila</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547437" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547437" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Penner, T. V.; Lorente Cobo, N.; Patel, D. T.; Patel, D. H.; Savchenko, A.; Brassinga, A. K. C.; Prehna, G.</p>
        <p class="info">Score: 4.3, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547437' target='https://doi.org/10.1101/2023.07.03.547437'> 10.1101/2023.07.03.547437</a></p>
        <p class="abstract">Legionella are freshwater Gram-negative bacteria that in their normal environment infect protozoa. However, this adaptation also allows Legionella to infect human alveolar macrophages and cause pneumonia. Central to Legionella pathogenesis are more than 330 secreted effectors, of which there are 9 core effectors that are conserved in all pathogenic species. Despite their importance, the biochemical function of several core effectors remains unclear. To address this, we have taken a structural approach to characterize the core effector of unknown function LceB, or Lpg1356, from Legionella pneumophila. Here we solve an X-ray crystal structure of LceB using an AlphaFold model for molecular replacement. The experimental structure shows that LceB adopts a Sel1-like repeat fold as predicted. However, the crystal structure captured multiple conformations of LceB all of which differed from the AlphaFold model. Comparison of the predicted model and the experimental models suggests that LceB is highly flexible in solution. Additionally, molecular analysis of LceB using its close structural homologues reveals sequence and structural motifs of known biochemical function. Specifically, LceB harbors a repeated KAAEQG motif that both stabilizes the Sel1-like repeat fold and is known to participate in protein-protein interactions with eukaryotic host proteins. We also observe that LceB forms several higher-order oligomers in solution. Overall, our results have revealed that LceB has conformational flexibility, self-associates, and contains a molecular surface for binding a target host-cell protein. Additionally, our data provides structural insights into the Sel1-like repeat family of proteins that remain poorly studied.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.547815">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.547815" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.547815">
        <p class="paperTitle">Structure and Biosynthesis of Hectoramide B, a Linear Depsipeptide from the Marine Cyanobacterium Moorena producens JHB Discovered via Co-culture with Candida albicans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.547815" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.547815" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ngo, T.-E.; Ecker, A. K.; Guild, A.; Remmel, A.; Boudreau, P. B.; Alexander, K. L.; Naman, C. B.; Glukhov, E.; Avalon, N. E.; Shende, V. V.; Gerwick, L.; Gerwick, W. H.</p>
        <p class="info">Score: 3.8, Published: 2023-07-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.547815' target='https://doi.org/10.1101/2023.07.06.547815'> 10.1101/2023.07.06.547815</a></p>
        <p class="abstract">The tropical marine cyanobacterium Moorena producens JHB is a prolific source of secondary metabolites with potential biomedical utility. Previous studies of this strain led to the discovery of several novel compounds such as the hectochlorins and jamaicamides; however, bioinformatic analyses of its genome suggested that there were many more cryptic biosynthetic gene clusters yet to be characterized. To potentially stimulate the production of novel compounds from this strain, it was co-cultured with Candida albicans. From this experiment, we observed the increased production of a new compound that we characterize here as hectoramide B. Bioinformatic analysis of the M. producens JHB genome enabled the identification of a putative biosynthetic gene cluster responsible for hectoramide B biosynthesis. This work demonstrates that co-culture competition experiments can be a valuable method to facilitate the discovery of novel natural products from cyanobacteria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.22.546187">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.22.546187" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.22.546187">
        <p class="paperTitle">HuR-miRNA complex Activates RAS GTPase RalA to Facilitate Endosome Targeting and Extracellular Export of miRNAs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.22.546187" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.22.546187" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ghosh, S.; Choudhury, S. H.; Mukherjee, K.; Bhattacharyya, S. N.</p>
        <p class="info">Score: 3.7, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.22.546187' target='https://doi.org/10.1101/2023.06.22.546187'> 10.1101/2023.06.22.546187</a></p>
        <p class="abstract">Extracellular vesicles-mediated exchange of miRNA cargos between diverse types of mammalian cells is a major mechanism of controlling cellular miRNA levels and activity and thus to regulate expression of miRNA-target genes in both donor and recipient cells. Despite tremendous excitement related to extracellular vesicles-associated miRNAs as biomarkers or having therapeutic potential, the mechanism of selective packaging of miRNAs into endosomes and multivesicular bodies for subsequent extracellular export is a poorly studied area due to lack of assay system to study such processes in vitro. We have developed an in vitro assay with endosomes isolated from mammalian macrophage cells to follow miRNA packaging into endocytic organelles. The synthetic miRNAs, used in the assay, get imported inside the isolated endosomes during the in vitro reaction and become protected from RNase in a time and concentration dependent manner. The selective miRNA accumulation inside endosomes requires both ATP and GTP hydrolysis and the miRNA binding protein HuR. The HuR-miRNA complex binds and stimulates the endosomal RalA GTPase to facilitate the import of miRNAs into endosomes and their subsequent export as part of the extracellular vesicles. The endosomal targeting of miRNAs is also very much dependent on endosome maturation process that is controlled by Rab5 protein and ATP.

Graphical Abstract{whitebullet} miRNAs get imported to endosomes in vitro and are become protected to RNase
{whitebullet}Endosomal miRNA import is sequence specific and requires miRNA exporter protein HuR
{whitebullet}HuR-miRNA complex activates RalA GTPases to complete miRNA import.
{whitebullet}Rab5 protein and ATP hydrolysis is required for endosome maturation and miRNA import


O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=149 SRC=&#34;FIGDIR/small/546187v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (31K):
org.highwire.dtl.DTLVardef@217e52org.highwire.dtl.DTLVardef@ec6190org.highwire.dtl.DTLVardef@f972f5org.highwire.dtl.DTLVardef@ea67a4_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
