<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A systems-scale, integrated view of the ubiquitylation site occupancy and dynamics
Authors: Prus, G.; Satpathy, S.; Weinert, B.; Narita, T.; Choudhary, C.
Score: 32.7, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549470
Ubiquitylation regulates virtually all proteins and biological processes in a cell. However, the global site-specific occupancy (stoichiometry) and turnover rate of ubiquitylation have never been quantified. Here, we present the first integrated picture of ubiquitylation site occupancy and half-life. Ubiquitylation occupancy spans four orders of magnitude, but the median ubiquitylation site occupancy is three orders of magnitude lower than that of phosphorylation.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="A systems-scale, integrated view of the ubiquitylation site occupancy and dynamics
Authors: Prus, G.; Satpathy, S.; Weinert, B.; Narita, T.; Choudhary, C.
Score: 32.7, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549470
Ubiquitylation regulates virtually all proteins and biological processes in a cell. However, the global site-specific occupancy (stoichiometry) and turnover rate of ubiquitylation have never been quantified. Here, we present the first integrated picture of ubiquitylation site occupancy and half-life. Ubiquitylation occupancy spans four orders of magnitude, but the median ubiquitylation site occupancy is three orders of magnitude lower than that of phosphorylation." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-23T10:36:34+00:00" />
<meta property="article:modified_time" content="2023-07-23T10:36:34+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="A systems-scale, integrated view of the ubiquitylation site occupancy and dynamics
Authors: Prus, G.; Satpathy, S.; Weinert, B.; Narita, T.; Choudhary, C.
Score: 32.7, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549470
Ubiquitylation regulates virtually all proteins and biological processes in a cell. However, the global site-specific occupancy (stoichiometry) and turnover rate of ubiquitylation have never been quantified. Here, we present the first integrated picture of ubiquitylation site occupancy and half-life. Ubiquitylation occupancy spans four orders of magnitude, but the median ubiquitylation site occupancy is three orders of magnitude lower than that of phosphorylation."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "A systems-scale, integrated view of the ubiquitylation site occupancy and dynamics\nAuthors: Prus, G.; Satpathy, S.; Weinert, B.; Narita, T.; Choudhary, C.\nScore: 32.7, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549470\nUbiquitylation regulates virtually all proteins and biological processes in a cell. However, the global site-specific occupancy (stoichiometry) and turnover rate of ubiquitylation have never been quantified. Here, we present the first integrated picture of ubiquitylation site occupancy and half-life. Ubiquitylation occupancy spans four orders of magnitude, but the median ubiquitylation site occupancy is three orders of magnitude lower than that of phosphorylation.",
  "keywords": [
    
  ],
  "articleBody": " A systems-scale, integrated view of the ubiquitylation site occupancy and dynamics\nAuthors: Prus, G.; Satpathy, S.; Weinert, B.; Narita, T.; Choudhary, C.\nScore: 32.7, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549470\nUbiquitylation regulates virtually all proteins and biological processes in a cell. However, the global site-specific occupancy (stoichiometry) and turnover rate of ubiquitylation have never been quantified. Here, we present the first integrated picture of ubiquitylation site occupancy and half-life. Ubiquitylation occupancy spans four orders of magnitude, but the median ubiquitylation site occupancy is three orders of magnitude lower than that of phosphorylation. The occupancy, turnover rate, and the regulation of sites by proteasome inhibitors show strong interrelationships. These properties can discriminate signaling-relevant sites from the sites involved in proteasomal degradation. The sites strongly upregulated by proteasome inhibitors have a longer half-life, and the half-life increases with increasing protein length. Importantly, a previously unknown surveillance mechanism rapidly deubiquitylates all ubiquitin-specific E1 and E2 enzymes and protects them against bystander ubiquitylation accumulation. This work reveals general principles of ubiquitylation-dependent governance and offers conceptual insights into the dynamic regulation of the cell. HighlightsO_LIUbiquitylation site occupancy is 3 orders of magnitude lower than phosphorylation C_LIO_LIThe highest 80% and the lowest 20% occupancy sites have distinct properties C_LIO_LIHigh occupancy sites are concentrated in the cytoplasmic domains of SLC proteins C_LIO_LIA dedicated mechanism prevevents ubiquitylation accumulation in E1s and E2s C_LI\nCoenzyme Q4 is a functional substitute for coenzyme Q10 and can be targeted to the mitochondria\nAuthors: Steenberge, L. H.; Sung, A. Y.; Fan, J.; Pagliarini, D. J.\nScore: 16.6, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549963\nCoenzyme Q10 (CoQ10) is an important cofactor and antioxidant for numerous cellular processes, and its deficiency has been linked to human disorders including mitochondrial disease, heart failure, Parkinsons disease, and hypertension. Unfortunately, treatment with exogenous oral CoQ10 is often ineffective, likely due to the extreme hydrophobicity and high molecular weight of CoQ10. Here, we show that less hydrophobic CoQ species with shorter isoprenoid tails can serve as viable substitutes for CoQ10 in human cells. We demonstrate that CoQ4 can perform multiple functions of CoQ10 in CoQ-deficient cells at markedly lower treatment concentrations, motivating further investigation of CoQ4 as a supplement for CoQ10 deficiencies. In addition, we describe the synthesis and evaluation of an initial set of compounds designed to target CoQ4 selectively to mitochondria using triphenylphosphonium (TPP). Our results indicate that select versions of these compounds can successfully be delivered to mitochondria in a cell model and be cleaved to produce CoQ4, laying the groundwork for further development.\nA bespoke analytical workflow for the confident identification of sulfopeptides and their discrimination from phosphopeptides\nAuthors: Daly, L. A.; Byrne, D. P.; Perkins, S.; Brownridge, P. J.; McDonnell, E.; Jones, A. R.; Eyers, P. A.; Eyers, C. E.\nScore: 11.3, Published: 2023-07-15 DOI: 10.1101/2023.07.15.549150\nProtein tyrosine sulfation (sY) is a post-translational modification (PTM) catalysed by Golgi-resident Tyrosyl Protein SulfoTransferases (TPSTs). Information on protein tyrosine sulfation is currently limited to [~]50 human proteins with only a handful having verified sites of sulfation. The contribution of this chemical moiety for the regulation of biological processes, both inside and outside the cell, remains poorly defined, in large part due to analytical limitations. Mass spectrometry-based proteomics is the method of choice for PTM analysis, but has yet to be applied for the systematic investigation and large-scale analysis of biomolecular sulfation (constituting the sulfome), primarily due to issues associated with discrimination of sY-from phosphotyrosine (pY)-containing peptides. In this study, we developed a mass spectrometry (MS)-based workflow centred on the characterization of sY-peptides, incorporating optimised Zr4+-IMAC and TiO2 enrichment strategies. Extensive characterization of a panel of sY- and pY-peptides using an array of MS fragmentation regimes (CID, HCD, EThcC, ETciD, UVPD) highlights differences in the ability to generate site-determining product ions, which can be exploited to differentiate sulfated peptides from nominally isobaric phosphopeptides based on precursor ion neutral loss at low collision energy. Application of our analytical workflow to a HEK-293 cell extracellular secretome facilitated identification of 21 new sulfotyrosine-containing proteins, several of which we validate enzymatically using in vitro sulfation assays. This study demonstrates the applicability of our strategy for confident, high-throughput, sulfomics studies, and reveals new sY interplay between enzymes relevant to both protein and glycan sulfation.\nEnhancing Inter-link Coverage in Cross-Linking Mass Spectrometry through Context-Sensitive Subgrouping and Decoy Fusion\nAuthors: Bogdanow, B.; Wang, C.; Ruwolt, M.; Ruta, J.; Mühlberg, L.; Zeng, W.-F.; Elofsson, A.; Liu, F.\nScore: 9.2, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549678\nIn cross-linking mass spectrometry, sensitivity and specificity in assigning mass spectra to cross-links between different proteins (inter-links) remains challenging. Here, we report on limitations of commonly used concatenated target-decoy searches and propose a target-decoy competition strategy on a fused database as a solution. Further, we capitalize on context-divergent error rates by implementing a novel context-sensitive subgrouping strategy. This approach increases inter-link coverage by ~ 30 - 75 % across XL-MS datasets, maintains low error rates, and preserves structural accuracy.\nFAIMS-enabled N-terminomics analysis reveals novel legumain substrates in murine spleen\nAuthors: Ziegler, A. R.; Dufour, A.; Scott, N. E.; Edgington-Mitchell, L. E.\nScore: 8.9, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549248\nAberrant levels of the asparaginyl endopeptidase legumain have been linked to inflammation, neurodegeneration and cancer, yet our understanding of this protease is incomplete. Systematic attempts to identify legumain substrates have previously been confined to in vitro studies, which fail to mirror physiological conditions and obscure biologically relevant cleavage events. Using high-field asymmetric waveform ion mobility spectrometry (FAIMS), we developed a sensitive and streamlined approach for proteome and N-terminome analyses in a single analytical method without the need for N-termini enrichment. Compared to unfractionated proteomic analysis, we demonstrate FAIMS fractionation improves neo-N- termini identification by \u003e2.5 fold, resulting in identification of \u003e2,882 unique neo-N-termini from limited sample amounts. Within murine spleens, this approach identifies 6,366 proteins and 2,528 unique neo-N-termini, with 235 cleavage events enriched in wild-type compared to legumain-deficient spleens. Among these, 119 neo-N-termini arose from asparaginyl endopeptidase activities, representing novel putative physiological legumain substrates. The direct cleavage of selected substrates by legumain was confirmed using in vitro assays, providing support for the existence of physiologically relevant extra-lysosomal legumain activity. Combined, these data shed critical light on the functions of legumain and demonstrates the utility of FAIMS as an accessible method to improve depth and quality of N- terminomics studies.\nInteraction with the membrane-anchored protein CHIC2 constrains the ubiquitin ligase activity of CHIP\nAuthors: Callahan, M. D.; Hodul, M.; Carroll, E. C.; Ravalin, M.; Nadel, C. M.; de Silva, A. R.; Cupo, A. R.; McDermott, L. E.; Nix, J. C.; Page, R. C.; Kao, A. W.; Gestwicki, J. E.\nScore: 8.3, Published: 2023-07-17 DOI: 10.1101/2023.07.17.549407\nMaintenance of cellular health requires the proper regulation of E3 ubiquitin ligases. The E3 ligase CHIP is canonically regulated by its interactions with the molecular chaperones Hsp70 and Hsp90, which focus CHIPs ubiquitination activity on misfolded proteins. Here, we report a chaperone-independent interaction of CHIP with the membrane-anchored protein CHIC2, which strongly attenuates CHIPs ligase activity. We show that CHIC2 outcompetes abundant, cytosolic chaperones through its exquisite CHIP selectivity, rather than through enhanced affinity. In proteomic experiments, we find that CHIC2 knockout phenocopies CHIP knockout in certain cell types, implying that chaperone-independent interactions can sometimes predominate CHIPs functions. Furthermore, loss of the CHIP-CHIC2 interaction induces neurodegeneration and shortens lifespan in C. elegans, demonstrating that formation of this chaperone-independent complex is important in animals. We propose that CHIC2 attenuates CHIP activity at the membrane, offering a novel mechanism by which this ubiquitin ligase can be regulated.\nStructural insights into complex I deficiency and assembly from the disease-related ndufs4-/- mouse\nAuthors: Yin, Z.; Agip, A.-N. A.; Bridges, H.; Hirst, J.\nScore: 5.6, Published: 2023-07-17 DOI: 10.1101/2023.07.17.549284\nRespiratory complex I (NADH:ubiquinone oxidoreductase) is essential for cellular energy production and NAD+ homeostasis. Complex I mutations cause neuromuscular, mitochondrial diseases, such as Leigh Syndrome, but their molecular-level consequences remain poorly understood. Here, we use a popular complex I-linked mitochondrial disease model, the ndufs4-/- mouse, to define the structural, biochemical and functional consequences of the absence of subunit NDUFS4. Cryo-EM analyses of mouse-heart ndufs4-/- complex I revealed a loose association of the NADH-dehydrogenase module, and discrete classes containing either assembly factor NDUFAF2 or subunit NDUFS6. Subunit NDUFA12 (that replaces its paralogue NDUFAF2 in mature complex I) is absent from all classes, compounding the deletion of NDUFS4 and preventing maturation of an NDUFS4-free but otherwise complete enzyme. We propose NDUFAF2 as the recruiter of the NADH-dehydrogenase module during assembly of the complex. Our results provide new molecular-level understanding of the ndufs4-/- mouse model and complex I-linked mitochondrial disease.\nMapping the architecture of the initiating phosphoglycosyl transferase from S. enterica O-antigen biosynthesis in a liponanoparticle.\nAuthors: Dodge, G. J.; Anderson, A. J.; He, Y.; Liu, W.; Viner, R.; Imperiali, B.\nScore: 12.6, Published: 2023-07-18 DOI: 10.1101/2023.06.16.545297\nBacterial cell surface glycoconjugates are critical for cell survival and for interactions between bacteria and their hosts. Consequently, the pathways responsible for their biosynthesis have untapped potential as therapeutic targets. The localization of many glycoconjugate biosynthesis enzymes to the membrane represents a significant challenge for expressing, purifying, and characterizing these enzymes. Here, we leverage cutting-edge methods to stabilize, purify, and structurally characterize WbaP, a phosphoglycosyl transferase (PGT) from Salmonella enterica (LT2) O-antigen biosynthesis without detergent solubilization from the lipid bilayer. From a functional perspective, these studies establish WbaP as a homodimer, reveal the structural elements responsible for oligomerization, shed light on the regulatory role of a domain of unknown function embedded within WbaP, and identify conserved structural motifs between PGTs and functionally unrelated UDP-sugar dehydratases. From a technological perspective, the strategy developed here is generalizable and provides a toolkit for studying small membrane proteins embedded in liponanoparticles beyond PGTs.\nStrategy of selection and optimization of single domain antibodies targeting the PHF6 linear peptide within the Tau intrinsically disordered protein\nAuthors: Mortelecque, J.; Zejneli, O.; Begard, S.; Nguyen, M.; Cantrelle, F.-X.; Hanoulle, X.; Rain, J.-C.; Colin, M.; Buee, L.; Landrieu, I.; Danis, C.; Dupre, E.\nScore: 4.5, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549252\nThe use of VHHs (Variable domain of the Heavy-chain of the Heavy-chain-only antibodies) as disease-modifying biomolecules in neurodegenerative disorders holds promises including to target aggregation-sensitive proteins. Exploitation of their clinical values dependents however on the capacity to deliver VHHs with optimal physico-chemical properties for their specific context of use. We described previously a VHH with high therapeutic potential in a family of neurodegenerative diseases called tauopathies. The activity of this promising parent VHH named Z70 relies on its binding within the central region of the Tau protein. Accordingly, we carried out random mutagenesis followed by yeast two-hybrid screening to obtain optimized variants. The VHHs selected from this initial screen targeted the same epitope as VHH Z70 as shown using nuclear magnetic resonance spectroscopy and had indeed improved binding affinities according to dissociation constant values obtained by surface plasmon resonance spectroscopy. The improved affinities can be partially rationalized based on three-dimensional structures of three complexes consisting of an optimized VHH and a peptide containing the Tau epitope. Interestingly, the ability of the VHH variants to inhibit Tau aggregation and seeding could not be predicted from their affinity alone. We indeed showed that the in vitro and in cellulo VHH stabilities are other limiting key factors to their efficacy. Our results demonstrate that only a complete pipeline of experiments, here described, permits a rational selection of optimized VHH variants, resulting in our capacity to propose two VHH variants derived from the parent Z70 for their next development steps.\nDirect, quantitative, and comprehensive analysis of tRNA acylation using intact tRNA liquid-chromatography mass-spectrometry\nAuthors: Fricke, R.; Knudson, I.; Schepartz, A.\nScore: 5.7, Published: 2023-07-14 DOI: 10.1101/2023.07.14.549096\nAminoacyl-tRNA synthetases (aaRSs) provide the functional and essential link between the sequence of an mRNA and the protein it encodes. aaRS enzymes catalyze a two-step chemical reaction that acylates specific tRNAs with a cognate -amino acid. In addition to their role in translation, acylated tRNAs contribute to non-ribosomal natural product biosynthesis and are implicated in multiple human diseases. From the standpoint of synthetic biology, the acylation of tRNAs with a non-canonical -amino acid (ncAA) or more recently, a non--amino acid monomer (nAA) is a critical first step in the incorporation of these monomers into proteins, where they can be used for fundamental and applied science. These endeavors all demand an understanding of aaRS activity and specificity. Although a number of methods to monitor aaRS function in vitro or in vivo have been developed, many evaluate only the first step of the two-step reaction, require the use of radioactivity, or are slow, difficult to generalize, or both. Here we describe an LC-MS assay that rapidly, quantitatively, and directly monitors aaRS activity by detecting the intact acyl-tRNA product. After a simple tRNA acylation reaction workup, acyl- and non-acyl-tRNA molecules are resolved using ion-pairing reverse phase chromatography and their exact masses are determined using high-resolution time-of-flight mass spectrometry. The intact tRNA assay we describe is fast, simple, and quantifies reaction yields as low as 0.23%. The assay can also be employed on tRNAs acylated with flexizyme to detect products that are undetectable using standard techniques. The protocol requires basic expertise in molecular biology, mass spectrometry, and RNAse-free techniques.\n",
  "wordCount" : "2181",
  "inLanguage": "en",
  "datePublished": "2023-07-23T10:36:34Z",
  "dateModified": "2023-07-23T10:36:34Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta"><span>updated on July 23, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549470">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549470" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549470">
        <p class="paperTitle">A systems-scale, integrated view of the ubiquitylation site occupancy and dynamics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549470" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549470" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Prus, G.; Satpathy, S.; Weinert, B.; Narita, T.; Choudhary, C.</p>
        <p class="info">Score: 32.7, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549470' target='https://doi.org/10.1101/2023.07.19.549470'> 10.1101/2023.07.19.549470</a></p>
        <p class="abstract">Ubiquitylation regulates virtually all proteins and biological processes in a cell. However, the global site-specific occupancy (stoichiometry) and turnover rate of ubiquitylation have never been quantified. Here, we present the first integrated picture of ubiquitylation site occupancy and half-life. Ubiquitylation occupancy spans four orders of magnitude, but the median ubiquitylation site occupancy is three orders of magnitude lower than that of phosphorylation. The occupancy, turnover rate, and the regulation of sites by proteasome inhibitors show strong interrelationships. These properties can discriminate signaling-relevant sites from the sites involved in proteasomal degradation. The sites strongly upregulated by proteasome inhibitors have a longer half-life, and the half-life increases with increasing protein length. Importantly, a previously unknown surveillance mechanism rapidly deubiquitylates all ubiquitin-specific E1 and E2 enzymes and protects them against bystander ubiquitylation accumulation. This work reveals general principles of ubiquitylation-dependent governance and offers conceptual insights into the dynamic regulation of the cell.

HighlightsO_LIUbiquitylation site occupancy is 3 orders of magnitude lower than phosphorylation
C_LIO_LIThe highest 80% and the lowest 20% occupancy sites have distinct properties
C_LIO_LIHigh occupancy sites are concentrated in the cytoplasmic domains of SLC proteins
C_LIO_LIA dedicated mechanism prevevents ubiquitylation accumulation in E1s and E2s
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549963">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549963" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549963">
        <p class="paperTitle">Coenzyme Q4 is a functional substitute for coenzyme Q10 and can be targeted to the mitochondria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549963" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549963" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Steenberge, L. H.; Sung, A. Y.; Fan, J.; Pagliarini, D. J.</p>
        <p class="info">Score: 16.6, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549963' target='https://doi.org/10.1101/2023.07.20.549963'> 10.1101/2023.07.20.549963</a></p>
        <p class="abstract">Coenzyme Q10 (CoQ10) is an important cofactor and antioxidant for numerous cellular processes, and its deficiency has been linked to human disorders including mitochondrial disease, heart failure, Parkinsons disease, and hypertension. Unfortunately, treatment with exogenous oral CoQ10 is often ineffective, likely due to the extreme hydrophobicity and high molecular weight of CoQ10. Here, we show that less hydrophobic CoQ species with shorter isoprenoid tails can serve as viable substitutes for CoQ10 in human cells. We demonstrate that CoQ4 can perform multiple functions of CoQ10 in CoQ-deficient cells at markedly lower treatment concentrations, motivating further investigation of CoQ4 as a supplement for CoQ10 deficiencies. In addition, we describe the synthesis and evaluation of an initial set of compounds designed to target CoQ4 selectively to mitochondria using triphenylphosphonium (TPP). Our results indicate that select versions of these compounds can successfully be delivered to mitochondria in a cell model and be cleaved to produce CoQ4, laying the groundwork for further development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.15.549150">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.15.549150" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.15.549150">
        <p class="paperTitle">A bespoke analytical workflow for the confident identification of sulfopeptides and their discrimination from phosphopeptides</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.15.549150" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.15.549150" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Daly, L. A.; Byrne, D. P.; Perkins, S.; Brownridge, P. J.; McDonnell, E.; Jones, A. R.; Eyers, P. A.; Eyers, C. E.</p>
        <p class="info">Score: 11.3, Published: 2023-07-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.15.549150' target='https://doi.org/10.1101/2023.07.15.549150'> 10.1101/2023.07.15.549150</a></p>
        <p class="abstract">Protein tyrosine sulfation (sY) is a post-translational modification (PTM) catalysed by Golgi-resident Tyrosyl Protein SulfoTransferases (TPSTs). Information on protein tyrosine sulfation is currently limited to [~]50 human proteins with only a handful having verified sites of sulfation. The contribution of this chemical moiety for the regulation of biological processes, both inside and outside the cell, remains poorly defined, in large part due to analytical limitations. Mass spectrometry-based proteomics is the method of choice for PTM analysis, but has yet to be applied for the systematic investigation and large-scale analysis of biomolecular sulfation (constituting the  sulfome), primarily due to issues associated with discrimination of sY-from phosphotyrosine (pY)-containing peptides. In this study, we developed a mass spectrometry (MS)-based workflow centred on the characterization of sY-peptides, incorporating optimised Zr4&#43;-IMAC and TiO2 enrichment strategies. Extensive characterization of a panel of sY- and pY-peptides using an array of MS fragmentation regimes (CID, HCD, EThcC, ETciD, UVPD) highlights differences in the ability to generate site-determining product ions, which can be exploited to differentiate sulfated peptides from nominally isobaric phosphopeptides based on precursor ion neutral loss at low collision energy. Application of our analytical workflow to a HEK-293 cell extracellular secretome facilitated identification of 21 new sulfotyrosine-containing proteins, several of which we validate enzymatically using in vitro sulfation assays. This study demonstrates the applicability of our strategy for confident, high-throughput,  sulfomics studies, and reveals new sY interplay between enzymes relevant to both protein and glycan sulfation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549678">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549678" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549678">
        <p class="paperTitle">Enhancing Inter-link Coverage in Cross-Linking Mass Spectrometry through Context-Sensitive Subgrouping and Decoy Fusion</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549678" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549678" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bogdanow, B.; Wang, C.; Ruwolt, M.; Ruta, J.; Mühlberg, L.; Zeng, W.-F.; Elofsson, A.; Liu, F.</p>
        <p class="info">Score: 9.2, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549678' target='https://doi.org/10.1101/2023.07.19.549678'> 10.1101/2023.07.19.549678</a></p>
        <p class="abstract">In cross-linking mass spectrometry, sensitivity and specificity in assigning mass spectra to cross-links between different proteins (inter-links) remains challenging. Here, we report on limitations of commonly used concatenated target-decoy searches and propose a target-decoy competition strategy on a fused database as a solution. Further, we capitalize on context-divergent error rates by implementing a novel context-sensitive subgrouping strategy. This approach increases inter-link coverage by ~ 30 - 75 % across XL-MS datasets, maintains low error rates, and preserves structural accuracy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549248">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549248" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549248">
        <p class="paperTitle">FAIMS-enabled N-terminomics analysis reveals novel legumain substrates in murine spleen</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549248" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549248" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ziegler, A. R.; Dufour, A.; Scott, N. E.; Edgington-Mitchell, L. E.</p>
        <p class="info">Score: 8.9, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549248' target='https://doi.org/10.1101/2023.07.18.549248'> 10.1101/2023.07.18.549248</a></p>
        <p class="abstract">Aberrant levels of the asparaginyl endopeptidase legumain have been linked to inflammation, neurodegeneration and cancer, yet our understanding of this protease is incomplete. Systematic attempts to identify legumain substrates have previously been confined to in vitro studies, which fail to mirror physiological conditions and obscure biologically relevant cleavage events. Using high-field asymmetric waveform ion mobility spectrometry (FAIMS), we developed a sensitive and streamlined approach for proteome and N-terminome analyses in a single analytical method without the need for N-termini enrichment. Compared to unfractionated proteomic analysis, we demonstrate FAIMS fractionation improves neo-N- termini identification by &gt;2.5 fold, resulting in identification of &gt;2,882 unique neo-N-termini from limited sample amounts. Within murine spleens, this approach identifies 6,366 proteins and 2,528 unique neo-N-termini, with 235 cleavage events enriched in wild-type compared to legumain-deficient spleens. Among these, 119 neo-N-termini arose from asparaginyl endopeptidase activities, representing novel putative physiological legumain substrates. The direct cleavage of selected substrates by legumain was confirmed using in vitro assays, providing support for the existence of physiologically relevant extra-lysosomal legumain activity. Combined, these data shed critical light on the functions of legumain and demonstrates the utility of FAIMS as an accessible method to improve depth and quality of N- terminomics studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549407">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549407" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549407">
        <p class="paperTitle">Interaction with the membrane-anchored protein CHIC2 constrains the ubiquitin ligase activity of CHIP</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549407" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549407" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Callahan, M. D.; Hodul, M.; Carroll, E. C.; Ravalin, M.; Nadel, C. M.; de Silva, A. R.; Cupo, A. R.; McDermott, L. E.; Nix, J. C.; Page, R. C.; Kao, A. W.; Gestwicki, J. E.</p>
        <p class="info">Score: 8.3, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549407' target='https://doi.org/10.1101/2023.07.17.549407'> 10.1101/2023.07.17.549407</a></p>
        <p class="abstract">Maintenance of cellular health requires the proper regulation of E3 ubiquitin ligases. The E3 ligase CHIP is canonically regulated by its interactions with the molecular chaperones Hsp70 and Hsp90, which focus CHIPs ubiquitination activity on misfolded proteins. Here, we report a chaperone-independent interaction of CHIP with the membrane-anchored protein CHIC2, which strongly attenuates CHIPs ligase activity. We show that CHIC2 outcompetes abundant, cytosolic chaperones through its exquisite CHIP selectivity, rather than through enhanced affinity. In proteomic experiments, we find that CHIC2 knockout phenocopies CHIP knockout in certain cell types, implying that chaperone-independent interactions can sometimes predominate CHIPs functions. Furthermore, loss of the CHIP-CHIC2 interaction induces neurodegeneration and shortens lifespan in C. elegans, demonstrating that formation of this chaperone-independent complex is important in animals. We propose that CHIC2 attenuates CHIP activity at the membrane, offering a novel mechanism by which this ubiquitin ligase can be regulated.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549284">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549284" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549284">
        <p class="paperTitle">Structural insights into complex I deficiency and assembly from the disease-related ndufs4-/- mouse</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549284" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549284" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yin, Z.; Agip, A.-N. A.; Bridges, H.; Hirst, J.</p>
        <p class="info">Score: 5.6, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549284' target='https://doi.org/10.1101/2023.07.17.549284'> 10.1101/2023.07.17.549284</a></p>
        <p class="abstract">Respiratory complex I (NADH:ubiquinone oxidoreductase) is essential for cellular energy production and NAD&#43; homeostasis. Complex I mutations cause neuromuscular, mitochondrial diseases, such as Leigh Syndrome, but their molecular-level consequences remain poorly understood. Here, we use a popular complex I-linked mitochondrial disease model, the ndufs4-/- mouse, to define the structural, biochemical and functional consequences of the absence of subunit NDUFS4. Cryo-EM analyses of mouse-heart ndufs4-/- complex I revealed a loose association of the NADH-dehydrogenase module, and discrete classes containing either assembly factor NDUFAF2 or subunit NDUFS6. Subunit NDUFA12 (that replaces its paralogue NDUFAF2 in mature complex I) is absent from all classes, compounding the deletion of NDUFS4 and preventing maturation of an NDUFS4-free but otherwise complete enzyme. We propose NDUFAF2 as the recruiter of the NADH-dehydrogenase module during assembly of the complex. Our results provide new molecular-level understanding of the ndufs4-/- mouse model and complex I-linked mitochondrial disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.16.545297">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.16.545297" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.16.545297">
        <p class="paperTitle">Mapping the architecture of the initiating phosphoglycosyl transferase from S. enterica O-antigen biosynthesis in a liponanoparticle.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.16.545297" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.16.545297" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dodge, G. J.; Anderson, A. J.; He, Y.; Liu, W.; Viner, R.; Imperiali, B.</p>
        <p class="info">Score: 12.6, Published: 2023-07-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.16.545297' target='https://doi.org/10.1101/2023.06.16.545297'> 10.1101/2023.06.16.545297</a></p>
        <p class="abstract">Bacterial cell surface glycoconjugates are critical for cell survival and for interactions between bacteria and their hosts. Consequently, the pathways responsible for their biosynthesis have untapped potential as therapeutic targets. The localization of many glycoconjugate biosynthesis enzymes to the membrane represents a significant challenge for expressing, purifying, and characterizing these enzymes. Here, we leverage cutting-edge methods to stabilize, purify, and structurally characterize WbaP, a phosphoglycosyl transferase (PGT) from Salmonella enterica (LT2) O-antigen biosynthesis without detergent solubilization from the lipid bilayer. From a functional perspective, these studies establish WbaP as a homodimer, reveal the structural elements responsible for oligomerization, shed light on the regulatory role of a domain of unknown function embedded within WbaP, and identify conserved structural motifs between PGTs and functionally unrelated UDP-sugar dehydratases. From a technological perspective, the strategy developed here is generalizable and provides a toolkit for studying small membrane proteins embedded in liponanoparticles beyond PGTs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549252">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549252" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549252">
        <p class="paperTitle">Strategy of selection and optimization of single domain antibodies targeting the PHF6 linear peptide within the Tau intrinsically disordered protein</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549252" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549252" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mortelecque, J.; Zejneli, O.; Begard, S.; Nguyen, M.; Cantrelle, F.-X.; Hanoulle, X.; Rain, J.-C.; Colin, M.; Buee, L.; Landrieu, I.; Danis, C.; Dupre, E.</p>
        <p class="info">Score: 4.5, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549252' target='https://doi.org/10.1101/2023.07.18.549252'> 10.1101/2023.07.18.549252</a></p>
        <p class="abstract">The use of VHHs (Variable domain of the Heavy-chain of the Heavy-chain-only antibodies) as disease-modifying biomolecules in neurodegenerative disorders holds promises including to target aggregation-sensitive proteins. Exploitation of their clinical values dependents however on the capacity to deliver VHHs with optimal physico-chemical properties for their specific context of use. We described previously a VHH with high therapeutic potential in a family of neurodegenerative diseases called tauopathies. The activity of this promising parent VHH named Z70 relies on its binding within the central region of the Tau protein. Accordingly, we carried out random mutagenesis followed by yeast two-hybrid screening to obtain optimized variants. The VHHs selected from this initial screen targeted the same epitope as VHH Z70 as shown using nuclear magnetic resonance spectroscopy and had indeed improved binding affinities according to dissociation constant values obtained by surface plasmon resonance spectroscopy. The improved affinities can be partially rationalized based on three-dimensional structures of three complexes consisting of an optimized VHH and a peptide containing the Tau epitope. Interestingly, the ability of the VHH variants to inhibit Tau aggregation and seeding could not be predicted from their affinity alone. We indeed showed that the in vitro and in cellulo VHH stabilities are other limiting key factors to their efficacy. Our results demonstrate that only a complete pipeline of experiments, here described, permits a rational selection of optimized VHH variants, resulting in our capacity to propose two VHH variants derived from the parent Z70 for their next development steps.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.549096">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.549096" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.549096">
        <p class="paperTitle">Direct, quantitative, and comprehensive analysis of tRNA acylation using intact tRNA liquid-chromatography mass-spectrometry</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.549096" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.549096" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fricke, R.; Knudson, I.; Schepartz, A.</p>
        <p class="info">Score: 5.7, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.549096' target='https://doi.org/10.1101/2023.07.14.549096'> 10.1101/2023.07.14.549096</a></p>
        <p class="abstract">Aminoacyl-tRNA synthetases (aaRSs) provide the functional and essential link between the sequence of an mRNA and the protein it encodes. aaRS enzymes catalyze a two-step chemical reaction that acylates specific tRNAs with a cognate -amino acid. In addition to their role in translation, acylated tRNAs contribute to non-ribosomal natural product biosynthesis and are implicated in multiple human diseases. From the standpoint of synthetic biology, the acylation of tRNAs with a non-canonical -amino acid (ncAA) or more recently, a non--amino acid monomer (nAA) is a critical first step in the incorporation of these monomers into proteins, where they can be used for fundamental and applied science. These endeavors all demand an understanding of aaRS activity and specificity. Although a number of methods to monitor aaRS function in vitro or in vivo have been developed, many evaluate only the first step of the two-step reaction, require the use of radioactivity, or are slow, difficult to generalize, or both. Here we describe an LC-MS assay that rapidly, quantitatively, and directly monitors aaRS activity by detecting the intact acyl-tRNA product. After a simple tRNA acylation reaction workup, acyl- and non-acyl-tRNA molecules are resolved using ion-pairing reverse phase chromatography and their exact masses are determined using high-resolution time-of-flight mass spectrometry. The intact tRNA assay we describe is fast, simple, and quantifies reaction yields as low as 0.23%. The assay can also be employed on tRNAs acylated with flexizyme to detect products that are undetectable using standard techniques. The protocol requires basic expertise in molecular biology, mass spectrometry, and RNAse-free techniques.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
