<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="CK-666 and CK-869 differentially inhibit Arp2/3 iso-complexes
Authors: Cao, L.; Huang, S.; Basant, A.; Mladenov, M.; Way, M.
Score: 23.5, Published: 2023-11-26 DOI: 10.1101/2023.11.26.568719
The inhibitors, CK-666 and CK-869, are widely used to probe the function of actin nucleation by the Arp2/3 complex in vitro and in cells. However, in mammals, the Arp2/3 complex consists of 8 iso-complexes, as three of its subunits (Arp3, ArpC1, ArpC5) are encoded by two different genes.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="CK-666 and CK-869 differentially inhibit Arp2/3 iso-complexes
Authors: Cao, L.; Huang, S.; Basant, A.; Mladenov, M.; Way, M.
Score: 23.5, Published: 2023-11-26 DOI: 10.1101/2023.11.26.568719
The inhibitors, CK-666 and CK-869, are widely used to probe the function of actin nucleation by the Arp2/3 complex in vitro and in cells. However, in mammals, the Arp2/3 complex consists of 8 iso-complexes, as three of its subunits (Arp3, ArpC1, ArpC5) are encoded by two different genes." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-29T10:40:14+00:00" />
<meta property="article:modified_time" content="2023-11-29T10:40:14+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="CK-666 and CK-869 differentially inhibit Arp2/3 iso-complexes
Authors: Cao, L.; Huang, S.; Basant, A.; Mladenov, M.; Way, M.
Score: 23.5, Published: 2023-11-26 DOI: 10.1101/2023.11.26.568719
The inhibitors, CK-666 and CK-869, are widely used to probe the function of actin nucleation by the Arp2/3 complex in vitro and in cells. However, in mammals, the Arp2/3 complex consists of 8 iso-complexes, as three of its subunits (Arp3, ArpC1, ArpC5) are encoded by two different genes."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "CK-666 and CK-869 differentially inhibit Arp2/3 iso-complexes\nAuthors: Cao, L.; Huang, S.; Basant, A.; Mladenov, M.; Way, M.\nScore: 23.5, Published: 2023-11-26 DOI: 10.1101/2023.11.26.568719\nThe inhibitors, CK-666 and CK-869, are widely used to probe the function of actin nucleation by the Arp2/3 complex in vitro and in cells. However, in mammals, the Arp2/3 complex consists of 8 iso-complexes, as three of its subunits (Arp3, ArpC1, ArpC5) are encoded by two different genes.",
  "keywords": [
    
  ],
  "articleBody": " CK-666 and CK-869 differentially inhibit Arp2/3 iso-complexes\nAuthors: Cao, L.; Huang, S.; Basant, A.; Mladenov, M.; Way, M.\nScore: 23.5, Published: 2023-11-26 DOI: 10.1101/2023.11.26.568719\nThe inhibitors, CK-666 and CK-869, are widely used to probe the function of actin nucleation by the Arp2/3 complex in vitro and in cells. However, in mammals, the Arp2/3 complex consists of 8 iso-complexes, as three of its subunits (Arp3, ArpC1, ArpC5) are encoded by two different genes. Here, we used recombinant complexes with defined subunit composition to assess the activity of CK-666 and CK-869 against iso-complexes. We demonstrate that while both inhibitors prevent linear actin filament formation when either ArpC1A- or ArpC1B-containing complexes are activated by SPIN90, inhibition of actin branching depends on iso-complex composition. Both drugs inhibited actin branch formation by VCA-activated complexes containing ArpC1A, but only CK-869 can inhibit ArpC1B-containing complexes. Consistent with this, in bone marrow-derived macrophages which express very low levels of ArpC1A, CK-869 but not CK-666, impacted phagocytosis and migration. We also found that CK-869 is only able to inhibit Arp3-but not Arp3B-containing iso-complexes. Our analyses have important implications for the interpretation of results using CK-666 and CK-869, given that the relative expression levels of ArpC1 and Arp3 isoforms in cells and tissues remains largely unknown.\nHuman DDX6 regulates translation and decay of inefficiently translated mRNAs\nAuthors: Weber, R.; Wohlbold, L.; Chang, C.-T.\nScore: 18.2, Published: 2023-10-30 DOI: 10.1101/2023.10.30.564346\nRecent findings indicate that the translation elongation rate influences mRNA stability. One of the factors that has been implicated in this link between mRNA decay and translation speed is the yeast DEAD-box helicase Dhh1p. Here, we demonstrate that the human ortholog of Dhh1p, DDX6, triggers deadenylation-dependent decay of inefficiently translated mRNAs in human cells. DDX6 interacts with the ribosome through the Phe-Asp-Phe (FDF) motif in its RecA2 domain. Furthermore, RecA2-mediated interactions and ATPase activity are both required for DDX6 to destabilize inefficiently translated mRNAs. Using ribosome profiling and RNA sequencing, we identified two classes of endogenous mRNAs that are regulated in a DDX6-dependent manner. The identified targets are either translationally regulated or regulated at the steady-state-level and either exhibit signatures of poor overall translation or of locally reduced ribosome translocation rates. Transferring the identified sequence stretches into a reporter mRNA caused translation-and DDX6-dependent degradation of the reporter mRNA. In summary, these results identify DDX6 as a crucial regulator of mRNA translation and decay triggered by slow ribosome movement and provide insights into the mechanism by which DDX6 destabilizes inefficiently translated mRNAs.\nFast and accurate disulfide bridge detection\nAuthors: Heissel, S.; He, Y.; Jankevics, A.; Shi, Y.; Molina, H.; Viner, R.; Scheltema, R. A.\nScore: 10.4, Published: 2023-11-22 DOI: 10.1101/2023.11.22.568205\nRecombinant expression of proteins, propelled by therapeutic antibodies, has evolved into a multi-billion-dollar industry. Essential here is quality control assessment of critical attributes such as sequence fidelity, proper folding, and post-translational modifications (PTMs). Errors can lead to diminished bioactivity and, in the context of therapeutic proteins, an elevated risk for immunogenicity. Over the years, many techniques were developed and applied to validate proteins in a standardized and high-throughput fashion. One parameter has, however, so far been challenging to assess. Disulfide bridges, covalent bonds linking two Cysteine residues, assist in the correct folding and stability of proteins and thus have a major influence on their efficacy. Mass spectrometry promises to be an optimal technique to uncover them in a fast and accurate fashion. In this work, we present a unique combination of sample preparation, data acquisition and analysis facilitating the rapid and accurate assessment of disulfide bridges in purified proteins. Through microwave-assisted acid hydrolysis (MAAH), the proteins are digested rapidly and artifact-free into peptides, with a substantial degree of overlap over the sequence. The nonspecific nature of this procedure, however, introduces chemical background which is efficiently removed by integrating ion mobility preceding the mass spectrometric measurement. The nonspecific nature of the digestion step additionally necessitates new developments in data analysis, for which we extended the XlinkX node in Proteome Discoverer (XlinkX/PD) to efficiently process the data and ensure correctness through effective false discovery rate correction. The entire workflow can be completed within one hour, allowing for high-throughput, high-accuracy disulfide mapping.\nStructure of the human 80S ribosome at 1.9 A resolution - the molecular role of chemical modifications and ions in RNA\nAuthors: Holvec, S.; Barchet, C.; Lechner, A.; Frechin, L.; De Silva, S. N. T.; Hazemann, I.; Wolff, P.; von Loeffelholz, O.; Klaholz, B. P.\nScore: 10.2, Published: 2023-11-24 DOI: 10.1101/2023.11.23.568509\nThe ribosomal RNA of the human protein synthesis machinery comprises numerous chemical modifications that are introduced during ribosome biogenesis. We present the 1.9 [A] resolution cryo-EM structure of the 80S human ribosome resolving numerous new rRNA modifications and functionally important ions such as Zn2+, K+ and Mg2+ including their associated individual water molecules. 2-O-methylation, pseudo-uridine and base modifications were confirmed by mass spectrometry resulting in a complete investigation of the \u003e 230 sites many of which could not be addressed previously. They choreograph key interactions within the RNA and at the interface with proteins, including at the ribosomal subunit interfaces of the fully assembled 80S ribosome. Uridine isomerisation turns out to be a key mechanism for U-A base pair stabilisation in RNA in general. The structural environment of chemical modifications \u0026 ions is primordial for the RNA architecture of the mature human ribosome, hence providing a structural framework to address their role in healthy context and human diseases.\nStructure of the γ-tubulin ring complex-capped microtubule\nAuthors: Aher, A.; Urnavicius, L.; Xue, A.; Neselu, K.; Kapoor, T.\nScore: 16.4, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567916\nMicrotubules are composed of /{beta}-tubulin dimers positioned head-to-tail to form protofilaments that associate laterally in varying numbers. It is not known how cellular microtubules assemble with the canonical 13-protofilament architecture, resulting in micrometer-scale /{beta}-tubulin tracks for intracellular transport that align with, rather than spiral along, the filaments long-axis. We report that the human [~]2.3MDa {gamma}-tubulin ring complex ({gamma}-TuRC), an essential regulator of microtubule formation that contains 14 {gamma}-tubulins, selectively nucleates 13-protofilament microtubules. Cryo-EM reconstructions of {gamma}-TuRC-capped microtubule minus-ends reveal the extensive intra- and inter-domain motions of {gamma}-TuRC subunits that accommodate its actin-containing luminal bridge and establish lateral and longitudinal interactions between {gamma}- and -tubulins. Our structures reveal how free {gamma}-TuRC, an inefficient nucleation template due to its splayed conformation, transforms into a stable cap that blocks addition or loss of /{beta}-tubulins from minus-ends and sets the lattice architecture of cellular microtubules. One Sentence SummaryStructural insights into how the {gamma}-tubulin ring complex nucleates and caps a 13-protofilament microtubule.\nBioorthogonal metabolic labeling of the virulence factor phenolic glycolipid in mycobacteria\nAuthors: Guzman, L. E.; Cambier, C.; Cheng, T.-Y.; Naqvi, K. F.; Shiloh, M. U.; Moody, D. B.; Bertozzi, C.\nScore: 6.0, Published: 2023-11-28 DOI: 10.1101/2023.11.28.569059\nSurface lipids on pathogenic mycobacteria modulate infection outcomes by regulating host immune responses. Phenolic glycolipid (PGL) is a host-modulating surface lipid that varies among clinical Mycobacterium tuberculosis strains. PGL is also found in Mycobacterium marinum where it promotes infection of zebrafish through effects on the innate immune system. Given the important role this lipid plays in the host-pathogen relationship, tools for profiling its abundance, spatial distribution, and dynamics are needed. Here we report a strategy for imaging PGL in live mycobacteria using bioorthogonal metabolic labeling. We functionalized the PGL precursor p-hydroxybenzoic acid (pHB) with an azide group (3-azido pHB). When fed to mycobacteria, 3-azido pHB was incorporated into the cell surface, which could then be visualized via bioorthogonal conjugation of a fluorescent probe. We confirmed that 3-azido pHB incorporates into PGL using mass spectrometry methods and demonstrated selectivity for PGL-producing Mycobacterium marinum and Mycobacterium tuberculosis strains. Finally, we applied this metabolic labeling strategy to study the dynamics of PGL within the mycobacterial membrane. This new tool enables visualization of PGL which may facilitate studies of mycobacterial pathogenesis.\nFluorescent human RPA to track assembly dynamics on DNA\nAuthors: Kaushik, V.; Chadda, R.; Kuppa, S.; Pokhrel, N.; Vayyeti, A.; Grady, S.; Arnatt, C.; Antony, E.\nScore: 4.1, Published: 2023-11-23 DOI: 10.1101/2023.11.23.568455\nDNA metabolic processes including replication, repair, recombination, and telomere maintenance occur on single-stranded DNA (ssDNA). In each of these complex processes, dozens of proteins function together on the ssDNA template. However, when double-stranded DNA is unwound, the transiently open ssDNA is protected and coated by the high affinity heterotrimeric ssDNA binding Replication Protein A (RPA). Almost all downstream DNA processes must first remodel/remove RPA or function alongside to access the ssDNA occluded under RPA. Formation of RPA-ssDNA complexes trigger the DNA damage checkpoint response and is a key step in activating most DNA repair and recombination pathways. Thus, in addition to protecting the exposed ssDNA, RPA functions as a gatekeeper to define functional specificity in DNA maintenance and genomic integrity. RPA achieves functional dexterity through a multi-domain architecture utilizing several DNA binding and protein-interaction domains connected by flexible linkers. This flexible and modular architecture enables RPA to adopt a myriad of configurations tailored for specific DNA metabolic roles. To experimentally capture the dynamics of the domains of RPA upon binding to ssDNA and interacting proteins we here describe the generation of active site-specific fluorescent versions of human RPA (RPA) using 4-azido-L-phenylalanine (4AZP) incorporation and click chemistry. This approach can also be applied to site-specific modifications of other multi-domain proteins. Fluorescence-enhancement through non-canonical amino acids (FEncAA) and Forster Resonance Energy Transfer (FRET) assays for measuring dynamics of RPA on DNA are also described. HighlightsO_LIRPA is an essential protein for most DNA metabolic processes including replication, repair, and recombination. C_LIO_LIRPA is a ssDNA binding protein made of six domains situated across RPA70, RPA32 and RPA14 subunits. Four high affinity DNA binding domains engage the DNA. C_LIO_LISite-specific fluorescent probes were incorporated into two domains of RPA and report on ssDNA binding dynamics. C_LIO_LIBulk-level kinetic and single-molecule assays are described to monitor the binding and remodeling of individual RPA domains on ssDNA. C_LI\nExpanding the GPCR-RAMP interactome\nAuthors: Kotliar, I. B.; Bendes, A.; Dahl, L.; Chen, Y.; Saarinen, M.; Ceraudo, E.; Dodig, T.; Uhlen, M.; Svenningsson, P.; Schwenk, J.; Sakmar, T.\nScore: 4.1, Published: 2023-11-23 DOI: 10.1101/2023.11.22.568247\nReceptor activity-modifying proteins (RAMPs) can form complexes with G protein-coupled receptors (GPCRs) and regulate their cellular trafficking and pharmacology. RAMP interactions have been identified for about 50 GPCRs, but only a few GPCR-RAMP complexes have been studied in detail. To elucidate a complete interactome between GPCRs and the three RAMPs, we developed a customized library of 215 Dual Epitope-Tagged (DuET) GPCRs representing all GPCR subfamilies. Using a multiplexed suspension bead array (SBA) assay, we identified 122 GPCRs that showed strong evidence for interaction with at least one RAMP. We screened for native interactions in three cell lines and found 23 GPCRs that formed complexes with RAMPs. Mapping the GPCR-RAMP interactome expands the current system-wide functional characterization of RAMP-interacting GPCRs to inform the design of selective GPCR-targeted therapeutics. One-Sentence SummaryNovel complexes between G protein-coupled receptors and interacting proteins point to a system-wide regulation of GPCR function.\nSub-single-turnover quantification of enzyme catalysis at ultrahigh throughput via a versatile NAD(P)H coupled assay in microdroplets\nAuthors: Penner, M.; Klein, O. J.; Gantz, M.; Boss, S.; Barker, P.; Dupree, P.; Hollfelder, F.\nScore: 4.0, Published: 2023-11-22 DOI: 10.1101/2023.11.22.568356\nEnzyme engineering and discovery are crucial for a future sustainable bioeconomy, and harvesting new biocatalysts from large libraries through directed evolution or functional metagenomics requires accessible, rapid assays. Ultra-high throughput screening can often require an optical readout, leading to the use of model substrates that may not accurately report on activity for the target reaction and may require bespoke synthesis. In contrast, coupled assays represent a modular plug-and-play system, where any pairing of enzyme/substrate may be investigated, if the reaction can produce a common intermediate which links the catalytic reaction to a detection cascade readout. Here we establish a detection cascade, producing a fluorescent readout in response to NAD(P)H via glutathione reductase and a subsequent thiol-mediated uncaging reaction, with a 30 nM detection limit. We demonstrate its utility for the glycosidase AxyAgu115A (producing monosaccharides from a natural biofuel feedstock) and report a three orders of magnitude improved sensitivity compared to absorbance-based systems, so that less than one catalytic turnover per enzyme molecule expressed from a single cell is detectable. These advantages are brought to bear in plate formats, but also in picoliter emulsion droplets, where enrichments of 950-fold suggest that large libraries can be interrogated against a specific query substrate.\nPhytopathogenic cyclic glucohexadecaose from an inverting transglycosylase\nAuthors: Motouchi, S.; Komba, S.; Nakai, H.; Nakajima, M.\nScore: 3.8, Published: 2023-11-27 DOI: 10.1101/2023.11.26.568766\nXanthomonas species contain numerous notoriously well-known plant pathogens. Among various pathogenic factors, the role of -1,6-cyclized {beta}-1,2-glucohexadecaose (C{beta}G16) produced by Xanthomonas campestris pv. campestris was shown previously to be vital for infecting model organisms Arabidopsis thaliana and Nicotiana benthamiana. However, enzymes responsible for biosynthesising C{beta}G16 are essentially unknown, which limits the generation of agrichemicals that inhibit C{beta}G16 synthesis. In this study, we discovered that OpgD from X. campestris pv. campestris converts linear {beta}-1,2-glucan to C{beta}G16. Structural and functional analyses revealed that OpgD from X. campestris pv. campestris possesses an anomer-inverting transglycosylation mechanism, which is unprecedented among carbohydrate-active enzymes. The discovery of this unprecedented glucan-generating mechanism reveals a new foundation for the enzymatic synthesis of carbohydrates. Furthermore, identifying C{beta}G16 synthase highly conserved in Xanthomonas provides a broadly adaptable drug target for new-genre agrichemicals that overcome antimicrobial-resistant bacterial issues.\n",
  "wordCount" : "2207",
  "inLanguage": "en",
  "datePublished": "2023-11-29T10:40:14Z",
  "dateModified": "2023-11-29T10:40:14Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 29, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.26.568719">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.26.568719" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.26.568719">
        <p class="paperTitle">CK-666 and CK-869 differentially inhibit Arp2/3 iso-complexes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.26.568719" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.26.568719" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cao, L.; Huang, S.; Basant, A.; Mladenov, M.; Way, M.</p>
        <p class="info">Score: 23.5, Published: 2023-11-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.26.568719' target='https://doi.org/10.1101/2023.11.26.568719'> 10.1101/2023.11.26.568719</a></p>
        <p class="abstract">The inhibitors, CK-666 and CK-869, are widely used to probe the function of actin nucleation by the Arp2/3 complex in vitro and in cells. However, in mammals, the Arp2/3 complex consists of 8 iso-complexes, as three of its subunits (Arp3, ArpC1, ArpC5) are encoded by two different genes. Here, we used recombinant complexes with defined subunit composition to assess the activity of CK-666 and CK-869 against iso-complexes. We demonstrate that while both inhibitors prevent linear actin filament formation when either ArpC1A- or ArpC1B-containing complexes are activated by SPIN90, inhibition of actin branching depends on iso-complex composition. Both drugs inhibited actin branch formation by VCA-activated complexes containing ArpC1A, but only CK-869 can inhibit ArpC1B-containing complexes. Consistent with this, in bone marrow-derived macrophages which express very low levels of ArpC1A, CK-869 but not CK-666, impacted phagocytosis and migration. We also found that CK-869 is only able to inhibit Arp3-but not Arp3B-containing iso-complexes. Our analyses have important implications for the interpretation of results using CK-666 and CK-869, given that the relative expression levels of ArpC1 and Arp3 isoforms in cells and tissues remains largely unknown.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564346">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564346" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564346">
        <p class="paperTitle">Human DDX6 regulates translation and decay of inefficiently translated mRNAs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564346" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564346" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Weber, R.; Wohlbold, L.; Chang, C.-T.</p>
        <p class="info">Score: 18.2, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564346' target='https://doi.org/10.1101/2023.10.30.564346'> 10.1101/2023.10.30.564346</a></p>
        <p class="abstract">Recent findings indicate that the translation elongation rate influences mRNA stability. One of the factors that has been implicated in this link between mRNA decay and translation speed is the yeast DEAD-box helicase Dhh1p. Here, we demonstrate that the human ortholog of Dhh1p, DDX6, triggers deadenylation-dependent decay of inefficiently translated mRNAs in human cells. DDX6 interacts with the ribosome through the Phe-Asp-Phe (FDF) motif in its RecA2 domain. Furthermore, RecA2-mediated interactions and ATPase activity are both required for DDX6 to destabilize inefficiently translated mRNAs. Using ribosome profiling and RNA sequencing, we identified two classes of endogenous mRNAs that are regulated in a DDX6-dependent manner. The identified targets are either translationally regulated or regulated at the steady-state-level and either exhibit signatures of poor overall translation or of locally reduced ribosome translocation rates. Transferring the identified sequence stretches into a reporter mRNA caused translation-and DDX6-dependent degradation of the reporter mRNA. In summary, these results identify DDX6 as a crucial regulator of mRNA translation and decay triggered by slow ribosome movement and provide insights into the mechanism by which DDX6 destabilizes inefficiently translated mRNAs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.568205">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.568205" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.568205">
        <p class="paperTitle">Fast and accurate disulfide bridge detection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.568205" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.568205" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Heissel, S.; He, Y.; Jankevics, A.; Shi, Y.; Molina, H.; Viner, R.; Scheltema, R. A.</p>
        <p class="info">Score: 10.4, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.568205' target='https://doi.org/10.1101/2023.11.22.568205'> 10.1101/2023.11.22.568205</a></p>
        <p class="abstract">Recombinant expression of proteins, propelled by therapeutic antibodies, has evolved into a multi-billion-dollar industry. Essential here is quality control assessment of critical attributes such as sequence fidelity, proper folding, and post-translational modifications (PTMs). Errors can lead to diminished bioactivity and, in the context of therapeutic proteins, an elevated risk for immunogenicity. Over the years, many techniques were developed and applied to validate proteins in a standardized and high-throughput fashion. One parameter has, however, so far been challenging to assess. Disulfide bridges, covalent bonds linking two Cysteine residues, assist in the correct folding and stability of proteins and thus have a major influence on their efficacy. Mass spectrometry promises to be an optimal technique to uncover them in a fast and accurate fashion. In this work, we present a unique combination of sample preparation, data acquisition and analysis facilitating the rapid and accurate assessment of disulfide bridges in purified proteins. Through microwave-assisted acid hydrolysis (MAAH), the proteins are digested rapidly and artifact-free into peptides, with a substantial degree of overlap over the sequence. The nonspecific nature of this procedure, however, introduces chemical background which is efficiently removed by integrating ion mobility preceding the mass spectrometric measurement. The nonspecific nature of the digestion step additionally necessitates new developments in data analysis, for which we extended the XlinkX node in Proteome Discoverer (XlinkX/PD) to efficiently process the data and ensure correctness through effective false discovery rate correction. The entire workflow can be completed within one hour, allowing for high-throughput, high-accuracy disulfide mapping.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.568509">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.568509" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.568509">
        <p class="paperTitle">Structure of the human 80S ribosome at 1.9 A resolution - the molecular role of chemical modifications and ions in RNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.568509" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.568509" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Holvec, S.; Barchet, C.; Lechner, A.; Frechin, L.; De Silva, S. N. T.; Hazemann, I.; Wolff, P.; von Loeffelholz, O.; Klaholz, B. P.</p>
        <p class="info">Score: 10.2, Published: 2023-11-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.568509' target='https://doi.org/10.1101/2023.11.23.568509'> 10.1101/2023.11.23.568509</a></p>
        <p class="abstract">The ribosomal RNA of the human protein synthesis machinery comprises numerous chemical modifications that are introduced during ribosome biogenesis. We present the 1.9 [A] resolution cryo-EM structure of the 80S human ribosome resolving numerous new rRNA modifications and functionally important ions such as Zn2&#43;, K&#43; and Mg2&#43; including their associated individual water molecules. 2-O-methylation, pseudo-uridine and base modifications were confirmed by mass spectrometry resulting in a complete investigation of the &gt; 230 sites many of which could not be addressed previously. They choreograph key interactions within the RNA and at the interface with proteins, including at the ribosomal subunit interfaces of the fully assembled 80S ribosome. Uridine isomerisation turns out to be a key mechanism for U-A base pair stabilisation in RNA in general. The structural environment of chemical modifications &amp; ions is primordial for the RNA architecture of the mature human ribosome, hence providing a structural framework to address their role in healthy context and human diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567916">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567916" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567916">
        <p class="paperTitle">Structure of the γ-tubulin ring complex-capped microtubule</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567916" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567916" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aher, A.; Urnavicius, L.; Xue, A.; Neselu, K.; Kapoor, T.</p>
        <p class="info">Score: 16.4, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567916' target='https://doi.org/10.1101/2023.11.20.567916'> 10.1101/2023.11.20.567916</a></p>
        <p class="abstract">Microtubules are composed of /{beta}-tubulin dimers positioned head-to-tail to form protofilaments that associate laterally in varying numbers. It is not known how cellular microtubules assemble with the canonical 13-protofilament architecture, resulting in micrometer-scale /{beta}-tubulin tracks for intracellular transport that align with, rather than spiral along, the filaments long-axis. We report that the human [~]2.3MDa {gamma}-tubulin ring complex ({gamma}-TuRC), an essential regulator of microtubule formation that contains 14 {gamma}-tubulins, selectively nucleates 13-protofilament microtubules. Cryo-EM reconstructions of {gamma}-TuRC-capped microtubule minus-ends reveal the extensive intra- and inter-domain motions of {gamma}-TuRC subunits that accommodate its actin-containing luminal bridge and establish lateral and longitudinal interactions between {gamma}- and -tubulins. Our structures reveal how free {gamma}-TuRC, an inefficient nucleation template due to its splayed conformation, transforms into a stable cap that blocks addition or loss of /{beta}-tubulins from minus-ends and sets the lattice architecture of cellular microtubules.

One Sentence SummaryStructural insights into how the {gamma}-tubulin ring complex nucleates and caps a 13-protofilament microtubule.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.569059">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.569059" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.569059">
        <p class="paperTitle">Bioorthogonal metabolic labeling of the virulence factor phenolic glycolipid in mycobacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.569059" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.569059" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guzman, L. E.; Cambier, C.; Cheng, T.-Y.; Naqvi, K. F.; Shiloh, M. U.; Moody, D. B.; Bertozzi, C.</p>
        <p class="info">Score: 6.0, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.569059' target='https://doi.org/10.1101/2023.11.28.569059'> 10.1101/2023.11.28.569059</a></p>
        <p class="abstract">Surface lipids on pathogenic mycobacteria modulate infection outcomes by regulating host immune responses. Phenolic glycolipid (PGL) is a host-modulating surface lipid that varies among clinical Mycobacterium tuberculosis strains. PGL is also found in Mycobacterium marinum where it promotes infection of zebrafish through effects on the innate immune system. Given the important role this lipid plays in the host-pathogen relationship, tools for profiling its abundance, spatial distribution, and dynamics are needed. Here we report a strategy for imaging PGL in live mycobacteria using bioorthogonal metabolic labeling. We functionalized the PGL precursor p-hydroxybenzoic acid (pHB) with an azide group (3-azido pHB). When fed to mycobacteria, 3-azido pHB was incorporated into the cell surface, which could then be visualized via bioorthogonal conjugation of a fluorescent probe. We confirmed that 3-azido pHB incorporates into PGL using mass spectrometry methods and demonstrated selectivity for PGL-producing Mycobacterium marinum and Mycobacterium tuberculosis strains. Finally, we applied this metabolic labeling strategy to study the dynamics of PGL within the mycobacterial membrane. This new tool enables visualization of PGL which may facilitate studies of mycobacterial pathogenesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.568455">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.568455" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.568455">
        <p class="paperTitle">Fluorescent human RPA to track assembly dynamics on DNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.568455" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.568455" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kaushik, V.; Chadda, R.; Kuppa, S.; Pokhrel, N.; Vayyeti, A.; Grady, S.; Arnatt, C.; Antony, E.</p>
        <p class="info">Score: 4.1, Published: 2023-11-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.568455' target='https://doi.org/10.1101/2023.11.23.568455'> 10.1101/2023.11.23.568455</a></p>
        <p class="abstract">DNA metabolic processes including replication, repair, recombination, and telomere maintenance occur on single-stranded DNA (ssDNA). In each of these complex processes, dozens of proteins function together on the ssDNA template. However, when double-stranded DNA is unwound, the transiently open ssDNA is protected and coated by the high affinity heterotrimeric ssDNA binding Replication Protein A (RPA). Almost all downstream DNA processes must first remodel/remove RPA or function alongside to access the ssDNA occluded under RPA. Formation of RPA-ssDNA complexes trigger the DNA damage checkpoint response and is a key step in activating most DNA repair and recombination pathways. Thus, in addition to protecting the exposed ssDNA, RPA functions as a gatekeeper to define functional specificity in DNA maintenance and genomic integrity. RPA achieves functional dexterity through a multi-domain architecture utilizing several DNA binding and protein-interaction domains connected by flexible linkers. This flexible and modular architecture enables RPA to adopt a myriad of configurations tailored for specific DNA metabolic roles. To experimentally capture the dynamics of the domains of RPA upon binding to ssDNA and interacting proteins we here describe the generation of active site-specific fluorescent versions of human RPA (RPA) using 4-azido-L-phenylalanine (4AZP) incorporation and click chemistry. This approach can also be applied to site-specific modifications of other multi-domain proteins. Fluorescence-enhancement through non-canonical amino acids (FEncAA) and Forster Resonance Energy Transfer (FRET) assays for measuring dynamics of RPA on DNA are also described.

HighlightsO_LIRPA is an essential protein for most DNA metabolic processes including replication, repair, and recombination.
C_LIO_LIRPA is a ssDNA binding protein made of six domains situated across RPA70, RPA32 and RPA14 subunits. Four high affinity DNA binding domains engage the DNA.
C_LIO_LISite-specific fluorescent probes were incorporated into two domains of RPA and report on ssDNA binding dynamics.
C_LIO_LIBulk-level kinetic and single-molecule assays are described to monitor the binding and remodeling of individual RPA domains on ssDNA.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.568247">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.568247" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.568247">
        <p class="paperTitle">Expanding the GPCR-RAMP interactome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.568247" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.568247" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kotliar, I. B.; Bendes, A.; Dahl, L.; Chen, Y.; Saarinen, M.; Ceraudo, E.; Dodig, T.; Uhlen, M.; Svenningsson, P.; Schwenk, J.; Sakmar, T.</p>
        <p class="info">Score: 4.1, Published: 2023-11-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.568247' target='https://doi.org/10.1101/2023.11.22.568247'> 10.1101/2023.11.22.568247</a></p>
        <p class="abstract">Receptor activity-modifying proteins (RAMPs) can form complexes with G protein-coupled receptors (GPCRs) and regulate their cellular trafficking and pharmacology. RAMP interactions have been identified for about 50 GPCRs, but only a few GPCR-RAMP complexes have been studied in detail. To elucidate a complete interactome between GPCRs and the three RAMPs, we developed a customized library of 215 Dual Epitope-Tagged (DuET) GPCRs representing all GPCR subfamilies. Using a multiplexed suspension bead array (SBA) assay, we identified 122 GPCRs that showed strong evidence for interaction with at least one RAMP. We screened for native interactions in three cell lines and found 23 GPCRs that formed complexes with RAMPs. Mapping the GPCR-RAMP interactome expands the current system-wide functional characterization of RAMP-interacting GPCRs to inform the design of selective GPCR-targeted therapeutics.

One-Sentence SummaryNovel complexes between G protein-coupled receptors and interacting proteins point to a system-wide regulation of GPCR function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.568356">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.568356" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.568356">
        <p class="paperTitle">Sub-single-turnover quantification of enzyme catalysis at ultrahigh throughput via a versatile NAD(P)H coupled assay in microdroplets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.568356" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.568356" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Penner, M.; Klein, O. J.; Gantz, M.; Boss, S.; Barker, P.; Dupree, P.; Hollfelder, F.</p>
        <p class="info">Score: 4.0, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.568356' target='https://doi.org/10.1101/2023.11.22.568356'> 10.1101/2023.11.22.568356</a></p>
        <p class="abstract">Enzyme engineering and discovery are crucial for a future sustainable bioeconomy, and harvesting new biocatalysts from large libraries through directed evolution or functional metagenomics requires accessible, rapid assays. Ultra-high throughput screening can often require an optical readout, leading to the use of model substrates that may not accurately report on activity for the target reaction and may require bespoke synthesis. In contrast, coupled assays represent a modular  plug-and-play system, where any pairing of enzyme/substrate may be investigated, if the reaction can produce a common intermediate which links the catalytic reaction to a detection cascade readout. Here we establish a detection cascade, producing a fluorescent readout in response to NAD(P)H via glutathione reductase and a subsequent thiol-mediated uncaging reaction, with a 30 nM detection limit. We demonstrate its utility for the glycosidase AxyAgu115A (producing monosaccharides from a natural biofuel feedstock) and report a three orders of magnitude improved sensitivity compared to absorbance-based systems, so that less than one catalytic turnover per enzyme molecule expressed from a single cell is detectable. These advantages are brought to bear in plate formats, but also in picoliter emulsion droplets, where enrichments of 950-fold suggest that large libraries can be interrogated against a specific query substrate.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.26.568766">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.26.568766" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.26.568766">
        <p class="paperTitle">Phytopathogenic cyclic glucohexadecaose from an inverting transglycosylase</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.26.568766" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.26.568766" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Motouchi, S.; Komba, S.; Nakai, H.; Nakajima, M.</p>
        <p class="info">Score: 3.8, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.26.568766' target='https://doi.org/10.1101/2023.11.26.568766'> 10.1101/2023.11.26.568766</a></p>
        <p class="abstract">Xanthomonas species contain numerous notoriously well-known plant pathogens. Among various pathogenic factors, the role of -1,6-cyclized {beta}-1,2-glucohexadecaose (C{beta}G16) produced by Xanthomonas campestris pv. campestris was shown previously to be vital for infecting model organisms Arabidopsis thaliana and Nicotiana benthamiana. However, enzymes responsible for biosynthesising C{beta}G16 are essentially unknown, which limits the generation of agrichemicals that inhibit C{beta}G16 synthesis. In this study, we discovered that OpgD from X. campestris pv. campestris converts linear {beta}-1,2-glucan to C{beta}G16. Structural and functional analyses revealed that OpgD from X. campestris pv. campestris possesses an anomer-inverting transglycosylation mechanism, which is unprecedented among carbohydrate-active enzymes. The discovery of this unprecedented glucan-generating mechanism reveals a new foundation for the enzymatic synthesis of carbohydrates. Furthermore, identifying C{beta}G16 synthase highly conserved in Xanthomonas provides a broadly adaptable drug target for new-genre agrichemicals that overcome antimicrobial-resistant bacterial issues.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
