<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Small-molecule binding and sensing with a designed protein family
Authors: Lee, G. R.; Pellock, S. J.; Norn, C.; Tischer, D.; Dauparas, J.; Anishchenko, I.; Mercer, J. A. M.; Kang, A.; Bera, A.; Nguyen, H.; Goreshnik, I.; Vafeados, D.; Roullier, N.; Han, H. L.; Coventry, B.; Haddox, H. K.; Liu, D. R.; Yeh, A. H.-W.; Baker, D.
Score: 24.1, Published: 2023-11-02 DOI: 10.1101/2023.11.01.565201
Despite transformative advances in protein design with deep learning, the design of small-molecule-binding proteins and sensors for arbitrary ligands remains a grand challenge.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Small-molecule binding and sensing with a designed protein family
Authors: Lee, G. R.; Pellock, S. J.; Norn, C.; Tischer, D.; Dauparas, J.; Anishchenko, I.; Mercer, J. A. M.; Kang, A.; Bera, A.; Nguyen, H.; Goreshnik, I.; Vafeados, D.; Roullier, N.; Han, H. L.; Coventry, B.; Haddox, H. K.; Liu, D. R.; Yeh, A. H.-W.; Baker, D.
Score: 24.1, Published: 2023-11-02 DOI: 10.1101/2023.11.01.565201
Despite transformative advances in protein design with deep learning, the design of small-molecule-binding proteins and sensors for arbitrary ligands remains a grand challenge." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-12T10:37:43+00:00" />
<meta property="article:modified_time" content="2023-11-12T10:37:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Small-molecule binding and sensing with a designed protein family
Authors: Lee, G. R.; Pellock, S. J.; Norn, C.; Tischer, D.; Dauparas, J.; Anishchenko, I.; Mercer, J. A. M.; Kang, A.; Bera, A.; Nguyen, H.; Goreshnik, I.; Vafeados, D.; Roullier, N.; Han, H. L.; Coventry, B.; Haddox, H. K.; Liu, D. R.; Yeh, A. H.-W.; Baker, D.
Score: 24.1, Published: 2023-11-02 DOI: 10.1101/2023.11.01.565201
Despite transformative advances in protein design with deep learning, the design of small-molecule-binding proteins and sensors for arbitrary ligands remains a grand challenge."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Small-molecule binding and sensing with a designed protein family\nAuthors: Lee, G. R.; Pellock, S. J.; Norn, C.; Tischer, D.; Dauparas, J.; Anishchenko, I.; Mercer, J. A. M.; Kang, A.; Bera, A.; Nguyen, H.; Goreshnik, I.; Vafeados, D.; Roullier, N.; Han, H. L.; Coventry, B.; Haddox, H. K.; Liu, D. R.; Yeh, A. H.-W.; Baker, D.\nScore: 24.1, Published: 2023-11-02 DOI: 10.1101/2023.11.01.565201\nDespite transformative advances in protein design with deep learning, the design of small-molecule-binding proteins and sensors for arbitrary ligands remains a grand challenge.",
  "keywords": [
    
  ],
  "articleBody": " Small-molecule binding and sensing with a designed protein family\nAuthors: Lee, G. R.; Pellock, S. J.; Norn, C.; Tischer, D.; Dauparas, J.; Anishchenko, I.; Mercer, J. A. M.; Kang, A.; Bera, A.; Nguyen, H.; Goreshnik, I.; Vafeados, D.; Roullier, N.; Han, H. L.; Coventry, B.; Haddox, H. K.; Liu, D. R.; Yeh, A. H.-W.; Baker, D.\nScore: 24.1, Published: 2023-11-02 DOI: 10.1101/2023.11.01.565201\nDespite transformative advances in protein design with deep learning, the design of small-molecule-binding proteins and sensors for arbitrary ligands remains a grand challenge. Here we combine deep learning and physics-based methods to generate a family of proteins with diverse and designable pocket geometries, which we employ to computationally design binders for six chemically and structurally distinct small-molecule targets. Biophysical characterization of the designed binders revealed nanomolar to low micromolar binding affinities and atomic-level design accuracy. The bound ligands are exposed at one edge of the binding pocket, enabling the de novo design of chemically induced dimerization (CID) systems; we take advantage of this to create a biosensor with nanomolar sensitivity for cortisol. Our approach provides a general method to design proteins that bind and sense small molecules for a wide range of analytical, environmental, and biomedical applications.\nElectric fields determine carbapenemase activity in class A Î²-lactamases\nAuthors: Jabeen, H.; Beer, M.; Spencer, J.; van der Kamp, M. W.; Bunzel, H. A.; Mulholland, A. J.\nScore: 10.9, Published: 2023-11-05 DOI: 10.1101/2023.11.04.565607\nAntimicrobial resistance is a public health crisis. Limited understanding of the catalytic drivers in resistance-mediating enzymes such as {beta}-lactamases hinders our ability to combat this crisis. Here, we dissect the catalytic contributions of active-site electric fields in class A {beta}-lactamases. We studied the enzymatic hydrolysis of a carbapenem antibiotic by QM/MM molecular dynamics simulations and quantified active-site fields with a custom-made script. We discovered that the fields correlate well with activity and identified seven positions, some distal, that distinguish efficient carbapenemases. Electric-field analysis may help predict the activity of {beta}-lactamases and guide antibiotic and enzyme design. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=109 SRC=\"FIGDIR/small/565607v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (27K): org.highwire.dtl.DTLVardef@1be29d6org.highwire.dtl.DTLVardef@1acbe37org.highwire.dtl.DTLVardef@1fc4835org.highwire.dtl.DTLVardef@1e8b9a0_HPS_FORMAT_FIGEXP M_FIG C_FIG Electric field script: www.github.com/bunzela/FieldTools\nDeuteration provides a general strategy to enhance azobenzene-based photopharmacology\nAuthors: Rossmann, K.; Gonzalez-Hernandez, A. J.; Bhuyan, R.; Boerjesson, K.; Levitz, J.; Broichhagen, J.\nScore: 10.8, Published: 2023-11-09 DOI: 10.1101/2023.11.09.566420\nHerein, we present deuterated azobenzene photoswitches as a general means of enhancing photopharmacological molecules. Deuteration improves azobenzene performance in terms of light sensitivity, photoswitch efficiency, and photoswitch kinetics with minimal alteration to the underlying structure of the photopharmacological ligand. We report synthesized deuterated azobenzene-based ligands for the optimized optical control of ion channel and G protein-coupled receptor function in live cells, setting the stage for the straightforward, widespread adoption of this approach.\nDesign of a symmetry-broken tetrahedral protein cage by a method of internal steric occlusion\nAuthors: Gladkov, N.; Scott, E. A.; Meador, K.; Lee, E. J.; Laganowsky, A. D.; Yeates, T. O.; Castells-Graells, R.\nScore: 10.3, Published: 2023-11-09 DOI: 10.1101/2023.11.08.566319\nMethods in protein design have made it possible to create large and complex, self-assembling protein cages with diverse applications. These have largely been based on highly symmetric forms exemplified by the Platonic solids. Prospective applications of protein cages would be expanded by strategies for breaking the designed symmetry, e.g., so that only one or a few (instead of many) copies of an exterior domain or motif might be displayed on their surfaces. Here we demonstrate a straightforward design approach for creating symmetry-broken protein cages able to display singular copies of outward-facing domains. We modify the subunit of an otherwise symmetric protein cage through fusion to a small inward-facing domain, only one copy of which can be accommodated in the cage interior. Using biochemical methods and native mass spectrometry, we show that co-expression of the original subunit and the modified subunit, which is further fused to an outward-facing anti-GFP DARPin domain, leads to self-assembly of a protein cage presenting just one copy of the DARPin protein on its exterior. This strategy of designed occlusion provides a facile route for creating new types of protein cages with unique properties.\nDesign of efficient artificial enzymes using crystallographically-enhanced conformational sampling\nAuthors: Rakotoharisoa, R. V.; Seifinoferest, B.; Zarifi, N.; Miller, J. D. M.; Rodriguez, J. M.; Thompson, M. C.; Chica, R. A.\nScore: 12.7, Published: 2023-11-02 DOI: 10.1101/2023.11.01.564846\nThe ability to create efficient artificial enzymes for any chemical reaction is of great interest. Here, we describe a computational design method for increasing catalytic efficiency of de novo enzymes to a level comparable to their natural counterparts without relying on directed evolution. Using structural ensembles generated from dynamics-based refinement against X-ray diffraction data collected from crystals of Kemp eliminases HG3 (kcat/KM 125 M-1 s-1) and KE70 (kcat/KM 57 M-1 s-1), we design from each enzyme [\u0026le;]10 sequences predicted to catalyze this reaction more efficiently. The most active designs display kcat/KM values improved by 100-250-fold, comparable to mutants obtained after screening thousands of variants in multiple rounds of directed evolution. Crystal structures show excellent agreement with computational models. Our work shows how computational design can generate efficient artificial enzymes by exploiting the true conformational ensemble to more effectively stabilize the transition state.\nUbiquitinated protein levels remain high during ageing in C. elegans\nAuthors: Daigle, S. E.; Doucet, S.; Lapierre, L. R.\nScore: 9.3, Published: 2023-11-08 DOI: 10.1101/2023.11.07.566011\nA wealth of literature in different model organisms have shown that proteostasis declines with age, which results in the accumulation of aggregated proteins that generally carry polyubiquitin chains. In Koyuncu et al. (Nature, 2021), the authors use a relatively mild homogenization approach to analyze total polyubiquitinated protein levels longitudinally in C. elegans and found that the overall level of polyubiquitinated proteins decreases in wild-type nematodes. Specifically, the substantial presence of polyubiquitinated proteins in early adulthood markedly decreases at day 10 and 15 of adulthood. This finding represents the central basis of their study where they subsequently suggest that age-dependent decrease in protein polyubiquitination in wild-type animals is due to enhanced deubiquitination. Here, we find that the mild homogenization of C. elegans used in Koyuncu et al. fails to properly extract all C. elegans proteins, because it largely omits relatively insoluble proteins in the pellet. When we perform complete homogenization of wild-type nematode proteins using SDS, sonication and heat, we unequivocally observe that polyubiquitinated protein levels do not decrease with age in C. elegans. Overall, the levels of polyubiquitinated proteins remains relatively constant post-reproduction. Therefore, our findings invalidate the main conclusion of Koyuncu et al. that the ubiquitinated proteome is rewired during ageing and demonstrate that C. elegans harbor polyubiquitinated proteins throughout their lifespan.\nDual-Probe Activity-Based Protein Profiling Reveals Site-Specific Differences in Protein Binding of EGFR-Directed Drugs\nAuthors: van Bergen, W.; Fiala, J.; Heck, A. J. R.; Baggelaar, M. P.\nScore: 8.9, Published: 2023-10-19 DOI: 10.1101/2023.10.19.562725\nComparative, dose-dependent analysis of interactions between small molecule drugs and their targets, as well as off-targets, in complex proteomes is crucial for selecting optimal drug candidates. The affinity of small molecules for targeted proteins is largely dictated by interactions between amino acid side chains and these drugs. Thus, studying drug-protein interactions at an amino acid resolution provides a comprehensive understanding of drug selectivity and efficacy. In this study, we further refined the site-specific activity-based protein profiling strategy, PhosID-ABPP, on a timsTOF HT mass spectrometer. This refinement enables dual dose-dependent competition of inhibitors within a single cellular proteome. Here, a comparative analysis of two activity-based probes (ABPs), developed to selectively target the epidermal growth factor receptor (EGFR), namely PF-06672131 and PF-6422899, facilitated the simultaneous identification of ABP-specific binding sites at a proteome-wide scale within a cellular proteome. Dose-dependent probe-binding preferences for proteinaceous cysteines, even at low nanomolar ABP concentrations, could be revealed. Notably, while both ABPs showed comparable affinities for the EGFR, PF-06672131 had a broader off-target reactivity profile. In contrast, PF-6422899 exhibited higher affinity for the ERBB2 receptor and bound to catalytic cysteines in several other enzymes, which is likely to disrupt their catalytic activity. Notably, PF-06672131 also effectively labeled ADP/ATP translocase proteins at a concentration of just 1 nanomolar. Additionally, analysis of different binding sites within the EGF receptor and the voltage-dependent anion channel 2 revealed secondary binding sites of both probes and provided insights into the binding poses of inhibitors on these proteins. Insights from the PhosID-ABPP analysis of these two ABPs serve as a valuable resource for understanding drug on- and off-target engagement in a dose- and site-specific manner.\nStructure and position-specific interactions of prion-like domains in transcription factor Efg1 phase separation\nAuthors: Zheng, T.; Wang, S.-H.; Fawzi, N. L.\nScore: 8.6, Published: 2023-11-09 DOI: 10.1101/2023.11.09.566450\nCandida albicans, a prominent member of the human microbiome, can make an opportunistic switch from commensal coexistence to pathogenicity accompanied by an epigenetic shift between the white and opaque cell states. This transcriptional switch is under precise regulation by a set of transcription factors (TFs), with Enhanced Filamentous Growth Protein 1 (Efg1) playing a central role. Previous research has emphasized the importance of Egf1's prion-like domain (PrLD) and the protein's ability to undergo phase separation for the white-to-opaque transition of C. albicans. However, the underlying molecular mechanisms of Efg1 phase separation have remained underexplored. In this study, we delved into the biophysical basis of Efg1 phase separation, revealing the significant contribution of both N-terminal (N) and C-terminal (C) PrLDs. Through NMR structural analysis, we found that Efg1 N-PrLD and C-PrLD are mostly disordered though have prominent partial -helical secondary structures in both domains. NMR titration experiments suggest that the partially helical structures in N-PrLD act as hubs for self-interaction as well as Efg1 interaction with RNA. Using condensed-phase NMR spectroscopy, we uncovered diverse amino acid interactions underlying Efg1 phase separation. Particularly, we highlight the indispensable role of tyrosine residues within the transient -helical structures of PrLDs particularly in the N-PrLD compared to the C-PrLD in stabilizing phase separation. Our study provides evidence that the transient -helical structure is present in the phase separated state and highlights the particular importance of aromatic residues within these structures for phase separation. Together, these results enhance the understanding of C. albicans TF interactions that lead to virulence and provide a crucial foundation for potential antifungal therapies targeting the transcriptional switch.\nAsymmetrical Bi-antennary Glycans Prepared by a Stop-and-Go Strategy Reveal Receptor Binding Evolution of Human Influenza A Viruses\nAuthors: Ma, S.; Liu, L.; Eggink, D.; Herfst, S.; Fouchier, R. A. M.; de Vries, R. P.; Boons, G.-J.\nScore: 6.3, Published: 2023-11-09 DOI: 10.1101/2023.11.08.566285\nGlycan binding properties of respiratory viruses have been difficult to probe due to a lack of biological relevant glycans for binding studies. Here, a stop-and-go chemoenzymatic methodology is presented that gave access to a panel of 32 asymmetrical bi-antennary N-glycans having various numbers of N-acetyl lactosamine (LacNAc) repeating units capped by 2,3- or 2,6-sialosides resembling structures found in airway tissues. It exploits that the branching enzymes MGAT1 and MGAT2 can utilize unnatural UDP-2-deoxy-2-trifluoro-N-acetamido-glucose (UDP-GlcNTFA) as donor. The TFA moiety of the resulting glycans can be hydrolyzed to give GlcNH2 at one of the antennae that temporarily blocks extension by glycosyl transferases. The N-glycans were printed as a microarray that was probed for receptor binding specificities of evolutionary distinct human A(H3N2) and A(H1N1)pdm09 viruses. It was found that not only the sialoside type but also the length of the LacNAc chain and presentation at the 1,3-antenna of N-glycans is critical for binding. Early A(H3N2) viruses bound to 2,6-sialosides at a single LacNAc moiety at the 1,3-antenna whereas later viruses required the sialoside to be presented at a tri-LacNAc moiety. Surprisingly, most of the A(H3N2) viruses that appeared after 2021 regained binding capacity to sialosides presented at a di-LacNAc moiety. As a result, these viruses agglutinate erythrocytes again, commonly employed for antigenic characterization of influenza viruses. Human A(H1N1)pdm09 viruses have similar receptor binding properties as recent A(H3N2) viruses. The data indicates that an asymmetric N-glycan having 2,6-sialoside at a di-LacNAc moiety is a commonly employed receptor by human influenza A viruses.\nReplication of 25 microsatellite sequences by human DNA polymerase delta holoenzymes is dependent on dNTP and RPA levels\nAuthors: Pytko, K. G.; Dannenberg, R. L.; Eckert, K. A.; Hedglin, M.\nScore: 6.0, Published: 2023-11-08 DOI: 10.1101/2023.11.07.566133\nDifficult-to-Replicate Sequences (DiToRS) are natural impediments in the human genome that inhibit DNA replication under endogenous replication. Some of the most widely-studied DiToRS are A+T-rich, high \"flexibility regions,\" including long stretches of perfect [AT/TA] microsatellite repeats that have the potential to collapse into hairpin structures when in single-stranded DNA (ssDNA) form and are sites of recurrent structural variation and double-stranded DNA (dsDNA) breaks. Currently, it is unclear how these flexibility regions impact DNA replication, greatly limiting our fundamental understanding of human genome stability. To investigate replication through flexibility regions, we utilized FRET to characterize the effects of the major ssDNA-binding complex, RPA, on the structure of perfect [AT/TA]25 microsatellite repeats and also re-constituted human lagging strand replication to quantitatively characterize initial encounters of pol {delta} holoenzymes with A+T-rich DNA template sequences. The results indicate that [AT/TA]25 sequences adopt hairpin structures that are unwound by RPA and pol {delta} holoenzymes support dNTP incorporation through the [AT/TA]25 sequences as well as an A+T-rich, non-structure forming sequence. Furthermore, the extent of dNTP incorporation is dependent on the sequence of the DNA template and the concentration of dNTPs. Importantly, the effects of RPA on the replication of [AT/TA]25 sequences are dependent on the concentration of dNTPs, whereas the effects of RPA on the replication of an A+T-rich, non-structure forming sequence are independent of dNTP concentration. Collectively, these results reveal complexities in lagging strand replication and provide novel insights into how flexibility regions contribute to genome instability.\n",
  "wordCount" : "2255",
  "inLanguage": "en",
  "datePublished": "2023-11-12T10:37:43Z",
  "dateModified": "2023-11-12T10:37:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 12, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565201">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565201" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565201">
        <p class="paperTitle">Small-molecule binding and sensing with a designed protein family</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565201" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565201" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, G. R.; Pellock, S. J.; Norn, C.; Tischer, D.; Dauparas, J.; Anishchenko, I.; Mercer, J. A. M.; Kang, A.; Bera, A.; Nguyen, H.; Goreshnik, I.; Vafeados, D.; Roullier, N.; Han, H. L.; Coventry, B.; Haddox, H. K.; Liu, D. R.; Yeh, A. H.-W.; Baker, D.</p>
        <p class="info">Score: 24.1, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565201' target='https://doi.org/10.1101/2023.11.01.565201'> 10.1101/2023.11.01.565201</a></p>
        <p class="abstract">Despite transformative advances in protein design with deep learning, the design of small-molecule-binding proteins and sensors for arbitrary ligands remains a grand challenge. Here we combine deep learning and physics-based methods to generate a family of proteins with diverse and designable pocket geometries, which we employ to computationally design binders for six chemically and structurally distinct small-molecule targets. Biophysical characterization of the designed binders revealed nanomolar to low micromolar binding affinities and atomic-level design accuracy. The bound ligands are exposed at one edge of the binding pocket, enabling the de novo design of chemically induced dimerization (CID) systems; we take advantage of this to create a biosensor with nanomolar sensitivity for cortisol. Our approach provides a general method to design proteins that bind and sense small molecules for a wide range of analytical, environmental, and biomedical applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.04.565607">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.04.565607" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.04.565607">
        <p class="paperTitle">Electric fields determine carbapenemase activity in class A Î²-lactamases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.04.565607" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.04.565607" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jabeen, H.; Beer, M.; Spencer, J.; van der Kamp, M. W.; Bunzel, H. A.; Mulholland, A. J.</p>
        <p class="info">Score: 10.9, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.04.565607' target='https://doi.org/10.1101/2023.11.04.565607'> 10.1101/2023.11.04.565607</a></p>
        <p class="abstract">Antimicrobial resistance is a public health crisis. Limited understanding of the catalytic drivers in resistance-mediating enzymes such as {beta}-lactamases hinders our ability to combat this crisis. Here, we dissect the catalytic contributions of active-site electric fields in class A {beta}-lactamases. We studied the enzymatic hydrolysis of a carbapenem antibiotic by QM/MM molecular dynamics simulations and quantified active-site fields with a custom-made script. We discovered that the fields correlate well with activity and identified seven positions, some distal, that distinguish efficient carbapenemases. Electric-field analysis may help predict the activity of {beta}-lactamases and guide antibiotic and enzyme design.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=109 SRC=&#34;FIGDIR/small/565607v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (27K):
org.highwire.dtl.DTLVardef@1be29d6org.highwire.dtl.DTLVardef@1acbe37org.highwire.dtl.DTLVardef@1fc4835org.highwire.dtl.DTLVardef@1e8b9a0_HPS_FORMAT_FIGEXP  M_FIG C_FIG Electric field script: www.github.com/bunzela/FieldTools</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.566420">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.566420" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.566420">
        <p class="paperTitle">Deuteration provides a general strategy to enhance azobenzene-based photopharmacology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.566420" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.566420" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rossmann, K.; Gonzalez-Hernandez, A. J.; Bhuyan, R.; Boerjesson, K.; Levitz, J.; Broichhagen, J.</p>
        <p class="info">Score: 10.8, Published: 2023-11-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.566420' target='https://doi.org/10.1101/2023.11.09.566420'> 10.1101/2023.11.09.566420</a></p>
        <p class="abstract">Herein, we present deuterated azobenzene photoswitches as a general means of enhancing photopharmacological molecules. Deuteration improves azobenzene performance in terms of light sensitivity, photoswitch efficiency, and photoswitch kinetics with minimal alteration to the underlying structure of the photopharmacological ligand. We report synthesized deuterated azobenzene-based ligands for the optimized optical control of ion channel and G protein-coupled receptor function in live cells, setting the stage for the straightforward, widespread adoption of this approach.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.08.566319">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.08.566319" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.08.566319">
        <p class="paperTitle">Design of a symmetry-broken tetrahedral protein cage by a method of internal steric occlusion</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.08.566319" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.08.566319" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gladkov, N.; Scott, E. A.; Meador, K.; Lee, E. J.; Laganowsky, A. D.; Yeates, T. O.; Castells-Graells, R.</p>
        <p class="info">Score: 10.3, Published: 2023-11-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.08.566319' target='https://doi.org/10.1101/2023.11.08.566319'> 10.1101/2023.11.08.566319</a></p>
        <p class="abstract">Methods in protein design have made it possible to create large and complex, self-assembling protein cages with diverse applications. These have largely been based on highly symmetric forms exemplified by the Platonic solids. Prospective applications of protein cages would be expanded by strategies for breaking the designed symmetry, e.g., so that only one or a few (instead of many) copies of an exterior domain or motif might be displayed on their surfaces. Here we demonstrate a straightforward design approach for creating symmetry-broken protein cages able to display singular copies of outward-facing domains. We modify the subunit of an otherwise symmetric protein cage through fusion to a small inward-facing domain, only one copy of which can be accommodated in the cage interior. Using biochemical methods and native mass spectrometry, we show that co-expression of the original subunit and the modified subunit, which is further fused to an outward-facing anti-GFP DARPin domain, leads to self-assembly of a protein cage presenting just one copy of the DARPin protein on its exterior. This strategy of designed occlusion provides a facile route for creating new types of protein cages with unique properties.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.564846">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.564846" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.564846">
        <p class="paperTitle">Design of efficient artificial enzymes using crystallographically-enhanced conformational sampling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.564846" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.564846" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rakotoharisoa, R. V.; Seifinoferest, B.; Zarifi, N.; Miller, J. D. M.; Rodriguez, J. M.; Thompson, M. C.; Chica, R. A.</p>
        <p class="info">Score: 12.7, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.564846' target='https://doi.org/10.1101/2023.11.01.564846'> 10.1101/2023.11.01.564846</a></p>
        <p class="abstract">The ability to create efficient artificial enzymes for any chemical reaction is of great interest. Here, we describe a computational design method for increasing catalytic efficiency of de novo enzymes to a level comparable to their natural counterparts without relying on directed evolution. Using structural ensembles generated from dynamics-based refinement against X-ray diffraction data collected from crystals of Kemp eliminases HG3 (kcat/KM 125 M-1 s-1) and KE70 (kcat/KM 57 M-1 s-1), we design from each enzyme [&amp;le;]10 sequences predicted to catalyze this reaction more efficiently. The most active designs display kcat/KM values improved by 100-250-fold, comparable to mutants obtained after screening thousands of variants in multiple rounds of directed evolution. Crystal structures show excellent agreement with computational models. Our work shows how computational design can generate efficient artificial enzymes by exploiting the true conformational ensemble to more effectively stabilize the transition state.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.566011">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.566011" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.566011">
        <p class="paperTitle">Ubiquitinated protein levels remain high during ageing in C. elegans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.566011" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.566011" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Daigle, S. E.; Doucet, S.; Lapierre, L. R.</p>
        <p class="info">Score: 9.3, Published: 2023-11-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.566011' target='https://doi.org/10.1101/2023.11.07.566011'> 10.1101/2023.11.07.566011</a></p>
        <p class="abstract">A wealth of literature in different model organisms have shown that proteostasis declines with age, which results in the accumulation of aggregated proteins that generally carry polyubiquitin chains. In Koyuncu et al. (Nature, 2021), the authors use a relatively mild homogenization approach to analyze total polyubiquitinated protein levels longitudinally in C. elegans and found that the overall level of polyubiquitinated proteins decreases in wild-type nematodes. Specifically, the substantial presence of polyubiquitinated proteins in early adulthood markedly decreases at day 10 and 15 of adulthood. This finding represents the central basis of their study where they subsequently suggest that age-dependent decrease in protein polyubiquitination in wild-type animals is due to enhanced deubiquitination. Here, we find that the mild homogenization of C. elegans used in Koyuncu et al. fails to properly extract all C. elegans proteins, because it largely omits relatively insoluble proteins in the pellet. When we perform complete homogenization of wild-type nematode proteins using SDS, sonication and heat, we unequivocally observe that polyubiquitinated protein levels do not decrease with age in C. elegans. Overall, the levels of polyubiquitinated proteins remains relatively constant post-reproduction. Therefore, our findings invalidate the main conclusion of Koyuncu et al. that the ubiquitinated proteome is rewired during ageing and demonstrate that C. elegans harbor polyubiquitinated proteins throughout their lifespan.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.19.562725">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.19.562725" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.19.562725">
        <p class="paperTitle">Dual-Probe Activity-Based Protein Profiling Reveals Site-Specific Differences in Protein Binding of EGFR-Directed Drugs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.19.562725" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.19.562725" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: van Bergen, W.; Fiala, J.; Heck, A. J. R.; Baggelaar, M. P.</p>
        <p class="info">Score: 8.9, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.19.562725' target='https://doi.org/10.1101/2023.10.19.562725'> 10.1101/2023.10.19.562725</a></p>
        <p class="abstract">Comparative, dose-dependent analysis of interactions between small molecule drugs and their targets, as well as off-targets, in complex proteomes is crucial for selecting optimal drug candidates. The affinity of small molecules for targeted proteins is largely dictated by interactions between amino acid side chains and these drugs. Thus, studying drug-protein interactions at an amino acid resolution provides a comprehensive understanding of drug selectivity and efficacy. In this study, we further refined the site-specific activity-based protein profiling strategy, PhosID-ABPP, on a timsTOF HT mass spectrometer. This refinement enables dual dose-dependent competition of inhibitors within a single cellular proteome. Here, a comparative analysis of two activity-based probes (ABPs), developed to selectively target the epidermal growth factor receptor (EGFR), namely PF-06672131 and PF-6422899, facilitated the simultaneous identification of ABP-specific binding sites at a proteome-wide scale within a cellular proteome. Dose-dependent probe-binding preferences for proteinaceous cysteines, even at low nanomolar ABP concentrations, could be revealed. Notably, while both ABPs showed comparable affinities for the EGFR, PF-06672131 had a broader off-target reactivity profile. In contrast, PF-6422899 exhibited higher affinity for the ERBB2 receptor and bound to catalytic cysteines in several other enzymes, which is likely to disrupt their catalytic activity. Notably, PF-06672131 also effectively labeled ADP/ATP translocase proteins at a concentration of just 1 nanomolar. Additionally, analysis of different binding sites within the EGF receptor and the voltage-dependent anion channel 2 revealed secondary binding sites of both probes and provided insights into the binding poses of inhibitors on these proteins. Insights from the PhosID-ABPP analysis of these two ABPs serve as a valuable resource for understanding drug on- and off-target engagement in a dose- and site-specific manner.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.566450">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.566450" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.566450">
        <p class="paperTitle">Structure and position-specific interactions of prion-like domains in transcription factor Efg1 phase separation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.566450" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.566450" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zheng, T.; Wang, S.-H.; Fawzi, N. L.</p>
        <p class="info">Score: 8.6, Published: 2023-11-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.566450' target='https://doi.org/10.1101/2023.11.09.566450'> 10.1101/2023.11.09.566450</a></p>
        <p class="abstract">Candida albicans, a prominent member of the human microbiome, can make an opportunistic switch from commensal coexistence to pathogenicity accompanied by an epigenetic shift between the white and opaque cell states. This transcriptional switch is under precise regulation by a set of transcription factors (TFs), with Enhanced Filamentous Growth Protein 1 (Efg1) playing a central role. Previous research has emphasized the importance of Egf1&#39;s prion-like domain (PrLD) and the protein&#39;s ability to undergo phase separation for the white-to-opaque transition of C. albicans. However, the underlying molecular mechanisms of Efg1 phase separation have remained underexplored. In this study, we delved into the biophysical basis of Efg1 phase separation, revealing the significant contribution of both N-terminal (N) and C-terminal (C) PrLDs. Through NMR structural analysis, we found that Efg1 N-PrLD and C-PrLD are mostly disordered though have prominent partial -helical secondary structures in both domains. NMR titration experiments suggest that the partially helical structures in N-PrLD act as hubs for self-interaction as well as Efg1 interaction with RNA. Using condensed-phase NMR spectroscopy, we uncovered diverse amino acid interactions underlying Efg1 phase separation. Particularly, we highlight the indispensable role of tyrosine residues within the transient -helical structures of PrLDs particularly in the N-PrLD compared to the C-PrLD in stabilizing phase separation. Our study provides evidence that the transient -helical structure is present in the phase separated state and highlights the particular importance of aromatic residues within these structures for phase separation. Together, these results enhance the understanding of C. albicans TF interactions that lead to virulence and provide a crucial foundation for potential antifungal therapies targeting the transcriptional switch.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.08.566285">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.08.566285" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.08.566285">
        <p class="paperTitle">Asymmetrical Bi-antennary Glycans Prepared by a Stop-and-Go Strategy Reveal Receptor Binding Evolution of Human Influenza A Viruses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.08.566285" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.08.566285" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ma, S.; Liu, L.; Eggink, D.; Herfst, S.; Fouchier, R. A. M.; de Vries, R. P.; Boons, G.-J.</p>
        <p class="info">Score: 6.3, Published: 2023-11-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.08.566285' target='https://doi.org/10.1101/2023.11.08.566285'> 10.1101/2023.11.08.566285</a></p>
        <p class="abstract">Glycan binding properties of respiratory viruses have been difficult to probe due to a lack of biological relevant glycans for binding studies. Here, a stop-and-go chemoenzymatic methodology is presented that gave access to a panel of 32 asymmetrical bi-antennary N-glycans having various numbers of N-acetyl lactosamine (LacNAc) repeating units capped by 2,3- or 2,6-sialosides resembling structures found in airway tissues. It exploits that the branching enzymes MGAT1 and MGAT2 can utilize unnatural UDP-2-deoxy-2-trifluoro-N-acetamido-glucose (UDP-GlcNTFA) as donor. The TFA moiety of the resulting glycans can be hydrolyzed to give GlcNH2 at one of the antennae that temporarily blocks extension by glycosyl transferases. The N-glycans were printed as a microarray that was probed for receptor binding specificities of evolutionary distinct human A(H3N2) and A(H1N1)pdm09 viruses. It was found that not only the sialoside type but also the length of the LacNAc chain and presentation at the 1,3-antenna of N-glycans is critical for binding. Early A(H3N2) viruses bound to 2,6-sialosides at a single LacNAc moiety at the 1,3-antenna whereas later viruses required the sialoside to be presented at a tri-LacNAc moiety. Surprisingly, most of the A(H3N2) viruses that appeared after 2021 regained binding capacity to sialosides presented at a di-LacNAc moiety. As a result, these viruses agglutinate erythrocytes again, commonly employed for antigenic characterization of influenza viruses. Human A(H1N1)pdm09 viruses have similar receptor binding properties as recent A(H3N2) viruses. The data indicates that an asymmetric N-glycan having 2,6-sialoside at a di-LacNAc moiety is a commonly employed receptor by human influenza A viruses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.566133">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.566133" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.566133">
        <p class="paperTitle">Replication of 25 microsatellite sequences by human DNA polymerase delta holoenzymes is dependent on dNTP and RPA levels</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.566133" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.566133" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pytko, K. G.; Dannenberg, R. L.; Eckert, K. A.; Hedglin, M.</p>
        <p class="info">Score: 6.0, Published: 2023-11-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.566133' target='https://doi.org/10.1101/2023.11.07.566133'> 10.1101/2023.11.07.566133</a></p>
        <p class="abstract">Difficult-to-Replicate Sequences (DiToRS) are natural impediments in the human genome that inhibit DNA replication under endogenous replication. Some of the most widely-studied DiToRS are A&#43;T-rich, high &#34;flexibility regions,&#34; including long stretches of perfect [AT/TA] microsatellite repeats that have the potential to collapse into hairpin structures when in single-stranded DNA (ssDNA) form and are sites of recurrent structural variation and double-stranded DNA (dsDNA) breaks. Currently, it is unclear how these flexibility regions impact DNA replication, greatly limiting our fundamental understanding of human genome stability. To investigate replication through flexibility regions, we utilized FRET to characterize the effects of the major ssDNA-binding complex, RPA, on the structure of perfect [AT/TA]25 microsatellite repeats and also re-constituted human lagging strand replication to quantitatively characterize initial encounters of pol {delta} holoenzymes with A&#43;T-rich DNA template sequences. The results indicate that [AT/TA]25 sequences adopt hairpin structures that are unwound by RPA and pol {delta} holoenzymes support dNTP incorporation through the [AT/TA]25 sequences as well as an A&#43;T-rich, non-structure forming sequence. Furthermore, the extent of dNTP incorporation is dependent on the sequence of the DNA template and the concentration of dNTPs. Importantly, the effects of RPA on the replication of [AT/TA]25 sequences are dependent on the concentration of dNTPs, whereas the effects of RPA on the replication of an A&#43;T-rich, non-structure forming sequence are independent of dNTP concentration. Collectively, these results reveal complexities in lagging strand replication and provide novel insights into how flexibility regions contribute to genome instability.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
