<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Improving protein expression, stability, and function with ProteinMPNN
Authors: Sumida, K. H.; Nunez Franco, R.; Kalvet, I.; Pellock, S. J.; Wicky, B. I. M.; Milles, L. F.; Dauparas, J.; Wang, J.; Kipnis, Y.; Jameson, N.; Kang, A.; De La Cruz, J.; Sankaran, B.; Bera, A. K.; Jimenez Oses, G.; Baker, D.
Score: 85.5, Published: 2023-10-03 DOI: 10.1101/2023.10.03.560713
Natural proteins are highly optimized for function, but are often difficult to produce at a scale suitable for biotechnological applications due to poor expression in heterologous systems, limited solubility, and sensitivity to temperature.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Improving protein expression, stability, and function with ProteinMPNN
Authors: Sumida, K. H.; Nunez Franco, R.; Kalvet, I.; Pellock, S. J.; Wicky, B. I. M.; Milles, L. F.; Dauparas, J.; Wang, J.; Kipnis, Y.; Jameson, N.; Kang, A.; De La Cruz, J.; Sankaran, B.; Bera, A. K.; Jimenez Oses, G.; Baker, D.
Score: 85.5, Published: 2023-10-03 DOI: 10.1101/2023.10.03.560713
Natural proteins are highly optimized for function, but are often difficult to produce at a scale suitable for biotechnological applications due to poor expression in heterologous systems, limited solubility, and sensitivity to temperature." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-08T10:37:16+00:00" />
<meta property="article:modified_time" content="2023-10-08T10:37:16+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Improving protein expression, stability, and function with ProteinMPNN
Authors: Sumida, K. H.; Nunez Franco, R.; Kalvet, I.; Pellock, S. J.; Wicky, B. I. M.; Milles, L. F.; Dauparas, J.; Wang, J.; Kipnis, Y.; Jameson, N.; Kang, A.; De La Cruz, J.; Sankaran, B.; Bera, A. K.; Jimenez Oses, G.; Baker, D.
Score: 85.5, Published: 2023-10-03 DOI: 10.1101/2023.10.03.560713
Natural proteins are highly optimized for function, but are often difficult to produce at a scale suitable for biotechnological applications due to poor expression in heterologous systems, limited solubility, and sensitivity to temperature."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Improving protein expression, stability, and function with ProteinMPNN\nAuthors: Sumida, K. H.; Nunez Franco, R.; Kalvet, I.; Pellock, S. J.; Wicky, B. I. M.; Milles, L. F.; Dauparas, J.; Wang, J.; Kipnis, Y.; Jameson, N.; Kang, A.; De La Cruz, J.; Sankaran, B.; Bera, A. K.; Jimenez Oses, G.; Baker, D.\nScore: 85.5, Published: 2023-10-03 DOI: 10.1101/2023.10.03.560713\nNatural proteins are highly optimized for function, but are often difficult to produce at a scale suitable for biotechnological applications due to poor expression in heterologous systems, limited solubility, and sensitivity to temperature.",
  "keywords": [
    
  ],
  "articleBody": " Improving protein expression, stability, and function with ProteinMPNN\nAuthors: Sumida, K. H.; Nunez Franco, R.; Kalvet, I.; Pellock, S. J.; Wicky, B. I. M.; Milles, L. F.; Dauparas, J.; Wang, J.; Kipnis, Y.; Jameson, N.; Kang, A.; De La Cruz, J.; Sankaran, B.; Bera, A. K.; Jimenez Oses, G.; Baker, D.\nScore: 85.5, Published: 2023-10-03 DOI: 10.1101/2023.10.03.560713\nNatural proteins are highly optimized for function, but are often difficult to produce at a scale suitable for biotechnological applications due to poor expression in heterologous systems, limited solubility, and sensitivity to temperature. Thus, a general method that improves the physical properties of native proteins while maintaining function could have wide utility for protein-based technologies. Here we show that the deep neural network ProteinMPNN together with evolutionary and structural information provides a route to increasing protein expression, stability, and function. For both myoglobin and tobacco etch virus (TEV) protease, we generated designs with improved expression, elevated melting temperatures, and improved function. For TEV protease, we identified multiple designs with improved catalytic activity as compared to the parent sequence and previously reported TEV variants. Our approach should be broadly useful for improving the expression, stability, and function of biotechnologically important proteins.\nTargeted Protein Relocalization via Protein Transport Coupling\nAuthors: Ng, C. S. C.; Liu, A.; Cui, B.; Banik, S. M.\nScore: 37.5, Published: 2023-10-04 DOI: 10.1101/2023.10.04.560943\nSubcellular protein localization regulates protein function and can be corrupted in cancers and neurodegenerative diseases. The localization of numerous proteins has been annotated, and pharmacologically relevant approaches for precise rewiring of localization to address disease-driving phenotypes would be an attractive targeted therapeutic approach. Molecules which harness the trafficking of a shuttle protein to control the subcellular localization of a target protein could provide an avenue for targeted protein relocalization for interactome-rewiring therapeutics. To realize this concept, we deploy a quantitative approach to identify features which govern the ability to hijack protein trafficking, develop a collection of shuttle proteins and ligands, and demonstrate relocalization of proteins bearing endogenous localization signals. Using a custom imaging analysis pipeline, we show that endogenous localization signals can be overcome through molecular coupling of target proteins to shuttle proteins containing sufficiently strong native localization sequences expressed in the necessary abundance. We develop nuclear hormone receptors as viable shuttles which can be harnessed with Targeted Relocalization Activating Molecules (TRAMs) to redistribute disease-driving mutant proteins such as SMARCB1Q318X, TDP43, and FUSR495X. Small molecule-mediated relocalization of FUSR495X to the nucleus from the cytoplasm reduced the number of cellular stress granules in a model of cellular stress. Using Cas9-mediated knock-in tagging, we demonstrate nuclear enrichment of both low abundance (FOXO3a) and high abundance (FKBP12) endogenous proteins via molecular coupling to nuclear hormone receptor trafficking. Finally, small molecule-mediated redistribution of NMNAT1 from nuclei to axons in primary neurons was able to slow axonal degeneration and pharmacologically mimic the WldS gain-of-function phenotype from mice resistant to certain types of neurodegeneration8. The concept of targeted protein relocalization could therefore nucleate approaches for treating disease through interactome rewiring.\nMapping the biochemical landscape of rubisco\nAuthors: Prywes, N.; Phillips, N. R.; Oltrogge, L. M.; de Pins, B.; Cowan, A. E.; Taylor-Kearney, L. J.; Chang, H. A.; Hall, L. N.; Bhatt, A.; Shih, P.; Milo, R.; Savage, D. F.\nScore: 21.1, Published: 2023-09-27 DOI: 10.1101/2023.09.27.559826\nThe enzyme rubisco catalyzes the first step of carbon assimilation in photosynthesis. Despite the massive flux of CO2 passing through this active site over billions of years, extant rubisco has relatively slow kinetics and is prone to off-target activity. In many growth regimes, this limits photosynthesis in planta. Many attempts have been made to improve the kinetic parameters of rubisco with limited success, potentially due to biochemical trade-offs. To understand the structural basis of constraints on rubisco, a comprehensive map of rubisco at the individual amino-acid level is needed. To that end we performed a deep mutational scan using a rubisco-dependent E. coli strain. By titrating CO2 concentrations it was possible to determine estimations for both catalytic rate (kcat) and substrate affinity (KM) of \u003e99% of rubisco point mutants. Some positions were found to act as \"rheostats\" where some amino-acid substitutions reduced -while others improved -affinity for CO2. No individual point mutation was found to substantially improve the catalytic rate, but a number of highly phylogenetically conserved positions were found to tolerate mutations, indicating that a large portion of rubiscos sequence space remains unexplored by nature and may serve as a resource for future protein engineering efforts. Together, these biochemical measurements inform our understanding of biochemical tradeoffs and will assist in future efforts to improve rubisco catalytic properties.\nMeasuring carbohydrate recognition profile of lectins on live cells using liquid glycan array (LiGA)\nAuthors: Sojitra, M.; Schmidt, E. N.; Lima, G. M.; Carpenter, E. J.; McCord, K. A.; Atrazhev, A.; Macauley, M. S.; Derda, R.\nScore: 11.9, Published: 2023-10-05 DOI: 10.1101/2023.10.04.559794\nGlycans constitute a significant fraction of biomolecular diversity on the surface of cells across all the species in all kingdoms of life. As the structure of glycans is not encoded by the DNA of the host organisms, it is impossible to use cutting-edge DNA technology to study the role of cellular glycosylation or to understand how cell-surface glycome is recognized by glycan-binding proteins (GBPs). To address this gap, we recently described a genetically-encoded liquid glycan array (LiGA) platform that allows profiling of glycan:GBP interactions on the surface of live cells in vitro and in vivo using next-generation sequencing (NGS). LiGA is a library of DNA-barcoded bacteriophages coated with 5-1500 copies of a glycan; the DNA barcode inside each bacteriophage encodes the structure and density of the displayed glycans. Deep sequencing of the glycophages associated with live cells yields a glycan-binding profile of GBPs displayed on the surface of such cells. This protocol provides detailed instructions of using LiGA to probe cell surface receptors and includes information on the preparation of glycophages, analysis by MALDI-TOF MS, the assembly of a LiGA library, and its deep-sequencing. Using the protocol detailed in this report, we measure a glycan-binding profile of the immunomodulatory Siglec-1, 2, -6, -7, and -9 expressed on the surface of different cell types and uncover previously unknown environment-dependent recognition of glycans by Siglec-receptors on the surface of live cells. Protocols similar to the one described in this report will make it possible to measure the precise glycan-binding profile of any GPBs displayed on the surface of any cell types.\nComputational prediction of structure, function and interaction of aphid salivary effector proteins\nAuthors: Waksman, T.; Astin, E.; Fisher, S. R.; Hunter, W.; Bos, J. I.\nScore: 11.4, Published: 2023-10-02 DOI: 10.1101/2023.10.02.560451\nSimilar to plant pathogens, phloem-feeding insects such as aphids deliver effector proteins inside their hosts that act to promote host susceptibility and enable feeding and infestation. Despite exciting progress towards identifying and characterizing effector proteins from these insects, their functions remain largely unknown. The recent ground-breaking development in protein structure prediction algorithms combined with the availability of proteomics and transcriptomic datasets for agriculturally important pests, such as the aphid Myzus persicae (green peach aphid), provides new opportunities to explore the structural and functional diversity of effector repertoires. In this study, we sought to gain insight into the the M. persicae effector repertoire by predicting and analysing the structures of a set of 71 effector candidate proteins. We used two protein structure prediction methods, AlphaFold and OmegaFold, which produced mutually consistent results. We observed a wide continuous spectrum of sizes and structures among the effector candidates, from disordered proteins to globular enzymes. We made use of the structural information and state-of-the-art computational methods to predict M. persicae effector protein properties, including function and interaction with host plant proteins. Overall, our investigation provides novel insights into the structure, function, and interaction prediction of aphid effector repertoires and will guide the necessary experimental characterization to address new hypotheses.\nPROxAb Shuttle: A non-covalent plug-and-play platform for the rapid generation of tumor-targeting antibody-PROTAC conjugates\nAuthors: Schneider, H.; Jaeger, S.; Koenning, D.; Rasche, N.; Schroeter, C.; Elter, D.; Evers, A.; Lecomte, M.; Riccardi Sirtori, F.; Schwarz, D.; Wegener, A.; Hartung, I.; Rieker, M.\nScore: 21.5, Published: 2023-09-29 DOI: 10.1101/2023.09.29.558399\nProteolysis-targeting chimeras (PROTACs) have evolved in recent years from an academic idea to a therapeutic modality with more than 25 active clinical programs. However, achieving oral bioavailability and cell-type specificity remains a challenge, especially for PROTACs recruiting the von Hippel-Lindau (VHL) E3 ligase. Herein, we present an unprecedented, plug- and-play platform for VHL-recruiting PROTACs to overcome these limitations. Our platform allows for the generation of non-covalent antibody-PROTAC complexes within minutes and obviates the need for prior PROTAC modification, antibody-drug linker chemistry optimization or bioconjugation. Our technology relies on camelid-derived antibody domains (VHHs) which can easily be engineered into existing therapeutic antibody scaffolds. The resulting targeted, bispecific fusion proteins can be complexed with PROTACs at defined PROTAC-to-antibody ratios and have been termed PROxAb Shuttles. PROxAb Shuttles can prolong the half-life of PROTACs from hours to days, demonstrate anti-tumor efficacy in vivo and have the potential for reloading in vivo to further boost efficacy.\nStructural and molecular basis of choline uptake into the brain by FLVCR2\nAuthors: Cater, R. J.; Mukherjee, D.; Gil Iturbe, E.; Erramilli, S. K.; Chen, T.; Koo, K.; Santander, N. G.; Reckers, A. R.; Kloss, B.; Gawda, T.; Choy, B. C.; Zhang, Z.; Clarke, O. B.; Yee, S. W.; Kossiakoff, A. K.; Quick, M.; Arnold, T.; Mancia, F.\nScore: 8.3, Published: 2023-10-05 DOI: 10.1101/2023.10.05.561059\nCholine is an essential nutrient that the human body needs in vast quantities for cell membrane synthesis, epigenetic modification, and neurotransmission. The brain has a particularly high demand for choline, but how it enters the brain has eluded the field for over fifty years. The MFS transporter FLVCR1 was recently determined to be a choline transporter, and while this protein is not highly expressed at the blood-brain barrier (BBB), its relative FLVCR2 is. Previous studies have shown that mutations in human Flvcr2 cause cerebral vascular abnormalities, hydrocephalus, and embryonic lethality, but the physiological role of FLVCR2 is unknown. Here, we demonstrate both in vivo and in vitro that FLVCR2 is a BBB choline transporter and is responsible for the majority of choline uptake into the brain. We also determine the structures of choline-bound FLVCR2 in the inward- and outward-facing states using cryo-electron microscopy to 2.49 and 2.77 Angstrom resolution, respectively. These results reveal how the brain obtains choline and provide molecular-level insights into how FLVCR2 binds choline in an aromatic cage and mediates its uptake. Our work could provide a novel framework for the targeted delivery of neurotherapeutics into the brain.\nIdentification and structural characterization of small-molecule inhibitors of PINK1\nAuthors: Rasool, S.; Shomali, T.; Truong, L.; Croteau, N.; Veyron, S.; Trempe, J.-F.\nScore: 7.0, Published: 2023-10-02 DOI: 10.1101/2023.10.02.560389\nMutations in PTEN-induced putative kinase 1 (PINK1) cause early-onset autosomal recessive Parkinsons Disease (PD). PINK1 is a Ser/Thr protein kinase which functions as a mitochondrial damage sensor and initiates mitochondrial quality control by accumulating on the damaged organelle. There, it will autophosphorylate and then phosphorylate ubiquitin chains, which in turn will recruit and activate Parkin, and E3 ubiquitin ligase also implicated in PD. Ubiquitylation of mitochondrial proteins leads to the autophagic degradation of the damaged organelle. Pharmacological modulation of PINK1 constitutes an appealing avenue to study its physiological function and develop PD therapeutics. In this study, we used a thermal shift assay to identify small-molecule inhibitors of PINK1. In vitro kinase activity assays demonstrate that these molecules are ATP competitive inhibitors that block ubiquitin phosphorylation. PRT062607 (a SYK inhibitor) is the most potent inhibitor of PINK1 in our screen and has an IC50 of 2 M against insect PINK1 and 1 M in HeLa cells expressing human PINK1. The crystal structures of PINK1 in complex with PRT062607 or CYC116 reveal how the compounds interact with the ATP-binding pocket. PRT062607 notably engages with the catalytic aspartate (type-1 inhibition) and causes a destabilization of insert-2 at the autophosphorylation dimer interface. Our findings provide a scaffold for the development of more selective and potent inhibitors of PINK1 that can be used as chemical probes.\nStructural dynamics of the active HER4 and HER2/HER4 complexes is finely tuned by different growth factors and glycosylation\nAuthors: Trenker, R.; Diwanji, D.; Bingham, T.; Verba, K. A.; Jura, N.\nScore: 4.8, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561161\nHuman Epidermal growth factor Receptor 4 (HER4) carries out essential functions in the development and maintenance of the cardiovascular and nervous systems. HER4 activation is regulated by a diverse group of extracellular ligands including the neuregulin (NRG) family and betacellulin (BTC), which promote HER4 homodimerization or heterodimerization with other HER receptors. Important cardiovascular functions of HER4 are exerted via heterodimerization with its close homolog and orphan receptor, HER2. To date structural insights into ligand-mediated HER4 activation have been limited to crystallographic studies of HER4 ectodomain homodimers in complex with NRG1b. Here we report cryo-EM structures of near full-length HER2/HER4 heterodimers and full-length HER4 homodimers bound to NRG1b and BTC. We show that the structures of the heterodimers bound to either ligand are nearly identical and that in both cases the HER2/HER4 heterodimer interface is less dynamic than those observed in structures of HER2/EGFR and HER2/HER3 heterodimers. In contrast, structures of full-length HER4 homodimers bound to NRG1b and BTC display more large-scale dynamics mirroring states previously reported for EGFR homodimers. Our structures also reveal the presence of multiple glycan modifications within HER4 ectodomains, modeled for the first time in HER receptors, that distinctively contribute to the stabilization of HER4 homodimer interfaces over those of HER2/HER4 heterodimers.\nDeep mutational scanning of Pneumocystis jirovecii dihydrofolate reductase reveals allosteric mechanism of resistance to an antifolate\nAuthors: Rouleau, F. D.; Dube, A. K.; Gagnon-Arsenault, I.; Dibyachintan, S.; Pageau, A.; Despres, P. C.; Lagüe, P.; Landry, C. R.\nScore: 6.3, Published: 2023-09-28 DOI: 10.1101/2023.09.27.559786\nPneumocystis jirovecii is a fungal pathogen that causes pneumocystis pneumonia, a disease that mainly affects immunocompromised individuals. This fungus has historically been hard to study because of our inability to grow it in vitro. One of the main drug targets in P. jirovecii is its dihydrofolate reductase (PjDHFR). Here, by using functional complementation of the bakers yeast ortholog, we show that PjDHFR can be inhibited by the antifolate methotrexate in a dose-dependent manner. Using deep mutational scanning of PjDHFR, we identify mutations conferring resistance to methotrexate. Thirty-one sites spanning the protein have at least one mutation that leads to resistance, for a total of 355 high-confidence resistance mutations. Most resistance-inducing mutations are found inside the active site, and many are structurally equivalent to mutations known to lead to resistance to different antifolates in other organisms. Some sites show specific resistance mutations, where only a single substitution confers resistance, whereas others are more permissive, as several substitutions at these sites confer resistance. Surprisingly, one of the permissive sites (F199) is without direct contact to either ligand or cofactor, suggesting that it acts through an allosteric mechanism. Modeling changes in binding energy between F199 mutants and drug shows that most mutations destabilize interactions between the protein and the drug. This evidence points towards a more important role of this position in resistance than previously estimated and highlights potential unknown allosteric mechanisms of resistance to antifolate in DHFRs. Our results offer unprecedented resources for the interpretation of mutation effects in the main drug target of an uncultivable fungal pathogen.\n",
  "wordCount" : "2540",
  "inLanguage": "en",
  "datePublished": "2023-10-08T10:37:16Z",
  "dateModified": "2023-10-08T10:37:16Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta"><span>updated on October 8, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.560713">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.560713" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.560713">
        <p class="paperTitle">Improving protein expression, stability, and function with ProteinMPNN</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.560713" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.560713" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sumida, K. H.; Nunez Franco, R.; Kalvet, I.; Pellock, S. J.; Wicky, B. I. M.; Milles, L. F.; Dauparas, J.; Wang, J.; Kipnis, Y.; Jameson, N.; Kang, A.; De La Cruz, J.; Sankaran, B.; Bera, A. K.; Jimenez Oses, G.; Baker, D.</p>
        <p class="info">Score: 85.5, Published: 2023-10-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.560713' target='https://doi.org/10.1101/2023.10.03.560713'> 10.1101/2023.10.03.560713</a></p>
        <p class="abstract">Natural proteins are highly optimized for function, but are often difficult to produce at a scale suitable for biotechnological applications due to poor expression in heterologous systems, limited solubility, and sensitivity to temperature. Thus, a general method that improves the physical properties of native proteins while maintaining function could have wide utility for protein-based technologies. Here we show that the deep neural network ProteinMPNN together with evolutionary and structural information provides a route to increasing protein expression, stability, and function. For both myoglobin and tobacco etch virus (TEV) protease, we generated designs with improved expression, elevated melting temperatures, and improved function. For TEV protease, we identified multiple designs with improved catalytic activity as compared to the parent sequence and previously reported TEV variants. Our approach should be broadly useful for improving the expression, stability, and function of biotechnologically important proteins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560943">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560943" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560943">
        <p class="paperTitle">Targeted Protein Relocalization via Protein Transport Coupling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560943" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560943" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ng, C. S. C.; Liu, A.; Cui, B.; Banik, S. M.</p>
        <p class="info">Score: 37.5, Published: 2023-10-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560943' target='https://doi.org/10.1101/2023.10.04.560943'> 10.1101/2023.10.04.560943</a></p>
        <p class="abstract">Subcellular protein localization regulates protein function and can be corrupted in cancers and neurodegenerative diseases. The localization of numerous proteins has been annotated, and pharmacologically relevant approaches for precise rewiring of localization to address disease-driving phenotypes would be an attractive targeted therapeutic approach. Molecules which harness the trafficking of a shuttle protein to control the subcellular localization of a target protein could provide an avenue for targeted protein relocalization for interactome-rewiring therapeutics. To realize this concept, we deploy a quantitative approach to identify features which govern the ability to hijack protein trafficking, develop a collection of shuttle proteins and ligands, and demonstrate relocalization of proteins bearing endogenous localization signals. Using a custom imaging analysis pipeline, we show that endogenous localization signals can be overcome through molecular coupling of target proteins to shuttle proteins containing sufficiently strong native localization sequences expressed in the necessary abundance. We develop nuclear hormone receptors as viable shuttles which can be harnessed with Targeted Relocalization Activating Molecules (TRAMs) to redistribute disease-driving mutant proteins such as SMARCB1Q318X, TDP43, and FUSR495X. Small molecule-mediated relocalization of FUSR495X to the nucleus from the cytoplasm reduced the number of cellular stress granules in a model of cellular stress. Using Cas9-mediated knock-in tagging, we demonstrate nuclear enrichment of both low abundance (FOXO3a) and high abundance (FKBP12) endogenous proteins via molecular coupling to nuclear hormone receptor trafficking. Finally, small molecule-mediated redistribution of NMNAT1 from nuclei to axons in primary neurons was able to slow axonal degeneration and pharmacologically mimic the WldS gain-of-function phenotype from mice resistant to certain types of neurodegeneration8. The concept of targeted protein relocalization could therefore nucleate approaches for treating disease through interactome rewiring.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.559826">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.559826" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.559826">
        <p class="paperTitle">Mapping the biochemical landscape of rubisco</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.559826" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.559826" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Prywes, N.; Phillips, N. R.; Oltrogge, L. M.; de Pins, B.; Cowan, A. E.; Taylor-Kearney, L. J.; Chang, H. A.; Hall, L. N.; Bhatt, A.; Shih, P.; Milo, R.; Savage, D. F.</p>
        <p class="info">Score: 21.1, Published: 2023-09-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.559826' target='https://doi.org/10.1101/2023.09.27.559826'> 10.1101/2023.09.27.559826</a></p>
        <p class="abstract">The enzyme rubisco catalyzes the first step of carbon assimilation in photosynthesis. Despite the massive flux of CO2 passing through this active site over billions of years, extant rubisco has relatively slow kinetics and is prone to off-target activity. In many growth regimes, this limits photosynthesis in planta. Many attempts have been made to improve the kinetic parameters of rubisco with limited success, potentially due to biochemical trade-offs. To understand the structural basis of constraints on rubisco, a comprehensive map of rubisco at the individual amino-acid level is needed. To that end we performed a deep mutational scan using a rubisco-dependent E. coli strain. By titrating CO2 concentrations it was possible to determine estimations for both catalytic rate (kcat) and substrate affinity (KM) of &gt;99% of rubisco point mutants. Some positions were found to act as &#34;rheostats&#34; where some amino-acid substitutions reduced -while others improved -affinity for CO2. No individual point mutation was found to substantially improve the catalytic rate, but a number of highly phylogenetically conserved positions were found to tolerate mutations, indicating that a large portion of rubiscos sequence space remains unexplored by nature and may serve as a resource for future protein engineering efforts. Together, these biochemical measurements inform our understanding of biochemical tradeoffs and will assist in future efforts to improve rubisco catalytic properties.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.559794">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.559794" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.559794">
        <p class="paperTitle">Measuring carbohydrate recognition profile of lectins on live cells using liquid glycan array (LiGA)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.559794" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.559794" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sojitra, M.; Schmidt, E. N.; Lima, G. M.; Carpenter, E. J.; McCord, K. A.; Atrazhev, A.; Macauley, M. S.; Derda, R.</p>
        <p class="info">Score: 11.9, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.559794' target='https://doi.org/10.1101/2023.10.04.559794'> 10.1101/2023.10.04.559794</a></p>
        <p class="abstract">Glycans constitute a significant fraction of biomolecular diversity on the surface of cells across all the species in all kingdoms of life. As the structure of glycans is not encoded by the DNA of the host organisms, it is impossible to use cutting-edge DNA technology to study the role of cellular glycosylation or to understand how cell-surface glycome is recognized by glycan-binding proteins (GBPs). To address this gap, we recently described a genetically-encoded liquid glycan array (LiGA) platform that allows profiling of glycan:GBP interactions on the surface of live cells in vitro and in vivo using next-generation sequencing (NGS). LiGA is a library of DNA-barcoded bacteriophages coated with 5-1500 copies of a glycan; the DNA barcode inside each bacteriophage encodes the structure and density of the displayed glycans. Deep sequencing of the glycophages associated with live cells yields a glycan-binding profile of GBPs displayed on the surface of such cells. This protocol provides detailed instructions of using LiGA to probe cell surface receptors and includes information on the preparation of glycophages, analysis by MALDI-TOF MS, the assembly of a LiGA library, and its deep-sequencing. Using the protocol detailed in this report, we measure a glycan-binding profile of the immunomodulatory Siglec-1, 2, -6, -7, and -9 expressed on the surface of different cell types and uncover previously unknown environment-dependent recognition of glycans by Siglec-receptors on the surface of live cells. Protocols similar to the one described in this report will make it possible to measure the precise glycan-binding profile of any GPBs displayed on the surface of any cell types.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560451">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560451" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560451">
        <p class="paperTitle">Computational prediction of structure, function and interaction of aphid salivary effector proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560451" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560451" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Waksman, T.; Astin, E.; Fisher, S. R.; Hunter, W.; Bos, J. I.</p>
        <p class="info">Score: 11.4, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560451' target='https://doi.org/10.1101/2023.10.02.560451'> 10.1101/2023.10.02.560451</a></p>
        <p class="abstract">Similar to plant pathogens, phloem-feeding insects such as aphids deliver effector proteins inside their hosts that act to promote host susceptibility and enable feeding and infestation. Despite exciting progress towards identifying and characterizing effector proteins from these insects, their functions remain largely unknown. The recent ground-breaking development in protein structure prediction algorithms combined with the availability of proteomics and transcriptomic datasets for agriculturally important pests, such as the aphid Myzus persicae (green peach aphid), provides new opportunities to explore the structural and functional diversity of effector repertoires. In this study, we sought to gain insight into the the M. persicae effector repertoire by predicting and analysing the structures of a set of 71 effector candidate proteins. We used two protein structure prediction methods, AlphaFold and OmegaFold, which produced mutually consistent results. We observed a wide continuous spectrum of sizes and structures among the effector candidates, from disordered proteins to globular enzymes. We made use of the structural information and state-of-the-art computational methods to predict M. persicae effector protein properties, including function and interaction with host plant proteins. Overall, our investigation provides novel insights into the structure, function, and interaction prediction of aphid effector repertoires and will guide the necessary experimental characterization to address new hypotheses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.558399">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.558399" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.558399">
        <p class="paperTitle">PROxAb Shuttle: A non-covalent plug-and-play platform for the rapid generation of tumor-targeting antibody-PROTAC conjugates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.558399" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.558399" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schneider, H.; Jaeger, S.; Koenning, D.; Rasche, N.; Schroeter, C.; Elter, D.; Evers, A.; Lecomte, M.; Riccardi Sirtori, F.; Schwarz, D.; Wegener, A.; Hartung, I.; Rieker, M.</p>
        <p class="info">Score: 21.5, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.558399' target='https://doi.org/10.1101/2023.09.29.558399'> 10.1101/2023.09.29.558399</a></p>
        <p class="abstract">Proteolysis-targeting chimeras (PROTACs) have evolved in recent years from an academic idea to a therapeutic modality with more than 25 active clinical programs. However, achieving oral bioavailability and cell-type specificity remains a challenge, especially for PROTACs recruiting the von Hippel-Lindau (VHL) E3 ligase. Herein, we present an unprecedented, plug- and-play platform for VHL-recruiting PROTACs to overcome these limitations. Our platform allows for the generation of non-covalent antibody-PROTAC complexes within minutes and obviates the need for prior PROTAC modification, antibody-drug linker chemistry optimization or bioconjugation. Our technology relies on camelid-derived antibody domains (VHHs) which can easily be engineered into existing therapeutic antibody scaffolds. The resulting targeted, bispecific fusion proteins can be complexed with PROTACs at defined PROTAC-to-antibody ratios and have been termed PROxAb Shuttles. PROxAb Shuttles can prolong the half-life of PROTACs from hours to days, demonstrate anti-tumor efficacy in vivo and have the potential for reloading in vivo to further boost efficacy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.561059">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.561059" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.561059">
        <p class="paperTitle">Structural and molecular basis of choline uptake into the brain by FLVCR2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.561059" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.561059" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cater, R. J.; Mukherjee, D.; Gil Iturbe, E.; Erramilli, S. K.; Chen, T.; Koo, K.; Santander, N. G.; Reckers, A. R.; Kloss, B.; Gawda, T.; Choy, B. C.; Zhang, Z.; Clarke, O. B.; Yee, S. W.; Kossiakoff, A. K.; Quick, M.; Arnold, T.; Mancia, F.</p>
        <p class="info">Score: 8.3, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.561059' target='https://doi.org/10.1101/2023.10.05.561059'> 10.1101/2023.10.05.561059</a></p>
        <p class="abstract">Choline is an essential nutrient that the human body needs in vast quantities for cell membrane synthesis, epigenetic modification, and neurotransmission. The brain has a particularly high demand for choline, but how it enters the brain has eluded the field for over fifty years. The MFS transporter FLVCR1 was recently determined to be a choline transporter, and while this protein is not highly expressed at the blood-brain barrier (BBB), its relative FLVCR2 is. Previous studies have shown that mutations in human Flvcr2 cause cerebral vascular abnormalities, hydrocephalus, and embryonic lethality, but the physiological role of FLVCR2 is unknown. Here, we demonstrate both in vivo and in vitro that FLVCR2 is a BBB choline transporter and is responsible for the majority of choline uptake into the brain. We also determine the structures of choline-bound FLVCR2 in the inward- and outward-facing states using cryo-electron microscopy to 2.49 and 2.77 Angstrom resolution, respectively. These results reveal how the brain obtains choline and provide molecular-level insights into how FLVCR2 binds choline in an aromatic cage and mediates its uptake. Our work could provide a novel framework for the targeted delivery of neurotherapeutics into the brain.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560389">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560389" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560389">
        <p class="paperTitle">Identification and structural characterization of small-molecule inhibitors of PINK1</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560389" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560389" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rasool, S.; Shomali, T.; Truong, L.; Croteau, N.; Veyron, S.; Trempe, J.-F.</p>
        <p class="info">Score: 7.0, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560389' target='https://doi.org/10.1101/2023.10.02.560389'> 10.1101/2023.10.02.560389</a></p>
        <p class="abstract">Mutations in PTEN-induced putative kinase 1 (PINK1) cause early-onset autosomal recessive Parkinsons Disease (PD). PINK1 is a Ser/Thr protein kinase which functions as a mitochondrial damage sensor and initiates mitochondrial quality control by accumulating on the damaged organelle. There, it will autophosphorylate and then phosphorylate ubiquitin chains, which in turn will recruit and activate Parkin, and E3 ubiquitin ligase also implicated in PD. Ubiquitylation of mitochondrial proteins leads to the autophagic degradation of the damaged organelle. Pharmacological modulation of PINK1 constitutes an appealing avenue to study its physiological function and develop PD therapeutics. In this study, we used a thermal shift assay to identify small-molecule inhibitors of PINK1. In vitro kinase activity assays demonstrate that these molecules are ATP competitive inhibitors that block ubiquitin phosphorylation. PRT062607 (a SYK inhibitor) is the most potent inhibitor of PINK1 in our screen and has an IC50 of 2 M against insect PINK1 and 1 M in HeLa cells expressing human PINK1. The crystal structures of PINK1 in complex with PRT062607 or CYC116 reveal how the compounds interact with the ATP-binding pocket. PRT062607 notably engages with the catalytic aspartate (type-1 inhibition) and causes a destabilization of insert-2 at the autophosphorylation dimer interface. Our findings provide a scaffold for the development of more selective and potent inhibitors of PINK1 that can be used as chemical probes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561161">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561161" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561161">
        <p class="paperTitle">Structural dynamics of the active HER4 and HER2/HER4 complexes is finely tuned by different growth factors and glycosylation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561161" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561161" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Trenker, R.; Diwanji, D.; Bingham, T.; Verba, K. A.; Jura, N.</p>
        <p class="info">Score: 4.8, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561161' target='https://doi.org/10.1101/2023.10.06.561161'> 10.1101/2023.10.06.561161</a></p>
        <p class="abstract">Human Epidermal growth factor Receptor 4 (HER4) carries out essential functions in the development and maintenance of the cardiovascular and nervous systems. HER4 activation is regulated by a diverse group of extracellular ligands including the neuregulin (NRG) family and betacellulin (BTC), which promote HER4 homodimerization or heterodimerization with other HER receptors. Important cardiovascular functions of HER4 are exerted via heterodimerization with its close homolog and orphan receptor, HER2. To date structural insights into ligand-mediated HER4 activation have been limited to crystallographic studies of HER4 ectodomain homodimers in complex with NRG1b. Here we report cryo-EM structures of near full-length HER2/HER4 heterodimers and full-length HER4 homodimers bound to NRG1b and BTC. We show that the structures of the heterodimers bound to either ligand are nearly identical and that in both cases the HER2/HER4 heterodimer interface is less dynamic than those observed in structures of HER2/EGFR and HER2/HER3 heterodimers. In contrast, structures of full-length HER4 homodimers bound to NRG1b and BTC display more large-scale dynamics mirroring states previously reported for EGFR homodimers. Our structures also reveal the presence of multiple glycan modifications within HER4 ectodomains, modeled for the first time in HER receptors, that distinctively contribute to the stabilization of HER4 homodimer interfaces over those of HER2/HER4 heterodimers.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.559786">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.559786" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.559786">
        <p class="paperTitle">Deep mutational scanning of Pneumocystis jirovecii dihydrofolate reductase reveals allosteric mechanism of resistance to an antifolate</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.559786" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.559786" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rouleau, F. D.; Dube, A. K.; Gagnon-Arsenault, I.; Dibyachintan, S.; Pageau, A.; Despres, P. C.; Lagüe, P.; Landry, C. R.</p>
        <p class="info">Score: 6.3, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.559786' target='https://doi.org/10.1101/2023.09.27.559786'> 10.1101/2023.09.27.559786</a></p>
        <p class="abstract">Pneumocystis jirovecii is a fungal pathogen that causes pneumocystis pneumonia, a disease that mainly affects immunocompromised individuals. This fungus has historically been hard to study because of our inability to grow it in vitro. One of the main drug targets in P. jirovecii is its dihydrofolate reductase (PjDHFR). Here, by using functional complementation of the bakers yeast ortholog, we show that PjDHFR can be inhibited by the antifolate methotrexate in a dose-dependent manner. Using deep mutational scanning of PjDHFR, we identify mutations conferring resistance to methotrexate. Thirty-one sites spanning the protein have at least one mutation that leads to resistance, for a total of 355 high-confidence resistance mutations. Most resistance-inducing mutations are found inside the active site, and many are structurally equivalent to mutations known to lead to resistance to different antifolates in other organisms. Some sites show specific resistance mutations, where only a single substitution confers resistance, whereas others are more permissive, as several substitutions at these sites confer resistance. Surprisingly, one of the permissive sites (F199) is without direct contact to either ligand or cofactor, suggesting that it acts through an allosteric mechanism. Modeling changes in binding energy between F199 mutants and drug shows that most mutations destabilize interactions between the protein and the drug. This evidence points towards a more important role of this position in resistance than previously estimated and highlights potential unknown allosteric mechanisms of resistance to antifolate in DHFRs. Our results offer unprecedented resources for the interpretation of mutation effects in the main drug target of an uncultivable fungal pathogen.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
