<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Design of a Cereblon construct for crystallographic and biophysical studies of protein degraders
Authors: Kroupova, A.; Spiteri, V. A.; Furihata, H.; Darren, D.; Ramachandran, S.; Rutter, Z.; Chakraborti, S.; Haubrich, K.; Pethe, J.; Gonzales, D.; Wijaya, A.; Rodriguez-Rios, M.; Lynch, D. M.; Farnaby, W.; Nakasone, M.; Zollman, D.; Ciulli, A.
Score: 25.5, Published: 2024-01-20 DOI: 10.1101/2024.01.17.575503
The ubiquitin E3 ligase cereblon (CRBN) is the target of therapeutic drugs thalidomide and lenalidomide and is recruited by most targeted protein degraders (PROTACs and molecular glues) in clinical development.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Design of a Cereblon construct for crystallographic and biophysical studies of protein degraders
Authors: Kroupova, A.; Spiteri, V. A.; Furihata, H.; Darren, D.; Ramachandran, S.; Rutter, Z.; Chakraborti, S.; Haubrich, K.; Pethe, J.; Gonzales, D.; Wijaya, A.; Rodriguez-Rios, M.; Lynch, D. M.; Farnaby, W.; Nakasone, M.; Zollman, D.; Ciulli, A.
Score: 25.5, Published: 2024-01-20 DOI: 10.1101/2024.01.17.575503
The ubiquitin E3 ligase cereblon (CRBN) is the target of therapeutic drugs thalidomide and lenalidomide and is recruited by most targeted protein degraders (PROTACs and molecular glues) in clinical development." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-28T10:37:59+00:00" />
<meta property="article:modified_time" content="2024-01-28T10:37:59+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Design of a Cereblon construct for crystallographic and biophysical studies of protein degraders
Authors: Kroupova, A.; Spiteri, V. A.; Furihata, H.; Darren, D.; Ramachandran, S.; Rutter, Z.; Chakraborti, S.; Haubrich, K.; Pethe, J.; Gonzales, D.; Wijaya, A.; Rodriguez-Rios, M.; Lynch, D. M.; Farnaby, W.; Nakasone, M.; Zollman, D.; Ciulli, A.
Score: 25.5, Published: 2024-01-20 DOI: 10.1101/2024.01.17.575503
The ubiquitin E3 ligase cereblon (CRBN) is the target of therapeutic drugs thalidomide and lenalidomide and is recruited by most targeted protein degraders (PROTACs and molecular glues) in clinical development."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Design of a Cereblon construct for crystallographic and biophysical studies of protein degraders\nAuthors: Kroupova, A.; Spiteri, V. A.; Furihata, H.; Darren, D.; Ramachandran, S.; Rutter, Z.; Chakraborti, S.; Haubrich, K.; Pethe, J.; Gonzales, D.; Wijaya, A.; Rodriguez-Rios, M.; Lynch, D. M.; Farnaby, W.; Nakasone, M.; Zollman, D.; Ciulli, A.\nScore: 25.5, Published: 2024-01-20 DOI: 10.1101/2024.01.17.575503\nThe ubiquitin E3 ligase cereblon (CRBN) is the target of therapeutic drugs thalidomide and lenalidomide and is recruited by most targeted protein degraders (PROTACs and molecular glues) in clinical development.",
  "keywords": [
    
  ],
  "articleBody": " Design of a Cereblon construct for crystallographic and biophysical studies of protein degraders\nAuthors: Kroupova, A.; Spiteri, V. A.; Furihata, H.; Darren, D.; Ramachandran, S.; Rutter, Z.; Chakraborti, S.; Haubrich, K.; Pethe, J.; Gonzales, D.; Wijaya, A.; Rodriguez-Rios, M.; Lynch, D. M.; Farnaby, W.; Nakasone, M.; Zollman, D.; Ciulli, A.\nScore: 25.5, Published: 2024-01-20 DOI: 10.1101/2024.01.17.575503\nThe ubiquitin E3 ligase cereblon (CRBN) is the target of therapeutic drugs thalidomide and lenalidomide and is recruited by most targeted protein degraders (PROTACs and molecular glues) in clinical development. Biophysical and structural investigation of CRBN has been limited by current constructs that either require co-expression with the adaptor DDB1 or inadequately represent full-length protein, with high-resolution structures of degraders ternary complexes remaining rare. We present the design of CRBNmidi, a construct that readily expresses from E. coli with high yields as soluble, stable protein without DDB1. We benchmark CRBNmidi for wild-type functionality through a suite of biophysical techniques and solve high-resolution co-crystal structures of its binary and ternary complexes with degraders. We qualify CRBNmidi as an enabling tool to accelerate structure-based discovery of the next generation of CRBN based therapeutics. One sentence summaryA novel Cereblon construct (CRBNmidi) allows structural and biophysical enablement of ligand and degrader design\nByeTAC: Bypassing an E3 Ligase for Targeted Protein Degradation\nAuthors: Ali, E. M.; Loy, C.; Trader, D. J.\nScore: 25.0, Published: 2024-01-21 DOI: 10.1101/2024.01.20.576376\nTargeted protein degradation utilizing a bifunctional molecule to initiate ubiquitination and subsequent degradation by the 26S proteasome has been shown to be a powerful therapeutic intervention. Many bifunctional molecules, including covalent and non-covalent ligands to proteins of interest, have been developed. The traditional target protein degradation methodology targets the protein of interest in both healthy and diseased cell populations, and a therapeutic window is obtained based on the overexpression of the targeted protein. We report here a series of bifunctional degraders that do not rely on interacting with an E3 ligase, but rather a 26S proteasome subunit, which we have named ByeTACs: Bypassing E3 Targeting Chimeras. Rpn-13 is a non-essential ubiquitin receptor for the 26S proteasome. Cells under significant stress or require significant ubiquitin-dependent degradation of proteins for survival, incorporate Rpn-13 in the 26S to increase protein degradation rates. The targeted protein degraders reported here are bifunctional molecules that include a ligand to Rpn-13 and BRD4, the protein of interest we wish to degrade. We synthesized a suite of degraders with varying PEG chain lengths and showed that bifunctional molecules that incorporate a Rpn-13 binder (TCL1) and a BRD4 binder (JQ1) with a PEG linker of 3 or 4 units are the most effective to induce BRD4 degradation. We also demonstrate that our new targeted protein degraders are dependent upon proteasome activity and Rpn-13 expression levels. This establishes a new mechanism of action for our ByeTACs that can be employed for the targeted degradation of a wide variety of protein substrates.\nExpanding the ligandable proteome by paralog hopping with covalent probes\nAuthors: Zhang, Y.; Liu, Z.; Hirschi, M.; Brodsky, O.; Johnson, E.; Won, S. J.; Nagata, A.; Petroski, M. D.; Majmudar, J. D.; Niessen, S.; VanArsdale, T.; Gilbert, A. M.; Hayward, M. M.; Stewart, A. E.; Nager, A. R.; Melillo, B.; Cravatt, B. F.\nScore: 29.5, Published: 2024-01-20 DOI: 10.1101/2024.01.18.576274\nMore than half of the [~]20,000 protein-encoding human genes have at least one paralog. Chemical proteomics has uncovered many electrophile-sensitive cysteines that are exclusive to a subset of paralogous proteins. Here, we explore whether such covalent compound-cysteine interactions can be used to discover ligandable pockets in paralogs that lack the cysteine. Leveraging the covalent ligandability of C109 in the cyclin CCNE2, we mutated the corresponding residue in paralog CCNE1 to cysteine (N112C) and found through activity-based protein profiling (ABPP) that this mutant reacts stereoselectively and site-specifically with tryptoline acrylamides. We then converted the tryptoline acrylamide-N112C-CCNE1 interaction into a NanoBRET-ABPP assay capable of identifying compounds that reversibly inhibit both N112C- and WT-CCNE1:CDK2 complexes. X-ray crystallography revealed a cryptic allosteric pocket at the CCNE1:CDK2 interface adjacent to N112 that binds the reversible inhibitors. Our findings thus provide a roadmap for leveraging electrophile-cysteine interactions to extend the ligandability of the proteome beyond covalent chemistry.\nAutomated single-cell proteomics providing sufficient proteome depth to study complex biology beyond cell type classifications\nAuthors: Ctortecka, C.; Clark, N. M.; Boyle, B.; Seth, A.; Mani, D. R.; Udeshi, N.; Carr, S. A.\nScore: 16.0, Published: 2024-01-22 DOI: 10.1101/2024.01.20.576369\nMass spectrometry (MS)-based single-cell proteomics (SCP) has gained massive attention as a viable complement to other single cell approaches. The rapid technological and computational advances in the field have pushed the boundaries of sensitivity and throughput. However, reproducible quantification of thousands of proteins within a single cell at reasonable proteome depth to characterize biological phenomena remains a challenge. To address some of those limitations we present a combination of fully automated single cell sample preparation utilizing a dedicated chip within the picolitre dispensing robot, the cellenONE. The proteoCHIP EVO 96 can be directly interfaced with the Evosep One chromatographic system for in-line desalting and highly reproducible separation with a throughput of 80 samples per day. This, in combination with the Bruker timsTOF MS instruments, demonstrates double the identifications without manual sample handling. Moreover, relative to standard high-performance liquid chromatography, the Evosep One separation provides further 2-fold improvement in protein identifications. The implementation of the newest generation timsTOF Ultra with our proteoCHIP EVO 96-based sample preparation workflow reproducibly identifies up to 4,000 proteins per single HEK-293T without a carrier or match-between runs. Our current SCP depth spans over 4 orders of magnitude and identifies over 50 biologically relevant ubiquitin ligases. We complement our highly reproducible single-cell proteomics workflow to profile hundreds of lipopolysaccharide (LPS)-perturbed THP-1 cells and identified key regulatory proteins involved in interleukin and interferon signaling. This study demonstrates that the proteoCHIP EVO 96-based SCP sample preparation with the timsTOF Ultra provides sufficient proteome depth to study complex biology beyond cell-type classifications.\nMechanism of chaperone coordination during cotranslational protein folding in bacteria\nAuthors: Roeselova, A.; Maslen, S. L.; Shivakumaraswamy, S.; Pellowe, G. A.; Howell, S.; Joshi, D.; Redmond, J.; Kjaer, S.; Skehel, M.; Balchin, D.\nScore: 15.7, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576655\nProtein folding is assisted by molecular chaperones that bind nascent polypeptides during mRNA translation. Several structurally-distinct classes of chaperone promote de novo folding, suggesting that their activities are coordinated at the ribosome. We used biochemical reconstitution and structural proteomics to explore the molecular basis for cotranslational chaperone action in bacteria. We found that chaperone binding is disfavoured close to the ribosome, allowing folding to precede chaperone recruitment. Trigger factor subsequently recognises compact folding intermediates exposing extensive non-native surface and dictates DnaJ access to nascent chains. DnaJ uses a large surface to bind structurally diverse intermediates, and recruits DnaK to sequence-diverse solvent-accessible sites. Neither Trigger factor, DnaJ nor DnaK destabilize cotranslational folding intermediates. Instead, the chaperones collaborate to create a protected space for protein maturation that extends well beyond the ribosome exit tunnel. Our findings show how the chaperone network selects and modulates cotranslational folding intermediates.\nIn-Depth Mapping of DNA-PKcs Signaling Uncovers Conserved Features of Its Kinase Specificity\nAuthors: Marshall, S.; Navarro, M.; Ascencao, C.; Smolka, M. B.\nScore: 8.9, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576037\nDNA-PKcs is a DNA damage sensor kinase with established roles in DNA double-strand break repair via non-homologous end joining. Recent studies have revealed additional roles of DNA-PKcs in the regulation of transcription, translation and DNA replication. However, the substrates through which DNA-PKcs regulates these processes remain largely undefined. Here we utilized quantitative phosphoproteomics to generate a high coverage map of DNA-PKcs signaling in response to ionizing radiation and mapped its interplay with the ATM kinase. Beyond the detection of the canonical S/T-Q phosphorylation motif, we uncovered a non-canonical mode of DNA-PKcs signaling targeting S/T-{psi}-D/E motifs. Cross-species analysis in mouse pre-B and human HCT116 cell lines revealed splicing factors and transcriptional regulators phosphorylated at this novel motif, several of which contain SAP domains. These findings expand the list of DNA-PKcs and ATM substrates and establish a novel preferential phosphorylation motif for DNA-PKcs that connects it to proteins involved in nucleotide processes and interactions.\nKinases in motion: impact of protein and small molecule interactions on kinase conformations\nAuthors: Kugler, V.; Schwaighofer, S.; Feichtner, A.; Enzler, F.; Fleischmann, J.; Strich, S.; Schwarz, S.; Wilson, R.; Tschaikner, P.; Troppmair, J.; Sexl, V.; Meier, P.; Kaserer, T.; Stefan, E.\nScore: 10.2, Published: 2024-01-18 DOI: 10.1101/2024.01.11.575270\nProtein kinases act as central molecular switches in the control of cellular functions. Alterations in the regulation and function of protein kinases may provoke diseases including cancer. In this study we investigate the conformational states of such disease-associated kinases using the high sensitivity of the Kinase Conformation (KinCon)-reporter system. We first tracked BRAF-kinase activity conformation changes upon melanoma drug binding. Second, we also use the KinCon reporter technology to examine the impact of regulatory protein interactions on LKB1-kinase tumor suppressor functions. Third, we explore the conformational dynamics of RIP-kinases in response to TNF-pathway activation and small molecule interactions. Finally, we show that CDK4/6 interactions with regulatory proteins alter conformations which remain unaffected in the presence of clinically applied inhibitors. Apart from its predictive value, the KinCon technology helps identify cellular factors that impact drug efficacies. The understanding of the dynamics of full-length protein kinases when interacting with small molecule inhibitors or regulatory proteins is crucial for designing more effective therapeutic strategies.\nLeveraging co-evolutionary insights and AI-based structural modeling to unravel receptor-peptide ligand-binding mechanisms\nAuthors: Snoeck, S.; Lee, H. K.; Schmid, M. W.; Bender, K. W.; Neeracher, M. J.; Fernandez-Fernandez, A. D.; Santiago, J.; Zipfel, C.\nScore: 32.2, Published: 2024-01-18 DOI: 10.1101/2024.01.18.575556\nSecreted signaling peptides are central regulators of growth, development, and stress responses, but specific steps in the evolution of these peptides and their receptors are not well understood. In addition, the molecular mechanisms of peptide-receptor binding are only known for a few examples, primarily owing to the limited availability of structural capabilities to few laboratories worldwide. Plants have evolved a multitude of secreted signaling peptides and corresponding transmembrane receptors. Stress-responsive SERINE RICH ENDOGENOUS PEPTIDES (SCOOPs) were recently identified. Bioactive SCOOPs are proteolytically processed by subtilases and are perceived by the leucine-rich repeat receptor kinase MALE DISCOVERER 1-INTERACTING RECEPTOR-LIKE KINASE 2 (MIK2) in the model plant Arabidopsis thaliana. How SCOOPs and MIK2 have (co-)evolved, and how SCOOPs bind to MIK2 are however still unknown. Using in silico analysis of 350 plant genomes and subsequent functional testing, we revealed the conservation of MIK2 as SCOOP receptor within the plant order Brassicales. We then leveraged AlphaFold-Multimer and comparative genomics to identify two conserved putative SCOOP-MIK2 binding pockets across Brassicales MIK2 homologues predicted to interact with the SxS motif of otherwise sequence-divergent SCOOPs. Notably, mutagenesis of both predicted binding pockets compromised SCOOP binding to MIK2, SCOOP-induced complex formation between MIK2 and its co-receptor BRASSINOSTEROID INSENSITIVE 1-ASSOCIATED KINASE 1 (BAK1), and SCOOP-induced reactive oxygen species production; thus, confirming our in silico predictions. Collectively, in addition to revealing the elusive SCOOP-MIK2 binding mechanisms, our analytic pipeline combining phylogenomics, AI-based structural predictions, and experimental biochemical and physiological validation provides a blueprint for the elucidation of peptide ligand-receptor perception mechanisms. Significance statementThis study presents a rapid and inexpensive alternative to classical structure-based approaches for resolving ligand-receptor binding mechanisms. It relies on a multilayered bioinformatic approach that leverages genomic data across diverse species in combination with AI-based structural modeling to identify true ligand and receptor homologues, and subsequently predict their binding mechanisms. In silico findings were validated by multiple experimental approaches, which investigated the effect of amino acid changes in the proposed binding pockets on ligand-binding, complex formation with a co-receptor essential for downstream signaling, and activation of downstream signaling. Our analysis combining evolutionary insights, in silico modeling and functional validation provides a framework for structure-function analysis of other peptide-receptor pairs, which could be easily implemented by most laboratories.\nDirect interrogation of context-dependent GPCR activity with a universal biosensor platform\nAuthors: Janicot, R.; Maziarz, M.; Park, J.-C.; Luebbers, A.; Green, E.; Zhao, J.; Philibert, C. E.; Zhang, H.; Layne, M. D.; Wu, J. C.; Garcia-Marcos, M.\nScore: 29.5, Published: 2024-01-02 DOI: 10.1101/2024.01.02.573921\nG protein-coupled receptors (GPCRs) are the largest family of druggable proteins in the human genome, but progress in understanding and targeting them is hindered by the lack of tools to reliably measure their nuanced behavior in physiologically-relevant contexts. Here, we developed a collection of compact ONE vector G-protein Optical (ONE-GO) biosensor constructs as a scalable platform that can be conveniently deployed to measure G-protein activation by virtually any GPCR with high fidelity even when expressed endogenously in primary cells. By characterizing dozens of GPCRs across many cell types like primary cardiovascular cells or neurons, we revealed new insights into the molecular basis for G-protein coupling selectivity of GPCRs, pharmacogenomic profiles of anti-psychotics on naturally-occurring GPCR variants, and G-protein subtype signaling bias by endogenous GPCRs depending on cell type or upon inducing disease-like states. In summary, this open-source platform makes the direct interrogation of context-dependent GPCR activity broadly accessible.\nPhenotypic screening of covalent compound libraries identifies chloromethyl ketone antibiotics and MiaA as a new target\nAuthors: Jin, Y.; Jana, S.; Abbasov, M.; Lin, H.\nScore: 5.6, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576576\nThe emerging antibiotic resistance requires the development of new antibiotics working on novel bacterial targets. Here, we reported an antibiotic discovery workflow by combining the cysteine-reactive compound library phenotypic screening with activity-based protein profiling, which enables the rapid identification of lead compounds as well as new druggable targets in pathogens. Compounds featuring chloromethyl ketone scaffolds exhibited a notably high hit rate against both gram-negative and gram-positive bacterial strains, but not the more commonly used warheads such as acrylamide or chloroacetamide. Target identification of the lead compound, 10-F05, revealed that its primary targets in S. flexneri are FabH Cys112 and MiaA Cys273. We validated the target relevance through biochemical and genetic interactions. Mechanistic studies revealed modification of MiaA by 10-F05 impair substrate tRNA binding, leading to decreased bacterial stress resistance and virulence. Our findings underscore chloromethyl ketone as a novel antibacterial warhead in covalent antibiotic design. The study showcases that combining covalent compound library phenotypic screening with chemoproteomics is an efficient way to identify new drug targets as well as lead compounds, with the potential to open new research directions in drug discovery and chemical biology. Graphic Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=121 SRC=\"FIGDIR/small/576576v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (21K): org.highwire.dtl.DTLVardef@1e76b4borg.highwire.dtl.DTLVardef@3509e7org.highwire.dtl.DTLVardef@1d1cccorg.highwire.dtl.DTLVardef@14da278_HPS_FORMAT_FIGEXP M_FIG C_FIG\n",
  "wordCount" : "2386",
  "inLanguage": "en",
  "datePublished": "2024-01-28T10:37:59Z",
  "dateModified": "2024-01-28T10:37:59Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 28, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.575503">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.575503" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.575503">
        <p class="paperTitle">Design of a Cereblon construct for crystallographic and biophysical studies of protein degraders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.575503" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.575503" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kroupova, A.; Spiteri, V. A.; Furihata, H.; Darren, D.; Ramachandran, S.; Rutter, Z.; Chakraborti, S.; Haubrich, K.; Pethe, J.; Gonzales, D.; Wijaya, A.; Rodriguez-Rios, M.; Lynch, D. M.; Farnaby, W.; Nakasone, M.; Zollman, D.; Ciulli, A.</p>
        <p class="info">Score: 25.5, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.575503' target='https://doi.org/10.1101/2024.01.17.575503'> 10.1101/2024.01.17.575503</a></p>
        <p class="abstract">The ubiquitin E3 ligase cereblon (CRBN) is the target of therapeutic drugs thalidomide and lenalidomide and is recruited by most targeted protein degraders (PROTACs and molecular glues) in clinical development. Biophysical and structural investigation of CRBN has been limited by current constructs that either require co-expression with the adaptor DDB1 or inadequately represent full-length protein, with high-resolution structures of degraders ternary complexes remaining rare. We present the design of CRBNmidi, a construct that readily expresses from E. coli with high yields as soluble, stable protein without DDB1. We benchmark CRBNmidi for wild-type functionality through a suite of biophysical techniques and solve high-resolution co-crystal structures of its binary and ternary complexes with degraders. We qualify CRBNmidi as an enabling tool to accelerate structure-based discovery of the next generation of CRBN based therapeutics.

One sentence summaryA novel Cereblon construct (CRBNmidi) allows structural and biophysical enablement of ligand and degrader design</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576376">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576376" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576376">
        <p class="paperTitle">ByeTAC: Bypassing an E3 Ligase for Targeted Protein Degradation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576376" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576376" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ali, E. M.; Loy, C.; Trader, D. J.</p>
        <p class="info">Score: 25.0, Published: 2024-01-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576376' target='https://doi.org/10.1101/2024.01.20.576376'> 10.1101/2024.01.20.576376</a></p>
        <p class="abstract">Targeted protein degradation utilizing a bifunctional molecule to initiate ubiquitination and subsequent degradation by the 26S proteasome has been shown to be a powerful therapeutic intervention. Many bifunctional molecules, including covalent and non-covalent ligands to proteins of interest, have been developed. The traditional target protein degradation methodology targets the protein of interest in both healthy and diseased cell populations, and a therapeutic window is obtained based on the overexpression of the targeted protein. We report here a series of bifunctional degraders that do not rely on interacting with an E3 ligase, but rather a 26S proteasome subunit, which we have named ByeTACs: Bypassing E3 Targeting Chimeras. Rpn-13 is a non-essential ubiquitin receptor for the 26S proteasome. Cells under significant stress or require significant ubiquitin-dependent degradation of proteins for survival, incorporate Rpn-13 in the 26S to increase protein degradation rates. The targeted protein degraders reported here are bifunctional molecules that include a ligand to Rpn-13 and BRD4, the protein of interest we wish to degrade. We synthesized a suite of degraders with varying PEG chain lengths and showed that bifunctional molecules that incorporate a Rpn-13 binder (TCL1) and a BRD4 binder (JQ1) with a PEG linker of 3 or 4 units are the most effective to induce BRD4 degradation. We also demonstrate that our new targeted protein degraders are dependent upon proteasome activity and Rpn-13 expression levels. This establishes a new mechanism of action for our ByeTACs that can be employed for the targeted degradation of a wide variety of protein substrates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576274">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576274" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576274">
        <p class="paperTitle">Expanding the ligandable proteome by paralog hopping with covalent probes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576274" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576274" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, Y.; Liu, Z.; Hirschi, M.; Brodsky, O.; Johnson, E.; Won, S. J.; Nagata, A.; Petroski, M. D.; Majmudar, J. D.; Niessen, S.; VanArsdale, T.; Gilbert, A. M.; Hayward, M. M.; Stewart, A. E.; Nager, A. R.; Melillo, B.; Cravatt, B. F.</p>
        <p class="info">Score: 29.5, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576274' target='https://doi.org/10.1101/2024.01.18.576274'> 10.1101/2024.01.18.576274</a></p>
        <p class="abstract">More than half of the [~]20,000 protein-encoding human genes have at least one paralog. Chemical proteomics has uncovered many electrophile-sensitive cysteines that are exclusive to a subset of paralogous proteins. Here, we explore whether such covalent compound-cysteine interactions can be used to discover ligandable pockets in paralogs that lack the cysteine. Leveraging the covalent ligandability of C109 in the cyclin CCNE2, we mutated the corresponding residue in paralog CCNE1 to cysteine (N112C) and found through activity-based protein profiling (ABPP) that this mutant reacts stereoselectively and site-specifically with tryptoline acrylamides. We then converted the tryptoline acrylamide-N112C-CCNE1 interaction into a NanoBRET-ABPP assay capable of identifying compounds that reversibly inhibit both N112C- and WT-CCNE1:CDK2 complexes. X-ray crystallography revealed a cryptic allosteric pocket at the CCNE1:CDK2 interface adjacent to N112 that binds the reversible inhibitors. Our findings thus provide a roadmap for leveraging electrophile-cysteine interactions to extend the ligandability of the proteome beyond covalent chemistry.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576369">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576369" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576369">
        <p class="paperTitle">Automated single-cell proteomics providing sufficient proteome depth to study complex biology beyond cell type classifications</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576369" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576369" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ctortecka, C.; Clark, N. M.; Boyle, B.; Seth, A.; Mani, D. R.; Udeshi, N.; Carr, S. A.</p>
        <p class="info">Score: 16.0, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576369' target='https://doi.org/10.1101/2024.01.20.576369'> 10.1101/2024.01.20.576369</a></p>
        <p class="abstract">Mass spectrometry (MS)-based single-cell proteomics (SCP) has gained massive attention as a viable complement to other single cell approaches. The rapid technological and computational advances in the field have pushed the boundaries of sensitivity and throughput. However, reproducible quantification of thousands of proteins within a single cell at reasonable proteome depth to characterize biological phenomena remains a challenge. To address some of those limitations we present a combination of fully automated single cell sample preparation utilizing a dedicated chip within the picolitre dispensing robot, the cellenONE. The proteoCHIP EVO 96 can be directly interfaced with the Evosep One chromatographic system for in-line desalting and highly reproducible separation with a throughput of 80 samples per day. This, in combination with the Bruker timsTOF MS instruments, demonstrates double the identifications without manual sample handling. Moreover, relative to standard high-performance liquid chromatography, the Evosep One separation provides further 2-fold improvement in protein identifications. The implementation of the newest generation timsTOF Ultra with our proteoCHIP EVO 96-based sample preparation workflow reproducibly identifies up to 4,000 proteins per single HEK-293T without a carrier or match-between runs. Our current SCP depth spans over 4 orders of magnitude and identifies over 50 biologically relevant ubiquitin ligases. We complement our highly reproducible single-cell proteomics workflow to profile hundreds of lipopolysaccharide (LPS)-perturbed THP-1 cells and identified key regulatory proteins involved in interleukin and interferon signaling. This study demonstrates that the proteoCHIP EVO 96-based SCP sample preparation with the timsTOF Ultra provides sufficient proteome depth to study complex biology beyond cell-type classifications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.576655">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.576655" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.576655">
        <p class="paperTitle">Mechanism of chaperone coordination during cotranslational protein folding in bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.576655" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.576655" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Roeselova, A.; Maslen, S. L.; Shivakumaraswamy, S.; Pellowe, G. A.; Howell, S.; Joshi, D.; Redmond, J.; Kjaer, S.; Skehel, M.; Balchin, D.</p>
        <p class="info">Score: 15.7, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.576655' target='https://doi.org/10.1101/2024.01.22.576655'> 10.1101/2024.01.22.576655</a></p>
        <p class="abstract">Protein folding is assisted by molecular chaperones that bind nascent polypeptides during mRNA translation. Several structurally-distinct classes of chaperone promote de novo folding, suggesting that their activities are coordinated at the ribosome. We used biochemical reconstitution and structural proteomics to explore the molecular basis for cotranslational chaperone action in bacteria. We found that chaperone binding is disfavoured close to the ribosome, allowing folding to precede chaperone recruitment. Trigger factor subsequently recognises compact folding intermediates exposing extensive non-native surface and dictates DnaJ access to nascent chains. DnaJ uses a large surface to bind structurally diverse intermediates, and recruits DnaK to sequence-diverse solvent-accessible sites. Neither Trigger factor, DnaJ nor DnaK destabilize cotranslational folding intermediates. Instead, the chaperones collaborate to create a protected space for protein maturation that extends well beyond the ribosome exit tunnel. Our findings show how the chaperone network selects and modulates cotranslational folding intermediates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.576037">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.576037" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.576037">
        <p class="paperTitle">In-Depth Mapping of DNA-PKcs Signaling Uncovers Conserved Features of Its Kinase Specificity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.576037" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.576037" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Marshall, S.; Navarro, M.; Ascencao, C.; Smolka, M. B.</p>
        <p class="info">Score: 8.9, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.576037' target='https://doi.org/10.1101/2024.01.17.576037'> 10.1101/2024.01.17.576037</a></p>
        <p class="abstract">DNA-PKcs is a DNA damage sensor kinase with established roles in DNA double-strand break repair via non-homologous end joining. Recent studies have revealed additional roles of DNA-PKcs in the regulation of transcription, translation and DNA replication. However, the substrates through which DNA-PKcs regulates these processes remain largely undefined. Here we utilized quantitative phosphoproteomics to generate a high coverage map of DNA-PKcs signaling in response to ionizing radiation and mapped its interplay with the ATM kinase. Beyond the detection of the canonical S/T-Q phosphorylation motif, we uncovered a non-canonical mode of DNA-PKcs signaling targeting S/T-{psi}-D/E motifs. Cross-species analysis in mouse pre-B and human HCT116 cell lines revealed splicing factors and transcriptional regulators phosphorylated at this novel motif, several of which contain SAP domains. These findings expand the list of DNA-PKcs and ATM substrates and establish a novel preferential phosphorylation motif for DNA-PKcs that connects it to proteins involved in nucleotide processes and interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575270">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575270" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575270">
        <p class="paperTitle">Kinases in motion: impact of protein and small molecule interactions on kinase conformations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575270" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575270" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kugler, V.; Schwaighofer, S.; Feichtner, A.; Enzler, F.; Fleischmann, J.; Strich, S.; Schwarz, S.; Wilson, R.; Tschaikner, P.; Troppmair, J.; Sexl, V.; Meier, P.; Kaserer, T.; Stefan, E.</p>
        <p class="info">Score: 10.2, Published: 2024-01-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575270' target='https://doi.org/10.1101/2024.01.11.575270'> 10.1101/2024.01.11.575270</a></p>
        <p class="abstract">Protein kinases act as central molecular switches in the control of cellular functions. Alterations in the regulation and function of protein kinases may provoke diseases including cancer. In this study we investigate the conformational states of such disease-associated kinases using the high sensitivity of the Kinase Conformation (KinCon)-reporter system. We first tracked BRAF-kinase activity conformation changes upon melanoma drug binding. Second, we also use the KinCon reporter technology to examine the impact of regulatory protein interactions on LKB1-kinase tumor suppressor functions. Third, we explore the conformational dynamics of RIP-kinases in response to TNF-pathway activation and small molecule interactions. Finally, we show that CDK4/6 interactions with regulatory proteins alter conformations which remain unaffected in the presence of clinically applied inhibitors. Apart from its predictive value, the KinCon technology helps identify cellular factors that impact drug efficacies. The understanding of the dynamics of full-length protein kinases when interacting with small molecule inhibitors or regulatory proteins is crucial for designing more effective therapeutic strategies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.575556">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.575556" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.575556">
        <p class="paperTitle">Leveraging co-evolutionary insights and AI-based structural modeling to unravel receptor-peptide ligand-binding mechanisms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.575556" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.575556" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Snoeck, S.; Lee, H. K.; Schmid, M. W.; Bender, K. W.; Neeracher, M. J.; Fernandez-Fernandez, A. D.; Santiago, J.; Zipfel, C.</p>
        <p class="info">Score: 32.2, Published: 2024-01-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.575556' target='https://doi.org/10.1101/2024.01.18.575556'> 10.1101/2024.01.18.575556</a></p>
        <p class="abstract">Secreted signaling peptides are central regulators of growth, development, and stress responses, but specific steps in the evolution of these peptides and their receptors are not well understood. In addition, the molecular mechanisms of peptide-receptor binding are only known for a few examples, primarily owing to the limited availability of structural capabilities to few laboratories worldwide. Plants have evolved a multitude of secreted signaling peptides and corresponding transmembrane receptors. Stress-responsive SERINE RICH ENDOGENOUS PEPTIDES (SCOOPs) were recently identified. Bioactive SCOOPs are proteolytically processed by subtilases and are perceived by the leucine-rich repeat receptor kinase MALE DISCOVERER 1-INTERACTING RECEPTOR-LIKE KINASE 2 (MIK2) in the model plant Arabidopsis thaliana. How SCOOPs and MIK2 have (co-)evolved, and how SCOOPs bind to MIK2 are however still unknown. Using in silico analysis of 350 plant genomes and subsequent functional testing, we revealed the conservation of MIK2 as SCOOP receptor within the plant order Brassicales. We then leveraged AlphaFold-Multimer and comparative genomics to identify two conserved putative SCOOP-MIK2 binding pockets across Brassicales MIK2 homologues predicted to interact with the  SxS motif of otherwise sequence-divergent SCOOPs. Notably, mutagenesis of both predicted binding pockets compromised SCOOP binding to MIK2, SCOOP-induced complex formation between MIK2 and its co-receptor BRASSINOSTEROID INSENSITIVE 1-ASSOCIATED KINASE 1 (BAK1), and SCOOP-induced reactive oxygen species production; thus, confirming our in silico predictions. Collectively, in addition to revealing the elusive SCOOP-MIK2 binding mechanisms, our analytic pipeline combining phylogenomics, AI-based structural predictions, and experimental biochemical and physiological validation provides a blueprint for the elucidation of peptide ligand-receptor perception mechanisms.

Significance statementThis study presents a rapid and inexpensive alternative to classical structure-based approaches for resolving ligand-receptor binding mechanisms. It relies on a multilayered bioinformatic approach that leverages genomic data across diverse species in combination with AI-based structural modeling to identify true ligand and receptor homologues, and subsequently predict their binding mechanisms. In silico findings were validated by multiple experimental approaches, which investigated the effect of amino acid changes in the proposed binding pockets on ligand-binding, complex formation with a co-receptor essential for downstream signaling, and activation of downstream signaling. Our analysis combining evolutionary insights, in silico modeling and functional validation provides a framework for structure-function analysis of other peptide-receptor pairs, which could be easily implemented by most laboratories.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573921">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573921" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573921">
        <p class="paperTitle">Direct interrogation of context-dependent GPCR activity with a universal biosensor platform</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573921" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573921" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Janicot, R.; Maziarz, M.; Park, J.-C.; Luebbers, A.; Green, E.; Zhao, J.; Philibert, C. E.; Zhang, H.; Layne, M. D.; Wu, J. C.; Garcia-Marcos, M.</p>
        <p class="info">Score: 29.5, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573921' target='https://doi.org/10.1101/2024.01.02.573921'> 10.1101/2024.01.02.573921</a></p>
        <p class="abstract">G protein-coupled receptors (GPCRs) are the largest family of druggable proteins in the human genome, but progress in understanding and targeting them is hindered by the lack of tools to reliably measure their nuanced behavior in physiologically-relevant contexts. Here, we developed a collection of compact ONE vector G-protein Optical (ONE-GO) biosensor constructs as a scalable platform that can be conveniently deployed to measure G-protein activation by virtually any GPCR with high fidelity even when expressed endogenously in primary cells. By characterizing dozens of GPCRs across many cell types like primary cardiovascular cells or neurons, we revealed new insights into the molecular basis for G-protein coupling selectivity of GPCRs, pharmacogenomic profiles of anti-psychotics on naturally-occurring GPCR variants, and G-protein subtype signaling bias by endogenous GPCRs depending on cell type or upon inducing disease-like states. In summary, this open-source platform makes the direct interrogation of context-dependent GPCR activity broadly accessible.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.576576">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.576576" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.576576">
        <p class="paperTitle">Phenotypic screening of covalent compound libraries identifies chloromethyl ketone antibiotics and MiaA as a new target</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.576576" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.576576" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jin, Y.; Jana, S.; Abbasov, M.; Lin, H.</p>
        <p class="info">Score: 5.6, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.576576' target='https://doi.org/10.1101/2024.01.22.576576'> 10.1101/2024.01.22.576576</a></p>
        <p class="abstract">The emerging antibiotic resistance requires the development of new antibiotics working on novel bacterial targets. Here, we reported an antibiotic discovery workflow by combining the cysteine-reactive compound library phenotypic screening with activity-based protein profiling, which enables the rapid identification of lead compounds as well as new druggable targets in pathogens. Compounds featuring chloromethyl ketone scaffolds exhibited a notably high hit rate against both gram-negative and gram-positive bacterial strains, but not the more commonly used warheads such as acrylamide or chloroacetamide. Target identification of the lead compound, 10-F05, revealed that its primary targets in S. flexneri are FabH Cys112 and MiaA Cys273. We validated the target relevance through biochemical and genetic interactions. Mechanistic studies revealed modification of MiaA by 10-F05 impair substrate tRNA binding, leading to decreased bacterial stress resistance and virulence. Our findings underscore chloromethyl ketone as a novel antibacterial warhead in covalent antibiotic design. The study showcases that combining covalent compound library phenotypic screening with chemoproteomics is an efficient way to identify new drug targets as well as lead compounds, with the potential to open new research directions in drug discovery and chemical biology.

Graphic Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=121 SRC=&#34;FIGDIR/small/576576v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (21K):
org.highwire.dtl.DTLVardef@1e76b4borg.highwire.dtl.DTLVardef@3509e7org.highwire.dtl.DTLVardef@1d1cccorg.highwire.dtl.DTLVardef@14da278_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
