<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A double-ring of human RAD52 remodels replication forks restricting fork reversal
Authors: Honda, M.; Razzaghi, M.; Gaur, P.; Malacaria, E.; Di Biagi, L.; Aiello, F. A.; Paintsil, E. A.; Stanfield, A.; Deppe, B. J.; Gakhar, L.; Schnicker, N. J.; Spies, M. A.; Pichierri, P.; Spies, M.
Score: 23.4, Published: 2023-11-14 DOI: 10.1101/2023.11.14.566657
Human RAD521,2 is a multifunctional DNA repair protein involved in several cellular events that support genome stability including protection of stalled DNA replication forks from excessive degradation3-7.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="A double-ring of human RAD52 remodels replication forks restricting fork reversal
Authors: Honda, M.; Razzaghi, M.; Gaur, P.; Malacaria, E.; Di Biagi, L.; Aiello, F. A.; Paintsil, E. A.; Stanfield, A.; Deppe, B. J.; Gakhar, L.; Schnicker, N. J.; Spies, M. A.; Pichierri, P.; Spies, M.
Score: 23.4, Published: 2023-11-14 DOI: 10.1101/2023.11.14.566657
Human RAD521,2 is a multifunctional DNA repair protein involved in several cellular events that support genome stability including protection of stalled DNA replication forks from excessive degradation3-7." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-22T10:41:31+00:00" />
<meta property="article:modified_time" content="2023-11-22T10:41:31+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="A double-ring of human RAD52 remodels replication forks restricting fork reversal
Authors: Honda, M.; Razzaghi, M.; Gaur, P.; Malacaria, E.; Di Biagi, L.; Aiello, F. A.; Paintsil, E. A.; Stanfield, A.; Deppe, B. J.; Gakhar, L.; Schnicker, N. J.; Spies, M. A.; Pichierri, P.; Spies, M.
Score: 23.4, Published: 2023-11-14 DOI: 10.1101/2023.11.14.566657
Human RAD521,2 is a multifunctional DNA repair protein involved in several cellular events that support genome stability including protection of stalled DNA replication forks from excessive degradation3-7."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "A double-ring of human RAD52 remodels replication forks restricting fork reversal\nAuthors: Honda, M.; Razzaghi, M.; Gaur, P.; Malacaria, E.; Di Biagi, L.; Aiello, F. A.; Paintsil, E. A.; Stanfield, A.; Deppe, B. J.; Gakhar, L.; Schnicker, N. J.; Spies, M. A.; Pichierri, P.; Spies, M.\nScore: 23.4, Published: 2023-11-14 DOI: 10.1101/2023.11.14.566657\nHuman RAD521,2 is a multifunctional DNA repair protein involved in several cellular events that support genome stability including protection of stalled DNA replication forks from excessive degradation3-7.",
  "keywords": [
    
  ],
  "articleBody": " A double-ring of human RAD52 remodels replication forks restricting fork reversal\nAuthors: Honda, M.; Razzaghi, M.; Gaur, P.; Malacaria, E.; Di Biagi, L.; Aiello, F. A.; Paintsil, E. A.; Stanfield, A.; Deppe, B. J.; Gakhar, L.; Schnicker, N. J.; Spies, M. A.; Pichierri, P.; Spies, M.\nScore: 23.4, Published: 2023-11-14 DOI: 10.1101/2023.11.14.566657\nHuman RAD521,2 is a multifunctional DNA repair protein involved in several cellular events that support genome stability including protection of stalled DNA replication forks from excessive degradation3-7. In its gatekeeper role, RAD52 binds to and stabilizes stalled replication forks during replication stress protecting them from reversal by SMARCAL15. The structural and molecular mechanism of the RAD52-mediated fork protection remains elusive. Here, using P1 nuclease sensitivity, biochemical and single-molecule analyses we show that RAD52 dynamically remodels replication forks through its strand exchange activity. The presence of the ssDNA binding protein RPA at the fork modulates the kinetics of the strand exchange without impeding the reaction outcome. Mass photometry and single-particle cryo-electron microscopy show that the replication fork promotes a unique nucleoprotein structure containing head-to-head arrangement of two undecameric RAD52 rings with an extended positively charged surface that accommodates all three arms of the replication fork. We propose that the formation and continuity of this surface is important for the strand exchange reaction and for competition with SMARCAL1.\nStructure of the γ-tubulin ring complex-capped microtubule\nAuthors: Aher, A.; Urnavicius, L.; Xue, A.; Neselu, K.; Kapoor, T.\nScore: 12.6, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567916\nMicrotubules are composed of /{beta}-tubulin dimers positioned head-to-tail to form protofilaments that associate laterally in varying numbers. It is not known how cellular microtubules assemble with the canonical 13-protofilament architecture, resulting in micrometer-scale /{beta}-tubulin tracks for intracellular transport that align with, rather than spiral along, the filament's long-axis. We report that the human ~2.3MDa {gamma}-tubulin ring complex ({gamma}-TuRC), an essential regulator of microtubule formation that contains 14 {gamma}-tubulins, selectively nucleates 13-protofilament microtubules. Cryo-EM reconstructions of {gamma}-TuRC-capped microtubule minus-ends reveal the extensive intra- and inter-domain motions of {gamma}-TuRC subunits that accommodate its actin-containing luminal bridge and establish lateral and longitudinal interactions between {gamma}- and -tubulins. Our structures reveal how free {gamma}-TuRC, an inefficient nucleation template due to its splayed conformation, transforms into a stable cap that blocks addition or loss of /{beta}-tubulins from minus-ends and sets the lattice architecture of cellular microtubules.\nFrom hot to cold: dissecting lipidome adaptation in Mycoplasma mycoides and the Minimal Cell JCVI-Syn3B\nAuthors: Safronova, N.; Junghans, L.; Saenz, J. P.\nScore: 11.6, Published: 2023-11-21 DOI: 10.1101/2023.11.10.566608\nCell membranes insulate and mediate interactions between life and its environment, with lipids determining their properties and functions. However, the intricacies of how cells adjust their lipidome compositions to tune membrane properties remain relatively undefined. The complexity of most model organisms has made it challenging to characterize lipidomic adaptation. An ideal model system would be a relatively simple organism with a single membrane that can adapt to environmental changes, particularly temperature, which is known to affect membrane properties. To this end, we used quantitative shotgun lipidomics to analyze temperature adaptation in Mycoplasma mycoides and its minimal synthetic counterpart, JCVI-Syn3B. Comparing with lipidomes from eukaryotes and bacteria, we observed a universal logarithmic distribution of lipid abundances. Additionally, the extent of lipid remodeling needed for temperature adaptation appears relatively constrained, irrespective of lipidomic or organismal complexity. Through lipid features analysis, we demonstrate head group-specific acyl chain remodeling as characteristic of temperature-induced lipidome adaptation and its deficiency in Syn3B is associated with impaired homeoviscous adaptation. Temporal analysis uncovers a two-stage cold adaptation process: swift cholesterol and cardiolipin shifts followed by gradual acyl chain modifications. This work provides an in-depth analysis of lipidome adaptation in minimal cells, laying a foundation to probe the fundamental design principles of living membranes.\nMassively parallel mapping of substrate cleavage sites defines dipeptidyl peptidase four subsite cooperativity\nAuthors: Gudipati, R. K.; Gaidatzis, D.; Seebacher, J.; Muehlhaeusser, S.; Kempf, G.; Cavadini, S.; Hess, D.; Soneson, C.; Grosshans, H.\nScore: 7.8, Published: 2023-11-17 DOI: 10.1101/2023.11.17.567387\nSubstrate specificity determines protease functions in physiology and in clinical and biotechnological application. However, affordable and unbiased assays for large-scale quantification of substrate cleavage have been lacking. Here, we develop hiMAPS (high-throughput mapping of protease cleavage sites), a cheap, mass spectrometry-based assay, to derive cleavage motifs for human Dipeptidyl Peptidase Four (DPP4), a key regulator of blood glucose levels. We recapitulate the known benefit of proline in the penultimate (P1) position from the substrate N-terminus, extend it to additional residues, and identify and quantify combinatorial interactions of P1 with its neighbors. These findings reveal extensive cooperativity among the enzymes active site subsites and allow us to derive a sequence motif that predicts substrate turnover. We show that this information provides new opportunities to engineer stabilized versions of a key substrate of DPP4, the small peptide hormone GLP-1, whose derivatives are used for clinical treatment of type 2 diabetes and obesity. Finally, structural and biochemical characterization of a DPP4 homologue, C. elegans DPF-3, reveal specific subsite differences and a distinct cleavage motif, providing insight into the mechanistic basis of the observed specificity. Collectively, our findings present a broadly applicable framework to high-throughput mapping of protease cleavage sites, extend our understanding of protease specificity, and provide a chemical space for substrate engineering of a clinically relevant family of exopeptidases.\nA liquid-to-solid phase transition of biomolecular condensates drives in vivo formation of yeast amyloids and prions\nAuthors: Alieva, M. K.; Nikishina, S. B.; Kireev, I. I.; Golyshev, S. A.; Tyurin-Kuzmin, P. A.; Ivanova, L. V.; Alexandrov, A. I.; Kushnirov, V. V.; Dergalev, A. A.\nScore: 7.3, Published: 2023-11-18 DOI: 10.1101/2023.11.17.566816\nLiquid-liquid phase separation (LLPS) and liquid-solid phase transition (LSPT) of amyloidogenic proteins are now being intensively studied as a potentially widespread mechanism of pathological amyloids formation. However, the possibility and importance of such a mechanism in living systems is still questionable. Here, we investigated the possibility of such LSPT for a series of yeast prion proteins-based constructs overproduced in yeast cells lacking any pre-existing amyloid template. By combining fluorescence and electron microscopy with biochemical and genetic approaches, we have shown that three such constructs (containing the prion domains (PDs) of either Sup35, Rnq1 or Mot3 proteins) form amyloid fibrils via the intermediate stage of liquid-like condensates, that age over time into the more solid-like hydrogels and amyloid bodies. In turn, LSPT of these constructs triggers prion conversion of the corresponding wild-type protein. Two other constructs studied (Ure2- and Sap30-based) are unable to phase separate in vivo and their amyloidogenesis is therefore strongly suppressed. Using PrK-resistant amyloid core mapping, we showed that Sup35PD amyloids formed via LSPT have a different molecular architecture compared to those formed via amyloid cross-seeding. Finally, we showed that physiological LLPS of wild-type Sup35 protein can increase its prion conversion in yeast.\nDevelopment of supramolecular anticoagulants with on-demand reversibility\nAuthors: Dockerill, M.; Ford, D. J.; Angerani, S.; Alwis, I.; Ripoll-Rozada, J.; Smythe, R. E.; Liu, J. S. T.; Barbosa Pereira, P. J.; Jackson, S. P.; Payne, R. J.; Winssinger, N.\nScore: 5.4, Published: 2023-11-12 DOI: 10.1101/2023.11.12.566735\nDrugs are administered at a dosing schedule set by their therapeutic index and termination of action is achieved by clearance and metabolism of the drug (hours to days for small molecules, weeks to months for biologics). In some cases, it is important to achieve a fast reversal of the drugs action to overcome dangerous side effects or in response to unforeseen events. A case in point is for anticoagulant drugs. Here we report a general strategy to achieve on-demand reversibility by leveraging supramolecular assembly of drug fragments and showcase the approach with thrombin-inhibiting anticoagulants. In our supramolecular drug design, the action of the drug is reinforced by a dynamic hybridisation of peptide nucleic acids (PNAs) between drug fragments. We show that this design enables the generation of very potent bivalent direct thrombin inhibitors (Ki 74 pM) and this inhibition can be reversed through the use of a PNA antidote. We demonstrate that these supramolecular inhibitors exhibit potent anticoagulant activity in vitro and in vivo and that this activity can also be reversed on demand.\nSingle-Molecule Direct RNA Sequencing Reveals the Shaping of Epitranscriptome Across Multiple Species\nAuthors: Luo, G.-z.; Xie, Y.-y.; Zhong, Z.-d.; Chen, H.-x.\nScore: 4.7, Published: 2023-11-16 DOI: 10.1101/2023.11.16.567334\nThe significance of RNA modification in gene regulation has been widely recognized. To understand the transcriptome-wide landscape and its underlying mechanisms, prevailing mapping strategies have been developed. However, these short-reads based techniques are primarily focused at the gene level, overlooking the nature of RNA as multiple copies within one cell. Third-generation sequencing (TGS) platforms provide direct RNA sequencing at the resolution of individual RNA molecules, offering the promise of detecting RNA modifications and RNA processing events simultaneously. In this study, we introduce SingleMod, a deep learning model tailored for the precise mapping of m6A modifications within single RNA molecules using nanopore direct RNA sequencing (DRS). We systematically dissect the transcriptome-wide m6A profiles in single-molecule and single-base resolution, refining our understanding of the genomics of m6A and revealing an additive mode through which m6A shapes the epitranscriptome. Through comparative analyses across diverse species, we identify three distinct m6A distribution patterns and unveil an exclusion-inclusion deposition mode that governs m6A biogenesis. Furthermore, we introduce a unified quantitative model that delineates this dual-mode in various species. This study pioneers single-molecule m6A landscape exploration across multiple species, deepening our understanding of m6A, including its genomics, biogenesis, mechanisms, and biological implications.\nCholesterol regulates plasma membrane bending by prominin-family proteins\nAuthors: Bell, T. A.; Luce, B. E.; Hakim, P.; Dardari, H.; Ananda, V. Y.; Nguyen, T. H.; Monshizadeh, A.; Chao, L. H.\nScore: 7.1, Published: 2023-11-10 DOI: 10.1101/2023.11.08.566258\nProminin-1 (Prom1) is a pentaspan membrane protein that associates with curved regions of the plasma membrane. Prom1 localizes to cholesterol-rich domains and requires membrane cholesterol to support membrane remodeling. Membrane bending activity is particularly evident in photoreceptors, where Prom1 mutations cause loss of outer segment disk homeostasis leading to cone-rod retinal dystrophy (CCRD). However, the mechanistic link between prominin-dependent cholesterol binding, membrane remodeling, and retinal disease remains unclear. Here, we characterize the membrane bending function and specific cholesterol binding activity of Prom1 and its proposed homolog Tweety homology 1 (Ttyh1) in extracellular vesicles (EVs). Prom1 and Ttyh1 induce formation of EVs in cultured mammalian cells that are biophysically similar. Though both proteins bend membranes and form EVs at the plasma membrane, Ttyh1 lacks a stable interaction with cholesterol that is present in Prom1. Correspondingly, Ttyh1 forms EVs that are more deformed than those produced by Prom1. An evolutionarily conserved and retinal disease-associated Prom1 residue (Trp-795) is necessary for cholesterol binding, EV membrane deformation, and efficient trafficking to the plasma membrane. Removal of N-glycan moieties from Prom1 biases the enzyme toward a cholesterol-bound state. We propose that Prom1 and Ttyh1 are both members of a single prominin family of membrane bending proteins, that Ttyh1 is a constitutively active member of this family, and that Prom1 is regulated by cholesterol binding and N-glycosylation. These findings shed light on mechanisms of prominin family function in disease and help unify models of prominin function across diverse cell types. Significance StatementMammalian cells dynamically shape the plasma membrane to achieve specialized functions. Prominin-1 (Prom1) is a membrane protein that promotes bending and shaping of the membrane, notably in photoreceptors. Here, the authors find that membrane cholesterol regulates the membrane bending activity of Prom1 in extracellular vesicles. Membrane bending can be tuned by altering cholesterol levels and may be coupled to different Prom1 N-glycosylation states. A distant homolog of Prom1 called Ttyh1 is also shown to form extracellular vesicles and supports more membrane bending than Prom1 though it does not bind cholesterol. This work unites the Prom and Ttyh protein families into a single functional group and helps explain how membrane cholesterol functionalizes prominin-family proteins for different roles in different cell types.\nDevelopment of UHPLC MS/MS method for determination and quantification of endocannabinoids in cerebrospinal fluid.\nAuthors: Hooshmand, K.; Ismoilova, V.; Wretlind, A.; Simonsen, A. H.; Hasselbalch, S. G.; Legido-Quigley, C.\nScore: 3.4, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567867\nEndocannabinoids (eCBs) are endogenous lipids that activate cannabinoid receptors. Their role in neuroinflammation is of a great interest due to the range of physiological effects they exhibit, potentially serving as bioindicators of pathologies. However, they are present at nanomolar concentrations in biological samples, posing challenges for detection and quantification. A method for quantitative analysis of eight exogenous eCB internal standards (anandamide-d4, docosahexaenoyl ethanolamide-d4, eicosapentaenoyl ethanolamide-d4, linoleoyl ethanolamide-d4, oleoyl ethanolamide-d4, palmitoyl ethanolamide-d5, stearoyl ethanolamide-d3 and palmitanilide) in cerebrospinal fluid is reported here. Six monophasic solvents (methanol, acetonitrile, chloroform/methanol/water, acetonitrile/acetone, isopropanol/acetonitrile/water, and ethanol) were compared in their extraction efficiency. Detection of analytes was performed using an ultra-high performance liquid chromatography method coupled to tandem mass spectrometry (UHPLC-MS/MS) in dynamic multiple reaction monitoring (dMRM) mode. The method was further optimised based on accuracy, precision, matrix effect, linearity, limits of detection and quantification. The method was applied to a cohort of healthy individuals (n=33) to identify and estimate the concentration of eCBs in CSF.\nDevelopment of LRRK2 designed ankyrin-repeat proteins\nAuthors: Dederer, V.; Sanz Murillo, M.; Karasmanis, E. P.; Hatch, K. S.; Chatterjee, D.; Preuss, F.; Abdul Azeez, K. R.; Nguyen, L. V.; Galicia, C.; Dreier, B.; Pluckthun, A.; Versees, W.; Mathea, S.; Leschziner, A. E.; Reck-Peterson, S. L.; Knapp, S.\nScore: 3.2, Published: 2023-11-15 DOI: 10.1101/2023.11.14.567123\nLeucine rich repeat kinase 2 (LRRK2) is a large multidomain protein containing two catalytic domains, a kinase and a GTPase, as well protein interactions domains, including a WD40 domain. The association of increased LRRK2 kinase activity with both the familial and sporadic forms of Parkinsons disease (PD) has led to intense interest in determining its cellular function. However, small molecule probes that can bind to LRRK2 and report on or affect its activity are needed. Here, we identified a series of high-affinity LRRK2-binding designed ankyrin-repeat proteins (DARPins). One of these DARPins (E11) bound to the LRRK2 WD40 domain with high affinity. LRRK2 bound to DARPin E11 showed improved behavior on cryo-EM grids, resulting in higher resolution LRRK2 structures. DARPin E11 did not affect the catalytic activity of a truncated form of LRRK2 in vitro but decreased the phosphorylation of Rab8A, a LRRK2 substrate, in cells. We also found that DARPin E11 disrupts the formation of microtubule-associated LRRK2 filaments in cells, which are known to require WD40-based dimerization. Thus, DARPin E11 is a new tool to explore the function and dysfunction of LRRK2 and guide the development of LRRK2 kinase inhibitors that target the WD40 domain instead of the kinase.\n",
  "wordCount" : "2407",
  "inLanguage": "en",
  "datePublished": "2023-11-22T10:41:31Z",
  "dateModified": "2023-11-22T10:41:31Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 22, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.14.566657">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.14.566657" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.14.566657">
        <p class="paperTitle">A double-ring of human RAD52 remodels replication forks restricting fork reversal</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.14.566657" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.14.566657" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Honda, M.; Razzaghi, M.; Gaur, P.; Malacaria, E.; Di Biagi, L.; Aiello, F. A.; Paintsil, E. A.; Stanfield, A.; Deppe, B. J.; Gakhar, L.; Schnicker, N. J.; Spies, M. A.; Pichierri, P.; Spies, M.</p>
        <p class="info">Score: 23.4, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.14.566657' target='https://doi.org/10.1101/2023.11.14.566657'> 10.1101/2023.11.14.566657</a></p>
        <p class="abstract">Human RAD521,2 is a multifunctional DNA repair protein involved in several cellular events that support genome stability including protection of stalled DNA replication forks from excessive degradation3-7. In its gatekeeper role, RAD52 binds to and stabilizes stalled replication forks during replication stress protecting them from reversal by SMARCAL15. The structural and molecular mechanism of the RAD52-mediated fork protection remains elusive. Here, using P1 nuclease sensitivity, biochemical and single-molecule analyses we show that RAD52 dynamically remodels replication forks through its strand exchange activity. The presence of the ssDNA binding protein RPA at the fork modulates the kinetics of the strand exchange without impeding the reaction outcome. Mass photometry and single-particle cryo-electron microscopy show that the replication fork promotes a unique nucleoprotein structure containing head-to-head arrangement of two undecameric RAD52 rings with an extended positively charged surface that accommodates all three arms of the replication fork. We propose that the formation and continuity of this surface is important for the strand exchange reaction and for competition with SMARCAL1.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567916">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567916" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567916">
        <p class="paperTitle">Structure of the γ-tubulin ring complex-capped microtubule</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567916" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567916" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aher, A.; Urnavicius, L.; Xue, A.; Neselu, K.; Kapoor, T.</p>
        <p class="info">Score: 12.6, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567916' target='https://doi.org/10.1101/2023.11.20.567916'> 10.1101/2023.11.20.567916</a></p>
        <p class="abstract">Microtubules are composed of /{beta}-tubulin dimers positioned head-to-tail to form protofilaments that associate laterally in varying numbers. It is not known how cellular microtubules assemble with the canonical 13-protofilament architecture, resulting in micrometer-scale /{beta}-tubulin tracks for intracellular transport that align with, rather than spiral along, the filament&#39;s long-axis. We report that the human ~2.3MDa {gamma}-tubulin ring complex ({gamma}-TuRC), an essential regulator of microtubule formation that contains 14 {gamma}-tubulins, selectively nucleates 13-protofilament microtubules. Cryo-EM reconstructions of {gamma}-TuRC-capped microtubule minus-ends reveal the extensive intra- and inter-domain motions of {gamma}-TuRC subunits that accommodate its actin-containing luminal bridge and establish lateral and longitudinal interactions between {gamma}- and -tubulins. Our structures reveal how free {gamma}-TuRC, an inefficient nucleation template due to its splayed conformation, transforms into a stable cap that blocks addition or loss of /{beta}-tubulins from minus-ends and sets the lattice architecture of cellular microtubules.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.566608">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.566608" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.566608">
        <p class="paperTitle">From hot to cold: dissecting lipidome adaptation in Mycoplasma mycoides and the Minimal Cell JCVI-Syn3B</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.566608" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.566608" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Safronova, N.; Junghans, L.; Saenz, J. P.</p>
        <p class="info">Score: 11.6, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.566608' target='https://doi.org/10.1101/2023.11.10.566608'> 10.1101/2023.11.10.566608</a></p>
        <p class="abstract">Cell membranes insulate and mediate interactions between life and its environment, with lipids determining their properties and functions. However, the intricacies of how cells adjust their lipidome compositions to tune membrane properties remain relatively undefined. The complexity of most model organisms has made it challenging to characterize lipidomic adaptation. An ideal model system would be a relatively simple organism with a single membrane that can adapt to environmental changes, particularly temperature, which is known to affect membrane properties. To this end, we used quantitative shotgun lipidomics to analyze temperature adaptation in Mycoplasma mycoides and its minimal synthetic counterpart, JCVI-Syn3B. Comparing with lipidomes from eukaryotes and bacteria, we observed a universal logarithmic distribution of lipid abundances. Additionally, the extent of lipid remodeling needed for temperature adaptation appears relatively constrained, irrespective of lipidomic or organismal complexity. Through lipid features analysis, we demonstrate head group-specific acyl chain remodeling as characteristic of temperature-induced lipidome adaptation and its deficiency in Syn3B is associated with impaired homeoviscous adaptation. Temporal analysis uncovers a two-stage cold adaptation process: swift cholesterol and cardiolipin shifts followed by gradual acyl chain modifications. This work provides an in-depth analysis of lipidome adaptation in minimal cells, laying a foundation to probe the fundamental design principles of living membranes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.17.567387">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.17.567387" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.17.567387">
        <p class="paperTitle">Massively parallel mapping of substrate cleavage sites defines dipeptidyl peptidase four subsite cooperativity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.17.567387" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.17.567387" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gudipati, R. K.; Gaidatzis, D.; Seebacher, J.; Muehlhaeusser, S.; Kempf, G.; Cavadini, S.; Hess, D.; Soneson, C.; Grosshans, H.</p>
        <p class="info">Score: 7.8, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.17.567387' target='https://doi.org/10.1101/2023.11.17.567387'> 10.1101/2023.11.17.567387</a></p>
        <p class="abstract">Substrate specificity determines protease functions in physiology and in clinical and biotechnological application. However, affordable and unbiased assays for large-scale quantification of substrate cleavage have been lacking. Here, we develop hiMAPS (high-throughput mapping of protease cleavage sites), a cheap, mass spectrometry-based assay, to derive cleavage motifs for human Dipeptidyl Peptidase Four (DPP4), a key regulator of blood glucose levels. We recapitulate the known benefit of proline in the penultimate (P1) position from the substrate N-terminus, extend it to additional residues, and identify and quantify combinatorial interactions of P1 with its neighbors. These findings reveal extensive cooperativity among the enzymes active site subsites and allow us to derive a sequence motif that predicts substrate turnover. We show that this information provides new opportunities to engineer stabilized versions of a key substrate of DPP4, the small peptide hormone GLP-1, whose derivatives are used for clinical treatment of type 2 diabetes and obesity. Finally, structural and biochemical characterization of a DPP4 homologue, C. elegans DPF-3, reveal specific subsite differences and a distinct cleavage motif, providing insight into the mechanistic basis of the observed specificity. Collectively, our findings present a broadly applicable framework to high-throughput mapping of protease cleavage sites, extend our understanding of protease specificity, and provide a chemical space for substrate engineering of a clinically relevant family of exopeptidases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.17.566816">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.17.566816" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.17.566816">
        <p class="paperTitle">A liquid-to-solid phase transition of biomolecular condensates drives in vivo formation of yeast amyloids and prions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.17.566816" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.17.566816" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alieva, M. K.; Nikishina, S. B.; Kireev, I. I.; Golyshev, S. A.; Tyurin-Kuzmin, P. A.; Ivanova, L. V.; Alexandrov, A. I.; Kushnirov, V. V.; Dergalev, A. A.</p>
        <p class="info">Score: 7.3, Published: 2023-11-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.17.566816' target='https://doi.org/10.1101/2023.11.17.566816'> 10.1101/2023.11.17.566816</a></p>
        <p class="abstract">Liquid-liquid phase separation (LLPS) and liquid-solid phase transition (LSPT) of amyloidogenic proteins are now being intensively studied as a potentially widespread mechanism of pathological amyloids formation. However, the possibility and importance of such a mechanism in living systems is still questionable. Here, we investigated the possibility of such LSPT for a series of yeast prion proteins-based constructs overproduced in yeast cells lacking any pre-existing amyloid template. By combining fluorescence and electron microscopy with biochemical and genetic approaches, we have shown that three such constructs (containing the prion domains (PDs) of either Sup35, Rnq1 or Mot3 proteins) form amyloid fibrils via the intermediate stage of liquid-like condensates, that age over time into the more solid-like hydrogels and amyloid bodies. In turn, LSPT of these constructs triggers prion conversion of the corresponding wild-type protein. Two other constructs studied (Ure2- and Sap30-based) are unable to phase separate in vivo and their amyloidogenesis is therefore strongly suppressed. Using PrK-resistant amyloid core mapping, we showed that Sup35PD amyloids formed via LSPT have a different molecular architecture compared to those formed via amyloid cross-seeding. Finally, we showed that physiological LLPS of wild-type Sup35 protein can increase its prion conversion in yeast.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.12.566735">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.12.566735" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.12.566735">
        <p class="paperTitle">Development of supramolecular anticoagulants with on-demand reversibility</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.12.566735" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.12.566735" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dockerill, M.; Ford, D. J.; Angerani, S.; Alwis, I.; Ripoll-Rozada, J.; Smythe, R. E.; Liu, J. S. T.; Barbosa Pereira, P. J.; Jackson, S. P.; Payne, R. J.; Winssinger, N.</p>
        <p class="info">Score: 5.4, Published: 2023-11-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.12.566735' target='https://doi.org/10.1101/2023.11.12.566735'> 10.1101/2023.11.12.566735</a></p>
        <p class="abstract">Drugs are administered at a dosing schedule set by their therapeutic index and termination of action is achieved by clearance and metabolism of the drug (hours to days for small molecules, weeks to months for biologics). In some cases, it is important to achieve a fast reversal of the drugs action to overcome dangerous side effects or in response to unforeseen events. A case in point is for anticoagulant drugs. Here we report a general strategy to achieve on-demand reversibility by leveraging supramolecular assembly of drug fragments and showcase the approach with thrombin-inhibiting anticoagulants. In our supramolecular drug design, the action of the drug is reinforced by a dynamic hybridisation of peptide nucleic acids (PNAs) between drug fragments. We show that this design enables the generation of very potent bivalent direct thrombin inhibitors (Ki 74 pM) and this inhibition can be reversed through the use of a PNA antidote. We demonstrate that these supramolecular inhibitors exhibit potent anticoagulant activity in vitro and in vivo and that this activity can also be reversed on demand.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.567334">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.567334" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.567334">
        <p class="paperTitle">Single-Molecule Direct RNA Sequencing Reveals the Shaping of Epitranscriptome Across Multiple Species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.567334" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.567334" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Luo, G.-z.; Xie, Y.-y.; Zhong, Z.-d.; Chen, H.-x.</p>
        <p class="info">Score: 4.7, Published: 2023-11-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.567334' target='https://doi.org/10.1101/2023.11.16.567334'> 10.1101/2023.11.16.567334</a></p>
        <p class="abstract">The significance of RNA modification in gene regulation has been widely recognized. To understand the transcriptome-wide landscape and its underlying mechanisms, prevailing mapping strategies have been developed. However, these short-reads based techniques are primarily focused at the gene level, overlooking the nature of RNA as multiple copies within one cell. Third-generation sequencing (TGS) platforms provide direct RNA sequencing at the resolution of individual RNA molecules, offering the promise of detecting RNA modifications and RNA processing events simultaneously. In this study, we introduce SingleMod, a deep learning model tailored for the precise mapping of m6A modifications within single RNA molecules using nanopore direct RNA sequencing (DRS). We systematically dissect the transcriptome-wide m6A profiles in single-molecule and single-base resolution, refining our understanding of the genomics of m6A and revealing an additive mode through which m6A shapes the epitranscriptome. Through comparative analyses across diverse species, we identify three distinct m6A distribution patterns and unveil an exclusion-inclusion deposition mode that governs m6A biogenesis. Furthermore, we introduce a unified quantitative model that delineates this dual-mode in various species. This study pioneers single-molecule m6A landscape exploration across multiple species, deepening our understanding of m6A, including its genomics, biogenesis, mechanisms, and biological implications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.08.566258">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.08.566258" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.08.566258">
        <p class="paperTitle">Cholesterol regulates plasma membrane bending by prominin-family proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.08.566258" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.08.566258" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bell, T. A.; Luce, B. E.; Hakim, P.; Dardari, H.; Ananda, V. Y.; Nguyen, T. H.; Monshizadeh, A.; Chao, L. H.</p>
        <p class="info">Score: 7.1, Published: 2023-11-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.08.566258' target='https://doi.org/10.1101/2023.11.08.566258'> 10.1101/2023.11.08.566258</a></p>
        <p class="abstract">Prominin-1 (Prom1) is a pentaspan membrane protein that associates with curved regions of the plasma membrane. Prom1 localizes to cholesterol-rich domains and requires membrane cholesterol to support membrane remodeling. Membrane bending activity is particularly evident in photoreceptors, where Prom1 mutations cause loss of outer segment disk homeostasis leading to cone-rod retinal dystrophy (CCRD). However, the mechanistic link between prominin-dependent cholesterol binding, membrane remodeling, and retinal disease remains unclear. Here, we characterize the membrane bending function and specific cholesterol binding activity of Prom1 and its proposed homolog Tweety homology 1 (Ttyh1) in extracellular vesicles (EVs). Prom1 and Ttyh1 induce formation of EVs in cultured mammalian cells that are biophysically similar. Though both proteins bend membranes and form EVs at the plasma membrane, Ttyh1 lacks a stable interaction with cholesterol that is present in Prom1. Correspondingly, Ttyh1 forms EVs that are more deformed than those produced by Prom1. An evolutionarily conserved and retinal disease-associated Prom1 residue (Trp-795) is necessary for cholesterol binding, EV membrane deformation, and efficient trafficking to the plasma membrane. Removal of N-glycan moieties from Prom1 biases the enzyme toward a cholesterol-bound state. We propose that Prom1 and Ttyh1 are both members of a single prominin family of membrane bending proteins, that Ttyh1 is a constitutively active member of this family, and that Prom1 is regulated by cholesterol binding and N-glycosylation. These findings shed light on mechanisms of prominin family function in disease and help unify models of prominin function across diverse cell types.

Significance StatementMammalian cells dynamically shape the plasma membrane to achieve specialized functions. Prominin-1 (Prom1) is a membrane protein that promotes bending and shaping of the membrane, notably in photoreceptors. Here, the authors find that membrane cholesterol regulates the membrane bending activity of Prom1 in extracellular vesicles. Membrane bending can be tuned by altering cholesterol levels and may be coupled to different Prom1 N-glycosylation states. A distant homolog of Prom1 called Ttyh1 is also shown to form extracellular vesicles and supports more membrane bending than Prom1 though it does not bind cholesterol. This work unites the Prom and Ttyh protein families into a single functional group and helps explain how membrane cholesterol functionalizes prominin-family proteins for different roles in different cell types.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567867">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567867" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567867">
        <p class="paperTitle">Development of UHPLC MS/MS method for determination and quantification of endocannabinoids in cerebrospinal fluid.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567867" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567867" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hooshmand, K.; Ismoilova, V.; Wretlind, A.; Simonsen, A. H.; Hasselbalch, S. G.; Legido-Quigley, C.</p>
        <p class="info">Score: 3.4, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567867' target='https://doi.org/10.1101/2023.11.20.567867'> 10.1101/2023.11.20.567867</a></p>
        <p class="abstract">Endocannabinoids (eCBs) are endogenous lipids that activate cannabinoid receptors. Their role in neuroinflammation is of a great interest due to the range of physiological effects they exhibit, potentially serving as bioindicators of pathologies. However, they are present at nanomolar concentrations in biological samples, posing challenges for detection and quantification. A method for quantitative analysis of eight exogenous eCB internal standards (anandamide-d4, docosahexaenoyl ethanolamide-d4, eicosapentaenoyl ethanolamide-d4, linoleoyl ethanolamide-d4, oleoyl ethanolamide-d4, palmitoyl ethanolamide-d5, stearoyl ethanolamide-d3 and palmitanilide) in cerebrospinal fluid is reported here. Six monophasic solvents (methanol, acetonitrile, chloroform/methanol/water, acetonitrile/acetone, isopropanol/acetonitrile/water, and ethanol) were compared in their extraction efficiency. Detection of analytes was performed using an ultra-high performance liquid chromatography method coupled to tandem mass spectrometry (UHPLC-MS/MS) in dynamic multiple reaction monitoring (dMRM) mode. The method was further optimised based on accuracy, precision, matrix effect, linearity, limits of detection and quantification. The method was applied to a cohort of healthy individuals (n=33) to identify and estimate the concentration of eCBs in CSF.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.14.567123">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.14.567123" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.14.567123">
        <p class="paperTitle">Development of LRRK2 designed ankyrin-repeat proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.14.567123" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.14.567123" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dederer, V.; Sanz Murillo, M.; Karasmanis, E. P.; Hatch, K. S.; Chatterjee, D.; Preuss, F.; Abdul Azeez, K. R.; Nguyen, L. V.; Galicia, C.; Dreier, B.; Pluckthun, A.; Versees, W.; Mathea, S.; Leschziner, A. E.; Reck-Peterson, S. L.; Knapp, S.</p>
        <p class="info">Score: 3.2, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.14.567123' target='https://doi.org/10.1101/2023.11.14.567123'> 10.1101/2023.11.14.567123</a></p>
        <p class="abstract">Leucine rich repeat kinase 2 (LRRK2) is a large multidomain protein containing two catalytic domains, a kinase and a GTPase, as well protein interactions domains, including a WD40 domain. The association of increased LRRK2 kinase activity with both the familial and sporadic forms of Parkinsons disease (PD) has led to intense interest in determining its cellular function. However, small molecule probes that can bind to LRRK2 and report on or affect its activity are needed. Here, we identified a series of high-affinity LRRK2-binding designed ankyrin-repeat proteins (DARPins). One of these DARPins (E11) bound to the LRRK2 WD40 domain with high affinity. LRRK2 bound to DARPin E11 showed improved behavior on cryo-EM grids, resulting in higher resolution LRRK2 structures. DARPin E11 did not affect the catalytic activity of a truncated form of LRRK2 in vitro but decreased the phosphorylation of Rab8A, a LRRK2 substrate, in cells. We also found that DARPin E11 disrupts the formation of microtubule-associated LRRK2 filaments in cells, which are known to require WD40-based dimerization. Thus, DARPin E11 is a new tool to explore the function and dysfunction of LRRK2 and guide the development of LRRK2 kinase inhibitors that target the WD40 domain instead of the kinase.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
