<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A systems-scale, integrated view of the ubiquitylation site occupancy and dynamics
Authors: Prus, G.; Satpathy, S.; Weinert, B.; Narita, T.; Choudhary, C.
Score: 33.2, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549470
Ubiquitylation regulates virtually all proteins and biological processes in a cell. However, the global site-specific occupancy (stoichiometry) and turnover rate of ubiquitylation have never been quantified. Here, we present the first integrated picture of ubiquitylation site occupancy and half-life. Ubiquitylation occupancy spans four orders of magnitude, but the median ubiquitylation site occupancy is three orders of magnitude lower than that of phosphorylation.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="A systems-scale, integrated view of the ubiquitylation site occupancy and dynamics
Authors: Prus, G.; Satpathy, S.; Weinert, B.; Narita, T.; Choudhary, C.
Score: 33.2, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549470
Ubiquitylation regulates virtually all proteins and biological processes in a cell. However, the global site-specific occupancy (stoichiometry) and turnover rate of ubiquitylation have never been quantified. Here, we present the first integrated picture of ubiquitylation site occupancy and half-life. Ubiquitylation occupancy spans four orders of magnitude, but the median ubiquitylation site occupancy is three orders of magnitude lower than that of phosphorylation." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-26T10:37:30+00:00" />
<meta property="article:modified_time" content="2023-07-26T10:37:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="A systems-scale, integrated view of the ubiquitylation site occupancy and dynamics
Authors: Prus, G.; Satpathy, S.; Weinert, B.; Narita, T.; Choudhary, C.
Score: 33.2, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549470
Ubiquitylation regulates virtually all proteins and biological processes in a cell. However, the global site-specific occupancy (stoichiometry) and turnover rate of ubiquitylation have never been quantified. Here, we present the first integrated picture of ubiquitylation site occupancy and half-life. Ubiquitylation occupancy spans four orders of magnitude, but the median ubiquitylation site occupancy is three orders of magnitude lower than that of phosphorylation."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "A systems-scale, integrated view of the ubiquitylation site occupancy and dynamics\nAuthors: Prus, G.; Satpathy, S.; Weinert, B.; Narita, T.; Choudhary, C.\nScore: 33.2, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549470\nUbiquitylation regulates virtually all proteins and biological processes in a cell. However, the global site-specific occupancy (stoichiometry) and turnover rate of ubiquitylation have never been quantified. Here, we present the first integrated picture of ubiquitylation site occupancy and half-life. Ubiquitylation occupancy spans four orders of magnitude, but the median ubiquitylation site occupancy is three orders of magnitude lower than that of phosphorylation.",
  "keywords": [
    
  ],
  "articleBody": " A systems-scale, integrated view of the ubiquitylation site occupancy and dynamics\nAuthors: Prus, G.; Satpathy, S.; Weinert, B.; Narita, T.; Choudhary, C.\nScore: 33.2, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549470\nUbiquitylation regulates virtually all proteins and biological processes in a cell. However, the global site-specific occupancy (stoichiometry) and turnover rate of ubiquitylation have never been quantified. Here, we present the first integrated picture of ubiquitylation site occupancy and half-life. Ubiquitylation occupancy spans four orders of magnitude, but the median ubiquitylation site occupancy is three orders of magnitude lower than that of phosphorylation. The occupancy, turnover rate, and the regulation of sites by proteasome inhibitors show strong interrelationships. These properties can discriminate signaling-relevant sites from the sites involved in proteasomal degradation. The sites strongly upregulated by proteasome inhibitors have a longer half-life, and the half-life increases with increasing protein length. Importantly, a previously unknown surveillance mechanism rapidly deubiquitylates all ubiquitin-specific E1 and E2 enzymes and protects them against bystander ubiquitylation accumulation. This work reveals general principles of ubiquitylation-dependent governance and offers conceptual insights into the dynamic regulation of the cell. HighlightsO_LIUbiquitylation site occupancy is 3 orders of magnitude lower than phosphorylation C_LIO_LIThe highest 80% and the lowest 20% occupancy sites have distinct properties C_LIO_LIHigh occupancy sites are concentrated in the cytoplasmic domains of SLC proteins C_LIO_LIA dedicated mechanism prevevents ubiquitylation accumulation in E1s and E2s C_LI\nCoenzyme Q4 is a functional substitute for coenzyme Q10 and can be targeted to the mitochondria\nAuthors: Steenberge, L. H.; Sung, A. Y.; Fan, J.; Pagliarini, D. J.\nScore: 26.5, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549963\nCoenzyme Q10 (CoQ10) is an important cofactor and antioxidant for numerous cellular processes, and its deficiency has been linked to human disorders including mitochondrial disease, heart failure, Parkinsons disease, and hypertension. Unfortunately, treatment with exogenous oral CoQ10 is often ineffective, likely due to the extreme hydrophobicity and high molecular weight of CoQ10. Here, we show that less hydrophobic CoQ species with shorter isoprenoid tails can serve as viable substitutes for CoQ10 in human cells. We demonstrate that CoQ4 can perform multiple functions of CoQ10 in CoQ-deficient cells at markedly lower treatment concentrations, motivating further investigation of CoQ4 as a supplement for CoQ10 deficiencies. In addition, we describe the synthesis and evaluation of an initial set of compounds designed to target CoQ4 selectively to mitochondria using triphenylphosphonium (TPP). Our results indicate that select versions of these compounds can successfully be delivered to mitochondria in a cell model and be cleaved to produce CoQ4, laying the groundwork for further development.\nEnhancing Inter-link Coverage in Cross-Linking Mass Spectrometry through Context-Sensitive Subgrouping and Decoy Fusion\nAuthors: Bogdanow, B.; Wang, C.; Ruwolt, M.; Ruta, J.; Mühlberg, L.; Zeng, W.-F.; Elofsson, A.; Liu, F.\nScore: 10.0, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549678\nIn cross-linking mass spectrometry, sensitivity and specificity in assigning mass spectra to cross-links between different proteins (inter-links) remains challenging. Here, we report on limitations of commonly used concatenated target-decoy searches and propose a target-decoy competition strategy on a fused database as a solution. Further, we capitalize on context-divergent error rates by implementing a novel context-sensitive subgrouping strategy. This approach increases inter-link coverage by [~] 30 - 75 % across XL-MS datasets, maintains low error rates, and preserves structural accuracy.\nFAIMS-enabled N-terminomics analysis reveals novel legumain substrates in murine spleen\nAuthors: Ziegler, A. R.; Dufour, A.; Scott, N. E.; Edgington-Mitchell, L. E.\nScore: 8.9, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549248\nAberrant levels of the asparaginyl endopeptidase legumain have been linked to inflammation, neurodegeneration and cancer, yet our understanding of this protease is incomplete. Systematic attempts to identify legumain substrates have previously been confined to in vitro studies, which fail to mirror physiological conditions and obscure biologically relevant cleavage events. Using high-field asymmetric waveform ion mobility spectrometry (FAIMS), we developed a sensitive and streamlined approach for proteome and N-terminome analyses in a single analytical method without the need for N-termini enrichment. Compared to unfractionated proteomic analysis, we demonstrate FAIMS fractionation improves neo-N- termini identification by \u003e2.5 fold, resulting in identification of \u003e2,882 unique neo-N-termini from limited sample amounts. Within murine spleens, this approach identifies 6,366 proteins and 2,528 unique neo-N-termini, with 235 cleavage events enriched in wild-type compared to legumain-deficient spleens. Among these, 119 neo-N-termini arose from asparaginyl endopeptidase activities, representing novel putative physiological legumain substrates. The direct cleavage of selected substrates by legumain was confirmed using in vitro assays, providing support for the existence of physiologically relevant extra-lysosomal legumain activity. Combined, these data shed critical light on the functions of legumain and demonstrates the utility of FAIMS as an accessible method to improve depth and quality of N- terminomics studies.\nInteraction with the membrane-anchored protein CHIC2 constrains the ubiquitin ligase activity of CHIP\nAuthors: Callahan, M. D.; Hodul, M.; Carroll, E. C.; Ravalin, M.; Nadel, C. M.; de Silva, A. R.; Cupo, A. R.; McDermott, L. E.; Nix, J. C.; Page, R. C.; Kao, A. W.; Gestwicki, J. E.\nScore: 8.8, Published: 2023-07-17 DOI: 10.1101/2023.07.17.549407\nMaintenance of cellular health requires the proper regulation of E3 ubiquitin ligases. The E3 ligase CHIP is canonically regulated by its interactions with the molecular chaperones Hsp70 and Hsp90, which focus CHIPs ubiquitination activity on misfolded proteins. Here, we report a chaperone-independent interaction of CHIP with the membrane-anchored protein CHIC2, which strongly attenuates CHIPs ligase activity. We show that CHIC2 outcompetes abundant, cytosolic chaperones through its exquisite CHIP selectivity, rather than through enhanced affinity. In proteomic experiments, we find that CHIC2 knockout phenocopies CHIP knockout in certain cell types, implying that chaperone-independent interactions can sometimes predominate CHIPs functions. Furthermore, loss of the CHIP-CHIC2 interaction induces neurodegeneration and shortens lifespan in C. elegans, demonstrating that formation of this chaperone-independent complex is important in animals. We propose that CHIC2 attenuates CHIP activity at the membrane, offering a novel mechanism by which this ubiquitin ligase can be regulated.\nThe hGID GID4 E3 ubiquitin ligase complex targets ARHGAP11A to regulate cell migration\nAuthors: Bagci, H.; Winkler, M.; Uliana, F.; Boulais, J.; Mohamed, W. I.; Park, S. L.; Cote, J.-F.; Peter, M.\nScore: 8.5, Published: 2023-07-20 DOI: 10.1101/2023.07.20.549906\nThe human CTLH/GID (hGID) complex emerged as an important E3 ligase regulating multiple cellular processes, including cell cycle progression and metabolic activity. However, the range of biological functions controlled by hGID remains unexplored. Here, we show that the hGID substrate receptor GID4 regulates cell growth and migration. Biochemical and cellular assays combined with proximity-dependent biotinylation (BioID2) revealed that the hGIDGID4 E3-ligase targets the Rho-GAP ARHGAP11A for degradation. Depletion of GID4 or impeding the GID4 substrate binding pocket impairs motility and directed cell movement, whereas knockdown of ARHGAP11A significantly restores the cell migration defect. We found that GID4 controls cell migration by degrading ARHGAP11A thereby preventing its accumulation at the cell periphery where it inactivates RhoA activity. Together, we identified a unique function for GID4, as well as a wide range of substrate profiles beyond Pro/N-degron motifs, which pave the way for deciphering additional pathways regulated by hGID E3 ligase activity through its GID4 substrate receptor.\nStructural characterisation of the complete cycle of sliding clamp loading in Escherichia coli\nAuthors: Xu, Z.-Q.; Jergic, S.; Lo, A. T. Y.; Pradhan, A. C.; Brown, S. H. J.; Bouwer, J. C.; Ghodke, H.; Lewis, P. J.; Tolun, G.; Oakley, A. J.; Dixon, N. E.\nScore: 8.1, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549978\nRing-shaped DNA sliding clamps are essential for DNA replication and genome maintenance. Clamps need to be opened or trapped open and chaperoned onto DNA by clamp loader complexes (CLCs). Detailed understanding of the mechanisms by which CLCs open and place clamps around DNA remains limited. Here, we present a series of six structures of the Escherichia coli CLC bound to an open or closed clamp on and off a primer-template DNA that represent all intermediates in the clamp loading process. We show that the ATP-bound CLC first binds to a clamp, then constricts to hold onto it. The CLC then expands to open the clamp with a gap large enough for double-stranded DNA to enter. Upon binding to DNA, the CLC constricts slightly, allowing ATP hydrolysis and clamp closing around DNA. Although both yeast and E. coli CLCs open clamps by crab claw-like motions, they do it by the CLC expanding in opposite directions. These structures provide critical high-resolution snapshots of clamp loading by the E. coli CLC, revealing how the molecular machine works.\nStructural insights into complex I deficiency and assembly from the disease-related ndufs4-/- mouse\nAuthors: Yin, Z.; Agip, A.-N. A.; Bridges, H.; Hirst, J.\nScore: 5.6, Published: 2023-07-25 DOI: 10.1101/2023.07.17.549284\nRespiratory complex I (NADH:ubiquinone oxidoreductase) is essential for cellular energy production and NAD+ homeostasis. Complex I mutations cause neuromuscular, mitochondrial diseases, such as Leigh Syndrome, but their molecular-level consequences remain poorly understood. Here, we use a popular complex I-linked mitochondrial disease model, the ndufs4-/- mouse, to define the structural, biochemical and functional consequences of the absence of subunit NDUFS4. Cryo-EM analyses of mouse-heart ndufs4-/- complex I revealed a loose association of the NADH-dehydrogenase module, and discrete classes containing either assembly factor NDUFAF2 or subunit NDUFS6. Subunit NDUFA12 (that replaces its paralogue NDUFAF2 in mature complex I) is absent from all classes, compounding the deletion of NDUFS4 and preventing maturation of an NDUFS4-free but otherwise complete enzyme. We propose NDUFAF2 as the recruiter of the NADH-dehydrogenase module during assembly of the complex. Our results provide new molecular-level understanding of the ndufs4-/- mouse model and complex I-linked mitochondrial disease.\nMapping the architecture of the initiating phosphoglycosyl transferase from S. enterica O-antigen biosynthesis in a liponanoparticle.\nAuthors: Dodge, G. J.; Anderson, A. J.; He, Y.; Liu, W.; Viner, R.; Imperiali, B.\nScore: 12.6, Published: 2023-07-18 DOI: 10.1101/2023.06.16.545297\nBacterial cell surface glycoconjugates are critical for cell survival and for interactions between bacteria and their hosts. Consequently, the pathways responsible for their biosynthesis have untapped potential as therapeutic targets. The localization of many glycoconjugate biosynthesis enzymes to the membrane represents a significant challenge for expressing, purifying, and characterizing these enzymes. Here, we leverage cutting-edge methods to stabilize, purify, and structurally characterize WbaP, a phosphoglycosyl transferase (PGT) from Salmonella enterica (LT2) O-antigen biosynthesis without detergent solubilization from the lipid bilayer. From a functional perspective, these studies establish WbaP as a homodimer, reveal the structural elements responsible for oligomerization, shed light on the regulatory role of a domain of unknown function embedded within WbaP, and identify conserved structural motifs between PGTs and functionally unrelated UDP-sugar dehydratases. From a technological perspective, the strategy developed here is generalizable and provides a toolkit for studying small membrane proteins embedded in liponanoparticles beyond PGTs.\nSurprising variety in the USP deubiquitinase catalytic mechanism\nAuthors: Keijzer, N.; Priyanka, A.; Stijf-Bultsma, Y.; Fish, A.; Gersch, M.; Sixma, T. K.\nScore: 5.0, Published: 2023-07-24 DOI: 10.1101/2023.07.24.550302\nThe USP family of deubiquitinases (DUBs) controls many ubiquitin-dependent signaling events. This generates therapeutic potential, with active-site inhibitors in preclinical and clinical studies. Understanding of the USP active site was so far primarily guided by USP7 data, where the catalytic triad consists of cysteine, histidine and a third residue (first critical residue), which polarizes the histidine through a hydrogen bond. A conserved aspartate (second critical residue) is directly adjacent to this first critical residue. Here we study the roles of these critical residues in a subset of USPs and reveal a remarkable variety in function. While USP7 relies on the first critical residue for catalysis, this residue is dispensable in USP1, USP15, USP40 and USP48. Instead, their second critical residue is vital for catalysis. Interestingly, without their respective vital residue USP7, USP15 and USP40 can still perform nucleophilic attack. The diverging catalytic mechanisms of USP1 and USP7 are independent of substrate and retained in cells for USP1. The unexpected variety of catalytic mechanisms in this well-conserved protein family may generate opportunities for selective targeting of individual USPs.\n",
  "wordCount" : "1943",
  "inLanguage": "en",
  "datePublished": "2023-07-26T10:37:30Z",
  "dateModified": "2023-07-26T10:37:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta"><span>updated on July 26, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549470">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549470" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549470">
        <p class="paperTitle">A systems-scale, integrated view of the ubiquitylation site occupancy and dynamics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549470" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549470" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Prus, G.; Satpathy, S.; Weinert, B.; Narita, T.; Choudhary, C.</p>
        <p class="info">Score: 33.2, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549470' target='https://doi.org/10.1101/2023.07.19.549470'> 10.1101/2023.07.19.549470</a></p>
        <p class="abstract">Ubiquitylation regulates virtually all proteins and biological processes in a cell. However, the global site-specific occupancy (stoichiometry) and turnover rate of ubiquitylation have never been quantified. Here, we present the first integrated picture of ubiquitylation site occupancy and half-life. Ubiquitylation occupancy spans four orders of magnitude, but the median ubiquitylation site occupancy is three orders of magnitude lower than that of phosphorylation. The occupancy, turnover rate, and the regulation of sites by proteasome inhibitors show strong interrelationships. These properties can discriminate signaling-relevant sites from the sites involved in proteasomal degradation. The sites strongly upregulated by proteasome inhibitors have a longer half-life, and the half-life increases with increasing protein length. Importantly, a previously unknown surveillance mechanism rapidly deubiquitylates all ubiquitin-specific E1 and E2 enzymes and protects them against bystander ubiquitylation accumulation. This work reveals general principles of ubiquitylation-dependent governance and offers conceptual insights into the dynamic regulation of the cell.

HighlightsO_LIUbiquitylation site occupancy is 3 orders of magnitude lower than phosphorylation
C_LIO_LIThe highest 80% and the lowest 20% occupancy sites have distinct properties
C_LIO_LIHigh occupancy sites are concentrated in the cytoplasmic domains of SLC proteins
C_LIO_LIA dedicated mechanism prevevents ubiquitylation accumulation in E1s and E2s
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549963">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549963" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549963">
        <p class="paperTitle">Coenzyme Q4 is a functional substitute for coenzyme Q10 and can be targeted to the mitochondria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549963" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549963" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Steenberge, L. H.; Sung, A. Y.; Fan, J.; Pagliarini, D. J.</p>
        <p class="info">Score: 26.5, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549963' target='https://doi.org/10.1101/2023.07.20.549963'> 10.1101/2023.07.20.549963</a></p>
        <p class="abstract">Coenzyme Q10 (CoQ10) is an important cofactor and antioxidant for numerous cellular processes, and its deficiency has been linked to human disorders including mitochondrial disease, heart failure, Parkinsons disease, and hypertension. Unfortunately, treatment with exogenous oral CoQ10 is often ineffective, likely due to the extreme hydrophobicity and high molecular weight of CoQ10. Here, we show that less hydrophobic CoQ species with shorter isoprenoid tails can serve as viable substitutes for CoQ10 in human cells. We demonstrate that CoQ4 can perform multiple functions of CoQ10 in CoQ-deficient cells at markedly lower treatment concentrations, motivating further investigation of CoQ4 as a supplement for CoQ10 deficiencies. In addition, we describe the synthesis and evaluation of an initial set of compounds designed to target CoQ4 selectively to mitochondria using triphenylphosphonium (TPP). Our results indicate that select versions of these compounds can successfully be delivered to mitochondria in a cell model and be cleaved to produce CoQ4, laying the groundwork for further development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549678">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549678" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549678">
        <p class="paperTitle">Enhancing Inter-link Coverage in Cross-Linking Mass Spectrometry through Context-Sensitive Subgrouping and Decoy Fusion</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549678" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549678" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bogdanow, B.; Wang, C.; Ruwolt, M.; Ruta, J.; Mühlberg, L.; Zeng, W.-F.; Elofsson, A.; Liu, F.</p>
        <p class="info">Score: 10.0, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549678' target='https://doi.org/10.1101/2023.07.19.549678'> 10.1101/2023.07.19.549678</a></p>
        <p class="abstract">In cross-linking mass spectrometry, sensitivity and specificity in assigning mass spectra to cross-links between different proteins (inter-links) remains challenging. Here, we report on limitations of commonly used concatenated target-decoy searches and propose a target-decoy competition strategy on a fused database as a solution. Further, we capitalize on context-divergent error rates by implementing a novel context-sensitive subgrouping strategy. This approach increases inter-link coverage by [~] 30 - 75 % across XL-MS datasets, maintains low error rates, and preserves structural accuracy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549248">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549248" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549248">
        <p class="paperTitle">FAIMS-enabled N-terminomics analysis reveals novel legumain substrates in murine spleen</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549248" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549248" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ziegler, A. R.; Dufour, A.; Scott, N. E.; Edgington-Mitchell, L. E.</p>
        <p class="info">Score: 8.9, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549248' target='https://doi.org/10.1101/2023.07.18.549248'> 10.1101/2023.07.18.549248</a></p>
        <p class="abstract">Aberrant levels of the asparaginyl endopeptidase legumain have been linked to inflammation, neurodegeneration and cancer, yet our understanding of this protease is incomplete. Systematic attempts to identify legumain substrates have previously been confined to in vitro studies, which fail to mirror physiological conditions and obscure biologically relevant cleavage events. Using high-field asymmetric waveform ion mobility spectrometry (FAIMS), we developed a sensitive and streamlined approach for proteome and N-terminome analyses in a single analytical method without the need for N-termini enrichment. Compared to unfractionated proteomic analysis, we demonstrate FAIMS fractionation improves neo-N- termini identification by &gt;2.5 fold, resulting in identification of &gt;2,882 unique neo-N-termini from limited sample amounts. Within murine spleens, this approach identifies 6,366 proteins and 2,528 unique neo-N-termini, with 235 cleavage events enriched in wild-type compared to legumain-deficient spleens. Among these, 119 neo-N-termini arose from asparaginyl endopeptidase activities, representing novel putative physiological legumain substrates. The direct cleavage of selected substrates by legumain was confirmed using in vitro assays, providing support for the existence of physiologically relevant extra-lysosomal legumain activity. Combined, these data shed critical light on the functions of legumain and demonstrates the utility of FAIMS as an accessible method to improve depth and quality of N- terminomics studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549407">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549407" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549407">
        <p class="paperTitle">Interaction with the membrane-anchored protein CHIC2 constrains the ubiquitin ligase activity of CHIP</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549407" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549407" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Callahan, M. D.; Hodul, M.; Carroll, E. C.; Ravalin, M.; Nadel, C. M.; de Silva, A. R.; Cupo, A. R.; McDermott, L. E.; Nix, J. C.; Page, R. C.; Kao, A. W.; Gestwicki, J. E.</p>
        <p class="info">Score: 8.8, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549407' target='https://doi.org/10.1101/2023.07.17.549407'> 10.1101/2023.07.17.549407</a></p>
        <p class="abstract">Maintenance of cellular health requires the proper regulation of E3 ubiquitin ligases. The E3 ligase CHIP is canonically regulated by its interactions with the molecular chaperones Hsp70 and Hsp90, which focus CHIPs ubiquitination activity on misfolded proteins. Here, we report a chaperone-independent interaction of CHIP with the membrane-anchored protein CHIC2, which strongly attenuates CHIPs ligase activity. We show that CHIC2 outcompetes abundant, cytosolic chaperones through its exquisite CHIP selectivity, rather than through enhanced affinity. In proteomic experiments, we find that CHIC2 knockout phenocopies CHIP knockout in certain cell types, implying that chaperone-independent interactions can sometimes predominate CHIPs functions. Furthermore, loss of the CHIP-CHIC2 interaction induces neurodegeneration and shortens lifespan in C. elegans, demonstrating that formation of this chaperone-independent complex is important in animals. We propose that CHIC2 attenuates CHIP activity at the membrane, offering a novel mechanism by which this ubiquitin ligase can be regulated.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549906">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549906" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549906">
        <p class="paperTitle">The hGID GID4 E3 ubiquitin ligase complex targets ARHGAP11A to regulate cell migration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549906" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549906" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bagci, H.; Winkler, M.; Uliana, F.; Boulais, J.; Mohamed, W. I.; Park, S. L.; Cote, J.-F.; Peter, M.</p>
        <p class="info">Score: 8.5, Published: 2023-07-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549906' target='https://doi.org/10.1101/2023.07.20.549906'> 10.1101/2023.07.20.549906</a></p>
        <p class="abstract">The human CTLH/GID (hGID) complex emerged as an important E3 ligase regulating multiple cellular processes, including cell cycle progression and metabolic activity. However, the range of biological functions controlled by hGID remains unexplored. Here, we show that the hGID substrate receptor GID4 regulates cell growth and migration. Biochemical and cellular assays combined with proximity-dependent biotinylation (BioID2) revealed that the hGIDGID4 E3-ligase targets the Rho-GAP ARHGAP11A for degradation. Depletion of GID4 or impeding the GID4 substrate binding pocket impairs motility and directed cell movement, whereas knockdown of ARHGAP11A significantly restores the cell migration defect. We found that GID4 controls cell migration by degrading ARHGAP11A thereby preventing its accumulation at the cell periphery where it inactivates RhoA activity. Together, we identified a unique function for GID4, as well as a wide range of substrate profiles beyond Pro/N-degron motifs, which pave the way for deciphering additional pathways regulated by hGID E3 ligase activity through its GID4 substrate receptor.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549978">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549978" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549978">
        <p class="paperTitle">Structural characterisation of the complete cycle of sliding clamp loading in Escherichia coli</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549978" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549978" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xu, Z.-Q.; Jergic, S.; Lo, A. T. Y.; Pradhan, A. C.; Brown, S. H. J.; Bouwer, J. C.; Ghodke, H.; Lewis, P. J.; Tolun, G.; Oakley, A. J.; Dixon, N. E.</p>
        <p class="info">Score: 8.1, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549978' target='https://doi.org/10.1101/2023.07.20.549978'> 10.1101/2023.07.20.549978</a></p>
        <p class="abstract">Ring-shaped DNA sliding clamps are essential for DNA replication and genome maintenance. Clamps need to be opened or trapped open and chaperoned onto DNA by clamp loader complexes (CLCs). Detailed understanding of the mechanisms by which CLCs open and place clamps around DNA remains limited. Here, we present a series of six structures of the Escherichia coli CLC bound to an open or closed clamp on and off a primer-template DNA that represent all intermediates in the clamp loading process. We show that the ATP-bound CLC first binds to a clamp, then constricts to hold onto it. The CLC then expands to open the clamp with a gap large enough for double-stranded DNA to enter. Upon binding to DNA, the CLC constricts slightly, allowing ATP hydrolysis and clamp closing around DNA. Although both yeast and E. coli CLCs open clamps by crab claw-like motions, they do it by the CLC expanding in opposite directions. These structures provide critical high-resolution snapshots of clamp loading by the E. coli CLC, revealing how the molecular machine works.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549284">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549284" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549284">
        <p class="paperTitle">Structural insights into complex I deficiency and assembly from the disease-related ndufs4-/- mouse</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549284" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549284" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yin, Z.; Agip, A.-N. A.; Bridges, H.; Hirst, J.</p>
        <p class="info">Score: 5.6, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549284' target='https://doi.org/10.1101/2023.07.17.549284'> 10.1101/2023.07.17.549284</a></p>
        <p class="abstract">Respiratory complex I (NADH:ubiquinone oxidoreductase) is essential for cellular energy production and NAD&#43; homeostasis. Complex I mutations cause neuromuscular, mitochondrial diseases, such as Leigh Syndrome, but their molecular-level consequences remain poorly understood. Here, we use a popular complex I-linked mitochondrial disease model, the ndufs4-/- mouse, to define the structural, biochemical and functional consequences of the absence of subunit NDUFS4. Cryo-EM analyses of mouse-heart ndufs4-/- complex I revealed a loose association of the NADH-dehydrogenase module, and discrete classes containing either assembly factor NDUFAF2 or subunit NDUFS6. Subunit NDUFA12 (that replaces its paralogue NDUFAF2 in mature complex I) is absent from all classes, compounding the deletion of NDUFS4 and preventing maturation of an NDUFS4-free but otherwise complete enzyme. We propose NDUFAF2 as the recruiter of the NADH-dehydrogenase module during assembly of the complex. Our results provide new molecular-level understanding of the ndufs4-/- mouse model and complex I-linked mitochondrial disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.16.545297">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.16.545297" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.16.545297">
        <p class="paperTitle">Mapping the architecture of the initiating phosphoglycosyl transferase from S. enterica O-antigen biosynthesis in a liponanoparticle.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.16.545297" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.16.545297" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dodge, G. J.; Anderson, A. J.; He, Y.; Liu, W.; Viner, R.; Imperiali, B.</p>
        <p class="info">Score: 12.6, Published: 2023-07-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.16.545297' target='https://doi.org/10.1101/2023.06.16.545297'> 10.1101/2023.06.16.545297</a></p>
        <p class="abstract">Bacterial cell surface glycoconjugates are critical for cell survival and for interactions between bacteria and their hosts. Consequently, the pathways responsible for their biosynthesis have untapped potential as therapeutic targets. The localization of many glycoconjugate biosynthesis enzymes to the membrane represents a significant challenge for expressing, purifying, and characterizing these enzymes. Here, we leverage cutting-edge methods to stabilize, purify, and structurally characterize WbaP, a phosphoglycosyl transferase (PGT) from Salmonella enterica (LT2) O-antigen biosynthesis without detergent solubilization from the lipid bilayer. From a functional perspective, these studies establish WbaP as a homodimer, reveal the structural elements responsible for oligomerization, shed light on the regulatory role of a domain of unknown function embedded within WbaP, and identify conserved structural motifs between PGTs and functionally unrelated UDP-sugar dehydratases. From a technological perspective, the strategy developed here is generalizable and provides a toolkit for studying small membrane proteins embedded in liponanoparticles beyond PGTs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550302">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550302" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550302">
        <p class="paperTitle">Surprising variety in the USP deubiquitinase catalytic mechanism</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550302" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550302" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Keijzer, N.; Priyanka, A.; Stijf-Bultsma, Y.; Fish, A.; Gersch, M.; Sixma, T. K.</p>
        <p class="info">Score: 5.0, Published: 2023-07-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550302' target='https://doi.org/10.1101/2023.07.24.550302'> 10.1101/2023.07.24.550302</a></p>
        <p class="abstract">The USP family of deubiquitinases (DUBs) controls many ubiquitin-dependent signaling events. This generates therapeutic potential, with active-site inhibitors in preclinical and clinical studies. Understanding of the USP active site was so far primarily guided by USP7 data, where the catalytic triad consists of cysteine, histidine and a third residue (first critical residue), which polarizes the histidine through a hydrogen bond. A conserved aspartate (second critical residue) is directly adjacent to this first critical residue. Here we study the roles of these critical residues in a subset of USPs and reveal a remarkable variety in function. While USP7 relies on the first critical residue for catalysis, this residue is dispensable in USP1, USP15, USP40 and USP48. Instead, their second critical residue is vital for catalysis. Interestingly, without their respective vital residue USP7, USP15 and USP40 can still perform nucleophilic attack. The diverging catalytic mechanisms of USP1 and USP7 are independent of substrate and retained in cells for USP1. The unexpected variety of catalytic mechanisms in this well-conserved protein family may generate opportunities for selective targeting of individual USPs.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
