<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Intrinsically Disordered Regions Promote Protein Refoldability and Facilitate Retrieval from Biomolecular Condensates
Authors: To, P.; Bhagwat, A. M.; Tarbox, H. E.; Ecer, A.; Wendorff, H.; Jamieson, Z.; Trcek, T.; Fried, S. D.
Score: 110.3, Published: 2023-06-25 DOI: 10.1101/2023.06.25.546465
Many eukaryotic proteins contain intrinsically disordered regions (IDRs) that intersperse globular folded domains, in contrast with bacterial proteins which are typically highly globular. Recent years have seen great progress in identifying biological functions associated with these elusive protein sequence: in specific cases, they mediate liquid-liquid phase separation, perform molecular recognition, or act as sensors to changes in the environment.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Intrinsically Disordered Regions Promote Protein Refoldability and Facilitate Retrieval from Biomolecular Condensates
Authors: To, P.; Bhagwat, A. M.; Tarbox, H. E.; Ecer, A.; Wendorff, H.; Jamieson, Z.; Trcek, T.; Fried, S. D.
Score: 110.3, Published: 2023-06-25 DOI: 10.1101/2023.06.25.546465
Many eukaryotic proteins contain intrinsically disordered regions (IDRs) that intersperse globular folded domains, in contrast with bacterial proteins which are typically highly globular. Recent years have seen great progress in identifying biological functions associated with these elusive protein sequence: in specific cases, they mediate liquid-liquid phase separation, perform molecular recognition, or act as sensors to changes in the environment." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-06-28T11:45:43+00:00" />
<meta property="article:modified_time" content="2023-06-28T11:45:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Intrinsically Disordered Regions Promote Protein Refoldability and Facilitate Retrieval from Biomolecular Condensates
Authors: To, P.; Bhagwat, A. M.; Tarbox, H. E.; Ecer, A.; Wendorff, H.; Jamieson, Z.; Trcek, T.; Fried, S. D.
Score: 110.3, Published: 2023-06-25 DOI: 10.1101/2023.06.25.546465
Many eukaryotic proteins contain intrinsically disordered regions (IDRs) that intersperse globular folded domains, in contrast with bacterial proteins which are typically highly globular. Recent years have seen great progress in identifying biological functions associated with these elusive protein sequence: in specific cases, they mediate liquid-liquid phase separation, perform molecular recognition, or act as sensors to changes in the environment."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Intrinsically Disordered Regions Promote Protein Refoldability and Facilitate Retrieval from Biomolecular Condensates\nAuthors: To, P.; Bhagwat, A. M.; Tarbox, H. E.; Ecer, A.; Wendorff, H.; Jamieson, Z.; Trcek, T.; Fried, S. D.\nScore: 110.3, Published: 2023-06-25 DOI: 10.1101/2023.06.25.546465\nMany eukaryotic proteins contain intrinsically disordered regions (IDRs) that intersperse globular folded domains, in contrast with bacterial proteins which are typically highly globular. Recent years have seen great progress in identifying biological functions associated with these elusive protein sequence: in specific cases, they mediate liquid-liquid phase separation, perform molecular recognition, or act as sensors to changes in the environment.",
  "keywords": [
    
  ],
  "articleBody": " Intrinsically Disordered Regions Promote Protein Refoldability and Facilitate Retrieval from Biomolecular Condensates\nAuthors: To, P.; Bhagwat, A. M.; Tarbox, H. E.; Ecer, A.; Wendorff, H.; Jamieson, Z.; Trcek, T.; Fried, S. D.\nScore: 110.3, Published: 2023-06-25 DOI: 10.1101/2023.06.25.546465\nMany eukaryotic proteins contain intrinsically disordered regions (IDRs) that intersperse globular folded domains, in contrast with bacterial proteins which are typically highly globular. Recent years have seen great progress in identifying biological functions associated with these elusive protein sequence: in specific cases, they mediate liquid-liquid phase separation, perform molecular recognition, or act as sensors to changes in the environment. Nevertheless, only a small number of IDRs have annotated functions despite their presence in 64% of yeast proteins, stimulating some to question what 'general purpose' they may serve. Here, by interrogating the refoldability of two fungal proteomes (Saccharomyces cerevisiae and Neurosporra crassa), we show that IDRs render their host proteins more refoldable from the denatured state, allowing them to cohere more closely to Anfinsen's thermodynamic hypothesis. The data provide an exceptionally clear picture of which biophysical and topological characteristics enable refoldability. Moreover, we find that almost all yeast proteins that partition into stress granules during heat shock are refoldable, a finding that holds for other condensates such as P-bodies and the nucleolus. Finally, we find that the Hsp104 unfoldase is the principal actor in mediating disassembly of heat stress granules and that the efficiency with which condensed proteins are returned to the soluble phase is also well explained by refoldability. Hence, these studies establish spontaneous refoldability as an adaptive trait that endows proteins with the capacity to reform their native soluble structures following their extraction from condensates. Altogether, our results provide an intuitive model for the function of IDRs in many multidomain proteins and clarifies their relationship to the phenomenon of biomolecular condensation.\nEvaluating the performance of the Astral mass analyzer for quantitative proteomics using data independent acquisition\nAuthors: Heil, L. R.; Damoc, N. E.; Arrey, T. N.; Pashkova, A.; Denisov, E.; Petzoldt, J.; Peterson, A.; Hsu, C.; Searle, B. C.; Shulman, N.; Riffle, M.; Connolly, B.; MacLean, B. X.; Remes, P. M.; Senko, M.; Stewart, H.; Hock, C.; Makarov, A.; Hermanson, D.; Zabrouskov, V.; Wu, C. C.; MacCoss, M. J.\nScore: 53.9, Published: 2023-06-03 DOI: 10.1101/2023.06.03.543570\nWe evaluate the quantitative performance of the newly released Asymmetric Track Lossless (Astral) analyzer. Using data independent acquisition, the Thermo Scientific Orbitrap Astral mass spectrometer quantifies 5 times more peptides per unit time than state-of-the-art Thermo Scientific Orbitrap mass spectrometers, which have long been the gold standard for high resolution quantitative proteomics. Our results demonstrate that the Orbitrap Astral mass spectrometer can produce high quality quantitative measurements across a wide dynamic range. We also use a newly developed extra-cellular vesicle enrichment protocol to reach new depths of coverage in the plasma proteome, quantifying over 5,000 plasma proteins in a 60-minute gradient with the Orbitrap Astral mass spectrometer. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=111 SRC=\"FIGDIR/small/543570v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (28K): org.highwire.dtl.DTLVardef@128156dorg.highwire.dtl.DTLVardef@96c051org.highwire.dtl.DTLVardef@12c25f1org.highwire.dtl.DTLVardef@16eff26_HPS_FORMAT_FIGEXP M_FIG C_FIG\nUnraveling the mechanisms of PAMless DNA interrogation by SpRY Cas9\nAuthors: Hibshman, G. N.; Bravo, J. P.; Zhang, H.; Dangerfield, T. L.; Finkelstein, I. J.; Johnson, K. A.; Taylor, D. W.\nScore: 13.7, Published: 2023-06-22 DOI: 10.1101/2023.06.22.546082\nCRISPR-Cas9 is a powerful tool for genome editing, but the strict requirement for an \"NGG\" protospacer-adjacent motif (PAM) sequence immediately adjacent to the DNA target limits the number of editable genes. To overcome the PAM requirement, a recently developed Cas9 variant, called SpRY-Cas9 was engineered to be \"PAMless\" (1, 2). However, the molecular mechanisms of how SpRY can recognize all potential PAM sequences and still accurately identify DNA targets have not been investigated. Here, we combined enzyme kinetics, cryo-EM, and single-molecule imaging to determine how SpRY interrogates DNA and recognizes target sites for cleavage. Divergent PAM sequences can be accommodated through conformational flexibility within the PAM-interacting region of SpRY, which facilitates tight binding to off-target DNA sequences. Once SpRY correctly identifies a target site, nuclease activation occurs [~]1,000-fold slower than for Streptococcus pyogenes Cas9, enabling us to directly visualize multiple on-pathway intermediate states. Insights gained from our intermediate structures prompted rationally designed mutants with improved DNA cleavage efficiency. Our findings shed light on the molecular mechanisms of PAMless genome editing with SpRY and provide a framework for the design of future genome editing tools with improved versatility, precision, and efficiency.\nContext-aware geometric deep learning for protein sequence design\nAuthors: Krapp, L.; Meireles, F.; Abriata, L.; Dal Peraro, M.\nScore: 29.8, Published: 2023-06-19 DOI: 10.1101/2023.06.19.545381\nProtein design and engineering are evolving at an unprecedented pace leveraging the advances of deep learning. Current models nonetheless cannot natively consider non-protein entities within the design process. Here we introduce a deep learning approach based solely on a geometric transformer of atomic coordinates that predicts protein sequences from backbone scaffolds aware of the restraints imposed by diverse molecular environments. This new concept is anticipated to improve the design versatility for engineering proteins with desired functions.\nBimodular Architecture of Bacterial Effector SAP05 Drives Ubiquitin-Independent Targeted Protein Degradation\nAuthors: Liu, Q.; Maqbool, A.; Mirkin, F. G.; Singh, Y.; Stevenson, C. E.; Lawson, D. M.; Kamoun, S.; Huang, W.; Hogenhout, S. A.\nScore: 35.7, Published: 2023-06-19 DOI: 10.1101/2023.06.19.545293\nIn eukaryotes, targeted protein degradation (TPD) typically depends on a series of interactions among ubiquitin ligases that transfer ubiquitin molecules to substrates leading to degradation by the 26S proteasome. We previously discovered that the bacterial effector protein SAP05 mediates ubiquitin-independent TPD. SAP05 forms a ternary complex via interactions with the von Willebrand Factor Type A (vWA) domain of the proteasomal ubiquitin receptor Rpn10 and the Zinc-finger (ZnF) domains of the SQUAMOSA-PROMOTER BINDING PROTEIN-LIKE (SPL) and GATA BINDING FACTOR (GATA) transcription factors (TFs). This leads to direct TPD of the TFs by the 26S proteasome. Here, we report the crystal structures of the SAP05-vWA complex at 2.17 [A] resolution and of the SAP05-ZnF(SPL5) complex at 2.20 [A] resolution. Structural analyses revealed that SAP05 displays a unique bimodular architecture with two distinct non-overlapping surfaces, a loop surface with three protruding loops that form electrostatic interactions with ZnF, and a sheet surface featuring two {beta}-sheets, loops and -helices that establish polar interactions with vWA. SAP05 binding to ZnF TFs involves single amino acids responsible for multiple contacts, while SAP05 binding to vWA is more stable due to the necessity of multiple mutations to break the interaction. In addition, positioning of the SAP05 complex on the 26S proteasome points to a mechanism of protein degradation. Collectively, our findings demonstrate how a small bacterial bimodular protein can bypass the canonical UPS cellular proteolysis pathway, enabling ubiquitin-independent TPD in eukaryotic cells. This knowledge holds significant potential for the creation of novel TPD technologies.\nFluorogenesis: Inducing Fluorescence in a Non-Fluorescent Protein Through Photoinduced Chromophore Transfer of a Genetically Encoded Chromophore\nAuthors: Kumar, Y.; Singh, R. K.; Ojha, M.; Pushpavanam, K.\nScore: 9.7, Published: 2023-06-25 DOI: 10.1101/2023.06.24.546416\nFluorescent proteins, while essential for bioimaging, are limited to visualizing cellular localization without offering additional functionality. We report for the first time a strategy to expand the chemical, structural, and functional diversity of fluorescent proteins by harnessing light to induce red fluorescence in a previously non-fluorescent protein. We accomplish this by inducing the transfer of the genetically encoded chromophore from a photocleavable protein (PhoCl1) to a non-fluorescent kinase (MjRibK) inducing red fluorescence in the latter. We have employed analytical and spectroscopic techniques to validate the presence of red fluorescence in MjRibK. Furthermore, molecular dynamics simulations were carried out to investigate the amino acid residues of MjRibK involved in the generation of red fluorescence. Finally, we demonstrate the ability of the red fluorescent MjRibK to operate as a cyclable high-temperature sensor. We anticipate that this light-induced chromophore transfer strategy will open new possibilities for developing multifunctional genetically encoded fluorescent sensors.\nMag-Net: Rapid enrichment of membrane-bound particles enables high coverage quantitative analysis of the plasma proteome\nAuthors: Wu, C. C.; Tsantilas, K. A.; Park, J.; Plubell, D. L.; Naicker, P.; Govender, I.; Buthelezi, S.; Stoychev, S.; Jordaan, J.; Merrihew, G. E.; Huang, E.; Parker, E. D.; Riffle, M.; Hoofnagle, A. N.; MacCoss, M. J.\nScore: 28.3, Published: 2023-06-11 DOI: 10.1101/2023.06.10.544439\nMembrane-bound particles in plasma are composed of exosomes, microvesicles, and apoptotic bodies and represent [~]1-2% of the total protein composition. Proteomic interrogation of this subset of plasma proteins augments the representation of tissue-specific proteins, representing a \"liquid biopsy,\" while enabling the detection of proteins that would otherwise be beyond the dynamic range of liquid chromatography-tandem mass spectrometry in unfractionated plasma. We have developed a one-step enrichment strategy (Mag-Net) using hyper-porous strong-anion exchange magnetic microparticles to sieve membrane-bound particles from plasma. The Mag-Net method is robust, reproducible, inexpensive, and requires \u003c100 L plasma input. Coupled to a quantitative data-independent mass spectrometry analytical strategy, we demonstrate that we can routinely collect results for \u003e37,000 peptides from \u003e4,000 plasma proteins with high precision. We demonstrate excellent quantitative accuracy and analytical reproducibility of the protocol.\nTargeting the conserved active site of splicing machines with specific and selective small molecule modulators\nAuthors: Silvestri, I.; Manigrasso, J.; Andreani, A.; Brindani, N.; De Vivo, M.; Marcia, M.\nScore: 9.2, Published: 2023-06-22 DOI: 10.1101/2023.06.21.545906\nThe self-splicing group II introns are bacterial and organellar ancestors of the nuclear spliceosome and retro-transposable elements of pharmacological and biotechnological importance. Integrating enzymatic, crystallographic, and simulation studies, we demonstrate how these introns recognize small molecules through their conserved active site. These RNA-binding small molecules selectively inhibit the two steps of splicing by adopting distinctive poses at different stages of catalysis, and by preventing crucial active site conformational changes that are essential for splicing progression. Our data exemplify the enormous power of RNA binders to mechanistically probe vital cellular pathways. Most importantly, by proving that the evolutionarily-conserved RNA core of splicing machines can recognize small molecules specifically, our work puts solid bases for the rational design of splicing modulators not only against bacterial and organellar introns, but also against the human spliceosome, which is a validated drug target for the treatment of congenital diseases and cancers.\nOmics Scale Quantitative Mass Spectrometry Imaging of Lipids in Brain Tissue using a Multi-Class Internal Standard Mixture\nAuthors: Vandenbosch, M.; Mutuku, S. M.; Mantas, M. J. Q.; Patterson, N. H.; Hallmark, T.; Claesen, M.; Heeren, R. M. A.; Hatcher, N. G.; Verbeeck, N.; Ekroos, K.; Ellis, S. R.\nScore: 7.8, Published: 2023-06-22 DOI: 10.1101/2023.06.21.546027\nMass spectrometry imaging (MSI) has accelerated the understanding of lipid metabolism and spatial distribution in tissues and cells. However, few MSI studies have approached lipid imaging quantitatively and those that have focus on a single lipid class. Herein, we overcome limitation of quantitative MSI (Q-MSI) by using a multi-class internal standard lipid mixture that is sprayed homogenously over the tissue surface with analytical concentrations that reflects endogenous brain lipid levels. Using this approach we have performed Q-MSI for 13 lipid classes representing \u003e200 sum-composition lipid species. This was carried out using both MALDI (negative ion mode) and MALDI-2 (positive ion mode) and pixel-wise normalisation of each lipid species signal to the corresponding class-specific IS an approach analogous to that widely used for shotgun lipidomics from biological extracts. This approach allows pixel concentrations of lipids to be reported in pmol/mm2. Q-MSI of lipids covered 3 orders of magnitude in dynamic range and revealed subtle change sin in distribution compared to conventional total-ion-current normalisation approaches. The robustness of the method was evaluated by repeating experiments in two laboratories on biological replicates using both timsTOF and Orbitrap mass spectrometers operated with a ~4-fold difference in mass resolution power. There was a strong overall correlation in the Q-MSI result obtained using the two approaches with outliers mostly rationalised by isobaric interferences that are only resolved with the Orbitrap system or the higher sensitivity of one instrument for particular lipid species, particularly for lipids detected at low intensity. These data provide insight into how mass resolving power can affect Q-MSI data. This approach opens up the possibility of performing large-scale Q-MSI studies across numerous lipid classes and reveal how absolute lipid concentrations vary throughout and between biological tissues.\nType III-B CRISPR-Cas signaling-based cascade of proteolytic cleavages\nAuthors: Steens, J.; Bravo, J. P.; Salazar, C. R. P.; Yildiz, C.; Amieiro, A. M.; Kostlbacher, S.; Prinsen, S. H. P.; Patinios, C.; Bardis, A.; Barendrecht, A.; Scheltema, R. A.; Ettema, T. J. G.; van der Oost, J.; Taylor, D. W.; Staals, R.\nScore: 6.7, Published: 2023-06-23 DOI: 10.1101/2023.06.23.546230\nType III CRISPR-Cas systems provide a sequence-specific adaptive immune response that protects prokaryotic hosts against viruses and other foreign genetic invaders. These crRNA-guided Cas effector complexes bind and cleave complementary RNA targets. Specific target binding stimulates the Cas10 subunit to generate cyclic oligoadenylate (cOA) signaling molecules, that in turn allosterically activate proteins carrying cognate sensory domains: CARF or SAVED. Here, we characterize an elaborate set of genes associated with the type III-B CRISPR-Cas system from Haliangium ochraceum, which includes a signal transduction module of a CBASS defense system with two caspase-like proteases, SAVED-CHAT and PCaspase (Prokaryotic Caspase). We show that binding of a 3-nucleotide cOA (cA3) to the SAVED domain of SAVED-CHAT induces its oligomerization into long filaments that activate the proteolytic activity of the CHAT domain. Surprisingly, we find that activated SAVED-CHAT specifically cleaves and activates the second protease, PCaspase. In turn, activated PCaspase cleaves a multitude of other proteins, including a putative sigma factor and a PCaspase-inhibitor. We expressed the type III-B system and its associated genes in E. coli and observed a strong abortive phenotype when offering a complementary target RNA, but only in the presence of both SAVED-CHAT and PCaspase. Together, our findings show an intriguing cascade of proteolytic activities (conceptually similar to eukaryotic caspases) in this bacterial immune system that reveals yet another strategy to effectively defend against mobile genetic elements.\n",
  "wordCount" : "2281",
  "inLanguage": "en",
  "datePublished": "2023-06-28T11:45:43Z",
  "dateModified": "2023-06-28T11:45:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta"><span>updated on June 28, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.25.546465">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.25.546465" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.25.546465">
        <p class="paperTitle">Intrinsically Disordered Regions Promote Protein Refoldability and Facilitate Retrieval from Biomolecular Condensates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.25.546465" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.25.546465" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: To, P.; Bhagwat, A. M.; Tarbox, H. E.; Ecer, A.; Wendorff, H.; Jamieson, Z.; Trcek, T.; Fried, S. D.</p>
        <p class="info">Score: 110.3, Published: 2023-06-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.25.546465' target='https://doi.org/10.1101/2023.06.25.546465'> 10.1101/2023.06.25.546465</a></p>
        <p class="abstract">Many eukaryotic proteins contain intrinsically disordered regions (IDRs) that intersperse globular folded domains, in contrast with bacterial proteins which are typically highly globular. Recent years have seen great progress in identifying biological functions associated with these elusive protein sequence: in specific cases, they mediate liquid-liquid phase separation, perform molecular recognition, or act as sensors to changes in the environment. Nevertheless, only a small number of IDRs have annotated functions despite their presence in 64% of yeast proteins, stimulating some to question what &#39;general purpose&#39; they may serve. Here, by interrogating the refoldability of two fungal proteomes (Saccharomyces cerevisiae and Neurosporra crassa), we show that IDRs render their host proteins more refoldable from the denatured state, allowing them to cohere more closely to Anfinsen&#39;s thermodynamic hypothesis. The data provide an exceptionally clear picture of which biophysical and topological characteristics enable refoldability. Moreover, we find that almost all yeast proteins that partition into stress granules during heat shock are refoldable, a finding that holds for other condensates such as P-bodies and the nucleolus. Finally, we find that the Hsp104 unfoldase is the principal actor in mediating disassembly of heat stress granules and that the efficiency with which condensed proteins are returned to the soluble phase is also well explained by refoldability. Hence, these studies establish spontaneous refoldability as an adaptive trait that endows proteins with the capacity to reform their native soluble structures following their extraction from condensates. Altogether, our results provide an intuitive model for the function of IDRs in many multidomain proteins and clarifies their relationship to the phenomenon of biomolecular condensation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.03.543570">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.03.543570" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.03.543570">
        <p class="paperTitle">Evaluating the performance of the Astral mass analyzer for quantitative proteomics using data independent acquisition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.03.543570" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.03.543570" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Heil, L. R.; Damoc, N. E.; Arrey, T. N.; Pashkova, A.; Denisov, E.; Petzoldt, J.; Peterson, A.; Hsu, C.; Searle, B. C.; Shulman, N.; Riffle, M.; Connolly, B.; MacLean, B. X.; Remes, P. M.; Senko, M.; Stewart, H.; Hock, C.; Makarov, A.; Hermanson, D.; Zabrouskov, V.; Wu, C. C.; MacCoss, M. J.</p>
        <p class="info">Score: 53.9, Published: 2023-06-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.03.543570' target='https://doi.org/10.1101/2023.06.03.543570'> 10.1101/2023.06.03.543570</a></p>
        <p class="abstract">We evaluate the quantitative performance of the newly released Asymmetric Track Lossless (Astral) analyzer. Using data independent acquisition, the Thermo Scientific Orbitrap Astral mass spectrometer quantifies 5 times more peptides per unit time than state-of-the-art Thermo Scientific Orbitrap mass spectrometers, which have long been the gold standard for high resolution quantitative proteomics. Our results demonstrate that the Orbitrap Astral mass spectrometer can produce high quality quantitative measurements across a wide dynamic range. We also use a newly developed extra-cellular vesicle enrichment protocol to reach new depths of coverage in the plasma proteome, quantifying over 5,000 plasma proteins in a 60-minute gradient with the Orbitrap Astral mass spectrometer.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=111 SRC=&#34;FIGDIR/small/543570v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (28K):
org.highwire.dtl.DTLVardef@128156dorg.highwire.dtl.DTLVardef@96c051org.highwire.dtl.DTLVardef@12c25f1org.highwire.dtl.DTLVardef@16eff26_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.22.546082">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.22.546082" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.22.546082">
        <p class="paperTitle">Unraveling the mechanisms of PAMless DNA interrogation by SpRY Cas9</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.22.546082" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.22.546082" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hibshman, G. N.; Bravo, J. P.; Zhang, H.; Dangerfield, T. L.; Finkelstein, I. J.; Johnson, K. A.; Taylor, D. W.</p>
        <p class="info">Score: 13.7, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.22.546082' target='https://doi.org/10.1101/2023.06.22.546082'> 10.1101/2023.06.22.546082</a></p>
        <p class="abstract">CRISPR-Cas9 is a powerful tool for genome editing, but the strict requirement for an &#34;NGG&#34; protospacer-adjacent motif (PAM) sequence immediately adjacent to the DNA target limits the number of editable genes. To overcome the PAM requirement, a recently developed Cas9 variant, called SpRY-Cas9 was engineered to be &#34;PAMless&#34; (1, 2). However, the molecular mechanisms of how SpRY can recognize all potential PAM sequences and still accurately identify DNA targets have not been investigated. Here, we combined enzyme kinetics, cryo-EM, and single-molecule imaging to determine how SpRY interrogates DNA and recognizes target sites for cleavage. Divergent PAM sequences can be accommodated through conformational flexibility within the PAM-interacting region of SpRY, which facilitates tight binding to off-target DNA sequences. Once SpRY correctly identifies a target site, nuclease activation occurs [~]1,000-fold slower than for Streptococcus pyogenes Cas9, enabling us to directly visualize multiple on-pathway intermediate states. Insights gained from our intermediate structures prompted rationally designed mutants with improved DNA cleavage efficiency. Our findings shed light on the molecular mechanisms of PAMless genome editing with SpRY and provide a framework for the design of future genome editing tools with improved versatility, precision, and efficiency.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.545381">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.545381" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.545381">
        <p class="paperTitle">Context-aware geometric deep learning for protein sequence design</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.545381" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.545381" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krapp, L.; Meireles, F.; Abriata, L.; Dal Peraro, M.</p>
        <p class="info">Score: 29.8, Published: 2023-06-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.545381' target='https://doi.org/10.1101/2023.06.19.545381'> 10.1101/2023.06.19.545381</a></p>
        <p class="abstract">Protein design and engineering are evolving at an unprecedented pace leveraging the advances of deep learning. Current models nonetheless cannot natively consider non-protein entities within the design process. Here we introduce a deep learning approach based solely on a geometric transformer of atomic coordinates that predicts protein sequences from backbone scaffolds aware of the restraints imposed by diverse molecular environments. This new concept is anticipated to improve the design versatility for engineering proteins with desired functions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.545293">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.545293" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.545293">
        <p class="paperTitle">Bimodular Architecture of Bacterial Effector SAP05 Drives Ubiquitin-Independent Targeted Protein Degradation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.545293" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.545293" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, Q.; Maqbool, A.; Mirkin, F. G.; Singh, Y.; Stevenson, C. E.; Lawson, D. M.; Kamoun, S.; Huang, W.; Hogenhout, S. A.</p>
        <p class="info">Score: 35.7, Published: 2023-06-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.545293' target='https://doi.org/10.1101/2023.06.19.545293'> 10.1101/2023.06.19.545293</a></p>
        <p class="abstract">In eukaryotes, targeted protein degradation (TPD) typically depends on a series of interactions among ubiquitin ligases that transfer ubiquitin molecules to substrates leading to degradation by the 26S proteasome. We previously discovered that the bacterial effector protein SAP05 mediates ubiquitin-independent TPD. SAP05 forms a ternary complex via interactions with the von Willebrand Factor Type A (vWA) domain of the proteasomal ubiquitin receptor Rpn10 and the Zinc-finger (ZnF) domains of the SQUAMOSA-PROMOTER BINDING PROTEIN-LIKE (SPL) and GATA BINDING FACTOR (GATA) transcription factors (TFs). This leads to direct TPD of the TFs by the 26S proteasome. Here, we report the crystal structures of the SAP05-vWA complex at 2.17 [A] resolution and of the SAP05-ZnF(SPL5) complex at 2.20 [A] resolution. Structural analyses revealed that SAP05 displays a unique bimodular architecture with two distinct non-overlapping surfaces, a  loop surface with three protruding loops that form electrostatic interactions with ZnF, and a  sheet surface featuring two {beta}-sheets, loops and -helices that establish polar interactions with vWA. SAP05 binding to ZnF TFs involves single amino acids responsible for multiple contacts, while SAP05 binding to vWA is more stable due to the necessity of multiple mutations to break the interaction. In addition, positioning of the SAP05 complex on the 26S proteasome points to a mechanism of protein degradation. Collectively, our findings demonstrate how a small bacterial bimodular protein can bypass the canonical UPS cellular proteolysis pathway, enabling ubiquitin-independent TPD in eukaryotic cells. This knowledge holds significant potential for the creation of novel TPD technologies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.24.546416">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.24.546416" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.24.546416">
        <p class="paperTitle">Fluorogenesis: Inducing Fluorescence in a Non-Fluorescent Protein Through Photoinduced Chromophore Transfer of a Genetically Encoded Chromophore</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.24.546416" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.24.546416" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kumar, Y.; Singh, R. K.; Ojha, M.; Pushpavanam, K.</p>
        <p class="info">Score: 9.7, Published: 2023-06-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.24.546416' target='https://doi.org/10.1101/2023.06.24.546416'> 10.1101/2023.06.24.546416</a></p>
        <p class="abstract">Fluorescent proteins, while essential for bioimaging, are limited to visualizing cellular localization without offering additional functionality. We report for the first time a strategy to expand the chemical, structural, and functional diversity of fluorescent proteins by harnessing light to induce red fluorescence in a previously non-fluorescent protein. We accomplish this by inducing the transfer of the genetically encoded chromophore from a photocleavable protein (PhoCl1) to a non-fluorescent kinase (MjRibK) inducing red fluorescence in the latter. We have employed analytical and spectroscopic techniques to validate the presence of red fluorescence in MjRibK. Furthermore, molecular dynamics simulations were carried out to investigate the amino acid residues of MjRibK involved in the generation of red fluorescence. Finally, we demonstrate the ability of the red fluorescent MjRibK to operate as a cyclable high-temperature sensor. We anticipate that this light-induced chromophore transfer strategy will open new possibilities for developing multifunctional genetically encoded fluorescent sensors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.10.544439">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.10.544439" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.10.544439">
        <p class="paperTitle">Mag-Net: Rapid enrichment of membrane-bound particles enables high coverage quantitative analysis of the plasma proteome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.10.544439" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.10.544439" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wu, C. C.; Tsantilas, K. A.; Park, J.; Plubell, D. L.; Naicker, P.; Govender, I.; Buthelezi, S.; Stoychev, S.; Jordaan, J.; Merrihew, G. E.; Huang, E.; Parker, E. D.; Riffle, M.; Hoofnagle, A. N.; MacCoss, M. J.</p>
        <p class="info">Score: 28.3, Published: 2023-06-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.10.544439' target='https://doi.org/10.1101/2023.06.10.544439'> 10.1101/2023.06.10.544439</a></p>
        <p class="abstract">Membrane-bound particles in plasma are composed of exosomes, microvesicles, and apoptotic bodies and represent [~]1-2% of the total protein composition. Proteomic interrogation of this subset of plasma proteins augments the representation of tissue-specific proteins, representing a &#34;liquid biopsy,&#34; while enabling the detection of proteins that would otherwise be beyond the dynamic range of liquid chromatography-tandem mass spectrometry in unfractionated plasma. We have developed a one-step enrichment strategy (Mag-Net) using hyper-porous strong-anion exchange magnetic microparticles to sieve membrane-bound particles from plasma. The Mag-Net method is robust, reproducible, inexpensive, and requires &lt;100 L plasma input. Coupled to a quantitative data-independent mass spectrometry analytical strategy, we demonstrate that we can routinely collect results for &gt;37,000 peptides from &gt;4,000 plasma proteins with high precision. We demonstrate excellent quantitative accuracy and analytical reproducibility of the protocol.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545906">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545906" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545906">
        <p class="paperTitle">Targeting the conserved active site of splicing machines with specific and selective small molecule modulators</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545906" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545906" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Silvestri, I.; Manigrasso, J.; Andreani, A.; Brindani, N.; De Vivo, M.; Marcia, M.</p>
        <p class="info">Score: 9.2, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545906' target='https://doi.org/10.1101/2023.06.21.545906'> 10.1101/2023.06.21.545906</a></p>
        <p class="abstract">The self-splicing group II introns are bacterial and organellar ancestors of the nuclear spliceosome and retro-transposable elements of pharmacological and biotechnological importance. Integrating enzymatic, crystallographic, and simulation studies, we demonstrate how these introns recognize small molecules through their conserved active site. These RNA-binding small molecules selectively inhibit the two steps of splicing by adopting distinctive poses at different stages of catalysis, and by preventing crucial active site conformational changes that are essential for splicing progression. Our data exemplify the enormous power of RNA binders to mechanistically probe vital cellular pathways. Most importantly, by proving that the evolutionarily-conserved RNA core of splicing machines can recognize small molecules specifically, our work puts solid bases for the rational design of splicing modulators not only against bacterial and organellar introns, but also against the human spliceosome, which is a validated drug target for the treatment of congenital diseases and cancers.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.546027">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.546027" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.546027">
        <p class="paperTitle">Omics Scale Quantitative Mass Spectrometry Imaging of Lipids in Brain Tissue using a Multi-Class Internal Standard Mixture</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.546027" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.546027" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vandenbosch, M.; Mutuku, S. M.; Mantas, M. J. Q.; Patterson, N. H.; Hallmark, T.; Claesen, M.; Heeren, R. M. A.; Hatcher, N. G.; Verbeeck, N.; Ekroos, K.; Ellis, S. R.</p>
        <p class="info">Score: 7.8, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.546027' target='https://doi.org/10.1101/2023.06.21.546027'> 10.1101/2023.06.21.546027</a></p>
        <p class="abstract">Mass spectrometry imaging (MSI) has accelerated the understanding of lipid metabolism and spatial distribution in tissues and cells. However, few MSI studies have approached lipid imaging quantitatively and those that have focus on a single lipid class. Herein, we overcome limitation of quantitative MSI (Q-MSI) by using a multi-class internal standard lipid mixture that is sprayed homogenously over the tissue surface with analytical concentrations that reflects endogenous brain lipid levels. Using this approach we have performed Q-MSI for 13 lipid classes representing &gt;200 sum-composition lipid species. This was carried out using both MALDI (negative ion mode) and MALDI-2 (positive ion mode) and pixel-wise normalisation of each lipid species signal to the corresponding class-specific IS an approach analogous to that widely used for shotgun lipidomics from biological extracts. This approach allows pixel concentrations of lipids to be reported in pmol/mm2. Q-MSI of lipids covered 3 orders of magnitude in dynamic range and revealed subtle change sin in distribution compared to conventional total-ion-current normalisation approaches. The robustness of the method was evaluated by repeating experiments in two laboratories on biological replicates using both timsTOF and Orbitrap mass spectrometers operated with a ~4-fold difference in mass resolution power. There was a strong overall correlation in the Q-MSI result obtained using the two approaches with outliers mostly rationalised by isobaric interferences that are only resolved with the Orbitrap system or the higher sensitivity of one instrument for particular lipid species, particularly for lipids detected at low intensity. These data provide insight into how mass resolving power can affect Q-MSI data. This approach opens up the possibility of performing large-scale Q-MSI studies across numerous lipid classes and reveal how absolute lipid concentrations vary throughout and between biological tissues.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546230">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546230" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546230">
        <p class="paperTitle">Type III-B CRISPR-Cas signaling-based cascade of proteolytic cleavages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546230" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546230" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Steens, J.; Bravo, J. P.; Salazar, C. R. P.; Yildiz, C.; Amieiro, A. M.; Kostlbacher, S.; Prinsen, S. H. P.; Patinios, C.; Bardis, A.; Barendrecht, A.; Scheltema, R. A.; Ettema, T. J. G.; van der Oost, J.; Taylor, D. W.; Staals, R.</p>
        <p class="info">Score: 6.7, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546230' target='https://doi.org/10.1101/2023.06.23.546230'> 10.1101/2023.06.23.546230</a></p>
        <p class="abstract">Type III CRISPR-Cas systems provide a sequence-specific adaptive immune response that protects prokaryotic hosts against viruses and other foreign genetic invaders. These crRNA-guided Cas effector complexes bind and cleave complementary RNA targets. Specific target binding stimulates the Cas10 subunit to generate cyclic oligoadenylate (cOA) signaling molecules, that in turn allosterically activate proteins carrying cognate sensory domains: CARF or SAVED. Here, we characterize an elaborate set of genes associated with the type III-B CRISPR-Cas system from Haliangium ochraceum, which includes a signal transduction module of a CBASS defense system with two caspase-like proteases, SAVED-CHAT and PCaspase (Prokaryotic Caspase). We show that binding of a 3-nucleotide cOA (cA3) to the SAVED domain of SAVED-CHAT induces its oligomerization into long filaments that activate the proteolytic activity of the CHAT domain. Surprisingly, we find that activated SAVED-CHAT specifically cleaves and activates the second protease, PCaspase. In turn, activated PCaspase cleaves a multitude of other proteins, including a putative sigma factor and a PCaspase-inhibitor. We expressed the type III-B system and its associated genes in E. coli and observed a strong abortive phenotype when offering a complementary target RNA, but only in the presence of both SAVED-CHAT and PCaspase. Together, our findings show an intriguing cascade of proteolytic activities (conceptually similar to eukaryotic caspases) in this bacterial immune system that reveals yet another strategy to effectively defend against mobile genetic elements.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
