<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Leveraging co-evolutionary insights and AI-based structural modeling to unravel receptor-peptide ligand-binding mechanisms
Authors: Snoeck, S.; Lee, H. K.; Schmid, M. W.; Bender, K. W.; Neeracher, M. J.; Fernandez-Fernandez, A. D.; Santiago, J.; Zipfel, C.
Score: 31.0, Published: 2024-01-18 DOI: 10.1101/2024.01.18.575556
Secreted signaling peptides are central regulators of growth, development, and stress responses, but specific steps in the evolution of these peptides and their receptors are not well understood. In addition, the molecular mechanisms of peptide-receptor binding are only known for a few examples, primarily owing to the limited availability of structural capabilities to few laboratories worldwide.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Leveraging co-evolutionary insights and AI-based structural modeling to unravel receptor-peptide ligand-binding mechanisms
Authors: Snoeck, S.; Lee, H. K.; Schmid, M. W.; Bender, K. W.; Neeracher, M. J.; Fernandez-Fernandez, A. D.; Santiago, J.; Zipfel, C.
Score: 31.0, Published: 2024-01-18 DOI: 10.1101/2024.01.18.575556
Secreted signaling peptides are central regulators of growth, development, and stress responses, but specific steps in the evolution of these peptides and their receptors are not well understood. In addition, the molecular mechanisms of peptide-receptor binding are only known for a few examples, primarily owing to the limited availability of structural capabilities to few laboratories worldwide." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-24T10:40:04+00:00" />
<meta property="article:modified_time" content="2024-01-24T10:40:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Leveraging co-evolutionary insights and AI-based structural modeling to unravel receptor-peptide ligand-binding mechanisms
Authors: Snoeck, S.; Lee, H. K.; Schmid, M. W.; Bender, K. W.; Neeracher, M. J.; Fernandez-Fernandez, A. D.; Santiago, J.; Zipfel, C.
Score: 31.0, Published: 2024-01-18 DOI: 10.1101/2024.01.18.575556
Secreted signaling peptides are central regulators of growth, development, and stress responses, but specific steps in the evolution of these peptides and their receptors are not well understood. In addition, the molecular mechanisms of peptide-receptor binding are only known for a few examples, primarily owing to the limited availability of structural capabilities to few laboratories worldwide."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Leveraging co-evolutionary insights and AI-based structural modeling to unravel receptor-peptide ligand-binding mechanisms\nAuthors: Snoeck, S.; Lee, H. K.; Schmid, M. W.; Bender, K. W.; Neeracher, M. J.; Fernandez-Fernandez, A. D.; Santiago, J.; Zipfel, C.\nScore: 31.0, Published: 2024-01-18 DOI: 10.1101/2024.01.18.575556\nSecreted signaling peptides are central regulators of growth, development, and stress responses, but specific steps in the evolution of these peptides and their receptors are not well understood. In addition, the molecular mechanisms of peptide-receptor binding are only known for a few examples, primarily owing to the limited availability of structural capabilities to few laboratories worldwide.",
  "keywords": [
    
  ],
  "articleBody": " Leveraging co-evolutionary insights and AI-based structural modeling to unravel receptor-peptide ligand-binding mechanisms\nAuthors: Snoeck, S.; Lee, H. K.; Schmid, M. W.; Bender, K. W.; Neeracher, M. J.; Fernandez-Fernandez, A. D.; Santiago, J.; Zipfel, C.\nScore: 31.0, Published: 2024-01-18 DOI: 10.1101/2024.01.18.575556\nSecreted signaling peptides are central regulators of growth, development, and stress responses, but specific steps in the evolution of these peptides and their receptors are not well understood. In addition, the molecular mechanisms of peptide-receptor binding are only known for a few examples, primarily owing to the limited availability of structural capabilities to few laboratories worldwide. Plants have evolved a multitude of secreted signaling peptides and corresponding transmembrane receptors. Stress-responsive SERINE RICH ENDOGENOUS PEPTIDES (SCOOPs) were recently identified. Bioactive SCOOPs are proteolytically processed by subtilases and are perceived by the leucine-rich repeat receptor kinase MALE DISCOVERER 1-INTERACTING RECEPTOR-LIKE KINASE 2 (MIK2) in the model plant Arabidopsis thaliana. How SCOOPs and MIK2 have (co-)evolved, and how SCOOPs bind to MIK2 are however still unknown. Using in silico analysis of 350 plant genomes and subsequent functional testing, we revealed the conservation of MIK2 as SCOOP receptor within the plant order Brassicales. We then leveraged AlphaFold-Multimer and comparative genomics to identify two conserved putative SCOOP-MIK2 binding pockets across Brassicales MIK2 homologues predicted to interact with the SxS motif of otherwise sequence-divergent SCOOPs. Notably, mutagenesis of both predicted binding pockets compromised SCOOP binding to MIK2, SCOOP-induced complex formation between MIK2 and its co-receptor BRASSINOSTEROID INSENSITIVE 1-ASSOCIATED KINASE 1 (BAK1), and SCOOP-induced reactive oxygen species production; thus, confirming our in silico predictions. Collectively, in addition to revealing the elusive SCOOP-MIK2 binding mechanisms, our analytic pipeline combining phylogenomics, AI-based structural predictions, and experimental biochemical and physiological validation provides a blueprint for the elucidation of peptide ligand-receptor perception mechanisms. Significance statementThis study presents a rapid and inexpensive alternative to classical structure-based approaches for resolving ligand-receptor binding mechanisms. It relies on a multilayered bioinformatic approach that leverages genomic data across diverse species in combination with AI-based structural modeling to identify true ligand and receptor homologues, and subsequently predict their binding mechanisms. In silico findings were validated by multiple experimental approaches, which investigated the effect of amino acid changes in the proposed binding pockets on ligand-binding, complex formation with a co-receptor essential for downstream signaling, and activation of downstream signaling. Our analysis combining evolutionary insights, in silico modeling and functional validation provides a framework for structure-function analysis of other peptide-receptor pairs, which could be easily implemented by most laboratories.\nExpanding the ligandable proteome by paralog hopping with covalent probes\nAuthors: Zhang, Y.; Liu, Z.; Hirschi, M.; Brodsky, O.; Johnson, E.; Won, S. J.; Nagata, A.; Petroski, M. D.; Majmudar, J. D.; Niessen, S.; VanArsdale, T.; Gilbert, A. M.; Hayward, M. M.; Stewart, A. E.; Nager, A. R.; Melillo, B.; Cravatt, B. F.\nScore: 29.5, Published: 2024-01-20 DOI: 10.1101/2024.01.18.576274\nMore than half of the [~]20,000 protein-encoding human genes have at least one paralog. Chemical proteomics has uncovered many electrophile-sensitive cysteines that are exclusive to a subset of paralogous proteins. Here, we explore whether such covalent compound-cysteine interactions can be used to discover ligandable pockets in paralogs that lack the cysteine. Leveraging the covalent ligandability of C109 in the cyclin CCNE2, we mutated the corresponding residue in paralog CCNE1 to cysteine (N112C) and found through activity-based protein profiling (ABPP) that this mutant reacts stereoselectively and site-specifically with tryptoline acrylamides. We then converted the tryptoline acrylamide-N112C-CCNE1 interaction into a NanoBRET-ABPP assay capable of identifying compounds that reversibly inhibit both N112C- and WT-CCNE1:CDK2 complexes. X-ray crystallography revealed a cryptic allosteric pocket at the CCNE1:CDK2 interface adjacent to N112 that binds the reversible inhibitors. Our findings thus provide a roadmap for leveraging electrophile-cysteine interactions to extend the ligandability of the proteome beyond covalent chemistry.\nDesign of a Cereblon construct for crystallographic and biophysical studies of protein degraders\nAuthors: Kroupova, A.; Spiteri, V. A.; Furihata, H.; Darren, D.; Ramachandran, S.; Rutter, Z.; Chakraborti, S.; Haubrich, K.; Pethe, J.; Gonzales, D.; Wijaya, A.; Rodriguez-Rios, M.; Lynch, D. M.; Farnaby, W.; Nakasone, M.; Zollman, D.; Ciulli, A.\nScore: 23.3, Published: 2024-01-20 DOI: 10.1101/2024.01.17.575503\nThe ubiquitin E3 ligase cereblon (CRBN) is the target of therapeutic drugs thalidomide and lenalidomide and is recruited by most targeted protein degraders (PROTACs and molecular glues) in clinical development. Biophysical and structural investigation of CRBN has been limited by current constructs that either require co-expression with the adaptor DDB1 or inadequately represent full-length protein, with high-resolution structures of degraders ternary complexes remaining rare. We present the design of CRBNmidi, a construct that readily expresses from E. coli with high yields as soluble, stable protein without DDB1. We benchmark CRBNmidi for wild-type functionality through a suite of biophysical techniques and solve high-resolution co-crystal structures of its binary and ternary complexes with degraders. We qualify CRBNmidi as an enabling tool to accelerate structure-based discovery of the next generation of CRBN based therapeutics. One sentence summaryA novel Cereblon construct (CRBNmidi) allows structural and biophysical enablement of ligand and degrader design\nByeTAC: Bypassing an E3 Ligase for Targeted Protein Degradation\nAuthors: Ali, E. M.; Loy, C.; Trader, D. J.\nScore: 22.5, Published: 2024-01-21 DOI: 10.1101/2024.01.20.576376\nTargeted protein degradation utilizing a bifunctional molecule to initiate ubiquitination and subsequent degradation by the 26S proteasome has been shown to be a powerful therapeutic intervention. Many bifunctional molecules, including covalent and non-covalent ligands to proteins of interest, have been developed. The traditional target protein degradation methodology targets the protein of interest in both healthy and diseased cell populations, and a therapeutic window is obtained based on the overexpression of the targeted protein. We report here a series of bifunctional degraders that do not rely on interacting with an E3 ligase, but rather a 26S proteasome subunit, which we have named ByeTACs: Bypassing E3 Targeting Chimeras. Rpn-13 is a non-essential ubiquitin receptor for the 26S proteasome. Cells under significant stress or require significant ubiquitin-dependent degradation of proteins for survival, incorporate Rpn-13 in the 26S to increase protein degradation rates. The targeted protein degraders reported here are bifunctional molecules that include a ligand to Rpn-13 and BRD4, the protein of interest we wish to degrade. We synthesized a suite of degraders with varying PEG chain lengths and showed that bifunctional molecules that incorporate a Rpn-13 binder (TCL1) and a BRD4 binder (JQ1) with a PEG linker of 3 or 4 units are the most effective to induce BRD4 degradation. We also demonstrate that our new targeted protein degraders are dependent upon proteasome activity and Rpn-13 expression levels. This establishes a new mechanism of action for our ByeTACs that can be employed for the targeted degradation of a wide variety of protein substrates.\nMulti-state kinetics of the syringe-like injection mechanism of Tc toxins\nAuthors: Ng`ang`a, P. N.; Folz, J.; Kucher, S.; Roderer, D.; Xu, Y.; Sitsel, O.; Belyy, A.; Prumbaum, D.; Kuehnemuth, R.; Assafa, T. E.; Dong, M.; Seidel, C. A. M.; Bordignon, E.; Raunser, S.\nScore: 22.1, Published: 2024-01-16 DOI: 10.1101/2024.01.16.575634\nTc toxins are virulence factors of many insects and human pathogenic bacteria. They attach as soluble prepores to receptors on host cells and following acidification in the late endosome, perforate the cell membrane like a syringe to translocate toxic enzymes into the host cell through their pore-forming channel. Although this complex transformation has been structurally well studied, the functional aspects of this large-scale rearrangement, such as the reaction pathway with possible intermediate states and the resulting temporal evolution have remained elusive. Here, we used an integrated biophysical approach to monitor the prepore-to-pore transition and found that it takes [~]28 h when induced by high pH in the absence of other factors. In the presence of liposomes, an increasingly high pH or receptors, such as heparin or Vsg, the probability to transform prepores to pores increases by a factor of up to 4. This effect can also be mimicked by biotinylation or site-directed mutagenesis of the shell, demonstrating that shell destabilization is a crucial step in prepore-to-pore transition. We show that shell opening is a heterogeneous process with transition times ranging from 60 ms to 1.6 s and resolve three sequential intermediate states: an initial transient intermediate during shell destabilization, a first stable intermediate where the receptor-binding domains on the shell rearrange and a second stable intermediate with an open shell. In contrast, the ejection of the pore-forming channel from the open shell is highly cooperative with a transition time of \u003c 60 ms. This detailed knowledge of the Tc toxin mechanism of action, even in the absence of receptors, is important for the future application of Tc toxins as biomedical devices or biopesticides.\nCell Cycle Regulation has Shaped Budding Yeast Replication Origin Structure and Function\nAuthors: Diffley, J.; Lim, C. T.; Miller, T. C.; Tan, K. W.; Talele, s.; Early, A.; East, P.; Sanchez, H.; Dekker, N. H.; Costa, A.\nScore: 15.5, Published: 2024-01-10 DOI: 10.1101/2024.01.10.575016\nEukaryotic DNA replication initiates from multiple genomic loci known as origins. At budding yeast origins like ARS1, a double hexamer (DH) of the MCM replicative helicase is assembled by Origin Recognition Complex (ORC), Cdc6 and Cdt1 via sequential hexamer loading from two opposed ORC binding sites. Cyclin Dependent Kinase (CDK) inhibits DH assembly, which prevents re-replication by restricting helicase loading to G1 phase. Here we show that an intrinsically disordered region (IDR) in the Orc2 subunit promotes interaction between ORC and the first loaded, closed-ring MCM hexamer (the MO intermediate); CDK phosphorylation of this IDR blocks MO formation and DH assembly. We show that MO functions by stabilising ORC at the lower affinity binding sites required for second hexamer loading. Origins comprising two high affinity ORC sites can assemble DH efficiently without MO by independently loading single hexamers; these origins escape CDK inhibition in vitro and in vivo. Our work reveals mechanistic plasticity in MCM loading with implications for understanding how CDK regulation has shaped yeast origin evolution and how natural origins might escape cell cycle regulation. We also identify key steps common to loading pathways, with implications for understanding how MCM is loaded in other eukaryotes.\nAutomated single-cell proteomics providing sufficient proteome depth to study complex biology beyond cell type classifications\nAuthors: Ctortecka, C.; Clark, N. M.; Boyle, B.; Seth, A.; Mani, D. R.; Udeshi, N.; Carr, S. A.\nScore: 12.2, Published: 2024-01-22 DOI: 10.1101/2024.01.20.576369\nMass spectrometry (MS)-based single-cell proteomics (SCP) has gained massive attention as a viable complement to other single cell approaches. The rapid technological and computational advances in the field have pushed the boundaries of sensitivity and throughput. However, reproducible quantification of thousands of proteins within a single cell at reasonable proteome depth to characterize biological phenomena remains a challenge. To address some of those limitations we present a combination of fully automated single cell sample preparation utilizing a dedicated chip within the picolitre dispensing robot, the cellenONE. The proteoCHIP EVO 96 can be directly interfaced with the Evosep One chromatographic system for in-line desalting and highly reproducible separation with a throughput of 80 samples per day. This, in combination with the Bruker timsTOF MS instruments, demonstrates double the identifications without manual sample handling. Moreover, relative to standard high-performance liquid chromatography, the Evosep One separation provides further 2-fold improvement in protein identifications. The implementation of the newest generation timsTOF Ultra with our proteoCHIP EVO 96-based sample preparation workflow reproducibly identifies up to 4,000 proteins per single HEK-293T without a carrier or match-between runs. Our current SCP depth spans over 4 orders of magnitude and identifies over 50 biologically relevant ubiquitin ligases. We complement our highly reproducible single-cell proteomics workflow to profile hundreds of lipopolysaccharide (LPS)-perturbed THP-1 cells and identified key regulatory proteins involved in interleukin and interferon signaling. This study demonstrates that the proteoCHIP EVO 96-based SCP sample preparation with the timsTOF Ultra provides sufficient proteome depth to study complex biology beyond cell-type classifications.\nDefining a small-molecule stimulator of the human Hsp70-disaggregase system with selectivity for DnaJB proteins\nAuthors: Chuang, E.; Cupo, R. R.; Bevan, J. R.; Rice, M. L.; Mao, S.; Gorenberg, E. L.; Mack, K. L.; Huryn, D. M.; Wipf, P.; Brodsky, J. L.; Shorter, J.\nScore: 9.2, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575109\nHsp70, Hsp40, and Hsp110 form a human protein-disaggregase system that solubilizes and reactivates proteins trapped in aggregated states. However, this system fails to maintain proteostasis in fatal neurodegenerative diseases. Here, we potentiate the human Hsp70-disaggregase system pharmacologically. By scouring a collection of dihydropyrimidines, we disambiguate a small molecule that specifically stimulates the Hsp70-disaggregase system against disordered aggregates and -synuclein fibrils. The newly identified lead compound stimulates the disaggregase activity of multiple active human Hsp70, Hsp40, Hsp110 chaperone sets, with selectivity for combinations that include DnaJB1 or DnaJB4 as the Hsp40. We find that the relative stoichiometry of Hsp70, Hsp40, and Hsp110 dictates disaggregase activity. Remarkably, our lead compound shifts the composition of active chaperone stoichiometries by preferentially activating combinations with lower DnaJB1 concentrations. Our findings unveil a small molecule that stimulates the Hsp70-disaggregase system, even at suboptimal chaperone stoichiometries, which could be developed for the treatment of neurodegenerative diseases. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=113 SRC=\"FIGDIR/small/575109v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (31K): org.highwire.dtl.DTLVardef@1f1b574org.highwire.dtl.DTLVardef@1bfd3aborg.highwire.dtl.DTLVardef@e191e3org.highwire.dtl.DTLVardef@130d19c_HPS_FORMAT_FIGEXP M_FIG C_FIG\nStructure of hepatitis B/D antiviral drug Bulevirtide bound to its receptor protein NTCP\nAuthors: Liu, H.; Zakrzewicz, D.; Nosol, K.; Irobalieva, R. N.; Mukherjee, S.; Bang-Soerensen, R.; Goldmann, N.; Kunz, S.; Rossi, L.; Kossiakoff, A. A.; Urban, S.; Glebe, D.; Geyer, J.; Locher, K. P.\nScore: 3.9, Published: 2024-01-22 DOI: 10.1101/2024.01.22.576510\nCellular entry of the hepatitis B and D viruses (HBV/HDV) require binding of the viral surface polypeptide preS1 to the hepatobiliary transporter NTCP. This interaction can be blocked by bulevirtide (BLV, formerly Myrcludex B), a preS1 derivative and approved drug for treating HDV infection. To elucidate the basis of this inhibitory function, we determined a cryo-EM structure of BLV-bound human NTCP. BLV forms two domains, a plug lodged in the bile salt transport tunnel of NTCP and a string that covers the receptors extracellular surface. The N-terminally attached myristoyl group of BLV interacts with the lipid-exposed surface of NTCP. Our structure reveals how BLV inhibits bile salt transport, rationalizes NTCP mutations that decrease the risk of HBV/HDV infection, and provides a basis for understanding the host specificity of HBV/HDV. Our results provide opportunities for structure-guided development of inhibitors that target HBV/HDV docking to NTCP.\nPhenotypic screening of covalent compound libraries identifies chloromethyl ketone antibiotics and MiaA as a new target\nAuthors: Jin, Y.; Jana, S.; Abbasov, M.; Lin, H.\nScore: 3.5, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576576\nThe emerging antibiotic resistance requires the development of new antibiotics working on novel bacterial targets. Here, we reported an antibiotic discovery workflow by combining the cysteine-reactive compound library phenotypic screening with activity-based protein profiling, which enables the rapid identification of lead compounds as well as new druggable targets in pathogens. Compounds featuring chloromethyl ketone scaffolds exhibited a notably high hit rate against both gram-negative and gram-positive bacterial strains, but not the more commonly used warheads such as acrylamide or chloroacetamide. Target identification of the lead compound, 10-F05, revealed that its primary targets in S. flexneri are FabH Cys112 and MiaA Cys273. We validated the target relevance through biochemical and genetic interactions. Mechanistic studies revealed modification of MiaA by 10-F05 impair substrate tRNA binding, leading to decreased bacterial stress resistance and virulence. Our findings underscore chloromethyl ketone as a novel antibacterial warhead in covalent antibiotic design. The study showcases that combining covalent compound library phenotypic screening with chemoproteomics is an efficient way to identify new drug targets as well as lead compounds, with the potential to open new research directions in drug discovery and chemical biology.\n",
  "wordCount" : "2578",
  "inLanguage": "en",
  "datePublished": "2024-01-24T10:40:04Z",
  "dateModified": "2024-01-24T10:40:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 24, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.575556">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.575556" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.575556">
        <p class="paperTitle">Leveraging co-evolutionary insights and AI-based structural modeling to unravel receptor-peptide ligand-binding mechanisms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.575556" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.575556" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Snoeck, S.; Lee, H. K.; Schmid, M. W.; Bender, K. W.; Neeracher, M. J.; Fernandez-Fernandez, A. D.; Santiago, J.; Zipfel, C.</p>
        <p class="info">Score: 31.0, Published: 2024-01-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.575556' target='https://doi.org/10.1101/2024.01.18.575556'> 10.1101/2024.01.18.575556</a></p>
        <p class="abstract">Secreted signaling peptides are central regulators of growth, development, and stress responses, but specific steps in the evolution of these peptides and their receptors are not well understood. In addition, the molecular mechanisms of peptide-receptor binding are only known for a few examples, primarily owing to the limited availability of structural capabilities to few laboratories worldwide. Plants have evolved a multitude of secreted signaling peptides and corresponding transmembrane receptors. Stress-responsive SERINE RICH ENDOGENOUS PEPTIDES (SCOOPs) were recently identified. Bioactive SCOOPs are proteolytically processed by subtilases and are perceived by the leucine-rich repeat receptor kinase MALE DISCOVERER 1-INTERACTING RECEPTOR-LIKE KINASE 2 (MIK2) in the model plant Arabidopsis thaliana. How SCOOPs and MIK2 have (co-)evolved, and how SCOOPs bind to MIK2 are however still unknown. Using in silico analysis of 350 plant genomes and subsequent functional testing, we revealed the conservation of MIK2 as SCOOP receptor within the plant order Brassicales. We then leveraged AlphaFold-Multimer and comparative genomics to identify two conserved putative SCOOP-MIK2 binding pockets across Brassicales MIK2 homologues predicted to interact with the  SxS motif of otherwise sequence-divergent SCOOPs. Notably, mutagenesis of both predicted binding pockets compromised SCOOP binding to MIK2, SCOOP-induced complex formation between MIK2 and its co-receptor BRASSINOSTEROID INSENSITIVE 1-ASSOCIATED KINASE 1 (BAK1), and SCOOP-induced reactive oxygen species production; thus, confirming our in silico predictions. Collectively, in addition to revealing the elusive SCOOP-MIK2 binding mechanisms, our analytic pipeline combining phylogenomics, AI-based structural predictions, and experimental biochemical and physiological validation provides a blueprint for the elucidation of peptide ligand-receptor perception mechanisms.

Significance statementThis study presents a rapid and inexpensive alternative to classical structure-based approaches for resolving ligand-receptor binding mechanisms. It relies on a multilayered bioinformatic approach that leverages genomic data across diverse species in combination with AI-based structural modeling to identify true ligand and receptor homologues, and subsequently predict their binding mechanisms. In silico findings were validated by multiple experimental approaches, which investigated the effect of amino acid changes in the proposed binding pockets on ligand-binding, complex formation with a co-receptor essential for downstream signaling, and activation of downstream signaling. Our analysis combining evolutionary insights, in silico modeling and functional validation provides a framework for structure-function analysis of other peptide-receptor pairs, which could be easily implemented by most laboratories.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576274">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576274" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576274">
        <p class="paperTitle">Expanding the ligandable proteome by paralog hopping with covalent probes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576274" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576274" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, Y.; Liu, Z.; Hirschi, M.; Brodsky, O.; Johnson, E.; Won, S. J.; Nagata, A.; Petroski, M. D.; Majmudar, J. D.; Niessen, S.; VanArsdale, T.; Gilbert, A. M.; Hayward, M. M.; Stewart, A. E.; Nager, A. R.; Melillo, B.; Cravatt, B. F.</p>
        <p class="info">Score: 29.5, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576274' target='https://doi.org/10.1101/2024.01.18.576274'> 10.1101/2024.01.18.576274</a></p>
        <p class="abstract">More than half of the [~]20,000 protein-encoding human genes have at least one paralog. Chemical proteomics has uncovered many electrophile-sensitive cysteines that are exclusive to a subset of paralogous proteins. Here, we explore whether such covalent compound-cysteine interactions can be used to discover ligandable pockets in paralogs that lack the cysteine. Leveraging the covalent ligandability of C109 in the cyclin CCNE2, we mutated the corresponding residue in paralog CCNE1 to cysteine (N112C) and found through activity-based protein profiling (ABPP) that this mutant reacts stereoselectively and site-specifically with tryptoline acrylamides. We then converted the tryptoline acrylamide-N112C-CCNE1 interaction into a NanoBRET-ABPP assay capable of identifying compounds that reversibly inhibit both N112C- and WT-CCNE1:CDK2 complexes. X-ray crystallography revealed a cryptic allosteric pocket at the CCNE1:CDK2 interface adjacent to N112 that binds the reversible inhibitors. Our findings thus provide a roadmap for leveraging electrophile-cysteine interactions to extend the ligandability of the proteome beyond covalent chemistry.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.575503">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.575503" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.575503">
        <p class="paperTitle">Design of a Cereblon construct for crystallographic and biophysical studies of protein degraders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.575503" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.575503" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kroupova, A.; Spiteri, V. A.; Furihata, H.; Darren, D.; Ramachandran, S.; Rutter, Z.; Chakraborti, S.; Haubrich, K.; Pethe, J.; Gonzales, D.; Wijaya, A.; Rodriguez-Rios, M.; Lynch, D. M.; Farnaby, W.; Nakasone, M.; Zollman, D.; Ciulli, A.</p>
        <p class="info">Score: 23.3, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.575503' target='https://doi.org/10.1101/2024.01.17.575503'> 10.1101/2024.01.17.575503</a></p>
        <p class="abstract">The ubiquitin E3 ligase cereblon (CRBN) is the target of therapeutic drugs thalidomide and lenalidomide and is recruited by most targeted protein degraders (PROTACs and molecular glues) in clinical development. Biophysical and structural investigation of CRBN has been limited by current constructs that either require co-expression with the adaptor DDB1 or inadequately represent full-length protein, with high-resolution structures of degraders ternary complexes remaining rare. We present the design of CRBNmidi, a construct that readily expresses from E. coli with high yields as soluble, stable protein without DDB1. We benchmark CRBNmidi for wild-type functionality through a suite of biophysical techniques and solve high-resolution co-crystal structures of its binary and ternary complexes with degraders. We qualify CRBNmidi as an enabling tool to accelerate structure-based discovery of the next generation of CRBN based therapeutics.

One sentence summaryA novel Cereblon construct (CRBNmidi) allows structural and biophysical enablement of ligand and degrader design</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576376">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576376" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576376">
        <p class="paperTitle">ByeTAC: Bypassing an E3 Ligase for Targeted Protein Degradation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576376" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576376" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ali, E. M.; Loy, C.; Trader, D. J.</p>
        <p class="info">Score: 22.5, Published: 2024-01-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576376' target='https://doi.org/10.1101/2024.01.20.576376'> 10.1101/2024.01.20.576376</a></p>
        <p class="abstract">Targeted protein degradation utilizing a bifunctional molecule to initiate ubiquitination and subsequent degradation by the 26S proteasome has been shown to be a powerful therapeutic intervention. Many bifunctional molecules, including covalent and non-covalent ligands to proteins of interest, have been developed. The traditional target protein degradation methodology targets the protein of interest in both healthy and diseased cell populations, and a therapeutic window is obtained based on the overexpression of the targeted protein. We report here a series of bifunctional degraders that do not rely on interacting with an E3 ligase, but rather a 26S proteasome subunit, which we have named ByeTACs: Bypassing E3 Targeting Chimeras. Rpn-13 is a non-essential ubiquitin receptor for the 26S proteasome. Cells under significant stress or require significant ubiquitin-dependent degradation of proteins for survival, incorporate Rpn-13 in the 26S to increase protein degradation rates. The targeted protein degraders reported here are bifunctional molecules that include a ligand to Rpn-13 and BRD4, the protein of interest we wish to degrade. We synthesized a suite of degraders with varying PEG chain lengths and showed that bifunctional molecules that incorporate a Rpn-13 binder (TCL1) and a BRD4 binder (JQ1) with a PEG linker of 3 or 4 units are the most effective to induce BRD4 degradation. We also demonstrate that our new targeted protein degraders are dependent upon proteasome activity and Rpn-13 expression levels. This establishes a new mechanism of action for our ByeTACs that can be employed for the targeted degradation of a wide variety of protein substrates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.16.575634">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.16.575634" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.16.575634">
        <p class="paperTitle">Multi-state kinetics of the syringe-like injection mechanism of Tc toxins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.16.575634" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.16.575634" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ng`ang`a, P. N.; Folz, J.; Kucher, S.; Roderer, D.; Xu, Y.; Sitsel, O.; Belyy, A.; Prumbaum, D.; Kuehnemuth, R.; Assafa, T. E.; Dong, M.; Seidel, C. A. M.; Bordignon, E.; Raunser, S.</p>
        <p class="info">Score: 22.1, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.16.575634' target='https://doi.org/10.1101/2024.01.16.575634'> 10.1101/2024.01.16.575634</a></p>
        <p class="abstract">Tc toxins are virulence factors of many insects and human pathogenic bacteria. They attach as soluble prepores to receptors on host cells and following acidification in the late endosome, perforate the cell membrane like a syringe to translocate toxic enzymes into the host cell through their pore-forming channel. Although this complex transformation has been structurally well studied, the functional aspects of this large-scale rearrangement, such as the reaction pathway with possible intermediate states and the resulting temporal evolution have remained elusive. Here, we used an integrated biophysical approach to monitor the prepore-to-pore transition and found that it takes [~]28 h when induced by high pH in the absence of other factors. In the presence of liposomes, an increasingly high pH or receptors, such as heparin or Vsg, the probability to transform prepores to pores increases by a factor of up to 4. This effect can also be mimicked by biotinylation or site-directed mutagenesis of the shell, demonstrating that shell destabilization is a crucial step in prepore-to-pore transition. We show that shell opening is a heterogeneous process with transition times ranging from 60 ms to 1.6 s and resolve three sequential intermediate states: an initial transient intermediate during shell destabilization, a first stable intermediate where the receptor-binding domains on the shell rearrange and a second stable intermediate with an open shell. In contrast, the ejection of the pore-forming channel from the open shell is highly cooperative with a transition time of &lt; 60 ms. This detailed knowledge of the Tc toxin mechanism of action, even in the absence of receptors, is important for the future application of Tc toxins as biomedical devices or biopesticides.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.575016">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.575016" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.575016">
        <p class="paperTitle">Cell Cycle Regulation has Shaped Budding Yeast Replication Origin Structure and Function</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.575016" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.575016" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Diffley, J.; Lim, C. T.; Miller, T. C.; Tan, K. W.; Talele, s.; Early, A.; East, P.; Sanchez, H.; Dekker, N. H.; Costa, A.</p>
        <p class="info">Score: 15.5, Published: 2024-01-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.575016' target='https://doi.org/10.1101/2024.01.10.575016'> 10.1101/2024.01.10.575016</a></p>
        <p class="abstract">Eukaryotic DNA replication initiates from multiple genomic loci known as origins. At budding yeast origins like ARS1, a double hexamer (DH) of the MCM replicative helicase is assembled by Origin Recognition Complex (ORC), Cdc6 and Cdt1 via sequential hexamer loading from two opposed ORC binding sites. Cyclin Dependent Kinase (CDK) inhibits DH assembly, which prevents re-replication by restricting helicase loading to G1 phase. Here we show that an intrinsically disordered region (IDR) in the Orc2 subunit promotes interaction between ORC and the first loaded, closed-ring MCM hexamer (the MO intermediate); CDK phosphorylation of this IDR blocks MO formation and DH assembly. We show that MO functions by stabilising ORC at the lower affinity binding sites required for second hexamer loading. Origins comprising two high affinity ORC sites can assemble DH efficiently without MO by independently loading single hexamers; these origins escape CDK inhibition in vitro and in vivo. Our work reveals mechanistic plasticity in MCM loading with implications for understanding how CDK regulation has shaped yeast origin evolution and how natural origins might escape cell cycle regulation. We also identify key steps common to loading pathways, with implications for understanding how MCM is loaded in other eukaryotes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576369">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576369" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576369">
        <p class="paperTitle">Automated single-cell proteomics providing sufficient proteome depth to study complex biology beyond cell type classifications</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576369" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576369" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ctortecka, C.; Clark, N. M.; Boyle, B.; Seth, A.; Mani, D. R.; Udeshi, N.; Carr, S. A.</p>
        <p class="info">Score: 12.2, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576369' target='https://doi.org/10.1101/2024.01.20.576369'> 10.1101/2024.01.20.576369</a></p>
        <p class="abstract">Mass spectrometry (MS)-based single-cell proteomics (SCP) has gained massive attention as a viable complement to other single cell approaches. The rapid technological and computational advances in the field have pushed the boundaries of sensitivity and throughput. However, reproducible quantification of thousands of proteins within a single cell at reasonable proteome depth to characterize biological phenomena remains a challenge. To address some of those limitations we present a combination of fully automated single cell sample preparation utilizing a dedicated chip within the picolitre dispensing robot, the cellenONE. The proteoCHIP EVO 96 can be directly interfaced with the Evosep One chromatographic system for in-line desalting and highly reproducible separation with a throughput of 80 samples per day. This, in combination with the Bruker timsTOF MS instruments, demonstrates double the identifications without manual sample handling. Moreover, relative to standard high-performance liquid chromatography, the Evosep One separation provides further 2-fold improvement in protein identifications. The implementation of the newest generation timsTOF Ultra with our proteoCHIP EVO 96-based sample preparation workflow reproducibly identifies up to 4,000 proteins per single HEK-293T without a carrier or match-between runs. Our current SCP depth spans over 4 orders of magnitude and identifies over 50 biologically relevant ubiquitin ligases. We complement our highly reproducible single-cell proteomics workflow to profile hundreds of lipopolysaccharide (LPS)-perturbed THP-1 cells and identified key regulatory proteins involved in interleukin and interferon signaling. This study demonstrates that the proteoCHIP EVO 96-based SCP sample preparation with the timsTOF Ultra provides sufficient proteome depth to study complex biology beyond cell-type classifications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575109">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575109" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575109">
        <p class="paperTitle">Defining a small-molecule stimulator of the human Hsp70-disaggregase system with selectivity for DnaJB proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575109" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575109" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chuang, E.; Cupo, R. R.; Bevan, J. R.; Rice, M. L.; Mao, S.; Gorenberg, E. L.; Mack, K. L.; Huryn, D. M.; Wipf, P.; Brodsky, J. L.; Shorter, J.</p>
        <p class="info">Score: 9.2, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575109' target='https://doi.org/10.1101/2024.01.11.575109'> 10.1101/2024.01.11.575109</a></p>
        <p class="abstract">Hsp70, Hsp40, and Hsp110 form a human protein-disaggregase system that solubilizes and reactivates proteins trapped in aggregated states. However, this system fails to maintain proteostasis in fatal neurodegenerative diseases. Here, we potentiate the human Hsp70-disaggregase system pharmacologically. By scouring a collection of dihydropyrimidines, we disambiguate a small molecule that specifically stimulates the Hsp70-disaggregase system against disordered aggregates and -synuclein fibrils. The newly identified lead compound stimulates the disaggregase activity of multiple active human Hsp70, Hsp40, Hsp110 chaperone sets, with selectivity for combinations that include DnaJB1 or DnaJB4 as the Hsp40. We find that the relative stoichiometry of Hsp70, Hsp40, and Hsp110 dictates disaggregase activity. Remarkably, our lead compound shifts the composition of active chaperone stoichiometries by preferentially activating combinations with lower DnaJB1 concentrations. Our findings unveil a small molecule that stimulates the Hsp70-disaggregase system, even at suboptimal chaperone stoichiometries, which could be developed for the treatment of neurodegenerative diseases.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=113 SRC=&#34;FIGDIR/small/575109v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (31K):
org.highwire.dtl.DTLVardef@1f1b574org.highwire.dtl.DTLVardef@1bfd3aborg.highwire.dtl.DTLVardef@e191e3org.highwire.dtl.DTLVardef@130d19c_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.576510">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.576510" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.576510">
        <p class="paperTitle">Structure of hepatitis B/D antiviral drug Bulevirtide bound to its receptor protein NTCP</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.576510" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.576510" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, H.; Zakrzewicz, D.; Nosol, K.; Irobalieva, R. N.; Mukherjee, S.; Bang-Soerensen, R.; Goldmann, N.; Kunz, S.; Rossi, L.; Kossiakoff, A. A.; Urban, S.; Glebe, D.; Geyer, J.; Locher, K. P.</p>
        <p class="info">Score: 3.9, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.576510' target='https://doi.org/10.1101/2024.01.22.576510'> 10.1101/2024.01.22.576510</a></p>
        <p class="abstract">Cellular entry of the hepatitis B and D viruses (HBV/HDV) require binding of the viral surface polypeptide preS1 to the hepatobiliary transporter NTCP. This interaction can be blocked by bulevirtide (BLV, formerly Myrcludex B), a preS1 derivative and approved drug for treating HDV infection. To elucidate the basis of this inhibitory function, we determined a cryo-EM structure of BLV-bound human NTCP. BLV forms two domains, a plug lodged in the bile salt transport tunnel of NTCP and a string that covers the receptors extracellular surface. The N-terminally attached myristoyl group of BLV interacts with the lipid-exposed surface of NTCP. Our structure reveals how BLV inhibits bile salt transport, rationalizes NTCP mutations that decrease the risk of HBV/HDV infection, and provides a basis for understanding the host specificity of HBV/HDV. Our results provide opportunities for structure-guided development of inhibitors that target HBV/HDV docking to NTCP.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.576576">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.576576" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.576576">
        <p class="paperTitle">Phenotypic screening of covalent compound libraries identifies chloromethyl ketone antibiotics and MiaA as a new target</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.576576" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.576576" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jin, Y.; Jana, S.; Abbasov, M.; Lin, H.</p>
        <p class="info">Score: 3.5, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.576576' target='https://doi.org/10.1101/2024.01.22.576576'> 10.1101/2024.01.22.576576</a></p>
        <p class="abstract">The emerging antibiotic resistance requires the development of new antibiotics working on novel bacterial targets. Here, we reported an antibiotic discovery workflow by combining the cysteine-reactive compound library phenotypic screening with activity-based protein profiling, which enables the rapid identification of lead compounds as well as new druggable targets in pathogens. Compounds featuring chloromethyl ketone scaffolds exhibited a notably high hit rate against both gram-negative and gram-positive bacterial strains, but not the more commonly used warheads such as acrylamide or chloroacetamide. Target identification of the lead compound, 10-F05, revealed that its primary targets in S. flexneri are FabH Cys112 and MiaA Cys273. We validated the target relevance through biochemical and genetic interactions. Mechanistic studies revealed modification of MiaA by 10-F05 impair substrate tRNA binding, leading to decreased bacterial stress resistance and virulence. Our findings underscore chloromethyl ketone as a novel antibacterial warhead in covalent antibiotic design. The study showcases that combining covalent compound library phenotypic screening with chemoproteomics is an efficient way to identify new drug targets as well as lead compounds, with the potential to open new research directions in drug discovery and chemical biology.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
