<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Generalized Biomolecular Modeling and Design with RoseTTAFold All-Atom
Authors: Krishna, R.; Wang, J.; Ahern, W.; Sturmfels, P.; Venkatesh, P.; Kalvet, I.; Lee, G. R.; Morey-Burrows, F. S.; Anishchenko, I.; Humphreys, I. R.; McHugh, R.; Vafeados, D.; Li, X.; Sutherland, G. A.; Hitchcock, A.; Hunter, C. N.; Baek, M.; DiMaio, F.; Baker, D.
Score: 234.7, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561603
Although AlphaFold2 (AF2) and RoseTTAFold (RF) have transformed structural biology by enabling high-accuracy protein structure modeling, they are unable to model covalent modifications or interactions with small molecules and other non-protein molecules that can play key roles in biological function.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Generalized Biomolecular Modeling and Design with RoseTTAFold All-Atom
Authors: Krishna, R.; Wang, J.; Ahern, W.; Sturmfels, P.; Venkatesh, P.; Kalvet, I.; Lee, G. R.; Morey-Burrows, F. S.; Anishchenko, I.; Humphreys, I. R.; McHugh, R.; Vafeados, D.; Li, X.; Sutherland, G. A.; Hitchcock, A.; Hunter, C. N.; Baek, M.; DiMaio, F.; Baker, D.
Score: 234.7, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561603
Although AlphaFold2 (AF2) and RoseTTAFold (RF) have transformed structural biology by enabling high-accuracy protein structure modeling, they are unable to model covalent modifications or interactions with small molecules and other non-protein molecules that can play key roles in biological function." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-11T10:39:46+00:00" />
<meta property="article:modified_time" content="2023-10-11T10:39:46+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Generalized Biomolecular Modeling and Design with RoseTTAFold All-Atom
Authors: Krishna, R.; Wang, J.; Ahern, W.; Sturmfels, P.; Venkatesh, P.; Kalvet, I.; Lee, G. R.; Morey-Burrows, F. S.; Anishchenko, I.; Humphreys, I. R.; McHugh, R.; Vafeados, D.; Li, X.; Sutherland, G. A.; Hitchcock, A.; Hunter, C. N.; Baek, M.; DiMaio, F.; Baker, D.
Score: 234.7, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561603
Although AlphaFold2 (AF2) and RoseTTAFold (RF) have transformed structural biology by enabling high-accuracy protein structure modeling, they are unable to model covalent modifications or interactions with small molecules and other non-protein molecules that can play key roles in biological function."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Generalized Biomolecular Modeling and Design with RoseTTAFold All-Atom\nAuthors: Krishna, R.; Wang, J.; Ahern, W.; Sturmfels, P.; Venkatesh, P.; Kalvet, I.; Lee, G. R.; Morey-Burrows, F. S.; Anishchenko, I.; Humphreys, I. R.; McHugh, R.; Vafeados, D.; Li, X.; Sutherland, G. A.; Hitchcock, A.; Hunter, C. N.; Baek, M.; DiMaio, F.; Baker, D.\nScore: 234.7, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561603\nAlthough AlphaFold2 (AF2) and RoseTTAFold (RF) have transformed structural biology by enabling high-accuracy protein structure modeling, they are unable to model covalent modifications or interactions with small molecules and other non-protein molecules that can play key roles in biological function.",
  "keywords": [
    
  ],
  "articleBody": " Generalized Biomolecular Modeling and Design with RoseTTAFold All-Atom\nAuthors: Krishna, R.; Wang, J.; Ahern, W.; Sturmfels, P.; Venkatesh, P.; Kalvet, I.; Lee, G. R.; Morey-Burrows, F. S.; Anishchenko, I.; Humphreys, I. R.; McHugh, R.; Vafeados, D.; Li, X.; Sutherland, G. A.; Hitchcock, A.; Hunter, C. N.; Baek, M.; DiMaio, F.; Baker, D.\nScore: 234.7, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561603\nAlthough AlphaFold2 (AF2) and RoseTTAFold (RF) have transformed structural biology by enabling high-accuracy protein structure modeling, they are unable to model covalent modifications or interactions with small molecules and other non-protein molecules that can play key roles in biological function. Here, we describe RoseTTAFold All-Atom (RFAA), a deep network capable of modeling full biological assemblies containing proteins, nucleic acids, small molecules, metals, and covalent modifications given the sequences of the polymers and the atomic bonded geometry of the small molecules and covalent modifications. Following training on structures of full biological assemblies in the Protein Data Bank (PDB), RFAA has comparable protein structure prediction accuracy to AF2, excellent performance in CAMEO for flexible backbone small molecule docking, and reasonable prediction accuracy for protein covalent modifications and assemblies of proteins with multiple nucleic acid chains and small molecules which, to our knowledge, no existing method can model simultaneously. By fine-tuning on diffusive denoising tasks, we develop RFdiffusion All-Atom (RFdiffusionAA), which generates binding pockets by directly building protein structures around small molecules and other non-protein molecules. Starting from random distributions of amino acid residues surrounding target small molecules, we design and experimentally validate proteins that bind the cardiac disease therapeutic digoxigenin, the enzymatic cofactor heme, and optically active bilin molecules with potential for expanding the range of wavelengths captured by photosynthesis. We anticipate that RFAA and RFdiffusionAA will be widely useful for modeling and designing complex biomolecular systems.\nImproving protein expression, stability, and function with ProteinMPNN\nAuthors: Sumida, K. H.; Nunez Franco, R.; Kalvet, I.; Pellock, S. J.; Wicky, B. I. M.; Milles, L. F.; Dauparas, J.; Wang, J.; Kipnis, Y.; Jameson, N.; Kang, A.; De La Cruz, J.; Sankaran, B.; Bera, A. K.; Jimenez Oses, G.; Baker, D.\nScore: 86.8, Published: 2023-10-03 DOI: 10.1101/2023.10.03.560713\nNatural proteins are highly optimized for function, but are often difficult to produce at a scale suitable for biotechnological applications due to poor expression in heterologous systems, limited solubility, and sensitivity to temperature. Thus, a general method that improves the physical properties of native proteins while maintaining function could have wide utility for protein-based technologies. Here we show that the deep neural network ProteinMPNN together with evolutionary and structural information provides a route to increasing protein expression, stability, and function. For both myoglobin and tobacco etch virus (TEV) protease, we generated designs with improved expression, elevated melting temperatures, and improved function. For TEV protease, we identified multiple designs with improved catalytic activity as compared to the parent sequence and previously reported TEV variants. Our approach should be broadly useful for improving the expression, stability, and function of biotechnologically important proteins.\nTargeted Protein Relocalization via Protein Transport Coupling\nAuthors: Ng, C. S. C.; Liu, A.; Cui, B.; Banik, S. M.\nScore: 38.5, Published: 2023-10-04 DOI: 10.1101/2023.10.04.560943\nSubcellular protein localization regulates protein function and can be corrupted in cancers1 and neurodegenerative diseases2-4. The localization of numerous proteins has been annotated5-7, and pharmacologically relevant approaches for precise rewiring of localization to address disease-driving phenotypes would be an attractive targeted therapeutic approach. Molecules which harness the trafficking of a shuttle protein to control the subcellular localization of a target protein could provide an avenue for targeted protein relocalization for interactome-rewiring therapeutics. To realize this concept, we deploy a quantitative approach to identify features which govern the ability to hijack protein trafficking, develop a collection of shuttle proteins and ligands, and demonstrate relocalization of proteins bearing endogenous localization signals. Using a custom imaging analysis pipeline, we show that endogenous localization signals can be overcome through molecular coupling of target proteins to shuttle proteins containing sufficiently strong native localization sequences expressed in the necessary abundance. We develop nuclear hormone receptors as viable shuttles which can be harnessed with Targeted Relocalization Activating Molecules (TRAMs) to redistribute disease-driving mutant proteins such as SMARCB1Q318X, TDP43 {Delta}NLS, and FUSR495X. Small molecule-mediated relocalization of FUSR495X to the nucleus from the cytoplasm reduced the number of cellular stress granules in a model of cellular stress. Using Cas9-mediated knock-in tagging, we demonstrate nuclear enrichment of both low abundance (FOXO3a) and high abundance (FKBP12) endogenous proteins via molecular coupling to nuclear hormone receptor trafficking. Finally, small molecule-mediated redistribution of NMNAT1 from nuclei to axons in primary neurons was able to slow axonal degeneration and pharmacologically mimic the WldS gain-of-function phenotype from mice resistant to certain types of neurodegeneration8. The concept of targeted protein relocalization could therefore nucleate approaches for treating disease through interactome rewiring.\nTargeted protein degradation via intramolecular bivalent glues\nAuthors: Hsia, O.; Hinterndorfer, M.; Cowan, A. D.; Iso, K.; Ishida, T.; Sundaramoorthy, R.; Nakasone, M. A.; Imrichova, H.; Schatz, C.; Rukavina, A.; Husnjak, K.; Wegner, M.; Correa-Saez, A.; Craigon, C.; Casement, R.; Maniaci, C.; Testa, A.; Kaulich, M.; Dikic, I.; Winter, G. E.; Ciulli, A.\nScore: 62.3, Published: 2023-10-07 DOI: 10.1101/2023.02.14.528511\nTargeted protein degradation is a pharmacological modality based on the induced proximity of an E3 ubiquitin ligase and a target protein to promote target ubiquitination and proteasomal degradation. This has been achieved either via bifunctional compounds (PROTACs) composed of two separate warheads that individually bind the target and E3 ligase, or via molecular glues that monovalently bind either the ligase or the target1-4. Using orthogonal genetic screening, biophysical characterization, and structural reconstitution, we investigate the mode of action of bifunctional BRD2/4 degraders (IBG1-4) and find that - instead of connecting target and ligase in trans as PROTACs do - they simultaneously engage two adjacent domains of the target protein in cis. This conformational change glues BRD4 to the E3 ligases DCAF11 or DCAF16, leveraging intrinsic target-ligase affinities which, albeit pre-existing, do not translate to BRD4 degradation in absence of compound. Structural insights into the ternary BRD4:IBG1:DCAF16 complex guided the rational design of improved degraders of low picomolar potency. We thus introduce a new modality in targeted protein degradation, termed intramolecular bivalent glues (IBGs), which work by bridging protein domains to enhance surface complementarity with E3 ligases for productive ubiquitination and degradation.\nMeasuring carbohydrate recognition profile of lectins on live cells using liquid glycan array (LiGA)\nAuthors: Sojitra, M.; Schmidt, E. N.; Lima, G. M.; Carpenter, E. J.; McCord, K. A.; Atrazhev, A.; Macauley, M. S.; Derda, R.\nScore: 11.9, Published: 2023-10-05 DOI: 10.1101/2023.10.04.559794\nGlycans constitute a significant fraction of biomolecular diversity on the surface of cells across all the species in all kingdoms of life. As the structure of glycans is not encoded by the DNA of the host organisms, it is impossible to use cutting-edge DNA technology to study the role of cellular glycosylation or to understand how cell-surface glycome is recognized by glycan-binding proteins (GBPs). To address this gap, we recently described a genetically-encoded liquid glycan array (LiGA) platform that allows profiling of glycan:GBP interactions on the surface of live cells in vitro and in vivo using next-generation sequencing (NGS). LiGA is a library of DNA-barcoded bacteriophages coated with 5-1500 copies of a glycan; the DNA barcode inside each bacteriophage encodes the structure and density of the displayed glycans. Deep sequencing of the glycophages associated with live cells yields a glycan-binding profile of GBPs displayed on the surface of such cells. This protocol provides detailed instructions of using LiGA to probe cell surface receptors and includes information on the preparation of glycophages, analysis by MALDI-TOF MS, the assembly of a LiGA library, and its deep-sequencing. Using the protocol detailed in this report, we measure a glycan-binding profile of the immunomodulatory SiglecLJ1, -2, -6, -7, and -9 expressed on the surface of different cell types and uncover previously unknown environment-dependent recognition of glycans by Siglec-receptors on the surface of live cells. Protocols similar to the one described in this report will make it possible to measure the precise glycan-binding profile of any GPBs displayed on the surface of any cell types.\nComputational prediction of structure, function and interaction of aphid salivary effector proteins\nAuthors: Waksman, T.; Astin, E.; Fisher, S. R.; Hunter, W.; Bos, J. I.\nScore: 13.0, Published: 2023-10-02 DOI: 10.1101/2023.10.02.560451\nSimilar to plant pathogens, phloem-feeding insects such as aphids deliver effector proteins inside their hosts that act to promote host susceptibility and enable feeding and infestation. Despite exciting progress towards identifying and characterizing effector proteins from these insects, their functions remain largely unknown. The recent ground-breaking development in protein structure prediction algorithms combined with the availability of proteomics and transcriptomic datasets for agriculturally important pests, such as the aphid Myzus persicae (green peach aphid), provides new opportunities to explore the structural and functional diversity of effector repertoires. In this study, we sought to gain insight into the the M. persicae effector repertoire by predicting and analysing the structures of a set of 71 effector candidate proteins. We used two protein structure prediction methods, AlphaFold and OmegaFold, which produced mutually consistent results. We observed a wide continuous spectrum of sizes and structures among the effector candidates, from disordered proteins to globular enzymes. We made use of the structural information and state-of-the-art computational methods to predict M. persicae effector protein properties, including function and interaction with host plant proteins. Overall, our investigation provides novel insights into the structure, function, and interaction prediction of aphid effector repertoires and will guide the necessary experimental characterization to address new hypotheses.\nStructural and molecular basis of choline uptake into the brain by FLVCR2\nAuthors: Cater, R. J.; Mukherjee, D.; Gil Iturbe, E.; Erramilli, S. K.; Chen, T.; Koo, K.; Santander, N. G.; Reckers, A. R.; Kloss, B.; Gawda, T.; Choy, B. C.; Zhang, Z.; Clarke, O. B.; Yee, S. W.; Kossiakoff, A. K.; Quick, M.; Arnold, T.; Mancia, F.\nScore: 9.7, Published: 2023-10-05 DOI: 10.1101/2023.10.05.561059\nCholine is an essential nutrient that the human body needs in vast quantities for cell membrane synthesis, epigenetic modification, and neurotransmission. The brain has a particularly high demand for choline, but how it enters the brain has eluded the field for over fifty years. The MFS transporter FLVCR1 was recently determined to be a choline transporter, and while this protein is not highly expressed at the blood-brain barrier (BBB), its relative FLVCR2 is. Previous studies have shown that mutations in human Flvcr2 cause cerebral vascular abnormalities, hydrocephalus, and embryonic lethality, but the physiological role of FLVCR2 is unknown. Here, we demonstrate both in vivo and in vitro that FLVCR2 is a BBB choline transporter and is responsible for the majority of choline uptake into the brain. We also determine the structures of choline-bound FLVCR2 in the inward- and outward-facing states using cryo-electron microscopy to 2.49 and 2.77 [A] resolution, respectively. These results reveal how the brain obtains choline and provide molecular-level insights into how FLVCR2 binds choline in an aromatic cage and mediates its uptake. Our work could provide a novel framework for the targeted delivery of neurotherapeutics into the brain.\nStructural insights into the FtsEX-EnvC complex regulation on septal peptidoglycan hydrolysis in Vibrio cholerae\nAuthors: Hao, A.; Suo, Y.; Lee, S.-Y.\nScore: 9.0, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561269\nDuring bacterial cell division, hydrolysis of septal peptidoglycan (sPG) is crucial for cell separation. This sPG hydrolysis is performed by the enzyme amidases whose activity is regulated by the integral membrane protein complex FtsEX-EnvC. FtsEX is an ATP-binding cassette transporter and EnvC is a long coiled-coiled protein that interacts with and activates the amidases. The molecular mechanism by which the FtsEX-EnvC complex activates amidases remains largely unclear. We present the cryo-electron microscopy structure of the FtsEX-EnvC complex from the pathogenic bacteria V. cholerae (FtsEX-EnvCVC). FtsEX-EnvCVC in the presence of ADP adopts a distinct conformation where EnvC is horizontally extended rather than vertically extended. Subsequent structural studies suggest that EnvC can swing between these conformations in space in a nucleotide-dependent manner. Our structural analysis and functional studies suggest that FtsEX-EnvCvc may employ spatial control of EnvC for amidase activation, suggesting species-specific diversity in the FtsEX-EnvC regulation on septal peptidoglycan hydrolysis.\nDissecting telomerase RNA structural heterogeneity in living human cells with DMS-MaPseq\nAuthors: Forino, N. M.; Woo, J. Z.; Zaug, A. J.; Jimenez, A. G.; Cech, T. R.; Rouskin, S.; Stone, M.\nScore: 8.6, Published: 2023-10-05 DOI: 10.1101/2023.10.04.560962\nTelomerase is a specialized reverse transcriptase that uses an intrinsic RNA subunit as the template for telomeric DNA synthesis. Biogenesis of human telomerase requires its RNA subunit (hTR) to fold into a multi-domain architecture that includes the template-containing pseudoknot (t/PK) and the three-way junction (CR4/5). These two hTR domains bind the telomerase reverse transcriptase (hTERT) protein and are thus essential for telomerase catalytic activity. Here, we probe the structure of hTR in living cells using dimethyl sulfate mutational profiling with sequencing (DMS-MaPseq) and ensemble deconvolution analysis. Unexpectedly, approximately 15% of the steady state population of hTR has a CR4/5 conformation lacking features required for hTERT binding. Mutagenesis demonstrates that stabilization of the alternative CR4/5 conformation is detrimental to telomerase assembly and activity. We propose that this misfolded portion of the cellular hTR pool is either slowly refolded or degraded. Thus, kinetic traps for RNA folding that have been so well-studied in vitro may also present barriers for assembly of ribonucleoprotein complexes in vivo.\nEngineered blue-shifted melanopsins for subcellular optogenetics\nAuthors: Wijayaratna, D.; Sacchetta, F.; Pedraza Gonzalez, L.; Fanelli, F.; Sugihara, T.; Koyanagi, M.; Piyawardana, S.; Ghotra, K.; Thotamune, W.; Terakita, A.; Olivucci, M.; Karunarathne, A.\nScore: 5.9, Published: 2023-10-08 DOI: 10.1101/2023.10.07.561352\nMelanopsin (MeOp) is a G protein-coupled Receptor (GPCR) family photopigment, expressed in intrinsically photosensitive retinal ganglion cells (ipRGCs) that display remarkable functional diversity. In addition to non-image-forming visual functions, MeOp also controls signaling underlying the retina development, circadian clock, mood, and behavior. MeOp is bistable, recycles retinal, and can function under low retinaldehyde availability. It also activates multiple G protein heterotrimers. Though MeOp could be a versatile optogenetic tool, its potential, especially its utility for subcellular signaling control, is hampered by the broader spectral sensitivity spanning the entire visible range. Here, we use a recently reported in silico technology called Automatic Rhodopsin Modeling (ARM) to identify blue-shifting mutations of MeOp and, ultimately, allow for imaging biosensors with red light without activating the opsin. Accordingly, ARM was used to construct validated quantum mechanics/molecular mechanics (QM/MM) models for mouse MeOp (mMeOp) to search and optimize a set of mutants featuring a blue-shifted light absorption. We demonstrate that four mutants of such can be successfully expressed and display the required resistance to activation by red light; however, they are activated by yellow, green, and blue light. Localized subcellular optical activation of these mutants in macrophage cells showed localized PIP3 generation and cell migration. Further characterization showed that MeOp blue-shifted mutants are also bistable. Altogether, our data demonstrate the computer-aided engineering feasibility of opsins with desired spectral properties for subcellular optogenetic applications.\n",
  "wordCount" : "2474",
  "inLanguage": "en",
  "datePublished": "2023-10-11T10:39:46Z",
  "dateModified": "2023-10-11T10:39:46Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta"><span>updated on October 11, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561603">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561603" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561603">
        <p class="paperTitle">Generalized Biomolecular Modeling and Design with RoseTTAFold All-Atom</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561603" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561603" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krishna, R.; Wang, J.; Ahern, W.; Sturmfels, P.; Venkatesh, P.; Kalvet, I.; Lee, G. R.; Morey-Burrows, F. S.; Anishchenko, I.; Humphreys, I. R.; McHugh, R.; Vafeados, D.; Li, X.; Sutherland, G. A.; Hitchcock, A.; Hunter, C. N.; Baek, M.; DiMaio, F.; Baker, D.</p>
        <p class="info">Score: 234.7, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561603' target='https://doi.org/10.1101/2023.10.09.561603'> 10.1101/2023.10.09.561603</a></p>
        <p class="abstract">Although AlphaFold2 (AF2) and RoseTTAFold (RF) have transformed structural biology by enabling high-accuracy protein structure modeling, they are unable to model covalent modifications or interactions with small molecules and other non-protein molecules that can play key roles in biological function. Here, we describe RoseTTAFold All-Atom (RFAA), a deep network capable of modeling full biological assemblies containing proteins, nucleic acids, small molecules, metals, and covalent modifications given the sequences of the polymers and the atomic bonded geometry of the small molecules and covalent modifications. Following training on structures of full biological assemblies in the Protein Data Bank (PDB), RFAA has comparable protein structure prediction accuracy to AF2, excellent performance in CAMEO for flexible backbone small molecule docking, and reasonable prediction accuracy for protein covalent modifications and assemblies of proteins with multiple nucleic acid chains and small molecules which, to our knowledge, no existing method can model simultaneously. By fine-tuning on diffusive denoising tasks, we develop RFdiffusion All-Atom (RFdiffusionAA), which generates binding pockets by directly building protein structures around small molecules and other non-protein molecules. Starting from random distributions of amino acid residues surrounding target small molecules, we design and experimentally validate proteins that bind the cardiac disease therapeutic digoxigenin, the enzymatic cofactor heme, and optically active bilin molecules with potential for expanding the range of wavelengths captured by photosynthesis. We anticipate that RFAA and RFdiffusionAA will be widely useful for modeling and designing complex biomolecular systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.560713">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.560713" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.560713">
        <p class="paperTitle">Improving protein expression, stability, and function with ProteinMPNN</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.560713" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.560713" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sumida, K. H.; Nunez Franco, R.; Kalvet, I.; Pellock, S. J.; Wicky, B. I. M.; Milles, L. F.; Dauparas, J.; Wang, J.; Kipnis, Y.; Jameson, N.; Kang, A.; De La Cruz, J.; Sankaran, B.; Bera, A. K.; Jimenez Oses, G.; Baker, D.</p>
        <p class="info">Score: 86.8, Published: 2023-10-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.560713' target='https://doi.org/10.1101/2023.10.03.560713'> 10.1101/2023.10.03.560713</a></p>
        <p class="abstract">Natural proteins are highly optimized for function, but are often difficult to produce at a scale suitable for biotechnological applications due to poor expression in heterologous systems, limited solubility, and sensitivity to temperature. Thus, a general method that improves the physical properties of native proteins while maintaining function could have wide utility for protein-based technologies. Here we show that the deep neural network ProteinMPNN together with evolutionary and structural information provides a route to increasing protein expression, stability, and function. For both myoglobin and tobacco etch virus (TEV) protease, we generated designs with improved expression, elevated melting temperatures, and improved function. For TEV protease, we identified multiple designs with improved catalytic activity as compared to the parent sequence and previously reported TEV variants. Our approach should be broadly useful for improving the expression, stability, and function of biotechnologically important proteins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560943">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560943" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560943">
        <p class="paperTitle">Targeted Protein Relocalization via Protein Transport Coupling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560943" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560943" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ng, C. S. C.; Liu, A.; Cui, B.; Banik, S. M.</p>
        <p class="info">Score: 38.5, Published: 2023-10-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560943' target='https://doi.org/10.1101/2023.10.04.560943'> 10.1101/2023.10.04.560943</a></p>
        <p class="abstract">Subcellular protein localization regulates protein function and can be corrupted in cancers1 and neurodegenerative diseases2-4. The localization of numerous proteins has been annotated5-7, and pharmacologically relevant approaches for precise rewiring of localization to address disease-driving phenotypes would be an attractive targeted therapeutic approach. Molecules which harness the trafficking of a shuttle protein to control the subcellular localization of a target protein could provide an avenue for targeted protein relocalization for interactome-rewiring therapeutics. To realize this concept, we deploy a quantitative approach to identify features which govern the ability to hijack protein trafficking, develop a collection of shuttle proteins and ligands, and demonstrate relocalization of proteins bearing endogenous localization signals. Using a custom imaging analysis pipeline, we show that endogenous localization signals can be overcome through molecular coupling of target proteins to shuttle proteins containing sufficiently strong native localization sequences expressed in the necessary abundance. We develop nuclear hormone receptors as viable shuttles which can be harnessed with Targeted Relocalization Activating Molecules (TRAMs) to redistribute disease-driving mutant proteins such as SMARCB1Q318X, TDP43 {Delta}NLS, and FUSR495X. Small molecule-mediated relocalization of FUSR495X to the nucleus from the cytoplasm reduced the number of cellular stress granules in a model of cellular stress. Using Cas9-mediated knock-in tagging, we demonstrate nuclear enrichment of both low abundance (FOXO3a) and high abundance (FKBP12) endogenous proteins via molecular coupling to nuclear hormone receptor trafficking. Finally, small molecule-mediated redistribution of NMNAT1 from nuclei to axons in primary neurons was able to slow axonal degeneration and pharmacologically mimic the WldS gain-of-function phenotype from mice resistant to certain types of neurodegeneration8. The concept of targeted protein relocalization could therefore nucleate approaches for treating disease through interactome rewiring.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.02.14.528511">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.02.14.528511" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.02.14.528511">
        <p class="paperTitle">Targeted protein degradation via intramolecular bivalent glues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.02.14.528511" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.02.14.528511" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hsia, O.; Hinterndorfer, M.; Cowan, A. D.; Iso, K.; Ishida, T.; Sundaramoorthy, R.; Nakasone, M. A.; Imrichova, H.; Schatz, C.; Rukavina, A.; Husnjak, K.; Wegner, M.; Correa-Saez, A.; Craigon, C.; Casement, R.; Maniaci, C.; Testa, A.; Kaulich, M.; Dikic, I.; Winter, G. E.; Ciulli, A.</p>
        <p class="info">Score: 62.3, Published: 2023-10-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.02.14.528511' target='https://doi.org/10.1101/2023.02.14.528511'> 10.1101/2023.02.14.528511</a></p>
        <p class="abstract">Targeted protein degradation is a pharmacological modality based on the induced proximity of an E3 ubiquitin ligase and a target protein to promote target ubiquitination and proteasomal degradation. This has been achieved either via bifunctional compounds (PROTACs) composed of two separate warheads that individually bind the target and E3 ligase, or via molecular glues that monovalently bind either the ligase or the target1-4. Using orthogonal genetic screening, biophysical characterization, and structural reconstitution, we investigate the mode of action of bifunctional BRD2/4 degraders (IBG1-4) and find that - instead of connecting target and ligase in trans as PROTACs do - they simultaneously engage two adjacent domains of the target protein in cis. This conformational change glues BRD4 to the E3 ligases DCAF11 or DCAF16, leveraging intrinsic target-ligase affinities which, albeit pre-existing, do not translate to BRD4 degradation in absence of compound. Structural insights into the ternary BRD4:IBG1:DCAF16 complex guided the rational design of improved degraders of low picomolar potency. We thus introduce a new modality in targeted protein degradation, termed intramolecular bivalent glues (IBGs), which work by bridging protein domains to enhance surface complementarity with E3 ligases for productive ubiquitination and degradation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.559794">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.559794" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.559794">
        <p class="paperTitle">Measuring carbohydrate recognition profile of lectins on live cells using liquid glycan array (LiGA)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.559794" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.559794" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sojitra, M.; Schmidt, E. N.; Lima, G. M.; Carpenter, E. J.; McCord, K. A.; Atrazhev, A.; Macauley, M. S.; Derda, R.</p>
        <p class="info">Score: 11.9, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.559794' target='https://doi.org/10.1101/2023.10.04.559794'> 10.1101/2023.10.04.559794</a></p>
        <p class="abstract">Glycans constitute a significant fraction of biomolecular diversity on the surface of cells across all the species in all kingdoms of life. As the structure of glycans is not encoded by the DNA of the host organisms, it is impossible to use cutting-edge DNA technology to study the role of cellular glycosylation or to understand how cell-surface glycome is recognized by glycan-binding proteins (GBPs). To address this gap, we recently described a genetically-encoded liquid glycan array (LiGA) platform that allows profiling of glycan:GBP interactions on the surface of live cells in vitro and in vivo using next-generation sequencing (NGS). LiGA is a library of DNA-barcoded bacteriophages coated with 5-1500 copies of a glycan; the DNA barcode inside each bacteriophage encodes the structure and density of the displayed glycans. Deep sequencing of the glycophages associated with live cells yields a glycan-binding profile of GBPs displayed on the surface of such cells. This protocol provides detailed instructions of using LiGA to probe cell surface receptors and includes information on the preparation of glycophages, analysis by MALDI-TOF MS, the assembly of a LiGA library, and its deep-sequencing. Using the protocol detailed in this report, we measure a glycan-binding profile of the immunomodulatory SiglecLJ1, -2, -6, -7, and -9 expressed on the surface of different cell types and uncover previously unknown environment-dependent recognition of glycans by Siglec-receptors on the surface of live cells. Protocols similar to the one described in this report will make it possible to measure the precise glycan-binding profile of any GPBs displayed on the surface of any cell types.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560451">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560451" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560451">
        <p class="paperTitle">Computational prediction of structure, function and interaction of aphid salivary effector proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560451" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560451" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Waksman, T.; Astin, E.; Fisher, S. R.; Hunter, W.; Bos, J. I.</p>
        <p class="info">Score: 13.0, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560451' target='https://doi.org/10.1101/2023.10.02.560451'> 10.1101/2023.10.02.560451</a></p>
        <p class="abstract">Similar to plant pathogens, phloem-feeding insects such as aphids deliver effector proteins inside their hosts that act to promote host susceptibility and enable feeding and infestation. Despite exciting progress towards identifying and characterizing effector proteins from these insects, their functions remain largely unknown. The recent ground-breaking development in protein structure prediction algorithms combined with the availability of proteomics and transcriptomic datasets for agriculturally important pests, such as the aphid Myzus persicae (green peach aphid), provides new opportunities to explore the structural and functional diversity of effector repertoires. In this study, we sought to gain insight into the the M. persicae effector repertoire by predicting and analysing the structures of a set of 71 effector candidate proteins. We used two protein structure prediction methods, AlphaFold and OmegaFold, which produced mutually consistent results. We observed a wide continuous spectrum of sizes and structures among the effector candidates, from disordered proteins to globular enzymes. We made use of the structural information and state-of-the-art computational methods to predict M. persicae effector protein properties, including function and interaction with host plant proteins. Overall, our investigation provides novel insights into the structure, function, and interaction prediction of aphid effector repertoires and will guide the necessary experimental characterization to address new hypotheses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.561059">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.561059" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.561059">
        <p class="paperTitle">Structural and molecular basis of choline uptake into the brain by FLVCR2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.561059" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.561059" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cater, R. J.; Mukherjee, D.; Gil Iturbe, E.; Erramilli, S. K.; Chen, T.; Koo, K.; Santander, N. G.; Reckers, A. R.; Kloss, B.; Gawda, T.; Choy, B. C.; Zhang, Z.; Clarke, O. B.; Yee, S. W.; Kossiakoff, A. K.; Quick, M.; Arnold, T.; Mancia, F.</p>
        <p class="info">Score: 9.7, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.561059' target='https://doi.org/10.1101/2023.10.05.561059'> 10.1101/2023.10.05.561059</a></p>
        <p class="abstract">Choline is an essential nutrient that the human body needs in vast quantities for cell membrane synthesis, epigenetic modification, and neurotransmission. The brain has a particularly high demand for choline, but how it enters the brain has eluded the field for over fifty years. The MFS transporter FLVCR1 was recently determined to be a choline transporter, and while this protein is not highly expressed at the blood-brain barrier (BBB), its relative FLVCR2 is. Previous studies have shown that mutations in human Flvcr2 cause cerebral vascular abnormalities, hydrocephalus, and embryonic lethality, but the physiological role of FLVCR2 is unknown. Here, we demonstrate both in vivo and in vitro that FLVCR2 is a BBB choline transporter and is responsible for the majority of choline uptake into the brain. We also determine the structures of choline-bound FLVCR2 in the inward- and outward-facing states using cryo-electron microscopy to 2.49 and 2.77 [A] resolution, respectively. These results reveal how the brain obtains choline and provide molecular-level insights into how FLVCR2 binds choline in an aromatic cage and mediates its uptake. Our work could provide a novel framework for the targeted delivery of neurotherapeutics into the brain.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561269">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561269" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561269">
        <p class="paperTitle">Structural insights into the FtsEX-EnvC complex regulation on septal peptidoglycan hydrolysis in Vibrio cholerae</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561269" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561269" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hao, A.; Suo, Y.; Lee, S.-Y.</p>
        <p class="info">Score: 9.0, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561269' target='https://doi.org/10.1101/2023.10.06.561269'> 10.1101/2023.10.06.561269</a></p>
        <p class="abstract">During bacterial cell division, hydrolysis of septal peptidoglycan (sPG) is crucial for cell separation. This sPG hydrolysis is performed by the enzyme amidases whose activity is regulated by the integral membrane protein complex FtsEX-EnvC. FtsEX is an ATP-binding cassette transporter and EnvC is a long coiled-coiled protein that interacts with and activates the amidases. The molecular mechanism by which the FtsEX-EnvC complex activates amidases remains largely unclear. We present the cryo-electron microscopy structure of the FtsEX-EnvC complex from the pathogenic bacteria V. cholerae (FtsEX-EnvCVC). FtsEX-EnvCVC in the presence of ADP adopts a distinct conformation where EnvC is horizontally extended rather than vertically extended. Subsequent structural studies suggest that EnvC can swing between these conformations in space in a nucleotide-dependent manner. Our structural analysis and functional studies suggest that FtsEX-EnvCvc may employ spatial control of EnvC for amidase activation, suggesting species-specific diversity in the FtsEX-EnvC regulation on septal peptidoglycan hydrolysis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560962">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560962" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560962">
        <p class="paperTitle">Dissecting telomerase RNA structural heterogeneity in living human cells with DMS-MaPseq</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560962" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560962" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Forino, N. M.; Woo, J. Z.; Zaug, A. J.; Jimenez, A. G.; Cech, T. R.; Rouskin, S.; Stone, M.</p>
        <p class="info">Score: 8.6, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560962' target='https://doi.org/10.1101/2023.10.04.560962'> 10.1101/2023.10.04.560962</a></p>
        <p class="abstract">Telomerase is a specialized reverse transcriptase that uses an intrinsic RNA subunit as the template for telomeric DNA synthesis. Biogenesis of human telomerase requires its RNA subunit (hTR) to fold into a multi-domain architecture that includes the template-containing pseudoknot (t/PK) and the three-way junction (CR4/5). These two hTR domains bind the telomerase reverse transcriptase (hTERT) protein and are thus essential for telomerase catalytic activity. Here, we probe the structure of hTR in living cells using dimethyl sulfate mutational profiling with sequencing (DMS-MaPseq) and ensemble deconvolution analysis. Unexpectedly, approximately 15% of the steady state population of hTR has a CR4/5 conformation lacking features required for hTERT binding. Mutagenesis demonstrates that stabilization of the alternative CR4/5 conformation is detrimental to telomerase assembly and activity. We propose that this misfolded portion of the cellular hTR pool is either slowly refolded or degraded. Thus, kinetic traps for RNA folding that have been so well-studied in vitro may also present barriers for assembly of ribonucleoprotein complexes in vivo.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.07.561352">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.07.561352" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.07.561352">
        <p class="paperTitle">Engineered blue-shifted melanopsins for subcellular optogenetics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.07.561352" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.07.561352" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wijayaratna, D.; Sacchetta, F.; Pedraza Gonzalez, L.; Fanelli, F.; Sugihara, T.; Koyanagi, M.; Piyawardana, S.; Ghotra, K.; Thotamune, W.; Terakita, A.; Olivucci, M.; Karunarathne, A.</p>
        <p class="info">Score: 5.9, Published: 2023-10-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.07.561352' target='https://doi.org/10.1101/2023.10.07.561352'> 10.1101/2023.10.07.561352</a></p>
        <p class="abstract">Melanopsin (MeOp) is a G protein-coupled Receptor (GPCR) family photopigment, expressed in intrinsically photosensitive retinal ganglion cells (ipRGCs) that display remarkable functional diversity. In addition to non-image-forming visual functions, MeOp also controls signaling underlying the retina development, circadian clock, mood, and behavior. MeOp is bistable, recycles retinal, and can function under low retinaldehyde availability. It also activates multiple G protein heterotrimers. Though MeOp could be a versatile optogenetic tool, its potential, especially its utility for subcellular signaling control, is hampered by the broader spectral sensitivity spanning the entire visible range. Here, we use a recently reported in silico technology called Automatic Rhodopsin Modeling (ARM) to identify blue-shifting mutations of MeOp and, ultimately, allow for imaging biosensors with red light without activating the opsin. Accordingly, ARM was used to construct validated quantum mechanics/molecular mechanics (QM/MM) models for mouse MeOp (mMeOp) to search and optimize a set of mutants featuring a blue-shifted light absorption. We demonstrate that four mutants of such can be successfully expressed and display the required resistance to activation by red light; however, they are activated by yellow, green, and blue light. Localized subcellular optical activation of these mutants in macrophage cells showed localized PIP3 generation and cell migration. Further characterization showed that MeOp blue-shifted mutants are also bistable. Altogether, our data demonstrate the computer-aided engineering feasibility of opsins with desired spectral properties for subcellular optogenetic applications.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
