<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="PROxAb Shuttle: A non-covalent plug-and-play platform for the rapid generation of tumor-targeting antibody-PROTAC conjugates
Authors: Schneider, H.; Jaeger, S.; Koenning, D.; Rasche, N.; Schroeter, C.; Elter, D.; Evers, A.; Lecomte, M.; Riccardi Sirtori, F.; Schwarz, D.; Wegener, A.; Hartung, I.; Rieker, M.
Score: 21.5, Published: 2023-09-29 DOI: 10.1101/2023.09.29.558399
Proteolysis-targeting chimeras (PROTACs) have evolved in recent years from an academic idea to a therapeutic modality with more than 25 active clinical programs.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="PROxAb Shuttle: A non-covalent plug-and-play platform for the rapid generation of tumor-targeting antibody-PROTAC conjugates
Authors: Schneider, H.; Jaeger, S.; Koenning, D.; Rasche, N.; Schroeter, C.; Elter, D.; Evers, A.; Lecomte, M.; Riccardi Sirtori, F.; Schwarz, D.; Wegener, A.; Hartung, I.; Rieker, M.
Score: 21.5, Published: 2023-09-29 DOI: 10.1101/2023.09.29.558399
Proteolysis-targeting chimeras (PROTACs) have evolved in recent years from an academic idea to a therapeutic modality with more than 25 active clinical programs." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-04T10:39:33+00:00" />
<meta property="article:modified_time" content="2023-10-04T10:39:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="PROxAb Shuttle: A non-covalent plug-and-play platform for the rapid generation of tumor-targeting antibody-PROTAC conjugates
Authors: Schneider, H.; Jaeger, S.; Koenning, D.; Rasche, N.; Schroeter, C.; Elter, D.; Evers, A.; Lecomte, M.; Riccardi Sirtori, F.; Schwarz, D.; Wegener, A.; Hartung, I.; Rieker, M.
Score: 21.5, Published: 2023-09-29 DOI: 10.1101/2023.09.29.558399
Proteolysis-targeting chimeras (PROTACs) have evolved in recent years from an academic idea to a therapeutic modality with more than 25 active clinical programs."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "PROxAb Shuttle: A non-covalent plug-and-play platform for the rapid generation of tumor-targeting antibody-PROTAC conjugates\nAuthors: Schneider, H.; Jaeger, S.; Koenning, D.; Rasche, N.; Schroeter, C.; Elter, D.; Evers, A.; Lecomte, M.; Riccardi Sirtori, F.; Schwarz, D.; Wegener, A.; Hartung, I.; Rieker, M.\nScore: 21.5, Published: 2023-09-29 DOI: 10.1101/2023.09.29.558399\nProteolysis-targeting chimeras (PROTACs) have evolved in recent years from an academic idea to a therapeutic modality with more than 25 active clinical programs.",
  "keywords": [
    
  ],
  "articleBody": " PROxAb Shuttle: A non-covalent plug-and-play platform for the rapid generation of tumor-targeting antibody-PROTAC conjugates\nAuthors: Schneider, H.; Jaeger, S.; Koenning, D.; Rasche, N.; Schroeter, C.; Elter, D.; Evers, A.; Lecomte, M.; Riccardi Sirtori, F.; Schwarz, D.; Wegener, A.; Hartung, I.; Rieker, M.\nScore: 21.5, Published: 2023-09-29 DOI: 10.1101/2023.09.29.558399\nProteolysis-targeting chimeras (PROTACs) have evolved in recent years from an academic idea to a therapeutic modality with more than 25 active clinical programs. However, achieving oral bioavailability and cell-type specificity remains a challenge, especially for PROTACs recruiting the von Hippel-Lindau (VHL) E3 ligase. Herein, we present an unprecedented, plug-and-play platform for VHL-recruiting PROTACs to overcome these limitations. Our platform allows for the generation of non-covalent antibody-PROTAC complexes within minutes and obviates the need for prior PROTAC modification, antibody-drug linker chemistry optimization or bioconjugation. Our technology relies on camelid-derived antibody domains (VHHs) which can easily be engineered into existing therapeutic antibody scaffolds. The resulting targeted, bispecific fusion proteins can be complexed with PROTACs at defined PROTAC-to-antibody ratios and have been termed PROxAb Shuttles. PROxAb Shuttles can prolong the half-life of PROTACs from hours to days, demonstrate anti-tumor efficacy in vivo and have the potential for reloading in vivo to further boost efficacy.\nMapping the biochemical landscape of rubisco\nAuthors: Prywes, N.; Phillips, N. R.; Oltrogge, L. M.; de Pins, B.; Cowan, A. E.; Taylor-Kearney, L. J.; Chang, H. A.; Hall, L. N.; Bhatt, A.; Shih, P.; Milo, R.; Savage, D. F.\nScore: 21.1, Published: 2023-09-27 DOI: 10.1101/2023.09.27.559826\nThe enzyme rubisco catalyzes the first step of carbon assimilation in photosynthesis. Despite the massive flux of CO2 passing through this active site over billions of years, extant rubisco has relatively slow kinetics and is prone to off-target activity. In many growth regimes, this limits photosynthesis in planta. Many attempts have been made to improve the kinetic parameters of rubisco with limited success, potentially due to biochemical trade-offs. To understand the structural basis of constraints on rubisco, a comprehensive map of rubisco at the individual amino-acid level is needed. To that end we performed a deep mutational scan using a rubisco-dependent E. coli strain. By titrating CO2 concentrations it was possible to determine estimations for both catalytic rate (kcat) and substrate affinity (KM) of \u003e99% of rubisco point mutants. Some positions were found to act as \"rheostats\" where some amino-acid substitutions reduced -while others improved -affinity for CO2. No individual point mutation was found to substantially improve the catalytic rate, but a number of highly phylogenetically conserved positions were found to tolerate mutations, indicating that a large portion of rubiscos sequence space remains unexplored by nature and may serve as a resource for future protein engineering efforts. Together, these biochemical measurements inform our understanding of biochemical tradeoffs and will assist in future efforts to improve rubisco catalytic properties.\nUsing a function-first \"scout fragment\"-based approach to develop allosteric covalent inhibitors of conformationally dynamic helicase mechanoenzymes\nAuthors: Ramsey, J. R.; Shelton, P. M. M.; Heiss, T. K.; Olinares, P. D. B.; Vostal, L. E.; Soileau, H.; Grasso, M.; Warrington, S.; Adaniya, S.; Miller, M.; Sun, S.; Huggins, D. J.; Myers, R. W.; Chait, B. T.; Vinogradova, E. V.; Kapoor, T. M.\nScore: 16.4, Published: 2023-09-26 DOI: 10.1101/2023.09.25.559391\nHelicases, classified into six superfamilies, are mechanoenzymes that utilize energy derived from ATP hydrolysis to remodel DNA and RNA substrates. These enzymes have key roles in diverse cellular processes, such as genome replication and maintenance, ribosome assembly and translation. Helicases with essential functions only in certain cancer cells have been identified and helicases expressed by certain viruses are required for their pathogenicity. As a result, helicases are important targets for chemical probes and therapeutics. However, it has been very challenging to develop selective chemical inhibitors for helicases, enzymes with highly dynamic conformations. We envisioned that electrophilic scout fragments, which have been used for chemical proteomic based profiling, could be leveraged to develop covalent inhibitors of helicases. We adopted a function-first approach, combining enzymatic assays with enantiomeric probe pairs and mass spectrometry, to develop a covalent inhibitor that selectively targets an allosteric site in SARS-CoV-2 nsp13, a superfamily-1 helicase. Further, we demonstrate that scout fragments inhibit the activity of two human superfamily-2 helicases, BLM and WRN, involved in genome maintenance. Together, our findings suggest a covalent inhibitor discovery approach to target helicases and potentially other conformationally dynamic mechanoenzymes.\nTranscriptome-wide RNA structure probing with temporal resolution\nAuthors: Wang, D.; Tang, Y.; Li, A.; Sun, L.; Kleiner, R. E.\nScore: 15.5, Published: 2023-09-29 DOI: 10.1101/2023.09.28.560059\nReactive small-molecule probes are widely used for RNA structure probing, however current approaches largely measure average RNA transcript dynamics and do not resolve structural differences that occur during folding or transcript maturation. Here, we present SNIPER-seq, an RNA structure probing method relying upon metabolic labeling with 2'-aminodeoxycytidine, structure-dependent 2'-amino reaction with an aromatic isothiocyanate, and high-throughput RNA sequencing. Our method maps cellular RNA structure transcriptome-wide with temporal resolution enabling determination of transcript age-dependent RNA structural dynamics. We benchmark our approach against known RNA structures and investigate the dynamics of human 5S rRNA during ribosome biogenesis, revealing specific structural changes in 5S rRNA loops that occur over the course of several hours. Taken together, our work sheds light on the maturation and coordinated conformational changes that take place during ribosome biogenesis and provides a general strategy for surveying evolving RNA structural dynamics across the transcriptome.\nImproving protein expression, stability, and function with ProteinMPNN\nAuthors: Sumida, K. H.; Nunez Franco, R.; Kalvet, I.; Pellock, S. J.; Wicky, B. I. M.; Milles, L. F.; Dauparas, J.; Wang, J.; Kipnis, Y.; Jameson, N.; Kang, A.; De La Cruz, J.; Sankaran, B.; Bera, A. K.; Jimenez Oses, G.; Baker, D.\nScore: 13.4, Published: 2023-10-03 DOI: 10.1101/2023.10.03.560713\nNatural proteins are highly optimized for function, but are often difficult to produce at a scale suitable for biotechnological applications due to poor expression in heterologous systems, limited solubility, and sensitivity to temperature. Thus, a general method that improves the physical properties of native proteins while maintaining function could have wide utility for protein-based technologies. Here we show that the deep neural network ProteinMPNN together with evolutionary and structural information provides a route to increasing protein expression, stability, and function. For both myoglobin and tobacco etch virus (TEV) protease, we generated designs with improved expression, elevated melting temperatures, and improved function. For TEV protease, we identified multiple designs with improved catalytic activity as compared to the parent sequence and previously reported TEV variants. Our approach should be broadly useful for improving the expression, stability, and function of biotechnologically important proteins.\nStructural pathway for class III PI 3-kinase activation by the myristoylated GTP-binding pseudokinase VPS15\nAuthors: Cook, A. S. I.; Chen, M. H.; Ren, X.; Rao, S.; Garcia, S. N.; Claveras Cabezudo, A.; Iavarone, A. T.; Hummer, G.; Hurley, J. H.\nScore: 11.0, Published: 2023-09-28 DOI: 10.1101/2023.09.28.559894\nThe class III phosphatidylinositol (PI) 3-kinase complexes I and II (PI3KC3-C1 and -C2) are central to the initiation of macroautophagy and endosomal maturation, respectively. Through three-dimensional classification of a large cryo-EM dataset of human PI3KC3-C1 bound to the small GTPase RAB1A, we were able to map the structural pathway of enzyme activation. The inactive conformation is stabilized by an N-myristoyl modification of the pseudokinase (PK) subunit VPS15. The N-myristate is sequestered in the N-lobe of the VPS15 PK domain, which stabilizes a series of interactions whereby VPS15 sequesters and blocks the catalytic and membrane binding units of the VPS34 lipid kinase. In the activated conformation, the N-myristate and the VPS34 lipid kinase domain are liberated to interact with membranes and catalyze PI3P formation. The VPS15 PK domain contains a unique Arg at the gatekeeper position and binds tightly to GTP. GTP binding structurally stabilizes the N-myristate \"in\" conformation, which promotes the inactive conformation. This pathway provides a general mechanism for PI3KC3 activation in autophagy and endosome biogenesis and a roadmap for their pharmacological upregulation.\nIdentification and structural characterization of small-molecule inhibitors of PINK1\nAuthors: Rasool, S.; Shomali, T.; Truong, L.; Croteau, N.; Veyron, S.; Trempe, J.-F.\nScore: 7.0, Published: 2023-10-02 DOI: 10.1101/2023.10.02.560389\nMutations in PTEN-induced putative kinase 1 (PINK1) cause early-onset autosomal recessive Parkinson's Disease (PD). PINK1 is a Ser/Thr protein kinase which functions as a mitochondrial damage sensor and initiates mitochondrial quality control by accumulating on the damaged organelle. There, it will autophosphorylate and then phosphorylate ubiquitin chains, which in turn will recruit and activate Parkin, and E3 ubiquitin ligase also implicated in PD. Ubiquitylation of mitochondrial proteins leads to the autophagic degradation of the damaged organelle. Pharmacological modulation of PINK1 constitutes an appealing avenue to study its physiological function and develop PD therapeutics. In this study, we used a thermal shift assay to identify small-molecule inhibitors of PINK1. In vitro kinase activity assays demonstrate that these molecules are ATP competitive inhibitors that block ubiquitin phosphorylation. PRT062607 (a SYK inhibitor) is the most potent inhibitor of PINK1 in our screen and has an IC50 of 2 M against insect PINK1 and 1 M in HeLa cells expressing human PINK1. The crystal structures of PINK1 in complex with PRT062607 or CYC116 reveal how the compounds interact with the ATP-binding pocket. PRT062607 notably engages with the catalytic aspartate (type-1 inhibition) and causes a destabilization of insert-2 at the autophosphorylation dimer interface. Our findings provide a scaffold for the development of more selective and potent inhibitors of PINK1 that can be used as chemical probes.\nCharacterising the Protein-Protein Interaction Between MDM2 and 14-3-3σ; Proof of Concept for Small Molecule Stabilisation\nAuthors: Ward, J. A.; Romartinez-Alonso, B.; Kay, D. F.; Bellamy-Carter, J.; Thurairajah, B.; Basran, J.; Kwon, H.; Leney, A. C.; Macip, S.; Roversi, P.; Muskett, F. W.; Doveston, R. G.\nScore: 6.9, Published: 2023-09-26 DOI: 10.1101/2023.09.26.559467\nMouse Double Minute 2 (MDM2) is a key negative regulator of the tumour suppressor protein p53. MDM2 overexpression occurs in many types of cancer and results in the suppression of wild type p53. The 14-3-3 family of adaptor proteins are known to bind to MDM2 and the 14-3-3{sigma} isoform controls MDM2 cellular localisation and stability to inhibit its activity. Therefore, small molecule stabilisation of the 14-3-3{sigma}/MDM2 protein-protein interaction (PPI) is a potential therapeutic strategy for the treatment of cancer. In this work we provide a detailed biophysical and structural characterisation of the phosphorylation-dependent interaction between 14-3-3{sigma} and peptides that mimic the 14-3-3 binding motifs within MDM2. The data show that di-phosphorylation of MDM2 at S166 and S186 is essential for high affinity 14-3-3 binding and that the binary complex formed involves one MDM2 di-phosphorylated peptide bound to a dimer of 14-3-3{sigma}. Each of the two phosphorylated stretches of MDM2 occupies one of the two binding grooves of a 14-3-3{sigma} dimer, a novel model for binding of di-phosphorylated peptides to 14-3-3 proteins. In addition, we show that the 14-3-3{sigma}/MDM2 interaction is amenable to small molecule stabilisation. The natural product fusicoccin A forms a ternary complex with a 14-3-3{sigma} dimer and an MDM2 di-phosphorylated peptide resulting in stablisation of the 14-3-3{sigma}/MDM2 PPI. This work serves as a proof-of-concept of the drugability of the 14-3-3/MDM2 PPI and paves the way toward the development of more selective and efficacious small molecule stabilisers.\nDeep mutational scanning of Pneumocystis jirovecii dihydrofolate reductase reveals allosteric mechanism of resistance to an antifolate\nAuthors: Rouleau, F. D.; Dube, A. K.; Gagnon-Arsenault, I.; Dibyachintan, S.; Pageau, A.; Despres, P. C.; Lagüe, P.; Landry, C. R.\nScore: 5.3, Published: 2023-09-28 DOI: 10.1101/2023.09.27.559786\nPneumocystis jirovecii is a fungal pathogen that causes pneumocystis pneumonia, a disease that mainly affects immunocompromised individuals. This fungus has historically been hard to study because of our inability to grow it in vitro. One of the main drug targets in P. jirovecii is its dihydrofolate reductase (PjDHFR). Here, by using functional complementation of the bakers yeast ortholog, we show that PjDHFR can be inhibited by the antifolate methotrexate in a dose-dependent manner. Using deep mutational scanning of PjDHFR, we identify mutations conferring resistance to methotrexate. Thirty-one sites spanning the protein have at least one mutation that leads to resistance, for a total of 355 high-confidence resistance mutations. Most resistance-inducing mutations are found inside the active site, and many are structurally equivalent to mutations known to lead to resistance to different antifolates in other organisms. Some sites show specific resistance mutations, where only a single substitution confers resistance, whereas others are more permissive, as several substitutions at these sites confer resistance. Surprisingly, one of the permissive sites (F199) is without direct contact to either ligand or cofactor, suggesting that it acts through an allosteric mechanism. Modeling changes in binding energy between F199 mutants and drug shows that most mutations destabilize interactions between the protein and the drug. This evidence points towards a more important role of this position in resistance than previously estimated and highlights potential unknown allosteric mechanisms of resistance to antifolate in DHFRs. Our results offer unprecedented resources for the interpretation of mutation effects in the main drug target of an uncultivable fungal pathogen.\nBroad receptor tropism and immunogenicity of a clade 3 sarbecovirus\nAuthors: Lee, J.; Zepeda, S.; Park, Y.-J.; Taylor, A. L.; Quispe, J.; Stewart, C.; Leaf, E. M.; Treichel, C.; Corti, D.; King, N. P.; Starr, T. N.; Veesler, D.\nScore: 78.8, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557371\nAlthough Rhinolophus bats harbor diverse clade 3 sarbecoviruses, the structural determinants of receptor tropism along with the antigenicity of their spike (S) glycoproteins remain uncharacterized. Here, we show that the African Rinolophus bat clade 3 sarbecovirus PRD-0038 S has a broad ACE2 usage and that RBD mutations further expand receptor promiscuity and enable human ACE2 utilization. We determined a cryoEM structure of the PRD-0038 RBD bound to R. alcyone ACE2, explaining receptor tropism and highlighting differences with SARS-CoV-1 and SARS-CoV-2. Characterization of PRD-0038 S using cryoEM and monoclonal antibody reactivity revealed its distinct antigenicity relative to SARS-CoV-2 and identified PRD-0038 cross-neutralizing antibodies for pandemic preparedness. PRD-0038 S vaccination elicited greater titers of antibodies cross-reacting with vaccine-mismatched clade 2 and clade 1a sarbecoviruses compared to SARS-CoV-2 S due to broader antigenic targeting, motivating the inclusion of clade 3 antigens in next-generation vaccines for enhanced resilience to viral evolution.\n",
  "wordCount" : "2331",
  "inLanguage": "en",
  "datePublished": "2023-10-04T10:39:33Z",
  "dateModified": "2023-10-04T10:39:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta"><span>updated on October 4, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.558399">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.558399" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.558399">
        <p class="paperTitle">PROxAb Shuttle: A non-covalent plug-and-play platform for the rapid generation of tumor-targeting antibody-PROTAC conjugates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.558399" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.558399" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schneider, H.; Jaeger, S.; Koenning, D.; Rasche, N.; Schroeter, C.; Elter, D.; Evers, A.; Lecomte, M.; Riccardi Sirtori, F.; Schwarz, D.; Wegener, A.; Hartung, I.; Rieker, M.</p>
        <p class="info">Score: 21.5, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.558399' target='https://doi.org/10.1101/2023.09.29.558399'> 10.1101/2023.09.29.558399</a></p>
        <p class="abstract">Proteolysis-targeting chimeras (PROTACs) have evolved in recent years from an academic idea to a therapeutic modality with more than 25 active clinical programs. However, achieving oral bioavailability and cell-type specificity remains a challenge, especially for PROTACs recruiting the von Hippel-Lindau (VHL) E3 ligase. Herein, we present an unprecedented, plug-and-play platform for VHL-recruiting PROTACs to overcome these limitations. Our platform allows for the generation of non-covalent antibody-PROTAC complexes within minutes and obviates the need for prior PROTAC modification, antibody-drug linker chemistry optimization or bioconjugation. Our technology relies on camelid-derived antibody domains (VHHs) which can easily be engineered into existing therapeutic antibody scaffolds. The resulting targeted, bispecific fusion proteins can be complexed with PROTACs at defined PROTAC-to-antibody ratios and have been termed PROxAb Shuttles. PROxAb Shuttles can prolong the half-life of PROTACs from hours to days, demonstrate anti-tumor efficacy in vivo and have the potential for reloading in vivo to further boost efficacy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.559826">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.559826" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.559826">
        <p class="paperTitle">Mapping the biochemical landscape of rubisco</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.559826" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.559826" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Prywes, N.; Phillips, N. R.; Oltrogge, L. M.; de Pins, B.; Cowan, A. E.; Taylor-Kearney, L. J.; Chang, H. A.; Hall, L. N.; Bhatt, A.; Shih, P.; Milo, R.; Savage, D. F.</p>
        <p class="info">Score: 21.1, Published: 2023-09-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.559826' target='https://doi.org/10.1101/2023.09.27.559826'> 10.1101/2023.09.27.559826</a></p>
        <p class="abstract">The enzyme rubisco catalyzes the first step of carbon assimilation in photosynthesis. Despite the massive flux of CO2 passing through this active site over billions of years, extant rubisco has relatively slow kinetics and is prone to off-target activity. In many growth regimes, this limits photosynthesis in planta. Many attempts have been made to improve the kinetic parameters of rubisco with limited success, potentially due to biochemical trade-offs. To understand the structural basis of constraints on rubisco, a comprehensive map of rubisco at the individual amino-acid level is needed. To that end we performed a deep mutational scan using a rubisco-dependent E. coli strain. By titrating CO2 concentrations it was possible to determine estimations for both catalytic rate (kcat) and substrate affinity (KM) of &gt;99% of rubisco point mutants. Some positions were found to act as &#34;rheostats&#34; where some amino-acid substitutions reduced -while others improved -affinity for CO2. No individual point mutation was found to substantially improve the catalytic rate, but a number of highly phylogenetically conserved positions were found to tolerate mutations, indicating that a large portion of rubiscos sequence space remains unexplored by nature and may serve as a resource for future protein engineering efforts. Together, these biochemical measurements inform our understanding of biochemical tradeoffs and will assist in future efforts to improve rubisco catalytic properties.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559391">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559391" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559391">
        <p class="paperTitle">Using a function-first &#34;scout fragment&#34;-based approach to develop allosteric covalent inhibitors of conformationally dynamic helicase mechanoenzymes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559391" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559391" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ramsey, J. R.; Shelton, P. M. M.; Heiss, T. K.; Olinares, P. D. B.; Vostal, L. E.; Soileau, H.; Grasso, M.; Warrington, S.; Adaniya, S.; Miller, M.; Sun, S.; Huggins, D. J.; Myers, R. W.; Chait, B. T.; Vinogradova, E. V.; Kapoor, T. M.</p>
        <p class="info">Score: 16.4, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559391' target='https://doi.org/10.1101/2023.09.25.559391'> 10.1101/2023.09.25.559391</a></p>
        <p class="abstract">Helicases, classified into six superfamilies, are mechanoenzymes that utilize energy derived from ATP hydrolysis to remodel DNA and RNA substrates. These enzymes have key roles in diverse cellular processes, such as genome replication and maintenance, ribosome assembly and translation. Helicases with essential functions only in certain cancer cells have been identified and helicases expressed by certain viruses are required for their pathogenicity. As a result, helicases are important targets for chemical probes and therapeutics. However, it has been very challenging to develop selective chemical inhibitors for helicases, enzymes with highly dynamic conformations. We envisioned that electrophilic  scout fragments, which have been used for chemical proteomic based profiling, could be leveraged to develop covalent inhibitors of helicases. We adopted a function-first approach, combining enzymatic assays with enantiomeric probe pairs and mass spectrometry, to develop a covalent inhibitor that selectively targets an allosteric site in SARS-CoV-2 nsp13, a superfamily-1 helicase. Further, we demonstrate that scout fragments inhibit the activity of two human superfamily-2 helicases, BLM and WRN, involved in genome maintenance. Together, our findings suggest a covalent inhibitor discovery approach to target helicases and potentially other conformationally dynamic mechanoenzymes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.28.560059">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.28.560059" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.28.560059">
        <p class="paperTitle">Transcriptome-wide RNA structure probing with temporal resolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.28.560059" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.28.560059" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, D.; Tang, Y.; Li, A.; Sun, L.; Kleiner, R. E.</p>
        <p class="info">Score: 15.5, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.28.560059' target='https://doi.org/10.1101/2023.09.28.560059'> 10.1101/2023.09.28.560059</a></p>
        <p class="abstract">Reactive small-molecule probes are widely used for RNA structure probing, however current approaches largely measure average RNA transcript dynamics and do not resolve structural differences that occur during folding or transcript maturation. Here, we present SNIPER-seq, an RNA structure probing method relying upon metabolic labeling with 2&#39;-aminodeoxycytidine, structure-dependent 2&#39;-amino reaction with an aromatic isothiocyanate, and high-throughput RNA sequencing. Our method maps cellular RNA structure transcriptome-wide with temporal resolution enabling determination of transcript age-dependent RNA structural dynamics. We benchmark our approach against known RNA structures and investigate the dynamics of human 5S rRNA during ribosome biogenesis, revealing specific structural changes in 5S rRNA loops that occur over the course of several hours. Taken together, our work sheds light on the maturation and coordinated conformational changes that take place during ribosome biogenesis and provides a general strategy for surveying evolving RNA structural dynamics across the transcriptome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.560713">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.560713" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.560713">
        <p class="paperTitle">Improving protein expression, stability, and function with ProteinMPNN</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.560713" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.560713" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sumida, K. H.; Nunez Franco, R.; Kalvet, I.; Pellock, S. J.; Wicky, B. I. M.; Milles, L. F.; Dauparas, J.; Wang, J.; Kipnis, Y.; Jameson, N.; Kang, A.; De La Cruz, J.; Sankaran, B.; Bera, A. K.; Jimenez Oses, G.; Baker, D.</p>
        <p class="info">Score: 13.4, Published: 2023-10-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.560713' target='https://doi.org/10.1101/2023.10.03.560713'> 10.1101/2023.10.03.560713</a></p>
        <p class="abstract">Natural proteins are highly optimized for function, but are often difficult to produce at a scale suitable for biotechnological applications due to poor expression in heterologous systems, limited solubility, and sensitivity to temperature. Thus, a general method that improves the physical properties of native proteins while maintaining function could have wide utility for protein-based technologies. Here we show that the deep neural network ProteinMPNN together with evolutionary and structural information provides a route to increasing protein expression, stability, and function. For both myoglobin and tobacco etch virus (TEV) protease, we generated designs with improved expression, elevated melting temperatures, and improved function. For TEV protease, we identified multiple designs with improved catalytic activity as compared to the parent sequence and previously reported TEV variants. Our approach should be broadly useful for improving the expression, stability, and function of biotechnologically important proteins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.28.559894">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.28.559894" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.28.559894">
        <p class="paperTitle">Structural pathway for class III PI 3-kinase activation by the myristoylated GTP-binding pseudokinase VPS15</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.28.559894" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.28.559894" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cook, A. S. I.; Chen, M. H.; Ren, X.; Rao, S.; Garcia, S. N.; Claveras Cabezudo, A.; Iavarone, A. T.; Hummer, G.; Hurley, J. H.</p>
        <p class="info">Score: 11.0, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.28.559894' target='https://doi.org/10.1101/2023.09.28.559894'> 10.1101/2023.09.28.559894</a></p>
        <p class="abstract">The class III phosphatidylinositol (PI) 3-kinase complexes I and II (PI3KC3-C1 and -C2) are central to the initiation of macroautophagy and endosomal maturation, respectively. Through three-dimensional classification of a large cryo-EM dataset of human PI3KC3-C1 bound to the small GTPase RAB1A, we were able to map the structural pathway of enzyme activation. The inactive conformation is stabilized by an N-myristoyl modification of the pseudokinase (PK) subunit VPS15. The N-myristate is sequestered in the N-lobe of the VPS15 PK domain, which stabilizes a series of interactions whereby VPS15 sequesters and blocks the catalytic and membrane binding units of the VPS34 lipid kinase. In the activated conformation, the N-myristate and the VPS34 lipid kinase domain are liberated to interact with membranes and catalyze PI3P formation. The VPS15 PK domain contains a unique Arg at the gatekeeper position and binds tightly to GTP. GTP binding structurally stabilizes the N-myristate &#34;in&#34; conformation, which promotes the inactive conformation. This pathway provides a general mechanism for PI3KC3 activation in autophagy and endosome biogenesis and a roadmap for their pharmacological upregulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560389">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560389" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560389">
        <p class="paperTitle">Identification and structural characterization of small-molecule inhibitors of PINK1</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560389" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560389" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rasool, S.; Shomali, T.; Truong, L.; Croteau, N.; Veyron, S.; Trempe, J.-F.</p>
        <p class="info">Score: 7.0, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560389' target='https://doi.org/10.1101/2023.10.02.560389'> 10.1101/2023.10.02.560389</a></p>
        <p class="abstract">Mutations in PTEN-induced putative kinase 1 (PINK1) cause early-onset autosomal recessive Parkinson&#39;s Disease (PD). PINK1 is a Ser/Thr protein kinase which functions as a mitochondrial damage sensor and initiates mitochondrial quality control by accumulating on the damaged organelle. There, it will autophosphorylate and then phosphorylate ubiquitin chains, which in turn will recruit and activate Parkin, and E3 ubiquitin ligase also implicated in PD. Ubiquitylation of mitochondrial proteins leads to the autophagic degradation of the damaged organelle. Pharmacological modulation of PINK1 constitutes an appealing avenue to study its physiological function and develop PD therapeutics. In this study, we used a thermal shift assay to identify small-molecule inhibitors of PINK1. In vitro kinase activity assays demonstrate that these molecules are ATP competitive inhibitors that block ubiquitin phosphorylation. PRT062607 (a SYK inhibitor) is the most potent inhibitor of PINK1 in our screen and has an IC50 of 2 M against insect PINK1 and 1 M in HeLa cells expressing human PINK1. The crystal structures of PINK1 in complex with PRT062607 or CYC116 reveal how the compounds interact with the ATP-binding pocket. PRT062607 notably engages with the catalytic aspartate (type-1 inhibition) and causes a destabilization of insert-2 at the autophosphorylation dimer interface. Our findings provide a scaffold for the development of more selective and potent inhibitors of PINK1 that can be used as chemical probes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559467">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559467" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559467">
        <p class="paperTitle">Characterising the Protein-Protein Interaction Between MDM2 and 14-3-3σ; Proof of Concept for Small Molecule Stabilisation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559467" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559467" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ward, J. A.; Romartinez-Alonso, B.; Kay, D. F.; Bellamy-Carter, J.; Thurairajah, B.; Basran, J.; Kwon, H.; Leney, A. C.; Macip, S.; Roversi, P.; Muskett, F. W.; Doveston, R. G.</p>
        <p class="info">Score: 6.9, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559467' target='https://doi.org/10.1101/2023.09.26.559467'> 10.1101/2023.09.26.559467</a></p>
        <p class="abstract">Mouse Double Minute 2 (MDM2) is a key negative regulator of the tumour suppressor protein p53. MDM2 overexpression occurs in many types of cancer and results in the suppression of wild type p53. The 14-3-3 family of adaptor proteins are known to bind to MDM2 and the 14-3-3{sigma} isoform controls MDM2 cellular localisation and stability to inhibit its activity. Therefore, small molecule stabilisation of the 14-3-3{sigma}/MDM2 protein-protein interaction (PPI) is a potential therapeutic strategy for the treatment of cancer. In this work we provide a detailed biophysical and structural characterisation of the phosphorylation-dependent interaction between 14-3-3{sigma} and peptides that mimic the 14-3-3 binding motifs within MDM2. The data show that di-phosphorylation of MDM2 at S166 and S186 is essential for high affinity 14-3-3 binding and that the binary complex formed involves one MDM2 di-phosphorylated peptide bound to a dimer of 14-3-3{sigma}. Each of the two phosphorylated stretches of MDM2 occupies one of the two binding grooves of a 14-3-3{sigma} dimer, a novel model for binding of di-phosphorylated peptides to 14-3-3 proteins. In addition, we show that the 14-3-3{sigma}/MDM2 interaction is amenable to small molecule stabilisation. The natural product fusicoccin A forms a ternary complex with a 14-3-3{sigma} dimer and an MDM2 di-phosphorylated peptide resulting in stablisation of the 14-3-3{sigma}/MDM2 PPI. This work serves as a proof-of-concept of the drugability of the 14-3-3/MDM2 PPI and paves the way toward the development of more selective and efficacious small molecule stabilisers.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.559786">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.559786" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.559786">
        <p class="paperTitle">Deep mutational scanning of Pneumocystis jirovecii dihydrofolate reductase reveals allosteric mechanism of resistance to an antifolate</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.559786" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.559786" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rouleau, F. D.; Dube, A. K.; Gagnon-Arsenault, I.; Dibyachintan, S.; Pageau, A.; Despres, P. C.; Lagüe, P.; Landry, C. R.</p>
        <p class="info">Score: 5.3, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.559786' target='https://doi.org/10.1101/2023.09.27.559786'> 10.1101/2023.09.27.559786</a></p>
        <p class="abstract">Pneumocystis jirovecii is a fungal pathogen that causes pneumocystis pneumonia, a disease that mainly affects immunocompromised individuals. This fungus has historically been hard to study because of our inability to grow it in vitro. One of the main drug targets in P. jirovecii is its dihydrofolate reductase (PjDHFR). Here, by using functional complementation of the bakers yeast ortholog, we show that PjDHFR can be inhibited by the antifolate methotrexate in a dose-dependent manner. Using deep mutational scanning of PjDHFR, we identify mutations conferring resistance to methotrexate. Thirty-one sites spanning the protein have at least one mutation that leads to resistance, for a total of 355 high-confidence resistance mutations. Most resistance-inducing mutations are found inside the active site, and many are structurally equivalent to mutations known to lead to resistance to different antifolates in other organisms. Some sites show specific resistance mutations, where only a single substitution confers resistance, whereas others are more permissive, as several substitutions at these sites confer resistance. Surprisingly, one of the permissive sites (F199) is without direct contact to either ligand or cofactor, suggesting that it acts through an allosteric mechanism. Modeling changes in binding energy between F199 mutants and drug shows that most mutations destabilize interactions between the protein and the drug. This evidence points towards a more important role of this position in resistance than previously estimated and highlights potential unknown allosteric mechanisms of resistance to antifolate in DHFRs. Our results offer unprecedented resources for the interpretation of mutation effects in the main drug target of an uncultivable fungal pathogen.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557371">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557371" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557371">
        <p class="paperTitle">Broad receptor tropism and immunogenicity of a clade 3 sarbecovirus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557371" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557371" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, J.; Zepeda, S.; Park, Y.-J.; Taylor, A. L.; Quispe, J.; Stewart, C.; Leaf, E. M.; Treichel, C.; Corti, D.; King, N. P.; Starr, T. N.; Veesler, D.</p>
        <p class="info">Score: 78.8, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557371' target='https://doi.org/10.1101/2023.09.12.557371'> 10.1101/2023.09.12.557371</a></p>
        <p class="abstract">Although Rhinolophus bats harbor diverse clade 3 sarbecoviruses, the structural determinants of receptor tropism along with the antigenicity of their spike (S) glycoproteins remain uncharacterized. Here, we show that the African Rinolophus bat clade 3 sarbecovirus PRD-0038 S has a broad ACE2 usage and that RBD mutations further expand receptor promiscuity and enable human ACE2 utilization. We determined a cryoEM structure of the PRD-0038 RBD bound to R. alcyone ACE2, explaining receptor tropism and highlighting differences with SARS-CoV-1 and SARS-CoV-2. Characterization of PRD-0038 S using cryoEM and monoclonal antibody reactivity revealed its distinct antigenicity relative to SARS-CoV-2 and identified PRD-0038 cross-neutralizing antibodies for pandemic preparedness. PRD-0038 S vaccination elicited greater titers of antibodies cross-reacting with vaccine-mismatched clade 2 and clade 1a sarbecoviruses compared to SARS-CoV-2 S due to broader antigenic targeting, motivating the inclusion of clade 3 antigens in next-generation vaccines for enhanced resilience to viral evolution.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
