<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Structure of the γ-tubulin ring complex-capped microtubule
Authors: Aher, A.; Urnavicius, L.; Xue, A.; Neselu, K.; Kapoor, T.
Score: 16.4, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567916
Microtubules are composed of /{beta}-tubulin dimers positioned head-to-tail to form protofilaments that associate laterally in varying numbers. It is not known how cellular microtubules assemble with the canonical 13-protofilament architecture, resulting in micrometer-scale /{beta}-tubulin tracks for intracellular transport that align with, rather than spiral along, the filaments long-axis.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Structure of the γ-tubulin ring complex-capped microtubule
Authors: Aher, A.; Urnavicius, L.; Xue, A.; Neselu, K.; Kapoor, T.
Score: 16.4, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567916
Microtubules are composed of /{beta}-tubulin dimers positioned head-to-tail to form protofilaments that associate laterally in varying numbers. It is not known how cellular microtubules assemble with the canonical 13-protofilament architecture, resulting in micrometer-scale /{beta}-tubulin tracks for intracellular transport that align with, rather than spiral along, the filaments long-axis." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-26T10:37:33+00:00" />
<meta property="article:modified_time" content="2023-11-26T10:37:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Structure of the γ-tubulin ring complex-capped microtubule
Authors: Aher, A.; Urnavicius, L.; Xue, A.; Neselu, K.; Kapoor, T.
Score: 16.4, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567916
Microtubules are composed of /{beta}-tubulin dimers positioned head-to-tail to form protofilaments that associate laterally in varying numbers. It is not known how cellular microtubules assemble with the canonical 13-protofilament architecture, resulting in micrometer-scale /{beta}-tubulin tracks for intracellular transport that align with, rather than spiral along, the filaments long-axis."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Structure of the γ-tubulin ring complex-capped microtubule\nAuthors: Aher, A.; Urnavicius, L.; Xue, A.; Neselu, K.; Kapoor, T.\nScore: 16.4, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567916\nMicrotubules are composed of /{beta}-tubulin dimers positioned head-to-tail to form protofilaments that associate laterally in varying numbers. It is not known how cellular microtubules assemble with the canonical 13-protofilament architecture, resulting in micrometer-scale /{beta}-tubulin tracks for intracellular transport that align with, rather than spiral along, the filaments long-axis.",
  "keywords": [
    
  ],
  "articleBody": " Structure of the γ-tubulin ring complex-capped microtubule\nAuthors: Aher, A.; Urnavicius, L.; Xue, A.; Neselu, K.; Kapoor, T.\nScore: 16.4, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567916\nMicrotubules are composed of /{beta}-tubulin dimers positioned head-to-tail to form protofilaments that associate laterally in varying numbers. It is not known how cellular microtubules assemble with the canonical 13-protofilament architecture, resulting in micrometer-scale /{beta}-tubulin tracks for intracellular transport that align with, rather than spiral along, the filaments long-axis. We report that the human [~]2.3MDa {gamma}-tubulin ring complex ({gamma}-TuRC), an essential regulator of microtubule formation that contains 14 {gamma}-tubulins, selectively nucleates 13-protofilament microtubules. Cryo-EM reconstructions of {gamma}-TuRC-capped microtubule minus-ends reveal the extensive intra- and inter-domain motions of {gamma}-TuRC subunits that accommodate its actin-containing luminal bridge and establish lateral and longitudinal interactions between {gamma}- and -tubulins. Our structures reveal how free {gamma}-TuRC, an inefficient nucleation template due to its splayed conformation, transforms into a stable cap that blocks addition or loss of /{beta}-tubulins from minus-ends and sets the lattice architecture of cellular microtubules. One Sentence SummaryStructural insights into how the {gamma}-tubulin ring complex nucleates and caps a 13-protofilament microtubule.\nFast and accurate disulfide bridge detection\nAuthors: Heissel, S.; He, Y.; Jankevics, A.; Shi, Y.; Molina, H.; Viner, R.; Scheltema, R. A.\nScore: 10.4, Published: 2023-11-22 DOI: 10.1101/2023.11.22.568205\nRecombinant expression of proteins, propelled by therapeutic antibodies, has evolved into a multi-billion-dollar industry. Essential here is quality control assessment of critical attributes such as sequence fidelity, proper folding, and post-translational modifications (PTMs). Errors can lead to diminished bioactivity and, in the context of therapeutic proteins, an elevated risk for immunogenicity. Over the years, many techniques were developed and applied to validate proteins in a standardized and high-throughput fashion. One parameter has, however, so far been challenging to assess. Disulfide bridges, covalent bonds linking two Cysteine residues, assist in the correct folding and stability of proteins and thus have a major influence on their efficacy. Mass spectrometry promises to be an optimal technique to uncover them in a fast and accurate fashion. In this work, we present a unique combination of sample preparation, data acquisition and analysis facilitating the rapid and accurate assessment of disulfide bridges in purified proteins. Through microwave-assisted acid hydrolysis (MAAH), the proteins are digested rapidly and artifact-free into peptides, with a substantial degree of overlap over the sequence. The nonspecific nature of this procedure, however, introduces chemical background which is efficiently removed by integrating ion mobility preceding the mass spectrometric measurement. The nonspecific nature of the digestion step additionally necessitates new developments in data analysis, for which we extended the XlinkX node in Proteome Discoverer (XlinkX/PD) to efficiently process the data and ensure correctness through effective false discovery rate correction. The entire workflow can be completed within one hour, allowing for high-throughput, high-accuracy disulfide mapping.\nStructure of the human 80S ribosome at 1.9 A resolution - the molecular role of chemical modifications and ions in RNA\nAuthors: Holvec, S.; Barchet, C.; Lechner, A.; Frechin, L.; De Silva, S. N. T.; Hazemann, I.; Wolff, P.; von Loeffelholz, O.; Klaholz, B. P.\nScore: 9.9, Published: 2023-11-24 DOI: 10.1101/2023.11.23.568509\nThe ribosomal RNA of the human protein synthesis machinery comprises numerous chemical modifications that are introduced during ribosome biogenesis. We present the 1.9 [A] resolution cryo-EM structure of the 80S human ribosome resolving numerous new rRNA modifications and functionally important ions such as Zn2+, K+ and Mg2+ including their associated individual water molecules. 2'-O-methylation, pseudo-uridine and base modifications were confirmed by mass spectrometry resulting in a complete investigation of the \u003e 230 sites many of which could not be addressed previously. They choreograph key interactions within the RNA and at the interface with proteins, including at the ribosomal subunit interfaces of the fully assembled 80S ribosome. Uridine isomerisation turns out to be a key mechanism for U-A base pair stabilisation in RNA in general. The structural environment of chemical modifications \u0026 ions is primordial for the RNA architecture of the mature human ribosome, hence providing a structural framework to address their role in healthy context and human diseases.\nMassively parallel mapping of substrate cleavage sites defines dipeptidyl peptidase four subsite cooperativity\nAuthors: Gudipati, R. K.; Gaidatzis, D.; Seebacher, J.; Muehlhaeusser, S.; Kempf, G.; Cavadini, S.; Hess, D.; Soneson, C.; Grosshans, H.\nScore: 7.8, Published: 2023-11-17 DOI: 10.1101/2023.11.17.567387\nSubstrate specificity determines protease functions in physiology and in clinical and biotechnological application. However, affordable and unbiased assays for large-scale quantification of substrate cleavage have been lacking. Here, we develop hiMAPS (high-throughput mapping of protease cleavage sites), a cheap, mass spectrometry-based assay, to derive cleavage motifs for human Dipeptidyl Peptidase Four (DPP4), a key regulator of blood glucose levels. We recapitulate the known benefit of proline in the penultimate (P1) position from the substrate N-terminus, extend it to additional residues, and identify and quantify combinatorial interactions of P1 with its neighbors. These findings reveal extensive cooperativity among the enzymes active site subsites and allow us to derive a sequence motif that predicts substrate turnover. We show that this information provides new opportunities to engineer stabilized versions of a key substrate of DPP4, the small peptide hormone GLP-1, whose derivatives are used for clinical treatment of type 2 diabetes and obesity. Finally, structural and biochemical characterization of a DPP4 homologue, C. elegans DPF-3, reveal specific subsite differences and a distinct cleavage motif, providing insight into the mechanistic basis of the observed specificity. Collectively, our findings present a broadly applicable framework to high-throughput mapping of protease cleavage sites, extend our understanding of protease specificity, and provide a chemical space for substrate engineering of a clinically relevant family of exopeptidases.\nA liquid-to-solid phase transition of biomolecular condensates drives in vivo formation of yeast amyloids and prions\nAuthors: Alieva, M. K.; Nikishina, S. B.; Kireev, I. I.; Golyshev, S. A.; Tyurin-Kuzmin, P. A.; Ivanova, L. V.; Alexandrov, A. I.; Kushnirov, V. V.; Dergalev, A. A.\nScore: 6.8, Published: 2023-11-18 DOI: 10.1101/2023.11.17.566816\nLiquid-liquid phase separation (LLPS) and liquid-solid phase transition (LSPT) of amyloidogenic proteins are now being intensively studied as a potentially widespread mechanism of pathological amyloids formation. However, the possibility and importance of such a mechanism in living systems is still questionable. Here, we investigated the possibility of such LSPT for a series of yeast prion proteins-based constructs overproduced in yeast cells lacking any pre-existing amyloid template. By combining fluorescence and electron microscopy with biochemical and genetic approaches, we have shown that three such constructs (containing the prion domains (PDs) of either Sup35, Rnq1 or Mot3 proteins) form amyloid fibrils via the intermediate stage of liquid-like condensates, that age over time into the more solid-like hydrogels and amyloid bodies. In turn, LSPT of these constructs triggers prion conversion of the corresponding wild-type protein. Two other constructs studied (Ure2- and Sap30-based) are unable to phase separate in vivo and their amyloidogenesis is therefore strongly suppressed. Using PrK-resistant amyloid core mapping, we showed that Sup35PD amyloids formed via LSPT have a different molecular architecture compared to those formed via amyloid cross-seeding. Finally, we showed that physiological LLPS of wild-type Sup35 protein can increase its prion conversion in yeast.\nSub-single-turnover quantification of enzyme catalysis at ultrahigh throughput via a versatile NAD(P)H coupled assay in microdroplets\nAuthors: Penner, M.; Klein, O. J.; Gantz, M.; Boss, S.; Barker, P.; Dupree, P.; Hollfelder, F.\nScore: 4.0, Published: 2023-11-22 DOI: 10.1101/2023.11.22.568356\nEnzyme engineering and discovery are crucial for a future sustainable bioeconomy, and harvesting new biocatalysts from large libraries through directed evolution or functional metagenomics requires accessible, rapid assays. Ultra-high throughput screening can often require an optical readout, leading to the use of model substrates that may not accurately report on activity for the target reaction and may require bespoke synthesis. In contrast, coupled assays represent a modular plug-and-play system, where any pairing of enzyme/substrate may be investigated, if the reaction can produce a common intermediate which links the catalytic reaction to a detection cascade readout. Here we establish a detection cascade, producing a fluorescent readout in response to NAD(P)H via glutathione reductase and a subsequent thiol-mediated uncaging reaction, with a 30 nM detection limit. We demonstrate its utility for the glycosidase AxyAgu115A (producing monosaccharides from a natural biofuel feedstock) and report a three orders of magnitude improved sensitivity compared to absorbance-based systems, so that less than one catalytic turnover per enzyme molecule expressed from a single cell is detectable. These advantages are brought to bear in plate formats, but also in picoliter emulsion droplets, where enrichments of 950-fold suggest that large libraries can be interrogated against a specific query substrate.\nFluorescent human RPA to track assembly dynamics on DNA\nAuthors: Kaushik, V.; Chadda, R.; Kuppa, S.; Pokhrel, N.; Vayyeti, A.; Grady, S.; Arnatt, C.; Antony, E.\nScore: 3.9, Published: 2023-11-23 DOI: 10.1101/2023.11.23.568455\nDNA metabolic processes including replication, repair, recombination, and telomere maintenance occur on single-stranded DNA (ssDNA). In each of these complex processes, dozens of proteins function together on the ssDNA template. However, when double-stranded DNA is unwound, the transiently open ssDNA is protected and coated by the high affinity heterotrimeric ssDNA binding Replication Protein A (RPA). Almost all downstream DNA processes must first remodel/remove RPA or function alongside to access the ssDNA occluded under RPA. Formation of RPA-ssDNA complexes trigger the DNA damage checkpoint response and is a key step in activating most DNA repair and recombination pathways. Thus, in addition to protecting the exposed ssDNA, RPA functions as a gatekeeper to define functional specificity in DNA maintenance and genomic integrity. RPA achieves functional dexterity through a multi-domain architecture utilizing several DNA binding and protein-interaction domains connected by flexible linkers. This flexible and modular architecture enables RPA to adopt a myriad of configurations tailored for specific DNA metabolic roles. To experimentally capture the dynamics of the domains of RPA upon binding to ssDNA and interacting proteins we here describe the generation of active site-specific fluorescent versions of human RPA (RPA) using 4-azido-L-phenylalanine (4AZP) incorporation and click chemistry. This approach can also be applied to site-specific modifications of other multi-domain proteins. Fluorescence-enhancement through non-canonical amino acids (FEncAA) and Forster Resonance Energy Transfer (FRET) assays for measuring dynamics of RPA on DNA are also described.\nExpanding the GPCR-RAMP interactome\nAuthors: Kotliar, I. B.; Bendes, A.; Dahl, L.; Chen, Y.; Saarinen, M.; Ceraudo, E.; Dodig, T.; Uhlen, M.; Svenningsson, P.; Schwenk, J.; Sakmar, T.\nScore: 3.9, Published: 2023-11-23 DOI: 10.1101/2023.11.22.568247\nReceptor activity-modifying proteins (RAMPs) can form complexes with G protein-coupled receptors (GPCRs) and regulate their cellular trafficking and pharmacology. RAMP interactions have been identified for about 50 GPCRs, but only a few GPCR-RAMP complexes have been studied in detail. To elucidate a complete interactome between GPCRs and the three RAMPs, we developed a customized library of 215 Dual Epitope-Tagged (DuET) GPCRs representing all GPCR subfamilies. Using a multiplexed suspension bead array (SBA) assay, we identified 122 GPCRs that showed strong evidence for interaction with at least one RAMP. We screened for native interactions in three cell lines and found 23 GPCRs that formed complexes with RAMPs. Mapping the GPCR-RAMP interactome expands the current system-wide functional characterization of RAMP-interacting GPCRs to inform the design of selective GPCR-targeted therapeutics.\nDevelopment of supramolecular anticoagulants with on-demand reversibility\nAuthors: Dockerill, M.; Ford, D. J.; Angerani, S.; Alwis, I.; Ripoll-Rozada, J.; Smythe, R. E.; Liu, J. S. T.; Barbosa Pereira, P. J.; Jackson, S. P.; Payne, R. J.; Winssinger, N.\nScore: 5.9, Published: 2023-11-12 DOI: 10.1101/2023.11.12.566735\nDrugs are administered at a dosing schedule set by their therapeutic index and termination of action is achieved by clearance and metabolism of the drug (hours to days for small molecules, weeks to months for biologics). In some cases, it is important to achieve a fast reversal of the drugs action to overcome dangerous side effects or in response to unforeseen events. A case in point is for anticoagulant drugs. Here we report a general strategy to achieve on-demand reversibility by leveraging supramolecular assembly of drug fragments and showcase the approach with thrombin-inhibiting anticoagulants. In our supramolecular drug design, the action of the drug is reinforced by a dynamic hybridisation of peptide nucleic acids (PNAs) between drug fragments. We show that this design enables the generation of very potent bivalent direct thrombin inhibitors (Ki 74 pM) and this inhibition can be reversed through the use of a PNA antidote. We demonstrate that these supramolecular inhibitors exhibit potent anticoagulant activity in vitro and in vivo and that this activity can also be reversed on demand.\nCholesterol regulates plasma membrane bending by prominin-family proteins\nAuthors: Bell, T. A.; Luce, B. E.; Hakim, P.; Dardari, H.; Ananda, V. Y.; Nguyen, T. H.; Monshizadeh, A.; Chao, L. H.\nScore: 7.1, Published: 2023-11-10 DOI: 10.1101/2023.11.08.566258\nProminin-1 (Prom1) is a pentaspan membrane protein that associates with curved regions of the plasma membrane. Prom1 localizes to cholesterol-rich domains and requires membrane cholesterol to support membrane remodeling. Membrane bending activity is particularly evident in photoreceptors, where Prom1 mutations cause loss of outer segment disk homeostasis leading to cone-rod retinal dystrophy (CCRD). However, the mechanistic link between prominin-dependent cholesterol binding, membrane remodeling, and retinal disease remains unclear. Here, we characterize the membrane bending function and specific cholesterol binding activity of Prom1 and its proposed homolog Tweety homology 1 (Ttyh1) in extracellular vesicles (EVs). Prom1 and Ttyh1 induce formation of EVs in cultured mammalian cells that are biophysically similar. Though both proteins bend membranes and form EVs at the plasma membrane, Ttyh1 lacks a stable interaction with cholesterol that is present in Prom1. Correspondingly, Ttyh1 forms EVs that are more deformed than those produced by Prom1. An evolutionarily conserved and retinal disease-associated Prom1 residue (Trp-795) is necessary for cholesterol binding, EV membrane deformation, and efficient trafficking to the plasma membrane. Removal of N-glycan moieties from Prom1 biases the enzyme toward a cholesterol-bound state. We propose that Prom1 and Ttyh1 are both members of a single prominin family of membrane bending proteins, that Ttyh1 is a constitutively active member of this family, and that Prom1 is regulated by cholesterol binding and N-glycosylation. These findings shed light on mechanisms of prominin family function in disease and help unify models of prominin function across diverse cell types. Significance StatementMammalian cells dynamically shape the plasma membrane to achieve specialized functions. Prominin-1 (Prom1) is a membrane protein that promotes bending and shaping of the membrane, notably in photoreceptors. Here, the authors find that membrane cholesterol regulates the membrane bending activity of Prom1 in extracellular vesicles. Membrane bending can be tuned by altering cholesterol levels and may be coupled to different Prom1 N-glycosylation states. A distant homolog of Prom1 called Ttyh1 is also shown to form extracellular vesicles and supports more membrane bending than Prom1 though it does not bind cholesterol. This work unites the Prom and Ttyh protein families into a single functional group and helps explain how membrane cholesterol functionalizes prominin-family proteins for different roles in different cell types.\n",
  "wordCount" : "2450",
  "inLanguage": "en",
  "datePublished": "2023-11-26T10:37:33Z",
  "dateModified": "2023-11-26T10:37:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 26, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567916">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567916" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567916">
        <p class="paperTitle">Structure of the γ-tubulin ring complex-capped microtubule</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567916" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567916" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aher, A.; Urnavicius, L.; Xue, A.; Neselu, K.; Kapoor, T.</p>
        <p class="info">Score: 16.4, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567916' target='https://doi.org/10.1101/2023.11.20.567916'> 10.1101/2023.11.20.567916</a></p>
        <p class="abstract">Microtubules are composed of /{beta}-tubulin dimers positioned head-to-tail to form protofilaments that associate laterally in varying numbers. It is not known how cellular microtubules assemble with the canonical 13-protofilament architecture, resulting in micrometer-scale /{beta}-tubulin tracks for intracellular transport that align with, rather than spiral along, the filaments long-axis. We report that the human [~]2.3MDa {gamma}-tubulin ring complex ({gamma}-TuRC), an essential regulator of microtubule formation that contains 14 {gamma}-tubulins, selectively nucleates 13-protofilament microtubules. Cryo-EM reconstructions of {gamma}-TuRC-capped microtubule minus-ends reveal the extensive intra- and inter-domain motions of {gamma}-TuRC subunits that accommodate its actin-containing luminal bridge and establish lateral and longitudinal interactions between {gamma}- and -tubulins. Our structures reveal how free {gamma}-TuRC, an inefficient nucleation template due to its splayed conformation, transforms into a stable cap that blocks addition or loss of /{beta}-tubulins from minus-ends and sets the lattice architecture of cellular microtubules.

One Sentence SummaryStructural insights into how the {gamma}-tubulin ring complex nucleates and caps a 13-protofilament microtubule.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.568205">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.568205" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.568205">
        <p class="paperTitle">Fast and accurate disulfide bridge detection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.568205" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.568205" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Heissel, S.; He, Y.; Jankevics, A.; Shi, Y.; Molina, H.; Viner, R.; Scheltema, R. A.</p>
        <p class="info">Score: 10.4, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.568205' target='https://doi.org/10.1101/2023.11.22.568205'> 10.1101/2023.11.22.568205</a></p>
        <p class="abstract">Recombinant expression of proteins, propelled by therapeutic antibodies, has evolved into a multi-billion-dollar industry. Essential here is quality control assessment of critical attributes such as sequence fidelity, proper folding, and post-translational modifications (PTMs). Errors can lead to diminished bioactivity and, in the context of therapeutic proteins, an elevated risk for immunogenicity. Over the years, many techniques were developed and applied to validate proteins in a standardized and high-throughput fashion. One parameter has, however, so far been challenging to assess. Disulfide bridges, covalent bonds linking two Cysteine residues, assist in the correct folding and stability of proteins and thus have a major influence on their efficacy. Mass spectrometry promises to be an optimal technique to uncover them in a fast and accurate fashion. In this work, we present a unique combination of sample preparation, data acquisition and analysis facilitating the rapid and accurate assessment of disulfide bridges in purified proteins. Through microwave-assisted acid hydrolysis (MAAH), the proteins are digested rapidly and artifact-free into peptides, with a substantial degree of overlap over the sequence. The nonspecific nature of this procedure, however, introduces chemical background which is efficiently removed by integrating ion mobility preceding the mass spectrometric measurement. The nonspecific nature of the digestion step additionally necessitates new developments in data analysis, for which we extended the XlinkX node in Proteome Discoverer (XlinkX/PD) to efficiently process the data and ensure correctness through effective false discovery rate correction. The entire workflow can be completed within one hour, allowing for high-throughput, high-accuracy disulfide mapping.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.568509">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.568509" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.568509">
        <p class="paperTitle">Structure of the human 80S ribosome at 1.9 A resolution - the molecular role of chemical modifications and ions in RNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.568509" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.568509" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Holvec, S.; Barchet, C.; Lechner, A.; Frechin, L.; De Silva, S. N. T.; Hazemann, I.; Wolff, P.; von Loeffelholz, O.; Klaholz, B. P.</p>
        <p class="info">Score: 9.9, Published: 2023-11-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.568509' target='https://doi.org/10.1101/2023.11.23.568509'> 10.1101/2023.11.23.568509</a></p>
        <p class="abstract">The ribosomal RNA of the human protein synthesis machinery comprises numerous chemical modifications that are introduced during ribosome biogenesis. We present the 1.9 [A] resolution cryo-EM structure of the 80S human ribosome resolving numerous new rRNA modifications and functionally important ions such as Zn2&#43;, K&#43; and Mg2&#43; including their associated individual water molecules. 2&#39;-O-methylation, pseudo-uridine and base modifications were confirmed by mass spectrometry resulting in a complete investigation of the &gt; 230 sites many of which could not be addressed previously. They choreograph key interactions within the RNA and at the interface with proteins, including at the ribosomal subunit interfaces of the fully assembled 80S ribosome. Uridine isomerisation turns out to be a key mechanism for U-A base pair stabilisation in RNA in general. The structural environment of chemical modifications &amp; ions is primordial for the RNA architecture of the mature human ribosome, hence providing a structural framework to address their role in healthy context and human diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.17.567387">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.17.567387" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.17.567387">
        <p class="paperTitle">Massively parallel mapping of substrate cleavage sites defines dipeptidyl peptidase four subsite cooperativity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.17.567387" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.17.567387" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gudipati, R. K.; Gaidatzis, D.; Seebacher, J.; Muehlhaeusser, S.; Kempf, G.; Cavadini, S.; Hess, D.; Soneson, C.; Grosshans, H.</p>
        <p class="info">Score: 7.8, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.17.567387' target='https://doi.org/10.1101/2023.11.17.567387'> 10.1101/2023.11.17.567387</a></p>
        <p class="abstract">Substrate specificity determines protease functions in physiology and in clinical and biotechnological application. However, affordable and unbiased assays for large-scale quantification of substrate cleavage have been lacking. Here, we develop hiMAPS (high-throughput mapping of protease cleavage sites), a cheap, mass spectrometry-based assay, to derive cleavage motifs for human Dipeptidyl Peptidase Four (DPP4), a key regulator of blood glucose levels. We recapitulate the known benefit of proline in the penultimate (P1) position from the substrate N-terminus, extend it to additional residues, and identify and quantify combinatorial interactions of P1 with its neighbors. These findings reveal extensive cooperativity among the enzymes active site subsites and allow us to derive a sequence motif that predicts substrate turnover. We show that this information provides new opportunities to engineer stabilized versions of a key substrate of DPP4, the small peptide hormone GLP-1, whose derivatives are used for clinical treatment of type 2 diabetes and obesity. Finally, structural and biochemical characterization of a DPP4 homologue, C. elegans DPF-3, reveal specific subsite differences and a distinct cleavage motif, providing insight into the mechanistic basis of the observed specificity. Collectively, our findings present a broadly applicable framework to high-throughput mapping of protease cleavage sites, extend our understanding of protease specificity, and provide a chemical space for substrate engineering of a clinically relevant family of exopeptidases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.17.566816">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.17.566816" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.17.566816">
        <p class="paperTitle">A liquid-to-solid phase transition of biomolecular condensates drives in vivo formation of yeast amyloids and prions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.17.566816" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.17.566816" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alieva, M. K.; Nikishina, S. B.; Kireev, I. I.; Golyshev, S. A.; Tyurin-Kuzmin, P. A.; Ivanova, L. V.; Alexandrov, A. I.; Kushnirov, V. V.; Dergalev, A. A.</p>
        <p class="info">Score: 6.8, Published: 2023-11-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.17.566816' target='https://doi.org/10.1101/2023.11.17.566816'> 10.1101/2023.11.17.566816</a></p>
        <p class="abstract">Liquid-liquid phase separation (LLPS) and liquid-solid phase transition (LSPT) of amyloidogenic proteins are now being intensively studied as a potentially widespread mechanism of pathological amyloids formation. However, the possibility and importance of such a mechanism in living systems is still questionable. Here, we investigated the possibility of such LSPT for a series of yeast prion proteins-based constructs overproduced in yeast cells lacking any pre-existing amyloid template. By combining fluorescence and electron microscopy with biochemical and genetic approaches, we have shown that three such constructs (containing the prion domains (PDs) of either Sup35, Rnq1 or Mot3 proteins) form amyloid fibrils via the intermediate stage of liquid-like condensates, that age over time into the more solid-like hydrogels and amyloid bodies. In turn, LSPT of these constructs triggers prion conversion of the corresponding wild-type protein. Two other constructs studied (Ure2- and Sap30-based) are unable to phase separate in vivo and their amyloidogenesis is therefore strongly suppressed. Using PrK-resistant amyloid core mapping, we showed that Sup35PD amyloids formed via LSPT have a different molecular architecture compared to those formed via amyloid cross-seeding. Finally, we showed that physiological LLPS of wild-type Sup35 protein can increase its prion conversion in yeast.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.568356">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.568356" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.568356">
        <p class="paperTitle">Sub-single-turnover quantification of enzyme catalysis at ultrahigh throughput via a versatile NAD(P)H coupled assay in microdroplets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.568356" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.568356" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Penner, M.; Klein, O. J.; Gantz, M.; Boss, S.; Barker, P.; Dupree, P.; Hollfelder, F.</p>
        <p class="info">Score: 4.0, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.568356' target='https://doi.org/10.1101/2023.11.22.568356'> 10.1101/2023.11.22.568356</a></p>
        <p class="abstract">Enzyme engineering and discovery are crucial for a future sustainable bioeconomy, and harvesting new biocatalysts from large libraries through directed evolution or functional metagenomics requires accessible, rapid assays. Ultra-high throughput screening can often require an optical readout, leading to the use of model substrates that may not accurately report on activity for the target reaction and may require bespoke synthesis. In contrast, coupled assays represent a modular  plug-and-play system, where any pairing of enzyme/substrate may be investigated, if the reaction can produce a common intermediate which links the catalytic reaction to a detection cascade readout. Here we establish a detection cascade, producing a fluorescent readout in response to NAD(P)H via glutathione reductase and a subsequent thiol-mediated uncaging reaction, with a 30 nM detection limit. We demonstrate its utility for the glycosidase AxyAgu115A (producing monosaccharides from a natural biofuel feedstock) and report a three orders of magnitude improved sensitivity compared to absorbance-based systems, so that less than one catalytic turnover per enzyme molecule expressed from a single cell is detectable. These advantages are brought to bear in plate formats, but also in picoliter emulsion droplets, where enrichments of 950-fold suggest that large libraries can be interrogated against a specific query substrate.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.568455">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.568455" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.568455">
        <p class="paperTitle">Fluorescent human RPA to track assembly dynamics on DNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.568455" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.568455" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kaushik, V.; Chadda, R.; Kuppa, S.; Pokhrel, N.; Vayyeti, A.; Grady, S.; Arnatt, C.; Antony, E.</p>
        <p class="info">Score: 3.9, Published: 2023-11-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.568455' target='https://doi.org/10.1101/2023.11.23.568455'> 10.1101/2023.11.23.568455</a></p>
        <p class="abstract">DNA metabolic processes including replication, repair, recombination, and telomere maintenance occur on single-stranded DNA (ssDNA). In each of these complex processes, dozens of proteins function together on the ssDNA template. However, when double-stranded DNA is unwound, the transiently open ssDNA is protected and coated by the high affinity heterotrimeric ssDNA binding Replication Protein A (RPA). Almost all downstream DNA processes must first remodel/remove RPA or function alongside to access the ssDNA occluded under RPA. Formation of RPA-ssDNA complexes trigger the DNA damage checkpoint response and is a key step in activating most DNA repair and recombination pathways. Thus, in addition to protecting the exposed ssDNA, RPA functions as a gatekeeper to define functional specificity in DNA maintenance and genomic integrity. RPA achieves functional dexterity through a multi-domain architecture utilizing several DNA binding and protein-interaction domains connected by flexible linkers. This flexible and modular architecture enables RPA to adopt a myriad of configurations tailored for specific DNA metabolic roles. To experimentally capture the dynamics of the domains of RPA upon binding to ssDNA and interacting proteins we here describe the generation of active site-specific fluorescent versions of human RPA (RPA) using 4-azido-L-phenylalanine (4AZP) incorporation and click chemistry. This approach can also be applied to site-specific modifications of other multi-domain proteins. Fluorescence-enhancement through non-canonical amino acids (FEncAA) and Forster Resonance Energy Transfer (FRET) assays for measuring dynamics of RPA on DNA are also described.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.568247">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.568247" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.568247">
        <p class="paperTitle">Expanding the GPCR-RAMP interactome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.568247" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.568247" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kotliar, I. B.; Bendes, A.; Dahl, L.; Chen, Y.; Saarinen, M.; Ceraudo, E.; Dodig, T.; Uhlen, M.; Svenningsson, P.; Schwenk, J.; Sakmar, T.</p>
        <p class="info">Score: 3.9, Published: 2023-11-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.568247' target='https://doi.org/10.1101/2023.11.22.568247'> 10.1101/2023.11.22.568247</a></p>
        <p class="abstract">Receptor activity-modifying proteins (RAMPs) can form complexes with G protein-coupled receptors (GPCRs) and regulate their cellular trafficking and pharmacology. RAMP interactions have been identified for about 50 GPCRs, but only a few GPCR-RAMP complexes have been studied in detail. To elucidate a complete interactome between GPCRs and the three RAMPs, we developed a customized library of 215 Dual Epitope-Tagged (DuET) GPCRs representing all GPCR subfamilies. Using a multiplexed suspension bead array (SBA) assay, we identified 122 GPCRs that showed strong evidence for interaction with at least one RAMP. We screened for native interactions in three cell lines and found 23 GPCRs that formed complexes with RAMPs. Mapping the GPCR-RAMP interactome expands the current system-wide functional characterization of RAMP-interacting GPCRs to inform the design of selective GPCR-targeted therapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.12.566735">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.12.566735" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.12.566735">
        <p class="paperTitle">Development of supramolecular anticoagulants with on-demand reversibility</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.12.566735" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.12.566735" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dockerill, M.; Ford, D. J.; Angerani, S.; Alwis, I.; Ripoll-Rozada, J.; Smythe, R. E.; Liu, J. S. T.; Barbosa Pereira, P. J.; Jackson, S. P.; Payne, R. J.; Winssinger, N.</p>
        <p class="info">Score: 5.9, Published: 2023-11-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.12.566735' target='https://doi.org/10.1101/2023.11.12.566735'> 10.1101/2023.11.12.566735</a></p>
        <p class="abstract">Drugs are administered at a dosing schedule set by their therapeutic index and termination of action is achieved by clearance and metabolism of the drug (hours to days for small molecules, weeks to months for biologics). In some cases, it is important to achieve a fast reversal of the drugs action to overcome dangerous side effects or in response to unforeseen events. A case in point is for anticoagulant drugs. Here we report a general strategy to achieve on-demand reversibility by leveraging supramolecular assembly of drug fragments and showcase the approach with thrombin-inhibiting anticoagulants. In our supramolecular drug design, the action of the drug is reinforced by a dynamic hybridisation of peptide nucleic acids (PNAs) between drug fragments. We show that this design enables the generation of very potent bivalent direct thrombin inhibitors (Ki 74 pM) and this inhibition can be reversed through the use of a PNA antidote. We demonstrate that these supramolecular inhibitors exhibit potent anticoagulant activity in vitro and in vivo and that this activity can also be reversed on demand.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.08.566258">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.08.566258" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.08.566258">
        <p class="paperTitle">Cholesterol regulates plasma membrane bending by prominin-family proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.08.566258" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.08.566258" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bell, T. A.; Luce, B. E.; Hakim, P.; Dardari, H.; Ananda, V. Y.; Nguyen, T. H.; Monshizadeh, A.; Chao, L. H.</p>
        <p class="info">Score: 7.1, Published: 2023-11-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.08.566258' target='https://doi.org/10.1101/2023.11.08.566258'> 10.1101/2023.11.08.566258</a></p>
        <p class="abstract">Prominin-1 (Prom1) is a pentaspan membrane protein that associates with curved regions of the plasma membrane. Prom1 localizes to cholesterol-rich domains and requires membrane cholesterol to support membrane remodeling. Membrane bending activity is particularly evident in photoreceptors, where Prom1 mutations cause loss of outer segment disk homeostasis leading to cone-rod retinal dystrophy (CCRD). However, the mechanistic link between prominin-dependent cholesterol binding, membrane remodeling, and retinal disease remains unclear. Here, we characterize the membrane bending function and specific cholesterol binding activity of Prom1 and its proposed homolog Tweety homology 1 (Ttyh1) in extracellular vesicles (EVs). Prom1 and Ttyh1 induce formation of EVs in cultured mammalian cells that are biophysically similar. Though both proteins bend membranes and form EVs at the plasma membrane, Ttyh1 lacks a stable interaction with cholesterol that is present in Prom1. Correspondingly, Ttyh1 forms EVs that are more deformed than those produced by Prom1. An evolutionarily conserved and retinal disease-associated Prom1 residue (Trp-795) is necessary for cholesterol binding, EV membrane deformation, and efficient trafficking to the plasma membrane. Removal of N-glycan moieties from Prom1 biases the enzyme toward a cholesterol-bound state. We propose that Prom1 and Ttyh1 are both members of a single prominin family of membrane bending proteins, that Ttyh1 is a constitutively active member of this family, and that Prom1 is regulated by cholesterol binding and N-glycosylation. These findings shed light on mechanisms of prominin family function in disease and help unify models of prominin function across diverse cell types.

Significance StatementMammalian cells dynamically shape the plasma membrane to achieve specialized functions. Prominin-1 (Prom1) is a membrane protein that promotes bending and shaping of the membrane, notably in photoreceptors. Here, the authors find that membrane cholesterol regulates the membrane bending activity of Prom1 in extracellular vesicles. Membrane bending can be tuned by altering cholesterol levels and may be coupled to different Prom1 N-glycosylation states. A distant homolog of Prom1 called Ttyh1 is also shown to form extracellular vesicles and supports more membrane bending than Prom1 though it does not bind cholesterol. This work unites the Prom and Ttyh protein families into a single functional group and helps explain how membrane cholesterol functionalizes prominin-family proteins for different roles in different cell types.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
