<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Glutamylation of Npm2 and Nap1 acidic disordered regions increases DNA charge mimicry to enhance chaperone efficiency
Authors: Lorton, B. M.; Warren, C.; Ilyas, H.; Nandigrami, P.; Hegde, S.; Cahill, S.; Lehman, S. M.; Shabanowitz, J.; Hunt, D.; Fiser, A.; Cowburn, D.; Shechter, D.
Score: 16.0, Published: 2023-09-19 DOI: 10.1101/2023.09.18.558337
Histone chaperones-structurally diverse, non-catalytic proteins enriched with acidic intrinsically disordered regions (IDRs)-protect histones from spurious nucleic acid interactions and guide their deposition into and out of nucleosomes.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Glutamylation of Npm2 and Nap1 acidic disordered regions increases DNA charge mimicry to enhance chaperone efficiency
Authors: Lorton, B. M.; Warren, C.; Ilyas, H.; Nandigrami, P.; Hegde, S.; Cahill, S.; Lehman, S. M.; Shabanowitz, J.; Hunt, D.; Fiser, A.; Cowburn, D.; Shechter, D.
Score: 16.0, Published: 2023-09-19 DOI: 10.1101/2023.09.18.558337
Histone chaperones-structurally diverse, non-catalytic proteins enriched with acidic intrinsically disordered regions (IDRs)-protect histones from spurious nucleic acid interactions and guide their deposition into and out of nucleosomes." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-24T10:37:06+00:00" />
<meta property="article:modified_time" content="2023-09-24T10:37:06+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Glutamylation of Npm2 and Nap1 acidic disordered regions increases DNA charge mimicry to enhance chaperone efficiency
Authors: Lorton, B. M.; Warren, C.; Ilyas, H.; Nandigrami, P.; Hegde, S.; Cahill, S.; Lehman, S. M.; Shabanowitz, J.; Hunt, D.; Fiser, A.; Cowburn, D.; Shechter, D.
Score: 16.0, Published: 2023-09-19 DOI: 10.1101/2023.09.18.558337
Histone chaperones-structurally diverse, non-catalytic proteins enriched with acidic intrinsically disordered regions (IDRs)-protect histones from spurious nucleic acid interactions and guide their deposition into and out of nucleosomes."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Glutamylation of Npm2 and Nap1 acidic disordered regions increases DNA charge mimicry to enhance chaperone efficiency\nAuthors: Lorton, B. M.; Warren, C.; Ilyas, H.; Nandigrami, P.; Hegde, S.; Cahill, S.; Lehman, S. M.; Shabanowitz, J.; Hunt, D.; Fiser, A.; Cowburn, D.; Shechter, D.\nScore: 16.0, Published: 2023-09-19 DOI: 10.1101/2023.09.18.558337\nHistone chaperones-structurally diverse, non-catalytic proteins enriched with acidic intrinsically disordered regions (IDRs)-protect histones from spurious nucleic acid interactions and guide their deposition into and out of nucleosomes.",
  "keywords": [
    
  ],
  "articleBody": " Glutamylation of Npm2 and Nap1 acidic disordered regions increases DNA charge mimicry to enhance chaperone efficiency\nAuthors: Lorton, B. M.; Warren, C.; Ilyas, H.; Nandigrami, P.; Hegde, S.; Cahill, S.; Lehman, S. M.; Shabanowitz, J.; Hunt, D.; Fiser, A.; Cowburn, D.; Shechter, D.\nScore: 16.0, Published: 2023-09-19 DOI: 10.1101/2023.09.18.558337\nHistone chaperones-structurally diverse, non-catalytic proteins enriched with acidic intrinsically disordered regions (IDRs)-protect histones from spurious nucleic acid interactions and guide their deposition into and out of nucleosomes. Despite their conservation and ubiquity, the function of the chaperone acidic IDRs remains unclear. Here, we show that the Xenopus laevis Npm2 and Nap1 acidic IDRs are substrates for TTLL4 (Tubulin Tyrosine Ligase Like 4)-catalyzed post-translational glutamate-glutamylation. We demonstrate that, to bind, stabilize, and deposit histones into nucleosomes, chaperone acidic IDRs function as DNA mimetics. Our biochemical, computational, and biophysical studies reveal that glutamylation of these chaperone polyelectrolyte acidic stretches functions to enhance DNA electrostatic mimicry, promoting the binding and stabilization of H2A/H2B heterodimers and facilitating nucleosome assembly. This discovery provides insights into both the previously unclear function of the acidic IDRs and the regulatory role of post-translational modifications in chromatin dynamics.\nStructural and dynamic changes in P-Rex1 upon activation by PIP3 and inhibition by IP4\nAuthors: Ravala, S. K.; Adame-Garcia, S. R.; Li, S.; Chen, C.-L.; Cianfrocco, M. A.; Gutkind, J. S.; Cash, J. N.; Tesmer, J. J. G.\nScore: 13.6, Published: 2023-09-16 DOI: 10.1101/2023.09.15.557836\nPIP3-dependent Rac exchanger 1 (P-Rex1) is abundantly expressed in neutrophils and plays central roles in chemotaxis and cancer metastasis by serving as a guanine nucleotide exchange factor (GEF) for Rac. The enzyme is synergistically activated by PIP3 and the heterotrimeric G{beta}{gamma} subunits, but mechanistic details remain poorly understood. While investigating the regulation of P-Rex1 by PIP3, we discovered that Ins(1,3,4,5)P4 (IP4) inhibits P-Rex1 activity and induces large decreases in backbone dynamics in diverse regions of the protein. Cryo-electron microscopy analysis of the P-Rex1{middle dot}IP4 complex revealed a conformation wherein the pleckstrin homology (PH) domain occludes the active site of the Dbl homology (DH) domain. This configuration is stabilized by interactions between the first DEP domain (DEP1) and the DH domain and between the PH domain and a 4-helix bundle (4HB) subdomain that extends from the C-terminal domain of P-Rex1. Disruption of the DH-DEP1 interface in a DH/PH-DEP1 fragment enhanced activity and led to a more extended conformation in solution, whereas mutations that constrain the occluded conformation led to decreased GEF activity. Variants of full-length P-Rex1 in which the DH-DEP1 and PH-4HB interfaces were disturbed exhibited enhanced activity during chemokine-induced cell migration, confirming that the observed structure represents the autoinhibited state in living cells. Interactions with PIP3-containing liposomes led to disruption of these interfaces and increased dynamics protein-wide. Our results further suggest that inositol phosphates such as IP4 help to inhibit basal P-Rex1 activity in neutrophils, similar to their inhibitory effects on phosphatidylinositol-3-kinase.\nSubstrate selectivity and catalytic activation of the type III CRISPR ancillary nuclease Can2\nAuthors: Jungfer, K.; Sigg, A.; Jinek, M.\nScore: 12.7, Published: 2023-09-18 DOI: 10.1101/2023.09.18.558242\nType III CRISPR-Cas systems provide adaptive immunity against foreign mobile genetic elements through RNA-guided interference. Sequence-specific recognition of RNA targets by the type III effector complex triggers the generation of cyclic oligoadenylate (cOA) second messengers that activate ancillary effector proteins, thus reinforcing the host immune response. The ancillary nuclease Can2 is activated by cyclic tetra-AMP (cA4); however, the mechanisms underlying cA4-mediated activation and substrate selectivity remain elusive. Here we report crystal structures of Thermoanaerobacter brockii Can2 (TbrCan2) in substrate- and product-bound complexes. We show that TbrCan2 is a single strand-selective DNase and RNase that binds substrates via a conserved SxTTS active site motif, and reveal molecular interactions underpinning its sequence preference for CA dinucleotides. Furthermore, we identify a molecular interaction relay linking the cA4 binding site and the nuclease catalytic site to enable divalent metal cation coordination and catalytic activation. These findings provide key insights into the molecular mechanisms of Can2 nucleases in type III CRISPR-Cas immunity and may guide their technological development for nucleic acid detection applications.\nProteome-scale characterisation of protein motif interactome rewiring by disease mutations\nAuthors: Kliche, J.; Simonetti, L.; Krystkowiak, I.; Kuss, H.; Diallo, M.; Rask, E.; Nilsson, J.; Davey, N.; Ivarsson, Y.\nScore: 10.2, Published: 2023-09-18 DOI: 10.1101/2023.09.18.558189\nWhole genome and exome sequencing are reporting on hundreds of thousands of missense mutations. Taking a pan-disease approach, we explored how mutations in the intrinsically disordered regions (IDRs) break or generate short linear motifs. We created a peptide-phage display library tiling peptides overlapping 12,301 disease-associated mutations from the IDRs of the human proteome. The mutations are linked to diverse diseases including cancer, metabolic diseases and neurological diseases. By screening 80 human proteins we found 367 mutation-modulated interactions, with half of the mutations diminishing binding, and half enhancing or creating novel interaction interfaces. The effects of the mutations were confirmed by affinity measurements. In cellular assays, the effects of motif-disruptive mutations were validated, including loss of a nuclear localisation signal in the cell division control protein CDC45 by a mutation associated with Meier-Gorlin syndrome. The study provides a panoramic view of how disease-associated mutations perturb and rewire the motif-based interactome.\nDe novo designed Hsp70 activator dissolves intracellular condensates\nAuthors: Zhang, J. Z.; Greenwood, N.; Hernandez, J.; Cuperus, J. T.; Huang, B.; Ryder, B. D.; Queitsch, C.; Gestwicki, J. E.; Baker, D.\nScore: 8.0, Published: 2023-09-19 DOI: 10.1101/2023.09.18.558356\nProtein quality control (PQC) is carried out in part by the chaperone Hsp70, in concert with adapters of the J-domain protein (JDP) family. The JDPs, also called Hsp40s, are thought to recruit Hsp70 into complexes with specific client proteins. However, the molecular principles regulating this process are not well understood. We describe the de novo design of a set of Hsp70 binding proteins that either inhibited or stimulated Hsp70s ATPase activity; a stimulating design promoted the refolding of denatured luciferase in vitro, similar to native JDPs. Targeting of this design to intracellular condensates resulted in their nearly complete dissolution. The designs inform our understanding of chaperone structure-function relationships and provide a general and modular way to target PQC systems to condensates and other cellular targets.\nA CRISPR activation screen identifies FBXO22 as an E3 ligase supporting targeted protein degradation\nAuthors: Basu, A. A.; Zhang, C.; Riha, I. A.; Magassa, A.; Ko, F.; Zhang, X.\nScore: 7.2, Published: 2023-09-16 DOI: 10.1101/2023.09.15.557708\nTargeted protein degradation (TPD) represents a potent chemical biology paradigm that leverages the cellular degradation machinery to pharmacologically eliminate specific proteins of interest. Although multiple E3 ligases have been discovered to facilitate TPD, there exists a compelling requirement to diversify the pool of E3 ligases available for such applications. This expansion will broaden the scope of potential protein targets, accommodating those with varying subcellular localizations and expression patterns. In this study, we describe a CRISPR-based transcriptional activation screen focused on human E3 ligases, with the goal of identifying E3 ligases that can facilitate heterobifunctional compound-mediated target degradation. This approach allows us to address the limitations associated with investigating candidate degrader molecules in specific cell lines that either lack or have low levels of the desired E3 ligases. Through this approach, we identified a candidate proteolysis-targeting chimera (PROTAC), 22-SLF, that induces the degradation of FKBP12 when the FBXO22 gene transcription is activated. 22-SLF induced the degradation of endogenous FKBP12 in a FBXO22-dependent manner across multiple cancer cell lines. Subsequent mechanistic investigations revealed that 22-SLF interacts with C227 and/or C228 in FBXO22 to achieve the target degradation. Finally, we demonstrated the versatility of FBXO22-based PROTACs by effectively degrading another endogenous protein BRD4. This study uncovers FBXO22 as an E3 ligase capable of supporting ligand-induced protein degradation through electrophilic PROTACs. The platform we have developed can readily be applied to elucidate protein degradation pathways by identifying E3 ligases that facilitate either small molecule-induced or endogenous protein degradation.\nEvolutionary engineering a larger porin using a loop-to-hairpin mechanism\nAuthors: Dhar, R.; Bowman, A. M.; Hatungimana, B.; Slusky, J. S. G.\nScore: 7.7, Published: 2023-09-20 DOI: 10.1101/2023.06.14.544993\nIn protein evolution, diversification is generally driven by genetic duplication. The hallmarks of this mechanism are visible in the repeating topology of various proteins. In outer membrane {beta}-barrels, duplication is visible with {beta}-hairpins as the repeating unit of the barrel. In contrast to the overall use of duplication in diversification, a computational study hypothesized evolutionary mechanisms other than hairpin duplications leading to increases in the number of strands in outer membrane {beta}-barrels. Specifically, the topology of some 16- and 18-stranded {beta}-barrels appear to have evolved through a loop to {beta}-hairpin transition. Here we test this novel evolutionary mechanism by creating a chimeric protein from an 18-stranded {beta}-barrel and an evolutionarily related 16-stranded {beta}-barrel. The chimeric combination of the two was created by replacing loop L3 of the 16-stranded barrel with the sequentially matched transmembrane {beta}-hairpin region of the 18-stranded barrel. We find the resulting chimeric protein is stable and has characteristics of increased strand number. This study provides the first experimental evidence supporting the evolution through a loop to {beta}-hairpin transition. HighlightsO_LIWe find evidence supporting a novel diversification mechanism in membrane {beta}-barrels C_LIO_LIThe mechanism is the conversion of an extracellular loop to transmembrane {beta}-hairpin C_LIO_LIA chimeric protein modeling this mechanism folds stably in the membrane C_LIO_LIThe chimera has more {beta}-structure and a larger pore, consistent with a loop-to-hairpin transition C_LI\nBroad receptor tropism and immunogenicity of a clade 3 sarbecovirus\nAuthors: Lee, J.; Zepeda, S.; Park, Y.-J.; Taylor, A. L.; Quispe, J.; Stewart, C.; Leaf, E. M.; Treichel, C.; Corti, D.; King, N. P.; Starr, T. N.; Veesler, D.\nScore: 88.9, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557371\nAlthough Rhinolophus bats harbor diverse clade 3 sarbecoviruses, the structural determinants of receptor tropism along with the antigenicity of their spike (S) glycoproteins remain uncharacterized. Here, we show that the African Rinolophus bat clade 3 sarbecovirus PRD-0038 S has a broad ACE2 usage and that RBD mutations further expand receptor promiscuity and enable human ACE2 utilization. We determined a cryoEM structure of the PRD-0038 RBD bound to R. alcyone ACE2, explaining receptor tropism and highlighting differences with SARS-CoV-1 and SARS-CoV-2. Characterization of PRD-0038 S using cryoEM and monoclonal antibody reactivity revealed its distinct antigenicity relative to SARS-CoV-2 and identified PRD-0038 cross-neutralizing antibodies for pandemic preparedness. PRD-0038 S vaccination elicited greater titers of antibodies cross-reacting with vaccine-mismatched clade 2 and clade 1a sarbecoviruses compared to SARS-CoV-2 S due to broader antigenic targeting, motivating the inclusion of clade 3 antigens in next-generation vaccines for enhanced resilience to viral evolution.\nRecognition of highly branched N-glycans of the porcine whipworm by the immune system\nAuthors: Eckmair, B.; Gao, C.; Mehta, A. Y.; Dutkiewicz, Z.; Vanbeselaere, J.; Cummings, R. D.; Paschinger, K.; Wilson, I. B. H.\nScore: 5.3, Published: 2023-09-21 DOI: 10.1101/2023.09.21.557549\nGlycans are key to host-pathogen interactions, whereby recognition by the host and immunomodulation by the pathogen can be mediated by carbohydrate binding proteins, such as lectins of the innate immune system, and their glycoconjugate ligands. Previous studies have shown that excretory-secretory products of the porcine nematode parasite Trichuris suis exert immunomodulatory effects in a glycan-dependent manner. To better understand the mechanisms of these interactions, we prepared N-glycans from T. suis and both analyzed their structures and used them to generate a natural glycan microarray. With this array we explored the interactions of glycans with C-type lectins, C-reactive protein and sera from T. suis infected pigs. Glycans containing LacdiNAc and phosphorylcholine-modified glycans were associated with the highest binding by most of these proteins. In-depth analysis revealed not only fucosylated LacdiNAc motifs with and without phosphorylcholine moieties, but phosphorylcholine-modified mannose and N-acetylhexosamine-substituted fucose residues, in the context of maximally tetraantennary N-glycan scaffolds. Furthermore, O-glycans also contained fucosylated motifs. In summary, the glycans of T. suis are recognized by both the innate and adaptive immune systems, and also exhibit species-specific features distinguishing its glycome from those of other nematodes.\nThe Pex6 N1 domain is required for Pex15 binding and proper assembly with Pex1\nAuthors: Ali, B. A.; Judy, R. M.; Chowdhury, S.; Jacobsen, N. K.; Castanzo, D. T.; Carr, K. L.; Richardson, C. D.; Lander, G. C.; Martin, A.; Gardner, B. M.\nScore: 3.8, Published: 2023-09-16 DOI: 10.1101/2023.09.15.557798\nThe heterohexameric AAA-ATPase Pex1/Pex6 is essential for the formation and maintenance of peroxisomes. Pex1/Pex6, similar to other AAA-ATPases, uses the energy from ATP hydrolysis to mechanically thread substrate proteins through its central pore, thereby unfolding them. In related AAA-ATPase motors, substrates are recruited through binding to the motors N-terminal domains or N-terminally bound co-factors. Here we use structural and biochemical techniques to characterize the function of the N1 domain in Pex6 from budding yeast, S. cerevisiae. We found that although Pex1/{Lambda}N1-Pex6 is an active ATPase in vitro, it does not support Pex1/Pex6 function at the peroxisome in vivo. An X-ray crystal structure of the isolated Pex6 N1 domain shows that the Pex6 N1 domain shares the same fold as the N terminal domains of PEX1, CDC48, or NSF, despite poor sequence conservation. Integrating this structure with a cryo-EM reconstruction of Pex1/Pex6, AlphaFold2 predictions, and biochemical assays shows that Pex6 N1 mediates binding to both the peroxisomal membrane tether Pex15 and an extended loop from the D2 ATPase domain of Pex1 that influences Pex1/Pex6 heterohexamer stability. Given the direct interactions with both Pex15 and the D2 ATPase domains, the Pex6 N1 domain is poised to coordinate binding of co-factors and substrates with Pex1/Pex6 ATPase activity.\n",
  "wordCount" : "2211",
  "inLanguage": "en",
  "datePublished": "2023-09-24T10:37:06Z",
  "dateModified": "2023-09-24T10:37:06Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta"><span>updated on September 24, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558337">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558337" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558337">
        <p class="paperTitle">Glutamylation of Npm2 and Nap1 acidic disordered regions increases DNA charge mimicry to enhance chaperone efficiency</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558337" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558337" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lorton, B. M.; Warren, C.; Ilyas, H.; Nandigrami, P.; Hegde, S.; Cahill, S.; Lehman, S. M.; Shabanowitz, J.; Hunt, D.; Fiser, A.; Cowburn, D.; Shechter, D.</p>
        <p class="info">Score: 16.0, Published: 2023-09-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558337' target='https://doi.org/10.1101/2023.09.18.558337'> 10.1101/2023.09.18.558337</a></p>
        <p class="abstract">Histone chaperones-structurally diverse, non-catalytic proteins enriched with acidic intrinsically disordered regions (IDRs)-protect histones from spurious nucleic acid interactions and guide their deposition into and out of nucleosomes. Despite their conservation and ubiquity, the function of the chaperone acidic IDRs remains unclear. Here, we show that the Xenopus laevis Npm2 and Nap1 acidic IDRs are substrates for TTLL4 (Tubulin Tyrosine Ligase Like 4)-catalyzed post-translational glutamate-glutamylation. We demonstrate that, to bind, stabilize, and deposit histones into nucleosomes, chaperone acidic IDRs function as DNA mimetics. Our biochemical, computational, and biophysical studies reveal that glutamylation of these chaperone polyelectrolyte acidic stretches functions to enhance DNA electrostatic mimicry, promoting the binding and stabilization of H2A/H2B heterodimers and facilitating nucleosome assembly. This discovery provides insights into both the previously unclear function of the acidic IDRs and the regulatory role of post-translational modifications in chromatin dynamics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.15.557836">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.15.557836" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.15.557836">
        <p class="paperTitle">Structural and dynamic changes in P-Rex1 upon activation by PIP3 and inhibition by IP4</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.15.557836" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.15.557836" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ravala, S. K.; Adame-Garcia, S. R.; Li, S.; Chen, C.-L.; Cianfrocco, M. A.; Gutkind, J. S.; Cash, J. N.; Tesmer, J. J. G.</p>
        <p class="info">Score: 13.6, Published: 2023-09-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.15.557836' target='https://doi.org/10.1101/2023.09.15.557836'> 10.1101/2023.09.15.557836</a></p>
        <p class="abstract">PIP3-dependent Rac exchanger 1 (P-Rex1) is abundantly expressed in neutrophils and plays central roles in chemotaxis and cancer metastasis by serving as a guanine nucleotide exchange factor (GEF) for Rac. The enzyme is synergistically activated by PIP3 and the heterotrimeric G{beta}{gamma} subunits, but mechanistic details remain poorly understood. While investigating the regulation of P-Rex1 by PIP3, we discovered that Ins(1,3,4,5)P4 (IP4) inhibits P-Rex1 activity and induces large decreases in backbone dynamics in diverse regions of the protein. Cryo-electron microscopy analysis of the P-Rex1{middle dot}IP4 complex revealed a conformation wherein the pleckstrin homology (PH) domain occludes the active site of the Dbl homology (DH) domain. This configuration is stabilized by interactions between the first DEP domain (DEP1) and the DH domain and between the PH domain and a 4-helix bundle (4HB) subdomain that extends from the C-terminal domain of P-Rex1. Disruption of the DH-DEP1 interface in a DH/PH-DEP1 fragment enhanced activity and led to a more extended conformation in solution, whereas mutations that constrain the occluded conformation led to decreased GEF activity. Variants of full-length P-Rex1 in which the DH-DEP1 and PH-4HB interfaces were disturbed exhibited enhanced activity during chemokine-induced cell migration, confirming that the observed structure represents the autoinhibited state in living cells. Interactions with PIP3-containing liposomes led to disruption of these interfaces and increased dynamics protein-wide. Our results further suggest that inositol phosphates such as IP4 help to inhibit basal P-Rex1 activity in neutrophils, similar to their inhibitory effects on phosphatidylinositol-3-kinase.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558242">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558242" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558242">
        <p class="paperTitle">Substrate selectivity and catalytic activation of the type III CRISPR ancillary nuclease Can2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558242" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558242" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jungfer, K.; Sigg, A.; Jinek, M.</p>
        <p class="info">Score: 12.7, Published: 2023-09-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558242' target='https://doi.org/10.1101/2023.09.18.558242'> 10.1101/2023.09.18.558242</a></p>
        <p class="abstract">Type III CRISPR-Cas systems provide adaptive immunity against foreign mobile genetic elements through RNA-guided interference. Sequence-specific recognition of RNA targets by the type III effector complex triggers the generation of cyclic oligoadenylate (cOA) second messengers that activate ancillary effector proteins, thus reinforcing the host immune response. The ancillary nuclease Can2 is activated by cyclic tetra-AMP (cA4); however, the mechanisms underlying cA4-mediated activation and substrate selectivity remain elusive. Here we report crystal structures of Thermoanaerobacter brockii Can2 (TbrCan2) in substrate- and product-bound complexes. We show that TbrCan2 is a single strand-selective DNase and RNase that binds substrates via a conserved SxTTS active site motif, and reveal molecular interactions underpinning its sequence preference for CA dinucleotides. Furthermore, we identify a molecular interaction relay linking the cA4 binding site and the nuclease catalytic site to enable divalent metal cation coordination and catalytic activation. These findings provide key insights into the molecular mechanisms of Can2 nucleases in type III CRISPR-Cas immunity and may guide their technological development for nucleic acid detection applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558189">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558189" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558189">
        <p class="paperTitle">Proteome-scale characterisation of protein motif interactome rewiring by disease mutations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558189" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558189" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kliche, J.; Simonetti, L.; Krystkowiak, I.; Kuss, H.; Diallo, M.; Rask, E.; Nilsson, J.; Davey, N.; Ivarsson, Y.</p>
        <p class="info">Score: 10.2, Published: 2023-09-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558189' target='https://doi.org/10.1101/2023.09.18.558189'> 10.1101/2023.09.18.558189</a></p>
        <p class="abstract">Whole genome and exome sequencing are reporting on hundreds of thousands of missense mutations. Taking a pan-disease approach, we explored how mutations in the intrinsically disordered regions (IDRs) break or generate short linear motifs. We created a peptide-phage display library tiling peptides overlapping 12,301 disease-associated mutations from the IDRs of the human proteome. The mutations are linked to diverse diseases including cancer, metabolic diseases and neurological diseases. By screening 80 human proteins we found 367 mutation-modulated interactions, with half of the mutations diminishing binding, and half enhancing or creating novel interaction interfaces. The effects of the mutations were confirmed by affinity measurements. In cellular assays, the effects of motif-disruptive mutations were validated, including loss of a nuclear localisation signal in the cell division control protein CDC45 by a mutation associated with Meier-Gorlin syndrome. The study provides a panoramic view of how disease-associated mutations perturb and rewire the motif-based interactome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558356">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558356" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558356">
        <p class="paperTitle">De novo designed Hsp70 activator dissolves intracellular condensates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558356" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558356" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, J. Z.; Greenwood, N.; Hernandez, J.; Cuperus, J. T.; Huang, B.; Ryder, B. D.; Queitsch, C.; Gestwicki, J. E.; Baker, D.</p>
        <p class="info">Score: 8.0, Published: 2023-09-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558356' target='https://doi.org/10.1101/2023.09.18.558356'> 10.1101/2023.09.18.558356</a></p>
        <p class="abstract">Protein quality control (PQC) is carried out in part by the chaperone Hsp70, in concert with adapters of the J-domain protein (JDP) family. The JDPs, also called Hsp40s, are thought to recruit Hsp70 into complexes with specific client proteins. However, the molecular principles regulating this process are not well understood. We describe the de novo design of a set of Hsp70 binding proteins that either inhibited or stimulated Hsp70s ATPase activity; a stimulating design promoted the refolding of denatured luciferase in vitro, similar to native JDPs. Targeting of this design to intracellular condensates resulted in their nearly complete dissolution. The designs inform our understanding of chaperone structure-function relationships and provide a general and modular way to target PQC systems to condensates and other cellular targets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.15.557708">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.15.557708" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.15.557708">
        <p class="paperTitle">A CRISPR activation screen identifies FBXO22 as an E3 ligase supporting targeted protein degradation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.15.557708" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.15.557708" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Basu, A. A.; Zhang, C.; Riha, I. A.; Magassa, A.; Ko, F.; Zhang, X.</p>
        <p class="info">Score: 7.2, Published: 2023-09-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.15.557708' target='https://doi.org/10.1101/2023.09.15.557708'> 10.1101/2023.09.15.557708</a></p>
        <p class="abstract">Targeted protein degradation (TPD) represents a potent chemical biology paradigm that leverages the cellular degradation machinery to pharmacologically eliminate specific proteins of interest. Although multiple E3 ligases have been discovered to facilitate TPD, there exists a compelling requirement to diversify the pool of E3 ligases available for such applications. This expansion will broaden the scope of potential protein targets, accommodating those with varying subcellular localizations and expression patterns. In this study, we describe a CRISPR-based transcriptional activation screen focused on human E3 ligases, with the goal of identifying E3 ligases that can facilitate heterobifunctional compound-mediated target degradation. This approach allows us to address the limitations associated with investigating candidate degrader molecules in specific cell lines that either lack or have low levels of the desired E3 ligases. Through this approach, we identified a candidate proteolysis-targeting chimera (PROTAC), 22-SLF, that induces the degradation of FKBP12 when the FBXO22 gene transcription is activated. 22-SLF induced the degradation of endogenous FKBP12 in a FBXO22-dependent manner across multiple cancer cell lines. Subsequent mechanistic investigations revealed that 22-SLF interacts with C227 and/or C228 in FBXO22 to achieve the target degradation. Finally, we demonstrated the versatility of FBXO22-based PROTACs by effectively degrading another endogenous protein BRD4. This study uncovers FBXO22 as an E3 ligase capable of supporting ligand-induced protein degradation through electrophilic PROTACs. The platform we have developed can readily be applied to elucidate protein degradation pathways by identifying E3 ligases that facilitate either small molecule-induced or endogenous protein degradation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.14.544993">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.14.544993" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.14.544993">
        <p class="paperTitle">Evolutionary engineering a larger porin using a loop-to-hairpin mechanism</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.14.544993" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.14.544993" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dhar, R.; Bowman, A. M.; Hatungimana, B.; Slusky, J. S. G.</p>
        <p class="info">Score: 7.7, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.14.544993' target='https://doi.org/10.1101/2023.06.14.544993'> 10.1101/2023.06.14.544993</a></p>
        <p class="abstract">In protein evolution, diversification is generally driven by genetic duplication. The hallmarks of this mechanism are visible in the repeating topology of various proteins. In outer membrane {beta}-barrels, duplication is visible with {beta}-hairpins as the repeating unit of the barrel. In contrast to the overall use of duplication in diversification, a computational study hypothesized evolutionary mechanisms other than hairpin duplications leading to increases in the number of strands in outer membrane {beta}-barrels. Specifically, the topology of some 16- and 18-stranded {beta}-barrels appear to have evolved through a loop to {beta}-hairpin transition. Here we test this novel evolutionary mechanism by creating a chimeric protein from an 18-stranded {beta}-barrel and an evolutionarily related 16-stranded {beta}-barrel. The chimeric combination of the two was created by replacing loop L3 of the 16-stranded barrel with the sequentially matched transmembrane {beta}-hairpin region of the 18-stranded barrel. We find the resulting chimeric protein is stable and has characteristics of increased strand number. This study provides the first experimental evidence supporting the evolution through a loop to {beta}-hairpin transition.

HighlightsO_LIWe find evidence supporting a novel diversification mechanism in membrane {beta}-barrels
C_LIO_LIThe mechanism is the conversion of an extracellular loop to transmembrane {beta}-hairpin
C_LIO_LIA chimeric protein modeling this mechanism folds stably in the membrane
C_LIO_LIThe chimera has more {beta}-structure and a larger pore, consistent with a loop-to-hairpin transition
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557371">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557371" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557371">
        <p class="paperTitle">Broad receptor tropism and immunogenicity of a clade 3 sarbecovirus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557371" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557371" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, J.; Zepeda, S.; Park, Y.-J.; Taylor, A. L.; Quispe, J.; Stewart, C.; Leaf, E. M.; Treichel, C.; Corti, D.; King, N. P.; Starr, T. N.; Veesler, D.</p>
        <p class="info">Score: 88.9, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557371' target='https://doi.org/10.1101/2023.09.12.557371'> 10.1101/2023.09.12.557371</a></p>
        <p class="abstract">Although Rhinolophus bats harbor diverse clade 3 sarbecoviruses, the structural determinants of receptor tropism along with the antigenicity of their spike (S) glycoproteins remain uncharacterized. Here, we show that the African Rinolophus bat clade 3 sarbecovirus PRD-0038 S has a broad ACE2 usage and that RBD mutations further expand receptor promiscuity and enable human ACE2 utilization. We determined a cryoEM structure of the PRD-0038 RBD bound to R. alcyone ACE2, explaining receptor tropism and highlighting differences with SARS-CoV-1 and SARS-CoV-2. Characterization of PRD-0038 S using cryoEM and monoclonal antibody reactivity revealed its distinct antigenicity relative to SARS-CoV-2 and identified PRD-0038 cross-neutralizing antibodies for pandemic preparedness. PRD-0038 S vaccination elicited greater titers of antibodies cross-reacting with vaccine-mismatched clade 2 and clade 1a sarbecoviruses compared to SARS-CoV-2 S due to broader antigenic targeting, motivating the inclusion of clade 3 antigens in next-generation vaccines for enhanced resilience to viral evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.557549">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.557549" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.557549">
        <p class="paperTitle">Recognition of highly branched N-glycans of the porcine whipworm by the immune system</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.557549" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.557549" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eckmair, B.; Gao, C.; Mehta, A. Y.; Dutkiewicz, Z.; Vanbeselaere, J.; Cummings, R. D.; Paschinger, K.; Wilson, I. B. H.</p>
        <p class="info">Score: 5.3, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.557549' target='https://doi.org/10.1101/2023.09.21.557549'> 10.1101/2023.09.21.557549</a></p>
        <p class="abstract">Glycans are key to host-pathogen interactions, whereby recognition by the host and immunomodulation by the pathogen can be mediated by carbohydrate binding proteins, such as lectins of the innate immune system, and their glycoconjugate ligands. Previous studies have shown that excretory-secretory products of the porcine nematode parasite Trichuris suis exert immunomodulatory effects in a glycan-dependent manner. To better understand the mechanisms of these interactions, we prepared N-glycans from T. suis and both analyzed their structures and used them to generate a natural glycan microarray. With this array we explored the interactions of glycans with C-type lectins, C-reactive protein and sera from T. suis infected pigs. Glycans containing LacdiNAc and phosphorylcholine-modified glycans were associated with the highest binding by most of these proteins. In-depth analysis revealed not only fucosylated LacdiNAc motifs with and without phosphorylcholine moieties, but phosphorylcholine-modified mannose and N-acetylhexosamine-substituted fucose residues, in the context of maximally tetraantennary N-glycan scaffolds. Furthermore, O-glycans also contained fucosylated motifs. In summary, the glycans of T. suis are recognized by both the innate and adaptive immune systems, and also exhibit species-specific features distinguishing its glycome from those of other nematodes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.15.557798">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.15.557798" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.15.557798">
        <p class="paperTitle">The Pex6 N1 domain is required for Pex15 binding and proper assembly with Pex1</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.15.557798" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.15.557798" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ali, B. A.; Judy, R. M.; Chowdhury, S.; Jacobsen, N. K.; Castanzo, D. T.; Carr, K. L.; Richardson, C. D.; Lander, G. C.; Martin, A.; Gardner, B. M.</p>
        <p class="info">Score: 3.8, Published: 2023-09-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.15.557798' target='https://doi.org/10.1101/2023.09.15.557798'> 10.1101/2023.09.15.557798</a></p>
        <p class="abstract">The heterohexameric AAA-ATPase Pex1/Pex6 is essential for the formation and maintenance of peroxisomes. Pex1/Pex6, similar to other AAA-ATPases, uses the energy from ATP hydrolysis to mechanically thread substrate proteins through its central pore, thereby unfolding them. In related AAA-ATPase motors, substrates are recruited through binding to the motors N-terminal domains or N-terminally bound co-factors. Here we use structural and biochemical techniques to characterize the function of the N1 domain in Pex6 from budding yeast, S. cerevisiae. We found that although Pex1/{Lambda}N1-Pex6 is an active ATPase in vitro, it does not support Pex1/Pex6 function at the peroxisome in vivo. An X-ray crystal structure of the isolated Pex6 N1 domain shows that the Pex6 N1 domain shares the same fold as the N terminal domains of PEX1, CDC48, or NSF, despite poor sequence conservation. Integrating this structure with a cryo-EM reconstruction of Pex1/Pex6, AlphaFold2 predictions, and biochemical assays shows that Pex6 N1 mediates binding to both the peroxisomal membrane tether Pex15 and an extended loop from the D2 ATPase domain of Pex1 that influences Pex1/Pex6 heterohexamer stability. Given the direct interactions with both Pex15 and the D2 ATPase domains, the Pex6 N1 domain is poised to coordinate binding of co-factors and substrates with Pex1/Pex6 ATPase activity.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
