<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Generalized Biomolecular Modeling and Design with RoseTTAFold All-Atom
Authors: Krishna, R.; Wang, J.; Ahern, W.; Sturmfels, P.; Venkatesh, P.; Kalvet, I.; Lee, G. R.; Morey-Burrows, F. S.; Anishchenko, I.; Humphreys, I. R.; McHugh, R.; Vafeados, D.; Li, X.; Sutherland, G. A.; Hitchcock, A.; Hunter, C. N.; Baek, M.; DiMaio, F.; Baker, D.
Score: 332.0, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561603
Although AlphaFold2 (AF2) and RoseTTAFold (RF) have transformed structural biology by enabling high-accuracy protein structure modeling, they are unable to model covalent modifications or interactions with small molecules and other non-protein molecules that can play key roles in biological function.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Generalized Biomolecular Modeling and Design with RoseTTAFold All-Atom
Authors: Krishna, R.; Wang, J.; Ahern, W.; Sturmfels, P.; Venkatesh, P.; Kalvet, I.; Lee, G. R.; Morey-Burrows, F. S.; Anishchenko, I.; Humphreys, I. R.; McHugh, R.; Vafeados, D.; Li, X.; Sutherland, G. A.; Hitchcock, A.; Hunter, C. N.; Baek, M.; DiMaio, F.; Baker, D.
Score: 332.0, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561603
Although AlphaFold2 (AF2) and RoseTTAFold (RF) have transformed structural biology by enabling high-accuracy protein structure modeling, they are unable to model covalent modifications or interactions with small molecules and other non-protein molecules that can play key roles in biological function." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-22T10:37:23+00:00" />
<meta property="article:modified_time" content="2023-10-22T10:37:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Generalized Biomolecular Modeling and Design with RoseTTAFold All-Atom
Authors: Krishna, R.; Wang, J.; Ahern, W.; Sturmfels, P.; Venkatesh, P.; Kalvet, I.; Lee, G. R.; Morey-Burrows, F. S.; Anishchenko, I.; Humphreys, I. R.; McHugh, R.; Vafeados, D.; Li, X.; Sutherland, G. A.; Hitchcock, A.; Hunter, C. N.; Baek, M.; DiMaio, F.; Baker, D.
Score: 332.0, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561603
Although AlphaFold2 (AF2) and RoseTTAFold (RF) have transformed structural biology by enabling high-accuracy protein structure modeling, they are unable to model covalent modifications or interactions with small molecules and other non-protein molecules that can play key roles in biological function."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Generalized Biomolecular Modeling and Design with RoseTTAFold All-Atom\nAuthors: Krishna, R.; Wang, J.; Ahern, W.; Sturmfels, P.; Venkatesh, P.; Kalvet, I.; Lee, G. R.; Morey-Burrows, F. S.; Anishchenko, I.; Humphreys, I. R.; McHugh, R.; Vafeados, D.; Li, X.; Sutherland, G. A.; Hitchcock, A.; Hunter, C. N.; Baek, M.; DiMaio, F.; Baker, D.\nScore: 332.0, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561603\nAlthough AlphaFold2 (AF2) and RoseTTAFold (RF) have transformed structural biology by enabling high-accuracy protein structure modeling, they are unable to model covalent modifications or interactions with small molecules and other non-protein molecules that can play key roles in biological function.",
  "keywords": [
    
  ],
  "articleBody": " Generalized Biomolecular Modeling and Design with RoseTTAFold All-Atom\nAuthors: Krishna, R.; Wang, J.; Ahern, W.; Sturmfels, P.; Venkatesh, P.; Kalvet, I.; Lee, G. R.; Morey-Burrows, F. S.; Anishchenko, I.; Humphreys, I. R.; McHugh, R.; Vafeados, D.; Li, X.; Sutherland, G. A.; Hitchcock, A.; Hunter, C. N.; Baek, M.; DiMaio, F.; Baker, D.\nScore: 332.0, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561603\nAlthough AlphaFold2 (AF2) and RoseTTAFold (RF) have transformed structural biology by enabling high-accuracy protein structure modeling, they are unable to model covalent modifications or interactions with small molecules and other non-protein molecules that can play key roles in biological function. Here, we describe RoseTTAFold All-Atom (RFAA), a deep network capable of modeling full biological assemblies containing proteins, nucleic acids, small molecules, metals, and covalent modifications given the sequences of the polymers and the atomic bonded geometry of the small molecules and covalent modifications. Following training on structures of full biological assemblies in the Protein Data Bank (PDB), RFAA has comparable protein structure prediction accuracy to AF2, excellent performance in CAMEO for flexible backbone small molecule docking, and reasonable prediction accuracy for protein covalent modifications and assemblies of proteins with multiple nucleic acid chains and small molecules which, to our knowledge, no existing method can model simultaneously. By fine-tuning on diffusive denoising tasks, we develop RFdiffusion All-Atom (RFdiffusionAA), which generates binding pockets by directly building protein structures around small molecules and other non-protein molecules. Starting from random distributions of amino acid residues surrounding target small molecules, we design and experimentally validate proteins that bind the cardiac disease therapeutic digoxigenin, the enzymatic cofactor heme, and optically active bilin molecules with potential for expanding the range of wavelengths captured by photosynthesis. We anticipate that RFAA and RFdiffusionAA will be widely useful for modeling and designing complex biomolecular systems.\nStructural and molecular basis of choline uptake into the brain by FLVCR2\nAuthors: Cater, R. J.; Mukherjee, D.; Gil Iturbe, E.; Erramilli, S. K.; Chen, T.; Koo, K.; Santander, N. G.; Reckers, A. R.; Kloss, B.; Gawda, T.; Choy, B. C.; Zhang, Z.; Clarke, O. B.; Yee, S. W.; Kossiakoff, A. K.; Quick, M.; Arnold, T.; Mancia, F.\nScore: 23.7, Published: 2023-10-05 DOI: 10.1101/2023.10.05.561059\nCholine is an essential nutrient that the human body needs in vast quantities for cell membrane synthesis, epigenetic modification, and neurotransmission. The brain has a particularly high demand for choline, but how it enters the brain has eluded the field for over fifty years. The MFS transporter FLVCR1 was recently determined to be a choline transporter, and while this protein is not highly expressed at the blood-brain barrier (BBB), its relative FLVCR2 is. Previous studies have shown that mutations in human Flvcr2 cause cerebral vascular abnormalities, hydrocephalus, and embryonic lethality, but the physiological role of FLVCR2 is unknown. Here, we demonstrate both in vivo and in vitro that FLVCR2 is a BBB choline transporter and is responsible for the majority of choline uptake into the brain. We also determine the structures of choline-bound FLVCR2 in the inward- and outward-facing states using cryo-electron microscopy to 2.49 and 2.77 [A] resolution, respectively. These results reveal how the brain obtains choline and provide molecular-level insights into how FLVCR2 binds choline in an aromatic cage and mediates its uptake. Our work could provide a novel framework for the targeted delivery of neurotherapeutics into the brain.\nDefining the Cell Surface Cysteinome using Two-step Enrichment Proteomics\nAuthors: Yan, T.; Boatner, L.; Cui, L.; Tontonoz, P.; Backus, K.\nScore: 15.5, Published: 2023-10-19 DOI: 10.1101/2023.10.17.562832\nThe plasma membrane proteome is a rich resource of functional and therapeutically relevant protein targets. Distinguished by high hydrophobicity, heavy glycosylation, disulfide-rich sequences, and low overall abundance, the cell surface proteome remains undersampled in established proteomic pipelines, including our own cysteine chemoproteomics platforms. Here we paired cell surface glycoprotein capture with cysteine chemoproteomics to establish a two-stage enrichment method that enables chemoproteomic profiling of cell Surface Cysteinome. Our \"Cys-Surf\" platform captures \u003e2,800 total membrane protein cysteines in 1,046 proteins, including 1,907 residues not previously captured by bulk proteomic analysis. By pairing Cys-Surf with an isotopic chemoproteomic readout, we uncovered 821 total ligandable cysteines, including known and novel sites. Cys-Surf also robustly delineates redox-sensitive cysteines, including cysteines prone to activation-dependent changes to cysteine oxidation state and residues sensitive to addition of exogenous reductants. Exemplifying the capacity of Cys-Surf to delineate functionally important cysteines, we identified a redox sensitive cysteine in the low-density lipoprotein receptor (LDLR) that impacts both the protein localization and uptake of LDL particles. Taken together, the Cys-Surf platform, distinguished by its two-stage enrichment paradigm, represents a tailored approach to delineate the functional and therapeutic potential of the plasma membrane cysteinome.\nThe guide RNA sequence dictates the slicing kinetics and conformational dynamics of the Argonaute silencing complex\nAuthors: Wang, P. Y.; Bartel, D. P.\nScore: 14.8, Published: 2023-10-16 DOI: 10.1101/2023.10.15.562437\nThe RNA-induced silencing complex (RISC), which powers RNA interference (RNAi), consists of a guide RNA and an Argonaute protein that slices target RNAs complementary to the guide. We find that for different guide-RNA sequences, slicing rates of perfectly paired, bound targets can be surprisingly different (\u003e250-fold range), and that faster slicing confers greater knockdown efficacy in cells. Four sequence determinants underlie much of this variation in slicing rates. Analyses of these determinants implicate a structural distortion at guide nucleotides 6-7 in promoting slicing, and indicate that, in contrast to the prevailing model, slicing is primarily limited by the rate at which the enzyme-substrate complex reaches a slicing-competent conformation. Moreover, different guide sequences have an unanticipated, 600-fold range in 3'-mismatch tolerance, which is attributable to the GC-content of their central regions. Together, our analyses identify sequence determinants of RISC activity and provide biochemical, conformational, and functional rationale for their action.\nCryo-EM structure of the CDK2-cyclin A-CDC25A complex\nAuthors: Rowland, R. J.; Korolchuk, S.; Salamina, M.; Ault, J. R.; Hart, S.; Turkenberg, J. P.; Blaza, J. N.; Noble, M. E.; Endicott, J. A.\nScore: 14.6, Published: 2023-10-17 DOI: 10.1101/2023.10.17.562665\nThe cell division cycle 25 phosphatases CDC25A, B and C regulate cell cycle transitions by dephosphorylating residues in the conserved glycine-rich motif of cyclin-dependent protein kinases (CDKs) to activate CDK activity. Here, we present the cryogenic-electron microscopy (cryo-EM) structure of CDK2-cyclin A in complex with CDC25A at 2.91 [A] resolution, providing a detailed structural analysis of the overall complex architecture and key protein-protein interactions that underpin this 86 kDa complex. We further reveal an unanticipated CDC25A C-terminal helix that is critical for complex formation. Sequence conservation analysis suggests CDK1/2-cyclin A, CDK1-cyclin B and CDK2/3-cyclin E are suitable binding partners for CDC25A, whilst CDK4/6-cyclin D complexes appear unlikely substrates. A comparative structural analysis of CDK-containing complexes also confirms the functional importance of the conserved CDK1/2 GDSEID motif. This structure improves our understanding of the roles of CDC25 phosphatases in CDK regulation and may inform the development of CDC25- targeting anticancer strategies.\nHeterogeneity of Tethered Agonist Signaling in Adhesion G Protein-Coupled Receptors\nAuthors: Dates, A. N.; Jones, D. T. D.; Smith, J. S.; Skiba, M. A.; Rich, M. F.; Burruss, M. M.; Kruse, A. C.; Blacklow, S. C.\nScore: 16.4, Published: 2023-10-19 DOI: 10.1101/2022.07.14.500097\nAdhesion G Protein-Coupled Receptor (aGPCR) signaling influences development and homeostasis in a wide range of tissues. In the current model for aGPCR signaling, ligand binding liberates or unmasks a highly conserved tethered agonist (TA) that acts as an intramolecular ligand to stimulate G protein coupling. However, a systematic, comprehensive examination of signaling has not been performed for all aGPCR family members. Here, we report a platform for profiling TA-dependent activities of aGPCRs in several different assays and apply it to all 33 human family members. Activity profiling identified a heterogeneity of responses among these aGPCRs, with only ~50% showing robust, TA-dependent signals. AlphaFold2 predictions assessing TA engagement in the predicted intramolecular binding pocket aligned with the TA-dependence of the cellular responses. The signaling information in this dataset is a comprehensive resource relevant for the investigation of all human aGPCRs and for targeting aGPCRs therapeutically.\nEx vivo structures from spinach leaves\nAuthors: Wang, J.; Johansen, N. T.; Gamon, L. F.; Zhao, Z.; Guo, Z.; Wang, Y.; Fuglsang, A. T.; Gourdon, P.; Wang, K.\nScore: 8.8, Published: 2023-10-14 DOI: 10.1101/2023.10.13.562012\nEx vivo structure determination of macromolecules from native source is gaining increasing attention from the scientific community, as the method can be employed to dissect the function of important, multi-component molecular machines. However, the existing ex vivo procedures often require genome manipulation or availability high-affinity binders, limiting the general applicability. Here, we report simple yet robust principles for isolation of protein complexes from enriched native biological material, enabling cryoEM-facilitated high-resolution structure determination. We report the structures of ten separate membrane and soluble protein complexes determined from spinach leaves. Moreover, the developed pipeline is likely adaptable to essentially any biological system. As such, the approach may represent an attractive avenue for future structural proteomics efforts.\nExploiting the Cullin E3 Ligase Adaptor Protein SKP1 for Targeted Protein Degradation\nAuthors: Hong, S. H.; Osa, A.; Wertz, I. E.; Nomura, D.\nScore: 5.3, Published: 2023-10-21 DOI: 10.1101/2023.10.20.563371\nTargeted protein degradation with Proteolysis Targeting Chimeras (PROTACs) is a powerful therapeutic modality for eliminating disease-causing proteins through targeted ubiquitination and proteasome-mediated degradation. Most PROTACs have exploited substrate receptors of Cullin-RING E3 ubiquitin ligases such as cereblon and VHL. Whether core, shared, and essential components of the Cullin-RING E3 ubiquitin ligase complex can be used for PROTAC applications remains less explored. Here, we discovered a cysteine-reactive covalent recruiter EN884 against the SKP1 adapter protein of the SKP1-CUL1-F-box containing SCF complex. We further showed that this recruiter can be used in PROTAC applications to degrade neo-substrate proteins such as BRD4 and the androgen receptor in a SKP1- and proteasome-dependent manner. Our studies demonstrate that core and essential adapter proteins within the Cullin-RING E3 ubiquitin ligase complex can be exploited for targeted protein degradation applications and that covalent chemoproteomic strategies can enable recruiter discovery against these targets.\nTrifunctional sphinganine: a new tool to dissect sphingolipid function\nAuthors: Farley, S. E.; Stein, F.; Haberkant, P.; Tafesse, F. G.; Schultz, C.\nScore: 4.8, Published: 2023-10-16 DOI: 10.1101/2023.10.16.562528\nFunctions of the sphingolipids sphingosine and sphinganine in cells are not well established. While some signaling roles for sphingosine have been elucidated, the closely related sphinganine has been described only insofar as it does not elicit many of the same signaling responses. The underlying mechanisms behind the cell biological differences between these lipids are not well understood. Here, we prepared multifunctionalized derivatives of the two lipid species that only differ in a single double bond of the carbon backbone. Using these novel probes, we were able to define their spatiotemporal distribution within cells. Furthermore, we used these tools to systematically map the protein interactomes of both lipids. The lipid-protein conjugates, prepared through photo-crosslinking in live cells and extraction via click chemistry to azide beads, revealed significant differences in the captured proteins, highlighting their distinct roles in various cellular processes. This work elucidates mechanistic differences between these critical lipids and sets the foundation for further studies on the functions of sphingosine and sphinganine.\nThe role of RNA in the maintenance of chromatin domains as revealed by antibody mediated proximity labelling coupled to mass spectrometry\nAuthors: Choudhury, R.; Venkatasubramani, A. V.; Jie, H.; Borso, M.; Forne, I.; Imhof, A.\nScore: 2.2, Published: 2023-10-18 DOI: 10.1101/2023.10.16.562458\nEukaryotic chromatin is organised in individual domains, which are defined by their location within the nuclear space, the molecular interactions within each other and their distinct proteomic compositions. In contrast to cellular organelles surrounded by lipid membranes, the composition of distinct chromatin domains is rather ill described and highly dynamic. To identify proteins associated with a specific chromatin domain and to better understand how those domains are established and maintained, we used a new method, which we termed AMPL-MS (Antibody mediated proximity labelling mass spectrometry). This method is based on antibodies and does not require the expression of fusion proteins and therefore constitutes a versatile and very sensitive method to characterize chromatin domains containing specific signature proteins or histone modifications. We used AMPL-MS to characterize the composition of chromocenter as well as the chromosome territory containing the hyperactive X-chromosome in Drosophila. As an outcome of this method, we show the importance of RNA in maintaining the integrity of these domains as treatment with RNAse alters their proteomic composition. Our data demonstrate the power of AMPL-MS to characterize the composition of non-membranous nuclear domains.\n",
  "wordCount" : "2054",
  "inLanguage": "en",
  "datePublished": "2023-10-22T10:37:23Z",
  "dateModified": "2023-10-22T10:37:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta"><span>updated on October 22, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561603">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561603" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561603">
        <p class="paperTitle">Generalized Biomolecular Modeling and Design with RoseTTAFold All-Atom</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561603" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561603" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krishna, R.; Wang, J.; Ahern, W.; Sturmfels, P.; Venkatesh, P.; Kalvet, I.; Lee, G. R.; Morey-Burrows, F. S.; Anishchenko, I.; Humphreys, I. R.; McHugh, R.; Vafeados, D.; Li, X.; Sutherland, G. A.; Hitchcock, A.; Hunter, C. N.; Baek, M.; DiMaio, F.; Baker, D.</p>
        <p class="info">Score: 332.0, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561603' target='https://doi.org/10.1101/2023.10.09.561603'> 10.1101/2023.10.09.561603</a></p>
        <p class="abstract">Although AlphaFold2 (AF2) and RoseTTAFold (RF) have transformed structural biology by enabling high-accuracy protein structure modeling, they are unable to model covalent modifications or interactions with small molecules and other non-protein molecules that can play key roles in biological function. Here, we describe RoseTTAFold All-Atom (RFAA), a deep network capable of modeling full biological assemblies containing proteins, nucleic acids, small molecules, metals, and covalent modifications given the sequences of the polymers and the atomic bonded geometry of the small molecules and covalent modifications. Following training on structures of full biological assemblies in the Protein Data Bank (PDB), RFAA has comparable protein structure prediction accuracy to AF2, excellent performance in CAMEO for flexible backbone small molecule docking, and reasonable prediction accuracy for protein covalent modifications and assemblies of proteins with multiple nucleic acid chains and small molecules which, to our knowledge, no existing method can model simultaneously. By fine-tuning on diffusive denoising tasks, we develop RFdiffusion All-Atom (RFdiffusionAA), which generates binding pockets by directly building protein structures around small molecules and other non-protein molecules. Starting from random distributions of amino acid residues surrounding target small molecules, we design and experimentally validate proteins that bind the cardiac disease therapeutic digoxigenin, the enzymatic cofactor heme, and optically active bilin molecules with potential for expanding the range of wavelengths captured by photosynthesis. We anticipate that RFAA and RFdiffusionAA will be widely useful for modeling and designing complex biomolecular systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.561059">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.561059" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.561059">
        <p class="paperTitle">Structural and molecular basis of choline uptake into the brain by FLVCR2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.561059" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.561059" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cater, R. J.; Mukherjee, D.; Gil Iturbe, E.; Erramilli, S. K.; Chen, T.; Koo, K.; Santander, N. G.; Reckers, A. R.; Kloss, B.; Gawda, T.; Choy, B. C.; Zhang, Z.; Clarke, O. B.; Yee, S. W.; Kossiakoff, A. K.; Quick, M.; Arnold, T.; Mancia, F.</p>
        <p class="info">Score: 23.7, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.561059' target='https://doi.org/10.1101/2023.10.05.561059'> 10.1101/2023.10.05.561059</a></p>
        <p class="abstract">Choline is an essential nutrient that the human body needs in vast quantities for cell membrane synthesis, epigenetic modification, and neurotransmission. The brain has a particularly high demand for choline, but how it enters the brain has eluded the field for over fifty years. The MFS transporter FLVCR1 was recently determined to be a choline transporter, and while this protein is not highly expressed at the blood-brain barrier (BBB), its relative FLVCR2 is. Previous studies have shown that mutations in human Flvcr2 cause cerebral vascular abnormalities, hydrocephalus, and embryonic lethality, but the physiological role of FLVCR2 is unknown. Here, we demonstrate both in vivo and in vitro that FLVCR2 is a BBB choline transporter and is responsible for the majority of choline uptake into the brain. We also determine the structures of choline-bound FLVCR2 in the inward- and outward-facing states using cryo-electron microscopy to 2.49 and 2.77 [A] resolution, respectively. These results reveal how the brain obtains choline and provide molecular-level insights into how FLVCR2 binds choline in an aromatic cage and mediates its uptake. Our work could provide a novel framework for the targeted delivery of neurotherapeutics into the brain.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562832">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562832" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562832">
        <p class="paperTitle">Defining the Cell Surface Cysteinome using Two-step Enrichment Proteomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562832" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562832" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yan, T.; Boatner, L.; Cui, L.; Tontonoz, P.; Backus, K.</p>
        <p class="info">Score: 15.5, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562832' target='https://doi.org/10.1101/2023.10.17.562832'> 10.1101/2023.10.17.562832</a></p>
        <p class="abstract">The plasma membrane proteome is a rich resource of functional and therapeutically relevant protein targets. Distinguished by high hydrophobicity, heavy glycosylation, disulfide-rich sequences, and low overall abundance, the cell surface proteome remains undersampled in established proteomic pipelines, including our own cysteine chemoproteomics platforms. Here we paired cell surface glycoprotein capture with cysteine chemoproteomics to establish a two-stage enrichment method that enables chemoproteomic profiling of cell Surface Cysteinome. Our &#34;Cys-Surf&#34; platform captures &gt;2,800 total membrane protein cysteines in 1,046 proteins, including 1,907 residues not previously captured by bulk proteomic analysis. By pairing Cys-Surf with an isotopic chemoproteomic readout, we uncovered 821 total ligandable cysteines, including known and novel sites. Cys-Surf also robustly delineates redox-sensitive cysteines, including cysteines prone to activation-dependent changes to cysteine oxidation state and residues sensitive to addition of exogenous reductants. Exemplifying the capacity of Cys-Surf to delineate functionally important cysteines, we identified a redox sensitive cysteine in the low-density lipoprotein receptor (LDLR) that impacts both the protein localization and uptake of LDL particles. Taken together, the Cys-Surf platform, distinguished by its two-stage enrichment paradigm, represents a tailored approach to delineate the functional and therapeutic potential of the plasma membrane cysteinome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.15.562437">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.15.562437" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.15.562437">
        <p class="paperTitle">The guide RNA sequence dictates the slicing kinetics and conformational dynamics of the Argonaute silencing complex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.15.562437" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.15.562437" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, P. Y.; Bartel, D. P.</p>
        <p class="info">Score: 14.8, Published: 2023-10-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.15.562437' target='https://doi.org/10.1101/2023.10.15.562437'> 10.1101/2023.10.15.562437</a></p>
        <p class="abstract">The RNA-induced silencing complex (RISC), which powers RNA interference (RNAi), consists of a guide RNA and an Argonaute protein that slices target RNAs complementary to the guide. We find that for different guide-RNA sequences, slicing rates of perfectly paired, bound targets can be surprisingly different (&gt;250-fold range), and that faster slicing confers greater knockdown efficacy in cells. Four sequence determinants underlie much of this variation in slicing rates. Analyses of these determinants implicate a structural distortion at guide nucleotides 6-7 in promoting slicing, and indicate that, in contrast to the prevailing model, slicing is primarily limited by the rate at which the enzyme-substrate complex reaches a slicing-competent conformation. Moreover, different guide sequences have an unanticipated, 600-fold range in 3&#39;-mismatch tolerance, which is attributable to the GC-content of their central regions. Together, our analyses identify sequence determinants of RISC activity and provide biochemical, conformational, and functional rationale for their action.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562665">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562665" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562665">
        <p class="paperTitle">Cryo-EM structure of the CDK2-cyclin A-CDC25A complex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562665" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562665" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rowland, R. J.; Korolchuk, S.; Salamina, M.; Ault, J. R.; Hart, S.; Turkenberg, J. P.; Blaza, J. N.; Noble, M. E.; Endicott, J. A.</p>
        <p class="info">Score: 14.6, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562665' target='https://doi.org/10.1101/2023.10.17.562665'> 10.1101/2023.10.17.562665</a></p>
        <p class="abstract">The cell division cycle 25 phosphatases CDC25A, B and C regulate cell cycle transitions by dephosphorylating residues in the conserved glycine-rich motif of cyclin-dependent protein kinases (CDKs) to activate CDK activity. Here, we present the cryogenic-electron microscopy (cryo-EM) structure of CDK2-cyclin A in complex with CDC25A at 2.91 [A] resolution, providing a detailed structural analysis of the overall complex architecture and key protein-protein interactions that underpin this 86 kDa complex. We further reveal an unanticipated CDC25A C-terminal helix that is critical for complex formation. Sequence conservation analysis suggests CDK1/2-cyclin A, CDK1-cyclin B and CDK2/3-cyclin E are suitable binding partners for CDC25A, whilst CDK4/6-cyclin D complexes appear unlikely substrates. A comparative structural analysis of CDK-containing complexes also confirms the functional importance of the conserved CDK1/2 GDSEID motif. This structure improves our understanding of the roles of CDC25 phosphatases in CDK regulation and may inform the development of CDC25- targeting anticancer strategies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.07.14.500097">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.07.14.500097" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.07.14.500097">
        <p class="paperTitle">Heterogeneity of Tethered Agonist Signaling in Adhesion G Protein-Coupled Receptors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.07.14.500097" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.07.14.500097" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dates, A. N.; Jones, D. T. D.; Smith, J. S.; Skiba, M. A.; Rich, M. F.; Burruss, M. M.; Kruse, A. C.; Blacklow, S. C.</p>
        <p class="info">Score: 16.4, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.07.14.500097' target='https://doi.org/10.1101/2022.07.14.500097'> 10.1101/2022.07.14.500097</a></p>
        <p class="abstract">Adhesion G Protein-Coupled Receptor (aGPCR) signaling influences development and homeostasis in a wide range of tissues. In the current model for aGPCR signaling, ligand binding liberates or unmasks a highly conserved tethered agonist (TA) that acts as an intramolecular ligand to stimulate G protein coupling. However, a systematic, comprehensive examination of signaling has not been performed for all aGPCR family members. Here, we report a platform for profiling TA-dependent activities of aGPCRs in several different assays and apply it to all 33 human family members. Activity profiling identified a heterogeneity of responses among these aGPCRs, with only ~50% showing robust, TA-dependent signals. AlphaFold2 predictions assessing TA engagement in the predicted intramolecular binding pocket aligned with the TA-dependence of the cellular responses. The signaling information in this dataset is a comprehensive resource relevant for the investigation of all human aGPCRs and for targeting aGPCRs therapeutically.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562012">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562012" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562012">
        <p class="paperTitle">Ex vivo structures from spinach leaves</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562012" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562012" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, J.; Johansen, N. T.; Gamon, L. F.; Zhao, Z.; Guo, Z.; Wang, Y.; Fuglsang, A. T.; Gourdon, P.; Wang, K.</p>
        <p class="info">Score: 8.8, Published: 2023-10-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562012' target='https://doi.org/10.1101/2023.10.13.562012'> 10.1101/2023.10.13.562012</a></p>
        <p class="abstract">Ex vivo structure determination of macromolecules from native source is gaining increasing attention from the scientific community, as the method can be employed to dissect the function of important, multi-component molecular machines. However, the existing ex vivo procedures often require genome manipulation or availability high-affinity binders, limiting the general applicability. Here, we report simple yet robust principles for isolation of protein complexes from enriched native biological material, enabling cryoEM-facilitated high-resolution structure determination. We report the structures of ten separate membrane and soluble protein complexes determined from spinach leaves. Moreover, the developed pipeline is likely adaptable to essentially any biological system. As such, the approach may represent an attractive avenue for future structural proteomics efforts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.563371">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.563371" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.563371">
        <p class="paperTitle">Exploiting the Cullin E3 Ligase Adaptor Protein SKP1 for Targeted Protein Degradation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.563371" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.563371" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hong, S. H.; Osa, A.; Wertz, I. E.; Nomura, D.</p>
        <p class="info">Score: 5.3, Published: 2023-10-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.563371' target='https://doi.org/10.1101/2023.10.20.563371'> 10.1101/2023.10.20.563371</a></p>
        <p class="abstract">Targeted protein degradation with Proteolysis Targeting Chimeras (PROTACs) is a powerful therapeutic modality for eliminating disease-causing proteins through targeted ubiquitination and proteasome-mediated degradation. Most PROTACs have exploited substrate receptors of Cullin-RING E3 ubiquitin ligases such as cereblon and VHL. Whether core, shared, and essential components of the Cullin-RING E3 ubiquitin ligase complex can be used for PROTAC applications remains less explored. Here, we discovered a cysteine-reactive covalent recruiter EN884 against the SKP1 adapter protein of the SKP1-CUL1-F-box containing SCF complex. We further showed that this recruiter can be used in PROTAC applications to degrade neo-substrate proteins such as BRD4 and the androgen receptor in a SKP1- and proteasome-dependent manner. Our studies demonstrate that core and essential adapter proteins within the Cullin-RING E3 ubiquitin ligase complex can be exploited for targeted protein degradation applications and that covalent chemoproteomic strategies can enable recruiter discovery against these targets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.16.562528">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.16.562528" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.16.562528">
        <p class="paperTitle">Trifunctional sphinganine: a new tool to dissect sphingolipid function</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.16.562528" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.16.562528" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Farley, S. E.; Stein, F.; Haberkant, P.; Tafesse, F. G.; Schultz, C.</p>
        <p class="info">Score: 4.8, Published: 2023-10-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.16.562528' target='https://doi.org/10.1101/2023.10.16.562528'> 10.1101/2023.10.16.562528</a></p>
        <p class="abstract">Functions of the sphingolipids sphingosine and sphinganine in cells are not well established. While some signaling roles for sphingosine have been elucidated, the closely related sphinganine has been described only insofar as it does not elicit many of the same signaling responses. The underlying mechanisms behind the cell biological differences between these lipids are not well understood. Here, we prepared multifunctionalized derivatives of the two lipid species that only differ in a single double bond of the carbon backbone. Using these novel probes, we were able to define their spatiotemporal distribution within cells. Furthermore, we used these tools to systematically map the protein interactomes of both lipids. The lipid-protein conjugates, prepared through photo-crosslinking in live cells and extraction via click chemistry to azide beads, revealed significant differences in the captured proteins, highlighting their distinct roles in various cellular processes. This work elucidates mechanistic differences between these critical lipids and sets the foundation for further studies on the functions of sphingosine and sphinganine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.16.562458">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.16.562458" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.16.562458">
        <p class="paperTitle">The role of RNA in the maintenance of chromatin domains as revealed by antibody mediated proximity labelling coupled to mass spectrometry</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.16.562458" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.16.562458" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Choudhury, R.; Venkatasubramani, A. V.; Jie, H.; Borso, M.; Forne, I.; Imhof, A.</p>
        <p class="info">Score: 2.2, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.16.562458' target='https://doi.org/10.1101/2023.10.16.562458'> 10.1101/2023.10.16.562458</a></p>
        <p class="abstract">Eukaryotic chromatin is organised in individual domains, which are defined by their location within the nuclear space, the molecular interactions within each other and their distinct proteomic compositions. In contrast to cellular organelles surrounded by lipid membranes, the composition of distinct chromatin domains is rather ill described and highly dynamic. To identify proteins associated with a specific chromatin domain and to better understand how those domains are established and maintained, we used a new method, which we termed AMPL-MS (Antibody mediated proximity labelling mass spectrometry). This method is based on antibodies and does not require the expression of fusion proteins and therefore constitutes a versatile and very sensitive method to characterize chromatin domains containing specific signature proteins or histone modifications. We used AMPL-MS to characterize the composition of chromocenter as well as the chromosome territory containing the hyperactive X-chromosome in Drosophila. As an outcome of this method, we show the importance of RNA in maintaining the integrity of these domains as treatment with RNAse alters their proteomic composition. Our data demonstrate the power of AMPL-MS to characterize the composition of non-membranous nuclear domains.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
