<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="DNA end sensing and cleavage by the Shedu anti-phage defense system
Authors: Loeff, L.; Walter, A.; Rosalen, G. T.; Jinek, M.
Score: 39.2, Published: 2023-08-12 DOI: 10.1101/2023.08.10.552762
The detection of molecular patterns associated with invading pathogens is a hallmark of innate immune systems. Prokaryotes deploy sophisticated host-defense mechanisms in innate anti-phage immunity. Shedu is a single-component defense system comprising a putative nuclease SduA. Here we report cryo-EM structures of apo- and dsDNA-bound tetrameric SduA assemblies, revealing that the N-terminal domains of SduA form a clamp that recognizes free DNA ends.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="DNA end sensing and cleavage by the Shedu anti-phage defense system
Authors: Loeff, L.; Walter, A.; Rosalen, G. T.; Jinek, M.
Score: 39.2, Published: 2023-08-12 DOI: 10.1101/2023.08.10.552762
The detection of molecular patterns associated with invading pathogens is a hallmark of innate immune systems. Prokaryotes deploy sophisticated host-defense mechanisms in innate anti-phage immunity. Shedu is a single-component defense system comprising a putative nuclease SduA. Here we report cryo-EM structures of apo- and dsDNA-bound tetrameric SduA assemblies, revealing that the N-terminal domains of SduA form a clamp that recognizes free DNA ends." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-16T10:37:20+00:00" />
<meta property="article:modified_time" content="2023-08-16T10:37:20+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="DNA end sensing and cleavage by the Shedu anti-phage defense system
Authors: Loeff, L.; Walter, A.; Rosalen, G. T.; Jinek, M.
Score: 39.2, Published: 2023-08-12 DOI: 10.1101/2023.08.10.552762
The detection of molecular patterns associated with invading pathogens is a hallmark of innate immune systems. Prokaryotes deploy sophisticated host-defense mechanisms in innate anti-phage immunity. Shedu is a single-component defense system comprising a putative nuclease SduA. Here we report cryo-EM structures of apo- and dsDNA-bound tetrameric SduA assemblies, revealing that the N-terminal domains of SduA form a clamp that recognizes free DNA ends."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "DNA end sensing and cleavage by the Shedu anti-phage defense system\nAuthors: Loeff, L.; Walter, A.; Rosalen, G. T.; Jinek, M.\nScore: 39.2, Published: 2023-08-12 DOI: 10.1101/2023.08.10.552762\nThe detection of molecular patterns associated with invading pathogens is a hallmark of innate immune systems. Prokaryotes deploy sophisticated host-defense mechanisms in innate anti-phage immunity. Shedu is a single-component defense system comprising a putative nuclease SduA. Here we report cryo-EM structures of apo- and dsDNA-bound tetrameric SduA assemblies, revealing that the N-terminal domains of SduA form a clamp that recognizes free DNA ends.",
  "keywords": [
    
  ],
  "articleBody": " DNA end sensing and cleavage by the Shedu anti-phage defense system\nAuthors: Loeff, L.; Walter, A.; Rosalen, G. T.; Jinek, M.\nScore: 39.2, Published: 2023-08-12 DOI: 10.1101/2023.08.10.552762\nThe detection of molecular patterns associated with invading pathogens is a hallmark of innate immune systems. Prokaryotes deploy sophisticated host-defense mechanisms in innate anti-phage immunity. Shedu is a single-component defense system comprising a putative nuclease SduA. Here we report cryo-EM structures of apo- and dsDNA-bound tetrameric SduA assemblies, revealing that the N-terminal domains of SduA form a clamp that recognizes free DNA ends. End binding positions the DNA over the PD/ExK nuclease domain, resulting in dsDNA nicking at a fixed distance from the 5-prime end. The end-directed DNA nicking activity of Shedu prevents propagation of linear DNA in vivo. Finally, we show that phages escape Shedu immunity by suppressing their recombination-dependent DNA replication pathway. Taken together, these results define the antiviral mechanism of Shedu systems, underlining the paradigm that recognition of pathogen-specific nucleic acid structures is a conserved feature of innate immunity across all domains of life.\nThe ER-SURF pathway uses ER-mitochondria contact sites for protein targeting to mitochondria\nAuthors: Koch, C.; Raeschle, M.; Prescianotto-Baschong, C.; Spang, A.; Herrmann, J. M.\nScore: 33.7, Published: 2023-08-11 DOI: 10.1101/2023.08.10.552816\nMost mitochondrial proteins are synthesized on cytosolic ribosomes and imported into mitochondria in a post-translational reaction. Mitochondrial precursor proteins which use the ER-SURF pathway employ the surface of the endoplasmic reticulum (ER) as an important sorting platform. How they reach the mitochondrial import machinery from the ER is not known. Here we show that mitochondrial contact sites play a crucial role in the ER-to-mitochondria transfer of precursor proteins. The ER encounter structure (ERMES) and Tom70 are part of two parallel and partially redundant ER-to-mitochondria transfer routes. When ER-to-mitochondria transfer is prevented, many mitochondrial precursor proteins associate with ER membranes, resulting in mitochondrial dysfunction. Our observations support an active role of the ER in mitochondrial protein biogenesis.\nStructure and activation mechanism of the Makes caterpillars floppy 1 toxin\nAuthors: Belyy, A.; Heilen, P.; Hagel, P.; Hofnagel, O.; Raunser, S.\nScore: 18.9, Published: 2023-08-14 DOI: 10.1101/2023.08.14.553178\nThe bacterial Makes caterpillars floppy 1 (Mcf1) toxin promotes apoptosis in insects, leading to loss of body turgor and death. The molecular mechanism underlying Mcf1 intoxication is poorly understood. Here, we present the cryo-EM structure of Mcf1 from Photorhabdus luminescens, revealing a seahorse-like shape with a head and tail. While the three head domains contain two effectors, an activator-binding domain (ABD) and an autoprotease, the tail consists of two translocation as well as three receptor-binding domains. Rearrangement of the tail moves the C-terminus away from the ABD and allows binding of the host cell ADP-ribosylation factor 3, inducing conformational changes that position the cleavage site closer to the protease. This unique activation mechanism that is based on a hook-loop interaction results in three autocleavage reactions and the release of two toxic effectors. Unexpectedly, the BH3-like domain containing ABD is not an active effector. Our findings allow us to understand key steps of Mcf1 intoxication at the molecular level.\nOne-Tip enables comprehensive proteome coverage in minimal cells and single zygotes\nAuthors: Ye, Z.; Sabatier, P.; Martin-Gonzalez, J.; Eguchi, A.; Bekker-Jensen, D. B.; Bache, N.; Olsen, J. V.\nScore: 15.9, Published: 2023-08-11 DOI: 10.1101/2023.08.10.552756\nWe present One-Tip, a lossless proteomics methodology that seamlessly combines swift, one-pot sample preparation with narrow-window data-independent acquisition mass spectrometric analysis. With simplest sample processing, One-Tip reproducibly identifies \u003e 9,000 proteins from [~]1000 cells and [~] 6,000 proteins in a single mouse zygote with a throughput of 40 samples-per-day. This easy-to-use method expands capabilities of proteomics research by enabling greater depth, scalability and throughput covering low to high input samples.\nSHRIMP: Genetically Encoded mScarlet-derived Red Fluorescent Hydrogen Peroxide Sensor with High Brightness and Minimal Photoactivation\nAuthors: Pang, Y.; Zhang, Y.; Zhang, J.; Li, Z.; He, Y.; Wang, Y.; Oberholzer, J.; Ai, H.\nScore: 14.5, Published: 2023-08-09 DOI: 10.1101/2023.08.09.552302\nRed fluorescent protein (RFP) derived indicators are popular due to advantages such as increased imaging depth and reduced autofluorescence and cytotoxicity. However, most RFP-based indicators have low brightness and are susceptible to blue-light-induced photoactivation. In this study, we aimed to overcome the limitations of existing red fluorescent indicators. We utilized mScarlet-I, a highly bright and robust monomeric RFP, to develop a circularly permuted variant called cpmScarlet. We further engineered cpmScarlet into a novel red fluorescent indicator specifically for hydrogen peroxide (H2O2), a crucial reactive oxygen species (ROS) involved in redox signaling and oxidative stress. The resultant indicator, SHRIMP (mScarlet-derived H2O2 Redox Indicator with Minimal Photoactivation), exhibited excitation and emission peaks at [~]570 and 595 nm, respectively, and demonstrated a maximum five-fold fluorescence turn-off response to H2O2. Importantly, SHRIMP was not susceptible to blue-light-induced photoactivation and showed high brightness both in its purified protein form and when expressed in mammalian cells. We successfully employed SHRIMP to visualize H2O2 dynamics in mammalian cells with exogenously added H2O2 and in activated macrophages. Additionally, we demonstrated its utility for multiparameter imaging by co-expressing SHRIMP with GCaMP6m, a green fluorescent calcium indicator, enabling simultaneous monitoring of H2O2 and calcium dynamics in mammalian cells in response to thapsigargin (TG) and epidermal growth factor (EGF) stimulation. Furthermore, we expressed SHRIMP in isolated primary mouse islet tissue, and SHRIMP exhibited excellent brightness and capability for effective detection of H2O2 production during streptozotocin (STZ)-induced {beta}-cell damage. This study successfully transformed mScarlet-I, a bright and robust monomeric RFP, into a circularly permuted variant (cpmScarlet) and developed the first cpmScarlet-based genetically encoded fluorescent indicator called SHRIMP. SHRIMP exhibits high brightness and insensitivity to photoactivation and is a valuable tool for real-time monitoring of H2O2 dynamics in various biological systems. Further research may yield an expanded family of cpmScarlet-based red fluorescent indicators with enhanced photophysical properties.\nTargeted Protein Degradation through Recruitment of the CUL4A Complex Adaptor Protein DDB1\nAuthors: Meyers, M.; Cismoski, S.; Panidapu, A.; Chie-Leon, B.; Nomura, D. K.\nScore: 10.3, Published: 2023-08-12 DOI: 10.1101/2023.08.11.553046\nTargeted protein degradation has arisen as a powerful therapeutic modality for eliminating proteins. Thus far, most heterobifunctional Proteolysis Targeting Chimeras (PROTACs) have utilized recruiters against substrate receptors of Cullin RING E3 ubiquitin ligases, such as cereblon and VHL. However, previous studies have surprisingly uncovered molecular glue degraders that exploit a CUL4A adaptor protein DDB1 to degrade neosubstrate proteins. Here, we sought to investigate whether DDB1 recruiters can be discovered that can be exploited for PROTAC applications. We utilized activity-based protein profiling and cysteine chemoproteomic screening to identify a covalent recruiter that targets C173 on DDB1 and exploited this recruiter to develop PROTACs against BRD4 and androgen receptor (AR). We demonstrated that the BRD4 PROTAC results in selective degradation of the short BRD4 isoform over the long isoform in a proteasome, NEDDylation, and DDB1-dependent manner. We also demonstrated degradation of AR with the AR PROTAC in prostate cancer cells. Our study demonstrated that covalent chemoproteomic approaches can be used to discover recruiters against Cullin RING adapter proteins and that these recruiters can be used for PROTAC applications to degrade neo-substrates.\nIdentification of factors that prevent ribosome stalling during early elongation\nAuthors: Hong, H.-R.; Prince, C. R.; Wu, L.; Feaga, H. A.\nScore: 29.1, Published: 2023-08-04 DOI: 10.1101/2023.08.04.552005\nProtein synthesis is catalyzed by the ribosome and a host of highly conserved elongation factors. Most elongation factors that are conserved in all domains of life are essential, such as EF-Tu (e/aEF1A) and EF-G (e/aEF2). In contrast, the universally conserved elongation factor P (EF-P/eIF5A) is essential in eukaryotes but is dispensable in most bacteria. EF-P prevents ribosome stalling at difficult-to translate sequences, especially polyprolines. Since efp deletion phenotypes range from modest to lethal in different bacterial species, we hypothesized that some bacteria encode an uncharacterized elongation factor with compensatory functions. To identify this factor, we used Tn-seq to screen for genes that are essential in the absence of EF-P in Bacillus subtilis. This screen identified YfmR, a member of the ABCF family of ATPases, as a translation factor that is essential when efp is deleted. We find that depleting YfmR from {Delta}efp cells decreases actively translating ribosomes and increases free ribosomal subunits bound to initiator tRNA, suggesting that ribosomes stall in early elongation. We also find that deleting efp from the spore-forming pathogen Bacillus anthracis causes a severe survival defect in liquid culture and in the presence of macrophages. B. anthracis {Delta}efp also does not produce detectable spores. We find that heterologous expression of B. subtilis YfmR in B. anthracis {Delta}efp cells partially rescues the severe growth and sporulation defects of this mutant. Our results therefore identify YfmR as an important translation factor that becomes essential in the absence of EF-P, and suggest that the essential function of YfmR and EF-P is to assist translation during early elongation. SignificanceTranslation is one of the most ancient and energetically demanding process that occurs in the cell. Ribosomes constitute more than 60% of cellular mass in actively growing cells, and ribosomes are a major target of antimicrobials and chemotherapeutics. Here, we report evidence that both EF-P and the uncharacterized protein YfmR are important for early translation elongation in bacteria, and that this activity is essential. YfmR is a member of the ABCF family of ATPases whose role in translation is only beginning to be understood. Given the broad distribution of ABCFs from bacteria to fungi, we expect our results to have implications for understanding the crucial first steps of translation elongation in diverse organisms.\nFucosylated glycoproteins and fucosylated glycolipids play opposing roles in cholera intoxication\nAuthors: Ghorashi, A. C.; Boucher, A.; Archer-hartmann, S. A.; Murray, N. B.; Konada, R. S. R.; Zhang, X.; Xing, C.; Azadi, P.; Yrlid, U.; Kohler, J. J.\nScore: 8.3, Published: 2023-08-03 DOI: 10.1101/2023.08.02.551727\nCholera toxin (CT) is the etiological agent of cholera. Here we report that multiple classes of fucosylated glycoconjugates function in CT binding and intoxication of intestinal epithelial cells. In Colo205 cells, knockout of B3GNT5, the enzyme required for synthesis of lacto- and neolacto-series glycosphingolipids (GSLs), reduces CT binding but sensitizes cells to intoxication. Overexpressing B3GNT5 to generate more fucosylated GSLs confers protection against intoxication, indicating that fucosylated GSLs act as decoy receptors for CT. Knockout (KO) of B3GALT5 causes increased production of fucosylated O-linked and N-linked glycoproteins, and leads to increased CT binding and intoxication. Knockout of B3GNT5 in B3GALT5 KO cells eliminates production of fucosylated GSLs but increases intoxication, identifying fucosylated glycoproteins as functional receptors for CT. These findings provide insight into molecular determinants regulating CT sensitivity of host cells.\nMulti-omic stratification of the missense variant cysteinome\nAuthors: Desai, H. S.; Ofori, S.; Boatner, L. M.; Yu, F.; Villaneuva, M.; Ung, N.; Nesvizhskii, A.; Backus, K. M.\nScore: 7.3, Published: 2023-08-14 DOI: 10.1101/2023.08.12.553095\nCancer genomes are rife with genetic variants; one key outcome of this variation is gain-of-cysteine, which is the most frequently acquired amino acid due to missense variants in COSMIC. Acquired cysteines are both driver mutations and sites targeted by precision therapies. However, despite their ubiquity, nearly all acquired cysteines remain uncharacterized. Here, we pair cysteine chemoproteomics, a technique that enables proteome-wide pinpointing of functional, redox sensitive, and potentially druggable residues, with genomics to reveal the hidden landscape of cysteine acquisition. For both cancer and healthy genomes, we find that cysteine acquisition is a ubiquitous consequence of genetic variation that is further elevated in the context of decreased DNA repair. Our chemoproteogenomics platform integrates chemoproteomic, whole exome, and RNA-seq data, with a customized 2-stage false discovery rate (FDR) error controlled proteomic search, further enhanced with a user-friendly FragPipe interface. Integration of CADD predictions of deleteriousness revealed marked enrichment for likely damaging variants that result in acquisition of cysteine. By deploying chemoproteogenomics across 11 cell lines, we identify 116 gain-of-cysteines, of which 10 were liganded by electrophilic druglike molecules. Reference cysteines proximal to missense variants were also found to be pervasive, 791 in total, supporting heretofore untapped opportunities for proteoform-specific chemical probe development campaigns. As chemoproteogenomics is further distinguished by sample-matched combinatorial variant databases and compatible with redox proteomics and small molecule screening, we expect widespread utility in guiding proteoform-specific biology and therapeutic discovery.\nBifunctional Monosaccharides Preferentially Localize to Nuclear Subcompartments\nAuthors: Barahtjan, P.; Iglesias-Artola, J. M.; Boehlig, K.; Lohmann, A.; Nadler, A.\nScore: 7.3, Published: 2023-08-11 DOI: 10.1101/2023.08.11.552905\nRecent progress in glycan research has been driven by widespread implementations of metabolic oligosaccharide engineering. Complementing existing approaches, we here introduce bifunctional, UV-crosslinkable and clickable N-acetylglucosamine and N-acetylgalactosamine analogues, which enable direct visualization of the intracellular probe distribution as well as distinguishing monomeric and macromolecule-bound fractions. Using this feature, we find that monomeric N-acetylmonosaccharides partition into RNA-rich nuclear compartments such as nuclear speckles and nucleoli. This suggests the existence of spatially separated N-acetylmonosccharide pools within the nucleoplasm. Taken together, bifunctional N-acetylmonosaccharide probes are a powerful discovery tool for probing intracellular localization of monosaccharides and glycosylated macromolecules.\n",
  "wordCount" : "2120",
  "inLanguage": "en",
  "datePublished": "2023-08-16T10:37:20Z",
  "dateModified": "2023-08-16T10:37:20Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta"><span>updated on August 16, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552762">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552762" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552762">
        <p class="paperTitle">DNA end sensing and cleavage by the Shedu anti-phage defense system</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552762" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552762" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Loeff, L.; Walter, A.; Rosalen, G. T.; Jinek, M.</p>
        <p class="info">Score: 39.2, Published: 2023-08-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552762' target='https://doi.org/10.1101/2023.08.10.552762'> 10.1101/2023.08.10.552762</a></p>
        <p class="abstract">The detection of molecular patterns associated with invading pathogens is a hallmark of innate immune systems. Prokaryotes deploy sophisticated host-defense mechanisms in innate anti-phage immunity. Shedu is a single-component defense system comprising a putative nuclease SduA. Here we report cryo-EM structures of apo- and dsDNA-bound tetrameric SduA assemblies, revealing that the N-terminal domains of SduA form a clamp that recognizes free DNA ends. End binding positions the DNA over the PD/ExK nuclease domain, resulting in dsDNA nicking at a fixed distance from the 5-prime end. The end-directed DNA nicking activity of Shedu prevents propagation of linear DNA in vivo. Finally, we show that phages escape Shedu immunity by suppressing their recombination-dependent DNA replication pathway. Taken together, these results define the antiviral mechanism of Shedu systems, underlining the paradigm that recognition of pathogen-specific nucleic acid structures is a conserved feature of innate immunity across all domains of life.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552816">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552816" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552816">
        <p class="paperTitle">The ER-SURF pathway uses ER-mitochondria contact sites for protein targeting to mitochondria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552816" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552816" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Koch, C.; Raeschle, M.; Prescianotto-Baschong, C.; Spang, A.; Herrmann, J. M.</p>
        <p class="info">Score: 33.7, Published: 2023-08-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552816' target='https://doi.org/10.1101/2023.08.10.552816'> 10.1101/2023.08.10.552816</a></p>
        <p class="abstract">Most mitochondrial proteins are synthesized on cytosolic ribosomes and imported into mitochondria in a post-translational reaction. Mitochondrial precursor proteins which use the ER-SURF pathway employ the surface of the endoplasmic reticulum (ER) as an important sorting platform. How they reach the mitochondrial import machinery from the ER is not known. Here we show that mitochondrial contact sites play a crucial role in the ER-to-mitochondria transfer of precursor proteins. The ER encounter structure (ERMES) and Tom70 are part of two parallel and partially redundant ER-to-mitochondria transfer routes. When ER-to-mitochondria transfer is prevented, many mitochondrial precursor proteins associate with ER membranes, resulting in mitochondrial dysfunction. Our observations support an active role of the ER in mitochondrial protein biogenesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.14.553178">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.14.553178" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.14.553178">
        <p class="paperTitle">Structure and activation mechanism of the Makes caterpillars floppy 1 toxin</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.14.553178" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.14.553178" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Belyy, A.; Heilen, P.; Hagel, P.; Hofnagel, O.; Raunser, S.</p>
        <p class="info">Score: 18.9, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.14.553178' target='https://doi.org/10.1101/2023.08.14.553178'> 10.1101/2023.08.14.553178</a></p>
        <p class="abstract">The bacterial Makes caterpillars floppy 1 (Mcf1) toxin promotes apoptosis in insects, leading to loss of body turgor and death. The molecular mechanism underlying Mcf1 intoxication is poorly understood. Here, we present the cryo-EM structure of Mcf1 from Photorhabdus luminescens, revealing a seahorse-like shape with a head and tail. While the three head domains contain two effectors, an activator-binding domain (ABD) and an autoprotease, the tail consists of two translocation as well as three receptor-binding domains. Rearrangement of the tail moves the C-terminus away from the ABD and allows binding of the host cell ADP-ribosylation factor 3, inducing conformational changes that position the cleavage site closer to the protease. This unique activation mechanism that is based on a hook-loop interaction results in three autocleavage reactions and the release of two toxic effectors. Unexpectedly, the BH3-like domain containing ABD is not an active effector. Our findings allow us to understand key steps of Mcf1 intoxication at the molecular level.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552756">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552756" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552756">
        <p class="paperTitle">One-Tip enables comprehensive proteome coverage in minimal cells and single zygotes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552756" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552756" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ye, Z.; Sabatier, P.; Martin-Gonzalez, J.; Eguchi, A.; Bekker-Jensen, D. B.; Bache, N.; Olsen, J. V.</p>
        <p class="info">Score: 15.9, Published: 2023-08-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552756' target='https://doi.org/10.1101/2023.08.10.552756'> 10.1101/2023.08.10.552756</a></p>
        <p class="abstract">We present One-Tip, a lossless proteomics methodology that seamlessly combines swift, one-pot sample preparation with narrow-window data-independent acquisition mass spectrometric analysis. With simplest sample processing, One-Tip reproducibly identifies &gt; 9,000 proteins from [~]1000 cells and [~] 6,000 proteins in a single mouse zygote with a throughput of 40 samples-per-day. This easy-to-use method expands capabilities of proteomics research by enabling greater depth, scalability and throughput covering low to high input samples.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.552302">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.552302" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.552302">
        <p class="paperTitle">SHRIMP: Genetically Encoded mScarlet-derived Red Fluorescent Hydrogen Peroxide Sensor with High Brightness and Minimal Photoactivation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.552302" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.552302" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pang, Y.; Zhang, Y.; Zhang, J.; Li, Z.; He, Y.; Wang, Y.; Oberholzer, J.; Ai, H.</p>
        <p class="info">Score: 14.5, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.552302' target='https://doi.org/10.1101/2023.08.09.552302'> 10.1101/2023.08.09.552302</a></p>
        <p class="abstract">Red fluorescent protein (RFP) derived indicators are popular due to advantages such as increased imaging depth and reduced autofluorescence and cytotoxicity. However, most RFP-based indicators have low brightness and are susceptible to blue-light-induced photoactivation. In this study, we aimed to overcome the limitations of existing red fluorescent indicators. We utilized mScarlet-I, a highly bright and robust monomeric RFP, to develop a circularly permuted variant called cpmScarlet. We further engineered cpmScarlet into a novel red fluorescent indicator specifically for hydrogen peroxide (H2O2), a crucial reactive oxygen species (ROS) involved in redox signaling and oxidative stress. The resultant indicator, SHRIMP (mScarlet-derived H2O2 Redox Indicator with Minimal Photoactivation), exhibited excitation and emission peaks at [~]570 and 595 nm, respectively, and demonstrated a maximum five-fold fluorescence turn-off response to H2O2. Importantly, SHRIMP was not susceptible to blue-light-induced photoactivation and showed high brightness both in its purified protein form and when expressed in mammalian cells. We successfully employed SHRIMP to visualize H2O2 dynamics in mammalian cells with exogenously added H2O2 and in activated macrophages. Additionally, we demonstrated its utility for multiparameter imaging by co-expressing SHRIMP with GCaMP6m, a green fluorescent calcium indicator, enabling simultaneous monitoring of H2O2 and calcium dynamics in mammalian cells in response to thapsigargin (TG) and epidermal growth factor (EGF) stimulation. Furthermore, we expressed SHRIMP in isolated primary mouse islet tissue, and SHRIMP exhibited excellent brightness and capability for effective detection of H2O2 production during streptozotocin (STZ)-induced {beta}-cell damage. This study successfully transformed mScarlet-I, a bright and robust monomeric RFP, into a circularly permuted variant (cpmScarlet) and developed the first cpmScarlet-based genetically encoded fluorescent indicator called SHRIMP. SHRIMP exhibits high brightness and insensitivity to photoactivation and is a valuable tool for real-time monitoring of H2O2 dynamics in various biological systems. Further research may yield an expanded family of cpmScarlet-based red fluorescent indicators with enhanced photophysical properties.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.553046">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.553046" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.553046">
        <p class="paperTitle">Targeted Protein Degradation through Recruitment of the CUL4A Complex Adaptor Protein DDB1</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.553046" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.553046" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Meyers, M.; Cismoski, S.; Panidapu, A.; Chie-Leon, B.; Nomura, D. K.</p>
        <p class="info">Score: 10.3, Published: 2023-08-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.553046' target='https://doi.org/10.1101/2023.08.11.553046'> 10.1101/2023.08.11.553046</a></p>
        <p class="abstract">Targeted protein degradation has arisen as a powerful therapeutic modality for eliminating proteins. Thus far, most heterobifunctional Proteolysis Targeting Chimeras (PROTACs) have utilized recruiters against substrate receptors of Cullin RING E3 ubiquitin ligases, such as cereblon and VHL. However, previous studies have surprisingly uncovered molecular glue degraders that exploit a CUL4A adaptor protein DDB1 to degrade neosubstrate proteins. Here, we sought to investigate whether DDB1 recruiters can be discovered that can be exploited for PROTAC applications. We utilized activity-based protein profiling and cysteine chemoproteomic screening to identify a covalent recruiter that targets C173 on DDB1 and exploited this recruiter to develop PROTACs against BRD4 and androgen receptor (AR). We demonstrated that the BRD4 PROTAC results in selective degradation of the short BRD4 isoform over the long isoform in a proteasome, NEDDylation, and DDB1-dependent manner. We also demonstrated degradation of AR with the AR PROTAC in prostate cancer cells. Our study demonstrated that covalent chemoproteomic approaches can be used to discover recruiters against Cullin RING adapter proteins and that these recruiters can be used for PROTAC applications to degrade neo-substrates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.552005">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.552005" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.552005">
        <p class="paperTitle">Identification of factors that prevent ribosome stalling during early elongation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.552005" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.552005" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hong, H.-R.; Prince, C. R.; Wu, L.; Feaga, H. A.</p>
        <p class="info">Score: 29.1, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.552005' target='https://doi.org/10.1101/2023.08.04.552005'> 10.1101/2023.08.04.552005</a></p>
        <p class="abstract">Protein synthesis is catalyzed by the ribosome and a host of highly conserved elongation factors. Most elongation factors that are conserved in all domains of life are essential, such as EF-Tu (e/aEF1A) and EF-G (e/aEF2). In contrast, the universally conserved elongation factor P (EF-P/eIF5A) is essential in eukaryotes but is dispensable in most bacteria. EF-P prevents ribosome stalling at difficult-to translate sequences, especially polyprolines. Since efp deletion phenotypes range from modest to lethal in different bacterial species, we hypothesized that some bacteria encode an uncharacterized elongation factor with compensatory functions. To identify this factor, we used Tn-seq to screen for genes that are essential in the absence of EF-P in Bacillus subtilis. This screen identified YfmR, a member of the ABCF family of ATPases, as a translation factor that is essential when efp is deleted. We find that depleting YfmR from {Delta}efp cells decreases actively translating ribosomes and increases free ribosomal subunits bound to initiator tRNA, suggesting that ribosomes stall in early elongation. We also find that deleting efp from the spore-forming pathogen Bacillus anthracis causes a severe survival defect in liquid culture and in the presence of macrophages. B. anthracis {Delta}efp also does not produce detectable spores. We find that heterologous expression of B. subtilis YfmR in B. anthracis {Delta}efp cells partially rescues the severe growth and sporulation defects of this mutant. Our results therefore identify YfmR as an important translation factor that becomes essential in the absence of EF-P, and suggest that the essential function of YfmR and EF-P is to assist translation during early elongation.

SignificanceTranslation is one of the most ancient and energetically demanding process that occurs in the cell. Ribosomes constitute more than 60% of cellular mass in actively growing cells, and ribosomes are a major target of antimicrobials and chemotherapeutics. Here, we report evidence that both EF-P and the uncharacterized protein YfmR are important for early translation elongation in bacteria, and that this activity is essential. YfmR is a member of the ABCF family of ATPases whose role in translation is only beginning to be understood. Given the broad distribution of ABCFs from bacteria to fungi, we expect our results to have implications for understanding the crucial first steps of translation elongation in diverse organisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.551727">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.551727" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.551727">
        <p class="paperTitle">Fucosylated glycoproteins and fucosylated glycolipids play opposing roles in cholera intoxication</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.551727" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.551727" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ghorashi, A. C.; Boucher, A.; Archer-hartmann, S. A.; Murray, N. B.; Konada, R. S. R.; Zhang, X.; Xing, C.; Azadi, P.; Yrlid, U.; Kohler, J. J.</p>
        <p class="info">Score: 8.3, Published: 2023-08-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.551727' target='https://doi.org/10.1101/2023.08.02.551727'> 10.1101/2023.08.02.551727</a></p>
        <p class="abstract">Cholera toxin (CT) is the etiological agent of cholera. Here we report that multiple classes of fucosylated glycoconjugates function in CT binding and intoxication of intestinal epithelial cells. In Colo205 cells, knockout of B3GNT5, the enzyme required for synthesis of lacto- and neolacto-series glycosphingolipids (GSLs), reduces CT binding but sensitizes cells to intoxication. Overexpressing B3GNT5 to generate more fucosylated GSLs confers protection against intoxication, indicating that fucosylated GSLs act as decoy receptors for CT. Knockout (KO) of B3GALT5 causes increased production of fucosylated O-linked and N-linked glycoproteins, and leads to increased CT binding and intoxication. Knockout of B3GNT5 in B3GALT5 KO cells eliminates production of fucosylated GSLs but increases intoxication, identifying fucosylated glycoproteins as functional receptors for CT. These findings provide insight into molecular determinants regulating CT sensitivity of host cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.12.553095">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.12.553095" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.12.553095">
        <p class="paperTitle">Multi-omic stratification of the missense variant cysteinome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.12.553095" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.12.553095" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Desai, H. S.; Ofori, S.; Boatner, L. M.; Yu, F.; Villaneuva, M.; Ung, N.; Nesvizhskii, A.; Backus, K. M.</p>
        <p class="info">Score: 7.3, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.12.553095' target='https://doi.org/10.1101/2023.08.12.553095'> 10.1101/2023.08.12.553095</a></p>
        <p class="abstract">Cancer genomes are rife with genetic variants; one key outcome of this variation is gain-of-cysteine, which is the most frequently acquired amino acid due to missense variants in COSMIC. Acquired cysteines are both driver mutations and sites targeted by precision therapies. However, despite their ubiquity, nearly all acquired cysteines remain uncharacterized. Here, we pair cysteine chemoproteomics, a technique that enables proteome-wide pinpointing of functional, redox sensitive, and potentially druggable residues, with genomics to reveal the hidden landscape of cysteine acquisition. For both cancer and healthy genomes, we find that cysteine acquisition is a ubiquitous consequence of genetic variation that is further elevated in the context of decreased DNA repair. Our chemoproteogenomics platform integrates chemoproteomic, whole exome, and RNA-seq data, with a customized 2-stage false discovery rate (FDR) error controlled proteomic search, further enhanced with a user-friendly FragPipe interface. Integration of CADD predictions of deleteriousness revealed marked enrichment for likely damaging variants that result in acquisition of cysteine. By deploying chemoproteogenomics across 11 cell lines, we identify 116 gain-of-cysteines, of which 10 were liganded by electrophilic druglike molecules. Reference cysteines proximal to missense variants were also found to be pervasive, 791 in total, supporting heretofore untapped opportunities for proteoform-specific chemical probe development campaigns. As chemoproteogenomics is further distinguished by sample-matched combinatorial variant databases and compatible with redox proteomics and small molecule screening, we expect widespread utility in guiding proteoform-specific biology and therapeutic discovery.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.552905">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.552905" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.552905">
        <p class="paperTitle">Bifunctional Monosaccharides Preferentially Localize to Nuclear Subcompartments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.552905" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.552905" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Barahtjan, P.; Iglesias-Artola, J. M.; Boehlig, K.; Lohmann, A.; Nadler, A.</p>
        <p class="info">Score: 7.3, Published: 2023-08-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.552905' target='https://doi.org/10.1101/2023.08.11.552905'> 10.1101/2023.08.11.552905</a></p>
        <p class="abstract">Recent progress in glycan research has been driven by widespread implementations of metabolic oligosaccharide engineering. Complementing existing approaches, we here introduce bifunctional, UV-crosslinkable and clickable N-acetylglucosamine and N-acetylgalactosamine analogues, which enable direct visualization of the intracellular probe distribution as well as distinguishing monomeric and macromolecule-bound fractions. Using this feature, we find that monomeric N-acetylmonosaccharides partition into RNA-rich nuclear compartments such as nuclear speckles and nucleoli. This suggests the existence of spatially separated N-acetylmonosccharide pools within the nucleoplasm. Taken together, bifunctional N-acetylmonosaccharide probes are a powerful discovery tool for probing intracellular localization of monosaccharides and glycosylated macromolecules.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
