<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="The potential of plasma HLA peptides beyond neoepitopes
Authors: Wahle, M.; Thielert, M.; Zwiebel, M.; Skowronek, P.; Zeng, W.-F.; Mann, M.
Score: 27.3, Published: 2023-09-05 DOI: 10.1101/2023.09.05.556309
Distinction of non-self from self is the major task of the immune system. Immunopeptidomics studies the peptide repertoire presented by the human leukocyte antigen (HLA) protein, usually on tissues. However, HLA peptides are also bound to plasma soluble HLA (sHLA), but little is known about their origin and potential for biomarker discovery in this readily available biofluid.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="The potential of plasma HLA peptides beyond neoepitopes
Authors: Wahle, M.; Thielert, M.; Zwiebel, M.; Skowronek, P.; Zeng, W.-F.; Mann, M.
Score: 27.3, Published: 2023-09-05 DOI: 10.1101/2023.09.05.556309
Distinction of non-self from self is the major task of the immune system. Immunopeptidomics studies the peptide repertoire presented by the human leukocyte antigen (HLA) protein, usually on tissues. However, HLA peptides are also bound to plasma soluble HLA (sHLA), but little is known about their origin and potential for biomarker discovery in this readily available biofluid." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-13T10:39:03+00:00" />
<meta property="article:modified_time" content="2023-09-13T10:39:03+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="The potential of plasma HLA peptides beyond neoepitopes
Authors: Wahle, M.; Thielert, M.; Zwiebel, M.; Skowronek, P.; Zeng, W.-F.; Mann, M.
Score: 27.3, Published: 2023-09-05 DOI: 10.1101/2023.09.05.556309
Distinction of non-self from self is the major task of the immune system. Immunopeptidomics studies the peptide repertoire presented by the human leukocyte antigen (HLA) protein, usually on tissues. However, HLA peptides are also bound to plasma soluble HLA (sHLA), but little is known about their origin and potential for biomarker discovery in this readily available biofluid."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "The potential of plasma HLA peptides beyond neoepitopes\nAuthors: Wahle, M.; Thielert, M.; Zwiebel, M.; Skowronek, P.; Zeng, W.-F.; Mann, M.\nScore: 27.3, Published: 2023-09-05 DOI: 10.1101/2023.09.05.556309\nDistinction of non-self from self is the major task of the immune system. Immunopeptidomics studies the peptide repertoire presented by the human leukocyte antigen (HLA) protein, usually on tissues. However, HLA peptides are also bound to plasma soluble HLA (sHLA), but little is known about their origin and potential for biomarker discovery in this readily available biofluid.",
  "keywords": [
    
  ],
  "articleBody": " The potential of plasma HLA peptides beyond neoepitopes\nAuthors: Wahle, M.; Thielert, M.; Zwiebel, M.; Skowronek, P.; Zeng, W.-F.; Mann, M.\nScore: 27.3, Published: 2023-09-05 DOI: 10.1101/2023.09.05.556309\nDistinction of non-self from self is the major task of the immune system. Immunopeptidomics studies the peptide repertoire presented by the human leukocyte antigen (HLA) protein, usually on tissues. However, HLA peptides are also bound to plasma soluble HLA (sHLA), but little is known about their origin and potential for biomarker discovery in this readily available biofluid. Currently, immunopeptidomics is hampered by complex workflows and limited sensitivity, generally requiring several mL of plasma for the detection of hundreds of HLA peptides. Here, we take advantage of recent improvements in the throughput and sensitivity of mass spectrometry (MS)-based proteomics to develop a highly-sensitive, automated and economical workflow for HLA peptide analysis, termed Immunopeptidomics by Biotinylated Antibodies and Streptavidin (IMBAS). IMBAS-MS quantifies more than 5,000 HLA class I peptides from only 200 L of plasma, in just 30 minutes. Our technology revealed that the plasma immunopeptidome of healthy donors is remarkably stable throughout a year and strongly correlated between individuals with overlapping HLA types. Immunopeptides originating from diverse tissues, including the brain, are proportionately represented. We conclude that sHLAs are a promising avenue for immunology and precision oncology.\nRosetta Energy Analysis of AlphaFold2 models: Point Mutations and Conformational Ensembles\nAuthors: Stein, R. A.; Mchaourab, H. S.\nScore: 25.1, Published: 2023-09-05 DOI: 10.1101/2023.09.05.556364\nAlphaFold2s ability to accurately predict protein structures from a multiple sequence alignment (MSA) has raised many questions about the utility of the models generated in downstream structural analysis. Two outstanding questions are the prediction of the consequences of point mutations and the completeness of the landscape of protein conformational ensembles. We previously developed a method, SPEACH_AF, to obtain alternate conformations by introducing residue substitutions across the MSA and not just within the input sequence. Here, we compared the structural and energetic consequences of having the mutation(s) in the input sequence versus in the whole MSA (SPEACH_AF). Both methods yielded models different from the wild-type sequence, with more robust changes when the mutation(s) were in the whole MSA. To evaluate models of conformational diversity, we used SPEACH_AF and a new MSA subsampling method, AF_cluster, combined with model relaxation in Rosetta. We find that the energetics of the conformations generated by AlphaFold2 correspond to those seen in experimental crystal structures and explored by standard molecular dynamic methods. Combined, the results support the fact that AlphaFold2 can predict structural changes due to point mutations and has learned information about protein structural energetics that are encoded in the MSA.\nDroplet based low input proteomic platform for rare cell populations\nAuthors: Waas, M.; Khoo, A.; Tharmapalan, P.; McCloskey, C. W.; Govindarajan, M.; Zhang, B.; Khan, S.; Waterhouse, P.; Khokha, R.; Kislinger, T.\nScore: 22.4, Published: 2023-09-11 DOI: 10.1101/2023.09.11.557098\nDeep proteomic profiling of rare cell populations has been constrained by sample input requirements. Here, we present DROPPS, an accessible low-input platform that generates high-fidelity proteomic profiles of 100 - 2,500 cells. By applying DROPPS within the mammary epithelium, we elucidated the connection between mitochondrial activity and clonogenicity, discovering and validating CD36 as a marker of progenitor capacity in the basal cell compartment. We anticipate DROPPS will accelerate biology-driven proteomic research for a multitude of rare cell populations.\nAlphaFold2 can predict single-mutation effects\nAuthors: Mcbride, J. M.; Reinharz, V.; Polev, K.; Grzybowski, B. A.; Tlusty, T.\nScore: 124.9, Published: 2023-09-03 DOI: 10.1101/2022.04.14.488301\nAlphaFold2 (AF) is a promising tool, but is it accurate enough to predict single mutation effects? Here, we report that the localized structural deformation between protein pairs differing by only 1-3 mutations - as measured by the effective strain - is correlated across 3,901 experimental and AF-predicted structures. Furthermore, analysis of [~]11,000 proteins shows that the local structural change correlates with various phenotypic changes. These findings suggest that AF can predict the range and magnitude of single-mutation effects on average, and we propose a method to improve precision of AF predictions and to indicate when predictions are unreliable.\nA ubiquitin-specific, proximity-based labeling approach for the identification of ubiquitin ligase substrates\nAuthors: Bhogaraju, S.; Behrends, C.; Mukhopadhyay, U.; Levantovsky, S.; Stein, F.; Gharbi, S.\nScore: 21.8, Published: 2023-09-04 DOI: 10.1101/2023.09.04.556194\nUbiquitination of proteins is central to protein homeostasis and other cellular processes including DNA repair, vesicular transport, cell-division etc. The process of ubiquitination is conserved from yeast to humans and is carried out by the sequential action of three enzymes: E1, E2 and E3. There are an estimated \u003e600 E3 ligases in humans that execute ubiquitination of specific target proteins in a spatio-temporal manner to elicit desired signaling effects. Here, we developed a simple, proximity-based labeling method to selectively biotinylate substrates of a given ubiquitin ligase. Our method exploits the proximity and the relative orientation of the E3-ligase catalytic domain with respect to ubiquitin observed in the enzymatic intermediate-state structures of E3-E2[~]Ub. By fusing the biotin ligase BirA and an Avi tag variant to the candidate E3 ligase and ubiquitin, respectively, we were able to specifically enrich the bona fide substrates and potential new substrates of the ligase using a one-step Streptavidin pulldown under denaturing conditions. As proof-of-principle, we applied our method, which we named Ub-POD, to the RING E3 ligase Rad18. Rad18 ubiquitinates DNA-sliding clamp PCNA upon UV-induced DNA damage. We identified PCNA and several other critical players in the DNA damage repair pathway in a single Ub-POD experiment. We further discovered an unintended but important application of Ub-POD, where we were able to pin down the cellular localization of Rad18-mediated ubiquitination to the damaged DNA nuclear puncta through Streptavidin immunofluorescence. We also applied our method to TRAF6, another RING ubiquitin ligase involved in TNF signaling, successfully identifying known and potentially new substrates. Finally, we adapted our method to the U-box-type E3 ubiquitin ligase CHIP to demonstrate that we can identify substrates of two major classes of mammalian ubiquitin ligases. We anticipate that our method and principle could be widely adapted to all classes of ubiquitin ligases to identify substrates and localize the cellular site(s) of ubiquitination.\nStructural Mechanisms for VMAT2 inhibition by tetrabenazine\nAuthors: Dalton, M. P.; Cheng, M. H.; Bahar, I.; Coleman, J. A.\nScore: 12.9, Published: 2023-09-05 DOI: 10.1101/2023.09.05.556211\nThe vesicular monoamine transporter 2 (VMAT2) is a proton-dependent antiporter responsible for loading monoamine neurotransmitters into synaptic vesicles. Dysregulation of VMAT2 can lead to several neuropsychiatric disorders including Parkinsons disease and schizophrenia. Furthermore, drugs such as amphetamine and MDMA are known to act on VMAT2, exemplifying its role in the mechanisms of actions for drugs of abuse. Despite VMAT2s importance, there remains a critical lack of mechanistic understanding, largely driven by a lack of structural information. Here we report a 3.3 [A] resolution cryo-EM structure of VMAT2 complexed with TBZ, a non-competitive inhibitor used in the treatment of Huntingtons chorea. We find TBZ interacts with residues in a central binding site, locking VMAT2 in an occluded conformation and providing a mechanistic basis for non-competitive inhibition. We further identify residues critical for intracellular and luminal gating, including a cluster of hydrophobic residues which are involved in a luminal gating strategy. Our structure also highlights three distinct polar networks that may determine VMAT2 conformational change and play a role in proton transduction. The structure elucidates mechanisms of VMAT2 inhibition and transport, providing insights into VMAT2 architecture, function, and the design of small-molecule therapeutics.\nNegative allosteric modulation of the μ-opioid receptor\nAuthors: O'Brien, E. S.; Rangari, V. A.; El Daibani, A.; Eans, S. O.; White, B.; Wang, H.; Shiimura, Y.; Krishna Kumar, K.; Appourchaux, K.; Huang, W.; Zhang, C.; Mathiesen, J. M.; Che, T.; McLaughlin, J. P.; Majumdar, S.; Kobilka, B. K.\nScore: 12.4, Published: 2023-09-10 DOI: 10.1101/2023.09.08.556921\nThe opioid receptor (OR) is a well-established target for analgesia, yet conventional opioid receptor agonists cause serious adverse effects, notably addiction and respiratory depression, which have led to the present opioid overdose epidemic. OR negative allosteric modulators (NAMs) may serve as powerful tools in preventing opioid overdose deaths, but promising chemical scaffolds remain elusive. We screened a large DNA-encoded chemical library against inactive OR, counter-screening with active, G-protein and agonist bound receptor to steer selections toward functional negative allosteric modulators. We discovered a NAM compound with high and selective enrichment to inactive OR; the molecule potently blocks the activity of orthosteric agonists and enhances the affinity of the key opioid overdose reversal molecule, naloxone. It accomplishes this by binding to a site on the extracellular vestibule proximal to naloxone, stabilizing a unique inactive conformation of the extracellular portions of the second and seventh transmembrane helices. The NAM perturbs orthosteric ligand kinetics in therapeutically desirable ways and works cooperatively with low doses of naloxone in vivo to inhibit morphine-induced antinociception, respiratory depression and conditioned place preference while minimizing withdrawal behaviors. Our results provide detailed structural insights into the mechanism of a negative allosteric modulator for the OR, and demonstrate how it can be exploited in vivo.\nConformational changes in the Niemann-Pick Type C1 protein NCR1 drive sterol translocation\nAuthors: Frain, K. M.; Dedic, E.; Nel, L.; Bohush, A.; Olesen, E.; Stokes, D. L.; Pedersen, B. P.\nScore: 9.0, Published: 2023-09-08 DOI: 10.1101/2023.09.08.556848\nThe membrane protein Niemann-Pick Type C1 protein (NPC1, named NCR1 in yeast) is central to sterol homeostasis in eukaryotes. Saccharomyces cerevisiae NCR1 is localized to the vacuolar membrane, where it is suggested to carry sterols across the protective glycocalyx and deposit them into the vacuolar membrane. However, documentation of a vacuolar glycocalyx in fungi is lacking and the mechanism for sterol translocation has remained unclear. Here we provide evidence that a glycocalyx is indeed present inside isolated Saccharomyces cerevisiae vacuoles, and report four cryo-EM structures of NCR1 in two distinct conformations that elucidate how it moves sterol through the glycocalyx. The two conformations, named 'tense' and 'relaxed', illustrate movement of sterol through a tunnel formed by the luminal domains. Based on these structures and on comparison with other members of the Resistance-Nodulation-Division (RND) superfamily we propose a transport model that links changes in the luminal domains with a cycle of protonation and deprotonation within the transmembrane region of the protein. Our model suggests that NPC proteins work by a generalized RND mechanism where the transmembrane domains form a 'motor-unit' that sequentially adopts a tense and relaxed conformation to drive changes in luminal/extracellular domains.\nMachine learning to classify mutational hotspots from molecular dynamics simulations\nAuthors: Davies, J.; Menzies, G. E.\nScore: 8.5, Published: 2023-09-10 DOI: 10.1101/2023.09.07.556625\nBenzo(a)pyrene, a notorious DNA-damaging carcinogen, belongs to the family of polycyclic aromatic hydrocarbons commonly found in tobacco smoke. Surprisingly, nucleotide excision repair (NER) machinery exhibits inefficiency in recognising specific bulky DNA adducts including Benzo(a)pyrene Diol-Epoxide (BPDE), a Benzo(a)pyrene metabolite. While sequence context is emerging as the leading factor linking the inadequate NER response to BPDE adducts, the precise structural attributes governing these disparities remain inadequately understood. We therefore combined the domains of molecular dynamics and machine learning to conduct a comprehensive assessment of helical distortion caused by BPDE-Guanine adducts in multiple gene contexts. Specifically, we implemented a dual approach involving a random forest classification-based analysis and subsequent feature selection to identify precise topological features that may distinguish adduct sites of variable repair capacity. Our models were trained using helical data extracted from duplexes representing both BPDE hotspot and non-hotspot sites within the TP53 gene, then applied to sites within TP53, cII, and lacZ genes. We show our optimised model consistently achieved exceptional performance, with accuracy, precision, and f1 scores exceeding 91%. Our feature selection approach uncovered that discernible variance in regional base pair rotation played a pivotal role in informing the decisions of our model. Notably, these disparities were highly conserved among TP53 and lacZ duplexes and appeared to be influenced by the regional GC content. As such, our findings suggest that there are indeed conserved topological features distinguishing hotspots and non-hotpot sites, highlighting regional GC content as a potential biomarker for mutation.\nFaa1 membrane binding drives positive feedback in autophagosome biogenesis via fatty acid activation\nAuthors: Baumann, V.; Achleitner, S.; Tulli, S.; Schuschnig, M.; Klune, L.; Martens, S.\nScore: 8.3, Published: 2023-09-09 DOI: 10.1101/2023.09.09.556965\nAutophagy serves as a stress response pathway by mediating the degradation of cellular material within lysosomes. In autophagy this material is encapsulated in double membrane vesicles termed autophagosomes, which form from precursors referred to as phagophores. Phagophores grow by lipid influx from the endoplasmic reticulum into Atg9-positive compartments and local lipid synthesis provides lipids for their expansion. How phagophore nucleation and expansion are coordinated with lipid synthesis is unclear. Here, we show that Faa1, an enzyme activating fatty acids, is directly recruited to Atg9 vesicles. We further show that Faa1 binds to negatively charged membranes. We define the membrane binding surface in Faa1 and show that membrane binding is required for its enzymatic activity. In cells, membrane binding by Faa1 is required for its recruitment to phagophores and promotes autophagosome biogenesis. Our results suggest a positive feedback loop coupling phagophore nucleation and expansion to lipid synthesis.\n",
  "wordCount" : "2156",
  "inLanguage": "en",
  "datePublished": "2023-09-13T10:39:03Z",
  "dateModified": "2023-09-13T10:39:03Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta"><span>updated on September 13, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.05.556309">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.05.556309" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.05.556309">
        <p class="paperTitle">The potential of plasma HLA peptides beyond neoepitopes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.05.556309" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.05.556309" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wahle, M.; Thielert, M.; Zwiebel, M.; Skowronek, P.; Zeng, W.-F.; Mann, M.</p>
        <p class="info">Score: 27.3, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.05.556309' target='https://doi.org/10.1101/2023.09.05.556309'> 10.1101/2023.09.05.556309</a></p>
        <p class="abstract">Distinction of non-self from self is the major task of the immune system. Immunopeptidomics studies the peptide repertoire presented by the human leukocyte antigen (HLA) protein, usually on tissues. However, HLA peptides are also bound to plasma soluble HLA (sHLA), but little is known about their origin and potential for biomarker discovery in this readily available biofluid. Currently, immunopeptidomics is hampered by complex workflows and limited sensitivity, generally requiring several mL of plasma for the detection of hundreds of HLA peptides. Here, we take advantage of recent improvements in the throughput and sensitivity of mass spectrometry (MS)-based proteomics to develop a highly-sensitive, automated and economical workflow for HLA peptide analysis, termed Immunopeptidomics by Biotinylated Antibodies and Streptavidin (IMBAS). IMBAS-MS quantifies more than 5,000 HLA class I peptides from only 200 L of plasma, in just 30 minutes. Our technology revealed that the plasma immunopeptidome of healthy donors is remarkably stable throughout a year and strongly correlated between individuals with overlapping HLA types. Immunopeptides originating from diverse tissues, including the brain, are proportionately represented. We conclude that sHLAs are a promising avenue for immunology and precision oncology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.05.556364">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.05.556364" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.05.556364">
        <p class="paperTitle">Rosetta Energy Analysis of AlphaFold2 models: Point Mutations and Conformational Ensembles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.05.556364" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.05.556364" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stein, R. A.; Mchaourab, H. S.</p>
        <p class="info">Score: 25.1, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.05.556364' target='https://doi.org/10.1101/2023.09.05.556364'> 10.1101/2023.09.05.556364</a></p>
        <p class="abstract">AlphaFold2s ability to accurately predict protein structures from a multiple sequence alignment (MSA) has raised many questions about the utility of the models generated in downstream structural analysis. Two outstanding questions are the prediction of the consequences of point mutations and the completeness of the landscape of protein conformational ensembles. We previously developed a method, SPEACH_AF, to obtain alternate conformations by introducing residue substitutions across the MSA and not just within the input sequence. Here, we compared the structural and energetic consequences of having the mutation(s) in the input sequence versus in the whole MSA (SPEACH_AF). Both methods yielded models different from the wild-type sequence, with more robust changes when the mutation(s) were in the whole MSA. To evaluate models of conformational diversity, we used SPEACH_AF and a new MSA subsampling method, AF_cluster, combined with model relaxation in Rosetta. We find that the energetics of the conformations generated by AlphaFold2 correspond to those seen in experimental crystal structures and explored by standard molecular dynamic methods. Combined, the results support the fact that AlphaFold2 can predict structural changes due to point mutations and has learned information about protein structural energetics that are encoded in the MSA.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557098">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557098" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557098">
        <p class="paperTitle">Droplet based low input proteomic platform for rare cell populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557098" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557098" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Waas, M.; Khoo, A.; Tharmapalan, P.; McCloskey, C. W.; Govindarajan, M.; Zhang, B.; Khan, S.; Waterhouse, P.; Khokha, R.; Kislinger, T.</p>
        <p class="info">Score: 22.4, Published: 2023-09-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557098' target='https://doi.org/10.1101/2023.09.11.557098'> 10.1101/2023.09.11.557098</a></p>
        <p class="abstract">Deep proteomic profiling of rare cell populations has been constrained by sample input requirements. Here, we present DROPPS, an accessible low-input platform that generates high-fidelity proteomic profiles of 100 - 2,500 cells. By applying DROPPS within the mammary epithelium, we elucidated the connection between mitochondrial activity and clonogenicity, discovering and validating CD36 as a marker of progenitor capacity in the basal cell compartment. We anticipate DROPPS will accelerate biology-driven proteomic research for a multitude of rare cell populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.04.14.488301">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.04.14.488301" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.04.14.488301">
        <p class="paperTitle">AlphaFold2 can predict single-mutation effects</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.04.14.488301" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.04.14.488301" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mcbride, J. M.; Reinharz, V.; Polev, K.; Grzybowski, B. A.; Tlusty, T.</p>
        <p class="info">Score: 124.9, Published: 2023-09-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.04.14.488301' target='https://doi.org/10.1101/2022.04.14.488301'> 10.1101/2022.04.14.488301</a></p>
        <p class="abstract">AlphaFold2 (AF) is a promising tool, but is it accurate enough to predict single mutation effects? Here, we report that the localized structural deformation between protein pairs differing by only 1-3 mutations - as measured by the effective strain - is correlated across 3,901 experimental and AF-predicted structures. Furthermore, analysis of [~]11,000 proteins shows that the local structural change correlates with various phenotypic changes. These findings suggest that AF can predict the range and magnitude of single-mutation effects on average, and we propose a method to improve precision of AF predictions and to indicate when predictions are unreliable.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.04.556194">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.04.556194" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.04.556194">
        <p class="paperTitle">A ubiquitin-specific, proximity-based labeling approach for the identification of ubiquitin ligase substrates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.04.556194" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.04.556194" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bhogaraju, S.; Behrends, C.; Mukhopadhyay, U.; Levantovsky, S.; Stein, F.; Gharbi, S.</p>
        <p class="info">Score: 21.8, Published: 2023-09-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.04.556194' target='https://doi.org/10.1101/2023.09.04.556194'> 10.1101/2023.09.04.556194</a></p>
        <p class="abstract">Ubiquitination of proteins is central to protein homeostasis and other cellular processes including DNA repair, vesicular transport, cell-division etc. The process of ubiquitination is conserved from yeast to humans and is carried out by the sequential action of three enzymes: E1, E2 and E3. There are an estimated &gt;600 E3 ligases in humans that execute ubiquitination of specific target proteins in a spatio-temporal manner to elicit desired signaling effects. Here, we developed a simple, proximity-based labeling method to selectively biotinylate substrates of a given ubiquitin ligase. Our method exploits the proximity and the relative orientation of the E3-ligase catalytic domain with respect to ubiquitin observed in the enzymatic intermediate-state structures of E3-E2[~]Ub. By fusing the biotin ligase BirA and an Avi tag variant to the candidate E3 ligase and ubiquitin, respectively, we were able to specifically enrich the bona fide substrates and potential new substrates of the ligase using a one-step Streptavidin pulldown under denaturing conditions. As proof-of-principle, we applied our method, which we named Ub-POD, to the RING E3 ligase Rad18. Rad18 ubiquitinates DNA-sliding clamp PCNA upon UV-induced DNA damage. We identified PCNA and several other critical players in the DNA damage repair pathway in a single Ub-POD experiment. We further discovered an unintended but important application of Ub-POD, where we were able to pin down the cellular localization of Rad18-mediated ubiquitination to the damaged DNA nuclear puncta through Streptavidin immunofluorescence. We also applied our method to TRAF6, another RING ubiquitin ligase involved in TNF signaling, successfully identifying known and potentially new substrates. Finally, we adapted our method to the U-box-type E3 ubiquitin ligase CHIP to demonstrate that we can identify substrates of two major classes of mammalian ubiquitin ligases. We anticipate that our method and principle could be widely adapted to all classes of ubiquitin ligases to identify substrates and localize the cellular site(s) of ubiquitination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.05.556211">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.05.556211" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.05.556211">
        <p class="paperTitle">Structural Mechanisms for VMAT2 inhibition by tetrabenazine</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.05.556211" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.05.556211" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dalton, M. P.; Cheng, M. H.; Bahar, I.; Coleman, J. A.</p>
        <p class="info">Score: 12.9, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.05.556211' target='https://doi.org/10.1101/2023.09.05.556211'> 10.1101/2023.09.05.556211</a></p>
        <p class="abstract">The vesicular monoamine transporter 2 (VMAT2) is a proton-dependent antiporter responsible for loading monoamine neurotransmitters into synaptic vesicles. Dysregulation of VMAT2 can lead to several neuropsychiatric disorders including Parkinsons disease and schizophrenia.

Furthermore, drugs such as amphetamine and MDMA are known to act on VMAT2, exemplifying its role in the mechanisms of actions for drugs of abuse. Despite VMAT2s importance, there remains a critical lack of mechanistic understanding, largely driven by a lack of structural information. Here we report a 3.3 [A] resolution cryo-EM structure of VMAT2 complexed with TBZ, a non-competitive inhibitor used in the treatment of Huntingtons chorea. We find TBZ interacts with residues in a central binding site, locking VMAT2 in an occluded conformation and providing a mechanistic basis for non-competitive inhibition. We further identify residues critical for intracellular and luminal gating, including a cluster of hydrophobic residues which are involved in a luminal gating strategy. Our structure also highlights three distinct polar networks that may determine VMAT2 conformational change and play a role in proton transduction. The structure elucidates mechanisms of VMAT2 inhibition and transport, providing insights into VMAT2 architecture, function, and the design of small-molecule therapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556921">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556921" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556921">
        <p class="paperTitle">Negative allosteric modulation of the μ-opioid receptor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556921" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556921" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: O&#39;Brien, E. S.; Rangari, V. A.; El Daibani, A.; Eans, S. O.; White, B.; Wang, H.; Shiimura, Y.; Krishna Kumar, K.; Appourchaux, K.; Huang, W.; Zhang, C.; Mathiesen, J. M.; Che, T.; McLaughlin, J. P.; Majumdar, S.; Kobilka, B. K.</p>
        <p class="info">Score: 12.4, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556921' target='https://doi.org/10.1101/2023.09.08.556921'> 10.1101/2023.09.08.556921</a></p>
        <p class="abstract">The  opioid receptor (OR) is a well-established target for analgesia, yet conventional opioid receptor agonists cause serious adverse effects, notably addiction and respiratory depression, which have led to the present opioid overdose epidemic. OR negative allosteric modulators (NAMs) may serve as powerful tools in preventing opioid overdose deaths, but promising chemical scaffolds remain elusive. We screened a large DNA-encoded chemical library against inactive OR, counter-screening with active, G-protein and agonist bound receptor to steer selections toward functional negative allosteric modulators. We discovered a NAM compound with high and selective enrichment to inactive OR; the molecule potently blocks the activity of orthosteric agonists and enhances the affinity of the key opioid overdose reversal molecule, naloxone. It accomplishes this by binding to a site on the extracellular vestibule proximal to naloxone, stabilizing a unique inactive conformation of the extracellular portions of the second and seventh transmembrane helices. The NAM perturbs orthosteric ligand kinetics in therapeutically desirable ways and works cooperatively with low doses of naloxone in vivo to inhibit morphine-induced antinociception, respiratory depression and conditioned place preference while minimizing withdrawal behaviors. Our results provide detailed structural insights into the mechanism of a negative allosteric modulator for the OR, and demonstrate how it can be exploited in vivo.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556848">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556848" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556848">
        <p class="paperTitle">Conformational changes in the Niemann-Pick Type C1 protein NCR1 drive sterol translocation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556848" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556848" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Frain, K. M.; Dedic, E.; Nel, L.; Bohush, A.; Olesen, E.; Stokes, D. L.; Pedersen, B. P.</p>
        <p class="info">Score: 9.0, Published: 2023-09-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556848' target='https://doi.org/10.1101/2023.09.08.556848'> 10.1101/2023.09.08.556848</a></p>
        <p class="abstract">The membrane protein Niemann-Pick Type C1 protein (NPC1, named NCR1 in yeast) is central to sterol homeostasis in eukaryotes. Saccharomyces cerevisiae NCR1 is localized to the vacuolar membrane, where it is suggested to carry sterols across the protective glycocalyx and deposit them into the vacuolar membrane. However, documentation of a vacuolar glycocalyx in fungi is lacking and the mechanism for sterol translocation has remained unclear. Here we provide evidence that a glycocalyx is indeed present inside isolated Saccharomyces cerevisiae vacuoles, and report four cryo-EM structures of NCR1 in two distinct conformations that elucidate how it moves sterol through the glycocalyx. The two conformations, named &#39;tense&#39; and &#39;relaxed&#39;, illustrate movement of sterol through a tunnel formed by the luminal domains. Based on these structures and on comparison with other members of the Resistance-Nodulation-Division (RND) superfamily we propose a transport model that links changes in the luminal domains with a cycle of protonation and deprotonation within the transmembrane region of the protein. Our model suggests that NPC proteins work by a generalized RND mechanism where the transmembrane domains form a &#39;motor-unit&#39; that sequentially adopts a tense and relaxed conformation to drive changes in luminal/extracellular domains.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556625">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556625" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556625">
        <p class="paperTitle">Machine learning to classify mutational hotspots from molecular dynamics simulations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556625" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556625" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Davies, J.; Menzies, G. E.</p>
        <p class="info">Score: 8.5, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556625' target='https://doi.org/10.1101/2023.09.07.556625'> 10.1101/2023.09.07.556625</a></p>
        <p class="abstract">Benzo(a)pyrene, a notorious DNA-damaging carcinogen, belongs to the family of polycyclic aromatic hydrocarbons commonly found in tobacco smoke. Surprisingly, nucleotide excision repair (NER) machinery exhibits inefficiency in recognising specific bulky DNA adducts including Benzo(a)pyrene Diol-Epoxide (BPDE), a Benzo(a)pyrene metabolite. While sequence context is emerging as the leading factor linking the inadequate NER response to BPDE adducts, the precise structural attributes governing these disparities remain inadequately understood. We therefore combined the domains of molecular dynamics and machine learning to conduct a comprehensive assessment of helical distortion caused by BPDE-Guanine adducts in multiple gene contexts. Specifically, we implemented a dual approach involving a random forest classification-based analysis and subsequent feature selection to identify precise topological features that may distinguish adduct sites of variable repair capacity. Our models were trained using helical data extracted from duplexes representing both BPDE hotspot and non-hotspot sites within the TP53 gene, then applied to sites within TP53, cII, and lacZ genes. We show our optimised model consistently achieved exceptional performance, with accuracy, precision, and f1 scores exceeding 91%. Our feature selection approach uncovered that discernible variance in regional base pair rotation played a pivotal role in informing the decisions of our model. Notably, these disparities were highly conserved among TP53 and lacZ duplexes and appeared to be influenced by the regional GC content. As such, our findings suggest that there are indeed conserved topological features distinguishing hotspots and non-hotpot sites, highlighting regional GC content as a potential biomarker for mutation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.09.556965">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.09.556965" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.09.556965">
        <p class="paperTitle">Faa1 membrane binding drives positive feedback in autophagosome biogenesis via fatty acid activation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.09.556965" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.09.556965" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Baumann, V.; Achleitner, S.; Tulli, S.; Schuschnig, M.; Klune, L.; Martens, S.</p>
        <p class="info">Score: 8.3, Published: 2023-09-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.09.556965' target='https://doi.org/10.1101/2023.09.09.556965'> 10.1101/2023.09.09.556965</a></p>
        <p class="abstract">Autophagy serves as a stress response pathway by mediating the degradation of cellular material within lysosomes. In autophagy this material is encapsulated in double membrane vesicles termed autophagosomes, which form from precursors referred to as phagophores. Phagophores grow by lipid influx from the endoplasmic reticulum into Atg9-positive compartments and local lipid synthesis provides lipids for their expansion. How phagophore nucleation and expansion are coordinated with lipid synthesis is unclear. Here, we show that Faa1, an enzyme activating fatty acids, is directly recruited to Atg9 vesicles. We further show that Faa1 binds to negatively charged membranes. We define the membrane binding surface in Faa1 and show that membrane binding is required for its enzymatic activity. In cells, membrane binding by Faa1 is required for its recruitment to phagophores and promotes autophagosome biogenesis. Our results suggest a positive feedback loop coupling phagophore nucleation and expansion to lipid synthesis.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
