<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Targeted protein editing with an antibody-based system
Authors: Rimon, O.; Konc, J.; Roy Chowdhury, V.; Bernardes, G. J. L.; Vendruscolo, M.
Score: 18.8, Published: 2023-07-29 DOI: 10.1101/2023.07.28.551006
The chemical modification of proteins is one of the major mechanisms used to regulate the trafficking and function of these macromolecules in the cell. It is therefore of great interest to develop tools to exploit this type of modifications for applications in molecular biology, medicine and biotechnology.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Targeted protein editing with an antibody-based system
Authors: Rimon, O.; Konc, J.; Roy Chowdhury, V.; Bernardes, G. J. L.; Vendruscolo, M.
Score: 18.8, Published: 2023-07-29 DOI: 10.1101/2023.07.28.551006
The chemical modification of proteins is one of the major mechanisms used to regulate the trafficking and function of these macromolecules in the cell. It is therefore of great interest to develop tools to exploit this type of modifications for applications in molecular biology, medicine and biotechnology." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-30T10:36:57+00:00" />
<meta property="article:modified_time" content="2023-07-30T10:36:57+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Targeted protein editing with an antibody-based system
Authors: Rimon, O.; Konc, J.; Roy Chowdhury, V.; Bernardes, G. J. L.; Vendruscolo, M.
Score: 18.8, Published: 2023-07-29 DOI: 10.1101/2023.07.28.551006
The chemical modification of proteins is one of the major mechanisms used to regulate the trafficking and function of these macromolecules in the cell. It is therefore of great interest to develop tools to exploit this type of modifications for applications in molecular biology, medicine and biotechnology."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Targeted protein editing with an antibody-based system\nAuthors: Rimon, O.; Konc, J.; Roy Chowdhury, V.; Bernardes, G. J. L.; Vendruscolo, M.\nScore: 18.8, Published: 2023-07-29 DOI: 10.1101/2023.07.28.551006\nThe chemical modification of proteins is one of the major mechanisms used to regulate the trafficking and function of these macromolecules in the cell. It is therefore of great interest to develop tools to exploit this type of modifications for applications in molecular biology, medicine and biotechnology.",
  "keywords": [
    
  ],
  "articleBody": " Targeted protein editing with an antibody-based system\nAuthors: Rimon, O.; Konc, J.; Roy Chowdhury, V.; Bernardes, G. J. L.; Vendruscolo, M.\nScore: 18.8, Published: 2023-07-29 DOI: 10.1101/2023.07.28.551006\nThe chemical modification of proteins is one of the major mechanisms used to regulate the trafficking and function of these macromolecules in the cell. It is therefore of great interest to develop tools to exploit this type of modifications for applications in molecular biology, medicine and biotechnology. Here we present a method of using antibodies to perform post-translational covalent modifications of endogenous proteins in complex environments by exploiting proximity-driven chemistry. The method is based on the ability of antibodies to hold a weakly reactive group adjacently to its intended site of reaction by binding the target protein on a nearby epitope. We illustrate this approach by targeting the green fluorescent protein in increasingly complex environments.\nCoenzyme Q4 is a functional substitute for coenzyme Q10 and can be targeted to the mitochondria\nAuthors: Steenberge, L. H.; Sung, A. Y.; Fan, J.; Pagliarini, D. J.\nScore: 26.7, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549963\nCoenzyme Q10 (CoQ10) is an important cofactor and antioxidant for numerous cellular processes, and its deficiency has been linked to human disorders including mitochondrial disease, heart failure, Parkinsons disease, and hypertension. Unfortunately, treatment with exogenous oral CoQ10 is often ineffective, likely due to the extreme hydrophobicity and high molecular weight of CoQ10. Here, we show that less hydrophobic CoQ species with shorter isoprenoid tails can serve as viable substitutes for CoQ10 in human cells. We demonstrate that CoQ4 can perform multiple functions of CoQ10 in CoQ-deficient cells at markedly lower treatment concentrations, motivating further investigation of CoQ4 as a supplement for CoQ10 deficiencies. In addition, we describe the synthesis and evaluation of an initial set of compounds designed to target CoQ4 selectively to mitochondria using triphenylphosphonium (TPP). Our results indicate that select versions of these compounds can successfully be delivered to mitochondria in a cell model and be cleaved to produce CoQ4, laying the groundwork for further development.\nStructural characterisation of the complete cycle of sliding clamp loading in Escherichia coli\nAuthors: Xu, Z.-Q.; Jergic, S.; Lo, A. T. Y.; Pradhan, A. C.; Brown, S. H. J.; Bouwer, J. C.; Ghodke, H.; Lewis, P. J.; Tolun, G.; Oakley, A. J.; Dixon, N. E.\nScore: 11.3, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549978\nRing-shaped DNA sliding clamps are essential for DNA replication and genome maintenance. Clamps need to be opened or trapped open and chaperoned onto DNA by clamp loader complexes (CLCs). Detailed understanding of the mechanisms by which CLCs open and place clamps around DNA remains limited. Here, we present a series of six structures of the Escherichia coli CLC bound to an open or closed clamp on and off a primer-template DNA that represent all intermediates in the clamp loading process. We show that the ATP-bound CLC first binds to a clamp, then constricts to hold onto it. The CLC then expands to open the clamp with a gap large enough for double-stranded DNA to enter. Upon binding to DNA, the CLC constricts slightly, allowing ATP hydrolysis and clamp closing around DNA. Although both yeast and E. coli CLCs open clamps by crab claw-like motions, they do it by the CLC expanding in opposite directions. These structures provide critical high-resolution snapshots of clamp loading by the E. coli CLC, revealing how the molecular machine works.\nShared Structural Features of Miro Binding Control Mitochondrial Homeostasis\nAuthors: Covill-Cooke, C.; Kwizera, B.; Lopez-Domenech, G.; Thompson, C. O. D.; Cheung, N. J.; Cerezo, E.; Peterka, M.; Kittler, J. T.; Kornmann, B.\nScore: 9.7, Published: 2023-07-25 DOI: 10.1101/2023.07.25.550462\nMiro proteins are universally conserved mitochondrial calcium-binding GTPases that regulate a multitude of mitochondrial processes, including transport, clearance and lipid trafficking. Miro binds a variety of client proteins involved in these functions. How this binding is operated at the molecular level and whether and how it is important for mitochondrial health, however, remains unknown. Here, we show that known Miro clients all use a similar short motif to bind the same structural element: a highly conserved hydrophobic pocket in the calcium-binding domain of Miro. Using these Miro-binding motifs, we identified direct interactors de novo, including yeast Mdm34, and mammalian MTFR1/2/1L, VPS13D and Parkin. Given the shared binding mechanism and conservation across eukaryotes, we propose that Miro is a universal mitochondrial adaptor coordinating mitochondrial health. One-Sentence SummaryFunctionally diverse mitochondrial proteins interact with a conserved hydrophobic pocket on the calcium-binding Miro-GTPases.\nThe hGID GID4 E3 ubiquitin ligase complex targets ARHGAP11A to regulate cell migration\nAuthors: Bagci, H.; Winkler, M.; Uliana, F.; Boulais, J.; Mohamed, W. I.; Park, S. L.; Cote, J.-F.; Peter, M.\nScore: 8.5, Published: 2023-07-20 DOI: 10.1101/2023.07.20.549906\nThe human CTLH/GID (hGID) complex emerged as an important E3 ligase regulating multiple cellular processes, including cell cycle progression and metabolic activity. However, the range of biological functions controlled by hGID remains unexplored. Here, we show that the hGID substrate receptor GID4 regulates cell growth and migration. Biochemical and cellular assays combined with proximity-dependent biotinylation (BioID2) revealed that the hGIDGID4 E3-ligase targets the Rho-GAP ARHGAP11A for degradation. Depletion of GID4 or impeding the GID4 substrate binding pocket impairs motility and directed cell movement, whereas knockdown of ARHGAP11A significantly restores the cell migration defect. We found that GID4 controls cell migration by degrading ARHGAP11A thereby preventing its accumulation at the cell periphery where it inactivates RhoA activity. Together, we identified a unique function for GID4, as well as a wide range of substrate profiles beyond Pro/N-degron motifs, which pave the way for deciphering additional pathways regulated by hGID E3 ligase activity through its GID4 substrate receptor.\nA practical and robust method to evaluate metabolic fluxes in pancreatic islets\nAuthors: Rocha, D. S.; Manucci, A. C.; Bruni-Cardoso, A.; Kowaltowski, A. J.; Vilas-Boas, E. A.\nScore: 5.8, Published: 2023-07-26 DOI: 10.1101/2023.07.25.550538\nAims/hypothesisEfficient mitochondrial oxidative phosphorylation is essential for pancreatic beta cell responses to nutrient levels. Consequently, the evaluation of mitochondrial oxygen consumption and ATP production is important to investigate essential aspects of pancreatic islet pathophysiology. Currently, most studies use cell lines instead of primary islets due to difficulties in measuring primary islet respiration, which requires specific equipment and consumables that are expensive, complicated to use, and poorly reproducible. The aim of this study is to establish a robust and practical method to assess primary islet metabolic fluxes using Extracellular Flux Technology and standard commercial consumables. MethodsPancreatic islets were isolated from 8 to 12-week-old mice and rats, and submitted to a dispersion protocol using trypsin. Dispersed islets were adhered overnight to pre-coated standard Seahorse microplates, and oxygen consumption rates were evaluated using a Seahorse Extracellular Flux Analyzer. We also validated the functionality of dispersed islets by analyzing glucose-stimulated insulin secretion (GSIS) and calcium (Ca2+) influx in response to different modulators by fluorescence microscopy. ResultsWe provide a detailed protocol with all steps necessary to optimize islet isolation and dispersion, in order to achieve a high yield of functional islets and perform metabolic flux analysis. With this method, which requires only a few islets per replicate, both rat and mouse islets present robust basal respiration and proper response to mitochondrial modulators (oligomycin, CCCP, antimycin and rotenone) and glucose addition. Both oligomycin and CCCP concentrations were titrated. Our method was also validated by other functional assays, which show these cells present conserved Ca2+ influx and insulin secretion in response to glucose. Conclusions/interpretationWe established a practical and robust method to assess ex vivo islet metabolic fluxes and oxidative phosphorylation. Our findings cover an important gap in primary islet physiology studies, providing a valuable tool we hope is useful to uncover basic beta cell metabolic mechanisms, as well as for translational investigations, such as pharmacological candidate discovery and islet transplantation protocols. Research in context What is already known about this subject?O_LIPancreatic beta cells efficiently couple oxidative phosphorylation and ATP production with insulin secretion; mitochondrial ATP production is crucial for proper insulin secretion. C_LIO_LIMost studies of beta cell respiration use cell lines instead of primary islets, which are a much more robust model to evaluate beta cell function. C_LIO_LIThe few works with primary islet respiration use specific equipment and consumables that are expensive, complicated, and poorly reproducible. C_LI What is the key question?O_LIIs it possible to develop a practical method to evaluate metabolic fluxes and ATP production in isolated islets, using the standard Seahorse Extracellular Flux Technology? C_LI What are the new findings?O_LIWe optimized rodent islet isolation and functional analysis protocols using standard extracellular flux analysis equipment and consumables. C_LIO_LIOur method allows for increased islet yield and robust islet respiration measurements. C_LI How might this impact on clinical practice in the foreseeable future?O_LIQuantitative measurements of metabolic fluxes and oxidative phosphorylation are the cornerstone of new discoveries in beta cells, and can contribute toward the establishment of new cellular protocols, such as for cell transplantation, as well as the development of new pharmacological agents targeted to these cells. C_LI\nDeep mutational scanning and machine learning uncover antimicrobial peptide features driving membrane selectivity\nAuthors: Randall, J. R.; Vieira, L.; Wilke, C. O.; Davies, B. W.\nScore: 5.5, Published: 2023-07-29 DOI: 10.1101/2023.07.28.551017\nAntimicrobial peptides are critical actors of the innate immune system. Those with direct antibacterial activity often kill via cell membrane lysis, but have difficulty distinguishing between target cells and mammalian cells complicating therapeutic use. Here, we use a new technique called deep mutational surface localized antimicrobial display (dmSLAY) to rapidly elucidate single residue importance and flexibility for Protegrin-1, a potent yet toxic host-defense peptide. Subsequent biochemical analyses reveal that bacterial membrane selectivity is improved for Protegrin-1 variants which maintain membrane bound secondary structure, avoid large aromatic residues, and mutate cysteine pairs. Machine learning is used to expand our analysis to encompass over five million sequence variations and uncover full mutational profiles which promote antibacterial potency, toxicity, and specificity for bacterial or mammalian cell membranes. Our results describe an innovative, high-throughput approach for rapidly elucidating antimicrobial peptide sequence-structure-function relationships which can be used to inform future clinical antibiotic design.\nSurprising variety in the USP deubiquitinase catalytic mechanism\nAuthors: Keijzer, N.; Priyanka, A.; Stijf-Bultsma, Y.; Fish, A.; Gersch, M.; Sixma, T. K.\nScore: 5.0, Published: 2023-07-24 DOI: 10.1101/2023.07.24.550302\nThe USP family of deubiquitinases (DUBs) controls many ubiquitin-dependent signaling events. This generates therapeutic potential, with active-site inhibitors in preclinical and clinical studies. Understanding of the USP active site was so far primarily guided by USP7 data, where the catalytic triad consists of cysteine, histidine and a third residue (first critical residue), which polarizes the histidine through a hydrogen bond. A conserved aspartate (second critical residue) is directly adjacent to this first critical residue. Here we study the roles of these critical residues in a subset of USPs and reveal a remarkable variety in function. While USP7 relies on the first critical residue for catalysis, this residue is dispensable in USP1, USP15, USP40 and USP48. Instead, their second critical residue is vital for catalysis. Interestingly, without their respective vital residue USP7, USP15 and USP40 can still perform nucleophilic attack. The diverging catalytic mechanisms of USP1 and USP7 are independent of substrate and retained in cells for USP1. The unexpected variety of catalytic mechanisms in this well-conserved protein family may generate opportunities for selective targeting of individual USPs. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=103 SRC=\"FIGDIR/small/550302v2_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (22K): org.highwire.dtl.DTLVardef@3c5068org.highwire.dtl.DTLVardef@1c3070corg.highwire.dtl.DTLVardef@103c795org.highwire.dtl.DTLVardef@1242471_HPS_FORMAT_FIGEXP M_FIG C_FIG SynopsisO_LIThe roles of the highly conserved critical residues in USP active sites are poorly understood. Here we show that these two residues have varying importance for catalysis between different USPs. C_LIO_LIExcept for USP7, the majority of USPs does not rely on the canonical third catalytic residue (first critical residue). Instead, the USPs tested rely primarily on the highly conserved second critical residue. C_LIO_LIIn some USPs, either critical residues can accommodate nucleophilic attack (USP7, USP40, USP15). USP1 and USP48 are unable to perform the nucleophilic attack without the second critical residue. C_LI\nCrystal structure and biochemical activity of the macro domain from rubella virus p150\nAuthors: Stoll, G. A.; Zhang, L.; Modis, Y.\nScore: 4.3, Published: 2023-07-28 DOI: 10.1101/2023.07.28.550942\nRubella virus remains a global health threat. Rubella infections during pregnancy can cause serious congenital pathology and no antiviral treatments are available. Rubella virus encodes a nonstructural polyprotein with RNA polymerase, methyltransferase, and papain-like cysteine protease activities, along with a putative macro domain of unknown function. Macro domains bind ADP-ribose adducts, a post-translational modification that plays a key role in host-virus conflicts. Some macro domains can also remove the mono-ADP-ribose adduct or degrade poly-ADP-ribose chains. Here, we report high-resolution crystal structures of the macro domain from rubella virus nonstructural protein p150, with and without ADP-ribose bound. The overall fold is most similar to macroD-type macro domains from various nonviral species. The specific composition and structure of the residues that poised for catalysis or coordinate ADP-ribose in the rubella virus macro domain are most similar to those of macro domains from alphaviruses. Isothermal calorimetry and enzymatic assays show that the rubella virus macro domain binds ADP-ribose in solution and has mono-ADP-ribosylhydrolase (de-MARylation) activity.\nGenetic and functional diversity of beta-N-acetylgalactosamine residue-targeting glycosidases expanded by deep-sea metagenome\nAuthors: Sumida, T.; Hiraoka, S.; Usui, K.; Ishiwata, A.; Sengoku, T.; Stubbs, K. A.; Tanaka, K.; Deguchi, S.; Fushinobu, S.; Nunoura, T.\nScore: 4.1, Published: 2023-07-28 DOI: 10.1101/2023.07.28.550916\nBeta-N-Acetylgalactosamine-containing glycans play essential roles in several biological processes, including cell adhesion, signal transduction, and immune responses. Beta-N-Acetylgalactosaminidases hydrolyze beta-N-acetylgalactosamine linkages of various glycoconjugates. However, their biological significance remains ambiguous, primarily because only one type of enzyme, exo-beta-N-acetylgalactosaminidases that specifically act on beta-N-acetylgalactosamine residues, has been documented so far. In this study, we identified three novel glycoside hydrolase families distributed among all three domains of life and characterized eight novel beta-N-acetylgalactosaminidases and beta-N-acetylhexosaminidase through sequence-based screening of deep-sea metagenomes and subsequent searching of public protein databases. Despite low sequence similarity, the crystal structures of these enzymes demonstrate that all enzymes share a prototype structure and diversify their substrate specificities (endo-, dual-endo/exo-, and exo-) through the accumulation of mutations and insertional amino acid sequences. The diverse beta-N-acetylgalactosaminidases reported in this study could facilitate the comprehension of their structures and functions and present novel evolutionary pathways for expanding their substrate specificity.\n",
  "wordCount" : "2339",
  "inLanguage": "en",
  "datePublished": "2023-07-30T10:36:57Z",
  "dateModified": "2023-07-30T10:36:57Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta"><span>updated on July 30, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.551006">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.551006" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.551006">
        <p class="paperTitle">Targeted protein editing with an antibody-based system</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.551006" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.551006" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rimon, O.; Konc, J.; Roy Chowdhury, V.; Bernardes, G. J. L.; Vendruscolo, M.</p>
        <p class="info">Score: 18.8, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.551006' target='https://doi.org/10.1101/2023.07.28.551006'> 10.1101/2023.07.28.551006</a></p>
        <p class="abstract">The chemical modification of proteins is one of the major mechanisms used to regulate the trafficking and function of these macromolecules in the cell. It is therefore of great interest to develop tools to exploit this type of modifications for applications in molecular biology, medicine and biotechnology. Here we present a method of using antibodies to perform post-translational covalent modifications of endogenous proteins in complex environments by exploiting proximity-driven chemistry. The method is based on the ability of antibodies to hold a weakly reactive group adjacently to its intended site of reaction by binding the target protein on a nearby epitope. We illustrate this approach by targeting the green fluorescent protein in increasingly complex environments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549963">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549963" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549963">
        <p class="paperTitle">Coenzyme Q4 is a functional substitute for coenzyme Q10 and can be targeted to the mitochondria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549963" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549963" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Steenberge, L. H.; Sung, A. Y.; Fan, J.; Pagliarini, D. J.</p>
        <p class="info">Score: 26.7, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549963' target='https://doi.org/10.1101/2023.07.20.549963'> 10.1101/2023.07.20.549963</a></p>
        <p class="abstract">Coenzyme Q10 (CoQ10) is an important cofactor and antioxidant for numerous cellular processes, and its deficiency has been linked to human disorders including mitochondrial disease, heart failure, Parkinsons disease, and hypertension. Unfortunately, treatment with exogenous oral CoQ10 is often ineffective, likely due to the extreme hydrophobicity and high molecular weight of CoQ10. Here, we show that less hydrophobic CoQ species with shorter isoprenoid tails can serve as viable substitutes for CoQ10 in human cells. We demonstrate that CoQ4 can perform multiple functions of CoQ10 in CoQ-deficient cells at markedly lower treatment concentrations, motivating further investigation of CoQ4 as a supplement for CoQ10 deficiencies. In addition, we describe the synthesis and evaluation of an initial set of compounds designed to target CoQ4 selectively to mitochondria using triphenylphosphonium (TPP). Our results indicate that select versions of these compounds can successfully be delivered to mitochondria in a cell model and be cleaved to produce CoQ4, laying the groundwork for further development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549978">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549978" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549978">
        <p class="paperTitle">Structural characterisation of the complete cycle of sliding clamp loading in Escherichia coli</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549978" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549978" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xu, Z.-Q.; Jergic, S.; Lo, A. T. Y.; Pradhan, A. C.; Brown, S. H. J.; Bouwer, J. C.; Ghodke, H.; Lewis, P. J.; Tolun, G.; Oakley, A. J.; Dixon, N. E.</p>
        <p class="info">Score: 11.3, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549978' target='https://doi.org/10.1101/2023.07.20.549978'> 10.1101/2023.07.20.549978</a></p>
        <p class="abstract">Ring-shaped DNA sliding clamps are essential for DNA replication and genome maintenance. Clamps need to be opened or trapped open and chaperoned onto DNA by clamp loader complexes (CLCs). Detailed understanding of the mechanisms by which CLCs open and place clamps around DNA remains limited. Here, we present a series of six structures of the Escherichia coli CLC bound to an open or closed clamp on and off a primer-template DNA that represent all intermediates in the clamp loading process. We show that the ATP-bound CLC first binds to a clamp, then constricts to hold onto it. The CLC then expands to open the clamp with a gap large enough for double-stranded DNA to enter. Upon binding to DNA, the CLC constricts slightly, allowing ATP hydrolysis and clamp closing around DNA. Although both yeast and E. coli CLCs open clamps by crab claw-like motions, they do it by the CLC expanding in opposite directions. These structures provide critical high-resolution snapshots of clamp loading by the E. coli CLC, revealing how the molecular machine works.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550462">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550462" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550462">
        <p class="paperTitle">Shared Structural Features of Miro Binding Control Mitochondrial Homeostasis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550462" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550462" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Covill-Cooke, C.; Kwizera, B.; Lopez-Domenech, G.; Thompson, C. O. D.; Cheung, N. J.; Cerezo, E.; Peterka, M.; Kittler, J. T.; Kornmann, B.</p>
        <p class="info">Score: 9.7, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550462' target='https://doi.org/10.1101/2023.07.25.550462'> 10.1101/2023.07.25.550462</a></p>
        <p class="abstract">Miro proteins are universally conserved mitochondrial calcium-binding GTPases that regulate a multitude of mitochondrial processes, including transport, clearance and lipid trafficking. Miro binds a variety of client proteins involved in these functions. How this binding is operated at the molecular level and whether and how it is important for mitochondrial health, however, remains unknown. Here, we show that known Miro clients all use a similar short motif to bind the same structural element: a highly conserved hydrophobic pocket in the calcium-binding domain of Miro. Using these Miro-binding motifs, we identified direct interactors de novo, including yeast Mdm34, and mammalian MTFR1/2/1L, VPS13D and Parkin. Given the shared binding mechanism and conservation across eukaryotes, we propose that Miro is a universal mitochondrial adaptor coordinating mitochondrial health.

One-Sentence SummaryFunctionally diverse mitochondrial proteins interact with a conserved hydrophobic pocket on the calcium-binding Miro-GTPases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549906">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549906" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549906">
        <p class="paperTitle">The hGID GID4 E3 ubiquitin ligase complex targets ARHGAP11A to regulate cell migration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549906" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549906" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bagci, H.; Winkler, M.; Uliana, F.; Boulais, J.; Mohamed, W. I.; Park, S. L.; Cote, J.-F.; Peter, M.</p>
        <p class="info">Score: 8.5, Published: 2023-07-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549906' target='https://doi.org/10.1101/2023.07.20.549906'> 10.1101/2023.07.20.549906</a></p>
        <p class="abstract">The human CTLH/GID (hGID) complex emerged as an important E3 ligase regulating multiple cellular processes, including cell cycle progression and metabolic activity. However, the range of biological functions controlled by hGID remains unexplored. Here, we show that the hGID substrate receptor GID4 regulates cell growth and migration. Biochemical and cellular assays combined with proximity-dependent biotinylation (BioID2) revealed that the hGIDGID4 E3-ligase targets the Rho-GAP ARHGAP11A for degradation. Depletion of GID4 or impeding the GID4 substrate binding pocket impairs motility and directed cell movement, whereas knockdown of ARHGAP11A significantly restores the cell migration defect. We found that GID4 controls cell migration by degrading ARHGAP11A thereby preventing its accumulation at the cell periphery where it inactivates RhoA activity. Together, we identified a unique function for GID4, as well as a wide range of substrate profiles beyond Pro/N-degron motifs, which pave the way for deciphering additional pathways regulated by hGID E3 ligase activity through its GID4 substrate receptor.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550538">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550538" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550538">
        <p class="paperTitle">A practical and robust method to evaluate metabolic fluxes in pancreatic islets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550538" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550538" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rocha, D. S.; Manucci, A. C.; Bruni-Cardoso, A.; Kowaltowski, A. J.; Vilas-Boas, E. A.</p>
        <p class="info">Score: 5.8, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550538' target='https://doi.org/10.1101/2023.07.25.550538'> 10.1101/2023.07.25.550538</a></p>
        <p class="abstract">Aims/hypothesisEfficient mitochondrial oxidative phosphorylation is essential for pancreatic beta cell responses to nutrient levels. Consequently, the evaluation of mitochondrial oxygen consumption and ATP production is important to investigate essential aspects of pancreatic islet pathophysiology. Currently, most studies use cell lines instead of primary islets due to difficulties in measuring primary islet respiration, which requires specific equipment and consumables that are expensive, complicated to use, and poorly reproducible. The aim of this study is to establish a robust and practical method to assess primary islet metabolic fluxes using Extracellular Flux Technology and standard commercial consumables.

MethodsPancreatic islets were isolated from 8 to 12-week-old mice and rats, and submitted to a dispersion protocol using trypsin. Dispersed islets were adhered overnight to pre-coated standard Seahorse microplates, and oxygen consumption rates were evaluated using a Seahorse Extracellular Flux Analyzer. We also validated the functionality of dispersed islets by analyzing glucose-stimulated insulin secretion (GSIS) and calcium (Ca2&#43;) influx in response to different modulators by fluorescence microscopy.

ResultsWe provide a detailed protocol with all steps necessary to optimize islet isolation and dispersion, in order to achieve a high yield of functional islets and perform metabolic flux analysis. With this method, which requires only a few islets per replicate, both rat and mouse islets present robust basal respiration and proper response to mitochondrial modulators (oligomycin, CCCP, antimycin and rotenone) and glucose addition. Both oligomycin and CCCP concentrations were titrated. Our method was also validated by other functional assays, which show these cells present conserved Ca2&#43; influx and insulin secretion in response to glucose.

Conclusions/interpretationWe established a practical and robust method to assess ex vivo islet metabolic fluxes and oxidative phosphorylation. Our findings cover an important gap in primary islet physiology studies, providing a valuable tool we hope is useful to uncover basic beta cell metabolic mechanisms, as well as for translational investigations, such as pharmacological candidate discovery and islet transplantation protocols.

Research in context What is already known about this subject?O_LIPancreatic beta cells efficiently couple oxidative phosphorylation and ATP production with insulin secretion; mitochondrial ATP production is crucial for proper insulin secretion.
C_LIO_LIMost studies of beta cell respiration use cell lines instead of primary islets, which are a much more robust model to evaluate beta cell function.
C_LIO_LIThe few works with primary islet respiration use specific equipment and consumables that are expensive, complicated, and poorly reproducible.
C_LI

What is the key question?O_LIIs it possible to develop a practical method to evaluate metabolic fluxes and ATP production in isolated islets, using the standard Seahorse Extracellular Flux Technology?
C_LI

What are the new findings?O_LIWe optimized rodent islet isolation and functional analysis protocols using standard extracellular flux analysis equipment and consumables.
C_LIO_LIOur method allows for increased islet yield and robust islet respiration measurements.
C_LI

How might this impact on clinical practice in the foreseeable future?O_LIQuantitative measurements of metabolic fluxes and oxidative phosphorylation are the cornerstone of new discoveries in beta cells, and can contribute toward the establishment of new cellular protocols, such as for cell transplantation, as well as the development of new pharmacological agents targeted to these cells.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.551017">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.551017" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.551017">
        <p class="paperTitle">Deep mutational scanning and machine learning uncover antimicrobial peptide features driving membrane selectivity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.551017" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.551017" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Randall, J. R.; Vieira, L.; Wilke, C. O.; Davies, B. W.</p>
        <p class="info">Score: 5.5, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.551017' target='https://doi.org/10.1101/2023.07.28.551017'> 10.1101/2023.07.28.551017</a></p>
        <p class="abstract">Antimicrobial peptides are critical actors of the innate immune system. Those with direct antibacterial activity often kill via cell membrane lysis, but have difficulty distinguishing between target cells and mammalian cells complicating therapeutic use. Here, we use a new technique called deep mutational surface localized antimicrobial display (dmSLAY) to rapidly elucidate single residue importance and flexibility for Protegrin-1, a potent yet toxic host-defense peptide. Subsequent biochemical analyses reveal that bacterial membrane selectivity is improved for Protegrin-1 variants which maintain membrane bound secondary structure, avoid large aromatic residues, and mutate cysteine pairs. Machine learning is used to expand our analysis to encompass over five million sequence variations and uncover full mutational profiles which promote antibacterial potency, toxicity, and specificity for bacterial or mammalian cell membranes. Our results describe an innovative, high-throughput approach for rapidly elucidating antimicrobial peptide sequence-structure-function relationships which can be used to inform future clinical antibiotic design.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550302">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550302" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550302">
        <p class="paperTitle">Surprising variety in the USP deubiquitinase catalytic mechanism</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550302" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550302" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Keijzer, N.; Priyanka, A.; Stijf-Bultsma, Y.; Fish, A.; Gersch, M.; Sixma, T. K.</p>
        <p class="info">Score: 5.0, Published: 2023-07-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550302' target='https://doi.org/10.1101/2023.07.24.550302'> 10.1101/2023.07.24.550302</a></p>
        <p class="abstract">The USP family of deubiquitinases (DUBs) controls many ubiquitin-dependent signaling events. This generates therapeutic potential, with active-site inhibitors in preclinical and clinical studies.

Understanding of the USP active site was so far primarily guided by USP7 data, where the catalytic triad consists of cysteine, histidine and a third residue (first critical residue), which polarizes the histidine through a hydrogen bond. A conserved aspartate (second critical residue) is directly adjacent to this first critical residue.

Here we study the roles of these critical residues in a subset of USPs and reveal a remarkable variety in function. While USP7 relies on the first critical residue for catalysis, this residue is dispensable in USP1, USP15, USP40 and USP48. Instead, their second critical residue is vital for catalysis.

Interestingly, without their respective vital residue USP7, USP15 and USP40 can still perform nucleophilic attack. The diverging catalytic mechanisms of USP1 and USP7 are independent of substrate and retained in cells for USP1. The unexpected variety of catalytic mechanisms in this well-conserved protein family may generate opportunities for selective targeting of individual USPs.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=103 SRC=&#34;FIGDIR/small/550302v2_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (22K):
org.highwire.dtl.DTLVardef@3c5068org.highwire.dtl.DTLVardef@1c3070corg.highwire.dtl.DTLVardef@103c795org.highwire.dtl.DTLVardef@1242471_HPS_FORMAT_FIGEXP  M_FIG C_FIG SynopsisO_LIThe roles of the highly conserved critical residues in USP active sites are poorly understood. Here we show that these two residues have varying importance for catalysis between different USPs.
C_LIO_LIExcept for USP7, the majority of USPs does not rely on the canonical third catalytic residue (first critical residue). Instead, the USPs tested rely primarily on the highly conserved second critical residue.
C_LIO_LIIn some USPs, either critical residues can accommodate nucleophilic attack (USP7, USP40, USP15). USP1 and USP48 are unable to perform the nucleophilic attack without the second critical residue.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.550942">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.550942" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.550942">
        <p class="paperTitle">Crystal structure and biochemical activity of the macro domain from rubella virus p150</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.550942" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.550942" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stoll, G. A.; Zhang, L.; Modis, Y.</p>
        <p class="info">Score: 4.3, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.550942' target='https://doi.org/10.1101/2023.07.28.550942'> 10.1101/2023.07.28.550942</a></p>
        <p class="abstract">Rubella virus remains a global health threat. Rubella infections during pregnancy can cause serious congenital pathology and no antiviral treatments are available. Rubella virus encodes a nonstructural polyprotein with RNA polymerase, methyltransferase, and papain-like cysteine protease activities, along with a putative macro domain of unknown function. Macro domains bind ADP-ribose adducts, a post-translational modification that plays a key role in host-virus conflicts. Some macro domains can also remove the mono-ADP-ribose adduct or degrade poly-ADP-ribose chains. Here, we report high-resolution crystal structures of the macro domain from rubella virus nonstructural protein p150, with and without ADP-ribose bound. The overall fold is most similar to macroD-type macro domains from various nonviral species. The specific composition and structure of the residues that poised for catalysis or coordinate ADP-ribose in the rubella virus macro domain are most similar to those of macro domains from alphaviruses. Isothermal calorimetry and enzymatic assays show that the rubella virus macro domain binds ADP-ribose in solution and has mono-ADP-ribosylhydrolase (de-MARylation) activity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.550916">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.550916" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.550916">
        <p class="paperTitle">Genetic and functional diversity of beta-N-acetylgalactosamine residue-targeting glycosidases expanded by deep-sea metagenome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.550916" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.550916" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sumida, T.; Hiraoka, S.; Usui, K.; Ishiwata, A.; Sengoku, T.; Stubbs, K. A.; Tanaka, K.; Deguchi, S.; Fushinobu, S.; Nunoura, T.</p>
        <p class="info">Score: 4.1, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.550916' target='https://doi.org/10.1101/2023.07.28.550916'> 10.1101/2023.07.28.550916</a></p>
        <p class="abstract">Beta-N-Acetylgalactosamine-containing glycans play essential roles in several biological processes, including cell adhesion, signal transduction, and immune responses. Beta-N-Acetylgalactosaminidases hydrolyze beta-N-acetylgalactosamine linkages of various glycoconjugates. However, their biological significance remains ambiguous, primarily because only one type of enzyme, exo-beta-N-acetylgalactosaminidases that specifically act on beta-N-acetylgalactosamine residues, has been documented so far. In this study, we identified three novel glycoside hydrolase families distributed among all three domains of life and characterized eight novel beta-N-acetylgalactosaminidases and beta-N-acetylhexosaminidase through sequence-based screening of deep-sea metagenomes and subsequent searching of public protein databases. Despite low sequence similarity, the crystal structures of these enzymes demonstrate that all enzymes share a prototype structure and diversify their substrate specificities (endo-, dual-endo/exo-, and exo-) through the accumulation of mutations and insertional amino acid sequences. The diverse beta-N-acetylgalactosaminidases reported in this study could facilitate the comprehension of their structures and functions and present novel evolutionary pathways for expanding their substrate specificity.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
