<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Viral proteins activate PARIS-mediated tRNA degradation and viral tRNAs rescue infection
Authors: Burman, N.; Belukhina, S.; Depardieu, F.; Wilkinson, R. A.; Skutel, M.; Santiago-Frangos, A.; Graham, A. B.; Livenskyi, A.; Chechenina, A.; Morozova, N.; Zahl, T.; Henriques, W. S.; Buyukyoruk, M.; Rouillon, C.; Shyrokova, L.; Kurata, T.; Hauryliuk, V.; Severinov, K.; Groseille, J.; Thierry, A.; Koszul, R.; Tesson, F.; Bernheim, A.; Bikard, D.; Wiedenheft, B.; Isaev, A.
Score: 60.2, Published: 2024-01-09 DOI: 10.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Viral proteins activate PARIS-mediated tRNA degradation and viral tRNAs rescue infection
Authors: Burman, N.; Belukhina, S.; Depardieu, F.; Wilkinson, R. A.; Skutel, M.; Santiago-Frangos, A.; Graham, A. B.; Livenskyi, A.; Chechenina, A.; Morozova, N.; Zahl, T.; Henriques, W. S.; Buyukyoruk, M.; Rouillon, C.; Shyrokova, L.; Kurata, T.; Hauryliuk, V.; Severinov, K.; Groseille, J.; Thierry, A.; Koszul, R.; Tesson, F.; Bernheim, A.; Bikard, D.; Wiedenheft, B.; Isaev, A.
Score: 60.2, Published: 2024-01-09 DOI: 10." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-10T10:40:05+00:00" />
<meta property="article:modified_time" content="2024-01-10T10:40:05+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Viral proteins activate PARIS-mediated tRNA degradation and viral tRNAs rescue infection
Authors: Burman, N.; Belukhina, S.; Depardieu, F.; Wilkinson, R. A.; Skutel, M.; Santiago-Frangos, A.; Graham, A. B.; Livenskyi, A.; Chechenina, A.; Morozova, N.; Zahl, T.; Henriques, W. S.; Buyukyoruk, M.; Rouillon, C.; Shyrokova, L.; Kurata, T.; Hauryliuk, V.; Severinov, K.; Groseille, J.; Thierry, A.; Koszul, R.; Tesson, F.; Bernheim, A.; Bikard, D.; Wiedenheft, B.; Isaev, A.
Score: 60.2, Published: 2024-01-09 DOI: 10."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Viral proteins activate PARIS-mediated tRNA degradation and viral tRNAs rescue infection\nAuthors: Burman, N.; Belukhina, S.; Depardieu, F.; Wilkinson, R. A.; Skutel, M.; Santiago-Frangos, A.; Graham, A. B.; Livenskyi, A.; Chechenina, A.; Morozova, N.; Zahl, T.; Henriques, W. S.; Buyukyoruk, M.; Rouillon, C.; Shyrokova, L.; Kurata, T.; Hauryliuk, V.; Severinov, K.; Groseille, J.; Thierry, A.; Koszul, R.; Tesson, F.; Bernheim, A.; Bikard, D.; Wiedenheft, B.; Isaev, A.\nScore: 60.2, Published: 2024-01-09 DOI: 10.",
  "keywords": [
    
  ],
  "articleBody": " Viral proteins activate PARIS-mediated tRNA degradation and viral tRNAs rescue infection\nAuthors: Burman, N.; Belukhina, S.; Depardieu, F.; Wilkinson, R. A.; Skutel, M.; Santiago-Frangos, A.; Graham, A. B.; Livenskyi, A.; Chechenina, A.; Morozova, N.; Zahl, T.; Henriques, W. S.; Buyukyoruk, M.; Rouillon, C.; Shyrokova, L.; Kurata, T.; Hauryliuk, V.; Severinov, K.; Groseille, J.; Thierry, A.; Koszul, R.; Tesson, F.; Bernheim, A.; Bikard, D.; Wiedenheft, B.; Isaev, A.\nScore: 60.2, Published: 2024-01-09 DOI: 10.1101/2024.01.02.573894\nViruses compete with each other for limited cellular resources, and some viruses deliver defense mechanisms that protect the host from competing genetic parasites. PARIS is a defense system, often encoded in viral genomes, that is composed of a 53 kDa ABC ATPase (AriA) and a 35 kDa TOPRIM nuclease (AriB). Here we show that AriA and AriB assemble into a 425 kDa supramolecular immune complex. We use cryo-EM to determine the structure of this complex which explains how six molecules of AriA assemble into a propeller-shaped scaffold that coordinates three subunits of AriB. ATP-dependent detection of foreign proteins triggers the release of AriB, which assembles into a homodimeric nuclease that blocks infection by cleaving the host tRNALys. Phage T5 subverts PARIS immunity through expression of a tRNALys variant that prevent PARIS-mediated cleavage, and thereby restores viral infection. Collectively, these data explain how AriA functions as an ATP-dependent sensor that detects viral proteins and activates the AriB toxin. PARIS is one of an emerging set of immune systems that form macromolecular complexes for the recognition of foreign proteins, rather than foreign nucleic acids.\nDirect interrogation of context-dependent GPCR activity with a universal biosensor platform\nAuthors: Janicot, R.; Maziarz, M.; Park, J.-C.; Luebbers, A.; Green, E.; Zhao, J.; Philibert, C. E.; Zhang, H.; Layne, M. D.; Wu, J. C.; Garcia-Marcos, M.\nScore: 24.9, Published: 2024-01-02 DOI: 10.1101/2024.01.02.573921\nG protein-coupled receptors (GPCRs) are the largest family of druggable proteins in the human genome, but progress in understanding and targeting them is hindered by the lack of tools to reliably measure their nuanced behavior in physiologically-relevant contexts. Here, we developed a collection of compact ONE vector G-protein Optical (ONE-GO) biosensor constructs as a scalable platform that can be conveniently deployed to measure G-protein activation by virtually any GPCR with high fidelity even when expressed endogenously in primary cells. By characterizing dozens of GPCRs across many cell types like primary cardiovascular cells or neurons, we revealed new insights into the molecular basis for G-protein coupling selectivity of GPCRs, pharmacogenomic profiles of anti-psychotics on naturally-occurring GPCR variants, and G-protein subtype signaling bias by endogenous GPCRs depending on cell type or upon inducing disease-like states. In summary, this open-source platform makes the direct interrogation of context-dependent GPCR activity broadly accessible.\nGenetically encoded tool for manipulation of Î”{Psi}m identifies the latter as the driver of integrative stress response induced by ATP Synthase dysfunction\nAuthors: Choe, M.; Titov, D. V.\nScore: 32.5, Published: 2023-12-27 DOI: 10.1101/2023.12.27.573435\nMitochondrial membrane potential ({Delta}{Psi}m) is one of the key parameters controlling cellular bioenergetics. Investigation of the role of {Delta}{Psi}m in live cells is complicated by a lack of tools for its direct manipulation without off-target effects. Here, we adopted the uncoupling protein UCP1 from brown adipocytes as a genetically encoded tool for direct manipulation of {Delta}{Psi}m. We validated the ability of exogenously expressed UCP1 to induce uncoupled respiration and lower {Delta}{Psi}m in mammalian cells. UCP1 expression lowered {Delta}{Psi}m to the same extent as chemical uncouplers but did not inhibit cell proliferation, suggesting that it manipulates {Delta}{Psi}m without the off-target effects of chemical uncouplers. Using UCP1, we revealed that elevated {Delta}{Psi}m is the driver of the Integrated Stress Response induced by ATP synthase inhibition in mammalian cells.\nSwinging lever mechanism of myosin directly demonstrated by time-resolved cryoEM\nAuthors: Klebl, D. P.; McMillan, S. N.; Risi, C.; Forgacs, E.; Virok, B.; Atherton, J.; Stofella, M.; Winkelmann, D.; Sobott, F.; Galkin, V.; Knight, P. J.; Muench, S. P.; Scarff, C.; White, H. D.\nScore: 16.9, Published: 2024-01-06 DOI: 10.1101/2024.01.05.574365\nMyosins are essential for producing force and movement in cells through their interactions with F-actin. Generation of movement is proposed to occur through structural changes within the myosin motor domain, fuelled by ATP hydrolysis, that are amplified by a lever swing1, transitioning myosin from a primed (pre-powerstroke) state to a post-powerstroke state. However, the initial, primed actomyosin state, proposed to form prior to lever swing, has never been observed. Nor has the mechanism by which actin catalyses myosin ATPase activity been resolved. To address this, we performed time-resolved cryoEM of a myosin-5 mutant having slow hydrolysis product release. Primed actomyosin was captured 10 ms after mixing primed myosin with F-actin, whereas post-powerstroke actomyosin predominated at 120 ms, with no abundant intermediate structures. The structures were solved to 4.4[A] and 4.2[A] global resolution respectively. The primed motor binds to actin through its lower 50 kDa subdomain, with the actin-binding cleft open and Pi release prohibited. N-terminal actin interactions with myosin promote rotation of the upper 50 kDa subdomain, which closes the actin-binding cleft, and enables Pi release. Formation of upper 50 kDa subdomain interactions with actin creates the strong-binding interface required for effective force production. The myosin-5 lever swings through an angle of 93{degrees}, predominantly along the actin axis, with little twisting, to produce the post-powerstroke state. The magnitude of the lever swing matches the typical step length of myosin-5 walking along actin. These time-resolved structures directly demonstrate the swinging lever mechanism, ending decades of conjecture on how myosin produces force and movement.\nMicroED structure of the C11 cysteine protease Clostripain\nAuthors: Ruma, Y. N.; Bu, G.; Gonen, T.\nScore: 15.2, Published: 2024-01-05 DOI: 10.1101/2024.01.04.574240\nClostripain secreted from Clostridium histolyticum is the founding member of the C11 family of Clan CD cysteine peptidases, which is an important group of peptidases secreted by numerous bacteria. Clostripain is an arginine specific endopeptidase. Because of its efficacy as a cysteine peptidase, it is widely used in laboratory settings. Despite its importance the structure of clostripain remains unsolved. Here we describe the first structure of an active form of C. histolyticum Clostripain determined at 3.6 [A] resolution using microcrystal electron diffraction (MicroED). The structure was determined from a single nanocrystal after focused ion beam milling. The structure of Clostripain shows a typical Clan CD /{beta}/ sandwich architecture and the Cys231/His176 catalytic dyad in the active site. It has a large electronegative substrate binding pocket showing its ability to accommodate large and diverse substrates. A loop in the heavy chain formed between residues 452 to 457 is potentially important for substrate binding. In conclusion, this result demonstrates the importance of MicroED to determine the unknown structure of macromolecules such as Clostripain, which can be further used as a platform to study substrate binding and design of potential inhibitors against this class of peptidases.\nHuman coronavirus HKU1 recognition of the TMPRSS2 host receptor\nAuthors: McCallum, M.; Park, Y.-J.; Stewart, C.; Sprouse, K. R.; Brown, J.; Tortorici, M. A.; Gibson, C.; Wong, E.; Ieven, M.; Telenti, A.; Veesler, D.\nScore: 15.0, Published: 2024-01-09 DOI: 10.1101/2024.01.09.574565\nThe human coronavirus HKU1 spike (S) glycoprotein engages host cell surface sialoglycans and transmembrane protease serine 2 (TMPRSS2) to initiate infection. The molecular basis of HKU1 binding to TMPRSS2 and determinants of host receptor tropism remain elusive. Here, we designed an active human TMPRSS2 construct enabling high-yield recombinant production in human cells of this key therapeutic target. We determined a cryo-electron microscopy structure of the HKU1 RBD bound to human TMPRSS2 providing a blueprint of the interactions supporting viral entry and explaining the specificity for TMPRSS2 among human type 2 transmembrane serine proteases. We found that human, rat, hamster and camel TMPRSS2 promote HKU1 S-mediated entry into cells and identified key residues governing host receptor usage. Our data show that serum antibodies targeting the HKU1 RBD TMPRSS2 binding-site are key for neutralization and that HKU1 uses conformational masking and glycan shielding to balance immune evasion and receptor engagement.\nQuinone extraction drives atmospheric carbon monoxide oxidation in bacteria\nAuthors: Kropp, A.; Gillett, D. L.; Venugopal, H.; Gonzalvez, M. A.; Lingford, J. P.; Barlow, C. K.; Zhang, J.; Greening, C.; Grinter, R.\nScore: 14.3, Published: 2024-01-09 DOI: 10.1101/2024.01.09.574806\nDiverse bacteria and archaea use atmospheric carbon monoxide (CO) as an energy source during long-term survival. This process enhances the biodiversity of soil and marine ecosystems globally and removes 250 million tonnes of a toxic, climate-relevant pollutant from the atmosphere each year. Bacteria use [MoCu]-carbon monoxide dehydrogenases (Mo-CODH) to convert CO to carbon dioxide, then transfer the liberated high-energy electrons to the aerobic respiratory chain. However, given no high-affinity Mo-CODH has been purified, it is unknown how these enzymes oxidise CO at low concentrations and interact with the respiratory chain. Here we resolve these knowledge gaps by analysing Mo-CODH (CoxSML) and its hypothetical partner CoxG from Mycobacterium smegmatis. Kinetic and electrochemical analyses show purified Mo-CODH is a highly active high-affinity enzyme (Km = 139 nM, kcat = 54.2 s-1). Based on its 1.85 Angstrom resolution cryoEM structure, Mo-CODH forms a CoxSML homodimer similar to characterised low-affinity homologs, but has distinct active site coordination and narrower gas channels that may modulate affinity. We provide structural, biochemical, and genetic evidence that Mo-CODH transfers CO-derived electrons to the aerobic respiratory chain via the membrane-bound menaquinone-binding protein CoxG. Consistently, CoxG is required for CO-driven respiration, extracts menaquinone from mycobacterial membranes, and binds quinones in a hydrophobic pocket. Finally, we show that Mo-CODH and CoxG genetically and structurally associate in diverse bacteria and archaea. These findings reveal the basis of a biogeochemically and ecologically important process, while demonstrating that the newly discovered process of long-range quinone transport is a general mechanism of energy conservation, which convergently evolved on multiple occasions.\nA genetically-encoded fluorescent biosensor for visualization of acetyl-CoA in live cells\nAuthors: Smith, J. J.; Valentino, T. R.; Ablicki, A. H.; Banerjee, R.; Colligan, A. R.; Eckert, D. M.; Desjardins, G. A.; Diehl, K. L.\nScore: 24.1, Published: 2024-01-01 DOI: 10.1101/2023.12.31.573774\nAcetyl-coenzyme A is a central metabolite that participates in many cellular pathways. Evidence suggests that acetyl-CoA production and consumption are highly compartmentalized in mammalian cells. Yet methods to measure acetyl-CoA in living cells are lacking. In this work, we engineer an acetyl-CoA biosensor from the bacterial protein PanZ and circularly permuted green fluorescent protein (cpGFP). We biochemically characterize the sensor and demonstrate its selectivity for acetyl-CoA over other CoA species. We then deploy the biosensor in E. coli and HeLa cells to demonstrate its utility in living cells. In E. coli, we show that the biosensor enables detection of rapid changes in acetyl-CoA levels. In human cells, we show that the biosensor enables subcellular detection and reveals the compartmentalization of acetyl-CoA metabolism.\nStructure of a lasso peptide bound ETB receptor provides insights into the mechanism of GPCR inverse agonism\nAuthors: Shihoya, W.; Akasaka, H.; Jordan, P. A.; Lechne, A.; Okada, B. K.; Cruz, G. C. M. d.; Sano, F. K.; Tanaka, T.; Kawahara, R.; Chaudhari, R.; Masamune, H.; Mark, B. J.; Nureki, O.\nScore: 11.0, Published: 2024-01-01 DOI: 10.1101/2023.12.30.573741\nLasso peptides exhibit a unique lariat-like knotted structure imparting exceptional stability and thus show promise as therapeutic agents that target cell-surface receptors. One such receptor is the human endothelin ETB receptor, which is implicated in challenging cancers with poor immunotherapy responsiveness. The Streptomyces-derived lasso peptide, RES-701-3, is a selective inhibitor for ETB and a compelling candidate for therapeutic development. However, meager production from a genetically recalcitrant host has limited further structure-activity relationship studies of this potent inhibitor. Here, we report cryo-electron microscopy structures of ETB receptor in both its apo form and complex with RES-701-3, facilitated by a calcineurin-fusion strategy. Hydrophobic interactions between RES-701-3 and the transmembrane region of the receptor, especially involving two tryptophan residues, play a crucial role in RES-701-3 binding. Furthermore, RES-701-3 prevents conformational changes associated with G-protein coupling, explaining its inverse agonist activity. A comparative analysis with other lasso peptides and their target proteins highlights the potential of lasso peptides as precise drug candidates for G-protein-coupled receptors. This structural insight into RES-701-3 binding to ETB receptor offers valuable information for the development of novel therapeutics targeting this receptor and provides a broader understanding of lasso peptide interactions with human cell-surface receptors.\nSpatially resolved quantification of oxygen consumption rate in ex vivo lymph node slices\nAuthors: Anbaei, P.; Stevens, M. G.; Ball, A. G.; Bullock, T. N. J.; Pompano, R. R.\nScore: 9.1, Published: 2024-01-04 DOI: 10.1101/2024.01.03.573955\nCellular metabolism has been closely linked to activation state in cells of the immune system, and the oxygen consumption rate (OCR) in particular serves as a valuable metric for assessing metabolic activity. Several oxygen sensing assays have been reported for cells in standard culture conditions. However, none have provided a spatially resolved, optical measurement of local oxygen consumption in intact tissue samples, making it challenging to understand regional dynamics of consumption. Therefore, here we established a system to monitor the rates of oxygen consumption in ex vivo tissue slices, using murine lymphoid tissue as a case study. By integrating an optical oxygen sensor into a sealed perfusion chamber and incorporating appropriate correction for photobleaching of the sensor and of tissue autofluorescence, we were able to visualize and quantify rates of oxygen consumption in tissue. This method revealed for the first time that the rate of oxygen consumption in naive lymphoid tissue was higher in the T cell region compared to the B cell and cortical regions. To validate the method, we measured OCR in the T cell regions of naive lymph node slices using the optical assay and estimated the consumption rate per cell. The predictions from the optical assay were similar to reported values and were not significantly different from those of the Seahorse metabolic assay, a gold standard method for measuring OCR in cell suspensions. Finally, we used this method to quantify the rate of onset of tissue hypoxia for lymph node slices cultured in a sealed chamber and showed that continuous perfusion was sufficient to maintain oxygenation. In summary, this work establishes a method to monitor oxygen consumption with regional resolution in intact tissue explants, suitable for future use to compare tissue culture conditions and responses to stimulation. TOC image O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=95 SRC=\"FIGDIR/small/573955v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (34K): org.highwire.dtl.DTLVardef@1e20f5dorg.highwire.dtl.DTLVardef@5eb0f8org.highwire.dtl.DTLVardef@b47541org.highwire.dtl.DTLVardef@17e1033_HPS_FORMAT_FIGEXP M_FIG C_FIG\n",
  "wordCount" : "2361",
  "inLanguage": "en",
  "datePublished": "2024-01-10T10:40:05Z",
  "dateModified": "2024-01-10T10:40:05Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 10, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573894">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573894" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573894">
        <p class="paperTitle">Viral proteins activate PARIS-mediated tRNA degradation and viral tRNAs rescue infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573894" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573894" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Burman, N.; Belukhina, S.; Depardieu, F.; Wilkinson, R. A.; Skutel, M.; Santiago-Frangos, A.; Graham, A. B.; Livenskyi, A.; Chechenina, A.; Morozova, N.; Zahl, T.; Henriques, W. S.; Buyukyoruk, M.; Rouillon, C.; Shyrokova, L.; Kurata, T.; Hauryliuk, V.; Severinov, K.; Groseille, J.; Thierry, A.; Koszul, R.; Tesson, F.; Bernheim, A.; Bikard, D.; Wiedenheft, B.; Isaev, A.</p>
        <p class="info">Score: 60.2, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573894' target='https://doi.org/10.1101/2024.01.02.573894'> 10.1101/2024.01.02.573894</a></p>
        <p class="abstract">Viruses compete with each other for limited cellular resources, and some viruses deliver defense mechanisms that protect the host from competing genetic parasites. PARIS is a defense system, often encoded in viral genomes, that is composed of a 53 kDa ABC ATPase (AriA) and a 35 kDa TOPRIM nuclease (AriB). Here we show that AriA and AriB assemble into a 425 kDa supramolecular immune complex. We use cryo-EM to determine the structure of this complex which explains how six molecules of AriA assemble into a propeller-shaped scaffold that coordinates three subunits of AriB. ATP-dependent detection of foreign proteins triggers the release of AriB, which assembles into a homodimeric nuclease that blocks infection by cleaving the host tRNALys. Phage T5 subverts PARIS immunity through expression of a tRNALys variant that prevent PARIS-mediated cleavage, and thereby restores viral infection. Collectively, these data explain how AriA functions as an ATP-dependent sensor that detects viral proteins and activates the AriB toxin. PARIS is one of an emerging set of immune systems that form macromolecular complexes for the recognition of foreign proteins, rather than foreign nucleic acids.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573921">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573921" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573921">
        <p class="paperTitle">Direct interrogation of context-dependent GPCR activity with a universal biosensor platform</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573921" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573921" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Janicot, R.; Maziarz, M.; Park, J.-C.; Luebbers, A.; Green, E.; Zhao, J.; Philibert, C. E.; Zhang, H.; Layne, M. D.; Wu, J. C.; Garcia-Marcos, M.</p>
        <p class="info">Score: 24.9, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573921' target='https://doi.org/10.1101/2024.01.02.573921'> 10.1101/2024.01.02.573921</a></p>
        <p class="abstract">G protein-coupled receptors (GPCRs) are the largest family of druggable proteins in the human genome, but progress in understanding and targeting them is hindered by the lack of tools to reliably measure their nuanced behavior in physiologically-relevant contexts. Here, we developed a collection of compact ONE vector G-protein Optical (ONE-GO) biosensor constructs as a scalable platform that can be conveniently deployed to measure G-protein activation by virtually any GPCR with high fidelity even when expressed endogenously in primary cells. By characterizing dozens of GPCRs across many cell types like primary cardiovascular cells or neurons, we revealed new insights into the molecular basis for G-protein coupling selectivity of GPCRs, pharmacogenomic profiles of anti-psychotics on naturally-occurring GPCR variants, and G-protein subtype signaling bias by endogenous GPCRs depending on cell type or upon inducing disease-like states. In summary, this open-source platform makes the direct interrogation of context-dependent GPCR activity broadly accessible.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.27.573435">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.27.573435" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.27.573435">
        <p class="paperTitle">Genetically encoded tool for manipulation of Î”{Psi}m identifies the latter as the driver of integrative stress response induced by ATP Synthase dysfunction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.27.573435" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.27.573435" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Choe, M.; Titov, D. V.</p>
        <p class="info">Score: 32.5, Published: 2023-12-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.27.573435' target='https://doi.org/10.1101/2023.12.27.573435'> 10.1101/2023.12.27.573435</a></p>
        <p class="abstract">Mitochondrial membrane potential ({Delta}{Psi}m) is one of the key parameters controlling cellular bioenergetics. Investigation of the role of {Delta}{Psi}m in live cells is complicated by a lack of tools for its direct manipulation without off-target effects. Here, we adopted the uncoupling protein UCP1 from brown adipocytes as a genetically encoded tool for direct manipulation of {Delta}{Psi}m. We validated the ability of exogenously expressed UCP1 to induce uncoupled respiration and lower {Delta}{Psi}m in mammalian cells. UCP1 expression lowered {Delta}{Psi}m to the same extent as chemical uncouplers but did not inhibit cell proliferation, suggesting that it manipulates {Delta}{Psi}m without the off-target effects of chemical uncouplers. Using UCP1, we revealed that elevated {Delta}{Psi}m is the driver of the Integrated Stress Response induced by ATP synthase inhibition in mammalian cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574365">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574365" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574365">
        <p class="paperTitle">Swinging lever mechanism of myosin directly demonstrated by time-resolved cryoEM</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574365" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574365" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Klebl, D. P.; McMillan, S. N.; Risi, C.; Forgacs, E.; Virok, B.; Atherton, J.; Stofella, M.; Winkelmann, D.; Sobott, F.; Galkin, V.; Knight, P. J.; Muench, S. P.; Scarff, C.; White, H. D.</p>
        <p class="info">Score: 16.9, Published: 2024-01-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574365' target='https://doi.org/10.1101/2024.01.05.574365'> 10.1101/2024.01.05.574365</a></p>
        <p class="abstract">Myosins are essential for producing force and movement in cells through their interactions with F-actin. Generation of movement is proposed to occur through structural changes within the myosin motor domain, fuelled by ATP hydrolysis, that are amplified by a lever swing1, transitioning myosin from a primed (pre-powerstroke) state to a post-powerstroke state. However, the initial, primed actomyosin state, proposed to form prior to lever swing, has never been observed. Nor has the mechanism by which actin catalyses myosin ATPase activity been resolved. To address this, we performed time-resolved cryoEM of a myosin-5 mutant having slow hydrolysis product release. Primed actomyosin was captured 10 ms after mixing primed myosin with F-actin, whereas post-powerstroke actomyosin predominated at 120 ms, with no abundant intermediate structures. The structures were solved to 4.4[A] and 4.2[A] global resolution respectively. The primed motor binds to actin through its lower 50 kDa subdomain, with the actin-binding cleft open and Pi release prohibited. N-terminal actin interactions with myosin promote rotation of the upper 50 kDa subdomain, which closes the actin-binding cleft, and enables Pi release. Formation of upper 50 kDa subdomain interactions with actin creates the strong-binding interface required for effective force production. The myosin-5 lever swings through an angle of 93{degrees}, predominantly along the actin axis, with little twisting, to produce the post-powerstroke state. The magnitude of the lever swing matches the typical step length of myosin-5 walking along actin. These time-resolved structures directly demonstrate the swinging lever mechanism, ending decades of conjecture on how myosin produces force and movement.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.574240">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.574240" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.574240">
        <p class="paperTitle">MicroED structure of the C11 cysteine protease Clostripain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.574240" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.574240" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ruma, Y. N.; Bu, G.; Gonen, T.</p>
        <p class="info">Score: 15.2, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.574240' target='https://doi.org/10.1101/2024.01.04.574240'> 10.1101/2024.01.04.574240</a></p>
        <p class="abstract">Clostripain secreted from Clostridium histolyticum is the founding member of the C11 family of Clan CD cysteine peptidases, which is an important group of peptidases secreted by numerous bacteria. Clostripain is an arginine specific endopeptidase. Because of its efficacy as a cysteine peptidase, it is widely used in laboratory settings. Despite its importance the structure of clostripain remains unsolved. Here we describe the first structure of an active form of C. histolyticum Clostripain determined at 3.6 [A] resolution using microcrystal electron diffraction (MicroED). The structure was determined from a single nanocrystal after focused ion beam milling. The structure of Clostripain shows a typical Clan CD /{beta}/ sandwich architecture and the Cys231/His176 catalytic dyad in the active site. It has a large electronegative substrate binding pocket showing its ability to accommodate large and diverse substrates. A loop in the heavy chain formed between residues 452 to 457 is potentially important for substrate binding. In conclusion, this result demonstrates the importance of MicroED to determine the unknown structure of macromolecules such as Clostripain, which can be further used as a platform to study substrate binding and design of potential inhibitors against this class of peptidases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.574565">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.574565" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.574565">
        <p class="paperTitle">Human coronavirus HKU1 recognition of the TMPRSS2 host receptor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.574565" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.574565" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McCallum, M.; Park, Y.-J.; Stewart, C.; Sprouse, K. R.; Brown, J.; Tortorici, M. A.; Gibson, C.; Wong, E.; Ieven, M.; Telenti, A.; Veesler, D.</p>
        <p class="info">Score: 15.0, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.574565' target='https://doi.org/10.1101/2024.01.09.574565'> 10.1101/2024.01.09.574565</a></p>
        <p class="abstract">The human coronavirus HKU1 spike (S) glycoprotein engages host cell surface sialoglycans and transmembrane protease serine 2 (TMPRSS2) to initiate infection. The molecular basis of HKU1 binding to TMPRSS2 and determinants of host receptor tropism remain elusive. Here, we designed an active human TMPRSS2 construct enabling high-yield recombinant production in human cells of this key therapeutic target. We determined a cryo-electron microscopy structure of the HKU1 RBD bound to human TMPRSS2 providing a blueprint of the interactions supporting viral entry and explaining the specificity for TMPRSS2 among human type 2 transmembrane serine proteases. We found that human, rat, hamster and camel TMPRSS2 promote HKU1 S-mediated entry into cells and identified key residues governing host receptor usage. Our data show that serum antibodies targeting the HKU1 RBD TMPRSS2 binding-site are key for neutralization and that HKU1 uses conformational masking and glycan shielding to balance immune evasion and receptor engagement.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.574806">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.574806" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.574806">
        <p class="paperTitle">Quinone extraction drives atmospheric carbon monoxide oxidation in bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.574806" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.574806" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kropp, A.; Gillett, D. L.; Venugopal, H.; Gonzalvez, M. A.; Lingford, J. P.; Barlow, C. K.; Zhang, J.; Greening, C.; Grinter, R.</p>
        <p class="info">Score: 14.3, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.574806' target='https://doi.org/10.1101/2024.01.09.574806'> 10.1101/2024.01.09.574806</a></p>
        <p class="abstract">Diverse bacteria and archaea use atmospheric carbon monoxide (CO) as an energy source during long-term survival. This process enhances the biodiversity of soil and marine ecosystems globally and removes 250 million tonnes of a toxic, climate-relevant pollutant from the atmosphere each year. Bacteria use [MoCu]-carbon monoxide dehydrogenases (Mo-CODH) to convert CO to carbon dioxide, then transfer the liberated high-energy electrons to the aerobic respiratory chain. However, given no high-affinity Mo-CODH has been purified, it is unknown how these enzymes oxidise CO at low concentrations and interact with the respiratory chain. Here we resolve these knowledge gaps by analysing Mo-CODH (CoxSML) and its hypothetical partner CoxG from Mycobacterium smegmatis. Kinetic and electrochemical analyses show purified Mo-CODH is a highly active high-affinity enzyme (Km = 139 nM, kcat = 54.2 s-1). Based on its 1.85 Angstrom resolution cryoEM structure, Mo-CODH forms a CoxSML homodimer similar to characterised low-affinity homologs, but has distinct active site coordination and narrower gas channels that may modulate affinity. We provide structural, biochemical, and genetic evidence that Mo-CODH transfers CO-derived electrons to the aerobic respiratory chain via the membrane-bound menaquinone-binding protein CoxG. Consistently, CoxG is required for CO-driven respiration, extracts menaquinone from mycobacterial membranes, and binds quinones in a hydrophobic pocket. Finally, we show that Mo-CODH and CoxG genetically and structurally associate in diverse bacteria and archaea. These findings reveal the basis of a biogeochemically and ecologically important process, while demonstrating that the newly discovered process of long-range quinone transport is a general mechanism of energy conservation, which convergently evolved on multiple occasions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.31.573774">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.31.573774" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.31.573774">
        <p class="paperTitle">A genetically-encoded fluorescent biosensor for visualization of acetyl-CoA in live cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.31.573774" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.31.573774" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smith, J. J.; Valentino, T. R.; Ablicki, A. H.; Banerjee, R.; Colligan, A. R.; Eckert, D. M.; Desjardins, G. A.; Diehl, K. L.</p>
        <p class="info">Score: 24.1, Published: 2024-01-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.31.573774' target='https://doi.org/10.1101/2023.12.31.573774'> 10.1101/2023.12.31.573774</a></p>
        <p class="abstract">Acetyl-coenzyme A is a central metabolite that participates in many cellular pathways. Evidence suggests that acetyl-CoA production and consumption are highly compartmentalized in mammalian cells. Yet methods to measure acetyl-CoA in living cells are lacking. In this work, we engineer an acetyl-CoA biosensor from the bacterial protein PanZ and circularly permuted green fluorescent protein (cpGFP). We biochemically characterize the sensor and demonstrate its selectivity for acetyl-CoA over other CoA species. We then deploy the biosensor in E. coli and HeLa cells to demonstrate its utility in living cells. In E. coli, we show that the biosensor enables detection of rapid changes in acetyl-CoA levels. In human cells, we show that the biosensor enables subcellular detection and reveals the compartmentalization of acetyl-CoA metabolism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.30.573741">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.30.573741" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.30.573741">
        <p class="paperTitle">Structure of a lasso peptide bound ETB receptor provides insights into the mechanism of GPCR inverse agonism</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.30.573741" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.30.573741" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shihoya, W.; Akasaka, H.; Jordan, P. A.; Lechne, A.; Okada, B. K.; Cruz, G. C. M. d.; Sano, F. K.; Tanaka, T.; Kawahara, R.; Chaudhari, R.; Masamune, H.; Mark, B. J.; Nureki, O.</p>
        <p class="info">Score: 11.0, Published: 2024-01-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.30.573741' target='https://doi.org/10.1101/2023.12.30.573741'> 10.1101/2023.12.30.573741</a></p>
        <p class="abstract">Lasso peptides exhibit a unique lariat-like knotted structure imparting exceptional stability and thus show promise as therapeutic agents that target cell-surface receptors. One such receptor is the human endothelin ETB receptor, which is implicated in challenging cancers with poor immunotherapy responsiveness. The Streptomyces-derived lasso peptide, RES-701-3, is a selective inhibitor for ETB and a compelling candidate for therapeutic development. However, meager production from a genetically recalcitrant host has limited further structure-activity relationship studies of this potent inhibitor. Here, we report cryo-electron microscopy structures of ETB receptor in both its apo form and complex with RES-701-3, facilitated by a calcineurin-fusion strategy. Hydrophobic interactions between RES-701-3 and the transmembrane region of the receptor, especially involving two tryptophan residues, play a crucial role in RES-701-3 binding. Furthermore, RES-701-3 prevents conformational changes associated with G-protein coupling, explaining its inverse agonist activity. A comparative analysis with other lasso peptides and their target proteins highlights the potential of lasso peptides as precise drug candidates for G-protein-coupled receptors. This structural insight into RES-701-3 binding to ETB receptor offers valuable information for the development of novel therapeutics targeting this receptor and provides a broader understanding of lasso peptide interactions with human cell-surface receptors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.573955">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.573955" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.573955">
        <p class="paperTitle">Spatially resolved quantification of oxygen consumption rate in ex vivo lymph node slices</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.573955" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.573955" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Anbaei, P.; Stevens, M. G.; Ball, A. G.; Bullock, T. N. J.; Pompano, R. R.</p>
        <p class="info">Score: 9.1, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.573955' target='https://doi.org/10.1101/2024.01.03.573955'> 10.1101/2024.01.03.573955</a></p>
        <p class="abstract">Cellular metabolism has been closely linked to activation state in cells of the immune system, and the oxygen consumption rate (OCR) in particular serves as a valuable metric for assessing metabolic activity. Several oxygen sensing assays have been reported for cells in standard culture conditions. However, none have provided a spatially resolved, optical measurement of local oxygen consumption in intact tissue samples, making it challenging to understand regional dynamics of consumption. Therefore, here we established a system to monitor the rates of oxygen consumption in ex vivo tissue slices, using murine lymphoid tissue as a case study. By integrating an optical oxygen sensor into a sealed perfusion chamber and incorporating appropriate correction for photobleaching of the sensor and of tissue autofluorescence, we were able to visualize and quantify rates of oxygen consumption in tissue. This method revealed for the first time that the rate of oxygen consumption in naive lymphoid tissue was higher in the T cell region compared to the B cell and cortical regions. To validate the method, we measured OCR in the T cell regions of naive lymph node slices using the optical assay and estimated the consumption rate per cell. The predictions from the optical assay were similar to reported values and were not significantly different from those of the Seahorse metabolic assay, a gold standard method for measuring OCR in cell suspensions. Finally, we used this method to quantify the rate of onset of tissue hypoxia for lymph node slices cultured in a sealed chamber and showed that continuous perfusion was sufficient to maintain oxygenation. In summary, this work establishes a method to monitor oxygen consumption with regional resolution in intact tissue explants, suitable for future use to compare tissue culture conditions and responses to stimulation.

TOC image

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=95 SRC=&#34;FIGDIR/small/573955v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (34K):
org.highwire.dtl.DTLVardef@1e20f5dorg.highwire.dtl.DTLVardef@5eb0f8org.highwire.dtl.DTLVardef@b47541org.highwire.dtl.DTLVardef@17e1033_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
