<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Highly specific intracellular ubiquitination of a small molecule
Authors: Li, W.; Garcia-Rivera, E. M.; Mitchell, D. C.; Chick, J. M.; Maetani, M.; Knapp, J. M.; Matthews, G. M.; Shirasaki, R.; de Matos Simoes, R.; Viswanathan, V.; Pulice, J. L.; Rees, M. G.; Roth, J. A.; Gygi, S. P.; Mitsiades, C. S.; Kadoch, C.; Schreiber, S. L.; Ostrem, J. M. L.
Score: 75.3, Published: 2024-01-28 DOI: 10.1101/2024.01.26.577493
Ubiquitin is a small, highly conserved protein that acts as a posttranslational modification in eukaryotes.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Highly specific intracellular ubiquitination of a small molecule
Authors: Li, W.; Garcia-Rivera, E. M.; Mitchell, D. C.; Chick, J. M.; Maetani, M.; Knapp, J. M.; Matthews, G. M.; Shirasaki, R.; de Matos Simoes, R.; Viswanathan, V.; Pulice, J. L.; Rees, M. G.; Roth, J. A.; Gygi, S. P.; Mitsiades, C. S.; Kadoch, C.; Schreiber, S. L.; Ostrem, J. M. L.
Score: 75.3, Published: 2024-01-28 DOI: 10.1101/2024.01.26.577493
Ubiquitin is a small, highly conserved protein that acts as a posttranslational modification in eukaryotes." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-04T10:38:39+00:00" />
<meta property="article:modified_time" content="2024-02-04T10:38:39+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Highly specific intracellular ubiquitination of a small molecule
Authors: Li, W.; Garcia-Rivera, E. M.; Mitchell, D. C.; Chick, J. M.; Maetani, M.; Knapp, J. M.; Matthews, G. M.; Shirasaki, R.; de Matos Simoes, R.; Viswanathan, V.; Pulice, J. L.; Rees, M. G.; Roth, J. A.; Gygi, S. P.; Mitsiades, C. S.; Kadoch, C.; Schreiber, S. L.; Ostrem, J. M. L.
Score: 75.3, Published: 2024-01-28 DOI: 10.1101/2024.01.26.577493
Ubiquitin is a small, highly conserved protein that acts as a posttranslational modification in eukaryotes."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Highly specific intracellular ubiquitination of a small molecule\nAuthors: Li, W.; Garcia-Rivera, E. M.; Mitchell, D. C.; Chick, J. M.; Maetani, M.; Knapp, J. M.; Matthews, G. M.; Shirasaki, R.; de Matos Simoes, R.; Viswanathan, V.; Pulice, J. L.; Rees, M. G.; Roth, J. A.; Gygi, S. P.; Mitsiades, C. S.; Kadoch, C.; Schreiber, S. L.; Ostrem, J. M. L.\nScore: 75.3, Published: 2024-01-28 DOI: 10.1101/2024.01.26.577493\nUbiquitin is a small, highly conserved protein that acts as a posttranslational modification in eukaryotes.",
  "keywords": [
    
  ],
  "articleBody": " Highly specific intracellular ubiquitination of a small molecule\nAuthors: Li, W.; Garcia-Rivera, E. M.; Mitchell, D. C.; Chick, J. M.; Maetani, M.; Knapp, J. M.; Matthews, G. M.; Shirasaki, R.; de Matos Simoes, R.; Viswanathan, V.; Pulice, J. L.; Rees, M. G.; Roth, J. A.; Gygi, S. P.; Mitsiades, C. S.; Kadoch, C.; Schreiber, S. L.; Ostrem, J. M. L.\nScore: 75.3, Published: 2024-01-28 DOI: 10.1101/2024.01.26.577493\nUbiquitin is a small, highly conserved protein that acts as a posttranslational modification in eukaryotes. Ubiquitination of proteins frequently serves as a degradation signal, marking them for disposal by the proteasome. Here, we report a novel small molecule from a diversity-oriented synthesis library, BRD1732, that is directly ubiquitinated in cells, resulting in dramatic accumulation of inactive ubiquitin monomers and polyubiquitin chains causing broad inhibition of the ubiquitin-proteasome system. Ubiquitination of BRD1732 and its associated cytotoxicity are stereospecific and dependent upon two homologous E3 ubiquitin ligases, RNF19A and RNF19B. Our finding opens the possibility for indirect ubiquitination of a target through a ubiquitinated bifunctional small molecule, and more broadly raises the potential for posttranslational modification in trans.\nLarge-scale characterization of drug mechanism of action using proteome-wide thermal shift assays\nAuthors: Van Vranken, J. G.; Li, J.; Mintseris, J.; Gadzuk-Shea, M.; Gygi, S. P.; Schweppe, D.\nScore: 31.5, Published: 2024-01-27 DOI: 10.1101/2024.01.26.577428\nIn response to an ever-increasing demand of new small molecules therapeutics, numerous chemical and genetic tools have been developed to interrogate compound mechanism of action. Owing to its ability to characterize compound-dependent changes in thermal stability, the proteome-wide thermal shift assay has emerged as a powerful tool in this arsenal. The most recent iterations have drastically improved the overall efficiency of these assays, providing an opportunity to screen compounds at a previously unprecedented rate. Taking advantage of this advance, we quantified 1.498 million thermal stability measurements in response to multiple classes of therapeutic and tool compounds (96 compounds in living cells and 70 compounds in lysates). When interrogating the dataset as a whole, approximately 80% of compounds (with quantifiable targets) caused a significant change in the thermal stability of an annotated target. There was also a wealth of evidence portending off-target engagement despite the extensive use of the compounds in the laboratory and/or clinic. Finally, the combined application of cell- and lysate-based assays, aided in the classification of primary (direct ligand binding) and secondary (indirect) changes in thermal stability. Overall, this study highlights the value of these assays in the drug development process by affording an unbiased and reliable assessment of compound mechanism of action.\nMolecular basis of ligand promiscuity, structural mimicry, and atypical dimerization in the chemokine receptors\nAuthors: Saha, S.; Sano, F. K.; Sharma, S.; Ganguly, M.; Saha, S.; Akasaka, H.; Kobayashi, T. A.; Zaidi, N.; Mishra, S.; Dalal, A.; Mohapatra, S.; Yadav, M.; Itoh, Y.; Leurs, R.; Chevigne, A.; Banerjee, R.; Shihoya, W.; Nureki, O.; SHUKLA, A. K.\nScore: 21.6, Published: 2024-02-02 DOI: 10.1101/2024.02.01.578380\nSelectivity of natural agonists for their cognate receptors is one of the hallmarks of the members of GPCR family, and it is crucial for the specificity of downstream signal-transduction. However, this selectivity often breaks down in the chemokine receptor subfamily, wherein a high degree of promiscuity is observed with one receptor recognizing multiple chemokines and one chemokine binding to multiple receptors. The molecular determinants of such a striking promiscuity for natural ligands in the chemokine-chemokine receptor system remain mostly elusive and represent an important knowledge gap in our current understanding. Here, we carry out a comprehensive transducer-coupling analysis, testing all known C-X-C chemokines on every C-X-C type chemokine receptor, to generate a global fingerprint of the selectivity and promiscuity encoded within this system. Taking lead from our finding, we determined cryo-EM structures of the most promiscuous receptor, CXCR2, in complex with every interacting chemokine, and deciphered the conserved molecular signatures and distinct binding modalities. While most chemokines position themselves on the receptor as a dimer, CXCL6 exhibits a monomeric binding pose induced by a previously unanticipated reorientation of its carboxyl-terminal -helix, leading to disruption of the dimer interface. Surprisingly, one of the chemokines, CXCL5, induces a ligand-swapped dimer of CXCR2, the first of its kind observed in class A GPCRs, wherein each protomer of the ligand engages its own receptor without any discernible receptor-receptor interface. These unique observations provide a possible structural mechanism for inherent functional specialization encoded in chemokines despite their convergence to a common receptor. Furthermore, we also determined cryo-EM structures of CXCR3 in complex with G-protein-biased and {beta}-arrestin-biased small molecule agonists that elucidate distinct allosteric modulations in the receptor driving their divergent transducer-coupling bias. Guided by structural analysis and experimental validation, we discover that in contrast to previously held notion, small molecule agonists of CXCR3 display robust agonism at CXCR7, an intrinsically biased, {beta}-arrestin-coupled receptor, making them first-in-class dual agonists for chemokine receptors with exclusive {beta}arr-bias at CXCR7. Taken together, our study provides molecular insights into ligand promiscuity and signaling bias at the chemokine receptors, and also demonstrates a proof of principle that naturally encoded structural mimicry can be recapitulated using synthetic pharmacophores with potential implications for developing novel therapeutics.\nACKR3 Proximity Labeling Identifies Novel G protein- and β-arrestin-independent GPCR Interacting Proteins\nAuthors: Hicks, C.; Gardner, J.; Eiger, D. S.; Camarda, N.; Pham, U.; Dhar, S.; Rodriguez, H.; Chundi, A.; Rajagopal, S.\nScore: 20.1, Published: 2024-01-28 DOI: 10.1101/2024.01.27.577545\nThe canonical paradigm of GPCR signaling recognizes G proteins and {beta}-arrestins as the two primary transducers that promote GPCR signaling. Recent evidence suggests the atypical chemokine receptor 3 (ACKR3) does not couple to G proteins, and {beta}-arrestins are dispensable for some of its functions. Here, we employed proximity labeling to identify proteins that interact with ACKR3 in cells devoid of {beta}-arrestin. We identified proteins involved in the endocytic machinery and evaluated a subset of proteins conserved across several GPCR-based proximity labeling experiments. We discovered that the bone morphogenic protein 2-inducible kinase (BMP2K) interacts with many different GPCRs with varying dependency on {beta}-arrestin. Together, our work highlights the existence of modulators that can act independently of G proteins and {beta}-arrestins to regulate GPCR signaling and provides important evidence for other targets that may regulate GPCR signaling. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=190 SRC=\"FIGDIR/small/577545v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (36K): org.highwire.dtl.DTLVardef@c61306org.highwire.dtl.DTLVardef@b2cdcorg.highwire.dtl.DTLVardef@1ae29cborg.highwire.dtl.DTLVardef@13a04a1_HPS_FORMAT_FIGEXP M_FIG C_FIG\nChallenging the Astral mass analyzer - going beyond 5200 proteins per single-cell at unseen quantitative accuracy to study cellular heterogeneity.\nAuthors: Bubis, J. A.; Arrey, T. N.; Damoc, E.; Delanghe, B.; Slovakova, J.; Sommer, T. M.; Kagawa, H.; Pichler, P.; Rivron, N. C.; Mechtler, K.; Matzinger, M.\nScore: 18.6, Published: 2024-02-01 DOI: 10.1101/2024.02.01.578358\nA detailed proteome map is crucial for understanding molecular pathways and protein functions. Despite significant advancements in sample preparation, instrumentation, and data analysis, single-cell proteomics is currently limited by proteomic depth and quantitative performance. We combine a zero dead-end volume chromatographic column running at high throughput with the Thermo Scientific Orbitrap Astral mass spectrometer running in DIA mode. We yield unprecedented depth of proteome coverage as well as accuracy and precision for quantification of ultra-low input amounts. Using a tailored library, we identify up to 7400 protein groups from as little as 250 pg HeLa at a throughput of 50 samples per day (SPD). We benchmark multiple data analysis strategies, estimate their influence on FDR and show that FDR on protein level can easily be maintained at 1 %. Using a two-proteome mix, we check for optimal parameters of quantification and show that fold change differences of 2 can still be successfully determined at single-cell level inputs. Eventually, we apply our workflow to A549 cells yielding a proteome coverage of up to 5200 protein groups from a single cell, which allows the observation of heterogeneity in a cellular population and studying dependencies between cell size and cell-cycle phase. Additionally, our workflow enables us to distinguish between in vitro analogs of two human blastocyst lineages: naive human pluripotent stem cells (epiblast) and trophectoderm (TE)-like cells. Gene Ontology analysis of enriched proteins in TE-like cells harmoniously aligns with transcriptomic data, indicating that single-cell proteomics possesses the capability to identify biologically relevant differences between these two lineages within the blastocyst.\nVisualizing chaperone-mediated multistep assembly of the human 20S proteasome\nAuthors: Adolf, F.; Du, J.; Goodall, E. A.; Walsh, R. M.; rawson, S.; von Gronaua, S.; Harper, J. W.; Hanna, J.; Schulman, B. A.\nScore: 15.6, Published: 2024-01-28 DOI: 10.1101/2024.01.27.577538\nDedicated assembly factors orchestrate stepwise production of many molecular machines, including the 28-subunit proteasome core particle (CP) that mediates protein degradation. Here, we report cryo-EM reconstructions of seven recombinant human subcomplexes that visualize all five chaperones and the three active site propeptides across a wide swath of the assembly pathway. Comparison of these chaperone-bound intermediates and a matching mature CP reveals molecular mechanisms determining the order of successive subunit additions, and how proteasome subcomplexes and assembly factors structurally adapt upon progressive subunit incorporation to stabilize intermediates, facilitate the formation of subsequent intermediates, and ultimately rearrange to coordinate proteolytic activation with gated access to active sites. The structural findings reported here explain many previous biochemical and genetic observations. This work establishes a methodologic approach for structural analysis of multiprotein complex assembly intermediates, illuminates specific functions of assembly factors, and reveals conceptual principles underlying human proteasome biogenesis.\nDegradome analysis to identify direct protein substrates of small-molecule degraders\nAuthors: Jochem, M.; Schrempf, A.; Wagner, L.-M.; Cisneros, J.; Ng, A.; Winter, G. E.; Krijgsveld, J.\nScore: 11.5, Published: 2024-01-28 DOI: 10.1101/2024.01.28.577572\nTargeted protein degradation (TPD) has emerged as a powerful strategy to selectively eliminate cellular proteins using small-molecule degraders, offering therapeutic promise for targeting proteins that are otherwise undruggable. However, a remaining challenge is to unambiguously identify primary TPD targets that are distinct from secondary downstream effects in the proteome. Here we introduce an approach that combines stable isotope labeling and click-chemistry for selective quantification of protein degradation by mass spectrometry, excluding confounding effects of altered transcription and translation induced by target depletion. We show that the approach efficiently operates at the time scale of TPD (hours) and we demonstrate its utility by analyzing the Cyclin K degraders dCeMM2 and dCeMM4, which induce widespread transcriptional downregulation, and the GSPT1 degrader CC-885, an inhibitor of protein translation. Additionally, we apply it to characterize compound 1, a previously uncharacterized degrader, and identify the zinc-finger protein FIZ1 as a degraded target.\nThe structure of the Orm2-containing serine palmitoyltransferase complex reveals distinct inhibitory potentials of yeast Orm proteins.\nAuthors: Körner, C.; Schäfer, J.-H.; Esch, B. M.; Parey, K.; Walter, S.; Teis, D.; Januliene, D.; Schmidt, O.; Moeller, A.; Fröhlich, F.\nScore: 10.0, Published: 2024-01-30 DOI: 10.1101/2024.01.30.577963\nThe levels of sphingolipids are crucial determinants of neurodegenerative disorders. Therefore, sphingolipid levels have to be tightly regulated. The Orm protein family and ceramides act as inhibitors in the rate-limiting step of sphingolipid biosynthesis - the condensation of L-serine and palmitoyl-CoA. The two yeast isoforms Orm1 and Orm2 form a complex with the serine palmitoyltransferase (SPT). While the sequences of the Orm proteins are highly similar, however, Orm1 and Orm2 are differentially regulated in yeast cells. The mechanistic details of the differential regulation remain elusive. To elucidate the regulatory mechanism, we determined the cryo-electron microscopy structure of the SPT complex containing Orm2. Through in vitro activity assays and a newly developed plasmid coverage assay combined with targeted lipidomics, we demonstrate that the SPT complex with Orm2 exhibits lower activity in comparison to the complex containing Orm1. This collectively suggests a higher inhibitory potential of Orm2, despite the remarkably similar structures of the Orm1- and Orm2 containing complexes. The high conservation of the SPT from yeast to man implies that the three human ORMDL isoforms could also have different regulatory capacities, which might be essential to understanding their role in sphingolipid-mediated neurodegenerative disorders. HighlightsO_LICryo-EM structures of serine palmitoyltransferase with Orm2 from S. cerevisiae C_LIO_LICeramide-mediated inhibition on SPT activity is conserved amongst SPTs C_LIO_LIThe two yeast Orm isoforms have different regulatory capacity C_LI\nRapid QC-MS - Interactive Dashboard for Synchronous Mass Spectrometry Data Acquisition Quality Control\nAuthors: Sandhu, W.; Gray, I. J.; Lin, S.; Elias, J. E.; DeFelice, B. C.\nScore: 9.4, Published: 2024-02-01 DOI: 10.1101/2024.01.30.578059\nConsistently collecting high quality liquid chromatography-coupled tandem mass spectrometry (LC-MS/MS) data is a time consuming hurdle for untargeted workflows. Analytical controls such as internal and biological standards are commonly included in high throughput workflows, helping researchers recognize low integrity specimens regardless of their biological source. However, evaluating these standards as data are collected has remained a considerable bottleneck - in both person hours and accuracy. Here we present Rapid QC-MS, an automated, interactive dashboard for assessing LC-MS/MS data quality. Minutes after a new data file is written, a browser-viewable dashboard is updated with quality control results spanning multiple performance dimensions such as instrument sensitivity, in-run retention time shifts, and mass accuracy drift. Rapid QC-MS provides interactive visualizations that help users recognize acute deviations in these performance metrics, as well as gradual drifts over periods of hours, days, months, or years. Rapid QC-MS is open-source, simple to install, and highly configurable. By integrating open source python libraries and widely used MS analysis software, it can adapt to any LC-MS/MS workflow. Rapid QC-MS runs locally and offers optional remote quality control by syncing with Google Drive. Furthermore, Rapid QC-MS can operate in a semiautonomous fashion, alerting users to specimens with potentially poor analytical integrity via frequently used messaging applications. Rapid QC-MS offers a fast, straightforward approach to help users collect high quality untargeted LC-MS/MS data by eliminating many of the most time consuming steps in manual data curation. Download for free: https://github.com/czbiohub-sf/Rapid-QC-MS\nLate consolidation of rRNA structure during co-transcriptional assembly in E. coli by time-resolved DMS footprinting\nAuthors: Hao, Y.; Hulscher, R. M.; Zinshteyn, B.; Woodson, S.\nScore: 9.8, Published: 2024-01-10 DOI: 10.1101/2024.01.10.574868\nThe production of new ribosomes requires proper folding of the rRNA and the addition of more than 50 ribosomal proteins. The structures of some assembly intermediates have been determined by cryo-electron microscopy, yet these structures do not provide information on the folding dynamics of the rRNA. To visualize the changes in rRNA structure during ribosome assembly in E. coli cells, transcripts were pulse-labeled with 4-thiouridine and the structure of newly made rRNA probed at various times by dimethyl sulfate modification and mutational profiling sequencing (4U-DMS-MaPseq). The in-cell DMS modification patterns revealed that many long-range rRNA tertiary interactions and protein binding sites through the 16S and 23S rRNA remain partially unfolded 1.5 min after transcription. By contrast, the active sites were continually shielded from DMS modification, suggesting that these critical regions are guarded by cellular factors throughout assembly. Later, bases near the peptidyl tRNA site exhibited specific rearrangements consistent with the binding and release of assembly factors. Time-dependent structure-probing in cells suggests that many tertiary interactions throughout the new ribosomal subunits remain mobile or unfolded until the late stages of subunit maturation.\n",
  "wordCount" : "2458",
  "inLanguage": "en",
  "datePublished": "2024-02-04T10:38:39Z",
  "dateModified": "2024-02-04T10:38:39Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 4, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.26.577493">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.26.577493" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.26.577493">
        <p class="paperTitle">Highly specific intracellular ubiquitination of a small molecule</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.26.577493" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.26.577493" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, W.; Garcia-Rivera, E. M.; Mitchell, D. C.; Chick, J. M.; Maetani, M.; Knapp, J. M.; Matthews, G. M.; Shirasaki, R.; de Matos Simoes, R.; Viswanathan, V.; Pulice, J. L.; Rees, M. G.; Roth, J. A.; Gygi, S. P.; Mitsiades, C. S.; Kadoch, C.; Schreiber, S. L.; Ostrem, J. M. L.</p>
        <p class="info">Score: 75.3, Published: 2024-01-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.26.577493' target='https://doi.org/10.1101/2024.01.26.577493'> 10.1101/2024.01.26.577493</a></p>
        <p class="abstract">Ubiquitin is a small, highly conserved protein that acts as a posttranslational modification in eukaryotes. Ubiquitination of proteins frequently serves as a degradation signal, marking them for disposal by the proteasome. Here, we report a novel small molecule from a diversity-oriented synthesis library, BRD1732, that is directly ubiquitinated in cells, resulting in dramatic accumulation of inactive ubiquitin monomers and polyubiquitin chains causing broad inhibition of the ubiquitin-proteasome system. Ubiquitination of BRD1732 and its associated cytotoxicity are stereospecific and dependent upon two homologous E3 ubiquitin ligases, RNF19A and RNF19B. Our finding opens the possibility for indirect ubiquitination of a target through a ubiquitinated bifunctional small molecule, and more broadly raises the potential for posttranslational modification in trans.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.26.577428">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.26.577428" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.26.577428">
        <p class="paperTitle">Large-scale characterization of drug mechanism of action using proteome-wide thermal shift assays</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.26.577428" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.26.577428" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Van Vranken, J. G.; Li, J.; Mintseris, J.; Gadzuk-Shea, M.; Gygi, S. P.; Schweppe, D.</p>
        <p class="info">Score: 31.5, Published: 2024-01-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.26.577428' target='https://doi.org/10.1101/2024.01.26.577428'> 10.1101/2024.01.26.577428</a></p>
        <p class="abstract">In response to an ever-increasing demand of new small molecules therapeutics, numerous chemical and genetic tools have been developed to interrogate compound mechanism of action. Owing to its ability to characterize compound-dependent changes in thermal stability, the proteome-wide thermal shift assay has emerged as a powerful tool in this arsenal. The most recent iterations have drastically improved the overall efficiency of these assays, providing an opportunity to screen compounds at a previously unprecedented rate. Taking advantage of this advance, we quantified 1.498 million thermal stability measurements in response to multiple classes of therapeutic and tool compounds (96 compounds in living cells and 70 compounds in lysates). When interrogating the dataset as a whole, approximately 80% of compounds (with quantifiable targets) caused a significant change in the thermal stability of an annotated target. There was also a wealth of evidence portending off-target engagement despite the extensive use of the compounds in the laboratory and/or clinic. Finally, the combined application of cell- and lysate-based assays, aided in the classification of primary (direct ligand binding) and secondary (indirect) changes in thermal stability. Overall, this study highlights the value of these assays in the drug development process by affording an unbiased and reliable assessment of compound mechanism of action.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578380">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578380" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578380">
        <p class="paperTitle">Molecular basis of ligand promiscuity, structural mimicry, and atypical dimerization in the chemokine receptors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578380" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578380" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Saha, S.; Sano, F. K.; Sharma, S.; Ganguly, M.; Saha, S.; Akasaka, H.; Kobayashi, T. A.; Zaidi, N.; Mishra, S.; Dalal, A.; Mohapatra, S.; Yadav, M.; Itoh, Y.; Leurs, R.; Chevigne, A.; Banerjee, R.; Shihoya, W.; Nureki, O.; SHUKLA, A. K.</p>
        <p class="info">Score: 21.6, Published: 2024-02-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578380' target='https://doi.org/10.1101/2024.02.01.578380'> 10.1101/2024.02.01.578380</a></p>
        <p class="abstract">Selectivity of natural agonists for their cognate receptors is one of the hallmarks of the members of GPCR family, and it is crucial for the specificity of downstream signal-transduction. However, this selectivity often breaks down in the chemokine receptor subfamily, wherein a high degree of promiscuity is observed with one receptor recognizing multiple chemokines and one chemokine binding to multiple receptors. The molecular determinants of such a striking promiscuity for natural ligands in the chemokine-chemokine receptor system remain mostly elusive and represent an important knowledge gap in our current understanding. Here, we carry out a comprehensive transducer-coupling analysis, testing all known C-X-C chemokines on every C-X-C type chemokine receptor, to generate a global fingerprint of the selectivity and promiscuity encoded within this system. Taking lead from our finding, we determined cryo-EM structures of the most promiscuous receptor, CXCR2, in complex with every interacting chemokine, and deciphered the conserved molecular signatures and distinct binding modalities. While most chemokines position themselves on the receptor as a dimer, CXCL6 exhibits a monomeric binding pose induced by a previously unanticipated reorientation of its carboxyl-terminal -helix, leading to disruption of the dimer interface. Surprisingly, one of the chemokines, CXCL5, induces a ligand-swapped dimer of CXCR2, the first of its kind observed in class A GPCRs, wherein each protomer of the ligand engages its own receptor without any discernible receptor-receptor interface. These unique observations provide a possible structural mechanism for inherent functional specialization encoded in chemokines despite their convergence to a common receptor. Furthermore, we also determined cryo-EM structures of CXCR3 in complex with G-protein-biased and {beta}-arrestin-biased small molecule agonists that elucidate distinct allosteric modulations in the receptor driving their divergent transducer-coupling bias. Guided by structural analysis and experimental validation, we discover that in contrast to previously held notion, small molecule agonists of CXCR3 display robust agonism at CXCR7, an intrinsically biased, {beta}-arrestin-coupled receptor, making them first-in-class dual agonists for chemokine receptors with exclusive {beta}arr-bias at CXCR7. Taken together, our study provides molecular insights into ligand promiscuity and signaling bias at the chemokine receptors, and also demonstrates a proof of principle that naturally encoded structural mimicry can be recapitulated using synthetic pharmacophores with potential implications for developing novel therapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.27.577545">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.27.577545" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.27.577545">
        <p class="paperTitle">ACKR3 Proximity Labeling Identifies Novel G protein- and β-arrestin-independent GPCR Interacting Proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.27.577545" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.27.577545" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hicks, C.; Gardner, J.; Eiger, D. S.; Camarda, N.; Pham, U.; Dhar, S.; Rodriguez, H.; Chundi, A.; Rajagopal, S.</p>
        <p class="info">Score: 20.1, Published: 2024-01-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.27.577545' target='https://doi.org/10.1101/2024.01.27.577545'> 10.1101/2024.01.27.577545</a></p>
        <p class="abstract">The canonical paradigm of GPCR signaling recognizes G proteins and {beta}-arrestins as the two primary transducers that promote GPCR signaling. Recent evidence suggests the atypical chemokine receptor 3 (ACKR3) does not couple to G proteins, and {beta}-arrestins are dispensable for some of its functions. Here, we employed proximity labeling to identify proteins that interact with ACKR3 in cells devoid of {beta}-arrestin. We identified proteins involved in the endocytic machinery and evaluated a subset of proteins conserved across several GPCR-based proximity labeling experiments. We discovered that the bone morphogenic protein 2-inducible kinase (BMP2K) interacts with many different GPCRs with varying dependency on {beta}-arrestin. Together, our work highlights the existence of modulators that can act independently of G proteins and {beta}-arrestins to regulate GPCR signaling and provides important evidence for other targets that may regulate GPCR signaling.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=190 SRC=&#34;FIGDIR/small/577545v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (36K):
org.highwire.dtl.DTLVardef@c61306org.highwire.dtl.DTLVardef@b2cdcorg.highwire.dtl.DTLVardef@1ae29cborg.highwire.dtl.DTLVardef@13a04a1_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578358">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578358" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578358">
        <p class="paperTitle">Challenging the Astral mass analyzer - going beyond 5200 proteins per single-cell at unseen quantitative accuracy to study cellular heterogeneity.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578358" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578358" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bubis, J. A.; Arrey, T. N.; Damoc, E.; Delanghe, B.; Slovakova, J.; Sommer, T. M.; Kagawa, H.; Pichler, P.; Rivron, N. C.; Mechtler, K.; Matzinger, M.</p>
        <p class="info">Score: 18.6, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578358' target='https://doi.org/10.1101/2024.02.01.578358'> 10.1101/2024.02.01.578358</a></p>
        <p class="abstract">A detailed proteome map is crucial for understanding molecular pathways and protein functions. Despite significant advancements in sample preparation, instrumentation, and data analysis, single-cell proteomics is currently limited by proteomic depth and quantitative performance. We combine a zero dead-end volume chromatographic column running at high throughput with the Thermo Scientific Orbitrap Astral mass spectrometer running in DIA mode. We yield unprecedented depth of proteome coverage as well as accuracy and precision for quantification of ultra-low input amounts. Using a tailored library, we identify up to 7400 protein groups from as little as 250 pg HeLa at a throughput of 50 samples per day (SPD). We benchmark multiple data analysis strategies, estimate their influence on FDR and show that FDR on protein level can easily be maintained at 1 %. Using a two-proteome mix, we check for optimal parameters of quantification and show that fold change differences of 2 can still be successfully determined at single-cell level inputs. Eventually, we apply our workflow to A549 cells yielding a proteome coverage of up to 5200 protein groups from a single cell, which allows the observation of heterogeneity in a cellular population and studying dependencies between cell size and cell-cycle phase. Additionally, our workflow enables us to distinguish between in vitro analogs of two human blastocyst lineages: naive human pluripotent stem cells (epiblast) and trophectoderm (TE)-like cells. Gene Ontology analysis of enriched proteins in TE-like cells harmoniously aligns with transcriptomic data, indicating that single-cell proteomics possesses the capability to identify biologically relevant differences between these two lineages within the blastocyst.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.27.577538">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.27.577538" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.27.577538">
        <p class="paperTitle">Visualizing chaperone-mediated multistep assembly of the human 20S proteasome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.27.577538" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.27.577538" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Adolf, F.; Du, J.; Goodall, E. A.; Walsh, R. M.; rawson, S.; von Gronaua, S.; Harper, J. W.; Hanna, J.; Schulman, B. A.</p>
        <p class="info">Score: 15.6, Published: 2024-01-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.27.577538' target='https://doi.org/10.1101/2024.01.27.577538'> 10.1101/2024.01.27.577538</a></p>
        <p class="abstract">Dedicated assembly factors orchestrate stepwise production of many molecular machines, including the 28-subunit proteasome core particle (CP) that mediates protein degradation. Here, we report cryo-EM reconstructions of seven recombinant human subcomplexes that visualize all five chaperones and the three active site propeptides across a wide swath of the assembly pathway. Comparison of these chaperone-bound intermediates and a matching mature CP reveals molecular mechanisms determining the order of successive subunit additions, and how proteasome subcomplexes and assembly factors structurally adapt upon progressive subunit incorporation to stabilize intermediates, facilitate the formation of subsequent intermediates, and ultimately rearrange to coordinate proteolytic activation with gated access to active sites. The structural findings reported here explain many previous biochemical and genetic observations. This work establishes a methodologic approach for structural analysis of multiprotein complex assembly intermediates, illuminates specific functions of assembly factors, and reveals conceptual principles underlying human proteasome biogenesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.28.577572">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.28.577572" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.28.577572">
        <p class="paperTitle">Degradome analysis to identify direct protein substrates of small-molecule degraders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.28.577572" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.28.577572" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jochem, M.; Schrempf, A.; Wagner, L.-M.; Cisneros, J.; Ng, A.; Winter, G. E.; Krijgsveld, J.</p>
        <p class="info">Score: 11.5, Published: 2024-01-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.28.577572' target='https://doi.org/10.1101/2024.01.28.577572'> 10.1101/2024.01.28.577572</a></p>
        <p class="abstract">Targeted protein degradation (TPD) has emerged as a powerful strategy to selectively eliminate cellular proteins using small-molecule degraders, offering therapeutic promise for targeting proteins that are otherwise undruggable. However, a remaining challenge is to unambiguously identify primary TPD targets that are distinct from secondary downstream effects in the proteome. Here we introduce an approach that combines stable isotope labeling and click-chemistry for selective quantification of protein degradation by mass spectrometry, excluding confounding effects of altered transcription and translation induced by target depletion. We show that the approach efficiently operates at the time scale of TPD (hours) and we demonstrate its utility by analyzing the Cyclin K degraders dCeMM2 and dCeMM4, which induce widespread transcriptional downregulation, and the GSPT1 degrader CC-885, an inhibitor of protein translation. Additionally, we apply it to characterize compound 1, a previously uncharacterized degrader, and identify the zinc-finger protein FIZ1 as a degraded target.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.30.577963">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.30.577963" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.30.577963">
        <p class="paperTitle">The structure of the Orm2-containing serine palmitoyltransferase complex reveals distinct inhibitory potentials of yeast Orm proteins.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.30.577963" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.30.577963" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Körner, C.; Schäfer, J.-H.; Esch, B. M.; Parey, K.; Walter, S.; Teis, D.; Januliene, D.; Schmidt, O.; Moeller, A.; Fröhlich, F.</p>
        <p class="info">Score: 10.0, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.30.577963' target='https://doi.org/10.1101/2024.01.30.577963'> 10.1101/2024.01.30.577963</a></p>
        <p class="abstract">The levels of sphingolipids are crucial determinants of neurodegenerative disorders. Therefore, sphingolipid levels have to be tightly regulated. The Orm protein family and ceramides act as inhibitors in the rate-limiting step of sphingolipid biosynthesis - the condensation of L-serine and palmitoyl-CoA. The two yeast isoforms Orm1 and Orm2 form a complex with the serine palmitoyltransferase (SPT). While the sequences of the Orm proteins are highly similar, however, Orm1 and Orm2 are differentially regulated in yeast cells. The mechanistic details of the differential regulation remain elusive. To elucidate the regulatory mechanism, we determined the cryo-electron microscopy structure of the SPT complex containing Orm2. Through in vitro activity assays and a newly developed plasmid coverage assay combined with targeted lipidomics, we demonstrate that the SPT complex with Orm2 exhibits lower activity in comparison to the complex containing Orm1. This collectively suggests a higher inhibitory potential of Orm2, despite the remarkably similar structures of the Orm1- and Orm2 containing complexes. The high conservation of the SPT from yeast to man implies that the three human ORMDL isoforms could also have different regulatory capacities, which might be essential to understanding their role in sphingolipid-mediated neurodegenerative disorders.

HighlightsO_LICryo-EM structures of serine palmitoyltransferase with Orm2 from S. cerevisiae
C_LIO_LICeramide-mediated inhibition on SPT activity is conserved amongst SPTs
C_LIO_LIThe two yeast Orm isoforms have different regulatory capacity
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.30.578059">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.30.578059" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.30.578059">
        <p class="paperTitle">Rapid QC-MS - Interactive Dashboard for Synchronous Mass Spectrometry Data Acquisition Quality Control</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.30.578059" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.30.578059" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sandhu, W.; Gray, I. J.; Lin, S.; Elias, J. E.; DeFelice, B. C.</p>
        <p class="info">Score: 9.4, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.30.578059' target='https://doi.org/10.1101/2024.01.30.578059'> 10.1101/2024.01.30.578059</a></p>
        <p class="abstract">Consistently collecting high quality liquid chromatography-coupled tandem mass spectrometry (LC-MS/MS) data is a time consuming hurdle for untargeted workflows. Analytical controls such as internal and biological standards are commonly included in high throughput workflows, helping researchers recognize low integrity specimens regardless of their biological source. However, evaluating these standards as data are collected has remained a considerable bottleneck - in both person hours and accuracy. Here we present Rapid QC-MS, an automated, interactive dashboard for assessing LC-MS/MS data quality. Minutes after a new data file is written, a browser-viewable dashboard is updated with quality control results spanning multiple performance dimensions such as instrument sensitivity, in-run retention time shifts, and mass accuracy drift. Rapid QC-MS provides interactive visualizations that help users recognize acute deviations in these performance metrics, as well as gradual drifts over periods of hours, days, months, or years. Rapid QC-MS is open-source, simple to install, and highly configurable. By integrating open source python libraries and widely used MS analysis software, it can adapt to any LC-MS/MS workflow. Rapid QC-MS runs locally and offers optional remote quality control by syncing with Google Drive. Furthermore, Rapid QC-MS can operate in a semiautonomous fashion, alerting users to specimens with potentially poor analytical integrity via frequently used messaging applications. Rapid QC-MS offers a fast, straightforward approach to help users collect high quality untargeted LC-MS/MS data by eliminating many of the most time consuming steps in manual data curation. Download for free: https://github.com/czbiohub-sf/Rapid-QC-MS</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.574868">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.574868" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.574868">
        <p class="paperTitle">Late consolidation of rRNA structure during co-transcriptional assembly in E. coli by time-resolved DMS footprinting</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.574868" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.574868" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hao, Y.; Hulscher, R. M.; Zinshteyn, B.; Woodson, S.</p>
        <p class="info">Score: 9.8, Published: 2024-01-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.574868' target='https://doi.org/10.1101/2024.01.10.574868'> 10.1101/2024.01.10.574868</a></p>
        <p class="abstract">The production of new ribosomes requires proper folding of the rRNA and the addition of more than 50 ribosomal proteins. The structures of some assembly intermediates have been determined by cryo-electron microscopy, yet these structures do not provide information on the folding dynamics of the rRNA. To visualize the changes in rRNA structure during ribosome assembly in E. coli cells, transcripts were pulse-labeled with 4-thiouridine and the structure of newly made rRNA probed at various times by dimethyl sulfate modification and mutational profiling sequencing (4U-DMS-MaPseq). The in-cell DMS modification patterns revealed that many long-range rRNA tertiary interactions and protein binding sites through the 16S and 23S rRNA remain partially unfolded 1.5 min after transcription. By contrast, the active sites were continually shielded from DMS modification, suggesting that these critical regions are guarded by cellular factors throughout assembly. Later, bases near the peptidyl tRNA site exhibited specific rearrangements consistent with the binding and release of assembly factors. Time-dependent structure-probing in cells suggests that many tertiary interactions throughout the new ribosomal subunits remain mobile or unfolded until the late stages of subunit maturation.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
