<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Intrinsically Disordered Regions Promote Protein Refoldability and Facilitate Retrieval from Biomolecular Condensates
Authors: To, P.; Bhagwat, A. M.; Tarbox, H. E.; Ecer, A.; Wendorff, H.; Jamieson, Z.; Trcek, T.; Fried, S. D.
Score: 129.0, Published: 2023-06-25 DOI: 10.1101/2023.06.25.546465
Many eukaryotic proteins contain intrinsically disordered regions (IDRs) that intersperse globular folded domains, in contrast with bacterial proteins which are typically highly globular1, 2. Recent years have seen great progress in identifying biological functions associated with these elusive protein sequence: in specific cases, they mediate liquid liquid phase separation3, perform molecular recognition4, or act as sensors to changes in the environment5.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Intrinsically Disordered Regions Promote Protein Refoldability and Facilitate Retrieval from Biomolecular Condensates
Authors: To, P.; Bhagwat, A. M.; Tarbox, H. E.; Ecer, A.; Wendorff, H.; Jamieson, Z.; Trcek, T.; Fried, S. D.
Score: 129.0, Published: 2023-06-25 DOI: 10.1101/2023.06.25.546465
Many eukaryotic proteins contain intrinsically disordered regions (IDRs) that intersperse globular folded domains, in contrast with bacterial proteins which are typically highly globular1, 2. Recent years have seen great progress in identifying biological functions associated with these elusive protein sequence: in specific cases, they mediate liquid liquid phase separation3, perform molecular recognition4, or act as sensors to changes in the environment5." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-05T10:42:43+00:00" />
<meta property="article:modified_time" content="2023-07-05T10:42:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Intrinsically Disordered Regions Promote Protein Refoldability and Facilitate Retrieval from Biomolecular Condensates
Authors: To, P.; Bhagwat, A. M.; Tarbox, H. E.; Ecer, A.; Wendorff, H.; Jamieson, Z.; Trcek, T.; Fried, S. D.
Score: 129.0, Published: 2023-06-25 DOI: 10.1101/2023.06.25.546465
Many eukaryotic proteins contain intrinsically disordered regions (IDRs) that intersperse globular folded domains, in contrast with bacterial proteins which are typically highly globular1, 2. Recent years have seen great progress in identifying biological functions associated with these elusive protein sequence: in specific cases, they mediate liquid liquid phase separation3, perform molecular recognition4, or act as sensors to changes in the environment5."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Intrinsically Disordered Regions Promote Protein Refoldability and Facilitate Retrieval from Biomolecular Condensates\nAuthors: To, P.; Bhagwat, A. M.; Tarbox, H. E.; Ecer, A.; Wendorff, H.; Jamieson, Z.; Trcek, T.; Fried, S. D.\nScore: 129.0, Published: 2023-06-25 DOI: 10.1101/2023.06.25.546465\nMany eukaryotic proteins contain intrinsically disordered regions (IDRs) that intersperse globular folded domains, in contrast with bacterial proteins which are typically highly globular1, 2. Recent years have seen great progress in identifying biological functions associated with these elusive protein sequence: in specific cases, they mediate liquid liquid phase separation3, perform molecular recognition4, or act as sensors to changes in the environment5.",
  "keywords": [
    
  ],
  "articleBody": " Intrinsically Disordered Regions Promote Protein Refoldability and Facilitate Retrieval from Biomolecular Condensates\nAuthors: To, P.; Bhagwat, A. M.; Tarbox, H. E.; Ecer, A.; Wendorff, H.; Jamieson, Z.; Trcek, T.; Fried, S. D.\nScore: 129.0, Published: 2023-06-25 DOI: 10.1101/2023.06.25.546465\nMany eukaryotic proteins contain intrinsically disordered regions (IDRs) that intersperse globular folded domains, in contrast with bacterial proteins which are typically highly globular1, 2. Recent years have seen great progress in identifying biological functions associated with these elusive protein sequence: in specific cases, they mediate liquid liquid phase separation3, perform molecular recognition4, or act as sensors to changes in the environment5. Nevertheless, only a small number of IDRs have annotated functions6 despite their presence in 64% of yeast proteins,7 stimulating some to question what general purpose they may serve8, 9. Here, by interrogating the refoldability of two fungal proteomes (Saccharomyces cerevisiae and Neurosporra crassa), we show that IDRs render their host proteins more refoldable from the denatured state, allowing them to cohere more closely to Anfinsens thermodynamic hypothesis10, 11. The data provide an exceptionally clear picture of which biophysical and topological characteristics enable refoldability. Moreover, we find that almost all yeast proteins that partition into stress granules during heat shock are refoldable, a finding that holds for other condensates such as P-bodies and the nucleolus. Finally, we find that the Hsp104 unfoldase12 is the principal actor in mediating disassembly of heat stress granules and that the efficiency with which condensed proteins are returned to the soluble phase is also well explained by refoldability. Hence, these studies establish spontaneous refoldability as an adaptive trait that endows proteins with the capacity to reform their native soluble structures following their extraction from condensates. Altogether, our results provide an intuitive model for the function of IDRs in many multidomain proteins and clarifies their relationship to the phenomenon of biomolecular condensation.\nAn engineered biosensor enables dynamic aspartate measurements in living cells\nAuthors: Davidsen, K.; Marvin, J. S.; Aggarwal, A.; Brown, T. A.; Sullivan, L. B.\nScore: 18.6, Published: 2023-06-27 DOI: 10.1101/2023.06.27.546775\nIntracellular levels of the amino acid aspartate are responsive to changes in metabolism in mammalian cells and can correspondingly alter cell function, highlighting the need for robust tools to measure aspartate abundance. However, comprehensive understanding of aspartate metabolism has been limited by the throughput, cost, and static nature of the mass spectrometry based measurements that are typically employed to measure aspartate levels. To address these issues, we have developed a GFP-based sensor of aspartate (jAspSnFR3), where the fluorescence intensity corresponds to aspartate concentration. As a purified protein, the sensor has a 20-fold increase in fluorescence upon aspartate saturation, with dose dependent fluorescence changes covering a physiologically relevant aspartate concentration range and no significant off target binding. Expressed in mammalian cell lines, sensor intensity correlated with aspartate levels measured by mass spectrometry and could resolve temporal changes in intracellular aspartate from genetic, pharmacological, and nutritional manipulations. These data demonstrate the utility of jAspSnFR3 and highlight the opportunities it provides for temporally resolved and high throughput applications of variables that affect aspartate levels.\nStructure-guided engineering of biased-agonism in the human niacin receptor via single amino acid substitution\nAuthors: Yadav, M.; Sarma, P.; Ganguly, M.; Mishra, S.; Maharana, J.; Zaidi, N.; Dalal, A.; Singh, V.; Saha, S.; Mahajan, G.; Sharma, S.; Chami, M.; Banerjee, R.; Shukla, A. K.\nScore: 15.3, Published: 2023-07-03 DOI: 10.1101/2023.07.03.547505\nThe Hydroxycarboxylic acid receptor 2 (HCA2), also known as the niacin receptor or GPR109A, is a prototypical G protein-coupled receptor that plays a central role in the inhibition of lipolytic and atherogenic activities in our body. Interestingly, GPR109A activation also results in vasodilation that is linked to the side-effect of flushing associated with dyslipidemia drugs such as niacin. This receptor continues to be a key target for developing novel pharmacophores and lead compounds as potential therapeutics in dyslipidemia with minimized flushing response, however, the lack of structural insights into agonist-binding and receptor activation has limited the efforts. Here, we present five different cryo-EM structures of the GPR109A-G-protein complexes with the receptor bound to dyslipidemia drugs, niacin or acipimox, non-flushing agonists, MK6892 or GSK256073, and recently approved psoriasis drug, monomethyl fumarate (MMF). These structures allow us to visualize the binding mechanism of agonists with a conserved molecular interaction network, and elucidate the previously lacking molecular basis of receptor activation and transducer-coupling. Importantly, cellular pharmacology experiments, guided by the structural framework determined here, elucidate pathway-selective biased signaling elicited by the non-flushing agonists. Finally, taking lead from the structural insights, we successfully engineered receptor mutants via single amino acid substitutions that either fail to elicit agonist-induced transducer-coupling or exhibits G-protein signaling bias. Taken together, our study provides previously lacking structural framework to understand the agonist-binding and activation of GPR109A, and opens up the possibilities of structure-guided novel drug discovery targeting this therapeutically important receptor.\nType III-B CRISPR-Cas signaling-based cascade of proteolytic cleavages\nAuthors: Steens, J.; Bravo, J. P.; Salazar, C. R. P.; Yildiz, C.; Amieiro, A. M.; Kostlbacher, S.; Prinsen, S. H. P.; Patinios, C.; Bardis, A.; Barendrecht, A.; Scheltema, R. A.; Ettema, T. J. G.; van der Oost, J.; Taylor, D. W.; Staals, R.\nScore: 10.2, Published: 2023-06-23 DOI: 10.1101/2023.06.23.546230\nType III CRISPR-Cas systems provide a sequence-specific adaptive immune response that protects prokaryotic hosts against viruses and other foreign genetic invaders. These crRNA-guided Cas effector complexes bind and cleave complementary RNA targets. Specific target binding stimulates the Cas10 subunit to generate cyclic oligoadenylate (cOA) signaling molecules, that in turn allosterically activate proteins carrying cognate sensory domains: CARF or SAVED. Here, we characterize an elaborate set of genes associated with the type III-B CRISPR-Cas system from Haliangium ochraceum, which includes a signal transduction module of a CBASS defense system with two caspase-like proteases, SAVED-CHAT and PCaspase (Prokaryotic Caspase). We show that binding of a 3-nucleotide cOA (cA3) to the SAVED domain of SAVED-CHAT induces its oligomerization into long filaments that activate the proteolytic activity of the CHAT domain. Surprisingly, we find that activated SAVED-CHAT specifically cleaves and activates the second protease, PCaspase. In turn, activated PCaspase cleaves a multitude of other proteins, including a putative sigma factor and a PCaspase-inhibitor. We expressed the type III-B system and its associated genes in E. coli and observed a strong abortive phenotype when offering a complementary target RNA, but only in the presence of both SAVED-CHAT and PCaspase. Together, our findings show an intriguing cascade of proteolytic activities (conceptually similar to eukaryotic caspases) in this bacterial immune system that reveals yet another strategy to effectively defend against mobile genetic elements.\nDecoding the comprehensive substrate-specificity and evidence of altered site-specific collagen prolyl-3-hydroxylation, lysyl-hydroxylation, and lysyl O-glycosylation in P4ha1 and P4ha2 deleted mutant mice\nAuthors: Sarohi, V.; Basak, T.\nScore: 4.1, Published: 2023-06-29 DOI: 10.1101/2023.06.28.546985\nCollagens, the most abundant proteins in mammals, play pivotal roles in the maintenance of tissue structure, functions, cell-to-cell communication, cellular migration, behavior, and growth. Collagens are highly complex in structure due to the dynamic post-translational modifications (PTMs) such as hydroxylations (on prolines and lysine residues) and O-glycosylation (on hydroxylysines) enzymatically catalyzed during biosynthesis. The most prevalent modification in fibrillar collagens is prolyl 4-hydroxylation catalyzed by collagen prolyl 4-hydroxylases (C-P4hs). Prolyl 4-hydroxylation on collagens plays a critical role in collagen biosynthesis, thermostability, and cell-collagen interactions. However, the site-specificity of prolyl 4-hydroxylase 1 (P4ha1) and P4ha2 is not comprehensively studied yet. Further, the effect of P4ha1 and P4ha2 on the plethora of other site-specific collagen PTMs is not known to date. In-depth mass-spectrometry data (PXD008802) analysis of mice skin collagen I extracted from wild-type and different deletion mutants of C-P4hs revealed that partial or full deletion of prolyl 4-hydroxylases (P4ha1 and P4ha2) significantly decreases collagen deposition in ECM hinting towards perturbed biosynthesis. A total of 421 site-specific PTMs on fibrillar collagen chains (Col1a1, Col1a2, and Col3a1) were identified. Further, novel 23 P4ha1 specific, 8 P4ha2 specific, and 18 C-P4hs promiscuous sites on fibrillar collagen chains were identified. Partial deletion of P4ha1 and full deletion of P4ha2 also resulted in altered levels of the site-specific prolyl-3-hydroxylation occupancy in collagen I. Surprisingly, an increased level of site-specific lysyl hydroxylation (Col1a1-K731, Col1a2-K183,315) was documented upon partial deletion of P4ha1 and full deletion of P4ha2. Our findings showcased that the activity of prolyl 4-hydroxylases is not limited to 4-hydroxylation of specific proline sites, but simultaneously can perturb the entire biosynthetic network by modulating prolyl 3-hydroxylation and lysyl hydroxylation occupancy levels in the fibrillar collagen chains in a site-specific manner.\nDecreased efficacy of a COVID-19 vaccine due to mutations present in early SARS-CoV-2 variants of concern\nAuthors: Weidenbacher, P. A.; Friedland, N.; Sanyal, M.; Morris, M. K.; Do, J.; Hanson, C.; Kim, P. S.\nScore: 4.1, Published: 2023-06-29 DOI: 10.1101/2023.06.27.546764\nWith the SARS-CoV-2 virus still circulating and evolving, there remains an outstanding question if variant-specific vaccines represent the optimal path forward, or if other strategies might be more efficacious towards providing broad protection against emerging variants. Here, we examine the efficacy of strain-specific variants of our previously reported, pan-sarbecovirus vaccine candidate, DCFHP-alum, a ferritin nanoparticle functionalized with an engineered form of the SARS-CoV-2 spike protein. In non-human primates, DCFHP-alum elicits neutralizing antibodies against all known VOCs that have emerged to date and SARS-CoV-1. During development of the DCFHP antigen, we investigated the incorporation of strain-specific mutations from the major VOCs that had emerged to date: D614G, Epsilon, Alpha, Beta, and Gamma. Here, we report the biochemical and immunological characterizations that led us to choose the ancestral Wuhan-1 sequence as the basis for the final DCFHP antigen design. Specifically, we show by size exclusion chromatography and differential scanning fluorimetry that mutations in the VOCs adversely alter the antigens structure and stability. More importantly, we determined that DCFHP without strain-specific mutations elicits the most robust, cross-reactive response in both pseudovirus and live virus neutralization assays. Our data suggest potential limitations to the variant-chasing approach in the development of protein nanoparticle vaccines, but also have implications for other approaches including mRNA-based vaccines.\nStructural characterization of the Sel1-like repeat protein LceB from Legionella pneumophila\nAuthors: Penner, T. V.; Lorente Cobo, N.; Patel, D. T.; Patel, D. H.; Savchenko, A.; Brassinga, A. K. C.; Prehna, G.\nScore: 3.9, Published: 2023-07-03 DOI: 10.1101/2023.07.03.547437\nLegionella are freshwater Gram-negative bacteria that in their normal environment infect protozoa. However, this adaptation also allows Legionella to infect human alveolar macrophages and cause pneumonia. Central to Legionella pathogenesis are more than 330 secreted effectors, of which there are 9 core effectors that are conserved in all pathogenic species. Despite their importance, the biochemical function of several core effectors remains unclear. To address this, we have taken a structural approach to characterize the core effector of unknown function LceB, or Lpg1356, from Legionella pneumophila. Here we solve an X-ray crystal structure of LceB using an AlphaFold model for molecular replacement. The experimental structure shows that LceB adopts a Sel1-like repeat fold as predicted. However, the crystal structure captured multiple conformations of LceB all of which differed from the AlphaFold model. Comparison of the predicted model and the experimental models suggests that LceB is highly flexible in solution. Additionally, molecular analysis of LceB using its close structural homologues reveals sequence and structural motifs of known biochemical function. Specifically, LceB harbors a repeated KAAEQG motif that both stabilizes the Sel1-like repeat fold and is known to participate in protein-protein interactions with eukaryotic host proteins. We also observe that LceB forms several higher-order oligomers in solution. Overall, our results have revealed that LceB has conformational flexibility, self-associates, and contains a molecular surface for binding a target host-cell protein. Additionally, our data provides structural insights into the Sel1-like repeat family of proteins that remain poorly studied.\nBimodular Architecture of Bacterial Effector SAP05 Drives Ubiquitin-Independent Targeted Protein Degradation\nAuthors: Liu, Q.; Maqbool, A.; Mirkin, F. G.; Singh, Y.; Stevenson, C. E.; Lawson, D. M.; Kamoun, S.; Huang, W.; Hogenhout, S. A.\nScore: 37.0, Published: 2023-06-19 DOI: 10.1101/2023.06.19.545293\nIn eukaryotes, targeted protein degradation (TPD) typically depends on a series of interactions among ubiquitin ligases that transfer ubiquitin molecules to substrates leading to degradation by the 26S proteasome. We previously discovered that the bacterial effector protein SAP05 mediates ubiquitin-independent TPD. SAP05 forms a ternary complex via interactions with the von Willebrand Factor Type A (vWA) domain of the proteasomal ubiquitin receptor Rpn10 and the Zinc-finger (ZnF) domains of the SQUAMOSA-PROMOTER BINDING PROTEIN-LIKE (SPL) and GATA BINDING FACTOR (GATA) transcription factors (TFs). This leads to direct TPD of the TFs by the 26S proteasome. Here, we report the crystal structures of the SAP05-vWA complex at 2.17 [A] resolution and of the SAP05-ZnF(SPL5) complex at 2.20 [A] resolution. Structural analyses revealed that SAP05 displays a unique bimodular architecture with two distinct non-overlapping surfaces, a loop surface with three protruding loops that form electrostatic interactions with ZnF, and a sheet surface featuring two {beta}-sheets, loops and -helices that establish polar interactions with vWA. SAP05 binding to ZnF TFs involves single amino acids responsible for multiple contacts, while SAP05 binding to vWA is more stable due to the necessity of multiple mutations to break the interaction. In addition, positioning of the SAP05 complex on the 26S proteasome points to a mechanism of protein degradation. Collectively, our findings demonstrate how a small bacterial bimodular protein can bypass the canonical UPS cellular proteolysis pathway, enabling ubiquitin-independent TPD in eukaryotic cells. This knowledge holds significant potential for the creation of novel TPD technologies.\nHow Myosin VI Traps its Off-State, is Activated and Dimerizes\nAuthors: Canon, L.; Kikuti, C.; Planelles-Herrero, V. J.; Lin, T.; Mayeux, F.; Sirkia, H.; Lee, Y. i.; Heidsieck, L.; Velikovsky, L.; David, A.; Liu, X.; Moussaoui, D.; Forest, E.; Hook, P.; Petersen, K. J.; Di Cicco, A.; Sires-Campos, J.; Levy, D.; Delevoye, C.; Sweeney, H. L.; Houdusse, A.\nScore: 3.7, Published: 2023-06-30 DOI: 10.1101/2023.06.30.547185\nMyosin VI (Myo6) is the only minus-end directed nanomotor on actin, allowing it to uniquely contribute to numerous cellular functions. As for other nanomotors, proper functioning of Myo6 relies on precise spatio-temporal control of motor activity via a poorly defined off-state and interactions with partners. Our structural, functional, and cellular studies reveal key features of myosin regulation and indicate that not all partners can activate Myo6. TOM1 and Dab2 cannot bind the off-state while, GIPC1 binds Myo6, releases its auto-inhibition and triggers proximal dimerization. Myo6 partners thus differentially recruit Myo6. We solved a crystal structure of the proximal dimerization domain, and show that its disruption compromises endocytosis in HeLa cells, emphasizing the importance of Myo6 dimerization. Finally, we show that the L926Q deafness mutation disrupts Myo6 auto-inhibition and indirectly impairs proximal dimerization. Our study thus demonstrates the importance of partners in the control of Myo6 auto-inhibition, localization, and activation.\nTargeting the conserved active site of splicing machines with specific and selective small molecule modulators\nAuthors: Silvestri, I.; Manigrasso, J.; Andreani, A.; Brindani, N.; De Vivo, M.; Marcia, M.\nScore: 11.6, Published: 2023-06-22 DOI: 10.1101/2023.06.21.545906\nThe self-splicing group II introns are bacterial and organellar ancestors of the nuclear spliceosome and retro-transposable elements of pharmacological and biotechnological importance. Integrating enzymatic, crystallographic, and simulation studies, we demonstrate how these introns recognize small molecules through their conserved active site. These RNA-binding small molecules selectively inhibit the two steps of splicing by adopting distinctive poses at different stages of catalysis, and by preventing crucial active site conformational changes that are essential for splicing progression. Our data exemplify the enormous power of RNA binders to mechanistically probe vital cellular pathways. Most importantly, by proving that the evolutionarily-conserved RNA core of splicing machines can recognize small molecules specifically, our work puts solid bases for the rational design of splicing modulators not only against bacterial and organellar introns, but also against the human spliceosome, which is a validated drug target for the treatment of congenital diseases and cancers.\n",
  "wordCount" : "2606",
  "inLanguage": "en",
  "datePublished": "2023-07-05T10:42:43Z",
  "dateModified": "2023-07-05T10:42:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta"><span>updated on July 5, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.25.546465">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.25.546465" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.25.546465">
        <p class="paperTitle">Intrinsically Disordered Regions Promote Protein Refoldability and Facilitate Retrieval from Biomolecular Condensates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.25.546465" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.25.546465" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: To, P.; Bhagwat, A. M.; Tarbox, H. E.; Ecer, A.; Wendorff, H.; Jamieson, Z.; Trcek, T.; Fried, S. D.</p>
        <p class="info">Score: 129.0, Published: 2023-06-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.25.546465' target='https://doi.org/10.1101/2023.06.25.546465'> 10.1101/2023.06.25.546465</a></p>
        <p class="abstract">Many eukaryotic proteins contain intrinsically disordered regions (IDRs) that intersperse globular folded domains, in contrast with bacterial proteins which are typically highly globular1, 2. Recent years have seen great progress in identifying biological functions associated with these elusive protein sequence: in specific cases, they mediate liquid liquid phase separation3, perform molecular recognition4, or act as sensors to changes in the environment5. Nevertheless, only a small number of IDRs have annotated functions6 despite their presence in 64% of yeast proteins,7 stimulating some to question what  general purpose they may serve8, 9. Here, by interrogating the refoldability of two fungal proteomes (Saccharomyces cerevisiae and Neurosporra crassa), we show that IDRs render their host proteins more refoldable from the denatured state, allowing them to cohere more closely to Anfinsens thermodynamic hypothesis10, 11. The data provide an exceptionally clear picture of which biophysical and topological characteristics enable refoldability. Moreover, we find that almost all yeast proteins that partition into stress granules during heat shock are refoldable, a finding that holds for other condensates such as P-bodies and the nucleolus. Finally, we find that the Hsp104 unfoldase12 is the principal actor in mediating disassembly of heat stress granules and that the efficiency with which condensed proteins are returned to the soluble phase is also well explained by refoldability. Hence, these studies establish spontaneous refoldability as an adaptive trait that endows proteins with the capacity to reform their native soluble structures following their extraction from condensates. Altogether, our results provide an intuitive model for the function of IDRs in many multidomain proteins and clarifies their relationship to the phenomenon of biomolecular condensation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546775">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546775" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546775">
        <p class="paperTitle">An engineered biosensor enables dynamic aspartate measurements in living cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546775" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546775" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Davidsen, K.; Marvin, J. S.; Aggarwal, A.; Brown, T. A.; Sullivan, L. B.</p>
        <p class="info">Score: 18.6, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546775' target='https://doi.org/10.1101/2023.06.27.546775'> 10.1101/2023.06.27.546775</a></p>
        <p class="abstract">Intracellular levels of the amino acid aspartate are responsive to changes in metabolism in mammalian cells and can correspondingly alter cell function, highlighting the need for robust tools to measure aspartate abundance. However, comprehensive understanding of aspartate metabolism has been limited by the throughput, cost, and static nature of the mass spectrometry based measurements that are typically employed to measure aspartate levels. To address these issues, we have developed a GFP-based sensor of aspartate (jAspSnFR3), where the fluorescence intensity corresponds to aspartate concentration. As a purified protein, the sensor has a 20-fold increase in fluorescence upon aspartate saturation, with dose dependent fluorescence changes covering a physiologically relevant aspartate concentration range and no significant off target binding. Expressed in mammalian cell lines, sensor intensity correlated with aspartate levels measured by mass spectrometry and could resolve temporal changes in intracellular aspartate from genetic, pharmacological, and nutritional manipulations. These data demonstrate the utility of jAspSnFR3 and highlight the opportunities it provides for temporally resolved and high throughput applications of variables that affect aspartate levels.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547505">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547505" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547505">
        <p class="paperTitle">Structure-guided engineering of biased-agonism in the human niacin receptor via single amino acid substitution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547505" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547505" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yadav, M.; Sarma, P.; Ganguly, M.; Mishra, S.; Maharana, J.; Zaidi, N.; Dalal, A.; Singh, V.; Saha, S.; Mahajan, G.; Sharma, S.; Chami, M.; Banerjee, R.; Shukla, A. K.</p>
        <p class="info">Score: 15.3, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547505' target='https://doi.org/10.1101/2023.07.03.547505'> 10.1101/2023.07.03.547505</a></p>
        <p class="abstract">The Hydroxycarboxylic acid receptor 2 (HCA2), also known as the niacin receptor or GPR109A, is a prototypical G protein-coupled receptor that plays a central role in the inhibition of lipolytic and atherogenic activities in our body. Interestingly, GPR109A activation also results in vasodilation that is linked to the side-effect of flushing associated with dyslipidemia drugs such as niacin. This receptor continues to be a key target for developing novel pharmacophores and lead compounds as potential therapeutics in dyslipidemia with minimized flushing response, however, the lack of structural insights into agonist-binding and receptor activation has limited the efforts. Here, we present five different cryo-EM structures of the GPR109A-G-protein complexes with the receptor bound to dyslipidemia drugs, niacin or acipimox, non-flushing agonists, MK6892 or GSK256073, and recently approved psoriasis drug, monomethyl fumarate (MMF). These structures allow us to visualize the binding mechanism of agonists with a conserved molecular interaction network, and elucidate the previously lacking molecular basis of receptor activation and transducer-coupling. Importantly, cellular pharmacology experiments, guided by the structural framework determined here, elucidate pathway-selective biased signaling elicited by the non-flushing agonists. Finally, taking lead from the structural insights, we successfully engineered receptor mutants via single amino acid substitutions that either fail to elicit agonist-induced transducer-coupling or exhibits G-protein signaling bias. Taken together, our study provides previously lacking structural framework to understand the agonist-binding and activation of GPR109A, and opens up the possibilities of structure-guided novel drug discovery targeting this therapeutically important receptor.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546230">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546230" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546230">
        <p class="paperTitle">Type III-B CRISPR-Cas signaling-based cascade of proteolytic cleavages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546230" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546230" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Steens, J.; Bravo, J. P.; Salazar, C. R. P.; Yildiz, C.; Amieiro, A. M.; Kostlbacher, S.; Prinsen, S. H. P.; Patinios, C.; Bardis, A.; Barendrecht, A.; Scheltema, R. A.; Ettema, T. J. G.; van der Oost, J.; Taylor, D. W.; Staals, R.</p>
        <p class="info">Score: 10.2, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546230' target='https://doi.org/10.1101/2023.06.23.546230'> 10.1101/2023.06.23.546230</a></p>
        <p class="abstract">Type III CRISPR-Cas systems provide a sequence-specific adaptive immune response that protects prokaryotic hosts against viruses and other foreign genetic invaders. These crRNA-guided Cas effector complexes bind and cleave complementary RNA targets. Specific target binding stimulates the Cas10 subunit to generate cyclic oligoadenylate (cOA) signaling molecules, that in turn allosterically activate proteins carrying cognate sensory domains: CARF or SAVED. Here, we characterize an elaborate set of genes associated with the type III-B CRISPR-Cas system from Haliangium ochraceum, which includes a signal transduction module of a CBASS defense system with two caspase-like proteases, SAVED-CHAT and PCaspase (Prokaryotic Caspase). We show that binding of a 3-nucleotide cOA (cA3) to the SAVED domain of SAVED-CHAT induces its oligomerization into long filaments that activate the proteolytic activity of the CHAT domain. Surprisingly, we find that activated SAVED-CHAT specifically cleaves and activates the second protease, PCaspase. In turn, activated PCaspase cleaves a multitude of other proteins, including a putative sigma factor and a PCaspase-inhibitor. We expressed the type III-B system and its associated genes in E. coli and observed a strong abortive phenotype when offering a complementary target RNA, but only in the presence of both SAVED-CHAT and PCaspase. Together, our findings show an intriguing cascade of proteolytic activities (conceptually similar to eukaryotic caspases) in this bacterial immune system that reveals yet another strategy to effectively defend against mobile genetic elements.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546985">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546985" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546985">
        <p class="paperTitle">Decoding the comprehensive substrate-specificity and evidence of altered site-specific collagen prolyl-3-hydroxylation, lysyl-hydroxylation, and lysyl O-glycosylation in P4ha1 and P4ha2 deleted mutant mice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546985" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546985" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sarohi, V.; Basak, T.</p>
        <p class="info">Score: 4.1, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546985' target='https://doi.org/10.1101/2023.06.28.546985'> 10.1101/2023.06.28.546985</a></p>
        <p class="abstract">Collagens, the most abundant proteins in mammals, play pivotal roles in the maintenance of tissue structure, functions, cell-to-cell communication, cellular migration, behavior, and growth. Collagens are highly complex in structure due to the dynamic post-translational modifications (PTMs) such as hydroxylations (on prolines and lysine residues) and O-glycosylation (on hydroxylysines) enzymatically catalyzed during biosynthesis. The most prevalent modification in fibrillar collagens is prolyl 4-hydroxylation catalyzed by collagen prolyl 4-hydroxylases (C-P4hs). Prolyl 4-hydroxylation on collagens plays a critical role in collagen biosynthesis, thermostability, and cell-collagen interactions. However, the site-specificity of prolyl 4-hydroxylase 1 (P4ha1) and P4ha2 is not comprehensively studied yet. Further, the effect of P4ha1 and P4ha2 on the plethora of other site-specific collagen PTMs is not known to date. In-depth mass-spectrometry data (PXD008802) analysis of mice skin collagen I extracted from wild-type and different deletion mutants of C-P4hs revealed that partial or full deletion of prolyl 4-hydroxylases (P4ha1 and P4ha2) significantly decreases collagen deposition in ECM hinting towards perturbed biosynthesis. A total of 421 site-specific PTMs on fibrillar collagen chains (Col1a1, Col1a2, and Col3a1) were identified. Further, novel 23 P4ha1 specific, 8 P4ha2 specific, and 18 C-P4hs promiscuous sites on fibrillar collagen chains were identified. Partial deletion of P4ha1 and full deletion of P4ha2 also resulted in altered levels of the site-specific prolyl-3-hydroxylation occupancy in collagen I. Surprisingly, an increased level of site-specific lysyl hydroxylation (Col1a1-K731, Col1a2-K183,315) was documented upon partial deletion of P4ha1 and full deletion of P4ha2. Our findings showcased that the activity of prolyl 4-hydroxylases is not limited to 4-hydroxylation of specific proline sites, but simultaneously can perturb the entire biosynthetic network by modulating prolyl 3-hydroxylation and lysyl hydroxylation occupancy levels in the fibrillar collagen chains in a site-specific manner.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546764">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546764" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546764">
        <p class="paperTitle">Decreased efficacy of a COVID-19 vaccine due to mutations present in early SARS-CoV-2 variants of concern</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546764" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546764" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Weidenbacher, P. A.; Friedland, N.; Sanyal, M.; Morris, M. K.; Do, J.; Hanson, C.; Kim, P. S.</p>
        <p class="info">Score: 4.1, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546764' target='https://doi.org/10.1101/2023.06.27.546764'> 10.1101/2023.06.27.546764</a></p>
        <p class="abstract">With the SARS-CoV-2 virus still circulating and evolving, there remains an outstanding question if variant-specific vaccines represent the optimal path forward, or if other strategies might be more efficacious towards providing broad protection against emerging variants. Here, we examine the efficacy of strain-specific variants of our previously reported, pan-sarbecovirus vaccine candidate, DCFHP-alum, a ferritin nanoparticle functionalized with an engineered form of the SARS-CoV-2 spike protein. In non-human primates, DCFHP-alum elicits neutralizing antibodies against all known VOCs that have emerged to date and SARS-CoV-1. During development of the DCFHP antigen, we investigated the incorporation of strain-specific mutations from the major VOCs that had emerged to date: D614G, Epsilon, Alpha, Beta, and Gamma. Here, we report the biochemical and immunological characterizations that led us to choose the ancestral Wuhan-1 sequence as the basis for the final DCFHP antigen design. Specifically, we show by size exclusion chromatography and differential scanning fluorimetry that mutations in the VOCs adversely alter the antigens structure and stability. More importantly, we determined that DCFHP without strain-specific mutations elicits the most robust, cross-reactive response in both pseudovirus and live virus neutralization assays. Our data suggest potential limitations to the variant-chasing approach in the development of protein nanoparticle vaccines, but also have implications for other approaches including mRNA-based vaccines.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547437">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547437" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547437">
        <p class="paperTitle">Structural characterization of the Sel1-like repeat protein LceB from Legionella pneumophila</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547437" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547437" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Penner, T. V.; Lorente Cobo, N.; Patel, D. T.; Patel, D. H.; Savchenko, A.; Brassinga, A. K. C.; Prehna, G.</p>
        <p class="info">Score: 3.9, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547437' target='https://doi.org/10.1101/2023.07.03.547437'> 10.1101/2023.07.03.547437</a></p>
        <p class="abstract">Legionella are freshwater Gram-negative bacteria that in their normal environment infect protozoa. However, this adaptation also allows Legionella to infect human alveolar macrophages and cause pneumonia. Central to Legionella pathogenesis are more than 330 secreted effectors, of which there are 9 core effectors that are conserved in all pathogenic species. Despite their importance, the biochemical function of several core effectors remains unclear. To address this, we have taken a structural approach to characterize the core effector of unknown function LceB, or Lpg1356, from Legionella pneumophila. Here we solve an X-ray crystal structure of LceB using an AlphaFold model for molecular replacement. The experimental structure shows that LceB adopts a Sel1-like repeat fold as predicted. However, the crystal structure captured multiple conformations of LceB all of which differed from the AlphaFold model. Comparison of the predicted model and the experimental models suggests that LceB is highly flexible in solution. Additionally, molecular analysis of LceB using its close structural homologues reveals sequence and structural motifs of known biochemical function. Specifically, LceB harbors a repeated KAAEQG motif that both stabilizes the Sel1-like repeat fold and is known to participate in protein-protein interactions with eukaryotic host proteins. We also observe that LceB forms several higher-order oligomers in solution. Overall, our results have revealed that LceB has conformational flexibility, self-associates, and contains a molecular surface for binding a target host-cell protein. Additionally, our data provides structural insights into the Sel1-like repeat family of proteins that remain poorly studied.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.545293">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.545293" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.545293">
        <p class="paperTitle">Bimodular Architecture of Bacterial Effector SAP05 Drives Ubiquitin-Independent Targeted Protein Degradation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.545293" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.545293" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, Q.; Maqbool, A.; Mirkin, F. G.; Singh, Y.; Stevenson, C. E.; Lawson, D. M.; Kamoun, S.; Huang, W.; Hogenhout, S. A.</p>
        <p class="info">Score: 37.0, Published: 2023-06-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.545293' target='https://doi.org/10.1101/2023.06.19.545293'> 10.1101/2023.06.19.545293</a></p>
        <p class="abstract">In eukaryotes, targeted protein degradation (TPD) typically depends on a series of interactions among ubiquitin ligases that transfer ubiquitin molecules to substrates leading to degradation by the 26S proteasome. We previously discovered that the bacterial effector protein SAP05 mediates ubiquitin-independent TPD. SAP05 forms a ternary complex via interactions with the von Willebrand Factor Type A (vWA) domain of the proteasomal ubiquitin receptor Rpn10 and the Zinc-finger (ZnF) domains of the SQUAMOSA-PROMOTER BINDING PROTEIN-LIKE (SPL) and GATA BINDING FACTOR (GATA) transcription factors (TFs). This leads to direct TPD of the TFs by the 26S proteasome. Here, we report the crystal structures of the SAP05-vWA complex at 2.17 [A] resolution and of the SAP05-ZnF(SPL5) complex at 2.20 [A] resolution. Structural analyses revealed that SAP05 displays a unique bimodular architecture with two distinct non-overlapping surfaces, a  loop surface with three protruding loops that form electrostatic interactions with ZnF, and a  sheet surface featuring two {beta}-sheets, loops and -helices that establish polar interactions with vWA. SAP05 binding to ZnF TFs involves single amino acids responsible for multiple contacts, while SAP05 binding to vWA is more stable due to the necessity of multiple mutations to break the interaction. In addition, positioning of the SAP05 complex on the 26S proteasome points to a mechanism of protein degradation. Collectively, our findings demonstrate how a small bacterial bimodular protein can bypass the canonical UPS cellular proteolysis pathway, enabling ubiquitin-independent TPD in eukaryotic cells. This knowledge holds significant potential for the creation of novel TPD technologies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547185">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547185" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547185">
        <p class="paperTitle">How Myosin VI Traps its Off-State, is Activated and Dimerizes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547185" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547185" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Canon, L.; Kikuti, C.; Planelles-Herrero, V. J.; Lin, T.; Mayeux, F.; Sirkia, H.; Lee, Y. i.; Heidsieck, L.; Velikovsky, L.; David, A.; Liu, X.; Moussaoui, D.; Forest, E.; Hook, P.; Petersen, K. J.; Di Cicco, A.; Sires-Campos, J.; Levy, D.; Delevoye, C.; Sweeney, H. L.; Houdusse, A.</p>
        <p class="info">Score: 3.7, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547185' target='https://doi.org/10.1101/2023.06.30.547185'> 10.1101/2023.06.30.547185</a></p>
        <p class="abstract">Myosin VI (Myo6) is the only minus-end directed nanomotor on actin, allowing it to uniquely contribute to numerous cellular functions. As for other nanomotors, proper functioning of Myo6 relies on precise spatio-temporal control of motor activity via a poorly defined off-state and interactions with partners. Our structural, functional, and cellular studies reveal key features of myosin regulation and indicate that not all partners can activate Myo6. TOM1 and Dab2 cannot bind the off-state while, GIPC1 binds Myo6, releases its auto-inhibition and triggers proximal dimerization. Myo6 partners thus differentially recruit Myo6. We solved a crystal structure of the proximal dimerization domain, and show that its disruption compromises endocytosis in HeLa cells, emphasizing the importance of Myo6 dimerization. Finally, we show that the L926Q deafness mutation disrupts Myo6 auto-inhibition and indirectly impairs proximal dimerization. Our study thus demonstrates the importance of partners in the control of Myo6 auto-inhibition, localization, and activation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545906">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545906" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545906">
        <p class="paperTitle">Targeting the conserved active site of splicing machines with specific and selective small molecule modulators</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545906" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545906" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Silvestri, I.; Manigrasso, J.; Andreani, A.; Brindani, N.; De Vivo, M.; Marcia, M.</p>
        <p class="info">Score: 11.6, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545906' target='https://doi.org/10.1101/2023.06.21.545906'> 10.1101/2023.06.21.545906</a></p>
        <p class="abstract">The self-splicing group II introns are bacterial and organellar ancestors of the nuclear spliceosome and retro-transposable elements of pharmacological and biotechnological importance. Integrating enzymatic, crystallographic, and simulation studies, we demonstrate how these introns recognize small molecules through their conserved active site. These RNA-binding small molecules selectively inhibit the two steps of splicing by adopting distinctive poses at different stages of catalysis, and by preventing crucial active site conformational changes that are essential for splicing progression. Our data exemplify the enormous power of RNA binders to mechanistically probe vital cellular pathways. Most importantly, by proving that the evolutionarily-conserved RNA core of splicing machines can recognize small molecules specifically, our work puts solid bases for the rational design of splicing modulators not only against bacterial and organellar introns, but also against the human spliceosome, which is a validated drug target for the treatment of congenital diseases and cancers.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
