<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Atomic context-conditioned protein sequence design using LigandMPNN
Authors: Dauparas, J.; Lee, G. R.; Pecoraro, R.; An, L.; Anishchenko, I.; Glasscock, C.; Baker, D.
Score: 38.8, Published: 2023-12-23 DOI: 10.1101/2023.12.22.573103
Protein sequence design in the context of small molecules, nucleotides, and metals is critical to enzyme and small molecule binder and sensor design, but current state-of-the-art deep learning-based sequence design methods are unable to model non-protein atoms and molecules. Here, we describe a deep learning-based protein sequence design method called LigandMPNN that explicitly models all non-protein components of biomolecular systems.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Atomic context-conditioned protein sequence design using LigandMPNN
Authors: Dauparas, J.; Lee, G. R.; Pecoraro, R.; An, L.; Anishchenko, I.; Glasscock, C.; Baker, D.
Score: 38.8, Published: 2023-12-23 DOI: 10.1101/2023.12.22.573103
Protein sequence design in the context of small molecules, nucleotides, and metals is critical to enzyme and small molecule binder and sensor design, but current state-of-the-art deep learning-based sequence design methods are unable to model non-protein atoms and molecules. Here, we describe a deep learning-based protein sequence design method called LigandMPNN that explicitly models all non-protein components of biomolecular systems." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-31T10:38:19+00:00" />
<meta property="article:modified_time" content="2023-12-31T10:38:19+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Atomic context-conditioned protein sequence design using LigandMPNN
Authors: Dauparas, J.; Lee, G. R.; Pecoraro, R.; An, L.; Anishchenko, I.; Glasscock, C.; Baker, D.
Score: 38.8, Published: 2023-12-23 DOI: 10.1101/2023.12.22.573103
Protein sequence design in the context of small molecules, nucleotides, and metals is critical to enzyme and small molecule binder and sensor design, but current state-of-the-art deep learning-based sequence design methods are unable to model non-protein atoms and molecules. Here, we describe a deep learning-based protein sequence design method called LigandMPNN that explicitly models all non-protein components of biomolecular systems."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Atomic context-conditioned protein sequence design using LigandMPNN\nAuthors: Dauparas, J.; Lee, G. R.; Pecoraro, R.; An, L.; Anishchenko, I.; Glasscock, C.; Baker, D.\nScore: 38.8, Published: 2023-12-23 DOI: 10.1101/2023.12.22.573103\nProtein sequence design in the context of small molecules, nucleotides, and metals is critical to enzyme and small molecule binder and sensor design, but current state-of-the-art deep learning-based sequence design methods are unable to model non-protein atoms and molecules. Here, we describe a deep learning-based protein sequence design method called LigandMPNN that explicitly models all non-protein components of biomolecular systems.",
  "keywords": [
    
  ],
  "articleBody": " Atomic context-conditioned protein sequence design using LigandMPNN\nAuthors: Dauparas, J.; Lee, G. R.; Pecoraro, R.; An, L.; Anishchenko, I.; Glasscock, C.; Baker, D.\nScore: 38.8, Published: 2023-12-23 DOI: 10.1101/2023.12.22.573103\nProtein sequence design in the context of small molecules, nucleotides, and metals is critical to enzyme and small molecule binder and sensor design, but current state-of-the-art deep learning-based sequence design methods are unable to model non-protein atoms and molecules. Here, we describe a deep learning-based protein sequence design method called LigandMPNN that explicitly models all non-protein components of biomolecular systems. LigandMPNN significantly outperforms Rosetta and ProteinMPNN on native backbone sequence recovery for residues interacting with small molecules (63.3% vs. 50.4% \u0026 50.5%), nucleotides (50.5% vs. 35.2% \u0026 34.0%), and metals (77.5% vs. 36.0% \u0026 40.6%). LigandMPNN generates not only sequences but also sidechain conformations to allow detailed evaluation of binding interactions. Experimental characterization demonstrates that LigandMPNN can generate small molecule and DNA-binding proteins with high affinity and specificity. One-sentence summaryWe present a deep learning-based protein sequence design method that allows explicit modeling of small molecule, nucleotide, metal, and other atomic contexts.\nMechanism of human PINK1 activation at the TOM complex in a reconstituted system\nAuthors: Raimi, O.; Ojha, H.; Ehses, K.; Dederer, V.; Lange, S.; Rivera, C. P.; Deegan, T.; Chen, Y.; Wightman, M.; Toth, R.; Labib, K.; Mathea, S.; Ranson, N.; Fernandez-Busnadiego, R. F.; Muqit, M.\nScore: 20.5, Published: 2023-12-23 DOI: 10.1101/2023.12.23.573181\nLoss of function mutations in PTEN-induced kinase 1 (PINK1) are a frequent cause of early-onset Parkinsons disease (PD). Stabilisation of PINK1 at the Translocase of Outer Membrane (TOM) complex of damaged mitochondria is a critical step for its activation. To date the mechanism of how PINK1 is activated in the TOM complex is unclear. Herein we report co-expression of human PINK1 and all seven TOM subunits in Saccharomyces cerevisiae is sufficient for PINK1 activation. We use this reconstitution system to systematically assess the role of each TOM subunit towards PINK1 activation. We unambiguously demonstrate that the TOM20 and TOM70 receptor subunits are required for optimal PINK1 activation and map their sites of interaction with PINK1 using AlphaFold structural modelling and mutagenesis. We also demonstrate an essential role of the pore-containing subunit TOM40 and its structurally associated subunits TOM7 and TOM22 for PINK1 activation. These molecular findings will aid in the development of small molecule activators of PINK1 as a therapeutic strategy for PD.\nStructure of the minor spliceosomal U11 snRNP\nAuthors: Zhao, J.; Peter, D.; Brandina, I.; Liu, X.; Galej, W. P.\nScore: 17.9, Published: 2023-12-22 DOI: 10.1101/2023.12.22.573053\nThe minor spliceosome catalyses the excision of U12-dependent introns from pre-mRNAs. These introns are rare, but their removal is critical for cell viability. We obtained a cryo-EM reconstruction of the 13-subunit U11 snRNP complex, revealing structures of U11 snRNA and five minor spliceosome-specific factors. U11 snRNP appears strikingly different from the equivalent major spliceosome U1 snRNP. SNRNP25 and SNRNP35 form a dimer, which specifically recognises U11 snRNA. PDCD7 forms extended helices, which bridge SNRNP25 and SNRNP48, located at the distal ends of the particle. SNRNP48 forms multiple interfaces with U11 snRNP and, together with ZMAT5, are positioned near the 5-end of the U11 snRNA and likely stabilise the binding of the incoming 5SS. Our structure provides mechanistic insights into U12-dependent intron recognition and the evolution of the splicing machinery.\nDe novo design of diverse small molecule binders and sensors using Shape Complementary Pseudocycles\nAuthors: An, L.; Said, M.; Tran, L.; Majumder, S.; Goreshnik, I.; Lee, G. R.; Juergens, D.; Dauparas, J.; Anishchenko, I.; Coventry, B.; Bera, A. K.; Kang, A.; Levine, P. M.; Alvarez, V.; Pillai, A.; Norn, C.; Feldman, D.; Zorine, D.; Hicks, D. R.; Li, X.; Sanchez, M. G.; Vafeados, D. K.; Salveson, P. J.; Vorobieva, A. A.; Baker, D.\nScore: 44.1, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572602\nA general method for designing proteins to bind and sense any small molecule of interest would be widely useful. Due to the small number of atoms to interact with, binding to small molecules with high affinity requires highly shape complementary pockets, and transducing binding events into signals is challenging. Here we describe an integrated deep learning and energy based approach for designing high shape complementarity binders to small molecules that are poised for downstream sensing applications. We employ deep learning generated psuedocycles with repeating structural units surrounding central pockets; depending on the geometry of the structural unit and repeat number, these pockets span wide ranges of sizes and shapes. For a small molecule target of interest, we extensively sample high shape complementarity pseudocycles to generate large numbers of customized potential binding pockets; the ligand binding poses and the interacting interfaces are then optimized for high affinity binding. We computationally design binders to four diverse molecules, including for the first time polar flexible molecules such as methotrexate and thyroxine, which are expressed at high levels and have nanomolar affinities straight out of the computer. Co-crystal structures are nearly identical to the design models. Taking advantage of the modular repeating structure of pseudocycles and central location of the binding pockets, we constructed low noise nanopore sensors and chemically induced dimerization systems by splitting the binders into domains which assemble into the original pseudocycle pocket upon target molecule addition. One Sentence SummaryWe use a pseuodocycle-based shape complementarity optimizing approach to design nanomolar binders to diverse ligands, including the flexible and polar methotrexate and thyroxine, that can be directly converted into ligand-gated nanopores and chemically induced dimerization systems.\nChemical probes to interrogate the extreme environment of mosquito larval guts\nAuthors: Guzman, L. E.; Wijetunge, A. N.; Riske, B. F.; Massani, B. B.; Riehle, M. A.; Jewett, J. C.\nScore: 12.4, Published: 2023-12-27 DOI: 10.1101/2023.12.27.573438\nMosquito control methods are vital for the spread of life-threatening illnesses such as dengue fever, malaria, and yellow fever. Vector control technologies must be selective to minimize deleterious effects to our ecosystem. Successful methods that control mosquito larva populations utilize the uniquely high alkaline nature of the midgut. Here, we present novel protected triazabutadienes (pTBD) which are deprotected under basic conditions of the larval midgut, releasing an aryl diazonium ion (ADI) that results in protein modification. The probes contain a bioorthogonal terminal alkyne handle, enabling a selective Cu-click reaction with an azido-fluorophore for quantification by SDS PAGE and visualization using fluorescence microscopy. A control TBD, unable to release an ADI, did not label the midgut. We envision our chemical probes will aid in the development of new selective mosquito control methods thus preventing the spread of mosquito-borne illnesses with minimal impact on other organisms in the ecosystem.\nStructural insights into rice KAI2 receptor provide functional implications for perception and signal transduction\nAuthors: Guercio, A. M.; Gilio, A. K.; Pawlak, J.; Shabek, N.\nScore: 8.1, Published: 2023-12-29 DOI: 10.1101/2023.12.28.573577\nKAI2 proteins, classified as plant /{beta} hydrolase receptors, are capable of perceiving smoke-derived butenolide signals and endogenous, yet unidentified KAI2-ligands (KLs). While the number of functional KAI2 receptors varies among land plant species, rice possesses only one KAI2 gene. Rice, a significant crop and representative of grasses, relies on KAI2-mediated Arbuscular mycorrhiza (AM) symbioses to flourish in traditionally arid and nutrient-poor environments. This study presents the first crystal structure of an active rice KAI2 hydrolase receptor. Our structural and biochemical analyses uncover grass-unique pocket residues influencing ligand sensitivity and hydrolytic activity. Through structure-guided analysis, we identify a specific residue whose mutation enables the increase or decrease of ligand perception, catalytic activity, and signal transduction. Furthermore, we investigate rice KAI2-mediated signaling by examining its ability to form a complex with its binding partner, the F-box protein DWARF3 (D3) ubiquitin ligase, demonstrating the significant role of hydrophobic interactions in the KAI2-D3 interface. This study provides new insights into the diverse and pivotal roles of the KAI2 signaling pathway in the plant kingdom, particularly in grasses.\nStructural analysis of the dynamic ribosome-translocon complex\nAuthors: Lewis, A. J. O.; Hegde, R. S.\nScore: 21.5, Published: 2023-12-22 DOI: 10.1101/2023.12.22.572959\nThe protein translocon at the endoplasmic reticulum comprises the Sec61 translocation channel and numerous accessory factors that collectively facilitate the biogenesis of secretory and membrane proteins. Here, we leveraged recent advances in cryo-EM and structure prediction to derive insights into several novel configurations of the ribosome-translocon complex. We show how a transmembrane domain (TMD) in a looped configuration passes through the Sec61 lateral gate during membrane insertion; how a nascent chain can bind and constrain the conformation of ribosomal protein uL22; and how the translocon-associated protein (TRAP) complex can adjust its position during different stages of protein biogenesis. Most unexpectedly, we find that a large proportion of translocon complexes contains RAMP4 intercalated into Sec61s lateral gate, widening Sec61s central pore and contributing to its hydrophilic interior. These structures lead to mechanistic hypotheses for translocon function and highlight a remarkably plastic machinery whose conformations and composition adjust dynamically to its diverse range of substrates.\nSystematic analysis of cotranscriptional RNA folding using transcription elongation complex display\nAuthors: Kelly, S. L.; Strobel, E. J.\nScore: 5.5, Published: 2023-12-23 DOI: 10.1101/2023.12.22.573115\nRNA can fold into structures that mediate diverse cellular functions. Understanding how RNA primary sequence directs the formation of functional structures requires methods that can comprehensively assess how changes in an RNA sequence affect its structure and function. Here we have developed a platform for performing high-throughput cotranscriptional RNA biochemical assays, called Transcription Elongation Complex display (TECdisplay). TECdisplay measures RNA function by fractionating a TEC library based on the activity of cotranscriptionally displayed nascent RNA. In this way, RNA function is measured as the distribution of template DNA molecules between fractions of the transcription reaction. This approach circumvents typical RNA sequencing library preparation steps that can cause technical bias. We used TECdisplay to characterize the transcription antitermination activity of 32,768 variants of the Clostridium beijerinckii pfl ZTP riboswitch designed to perturb steps within its cotranscriptional folding pathway. Our findings establish TECdisplay as an accessible platform for high-throughput RNA biochemical assays.\nMapping adipocyte interactome networks by HaloTag enrichment mass-spectrometry\nAuthors: Yazaki, J.; Yamanashi, T.; Nemoto, S.; Kobayashi, A.; Han, Y.; Hasegawa, T.; Iwase, A.; Ishikawa, M.; Konno, R.; Imami, K.; Kawashima, Y.; Seita, J.\nScore: 4.8, Published: 2023-12-25 DOI: 10.1101/2023.12.24.573280\nMapping protein interaction complexes in their natural state in vivo represents the holy grail of protein network analysis. Detection of protein interaction stoichiometry has been an important technical challenge, as few studies have focused this, yet this may be solved by artificial intelligence and proteomics. Here, we describe the development of HaloMS, a high-throughput HaloTag-based affinity purification-mass spectrometry assay for protein interaction discovery. The approach enables the rapid capture of newly expressed proteins, eliminating tedious conventional one-by-one assay. As a proof-of-principle, we used HaloMS to evaluate protein complex interactions of 17 regulatory proteins in human adipocytes. The adipocyte interactome network was validated using an in vitro pull-down assay and artificial intelligence-based prediction tools. The application of HaloMS to probe adipocyte differentiation facilitated the identification of previously unknown transcription factor-protein complexes, revealing proteome-wide human adipocyte transcription factor networks, and shedding light on how different pathways are integrated.\nIdentification of SLC25A46 interaction interfaces with mitochondrial membrane fusogens Mfn2 and Opa1\nAuthors: Boopathy, S.; Lugo, C. M.; Luce, B. E.; McDonald, J. L.; Hakim, P.; Ponce, J.; Ueberheide, B.; Chao, L. H.\nScore: 4.3, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573615\nMitochondrial fusion requires the sequential merger of four bilayers to two. The outer-membrane solute carrier protein SLC25A46 interacts with both the outer and inner-membrane dynamin family GTPases Mfn1/2 and Opa1. While SLC25A46 levels are known affect mitochondrial morphology, how SLC25A46 interacts with Mfn1/2 and Opa1 to regulate membrane fusion is not understood. In this study, we use crosslinking mass-spectrometry and AlphaFold 2 modeling to identify interfaces mediating a SLC25A46-Opa1-Mfn1/2 complex. We reveal that the bundle signaling element of Opa1 interacts with SLC25A46, and the helical repeat 1 region of Mfn2 interacts with the SLC25A46 N-terminus. We validate these newly identified interaction interfaces and show that they play a role in mitochondrial network maintenance.\n",
  "wordCount" : "1974",
  "inLanguage": "en",
  "datePublished": "2023-12-31T10:38:19Z",
  "dateModified": "2023-12-31T10:38:19Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 31, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.573103">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.573103" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.573103">
        <p class="paperTitle">Atomic context-conditioned protein sequence design using LigandMPNN</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.573103" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.573103" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dauparas, J.; Lee, G. R.; Pecoraro, R.; An, L.; Anishchenko, I.; Glasscock, C.; Baker, D.</p>
        <p class="info">Score: 38.8, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.573103' target='https://doi.org/10.1101/2023.12.22.573103'> 10.1101/2023.12.22.573103</a></p>
        <p class="abstract">Protein sequence design in the context of small molecules, nucleotides, and metals is critical to enzyme and small molecule binder and sensor design, but current state-of-the-art deep learning-based sequence design methods are unable to model non-protein atoms and molecules. Here, we describe a deep learning-based protein sequence design method called LigandMPNN that explicitly models all non-protein components of biomolecular systems. LigandMPNN significantly outperforms Rosetta and ProteinMPNN on native backbone sequence recovery for residues interacting with small molecules (63.3% vs. 50.4% &amp; 50.5%), nucleotides (50.5% vs. 35.2% &amp; 34.0%), and metals (77.5% vs. 36.0% &amp; 40.6%). LigandMPNN generates not only sequences but also sidechain conformations to allow detailed evaluation of binding interactions. Experimental characterization demonstrates that LigandMPNN can generate small molecule and DNA-binding proteins with high affinity and specificity.

One-sentence summaryWe present a deep learning-based protein sequence design method that allows explicit modeling of small molecule, nucleotide, metal, and other atomic contexts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.573181">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.573181" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.573181">
        <p class="paperTitle">Mechanism of human PINK1 activation at the TOM complex in a reconstituted system</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.573181" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.573181" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Raimi, O.; Ojha, H.; Ehses, K.; Dederer, V.; Lange, S.; Rivera, C. P.; Deegan, T.; Chen, Y.; Wightman, M.; Toth, R.; Labib, K.; Mathea, S.; Ranson, N.; Fernandez-Busnadiego, R. F.; Muqit, M.</p>
        <p class="info">Score: 20.5, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.573181' target='https://doi.org/10.1101/2023.12.23.573181'> 10.1101/2023.12.23.573181</a></p>
        <p class="abstract">Loss of function mutations in PTEN-induced kinase 1 (PINK1) are a frequent cause of early-onset Parkinsons disease (PD). Stabilisation of PINK1 at the Translocase of Outer Membrane (TOM) complex of damaged mitochondria is a critical step for its activation. To date the mechanism of how PINK1 is activated in the TOM complex is unclear. Herein we report co-expression of human PINK1 and all seven TOM subunits in Saccharomyces cerevisiae is sufficient for PINK1 activation. We use this reconstitution system to systematically assess the role of each TOM subunit towards PINK1 activation. We unambiguously demonstrate that the TOM20 and TOM70 receptor subunits are required for optimal PINK1 activation and map their sites of interaction with PINK1 using AlphaFold structural modelling and mutagenesis. We also demonstrate an essential role of the pore-containing subunit TOM40 and its structurally associated subunits TOM7 and TOM22 for PINK1 activation. These molecular findings will aid in the development of small molecule activators of PINK1 as a therapeutic strategy for PD.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.573053">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.573053" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.573053">
        <p class="paperTitle">Structure of the minor spliceosomal U11 snRNP</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.573053" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.573053" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhao, J.; Peter, D.; Brandina, I.; Liu, X.; Galej, W. P.</p>
        <p class="info">Score: 17.9, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.573053' target='https://doi.org/10.1101/2023.12.22.573053'> 10.1101/2023.12.22.573053</a></p>
        <p class="abstract">The minor spliceosome catalyses the excision of U12-dependent introns from pre-mRNAs. These introns are rare, but their removal is critical for cell viability. We obtained a cryo-EM reconstruction of the 13-subunit U11 snRNP complex, revealing structures of U11 snRNA and five minor spliceosome-specific factors. U11 snRNP appears strikingly different from the equivalent major spliceosome U1 snRNP. SNRNP25 and SNRNP35 form a dimer, which specifically recognises U11 snRNA. PDCD7 forms extended helices, which bridge SNRNP25 and SNRNP48, located at the distal ends of the particle. SNRNP48 forms multiple interfaces with U11 snRNP and, together with ZMAT5, are positioned near the 5-end of the U11 snRNA and likely stabilise the binding of the incoming 5SS. Our structure provides mechanistic insights into U12-dependent intron recognition and the evolution of the splicing machinery.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572602">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572602" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572602">
        <p class="paperTitle">De novo design of diverse small molecule binders and sensors using Shape Complementary Pseudocycles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572602" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572602" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: An, L.; Said, M.; Tran, L.; Majumder, S.; Goreshnik, I.; Lee, G. R.; Juergens, D.; Dauparas, J.; Anishchenko, I.; Coventry, B.; Bera, A. K.; Kang, A.; Levine, P. M.; Alvarez, V.; Pillai, A.; Norn, C.; Feldman, D.; Zorine, D.; Hicks, D. R.; Li, X.; Sanchez, M. G.; Vafeados, D. K.; Salveson, P. J.; Vorobieva, A. A.; Baker, D.</p>
        <p class="info">Score: 44.1, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572602' target='https://doi.org/10.1101/2023.12.20.572602'> 10.1101/2023.12.20.572602</a></p>
        <p class="abstract">A general method for designing proteins to bind and sense any small molecule of interest would be widely useful. Due to the small number of atoms to interact with, binding to small molecules with high affinity requires highly shape complementary pockets, and transducing binding events into signals is challenging. Here we describe an integrated deep learning and energy based approach for designing high shape complementarity binders to small molecules that are poised for downstream sensing applications. We employ deep learning generated psuedocycles with repeating structural units surrounding central pockets; depending on the geometry of the structural unit and repeat number, these pockets span wide ranges of sizes and shapes. For a small molecule target of interest, we extensively sample high shape complementarity pseudocycles to generate large numbers of customized potential binding pockets; the ligand binding poses and the interacting interfaces are then optimized for high affinity binding. We computationally design binders to four diverse molecules, including for the first time polar flexible molecules such as methotrexate and thyroxine, which are expressed at high levels and have nanomolar affinities straight out of the computer. Co-crystal structures are nearly identical to the design models. Taking advantage of the modular repeating structure of pseudocycles and central location of the binding pockets, we constructed low noise nanopore sensors and chemically induced dimerization systems by splitting the binders into domains which assemble into the original pseudocycle pocket upon target molecule addition.

One Sentence SummaryWe use a pseuodocycle-based shape complementarity optimizing approach to design nanomolar binders to diverse ligands, including the flexible and polar methotrexate and thyroxine, that can be directly converted into ligand-gated nanopores and chemically induced dimerization systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.27.573438">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.27.573438" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.27.573438">
        <p class="paperTitle">Chemical probes to interrogate the extreme environment of mosquito larval guts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.27.573438" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.27.573438" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guzman, L. E.; Wijetunge, A. N.; Riske, B. F.; Massani, B. B.; Riehle, M. A.; Jewett, J. C.</p>
        <p class="info">Score: 12.4, Published: 2023-12-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.27.573438' target='https://doi.org/10.1101/2023.12.27.573438'> 10.1101/2023.12.27.573438</a></p>
        <p class="abstract">Mosquito control methods are vital for the spread of life-threatening illnesses such as dengue fever, malaria, and yellow fever. Vector control technologies must be selective to minimize deleterious effects to our ecosystem. Successful methods that control mosquito larva populations utilize the uniquely high alkaline nature of the midgut. Here, we present novel protected triazabutadienes (pTBD) which are deprotected under basic conditions of the larval midgut, releasing an aryl diazonium ion (ADI) that results in protein modification. The probes contain a bioorthogonal terminal alkyne handle, enabling a selective Cu-click reaction with an azido-fluorophore for quantification by SDS PAGE and visualization using fluorescence microscopy. A control TBD, unable to release an ADI, did not label the midgut. We envision our chemical probes will aid in the development of new selective mosquito control methods thus preventing the spread of mosquito-borne illnesses with minimal impact on other organisms in the ecosystem.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.573577">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.573577" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.573577">
        <p class="paperTitle">Structural insights into rice KAI2 receptor provide functional implications for perception and signal transduction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.573577" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.573577" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guercio, A. M.; Gilio, A. K.; Pawlak, J.; Shabek, N.</p>
        <p class="info">Score: 8.1, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.573577' target='https://doi.org/10.1101/2023.12.28.573577'> 10.1101/2023.12.28.573577</a></p>
        <p class="abstract">KAI2 proteins, classified as plant /{beta} hydrolase receptors, are capable of perceiving smoke-derived butenolide signals and endogenous, yet unidentified KAI2-ligands (KLs). While the number of functional KAI2 receptors varies among land plant species, rice possesses only one KAI2 gene. Rice, a significant crop and representative of grasses, relies on KAI2-mediated Arbuscular mycorrhiza (AM) symbioses to flourish in traditionally arid and nutrient-poor environments. This study presents the first crystal structure of an active rice KAI2 hydrolase receptor. Our structural and biochemical analyses uncover grass-unique pocket residues influencing ligand sensitivity and hydrolytic activity. Through structure-guided analysis, we identify a specific residue whose mutation enables the increase or decrease of ligand perception, catalytic activity, and signal transduction. Furthermore, we investigate rice KAI2-mediated signaling by examining its ability to form a complex with its binding partner, the F-box protein DWARF3 (D3) ubiquitin ligase, demonstrating the significant role of hydrophobic interactions in the KAI2-D3 interface. This study provides new insights into the diverse and pivotal roles of the KAI2 signaling pathway in the plant kingdom, particularly in grasses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.572959">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.572959" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.572959">
        <p class="paperTitle">Structural analysis of the dynamic ribosome-translocon complex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.572959" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.572959" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lewis, A. J. O.; Hegde, R. S.</p>
        <p class="info">Score: 21.5, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.572959' target='https://doi.org/10.1101/2023.12.22.572959'> 10.1101/2023.12.22.572959</a></p>
        <p class="abstract">The protein translocon at the endoplasmic reticulum comprises the Sec61 translocation channel and numerous accessory factors that collectively facilitate the biogenesis of secretory and membrane proteins. Here, we leveraged recent advances in cryo-EM and structure prediction to derive insights into several novel configurations of the ribosome-translocon complex. We show how a transmembrane domain (TMD) in a looped configuration passes through the Sec61 lateral gate during membrane insertion; how a nascent chain can bind and constrain the conformation of ribosomal protein uL22; and how the translocon-associated protein (TRAP) complex can adjust its position during different stages of protein biogenesis. Most unexpectedly, we find that a large proportion of translocon complexes contains RAMP4 intercalated into Sec61s lateral gate, widening Sec61s central pore and contributing to its hydrophilic interior. These structures lead to mechanistic hypotheses for translocon function and highlight a remarkably plastic machinery whose conformations and composition adjust dynamically to its diverse range of substrates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.573115">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.573115" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.573115">
        <p class="paperTitle">Systematic analysis of cotranscriptional RNA folding using transcription elongation complex display</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.573115" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.573115" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kelly, S. L.; Strobel, E. J.</p>
        <p class="info">Score: 5.5, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.573115' target='https://doi.org/10.1101/2023.12.22.573115'> 10.1101/2023.12.22.573115</a></p>
        <p class="abstract">RNA can fold into structures that mediate diverse cellular functions. Understanding how RNA primary sequence directs the formation of functional structures requires methods that can comprehensively assess how changes in an RNA sequence affect its structure and function. Here we have developed a platform for performing high-throughput cotranscriptional RNA biochemical assays, called Transcription Elongation Complex display (TECdisplay). TECdisplay measures RNA function by fractionating a TEC library based on the activity of cotranscriptionally displayed nascent RNA. In this way, RNA function is measured as the distribution of template DNA molecules between fractions of the transcription reaction. This approach circumvents typical RNA sequencing library preparation steps that can cause technical bias. We used TECdisplay to characterize the transcription antitermination activity of 32,768 variants of the Clostridium beijerinckii pfl ZTP riboswitch designed to perturb steps within its cotranscriptional folding pathway. Our findings establish TECdisplay as an accessible platform for high-throughput RNA biochemical assays.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.24.573280">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.24.573280" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.24.573280">
        <p class="paperTitle">Mapping adipocyte interactome networks by HaloTag enrichment mass-spectrometry</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.24.573280" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.24.573280" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yazaki, J.; Yamanashi, T.; Nemoto, S.; Kobayashi, A.; Han, Y.; Hasegawa, T.; Iwase, A.; Ishikawa, M.; Konno, R.; Imami, K.; Kawashima, Y.; Seita, J.</p>
        <p class="info">Score: 4.8, Published: 2023-12-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.24.573280' target='https://doi.org/10.1101/2023.12.24.573280'> 10.1101/2023.12.24.573280</a></p>
        <p class="abstract">Mapping protein interaction complexes in their natural state in vivo represents the holy grail of protein network analysis. Detection of protein interaction stoichiometry has been an important technical challenge, as few studies have focused this, yet this may be solved by artificial intelligence and proteomics. Here, we describe the development of HaloMS, a high-throughput HaloTag-based affinity purification-mass spectrometry assay for protein interaction discovery. The approach enables the rapid capture of newly expressed proteins, eliminating tedious conventional one-by-one assay. As a proof-of-principle, we used HaloMS to evaluate protein complex interactions of 17 regulatory proteins in human adipocytes. The adipocyte interactome network was validated using an in vitro pull-down assay and artificial intelligence-based prediction tools. The application of HaloMS to probe adipocyte differentiation facilitated the identification of previously unknown transcription factor-protein complexes, revealing proteome-wide human adipocyte transcription factor networks, and shedding light on how different pathways are integrated.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.573615">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.573615" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.573615">
        <p class="paperTitle">Identification of SLC25A46 interaction interfaces with mitochondrial membrane fusogens Mfn2 and Opa1</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.573615" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.573615" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Boopathy, S.; Lugo, C. M.; Luce, B. E.; McDonald, J. L.; Hakim, P.; Ponce, J.; Ueberheide, B.; Chao, L. H.</p>
        <p class="info">Score: 4.3, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.573615' target='https://doi.org/10.1101/2023.12.29.573615'> 10.1101/2023.12.29.573615</a></p>
        <p class="abstract">Mitochondrial fusion requires the sequential merger of four bilayers to two. The outer-membrane solute carrier protein SLC25A46 interacts with both the outer and inner-membrane dynamin family GTPases Mfn1/2 and Opa1. While SLC25A46 levels are known affect mitochondrial morphology, how SLC25A46 interacts with Mfn1/2 and Opa1 to regulate membrane fusion is not understood. In this study, we use crosslinking mass-spectrometry and AlphaFold 2 modeling to identify interfaces mediating a SLC25A46-Opa1-Mfn1/2 complex. We reveal that the bundle signaling element of Opa1 interacts with SLC25A46, and the helical repeat 1 region of Mfn2 interacts with the SLC25A46 N-terminus. We validate these newly identified interaction interfaces and show that they play a role in mitochondrial network maintenance.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
