<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A broadly generalizable stabilization strategy for sarbecovirus fusion machinery vaccines
Authors: Lee, J.; Stewart, C.; Schaefer, A.; Leaf, E. M.; Park, Y.-J.; Asarnow, D.; Powers, J. M.; Treichel, C.; Corti, D.; Baric, R.; King, N. P.; Veesler, D.
Score: 73.5, Published: 2023-12-13 DOI: 10.1101/2023.12.12.571160
Continuous evolution of SARS-CoV-2 alters the antigenicity of the immunodominant spike (S) receptor-binding domain and N-terminal domain, undermining the efficacy of vaccines and monoclonal antibody therapies.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="A broadly generalizable stabilization strategy for sarbecovirus fusion machinery vaccines
Authors: Lee, J.; Stewart, C.; Schaefer, A.; Leaf, E. M.; Park, Y.-J.; Asarnow, D.; Powers, J. M.; Treichel, C.; Corti, D.; Baric, R.; King, N. P.; Veesler, D.
Score: 73.5, Published: 2023-12-13 DOI: 10.1101/2023.12.12.571160
Continuous evolution of SARS-CoV-2 alters the antigenicity of the immunodominant spike (S) receptor-binding domain and N-terminal domain, undermining the efficacy of vaccines and monoclonal antibody therapies." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-20T10:36:27+00:00" />
<meta property="article:modified_time" content="2023-12-20T10:36:27+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="A broadly generalizable stabilization strategy for sarbecovirus fusion machinery vaccines
Authors: Lee, J.; Stewart, C.; Schaefer, A.; Leaf, E. M.; Park, Y.-J.; Asarnow, D.; Powers, J. M.; Treichel, C.; Corti, D.; Baric, R.; King, N. P.; Veesler, D.
Score: 73.5, Published: 2023-12-13 DOI: 10.1101/2023.12.12.571160
Continuous evolution of SARS-CoV-2 alters the antigenicity of the immunodominant spike (S) receptor-binding domain and N-terminal domain, undermining the efficacy of vaccines and monoclonal antibody therapies."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "A broadly generalizable stabilization strategy for sarbecovirus fusion machinery vaccines\nAuthors: Lee, J.; Stewart, C.; Schaefer, A.; Leaf, E. M.; Park, Y.-J.; Asarnow, D.; Powers, J. M.; Treichel, C.; Corti, D.; Baric, R.; King, N. P.; Veesler, D.\nScore: 73.5, Published: 2023-12-13 DOI: 10.1101/2023.12.12.571160\nContinuous evolution of SARS-CoV-2 alters the antigenicity of the immunodominant spike (S) receptor-binding domain and N-terminal domain, undermining the efficacy of vaccines and monoclonal antibody therapies.",
  "keywords": [
    
  ],
  "articleBody": " A broadly generalizable stabilization strategy for sarbecovirus fusion machinery vaccines\nAuthors: Lee, J.; Stewart, C.; Schaefer, A.; Leaf, E. M.; Park, Y.-J.; Asarnow, D.; Powers, J. M.; Treichel, C.; Corti, D.; Baric, R.; King, N. P.; Veesler, D.\nScore: 73.5, Published: 2023-12-13 DOI: 10.1101/2023.12.12.571160\nContinuous evolution of SARS-CoV-2 alters the antigenicity of the immunodominant spike (S) receptor-binding domain and N-terminal domain, undermining the efficacy of vaccines and monoclonal antibody therapies. To overcome this challenge, we set out to develop a vaccine focusing antibody responses on the highly conserved but metastable S2 subunit, which folds as a spring-loaded fusion machinery. Here, we describe a protein design strategy enabling prefusion-stabilization of the SARS-CoV-2 S2 subunit and high yield recombinant expression of trimers with native structure and antigenicity. We demonstrate that our design strategy is broadly generalizable to all sarbecoviruses, as exemplified with the SARS-CoV-1 (clade 1a) and PRD-0038 (clade 3) S2 fusion machineries. Immunization of mice with a prefusion-stabilized SARS-CoV-2 S2 trimer vaccine elicits broadly reactive sarbecovirus antibody responses and neutralizing antibody titers of comparable magnitude against Wuhan-Hu-1 and the immune evasive XBB.1.5 variant. Vaccinated mice were protected from weight loss and disease upon challenge with SARS-CoV-2 XBB.1.5, providing proof-of-principle for fusion machinery sarbecovirus vaccines motivating future development.\nMFSD1 in complex with its accessory subunit GLMP functions as a general dipeptide uniporter in lysosomes\nAuthors: Jungnickel, K. E. J.; Guelle, O.; Iguchi, M.; Dong, W.; Kotov, V.; Gabriel, F.; Debacker, C.; Dairou, J.; McCort-Tranchepain, I.; Laqtom, N. N.; Chan, S. H.; Ejima, A.; Sato, K.; Massa Lopez, D.; Saftig, P.; Mehdipour, A. R.; Abu-Remaileh, M.; Gasnier, B.; Low, C.; Damme, M.\nScore: 22.5, Published: 2023-12-15 DOI: 10.1101/2023.12.15.570541\nLysosomal degradation of macromolecules in lysosomes produces diverse small metabolites exported by specific transporters for reuse in biosynthetic pathways. Here, we deorphanized the Major Facilitator Superfamily Domain Containing 1 (MFSD1) protein, which forms a tight complex with the Glycosylated Lysosomal Membrane Protein (GLMP) in the lysosomal membrane. Untargeted metabolomics analysis of MFSD1-deficient mouse lysosomes revealed an increase in cationic dipeptides. Purified MFSD1 selectively bound diverse dipeptides, while electrophysiological, isotope tracer, and fluorescence-based studies in Xenopus oocytes and proteoliposomes showed that MFSD1/GLMP acts as a uniporter for cationic and neutral dipeptides. Cryo-EM structure of the dipeptide-bound MFSD1/GLMP complex in outward-open conformation characterized the heterodimer interface and, in combination with molecular dynamics simulations, provided a structural basis for its selectivity towards diverse dipeptides. Together, our data identify MFSD1 as a general lysosomal dipeptide uniporter, providing an alternative route to recycle lysosomal proteolysis products when lysosomal amino acid exporters are overloaded.\nClosure of the gamma-tubulin ring complex by CDK5RAP2 activates microtubule nucleation\nAuthors: Xu, Y.; Munoz-Hernandez, H.; Krutyholowa, R.; Marxer, F.; Cetin, F.; Wieczorek, M.\nScore: 18.7, Published: 2023-12-14 DOI: 10.1101/2023.12.14.571518\nMicrotubule nucleation in cells is templated by the {gamma}-tubulin ring complex ({gamma}-TuRC), a 2.3 MDa multiprotein assembly concentrated at microtubule organizing centers (MTOCs). Current {gamma}-TuRC structures exhibit an open conformation that deviates from the geometry of /{beta}-tubulin in the microtubule, potentially explaining their low in vitro microtubule-nucleating activity. Several proteins have been proposed to activate the {gamma}-TuRC, but the mechanisms underlying activation are not known. Here, we isolated the porcine {gamma}-TuRC using CDK5RAP2s centrosomin motif 1 (CM1) and determined its structure with cryo-electron microscopy. 3D heterogeneity analysis revealed an unexpected conformation of the {gamma}-TuRC, in which five protein modules containing MZT2, GCP2, and CDK5RAP2 decorate the outer face of the holocomplex. These decorations drive a long-range constriction of the {gamma}-tubulin ring, bringing the GCP2/GCP3-rich core of the complex in close agreement with the architecture of a microtubule. A purified CDK5RAP2 fragment stimulated the microtubule nucleating-activity of the porcine {gamma}-TuRC as well as a reconstituted, CM1-free human complex in single molecule assays. Our results show that CDK5RAP2 activates the {gamma}-TuRC by promoting {gamma}-tubulin ring closure, providing a structural mechanism for the regulation of microtubule nucleation by CM1 motif proteins in mammals and revealing conformational transitions in {gamma}-tubulin that prime it for templating microtubule nucleation at MTOCs.\nOptimized mucin-selective enrichment strategy to probe the mucinome\nAuthors: Mahoney, K. E.; Chang, V.; Lucas, T. M.; Maruszko, K.; Malaker, S. A.\nScore: 17.0, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572204\nMucin-domain glycoproteins are densely O-glycosylated and play critical roles in a host of healthy and disease-driven biological functions. Previously, we developed a mucin-selective enrichment strategy by employing a catalytically inactive mucinase (StcE) conjugated to solid support. While this method was effective, it suffered from low throughput and high sample requirements. Further, the elution step required boiling in SDS, thus necessitating an in-gel digest with trypsin. Here, we optimized our previous enrichment method to include elution conditions amenable to mucinase digestion and downstream analysis with mass spectrometry. This increased throughput and lowered sample input while maintaining mucin selectivity and enhancing glycopeptide signal. We then benchmarked this technique against different O-glycan binding moieties for their ability to enrich mucins from various cell lines and human serum. Overall, the new method outperformed our previous procedure and all other enrichment techniques tested. This allowed for effective isolation of more mucin-domain glycoproteins, resulting in a high number of O-glycopeptides, thus enhancing our ability to analyze the mucinome.\nMultivalent DNA-encoded lectins on phage enable detecting compositional glycocalyx differences\nAuthors: Lima, G. M.; Chenarboo, Z. J.; Sojitra, M.; Sarkar, S.; Carpenter, E. J.; Yang, C. Y.-L.; Schmidt, E.; Lai, J.; Atrazhev, A.; Yazdan, D.; Peng, C.; Volker, E. A.; Ho, R.; Monteiro, G.; Lai, R.; Mahal, L. K.; Macauley, M. S.; Derda, R.\nScore: 11.6, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571601\nSelective detection of disease-associated changes in the cellular glycocalyx is a foundation of modern targeted therapies. Detecting minor changes in the density and identity of glycans on the cell surface is a technological challenge exacerbated by lack of 1:1 correspondence between cellular DNA/RNA and glycan structures on cell surface. We demonstrate that multivalent displays of up to 300 lectins on DNA-barcoded M13 phage on a liquid lectin array (LiLA), detects subtle differences in composition and density of glycans on cells ex vivo and in immune cells or organs in animals. For example, constructs displaying 73 copies of diCBM40 lectin per 700x5 nm virion ({varphi}-CBM73) exhibit non-linear ON/OFF-like recognition of sialoglycans on the surface of normal and cancer cells. In contrast, a high-valency {varphi}-CBM290 display, or soluble diCBM40, exhibit canonical progressive scaling in binding with increased epitope density; these constructs cannot amplify the subtle differences detected by {varphi}-CBM73. Similarly, multivalent displays of diCBM40 and Siglec-7 detect differences in the glycocalyx between stem-like and non-stem populations in cancer cells that are not detected with soluble lectins. Multivalent display of lectins on M13 scaffold with protected DNA inside the phage offer non-destructive detection of minor differences in glycocalyx in cells in vitro and in vivo not feasible to currently available technologies.\nBiosynthesis of Strained Amino Acids Through a PLP-Dependent Enzyme via Cryptic Halogenation\nAuthors: Sosa, M. B.; Leeman, J. T.; Washington, L. J.; Scheller, H. V.; Chang, M. C. Y.\nScore: 10.8, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571568\nAmino acids (AAs) are modular and modifiable building blocks which nature uses to synthesize both macromolecules, such as proteins, and small molecule natural products, such as alkaloids and non-ribosomal peptides (NRPs). While the 20 main proteinogenic AAs display relatively limited side-chain diversity, a wide range of non-canonical amino acids (ncAAs) exist that are not used by the ribosome for protein synthesis but contain a broad array of structural features and functional groups not found in proteinogenic AAs. In this communication, we report the discovery of the biosynthetic pathway for a new ncAA, pazamine, which contains a cyclopropane ring formed in two steps. In the first step, a chlorine is added onto the C4 position of lysine by a radical halogenase PazA. The cyclopropane ring is then formed in the next step by a pyridoxal-5-phosphate-dependent enzyme, PazB, via an SN2-like attack onto C4 to eliminate chloride. Genetic studies of this pathway in the native host, Pseudomonas azotoformans, show that pazamine and its succinylated derivative, pazamide, potentially inhibit ethylene biosynthesis in growing plants based on alterations in the root phenotype of Arabidopsis thaliana seedlings. We further show that PazB can be utilized to make an alternative cyclobutane-containing AA. These discoveries may lead to advances in biocatalytic production of specialty chemicals and agricultural biotechnology.\nHuman Pannexin 1 Channel is NOT Phosphorylated by Src Tyrosine Kinase at Tyr199 and Tyr309\nAuthors: Ruan, Z.; Du, J.; Lu, W.; Lee, J.; Li, Y.\nScore: 10.2, Published: 2023-12-12 DOI: 10.1101/2023.09.10.557063\nProtein phosphorylation is one of the major molecular mechanisms regulating protein activity and function throughout the cell. Pannexin 1 (PANX1) is a large-pore channel permeable to ATP and other cellular metabolites. Its tyrosine phosphorylation and subsequent activation have been found to play critical roles in diverse cellular conditions, including neuronal cell death, acute inflammation, and smooth muscle contraction. Specifically, the non-receptor kinase Src has been reported to phosphorylate Tyr198 and Tyr308 of mouse PANX1 (equivalent to Tyr199 and Tyr309 of human PANX1), resulting in channel opening and ATP release. Although the Src-dependent PANX1 activation mechanism has been widely discussed in the literature, independent validation of the tyrosine phosphorylation of PANX1 has been lacking. Here, we show that commercially available antibodies against the two phosphorylation sites mentioned above--which were used to identify endogenous PANX1 phosphorylation at these two sites-- are nonspecific and should not be used to interpret results related to PANX1 phosphorylation. We further provide evidence that neither tyrosine residue is a major phosphorylation site for Src kinase in heterologous expression systems. We call on the field to re-examine the existing paradigm of tyrosine phosphorylation-dependent activation of the PANX1 channel.\nCaulobacter crescentus RNase E condensation contributes to autoregulation and fitness\nAuthors: Nandana, V.; Al-Husini, N.; Vaishnav, A.; Dilrangi, K. H.; Schrader, J. M.\nScore: 7.7, Published: 2023-12-15 DOI: 10.1101/2023.12.15.571756\nRNase E is the most common RNA decay nuclease in bacteria, setting the global mRNA decay rate and scaffolding formation of the RNA degradosome complex and BR-bodies. To properly set the global mRNA decay rate, RNase E from Escherichia coli and neighboring gamma-proteobacteria were found to autoregulate RNase E levels via the decay of its mRNA's 5' UTR. While the 5' UTR is absent from other groups of bacteria in the Rfam database, we identified that the alpha-proteobacterium Caulobacter crescentus RNase E contains a similar 5' UTR structure that promotes RNase E autoregulation. In both bacteria, the C-terminal IDR of RNase E is required for proper autoregulation to occur, and this IDR is also necessary and sufficient for RNase E to phase-separate, generating BR-bodies. Using in vitro purified RNase E, we find that the IDR's ability to promote phase-separation correlates with enhanced 5' UTR cleavage, suggesting that phase-separation of RNase E with the 5' UTR enhances autoregulation. Finally, using growth competition experiments we find that a strain capable of autoregulation rapidly outcompetes a strain with a 5' UTR mutation that cannot autoregulate, suggesting autoregulation promotes optimal cellular fitness.\nA cyclic peptide toolkit reveals mechanistic principles of peptidylarginine deiminase IV (PADI4) regulation\nAuthors: Bertran, M. T.; Walmsley, R.; Cummings, T. F. M.; Aramburu, I.; Benton, D. J.; Assalaarachchi, J.; Chasampalioti, M.; Swanton, T.; Joshi, D.; Federico, S.; Okkenhaug, H.; Yu, L.; Oxley, D.; Walker, S.; Papayannopoulos, V.; Suga, H.; Christophorou, M. A.; Walport, L. J.\nScore: 6.7, Published: 2023-12-12 DOI: 10.1101/2023.12.12.571217\nPeptidylarginine deiminase IV (PADI4) deregulation promotes the development of autoimmunity, cancer, atherosclerosis and age-related tissue fibrosis. Genetic or pharmacological PADI4 inhibition is therapeutically efficacious in mice, but no clinically relevant inhibitors currently exist. PADI4 additionally mediates immune responses and cellular reprogramming, although the full extent of its physiological roles is unexplored. Despite detailed molecular knowledge of PADI4 activation in vitro, we lack understanding of its regulation within cells, largely due to lack of appropriate systems and tools. Here, we developed and applied a set of potent and selective PADI4 modulators. Using the mRNA- display-based RaPID system, we screened \u003e1012 cyclic peptides for high-affinity, conformation-selective binders. We report PADI4_3, an inhibitor specific for the active conformation of PADI4; PADI4_7, an inert binder, which we functionalised for the isolation and study of cellular PADI4; and PADI4_11, a first-in-class activator. Using a newly developed method for the quantification of cellular PADI4 activity, we show that PADI4_3 and PADI4_11 are effective in cells. Structural studies with PADI4_11 reveal an allosteric binding mode that may reflect the mechanism that promotes cellular PADI4 activation. This work offers new understanding of PADI4 regulation and a toolkit for the study and modulation of PADI4 across physiological and pathophysiological contexts.\nFormation of the pyruvoyl-dependent proline reductase Prd from Clostridioides difficile requires the maturation enzyme PrdH\nAuthors: Behlendorf, C.; Diwo, M. G.; Neumann-Schaal, M.; Fuchs, M.; Faber, F.; Blankenfeldt, W.\nScore: 4.6, Published: 2023-12-16 DOI: 10.1101/2023.12.15.571967\nStickland fermentation, the coupled oxidation and reduction of amino acid pairs, is a major pathway for obtaining energy in the nosocomial bacterium Clostridioides difficile. D-proline is the preferred substrate for the reductive path, making it not only a key component of the general metabolism but also impacting on the expression of the clostridial toxins TcdA and TcdB. D-proline reduction is catalyzed by the proline reductase Prd, which belongs to the pyruvoyl-dependent enzymes. These enzymes are translated as inactive proenzymes and require subsequent processing to install the covalently bound pyruvate. Whereas pyruvoyl formation by intramolecular serinolysis has been studied in unrelated enzymes, details about pyruvoyl generation by cysteinolysis such as in Prd are lacking. Here we show that Prd maturation requires a small dimeric protein that we have named PrdH. PrdH is co-encoded with the PrdA and PrdB subunits of Prd and also found in species producing similar reductases. By producing stable variants of PrdA and PrdB, we demonstrate that PrdH-mediated cleavage and pyruvoyl formation in the PrdA subunit require PrdB, which can be harnessed to produce active recombinant Prd for subsequent analyses. We further created PrdA- and PrdH-mutants to get insight into the interaction of the components and into the processing reaction itself. Finally, we show that deletion of prdH in C. difficile renders the corresponding mutant blind to proline, suggesting that this processing factor is essential for proline utilization. Due to the link between Stickland fermentation and pathogenesis, we suggest PrdH may be an attractive target for drug development.\n",
  "wordCount" : "2349",
  "inLanguage": "en",
  "datePublished": "2023-12-20T10:36:27Z",
  "dateModified": "2023-12-20T10:36:27Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 20, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.12.571160">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.12.571160" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.12.571160">
        <p class="paperTitle">A broadly generalizable stabilization strategy for sarbecovirus fusion machinery vaccines</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.12.571160" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.12.571160" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, J.; Stewart, C.; Schaefer, A.; Leaf, E. M.; Park, Y.-J.; Asarnow, D.; Powers, J. M.; Treichel, C.; Corti, D.; Baric, R.; King, N. P.; Veesler, D.</p>
        <p class="info">Score: 73.5, Published: 2023-12-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.12.571160' target='https://doi.org/10.1101/2023.12.12.571160'> 10.1101/2023.12.12.571160</a></p>
        <p class="abstract">Continuous evolution of SARS-CoV-2 alters the antigenicity of the immunodominant spike (S) receptor-binding domain and N-terminal domain, undermining the efficacy of vaccines and monoclonal antibody therapies. To overcome this challenge, we set out to develop a vaccine focusing antibody responses on the highly conserved but metastable S2 subunit, which folds as a spring-loaded fusion machinery. Here, we describe a protein design strategy enabling prefusion-stabilization of the SARS-CoV-2 S2 subunit and high yield recombinant expression of trimers with native structure and antigenicity. We demonstrate that our design strategy is broadly generalizable to all sarbecoviruses, as exemplified with the SARS-CoV-1 (clade 1a) and PRD-0038 (clade 3) S2 fusion machineries. Immunization of mice with a prefusion-stabilized SARS-CoV-2 S2 trimer vaccine elicits broadly reactive sarbecovirus antibody responses and neutralizing antibody titers of comparable magnitude against Wuhan-Hu-1 and the immune evasive XBB.1.5 variant. Vaccinated mice were protected from weight loss and disease upon challenge with SARS-CoV-2 XBB.1.5, providing proof-of-principle for fusion machinery sarbecovirus vaccines motivating future development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.570541">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.570541" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.570541">
        <p class="paperTitle">MFSD1 in complex with its accessory subunit GLMP functions as a general dipeptide uniporter in lysosomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.570541" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.570541" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jungnickel, K. E. J.; Guelle, O.; Iguchi, M.; Dong, W.; Kotov, V.; Gabriel, F.; Debacker, C.; Dairou, J.; McCort-Tranchepain, I.; Laqtom, N. N.; Chan, S. H.; Ejima, A.; Sato, K.; Massa Lopez, D.; Saftig, P.; Mehdipour, A. R.; Abu-Remaileh, M.; Gasnier, B.; Low, C.; Damme, M.</p>
        <p class="info">Score: 22.5, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.570541' target='https://doi.org/10.1101/2023.12.15.570541'> 10.1101/2023.12.15.570541</a></p>
        <p class="abstract">Lysosomal degradation of macromolecules in lysosomes produces diverse small metabolites exported by specific transporters for reuse in biosynthetic pathways. Here, we deorphanized the Major Facilitator Superfamily Domain Containing 1 (MFSD1) protein, which forms a tight complex with the Glycosylated Lysosomal Membrane Protein (GLMP) in the lysosomal membrane. Untargeted metabolomics analysis of MFSD1-deficient mouse lysosomes revealed an increase in cationic dipeptides. Purified MFSD1 selectively bound diverse dipeptides, while electrophysiological, isotope tracer, and fluorescence-based studies in Xenopus oocytes and proteoliposomes showed that MFSD1/GLMP acts as a uniporter for cationic and neutral dipeptides. Cryo-EM structure of the dipeptide-bound MFSD1/GLMP complex in outward-open conformation characterized the heterodimer interface and, in combination with molecular dynamics simulations, provided a structural basis for its selectivity towards diverse dipeptides. Together, our data identify MFSD1 as a general lysosomal dipeptide uniporter, providing an alternative route to recycle lysosomal proteolysis products when lysosomal amino acid exporters are overloaded.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.571518">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.571518" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.571518">
        <p class="paperTitle">Closure of the gamma-tubulin ring complex by CDK5RAP2 activates microtubule nucleation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.571518" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.571518" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xu, Y.; Munoz-Hernandez, H.; Krutyholowa, R.; Marxer, F.; Cetin, F.; Wieczorek, M.</p>
        <p class="info">Score: 18.7, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.571518' target='https://doi.org/10.1101/2023.12.14.571518'> 10.1101/2023.12.14.571518</a></p>
        <p class="abstract">Microtubule nucleation in cells is templated by the {gamma}-tubulin ring complex ({gamma}-TuRC), a 2.3 MDa multiprotein assembly concentrated at microtubule organizing centers (MTOCs). Current {gamma}-TuRC structures exhibit an open conformation that deviates from the geometry of /{beta}-tubulin in the microtubule, potentially explaining their low in vitro microtubule-nucleating activity. Several proteins have been proposed to activate the {gamma}-TuRC, but the mechanisms underlying activation are not known. Here, we isolated the porcine {gamma}-TuRC using CDK5RAP2s centrosomin motif 1 (CM1) and determined its structure with cryo-electron microscopy. 3D heterogeneity analysis revealed an unexpected conformation of the {gamma}-TuRC, in which five protein modules containing MZT2, GCP2, and CDK5RAP2 decorate the outer face of the holocomplex. These decorations drive a long-range constriction of the {gamma}-tubulin ring, bringing the GCP2/GCP3-rich core of the complex in close agreement with the architecture of a microtubule. A purified CDK5RAP2 fragment stimulated the microtubule nucleating-activity of the porcine {gamma}-TuRC as well as a reconstituted, CM1-free human complex in single molecule assays. Our results show that CDK5RAP2 activates the {gamma}-TuRC by promoting {gamma}-tubulin ring closure, providing a structural mechanism for the regulation of microtubule nucleation by CM1 motif proteins in mammals and revealing conformational transitions in {gamma}-tubulin that prime it for templating microtubule nucleation at MTOCs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572204">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572204" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572204">
        <p class="paperTitle">Optimized mucin-selective enrichment strategy to probe the mucinome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572204" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572204" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mahoney, K. E.; Chang, V.; Lucas, T. M.; Maruszko, K.; Malaker, S. A.</p>
        <p class="info">Score: 17.0, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572204' target='https://doi.org/10.1101/2023.12.18.572204'> 10.1101/2023.12.18.572204</a></p>
        <p class="abstract">Mucin-domain glycoproteins are densely O-glycosylated and play critical roles in a host of healthy and disease-driven biological functions. Previously, we developed a mucin-selective enrichment strategy by employing a catalytically inactive mucinase (StcE) conjugated to solid support. While this method was effective, it suffered from low throughput and high sample requirements. Further, the elution step required boiling in SDS, thus necessitating an in-gel digest with trypsin. Here, we optimized our previous enrichment method to include elution conditions amenable to mucinase digestion and downstream analysis with mass spectrometry. This increased throughput and lowered sample input while maintaining mucin selectivity and enhancing glycopeptide signal. We then benchmarked this technique against different O-glycan binding moieties for their ability to enrich mucins from various cell lines and human serum. Overall, the new method outperformed our previous procedure and all other enrichment techniques tested. This allowed for effective isolation of more mucin-domain glycoproteins, resulting in a high number of O-glycopeptides, thus enhancing our ability to analyze the mucinome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571601">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571601" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571601">
        <p class="paperTitle">Multivalent DNA-encoded lectins on phage enable detecting compositional glycocalyx differences</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571601" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571601" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lima, G. M.; Chenarboo, Z. J.; Sojitra, M.; Sarkar, S.; Carpenter, E. J.; Yang, C. Y.-L.; Schmidt, E.; Lai, J.; Atrazhev, A.; Yazdan, D.; Peng, C.; Volker, E. A.; Ho, R.; Monteiro, G.; Lai, R.; Mahal, L. K.; Macauley, M. S.; Derda, R.</p>
        <p class="info">Score: 11.6, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571601' target='https://doi.org/10.1101/2023.12.13.571601'> 10.1101/2023.12.13.571601</a></p>
        <p class="abstract">Selective detection of disease-associated changes in the cellular glycocalyx is a foundation of modern targeted therapies. Detecting minor changes in the density and identity of glycans on the cell surface is a technological challenge exacerbated by lack of 1:1 correspondence between cellular DNA/RNA and glycan structures on cell surface. We demonstrate that multivalent displays of up to 300 lectins on DNA-barcoded M13 phage on a liquid lectin array (LiLA), detects subtle differences in composition and density of glycans on cells ex vivo and in immune cells or organs in animals. For example, constructs displaying 73 copies of diCBM40 lectin per 700x5 nm virion ({varphi}-CBM73) exhibit non-linear ON/OFF-like recognition of sialoglycans on the surface of normal and cancer cells. In contrast, a high-valency {varphi}-CBM290 display, or soluble diCBM40, exhibit canonical progressive scaling in binding with increased epitope density; these constructs cannot amplify the subtle differences detected by {varphi}-CBM73. Similarly, multivalent displays of diCBM40 and Siglec-7 detect differences in the glycocalyx between stem-like and non-stem populations in cancer cells that are not detected with soluble lectins. Multivalent display of lectins on M13 scaffold with protected DNA inside the phage offer non-destructive detection of minor differences in glycocalyx in cells in vitro and in vivo not feasible to currently available technologies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571568">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571568" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571568">
        <p class="paperTitle">Biosynthesis of Strained Amino Acids Through a PLP-Dependent Enzyme via Cryptic Halogenation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571568" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571568" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sosa, M. B.; Leeman, J. T.; Washington, L. J.; Scheller, H. V.; Chang, M. C. Y.</p>
        <p class="info">Score: 10.8, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571568' target='https://doi.org/10.1101/2023.12.13.571568'> 10.1101/2023.12.13.571568</a></p>
        <p class="abstract">Amino acids (AAs) are modular and modifiable building blocks which nature uses to synthesize both macromolecules, such as proteins, and small molecule natural products, such as alkaloids and non-ribosomal peptides (NRPs). While the 20 main proteinogenic AAs display relatively limited side-chain diversity, a wide range of non-canonical amino acids (ncAAs) exist that are not used by the ribosome for protein synthesis but contain a broad array of structural features and functional groups not found in proteinogenic AAs. In this communication, we report the discovery of the biosynthetic pathway for a new ncAA, pazamine, which contains a cyclopropane ring formed in two steps. In the first step, a chlorine is added onto the C4 position of lysine by a radical halogenase PazA. The cyclopropane ring is then formed in the next step by a pyridoxal-5-phosphate-dependent enzyme, PazB, via an SN2-like attack onto C4 to eliminate chloride. Genetic studies of this pathway in the native host, Pseudomonas azotoformans, show that pazamine and its succinylated derivative, pazamide, potentially inhibit ethylene biosynthesis in growing plants based on alterations in the root phenotype of Arabidopsis thaliana seedlings. We further show that PazB can be utilized to make an alternative cyclobutane-containing AA. These discoveries may lead to advances in biocatalytic production of specialty chemicals and agricultural biotechnology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.10.557063">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.10.557063" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.10.557063">
        <p class="paperTitle">Human Pannexin 1 Channel is NOT Phosphorylated by Src Tyrosine Kinase at Tyr199 and Tyr309</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.10.557063" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.10.557063" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ruan, Z.; Du, J.; Lu, W.; Lee, J.; Li, Y.</p>
        <p class="info">Score: 10.2, Published: 2023-12-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.10.557063' target='https://doi.org/10.1101/2023.09.10.557063'> 10.1101/2023.09.10.557063</a></p>
        <p class="abstract">Protein phosphorylation is one of the major molecular mechanisms regulating protein activity and function throughout the cell. Pannexin 1 (PANX1) is a large-pore channel permeable to ATP and other cellular metabolites. Its tyrosine phosphorylation and subsequent activation have been found to play critical roles in diverse cellular conditions, including neuronal cell death, acute inflammation, and smooth muscle contraction. Specifically, the non-receptor kinase Src has been reported to phosphorylate Tyr198 and Tyr308 of mouse PANX1 (equivalent to Tyr199 and Tyr309 of human PANX1), resulting in channel opening and ATP release. Although the Src-dependent PANX1 activation mechanism has been widely discussed in the literature, independent validation of the tyrosine phosphorylation of PANX1 has been lacking. Here, we show that commercially available antibodies against the two phosphorylation sites mentioned above--which were used to identify endogenous PANX1 phosphorylation at these two sites-- are nonspecific and should not be used to interpret results related to PANX1 phosphorylation. We further provide evidence that neither tyrosine residue is a major phosphorylation site for Src kinase in heterologous expression systems. We call on the field to re-examine the existing paradigm of tyrosine phosphorylation-dependent activation of the PANX1 channel.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.571756">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.571756" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.571756">
        <p class="paperTitle">Caulobacter crescentus RNase E condensation contributes to autoregulation and fitness</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.571756" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.571756" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nandana, V.; Al-Husini, N.; Vaishnav, A.; Dilrangi, K. H.; Schrader, J. M.</p>
        <p class="info">Score: 7.7, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.571756' target='https://doi.org/10.1101/2023.12.15.571756'> 10.1101/2023.12.15.571756</a></p>
        <p class="abstract">RNase E is the most common RNA decay nuclease in bacteria, setting the global mRNA decay rate and scaffolding formation of the RNA degradosome complex and BR-bodies. To properly set the global mRNA decay rate, RNase E from Escherichia coli and neighboring gamma-proteobacteria were found to autoregulate RNase E levels via the decay of its mRNA&#39;s 5&#39; UTR. While the 5&#39; UTR is absent from other groups of bacteria in the Rfam database, we identified that the alpha-proteobacterium Caulobacter crescentus RNase E contains a similar 5&#39; UTR structure that promotes RNase E autoregulation. In both bacteria, the C-terminal IDR of RNase E is required for proper autoregulation to occur, and this IDR is also necessary and sufficient for RNase E to phase-separate, generating BR-bodies. Using in vitro purified RNase E, we find that the IDR&#39;s ability to promote phase-separation correlates with enhanced 5&#39; UTR cleavage, suggesting that phase-separation of RNase E with the 5&#39; UTR enhances autoregulation. Finally, using growth competition experiments we find that a strain capable of autoregulation rapidly outcompetes a strain with a 5&#39; UTR mutation that cannot autoregulate, suggesting autoregulation promotes optimal cellular fitness.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.12.571217">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.12.571217" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.12.571217">
        <p class="paperTitle">A cyclic peptide toolkit reveals mechanistic principles of peptidylarginine deiminase IV (PADI4) regulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.12.571217" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.12.571217" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bertran, M. T.; Walmsley, R.; Cummings, T. F. M.; Aramburu, I.; Benton, D. J.; Assalaarachchi, J.; Chasampalioti, M.; Swanton, T.; Joshi, D.; Federico, S.; Okkenhaug, H.; Yu, L.; Oxley, D.; Walker, S.; Papayannopoulos, V.; Suga, H.; Christophorou, M. A.; Walport, L. J.</p>
        <p class="info">Score: 6.7, Published: 2023-12-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.12.571217' target='https://doi.org/10.1101/2023.12.12.571217'> 10.1101/2023.12.12.571217</a></p>
        <p class="abstract">Peptidylarginine deiminase IV (PADI4) deregulation promotes the development of autoimmunity, cancer, atherosclerosis and age-related tissue fibrosis. Genetic or pharmacological PADI4 inhibition is therapeutically efficacious in mice, but no clinically relevant inhibitors currently exist. PADI4 additionally mediates immune responses and cellular reprogramming, although the full extent of its physiological roles is unexplored. Despite detailed molecular knowledge of PADI4 activation in vitro, we lack understanding of its regulation within cells, largely due to lack of appropriate systems and tools. Here, we developed and applied a set of potent and selective PADI4 modulators. Using the mRNA- display-based RaPID system, we screened &gt;1012 cyclic peptides for high-affinity, conformation-selective binders. We report PADI4_3, an inhibitor specific for the active conformation of PADI4; PADI4_7, an inert binder, which we functionalised for the isolation and study of cellular PADI4; and PADI4_11, a first-in-class activator. Using a newly developed method for the quantification of cellular PADI4 activity, we show that PADI4_3 and PADI4_11 are effective in cells. Structural studies with PADI4_11 reveal an allosteric binding mode that may reflect the mechanism that promotes cellular PADI4 activation. This work offers new understanding of PADI4 regulation and a toolkit for the study and modulation of PADI4 across physiological and pathophysiological contexts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.571967">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.571967" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.571967">
        <p class="paperTitle">Formation of the pyruvoyl-dependent proline reductase Prd from Clostridioides difficile requires the maturation enzyme PrdH</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.571967" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.571967" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Behlendorf, C.; Diwo, M. G.; Neumann-Schaal, M.; Fuchs, M.; Faber, F.; Blankenfeldt, W.</p>
        <p class="info">Score: 4.6, Published: 2023-12-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.571967' target='https://doi.org/10.1101/2023.12.15.571967'> 10.1101/2023.12.15.571967</a></p>
        <p class="abstract">Stickland fermentation, the coupled oxidation and reduction of amino acid pairs, is a major pathway for obtaining energy in the nosocomial bacterium Clostridioides difficile. D-proline is the preferred substrate for the reductive path, making it not only a key component of the general metabolism but also impacting on the expression of the clostridial toxins TcdA and TcdB. D-proline reduction is catalyzed by the proline reductase Prd, which belongs to the pyruvoyl-dependent enzymes. These enzymes are translated as inactive proenzymes and require subsequent processing to install the covalently bound pyruvate. Whereas pyruvoyl formation by intramolecular serinolysis has been studied in unrelated enzymes, details about pyruvoyl generation by cysteinolysis such as in Prd are lacking. Here we show that Prd maturation requires a small dimeric protein that we have named PrdH. PrdH is co-encoded with the PrdA and PrdB subunits of Prd and also found in species producing similar reductases. By producing stable variants of PrdA and PrdB, we demonstrate that PrdH-mediated cleavage and pyruvoyl formation in the PrdA subunit require PrdB, which can be harnessed to produce active recombinant Prd for subsequent analyses. We further created PrdA- and PrdH-mutants to get insight into the interaction of the components and into the processing reaction itself. Finally, we show that deletion of prdH in C. difficile renders the corresponding mutant blind to proline, suggesting that this processing factor is essential for proline utilization. Due to the link between Stickland fermentation and pathogenesis, we suggest PrdH may be an attractive target for drug development.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
