<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Engineering and Evolution of the Complete Reductive Glycine Pathway in Saccharomyces cerevisiae for Formate and CO2 Assimilation
Authors: Bysani, V. R.; Alam, A. S. M.; Bar-Even, A.; Machens, F.
Score: 16.6, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548313
Using captured CO2 and C1-feedstocks like formate and methanol derived from electrochemical activation of CO2 are key solutions for transforming industrial processes towards a circular carbon economy. Engineering formate and CO2-based growth in the biotechnologically relevant yeast Saccharomyces cerevisiae could boost the emergence of a formate-mediated circular bio-economy.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Engineering and Evolution of the Complete Reductive Glycine Pathway in Saccharomyces cerevisiae for Formate and CO2 Assimilation
Authors: Bysani, V. R.; Alam, A. S. M.; Bar-Even, A.; Machens, F.
Score: 16.6, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548313
Using captured CO2 and C1-feedstocks like formate and methanol derived from electrochemical activation of CO2 are key solutions for transforming industrial processes towards a circular carbon economy. Engineering formate and CO2-based growth in the biotechnologically relevant yeast Saccharomyces cerevisiae could boost the emergence of a formate-mediated circular bio-economy." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-19T10:38:24+00:00" />
<meta property="article:modified_time" content="2023-07-19T10:38:24+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Engineering and Evolution of the Complete Reductive Glycine Pathway in Saccharomyces cerevisiae for Formate and CO2 Assimilation
Authors: Bysani, V. R.; Alam, A. S. M.; Bar-Even, A.; Machens, F.
Score: 16.6, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548313
Using captured CO2 and C1-feedstocks like formate and methanol derived from electrochemical activation of CO2 are key solutions for transforming industrial processes towards a circular carbon economy. Engineering formate and CO2-based growth in the biotechnologically relevant yeast Saccharomyces cerevisiae could boost the emergence of a formate-mediated circular bio-economy."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Engineering and Evolution of the Complete Reductive Glycine Pathway in Saccharomyces cerevisiae for Formate and CO2 Assimilation\nAuthors: Bysani, V. R.; Alam, A. S. M.; Bar-Even, A.; Machens, F.\nScore: 16.6, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548313\nUsing captured CO2 and C1-feedstocks like formate and methanol derived from electrochemical activation of CO2 are key solutions for transforming industrial processes towards a circular carbon economy. Engineering formate and CO2-based growth in the biotechnologically relevant yeast Saccharomyces cerevisiae could boost the emergence of a formate-mediated circular bio-economy.",
  "keywords": [
    
  ],
  "articleBody": " Engineering and Evolution of the Complete Reductive Glycine Pathway in Saccharomyces cerevisiae for Formate and CO2 Assimilation\nAuthors: Bysani, V. R.; Alam, A. S. M.; Bar-Even, A.; Machens, F.\nScore: 16.6, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548313\nUsing captured CO2 and C1-feedstocks like formate and methanol derived from electrochemical activation of CO2 are key solutions for transforming industrial processes towards a circular carbon economy. Engineering formate and CO2-based growth in the biotechnologically relevant yeast Saccharomyces cerevisiae could boost the emergence of a formate-mediated circular bio-economy. This study adopts a growth-coupled selection scheme for modular implementation of the Reductive Glycine Pathway (RGP) and subsequent Adaptive Laboratory Evolution (ALE) to enable formate and CO2 assimilation for biomass formation in yeast. We first constructed a serine biosensor strain and then implemented the serine synthesis module of the RGP into yeast, establishing glycine and serine synthesis from formate and CO2. ALE improved the RGP-dependent growth by 8-fold. 13C-labeling experiments reveal glycine, serine, and pyruvate synthesis via the RGP, demonstrating the complete pathway activity. Further, we reestablished formate and CO2-dependent growth in non-evolved biosensor strains via reverse-engineering a mutation in GDH1 identified from ALE. This mutation led to significantly more 13C-formate assimilation than in WT without any selection or overexpression of the RGP. Overall, we demonstrated the activity of the complete RGP, showing evidence for carbon transfer from formate to pyruvate coupled with CO2 assimilation.\nA bespoke analytical workflow for the confident identification of sulfopeptides and their discrimination from phosphopeptides\nAuthors: Daly, L. A.; Byrne, D. P.; Perkins, S.; Brownridge, P. J.; McDonnell, E.; Jones, A. R.; Eyers, P. A.; Eyers, C. E.\nScore: 11.3, Published: 2023-07-15 DOI: 10.1101/2023.07.15.549150\nProtein tyrosine sulfation (sY) is a post-translational modification (PTM) catalysed by Golgi-resident Tyrosyl Protein Sul-foTransferases (TPSTs). Information on protein tyrosine sulfation is currently limited to ~50 human proteins with only a handful having verified sites of sulfation. The contribution of this chemical moiety for the regulation of biological processes, both inside and outside the cell, remains poorly defined, in large part due to analytical limitations. Mass spectrometry-based proteomics is the method of choice for PTM analysis, but has yet to be applied for the systematic investigation and large-scale analysis of biomolecular sulfation (constituting the sulfome), primarily due to issues associated with discrimination of sY- from phosphotyrosine (pY)-containing peptides. In this study, we developed a mass spectrometry (MS)-based workflow centred on the characterization of sY-peptides, incorporating optimised Zr4+-IMAC and TiO2 enrichment strategies. Extensive characterization of a panel of sY- and pY-peptides using an array of MS fragmentation regimes (CID, HCD, EThcC, ETciD, UVPD) highlights differences in the ability to generate site-determining product ions, which can be exploited to differentiate sulfated peptides from nominally isobaric phosphopeptides based on precursor ion neutral loss at low collision energy. Application of our analytical workflow to a HEK-293 cell extracellular secretome facilitated identification of 21 new sulfotyrosine-containing proteins, several of which we validate enzymatically using in vitro sulfation assays. This study demonstrates the applicability of our strategy for confident, high-throughput, sulfomics studies, and reveals new sY interplay between enzymes relevant to both protein and glycan sulfation.\nDiversity and modularity of tyrosine-accepting tRNA-like structures\nAuthors: Sherlock, M. E.; Langeberg, C. J.; Kieft, J. S.\nScore: 15.3, Published: 2023-07-10 DOI: 10.1101/2023.07.08.548219\nCertain positive-sense single-stranded RNA viruses contain elements at their 3 termini that structurally mimic tRNAs. These tRNA-like structures (TLSs) are classified based on which amino acid is covalently added to the 3 end by host aminoacyl-tRNA synthetase. Recently, a cryoEM reconstruction of a representative tyrosine-accepting tRNA-like structure (TLSTyr) from brome mosaic virus (BMV) revealed a unique mode of recognition of the viral anticodon-mimicking domain by tyrosyl-tRNA synthetase. Some viruses in the hordeivirus genus of Virgaviridae are also selectively aminoacylated with tyrosine, yet these TLS RNAs have a different architecture in the 5 domain that comprises the atypical anticodon loop mimic. Herein, we present bioinformatic and biochemical data supporting a distinct secondary structure for the 5' domain of the hordeivirus TLSTyr compared to those in Bromoviridae. Despite forming a different secondary structure, the 5' domain is necessary to achieve robust in vitro aminoacylation. Furthermore, a chimeric RNA containing the 5' domain from the BMV TLSTyr and the 3' domain from a hordeivirus TLSTyr are aminoacylated, illustrating modularity in these structured RNA elements. We propose that the structurally distinct 5' domain of the hordeivirus TLSTyrs performs the same role in mimicking the anticodon loop as its counterpart in the BMV TLSTyr. Finally, these structurally and phylogenetically divergent types of TLSTyr provide insight into the evolutionary connections between all classes of viral tRNA-like structures.\nAlgal kainoid synthases exhibit substrate-dependent hydroxylation and cyclization activities\nAuthors: Hopiavuori, A. R.; McKinnie, S. M. K.\nScore: 8.3, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548461\nFeII/-ketoglutarate-dependent dioxygenases (Fe/KG) are a large enzyme family that functionalize C-H bonds on diverse organic substrates. Although Fe/KG homologs catalyze an array of chemically useful reactions, hydroxylation typically predominates. Microalgal DabC uniquely forms a novel C-C bond to construct the bioactive pyrrolidine ring in domoic acid biosynthesis. However, this kainoid synthase exclusively performs a stereospecific hydroxylation reaction on its cis substrate regioisomer. Mechanistic and kinetic analyses with native and alternative substrates identified a 20-fold rate increase in DabC radical cyclization over {beta}-hydroxylation, with no observable 1,5-hydrogen atom transfer. Moreover, this dual activity was conserved among macroalgal RadC1 and KabC homologs and provided insight into substrate recognition and reactivity trends. Investigation of this substrate-dependent chemistry improves our understanding of Fe/KG enzymes and their biocatalytic application.\nDecrypting the languages of allostery in membrane-bound K-Ras4B using four complementary in silico approaches\nAuthors: Castelli, M.; Marchetti, F.; Serapian, S. A.; Colombo, G.\nScore: 7.8, Published: 2023-07-14 DOI: 10.1101/2023.07.14.549022\nProteins are evolved molecular machines whose diverse biochemical functions are (also) dynamically regulated by allostery, through which even faraway protein residues can conformationally communicate. Allostery can express itself in different ways, akin to different \"languages\": allosteric control pathways predominating in an unperturbed protein are superseded by others as soon as a perturbation arises--e.g, mutation--that alters its function (pathologically or not). Accurately modeling these often-unintuitive phenomena could therefore help explain functional changes in a specific protein whenever they are unclear. Unbiased molecular dynamics (MD) simulations are a possibility; however, since allostery can operate at longer timescales than those accessible by MD, simulations require integration with a reliable method able to, e.g., detect regions of incipient allosteric change, or likely perturbation pathways. Several such (valid) methods exist, but are typically applied singularly, only yielding a partial (\"monolinguistic\") picture of allostery: we argue their joint application could significantly enrich this picture. To prove this, we perform unbiased MD simulations ([~]100 {micro}s) of the widely studied, allosterically active oncotarget K-Ras4B, solvated and embedded in a phospholipid membrane, proceeding to decrypt its allostery using four showcase \"languages\": Distance Fluctuation analysis and the Shortest Path Map capture allosteric communication hotspots at equilibrium; Anisotropic Thermal Diffusion and Dynamic Non-Equilibrium MD assess them once the GTP that oncogenically activates K-Ras4B is, respectively, either superheated or hydrolyzed. \"Languages\" provide a uniquely articulate, mutually coherent, experimentally consistent picture of allostery in K-Ras4B. At equilibrium, pathways stretch from the membrane-embedded hypervariable region all the way to the active site, touching known flexible allosteric \"switches\" and proposed pockets. Upon GTP cleavage/perturbation, known to inactivate K-Ras4B, allosteric signals most reverberate on switches and interfaces that recruit effector proteins. Our work highlights the benefits of integrating different allostery detection methods, even in conjunction with unbiased MD.\nDirect, quantitative, and comprehensive analysis of tRNA acylation using intact tRNA liquid-chromatography mass-spectrometry\nAuthors: Fricke, R.; Knudson, I.; Schepartz, A.\nScore: 5.2, Published: 2023-07-14 DOI: 10.1101/2023.07.14.549096\nAminoacyl-tRNA synthetases (aaRSs) provide the functional and essential link between the sequence of an mRNA and the protein it encodes. aaRS enzymes catalyze a two-step chemical reaction that acylates specific tRNAs with a cognate -amino acid. In addition to their role in translation, acylated tRNAs contribute to non-ribosomal natural product biosynthesis and are implicated in multiple human diseases. From the standpoint of synthetic biology, the acylation of tRNAs with a non-canonical -amino acid (ncAA) or more recently, a non--amino acid monomer (nAA) is a critical first step in the incorporation of these monomers into proteins, where they can be used for fundamental and applied science. These endeavors all demand an understanding of aaRS activity and specificity. Although a number of methods to monitor aaRS function in vitro or in vivo have been developed, many evaluate only the first step of the two-step reaction, require the use of radioactivity, or are slow, difficult to generalize, or both. Here we describe an LC-MS assay that rapidly, quantitatively, and directly monitors aaRS activity by detecting the intact acyl-tRNA product. After a simple tRNA acylation reaction workup, acyl- and non-acyl-tRNA molecules are resolved using ion-pairing reverse phase chromatography and their exact masses are determined using high-resolution time-of-flight mass spectrometry. The intact tRNA assay we describe is fast, simple, and quantifies reaction yields as low as 0.23%. The assay can also be employed on tRNAs acylated with flexizyme to detect products that are undetectable using standard techniques. The protocol requires basic expertise in molecular biology, mass spectrometry, and RNAse-free techniques.\nNanobody repertoire generated against the spike protein of ancestral SARS-CoV-2 remains efficacious against the rapidly evolving virus\nAuthors: Ketaren, N. E.; Mast, F. D.; Fridy, P. C.; Olivier, J. P.; Sanyal, T.; Sali, A.; Chait, B. T.; Rout, M. P.; Aitchison, J.\nScore: 5.2, Published: 2023-07-14 DOI: 10.1101/2023.07.14.549041\nTo date, all major modes of monoclonal antibody therapy targeting SARS-CoV-2 have lost significant efficacy against the latest circulating variants. As SARS-CoV-2 omicron sublineages account for over 90% of COVID-19 infections, evasion of immune responses generated by vaccination or exposure to previous variants poses a significant challenge. A compelling new therapeutic strategy against SARS-CoV-2 is that of single domain antibodies, termed nanobodies, which address certain limitations of monoclonal antibodies. Here we demonstrate that our high-affinity nanobody repertoire, generated against wild-type SARS-CoV-2 spike protein (Mast, Fridy et al. 2021), remains effective against variants of concern, including omicron BA.4/BA.5; a subset is predicted to counter resistance in emerging XBB and BQ.1.1 sublineages. Furthermore, we reveal the synergistic potential of nanobody cocktails in neutralizing emerging variants. Our study highlights the power of nanobody technology as a versatile therapeutic and diagnostic tool to combat rapidly evolving infectious diseases such as SARS-CoV-2.\nDual engagement of the nucleosomal acidic patches is essential for deposition of histone H2A.Z by SWR1C\nAuthors: Baier, A. S.; Gioacchini, N.; Eek, P.; Tan, S.; Peterson, C. L.\nScore: 13.1, Published: 2023-07-15 DOI: 10.1101/2021.12.08.471801\nThe SWR1C chromatin remodeling enzyme catalyzes the ATP-dependent exchange of nucleosomal histone H2A for the histone variant H2A.Z, a key variant involved in a multitude of nuclear functions. How the 14-subunit SWR1C engages the nucleosomal substrate remains largely unknown. Numerous studies on the ISWI, CHD1, and SWI/SNF families of chromatin remodeling enzymes have demonstrated key roles for the nucleosomal acidic patch for remodeling activity, however a role for this nucleosomal epitope in nucleosome editing by SWR1C has not been tested. Here, we employ a variety of biochemical assays to demonstrate an essential role for the acidic patch in the H2A.Z exchange reaction. Utilizing asymmetrically assembled nucleosomes, we demonstrate that the acidic patches on each face of the nucleosome are required for SWR1C-mediated dimer exchange, suggesting SWR1C engages the nucleosome in a \"pincer-like\" conformation, engaging both patches simultaneously. Loss of a single acidic patch results in loss of high affinity nucleosome binding and nucleosomal stimulation of ATPase activity. We identify a conserved arginine-rich motif within the Swc5 subunit that binds the acidic patch and is key for dimer exchange activity. In addition, our cryoEM structure of a Swc5-nucleosome complex suggests that promoter proximal, histone H2B ubiquitinylation may regulate H2A.Z deposition. Together these findings provide new insights into how SWR1C engages its nucleosomal substrate to promote efficient H2A.Z deposition.\nThe AAA+ protein Msp1 selects substrates by recognizing a hydrophobic mismatch between the substrate transmembrane domain and the lipid bilayer\nAuthors: Fresenius, H. L.; Gaur, D.; Wohlever, M. L.\nScore: 2.8, Published: 2023-07-11 DOI: 10.1101/2023.07.11.548587\nAn essential aspect of protein quality control is enzymatic removal of membrane proteins from the lipid bilayer. Failures in this essential cellular process are associated with neurodegenerative diseases and cancer. Msp1 is a AAA+ (ATPases Associated with diverse cellular Activities) protein that removes mistargeted proteins from the outer mitochondrial membrane (OMM). How Msp1 selectively recognizes and extracts substrates within the complex OMM ecosystem, and the role of the lipid bilayer on these processes is unknown. Here, we describe the development of fully defined, rapid, and quantitative extraction assay that retains physiological substrate selectivity. Using this new assay, we systematically modified both substrates and the lipid environment to demonstrate that Msp1 recognizes substrates by a hydrophobic mismatch between the substrate TMD and the lipid bilayer. We further demonstrate that the rate limiting step in Msp1 activity is extraction of the TMD from the lipid bilayer. Together, these results provide foundational insights into how the lipid bilayer influences AAA+ mediated membrane protein extraction.\nThe pathogenic T42A mutation in SHP2 rewires interaction specificity and enhances signaling\nAuthors: van Vlimmeren, A. E.; Voleti, R.; Chartier, C. A.; Jiang, Z.; Karandur, D.; Shah, N. H.\nScore: 6.3, Published: 2023-07-10 DOI: 10.1101/2023.07.10.548257\nMutations in the tyrosine phosphatase SHP2 are associated with various human diseases. Most of these mutations increase the basal catalytic activity of SHP2 by disrupting auto-inhibitory interactions between its phosphatase domain and N-terminal SH2 (phosphotyrosine recognition) domain. By contrast, several other disease-associated mutations located in the ligand-binding pockets of the N- or C-terminal SH2 domains do not increase basal activity and likely exert their pathogenicity through alternative mechanisms. We lack a molecular understanding of how these SH2 mutations impact SHP2 structure, activity, and signaling. Here, we characterize five SHP2 SH2 domain ligand-binding pocket mutants through a combination of high-throughput biochemical screens, biophysical and biochemical measurements, molecular dynamics simulations, and cellular assays. We show that, while several of these mutants alter binding affinity to phosphorylation sites, the T42A mutation in the N-SH2 domain is unique in that it also alters ligand-binding specificity. The functional consequence of this altered specificity is that the T42A mutant has biased sensitivity toward a subset of activating ligands. Our study highlights an example of a nuanced mechanism of action for a disease-associated mutation, characterized by a change in protein-protein interaction specificity that alters enzyme activation.\n",
  "wordCount" : "2373",
  "inLanguage": "en",
  "datePublished": "2023-07-19T10:38:24Z",
  "dateModified": "2023-07-19T10:38:24Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta"><span>updated on July 19, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548313">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548313" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548313">
        <p class="paperTitle">Engineering and Evolution of the Complete Reductive Glycine Pathway in Saccharomyces cerevisiae for Formate and CO2 Assimilation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548313" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548313" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bysani, V. R.; Alam, A. S. M.; Bar-Even, A.; Machens, F.</p>
        <p class="info">Score: 16.6, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548313' target='https://doi.org/10.1101/2023.07.10.548313'> 10.1101/2023.07.10.548313</a></p>
        <p class="abstract">Using captured CO2 and C1-feedstocks like formate and methanol derived from electrochemical activation of CO2 are key solutions for transforming industrial processes towards a circular carbon economy. Engineering formate and CO2-based growth in the biotechnologically relevant yeast Saccharomyces cerevisiae could boost the emergence of a formate-mediated circular bio-economy. This study adopts a growth-coupled selection scheme for modular implementation of the Reductive Glycine Pathway (RGP) and subsequent Adaptive Laboratory Evolution (ALE) to enable formate and CO2 assimilation for biomass formation in yeast. We first constructed a serine biosensor strain and then implemented the serine synthesis module of the RGP into yeast, establishing glycine and serine synthesis from formate and CO2. ALE improved the RGP-dependent growth by 8-fold. 13C-labeling experiments reveal glycine, serine, and pyruvate synthesis via the RGP, demonstrating the complete pathway activity. Further, we reestablished formate and CO2-dependent growth in non-evolved biosensor strains via reverse-engineering a mutation in GDH1 identified from ALE. This mutation led to significantly more 13C-formate assimilation than in WT without any selection or overexpression of the RGP. Overall, we demonstrated the activity of the complete RGP, showing evidence for carbon transfer from formate to pyruvate coupled with CO2 assimilation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.15.549150">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.15.549150" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.15.549150">
        <p class="paperTitle">A bespoke analytical workflow for the confident identification of sulfopeptides and their discrimination from phosphopeptides</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.15.549150" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.15.549150" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Daly, L. A.; Byrne, D. P.; Perkins, S.; Brownridge, P. J.; McDonnell, E.; Jones, A. R.; Eyers, P. A.; Eyers, C. E.</p>
        <p class="info">Score: 11.3, Published: 2023-07-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.15.549150' target='https://doi.org/10.1101/2023.07.15.549150'> 10.1101/2023.07.15.549150</a></p>
        <p class="abstract">Protein tyrosine sulfation (sY) is a post-translational modification (PTM) catalysed by Golgi-resident Tyrosyl Protein Sul-foTransferases (TPSTs). Information on protein tyrosine sulfation is currently limited to ~50 human proteins with only a handful having verified sites of sulfation. The contribution of this chemical moiety for the regulation of biological processes, both inside and outside the cell, remains poorly defined, in large part due to analytical limitations. Mass spectrometry-based proteomics is the method of choice for PTM analysis, but has yet to be applied for the systematic investigation and large-scale analysis of biomolecular sulfation (constituting the sulfome), primarily due to issues associated with discrimination of sY- from phosphotyrosine (pY)-containing peptides. In this study, we developed a mass spectrometry (MS)-based workflow centred on the characterization of sY-peptides, incorporating optimised Zr4&#43;-IMAC and TiO2 enrichment strategies. Extensive characterization of a panel of sY- and pY-peptides using an array of MS fragmentation regimes (CID, HCD, EThcC, ETciD, UVPD) highlights differences in the ability to generate site-determining product ions, which can be exploited to differentiate sulfated peptides from nominally isobaric phosphopeptides based on precursor ion neutral loss at low collision energy. Application of our analytical workflow to a HEK-293 cell extracellular secretome facilitated identification of 21 new sulfotyrosine-containing proteins, several of which we validate enzymatically using in vitro sulfation assays. This study demonstrates the applicability of our strategy for confident, high-throughput, sulfomics studies, and reveals new sY interplay between enzymes relevant to both protein and glycan sulfation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.08.548219">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.08.548219" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.08.548219">
        <p class="paperTitle">Diversity and modularity of tyrosine-accepting tRNA-like structures</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.08.548219" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.08.548219" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sherlock, M. E.; Langeberg, C. J.; Kieft, J. S.</p>
        <p class="info">Score: 15.3, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.08.548219' target='https://doi.org/10.1101/2023.07.08.548219'> 10.1101/2023.07.08.548219</a></p>
        <p class="abstract">Certain positive-sense single-stranded RNA viruses contain elements at their 3 termini that structurally mimic tRNAs. These tRNA-like structures (TLSs) are classified based on which amino acid is covalently added to the 3 end by host aminoacyl-tRNA synthetase. Recently, a cryoEM reconstruction of a representative tyrosine-accepting tRNA-like structure (TLSTyr) from brome mosaic virus (BMV) revealed a unique mode of recognition of the viral anticodon-mimicking domain by tyrosyl-tRNA synthetase. Some viruses in the hordeivirus genus of Virgaviridae are also selectively aminoacylated with tyrosine, yet these TLS RNAs have a different architecture in the 5 domain that comprises the atypical anticodon loop mimic. Herein, we present bioinformatic and biochemical data supporting a distinct secondary structure for the 5&#39; domain of the hordeivirus TLSTyr compared to those in Bromoviridae. Despite forming a different secondary structure, the 5&#39; domain is necessary to achieve robust in vitro aminoacylation. Furthermore, a chimeric RNA containing the 5&#39; domain from the BMV TLSTyr and the 3&#39; domain from a hordeivirus TLSTyr are aminoacylated, illustrating modularity in these structured RNA elements. We propose that the structurally distinct 5&#39; domain of the hordeivirus TLSTyrs performs the same role in mimicking the anticodon loop as its counterpart in the BMV TLSTyr. Finally, these structurally and phylogenetically divergent types of TLSTyr provide insight into the evolutionary connections between all classes of viral tRNA-like structures.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548461">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548461" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548461">
        <p class="paperTitle">Algal kainoid synthases exhibit substrate-dependent hydroxylation and cyclization activities</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548461" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548461" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hopiavuori, A. R.; McKinnie, S. M. K.</p>
        <p class="info">Score: 8.3, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548461' target='https://doi.org/10.1101/2023.07.10.548461'> 10.1101/2023.07.10.548461</a></p>
        <p class="abstract">FeII/-ketoglutarate-dependent dioxygenases (Fe/KG) are a large enzyme family that functionalize C-H bonds on diverse organic substrates. Although Fe/KG homologs catalyze an array of chemically useful reactions, hydroxylation typically predominates. Microalgal DabC uniquely forms a novel C-C bond to construct the bioactive pyrrolidine ring in domoic acid biosynthesis. However, this kainoid synthase exclusively performs a stereospecific hydroxylation reaction on its cis substrate regioisomer. Mechanistic and kinetic analyses with native and alternative substrates identified a 20-fold rate increase in DabC radical cyclization over {beta}-hydroxylation, with no observable 1,5-hydrogen atom transfer. Moreover, this dual activity was conserved among macroalgal RadC1 and KabC homologs and provided insight into substrate recognition and reactivity trends. Investigation of this substrate-dependent chemistry improves our understanding of Fe/KG enzymes and their biocatalytic application.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.549022">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.549022" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.549022">
        <p class="paperTitle">Decrypting the languages of allostery in membrane-bound K-Ras4B using four complementary in silico approaches</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.549022" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.549022" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Castelli, M.; Marchetti, F.; Serapian, S. A.; Colombo, G.</p>
        <p class="info">Score: 7.8, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.549022' target='https://doi.org/10.1101/2023.07.14.549022'> 10.1101/2023.07.14.549022</a></p>
        <p class="abstract">Proteins are evolved molecular machines whose diverse biochemical functions are (also) dynamically regulated by allostery, through which even faraway protein residues can conformationally communicate. Allostery can express itself in different ways, akin to different &#34;languages&#34;: allosteric control pathways predominating in an unperturbed protein are superseded by others as soon as a perturbation arises--e.g, mutation--that alters its function (pathologically or not). Accurately modeling these often-unintuitive phenomena could therefore help explain functional changes in a specific protein whenever they are unclear. Unbiased molecular dynamics (MD) simulations are a possibility; however, since allostery can operate at longer timescales than those accessible by MD, simulations require integration with a reliable method able to, e.g., detect regions of incipient allosteric change, or likely perturbation pathways. Several such (valid) methods exist, but are typically applied singularly, only yielding a partial (&#34;monolinguistic&#34;) picture of allostery: we argue their joint application could significantly enrich this picture. To prove this, we perform unbiased MD simulations ([~]100 {micro}s) of the widely studied, allosterically active oncotarget K-Ras4B, solvated and embedded in a phospholipid membrane, proceeding to decrypt its allostery using four showcase &#34;languages&#34;: Distance Fluctuation analysis and the Shortest Path Map capture allosteric communication hotspots at equilibrium; Anisotropic Thermal Diffusion and Dynamic Non-Equilibrium MD assess them once the GTP that oncogenically activates K-Ras4B is, respectively, either superheated or hydrolyzed. &#34;Languages&#34; provide a uniquely articulate, mutually coherent, experimentally consistent picture of allostery in K-Ras4B. At equilibrium, pathways stretch from the membrane-embedded hypervariable region all the way to the active site, touching known flexible allosteric &#34;switches&#34; and proposed pockets. Upon GTP cleavage/perturbation, known to inactivate K-Ras4B, allosteric signals most reverberate on switches and interfaces that recruit effector proteins. Our work highlights the benefits of integrating different allostery detection methods, even in conjunction with unbiased MD.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.549096">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.549096" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.549096">
        <p class="paperTitle">Direct, quantitative, and comprehensive analysis of tRNA acylation using intact tRNA liquid-chromatography mass-spectrometry</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.549096" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.549096" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fricke, R.; Knudson, I.; Schepartz, A.</p>
        <p class="info">Score: 5.2, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.549096' target='https://doi.org/10.1101/2023.07.14.549096'> 10.1101/2023.07.14.549096</a></p>
        <p class="abstract">Aminoacyl-tRNA synthetases (aaRSs) provide the functional and essential link between the sequence of an mRNA and the protein it encodes. aaRS enzymes catalyze a two-step chemical reaction that acylates specific tRNAs with a cognate -amino acid. In addition to their role in translation, acylated tRNAs contribute to non-ribosomal natural product biosynthesis and are implicated in multiple human diseases. From the standpoint of synthetic biology, the acylation of tRNAs with a non-canonical -amino acid (ncAA) or more recently, a non--amino acid monomer (nAA) is a critical first step in the incorporation of these monomers into proteins, where they can be used for fundamental and applied science. These endeavors all demand an understanding of aaRS activity and specificity. Although a number of methods to monitor aaRS function in vitro or in vivo have been developed, many evaluate only the first step of the two-step reaction, require the use of radioactivity, or are slow, difficult to generalize, or both. Here we describe an LC-MS assay that rapidly, quantitatively, and directly monitors aaRS activity by detecting the intact acyl-tRNA product. After a simple tRNA acylation reaction workup, acyl- and non-acyl-tRNA molecules are resolved using ion-pairing reverse phase chromatography and their exact masses are determined using high-resolution time-of-flight mass spectrometry. The intact tRNA assay we describe is fast, simple, and quantifies reaction yields as low as 0.23%. The assay can also be employed on tRNAs acylated with flexizyme to detect products that are undetectable using standard techniques. The protocol requires basic expertise in molecular biology, mass spectrometry, and RNAse-free techniques.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.549041">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.549041" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.549041">
        <p class="paperTitle">Nanobody repertoire generated against the spike protein of ancestral SARS-CoV-2 remains efficacious against the rapidly evolving virus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.549041" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.549041" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ketaren, N. E.; Mast, F. D.; Fridy, P. C.; Olivier, J. P.; Sanyal, T.; Sali, A.; Chait, B. T.; Rout, M. P.; Aitchison, J.</p>
        <p class="info">Score: 5.2, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.549041' target='https://doi.org/10.1101/2023.07.14.549041'> 10.1101/2023.07.14.549041</a></p>
        <p class="abstract">To date, all major modes of monoclonal antibody therapy targeting SARS-CoV-2 have lost significant efficacy against the latest circulating variants. As SARS-CoV-2 omicron sublineages account for over 90% of COVID-19 infections, evasion of immune responses generated by vaccination or exposure to previous variants poses a significant challenge. A compelling new therapeutic strategy against SARS-CoV-2 is that of single domain antibodies, termed nanobodies, which address certain limitations of monoclonal antibodies. Here we demonstrate that our high-affinity nanobody repertoire, generated against wild-type SARS-CoV-2 spike protein (Mast, Fridy et al. 2021), remains effective against variants of concern, including omicron BA.4/BA.5; a subset is predicted to counter resistance in emerging XBB and BQ.1.1 sublineages. Furthermore, we reveal the synergistic potential of nanobody cocktails in neutralizing emerging variants. Our study highlights the power of nanobody technology as a versatile therapeutic and diagnostic tool to combat rapidly evolving infectious diseases such as SARS-CoV-2.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2021.12.08.471801">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2021.12.08.471801" aria-expanded="false" aria-controls="flush-collapse10.1101/2021.12.08.471801">
        <p class="paperTitle">Dual engagement of the nucleosomal acidic patches is essential for deposition of histone H2A.Z by SWR1C</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2021.12.08.471801" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2021.12.08.471801" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Baier, A. S.; Gioacchini, N.; Eek, P.; Tan, S.; Peterson, C. L.</p>
        <p class="info">Score: 13.1, Published: 2023-07-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2021.12.08.471801' target='https://doi.org/10.1101/2021.12.08.471801'> 10.1101/2021.12.08.471801</a></p>
        <p class="abstract">The SWR1C chromatin remodeling enzyme catalyzes the ATP-dependent exchange of nucleosomal histone H2A for the histone variant H2A.Z, a key variant involved in a multitude of nuclear functions. How the 14-subunit SWR1C engages the nucleosomal substrate remains largely unknown. Numerous studies on the ISWI, CHD1, and SWI/SNF families of chromatin remodeling enzymes have demonstrated key roles for the nucleosomal acidic patch for remodeling activity, however a role for this nucleosomal epitope in nucleosome editing by SWR1C has not been tested. Here, we employ a variety of biochemical assays to demonstrate an essential role for the acidic patch in the H2A.Z exchange reaction. Utilizing asymmetrically assembled nucleosomes, we demonstrate that the acidic patches on each face of the nucleosome are required for SWR1C-mediated dimer exchange, suggesting SWR1C engages the nucleosome in a &#34;pincer-like&#34; conformation, engaging both patches simultaneously. Loss of a single acidic patch results in loss of high affinity nucleosome binding and nucleosomal stimulation of ATPase activity. We identify a conserved arginine-rich motif within the Swc5 subunit that binds the acidic patch and is key for dimer exchange activity. In addition, our cryoEM structure of a Swc5-nucleosome complex suggests that promoter proximal, histone H2B ubiquitinylation may regulate H2A.Z deposition. Together these findings provide new insights into how SWR1C engages its nucleosomal substrate to promote efficient H2A.Z deposition.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.548587">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.548587" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.548587">
        <p class="paperTitle">The AAA&#43; protein Msp1 selects substrates by recognizing a hydrophobic mismatch between the substrate transmembrane domain and the lipid bilayer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.548587" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.548587" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fresenius, H. L.; Gaur, D.; Wohlever, M. L.</p>
        <p class="info">Score: 2.8, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.548587' target='https://doi.org/10.1101/2023.07.11.548587'> 10.1101/2023.07.11.548587</a></p>
        <p class="abstract">An essential aspect of protein quality control is enzymatic removal of membrane proteins from the lipid bilayer. Failures in this essential cellular process are associated with neurodegenerative diseases and cancer. Msp1 is a AAA&#43; (ATPases Associated with diverse cellular Activities) protein that removes mistargeted proteins from the outer mitochondrial membrane (OMM). How Msp1 selectively recognizes and extracts substrates within the complex OMM ecosystem, and the role of the lipid bilayer on these processes is unknown. Here, we describe the development of fully defined, rapid, and quantitative extraction assay that retains physiological substrate selectivity. Using this new assay, we systematically modified both substrates and the lipid environment to demonstrate that Msp1 recognizes substrates by a hydrophobic mismatch between the substrate TMD and the lipid bilayer. We further demonstrate that the rate limiting step in Msp1 activity is extraction of the TMD from the lipid bilayer. Together, these results provide foundational insights into how the lipid bilayer influences AAA&#43; mediated membrane protein extraction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548257">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548257" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548257">
        <p class="paperTitle">The pathogenic T42A mutation in SHP2 rewires interaction specificity and enhances signaling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548257" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548257" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: van Vlimmeren, A. E.; Voleti, R.; Chartier, C. A.; Jiang, Z.; Karandur, D.; Shah, N. H.</p>
        <p class="info">Score: 6.3, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548257' target='https://doi.org/10.1101/2023.07.10.548257'> 10.1101/2023.07.10.548257</a></p>
        <p class="abstract">Mutations in the tyrosine phosphatase SHP2 are associated with various human diseases. Most of these mutations increase the basal catalytic activity of SHP2 by disrupting auto-inhibitory interactions between its phosphatase domain and N-terminal SH2 (phosphotyrosine recognition) domain. By contrast, several other disease-associated mutations located in the ligand-binding pockets of the N- or C-terminal SH2 domains do not increase basal activity and likely exert their pathogenicity through alternative mechanisms. We lack a molecular understanding of how these SH2 mutations impact SHP2 structure, activity, and signaling. Here, we characterize five SHP2 SH2 domain ligand-binding pocket mutants through a combination of high-throughput biochemical screens, biophysical and biochemical measurements, molecular dynamics simulations, and cellular assays. We show that, while several of these mutants alter binding affinity to phosphorylation sites, the T42A mutation in the N-SH2 domain is unique in that it also alters ligand-binding specificity. The functional consequence of this altered specificity is that the T42A mutant has biased sensitivity toward a subset of activating ligands. Our study highlights an example of a nuanced mechanism of action for a disease-associated mutation, characterized by a change in protein-protein interaction specificity that alters enzyme activation.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
