<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Intrinsically Disordered Regions Promote Protein Refoldability and Facilitate Retrieval from Biomolecular Condensates
Authors: To, P.; Bhagwat, A. M.; Tarbox, H. E.; Ecer, A.; Wendorff, H.; Jamieson, Z.; Trcek, T.; Fried, S. D.
Score: 129.8, Published: 2023-06-25 DOI: 10.1101/2023.06.25.546465
Many eukaryotic proteins contain intrinsically disordered regions (IDRs) that intersperse globular folded domains, in contrast with bacterial proteins which are typically highly globular1, 2. Recent years have seen great progress in identifying biological functions associated with these elusive protein sequence: in specific cases, they mediate liquid liquid phase separation3, perform molecular recognition4, or act as sensors to changes in the environment5.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Intrinsically Disordered Regions Promote Protein Refoldability and Facilitate Retrieval from Biomolecular Condensates
Authors: To, P.; Bhagwat, A. M.; Tarbox, H. E.; Ecer, A.; Wendorff, H.; Jamieson, Z.; Trcek, T.; Fried, S. D.
Score: 129.8, Published: 2023-06-25 DOI: 10.1101/2023.06.25.546465
Many eukaryotic proteins contain intrinsically disordered regions (IDRs) that intersperse globular folded domains, in contrast with bacterial proteins which are typically highly globular1, 2. Recent years have seen great progress in identifying biological functions associated with these elusive protein sequence: in specific cases, they mediate liquid liquid phase separation3, perform molecular recognition4, or act as sensors to changes in the environment5." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-02T10:39:32+00:00" />
<meta property="article:modified_time" content="2023-07-02T10:39:32+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Intrinsically Disordered Regions Promote Protein Refoldability and Facilitate Retrieval from Biomolecular Condensates
Authors: To, P.; Bhagwat, A. M.; Tarbox, H. E.; Ecer, A.; Wendorff, H.; Jamieson, Z.; Trcek, T.; Fried, S. D.
Score: 129.8, Published: 2023-06-25 DOI: 10.1101/2023.06.25.546465
Many eukaryotic proteins contain intrinsically disordered regions (IDRs) that intersperse globular folded domains, in contrast with bacterial proteins which are typically highly globular1, 2. Recent years have seen great progress in identifying biological functions associated with these elusive protein sequence: in specific cases, they mediate liquid liquid phase separation3, perform molecular recognition4, or act as sensors to changes in the environment5."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Intrinsically Disordered Regions Promote Protein Refoldability and Facilitate Retrieval from Biomolecular Condensates\nAuthors: To, P.; Bhagwat, A. M.; Tarbox, H. E.; Ecer, A.; Wendorff, H.; Jamieson, Z.; Trcek, T.; Fried, S. D.\nScore: 129.8, Published: 2023-06-25 DOI: 10.1101/2023.06.25.546465\nMany eukaryotic proteins contain intrinsically disordered regions (IDRs) that intersperse globular folded domains, in contrast with bacterial proteins which are typically highly globular1, 2. Recent years have seen great progress in identifying biological functions associated with these elusive protein sequence: in specific cases, they mediate liquid liquid phase separation3, perform molecular recognition4, or act as sensors to changes in the environment5.",
  "keywords": [
    
  ],
  "articleBody": " Intrinsically Disordered Regions Promote Protein Refoldability and Facilitate Retrieval from Biomolecular Condensates\nAuthors: To, P.; Bhagwat, A. M.; Tarbox, H. E.; Ecer, A.; Wendorff, H.; Jamieson, Z.; Trcek, T.; Fried, S. D.\nScore: 129.8, Published: 2023-06-25 DOI: 10.1101/2023.06.25.546465\nMany eukaryotic proteins contain intrinsically disordered regions (IDRs) that intersperse globular folded domains, in contrast with bacterial proteins which are typically highly globular1, 2. Recent years have seen great progress in identifying biological functions associated with these elusive protein sequence: in specific cases, they mediate liquid liquid phase separation3, perform molecular recognition4, or act as sensors to changes in the environment5. Nevertheless, only a small number of IDRs have annotated functions6 despite their presence in 64% of yeast proteins,7 stimulating some to question what general purpose they may serve8, 9. Here, by interrogating the refoldability of two fungal proteomes (Saccharomyces cerevisiae and Neurosporra crassa), we show that IDRs render their host proteins more refoldable from the denatured state, allowing them to cohere more closely to Anfinsens thermodynamic hypothesis10, 11. The data provide an exceptionally clear picture of which biophysical and topological characteristics enable refoldability. Moreover, we find that almost all yeast proteins that partition into stress granules during heat shock are refoldable, a finding that holds for other condensates such as P-bodies and the nucleolus. Finally, we find that the Hsp104 unfoldase12 is the principal actor in mediating disassembly of heat stress granules and that the efficiency with which condensed proteins are returned to the soluble phase is also well explained by refoldability. Hence, these studies establish spontaneous refoldability as an adaptive trait that endows proteins with the capacity to reform their native soluble structures following their extraction from condensates. Altogether, our results provide an intuitive model for the function of IDRs in many multidomain proteins and clarifies their relationship to the phenomenon of biomolecular condensation.\nEvaluating the performance of the Astral mass analyzer for quantitative proteomics using data independent acquisition\nAuthors: Heil, L. R.; Damoc, N. E.; Arrey, T. N.; Pashkova, A.; Denisov, E.; Petzoldt, J.; Peterson, A.; Hsu, C.; Searle, B. C.; Shulman, N.; Riffle, M.; Connolly, B.; MacLean, B. X.; Remes, P. M.; Senko, M.; Stewart, H.; Hock, C.; Makarov, A.; Hermanson, D.; Zabrouskov, V.; Wu, C. C.; MacCoss, M. J.\nScore: 54.8, Published: 2023-06-03 DOI: 10.1101/2023.06.03.543570\nWe evaluate the quantitative performance of the newly released Asymmetric Track Lossless (Astral) analyzer. Using data independent acquisition, the Thermo Scientific Orbitrap Astral mass spectrometer quantifies 5 times more peptides per unit time than state-of-the-art Thermo Scientific Orbitrap mass spectrometers, which have long been the gold standard for high resolution quantitative proteomics. Our results demonstrate that the Orbitrap Astral mass spectrometer can produce high quality quantitative measurements across a wide dynamic range. We also use a newly developed extra-cellular vesicle enrichment protocol to reach new depths of coverage in the plasma proteome, quantifying over 5,000 plasma proteins in a 60-minute gradient with the Orbitrap Astral mass spectrometer. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=111 SRC=\"FIGDIR/small/543570v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (28K): org.highwire.dtl.DTLVardef@128156dorg.highwire.dtl.DTLVardef@96c051org.highwire.dtl.DTLVardef@12c25f1org.highwire.dtl.DTLVardef@16eff26_HPS_FORMAT_FIGEXP M_FIG C_FIG\nTargeting the conserved active site of splicing machines with specific and selective small molecule modulators\nAuthors: Silvestri, I.; Manigrasso, J.; Andreani, A.; Brindani, N.; De Vivo, M.; Marcia, M.\nScore: 11.6, Published: 2023-06-22 DOI: 10.1101/2023.06.21.545906\nThe self-splicing group II introns are bacterial and organellar ancestors of the nuclear spliceosome and retro-transposable elements of pharmacological and biotechnological importance. Integrating enzymatic, crystallographic, and simulation studies, we demonstrate how these introns recognize small molecules through their conserved active site. These RNA-binding small molecules selectively inhibit the two steps of splicing by adopting distinctive poses at different stages of catalysis, and by preventing crucial active site conformational changes that are essential for splicing progression. Our data exemplify the enormous power of RNA binders to mechanistically probe vital cellular pathways. Most importantly, by proving that the evolutionarily-conserved RNA core of splicing machines can recognize small molecules specifically, our work puts solid bases for the rational design of splicing modulators not only against bacterial and organellar introns, but also against the human spliceosome, which is a validated drug target for the treatment of congenital diseases and cancers.\nAn engineered biosensor enables dynamic aspartate measurements in living cells\nAuthors: Davidsen, K.; Marvin, J. S.; Aggarwal, A.; Brown, T. A.; Sullivan, L. B.\nScore: 10.0, Published: 2023-06-27 DOI: 10.1101/2023.06.27.546775\nIntracellular levels of the amino acid aspartate are responsive to changes in metabolism in mammalian cells and can correspondingly alter cell function, highlighting the need for robust tools to measure aspartate abundance. However, comprehensive understanding of aspartate metabolism has been limited by the throughput, cost, and static nature of the mass spectrometry based measurements that are typically employed to measure aspartate levels. To address these issues, we have developed a GFP-based sensor of aspartate (jAspSnFR3), where the fluorescence intensity corresponds to aspartate concentration. As a purified protein, the sensor has a 20-fold increase in fluorescence upon aspartate saturation, with dose dependent fluorescence changes covering a physiologically relevant aspartate concentration range and no significant off target binding. Expressed in mammalian cell lines, sensor intensity correlated with aspartate levels measured by mass spectrometry and could resolve temporal changes in intracellular aspartate from genetic, pharmacological, and nutritional manipulations. These data demonstrate the utility of jAspSnFR3 and highlight the opportunities it provides for temporally resolved and high throughput applications of variables that affect aspartate levels.\nFluorogenesis: Inducing Fluorescence in a Non-Fluorescent Protein Through Photoinduced Chromophore Transfer of a Genetically Encoded Chromophore\nAuthors: Kumar, Y.; Singh, R. K.; Ojha, M.; Pushpavanam, K.\nScore: 9.9, Published: 2023-06-25 DOI: 10.1101/2023.06.24.546416\nFluorescent proteins, while essential for bioimaging, are limited to visualizing cellular localization without offering additional functionality. We report for the first time a strategy to expand the chemical, structural, and functional diversity of fluorescent proteins by harnessing light to induce red fluorescence in a previously non-fluorescent protein. We accomplish this by inducing the transfer of the genetically encoded chromophore from a photocleavable protein (PhoCl1) to a non-fluorescent kinase (MjRibK) inducing red fluorescence in the latter. We have employed analytical and spectroscopic techniques to validate the presence of red fluorescence in MjRibK. Furthermore, molecular dynamics simulations were carried out to investigate the amino acid residues of MjRibK involved in the generation of red fluorescence. Finally, we demonstrate the ability of the red fluorescent MjRibK to operate as a cyclable high-temperature sensor. We anticipate that this light-induced chromophore transfer strategy will open new possibilities for developing multifunctional genetically encoded fluorescent sensors.\nType III-B CRISPR-Cas signaling-based cascade of proteolytic cleavages\nAuthors: Steens, J.; Bravo, J. P.; Salazar, C. R. P.; Yildiz, C.; Amieiro, A. M.; Kostlbacher, S.; Prinsen, S. H. P.; Patinios, C.; Bardis, A.; Barendrecht, A.; Scheltema, R. A.; Ettema, T. J. G.; van der Oost, J.; Taylor, D. W.; Staals, R.\nScore: 10.2, Published: 2023-06-23 DOI: 10.1101/2023.06.23.546230\nType III CRISPR-Cas systems provide a sequence-specific adaptive immune response that protects prokaryotic hosts against viruses and other foreign genetic invaders. These crRNA-guided Cas effector complexes bind and cleave complementary RNA targets. Specific target binding stimulates the Cas10 subunit to generate cyclic oligoadenylate (cOA) signaling molecules, that in turn allosterically activate proteins carrying cognate sensory domains: CARF or SAVED. Here, we characterize an elaborate set of genes associated with the type III-B CRISPR-Cas system from Haliangium ochraceum, which includes a signal transduction module of a CBASS defense system with two caspase-like proteases, SAVED-CHAT and PCaspase (Prokaryotic Caspase). We show that binding of a 3-nucleotide cOA (cA3) to the SAVED domain of SAVED-CHAT induces its oligomerization into long filaments that activate the proteolytic activity of the CHAT domain. Surprisingly, we find that activated SAVED-CHAT specifically cleaves and activates the second protease, PCaspase. In turn, activated PCaspase cleaves a multitude of other proteins, including a putative sigma factor and a PCaspase-inhibitor. We expressed the type III-B system and its associated genes in E. coli and observed a strong abortive phenotype when offering a complementary target RNA, but only in the presence of both SAVED-CHAT and PCaspase. Together, our findings show an intriguing cascade of proteolytic activities (conceptually similar to eukaryotic caspases) in this bacterial immune system that reveals yet another strategy to effectively defend against mobile genetic elements.\n4-Thiaproline accelerates the slow folding phase of proteins containing cis prolines in the native state by two orders of magnitude\nAuthors: O' Loughlin, J.; Zinovjev, K.; Napolitano, S.; Van der Kamp, M.; Rubini, M.\nScore: 5.3, Published: 2023-06-23 DOI: 10.1101/2023.06.23.546227\nThe cis/trans isomerization of peptidyl-prolyl peptide bonds is often the bottleneck of the refolding reaction for proteins containing cis proline residues in the native state. Proline (Pro) analogues, especially C4-substituted fluoroprolines, have been widely used in protein engineering to enhance the thermodynamic stability of peptides and proteins and to investigate folding kinetics. 4-thiaproline (Thp) has been shown to bias the ring pucker of Pro, to increase the cis population percentage of model peptides in comparison to Pro, and to diminish the activation energy barrier for the cis/trans isomerization reaction. Despite its intriguing properties, Thp has been seldom incorporated into proteins. Moreover, the impact of Thp on the folding kinetics of globular proteins has never been reported. In this study, we show that upon incorporation of Thp at cisPro76 into the thioredoxin variant Trx1P the half-life of the refolding reaction decreased from [~]2 hours to [~]35 seconds. A dramatic acceleration of the refolding rate could be observed also for the protein pseudo wild-type barstar upon replacement of cisPro48 with Thp. Quantum chemical calculations revealed that the replacement of the C{gamma}H2 group by a sulfur atom in the pyrrolidine ring, lowers the barrier for cis/trans rotation due to a weakened peptide bond. The protein variants retained their thermodynamic stability upon incorporation of Thp, while the catalytic and enzymatic activities of the modified Trx1P remained unchanged. Our results show that the Pro isostere Thp might eliminate the bottleneck of the refolding reaction of proteins containing cis proline residues in the native state, independent from the local structural environment.\nBioorthogonal caging-group-free photoactivatable probes for minimal-linkage-error nanoscopy\nAuthors: Aktalay, A.; Lincoln, R.; Heynck, L.; Lima, M. A. D. R. B. F.; Butkevich, A. N.; Bossi, M. L.; Hell, S. W.\nScore: 5.5, Published: 2023-06-22 DOI: 10.1101/2023.06.21.545866\nHere we describe highly compact, click compatible, and photoactivatable dyes for super-resolution fluorescence microscopy (nanoscopy). By combining the photoactivatable xanthone (PaX) core with a tetrazine group, we achieve minimally sized and highly sensitive molecular dyads for selective labeling of unnatural amino acids introduced by genetic code expansion. We exploit the excited state quenching properties of the tetrazine group to attenuate the photoactivation rates of the PaX, and further reduce the overall fluorescence emission of the photogenerated fluorophore, providing two mechanisms of selectivity to reduce off-target signal. Coupled with MINFLUX nanoscopy, we demonstrate our dyads in the minimal-linkage-error imaging of vimentin filaments, demonstrating the molecular scale precision in fluorophore positioning.\nDecoding the comprehensive substrate-specificity and evidence of altered site-specific collagen prolyl-3-hydroxylation, lysyl-hydroxylation, and lysyl O-glycosylation in P4ha1 and P4ha2 deleted mutant mice\nAuthors: Sarohi, V.; Basak, T.\nScore: 4.1, Published: 2023-06-29 DOI: 10.1101/2023.06.28.546985\nCollagens, the most abundant proteins in mammals, play pivotal roles in the maintenance of tissue structure, functions, cell-to-cell communication, cellular migration, behavior, and growth. Collagens are highly complex in structure due to the dynamic post-translational modifications (PTMs) such as hydroxylations (on prolines and lysine residues) and O-glycosylation (on hydroxylysines) enzymatically catalyzed during biosynthesis. The most prevalent modification in fibrillar collagens is prolyl 4-hydroxylation catalyzed by collagen prolyl 4-hydroxylases (C-P4hs). Prolyl 4-hydroxylation on collagens plays a critical role in collagen biosynthesis, thermostability, and cell-collagen interactions. However, the site-specificity of prolyl 4-hydroxylase 1 (P4ha1) and P4ha2 is not comprehensively studied yet. Further, the effect of P4ha1 and P4ha2 on the plethora of other site-specific collagen PTMs is not known to date. In-depth mass-spectrometry data (PXD008802) analysis of mice skin collagen I extracted from wild-type and different deletion mutants of C-P4hs revealed that partial or full deletion of prolyl 4-hydroxylases (P4ha1 and P4ha2) significantly decreases collagen deposition in ECM hinting towards perturbed biosynthesis. A total of 421 site-specific PTMs on fibrillar collagen chains (Col1a1, Col1a2, and Col3a1) were identified. Further, novel 23 P4ha1 specific, 8 P4ha2 specific, and 18 C-P4hs promiscuous sites on fibrillar collagen chains were identified. Partial deletion of P4ha1 and full deletion of P4ha2 also resulted in altered levels of the site-specific prolyl-3-hydroxylation occupancy in collagen I. Surprisingly, an increased level of site-specific lysyl hydroxylation (Col1a1-K731, Col1a2-K183,315) was documented upon partial deletion of P4ha1 and full deletion of P4ha2. Our findings showcased that the activity of prolyl 4-hydroxylases is not limited to 4-hydroxylation of specific proline sites, but simultaneously can perturb the entire biosynthetic network by modulating prolyl 3-hydroxylation and lysyl hydroxylation occupancy levels in the fibrillar collagen chains in a site-specific manner.\nDecreased efficacy of a COVID-19 vaccine due to mutations present in early SARS-CoV-2 variants of concern\nAuthors: Weidenbacher, P. A.; Friedland, N.; Sanyal, M.; Morris, M. K.; Do, J.; Hanson, C.; Kim, P. S.\nScore: 4.1, Published: 2023-06-29 DOI: 10.1101/2023.06.27.546764\nWith the SARS-CoV-2 virus still circulating and evolving, there remains an outstanding question if variant-specific vaccines represent the optimal path forward, or if other strategies might be more efficacious towards providing broad protection against emerging variants. Here, we examine the efficacy of strain-specific variants of our previously reported, pan-sarbecovirus vaccine candidate, DCFHP-alum, a ferritin nanoparticle functionalized with an engineered form of the SARS-CoV-2 spike protein. In non-human primates, DCFHP-alum elicits neutralizing antibodies against all known VOCs that have emerged to date and SARS-CoV-1. During development of the DCFHP antigen, we investigated the incorporation of strain-specific mutations from the major VOCs that had emerged to date: D614G, Epsilon, Alpha, Beta, and Gamma. Here, we report the biochemical and immunological characterizations that led us to choose the ancestral Wuhan-1 sequence as the basis for the final DCFHP antigen design. Specifically, we show by size exclusion chromatography and differential scanning fluorimetry that mutations in the VOCs adversely alter the antigen's structure and stability. More importantly, we determined that DCFHP without strain-specific mutations elicits the most robust, cross-reactive response in both pseudovirus and live virus neutralization assays. Our data suggest potential limitations to the variant-chasing approach in the development of protein nanoparticle vaccines, but also have implications for other approaches including mRNA-based vaccines.\n",
  "wordCount" : "2350",
  "inLanguage": "en",
  "datePublished": "2023-07-02T10:39:32Z",
  "dateModified": "2023-07-02T10:39:32Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta"><span>updated on July 2, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.25.546465">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.25.546465" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.25.546465">
        <p class="paperTitle">Intrinsically Disordered Regions Promote Protein Refoldability and Facilitate Retrieval from Biomolecular Condensates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.25.546465" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.25.546465" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: To, P.; Bhagwat, A. M.; Tarbox, H. E.; Ecer, A.; Wendorff, H.; Jamieson, Z.; Trcek, T.; Fried, S. D.</p>
        <p class="info">Score: 129.8, Published: 2023-06-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.25.546465' target='https://doi.org/10.1101/2023.06.25.546465'> 10.1101/2023.06.25.546465</a></p>
        <p class="abstract">Many eukaryotic proteins contain intrinsically disordered regions (IDRs) that intersperse globular folded domains, in contrast with bacterial proteins which are typically highly globular1, 2. Recent years have seen great progress in identifying biological functions associated with these elusive protein sequence: in specific cases, they mediate liquid liquid phase separation3, perform molecular recognition4, or act as sensors to changes in the environment5. Nevertheless, only a small number of IDRs have annotated functions6 despite their presence in 64% of yeast proteins,7 stimulating some to question what  general purpose they may serve8, 9. Here, by interrogating the refoldability of two fungal proteomes (Saccharomyces cerevisiae and Neurosporra crassa), we show that IDRs render their host proteins more refoldable from the denatured state, allowing them to cohere more closely to Anfinsens thermodynamic hypothesis10, 11. The data provide an exceptionally clear picture of which biophysical and topological characteristics enable refoldability. Moreover, we find that almost all yeast proteins that partition into stress granules during heat shock are refoldable, a finding that holds for other condensates such as P-bodies and the nucleolus. Finally, we find that the Hsp104 unfoldase12 is the principal actor in mediating disassembly of heat stress granules and that the efficiency with which condensed proteins are returned to the soluble phase is also well explained by refoldability. Hence, these studies establish spontaneous refoldability as an adaptive trait that endows proteins with the capacity to reform their native soluble structures following their extraction from condensates. Altogether, our results provide an intuitive model for the function of IDRs in many multidomain proteins and clarifies their relationship to the phenomenon of biomolecular condensation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.03.543570">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.03.543570" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.03.543570">
        <p class="paperTitle">Evaluating the performance of the Astral mass analyzer for quantitative proteomics using data independent acquisition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.03.543570" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.03.543570" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Heil, L. R.; Damoc, N. E.; Arrey, T. N.; Pashkova, A.; Denisov, E.; Petzoldt, J.; Peterson, A.; Hsu, C.; Searle, B. C.; Shulman, N.; Riffle, M.; Connolly, B.; MacLean, B. X.; Remes, P. M.; Senko, M.; Stewart, H.; Hock, C.; Makarov, A.; Hermanson, D.; Zabrouskov, V.; Wu, C. C.; MacCoss, M. J.</p>
        <p class="info">Score: 54.8, Published: 2023-06-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.03.543570' target='https://doi.org/10.1101/2023.06.03.543570'> 10.1101/2023.06.03.543570</a></p>
        <p class="abstract">We evaluate the quantitative performance of the newly released Asymmetric Track Lossless (Astral) analyzer. Using data independent acquisition, the Thermo Scientific Orbitrap Astral mass spectrometer quantifies 5 times more peptides per unit time than state-of-the-art Thermo Scientific Orbitrap mass spectrometers, which have long been the gold standard for high resolution quantitative proteomics. Our results demonstrate that the Orbitrap Astral mass spectrometer can produce high quality quantitative measurements across a wide dynamic range. We also use a newly developed extra-cellular vesicle enrichment protocol to reach new depths of coverage in the plasma proteome, quantifying over 5,000 plasma proteins in a 60-minute gradient with the Orbitrap Astral mass spectrometer.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=111 SRC=&#34;FIGDIR/small/543570v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (28K):
org.highwire.dtl.DTLVardef@128156dorg.highwire.dtl.DTLVardef@96c051org.highwire.dtl.DTLVardef@12c25f1org.highwire.dtl.DTLVardef@16eff26_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545906">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545906" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545906">
        <p class="paperTitle">Targeting the conserved active site of splicing machines with specific and selective small molecule modulators</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545906" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545906" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Silvestri, I.; Manigrasso, J.; Andreani, A.; Brindani, N.; De Vivo, M.; Marcia, M.</p>
        <p class="info">Score: 11.6, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545906' target='https://doi.org/10.1101/2023.06.21.545906'> 10.1101/2023.06.21.545906</a></p>
        <p class="abstract">The self-splicing group II introns are bacterial and organellar ancestors of the nuclear spliceosome and retro-transposable elements of pharmacological and biotechnological importance. Integrating enzymatic, crystallographic, and simulation studies, we demonstrate how these introns recognize small molecules through their conserved active site. These RNA-binding small molecules selectively inhibit the two steps of splicing by adopting distinctive poses at different stages of catalysis, and by preventing crucial active site conformational changes that are essential for splicing progression. Our data exemplify the enormous power of RNA binders to mechanistically probe vital cellular pathways. Most importantly, by proving that the evolutionarily-conserved RNA core of splicing machines can recognize small molecules specifically, our work puts solid bases for the rational design of splicing modulators not only against bacterial and organellar introns, but also against the human spliceosome, which is a validated drug target for the treatment of congenital diseases and cancers.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546775">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546775" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546775">
        <p class="paperTitle">An engineered biosensor enables dynamic aspartate measurements in living cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546775" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546775" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Davidsen, K.; Marvin, J. S.; Aggarwal, A.; Brown, T. A.; Sullivan, L. B.</p>
        <p class="info">Score: 10.0, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546775' target='https://doi.org/10.1101/2023.06.27.546775'> 10.1101/2023.06.27.546775</a></p>
        <p class="abstract">Intracellular levels of the amino acid aspartate are responsive to changes in metabolism in mammalian cells and can correspondingly alter cell function, highlighting the need for robust tools to measure aspartate abundance. However, comprehensive understanding of aspartate metabolism has been limited by the throughput, cost, and static nature of the mass spectrometry based measurements that are typically employed to measure aspartate levels. To address these issues, we have developed a GFP-based sensor of aspartate (jAspSnFR3), where the fluorescence intensity corresponds to aspartate concentration. As a purified protein, the sensor has a 20-fold increase in fluorescence upon aspartate saturation, with dose dependent fluorescence changes covering a physiologically relevant aspartate concentration range and no significant off target binding. Expressed in mammalian cell lines, sensor intensity correlated with aspartate levels measured by mass spectrometry and could resolve temporal changes in intracellular aspartate from genetic, pharmacological, and nutritional manipulations. These data demonstrate the utility of jAspSnFR3 and highlight the opportunities it provides for temporally resolved and high throughput applications of variables that affect aspartate levels.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.24.546416">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.24.546416" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.24.546416">
        <p class="paperTitle">Fluorogenesis: Inducing Fluorescence in a Non-Fluorescent Protein Through Photoinduced Chromophore Transfer of a Genetically Encoded Chromophore</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.24.546416" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.24.546416" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kumar, Y.; Singh, R. K.; Ojha, M.; Pushpavanam, K.</p>
        <p class="info">Score: 9.9, Published: 2023-06-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.24.546416' target='https://doi.org/10.1101/2023.06.24.546416'> 10.1101/2023.06.24.546416</a></p>
        <p class="abstract">Fluorescent proteins, while essential for bioimaging, are limited to visualizing cellular localization without offering additional functionality. We report for the first time a strategy to expand the chemical, structural, and functional diversity of fluorescent proteins by harnessing light to induce red fluorescence in a previously non-fluorescent protein. We accomplish this by inducing the transfer of the genetically encoded chromophore from a photocleavable protein (PhoCl1) to a non-fluorescent kinase (MjRibK) inducing red fluorescence in the latter. We have employed analytical and spectroscopic techniques to validate the presence of red fluorescence in MjRibK. Furthermore, molecular dynamics simulations were carried out to investigate the amino acid residues of MjRibK involved in the generation of red fluorescence. Finally, we demonstrate the ability of the red fluorescent MjRibK to operate as a cyclable high-temperature sensor. We anticipate that this light-induced chromophore transfer strategy will open new possibilities for developing multifunctional genetically encoded fluorescent sensors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546230">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546230" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546230">
        <p class="paperTitle">Type III-B CRISPR-Cas signaling-based cascade of proteolytic cleavages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546230" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546230" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Steens, J.; Bravo, J. P.; Salazar, C. R. P.; Yildiz, C.; Amieiro, A. M.; Kostlbacher, S.; Prinsen, S. H. P.; Patinios, C.; Bardis, A.; Barendrecht, A.; Scheltema, R. A.; Ettema, T. J. G.; van der Oost, J.; Taylor, D. W.; Staals, R.</p>
        <p class="info">Score: 10.2, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546230' target='https://doi.org/10.1101/2023.06.23.546230'> 10.1101/2023.06.23.546230</a></p>
        <p class="abstract">Type III CRISPR-Cas systems provide a sequence-specific adaptive immune response that protects prokaryotic hosts against viruses and other foreign genetic invaders. These crRNA-guided Cas effector complexes bind and cleave complementary RNA targets. Specific target binding stimulates the Cas10 subunit to generate cyclic oligoadenylate (cOA) signaling molecules, that in turn allosterically activate proteins carrying cognate sensory domains: CARF or SAVED. Here, we characterize an elaborate set of genes associated with the type III-B CRISPR-Cas system from Haliangium ochraceum, which includes a signal transduction module of a CBASS defense system with two caspase-like proteases, SAVED-CHAT and PCaspase (Prokaryotic Caspase). We show that binding of a 3-nucleotide cOA (cA3) to the SAVED domain of SAVED-CHAT induces its oligomerization into long filaments that activate the proteolytic activity of the CHAT domain. Surprisingly, we find that activated SAVED-CHAT specifically cleaves and activates the second protease, PCaspase. In turn, activated PCaspase cleaves a multitude of other proteins, including a putative sigma factor and a PCaspase-inhibitor. We expressed the type III-B system and its associated genes in E. coli and observed a strong abortive phenotype when offering a complementary target RNA, but only in the presence of both SAVED-CHAT and PCaspase. Together, our findings show an intriguing cascade of proteolytic activities (conceptually similar to eukaryotic caspases) in this bacterial immune system that reveals yet another strategy to effectively defend against mobile genetic elements.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546227">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546227" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546227">
        <p class="paperTitle">4-Thiaproline accelerates the slow folding phase of proteins containing cis prolines in the native state by two orders of magnitude</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546227" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546227" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: O&#39; Loughlin, J.; Zinovjev, K.; Napolitano, S.; Van der Kamp, M.; Rubini, M.</p>
        <p class="info">Score: 5.3, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546227' target='https://doi.org/10.1101/2023.06.23.546227'> 10.1101/2023.06.23.546227</a></p>
        <p class="abstract">The cis/trans isomerization of peptidyl-prolyl peptide bonds is often the bottleneck of the refolding reaction for proteins containing cis proline residues in the native state. Proline (Pro) analogues, especially C4-substituted fluoroprolines, have been widely used in protein engineering to enhance the thermodynamic stability of peptides and proteins and to investigate folding kinetics. 4-thiaproline (Thp) has been shown to bias the ring pucker of Pro, to increase the cis population percentage of model peptides in comparison to Pro, and to diminish the activation energy barrier for the cis/trans isomerization reaction. Despite its intriguing properties, Thp has been seldom incorporated into proteins. Moreover, the impact of Thp on the folding kinetics of globular proteins has never been reported. In this study, we show that upon incorporation of Thp at cisPro76 into the thioredoxin variant Trx1P the half-life of the refolding reaction decreased from [~]2 hours to [~]35 seconds. A dramatic acceleration of the refolding rate could be observed also for the protein pseudo wild-type barstar upon replacement of cisPro48 with Thp. Quantum chemical calculations revealed that the replacement of the C{gamma}H2 group by a sulfur atom in the pyrrolidine ring, lowers the barrier for cis/trans rotation due to a weakened peptide bond. The protein variants retained their thermodynamic stability upon incorporation of Thp, while the catalytic and enzymatic activities of the modified Trx1P remained unchanged. Our results show that the Pro isostere Thp might eliminate the bottleneck of the refolding reaction of proteins containing cis proline residues in the native state, independent from the local structural environment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545866">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545866" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545866">
        <p class="paperTitle">Bioorthogonal caging-group-free photoactivatable probes for minimal-linkage-error nanoscopy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545866" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545866" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aktalay, A.; Lincoln, R.; Heynck, L.; Lima, M. A. D. R. B. F.; Butkevich, A. N.; Bossi, M. L.; Hell, S. W.</p>
        <p class="info">Score: 5.5, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545866' target='https://doi.org/10.1101/2023.06.21.545866'> 10.1101/2023.06.21.545866</a></p>
        <p class="abstract">Here we describe highly compact, click compatible, and photoactivatable dyes for super-resolution fluorescence microscopy (nanoscopy). By combining the photoactivatable xanthone (PaX) core with a tetrazine group, we achieve minimally sized and highly sensitive molecular dyads for selective labeling of unnatural amino acids introduced by genetic code expansion. We exploit the excited state quenching properties of the tetrazine group to attenuate the photoactivation rates of the PaX, and further reduce the overall fluorescence emission of the photogenerated fluorophore, providing two mechanisms of selectivity to reduce off-target signal. Coupled with MINFLUX nanoscopy, we demonstrate our dyads in the minimal-linkage-error imaging of vimentin filaments, demonstrating the molecular scale precision in fluorophore positioning.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546985">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546985" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546985">
        <p class="paperTitle">Decoding the comprehensive substrate-specificity and evidence of altered site-specific collagen prolyl-3-hydroxylation, lysyl-hydroxylation, and lysyl O-glycosylation in P4ha1 and P4ha2 deleted mutant mice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546985" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546985" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sarohi, V.; Basak, T.</p>
        <p class="info">Score: 4.1, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546985' target='https://doi.org/10.1101/2023.06.28.546985'> 10.1101/2023.06.28.546985</a></p>
        <p class="abstract">Collagens, the most abundant proteins in mammals, play pivotal roles in the maintenance of tissue structure, functions, cell-to-cell communication, cellular migration, behavior, and growth. Collagens are highly complex in structure due to the dynamic post-translational modifications (PTMs) such as hydroxylations (on prolines and lysine residues) and O-glycosylation (on hydroxylysines) enzymatically catalyzed during biosynthesis. The most prevalent modification in fibrillar collagens is prolyl 4-hydroxylation catalyzed by collagen prolyl 4-hydroxylases (C-P4hs). Prolyl 4-hydroxylation on collagens plays a critical role in collagen biosynthesis, thermostability, and cell-collagen interactions. However, the site-specificity of prolyl 4-hydroxylase 1 (P4ha1) and P4ha2 is not comprehensively studied yet. Further, the effect of P4ha1 and P4ha2 on the plethora of other site-specific collagen PTMs is not known to date. In-depth mass-spectrometry data (PXD008802) analysis of mice skin collagen I extracted from wild-type and different deletion mutants of C-P4hs revealed that partial or full deletion of prolyl 4-hydroxylases (P4ha1 and P4ha2) significantly decreases collagen deposition in ECM hinting towards perturbed biosynthesis. A total of 421 site-specific PTMs on fibrillar collagen chains (Col1a1, Col1a2, and Col3a1) were identified. Further, novel 23 P4ha1 specific, 8 P4ha2 specific, and 18 C-P4hs promiscuous sites on fibrillar collagen chains were identified. Partial deletion of P4ha1 and full deletion of P4ha2 also resulted in altered levels of the site-specific prolyl-3-hydroxylation occupancy in collagen I. Surprisingly, an increased level of site-specific lysyl hydroxylation (Col1a1-K731, Col1a2-K183,315) was documented upon partial deletion of P4ha1 and full deletion of P4ha2. Our findings showcased that the activity of prolyl 4-hydroxylases is not limited to 4-hydroxylation of specific proline sites, but simultaneously can perturb the entire biosynthetic network by modulating prolyl 3-hydroxylation and lysyl hydroxylation occupancy levels in the fibrillar collagen chains in a site-specific manner.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546764">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546764" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546764">
        <p class="paperTitle">Decreased efficacy of a COVID-19 vaccine due to mutations present in early SARS-CoV-2 variants of concern</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546764" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546764" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Weidenbacher, P. A.; Friedland, N.; Sanyal, M.; Morris, M. K.; Do, J.; Hanson, C.; Kim, P. S.</p>
        <p class="info">Score: 4.1, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546764' target='https://doi.org/10.1101/2023.06.27.546764'> 10.1101/2023.06.27.546764</a></p>
        <p class="abstract">With the SARS-CoV-2 virus still circulating and evolving, there remains an outstanding question if variant-specific vaccines represent the optimal path forward, or if other strategies might be more efficacious towards providing broad protection against emerging variants. Here, we examine the efficacy of strain-specific variants of our previously reported, pan-sarbecovirus vaccine candidate, DCFHP-alum, a ferritin nanoparticle functionalized with an engineered form of the SARS-CoV-2 spike protein. In non-human primates, DCFHP-alum elicits neutralizing antibodies against all known VOCs that have emerged to date and SARS-CoV-1. During development of the DCFHP antigen, we investigated the incorporation of strain-specific mutations from the major VOCs that had emerged to date: D614G, Epsilon, Alpha, Beta, and Gamma. Here, we report the biochemical and immunological characterizations that led us to choose the ancestral Wuhan-1 sequence as the basis for the final DCFHP antigen design. Specifically, we show by size exclusion chromatography and differential scanning fluorimetry that mutations in the VOCs adversely alter the antigen&#39;s structure and stability. More importantly, we determined that DCFHP without strain-specific mutations elicits the most robust, cross-reactive response in both pseudovirus and live virus neutralization assays. Our data suggest potential limitations to the variant-chasing approach in the development of protein nanoparticle vaccines, but also have implications for other approaches including mRNA-based vaccines.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
