<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Target DNA-dependent activation mechanism of the prokaryotic immune system SPARTA
Authors: Finocchio, G.; Koopal, B.; Potocnik, A.; Heijstek, C.; Jinek, M.; Swarts, D. C.
Score: 22.3, Published: 2023-09-25 DOI: 10.1101/2023.09.23.559129
In both prokaryotic and eukaryotic innate immune systems, TIR domains function as NADases that degrade the key metabolite NAD&#43; or generate signaling molecules. Catalytic activation of TIR domains requires oligomerization, but how this is achieved varies in distinct immune systems.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Target DNA-dependent activation mechanism of the prokaryotic immune system SPARTA
Authors: Finocchio, G.; Koopal, B.; Potocnik, A.; Heijstek, C.; Jinek, M.; Swarts, D. C.
Score: 22.3, Published: 2023-09-25 DOI: 10.1101/2023.09.23.559129
In both prokaryotic and eukaryotic innate immune systems, TIR domains function as NADases that degrade the key metabolite NAD&#43; or generate signaling molecules. Catalytic activation of TIR domains requires oligomerization, but how this is achieved varies in distinct immune systems." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-27T10:39:11+00:00" />
<meta property="article:modified_time" content="2023-09-27T10:39:11+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Target DNA-dependent activation mechanism of the prokaryotic immune system SPARTA
Authors: Finocchio, G.; Koopal, B.; Potocnik, A.; Heijstek, C.; Jinek, M.; Swarts, D. C.
Score: 22.3, Published: 2023-09-25 DOI: 10.1101/2023.09.23.559129
In both prokaryotic and eukaryotic innate immune systems, TIR domains function as NADases that degrade the key metabolite NAD&#43; or generate signaling molecules. Catalytic activation of TIR domains requires oligomerization, but how this is achieved varies in distinct immune systems."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Target DNA-dependent activation mechanism of the prokaryotic immune system SPARTA\nAuthors: Finocchio, G.; Koopal, B.; Potocnik, A.; Heijstek, C.; Jinek, M.; Swarts, D. C.\nScore: 22.3, Published: 2023-09-25 DOI: 10.1101/2023.09.23.559129\nIn both prokaryotic and eukaryotic innate immune systems, TIR domains function as NADases that degrade the key metabolite NAD\u0026#43; or generate signaling molecules. Catalytic activation of TIR domains requires oligomerization, but how this is achieved varies in distinct immune systems.",
  "keywords": [
    
  ],
  "articleBody": " Target DNA-dependent activation mechanism of the prokaryotic immune system SPARTA\nAuthors: Finocchio, G.; Koopal, B.; Potocnik, A.; Heijstek, C.; Jinek, M.; Swarts, D. C.\nScore: 22.3, Published: 2023-09-25 DOI: 10.1101/2023.09.23.559129\nIn both prokaryotic and eukaryotic innate immune systems, TIR domains function as NADases that degrade the key metabolite NAD+ or generate signaling molecules. Catalytic activation of TIR domains requires oligomerization, but how this is achieved varies in distinct immune systems. In the Short prokaryotic Argonaute (pAgo)/TIR-APAZ (SPARTA) immune system, TIR NADase activity is triggered upon guide RNA-mediated recognition of invading DNA by an unknown mechanism. Here, we describe cryo-EM structures of SPARTA in the inactive monomeric and target DNA-activated tetrameric states. The monomeric SPARTA structure reveals that in the absence of target DNA, a C-terminal tail of TIR-APAZ occupies the nucleic acid binding cleft formed by the pAgo and TIR-APAZ subunits, suppressing SPARTA activation. In the active tetrameric SPARTA complex, guide RNA-mediated target DNA binding displaces the C-terminal tail and induces conformational changes in pAgo that facilitate SPARTA-SPARTA dimerization. Concurrent release and rotation of one TIR domain allow it to form a composite NADase catalytic site with the other TIR domain within the dimer, and generate a self-complementary interface that mediates cooperative tetramerization. Combined, this study provides critical insights into the structural architecture of SPARTA and the molecular mechanism underlying target DNA-dependent oligomerization and catalytic activation.\nGlutamylation of Npm2 and Nap1 acidic disordered regions increases DNA charge mimicry to enhance chaperone efficiency\nAuthors: Lorton, B. M.; Warren, C.; Ilyas, H.; Nandigrami, P.; Hegde, S.; Cahill, S.; Lehman, S. M.; Shabanowitz, J.; Hunt, D.; Fiser, A.; Cowburn, D.; Shechter, D.\nScore: 16.0, Published: 2023-09-19 DOI: 10.1101/2023.09.18.558337\nHistone chaperones-structurally diverse, non-catalytic proteins enriched with acidic intrinsically disordered regions (IDRs)-protect histones from spurious nucleic acid interactions and guide their deposition into and out of nucleosomes. Despite their conservation and ubiquity, the function of the chaperone acidic IDRs remains unclear. Here, we show that the Xenopus laevis Npm2 and Nap1 acidic IDRs are substrates for TTLL4 (Tubulin Tyrosine Ligase Like 4)-catalyzed post-translational glutamate-glutamylation. We demonstrate that, to bind, stabilize, and deposit histones into nucleosomes, chaperone acidic IDRs function as DNA mimetics. Our biochemical, computational, and biophysical studies reveal that glutamylation of these chaperone polyelectrolyte acidic stretches functions to enhance DNA electrostatic mimicry, promoting the binding and stabilization of H2A/H2B heterodimers and facilitating nucleosome assembly. This discovery provides insights into both the previously unclear function of the acidic IDRs and the regulatory role of post-translational modifications in chromatin dynamics.\nSubstrate selectivity and catalytic activation of the type III CRISPR ancillary nuclease Can2\nAuthors: Jungfer, K.; Sigg, A.; Jinek, M.\nScore: 13.5, Published: 2023-09-18 DOI: 10.1101/2023.09.18.558242\nType III CRISPR-Cas systems provide adaptive immunity against foreign mobile genetic elements through RNA-guided interference. Sequence-specific recognition of RNA targets by the type III effector complex triggers the generation of cyclic oligoadenylate (cOA) second messengers that activate ancillary effector proteins, thus reinforcing the host immune response. The ancillary nuclease Can2 is activated by cyclic tetra-AMP (cA4); however, the mechanisms underlying cA4-mediated activation and substrate selectivity remain elusive. Here we report crystal structures of Thermoanaerobacter brockii Can2 (TbrCan2) in substrate- and product-bound complexes. We show that TbrCan2 is a single strand-selective DNase and RNase that binds substrates via a conserved SxTTS active site motif, and reveal molecular interactions underpinning its sequence preference for CA dinucleotides. Furthermore, we identify a molecular interaction relay linking the cA4 binding site and the nuclease catalytic site to enable divalent metal cation coordination and catalytic activation. These findings provide key insights into the molecular mechanisms of Can2 nucleases in type III CRISPR-Cas immunity and may guide their technological development for nucleic acid detection applications.\nDe novo designed Hsp70 activator dissolves intracellular condensates\nAuthors: Zhang, J. Z.; Greenwood, N.; Hernandez, J.; Cuperus, J. T.; Huang, B.; Ryder, B. D.; Queitsch, C.; Gestwicki, J. E.; Baker, D.\nScore: 10.1, Published: 2023-09-19 DOI: 10.1101/2023.09.18.558356\nProtein quality control (PQC) is carried out in part by the chaperone Hsp70, in concert with adapters of the J-domain protein (JDP) family. The JDPs, also called Hsp40s, are thought to recruit Hsp70 into complexes with specific client proteins. However, the molecular principles regulating this process are not well understood. We describe the de novo design of a set of Hsp70 binding proteins that either inhibited or stimulated Hsp70s ATPase activity; a stimulating design promoted the refolding of denatured luciferase in vitro, similar to native JDPs. Targeting of this design to intracellular condensates resulted in their nearly complete dissolution. The designs inform our understanding of chaperone structure-function relationships and provide a general and modular way to target PQC systems to condensates and other cellular targets.\nEvolutionary engineering a larger porin using a loop-to-hairpin mechanism\nAuthors: Dhar, R.; Bowman, A. M.; Hatungimana, B.; Slusky, J. S. G.\nScore: 7.7, Published: 2023-09-20 DOI: 10.1101/2023.06.14.544993\nIn protein evolution, diversification is generally driven by genetic duplication. The hallmarks of this mechanism are visible in the repeating topology of various proteins. In outer membrane {beta}-barrels, duplication is visible with {beta}-hairpins as the repeating unit of the barrel. In contrast to the overall use of duplication in diversification, a computational study hypothesized evolutionary mechanisms other than hairpin duplications leading to increases in the number of strands in outer membrane {beta}-barrels. Specifically, the topology of some 16- and 18-stranded {beta}-barrels appear to have evolved through a loop to {beta}-hairpin transition. Here we test this novel evolutionary mechanism by creating a chimeric protein from an 18-stranded {beta}-barrel and an evolutionarily related 16-stranded {beta}-barrel. The chimeric combination of the two was created by replacing loop L3 of the 16-stranded barrel with the sequentially matched transmembrane {beta}-hairpin region of the 18-stranded barrel. We find the resulting chimeric protein is stable and has characteristics of increased strand number. This study provides the first experimental evidence supporting the evolution through a loop to {beta}-hairpin transition. HighlightsO_LIWe find evidence supporting a novel diversification mechanism in membrane {beta}-barrels C_LIO_LIThe mechanism is the conversion of an extracellular loop to transmembrane {beta}-hairpin C_LIO_LIA chimeric protein modeling this mechanism folds stably in the membrane C_LIO_LIThe chimera has more {beta}-structure and a larger pore, consistent with a loop-to-hairpin transition C_LI\nC. elegans Dicer acts with the RIG-I-like helicase DRH-1 and RDE-4 to cleave dsRNA\nAuthors: Consalvo, C. D.; Aderounmu, A. M.; Donelick, H. M.; Aruscavage, P. J.; Eckert, D. M.; Shen, P. S.; Bass, B. L.\nScore: 6.8, Published: 2023-09-21 DOI: 10.1101/2023.09.21.558868\nAbstract/SummaryInvertebrates use the endoribonuclease Dicer to cleave viral dsRNA during antiviral defense, while vertebrates use RIG-I-like Receptors (RLRs), which bind viral dsRNA to trigger an interferon response. While some invertebrate Dicers act alone during antiviral defense, C. elegans Dicer acts in a complex with a dsRNA binding protein called RDE-4, and an RLR ortholog called DRH-1. We used biochemical and structural techniques to provide mechanistic insight into how these proteins function together. We found RDE-4 is important for ATP- independent and ATP-dependent cleavage reactions, while helicase domains of both DCR-1 and DRH-1 contribute to ATP-dependent cleavage. DRH-1 plays the dominant role in ATP hydrolysis, and like mammalian RLRs, has an N-terminal domain that functions in autoinhibition. A cryo-EM structure indicates DRH-1 interacts with DCR-1s helicase domain, suggesting this interaction relieves autoinhibition. Our study unravels the mechanistic basis of the collaboration between two helicases from typically distinct innate immune defense pathways.\nBioluminescent detection of viral surface proteins using branched multivalent protein switches\nAuthors: Grawe, A.; Spruit, C. M.; de Vries, R. P.; Merkx, M.\nScore: 5.4, Published: 2023-09-22 DOI: 10.1101/2023.09.22.558942\nFast and reliable virus diagnostics is key to prevent the spread of viruses in populations. A hallmark of viruses is the presence of multivalent surface proteins, a property that can be harnessed to control conformational switching in sensor proteins. Here, we introduce a new sensor platform (dark-LUX) for the detection of viral surface proteins consisting of a general bioluminescent framework that can be post-translationally functionalized with separately expressed binding domains. The platform relies on 1) plug-and-play bioconjugation of different binding proteins via SpyTag/SpyCatcher technology to create branched protein structures, 2) an optimized turn-on bioluminescent switch based on complementation of the split-luciferase NanoBiT upon target binding and 3) straightforward exploration of the protein linker space. The influenza A virus (IAV) surface proteins hemagglutinin (HA) and neuraminidase (NA) were used as relevant multivalent targets to establish proof of principle and optimize relevant parameters such as linker properties, choice of target binding domains and the optimal combination of the competing NanoBiT components SmBiT and DarkBiT. The sensor framework allows rapid conjugation and exchange of various binding domains including scFvs, nanobodies and de novo designed binders for a variety of targets, including the construction of a heterobivalent switch that targets the head and stem region of hemagglutinin. The modularity of the platform thus allows straightforward optimization of binding domains and scaffold properties for existing viral targets, and is well suited to quickly adapt bioluminescent sensor proteins to effectively detect newly evolving viral epitopes.\nRecognition of highly branched N-glycans of the porcine whipworm by the immune system\nAuthors: Eckmair, B.; Gao, C.; Mehta, A. Y.; Dutkiewicz, Z.; Vanbeselaere, J.; Cummings, R. D.; Paschinger, K.; Wilson, I. B. H.\nScore: 5.3, Published: 2023-09-21 DOI: 10.1101/2023.09.21.557549\nGlycans are key to host-pathogen interactions, whereby recognition by the host and immunomodulation by the pathogen can be mediated by carbohydrate binding proteins, such as lectins of the innate immune system, and their glycoconjugate ligands. Previous studies have shown that excretory-secretory products of the porcine nematode parasite Trichuris suis exert immunomodulatory effects in a glycan-dependent manner. To better understand the mechanisms of these interactions, we prepared N-glycans from T. suis and both analyzed their structures and used them to generate a natural glycan microarray. With this array we explored the interactions of glycans with C-type lectins, C-reactive protein and sera from T. suis infected pigs. Glycans containing LacdiNAc and phosphorylcholine-modified glycans were associated with the highest binding by most of these proteins. In-depth analysis revealed not only fucosylated LacdiNAc motifs with and without phosphorylcholine moieties, but phosphorylcholine-modified mannose and N-acetylhexosamine-substituted fucose residues, in the context of maximally tetraantennary N-glycan scaffolds. Furthermore, O-glycans also contained fucosylated motifs. In summary, the glycans of T. suis are recognized by both the innate and adaptive immune systems, and also exhibit species-specific features distinguishing its glycome from those of other nematodes.\nLight-sensitive phosphorylation regulates enzyme activity and filament assembly of human IMPDH1 retinal splice variants\nAuthors: Calise, S. J.; O'Neill, A. G.; Burrell, A. L.; Dickinson, M. S.; Molfino, J.; Clarke, C.; Quispe, J.; Sokolov, D.; Buey, R. M.; Kollman, J. M.\nScore: 4.9, Published: 2023-09-21 DOI: 10.1101/2023.09.21.558867\nInosine monophosphate dehydrogenase (IMPDH) is the rate-limiting enzyme in de novo guanosine triphosphate (GTP) synthesis and is controlled by feedback inhibition and allosteric regulation. IMPDH assembles into micron-scale filaments in cells, which desensitizes the enzyme to feedback inhibition by GTP and boosts nucleotide production. The vertebrate retina expresses two tissue-specific splice variants IMPDH1(546) and IMPDH1(595). IMPDH1(546) filaments adopt high and low activity conformations, while IMPDH1(595) filaments maintain high activity. In bovine retinas, residue S477 is preferentially phosphorylated in the dark, but the effects on IMPDH1 activity and regulation are unclear. Here, we generated phosphomimetic mutants to investigate structural and functional consequences of phosphorylation in IMPDH1 variants. The S477D mutation re-sensitized both variants to GTP inhibition, but only blocked assembly of IMPDH1(595) filaments and not IMPDH1(546) filaments. Cryo-EM structures of both variants showed that S477D specifically blocks assembly of the high activity assembly interface, still allowing assembly of low activity IMPDH1(546) filaments. Finally, we discovered that S477D exerts a dominant-negative effect in cells, preventing endogenous IMPDH filament assembly. By modulating the structure and higher-order assembly of IMPDH, phosphorylation at S477 acts as a mechanism for downregulating retinal GTP synthesis in the dark, when nucleotide turnover is decreased. Like IMPDH1, many other metabolic enzymes dynamically assemble filamentous polymers that allosterically regulate activity. Our work suggests that posttranslational modifications may be yet another layer of regulatory control to finely tune activity by modulating filament assembly in response to changing metabolic demands. SIGNIFICANCE STATEMENTOver 20 different metabolic enzymes form micron-scale filaments in cells, suggesting that filament assembly is a conserved mechanism for regulating diverse metabolic pathways. Filament assembly regulates catalytic activity of many of these enzymes, including inosine monophosphate dehydrogenase (IMPDH), the rate-limiting enzyme in de novo GTP biosynthesis. The vertebrate retina expresses two IMPDH1 splice variants that are critical for maintaining nucleotide levels required for phototransduction. Here, we show that filament assembly by these variants is itself controlled by phosphorylation at a single residue, adding further complexity to the tight regulation of nucleotide metabolism in the retina. Phosphorylation and other posttranslational modifications are likely to be a general regulatory mechanism controlling filament assembly by enzymes in many different metabolic pathways.\nMethionine Alkylation as an Approach to Quantify Methionine Oxidation Using Mass Spectrometry\nAuthors: Hoare, M.; Tan, R.; Welle, K.; Swovick, K.; Hryhorenko, J. R.; Ghaemmaghami, S.\nScore: 4.0, Published: 2023-09-21 DOI: 10.1101/2023.09.21.558873\nPost-translational oxidation of methionine residues can destabilize proteins or modify their functions. Although levels of methionine oxidation can provide important information regarding the structural integrity and regulation of proteins, their quantitation is often challenging as analytical procedures in and of themselves can artifactually oxidize methionines. Here, we develop a mass spectrometry-based method called Methionine Oxidation by Blocking with Alkylation (MObBa) that quantifies methionine oxidation by selectively alkylating and blocking unoxidized methionines. Thus, alkylated methionines can be used as a stable proxy for unoxidized methionines. Using proof of concept experiments, we demonstrate that MObBa can be used to measure methionine oxidation levels within individual synthetic peptides and on proteome-wide scales. MObBa may provide a straightforward experimental strategy for mass spectrometric quantitation of methionine oxidation. Significance StatementOver time, cellular proteins can become oxidatively damaged by reactive oxygen species (ROS). A residue that is particularly prone to oxidative damage is methionine. Here, we develop and validate a methodology for detecting and quantifying levels of methionine oxidation by mass spectrometry. This approach has a number of practical advantages over methods currently available for analysis of methionine oxidation. The ability to accurately quantify methionine oxidation will provide important insights into factors that influence protein homeostasis within a cell.\n",
  "wordCount" : "2342",
  "inLanguage": "en",
  "datePublished": "2023-09-27T10:39:11Z",
  "dateModified": "2023-09-27T10:39:11Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta"><span>updated on September 27, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.23.559129">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.23.559129" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.23.559129">
        <p class="paperTitle">Target DNA-dependent activation mechanism of the prokaryotic immune system SPARTA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.23.559129" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.23.559129" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Finocchio, G.; Koopal, B.; Potocnik, A.; Heijstek, C.; Jinek, M.; Swarts, D. C.</p>
        <p class="info">Score: 22.3, Published: 2023-09-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.23.559129' target='https://doi.org/10.1101/2023.09.23.559129'> 10.1101/2023.09.23.559129</a></p>
        <p class="abstract">In both prokaryotic and eukaryotic innate immune systems, TIR domains function as NADases that degrade the key metabolite NAD&#43; or generate signaling molecules. Catalytic activation of TIR domains requires oligomerization, but how this is achieved varies in distinct immune systems. In the Short prokaryotic Argonaute (pAgo)/TIR-APAZ (SPARTA) immune system, TIR NADase activity is triggered upon guide RNA-mediated recognition of invading DNA by an unknown mechanism. Here, we describe cryo-EM structures of SPARTA in the inactive monomeric and target DNA-activated tetrameric states. The monomeric SPARTA structure reveals that in the absence of target DNA, a C-terminal tail of TIR-APAZ occupies the nucleic acid binding cleft formed by the pAgo and TIR-APAZ subunits, suppressing SPARTA activation. In the active tetrameric SPARTA complex, guide RNA-mediated target DNA binding displaces the C-terminal tail and induces conformational changes in pAgo that facilitate SPARTA-SPARTA dimerization. Concurrent release and rotation of one TIR domain allow it to form a composite NADase catalytic site with the other TIR domain within the dimer, and generate a self-complementary interface that mediates cooperative tetramerization. Combined, this study provides critical insights into the structural architecture of SPARTA and the molecular mechanism underlying target DNA-dependent oligomerization and catalytic activation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558337">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558337" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558337">
        <p class="paperTitle">Glutamylation of Npm2 and Nap1 acidic disordered regions increases DNA charge mimicry to enhance chaperone efficiency</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558337" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558337" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lorton, B. M.; Warren, C.; Ilyas, H.; Nandigrami, P.; Hegde, S.; Cahill, S.; Lehman, S. M.; Shabanowitz, J.; Hunt, D.; Fiser, A.; Cowburn, D.; Shechter, D.</p>
        <p class="info">Score: 16.0, Published: 2023-09-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558337' target='https://doi.org/10.1101/2023.09.18.558337'> 10.1101/2023.09.18.558337</a></p>
        <p class="abstract">Histone chaperones-structurally diverse, non-catalytic proteins enriched with acidic intrinsically disordered regions (IDRs)-protect histones from spurious nucleic acid interactions and guide their deposition into and out of nucleosomes. Despite their conservation and ubiquity, the function of the chaperone acidic IDRs remains unclear. Here, we show that the Xenopus laevis Npm2 and Nap1 acidic IDRs are substrates for TTLL4 (Tubulin Tyrosine Ligase Like 4)-catalyzed post-translational glutamate-glutamylation. We demonstrate that, to bind, stabilize, and deposit histones into nucleosomes, chaperone acidic IDRs function as DNA mimetics. Our biochemical, computational, and biophysical studies reveal that glutamylation of these chaperone polyelectrolyte acidic stretches functions to enhance DNA electrostatic mimicry, promoting the binding and stabilization of H2A/H2B heterodimers and facilitating nucleosome assembly. This discovery provides insights into both the previously unclear function of the acidic IDRs and the regulatory role of post-translational modifications in chromatin dynamics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558242">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558242" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558242">
        <p class="paperTitle">Substrate selectivity and catalytic activation of the type III CRISPR ancillary nuclease Can2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558242" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558242" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jungfer, K.; Sigg, A.; Jinek, M.</p>
        <p class="info">Score: 13.5, Published: 2023-09-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558242' target='https://doi.org/10.1101/2023.09.18.558242'> 10.1101/2023.09.18.558242</a></p>
        <p class="abstract">Type III CRISPR-Cas systems provide adaptive immunity against foreign mobile genetic elements through RNA-guided interference. Sequence-specific recognition of RNA targets by the type III effector complex triggers the generation of cyclic oligoadenylate (cOA) second messengers that activate ancillary effector proteins, thus reinforcing the host immune response. The ancillary nuclease Can2 is activated by cyclic tetra-AMP (cA4); however, the mechanisms underlying cA4-mediated activation and substrate selectivity remain elusive. Here we report crystal structures of Thermoanaerobacter brockii Can2 (TbrCan2) in substrate- and product-bound complexes. We show that TbrCan2 is a single strand-selective DNase and RNase that binds substrates via a conserved SxTTS active site motif, and reveal molecular interactions underpinning its sequence preference for CA dinucleotides. Furthermore, we identify a molecular interaction relay linking the cA4 binding site and the nuclease catalytic site to enable divalent metal cation coordination and catalytic activation. These findings provide key insights into the molecular mechanisms of Can2 nucleases in type III CRISPR-Cas immunity and may guide their technological development for nucleic acid detection applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558356">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558356" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558356">
        <p class="paperTitle">De novo designed Hsp70 activator dissolves intracellular condensates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558356" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558356" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, J. Z.; Greenwood, N.; Hernandez, J.; Cuperus, J. T.; Huang, B.; Ryder, B. D.; Queitsch, C.; Gestwicki, J. E.; Baker, D.</p>
        <p class="info">Score: 10.1, Published: 2023-09-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558356' target='https://doi.org/10.1101/2023.09.18.558356'> 10.1101/2023.09.18.558356</a></p>
        <p class="abstract">Protein quality control (PQC) is carried out in part by the chaperone Hsp70, in concert with adapters of the J-domain protein (JDP) family. The JDPs, also called Hsp40s, are thought to recruit Hsp70 into complexes with specific client proteins. However, the molecular principles regulating this process are not well understood. We describe the de novo design of a set of Hsp70 binding proteins that either inhibited or stimulated Hsp70s ATPase activity; a stimulating design promoted the refolding of denatured luciferase in vitro, similar to native JDPs. Targeting of this design to intracellular condensates resulted in their nearly complete dissolution. The designs inform our understanding of chaperone structure-function relationships and provide a general and modular way to target PQC systems to condensates and other cellular targets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.14.544993">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.14.544993" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.14.544993">
        <p class="paperTitle">Evolutionary engineering a larger porin using a loop-to-hairpin mechanism</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.14.544993" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.14.544993" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dhar, R.; Bowman, A. M.; Hatungimana, B.; Slusky, J. S. G.</p>
        <p class="info">Score: 7.7, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.14.544993' target='https://doi.org/10.1101/2023.06.14.544993'> 10.1101/2023.06.14.544993</a></p>
        <p class="abstract">In protein evolution, diversification is generally driven by genetic duplication. The hallmarks of this mechanism are visible in the repeating topology of various proteins. In outer membrane {beta}-barrels, duplication is visible with {beta}-hairpins as the repeating unit of the barrel. In contrast to the overall use of duplication in diversification, a computational study hypothesized evolutionary mechanisms other than hairpin duplications leading to increases in the number of strands in outer membrane {beta}-barrels. Specifically, the topology of some 16- and 18-stranded {beta}-barrels appear to have evolved through a loop to {beta}-hairpin transition. Here we test this novel evolutionary mechanism by creating a chimeric protein from an 18-stranded {beta}-barrel and an evolutionarily related 16-stranded {beta}-barrel. The chimeric combination of the two was created by replacing loop L3 of the 16-stranded barrel with the sequentially matched transmembrane {beta}-hairpin region of the 18-stranded barrel. We find the resulting chimeric protein is stable and has characteristics of increased strand number. This study provides the first experimental evidence supporting the evolution through a loop to {beta}-hairpin transition.

HighlightsO_LIWe find evidence supporting a novel diversification mechanism in membrane {beta}-barrels
C_LIO_LIThe mechanism is the conversion of an extracellular loop to transmembrane {beta}-hairpin
C_LIO_LIA chimeric protein modeling this mechanism folds stably in the membrane
C_LIO_LIThe chimera has more {beta}-structure and a larger pore, consistent with a loop-to-hairpin transition
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558868">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558868" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558868">
        <p class="paperTitle">C. elegans Dicer acts with the RIG-I-like helicase DRH-1 and RDE-4 to cleave dsRNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558868" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558868" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Consalvo, C. D.; Aderounmu, A. M.; Donelick, H. M.; Aruscavage, P. J.; Eckert, D. M.; Shen, P. S.; Bass, B. L.</p>
        <p class="info">Score: 6.8, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558868' target='https://doi.org/10.1101/2023.09.21.558868'> 10.1101/2023.09.21.558868</a></p>
        <p class="abstract">Abstract/SummaryInvertebrates use the endoribonuclease Dicer to cleave viral dsRNA during antiviral defense, while vertebrates use RIG-I-like Receptors (RLRs), which bind viral dsRNA to trigger an interferon response. While some invertebrate Dicers act alone during antiviral defense, C. elegans Dicer acts in a complex with a dsRNA binding protein called RDE-4, and an RLR ortholog called DRH-1. We used biochemical and structural techniques to provide mechanistic insight into how these proteins function together. We found RDE-4 is important for ATP- independent and ATP-dependent cleavage reactions, while helicase domains of both DCR-1 and DRH-1 contribute to ATP-dependent cleavage. DRH-1 plays the dominant role in ATP hydrolysis, and like mammalian RLRs, has an N-terminal domain that functions in autoinhibition. A cryo-EM structure indicates DRH-1 interacts with DCR-1s helicase domain, suggesting this interaction relieves autoinhibition. Our study unravels the mechanistic basis of the collaboration between two helicases from typically distinct innate immune defense pathways.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.558942">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.558942" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.558942">
        <p class="paperTitle">Bioluminescent detection of viral surface proteins using branched multivalent protein switches</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.558942" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.558942" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Grawe, A.; Spruit, C. M.; de Vries, R. P.; Merkx, M.</p>
        <p class="info">Score: 5.4, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.558942' target='https://doi.org/10.1101/2023.09.22.558942'> 10.1101/2023.09.22.558942</a></p>
        <p class="abstract">Fast and reliable virus diagnostics is key to prevent the spread of viruses in populations. A hallmark of viruses is the presence of multivalent surface proteins, a property that can be harnessed to control conformational switching in sensor proteins. Here, we introduce a new sensor platform (dark-LUX) for the detection of viral surface proteins consisting of a general bioluminescent framework that can be post-translationally functionalized with separately expressed binding domains. The platform relies on 1) plug-and-play bioconjugation of different binding proteins via SpyTag/SpyCatcher technology to create branched protein structures, 2) an optimized turn-on bioluminescent switch based on complementation of the split-luciferase NanoBiT upon target binding and 3) straightforward exploration of the protein linker space. The influenza A virus (IAV) surface proteins hemagglutinin (HA) and neuraminidase (NA) were used as relevant multivalent targets to establish proof of principle and optimize relevant parameters such as linker properties, choice of target binding domains and the optimal combination of the competing NanoBiT components SmBiT and DarkBiT. The sensor framework allows rapid conjugation and exchange of various binding domains including scFvs, nanobodies and de novo designed binders for a variety of targets, including the construction of a heterobivalent switch that targets the head and stem region of hemagglutinin. The modularity of the platform thus allows straightforward optimization of binding domains and scaffold properties for existing viral targets, and is well suited to quickly adapt bioluminescent sensor proteins to effectively detect newly evolving viral epitopes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.557549">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.557549" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.557549">
        <p class="paperTitle">Recognition of highly branched N-glycans of the porcine whipworm by the immune system</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.557549" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.557549" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eckmair, B.; Gao, C.; Mehta, A. Y.; Dutkiewicz, Z.; Vanbeselaere, J.; Cummings, R. D.; Paschinger, K.; Wilson, I. B. H.</p>
        <p class="info">Score: 5.3, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.557549' target='https://doi.org/10.1101/2023.09.21.557549'> 10.1101/2023.09.21.557549</a></p>
        <p class="abstract">Glycans are key to host-pathogen interactions, whereby recognition by the host and immunomodulation by the pathogen can be mediated by carbohydrate binding proteins, such as lectins of the innate immune system, and their glycoconjugate ligands. Previous studies have shown that excretory-secretory products of the porcine nematode parasite Trichuris suis exert immunomodulatory effects in a glycan-dependent manner. To better understand the mechanisms of these interactions, we prepared N-glycans from T. suis and both analyzed their structures and used them to generate a natural glycan microarray. With this array we explored the interactions of glycans with C-type lectins, C-reactive protein and sera from T. suis infected pigs. Glycans containing LacdiNAc and phosphorylcholine-modified glycans were associated with the highest binding by most of these proteins. In-depth analysis revealed not only fucosylated LacdiNAc motifs with and without phosphorylcholine moieties, but phosphorylcholine-modified mannose and N-acetylhexosamine-substituted fucose residues, in the context of maximally tetraantennary N-glycan scaffolds. Furthermore, O-glycans also contained fucosylated motifs. In summary, the glycans of T. suis are recognized by both the innate and adaptive immune systems, and also exhibit species-specific features distinguishing its glycome from those of other nematodes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558867">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558867" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558867">
        <p class="paperTitle">Light-sensitive phosphorylation regulates enzyme activity and filament assembly of human IMPDH1 retinal splice variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558867" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558867" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Calise, S. J.; O&#39;Neill, A. G.; Burrell, A. L.; Dickinson, M. S.; Molfino, J.; Clarke, C.; Quispe, J.; Sokolov, D.; Buey, R. M.; Kollman, J. M.</p>
        <p class="info">Score: 4.9, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558867' target='https://doi.org/10.1101/2023.09.21.558867'> 10.1101/2023.09.21.558867</a></p>
        <p class="abstract">Inosine monophosphate dehydrogenase (IMPDH) is the rate-limiting enzyme in de novo guanosine triphosphate (GTP) synthesis and is controlled by feedback inhibition and allosteric regulation. IMPDH assembles into micron-scale filaments in cells, which desensitizes the enzyme to feedback inhibition by GTP and boosts nucleotide production. The vertebrate retina expresses two tissue-specific splice variants IMPDH1(546) and IMPDH1(595). IMPDH1(546) filaments adopt high and low activity conformations, while IMPDH1(595) filaments maintain high activity. In bovine retinas, residue S477 is preferentially phosphorylated in the dark, but the effects on IMPDH1 activity and regulation are unclear. Here, we generated phosphomimetic mutants to investigate structural and functional consequences of phosphorylation in IMPDH1 variants. The S477D mutation re-sensitized both variants to GTP inhibition, but only blocked assembly of IMPDH1(595) filaments and not IMPDH1(546) filaments. Cryo-EM structures of both variants showed that S477D specifically blocks assembly of the high activity assembly interface, still allowing assembly of low activity IMPDH1(546) filaments. Finally, we discovered that S477D exerts a dominant-negative effect in cells, preventing endogenous IMPDH filament assembly. By modulating the structure and higher-order assembly of IMPDH, phosphorylation at S477 acts as a mechanism for downregulating retinal GTP synthesis in the dark, when nucleotide turnover is decreased. Like IMPDH1, many other metabolic enzymes dynamically assemble filamentous polymers that allosterically regulate activity. Our work suggests that posttranslational modifications may be yet another layer of regulatory control to finely tune activity by modulating filament assembly in response to changing metabolic demands.

SIGNIFICANCE STATEMENTOver 20 different metabolic enzymes form micron-scale filaments in cells, suggesting that filament assembly is a conserved mechanism for regulating diverse metabolic pathways. Filament assembly regulates catalytic activity of many of these enzymes, including inosine monophosphate dehydrogenase (IMPDH), the rate-limiting enzyme in de novo GTP biosynthesis. The vertebrate retina expresses two IMPDH1 splice variants that are critical for maintaining nucleotide levels required for phototransduction. Here, we show that filament assembly by these variants is itself controlled by phosphorylation at a single residue, adding further complexity to the tight regulation of nucleotide metabolism in the retina. Phosphorylation and other posttranslational modifications are likely to be a general regulatory mechanism controlling filament assembly by enzymes in many different metabolic pathways.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558873">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558873" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558873">
        <p class="paperTitle">Methionine Alkylation as an Approach to Quantify Methionine Oxidation Using Mass Spectrometry</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558873" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558873" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hoare, M.; Tan, R.; Welle, K.; Swovick, K.; Hryhorenko, J. R.; Ghaemmaghami, S.</p>
        <p class="info">Score: 4.0, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558873' target='https://doi.org/10.1101/2023.09.21.558873'> 10.1101/2023.09.21.558873</a></p>
        <p class="abstract">Post-translational oxidation of methionine residues can destabilize proteins or modify their functions. Although levels of methionine oxidation can provide important information regarding the structural integrity and regulation of proteins, their quantitation is often challenging as analytical procedures in and of themselves can artifactually oxidize methionines. Here, we develop a mass spectrometry-based method called Methionine Oxidation by Blocking with Alkylation (MObBa) that quantifies methionine oxidation by selectively alkylating and blocking unoxidized methionines. Thus, alkylated methionines can be used as a stable proxy for unoxidized methionines. Using proof of concept experiments, we demonstrate that MObBa can be used to measure methionine oxidation levels within individual synthetic peptides and on proteome-wide scales. MObBa may provide a straightforward experimental strategy for mass spectrometric quantitation of methionine oxidation.

Significance StatementOver time, cellular proteins can become oxidatively damaged by reactive oxygen species (ROS). A residue that is particularly prone to oxidative damage is methionine. Here, we develop and validate a methodology for detecting and quantifying levels of methionine oxidation by mass spectrometry. This approach has a number of practical advantages over methods currently available for analysis of methionine oxidation. The ability to accurately quantify methionine oxidation will provide important insights into factors that influence protein homeostasis within a cell.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
