<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="De novo design of allosterically switchable protein assemblies
Authors: Pillai, A.; Idris, A.; Philomin, A.; Weidle, C.; Skotheim, R.; Leung, P. J. Y.; Broerman, A.; Demakis, C.; Borst, A. J.; Praetorius, F.; Baker, D.
Score: 32.7, Published: 2023-11-02 DOI: 10.1101/2023.11.01.565167
Allosteric modulation of protein function, wherein the binding of an effector to a protein triggers conformational changes at distant functional sites, plays a central role in the control of metabolism and cell signaling1-3.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="De novo design of allosterically switchable protein assemblies
Authors: Pillai, A.; Idris, A.; Philomin, A.; Weidle, C.; Skotheim, R.; Leung, P. J. Y.; Broerman, A.; Demakis, C.; Borst, A. J.; Praetorius, F.; Baker, D.
Score: 32.7, Published: 2023-11-02 DOI: 10.1101/2023.11.01.565167
Allosteric modulation of protein function, wherein the binding of an effector to a protein triggers conformational changes at distant functional sites, plays a central role in the control of metabolism and cell signaling1-3." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-08T10:39:23+00:00" />
<meta property="article:modified_time" content="2023-11-08T10:39:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="De novo design of allosterically switchable protein assemblies
Authors: Pillai, A.; Idris, A.; Philomin, A.; Weidle, C.; Skotheim, R.; Leung, P. J. Y.; Broerman, A.; Demakis, C.; Borst, A. J.; Praetorius, F.; Baker, D.
Score: 32.7, Published: 2023-11-02 DOI: 10.1101/2023.11.01.565167
Allosteric modulation of protein function, wherein the binding of an effector to a protein triggers conformational changes at distant functional sites, plays a central role in the control of metabolism and cell signaling1-3."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "De novo design of allosterically switchable protein assemblies\nAuthors: Pillai, A.; Idris, A.; Philomin, A.; Weidle, C.; Skotheim, R.; Leung, P. J. Y.; Broerman, A.; Demakis, C.; Borst, A. J.; Praetorius, F.; Baker, D.\nScore: 32.7, Published: 2023-11-02 DOI: 10.1101/2023.11.01.565167\nAllosteric modulation of protein function, wherein the binding of an effector to a protein triggers conformational changes at distant functional sites, plays a central role in the control of metabolism and cell signaling1-3.",
  "keywords": [
    
  ],
  "articleBody": " De novo design of allosterically switchable protein assemblies\nAuthors: Pillai, A.; Idris, A.; Philomin, A.; Weidle, C.; Skotheim, R.; Leung, P. J. Y.; Broerman, A.; Demakis, C.; Borst, A. J.; Praetorius, F.; Baker, D.\nScore: 32.7, Published: 2023-11-02 DOI: 10.1101/2023.11.01.565167\nAllosteric modulation of protein function, wherein the binding of an effector to a protein triggers conformational changes at distant functional sites, plays a central role in the control of metabolism and cell signaling1-3. There has been considerable interest in designing allosteric systems, both to gain insight into the mechanisms underlying such \"action at a distance\" modulation and to create synthetic proteins whose functions can be regulated by effectors4-7. However, emulating the subtle conformational changes distributed across many residues, characteristic of natural allosteric proteins, is a significant challenge8,9. Here, inspired by the classic Monod-Changeux-Wyman model of cooperativity10, we investigate the de novo design of allostery through rigid-body coupling of designed effector-switchable hinge modules11 to protein interfaces12 that direct the formation of alternative oligomeric states. We find that this approach can be used to generate a wide variety of allosterically switchable systems, including cyclic rings that incorporate or eject subunits in response to effector binding and dihedral cages that undergo effector-induced disassembly. Size-exclusion chromatography, mass photometry13, and electron microscopy reveal that these designed allosteric protein assemblies closely resemble the design models in both the presence and absence of effectors and can have ligand-binding cooperativity comparable to classic natural systems such as hemoglobin14. Our results indicate that allostery can arise from global coupling of the energetics of protein substructures without optimized sidechain-sidechain allosteric communication pathways and provide a roadmap for generating allosterically triggerable delivery systems, protein nanomachines, and cellular feedback control circuitry.\nPervasive aggregation and depletion of host and viral proteins in response to cysteine-reactive electrophilic compounds\nAuthors: Julio, A. R.; Shikwana, F.; Truong, C.; Burton, N. R.; Dominguez, E.; Turmon, A.; Cao, J.; Backus, K.\nScore: 27.0, Published: 2023-11-01 DOI: 10.1101/2023.10.30.564067\nProtein homeostasis is tightly regulated, with damaged or misfolded proteins quickly eliminated by the proteasome and autophagosome pathways. By co-opting these processes, targeted protein degradation technologies enable pharmacological manipulation of protein abundance. Recently, cysteine-reactive molecules have been added to the degrader toolbox, which offer the benefit of unlocking the therapeutic potential of undruggable protein targets. The proteome-wide impact of these molecules remains to be fully understood and given the general reactivity of many classes of cysteine-reactive electrophiles, on- and off-target effects are likely. Using chemical proteomics, we identified a cysteine-reactive small molecule degrader of the SARS-CoV-2 non-structural protein 14 (nsp14), which effects degradation through direct modification of cysteines in both nsp14 and in host chaperones together with activation of global cell stress response pathways. We find that cysteine-reactive electrophiles increase global protein ubiquitylation, trigger proteasome activation, and result in widespread aggregation and depletion of host proteins, including components of the nuclear pore complex. Formation of stress granules was also found to be a remarkably ubiquitous cellular response to nearly all cysteine-reactive compounds and degraders. Collectively, our study sheds light on complexities of covalent target protein degradation and highlights untapped opportunities in manipulating and characterizing proteostasis processes via deciphering the cysteine-centric regulation of stress response pathways.\nSmall-molecule binding and sensing with a designed protein family\nAuthors: Lee, G. R.; Pellock, S. J.; Norn, C.; Tischer, D.; Dauparas, J.; Anishchenko, I.; Mercer, J. A. M.; Kang, A.; Bera, A.; Nguyen, H.; Goreshnik, I.; Vafeados, D.; Roullier, N.; Han, H. L.; Coventry, B.; Haddox, H. K.; Liu, D. R.; Yeh, A. H.-W.; Baker, D.\nScore: 22.7, Published: 2023-11-02 DOI: 10.1101/2023.11.01.565201\nDespite transformative advances in protein design with deep learning, the design of small-molecule-binding proteins and sensors for arbitrary ligands remains a grand challenge. Here we combine deep learning and physics-based methods to generate a family of proteins with diverse and designable pocket geometries, which we employ to computationally design binders for six chemically and structurally distinct small-molecule targets. Biophysical characterization of the designed binders revealed nanomolar to low micromolar binding affinities and atomic-level design accuracy. The bound ligands are exposed at one edge of the binding pocket, enabling the de novo design of chemically induced dimerization (CID) systems; we take advantage of this to create a biosensor with nanomolar sensitivity for cortisol. Our approach provides a general method to design proteins that bind and sense small molecules for a wide range of analytical, environmental, and biomedical applications.\nCovalent Degrader of the Oncogenic Transcription Factor β-Catenin\nAuthors: Gowans, F. A.; Forte, N.; Hatcher, J.; Huang, O. W.; Wang, Y.; Poblano, B. E. A.; Wertz, I. E.; Nomura, D. K.\nScore: 18.3, Published: 2023-11-02 DOI: 10.1101/2023.10.31.565018\n{beta}-catenin (CTNNB1) is an oncogenic transcription factor that is important in cell-cell adhesion and transcription of cell proliferation and survival genes that drives the pathogenesis of many different types of cancers. However, direct pharmacological targeting of CTNNB1 has remained challenging deeming this transcription factor as \"undruggable.\" Here, we have performed a screen with a library of cysteine-reactive covalent ligands to identify a monovalent degrader EN83 that depletes CTNNB1 in a ubiquitin-proteasome-dependent manner. We show that EN83 directly and covalently targets CTNNB1 through targeting four distinct cysteines within the armadillo repeat domain--C439, C466, C520, and C619--leading to a destabilization of CTNNB1. Using covalent chemoproteomic approaches, we show that EN83 directly engages CTNNB1 in cells with a moderate degree of selectivity. We further demonstrate that direct covalent targeting of three of these four cysteines--C466, C520, and C619--in cells contributes to CTNNB1 degradation in cells. We also demonstrate that EN83 can be further optimized to yield more potent CTNNB1 binders and degraders. Our results show that chemoproteomic approaches can be used to covalently target and degrade challenging transcription factors like CTNNB1 through a destabilization-mediated degradation.\nElectric fields determine carbapenemase activity in class A β-lactamases\nAuthors: Jabeen, H.; Beer, M.; Spencer, J.; van der Kamp, M. W.; Bunzel, H. A.; Mulholland, A. J.\nScore: 10.4, Published: 2023-11-05 DOI: 10.1101/2023.11.04.565607\nAntimicrobial resistance is a public health crisis. Limited understanding of the catalytic drivers in resistance-mediating enzymes such as {beta}-lactamases hinders our ability to combat this crisis. Here, we dissect the catalytic contributions of active-site electric fields in class A {beta}-lactamases. We studied the enzymatic hydrolysis of a carbapenem antibiotic by QM/MM molecular dynamics simulations and quantified active-site fields with a custom-made script. We discovered that the fields correlate well with activity and identified seven positions, some distal, that distinguish efficient carbapenemases. Electric-field analysis may help predict the activity of {beta}-lactamases and guide antibiotic and enzyme design. Electric field script: https://www.github.com/bunzela/FieldTools\nDesign of efficient artificial enzymes using crystallographically-enhanced conformational sampling\nAuthors: Rakotoharisoa, R. V.; Seifinoferest, B.; Zarifi, N.; Miller, J. D. M.; Rodriguez, J. M.; Thompson, M. C.; Chica, R. A.\nScore: 10.0, Published: 2023-11-02 DOI: 10.1101/2023.11.01.564846\nThe ability to create efficient artificial enzymes for any chemical reaction is of great interest. Here, we describe a computational design method for increasing catalytic efficiency of de novo enzymes to a level comparable to their natural counterparts without relying on directed evolution. Using structural ensembles generated from dynamics-based refinement against X-ray diffraction data collected from crystals of Kemp eliminases HG3 (kcat/KM 125 M-1 s-1) and KE70 (kcat/KM 57 M-1 s-1), we design from each enzyme [\u0026le;]10 sequences predicted to catalyze this reaction more efficiently. The most active designs display kcat/KM values improved by 100-250-fold, comparable to mutants obtained after screening thousands of variants in multiple rounds of directed evolution. Crystal structures show excellent agreement with computational models. Our work shows how computational design can generate efficient artificial enzymes by exploiting the true conformational ensemble to more effectively stabilize the transition state.\nProximity proteomics reveals UCH-L1 as an NLRP3 interactor that modulates IL-1β production in human macrophages and microglia\nAuthors: Liang, Z.; Damianou, A.; Vendrell, I.; Jenkins, E.; Lassen, F. H.; Washer, S. J.; Liu, G.; Yi, G.; Lou, H.; Cao, F.; Zheng, X.; Fernandes, R.; Dong, T.; Tate, E. W.; Daniel, E. D.; Kessler, B. M.\nScore: 6.5, Published: 2023-10-24 DOI: 10.1101/2023.10.09.561576\nActivation of the NACHT, LRR family pyrin domain containing 3 (NLRP3) inflammasome complex is an essential innate immune signalling mechanism. To reveal how human NLRP3 inflammasome assembly and activation are controlled, in particular by components of the ubiquitin system, proximity labelling, affinity purification and RNAi screening approaches were performed. Our study provides an intricate time-resolved molecular map of different phases of NLRP3 inflammasome activation. Also, we show that ubiquitin C-terminal hydrolase 1 (UCH-L1) interacts with the NACHT domain of NLRP3. Downregulation of UCH-L1 decreases pro-IL-1{beta} levels. UCH-L1 chemical inhibition with small molecules interfered with NLRP3 puncta formation and ASC oligomerization, leading to altered IL-1{beta} cleavage and secretion, particularly in microglia cells, which exhibited elevated UCH-L1 expression as compared to monocytes/macrophages. Altogether, we profiled NLRP3 inflammasome activation dynamics and highlight UCH-L1 as an important modulator of NLRP3-mediated IL-1{beta} production, suggesting that a pharmacological inhibitor of UCH-L1 may decrease inflammation-associated pathologies.\nA minimal sequence motif drives selective tRNA dihydrouridylation by hDUS2\nAuthors: Ji, J.; Yu, N. J.; Kleiner, R. E.\nScore: 4.5, Published: 2023-11-05 DOI: 10.1101/2023.11.04.565616\nThe post-transcriptional reduction of uridine to dihydrouridine (D) by dihydrouridine synthase (DUS) enzymes is among the most ubiquitous transformations in RNA biology. D is found at multiple sites in tRNAs and studies in yeast have proposed that each of the four eukaryotic DUS enzymes modifies a different site, however the molecular basis for this exquisite selectivity is unknown and human DUS enzymes have remained largely uncharacterized. Here we investigate the substrate specificity of human dihydrouridine synthase 2 (hDUS2) using mechanism-based crosslinking with 5-bromouridine (5-BrUrd)-modified oligonucleotide probes and in vitro dihydrouridylation assays. We find that hDUS2 modifies U20 in the D loop of diverse tRNA substrates and identify a minimal GU motif within the tRNA tertiary fold required for directing its activity. Further, we use our mechanism-based platform to screen small molecule inhibitors of hDUS2, a potential anti-cancer target. Our work elucidates the principles of substrate modification by a conserved DUS and provides a general platform to studying RNA modifying enzymes with sequence-defined activity-based probes.\nGeneralized Biomolecular Modeling and Design with RoseTTAFold All-Atom\nAuthors: Krishna, R.; Wang, J.; Ahern, W.; Sturmfels, P.; Venkatesh, P.; Kalvet, I.; Lee, G. R.; Morey-Burrows, F. S.; Anishchenko, I.; Humphreys, I. R.; McHugh, R.; Vafeados, D.; Li, X.; Sutherland, G. A.; Hitchcock, A.; Hunter, C. N.; Baek, M.; DiMaio, F.; Baker, D.\nScore: 347.5, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561603\nAlthough AlphaFold2 (AF2) and RoseTTAFold (RF) have transformed structural biology by enabling high-accuracy protein structure modeling, they are unable to model covalent modifications or interactions with small molecules and other non-protein molecules that can play key roles in biological function. Here, we describe RoseTTAFold All-Atom (RFAA), a deep network capable of modeling full biological assemblies containing proteins, nucleic acids, small molecules, metals, and covalent modifications given the sequences of the polymers and the atomic bonded geometry of the small molecules and covalent modifications. Following training on structures of full biological assemblies in the Protein Data Bank (PDB), RFAA has comparable protein structure prediction accuracy to AF2, excellent performance in CAMEO for flexible backbone small molecule docking, and reasonable prediction accuracy for protein covalent modifications and assemblies of proteins with multiple nucleic acid chains and small molecules which, to our knowledge, no existing method can model simultaneously. By fine-tuning on diffusive denoising tasks, we develop RFdiffusion All-Atom (RFdiffusionAA), which generates binding pockets by directly building protein structures around small molecules and other non-protein molecules. Starting from random distributions of amino acid residues surrounding target small molecules, we design and experimentally validate proteins that bind the cardiac disease therapeutic digoxigenin, the enzymatic cofactor heme, and optically active bilin molecules with potential for expanding the range of wavelengths captured by photosynthesis. We anticipate that RFAA and RFdiffusionAA will be widely useful for modeling and designing complex biomolecular systems.\nRecruitment of FBXO22 for Targeted Degradation of NSD2\nAuthors: Nie, D. Y.; Tabor, J. R.; Li, J.; Kutera, M.; St-Germain, J.; Hanley, R. P.; WOLF, E. Z.; Paulakonis, E.; Kenney, T. M. G.; Duan, S.; Shrestha, S.; Owens, D. D.; Pon, A.; Szewczyk, M.; Lamberto, A. J.; Menes, M.; Li, F.; Barsyte-Lovejoy, D.; Brown, N. G.; Barsotti, A. M.; Stamford, A. W.; Collins, J. L.; Wilson, D. J.; Raught, B.; Licht, J.; James, L.; Arrowsmith, C.\nScore: 4.0, Published: 2023-11-02 DOI: 10.1101/2023.11.01.564830\nTargeted protein degradation (TPD) is an emerging therapeutic strategy that would benefit from new chemical entities with which to recruit a wider variety of ubiquitin E3 ligases to target proteins for proteasomal degradation. Here, we describe a TPD strategy involving the recruitment of FBXO22 to induce degradation of the histone methyltransferase and oncogene NSD2. UNC8732 facilitates FBXO22-mediated degradation of NSD2 in acute lymphoblastic leukemia cells harboring the NSD2 gain of function mutation p.E1099K, resulting in growth suppression, apoptosis, and reversal of drug resistance. The primary amine of UNC8732 is metabolized to an aldehyde species, which engages C326 of FBXO22 in a covalent and reversible manner to recruit the SCFFBXO22 Cullin complex. We further demonstrate that a previously reported alkyl amine-containing degrader targeting XIAP is similarly dependent on SCFFBXO22. Overall, we present a highly potent NSD2 degrader for the exploration of NSD2 disease phenotypes and a novel FBXO22-dependent TPD strategy.\n",
  "wordCount" : "2176",
  "inLanguage": "en",
  "datePublished": "2023-11-08T10:39:23Z",
  "dateModified": "2023-11-08T10:39:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 8, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565167">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565167" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565167">
        <p class="paperTitle">De novo design of allosterically switchable protein assemblies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565167" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565167" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pillai, A.; Idris, A.; Philomin, A.; Weidle, C.; Skotheim, R.; Leung, P. J. Y.; Broerman, A.; Demakis, C.; Borst, A. J.; Praetorius, F.; Baker, D.</p>
        <p class="info">Score: 32.7, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565167' target='https://doi.org/10.1101/2023.11.01.565167'> 10.1101/2023.11.01.565167</a></p>
        <p class="abstract">Allosteric modulation of protein function, wherein the binding of an effector to a protein triggers conformational changes at distant functional sites, plays a central role in the control of metabolism and cell signaling1-3. There has been considerable interest in designing allosteric systems, both to gain insight into the mechanisms underlying such &#34;action at a distance&#34; modulation and to create synthetic proteins whose functions can be regulated by effectors4-7. However, emulating the subtle conformational changes distributed across many residues, characteristic of natural allosteric proteins, is a significant challenge8,9. Here, inspired by the classic Monod-Changeux-Wyman model of cooperativity10, we investigate the de novo design of allostery through rigid-body coupling of designed effector-switchable hinge modules11 to protein interfaces12 that direct the formation of alternative oligomeric states. We find that this approach can be used to generate a wide variety of allosterically switchable systems, including cyclic rings that incorporate or eject subunits in response to effector binding and dihedral cages that undergo effector-induced disassembly. Size-exclusion chromatography, mass photometry13, and electron microscopy reveal that these designed allosteric protein assemblies closely resemble the design models in both the presence and absence of effectors and can have ligand-binding cooperativity comparable to classic natural systems such as hemoglobin14. Our results indicate that allostery can arise from global coupling of the energetics of protein substructures without optimized sidechain-sidechain allosteric communication pathways and provide a roadmap for generating allosterically triggerable delivery systems, protein nanomachines, and cellular feedback control circuitry.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564067">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564067" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564067">
        <p class="paperTitle">Pervasive aggregation and depletion of host and viral proteins in response to cysteine-reactive electrophilic compounds</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564067" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564067" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Julio, A. R.; Shikwana, F.; Truong, C.; Burton, N. R.; Dominguez, E.; Turmon, A.; Cao, J.; Backus, K.</p>
        <p class="info">Score: 27.0, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564067' target='https://doi.org/10.1101/2023.10.30.564067'> 10.1101/2023.10.30.564067</a></p>
        <p class="abstract">Protein homeostasis is tightly regulated, with damaged or misfolded proteins quickly eliminated by the proteasome and autophagosome pathways. By co-opting these processes, targeted protein degradation technologies enable pharmacological manipulation of protein abundance. Recently, cysteine-reactive molecules have been added to the degrader toolbox, which offer the benefit of unlocking the therapeutic potential of undruggable protein targets. The proteome-wide impact of these molecules remains to be fully understood and given the general reactivity of many classes of cysteine-reactive electrophiles, on- and off-target effects are likely. Using chemical proteomics, we identified a cysteine-reactive small molecule degrader of the SARS-CoV-2 non-structural protein 14 (nsp14), which effects degradation through direct modification of cysteines in both nsp14 and in host chaperones together with activation of global cell stress response pathways. We find that cysteine-reactive electrophiles increase global protein ubiquitylation, trigger proteasome activation, and result in widespread aggregation and depletion of host proteins, including components of the nuclear pore complex. Formation of stress granules was also found to be a remarkably ubiquitous cellular response to nearly all cysteine-reactive compounds and degraders. Collectively, our study sheds light on complexities of covalent target protein degradation and highlights untapped opportunities in manipulating and characterizing proteostasis processes via deciphering the cysteine-centric regulation of stress response pathways.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565201">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565201" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565201">
        <p class="paperTitle">Small-molecule binding and sensing with a designed protein family</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565201" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565201" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, G. R.; Pellock, S. J.; Norn, C.; Tischer, D.; Dauparas, J.; Anishchenko, I.; Mercer, J. A. M.; Kang, A.; Bera, A.; Nguyen, H.; Goreshnik, I.; Vafeados, D.; Roullier, N.; Han, H. L.; Coventry, B.; Haddox, H. K.; Liu, D. R.; Yeh, A. H.-W.; Baker, D.</p>
        <p class="info">Score: 22.7, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565201' target='https://doi.org/10.1101/2023.11.01.565201'> 10.1101/2023.11.01.565201</a></p>
        <p class="abstract">Despite transformative advances in protein design with deep learning, the design of small-molecule-binding proteins and sensors for arbitrary ligands remains a grand challenge. Here we combine deep learning and physics-based methods to generate a family of proteins with diverse and designable pocket geometries, which we employ to computationally design binders for six chemically and structurally distinct small-molecule targets. Biophysical characterization of the designed binders revealed nanomolar to low micromolar binding affinities and atomic-level design accuracy. The bound ligands are exposed at one edge of the binding pocket, enabling the de novo design of chemically induced dimerization (CID) systems; we take advantage of this to create a biosensor with nanomolar sensitivity for cortisol. Our approach provides a general method to design proteins that bind and sense small molecules for a wide range of analytical, environmental, and biomedical applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.565018">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.565018" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.565018">
        <p class="paperTitle">Covalent Degrader of the Oncogenic Transcription Factor β-Catenin</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.565018" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.565018" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gowans, F. A.; Forte, N.; Hatcher, J.; Huang, O. W.; Wang, Y.; Poblano, B. E. A.; Wertz, I. E.; Nomura, D. K.</p>
        <p class="info">Score: 18.3, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.565018' target='https://doi.org/10.1101/2023.10.31.565018'> 10.1101/2023.10.31.565018</a></p>
        <p class="abstract">{beta}-catenin (CTNNB1) is an oncogenic transcription factor that is important in cell-cell adhesion and transcription of cell proliferation and survival genes that drives the pathogenesis of many different types of cancers. However, direct pharmacological targeting of CTNNB1 has remained challenging deeming this transcription factor as &#34;undruggable.&#34; Here, we have performed a screen with a library of cysteine-reactive covalent ligands to identify a monovalent degrader EN83 that depletes CTNNB1 in a ubiquitin-proteasome-dependent manner. We show that EN83 directly and covalently targets CTNNB1 through targeting four distinct cysteines within the armadillo repeat domain--C439, C466, C520, and C619--leading to a destabilization of CTNNB1. Using covalent chemoproteomic approaches, we show that EN83 directly engages CTNNB1 in cells with a moderate degree of selectivity. We further demonstrate that direct covalent targeting of three of these four cysteines--C466, C520, and C619--in cells contributes to CTNNB1 degradation in cells. We also demonstrate that EN83 can be further optimized to yield more potent CTNNB1 binders and degraders. Our results show that chemoproteomic approaches can be used to covalently target and degrade challenging transcription factors like CTNNB1 through a destabilization-mediated degradation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.04.565607">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.04.565607" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.04.565607">
        <p class="paperTitle">Electric fields determine carbapenemase activity in class A β-lactamases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.04.565607" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.04.565607" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jabeen, H.; Beer, M.; Spencer, J.; van der Kamp, M. W.; Bunzel, H. A.; Mulholland, A. J.</p>
        <p class="info">Score: 10.4, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.04.565607' target='https://doi.org/10.1101/2023.11.04.565607'> 10.1101/2023.11.04.565607</a></p>
        <p class="abstract">Antimicrobial resistance is a public health crisis. Limited understanding of the catalytic drivers in resistance-mediating enzymes such as {beta}-lactamases hinders our ability to combat this crisis. Here, we dissect the catalytic contributions of active-site electric fields in class A {beta}-lactamases. We studied the enzymatic hydrolysis of a carbapenem antibiotic by QM/MM molecular dynamics simulations and quantified active-site fields with a custom-made script. We discovered that the fields correlate well with activity and identified seven positions, some distal, that distinguish efficient carbapenemases. Electric-field analysis may help predict the activity of {beta}-lactamases and guide antibiotic and enzyme design. Electric field script: https://www.github.com/bunzela/FieldTools</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.564846">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.564846" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.564846">
        <p class="paperTitle">Design of efficient artificial enzymes using crystallographically-enhanced conformational sampling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.564846" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.564846" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rakotoharisoa, R. V.; Seifinoferest, B.; Zarifi, N.; Miller, J. D. M.; Rodriguez, J. M.; Thompson, M. C.; Chica, R. A.</p>
        <p class="info">Score: 10.0, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.564846' target='https://doi.org/10.1101/2023.11.01.564846'> 10.1101/2023.11.01.564846</a></p>
        <p class="abstract">The ability to create efficient artificial enzymes for any chemical reaction is of great interest. Here, we describe a computational design method for increasing catalytic efficiency of de novo enzymes to a level comparable to their natural counterparts without relying on directed evolution. Using structural ensembles generated from dynamics-based refinement against X-ray diffraction data collected from crystals of Kemp eliminases HG3 (kcat/KM 125 M-1 s-1) and KE70 (kcat/KM 57 M-1 s-1), we design from each enzyme [&amp;le;]10 sequences predicted to catalyze this reaction more efficiently. The most active designs display kcat/KM values improved by 100-250-fold, comparable to mutants obtained after screening thousands of variants in multiple rounds of directed evolution. Crystal structures show excellent agreement with computational models. Our work shows how computational design can generate efficient artificial enzymes by exploiting the true conformational ensemble to more effectively stabilize the transition state.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561576">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561576" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561576">
        <p class="paperTitle">Proximity proteomics reveals UCH-L1 as an NLRP3 interactor that modulates IL-1β production in human macrophages and microglia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561576" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561576" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liang, Z.; Damianou, A.; Vendrell, I.; Jenkins, E.; Lassen, F. H.; Washer, S. J.; Liu, G.; Yi, G.; Lou, H.; Cao, F.; Zheng, X.; Fernandes, R.; Dong, T.; Tate, E. W.; Daniel, E. D.; Kessler, B. M.</p>
        <p class="info">Score: 6.5, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561576' target='https://doi.org/10.1101/2023.10.09.561576'> 10.1101/2023.10.09.561576</a></p>
        <p class="abstract">Activation of the NACHT, LRR family pyrin domain containing 3 (NLRP3) inflammasome complex is an essential innate immune signalling mechanism. To reveal how human NLRP3 inflammasome assembly and activation are controlled, in particular by components of the ubiquitin system, proximity labelling, affinity purification and RNAi screening approaches were performed. Our study provides an intricate time-resolved molecular map of different phases of NLRP3 inflammasome activation. Also, we show that ubiquitin C-terminal hydrolase 1 (UCH-L1) interacts with the NACHT domain of NLRP3. Downregulation of UCH-L1 decreases pro-IL-1{beta} levels. UCH-L1 chemical inhibition with small molecules interfered with NLRP3 puncta formation and ASC oligomerization, leading to altered IL-1{beta} cleavage and secretion, particularly in microglia cells, which exhibited elevated UCH-L1 expression as compared to monocytes/macrophages. Altogether, we profiled NLRP3 inflammasome activation dynamics and highlight UCH-L1 as an important modulator of NLRP3-mediated IL-1{beta} production, suggesting that a pharmacological inhibitor of UCH-L1 may decrease inflammation-associated pathologies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.04.565616">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.04.565616" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.04.565616">
        <p class="paperTitle">A minimal sequence motif drives selective tRNA dihydrouridylation by hDUS2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.04.565616" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.04.565616" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ji, J.; Yu, N. J.; Kleiner, R. E.</p>
        <p class="info">Score: 4.5, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.04.565616' target='https://doi.org/10.1101/2023.11.04.565616'> 10.1101/2023.11.04.565616</a></p>
        <p class="abstract">The post-transcriptional reduction of uridine to dihydrouridine (D) by dihydrouridine synthase (DUS) enzymes is among the most ubiquitous transformations in RNA biology. D is found at multiple sites in tRNAs and studies in yeast have proposed that each of the four eukaryotic DUS enzymes modifies a different site, however the molecular basis for this exquisite selectivity is unknown and human DUS enzymes have remained largely uncharacterized. Here we investigate the substrate specificity of human dihydrouridine synthase 2 (hDUS2) using mechanism-based crosslinking with 5-bromouridine (5-BrUrd)-modified oligonucleotide probes and in vitro dihydrouridylation assays. We find that hDUS2 modifies U20 in the D loop of diverse tRNA substrates and identify a minimal GU motif within the tRNA tertiary fold required for directing its activity. Further, we use our mechanism-based platform to screen small molecule inhibitors of hDUS2, a potential anti-cancer target. Our work elucidates the principles of substrate modification by a conserved DUS and provides a general platform to studying RNA modifying enzymes with sequence-defined activity-based probes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561603">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561603" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561603">
        <p class="paperTitle">Generalized Biomolecular Modeling and Design with RoseTTAFold All-Atom</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561603" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561603" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krishna, R.; Wang, J.; Ahern, W.; Sturmfels, P.; Venkatesh, P.; Kalvet, I.; Lee, G. R.; Morey-Burrows, F. S.; Anishchenko, I.; Humphreys, I. R.; McHugh, R.; Vafeados, D.; Li, X.; Sutherland, G. A.; Hitchcock, A.; Hunter, C. N.; Baek, M.; DiMaio, F.; Baker, D.</p>
        <p class="info">Score: 347.5, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561603' target='https://doi.org/10.1101/2023.10.09.561603'> 10.1101/2023.10.09.561603</a></p>
        <p class="abstract">Although AlphaFold2 (AF2) and RoseTTAFold (RF) have transformed structural biology by enabling high-accuracy protein structure modeling, they are unable to model covalent modifications or interactions with small molecules and other non-protein molecules that can play key roles in biological function. Here, we describe RoseTTAFold All-Atom (RFAA), a deep network capable of modeling full biological assemblies containing proteins, nucleic acids, small molecules, metals, and covalent modifications given the sequences of the polymers and the atomic bonded geometry of the small molecules and covalent modifications. Following training on structures of full biological assemblies in the Protein Data Bank (PDB), RFAA has comparable protein structure prediction accuracy to AF2, excellent performance in CAMEO for flexible backbone small molecule docking, and reasonable prediction accuracy for protein covalent modifications and assemblies of proteins with multiple nucleic acid chains and small molecules which, to our knowledge, no existing method can model simultaneously. By fine-tuning on diffusive denoising tasks, we develop RFdiffusion All-Atom (RFdiffusionAA), which generates binding pockets by directly building protein structures around small molecules and other non-protein molecules. Starting from random distributions of amino acid residues surrounding target small molecules, we design and experimentally validate proteins that bind the cardiac disease therapeutic digoxigenin, the enzymatic cofactor heme, and optically active bilin molecules with potential for expanding the range of wavelengths captured by photosynthesis. We anticipate that RFAA and RFdiffusionAA will be widely useful for modeling and designing complex biomolecular systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.564830">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.564830" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.564830">
        <p class="paperTitle">Recruitment of FBXO22 for Targeted Degradation of NSD2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.564830" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.564830" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nie, D. Y.; Tabor, J. R.; Li, J.; Kutera, M.; St-Germain, J.; Hanley, R. P.; WOLF, E. Z.; Paulakonis, E.; Kenney, T. M. G.; Duan, S.; Shrestha, S.; Owens, D. D.; Pon, A.; Szewczyk, M.; Lamberto, A. J.; Menes, M.; Li, F.; Barsyte-Lovejoy, D.; Brown, N. G.; Barsotti, A. M.; Stamford, A. W.; Collins, J. L.; Wilson, D. J.; Raught, B.; Licht, J.; James, L.; Arrowsmith, C.</p>
        <p class="info">Score: 4.0, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.564830' target='https://doi.org/10.1101/2023.11.01.564830'> 10.1101/2023.11.01.564830</a></p>
        <p class="abstract">Targeted protein degradation (TPD) is an emerging therapeutic strategy that would benefit from new chemical entities with which to recruit a wider variety of ubiquitin E3 ligases to target proteins for proteasomal degradation. Here, we describe a TPD strategy involving the recruitment of FBXO22 to induce degradation of the histone methyltransferase and oncogene NSD2. UNC8732 facilitates FBXO22-mediated degradation of NSD2 in acute lymphoblastic leukemia cells harboring the NSD2 gain of function mutation p.E1099K, resulting in growth suppression, apoptosis, and reversal of drug resistance. The primary amine of UNC8732 is metabolized to an aldehyde species, which engages C326 of FBXO22 in a covalent and reversible manner to recruit the SCFFBXO22 Cullin complex. We further demonstrate that a previously reported alkyl amine-containing degrader targeting XIAP is similarly dependent on SCFFBXO22. Overall, we present a highly potent NSD2 degrader for the exploration of NSD2 disease phenotypes and a novel FBXO22-dependent TPD strategy.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
