<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Mechanism of degrader-targeted protein ubiquitinability
Authors: Crowe, C.; Nakasone, M.; Chandler, S.; Tatham, M. H.; Makukhin, N.; Hay, R. T.; Ciulli, A.
Score: 59.7, Published: 2024-02-05 DOI: 10.1101/2024.02.05.578957
Small molecule degraders of disease-driving proteins offer a clinically proven modality with enhanced therapeutic efficacy and the potential to tackle previously undrugged targets. Thermodynamically stable and kinetically long-lived degrader-mediated ternary complexes can drive faster, more profound and durable target degradation, however the mechanistic features by which they impact on target ubiquitination remain elusive.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Mechanism of degrader-targeted protein ubiquitinability
Authors: Crowe, C.; Nakasone, M.; Chandler, S.; Tatham, M. H.; Makukhin, N.; Hay, R. T.; Ciulli, A.
Score: 59.7, Published: 2024-02-05 DOI: 10.1101/2024.02.05.578957
Small molecule degraders of disease-driving proteins offer a clinically proven modality with enhanced therapeutic efficacy and the potential to tackle previously undrugged targets. Thermodynamically stable and kinetically long-lived degrader-mediated ternary complexes can drive faster, more profound and durable target degradation, however the mechanistic features by which they impact on target ubiquitination remain elusive." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-14T10:38:48+00:00" />
<meta property="article:modified_time" content="2024-02-14T10:38:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Mechanism of degrader-targeted protein ubiquitinability
Authors: Crowe, C.; Nakasone, M.; Chandler, S.; Tatham, M. H.; Makukhin, N.; Hay, R. T.; Ciulli, A.
Score: 59.7, Published: 2024-02-05 DOI: 10.1101/2024.02.05.578957
Small molecule degraders of disease-driving proteins offer a clinically proven modality with enhanced therapeutic efficacy and the potential to tackle previously undrugged targets. Thermodynamically stable and kinetically long-lived degrader-mediated ternary complexes can drive faster, more profound and durable target degradation, however the mechanistic features by which they impact on target ubiquitination remain elusive."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Mechanism of degrader-targeted protein ubiquitinability\nAuthors: Crowe, C.; Nakasone, M.; Chandler, S.; Tatham, M. H.; Makukhin, N.; Hay, R. T.; Ciulli, A.\nScore: 59.7, Published: 2024-02-05 DOI: 10.1101/2024.02.05.578957\nSmall molecule degraders of disease-driving proteins offer a clinically proven modality with enhanced therapeutic efficacy and the potential to tackle previously undrugged targets. Thermodynamically stable and kinetically long-lived degrader-mediated ternary complexes can drive faster, more profound and durable target degradation, however the mechanistic features by which they impact on target ubiquitination remain elusive.",
  "keywords": [
    
  ],
  "articleBody": " Mechanism of degrader-targeted protein ubiquitinability\nAuthors: Crowe, C.; Nakasone, M.; Chandler, S.; Tatham, M. H.; Makukhin, N.; Hay, R. T.; Ciulli, A.\nScore: 59.7, Published: 2024-02-05 DOI: 10.1101/2024.02.05.578957\nSmall molecule degraders of disease-driving proteins offer a clinically proven modality with enhanced therapeutic efficacy and the potential to tackle previously undrugged targets. Thermodynamically stable and kinetically long-lived degrader-mediated ternary complexes can drive faster, more profound and durable target degradation, however the mechanistic features by which they impact on target ubiquitination remain elusive. Here, we solve cryo-EM structures of the VHL Cullin 2 RING E3 ligase complexed with degrader MZ1, target protein Brd4BD2 and primed for catalysis with its cognate E2-ubiquitin bound. We find that Brd4BD2 adopts a favourable orientation towards the E2 active site. In vitro ubiquitination coupled with mass spectrometry illuminates a patch of ubiquitinable lysines on one face of Brd4BD2, with Lys456 showing optimal distance and geometry for nucleophilic attack. Our results demonstrate the proficiency of MZ1 in directing the substrate towards catalysis, explains the favourability of Brd4BD2 for ubiquitination, and reveals the flexibility of the enzyme in capturing sub-optimal lysines. We propose a model for ubiquitinability of degrader-recruited targets that provides a mechanistic blueprint for further rational drug design and optimization. One-Sentence SummaryStructural assembly a PROTAC-mediated complex of whole Cullin RING E3 ligase with bound target and E2-ubiquitin reveals structural and mechanistic insights of specificity for target protein ubiquitination.\nStructural basis for the inhibition of PRC2 by active transcription histone posttranslational modifications\nAuthors: Cookis, T.; Lydecker, A.; Sauer, P. V.; Kasinath, V.; Nogales, E.\nScore: 25.5, Published: 2024-02-10 DOI: 10.1101/2024.02.09.579730\nPolycomb repressive complex 2 (PRC2) is an epigenetic regulator essential for embryonic development and maintenance of cell identity that trimethylates histone H3 at lysine 27 (H3K27me3) leading to gene silencing. PRC2 is regulated by association with protein cofactors and crosstalk with histone posttranslational modifications. Trimethylated histone H3 K4 (H3K4me3) and K36 (H3K36me3) localize to sites of active transcription where H3K27me3 is absent and inhibit PRC2 activity through unknown mechanisms. Using cryo-electron microscopy we reveal that histone H3 tails modified with H3K36me3 engage poorly with the PRC2 active site and preclude its effective interaction with chromatin, while the H3K4me3 modification binds to the allosteric site in the EED subunit, acting as an antagonist that competes with allosteric activators required for the spreading of the H3K27me3 repressive mark. Thus, the location along the H3 tail of the H3K4me3 and H3K36me3 modifications allow them to target two essential requirements for efficient trimethylation of histone H3K27. We further show that the JARID2 cofactor modulates PRC2 activity in the presence of these histone modifications.\nPersonalized Molecular Signatures of Insulin Resistance and Type 2 Diabetes\nAuthors: Larsen, J.; Stocks, B.; Henderson, J.; Andersson, D.; Backdahl, J.; Eriksson-Hogling, D.; Stidsen, J.; Sakamoto, K.; Hojlund, K.; Ryden, M.; Zierath, J. R.; Krook, A.; Deshmukh, A. S.\nScore: 20.2, Published: 2024-02-07 DOI: 10.1101/2024.02.06.578994\nHighlightsO_LIAdvanced proteomics analysis reveals personalized signatures of insulin resistance C_LIO_LIFasting muscle proteome and phosphoproteome predicts whole-body insulin sensitivity C_LIO_LIInsulin-stimulated phosphoproteome reveals selective insulin resistance signatures C_LIO_LIPhosphoproteome and proteome atlas explains sex-specific muscle metabolism C_LI O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=143 SRC=\"FIGDIR/small/578994v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (35K): org.highwire.dtl.DTLVardef@1d0cf48org.highwire.dtl.DTLVardef@1d88607org.highwire.dtl.DTLVardef@47533corg.highwire.dtl.DTLVardef@a9a3fb_HPS_FORMAT_FIGEXP M_FIG O_FLOATNOGraphical AbstractC_FLOATNO C_FIG Insulin resistance is a hallmark of type 2 diabetes, which is a highly heterogeneous disease with diverse pathology. Understanding the molecular signatures of insulin resistance and its association with individual phenotypic traits is crucial for advancing precision medicine in type 2 diabetes. Utilizing cutting-edge proteomics technology, we mapped the proteome and phosphoproteome of skeletal muscle from \u003e120 men and women with normal glucose tolerance or type 2 diabetes, with varying degrees of insulin sensitivity. Leveraging deep in vivo phenotyping, we reveal that fasting proteome and phosphoproteome signatures strongly predict insulin sensitivity. Furthermore, the insulin-stimulated phosphoproteome revealed both dysregulated and preserved signaling nodes - even in individuals with severe insulin resistance. While substantial sex-specific differences in the proteome and phosphoproteome were identified, molecular signatures of insulin resistance remained largely similar between men and women. These findings underscore the need for precision medicine approaches in type 2 diabetes care, acknowledging disease heterogeneity.\nSTREAMLINED PROTEOME-WIDE IDENTIFICATION OF DRUG TARGETS INDICATES ORGAN-SPECIFIC ENGAGEMENT\nAuthors: Batth, T. S.; Locard-Paulet, M.; Doncheva, N. T.; Lopez-Mendez, B.; Jensen, L. J.; Olsen, J. V.\nScore: 14.2, Published: 2024-02-08 DOI: 10.1101/2024.02.08.578880\nProteins are the primary targets of almost all small molecule drugs. However, even the most selectively designed drugs can potentially target several unknown proteins. Identification of potential drug targets can facilitate design of new drugs and repurposing of existing ones. Current state-of-the-art proteomics methodologies enable screening of thousands of proteins against a limited number of drug molecules. Here we report the development of a label-free quantitative proteomics approach that enables proteome-wide screening of small organic molecules in a scalable, reproducible, and rapid manner by streamlining the proteome integral solubility alteration (PISA) assay. We used rat organs ex-vivo to determine organ specific targets of medical drugs and enzyme inhibitors to identify novel drug targets for common drugs such as Ibuprofen. Finally, global drug profiling revealed overarching trends of how small molecules affect the proteome through either direct or indirect protein interactions.\nStructural characterization of the ACDC domain from ApiAP2 proteins of the malaria parasite\nAuthors: Le Berre, M.; Tubiana, T.; Reutersward Waldner, P.; Lazar, N.; Gallay Li de la Sierra, I.; Santos, J. M.; Llinas, M.; Nessler, S.\nScore: 6.7, Published: 2024-02-10 DOI: 10.1101/2024.02.09.579679\nThe Apicomplexan AP2 (ApiAP2) proteins are the best characterized family of DNA-binding proteins in the malaria parasite. Apart from the AP2 DNA-binding domain, there is little sequence similarity between ApiAP2 proteins and no other functional domains have been extensively characterized. One protein domain, which is present in a subset of the ApiAP2 proteins, is the conserved AP2-coincident domain mostly at the C-terminus (ACDC domain). Here we solved for the first time the crystal structure of the ACDC domain from two distinct Plasmodium falciparum ApiAP2 proteins and one orthologue from P. vivax, revealing a non-canonical four-helix bundle. Despite little sequence conservation between the ACDC domains from the two proteins, the structures are remarkably similar and do not resemble that of any other known protein domains. Due to their unique protein architecture and lack of homologues in the human genome, we performed in silico docking calculations against a library of known antimalarial compounds and we identified a small molecule that can potentially bind to any Apicomplexan ACDC domain within a pocket highly conserved amongst ApiAP2 proteins. Inhibitors based on this compound would disrupt the function of the ACDC domain and thus of the ApiAP2 proteins containing it, providing a new therapeutic window for targeting the malaria parasite and other Apicomplexans.\nBiochemical fractionation of human α-Synuclein in a Drosophila model of synucleinopathies\nAuthors: Imomnazarov, K.; Lopez-Scarim, J.; Bagheri, I.; Joers, V.; Tansey, M. G.; Martin-Pena, A.\nScore: 4.5, Published: 2024-02-13 DOI: 10.1101/2024.02.05.579034\nSynucleinopathies are a group of central nervous system pathologies that are characterized by neuronal accumulation of misfolded and aggregated -synuclein in proteinaceous depositions known as Lewy Bodies (LBs). The transition of -synuclein from its physiological to pathological form has been associated with several post-translational modifications such as phosphorylation and an increasing degree of insolubility, which also correlate with disease progression in post-mortem specimens from human patients. Neuronal expression of -synuclein in model organisms, including Drosophila melanogaster, has been a typical approach employed to study its physiological effects. Biochemical analysis of -synuclein solubility via high-speed ultracentrifugation with buffers of increasing detergent strength offers a potent method for identification of -synuclein biochemical properties and the associated pathology stage. Unfortunately, the development of a robust and reproducible method for evaluation of human -synuclein solubility isolated from Drosophila tissues has remained elusive. Here, we tested different detergents for their ability to solubilize human -synuclein carrying the pathological mutation A53T from brains of aged flies. We also assessed the effect of sonication on solubility of human -synuclein and optimized a protocol to discriminate relative amounts of soluble/insoluble human -synuclein from dopaminergic neurons of the Drosophila brain. Our data established that, using a 5% SDS buffer, the 3-step protocol distinguishes between cytosolic soluble proteins in fraction 1, detergent-soluble proteins in fraction 2 and insoluble proteins in fraction 3. This protocol shows that sonication breaks down -synuclein insoluble complexes from the fly brain, making them soluble in the SDS buffer and enriching fraction 2 of the protocol.\nLung Mycobacterium tuberculosis infection perturbs metabolic pathways in non-pulmonary tissues\nAuthors: Pahwa, F.; Chaudhary, S.; Dayal, A.; Nanda, R.\nScore: 3.9, Published: 2024-02-09 DOI: 10.1101/2024.02.09.579656\nMycobacterium tuberculosis (Mtb), through aerosol, reaches the lungs to cause pulmonary tuberculosis (TB); however, it may also affect the metabolism of other tissues in age-specific ways. In this study, female C57BL/6 mice (2 and 5 months old; M) were infected with a low aerosol dose (100-200 cfu) of Mtb H37Rv to monitor tissue mycobacterial load and multi-tissue metabolite profiling using gas chromatography and mass spectrometry (GC-MS). 5M C57BL/6 mice showed separate tissue metabolic phenotype with significantly higher lung aspartic acid, fecal oxalic acid and tryptophan levels with lower liver lysine and aspartic acid and fecal phenylalanine levels (log2FC: 5M/2M\u003e {+/-}1.0, p\u003c0.1) compared to 2M young controls. Upon Mtb infection, the lung mycobacterial load of 2M and 5M mice were similar till 6 weeks post-infection. However, significantly higher lung phosphoric acid, malonic acid and lower mannose levels (log2FC: Mtb infected/healthy\u003e {+/-}1.0, p\u003c0.1) were observed in Mtb-infected 5M C57BL/6 mice. Meanwhile, Mtb-infected 2M mice showed higher liver xylose and lower lysine levels. The thigh muscles of Mtb-infected 2M and 5M mice showed increased malic acid and oxalic acid and decreased glycine, serine, and glycerol levels. Fecal aspartic acid level was higher in Mtb-infected 5M mice, while a decreased abundance of fecal lysine was observed in Mtb-infected 2M mice. Overall, this study demonstrates a deregulated tissue-specific amino acid metabolism in Mtb-infected mice groups of different age groups, which might be targeted for managing TB infection-related adverse effects.\nSARS-CoV-2 Omicron lineage XBB spike structures, conformations, antigenicity, and receptor recognition\nAuthors: Zhang, Q. E.; Lindenberger, J.; Parsons, R.; Thakur, B.; Parks, R.; Park, C. S.; Huang, X.; Sammour, S.; Janowska, K.; Spence, T. N.; Edwards, R. J.; Martin, M.; Williams, W. B.; Gobeil, S.; Montefiori, D. C.; Korber, B.; Saunders, K. O.; Haynes, B. F.; Haynes, B. F.; Henderson, R.; Acharya, P.\nScore: 3.6, Published: 2024-02-13 DOI: 10.1101/2024.02.12.580004\nA recombinant lineage of the SARS-CoV-2 Omicron variant, named XBB, appeared in late 2022 and evolved descendants that successively swept local and global populations. XBB lineage members were noted for their improved immune evasion and transmissibility. Here, we determine cryo-EM structures of XBB.1.5, XBB.1.16 and EG.5 spike (S) ectodomains to reveal enhanced occupancy of the receptor inaccessible closed state. Interprotomer receptor binding domain (RBD) interactions previously observed in BA.1 and BA.2 were retained to reinforce the 3-RBD-down state. Improved stability of XBB.1.5 and XBB.1.16 RBD compensated for loss of stability caused by early Omicron mutations, while the F456L substitution reduced EG.5 RBD stability. Long-range impacts of S1 subunit mutations affected conformation and epitope presentation in the S2 subunit. Taken together, our results feature a theme of iterative optimization of S protein stability as Omicron continues to evolve, while maintaining high affinity receptor binding and bolstering immune evasion.\nStructural insights into CXCR4 modulation and oligomerization\nAuthors: Saotome, K.; McGoldrick, L. L.; Ho, J.-H.; Ramlall, T. F.; Shah, S.; Moore, M. J.; Kim, J. H.; Leidich, R.; Olson, W. C.; Franklin, M. C.\nScore: 3.4, Published: 2024-02-10 DOI: 10.1101/2024.02.09.579708\nActivation of the chemokine receptor CXCR4 by its chemokine ligand CXCL12 regulates diverse cellular processes. CXCR4 also serves as a key target for diseases such as cancer and HIV. Previously reported crystal structures of CXCR4 bound to antagonists revealed the architecture of an inactive, homodimeric receptor. However, many structural aspects of CXCR4 remain poorly understood, including its activation by CXCL12, as well as its assembly into higher-order oligomers. Here, we use cryoelectron microscopy (cryoEM) to investigate various modes of CXCR4 regulation in the presence and absence of Gi protein. CXCL12 activates CXCR4 by inserting its N-terminus deep into the CXCR4 orthosteric pocket. The binding of FDA-approved antagonist AMD3100 is stabilized by electrostatic interactions with acidic residues in the 7 transmembrane helix bundle. A potent antibody blocker, REGN7663, binds across the extracellular face of CXCR4 and inserts its CDR-H3 loop into the orthosteric pocket. Trimeric and tetrameric structures of CXCR4 reveal, to our knowledge, previously undescribed modes of GPCR oligomerization. Remarkably, CXCR4 adopts distinct subunit conformations in trimeric and tetrameric assemblies, highlighting how oligomerization could allosterically regulate chemokine receptor function.\nSpatial top-down proteomics for the functional characterization of human kidney\nAuthors: Zemaitis, K. J.; Fulcher, J. M.; Kumar, R.; Degnan, D. J.; Lewis, L. A.; Liao, Y.-C.; Velickovic, M.; Williams, S. M.; Moore, R. J.; Bramer, L. M.; Velickovic, D.; Zhu, Y.; Zhou, M.; Pasa-Tolic, L.\nScore: 3.2, Published: 2024-02-13 DOI: 10.1101/2024.02.13.580062\nBackground: The Human Proteome Project has credibly detected nearly 93% of the roughly 20,000 proteins which are predicted by the human genome. However, the proteome is enigmatic, where alterations in amino acid sequences from polymorphisms and alternative splicing, errors in translation, and post-translational modifications result in a proteome depth estimated at several million unique proteoforms. Recently mass spectrometry has been demonstrated in several landmark efforts mapping the human proteoform landscape in bulk analyses. Herein, we developed an integrated workflow for characterizing proteoforms from human tissue in a spatially resolved manner by coupling laser capture microdissection, nanoliter-scale sample preparation, and mass spectrometry imaging. Results: Using healthy human kidney sections as the case study, we focused our analyses on the major functional tissue units including glomeruli, tubules, and medullary rays. After laser capture microdissection, these isolated functional tissue units were processed with microPOTS (microdroplet processing in one-pot for trace samples) for sensitive top-down proteomics measurement. This provided a quantitative database of 616 proteoforms that was further leveraged as a library for mass spectrometry imaging with near-cellular spatial resolution over the entire section. Notably, several mitochondrial proteoforms were found to be differentially abundant between glomeruli and convoluted tubules, and further spatial contextualization was provided by mass spectrometry imaging confirming unique differences identified by microPOTS, and further expanding the field-of-view for unique distributions such as enhanced abundance of a truncated form (1-74) of ubiquitin within cortical regions. Conclusions: We developed an integrated workflow to directly identify proteoforms and reveal their spatial distributions. Where of the 20 differentially abundant proteoforms identified as discriminate between tubules and glomeruli by microPOTS, the vast majority of tubular proteoforms were of mitochondrial origin (8 of 10) where discriminate proteoforms in glomeruli were primarily hemoglobin subunits (9 of 10). These trends were also identified within ion images demonstrating spatially resolved characterization of proteoforms that has the potential to reshape discovery-based proteomics because the proteoforms are the ultimate effector of cellular functions. Applications of this technology have the potential to unravel etiology and pathophysiology of disease states, informing on biologically active proteoforms, which remodel the proteomic landscape in chronic and acute disorders.\n",
  "wordCount" : "2477",
  "inLanguage": "en",
  "datePublished": "2024-02-14T10:38:48Z",
  "dateModified": "2024-02-14T10:38:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 14, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.05.578957">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.05.578957" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.05.578957">
        <p class="paperTitle">Mechanism of degrader-targeted protein ubiquitinability</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.05.578957" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.05.578957" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Crowe, C.; Nakasone, M.; Chandler, S.; Tatham, M. H.; Makukhin, N.; Hay, R. T.; Ciulli, A.</p>
        <p class="info">Score: 59.7, Published: 2024-02-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.05.578957' target='https://doi.org/10.1101/2024.02.05.578957'> 10.1101/2024.02.05.578957</a></p>
        <p class="abstract">Small molecule degraders of disease-driving proteins offer a clinically proven modality with enhanced therapeutic efficacy and the potential to tackle previously undrugged targets. Thermodynamically stable and kinetically long-lived degrader-mediated ternary complexes can drive faster, more profound and durable target degradation, however the mechanistic features by which they impact on target ubiquitination remain elusive. Here, we solve cryo-EM structures of the VHL Cullin 2 RING E3 ligase complexed with degrader MZ1, target protein Brd4BD2 and primed for catalysis with its cognate E2-ubiquitin bound. We find that Brd4BD2 adopts a favourable orientation towards the E2 active site. In vitro ubiquitination coupled with mass spectrometry illuminates a patch of ubiquitinable lysines on one face of Brd4BD2, with Lys456 showing optimal distance and geometry for nucleophilic attack. Our results demonstrate the proficiency of MZ1 in directing the substrate towards catalysis, explains the favourability of Brd4BD2 for ubiquitination, and reveals the flexibility of the enzyme in capturing sub-optimal lysines. We propose a model for ubiquitinability of degrader-recruited targets that provides a mechanistic blueprint for further rational drug design and optimization.

One-Sentence SummaryStructural assembly a PROTAC-mediated complex of whole Cullin RING E3 ligase with bound target and E2-ubiquitin reveals structural and mechanistic insights of specificity for target protein ubiquitination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579730">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579730" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579730">
        <p class="paperTitle">Structural basis for the inhibition of PRC2 by active transcription histone posttranslational modifications</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579730" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579730" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cookis, T.; Lydecker, A.; Sauer, P. V.; Kasinath, V.; Nogales, E.</p>
        <p class="info">Score: 25.5, Published: 2024-02-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579730' target='https://doi.org/10.1101/2024.02.09.579730'> 10.1101/2024.02.09.579730</a></p>
        <p class="abstract">Polycomb repressive complex 2 (PRC2) is an epigenetic regulator essential for embryonic development and maintenance of cell identity that trimethylates histone H3 at lysine 27 (H3K27me3) leading to gene silencing. PRC2 is regulated by association with protein cofactors and crosstalk with histone posttranslational modifications. Trimethylated histone H3 K4 (H3K4me3) and K36 (H3K36me3) localize to sites of active transcription where H3K27me3 is absent and inhibit PRC2 activity through unknown mechanisms. Using cryo-electron microscopy we reveal that histone H3 tails modified with H3K36me3 engage poorly with the PRC2 active site and preclude its effective interaction with chromatin, while the H3K4me3 modification binds to the allosteric site in the EED subunit, acting as an antagonist that competes with allosteric activators required for the spreading of the H3K27me3 repressive mark. Thus, the location along the H3 tail of the H3K4me3 and H3K36me3 modifications allow them to target two essential requirements for efficient trimethylation of histone H3K27. We further show that the JARID2 cofactor modulates PRC2 activity in the presence of these histone modifications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.578994">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.578994" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.578994">
        <p class="paperTitle">Personalized Molecular Signatures of Insulin Resistance and Type 2 Diabetes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.578994" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.578994" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Larsen, J.; Stocks, B.; Henderson, J.; Andersson, D.; Backdahl, J.; Eriksson-Hogling, D.; Stidsen, J.; Sakamoto, K.; Hojlund, K.; Ryden, M.; Zierath, J. R.; Krook, A.; Deshmukh, A. S.</p>
        <p class="info">Score: 20.2, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.578994' target='https://doi.org/10.1101/2024.02.06.578994'> 10.1101/2024.02.06.578994</a></p>
        <p class="abstract">HighlightsO_LIAdvanced proteomics analysis reveals personalized signatures of insulin resistance
C_LIO_LIFasting muscle proteome and phosphoproteome predicts whole-body insulin sensitivity
C_LIO_LIInsulin-stimulated phosphoproteome reveals selective insulin resistance signatures
C_LIO_LIPhosphoproteome and proteome atlas explains sex-specific muscle metabolism
C_LI



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=143 SRC=&#34;FIGDIR/small/578994v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (35K):
org.highwire.dtl.DTLVardef@1d0cf48org.highwire.dtl.DTLVardef@1d88607org.highwire.dtl.DTLVardef@47533corg.highwire.dtl.DTLVardef@a9a3fb_HPS_FORMAT_FIGEXP  M_FIG O_FLOATNOGraphical AbstractC_FLOATNO C_FIG Insulin resistance is a hallmark of type 2 diabetes, which is a highly heterogeneous disease with diverse pathology. Understanding the molecular signatures of insulin resistance and its association with individual phenotypic traits is crucial for advancing precision medicine in type 2 diabetes. Utilizing cutting-edge proteomics technology, we mapped the proteome and phosphoproteome of skeletal muscle from &gt;120 men and women with normal glucose tolerance or type 2 diabetes, with varying degrees of insulin sensitivity. Leveraging deep in vivo phenotyping, we reveal that fasting proteome and phosphoproteome signatures strongly predict insulin sensitivity. Furthermore, the insulin-stimulated phosphoproteome revealed both dysregulated and preserved signaling nodes - even in individuals with severe insulin resistance. While substantial sex-specific differences in the proteome and phosphoproteome were identified, molecular signatures of insulin resistance remained largely similar between men and women. These findings underscore the need for precision medicine approaches in type 2 diabetes care, acknowledging disease heterogeneity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.08.578880">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.08.578880" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.08.578880">
        <p class="paperTitle">STREAMLINED PROTEOME-WIDE IDENTIFICATION OF DRUG TARGETS INDICATES ORGAN-SPECIFIC ENGAGEMENT</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.08.578880" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.08.578880" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Batth, T. S.; Locard-Paulet, M.; Doncheva, N. T.; Lopez-Mendez, B.; Jensen, L. J.; Olsen, J. V.</p>
        <p class="info">Score: 14.2, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.08.578880' target='https://doi.org/10.1101/2024.02.08.578880'> 10.1101/2024.02.08.578880</a></p>
        <p class="abstract">Proteins are the primary targets of almost all small molecule drugs. However, even the most selectively designed drugs can potentially target several unknown proteins. Identification of potential drug targets can facilitate design of new drugs and repurposing of existing ones. Current state-of-the-art proteomics methodologies enable screening of thousands of proteins against a limited number of drug molecules. Here we report the development of a label-free quantitative proteomics approach that enables proteome-wide screening of small organic molecules in a scalable, reproducible, and rapid manner by streamlining the proteome integral solubility alteration (PISA) assay. We used rat organs ex-vivo to determine organ specific targets of medical drugs and enzyme inhibitors to identify novel drug targets for common drugs such as Ibuprofen. Finally, global drug profiling revealed overarching trends of how small molecules affect the proteome through either direct or indirect protein interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579679">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579679" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579679">
        <p class="paperTitle">Structural characterization of the ACDC domain from ApiAP2 proteins of the malaria parasite</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579679" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579679" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Le Berre, M.; Tubiana, T.; Reutersward Waldner, P.; Lazar, N.; Gallay Li de la Sierra, I.; Santos, J. M.; Llinas, M.; Nessler, S.</p>
        <p class="info">Score: 6.7, Published: 2024-02-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579679' target='https://doi.org/10.1101/2024.02.09.579679'> 10.1101/2024.02.09.579679</a></p>
        <p class="abstract">The Apicomplexan AP2 (ApiAP2) proteins are the best characterized family of DNA-binding proteins in the malaria parasite. Apart from the AP2 DNA-binding domain, there is little sequence similarity between ApiAP2 proteins and no other functional domains have been extensively characterized. One protein domain, which is present in a subset of the ApiAP2 proteins, is the conserved AP2-coincident domain mostly at the C-terminus (ACDC domain). Here we solved for the first time the crystal structure of the ACDC domain from two distinct Plasmodium falciparum ApiAP2 proteins and one orthologue from P. vivax, revealing a non-canonical four-helix bundle. Despite little sequence conservation between the ACDC domains from the two proteins, the structures are remarkably similar and do not resemble that of any other known protein domains. Due to their unique protein architecture and lack of homologues in the human genome, we performed in silico docking calculations against a library of known antimalarial compounds and we identified a small molecule that can potentially bind to any Apicomplexan ACDC domain within a pocket highly conserved amongst ApiAP2 proteins. Inhibitors based on this compound would disrupt the function of the ACDC domain and thus of the ApiAP2 proteins containing it, providing a new therapeutic window for targeting the malaria parasite and other Apicomplexans.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.05.579034">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.05.579034" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.05.579034">
        <p class="paperTitle">Biochemical fractionation of human α-Synuclein in a Drosophila model of synucleinopathies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.05.579034" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.05.579034" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Imomnazarov, K.; Lopez-Scarim, J.; Bagheri, I.; Joers, V.; Tansey, M. G.; Martin-Pena, A.</p>
        <p class="info">Score: 4.5, Published: 2024-02-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.05.579034' target='https://doi.org/10.1101/2024.02.05.579034'> 10.1101/2024.02.05.579034</a></p>
        <p class="abstract">Synucleinopathies are a group of central nervous system pathologies that are characterized by neuronal accumulation of misfolded and aggregated -synuclein in proteinaceous depositions known as Lewy Bodies (LBs). The transition of -synuclein from its physiological to pathological form has been associated with several post-translational modifications such as phosphorylation and an increasing degree of insolubility, which also correlate with disease progression in post-mortem specimens from human patients. Neuronal expression of -synuclein in model organisms, including Drosophila melanogaster, has been a typical approach employed to study its physiological effects. Biochemical analysis of -synuclein solubility via high-speed ultracentrifugation with buffers of increasing detergent strength offers a potent method for identification of -synuclein biochemical properties and the associated pathology stage. Unfortunately, the development of a robust and reproducible method for evaluation of human -synuclein solubility isolated from Drosophila tissues has remained elusive. Here, we tested different detergents for their ability to solubilize human -synuclein carrying the pathological mutation A53T from brains of aged flies. We also assessed the effect of sonication on solubility of human -synuclein and optimized a protocol to discriminate relative amounts of soluble/insoluble human -synuclein from dopaminergic neurons of the Drosophila brain. Our data established that, using a 5% SDS buffer, the 3-step protocol distinguishes between cytosolic soluble proteins in fraction 1, detergent-soluble proteins in fraction 2 and insoluble proteins in fraction 3. This protocol shows that sonication breaks down -synuclein insoluble complexes from the fly brain, making them soluble in the SDS buffer and enriching fraction 2 of the protocol.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579656">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579656" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579656">
        <p class="paperTitle">Lung Mycobacterium tuberculosis infection perturbs metabolic pathways in non-pulmonary tissues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579656" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579656" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pahwa, F.; Chaudhary, S.; Dayal, A.; Nanda, R.</p>
        <p class="info">Score: 3.9, Published: 2024-02-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579656' target='https://doi.org/10.1101/2024.02.09.579656'> 10.1101/2024.02.09.579656</a></p>
        <p class="abstract">Mycobacterium tuberculosis (Mtb), through aerosol, reaches the lungs to cause pulmonary tuberculosis (TB); however, it may also affect the metabolism of other tissues in age-specific ways. In this study, female C57BL/6 mice (2 and 5 months old; M) were infected with a low aerosol dose (100-200 cfu) of Mtb H37Rv to monitor tissue mycobacterial load and multi-tissue metabolite profiling using gas chromatography and mass spectrometry (GC-MS). 5M C57BL/6 mice showed separate tissue metabolic phenotype with significantly higher lung aspartic acid, fecal oxalic acid and tryptophan levels with lower liver lysine and aspartic acid and fecal phenylalanine levels (log2FC: 5M/2M&gt; {&#43;/-}1.0, p&lt;0.1) compared to 2M young controls. Upon Mtb infection, the lung mycobacterial load of 2M and 5M mice were similar till 6 weeks post-infection. However, significantly higher lung phosphoric acid, malonic acid and lower mannose levels (log2FC: Mtb infected/healthy&gt; {&#43;/-}1.0, p&lt;0.1) were observed in Mtb-infected 5M C57BL/6 mice. Meanwhile, Mtb-infected 2M mice showed higher liver xylose and lower lysine levels. The thigh muscles of Mtb-infected 2M and 5M mice showed increased malic acid and oxalic acid and decreased glycine, serine, and glycerol levels. Fecal aspartic acid level was higher in Mtb-infected 5M mice, while a decreased abundance of fecal lysine was observed in Mtb-infected 2M mice. Overall, this study demonstrates a deregulated tissue-specific amino acid metabolism in Mtb-infected mice groups of different age groups, which might be targeted for managing TB infection-related adverse effects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.12.580004">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.12.580004" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.12.580004">
        <p class="paperTitle">SARS-CoV-2 Omicron lineage XBB spike structures, conformations, antigenicity, and receptor recognition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.12.580004" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.12.580004" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, Q. E.; Lindenberger, J.; Parsons, R.; Thakur, B.; Parks, R.; Park, C. S.; Huang, X.; Sammour, S.; Janowska, K.; Spence, T. N.; Edwards, R. J.; Martin, M.; Williams, W. B.; Gobeil, S.; Montefiori, D. C.; Korber, B.; Saunders, K. O.; Haynes, B. F.; Haynes, B. F.; Henderson, R.; Acharya, P.</p>
        <p class="info">Score: 3.6, Published: 2024-02-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.12.580004' target='https://doi.org/10.1101/2024.02.12.580004'> 10.1101/2024.02.12.580004</a></p>
        <p class="abstract">A recombinant lineage of the SARS-CoV-2 Omicron variant, named XBB, appeared in late 2022 and evolved descendants that successively swept local and global populations. XBB lineage members were noted for their improved immune evasion and transmissibility. Here, we determine cryo-EM structures of XBB.1.5, XBB.1.16 and EG.5 spike (S) ectodomains to reveal enhanced occupancy of the receptor inaccessible closed state. Interprotomer receptor binding domain (RBD) interactions previously observed in BA.1 and BA.2 were retained to reinforce the 3-RBD-down state. Improved stability of XBB.1.5 and XBB.1.16 RBD compensated for loss of stability caused by early Omicron mutations, while the F456L substitution reduced EG.5 RBD stability. Long-range impacts of S1 subunit mutations affected conformation and epitope presentation in the S2 subunit. Taken together, our results feature a theme of iterative optimization of S protein stability as Omicron continues to evolve, while maintaining high affinity receptor binding and bolstering immune evasion.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579708">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579708" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579708">
        <p class="paperTitle">Structural insights into CXCR4 modulation and oligomerization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579708" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579708" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Saotome, K.; McGoldrick, L. L.; Ho, J.-H.; Ramlall, T. F.; Shah, S.; Moore, M. J.; Kim, J. H.; Leidich, R.; Olson, W. C.; Franklin, M. C.</p>
        <p class="info">Score: 3.4, Published: 2024-02-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579708' target='https://doi.org/10.1101/2024.02.09.579708'> 10.1101/2024.02.09.579708</a></p>
        <p class="abstract">Activation of the chemokine receptor CXCR4 by its chemokine ligand CXCL12 regulates diverse cellular processes. CXCR4 also serves as a key target for diseases such as cancer and HIV. Previously reported crystal structures of CXCR4 bound to antagonists revealed the architecture of an inactive, homodimeric receptor. However, many structural aspects of CXCR4 remain poorly understood, including its activation by CXCL12, as well as its assembly into higher-order oligomers. Here, we use cryoelectron microscopy (cryoEM) to investigate various modes of CXCR4 regulation in the presence and absence of Gi protein. CXCL12 activates CXCR4 by inserting its N-terminus deep into the CXCR4 orthosteric pocket. The binding of FDA-approved antagonist AMD3100 is stabilized by electrostatic interactions with acidic residues in the 7 transmembrane helix bundle. A potent antibody blocker, REGN7663, binds across the extracellular face of CXCR4 and inserts its CDR-H3 loop into the orthosteric pocket. Trimeric and tetrameric structures of CXCR4 reveal, to our knowledge, previously undescribed modes of GPCR oligomerization. Remarkably, CXCR4 adopts distinct subunit conformations in trimeric and tetrameric assemblies, highlighting how oligomerization could allosterically regulate chemokine receptor function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.13.580062">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.13.580062" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.13.580062">
        <p class="paperTitle">Spatial top-down proteomics for the functional characterization of human kidney</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.13.580062" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.13.580062" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zemaitis, K. J.; Fulcher, J. M.; Kumar, R.; Degnan, D. J.; Lewis, L. A.; Liao, Y.-C.; Velickovic, M.; Williams, S. M.; Moore, R. J.; Bramer, L. M.; Velickovic, D.; Zhu, Y.; Zhou, M.; Pasa-Tolic, L.</p>
        <p class="info">Score: 3.2, Published: 2024-02-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.13.580062' target='https://doi.org/10.1101/2024.02.13.580062'> 10.1101/2024.02.13.580062</a></p>
        <p class="abstract">Background: The Human Proteome Project has credibly detected nearly 93% of the roughly 20,000 proteins which are predicted by the human genome. However, the proteome is enigmatic, where alterations in amino acid sequences from polymorphisms and alternative splicing, errors in translation, and post-translational modifications result in a proteome depth estimated at several million unique proteoforms. Recently mass spectrometry has been demonstrated in several landmark efforts mapping the human proteoform landscape in bulk analyses. Herein, we developed an integrated workflow for characterizing proteoforms from human tissue in a spatially resolved manner by coupling laser capture microdissection, nanoliter-scale sample preparation, and mass spectrometry imaging. Results: Using healthy human kidney sections as the case study, we focused our analyses on the major functional tissue units including glomeruli, tubules, and medullary rays. After laser capture microdissection, these isolated functional tissue units were processed with microPOTS (microdroplet processing in one-pot for trace samples) for sensitive top-down proteomics measurement. This provided a quantitative database of 616 proteoforms that was further leveraged as a library for mass spectrometry imaging with near-cellular spatial resolution over the entire section. Notably, several mitochondrial proteoforms were found to be differentially abundant between glomeruli and convoluted tubules, and further spatial contextualization was provided by mass spectrometry imaging confirming unique differences identified by microPOTS, and further expanding the field-of-view for unique distributions such as enhanced abundance of a truncated form (1-74) of ubiquitin within cortical regions. Conclusions: We developed an integrated workflow to directly identify proteoforms and reveal their spatial distributions. Where of the 20 differentially abundant proteoforms identified as discriminate between tubules and glomeruli by microPOTS, the vast majority of tubular proteoforms were of mitochondrial origin (8 of 10) where discriminate proteoforms in glomeruli were primarily hemoglobin subunits (9 of 10). These trends were also identified within ion images demonstrating spatially resolved characterization of proteoforms that has the potential to reshape discovery-based proteomics because the proteoforms are the ultimate effector of cellular functions. Applications of this technology have the potential to unravel etiology and pathophysiology of disease states, informing on biologically active proteoforms, which remodel the proteomic landscape in chronic and acute disorders.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
