<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="The resting state of the human T-cell receptor
Authors: Notti, R. Q.; Yi, F.; Heissel, S.; Molina, H.; Klebanoff, C. A.; Walz, T.
Score: 35.5, Published: 2023-08-22 DOI: 10.1101/2023.08.22.554360
The T-cell receptor (TCR) is central to the ligand-dependent activation of T lymphocytes and as such orchestrates both adaptive and pathologic immune processes1. However, major questions remain regarding the structure and function of the human TCR2-4. Here, we present cryogenic electron microscopy structures for the unliganded human TCR in a native-like lipid bilayer, revealing three related conformations that are distinct from previously reported structures of receptors in detergent.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="The resting state of the human T-cell receptor
Authors: Notti, R. Q.; Yi, F.; Heissel, S.; Molina, H.; Klebanoff, C. A.; Walz, T.
Score: 35.5, Published: 2023-08-22 DOI: 10.1101/2023.08.22.554360
The T-cell receptor (TCR) is central to the ligand-dependent activation of T lymphocytes and as such orchestrates both adaptive and pathologic immune processes1. However, major questions remain regarding the structure and function of the human TCR2-4. Here, we present cryogenic electron microscopy structures for the unliganded human TCR in a native-like lipid bilayer, revealing three related conformations that are distinct from previously reported structures of receptors in detergent." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-03T10:37:18+00:00" />
<meta property="article:modified_time" content="2023-09-03T10:37:18+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="The resting state of the human T-cell receptor
Authors: Notti, R. Q.; Yi, F.; Heissel, S.; Molina, H.; Klebanoff, C. A.; Walz, T.
Score: 35.5, Published: 2023-08-22 DOI: 10.1101/2023.08.22.554360
The T-cell receptor (TCR) is central to the ligand-dependent activation of T lymphocytes and as such orchestrates both adaptive and pathologic immune processes1. However, major questions remain regarding the structure and function of the human TCR2-4. Here, we present cryogenic electron microscopy structures for the unliganded human TCR in a native-like lipid bilayer, revealing three related conformations that are distinct from previously reported structures of receptors in detergent."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "The resting state of the human T-cell receptor\nAuthors: Notti, R. Q.; Yi, F.; Heissel, S.; Molina, H.; Klebanoff, C. A.; Walz, T.\nScore: 35.5, Published: 2023-08-22 DOI: 10.1101/2023.08.22.554360\nThe T-cell receptor (TCR) is central to the ligand-dependent activation of T lymphocytes and as such orchestrates both adaptive and pathologic immune processes1. However, major questions remain regarding the structure and function of the human TCR2-4. Here, we present cryogenic electron microscopy structures for the unliganded human TCR in a native-like lipid bilayer, revealing three related conformations that are distinct from previously reported structures of receptors in detergent.",
  "keywords": [
    
  ],
  "articleBody": " The resting state of the human T-cell receptor\nAuthors: Notti, R. Q.; Yi, F.; Heissel, S.; Molina, H.; Klebanoff, C. A.; Walz, T.\nScore: 35.5, Published: 2023-08-22 DOI: 10.1101/2023.08.22.554360\nThe T-cell receptor (TCR) is central to the ligand-dependent activation of T lymphocytes and as such orchestrates both adaptive and pathologic immune processes1. However, major questions remain regarding the structure and function of the human TCR2-4. Here, we present cryogenic electron microscopy structures for the unliganded human TCR in a native-like lipid bilayer, revealing three related conformations that are distinct from previously reported structures of receptors in detergent. These new \"closed and compacted\" conformations afford insights into the interactions between the TCR and the membrane, including conserved surface patches that make extensive outer leaflet contact, and suggest novel conformational regulation by glycans. We show that the closed/compacted conformations, not the extended one previously reported in detergent5-8, represent the unliganded resting state for the TCR in vivo, underscoring the importance of structural interrogation of membrane proteins in native-like environments. We use conformation-locking disulfide mutants to show that juxtamembrane linker extension is necessary for ligand-dependent TCR activation, demonstrating that TCR-intrinsic conformational change is necessary for TCR activation and opening numerous avenues for immunoreceptor engineering.\niATPSnFR2: a high dynamic range fluorescent sensor for monitoring intracellular ATP\nAuthors: Marvin, J. S.; Kokotos, A. C.; Kumar, M.; Pulido, C.; Tkachuk, A. N.; Yao, J. S.; Brown, T. A.; Ryan, T. A.\nScore: 15.8, Published: 2023-08-24 DOI: 10.1101/2023.08.24.554624\nWe developed a significantly improved genetically encoded quantitative adenosine triphosphate (ATP) sensor to provide real-time dynamics of ATP levels in subcellular compartments. iATPSnFR2 is a variant of iATPSnFR1, a previously developed sensor that has circularly permuted super-folder GFP inserted between the ATP-binding helices of the{varepsilon} -subunit of a bacterial F0-F1 ATPase. Optimizing the linkers joining the two domains resulted in a [~] 5-6 fold improvement in the dynamic range compared to the previous generation sensor, with excellent discrimination against other analytes and affinity variants varying from 4 M to 500 M. A chimeric version of this sensor fused to either the HaloTag protein or a suitably spectrally separated fluorescent protein, provides a ratiometric readout allowing comparisons of ATP across cellular regions. Subcellular targeting of the sensor to nerve terminals reveals previously uncharacterized single synapse metabolic signatures, while targeting to the mitochondrial matrix allowed direct quantitative probing of oxidative phosphorylation dynamics. Significance StatementAdenosine triphosphate (ATP) is a key metabolite necessary for cellular life. Here we develop a next-generation genetically encoded ratiometric fluorescent ATP sensor that allows subcellular tracking of ATP levels in living cells. The large dynamic range makes it possible to follow the dynamics of this metabolite across cells and subcellular regions under different metabolic stressors. We expect that iATPSnFR2 will provide researchers with exciting new opportunities to study ATP dynamics with temporal and spatial resolution that has, until now, been unavailable.\nSCP Viz - A universal graphical user interface for single protein analysis in single cell proteomics datasets\nAuthors: Warshanna, A.; Orsburn, B.\nScore: 12.8, Published: 2023-08-29 DOI: 10.1101/2023.08.29.555397\nSingle cell proteomics (SCP) requires the analysis of dozens to thousands of single human cells to draw biological conclusions. However, assessing of the abundance of single proteins in output data presents a considerable challenge, and no simple universal solutions currently exist. To address this, we developed SCP Viz, a statistical package with a graphical user interface that can handle small and large scale SCP output from any instrument or data processing software. In this software, the abundance of individual proteins can be plotted in a variety of ways, using either unadjusted or normalized outputs. These outputs can also be transformed or imputed within the software. SCP Viz offers a variety of plotting options which can help identify significantly altered proteins between groups, both before and after quantitative transformations. Upon the discovery of subpopulations of single cells, users can easily regroup the cells of interest using straightforward text-based filters. When used in this way, SCP Viz allows users to visualize proteomic heterogeneity at the level of individual proteins, cells, or identified subcellular populations. SCP Viz is compatible with output files from MaxQuant, FragPipe, SpectroNaut, and Proteome Discoverer, and should work equally well with other formats. SCP Viz is publicly available at https://github.com/orsburn/SCPViz. For demonstrations, users can download our test data from GitHub and use an online version that accepts user input for analysis at https://orsburnlab.shinyapps.io/SCPViz/. Abstract graphic O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=167 SRC=\"FIGDIR/small/555397v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (27K): org.highwire.dtl.DTLVardef@1e522f5org.highwire.dtl.DTLVardef@5a31f5org.highwire.dtl.DTLVardef@185dc6borg.highwire.dtl.DTLVardef@149f594_HPS_FORMAT_FIGEXP M_FIG C_FIG\nA conserved ion channel function of STING mediates non-canonical autophagy and cell death\nAuthors: Xun, J.; Zhang, Z.; Lyu, B.; Lu, D.; Yang, H.; Shang, G.; Tan, J. X.\nScore: 12.1, Published: 2023-08-27 DOI: 10.1101/2023.08.26.554976\nThe cGAS/STING pathway triggers inflammation in response to diverse cellular stresses such as infection, cellular damage, senescence, normal aging, and age-related disease. Besides inflammation, STING also triggers non-canonical autophagy and cell death, the former of which requires the proton pump V-ATPase- mediated LC3 lipidation to single membrane STING vesicles. V-ATPase is known to sense organelle de- acidification in other contexts and recruits the ATG16L1 complex for direct conjugation of LC3/ATG8 to single membranes (CASM). However, it is unclear how STING activates V-ATPase for non-canonical autophagy. Here we report that upon STING activation, the transmembrane domain (TMD) of STING significantly reorganizes and forms an electron-sparse pore in the center. Cellular imaging and in vitro ion flux assays revealed that STING is critical for proton efflux and pH neutralization of Golgi-derived STING vesicles. A chemical ligand of STING, C53, which binds to and blocks the channel of STING strongly inhibited STING-mediated proton flux in vitro and vesicular de-acidification in cells. C53 also abolished STING-dependent LC3 lipidation and cell death. Thus, the ion channel function of STING activates non-canonical autophagy and cell death through vesicle de-acidification.\nProtein Coronas on Functionalized Nanoparticles Enable Quantitative and Precise Large-Scale Deep Plasma Proteomics\nAuthors: Huang, T.; Wang, J.; Stukalov, A.; Donovan, M. K. R.; Ferdosi, S.; Williamson, L.; Just, S.; Castro, G.; Cantrell, L. S.; Elgierari, E.; Benz, R. W.; Huang, Y.; Motamedchaboki, K.; Hakimi, A.; Arrey, T.; Damoc, E.; Kreimer, S.; Farokhzad, O. C.; Batzoglou, S.; Siddiqui, A.; Van Eyk, J. E.; Hornburg, D.\nScore: 11.4, Published: 2023-08-29 DOI: 10.1101/2023.08.28.555225\nBackgroundThe wide dynamic range of circulating proteins coupled with the diversity of proteoforms present in plasma has historically impeded comprehensive and quantitative characterization of the plasma proteome at scale. Automated nanoparticle (NP) protein corona-based proteomics workflows can efficiently compress the dynamic range of protein abundances into a mass spectrometry (MS)-accessible detection range. This enhances the depth and scalability of quantitative MS-based methods, which can elucidate the molecular mechanisms of biological processes, discover new protein biomarkers, and improve comprehensiveness of MS-based diagnostics. MethodsInvestigating multi-species spike-in experiments and a cohort, we investigated fold-change accuracy, linearity, precision, and statistical power for the using the Proteograph Product Suite, a deep plasma proteomics workflow, in conjunction with multiple MS instruments. ResultsWe show that NP-based workflows enable accurate identification (false discovery rate of 1%) of more than 6,000 proteins from plasma (Orbitrap Astral) and, compared to a gold standard neat plasma workflow that is limited to the detection of hundreds of plasma proteins, facilitate quantification of more proteins with accurate fold-changes, high linearity, and precision. Furthermore, we demonstrate high statistical power for the discovery of biomarkers in small- and large-scale cohorts. ConclusionsThe automated NP workflow enables high-throughput, deep, and quantitative plasma proteomics investigation with sufficient power to discover new biomarker signatures with a peptide level resolution.\nAutomating data analysis for hydrogen/deuterium exchange mass spectrometry using data-independent acquisition methodology\nAuthors: Filandr, F.; Sarpe, V.; Raval, S.; Crowder, D. A.; Khan, M. F.; Douglas, P.; Coales, S.; Viner, R.; Syed, A.; Tainer, J. A.; Lees-MIller, S. P.; Schriemer, D. C.\nScore: 9.2, Published: 2023-08-26 DOI: 10.1101/2023.08.25.554852\nWe developed a hydrogen/deuterium exchange workflow coupled to tandem mass spectrometry (HX-MS2) that supports the acquisition of peptide fragment ions alongside their peptide precursors. The approach enables true auto-validation of HX data by mining a rich set of deuterated fragments, generated by collisional-induced dissociation (CID), to simultaneously confirm the peptide ID and authenticate MS1-based deuteration calculations. The high redundancy provided by the fragments supports a confidence assessment of deuterium calculations using a combinatorial strategy. The approach requires data-independent acquisition (DIA) methods that are available on most MS platforms, making the switch to HX-MS2 straightforward. Importantly, we find that HX-DIA enables a proteomics-grade approach and wide-spread applications. Considerable time is saved through auto-validation and complex samples can now be characterized and at higher throughput. We illustrate these advantages in a drug binding analysis of the ultra-large protein kinase DNA-PKcs, isolated directly from mammalian cells.\nAllosteric activation of the co-receptor BAK1 by the EFR receptor kinase initiates immune signaling\nAuthors: Mühlenbeck, H.; Tsutsui, Y.; Lemmon, M.; Bender, K. W.; Zipfel, C.\nScore: 19.2, Published: 2023-08-25 DOI: 10.1101/2023.08.23.554490\nTransmembrane signaling by plant receptor kinases (RKs) has long been thought to involve reciprocal trans-phosphorylation of their intracellular kinase domains. The fact that many of these are pseudokinase domains, however, suggests that additional mechanisms must govern RK signaling activation. Non-catalytic (pseudo)kinase signaling mechanisms have been described in metazoans, but information is scarce for plants. Recently, a non-catalytic function was reported for the leucine-rich repeat (LRR)-RK subfamily XIIa member EFR (ELONGATION FACTOR TU RECEPTOR) and phosphorylation-dependent conformational changes were proposed to regulate signaling of RKs with non-RD kinase domains. Here, using EFR as a model, we describe a non-catalytic activation mechanism for LRR-RKs with non-RD kinase domains. EFR is an active kinase, but a kinase-dead variant retains the ability to enhance catalytic activity of its co-receptor kinase BAK1/SERK3 (BRASSINOSTEROID INSENSITIVE 1-ASSOCIATED KINASE 1/SOMATIC EMBRYOGENESIS RECEPTOR KINASE 3). Applying hydrogen-deuterium exchange mass spectrometry (HDX-MS) analysis and designing homology-based intragenic suppressor mutations, we provide evidence that the EFR kinase domain must adopt its active conformation in order to activate BAK1 allosterically, likely by supporting C-helix positioning in BAK1. Our results suggest a conformational toggle model for signaling, in which BAK1 first phosphorylates EFR in the activation loop to stabilize its active conformation, allowing EFR in turn to allosterically activate BAK1.\nPigment Binding in The Light-Dependent Protochlorophyllide Oxidoreductase\nAuthors: Pesara, P.; Szafran, K.; Nguyen, H. C.; Sirohiwal, A.; Pantazis, D. A.; Gabruk, M.\nScore: 7.9, Published: 2023-08-29 DOI: 10.1101/2023.08.28.555081\nThe Light-Dependent Protochlorophyllide Oxidoreductase (LPOR) is a key enzyme in chlorophyll biosynthesis and its photocatalytic mechanism has long intrigued researchers. However, the lack of structural data for the active complex has impeded understanding of its reaction mechanism. A recent high-resolution structure of enzyme in the active conformation has established a robust foundation for validating hypotheses concerning pigment binding, residue involvement, and consequently, the reaction mechanism. Surprisingly, this new structure challenges previously proposed mechanisms, especially concerning the orientation of the bound protochlorophyllide (Pchlide) pigment. In this study, we employ molecular dynamics and hybrid quantum-mechanics/molecular-mechanics (QM/MM) simulations along with site-directed mutagenesis to compare two Pchlide binding modes: one aligned with previous proposals (mode A), and the other consistent with the recent experimental data (mode B). Binding energy calculations reveal thermodynamic instability of binding mode A due to nonspecific interactions, while mode B exhibits distinct stabilizing interactions yielding favorable binding. QM/MM-based local energy decomposition analysis unravels a complex interaction network that reinforces pigment stabilization in this conformation. Notably, interactions involving Tyr177, His319, and the carboxyl group at C131 influence the pigments excited state energy and potentially contributing to the substrate specificity of the enzyme. Our results uniformly favor binding mode B as represented in the new cryo-EM structure, over the previously assumed mode A. These findings challenge established interpretations and underscore the need for a comprehensive re-evaluation of the reaction mechanism of LPOR that correctly considers pigment interactions and substrate orientation in the binding pocket. Significance StatementA crucial step in the biosynthesis of the all-important photosynthetic pigment chlorophyll is the reduction of a double C=C bond in its precursor protochlorophyllide (PChlide). This is catalyzed by the Light-Dependent Protochlorophyllide Oxidoreductase (LPOR) via an extremely rare example of a biological photocatalytic reaction. Understanding of the LPOR mechanism has been hindered by limited insight into the structure of its active complex. A recent high-resolution LPOR cryo-EM structure substantiates pigment binding, residue interactions, and the reaction mechanism, but contrasts markedly with all previous assumptions regarding the binding mode of the substrate PChlide. Using molecular dynamics simulations, quantum-mechanics/molecular-mechanics calculations, and mutagenesis, we compare and evaluate the two possible Pchlide binding modes, the one assumed previously (mode A) and the one supported by recent data (mode B). Our findings conclusively favor mode B, challenging prior assumptions and pointing toward novel mechanistic possibilities for this unique photocatalytic reaction.\nDiscovery of 2-amide-3-methylester thiophenes inhibiting SARS-CoV-2 ADP-ribosyl hydrolysing macrodomain and coronavirus replication\nAuthors: Wazir, S.; Parviainen, T. A. O.; Maksimainen, M. M.; Duong, M. T. H.; Pfannenstiel, J. J.; Cluff, D.; Sowa, S. T.; Galera-Prat, A.; Ferraris, D. V.; Fehr, A.; Heiskanen, J. P.; Lehtiö, L.\nScore: 6.1, Published: 2023-08-29 DOI: 10.1101/2023.08.28.555062\nThe COVID-19 pandemic caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) virus has made it clear that further development of antiviral therapies will be needed to combat additional SARS-CoV-2 variants or novel CoVs. Here, we describe small molecule inhibitors for SARS-CoV-2 Mac1, which counters ADP-ribosylation mediated innate immune responses. The compounds inhibiting Mac1 were discovered through high-throughput screening (HTS) using a protein FRET-based competition assay and the best hit compound had an IC50 of 14 M. Three validated HTS hits have the same 2-amide-3-methylester thiophene scaffold and the scaffold was selected for structure-activity relationship (SAR) studies through commercial and synthesized analogs. We studied the compound binding mode in detail using X-ray crystallography and this allowed us to focus on specific features of the compound and design analogs. Compound 27 (MDOLL-0229) had an IC50 of 2.1 M and was generally selective for CoV Mac1 proteins after profiling for activity against a panel of viral and human ADP-ribose binding proteins. The improved potency allowed testing of its effect on virus replication and indeed, 27 inhibited replication of a mouse hepatitis virus, a prototype CoV. Compound 27 is the first Mac1 targeted small molecule demonstrated to inhibit coronavirus replication in a cell model. This, together with its well-defined binding mode, makes 27 a good candidate for further hit/lead-optimization efforts.\nDiscovery and engineering of AiEvo2, a novel Cas12a nuclease for human gene editing applications\nAuthors: Sharrar, A.; Arake de Tacca, L.; Meacham, Z.; Staples-Ager, J.; Collingwood, T.; Rabuka, D.; Schelle, M.\nScore: 5.6, Published: 2023-08-28 DOI: 10.1101/2023.08.28.554880\nThe precision of gene editing technology is critical to creating safe and effective therapies for treating human disease. While the programmability of CRISPR-Cas systems has allowed for rapid innovation of new gene editing techniques, the off-target activity of these enzymes has hampered clinical development for novel therapeutics. Here we report the identification and characterization of a novel CRISPR-Cas12a enzyme from Acinetobacter indicus (AiCas12a). We then engineer the nuclease (termed AiEvo2) for increased specificity, PAM recognition, and efficacy on a variety of human clinical targets. AiEvo2 is highly precise and able to efficiently discriminate between normal and disease-causing alleles in Huntingtons patient derived cells by taking advantage of a single nucleotide polymorphism on the disease-associated allele. AiEvo2 efficiently edits several liver-associated target genes including PCSK9 and TTR when delivered to primary hepatocytes as mRNA encapsulated in a lipid nanoparticle. The enzyme also engineers an effective CD19 CAR-T therapy from primary human T cells using multiplexed simultaneous editing and CAR insertion. To further ensure precise editing, we engineered an anti-CRISPR protein (ErAcr) to selectively inhibit off-target gene editing while retaining therapeutic on-target editing. The engineered AiEvo2 nuclease coupled with a novel ErAcr protein represents a new way to control the fidelity of editing and improve the safety and efficacy of gene editing therapies.\n",
  "wordCount" : "2603",
  "inLanguage": "en",
  "datePublished": "2023-09-03T10:37:18Z",
  "dateModified": "2023-09-03T10:37:18Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta"><span>updated on September 3, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.554360">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.554360" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.554360">
        <p class="paperTitle">The resting state of the human T-cell receptor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.554360" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.554360" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Notti, R. Q.; Yi, F.; Heissel, S.; Molina, H.; Klebanoff, C. A.; Walz, T.</p>
        <p class="info">Score: 35.5, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.554360' target='https://doi.org/10.1101/2023.08.22.554360'> 10.1101/2023.08.22.554360</a></p>
        <p class="abstract">The T-cell receptor (TCR) is central to the ligand-dependent activation of T lymphocytes and as such orchestrates both adaptive and pathologic immune processes1. However, major questions remain regarding the structure and function of the human TCR2-4. Here, we present cryogenic electron microscopy structures for the unliganded human TCR in a native-like lipid bilayer, revealing three related conformations that are distinct from previously reported structures of receptors in detergent. These new &#34;closed and compacted&#34; conformations afford insights into the interactions between the TCR and the membrane, including conserved surface patches that make extensive outer leaflet contact, and suggest novel conformational regulation by glycans. We show that the closed/compacted conformations, not the extended one previously reported in detergent5-8, represent the unliganded resting state for the TCR in vivo, underscoring the importance of structural interrogation of membrane proteins in native-like environments. We use conformation-locking disulfide mutants to show that juxtamembrane linker extension is necessary for ligand-dependent TCR activation, demonstrating that TCR-intrinsic conformational change is necessary for TCR activation and opening numerous avenues for immunoreceptor engineering.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554624">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554624" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554624">
        <p class="paperTitle">iATPSnFR2: a high dynamic range fluorescent sensor for monitoring intracellular ATP</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554624" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554624" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Marvin, J. S.; Kokotos, A. C.; Kumar, M.; Pulido, C.; Tkachuk, A. N.; Yao, J. S.; Brown, T. A.; Ryan, T. A.</p>
        <p class="info">Score: 15.8, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554624' target='https://doi.org/10.1101/2023.08.24.554624'> 10.1101/2023.08.24.554624</a></p>
        <p class="abstract">We developed a significantly improved genetically encoded quantitative adenosine triphosphate (ATP) sensor to provide real-time dynamics of ATP levels in subcellular compartments. iATPSnFR2 is a variant of iATPSnFR1, a previously developed sensor that has circularly permuted super-folder GFP inserted between the ATP-binding helices of the{varepsilon} -subunit of a bacterial F0-F1 ATPase. Optimizing the linkers joining the two domains resulted in a [~] 5-6 fold improvement in the dynamic range compared to the previous generation sensor, with excellent discrimination against other analytes and affinity variants varying from 4 M to 500 M. A chimeric version of this sensor fused to either the HaloTag protein or a suitably spectrally separated fluorescent protein, provides a ratiometric readout allowing comparisons of ATP across cellular regions. Subcellular targeting of the sensor to nerve terminals reveals previously uncharacterized single synapse metabolic signatures, while targeting to the mitochondrial matrix allowed direct quantitative probing of oxidative phosphorylation dynamics.

Significance StatementAdenosine triphosphate (ATP) is a key metabolite necessary for cellular life. Here we develop a next-generation genetically encoded ratiometric fluorescent ATP sensor that allows subcellular tracking of ATP levels in living cells. The large dynamic range makes it possible to follow the dynamics of this metabolite across cells and subcellular regions under different metabolic stressors. We expect that iATPSnFR2 will provide researchers with exciting new opportunities to study ATP dynamics with temporal and spatial resolution that has, until now, been unavailable.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.555397">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.555397" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.555397">
        <p class="paperTitle">SCP Viz - A universal graphical user interface for single protein analysis in single cell proteomics datasets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.555397" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.555397" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Warshanna, A.; Orsburn, B.</p>
        <p class="info">Score: 12.8, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.555397' target='https://doi.org/10.1101/2023.08.29.555397'> 10.1101/2023.08.29.555397</a></p>
        <p class="abstract">Single cell proteomics (SCP) requires the analysis of dozens to thousands of single human cells to draw biological conclusions. However, assessing of the abundance of single proteins in output data presents a considerable challenge, and no simple universal solutions currently exist. To address this, we developed SCP Viz, a statistical package with a graphical user interface that can handle small and large scale SCP output from any instrument or data processing software. In this software, the abundance of individual proteins can be plotted in a variety of ways, using either unadjusted or normalized outputs. These outputs can also be transformed or imputed within the software. SCP Viz offers a variety of plotting options which can help identify significantly altered proteins between groups, both before and after quantitative transformations. Upon the discovery of subpopulations of single cells, users can easily regroup the cells of interest using straightforward text-based filters. When used in this way, SCP Viz allows users to visualize proteomic heterogeneity at the level of individual proteins, cells, or identified subcellular populations. SCP Viz is compatible with output files from MaxQuant, FragPipe, SpectroNaut, and Proteome Discoverer, and should work equally well with other formats. SCP Viz is publicly available at https://github.com/orsburn/SCPViz. For demonstrations, users can download our test data from GitHub and use an online version that accepts user input for analysis at https://orsburnlab.shinyapps.io/SCPViz/.

Abstract graphic

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=167 SRC=&#34;FIGDIR/small/555397v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (27K):
org.highwire.dtl.DTLVardef@1e522f5org.highwire.dtl.DTLVardef@5a31f5org.highwire.dtl.DTLVardef@185dc6borg.highwire.dtl.DTLVardef@149f594_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.26.554976">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.26.554976" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.26.554976">
        <p class="paperTitle">A conserved ion channel function of STING mediates non-canonical autophagy and cell death</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.26.554976" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.26.554976" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xun, J.; Zhang, Z.; Lyu, B.; Lu, D.; Yang, H.; Shang, G.; Tan, J. X.</p>
        <p class="info">Score: 12.1, Published: 2023-08-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.26.554976' target='https://doi.org/10.1101/2023.08.26.554976'> 10.1101/2023.08.26.554976</a></p>
        <p class="abstract">The cGAS/STING pathway triggers inflammation in response to diverse cellular stresses such as infection, cellular damage, senescence, normal aging, and age-related disease. Besides inflammation, STING also triggers non-canonical autophagy and cell death, the former of which requires the proton pump V-ATPase- mediated LC3 lipidation to single membrane STING vesicles. V-ATPase is known to sense organelle de- acidification in other contexts and recruits the ATG16L1 complex for direct conjugation of LC3/ATG8 to single membranes (CASM). However, it is unclear how STING activates V-ATPase for non-canonical autophagy. Here we report that upon STING activation, the transmembrane domain (TMD) of STING significantly reorganizes and forms an electron-sparse pore in the center. Cellular imaging and in vitro ion flux assays revealed that STING is critical for proton efflux and pH neutralization of Golgi-derived STING vesicles. A chemical ligand of STING, C53, which binds to and blocks the channel of STING strongly inhibited STING-mediated proton flux in vitro and vesicular de-acidification in cells. C53 also abolished STING-dependent LC3 lipidation and cell death. Thus, the ion channel function of STING activates non-canonical autophagy and cell death through vesicle de-acidification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555225">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555225" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555225">
        <p class="paperTitle">Protein Coronas on Functionalized Nanoparticles Enable Quantitative and Precise Large-Scale Deep Plasma Proteomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555225" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555225" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Huang, T.; Wang, J.; Stukalov, A.; Donovan, M. K. R.; Ferdosi, S.; Williamson, L.; Just, S.; Castro, G.; Cantrell, L. S.; Elgierari, E.; Benz, R. W.; Huang, Y.; Motamedchaboki, K.; Hakimi, A.; Arrey, T.; Damoc, E.; Kreimer, S.; Farokhzad, O. C.; Batzoglou, S.; Siddiqui, A.; Van Eyk, J. E.; Hornburg, D.</p>
        <p class="info">Score: 11.4, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555225' target='https://doi.org/10.1101/2023.08.28.555225'> 10.1101/2023.08.28.555225</a></p>
        <p class="abstract">BackgroundThe wide dynamic range of circulating proteins coupled with the diversity of proteoforms present in plasma has historically impeded comprehensive and quantitative characterization of the plasma proteome at scale. Automated nanoparticle (NP) protein corona-based proteomics workflows can efficiently compress the dynamic range of protein abundances into a mass spectrometry (MS)-accessible detection range. This enhances the depth and scalability of quantitative MS-based methods, which can elucidate the molecular mechanisms of biological processes, discover new protein biomarkers, and improve comprehensiveness of MS-based diagnostics.

MethodsInvestigating multi-species spike-in experiments and a cohort, we investigated fold-change accuracy, linearity, precision, and statistical power for the using the Proteograph Product Suite, a deep plasma proteomics workflow, in conjunction with multiple MS instruments.

ResultsWe show that NP-based workflows enable accurate identification (false discovery rate of 1%) of more than 6,000 proteins from plasma (Orbitrap Astral) and, compared to a gold standard neat plasma workflow that is limited to the detection of hundreds of plasma proteins, facilitate quantification of more proteins with accurate fold-changes, high linearity, and precision. Furthermore, we demonstrate high statistical power for the discovery of biomarkers in small- and large-scale cohorts.

ConclusionsThe automated NP workflow enables high-throughput, deep, and quantitative plasma proteomics investigation with sufficient power to discover new biomarker signatures with a peptide level resolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.554852">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.554852" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.554852">
        <p class="paperTitle">Automating data analysis for hydrogen/deuterium exchange mass spectrometry using data-independent acquisition methodology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.554852" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.554852" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Filandr, F.; Sarpe, V.; Raval, S.; Crowder, D. A.; Khan, M. F.; Douglas, P.; Coales, S.; Viner, R.; Syed, A.; Tainer, J. A.; Lees-MIller, S. P.; Schriemer, D. C.</p>
        <p class="info">Score: 9.2, Published: 2023-08-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.554852' target='https://doi.org/10.1101/2023.08.25.554852'> 10.1101/2023.08.25.554852</a></p>
        <p class="abstract">We developed a hydrogen/deuterium exchange workflow coupled to tandem mass spectrometry (HX-MS2) that supports the acquisition of peptide fragment ions alongside their peptide precursors. The approach enables true auto-validation of HX data by mining a rich set of deuterated fragments, generated by collisional-induced dissociation (CID), to simultaneously confirm the peptide ID and authenticate MS1-based deuteration calculations. The high redundancy provided by the fragments supports a confidence assessment of deuterium calculations using a combinatorial strategy. The approach requires data-independent acquisition (DIA) methods that are available on most MS platforms, making the switch to HX-MS2 straightforward. Importantly, we find that HX-DIA enables a proteomics-grade approach and wide-spread applications. Considerable time is saved through auto-validation and complex samples can now be characterized and at higher throughput. We illustrate these advantages in a drug binding analysis of the ultra-large protein kinase DNA-PKcs, isolated directly from mammalian cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554490">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554490" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554490">
        <p class="paperTitle">Allosteric activation of the co-receptor BAK1 by the EFR receptor kinase initiates immune signaling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554490" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554490" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mühlenbeck, H.; Tsutsui, Y.; Lemmon, M.; Bender, K. W.; Zipfel, C.</p>
        <p class="info">Score: 19.2, Published: 2023-08-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554490' target='https://doi.org/10.1101/2023.08.23.554490'> 10.1101/2023.08.23.554490</a></p>
        <p class="abstract">Transmembrane signaling by plant receptor kinases (RKs) has long been thought to involve reciprocal trans-phosphorylation of their intracellular kinase domains. The fact that many of these are pseudokinase domains, however, suggests that additional mechanisms must govern RK signaling activation. Non-catalytic (pseudo)kinase signaling mechanisms have been described in metazoans, but information is scarce for plants. Recently, a non-catalytic function was reported for the leucine-rich repeat (LRR)-RK subfamily XIIa member EFR (ELONGATION FACTOR TU RECEPTOR) and phosphorylation-dependent conformational changes were proposed to regulate signaling of RKs with non-RD kinase domains. Here, using EFR as a model, we describe a non-catalytic activation mechanism for LRR-RKs with non-RD kinase domains. EFR is an active kinase, but a kinase-dead variant retains the ability to enhance catalytic activity of its co-receptor kinase BAK1/SERK3 (BRASSINOSTEROID INSENSITIVE 1-ASSOCIATED KINASE 1/SOMATIC EMBRYOGENESIS RECEPTOR KINASE 3). Applying hydrogen-deuterium exchange mass spectrometry (HDX-MS) analysis and designing homology-based intragenic suppressor mutations, we provide evidence that the EFR kinase domain must adopt its active conformation in order to activate BAK1 allosterically, likely by supporting C-helix positioning in BAK1. Our results suggest a conformational toggle model for signaling, in which BAK1 first phosphorylates EFR in the activation loop to stabilize its active conformation, allowing EFR in turn to allosterically activate BAK1.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555081">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555081" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555081">
        <p class="paperTitle">Pigment Binding in The Light-Dependent Protochlorophyllide Oxidoreductase</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555081" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555081" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pesara, P.; Szafran, K.; Nguyen, H. C.; Sirohiwal, A.; Pantazis, D. A.; Gabruk, M.</p>
        <p class="info">Score: 7.9, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555081' target='https://doi.org/10.1101/2023.08.28.555081'> 10.1101/2023.08.28.555081</a></p>
        <p class="abstract">The Light-Dependent Protochlorophyllide Oxidoreductase (LPOR) is a key enzyme in chlorophyll biosynthesis and its photocatalytic mechanism has long intrigued researchers. However, the lack of structural data for the active complex has impeded understanding of its reaction mechanism. A recent high-resolution structure of enzyme in the active conformation has established a robust foundation for validating hypotheses concerning pigment binding, residue involvement, and consequently, the reaction mechanism. Surprisingly, this new structure challenges previously proposed mechanisms, especially concerning the orientation of the bound protochlorophyllide (Pchlide) pigment. In this study, we employ molecular dynamics and hybrid quantum-mechanics/molecular-mechanics (QM/MM) simulations along with site-directed mutagenesis to compare two Pchlide binding modes: one aligned with previous proposals (mode A), and the other consistent with the recent experimental data (mode B). Binding energy calculations reveal thermodynamic instability of binding mode A due to nonspecific interactions, while mode B exhibits distinct stabilizing interactions yielding favorable binding. QM/MM-based local energy decomposition analysis unravels a complex interaction network that reinforces pigment stabilization in this conformation. Notably, interactions involving Tyr177, His319, and the carboxyl group at C131 influence the pigments excited state energy and potentially contributing to the substrate specificity of the enzyme. Our results uniformly favor binding mode B as represented in the new cryo-EM structure, over the previously assumed mode A. These findings challenge established interpretations and underscore the need for a comprehensive re-evaluation of the reaction mechanism of LPOR that correctly considers pigment interactions and substrate orientation in the binding pocket.

Significance StatementA crucial step in the biosynthesis of the all-important photosynthetic pigment chlorophyll is the reduction of a double C=C bond in its precursor protochlorophyllide (PChlide). This is catalyzed by the Light-Dependent Protochlorophyllide Oxidoreductase (LPOR) via an extremely rare example of a biological photocatalytic reaction. Understanding of the LPOR mechanism has been hindered by limited insight into the structure of its active complex. A recent high-resolution LPOR cryo-EM structure substantiates pigment binding, residue interactions, and the reaction mechanism, but contrasts markedly with all previous assumptions regarding the binding mode of the substrate PChlide. Using molecular dynamics simulations, quantum-mechanics/molecular-mechanics calculations, and mutagenesis, we compare and evaluate the two possible Pchlide binding modes, the one assumed previously (mode A) and the one supported by recent data (mode B). Our findings conclusively favor mode B, challenging prior assumptions and pointing toward novel mechanistic possibilities for this unique photocatalytic reaction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555062">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555062" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555062">
        <p class="paperTitle">Discovery of 2-amide-3-methylester thiophenes inhibiting SARS-CoV-2 ADP-ribosyl hydrolysing macrodomain and coronavirus replication</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555062" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555062" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wazir, S.; Parviainen, T. A. O.; Maksimainen, M. M.; Duong, M. T. H.; Pfannenstiel, J. J.; Cluff, D.; Sowa, S. T.; Galera-Prat, A.; Ferraris, D. V.; Fehr, A.; Heiskanen, J. P.; Lehtiö, L.</p>
        <p class="info">Score: 6.1, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555062' target='https://doi.org/10.1101/2023.08.28.555062'> 10.1101/2023.08.28.555062</a></p>
        <p class="abstract">The COVID-19 pandemic caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) virus has made it clear that further development of antiviral therapies will be needed to combat additional SARS-CoV-2 variants or novel CoVs. Here, we describe small molecule inhibitors for SARS-CoV-2 Mac1, which counters ADP-ribosylation mediated innate immune responses. The compounds inhibiting Mac1 were discovered through high-throughput screening (HTS) using a protein FRET-based competition assay and the best hit compound had an IC50 of 14 M. Three validated HTS hits have the same 2-amide-3-methylester thiophene scaffold and the scaffold was selected for structure-activity relationship (SAR) studies through commercial and synthesized analogs. We studied the compound binding mode in detail using X-ray crystallography and this allowed us to focus on specific features of the compound and design analogs. Compound 27 (MDOLL-0229) had an IC50 of 2.1 M and was generally selective for CoV Mac1 proteins after profiling for activity against a panel of viral and human ADP-ribose binding proteins. The improved potency allowed testing of its effect on virus replication and indeed, 27 inhibited replication of a mouse hepatitis virus, a prototype CoV. Compound 27 is the first Mac1 targeted small molecule demonstrated to inhibit coronavirus replication in a cell model. This, together with its well-defined binding mode, makes 27 a good candidate for further hit/lead-optimization efforts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.554880">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.554880" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.554880">
        <p class="paperTitle">Discovery and engineering of AiEvo2, a novel Cas12a nuclease for human gene editing applications</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.554880" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.554880" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sharrar, A.; Arake de Tacca, L.; Meacham, Z.; Staples-Ager, J.; Collingwood, T.; Rabuka, D.; Schelle, M.</p>
        <p class="info">Score: 5.6, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.554880' target='https://doi.org/10.1101/2023.08.28.554880'> 10.1101/2023.08.28.554880</a></p>
        <p class="abstract">The precision of gene editing technology is critical to creating safe and effective therapies for treating human disease. While the programmability of CRISPR-Cas systems has allowed for rapid innovation of new gene editing techniques, the off-target activity of these enzymes has hampered clinical development for novel therapeutics. Here we report the identification and characterization of a novel CRISPR-Cas12a enzyme from Acinetobacter indicus (AiCas12a). We then engineer the nuclease (termed AiEvo2) for increased specificity, PAM recognition, and efficacy on a variety of human clinical targets. AiEvo2 is highly precise and able to efficiently discriminate between normal and disease-causing alleles in Huntingtons patient derived cells by taking advantage of a single nucleotide polymorphism on the disease-associated allele. AiEvo2 efficiently edits several liver-associated target genes including PCSK9 and TTR when delivered to primary hepatocytes as mRNA encapsulated in a lipid nanoparticle. The enzyme also engineers an effective CD19 CAR-T therapy from primary human T cells using multiplexed simultaneous editing and CAR insertion. To further ensure precise editing, we engineered an anti-CRISPR protein (ErAcr) to selectively inhibit off-target gene editing while retaining therapeutic on-target editing. The engineered AiEvo2 nuclease coupled with a novel ErAcr protein represents a new way to control the fidelity of editing and improve the safety and efficacy of gene editing therapies.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
