<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Broad receptor tropism and immunogenicity of a clade 3 sarbecovirus
Authors: Lee, J.; Zepeda, S.; Park, Y.-J.; Taylor, A. L.; Quispe, J.; Stewart, C.; Leaf, E. M.; Treichel, C.; Corti, D.; King, N. P.; Starr, T. N.; Veesler, D.
Score: 102.9, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557371
Although Rhinolophus bats harbor diverse clade 3 sarbecoviruses, the structural determinants of receptor tropism along with the antigenicity of their spike (S) glycoproteins remain uncharacterized.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Broad receptor tropism and immunogenicity of a clade 3 sarbecovirus
Authors: Lee, J.; Zepeda, S.; Park, Y.-J.; Taylor, A. L.; Quispe, J.; Stewart, C.; Leaf, E. M.; Treichel, C.; Corti, D.; King, N. P.; Starr, T. N.; Veesler, D.
Score: 102.9, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557371
Although Rhinolophus bats harbor diverse clade 3 sarbecoviruses, the structural determinants of receptor tropism along with the antigenicity of their spike (S) glycoproteins remain uncharacterized." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-17T10:36:50+00:00" />
<meta property="article:modified_time" content="2023-09-17T10:36:50+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Broad receptor tropism and immunogenicity of a clade 3 sarbecovirus
Authors: Lee, J.; Zepeda, S.; Park, Y.-J.; Taylor, A. L.; Quispe, J.; Stewart, C.; Leaf, E. M.; Treichel, C.; Corti, D.; King, N. P.; Starr, T. N.; Veesler, D.
Score: 102.9, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557371
Although Rhinolophus bats harbor diverse clade 3 sarbecoviruses, the structural determinants of receptor tropism along with the antigenicity of their spike (S) glycoproteins remain uncharacterized."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Broad receptor tropism and immunogenicity of a clade 3 sarbecovirus\nAuthors: Lee, J.; Zepeda, S.; Park, Y.-J.; Taylor, A. L.; Quispe, J.; Stewart, C.; Leaf, E. M.; Treichel, C.; Corti, D.; King, N. P.; Starr, T. N.; Veesler, D.\nScore: 102.9, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557371\nAlthough Rhinolophus bats harbor diverse clade 3 sarbecoviruses, the structural determinants of receptor tropism along with the antigenicity of their spike (S) glycoproteins remain uncharacterized.",
  "keywords": [
    
  ],
  "articleBody": " Broad receptor tropism and immunogenicity of a clade 3 sarbecovirus\nAuthors: Lee, J.; Zepeda, S.; Park, Y.-J.; Taylor, A. L.; Quispe, J.; Stewart, C.; Leaf, E. M.; Treichel, C.; Corti, D.; King, N. P.; Starr, T. N.; Veesler, D.\nScore: 102.9, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557371\nAlthough Rhinolophus bats harbor diverse clade 3 sarbecoviruses, the structural determinants of receptor tropism along with the antigenicity of their spike (S) glycoproteins remain uncharacterized. Here, we show that the African Rinolophus bat clade 3 sarbecovirus PRD-0038 S has a broad ACE2 usage and that RBD mutations further expand receptor promiscuity and enable human ACE2 utilization. We determined a cryoEM structure of the PRD-0038 RBD bound to R. alcyone ACE2, explaining receptor tropism and highlighting differences with SARS-CoV-1 and SARS-CoV-2. Characterization of PRD-0038 S using cryoEM and monoclonal antibody reactivity revealed its distinct antigenicity relative to SARS-CoV-2 and identified PRD-0038 cross-neutralizing antibodies for pandemic preparedness. PRD-0038 S vaccination elicited greater titers of antibodies cross-reacting with vaccine-mismatched clade 2 and clade 1a sarbecoviruses compared to SARS-CoV-2 S due to broader antigenic targeting, motivating the inclusion of clade 3 antigens in next-generation vaccines for enhanced resilience to viral evolution.\nThe potential of plasma HLA peptides beyond neoepitopes\nAuthors: Wahle, M.; Thielert, M.; Zwiebel, M.; Skowronek, P.; Zeng, W.-F.; Mann, M.\nScore: 28.7, Published: 2023-09-05 DOI: 10.1101/2023.09.05.556309\nDistinction of non-self from self is the major task of the immune system. Immunopeptidomics studies the peptide repertoire presented by the human leukocyte antigen (HLA) protein, usually on tissues. However, HLA peptides are also bound to plasma soluble HLA (sHLA), but little is known about their origin and potential for biomarker discovery in this readily available biofluid. Currently, immunopeptidomics is hampered by complex workflows and limited sensitivity, generally requiring several mL of plasma for the detection of hundreds of HLA peptides. Here, we take advantage of recent improvements in the throughput and sensitivity of mass spectrometry (MS)-based proteomics to develop a highly-sensitive, automated and economical workflow for HLA peptide analysis, termed Immunopeptidomics by Biotinylated Antibodies and Streptavidin (IMBAS). IMBAS-MS quantifies more than 5,000 HLA class I peptides from only 200 L of plasma, in just 30 minutes. Our technology revealed that the plasma immunopeptidome of healthy donors is remarkably stable throughout a year and strongly correlated between individuals with overlapping HLA types. Immunopeptides originating from diverse tissues, including the brain, are proportionately represented. We conclude that sHLAs are a promising avenue for immunology and precision oncology.\nDroplet based low input proteomic platform for rare cell populations\nAuthors: Waas, M.; Khoo, A.; Tharmapalan, P.; McCloskey, C. W.; Govindarajan, M.; Zhang, B.; Khan, S.; Waterhouse, P.; Khokha, R.; Kislinger, T.\nScore: 23.6, Published: 2023-09-11 DOI: 10.1101/2023.09.11.557098\nDeep proteomic profiling of rare cell populations has been constrained by sample input requirements. Here, we present DROPPS, an accessible low-input platform that generates high-fidelity proteomic profiles of 100 - 2,500 cells. By applying DROPPS within the mammary epithelium, we elucidated the connection between mitochondrial activity and clonogenicity, discovering and validating CD36 as a marker of progenitor capacity in the basal cell compartment. We anticipate DROPPS will accelerate biology-driven proteomic research for a multitude of rare cell populations.\nZinc is Essential for the Copper Reductase Activity of Yeast Nucleosomes\nAuthors: Vogelauer, M.; Cheng, C.; Karimian, A.; Golshan Iranpour, H.; Kurdistani, S. K.\nScore: 15.8, Published: 2023-09-14 DOI: 10.1101/2023.09.14.557765\nThe histone H3-H4 tetramer is a copper reductase enzyme, facilitating the production of cuprous (Cu1+) ions for distribution to copper-dependent enzymes. It was, however, unknown if this enzymatic activity occurred within nucleosomes. To investigate this, we obtained native nucleosomes from Saccharomyces cerevisiae using micrococcal nuclease digestion of chromatin in isolated nuclei and ion-exchange chromatographic purification. The purified nucleosomal fragments robustly reduced Cu2+ to Cu1+ ions, with the optimal activity dependent on the presence of zinc ions. Mutation of the histone H3 histidine 113 (H3H113) residue at the active site substantially reduced the enzymatic activity of nucleosomes, underscoring the catalytic role of histone H3. Consistently, limiting zinc ions reduced intracellular Cu1+ levels and compromised growth, phenotypes that were mitigated by genetically enhancing the copper reductase activity of histone H3. These results indicate that yeast nucleosomes possess copper reductase activity, suggesting that the fundamental unit of eukaryotic chromatin is an enzyme complex.\nNegative allosteric modulation of the μ-opioid receptor\nAuthors: O'Brien, E. S.; Rangari, V. A.; El Daibani, A.; Eans, S. O.; White, B.; Wang, H.; Shiimura, Y.; Krishna Kumar, K.; Appourchaux, K.; Huang, W.; Zhang, C.; Mathiesen, J. M.; Che, T.; McLaughlin, J. P.; Majumdar, S.; Kobilka, B. K.\nScore: 13.1, Published: 2023-09-10 DOI: 10.1101/2023.09.08.556921\nThe {micro}-opioid receptor ({micro}OR) is a well-established target for analgesia, yet conventional opioid receptor agonists cause serious adverse effects, notably addiction and respiratory depression, which have led to the present opioid overdose epidemic. {micro}OR negative allosteric modulators (NAMs) may serve as powerful tools in preventing opioid overdose deaths, but promising chemical scaffolds remain elusive. We screened a large DNA-encoded chemical library against inactive {micro}OR, counter-screening with active, G-protein and agonist bound receptor to \"steer\" selections toward functional negative allosteric modulators. We discovered a NAM compound with high and selective enrichment to inactive {micro}OR; the molecule potently blocks the activity of orthosteric agonists and enhances the affinity of the key opioid overdose reversal molecule, naloxone. It accomplishes this by binding to a site on the extracellular vestibule proximal to naloxone, stabilizing a unique inactive conformation of the extracellular portions of the second and seventh transmembrane helices. The NAM perturbs orthosteric ligand kinetics in therapeutically desirable ways and works cooperatively with low doses of naloxone in vivo to inhibit morphine-induced antinociception, respiratory depression and conditioned place preference while minimizing withdrawal behaviors. Our results provide detailed structural insights into the mechanism of a negative allosteric modulator for the {micro}OR and demonstrate how it can be exploited in vivo.\nPhotoswitchable binders enable temporal dissection of endogenous protein function\nAuthors: Westberg, M.; Song, D.; Duong, V.; Fernandez, D.; Huang, P.-S.; Lin, M. Z.\nScore: 12.8, Published: 2023-09-14 DOI: 10.1101/2023.09.14.557687\nGeneral methods for spatiotemporal control of specific endogenous proteins would be broadly useful for probing protein function in living cells. Synthetic protein binders that bind and inhibit endogenous protein targets can be obtained from nanobodies, designed ankyrin repeat proteins (DARPins), and other small protein scaffolds, but generalizable methods to control their binding activity are lacking. Here, we report robust single-chain photoswitchable DARPins (psDARPins) for bidirectional optical control of endogenous proteins. We created topological variants of the DARPin scaffold by computer-aided design so fusion of photodissociable dimeric Dronpa (pdDronpa) results in occlusion of target binding at baseline. Cyan light induces pdDronpa dissociation to expose the binding surface (paratope), while violet light restores pdDronpa dimerization and paratope caging. Since the DARPin redesign leaves the paratope intact, the approach was easily applied to existing DARPins for GFP, ERK, and Ras, as demonstrated by relocalizing GFP-family proteins and inhibiting endogenous ERK and Ras with optical control. Finally, a Ras-targeted psDARPin was used to determine that, following EGF-activation of EGFR, Ras is required for sustained EGFR to ERK signaling. In summary, psDARPins provide a generalizable strategy for precise spatiotemporal dissection of endogenous protein function.\nBiological and Genetic Determinants of Red Blood Cell Glycolysis\nAuthors: Nemkov, T.; Stephenson, D.; Earley, E. J.; Keele, G. R.; Hay, A.; Erickson, C.; Dzieciatkowska, M.; Key, A. M.; Moore, A.; Stone, M.; Deng, X.; Kleinman, S.; Spitalnik, S. L.; Hod, E. A.; Hansen, K.; Churchill, G. A.; Roubinian, N.; Norris, P. J.; Busch, M. P.; Zimring, J. C.; Page, G. P.; D'Alessandro, A.\nScore: 10.7, Published: 2023-09-11 DOI: 10.1101/2023.09.11.557250\nGlycolysis is a central metabolic pathway in health and disease,1 an essential one in mature red blood cells (RBCs), which rely on the Embden-Meyerhof-Parnas pathway, i.e., glycolysis, as the sole source of energy generation.2 During aging in vivo and in vitro under blood bank conditions,3 the adenosine triphosphate (ATP) generated via glycolysis in mitochondria-devoid mature RBCs is essential for the modulation of oxygen kinetics,4 deformability2 and, ultimately, intra- and extra-vascular hemolysis.3 Here we draw upon a cohort of 13,029 volunteers from the Recipient Epidemiology and Donor Evaluation Study (REDS) to identify biological and genetic traits associated with heterogeneity in glycolytic phenotypes. Age, sex and ethnicity influenced glycolysis, with additional effects due to genetic polymorphisms in regions coding for rate-limiting enzymes of glycolysis,1 phosphofructokinase 1 (PFKP) and hexokinase 1 (HK1), and for the ADP-ribosyl cyclase CD38. The gene-metabolite associations were validated by testing glycolysis in fresh and stored RBCs from 350 Diversity Outbred mice.5 These findings were further corroborated via multi-omics analyses of 1,929 samples at storage day 10, 23 and 42 in 643 RBC units from donors who were selected amongst the index cohort based on their extreme hemolytic propensity. Finally, we show that glycolysis, ATP levels, breakdown and deamination into hypoxanthine are associated with increased hemolytic propensity and vesiculation ex vivo, as well as with extravascular hemolysis in vivo, both in healthy autologous transfusion recipients from the Donor Iron Deficiency Study and in 4,700 heterologous critically ill patients receiving blood products from the REDS donors. This work advances our understanding of genetic and non-genetic factors that regulate glycolysis in mature RBCs, a simplified model of mammalian cell metabolism,6 a cell type where dysregulated glycolysis holds broader implications in hemolytic disorders, offering potential avenues for novel therapeutic interventions and transfusion strategies. Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=200 SRC=\"FIGDIR/small/557250v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (90K): org.highwire.dtl.DTLVardef@1c2ee13org.highwire.dtl.DTLVardef@9c34c3org.highwire.dtl.DTLVardef@14bfeeeorg.highwire.dtl.DTLVardef@4df457_HPS_FORMAT_FIGEXP M_FIG C_FIG\nMachine learning to classify mutational hotspots from molecular dynamics simulations\nAuthors: Davies, J.; Menzies, G. E.\nScore: 9.7, Published: 2023-09-10 DOI: 10.1101/2023.09.07.556625\nBenzo[a]pyrene, a notorious DNA-damaging carcinogen, belongs to the family of polycyclic aromatic hydrocarbons commonly found in tobacco smoke. Surprisingly, nucleotide excision repair (NER) machinery exhibits inefficiency in recognising specific bulky DNA adducts including Benzo[a]pyrene Diol-Epoxide (BPDE), a Benzo[a]pyrene metabolite. While sequence context is emerging as the leading factor linking the inadequate NER response to BPDE adducts, the precise structural attributes governing these disparities remain inadequately understood. We therefore combined the domains of molecular dynamics and machine learning to conduct a comprehensive assessment of helical distortion caused by BPDE-Guanine adducts in multiple gene contexts. Specifically, we implemented a dual approach involving a random forest classification-based analysis and subsequent feature selection to identify precise topological features that may distinguish adduct sites of variable repair capacity. Our models were trained using helical data extracted from duplexes representing both BPDE hotspot and non-hotspot sites within the TP53 gene, then applied to sites within TP53, cII, and lacZ genes. We show our optimised model consistently achieved exceptional performance, with accuracy, precision, and f1 scores exceeding 91%. Our feature selection approach uncovered that discernible variance in regional base pair rotation played a pivotal role in informing the decisions of our model. Notably, these disparities were highly conserved among TP53 and lacZ duplexes and appeared to be influenced by the regional GC content. As such, our findings suggest that there are indeed conserved topological features distinguishing hotspots and non-hotpot sites, highlighting regional GC content as a potential biomarker for mutation. Author SummaryAlthough much is known about DNA repair processes, we are still lacking some fundamental understanding relating to DNA sequence and mutation rates, specifically why some sequences mutate at a higher rate or are repaired less than others. We believe that by using a combination of Molecular Simulation and Machine Learning (ML) we can measure which structural features are present in sequences which mutate at higher rates in cancer gene and lab-based test assays frequently used to investigate toxicology. Here we have run Molecular Dynamics on five sets of DNA sequences with and without a carcinogen found in cigarette smoke to allow us to study the mutation event that would need to be repaired. We have measured their helical and base stacking properties. We have used ML to successfully differentiate between low and high mutating sequences using this model allowing us to begin to elucidate the structural features these groups have in common. We believe this method could have wide reaching uses, it could be applied to any gene context and mutation event and indeed the knowledge of the structural features which are best repaired gives us insight into the biophysics of DNA repair adding knowledge to the drug design pipeline.\nFaa1 membrane binding drives positive feedback in autophagosome biogenesis via fatty acid activation\nAuthors: Baumann, V.; Achleitner, S.; Tulli, S.; Schuschnig, M.; Klune, L.; Martens, S.\nScore: 9.2, Published: 2023-09-09 DOI: 10.1101/2023.09.09.556965\nAutophagy serves as a stress response pathway by mediating the degradation of cellular material within lysosomes. In autophagy this material is encapsulated in double membrane vesicles termed autophagosomes, which form from precursors referred to as phagophores. Phagophores grow by lipid influx from the endoplasmic reticulum into Atg9-positive compartments and local lipid synthesis provides lipids for their expansion. How phagophore nucleation and expansion are coordinated with lipid synthesis is unclear. Here, we show that Faa1, an enzyme activating fatty acids, is directly recruited to Atg9 vesicles. We further show that Faa1 binds to negatively charged membranes. We define the membrane binding surface in Faa1 and show that membrane binding is required for its enzymatic activity. In cells, membrane binding by Faa1 is required for its recruitment to phagophores and promotes autophagosome biogenesis. Our results suggest a positive feedback loop coupling phagophore nucleation and expansion to lipid synthesis. SummaryBaumann, Achleitner, Tulli et al. dissect Faa1 function and recruitment during autophagy. They discover that Faa1 directly binds membranes via a positively charged surface. This is a prerequisite for Faa1s enzymatic activity sustaining autophagosome biogenesis.\nConformational changes in the Niemann-Pick Type C1 protein NCR1 drive sterol translocation\nAuthors: Frain, K. M.; Dedic, E.; Nel, L.; Bohush, A.; Olesen, E.; Stokes, D. L.; Pedersen, B. P.\nScore: 9.0, Published: 2023-09-08 DOI: 10.1101/2023.09.08.556848\nThe membrane protein Niemann-Pick Type C1 protein (NPC1, named NCR1 in yeast) is central to sterol homeostasis in eukaryotes. Saccharomyces cerevisiae NCR1 is localized to the vacuolar membrane, where it is suggested to carry sterols across the protective glycocalyx and deposit them into the vacuolar membrane. However, documentation of a vacuolar glycocalyx in fungi is lacking and the mechanism for sterol translocation has remained unclear. Here we provide evidence that a glycocalyx is indeed present inside isolated Saccharomyces cerevisiae vacuoles, and report four cryo-EM structures of NCR1 in two distinct conformations that elucidate how it moves sterol through the glycocalyx. The two conformations, named \"tense\" and \"relaxed\", illustrate movement of sterol through a tunnel formed by the luminal domains. Based on these structures and on comparison with other members of the Resistance-Nodulation-Division (RND) superfamily we propose a transport model that links changes in the luminal domains with a cycle of protonation and deprotonation within the transmembrane region of the protein. Our model suggests that NPC proteins work by a generalized RND mechanism where the transmembrane domains form a motor-unit that sequentially adopts a tense and relaxed conformation to drive changes in luminal/extracellular domains. SIGNIFICANCE STATEMENTNiemann-Pick Type C1 (NPC1, named NCR1 in yeast) proteins play a critical role in sterol homeostasis by facilitating the integration of sterols into membranes of acidic organelles like lysosomes and vacuoles. The inner surface of these organelles membranes is shielded by the glycocalyx. Here, we provide evidence that a glycocalyx is present in vacuoles from Saccharomyces cerevisiae and demonstrate that NCR1 transports sterols across it by undergoing conformational changes. Our structures suggest a transport model where sterol transport is linked to proton-driven changes in the transmembrane region. This work sheds light on the mechanism of NPC1 protein function and has broad implications for understanding lysosomal storage disorders and for mechanisms employed by members of the Resistance-Nodulation-Division (RND) superfamily.\n",
  "wordCount" : "2582",
  "inLanguage": "en",
  "datePublished": "2023-09-17T10:36:50Z",
  "dateModified": "2023-09-17T10:36:50Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta"><span>updated on September 17, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557371">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557371" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557371">
        <p class="paperTitle">Broad receptor tropism and immunogenicity of a clade 3 sarbecovirus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557371" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557371" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, J.; Zepeda, S.; Park, Y.-J.; Taylor, A. L.; Quispe, J.; Stewart, C.; Leaf, E. M.; Treichel, C.; Corti, D.; King, N. P.; Starr, T. N.; Veesler, D.</p>
        <p class="info">Score: 102.9, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557371' target='https://doi.org/10.1101/2023.09.12.557371'> 10.1101/2023.09.12.557371</a></p>
        <p class="abstract">Although Rhinolophus bats harbor diverse clade 3 sarbecoviruses, the structural determinants of receptor tropism along with the antigenicity of their spike (S) glycoproteins remain uncharacterized. Here, we show that the African Rinolophus bat clade 3 sarbecovirus PRD-0038 S has a broad ACE2 usage and that RBD mutations further expand receptor promiscuity and enable human ACE2 utilization. We determined a cryoEM structure of the PRD-0038 RBD bound to R. alcyone ACE2, explaining receptor tropism and highlighting differences with SARS-CoV-1 and SARS-CoV-2. Characterization of PRD-0038 S using cryoEM and monoclonal antibody reactivity revealed its distinct antigenicity relative to SARS-CoV-2 and identified PRD-0038 cross-neutralizing antibodies for pandemic preparedness. PRD-0038 S vaccination elicited greater titers of antibodies cross-reacting with vaccine-mismatched clade 2 and clade 1a sarbecoviruses compared to SARS-CoV-2 S due to broader antigenic targeting, motivating the inclusion of clade 3 antigens in next-generation vaccines for enhanced resilience to viral evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.05.556309">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.05.556309" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.05.556309">
        <p class="paperTitle">The potential of plasma HLA peptides beyond neoepitopes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.05.556309" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.05.556309" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wahle, M.; Thielert, M.; Zwiebel, M.; Skowronek, P.; Zeng, W.-F.; Mann, M.</p>
        <p class="info">Score: 28.7, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.05.556309' target='https://doi.org/10.1101/2023.09.05.556309'> 10.1101/2023.09.05.556309</a></p>
        <p class="abstract">Distinction of non-self from self is the major task of the immune system. Immunopeptidomics studies the peptide repertoire presented by the human leukocyte antigen (HLA) protein, usually on tissues. However, HLA peptides are also bound to plasma soluble HLA (sHLA), but little is known about their origin and potential for biomarker discovery in this readily available biofluid. Currently, immunopeptidomics is hampered by complex workflows and limited sensitivity, generally requiring several mL of plasma for the detection of hundreds of HLA peptides. Here, we take advantage of recent improvements in the throughput and sensitivity of mass spectrometry (MS)-based proteomics to develop a highly-sensitive, automated and economical workflow for HLA peptide analysis, termed Immunopeptidomics by Biotinylated Antibodies and Streptavidin (IMBAS). IMBAS-MS quantifies more than 5,000 HLA class I peptides from only 200 L of plasma, in just 30 minutes. Our technology revealed that the plasma immunopeptidome of healthy donors is remarkably stable throughout a year and strongly correlated between individuals with overlapping HLA types. Immunopeptides originating from diverse tissues, including the brain, are proportionately represented. We conclude that sHLAs are a promising avenue for immunology and precision oncology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557098">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557098" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557098">
        <p class="paperTitle">Droplet based low input proteomic platform for rare cell populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557098" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557098" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Waas, M.; Khoo, A.; Tharmapalan, P.; McCloskey, C. W.; Govindarajan, M.; Zhang, B.; Khan, S.; Waterhouse, P.; Khokha, R.; Kislinger, T.</p>
        <p class="info">Score: 23.6, Published: 2023-09-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557098' target='https://doi.org/10.1101/2023.09.11.557098'> 10.1101/2023.09.11.557098</a></p>
        <p class="abstract">Deep proteomic profiling of rare cell populations has been constrained by sample input requirements. Here, we present DROPPS, an accessible low-input platform that generates high-fidelity proteomic profiles of 100 - 2,500 cells. By applying DROPPS within the mammary epithelium, we elucidated the connection between mitochondrial activity and clonogenicity, discovering and validating CD36 as a marker of progenitor capacity in the basal cell compartment. We anticipate DROPPS will accelerate biology-driven proteomic research for a multitude of rare cell populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.14.557765">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.14.557765" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.14.557765">
        <p class="paperTitle">Zinc is Essential for the Copper Reductase Activity of Yeast Nucleosomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.14.557765" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.14.557765" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vogelauer, M.; Cheng, C.; Karimian, A.; Golshan Iranpour, H.; Kurdistani, S. K.</p>
        <p class="info">Score: 15.8, Published: 2023-09-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.14.557765' target='https://doi.org/10.1101/2023.09.14.557765'> 10.1101/2023.09.14.557765</a></p>
        <p class="abstract">The histone H3-H4 tetramer is a copper reductase enzyme, facilitating the production of cuprous (Cu1&#43;) ions for distribution to copper-dependent enzymes. It was, however, unknown if this enzymatic activity occurred within nucleosomes. To investigate this, we obtained native nucleosomes from Saccharomyces cerevisiae using micrococcal nuclease digestion of chromatin in isolated nuclei and ion-exchange chromatographic purification. The purified nucleosomal fragments robustly reduced Cu2&#43; to Cu1&#43; ions, with the optimal activity dependent on the presence of zinc ions. Mutation of the histone H3 histidine 113 (H3H113) residue at the active site substantially reduced the enzymatic activity of nucleosomes, underscoring the catalytic role of histone H3. Consistently, limiting zinc ions reduced intracellular Cu1&#43; levels and compromised growth, phenotypes that were mitigated by genetically enhancing the copper reductase activity of histone H3. These results indicate that yeast nucleosomes possess copper reductase activity, suggesting that the fundamental unit of eukaryotic chromatin is an enzyme complex.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556921">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556921" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556921">
        <p class="paperTitle">Negative allosteric modulation of the μ-opioid receptor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556921" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556921" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: O&#39;Brien, E. S.; Rangari, V. A.; El Daibani, A.; Eans, S. O.; White, B.; Wang, H.; Shiimura, Y.; Krishna Kumar, K.; Appourchaux, K.; Huang, W.; Zhang, C.; Mathiesen, J. M.; Che, T.; McLaughlin, J. P.; Majumdar, S.; Kobilka, B. K.</p>
        <p class="info">Score: 13.1, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556921' target='https://doi.org/10.1101/2023.09.08.556921'> 10.1101/2023.09.08.556921</a></p>
        <p class="abstract">The {micro}-opioid receptor ({micro}OR) is a well-established target for analgesia, yet conventional opioid receptor agonists cause serious adverse effects, notably addiction and respiratory depression, which have led to the present opioid overdose epidemic. {micro}OR negative allosteric modulators (NAMs) may serve as powerful tools in preventing opioid overdose deaths, but promising chemical scaffolds remain elusive. We screened a large DNA-encoded chemical library against inactive {micro}OR, counter-screening with active, G-protein and agonist bound receptor to &#34;steer&#34; selections toward functional negative allosteric modulators. We discovered a NAM compound with high and selective enrichment to inactive {micro}OR; the molecule potently blocks the activity of orthosteric agonists and enhances the affinity of the key opioid overdose reversal molecule, naloxone. It accomplishes this by binding to a site on the extracellular vestibule proximal to naloxone, stabilizing a unique inactive conformation of the extracellular portions of the second and seventh transmembrane helices. The NAM perturbs orthosteric ligand kinetics in therapeutically desirable ways and works cooperatively with low doses of naloxone in vivo to inhibit morphine-induced antinociception, respiratory depression and conditioned place preference while minimizing withdrawal behaviors. Our results provide detailed structural insights into the mechanism of a negative allosteric modulator for the {micro}OR and demonstrate how it can be exploited in vivo.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.14.557687">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.14.557687" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.14.557687">
        <p class="paperTitle">Photoswitchable binders enable temporal dissection of endogenous protein function</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.14.557687" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.14.557687" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Westberg, M.; Song, D.; Duong, V.; Fernandez, D.; Huang, P.-S.; Lin, M. Z.</p>
        <p class="info">Score: 12.8, Published: 2023-09-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.14.557687' target='https://doi.org/10.1101/2023.09.14.557687'> 10.1101/2023.09.14.557687</a></p>
        <p class="abstract">General methods for spatiotemporal control of specific endogenous proteins would be broadly useful for probing protein function in living cells. Synthetic protein binders that bind and inhibit endogenous protein targets can be obtained from nanobodies, designed ankyrin repeat proteins (DARPins), and other small protein scaffolds, but generalizable methods to control their binding activity are lacking. Here, we report robust single-chain photoswitchable DARPins (psDARPins) for bidirectional optical control of endogenous proteins. We created topological variants of the DARPin scaffold by computer-aided design so fusion of photodissociable dimeric Dronpa (pdDronpa) results in occlusion of target binding at baseline. Cyan light induces pdDronpa dissociation to expose the binding surface (paratope), while violet light restores pdDronpa dimerization and paratope caging. Since the DARPin redesign leaves the paratope intact, the approach was easily applied to existing DARPins for GFP, ERK, and Ras, as demonstrated by relocalizing GFP-family proteins and inhibiting endogenous ERK and Ras with optical control. Finally, a Ras-targeted psDARPin was used to determine that, following EGF-activation of EGFR, Ras is required for sustained EGFR to ERK signaling. In summary, psDARPins provide a generalizable strategy for precise spatiotemporal dissection of endogenous protein function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557250">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557250" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557250">
        <p class="paperTitle">Biological and Genetic Determinants of Red Blood Cell Glycolysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557250" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557250" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nemkov, T.; Stephenson, D.; Earley, E. J.; Keele, G. R.; Hay, A.; Erickson, C.; Dzieciatkowska, M.; Key, A. M.; Moore, A.; Stone, M.; Deng, X.; Kleinman, S.; Spitalnik, S. L.; Hod, E. A.; Hansen, K.; Churchill, G. A.; Roubinian, N.; Norris, P. J.; Busch, M. P.; Zimring, J. C.; Page, G. P.; D&#39;Alessandro, A.</p>
        <p class="info">Score: 10.7, Published: 2023-09-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557250' target='https://doi.org/10.1101/2023.09.11.557250'> 10.1101/2023.09.11.557250</a></p>
        <p class="abstract">Glycolysis is a central metabolic pathway in health and disease,1 an essential one in mature red blood cells (RBCs), which rely on the Embden-Meyerhof-Parnas pathway, i.e., glycolysis, as the sole source of energy generation.2 During aging in vivo and in vitro under blood bank conditions,3 the adenosine triphosphate (ATP) generated via glycolysis in mitochondria-devoid mature RBCs is essential for the modulation of oxygen kinetics,4 deformability2 and, ultimately, intra- and extra-vascular hemolysis.3 Here we draw upon a cohort of 13,029 volunteers from the Recipient Epidemiology and Donor Evaluation Study (REDS) to identify biological and genetic traits associated with heterogeneity in glycolytic phenotypes. Age, sex and ethnicity influenced glycolysis, with additional effects due to genetic polymorphisms in regions coding for rate-limiting enzymes of glycolysis,1 phosphofructokinase 1 (PFKP) and hexokinase 1 (HK1), and for the ADP-ribosyl cyclase CD38. The gene-metabolite associations were validated by testing glycolysis in fresh and stored RBCs from 350 Diversity Outbred mice.5 These findings were further corroborated via multi-omics analyses of 1,929 samples at storage day 10, 23 and 42 in 643 RBC units from donors who were selected amongst the index cohort based on their extreme hemolytic propensity. Finally, we show that glycolysis, ATP levels, breakdown and deamination into hypoxanthine are associated with increased hemolytic propensity and vesiculation ex vivo, as well as with extravascular hemolysis in vivo, both in healthy autologous transfusion recipients from the Donor Iron Deficiency Study and in 4,700 heterologous critically ill patients receiving blood products from the REDS donors. This work advances our understanding of genetic and non-genetic factors that regulate glycolysis in mature RBCs, a simplified model of mammalian cell metabolism,6 a cell type where dysregulated glycolysis holds broader implications in hemolytic disorders, offering potential avenues for novel therapeutic interventions and transfusion strategies.

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=200 SRC=&#34;FIGDIR/small/557250v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (90K):
org.highwire.dtl.DTLVardef@1c2ee13org.highwire.dtl.DTLVardef@9c34c3org.highwire.dtl.DTLVardef@14bfeeeorg.highwire.dtl.DTLVardef@4df457_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556625">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556625" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556625">
        <p class="paperTitle">Machine learning to classify mutational hotspots from molecular dynamics simulations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556625" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556625" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Davies, J.; Menzies, G. E.</p>
        <p class="info">Score: 9.7, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556625' target='https://doi.org/10.1101/2023.09.07.556625'> 10.1101/2023.09.07.556625</a></p>
        <p class="abstract">Benzo[a]pyrene, a notorious DNA-damaging carcinogen, belongs to the family of polycyclic aromatic hydrocarbons commonly found in tobacco smoke. Surprisingly, nucleotide excision repair (NER) machinery exhibits inefficiency in recognising specific bulky DNA adducts including Benzo[a]pyrene Diol-Epoxide (BPDE), a Benzo[a]pyrene metabolite. While sequence context is emerging as the leading factor linking the inadequate NER response to BPDE adducts, the precise structural attributes governing these disparities remain inadequately understood. We therefore combined the domains of molecular dynamics and machine learning to conduct a comprehensive assessment of helical distortion caused by BPDE-Guanine adducts in multiple gene contexts. Specifically, we implemented a dual approach involving a random forest classification-based analysis and subsequent feature selection to identify precise topological features that may distinguish adduct sites of variable repair capacity. Our models were trained using helical data extracted from duplexes representing both BPDE hotspot and non-hotspot sites within the TP53 gene, then applied to sites within TP53, cII, and lacZ genes.

We show our optimised model consistently achieved exceptional performance, with accuracy, precision, and f1 scores exceeding 91%. Our feature selection approach uncovered that discernible variance in regional base pair rotation played a pivotal role in informing the decisions of our model. Notably, these disparities were highly conserved among TP53 and lacZ duplexes and appeared to be influenced by the regional GC content. As such, our findings suggest that there are indeed conserved topological features distinguishing hotspots and non-hotpot sites, highlighting regional GC content as a potential biomarker for mutation.

Author SummaryAlthough much is known about DNA repair processes, we are still lacking some fundamental understanding relating to DNA sequence and mutation rates, specifically why some sequences mutate at a higher rate or are repaired less than others. We believe that by using a combination of Molecular Simulation and Machine Learning (ML) we can measure which structural features are present in sequences which mutate at higher rates in cancer gene and lab-based test assays frequently used to investigate toxicology.

Here we have run Molecular Dynamics on five sets of DNA sequences with and without a carcinogen found in cigarette smoke to allow us to study the mutation event that would need to be repaired. We have measured their helical and base stacking properties. We have used ML to successfully differentiate between low and high mutating sequences using this model allowing us to begin to elucidate the structural features these groups have in common.

We believe this method could have wide reaching uses, it could be applied to any gene context and mutation event and indeed the knowledge of the structural features which are best repaired gives us insight into the biophysics of DNA repair adding knowledge to the drug design pipeline.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.09.556965">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.09.556965" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.09.556965">
        <p class="paperTitle">Faa1 membrane binding drives positive feedback in autophagosome biogenesis via fatty acid activation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.09.556965" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.09.556965" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Baumann, V.; Achleitner, S.; Tulli, S.; Schuschnig, M.; Klune, L.; Martens, S.</p>
        <p class="info">Score: 9.2, Published: 2023-09-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.09.556965' target='https://doi.org/10.1101/2023.09.09.556965'> 10.1101/2023.09.09.556965</a></p>
        <p class="abstract">Autophagy serves as a stress response pathway by mediating the degradation of cellular material within lysosomes. In autophagy this material is encapsulated in double membrane vesicles termed autophagosomes, which form from precursors referred to as phagophores. Phagophores grow by lipid influx from the endoplasmic reticulum into Atg9-positive compartments and local lipid synthesis provides lipids for their expansion. How phagophore nucleation and expansion are coordinated with lipid synthesis is unclear. Here, we show that Faa1, an enzyme activating fatty acids, is directly recruited to Atg9 vesicles. We further show that Faa1 binds to negatively charged membranes. We define the membrane binding surface in Faa1 and show that membrane binding is required for its enzymatic activity. In cells, membrane binding by Faa1 is required for its recruitment to phagophores and promotes autophagosome biogenesis. Our results suggest a positive feedback loop coupling phagophore nucleation and expansion to lipid synthesis.

SummaryBaumann, Achleitner, Tulli et al. dissect Faa1 function and recruitment during autophagy. They discover that Faa1 directly binds membranes via a positively charged surface. This is a prerequisite for Faa1s enzymatic activity sustaining autophagosome biogenesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556848">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556848" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556848">
        <p class="paperTitle">Conformational changes in the Niemann-Pick Type C1 protein NCR1 drive sterol translocation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556848" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556848" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Frain, K. M.; Dedic, E.; Nel, L.; Bohush, A.; Olesen, E.; Stokes, D. L.; Pedersen, B. P.</p>
        <p class="info">Score: 9.0, Published: 2023-09-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556848' target='https://doi.org/10.1101/2023.09.08.556848'> 10.1101/2023.09.08.556848</a></p>
        <p class="abstract">The membrane protein Niemann-Pick Type C1 protein (NPC1, named NCR1 in yeast) is central to sterol homeostasis in eukaryotes. Saccharomyces cerevisiae NCR1 is localized to the vacuolar membrane, where it is suggested to carry sterols across the protective glycocalyx and deposit them into the vacuolar membrane. However, documentation of a vacuolar glycocalyx in fungi is lacking and the mechanism for sterol translocation has remained unclear. Here we provide evidence that a glycocalyx is indeed present inside isolated Saccharomyces cerevisiae vacuoles, and report four cryo-EM structures of NCR1 in two distinct conformations that elucidate how it moves sterol through the glycocalyx. The two conformations, named &#34;tense&#34; and &#34;relaxed&#34;, illustrate movement of sterol through a tunnel formed by the luminal domains. Based on these structures and on comparison with other members of the Resistance-Nodulation-Division (RND) superfamily we propose a transport model that links changes in the luminal domains with a cycle of protonation and deprotonation within the transmembrane region of the protein. Our model suggests that NPC proteins work by a generalized RND mechanism where the transmembrane domains form a motor-unit that sequentially adopts a tense and relaxed conformation to drive changes in luminal/extracellular domains.

SIGNIFICANCE STATEMENTNiemann-Pick Type C1 (NPC1, named NCR1 in yeast) proteins play a critical role in sterol homeostasis by facilitating the integration of sterols into membranes of acidic organelles like lysosomes and vacuoles. The inner surface of these organelles membranes is shielded by the glycocalyx. Here, we provide evidence that a glycocalyx is present in vacuoles from Saccharomyces cerevisiae and demonstrate that NCR1 transports sterols across it by undergoing conformational changes. Our structures suggest a transport model where sterol transport is linked to proton-driven changes in the transmembrane region. This work sheds light on the mechanism of NPC1 protein function and has broad implications for understanding lysosomal storage disorders and for mechanisms employed by members of the Resistance-Nodulation-Division (RND) superfamily.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
