<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="The resting state of the human T-cell receptor
Authors: Notti, R. Q.; Yi, F.; Heissel, S.; Molina, H.; Klebanoff, C. A.; Walz, T.
Score: 28.7, Published: 2023-08-22 DOI: 10.1101/2023.08.22.554360
The T-cell receptor (TCR) is central to the ligand-dependent activation of T lymphocytes and as such orchestrates both adaptive and pathologic immune processes1. However, major questions remain regarding the structure and function of the human TCR2-4. Here, we present cryogenic electron microscopy structures for the unliganded human TCR in a native-like lipid bilayer, revealing three related conformations that are distinct from previously reported structures of receptors in detergent.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="The resting state of the human T-cell receptor
Authors: Notti, R. Q.; Yi, F.; Heissel, S.; Molina, H.; Klebanoff, C. A.; Walz, T.
Score: 28.7, Published: 2023-08-22 DOI: 10.1101/2023.08.22.554360
The T-cell receptor (TCR) is central to the ligand-dependent activation of T lymphocytes and as such orchestrates both adaptive and pathologic immune processes1. However, major questions remain regarding the structure and function of the human TCR2-4. Here, we present cryogenic electron microscopy structures for the unliganded human TCR in a native-like lipid bilayer, revealing three related conformations that are distinct from previously reported structures of receptors in detergent." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-30T10:38:23+00:00" />
<meta property="article:modified_time" content="2023-08-30T10:38:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="The resting state of the human T-cell receptor
Authors: Notti, R. Q.; Yi, F.; Heissel, S.; Molina, H.; Klebanoff, C. A.; Walz, T.
Score: 28.7, Published: 2023-08-22 DOI: 10.1101/2023.08.22.554360
The T-cell receptor (TCR) is central to the ligand-dependent activation of T lymphocytes and as such orchestrates both adaptive and pathologic immune processes1. However, major questions remain regarding the structure and function of the human TCR2-4. Here, we present cryogenic electron microscopy structures for the unliganded human TCR in a native-like lipid bilayer, revealing three related conformations that are distinct from previously reported structures of receptors in detergent."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "The resting state of the human T-cell receptor\nAuthors: Notti, R. Q.; Yi, F.; Heissel, S.; Molina, H.; Klebanoff, C. A.; Walz, T.\nScore: 28.7, Published: 2023-08-22 DOI: 10.1101/2023.08.22.554360\nThe T-cell receptor (TCR) is central to the ligand-dependent activation of T lymphocytes and as such orchestrates both adaptive and pathologic immune processes1. However, major questions remain regarding the structure and function of the human TCR2-4. Here, we present cryogenic electron microscopy structures for the unliganded human TCR in a native-like lipid bilayer, revealing three related conformations that are distinct from previously reported structures of receptors in detergent.",
  "keywords": [
    
  ],
  "articleBody": " The resting state of the human T-cell receptor\nAuthors: Notti, R. Q.; Yi, F.; Heissel, S.; Molina, H.; Klebanoff, C. A.; Walz, T.\nScore: 28.7, Published: 2023-08-22 DOI: 10.1101/2023.08.22.554360\nThe T-cell receptor (TCR) is central to the ligand-dependent activation of T lymphocytes and as such orchestrates both adaptive and pathologic immune processes1. However, major questions remain regarding the structure and function of the human TCR2-4. Here, we present cryogenic electron microscopy structures for the unliganded human TCR in a native-like lipid bilayer, revealing three related conformations that are distinct from previously reported structures of receptors in detergent. These new \"closed and compacted\" conformations afford insights into the interactions between the TCR and the membrane, including conserved surface patches that make extensive outer leaflet contact, and suggest novel conformational regulation by glycans. We show that the closed/compacted conformations, not the extended one previously reported in detergent5-8, represent the unliganded resting state for the TCR in vivo, underscoring the importance of structural interrogation of membrane proteins in native-like environments. We use conformation-locking disulfide mutants to show that juxtamembrane linker extension is necessary for ligand-dependent TCR activation, demonstrating that TCR-intrinsic conformational change is necessary for TCR activation and opening numerous avenues for immunoreceptor engineering.\nAllosteric activation of the co-receptor BAK1 by the EFR receptor kinase initiates immune signaling\nAuthors: Mühlenbeck, H.; Tsutsui, Y.; Lemmon, M.; Bender, K. W.; Zipfel, C.\nScore: 18.4, Published: 2023-08-25 DOI: 10.1101/2023.08.23.554490\nTransmembrane signaling by plant receptor kinases (RKs) has long been thought to involve reciprocal trans-phosphorylation of their intracellular kinase domains. The fact that many of these are pseudokinase domains, however, suggests that additional mechanisms must govern RK signaling activation. Non-catalytic (pseudo)kinase signaling mechanisms have been described in metazoans, but information is scarce for plants. Recently, a non-catalytic function was reported for the leucine-rich repeat (LRR)-RK subfamily XIIa member EFR (ELONGATION FACTOR TU RECEPTOR) and phosphorylation-dependent conformational changes were proposed to regulate signaling of RKs with non-RD kinase domains. Here, using EFR as a model, we describe a non-catalytic activation mechanism for LRR-RKs with non-RD kinase domains. EFR is an active kinase, but a kinase-dead variant retains the ability to enhance catalytic activity of its co-receptor kinase BAK1/SERK3 (BRASSINOSTEROID INSENSITIVE 1-ASSOCIATED KINASE 1/SOMATIC EMBRYOGENESIS RECEPTOR KINASE 3). Applying hydrogen-deuterium exchange mass spectrometry (HDX-MS) analysis and designing homology-based intragenic suppressor mutations, we provide evidence that the EFR kinase domain must adopt its active conformation in order to activate BAK1 allosterically, likely by supporting C-helix positioning in BAK1. Our results suggest a conformational toggle model for signaling, in which BAK1 first phosphorylates EFR in the activation loop to stabilize its active conformation, allowing EFR in turn to allosterically activate BAK1.\niATPSnFR2: a high dynamic range fluorescent sensor for monitoring intracellular ATP\nAuthors: Marvin, J. S.; Kokotos, A. C.; Kumar, M.; Pulido, C.; Tkachuk, A. N.; Yao, J. S.; Brown, T. A.; Ryan, T. A.\nScore: 15.2, Published: 2023-08-24 DOI: 10.1101/2023.08.24.554624\nWe developed a significantly improved genetically encoded quantitative adenosine triphosphate (ATP) sensor to provide real-time dynamics of ATP levels in subcellular compartments. iATPSnFR2 is a variant of iATPSnFR1, a previously developed sensor that has circularly permuted super-folder GFP inserted between the ATP-binding helices of the{varepsilon} -subunit of a bacterial F0-F1 ATPase. Optimizing the linkers joining the two domains resulted in a [~] 5-6 fold improvement in the dynamic range compared to the previous generation sensor, with excellent discrimination against other analytes and affinity variants varying from 4 M to 500 M. A chimeric version of this sensor fused to either the HaloTag protein or a suitably spectrally separated fluorescent protein, provides a ratiometric readout allowing comparisons of ATP across cellular regions. Subcellular targeting of the sensor to nerve terminals reveals previously uncharacterized single synapse metabolic signatures, while targeting to the mitochondrial matrix allowed direct quantitative probing of oxidative phosphorylation dynamics. Significance StatementAdenosine triphosphate (ATP) is a key metabolite necessary for cellular life. Here we develop a next-generation genetically encoded ratiometric fluorescent ATP sensor that allows subcellular tracking of ATP levels in living cells. The large dynamic range makes it possible to follow the dynamics of this metabolite across cells and subcellular regions under different metabolic stressors. We expect that iATPSnFR2 will provide researchers with exciting new opportunities to study ATP dynamics with temporal and spatial resolution that has, until now, been unavailable.\nDe novo design of buttressed loops for sculpting protein functions\nAuthors: Jiang, H.; Jude, K. M.; Wu, K.; Fallas, J.; Ueda, G.; Brunette, T.; Hicks, D.; Pyles, H.; Yang, A.; Carter, L.; Lamb, M.; Li, X.; Levine, P. M.; Stewart, L. J.; Garcia, K. C.; Baker, D.\nScore: 14.8, Published: 2023-08-23 DOI: 10.1101/2023.08.22.554384\nIn natural proteins, structured loops play central roles in molecular recognition, signal transduction and enzyme catalysis. However, because of the intrinsic flexibility and irregularity of loop regions, organizing multiple structured loops at protein functional sites has been very difficult to achieve by de novo protein design. Here we describe a solution to this problem that generates structured loops buttressed by extensive hydrogen bonding interactions with two neighboring loops and with secondary structure elements. We use this approach to design tandem repeat proteins with buttressed loops ranging from 9 to 14 residues in length. Experimental characterization shows the designs are folded and monodisperse, highly soluble, and thermally stable. Crystal structures are in close agreement with the computational design models, with the loops structured and buttressed by their neighbors as designed. We demonstrate the functionality afforded by loop buttressing by designing and characterizing binders for extended peptides in which the loops form one side of an extended binding pocket. The ability to design multiple structured loops should contribute quite generally to efforts to design new protein functions.\nBromodomain Factor 5 as a Target for Antileishmanial Drug Discovery\nAuthors: Russell, C. N.; Carter, J. L.; Borgia, J. M.; Bush, J.; Calderon, F.; Gabarro, R.; Conway, S. J.; Mottram, J. C.; Wilkinson, A. J.; Jones, N. G.\nScore: 11.1, Published: 2023-08-21 DOI: 10.1101/2023.08.21.554125\nLeishmaniases are a collection of neglected tropical diseases caused by kinetoplastid parasites in the genus Leishmania. Current chemotherapies are severely limited and the need for new antileishmanials is of pressing international importance. Bromodomains are epigenetic reader domains that have shown promising therapeutic potential for cancer therapy and may also present an attractive target to treat parasitic diseases. Here, we investigate Leishmania donovani bromodomain factor 5 (LdBDF5) as a target for antileishmanial drug discovery. LdBDF5 contains pair of bromodomains (BD5.1 and BD5.2) in an N-terminal tandem repeat. We purified recombinant bromodomains of L. donovani BDF5 and determined the structure of BD5.2 by X-ray crystallography. Using a histone peptide microarray and fluorescence polarisation assay, we identified binding interactions of LdBDF5 bromodomains with acetylated peptides derived from histones H2B and H4. In orthogonal biophysical assays including thermal shift assays, fluorescence polarisation and NMR, we showed that BDF5 bromodomains bind to human bromodomain inhibitors SGC-CBP30, bromosporine and I-BRD9, moreover, SGC-CBP30 exhibited activity against Leishmania promastigotes in cell viability assays. These findings exemplify the potential BDF5 holds as a drug target in Leishmania and provide a foundation for the future development of optimised antileishmanial compounds targeting this epigenetic reader protein. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=113 SRC=\"FIGDIR/small/554125v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (25K): org.highwire.dtl.DTLVardef@396addorg.highwire.dtl.DTLVardef@196e1a4org.highwire.dtl.DTLVardef@b81666org.highwire.dtl.DTLVardef@1c6a1ae_HPS_FORMAT_FIGEXP M_FIG C_FIG\nA conserved ion channel function of STING mediates non-canonical autophagy and cell death\nAuthors: Xun, J.; Zhang, Z.; Lyu, B.; Lu, D.; Yang, H.; Shang, G.; Tan, J. X.\nScore: 11.0, Published: 2023-08-27 DOI: 10.1101/2023.08.26.554976\nThe cGAS/STING pathway triggers inflammation in response to diverse cellular stresses such as infection, cellular damage, senescence, normal aging, and age-related disease. Besides inflammation, STING also triggers non-canonical autophagy and cell death, the former of which requires the proton pump V-ATPase-mediated LC3 lipidation to single membrane STING vesicles. V-ATPase is known to sense organelle de-acidification in other contexts and recruits the ATG16L1 complex for direct conjugation of LC3/ATG8 to single membranes (CASM). However, it is unclear how STING activates V-ATPase for non-canonical autophagy. Here we report that upon STING activation, the transmembrane domain (TMD) of STING significantly reorganizes and forms an electron-sparse pore in the center. Cellular imaging and in vitro ion flux assays revealed that STING is critical for proton efflux and pH neutralization of Golgi-derived STING vesicles. A chemical ligand of STING, C53, which binds to and blocks the channel of STING strongly inhibited STING-mediated proton flux in vitro and vesicular de-acidification in cells. C53 also abolished STING-dependent LC3 lipidation and cell death. Thus, the ion channel function of STING activates non-canonical autophagy and cell death through vesicle de-acidification.\nBlocking oligomerization is the most viable strategy to inhibit STING\nAuthors: Chan, R. J. M.; Cao, X.; Ergun, S. L.; Njomen, E.; Lynch, S. R.; Cravatt, B. F.; Li, L.\nScore: 15.2, Published: 2023-08-12 DOI: 10.1101/2023.08.11.553045\nAutoimmune disorders are frequently exacerbated by aberrant STING signaling. While STING inhibitors are highly sought after, no viable therapeutic strategies have been reported. When stimulated, STING undergoes a multi-step mechanism to activate downstream interferon-stimulated genes (ISGs), including palmitoylation, translocation from the ER to the Golgi, and oligomerization, but it remains unclear which step(s) are strictly required and therefore should be targeted by potential STING inhibitors. Here, we comprehensively established the order and determined the necessity of each STING activation step. We find that ER-to-Golgi translocation is dispensable for downstream ISG activation, and a previously studied palmitoylation site, C91, is not required for activation in all contexts. Cysteine reactivity mapping and metabolic labeling identified a new palmitoylation site, C64, that is basally palmitoylated and required for STING activation, but is not druggable due to poor solvent accessibility. We confirm that the oligomerization step of STING activation is strictly required, and therefore developed a short peptide mimicking its natural autoinhibitory domain that effectively inhibits STING activation: a proof of concept for the first viable strategy for developing STING inhibitors.\nSUPREM: an engineered non-site-specific m6A RNA methyltransferase with highly improved efficiency\nAuthors: Ochiai, Y.; Clifton, B. E.; Coz, M. L.; Terenzio, M.; Laurino, P.\nScore: 5.5, Published: 2023-08-23 DOI: 10.1101/2023.08.23.554385\nm6A RNA methylation plays a key role in RNA processing and translational regulation, influencing both normal physiological and pathological processes. Yet, current techniques for studying RNA methylation struggle to isolate the effects of individual m6A modifications. Engineering of RNA methyltransferases (RNMTs) could enable development of improved synthetic biology tools to manipulate RNA methylation, but is challenging due to limited understanding of structure-function relationships in RNMTs. Herein, using ancestral sequence reconstruction we explore the sequence space of the bacterial DNA methyltransferase EcoGII (M.EcoGII), a promising target for protein engineering due to its lack of sequence specificity and its residual activity on RNA. We thereby created an efficient non-specific RNMT termed SUPREM, which exhibits 8-fold higher expression levels, 7 {degrees}C higher thermostability, and 12-fold greater m6A RNA methylation activity compared with M.EcoGII. Immunofluorescent staining confirmed SUPREMs higher RNA methylation activity compared with M.EcoGII in mammalian cells. Additionally, Nanopore direct RNA sequencing highlighted that SUPREM is capable of methylating a larger number of RNA methylation sites than M.EcoGII. Through phylogenetic and mutational analysis, we identified a critical residue for the enhanced RNA methylation activity of SUPREM. Collectively, our findings indicate that SUPREM holds promise as a versatile tool for in vivo RNA methylation and labeling.\nMicroglia mediate contact-independent neuronal pruning via secreted Neuraminidase-3 associated with extracellular vesicles\nAuthors: Delaveris, C. S.; Wang, C. L.; Riley, N. M.; Li, S.; Kulkarni, R. U.; Bertozzi, C. R.\nScore: 17.0, Published: 2023-08-22 DOI: 10.1101/2023.08.21.554214\nNeurons communicate with each other through electrochemical transmission at synapses. Microglia, the resident immune cells of the central nervous system, can prune these synapses through a variety of contact-dependent and -independent means. Microglial secretion of active sialidase enzymes upon exposure to inflammatory stimuli is one unexplored mechanism of pruning. Recent work from our lab showed that treatment of neurons with bacterial sialidases disrupts neuronal network connectivity. Here, we find that activated microglia secrete Neuraminidase-3 (Neu3) associated with fusogenic extracellular vesicles. Furthermore, we show Neu3 mediates contact-independent pruning of neurons and subsequent disruption of neuronal networks through neuronal glycocalyx remodeling. We observe that NEU3 is transcriptionally upregulated upon exposure to inflammatory stimuli, and that a genetic knock-out of NEU3 abrogates the sialidase activity of inflammatory microglial secretions. Moreover, we demonstrate that Neu3 is associated with a subpopulation of extracellular vesicles, possibly exosomes, that are secreted by microglia upon inflammatory insult. Finally, we demonstrate that Neu3 is both necessary and sufficient to both desialylate neurons and decrease neuronal network connectivity. These results implicate Neu3 in remodeling of the glycocalyx leading to aberrant network-level activity of neurons, with implications in neuroinflammatory diseases such as Parkinsons disease and Alzheimers disease. Graphical AbstractNeuroinflammation induces secretion of the sialidase Neu3 via extracellular vesicles from microglia that prune neuronal synapses and disrupt neuronal communication. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=76 SRC=\"FIGDIR/small/554214v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (25K): org.highwire.dtl.DTLVardef@137c0f7org.highwire.dtl.DTLVardef@15dab37org.highwire.dtl.DTLVardef@93e0fforg.highwire.dtl.DTLVardef@98be10_HPS_FORMAT_FIGEXP M_FIG C_FIG\nMolecularly imprinted nanoparticles reveal new insights into Pyk2 tyrosine kinase regulatory features\nAuthors: Palhano Zanela, T. M.; Zangiabadi, M.; Zhao, Y.; Underbakke, E. S.\nScore: 5.1, Published: 2023-08-05 DOI: 10.1101/2023.08.05.552112\nPyk2 is a multi-domain non-receptor tyrosine kinase that serves dual roles as signaling enzyme and scaffold. Pyk2 activation involves a multi-stage cascade of conformational rearrangements and protein interactions initiated by autophosphorylation of a linker site. Linker phosphorylation recruits Src kinase, and Src-mediated phosphorylation of the Pyk2 activation loop confers full activation. The regulated accessibility of the initial Pyk2 autophosphorylation site remains unclear. We employed peptide-binding molecularly imprinted nanoparticles (MINPs) to probe the regulatory conformations controlling Pyk2 activation. MINPs differentiating local structure and phosphorylation state revealed that the Pyk2 autophosphorylation site is protected in the autoinhibited state. Activity profiling of Pyk2 variants implicated FERM and linker residues responsible for constraining the autophosphorylation site. MINPs targeting each Src docking site disrupt the higher-order kinase interactions critical for activation complex maturation. Ultimately, MINPs targeting key regulatory motifs establish a useful toolkit for probing successive activational stages in the higher-order Pyk2 activation complex. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=198 SRC=\"FIGDIR/small/552112v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (65K): org.highwire.dtl.DTLVardef@12a2d58org.highwire.dtl.DTLVardef@70498org.highwire.dtl.DTLVardef@bed5beorg.highwire.dtl.DTLVardef@c2df97_HPS_FORMAT_FIGEXP M_FIG C_FIG\n",
  "wordCount" : "2314",
  "inLanguage": "en",
  "datePublished": "2023-08-30T10:38:23Z",
  "dateModified": "2023-08-30T10:38:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta"><span>updated on August 30, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.554360">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.554360" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.554360">
        <p class="paperTitle">The resting state of the human T-cell receptor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.554360" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.554360" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Notti, R. Q.; Yi, F.; Heissel, S.; Molina, H.; Klebanoff, C. A.; Walz, T.</p>
        <p class="info">Score: 28.7, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.554360' target='https://doi.org/10.1101/2023.08.22.554360'> 10.1101/2023.08.22.554360</a></p>
        <p class="abstract">The T-cell receptor (TCR) is central to the ligand-dependent activation of T lymphocytes and as such orchestrates both adaptive and pathologic immune processes1. However, major questions remain regarding the structure and function of the human TCR2-4. Here, we present cryogenic electron microscopy structures for the unliganded human TCR in a native-like lipid bilayer, revealing three related conformations that are distinct from previously reported structures of receptors in detergent. These new &#34;closed and compacted&#34; conformations afford insights into the interactions between the TCR and the membrane, including conserved surface patches that make extensive outer leaflet contact, and suggest novel conformational regulation by glycans. We show that the closed/compacted conformations, not the extended one previously reported in detergent5-8, represent the unliganded resting state for the TCR in vivo, underscoring the importance of structural interrogation of membrane proteins in native-like environments. We use conformation-locking disulfide mutants to show that juxtamembrane linker extension is necessary for ligand-dependent TCR activation, demonstrating that TCR-intrinsic conformational change is necessary for TCR activation and opening numerous avenues for immunoreceptor engineering.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554490">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554490" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554490">
        <p class="paperTitle">Allosteric activation of the co-receptor BAK1 by the EFR receptor kinase initiates immune signaling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554490" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554490" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mühlenbeck, H.; Tsutsui, Y.; Lemmon, M.; Bender, K. W.; Zipfel, C.</p>
        <p class="info">Score: 18.4, Published: 2023-08-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554490' target='https://doi.org/10.1101/2023.08.23.554490'> 10.1101/2023.08.23.554490</a></p>
        <p class="abstract">Transmembrane signaling by plant receptor kinases (RKs) has long been thought to involve reciprocal trans-phosphorylation of their intracellular kinase domains. The fact that many of these are pseudokinase domains, however, suggests that additional mechanisms must govern RK signaling activation. Non-catalytic (pseudo)kinase signaling mechanisms have been described in metazoans, but information is scarce for plants. Recently, a non-catalytic function was reported for the leucine-rich repeat (LRR)-RK subfamily XIIa member EFR (ELONGATION FACTOR TU RECEPTOR) and phosphorylation-dependent conformational changes were proposed to regulate signaling of RKs with non-RD kinase domains. Here, using EFR as a model, we describe a non-catalytic activation mechanism for LRR-RKs with non-RD kinase domains. EFR is an active kinase, but a kinase-dead variant retains the ability to enhance catalytic activity of its co-receptor kinase BAK1/SERK3 (BRASSINOSTEROID INSENSITIVE 1-ASSOCIATED KINASE 1/SOMATIC EMBRYOGENESIS RECEPTOR KINASE 3). Applying hydrogen-deuterium exchange mass spectrometry (HDX-MS) analysis and designing homology-based intragenic suppressor mutations, we provide evidence that the EFR kinase domain must adopt its active conformation in order to activate BAK1 allosterically, likely by supporting C-helix positioning in BAK1. Our results suggest a conformational toggle model for signaling, in which BAK1 first phosphorylates EFR in the activation loop to stabilize its active conformation, allowing EFR in turn to allosterically activate BAK1.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554624">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554624" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554624">
        <p class="paperTitle">iATPSnFR2: a high dynamic range fluorescent sensor for monitoring intracellular ATP</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554624" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554624" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Marvin, J. S.; Kokotos, A. C.; Kumar, M.; Pulido, C.; Tkachuk, A. N.; Yao, J. S.; Brown, T. A.; Ryan, T. A.</p>
        <p class="info">Score: 15.2, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554624' target='https://doi.org/10.1101/2023.08.24.554624'> 10.1101/2023.08.24.554624</a></p>
        <p class="abstract">We developed a significantly improved genetically encoded quantitative adenosine triphosphate (ATP) sensor to provide real-time dynamics of ATP levels in subcellular compartments. iATPSnFR2 is a variant of iATPSnFR1, a previously developed sensor that has circularly permuted super-folder GFP inserted between the ATP-binding helices of the{varepsilon} -subunit of a bacterial F0-F1 ATPase. Optimizing the linkers joining the two domains resulted in a [~] 5-6 fold improvement in the dynamic range compared to the previous generation sensor, with excellent discrimination against other analytes and affinity variants varying from 4 M to 500 M. A chimeric version of this sensor fused to either the HaloTag protein or a suitably spectrally separated fluorescent protein, provides a ratiometric readout allowing comparisons of ATP across cellular regions. Subcellular targeting of the sensor to nerve terminals reveals previously uncharacterized single synapse metabolic signatures, while targeting to the mitochondrial matrix allowed direct quantitative probing of oxidative phosphorylation dynamics.

Significance StatementAdenosine triphosphate (ATP) is a key metabolite necessary for cellular life. Here we develop a next-generation genetically encoded ratiometric fluorescent ATP sensor that allows subcellular tracking of ATP levels in living cells. The large dynamic range makes it possible to follow the dynamics of this metabolite across cells and subcellular regions under different metabolic stressors. We expect that iATPSnFR2 will provide researchers with exciting new opportunities to study ATP dynamics with temporal and spatial resolution that has, until now, been unavailable.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.554384">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.554384" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.554384">
        <p class="paperTitle">De novo design of buttressed loops for sculpting protein functions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.554384" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.554384" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jiang, H.; Jude, K. M.; Wu, K.; Fallas, J.; Ueda, G.; Brunette, T.; Hicks, D.; Pyles, H.; Yang, A.; Carter, L.; Lamb, M.; Li, X.; Levine, P. M.; Stewart, L. J.; Garcia, K. C.; Baker, D.</p>
        <p class="info">Score: 14.8, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.554384' target='https://doi.org/10.1101/2023.08.22.554384'> 10.1101/2023.08.22.554384</a></p>
        <p class="abstract">In natural proteins, structured loops play central roles in molecular recognition, signal transduction and enzyme catalysis. However, because of the intrinsic flexibility and irregularity of loop regions, organizing multiple structured loops at protein functional sites has been very difficult to achieve by de novo protein design. Here we describe a solution to this problem that generates structured loops buttressed by extensive hydrogen bonding interactions with two neighboring loops and with secondary structure elements. We use this approach to design tandem repeat proteins with buttressed loops ranging from 9 to 14 residues in length. Experimental characterization shows the designs are folded and monodisperse, highly soluble, and thermally stable. Crystal structures are in close agreement with the computational design models, with the loops structured and buttressed by their neighbors as designed. We demonstrate the functionality afforded by loop buttressing by designing and characterizing binders for extended peptides in which the loops form one side of an extended binding pocket. The ability to design multiple structured loops should contribute quite generally to efforts to design new protein functions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554125">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554125" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554125">
        <p class="paperTitle">Bromodomain Factor 5 as a Target for Antileishmanial Drug Discovery</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554125" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554125" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Russell, C. N.; Carter, J. L.; Borgia, J. M.; Bush, J.; Calderon, F.; Gabarro, R.; Conway, S. J.; Mottram, J. C.; Wilkinson, A. J.; Jones, N. G.</p>
        <p class="info">Score: 11.1, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554125' target='https://doi.org/10.1101/2023.08.21.554125'> 10.1101/2023.08.21.554125</a></p>
        <p class="abstract">Leishmaniases are a collection of neglected tropical diseases caused by kinetoplastid parasites in the genus Leishmania. Current chemotherapies are severely limited and the need for new antileishmanials is of pressing international importance. Bromodomains are epigenetic reader domains that have shown promising therapeutic potential for cancer therapy and may also present an attractive target to treat parasitic diseases. Here, we investigate Leishmania donovani bromodomain factor 5 (LdBDF5) as a target for antileishmanial drug discovery. LdBDF5 contains pair of bromodomains (BD5.1 and BD5.2) in an N-terminal tandem repeat. We purified recombinant bromodomains of L. donovani BDF5 and determined the structure of BD5.2 by X-ray crystallography. Using a histone peptide microarray and fluorescence polarisation assay, we identified binding interactions of LdBDF5 bromodomains with acetylated peptides derived from histones H2B and H4. In orthogonal biophysical assays including thermal shift assays, fluorescence polarisation and NMR, we showed that BDF5 bromodomains bind to human bromodomain inhibitors SGC-CBP30, bromosporine and I-BRD9, moreover, SGC-CBP30 exhibited activity against Leishmania promastigotes in cell viability assays. These findings exemplify the potential BDF5 holds as a drug target in Leishmania and provide a foundation for the future development of optimised antileishmanial compounds targeting this epigenetic reader protein.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=113 SRC=&#34;FIGDIR/small/554125v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (25K):
org.highwire.dtl.DTLVardef@396addorg.highwire.dtl.DTLVardef@196e1a4org.highwire.dtl.DTLVardef@b81666org.highwire.dtl.DTLVardef@1c6a1ae_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.26.554976">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.26.554976" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.26.554976">
        <p class="paperTitle">A conserved ion channel function of STING mediates non-canonical autophagy and cell death</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.26.554976" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.26.554976" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xun, J.; Zhang, Z.; Lyu, B.; Lu, D.; Yang, H.; Shang, G.; Tan, J. X.</p>
        <p class="info">Score: 11.0, Published: 2023-08-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.26.554976' target='https://doi.org/10.1101/2023.08.26.554976'> 10.1101/2023.08.26.554976</a></p>
        <p class="abstract">The cGAS/STING pathway triggers inflammation in response to diverse cellular stresses such as infection, cellular damage, senescence, normal aging, and age-related disease. Besides inflammation, STING also triggers non-canonical autophagy and cell death, the former of which requires the proton pump V-ATPase-mediated LC3 lipidation to single membrane STING vesicles. V-ATPase is known to sense organelle de-acidification in other contexts and recruits the ATG16L1 complex for direct conjugation of LC3/ATG8 to single membranes (CASM). However, it is unclear how STING activates V-ATPase for non-canonical autophagy. Here we report that upon STING activation, the transmembrane domain (TMD) of STING significantly reorganizes and forms an electron-sparse pore in the center. Cellular imaging and in vitro ion flux assays revealed that STING is critical for proton efflux and pH neutralization of Golgi-derived STING vesicles. A chemical ligand of STING, C53, which binds to and blocks the channel of STING strongly inhibited STING-mediated proton flux in vitro and vesicular de-acidification in cells. C53 also abolished STING-dependent LC3 lipidation and cell death. Thus, the ion channel function of STING activates non-canonical autophagy and cell death through vesicle de-acidification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.553045">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.553045" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.553045">
        <p class="paperTitle">Blocking oligomerization is the most viable strategy to inhibit STING</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.553045" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.553045" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chan, R. J. M.; Cao, X.; Ergun, S. L.; Njomen, E.; Lynch, S. R.; Cravatt, B. F.; Li, L.</p>
        <p class="info">Score: 15.2, Published: 2023-08-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.553045' target='https://doi.org/10.1101/2023.08.11.553045'> 10.1101/2023.08.11.553045</a></p>
        <p class="abstract">Autoimmune disorders are frequently exacerbated by aberrant STING signaling. While STING inhibitors are highly sought after, no viable therapeutic strategies have been reported. When stimulated, STING undergoes a multi-step mechanism to activate downstream interferon-stimulated genes (ISGs), including palmitoylation, translocation from the ER to the Golgi, and oligomerization, but it remains unclear which step(s) are strictly required and therefore should be targeted by potential STING inhibitors. Here, we comprehensively established the order and determined the necessity of each STING activation step. We find that ER-to-Golgi translocation is dispensable for downstream ISG activation, and a previously studied palmitoylation site, C91, is not required for activation in all contexts. Cysteine reactivity mapping and metabolic labeling identified a new palmitoylation site, C64, that is basally palmitoylated and required for STING activation, but is not druggable due to poor solvent accessibility. We confirm that the oligomerization step of STING activation is strictly required, and therefore developed a short peptide mimicking its natural autoinhibitory domain that effectively inhibits STING activation: a proof of concept for the first viable strategy for developing STING inhibitors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554385">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554385" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554385">
        <p class="paperTitle">SUPREM: an engineered non-site-specific m6A RNA methyltransferase with highly improved efficiency</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554385" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554385" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ochiai, Y.; Clifton, B. E.; Coz, M. L.; Terenzio, M.; Laurino, P.</p>
        <p class="info">Score: 5.5, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554385' target='https://doi.org/10.1101/2023.08.23.554385'> 10.1101/2023.08.23.554385</a></p>
        <p class="abstract">m6A RNA methylation plays a key role in RNA processing and translational regulation, influencing both normal physiological and pathological processes. Yet, current techniques for studying RNA methylation struggle to isolate the effects of individual m6A modifications. Engineering of RNA methyltransferases (RNMTs) could enable development of improved synthetic biology tools to manipulate RNA methylation, but is challenging due to limited understanding of structure-function relationships in RNMTs. Herein, using ancestral sequence reconstruction we explore the sequence space of the bacterial DNA methyltransferase EcoGII (M.EcoGII), a promising target for protein engineering due to its lack of sequence specificity and its residual activity on RNA. We thereby created an efficient non-specific RNMT termed SUPREM, which exhibits 8-fold higher expression levels, 7 {degrees}C higher thermostability, and 12-fold greater m6A RNA methylation activity compared with M.EcoGII. Immunofluorescent staining confirmed SUPREMs higher RNA methylation activity compared with M.EcoGII in mammalian cells. Additionally, Nanopore direct RNA sequencing highlighted that SUPREM is capable of methylating a larger number of RNA methylation sites than M.EcoGII. Through phylogenetic and mutational analysis, we identified a critical residue for the enhanced RNA methylation activity of SUPREM. Collectively, our findings indicate that SUPREM holds promise as a versatile tool for in vivo RNA methylation and labeling.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554214">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554214" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554214">
        <p class="paperTitle">Microglia mediate contact-independent neuronal pruning via secreted Neuraminidase-3 associated with extracellular vesicles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554214" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554214" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Delaveris, C. S.; Wang, C. L.; Riley, N. M.; Li, S.; Kulkarni, R. U.; Bertozzi, C. R.</p>
        <p class="info">Score: 17.0, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554214' target='https://doi.org/10.1101/2023.08.21.554214'> 10.1101/2023.08.21.554214</a></p>
        <p class="abstract">Neurons communicate with each other through electrochemical transmission at synapses. Microglia, the resident immune cells of the central nervous system, can prune these synapses through a variety of contact-dependent and -independent means. Microglial secretion of active sialidase enzymes upon exposure to inflammatory stimuli is one unexplored mechanism of pruning. Recent work from our lab showed that treatment of neurons with bacterial sialidases disrupts neuronal network connectivity. Here, we find that activated microglia secrete Neuraminidase-3 (Neu3) associated with fusogenic extracellular vesicles. Furthermore, we show Neu3 mediates contact-independent pruning of neurons and subsequent disruption of neuronal networks through neuronal glycocalyx remodeling. We observe that NEU3 is transcriptionally upregulated upon exposure to inflammatory stimuli, and that a genetic knock-out of NEU3 abrogates the sialidase activity of inflammatory microglial secretions. Moreover, we demonstrate that Neu3 is associated with a subpopulation of extracellular vesicles, possibly exosomes, that are secreted by microglia upon inflammatory insult. Finally, we demonstrate that Neu3 is both necessary and sufficient to both desialylate neurons and decrease neuronal network connectivity. These results implicate Neu3 in remodeling of the glycocalyx leading to aberrant network-level activity of neurons, with implications in neuroinflammatory diseases such as Parkinsons disease and Alzheimers disease.

Graphical AbstractNeuroinflammation induces secretion of the sialidase Neu3 via extracellular vesicles from microglia that prune neuronal synapses and disrupt neuronal communication.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=76 SRC=&#34;FIGDIR/small/554214v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (25K):
org.highwire.dtl.DTLVardef@137c0f7org.highwire.dtl.DTLVardef@15dab37org.highwire.dtl.DTLVardef@93e0fforg.highwire.dtl.DTLVardef@98be10_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.05.552112">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.05.552112" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.05.552112">
        <p class="paperTitle">Molecularly imprinted nanoparticles reveal new insights into Pyk2 tyrosine kinase regulatory features</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.05.552112" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.05.552112" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Palhano Zanela, T. M.; Zangiabadi, M.; Zhao, Y.; Underbakke, E. S.</p>
        <p class="info">Score: 5.1, Published: 2023-08-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.05.552112' target='https://doi.org/10.1101/2023.08.05.552112'> 10.1101/2023.08.05.552112</a></p>
        <p class="abstract">Pyk2 is a multi-domain non-receptor tyrosine kinase that serves dual roles as signaling enzyme and scaffold. Pyk2 activation involves a multi-stage cascade of conformational rearrangements and protein interactions initiated by autophosphorylation of a linker site. Linker phosphorylation recruits Src kinase, and Src-mediated phosphorylation of the Pyk2 activation loop confers full activation. The regulated accessibility of the initial Pyk2 autophosphorylation site remains unclear. We employed peptide-binding molecularly imprinted nanoparticles (MINPs) to probe the regulatory conformations controlling Pyk2 activation. MINPs differentiating local structure and phosphorylation state revealed that the Pyk2 autophosphorylation site is protected in the autoinhibited state. Activity profiling of Pyk2 variants implicated FERM and linker residues responsible for constraining the autophosphorylation site. MINPs targeting each Src docking site disrupt the higher-order kinase interactions critical for activation complex maturation. Ultimately, MINPs targeting key regulatory motifs establish a useful toolkit for probing successive activational stages in the higher-order Pyk2 activation complex.

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=198 SRC=&#34;FIGDIR/small/552112v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (65K):
org.highwire.dtl.DTLVardef@12a2d58org.highwire.dtl.DTLVardef@70498org.highwire.dtl.DTLVardef@bed5beorg.highwire.dtl.DTLVardef@c2df97_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
