<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Generalized Biomolecular Modeling and Design with RoseTTAFold All-Atom
Authors: Krishna, R.; Wang, J.; Ahern, W.; Sturmfels, P.; Venkatesh, P.; Kalvet, I.; Lee, G. R.; Morey-Burrows, F. S.; Anishchenko, I.; Humphreys, I. R.; McHugh, R.; Vafeados, D.; Li, X.; Sutherland, G. A.; Hitchcock, A.; Hunter, C. N.; Baek, M.; DiMaio, F.; Baker, D.
Score: 288.2, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561603
Although AlphaFold2 (AF2) and RoseTTAFold (RF) have transformed structural biology by enabling high-accuracy protein structure modeling, they are unable to model covalent modifications or interactions with small molecules and other non-protein molecules that can play key roles in biological function.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Generalized Biomolecular Modeling and Design with RoseTTAFold All-Atom
Authors: Krishna, R.; Wang, J.; Ahern, W.; Sturmfels, P.; Venkatesh, P.; Kalvet, I.; Lee, G. R.; Morey-Burrows, F. S.; Anishchenko, I.; Humphreys, I. R.; McHugh, R.; Vafeados, D.; Li, X.; Sutherland, G. A.; Hitchcock, A.; Hunter, C. N.; Baek, M.; DiMaio, F.; Baker, D.
Score: 288.2, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561603
Although AlphaFold2 (AF2) and RoseTTAFold (RF) have transformed structural biology by enabling high-accuracy protein structure modeling, they are unable to model covalent modifications or interactions with small molecules and other non-protein molecules that can play key roles in biological function." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-18T10:38:41+00:00" />
<meta property="article:modified_time" content="2023-10-18T10:38:41+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Generalized Biomolecular Modeling and Design with RoseTTAFold All-Atom
Authors: Krishna, R.; Wang, J.; Ahern, W.; Sturmfels, P.; Venkatesh, P.; Kalvet, I.; Lee, G. R.; Morey-Burrows, F. S.; Anishchenko, I.; Humphreys, I. R.; McHugh, R.; Vafeados, D.; Li, X.; Sutherland, G. A.; Hitchcock, A.; Hunter, C. N.; Baek, M.; DiMaio, F.; Baker, D.
Score: 288.2, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561603
Although AlphaFold2 (AF2) and RoseTTAFold (RF) have transformed structural biology by enabling high-accuracy protein structure modeling, they are unable to model covalent modifications or interactions with small molecules and other non-protein molecules that can play key roles in biological function."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Generalized Biomolecular Modeling and Design with RoseTTAFold All-Atom\nAuthors: Krishna, R.; Wang, J.; Ahern, W.; Sturmfels, P.; Venkatesh, P.; Kalvet, I.; Lee, G. R.; Morey-Burrows, F. S.; Anishchenko, I.; Humphreys, I. R.; McHugh, R.; Vafeados, D.; Li, X.; Sutherland, G. A.; Hitchcock, A.; Hunter, C. N.; Baek, M.; DiMaio, F.; Baker, D.\nScore: 288.2, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561603\nAlthough AlphaFold2 (AF2) and RoseTTAFold (RF) have transformed structural biology by enabling high-accuracy protein structure modeling, they are unable to model covalent modifications or interactions with small molecules and other non-protein molecules that can play key roles in biological function.",
  "keywords": [
    
  ],
  "articleBody": " Generalized Biomolecular Modeling and Design with RoseTTAFold All-Atom\nAuthors: Krishna, R.; Wang, J.; Ahern, W.; Sturmfels, P.; Venkatesh, P.; Kalvet, I.; Lee, G. R.; Morey-Burrows, F. S.; Anishchenko, I.; Humphreys, I. R.; McHugh, R.; Vafeados, D.; Li, X.; Sutherland, G. A.; Hitchcock, A.; Hunter, C. N.; Baek, M.; DiMaio, F.; Baker, D.\nScore: 288.2, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561603\nAlthough AlphaFold2 (AF2) and RoseTTAFold (RF) have transformed structural biology by enabling high-accuracy protein structure modeling, they are unable to model covalent modifications or interactions with small molecules and other non-protein molecules that can play key roles in biological function. Here, we describe RoseTTAFold All-Atom (RFAA), a deep network capable of modeling full biological assemblies containing proteins, nucleic acids, small molecules, metals, and covalent modifications given the sequences of the polymers and the atomic bonded geometry of the small molecules and covalent modifications. Following training on structures of full biological assemblies in the Protein Data Bank (PDB), RFAA has comparable protein structure prediction accuracy to AF2, excellent performance in CAMEO for flexible backbone small molecule docking, and reasonable prediction accuracy for protein covalent modifications and assemblies of proteins with multiple nucleic acid chains and small molecules which, to our knowledge, no existing method can model simultaneously. By fine-tuning on diffusive denoising tasks, we develop RFdiffusion All-Atom (RFdiffusionAA), which generates binding pockets by directly building protein structures around small molecules and other non-protein molecules. Starting from random distributions of amino acid residues surrounding target small molecules, we design and experimentally validate proteins that bind the cardiac disease therapeutic digoxigenin, the enzymatic cofactor heme, and optically active bilin molecules with potential for expanding the range of wavelengths captured by photosynthesis. We anticipate that RFAA and RFdiffusionAA will be widely useful for modeling and designing complex biomolecular systems.\nOptimising Spectronaut search parameters to improve data quality with minimal proteome coverage reductions in DIA analyses of heterogeneous samples\nAuthors: Baker, C. P.; Bruderer, R.; Abbott, J. C.; Arthur, S.; Brenes, A. J.\nScore: 24.0, Published: 2023-10-11 DOI: 10.1101/2023.10.11.561927\nData independent acquisition has seen breakthroughs that enable comprehensive proteome profiling using short gradients. As the proteome coverage continues to increase, the quality of the data generated becomes much more relevant. Using Spectronaut, we show that the default search parameters can be easily optimised to minimise the occurrence of false positives across different samples. Using an immunological infection model system to demonstrate the impact of adjusting search settings we analysed mouse macrophages and compared their proteome to macrophages spiked with Candida albicans. This experimental system enabled the identification of false positives since Candida albicans peptides and proteins should not be present in the mouse only samples. We show that adjusting the search parameters reduced false positive identifications by 89% at the peptide and protein level, thereby considerably increasing the quality of the data. We also show that these optimised parameters incur a moderate cost, only reducing the overall number of true positive identifications across each biological replicate by \u003c6.7% at both the peptide and protein level. We believe the value of our updated search parameters extends beyond a two-organism analysis and would be of great value to any DIA experiment analysing heterogenous populations of cell types or tissues.\nShared and Distinct Mechanisms of UBA1 Inactivation Across Different Diseases\nAuthors: Collins, J. C.; Magaziner, S. J.; English, M.; Hassan, B.; Chen, X.; Balanda, N.; Anderson, M.; Lam, A.; Fernandez-Pol, S.; Kwong, B.; Greenberg, P. L.; Terrier, B.; Likhite, M. E.; Kosmider, O.; Wang, Y.; Samara, N. L.; Walters, K. J.; Beck, D. B.; Werner, A.\nScore: 14.0, Published: 2023-10-10 DOI: 10.1101/2023.10.10.561769\nMost cellular ubiquitin signaling is initiated by UBA1, which activates and transfers ubiquitin to tens of E2 enzymes. Clonally acquired UBA1 missense mutations cause an inflammatory-hematologic overlap disease called VEXAS (vacuoles, E1, X-linked, autoinflammatory, somatic) syndrome. Despite extensive clinical investigation into this lethal disease, little is known about the underlying molecular mechanisms. Here, by dissecting VEXAS-causing UBA1 mutations, we discovered that p.Met41 mutations alter cytoplasmic isoform expression, whereas other mutations reduce catalytic activity of nuclear and cytoplasmic isoforms by diverse mechanisms, including aberrant oxyester formation. Strikingly, non-p.Met41 mutations most prominently affect transthioesterification, revealing ubiquitin transfer to cytoplasmic E2 enzymes as a shared property of pathogenesis amongst different VEXAS syndrome genotypes. A similar E2 charging bottleneck exists in some lung cancer-associated UBA1 mutations, but not in spinal muscular atrophy-causing UBA1 mutations, which instead, render UBA1 thermolabile. Collectively, our results highlight the precision of conformational changes required for faithful ubiquitin transfer, define distinct and shared mechanisms of UBA1 inactivation in diverse diseases, and suggest that specific E1-E2 modules control different aspects of tissue differentiation and maintenance.\nStructural and molecular basis of choline uptake into the brain by FLVCR2\nAuthors: Cater, R. J.; Mukherjee, D.; Gil Iturbe, E.; Erramilli, S. K.; Chen, T.; Koo, K.; Santander, N. G.; Reckers, A. R.; Kloss, B.; Gawda, T.; Choy, B. C.; Zhang, Z.; Clarke, O. B.; Yee, S. W.; Kossiakoff, A. K.; Quick, M.; Arnold, T.; Mancia, F.\nScore: 19.2, Published: 2023-10-05 DOI: 10.1101/2023.10.05.561059\nCholine is an essential nutrient that the human body needs in vast quantities for cell membrane synthesis, epigenetic modification, and neurotransmission. The brain has a particularly high demand for choline, but how it enters the brain has eluded the field for over fifty years. The MFS transporter FLVCR1 was recently determined to be a choline transporter, and while this protein is not highly expressed at the blood-brain barrier (BBB), its relative FLVCR2 is. Previous studies have shown that mutations in human Flvcr2 cause cerebral vascular abnormalities, hydrocephalus, and embryonic lethality, but the physiological role of FLVCR2 is unknown. Here, we demonstrate both in vivo and in vitro that FLVCR2 is a BBB choline transporter and is responsible for the majority of choline uptake into the brain. We also determine the structures of choline-bound FLVCR2 in the inward- and outward-facing states using cryo-electron microscopy to 2.49 and 2.77 [A] resolution, respectively. These results reveal how the brain obtains choline and provide molecular-level insights into how FLVCR2 binds choline in an aromatic cage and mediates its uptake. Our work could provide a novel framework for the targeted delivery of neurotherapeutics into the brain.\nStructure of the endosomal CORVET tethering complex\nAuthors: Shvarev, D.; Koenig, C.; Susan, N.; Langemeyer, L.; Walter, S.; Perz, A.; Froehlich, F.; Ungermann, C.; Moeller, A.\nScore: 9.2, Published: 2023-10-13 DOI: 10.1101/2023.10.13.562242\nCells depend on their endolysosomal system for nutrient uptake and downregulation of plasma membrane proteins. These processes rely on endosomal maturation, which requires multiple membrane fusion steps. Early endosome fusion is promoted by the Rab5 GTPase and its effector, the hexameric CORVET tethering complex, which is homologous to the lysosomal HOPS. How these related complexes recognize their specific target membranes remains entirely elusive. Here, we solved the structure of CORVET by cryo-electron microscopy and revealed its minimal requirements for membrane tethering. As expected, the core of CORVET and HOPS resembles each other. However, the function-defining subunits show marked structural differences. Notably, we discovered that unlike HOPS, CORVET depends not only on Rab5 but also on phosphatidylinositol-3-phosphate (PI3P) and membrane packaging defects for tethering, implying that an organelle-specific membrane code enables fusion. Our data suggest that both shape and membrane interactions of CORVET and HOPS are conserved in metazoans, thus providing a paradigm how tethering complexes function.\nThe guide RNA sequence dictates the slicing kinetics and conformational dynamics of the Argonaute silencing complex\nAuthors: Wang, P. Y.; Bartel, D. P.\nScore: 8.8, Published: 2023-10-16 DOI: 10.1101/2023.10.15.562437\nThe RNA-induced silencing complex (RISC), which powers RNA interference (RNAi), consists of a guide RNA and an Argonaute protein that slices target RNAs complementary to the guide. We find that for different guide-RNA sequences, slicing rates of perfectly paired, bound targets can be surprisingly different (\u003e250-fold range), and that faster slicing confers greater knockdown efficacy in cells. Four sequence determinants underlie much of this variation in slicing rates. Analyses of these determinants implicate a structural distortion at guide nucleotides 6-7 in promoting slicing, and indicate that, in contrast to the prevailing model, slicing is primarily limited by the rate at which the enzyme-substrate complex reaches a slicing-competent conformation. Moreover, different guide sequences have an unanticipated, 600-fold range in 3'-mismatch tolerance, which is attributable to the GC-content of their central regions. Together, our analyses identify sequence determinants of RISC activity and provide biochemical, conformational, and functional rationale for their action.\nTMEM65-dependent Ca2+ extrusion safeguards mitochondrial homeostasis\nAuthors: Vetralla, M.; Wischhof, L.; Cadenelli, V.; Scifo, E.; Ehninger, D.; Rizzuto, R.; Bano, D.; De Stefani, D.\nScore: 8.5, Published: 2023-10-11 DOI: 10.1101/2023.10.10.561661\nThe bidirectional transport of Ca2+ into and out of mitochondria is a conserved biological process controlling multiple events, including metabolism, signaling, and cell fate. In the presence of membrane potential driving mitochondrial Ca2+ accumulation, transient changes of [Ca2+]mt in response to cytosolic [Ca2+] variations are ensured by a molecular machinery for Ca2+ influx and efflux embedded in the inner mitochondrial membrane (IMM). While it is well established that influx relies on the Mitochondrial Calcium Uniporter (MCU), efflux was expected to be molecularly diversified, given the occurrence of functionally different exchange pathways with either Na+ or H+1. Accordingly, dedicated transporters ensure proper Ca2+ homeostasis and tightly regulated mitochondrial bioenergetics, but the process is not yet fully elucidated. We here demonstrate that TMEM65, a protein with an unknown biological function, is a fundamental component of the Ca2+ efflux machinery of mitochondria. As the MCU, TMEM65 has a broad tissue expression and localizes to the IMM. Its overexpression dramatically enhances Na+- and Li+-dependent mitochondrial Ca2+ extrusion, which is abrogated by the pharmacological inhibitor CGP-37157. Conversely, its downregulation chronically elevates resting mitochondrial Ca2+ levels and inhibits efficient Ca2+ efflux upon cellular activation, culminating in mitochondrial calcium overload and causing organelle dysfunction. Since TMEM65 has been associated with a severe human mitochondrial disease2, we deleted the TMEM65 homologues in Caenorhabditis elegans (CeTMEM65). While the two CeTMEM65 orthologs are dispensable for the survival at permissive growing conditions, their loss undermines embryonic developments when eggs are exposed to mild temperature-stress. In this regard, we find that CeTMEM65 (null) alleles cause necrotic lesions that are suppressed by inhibiting the mitochondrial calcium uniporter MCU-1. Overall, these results unambiguously assign a primary role in mitochondrial Ca2+ homeostasis to the orphan protein TMEM65. More importantly, our findings describe a novel molecular component that may be relevant in pathological settings in which excessive mitochondrial Ca2+ accumulation critically contribute to degenerative pathways.\nRedox Regulation of Brain Selective Kinases BRSK1/2: Implications for Dynamic Control of the Eukaryotic AMPK family through Cys-based mechanisms\nAuthors: Bendzunas, G.; Byrne, D. P.; Shrestha, S.; Daly, L. A.; Oswald, S. O.; Katiyar, S.; Venkat, A.; Yeung, W.; Eyers, C. E.; Eyers, P. A.; Kannan, N.\nScore: 7.8, Published: 2023-10-06 DOI: 10.1101/2023.10.05.561145\nIn eukaryotes, protein kinase signaling is regulated by a diverse array of post- translational modifications (PTMs). While regulation by activation segment phosphorylation in Ser/Thr kinases is well understood, relatively little is known about how oxidation of cysteine (Cys) amino acids modulate catalysis. In this study, we investigate redox regulation of the AMPK-related Brain-selective kinases (BRSK) 1 and 2, and detail how broad catalytic activity is directly regulated through reversible oxidation and reduction of evolutionarily conserved Cys residues within the catalytic domain. We show that redox-dependent control of BRSKs is a dynamic and multilayered process involving oxidative modifications of several Cys residues, including the formation of intra-molecular disulfide bonds involving a pair of Cys residues near the catalytic HRD motif and a highly conserved T-Loop Cys with a BRSK-specific Cys within an unusual CPE motif at the end of the activation segment. Consistently, mutation of the CPE-Cys increases catalytic activity in vitro and drives phosphorylation of the BRSK substrate Tau in cells. Molecular modeling and molecular dynamics simulations indicate that oxidation of the CPE-Cys destabilizes a conserved salt bridge network critical for allosteric activation. The occurrence of spatially proximal Cys amino acids in diverse Ser/Thr protein kinase families suggests that disulfide mediated control of catalytic activity may be a prevalent mechanism for regulation within the broader AMPK family.\nInhibition of Asparagine Synthetase Effectively Retards Polycystic Kidney Disease Progression\nAuthors: Podrini, C.; Clerici, S.; Tronci, L.; Stefanoni, D.; Canu, T.; Chiaravalli, M.; Spies, D.; Costa, A. S. H.; Esposito, A.; D'Alessandro, A.; Frezza, C.; Bachi, A.; Boletta, A.\nScore: 5.7, Published: 2023-10-10 DOI: 10.1101/2023.10.10.561720\nPolycystic Kidney Disease (PKD) is a genetic disorder characterized by bilateral cyst formation. We showed that PKD cells and kidneys display metabolic alterations, including the Warburg effect and glutaminolysis, sustained in vitro by the enzyme asparagine synthetase (ASNS). Here, we used antisense oligonucleotides (ASO) against Asns in orthologous and slowly progressive PKD murine models and show that treatment leads to a drastic reduction of total kidney volume (measured by MRI) and a prominent rescue of renal function in the mouse. Mechanistically, the upregulation of an ATF4-ASNS axis in PKD is driven by the amino acid response (AAR) branch of the integrated stress response (ISR). Metabolic profiling of PKD or control kidneys treated with Asns-ASO or Scr-ASO revealed major changes in the mutants, several of which are rescued by Asns silencing in vivo. Indeed, ASNS drives glutamine-dependent de novo pyrimidine synthesis and proliferation in cystic epithelia. Notably, while several metabolic pathways were completely corrected by Asns-ASO, glycolysis was only partially restored. Accordingly, combining the glycolytic inhibitor 2DG with Asns-ASO further improved efficacy. Our studies identify a new therapeutic target and novel metabolic vulnerabilities in PKD.\nEx vivo structures from spinach leaves\nAuthors: Wang, J.; Johansen, N. T.; Gamon, L. F.; Zhao, Z.; Guo, Z.; Wang, Y.; Fuglsang, A. T.; Gourdon, P.; Wang, K.\nScore: 5.1, Published: 2023-10-14 DOI: 10.1101/2023.10.13.562012\nEx vivo structure determination of macromolecules from native source is gaining increasing attention from the scientific community, as the method can be employed to dissect the function of important, multi-component molecular machines. However, the existing ex vivo procedures often require genome manipulation or availability high-affinity binders, limiting the general applicability. Here, we report simple yet robust principles for isolation of protein complexes from enriched native biological material, enabling cryoEM-facilitated high-resolution structure determination. We report the structures of ten separate membrane and soluble protein complexes determined from spinach leaves. Moreover, the developed pipeline is likely adaptable to essentially any biological system. As such, the approach may represent an attractive avenue for future structural proteomics efforts.\n",
  "wordCount" : "2373",
  "inLanguage": "en",
  "datePublished": "2023-10-18T10:38:41Z",
  "dateModified": "2023-10-18T10:38:41Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta"><span>updated on October 18, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561603">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561603" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561603">
        <p class="paperTitle">Generalized Biomolecular Modeling and Design with RoseTTAFold All-Atom</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561603" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561603" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krishna, R.; Wang, J.; Ahern, W.; Sturmfels, P.; Venkatesh, P.; Kalvet, I.; Lee, G. R.; Morey-Burrows, F. S.; Anishchenko, I.; Humphreys, I. R.; McHugh, R.; Vafeados, D.; Li, X.; Sutherland, G. A.; Hitchcock, A.; Hunter, C. N.; Baek, M.; DiMaio, F.; Baker, D.</p>
        <p class="info">Score: 288.2, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561603' target='https://doi.org/10.1101/2023.10.09.561603'> 10.1101/2023.10.09.561603</a></p>
        <p class="abstract">Although AlphaFold2 (AF2) and RoseTTAFold (RF) have transformed structural biology by enabling high-accuracy protein structure modeling, they are unable to model covalent modifications or interactions with small molecules and other non-protein molecules that can play key roles in biological function. Here, we describe RoseTTAFold All-Atom (RFAA), a deep network capable of modeling full biological assemblies containing proteins, nucleic acids, small molecules, metals, and covalent modifications given the sequences of the polymers and the atomic bonded geometry of the small molecules and covalent modifications. Following training on structures of full biological assemblies in the Protein Data Bank (PDB), RFAA has comparable protein structure prediction accuracy to AF2, excellent performance in CAMEO for flexible backbone small molecule docking, and reasonable prediction accuracy for protein covalent modifications and assemblies of proteins with multiple nucleic acid chains and small molecules which, to our knowledge, no existing method can model simultaneously. By fine-tuning on diffusive denoising tasks, we develop RFdiffusion All-Atom (RFdiffusionAA), which generates binding pockets by directly building protein structures around small molecules and other non-protein molecules. Starting from random distributions of amino acid residues surrounding target small molecules, we design and experimentally validate proteins that bind the cardiac disease therapeutic digoxigenin, the enzymatic cofactor heme, and optically active bilin molecules with potential for expanding the range of wavelengths captured by photosynthesis. We anticipate that RFAA and RFdiffusionAA will be widely useful for modeling and designing complex biomolecular systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.11.561927">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.11.561927" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.11.561927">
        <p class="paperTitle">Optimising Spectronaut search parameters to improve data quality with minimal proteome coverage reductions in DIA analyses of heterogeneous samples</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.11.561927" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.11.561927" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Baker, C. P.; Bruderer, R.; Abbott, J. C.; Arthur, S.; Brenes, A. J.</p>
        <p class="info">Score: 24.0, Published: 2023-10-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.11.561927' target='https://doi.org/10.1101/2023.10.11.561927'> 10.1101/2023.10.11.561927</a></p>
        <p class="abstract">Data independent acquisition has seen breakthroughs that enable comprehensive proteome profiling using short gradients. As the proteome coverage continues to increase, the quality of the data generated becomes much more relevant. Using Spectronaut, we show that the default search parameters can be easily optimised to minimise the occurrence of false positives across different samples. Using an immunological infection model system to demonstrate the impact of adjusting search settings we analysed mouse macrophages and compared their proteome to macrophages spiked with Candida albicans. This experimental system enabled the identification of  false positives since Candida albicans peptides and proteins should not be present in the mouse only samples. We show that adjusting the search parameters reduced  false positive identifications by 89% at the peptide and protein level, thereby considerably increasing the quality of the data. We also show that these optimised parameters incur a moderate cost, only reducing the overall number of  true positive identifications across each biological replicate by &lt;6.7% at both the peptide and protein level. We believe the value of our updated search parameters extends beyond a two-organism analysis and would be of great value to any DIA experiment analysing heterogenous populations of cell types or tissues.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561769">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561769" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561769">
        <p class="paperTitle">Shared and Distinct Mechanisms of UBA1 Inactivation Across Different Diseases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561769" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561769" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Collins, J. C.; Magaziner, S. J.; English, M.; Hassan, B.; Chen, X.; Balanda, N.; Anderson, M.; Lam, A.; Fernandez-Pol, S.; Kwong, B.; Greenberg, P. L.; Terrier, B.; Likhite, M. E.; Kosmider, O.; Wang, Y.; Samara, N. L.; Walters, K. J.; Beck, D. B.; Werner, A.</p>
        <p class="info">Score: 14.0, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561769' target='https://doi.org/10.1101/2023.10.10.561769'> 10.1101/2023.10.10.561769</a></p>
        <p class="abstract">Most cellular ubiquitin signaling is initiated by UBA1, which activates and transfers ubiquitin to tens of E2 enzymes. Clonally acquired UBA1 missense mutations cause an inflammatory-hematologic overlap disease called VEXAS (vacuoles, E1, X-linked, autoinflammatory, somatic) syndrome. Despite extensive clinical investigation into this lethal disease, little is known about the underlying molecular mechanisms. Here, by dissecting VEXAS-causing UBA1 mutations, we discovered that p.Met41 mutations alter cytoplasmic isoform expression, whereas other mutations reduce catalytic activity of nuclear and cytoplasmic isoforms by diverse mechanisms, including aberrant oxyester formation. Strikingly, non-p.Met41 mutations most prominently affect transthioesterification, revealing ubiquitin transfer to cytoplasmic E2 enzymes as a shared property of pathogenesis amongst different VEXAS syndrome genotypes. A similar E2 charging bottleneck exists in some lung cancer-associated UBA1 mutations, but not in spinal muscular atrophy-causing UBA1 mutations, which instead, render UBA1 thermolabile. Collectively, our results highlight the precision of conformational changes required for faithful ubiquitin transfer, define distinct and shared mechanisms of UBA1 inactivation in diverse diseases, and suggest that specific E1-E2 modules control different aspects of tissue differentiation and maintenance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.561059">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.561059" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.561059">
        <p class="paperTitle">Structural and molecular basis of choline uptake into the brain by FLVCR2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.561059" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.561059" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cater, R. J.; Mukherjee, D.; Gil Iturbe, E.; Erramilli, S. K.; Chen, T.; Koo, K.; Santander, N. G.; Reckers, A. R.; Kloss, B.; Gawda, T.; Choy, B. C.; Zhang, Z.; Clarke, O. B.; Yee, S. W.; Kossiakoff, A. K.; Quick, M.; Arnold, T.; Mancia, F.</p>
        <p class="info">Score: 19.2, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.561059' target='https://doi.org/10.1101/2023.10.05.561059'> 10.1101/2023.10.05.561059</a></p>
        <p class="abstract">Choline is an essential nutrient that the human body needs in vast quantities for cell membrane synthesis, epigenetic modification, and neurotransmission. The brain has a particularly high demand for choline, but how it enters the brain has eluded the field for over fifty years. The MFS transporter FLVCR1 was recently determined to be a choline transporter, and while this protein is not highly expressed at the blood-brain barrier (BBB), its relative FLVCR2 is. Previous studies have shown that mutations in human Flvcr2 cause cerebral vascular abnormalities, hydrocephalus, and embryonic lethality, but the physiological role of FLVCR2 is unknown. Here, we demonstrate both in vivo and in vitro that FLVCR2 is a BBB choline transporter and is responsible for the majority of choline uptake into the brain. We also determine the structures of choline-bound FLVCR2 in the inward- and outward-facing states using cryo-electron microscopy to 2.49 and 2.77 [A] resolution, respectively. These results reveal how the brain obtains choline and provide molecular-level insights into how FLVCR2 binds choline in an aromatic cage and mediates its uptake. Our work could provide a novel framework for the targeted delivery of neurotherapeutics into the brain.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562242">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562242" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562242">
        <p class="paperTitle">Structure of the endosomal CORVET tethering complex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562242" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562242" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shvarev, D.; Koenig, C.; Susan, N.; Langemeyer, L.; Walter, S.; Perz, A.; Froehlich, F.; Ungermann, C.; Moeller, A.</p>
        <p class="info">Score: 9.2, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562242' target='https://doi.org/10.1101/2023.10.13.562242'> 10.1101/2023.10.13.562242</a></p>
        <p class="abstract">Cells depend on their endolysosomal system for nutrient uptake and downregulation of plasma membrane proteins. These processes rely on endosomal maturation, which requires multiple membrane fusion steps. Early endosome fusion is promoted by the Rab5 GTPase and its effector, the hexameric CORVET tethering complex, which is homologous to the lysosomal HOPS. How these related complexes recognize their specific target membranes remains entirely elusive. Here, we solved the structure of CORVET by cryo-electron microscopy and revealed its minimal requirements for membrane tethering. As expected, the core of CORVET and HOPS resembles each other. However, the function-defining subunits show marked structural differences. Notably, we discovered that unlike HOPS, CORVET depends not only on Rab5 but also on phosphatidylinositol-3-phosphate (PI3P) and membrane packaging defects for tethering, implying that an organelle-specific membrane code enables fusion. Our data suggest that both shape and membrane interactions of CORVET and HOPS are conserved in metazoans, thus providing a paradigm how tethering complexes function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.15.562437">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.15.562437" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.15.562437">
        <p class="paperTitle">The guide RNA sequence dictates the slicing kinetics and conformational dynamics of the Argonaute silencing complex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.15.562437" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.15.562437" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, P. Y.; Bartel, D. P.</p>
        <p class="info">Score: 8.8, Published: 2023-10-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.15.562437' target='https://doi.org/10.1101/2023.10.15.562437'> 10.1101/2023.10.15.562437</a></p>
        <p class="abstract">The RNA-induced silencing complex (RISC), which powers RNA interference (RNAi), consists of a guide RNA and an Argonaute protein that slices target RNAs complementary to the guide. We find that for different guide-RNA sequences, slicing rates of perfectly paired, bound targets can be surprisingly different (&gt;250-fold range), and that faster slicing confers greater knockdown efficacy in cells. Four sequence determinants underlie much of this variation in slicing rates. Analyses of these determinants implicate a structural distortion at guide nucleotides 6-7 in promoting slicing, and indicate that, in contrast to the prevailing model, slicing is primarily limited by the rate at which the enzyme-substrate complex reaches a slicing-competent conformation. Moreover, different guide sequences have an unanticipated, 600-fold range in 3&#39;-mismatch tolerance, which is attributable to the GC-content of their central regions. Together, our analyses identify sequence determinants of RISC activity and provide biochemical, conformational, and functional rationale for their action.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561661">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561661" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561661">
        <p class="paperTitle">TMEM65-dependent Ca2&#43; extrusion safeguards mitochondrial homeostasis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561661" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561661" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vetralla, M.; Wischhof, L.; Cadenelli, V.; Scifo, E.; Ehninger, D.; Rizzuto, R.; Bano, D.; De Stefani, D.</p>
        <p class="info">Score: 8.5, Published: 2023-10-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561661' target='https://doi.org/10.1101/2023.10.10.561661'> 10.1101/2023.10.10.561661</a></p>
        <p class="abstract">The bidirectional transport of Ca2&#43; into and out of mitochondria is a conserved biological process controlling multiple events, including metabolism, signaling, and cell fate. In the presence of membrane potential driving mitochondrial Ca2&#43; accumulation, transient changes of [Ca2&#43;]mt in response to cytosolic [Ca2&#43;] variations are ensured by a molecular machinery for Ca2&#43; influx and efflux embedded in the inner mitochondrial membrane (IMM). While it is well established that influx relies on the Mitochondrial Calcium Uniporter (MCU), efflux was expected to be molecularly diversified, given the occurrence of functionally different exchange pathways with either Na&#43; or H&#43;1. Accordingly, dedicated transporters ensure proper Ca2&#43; homeostasis and tightly regulated mitochondrial bioenergetics, but the process is not yet fully elucidated. We here demonstrate that TMEM65, a protein with an unknown biological function, is a fundamental component of the Ca2&#43; efflux machinery of mitochondria. As the MCU, TMEM65 has a broad tissue expression and localizes to the IMM. Its overexpression dramatically enhances Na&#43;- and Li&#43;-dependent mitochondrial Ca2&#43; extrusion, which is abrogated by the pharmacological inhibitor CGP-37157. Conversely, its downregulation chronically elevates resting mitochondrial Ca2&#43; levels and inhibits efficient Ca2&#43; efflux upon cellular activation, culminating in mitochondrial calcium overload and causing organelle dysfunction. Since TMEM65 has been associated with a severe human mitochondrial disease2, we deleted the TMEM65 homologues in Caenorhabditis elegans (CeTMEM65). While the two CeTMEM65 orthologs are dispensable for the survival at permissive growing conditions, their loss undermines embryonic developments when eggs are exposed to mild temperature-stress. In this regard, we find that CeTMEM65 (null) alleles cause necrotic lesions that are suppressed by inhibiting the mitochondrial calcium uniporter MCU-1. Overall, these results unambiguously assign a primary role in mitochondrial Ca2&#43; homeostasis to the orphan protein TMEM65. More importantly, our findings describe a novel molecular component that may be relevant in pathological settings in which excessive mitochondrial Ca2&#43; accumulation critically contribute to degenerative pathways.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.561145">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.561145" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.561145">
        <p class="paperTitle">Redox Regulation of Brain Selective Kinases BRSK1/2: Implications for Dynamic Control of the Eukaryotic AMPK family through Cys-based mechanisms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.561145" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.561145" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bendzunas, G.; Byrne, D. P.; Shrestha, S.; Daly, L. A.; Oswald, S. O.; Katiyar, S.; Venkat, A.; Yeung, W.; Eyers, C. E.; Eyers, P. A.; Kannan, N.</p>
        <p class="info">Score: 7.8, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.561145' target='https://doi.org/10.1101/2023.10.05.561145'> 10.1101/2023.10.05.561145</a></p>
        <p class="abstract">In eukaryotes, protein kinase signaling is regulated by a diverse array of post- translational modifications (PTMs). While regulation by activation segment phosphorylation in Ser/Thr kinases is well understood, relatively little is known about how oxidation of cysteine (Cys) amino acids modulate catalysis. In this study, we investigate redox regulation of the AMPK-related Brain-selective kinases (BRSK) 1 and 2, and detail how broad catalytic activity is directly regulated through reversible oxidation and reduction of evolutionarily conserved Cys residues within the catalytic domain. We show that redox-dependent control of BRSKs is a dynamic and multilayered process involving oxidative modifications of several Cys residues, including the formation of intra-molecular disulfide bonds involving a pair of Cys residues near the catalytic HRD motif and a highly conserved T-Loop Cys with a BRSK-specific Cys within an unusual CPE motif at the end of the activation segment. Consistently, mutation of the CPE-Cys increases catalytic activity in vitro and drives phosphorylation of the BRSK substrate Tau in cells. Molecular modeling and molecular dynamics simulations indicate that oxidation of the CPE-Cys destabilizes a conserved salt bridge network critical for allosteric activation. The occurrence of spatially proximal Cys amino acids in diverse Ser/Thr protein kinase families suggests that disulfide mediated control of catalytic activity may be a prevalent mechanism for regulation within the broader AMPK family.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561720">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561720" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561720">
        <p class="paperTitle">Inhibition of Asparagine Synthetase Effectively Retards Polycystic Kidney Disease Progression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561720" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561720" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Podrini, C.; Clerici, S.; Tronci, L.; Stefanoni, D.; Canu, T.; Chiaravalli, M.; Spies, D.; Costa, A. S. H.; Esposito, A.; D&#39;Alessandro, A.; Frezza, C.; Bachi, A.; Boletta, A.</p>
        <p class="info">Score: 5.7, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561720' target='https://doi.org/10.1101/2023.10.10.561720'> 10.1101/2023.10.10.561720</a></p>
        <p class="abstract">Polycystic Kidney Disease (PKD) is a genetic disorder characterized by bilateral cyst formation. We showed that PKD cells and kidneys display metabolic alterations, including the Warburg effect and glutaminolysis, sustained in vitro by the enzyme asparagine synthetase (ASNS). Here, we used antisense oligonucleotides (ASO) against Asns in orthologous and slowly progressive PKD murine models and show that treatment leads to a drastic reduction of total kidney volume (measured by MRI) and a prominent rescue of renal function in the mouse. Mechanistically, the upregulation of an ATF4-ASNS axis in PKD is driven by the amino acid response (AAR) branch of the integrated stress response (ISR). Metabolic profiling of PKD or control kidneys treated with Asns-ASO or Scr-ASO revealed major changes in the mutants, several of which are rescued by Asns silencing in vivo. Indeed, ASNS drives glutamine-dependent de novo pyrimidine synthesis and proliferation in cystic epithelia. Notably, while several metabolic pathways were completely corrected by Asns-ASO, glycolysis was only partially restored. Accordingly, combining the glycolytic inhibitor 2DG with Asns-ASO further improved efficacy. Our studies identify a new therapeutic target and novel metabolic vulnerabilities in PKD.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562012">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562012" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562012">
        <p class="paperTitle">Ex vivo structures from spinach leaves</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562012" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562012" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, J.; Johansen, N. T.; Gamon, L. F.; Zhao, Z.; Guo, Z.; Wang, Y.; Fuglsang, A. T.; Gourdon, P.; Wang, K.</p>
        <p class="info">Score: 5.1, Published: 2023-10-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562012' target='https://doi.org/10.1101/2023.10.13.562012'> 10.1101/2023.10.13.562012</a></p>
        <p class="abstract">Ex vivo structure determination of macromolecules from native source is gaining increasing attention from the scientific community, as the method can be employed to dissect the function of important, multi-component molecular machines. However, the existing ex vivo procedures often require genome manipulation or availability high-affinity binders, limiting the general applicability. Here, we report simple yet robust principles for isolation of protein complexes from enriched native biological material, enabling cryoEM-facilitated high-resolution structure determination. We report the structures of ten separate membrane and soluble protein complexes determined from spinach leaves. Moreover, the developed pipeline is likely adaptable to essentially any biological system. As such, the approach may represent an attractive avenue for future structural proteomics efforts.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
