<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Structure of the yeast ceramide synthase
Authors: Schäfer, J.-H.; Clausmeyer, L.; Körner, C.; Esch, B. M.; Wolf, V. N.; Walter, S.; Januliene, D.; Moeller, A.; Fröhlich, F.
Score: 17.4, Published: 2023-10-24 DOI: 10.1101/2023.10.24.563787
Ceramides play a pivotal role as essential lipids, serving as foundational components for complex sphingolipids and potent signaling molecules. Ceramides are the products of the N-acylation of a sphingoid base and a CoA-activated fatty acid. This reaction is catalyzed by the enzymes of the evolutionarily conserved ceramide synthase (CerS) family.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Structure of the yeast ceramide synthase
Authors: Schäfer, J.-H.; Clausmeyer, L.; Körner, C.; Esch, B. M.; Wolf, V. N.; Walter, S.; Januliene, D.; Moeller, A.; Fröhlich, F.
Score: 17.4, Published: 2023-10-24 DOI: 10.1101/2023.10.24.563787
Ceramides play a pivotal role as essential lipids, serving as foundational components for complex sphingolipids and potent signaling molecules. Ceramides are the products of the N-acylation of a sphingoid base and a CoA-activated fatty acid. This reaction is catalyzed by the enzymes of the evolutionarily conserved ceramide synthase (CerS) family." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-01T10:38:40+00:00" />
<meta property="article:modified_time" content="2023-11-01T10:38:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Structure of the yeast ceramide synthase
Authors: Schäfer, J.-H.; Clausmeyer, L.; Körner, C.; Esch, B. M.; Wolf, V. N.; Walter, S.; Januliene, D.; Moeller, A.; Fröhlich, F.
Score: 17.4, Published: 2023-10-24 DOI: 10.1101/2023.10.24.563787
Ceramides play a pivotal role as essential lipids, serving as foundational components for complex sphingolipids and potent signaling molecules. Ceramides are the products of the N-acylation of a sphingoid base and a CoA-activated fatty acid. This reaction is catalyzed by the enzymes of the evolutionarily conserved ceramide synthase (CerS) family."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Structure of the yeast ceramide synthase\nAuthors: Schäfer, J.-H.; Clausmeyer, L.; Körner, C.; Esch, B. M.; Wolf, V. N.; Walter, S.; Januliene, D.; Moeller, A.; Fröhlich, F.\nScore: 17.4, Published: 2023-10-24 DOI: 10.1101/2023.10.24.563787\nCeramides play a pivotal role as essential lipids, serving as foundational components for complex sphingolipids and potent signaling molecules. Ceramides are the products of the N-acylation of a sphingoid base and a CoA-activated fatty acid. This reaction is catalyzed by the enzymes of the evolutionarily conserved ceramide synthase (CerS) family.",
  "keywords": [
    
  ],
  "articleBody": " Structure of the yeast ceramide synthase\nAuthors: Schäfer, J.-H.; Clausmeyer, L.; Körner, C.; Esch, B. M.; Wolf, V. N.; Walter, S.; Januliene, D.; Moeller, A.; Fröhlich, F.\nScore: 17.4, Published: 2023-10-24 DOI: 10.1101/2023.10.24.563787\nCeramides play a pivotal role as essential lipids, serving as foundational components for complex sphingolipids and potent signaling molecules. Ceramides are the products of the N-acylation of a sphingoid base and a CoA-activated fatty acid. This reaction is catalyzed by the enzymes of the evolutionarily conserved ceramide synthase (CerS) family. Yet, the precise structural details and catalytic mechanisms of CerSs have remained elusive. Here, we employed cryo-EM single particle analysis to unravel the structure of the yeast ceramide synthase complex in both an active and a fumonisin B1 inhibited state. Our findings shed light on the complexs architecture, revealing a dimer of Lip1 subunits bound to the two catalytic subunits, Lag1 and Lac1. Each catalytic subunit forms a hydrophobic crevice that is accessible from both the cytosolic site as well as from the intermembrane space of the endoplasmic reticulum (ER). Within this cavity, we identify amino acids forming the active center and a sphingoid base, one of the substrates of the complex. Together, this suggests a pre-loaded state of the CerS. Additionally, the fumonisin B1 bound structure reveals the inhibitory mechanism by blocking the cytosolic acyl-CoA binding site.\nGeneralized Biomolecular Modeling and Design with RoseTTAFold All-Atom\nAuthors: Krishna, R.; Wang, J.; Ahern, W.; Sturmfels, P.; Venkatesh, P.; Kalvet, I.; Lee, G. R.; Morey-Burrows, F. S.; Anishchenko, I.; Humphreys, I. R.; McHugh, R.; Vafeados, D.; Li, X.; Sutherland, G. A.; Hitchcock, A.; Hunter, C. N.; Baek, M.; DiMaio, F.; Baker, D.\nScore: 343.5, Published: 2023-10-09 DOI: 10.1101/2023.10.09.561603\nAlthough AlphaFold2 (AF2) and RoseTTAFold (RF) have transformed structural biology by enabling high-accuracy protein structure modeling, they are unable to model covalent modifications or interactions with small molecules and other non-protein molecules that can play key roles in biological function. Here, we describe RoseTTAFold All-Atom (RFAA), a deep network capable of modeling full biological assemblies containing proteins, nucleic acids, small molecules, metals, and covalent modifications given the sequences of the polymers and the atomic bonded geometry of the small molecules and covalent modifications. Following training on structures of full biological assemblies in the Protein Data Bank (PDB), RFAA has comparable protein structure prediction accuracy to AF2, excellent performance in CAMEO for flexible backbone small molecule docking, and reasonable prediction accuracy for protein covalent modifications and assemblies of proteins with multiple nucleic acid chains and small molecules which, to our knowledge, no existing method can model simultaneously. By fine-tuning on diffusive denoising tasks, we develop RFdiffusion All-Atom (RFdiffusionAA), which generates binding pockets by directly building protein structures around small molecules and other non-protein molecules. Starting from random distributions of amino acid residues surrounding target small molecules, we design and experimentally validate proteins that bind the cardiac disease therapeutic digoxigenin, the enzymatic cofactor heme, and optically active bilin molecules with potential for expanding the range of wavelengths captured by photosynthesis. We anticipate that RFAA and RFdiffusionAA will be widely useful for modeling and designing complex biomolecular systems.\nReconstitution of cytolinker-mediated crosstalk between actin and vimentin\nAuthors: Isturiz Petitjean, I.; Tran, Q. D.; Goutou, A.; Kabir, Z.; Wiche, G.; Leduc, C.; Koenderink, G. H.\nScore: 9.3, Published: 2023-10-28 DOI: 10.1101/2023.10.27.564417\nCell shape and motility are determined by the cytoskeleton, an interpenetrating network of actin filaments, microtubules, and intermediate filaments. The biophysical properties of each filament type individually have been studied extensively by cell-free reconstitution. By contrast, the interactions between the three cytoskeletal networks are relatively unexplored. They are coupled via crosslinkers of the plakin family such as plectin. These are challenging proteins for reconstitution because of their giant size and multidomain structure. Here we engineer a recombinant actin-vimentin crosslinker protein called 'ACTIF' that provides a minimal model system for plectin, recapitulating its modular design with actin-binding and intermediate filament-binding domains separated by a coiled-coil linker for dimerisation. We show by fluorescence and electron microscopy that ACTIF has a high binding affinity for vimentin and actin and creates mixed actin-vimentin bundles. Rheology measurements show that ACTIF-mediated crosslinking strongly stiffens actin-vimentin composites. Finally, we demonstrate the modularity of this approach by creating an ACTIF variant with the intermediate filament binding domain of Adenomatous Polyposis Coli. Our protein engineering approach provides a new cell-free system for the biophysical characterization of intermediate filament-binding crosslinkers and for understanding the mechanical synergy between actin and vimentin in mesenchymal cells.\nEvolution and Diversification of Carboxylesterase-like Cyclases in Aspidosperma and Iboga Alkaloid Biosynthesis\nAuthors: DeMars, M. D.; O'Connor, S. E.\nScore: 9.1, Published: 2023-10-24 DOI: 10.1101/2023.10.24.563752\nMonoterpene indole alkaloids (MIAs) are a large and diverse class of plant natural products, and their biosynthetic construction has been a subject of intensive study for many years. The enzymatic basis for the production of aspidosperma and iboga alkaloids, which are produced exclusively by members of the Apocynaceae plant family, has recently been discovered. Three carboxylesterase (CXE)-like enzymes from Catharanthus roseus and Tabernanthe iboga catalyze regio- and enantiodivergent [4+2] cycloaddition reactions to generate the aspidosperma (tabersonine synthase, TS) and iboga (coronaridine synthase, CorS; catharanthine synthase, CS) scaffolds from a common biosynthetic intermediate. Here, we use a combined phylogenetic and biochemical approach to investigate the evolution and functional diversification of these cyclase enzymes. Through ancestral sequence reconstruction, we provide evidence for initial evolution of TS from an ancestral CXE followed by emergence of CorS in two separate lineages, leading in turn to CS exclusively in the Catharanthus genus. This progression from aspidosperma to iboga alkaloid biosynthesis is consistent with the chemotaxonomic distribution of these MIAs. We subsequently generate and test a panel of chimeras based on the ancestral cyclases to probe the molecular basis for differential cyclization activity. Finally, we show through partial heterologous reconstitution of tabersonine biosynthesis using non-pathway enzymes how aspidosperma alkaloids could have first appeared as \"underground metabolites\" via recruitment of promiscuous enzymes from common protein families. Our results provide insight into the evolution of biosynthetic enzymes and how new secondary metabolic pathways can emerge through small but important sequence changes following co-option of preexisting enzymatic functions.\nCyclic peptide inhibitors stabilize Gq/11 heterotrimers\nAuthors: Muehle, J.; Rodrigues, M. J.; Alenfelder, J.; Juergenliemke, L.; Guixa-Gonzalez, R.; Bacchin, A.; Andres, F.; Hennig, M.; Schihada, H.; Cruesemann, M.; Koenig, G.; Kostenis, E.; Schertler, G.; Deupi, X.\nScore: 8.8, Published: 2023-10-24 DOI: 10.1101/2023.10.24.563737\nHeterotrimeric G proteins play a central role in cellular signaling, acting as switchable molecular regulators. Consequently, pharmacological agents to control G protein activity are of utmost importance to advance our understanding of this signal transduction system. The natural depsipeptides FR900359 (FR) and YM-254890 (YM) are two highly specific and widely used inhibitors of heterotrimeric Gq/11 proteins. These compounds have traditionally been understood to inhibit GDP dissociation by preventing the separation of the GTPase and -helical domains of the G subunit. In this work, we have determined the high-resolution crystal structures of FR and YM bound to heterotrimeric G11 and used them to explain the molecular basis underlying their efficient suppression of G protein signaling. Notably, our data show that FR and YM also function as stabilizers of the interface between the G and G{beta} subunits, acting as molecular adhesives that stabilize the entire heterotrimer. Our results reveal unrecognized mechanistic features that explain how FR and YM effectively blunt Gq/11 signaling in living cells.\nMulti-scale photocatalytic proximity labeling reveals cell surface neighbors on and between cells\nAuthors: Lin, Z.; Schaefer, K.; Lui, I.; Yao, Z.; Fossati, A. A.; Swaney, D. L.; Palar, A.; Sali, A.; Wells, J. A.\nScore: 8.1, Published: 2023-10-29 DOI: 10.1101/2023.10.28.564055\nThe cell membrane proteome is the primary biohub for cell communication, yet we are only beginning to understand the dynamic protein neighborhoods that form on the cell surface and between cells. Proximity labeling proteomics (PLP) strategies using chemically reactive probes are powerful approaches to yield snapshots of protein neighborhoods but are currently limited to one single resolution based on the probe labeling radius. Here, we describe a multi-scale PLP method with tunable resolution using a commercially available histological dye, Eosin Y, which upon visible light illumination, activates three different photo-probes with labeling radii ranging from ~100 to 3000 [a]. We applied this platform to profile neighborhoods of the oncogenic epidermal growth factor receptor (EGFR) and orthogonally validated \u003e20 neighbors using immuno-assays and AlphaFold-Multimer prediction that generated plausible binary interaction models. We further profiled the protein neighborhoods of cell-cell synapses induced by bi-specific T-cell engagers (BiTEs) and chimeric antigen receptor (CAR)T cells at longer length scales. This integrated multi-scale PLP platform maps local and distal protein networks on cell surfaces and between cells. We believe this information will aid in the systematic construction of the cell surface interactome and reveal new opportunities for immunotherapeutics.\nDetermining the defining lengths between mature microRNAs/small interfering RNAs and tinyRNAs\nAuthors: Sim, G.; Kehling, A. C.; Park, M. S.; Divoky, C.; Zhang, H.; Malhotra, N.; Secor, J.; Nakanishi, K.\nScore: 7.8, Published: 2023-10-28 DOI: 10.1101/2023.10.27.564437\nMicroRNAs (miRNAs) and small interfering RNAs (siRNAs) are loaded into Argonaute (AGO) proteins, forming RNA-induced silencing complexes (RISCs). The assembly process establishes the seed, central, 3' supplementary, and tail regions across the loaded guide, enabling the RISC to recognize and cleave target RNAs. This guide segmentation is caused by anchoring the 3' end at the AGO PAZ domain, but the minimum guide length required for the conformation remains to be studied because there was no method by which to do so. Using a 3'[-\u0026gt;]5' exonuclease ISG20, we determined the lengths of AGO-associated miR-20a and let-7a with 3' ends that no longer reach the PAZ domain. Unexpectedly, miR-20a and let-7a needed different lengths, 19 and 20 nt, respectively, to maintain their RISC conformation. This difference can be explained by the low affinity of the PAZ domain for the adenosine at g19 of let-7a, suggesting that the tail-region sequence slightly alters the minimum guide length. We also present that 17-nt guides are sufficiently short enough to function as tinyRNAs (tyRNAs) whose 3' ends are not anchored at the PAZ domain. Since tyRNAs do not have the prerequisite anchoring for the standardized guide segmentation, they would recognize targets differently from miRNAs and siRNAs.\nChemoproteomics identifies proteoform-selective caspase-2 inhibitors\nAuthors: Castellon, J. O.; Ofori, S.; Armenta, E.; Burton, N.; Boatner, L. M.; Takayoshi, E. E.; Faragalla, M.; Zhou, A.; Tran, K.; Shek, J.; Yan, T.; Desai, H. S.; Backus, K. M.\nScore: 7.5, Published: 2023-10-26 DOI: 10.1101/2023.10.25.563785\nCaspases are a highly conserved family of cysteine-aspartyl proteases known for their essential roles in regulating apoptosis, inflammation, cell differentiation, and proliferation. Complementary to genetic approaches, small-molecule probes have emerged as useful tools for modulating caspase activity. However, due to the high sequence and structure homology of all twelve human caspases, achieving selectivity remains a central challenge for caspase-directed small-molecule inhibitor development efforts. Here, using mass spectrometry-based chemoproteomics, we first identify a highly reactive non-catalytic cysteine that is unique to caspase-2. By combining both gel-based activity-based protein profiling (ABPP) and a tobacco etch virus (TEV) protease activation assay, we then identify covalent lead compounds that react preferentially with this cysteine and afford a complete blockade of caspase-2 activity. Inhibitory activity is restricted to the zymogen or precursor form of monomeric caspase-2. Focused analogue synthesis combined with chemoproteomic target engagement analysis in cellular lysates and in cells yielded both pan-caspase reactive molecules and caspase-2 selective lead compounds together with a structurally matched inactive control. Application of this focused set of tool compounds to stratify caspase contributions to initiation of intrinsic apoptosis, supports compensatory caspase-9 activity in the context of caspase-2 inactivation. More broadly, our study highlights future opportunities for the development of proteoform-selective caspase inhibitors that target non-conserved and non-catalytic cysteine residues.\nA Toxoplasma gondii O-glycosyltransferase that modulates bradyzoite cyst wall rigidity is structurally and functionally distinct from host homologues\nAuthors: Kumar, P.; Tomita, T.; Gerken, T. A.; Ballard, C. J.; Lee, Y. S.; Weiss, L. M.; Samara, N. L.\nScore: 6.2, Published: 2023-10-26 DOI: 10.1101/2023.10.26.564164\nInfection with the Apicomplexan protozoan Toxoplasma gondii can be life-threatening in immunocompromised hosts. Transmission frequently occurs through the oral ingestion of T. gondii bradyzoite cysts, which transition to tachyzoites, disseminate, and then form cysts containing bradyzoites in the central nervous system, resulting in latent infection. There are currently no effective treatments to cure latent infection. Bradyzoites are encapsulated by a cyst wall that is critical for immune evasion, survival, and transmission. Cyst wall rigidity is influenced by the O-glycosylation of the mucin domain of the cyst wall protein CST1 by mucin-type O-glycosyltransferases (GalNAc-Ts). Here, we report the first structures of a protozoan GalNAc-T, T.gondii-GalNAc-T3 in the apo state and in complex with glycopeptide substrates. The structures reveal features that are strictly conserved in Apicomplexan homologues of T.gondii-GalNAc-T3, including a unique 2nd metal binding site that is coupled to substrate binding and enzymatic activity in vitro and cyst wall O-glycosylation in T. gondii. Additional structural features illustrate the divergence of GalNAc-Ts from parasite to host and highlight multiple druggable sites in T.gondii-GalNAc-T3 and its homologues in Apicomplexa that are responsible for a wide range of parasitic diseases.\nCo-transcriptional capping using an RNA capping enzyme-T7 RNA polymerase fusion protein\nAuthors: Chan, S. H.; Tirole, L.; Kneller, D. W.; Kelley, T. M.; Dai, N.; Robb, G. B.\nScore: 5.3, Published: 2023-10-28 DOI: 10.1101/2023.10.28.564488\nmRNA vaccines and therapeutics are highly effective and can be developed and manufactured with a relatively short lead time. Here we report an all-enzyme platform to generate capped synthetic RNA in a one-step process based on an RNA capping enzyme-T7 RNA polymerase fusion protein. Under standard in vitro transcription reaction conditions, the fusion protein, in conjunction with an RNA cap 2'-O-methyltransferase, can generate synthetic mRNA with up to 95% of Cap-1 incorporation, greatly simplifying mRNA manufacturing workflows.\n",
  "wordCount" : "2282",
  "inLanguage": "en",
  "datePublished": "2023-11-01T10:38:40Z",
  "dateModified": "2023-11-01T10:38:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 1, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563787">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563787" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563787">
        <p class="paperTitle">Structure of the yeast ceramide synthase</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563787" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563787" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schäfer, J.-H.; Clausmeyer, L.; Körner, C.; Esch, B. M.; Wolf, V. N.; Walter, S.; Januliene, D.; Moeller, A.; Fröhlich, F.</p>
        <p class="info">Score: 17.4, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563787' target='https://doi.org/10.1101/2023.10.24.563787'> 10.1101/2023.10.24.563787</a></p>
        <p class="abstract">Ceramides play a pivotal role as essential lipids, serving as foundational components for complex sphingolipids and potent signaling molecules. Ceramides are the products of the N-acylation of a sphingoid base and a CoA-activated fatty acid. This reaction is catalyzed by the enzymes of the evolutionarily conserved ceramide synthase (CerS) family. Yet, the precise structural details and catalytic mechanisms of CerSs have remained elusive. Here, we employed cryo-EM single particle analysis to unravel the structure of the yeast ceramide synthase complex in both an active and a fumonisin B1 inhibited state. Our findings shed light on the complexs architecture, revealing a dimer of Lip1 subunits bound to the two catalytic subunits, Lag1 and Lac1. Each catalytic subunit forms a hydrophobic crevice that is accessible from both the cytosolic site as well as from the intermembrane space of the endoplasmic reticulum (ER). Within this cavity, we identify amino acids forming the active center and a sphingoid base, one of the substrates of the complex. Together, this suggests a pre-loaded state of the CerS. Additionally, the fumonisin B1 bound structure reveals the inhibitory mechanism by blocking the cytosolic acyl-CoA binding site.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561603">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561603" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561603">
        <p class="paperTitle">Generalized Biomolecular Modeling and Design with RoseTTAFold All-Atom</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561603" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561603" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krishna, R.; Wang, J.; Ahern, W.; Sturmfels, P.; Venkatesh, P.; Kalvet, I.; Lee, G. R.; Morey-Burrows, F. S.; Anishchenko, I.; Humphreys, I. R.; McHugh, R.; Vafeados, D.; Li, X.; Sutherland, G. A.; Hitchcock, A.; Hunter, C. N.; Baek, M.; DiMaio, F.; Baker, D.</p>
        <p class="info">Score: 343.5, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561603' target='https://doi.org/10.1101/2023.10.09.561603'> 10.1101/2023.10.09.561603</a></p>
        <p class="abstract">Although AlphaFold2 (AF2) and RoseTTAFold (RF) have transformed structural biology by enabling high-accuracy protein structure modeling, they are unable to model covalent modifications or interactions with small molecules and other non-protein molecules that can play key roles in biological function. Here, we describe RoseTTAFold All-Atom (RFAA), a deep network capable of modeling full biological assemblies containing proteins, nucleic acids, small molecules, metals, and covalent modifications given the sequences of the polymers and the atomic bonded geometry of the small molecules and covalent modifications. Following training on structures of full biological assemblies in the Protein Data Bank (PDB), RFAA has comparable protein structure prediction accuracy to AF2, excellent performance in CAMEO for flexible backbone small molecule docking, and reasonable prediction accuracy for protein covalent modifications and assemblies of proteins with multiple nucleic acid chains and small molecules which, to our knowledge, no existing method can model simultaneously. By fine-tuning on diffusive denoising tasks, we develop RFdiffusion All-Atom (RFdiffusionAA), which generates binding pockets by directly building protein structures around small molecules and other non-protein molecules. Starting from random distributions of amino acid residues surrounding target small molecules, we design and experimentally validate proteins that bind the cardiac disease therapeutic digoxigenin, the enzymatic cofactor heme, and optically active bilin molecules with potential for expanding the range of wavelengths captured by photosynthesis. We anticipate that RFAA and RFdiffusionAA will be widely useful for modeling and designing complex biomolecular systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564417">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564417" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564417">
        <p class="paperTitle">Reconstitution of cytolinker-mediated crosstalk between actin and vimentin</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564417" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564417" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Isturiz Petitjean, I.; Tran, Q. D.; Goutou, A.; Kabir, Z.; Wiche, G.; Leduc, C.; Koenderink, G. H.</p>
        <p class="info">Score: 9.3, Published: 2023-10-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564417' target='https://doi.org/10.1101/2023.10.27.564417'> 10.1101/2023.10.27.564417</a></p>
        <p class="abstract">Cell shape and motility are determined by the cytoskeleton, an interpenetrating network of actin filaments, microtubules, and intermediate filaments. The biophysical properties of each filament type individually have been studied extensively by cell-free reconstitution. By contrast, the interactions between the three cytoskeletal networks are relatively unexplored. They are coupled via crosslinkers of the plakin family such as plectin. These are challenging proteins for reconstitution because of their giant size and multidomain structure. Here we engineer a recombinant actin-vimentin crosslinker protein called &#39;ACTIF&#39; that provides a minimal model system for plectin, recapitulating its modular design with actin-binding and intermediate filament-binding domains separated by a coiled-coil linker for dimerisation. We show by fluorescence and electron microscopy that ACTIF has a high binding affinity for vimentin and actin and creates mixed actin-vimentin bundles. Rheology measurements show that ACTIF-mediated crosslinking strongly stiffens actin-vimentin composites. Finally, we demonstrate the modularity of this approach by creating an ACTIF variant with the intermediate filament binding domain of Adenomatous Polyposis Coli. Our protein engineering approach provides a new cell-free system for the biophysical characterization of intermediate filament-binding crosslinkers and for understanding the mechanical synergy between actin and vimentin in mesenchymal cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563752">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563752" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563752">
        <p class="paperTitle">Evolution and Diversification of Carboxylesterase-like  Cyclases in Aspidosperma and Iboga Alkaloid Biosynthesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563752" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563752" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: DeMars, M. D.; O&#39;Connor, S. E.</p>
        <p class="info">Score: 9.1, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563752' target='https://doi.org/10.1101/2023.10.24.563752'> 10.1101/2023.10.24.563752</a></p>
        <p class="abstract">Monoterpene indole alkaloids (MIAs) are a large and diverse class of plant natural products, and their biosynthetic construction has been a subject of intensive study for many years. The enzymatic basis for the production of aspidosperma and iboga alkaloids, which are produced exclusively by members of the Apocynaceae plant family, has recently been discovered. Three carboxylesterase (CXE)-like enzymes from Catharanthus roseus and Tabernanthe iboga catalyze regio- and enantiodivergent [4&#43;2] cycloaddition reactions to generate the aspidosperma (tabersonine synthase, TS) and iboga (coronaridine synthase, CorS; catharanthine synthase, CS) scaffolds from a common biosynthetic intermediate. Here, we use a combined phylogenetic and biochemical approach to investigate the evolution and functional diversification of these cyclase enzymes. Through ancestral sequence reconstruction, we provide evidence for initial evolution of TS from an ancestral CXE followed by emergence of CorS in two separate lineages, leading in turn to CS exclusively in the Catharanthus genus. This progression from aspidosperma to iboga alkaloid biosynthesis is consistent with the chemotaxonomic distribution of these MIAs. We subsequently generate and test a panel of chimeras based on the ancestral cyclases to probe the molecular basis for differential cyclization activity. Finally, we show through partial heterologous reconstitution of tabersonine biosynthesis using non-pathway enzymes how aspidosperma alkaloids could have first appeared as &#34;underground metabolites&#34; via recruitment of promiscuous enzymes from common protein families. Our results provide insight into the evolution of biosynthetic enzymes and how new secondary metabolic pathways can emerge through small but important sequence changes following co-option of preexisting enzymatic functions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563737">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563737" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563737">
        <p class="paperTitle">Cyclic peptide inhibitors stabilize Gq/11 heterotrimers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563737" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563737" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Muehle, J.; Rodrigues, M. J.; Alenfelder, J.; Juergenliemke, L.; Guixa-Gonzalez, R.; Bacchin, A.; Andres, F.; Hennig, M.; Schihada, H.; Cruesemann, M.; Koenig, G.; Kostenis, E.; Schertler, G.; Deupi, X.</p>
        <p class="info">Score: 8.8, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563737' target='https://doi.org/10.1101/2023.10.24.563737'> 10.1101/2023.10.24.563737</a></p>
        <p class="abstract">Heterotrimeric G proteins play a central role in cellular signaling, acting as switchable molecular regulators. Consequently, pharmacological agents to control G protein activity are of utmost importance to advance our understanding of this signal transduction system. The natural depsipeptides FR900359 (FR) and YM-254890 (YM) are two highly specific and widely used inhibitors of heterotrimeric Gq/11 proteins. These compounds have traditionally been understood to inhibit GDP dissociation by preventing the separation of the GTPase and -helical domains of the G subunit. In this work, we have determined the high-resolution crystal structures of FR and YM bound to heterotrimeric G11 and used them to explain the molecular basis underlying their efficient suppression of G protein signaling. Notably, our data show that FR and YM also function as stabilizers of the interface between the G and G{beta} subunits, acting as  molecular adhesives that stabilize the entire heterotrimer. Our results reveal unrecognized mechanistic features that explain how FR and YM effectively blunt Gq/11 signaling in living cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.28.564055">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.28.564055" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.28.564055">
        <p class="paperTitle">Multi-scale photocatalytic proximity labeling reveals cell surface neighbors on and between cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.28.564055" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.28.564055" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lin, Z.; Schaefer, K.; Lui, I.; Yao, Z.; Fossati, A. A.; Swaney, D. L.; Palar, A.; Sali, A.; Wells, J. A.</p>
        <p class="info">Score: 8.1, Published: 2023-10-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.28.564055' target='https://doi.org/10.1101/2023.10.28.564055'> 10.1101/2023.10.28.564055</a></p>
        <p class="abstract">The cell membrane proteome is the primary biohub for cell communication, yet we are only beginning to understand the dynamic protein neighborhoods that form on the cell surface and between cells. Proximity labeling proteomics (PLP) strategies using chemically reactive probes are powerful approaches to yield snapshots of protein neighborhoods but are currently limited to one single resolution based on the probe labeling radius. Here, we describe a multi-scale PLP method with tunable resolution using a commercially available histological dye, Eosin Y, which upon visible light illumination, activates three different photo-probes with labeling radii ranging from ~100 to 3000 [a]. We applied this platform to profile neighborhoods of the oncogenic epidermal growth factor receptor (EGFR) and orthogonally validated &gt;20 neighbors using immuno-assays and AlphaFold-Multimer prediction that generated plausible binary interaction models. We further profiled the protein neighborhoods of cell-cell synapses induced by bi-specific T-cell engagers (BiTEs) and chimeric antigen receptor (CAR)T cells at longer length scales. This integrated multi-scale PLP platform maps local and distal protein networks on cell surfaces and between cells. We believe this information will aid in the systematic construction of the cell surface interactome and reveal new opportunities for immunotherapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564437">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564437" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564437">
        <p class="paperTitle">Determining the defining lengths between mature microRNAs/small interfering RNAs and tinyRNAs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564437" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564437" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sim, G.; Kehling, A. C.; Park, M. S.; Divoky, C.; Zhang, H.; Malhotra, N.; Secor, J.; Nakanishi, K.</p>
        <p class="info">Score: 7.8, Published: 2023-10-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564437' target='https://doi.org/10.1101/2023.10.27.564437'> 10.1101/2023.10.27.564437</a></p>
        <p class="abstract">MicroRNAs (miRNAs) and small interfering RNAs (siRNAs) are loaded into Argonaute (AGO) proteins, forming RNA-induced silencing complexes (RISCs). The assembly process establishes the seed, central, 3&#39; supplementary, and tail regions across the loaded guide, enabling the RISC to recognize and cleave target RNAs. This guide segmentation is caused by anchoring the 3&#39; end at the AGO PAZ domain, but the minimum guide length required for the conformation remains to be studied because there was no method by which to do so. Using a 3&#39;[-&amp;gt;]5&#39; exonuclease ISG20, we determined the lengths of AGO-associated miR-20a and let-7a with 3&#39; ends that no longer reach the PAZ domain. Unexpectedly, miR-20a and let-7a needed different lengths, 19 and 20 nt, respectively, to maintain their RISC conformation. This difference can be explained by the low affinity of the PAZ domain for the adenosine at g19 of let-7a, suggesting that the tail-region sequence slightly alters the minimum guide length. We also present that 17-nt guides are sufficiently short enough to function as tinyRNAs (tyRNAs) whose 3&#39; ends are not anchored at the PAZ domain. Since tyRNAs do not have the prerequisite anchoring for the standardized guide segmentation, they would recognize targets differently from miRNAs and siRNAs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.563785">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.563785" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.563785">
        <p class="paperTitle">Chemoproteomics identifies proteoform-selective caspase-2 inhibitors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.563785" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.563785" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Castellon, J. O.; Ofori, S.; Armenta, E.; Burton, N.; Boatner, L. M.; Takayoshi, E. E.; Faragalla, M.; Zhou, A.; Tran, K.; Shek, J.; Yan, T.; Desai, H. S.; Backus, K. M.</p>
        <p class="info">Score: 7.5, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.563785' target='https://doi.org/10.1101/2023.10.25.563785'> 10.1101/2023.10.25.563785</a></p>
        <p class="abstract">Caspases are a highly conserved family of cysteine-aspartyl proteases known for their essential roles in regulating apoptosis, inflammation, cell differentiation, and proliferation. Complementary to genetic approaches, small-molecule probes have emerged as useful tools for modulating caspase activity. However, due to the high sequence and structure homology of all twelve human caspases, achieving selectivity remains a central challenge for caspase-directed small-molecule inhibitor development efforts. Here, using mass spectrometry-based chemoproteomics, we first identify a highly reactive non-catalytic cysteine that is unique to caspase-2. By combining both gel-based activity-based protein profiling (ABPP) and a tobacco etch virus (TEV) protease activation assay, we then identify covalent lead compounds that react preferentially with this cysteine and afford a complete blockade of caspase-2 activity. Inhibitory activity is restricted to the zymogen or precursor form of monomeric caspase-2. Focused analogue synthesis combined with chemoproteomic target engagement analysis in cellular lysates and in cells yielded both pan-caspase reactive molecules and caspase-2 selective lead compounds together with a structurally matched inactive control. Application of this focused set of tool compounds to stratify caspase contributions to initiation of intrinsic apoptosis, supports compensatory caspase-9 activity in the context of caspase-2 inactivation. More broadly, our study highlights future opportunities for the development of proteoform-selective caspase inhibitors that target non-conserved and non-catalytic cysteine residues.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564164">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564164" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564164">
        <p class="paperTitle">A Toxoplasma gondii O-glycosyltransferase that modulates bradyzoite cyst wall rigidity is structurally and functionally distinct from host homologues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564164" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564164" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kumar, P.; Tomita, T.; Gerken, T. A.; Ballard, C. J.; Lee, Y. S.; Weiss, L. M.; Samara, N. L.</p>
        <p class="info">Score: 6.2, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564164' target='https://doi.org/10.1101/2023.10.26.564164'> 10.1101/2023.10.26.564164</a></p>
        <p class="abstract">Infection with the Apicomplexan protozoan Toxoplasma gondii can be life-threatening in immunocompromised hosts. Transmission frequently occurs through the oral ingestion of T. gondii bradyzoite cysts, which transition to tachyzoites, disseminate, and then form cysts containing bradyzoites in the central nervous system, resulting in latent infection. There are currently no effective treatments to cure latent infection. Bradyzoites are encapsulated by a cyst wall that is critical for immune evasion, survival, and transmission. Cyst wall rigidity is influenced by the O-glycosylation of the mucin domain of the cyst wall protein CST1 by mucin-type O-glycosyltransferases (GalNAc-Ts). Here, we report the first structures of a protozoan GalNAc-T, T.gondii-GalNAc-T3 in the apo state and in complex with glycopeptide substrates. The structures reveal features that are strictly conserved in Apicomplexan homologues of T.gondii-GalNAc-T3, including a unique 2nd metal binding site that is coupled to substrate binding and enzymatic activity in vitro and cyst wall O-glycosylation in T. gondii. Additional structural features illustrate the divergence of GalNAc-Ts from parasite to host and highlight multiple druggable sites in T.gondii-GalNAc-T3 and its homologues in Apicomplexa that are responsible for a wide range of parasitic diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.28.564488">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.28.564488" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.28.564488">
        <p class="paperTitle">Co-transcriptional capping using an RNA capping enzyme-T7 RNA polymerase fusion protein</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.28.564488" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.28.564488" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chan, S. H.; Tirole, L.; Kneller, D. W.; Kelley, T. M.; Dai, N.; Robb, G. B.</p>
        <p class="info">Score: 5.3, Published: 2023-10-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.28.564488' target='https://doi.org/10.1101/2023.10.28.564488'> 10.1101/2023.10.28.564488</a></p>
        <p class="abstract">mRNA vaccines and therapeutics are highly effective and can be developed and manufactured with a relatively short lead time. Here we report an all-enzyme platform to generate capped synthetic RNA in a one-step process based on an RNA capping enzyme-T7 RNA polymerase fusion protein. Under standard in vitro transcription reaction conditions, the fusion protein, in conjunction with an RNA cap 2&#39;-O-methyltransferase, can generate synthetic mRNA with up to 95% of Cap-1 incorporation, greatly simplifying mRNA manufacturing workflows.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
