<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="DNA end sensing and cleavage by the Shedu anti-phage defense system
Authors: Loeff, L.; Walter, A.; Rosalen, G. T.; Jinek, M.
Score: 29.8, Published: 2023-08-12 DOI: 10.1101/2023.08.10.552762
The detection of molecular patterns associated with invading pathogens is a hallmark of innate immune systems. Prokaryotes deploy sophisticated host-defense mechanisms in innate anti-phage immunity. Shedu is a single-component defense system comprising a putative nuclease SduA. Here we report cryo-EM structures of apo- and dsDNA-bound tetrameric SduA assemblies, revealing that the N-terminal domains of SduA form a clamp that recognizes free DNA ends.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="DNA end sensing and cleavage by the Shedu anti-phage defense system
Authors: Loeff, L.; Walter, A.; Rosalen, G. T.; Jinek, M.
Score: 29.8, Published: 2023-08-12 DOI: 10.1101/2023.08.10.552762
The detection of molecular patterns associated with invading pathogens is a hallmark of innate immune systems. Prokaryotes deploy sophisticated host-defense mechanisms in innate anti-phage immunity. Shedu is a single-component defense system comprising a putative nuclease SduA. Here we report cryo-EM structures of apo- and dsDNA-bound tetrameric SduA assemblies, revealing that the N-terminal domains of SduA form a clamp that recognizes free DNA ends." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-13T10:36:01+00:00" />
<meta property="article:modified_time" content="2023-08-13T10:36:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="DNA end sensing and cleavage by the Shedu anti-phage defense system
Authors: Loeff, L.; Walter, A.; Rosalen, G. T.; Jinek, M.
Score: 29.8, Published: 2023-08-12 DOI: 10.1101/2023.08.10.552762
The detection of molecular patterns associated with invading pathogens is a hallmark of innate immune systems. Prokaryotes deploy sophisticated host-defense mechanisms in innate anti-phage immunity. Shedu is a single-component defense system comprising a putative nuclease SduA. Here we report cryo-EM structures of apo- and dsDNA-bound tetrameric SduA assemblies, revealing that the N-terminal domains of SduA form a clamp that recognizes free DNA ends."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "DNA end sensing and cleavage by the Shedu anti-phage defense system\nAuthors: Loeff, L.; Walter, A.; Rosalen, G. T.; Jinek, M.\nScore: 29.8, Published: 2023-08-12 DOI: 10.1101/2023.08.10.552762\nThe detection of molecular patterns associated with invading pathogens is a hallmark of innate immune systems. Prokaryotes deploy sophisticated host-defense mechanisms in innate anti-phage immunity. Shedu is a single-component defense system comprising a putative nuclease SduA. Here we report cryo-EM structures of apo- and dsDNA-bound tetrameric SduA assemblies, revealing that the N-terminal domains of SduA form a clamp that recognizes free DNA ends.",
  "keywords": [
    
  ],
  "articleBody": " DNA end sensing and cleavage by the Shedu anti-phage defense system\nAuthors: Loeff, L.; Walter, A.; Rosalen, G. T.; Jinek, M.\nScore: 29.8, Published: 2023-08-12 DOI: 10.1101/2023.08.10.552762\nThe detection of molecular patterns associated with invading pathogens is a hallmark of innate immune systems. Prokaryotes deploy sophisticated host-defense mechanisms in innate anti-phage immunity. Shedu is a single-component defense system comprising a putative nuclease SduA. Here we report cryo-EM structures of apo- and dsDNA-bound tetrameric SduA assemblies, revealing that the N-terminal domains of SduA form a clamp that recognizes free DNA ends. End binding positions the DNA over the PD/ExK nuclease domain, resulting in dsDNA nicking at a fixed distance from the 5-prime end. The end-directed DNA nicking activity of Shedu prevents propagation of linear DNA in vivo. Finally, we show that phages escape Shedu immunity by suppressing their recombination-dependent DNA replication pathway. Taken together, these results define the antiviral mechanism of Shedu systems, underlining the paradigm that recognition of pathogen-specific nucleic acid structures is a conserved feature of innate immunity across all domains of life.\nThe ER-SURF pathway uses ER-mitochondria contact sites for protein targeting to mitochondria\nAuthors: Koch, C.; Raeschle, M.; Prescianotto-Baschong, C.; Spang, A.; Herrmann, J. M.\nScore: 26.4, Published: 2023-08-11 DOI: 10.1101/2023.08.10.552816\nMost mitochondrial proteins are synthesized on cytosolic ribosomes and imported into mitochondria in a post-translational reaction. Mitochondrial precursor proteins which use the ER-SURF pathway employ the surface of the endoplasmic reticulum (ER) as an important sorting platform. How they reach the mitochondrial import machinery from the ER is not known. Here we show that mitochondrial contact sites play a crucial role in the ER-to-mitochondria transfer of precursor proteins. The ER encounter structure (ERMES) and Tom70 are part of two parallel and partially redundant ER-to-mitochondria transfer routes. When ER-to-mitochondria transfer is prevented, many mitochondrial precursor proteins associate with ER membranes, resulting in mitochondrial dysfunction. Our observations support an active role of the ER in mitochondrial protein biogenesis.\nRapid and automated design of two-component protein nanomaterials using ProteinMPNN\nAuthors: de Haas, R. J.; Brunette, N.; Goodson, A.; Dauparas, J.; Yi, S. Y.; Yang, E. C.; Dowling, Q.; Nguyen, H.; Kang, A.; Bera, A. K.; Sankaran, B.; de Vries, R.; Baker, D.; King, N. P.\nScore: 23.8, Published: 2023-08-04 DOI: 10.1101/2023.08.04.551935\nThe design of novel protein-protein interfaces using physics-based design methods such as Rosetta requires substantial computational resources and manual refinement by expert structural biologists. A new generation of deep learning methods promises to simplify protein-protein interface design and enable its application to a wide variety of problems by researchers from various scientific disciplines. Here we test the ability of a deep learning method for protein sequence design, ProteinMPNN, to design two-component tetrahedral protein nanomaterials and benchmark its performance against Rosetta. ProteinMPNN had a similar success rate to Rosetta, yielding 13 new experimentally confirmed assemblies, but required orders of magnitude less computation and no manual refinement. The interfaces designed by ProteinMPNN were substantially more polar than those designed by Rosetta, which facilitated in vitro assembly of the designed nanomaterials from independently purified components. Crystal structures of several of the assemblies confirmed the accuracy of the design method at high resolution. Our results showcase the potential of deep learning-based methods to unlock the widespread application of designed protein-protein interfaces and self-assembling protein nanomaterials in biotechnology.\nOne-Tip enables comprehensive proteome coverage in minimal cells and single zygotes\nAuthors: Ye, Z.; Sabatier, P.; Martin-Gonzalez, J.; Eguchi, A.; Bekker-Jensen, D. B.; Bache, N.; Olsen, J. V.\nScore: 15.1, Published: 2023-08-11 DOI: 10.1101/2023.08.10.552756\nWe present One-Tip, a lossless proteomics methodology that seamlessly combines swift, one-pot sample preparation with narrow-window data-independent acquisition mass spectrometric analysis. With simplest sample processing, One-Tip reproducibly identifies \u003e 9,000 proteins from ~1000 cells and ~ 6,000 proteins in a single mouse zygote with a throughput of 40 samples-per-day. This easy-to-use method expands capabilities of proteomics research by enabling greater depth, scalability and throughput covering low to high input samples.\nMolecular insights into the activation of Mre11-Rad50 endonuclease activity by Sae2/CtIP\nAuthors: Nicolas, Y.; Bret, H.; Cannavo, E.; Cejka, P.; Borde, V.; Guerois, R.\nScore: 13.1, Published: 2023-08-05 DOI: 10.1101/2023.08.05.552090\nIn S. cerevisiae, the Mre11-Rad50-Xrs2 (MRX)-Sae2 nuclease activity is required for the resection of DNA breaks with secondary structures or protein blocks, while in humans, the MRE11-RAD50-NBS1 (MRN) homologue with CtIP is needed to initiate DNA end resection of all breaks. Phosphorylated Sae2/CtIP stimulates the endonuclease activity of MRX/N. Structural insights into the activation of the Mre11 nuclease are available only for organisms lacking Sae2/CtIP, so little is known how Sae2/CtIP activates the nuclease ensemble. Here, we uncover the mechanism of Mre11 activation by Sae2 using a combination of AlphaFold2 structural modeling, biochemical and genetic assays. We show that Sae2 stabilizes the Mre11 nuclease in a conformation poised to cleave substrate DNA. Several designs of compensatory mutations establish how Sae2 activates MRX in vitro and in vivo, validating the structural model. Last, our study uncovers how human CtIP, despite considerable sequence divergence, employs a similar mechanism to activate MRN.\nIdentification of factors that prevent ribosome stalling during early elongation\nAuthors: Hong, H.-R.; Prince, C. R.; Wu, L.; Feaga, H. A.\nScore: 25.5, Published: 2023-08-04 DOI: 10.1101/2023.08.04.552005\nProtein synthesis is catalyzed by the ribosome and a host of highly conserved elongation factors. Most elongation factors that are conserved in all domains of life are essential, such as EF-Tu (e/aEF1A) and EF-G (e/aEF2). In contrast, the universally conserved elongation factor P (EF-P/eIF5A) is essential in eukaryotes but is dispensable in most bacteria. EF-P prevents ribosome stalling at difficult-to translate sequences, especially polyprolines. Since efp deletion phenotypes range from modest to lethal in different bacterial species, we hypothesized that some bacteria encode an uncharacterized elongation factor with compensatory functions. To identify this factor, we used Tn-seq to screen for genes that are essential in the absence of EF-P in Bacillus subtilis. This screen identified YfmR, a member of the ABCF family of ATPases, as a translation factor that is essential when efp is deleted. We find that depleting YfmR from {Delta}efp cells decreases actively translating ribosomes and increases free ribosomal subunits bound to initiator tRNA, suggesting that ribosomes stall in early elongation. We also find that deleting efp from the spore-forming pathogen Bacillus anthracis causes a severe survival defect in liquid culture and in the presence of macrophages. B. anthracis {Delta}efp also does not produce detectable spores. We find that heterologous expression of B. subtilis YfmR in B. anthracis {Delta}efp cells partially rescues the severe growth and sporulation defects of this mutant. Our results therefore identify YfmR as an important translation factor that becomes essential in the absence of EF-P, and suggest that the essential function of YfmR and EF-P is to assist translation during early elongation. SignificanceTranslation is one of the most ancient and energetically demanding process that occurs in the cell. Ribosomes constitute more than 60% of cellular mass in actively growing cells, and ribosomes are a major target of antimicrobials and chemotherapeutics. Here, we report evidence that both EF-P and the uncharacterized protein YfmR are important for early translation elongation in bacteria, and that this activity is essential. YfmR is a member of the ABCF family of ATPases whose role in translation is only beginning to be understood. Given the broad distribution of ABCFs from bacteria to fungi, we expect our results to have implications for understanding the crucial first steps of translation elongation in diverse organisms.\nTOM20 Gates PINK1 Activity And Mediates Its Tethering Of The TOM And TIM23 Translocases Upon Mitochondrial Stress\nAuthors: Eldeeb, M.; Bayne, A.; Fallahi, A.; Goiran, T.; McDougall, E.; Soumbasis, A.; Zorca, C.; Tabah, J.-J.; Thomas, R.; Karpilovsky, N.; Mathur, M.; Durcan, T.; Trempe, J.-F.; Fon, E.\nScore: 8.9, Published: 2023-08-07 DOI: 10.1101/2023.08.07.552252\nMutations in PTEN-induced putative kinase 1 (PINK1) cause autosomal recessive early onset Parkinson disease (PD). PINK1 is a Ser/Thr kinase that regulates mitochondrial quality control by triggering mitophagy mediated by the ubiquitin ligase Parkin. Upon mitochondrial damage, PINK1 accumulates on the outer mitochondrial membrane (OMM) forming a high molecular weight complex with the translocase of the outer membrane (TOM). PINK1 then phosphorylates ubiquitin, which enables recruitment and activation of Parkin followed by autophagic clearance of the damaged mitochondrion. Thus, Parkin-dependent mitophagy hinges on the stable accumulation of PINK1 on the TOM complex. Yet, the mechanism linking mitochondrial stressors to PINK1 accumulation and whether the translocases of the inner membrane (TIMs) are also involved, remain unclear. Herein, we demonstrate that mitochondrial stress induces the formation of a PINK1-TOM-TIM23 supercomplex in human cultured cell lines, dopamine neurons, and midbrain organoids. Moreover, we show that PINK1 is required to stably tether the TOM to TIM23 complexes in response to stress, such that the supercomplex fails to accumulate in cells lacking PINK1. This tethering is dependent on an interaction between the PINK1 NT-CTE module and the cytosolic domain of the Tom20 subunit of the TOM complex, the disruption of which, by either designer or PD-associated PINK1 mutations, inhibits downstream mitophagy. Together, the findings provide key insight into how PINK1 interfaces with the mitochondrial import machinery, with important implications for the mechanisms of mitochondrial quality control and PD pathogenesis.\nThe HuMet Repository: Watching human metabolism at work\nAuthors: Weinisch, P.; Raffler, J.; Römisch-Margl, W.; Arnold, M.; Mohney, R. P.; Rist, M. J.; Prehn, C.; Skurk, T.; Hauner, H.; Daniel, H.; Suhre, K.; Kastenmüller, G.\nScore: 6.3, Published: 2023-08-09 DOI: 10.1101/2023.08.08.550079\nThe human metabolism constantly responds to stimuli such as food intake, fasting, exercise, and stress, triggering adaptive biochemical processes across multiple metabolic pathways. To understand the role of these processes and disruptions thereof in health and disease, detailed documentation of healthy metabolic responses is needed but still scarce on a time-resolved metabolome-wide level. Here, we present the HuMet Repository, a web-based resource for exploring dynamic metabolic responses to six physiological challenges (exercise, 36 h fasting, oral glucose and lipid loads, mixed meal, cold stress) in healthy subjects. For building this resource, we integrated existing and newly derived metabolomics data measured in blood, urine, and breath samples of 15 young healthy men at up to 56 time points during the six highly standardized challenge tests conducted over four days. The data comprise 1.1 million data points acquired on multiple platforms with temporal profiles of 2,656 metabolites from a broad range of biochemical pathways. By embedding the dataset into an interactive web application, we enable users to easily access, search, filter, analyze, and visualize the time-resolved metabolomic readouts and derived results. Users can put metabolites into their larger context by identifying metabolites with similar trajectories or by visualizing metabolites within holistic metabolic networks to pinpoint pathways of interest. In three showcases, we outline the value of the repository for gaining biological insights and generating hypotheses by analyzing the wash-out of dietary markers, the complementarity of metabolomics platforms in dynamic versus cross-sectional data, and similarities and differences in systemic metabolic responses across challenges. With its comprehensive collection of time-resolved metabolomics data, the HuMet Repository, freely accessible at https://humet.org/, is a reference for normal, healthy responses to metabolic challenges in young males. It will enable researchers with and without computational expertise, to flexibly query the data for their own research into the dynamics of human metabolism.\nCryo-EM structure of DDM1-HELLS chimera bound to nucleosome reveals a mechanism of chromatin remodeling and disease regulation\nAuthors: Nartey, W.; Goodarzi, A. A.; Williams, G. J.\nScore: 4.8, Published: 2023-08-09 DOI: 10.1101/2023.08.09.551721\nHuman HELicase, Lymphoid Specific (HELLS), and plant homolog Deficient in DNA Methylation 1 (DDM1), belong to a distinct class of chromatin remodelers that play important roles in DNA repair, transcription, and maintenance of DNA methylation in heterochromatin. HELLS also promotes the growth of hard-to-treat cancers including glioblastoma and hepatocellular carcinoma. Here, we identify an auto-inhibitory HELLS N-terminal coiled-coil, unravelling a long-standing question of HELLS inactivity in vitro. Using cryo-EM, we determine the 3.5 [A] structure of an active DDM1-HELLS chimera in complex with a nucleosome. The structure reveals that a HELLS-specific insertion in the ATPase lobe 2 interacts with the nucleosome acidic patch to enhance chromatin remodeling. At the C-terminus, we resolve a unique motif, and disease hot spot, that binds and stabilizes the ATPase motor of the HELLS family of remodelers. Finally, we provide mechanistic insights for how post-translational modifications in the motor domain and midloop could modulate HELLS activity to regulate cancer stem cell state.\nTarget-aware Variational Auto-encoders for Ligand Generation with Multimodal Protein Representation Learning\nAuthors: Ngo, N. K.; Hy, T. S.\nScore: 4.7, Published: 2023-08-11 DOI: 10.1101/2023.08.10.552868\nWithout knowledge of specific pockets, generating ligands based on the global structure of a protein target plays a crucial role in drug discovery as it helps reduce the search space for potential drug-like candidates in the pipeline. However, contemporary methods require optimizing tailored networks for each protein, which is arduous and costly. To address this issue, we introduce TargetVAE, a target-aware variational auto-encoder that generates ligands with high binding affinities to arbitrary protein targets, guided by a novel multimodal deep neural network built based on graph Transformers as the prior for the generative model. This is the first effort to unify different representations of proteins (e.g., sequence of amino-acids, 3D structure) into a single model that we name as Protein Multimodal Network (PMN). Our multimodal architecture learns from the entire protein structures and is able to capture their sequential, topological and geometrical information. We showcase the superiority of our approach by conducting extensive experiments and evaluations, including the assessment of generative model quality, ligand generation for unseen targets, docking score computation, and binding affinity prediction. Empirical results demonstrate the promising performance of our proposed approach. Our software package is publicly available at https://github.com/HySonLab/Ligand_Generation\n",
  "wordCount" : "2261",
  "inLanguage": "en",
  "datePublished": "2023-08-13T10:36:01Z",
  "dateModified": "2023-08-13T10:36:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta"><span>updated on August 13, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552762">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552762" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552762">
        <p class="paperTitle">DNA end sensing and cleavage by the Shedu anti-phage defense system</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552762" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552762" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Loeff, L.; Walter, A.; Rosalen, G. T.; Jinek, M.</p>
        <p class="info">Score: 29.8, Published: 2023-08-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552762' target='https://doi.org/10.1101/2023.08.10.552762'> 10.1101/2023.08.10.552762</a></p>
        <p class="abstract">The detection of molecular patterns associated with invading pathogens is a hallmark of innate immune systems. Prokaryotes deploy sophisticated host-defense mechanisms in innate anti-phage immunity. Shedu is a single-component defense system comprising a putative nuclease SduA. Here we report cryo-EM structures of apo- and dsDNA-bound tetrameric SduA assemblies, revealing that the N-terminal domains of SduA form a clamp that recognizes free DNA ends. End binding positions the DNA over the PD/ExK nuclease domain, resulting in dsDNA nicking at a fixed distance from the 5-prime end. The end-directed DNA nicking activity of Shedu prevents propagation of linear DNA in vivo. Finally, we show that phages escape Shedu immunity by suppressing their recombination-dependent DNA replication pathway. Taken together, these results define the antiviral mechanism of Shedu systems, underlining the paradigm that recognition of pathogen-specific nucleic acid structures is a conserved feature of innate immunity across all domains of life.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552816">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552816" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552816">
        <p class="paperTitle">The ER-SURF pathway uses ER-mitochondria contact sites for protein targeting to mitochondria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552816" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552816" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Koch, C.; Raeschle, M.; Prescianotto-Baschong, C.; Spang, A.; Herrmann, J. M.</p>
        <p class="info">Score: 26.4, Published: 2023-08-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552816' target='https://doi.org/10.1101/2023.08.10.552816'> 10.1101/2023.08.10.552816</a></p>
        <p class="abstract">Most mitochondrial proteins are synthesized on cytosolic ribosomes and imported into mitochondria in a post-translational reaction. Mitochondrial precursor proteins which use the ER-SURF pathway employ the surface of the endoplasmic reticulum (ER) as an important sorting platform. How they reach the mitochondrial import machinery from the ER is not known. Here we show that mitochondrial contact sites play a crucial role in the ER-to-mitochondria transfer of precursor proteins. The ER encounter structure (ERMES) and Tom70 are part of two parallel and partially redundant ER-to-mitochondria transfer routes. When ER-to-mitochondria transfer is prevented, many mitochondrial precursor proteins associate with ER membranes, resulting in mitochondrial dysfunction. Our observations support an active role of the ER in mitochondrial protein biogenesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.551935">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.551935" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.551935">
        <p class="paperTitle">Rapid and automated design of two-component protein nanomaterials using ProteinMPNN</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.551935" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.551935" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: de Haas, R. J.; Brunette, N.; Goodson, A.; Dauparas, J.; Yi, S. Y.; Yang, E. C.; Dowling, Q.; Nguyen, H.; Kang, A.; Bera, A. K.; Sankaran, B.; de Vries, R.; Baker, D.; King, N. P.</p>
        <p class="info">Score: 23.8, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.551935' target='https://doi.org/10.1101/2023.08.04.551935'> 10.1101/2023.08.04.551935</a></p>
        <p class="abstract">The design of novel protein-protein interfaces using physics-based design methods such as Rosetta requires substantial computational resources and manual refinement by expert structural biologists. A new generation of deep learning methods promises to simplify protein-protein interface design and enable its application to a wide variety of problems by researchers from various scientific disciplines. Here we test the ability of a deep learning method for protein sequence design, ProteinMPNN, to design two-component tetrahedral protein nanomaterials and benchmark its performance against Rosetta. ProteinMPNN had a similar success rate to Rosetta, yielding 13 new experimentally confirmed assemblies, but required orders of magnitude less computation and no manual refinement. The interfaces designed by ProteinMPNN were substantially more polar than those designed by Rosetta, which facilitated in vitro assembly of the designed nanomaterials from independently purified components. Crystal structures of several of the assemblies confirmed the accuracy of the design method at high resolution. Our results showcase the potential of deep learning-based methods to unlock the widespread application of designed protein-protein interfaces and self-assembling protein nanomaterials in biotechnology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552756">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552756" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552756">
        <p class="paperTitle">One-Tip enables comprehensive proteome coverage in minimal cells and single zygotes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552756" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552756" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ye, Z.; Sabatier, P.; Martin-Gonzalez, J.; Eguchi, A.; Bekker-Jensen, D. B.; Bache, N.; Olsen, J. V.</p>
        <p class="info">Score: 15.1, Published: 2023-08-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552756' target='https://doi.org/10.1101/2023.08.10.552756'> 10.1101/2023.08.10.552756</a></p>
        <p class="abstract">We present One-Tip, a lossless proteomics methodology that seamlessly combines swift, one-pot sample preparation with narrow-window data-independent acquisition mass spectrometric analysis. With simplest sample processing, One-Tip reproducibly identifies &gt; 9,000 proteins from ~1000 cells and ~ 6,000 proteins in a single mouse zygote with a throughput of 40 samples-per-day. This easy-to-use method expands capabilities of proteomics research by enabling greater depth, scalability and throughput covering low to high input samples.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.05.552090">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.05.552090" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.05.552090">
        <p class="paperTitle">Molecular insights into the activation of Mre11-Rad50 endonuclease activity by Sae2/CtIP</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.05.552090" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.05.552090" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nicolas, Y.; Bret, H.; Cannavo, E.; Cejka, P.; Borde, V.; Guerois, R.</p>
        <p class="info">Score: 13.1, Published: 2023-08-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.05.552090' target='https://doi.org/10.1101/2023.08.05.552090'> 10.1101/2023.08.05.552090</a></p>
        <p class="abstract">In S. cerevisiae, the Mre11-Rad50-Xrs2 (MRX)-Sae2 nuclease activity is required for the resection of DNA breaks with secondary structures or protein blocks, while in humans, the MRE11-RAD50-NBS1 (MRN) homologue with CtIP is needed to initiate DNA end resection of all breaks. Phosphorylated Sae2/CtIP stimulates the endonuclease activity of MRX/N. Structural insights into the activation of the Mre11 nuclease are available only for organisms lacking Sae2/CtIP, so little is known how Sae2/CtIP activates the nuclease ensemble. Here, we uncover the mechanism of Mre11 activation by Sae2 using a combination of AlphaFold2 structural modeling, biochemical and genetic assays. We show that Sae2 stabilizes the Mre11 nuclease in a conformation poised to cleave substrate DNA. Several designs of compensatory mutations establish how Sae2 activates MRX in vitro and in vivo, validating the structural model. Last, our study uncovers how human CtIP, despite considerable sequence divergence, employs a similar mechanism to activate MRN.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.552005">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.552005" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.552005">
        <p class="paperTitle">Identification of factors that prevent ribosome stalling during early elongation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.552005" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.552005" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hong, H.-R.; Prince, C. R.; Wu, L.; Feaga, H. A.</p>
        <p class="info">Score: 25.5, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.552005' target='https://doi.org/10.1101/2023.08.04.552005'> 10.1101/2023.08.04.552005</a></p>
        <p class="abstract">Protein synthesis is catalyzed by the ribosome and a host of highly conserved elongation factors. Most elongation factors that are conserved in all domains of life are essential, such as EF-Tu (e/aEF1A) and EF-G (e/aEF2). In contrast, the universally conserved elongation factor P (EF-P/eIF5A) is essential in eukaryotes but is dispensable in most bacteria. EF-P prevents ribosome stalling at difficult-to translate sequences, especially polyprolines. Since efp deletion phenotypes range from modest to lethal in different bacterial species, we hypothesized that some bacteria encode an uncharacterized elongation factor with compensatory functions. To identify this factor, we used Tn-seq to screen for genes that are essential in the absence of EF-P in Bacillus subtilis. This screen identified YfmR, a member of the ABCF family of ATPases, as a translation factor that is essential when efp is deleted. We find that depleting YfmR from {Delta}efp cells decreases actively translating ribosomes and increases free ribosomal subunits bound to initiator tRNA, suggesting that ribosomes stall in early elongation. We also find that deleting efp from the spore-forming pathogen Bacillus anthracis causes a severe survival defect in liquid culture and in the presence of macrophages. B. anthracis {Delta}efp also does not produce detectable spores. We find that heterologous expression of B. subtilis YfmR in B. anthracis {Delta}efp cells partially rescues the severe growth and sporulation defects of this mutant. Our results therefore identify YfmR as an important translation factor that becomes essential in the absence of EF-P, and suggest that the essential function of YfmR and EF-P is to assist translation during early elongation.

SignificanceTranslation is one of the most ancient and energetically demanding process that occurs in the cell. Ribosomes constitute more than 60% of cellular mass in actively growing cells, and ribosomes are a major target of antimicrobials and chemotherapeutics. Here, we report evidence that both EF-P and the uncharacterized protein YfmR are important for early translation elongation in bacteria, and that this activity is essential. YfmR is a member of the ABCF family of ATPases whose role in translation is only beginning to be understood. Given the broad distribution of ABCFs from bacteria to fungi, we expect our results to have implications for understanding the crucial first steps of translation elongation in diverse organisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552252">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552252" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552252">
        <p class="paperTitle">TOM20 Gates PINK1 Activity And Mediates Its Tethering Of The TOM And TIM23 Translocases Upon Mitochondrial Stress</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552252" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552252" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eldeeb, M.; Bayne, A.; Fallahi, A.; Goiran, T.; McDougall, E.; Soumbasis, A.; Zorca, C.; Tabah, J.-J.; Thomas, R.; Karpilovsky, N.; Mathur, M.; Durcan, T.; Trempe, J.-F.; Fon, E.</p>
        <p class="info">Score: 8.9, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552252' target='https://doi.org/10.1101/2023.08.07.552252'> 10.1101/2023.08.07.552252</a></p>
        <p class="abstract">Mutations in PTEN-induced putative kinase 1 (PINK1) cause autosomal recessive early onset Parkinson disease (PD). PINK1 is a Ser/Thr kinase that regulates mitochondrial quality control by triggering mitophagy mediated by the ubiquitin ligase Parkin. Upon mitochondrial damage, PINK1 accumulates on the outer mitochondrial membrane (OMM) forming a high molecular weight complex with the translocase of the outer membrane (TOM). PINK1 then phosphorylates ubiquitin, which enables recruitment and activation of Parkin followed by autophagic clearance of the damaged mitochondrion. Thus, Parkin-dependent mitophagy hinges on the stable accumulation of PINK1 on the TOM complex. Yet, the mechanism linking mitochondrial stressors to PINK1 accumulation and whether the translocases of the inner membrane (TIMs) are also involved, remain unclear. Herein, we demonstrate that mitochondrial stress induces the formation of a PINK1-TOM-TIM23 supercomplex in human cultured cell lines, dopamine neurons, and midbrain organoids. Moreover, we show that PINK1 is required to stably tether the TOM to TIM23 complexes in response to stress, such that the supercomplex fails to accumulate in cells lacking PINK1. This tethering is dependent on an interaction between the PINK1 NT-CTE module and the cytosolic domain of the Tom20 subunit of the TOM complex, the disruption of which, by either designer or PD-associated PINK1 mutations, inhibits downstream mitophagy. Together, the findings provide key insight into how PINK1 interfaces with the mitochondrial import machinery, with important implications for the mechanisms of mitochondrial quality control and PD pathogenesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.550079">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.550079" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.550079">
        <p class="paperTitle">The HuMet Repository: Watching human metabolism at work</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.550079" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.550079" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Weinisch, P.; Raffler, J.; Römisch-Margl, W.; Arnold, M.; Mohney, R. P.; Rist, M. J.; Prehn, C.; Skurk, T.; Hauner, H.; Daniel, H.; Suhre, K.; Kastenmüller, G.</p>
        <p class="info">Score: 6.3, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.550079' target='https://doi.org/10.1101/2023.08.08.550079'> 10.1101/2023.08.08.550079</a></p>
        <p class="abstract">The human metabolism constantly responds to stimuli such as food intake, fasting, exercise, and stress, triggering adaptive biochemical processes across multiple metabolic pathways. To understand the role of these processes and disruptions thereof in health and disease, detailed documentation of healthy metabolic responses is needed but still scarce on a time-resolved metabolome-wide level.

Here, we present the HuMet Repository, a web-based resource for exploring dynamic metabolic responses to six physiological challenges (exercise, 36 h fasting, oral glucose and lipid loads, mixed meal, cold stress) in healthy subjects. For building this resource, we integrated existing and newly derived metabolomics data measured in blood, urine, and breath samples of 15 young healthy men at up to 56 time points during the six highly standardized challenge tests conducted over four days. The data comprise 1.1 million data points acquired on multiple platforms with temporal profiles of 2,656 metabolites from a broad range of biochemical pathways. By embedding the dataset into an interactive web application, we enable users to easily access, search, filter, analyze, and visualize the time-resolved metabolomic readouts and derived results. Users can put metabolites into their larger context by identifying metabolites with similar trajectories or by visualizing metabolites within holistic metabolic networks to pinpoint pathways of interest. In three showcases, we outline the value of the repository for gaining biological insights and generating hypotheses by analyzing the wash-out of dietary markers, the complementarity of metabolomics platforms in dynamic versus cross-sectional data, and similarities and differences in systemic metabolic responses across challenges.

With its comprehensive collection of time-resolved metabolomics data, the HuMet Repository, freely accessible at https://humet.org/, is a reference for normal, healthy responses to metabolic challenges in young males. It will enable researchers with and without computational expertise, to flexibly query the data for their own research into the dynamics of human metabolism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.551721">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.551721" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.551721">
        <p class="paperTitle">Cryo-EM structure of DDM1-HELLS chimera bound to nucleosome reveals a mechanism of chromatin remodeling and disease regulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.551721" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.551721" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nartey, W.; Goodarzi, A. A.; Williams, G. J.</p>
        <p class="info">Score: 4.8, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.551721' target='https://doi.org/10.1101/2023.08.09.551721'> 10.1101/2023.08.09.551721</a></p>
        <p class="abstract">Human HELicase, Lymphoid Specific (HELLS), and plant homolog Deficient in DNA Methylation 1 (DDM1), belong to a distinct class of chromatin remodelers that play important roles in DNA repair, transcription, and maintenance of DNA methylation in heterochromatin. HELLS also promotes the growth of hard-to-treat cancers including glioblastoma and hepatocellular carcinoma. Here, we identify an auto-inhibitory HELLS N-terminal coiled-coil, unravelling a long-standing question of HELLS inactivity in vitro. Using cryo-EM, we determine the 3.5 [A] structure of an active DDM1-HELLS chimera in complex with a nucleosome. The structure reveals that a HELLS-specific insertion in the ATPase lobe 2 interacts with the nucleosome acidic patch to enhance chromatin remodeling. At the C-terminus, we resolve a unique motif, and disease hot spot, that binds and stabilizes the ATPase motor of the HELLS family of remodelers. Finally, we provide mechanistic insights for how post-translational modifications in the motor domain and midloop could modulate HELLS activity to regulate cancer stem cell state.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552868">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552868" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552868">
        <p class="paperTitle">Target-aware Variational Auto-encoders for Ligand Generation with Multimodal Protein Representation Learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552868" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552868" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ngo, N. K.; Hy, T. S.</p>
        <p class="info">Score: 4.7, Published: 2023-08-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552868' target='https://doi.org/10.1101/2023.08.10.552868'> 10.1101/2023.08.10.552868</a></p>
        <p class="abstract">Without knowledge of specific pockets, generating ligands based on the global structure of a protein target plays a crucial role in drug discovery as it helps reduce the search space for potential drug-like candidates in the pipeline. However, contemporary methods require optimizing tailored networks for each protein, which is arduous and costly. To address this issue, we introduce TargetVAE, a target-aware variational auto-encoder that generates ligands with high binding affinities to arbitrary protein targets, guided by a novel multimodal deep neural network built based on graph Transformers as the prior for the generative model. This is the first effort to unify different representations of proteins (e.g., sequence of amino-acids, 3D structure) into a single model that we name as Protein Multimodal Network (PMN). Our multimodal architecture learns from the entire protein structures and is able to capture their sequential, topological and geometrical information. We showcase the superiority of our approach by conducting extensive experiments and evaluations, including the assessment of generative model quality, ligand generation for unseen targets, docking score computation, and binding affinity prediction. Empirical results demonstrate the promising performance of our proposed approach. Our software package is publicly available at https://github.com/HySonLab/Ligand_Generation</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
