<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Structural basis for the inhibition of PRC2 by active transcription histone posttranslational modifications
Authors: Cookis, T.; Lydecker, A.; Sauer, P. V.; Kasinath, V.; Nogales, E.
Score: 25.2, Published: 2024-02-10 DOI: 10.1101/2024.02.09.579730
Polycomb repressive complex 2 (PRC2) is an epigenetic regulator essential for embryonic development and maintenance of cell identity that trimethylates histone H3 at lysine 27 (H3K27me3) leading to gene silencing. PRC2 is regulated by association with protein cofactors and crosstalk with histone posttranslational modifications.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Structural basis for the inhibition of PRC2 by active transcription histone posttranslational modifications
Authors: Cookis, T.; Lydecker, A.; Sauer, P. V.; Kasinath, V.; Nogales, E.
Score: 25.2, Published: 2024-02-10 DOI: 10.1101/2024.02.09.579730
Polycomb repressive complex 2 (PRC2) is an epigenetic regulator essential for embryonic development and maintenance of cell identity that trimethylates histone H3 at lysine 27 (H3K27me3) leading to gene silencing. PRC2 is regulated by association with protein cofactors and crosstalk with histone posttranslational modifications." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-18T10:38:00+00:00" />
<meta property="article:modified_time" content="2024-02-18T10:38:00+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Structural basis for the inhibition of PRC2 by active transcription histone posttranslational modifications
Authors: Cookis, T.; Lydecker, A.; Sauer, P. V.; Kasinath, V.; Nogales, E.
Score: 25.2, Published: 2024-02-10 DOI: 10.1101/2024.02.09.579730
Polycomb repressive complex 2 (PRC2) is an epigenetic regulator essential for embryonic development and maintenance of cell identity that trimethylates histone H3 at lysine 27 (H3K27me3) leading to gene silencing. PRC2 is regulated by association with protein cofactors and crosstalk with histone posttranslational modifications."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Structural basis for the inhibition of PRC2 by active transcription histone posttranslational modifications\nAuthors: Cookis, T.; Lydecker, A.; Sauer, P. V.; Kasinath, V.; Nogales, E.\nScore: 25.2, Published: 2024-02-10 DOI: 10.1101/2024.02.09.579730\nPolycomb repressive complex 2 (PRC2) is an epigenetic regulator essential for embryonic development and maintenance of cell identity that trimethylates histone H3 at lysine 27 (H3K27me3) leading to gene silencing. PRC2 is regulated by association with protein cofactors and crosstalk with histone posttranslational modifications.",
  "keywords": [
    
  ],
  "articleBody": " Structural basis for the inhibition of PRC2 by active transcription histone posttranslational modifications\nAuthors: Cookis, T.; Lydecker, A.; Sauer, P. V.; Kasinath, V.; Nogales, E.\nScore: 25.2, Published: 2024-02-10 DOI: 10.1101/2024.02.09.579730\nPolycomb repressive complex 2 (PRC2) is an epigenetic regulator essential for embryonic development and maintenance of cell identity that trimethylates histone H3 at lysine 27 (H3K27me3) leading to gene silencing. PRC2 is regulated by association with protein cofactors and crosstalk with histone posttranslational modifications. Trimethylated histone H3 K4 (H3K4me3) and K36 (H3K36me3) localize to sites of active transcription where H3K27me3 is absent and inhibit PRC2 activity through unknown mechanisms. Using cryo-electron microscopy we reveal that histone H3 tails modified with H3K36me3 engage poorly with the PRC2 active site and preclude its effective interaction with chromatin, while the H3K4me3 modification binds to the allosteric site in the EED subunit, acting as an antagonist that competes with allosteric activators required for the spreading of the H3K27me3 repressive mark. Thus, the location along the H3 tail of the H3K4me3 and H3K36me3 modifications allow them to target two essential requirements for efficient trimethylation of histone H3K27. We further show that the JARID2 cofactor modulates PRC2 activity in the presence of these histone modifications.\nThe human Shu complex promotes RAD51 activity by modulating RPA dynamics on ssDNA.\nAuthors: Hengel, S. R.; Oppenheimer, K.; Smith, C.; Schaich, M.; Rein, H. L.; Martino, J.; Darrah, K.; Ezekwenna, O. C.; Burton, K. R.; Van Houten, B.; Spies, M.; Bernstein, K. A.\nScore: 18.6, Published: 2024-02-15 DOI: 10.1101/2024.02.14.580393\nTemplated DNA repair that occurs during homologous recombination and replication stress relies on RAD51. RAD51 activity is positively regulated by BRCA2 and the RAD51 paralogs. The Shu complex is a RAD51 paralog-containing complex consisting of SWSAP1 and SWS1. We demonstrate that SWSAP1-SWS1 binds RAD51, maintains RAD51 filament stability, and enables strand exchange. Using single molecule confocal fluorescence microscopy combined with optical tweezers, we show that SWSAP1-SWS1 decorates RAD51 filaments proficient for homologous recombination. We also find SWSAP1-SWS1 enhances RPA diffusion on ssDNA. Importantly, we show human sgSWSAP1 and sgSWS1 knockout cells are sensitive to pharmacological inhibition of PARP and APE1. Lastly, we identify cancer variants in SWSAP1 that alter SWS1 complex formation. Together, we show that SWSAP1-SWS1 stimulates RAD51- dependent high-fidelity repair and may be an important new cancer therapeutic target.\nSTREAMLINED PROTEOME-WIDE IDENTIFICATION OF DRUG TARGETS INDICATES ORGAN-SPECIFIC ENGAGEMENT\nAuthors: Batth, T. S.; Locard-Paulet, M.; Doncheva, N. T.; Lopez-Mendez, B.; Jensen, L. J.; Olsen, J. V.\nScore: 15.7, Published: 2024-02-08 DOI: 10.1101/2024.02.08.578880\nProteins are the primary targets of almost all small molecule drugs. However, even the most selectively designed drugs can potentially target several unknown proteins. Identification of potential drug targets can facilitate design of new drugs and repurposing of existing ones. Current state-of-the-art proteomics methodologies enable screening of thousands of proteins against a limited number of drug molecules. Here we report the development of a label-free quantitative proteomics approach that enables proteome-wide screening of small organic molecules in a scalable, reproducible, and rapid manner by streamlining the proteome integral solubility alteration (PISA) assay. We used rat organs ex-vivo to determine organ specific targets of medical drugs and enzyme inhibitors to identify novel drug targets for common drugs such as Ibuprofen. Finally, global drug profiling revealed overarching trends of how small molecules affect the proteome through either direct or indirect protein interactions.\nThe NAPstar family of NADP redox state sensors highlights glutathione as the primary mediator of anti-oxidative electron flux\nAuthors: Scherschel, M.; Niemeier, J.-O.; Jacobs, L.; Hoffmann, M.; Diederich, A.; Bell, C.; Hoehne, P.; Raetz, S.; Kroll, J.; Steinbeck, J.; Lichtenauer, S.; Multhoff, J.; Zimmermann, J.; Sadhanasatish, T.; Rothemann, R. A.; Grashoff, C.; Messens, J.; Ampofo, E.; Laschke, M. W.; Riemer, J.; Roma, L. P.; Schwarzlander, M.; Morgan, B.\nScore: 9.3, Published: 2024-02-15 DOI: 10.1101/2024.02.14.580349\nThe NADPH/NADP+ redox couple is central to metabolism and redox signalling. NADP redox state is differentially regulated by distinct enzymatic machineries at the subcellular compartment level. Nonetheless, a detailed understanding of subcellular NADP redox dynamics is limited by the availability of appropriate tools. Here, we introduce NAPstars, a family of genetically encoded, fluorescent protein-based NADP redox state biosensors. NAPstars offer real-time, specific, pH-resistant measurements, across a broad-range of NADP redox states, with subcellular resolution. We establish NAPstar measurements in yeast, plants and mammalian cell models, revealing a conserved robustness of cytosolic NADP redox homeostasis. NAPstars uncovered NADP redox oscillations linked to the cell cycle in yeast and illumination- and hypoxia-dependent NADP redox changes in plant leaves. By selectively impairing the glutathione and thioredoxin anti-oxidative pathways under acute oxidative challenge, NAPstars demonstrated an unexpected role for the glutathione system as the primary mediator of anti-oxidative electron flux that is conserved across eukaryotic kingdoms.\nExpanding the molecular grammar of polar residues and arginine in FUS prion-like domain phase separation and aggregation\nAuthors: Wake, N.; Weng, S.-L.; Zheng, T.; Wang, S.-H.; Kirilenko, V.; Mittal, J.; Fawzi, N. L.\nScore: 9.2, Published: 2024-02-15 DOI: 10.1101/2024.02.15.580391\nA molecular grammar governing low-complexity prion-like domains phase separation (PS) has been proposed based on mutagenesis experiments that identified tyrosine and arginine as primary drivers of phase separation via aromatic-aromatic and aromatic-arginine interactions. Here we show that additional residues make direct favorable contacts that contribute to phase separation, highlighting the need to account for these contributions in PS theories and models. We find that tyrosine and arginine make important contacts beyond only tyrosine-tyrosine and tyrosine-arginine, including arginine-arginine contacts. Among polar residues, glutamine in particular contributes to phase separation with sequence/position-specificity, making contacts with both tyrosine and arginine as well as other residues, both before phase separation and in condensed phases. For glycine, its flexibility, not its small solvation volume, favors phase separation by allowing favorable contacts between other residues and inhibits the liquid-to-solid (LST) transition. Polar residue types also make sequence-specific contributions to aggregation that go beyond simple rules, which for serine positions is linked to formation of an amyloid-core structure by the FUS low-complexity domain. Hence, here we propose a revised molecular grammar expanding the role of arginine and polar residues in prion-like domain protein phase separation and aggregation.\nStructural characterization of the ACDC domain from ApiAP2 proteins of the malaria parasite\nAuthors: Le Berre, M.; Tubiana, T.; Reutersward Waldner, P.; Lazar, N.; Gallay Li de la Sierra, I.; Santos, J. M.; Llinas, M.; Nessler, S.\nScore: 6.7, Published: 2024-02-10 DOI: 10.1101/2024.02.09.579679\nThe Apicomplexan AP2 (ApiAP2) proteins are the best characterized family of DNA-binding proteins in the malaria parasite. Apart from the AP2 DNA-binding domain, there is little sequence similarity between ApiAP2 proteins and no other functional domains have been extensively characterized. One protein domain, which is present in a subset of the ApiAP2 proteins, is the conserved AP2-coincident domain mostly at the C-terminus (ACDC domain). Here we solved for the first time the crystal structure of the ACDC domain from two distinct Plasmodium falciparum ApiAP2 proteins and one orthologue from P. vivax, revealing a non-canonical four-helix bundle. Despite little sequence conservation between the ACDC domains from the two proteins, the structures are remarkably similar and do not resemble that of any other known protein domains. Due to their unique protein architecture and lack of homologues in the human genome, we performed in silico docking calculations against a library of known antimalarial compounds and we identified a small molecule that can potentially bind to any Apicomplexan ACDC domain within a pocket highly conserved amongst ApiAP2 proteins. Inhibitors based on this compound would disrupt the function of the ACDC domain and thus of the ApiAP2 proteins containing it, providing a new therapeutic window for targeting the malaria parasite and other Apicomplexans.\nSARS-CoV-2 Omicron lineage XBB spike structures, conformations, antigenicity, and receptor recognition\nAuthors: Zhang, Q. E.; Lindenberger, J.; Parsons, R.; Thakur, B.; Parks, R.; Park, C. S.; Huang, X.; Sammour, S.; Janowska, K.; Spence, T. N.; Edwards, R. J.; Martin, M.; Williams, W. B.; Gobeil, S.; Montefiori, D. C.; Korber, B.; Saunders, K. O.; Haynes, B. F.; Haynes, B. F.; Henderson, R.; Acharya, P.\nScore: 5.8, Published: 2024-02-13 DOI: 10.1101/2024.02.12.580004\nA recombinant lineage of the SARS-CoV-2 Omicron variant, named XBB, appeared in late 2022 and evolved descendants that successively swept local and global populations. XBB lineage members were noted for their improved immune evasion and transmissibility. Here, we determine cryo-EM structures of XBB.1.5, XBB.1.16 and EG.5 spike (S) ectodomains to reveal enhanced occupancy of the receptor inaccessible closed state. Interprotomer receptor binding domain (RBD) interactions previously observed in BA.1 and BA.2 were retained to reinforce the 3-RBD-down state. Improved stability of XBB.1.5 and XBB.1.16 RBD compensated for loss of stability caused by early Omicron mutations, while the F456L substitution reduced EG.5 RBD stability. Long-range impacts of S1 subunit mutations affected conformation and epitope presentation in the S2 subunit. Taken together, our results feature a theme of iterative optimization of S protein stability as Omicron continues to evolve, while maintaining high affinity receptor binding and bolstering immune evasion.\nSpatial top-down proteomics for the functional characterization of human kidney\nAuthors: Zemaitis, K. J.; Fulcher, J. M.; Kumar, R.; Degnan, D. J.; Lewis, L. A.; Liao, Y.-C.; Velickovic, M.; Williams, S. M.; Moore, R. J.; Bramer, L. M.; Velickovic, D.; Zhu, Y.; Zhou, M.; Pasa-Tolic, L.\nScore: 5.8, Published: 2024-02-13 DOI: 10.1101/2024.02.13.580062\nBackgroundThe Human Proteome Project has credibly detected nearly 93% of the roughly 20,000 proteins which are predicted by the human genome. However, the proteome is enigmatic, where alterations in amino acid sequences from polymorphisms and alternative splicing, errors in translation, and post-translational modifications result in a proteome depth estimated at several million unique proteoforms. Recently mass spectrometry has been demonstrated in several landmark efforts mapping the human proteoform landscape in bulk analyses. Herein, we developed an integrated workflow for characterizing proteoforms from human tissue in a spatially resolved manner by coupling laser capture microdissection, nanoliter-scale sample preparation, and mass spectrometry imaging. ResultsUsing healthy human kidney sections as the case study, we focused our analyses on the major functional tissue units including glomeruli, tubules, and medullary rays. After laser capture microdissection, these isolated functional tissue units were processed with microPOTS (microdroplet processing in one-pot for trace samples) for sensitive top-down proteomics measurement. This provided a quantitative database of 616 proteoforms that was further leveraged as a library for mass spectrometry imaging with near-cellular spatial resolution over the entire section. Notably, several mitochondrial proteoforms were found to be differentially abundant between glomeruli and convoluted tubules, and further spatial contextualization was provided by mass spectrometry imaging confirming unique differences identified by microPOTS, and further expanding the field-of-view for unique distributions such as enhanced abundance of a truncated form (1-74) of ubiquitin within cortical regions. ConclusionsWe developed an integrated workflow to directly identify proteoforms and reveal their spatial distributions. Where of the 20 differentially abundant proteoforms identified as discriminate between tubules and glomeruli by microPOTS, the vast majority of tubular proteoforms were of mitochondrial origin (8 of 10) where discriminate proteoforms in glomeruli were primarily hemoglobin subunits (9 of 10). These trends were also identified within ion images demonstrating spatially resolved characterization of proteoforms that has the potential to reshape discovery-based proteomics because the proteoforms are the ultimate effector of cellular functions. Applications of this technology have the potential to unravel etiology and pathophysiology of disease states, informing on biologically active proteoforms, which remodel the proteomic landscape in chronic and acute disorders.\nMicromolar fluoride contamination arising from glass NMR tubes and a simple solution for biomolecular applications\nAuthors: Matwani, K.; Cornish, J.; DeBenedictis, E. A.; Heller, G. T.\nScore: 5.2, Published: 2024-02-12 DOI: 10.1101/2024.02.12.579991\nFluorine (19F) NMR is emerging as an invaluable analytical technique in chemistry, biochemistry, material science, and medicine, especially due to the inherent rarity of naturally occurring fluorine in biological, organic, and inorganic compounds. Thus, we were surprised to identify an unexpected peak in our 19F NMR spectra, corresponding to free fluoride, which appears to leach out from various types of new and unused glass NMR tubes over the course of several hours. We quantified this contaminant to be at micromolar concentrations for typical NMR sample volumes across multiple glass types and brands. We find that this artefact is undetectable for samples prepared in quartz NMR tubes within the timeframes of our experiments. We also observed that pre-soaking new glass NMR tubes combined with rinsing removes this contamination below micromolar levels. Given the increasing popularity of 19F NMR across a wide range of fields, the long collection times required for relaxation studies and samples of low concentrations, and the importance of avoiding contamination in all NMR experiments, we anticipate that our simple solution will be useful to biomolecular NMR spectroscopists.\nStructural insights into CXCR4 modulation and oligomerization\nAuthors: Saotome, K.; McGoldrick, L. L.; Ho, J.-H.; Ramlall, T. F.; Shah, S.; Moore, M. J.; Kim, J. H.; Leidich, R.; Olson, W. C.; Franklin, M. C.\nScore: 4.8, Published: 2024-02-10 DOI: 10.1101/2024.02.09.579708\nActivation of the chemokine receptor CXCR4 by its chemokine ligand CXCL12 regulates diverse cellular processes. CXCR4 also serves as a key target for diseases such as cancer and HIV. Previously reported crystal structures of CXCR4 bound to antagonists revealed the architecture of an inactive, homodimeric receptor. However, many structural aspects of CXCR4 remain poorly understood, including its activation by CXCL12, as well as its assembly into higher-order oligomers. Here, we use cryoelectron microscopy (cryoEM) to investigate various modes of CXCR4 regulation in the presence and absence of Gi protein. CXCL12 activates CXCR4 by inserting its N-terminus deep into the CXCR4 orthosteric pocket. The binding of FDA-approved antagonist AMD3100 is stabilized by electrostatic interactions with acidic residues in the 7 transmembrane helix bundle. A potent antibody blocker, REGN7663, binds across the extracellular face of CXCR4 and inserts its CDR-H3 loop into the orthosteric pocket. Trimeric and tetrameric structures of CXCR4 reveal, to our knowledge, previously undescribed modes of GPCR oligomerization. Remarkably, CXCR4 adopts distinct subunit conformations in trimeric and tetrameric assemblies, highlighting how oligomerization could allosterically regulate chemokine receptor function.\n",
  "wordCount" : "2305",
  "inLanguage": "en",
  "datePublished": "2024-02-18T10:38:00Z",
  "dateModified": "2024-02-18T10:38:00Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 18, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579730">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579730" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579730">
        <p class="paperTitle">Structural basis for the inhibition of PRC2 by active transcription histone posttranslational modifications</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579730" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579730" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cookis, T.; Lydecker, A.; Sauer, P. V.; Kasinath, V.; Nogales, E.</p>
        <p class="info">Score: 25.2, Published: 2024-02-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579730' target='https://doi.org/10.1101/2024.02.09.579730'> 10.1101/2024.02.09.579730</a></p>
        <p class="abstract">Polycomb repressive complex 2 (PRC2) is an epigenetic regulator essential for embryonic development and maintenance of cell identity that trimethylates histone H3 at lysine 27 (H3K27me3) leading to gene silencing. PRC2 is regulated by association with protein cofactors and crosstalk with histone posttranslational modifications. Trimethylated histone H3 K4 (H3K4me3) and K36 (H3K36me3) localize to sites of active transcription where H3K27me3 is absent and inhibit PRC2 activity through unknown mechanisms. Using cryo-electron microscopy we reveal that histone H3 tails modified with H3K36me3 engage poorly with the PRC2 active site and preclude its effective interaction with chromatin, while the H3K4me3 modification binds to the allosteric site in the EED subunit, acting as an antagonist that competes with allosteric activators required for the spreading of the H3K27me3 repressive mark. Thus, the location along the H3 tail of the H3K4me3 and H3K36me3 modifications allow them to target two essential requirements for efficient trimethylation of histone H3K27. We further show that the JARID2 cofactor modulates PRC2 activity in the presence of these histone modifications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.14.580393">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.14.580393" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.14.580393">
        <p class="paperTitle">The human Shu complex promotes RAD51 activity by modulating RPA dynamics on ssDNA.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.14.580393" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.14.580393" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hengel, S. R.; Oppenheimer, K.; Smith, C.; Schaich, M.; Rein, H. L.; Martino, J.; Darrah, K.; Ezekwenna, O. C.; Burton, K. R.; Van Houten, B.; Spies, M.; Bernstein, K. A.</p>
        <p class="info">Score: 18.6, Published: 2024-02-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.14.580393' target='https://doi.org/10.1101/2024.02.14.580393'> 10.1101/2024.02.14.580393</a></p>
        <p class="abstract">Templated DNA repair that occurs during homologous recombination and replication stress relies on RAD51. RAD51 activity is positively regulated by BRCA2 and the RAD51 paralogs. The Shu complex is a RAD51 paralog-containing complex consisting of SWSAP1 and SWS1. We demonstrate that SWSAP1-SWS1 binds RAD51, maintains RAD51 filament stability, and enables strand exchange. Using single molecule confocal fluorescence microscopy combined with optical tweezers, we show that SWSAP1-SWS1 decorates RAD51 filaments proficient for homologous recombination. We also find SWSAP1-SWS1 enhances RPA diffusion on ssDNA. Importantly, we show human sgSWSAP1 and sgSWS1 knockout cells are sensitive to pharmacological inhibition of PARP and APE1. Lastly, we identify cancer variants in SWSAP1 that alter SWS1 complex formation. Together, we show that SWSAP1-SWS1 stimulates RAD51- dependent high-fidelity repair and may be an important new cancer therapeutic target.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.08.578880">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.08.578880" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.08.578880">
        <p class="paperTitle">STREAMLINED PROTEOME-WIDE IDENTIFICATION OF DRUG TARGETS INDICATES ORGAN-SPECIFIC ENGAGEMENT</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.08.578880" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.08.578880" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Batth, T. S.; Locard-Paulet, M.; Doncheva, N. T.; Lopez-Mendez, B.; Jensen, L. J.; Olsen, J. V.</p>
        <p class="info">Score: 15.7, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.08.578880' target='https://doi.org/10.1101/2024.02.08.578880'> 10.1101/2024.02.08.578880</a></p>
        <p class="abstract">Proteins are the primary targets of almost all small molecule drugs. However, even the most selectively designed drugs can potentially target several unknown proteins. Identification of potential drug targets can facilitate design of new drugs and repurposing of existing ones. Current state-of-the-art proteomics methodologies enable screening of thousands of proteins against a limited number of drug molecules. Here we report the development of a label-free quantitative proteomics approach that enables proteome-wide screening of small organic molecules in a scalable, reproducible, and rapid manner by streamlining the proteome integral solubility alteration (PISA) assay. We used rat organs ex-vivo to determine organ specific targets of medical drugs and enzyme inhibitors to identify novel drug targets for common drugs such as Ibuprofen. Finally, global drug profiling revealed overarching trends of how small molecules affect the proteome through either direct or indirect protein interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.14.580349">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.14.580349" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.14.580349">
        <p class="paperTitle">The NAPstar family of NADP redox state sensors highlights glutathione as the primary mediator of anti-oxidative electron flux</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.14.580349" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.14.580349" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Scherschel, M.; Niemeier, J.-O.; Jacobs, L.; Hoffmann, M.; Diederich, A.; Bell, C.; Hoehne, P.; Raetz, S.; Kroll, J.; Steinbeck, J.; Lichtenauer, S.; Multhoff, J.; Zimmermann, J.; Sadhanasatish, T.; Rothemann, R. A.; Grashoff, C.; Messens, J.; Ampofo, E.; Laschke, M. W.; Riemer, J.; Roma, L. P.; Schwarzlander, M.; Morgan, B.</p>
        <p class="info">Score: 9.3, Published: 2024-02-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.14.580349' target='https://doi.org/10.1101/2024.02.14.580349'> 10.1101/2024.02.14.580349</a></p>
        <p class="abstract">The NADPH/NADP&#43; redox couple is central to metabolism and redox signalling. NADP redox state is differentially regulated by distinct enzymatic machineries at the subcellular compartment level. Nonetheless, a detailed understanding of subcellular NADP redox dynamics is limited by the availability of appropriate tools. Here, we introduce NAPstars, a family of genetically encoded, fluorescent protein-based NADP redox state biosensors. NAPstars offer real-time, specific, pH-resistant measurements, across a broad-range of NADP redox states, with subcellular resolution. We establish NAPstar measurements in yeast, plants and mammalian cell models, revealing a conserved robustness of cytosolic NADP redox homeostasis. NAPstars uncovered NADP redox oscillations linked to the cell cycle in yeast and illumination- and hypoxia-dependent NADP redox changes in plant leaves. By selectively impairing the glutathione and thioredoxin anti-oxidative pathways under acute oxidative challenge, NAPstars demonstrated an unexpected role for the glutathione system as the primary mediator of anti-oxidative electron flux that is conserved across eukaryotic kingdoms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.15.580391">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.15.580391" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.15.580391">
        <p class="paperTitle">Expanding the molecular grammar of polar residues and arginine in FUS prion-like domain phase separation and aggregation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.15.580391" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.15.580391" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wake, N.; Weng, S.-L.; Zheng, T.; Wang, S.-H.; Kirilenko, V.; Mittal, J.; Fawzi, N. L.</p>
        <p class="info">Score: 9.2, Published: 2024-02-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.15.580391' target='https://doi.org/10.1101/2024.02.15.580391'> 10.1101/2024.02.15.580391</a></p>
        <p class="abstract">A molecular grammar governing low-complexity prion-like domains phase separation (PS) has been proposed based on mutagenesis experiments that identified tyrosine and arginine as primary drivers of phase separation via aromatic-aromatic and aromatic-arginine interactions. Here we show that additional residues make direct favorable contacts that contribute to phase separation, highlighting the need to account for these contributions in PS theories and models. We find that tyrosine and arginine make important contacts beyond only tyrosine-tyrosine and tyrosine-arginine, including arginine-arginine contacts. Among polar residues, glutamine in particular contributes to phase separation with sequence/position-specificity, making contacts with both tyrosine and arginine as well as other residues, both before phase separation and in condensed phases. For glycine, its flexibility, not its small solvation volume, favors phase separation by allowing favorable contacts between other residues and inhibits the liquid-to-solid (LST) transition. Polar residue types also make sequence-specific contributions to aggregation that go beyond simple rules, which for serine positions is linked to formation of an amyloid-core structure by the FUS low-complexity domain. Hence, here we propose a revised molecular grammar expanding the role of arginine and polar residues in prion-like domain protein phase separation and aggregation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579679">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579679" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579679">
        <p class="paperTitle">Structural characterization of the ACDC domain from ApiAP2 proteins of the malaria parasite</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579679" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579679" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Le Berre, M.; Tubiana, T.; Reutersward Waldner, P.; Lazar, N.; Gallay Li de la Sierra, I.; Santos, J. M.; Llinas, M.; Nessler, S.</p>
        <p class="info">Score: 6.7, Published: 2024-02-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579679' target='https://doi.org/10.1101/2024.02.09.579679'> 10.1101/2024.02.09.579679</a></p>
        <p class="abstract">The Apicomplexan AP2 (ApiAP2) proteins are the best characterized family of DNA-binding proteins in the malaria parasite. Apart from the AP2 DNA-binding domain, there is little sequence similarity between ApiAP2 proteins and no other functional domains have been extensively characterized. One protein domain, which is present in a subset of the ApiAP2 proteins, is the conserved AP2-coincident domain mostly at the C-terminus (ACDC domain). Here we solved for the first time the crystal structure of the ACDC domain from two distinct Plasmodium falciparum ApiAP2 proteins and one orthologue from P. vivax, revealing a non-canonical four-helix bundle. Despite little sequence conservation between the ACDC domains from the two proteins, the structures are remarkably similar and do not resemble that of any other known protein domains. Due to their unique protein architecture and lack of homologues in the human genome, we performed in silico docking calculations against a library of known antimalarial compounds and we identified a small molecule that can potentially bind to any Apicomplexan ACDC domain within a pocket highly conserved amongst ApiAP2 proteins. Inhibitors based on this compound would disrupt the function of the ACDC domain and thus of the ApiAP2 proteins containing it, providing a new therapeutic window for targeting the malaria parasite and other Apicomplexans.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.12.580004">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.12.580004" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.12.580004">
        <p class="paperTitle">SARS-CoV-2 Omicron lineage XBB spike structures, conformations, antigenicity, and receptor recognition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.12.580004" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.12.580004" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, Q. E.; Lindenberger, J.; Parsons, R.; Thakur, B.; Parks, R.; Park, C. S.; Huang, X.; Sammour, S.; Janowska, K.; Spence, T. N.; Edwards, R. J.; Martin, M.; Williams, W. B.; Gobeil, S.; Montefiori, D. C.; Korber, B.; Saunders, K. O.; Haynes, B. F.; Haynes, B. F.; Henderson, R.; Acharya, P.</p>
        <p class="info">Score: 5.8, Published: 2024-02-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.12.580004' target='https://doi.org/10.1101/2024.02.12.580004'> 10.1101/2024.02.12.580004</a></p>
        <p class="abstract">A recombinant lineage of the SARS-CoV-2 Omicron variant, named XBB, appeared in late 2022 and evolved descendants that successively swept local and global populations. XBB lineage members were noted for their improved immune evasion and transmissibility. Here, we determine cryo-EM structures of XBB.1.5, XBB.1.16 and EG.5 spike (S) ectodomains to reveal enhanced occupancy of the receptor inaccessible closed state. Interprotomer receptor binding domain (RBD) interactions previously observed in BA.1 and BA.2 were retained to reinforce the 3-RBD-down state. Improved stability of XBB.1.5 and XBB.1.16 RBD compensated for loss of stability caused by early Omicron mutations, while the F456L substitution reduced EG.5 RBD stability. Long-range impacts of S1 subunit mutations affected conformation and epitope presentation in the S2 subunit. Taken together, our results feature a theme of iterative optimization of S protein stability as Omicron continues to evolve, while maintaining high affinity receptor binding and bolstering immune evasion.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.13.580062">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.13.580062" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.13.580062">
        <p class="paperTitle">Spatial top-down proteomics for the functional characterization of human kidney</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.13.580062" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.13.580062" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zemaitis, K. J.; Fulcher, J. M.; Kumar, R.; Degnan, D. J.; Lewis, L. A.; Liao, Y.-C.; Velickovic, M.; Williams, S. M.; Moore, R. J.; Bramer, L. M.; Velickovic, D.; Zhu, Y.; Zhou, M.; Pasa-Tolic, L.</p>
        <p class="info">Score: 5.8, Published: 2024-02-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.13.580062' target='https://doi.org/10.1101/2024.02.13.580062'> 10.1101/2024.02.13.580062</a></p>
        <p class="abstract">BackgroundThe Human Proteome Project has credibly detected nearly 93% of the roughly 20,000 proteins which are predicted by the human genome. However, the proteome is enigmatic, where alterations in amino acid sequences from polymorphisms and alternative splicing, errors in translation, and post-translational modifications result in a proteome depth estimated at several million unique proteoforms. Recently mass spectrometry has been demonstrated in several landmark efforts mapping the human proteoform landscape in bulk analyses. Herein, we developed an integrated workflow for characterizing proteoforms from human tissue in a spatially resolved manner by coupling laser capture microdissection, nanoliter-scale sample preparation, and mass spectrometry imaging.

ResultsUsing healthy human kidney sections as the case study, we focused our analyses on the major functional tissue units including glomeruli, tubules, and medullary rays. After laser capture microdissection, these isolated functional tissue units were processed with microPOTS (microdroplet processing in one-pot for trace samples) for sensitive top-down proteomics measurement. This provided a quantitative database of 616 proteoforms that was further leveraged as a library for mass spectrometry imaging with near-cellular spatial resolution over the entire section. Notably, several mitochondrial proteoforms were found to be differentially abundant between glomeruli and convoluted tubules, and further spatial contextualization was provided by mass spectrometry imaging confirming unique differences identified by microPOTS, and further expanding the field-of-view for unique distributions such as enhanced abundance of a truncated form (1-74) of ubiquitin within cortical regions.

ConclusionsWe developed an integrated workflow to directly identify proteoforms and reveal their spatial distributions. Where of the 20 differentially abundant proteoforms identified as discriminate between tubules and glomeruli by microPOTS, the vast majority of tubular proteoforms were of mitochondrial origin (8 of 10) where discriminate proteoforms in glomeruli were primarily hemoglobin subunits (9 of 10). These trends were also identified within ion images demonstrating spatially resolved characterization of proteoforms that has the potential to reshape discovery-based proteomics because the proteoforms are the ultimate effector of cellular functions. Applications of this technology have the potential to unravel etiology and pathophysiology of disease states, informing on biologically active proteoforms, which remodel the proteomic landscape in chronic and acute disorders.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.12.579991">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.12.579991" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.12.579991">
        <p class="paperTitle">Micromolar fluoride contamination arising from glass NMR tubes and a simple solution for biomolecular applications</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.12.579991" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.12.579991" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Matwani, K.; Cornish, J.; DeBenedictis, E. A.; Heller, G. T.</p>
        <p class="info">Score: 5.2, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.12.579991' target='https://doi.org/10.1101/2024.02.12.579991'> 10.1101/2024.02.12.579991</a></p>
        <p class="abstract">Fluorine (19F) NMR is emerging as an invaluable analytical technique in chemistry, biochemistry, material science, and medicine, especially due to the inherent rarity of naturally occurring fluorine in biological, organic, and inorganic compounds. Thus, we were surprised to identify an unexpected peak in our 19F NMR spectra, corresponding to free fluoride, which appears to leach out from various types of new and unused glass NMR tubes over the course of several hours. We quantified this contaminant to be at micromolar concentrations for typical NMR sample volumes across multiple glass types and brands. We find that this artefact is undetectable for samples prepared in quartz NMR tubes within the timeframes of our experiments. We also observed that pre-soaking new glass NMR tubes combined with rinsing removes this contamination below micromolar levels. Given the increasing popularity of 19F NMR across a wide range of fields, the long collection times required for relaxation studies and samples of low concentrations, and the importance of avoiding contamination in all NMR experiments, we anticipate that our simple solution will be useful to biomolecular NMR spectroscopists.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579708">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579708" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579708">
        <p class="paperTitle">Structural insights into CXCR4 modulation and oligomerization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579708" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579708" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Saotome, K.; McGoldrick, L. L.; Ho, J.-H.; Ramlall, T. F.; Shah, S.; Moore, M. J.; Kim, J. H.; Leidich, R.; Olson, W. C.; Franklin, M. C.</p>
        <p class="info">Score: 4.8, Published: 2024-02-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579708' target='https://doi.org/10.1101/2024.02.09.579708'> 10.1101/2024.02.09.579708</a></p>
        <p class="abstract">Activation of the chemokine receptor CXCR4 by its chemokine ligand CXCL12 regulates diverse cellular processes. CXCR4 also serves as a key target for diseases such as cancer and HIV. Previously reported crystal structures of CXCR4 bound to antagonists revealed the architecture of an inactive, homodimeric receptor. However, many structural aspects of CXCR4 remain poorly understood, including its activation by CXCL12, as well as its assembly into higher-order oligomers. Here, we use cryoelectron microscopy (cryoEM) to investigate various modes of CXCR4 regulation in the presence and absence of Gi protein. CXCL12 activates CXCR4 by inserting its N-terminus deep into the CXCR4 orthosteric pocket. The binding of FDA-approved antagonist AMD3100 is stabilized by electrostatic interactions with acidic residues in the 7 transmembrane helix bundle. A potent antibody blocker, REGN7663, binds across the extracellular face of CXCR4 and inserts its CDR-H3 loop into the orthosteric pocket. Trimeric and tetrameric structures of CXCR4 reveal, to our knowledge, previously undescribed modes of GPCR oligomerization. Remarkably, CXCR4 adopts distinct subunit conformations in trimeric and tetrameric assemblies, highlighting how oligomerization could allosterically regulate chemokine receptor function.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
