<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Structure and design of Langya virus glycoprotein antigens
Authors: Wang, Z.; McCallum, M.; Yan, L.; Sharkey, W.; Park, Y.-J.; Dang, H. V.; Amaya, M.; Person, A.; Broder, C. C.; Veesler, D. J.
Score: 31.2, Published: 2023-08-26 DOI: 10.1101/2023.08.20.554025
Langya virus (LayV) is a recently discovered henipavirus (HNV), isolated from febrile patients in China, harboring a genome distantly related to that of Nipah virus (NiV) and Hendra virus (HeV). HNV entry into host cells is mediated by the attachment (G) and fusion (F) glycoproteins which are the main targets of neutralizing antibodies.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Structure and design of Langya virus glycoprotein antigens
Authors: Wang, Z.; McCallum, M.; Yan, L.; Sharkey, W.; Park, Y.-J.; Dang, H. V.; Amaya, M.; Person, A.; Broder, C. C.; Veesler, D. J.
Score: 31.2, Published: 2023-08-26 DOI: 10.1101/2023.08.20.554025
Langya virus (LayV) is a recently discovered henipavirus (HNV), isolated from febrile patients in China, harboring a genome distantly related to that of Nipah virus (NiV) and Hendra virus (HeV). HNV entry into host cells is mediated by the attachment (G) and fusion (F) glycoproteins which are the main targets of neutralizing antibodies." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-27T10:37:02+00:00" />
<meta property="article:modified_time" content="2023-08-27T10:37:02+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Structure and design of Langya virus glycoprotein antigens
Authors: Wang, Z.; McCallum, M.; Yan, L.; Sharkey, W.; Park, Y.-J.; Dang, H. V.; Amaya, M.; Person, A.; Broder, C. C.; Veesler, D. J.
Score: 31.2, Published: 2023-08-26 DOI: 10.1101/2023.08.20.554025
Langya virus (LayV) is a recently discovered henipavirus (HNV), isolated from febrile patients in China, harboring a genome distantly related to that of Nipah virus (NiV) and Hendra virus (HeV). HNV entry into host cells is mediated by the attachment (G) and fusion (F) glycoproteins which are the main targets of neutralizing antibodies."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Structure and design of Langya virus glycoprotein antigens\nAuthors: Wang, Z.; McCallum, M.; Yan, L.; Sharkey, W.; Park, Y.-J.; Dang, H. V.; Amaya, M.; Person, A.; Broder, C. C.; Veesler, D. J.\nScore: 31.2, Published: 2023-08-26 DOI: 10.1101/2023.08.20.554025\nLangya virus (LayV) is a recently discovered henipavirus (HNV), isolated from febrile patients in China, harboring a genome distantly related to that of Nipah virus (NiV) and Hendra virus (HeV). HNV entry into host cells is mediated by the attachment (G) and fusion (F) glycoproteins which are the main targets of neutralizing antibodies.",
  "keywords": [
    
  ],
  "articleBody": " Structure and design of Langya virus glycoprotein antigens\nAuthors: Wang, Z.; McCallum, M.; Yan, L.; Sharkey, W.; Park, Y.-J.; Dang, H. V.; Amaya, M.; Person, A.; Broder, C. C.; Veesler, D. J.\nScore: 31.2, Published: 2023-08-26 DOI: 10.1101/2023.08.20.554025\nLangya virus (LayV) is a recently discovered henipavirus (HNV), isolated from febrile patients in China, harboring a genome distantly related to that of Nipah virus (NiV) and Hendra virus (HeV). HNV entry into host cells is mediated by the attachment (G) and fusion (F) glycoproteins which are the main targets of neutralizing antibodies. We show here that the LayV F and G glycoproteins promote membrane fusion with human, mouse and hamster target cells using a different, yet unknown, receptor than NiV and HeV and that NiV- and HeV-elicited monoclonal and polyclonal antibodies do not cross-react with LayV F and G. We determined cryo-electron microscopy structures of LayV F, in the prefusion and postfusion states, and of LayV G, revealing previously unknown conformational landscapes and explaining their distinct antigenicity relative to NiV and HeV. We computationally designed stabilized LayV G constructs and demonstrate the generalizability of an HNV F prefusion-stabilization strategy which will expedite vaccine development efforts. Our data will support the development of vaccines and therapeutics against LayV and closely related HNVs, such as the monoclonal antibodies characterized here.\nMicroglia mediate contact-independent neuronal pruning via secreted Neuraminidase-3 associated with extracellular vesicles\nAuthors: Delaveris, C. S.; Wang, C. L.; Riley, N. M.; Li, S.; Kulkarni, R. U.; Bertozzi, C. R.\nScore: 17.0, Published: 2023-08-22 DOI: 10.1101/2023.08.21.554214\nNeurons communicate with each other through electrochemical transmission at synapses. Microglia, the resident immune cells of the central nervous system, can prune these synapses through a variety of contact-dependent and -independent means. Microglial secretion of active sialidase enzymes upon exposure to inflammatory stimuli is one unexplored mechanism of pruning. Recent work from our lab showed that treatment of neurons with bacterial sialidases disrupts neuronal network connectivity. Here, we find that activated microglia secrete Neuraminidase-3 (Neu3) associated with fusogenic extracellular vesicles. Furthermore, we show Neu3 mediates contact-independent pruning of neurons and subsequent disruption of neuronal networks through neuronal glycocalyx remodeling. We observe that NEU3 is transcriptionally upregulated upon exposure to inflammatory stimuli, and that a genetic knock-out of NEU3 abrogates the sialidase activity of inflammatory microglial secretions. Moreover, we demonstrate that Neu3 is associated with a subpopulation of extracellular vesicles, possibly exosomes, that are secreted by microglia upon inflammatory insult. Finally, we demonstrate that Neu3 is both necessary and sufficient to both desialylate neurons and decrease neuronal network connectivity. These results implicate Neu3 in remodeling of the glycocalyx leading to aberrant network-level activity of neurons, with implications in neuroinflammatory diseases such as Parkinsons disease and Alzheimers disease. Graphical AbstractNeuroinflammation induces secretion of the sialidase Neu3 via extracellular vesicles from microglia that prune neuronal synapses and disrupt neuronal communication. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=76 SRC=\"FIGDIR/small/554214v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (25K): org.highwire.dtl.DTLVardef@1c5a5aaorg.highwire.dtl.DTLVardef@ff5a8eorg.highwire.dtl.DTLVardef@9237b6org.highwire.dtl.DTLVardef@187a6a3_HPS_FORMAT_FIGEXP M_FIG C_FIG\nDe novo design of buttressed loops for sculpting protein functions\nAuthors: Jiang, H.; Jude, K. M.; Wu, K.; Fallas, J.; Ueda, G.; Brunette, T.; Hicks, D.; Pyles, H.; Yang, A.; Carter, L.; Lamb, M.; Li, X.; Levine, P. M.; Stewart, L. J.; Garcia, K. C.; Baker, D.\nScore: 14.8, Published: 2023-08-23 DOI: 10.1101/2023.08.22.554384\nIn natural proteins, structured loops play central roles in molecular recognition, signal transduction and enzyme catalysis. However, because of the intrinsic flexibility and irregularity of loop regions, organizing multiple structured loops at protein functional sites has been very difficult to achieve by de novo protein design. Here we describe a solution to this problem that generates structured loops buttressed by extensive hydrogen bonding interactions with two neighboring loops and with secondary structure elements. We use this approach to design tandem repeat proteins with buttressed loops ranging from 9 to 14 residues in length. Experimental characterization shows the designs are folded and monodisperse, highly soluble, and thermally stable. Crystal structures are in close agreement with the computational design models, with the loops structured and buttressed by their neighbors as designed. We demonstrate the functionality afforded by loop buttressing by designing and characterizing binders for extended peptides in which the loops form one side of an extended binding pocket. The ability to design multiple structured loops should contribute quite generally to efforts to design new protein functions.\nAllosteric activation of the co-receptor BAK1 by the EFR receptor kinase initiates immune signaling\nAuthors: Mühlenbeck, H.; Tsutsui, Y.; Lemmon, M.; Bender, K. W.; Zipfel, C.\nScore: 14.7, Published: 2023-08-25 DOI: 10.1101/2023.08.23.554490\nTransmembrane signaling by plant receptor kinases (RKs) has long been thought to involve reciprocal trans-phosphorylation of their intracellular kinase domains. The fact that many of these are pseudokinase domains, however, suggests that additional mechanisms must govern RK signaling activation. Non-catalytic (pseudo)kinase signaling mechanisms have been described in metazoans, but information is scarce for plants. Recently, a non-catalytic function was reported for the leucine-rich repeat (LRR)-RK subfamily XIIa member EFR (ELONGATION FACTOR TU RECEPTOR) and phosphorylation-dependent conformational changes were proposed to regulate signaling of RKs with non-RD kinase domains. Here, using EFR as a model, we describe a non-catalytic activation mechanism for LRR-RKs with non-RD kinase domains. EFR is an active kinase, but a kinase-dead variant retains the ability to enhance catalytic activity of its co-receptor kinase BAK1/SERK3 (BRASSINOSTEROID INSENSITIVE 1-ASSOCIATED KINASE 1/SOMATIC EMBRYOGENESIS RECEPTOR KINASE 3). Applying hydrogen-deuterium exchange mass spectrometry (HDX-MS) analysis and designing homology-based intragenic suppressor mutations, we provide evidence that the EFR kinase domain must adopt its active conformation in order to activate BAK1 allosterically, likely by supporting C-helix positioning in BAK1. Our results suggest a conformational toggle model for signaling, in which BAK1 first phosphorylates EFR in the activation loop to stabilize its active conformation, allowing EFR in turn to allosterically activate BAK1.\nBlocking oligomerization is the most viable strategy to inhibit STING\nAuthors: Chan, R. J. M.; Cao, X.; Ergun, S. L.; Njomen, E.; Lynch, S. R.; Cravatt, B. F.; Li, L.\nScore: 16.0, Published: 2023-08-12 DOI: 10.1101/2023.08.11.553045\nAutoimmune disorders are frequently exacerbated by aberrant STING signaling. While STING inhibitors are highly sought after, no viable therapeutic strategies have been reported. When stimulated, STING undergoes a multi-step mechanism to activate downstream interferon-stimulated genes (ISGs), including palmitoylation, translocation from the ER to the Golgi, and oligomerization, but it remains unclear which step(s) are strictly required and therefore should be targeted by potential STING inhibitors. Here, we comprehensively established the order and determined the necessity of each STING activation step. We find that ER-to-Golgi translocation is dispensable for downstream ISG activation, and a previously studied palmitoylation site, C91, is not required for activation in all contexts. Cysteine reactivity mapping and metabolic labeling identified a new palmitoylation site, C64, that is basally palmitoylated and required for STING activation, but is not druggable due to poor solvent accessibility. We confirm that the oligomerization step of STING activation is strictly required, and therefore developed a short peptide mimicking its natural autoinhibitory domain that effectively inhibits STING activation: a proof of concept for the first viable strategy for developing STING inhibitors.\nBromodomain Factor 5 as a Target for Antileishmanial Drug Discovery\nAuthors: Russell, C. N.; Carter, J. L.; Borgia, J. M.; Bush, J.; Calderon, F.; Gabarro, R.; Conway, S. J.; Mottram, J. C.; Wilkinson, A. J.; Jones, N. G.\nScore: 11.1, Published: 2023-08-21 DOI: 10.1101/2023.08.21.554125\nLeishmaniases are a collection of neglected tropical diseases caused by kinetoplastid parasites in the genus Leishmania. Current chemotherapies are severely limited and the need for new antileishmanials is of pressing international importance. Bromodomains are epigenetic reader domains that have shown promising therapeutic potential for cancer therapy and may also present an attractive target to treat parasitic diseases. Here, we investigate Leishmania donovani bromodomain factor 5 (LdBDF5) as a target for antileishmanial drug discovery. LdBDF5 contains pair of bromodomains (BD5.1 and BD5.2) in an N-terminal tandem repeat. We purified recombinant bromodomains of L. donovani BDF5 and determined the structure of BD5.2 by X-ray crystallography. Using a histone peptide microarray and fluorescence polarisation assay, we identified binding interactions of LdBDF5 bromodomains with acetylated peptides derived from histones H2B and H4. In orthogonal biophysical assays including thermal shift assays, fluorescence polarisation and NMR, we showed that BDF5 bromodomains bind to human bromodomain inhibitors SGC-CBP30, bromosporine and I-BRD9, moreover, SGC-CBP30 exhibited activity against Leishmania promastigotes in cell viability assays. These findings exemplify the potential BDF5 holds as a drug target in Leishmania and provide a foundation for the future development of optimised antileishmanial compounds targeting this epigenetic reader protein. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=113 SRC=\"FIGDIR/small/554125v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (25K): org.highwire.dtl.DTLVardef@a6c345org.highwire.dtl.DTLVardef@5df8c8org.highwire.dtl.DTLVardef@1784d20org.highwire.dtl.DTLVardef@10fad26_HPS_FORMAT_FIGEXP M_FIG C_FIG\nCryo-EM of soft-landed β-galactosidase: Gas-phase and native structures are remarkably similar\nAuthors: Esser, T. K.; Boehning, J.; Onur, A.; Chinthapalli, D. K.; Eriksson, L.; Grabarics, M.; Fremdling, P.; Konijnenberg, A.; Makarov, A. A.; Botman, A.; Peter, C.; Benesch, J. L. P.; Robinson, C. V.; Gault, J.; Baker, L.; Bharat, T. A. M.; Rauschenbach, S.\nScore: 54.6, Published: 2023-08-17 DOI: 10.1101/2023.08.17.553673\nNative mass spectrometry (native MS) is a powerful technique that provides information on stoichiometry, interactions, homogeneity and shape of protein complexes. However, the extent of deviation between protein structures in the mass spectrometer and in solution remains a matter of debate. Here, we uncover the gas-phase structure of {beta}-galactosidase using single particle electron cryomicroscopy (cryo-EM) down to 2.6 [A] resolution, enabled by soft-landing of mass-selected protein complexes onto cold TEM grids and in-situ ice coating. We find that large parts of the secondary and tertiary structure are retained from solution, with dehydration-driven subunit reorientation leading to consistent compaction in the gas phase. Our work enables visualizing the structure of gas-phase protein complexes from numerous experimental scenarios at side-chain resolution and demonstrates the possibility of more controlled cryo-EM sample preparation. One Sentence SummaryElectrospray ion-beam deposition on cold grids and in-vacuum ice growth enable cryo-EM of mass-selected proteins at 2.6[A].\nEffects of phosphorylation on Drp1 activation by its receptors, actin, and cardiolipin\nAuthors: Liu, A.; Hatch, A. L.; Higgs, H.\nScore: 8.6, Published: 2023-08-20 DOI: 10.1101/2023.08.20.554022\nDrp1 is a dynamin family GTPase that is required for mitochondrial and peroxisomal division, in which it oligomerizes into a ring and constricts the underlying membrane in a GTP hydrolysis-dependent manner. Oligomerization increases Drp1 GTPase activity through interactions between neighboring GTPase domains. In cells, Drp1 is regulated by several factors including Drp1 receptors, actin filaments, cardiolipin, and phosphorylation at two sites: S579 and S600. Phosphorylation of S579 is widely regarded as activating, while S600 phosphorylation is commonly considered inhibiting. However, the direct effects of phosphorylation on Drp1 GTPase activity have not been investigated in detail. In this study, we compare the effects of S579 and S600 phosphorylation on purified Drp1, using phospho-mimetic mutants and in vitro phosphorylation. The oligomerization state of both phospho-mimetic mutants is shifted toward smaller oligomers. Both phospho-mimetic mutations maintain basal GTPase activity, but eliminate GTPase stimulation by actin and decrease GTPase stimulation by cardiolipin, Mff, and MiD49. Phosphorylation of S579 by Erk2 produces similar effects. When mixed with wild-type Drp1, both S579D and S600D phospho-mimetic mutants reduce the actin-stimulated GTPase activity of Drp1-WT. Conversely, a Drp1 mutant that lacks GTPase activity, the K38A mutant, stimulates Drp1-WT GTPase activity under both basal and actin-stimulated conditions, similar to previous results for dynamin-1. These results suggest that the effect of S579 phosphorylation is not to activate Drp1 directly, and likely requires additional factors for stimulation of mitochondrial fission in cells. In addition, our results suggest that nearest neighbor interactions within the Drp1 oligomer affect catalytic activity.\nExploiting Substrate Specificities of 6-O-Sulfotransferases to Enzymatically Synthesize Keratan Sulfate Oligosaccharides\nAuthors: Wu, Y.; Vos, G.; Huang, C.; Chapla, D.; Kimpel, A.; Moremen, K. W.; de Vries, R. P.; Boons, G.-J.\nScore: 5.9, Published: 2023-08-21 DOI: 10.1101/2023.08.20.554027\nKeratan sulfate (KS) is a glycosaminoglycan that is widely expressed in the extracellular matrix of various tissue types where it is involved in many biological processes. Herein, we describe a chemo-enzymatic approach to prepare well-defined KS oligosaccharides by exploiting known and newly discovered substrate specificities of relevant sulfotransferases. The premise of the approach is that recombinant GlcNAc-6-O-sulfotransferases (CHST2) only sulfates terminal GlcNAc moieties to give GlcNAc6S that can be galactosylated by B4GalT4. Furthermore, CHST1 can modify internal galactosides of a poly-LacNAc chain, however, it was found that a GlcNAc6S residue greatly increases the reactivity of CHST1 of a neighboring and internal galactoside. The presence of a 2,3-linked sialoside further modulates the site of modification by CHST1, and a galactoside flanked by 2,3-Neu5Ac and GlcNAc6S is preferentially sulfated over other Gal residues. The substrate specificities of CHST1 and 2 were exploited to prepare a panel of KS oligosaccharides including selectively sulfated N-glycans. The compounds and several other reference derivatives were used to construct a microarray that was probed for binding by several plant lectins, Siglec proteins and hemagglutinins of influenza viruses. It was found that not only the sulfation pattern but also presentation of epitopes as part of an O- or N-glycan determines binding properties.\nSCALiR: a web application for automating absolute quantification of mass spectrometry-based metabolomics data\nAuthors: Bishop, S. L.; Ponce-Alvarez, L.; Wacker, S.; Groves, R. A.; Lewis, I. A.\nScore: 5.0, Published: 2023-08-16 DOI: 10.1101/2023.08.16.551807\nMetabolomics is an important approach for studying complex biological systems. Quantitative liquid chromatography-mass spectrometry (LC-MS)-based metabolomics is becoming a mainstream strategy but presents several technical challenges that limit its widespread use. Computing metabolite concentrations using standard curves generated from standard mixtures of known concentrations is a labor-intensive process which is often performed manually. Currently, there are few options for open-source software tools that can automatically calculate metabolite concentrations. Herein, we introduce SCALiR (Standard Curve Application for determining Linear Ranges), a new web-based software tool specifically built for this task, which allows users to automatically transform LC-MS signal data into absolute quantitative data (https://www.lewisresearchgroup.org/software). The algorithm used in SCALiR automatically finds the equation of the line of best fit for each standard curve and uses this equation to calculate compound concentrations from their LC-MS signal. Using a standard mix containing 77 metabolites, we found excellent correlation between the concentrations calculated by SCALiR and the expected concentrations of each compound (R2 = 0.99) and that SCALiR reproducibly calculated concentrations of mid-range standards across ten analytical batches (average coefficient of variation 0.091). SCALiR offers users several advantages, including that it (1) is open-source and vendor agnostic; (2) requires only 10 seconds of analysis time to compute concentrations of \u003e75 compounds; (3) facilitates automation of quantitative workflows; and (4) performs deterministic evaluation of compound quantification limits. SCALiR provides the metabolomics community with a simple and rapid tool that enables rigorous and reproducible quantitative metabolomics studies.\n",
  "wordCount" : "2413",
  "inLanguage": "en",
  "datePublished": "2023-08-27T10:37:02Z",
  "dateModified": "2023-08-27T10:37:02Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta"><span>updated on August 27, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.20.554025">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.20.554025" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.20.554025">
        <p class="paperTitle">Structure and design of Langya virus glycoprotein antigens</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.20.554025" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.20.554025" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Z.; McCallum, M.; Yan, L.; Sharkey, W.; Park, Y.-J.; Dang, H. V.; Amaya, M.; Person, A.; Broder, C. C.; Veesler, D. J.</p>
        <p class="info">Score: 31.2, Published: 2023-08-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.20.554025' target='https://doi.org/10.1101/2023.08.20.554025'> 10.1101/2023.08.20.554025</a></p>
        <p class="abstract">Langya virus (LayV) is a recently discovered henipavirus (HNV), isolated from febrile patients in China, harboring a genome distantly related to that of Nipah virus (NiV) and Hendra virus (HeV). HNV entry into host cells is mediated by the attachment (G) and fusion (F) glycoproteins which are the main targets of neutralizing antibodies. We show here that the LayV F and G glycoproteins promote membrane fusion with human, mouse and hamster target cells using a different, yet unknown, receptor than NiV and HeV and that NiV- and HeV-elicited monoclonal and polyclonal antibodies do not cross-react with LayV F and G. We determined cryo-electron microscopy structures of LayV F, in the prefusion and postfusion states, and of LayV G, revealing previously unknown conformational landscapes and explaining their distinct antigenicity relative to NiV and HeV. We computationally designed stabilized LayV G constructs and demonstrate the generalizability of an HNV F prefusion-stabilization strategy which will expedite vaccine development efforts. Our data will support the development of vaccines and therapeutics against LayV and closely related HNVs, such as the monoclonal antibodies characterized here.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554214">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554214" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554214">
        <p class="paperTitle">Microglia mediate contact-independent neuronal pruning via secreted Neuraminidase-3 associated with extracellular vesicles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554214" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554214" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Delaveris, C. S.; Wang, C. L.; Riley, N. M.; Li, S.; Kulkarni, R. U.; Bertozzi, C. R.</p>
        <p class="info">Score: 17.0, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554214' target='https://doi.org/10.1101/2023.08.21.554214'> 10.1101/2023.08.21.554214</a></p>
        <p class="abstract">Neurons communicate with each other through electrochemical transmission at synapses. Microglia, the resident immune cells of the central nervous system, can prune these synapses through a variety of contact-dependent and -independent means. Microglial secretion of active sialidase enzymes upon exposure to inflammatory stimuli is one unexplored mechanism of pruning. Recent work from our lab showed that treatment of neurons with bacterial sialidases disrupts neuronal network connectivity. Here, we find that activated microglia secrete Neuraminidase-3 (Neu3) associated with fusogenic extracellular vesicles. Furthermore, we show Neu3 mediates contact-independent pruning of neurons and subsequent disruption of neuronal networks through neuronal glycocalyx remodeling. We observe that NEU3 is transcriptionally upregulated upon exposure to inflammatory stimuli, and that a genetic knock-out of NEU3 abrogates the sialidase activity of inflammatory microglial secretions. Moreover, we demonstrate that Neu3 is associated with a subpopulation of extracellular vesicles, possibly exosomes, that are secreted by microglia upon inflammatory insult. Finally, we demonstrate that Neu3 is both necessary and sufficient to both desialylate neurons and decrease neuronal network connectivity. These results implicate Neu3 in remodeling of the glycocalyx leading to aberrant network-level activity of neurons, with implications in neuroinflammatory diseases such as Parkinsons disease and Alzheimers disease.

Graphical AbstractNeuroinflammation induces secretion of the sialidase Neu3 via extracellular vesicles from microglia that prune neuronal synapses and disrupt neuronal communication.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=76 SRC=&#34;FIGDIR/small/554214v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (25K):
org.highwire.dtl.DTLVardef@1c5a5aaorg.highwire.dtl.DTLVardef@ff5a8eorg.highwire.dtl.DTLVardef@9237b6org.highwire.dtl.DTLVardef@187a6a3_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.554384">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.554384" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.554384">
        <p class="paperTitle">De novo design of buttressed loops for sculpting protein functions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.554384" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.554384" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jiang, H.; Jude, K. M.; Wu, K.; Fallas, J.; Ueda, G.; Brunette, T.; Hicks, D.; Pyles, H.; Yang, A.; Carter, L.; Lamb, M.; Li, X.; Levine, P. M.; Stewart, L. J.; Garcia, K. C.; Baker, D.</p>
        <p class="info">Score: 14.8, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.554384' target='https://doi.org/10.1101/2023.08.22.554384'> 10.1101/2023.08.22.554384</a></p>
        <p class="abstract">In natural proteins, structured loops play central roles in molecular recognition, signal transduction and enzyme catalysis. However, because of the intrinsic flexibility and irregularity of loop regions, organizing multiple structured loops at protein functional sites has been very difficult to achieve by de novo protein design. Here we describe a solution to this problem that generates structured loops buttressed by extensive hydrogen bonding interactions with two neighboring loops and with secondary structure elements. We use this approach to design tandem repeat proteins with buttressed loops ranging from 9 to 14 residues in length. Experimental characterization shows the designs are folded and monodisperse, highly soluble, and thermally stable. Crystal structures are in close agreement with the computational design models, with the loops structured and buttressed by their neighbors as designed. We demonstrate the functionality afforded by loop buttressing by designing and characterizing binders for extended peptides in which the loops form one side of an extended binding pocket. The ability to design multiple structured loops should contribute quite generally to efforts to design new protein functions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554490">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554490" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554490">
        <p class="paperTitle">Allosteric activation of the co-receptor BAK1 by the EFR receptor kinase initiates immune signaling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554490" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554490" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mühlenbeck, H.; Tsutsui, Y.; Lemmon, M.; Bender, K. W.; Zipfel, C.</p>
        <p class="info">Score: 14.7, Published: 2023-08-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554490' target='https://doi.org/10.1101/2023.08.23.554490'> 10.1101/2023.08.23.554490</a></p>
        <p class="abstract">Transmembrane signaling by plant receptor kinases (RKs) has long been thought to involve reciprocal trans-phosphorylation of their intracellular kinase domains. The fact that many of these are pseudokinase domains, however, suggests that additional mechanisms must govern RK signaling activation. Non-catalytic (pseudo)kinase signaling mechanisms have been described in metazoans, but information is scarce for plants. Recently, a non-catalytic function was reported for the leucine-rich repeat (LRR)-RK subfamily XIIa member EFR (ELONGATION FACTOR TU RECEPTOR) and phosphorylation-dependent conformational changes were proposed to regulate signaling of RKs with non-RD kinase domains. Here, using EFR as a model, we describe a non-catalytic activation mechanism for LRR-RKs with non-RD kinase domains. EFR is an active kinase, but a kinase-dead variant retains the ability to enhance catalytic activity of its co-receptor kinase BAK1/SERK3 (BRASSINOSTEROID INSENSITIVE 1-ASSOCIATED KINASE 1/SOMATIC EMBRYOGENESIS RECEPTOR KINASE 3). Applying hydrogen-deuterium exchange mass spectrometry (HDX-MS) analysis and designing homology-based intragenic suppressor mutations, we provide evidence that the EFR kinase domain must adopt its active conformation in order to activate BAK1 allosterically, likely by supporting C-helix positioning in BAK1. Our results suggest a conformational toggle model for signaling, in which BAK1 first phosphorylates EFR in the activation loop to stabilize its active conformation, allowing EFR in turn to allosterically activate BAK1.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.553045">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.553045" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.553045">
        <p class="paperTitle">Blocking oligomerization is the most viable strategy to inhibit STING</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.553045" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.553045" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chan, R. J. M.; Cao, X.; Ergun, S. L.; Njomen, E.; Lynch, S. R.; Cravatt, B. F.; Li, L.</p>
        <p class="info">Score: 16.0, Published: 2023-08-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.553045' target='https://doi.org/10.1101/2023.08.11.553045'> 10.1101/2023.08.11.553045</a></p>
        <p class="abstract">Autoimmune disorders are frequently exacerbated by aberrant STING signaling. While STING inhibitors are highly sought after, no viable therapeutic strategies have been reported. When stimulated, STING undergoes a multi-step mechanism to activate downstream interferon-stimulated genes (ISGs), including palmitoylation, translocation from the ER to the Golgi, and oligomerization, but it remains unclear which step(s) are strictly required and therefore should be targeted by potential STING inhibitors. Here, we comprehensively established the order and determined the necessity of each STING activation step. We find that ER-to-Golgi translocation is dispensable for downstream ISG activation, and a previously studied palmitoylation site, C91, is not required for activation in all contexts. Cysteine reactivity mapping and metabolic labeling identified a new palmitoylation site, C64, that is basally palmitoylated and required for STING activation, but is not druggable due to poor solvent accessibility. We confirm that the oligomerization step of STING activation is strictly required, and therefore developed a short peptide mimicking its natural autoinhibitory domain that effectively inhibits STING activation: a proof of concept for the first viable strategy for developing STING inhibitors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554125">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554125" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554125">
        <p class="paperTitle">Bromodomain Factor 5 as a Target for Antileishmanial Drug Discovery</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554125" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554125" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Russell, C. N.; Carter, J. L.; Borgia, J. M.; Bush, J.; Calderon, F.; Gabarro, R.; Conway, S. J.; Mottram, J. C.; Wilkinson, A. J.; Jones, N. G.</p>
        <p class="info">Score: 11.1, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554125' target='https://doi.org/10.1101/2023.08.21.554125'> 10.1101/2023.08.21.554125</a></p>
        <p class="abstract">Leishmaniases are a collection of neglected tropical diseases caused by kinetoplastid parasites in the genus Leishmania. Current chemotherapies are severely limited and the need for new antileishmanials is of pressing international importance. Bromodomains are epigenetic reader domains that have shown promising therapeutic potential for cancer therapy and may also present an attractive target to treat parasitic diseases. Here, we investigate Leishmania donovani bromodomain factor 5 (LdBDF5) as a target for antileishmanial drug discovery. LdBDF5 contains pair of bromodomains (BD5.1 and BD5.2) in an N-terminal tandem repeat. We purified recombinant bromodomains of L. donovani BDF5 and determined the structure of BD5.2 by X-ray crystallography. Using a histone peptide microarray and fluorescence polarisation assay, we identified binding interactions of LdBDF5 bromodomains with acetylated peptides derived from histones H2B and H4. In orthogonal biophysical assays including thermal shift assays, fluorescence polarisation and NMR, we showed that BDF5 bromodomains bind to human bromodomain inhibitors SGC-CBP30, bromosporine and I-BRD9, moreover, SGC-CBP30 exhibited activity against Leishmania promastigotes in cell viability assays. These findings exemplify the potential BDF5 holds as a drug target in Leishmania and provide a foundation for the future development of optimised antileishmanial compounds targeting this epigenetic reader protein.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=113 SRC=&#34;FIGDIR/small/554125v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (25K):
org.highwire.dtl.DTLVardef@a6c345org.highwire.dtl.DTLVardef@5df8c8org.highwire.dtl.DTLVardef@1784d20org.highwire.dtl.DTLVardef@10fad26_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.17.553673">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.17.553673" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.17.553673">
        <p class="paperTitle">Cryo-EM of soft-landed β-galactosidase: Gas-phase and native structures are remarkably similar</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.17.553673" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.17.553673" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Esser, T. K.; Boehning, J.; Onur, A.; Chinthapalli, D. K.; Eriksson, L.; Grabarics, M.; Fremdling, P.; Konijnenberg, A.; Makarov, A. A.; Botman, A.; Peter, C.; Benesch, J. L. P.; Robinson, C. V.; Gault, J.; Baker, L.; Bharat, T. A. M.; Rauschenbach, S.</p>
        <p class="info">Score: 54.6, Published: 2023-08-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.17.553673' target='https://doi.org/10.1101/2023.08.17.553673'> 10.1101/2023.08.17.553673</a></p>
        <p class="abstract">Native mass spectrometry (native MS) is a powerful technique that provides information on stoichiometry, interactions, homogeneity and shape of protein complexes. However, the extent of deviation between protein structures in the mass spectrometer and in solution remains a matter of debate. Here, we uncover the gas-phase structure of {beta}-galactosidase using single particle electron cryomicroscopy (cryo-EM) down to 2.6 [A] resolution, enabled by soft-landing of mass-selected protein complexes onto cold TEM grids and in-situ ice coating. We find that large parts of the secondary and tertiary structure are retained from solution, with dehydration-driven subunit reorientation leading to consistent compaction in the gas phase. Our work enables visualizing the structure of gas-phase protein complexes from numerous experimental scenarios at side-chain resolution and demonstrates the possibility of more controlled cryo-EM sample preparation.

One Sentence SummaryElectrospray ion-beam deposition on cold grids and in-vacuum ice growth enable cryo-EM of mass-selected proteins at 2.6[A].</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.20.554022">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.20.554022" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.20.554022">
        <p class="paperTitle">Effects of phosphorylation on Drp1 activation by its receptors, actin, and cardiolipin</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.20.554022" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.20.554022" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, A.; Hatch, A. L.; Higgs, H.</p>
        <p class="info">Score: 8.6, Published: 2023-08-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.20.554022' target='https://doi.org/10.1101/2023.08.20.554022'> 10.1101/2023.08.20.554022</a></p>
        <p class="abstract">Drp1 is a dynamin family GTPase that is required for mitochondrial and peroxisomal division, in which it oligomerizes into a ring and constricts the underlying membrane in a GTP hydrolysis-dependent manner. Oligomerization increases Drp1 GTPase activity through interactions between neighboring GTPase domains. In cells, Drp1 is regulated by several factors including Drp1 receptors, actin filaments, cardiolipin, and phosphorylation at two sites: S579 and S600. Phosphorylation of S579 is widely regarded as activating, while S600 phosphorylation is commonly considered inhibiting. However, the direct effects of phosphorylation on Drp1 GTPase activity have not been investigated in detail. In this study, we compare the effects of S579 and S600 phosphorylation on purified Drp1, using phospho-mimetic mutants and in vitro phosphorylation. The oligomerization state of both phospho-mimetic mutants is shifted toward smaller oligomers. Both phospho-mimetic mutations maintain basal GTPase activity, but eliminate GTPase stimulation by actin and decrease GTPase stimulation by cardiolipin, Mff, and MiD49. Phosphorylation of S579 by Erk2 produces similar effects. When mixed with wild-type Drp1, both S579D and S600D phospho-mimetic mutants reduce the actin-stimulated GTPase activity of Drp1-WT. Conversely, a Drp1 mutant that lacks GTPase activity, the K38A mutant, stimulates Drp1-WT GTPase activity under both basal and actin-stimulated conditions, similar to previous results for dynamin-1. These results suggest that the effect of S579 phosphorylation is not to activate Drp1 directly, and likely requires additional factors for stimulation of mitochondrial fission in cells. In addition, our results suggest that nearest neighbor interactions within the Drp1 oligomer affect catalytic activity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.20.554027">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.20.554027" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.20.554027">
        <p class="paperTitle">Exploiting Substrate Specificities of 6-O-Sulfotransferases to Enzymatically Synthesize Keratan Sulfate Oligosaccharides</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.20.554027" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.20.554027" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wu, Y.; Vos, G.; Huang, C.; Chapla, D.; Kimpel, A.; Moremen, K. W.; de Vries, R. P.; Boons, G.-J.</p>
        <p class="info">Score: 5.9, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.20.554027' target='https://doi.org/10.1101/2023.08.20.554027'> 10.1101/2023.08.20.554027</a></p>
        <p class="abstract">Keratan sulfate (KS) is a glycosaminoglycan that is widely expressed in the extracellular matrix of various tissue types where it is involved in many biological processes. Herein, we describe a chemo-enzymatic approach to prepare well-defined KS oligosaccharides by exploiting known and newly discovered substrate specificities of relevant sulfotransferases. The premise of the approach is that recombinant GlcNAc-6-O-sulfotransferases (CHST2) only sulfates terminal GlcNAc moieties to give GlcNAc6S that can be galactosylated by B4GalT4. Furthermore, CHST1 can modify internal galactosides of a poly-LacNAc chain, however, it was found that a GlcNAc6S residue greatly increases the reactivity of CHST1 of a neighboring and internal galactoside. The presence of a 2,3-linked sialoside further modulates the site of modification by CHST1, and a galactoside flanked by 2,3-Neu5Ac and GlcNAc6S is preferentially sulfated over other Gal residues. The substrate specificities of CHST1 and 2 were exploited to prepare a panel of KS oligosaccharides including selectively sulfated N-glycans. The compounds and several other reference derivatives were used to construct a microarray that was probed for binding by several plant lectins, Siglec proteins and hemagglutinins of influenza viruses. It was found that not only the sulfation pattern but also presentation of epitopes as part of an O- or N-glycan determines binding properties.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.551807">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.551807" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.551807">
        <p class="paperTitle">SCALiR: a web application for automating absolute quantification of mass spectrometry-based metabolomics data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.551807" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.551807" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bishop, S. L.; Ponce-Alvarez, L.; Wacker, S.; Groves, R. A.; Lewis, I. A.</p>
        <p class="info">Score: 5.0, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.551807' target='https://doi.org/10.1101/2023.08.16.551807'> 10.1101/2023.08.16.551807</a></p>
        <p class="abstract">Metabolomics is an important approach for studying complex biological systems. Quantitative liquid chromatography-mass spectrometry (LC-MS)-based metabolomics is becoming a mainstream strategy but presents several technical challenges that limit its widespread use. Computing metabolite concentrations using standard curves generated from standard mixtures of known concentrations is a labor-intensive process which is often performed manually. Currently, there are few options for open-source software tools that can automatically calculate metabolite concentrations. Herein, we introduce SCALiR (Standard Curve Application for determining Linear Ranges), a new web-based software tool specifically built for this task, which allows users to automatically transform LC-MS signal data into absolute quantitative data (https://www.lewisresearchgroup.org/software). The algorithm used in SCALiR automatically finds the equation of the line of best fit for each standard curve and uses this equation to calculate compound concentrations from their LC-MS signal. Using a standard mix containing 77 metabolites, we found excellent correlation between the concentrations calculated by SCALiR and the expected concentrations of each compound (R2 = 0.99) and that SCALiR reproducibly calculated concentrations of mid-range standards across ten analytical batches (average coefficient of variation 0.091). SCALiR offers users several advantages, including that it (1) is open-source and vendor agnostic; (2) requires only 10 seconds of analysis time to compute concentrations of &gt;75 compounds; (3) facilitates automation of quantitative workflows; and (4) performs deterministic evaluation of compound quantification limits. SCALiR provides the metabolomics community with a simple and rapid tool that enables rigorous and reproducible quantitative metabolomics studies.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
