<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biochemistry</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biochemistry | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Targeted protein editing with an antibody-based system
Authors: Rimon, O.; Konc, J.; Roy Chowdhury, V.; Bernardes, G. J. L.; Vendruscolo, M.
Score: 24.0, Published: 2023-07-29 DOI: 10.1101/2023.07.28.551006
The chemical modification of proteins is one of the major mechanisms used to regulate the trafficking and function of these macromolecules in the cell. It is therefore of great interest to develop tools to exploit this type of modifications for applications in molecular biology, medicine and biotechnology.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biochemistry/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biochemistry" />
<meta property="og:description" content="Targeted protein editing with an antibody-based system
Authors: Rimon, O.; Konc, J.; Roy Chowdhury, V.; Bernardes, G. J. L.; Vendruscolo, M.
Score: 24.0, Published: 2023-07-29 DOI: 10.1101/2023.07.28.551006
The chemical modification of proteins is one of the major mechanisms used to regulate the trafficking and function of these macromolecules in the cell. It is therefore of great interest to develop tools to exploit this type of modifications for applications in molecular biology, medicine and biotechnology." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biochemistry/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-02T10:38:09+00:00" />
<meta property="article:modified_time" content="2023-08-02T10:38:09+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biochemistry"/>
<meta name="twitter:description" content="Targeted protein editing with an antibody-based system
Authors: Rimon, O.; Konc, J.; Roy Chowdhury, V.; Bernardes, G. J. L.; Vendruscolo, M.
Score: 24.0, Published: 2023-07-29 DOI: 10.1101/2023.07.28.551006
The chemical modification of proteins is one of the major mechanisms used to regulate the trafficking and function of these macromolecules in the cell. It is therefore of great interest to develop tools to exploit this type of modifications for applications in molecular biology, medicine and biotechnology."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biochemistry",
      "item": "https://trxiv.yorks0n.com/posts/biochemistry/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biochemistry",
  "name": "biochemistry",
  "description": "Targeted protein editing with an antibody-based system\nAuthors: Rimon, O.; Konc, J.; Roy Chowdhury, V.; Bernardes, G. J. L.; Vendruscolo, M.\nScore: 24.0, Published: 2023-07-29 DOI: 10.1101/2023.07.28.551006\nThe chemical modification of proteins is one of the major mechanisms used to regulate the trafficking and function of these macromolecules in the cell. It is therefore of great interest to develop tools to exploit this type of modifications for applications in molecular biology, medicine and biotechnology.",
  "keywords": [
    
  ],
  "articleBody": " Targeted protein editing with an antibody-based system\nAuthors: Rimon, O.; Konc, J.; Roy Chowdhury, V.; Bernardes, G. J. L.; Vendruscolo, M.\nScore: 24.0, Published: 2023-07-29 DOI: 10.1101/2023.07.28.551006\nThe chemical modification of proteins is one of the major mechanisms used to regulate the trafficking and function of these macromolecules in the cell. It is therefore of great interest to develop tools to exploit this type of modifications for applications in molecular biology, medicine and biotechnology. Here we present a method of using antibodies to perform post-translational covalent modifications of endogenous proteins in complex environments by exploiting proximity-driven chemistry. The method is based on the ability of antibodies to hold a weakly reactive group adjacently to its intended site of reaction by binding the target protein on a nearby epitope. We illustrate this approach by targeting the green fluorescent protein in increasingly complex environments.\nGenetic and functional diversity of beta-N-acetylgalactosamine residue-targeting glycosidases expanded by deep-sea metagenome\nAuthors: Sumida, T.; Hiraoka, S.; Usui, K.; Ishiwata, A.; Sengoku, T.; Stubbs, K. A.; Tanaka, K.; Deguchi, S.; Fushinobu, S.; Nunoura, T.\nScore: 17.1, Published: 2023-07-28 DOI: 10.1101/2023.07.28.550916\n{beta}-N-Acetylgalactosamine-containing glycans play essential roles in several biological processes, including cell adhesion, signal transduction, and immune responses. {beta}-N-Acetylgalactosaminidases hydrolyze {beta}-N-acetylgalactosamine linkages of various glycoconjugates. However, their biological significance remains ambiguous, primarily because only one type of enzyme, exo-{beta}-N-acetylgalactosaminidases that specifically act on {beta}-N-acetylgalactosamine residues, has been documented so far. In this study, we identified three novel glycoside hydrolase families distributed among all three domains of life and characterized eight novel {beta}-N-acetylgalactosaminidases and {beta}-N-acetylhexosaminidase through sequence-based screening of deep-sea metagenomes and subsequent searching of public protein databases. Despite low sequence similarity, the crystal structures of these enzymes demonstrate that all enzymes share a prototype structure and diversify their substrate specificities (endo-, dual-endo/exo-, and exo-) through the accumulation of mutations and insertional amino acid sequences. The diverse {beta}-N-acetylgalactosaminidases reported in this study could facilitate the comprehension of their structures and functions and present novel evolutionary pathways for expanding their substrate specificity.\nShared Structural Features of Miro Binding Control Mitochondrial Homeostasis\nAuthors: Covill-Cooke, C.; Kwizera, B.; Lopez-Domenech, G.; Thompson, C. O. D.; Cheung, N. J.; Cerezo, E.; Peterka, M.; Kittler, J. T.; Kornmann, B.\nScore: 9.7, Published: 2023-07-25 DOI: 10.1101/2023.07.25.550462\nMiro proteins are universally conserved mitochondrial calcium-binding GTPases that regulate a multitude of mitochondrial processes, including transport, clearance and lipid trafficking. Miro binds a variety of client proteins involved in these functions. How this binding is operated at the molecular level and whether and how it is important for mitochondrial health, however, remains unknown. Here, we show that known Miro clients all use a similar short motif to bind the same structural element: a highly conserved hydrophobic pocket in the calcium-binding domain of Miro. Using these Miro-binding motifs, we identified direct interactors de novo, including yeast Mdm34, and mammalian MTFR1/2/1L, VPS13D and Parkin. Given the shared binding mechanism and conservation across eukaryotes, we propose that Miro is a universal mitochondrial adaptor coordinating mitochondrial health. One-Sentence SummaryFunctionally diverse mitochondrial proteins interact with a conserved hydrophobic pocket on the calcium-binding Miro-GTPases.\nStructure and function of a YeeE-YeeD complex for sophisticated thiosulfate uptake\nAuthors: Ikei, M.; Miyazaki, R.; Monden, K.; Naito, Y.; Takeuchi, A.; Takahashi, Y. S.; Tanaka, Y.; Mori, T.; Ichikawa, M.; Tsukazaki, T.\nScore: 7.1, Published: 2023-07-29 DOI: 10.1101/2023.07.28.550924\nUptake of thiosulfate ions as an inorganic sulfur source from the environment is important for bacterial sulfur assimilation. Recently, a selective thiosulfate uptake pathway involving membrane protein YeeE (TsuA) was characterized. However, the precise function of YeeE and a putative cofactor in the thiosulfate ion uptake pathway remained unclear. Here, we assessed selective thiosulfate transport via YeeE in vitro and characterized YeeD (TsuB) as an adjacent and essential cofactor for YeeE-mediated thiosulfate uptake in vivo. We further showed that YeeD possesses thiosulfate decomposition activity and that a conserved cysteine in YeeD was modified in several forms in the presence of thiosulfate. Finally, the crystal structure of a YeeE-YeeD fusion protein at 2.6-[A] resolution revealed their interactions. The association was evaluated by a binding assay using purified proteins. Based on these results, a model of the sophisticated uptake of thiosulfate ions by YeeE and YeeD is proposed.\nMissing-link fold reveals the evolutionary pathway between RNA polymerase and ribosomal proteins\nAuthors: Yagi, S.; Tagami, S.\nScore: 26.0, Published: 2023-07-05 DOI: 10.1101/2023.07.05.547881\nNumerous molecular machines are required to drive the central dogma of molecular biology. However, the means by which such numerous proteins emerged in the early evolutionary stage of life remains enigmatic. Many of them possess small {beta}-barrel folds with different topologies, represented by DPBB conserved in DNA and RNA polymerases, and RIFT, OB, and SH3 in ribosomal proteins. Here, we discovered that the previously reconstructed ancient DPBB sequence could also adopt a novel {beta}-barrel fold named DZBB, which shares similarities with RIFT and OB. Indeed, DZBB could be transformed into them through simple engineering experiments. Furthermore, the OB designs could be converted into SH3 by circular-permutation. These results indicate that these {beta}-barrels diversified quickly from a common ancestor at the beginning of the central dogma evolution.\nDeep mutational scanning and machine learning uncover antimicrobial peptide features driving membrane selectivity\nAuthors: Randall, J. R.; Vieira, L.; Wilke, C. O.; Davies, B. W.\nScore: 6.6, Published: 2023-07-29 DOI: 10.1101/2023.07.28.551017\nAntimicrobial peptides are critical actors of the innate immune system. Those with direct antibacterial activity often kill via cell membrane lysis, but have difficulty distinguishing between target cells and mammalian cells complicating therapeutic use. Here, we use a new technique called deep mutational surface localized antimicrobial display (dmSLAY) to rapidly elucidate single residue importance and flexibility for Protegrin-1, a potent yet toxic host-defense peptide. Subsequent biochemical analyses reveal that bacterial membrane selectivity is improved for Protegrin-1 variants which maintain membrane bound secondary structure, avoid large aromatic residues, and mutate cysteine pairs. Machine learning is used to expand our analysis to encompass over five million sequence variations and uncover full mutational profiles which promote antibacterial potency, toxicity, and specificity for bacterial or mammalian cell membranes. Our results describe an innovative, high-throughput approach for rapidly elucidating antimicrobial peptide sequence-structure-function relationships which can be used to inform future clinical antibiotic design.\nIdentification of Sequence Determinants for the ABHD14 Enzymes\nAuthors: Vaidya, K.; Rodrigues, G.; Gupta, S.; Devarajan, A.; Yeolekar, M.; Madhusudhan, M. S.; Kamat, S. S.\nScore: 6.1, Published: 2023-07-31 DOI: 10.1101/2023.07.30.551196\nOver the course of evolution, enzymes have developed remarkable functional diversity in catalyzing important chemical reactions across various organisms, and understanding how new enzyme functions might have evolved remains an important question in modern enzymology. To systematically annotate functions, based on protein sequences and available biochemical studies, enzymes with similar catalytic mechanisms and/or aspects of catalysis have been clustered together into an enzyme superfamily. Typically, enzymes within a superfamily have similar overall three-dimensional structures, conserved key catalytic residues, but large variations in substrate recognition sites and residues to accommodate the diverse biochemical reactions that are catalyzed within the superfamily. The serine hydrolases are an excellent example of such an enzyme superfamily, that based on known enzymatic activities and protein sequences, is split almost equally into the serine proteases and metabolic serine hydrolases. Within the metabolic serine hydrolases, are two outlying members, ABHD14A and ABHD14B, that have high sequence similarity, but their functions remained cryptic till recently. While ABHD14A still lacks any functional annotation to date, we recently showed that ABHD14B functions as a lysine deacetylase in mammals. Given their high sequence similarity, automated databases wrongly assign ABHD14A and ABHD14B as the same enzyme, and therefore, annotating functions to them in various organisms maybe problematic. In this paper, we present a bioinformatics study coupled to biochemical experiments, that identifies key sequence determinants for both ABHD14A and ABHD14B, and enables better classification for them. Additionally, we map these enzymes on an evolutionary timescale, and provide a resource in studying these interesting enzymes in different organisms.\nApplication of monolayer graphene to cryo-electron microscopy grids for high-resolution structure determination\nAuthors: Grassetti, A. V.; May, M. B.; Davis, J. H.\nScore: 5.9, Published: 2023-07-28 DOI: 10.1101/2023.07.28.550908\nIn cryogenic electron microscopy (cryo-EM), purified macromolecules are typically applied to a grid bearing a holey carbon foil, blotted to remove excess liquid and rapidly frozen in a roughly 20-100 nm thick layer of vitreous ice that is suspended across roughly 1 m-wide foil holes. The resulting sample is then imaged using cryogenic transmission electron microscopy and, after substantial image processing, near-atomic resolution structures can be determined. Despite cryo-EMs widespread adoption, sample preparation remains a severe bottleneck in cryo-EM workflows, with users often encountering challenges related to samples behaving poorly in the suspended vitreous ice. Recently, methods have been developed to modify cryo-EM grids with a single continuous layer of graphene, which acts as a support surface that often increases particle density in the imaged area and can reduce interactions between particles and the air-water interface. Here, we provide detailed protocols for the application of graphene to cryo-EM grids, and for rapidly assessing the relative hydrophilicity of the resulting grids. Additionally, we describe an EM-based method to confirm the presence of graphene by visualizing its characteristic diffraction pattern. Finally, we demonstrate the utility of these graphene supports by rapidly reconstructing a 2.7 [A] resolution density map of an exemplar Cas9 complex using a highly pure sample at a relatively low concentration. SUMMARYThe application of support layers, such as graphene, to cryo-electron microscopy grids can increase the density of particles imaged, limit particle interactions with the air-water interface, reduce the extent of beam-induced motion, and, in some instances, improve the distribution of particle orientations. This paper describes a robust protocol for coating cryo-EM grids with a monolayer of graphene for improved cryo-sample preparation.\nA practical and robust method to evaluate metabolic fluxes in pancreatic islets\nAuthors: Rocha, D. S.; Manucci, A. C.; Bruni-Cardoso, A.; Kowaltowski, A. J.; Vilas-Boas, E. A.\nScore: 5.8, Published: 2023-07-26 DOI: 10.1101/2023.07.25.550538\nAims/hypothesisEfficient mitochondrial oxidative phosphorylation is essential for pancreatic beta cell responses to nutrient levels. Consequently, the evaluation of mitochondrial oxygen consumption and ATP production is important to investigate essential aspects of pancreatic islet pathophysiology. Currently, most studies use cell lines instead of primary islets due to difficulties in measuring primary islet respiration, which requires specific equipment and consumables that are expensive, complicated to use, and poorly reproducible. The aim of this study is to establish a robust and practical method to assess primary islet metabolic fluxes using Extracellular Flux Technology and standard commercial consumables. MethodsPancreatic islets were isolated from 8 to 12-week-old mice and rats, and submitted to a dispersion protocol using trypsin. Dispersed islets were adhered overnight to pre-coated standard Seahorse microplates, and oxygen consumption rates were evaluated using a Seahorse Extracellular Flux Analyzer. We also validated the functionality of dispersed islets by analyzing glucose-stimulated insulin secretion (GSIS) and calcium (Ca2+) influx in response to different modulators by fluorescence microscopy. ResultsWe provide a detailed protocol with all steps necessary to optimize islet isolation and dispersion, in order to achieve a high yield of functional islets and perform metabolic flux analysis. With this method, which requires only a few islets per replicate, both rat and mouse islets present robust basal respiration and proper response to mitochondrial modulators (oligomycin, CCCP, antimycin and rotenone) and glucose addition. Both oligomycin and CCCP concentrations were titrated. Our method was also validated by other functional assays, which show these cells present conserved Ca2+ influx and insulin secretion in response to glucose. Conclusions/interpretationWe established a practical and robust method to assess ex vivo islet metabolic fluxes and oxidative phosphorylation. Our findings cover an important gap in primary islet physiology studies, providing a valuable tool we hope is useful to uncover basic beta cell metabolic mechanisms, as well as for translational investigations, such as pharmacological candidate discovery and islet transplantation protocols. Research in context What is already known about this subject?O_LIPancreatic beta cells efficiently couple oxidative phosphorylation and ATP production with insulin secretion; mitochondrial ATP production is crucial for proper insulin secretion. C_LIO_LIMost studies of beta cell respiration use cell lines instead of primary islets, which are a much more robust model to evaluate beta cell function. C_LIO_LIThe few works with primary islet respiration use specific equipment and consumables that are expensive, complicated, and poorly reproducible. C_LI What is the key question?O_LIIs it possible to develop a practical method to evaluate metabolic fluxes and ATP production in isolated islets, using the standard Seahorse Extracellular Flux Technology? C_LI What are the new findings?O_LIWe optimized rodent islet isolation and functional analysis protocols using standard extracellular flux analysis equipment and consumables. C_LIO_LIOur method allows for increased islet yield and robust islet respiration measurements. C_LI How might this impact on clinical practice in the foreseeable future?O_LIQuantitative measurements of metabolic fluxes and oxidative phosphorylation are the cornerstone of new discoveries in beta cells, and can contribute toward the establishment of new cellular protocols, such as for cell transplantation, as well as the development of new pharmacological agents targeted to these cells. C_LI\nCrystal structure and biochemical activity of the macro domain from rubella virus p150\nAuthors: Stoll, G. A.; Zhang, L.; Modis, Y.\nScore: 5.2, Published: 2023-07-28 DOI: 10.1101/2023.07.28.550942\nRubella virus remains a global health threat. Rubella infections during pregnancy can cause serious congenital pathology and no antiviral treatments are available. Rubella virus encodes a nonstructural polyprotein with RNA polymerase, methyltransferase, and papain-like cysteine protease activities, along with a putative macro domain of unknown function. Macro domains bind ADP-ribose adducts, a post-translational modification that plays a key role in host-virus conflicts. Some macro domains can also remove the mono-ADP-ribose adduct or degrade poly-ADP-ribose chains. Here, we report high-resolution crystal structures of the macro domain from rubella virus nonstructural protein p150, with and without ADP-ribose bound. The overall fold is most similar to macroD-type macro domains from various nonviral species. The specific composition and structure of the residues that poised for catalysis or coordinate ADP-ribose in the rubella virus macro domain are most similar to those of macro domains from alphaviruses. Isothermal calorimetry and enzymatic assays show that the rubella virus macro domain binds ADP-ribose in solution and has mono-ADP-ribosylhydrolase (de-MARylation) activity. IMPORTANCEOur work demonstrates that, like alpha- and coronaviruses, rubiviruses encode a mono-ADP-ribosylhydrolase with a structurally conserved macro domain fold to counteract MARylation by PARPs in the host innate immune response. Our structural data will guide future efforts to develop novel antiviral therapeutics against rubella or infections with related viruses.\n",
  "wordCount" : "2386",
  "inLanguage": "en",
  "datePublished": "2023-08-02T10:38:09Z",
  "dateModified": "2023-08-02T10:38:09Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biochemistry/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biochemistry
    </h1>
    <div class="post-meta"><span>updated on August 2, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.551006">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.551006" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.551006">
        <p class="paperTitle">Targeted protein editing with an antibody-based system</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.551006" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.551006" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rimon, O.; Konc, J.; Roy Chowdhury, V.; Bernardes, G. J. L.; Vendruscolo, M.</p>
        <p class="info">Score: 24.0, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.551006' target='https://doi.org/10.1101/2023.07.28.551006'> 10.1101/2023.07.28.551006</a></p>
        <p class="abstract">The chemical modification of proteins is one of the major mechanisms used to regulate the trafficking and function of these macromolecules in the cell. It is therefore of great interest to develop tools to exploit this type of modifications for applications in molecular biology, medicine and biotechnology. Here we present a method of using antibodies to perform post-translational covalent modifications of endogenous proteins in complex environments by exploiting proximity-driven chemistry. The method is based on the ability of antibodies to hold a weakly reactive group adjacently to its intended site of reaction by binding the target protein on a nearby epitope. We illustrate this approach by targeting the green fluorescent protein in increasingly complex environments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.550916">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.550916" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.550916">
        <p class="paperTitle">Genetic and functional diversity of beta-N-acetylgalactosamine residue-targeting glycosidases expanded by deep-sea metagenome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.550916" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.550916" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sumida, T.; Hiraoka, S.; Usui, K.; Ishiwata, A.; Sengoku, T.; Stubbs, K. A.; Tanaka, K.; Deguchi, S.; Fushinobu, S.; Nunoura, T.</p>
        <p class="info">Score: 17.1, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.550916' target='https://doi.org/10.1101/2023.07.28.550916'> 10.1101/2023.07.28.550916</a></p>
        <p class="abstract">{beta}-N-Acetylgalactosamine-containing glycans play essential roles in several biological processes, including cell adhesion, signal transduction, and immune responses. {beta}-N-Acetylgalactosaminidases hydrolyze {beta}-N-acetylgalactosamine linkages of various glycoconjugates. However, their biological significance remains ambiguous, primarily because only one type of enzyme, exo-{beta}-N-acetylgalactosaminidases that specifically act on {beta}-N-acetylgalactosamine residues, has been documented so far. In this study, we identified three novel glycoside hydrolase families distributed among all three domains of life and characterized eight novel {beta}-N-acetylgalactosaminidases and {beta}-N-acetylhexosaminidase through sequence-based screening of deep-sea metagenomes and subsequent searching of public protein databases. Despite low sequence similarity, the crystal structures of these enzymes demonstrate that all enzymes share a prototype structure and diversify their substrate specificities (endo-, dual-endo/exo-, and exo-) through the accumulation of mutations and insertional amino acid sequences. The diverse {beta}-N-acetylgalactosaminidases reported in this study could facilitate the comprehension of their structures and functions and present novel evolutionary pathways for expanding their substrate specificity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550462">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550462" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550462">
        <p class="paperTitle">Shared Structural Features of Miro Binding Control Mitochondrial Homeostasis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550462" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550462" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Covill-Cooke, C.; Kwizera, B.; Lopez-Domenech, G.; Thompson, C. O. D.; Cheung, N. J.; Cerezo, E.; Peterka, M.; Kittler, J. T.; Kornmann, B.</p>
        <p class="info">Score: 9.7, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550462' target='https://doi.org/10.1101/2023.07.25.550462'> 10.1101/2023.07.25.550462</a></p>
        <p class="abstract">Miro proteins are universally conserved mitochondrial calcium-binding GTPases that regulate a multitude of mitochondrial processes, including transport, clearance and lipid trafficking. Miro binds a variety of client proteins involved in these functions. How this binding is operated at the molecular level and whether and how it is important for mitochondrial health, however, remains unknown. Here, we show that known Miro clients all use a similar short motif to bind the same structural element: a highly conserved hydrophobic pocket in the calcium-binding domain of Miro. Using these Miro-binding motifs, we identified direct interactors de novo, including yeast Mdm34, and mammalian MTFR1/2/1L, VPS13D and Parkin. Given the shared binding mechanism and conservation across eukaryotes, we propose that Miro is a universal mitochondrial adaptor coordinating mitochondrial health.

One-Sentence SummaryFunctionally diverse mitochondrial proteins interact with a conserved hydrophobic pocket on the calcium-binding Miro-GTPases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.550924">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.550924" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.550924">
        <p class="paperTitle">Structure and function of a YeeE-YeeD complex for sophisticated thiosulfate uptake</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.550924" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.550924" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ikei, M.; Miyazaki, R.; Monden, K.; Naito, Y.; Takeuchi, A.; Takahashi, Y. S.; Tanaka, Y.; Mori, T.; Ichikawa, M.; Tsukazaki, T.</p>
        <p class="info">Score: 7.1, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.550924' target='https://doi.org/10.1101/2023.07.28.550924'> 10.1101/2023.07.28.550924</a></p>
        <p class="abstract">Uptake of thiosulfate ions as an inorganic sulfur source from the environment is important for bacterial sulfur assimilation. Recently, a selective thiosulfate uptake pathway involving membrane protein YeeE (TsuA) was characterized. However, the precise function of YeeE and a putative cofactor in the thiosulfate ion uptake pathway remained unclear. Here, we assessed selective thiosulfate transport via YeeE in vitro and characterized YeeD (TsuB) as an adjacent and essential cofactor for YeeE-mediated thiosulfate uptake in vivo. We further showed that YeeD possesses thiosulfate decomposition activity and that a conserved cysteine in YeeD was modified in several forms in the presence of thiosulfate. Finally, the crystal structure of a YeeE-YeeD fusion protein at 2.6-[A] resolution revealed their interactions. The association was evaluated by a binding assay using purified proteins. Based on these results, a model of the sophisticated uptake of thiosulfate ions by YeeE and YeeD is proposed.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.547881">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.547881" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.547881">
        <p class="paperTitle">Missing-link fold reveals the evolutionary pathway between RNA polymerase and ribosomal proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.547881" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.547881" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yagi, S.; Tagami, S.</p>
        <p class="info">Score: 26.0, Published: 2023-07-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.547881' target='https://doi.org/10.1101/2023.07.05.547881'> 10.1101/2023.07.05.547881</a></p>
        <p class="abstract">Numerous molecular machines are required to drive the central dogma of molecular biology. However, the means by which such numerous proteins emerged in the early evolutionary stage of life remains enigmatic. Many of them possess small {beta}-barrel folds with different topologies, represented by DPBB conserved in DNA and RNA polymerases, and RIFT, OB, and SH3 in ribosomal proteins. Here, we discovered that the previously reconstructed ancient DPBB sequence could also adopt a novel {beta}-barrel fold named DZBB, which shares similarities with RIFT and OB. Indeed, DZBB could be transformed into them through simple engineering experiments. Furthermore, the OB designs could be converted into SH3 by circular-permutation. These results indicate that these {beta}-barrels diversified quickly from a common ancestor at the beginning of the central dogma evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.551017">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.551017" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.551017">
        <p class="paperTitle">Deep mutational scanning and machine learning uncover antimicrobial peptide features driving membrane selectivity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.551017" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.551017" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Randall, J. R.; Vieira, L.; Wilke, C. O.; Davies, B. W.</p>
        <p class="info">Score: 6.6, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.551017' target='https://doi.org/10.1101/2023.07.28.551017'> 10.1101/2023.07.28.551017</a></p>
        <p class="abstract">Antimicrobial peptides are critical actors of the innate immune system. Those with direct antibacterial activity often kill via cell membrane lysis, but have difficulty distinguishing between target cells and mammalian cells complicating therapeutic use. Here, we use a new technique called deep mutational surface localized antimicrobial display (dmSLAY) to rapidly elucidate single residue importance and flexibility for Protegrin-1, a potent yet toxic host-defense peptide. Subsequent biochemical analyses reveal that bacterial membrane selectivity is improved for Protegrin-1 variants which maintain membrane bound secondary structure, avoid large aromatic residues, and mutate cysteine pairs. Machine learning is used to expand our analysis to encompass over five million sequence variations and uncover full mutational profiles which promote antibacterial potency, toxicity, and specificity for bacterial or mammalian cell membranes. Our results describe an innovative, high-throughput approach for rapidly elucidating antimicrobial peptide sequence-structure-function relationships which can be used to inform future clinical antibiotic design.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.30.551196">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.30.551196" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.30.551196">
        <p class="paperTitle">Identification of Sequence Determinants for the ABHD14 Enzymes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.30.551196" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.30.551196" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vaidya, K.; Rodrigues, G.; Gupta, S.; Devarajan, A.; Yeolekar, M.; Madhusudhan, M. S.; Kamat, S. S.</p>
        <p class="info">Score: 6.1, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.30.551196' target='https://doi.org/10.1101/2023.07.30.551196'> 10.1101/2023.07.30.551196</a></p>
        <p class="abstract">Over the course of evolution, enzymes have developed remarkable functional diversity in catalyzing important chemical reactions across various organisms, and understanding how new enzyme functions might have evolved remains an important question in modern enzymology. To systematically annotate functions, based on protein sequences and available biochemical studies, enzymes with similar catalytic mechanisms and/or aspects of catalysis have been clustered together into an enzyme superfamily. Typically, enzymes within a superfamily have similar overall three-dimensional structures, conserved key catalytic residues, but large variations in substrate recognition sites and residues to accommodate the diverse biochemical reactions that are catalyzed within the superfamily. The serine hydrolases are an excellent example of such an enzyme superfamily, that based on known enzymatic activities and protein sequences, is split almost equally into the serine proteases and metabolic serine hydrolases. Within the metabolic serine hydrolases, are two outlying members, ABHD14A and ABHD14B, that have high sequence similarity, but their functions remained cryptic till recently. While ABHD14A still lacks any functional annotation to date, we recently showed that ABHD14B functions as a lysine deacetylase in mammals. Given their high sequence similarity, automated databases wrongly assign ABHD14A and ABHD14B as the same enzyme, and therefore, annotating functions to them in various organisms maybe problematic. In this paper, we present a bioinformatics study coupled to biochemical experiments, that identifies key sequence determinants for both ABHD14A and ABHD14B, and enables better classification for them. Additionally, we map these enzymes on an evolutionary timescale, and provide a resource in studying these interesting enzymes in different organisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.550908">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.550908" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.550908">
        <p class="paperTitle">Application of monolayer graphene to cryo-electron microscopy grids for high-resolution structure determination</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.550908" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.550908" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Grassetti, A. V.; May, M. B.; Davis, J. H.</p>
        <p class="info">Score: 5.9, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.550908' target='https://doi.org/10.1101/2023.07.28.550908'> 10.1101/2023.07.28.550908</a></p>
        <p class="abstract">In cryogenic electron microscopy (cryo-EM), purified macromolecules are typically applied to a grid bearing a holey carbon foil, blotted to remove excess liquid and rapidly frozen in a roughly 20-100 nm thick layer of vitreous ice that is suspended across roughly 1 m-wide foil holes. The resulting sample is then imaged using cryogenic transmission electron microscopy and, after substantial image processing, near-atomic resolution structures can be determined. Despite cryo-EMs widespread adoption, sample preparation remains a severe bottleneck in cryo-EM workflows, with users often encountering challenges related to samples behaving poorly in the suspended vitreous ice. Recently, methods have been developed to modify cryo-EM grids with a single continuous layer of graphene, which acts as a support surface that often increases particle density in the imaged area and can reduce interactions between particles and the air-water interface. Here, we provide detailed protocols for the application of graphene to cryo-EM grids, and for rapidly assessing the relative hydrophilicity of the resulting grids. Additionally, we describe an EM-based method to confirm the presence of graphene by visualizing its characteristic diffraction pattern. Finally, we demonstrate the utility of these graphene supports by rapidly reconstructing a 2.7 [A] resolution density map of an exemplar Cas9 complex using a highly pure sample at a relatively low concentration.

SUMMARYThe application of support layers, such as graphene, to cryo-electron microscopy grids can increase the density of particles imaged, limit particle interactions with the air-water interface, reduce the extent of beam-induced motion, and, in some instances, improve the distribution of particle orientations. This paper describes a robust protocol for coating cryo-EM grids with a monolayer of graphene for improved cryo-sample preparation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550538">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550538" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550538">
        <p class="paperTitle">A practical and robust method to evaluate metabolic fluxes in pancreatic islets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550538" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550538" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rocha, D. S.; Manucci, A. C.; Bruni-Cardoso, A.; Kowaltowski, A. J.; Vilas-Boas, E. A.</p>
        <p class="info">Score: 5.8, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550538' target='https://doi.org/10.1101/2023.07.25.550538'> 10.1101/2023.07.25.550538</a></p>
        <p class="abstract">Aims/hypothesisEfficient mitochondrial oxidative phosphorylation is essential for pancreatic beta cell responses to nutrient levels. Consequently, the evaluation of mitochondrial oxygen consumption and ATP production is important to investigate essential aspects of pancreatic islet pathophysiology. Currently, most studies use cell lines instead of primary islets due to difficulties in measuring primary islet respiration, which requires specific equipment and consumables that are expensive, complicated to use, and poorly reproducible. The aim of this study is to establish a robust and practical method to assess primary islet metabolic fluxes using Extracellular Flux Technology and standard commercial consumables.

MethodsPancreatic islets were isolated from 8 to 12-week-old mice and rats, and submitted to a dispersion protocol using trypsin. Dispersed islets were adhered overnight to pre-coated standard Seahorse microplates, and oxygen consumption rates were evaluated using a Seahorse Extracellular Flux Analyzer. We also validated the functionality of dispersed islets by analyzing glucose-stimulated insulin secretion (GSIS) and calcium (Ca2&#43;) influx in response to different modulators by fluorescence microscopy.

ResultsWe provide a detailed protocol with all steps necessary to optimize islet isolation and dispersion, in order to achieve a high yield of functional islets and perform metabolic flux analysis. With this method, which requires only a few islets per replicate, both rat and mouse islets present robust basal respiration and proper response to mitochondrial modulators (oligomycin, CCCP, antimycin and rotenone) and glucose addition. Both oligomycin and CCCP concentrations were titrated. Our method was also validated by other functional assays, which show these cells present conserved Ca2&#43; influx and insulin secretion in response to glucose.

Conclusions/interpretationWe established a practical and robust method to assess ex vivo islet metabolic fluxes and oxidative phosphorylation. Our findings cover an important gap in primary islet physiology studies, providing a valuable tool we hope is useful to uncover basic beta cell metabolic mechanisms, as well as for translational investigations, such as pharmacological candidate discovery and islet transplantation protocols.

Research in context What is already known about this subject?O_LIPancreatic beta cells efficiently couple oxidative phosphorylation and ATP production with insulin secretion; mitochondrial ATP production is crucial for proper insulin secretion.
C_LIO_LIMost studies of beta cell respiration use cell lines instead of primary islets, which are a much more robust model to evaluate beta cell function.
C_LIO_LIThe few works with primary islet respiration use specific equipment and consumables that are expensive, complicated, and poorly reproducible.
C_LI

What is the key question?O_LIIs it possible to develop a practical method to evaluate metabolic fluxes and ATP production in isolated islets, using the standard Seahorse Extracellular Flux Technology?
C_LI

What are the new findings?O_LIWe optimized rodent islet isolation and functional analysis protocols using standard extracellular flux analysis equipment and consumables.
C_LIO_LIOur method allows for increased islet yield and robust islet respiration measurements.
C_LI

How might this impact on clinical practice in the foreseeable future?O_LIQuantitative measurements of metabolic fluxes and oxidative phosphorylation are the cornerstone of new discoveries in beta cells, and can contribute toward the establishment of new cellular protocols, such as for cell transplantation, as well as the development of new pharmacological agents targeted to these cells.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.550942">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.550942" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.550942">
        <p class="paperTitle">Crystal structure and biochemical activity of the macro domain from rubella virus p150</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.550942" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.550942" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stoll, G. A.; Zhang, L.; Modis, Y.</p>
        <p class="info">Score: 5.2, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.550942' target='https://doi.org/10.1101/2023.07.28.550942'> 10.1101/2023.07.28.550942</a></p>
        <p class="abstract">Rubella virus remains a global health threat. Rubella infections during pregnancy can cause serious congenital pathology and no antiviral treatments are available. Rubella virus encodes a nonstructural polyprotein with RNA polymerase, methyltransferase, and papain-like cysteine protease activities, along with a putative macro domain of unknown function. Macro domains bind ADP-ribose adducts, a post-translational modification that plays a key role in host-virus conflicts. Some macro domains can also remove the mono-ADP-ribose adduct or degrade poly-ADP-ribose chains. Here, we report high-resolution crystal structures of the macro domain from rubella virus nonstructural protein p150, with and without ADP-ribose bound. The overall fold is most similar to macroD-type macro domains from various nonviral species. The specific composition and structure of the residues that poised for catalysis or coordinate ADP-ribose in the rubella virus macro domain are most similar to those of macro domains from alphaviruses. Isothermal calorimetry and enzymatic assays show that the rubella virus macro domain binds ADP-ribose in solution and has mono-ADP-ribosylhydrolase (de-MARylation) activity.

IMPORTANCEOur work demonstrates that, like alpha- and coronaviruses, rubiviruses encode a mono-ADP-ribosylhydrolase with a structurally conserved macro domain fold to counteract MARylation by PARPs in the host innate immune response. Our structural data will guide future efforts to develop novel antiviral therapeutics against rubella or infections with related viruses.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
