<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>scientific communication and education</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>scientific communication and education | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Origami-based growing tube model for reproducing shell shapes
Authors: Ueda, M.; Fukunaga, N.; Yamashita, N.; Yokoyama, Y.; Kida, H.; Matsuda, K.
Score: 6.1, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549674
Mathematical models for shell morphology have been well-studied in theoretical morphology. The growing tube model proposed by Takashi Okamoto was developed to reproduce heteromorph ammonoids and can generate various shells by changing three parameters: curvature, torsion, and enlarging ratio. Previous studies in theoretical morphology have employed computers to visualize mathematical models.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/scientific-communication-and-education/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="scientific communication and education" />
<meta property="og:description" content="Origami-based growing tube model for reproducing shell shapes
Authors: Ueda, M.; Fukunaga, N.; Yamashita, N.; Yokoyama, Y.; Kida, H.; Matsuda, K.
Score: 6.1, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549674
Mathematical models for shell morphology have been well-studied in theoretical morphology. The growing tube model proposed by Takashi Okamoto was developed to reproduce heteromorph ammonoids and can generate various shells by changing three parameters: curvature, torsion, and enlarging ratio. Previous studies in theoretical morphology have employed computers to visualize mathematical models." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/scientific-communication-and-education/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-26T10:37:30+00:00" />
<meta property="article:modified_time" content="2023-07-26T10:37:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="scientific communication and education"/>
<meta name="twitter:description" content="Origami-based growing tube model for reproducing shell shapes
Authors: Ueda, M.; Fukunaga, N.; Yamashita, N.; Yokoyama, Y.; Kida, H.; Matsuda, K.
Score: 6.1, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549674
Mathematical models for shell morphology have been well-studied in theoretical morphology. The growing tube model proposed by Takashi Okamoto was developed to reproduce heteromorph ammonoids and can generate various shells by changing three parameters: curvature, torsion, and enlarging ratio. Previous studies in theoretical morphology have employed computers to visualize mathematical models."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "scientific communication and education",
      "item": "https://trxiv.yorks0n.com/posts/scientific-communication-and-education/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "scientific communication and education",
  "name": "scientific communication and education",
  "description": "Origami-based growing tube model for reproducing shell shapes\nAuthors: Ueda, M.; Fukunaga, N.; Yamashita, N.; Yokoyama, Y.; Kida, H.; Matsuda, K.\nScore: 6.1, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549674\nMathematical models for shell morphology have been well-studied in theoretical morphology. The growing tube model proposed by Takashi Okamoto was developed to reproduce heteromorph ammonoids and can generate various shells by changing three parameters: curvature, torsion, and enlarging ratio. Previous studies in theoretical morphology have employed computers to visualize mathematical models.",
  "keywords": [
    
  ],
  "articleBody": " Origami-based growing tube model for reproducing shell shapes\nAuthors: Ueda, M.; Fukunaga, N.; Yamashita, N.; Yokoyama, Y.; Kida, H.; Matsuda, K.\nScore: 6.1, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549674\nMathematical models for shell morphology have been well-studied in theoretical morphology. The growing tube model proposed by Takashi Okamoto was developed to reproduce heteromorph ammonoids and can generate various shells by changing three parameters: curvature, torsion, and enlarging ratio. Previous studies in theoretical morphology have employed computers to visualize mathematical models. However, computers are not accessible to everyone, and merely observing the graphics of the reproduced shells does not deepen understanding of the mathematical model. Therefore, in this study, we considered using origami, which is inexpensive and readily available, to make users understand mathematical models through hands-on folding. First, we simplified the crease pattern of seashells, originally devised by an origami artist Tomoko Fuse, to create a basic crease pattern. We then identified the crease pattern elements corresponding to the three parameters in the growing tube model and the aperture-apex position determiner. Based on them, we reproduced various shell shapes, including a heteromorph-ammonoid-like shape, with origami. Finally, we investigated the constraints between parameters in the origami-growing tube model. The origami-growing tube model is expected to help disseminate mathematical models and promote theoretical biology.\nIs biomedical research self-correcting? Modelling insights on the persistence of spurious science\nAuthors: Grimes, D. R.\nScore: 5.1, Published: 2023-07-18 DOI: 10.1101/2023.07.17.549436\nThe reality that volumes of published research are not reproducible has been increasingly recognised in recent years, notably in biomedical science. In many fields, spurious results are common, reducing trustworthiness of reported results. While this increases research waste, a common response is that science is ultimately self-correcting, and trustworthy science will eventually triumph. While this is likely true from a philosophy of science perspective, it does not yield information on how much effort is required to nullify suspect findings, nor factors that shape how quickly science may be correcting in the publish-or-perish environment scientists operate. There is also a paucity of information on how perverse incentives of the publishing ecosystem, which reward novel positive findings over null results, shaping the ability of published science to self-correct. Precisely what factors shape self-correction of science remain obscure, limiting our ability to mitigate harms. This modelling study illuminates these questions, introducing a simple model to capture dynamics of the publication ecosystem, exploring factors influencing research waste, trustworthiness, corrective effort, and time to correction. Results from this work indicate that research waste and corrective effort are highly dependent on field-specific false positive rates and the time delay before corrective results to spurious findings are propagated. The model also suggests conditions under which biomedical science is self-correcting, and those under which publication of correctives alone cannot stem the propagation of untrustworthy results. Finally, this work models a variety of potential mitigation strategies, including researcher and publication driven interventions. Significance statementIn biomedical science, there is increasing recognition that many results fail to replicate, impeding both scientific advances and trust in science. While science is self-correcting over long time-scales, there has been little work done on the factors that shape time to correction, the scale of corrective efforts, and the research waste generated in these endeavours. Similarly, there has been little work done on quantifying factors that might reduce negative impacts of spurious science. This work takes a modeling approach to illuminate these questions, uncovering new strategies for mitigating the impact of untrustworthy research.\nChatGPT identifies gender disparities in scientific peer review\nAuthors: Verharen, J. P. H.\nScore: 3.8, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549552\nThe peer review process is a critical step in ensuring the quality of scientific research. However, its subjectivity has raised concerns. To investigate this issue, I examined over 500 publicly available peer review reports from 200 published neuroscience papers in 2022-2023. OpenAIs generative artificial intelligence ChatGPT was used to analyze language use in these reports. This analysis found high levels of variability in how each reviewer scored the same paper, indicating the presence of subjectivity in the peer review process. The results also revealed that female first authors received less polite reviews than their male peers, indicating a gender bias in reviewing. Furthermore, published papers with a female senior author received more favorable reviews than papers with a male senior author, suggesting a gender disparity in academic publishing. This study highlights the potential of generative artificial intelligence in identifying areas of concern in scientific peer review and underscores the need to enhance transparency and objectivity in the scientific publishing process.\nDevelopment and Assessment of a Sustainable PhD Internship Program Supporting Diverse Biomedical Career Outcomes\nAuthors: Brandt, P. D.; Whittington, D.; Wood, K. D.; Holmquist, C. E.; Nogueira, A. T.; Gaines, C. H.; Brennwald, P. J.; Layton, R. L.\nScore: 16.4, Published: 2023-07-17 DOI: 10.1101/2023.07.13.548912\nA doctoral-level internship program was developed at the University of North Carolina at Chapel Hill with the intent to create customizable experiential learning opportunities for biomedical trainees to support career exploration, preparation, and transition into their post-graduate professional roles. We report the outcomes of this program over a five-year period. During that 5-year period, 123 internships took place at over 70 partner sites, representing at least 20 academic, for-profit, and non-profit career paths in the life sciences. A major goal of the program was to enhance trainees skill development and expertise in careers of interest. The benefits of the internship program for interns, host/employer, and supervisor/principal investigator were assessed using a mixed-methods approach, including surveys with closed- and open-ended responses as well as focus group interviews. Balancing stakeholder interests is key to creating a sustainable program with widespread support; hence, the level of support from internship hosts and faculty members were key metrics analyzed throughout. We hypothesized that once a successful internship program was implemented, faculty culture might shift to be more accepting of internships; indeed, the data quantifying faculty attitudes support this. Furthermore, host motivation and performance expectations of interns were compared with results achieved, and this data revealed both expected and surprising benefits to hosts. Data suggests a myriad of benefits for each stakeholder group, and themes are cataloged and discussed. Program outcomes, evaluation data, policies, resources, and best practices developed through the implementation of this program are shared to provide resources that facilitate the creation of similar internship programs at other institutions. Program development was initially spurred by National Institutes of Health pilot funding, thereafter, successfully transitioning from a grant-supported model, to an institutionally supported funding model to achieve long-term programmatic sustainability.\nUse of Large Language Models to Aid Analysis of Textual Data\nAuthors: Tai, R. H.; Bentley, L. R.; Xia, X.; Sitt, J. M.; Fankhauser, S. C.; Chicas-Mosier, A. M.; Monteith, B. G.\nScore: 1.2, Published: 2023-07-19 DOI: 10.1101/2023.07.17.549361\nThe increasing use of machine learning and Large Language Models (LLMs) opens up opportunities to use these artificially intelligent algorithms in novel ways. In this article, we propose a methodology using LLMs to support traditional deductive coding in qualitative research. We began our analysis with three different sample texts taken from existing interviews. Next, we created a codebook and inputted the sample text and codebook into an LLM. We asked the LLM to determine if the codes were present in a sample text provided and requested evidence to support the coding. The sample texts were inputted 160 times to record changes between iterations of the LLM response. Each iteration was analogous to a new coder deductively analyzing the given text with the codebook information. In our results, we present the outputs for these recursive analyses, along with a comparison of the LLM coding to evaluations made by human coders using traditional coding methods. We argue that LLM analysis can aid qualitative researchers by deductively coding transcripts, providing a systematic and reliable platform for code identification, and offering a means of avoiding analysis misalignment. Implications of using LLM in research praxis are discussed, along with current limitations.\nImpact Evaluation of Significant Feature Set in Cross Project for Defect Prediction through Hybrid Feature Selection in Multiclass\nAuthors: Aljaidi, M.; Gul, S.; Faiz, R.; Samara, G.; Alsarhan, A.; al-Qerem, A.\nScore: 1.1, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549868\nAbstractCross-project defect prediction (CPDP) is a significant way of defect identification in the project. In cross-project defect prediction, we extract knowledge from the source project and apply that learned knowledge to predict labels for the target project. However, the model performance can be affected by features that are insignificant and irrelevant. Hybrid feature selection (HFS) can play a significant role in achieving high prediction accuracy by selecting significant and only relevant features. Our aim is to explore effect of significant feature selection through a hybrid approach upon cross-project (CP) defect prediction for datasets which are multi-class in nature. We leveraged the strengths of Random Forest (RF) and Recursive Feature Elimination Cross Validation (RFECV) which can constructively select few features which are significant. The design of our controlled experiment is 1 Factor 2 Treatments (1F2T). Exploratory Data Analysis (EDA) proves that all versions of PROMISE repository are multi class and have duplicated rows in data, distribution gap among values, and imbalance classes. Hence after removing duplicated rows, reducing the gap present in distribution of data, and balancing classes, we selected significant feature set through Hybrid approach i.e. Random Forest (RF) and Recursive Feature Elimination Cross Validation (RFECV). We used Convolutional Neural Network (CNN) as a classifier to predict Cross project defects along with SoftMax as the last layer. Our experimental setup resulted in the average 78% prediction accuracy measure of all 14 versions in terms of AUC. Our experimental result showed that there is significant impact of HFS on defect prediction accuracy for different datasets in the CP.\nWhen Citizen Science meets teaching: A storified citizen science computer game to teach and contribute to genomic data analysis\nAuthors: Drogaris, C.; Butyaev, A.; Nazarova, E.; Sarrazin-Gendron, R.; Patel, H.; Singh, A.; Kadota, B.; Waldispuhl, J.\nScore: 9.9, Published: 2023-07-18 DOI: 10.1101/2022.05.23.492810\nIn the last decade, video games became a common vehicle for citizen science initiatives in life science, allowing participants to contribute to real scientific data analysis while learning about it. Since 2010, our scientific discovery game (SDG) Phylo enlists participants in comparative genomic data analysis. It is frequently used as a learning tool, but the activities were difficult to aggregate to build a coherent teaching activity. Here, we describe a strategy and series of recipes to facilitate the integration of SDGs in courses and implement this approach in Phylo. We developed new roles and functionalities enabling instructors to create assignments and monitor the progress of students. A story mode progressively introduces comparative genomics concepts, allowing users to learn and contribute to the analysis of real genomic sequences. Preliminary results from a user study suggest this framework may help to boost user motivation and clarify pedagogical objectives.\nSingle Cell Portal: an interactive home for single-cell genomics data\nAuthors: Tarhan, L.; Bistline, J.; Chang, J.; Galloway, B.; Hanna, E.; Weitz, E.\nScore: 2.6, Published: 2023-07-17 DOI: 10.1101/2023.07.13.548886\nSingle-cell omics research has the power to leave a deep impact on modern healthcare. Sharing data widely and freely advances this progress in both the academic and clinical spheres. We developed the Single Cell Portal (SCP) to maximize the impact of this work. SCP enables data sharing, supports dynamic results visualization, and facilitates scientific exploration across a large repository of single-cell datasets. SCPs data contributors maintain full control over how their data are shared and presented, without requiring web development expertise. Finally, SCP supports the entire lifecycle of a research project, from sparking an idea, to fine-tuning the data with collaborators, to sharing results in an accessible and interactive way. This paper highlights the most valuable ways in which SCP helps to advance single-cell research.\n",
  "wordCount" : "1947",
  "inLanguage": "en",
  "datePublished": "2023-07-26T10:37:30Z",
  "dateModified": "2023-07-26T10:37:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/scientific-communication-and-education/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      scientific communication and education
    </h1>
    <div class="post-meta"><span>updated on July 26, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549674">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549674" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549674">
        <p class="paperTitle">Origami-based growing tube model for reproducing shell shapes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549674" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549674" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ueda, M.; Fukunaga, N.; Yamashita, N.; Yokoyama, Y.; Kida, H.; Matsuda, K.</p>
        <p class="info">Score: 6.1, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549674' target='https://doi.org/10.1101/2023.07.19.549674'> 10.1101/2023.07.19.549674</a></p>
        <p class="abstract">Mathematical models for shell morphology have been well-studied in theoretical morphology. The growing tube model proposed by Takashi Okamoto was developed to reproduce heteromorph ammonoids and can generate various shells by changing three parameters: curvature, torsion, and enlarging ratio. Previous studies in theoretical morphology have employed computers to visualize mathematical models. However, computers are not accessible to everyone, and merely observing the graphics of the reproduced shells does not deepen understanding of the mathematical model. Therefore, in this study, we considered using origami, which is inexpensive and readily available, to make users understand mathematical models through hands-on folding. First, we simplified the crease pattern of seashells, originally devised by an origami artist Tomoko Fuse, to create a basic crease pattern. We then identified the crease pattern elements corresponding to the three parameters in the growing tube model and the aperture-apex position determiner. Based on them, we reproduced various shell shapes, including a heteromorph-ammonoid-like shape, with origami. Finally, we investigated the constraints between parameters in the origami-growing tube model. The origami-growing tube model is expected to help disseminate mathematical models and promote theoretical biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549436">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549436" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549436">
        <p class="paperTitle">Is biomedical research self-correcting? Modelling insights on the persistence of spurious science</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549436" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549436" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Grimes, D. R.</p>
        <p class="info">Score: 5.1, Published: 2023-07-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549436' target='https://doi.org/10.1101/2023.07.17.549436'> 10.1101/2023.07.17.549436</a></p>
        <p class="abstract">The reality that volumes of published research are not reproducible has been increasingly recognised in recent years, notably in biomedical science. In many fields, spurious results are common, reducing trustworthiness of reported results. While this increases research waste, a common response is that science is ultimately self-correcting, and trustworthy science will eventually triumph. While this is likely true from a philosophy of science perspective, it does not yield information on how much effort is required to nullify suspect findings, nor factors that shape how quickly science may be correcting in the publish-or-perish environment scientists operate. There is also a paucity of information on how perverse incentives of the publishing ecosystem, which reward novel positive findings over null results, shaping the ability of published science to self-correct. Precisely what factors shape self-correction of science remain obscure, limiting our ability to mitigate harms. This modelling study illuminates these questions, introducing a simple model to capture dynamics of the publication ecosystem, exploring factors influencing research waste, trustworthiness, corrective effort, and time to correction. Results from this work indicate that research waste and corrective effort are highly dependent on field-specific false positive rates and the time delay before corrective results to spurious findings are propagated. The model also suggests conditions under which biomedical science is self-correcting, and those under which publication of correctives alone cannot stem the propagation of untrustworthy results. Finally, this work models a variety of potential mitigation strategies, including researcher and publication driven interventions.

Significance statementIn biomedical science, there is increasing recognition that many results fail to replicate, impeding both scientific advances and trust in science. While science is self-correcting over long time-scales, there has been little work done on the factors that shape time to correction, the scale of corrective efforts, and the research waste generated in these endeavours. Similarly, there has been little work done on quantifying factors that might reduce negative impacts of spurious science. This work takes a modeling approach to illuminate these questions, uncovering new strategies for mitigating the impact of untrustworthy research.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549552">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549552" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549552">
        <p class="paperTitle">ChatGPT identifies gender disparities in scientific peer review</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549552" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549552" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Verharen, J. P. H.</p>
        <p class="info">Score: 3.8, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549552' target='https://doi.org/10.1101/2023.07.18.549552'> 10.1101/2023.07.18.549552</a></p>
        <p class="abstract">The peer review process is a critical step in ensuring the quality of scientific research. However, its subjectivity has raised concerns. To investigate this issue, I examined over 500 publicly available peer review reports from 200 published neuroscience papers in 2022-2023. OpenAIs generative artificial intelligence ChatGPT was used to analyze language use in these reports. This analysis found high levels of variability in how each reviewer scored the same paper, indicating the presence of subjectivity in the peer review process. The results also revealed that female first authors received less polite reviews than their male peers, indicating a gender bias in reviewing. Furthermore, published papers with a female senior author received more favorable reviews than papers with a male senior author, suggesting a gender disparity in academic publishing. This study highlights the potential of generative artificial intelligence in identifying areas of concern in scientific peer review and underscores the need to enhance transparency and objectivity in the scientific publishing process.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.548912">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.548912" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.548912">
        <p class="paperTitle">Development and Assessment of a Sustainable PhD Internship Program Supporting Diverse Biomedical Career Outcomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.548912" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.548912" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brandt, P. D.; Whittington, D.; Wood, K. D.; Holmquist, C. E.; Nogueira, A. T.; Gaines, C. H.; Brennwald, P. J.; Layton, R. L.</p>
        <p class="info">Score: 16.4, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.548912' target='https://doi.org/10.1101/2023.07.13.548912'> 10.1101/2023.07.13.548912</a></p>
        <p class="abstract">A doctoral-level internship program was developed at the University of North Carolina at Chapel Hill with the intent to create customizable experiential learning opportunities for biomedical trainees to support career exploration, preparation, and transition into their post-graduate professional roles. We report the outcomes of this program over a five-year period. During that 5-year period, 123 internships took place at over 70 partner sites, representing at least 20 academic, for-profit, and non-profit career paths in the life sciences. A major goal of the program was to enhance trainees skill development and expertise in careers of interest. The benefits of the internship program for interns, host/employer, and supervisor/principal investigator were assessed using a mixed-methods approach, including surveys with closed- and open-ended responses as well as focus group interviews. Balancing stakeholder interests is key to creating a sustainable program with widespread support; hence, the level of support from internship hosts and faculty members were key metrics analyzed throughout. We hypothesized that once a successful internship program was implemented, faculty culture might shift to be more accepting of internships; indeed, the data quantifying faculty attitudes support this. Furthermore, host motivation and performance expectations of interns were compared with results achieved, and this data revealed both expected and surprising benefits to hosts. Data suggests a myriad of benefits for each stakeholder group, and themes are cataloged and discussed. Program outcomes, evaluation data, policies, resources, and best practices developed through the implementation of this program are shared to provide resources that facilitate the creation of similar internship programs at other institutions. Program development was initially spurred by National Institutes of Health pilot funding, thereafter, successfully transitioning from a grant-supported model, to an institutionally supported funding model to achieve long-term programmatic sustainability.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549361">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549361" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549361">
        <p class="paperTitle">Use of Large Language Models to Aid Analysis of Textual Data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549361" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549361" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tai, R. H.; Bentley, L. R.; Xia, X.; Sitt, J. M.; Fankhauser, S. C.; Chicas-Mosier, A. M.; Monteith, B. G.</p>
        <p class="info">Score: 1.2, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549361' target='https://doi.org/10.1101/2023.07.17.549361'> 10.1101/2023.07.17.549361</a></p>
        <p class="abstract">The increasing use of machine learning and Large Language Models (LLMs) opens up opportunities to use these artificially intelligent algorithms in novel ways. In this article, we propose a methodology using LLMs to support traditional deductive coding in qualitative research. We began our analysis with three different sample texts taken from existing interviews. Next, we created a codebook and inputted the sample text and codebook into an LLM. We asked the LLM to determine if the codes were present in a sample text provided and requested evidence to support the coding. The sample texts were inputted 160 times to record changes between iterations of the LLM response. Each iteration was analogous to a new coder deductively analyzing the given text with the codebook information. In our results, we present the outputs for these recursive analyses, along with a comparison of the LLM coding to evaluations made by human coders using traditional coding methods. We argue that LLM analysis can aid qualitative researchers by deductively coding transcripts, providing a systematic and reliable platform for code identification, and offering a means of avoiding analysis misalignment. Implications of using LLM in research praxis are discussed, along with current limitations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549868">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549868" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549868">
        <p class="paperTitle">Impact Evaluation of Significant Feature Set in Cross Project for Defect Prediction through Hybrid Feature Selection in Multiclass</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549868" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549868" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aljaidi, M.; Gul, S.; Faiz, R.; Samara, G.; Alsarhan, A.; al-Qerem, A.</p>
        <p class="info">Score: 1.1, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549868' target='https://doi.org/10.1101/2023.07.20.549868'> 10.1101/2023.07.20.549868</a></p>
        <p class="abstract">AbstractCross-project defect prediction (CPDP) is a significant way of defect identification in the project. In cross-project defect prediction, we extract knowledge from the source project and apply that learned knowledge to predict labels for the target project. However, the model performance can be affected by features that are insignificant and irrelevant. Hybrid feature selection (HFS) can play a significant role in achieving high prediction accuracy by selecting significant and only relevant features. Our aim is to explore effect of significant feature selection through a hybrid approach upon cross-project (CP) defect prediction for datasets which are multi-class in nature. We leveraged the strengths of Random Forest (RF) and Recursive Feature Elimination Cross Validation (RFECV) which can constructively select few features which are significant. The design of our controlled experiment is 1 Factor 2 Treatments (1F2T). Exploratory Data Analysis (EDA) proves that all versions of PROMISE repository are multi class and have duplicated rows in data, distribution gap among values, and imbalance classes. Hence after removing duplicated rows, reducing the gap present in distribution of data, and balancing classes, we selected significant feature set through Hybrid approach i.e. Random Forest (RF) and Recursive Feature Elimination Cross Validation (RFECV). We used Convolutional Neural Network (CNN) as a classifier to predict Cross project defects along with SoftMax as the last layer. Our experimental setup resulted in the average 78% prediction accuracy measure of all 14 versions in terms of AUC. Our experimental result showed that there is significant impact of HFS on defect prediction accuracy for different datasets in the CP.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.05.23.492810">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.05.23.492810" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.05.23.492810">
        <p class="paperTitle">When Citizen Science meets teaching: A storified citizen science computer game to teach and contribute to genomic data analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.05.23.492810" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.05.23.492810" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Drogaris, C.; Butyaev, A.; Nazarova, E.; Sarrazin-Gendron, R.; Patel, H.; Singh, A.; Kadota, B.; Waldispuhl, J.</p>
        <p class="info">Score: 9.9, Published: 2023-07-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.05.23.492810' target='https://doi.org/10.1101/2022.05.23.492810'> 10.1101/2022.05.23.492810</a></p>
        <p class="abstract">In the last decade, video games became a common vehicle for citizen science initiatives in life science, allowing participants to contribute to real scientific data analysis while learning about it. Since 2010, our scientific discovery game (SDG) Phylo enlists participants in comparative genomic data analysis. It is frequently used as a learning tool, but the activities were difficult to aggregate to build a coherent teaching activity. Here, we describe a strategy and series of recipes to facilitate the integration of SDGs in courses and implement this approach in Phylo. We developed new roles and functionalities enabling instructors to create assignments and monitor the progress of students. A story mode progressively introduces comparative genomics concepts, allowing users to learn and contribute to the analysis of real genomic sequences. Preliminary results from a user study suggest this framework may help to boost user motivation and clarify pedagogical objectives.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.548886">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.548886" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.548886">
        <p class="paperTitle">Single Cell Portal: an interactive home for single-cell genomics data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.548886" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.548886" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tarhan, L.; Bistline, J.; Chang, J.; Galloway, B.; Hanna, E.; Weitz, E.</p>
        <p class="info">Score: 2.6, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.548886' target='https://doi.org/10.1101/2023.07.13.548886'> 10.1101/2023.07.13.548886</a></p>
        <p class="abstract">Single-cell omics research has the power to leave a deep impact on modern healthcare. Sharing data widely and freely advances this progress in both the academic and clinical spheres. We developed the Single Cell Portal (SCP) to maximize the impact of this work. SCP enables data sharing, supports dynamic results visualization, and facilitates scientific exploration across a large repository of single-cell datasets. SCPs data contributors maintain full control over how their data are shared and presented, without requiring web development expertise. Finally, SCP supports the entire lifecycle of a research project, from sparking an idea, to fine-tuning the data with collaborators, to sharing results in an accessible and interactive way. This paper highlights the most valuable ways in which SCP helps to advance single-cell research.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
