<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>health informatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>health informatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Exploratory electronic health record analysis with ehrapy
Authors: Heumos, L.; Ehmele, P.; Treis, T.; Upmeier zu Belzen, J.; Namsaraeva, A.; Horlava, N.; Shitov, V. A.; Zhang, X.; Zappia, L.; Knoll, R.; Lang, N. J.; Hetzel, L.; Virshup, I.; Sikkema, L.; Roellin, E.; Curion, F.; Eils, R.; Schiller, H. B.; Hilgendorff, A.; Theis, F.
Score: 52.8, Published: 2023-12-15 DOI: 10.1101/2023.12.11.23299816
With progressive digitalization of healthcare systems worldwide, large-scale collection of electronic health records (EHRs) has become commonplace.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/health-informatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="health informatics" />
<meta property="og:description" content="Exploratory electronic health record analysis with ehrapy
Authors: Heumos, L.; Ehmele, P.; Treis, T.; Upmeier zu Belzen, J.; Namsaraeva, A.; Horlava, N.; Shitov, V. A.; Zhang, X.; Zappia, L.; Knoll, R.; Lang, N. J.; Hetzel, L.; Virshup, I.; Sikkema, L.; Roellin, E.; Curion, F.; Eils, R.; Schiller, H. B.; Hilgendorff, A.; Theis, F.
Score: 52.8, Published: 2023-12-15 DOI: 10.1101/2023.12.11.23299816
With progressive digitalization of healthcare systems worldwide, large-scale collection of electronic health records (EHRs) has become commonplace." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/health-informatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-27T10:39:30+00:00" />
<meta property="article:modified_time" content="2023-12-27T10:39:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="health informatics"/>
<meta name="twitter:description" content="Exploratory electronic health record analysis with ehrapy
Authors: Heumos, L.; Ehmele, P.; Treis, T.; Upmeier zu Belzen, J.; Namsaraeva, A.; Horlava, N.; Shitov, V. A.; Zhang, X.; Zappia, L.; Knoll, R.; Lang, N. J.; Hetzel, L.; Virshup, I.; Sikkema, L.; Roellin, E.; Curion, F.; Eils, R.; Schiller, H. B.; Hilgendorff, A.; Theis, F.
Score: 52.8, Published: 2023-12-15 DOI: 10.1101/2023.12.11.23299816
With progressive digitalization of healthcare systems worldwide, large-scale collection of electronic health records (EHRs) has become commonplace."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "health informatics",
      "item": "https://trxiv.yorks0n.com/posts/health-informatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "health informatics",
  "name": "health informatics",
  "description": "Exploratory electronic health record analysis with ehrapy\nAuthors: Heumos, L.; Ehmele, P.; Treis, T.; Upmeier zu Belzen, J.; Namsaraeva, A.; Horlava, N.; Shitov, V. A.; Zhang, X.; Zappia, L.; Knoll, R.; Lang, N. J.; Hetzel, L.; Virshup, I.; Sikkema, L.; Roellin, E.; Curion, F.; Eils, R.; Schiller, H. B.; Hilgendorff, A.; Theis, F.\nScore: 52.8, Published: 2023-12-15 DOI: 10.1101/2023.12.11.23299816\nWith progressive digitalization of healthcare systems worldwide, large-scale collection of electronic health records (EHRs) has become commonplace.",
  "keywords": [
    
  ],
  "articleBody": " Exploratory electronic health record analysis with ehrapy\nAuthors: Heumos, L.; Ehmele, P.; Treis, T.; Upmeier zu Belzen, J.; Namsaraeva, A.; Horlava, N.; Shitov, V. A.; Zhang, X.; Zappia, L.; Knoll, R.; Lang, N. J.; Hetzel, L.; Virshup, I.; Sikkema, L.; Roellin, E.; Curion, F.; Eils, R.; Schiller, H. B.; Hilgendorff, A.; Theis, F.\nScore: 52.8, Published: 2023-12-15 DOI: 10.1101/2023.12.11.23299816\nWith progressive digitalization of healthcare systems worldwide, large-scale collection of electronic health records (EHRs) has become commonplace. However, an extensible framework for comprehensive exploratory analysis that accounts for data heterogeneity is missing. Here, we introduce ehrapy, a modular open-source Python framework designed for exploratory end-to-end analysis of heterogeneous epidemiology and electronic health record data. Ehrapy incorporates a series of analytical steps, from data extraction and quality control to the generation of low-dimensional representations. Complemented by rich statistical modules, ehrapy facilitates associating patients with disease states, differential comparison between patient clusters, survival analysis, trajectory inference, causal inference, and more. Leveraging ontologies, ehrapy further enables data sharing and training EHR deep learning models paving the way for foundational models in biomedical research. We demonstrated ehrapys features in five distinct examples: We first applied ehrapy to stratify patients affected by unspecified pneumonia into finer-grained phenotypes. Furthermore, we revealed biomarkers for significant differences in survival among these groups. Additionally, we quantify medication-class effects of pneumonia medications on length of stay. We further leveraged ehrapy to analyze cardiovascular risks across different data modalities. Finally, we reconstructed disease state trajectories in SARS-CoV-2 patients based on imaging data. Ehrapy thus provides a framework that we envision will standardize analysis pipelines on EHR data and serve as a cornerstone for the community.\nSARS-CoV-2 Infection is Associated with an Increase in New Diagnoses of Schizophrenia Spectrum and Psychotic Disorder: A Study Using the US National COVID Cohort Collaborative (N3C)\nAuthors: Rahman, A.; Russell, M.; Zheng, W.; Eckrich, D.; Ahmed, I.; on behalf of the N3C Consortium, Score: 704.2, Published: 2023-12-05 DOI: 10.1101/2023.12.05.23299473\nAmid the ongoing global repercussions of SARS-CoV-2, its crucial to comprehend its potential long-term psychiatric effects. Several recent studies have suggested a link between COVID-19 and subsequent mental health disorders. Our investigation joins this exploration, concentrating on Schizophrenia Spectrum and Psychotic Disorders (SSPD). Different from other studies, we took acute respiratory distress syndrome (ARDS) and COVID-19 lab negative cohorts as control groups to accurately gauge the impact of COVID-19 on SSPD. Data from 19,344,698 patients, sourced from the N3C Data Enclave platform, were methodically filtered to create propensity matched cohorts: ARDS (n = 222,337), COVID-positive (n = 219,264), and COVID-negative (n = 213,183). We systematically analyzed the hazard rate of new-onset SSPD across three distinct time intervals: 0-21 days, 22-90 days, and beyond 90 days post-infection. COVID-19 positive patients consistently exhibited a heightened hazard ratio (HR) across all intervals [0-21 days (HR: 4.6; CI: 3.7-5.7), 22-90 days (HR: 2.9; CI: 2.3 -3.8), beyond 90 days (HR: 1.7; CI: 1.5-1.)]. These are notably higher than both ARDS and COVID-19 lab-negative patients. Validations using various tests, including the Cochran Mantel Haenszel Test, Wald Test, and Log-rank Test confirmed these associations. Intriguingly, our data indicated that younger individuals face a heightened risk of SSPD after contracting COVID-19, a trend not observed in the ARDS and COVID-negative groups. These results, aligned with the known neurotropism of SARS-CoV-2 and earlier studies, accentuate the need for vigilant psychiatric assessment and support in the era of Long-COVID, especially among younger populations.\nThe molecular landscape of premature aging diseases defined by multilayer network exploration\nAuthors: Beust, C.; Valdeolivas, A.; Baptista, A.; Briere, G.; Levy, N.; Ozisik, O.; Baudot, A.\nScore: 4.8, Published: 2023-12-19 DOI: 10.1101/2023.12.19.23300210\nPremature Aging (PA) diseases are rare genetic disorders that mimic some aspects of physiological aging at an early age. Various causative genes of PA diseases have been identified in recent years, providing insights into some dysfunctional cellular functions. However, the identification of PA genes also revealed significant genetic heterogeneity and highlighted the gaps in our understanding of PA molecular mechanisms. Furthermore, many patients remain undiagnosed. Overall, the current lack of knowledge about PA diseases hinders the development of effective diagnosis and therapies and poses significant challenges to improving patient care. Here, we present a network-based approach to systematically unravel the cellular functions disrupted in PA diseases. Leveraging a novel community identification algorithm, we delved into a vast multilayer network of biological interactions to extract the communities of 67 PA diseases from their 132 associated genes. We found that these communities can be grouped into six distinct clusters, each reflecting specific cellular functions: DNA repair, cell cycle, transcription regulation, inflammation, cell communication, and vesicle-mediated transport. We propose that these clusters collectively represent the landscape of the molecular mechanisms that are perturbed in PA diseases, providing a framework for better understanding their pathogenesis. Intriguingly, most clusters also exhibited a significant enrichment in genes associated with physiological aging, suggesting a potential overlap between the molecular underpinnings of PA diseases and natural aging.\nA deep learning transformer model predicts high rates of undiagnosed rare disease in large electronic health systems\nAuthors: Jordan, D. M.; Vy, H. M. T.; Do, R.\nScore: 3.4, Published: 2023-12-24 DOI: 10.1101/2023.12.21.23300393\nIt is estimated that as many as 1 in 16 people worldwide suffer from rare diseases. Rare disease patients face difficulty finding diagnosis and treatment for their conditions, including long diagnostic odysseys, multiple incorrect diagnoses, and unavailable or prohibitively expensive treatments. As a result, it is likely that large electronic health record (EHR) systems include high numbers of participants suffering from undiagnosed rare disease. While this has been shown in detail for specific diseases, these studies are expensive and time consuming and have only been feasible to perform for a handful of the thousands of known rare diseases. The bulk of these undiagnosed cases are effectively hidden, with no straightforward way to differentiate them from healthy controls. The ability to access them at scale would enormously expand our capacity to study and develop drugs for rare diseases, adding to tools aimed at increasing availability of study cohorts for rare disease. In this study, we train a deep learning transformer algorithm, RarePT (Rare-Phenotype Prediction Transformer), to impute undiagnosed rare disease from EHR diagnosis codes in 436,407 participants in the UK Biobank and validated on an independent cohort from 3,333,560 individuals from the Mount Sinai Health System. We applied our model to 155 rare diagnosis codes with fewer than 250 cases each in the UK Biobank and predicted participants with elevated risk for each diagnosis, with the number of participants predicted to be at risk ranging from 85 to 22,000 for different diagnoses. These risk predictions are significantly associated with increased mortality for 65% of diagnoses, with disease burden expressed as disability-adjusted life years (DALY) for 73% of diagnoses, and with 72% of available disease-specific diagnostic tests. They are also highly enriched for known rare diagnoses in patients not included in the training set, with an odds ratio (OR) of 48.0 in cross-validation cohorts of the UK Biobank and an OR of 30.6 in the independent Mount Sinai Health System cohort. Most importantly, RarePT successfully screens for undiagnosed patients in 32 rare diseases with available diagnostic tests in the UK Biobank. Using the trained model to estimate the prevalence of undiagnosed disease in the UK Biobank for these 32 rare phenotypes, we find that at least 50% of patients remain undiagnosed for 20 of 32 diseases. These estimates provide empirical evidence of a high prevalence of undiagnosed rare disease, as well as demonstrating the enormous potential benefit of using RarePT to screen for undiagnosed rare disease patients in large electronic health systems.\nMachine learning in medicine using JavaScript: building web apps using TensorFlow.js for interpreting biomedical datasets\nAuthors: Pires, J. G.\nScore: 5.9, Published: 2023-12-21 DOI: 10.1101/2023.06.21.23291717\nContributions to medicine may come from different areas, and most of these areas are filled with researchers eager to contribute. In this paper, we aim to contribute through the intersection of machine learning and web development. We employed TensorFlow.js, a JavaScript-based library, to model biomedical datasets using neural networks obtained from Kaggle. The principal aim of this study is to present the capabilities of TensorFlow.js and promote its utility in the development of sophisticated machine learning models customized for web-based applications. We modeled three datasets: diabetes detection, surgery complications, and heart failure. While Python and R currently dominate, JavaScript and its derivatives are rapidly gaining ground, offering comparable performance and additional features associated with JavaScript. Kaggle, the public platform from which we downloaded our datasets, provides an extensive collection of biomedical datasets. Therefore, readers can easily test our discussed methods by using the provided codes with minor adjustments on any case of their interest. The results demonstrate an accuracy of 92% for diabetes detection, almost 100% for surgery complications, and 80% for heart failure. The possibilities are vast, and we believe that this is an excellent option for researchers focusing on web applications, particularly in the field of medicine.\nThe Applications of Laser Interstitial Thermal Therapy and Machine Learning in Neurosurgery: A Systematic Review\nAuthors: Bouras, A.; Patel, D.; Chetla, N.\nScore: 1.4, Published: 2023-12-26 DOI: 10.1101/2023.12.21.23300384\nBackground: The incorporation of Machine Learning (ML) into Laser Interstitial Thermal Therapy (LITT) represents a significant advancement in minimally invasive neurosurgery, partic- ularly for treating brain tumors, vascular malformations, and epileptogenic foci. This systematic review focuses on evaluating the integration and impact of ML in enhancing the efficacy, precision, and outcomes of LITT in neurosurgical procedures. Methods: An exhaustive search was conducted in major scientific databases for studies from 2015 to 2023 that specifically focused on the application of ML in LITT. The review assessed the development and implementation of ML algorithms in surgical planning, outcome prediction, and postoperative evaluation in LITT. Rigorous inclusion criteria were applied to select studies, and a combination of meta-analysis and qualitative synthesis was used to analyze the data. Results: The review synthesizes findings from a range of studies, including retrospective anal- yses and initial clinical trials. It highlights the role of ML in enhancing the selection criteria for LITT, optimizing surgical approaches, and improving patient-specific outcome predictions. While LITT showed favorable results in treating non-resectable lesions, the integration of ML was found to potentially refine these outcomes further. However, challenges such as the need for larger sample sizes, standardization of ML algorithms, and validation of these methods in clinical settings were noted. Conclusions: The integration of ML into LITT procedures marks a promising frontier in neurosurgery, offering potential improvements in surgical accuracy and patient outcomes. The evidence suggests a need for continued development and rigorous testing of ML applications in LITT. Future research should focus on the refinement and validation of ML algorithms for wider clinical adoption, ensuring that technological advancements align with patient safety and treatment efficacy.\nFRISTS: INTERPRETABLE TIME SERIES-BASED HEART FAILURE RISK PREDICTION\nAuthors: Lin, S.; Dong, X.; Wang, F.\nScore: 1.1, Published: 2023-12-22 DOI: 10.1101/2023.12.20.23299867\nAO_SCPLOWBSTRACTC_SCPLOWHeart failure is an incurable medical condition that affects millions of people globally. Developing prediction models is crucial to prevent patients from progressing to heart failure. Current heart failure prediction models struggle with achieving interpretability, a mandatory trait that allows the model to be applied in healthcare, while possessing high real-world accuracy. We introduce FRISTS, a novel HF prediction approach leveraging sequential times series-based modeling, such as long short-term memory (LSTM) networks, and feature selection. Our study utilized an extensive electronic health record (EHR) database and found that FRISTS outperformed traditional AI models (e.g., Random Forest Classifier, Logistic Regression, and Decision Tree) and more complex machine learning techniques (e.g., XGBoost, LSTM), yielding an average F1 score of 0.805 and receiver operating characteristic area under the curve (ROC AUC) value of 0.990. Our SHAP-inspired permutation method enables interpretation of the feature ranking, enhancing the result transparency and paving the way for a new class of interpretable models. This approach holds promise in enhancing clinical decision-making and patient care in the context of heart failure prevention.\nAugmentation of ChatGPT with Clinician-Informed Tools Improves Performance on Medical Calculation Tasks\nAuthors: Goodell, A. J.; Chu, S. N.; Rouholiman, D.; Chu, L. F.\nScore: 2.6, Published: 2023-12-15 DOI: 10.1101/2023.12.13.23299881\nAO_SCPLOWBSTRACTC_SCPLOWPrior work has shown that large language models (LLMs) have the ability to answer expert-level multiple choice questions in medicine, but are limited by both their tendency to hallucinate knowledge and their inherent inadequacy in performing basic mathematical operations. Unsurprisingly, early evidence suggests that LLMs perform poorly when asked to execute common clinical calculations. Recently, it has been demonstrated that LLMs have the capability of interacting with external programs and tools, presenting a possible remedy for this limitation. In this study, we explore the ability of ChatGPT (GPT-4, November 2023) to perform medical calculations, evaluating its performance across 48 diverse clinical calculation tasks. Our findings indicate that ChatGPT is an unreliable clinical calculator, delivering inaccurate responses in one-third of trials (n=212). To address this, we developed an open-source clinical calculation API (openmedcalc.org), which we then integrated with ChatGPT. We subsequently evaluated the performance of this augmented model by comparing it against standard ChatGPT using 75 clinical vignettes in three common clinical calculation tasks: Caprini VTE Risk, Wells DVT Criteria, and MELD-Na. The augmented model demonstrated a marked improvement in accuracy over unimproved ChatGPT. Our findings suggest that integration of machine-usable, clinician-informed tools can help alleviate the reliability limitations observed in medical LLMs.\nSARS-CoV-2 Orphan Gene ORF10 Contributes to More Severe COVID-19 Disease\nAuthors: Haltom, J. A.; Trovao, N. S.; Guarnieri, J.; Vincent, P.; Singh, U.; Tsoy, S.; O'Leary, C. A.; Bram, Y.; Widjaja, G. A.; Cen, Z.; Meller, R.; Baylin, S. B.; Moss, W. N.; Nikolau, B. J.; Enguita, F. J.; Wallace, D. C.; Beheshti, A.; Schwartz, R.; Wurtele, E. S.\nScore: 25.7, Published: 2023-11-29 DOI: 10.1101/2023.11.27.23298847\nThe orphan gene of SARS-CoV-2, ORF10, is the least studied gene in the virus responsible for the COVID-19 pandemic. Recent experimentation indicated ORF10 expression moderates innate immunity in vitro. However, whether ORF10 affects COVID-19 in humans remained unknown. We determine that the ORF10 sequence is identical to the Wuhan-Hu-1 ancestral haplotype in 95% of genomes across five variants of concern (VOC). Four ORF10 variants are associated with less virulent clinical outcomes in the human host: three of these affect ORF10 protein structure, one affects ORF10 RNA structural dynamics. RNA-Seq data from 2070 samples from diverse human cells and tissues reveals ORF10 accumulation is conditionally discordant from that of other SARS-CoV-2 transcripts. Expression of ORF10 in A549 and HEK293 cells perturbs immune-related gene expression networks, alters expression of the majority of mitochondrially-encoded genes of oxidative respiration, and leads to large shifts in levels of 14 newly-identified transcripts. We conclude ORF10 contributes to more severe COVID-19 clinical outcomes in the human host.\n",
  "wordCount" : "2439",
  "inLanguage": "en",
  "datePublished": "2023-12-27T10:39:30Z",
  "dateModified": "2023-12-27T10:39:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/health-informatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      health informatics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 27, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.11.23299816">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.11.23299816" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.11.23299816">
        <p class="paperTitle">Exploratory electronic health record analysis with ehrapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.11.23299816" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.11.23299816" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Heumos, L.; Ehmele, P.; Treis, T.; Upmeier zu Belzen, J.; Namsaraeva, A.; Horlava, N.; Shitov, V. A.; Zhang, X.; Zappia, L.; Knoll, R.; Lang, N. J.; Hetzel, L.; Virshup, I.; Sikkema, L.; Roellin, E.; Curion, F.; Eils, R.; Schiller, H. B.; Hilgendorff, A.; Theis, F.</p>
        <p class="info">Score: 52.8, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.11.23299816' target='https://doi.org/10.1101/2023.12.11.23299816'> 10.1101/2023.12.11.23299816</a></p>
        <p class="abstract">With progressive digitalization of healthcare systems worldwide, large-scale collection of electronic health records (EHRs) has become commonplace. However, an extensible framework for comprehensive exploratory analysis that accounts for data heterogeneity is missing. Here, we introduce ehrapy, a modular open-source Python framework designed for exploratory end-to-end analysis of heterogeneous epidemiology and electronic health record data. Ehrapy incorporates a series of analytical steps, from data extraction and quality control to the generation of low-dimensional representations. Complemented by rich statistical modules, ehrapy facilitates associating patients with disease states, differential comparison between patient clusters, survival analysis, trajectory inference, causal inference, and more. Leveraging ontologies, ehrapy further enables data sharing and training EHR deep learning models paving the way for foundational models in biomedical research. We demonstrated ehrapys features in five distinct examples: We first applied ehrapy to stratify patients affected by unspecified pneumonia into finer-grained phenotypes. Furthermore, we revealed biomarkers for significant differences in survival among these groups. Additionally, we quantify medication-class effects of pneumonia medications on length of stay. We further leveraged ehrapy to analyze cardiovascular risks across different data modalities. Finally, we reconstructed disease state trajectories in SARS-CoV-2 patients based on imaging data. Ehrapy thus provides a framework that we envision will standardize analysis pipelines on EHR data and serve as a cornerstone for the community.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.23299473">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.23299473" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.23299473">
        <p class="paperTitle">SARS-CoV-2 Infection is Associated with an Increase in New Diagnoses of Schizophrenia Spectrum and Psychotic Disorder: A Study Using the US National COVID Cohort Collaborative (N3C)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.23299473" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.23299473" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rahman, A.; Russell, M.; Zheng, W.; Eckrich, D.; Ahmed, I.; on behalf of the N3C Consortium,  </p>
        <p class="info">Score: 704.2, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.23299473' target='https://doi.org/10.1101/2023.12.05.23299473'> 10.1101/2023.12.05.23299473</a></p>
        <p class="abstract">Amid the ongoing global repercussions of SARS-CoV-2, its crucial to comprehend its potential long-term psychiatric effects. Several recent studies have suggested a link between COVID-19 and subsequent mental health disorders. Our investigation joins this exploration, concentrating on Schizophrenia Spectrum and Psychotic Disorders (SSPD). Different from other studies, we took acute respiratory distress syndrome (ARDS) and COVID-19 lab negative cohorts as control groups to accurately gauge the impact of COVID-19 on SSPD. Data from 19,344,698 patients, sourced from the N3C Data Enclave platform, were methodically filtered to create propensity matched cohorts: ARDS (n = 222,337), COVID-positive (n = 219,264), and COVID-negative (n = 213,183). We systematically analyzed the hazard rate of new-onset SSPD across three distinct time intervals: 0-21 days, 22-90 days, and beyond 90 days post-infection. COVID-19 positive patients consistently exhibited a heightened hazard ratio (HR) across all intervals [0-21 days (HR: 4.6; CI: 3.7-5.7), 22-90 days (HR: 2.9; CI: 2.3 -3.8), beyond 90 days (HR: 1.7; CI: 1.5-1.)]. These are notably higher than both ARDS and COVID-19 lab-negative patients. Validations using various tests, including the Cochran Mantel Haenszel Test, Wald Test, and Log-rank Test confirmed these associations. Intriguingly, our data indicated that younger individuals face a heightened risk of SSPD after contracting COVID-19, a trend not observed in the ARDS and COVID-negative groups. These results, aligned with the known neurotropism of SARS-CoV-2 and earlier studies, accentuate the need for vigilant psychiatric assessment and support in the era of Long-COVID, especially among younger populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.23300210">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.23300210" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.23300210">
        <p class="paperTitle">The molecular landscape of premature aging diseases defined by multilayer network exploration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.23300210" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.23300210" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Beust, C.; Valdeolivas, A.; Baptista, A.; Briere, G.; Levy, N.; Ozisik, O.; Baudot, A.</p>
        <p class="info">Score: 4.8, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.23300210' target='https://doi.org/10.1101/2023.12.19.23300210'> 10.1101/2023.12.19.23300210</a></p>
        <p class="abstract">Premature Aging (PA) diseases are rare genetic disorders that mimic some aspects of physiological aging at an early age. Various causative genes of PA diseases have been identified in recent years, providing insights into some dysfunctional cellular functions. However, the identification of PA genes also revealed significant genetic heterogeneity and highlighted the gaps in our understanding of PA molecular mechanisms. Furthermore, many patients remain undiagnosed. Overall, the current lack of knowledge about PA diseases hinders the development of effective diagnosis and therapies and poses significant challenges to improving patient care.

Here, we present a network-based approach to systematically unravel the cellular functions disrupted in PA diseases. Leveraging a novel community identification algorithm, we delved into a vast multilayer network of biological interactions to extract the communities of 67 PA diseases from their 132 associated genes. We found that these communities can be grouped into six distinct clusters, each reflecting specific cellular functions: DNA repair, cell cycle, transcription regulation, inflammation, cell communication, and vesicle-mediated transport. We propose that these clusters collectively represent the landscape of the molecular mechanisms that are perturbed in PA diseases, providing a framework for better understanding their pathogenesis. Intriguingly, most clusters also exhibited a significant enrichment in genes associated with physiological aging, suggesting a potential overlap between the molecular underpinnings of PA diseases and natural aging.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.23300393">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.23300393" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.23300393">
        <p class="paperTitle">A deep learning transformer model predicts high rates of undiagnosed rare disease in large electronic health systems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.23300393" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.23300393" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jordan, D. M.; Vy, H. M. T.; Do, R.</p>
        <p class="info">Score: 3.4, Published: 2023-12-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.23300393' target='https://doi.org/10.1101/2023.12.21.23300393'> 10.1101/2023.12.21.23300393</a></p>
        <p class="abstract">It is estimated that as many as 1 in 16 people worldwide suffer from rare diseases. Rare disease patients face difficulty finding diagnosis and treatment for their conditions, including long diagnostic odysseys, multiple incorrect diagnoses, and unavailable or prohibitively expensive treatments. As a result, it is likely that large electronic health record (EHR) systems include high numbers of participants suffering from undiagnosed rare disease. While this has been shown in detail for specific diseases, these studies are expensive and time consuming and have only been feasible to perform for a handful of the thousands of known rare diseases. The bulk of these undiagnosed cases are effectively hidden, with no straightforward way to differentiate them from healthy controls. The ability to access them at scale would enormously expand our capacity to study and develop drugs for rare diseases, adding to tools aimed at increasing availability of study cohorts for rare disease. In this study, we train a deep learning transformer algorithm, RarePT (Rare-Phenotype Prediction Transformer), to impute undiagnosed rare disease from EHR diagnosis codes in 436,407 participants in the UK Biobank and validated on an independent cohort from 3,333,560 individuals from the Mount Sinai Health System. We applied our model to 155 rare diagnosis codes with fewer than 250 cases each in the UK Biobank and predicted participants with elevated risk for each diagnosis, with the number of participants predicted to be at risk ranging from 85 to 22,000 for different diagnoses. These risk predictions are significantly associated with increased mortality for 65% of diagnoses, with disease burden expressed as disability-adjusted life years (DALY) for 73% of diagnoses, and with 72% of available disease-specific diagnostic tests. They are also highly enriched for known rare diagnoses in patients not included in the training set, with an odds ratio (OR) of 48.0 in cross-validation cohorts of the UK Biobank and an OR of 30.6 in the independent Mount Sinai Health System cohort. Most importantly, RarePT successfully screens for undiagnosed patients in 32 rare diseases with available diagnostic tests in the UK Biobank. Using the trained model to estimate the prevalence of undiagnosed disease in the UK Biobank for these 32 rare phenotypes, we find that at least 50% of patients remain undiagnosed for 20 of 32 diseases. These estimates provide empirical evidence of a high prevalence of undiagnosed rare disease, as well as demonstrating the enormous potential benefit of using RarePT to screen for undiagnosed rare disease patients in large electronic health systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.23291717">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.23291717" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.23291717">
        <p class="paperTitle">Machine learning in medicine using JavaScript: building web apps using TensorFlow.js for interpreting biomedical datasets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.23291717" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.23291717" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pires, J. G.</p>
        <p class="info">Score: 5.9, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.23291717' target='https://doi.org/10.1101/2023.06.21.23291717'> 10.1101/2023.06.21.23291717</a></p>
        <p class="abstract">Contributions to medicine may come from different areas, and most of these areas are filled with researchers eager to contribute. In this paper, we aim to contribute through the intersection of machine learning and web development. We employed TensorFlow.js, a JavaScript-based library, to model biomedical datasets using neural networks obtained from Kaggle. The principal aim of this study is to present the capabilities of TensorFlow.js and promote its utility in the development of sophisticated machine learning models customized for web-based applications. We modeled three datasets: diabetes detection, surgery complications, and heart failure. While Python and R currently dominate, JavaScript and its derivatives are rapidly gaining ground, offering comparable performance and additional features associated with JavaScript. Kaggle, the public platform from which we downloaded our datasets, provides an extensive collection of biomedical datasets. Therefore, readers can easily test our discussed methods by using the provided codes with minor adjustments on any case of their interest. The results demonstrate an accuracy of 92% for diabetes detection, almost 100% for surgery complications, and 80% for heart failure. The possibilities are vast, and we believe that this is an excellent option for researchers focusing on web applications, particularly in the field of medicine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.23300384">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.23300384" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.23300384">
        <p class="paperTitle">The Applications of Laser Interstitial Thermal Therapy and Machine Learning in Neurosurgery: A Systematic Review</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.23300384" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.23300384" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bouras, A.; Patel, D.; Chetla, N.</p>
        <p class="info">Score: 1.4, Published: 2023-12-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.23300384' target='https://doi.org/10.1101/2023.12.21.23300384'> 10.1101/2023.12.21.23300384</a></p>
        <p class="abstract">Background: The incorporation of Machine Learning (ML) into Laser Interstitial Thermal Therapy (LITT) represents a significant advancement in minimally invasive neurosurgery, partic- ularly for treating brain tumors, vascular malformations, and epileptogenic foci. This systematic review focuses on evaluating the integration and impact of ML in enhancing the efficacy, precision, and outcomes of LITT in neurosurgical procedures. Methods: An exhaustive search was conducted in major scientific databases for studies from 2015 to 2023 that specifically focused on the application of ML in LITT. The review assessed the development and implementation of ML algorithms in surgical planning, outcome prediction, and postoperative evaluation in LITT. Rigorous inclusion criteria were applied to select studies, and a combination of meta-analysis and qualitative synthesis was used to analyze the data. Results: The review synthesizes findings from a range of studies, including retrospective anal- yses and initial clinical trials. It highlights the role of ML in enhancing the selection criteria for LITT, optimizing surgical approaches, and improving patient-specific outcome predictions. While LITT showed favorable results in treating non-resectable lesions, the integration of ML was found to potentially refine these outcomes further. However, challenges such as the need for larger sample sizes, standardization of ML algorithms, and validation of these methods in clinical settings were noted. Conclusions: The integration of ML into LITT procedures marks a promising frontier in neurosurgery, offering potential improvements in surgical accuracy and patient outcomes. The evidence suggests a need for continued development and rigorous testing of ML applications in LITT. Future research should focus on the refinement and validation of ML algorithms for wider clinical adoption, ensuring that technological advancements align with patient safety and treatment efficacy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.23299867">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.23299867" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.23299867">
        <p class="paperTitle">FRISTS: INTERPRETABLE TIME SERIES-BASED HEART FAILURE RISK PREDICTION</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.23299867" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.23299867" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lin, S.; Dong, X.; Wang, F.</p>
        <p class="info">Score: 1.1, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.23299867' target='https://doi.org/10.1101/2023.12.20.23299867'> 10.1101/2023.12.20.23299867</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWHeart failure is an incurable medical condition that affects millions of people globally. Developing prediction models is crucial to prevent patients from progressing to heart failure. Current heart failure prediction models struggle with achieving interpretability, a mandatory trait that allows the model to be applied in healthcare, while possessing high real-world accuracy. We introduce FRISTS, a novel HF prediction approach leveraging sequential times series-based modeling, such as long short-term memory (LSTM) networks, and feature selection. Our study utilized an extensive electronic health record (EHR) database and found that FRISTS outperformed traditional AI models (e.g., Random Forest Classifier, Logistic Regression, and Decision Tree) and more complex machine learning techniques (e.g., XGBoost, LSTM), yielding an average F1 score of 0.805 and receiver operating characteristic area under the curve (ROC AUC) value of 0.990. Our SHAP-inspired permutation method enables interpretation of the feature ranking, enhancing the result transparency and paving the way for a new class of interpretable models. This approach holds promise in enhancing clinical decision-making and patient care in the context of heart failure prevention.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.23299881">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.23299881" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.23299881">
        <p class="paperTitle">Augmentation of ChatGPT with Clinician-Informed Tools Improves Performance on Medical Calculation Tasks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.23299881" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.23299881" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Goodell, A. J.; Chu, S. N.; Rouholiman, D.; Chu, L. F.</p>
        <p class="info">Score: 2.6, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.23299881' target='https://doi.org/10.1101/2023.12.13.23299881'> 10.1101/2023.12.13.23299881</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWPrior work has shown that large language models (LLMs) have the ability to answer expert-level multiple choice questions in medicine, but are limited by both their tendency to hallucinate knowledge and their inherent inadequacy in performing basic mathematical operations. Unsurprisingly, early evidence suggests that LLMs perform poorly when asked to execute common clinical calculations. Recently, it has been demonstrated that LLMs have the capability of interacting with external programs and tools, presenting a possible remedy for this limitation. In this study, we explore the ability of ChatGPT (GPT-4, November 2023) to perform medical calculations, evaluating its performance across 48 diverse clinical calculation tasks. Our findings indicate that ChatGPT is an unreliable clinical calculator, delivering inaccurate responses in one-third of trials (n=212). To address this, we developed an open-source clinical calculation API (openmedcalc.org), which we then integrated with ChatGPT. We subsequently evaluated the performance of this augmented model by comparing it against standard ChatGPT using 75 clinical vignettes in three common clinical calculation tasks: Caprini VTE Risk, Wells DVT Criteria, and MELD-Na. The augmented model demonstrated a marked improvement in accuracy over unimproved ChatGPT. Our findings suggest that integration of machine-usable, clinician-informed tools can help alleviate the reliability limitations observed in medical LLMs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.23298847">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.23298847" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.23298847">
        <p class="paperTitle">SARS-CoV-2 Orphan Gene ORF10 Contributes to More Severe COVID-19 Disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.23298847" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.23298847" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Haltom, J. A.; Trovao, N. S.; Guarnieri, J.; Vincent, P.; Singh, U.; Tsoy, S.; O&#39;Leary, C. A.; Bram, Y.; Widjaja, G. A.; Cen, Z.; Meller, R.; Baylin, S. B.; Moss, W. N.; Nikolau, B. J.; Enguita, F. J.; Wallace, D. C.; Beheshti, A.; Schwartz, R.; Wurtele, E. S.</p>
        <p class="info">Score: 25.7, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.23298847' target='https://doi.org/10.1101/2023.11.27.23298847'> 10.1101/2023.11.27.23298847</a></p>
        <p class="abstract">The orphan gene of SARS-CoV-2, ORF10, is the least studied gene in the virus responsible for the COVID-19 pandemic. Recent experimentation indicated ORF10 expression moderates innate immunity in vitro. However, whether ORF10 affects COVID-19 in humans remained unknown. We determine that the ORF10 sequence is identical to the Wuhan-Hu-1 ancestral haplotype in 95% of genomes across five variants of concern (VOC). Four ORF10 variants are associated with less virulent clinical outcomes in the human host: three of these affect ORF10 protein structure, one affects ORF10 RNA structural dynamics. RNA-Seq data from 2070 samples from diverse human cells and tissues reveals ORF10 accumulation is conditionally discordant from that of other SARS-CoV-2 transcripts. Expression of ORF10 in A549 and HEK293 cells perturbs immune-related gene expression networks, alters expression of the majority of mitochondrially-encoded genes of oxidative respiration, and leads to large shifts in levels of 14 newly-identified transcripts. We conclude ORF10 contributes to more severe COVID-19 clinical outcomes in the human host.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
