<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>health informatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>health informatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Retrieval Augmented Generation Enabled Generative Pre-Trained Transformer 4 (GPT-4) Performance for Clinical Trial Screening
Authors: Unlu, O.; Shin, J.; Mailly, C. J.; Oates, M. F.; Tucci, M. R.; Varugheese, M.; Wagholikar, K.; Wang, F.; Scirica, B. M.; Blood, A. J.; Aronson, S. J.
Score: 62.3, Published: 2024-02-08 DOI: 10.1101/2024.02.08.24302376
BackgroundSubject screening is a key aspect of all clinical trials; however, traditionally, it is a labor-intensive and error-prone task, demanding significant time and resources.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/health-informatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="health informatics" />
<meta property="og:description" content="Retrieval Augmented Generation Enabled Generative Pre-Trained Transformer 4 (GPT-4) Performance for Clinical Trial Screening
Authors: Unlu, O.; Shin, J.; Mailly, C. J.; Oates, M. F.; Tucci, M. R.; Varugheese, M.; Wagholikar, K.; Wang, F.; Scirica, B. M.; Blood, A. J.; Aronson, S. J.
Score: 62.3, Published: 2024-02-08 DOI: 10.1101/2024.02.08.24302376
BackgroundSubject screening is a key aspect of all clinical trials; however, traditionally, it is a labor-intensive and error-prone task, demanding significant time and resources." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/health-informatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-14T10:38:48+00:00" />
<meta property="article:modified_time" content="2024-02-14T10:38:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="health informatics"/>
<meta name="twitter:description" content="Retrieval Augmented Generation Enabled Generative Pre-Trained Transformer 4 (GPT-4) Performance for Clinical Trial Screening
Authors: Unlu, O.; Shin, J.; Mailly, C. J.; Oates, M. F.; Tucci, M. R.; Varugheese, M.; Wagholikar, K.; Wang, F.; Scirica, B. M.; Blood, A. J.; Aronson, S. J.
Score: 62.3, Published: 2024-02-08 DOI: 10.1101/2024.02.08.24302376
BackgroundSubject screening is a key aspect of all clinical trials; however, traditionally, it is a labor-intensive and error-prone task, demanding significant time and resources."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "health informatics",
      "item": "https://trxiv.yorks0n.com/posts/health-informatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "health informatics",
  "name": "health informatics",
  "description": "Retrieval Augmented Generation Enabled Generative Pre-Trained Transformer 4 (GPT-4) Performance for Clinical Trial Screening\nAuthors: Unlu, O.; Shin, J.; Mailly, C. J.; Oates, M. F.; Tucci, M. R.; Varugheese, M.; Wagholikar, K.; Wang, F.; Scirica, B. M.; Blood, A. J.; Aronson, S. J.\nScore: 62.3, Published: 2024-02-08 DOI: 10.1101/2024.02.08.24302376\nBackgroundSubject screening is a key aspect of all clinical trials; however, traditionally, it is a labor-intensive and error-prone task, demanding significant time and resources.",
  "keywords": [
    
  ],
  "articleBody": " Retrieval Augmented Generation Enabled Generative Pre-Trained Transformer 4 (GPT-4) Performance for Clinical Trial Screening\nAuthors: Unlu, O.; Shin, J.; Mailly, C. J.; Oates, M. F.; Tucci, M. R.; Varugheese, M.; Wagholikar, K.; Wang, F.; Scirica, B. M.; Blood, A. J.; Aronson, S. J.\nScore: 62.3, Published: 2024-02-08 DOI: 10.1101/2024.02.08.24302376\nBackgroundSubject screening is a key aspect of all clinical trials; however, traditionally, it is a labor-intensive and error-prone task, demanding significant time and resources. With the advent of large language models (LLMs) and related technologies, a paradigm shift in natural language processing capabilities offers a promising avenue for increasing both quality and efficiency of screening efforts. This study aimed to test the Retrieval-Augmented Generation (RAG) process enabled Generative Pretrained Transformer Version 4 (GPT-4) to accurately identify and report on inclusion and exclusion criteria for a clinical trial. MethodsThe Co-Operative Program for Implementation of Optimal Therapy in Heart Failure (COPILOT-HF) trial aims to recruit patients with symptomatic heart failure. As part of the screening process, a list of potentially eligible patients is created through an electronic health record (EHR) query. Currently, structured data in the EHR can only be used to determine 5 out of 6 inclusion and 5 out of 17 exclusion criteria. Trained, but non-licensed, study staff complete manual chart review to determine patient eligibility and record their assessment of the inclusion and exclusion criteria. We obtained the structured assessments completed by the study staff and clinical notes for the past two years and developed a workflow of clinical note-based question answering system powered by RAG architecture and GPT-4 that we named RECTIFIER (RAG-Enabled Clinical Trial Infrastructure for Inclusion Exclusion Review). We used notes from 100 patients as a development dataset, 282 patients as a validation dataset, and 1894 patients as a test set. An expert clinician completed a blinded review of patients charts to answer the eligibility questions and determine the \"gold standard\" answers. We calculated the sensitivity, specificity, accuracy, and Matthews correlation coefficient (MCC) for each question and screening method. We also performed bootstrapping to calculate the confidence intervals for each statistic. ResultsBoth RECTIFIER and study staff answers closely aligned with the expert clinician answers across criteria with accuracy ranging between 97.9% and 100% (MCC 0.837 and 1) for RECTIFIER and 91.7% and 100% (MCC 0.644 and 1) for study staff. RECTIFIER performed better than study staff to determine the inclusion criteria of \"symptomatic heart failure\" with an accuracy of 97.9% vs 91.7% and an MCC of 0.924 vs 0.721, respectively. Overall, the sensitivity and specificity of determining eligibility for the RECTIFIER was 92.3% (CI) and 93.9% (CI), and study staff was 90.1% (CI) and 83.6% (CI), respectively. ConclusionGPT-4 based solutions have the potential to improve efficiency and reduce costs in clinical trial screening. When incorporating new tools such as RECTIFIER, it is important to consider the potential hazards of automating the screening process and set up appropriate mitigation strategies such as final clinician review before patient engagement.\nUnequal access to diagnosis of myalgic encephalomyelitis in England\nAuthors: Ponting, C. P.; Samms, G. L.\nScore: 64.7, Published: 2024-02-01 DOI: 10.1101/2024.01.31.24302070\nBackgroundPeople with Myalgic Encephalomyelitis (ME/CFS; sometimes referred to as chronic fatigue syndrome) experience very poor health-related quality of life and only rarely recover. ME/CFS has no curative treatment and no single diagnostic test. Public health and policy decisions relevant to ME/CFS require knowledge of its prevalence and barriers to diagnosis. However, people with ME/CFS report lengthy diagnostic delays and widespread misunderstanding of their symptoms. Published prevalence estimates vary greatly by country, gender, age and ethnicity. MethodsHospital Episode Statistics data is routinely collected by the NHS in England together with patient age, gender and ethnicity. This data, downloaded from the Feasibility Self-Service of NHS DigiTrials, was used to stratify individuals with the ICD-10 code that best reflects ME/CFS symptoms (G93.3; \"Postviral fatigue syndrome\") according to their age, self-reported gender and ethnicity, General Practice and NHS England Integrated Care Board (ICB). ResultsIn all, 100,055 people in England had been diagnosed with ME/CFS (ICD-10:G93.3) between April 1 1989 and October 7 2023, 0.16% of all registered patients. Of these, 79,445 were females and 20,590 males, a female-to-male ratio of 3.88:1. Female relative to male prevalence peaked at about 6-to-1 in individuals fourth and fifth decades of life. Prevalence varied widely across the 42 ICBs: 0.086%-0.82% for females and 0.024%-0.21% for males. White individuals were approximately 5-fold more likely to be diagnosed with ME/CFS than others; black, Asian or Chinese ethnicities are associated with particularly low rates of ME/CFS diagnoses. This ethnicity bias is stronger than for other common diseases. Among active English GP practices, 176 (3%) had no registered ME/CFS patients. Eight ICBs (19%) each contained fewer than 8 other-than-white individuals with a G93.3 code despite their registers containing a total of 293,770 other-than-white patients. ConclusionThose who are disproportionately undiagnosed with ME/CFS are other-than-white ethnic groups, older females (\u003e60y), older males (\u003e80y), and people living in areas of multiple deprivation. The lifetime prevalence of ME/CFS for English females and males may be as high as 0.92% and 0.25%, respectively, or approximately 390,000 UK individuals overall. This improved estimate of ME/CFS prevalence allows more accurate assessment of the socioeconomic and disease burden imposed by ME/CFS.\nAn Integrative Network Approach for Longitudinal Stratification in Parkinson's Disease\nAuthors: Ryan, B.; Marioni, R.; Simpson, T. I.\nScore: 2.6, Published: 2024-01-26 DOI: 10.1101/2024.01.25.24301595\nParkinsons Disease (PD) is a neurodegenerative disorder characterized by motor symptoms resulting from the loss of dopamine-producing neurons in the brain. Currently, there is no cure for the disease which is in part due to the heterogeneity in patient symptoms, trajectories and manifestations. There is a known genetic component of PD and genomic datasets have helped to uncover some aspects of the disease. Understanding the longitudinal variability of PD is essential as it has been theorised that there are different triggers and underlying disease mechanisms at different points during disease progression. In this paper, we perform longitudinal and cross-sectional experiments to identify which data modalities or combinations of modalities are informative at different time points. We use clinical, genomic, and proteomic data from the Parkinsons Progression Markers Initiative. We validate the importance of flexible data integration by highlighting the varying combinations of data modalities for optimal stratification at different disease stages in idiopathic PD. We show there is a shared signal in the DNAm signatures of participants with a mutation in a causal gene of PD and participants with idiopathic PD. We also show that integration of SNPs and DNAm data modalities has potential for use as an early diagnostic tool for individuals with a genetic cause of PD.\nA deep-learning-based two-compartment predictive model (PKRNN-2CM) for vancomycin therapeutic drug monitoring\nAuthors: Mao, B.; Xie, Z.; Nigo, M.; Rasmy, L.; Zhi, D.\nScore: 4.5, Published: 2024-01-31 DOI: 10.1101/2024.01.30.24302025\nObjectiveVancomycin is a widely used antibiotic that requires therapeutic drug monitoring (TDM) for optimized individual dosage. The deep learning-based model PKRNN-1CM has shown the advantage of leveraging time series electronic health record (EHR) data for individualized estimation of vancomycin pharmacokinetic (PK) parameters. While one-compartment (1CM) PK models are commonly used because of their simplicity and previous trough-based clinical practices for dose adjustment, the pre-deep learning literature suggests the superiority of two-compartment models (2CM). Motivated by this, we introduce a novel deep-learning-based approach, PKRNN-2CM, for vancomycin TDM. MethodsPKRNN-2CM combines RNN-driven PK parameter estimation with a 2CM PK model to predict vancomycin concentration trajectories. Training on both simulated data and real-world EHR data allows for a comprehensive evaluation of its performance. ResultsExperiments based on simulated data highlight PKRNN-2CMs superiority over the simpler 1CM model PKRNN-1CM (PKRNN-2CM RMSE=1.30, PKRNN-1CM RMSE=2.50). Application to real data showcases significant improvement over PKRNN-1CM (PKRNN-2CM RMSE=5.62, PKRNN-1CM RMSE=5.84, two-sample unpaired t-test p-value=0.01), with potential further gains expected with non-trough level measurements. ConclusionPKRNN-2CM is an important improvement in vancomycin TDM, demonstrating enhanced accuracy and performance compared to the PKRNN-1CM model. This deep learning model holds potential for future individualized vancomycin TDM optimization and broader application in diverse clinical scenarios.\n",
  "wordCount" : "1324",
  "inLanguage": "en",
  "datePublished": "2024-02-14T10:38:48Z",
  "dateModified": "2024-02-14T10:38:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/health-informatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      health informatics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 14, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.08.24302376">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.08.24302376" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.08.24302376">
        <p class="paperTitle">Retrieval Augmented Generation Enabled Generative Pre-Trained Transformer 4 (GPT-4) Performance for Clinical Trial Screening</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.08.24302376" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.08.24302376" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Unlu, O.; Shin, J.; Mailly, C. J.; Oates, M. F.; Tucci, M. R.; Varugheese, M.; Wagholikar, K.; Wang, F.; Scirica, B. M.; Blood, A. J.; Aronson, S. J.</p>
        <p class="info">Score: 62.3, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.08.24302376' target='https://doi.org/10.1101/2024.02.08.24302376'> 10.1101/2024.02.08.24302376</a></p>
        <p class="abstract">BackgroundSubject screening is a key aspect of all clinical trials; however, traditionally, it is a labor-intensive and error-prone task, demanding significant time and resources. With the advent of large language models (LLMs) and related technologies, a paradigm shift in natural language processing capabilities offers a promising avenue for increasing both quality and efficiency of screening efforts. This study aimed to test the Retrieval-Augmented Generation (RAG) process enabled Generative Pretrained Transformer Version 4 (GPT-4) to accurately identify and report on inclusion and exclusion criteria for a clinical trial.

MethodsThe Co-Operative Program for Implementation of Optimal Therapy in Heart Failure (COPILOT-HF) trial aims to recruit patients with symptomatic heart failure. As part of the screening process, a list of potentially eligible patients is created through an electronic health record (EHR) query. Currently, structured data in the EHR can only be used to determine 5 out of 6 inclusion and 5 out of 17 exclusion criteria. Trained, but non-licensed, study staff complete manual chart review to determine patient eligibility and record their assessment of the inclusion and exclusion criteria. We obtained the structured assessments completed by the study staff and clinical notes for the past two years and developed a workflow of clinical note-based question answering system powered by RAG architecture and GPT-4 that we named RECTIFIER (RAG-Enabled Clinical Trial Infrastructure for Inclusion Exclusion Review). We used notes from 100 patients as a development dataset, 282 patients as a validation dataset, and 1894 patients as a test set. An expert clinician completed a blinded review of patients charts to answer the eligibility questions and determine the &#34;gold standard&#34; answers. We calculated the sensitivity, specificity, accuracy, and Matthews correlation coefficient (MCC) for each question and screening method. We also performed bootstrapping to calculate the confidence intervals for each statistic.

ResultsBoth RECTIFIER and study staff answers closely aligned with the expert clinician answers across criteria with accuracy ranging between 97.9% and 100% (MCC 0.837 and 1) for RECTIFIER and 91.7% and 100% (MCC 0.644 and 1) for study staff. RECTIFIER performed better than study staff to determine the inclusion criteria of &#34;symptomatic heart failure&#34; with an accuracy of 97.9% vs 91.7% and an MCC of 0.924 vs 0.721, respectively. Overall, the sensitivity and specificity of determining eligibility for the RECTIFIER was 92.3% (CI) and 93.9% (CI), and study staff was 90.1% (CI) and 83.6% (CI), respectively.

ConclusionGPT-4 based solutions have the potential to improve efficiency and reduce costs in clinical trial screening. When incorporating new tools such as RECTIFIER, it is important to consider the potential hazards of automating the screening process and set up appropriate mitigation strategies such as final clinician review before patient engagement.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.31.24302070">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.31.24302070" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.31.24302070">
        <p class="paperTitle">Unequal access to diagnosis of myalgic encephalomyelitis in England</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.31.24302070" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.31.24302070" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ponting, C. P.; Samms, G. L.</p>
        <p class="info">Score: 64.7, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.31.24302070' target='https://doi.org/10.1101/2024.01.31.24302070'> 10.1101/2024.01.31.24302070</a></p>
        <p class="abstract">BackgroundPeople with Myalgic Encephalomyelitis (ME/CFS; sometimes referred to as chronic fatigue syndrome) experience very poor health-related quality of life and only rarely recover. ME/CFS has no curative treatment and no single diagnostic test. Public health and policy decisions relevant to ME/CFS require knowledge of its prevalence and barriers to diagnosis. However, people with ME/CFS report lengthy diagnostic delays and widespread misunderstanding of their symptoms. Published prevalence estimates vary greatly by country, gender, age and ethnicity.

MethodsHospital Episode Statistics data is routinely collected by the NHS in England together with patient age, gender and ethnicity. This data, downloaded from the Feasibility Self-Service of NHS DigiTrials, was used to stratify individuals with the ICD-10 code that best reflects ME/CFS symptoms (G93.3; &#34;Postviral fatigue syndrome&#34;) according to their age, self-reported gender and ethnicity, General Practice and NHS England Integrated Care Board (ICB).

ResultsIn all, 100,055 people in England had been diagnosed with ME/CFS (ICD-10:G93.3) between April 1 1989 and October 7 2023, 0.16% of all registered patients. Of these, 79,445 were females and 20,590 males, a female-to-male ratio of 3.88:1. Female relative to male prevalence peaked at about 6-to-1 in individuals fourth and fifth decades of life. Prevalence varied widely across the 42 ICBs: 0.086%-0.82% for females and 0.024%-0.21% for males. White individuals were approximately 5-fold more likely to be diagnosed with ME/CFS than others; black, Asian or Chinese ethnicities are associated with particularly low rates of ME/CFS diagnoses. This ethnicity bias is stronger than for other common diseases. Among active English GP practices, 176 (3%) had no registered ME/CFS patients. Eight ICBs (19%) each contained fewer than 8 other-than-white individuals with a G93.3 code despite their registers containing a total of 293,770 other-than-white patients.

ConclusionThose who are disproportionately undiagnosed with ME/CFS are other-than-white ethnic groups, older females (&gt;60y), older males (&gt;80y), and people living in areas of multiple deprivation. The lifetime prevalence of ME/CFS for English females and males may be as high as 0.92% and 0.25%, respectively, or approximately 390,000 UK individuals overall. This improved estimate of ME/CFS prevalence allows more accurate assessment of the socioeconomic and disease burden imposed by ME/CFS.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.25.24301595">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.25.24301595" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.25.24301595">
        <p class="paperTitle">An Integrative Network Approach for Longitudinal Stratification in Parkinson&#39;s Disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.25.24301595" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.25.24301595" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ryan, B.; Marioni, R.; Simpson, T. I.</p>
        <p class="info">Score: 2.6, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.25.24301595' target='https://doi.org/10.1101/2024.01.25.24301595'> 10.1101/2024.01.25.24301595</a></p>
        <p class="abstract">Parkinsons Disease (PD) is a neurodegenerative disorder characterized by motor symptoms resulting from the loss of dopamine-producing neurons in the brain. Currently, there is no cure for the disease which is in part due to the heterogeneity in patient symptoms, trajectories and manifestations. There is a known genetic component of PD and genomic datasets have helped to uncover some aspects of the disease. Understanding the longitudinal variability of PD is essential as it has been theorised that there are different triggers and underlying disease mechanisms at different points during disease progression. In this paper, we perform longitudinal and cross-sectional experiments to identify which data modalities or combinations of modalities are informative at different time points. We use clinical, genomic, and proteomic data from the Parkinsons Progression Markers Initiative. We validate the importance of flexible data integration by highlighting the varying combinations of data modalities for optimal stratification at different disease stages in idiopathic PD. We show there is a shared signal in the DNAm signatures of participants with a mutation in a causal gene of PD and participants with idiopathic PD. We also show that integration of SNPs and DNAm data modalities has potential for use as an early diagnostic tool for individuals with a genetic cause of PD.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.30.24302025">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.30.24302025" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.30.24302025">
        <p class="paperTitle">A deep-learning-based two-compartment predictive model (PKRNN-2CM) for vancomycin therapeutic drug monitoring</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.30.24302025" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.30.24302025" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mao, B.; Xie, Z.; Nigo, M.; Rasmy, L.; Zhi, D.</p>
        <p class="info">Score: 4.5, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.30.24302025' target='https://doi.org/10.1101/2024.01.30.24302025'> 10.1101/2024.01.30.24302025</a></p>
        <p class="abstract">ObjectiveVancomycin is a widely used antibiotic that requires therapeutic drug monitoring (TDM) for optimized individual dosage. The deep learning-based model PKRNN-1CM has shown the advantage of leveraging time series electronic health record (EHR) data for individualized estimation of vancomycin pharmacokinetic (PK) parameters. While one-compartment (1CM) PK models are commonly used because of their simplicity and previous trough-based clinical practices for dose adjustment, the pre-deep learning literature suggests the superiority of two-compartment models (2CM). Motivated by this, we introduce a novel deep-learning-based approach, PKRNN-2CM, for vancomycin TDM.

MethodsPKRNN-2CM combines RNN-driven PK parameter estimation with a 2CM PK model to predict vancomycin concentration trajectories. Training on both simulated data and real-world EHR data allows for a comprehensive evaluation of its performance.

ResultsExperiments based on simulated data highlight PKRNN-2CMs superiority over the simpler 1CM model PKRNN-1CM (PKRNN-2CM RMSE=1.30, PKRNN-1CM RMSE=2.50). Application to real data showcases significant improvement over PKRNN-1CM (PKRNN-2CM RMSE=5.62, PKRNN-1CM RMSE=5.84, two-sample unpaired t-test p-value=0.01), with potential further gains expected with non-trough level measurements.

ConclusionPKRNN-2CM is an important improvement in vancomycin TDM, demonstrating enhanced accuracy and performance compared to the PKRNN-1CM model. This deep learning model holds potential for future individualized vancomycin TDM optimization and broader application in diverse clinical scenarios.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
