<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>health informatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>health informatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="SARS-CoV-2 Infection is Associated with an Increase in New Diagnoses of Schizophrenia Spectrum and Psychotic Disorder: A Study Using the US National COVID Cohort Collaborative (N3C)
Authors: Rahman, A.; Russell, M.; Zheng, W.; Eckrich, D.; Ahmed, I.; on behalf of the N3C Consortium, Score: 703.8, Published: 2023-12-05 DOI: 10.1101/2023.12.05.23299473
Amid the ongoing global repercussions of SARS-CoV-2, its crucial to comprehend its potential long-term psychiatric effects. Several recent studies have suggested a link between COVID-19 and subsequent mental health disorders.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/health-informatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="health informatics" />
<meta property="og:description" content="SARS-CoV-2 Infection is Associated with an Increase in New Diagnoses of Schizophrenia Spectrum and Psychotic Disorder: A Study Using the US National COVID Cohort Collaborative (N3C)
Authors: Rahman, A.; Russell, M.; Zheng, W.; Eckrich, D.; Ahmed, I.; on behalf of the N3C Consortium, Score: 703.8, Published: 2023-12-05 DOI: 10.1101/2023.12.05.23299473
Amid the ongoing global repercussions of SARS-CoV-2, its crucial to comprehend its potential long-term psychiatric effects. Several recent studies have suggested a link between COVID-19 and subsequent mental health disorders." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/health-informatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-31T10:38:19+00:00" />
<meta property="article:modified_time" content="2023-12-31T10:38:19+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="health informatics"/>
<meta name="twitter:description" content="SARS-CoV-2 Infection is Associated with an Increase in New Diagnoses of Schizophrenia Spectrum and Psychotic Disorder: A Study Using the US National COVID Cohort Collaborative (N3C)
Authors: Rahman, A.; Russell, M.; Zheng, W.; Eckrich, D.; Ahmed, I.; on behalf of the N3C Consortium, Score: 703.8, Published: 2023-12-05 DOI: 10.1101/2023.12.05.23299473
Amid the ongoing global repercussions of SARS-CoV-2, its crucial to comprehend its potential long-term psychiatric effects. Several recent studies have suggested a link between COVID-19 and subsequent mental health disorders."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "health informatics",
      "item": "https://trxiv.yorks0n.com/posts/health-informatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "health informatics",
  "name": "health informatics",
  "description": "SARS-CoV-2 Infection is Associated with an Increase in New Diagnoses of Schizophrenia Spectrum and Psychotic Disorder: A Study Using the US National COVID Cohort Collaborative (N3C)\nAuthors: Rahman, A.; Russell, M.; Zheng, W.; Eckrich, D.; Ahmed, I.; on behalf of the N3C Consortium, Score: 703.8, Published: 2023-12-05 DOI: 10.1101/2023.12.05.23299473\nAmid the ongoing global repercussions of SARS-CoV-2, its crucial to comprehend its potential long-term psychiatric effects. Several recent studies have suggested a link between COVID-19 and subsequent mental health disorders.",
  "keywords": [
    
  ],
  "articleBody": " SARS-CoV-2 Infection is Associated with an Increase in New Diagnoses of Schizophrenia Spectrum and Psychotic Disorder: A Study Using the US National COVID Cohort Collaborative (N3C)\nAuthors: Rahman, A.; Russell, M.; Zheng, W.; Eckrich, D.; Ahmed, I.; on behalf of the N3C Consortium, Score: 703.8, Published: 2023-12-05 DOI: 10.1101/2023.12.05.23299473\nAmid the ongoing global repercussions of SARS-CoV-2, its crucial to comprehend its potential long-term psychiatric effects. Several recent studies have suggested a link between COVID-19 and subsequent mental health disorders. Our investigation joins this exploration, concentrating on Schizophrenia Spectrum and Psychotic Disorders (SSPD). Different from other studies, we took acute respiratory distress syndrome (ARDS) and COVID-19 lab negative cohorts as control groups to accurately gauge the impact of COVID-19 on SSPD. Data from 19,344,698 patients, sourced from the N3C Data Enclave platform, were methodically filtered to create propensity matched cohorts: ARDS (n = 222,337), COVID-positive (n = 219,264), and COVID-negative (n = 213,183). We systematically analyzed the hazard rate of new-onset SSPD across three distinct time intervals: 0-21 days, 22-90 days, and beyond 90 days post-infection. COVID-19 positive patients consistently exhibited a heightened hazard ratio (HR) across all intervals [0-21 days (HR: 4.6; CI: 3.7-5.7), 22-90 days (HR: 2.9; CI: 2.3 -3.8), beyond 90 days (HR: 1.7; CI: 1.5-1.)]. These are notably higher than both ARDS and COVID-19 lab-negative patients. Validations using various tests, including the Cochran Mantel Haenszel Test, Wald Test, and Log-rank Test confirmed these associations. Intriguingly, our data indicated that younger individuals face a heightened risk of SSPD after contracting COVID-19, a trend not observed in the ARDS and COVID-negative groups. These results, aligned with the known neurotropism of SARS-CoV-2 and earlier studies, accentuate the need for vigilant psychiatric assessment and support in the era of Long-COVID, especially among younger populations.\nA deep learning transformer model predicts high rates of undiagnosed rare disease in large electronic health systems\nAuthors: Jordan, D. M.; Vy, H. M. T.; Do, R.\nScore: 3.7, Published: 2023-12-24 DOI: 10.1101/2023.12.21.23300393\nIt is estimated that as many as 1 in 16 people worldwide suffer from rare diseases. Rare disease patients face difficulty finding diagnosis and treatment for their conditions, including long diagnostic odysseys, multiple incorrect diagnoses, and unavailable or prohibitively expensive treatments. As a result, it is likely that large electronic health record (EHR) systems include high numbers of participants suffering from undiagnosed rare disease. While this has been shown in detail for specific diseases, these studies are expensive and time consuming and have only been feasible to perform for a handful of the thousands of known rare diseases. The bulk of these undiagnosed cases are effectively hidden, with no straightforward way to differentiate them from healthy controls. The ability to access them at scale would enormously expand our capacity to study and develop drugs for rare diseases, adding to tools aimed at increasing availability of study cohorts for rare disease. In this study, we train a deep learning transformer algorithm, RarePT (Rare-Phenotype Prediction Transformer), to impute undiagnosed rare disease from EHR diagnosis codes in 436,407 participants in the UK Biobank and validated on an independent cohort from 3,333,560 individuals from the Mount Sinai Health System. We applied our model to 155 rare diagnosis codes with fewer than 250 cases each in the UK Biobank and predicted participants with elevated risk for each diagnosis, with the number of participants predicted to be at risk ranging from 85 to 22,000 for different diagnoses. These risk predictions are significantly associated with increased mortality for 65% of diagnoses, with disease burden expressed as disability-adjusted life years (DALY) for 73% of diagnoses, and with 72% of available disease-specific diagnostic tests. They are also highly enriched for known rare diagnoses in patients not included in the training set, with an odds ratio (OR) of 48.0 in cross-validation cohorts of the UK Biobank and an OR of 30.6 in the independent Mount Sinai Health System cohort. Most importantly, RarePT successfully screens for undiagnosed patients in 32 rare diseases with available diagnostic tests in the UK Biobank. Using the trained model to estimate the prevalence of undiagnosed disease in the UK Biobank for these 32 rare phenotypes, we find that at least 50% of patients remain undiagnosed for 20 of 32 diseases. These estimates provide empirical evidence of a high prevalence of undiagnosed rare disease, as well as demonstrating the enormous potential benefit of using RarePT to screen for undiagnosed rare disease patients in large electronic health systems.\nThe Applications of Laser Interstitial Thermal Therapy and Machine Learning in Neurosurgery: A Systematic Review\nAuthors: Bouras, A.; Patel, D.; Chetla, N.\nScore: 1.4, Published: 2023-12-26 DOI: 10.1101/2023.12.21.23300384\nBackgroundThe incorporation of Machine Learning (ML) into Laser Interstitial Thermal Therapy (LITT) represents a significant advancement in minimally invasive neurosurgery, particularly for treating brain tumors, vascular malformations, and epileptogenic foci. This systematic review focuses on evaluating the integration and impact of ML in enhancing the efficacy, precision, and outcomes of LITT in neurosurgical procedures. MethodsAn exhaustive search was conducted in major scientific databases for studies from 2015 to 2023 that specifically focused on the application of ML in LITT. The review assessed the development and implementation of ML algorithms in surgical planning, outcome prediction, and postoperative evaluation in LITT. Rigorous inclusion criteria were applied to select studies, and a combination of meta-analysis and qualitative synthesis was used to analyze the data. ResultsThe review synthesizes findings from a range of studies, including retrospective analyses and initial clinical trials. It highlights the role of ML in enhancing the selection criteria for LITT, optimizing surgical approaches, and improving patient-specific outcome predictions. While LITT showed favorable results in treating non-resectable lesions, the integration of ML was found to potentially refine these outcomes further. However, challenges such as the need for larger sample sizes, standardization of ML algorithms, and validation of these methods in clinical settings were noted. ConclusionsThe integration of ML into LITT procedures marks a promising frontier in neurosurgery, offering potential improvements in surgical accuracy and patient outcomes. The evidence suggests a need for continued development and rigorous testing of ML applications in LITT. Future research should focus on the refinement and validation of ML algorithms for wider clinical adoption, ensuring that technological advancements align with patient safety and treatment efficacy.\nMachine learning in medicine using JavaScript: building web apps using TensorFlow.js for interpreting biomedical datasets\nAuthors: Pires, J. G.\nScore: 5.9, Published: 2023-12-21 DOI: 10.1101/2023.06.21.23291717\nContributions to medicine may come from different areas, and most of these areas are filled with researchers eager to contribute. In this paper, we aim to contribute through the intersection of machine learning and web development. We employed TensorFlow.js, a JavaScript-based library, to model biomedical datasets using neural networks obtained from Kaggle. The principal aim of this study is to present the capabilities of TensorFlow.js and promote its utility in the development of sophisticated machine learning models customized for web-based applications. We modeled three datasets: diabetes detection, surgery complications, and heart failure. While Python and R currently dominate, JavaScript and its derivatives are rapidly gaining ground, offering comparable performance and additional features associated with JavaScript. Kaggle, the public platform from which we downloaded our datasets, provides an extensive collection of biomedical datasets. Therefore, readers can easily test our discussed methods by using the provided codes with minor adjustments on any case of their interest. The results demonstrate an accuracy of 92% for diabetes detection, almost 100% for surgery complications, and 80% for heart failure. The possibilities are vast, and we believe that this is an excellent option for researchers focusing on web applications, particularly in the field of medicine.\nEmpowering Transformers for Evidence-Based Medicine\nAuthors: Mohammed, S.; Fiaidhi, J.; Shaik, H.\nScore: 0.8, Published: 2023-12-28 DOI: 10.1101/2023.12.25.23300520\nBreaking the barrier for practicing evidence-based medicine rely on effective methods for rapidly identifying relevant evidences from the body of biomedical literature. An important challenge confronted by the medical practitioners is the long time needed to browse, filter, summarize and compile information from different medical resources. Deep learning can help in solving this based on the automatic question answering (Q\u0026A) and transformers. However, Q\u0026A and transformers technologies are not trained to answer clinical queries that can be used for evidence-based practice nor it can respond to structured clinical questioning protocol like PICO (Patient/Problem, Intervention, Comparison and Outcome). This article describes the use of deep learning techniques for Q\u0026A that is based on transformer models like BERT and GPT to answer PICO clinical questions that can be used for evidence-based practice extracted from sound medical research resources like PubMed. We are reporting acceptable clinical answers that are supported by findings from PubMed. Our transformer methods are reaching an acceptable state of the art performance based on two staged bootstrapping process involving filtering relevant articles followed by identifying articles that support the requested outcome expressed by the PICO question. Moreover, we are also reporting experimentations to empower our bootstrapping techniques with patch attentions to the most important keywords in the clinical case and the PICO questions. Our bootstrapped patched with attention is showing relevancy of the evidences collected based on an entropy metrics.\nAnalysis of Pleiotropy for Testosterone and Lipid Profiles in Males and Females\nAuthors: Chattopadhyay, S.; Bhattacharyya, S.; Basu, S.\nScore: 0.5, Published: 2023-12-29 DOI: 10.1101/2023.12.27.23300595\nIn modern scientific studies, it is often imperative to determine whether a set of phenotypes is affected by a single factor. If such an influence is identified, it becomes essential to discern whether this effect is contingent upon categories such as sex or age group, and importantly, to understand whether this dependence is rooted in purely non-environmental reasons. The exploration of such dependencies often involves studying pleiotropy, a phenomenon wherein a single genetic locus impacts multiple traits. This heightened interest in uncovering dependencies by pleiotropy is fueled by the growing accessibility of summary statistics from genome-wide association studies (GWAS) and the establishment of thoroughly phenotyped sample collections. This advancement enables a systematic and comprehensive exploration of the genetic connections among various traits and diseases. additive genetic correlation illuminates the genetic connection between two traits, providing valuable insights into the shared biological pathways and underlying causal relationships between them. In this paper, we present a novel method to analyze such dependencies by studying additive genetic correlations between pairs of traits under consideration. Subsequently, we employ matrix comparison techniques to discern and elucidate sex-specific or age-group-specific associations, contributing to a deeper understanding of the nuanced dependencies within the studied traits. Our proposed method is computationally handy and requires only GWAS summary statistics. We validate our method by applying it to the UK Biobank data and present the results.\nHumanELY: Human evaluation of LLM yield, using a novel web-based evaluation tool\nAuthors: Awasthi, R.; Mishra, S.; Mahapatra, D.; Khanna, A.; Maheshwari, K.; Cywinski, J.; Papay, F.; Mathur, P.\nScore: 0.5, Published: 2023-12-30 DOI: 10.1101/2023.12.22.23300458\nAO_SCPLOWBSTRACTC_SCPLOWLarge language models (LLMs) have caught the imagination of researchers,developers and public in general the world over with their potential for transformation. Vast amounts of research and development resources are being provided to implement these models in all facets of life. Trained using billions of parameters, various measures of their accuracy and performance have been proposed and used in recent times. While many of the automated natural language assessment parameters measure LLM output performance for use of language, contextual outputs are still hard to measure and quantify. Hence, human evaluation is still an important measure of LLM performance,even though it has been applied variably and inconsistently due to lack of guidance and resource limitations. To provide a structured way to perform comprehensive human evaluation of LLM output, we propose the first guidance and tool called HumanELY. Our approach and tool built using prior knowledge helps perform evaluation of LLM outputs in a comprehensive, consistent, measurable and comparable manner. HumanELY comprises of five key evaluation metrics: relevance, coverage, coherence, harm and comparison. Additional submetrics within these five key metrics provide for Likert scale based human evaluation of LLM outputs. Our related webtool uses this HumanELY guidance to enable LLM evaluation and provide data for comparison against different users performing human evaluation. While all metrics may not be relevant and pertinent to all outputs, it is important to assess and address their use. Lastly, we demonstrate comparison of metrics used in HumanELY against some of the recent publications in the healthcare domain. We focused on the healthcare domain due to the need to demonstrate highest levels of accuracy and lowest levels of harm in a comprehensive manner. We anticipate our guidance and tool to be used for any domain where LLMs find an use case. Link to the HumanELY Toolhttps://www.brainxai.com/humanely\nAutomated self-service cohort selection for large-scale population sciences and observational research: The California Teachers Study Researcher Platform\nAuthors: Lacey, J. V.; Spielfogel, E. S.; Benbow, J. L.; Savage, K. E.; Lin, K.; Anderson, C. A. M.; Clague-DeHart, J.; Duffy, C. N.; Martinez, M. E.; Park, H. L.; Thompson, C. A.; Wang, S. S.; Chandra, S.\nScore: 0.5, Published: 2023-12-26 DOI: 10.1101/2023.12.22.23300461\nObjectiveCohort selection is ubiquitous and essential, but manual and ad hoc approaches are time-consuming, labor-intense, and difficult to scale. We sought to automate the task of cohort selection by building self-service tools that enable researchers to independently generate datasets for population sciences research. Materials and MethodsThe California Teachers Study (CTS) is a prospective observational study of 133,477 women who have been followed continuously since 1995. The CTS includes extensive survey-based and real-world data from cancer, hospitalization, and mortality linkages. We curated data from our data warehouse into a column-oriented database and developed a researcher-facing web application that guides researchers through the project lifecycle; captures researchers inputs; and automatically generates custom and analysis-ready data, code, dictionaries, and documentation. ResultsResearchers can register, access data, and propose projects on the CTS Researcher Platform via our CTS website. The Platform supports cohort and cross-sectional study designs for cancer, mortality, and any other ICD-based phenotypes or endpoints. User-friendly prompts and menus capture analytic design, inclusion/exclusion criteria, endpoint definitions, censoring rules, and covariate selection. Our platform empowers researchers everywhere to query, choose, review, and automatically and quickly receive custom data, analytic scripts, and documentation for their research projects. Research teams can review, revise, and update their choices anytime. DiscussionWe replaced inefficient traditional cohort-selection processes with an integrated self-service approach that simplifies and improves cohort selection for all stakeholders. Compared with manual methods, our solution is faster and more scalable, user-friendly, and collaborative. Other studies could re-configure our individual database, project-tracking, website, and data-delivery components for their own specific needs, or they could utilize other widely available solutions (e.g., alternative database or project-tracking tools) to enable similarly automated cohort-selection in their own settings. Our comprehensive and flexible framework could be adopted to improve cohort selection in other population sciences and observational research settings.\nDetecting QT prolongation From a Single-lead ECG With Deep Learning\nAuthors: Alam, R.; Aguirre, A.; Stultz, C.\nScore: 0.5, Published: 2023-12-24 DOI: 10.1101/2023.12.21.23300388\nBackground and AimsFor a number of antiarrhythmics, drug loading requires a 3-day hospitalization with monitoring for QT-prolongation. Automated QT monitoring with wearable ECG monitors would facilitate out-of-hospital care. We aim to develop a deep learning model that infers QT intervals from ECG lead-I - the lead most often acquired from ambulatory ECG monitors - and we use this model to detect clinically meaningful QT-prolongation episodes during Dofetilide drug loading. MethodsUsing 4.22 million 12-lead ECG recordings from 903.6 thousand patients at the Massachusetts General Hospital, we develop a deep learning model, QTNet, that infers QT intervals from lead-I. Over 3 million ECGs from 653 thousand patients are used to train the model and an internal-test set containing 633 thousand ECGs from 135 thousand patients was used for testing. QTNet is further evaluated on an external-validation set containing 3.1 million ECGs from 667 thousand patients at another institution. QTNet was used to detect Dofetilide-induced QT prolongation in a publicly available database (ECGRDVQ-dataset) containing ECGs from subjects enrolled in a clinical trial evaluating the effects of antiarrhythmic drugs. ResultsQTNet achieves mean absolute errors of 12.63ms (internal-test) and 12.30ms (external-validation) for estimating absolute QT intervals. The associated Pearson correlation coefficients are 0.91 (internal-test) and 0.92 (external-validation). For the ECGRDVQ-dataset, QTNet detects Dofetilide-induced QTc prolongation with 87% sensitivity and 77% specificity. The negative predictive value of the model is greater than 95% when the pre-test probability of drug-induced QTc prolongation is below 25%. ConclusionsDrug-induced QT prolongation risk can be tracked from ECG lead-I using deep learning. This research leads the path toward out-of-hospital care using wearable ECG devices for antiarrhythmic therapies. Whats New?O_LIUsing only Lead-I ECG, a novel deep neural network, QTNet, can estimate the QT intervals that are similar to those generated from the 12-lead ECG by the clinical ECG machines, with a mean absolute error of 12ms and a Pearson correlation coefficient of 0.91. C_LIO_LIThe same QTNet, when applied without any fine-tuning on an external population undergoing Dofetilide loading, can identify whether and when clinically critical QT prolongation occurs after the drug loading. In comparison to manual annotations of QT intervals by clinical experts, QTNet achieves 87% sensitivity and 77% specificity. C_LIO_LIQTNet is a novel regression model that can be used on Lead-I ECG streams, potentially from wearable devices at out-of-hospital settings, for health critical applications such as drug-induced QT prolongation tracking. C_LI\n",
  "wordCount" : "2859",
  "inLanguage": "en",
  "datePublished": "2023-12-31T10:38:19Z",
  "dateModified": "2023-12-31T10:38:19Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/health-informatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      health informatics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 31, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.23299473">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.23299473" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.23299473">
        <p class="paperTitle">SARS-CoV-2 Infection is Associated with an Increase in New Diagnoses of Schizophrenia Spectrum and Psychotic Disorder: A Study Using the US National COVID Cohort Collaborative (N3C)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.23299473" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.23299473" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rahman, A.; Russell, M.; Zheng, W.; Eckrich, D.; Ahmed, I.; on behalf of the N3C Consortium,  </p>
        <p class="info">Score: 703.8, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.23299473' target='https://doi.org/10.1101/2023.12.05.23299473'> 10.1101/2023.12.05.23299473</a></p>
        <p class="abstract">Amid the ongoing global repercussions of SARS-CoV-2, its crucial to comprehend its potential long-term psychiatric effects. Several recent studies have suggested a link between COVID-19 and subsequent mental health disorders. Our investigation joins this exploration, concentrating on Schizophrenia Spectrum and Psychotic Disorders (SSPD). Different from other studies, we took acute respiratory distress syndrome (ARDS) and COVID-19 lab negative cohorts as control groups to accurately gauge the impact of COVID-19 on SSPD. Data from 19,344,698 patients, sourced from the N3C Data Enclave platform, were methodically filtered to create propensity matched cohorts: ARDS (n = 222,337), COVID-positive (n = 219,264), and COVID-negative (n = 213,183). We systematically analyzed the hazard rate of new-onset SSPD across three distinct time intervals: 0-21 days, 22-90 days, and beyond 90 days post-infection. COVID-19 positive patients consistently exhibited a heightened hazard ratio (HR) across all intervals [0-21 days (HR: 4.6; CI: 3.7-5.7), 22-90 days (HR: 2.9; CI: 2.3 -3.8), beyond 90 days (HR: 1.7; CI: 1.5-1.)]. These are notably higher than both ARDS and COVID-19 lab-negative patients. Validations using various tests, including the Cochran Mantel Haenszel Test, Wald Test, and Log-rank Test confirmed these associations. Intriguingly, our data indicated that younger individuals face a heightened risk of SSPD after contracting COVID-19, a trend not observed in the ARDS and COVID-negative groups. These results, aligned with the known neurotropism of SARS-CoV-2 and earlier studies, accentuate the need for vigilant psychiatric assessment and support in the era of Long-COVID, especially among younger populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.23300393">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.23300393" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.23300393">
        <p class="paperTitle">A deep learning transformer model predicts high rates of undiagnosed rare disease in large electronic health systems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.23300393" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.23300393" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jordan, D. M.; Vy, H. M. T.; Do, R.</p>
        <p class="info">Score: 3.7, Published: 2023-12-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.23300393' target='https://doi.org/10.1101/2023.12.21.23300393'> 10.1101/2023.12.21.23300393</a></p>
        <p class="abstract">It is estimated that as many as 1 in 16 people worldwide suffer from rare diseases. Rare disease patients face difficulty finding diagnosis and treatment for their conditions, including long diagnostic odysseys, multiple incorrect diagnoses, and unavailable or prohibitively expensive treatments. As a result, it is likely that large electronic health record (EHR) systems include high numbers of participants suffering from undiagnosed rare disease. While this has been shown in detail for specific diseases, these studies are expensive and time consuming and have only been feasible to perform for a handful of the thousands of known rare diseases. The bulk of these undiagnosed cases are effectively hidden, with no straightforward way to differentiate them from healthy controls. The ability to access them at scale would enormously expand our capacity to study and develop drugs for rare diseases, adding to tools aimed at increasing availability of study cohorts for rare disease. In this study, we train a deep learning transformer algorithm, RarePT (Rare-Phenotype Prediction Transformer), to impute undiagnosed rare disease from EHR diagnosis codes in 436,407 participants in the UK Biobank and validated on an independent cohort from 3,333,560 individuals from the Mount Sinai Health System. We applied our model to 155 rare diagnosis codes with fewer than 250 cases each in the UK Biobank and predicted participants with elevated risk for each diagnosis, with the number of participants predicted to be at risk ranging from 85 to 22,000 for different diagnoses. These risk predictions are significantly associated with increased mortality for 65% of diagnoses, with disease burden expressed as disability-adjusted life years (DALY) for 73% of diagnoses, and with 72% of available disease-specific diagnostic tests. They are also highly enriched for known rare diagnoses in patients not included in the training set, with an odds ratio (OR) of 48.0 in cross-validation cohorts of the UK Biobank and an OR of 30.6 in the independent Mount Sinai Health System cohort. Most importantly, RarePT successfully screens for undiagnosed patients in 32 rare diseases with available diagnostic tests in the UK Biobank. Using the trained model to estimate the prevalence of undiagnosed disease in the UK Biobank for these 32 rare phenotypes, we find that at least 50% of patients remain undiagnosed for 20 of 32 diseases. These estimates provide empirical evidence of a high prevalence of undiagnosed rare disease, as well as demonstrating the enormous potential benefit of using RarePT to screen for undiagnosed rare disease patients in large electronic health systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.23300384">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.23300384" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.23300384">
        <p class="paperTitle">The Applications of Laser Interstitial Thermal Therapy and Machine Learning in Neurosurgery: A Systematic Review</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.23300384" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.23300384" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bouras, A.; Patel, D.; Chetla, N.</p>
        <p class="info">Score: 1.4, Published: 2023-12-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.23300384' target='https://doi.org/10.1101/2023.12.21.23300384'> 10.1101/2023.12.21.23300384</a></p>
        <p class="abstract">BackgroundThe incorporation of Machine Learning (ML) into Laser Interstitial Thermal Therapy (LITT) represents a significant advancement in minimally invasive neurosurgery, particularly for treating brain tumors, vascular malformations, and epileptogenic foci. This systematic review focuses on evaluating the integration and impact of ML in enhancing the efficacy, precision, and outcomes of LITT in neurosurgical procedures.

MethodsAn exhaustive search was conducted in major scientific databases for studies from 2015 to 2023 that specifically focused on the application of ML in LITT. The review assessed the development and implementation of ML algorithms in surgical planning, outcome prediction, and postoperative evaluation in LITT. Rigorous inclusion criteria were applied to select studies, and a combination of meta-analysis and qualitative synthesis was used to analyze the data.

ResultsThe review synthesizes findings from a range of studies, including retrospective analyses and initial clinical trials. It highlights the role of ML in enhancing the selection criteria for LITT, optimizing surgical approaches, and improving patient-specific outcome predictions. While LITT showed favorable results in treating non-resectable lesions, the integration of ML was found to potentially refine these outcomes further. However, challenges such as the need for larger sample sizes, standardization of ML algorithms, and validation of these methods in clinical settings were noted.

ConclusionsThe integration of ML into LITT procedures marks a promising frontier in neurosurgery, offering potential improvements in surgical accuracy and patient outcomes. The evidence suggests a need for continued development and rigorous testing of ML applications in LITT. Future research should focus on the refinement and validation of ML algorithms for wider clinical adoption, ensuring that technological advancements align with patient safety and treatment efficacy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.23291717">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.23291717" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.23291717">
        <p class="paperTitle">Machine learning in medicine using JavaScript: building web apps using TensorFlow.js for interpreting biomedical datasets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.23291717" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.23291717" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pires, J. G.</p>
        <p class="info">Score: 5.9, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.23291717' target='https://doi.org/10.1101/2023.06.21.23291717'> 10.1101/2023.06.21.23291717</a></p>
        <p class="abstract">Contributions to medicine may come from different areas, and most of these areas are filled with researchers eager to contribute. In this paper, we aim to contribute through the intersection of machine learning and web development. We employed TensorFlow.js, a JavaScript-based library, to model biomedical datasets using neural networks obtained from Kaggle. The principal aim of this study is to present the capabilities of TensorFlow.js and promote its utility in the development of sophisticated machine learning models customized for web-based applications. We modeled three datasets: diabetes detection, surgery complications, and heart failure. While Python and R currently dominate, JavaScript and its derivatives are rapidly gaining ground, offering comparable performance and additional features associated with JavaScript. Kaggle, the public platform from which we downloaded our datasets, provides an extensive collection of biomedical datasets. Therefore, readers can easily test our discussed methods by using the provided codes with minor adjustments on any case of their interest. The results demonstrate an accuracy of 92% for diabetes detection, almost 100% for surgery complications, and 80% for heart failure. The possibilities are vast, and we believe that this is an excellent option for researchers focusing on web applications, particularly in the field of medicine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.25.23300520">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.25.23300520" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.25.23300520">
        <p class="paperTitle">Empowering Transformers for Evidence-Based Medicine</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.25.23300520" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.25.23300520" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mohammed, S.; Fiaidhi, J.; Shaik, H.</p>
        <p class="info">Score: 0.8, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.25.23300520' target='https://doi.org/10.1101/2023.12.25.23300520'> 10.1101/2023.12.25.23300520</a></p>
        <p class="abstract">Breaking the barrier for practicing evidence-based medicine rely on effective methods for rapidly identifying relevant evidences from the body of biomedical literature. An important challenge confronted by the medical practitioners is the long time needed to browse, filter, summarize and compile information from different medical resources. Deep learning can help in solving this based on the automatic question answering (Q&amp;A) and transformers. However, Q&amp;A and transformers technologies are not trained to answer clinical queries that can be used for evidence-based practice nor it can respond to structured clinical questioning protocol like PICO (Patient/Problem, Intervention, Comparison and Outcome). This article describes the use of deep learning techniques for Q&amp;A that is based on transformer models like BERT and GPT to answer PICO clinical questions that can be used for evidence-based practice extracted from sound medical research resources like PubMed. We are reporting acceptable clinical answers that are supported by findings from PubMed. Our transformer methods are reaching an acceptable state of the art performance based on two staged bootstrapping process involving filtering relevant articles followed by identifying articles that support the requested outcome expressed by the PICO question. Moreover, we are also reporting experimentations to empower our bootstrapping techniques with patch attentions to the most important keywords in the clinical case and the PICO questions. Our bootstrapped patched with attention is showing relevancy of the evidences collected based on an entropy metrics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.27.23300595">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.27.23300595" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.27.23300595">
        <p class="paperTitle">Analysis of Pleiotropy for Testosterone and Lipid Profiles in Males and Females</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.27.23300595" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.27.23300595" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chattopadhyay, S.; Bhattacharyya, S.; Basu, S.</p>
        <p class="info">Score: 0.5, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.27.23300595' target='https://doi.org/10.1101/2023.12.27.23300595'> 10.1101/2023.12.27.23300595</a></p>
        <p class="abstract">In modern scientific studies, it is often imperative to determine whether a set of phenotypes is affected by a single factor. If such an influence is identified, it becomes essential to discern whether this effect is contingent upon categories such as sex or age group, and importantly, to understand whether this dependence is rooted in purely non-environmental reasons. The exploration of such dependencies often involves studying pleiotropy, a phenomenon wherein a single genetic locus impacts multiple traits. This heightened interest in uncovering dependencies by pleiotropy is fueled by the growing accessibility of summary statistics from genome-wide association studies (GWAS) and the establishment of thoroughly phenotyped sample collections. This advancement enables a systematic and comprehensive exploration of the genetic connections among various traits and diseases. additive genetic correlation illuminates the genetic connection between two traits, providing valuable insights into the shared biological pathways and underlying causal relationships between them. In this paper, we present a novel method to analyze such dependencies by studying additive genetic correlations between pairs of traits under consideration. Subsequently, we employ matrix comparison techniques to discern and elucidate sex-specific or age-group-specific associations, contributing to a deeper understanding of the nuanced dependencies within the studied traits. Our proposed method is computationally handy and requires only GWAS summary statistics. We validate our method by applying it to the UK Biobank data and present the results.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.23300458">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.23300458" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.23300458">
        <p class="paperTitle">HumanELY: Human evaluation of LLM yield, using a novel web-based evaluation tool</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.23300458" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.23300458" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Awasthi, R.; Mishra, S.; Mahapatra, D.; Khanna, A.; Maheshwari, K.; Cywinski, J.; Papay, F.; Mathur, P.</p>
        <p class="info">Score: 0.5, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.23300458' target='https://doi.org/10.1101/2023.12.22.23300458'> 10.1101/2023.12.22.23300458</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWLarge language models (LLMs) have caught the imagination of researchers,developers and public in general the world over with their potential for transformation. Vast amounts of research and development resources are being provided to implement these models in all facets of life. Trained using billions of parameters, various measures of their accuracy and performance have been proposed and used in recent times. While many of the automated natural language assessment parameters measure LLM output performance for use of language, contextual outputs are still hard to measure and quantify. Hence, human evaluation is still an important measure of LLM performance,even though it has been applied variably and inconsistently due to lack of guidance and resource limitations.

To provide a structured way to perform comprehensive human evaluation of LLM output, we propose the first guidance and tool called HumanELY. Our approach and tool built using prior knowledge helps perform evaluation of LLM outputs in a comprehensive, consistent, measurable and comparable manner. HumanELY comprises of five key evaluation metrics: relevance, coverage, coherence, harm and comparison. Additional submetrics within these five key metrics provide for Likert scale based human evaluation of LLM outputs. Our related webtool uses this HumanELY guidance to enable LLM evaluation and provide data for comparison against different users performing human evaluation. While all metrics may not be relevant and pertinent to all outputs, it is important to assess and address their use.

Lastly, we demonstrate comparison of metrics used in HumanELY against some of the recent publications in the healthcare domain. We focused on the healthcare domain due to the need to demonstrate highest levels of accuracy and lowest levels of harm in a comprehensive manner. We anticipate our guidance and tool to be used for any domain where LLMs find an use case.

Link to the HumanELY Toolhttps://www.brainxai.com/humanely</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.23300461">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.23300461" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.23300461">
        <p class="paperTitle">Automated self-service cohort selection for large-scale population sciences and observational research:  The California Teachers Study Researcher Platform</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.23300461" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.23300461" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lacey, J. V.; Spielfogel, E. S.; Benbow, J. L.; Savage, K. E.; Lin, K.; Anderson, C. A. M.; Clague-DeHart, J.; Duffy, C. N.; Martinez, M. E.; Park, H. L.; Thompson, C. A.; Wang, S. S.; Chandra, S.</p>
        <p class="info">Score: 0.5, Published: 2023-12-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.23300461' target='https://doi.org/10.1101/2023.12.22.23300461'> 10.1101/2023.12.22.23300461</a></p>
        <p class="abstract">ObjectiveCohort selection is ubiquitous and essential, but manual and ad hoc approaches are time-consuming, labor-intense, and difficult to scale. We sought to automate the task of cohort selection by building self-service tools that enable researchers to independently generate datasets for population sciences research.

Materials and MethodsThe California Teachers Study (CTS) is a prospective observational study of 133,477 women who have been followed continuously since 1995. The CTS includes extensive survey-based and real-world data from cancer, hospitalization, and mortality linkages. We curated data from our data warehouse into a column-oriented database and developed a researcher-facing web application that guides researchers through the project lifecycle; captures researchers inputs; and automatically generates custom and analysis-ready data, code, dictionaries, and documentation.

ResultsResearchers can register, access data, and propose projects on the CTS Researcher Platform via our CTS website. The Platform supports cohort and cross-sectional study designs for cancer, mortality, and any other ICD-based phenotypes or endpoints. User-friendly prompts and menus capture analytic design, inclusion/exclusion criteria, endpoint definitions, censoring rules, and covariate selection. Our platform empowers researchers everywhere to query, choose, review, and automatically and quickly receive custom data, analytic scripts, and documentation for their research projects. Research teams can review, revise, and update their choices anytime.

DiscussionWe replaced inefficient traditional cohort-selection processes with an integrated self-service approach that simplifies and improves cohort selection for all stakeholders. Compared with manual methods, our solution is faster and more scalable, user-friendly, and collaborative. Other studies could re-configure our individual database, project-tracking, website, and data-delivery components for their own specific needs, or they could utilize other widely available solutions (e.g., alternative database or project-tracking tools) to enable similarly automated cohort-selection in their own settings. Our comprehensive and flexible framework could be adopted to improve cohort selection in other population sciences and observational research settings.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.23300388">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.23300388" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.23300388">
        <p class="paperTitle">Detecting QT prolongation From a Single-lead ECG With Deep Learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.23300388" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.23300388" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alam, R.; Aguirre, A.; Stultz, C.</p>
        <p class="info">Score: 0.5, Published: 2023-12-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.23300388' target='https://doi.org/10.1101/2023.12.21.23300388'> 10.1101/2023.12.21.23300388</a></p>
        <p class="abstract">Background and AimsFor a number of antiarrhythmics, drug loading requires a 3-day hospitalization with monitoring for QT-prolongation. Automated QT monitoring with wearable ECG monitors would facilitate out-of-hospital care. We aim to develop a deep learning model that infers QT intervals from ECG lead-I - the lead most often acquired from ambulatory ECG monitors - and we use this model to detect clinically meaningful QT-prolongation episodes during Dofetilide drug loading.

MethodsUsing 4.22 million 12-lead ECG recordings from 903.6 thousand patients at the Massachusetts General Hospital, we develop a deep learning model, QTNet, that infers QT intervals from lead-I. Over 3 million ECGs from 653 thousand patients are used to train the model and an internal-test set containing 633 thousand ECGs from 135 thousand patients was used for testing. QTNet is further evaluated on an external-validation set containing 3.1 million ECGs from 667 thousand patients at another institution. QTNet was used to detect Dofetilide-induced QT prolongation in a publicly available database (ECGRDVQ-dataset) containing ECGs from subjects enrolled in a clinical trial evaluating the effects of antiarrhythmic drugs.

ResultsQTNet achieves mean absolute errors of 12.63ms (internal-test) and 12.30ms (external-validation) for estimating absolute QT intervals. The associated Pearson correlation coefficients are 0.91 (internal-test) and 0.92 (external-validation). For the ECGRDVQ-dataset, QTNet detects Dofetilide-induced QTc prolongation with 87% sensitivity and 77% specificity. The negative predictive value of the model is greater than 95% when the pre-test probability of drug-induced QTc prolongation is below 25%.

ConclusionsDrug-induced QT prolongation risk can be tracked from ECG lead-I using deep learning. This research leads the path toward out-of-hospital care using wearable ECG devices for antiarrhythmic therapies.

Whats New?O_LIUsing only Lead-I ECG, a novel deep neural network, QTNet, can estimate the QT intervals that are similar to those generated from the 12-lead ECG by the clinical ECG machines, with a mean absolute error of 12ms and a Pearson correlation coefficient of 0.91.
C_LIO_LIThe same QTNet, when applied without any fine-tuning on an external population undergoing Dofetilide loading, can identify whether and when clinically critical QT prolongation occurs after the drug loading. In comparison to manual annotations of QT intervals by clinical experts, QTNet achieves 87% sensitivity and 77% specificity.
C_LIO_LIQTNet is a novel regression model that can be used on Lead-I ECG streams, potentially from wearable devices at out-of-hospital settings, for health critical applications such as drug-induced QT prolongation tracking.
C_LI</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
