<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>health informatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>health informatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="SARS-CoV-2 Orphan Gene ORF10 Contributes to More Severe COVID-19 Disease
Authors: Haltom, J. A.; Trovao, N. S.; Guarnieri, J.; Vincent, P.; Singh, U.; Tsoy, S.; O&#39;Leary, C. A.; Bram, Y.; Widjaja, G. A.; Cen, Z.; Meller, R.; Baylin, S. B.; Moss, W. N.; Nikolau, B. J.; Enguita, F. J.; Wallace, D. C.; Beheshti, A.; Schwartz, R.; Wurtele, E. S.
Score: 24.9, Published: 2023-11-29 DOI: 10.1101/2023.11.27.23298847
The orphan gene of SARS-CoV-2, ORF10, is the least studied gene in the virus responsible for the COVID-19 pandemic.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/health-informatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="health informatics" />
<meta property="og:description" content="SARS-CoV-2 Orphan Gene ORF10 Contributes to More Severe COVID-19 Disease
Authors: Haltom, J. A.; Trovao, N. S.; Guarnieri, J.; Vincent, P.; Singh, U.; Tsoy, S.; O&#39;Leary, C. A.; Bram, Y.; Widjaja, G. A.; Cen, Z.; Meller, R.; Baylin, S. B.; Moss, W. N.; Nikolau, B. J.; Enguita, F. J.; Wallace, D. C.; Beheshti, A.; Schwartz, R.; Wurtele, E. S.
Score: 24.9, Published: 2023-11-29 DOI: 10.1101/2023.11.27.23298847
The orphan gene of SARS-CoV-2, ORF10, is the least studied gene in the virus responsible for the COVID-19 pandemic." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/health-informatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-06T10:40:04+00:00" />
<meta property="article:modified_time" content="2023-12-06T10:40:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="health informatics"/>
<meta name="twitter:description" content="SARS-CoV-2 Orphan Gene ORF10 Contributes to More Severe COVID-19 Disease
Authors: Haltom, J. A.; Trovao, N. S.; Guarnieri, J.; Vincent, P.; Singh, U.; Tsoy, S.; O&#39;Leary, C. A.; Bram, Y.; Widjaja, G. A.; Cen, Z.; Meller, R.; Baylin, S. B.; Moss, W. N.; Nikolau, B. J.; Enguita, F. J.; Wallace, D. C.; Beheshti, A.; Schwartz, R.; Wurtele, E. S.
Score: 24.9, Published: 2023-11-29 DOI: 10.1101/2023.11.27.23298847
The orphan gene of SARS-CoV-2, ORF10, is the least studied gene in the virus responsible for the COVID-19 pandemic."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "health informatics",
      "item": "https://trxiv.yorks0n.com/posts/health-informatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "health informatics",
  "name": "health informatics",
  "description": "SARS-CoV-2 Orphan Gene ORF10 Contributes to More Severe COVID-19 Disease\nAuthors: Haltom, J. A.; Trovao, N. S.; Guarnieri, J.; Vincent, P.; Singh, U.; Tsoy, S.; O\u0026#39;Leary, C. A.; Bram, Y.; Widjaja, G. A.; Cen, Z.; Meller, R.; Baylin, S. B.; Moss, W. N.; Nikolau, B. J.; Enguita, F. J.; Wallace, D. C.; Beheshti, A.; Schwartz, R.; Wurtele, E. S.\nScore: 24.9, Published: 2023-11-29 DOI: 10.1101/2023.11.27.23298847\nThe orphan gene of SARS-CoV-2, ORF10, is the least studied gene in the virus responsible for the COVID-19 pandemic.",
  "keywords": [
    
  ],
  "articleBody": " SARS-CoV-2 Orphan Gene ORF10 Contributes to More Severe COVID-19 Disease\nAuthors: Haltom, J. A.; Trovao, N. S.; Guarnieri, J.; Vincent, P.; Singh, U.; Tsoy, S.; O'Leary, C. A.; Bram, Y.; Widjaja, G. A.; Cen, Z.; Meller, R.; Baylin, S. B.; Moss, W. N.; Nikolau, B. J.; Enguita, F. J.; Wallace, D. C.; Beheshti, A.; Schwartz, R.; Wurtele, E. S.\nScore: 24.9, Published: 2023-11-29 DOI: 10.1101/2023.11.27.23298847\nThe orphan gene of SARS-CoV-2, ORF10, is the least studied gene in the virus responsible for the COVID-19 pandemic. Recent experimentation indicated ORF10 expression moderates innate immunity in vitro. However, whether ORF10 affects COVID-19 in humans remained unknown. We determine that the ORF10 sequence is identical to the Wuhan-Hu-1 ancestral haplotype in 95% of genomes across five variants of concern (VOC). Four ORF10 variants are associated with less virulent clinical outcomes in the human host: three of these affect ORF10 protein structure, one affects ORF10 RNA structural dynamics. RNA-Seq data from 2070 samples from diverse human cells and tissues reveals ORF10 accumulation is conditionally discordant from that of other SARS-CoV-2 transcripts. Expression of ORF10 in A549 and HEK293 cells perturbs immune-related gene expression networks, alters expression of the majority of mitochondrially-encoded genes of oxidative respiration, and leads to large shifts in levels of 14 newly-identified transcripts. We conclude ORF10 contributes to more severe COVID-19 clinical outcomes in the human host.\nSARS-CoV-2 Infection is Associated with an Increase in New Diagnoses of Schizophrenia Spectrum and Psychotic Disorder: A Study Using the US National COVID Cohort Collaborative (N3C)\nAuthors: Rahman, A.; Russell, M.; Zheng, W.; Eckrich, D.; Ahmed, I.; on behalf of the N3C Consortium, Score: 9.4, Published: 2023-12-05 DOI: 10.1101/2023.12.05.23299473\nAmid the ongoing global repercussions of SARS-CoV-2, its crucial to comprehend its potential long-term psychiatric effects. Several recent studies have suggested a link between COVID-19 and subsequent mental health disorders. Our investigation joinsthis exploration, concentrating on Schizophrenia Spectrum and Psychotic Disorders (SSPD). Different from other studies, we took acute respiratory distress syndrome (ARDS) and COVID-19 lab negative cohorts as control groups to accurately gauge the impact of COVID-19 on SSPD. Data from 19,344,698 patients, sourced from the N3C Data Enclave platform, were methodically filtered to create propensity matched cohorts: ARDS (n = 222,337), COVID-positive (n = 219,264), and COVID-negative (n = 213,183). We systematically analyzed the hazard rate of new-onset SSPD across three distinct time intervals: 0-21 days, 22-90 days, and beyond 90 days post-infection. COVID-19 positive patients consistently exhibited a heightened hazard ratio (HR) across all intervals [0-21 days (HR: 4.6; CI: 3.7-5.7), 22-90 days (HR: 2.9; CI: 2.3 -3.8), beyond 90 days (HR: 1.7; CI: 1.5-1.)]. These are notably higher than both ARDS and COVID-19 lab-negative patients. Validations using various tests, including the Cochran Mantel Haenszel Test, Wald Test, and Log-rank Test confirmed these associations. Intriguingly, our data indicated that younger individuals face a heightened risk of SSPD after contracting COVID-19, a trend not observed in the ARDS and COVID-negative groups. These results, aligned with the known neurotropism of SARS-CoV-2 and earlier studies, accentuate the need for vigilant psychiatric assessment and support in the era of Long-COVID, especially among younger populations.\nLarge Language Models and Medical Knowledge Grounding for Diagnosis Prediction\nAuthors: Gao, Y.; Li, R.; Croxford, E.; Tesch, S.; To, D.; Caskey, J.; Patterson, B. W.; Churpek, M. M.; Miller, T.; Dligach, D.; Afshar, M.\nScore: 6.0, Published: 2023-11-27 DOI: 10.1101/2023.11.24.23298641\nWhile Large Language Models (LLMs) have showcased their potential in diverse language tasks, their application in the healthcare arena needs to ensure the minimization of diagnostic errors and the prevention of patient harm. A Medical Knowledge Graph (KG) houses a wealth of structured medical concept relations sourced from authoritative references, such as UMLS, making it a valuable resource to ground LLMs diagnostic process in knowledge. In this paper, we examine the synergistic potential of LLMs and medical KG in predicting diagnoses given electronic health records (EHR), under the framework of Retrieval-augmented generation (RAG). We proposed a novel graph model: DO_SCPLOWRC_SCPLOW.KO_SCPLOWNOWSC_SCPLOW, that selects the most relevant pathology knowledge paths based on the medical problem descriptions. In order to evaluate DO_SCPLOWRC_SCPLOW.KO_SCPLOWNOWSC_SCPLOW, we developed the first comprehensive human evaluation approach to assess the performance of LLMs for diagnosis prediction and examine the rationale behind their decision-making processes, aimed at improving diagnostic safety. Using real-world hospital datasets, our study serves to enrich the discourse on the role of medical KGs in grounding medical knowledge into LLMs, revealing both challenges and opportunities in harnessing external knowledge for explainable diagnostic pathway and the realization of AI-augmented diagnostic decision support systems.\nChatGPT Influence on Medical Decision-Making, Bias, and Equity: A Randomized Study of Clinicians Evaluating Clinical Vignettes\nAuthors: Goh, E.; Bunning, B.; Khoong, E.; Gallo, R.; Milstein, A.; Centola, D.; Chen, J. H.\nScore: 7.2, Published: 2023-11-27 DOI: 10.1101/2023.11.24.23298844\nIn a randomized, pre-post intervention study, we evaluated the influence of a large language model (LLM) generative AI system on accuracy of physician decision-making and bias in healthcare. 50 US-licensed physicians reviewed a video clinical vignette, featuring actors representing different demographics (a White male or a Black female) with chest pain. Participants were asked to answer clinical questions around triage, risk, and treatment based on these vignettes, then asked to reconsider after receiving advice generated by ChatGPT+ (GPT4). The primary outcome was the accuracy of clinical decisions based on pre-established evidence-based guidelines. Results showed that physicians are willing to change their initial clinical impressions given AI assistance, and that this led to a significant improvement in clinical decision-making accuracy in a chest pain evaluation scenario without introducing or exacerbating existing race or gender biases. A survey of physician participants indicates that the majority expect LLM tools to play a significant role in clinical decision making.\nThe clinician-AI interface: intended use and explainability in FDA-cleared AI devices for medical image interpretation\nAuthors: McNamara, S. L.; Yi, P. H.; Lotter, W.\nScore: 3.0, Published: 2023-11-29 DOI: 10.1101/2023.11.28.23299132\nAs applications of AI in medicine continue to expand, there is an increasing focus on integration into clinical practice. An underappreciated aspect of clinical translation is where the AI fits into the clinical workflow, and in turn, the outputs generated by the AI to facilitate clinician interaction in this workflow. For instance, in the canonical use case of AI for medical image interpretation, the AI could prioritize cases before clinician review or even autonomously interpret the images without clinician review. A related aspect is explainability - does the AI generate outputs to help explain its predictions to clinicians? While many clinical AI workflows and explainability techniques have been proposed, a summative assessment of the current scope in clinical practice is lacking. Here, we evaluate the current state of FDA-cleared AI devices for medical image interpretation assistance in terms of intended clinical use, outputs generated, and types of explainability offered. We create a curated database focused on these aspects of the clinician-AI interface, where we find a high frequency of \"triage\" devices, notable variability in output characteristics across products, and often limited explainability of AI predictions. Altogether, we aim to increase transparency of the current landscape of the clinician-AI interface and highlight the need to rigorously assess which strategies ultimately lead to the best clinical outcomes.\nMulti-omics analysis of the molecular response to glucocorticoids - insights into shared genetic risk from psychiatric to medical disorders\nAuthors: Knauer-Arloth, J.; Hryhorzhevska, A.; Binder, E. B.\nScore: 1.2, Published: 2023-12-05 DOI: 10.1101/2023.12.05.23299430\nBackground: Glucocorticoids play a crucial role as mediators of negative health effects associated with chronic stress, including increased risk for psychiatric disorders as well as cardiovascular and metabolic diseases. This study investigates the impact of genetic variants and glucocorticoid receptor (GR)-activation on gene expression and DNA methylation in peripheral blood and the relationship of these variants with disease risk. Methods: We conducted a comprehensive molecular quantitative trait locus (QTL) analysis, mapping GR-methylation (me)QTLs, GR-expression (e)QTLs, and GR-expression quantitative trait methylation (eQTM) in a cohort of 199 individuals, with DNA methylation and RNA expression data collected before and after GR-activation with dexamethasone. A multi-level network analysis was employed to map the complex relationships between the transcriptome, epigenome, and genetic variation. Results: We identified 3,772 GR-meQTL CpGs corresponding to 114,134 local GR-meQTLs. eQTM and eQTL analyses revealed distinct genetic influences on RNA expression and DNA methylation. Multi-level network analysis uncovered GR-network trio QTLs, characterised by SNP-CpG-transcript combinations where meQTLs act as both eQTLs and eQTMs. These trios' genes demonstrated enrichment in immune response and cell activation pathways and showed a significant overlap with transcripts altered by GR-activation in the mouse brain. GR-trio variants were enriched in GWAS for bipolar disorder, schizophrenia, autoimmune and cardiovascular diseases and traits, cytokines levels and BMI. Conclusions: Genetic variants modulating the molecular effects of glucocorticoids are associated with psychiatric as well as medical diseases. Our findings support stress as a shared risk factor for transdiagnostic negative health outcomes and may lead to innovative interventions targeting shared underlying molecular mechanisms.\nBFCI at #SMM4H 2023: Integration of Machine Learning and TF-IDF for Covid-19 Tweets Analysis\nAuthors: Nayel, H.; Ashraf, N.; Aldawsari, M.\nScore: 1.2, Published: 2023-11-20 DOI: 10.1101/2023.11.18.23297862\nExtracting information from texts generated by users of social media platforms becomes a crucial task. In this paper, we describe the systems submitted to the SMM4H shared tasks 1 and 2. The aims of these two tasks are binary and multi-class classification of English tweets. We developed a machine learning-based model integrated with TF-IDF as a feature extraction approach. Four classification algorithms have been implemented namely, support vector machines, passive-aggressive classifier, multi-layer perceptron and random forest. For task 1, the passive-aggressive classifier reported f1-score of 63.7%. For task 2, multi-layer perceptron reported f1-score of 71.4%.\n",
  "wordCount" : "1597",
  "inLanguage": "en",
  "datePublished": "2023-12-06T10:40:04Z",
  "dateModified": "2023-12-06T10:40:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/health-informatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      health informatics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 6, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.23298847">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.23298847" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.23298847">
        <p class="paperTitle">SARS-CoV-2 Orphan Gene ORF10 Contributes to More Severe COVID-19 Disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.23298847" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.23298847" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Haltom, J. A.; Trovao, N. S.; Guarnieri, J.; Vincent, P.; Singh, U.; Tsoy, S.; O&#39;Leary, C. A.; Bram, Y.; Widjaja, G. A.; Cen, Z.; Meller, R.; Baylin, S. B.; Moss, W. N.; Nikolau, B. J.; Enguita, F. J.; Wallace, D. C.; Beheshti, A.; Schwartz, R.; Wurtele, E. S.</p>
        <p class="info">Score: 24.9, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.23298847' target='https://doi.org/10.1101/2023.11.27.23298847'> 10.1101/2023.11.27.23298847</a></p>
        <p class="abstract">The orphan gene of SARS-CoV-2, ORF10, is the least studied gene in the virus responsible for the COVID-19 pandemic. Recent experimentation indicated ORF10 expression moderates innate immunity in vitro. However, whether ORF10 affects COVID-19 in humans remained unknown. We determine that the ORF10 sequence is identical to the Wuhan-Hu-1 ancestral haplotype in 95% of genomes across five variants of concern (VOC). Four ORF10 variants are associated with less virulent clinical outcomes in the human host: three of these affect ORF10 protein structure, one affects ORF10 RNA structural dynamics. RNA-Seq data from 2070 samples from diverse human cells and tissues reveals ORF10 accumulation is conditionally discordant from that of other SARS-CoV-2 transcripts. Expression of ORF10 in A549 and HEK293 cells perturbs immune-related gene expression networks, alters expression of the majority of mitochondrially-encoded genes of oxidative respiration, and leads to large shifts in levels of 14 newly-identified transcripts. We conclude ORF10 contributes to more severe COVID-19 clinical outcomes in the human host.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.23299473">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.23299473" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.23299473">
        <p class="paperTitle">SARS-CoV-2 Infection is Associated with an Increase in New Diagnoses of Schizophrenia Spectrum and Psychotic Disorder: A Study Using the US National COVID Cohort Collaborative (N3C)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.23299473" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.23299473" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rahman, A.; Russell, M.; Zheng, W.; Eckrich, D.; Ahmed, I.; on behalf of the N3C Consortium,  </p>
        <p class="info">Score: 9.4, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.23299473' target='https://doi.org/10.1101/2023.12.05.23299473'> 10.1101/2023.12.05.23299473</a></p>
        <p class="abstract">Amid the ongoing global repercussions of SARS-CoV-2, its crucial to comprehend its potential long-term psychiatric effects. Several recent studies have suggested a link between COVID-19 and subsequent mental health disorders. Our investigation joinsthis exploration, concentrating on Schizophrenia Spectrum and Psychotic Disorders (SSPD). Different from other studies, we took acute respiratory distress syndrome (ARDS) and COVID-19 lab negative cohorts as control groups to accurately gauge the impact of COVID-19 on SSPD. Data from 19,344,698 patients, sourced from the N3C Data Enclave platform, were methodically filtered to create propensity matched cohorts: ARDS (n = 222,337), COVID-positive (n = 219,264), and COVID-negative (n = 213,183). We systematically analyzed the hazard rate of new-onset SSPD across three distinct time intervals: 0-21 days, 22-90 days, and beyond 90 days post-infection. COVID-19 positive patients consistently exhibited a heightened hazard ratio (HR) across all intervals [0-21 days (HR: 4.6; CI: 3.7-5.7), 22-90 days (HR: 2.9; CI: 2.3 -3.8), beyond 90 days (HR: 1.7; CI: 1.5-1.)]. These are notably higher than both ARDS and COVID-19 lab-negative patients. Validations using various tests, including the Cochran Mantel Haenszel Test, Wald Test, and Log-rank Test confirmed these associations. Intriguingly, our data indicated that younger individuals face a heightened risk of SSPD after contracting COVID-19, a trend not observed in the ARDS and COVID-negative groups. These results, aligned with the known neurotropism of SARS-CoV-2 and earlier studies, accentuate the need for vigilant psychiatric assessment and support in the era of Long-COVID, especially among younger populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.24.23298641">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.24.23298641" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.24.23298641">
        <p class="paperTitle">Large Language Models and Medical Knowledge Grounding for Diagnosis Prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.24.23298641" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.24.23298641" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gao, Y.; Li, R.; Croxford, E.; Tesch, S.; To, D.; Caskey, J.; Patterson, B. W.; Churpek, M. M.; Miller, T.; Dligach, D.; Afshar, M.</p>
        <p class="info">Score: 6.0, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.24.23298641' target='https://doi.org/10.1101/2023.11.24.23298641'> 10.1101/2023.11.24.23298641</a></p>
        <p class="abstract">While Large Language Models (LLMs) have showcased their potential in diverse language tasks, their application in the healthcare arena needs to ensure the minimization of diagnostic errors and the prevention of patient harm. A Medical Knowledge Graph (KG) houses a wealth of structured medical concept relations sourced from authoritative references, such as UMLS, making it a valuable resource to ground LLMs diagnostic process in knowledge. In this paper, we examine the synergistic potential of LLMs and medical KG in predicting diagnoses given electronic health records (EHR), under the framework of Retrieval-augmented generation (RAG). We proposed a novel graph model: DO_SCPLOWRC_SCPLOW.KO_SCPLOWNOWSC_SCPLOW, that selects the most relevant pathology knowledge paths based on the medical problem descriptions. In order to evaluate DO_SCPLOWRC_SCPLOW.KO_SCPLOWNOWSC_SCPLOW, we developed the first comprehensive human evaluation approach to assess the performance of LLMs for diagnosis prediction and examine the rationale behind their decision-making processes, aimed at improving diagnostic safety. Using real-world hospital datasets, our study serves to enrich the discourse on the role of medical KGs in grounding medical knowledge into LLMs, revealing both challenges and opportunities in harnessing external knowledge for explainable diagnostic pathway and the realization of AI-augmented diagnostic decision support systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.24.23298844">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.24.23298844" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.24.23298844">
        <p class="paperTitle">ChatGPT Influence on Medical Decision-Making, Bias, and Equity: A Randomized Study of Clinicians Evaluating Clinical Vignettes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.24.23298844" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.24.23298844" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Goh, E.; Bunning, B.; Khoong, E.; Gallo, R.; Milstein, A.; Centola, D.; Chen, J. H.</p>
        <p class="info">Score: 7.2, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.24.23298844' target='https://doi.org/10.1101/2023.11.24.23298844'> 10.1101/2023.11.24.23298844</a></p>
        <p class="abstract">In a randomized, pre-post intervention study, we evaluated the influence of a large language model (LLM) generative AI system on accuracy of physician decision-making and bias in healthcare. 50 US-licensed physicians reviewed a video clinical vignette, featuring actors representing different demographics (a White male or a Black female) with chest pain. Participants were asked to answer clinical questions around triage, risk, and treatment based on these vignettes, then asked to reconsider after receiving advice generated by ChatGPT&#43; (GPT4). The primary outcome was the accuracy of clinical decisions based on pre-established evidence-based guidelines. Results showed that physicians are willing to change their initial clinical impressions given AI assistance, and that this led to a significant improvement in clinical decision-making accuracy in a chest pain evaluation scenario without introducing or exacerbating existing race or gender biases. A survey of physician participants indicates that the majority expect LLM tools to play a significant role in clinical decision making.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.23299132">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.23299132" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.23299132">
        <p class="paperTitle">The clinician-AI interface: intended use and explainability in FDA-cleared AI devices for medical image interpretation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.23299132" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.23299132" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McNamara, S. L.; Yi, P. H.; Lotter, W.</p>
        <p class="info">Score: 3.0, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.23299132' target='https://doi.org/10.1101/2023.11.28.23299132'> 10.1101/2023.11.28.23299132</a></p>
        <p class="abstract">As applications of AI in medicine continue to expand, there is an increasing focus on integration into clinical practice. An underappreciated aspect of clinical translation is where the AI fits into the clinical workflow, and in turn, the outputs generated by the AI to facilitate clinician interaction in this workflow. For instance, in the canonical use case of AI for medical image interpretation, the AI could prioritize cases before clinician review or even autonomously interpret the images without clinician review. A related aspect is explainability - does the AI generate outputs to help explain its predictions to clinicians? While many clinical AI workflows and explainability techniques have been proposed, a summative assessment of the current scope in clinical practice is lacking. Here, we evaluate the current state of FDA-cleared AI devices for medical image interpretation assistance in terms of intended clinical use, outputs generated, and types of explainability offered. We create a curated database focused on these aspects of the clinician-AI interface, where we find a high frequency of &#34;triage&#34; devices, notable variability in output characteristics across products, and often limited explainability of AI predictions. Altogether, we aim to increase transparency of the current landscape of the clinician-AI interface and highlight the need to rigorously assess which strategies ultimately lead to the best clinical outcomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.23299430">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.23299430" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.23299430">
        <p class="paperTitle">Multi-omics analysis of the molecular response to glucocorticoids - insights into shared genetic risk from psychiatric to medical disorders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.23299430" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.23299430" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Knauer-Arloth, J.; Hryhorzhevska, A.; Binder, E. B.</p>
        <p class="info">Score: 1.2, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.23299430' target='https://doi.org/10.1101/2023.12.05.23299430'> 10.1101/2023.12.05.23299430</a></p>
        <p class="abstract">Background: Glucocorticoids play a crucial role as mediators of negative health effects associated with chronic stress, including increased risk for psychiatric disorders as well as cardiovascular and metabolic diseases. This study investigates the impact of genetic variants and glucocorticoid receptor (GR)-activation on gene expression and DNA methylation in peripheral blood and the relationship of these variants with disease risk. Methods: We conducted a comprehensive molecular quantitative trait locus (QTL) analysis, mapping GR-methylation (me)QTLs, GR-expression (e)QTLs, and GR-expression quantitative trait methylation (eQTM) in a cohort of 199 individuals, with DNA methylation and RNA expression data collected before and after GR-activation with dexamethasone. A multi-level network analysis was employed to map the complex relationships between the transcriptome, epigenome, and genetic variation. Results: We identified 3,772 GR-meQTL CpGs corresponding to 114,134 local GR-meQTLs. eQTM and eQTL analyses revealed distinct genetic influences on RNA expression and DNA methylation. Multi-level network analysis uncovered GR-network trio QTLs, characterised by SNP-CpG-transcript combinations where meQTLs act as both eQTLs and eQTMs. These trios&#39; genes demonstrated enrichment in immune response and cell activation pathways and showed a significant overlap with transcripts altered by GR-activation in the mouse brain. GR-trio variants were enriched in GWAS for bipolar disorder, schizophrenia, autoimmune and cardiovascular diseases and traits, cytokines levels and BMI. Conclusions: Genetic variants modulating the molecular effects of glucocorticoids are associated with psychiatric as well as medical diseases. Our findings support stress as a shared risk factor for transdiagnostic negative health outcomes and may lead to innovative interventions targeting shared underlying molecular mechanisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.18.23297862">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.18.23297862" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.18.23297862">
        <p class="paperTitle">BFCI at #SMM4H 2023: Integration of Machine Learning and TF-IDF for Covid-19 Tweets Analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.18.23297862" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.18.23297862" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nayel, H.; Ashraf, N.; Aldawsari, M.</p>
        <p class="info">Score: 1.2, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.18.23297862' target='https://doi.org/10.1101/2023.11.18.23297862'> 10.1101/2023.11.18.23297862</a></p>
        <p class="abstract">Extracting information from texts generated by users of social media platforms becomes a crucial task. In this paper, we describe the systems submitted to the SMM4H shared tasks 1 and 2. The aims of these two tasks are binary and multi-class classification of English tweets. We developed a machine learning-based model integrated with TF-IDF as a feature extraction approach. Four classification algorithms have been implemented namely, support vector machines, passive-aggressive classifier, multi-layer perceptron and random forest. For task 1, the passive-aggressive classifier reported f1-score of 63.7%. For task 2, multi-layer perceptron reported f1-score of 71.4%.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
