<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>health informatics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>health informatics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Automated Classification of At-home SARS-CoV-2 Lateral Flow Assay Test Results using Image Matching and Transfer Learning: multiple-pipeline study
Authors: Safarzadeh, M.; Herbert, C.; Wong, S. K.; Stamegna, P.; Guilarte-Walker, Y.; Wright, C.; Suvarna, T.; Nowak, C.; Kheterpal, V.; Pandey, S.; Wang, B.; Lin, H.; O&#39;Connor, L.; Hafer, N.; Luzuriaga, K.; Manabe, Y.; Broach, J.; Zai, A. H.; McManus, D. D.; Du, X.; Soni, A.
Score: 1.5, Published: 2024-01-04 DOI: 10.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/health-informatics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="health informatics" />
<meta property="og:description" content="Automated Classification of At-home SARS-CoV-2 Lateral Flow Assay Test Results using Image Matching and Transfer Learning: multiple-pipeline study
Authors: Safarzadeh, M.; Herbert, C.; Wong, S. K.; Stamegna, P.; Guilarte-Walker, Y.; Wright, C.; Suvarna, T.; Nowak, C.; Kheterpal, V.; Pandey, S.; Wang, B.; Lin, H.; O&#39;Connor, L.; Hafer, N.; Luzuriaga, K.; Manabe, Y.; Broach, J.; Zai, A. H.; McManus, D. D.; Du, X.; Soni, A.
Score: 1.5, Published: 2024-01-04 DOI: 10." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/health-informatics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-10T10:40:05+00:00" />
<meta property="article:modified_time" content="2024-01-10T10:40:05+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="health informatics"/>
<meta name="twitter:description" content="Automated Classification of At-home SARS-CoV-2 Lateral Flow Assay Test Results using Image Matching and Transfer Learning: multiple-pipeline study
Authors: Safarzadeh, M.; Herbert, C.; Wong, S. K.; Stamegna, P.; Guilarte-Walker, Y.; Wright, C.; Suvarna, T.; Nowak, C.; Kheterpal, V.; Pandey, S.; Wang, B.; Lin, H.; O&#39;Connor, L.; Hafer, N.; Luzuriaga, K.; Manabe, Y.; Broach, J.; Zai, A. H.; McManus, D. D.; Du, X.; Soni, A.
Score: 1.5, Published: 2024-01-04 DOI: 10."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "health informatics",
      "item": "https://trxiv.yorks0n.com/posts/health-informatics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "health informatics",
  "name": "health informatics",
  "description": "Automated Classification of At-home SARS-CoV-2 Lateral Flow Assay Test Results using Image Matching and Transfer Learning: multiple-pipeline study\nAuthors: Safarzadeh, M.; Herbert, C.; Wong, S. K.; Stamegna, P.; Guilarte-Walker, Y.; Wright, C.; Suvarna, T.; Nowak, C.; Kheterpal, V.; Pandey, S.; Wang, B.; Lin, H.; O\u0026#39;Connor, L.; Hafer, N.; Luzuriaga, K.; Manabe, Y.; Broach, J.; Zai, A. H.; McManus, D. D.; Du, X.; Soni, A.\nScore: 1.5, Published: 2024-01-04 DOI: 10.",
  "keywords": [
    
  ],
  "articleBody": " Automated Classification of At-home SARS-CoV-2 Lateral Flow Assay Test Results using Image Matching and Transfer Learning: multiple-pipeline study\nAuthors: Safarzadeh, M.; Herbert, C.; Wong, S. K.; Stamegna, P.; Guilarte-Walker, Y.; Wright, C.; Suvarna, T.; Nowak, C.; Kheterpal, V.; Pandey, S.; Wang, B.; Lin, H.; O'Connor, L.; Hafer, N.; Luzuriaga, K.; Manabe, Y.; Broach, J.; Zai, A. H.; McManus, D. D.; Du, X.; Soni, A.\nScore: 1.5, Published: 2024-01-04 DOI: 10.1101/2024.01.04.24300836\nIntroductionRapid antigen testing for SARS-CoV-2 is an important tool for the timely diagnosis of COVID-19, especially in at-home settings. However, the interpretation of test results can be subjective and prone to error. We describe an automated image analysis pipeline to accurately classify test types and results without human intervention using a dataset of 51,274 rapid antigen test images across three distinct test card brands. MethodsThe proposed method classifies participant-submitted images into four categories: positive for SARS-CoV-2, negative for SARS-CoV-2, invalid/uncertain, and unclassifiable. The model includes four stages: test card classification and region of interest detection using image-matching algorithms, elimination of invalid results using a developed Siamese neural network, and test result classification using transfer learning. ResultsThe model accuracy was very good for test-card classification (100%), region of interest detection (83.5%), and identification of invalid results ranging from 95.6% to 100% for different test types. Performance of the model for test result classification varied by tests; the models sensitivity, specificity, and precision for Abbott BinaxNOW was 0.761, 0.989, and 0.946, BD Veritor At-Home COVID-19 Test was 0.955, 0.993, and 0.877, and for QuickVue(R) At-Home OTC COVID-19 Test was 0.816, 0.988, and 0.930. ConclusionThe proposed method improved the interpretation of rapid antigen tests, particularly in invalid result detection compared to human-read, and offers a great opportunity for standardization of rapid antigen test interpretation and for providing feedback to participants with invalid tests.\nRobodoc: a conversational-AI based app for medical conversations\nAuthors: Pires, J. G.\nScore: 1.8, Published: 2024-01-02 DOI: 10.1101/2023.12.31.23300681\nArtificial Intelligence (AI) evolved in trends. Currently, the trend is Conversational Artificial Intelligence (CAI). Those models of AI are focused on text-related tasks, and their commonest applications are chatbots. On this paper, we explore a smart chatbot using the Large Language Models (LLMs) from openAI. I have used a tool called Teachable Machine (TM) from Google to apply transfer learning and create image-based models. I have built two image-based model: for X-ray and for OCT. The model of X-ray is able to detect viral and bacterial pneumonia, whereas the Optical coherence tomography (OCT) model can detect Drusen, Choroidal Neovascularization (CNV) and (Diabetic Macular Edema DME) conditions on the patients eyes image. I have also used TensorFlow.js from Google to create a diabetes detection model. All those models are integrated into a chatbot, that according to the message entered by a user, is able to use the models intelligently. Our results show a good integration between the models and the chatbot, with slight deviations from the expected behaviors. For the OCT model, we have also tested a stub function for medical appointments done by the bot, based on how serious is the patient condition. The future of artificial intelligence are public APIs, as I have shown that a complex model can be built, without a complex research infrastructure, and with low costs. Bioinformatics may have gained a new supporter towards more friendly interfaces on bioinformatics.\nApplying Multimodal Data Fusion based on Deep Learning Methods for the Diagnosis of Neglected Tropical Diseases: A Systematic Review and Meta-Analysis\nAuthors: Minyilu, Y.; Yimer, M. A.; Meshesha, M.\nScore: 1.4, Published: 2024-01-09 DOI: 10.1101/2024.01.07.24300957\nNeglected tropical diseases (NTDs) are the most prevalent diseases worldwide affecting one tenth of the world population. Although there are multiple approaches to diagnose these diseases, using skin manifestations and lesions caused as a result of these diseases along with other medical records is the preferred method. This fact triggers the need to explore and implement a deep learning -based diagnostic model using multimodal data fusion (MMDF) techniques to enhance the diagnostic process. This paper, thus, endeavored to present a thorough systematic review of studies regarding the implementation of MMDF techniques for the diagnoses of skin related NTDs. To achieve its objective, the study used the PRISMA method based on predefined questions and collected 427 articles from seven major and reputed sources and critically appraised each article. Although there are studies that demonstrate the utilization of deep learning methods for the diagnoses of NTDs, no previous studies were found regarding the implementation of MMDF for the diagnoses of skin related NTDs. Similar studies using MMDF for the diagnoses of other skin diseases, such as skin cancer, were collected and analyzed in this review to extract information about the implementation of these methods. In doing so, various studies are analyzed using six different parameters including research approaches, disease selected for diagnosis, dataset, algorithms, performance achievements and future directions. Accordingly, although all the studies analyzed used diverse research methods and datasets based on their problem, deep learning -based convolutional neural network algorithms are the most frequently used and best performing model in all studies reviewed.\nEvaluating and Improving the Performance and Racial Fairness of Algorithms for GFR Estimation\nAuthors: Zhang, L.; Richter, L. R.; Kim, T.; Hripcsak, G.\nScore: 1.0, Published: 2024-01-09 DOI: 10.1101/2024.01.07.24300943\nData-driven clinical prediction algorithms are used widely by clinicians. Understanding what factors can impact the performance and fairness of data-driven algorithms is an important step towards achieving equitable healthcare. To investigate the impact of modeling choices on the algorithmic performance and fairness, we make use of a case study to build a prediction algorithm for estimating glomerular filtration rate (GFR) based on the electronic health record (EHR). We compare three distinct approaches for estimating GFR: CKD-EPI equations, epidemiological models, and EHR-based models. For epidemiological models and EHR-based models, four machine learning models of varying computational complexity (i.e., linear regression, support vector machine, random forest regression, and neural network) were compared. Performance metrics included root mean squared error (RMSE), median difference, and the proportion of GFR estimates within 30% of the measured GFR value (P30). Differential performance between non-African American and African American group was used to assess algorithmic fairness with respect to race. Our study showed that the variable race had a negligible effect on error, accuracy, and differential performance. Furthermore, including more relevant clinical features (e.g., common comorbidities of chronic kidney disease) and using more complex machine learning models, namely random forest regression, significantly lowered the estimation error of GFR. However, the difference in performance between African American and non-African American patients did not decrease, where the estimation error for African American patients remained consistently higher than non-African American patients, indicating that more objective patient characteristics should be discovered and included to improve algorithm performance.\nHuman versus Artificial Intelligence: ChatGPT-4 Outperforming Bing, Bard, ChatGPT-3.5, and Humans in Clinical Chemistry Multiple-Choice Questions\nAuthors: Sallam, M.; Al-Salahat, K.; Eid, H.; Egger, J.; Puladi, B.\nScore: 1.0, Published: 2024-01-09 DOI: 10.1101/2024.01.08.24300995\nThe advances in large language models (LLMs) are evolving rapidly. Artificial intelligence (AI) chatbots based on LLMs excel in language understanding and generation, with potential utility to transform healthcare education and practice. However, it is important to assess the performance of such AI models in various topics to highlight its strengths and possible limitations. Therefore, this study aimed to evaluate the performance of ChatGPT (GPT-3.5 and GPT-4), Bing, and Bard compared to human students at a postgraduate masters (MSc) level in Medical Laboratory Sciences. The study design was based on the METRICS checklist for the design and reporting of AI-based studies in healthcare. The study utilized a dataset of 60 Clinical Chemistry multiple-choice questions (MCQs) initially conceived for assessment of 20 MSc students. The revised Blooms taxonomy was used as the framework for classifying the MCQs into four cognitive categories: Remember, Understand, Analyze, and Apply. A modified version of the CLEAR tool was used for assessment of the quality of AI-generated content, with Cohens {kappa} for inter-rater agreement. Compared to the mean students score which was 40/60 (66.8%), GPT-4 scored 54/60 (90.0%), followed by Bing (46/60, 76.7%), GPT-3.5 (44/60, 73.3%), and Bard (40/60, 66.7%). Statistically significant better performance was noted in lower cognitive domains (Remember and Understand) in GPT-3.5, GPT-4, and Bard. The CLEAR scores indicated that ChatGPT-4 performance was \"Excellent\" compared to \"Above average\" performance of ChatGPT-3.5, Bing, and Bard. The findings indicated that ChatGPT-4 excelled in the Clinical Chemistry exam, while ChatGPT-3.5, Bing, and Bard were above-average. Given that the MCQs were directed to postgraduate students with a high degree of specialization, the performance of these AI chatbots was remarkable. Due to the risks of academic dishonesty and possible dependence on these AI models, the appropriateness of MCQs as an assessment tool in higher education should be re-evaluated.\nCharacterising and differentiating cognitive and motor speed in older adults:a longitudinal birth cohort study\nAuthors: Bundil, I.; Baltruschat, S.; Zhang, J.\nScore: 1.0, Published: 2024-01-05 DOI: 10.1101/2024.01.04.24300822\nObjectivesInformation processing speed (IPS) has been proposed to be a key component in healthy ageing and cognitive functioning. Yet, current studies lack a consistent definition and specific influential characteristics. This study aimed at investigating IPS as a multifaceted concept by differentiating cognitive and motor IPS. Design, setting and participantsA retrospective data analysis using data from the Medical Research Council National Survey of Health and Development (MRC NSHD; a population-based cohort of UK adults born in 1946) at childhood (ages 8, 11, and 15) and adulthood (ages 60-64 and 68-70). Using structural equation modelling, we constructed two models of IPS with 2124 and 1776 participants, respectively. Outcome measuresMeasures of interest included IPS (i.e., letter cancellation, simple and choice reaction time), intelligence (i.e., childhood intelligence and NART), verbal memory, socio-economic status (SES) and cognitive functions measured by the Addenbrookes Cognitive Examination III, as well as a variety of health indexes. ResultsWe found distinct predictors for cognitive and motor IPS and how they relate to other cognitive functions in old age. In our first model, SES and anti-psychotic medication usage emerged as significant predictors for cognitive IPS, intelligence and smoking as predictors for motor IPS, while both share sex, memory, and anti-epileptic medication usage as common predictors. Notably, all differences between both IPS types ran in the same direction except for sex differences, with women performing better than men in cognitive IPS and vice versa in motor IPS. The second model showed that both IPS measures, as well as intelligence, memory, anti-psychotic and sedative medication usage explain cognitive functions later in life. ConclusionTaken together, these results shed further light on IPS as a whole by showing there are distinct types and that these measures directly relate to other cognitive functions. STRENGTH \u0026 LIMITATIONSO_LIA large longitudinal cohort data set with different measurements of information processing speed that are widely used C_LIO_LIInformation processing speed is not only related to variables measured at the same time but also to childhood and premorbid intelligence and cognitive functions in later life C_LIO_LILimitations of the cohort dataset include different response rates between waves, thus some variables were not available for all individuals at certain time points, and IPS scores were derived from a small number of trials C_LIO_LIThe study involved self-reported measures which might have increased the proportion of misclassification C_LI\nA web tool for the exploration of hospital prescription data in England\nAuthors: Sanderson, T.\nScore: 16.8, Published: 2023-12-15 DOI: 10.1101/2023.12.14.23299817\nMedicine prescription data is a valuable resource, with the power to reveal insights into healthcare trends, cost efficiencies, and geographic disparities. In England, primary care prescription data has been openly accessible for analysis for some time through a web tool, providing significant benefits. Since 2020, the National Health Service in England has also released data on secondary care prescriptions, i.e. prescribing within hospitals. This is an important dataset, but until now has been available only in a raw form that requires considerable technical skills to be used for even the analysis of basic trends. I have built a web application that enables anyone to easily analyse trends in this data, which is available at hospitalprescriptions.genomium.org.\nExploratory electronic health record analysis with ehrapy\nAuthors: Heumos, L.; Ehmele, P.; Treis, T.; Upmeier zu Belzen, J.; Namsaraeva, A.; Horlava, N.; Shitov, V. A.; Zhang, X.; Zappia, L.; Knoll, R.; Lang, N. J.; Hetzel, L.; Virshup, I.; Sikkema, L.; Roellin, E.; Curion, F.; Eils, R.; Schiller, H. B.; Hilgendorff, A.; Theis, F.\nScore: 54.2, Published: 2023-12-15 DOI: 10.1101/2023.12.11.23299816\nWith progressive digitalization of healthcare systems worldwide, large-scale collection of electronic health records (EHRs) has become commonplace. However, an extensible framework for comprehensive exploratory analysis that accounts for data heterogeneity is missing. Here, we introduce ehrapy, a modular open-source Python framework designed for exploratory end-to-end analysis of heterogeneous epidemiology and electronic health record data. Ehrapy incorporates a series of analytical steps, from data extraction and quality control to the generation of low-dimensional representations. Complemented by rich statistical modules, ehrapy facilitates associating patients with disease states, differential comparison between patient clusters, survival analysis, trajectory inference, causal inference, and more. Leveraging ontologies, ehrapy further enables data sharing and training EHR deep learning models paving the way for foundational models in biomedical research. We demonstrated ehrapys features in five distinct examples: We first applied ehrapy to stratify patients affected by unspecified pneumonia into finer-grained phenotypes. Furthermore, we revealed biomarkers for significant differences in survival among these groups. Additionally, we quantify medication-class effects of pneumonia medications on length of stay. We further leveraged ehrapy to analyze cardiovascular risks across different data modalities. Finally, we reconstructed disease state trajectories in SARS-CoV-2 patients based on imaging data. Ehrapy thus provides a framework that we envision will standardize analysis pipelines on EHR data and serve as a cornerstone for the community.\nTesting federated analytics across secure data environments using differing statistical approaches on cross-disciplinary data.\nAuthors: Gallier, S.; Topham, A.; Hodson, J.; McNulty, D.; Giles, T.; Cox, S.; Chaganty, J.; Cooper, L.; Perks, S.; Quinlan, P.; Sapey, E.\nScore: 0.8, Published: 2024-01-07 DOI: 10.1101/2024.01.06.23300659\nBACKGROUNDIntroducing data-driven technologies into health systems can enhance population health and streamline care delivery. The use of diverse and geographically varied data is key for tackling health and societal challenges, despite associated technical, ethical, and governance complexities. This study explored the efficacy of federated analytics using general linear models (GLMs) and machine learning (ML) models, comparing outcomes with non-federated data analysis. METHODSA Conditional Transformation Generative Adversarial Network was used to create two synthetic datasets (training set: N=10,000; test set: N=1,000), using real-world data from 381 asthma patients. To simulate a federated environment, the resulting data were distributed across nodes in a Microsoft Azure Trusted Research Environment (TRE). GLMs (one-way ANOVA) and ML models (gradient boosted decision trees) where then produced, using both federated and non-federated approaches. The consistency of predictions produced by the ML models were then compared between approaches, with predictive accuracy of the models quantified by the area under the receiver operating characteristic curve (AUROC). FINDINGSGLMs produced from federated data distributed between two TREs were identical to those produced using a non-federated approach. However, ML models produced by federated and non-federated approaches, and using different data distributions between TREs, were non-identical. Despite this, when applied to the test set, the classifications made by the federated models were consistent with the non-federated model in 84.7-90.4% of cases, which was similar to the consistency of repeated non-federated models (90.9-91.5%). Consequently, overall predictive accuracies for federated and non-federated models were similar (AUROC: 0.663-0.669). INTERPRETATIONThis study confirmed the robustness of GLMs utilising ANOVA within a federated framework, yielding consistent outcomes. Moreover, federated ML models demonstrated a high degree of classification agreement, with comparable accuracy to traditional non-federated models. These results highlight the viability of federated approaches for reliable and accurate data analysis in sensitive domains.\n",
  "wordCount" : "2602",
  "inLanguage": "en",
  "datePublished": "2024-01-10T10:40:05Z",
  "dateModified": "2024-01-10T10:40:05Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/health-informatics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      health informatics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 10, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.24300836">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.24300836" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.24300836">
        <p class="paperTitle">Automated Classification of At-home SARS-CoV-2 Lateral Flow Assay Test Results using Image Matching and Transfer Learning: multiple-pipeline study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.24300836" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.24300836" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Safarzadeh, M.; Herbert, C.; Wong, S. K.; Stamegna, P.; Guilarte-Walker, Y.; Wright, C.; Suvarna, T.; Nowak, C.; Kheterpal, V.; Pandey, S.; Wang, B.; Lin, H.; O&#39;Connor, L.; Hafer, N.; Luzuriaga, K.; Manabe, Y.; Broach, J.; Zai, A. H.; McManus, D. D.; Du, X.; Soni, A.</p>
        <p class="info">Score: 1.5, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.24300836' target='https://doi.org/10.1101/2024.01.04.24300836'> 10.1101/2024.01.04.24300836</a></p>
        <p class="abstract">IntroductionRapid antigen testing for SARS-CoV-2 is an important tool for the timely diagnosis of COVID-19, especially in at-home settings. However, the interpretation of test results can be subjective and prone to error. We describe an automated image analysis pipeline to accurately classify test types and results without human intervention using a dataset of 51,274 rapid antigen test images across three distinct test card brands.

MethodsThe proposed method classifies participant-submitted images into four categories: positive for SARS-CoV-2, negative for SARS-CoV-2, invalid/uncertain, and unclassifiable. The model includes four stages: test card classification and region of interest detection using image-matching algorithms, elimination of invalid results using a developed Siamese neural network, and test result classification using transfer learning.

ResultsThe model accuracy was very good for test-card classification (100%), region of interest detection (83.5%), and identification of invalid results ranging from 95.6% to 100% for different test types. Performance of the model for test result classification varied by tests; the models sensitivity, specificity, and precision for Abbott BinaxNOW was 0.761, 0.989, and 0.946, BD Veritor At-Home COVID-19 Test was 0.955, 0.993, and 0.877, and for QuickVue(R) At-Home OTC COVID-19 Test was 0.816, 0.988, and 0.930.

ConclusionThe proposed method improved the interpretation of rapid antigen tests, particularly in invalid result detection compared to human-read, and offers a great opportunity for standardization of rapid antigen test interpretation and for providing feedback to participants with invalid tests.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.31.23300681">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.31.23300681" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.31.23300681">
        <p class="paperTitle">Robodoc: a conversational-AI based app for medical conversations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.31.23300681" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.31.23300681" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pires, J. G.</p>
        <p class="info">Score: 1.8, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.31.23300681' target='https://doi.org/10.1101/2023.12.31.23300681'> 10.1101/2023.12.31.23300681</a></p>
        <p class="abstract">Artificial Intelligence (AI) evolved in trends. Currently, the trend is Conversational Artificial Intelligence (CAI). Those models of AI are focused on text-related tasks, and their commonest applications are chatbots. On this paper, we explore a smart chatbot using the Large Language Models (LLMs) from openAI. I have used a tool called Teachable Machine (TM) from Google to apply transfer learning and create image-based models. I have built two image-based model: for X-ray and for OCT. The model of X-ray is able to detect viral and bacterial pneumonia, whereas the Optical coherence tomography (OCT) model can detect Drusen, Choroidal Neovascularization (CNV) and (Diabetic Macular Edema DME) conditions on the patients eyes image. I have also used TensorFlow.js from Google to create a diabetes detection model. All those models are integrated into a chatbot, that according to the message entered by a user, is able to use the models intelligently. Our results show a good integration between the models and the chatbot, with slight deviations from the expected behaviors. For the OCT model, we have also tested a stub function for medical appointments done by the bot, based on how serious is the patient condition. The future of artificial intelligence are public APIs, as I have shown that a complex model can be built, without a complex research infrastructure, and with low costs. Bioinformatics may have gained a new supporter towards more friendly interfaces on bioinformatics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.07.24300957">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.07.24300957" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.07.24300957">
        <p class="paperTitle">Applying Multimodal Data Fusion based on Deep Learning Methods for the Diagnosis of Neglected Tropical Diseases: A Systematic Review and Meta-Analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.07.24300957" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.07.24300957" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Minyilu, Y.; Yimer, M. A.; Meshesha, M.</p>
        <p class="info">Score: 1.4, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.07.24300957' target='https://doi.org/10.1101/2024.01.07.24300957'> 10.1101/2024.01.07.24300957</a></p>
        <p class="abstract">Neglected tropical diseases (NTDs) are the most prevalent diseases worldwide affecting one tenth of the world population. Although there are multiple approaches to diagnose these diseases, using skin manifestations and lesions caused as a result of these diseases along with other medical records is the preferred method. This fact triggers the need to explore and implement a deep learning -based diagnostic model using multimodal data fusion (MMDF) techniques to enhance the diagnostic process. This paper, thus, endeavored to present a thorough systematic review of studies regarding the implementation of MMDF techniques for the diagnoses of skin related NTDs. To achieve its objective, the study used the PRISMA method based on predefined questions and collected 427 articles from seven major and reputed sources and critically appraised each article. Although there are studies that demonstrate the utilization of deep learning methods for the diagnoses of NTDs, no previous studies were found regarding the implementation of MMDF for the diagnoses of skin related NTDs. Similar studies using MMDF for the diagnoses of other skin diseases, such as skin cancer, were collected and analyzed in this review to extract information about the implementation of these methods. In doing so, various studies are analyzed using six different parameters including research approaches, disease selected for diagnosis, dataset, algorithms, performance achievements and future directions. Accordingly, although all the studies analyzed used diverse research methods and datasets based on their problem, deep learning -based convolutional neural network algorithms are the most frequently used and best performing model in all studies reviewed.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.07.24300943">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.07.24300943" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.07.24300943">
        <p class="paperTitle">Evaluating and Improving the Performance and Racial Fairness of Algorithms for GFR Estimation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.07.24300943" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.07.24300943" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, L.; Richter, L. R.; Kim, T.; Hripcsak, G.</p>
        <p class="info">Score: 1.0, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.07.24300943' target='https://doi.org/10.1101/2024.01.07.24300943'> 10.1101/2024.01.07.24300943</a></p>
        <p class="abstract">Data-driven clinical prediction algorithms are used widely by clinicians. Understanding what factors can impact the performance and fairness of data-driven algorithms is an important step towards achieving equitable healthcare. To investigate the impact of modeling choices on the algorithmic performance and fairness, we make use of a case study to build a prediction algorithm for estimating glomerular filtration rate (GFR) based on the electronic health record (EHR). We compare three distinct approaches for estimating GFR: CKD-EPI equations, epidemiological models, and EHR-based models. For epidemiological models and EHR-based models, four machine learning models of varying computational complexity (i.e., linear regression, support vector machine, random forest regression, and neural network) were compared. Performance metrics included root mean squared error (RMSE), median difference, and the proportion of GFR estimates within 30% of the measured GFR value (P30). Differential performance between non-African American and African American group was used to assess algorithmic fairness with respect to race. Our study showed that the variable race had a negligible effect on error, accuracy, and differential performance. Furthermore, including more relevant clinical features (e.g., common comorbidities of chronic kidney disease) and using more complex machine learning models, namely random forest regression, significantly lowered the estimation error of GFR. However, the difference in performance between African American and non-African American patients did not decrease, where the estimation error for African American patients remained consistently higher than non-African American patients, indicating that more objective patient characteristics should be discovered and included to improve algorithm performance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.24300995">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.24300995" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.24300995">
        <p class="paperTitle">Human versus Artificial Intelligence: ChatGPT-4 Outperforming Bing, Bard, ChatGPT-3.5, and Humans in Clinical Chemistry Multiple-Choice Questions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.24300995" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.24300995" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sallam, M.; Al-Salahat, K.; Eid, H.; Egger, J.; Puladi, B.</p>
        <p class="info">Score: 1.0, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.24300995' target='https://doi.org/10.1101/2024.01.08.24300995'> 10.1101/2024.01.08.24300995</a></p>
        <p class="abstract">The advances in large language models (LLMs) are evolving rapidly. Artificial intelligence (AI) chatbots based on LLMs excel in language understanding and generation, with potential utility to transform healthcare education and practice. However, it is important to assess the performance of such AI models in various topics to highlight its strengths and possible limitations. Therefore, this study aimed to evaluate the performance of ChatGPT (GPT-3.5 and GPT-4), Bing, and Bard compared to human students at a postgraduate masters (MSc) level in Medical Laboratory Sciences. The study design was based on the METRICS checklist for the design and reporting of AI-based studies in healthcare. The study utilized a dataset of 60 Clinical Chemistry multiple-choice questions (MCQs) initially conceived for assessment of 20 MSc students. The revised Blooms taxonomy was used as the framework for classifying the MCQs into four cognitive categories: Remember, Understand, Analyze, and Apply. A modified version of the CLEAR tool was used for assessment of the quality of AI-generated content, with Cohens {kappa} for inter-rater agreement. Compared to the mean students score which was 40/60 (66.8%), GPT-4 scored 54/60 (90.0%), followed by Bing (46/60, 76.7%), GPT-3.5 (44/60, 73.3%), and Bard (40/60, 66.7%). Statistically significant better performance was noted in lower cognitive domains (Remember and Understand) in GPT-3.5, GPT-4, and Bard. The CLEAR scores indicated that ChatGPT-4 performance was &#34;Excellent&#34; compared to &#34;Above average&#34; performance of ChatGPT-3.5, Bing, and Bard. The findings indicated that ChatGPT-4 excelled in the Clinical Chemistry exam, while ChatGPT-3.5, Bing, and Bard were above-average. Given that the MCQs were directed to postgraduate students with a high degree of specialization, the performance of these AI chatbots was remarkable. Due to the risks of academic dishonesty and possible dependence on these AI models, the appropriateness of MCQs as an assessment tool in higher education should be re-evaluated.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.24300822">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.24300822" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.24300822">
        <p class="paperTitle">Characterising and differentiating cognitive and motor speed in older adults:a longitudinal birth cohort study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.24300822" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.24300822" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bundil, I.; Baltruschat, S.; Zhang, J.</p>
        <p class="info">Score: 1.0, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.24300822' target='https://doi.org/10.1101/2024.01.04.24300822'> 10.1101/2024.01.04.24300822</a></p>
        <p class="abstract">ObjectivesInformation processing speed (IPS) has been proposed to be a key component in healthy ageing and cognitive functioning. Yet, current studies lack a consistent definition and specific influential characteristics. This study aimed at investigating IPS as a multifaceted concept by differentiating cognitive and motor IPS.

Design, setting and participantsA retrospective data analysis using data from the Medical Research Council National Survey of Health and Development (MRC NSHD; a population-based cohort of UK adults born in 1946) at childhood (ages 8, 11, and 15) and adulthood (ages 60-64 and 68-70). Using structural equation modelling, we constructed two models of IPS with 2124 and 1776 participants, respectively.

Outcome measuresMeasures of interest included IPS (i.e., letter cancellation, simple and choice reaction time), intelligence (i.e., childhood intelligence and NART), verbal memory, socio-economic status (SES) and cognitive functions measured by the Addenbrookes Cognitive Examination III, as well as a variety of health indexes.

ResultsWe found distinct predictors for cognitive and motor IPS and how they relate to other cognitive functions in old age. In our first model, SES and anti-psychotic medication usage emerged as significant predictors for cognitive IPS, intelligence and smoking as predictors for motor IPS, while both share sex, memory, and anti-epileptic medication usage as common predictors. Notably, all differences between both IPS types ran in the same direction except for sex differences, with women performing better than men in cognitive IPS and vice versa in motor IPS. The second model showed that both IPS measures, as well as intelligence, memory, anti-psychotic and sedative medication usage explain cognitive functions later in life.

ConclusionTaken together, these results shed further light on IPS as a whole by showing there are distinct types and that these measures directly relate to other cognitive functions.

STRENGTH &amp; LIMITATIONSO_LIA large longitudinal cohort data set with different measurements of information processing speed that are widely used
C_LIO_LIInformation processing speed is not only related to variables measured at the same time but also to childhood and premorbid intelligence and cognitive functions in later life
C_LIO_LILimitations of the cohort dataset include different response rates between waves, thus some variables were not available for all individuals at certain time points, and IPS scores were derived from a small number of trials
C_LIO_LIThe study involved self-reported measures which might have increased the proportion of misclassification
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.23299817">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.23299817" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.23299817">
        <p class="paperTitle">A web tool for the exploration of hospital prescription data in England</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.23299817" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.23299817" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sanderson, T.</p>
        <p class="info">Score: 16.8, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.23299817' target='https://doi.org/10.1101/2023.12.14.23299817'> 10.1101/2023.12.14.23299817</a></p>
        <p class="abstract">Medicine prescription data is a valuable resource, with the power to reveal insights into healthcare trends, cost efficiencies, and geographic disparities. In England, primary care prescription data has been openly accessible for analysis for some time through a web tool, providing significant benefits. Since 2020, the National Health Service in England has also released data on secondary care prescriptions, i.e. prescribing within hospitals. This is an important dataset, but until now has been available only in a raw form that requires considerable technical skills to be used for even the analysis of basic trends. I have built a web application that enables anyone to easily analyse trends in this data, which is available at hospitalprescriptions.genomium.org.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.11.23299816">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.11.23299816" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.11.23299816">
        <p class="paperTitle">Exploratory electronic health record analysis with ehrapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.11.23299816" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.11.23299816" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Heumos, L.; Ehmele, P.; Treis, T.; Upmeier zu Belzen, J.; Namsaraeva, A.; Horlava, N.; Shitov, V. A.; Zhang, X.; Zappia, L.; Knoll, R.; Lang, N. J.; Hetzel, L.; Virshup, I.; Sikkema, L.; Roellin, E.; Curion, F.; Eils, R.; Schiller, H. B.; Hilgendorff, A.; Theis, F.</p>
        <p class="info">Score: 54.2, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.11.23299816' target='https://doi.org/10.1101/2023.12.11.23299816'> 10.1101/2023.12.11.23299816</a></p>
        <p class="abstract">With progressive digitalization of healthcare systems worldwide, large-scale collection of electronic health records (EHRs) has become commonplace. However, an extensible framework for comprehensive exploratory analysis that accounts for data heterogeneity is missing. Here, we introduce ehrapy, a modular open-source Python framework designed for exploratory end-to-end analysis of heterogeneous epidemiology and electronic health record data. Ehrapy incorporates a series of analytical steps, from data extraction and quality control to the generation of low-dimensional representations. Complemented by rich statistical modules, ehrapy facilitates associating patients with disease states, differential comparison between patient clusters, survival analysis, trajectory inference, causal inference, and more. Leveraging ontologies, ehrapy further enables data sharing and training EHR deep learning models paving the way for foundational models in biomedical research. We demonstrated ehrapys features in five distinct examples: We first applied ehrapy to stratify patients affected by unspecified pneumonia into finer-grained phenotypes. Furthermore, we revealed biomarkers for significant differences in survival among these groups. Additionally, we quantify medication-class effects of pneumonia medications on length of stay. We further leveraged ehrapy to analyze cardiovascular risks across different data modalities. Finally, we reconstructed disease state trajectories in SARS-CoV-2 patients based on imaging data. Ehrapy thus provides a framework that we envision will standardize analysis pipelines on EHR data and serve as a cornerstone for the community.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.06.23300659">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.06.23300659" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.06.23300659">
        <p class="paperTitle">Testing federated analytics across secure data environments using differing statistical approaches on cross-disciplinary data.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.06.23300659" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.06.23300659" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gallier, S.; Topham, A.; Hodson, J.; McNulty, D.; Giles, T.; Cox, S.; Chaganty, J.; Cooper, L.; Perks, S.; Quinlan, P.; Sapey, E.</p>
        <p class="info">Score: 0.8, Published: 2024-01-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.06.23300659' target='https://doi.org/10.1101/2024.01.06.23300659'> 10.1101/2024.01.06.23300659</a></p>
        <p class="abstract">BACKGROUNDIntroducing data-driven technologies into health systems can enhance population health and streamline care delivery. The use of diverse and geographically varied data is key for tackling health and societal challenges, despite associated technical, ethical, and governance complexities. This study explored the efficacy of federated analytics using general linear models (GLMs) and machine learning (ML) models, comparing outcomes with non-federated data analysis.

METHODSA Conditional Transformation Generative Adversarial Network was used to create two synthetic datasets (training set: N=10,000; test set: N=1,000), using real-world data from 381 asthma patients. To simulate a federated environment, the resulting data were distributed across nodes in a Microsoft Azure Trusted Research Environment (TRE). GLMs (one-way ANOVA) and ML models (gradient boosted decision trees) where then produced, using both federated and non-federated approaches. The consistency of predictions produced by the ML models were then compared between approaches, with predictive accuracy of the models quantified by the area under the receiver operating characteristic curve (AUROC).

FINDINGSGLMs produced from federated data distributed between two TREs were identical to those produced using a non-federated approach. However, ML models produced by federated and non-federated approaches, and using different data distributions between TREs, were non-identical. Despite this, when applied to the test set, the classifications made by the federated models were consistent with the non-federated model in 84.7-90.4% of cases, which was similar to the consistency of repeated non-federated models (90.9-91.5%). Consequently, overall predictive accuracies for federated and non-federated models were similar (AUROC: 0.663-0.669).

INTERPRETATIONThis study confirmed the robustness of GLMs utilising ANOVA within a federated framework, yielding consistent outcomes. Moreover, federated ML models demonstrated a high degree of classification agreement, with comparable accuracy to traditional non-federated models. These results highlight the viability of federated approaches for reliable and accurate data analysis in sensitive domains.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
