<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Genome-wide Characterization of Diverse Bacteriophages Enabled by RNA-Binding CRISPRi
Authors: Adler, B. A.; Al-Shimary, M. J.; Patel, J. R.; Ambruster, E.; Colognori, D.; Charles, E. J.; Miller, K. V.; Lahiri, A.; Trinidad, M.; Boger, R.; Nomburg, J.; Beurnier, S.; Cui, M. L.; Barrangou, R.; Mutalik, V. K.; Schoeniger, J. S.; Pogliano, J. A.; Savage, D. F.; Doudna, J. A.; Cress, B. F.
Score: 70.0, Published: 2023-09-18 DOI: 10.1101/2023.09.18.558157
Bacteriophages constitute one of the largest sources of unknown gene content in the biosphere.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Genome-wide Characterization of Diverse Bacteriophages Enabled by RNA-Binding CRISPRi
Authors: Adler, B. A.; Al-Shimary, M. J.; Patel, J. R.; Ambruster, E.; Colognori, D.; Charles, E. J.; Miller, K. V.; Lahiri, A.; Trinidad, M.; Boger, R.; Nomburg, J.; Beurnier, S.; Cui, M. L.; Barrangou, R.; Mutalik, V. K.; Schoeniger, J. S.; Pogliano, J. A.; Savage, D. F.; Doudna, J. A.; Cress, B. F.
Score: 70.0, Published: 2023-09-18 DOI: 10.1101/2023.09.18.558157
Bacteriophages constitute one of the largest sources of unknown gene content in the biosphere." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-24T10:37:06+00:00" />
<meta property="article:modified_time" content="2023-09-24T10:37:06+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Genome-wide Characterization of Diverse Bacteriophages Enabled by RNA-Binding CRISPRi
Authors: Adler, B. A.; Al-Shimary, M. J.; Patel, J. R.; Ambruster, E.; Colognori, D.; Charles, E. J.; Miller, K. V.; Lahiri, A.; Trinidad, M.; Boger, R.; Nomburg, J.; Beurnier, S.; Cui, M. L.; Barrangou, R.; Mutalik, V. K.; Schoeniger, J. S.; Pogliano, J. A.; Savage, D. F.; Doudna, J. A.; Cress, B. F.
Score: 70.0, Published: 2023-09-18 DOI: 10.1101/2023.09.18.558157
Bacteriophages constitute one of the largest sources of unknown gene content in the biosphere."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Genome-wide Characterization of Diverse Bacteriophages Enabled by RNA-Binding CRISPRi\nAuthors: Adler, B. A.; Al-Shimary, M. J.; Patel, J. R.; Ambruster, E.; Colognori, D.; Charles, E. J.; Miller, K. V.; Lahiri, A.; Trinidad, M.; Boger, R.; Nomburg, J.; Beurnier, S.; Cui, M. L.; Barrangou, R.; Mutalik, V. K.; Schoeniger, J. S.; Pogliano, J. A.; Savage, D. F.; Doudna, J. A.; Cress, B. F.\nScore: 70.0, Published: 2023-09-18 DOI: 10.1101/2023.09.18.558157\nBacteriophages constitute one of the largest sources of unknown gene content in the biosphere.",
  "keywords": [
    
  ],
  "articleBody": " Genome-wide Characterization of Diverse Bacteriophages Enabled by RNA-Binding CRISPRi\nAuthors: Adler, B. A.; Al-Shimary, M. J.; Patel, J. R.; Ambruster, E.; Colognori, D.; Charles, E. J.; Miller, K. V.; Lahiri, A.; Trinidad, M.; Boger, R.; Nomburg, J.; Beurnier, S.; Cui, M. L.; Barrangou, R.; Mutalik, V. K.; Schoeniger, J. S.; Pogliano, J. A.; Savage, D. F.; Doudna, J. A.; Cress, B. F.\nScore: 70.0, Published: 2023-09-18 DOI: 10.1101/2023.09.18.558157\nBacteriophages constitute one of the largest sources of unknown gene content in the biosphere. Even for well-studied model phages, robust experimental approaches to identify and study their essential genes remain elusive. We uncover and exploit the conserved vulnerability of the phage transcriptome to facilitate genome-wide protein expression knockdown via programmable RNA-binding protein dRfxCas13d (CRISPRi-ART) across diverse phages and their host. Establishing the first broad-spectrum phage functional genomics platform, we predict over 90 essential genes across four phage genomes, a third of which have no known function. These results highlight hidden infection strategies encoded in the most abundant biological entities on earth and provide a facile platform to study them.\nImmune Evasion, Infectivity, and Fusogenicity of SARS-CoV-2 Omicron BA.2.86 and FLip Variants\nAuthors: Qu, P.; Xu, K.; Faraone, J. N.; Goodarzi, N.; Zheng, Y.-M.; Carlin, C.; Bednash, J. S.; Horowitz, J. C.; Mallampalli, R. K.; Saif, L. J.; Oltz, E. M.; Jones, D.; Gumina, R. J.; Liu, S.-L.\nScore: 132.7, Published: 2023-09-12 DOI: 10.1101/2023.09.11.557206\nEvolution of SARS-CoV-2 requires the reassessment of current vaccine measures. Here, we characterized BA.2.86 and the XBB-lineage variant FLip by investigating their neutralization alongside D614G, BA.1, BA.2, BA.4/5, XBB.1.5, and EG.5.1 by sera from 3-dose vaccinated and bivalent vaccinated healthcare workers, XBB.1.5-wave infected first responders, and monoclonal antibody (mAb) S309. We assessed the biology of the variant Spikes by measuring viral infectivity and membrane fusogenicity. BA.2.86 is less immune evasive compared to FLip and other XBB variants, consistent with antigenic distances. Importantly, distinct from XBB variants, mAb S309 was unable to neutralize BA.2.86, likely due to a D339H mutation based on modeling. BA.2.86 had relatively high fusogenicity and infectivity in CaLu-3 cells but low fusion and infectivity in 293T-ACE2 cells compared to some XBB variants, suggesting a potentially differences conformational stability of BA.2.86 Spike. Overall, our study underscores the importance of SARS-CoV-2 variant surveillance and the need for updated COVID-19 vaccines.\nTMPRSS2 activation of Omicron lineage Spike glycoproteins is regulated by TMPRSS2 cleavage of ACE2\nAuthors: Aggarwal, A.; Fichter, C.; Milogiannakis, V.; Akerman, A.; Ison, T.; Ruiz Silva, M.; Esneau, C.; Bartlett, N.; Burrell, L.; Patel, S.; Churchill, M.; Angelovich, T.; Parry, R.; Sng, J. D.; Neely, G. G.; Moreno, C. L.; Loo, L.; Kelleher, A. D.; Brilot, F.; Khromykh, A.; Turville, S. G.\nScore: 61.5, Published: 2023-09-22 DOI: 10.1101/2023.09.22.558930\nContinued high levels spread of SARS-CoV-2 globally enabled accumulation of changes within the Spike glycoprotein, leading to resistance to neutralising antibodies and concomitant changes to entry requirements that increased viral transmission fitness. Herein, we demonstrate a significant change in ACE2 and TMPRSS2 use by primary SARS-CoV-2 isolates that occurred upon arrival of Omicron lineages. Mechanistically we show this shift to be a function of two distinct ACE2 pools based on cleavage or non-cleavage of ACE2 by TMPRSS2 activity. In engineered cells overexpressing ACE2 and TMPRSS2, ACE2 was cleaved by TMPRSS2 and this led to either augmentation or progressive attenuation of pre-Omicron and Omicron lineages, respectfully. In contrast, TMPRSS2 resistant ACE2 restored infectivity across all Omicron lineages through enabling ACE2 binding that facilitated TMPRSS2 spike activation. Therefore, our data support the tropism shift of Omicron lineages to be a function of evolution towards the use of uncleaved pools of ACE2 with the latter consistent with its role as a chaperone for many tissue specific amino acid transport proteins.\nNeutralization Escape by SARS-CoV-2 Omicron Subvariant BA.2.86\nAuthors: Lasrado, N.; Collier, A.-r.; Hachmann, N.; Miller, J.; Rowe, M.; Schonberg, E.; Rodrigues, S.; LaPiana, A.; Patio, R.; Anand, T.; Fisher, J.; Mazurek, C.; Guan, R.; Wagh, K.; Theiler, J.; Korber, B.; Barouch, D. H.\nScore: 824.7, Published: 2023-09-05 DOI: 10.1101/2023.09.04.556272\nThe continued evolution of SARS-CoV-2 may lead to evasion of vaccine immunity and natural immunity. A highly mutated Omicron variant BA.2.86 has recently been identified with over 30 amino acid changes in Spike compared with BA.2 and XBB.1.5. As of September 4, 2023, BA.2.86 has been identified in 37 sequences from 10 countries, which is likely an underestimate due to limited surveillance. The ability of BA.2.86 to evade NAbs compared with other currently circulating Omicron variants remains unknown. Our data show that NAb responses to BA.2.86 were lower than to BA.2 but were comparable or slightly higher than to the current circulating recombinant variants XBB.1.5, XBB.1.16, EG.5, EG.5.1, and FL.1.5.1.\nA Parasite Odyssey: An RNA virus concealed in Toxoplasma gondii\nAuthors: Gupta, P.; Hiller, A.; Chowdhury, J.; Lim, D.; Lim, D.; Babaian, A.; Pereira, L.; Morales, A.\nScore: 31.2, Published: 2023-09-18 DOI: 10.1101/2023.09.17.558162\nWe are in the midst of the \"Platinum Age of Virus Discovery\", an era characterized by the exponential growth in the discovery of virus biodiversity. In humans (or any animal) there are more viruses than simply those directly infecting the animal cells. Viruses infect all species constituting the microbiome: including bacteria, fungi, and unicellular parasites. Thus the complexity of possible interactions between host, microbe, and viruses is unfathomable. To understand this network we must employ computationally assisted virology as a means of analyzing and interpreting the millions of available samples to decipher the myriad of ways in which viruses may intersect human health. From a global screen for human neuron-associated viruses, we identify a novel narnavirus Achaeavirus odysseus (Ao) which likely infects the neurotropic parasite Toxoplasma gondii. Previously, several parasitic protozoan viruses (PPVs) have been mechanistically established as triggers of host innate responses, and here we present in silico evidence that Ao is a plausible pro-inflammatory factor in mouse and human cells infected by T. gondii. T. gondii infects billions of people worldwide, yet the prognosis of toxoplasmosis disease is highly variable, and PPVs like Ao could function as a hitherto undescribed hypervirulence factor. More broadly, we explore phylogenetically proximal viruses to Ao and discover 18 new species of Achaeavirus, all found in vertebrate transcriptome and metatranscriptomes. While the Narnavirus samples making up this genus-like clade are derived from sheep, goat, bat, rabbit, chicken, and pigeon, the presence of virus is strongly predictive of parasitic (Apicomplexa) nucleic acid co-occurrence, supporting that these are a clade of parasite-infecting viruses. This is a computational proof-of-concept study in which we rapidly analyze millions of datasets from which we distil a mechanistically, ecologically, and phylogenetically refined hypothesis. We predict this highly diverged Ao RNA virus is biologically a T. gondii infection, and that Ao, and other viruses like it, will modulate this disease which afflicts billions worldwide.\nEvolutionary dynamics of proteinaceous MAMPs reveals intrabacterial antagonism of plant immune perception\nAuthors: Stevens, D. M.; Moreno-Perez, A.; Weisberg, A. J.; Ramsing, C.; Fliegmann, J.; Zhang, N.; Madrigal, M.; Martin, G.; Steinbrenner, A.; Felix, G.; Coaker, G.\nScore: 18.3, Published: 2023-09-21 DOI: 10.1101/2023.09.21.558511\nPlants and animals detect biomolecules termed Microbe-Associated Molecular Patterns (MAMPs) and induce immune responses. The impact of diverse multi-copy MAMPs on immune induction is unknown. We characterized the evolutionary trajectories of five proteinaceous MAMPs and the effect of copy number and sequence variation on plant immune outcomes. From 4,228 plant-associated bacterial genomes, 34,262 MAMPs were identified. Natural variation was constricted, enabling characterization of immune perception across thousands of bacteria. We characterized the immunogenic outcomes of two MAMPs with different evolutionary trajectories the Elongation Factor Tu (EF-Tu) epitope elf18 and Cold Shock Protein (CSP) epitope csp22. EF-Tu is a single-copy gene and 90% of elf18 epitopes are immunogenic. Conversely, CSPs are multi-copy with 99.8% of bacteria carrying at least one immunogenic CSP and only 54% of csp22 epitopes are immunogenic. We uncovered a new mechanism for immune evasion, intrabacterial antagonism, from a non-immunogenic actinobacterial CSP that blocks perception of immunogenic forms.\nA conserved cell-pole determinant organizes proper polar flagellum formation\nAuthors: Arroyo-Perez, E. E.; Hook, J. C.; Alvarado, A.; Wimmi, S.; Glatter, T.; Thormann, K. M.; Ringgaard, S.\nScore: 13.7, Published: 2023-09-21 DOI: 10.1101/2023.09.20.558563\nThe coordination of cell cycle progression and flagellar synthesis is a complex process in motile bacteria. In gamma-proteobacteria, the localization of the flagellum to the cell pole is mediated by the SRP-type GTPase FlhF. However, the mechanism of action of FlhF, and its relationship with the cell pole landmark protein HubP remain unclear. In this study, we discovered a novel protein called FipA that is required for normal FlhF activity and function in polar flagellar synthesis. We demonstrated that membrane-localized FipA interacts with FlhF and is required for normal flagellar synthesis in Vibrio parahaemolyticus, Pseudomonas putida, and Shewanella putrefaciens, and it does so independently of the polar localization mediated by HubP. FipA exhibits a dynamic localization pattern and is present at the designated pole before flagellar synthesis begins, suggesting its role in licensing flagellar formation. This discovery provides insight into a new pathway for regulating flagellum synthesis and coordinating cellular organization in bacteria that rely on polar flagellation and FlhF-dependent localization.\nA phage nucleus-associated RNA-binding protein is required for jumbo phage infection\nAuthors: Enustun, E.; Armbruster, E. G.; Lee, J.; Zhang, S.; Yee, B. A.; Gu, Y.; Deep, A.; Naritomi, J. T.; Liang, Q.; Aigner, S.; Adler, B. A.; Cress, B. F.; Doudna, J. A.; Chaikeeratisak, V.; Cleveland, D. W.; Ghassemian, M.; Yeo, G. W.; Pogliano, J.; Corbett, K. D.\nScore: 11.7, Published: 2023-09-22 DOI: 10.1101/2023.09.22.559000\nLarge-genome bacteriophages (jumbo phages) of the Chimalliviridae family assemble a nucleus-like compartment bounded by a protein shell that protects the replicating phage genome from host-encoded restriction enzymes and CRISPR/Cas nucleases. While the nuclear shell provides broad protection against host nucleases, it necessitates transport of mRNA out of the nucleus-like compartment for translation by host ribosomes, and transport of specific proteins into the nucleus-like compartment to support DNA replication and mRNA transcription. Here we identify a conserved phage nuclear shell-associated protein that we term Chimallin C (ChmC), which adopts a nucleic acid-binding fold, binds RNA with high affinity in vitro, and binds phage mRNAs in infected cells. ChmC also forms phase-separated condensates with RNA in vitro. Targeted knockdown of ChmC using mRNA-targeting dCas13d halts infections at an early stage. Taken together, our data suggest that the conserved ChmC protein acts as a chaperone for phage mRNAs, potentially stabilizing these mRNAs and driving their translocation through the nuclear shell to promote translation and infection progression.\nAsgard archaea defense systems and their roles in the origin of immunity in eukaryotes\nAuthors: Leao, P.; Little, M. E.; Appler, K. E.; Sahaya, D.; Aguilar-Pine, E.; Currie, K.; Finkelstein, I. J.; De Anda, V.; Baker, B. J.\nScore: 81.5, Published: 2023-09-13 DOI: 10.1101/2023.09.13.557551\nImmune systems are integral to survival against viral infection. Recently, dozens of new anti-viral systems have been characterized in bacteria1. Some of these systems are present in eukaryotes and appear to have originated in prokaryotes. However, little is known about these defense mechanisms in archaea. Here, we identified 2,610 complete defense systems in archaea related to eukaryotes, the Asgardarchaeota2. These comprise 89 unique systems, including argonaute, NLR, mokosh, viperin, lassamu, and CBASS. Asgard viperin (asVip) and argonaute (asAgo) proteins are present at high frequencies compared to bacteria and have structural homology to eukaryotes. Phylogenetic analyses revealed asVips are ancestral eukaryotic proteins. Heterologous expression of asVips in bacteria, including the lineage closest to eukaryotes, Hodarchaeales, showed robust anti-phage activity. Eukaryotic-and bacterial-argonaute proteins appear to have originated in the Asgardarchaeota, and have ancient structural characteristics. AsAgos appear to have argonaute-PIWI domains which are key components of the RNA interference (RNAi) in eukaryotes. Characterization of hundreds of new defense systems in the Asgardarchaeota revealed these archaea played important roles in the innovation of viral protection in eukaryotes. Given their relationship to eukaryotes, these defense systems may have applications in biomedicine and biotechnology.\nEndozoicomonas provides corals with steroid hormones during thermal stress\nAuthors: Ochsenkuhn, M.; Mohamed, A. R.; Haydon, T. D.; Coe, L. S.; Abrego, D.; Amin, S. A.\nScore: 11.0, Published: 2023-09-19 DOI: 10.1101/2023.09.19.558257\nSummary ParagraphRising temperatures are devastating coral populations throughout the globe1. The coral microbiome is believed to play a critical role in sustaining corals and enabling their adaptation to environmental changes, particularly thermal stress2-4. A ubiquitous group of coral-associated bacteria, known as Endozoicomonas, are hypothesized to provide their host with essential metabolites5. However, the nature of coral-Endozoicomonas symbiosis and the role these bacteria play in adaptation to thermal stress are largely unknown. Here, we show that symbiotic Endozoicomonas adapt to the host environment by gaining the ability to degrade coral-derived steroids and Symbiodiniaceae-derived galactose while losing the ability to synthesize iron-binding siderophores, a common feature of free-living marine bacteria6. More importantly, under thermal stress Endozoicomonas utilizes coral-derived cholesterol partially as a carbon source while simultaneously converting it to the hormones testosterone and progesterone. Both steroids prime the innate immune system and inhibit pathogenic bacteria and fungi7. These findings highlight an unknown interaction between corals and their microbiome that may be critical to coral health as oceans warm up. The ability of bacteria to synthesize eukaryotic steroid hormones underscores the importance of these molecules in inter-kingdom interactions and suggests that their origin may have evolved as a result of eukaryogenesis.\n",
  "wordCount" : "2201",
  "inLanguage": "en",
  "datePublished": "2023-09-24T10:37:06Z",
  "dateModified": "2023-09-24T10:37:06Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta"><span>updated on September 24, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558157">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558157" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558157">
        <p class="paperTitle">Genome-wide Characterization of Diverse Bacteriophages Enabled by RNA-Binding CRISPRi</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558157" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558157" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Adler, B. A.; Al-Shimary, M. J.; Patel, J. R.; Ambruster, E.; Colognori, D.; Charles, E. J.; Miller, K. V.; Lahiri, A.; Trinidad, M.; Boger, R.; Nomburg, J.; Beurnier, S.; Cui, M. L.; Barrangou, R.; Mutalik, V. K.; Schoeniger, J. S.; Pogliano, J. A.; Savage, D. F.; Doudna, J. A.; Cress, B. F.</p>
        <p class="info">Score: 70.0, Published: 2023-09-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558157' target='https://doi.org/10.1101/2023.09.18.558157'> 10.1101/2023.09.18.558157</a></p>
        <p class="abstract">Bacteriophages constitute one of the largest sources of unknown gene content in the biosphere. Even for well-studied model phages, robust experimental approaches to identify and study their essential genes remain elusive. We uncover and exploit the conserved vulnerability of the phage transcriptome to facilitate genome-wide protein expression knockdown via programmable RNA-binding protein dRfxCas13d (CRISPRi-ART) across diverse phages and their host. Establishing the first broad-spectrum phage functional genomics platform, we predict over 90 essential genes across four phage genomes, a third of which have no known function. These results highlight hidden infection strategies encoded in the most abundant biological entities on earth and provide a facile platform to study them.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557206">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557206" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557206">
        <p class="paperTitle">Immune Evasion, Infectivity, and Fusogenicity of SARS-CoV-2 Omicron BA.2.86 and FLip Variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557206" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557206" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Qu, P.; Xu, K.; Faraone, J. N.; Goodarzi, N.; Zheng, Y.-M.; Carlin, C.; Bednash, J. S.; Horowitz, J. C.; Mallampalli, R. K.; Saif, L. J.; Oltz, E. M.; Jones, D.; Gumina, R. J.; Liu, S.-L.</p>
        <p class="info">Score: 132.7, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557206' target='https://doi.org/10.1101/2023.09.11.557206'> 10.1101/2023.09.11.557206</a></p>
        <p class="abstract">Evolution of SARS-CoV-2 requires the reassessment of current vaccine measures. Here, we characterized BA.2.86 and the XBB-lineage variant FLip by investigating their neutralization alongside D614G, BA.1, BA.2, BA.4/5, XBB.1.5, and EG.5.1 by sera from 3-dose vaccinated and bivalent vaccinated healthcare workers, XBB.1.5-wave infected first responders, and monoclonal antibody (mAb) S309. We assessed the biology of the variant Spikes by measuring viral infectivity and membrane fusogenicity. BA.2.86 is less immune evasive compared to FLip and other XBB variants, consistent with antigenic distances. Importantly, distinct from XBB variants, mAb S309 was unable to neutralize BA.2.86, likely due to a D339H mutation based on modeling. BA.2.86 had relatively high fusogenicity and infectivity in CaLu-3 cells but low fusion and infectivity in 293T-ACE2 cells compared to some XBB variants, suggesting a potentially differences conformational stability of BA.2.86 Spike. Overall, our study underscores the importance of SARS-CoV-2 variant surveillance and the need for updated COVID-19 vaccines.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.558930">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.558930" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.558930">
        <p class="paperTitle">TMPRSS2 activation of Omicron lineage Spike glycoproteins is regulated by TMPRSS2 cleavage of ACE2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.558930" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.558930" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aggarwal, A.; Fichter, C.; Milogiannakis, V.; Akerman, A.; Ison, T.; Ruiz Silva, M.; Esneau, C.; Bartlett, N.; Burrell, L.; Patel, S.; Churchill, M.; Angelovich, T.; Parry, R.; Sng, J. D.; Neely, G. G.; Moreno, C. L.; Loo, L.; Kelleher, A. D.; Brilot, F.; Khromykh, A.; Turville, S. G.</p>
        <p class="info">Score: 61.5, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.558930' target='https://doi.org/10.1101/2023.09.22.558930'> 10.1101/2023.09.22.558930</a></p>
        <p class="abstract">Continued high levels spread of SARS-CoV-2 globally enabled accumulation of changes within the Spike glycoprotein, leading to resistance to neutralising antibodies and concomitant changes to entry requirements that increased viral transmission fitness. Herein, we demonstrate a significant change in ACE2 and TMPRSS2 use by primary SARS-CoV-2 isolates that occurred upon arrival of Omicron lineages. Mechanistically we show this shift to be a function of two distinct ACE2 pools based on cleavage or non-cleavage of ACE2 by TMPRSS2 activity. In engineered cells overexpressing ACE2 and TMPRSS2, ACE2 was cleaved by TMPRSS2 and this led to either augmentation or progressive attenuation of pre-Omicron and Omicron lineages, respectfully. In contrast, TMPRSS2 resistant ACE2 restored infectivity across all Omicron lineages through enabling ACE2 binding that facilitated TMPRSS2 spike activation. Therefore, our data support the tropism shift of Omicron lineages to be a function of evolution towards the use of uncleaved pools of ACE2 with the latter consistent with its role as a chaperone for many tissue specific amino acid transport proteins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.04.556272">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.04.556272" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.04.556272">
        <p class="paperTitle">Neutralization Escape by SARS-CoV-2 Omicron Subvariant BA.2.86</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.04.556272" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.04.556272" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lasrado, N.; Collier, A.-r.; Hachmann, N.; Miller, J.; Rowe, M.; Schonberg, E.; Rodrigues, S.; LaPiana, A.; Patio, R.; Anand, T.; Fisher, J.; Mazurek, C.; Guan, R.; Wagh, K.; Theiler, J.; Korber, B.; Barouch, D. H.</p>
        <p class="info">Score: 824.7, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.04.556272' target='https://doi.org/10.1101/2023.09.04.556272'> 10.1101/2023.09.04.556272</a></p>
        <p class="abstract">The continued evolution of SARS-CoV-2 may lead to evasion of vaccine immunity and natural immunity. A highly mutated Omicron variant BA.2.86 has recently been identified with over 30 amino acid changes in Spike compared with BA.2 and XBB.1.5. As of September 4, 2023, BA.2.86 has been identified in 37 sequences from 10 countries, which is likely an underestimate due to limited surveillance. The ability of BA.2.86 to evade NAbs compared with other currently circulating Omicron variants remains unknown. Our data show that NAb responses to BA.2.86 were lower than to BA.2 but were comparable or slightly higher than to the current circulating recombinant variants XBB.1.5, XBB.1.16, EG.5, EG.5.1, and FL.1.5.1.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.17.558162">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.17.558162" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.17.558162">
        <p class="paperTitle">A Parasite Odyssey: An RNA virus concealed in Toxoplasma gondii</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.17.558162" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.17.558162" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gupta, P.; Hiller, A.; Chowdhury, J.; Lim, D.; Lim, D.; Babaian, A.; Pereira, L.; Morales, A.</p>
        <p class="info">Score: 31.2, Published: 2023-09-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.17.558162' target='https://doi.org/10.1101/2023.09.17.558162'> 10.1101/2023.09.17.558162</a></p>
        <p class="abstract">We are in the midst of the &#34;Platinum Age of Virus Discovery&#34;, an era characterized by the exponential growth in the discovery of virus biodiversity. In humans (or any animal) there are more viruses than simply those directly infecting the animal cells. Viruses infect all species constituting the microbiome: including bacteria, fungi, and unicellular parasites. Thus the complexity of possible interactions between host, microbe, and viruses is unfathomable. To understand this network we must employ computationally assisted virology as a means of analyzing and interpreting the millions of available samples to decipher the myriad of ways in which viruses may intersect human health.

From a global screen for human neuron-associated viruses, we identify a novel narnavirus Achaeavirus odysseus (Ao) which likely infects the neurotropic parasite Toxoplasma gondii. Previously, several parasitic protozoan viruses (PPVs) have been mechanistically established as triggers of host innate responses, and here we present in silico evidence that Ao is a plausible pro-inflammatory factor in mouse and human cells infected by T. gondii. T. gondii infects billions of people worldwide, yet the prognosis of toxoplasmosis disease is highly variable, and PPVs like Ao could function as a hitherto undescribed hypervirulence factor. More broadly, we explore phylogenetically proximal viruses to Ao and discover 18 new species of Achaeavirus, all found in vertebrate transcriptome and metatranscriptomes. While the Narnavirus samples making up this genus-like clade are derived from sheep, goat, bat, rabbit, chicken, and pigeon, the presence of virus is strongly predictive of parasitic (Apicomplexa) nucleic acid co-occurrence, supporting that these are a clade of parasite-infecting viruses.

This is a computational proof-of-concept study in which we rapidly analyze millions of datasets from which we distil a mechanistically, ecologically, and phylogenetically refined hypothesis. We predict this highly diverged Ao RNA virus is biologically a T. gondii infection, and that Ao, and other viruses like it, will modulate this disease which afflicts billions worldwide.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558511">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558511" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558511">
        <p class="paperTitle">Evolutionary dynamics of proteinaceous MAMPs reveals intrabacterial antagonism of plant immune perception</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558511" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558511" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stevens, D. M.; Moreno-Perez, A.; Weisberg, A. J.; Ramsing, C.; Fliegmann, J.; Zhang, N.; Madrigal, M.; Martin, G.; Steinbrenner, A.; Felix, G.; Coaker, G.</p>
        <p class="info">Score: 18.3, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558511' target='https://doi.org/10.1101/2023.09.21.558511'> 10.1101/2023.09.21.558511</a></p>
        <p class="abstract">Plants and animals detect biomolecules termed Microbe-Associated Molecular Patterns (MAMPs) and induce immune responses. The impact of diverse multi-copy MAMPs on immune induction is unknown. We characterized the evolutionary trajectories of five proteinaceous MAMPs and the effect of copy number and sequence variation on plant immune outcomes. From 4,228 plant-associated bacterial genomes, 34,262 MAMPs were identified. Natural variation was constricted, enabling characterization of immune perception across thousands of bacteria. We characterized the immunogenic outcomes of two MAMPs with different evolutionary trajectories the Elongation Factor Tu (EF-Tu) epitope elf18 and Cold Shock Protein (CSP) epitope csp22. EF-Tu is a single-copy gene and 90% of elf18 epitopes are immunogenic. Conversely, CSPs are multi-copy with 99.8% of bacteria carrying at least one immunogenic CSP and only 54% of csp22 epitopes are immunogenic. We uncovered a new mechanism for immune evasion, intrabacterial antagonism, from a non-immunogenic actinobacterial CSP that blocks perception of immunogenic forms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.558563">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.558563" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.558563">
        <p class="paperTitle">A conserved cell-pole determinant organizes proper polar flagellum formation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.558563" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.558563" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Arroyo-Perez, E. E.; Hook, J. C.; Alvarado, A.; Wimmi, S.; Glatter, T.; Thormann, K. M.; Ringgaard, S.</p>
        <p class="info">Score: 13.7, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.558563' target='https://doi.org/10.1101/2023.09.20.558563'> 10.1101/2023.09.20.558563</a></p>
        <p class="abstract">The coordination of cell cycle progression and flagellar synthesis is a complex process in motile bacteria. In gamma-proteobacteria, the localization of the flagellum to the cell pole is mediated by the SRP-type GTPase FlhF. However, the mechanism of action of FlhF, and its relationship with the cell pole landmark protein HubP remain unclear. In this study, we discovered a novel protein called FipA that is required for normal FlhF activity and function in polar flagellar synthesis. We demonstrated that membrane-localized FipA interacts with FlhF and is required for normal flagellar synthesis in Vibrio parahaemolyticus, Pseudomonas putida, and Shewanella putrefaciens, and it does so independently of the polar localization mediated by HubP. FipA exhibits a dynamic localization pattern and is present at the designated pole before flagellar synthesis begins, suggesting its role in licensing flagellar formation. This discovery provides insight into a new pathway for regulating flagellum synthesis and coordinating cellular organization in bacteria that rely on polar flagellation and FlhF-dependent localization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.559000">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.559000" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.559000">
        <p class="paperTitle">A phage nucleus-associated RNA-binding protein is required for jumbo phage infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.559000" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.559000" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Enustun, E.; Armbruster, E. G.; Lee, J.; Zhang, S.; Yee, B. A.; Gu, Y.; Deep, A.; Naritomi, J. T.; Liang, Q.; Aigner, S.; Adler, B. A.; Cress, B. F.; Doudna, J. A.; Chaikeeratisak, V.; Cleveland, D. W.; Ghassemian, M.; Yeo, G. W.; Pogliano, J.; Corbett, K. D.</p>
        <p class="info">Score: 11.7, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.559000' target='https://doi.org/10.1101/2023.09.22.559000'> 10.1101/2023.09.22.559000</a></p>
        <p class="abstract">Large-genome bacteriophages (jumbo phages) of the Chimalliviridae family assemble a nucleus-like compartment bounded by a protein shell that protects the replicating phage genome from host-encoded restriction enzymes and CRISPR/Cas nucleases. While the nuclear shell provides broad protection against host nucleases, it necessitates transport of mRNA out of the nucleus-like compartment for translation by host ribosomes, and transport of specific proteins into the nucleus-like compartment to support DNA replication and mRNA transcription. Here we identify a conserved phage nuclear shell-associated protein that we term Chimallin C (ChmC), which adopts a nucleic acid-binding fold, binds RNA with high affinity in vitro, and binds phage mRNAs in infected cells. ChmC also forms phase-separated condensates with RNA in vitro. Targeted knockdown of ChmC using mRNA-targeting dCas13d halts infections at an early stage. Taken together, our data suggest that the conserved ChmC protein acts as a chaperone for phage mRNAs, potentially stabilizing these mRNAs and driving their translocation through the nuclear shell to promote translation and infection progression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.13.557551">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.13.557551" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.13.557551">
        <p class="paperTitle">Asgard archaea defense systems and their roles in the origin of immunity in eukaryotes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.13.557551" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.13.557551" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Leao, P.; Little, M. E.; Appler, K. E.; Sahaya, D.; Aguilar-Pine, E.; Currie, K.; Finkelstein, I. J.; De Anda, V.; Baker, B. J.</p>
        <p class="info">Score: 81.5, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.13.557551' target='https://doi.org/10.1101/2023.09.13.557551'> 10.1101/2023.09.13.557551</a></p>
        <p class="abstract">Immune systems are integral to survival against viral infection. Recently, dozens of new anti-viral systems have been characterized in bacteria1. Some of these systems are present in eukaryotes and appear to have originated in prokaryotes. However, little is known about these defense mechanisms in archaea. Here, we identified 2,610 complete defense systems in archaea related to eukaryotes, the Asgardarchaeota2. These comprise 89 unique systems, including argonaute, NLR, mokosh, viperin, lassamu, and CBASS. Asgard viperin (asVip) and argonaute (asAgo) proteins are present at high frequencies compared to bacteria and have structural homology to eukaryotes. Phylogenetic analyses revealed asVips are ancestral eukaryotic proteins. Heterologous expression of asVips in bacteria, including the lineage closest to eukaryotes, Hodarchaeales, showed robust anti-phage activity. Eukaryotic-and bacterial-argonaute proteins appear to have originated in the Asgardarchaeota, and have ancient structural characteristics. AsAgos appear to have argonaute-PIWI domains which are key components of the RNA interference (RNAi) in eukaryotes. Characterization of hundreds of new defense systems in the Asgardarchaeota revealed these archaea played important roles in the innovation of viral protection in eukaryotes. Given their relationship to eukaryotes, these defense systems may have applications in biomedicine and biotechnology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.19.558257">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.19.558257" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.19.558257">
        <p class="paperTitle">Endozoicomonas provides corals with steroid hormones during thermal stress</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.19.558257" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.19.558257" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ochsenkuhn, M.; Mohamed, A. R.; Haydon, T. D.; Coe, L. S.; Abrego, D.; Amin, S. A.</p>
        <p class="info">Score: 11.0, Published: 2023-09-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.19.558257' target='https://doi.org/10.1101/2023.09.19.558257'> 10.1101/2023.09.19.558257</a></p>
        <p class="abstract">Summary ParagraphRising temperatures are devastating coral populations throughout the globe1. The coral microbiome is believed to play a critical role in sustaining corals and enabling their adaptation to environmental changes, particularly thermal stress2-4. A ubiquitous group of coral-associated bacteria, known as Endozoicomonas, are hypothesized to provide their host with essential metabolites5. However, the nature of coral-Endozoicomonas symbiosis and the role these bacteria play in adaptation to thermal stress are largely unknown. Here, we show that symbiotic Endozoicomonas adapt to the host environment by gaining the ability to degrade coral-derived steroids and Symbiodiniaceae-derived galactose while losing the ability to synthesize iron-binding siderophores, a common feature of free-living marine bacteria6. More importantly, under thermal stress Endozoicomonas utilizes coral-derived cholesterol partially as a carbon source while simultaneously converting it to the hormones testosterone and progesterone. Both steroids prime the innate immune system and inhibit pathogenic bacteria and fungi7. These findings highlight an unknown interaction between corals and their microbiome that may be critical to coral health as oceans warm up. The ability of bacteria to synthesize eukaryotic steroid hormones underscores the importance of these molecules in inter-kingdom interactions and suggests that their origin may have evolved as a result of eukaryogenesis.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
