<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Persistence of an infectious form of SARS-CoV-2 post protease inhibitor treatment of permissive cells in vitro
Authors: Nair, M. S.; Luck, M. I.; Huang, Y.; Sabo, Y.; Ho, D. D.
Score: 67.9, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572655
Reports have described SARS-CoV-2 rebound in COVID-19 patients treated with nirmatrelvir, a 3CL protease inhibitor. The cause remains a mystery, although drug resistance, re-infection, and lack of adequate immune responses have been excluded. We now present virologic findings that provide a clue to the cause of viral rebound, which occurs in [~]20% of the treated cases.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Persistence of an infectious form of SARS-CoV-2 post protease inhibitor treatment of permissive cells in vitro
Authors: Nair, M. S.; Luck, M. I.; Huang, Y.; Sabo, Y.; Ho, D. D.
Score: 67.9, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572655
Reports have described SARS-CoV-2 rebound in COVID-19 patients treated with nirmatrelvir, a 3CL protease inhibitor. The cause remains a mystery, although drug resistance, re-infection, and lack of adequate immune responses have been excluded. We now present virologic findings that provide a clue to the cause of viral rebound, which occurs in [~]20% of the treated cases." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-03T10:39:43+00:00" />
<meta property="article:modified_time" content="2024-01-03T10:39:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Persistence of an infectious form of SARS-CoV-2 post protease inhibitor treatment of permissive cells in vitro
Authors: Nair, M. S.; Luck, M. I.; Huang, Y.; Sabo, Y.; Ho, D. D.
Score: 67.9, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572655
Reports have described SARS-CoV-2 rebound in COVID-19 patients treated with nirmatrelvir, a 3CL protease inhibitor. The cause remains a mystery, although drug resistance, re-infection, and lack of adequate immune responses have been excluded. We now present virologic findings that provide a clue to the cause of viral rebound, which occurs in [~]20% of the treated cases."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Persistence of an infectious form of SARS-CoV-2 post protease inhibitor treatment of permissive cells in vitro\nAuthors: Nair, M. S.; Luck, M. I.; Huang, Y.; Sabo, Y.; Ho, D. D.\nScore: 67.9, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572655\nReports have described SARS-CoV-2 rebound in COVID-19 patients treated with nirmatrelvir, a 3CL protease inhibitor. The cause remains a mystery, although drug resistance, re-infection, and lack of adequate immune responses have been excluded. We now present virologic findings that provide a clue to the cause of viral rebound, which occurs in [~]20% of the treated cases.",
  "keywords": [
    
  ],
  "articleBody": " Persistence of an infectious form of SARS-CoV-2 post protease inhibitor treatment of permissive cells in vitro\nAuthors: Nair, M. S.; Luck, M. I.; Huang, Y.; Sabo, Y.; Ho, D. D.\nScore: 67.9, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572655\nReports have described SARS-CoV-2 rebound in COVID-19 patients treated with nirmatrelvir, a 3CL protease inhibitor. The cause remains a mystery, although drug resistance, re-infection, and lack of adequate immune responses have been excluded. We now present virologic findings that provide a clue to the cause of viral rebound, which occurs in [~]20% of the treated cases. The persistence of an intermediary form of infectious SARS-CoV-2 was experimentally documented in vitro after treatment with nirmatrelvir or another 3CL protease inhibitor, but not with a polymerase inhibitor, remdesivir. This infectious intermediate decayed slowly with a half-life of [~]1 day, suggesting that its persistence could outlive the treatment course to re-ignited SARS-CoV-2 infection as the drug is eliminated. Additional studies are needed to define the nature of this viral intermediate, but our findings point to a particular direction for future investigation and offer a specific treatment recommendation that should be tested clinically.\nVirological characteristics of the SARS-CoV-2 JN.1 variant\nAuthors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Zahradnik, J.; Ito, J.; Sato, K.\nScore: 774.8, Published: 2023-12-09 DOI: 10.1101/2023.12.08.570782\nThe SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG.5.1 and HK.3. Comparing to XBB and BA.2, BA.2.86 carries more than 30 mutations in the spike (S) protein, indicating a high potential for immune evasion. BA.2.86 has evolved and its descendant, JN.1 (BA.2.86.1.1), emerged in late 2023. JN.1 harbors S:L455S and three mutations in non-S proteins. S:L455S is a hallmark mutation of JN.1: we have recently shown that HK.3 and other \"FLip\" variants carry S:L455F, which contributes to increased transmissibility and immune escape ability compared to the parental EG.5.1 variant. Here, we investigated the virological properties of JN.1.\nVirion morphology and on-virus spike protein structures of diverse SARS-CoV-2 variants\nAuthors: Ke, Z.; Peacock, T. P.; Brown, J. C.; Sheppard, C. M.; Croll, T. I.; Kotecha, A.; Goldhill, D. H.; Barclay, W. S.; Briggs, J. A.\nScore: 27.2, Published: 2023-12-22 DOI: 10.1101/2023.12.21.572824\nThe evolution of SARS-CoV-2 variants with increased fitness has been accompanied by structural changes in the spike (S) proteins that are the major target for the adaptive immune response. Single-particle cryo-EM analysis of soluble S from SARS-CoV-2 variants has revealed this structural adaptation at high-resolution. The analysis of S trimers in situ on intact virions has the potential to provide more functionally relevant insights into S structure and virion morphology. Here, we characterized B.1, Alpha, Beta, Gamma, Delta, Kappa, and Mu variants by cryo-electron microscopy and tomography, assessing S cleavage, virion morphology, S incorporation, \"in-situ\" high-resolution S structures and the range of S conformational states. We found no evidence for adaptive changes in virion morphology, but describe multiple different positions in the S protein where amino acid changes alter local protein structure. Considered together, our data is consistent with a model where amino acid changes at multiple positions from the top to the base of the spike cause structural changes that can modulate the conformational dynamics of S.\nPrecision Genome Editing Unveils a Breakthrough in Reversing Antibiotic Resistance: CRISPR/Cas9 Targeting of Multi-Drug Resistance Genes in Methicillin-Resistant Staphylococcus aureus\nAuthors: Ates, A.; Tastan, C.; Ermertcan, S.\nScore: 11.1, Published: 2024-01-01 DOI: 10.1101/2023.12.31.573511\nAntibiotic resistance poses a global health crisis, limiting the efficacy of available therapeutic agents and hindering the development of new antibiotics. The pharmaceutical industrys waning interest in antibiotic production further exacerbates this challenge. Addressing antibiotic resistance demands innovative solutions. Here, we explore the application of CRISPR-Cas-based antimicrobials as a pioneering approach to combat multidrug resistance. Our study focuses on the methicillin-resistant clinical Staphylococcus aureus (MRSA), a significant clinical threat. Utilizing CRISPR/Cas9 technology, we aimed to concurrently target methicillin (mecA), gentamicin (aacA), and ciprofloxacin (grlA, grlB) resistance genes, thereby altering the resistance profile and enhancing sensitivity to antibiotics. We engineered CRISPR plasmids containing sgRNAs specific to the target regions, which were subsequently electroporated into MRSA strains. Real-time Polymerase Chain Reaction (RT-PCR) assessed changes in resistance gene expression, while disk diffusion and broth microdilution methods determined alterations in resistance status. Western blotting analyzed changes in PBP2a expression, and Sanger sequence analysis confirmed mutations in target regions. Results revealed a statistically significant 1.5-fold decrease in mecA gene expression, 5.5-fold decrease in grlA gene, 6-fold decrease in grlB gene, and 4-fold decrease in aacA gene compared to the wild type strain, as determined by RT-PCR. Antibiotic susceptibility tests demonstrated the suppression of mecA, grlA, grlB, and aacA genes, resulting in the reversal of resistance to beta-lactam, quinolone, and aminoglycoside antibiotics. Western blot analysis showed 70% decrease in PBP2a expression, indicating the breakdown of beta-lactam resistance. Sanger sequence analysis confirmed point mutations in grlB and aacA genes, along with three-base mutations in grlA and mecA genes. Our findings underscore the potential of CRISPR/Cas9 technology to mitigate antibiotic resistance, offering a transformative strategy to restore the efficacy of existing antibiotics in the face of multidrug-resistant pathogens.\nTime-resolved transcriptomics reveal a mechanism of host niche defense: beneficial root endophytes deploy a host-protective antimicrobial GH18-CBM5 chitinase\nAuthors: Eichfeld, R.; Mahdi, L. K.; De Quattro, C.; Armbruster, L.; Endeshaw, A. B.; Miyauchi, S.; Hellmann, M. J.; Cord-Landwehr, S.; Grigoriev, I.; Peterson, D.; Singan, V.; Lail, K.; Savage, E.; Ng, V.; Langen, G.; Moerschbacher, B.; Zuccaro, A.\nScore: 10.8, Published: 2023-12-29 DOI: 10.1101/2023.12.29.572992\nAssociations between plants and beneficial root-endophytic fungi enhance plant performance by improving nutrient uptake, abiotic stress tolerance and disease resistance. To successfully colonize different host plants and defend their host niche against competing microbes, but also to cooperate with beneficial bacterial members of the microbiota, root endophytes such as Sebacinales secrete a multitude of tightly regulated effector-proteins and carbohydrate-active enzymes. However, the functions, specificity, and regulation of these proteins remain poorly understood. In this study, we employ time-resolved transcriptomics to analyse the gene expression profiles of two Sebacinales members interacting with organisms from different kingdoms of life. We identified crucial genes for plant colonization and intermicrobial competition, including a fungal GH18-CBM5 chitinase specifically upregulated in response to the phytopathogenic fungus Bipolaris sorokiniana. This chitinase protects the plant hosts against the pathogen, reducing fungal biomass and disease symptoms in barley and Arabidopsis thaliana. Our findings shed light on interaction partner specific gene expression in Sebacinales endophytes, with potential applications in enhancing plant health and resilience. Bullet pointsO_LIBoth Serendipita indica (Si) and Serendipita vermifera (Sv) show similar transcriptional responses to three host species and the phytopathogen Bipolaris sorokiniana (Bs), indicating common interaction principles between Sebacinales and plant hosts or fungi. C_LIO_LIThese shared mechanisms involve the activation of effector genes like small secreted proteins and carbohydrate-active enzymes. C_LIO_LICooperation with beneficial bacteria elicits only minimal transcriptomic alterations in Sebacinales compared to plants and Bs. C_LIO_LISebacinales respond to Bs by upregulating a specific GH18-CBM5 chitinase unique to Basidiomycota within the fungal kingdom, inhibiting Bs growth and reducing disease symptoms in Arabidopsis thaliana and barley. C_LI\nThe RNA-binding protein RbpB is a central regulator of polysaccharide utilization in gut Bacteroides\nAuthors: Ruttiger, A.-S.; Ryan, D.; Spiga, L.; Lamm-Schmidt, V.; Prezza, G.; Reichardt, S.; Barquist, L.; Faber, F.; Zhu, W.; Westermann, A. J.\nScore: 9.5, Published: 2023-12-22 DOI: 10.1101/2023.12.22.572801\nParamount to human health, symbiotic bacteria in the gastrointestinal tract rely on the breakdown of complex polysaccharides to thrive in this sugar-deprived environment. Gut Bacteroides are metabolic generalists and deploy dozens of polysaccharide utilization loci (PULs) to forage diverse dietary and host-derived glycans. The expression of the multi-protein PUL complexes is tightly regulated at the transcriptional level. However, how PULs are orchestrated at translational level in response to the fluctuating levels of their cognate substrates is unknown. Here, we identify the RNA-binding protein RbpB and a family of noncoding RNAs as key players in post-transcriptional PUL regulation. Ablation of RbpB in Bacteroides thetaiotaomicron displays compromised colonization in the mouse gut in a host diet-dependent manner. Current dogma holds that individual PULs are regulated by dedicated transcriptional regulators. We demonstrate that RbpB acts as a global RNA binder that directly interacts with several hundred cellular transcripts. This includes a paralogous noncoding RNA family comprised of 14 members, the FopS (family of paralogous sRNAs) cluster. Through a series of in-vitro and in-vivo assays, we reveal that FopS sRNAs repress the translation of a SusC-like glycan transporter when substrates are limited--an effect antagonized by RbpB. Together, this study implicates RNA-coordinated metabolic control as an important, yet previously overlooked, factor contributing to the in-vivo fitness of predominant microbiota species in dynamic nutrient landscapes.\nExtensive variation and strain-specificity in dengue virus susceptibility among African Aedes aegypti populations\nAuthors: Dabo, S.; Henrion-Lacritick, A.; Lecuyer, A.; Jiolle, D.; Paupy, C.; Ayala, D.; da Veiga Leal, S.; Badolo, A.; Vega-Rua, A.; Sylla, M.; Akorli, J.; Otoo, S.; Lutomiah, J.; Sang, R.; Mutebi, J.-P.; Saleh, M.-C.; Rose, N.; McBride, C.; Lambrechts, L.\nScore: 39.7, Published: 2023-12-14 DOI: 10.1101/2023.12.14.571617\nAfrican populations of the mosquito Aedes aegypti are usually considered less susceptible to infection by human-pathogenic flaviviruses than globally invasive populations found outside Africa. Although this contrast has been well documented for Zika virus (ZIKV), it is unclear to what extent it is true for dengue virus (DENV), the most prevalent flavivirus of humans. Addressing this question is complicated by substantial genetic diversity among DENV strains, most notably in the form of four genetic types (DENV1 to DENV4), that can lead to genetically specific interactions with mosquito populations. Here, we carried out a continent-wide survey of DENV susceptibility using a panel of field-derived Ae. aegypti colonies from across the African range of the species and a colony from Guadeloupe, French West Indies as non-African reference. We found considerable variation in the ability of African Ae. aegypti populations to acquire and replicate a panel of six DENV strains spanning the four DENV types. Although African Ae. aegypti populations were generally less susceptible than the reference non-African population from Guadeloupe, in several instances some African populations were equally or more susceptible than the Guadeloupe population. Moreover, the relative level of susceptibility between African mosquito populations depended on the DENV strain, indicating genetically specific interactions. We conclude that unlike ZIKV susceptibility, there is no clear-cut dichotomy in DENV susceptibility between African and non-African Ae. aegypti. DENV susceptibility of African Ae. aegypti populations is highly heterogeneous and largely governed by the specific pairing of mosquito population and DENV strain. Author summaryAfrican populations of the mosquito Aedes aegypti are usually thought to be less likely to get infected by flaviviruses compared to Ae. aegypti mosquitoes found outside Africa. While this has been well-demonstrated for Zika virus, it is not clear if the same is true for dengue virus, which is the most common flavivirus in humans. Studying this is complicated by the strain diversity of dengue virus, including four main genetic types, potentially causing different interactions. In this study, we compared several mosquito populations and found that, in general, African mosquitoes were less likely to get infected by dengue virus compared to mosquitoes from outside Africa. However, in some cases, African mosquitoes were just as or even more likely to get infected. The specific strain of dengue virus also influenced how likely African mosquitoes were to get infected, showing that the relationship between African mosquitoes and dengue virus is complex.\nQuantitative measurement of microbial growth rate with Raman microspectroscopy\nAuthors: Caro, T. A.; Kashyap, S.; Brown, G.; Chen, C.; Kopf, S.; Templeton, A.\nScore: 12.0, Published: 2023-12-17 DOI: 10.1101/2023.12.16.571966\nRates of microbial activity and growth are fundamental to understanding environmental geochemistry and ecology. However, measuring the heterogeneity of microbial activity at the single-cell level, especially within complex populations and environmental matrices, remains a forefront challenge. Stable Isotope Probing (SIP) is a standard method for assessing microbial activity and involves measuring the incorporation of an isotopically labeled compound into microbial biomass. Here, we assess the utility of Raman microspectroscopy as a SIP technique, specifically focusing on the measurement of deuterium (2H), a tracer of microbial biomass production. We generate calibrations of microbial biomass 2H values and find that Raman microspectroscopy reliably quantifies 2H incorporation ranging between 0 and 40 at. %. Applying the results of this calibration to a SIP model, we explicitly parameterize the factors controlling microbial growth quantification, demonstrating how Raman-SIP can measure the growth of microorganisms with doubling times ranging from hours to years. Furthermore, we correlatively compare our Raman-derived measurements with those of nanoscale secondary ion mass spectrometry (nanoSIMS) to compare the relative strengths of nanoSIMS- and Raman-based SIP approaches. We find that Raman microspectroscopy is a robust, accessible methodology that can readily differentiate and quantify the growth of individual microbial cells in complex samples.\nCyclo(Pro-Tyr) elicits conserved cellular damage in fungi by targeting the ATPase Pma1 in plasma membrane domains\nAuthors: Vela-Corcia, D.; Hierrezuelo, J.; Perez-Lorente, A. I.; Stincone, P.; Pakkir Shah, A. K.; Grelard, A.; de Vicente, A.; Perez-Garcia, A.; Loquet, A.; Petras, D.; Romero, D.\nScore: 6.0, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573620\nBioactive metabolites play a crucial role in shaping interactions among diverse organisms. In this study, we identified cyclo(Pro-Tyr), a metabolite produced by Bacillus velezensis, as a potent inhibitor of Botrytis cinerea and Caenorhabditis elegans, two potential cohabitant eukaryotic organisms. Based on our investigation, cyclo(Pro-Tyr) disrupts plasma membrane polarization, induces oxidative stress and increases membrane fluidity, which compromises fungal membrane integrity. These cytological and physiological changes induced by cyclo(Pro-Tyr) may be triggered by the destabilization of membrane microdomains containing the [H+]ATPase Pma1. In response to cyclo(Pro-Tyr) stress, fungal cells activate a transcriptomic and metabolomic response, which primarily involves lipid metabolism and Reactive Oxygen Species (ROS) detoxification, to mitigate membrane damage. This similar response occurs in the nematode C. elegans, suggesting that cyclo(Pro-Tyr) universally targets eukaryotic cellular membranes.\nVentilation does not affect influenza virus transmission efficiency in a ferret playpen setup\nAuthors: Rockey, N. C.; Le Sage, V.; Shephard, M.; Vargas-Maldonado, N.; French, A. J.; Walter, S.; Ferreri, L. M.; Holmes, K. E.; VanInsberghe, D.; Clack, H.; Prussin, A. J.; Lowen, A. C.; Marr, L.; Lakdawala, S.\nScore: 4.5, Published: 2023-12-26 DOI: 10.1101/2023.12.26.573248\nSustained community spread of influenza viruses relies on efficient person-to-person transmission. Current experimental transmission systems do not mimic environmental conditions (e.g., air exchange rates, flow patterns), host behaviors or exposure durations relevant to real-world settings. Therefore, results from these traditional systems may not be representative of influenza virus transmission in humans. To address this pitfall, we developed a modified, more realistic transmission setup and used it to investigate the impact of ventilation rates on transmission in a close-range, play-based scenario. In this setup, four immunologically naive recipient ferrets were exposed to a donor ferret infected with a genetically barcoded 2009 H1N1 virus (H1N1pdm09) for four hours. The ferrets interacted in a shared space that included toys, similar to a child care setting. The transmission efficiency was determined under conditions of low and high ventilation rates; air exchange rates of [~] 1.3 hr-1 and 23 hr-1, respectively. Despite the large difference in ventilation rate, transmission efficiency was the same, 50% in two independent replicate studies. The presence of infectious virus or viral RNA on surfaces and in air throughout the exposure area was similar regardless of ventilation rate. While high viral genetic diversity in donor ferret nasal washes was maintained during infection, recipient ferret nasal washes displayed low diversity, revealing a narrow transmission bottleneck regardless of ventilation rate. Our findings indicate that in exposures characterized by frequent close-range, play-based interactions and the presence of fomites, ventilation does not significantly impact transmission efficiency. SignificanceImproved ventilation in building has the potential to reduce transmission of respiratory viruses, but its effect in different settings is not well understood. We developed a novel system to study influenza virus transmission in the ferret animal model in an environment that mimics a child care center. We demonstrate that increased ventilation is not effective at disrupting transmission in this setting, suggesting that transmission occurs mainly at close-range or via fomites. Multiple interventions are needed to reduce the spread of influenza virus in this type of setting.\n",
  "wordCount" : "2694",
  "inLanguage": "en",
  "datePublished": "2024-01-03T10:39:43Z",
  "dateModified": "2024-01-03T10:39:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 3, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572655">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572655" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572655">
        <p class="paperTitle">Persistence of an infectious form of SARS-CoV-2 post protease inhibitor treatment of permissive cells in vitro</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572655" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572655" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nair, M. S.; Luck, M. I.; Huang, Y.; Sabo, Y.; Ho, D. D.</p>
        <p class="info">Score: 67.9, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572655' target='https://doi.org/10.1101/2023.12.20.572655'> 10.1101/2023.12.20.572655</a></p>
        <p class="abstract">Reports have described SARS-CoV-2 rebound in COVID-19 patients treated with nirmatrelvir, a 3CL protease inhibitor. The cause remains a mystery, although drug resistance, re-infection, and lack of adequate immune responses have been excluded. We now present virologic findings that provide a clue to the cause of viral rebound, which occurs in [~]20% of the treated cases. The persistence of an intermediary form of infectious SARS-CoV-2 was experimentally documented in vitro after treatment with nirmatrelvir or another 3CL protease inhibitor, but not with a polymerase inhibitor, remdesivir. This infectious intermediate decayed slowly with a half-life of [~]1 day, suggesting that its persistence could outlive the treatment course to re-ignited SARS-CoV-2 infection as the drug is eliminated. Additional studies are needed to define the nature of this viral intermediate, but our findings point to a particular direction for future investigation and offer a specific treatment recommendation that should be tested clinically.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.570782">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.570782" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.570782">
        <p class="paperTitle">Virological characteristics of the SARS-CoV-2 JN.1 variant</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.570782" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.570782" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium,  ; Zahradnik, J.; Ito, J.; Sato, K.</p>
        <p class="info">Score: 774.8, Published: 2023-12-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.570782' target='https://doi.org/10.1101/2023.12.08.570782'> 10.1101/2023.12.08.570782</a></p>
        <p class="abstract">The SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG.5.1 and HK.3. Comparing to XBB and BA.2, BA.2.86 carries more than 30 mutations in the spike (S) protein, indicating a high potential for immune evasion. BA.2.86 has evolved and its descendant, JN.1 (BA.2.86.1.1), emerged in late 2023. JN.1 harbors S:L455S and three mutations in non-S proteins. S:L455S is a hallmark mutation of JN.1: we have recently shown that HK.3 and other &#34;FLip&#34; variants carry S:L455F, which contributes to increased transmissibility and immune escape ability compared to the parental EG.5.1 variant. Here, we investigated the virological properties of JN.1.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572824">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572824" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572824">
        <p class="paperTitle">Virion morphology and on-virus spike protein structures of diverse SARS-CoV-2 variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572824" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572824" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ke, Z.; Peacock, T. P.; Brown, J. C.; Sheppard, C. M.; Croll, T. I.; Kotecha, A.; Goldhill, D. H.; Barclay, W. S.; Briggs, J. A.</p>
        <p class="info">Score: 27.2, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572824' target='https://doi.org/10.1101/2023.12.21.572824'> 10.1101/2023.12.21.572824</a></p>
        <p class="abstract">The evolution of SARS-CoV-2 variants with increased fitness has been accompanied by structural changes in the spike (S) proteins that are the major target for the adaptive immune response. Single-particle cryo-EM analysis of soluble S from SARS-CoV-2 variants has revealed this structural adaptation at high-resolution. The analysis of S trimers in situ on intact virions has the potential to provide more functionally relevant insights into S structure and virion morphology. Here, we characterized B.1, Alpha, Beta, Gamma, Delta, Kappa, and Mu variants by cryo-electron microscopy and tomography, assessing S cleavage, virion morphology, S incorporation, &#34;in-situ&#34; high-resolution S structures and the range of S conformational states. We found no evidence for adaptive changes in virion morphology, but describe multiple different positions in the S protein where amino acid changes alter local protein structure. Considered together, our data is consistent with a model where amino acid changes at multiple positions from the top to the base of the spike cause structural changes that can modulate the conformational dynamics of S.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.31.573511">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.31.573511" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.31.573511">
        <p class="paperTitle">Precision Genome Editing Unveils a Breakthrough in Reversing Antibiotic Resistance: CRISPR/Cas9 Targeting of Multi-Drug Resistance Genes in Methicillin-Resistant Staphylococcus aureus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.31.573511" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.31.573511" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ates, A.; Tastan, C.; Ermertcan, S.</p>
        <p class="info">Score: 11.1, Published: 2024-01-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.31.573511' target='https://doi.org/10.1101/2023.12.31.573511'> 10.1101/2023.12.31.573511</a></p>
        <p class="abstract">Antibiotic resistance poses a global health crisis, limiting the efficacy of available therapeutic agents and hindering the development of new antibiotics. The pharmaceutical industrys waning interest in antibiotic production further exacerbates this challenge. Addressing antibiotic resistance demands innovative solutions. Here, we explore the application of CRISPR-Cas-based antimicrobials as a pioneering approach to combat multidrug resistance. Our study focuses on the methicillin-resistant clinical Staphylococcus aureus (MRSA), a significant clinical threat. Utilizing CRISPR/Cas9 technology, we aimed to concurrently target methicillin (mecA), gentamicin (aacA), and ciprofloxacin (grlA, grlB) resistance genes, thereby altering the resistance profile and enhancing sensitivity to antibiotics. We engineered CRISPR plasmids containing sgRNAs specific to the target regions, which were subsequently electroporated into MRSA strains. Real-time Polymerase Chain Reaction (RT-PCR) assessed changes in resistance gene expression, while disk diffusion and broth microdilution methods determined alterations in resistance status. Western blotting analyzed changes in PBP2a expression, and Sanger sequence analysis confirmed mutations in target regions. Results revealed a statistically significant 1.5-fold decrease in mecA gene expression, 5.5-fold decrease in grlA gene, 6-fold decrease in grlB gene, and 4-fold decrease in aacA gene compared to the wild type strain, as determined by RT-PCR. Antibiotic susceptibility tests demonstrated the suppression of mecA, grlA, grlB, and aacA genes, resulting in the reversal of resistance to beta-lactam, quinolone, and aminoglycoside antibiotics. Western blot analysis showed 70% decrease in PBP2a expression, indicating the breakdown of beta-lactam resistance. Sanger sequence analysis confirmed point mutations in grlB and aacA genes, along with three-base mutations in grlA and mecA genes. Our findings underscore the potential of CRISPR/Cas9 technology to mitigate antibiotic resistance, offering a transformative strategy to restore the efficacy of existing antibiotics in the face of multidrug-resistant pathogens.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.572992">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.572992" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.572992">
        <p class="paperTitle">Time-resolved transcriptomics reveal a mechanism of host niche defense: beneficial root endophytes deploy a host-protective antimicrobial GH18-CBM5 chitinase</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.572992" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.572992" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eichfeld, R.; Mahdi, L. K.; De Quattro, C.; Armbruster, L.; Endeshaw, A. B.; Miyauchi, S.; Hellmann, M. J.; Cord-Landwehr, S.; Grigoriev, I.; Peterson, D.; Singan, V.; Lail, K.; Savage, E.; Ng, V.; Langen, G.; Moerschbacher, B.; Zuccaro, A.</p>
        <p class="info">Score: 10.8, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.572992' target='https://doi.org/10.1101/2023.12.29.572992'> 10.1101/2023.12.29.572992</a></p>
        <p class="abstract">Associations between plants and beneficial root-endophytic fungi enhance plant performance by improving nutrient uptake, abiotic stress tolerance and disease resistance. To successfully colonize different host plants and defend their host niche against competing microbes, but also to cooperate with beneficial bacterial members of the microbiota, root endophytes such as Sebacinales secrete a multitude of tightly regulated effector-proteins and carbohydrate-active enzymes. However, the functions, specificity, and regulation of these proteins remain poorly understood. In this study, we employ time-resolved transcriptomics to analyse the gene expression profiles of two Sebacinales members interacting with organisms from different kingdoms of life. We identified crucial genes for plant colonization and intermicrobial competition, including a fungal GH18-CBM5 chitinase specifically upregulated in response to the phytopathogenic fungus Bipolaris sorokiniana. This chitinase protects the plant hosts against the pathogen, reducing fungal biomass and disease symptoms in barley and Arabidopsis thaliana. Our findings shed light on interaction partner specific gene expression in Sebacinales endophytes, with potential applications in enhancing plant health and resilience.

Bullet pointsO_LIBoth Serendipita indica (Si) and Serendipita vermifera (Sv) show similar transcriptional responses to three host species and the phytopathogen Bipolaris sorokiniana (Bs), indicating common interaction principles between Sebacinales and plant hosts or fungi.
C_LIO_LIThese shared mechanisms involve the activation of effector genes like small secreted proteins and carbohydrate-active enzymes.
C_LIO_LICooperation with beneficial bacteria elicits only minimal transcriptomic alterations in Sebacinales compared to plants and Bs.
C_LIO_LISebacinales respond to Bs by upregulating a specific GH18-CBM5 chitinase unique to Basidiomycota within the fungal kingdom, inhibiting Bs growth and reducing disease symptoms in Arabidopsis thaliana and barley.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.572801">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.572801" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.572801">
        <p class="paperTitle">The RNA-binding protein RbpB is a central regulator of polysaccharide utilization in gut Bacteroides</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.572801" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.572801" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ruttiger, A.-S.; Ryan, D.; Spiga, L.; Lamm-Schmidt, V.; Prezza, G.; Reichardt, S.; Barquist, L.; Faber, F.; Zhu, W.; Westermann, A. J.</p>
        <p class="info">Score: 9.5, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.572801' target='https://doi.org/10.1101/2023.12.22.572801'> 10.1101/2023.12.22.572801</a></p>
        <p class="abstract">Paramount to human health, symbiotic bacteria in the gastrointestinal tract rely on the breakdown of complex polysaccharides to thrive in this sugar-deprived environment. Gut Bacteroides are metabolic generalists and deploy dozens of polysaccharide utilization loci (PULs) to forage diverse dietary and host-derived glycans. The expression of the multi-protein PUL complexes is tightly regulated at the transcriptional level. However, how PULs are orchestrated at translational level in response to the fluctuating levels of their cognate substrates is unknown. Here, we identify the RNA-binding protein RbpB and a family of noncoding RNAs as key players in post-transcriptional PUL regulation. Ablation of RbpB in Bacteroides thetaiotaomicron displays compromised colonization in the mouse gut in a host diet-dependent manner. Current dogma holds that individual PULs are regulated by dedicated transcriptional regulators. We demonstrate that RbpB acts as a global RNA binder that directly interacts with several hundred cellular transcripts. This includes a paralogous noncoding RNA family comprised of 14 members, the FopS (family of paralogous sRNAs) cluster. Through a series of in-vitro and in-vivo assays, we reveal that FopS sRNAs repress the translation of a SusC-like glycan transporter when substrates are limited--an effect antagonized by RbpB. Together, this study implicates RNA-coordinated metabolic control as an important, yet previously overlooked, factor contributing to the in-vivo fitness of predominant microbiota species in dynamic nutrient landscapes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.571617">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.571617" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.571617">
        <p class="paperTitle">Extensive variation and strain-specificity in dengue virus susceptibility among African Aedes aegypti populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.571617" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.571617" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dabo, S.; Henrion-Lacritick, A.; Lecuyer, A.; Jiolle, D.; Paupy, C.; Ayala, D.; da Veiga Leal, S.; Badolo, A.; Vega-Rua, A.; Sylla, M.; Akorli, J.; Otoo, S.; Lutomiah, J.; Sang, R.; Mutebi, J.-P.; Saleh, M.-C.; Rose, N.; McBride, C.; Lambrechts, L.</p>
        <p class="info">Score: 39.7, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.571617' target='https://doi.org/10.1101/2023.12.14.571617'> 10.1101/2023.12.14.571617</a></p>
        <p class="abstract">African populations of the mosquito Aedes aegypti are usually considered less susceptible to infection by human-pathogenic flaviviruses than globally invasive populations found outside Africa. Although this contrast has been well documented for Zika virus (ZIKV), it is unclear to what extent it is true for dengue virus (DENV), the most prevalent flavivirus of humans. Addressing this question is complicated by substantial genetic diversity among DENV strains, most notably in the form of four genetic types (DENV1 to DENV4), that can lead to genetically specific interactions with mosquito populations. Here, we carried out a continent-wide survey of DENV susceptibility using a panel of field-derived Ae. aegypti colonies from across the African range of the species and a colony from Guadeloupe, French West Indies as non-African reference. We found considerable variation in the ability of African Ae. aegypti populations to acquire and replicate a panel of six DENV strains spanning the four DENV types. Although African Ae. aegypti populations were generally less susceptible than the reference non-African population from Guadeloupe, in several instances some African populations were equally or more susceptible than the Guadeloupe population. Moreover, the relative level of susceptibility between African mosquito populations depended on the DENV strain, indicating genetically specific interactions. We conclude that unlike ZIKV susceptibility, there is no clear-cut dichotomy in DENV susceptibility between African and non-African Ae. aegypti. DENV susceptibility of African Ae. aegypti populations is highly heterogeneous and largely governed by the specific pairing of mosquito population and DENV strain.

Author summaryAfrican populations of the mosquito Aedes aegypti are usually thought to be less likely to get infected by flaviviruses compared to Ae. aegypti mosquitoes found outside Africa. While this has been well-demonstrated for Zika virus, it is not clear if the same is true for dengue virus, which is the most common flavivirus in humans. Studying this is complicated by the strain diversity of dengue virus, including four main genetic types, potentially causing different interactions. In this study, we compared several mosquito populations and found that, in general, African mosquitoes were less likely to get infected by dengue virus compared to mosquitoes from outside Africa. However, in some cases, African mosquitoes were just as or even more likely to get infected. The specific strain of dengue virus also influenced how likely African mosquitoes were to get infected, showing that the relationship between African mosquitoes and dengue virus is complex.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.16.571966">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.16.571966" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.16.571966">
        <p class="paperTitle">Quantitative measurement of microbial growth rate with Raman microspectroscopy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.16.571966" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.16.571966" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Caro, T. A.; Kashyap, S.; Brown, G.; Chen, C.; Kopf, S.; Templeton, A.</p>
        <p class="info">Score: 12.0, Published: 2023-12-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.16.571966' target='https://doi.org/10.1101/2023.12.16.571966'> 10.1101/2023.12.16.571966</a></p>
        <p class="abstract">Rates of microbial activity and growth are fundamental to understanding environmental geochemistry and ecology. However, measuring the heterogeneity of microbial activity at the single-cell level, especially within complex populations and environmental matrices, remains a forefront challenge. Stable Isotope Probing (SIP) is a standard method for assessing microbial activity and involves measuring the incorporation of an isotopically labeled compound into microbial biomass. Here, we assess the utility of Raman microspectroscopy as a SIP technique, specifically focusing on the measurement of deuterium (2H), a tracer of microbial biomass production. We generate calibrations of microbial biomass 2H values and find that Raman microspectroscopy reliably quantifies 2H incorporation ranging between 0 and 40 at. %. Applying the results of this calibration to a SIP model, we explicitly parameterize the factors controlling microbial growth quantification, demonstrating how Raman-SIP can measure the growth of microorganisms with doubling times ranging from hours to years. Furthermore, we correlatively compare our Raman-derived measurements with those of nanoscale secondary ion mass spectrometry (nanoSIMS) to compare the relative strengths of nanoSIMS- and Raman-based SIP approaches. We find that Raman microspectroscopy is a robust, accessible methodology that can readily differentiate and quantify the growth of individual microbial cells in complex samples.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.573620">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.573620" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.573620">
        <p class="paperTitle">Cyclo(Pro-Tyr) elicits conserved cellular damage in fungi by targeting the ATPase Pma1 in plasma membrane domains</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.573620" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.573620" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vela-Corcia, D.; Hierrezuelo, J.; Perez-Lorente, A. I.; Stincone, P.; Pakkir Shah, A. K.; Grelard, A.; de Vicente, A.; Perez-Garcia, A.; Loquet, A.; Petras, D.; Romero, D.</p>
        <p class="info">Score: 6.0, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.573620' target='https://doi.org/10.1101/2023.12.29.573620'> 10.1101/2023.12.29.573620</a></p>
        <p class="abstract">Bioactive metabolites play a crucial role in shaping interactions among diverse organisms. In this study, we identified cyclo(Pro-Tyr), a metabolite produced by Bacillus velezensis, as a potent inhibitor of Botrytis cinerea and Caenorhabditis elegans, two potential cohabitant eukaryotic organisms. Based on our investigation, cyclo(Pro-Tyr) disrupts plasma membrane polarization, induces oxidative stress and increases membrane fluidity, which compromises fungal membrane integrity. These cytological and physiological changes induced by cyclo(Pro-Tyr) may be triggered by the destabilization of membrane microdomains containing the [H&#43;]ATPase Pma1. In response to cyclo(Pro-Tyr) stress, fungal cells activate a transcriptomic and metabolomic response, which primarily involves lipid metabolism and Reactive Oxygen Species (ROS) detoxification, to mitigate membrane damage. This similar response occurs in the nematode C. elegans, suggesting that cyclo(Pro-Tyr) universally targets eukaryotic cellular membranes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.26.573248">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.26.573248" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.26.573248">
        <p class="paperTitle">Ventilation does not affect influenza virus transmission efficiency in a ferret playpen setup</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.26.573248" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.26.573248" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rockey, N. C.; Le Sage, V.; Shephard, M.; Vargas-Maldonado, N.; French, A. J.; Walter, S.; Ferreri, L. M.; Holmes, K. E.; VanInsberghe, D.; Clack, H.; Prussin, A. J.; Lowen, A. C.; Marr, L.; Lakdawala, S.</p>
        <p class="info">Score: 4.5, Published: 2023-12-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.26.573248' target='https://doi.org/10.1101/2023.12.26.573248'> 10.1101/2023.12.26.573248</a></p>
        <p class="abstract">Sustained community spread of influenza viruses relies on efficient person-to-person transmission. Current experimental transmission systems do not mimic environmental conditions (e.g., air exchange rates, flow patterns), host behaviors or exposure durations relevant to real-world settings. Therefore, results from these traditional systems may not be representative of influenza virus transmission in humans. To address this pitfall, we developed a modified, more realistic transmission setup and used it to investigate the impact of ventilation rates on transmission in a close-range, play-based scenario. In this setup, four immunologically naive recipient ferrets were exposed to a donor ferret infected with a genetically barcoded 2009 H1N1 virus (H1N1pdm09) for four hours. The ferrets interacted in a shared space that included toys, similar to a child care setting. The transmission efficiency was determined under conditions of low and high ventilation rates; air exchange rates of [~] 1.3 hr-1 and 23 hr-1, respectively. Despite the large difference in ventilation rate, transmission efficiency was the same, 50% in two independent replicate studies. The presence of infectious virus or viral RNA on surfaces and in air throughout the exposure area was similar regardless of ventilation rate. While high viral genetic diversity in donor ferret nasal washes was maintained during infection, recipient ferret nasal washes displayed low diversity, revealing a narrow transmission bottleneck regardless of ventilation rate. Our findings indicate that in exposures characterized by frequent close-range, play-based interactions and the presence of fomites, ventilation does not significantly impact transmission efficiency.

SignificanceImproved ventilation in building has the potential to reduce transmission of respiratory viruses, but its effect in different settings is not well understood. We developed a novel system to study influenza virus transmission in the ferret animal model in an environment that mimics a child care center. We demonstrate that increased ventilation is not effective at disrupting transmission in this setting, suggesting that transmission occurs mainly at close-range or via fomites. Multiple interventions are needed to reduce the spread of influenza virus in this type of setting.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
