<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Bacterial cGAS-like enzymes produce 2&#39;,3&#39;-cGAMP to activate an ion channel that restricts phage replication
Authors: Tak, U.; Walth, P.; Whiteley, A. T.
Score: 43.7, Published: 2023-07-24 DOI: 10.1101/2023.07.24.550367
The mammalian innate immune system uses cyclic GMP-AMP synthase (cGAS) to synthesize the cyclic dinucleotide 2&#39;,3&#39;-cGAMP during antiviral and antitumor immune responses. 2&#39;,3&#39;-cGAMP is a nucleotide second messenger that initiates inflammatory signaling by binding to and activating the stimulator of interferon genes (STING) receptor.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Bacterial cGAS-like enzymes produce 2&#39;,3&#39;-cGAMP to activate an ion channel that restricts phage replication
Authors: Tak, U.; Walth, P.; Whiteley, A. T.
Score: 43.7, Published: 2023-07-24 DOI: 10.1101/2023.07.24.550367
The mammalian innate immune system uses cyclic GMP-AMP synthase (cGAS) to synthesize the cyclic dinucleotide 2&#39;,3&#39;-cGAMP during antiviral and antitumor immune responses. 2&#39;,3&#39;-cGAMP is a nucleotide second messenger that initiates inflammatory signaling by binding to and activating the stimulator of interferon genes (STING) receptor." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-30T10:36:57+00:00" />
<meta property="article:modified_time" content="2023-07-30T10:36:57+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Bacterial cGAS-like enzymes produce 2&#39;,3&#39;-cGAMP to activate an ion channel that restricts phage replication
Authors: Tak, U.; Walth, P.; Whiteley, A. T.
Score: 43.7, Published: 2023-07-24 DOI: 10.1101/2023.07.24.550367
The mammalian innate immune system uses cyclic GMP-AMP synthase (cGAS) to synthesize the cyclic dinucleotide 2&#39;,3&#39;-cGAMP during antiviral and antitumor immune responses. 2&#39;,3&#39;-cGAMP is a nucleotide second messenger that initiates inflammatory signaling by binding to and activating the stimulator of interferon genes (STING) receptor."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Bacterial cGAS-like enzymes produce 2\u0026#39;,3\u0026#39;-cGAMP to activate an ion channel that restricts phage replication\nAuthors: Tak, U.; Walth, P.; Whiteley, A. T.\nScore: 43.7, Published: 2023-07-24 DOI: 10.1101/2023.07.24.550367\nThe mammalian innate immune system uses cyclic GMP-AMP synthase (cGAS) to synthesize the cyclic dinucleotide 2\u0026#39;,3\u0026#39;-cGAMP during antiviral and antitumor immune responses. 2\u0026#39;,3\u0026#39;-cGAMP is a nucleotide second messenger that initiates inflammatory signaling by binding to and activating the stimulator of interferon genes (STING) receptor.",
  "keywords": [
    
  ],
  "articleBody": " Bacterial cGAS-like enzymes produce 2',3'-cGAMP to activate an ion channel that restricts phage replication\nAuthors: Tak, U.; Walth, P.; Whiteley, A. T.\nScore: 43.7, Published: 2023-07-24 DOI: 10.1101/2023.07.24.550367\nThe mammalian innate immune system uses cyclic GMP-AMP synthase (cGAS) to synthesize the cyclic dinucleotide 2',3'-cGAMP during antiviral and antitumor immune responses. 2',3'-cGAMP is a nucleotide second messenger that initiates inflammatory signaling by binding to and activating the stimulator of interferon genes (STING) receptor. Bacteria also encode cGAS/DncV-like nucleotidyltransferases (CD-NTases) that produce nucleotide second messengers to initiate antiviral (antiphage) signaling. Bacterial CD-NTases produce a wide range of cyclic oligonucleotides but have not been documented to produce 2',3'-cGAMP. Here we discovered bacterial CD-NTases that produce 2',3'-cGAMP to restrict phage replication. Bacterial 2',3'-cGAMP binds to CD-NTase associated protein 14 (Cap14), a transmembrane protein of unknown function. Using electrophysiology, we show that Cap14 is a chloride-selective ion channel that is activated by 2',3'-cGAMP binding. Cap14 adopts a modular architecture, with an N-terminal transmembrane domain and a C-terminal nucleotide-binding SAVED domain. Domain-swapping experiments demonstrated the Cap14 transmembrane region could be substituted with a nuclease, thereby generating a biosensor that is selective for 2',3'-cGAMP. This study reveals that 2',3'-cGAMP signaling extends beyond metazoa to bacteria. Further, our findings suggest that transmembrane proteins of unknown function in bacterial immune pathways may broadly function as nucleotide-gated ion channels.\nGenomic epidemiology of European Aspergillus fumigatus causing COVID-19-associated pulmonary aspergillosis in Europe\nAuthors: Simmons, B.; Rhodes, J.; Rogers, T.; Talento, A. F.; Griffin, A.; Mansfield, M.; Abdolrasouli, A.; Verweij, P. E.; Bosch, T.; Schelenz, S.; Hemmings, S.; Sheehan, D.; Fisher, M. C.\nScore: 35.1, Published: 2023-07-24 DOI: 10.1101/2023.07.21.550109\nThe opportunistic fungus Aspergillus fumigatus has been found to cause coinfections in patients with severe SARS-CoV-2 virus infection, leading to COVID-19-associated pulmonary aspergillosis (CAPA). The CAPA all-cause mortality rate is approximately 50% and may be complicated by azole-resistance. Genomic epidemiology can help shed light on the genetics of A. fumigatus causing CAPA including the prevalence of alleles that are associated with azole-resistance. Here, a population genomic analysis of 21 CAPA isolates from four European countries is presented. The CAPA isolates were compared with A. fumigatus from a wider population of 167 non-CAPA clinical isolates and 73 environmental isolates. Bioinformatic analysis and antifungal susceptibility testing were performed to quantify resistance and identify possible genetically-encoded azole-resistant mechanisms. Phylogenetic analysis of the 21 CAPA isolates showed a lack of genetic distinction from the wider A. fumigatus population, with isolates distributed within two distinct clades (A and B), with the majority of the CAPA isolates in clade B (71.4%). The prevalence of phenotypic azole-resistance in CAPA was 14.3% (n=3/21); all three CAPA isolates contained a known resistance-associated cyp51A polymorphism. CAPA isolates are drawn from the wider A. fumigatus population rather than forming a unique genetic background showing that COVID-19 patients are susceptible to the entire A. fumigatus population. However, the relatively high prevalence of azole-resistance alleles that we document poses a threat to treatment success rates, warranting enhanced detection and surveillance of A. fumigatus genotypes in these patients. Furthermore, potential changes to antifungal first-line treatment guidelines may be needed to improve patient outcomes.\nA Taxonomically-informed Mass Spectrometry Search Tool for Microbial Metabolomics Data\nAuthors: Zuffa, S.; Schmid, R.; Bauermeister, A.; P. Gomes, P. W.; Caraballo-Rodriguez, A. M.; El Abiead, Y.; Aron, A. T.; Gentry, E. C.; Zemlin, J.; Meehan, M. J.; Avalon, N. E.; Cichewicz, R. H.; Buzun, E.; Carrillo Terrazas, M.; Hsu, C.-Y.; Oles, R.; Vasquez Ayala, A.; Zhao, J.; Chu, H.; Kuijpers, M. C. M.; Jackrel, S. L.; Tugizimana, F.; Nephali, L. P.; Dubery, I. A.; Madala, N. E.; Moreira, E. A.; Costa-Lotufo, L. V.; Lopes, N. P.; Rezende-Teixeira, P.; Jimenez, P. C.; Rimal, B.; Patterson, A. D.; Traxler, M. F.; Pessotti, R. d. C.; Alvarado-Villalobos, D.; Tamayo-Castillo, G.; Chaverri, P.; Escud\nScore: 39.0, Published: 2023-07-20 DOI: 10.1101/2023.07.20.549584\nMicrobeMASST, a taxonomically-informed mass spectrometry (MS) search tool, tackles limited microbial metabolite annotation in untargeted metabolomics experiments. Leveraging a curated database of \u003e60,000 microbial monocultures, users can search known and unknown MS/MS spectra and link them to their respective microbial producers via MS/MS fragmentation patterns. Identification of microbial-derived metabolites and relative producers, without a priori knowledge, will vastly enhance the understanding of microorganisms role in ecology and human health.\nEvolution and spread of a persistent P. aeruginosa outbreak clone over decades in a single US hospital.\nAuthors: Stribling, W.; Hall, L. R.; Powell, A.; Harless, C.; Martin, M. J.; Corey, B. W.; Snesrud, E.; Ong, A.; Maybank, R.; Stam, J.; Bartlett, K.; Jones, B. T.; Preston, L. N.; Lane, K. F.; Thompson, B.; Young, L. M.; Kwak, Y. I.; Barsoumian, A. E.; Markelz, A.-E.; Kiley, J. L.; Cybulski, R. J.; Bennett, J. W.; Mc Gann, P. T.; Lebreton, F.\nScore: 22.3, Published: 2023-07-25 DOI: 10.1101/2023.07.24.550326\nWhole-genome sequencing is revolutionizing outbreak investigation. In 2020, prospective and retrospective surveillance detected a P. aeruginosa outbreak with 254 isolates collected from 82 patients in 27 wards of a hospital. Its origin was dated to the late 90s, just after the facility opened, and patient-to-patient and environment-to-patient cases of transmission were inferred. Over time, two epidemic subclones evolved in separate hosts and hospital areas, including newly opened wards, and hospital-wide sampling confirmed reservoirs persisted in the plumbing. Pathoadaptive mutations in genes associated with virulence, cell wall biogenesis, and antibiotic resistance were identified. While the latter correlated with the acquisition of phenotypic resistances to 1st (cephalosporin), 2nd (carbapenems) and 3rd (colistin) lines of treatment, maximum parsimony suggested that a truncation in a lipopolysaccharide component coincided with the emergence of a subclone prevalent in chronic infections. Since initial identification, extensive infection control efforts have proved successful at slowing transmission.\nFirst Eurasian cases of SARS-CoV-2 seropositivity in a free-ranging urban population of wild fallow deer\nAuthors: Purves, K.; Brown, H.; Haverty, R.; Ryan, A.; Griffin, L.; McCormack, J.; O'Reilly, S. R.; Mallon, P. W.; Gautier, V.; Cassidy, J. P.; Fabre, A.; Carr, M. J.; Gonzalez, G.; Ciuti, S.; Fletcher, N.\nScore: 76.9, Published: 2023-07-07 DOI: 10.1101/2023.07.07.547941\nSevere acute respiratory syndrome coronavirus-2 (SARS-CoV-2) infects wildlife. Recent studies highlighted that variants of concern (VOC) may expand into novel animal reservoirs with the potential for reverse zoonosis. North American white-tailed deer are the only deer species in which SARS-CoV-2 has been documented, raising the question whether further reservoir species exist as new VOC emerge. Here, we report the first cases of deer SARS-CoV-2 seropositivity in Eurasia, in a city population of fallow deer in Dublin, Ireland. Deer were seronegative in 2020 (circulating variant in humans: Alpha), one animal was seropositive in 2021 (Delta variant), and 57% of animals tested in 2022 were seropositive (Omicron variant). Ex vivo, a clinical isolate of Omicron BA.1 infected fallow deer precision cut lung slice type-2 pneumocytes, also a major target of infection in human lungs. Our findings suggest a change in host tropism as new variants emerged in the human reservoir, highlighting the importance of continued wildlife disease monitoring and limiting human-wildlife contacts. Teaser: This study is the first report of SARS-CoV-2 seropositivity in fallow deer, highlighting expansion of viral variants into new host reservoirs.\nMechanisms of extracellular electron transfer in anaerobic methanotrophic archaea\nAuthors: Ouboter, H. T.; Mesman, R.; Sleutels, T.; Postma, J.; Wissink, M.; Jetten, M. S. M.; ter Heijne, A.; Berben, T.; Welte, C.\nScore: 13.5, Published: 2023-07-24 DOI: 10.1101/2023.07.24.550278\nAnaerobic methanotrophic (ANME) archaea are environmentally important uncultivated microorganisms mitigating the release of the potent greenhouse gas methane. During methane oxidation ANME archaea engage in extracellular electron transfer (EET) with other microorganisms, metal oxides, and electrodes, through a currently unknown mechanism. To shed light on this mechanism, we cultivated ANME-2d archaea (Ca. Methanoperedens) in bioelectrochemical systems and observed strong methane-dependent current (91-93% of total current) associated with high enrichment of Ca. Methanoperedens on the anode (up to 82% of the community) determined by metagenomics and transmission electron microscopy. Electrochemistry and metatranscriptomics indicated that the EET mechanism was similar at various electrode potentials pointing to the involvement of an so far uncharacterized short-range electron transport protein complex and OmcZ nanowires, suggesting a unique EET pathway in all ANME-2 archaea. Our findings furthermore indicate that bioelectrochemical cells might be powerful tools for the cultivation, and possibly isolation, of uncultured electroactive microorganisms.\nCommunity structure of known and previously unknown endobacteria associated with spores of arbuscular mycorrhizal fungi.\nAuthors: Lastovetsky, O. A.; Caruso, T.; Brennan, F.; Wall, D.; Pylni, S.; Doyle, E.\nScore: 12.8, Published: 2023-07-25 DOI: 10.1101/2023.07.25.550273\nArbuscular mycorrhizal fungi (AMF) are ubiquitous plant root symbionts which can house two endobacteria: Ca. Moeniiplasma glomeromycotorum (CaMg) and Ca. Glomeribacter gigasporarum (CaGg). However, little is known about their distribution and population structure in natural AMF populations and whether AMF can harbour other endobacteria. We isolated AMF from two environments and surveyed the surface-sterilized spores for endobacteria. We found that CaMg and CaGg differed significantly in distribution whereby CaMg were extremely abundant (80%) and CaGg were extremely rare (2%) in both environments. Unexpectedly, we discovered an additional and previously unknown level of bacterial diversity within AMF spores which extended beyond the known endosymbionts, with as many as 277 other bacterial taxa detected in individual spores. Detailed analysis of endobacterial communities inside AMF spores revealed that: (i)CaGg were not limited in distribution to the Gigasporacea family of AMF, as previously thought, (ii) CaMg community structure was driven by AMF host genotype, (iii) a significant inverse correlation existed between the diversity of CaMg and diversity of all other endobacteria. The latter suggests the existence of competition dynamics between different bacterial populations inside AMF spores and provides a basis for generation of testable hypotheses regarding the function of CaMg in AMF biology.\nDiversity and biotechnological potential of filamentous fungi isolated from sediments of Basque estuaries\nAuthors: Otamendi, A.; Agirrezabala, Z.; Perez-Cruz, C.; Liebana, R.; Berregi, I.; Alonso-Saez, L.; Duenas, M. T.; Lanzen, A.; Etxebeste, O.\nScore: 10.5, Published: 2023-07-28 DOI: 10.1101/2023.07.26.550687\nMarine environments harbor a vast diversity of microorganisms, which have developed multiple strategies to adapt to stress conditions such as high salt concentrations and nutrient scarcity and, thus, are a source of natural products and high-value enzymes, as they have the ability to degrade complex polymeric substrates. These features make marine microorganisms a valuable source for the development of new biotechnological tools. Here, we focused on the isolation of filamentous fungi, using sediment samples collected from estuaries of the Basque Country in the Bay of Biscay. The phenotype of these isolates was characterized based on the potential to grow on complex marine polysaccharides or to secrete pigments to the culture medium. Based on this screening, the genomes of two isolates of the order Hypocreales, Marquandomyces marquandii and Albophoma yamanashiensis, were sequenced and are described here for the first time. The analysis of their CAZYme (carbohydrate-active enzyme) and secondary metabolite gene-cluster repertoires suggest that these estuarine isolates could be used as a source of novel enzymatic activities and secondary metabolites.\nChromosome organization shapes replisome dynamics in Caulobacter crescentus\nAuthors: Zhang, C.; Joseph, A. M.; Casini, L.; Collier, J.; Badrinarayanan, A.; Manley, S.\nScore: 10.5, Published: 2023-07-22 DOI: 10.1101/2023.07.22.550130\nDNA replication in bacteria takes place on highly compacted chromosomes, where segregation, transcription, and repair must occur simultaneously. Within this dynamic environment, colocalization of sister replisomes has been observed in many bacterial species, driving the hypothesis that a physical linker may tether them together. However, replisome splitting has also been reported in many of the same species, leaving the principles behind replisome organization a long-standing puzzle. Here, by tracking the replisome {beta}-clamp subunit in live Caulobacter crescentus, we find that rapid DNA segregation can give rise to a second focus which resembles a replisome, but does not replicate DNA. Sister replisomes can remain colocalized, or split apart to travel along DNA separately upon disruption of chromosome inter-arm alignment. Furthermore, chromosome arm-specific replication-transcription conflicts differentially modify replication speed on the two arms, facilitate the decoupling of the two replisomes. With these observations, we conclude that the dynamic chromosome organization flexibly shapes the organization of sister replisomes, and we outline principles which can help to reconcile previously conflicting models of replisome architecture.\nExperimentally Evolved Staphylococcus aureus Survives in the Presence of Pseudomonas aeruginosa by Acquiring Mutations in the Amino Acid Transporter, GltT\nAuthors: Alexander, A. M.; Luu, J. M.; Raghuram, V.; Bottacin, G.; van Vliet, S.; Read, T. D.; Goldberg, J. B.\nScore: 9.5, Published: 2023-07-24 DOI: 10.1101/2023.07.24.550428\nStaphylococcus aureus and Pseudomonas aeruginosa are the most common bacterial pathogens isolated from cystic fibrosis (CF) related lung infections. When both of these opportunistic pathogens are found in a coinfection, CF patients tend to have higher rates of pulmonary exacerbations and experience a more rapid decrease in lung function. When cultured together under standard laboratory conditions, it is often observed that P. aeruginosa effectively inhibits S. aureus growth. Previous work from our group revealed that S. aureus from CF infections have isolate-specific survival capabilities when cocultured with P. aeruginosa. In this study, we designed a serial transfer evolution experiment to identify mutations that allow S. aureus to adapt to the presence of P. aeruginosa. Using S. aureus USA300 JE2 as our ancestral strain, populations of S. aureus were repeatedly cocultured with fresh P. aeruginosa strain, PAO1. After 8 coculture periods, S. aureus populations that survived better in the presence of PAO1 were observed. We found two independent mutations in the highly conserved S. aureus aspartate transporter, gltT, that were unique to evolved P. aeruginosa-tolerant isolates. Subsequent phenotypic testing demonstrated that gltT mutants have reduced uptake of glutamate and outcompete wild-type S. aureus when glutamate is absent from chemically-defined media. These findings together demonstrate that the presence of P. aeruginosa exerts selective pressure on S. aureus to alter its uptake and metabolism of key amino acids when the two bacteria are cultured together. ImportanceStaphylococcus aureus and Pseudomonas aeruginosa are the two most common bacterial pathogens that infect people with the genetic disease, cystic fibrosis (CF). They are often found together in CF-associated polymicrobial infections that are associated with worse patient prognosis. Understanding how these very different opportunistic pathogens influence each other in a shared environment is pertinent to improving the treatment of polymicrobial infections. While much attention has been brought to the interspecific interactions between S. aureus and P. aeruginosa, few studies have used experimental evolution methods to identify determinants of their competition and coexistence. Here, we use a serial transfer experimental evolution approach and identified a single genetic change associated with improved survival of S. aureus in the presence of P. aeruginosa. Our findings implicate metabolism of shared resources as an important factor in S. aureuss ability to survive in the presence of P. aeruginosa.\n",
  "wordCount" : "2421",
  "inLanguage": "en",
  "datePublished": "2023-07-30T10:36:57Z",
  "dateModified": "2023-07-30T10:36:57Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta"><span>updated on July 30, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550367">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550367" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550367">
        <p class="paperTitle">Bacterial cGAS-like enzymes produce 2&#39;,3&#39;-cGAMP to activate an ion channel that restricts phage replication</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550367" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550367" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tak, U.; Walth, P.; Whiteley, A. T.</p>
        <p class="info">Score: 43.7, Published: 2023-07-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550367' target='https://doi.org/10.1101/2023.07.24.550367'> 10.1101/2023.07.24.550367</a></p>
        <p class="abstract">The mammalian innate immune system uses cyclic GMP-AMP synthase (cGAS) to synthesize the cyclic dinucleotide 2&#39;,3&#39;-cGAMP during antiviral and antitumor immune responses. 2&#39;,3&#39;-cGAMP is a nucleotide second messenger that initiates inflammatory signaling by binding to and activating the stimulator of interferon genes (STING) receptor. Bacteria also encode cGAS/DncV-like nucleotidyltransferases (CD-NTases) that produce nucleotide second messengers to initiate antiviral (antiphage) signaling. Bacterial CD-NTases produce a wide range of cyclic oligonucleotides but have not been documented to produce 2&#39;,3&#39;-cGAMP. Here we discovered bacterial CD-NTases that produce 2&#39;,3&#39;-cGAMP to restrict phage replication. Bacterial 2&#39;,3&#39;-cGAMP binds to CD-NTase associated protein 14 (Cap14), a transmembrane protein of unknown function. Using electrophysiology, we show that Cap14 is a chloride-selective ion channel that is activated by 2&#39;,3&#39;-cGAMP binding. Cap14 adopts a modular architecture, with an N-terminal transmembrane domain and a C-terminal nucleotide-binding SAVED domain. Domain-swapping experiments demonstrated the Cap14 transmembrane region could be substituted with a nuclease, thereby generating a biosensor that is selective for 2&#39;,3&#39;-cGAMP. This study reveals that 2&#39;,3&#39;-cGAMP signaling extends beyond metazoa to bacteria. Further, our findings suggest that transmembrane proteins of unknown function in bacterial immune pathways may broadly function as nucleotide-gated ion channels.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.550109">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.550109" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.550109">
        <p class="paperTitle">Genomic epidemiology of European Aspergillus fumigatus causing COVID-19-associated pulmonary aspergillosis in Europe</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.550109" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.550109" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Simmons, B.; Rhodes, J.; Rogers, T.; Talento, A. F.; Griffin, A.; Mansfield, M.; Abdolrasouli, A.; Verweij, P. E.; Bosch, T.; Schelenz, S.; Hemmings, S.; Sheehan, D.; Fisher, M. C.</p>
        <p class="info">Score: 35.1, Published: 2023-07-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.550109' target='https://doi.org/10.1101/2023.07.21.550109'> 10.1101/2023.07.21.550109</a></p>
        <p class="abstract">The opportunistic fungus Aspergillus fumigatus has been found to cause coinfections in patients with severe SARS-CoV-2 virus infection, leading to COVID-19-associated pulmonary aspergillosis (CAPA). The CAPA all-cause mortality rate is approximately 50% and may be complicated by azole-resistance. Genomic epidemiology can help shed light on the genetics of A. fumigatus causing CAPA including the prevalence of alleles that are associated with azole-resistance. Here, a population genomic analysis of 21 CAPA isolates from four European countries is presented. The CAPA isolates were compared with A. fumigatus from a wider population of 167 non-CAPA clinical isolates and 73 environmental isolates. Bioinformatic analysis and antifungal susceptibility testing were performed to quantify resistance and identify possible genetically-encoded azole-resistant mechanisms. Phylogenetic analysis of the 21 CAPA isolates showed a lack of genetic distinction from the wider A. fumigatus population, with isolates distributed within two distinct clades (A and B), with the majority of the CAPA isolates in clade B (71.4%). The prevalence of phenotypic azole-resistance in CAPA was 14.3% (n=3/21); all three CAPA isolates contained a known resistance-associated cyp51A polymorphism. CAPA isolates are drawn from the wider A. fumigatus population rather than forming a unique genetic background showing that COVID-19 patients are susceptible to the entire A. fumigatus population. However, the relatively high prevalence of azole-resistance alleles that we document poses a threat to treatment success rates, warranting enhanced detection and surveillance of A. fumigatus genotypes in these patients. Furthermore, potential changes to antifungal first-line treatment guidelines may be needed to improve patient outcomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549584">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549584" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549584">
        <p class="paperTitle">A Taxonomically-informed Mass Spectrometry Search Tool for Microbial Metabolomics Data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549584" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549584" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zuffa, S.; Schmid, R.; Bauermeister, A.; P. Gomes, P. W.; Caraballo-Rodriguez, A. M.; El Abiead, Y.; Aron, A. T.; Gentry, E. C.; Zemlin, J.; Meehan, M. J.; Avalon, N. E.; Cichewicz, R. H.; Buzun, E.; Carrillo Terrazas, M.; Hsu, C.-Y.; Oles, R.; Vasquez Ayala, A.; Zhao, J.; Chu, H.; Kuijpers, M. C. M.; Jackrel, S. L.; Tugizimana, F.; Nephali, L. P.; Dubery, I. A.; Madala, N. E.; Moreira, E. A.; Costa-Lotufo, L. V.; Lopes, N. P.; Rezende-Teixeira, P.; Jimenez, P. C.; Rimal, B.; Patterson, A. D.; Traxler, M. F.; Pessotti, R. d. C.; Alvarado-Villalobos, D.; Tamayo-Castillo, G.; Chaverri, P.; Escud</p>
        <p class="info">Score: 39.0, Published: 2023-07-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549584' target='https://doi.org/10.1101/2023.07.20.549584'> 10.1101/2023.07.20.549584</a></p>
        <p class="abstract">MicrobeMASST, a taxonomically-informed mass spectrometry (MS) search tool, tackles limited microbial metabolite annotation in untargeted metabolomics experiments. Leveraging a curated database of &gt;60,000 microbial monocultures, users can search known and unknown MS/MS spectra and link them to their respective microbial producers via MS/MS fragmentation patterns. Identification of microbial-derived metabolites and relative producers, without a priori knowledge, will vastly enhance the understanding of microorganisms role in ecology and human health.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550326">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550326" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550326">
        <p class="paperTitle">Evolution and spread of a persistent 	P. aeruginosa outbreak clone over decades in a single US hospital.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550326" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550326" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stribling, W.; Hall, L. R.; Powell, A.; Harless, C.; Martin, M. J.; Corey, B. W.; Snesrud, E.; Ong, A.; Maybank, R.; Stam, J.; Bartlett, K.; Jones, B. T.; Preston, L. N.; Lane, K. F.; Thompson, B.; Young, L. M.; Kwak, Y. I.; Barsoumian, A. E.; Markelz, A.-E.; Kiley, J. L.; Cybulski, R. J.; Bennett, J. W.; Mc Gann, P. T.; Lebreton, F.</p>
        <p class="info">Score: 22.3, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550326' target='https://doi.org/10.1101/2023.07.24.550326'> 10.1101/2023.07.24.550326</a></p>
        <p class="abstract">Whole-genome sequencing is revolutionizing outbreak investigation. In 2020, prospective and retrospective surveillance detected a P. aeruginosa outbreak with 254 isolates collected from 82 patients in 27 wards of a hospital. Its origin was dated to the late 90s, just after the facility opened, and patient-to-patient and environment-to-patient cases of transmission were inferred. Over time, two epidemic subclones evolved in separate hosts and hospital areas, including newly opened wards, and hospital-wide sampling confirmed reservoirs persisted in the plumbing. Pathoadaptive mutations in genes associated with virulence, cell wall biogenesis, and antibiotic resistance were identified. While the latter correlated with the acquisition of phenotypic resistances to 1st (cephalosporin), 2nd (carbapenems) and 3rd (colistin) lines of treatment, maximum parsimony suggested that a truncation in a lipopolysaccharide component coincided with the emergence of a subclone prevalent in chronic infections. Since initial identification, extensive infection control efforts have proved successful at slowing transmission.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.547941">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.547941" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.547941">
        <p class="paperTitle">First Eurasian cases of SARS-CoV-2 seropositivity in a free-ranging urban population of wild fallow deer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.547941" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.547941" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Purves, K.; Brown, H.; Haverty, R.; Ryan, A.; Griffin, L.; McCormack, J.; O&#39;Reilly, S. R.; Mallon, P. W.; Gautier, V.; Cassidy, J. P.; Fabre, A.; Carr, M. J.; Gonzalez, G.; Ciuti, S.; Fletcher, N.</p>
        <p class="info">Score: 76.9, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.547941' target='https://doi.org/10.1101/2023.07.07.547941'> 10.1101/2023.07.07.547941</a></p>
        <p class="abstract">Severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) infects wildlife. Recent studies highlighted that variants of concern (VOC) may expand into novel animal reservoirs with the potential for reverse zoonosis. North American white-tailed deer are the only deer species in which SARS-CoV-2 has been documented, raising the question whether further reservoir species exist as new VOC emerge. Here, we report the first cases of deer SARS-CoV-2 seropositivity in Eurasia, in a city population of fallow deer in Dublin, Ireland. Deer were seronegative in 2020 (circulating variant in humans: Alpha), one animal was seropositive in 2021 (Delta variant), and 57% of animals tested in 2022 were seropositive (Omicron variant). Ex vivo, a clinical isolate of Omicron BA.1 infected fallow deer precision cut lung slice type-2 pneumocytes, also a major target of infection in human lungs. Our findings suggest a change in host tropism as new variants emerged in the human reservoir, highlighting the importance of continued wildlife disease monitoring and limiting human-wildlife contacts.

Teaser: This study is the first report of SARS-CoV-2 seropositivity in fallow deer, highlighting expansion of viral variants into new host reservoirs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550278">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550278" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550278">
        <p class="paperTitle">Mechanisms of extracellular electron transfer in anaerobic methanotrophic archaea</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550278" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550278" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ouboter, H. T.; Mesman, R.; Sleutels, T.; Postma, J.; Wissink, M.; Jetten, M. S. M.; ter Heijne, A.; Berben, T.; Welte, C.</p>
        <p class="info">Score: 13.5, Published: 2023-07-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550278' target='https://doi.org/10.1101/2023.07.24.550278'> 10.1101/2023.07.24.550278</a></p>
        <p class="abstract">Anaerobic methanotrophic (ANME) archaea are environmentally important uncultivated microorganisms mitigating the release of the potent greenhouse gas methane. During methane oxidation ANME archaea engage in extracellular electron transfer (EET) with other microorganisms, metal oxides, and electrodes, through a currently unknown mechanism. To shed light on this mechanism, we cultivated ANME-2d archaea (Ca. Methanoperedens) in bioelectrochemical systems and observed strong methane-dependent current (91-93% of total current) associated with high enrichment of  Ca. Methanoperedens on the anode (up to 82% of the community) determined by metagenomics and transmission electron microscopy. Electrochemistry and metatranscriptomics indicated that the EET mechanism was similar at various electrode potentials pointing to the involvement of an so far uncharacterized short-range electron transport protein complex and OmcZ nanowires, suggesting a unique EET pathway in all ANME-2 archaea. Our findings furthermore indicate that bioelectrochemical cells might be powerful tools for the cultivation, and possibly isolation, of uncultured electroactive microorganisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550273">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550273" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550273">
        <p class="paperTitle">Community structure of known and previously unknown endobacteria associated with spores of arbuscular mycorrhizal fungi.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550273" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550273" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lastovetsky, O. A.; Caruso, T.; Brennan, F.; Wall, D.; Pylni, S.; Doyle, E.</p>
        <p class="info">Score: 12.8, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550273' target='https://doi.org/10.1101/2023.07.25.550273'> 10.1101/2023.07.25.550273</a></p>
        <p class="abstract">Arbuscular mycorrhizal fungi (AMF) are ubiquitous plant root symbionts which can house two endobacteria: Ca. Moeniiplasma glomeromycotorum (CaMg) and Ca. Glomeribacter gigasporarum (CaGg). However, little is known about their distribution and population structure in natural AMF populations and whether AMF can harbour other endobacteria. We isolated AMF from two environments and surveyed the surface-sterilized spores for endobacteria. We found that CaMg and CaGg differed significantly in distribution whereby CaMg were extremely abundant (80%) and CaGg were extremely rare (2%) in both environments. Unexpectedly, we discovered an additional and previously unknown level of bacterial diversity within AMF spores which extended beyond the known endosymbionts, with as many as 277 other bacterial taxa detected in individual spores. Detailed analysis of endobacterial communities inside AMF spores revealed that: (i)CaGg were not limited in distribution to the Gigasporacea family of AMF, as previously thought, (ii) CaMg community structure was driven by AMF host genotype, (iii) a significant inverse correlation existed between the diversity of CaMg and diversity of all other endobacteria. The latter suggests the existence of competition dynamics between different bacterial populations inside AMF spores and provides a basis for generation of testable hypotheses regarding the function of CaMg in AMF biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.550687">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.550687" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.550687">
        <p class="paperTitle">Diversity and biotechnological potential of filamentous fungi isolated from sediments of Basque estuaries</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.550687" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.550687" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Otamendi, A.; Agirrezabala, Z.; Perez-Cruz, C.; Liebana, R.; Berregi, I.; Alonso-Saez, L.; Duenas, M. T.; Lanzen, A.; Etxebeste, O.</p>
        <p class="info">Score: 10.5, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.550687' target='https://doi.org/10.1101/2023.07.26.550687'> 10.1101/2023.07.26.550687</a></p>
        <p class="abstract">Marine environments harbor a vast diversity of microorganisms, which have developed multiple strategies to adapt to stress conditions such as high salt concentrations and nutrient scarcity and, thus, are a source of natural products and high-value enzymes, as they have the ability to degrade complex polymeric substrates. These features make marine microorganisms a valuable source for the development of new biotechnological tools. Here, we focused on the isolation of filamentous fungi, using sediment samples collected from estuaries of the Basque Country in the Bay of Biscay. The phenotype of these isolates was characterized based on the potential to grow on complex marine polysaccharides or to secrete pigments to the culture medium. Based on this screening, the genomes of two isolates of the order Hypocreales, Marquandomyces marquandii and Albophoma yamanashiensis, were sequenced and are described here for the first time. The analysis of their CAZYme (carbohydrate-active enzyme) and secondary metabolite gene-cluster repertoires suggest that these estuarine isolates could be used as a source of novel enzymatic activities and secondary metabolites.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.22.550130">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.22.550130" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.22.550130">
        <p class="paperTitle">Chromosome organization shapes replisome dynamics in Caulobacter crescentus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.22.550130" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.22.550130" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, C.; Joseph, A. M.; Casini, L.; Collier, J.; Badrinarayanan, A.; Manley, S.</p>
        <p class="info">Score: 10.5, Published: 2023-07-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.22.550130' target='https://doi.org/10.1101/2023.07.22.550130'> 10.1101/2023.07.22.550130</a></p>
        <p class="abstract">DNA replication in bacteria takes place on highly compacted chromosomes, where segregation, transcription, and repair must occur simultaneously. Within this dynamic environment, colocalization of sister replisomes has been observed in many bacterial species, driving the hypothesis that a physical linker may tether them together. However, replisome splitting has also been reported in many of the same species, leaving the principles behind replisome organization a long-standing puzzle. Here, by tracking the replisome {beta}-clamp subunit in live Caulobacter crescentus, we find that rapid DNA segregation can give rise to a second focus which resembles a replisome, but does not replicate DNA. Sister replisomes can remain colocalized, or split apart to travel along DNA separately upon disruption of chromosome inter-arm alignment. Furthermore, chromosome arm-specific replication-transcription conflicts differentially modify replication speed on the two arms, facilitate the decoupling of the two replisomes. With these observations, we conclude that the dynamic chromosome organization flexibly shapes the organization of sister replisomes, and we outline principles which can help to reconcile previously conflicting models of replisome architecture.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550428">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550428" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550428">
        <p class="paperTitle">Experimentally Evolved Staphylococcus aureus Survives in the Presence of Pseudomonas aeruginosa by Acquiring Mutations in the Amino Acid Transporter, GltT</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550428" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550428" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alexander, A. M.; Luu, J. M.; Raghuram, V.; Bottacin, G.; van Vliet, S.; Read, T. D.; Goldberg, J. B.</p>
        <p class="info">Score: 9.5, Published: 2023-07-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550428' target='https://doi.org/10.1101/2023.07.24.550428'> 10.1101/2023.07.24.550428</a></p>
        <p class="abstract">Staphylococcus aureus and Pseudomonas aeruginosa are the most common bacterial pathogens isolated from cystic fibrosis (CF) related lung infections. When both of these opportunistic pathogens are found in a coinfection, CF patients tend to have higher rates of pulmonary exacerbations and experience a more rapid decrease in lung function. When cultured together under standard laboratory conditions, it is often observed that P. aeruginosa effectively inhibits S. aureus growth. Previous work from our group revealed that S. aureus from CF infections have isolate-specific survival capabilities when cocultured with P. aeruginosa. In this study, we designed a serial transfer evolution experiment to identify mutations that allow S. aureus to adapt to the presence of P. aeruginosa. Using S. aureus USA300 JE2 as our ancestral strain, populations of S. aureus were repeatedly cocultured with fresh P. aeruginosa strain, PAO1. After 8 coculture periods, S. aureus populations that survived better in the presence of PAO1 were observed. We found two independent mutations in the highly conserved S. aureus aspartate transporter, gltT, that were unique to evolved P. aeruginosa-tolerant isolates. Subsequent phenotypic testing demonstrated that gltT mutants have reduced uptake of glutamate and outcompete wild-type S. aureus when glutamate is absent from chemically-defined media. These findings together demonstrate that the presence of P. aeruginosa exerts selective pressure on S. aureus to alter its uptake and metabolism of key amino acids when the two bacteria are cultured together.

ImportanceStaphylococcus aureus and Pseudomonas aeruginosa are the two most common bacterial pathogens that infect people with the genetic disease, cystic fibrosis (CF). They are often found together in CF-associated polymicrobial infections that are associated with worse patient prognosis. Understanding how these very different opportunistic pathogens influence each other in a shared environment is pertinent to improving the treatment of polymicrobial infections. While much attention has been brought to the interspecific interactions between S. aureus and P. aeruginosa, few studies have used experimental evolution methods to identify determinants of their competition and coexistence. Here, we use a serial transfer experimental evolution approach and identified a single genetic change associated with improved survival of S. aureus in the presence of P. aeruginosa. Our findings implicate metabolism of shared resources as an important factor in S. aureuss ability to survive in the presence of P. aeruginosa.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
