<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://yorks0n.github.io/TRxiv2" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/TRxiv2/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Type IV-A3 CRISPR-Cas systems drive inter-plasmid conflicts by acquiring spacers in trans
Authors: Benz, F.; Camara-Wilpert, S.; Russel, J.; Wandera, K. G.; Cepaite, R.; Ares-Arroyo, M.; Gomes-Filho, J. V.; Englert, F.; Kuehn, J.; Gloor, S.; Cuenod, A.; Aguila-Sans, M.; Maccario, L.; Egli, A.; Randau, L.; Pausch, P.; Rocha, E.; Beisel, C. L.; Madsen, J. S.; Bikard, D.; Hall, A. R.; Soerensen, S. J.; Pinilla-Redondo, R.
Score: 74.8, Published: 2023-06-23 DOI: 10.">
<meta name="author" content="">
<link rel="canonical" href="https://yorks0n.github.io/TRxiv2/posts/microbiology/">
<link crossorigin="anonymous" href="/TRxiv2/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/TRxiv2/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://yorks0n.github.io/TRxiv2/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://yorks0n.github.io/TRxiv2/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://yorks0n.github.io/TRxiv2/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://yorks0n.github.io/TRxiv2/apple-touch-icon.png">
<link rel="mask-icon" href="https://yorks0n.github.io/TRxiv2/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Type IV-A3 CRISPR-Cas systems drive inter-plasmid conflicts by acquiring spacers in trans
Authors: Benz, F.; Camara-Wilpert, S.; Russel, J.; Wandera, K. G.; Cepaite, R.; Ares-Arroyo, M.; Gomes-Filho, J. V.; Englert, F.; Kuehn, J.; Gloor, S.; Cuenod, A.; Aguila-Sans, M.; Maccario, L.; Egli, A.; Randau, L.; Pausch, P.; Rocha, E.; Beisel, C. L.; Madsen, J. S.; Bikard, D.; Hall, A. R.; Soerensen, S. J.; Pinilla-Redondo, R.
Score: 74.8, Published: 2023-06-23 DOI: 10." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://yorks0n.github.io/TRxiv2/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-06-28T08:34:30+00:00" />
<meta property="article:modified_time" content="2023-06-28T08:34:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Type IV-A3 CRISPR-Cas systems drive inter-plasmid conflicts by acquiring spacers in trans
Authors: Benz, F.; Camara-Wilpert, S.; Russel, J.; Wandera, K. G.; Cepaite, R.; Ares-Arroyo, M.; Gomes-Filho, J. V.; Englert, F.; Kuehn, J.; Gloor, S.; Cuenod, A.; Aguila-Sans, M.; Maccario, L.; Egli, A.; Randau, L.; Pausch, P.; Rocha, E.; Beisel, C. L.; Madsen, J. S.; Bikard, D.; Hall, A. R.; Soerensen, S. J.; Pinilla-Redondo, R.
Score: 74.8, Published: 2023-06-23 DOI: 10."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://yorks0n.github.io/TRxiv2/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://yorks0n.github.io/TRxiv2/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Type IV-A3 CRISPR-Cas systems drive inter-plasmid conflicts by acquiring spacers in trans\nAuthors: Benz, F.; Camara-Wilpert, S.; Russel, J.; Wandera, K. G.; Cepaite, R.; Ares-Arroyo, M.; Gomes-Filho, J. V.; Englert, F.; Kuehn, J.; Gloor, S.; Cuenod, A.; Aguila-Sans, M.; Maccario, L.; Egli, A.; Randau, L.; Pausch, P.; Rocha, E.; Beisel, C. L.; Madsen, J. S.; Bikard, D.; Hall, A. R.; Soerensen, S. J.; Pinilla-Redondo, R.\nScore: 74.8, Published: 2023-06-23 DOI: 10.",
  "keywords": [
    
  ],
  "articleBody": " Type IV-A3 CRISPR-Cas systems drive inter-plasmid conflicts by acquiring spacers in trans\nAuthors: Benz, F.; Camara-Wilpert, S.; Russel, J.; Wandera, K. G.; Cepaite, R.; Ares-Arroyo, M.; Gomes-Filho, J. V.; Englert, F.; Kuehn, J.; Gloor, S.; Cuenod, A.; Aguila-Sans, M.; Maccario, L.; Egli, A.; Randau, L.; Pausch, P.; Rocha, E.; Beisel, C. L.; Madsen, J. S.; Bikard, D.; Hall, A. R.; Soerensen, S. J.; Pinilla-Redondo, R.\nScore: 74.8, Published: 2023-06-23 DOI: 10.1101/2023.06.23.546257\nType IV-A CRISPR-Cas systems are primarily encoded on plasmids and form multi-subunit ribonucleoprotein complexes with unknown biological functions. In contrast to other CRISPR-Cas types, they lack the archetypical CRISPR acquisition module and encode a DinG helicase instead of a nuclease component. Type IV-A3 systems are carried by large conjugative plasmids that often harbor multiple antibiotic-resistance genes. Although their CRISPR array contents suggest a role in inter-plasmid conflicts, this function and the underlying mechanisms have remained unexplored. Here, we demonstrate that a plasmid-encoded type IV-A3 CRISPR-Cas system co-opts the type I-E adaptation machinery from its clinical Klebsiella pneumoniae host to update its CRISPR array. Furthermore, we demonstrate that robust interference of conjugative plasmids and phages is elicited through CRISPR RNA-dependent transcriptional repression. By targeting plasmid core functions, type IV-A3 can prevent the uptake of incoming plasmids, limit their horizontal transfer, and destabilize co-residing plasmids, altogether supporting type IV-A3s involvement in plasmid competition. Collectively, our findings shed light on the molecular mechanisms and ecological function of type IV-A3 systems and have broad implications for understanding and countering the spread of antibiotic resistance in clinically relevant strains.\nWaste not, want not: Revisiting the analysis that called into question the practice of rarefaction\nAuthors: Schloss, P. D.\nScore: 59.8, Published: 2023-06-26 DOI: 10.1101/2023.06.23.546312\nIn 2014, McMurdie and Holmes published the provocatively titled, \"Waste not, want not: why rarefying microbiome data is inadmissible\". The claims of their study have significantly altered how microbiome researchers control for the unavoidable uneven sequencing depths that are inherent in modern 16S rRNA gene sequencing. Confusion over the distinction between the definitions of rarefying and rarefaction continues to cloud the interpretation of their results. More importantly, the authors made a variety of choices in designing and analyzing their simulations that were potentially problematic. I identified 11 factors that could have compromised the results of the original study. I reproduced the original simulation results and assessed the impact of those 11 factors on the underlying conclusion that rarefying data is inadmissible. Throughout, the design of the original study made choices that caused rarefying and rarefaction to appear to perform worse than they truly did. Most important were the approaches used to assess ecological distances, the removal of samples with low sequencing depth, and not accounting for conditions where sequencing effort is confounded with treatment group. Although the original study criticized rarefying for the arbitrary removal of valid data, repeatedly rarefying data many times (i.e., rarefaction) incorporates all the data. In contrast, it is the removal of rare taxa that would appear to remove valid data. Overall, I show that rarefaction is the most robust approach to control for uneven sequencing effort when considered across a variety of alpha and beta diversity metrics. Importance: Over the past 10 years, the best method for normalizing the sequencing depth of samples characterized by 16S rRNA gene sequencing has been contentious. An often cited paper by McMurdie and Holmes forcefully argued that rarefying the number of sequence counts was \"inadmissible\" and should not be employed. However, I identified a number of problems with the design of their simulations and analysis that could compromise their results. In fact, when I reproduced and expanded upon their analysis, it was clear that rarefaction was actually the most robust approach for controlling for uneven sequencing effort across samples. Rarefaction limits the rate of falsely detecting and rejecting differences between treatment groups. Far from being \"inadmissible\", rarefaction is a valuable tool for analyzing microbiome sequence data.\nRarefaction is currently the best approach to control for uneven sequencing effort in amplicon sequence analyses\nAuthors: Schloss, P. D.\nScore: 56.0, Published: 2023-06-23 DOI: 10.1101/2023.06.23.546313\nConsidering it is common to find as much as 100-fold variation in the number of 16S rRNA gene sequences found across samples in a study, researchers need to control for the effect of uneven sequencing effort. How to do this has become a contentious question. Some have argued that rarefying or rarefaction is \"inadmissible\" because it omits valid data. A number of alternative approaches have been developed to normalize and rescale the data that purport to be invariant to the number of observations. I generated community distributions based on 12 published datasets where I was able to assess the ability of multiple methods to control for uneven sequencing effort. Rarefaction was the only method that could control for variation in uneven sequencing effort when measuring commonly used alpha and beta diversity metrics. Next, I compared the false detection rate and power to detect true differences between simulated communities with a known effect size using various alpha and beta diversity metrics. Although all methods of controlling for uneven sequencing effort had an acceptable false detection rate when samples were randomly assigned to two treatment groups, rarefaction was consistently able to control for differences in sequencing effort when sequencing depth was confounded with treatment group. Finally, the statistical power to detect differences in alpha and beta diversity metrics was consistently the highest when using rarefaction. These simulations underscore the importance of using rarefaction to normalize the number of sequences across samples in amplicon sequencing analyses. ImportanceSequencing 16S rRNA gene fragments has become a fundamental tool for understanding the diversity of microbial communities and the factors that affect their diversity. Due to technical challenges, it is common to observe wide variation in the number of sequences that are collected from different samples within the same study. However, the diversity metrics used by microbial ecologists are sensitive to differences in sequencing effort. Therefore, tools are needed to control for the uneven levels of sequencing. This simulation-based analysis shows that despite a longstanding controversy, rarefaction is the most robust approach to control for uneven sequencing effort. The controversy started because of confusion over the definition of rarefaction and violation of assumptions that are made by methods that have been borrowed from other fields. Microbial ecologists should use rarefaction.\nDiverse viral cas genes antagonize CRISPR immunity\nAuthors: Katz, M. A.; Sawyer, E. M.; Kozlova, A.; Williams, M. C.; Margolis, S. R.; Oriolt, L.; Johnson, M.; Bondy-Denomy, J.; Meeske, A. J.\nScore: 45.5, Published: 2023-06-24 DOI: 10.1101/2023.06.24.545427\nProkaryotic CRISPR-Cas immunity is subverted via anti-CRISPRs (Acrs), small proteins that inhibit Cas protein activities when expressed during the phage lytic cycle or from resident prophages or plasmids. CRISPR-Cas defenses are classified into 6 types and 33 subtypes, which employ a diverse suite of Cas effectors and differ in their mechanisms of interference. As Acrs often work via binding to a cognate Cas protein, inhibition is almost always limited to a single CRISPR type. Furthermore, while acr genes are frequently organized together in phage-associated gene clusters, how such inhibitors initially evolve has remained unclear. Here we have investigated the Acr content and inhibition specificity of a collection of Listeria isolates, which naturally harbor four diverse CRISPR-Cas systems (types I-B, II-A, II-C, and VI-A). We observed widespread antagonism of CRISPR, which we traced to 12 novel and 4 known Acr gene families encoded on endogenous mobile genetic elements. Among these were two Acrs that possess sequence homology to type I-B Cas proteins and assemble into a defective interference complex. Surprisingly, an additional type I-B Cas homolog did not affect type I immunity, but instead inhibited the RNA-targeting type VI CRISPR system through sequestration of crRNA. By probing the IMGVR database of viral genomes, we detected abundant orphan cas genes located within putative anti-defense gene clusters. We experimentally verified the Acr activity of one viral cas gene, a particularly broad-spectrum cas3 homolog that inhibits type I-B, II-A, and VI-A CRISPR immunity. Our observations provide direct evidence of Acr evolution via cas gene co-option, and new genes with potential for broad-spectrum control of genome editing technologies.\nVirome analysis of New Zealand's bats reveals cross-species viral transmission among the Coronaviridae\nAuthors: Waller, S. J.; Tortosa, P.; Thurley, T.; O'Donnell, C.; Jackson, R.; Dennis, G.; Grimwood, R. M.; Holmes, E. C.; McInnes, K.; Geoghegan, J. L.\nScore: 38.9, Published: 2023-06-20 DOI: 10.1101/2023.06.19.545647\n1.The lesser short-tailed bat (Mystacina tuberculata) and the long-tailed bat (Chalinolobus tuberculatus) are Aotearoa New Zealands only native extant terrestrial mammals and are believed to have migrated from Australia. Long-tailed bats arrived in New Zealand an estimated two million years ago and are closely related to other Australian bat species. Lesser short-tailed bats, in contrast, are the only extant species within the Mystacinidae and are estimated to have been living in isolation in New Zealand for the past 16-18 million years. Throughout this period of isolation, lesser short-tailed bats have become one of the most terrestrial bats in the world. Through a metatranscriptomic analysis of guano samples from eight locations across New Zealand we aimed to characterise the viromes of New Zealands bats and determine whether viruses have jumped between these species over the past two million years. High viral richness was observed among long-tailed bats with viruses spanning seven different viral families. In contrast, no bat-specific viruses were identified in lesser short-tailed bats. Both bat species harboured an abundance of likely dietary- and environmental-associated viruses. We also identified alphacoronaviruses in long-tailed bat guano that had previously been identified in lesser short-tailed bats, suggesting that these viruses had jumped the species barrier after long-tailed bats migrated to New Zealand. Of note, an alphacoronavirus species discovered here possessed a complete genome of only 22,416 nucleotides with entire deletions or truncations of several non-structural proteins, thereby representing what is possibly the shortest genome within the Coronaviridae identified to date. Overall, this study has revealed a diverse range of novel viruses harboured by New Zealands only native terrestrial mammals, in turn expanding our understanding of bat viral dynamics and evolution globally.\nThe F pilus serves as a conduit for the DNA during conjugation between physically distant bacteria\nAuthors: Goldlust, K.; Ducret, A.; Halte, M.; Dedieu-Berne, A.; Erhardt, M.; Lesterlin, C.\nScore: 25.8, Published: 2023-06-23 DOI: 10.1101/2023.06.21.545889\nHorizontal transfer of F-like plasmids by bacterial conjugation is responsible for disseminating antibiotic resistance and virulence determinants among pathogenic Enterobacteriaceae species, a growing health concern worldwide. Central to this process is the conjugative F pilus, a long extracellular filamentous polymer that extends from the surface of plasmid donor cells, allowing it to probe the environment and make contact with the recipient cell. It is well established the F pilus can retract to bring mating pair cells in tight contact before DNA transfer. However, whether DNA transfer can occur through the extended pilus has been a subject of active debate. In this study, we use live-cell microscopy to show that the F pilus can indeed serve as a conduit for the DNA during transfer between physically distant cells. Our findings enable us to propose a new model for conjugation that revises our understanding of the DNA transport mechanism and the dissemination of drug resistance in complex bacterial communities. One-Sentence SummaryPlasmid DNA passes through the F pilus during conjugational transfer between physically distant bacteria.\nCell arrangement impacts metabolic activity and antibiotic tolerance in Pseudomonas aeruginosa biofilms\nAuthors: Dayton, H.; Kiss, J.; Wei, M.; Chauhan, S.; LaMarre, E.; Cornell, W. C.; Janakiraman, A.; Min, W.; Tomer, R.; Price-Whelan, A.; Nirody, J. A.; Dietrich, L. E.\nScore: 22.5, Published: 2023-06-20 DOI: 10.1101/2023.06.20.545666\nCells must access resources to survive, and the anatomy of multicellular structures influences this access. In diverse multicellular eukaryotes, resources are provided by internal conduits that allow substances to travel more readily through tissue than they would via diffusion. Microbes growing in multicellular structures, called biofilms, are also affected by differential access to resources and we hypothesized that this is influenced by the physical arrangement of the cells. In this study, we examined the microanatomy of biofilms formed by the pathogenic bacterium Pseudomonas aeruginosa and discovered that clonal cells form striations that are packed lengthwise across most of a mature biofilms depth. We identified mutants, including those defective in pilus function and in O-antigen attachment, that show alterations to this lengthwise packing phenotype. Consistent with the notion that cellular arrangement affects access to resources within the biofilm, we found that while the wild type shows even distribution of tested substrates across depth, the mutants show accumulation of substrates at the biofilm boundaries. Furthermore, we found that altered cellular arrangement within biofilms affects the localization of metabolic activity, the survival of resident cells, and the susceptibility of subpopulations to antibiotic treatment. Our observations provide insight into cellular features that determine biofilm microanatomy, with consequences for physiological differentiation and drug sensitivity.\nHigh-accuracy meets high-throughput for microbiome profiling with near full-length 16S rRNA amplicon sequencing on the Nanopore platform\nAuthors: Lin, X.; Waring, K.; Tyson, J.; Ziels, R.\nScore: 38.4, Published: 2023-06-19 DOI: 10.1101/2023.06.19.544637\nAmplicon sequencing of small subunit (SSU) rRNA genes is a foundational method for studying microbial communities within various environmental, human, and engineered ecosystems. Currently, short-read platforms are commonly employed for high-throughput applications of SSU rRNA amplicon sequencing, but at the cost of poor taxonomic classification. The low-cost Oxford Nanopore Technologies (ONT) platform is capable of sequencing full-length SSU rRNA genes, but the lower raw-read accuracies of previous ONT sequencing chemistries have limited accurate taxonomic classification and de novo generation of operational taxonomic units (OTUs) and amplicon sequence variants (ASVs). Here, we examine the potential for Nanopore sequencing with newer (R10.4+) chemistry to provide high-throughput and high-accuracy full-length 16S rRNA gene amplicon sequencing. We present a sequencing workflow utilizing unique molecular identifiers (UMIs) for error-correction of SSU rRNA (e.g. 16S rRNA) gene amplicons, termed ssUMI. Using two synthetic microbial community standards, the ssUMI workflow generated consensus sequences with 99.99% mean accuracy using a minimum UMI subread coverage threshold of 3x, and was capable of generating error-free ASVs and 97% OTUs with no false-positives. Non-corrected Nanopore reads generated error-free 97% OTUs but with reduced detection sensitivity, and also generated false-positive ASVs. We showcase the cost-competitive and high-throughput scalability of the ssUMI workflow by sequencing 90 time-series samples from seven different wastewater matrices, generating ASVs that were tightly clustered based on sample matrix type. This work demonstrates that highly accurate full-length 16S rRNA gene amplicon sequencing on Nanopore is possible, paving the way to more accessible microbiome science.\n",
  "wordCount" : "2407",
  "inLanguage": "en",
  "datePublished": "2023-06-28T08:34:30Z",
  "dateModified": "2023-06-28T08:34:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://yorks0n.github.io/TRxiv2/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://yorks0n.github.io/TRxiv2/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://yorks0n.github.io/TRxiv2" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta"><span>updated on June 28, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546257">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546257" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546257">
        <p class="paperTitle">Type IV-A3 CRISPR-Cas systems drive inter-plasmid conflicts by acquiring spacers in trans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546257" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546257" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Benz, F.; Camara-Wilpert, S.; Russel, J.; Wandera, K. G.; Cepaite, R.; Ares-Arroyo, M.; Gomes-Filho, J. V.; Englert, F.; Kuehn, J.; Gloor, S.; Cuenod, A.; Aguila-Sans, M.; Maccario, L.; Egli, A.; Randau, L.; Pausch, P.; Rocha, E.; Beisel, C. L.; Madsen, J. S.; Bikard, D.; Hall, A. R.; Soerensen, S. J.; Pinilla-Redondo, R.</p>
        <p class="info">Score: 74.8, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546257' target='https://doi.org/10.1101/2023.06.23.546257'> 10.1101/2023.06.23.546257</a></p>
        <p class="abstract">Type IV-A CRISPR-Cas systems are primarily encoded on plasmids and form multi-subunit ribonucleoprotein complexes with unknown biological functions. In contrast to other CRISPR-Cas types, they lack the archetypical CRISPR acquisition module and encode a DinG helicase instead of a nuclease component. Type IV-A3 systems are carried by large conjugative plasmids that often harbor multiple antibiotic-resistance genes. Although their CRISPR array contents suggest a role in inter-plasmid conflicts, this function and the underlying mechanisms have remained unexplored. Here, we demonstrate that a plasmid-encoded type IV-A3 CRISPR-Cas system co-opts the type I-E adaptation machinery from its clinical Klebsiella pneumoniae host to update its CRISPR array. Furthermore, we demonstrate that robust interference of conjugative plasmids and phages is elicited through CRISPR RNA-dependent transcriptional repression. By targeting plasmid core functions, type IV-A3 can prevent the uptake of incoming plasmids, limit their horizontal transfer, and destabilize co-residing plasmids, altogether supporting type IV-A3s involvement in plasmid competition. Collectively, our findings shed light on the molecular mechanisms and ecological function of type IV-A3 systems and have broad implications for understanding and countering the spread of antibiotic resistance in clinically relevant strains.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546312">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546312" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546312">
        <p class="paperTitle">Waste not, want not: Revisiting the analysis that called into question the practice of rarefaction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546312" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546312" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schloss, P. D.</p>
        <p class="info">Score: 59.8, Published: 2023-06-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546312' target='https://doi.org/10.1101/2023.06.23.546312'> 10.1101/2023.06.23.546312</a></p>
        <p class="abstract">In 2014, McMurdie and Holmes published the provocatively titled, &#34;Waste not, want not: why rarefying microbiome data is inadmissible&#34;. The claims of their study have significantly altered how microbiome researchers control for the unavoidable uneven sequencing depths that are inherent in modern 16S rRNA gene sequencing. Confusion over the distinction between the definitions of rarefying and rarefaction continues to cloud the interpretation of their results. More importantly, the authors made a variety of choices in designing and analyzing their simulations that were potentially problematic. I identified 11 factors that could have compromised the results of the original study. I reproduced the original simulation results and assessed the impact of those 11 factors on the underlying conclusion that rarefying data is inadmissible. Throughout, the design of the original study made choices that caused rarefying and rarefaction to appear to perform worse than they truly did. Most important were the approaches used to assess ecological distances, the removal of samples with low sequencing depth, and not accounting for conditions where sequencing effort is confounded with treatment group. Although the original study criticized rarefying for the arbitrary removal of valid data, repeatedly rarefying data many times (i.e., rarefaction) incorporates all the data. In contrast, it is the removal of rare taxa that would appear to remove valid data. Overall, I show that rarefaction is the most robust approach to control for uneven sequencing effort when considered across a variety of alpha and beta diversity metrics. Importance: Over the past 10 years, the best method for normalizing the sequencing depth of samples characterized by 16S rRNA gene sequencing has been contentious. An often cited paper by McMurdie and Holmes forcefully argued that rarefying the number of sequence counts was &#34;inadmissible&#34; and should not be employed. However, I identified a number of problems with the design of their simulations and analysis that could compromise their results. In fact, when I reproduced and expanded upon their analysis, it was clear that rarefaction was actually the most robust approach for controlling for uneven sequencing effort across samples. Rarefaction limits the rate of falsely detecting and rejecting differences between treatment groups. Far from being &#34;inadmissible&#34;, rarefaction is a valuable tool for analyzing microbiome sequence data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546313">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546313" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546313">
        <p class="paperTitle">Rarefaction is currently the best approach to control for uneven sequencing effort in amplicon sequence analyses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546313" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546313" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schloss, P. D.</p>
        <p class="info">Score: 56.0, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546313' target='https://doi.org/10.1101/2023.06.23.546313'> 10.1101/2023.06.23.546313</a></p>
        <p class="abstract">Considering it is common to find as much as 100-fold variation in the number of 16S rRNA gene sequences found across samples in a study, researchers need to control for the effect of uneven sequencing effort. How to do this has become a contentious question. Some have argued that rarefying or rarefaction is &#34;inadmissible&#34; because it omits valid data. A number of alternative approaches have been developed to normalize and rescale the data that purport to be invariant to the number of observations. I generated community distributions based on 12 published datasets where I was able to assess the ability of multiple methods to control for uneven sequencing effort. Rarefaction was the only method that could control for variation in uneven sequencing effort when measuring commonly used alpha and beta diversity metrics. Next, I compared the false detection rate and power to detect true differences between simulated communities with a known effect size using various alpha and beta diversity metrics. Although all methods of controlling for uneven sequencing effort had an acceptable false detection rate when samples were randomly assigned to two treatment groups, rarefaction was consistently able to control for differences in sequencing effort when sequencing depth was confounded with treatment group. Finally, the statistical power to detect differences in alpha and beta diversity metrics was consistently the highest when using rarefaction. These simulations underscore the importance of using rarefaction to normalize the number of sequences across samples in amplicon sequencing analyses.

ImportanceSequencing 16S rRNA gene fragments has become a fundamental tool for understanding the diversity of microbial communities and the factors that affect their diversity. Due to technical challenges, it is common to observe wide variation in the number of sequences that are collected from different samples within the same study. However, the diversity metrics used by microbial ecologists are sensitive to differences in sequencing effort. Therefore, tools are needed to control for the uneven levels of sequencing. This simulation-based analysis shows that despite a longstanding controversy, rarefaction is the most robust approach to control for uneven sequencing effort. The controversy started because of confusion over the definition of rarefaction and violation of assumptions that are made by methods that have been borrowed from other fields. Microbial ecologists should use rarefaction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.24.545427">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.24.545427" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.24.545427">
        <p class="paperTitle">Diverse viral cas genes antagonize CRISPR immunity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.24.545427" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.24.545427" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Katz, M. A.; Sawyer, E. M.; Kozlova, A.; Williams, M. C.; Margolis, S. R.; Oriolt, L.; Johnson, M.; Bondy-Denomy, J.; Meeske, A. J.</p>
        <p class="info">Score: 45.5, Published: 2023-06-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.24.545427' target='https://doi.org/10.1101/2023.06.24.545427'> 10.1101/2023.06.24.545427</a></p>
        <p class="abstract">Prokaryotic CRISPR-Cas immunity is subverted via anti-CRISPRs (Acrs), small proteins that inhibit Cas protein activities when expressed during the phage lytic cycle or from resident prophages or plasmids. CRISPR-Cas defenses are classified into 6 types and 33 subtypes, which employ a diverse suite of Cas effectors and differ in their mechanisms of interference. As Acrs often work via binding to a cognate Cas protein, inhibition is almost always limited to a single CRISPR type. Furthermore, while acr genes are frequently organized together in phage-associated gene clusters, how such inhibitors initially evolve has remained unclear. Here we have investigated the Acr content and inhibition specificity of a collection of Listeria isolates, which naturally harbor four diverse CRISPR-Cas systems (types I-B, II-A, II-C, and VI-A). We observed widespread antagonism of CRISPR, which we traced to 12 novel and 4 known Acr gene families encoded on endogenous mobile genetic elements. Among these were two Acrs that possess sequence homology to type I-B Cas proteins and assemble into a defective interference complex. Surprisingly, an additional type I-B Cas homolog did not affect type I immunity, but instead inhibited the RNA-targeting type VI CRISPR system through sequestration of crRNA. By probing the IMGVR database of viral genomes, we detected abundant orphan cas genes located within putative anti-defense gene clusters. We experimentally verified the Acr activity of one viral cas gene, a particularly broad-spectrum cas3 homolog that inhibits type I-B, II-A, and VI-A CRISPR immunity. Our observations provide direct evidence of Acr evolution via cas gene co-option, and new genes with potential for broad-spectrum control of genome editing technologies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.545647">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.545647" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.545647">
        <p class="paperTitle">Virome analysis of New Zealand&#39;s bats reveals cross-species viral transmission among the Coronaviridae</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.545647" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.545647" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Waller, S. J.; Tortosa, P.; Thurley, T.; O&#39;Donnell, C.; Jackson, R.; Dennis, G.; Grimwood, R. M.; Holmes, E. C.; McInnes, K.; Geoghegan, J. L.</p>
        <p class="info">Score: 38.9, Published: 2023-06-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.545647' target='https://doi.org/10.1101/2023.06.19.545647'> 10.1101/2023.06.19.545647</a></p>
        <p class="abstract">1.The lesser short-tailed bat (Mystacina tuberculata) and the long-tailed bat (Chalinolobus tuberculatus) are Aotearoa New Zealands only native extant terrestrial mammals and are believed to have migrated from Australia. Long-tailed bats arrived in New Zealand an estimated two million years ago and are closely related to other Australian bat species. Lesser short-tailed bats, in contrast, are the only extant species within the Mystacinidae and are estimated to have been living in isolation in New Zealand for the past 16-18 million years. Throughout this period of isolation, lesser short-tailed bats have become one of the most terrestrial bats in the world.

Through a metatranscriptomic analysis of guano samples from eight locations across New Zealand we aimed to characterise the viromes of New Zealands bats and determine whether viruses have jumped between these species over the past two million years. High viral richness was observed among long-tailed bats with viruses spanning seven different viral families. In contrast, no bat-specific viruses were identified in lesser short-tailed bats. Both bat species harboured an abundance of likely dietary- and environmental-associated viruses. We also identified alphacoronaviruses in long-tailed bat guano that had previously been identified in lesser short-tailed bats, suggesting that these viruses had jumped the species barrier after long-tailed bats migrated to New Zealand. Of note, an alphacoronavirus species discovered here possessed a complete genome of only 22,416 nucleotides with entire deletions or truncations of several non-structural proteins, thereby representing what is possibly the shortest genome within the Coronaviridae identified to date. Overall, this study has revealed a diverse range of novel viruses harboured by New Zealands only native terrestrial mammals, in turn expanding our understanding of bat viral dynamics and evolution globally.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545889">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545889" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545889">
        <p class="paperTitle">The F pilus serves as a conduit for the DNA during conjugation between physically distant bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545889" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545889" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Goldlust, K.; Ducret, A.; Halte, M.; Dedieu-Berne, A.; Erhardt, M.; Lesterlin, C.</p>
        <p class="info">Score: 25.8, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545889' target='https://doi.org/10.1101/2023.06.21.545889'> 10.1101/2023.06.21.545889</a></p>
        <p class="abstract">Horizontal transfer of F-like plasmids by bacterial conjugation is responsible for disseminating antibiotic resistance and virulence determinants among pathogenic Enterobacteriaceae species, a growing health concern worldwide. Central to this process is the conjugative F pilus, a long extracellular filamentous polymer that extends from the surface of plasmid donor cells, allowing it to probe the environment and make contact with the recipient cell. It is well established the F pilus can retract to bring mating pair cells in tight contact before DNA transfer. However, whether DNA transfer can occur through the extended pilus has been a subject of active debate. In this study, we use live-cell microscopy to show that the F pilus can indeed serve as a conduit for the DNA during transfer between physically distant cells. Our findings enable us to propose a new model for conjugation that revises our understanding of the DNA transport mechanism and the dissemination of drug resistance in complex bacterial communities.

One-Sentence SummaryPlasmid DNA passes through the F pilus during conjugational transfer between physically distant bacteria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.20.545666">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.20.545666" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.20.545666">
        <p class="paperTitle">Cell arrangement impacts metabolic activity and antibiotic tolerance in Pseudomonas aeruginosa biofilms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.20.545666" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.20.545666" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dayton, H.; Kiss, J.; Wei, M.; Chauhan, S.; LaMarre, E.; Cornell, W. C.; Janakiraman, A.; Min, W.; Tomer, R.; Price-Whelan, A.; Nirody, J. A.; Dietrich, L. E.</p>
        <p class="info">Score: 22.5, Published: 2023-06-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.20.545666' target='https://doi.org/10.1101/2023.06.20.545666'> 10.1101/2023.06.20.545666</a></p>
        <p class="abstract">Cells must access resources to survive, and the anatomy of multicellular structures influences this access. In diverse multicellular eukaryotes, resources are provided by internal conduits that allow substances to travel more readily through tissue than they would via diffusion. Microbes growing in multicellular structures, called biofilms, are also affected by differential access to resources and we hypothesized that this is influenced by the physical arrangement of the cells. In this study, we examined the microanatomy of biofilms formed by the pathogenic bacterium Pseudomonas aeruginosa and discovered that clonal cells form striations that are packed lengthwise across most of a mature biofilms depth. We identified mutants, including those defective in pilus function and in O-antigen attachment, that show alterations to this lengthwise packing phenotype. Consistent with the notion that cellular arrangement affects access to resources within the biofilm, we found that while the wild type shows even distribution of tested substrates across depth, the mutants show accumulation of substrates at the biofilm boundaries. Furthermore, we found that altered cellular arrangement within biofilms affects the localization of metabolic activity, the survival of resident cells, and the susceptibility of subpopulations to antibiotic treatment. Our observations provide insight into cellular features that determine biofilm microanatomy, with consequences for physiological differentiation and drug sensitivity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.544637">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.544637" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.544637">
        <p class="paperTitle">High-accuracy meets high-throughput for microbiome profiling with near full-length 16S rRNA amplicon sequencing on the Nanopore platform</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.544637" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.544637" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lin, X.; Waring, K.; Tyson, J.; Ziels, R.</p>
        <p class="info">Score: 38.4, Published: 2023-06-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.544637' target='https://doi.org/10.1101/2023.06.19.544637'> 10.1101/2023.06.19.544637</a></p>
        <p class="abstract">Amplicon sequencing of small subunit (SSU) rRNA genes is a foundational method for studying microbial communities within various environmental, human, and engineered ecosystems. Currently, short-read platforms are commonly employed for high-throughput applications of SSU rRNA amplicon sequencing, but at the cost of poor taxonomic classification. The low-cost Oxford Nanopore Technologies (ONT) platform is capable of sequencing full-length SSU rRNA genes, but the lower raw-read accuracies of previous ONT sequencing chemistries have limited accurate taxonomic classification and de novo generation of operational taxonomic units (OTUs) and amplicon sequence variants (ASVs). Here, we examine the potential for Nanopore sequencing with newer (R10.4&#43;) chemistry to provide high-throughput and high-accuracy full-length 16S rRNA gene amplicon sequencing. We present a sequencing workflow utilizing unique molecular identifiers (UMIs) for error-correction of SSU rRNA (e.g. 16S rRNA) gene amplicons, termed ssUMI. Using two synthetic microbial community standards, the ssUMI workflow generated consensus sequences with 99.99% mean accuracy using a minimum UMI subread coverage threshold of 3x, and was capable of generating error-free ASVs and 97% OTUs with no false-positives. Non-corrected Nanopore reads generated error-free 97% OTUs but with reduced detection sensitivity, and also generated false-positive ASVs. We showcase the cost-competitive and high-throughput scalability of the ssUMI workflow by sequencing 90 time-series samples from seven different wastewater matrices, generating ASVs that were tightly clustered based on sample matrix type. This work demonstrates that highly accurate full-length 16S rRNA gene amplicon sequencing on Nanopore is possible, paving the way to more accessible microbiome science.</p>
      </div>
    </div>
  </div>
</div>









<script src="/TRxiv2/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://yorks0n.github.io/TRxiv2">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
