<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Virological characteristics of the SARS-CoV-2 BA.2.86 variant
Authors: Tamura, T.; Mizuma, K.; Nasser, H.; Deguchi, S.; Padilla-Blanco, M.; Uriu, K.; Tolentino, J. E. M.; Tsujino, S.; Suzuki, R.; Kojima, I.; Nao, N.; Shimizu, R.; Jonathan, M.; Kosugi, Y.; Guo, Z.; Hinay, A. A.; Putri, O.; Kim, Y.; Tanaka, Y. L.; Asakura, H.; Nagashima, M.; Sadamasu, K.; Yoshimura, K.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Saito, A.; Ito, J.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Virological characteristics of the SARS-CoV-2 BA.2.86 variant
Authors: Tamura, T.; Mizuma, K.; Nasser, H.; Deguchi, S.; Padilla-Blanco, M.; Uriu, K.; Tolentino, J. E. M.; Tsujino, S.; Suzuki, R.; Kojima, I.; Nao, N.; Shimizu, R.; Jonathan, M.; Kosugi, Y.; Guo, Z.; Hinay, A. A.; Putri, O.; Kim, Y.; Tanaka, Y. L.; Asakura, H.; Nagashima, M.; Sadamasu, K.; Yoshimura, K.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Saito, A.; Ito, J." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-08T10:39:23+00:00" />
<meta property="article:modified_time" content="2023-11-08T10:39:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Virological characteristics of the SARS-CoV-2 BA.2.86 variant
Authors: Tamura, T.; Mizuma, K.; Nasser, H.; Deguchi, S.; Padilla-Blanco, M.; Uriu, K.; Tolentino, J. E. M.; Tsujino, S.; Suzuki, R.; Kojima, I.; Nao, N.; Shimizu, R.; Jonathan, M.; Kosugi, Y.; Guo, Z.; Hinay, A. A.; Putri, O.; Kim, Y.; Tanaka, Y. L.; Asakura, H.; Nagashima, M.; Sadamasu, K.; Yoshimura, K.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Saito, A.; Ito, J."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Virological characteristics of the SARS-CoV-2 BA.2.86 variant\nAuthors: Tamura, T.; Mizuma, K.; Nasser, H.; Deguchi, S.; Padilla-Blanco, M.; Uriu, K.; Tolentino, J. E. M.; Tsujino, S.; Suzuki, R.; Kojima, I.; Nao, N.; Shimizu, R.; Jonathan, M.; Kosugi, Y.; Guo, Z.; Hinay, A. A.; Putri, O.; Kim, Y.; Tanaka, Y. L.; Asakura, H.; Nagashima, M.; Sadamasu, K.; Yoshimura, K.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Saito, A.; Ito, J.",
  "keywords": [
    
  ],
  "articleBody": " Virological characteristics of the SARS-CoV-2 BA.2.86 variant\nAuthors: Tamura, T.; Mizuma, K.; Nasser, H.; Deguchi, S.; Padilla-Blanco, M.; Uriu, K.; Tolentino, J. E. M.; Tsujino, S.; Suzuki, R.; Kojima, I.; Nao, N.; Shimizu, R.; Jonathan, M.; Kosugi, Y.; Guo, Z.; Hinay, A. A.; Putri, O.; Kim, Y.; Tanaka, Y. L.; Asakura, H.; Nagashima, M.; Sadamasu, K.; Yoshimura, K.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Saito, A.; Ito, J.; Irie, T.; Zahradnik, J.; Ikeda, T.; Takayama, K.; Matsuno, K.; Fukuhara, T.; Sato, K.\nScore: 323.8, Published: 2023-11-03 DOI: 10.1101/2023.11.02.565304\nIn late 2023, a lineage of SARS-CoV-2 emerged and was named the BA.2.86 variant. BA.2.86 is phylogenetically distinct from other Omicron sublineages identified so far, displaying an accumulation of over 30 amino acid mutations in its spike protein. Here, we performed multiscale investigations to reveal the virological characteristics of the BA.2.86 variant. Our epidemic dynamics modeling suggested that the relative reproduction number of BA.2.86 is significantly higher than that of EG.5.1. Experimental studies showed that four clinically-available antivirals were effective against BA.2.86. Although the fusogenicity of BA.2.86 spike is similar to that of the parental BA.2 spike, the intrinsic pathogenicity of BA.2.86 in hamsters was significantly lower than that of BA.2. Since the growth kinetics of BA.2.86 is significantly lower than that of BA.2 in both in vitro cell cultures and in vivo, it is suggested that the attenuated pathogenicity of BA.2.86 is due to its decreased replication capacity.\nSARS-CoV-2 infection leads to sustained testicular injury and functional impairments in K18 hACE2 mice\nAuthors: Giannakopoulos, S.; Ward, M. A.; Bakse, J.; Pak, J. H.; Nerurkar, V. R.; Tallquist, M. D.; Verma, S.\nScore: 107.3, Published: 2023-11-01 DOI: 10.1101/2023.10.31.565042\nCompromised male reproductive health is one of the symptoms of long COVID with a decrease in male fertility markers including testosterone levels and sperm count for months in recovering patients. However, the long-term impact of SARS-CoV-2 infection on testicular injury and underlying mechanisms remains unknown. We previously demonstrated a disrupted tissue architecture with no evidence of virus replication in the testis during the acute stage of the disease in K18-hACE2 mice. Here, we systematically delineate the consequences of SARS-CoV-2 infection on the testis injury and function both during the acute stage of the disease and up to 4 weeks after infection in survivor K18-hACE2 mice. The gross morphological defects included sloughing of healthy spermatids and spermatocytes into the lumen, lack of lumen, and increase in apoptotic cells that sustained for at least 2 weeks after infection. Testis injury correlated with systemic and testicular inflammation, and infiltration of immune cells in the interstitial space and seminiferous tubules. Transcriptomic analysis identified dysregulation of key pathways of testicular immune homeostasis, spermatogenesis, and cell death at the symptomatic and short-term recovery stages. Further, a significant reduction in testosterone levels was associated with transient reduction in sperm count and mouse fertility. Most of the testicular impairments except testosterone levels were resolved within 4 weeks, which is almost one spermatogenesis cycle in mice. These findings provide much-needed mechanistic insights beyond our current understanding of testicular pathogenesis, suggesting that recovering COVID-19 patients should be closely monitored to rescue the pathophysiological effects on male reproductive health.\nSynergistic anti-phage activity of bacterial defence systems\nAuthors: Wu, Y.; Garushyants, S. K.; van den Hurk, A.; Aparicio-Maldonado, C.; Kushwaha, S. K.; King, C. M.; Ou, Y.; Todeschini, T. C.; Clokie, M. R. J.; Millard, A. M.; Gencay, Y. E.; Koonin, E. V.; Nobrega, F. L.\nScore: 56.0, Published: 2023-11-06 DOI: 10.1101/2022.08.21.504612\nBacterial defence against phage predation involves diverse defence systems acting both individually and concurrently, yet their interactions remain poorly understood. We investigated \u003e100 defence systems in 42,925 bacterial genomes and identified numerous instances of their non-random co-occurrence and mutual exclusion. For several pairs of defence systems significantly co-occurring in Escherichia coli strains, we demonstrate synergistic anti-phage activity, including tmn synergising with defence systems containing sensory switch ATPase domains. Some of the defence system pairs that are mutually exclusive in E. coli significantly co-occur in other taxa, and one tested mutually exclusive pair showed synergy. These findings demonstrate compatibility and synergy between defence systems, allowing bacteria to adopt flexible strategies for phage defence, driven by specific environmental pressures. The evolution of bacterial immune repertoires seems to be shaped largely by selection for resistance to host-specific phages rather than by negative epistasis.\nA virally encoded high resolution screen of cytomegalovirus host dependencies\nAuthors: Finkel, Y.; Nachshon, A.; Aharon, E.; Arazi, T.; Simonovsky, E.; Dobesova, M.; Gluck, A.; Fisher, T.; Stanton, R. J.; Schwartz, M.; Stern-Ginossar, N.\nScore: 19.8, Published: 2023-10-30 DOI: 10.1101/2023.10.30.564793\nGenetic screens have transformed our ability to interrogate cellular factor requirements in infection, yet current approaches are limited in their sensitivity, biased towards early stages of infection and provide only simplistic phenotypic information which is often based on infected cell survival. Here, by engineering human cytomegalovirus to express sgRNA libraries directly from the viral genome, we developed a sensitive, versatile, viral centric approach that allows profiling of different stages along viral infection in a pooled format. Using this approach, which we termed VECOS (Virus Encoded CRISPR-based direct readOut Screening system), we identified hundreds of novel host dependency and restriction factors and quantified their direct effects on viral genome replication, viral particle secretion and infectiousness of secreted particles, providing a multi-dimensional perspective on viral-host interactions. These high resolution measurements reveal that perturbations that alter late stages in HCMV life cycle mostly regulate HCMV particle quality rather than quantity, defining correct virion assembly as a critical stage that is heavily reliant on viral-host interactions. Overall, VECOS facilitates systematic high resolution dissection of human proteins role along the infection cycle, providing a roadmap for in-depth dissection of host-herpesvirus interactions.\nA global survey of small RNA interactors identifies KhpA and KhpB as major RNA-binding proteins in Fusobacterium nucleatum\nAuthors: Zhu, Y.; Ponath, F.; Cosi, V.; Vogel, J.\nScore: 22.2, Published: 2023-10-30 DOI: 10.1101/2023.10.30.564711\nABSTRACTThe common oral microbe Fusobacterium nucleatum has recently drawn attention after it was found to colonize tumors throughout the human body. Fusobacteria are also interesting study systems for bacterial RNA biology as these early-branching species encode many small noncoding RNAs (sRNAs) but lack homologs of the common RNA-binding proteins (RBPs) CsrA, Hfq and ProQ. To search for alternate sRNA-associated RBPs in F. nucleatum, we performed a systematic mass spectrometry analysis of proteins that co-purified with 19 different sRNAs. This approach revealed strong enrichment of the KH domain proteins KhpA and KhpB with nearly all tested sRNAs, including the {sigma}E-dependent sRNA FoxI, a regulator of several envelope proteins. KhpA/B act as a dimer to bind sRNAs with low micromolar affinity and influence the stability of several of their target transcripts. Transcriptome studies combined with biochemical and genetic analyses suggest that KhpA/B have several physiological functions, including being required for ethanolamine utilization. Our RBP search and the discovery of KhpA and KhpB as major RBPs in F. nucleatum are important first steps in identifying key players of post-transcriptional control at the root of the bacterial phylogenetic tree.\nCas13-based amplification-free quantification of aberrant RNAs during influenza virus infection\nAuthors: Lamb, C.; Pitre, E.; Elshina, E.; Rigby, C.; Bisht, K.; Jalal, H.; Myhrvold, C.; te Velthuis, A. J. W.\nScore: 14.3, Published: 2023-11-03 DOI: 10.1101/2023.11.03.565460\nInfluenza A virus RNA synthesis produces full-length and aberrant RNA molecules, which include defective viral genomes (DVG) and mini viral RNAs (mvRNA). Sequencing approaches have shown that several hundred unique aberrant RNA species may be present during infection, and that they can vary in size, segment origin, and sequence. Moreover, a subset of aberrant RNA molecules can bind and activate host pathogen receptor retinoic acid-inducible gene I (RIG-I), leading to innate immune signaling and the expression of type I and III interferons. Understanding the kinetics and distribution of these immunostimulatory aberrant RNA sequences is important for modeling the outcomes of IAV infection. We here first show that reverse transcription and PCR steps can yield imperfect aberrant RNA quantification data in a sequence-dependent manner. Next, we developed an amplification-free LbuCas13a-based detection method to quantify mvRNA amplification kinetics and subcellular distributions. We show that our assay can quantify the copy numbers of 10 specific mvRNA sequences in total RNA from cell culture, animal tissue or clinical nasopharyngeal swab extracts. In addition, we find kinetic and distribution differences between immunostimulatory and non-immunostimulatory mvRNAs, as well as mvRNAs derived from different segments, during infection. Overall, our results reveal a hitherto hidden diversity in the behavior of IAV mvRNAs and they suggest that their production is linked to replication of the individual viral segments. Cas13 is therefore a valuable new tool in our repertoire for investigating the impact of aberrant RNAs on RNA virus infection.\nMultidrug resistance plasmids commonly reprogramme expression of metabolic genes in Escherichia coli\nAuthors: Hall, R. J.; Snaith, A. E.; Thomas, M.; Brockhurst, M.; McNally, A.\nScore: 13.0, Published: 2023-11-06 DOI: 10.1101/2023.11.06.565804\nMultidrug resistant Escherichia coli is a leading cause of global mortality. Transfer of plasmids carrying genes encoding beta-lactamases, carbapenamases, and colistin resistance genes between lineages is driving the rising rates of hard to treat nosocomial and community infections. Multidrug resistance (MDR) plasmid acquisition commonly causes transcriptional disruption, and whilst a number of studies have shown strain-specific fitness and transcriptional effects of an MDR plasmid across diverse bacterial lineages, fewer studies have compared impacts of different MDR plasmids in a common bacterial host. As such, our ability to predict which MDR plasmids are the most likely to be maintained and spread in bacterial populations is limited. Here, we introduced eight diverse MDR plasmids encoding resistances against a range of clinically important antibiotics into E. coli K-12 MG1655 and measured their fitness costs and transcriptional impacts. The scale of the transcriptional responses varied substantially between plasmids, ranging from \u003e650 to \u003c20 chromosomal genes being differentially expressed. However, neither the scale of regulatory disruption nor the plasmid size correlated with the magnitude of the plasmid fitness cost, which also varied between plasmids. The identities of differentially expressed genes varied among plasmids, although expression of certain metabolic genes and functions were convergently affected by multiple plasmids, including the downregulation of genes involved in L-methionine transport and metabolism. Our data show the complexity of interaction between host genetic background and plasmid genetic background in determining the impact of MDR plasmid acquisition on E. coli.\nIdentification of novel tail-anchored membrane proteins integrated by the bacterial twin-arginine translocase\nAuthors: Gallego-Parrilla, J. J.; SEVERI, E.; Chandra, G.; Palmer, T.\nScore: 12.7, Published: 2023-11-02 DOI: 10.1101/2023.11.02.564894\nThe twin arginine protein transport (Tat) system exports folded proteins across the cytoplasmic membranes of prokaryotes and the energy transducing membranes of plant thylakoids and mitochondria. Proteins are targeted to the Tat machinery by N-terminal signal peptides with a conserved twin arginine motif, and some substrates are exported as heterodimers where the signal peptide is present on one of the partner proteins. A subset of Tat substrates is found in the membrane. Tat-dependent membrane proteins usually have large globular domains and a single transmembrane helix present at the N- or C-terminus. Five Tat substrates that have C-terminal transmembrane helices have previously been characterised in the model bacterium Escherichia coli. Each of these is an iron-sulphur cluster-containing protein involved in electron transfer from hydrogen or formate. Here we have undertaken a bioinformatic search to identify further tail-anchored Tat substrates encoded in bacterial genomes. Our analysis has revealed additional tail-anchored iron-sulphur proteins associated in modules with either a b-type cytochrome or a quinol oxidase. We also identified further candidate tail-anchored Tat substrates, particularly among members of the actinobacterial genus, that are not predicted to contain cofactors. Using reporter assays we show experimentally that six of these have both N-terminal Tat signal peptides and C-terminal transmembrane helices. The newly-identified proteins include a carboxypeptidase and a predicted protease, and four sortase substrates for which membrane integration is a pre-requisite for covalent attachment to the cell wall.\nDesiccated cyanobacteria serve as efficient plasmid DNA carrier in space flight\nAuthors: Kakouridis, A.; Diamond, S.; Eng, T.; Mills, H. J.; Gamez-Holzhaus, O.; Summers, M. L.; Garcia-Pichel, F.; Mukhopadhyay, A.\nScore: 12.3, Published: 2023-11-01 DOI: 10.1101/2023.10.31.564425\nEffective transport of biological systems as cargo during space travel is a critical requirement to use synthetic biology and biomanufacturing in outer space. Bioproduction using microbes will drive the extent to which many human needs can be met in environments with limited resources. Vast repositories of biological parts and strains are available to meet this need, but their on-site availability requires effective transport. Here, we explore an approach that allows DNA plasmids, a ubiquitous synthetic biology part, to be safely transported to the International Space Station and back to the Kennedy Space center, without low-temperature or cryogenic stowage. Our approach relied on the cyanobacterium Nostoc punctiforme PC73102, that is naturally tolerant to prolonged desiccation. Desiccated N. punctiforme was able to carry the non-native pSCR119 plasmid as intracellular cargo safely to space and back. Upon return to the laboratory, the extracted plasmid showed no DNA damage or additional mutations and could be used as intended to transform the model Synbio host Escherichia coli to bestow Kanamycin resistance. This proof-of-concept study provides the foundation for a ruggedized transport host for DNA to environments where there is a need to reduce equipment and infrastructure for biological parts stowage and storage.\nQueuosine biosynthetic enzyme, QueE moonlights as a cell division regulator\nAuthors: Adeleye, S. A.; Yadavalli, S. S.\nScore: 11.0, Published: 2023-11-01 DOI: 10.1101/2023.10.31.565030\nIn many organisms, stress responses to adverse environments can trigger secondary functions of certain proteins by altering protein levels, localization, activity, or interaction partners. Escherichia coli cells respond to the presence of specific cationic antimicrobial peptides by strongly activating the PhoQ/PhoP two-component signaling system, which regulates genes important for growth under this stress. As part of this pathway, a biosynthetic enzyme called QueE, which catalyzes a step in the formation of queuosine (Q) tRNA modification is upregulated. When cellular QueE levels are high, it co-localizes with the central cell division protein FtsZ at the septal site, blocking division and resulting in filamentous growth. Here we show that QueE affects cell size in a dose-dependent manner. Using alanine scanning mutagenesis of amino acids in the catalytic active site, we pinpoint particular residues in QueE that contribute distinctly to each of its functions - Q biosynthesis or regulation of cell division, establishing QueE as a moonlighting protein. We further show that QueE orthologs from enterobacteria like Salmonella typhimurium and Klebsiella pneumoniae also cause filamentation in these organisms, but the more distant counterparts from Pseudomonas aeruginosa and Bacillus subtilis lack this ability. By comparative analysis of E. coli QueE with distant orthologs, we elucidate a unique region in this protein that is responsible for QueEs secondary function as a cell division regulator. A dual-function protein like QueE is an exception to the conventional model of \"one gene, one enzyme, one function\", which has divergent roles across a range of fundamental cellular processes including RNA modification and translation to cell division and stress response.\n",
  "wordCount" : "2484",
  "inLanguage": "en",
  "datePublished": "2023-11-08T10:39:23Z",
  "dateModified": "2023-11-08T10:39:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 8, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.565304">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.565304" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.565304">
        <p class="paperTitle">Virological characteristics of the SARS-CoV-2 BA.2.86 variant</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.565304" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.565304" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tamura, T.; Mizuma, K.; Nasser, H.; Deguchi, S.; Padilla-Blanco, M.; Uriu, K.; Tolentino, J. E. M.; Tsujino, S.; Suzuki, R.; Kojima, I.; Nao, N.; Shimizu, R.; Jonathan, M.; Kosugi, Y.; Guo, Z.; Hinay, A. A.; Putri, O.; Kim, Y.; Tanaka, Y. L.; Asakura, H.; Nagashima, M.; Sadamasu, K.; Yoshimura, K.; The Genotype to Phenotype Japan (G2P-Japan) Consortium,  ; Saito, A.; Ito, J.; Irie, T.; Zahradnik, J.; Ikeda, T.; Takayama, K.; Matsuno, K.; Fukuhara, T.; Sato, K.</p>
        <p class="info">Score: 323.8, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.565304' target='https://doi.org/10.1101/2023.11.02.565304'> 10.1101/2023.11.02.565304</a></p>
        <p class="abstract">In late 2023, a lineage of SARS-CoV-2 emerged and was named the BA.2.86 variant. BA.2.86 is phylogenetically distinct from other Omicron sublineages identified so far, displaying an accumulation of over 30 amino acid mutations in its spike protein. Here, we performed multiscale investigations to reveal the virological characteristics of the BA.2.86 variant. Our epidemic dynamics modeling suggested that the relative reproduction number of BA.2.86 is significantly higher than that of EG.5.1. Experimental studies showed that four clinically-available antivirals were effective against BA.2.86. Although the fusogenicity of BA.2.86 spike is similar to that of the parental BA.2 spike, the intrinsic pathogenicity of BA.2.86 in hamsters was significantly lower than that of BA.2. Since the growth kinetics of BA.2.86 is significantly lower than that of BA.2 in both in vitro cell cultures and in vivo, it is suggested that the attenuated pathogenicity of BA.2.86 is due to its decreased replication capacity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.565042">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.565042" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.565042">
        <p class="paperTitle">SARS-CoV-2 infection leads to sustained testicular injury and functional impairments in K18 hACE2 mice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.565042" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.565042" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Giannakopoulos, S.; Ward, M. A.; Bakse, J.; Pak, J. H.; Nerurkar, V. R.; Tallquist, M. D.; Verma, S.</p>
        <p class="info">Score: 107.3, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.565042' target='https://doi.org/10.1101/2023.10.31.565042'> 10.1101/2023.10.31.565042</a></p>
        <p class="abstract">Compromised male reproductive health is one of the symptoms of long COVID with a decrease in male fertility markers including testosterone levels and sperm count for months in recovering patients. However, the long-term impact of SARS-CoV-2 infection on testicular injury and underlying mechanisms remains unknown. We previously demonstrated a disrupted tissue architecture with no evidence of virus replication in the testis during the acute stage of the disease in K18-hACE2 mice. Here, we systematically delineate the consequences of SARS-CoV-2 infection on the testis injury and function both during the acute stage of the disease and up to 4 weeks after infection in survivor K18-hACE2 mice. The gross morphological defects included sloughing of healthy spermatids and spermatocytes into the lumen, lack of lumen, and increase in apoptotic cells that sustained for at least 2 weeks after infection. Testis injury correlated with systemic and testicular inflammation, and infiltration of immune cells in the interstitial space and seminiferous tubules. Transcriptomic analysis identified dysregulation of key pathways of testicular immune homeostasis, spermatogenesis, and cell death at the symptomatic and short-term recovery stages. Further, a significant reduction in testosterone levels was associated with transient reduction in sperm count and mouse fertility. Most of the testicular impairments except testosterone levels were resolved within 4 weeks, which is almost one spermatogenesis cycle in mice. These findings provide much-needed mechanistic insights beyond our current understanding of testicular pathogenesis, suggesting that recovering COVID-19 patients should be closely monitored to rescue the pathophysiological effects on male reproductive health.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.08.21.504612">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.08.21.504612" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.08.21.504612">
        <p class="paperTitle">Synergistic anti-phage activity of bacterial defence systems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.08.21.504612" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.08.21.504612" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wu, Y.; Garushyants, S. K.; van den Hurk, A.; Aparicio-Maldonado, C.; Kushwaha, S. K.; King, C. M.; Ou, Y.; Todeschini, T. C.; Clokie, M. R. J.; Millard, A. M.; Gencay, Y. E.; Koonin, E. V.; Nobrega, F. L.</p>
        <p class="info">Score: 56.0, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.08.21.504612' target='https://doi.org/10.1101/2022.08.21.504612'> 10.1101/2022.08.21.504612</a></p>
        <p class="abstract">Bacterial defence against phage predation involves diverse defence systems acting both individually and concurrently, yet their interactions remain poorly understood. We investigated &gt;100 defence systems in 42,925 bacterial genomes and identified numerous instances of their non-random co-occurrence and mutual exclusion. For several pairs of defence systems significantly co-occurring in Escherichia coli strains, we demonstrate synergistic anti-phage activity, including tmn synergising with defence systems containing sensory switch ATPase domains. Some of the defence system pairs that are mutually exclusive in E. coli significantly co-occur in other taxa, and one tested mutually exclusive pair showed synergy. These findings demonstrate compatibility and synergy between defence systems, allowing bacteria to adopt flexible strategies for phage defence, driven by specific environmental pressures. The evolution of bacterial immune repertoires seems to be shaped largely by selection for resistance to host-specific phages rather than by negative epistasis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564793">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564793" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564793">
        <p class="paperTitle">A virally encoded high resolution screen of cytomegalovirus host dependencies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564793" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564793" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Finkel, Y.; Nachshon, A.; Aharon, E.; Arazi, T.; Simonovsky, E.; Dobesova, M.; Gluck, A.; Fisher, T.; Stanton, R. J.; Schwartz, M.; Stern-Ginossar, N.</p>
        <p class="info">Score: 19.8, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564793' target='https://doi.org/10.1101/2023.10.30.564793'> 10.1101/2023.10.30.564793</a></p>
        <p class="abstract">Genetic screens have transformed our ability to interrogate cellular factor requirements in infection, yet current approaches are limited in their sensitivity, biased towards early stages of infection and provide only simplistic phenotypic information which is often based on infected cell survival. Here, by engineering human cytomegalovirus to express sgRNA libraries directly from the viral genome, we developed a sensitive, versatile, viral centric approach that allows profiling of different stages along viral infection in a pooled format. Using this approach, which we termed VECOS (Virus Encoded CRISPR-based direct readOut Screening system), we identified hundreds of novel host dependency and restriction factors and quantified their direct effects on viral genome replication, viral particle secretion and infectiousness of secreted particles, providing a multi-dimensional perspective on viral-host interactions. These high resolution measurements reveal that perturbations that alter late stages in HCMV life cycle mostly regulate HCMV particle quality rather than quantity, defining correct virion assembly as a critical stage that is heavily reliant on viral-host interactions. Overall, VECOS facilitates systematic high resolution dissection of human proteins role along the infection cycle, providing a roadmap for in-depth dissection of host-herpesvirus interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564711">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564711" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564711">
        <p class="paperTitle">A global survey of small RNA interactors identifies KhpA and KhpB as major RNA-binding proteins in Fusobacterium nucleatum</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564711" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564711" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhu, Y.; Ponath, F.; Cosi, V.; Vogel, J.</p>
        <p class="info">Score: 22.2, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564711' target='https://doi.org/10.1101/2023.10.30.564711'> 10.1101/2023.10.30.564711</a></p>
        <p class="abstract">ABSTRACTThe common oral microbe Fusobacterium nucleatum has recently drawn attention after it was found to colonize tumors throughout the human body. Fusobacteria are also interesting study systems for bacterial RNA biology as these early-branching species encode many small noncoding RNAs (sRNAs) but lack homologs of the common RNA-binding proteins (RBPs) CsrA, Hfq and ProQ. To search for alternate sRNA-associated RBPs in F. nucleatum, we performed a systematic mass spectrometry analysis of proteins that co-purified with 19 different sRNAs. This approach revealed strong enrichment of the KH domain proteins KhpA and KhpB with nearly all tested sRNAs, including the {sigma}E-dependent sRNA FoxI, a regulator of several envelope proteins. KhpA/B act as a dimer to bind sRNAs with low micromolar affinity and influence the stability of several of their target transcripts. Transcriptome studies combined with biochemical and genetic analyses suggest that KhpA/B have several physiological functions, including being required for ethanolamine utilization. Our RBP search and the discovery of KhpA and KhpB as major RBPs in F. nucleatum are important first steps in identifying key players of post-transcriptional control at the root of the bacterial phylogenetic tree.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.03.565460">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.03.565460" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.03.565460">
        <p class="paperTitle">Cas13-based amplification-free quantification of aberrant RNAs during influenza virus infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.03.565460" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.03.565460" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lamb, C.; Pitre, E.; Elshina, E.; Rigby, C.; Bisht, K.; Jalal, H.; Myhrvold, C.; te Velthuis, A. J. W.</p>
        <p class="info">Score: 14.3, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.03.565460' target='https://doi.org/10.1101/2023.11.03.565460'> 10.1101/2023.11.03.565460</a></p>
        <p class="abstract">Influenza A virus RNA synthesis produces full-length and aberrant RNA molecules, which include defective viral genomes (DVG) and mini viral RNAs (mvRNA). Sequencing approaches have shown that several hundred unique aberrant RNA species may be present during infection, and that they can vary in size, segment origin, and sequence. Moreover, a subset of aberrant RNA molecules can bind and activate host pathogen receptor retinoic acid-inducible gene I (RIG-I), leading to innate immune signaling and the expression of type I and III interferons. Understanding the kinetics and distribution of these immunostimulatory aberrant RNA sequences is important for modeling the outcomes of IAV infection. We here first show that reverse transcription and PCR steps can yield imperfect aberrant RNA quantification data in a sequence-dependent manner. Next, we developed an amplification-free LbuCas13a-based detection method to quantify mvRNA amplification kinetics and subcellular distributions. We show that our assay can quantify the copy numbers of 10 specific mvRNA sequences in total RNA from cell culture, animal tissue or clinical nasopharyngeal swab extracts. In addition, we find kinetic and distribution differences between immunostimulatory and non-immunostimulatory mvRNAs, as well as mvRNAs derived from different segments, during infection. Overall, our results reveal a hitherto hidden diversity in the behavior of IAV mvRNAs and they suggest that their production is linked to replication of the individual viral segments. Cas13 is therefore a valuable new tool in our repertoire for investigating the impact of aberrant RNAs on RNA virus infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565804">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565804" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565804">
        <p class="paperTitle">Multidrug resistance plasmids commonly reprogramme expression of metabolic genes in Escherichia coli</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565804" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565804" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hall, R. J.; Snaith, A. E.; Thomas, M.; Brockhurst, M.; McNally, A.</p>
        <p class="info">Score: 13.0, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565804' target='https://doi.org/10.1101/2023.11.06.565804'> 10.1101/2023.11.06.565804</a></p>
        <p class="abstract">Multidrug resistant Escherichia coli is a leading cause of global mortality. Transfer of plasmids carrying genes encoding beta-lactamases, carbapenamases, and colistin resistance genes between lineages is driving the rising rates of hard to treat nosocomial and community infections. Multidrug resistance (MDR) plasmid acquisition commonly causes transcriptional disruption, and whilst a number of studies have shown strain-specific fitness and transcriptional effects of an MDR plasmid across diverse bacterial lineages, fewer studies have compared impacts of different MDR plasmids in a common bacterial host. As such, our ability to predict which MDR plasmids are the most likely to be maintained and spread in bacterial populations is limited. Here, we introduced eight diverse MDR plasmids encoding resistances against a range of clinically important antibiotics into E. coli K-12 MG1655 and measured their fitness costs and transcriptional impacts. The scale of the transcriptional responses varied substantially between plasmids, ranging from &gt;650 to &lt;20 chromosomal genes being differentially expressed. However, neither the scale of regulatory disruption nor the plasmid size correlated with the magnitude of the plasmid fitness cost, which also varied between plasmids. The identities of differentially expressed genes varied among plasmids, although expression of certain metabolic genes and functions were convergently affected by multiple plasmids, including the downregulation of genes involved in L-methionine transport and metabolism. Our data show the complexity of interaction between host genetic background and plasmid genetic background in determining the impact of MDR plasmid acquisition on E. coli.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.564894">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.564894" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.564894">
        <p class="paperTitle">Identification of novel tail-anchored membrane proteins integrated by the bacterial twin-arginine translocase</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.564894" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.564894" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gallego-Parrilla, J. J.; SEVERI, E.; Chandra, G.; Palmer, T.</p>
        <p class="info">Score: 12.7, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.564894' target='https://doi.org/10.1101/2023.11.02.564894'> 10.1101/2023.11.02.564894</a></p>
        <p class="abstract">The twin arginine protein transport (Tat) system exports folded proteins across the cytoplasmic membranes of prokaryotes and the energy transducing membranes of plant thylakoids and mitochondria. Proteins are targeted to the Tat machinery by N-terminal signal peptides with a conserved twin arginine motif, and some substrates are exported as heterodimers where the signal peptide is present on one of the partner proteins. A subset of Tat substrates is found in the membrane. Tat-dependent membrane proteins usually have large globular domains and a single transmembrane helix present at the N- or C-terminus. Five Tat substrates that have C-terminal transmembrane helices have previously been characterised in the model bacterium Escherichia coli. Each of these is an iron-sulphur cluster-containing protein involved in electron transfer from hydrogen or formate. Here we have undertaken a bioinformatic search to identify further tail-anchored Tat substrates encoded in bacterial genomes. Our analysis has revealed additional tail-anchored iron-sulphur proteins associated in modules with either a b-type cytochrome or a quinol oxidase. We also identified further candidate tail-anchored Tat substrates, particularly among members of the actinobacterial genus, that are not predicted to contain cofactors. Using reporter assays we show experimentally that six of these have both N-terminal Tat signal peptides and C-terminal transmembrane helices. The newly-identified proteins include a carboxypeptidase and a predicted protease, and four sortase substrates for which membrane integration is a pre-requisite for covalent attachment to the cell wall.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.564425">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.564425" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.564425">
        <p class="paperTitle">Desiccated cyanobacteria serve as efficient plasmid DNA carrier in space flight</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.564425" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.564425" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kakouridis, A.; Diamond, S.; Eng, T.; Mills, H. J.; Gamez-Holzhaus, O.; Summers, M. L.; Garcia-Pichel, F.; Mukhopadhyay, A.</p>
        <p class="info">Score: 12.3, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.564425' target='https://doi.org/10.1101/2023.10.31.564425'> 10.1101/2023.10.31.564425</a></p>
        <p class="abstract">Effective transport of biological systems as cargo during space travel is a critical requirement to use synthetic biology and biomanufacturing in outer space. Bioproduction using microbes will drive the extent to which many human needs can be met in environments with limited resources. Vast repositories of biological parts and strains are available to meet this need, but their on-site availability requires effective transport. Here, we explore an approach that allows DNA plasmids, a ubiquitous synthetic biology part, to be safely transported to the International Space Station and back to the Kennedy Space center, without low-temperature or cryogenic stowage. Our approach relied on the cyanobacterium Nostoc punctiforme PC73102, that is naturally tolerant to prolonged desiccation. Desiccated N. punctiforme was able to carry the non-native pSCR119 plasmid as intracellular cargo safely to space and back. Upon return to the laboratory, the extracted plasmid showed no DNA damage or additional mutations and could be used as intended to transform the model Synbio host Escherichia coli to bestow Kanamycin resistance. This proof-of-concept study provides the foundation for a ruggedized transport host for DNA to environments where there is a need to reduce equipment and infrastructure for biological parts stowage and storage.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.565030">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.565030" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.565030">
        <p class="paperTitle">Queuosine biosynthetic enzyme, QueE moonlights as a cell division regulator</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.565030" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.565030" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Adeleye, S. A.; Yadavalli, S. S.</p>
        <p class="info">Score: 11.0, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.565030' target='https://doi.org/10.1101/2023.10.31.565030'> 10.1101/2023.10.31.565030</a></p>
        <p class="abstract">In many organisms, stress responses to adverse environments can trigger secondary functions of certain proteins by altering protein levels, localization, activity, or interaction partners. Escherichia coli cells respond to the presence of specific cationic antimicrobial peptides by strongly activating the PhoQ/PhoP two-component signaling system, which regulates genes important for growth under this stress. As part of this pathway, a biosynthetic enzyme called QueE, which catalyzes a step in the formation of queuosine (Q) tRNA modification is upregulated. When cellular QueE levels are high, it co-localizes with the central cell division protein FtsZ at the septal site, blocking division and resulting in filamentous growth. Here we show that QueE affects cell size in a dose-dependent manner. Using alanine scanning mutagenesis of amino acids in the catalytic active site, we pinpoint particular residues in QueE that contribute distinctly to each of its functions - Q biosynthesis or regulation of cell division, establishing QueE as a moonlighting protein. We further show that QueE orthologs from enterobacteria like Salmonella typhimurium and Klebsiella pneumoniae also cause filamentation in these organisms, but the more distant counterparts from Pseudomonas aeruginosa and Bacillus subtilis lack this ability. By comparative analysis of E. coli QueE with distant orthologs, we elucidate a unique region in this protein that is responsible for QueEs secondary function as a cell division regulator. A dual-function protein like QueE is an exception to the conventional model of &#34;one gene, one enzyme, one function&#34;, which has divergent roles across a range of fundamental cellular processes including RNA modification and translation to cell division and stress response.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
