<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Viroid-like colonists of human microbiomes
Authors: Zheludev, I. N.; Edgar, R. C.; Galiano, M. J. L.; De la Pena, M.; Babaian, A.; Bhatt, A. S.; Fire, A. Z.
Score: 829.4, Published: 2024-01-21 DOI: 10.1101/2024.01.20.576352
Here, we describe the &#34;Obelisks,&#34; a previously unrecognised class of viroid-like elements that we first identified in human gut metatranscriptomic data. &#34;Obelisks&#34; share several properties: (i) apparently circular RNA [~]1kb genome assemblies, (ii) predicted rod-like secondary structures encompassing the entire genome, and (iii) open reading frames coding for a novel protein superfamily, which we call the &#34;Oblins&#34;.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Viroid-like colonists of human microbiomes
Authors: Zheludev, I. N.; Edgar, R. C.; Galiano, M. J. L.; De la Pena, M.; Babaian, A.; Bhatt, A. S.; Fire, A. Z.
Score: 829.4, Published: 2024-01-21 DOI: 10.1101/2024.01.20.576352
Here, we describe the &#34;Obelisks,&#34; a previously unrecognised class of viroid-like elements that we first identified in human gut metatranscriptomic data. &#34;Obelisks&#34; share several properties: (i) apparently circular RNA [~]1kb genome assemblies, (ii) predicted rod-like secondary structures encompassing the entire genome, and (iii) open reading frames coding for a novel protein superfamily, which we call the &#34;Oblins&#34;." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-14T10:38:48+00:00" />
<meta property="article:modified_time" content="2024-02-14T10:38:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Viroid-like colonists of human microbiomes
Authors: Zheludev, I. N.; Edgar, R. C.; Galiano, M. J. L.; De la Pena, M.; Babaian, A.; Bhatt, A. S.; Fire, A. Z.
Score: 829.4, Published: 2024-01-21 DOI: 10.1101/2024.01.20.576352
Here, we describe the &#34;Obelisks,&#34; a previously unrecognised class of viroid-like elements that we first identified in human gut metatranscriptomic data. &#34;Obelisks&#34; share several properties: (i) apparently circular RNA [~]1kb genome assemblies, (ii) predicted rod-like secondary structures encompassing the entire genome, and (iii) open reading frames coding for a novel protein superfamily, which we call the &#34;Oblins&#34;."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Viroid-like colonists of human microbiomes\nAuthors: Zheludev, I. N.; Edgar, R. C.; Galiano, M. J. L.; De la Pena, M.; Babaian, A.; Bhatt, A. S.; Fire, A. Z.\nScore: 829.4, Published: 2024-01-21 DOI: 10.1101/2024.01.20.576352\nHere, we describe the \u0026#34;Obelisks,\u0026#34; a previously unrecognised class of viroid-like elements that we first identified in human gut metatranscriptomic data. \u0026#34;Obelisks\u0026#34; share several properties: (i) apparently circular RNA [~]1kb genome assemblies, (ii) predicted rod-like secondary structures encompassing the entire genome, and (iii) open reading frames coding for a novel protein superfamily, which we call the \u0026#34;Oblins\u0026#34;.",
  "keywords": [
    
  ],
  "articleBody": " Viroid-like colonists of human microbiomes\nAuthors: Zheludev, I. N.; Edgar, R. C.; Galiano, M. J. L.; De la Pena, M.; Babaian, A.; Bhatt, A. S.; Fire, A. Z.\nScore: 829.4, Published: 2024-01-21 DOI: 10.1101/2024.01.20.576352\nHere, we describe the \"Obelisks,\" a previously unrecognised class of viroid-like elements that we first identified in human gut metatranscriptomic data. \"Obelisks\" share several properties: (i) apparently circular RNA [~]1kb genome assemblies, (ii) predicted rod-like secondary structures encompassing the entire genome, and (iii) open reading frames coding for a novel protein superfamily, which we call the \"Oblins\". We find that Obelisks form their own distinct phylogenetic group with no detectable sequence or structural similarity to known biological agents. Further, Obelisks are prevalent in tested human microbiome metatranscriptomes with representatives detected in [~]7% of analysed stool metatranscriptomes (29/440) and in [~]50% of analysed oral metatranscriptomes (17/32). Obelisk compositions appear to differ between the anatomic sites and are capable of persisting in individuals, with continued presence over \u003e300 days observed in one case. Large scale searches identified 29,959 Obelisks (clustered at 90% nucleotide identity), with examples from all seven continents and in diverse ecological niches. From this search, a subset of Obelisks are identified to code for Obelisk-specific variants of the hammerhead type-III self-cleaving ribozyme. Lastly, we identified one case of a bacterial species (Streptococcus sanguinis) in which a subset of defined laboratory strains harboured a specific Obelisk RNA population. As such, Obelisks comprise a class of diverse RNAs that have colonised, and gone unnoticed in, human, and global microbiomes.\nA 20-year time-series of a freshwater lake reveals seasonal dynamics and environmental drivers of viral diversity, ecology, and evolution\nAuthors: Zhou, Z.; Tran, P. Q.; Martin, C.; Rohwer, R. R.; Baker, B.; McMahon, K.; Anantharaman, K.\nScore: 43.6, Published: 2024-02-06 DOI: 10.1101/2024.02.06.579183\nLong-term ecological studies are powerful tools to investigate microbiomes and ecosystem change but have mostly ignored viruses. Here, we leveraged a 20-year time-series of a freshwater lake to characterize 1.3 million viral genomes over time, seasonality, and environmental factors. We identified 578 auxiliary metabolic gene (AMG) clusters representing over 150,000 AMGs, the most abundant of which, including psbA for photosynthesis, pmoC for methane oxidation, and katG for hydrogen peroxide decomposition, were consistently represented in viruses across decades and seasons. We observed positive associations and niche differentiation between virus-host pairs during seasonal change including in keystone taxa, Cyanobacteria, methanotrophs, and Nanopelagicales. Environmental constraints, specifically inorganic carbon and ammonium influenced viral abundances over time, and highlighted roles of viruses in both \"top-down\" and \"bottom-up\" interactions. Key evolutionary processes shaping gene and genome-wide selection included favored fitness genes, reduced genomic heterogeneity, and dominant sub-populations carrying specific genes. Overall, our study advances understanding of diversity, ecological dynamics, and evolutionary trajectories of viruses in Earths microbiomes and ecosystems.\nMapping glycoprotein structure reveals defining events in the evolution of the Flaviviridae\nAuthors: Mifsud, J. C. O.; Lytras, S.; Oliver, M. R.; Toon, K.; Costa, V. A.; Holmes, E. C.; Grove, J.\nScore: 38.4, Published: 2024-02-06 DOI: 10.1101/2024.02.06.579159\nViral glycoproteins drive membrane fusion in enveloped viruses and determine host range, tissue tropism and pathogenesis. Despite their importance, there is a fragmentary understanding of glycoproteins within the Flaviviridae; for many species the glycoproteins have not yet been identified, for others, such as the hepaciviruses, the molecular mechanisms of membrane fusion remain uncharacterised. Here, we combine comprehensive phylogenetic analyses with systematic protein structure prediction to survey glycoproteins across the entire Flaviviridae. We discover class-II fusion systems, homologous to the orthoflavivirus E glycoprotein, in most species, including highly-divergent jingmenviruses and large genome flaviviruses. However, the E1E2 glycoproteins of the hepaci-, pegi- and pestiviruses are structurally distinct, may represent a novel class of fusion mechanism, and are strictly associated with infection of vertebrate hosts. By mapping glycoprotein distribution onto the underlying phylogeny we reveal a complex history of evolutionary events that have shaped the diverse virology and ecology of the Flaviviridae.\nA new archaeal virus that suppresses the transcription of host immunity genes\nAuthors: Turgeman-Grott, I.; Golan, N.; Neri, U.; Naki, D.; Altman-Price, N.; Eizenshtein, K.; Choudhary, D. K.; Levy, R.; Papke, R. T.; Himani, H.; Reshef, L.; Gophna, U.\nScore: 27.1, Published: 2024-02-12 DOI: 10.1101/2024.02.12.579488\nIn some extreme environments, archaeal cells have been shown to have chronic viral infections, and such infections are well-tolerated by the hosts and may potentially protect against more lethal infections by lytic viruses. We have discovered that a natural Haloferax strain (48N), which is closely related to the model organism Haloferax volcanii, is chronically infected by a lemon-shaped virus, which we could purify from the medium. The chronic infection by this virus, which we named LSV-48N, is never cleared, despite the multiple defense systems of the host that include CRISPR-Cas, and two CBASS systems. Curing 48N of its virus by genetic engineering, led to radical changes in the gene expression profile of 48N and a dramatic improvement in its growth rate. Remarkably, the cured 48N is the fastest-growing haloarchaeon reported to date, with a generation time of approximately 1 hour at 45C compared to the typical 2.5 hours of H. volcanii or its infected isogen, and faster than any known haloarchaeon. The virus subverts host defenses by reducing their transcription and interfering with the CRISPR spacer acquisition machinery. Our results suggest that the slow growth of many halophilic archaea could be due to the effects of proviruses within their genomes that consume resources and alter the gene expression of their hosts.\nPhages reconstitute NAD+ to counter bacterial immunity\nAuthors: Osterman, I.; Samra, H.; Rousset, F.; Loseva, E.; Itkin, M.; Malitsky, S.; Yirmiya, E.; Millman, A.; Sorek, R.\nScore: 23.1, Published: 2024-02-13 DOI: 10.1101/2024.02.11.579819\nBacteria defend against phage infection via a variety of antiphage defense systems. Many defense systems were recently shown to deplete cellular nicotinamide adenine dinucleotide (NAD+) in response to infection, by breaking NAD+ to ADP-ribose (ADPR) and nicotinamide. It was demonstrated that NAD+ depletion during infection deprives the phage from this essential molecule and impedes phage replication. Here we show that a substantial fraction of phages possess enzymatic pathways allowing reconstitution of NAD+ from its degradation products in infected cells. We describe NAD+ reconstitution pathway 1 (NARP1), a two-step pathway in which one enzyme phosphorylates ADPR to generate ADPR-pyrophosphate (ADPR-PP), and the second enzyme conjugates ADPR-PP and nicotinamide to generate NAD+. Phages encoding the NARP1 pathway can overcome a diverse set of defense systems, including Thoeris, DSR1, DSR2, SIR2-HerA, and SEFIR, all of which deplete NAD+ as part of their defensive mechanism. Phylogenetic analyses show that NARP1 is primarily encoded on phage genomes, suggesting a phage-specific function in countering bacterial defenses. A second pathway, NARP2, allows phages to overcome bacterial defenses by building NAD+ via metabolites different than ADPR-PP. Our findings report a unique immune evasion strategy where viruses rebuild molecules depleted by defense systems, thus overcoming host immunity.\nDeep mutational scanning reveals functional constraints and antigenic variability of Lassa virus glycoprotein complex\nAuthors: Carr, C. R.; Crawford, K. H.; Murphy, M.; Galloway, J. G.; Haddox, H. K.; Matsen, F. A.; Andersen, K. G.; King, N. P.; Bloom, J. D.\nScore: 20.4, Published: 2024-02-06 DOI: 10.1101/2024.02.05.579020\nLassa virus is estimated to cause thousands of human deaths per year, primarily due to spillovers from its natural host, Mastomys rodents. Efforts to create vaccines and antibody therapeutics must account for the evolutionary variability of Lassa viruss glycoprotein complex (GPC), which mediates viral entry into cells and is the target of neutralizing antibodies. To map the evolutionary space accessible to GPC, we use pseudovirus deep mutational scanning to measure how nearly all GPC amino-acid mutations affect cell entry and antibody neutralization. Our experiments define functional constraints throughout GPC. We quantify how GPC mutations affect neutralization by a panel of monoclonal antibodies and show that all antibodies are escaped by mutations that exist among natural Lassa virus lineages. Overall, our work describes a biosafety-level-2 method to elucidate the mutational space accessible to GPC and shows how prospective characterization of antigenic variation could aid design of therapeutics and vaccines.\nThe Cas10 nuclease activity relieves host dormancy to facilitate spacer acquisition and retention during type III-A CRISPR immunity\nAuthors: Aviram, N.; Shilton, A.; Lyn, N.; Reis, B. S.; Brivanlou, A.; Marraffini, L. A.\nScore: 20.2, Published: 2024-02-12 DOI: 10.1101/2024.02.11.579731\nA hallmark of CRISPR immunity is the acquisition of short viral DNA sequences, known as spacers, that are transcribed into guide RNAs to recognize complementary sequences. The staphylococcal type III-A CRISPR-Cas system uses guide RNAs to locate viral transcripts and start a response that displays two mechanisms of immunity. When immunity is triggered by an early-expressed phage RNA, degradation of viral ssDNA can cure the host from infection. In contrast, when the RNA guide targets a late-expressed transcript, defense requires the activity of Csm6, a non-specific RNase. Here we show that Csm6 triggers a growth arrest of the host that provides immunity at the population level which hinders viral propagation to allow the replication of non-infected cells. We demonstrate that this mechanism leads to defense against not only the target phage but also other viruses present in the population that fail to replicate in the arrested cells. On the other hand, dormancy limits the acquisition and retention of spacers that trigger it. We found that the ssDNase activity of type III-A systems is required for the re-growth of a subset of the arrested cells, presumably through the degradation of the phage DNA, ending target transcription and inactivating the immune response. Altogether, our work reveals a built-in mechanism within type III-A CRISPR-Cas systems that allows the exit from dormancy needed for the subsistence of spacers that provide broad-spectrum immunity.\nYraN is a helicase-associated nuclease fostering extended recombination events by natural transformation\nAuthors: Hardy, L.; Plantade, J.; Morales, V.; Mazzamurro, F.; Rocha, E. P. C.; Polard, P.; Charpentier, X.\nScore: 18.9, Published: 2024-02-09 DOI: 10.1101/2024.02.06.579203\nNatural transformation is a widespread molecular pathway of horizontal gene transfer involving the uptake and recombination of exogenous DNA. Exogenous DNA follows a pathway involving genes sequentially required for its capture, internalization, protection, and recombination with the chromosome. Most of these genes were identified through the isolation of transformation-defective mutants and/or based on their expression preceding natural transformation. Yet, genes required for key steps of the pathway remain elusive. We sought to identify any missing component by comparing Tn-seq data obtained in two distantly-related transformable diderm species, the human pathogen Legionella pneumophila and the cyanobacterium Synechococcus elongatus. We identified yraN, a widespread and highly conserved gene of unknown function required for natural transformation. We provide evidence that YraN is a nuclease associated with the ComM helicase, which cooperate to process the D-loop formed by the invasion of the transforming DNA in the chromosomal DNA strands. We propose a model in which cleavage of the displaced strand by YraN can promote the recombination of transforming DNA, leading to extended recombination events. The identification of this YraN/ComM nuclease/helicase system supports the hypothesis that bacteria possess a conserved pathway for the transport and recombination of exogenous DNA. SignificanceMany bacteria use a dedicated pathway to internalize and integrate extracellular DNA into their chromosome. This allows them to naturally acquire genes, or gene variants, that can confer them new traits, hence the term of natural transformation. Although reported nearly a century ago by Frederick Griffith, several aspects of the mechanism of natural transformation remain elusive. Specifically, it is not clear whether or not all the conserved molecular players of the pathway have been identified. We queried two distinct bacterial species for any gene that would be required for natural transformation. We confirmed all core players of the pathway, but also uncovered a highly conserved one, YraN. We provide evidence that YraN is an endonuclease, which in association with the ComM helicase, promotes the efficient integration of the extracellular DNA in the bacterial chromosome.\nDe novo mutations mediate phenotypic switching in an opportunistic human lung pathogen\nAuthors: Poret, A. J.; Schaefers, M. M.; Merakou, C.; Mansour, K. E.; Lagoudas, G. K.; Cross, A. R.; Goldberg, J. B.; Kishony, R.; Uluer, A. Z.; McAdam, A. J.; Blainey, P. C.; Vargas, S. O.; Lieberman, T. D.; Priebe, G. P.\nScore: 17.9, Published: 2024-02-06 DOI: 10.1101/2024.02.06.579193\nBacteria evolving within human hosts encounter selective tradeoffs that render mutations adaptive in one context and deleterious in another. Here, we report that the cystic fibrosis-associated pathogen Burkholderia dolosa overcomes in-human selective tradeoffs by acquiring successive point mutations that alternate phenotypes. We sequenced the whole genomes of 931 respiratory isolates from two recently infected patients and an epidemiologically-linked, chronically-infected patient. These isolates are contextualized using 112 historical genomes from the same outbreak strain. Within both newly infected patients, diverse parallel mutations that disrupt O-antigen expression quickly arose, comprising 29% and 63% of their B. dolosa communities by 3 years. The selection for loss of O-antigen starkly contrasts with our previous observation of parallel O-antigen-restoring mutations after many years of chronic infection in the historical outbreak. Experimental characterization revealed that O-antigen loss increases uptake in immune cells while decreasing competitiveness in the mouse lung. We propose that the balance of these pressures, and thus whether O-antigen expression is advantageous, depends on tissue localization and infection duration. These results suggest that mutation-driven alternation during infection may be more frequent than appreciated and is underestimated without dense temporal sampling.\nCryo-EM structure of a conjugative T4SS identifies a molecular switch regulating pilus biogenesis\nAuthors: Waksman, G.; MACE, K.\nScore: 15.3, Published: 2024-02-09 DOI: 10.1101/2024.02.08.579498\nConjugative Type IV Secretion Systems (T4SS) mediate bacterial conjugation, a process that enables the unidirectional exchange of genetic materials between a donor and a recipient bacterial cell. Bacterial conjugation is the primary means by which antibiotic resistance genes spread among bacterial populations (Barlow, 2009; Virolle et al, 2020). Conjugative T4SSs elaborate a long extracellular filament, termed \"pilus\" to connect with the recipient cell. Previously, we solved the cryo-EM structure of a conjugative T4SS. In this article, based on additional data, we present a more complete T4SS cryo-EM structure than that published earlier. Novel structural features include details of the mismatch symmetry within the OMCC, the presence of a 4th VirB8 subunit in the asymmetric unit of both the Arches and the IMC, and a hydrophobic VirB5 tip in the distal end of the stalk. However, more significantly, we provide previously-undescribed structural insights into the protein VirB10 and identify a novel regulation mechanism of T4SS-mediated pilus biogenesis by this protein, that we believe is a key checkpoint for this process.\n",
  "wordCount" : "2393",
  "inLanguage": "en",
  "datePublished": "2024-02-14T10:38:48Z",
  "dateModified": "2024-02-14T10:38:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 14, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576352">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576352" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576352">
        <p class="paperTitle">Viroid-like colonists of human microbiomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576352" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576352" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zheludev, I. N.; Edgar, R. C.; Galiano, M. J. L.; De la Pena, M.; Babaian, A.; Bhatt, A. S.; Fire, A. Z.</p>
        <p class="info">Score: 829.4, Published: 2024-01-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576352' target='https://doi.org/10.1101/2024.01.20.576352'> 10.1101/2024.01.20.576352</a></p>
        <p class="abstract">Here, we describe the &#34;Obelisks,&#34; a previously unrecognised class of viroid-like elements that we first identified in human gut metatranscriptomic data. &#34;Obelisks&#34; share several properties: (i) apparently circular RNA [~]1kb genome assemblies, (ii) predicted rod-like secondary structures encompassing the entire genome, and (iii) open reading frames coding for a novel protein superfamily, which we call the &#34;Oblins&#34;. We find that Obelisks form their own distinct phylogenetic group with no detectable sequence or structural similarity to known biological agents. Further, Obelisks are prevalent in tested human microbiome metatranscriptomes with representatives detected in [~]7% of analysed stool metatranscriptomes (29/440) and in [~]50% of analysed oral metatranscriptomes (17/32). Obelisk compositions appear to differ between the anatomic sites and are capable of persisting in individuals, with continued presence over &gt;300 days observed in one case. Large scale searches identified 29,959 Obelisks (clustered at 90% nucleotide identity), with examples from all seven continents and in diverse ecological niches. From this search, a subset of Obelisks are identified to code for Obelisk-specific variants of the hammerhead type-III self-cleaving ribozyme. Lastly, we identified one case of a bacterial species (Streptococcus sanguinis) in which a subset of defined laboratory strains harboured a specific Obelisk RNA population. As such, Obelisks comprise a class of diverse RNAs that have colonised, and gone unnoticed in, human, and global microbiomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579183">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579183" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579183">
        <p class="paperTitle">A 20-year time-series of a freshwater lake reveals seasonal dynamics and environmental drivers of viral diversity, ecology, and evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579183" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579183" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhou, Z.; Tran, P. Q.; Martin, C.; Rohwer, R. R.; Baker, B.; McMahon, K.; Anantharaman, K.</p>
        <p class="info">Score: 43.6, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579183' target='https://doi.org/10.1101/2024.02.06.579183'> 10.1101/2024.02.06.579183</a></p>
        <p class="abstract">Long-term ecological studies are powerful tools to investigate microbiomes and ecosystem change but have mostly ignored viruses. Here, we leveraged a 20-year time-series of a freshwater lake to characterize 1.3 million viral genomes over time, seasonality, and environmental factors. We identified 578 auxiliary metabolic gene (AMG) clusters representing over 150,000 AMGs, the most abundant of which, including psbA for photosynthesis, pmoC for methane oxidation, and katG for hydrogen peroxide decomposition, were consistently represented in viruses across decades and seasons. We observed positive associations and niche differentiation between virus-host pairs during seasonal change including in keystone taxa, Cyanobacteria, methanotrophs, and Nanopelagicales. Environmental constraints, specifically inorganic carbon and ammonium influenced viral abundances over time, and highlighted roles of viruses in both &#34;top-down&#34; and &#34;bottom-up&#34; interactions. Key evolutionary processes shaping gene and genome-wide selection included favored fitness genes, reduced genomic heterogeneity, and dominant sub-populations carrying specific genes. Overall, our study advances understanding of diversity, ecological dynamics, and evolutionary trajectories of viruses in Earths microbiomes and ecosystems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579159">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579159" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579159">
        <p class="paperTitle">Mapping glycoprotein structure reveals defining events in the evolution of the Flaviviridae</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579159" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579159" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mifsud, J. C. O.; Lytras, S.; Oliver, M. R.; Toon, K.; Costa, V. A.; Holmes, E. C.; Grove, J.</p>
        <p class="info">Score: 38.4, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579159' target='https://doi.org/10.1101/2024.02.06.579159'> 10.1101/2024.02.06.579159</a></p>
        <p class="abstract">Viral glycoproteins drive membrane fusion in enveloped viruses and determine host range, tissue tropism and pathogenesis. Despite their importance, there is a fragmentary understanding of glycoproteins within the Flaviviridae; for many species the glycoproteins have not yet been identified, for others, such as the hepaciviruses, the molecular mechanisms of membrane fusion remain uncharacterised. Here, we combine comprehensive phylogenetic analyses with systematic protein structure prediction to survey glycoproteins across the entire Flaviviridae. We discover class-II fusion systems, homologous to the orthoflavivirus E glycoprotein, in most species, including highly-divergent jingmenviruses and large genome flaviviruses. However, the E1E2 glycoproteins of the hepaci-, pegi- and pestiviruses are structurally distinct, may represent a novel class of fusion mechanism, and are strictly associated with infection of vertebrate hosts. By mapping glycoprotein distribution onto the underlying phylogeny we reveal a complex history of evolutionary events that have shaped the diverse virology and ecology of the Flaviviridae.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.12.579488">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.12.579488" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.12.579488">
        <p class="paperTitle">A new archaeal virus that suppresses the transcription of host immunity genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.12.579488" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.12.579488" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Turgeman-Grott, I.; Golan, N.; Neri, U.; Naki, D.; Altman-Price, N.; Eizenshtein, K.; Choudhary, D. K.; Levy, R.; Papke, R. T.; Himani, H.; Reshef, L.; Gophna, U.</p>
        <p class="info">Score: 27.1, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.12.579488' target='https://doi.org/10.1101/2024.02.12.579488'> 10.1101/2024.02.12.579488</a></p>
        <p class="abstract">In some extreme environments, archaeal cells have been shown to have chronic viral infections, and such infections are well-tolerated by the hosts and may potentially protect against more lethal infections by lytic viruses. We have discovered that a natural Haloferax strain (48N), which is closely related to the model organism Haloferax volcanii, is chronically infected by a lemon-shaped virus, which we could purify from the medium. The chronic infection by this virus, which we named LSV-48N, is never cleared, despite the multiple defense systems of the host that include CRISPR-Cas, and two CBASS systems. Curing 48N of its virus by genetic engineering, led to radical changes in the gene expression profile of 48N and a dramatic improvement in its growth rate. Remarkably, the cured 48N is the fastest-growing haloarchaeon reported to date, with a generation time of approximately 1 hour at 45C compared to the typical 2.5 hours of H. volcanii or its infected isogen, and faster than any known haloarchaeon. The virus subverts host defenses by reducing their transcription and interfering with the CRISPR spacer acquisition machinery. Our results suggest that the slow growth of many halophilic archaea could be due to the effects of proviruses within their genomes that consume resources and alter the gene expression of their hosts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.11.579819">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.11.579819" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.11.579819">
        <p class="paperTitle">Phages reconstitute NAD&#43; to counter bacterial immunity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.11.579819" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.11.579819" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Osterman, I.; Samra, H.; Rousset, F.; Loseva, E.; Itkin, M.; Malitsky, S.; Yirmiya, E.; Millman, A.; Sorek, R.</p>
        <p class="info">Score: 23.1, Published: 2024-02-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.11.579819' target='https://doi.org/10.1101/2024.02.11.579819'> 10.1101/2024.02.11.579819</a></p>
        <p class="abstract">Bacteria defend against phage infection via a variety of antiphage defense systems. Many defense systems were recently shown to deplete cellular nicotinamide adenine dinucleotide (NAD&#43;) in response to infection, by breaking NAD&#43; to ADP-ribose (ADPR) and nicotinamide. It was demonstrated that NAD&#43; depletion during infection deprives the phage from this essential molecule and impedes phage replication. Here we show that a substantial fraction of phages possess enzymatic pathways allowing reconstitution of NAD&#43; from its degradation products in infected cells. We describe NAD&#43; reconstitution pathway 1 (NARP1), a two-step pathway in which one enzyme phosphorylates ADPR to generate ADPR-pyrophosphate (ADPR-PP), and the second enzyme conjugates ADPR-PP and nicotinamide to generate NAD&#43;. Phages encoding the NARP1 pathway can overcome a diverse set of defense systems, including Thoeris, DSR1, DSR2, SIR2-HerA, and SEFIR, all of which deplete NAD&#43; as part of their defensive mechanism. Phylogenetic analyses show that NARP1 is primarily encoded on phage genomes, suggesting a phage-specific function in countering bacterial defenses. A second pathway, NARP2, allows phages to overcome bacterial defenses by building NAD&#43; via metabolites different than ADPR-PP. Our findings report a unique immune evasion strategy where viruses rebuild molecules depleted by defense systems, thus overcoming host immunity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.05.579020">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.05.579020" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.05.579020">
        <p class="paperTitle">Deep mutational scanning reveals functional constraints and antigenic variability of Lassa virus glycoprotein complex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.05.579020" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.05.579020" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carr, C. R.; Crawford, K. H.; Murphy, M.; Galloway, J. G.; Haddox, H. K.; Matsen, F. A.; Andersen, K. G.; King, N. P.; Bloom, J. D.</p>
        <p class="info">Score: 20.4, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.05.579020' target='https://doi.org/10.1101/2024.02.05.579020'> 10.1101/2024.02.05.579020</a></p>
        <p class="abstract">Lassa virus is estimated to cause thousands of human deaths per year, primarily due to spillovers from its natural host, Mastomys rodents. Efforts to create vaccines and antibody therapeutics must account for the evolutionary variability of Lassa viruss glycoprotein complex (GPC), which mediates viral entry into cells and is the target of neutralizing antibodies. To map the evolutionary space accessible to GPC, we use pseudovirus deep mutational scanning to measure how nearly all GPC amino-acid mutations affect cell entry and antibody neutralization. Our experiments define functional constraints throughout GPC. We quantify how GPC mutations affect neutralization by a panel of monoclonal antibodies and show that all antibodies are escaped by mutations that exist among natural Lassa virus lineages. Overall, our work describes a biosafety-level-2 method to elucidate the mutational space accessible to GPC and shows how prospective characterization of antigenic variation could aid design of therapeutics and vaccines.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.11.579731">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.11.579731" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.11.579731">
        <p class="paperTitle">The Cas10 nuclease activity relieves host dormancy to facilitate spacer acquisition and retention during type III-A CRISPR immunity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.11.579731" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.11.579731" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aviram, N.; Shilton, A.; Lyn, N.; Reis, B. S.; Brivanlou, A.; Marraffini, L. A.</p>
        <p class="info">Score: 20.2, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.11.579731' target='https://doi.org/10.1101/2024.02.11.579731'> 10.1101/2024.02.11.579731</a></p>
        <p class="abstract">A hallmark of CRISPR immunity is the acquisition of short viral DNA sequences, known as spacers, that are transcribed into guide RNAs to recognize complementary sequences. The staphylococcal type III-A CRISPR-Cas system uses guide RNAs to locate viral transcripts and start a response that displays two mechanisms of immunity. When immunity is triggered by an early-expressed phage RNA, degradation of viral ssDNA can cure the host from infection. In contrast, when the RNA guide targets a late-expressed transcript, defense requires the activity of Csm6, a non-specific RNase. Here we show that Csm6 triggers a growth arrest of the host that provides immunity at the population level which hinders viral propagation to allow the replication of non-infected cells. We demonstrate that this mechanism leads to defense against not only the target phage but also other viruses present in the population that fail to replicate in the arrested cells. On the other hand, dormancy limits the acquisition and retention of spacers that trigger it. We found that the ssDNase activity of type III-A systems is required for the re-growth of a subset of the arrested cells, presumably through the degradation of the phage DNA, ending target transcription and inactivating the immune response. Altogether, our work reveals a built-in mechanism within type III-A CRISPR-Cas systems that allows the exit from dormancy needed for the subsistence of spacers that provide broad-spectrum immunity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579203">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579203" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579203">
        <p class="paperTitle">YraN is a helicase-associated nuclease fostering extended recombination events by natural transformation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579203" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579203" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hardy, L.; Plantade, J.; Morales, V.; Mazzamurro, F.; Rocha, E. P. C.; Polard, P.; Charpentier, X.</p>
        <p class="info">Score: 18.9, Published: 2024-02-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579203' target='https://doi.org/10.1101/2024.02.06.579203'> 10.1101/2024.02.06.579203</a></p>
        <p class="abstract">Natural transformation is a widespread molecular pathway of horizontal gene transfer involving the uptake and recombination of exogenous DNA. Exogenous DNA follows a pathway involving genes sequentially required for its capture, internalization, protection, and recombination with the chromosome. Most of these genes were identified through the isolation of transformation-defective mutants and/or based on their expression preceding natural transformation. Yet, genes required for key steps of the pathway remain elusive. We sought to identify any missing component by comparing Tn-seq data obtained in two distantly-related transformable diderm species, the human pathogen Legionella pneumophila and the cyanobacterium Synechococcus elongatus. We identified yraN, a widespread and highly conserved gene of unknown function required for natural transformation. We provide evidence that YraN is a nuclease associated with the ComM helicase, which cooperate to process the D-loop formed by the invasion of the transforming DNA in the chromosomal DNA strands. We propose a model in which cleavage of the displaced strand by YraN can promote the recombination of transforming DNA, leading to extended recombination events. The identification of this YraN/ComM nuclease/helicase system supports the hypothesis that bacteria possess a conserved pathway for the transport and recombination of exogenous DNA.

SignificanceMany bacteria use a dedicated pathway to internalize and integrate extracellular DNA into their chromosome. This allows them to naturally acquire genes, or gene variants, that can confer them new traits, hence the term of natural transformation. Although reported nearly a century ago by Frederick Griffith, several aspects of the mechanism of natural transformation remain elusive. Specifically, it is not clear whether or not all the conserved molecular players of the pathway have been identified. We queried two distinct bacterial species for any gene that would be required for natural transformation. We confirmed all core players of the pathway, but also uncovered a highly conserved one, YraN. We provide evidence that YraN is an endonuclease, which in association with the ComM helicase, promotes the efficient integration of the extracellular DNA in the bacterial chromosome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579193">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579193" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579193">
        <p class="paperTitle">De novo mutations mediate phenotypic switching in an opportunistic human lung pathogen</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579193" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579193" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Poret, A. J.; Schaefers, M. M.; Merakou, C.; Mansour, K. E.; Lagoudas, G. K.; Cross, A. R.; Goldberg, J. B.; Kishony, R.; Uluer, A. Z.; McAdam, A. J.; Blainey, P. C.; Vargas, S. O.; Lieberman, T. D.; Priebe, G. P.</p>
        <p class="info">Score: 17.9, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579193' target='https://doi.org/10.1101/2024.02.06.579193'> 10.1101/2024.02.06.579193</a></p>
        <p class="abstract">Bacteria evolving within human hosts encounter selective tradeoffs that render mutations adaptive in one context and deleterious in another. Here, we report that the cystic fibrosis-associated pathogen Burkholderia dolosa overcomes in-human selective tradeoffs by acquiring successive point mutations that alternate phenotypes. We sequenced the whole genomes of 931 respiratory isolates from two recently infected patients and an epidemiologically-linked, chronically-infected patient. These isolates are contextualized using 112 historical genomes from the same outbreak strain. Within both newly infected patients, diverse parallel mutations that disrupt O-antigen expression quickly arose, comprising 29% and 63% of their B. dolosa communities by 3 years. The selection for loss of O-antigen starkly contrasts with our previous observation of parallel O-antigen-restoring mutations after many years of chronic infection in the historical outbreak. Experimental characterization revealed that O-antigen loss increases uptake in immune cells while decreasing competitiveness in the mouse lung. We propose that the balance of these pressures, and thus whether O-antigen expression is advantageous, depends on tissue localization and infection duration. These results suggest that mutation-driven alternation during infection may be more frequent than appreciated and is underestimated without dense temporal sampling.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.08.579498">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.08.579498" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.08.579498">
        <p class="paperTitle">Cryo-EM structure of a conjugative T4SS identifies a molecular switch regulating pilus biogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.08.579498" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.08.579498" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Waksman, G.; MACE, K.</p>
        <p class="info">Score: 15.3, Published: 2024-02-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.08.579498' target='https://doi.org/10.1101/2024.02.08.579498'> 10.1101/2024.02.08.579498</a></p>
        <p class="abstract">Conjugative Type IV Secretion Systems (T4SS) mediate bacterial conjugation, a process that enables the unidirectional exchange of genetic materials between a donor and a recipient bacterial cell. Bacterial conjugation is the primary means by which antibiotic resistance genes spread among bacterial populations (Barlow, 2009; Virolle et al, 2020). Conjugative T4SSs elaborate a long extracellular filament, termed &#34;pilus&#34; to connect with the recipient cell. Previously, we solved the cryo-EM structure of a conjugative T4SS. In this article, based on additional data, we present a more complete T4SS cryo-EM structure than that published earlier. Novel structural features include details of the mismatch symmetry within the OMCC, the presence of a 4th VirB8 subunit in the asymmetric unit of both the Arches and the IMC, and a hydrophobic VirB5 tip in the distal end of the stalk. However, more significantly, we provide previously-undescribed structural insights into the protein VirB10 and identify a novel regulation mechanism of T4SS-mediated pilus biogenesis by this protein, that we believe is a key checkpoint for this process.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
