<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Antigenicity and receptor affinity of SARS-CoV-2 BA.2.86 spike
Authors: Wang, Q.; Guo, Y.; Liu, L.; Schwanz, L. T.; Li, Z.; Ho, J.; Zhang, R. M.; Iketani, S.; Yu, J.; Huang, Y.; Qu, Y.; Valdez, R.; Lauring, A. S.; Gordon, A.; Wang, H. H.; Liu, L.; Ho, D. D.
Score: 158.7, Published: 2023-09-27 DOI: 10.1101/2023.09.24.559214
Although the COVID-19 pandemic has officially ended1, SARS-CoV-2 continues to spread and evolve. Recent infections have been dominated by XBB.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Antigenicity and receptor affinity of SARS-CoV-2 BA.2.86 spike
Authors: Wang, Q.; Guo, Y.; Liu, L.; Schwanz, L. T.; Li, Z.; Ho, J.; Zhang, R. M.; Iketani, S.; Yu, J.; Huang, Y.; Qu, Y.; Valdez, R.; Lauring, A. S.; Gordon, A.; Wang, H. H.; Liu, L.; Ho, D. D.
Score: 158.7, Published: 2023-09-27 DOI: 10.1101/2023.09.24.559214
Although the COVID-19 pandemic has officially ended1, SARS-CoV-2 continues to spread and evolve. Recent infections have been dominated by XBB." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-01T10:37:47+00:00" />
<meta property="article:modified_time" content="2023-10-01T10:37:47+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Antigenicity and receptor affinity of SARS-CoV-2 BA.2.86 spike
Authors: Wang, Q.; Guo, Y.; Liu, L.; Schwanz, L. T.; Li, Z.; Ho, J.; Zhang, R. M.; Iketani, S.; Yu, J.; Huang, Y.; Qu, Y.; Valdez, R.; Lauring, A. S.; Gordon, A.; Wang, H. H.; Liu, L.; Ho, D. D.
Score: 158.7, Published: 2023-09-27 DOI: 10.1101/2023.09.24.559214
Although the COVID-19 pandemic has officially ended1, SARS-CoV-2 continues to spread and evolve. Recent infections have been dominated by XBB."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Antigenicity and receptor affinity of SARS-CoV-2 BA.2.86 spike\nAuthors: Wang, Q.; Guo, Y.; Liu, L.; Schwanz, L. T.; Li, Z.; Ho, J.; Zhang, R. M.; Iketani, S.; Yu, J.; Huang, Y.; Qu, Y.; Valdez, R.; Lauring, A. S.; Gordon, A.; Wang, H. H.; Liu, L.; Ho, D. D.\nScore: 158.7, Published: 2023-09-27 DOI: 10.1101/2023.09.24.559214\nAlthough the COVID-19 pandemic has officially ended1, SARS-CoV-2 continues to spread and evolve. Recent infections have been dominated by XBB.",
  "keywords": [
    
  ],
  "articleBody": " Antigenicity and receptor affinity of SARS-CoV-2 BA.2.86 spike\nAuthors: Wang, Q.; Guo, Y.; Liu, L.; Schwanz, L. T.; Li, Z.; Ho, J.; Zhang, R. M.; Iketani, S.; Yu, J.; Huang, Y.; Qu, Y.; Valdez, R.; Lauring, A. S.; Gordon, A.; Wang, H. H.; Liu, L.; Ho, D. D.\nScore: 158.7, Published: 2023-09-27 DOI: 10.1101/2023.09.24.559214\nAlthough the COVID-19 pandemic has officially ended1, SARS-CoV-2 continues to spread and evolve. Recent infections have been dominated by XBB.1.5 and EG.5.1 subvariants2. A new subvariant designated BA.2.86 has just emerged, spreading to 21 countries in 5 continents3. This virus contains 34 spike mutations compared to its BA.2 predecessor, thereby raising concerns about its propensity to evade existing antibodies. We examined its antigenicity using human sera and monoclonal antibodies (mAbs). Reassuringly, BA.2.86 was not more resistant to human sera than XBB.1.5 and EG.5.1, indicating that the new subvariant would not have a growth advantage in this regard. Importantly, sera from patients who had XBB breakthrough infection exhibited robust neutralizing activity against all viruses tested, suggesting that upcoming XBB.1.5 monovalent vaccines could confer added protection. The finding that the longer genetic distance of BA.2.86 did not yield a larger antigenic distance was partially explained by the mAb data. While BA.2.86 showed greater resistance to mAbs to subdomain 1 (SD1) and receptor-binding domain (RBD) class 2 and 3 epitopes, it was more sensitive to mAbs to class 1 and 4/1 epitopes in the \"inner face\" of RBD that is exposed only when this domain is in the \"up\" position. We also identified six new spike mutations that mediate antibody resistance, including E554K that threatens SD1 mAbs in clinical development. The BA.2.86 spike also had a remarkably high receptor affinity. The ultimate trajectory of this new SARS-CoV-2 variant will soon be revealed by continuing surveillance, but its worldwide spread is worrisome.\nMechanism for Vipp1 spiral formation, ring biogenesis and membrane repair\nAuthors: Naskar, S.; Merino, A.; Espadas, J.; Singh, J.; Roux, A.; Colom, A.; Low, H. H.\nScore: 115.4, Published: 2023-09-26 DOI: 10.1101/2023.09.26.559607\nESCRT-III family proteins build dynamic filaments that remodel membrane. Although the transition of planar filaments into 3D membrane budding structures is fundamental for their function, the geometric changes in polymer architecture driving the transition remain obscure. Here we show how bacterial Vipp1 polymerises into dynamic planar sheets and spirals on membrane. The spirals converge to form a central ring like those known to bud membrane. To probe how Vipp1 morphs between polymers, we determine the architecture of multiple helical filaments. As well as describing filament constriction and membrane tubulation, the geometric relationship between helical and planar lattices enables Vipp1 sheets and spirals to be modelled. Moreover, the helical structures show filaments twisting - a process needed for Vipp1 to transition between planar and 3D architectures. Given the structural conservation between Vipp1 and ESCRT-III, our results may represent the broad changes in geometry required for some ESCRT-III filaments to switch between 2D and 3D forms.\nEvidence of antigenic drift in the fusion machinery core of SARS-CoV-2 spike\nAuthors: Tan, T. J. C.; Odle, A.; Lei, R.; Matreyek, K. A.; Perlman, S.; Wong, L.-Y. R.; Wu, N. C.\nScore: 46.0, Published: 2023-09-28 DOI: 10.1101/2023.09.27.559757\nAntigenic drift of SARS-CoV-2 is typically defined by mutations in the N-terminal domain and receptor binding domain of spike protein. In contrast, whether antigenic drift occurs in the S2 domain remains largely elusive. Here, we perform a deep mutational scanning experiment to identify S2 mutations that affect binding of SARS-CoV-2 spike to three S2 apex public antibodies. Our results indicate that spatially diverse mutations, including D950N and Q954H, which are observed in Delta and Omicron variants, respectively, weaken the binding of spike to these antibodies. Although S2 apex antibodies are known to be non-neutralizing, we show that they confer partial protection in vivo. We further demonstrate that such in vivo protection activity is diminished by the natural mutation D950N. Overall, this study indicates that the S2 domain of SARS-CoV-2 spike can undergo antigenic drift, which represents a potential challenge for the development of more universal coronavirus vaccines.\nDevelopment of a mutant aerosolized ACE2 that neutralizes SARS-CoV-2 in vivo\nAuthors: Kober, D. L.; Caballero Van Dyke, M. C.; Eitson, J. L. E. L.; Rosenbaum, D.; Schoggins, J. W.\nScore: 33.8, Published: 2023-09-27 DOI: 10.1101/2023.09.26.559550\nThe rapid evolution of variants of SARS-CoV-2 highlights the need for new therapies to prevent disease spread. SARS-CoV-2, like SARS-CoV-1, uses the human cell surface protein angiotensin-converting enzyme 2 (ACE2) as its native receptor. Here, we design and characterize a mutant ACE2 that enables rapid affinity purification of a dimeric protein by altering the active site to prevent autoproteolytic digestion of a C-terminal His10 epitope tag. In cultured cells, mutant ACE2 competitively inhibits lentiviral vectors pseudotyped with spike from multiple SARS-CoV-2 variants, as well as infectious SARS-CoV-2. Moreover, the protein can be nebulized and retains virus-binding properties. We developed a system for delivery of aerosolized ACE2 to K18-hACE2 mice and demonstrate protection by our modified ACE2 when delivered as a prophylactic agent. These results show proof-of-concept for an aerosolized delivery method to evaluate anti-SARS-CoV-2 agents in vivo and suggest a new tool in the ongoing fight against SARS-CoV-2 and other ACE2-dependent viruses.\nReversible conjugation of a CBASS nucleotide cyclase regulates immune response to phage infection\nAuthors: Kruger, L.; Gaskell-Mew, L.; Graham, S.; Shirran, S.; Hertel, R.; White, M. F.\nScore: 23.3, Published: 2023-09-29 DOI: 10.1101/2023.09.29.560134\nAntiviral defence systems build the prokaryotic immune system and their proper regulation is vital for survival and fitness. While it is important that they are readily available in case of infection, they need to be tightly controlled to prevent activation of an unnecessary cellular response. Here we describe how the bacterial cyclic oligonucleotide-based antiphage signalling system (CBASS) uses its intrinsic protein modification system to regulate the nucleotide cyclase. By integrating a Type II CBASS system from Bacillus cereus into the cognate host Bacillus subtilis, we show that the Cap2 protein conjugates the cyclase exclusively to the conserved phage shock protein A (PspA) in the absence of phage. This cyclase-PspA conjugation is reversed after infection by the isopeptidase Cap3. Finally, we propose a model in which the cyclase is held in an inactive state by conjugation to PspA in the absence of phage and that this conjugation is released upon infection, priming the cyclase for activation.\nA green-fluorescent siderophore protects bacterial communities from UV damage\nAuthors: Özkaya, O.; Kramer, J.; Wechsler, T. E.; Kümmerli, R.\nScore: 22.9, Published: 2023-09-26 DOI: 10.1101/2023.09.26.559457\nSunlight enables virtually all life on earth, but also entails harmful ultraviolet (UV) radiation inducing DNA damage. In response to UV stress, natural selection has favored both curative and preventive measures such as DNA repair mechanisms and UV-absorbing pigments. While UV protection by pigments is well documented in plants, animals and fungi, little is known about their protective role in bacteria. Here, we combine batch-culture and microscopy experiments to show that the siderophore pyoverdine, a fluorescent pigment produced by the opportunistic pathogen Pseudomonas aeruginosa offers high-level protection against UV radiation. Our results reveal that bacteria up-regulate pyoverdine production following UV exposure, seemingly as part of a general stress response. We further found that pyoverdine cannot curatively alleviate UV-damage but protects cells preventively and collectively from oncoming UV exposures through its accumulation in the environment. Altogether, our results reveal a new and non-canonical function of this iron-scavenging molecule, demonstrating that pyoverdine acts as a public sunscreen protecting bacterial populations from UV damage. Given that many bacteria produce pigments, such protection might be widespread in species colonizing habitats exposed to UV radiation.\nA gut meta-interactome map reveals modulation of human immunity by microbiome effectors\nAuthors: Young, V.; Dohai, B.; Hitch, T. C.; Hyden, P.; Weller, B.; Heusden, N. S.; Saha, D.; Macgregor, J. F.; Maseko, S. B.; Lin, C.-W.; Boujeant, M.; Choteau, S. A.; Ober, F.; Schwehn, P.; Rothballer, S.; Altmann, M.; Altmann, S.; Strobel, A.; Rothballer, M.; Tofaute, M. J.; Heinig, M.; Clavel, T.; Twizere, J.-C.; Vincentelli, R.; Boes, M.; Krappmann, D.; Falter, C.; Rattei, T.; Brun, C.; Zanzoni, A.; Falter-Braun, P.\nScore: 19.1, Published: 2023-09-25 DOI: 10.1101/2023.09.25.559292\nThe molecular mechanisms by which the gut microbiome influences human health remain largely unknown. Pseudomonadota is the third most abundant phylum in normal gut microbiomes. Several pathogens in this phylum can inject so-called virulence effector proteins into host cells. We report the identification of intact type 3 secretion systems (T3SS) in 5 - 20% of commensal Pseudomonadota in normal human gut microbiomes. To understand their functions, we experimentally generated a high-quality protein-protein meta-interactome map consisting of 1,263 interactions between 289 bacterial effectors and 430 human proteins. Effector targets are enriched for metabolic and immune functions and for genetic variation of microbiome-influenced traits including autoimmune diseases. We demonstrate that effectors modulate NF-{kappa}B signaling, cytokine secretion, and adhesion molecule expression. Finally, effectors are enriched in metagenomes of Crohns disease, but not ulcerative colitis patients pointing toward complex contributions to the etiology of inflammatory bowel diseases. Our results suggest that effector-host protein interactions are an important regulatory layer by which the microbiome impacts human health.\nThe impact of phage and phage resistance on microbial community dynamics\nAuthors: Alseth, E. O.; Custodio, R.; Sundius, S. A.; Kuske, R. A.; Brown, S. P.; Westra, E. R.\nScore: 15.4, Published: 2023-09-26 DOI: 10.1101/2023.09.26.559468\nWhere there are bacteria, there will be bacteriophages. These viruses are known to be important players in shaping the wider microbial community in which they are embedded, with potential implications for human health. On the other hand, bacteria possess a range of distinct immune mechanisms that provide protection against bacteriophages, including the mutation or complete loss of the phage receptor, and CRISPR-Cas adaptive immunity. Yet little is known about how interactions between phages and these different phage resistance mechanisms affect the wider microbial community in which they are embedded. Here, we conducted a 10-day, fully factorial evolution experiment to examine how phage impact the structure and dynamics of an artificial four-species bacterial community that includes either Pseudomonas aeruginosa wild type or an isogenic mutant unable to evolve phage resistance through CRISPR-Cas. Our results show that the microbial community structure is drastically altered by the addition of phage, with Acinetobacter baumannii becoming the dominant species and P. aeruginosa being driven nearly extinct, whereas P. aeruginosa outcompetes the other species in the absence of phage. Moreover, we find that a P. aeruginosa strain with the ability to evolve CRISPR-based resistance generally does better when in the presence of A. baumannii, but that this benefit is largely lost over time as phage is driven extinct. Combined, our data highlight how phage-targeting a dominant species allows for the competitive release of the strongest competitor whilst also contributing to community diversity maintenance and potentially preventing the reinvasion of the target species, and underline the importance of mapping community composition before therapeutically applying phage.\nA phage nucleus-associated RNA-binding protein is required for jumbo phage infection\nAuthors: Enustun, E.; Armbruster, E. G.; Lee, J.; Zhang, S.; Yee, B. A.; Gu, Y.; Deep, A.; Naritomi, J. T.; Liang, Q.; Aigner, S.; Adler, B. A.; Cress, B. F.; Doudna, J. A.; Chaikeeratisak, V.; Cleveland, D. W.; Ghassemian, M.; Yeo, G. W.; Pogliano, J.; Corbett, K. D.\nScore: 21.6, Published: 2023-09-22 DOI: 10.1101/2023.09.22.559000\nLarge-genome bacteriophages (jumbo phages) of the Chimalliviridae family assemble a nucleus-like compartment bounded by a protein shell that protects the replicating phage genome from host-encoded restriction enzymes and CRISPR/Cas nucleases. While the nuclear shell provides broad protection against host nucleases, it necessitates transport of mRNA out of the nucleus-like compartment for translation by host ribosomes, and transport of specific proteins into the nucleus-like compartment to support DNA replication and mRNA transcription. Here we identify a conserved phage nuclear shell-associated protein that we term Chimallin C (ChmC), which adopts a nucleic acid-binding fold, binds RNA with high affinity in vitro, and binds phage mRNAs in infected cells. ChmC also forms phase-separated condensates with RNA in vitro. Targeted knockdown of ChmC using mRNA-targeting dCas13d halts infections at an early stage. Taken together, our data suggest that the conserved ChmC protein acts as a chaperone for phage mRNAs, potentially stabilizing these mRNAs and driving their translocation through the nuclear shell to promote translation and infection progression.\nThe lactonase BxdA mediates metabolic adaptation of maize root bacteria to benzoxazinoids\nAuthors: Thoenen, L.; Kreuzer, M.; Florean, M.; Mateo, P.; Zuest, T.; Giroud, C.; Rouyer, L.; Gfeller, V.; Notter, M.; Knoch, E.; Hapfelmeier, S.; Becker, C.; Schandry, N.; Robert, C.; Koellner, T.; Bruggmann, R.; Erb, M.; Schlaeppi, K.\nScore: 13.1, Published: 2023-09-23 DOI: 10.1101/2023.09.22.559061\nRoot exudates contain secondary metabolites that affect the plants root microbiome. How microbes cope with these bioactive compounds, and how this ability shapes root microbiomes remain largely unknown. We investigated how maize root bacteria metabolise benzoxazinoids, the main specialised metabolites of maize. Diverse and abundant bacteria metabolised the major compound (6-methoxy-benzoxazolin-2-one, MBOA) in the maize rhizosphere to 2-amino-7-methoxyphenoxazin-3-one (AMPO). By contrast, bacteria isolated from Arabidopsis, which does not produce benzoxazinoids, were unable to metabolise MBOA. Among Microbacteria strains, this differential metabolisation allowed to identify a conserved gene cluster containing the lactonase bxdA. BxdA converts MBOA to AMPO in vitro and we show that this capacity provided bacteria a growth benefit under carbon-limiting conditions. Together these results reveal that maize root bacteria - through BxdA - are metabolically adapted to the benzoxazinoids of their host. We propose that metabolic adaptation to plant-specialised compounds shapes root bacterial communities across the plant kingdom.\n",
  "wordCount" : "2229",
  "inLanguage": "en",
  "datePublished": "2023-10-01T10:37:47Z",
  "dateModified": "2023-10-01T10:37:47Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta"><span>updated on October 1, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.24.559214">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.24.559214" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.24.559214">
        <p class="paperTitle">Antigenicity and receptor affinity of SARS-CoV-2 BA.2.86 spike</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.24.559214" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.24.559214" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Q.; Guo, Y.; Liu, L.; Schwanz, L. T.; Li, Z.; Ho, J.; Zhang, R. M.; Iketani, S.; Yu, J.; Huang, Y.; Qu, Y.; Valdez, R.; Lauring, A. S.; Gordon, A.; Wang, H. H.; Liu, L.; Ho, D. D.</p>
        <p class="info">Score: 158.7, Published: 2023-09-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.24.559214' target='https://doi.org/10.1101/2023.09.24.559214'> 10.1101/2023.09.24.559214</a></p>
        <p class="abstract">Although the COVID-19 pandemic has officially ended1, SARS-CoV-2 continues to spread and evolve. Recent infections have been dominated by XBB.1.5 and EG.5.1 subvariants2. A new subvariant designated BA.2.86 has just emerged, spreading to 21 countries in 5 continents3. This virus contains 34 spike mutations compared to its BA.2 predecessor, thereby raising concerns about its propensity to evade existing antibodies. We examined its antigenicity using human sera and monoclonal antibodies (mAbs). Reassuringly, BA.2.86 was not more resistant to human sera than XBB.1.5 and EG.5.1, indicating that the new subvariant would not have a growth advantage in this regard. Importantly, sera from patients who had XBB breakthrough infection exhibited robust neutralizing activity against all viruses tested, suggesting that upcoming XBB.1.5 monovalent vaccines could confer added protection. The finding that the longer genetic distance of BA.2.86 did not yield a larger antigenic distance was partially explained by the mAb data. While BA.2.86 showed greater resistance to mAbs to subdomain 1 (SD1) and receptor-binding domain (RBD) class 2 and 3 epitopes, it was more sensitive to mAbs to class 1 and 4/1 epitopes in the &#34;inner face&#34; of RBD that is exposed only when this domain is in the &#34;up&#34; position. We also identified six new spike mutations that mediate antibody resistance, including E554K that threatens SD1 mAbs in clinical development. The BA.2.86 spike also had a remarkably high receptor affinity. The ultimate trajectory of this new SARS-CoV-2 variant will soon be revealed by continuing surveillance, but its worldwide spread is worrisome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559607">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559607" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559607">
        <p class="paperTitle">Mechanism for Vipp1 spiral formation, ring biogenesis and membrane repair</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559607" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559607" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Naskar, S.; Merino, A.; Espadas, J.; Singh, J.; Roux, A.; Colom, A.; Low, H. H.</p>
        <p class="info">Score: 115.4, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559607' target='https://doi.org/10.1101/2023.09.26.559607'> 10.1101/2023.09.26.559607</a></p>
        <p class="abstract">ESCRT-III family proteins build dynamic filaments that remodel membrane. Although the transition of planar filaments into 3D membrane budding structures is fundamental for their function, the geometric changes in polymer architecture driving the transition remain obscure. Here we show how bacterial Vipp1 polymerises into dynamic planar sheets and spirals on membrane. The spirals converge to form a central ring like those known to bud membrane. To probe how Vipp1 morphs between polymers, we determine the architecture of multiple helical filaments. As well as describing filament constriction and membrane tubulation, the geometric relationship between helical and planar lattices enables Vipp1 sheets and spirals to be modelled. Moreover, the helical structures show filaments twisting - a process needed for Vipp1 to transition between planar and 3D architectures. Given the structural conservation between Vipp1 and ESCRT-III, our results may represent the broad changes in geometry required for some ESCRT-III filaments to switch between 2D and 3D forms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.559757">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.559757" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.559757">
        <p class="paperTitle">Evidence of antigenic drift in the fusion machinery core of SARS-CoV-2 spike</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.559757" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.559757" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tan, T. J. C.; Odle, A.; Lei, R.; Matreyek, K. A.; Perlman, S.; Wong, L.-Y. R.; Wu, N. C.</p>
        <p class="info">Score: 46.0, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.559757' target='https://doi.org/10.1101/2023.09.27.559757'> 10.1101/2023.09.27.559757</a></p>
        <p class="abstract">Antigenic drift of SARS-CoV-2 is typically defined by mutations in the N-terminal domain and receptor binding domain of spike protein. In contrast, whether antigenic drift occurs in the S2 domain remains largely elusive. Here, we perform a deep mutational scanning experiment to identify S2 mutations that affect binding of SARS-CoV-2 spike to three S2 apex public antibodies. Our results indicate that spatially diverse mutations, including D950N and Q954H, which are observed in Delta and Omicron variants, respectively, weaken the binding of spike to these antibodies. Although S2 apex antibodies are known to be non-neutralizing, we show that they confer partial protection in vivo. We further demonstrate that such in vivo protection activity is diminished by the natural mutation D950N. Overall, this study indicates that the S2 domain of SARS-CoV-2 spike can undergo antigenic drift, which represents a potential challenge for the development of more universal coronavirus vaccines.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559550">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559550" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559550">
        <p class="paperTitle">Development of a mutant aerosolized ACE2 that neutralizes SARS-CoV-2 in vivo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559550" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559550" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kober, D. L.; Caballero Van Dyke, M. C.; Eitson, J. L. E. L.; Rosenbaum, D.; Schoggins, J. W.</p>
        <p class="info">Score: 33.8, Published: 2023-09-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559550' target='https://doi.org/10.1101/2023.09.26.559550'> 10.1101/2023.09.26.559550</a></p>
        <p class="abstract">The rapid evolution of variants of SARS-CoV-2 highlights the need for new therapies to prevent disease spread. SARS-CoV-2, like SARS-CoV-1, uses the human cell surface protein angiotensin-converting enzyme 2 (ACE2) as its native receptor. Here, we design and characterize a mutant ACE2 that enables rapid affinity purification of a dimeric protein by altering the active site to prevent autoproteolytic digestion of a C-terminal His10 epitope tag. In cultured cells, mutant ACE2 competitively inhibits lentiviral vectors pseudotyped with spike from multiple SARS-CoV-2 variants, as well as infectious SARS-CoV-2. Moreover, the protein can be nebulized and retains virus-binding properties. We developed a system for delivery of aerosolized ACE2 to K18-hACE2 mice and demonstrate protection by our modified ACE2 when delivered as a prophylactic agent. These results show proof-of-concept for an aerosolized delivery method to evaluate anti-SARS-CoV-2 agents in vivo and suggest a new tool in the ongoing fight against SARS-CoV-2 and other ACE2-dependent viruses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560134">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560134" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560134">
        <p class="paperTitle">Reversible conjugation of a CBASS nucleotide cyclase regulates immune response to phage infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560134" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560134" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kruger, L.; Gaskell-Mew, L.; Graham, S.; Shirran, S.; Hertel, R.; White, M. F.</p>
        <p class="info">Score: 23.3, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560134' target='https://doi.org/10.1101/2023.09.29.560134'> 10.1101/2023.09.29.560134</a></p>
        <p class="abstract">Antiviral defence systems build the prokaryotic immune system and their proper regulation is vital for survival and fitness. While it is important that they are readily available in case of infection, they need to be tightly controlled to prevent activation of an unnecessary cellular response. Here we describe how the bacterial cyclic oligonucleotide-based antiphage signalling system (CBASS) uses its intrinsic protein modification system to regulate the nucleotide cyclase. By integrating a Type II CBASS system from Bacillus cereus into the cognate host Bacillus subtilis, we show that the Cap2 protein conjugates the cyclase exclusively to the conserved phage shock protein A (PspA) in the absence of phage. This cyclase-PspA conjugation is reversed after infection by the isopeptidase Cap3. Finally, we propose a model in which the cyclase is held in an inactive state by conjugation to PspA in the absence of phage and that this conjugation is released upon infection, priming the cyclase for activation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559457">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559457" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559457">
        <p class="paperTitle">A green-fluorescent siderophore protects bacterial communities from UV damage</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559457" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559457" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Özkaya, O.; Kramer, J.; Wechsler, T. E.; Kümmerli, R.</p>
        <p class="info">Score: 22.9, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559457' target='https://doi.org/10.1101/2023.09.26.559457'> 10.1101/2023.09.26.559457</a></p>
        <p class="abstract">Sunlight enables virtually all life on earth, but also entails harmful ultraviolet (UV) radiation inducing DNA damage. In response to UV stress, natural selection has favored both curative and preventive measures such as DNA repair mechanisms and UV-absorbing pigments. While UV protection by pigments is well documented in plants, animals and fungi, little is known about their protective role in bacteria. Here, we combine batch-culture and microscopy experiments to show that the siderophore pyoverdine, a fluorescent pigment produced by the opportunistic pathogen Pseudomonas aeruginosa offers high-level protection against UV radiation. Our results reveal that bacteria up-regulate pyoverdine production following UV exposure, seemingly as part of a general stress response. We further found that pyoverdine cannot curatively alleviate UV-damage but protects cells preventively and collectively from oncoming UV exposures through its accumulation in the environment. Altogether, our results reveal a new and non-canonical function of this iron-scavenging molecule, demonstrating that pyoverdine acts as a public sunscreen protecting bacterial populations from UV damage. Given that many bacteria produce pigments, such protection might be widespread in species colonizing habitats exposed to UV radiation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559292">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559292" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559292">
        <p class="paperTitle">A gut meta-interactome map reveals modulation of human immunity by microbiome effectors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559292" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559292" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Young, V.; Dohai, B.; Hitch, T. C.; Hyden, P.; Weller, B.; Heusden, N. S.; Saha, D.; Macgregor, J. F.; Maseko, S. B.; Lin, C.-W.; Boujeant, M.; Choteau, S. A.; Ober, F.; Schwehn, P.; Rothballer, S.; Altmann, M.; Altmann, S.; Strobel, A.; Rothballer, M.; Tofaute, M. J.; Heinig, M.; Clavel, T.; Twizere, J.-C.; Vincentelli, R.; Boes, M.; Krappmann, D.; Falter, C.; Rattei, T.; Brun, C.; Zanzoni, A.; Falter-Braun, P.</p>
        <p class="info">Score: 19.1, Published: 2023-09-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559292' target='https://doi.org/10.1101/2023.09.25.559292'> 10.1101/2023.09.25.559292</a></p>
        <p class="abstract">The molecular mechanisms by which the gut microbiome influences human health remain largely unknown. Pseudomonadota is the third most abundant phylum in normal gut microbiomes. Several pathogens in this phylum can inject so-called virulence effector proteins into host cells. We report the identification of intact type 3 secretion systems (T3SS) in 5 - 20% of commensal Pseudomonadota in normal human gut microbiomes. To understand their functions, we experimentally generated a high-quality protein-protein meta-interactome map consisting of 1,263 interactions between 289 bacterial effectors and 430 human proteins. Effector targets are enriched for metabolic and immune functions and for genetic variation of microbiome-influenced traits including autoimmune diseases. We demonstrate that effectors modulate NF-{kappa}B signaling, cytokine secretion, and adhesion molecule expression. Finally, effectors are enriched in metagenomes of Crohns disease, but not ulcerative colitis patients pointing toward complex contributions to the etiology of inflammatory bowel diseases. Our results suggest that effector-host protein interactions are an important regulatory layer by which the microbiome impacts human health.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559468">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559468" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559468">
        <p class="paperTitle">The impact of phage and phage resistance on microbial community dynamics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559468" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559468" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alseth, E. O.; Custodio, R.; Sundius, S. A.; Kuske, R. A.; Brown, S. P.; Westra, E. R.</p>
        <p class="info">Score: 15.4, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559468' target='https://doi.org/10.1101/2023.09.26.559468'> 10.1101/2023.09.26.559468</a></p>
        <p class="abstract">Where there are bacteria, there will be bacteriophages. These viruses are known to be important players in shaping the wider microbial community in which they are embedded, with potential implications for human health. On the other hand, bacteria possess a range of distinct immune mechanisms that provide protection against bacteriophages, including the mutation or complete loss of the phage receptor, and CRISPR-Cas adaptive immunity. Yet little is known about how interactions between phages and these different phage resistance mechanisms affect the wider microbial community in which they are embedded. Here, we conducted a 10-day, fully factorial evolution experiment to examine how phage impact the structure and dynamics of an artificial four-species bacterial community that includes either Pseudomonas aeruginosa wild type or an isogenic mutant unable to evolve phage resistance through CRISPR-Cas. Our results show that the microbial community structure is drastically altered by the addition of phage, with Acinetobacter baumannii becoming the dominant species and P. aeruginosa being driven nearly extinct, whereas P. aeruginosa outcompetes the other species in the absence of phage. Moreover, we find that a P. aeruginosa strain with the ability to evolve CRISPR-based resistance generally does better when in the presence of A. baumannii, but that this benefit is largely lost over time as phage is driven extinct. Combined, our data highlight how phage-targeting a dominant species allows for the competitive release of the strongest competitor whilst also contributing to community diversity maintenance and potentially preventing the reinvasion of the target species, and underline the importance of mapping community composition before therapeutically applying phage.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.559000">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.559000" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.559000">
        <p class="paperTitle">A phage nucleus-associated RNA-binding protein is required for jumbo phage infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.559000" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.559000" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Enustun, E.; Armbruster, E. G.; Lee, J.; Zhang, S.; Yee, B. A.; Gu, Y.; Deep, A.; Naritomi, J. T.; Liang, Q.; Aigner, S.; Adler, B. A.; Cress, B. F.; Doudna, J. A.; Chaikeeratisak, V.; Cleveland, D. W.; Ghassemian, M.; Yeo, G. W.; Pogliano, J.; Corbett, K. D.</p>
        <p class="info">Score: 21.6, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.559000' target='https://doi.org/10.1101/2023.09.22.559000'> 10.1101/2023.09.22.559000</a></p>
        <p class="abstract">Large-genome bacteriophages (jumbo phages) of the Chimalliviridae family assemble a nucleus-like compartment bounded by a protein shell that protects the replicating phage genome from host-encoded restriction enzymes and CRISPR/Cas nucleases. While the nuclear shell provides broad protection against host nucleases, it necessitates transport of mRNA out of the nucleus-like compartment for translation by host ribosomes, and transport of specific proteins into the nucleus-like compartment to support DNA replication and mRNA transcription. Here we identify a conserved phage nuclear shell-associated protein that we term Chimallin C (ChmC), which adopts a nucleic acid-binding fold, binds RNA with high affinity in vitro, and binds phage mRNAs in infected cells. ChmC also forms phase-separated condensates with RNA in vitro. Targeted knockdown of ChmC using mRNA-targeting dCas13d halts infections at an early stage. Taken together, our data suggest that the conserved ChmC protein acts as a chaperone for phage mRNAs, potentially stabilizing these mRNAs and driving their translocation through the nuclear shell to promote translation and infection progression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.559061">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.559061" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.559061">
        <p class="paperTitle">The lactonase BxdA mediates metabolic adaptation of maize root bacteria to benzoxazinoids</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.559061" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.559061" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Thoenen, L.; Kreuzer, M.; Florean, M.; Mateo, P.; Zuest, T.; Giroud, C.; Rouyer, L.; Gfeller, V.; Notter, M.; Knoch, E.; Hapfelmeier, S.; Becker, C.; Schandry, N.; Robert, C.; Koellner, T.; Bruggmann, R.; Erb, M.; Schlaeppi, K.</p>
        <p class="info">Score: 13.1, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.559061' target='https://doi.org/10.1101/2023.09.22.559061'> 10.1101/2023.09.22.559061</a></p>
        <p class="abstract">Root exudates contain secondary metabolites that affect the plants root microbiome. How microbes cope with these bioactive compounds, and how this ability shapes root microbiomes remain largely unknown. We investigated how maize root bacteria metabolise benzoxazinoids, the main specialised metabolites of maize. Diverse and abundant bacteria metabolised the major compound (6-methoxy-benzoxazolin-2-one, MBOA) in the maize rhizosphere to 2-amino-7-methoxyphenoxazin-3-one (AMPO). By contrast, bacteria isolated from Arabidopsis, which does not produce benzoxazinoids, were unable to metabolise MBOA. Among Microbacteria strains, this differential metabolisation allowed to identify a conserved gene cluster containing the lactonase bxdA. BxdA converts MBOA to AMPO in vitro and we show that this capacity provided bacteria a growth benefit under carbon-limiting conditions. Together these results reveal that maize root bacteria - through BxdA - are metabolically adapted to the benzoxazinoids of their host. We propose that metabolic adaptation to plant-specialised compounds shapes root bacterial communities across the plant kingdom.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
