<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Phospholipid transport to the bacterial outer membrane through an envelope-spanning bridge
Authors: Isom, G.; Cooper, B. F.; Clark, R.; Kudhail, A.; Ekiert, D. C.; Bhabha, G.; Khalid, S.
Score: 52.9, Published: 2023-10-05 DOI: 10.1101/2023.10.05.561070
The outer membrane of Gram-negative bacteria provides a formidable barrier, essential for both pathogenesis and antimicrobial resistance. Biogenesis of the outer membrane requires the transport of phospholipids across the cell envelope. Recently, YhdP was implicated as a major protagonist in the transport of phospholipids from the inner membrane to the outer membrane however the molecular mechanism of YhdP mediated transport remains elusive.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Phospholipid transport to the bacterial outer membrane through an envelope-spanning bridge
Authors: Isom, G.; Cooper, B. F.; Clark, R.; Kudhail, A.; Ekiert, D. C.; Bhabha, G.; Khalid, S.
Score: 52.9, Published: 2023-10-05 DOI: 10.1101/2023.10.05.561070
The outer membrane of Gram-negative bacteria provides a formidable barrier, essential for both pathogenesis and antimicrobial resistance. Biogenesis of the outer membrane requires the transport of phospholipids across the cell envelope. Recently, YhdP was implicated as a major protagonist in the transport of phospholipids from the inner membrane to the outer membrane however the molecular mechanism of YhdP mediated transport remains elusive." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-08T10:37:16+00:00" />
<meta property="article:modified_time" content="2023-10-08T10:37:16+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Phospholipid transport to the bacterial outer membrane through an envelope-spanning bridge
Authors: Isom, G.; Cooper, B. F.; Clark, R.; Kudhail, A.; Ekiert, D. C.; Bhabha, G.; Khalid, S.
Score: 52.9, Published: 2023-10-05 DOI: 10.1101/2023.10.05.561070
The outer membrane of Gram-negative bacteria provides a formidable barrier, essential for both pathogenesis and antimicrobial resistance. Biogenesis of the outer membrane requires the transport of phospholipids across the cell envelope. Recently, YhdP was implicated as a major protagonist in the transport of phospholipids from the inner membrane to the outer membrane however the molecular mechanism of YhdP mediated transport remains elusive."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Phospholipid transport to the bacterial outer membrane through an envelope-spanning bridge\nAuthors: Isom, G.; Cooper, B. F.; Clark, R.; Kudhail, A.; Ekiert, D. C.; Bhabha, G.; Khalid, S.\nScore: 52.9, Published: 2023-10-05 DOI: 10.1101/2023.10.05.561070\nThe outer membrane of Gram-negative bacteria provides a formidable barrier, essential for both pathogenesis and antimicrobial resistance. Biogenesis of the outer membrane requires the transport of phospholipids across the cell envelope. Recently, YhdP was implicated as a major protagonist in the transport of phospholipids from the inner membrane to the outer membrane however the molecular mechanism of YhdP mediated transport remains elusive.",
  "keywords": [
    
  ],
  "articleBody": " Phospholipid transport to the bacterial outer membrane through an envelope-spanning bridge\nAuthors: Isom, G.; Cooper, B. F.; Clark, R.; Kudhail, A.; Ekiert, D. C.; Bhabha, G.; Khalid, S.\nScore: 52.9, Published: 2023-10-05 DOI: 10.1101/2023.10.05.561070\nThe outer membrane of Gram-negative bacteria provides a formidable barrier, essential for both pathogenesis and antimicrobial resistance. Biogenesis of the outer membrane requires the transport of phospholipids across the cell envelope. Recently, YhdP was implicated as a major protagonist in the transport of phospholipids from the inner membrane to the outer membrane however the molecular mechanism of YhdP mediated transport remains elusive. Here, utilising AlphaFold, we observe YhdP to form an elongated assembly of 60 beta strands that curve to form a continuous hydrophobic groove. This architecture is consistent with our negative stain electron microscopy data which reveals YhdP to be approximately 250 Angstrom in length and thus sufficient to span the bacterial cell envelope. Furthermore, molecular dynamics simulations and in vivo bacterial growth assays indicate essential helical regions at the N- and C-termini of YhdP, that may embed into the inner and outer membranes respectively, reinforcing its envelope spanning nature. Our in vivo crosslinking data reveal phosphate-containing substrates captured along the length of the YhdP groove, providing direct evidence that YhdP transports phospholipids. This finding is congruent with our molecular dynamics simulations which demonstrate the propensity for inner membrane lipids to spontaneously enter the groove of YhdP. Collectively, our results support a model in which YhdP bridges the cell envelope, providing a hydrophobic environment for the transport of phospholipids to the outer membrane.\nAntigenicity and receptor affinity of SARS-CoV-2 BA.2.86 spike\nAuthors: Wang, Q.; Guo, Y.; Liu, L.; Schwanz, L. T.; Li, Z.; Ho, J.; Zhang, R. M.; Iketani, S.; Yu, J.; Huang, Y.; Qu, Y.; Valdez, R.; Lauring, A. S.; Gordon, A.; Wang, H. H.; Liu, L.; Ho, D. D.\nScore: 199.1, Published: 2023-09-27 DOI: 10.1101/2023.09.24.559214\nAlthough the COVID-19 pandemic has officially ended1, SARS-CoV-2 continues to spread and evolve. Recent infections have been dominated by XBB.1.5 and EG.5.1 subvariants2. A new subvariant designated BA.2.86 has just emerged, spreading to 21 countries in 5 continents3. This virus contains 34 spike mutations compared to its BA.2 predecessor, thereby raising concerns about its propensity to evade existing antibodies. We examined its antigenicity using human sera and monoclonal antibodies (mAbs). Reassuringly, BA.2.86 was not more resistant to human sera than XBB.1.5 and EG.5.1, indicating that the new subvariant would not have a growth advantage in this regard. Importantly, sera from patients who had XBB breakthrough infection exhibited robust neutralizing activity against all viruses tested, suggesting that upcoming XBB.1.5 monovalent vaccines could confer added protection. The finding that the longer genetic distance of BA.2.86 did not yield a larger antigenic distance was partially explained by the mAb data. While BA.2.86 showed greater resistance to mAbs to subdomain 1 (SD1) and receptor-binding domain (RBD) class 2 and 3 epitopes, it was more sensitive to mAbs to class 1 and 4/1 epitopes in the \"inner face\" of RBD that is exposed only when this domain is in the \"up\" position. We also identified six new spike mutations that mediate antibody resistance, including E554K that threatens SD1 mAbs in clinical development. The BA.2.86 spike also had a remarkably high receptor affinity. The ultimate trajectory of this new SARS-CoV-2 variant will soon be revealed by continuing surveillance, but its worldwide spread is worrisome.\nVirological characteristics correlating with SARS-CoV-2 spike protein fusogenicity\nAuthors: Begum, M. M.; Ichihara, K.; Takahashi, O.; Nasser, H.; Jonathan, M.; Tokunaga, K.; Yoshida, I.; Nagashima, M.; Sadamasu, K.; Yoshimura, K.; Sato, K.; Ikeda, T.\nScore: 37.8, Published: 2023-10-04 DOI: 10.1101/2023.10.03.560628\nThe severe acute respiratory syndrome coronavirus (SARS-CoV-2) spike (S) protein is essential in mediating membrane fusion of the virus with the target cells. Several reports demonstrated that SARS-CoV-2 S protein fusogenicity is reportedly closely associated with the intrinsic pathogenicity of the virus determined using hamster models. However, the association between S protein fusogenicity and other virological parameters remains elusive. In this study, we investigated the virological parameters of eleven previous variants of concern (VOCs) and variants of interest (VOIs) correlating with S protein fusogenicity. S protein fusogenicity was found to be strongly correlated with S1/S2 cleavage efficiency and plaque size formed by clinical isolates. However, S protein fusogenicity was less associated with pseudoviral infectivity, pseudovirus entry efficiency, and viral replication kinetics. Taken together, our results suggest that S1/S2 cleavage efficiency and plaque size could be potential indicators to predict the intrinsic pathogenicity of newly emerged SARS-CoV-2 variants.\nGlobal RNA interactome of nitrogen starved Escherichia coli uncovers a conserved post-transcriptional regulatory axis required for optimal growth recovery\nAuthors: McQuail, J.; Matera, G.; Gräfenhan, T.; Bischler, T.; Haberkant, P.; Stein, F.; Vogel, J.; Wigneshweraraj, R.\nScore: 18.8, Published: 2023-10-02 DOI: 10.1101/2023.10.02.560498\nThe RNA binding protein Hfq has a central role in the post-transcription control of gene expression in many bacteria. Numerous studies have mapped the transcriptome-wide Hfq-mediated RNA-RNA interactions in growing bacteria or bacteria that have entered short-term growth-arrest. To what extent post-transcriptional regulation underpins gene expression in growth-arrested bacteria remains unknown. Here, we used nitrogen (N) starvation as a model to study the Hfq-mediated RNA interactome as Escherichia coli enter, experience, and exit long-term growth arrest. We observe that the Hfq-mediated RNA interactome undergoes extensive changes during N starvation, with the conserved SdsR sRNA making the most interactions with different mRNA targets exclusively in long-term N-starved E. coli. Taking a proteomics approach, we reveal that in growth-arrested cells SdsR influences gene expression far beyond its direct mRNA targets. We demonstrate that the absence of SdsR significantly compromises the ability of the mutant bacteria to recover growth competitively from the long-term N-starved state and uncover a conserved post-transcriptional regulatory axis which underpins this process.\nEstablishment and comparative genomics of a high-quality collection of mosquito-associated bacterial isolates -- MosAIC (Mosquito-Associated Isolate Collection)\nAuthors: Foo, A.; Brettell, L. E.; Nichols, H. L.; 2022 UW-Madison Capstone in Microbiology Students, ; Medina Munoz, M.; Lysne, J.; Dhokiya, V.; Hoque, A. F.; Brackney, D. E.; Caragata, E. P.; Hutchinson, M.; Jacobs-Lorena, M.; Lampe, D. J.; Martin, E.; Valiente Moro, C.; Povelones, M.; Short, S. M.; Steven, B.; Xu, J.; Paustian, T. D.; Rondon, M. R.; Hughes, G. L.; Coon, K. L.; Heinz, E.\nScore: 16.8, Published: 2023-10-04 DOI: 10.1101/2023.10.04.560816\nMosquitoes transmit medically important human pathogens, including viruses like dengue virus and parasites such as Plasmodium spp., the causative agent of malaria. Mosquito microbiomes are critically important for the ability of mosquitoes to transmit disease-causing agents. However, while large collections of bacterial isolates and genomic data exist for vertebrate microbiomes, the vast majority of work in mosquitoes to date is based on 16S rRNA gene amplicon data that provides limited taxonomic resolution and no functional information. To address this gap and facilitate future studies using experimental microbiome manipulations, we generated a bacterial Mosquito-Associated Isolate Collection (MosAIC) consisting of 392 bacterial isolates with extensive metadata and high-quality draft genome assemblies that are publicly available for use by the scientific community. MosAIC encompasses 142 species spanning 29 bacterial families, with members of the Enterobacteriaceae comprising 40% of the collection. Phylogenomic analysis of three genera, Enterobacter, Serratia, and Elizabethkingia, reveal lineages of mosquito-associated bacteria isolated from different mosquito species in multiple laboratories. Investigation into species' pangenomes further reveals clusters of genes specific to these lineages, which are of interest for future work to identify functions underlying mosquito host association. Altogether, we describe the generation of a physical collection of mosquito-associated bacterial isolates, their genomic data, and analyses of selected groups in context of genome data from closely related isolates, providing a unique, highly valuable resource to investigate factors for bacterial colonisation and adaptation within mosquito hosts. Future efforts will expand the collection to include broader geographic and host species representation, especially from individuals collected from field populations, as well as other mosquito-associated microbes, including fungi, archaea, and protozoa.\nStreptomyces extracellular vesicles are a broad and permissive antimicrobial packaging and delivery system\nAuthors: Meyer, K. J.; Nodwell, J. R.\nScore: 16.3, Published: 2023-10-02 DOI: 10.1101/2023.10.02.560488\nStreptomyces are the primary source of bioactive specialized metabolites used in research and medicine, including many antimicrobials. These are presumed to be secreted and function as freely soluble compounds. However, increasing evidence suggests extracellular vesicles are an alternative secretion system. We assessed environmental and lab-adapted Streptomyces (sporulating filamentous actinomycetes) and found a majority of strains produce vesicles packaging antimicrobial compounds. The cargo included actinomycins, anthracyclines, candicidin, and actinorhodin, reflecting both diverse chemical properties and diverse antibacterial and antifungal activity. The production of the antimicrobials and vesicles was linked, although vesicle production continued in the absence of antimicrobials. We demonstrated that antimicrobial-containing vesicles achieve direct delivery of the cargo to other microbes. Notably, this delivery via membrane fusion occurred to a broad range of microbes including pathogenic bacteria and yeast. Vesicle encapsulation offers a broad and permissive packaging and delivery system for antimicrobial specialized metabolites, with important implications for ecology and translation. GRAPHICAL ABSTRACT O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=185 SRC=\"FIGDIR/small/560488v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (48K): org.highwire.dtl.DTLVardef@1d0eb3dorg.highwire.dtl.DTLVardef@1041fdcorg.highwire.dtl.DTLVardef@1c7b679org.highwire.dtl.DTLVardef@9cf51c_HPS_FORMAT_FIGEXP M_FIG C_FIG IMPORTANCEExtracellular vesicle encapsulation changes our picture of how antimicrobial metabolites function in the environment, and provides an alternative translational approach for delivery of antimicrobials. We find the majority of Streptomyces strains are capable of releasing antimicrobial vesicles, and at least four distinct classes of compounds can be packaged, suggesting this is widespread in nature. This is a striking departure from the primary paradigm of secretion and action of specialized metabolites as soluble compounds. Importantly the vesicles deliver antimicrobial metabolites directly to other microbes via membrane fusion, including to pathogenic bacteria and yeast. This suggests future applications in which lipid-encapsulated natural product antibiotics and antifungals could be used to solve some of the most pressing problems in drug resistance.\nThe Biofilm Lifestyle Shapes the Evolution of β-Lactamases\nAuthors: Lorentzen, O. M.; Haukefer, A. S. B.; Johnsen, P. J.; Frohlich, C.\nScore: 16.3, Published: 2023-10-02 DOI: 10.1101/2023.10.02.560492\nThe evolutionary relationship between the biofilm lifestyle and antibiotic resistance enzymes remains a subject of limited understanding. Here, we investigate how {beta}-lactamases affect biofilm formation in Vibrio cholerae and how selection for a biofilm lifestyle impacts the evolution of these enzymes. Seven genetically diverse {beta}-lactamases expressed in V. cholerae displayed a strong inhibitory effect on biofilm production, ranging from 17% to 61%. To understand how natural evolution affects this antagonistic pleiotropy under biofilm selecting conditions, we randomly mutagenized one {beta}-lactamase and selected for elevated biofilm formation. Our results revealed that biofilm evolution selects for mutations predominantly clustered around the {beta}-lactamases active site, yielding functional variants still proficient in {beta}-lactam hydrolysis without biofilm inhibition. Mutational analysis of evolved variants demonstrated that restoration of biofilm development could be achieved either independent of enzymatic function or by actively leveraging enzymatic activity to increase biofilm formation. Taken together, the biofilm lifestyle can impose a profound selective pressure on antimicrobial resistance enzymes. Shedding light on such evolutionary interplays is of great importance to understand the various factors driving antimicrobial resistance. Impact statement{beta}-lactamases inhibit biofilm formation and the selection for increased biofilm production can mitigate this antagonistic pleiotropic effect. The emergence of {beta}-lactamase variants avoiding biofilm inhibition strongly suggests that the biofilm lifestyle affects the evolutionary fate of these enzymes.\nSARS-CoV-2 N protein-induced Dicer, XPO5, SRSF3, and hnRNPA3 downregulation causes pneumonia\nAuthors: Luo, Y.-W.; Zhou, J.-P.; Ji, H.; Zheng, A.; Wang, X.; Dai, Z.; Luo, Z.; Cao, F.; Wang, X.-Y.; Bai, Y.; Chen, D.; Chen, Y.; Wang, Q.; Yang, Y.; Zhang, X.; Chiu, S.; Huang, A.-L.; Tang, K.-F.\nScore: 13.4, Published: 2023-10-04 DOI: 10.1101/2023.10.03.560426\nAge is a major risk factor for coronavirus disease (COVID-19)-associated severe pneumonia and mortality; however, the underlying mechanism remains unclear. Herein, we investigated whether age-related deregulation of RNAi components and RNA splicing factors affects COVID-19 severity. Decreased expression of RNAi components (Dicer and XPO5) and splicing factors (SRSF3 and hnRNPA3) correlated with increased severity of COVID-19 and SARS-CoV-2 nucleocapsid (N) protein-induced pneumonia. N protein induced autophagic degradation of Dicer, XPO5, SRSF3, and hnRNPA3, repressing miRNA biogenesis and RNA splicing and inducing DNA damage, proteotoxic stress, and pneumonia. Dicer, XPO5, SRSF3, and hnRNPA3 were downregulated with age in mouse lung tissues. Older mice experienced more severe N protein-induced pneumonia than younger mice. However, treatment with a poly(ADP-ribose) polymerase inhibitor (PJ34) or aromatase inhibitor (anastrozole) relieved N protein-induced pneumonia by restoring Dicer, XPO5, SRSF3, and hnRNPA3 expression. These findings will aid in developing improved treatments for SARS-CoV-2-associated pneumonia.\nDetection and characterization of novel luchacoviruses, genus Alphacoronavirus, shed in saliva and feces of meso-carnivores in the northeastern United States\nAuthors: Olarte-Castillo, X. A.; Plimpton, L. D.; McQueary, H. C.; Sun, Y.; Yu, Y. T.; Cover, S. C.; Richardson, A. N.; Jin, Y.; Grenier, J. K.; Cummings, K. J.; Bunting, E. M.; Diuk-Wasser, M.; Needle, D.; Schuler, K.; Stanhope, M. J.; Whittaker, G.; Goodman, L. B.\nScore: 16.5, Published: 2023-09-13 DOI: 10.1101/2023.05.31.541188\nSmall to mid-sized carnivores, or meso-carnivores, comprise a group of diverse mammals, many of which can adapt to anthropogenically disturbed environments. Wild meso-carnivores living in urban areas may get exposed to or spread pathogens to other species, including stray/feral domestic animals. Several coronaviruses (CoVs) have been detected in domesticated and farmed meso-carnivores, but knowledge of CoVs circulating in free-ranging wild meso-carnivores remains limited. In this study, we analyzed 321 samples collected between 2016 and 2022 from 9 species of free-ranging wild meso-carnivores and stray/feral domestic cats in the northeastern United States. Using a pan-CoV PCR, we screened tissues, feces, and saliva, nasal, and rectal swabs. We detected CoV RNA in fecal and saliva samples of animals in four species: fisher (Pekania pennanti), bobcat (Lynx rufus), red fox (Vulpes vulpes), and domestic cat (Felis catus). Next-generation sequencing revealed that all these viruses belonged to the Luchacovirus subgenus (Alphacoronavirus genus), previously reported only in rodents and lagomorphs (i.e., rabbits). Genetic comparison of the 3-end of the genome ([~]12,000bp) revealed that although the viruses detected group with, and have a genetic organization similar to other luchacoviruses, they are genetically distinct from those from rodents and lagomorphs. Genetic characterization of the spike protein revealed that the meso-carnivore luchacoviruses do not have an S1/S2 cleavage motif but do have highly variable structural loops containing cleavage motifs similar to those identified in certain pathogenic CoVs. This study highlights the importance of characterizing the spike protein of CoVs in wild species for further targeted epidemiologic monitoring. ImportanceSeveral coronaviruses (CoVs) have been detected in domesticated, farmed, and wild meso-carnivores, causing a wide range of diseases, and infecting diverse species, highlighting their important but understudied role in the epidemiology of these viruses. Assessing the viral diversity hosted in wildlife species is essential to understand their significance in the cross-species transmission of CoVs. Our focus here was on CoV discovery in meso-carnivores in the Northeast USA as a potential \"hotspot\" area with high density of humans and urban wildlife. This study identifies novel alphacoronaviruses circulating in multiple free-ranging wild and domestic species in this area and explores their potential epidemiological importance based on regions of the Spike gene that are relevant for virus-host interactions.\nDetection of Mycobacterium tuberculosis in human tissue via RNA in situ hybridization\nAuthors: Nargan, k.; Naidoo, T.; Msimang, M.; Nadeem, S.; Wells, G.; Hunter, R. L.; Hutton, A.; Lumamba, K.; Glasgow, J. N.; Benson, P. V.; Steyn, A. J.\nScore: 10.9, Published: 2023-10-05 DOI: 10.1101/2023.10.04.560963\nRationale: Accurate TB diagnosis is hampered by the variable efficacy of the widely-used Ziehl-Neelsen (ZN) staining method to identify Mycobacterium tuberculosis (Mtb) acid-fast bacilli (AFB). Here, we sought to circumvent this current limitation through direct detection of Mtb mRNA. Objectives: To employ RNAscope to determine the spatial distribution of Mtb mRNA within tuberculous human tissue, to appraise ZN-negative tissue from confirmed TB patients, and to provide proof-of-concept of RNAscope as a platform to inform TB diagnosis and Mtb biology. Methods: We examined ante- and postmortem human TB tissue using RNAscope to detect Mtb mRNA and a dual ZN/immunohistochemistry staining approach to identify AFB and bacilli producing antigen 85B (Ag85B). Measurements and main results: We adapted RNAscope for Mtb and identified intact and disintegrated Mtb bacilli and intra- and extracellular Mtb mRNA. Mtb mRNA was distributed zonally within necrotic and non-necrotic granulomas. We also found Mtb mRNA within, and adjacent to, necrotic granulomas in ZN-negative lung tissue and in Ag85B-positive bronchial epithelium. Intriguingly, we observed accumulation of Mtb mRNA and Ag85B in the cytoplasm of host cells. Notably, many AFB were negative for Ag85B staining. Mtb mRNA was observed in ZN-negative antemortem lymph node biopsies. Conclusions: RNAscope has diagnostic potential and can guide therapeutic intervention as it detects Mtb mRNA and morphology in ZN-negative tissues from TB patients, and Mtb mRNA in ZN-negative antemortem biopsies, respectively. Lastly, our data provide evidence that at least two phenotypically distinct populations of Mtb bacilli exist in vivo.\n",
  "wordCount" : "2783",
  "inLanguage": "en",
  "datePublished": "2023-10-08T10:37:16Z",
  "dateModified": "2023-10-08T10:37:16Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta"><span>updated on October 8, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.561070">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.561070" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.561070">
        <p class="paperTitle">Phospholipid transport to the bacterial outer membrane through an envelope-spanning bridge</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.561070" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.561070" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Isom, G.; Cooper, B. F.; Clark, R.; Kudhail, A.; Ekiert, D. C.; Bhabha, G.; Khalid, S.</p>
        <p class="info">Score: 52.9, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.561070' target='https://doi.org/10.1101/2023.10.05.561070'> 10.1101/2023.10.05.561070</a></p>
        <p class="abstract">The outer membrane of Gram-negative bacteria provides a formidable barrier, essential for both pathogenesis and antimicrobial resistance. Biogenesis of the outer membrane requires the transport of phospholipids across the cell envelope. Recently, YhdP was implicated as a major protagonist in the transport of phospholipids from the inner membrane to the outer membrane however the molecular mechanism of YhdP mediated transport remains elusive. Here, utilising AlphaFold, we observe YhdP to form an elongated assembly of 60 beta strands that curve to form a continuous hydrophobic groove. This architecture is consistent with our negative stain electron microscopy data which reveals YhdP to be approximately 250 Angstrom in length and thus sufficient to span the bacterial cell envelope. Furthermore, molecular dynamics simulations and in vivo bacterial growth assays indicate essential helical regions at the N- and C-termini of YhdP, that may embed into the inner and outer membranes respectively, reinforcing its envelope spanning nature. Our in vivo crosslinking data reveal phosphate-containing substrates captured along the length of the YhdP groove, providing direct evidence that YhdP transports phospholipids. This finding is congruent with our molecular dynamics simulations which demonstrate the propensity for inner membrane lipids to spontaneously enter the groove of YhdP. Collectively, our results support a model in which YhdP bridges the cell envelope, providing a hydrophobic environment for the transport of phospholipids to the outer membrane.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.24.559214">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.24.559214" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.24.559214">
        <p class="paperTitle">Antigenicity and receptor affinity of SARS-CoV-2 BA.2.86 spike</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.24.559214" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.24.559214" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Q.; Guo, Y.; Liu, L.; Schwanz, L. T.; Li, Z.; Ho, J.; Zhang, R. M.; Iketani, S.; Yu, J.; Huang, Y.; Qu, Y.; Valdez, R.; Lauring, A. S.; Gordon, A.; Wang, H. H.; Liu, L.; Ho, D. D.</p>
        <p class="info">Score: 199.1, Published: 2023-09-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.24.559214' target='https://doi.org/10.1101/2023.09.24.559214'> 10.1101/2023.09.24.559214</a></p>
        <p class="abstract">Although the COVID-19 pandemic has officially ended1, SARS-CoV-2 continues to spread and evolve. Recent infections have been dominated by XBB.1.5 and EG.5.1 subvariants2. A new subvariant designated BA.2.86 has just emerged, spreading to 21 countries in 5 continents3. This virus contains 34 spike mutations compared to its BA.2 predecessor, thereby raising concerns about its propensity to evade existing antibodies. We examined its antigenicity using human sera and monoclonal antibodies (mAbs). Reassuringly, BA.2.86 was not more resistant to human sera than XBB.1.5 and EG.5.1, indicating that the new subvariant would not have a growth advantage in this regard. Importantly, sera from patients who had XBB breakthrough infection exhibited robust neutralizing activity against all viruses tested, suggesting that upcoming XBB.1.5 monovalent vaccines could confer added protection. The finding that the longer genetic distance of BA.2.86 did not yield a larger antigenic distance was partially explained by the mAb data. While BA.2.86 showed greater resistance to mAbs to subdomain 1 (SD1) and receptor-binding domain (RBD) class 2 and 3 epitopes, it was more sensitive to mAbs to class 1 and 4/1 epitopes in the &#34;inner face&#34; of RBD that is exposed only when this domain is in the &#34;up&#34; position. We also identified six new spike mutations that mediate antibody resistance, including E554K that threatens SD1 mAbs in clinical development. The BA.2.86 spike also had a remarkably high receptor affinity. The ultimate trajectory of this new SARS-CoV-2 variant will soon be revealed by continuing surveillance, but its worldwide spread is worrisome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.560628">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.560628" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.560628">
        <p class="paperTitle">Virological characteristics correlating with SARS-CoV-2 spike protein fusogenicity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.560628" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.560628" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Begum, M. M.; Ichihara, K.; Takahashi, O.; Nasser, H.; Jonathan, M.; Tokunaga, K.; Yoshida, I.; Nagashima, M.; Sadamasu, K.; Yoshimura, K.; Sato, K.; Ikeda, T.</p>
        <p class="info">Score: 37.8, Published: 2023-10-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.560628' target='https://doi.org/10.1101/2023.10.03.560628'> 10.1101/2023.10.03.560628</a></p>
        <p class="abstract">The severe acute respiratory syndrome coronavirus (SARS-CoV-2) spike (S) protein is essential in mediating membrane fusion of the virus with the target cells. Several reports demonstrated that SARS-CoV-2 S protein fusogenicity is reportedly closely associated with the intrinsic pathogenicity of the virus determined using hamster models. However, the association between S protein fusogenicity and other virological parameters remains elusive. In this study, we investigated the virological parameters of eleven previous variants of concern (VOCs) and variants of interest (VOIs) correlating with S protein fusogenicity. S protein fusogenicity was found to be strongly correlated with S1/S2 cleavage efficiency and plaque size formed by clinical isolates. However, S protein fusogenicity was less associated with pseudoviral infectivity, pseudovirus entry efficiency, and viral replication kinetics. Taken together, our results suggest that S1/S2 cleavage efficiency and plaque size could be potential indicators to predict the intrinsic pathogenicity of newly emerged SARS-CoV-2 variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560498">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560498" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560498">
        <p class="paperTitle">Global RNA interactome of nitrogen starved Escherichia coli uncovers a conserved post-transcriptional regulatory axis required for optimal growth recovery</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560498" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560498" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McQuail, J.; Matera, G.; Gräfenhan, T.; Bischler, T.; Haberkant, P.; Stein, F.; Vogel, J.; Wigneshweraraj, R.</p>
        <p class="info">Score: 18.8, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560498' target='https://doi.org/10.1101/2023.10.02.560498'> 10.1101/2023.10.02.560498</a></p>
        <p class="abstract">The RNA binding protein Hfq has a central role in the post-transcription control of gene expression in many bacteria. Numerous studies have mapped the transcriptome-wide Hfq-mediated RNA-RNA interactions in growing bacteria or bacteria that have entered short-term growth-arrest. To what extent post-transcriptional regulation underpins gene expression in growth-arrested bacteria remains unknown. Here, we used nitrogen (N) starvation as a model to study the Hfq-mediated RNA interactome as Escherichia coli enter, experience, and exit long-term growth arrest. We observe that the Hfq-mediated RNA interactome undergoes extensive changes during N starvation, with the conserved SdsR sRNA making the most interactions with different mRNA targets exclusively in long-term N-starved E. coli. Taking a proteomics approach, we reveal that in growth-arrested cells SdsR influences gene expression far beyond its direct mRNA targets. We demonstrate that the absence of SdsR significantly compromises the ability of the mutant bacteria to recover growth competitively from the long-term N-starved state and uncover a conserved post-transcriptional regulatory axis which underpins this process.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560816">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560816" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560816">
        <p class="paperTitle">Establishment and comparative genomics of a high-quality collection of mosquito-associated bacterial isolates -- MosAIC (Mosquito-Associated Isolate Collection)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560816" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560816" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Foo, A.; Brettell, L. E.; Nichols, H. L.; 2022 UW-Madison Capstone in Microbiology Students,  ; Medina Munoz, M.; Lysne, J.; Dhokiya, V.; Hoque, A. F.; Brackney, D. E.; Caragata, E. P.; Hutchinson, M.; Jacobs-Lorena, M.; Lampe, D. J.; Martin, E.; Valiente Moro, C.; Povelones, M.; Short, S. M.; Steven, B.; Xu, J.; Paustian, T. D.; Rondon, M. R.; Hughes, G. L.; Coon, K. L.; Heinz, E.</p>
        <p class="info">Score: 16.8, Published: 2023-10-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560816' target='https://doi.org/10.1101/2023.10.04.560816'> 10.1101/2023.10.04.560816</a></p>
        <p class="abstract">Mosquitoes transmit medically important human pathogens, including viruses like dengue virus and parasites such as Plasmodium spp., the causative agent of malaria. Mosquito microbiomes are critically important for the ability of mosquitoes to transmit disease-causing agents. However, while large collections of bacterial isolates and genomic data exist for vertebrate microbiomes, the vast majority of work in mosquitoes to date is based on 16S rRNA gene amplicon data that provides limited taxonomic resolution and no functional information. To address this gap and facilitate future studies using experimental microbiome manipulations, we generated a bacterial Mosquito-Associated Isolate Collection (MosAIC) consisting of 392 bacterial isolates with extensive metadata and high-quality draft genome assemblies that are publicly available for use by the scientific community. MosAIC encompasses 142 species spanning 29 bacterial families, with members of the Enterobacteriaceae comprising 40% of the collection. Phylogenomic analysis of three genera, Enterobacter, Serratia, and Elizabethkingia, reveal lineages of mosquito-associated bacteria isolated from different mosquito species in multiple laboratories. Investigation into species&#39; pangenomes further reveals clusters of genes specific to these lineages, which are of interest for future work to identify functions underlying mosquito host association. Altogether, we describe the generation of a physical collection of mosquito-associated bacterial isolates, their genomic data, and analyses of selected groups in context of genome data from closely related isolates, providing a unique, highly valuable resource to investigate factors for bacterial colonisation and adaptation within mosquito hosts. Future efforts will expand the collection to include broader geographic and host species representation, especially from individuals collected from field populations, as well as other mosquito-associated microbes, including fungi, archaea, and protozoa.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560488">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560488" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560488">
        <p class="paperTitle">Streptomyces extracellular vesicles are a broad and permissive antimicrobial packaging and delivery system</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560488" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560488" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Meyer, K. J.; Nodwell, J. R.</p>
        <p class="info">Score: 16.3, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560488' target='https://doi.org/10.1101/2023.10.02.560488'> 10.1101/2023.10.02.560488</a></p>
        <p class="abstract">Streptomyces are the primary source of bioactive specialized metabolites used in research and medicine, including many antimicrobials. These are presumed to be secreted and function as freely soluble compounds. However, increasing evidence suggests extracellular vesicles are an alternative secretion system. We assessed environmental and lab-adapted Streptomyces (sporulating filamentous actinomycetes) and found a majority of strains produce vesicles packaging antimicrobial compounds. The cargo included actinomycins, anthracyclines, candicidin, and actinorhodin, reflecting both diverse chemical properties and diverse antibacterial and antifungal activity. The production of the antimicrobials and vesicles was linked, although vesicle production continued in the absence of antimicrobials. We demonstrated that antimicrobial-containing vesicles achieve direct delivery of the cargo to other microbes. Notably, this delivery via membrane fusion occurred to a broad range of microbes including pathogenic bacteria and yeast. Vesicle encapsulation offers a broad and permissive packaging and delivery system for antimicrobial specialized metabolites, with important implications for ecology and translation.

GRAPHICAL ABSTRACT

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=185 SRC=&#34;FIGDIR/small/560488v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (48K):
org.highwire.dtl.DTLVardef@1d0eb3dorg.highwire.dtl.DTLVardef@1041fdcorg.highwire.dtl.DTLVardef@1c7b679org.highwire.dtl.DTLVardef@9cf51c_HPS_FORMAT_FIGEXP  M_FIG C_FIG IMPORTANCEExtracellular vesicle encapsulation changes our picture of how antimicrobial metabolites function in the environment, and provides an alternative translational approach for delivery of antimicrobials. We find the majority of Streptomyces strains are capable of releasing antimicrobial vesicles, and at least four distinct classes of compounds can be packaged, suggesting this is widespread in nature. This is a striking departure from the primary paradigm of secretion and action of specialized metabolites as soluble compounds. Importantly the vesicles deliver antimicrobial metabolites directly to other microbes via membrane fusion, including to pathogenic bacteria and yeast. This suggests future applications in which lipid-encapsulated natural product antibiotics and antifungals could be used to solve some of the most pressing problems in drug resistance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560492">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560492" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560492">
        <p class="paperTitle">The Biofilm Lifestyle Shapes the Evolution of β-Lactamases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560492" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560492" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lorentzen, O. M.; Haukefer, A. S. B.; Johnsen, P. J.; Frohlich, C.</p>
        <p class="info">Score: 16.3, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560492' target='https://doi.org/10.1101/2023.10.02.560492'> 10.1101/2023.10.02.560492</a></p>
        <p class="abstract">The evolutionary relationship between the biofilm lifestyle and antibiotic resistance enzymes remains a subject of limited understanding. Here, we investigate how {beta}-lactamases affect biofilm formation in Vibrio cholerae and how selection for a biofilm lifestyle impacts the evolution of these enzymes. Seven genetically diverse {beta}-lactamases expressed in V. cholerae displayed a strong inhibitory effect on biofilm production, ranging from 17% to 61%. To understand how natural evolution affects this antagonistic pleiotropy under biofilm selecting conditions, we randomly mutagenized one {beta}-lactamase and selected for elevated biofilm formation. Our results revealed that biofilm evolution selects for mutations predominantly clustered around the {beta}-lactamases active site, yielding functional variants still proficient in {beta}-lactam hydrolysis without biofilm inhibition. Mutational analysis of evolved variants demonstrated that restoration of biofilm development could be achieved either independent of enzymatic function or by actively leveraging enzymatic activity to increase biofilm formation. Taken together, the biofilm lifestyle can impose a profound selective pressure on antimicrobial resistance enzymes. Shedding light on such evolutionary interplays is of great importance to understand the various factors driving antimicrobial resistance.

Impact statement{beta}-lactamases inhibit biofilm formation and the selection for increased biofilm production can mitigate this antagonistic pleiotropic effect. The emergence of {beta}-lactamase variants avoiding biofilm inhibition strongly suggests that the biofilm lifestyle affects the evolutionary fate of these enzymes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.560426">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.560426" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.560426">
        <p class="paperTitle">SARS-CoV-2 N protein-induced Dicer, XPO5, SRSF3, and hnRNPA3 downregulation causes pneumonia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.560426" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.560426" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Luo, Y.-W.; Zhou, J.-P.; Ji, H.; Zheng, A.; Wang, X.; Dai, Z.; Luo, Z.; Cao, F.; Wang, X.-Y.; Bai, Y.; Chen, D.; Chen, Y.; Wang, Q.; Yang, Y.; Zhang, X.; Chiu, S.; Huang, A.-L.; Tang, K.-F.</p>
        <p class="info">Score: 13.4, Published: 2023-10-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.560426' target='https://doi.org/10.1101/2023.10.03.560426'> 10.1101/2023.10.03.560426</a></p>
        <p class="abstract">Age is a major risk factor for coronavirus disease (COVID-19)-associated severe pneumonia and mortality; however, the underlying mechanism remains unclear. Herein, we investigated whether age-related deregulation of RNAi components and RNA splicing factors affects COVID-19 severity. Decreased expression of RNAi components (Dicer and XPO5) and splicing factors (SRSF3 and hnRNPA3) correlated with increased severity of COVID-19 and SARS-CoV-2 nucleocapsid (N) protein-induced pneumonia. N protein induced autophagic degradation of Dicer, XPO5, SRSF3, and hnRNPA3, repressing miRNA biogenesis and RNA splicing and inducing DNA damage, proteotoxic stress, and pneumonia. Dicer, XPO5, SRSF3, and hnRNPA3 were downregulated with age in mouse lung tissues. Older mice experienced more severe N protein-induced pneumonia than younger mice. However, treatment with a poly(ADP-ribose) polymerase inhibitor (PJ34) or aromatase inhibitor (anastrozole) relieved N protein-induced pneumonia by restoring Dicer, XPO5, SRSF3, and hnRNPA3 expression. These findings will aid in developing improved treatments for SARS-CoV-2-associated pneumonia.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.05.31.541188">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.05.31.541188" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.05.31.541188">
        <p class="paperTitle">Detection and characterization of novel luchacoviruses, genus Alphacoronavirus, shed in saliva and feces of meso-carnivores in the northeastern United States</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.05.31.541188" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.05.31.541188" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Olarte-Castillo, X. A.; Plimpton, L. D.; McQueary, H. C.; Sun, Y.; Yu, Y. T.; Cover, S. C.; Richardson, A. N.; Jin, Y.; Grenier, J. K.; Cummings, K. J.; Bunting, E. M.; Diuk-Wasser, M.; Needle, D.; Schuler, K.; Stanhope, M. J.; Whittaker, G.; Goodman, L. B.</p>
        <p class="info">Score: 16.5, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.05.31.541188' target='https://doi.org/10.1101/2023.05.31.541188'> 10.1101/2023.05.31.541188</a></p>
        <p class="abstract">Small to mid-sized carnivores, or meso-carnivores, comprise a group of diverse mammals, many of which can adapt to anthropogenically disturbed environments. Wild meso-carnivores living in urban areas may get exposed to or spread pathogens to other species, including stray/feral domestic animals. Several coronaviruses (CoVs) have been detected in domesticated and farmed meso-carnivores, but knowledge of CoVs circulating in free-ranging wild meso-carnivores remains limited. In this study, we analyzed 321 samples collected between 2016 and 2022 from 9 species of free-ranging wild meso-carnivores and stray/feral domestic cats in the northeastern United States. Using a pan-CoV PCR, we screened tissues, feces, and saliva, nasal, and rectal swabs. We detected CoV RNA in fecal and saliva samples of animals in four species: fisher (Pekania pennanti), bobcat (Lynx rufus), red fox (Vulpes vulpes), and domestic cat (Felis catus). Next-generation sequencing revealed that all these viruses belonged to the Luchacovirus subgenus (Alphacoronavirus genus), previously reported only in rodents and lagomorphs (i.e., rabbits). Genetic comparison of the 3-end of the genome ([~]12,000bp) revealed that although the viruses detected group with, and have a genetic organization similar to other luchacoviruses, they are genetically distinct from those from rodents and lagomorphs. Genetic characterization of the spike protein revealed that the meso-carnivore luchacoviruses do not have an S1/S2 cleavage motif but do have highly variable structural loops containing cleavage motifs similar to those identified in certain pathogenic CoVs. This study highlights the importance of characterizing the spike protein of CoVs in wild species for further targeted epidemiologic monitoring.

ImportanceSeveral coronaviruses (CoVs) have been detected in domesticated, farmed, and wild meso-carnivores, causing a wide range of diseases, and infecting diverse species, highlighting their important but understudied role in the epidemiology of these viruses. Assessing the viral diversity hosted in wildlife species is essential to understand their significance in the cross-species transmission of CoVs. Our focus here was on CoV discovery in meso-carnivores in the Northeast USA as a potential &#34;hotspot&#34; area with high density of humans and urban wildlife. This study identifies novel alphacoronaviruses circulating in multiple free-ranging wild and domestic species in this area and explores their potential epidemiological importance based on regions of the Spike gene that are relevant for virus-host interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560963">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560963" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560963">
        <p class="paperTitle">Detection of Mycobacterium tuberculosis in human tissue via RNA in situ hybridization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560963" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560963" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nargan, k.; Naidoo, T.; Msimang, M.; Nadeem, S.; Wells, G.; Hunter, R. L.; Hutton, A.; Lumamba, K.; Glasgow, J. N.; Benson, P. V.; Steyn, A. J.</p>
        <p class="info">Score: 10.9, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560963' target='https://doi.org/10.1101/2023.10.04.560963'> 10.1101/2023.10.04.560963</a></p>
        <p class="abstract">Rationale: Accurate TB diagnosis is hampered by the variable efficacy of the widely-used Ziehl-Neelsen (ZN) staining method to identify Mycobacterium tuberculosis (Mtb) acid-fast bacilli (AFB). Here, we sought to circumvent this current limitation through direct detection of Mtb mRNA. Objectives: To employ RNAscope to determine the spatial distribution of Mtb mRNA within tuberculous human tissue, to appraise ZN-negative tissue from confirmed TB patients, and to provide proof-of-concept of RNAscope as a platform to inform TB diagnosis and Mtb biology. Methods: We examined ante- and postmortem human TB tissue using RNAscope to detect Mtb mRNA and a dual ZN/immunohistochemistry staining approach to identify AFB and bacilli producing antigen 85B (Ag85B). Measurements and main results: We adapted RNAscope for Mtb and identified intact and disintegrated Mtb bacilli and intra- and extracellular Mtb mRNA. Mtb mRNA was distributed zonally within necrotic and non-necrotic granulomas. We also found Mtb mRNA within, and adjacent to, necrotic granulomas in ZN-negative lung tissue and in Ag85B-positive bronchial epithelium. Intriguingly, we observed accumulation of Mtb mRNA and Ag85B in the cytoplasm of host cells. Notably, many AFB were negative for Ag85B staining. Mtb mRNA was observed in ZN-negative antemortem lymph node biopsies. Conclusions: RNAscope has diagnostic potential and can guide therapeutic intervention as it detects Mtb mRNA and morphology in ZN-negative tissues from TB patients, and Mtb mRNA in ZN-negative antemortem biopsies, respectively. Lastly, our data provide evidence that at least two phenotypically distinct populations of Mtb bacilli exist in vivo.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
