<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Phenotyping the virulence of SARS-CoV-2 variants in hamsters by digital pathology and machine learning
Authors: Meehan, G. R.; Herder, V.; Allan, J.; Huang, X.; Kerr, K.; Mendonca, D. C.; Ilia, G.; Wright, D.; Nomikou, K.; Gu, Q.; Arias, S. M.; De Lorenzo, G.; Cowton, V.; Upfold, N.; Palmalux, N.; Brown, J. C.; Barclay, W.; Filipe, A. D. S.; Furnon, W.; Patel, A.; Palmarini, M.
Score: 915.1, Published: 2023-08-01 DOI: 10.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Phenotyping the virulence of SARS-CoV-2 variants in hamsters by digital pathology and machine learning
Authors: Meehan, G. R.; Herder, V.; Allan, J.; Huang, X.; Kerr, K.; Mendonca, D. C.; Ilia, G.; Wright, D.; Nomikou, K.; Gu, Q.; Arias, S. M.; De Lorenzo, G.; Cowton, V.; Upfold, N.; Palmalux, N.; Brown, J. C.; Barclay, W.; Filipe, A. D. S.; Furnon, W.; Patel, A.; Palmarini, M.
Score: 915.1, Published: 2023-08-01 DOI: 10." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-06T10:36:48+00:00" />
<meta property="article:modified_time" content="2023-08-06T10:36:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Phenotyping the virulence of SARS-CoV-2 variants in hamsters by digital pathology and machine learning
Authors: Meehan, G. R.; Herder, V.; Allan, J.; Huang, X.; Kerr, K.; Mendonca, D. C.; Ilia, G.; Wright, D.; Nomikou, K.; Gu, Q.; Arias, S. M.; De Lorenzo, G.; Cowton, V.; Upfold, N.; Palmalux, N.; Brown, J. C.; Barclay, W.; Filipe, A. D. S.; Furnon, W.; Patel, A.; Palmarini, M.
Score: 915.1, Published: 2023-08-01 DOI: 10."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Phenotyping the virulence of SARS-CoV-2 variants in hamsters by digital pathology and machine learning\nAuthors: Meehan, G. R.; Herder, V.; Allan, J.; Huang, X.; Kerr, K.; Mendonca, D. C.; Ilia, G.; Wright, D.; Nomikou, K.; Gu, Q.; Arias, S. M.; De Lorenzo, G.; Cowton, V.; Upfold, N.; Palmalux, N.; Brown, J. C.; Barclay, W.; Filipe, A. D. S.; Furnon, W.; Patel, A.; Palmarini, M.\nScore: 915.1, Published: 2023-08-01 DOI: 10.",
  "keywords": [
    
  ],
  "articleBody": " Phenotyping the virulence of SARS-CoV-2 variants in hamsters by digital pathology and machine learning\nAuthors: Meehan, G. R.; Herder, V.; Allan, J.; Huang, X.; Kerr, K.; Mendonca, D. C.; Ilia, G.; Wright, D.; Nomikou, K.; Gu, Q.; Arias, S. M.; De Lorenzo, G.; Cowton, V.; Upfold, N.; Palmalux, N.; Brown, J. C.; Barclay, W.; Filipe, A. D. S.; Furnon, W.; Patel, A.; Palmarini, M.\nScore: 915.1, Published: 2023-08-01 DOI: 10.1101/2023.08.01.551417\nSARS-CoV-2 has continued to evolve throughout the COVID-19 pandemic, giving rise to multiple variants of concern (VOCs) with different biological properties. As the pandemic progresses, it will be essential to test in near real time the potential of any new emerging variant to cause severe disease. BA.1 (Omicron) was shown to be attenuated compared to the previous VOCs like Delta, but it is possible that newly emerging variants may regain a virulent phenotype. Hamsters have been proven to be an exceedingly good model for SARS-CoV-2 pathogenesis. Here, we aimed to develop robust quantitative pipelines to assess the virulence of SARS-CoV-2 variants in hamsters. We used various approaches including RNAseq, RNA in situ hybridization, immunohistochemistry, and digital pathology, including software assisted whole section imaging and downstream automatic analyses enhanced by machine learning, to develop methods to assess and quantify virus-induced pulmonary lesions in an unbiased manner. Initially, we used Delta and Omicron to develop our experimental pipelines. We then assessed the virulence of recent Omicron sub-lineages including BA.5, XBB, BQ.1.18, BA.2 and BA.2.75. We show that in experimentally infected hamsters, accurate quantification of alveolar epithelial hyperplasia and macrophage infiltrates represent robust markers for assessing the extent of virus-induced pulmonary pathology, and hence virus virulence. In addition, using these pipelines, we could reveal how some Omicron sub-lineages (e.g., BA.2.75) have regained virulence compared to the original BA.1. Finally, to maximise the utility of the digital pathology pipelines reported in our study, we developed an online repository containing representative whole organ histopathology sections that can be visualised at variable magnifications (https://covid-atlas.cvr.gla.ac.uk). Overall, this pipeline can provide unbiased and invaluable data for rapidly assessing newly emerging variants and their potential to cause severe disease.\nEpigenetic liquid biopsies reveal elevated vascular endothelial cell turnover and erythropoiesis in asymptomatic COVID-19 patients\nAuthors: Ben Ami, R.; Loyfer, N.; Cohn, E.; Fialkoff, G.; Sharkia, I.; Bogot, N.; Kochan, D.; Kalak, G.; Jarjoui, A.; Chen-Shuali, C.; Azulai, H.; Barhoum, H.; Arish, N.; Greenberger, M. M.; Velleman, D.; Kurd, R.; Chetrit, E. B.; Bohm, D.; Wolak, T.; Quteineh, A.; Cann, G.; Glaser, B.; Friedman, N.; Kaplan, T.; Shemer, R.; Rokach, A.; Dor, Y.\nScore: 274.5, Published: 2023-08-01 DOI: 10.1101/2023.07.28.550957\nThe full spectrum of tissues affected by SARS-CoV-2 infection is crucial for deciphering the heterogenous clinical course of COVID-19. Here, we analyzed DNA methylation and histone modification patterns in circulating chromatin to assess cell type-specific turnover in severe and asymptomatic COVID-19 patients, in relation to clinical outcome. Patients with severe COVID-19 had a massive elevation of circulating cell-free DNA (cfDNA) levels, which originated in lung epithelial cells, cardiomyocytes, vascular endothelial cells and erythroblasts, suggesting increased cell death or turnover in these tissues. The immune response to infection was reflected by elevated B cell and monocyte/macrophage cfDNA levels, and by evidence of an interferon response in cells prior to cfDNA release. Strikingly, monocyte/macrophage cfDNA levels (but not monocyte counts), as well as lung epithelium cfDNA and vascular endothelial cfDNA, predicted clinical deterioration and duration of hospitalization. Asymptomatic patients had elevated levels of immune-derived cfDNA but did not show evidence of pulmonary or cardiac damage. Surprisingly, these patients showed elevated levels of vascular endothelial cell and erythroblast cfDNA, suggesting that sub-clinical vascular and erythrocyte turnover are universal features of COVID-19, independent of disease severity. Epigenetic liquid biopsies provide non-invasive means of monitoring COVID-19 patients, and reveal sub-clinical vascular damage and red blood cell turnover.\nA patchwork pathway of apparently recent origin enables degradation of the synthetic buffer compound TRIS in bacteria\nAuthors: Holert, J.; Borker, A.; Nuebel, L. L.; Daniel, R.; Poehlein, A.; Philipp, B.\nScore: 43.8, Published: 2023-08-01 DOI: 10.1101/2023.08.01.551466\nThe widely used synthetic chemical 2-amino-2-hydroxymethyl-propane-1,3-diol (TRIS) was long considered to be biologically inert. Herein, we describe a complete degradation pathway for the catabolism of TRIS in bacteria. By serendipity, Pseudomonas hunanensis Teo1 was isolated from sewage sludge and shown to be able to grow with TRIS as only carbon and nitrogen source. Genome sequencing and transcriptome analyses revealed two adjacent gene clusters embedded in a mobile genetic element on a conjugative plasmid to be upregulated during growth with TRIS. Conjugational transfer of this plasmid into P. putida KT2440 enabled this strain to grow with TRIS confirming that these clusters encode the complete TRIS degradation pathway. Heterologous gene expression in Escherichia coli revealed cluster I to encode a TRIS uptake protein, a TRIS dehydrogenase, and a TRIS aldehyde dehydrogenase, which catalyze the oxidation of TRIS into 2-hydroxymethylserine. Phylogenetic analysis suggests that the TRIS dehydrogenase proteins evolved from choline dehydrogenases. Genes encoded in cluster II encode a methylserine hydroxymethyltransferase and a D-serine dehydratase which could plausibly catalyze conversion of 2-hydroxymethylserine into pyruvate. Subsequent enrichments from wastewater purification systems led to the isolation of further TRIS-degrading bacteria from the Pseudomonas and Shinella genera which all carried the respective gene clusters in a similar genetic context as strain Teo1. Our data indicate that TRIS degradation evolved by gene assembly and enzyme adaptation from multiple independent metabolic pathways. Genomic database searches showed that TRIS degradation is now globally distributed, confirming its dissemination by horizontal gene transfer. Significance statementTRIS is a synthetic organic chemical that has been mass-produced since the 1950s and is now used in countless applications in industry, medicine, and households world-wide, in parts due to its presumed biological inertness. Our results challenge this common conception, showing that TRIS is biologically degradable by an apparently recently evolved pathway, providing insight into the evolution of metabolic pathways that make such ubiquitous chemicals available as growth substrates for bacteria. Such knowledge is valuable for developing new processes of wastewater purification in times of increasing water scarcity.\nBorg extrachromosomal elements of methane-oxidizing archaea have conserved and expressed genetic repertoires\nAuthors: Schoelmerich, M. C.; Ly, L.; West-Roberts, J.; Shi, L.-D.; Shen, C.; Malvankar, N.; Taib, N.; Gribaldo, S.; Woodcroft, B. J.; Al-Shayeb, B.; Dai, X.; Mozsary, C.; Hickey, S.; He, C.; Beaulaurier, J. A.; Juul, S.; Sachdeva, R.; Banfield, J.\nScore: 37.5, Published: 2023-08-01 DOI: 10.1101/2023.08.01.549754\nBorgs are huge extrachromosomal elements (ECE) of anaerobic methane-consuming \"Candidatus Methanoperedens\" archaea. Here, we used nanopore sequencing to validate published complete genomes curated from short reads and to reconstruct new genomes. 13 complete and four near-complete linear genomes share 40 genes that define a largely syntenous genome backbone. We use these conserved genes to identify new Borgs from peatland soil and to delineate Borg phylogeny, revealing two major clades. Remarkably, Borg genes encoding OmcZ nanowire-like electron-exporting cytochromes and cell surface proteins are more highly expressed than those of host Methanoperedens, indicating that Borgs augment the Methanoperedens activity in situ. We reconstructed the first complete 4.00 Mbp genome for a Methanoperedens that is inferred to be a Borg host and predicted its methylation motifs, which differ from pervasive TC and CC methylation motifs of the Borgs. Thus, methylation may enable Methanoperedens to distinguish their genomes from those of Borgs. Very high Borg to Methanoperedens ratios and structural predictions suggest that Borgs may be capable of encapsulation. The findings clearly define Borgs as a distinct class of ECE with shared genomic signatures, establish their diversification from a common ancestor with genetic inheritance, and raise the possibility of periodic existence outside of host cells.\nMissing microbial eukaryotes and misleading meta-omic conclusions\nAuthors: Krinos, A. I.; Mars Brisbin, M.; Hu, S. K.; Cohen, N. R.; Rynearson, T. A.; Follows, M. J.; Schulz, F.; Alexander, H.\nScore: 26.0, Published: 2023-07-30 DOI: 10.1101/2023.07.30.551153\nMeta-omics has become commonplace in the study of microbial eukaryotes. The explosion of available data has inspired large-scale analyses, including species or taxonomic group distribution mapping, gene catalog construction, and inference on the functional roles and activities of microbial eukaryotes in situ. However, genome and transcriptome databases are prone to misannotation biases, and meta-omic inventories may have no recoverable taxonomic annotation for more than half of assembled contigs or predicted proteins. Direct mapping solely to organisms of interest might introduce a problematic misattribution bias, while full databases can annotate any cataloged organism but may be imbalanced between taxa. Here, we explore the potential pitfalls of common approaches to taxonomic annotation of protistan meta-omic datasets. We argue that ongoing curation of genetic resources is critical in accurately annotating protists in situ in meta-omic datasets. Moreover, we propose that precise taxonomic annotation of meta-omic data is a clustering problem rather than a feasible alignment problem. We show that taxonomic membership of sequence clusters demonstrates more accurate estimated community composition than returning exact sequence labels, and overlap between clusters can address database shortcomings. Clustering approaches can be applied to diverse environments while continuing to exploit the wealth of annotation data collated in databases, and database selection and evaluation is a critical part of correctly annotating protistan taxonomy in environmental datasets. We re-analyze three environmental datasets at three levels of taxonomic hierarchy in order to illustrate the critical importance of both database completeness and curation in enabling accurate environmental interpretation.\nAn interbacterial lipase toxin with an unprecedented reverse domain arrangement defines a new class of type VII secretion system effector\nAuthors: Garrett, S. R.; Mietrach, N.; Deme, J.; Bitzer, A.; Yang, Y.; Ulhuq, F. R.; Kretschmer, D.; Heilbronner, S.; Smith, T. K.; Lea, S. M.; Palmer, T.\nScore: 21.4, Published: 2023-07-29 DOI: 10.1101/2023.07.29.551094\nThe type VII protein secretion system (T7SS) is found in many Gram-positive bacteria and in pathogenic mycobacteria. All T7SS substrate proteins described to date share a common helical domain architecture at the N-terminus that typically interacts with other helical partner proteins, forming a composite signal sequence for targeting to the T7SS. The C-terminal domains are functionally diverse and in Gram-positive bacteria such as Staphylococcus aureus often specify toxic anti-bacterial activity. Here we describe the first example of a new class of T7 substrate, TslA, that has an unexpected reverse domain organisation. TslA is widely found across Bacillota including Staphylococcus, Enterococcus and Listeria. We show that the S. aureus TslA N-terminal domain is a phospholipase A with anti-staphylococcal activity that is neutralised by the immunity lipoprotein TilA. Two small helical partner proteins, TlaA1 and TlaA2 are essential for T7-dependent secretion of TslA and at least one of these interacts with the TslA C-terminal domain to form a helical stack. Cryo-EM analysis of purified TslA complexes indicate that they share structural similarity with canonical T7 substrates. Our findings suggest that the T7SS has the extraordinary feature of recognising a secretion signal present at either end of a substrate.\nThe host RNA polymerase II C-terminal domain is the anchor for replication of the influenza virus genome\nAuthors: Krischuns, T.; Arragain, B.; Isel, C.; Paisant, S.; Budt, M.; Wolff, T.; Cusack, S.; Naffakh, N.\nScore: 18.5, Published: 2023-08-01 DOI: 10.1101/2023.08.01.551436\nThe current model is that the influenza virus polymerase (FluPol) binds either to host RNA polymerase II (RNAP II) or to the acidic nuclear phosphoprotein 32 (ANP32), which drives its conformation and activity towards transcription or replication of the viral genome, respectively. Here, we provide evidence that the FluPol-RNAP II binding interface has a so far overlooked function for replication of the viral genome. Using a combination of cell-based and in vitro approaches, we show that the RNAP II C-terminal-domain, jointly with ANP32, enhances FluPol replication activity and we propose a model in which the host RNAP II is the anchor for transcription and replication of the viral genome. Our data open new perspectives on the spatial coupling of viral transcription and replication and the coordinated balance between these two activities.\nThe Feather Epithelium Contributes to the Dissemination and Ecology of clade 2.3.4.4b H5 High Pathogenicity Avian Influenza Virus in Ducks\nAuthors: Gaide, N.; Filaire, F.; Bertran, K.; Crispo, M.; Dirat, M.; Secula, A.; Foret-Lucas, C.; Payre, B.; Perlas, A.; Cantero, G.; Majo, N.; Soubies, S.; GUERIN, J.-L.\nScore: 18.5, Published: 2023-07-26 DOI: 10.1101/2023.07.26.550633\nImmature feathers are known replication sites for high pathogenicity avian influenza viruses (HPAIVs) in poultry. However, it is unclear whether feathers play an active role in viral transmission. This study aims to investigate the contribution of the feather epithelium to the dissemination of clade 2.3.4.4b goose/Guangdong/1996 lineage H5 HPAIVs in the environment, based on natural and experimental infections of domestic ducks. During the 2016-22 outbreaks, H5 HPAIVs exhibited persistent and marked feather epitheliotropism in naturally infected commercial ducks. Infection of feathers resulted in epithelial necrosis, disruption, and the production and release of infectious virions. Viral and feather antigens colocalized in dust samples obtained from poultry barns housing naturally infected birds. In summary, the feather epithelium contributes to viral replication, and it is a likely source of environmental infectious material. This underestimated excretion route could greatly impact the ecology of HPAIVs, facilitating airborne and preening-related infections within a flock, and promoting prolonged viral infectivity and long-distance viral transmission between poultry farms. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=112 SRC=\"FIGDIR/small/550633v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (54K): org.highwire.dtl.DTLVardef@16f6b24org.highwire.dtl.DTLVardef@1e33c82org.highwire.dtl.DTLVardef@199b745org.highwire.dtl.DTLVardef@cf2024_HPS_FORMAT_FIGEXP M_FIG C_FIG\nRecombinant Rotavirus Expressing the Glycosylated S1 Protein of SARS-CoV-2\nAuthors: Philip, A. A.; Hu, S.; Patton, J. T.\nScore: 17.0, Published: 2023-08-01 DOI: 10.1101/2023.08.01.551500\nReverse genetic systems have been used to introduce heterologous sequences into the rotavirus segmented double-stranded (ds)RNA genome, enabling the generation of recombinant viruses that express foreign proteins and possibly serve as vaccine vectors. Notably, insertion of SARS-CoV-2 sequences into the segment 7 (NSP3) RNA of simian SA11 rotavirus was previously shown to result in the production of recombinant viruses that efficiently expressed the N-terminal domain (NTD) and the receptor-binding domain (RBD) of the S1 region of the SARS-CoV-2 spike protein. However, efforts to generate a similar recombinant (r) SA11 virus that efficiently expressed full-length S1 were less successful. In this study, we describe modifications to the S1-coding cassette inserted in the segment 7 RNA that allowed recovery of second-generation rSA11 viruses that efficiently expressed the [~]120-kDa S1 protein. The [~]120-kDa S1 products were shown to be glycosylated, based on treatment with endoglycosidase H, which reduced the protein to a size of [~]80 kDa. Co-pulldown assays demonstrated that the [~]120-kDa S1 proteins had affinity for the human ACE2 receptor. Although all the second-generation rSA11 viruses expressed glycosylated S1 with affinity for the ACE receptor, only the S1 product of one virus (rSA11/S1f) was appropriately recognized by anti-S1 antibody, suggesting the rSA11/S1f virus expressed an authentic form of S1. Probably due to the presence of FLAG tags on their S1 signal peptides, the S1 products of the other viruses (rSA11/3fS1 and rSA11/3fS1-His) may have undergone defective glycosylation, impeding antibody binding. In summary, these results indicate that recombinant rotaviruses can serve as expression vectors of foreign glycosylated proteins, raising the possibility of generating rotavirus-based vaccines that can induce protective immune responses against enteric and mucosal viruses with glycosylated capsid components, including SARS-CoV-2.\nButyrate Protects against SARS-CoV-2-induced Tissue Damage in Golden Hamsters.\nAuthors: Yu, H.; Yuan, L.; Yan, Z.; Zhou, M.; Ye, J.; Wu, K.; Chen, W.; Chen, R.; Xia, N.-S.; Guan, Y.; Zhu, H.\nScore: 18.7, Published: 2023-07-28 DOI: 10.1101/2023.07.27.550811\nButyrate, produced by gut microbe during dietary fiber fermentation, plays anti-inflammatory and antioxidant effects in chronic inflammation diseases, yet it remains to be explored whether butyrate has protective effects against viral infections. Here, we demonstrated that butyrate alleviated tissue injury in severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)-infected golden hamsters with supplementation of butyrate before and during the infection. Butyrate-treated hamsters showed augmentation of type I interferon (IFN) response and activation of endothelial cells without exaggerated inflammation. In addition, butyrate regulated redox homeostasis by enhancing the activity of superoxide dismutase (SOD) to inhibit excessive apoptotic cell death. Therefore, butyrate exhibited an effective prevention against SARS-CoV-2 by upregulating antiviral immune responses and promoting cell survival. IMPORTANCESince SARS-CoV-2 has caused severe disease characterized by acute respiratory distress syndrome (ARDS) in humans, it is essential to develop therapeutics based on relieving such severe clinical symptoms. Current therapy strategies mainly focus on individuals who have COVID-19, however, there is still a strong need for prevention and treatment of SARS-CoV-2 infection. This study showed that butyrate, a bacterial metabolite, improved the response of SARS-CoV-2-infected hamsters by reducing immunopathology caused by impaired antiviral defenses and inhibiting excessive apoptosis through reduction in oxidative stress.\n",
  "wordCount" : "2725",
  "inLanguage": "en",
  "datePublished": "2023-08-06T10:36:48Z",
  "dateModified": "2023-08-06T10:36:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta"><span>updated on August 6, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.551417">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.551417" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.551417">
        <p class="paperTitle">Phenotyping the virulence of SARS-CoV-2 variants in hamsters by digital pathology and machine learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.551417" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.551417" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Meehan, G. R.; Herder, V.; Allan, J.; Huang, X.; Kerr, K.; Mendonca, D. C.; Ilia, G.; Wright, D.; Nomikou, K.; Gu, Q.; Arias, S. M.; De Lorenzo, G.; Cowton, V.; Upfold, N.; Palmalux, N.; Brown, J. C.; Barclay, W.; Filipe, A. D. S.; Furnon, W.; Patel, A.; Palmarini, M.</p>
        <p class="info">Score: 915.1, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.551417' target='https://doi.org/10.1101/2023.08.01.551417'> 10.1101/2023.08.01.551417</a></p>
        <p class="abstract">SARS-CoV-2 has continued to evolve throughout the COVID-19 pandemic, giving rise to multiple variants of concern (VOCs) with different biological properties. As the pandemic progresses, it will be essential to test in near real time the potential of any new emerging variant to cause severe disease. BA.1 (Omicron) was shown to be attenuated compared to the previous VOCs like Delta, but it is possible that newly emerging variants may regain a virulent phenotype. Hamsters have been proven to be an exceedingly good model for SARS-CoV-2 pathogenesis. Here, we aimed to develop robust quantitative pipelines to assess the virulence of SARS-CoV-2 variants in hamsters. We used various approaches including RNAseq, RNA in situ hybridization, immunohistochemistry, and digital pathology, including software assisted whole section imaging and downstream automatic analyses enhanced by machine learning, to develop methods to assess and quantify virus-induced pulmonary lesions in an unbiased manner. Initially, we used Delta and Omicron to develop our experimental pipelines. We then assessed the virulence of recent Omicron sub-lineages including BA.5, XBB, BQ.1.18, BA.2 and BA.2.75. We show that in experimentally infected hamsters, accurate quantification of alveolar epithelial hyperplasia and macrophage infiltrates represent robust markers for assessing the extent of virus-induced pulmonary pathology, and hence virus virulence. In addition, using these pipelines, we could reveal how some Omicron sub-lineages (e.g., BA.2.75) have regained virulence compared to the original BA.1. Finally, to maximise the utility of the digital pathology pipelines reported in our study, we developed an online repository containing representative whole organ histopathology sections that can be visualised at variable magnifications (https://covid-atlas.cvr.gla.ac.uk). Overall, this pipeline can provide unbiased and invaluable data for rapidly assessing newly emerging variants and their potential to cause severe disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.550957">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.550957" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.550957">
        <p class="paperTitle">Epigenetic liquid biopsies reveal elevated vascular endothelial cell turnover and erythropoiesis in asymptomatic COVID-19 patients</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.550957" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.550957" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ben Ami, R.; Loyfer, N.; Cohn, E.; Fialkoff, G.; Sharkia, I.; Bogot, N.; Kochan, D.; Kalak, G.; Jarjoui, A.; Chen-Shuali, C.; Azulai, H.; Barhoum, H.; Arish, N.; Greenberger, M. M.; Velleman, D.; Kurd, R.; Chetrit, E. B.; Bohm, D.; Wolak, T.; Quteineh, A.; Cann, G.; Glaser, B.; Friedman, N.; Kaplan, T.; Shemer, R.; Rokach, A.; Dor, Y.</p>
        <p class="info">Score: 274.5, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.550957' target='https://doi.org/10.1101/2023.07.28.550957'> 10.1101/2023.07.28.550957</a></p>
        <p class="abstract">The full spectrum of tissues affected by SARS-CoV-2 infection is crucial for deciphering the heterogenous clinical course of COVID-19. Here, we analyzed DNA methylation and histone modification patterns in circulating chromatin to assess cell type-specific turnover in severe and asymptomatic COVID-19 patients, in relation to clinical outcome. Patients with severe COVID-19 had a massive elevation of circulating cell-free DNA (cfDNA) levels, which originated in lung epithelial cells, cardiomyocytes, vascular endothelial cells and erythroblasts, suggesting increased cell death or turnover in these tissues. The immune response to infection was reflected by elevated B cell and monocyte/macrophage cfDNA levels, and by evidence of an interferon response in cells prior to cfDNA release. Strikingly, monocyte/macrophage cfDNA levels (but not monocyte counts), as well as lung epithelium cfDNA and vascular endothelial cfDNA, predicted clinical deterioration and duration of hospitalization. Asymptomatic patients had elevated levels of immune-derived cfDNA but did not show evidence of pulmonary or cardiac damage. Surprisingly, these patients showed elevated levels of vascular endothelial cell and erythroblast cfDNA, suggesting that sub-clinical vascular and erythrocyte turnover are universal features of COVID-19, independent of disease severity. Epigenetic liquid biopsies provide non-invasive means of monitoring COVID-19 patients, and reveal sub-clinical vascular damage and red blood cell turnover.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.551466">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.551466" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.551466">
        <p class="paperTitle">A patchwork pathway of apparently recent origin enables degradation of the synthetic buffer compound TRIS in bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.551466" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.551466" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Holert, J.; Borker, A.; Nuebel, L. L.; Daniel, R.; Poehlein, A.; Philipp, B.</p>
        <p class="info">Score: 43.8, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.551466' target='https://doi.org/10.1101/2023.08.01.551466'> 10.1101/2023.08.01.551466</a></p>
        <p class="abstract">The widely used synthetic chemical 2-amino-2-hydroxymethyl-propane-1,3-diol (TRIS) was long considered to be biologically inert. Herein, we describe a complete degradation pathway for the catabolism of TRIS in bacteria. By serendipity, Pseudomonas hunanensis Teo1 was isolated from sewage sludge and shown to be able to grow with TRIS as only carbon and nitrogen source. Genome sequencing and transcriptome analyses revealed two adjacent gene clusters embedded in a mobile genetic element on a conjugative plasmid to be upregulated during growth with TRIS. Conjugational transfer of this plasmid into P. putida KT2440 enabled this strain to grow with TRIS confirming that these clusters encode the complete TRIS degradation pathway. Heterologous gene expression in Escherichia coli revealed cluster I to encode a TRIS uptake protein, a TRIS dehydrogenase, and a TRIS aldehyde dehydrogenase, which catalyze the oxidation of TRIS into 2-hydroxymethylserine. Phylogenetic analysis suggests that the TRIS dehydrogenase proteins evolved from choline dehydrogenases. Genes encoded in cluster II encode a methylserine hydroxymethyltransferase and a D-serine dehydratase which could plausibly catalyze conversion of 2-hydroxymethylserine into pyruvate. Subsequent enrichments from wastewater purification systems led to the isolation of further TRIS-degrading bacteria from the Pseudomonas and Shinella genera which all carried the respective gene clusters in a similar genetic context as strain Teo1.

Our data indicate that TRIS degradation evolved by gene assembly and enzyme adaptation from multiple independent metabolic pathways. Genomic database searches showed that TRIS degradation is now globally distributed, confirming its dissemination by horizontal gene transfer.

Significance statementTRIS is a synthetic organic chemical that has been mass-produced since the 1950s and is now used in countless applications in industry, medicine, and households world-wide, in parts due to its presumed biological inertness. Our results challenge this common conception, showing that TRIS is biologically degradable by an apparently recently evolved pathway, providing insight into the evolution of metabolic pathways that make such ubiquitous chemicals available as growth substrates for bacteria. Such knowledge is valuable for developing new processes of wastewater purification in times of increasing water scarcity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.549754">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.549754" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.549754">
        <p class="paperTitle">Borg extrachromosomal elements of methane-oxidizing archaea have conserved and expressed genetic repertoires</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.549754" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.549754" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schoelmerich, M. C.; Ly, L.; West-Roberts, J.; Shi, L.-D.; Shen, C.; Malvankar, N.; Taib, N.; Gribaldo, S.; Woodcroft, B. J.; Al-Shayeb, B.; Dai, X.; Mozsary, C.; Hickey, S.; He, C.; Beaulaurier, J. A.; Juul, S.; Sachdeva, R.; Banfield, J.</p>
        <p class="info">Score: 37.5, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.549754' target='https://doi.org/10.1101/2023.08.01.549754'> 10.1101/2023.08.01.549754</a></p>
        <p class="abstract">Borgs are huge extrachromosomal elements (ECE) of anaerobic methane-consuming &#34;Candidatus Methanoperedens&#34; archaea. Here, we used nanopore sequencing to validate published complete genomes curated from short reads and to reconstruct new genomes. 13 complete and four near-complete linear genomes share 40 genes that define a largely syntenous genome backbone. We use these conserved genes to identify new Borgs from peatland soil and to delineate Borg phylogeny, revealing two major clades. Remarkably, Borg genes encoding OmcZ nanowire-like electron-exporting cytochromes and cell surface proteins are more highly expressed than those of host Methanoperedens, indicating that Borgs augment the Methanoperedens activity in situ. We reconstructed the first complete 4.00 Mbp genome for a Methanoperedens that is inferred to be a Borg host and predicted its methylation motifs, which differ from pervasive TC and CC methylation motifs of the Borgs. Thus, methylation may enable Methanoperedens to distinguish their genomes from those of Borgs. Very high Borg to Methanoperedens ratios and structural predictions suggest that Borgs may be capable of encapsulation. The findings clearly define Borgs as a distinct class of ECE with shared genomic signatures, establish their diversification from a common ancestor with genetic inheritance, and raise the possibility of periodic existence outside of host cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.30.551153">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.30.551153" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.30.551153">
        <p class="paperTitle">Missing microbial eukaryotes and misleading meta-omic conclusions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.30.551153" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.30.551153" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krinos, A. I.; Mars Brisbin, M.; Hu, S. K.; Cohen, N. R.; Rynearson, T. A.; Follows, M. J.; Schulz, F.; Alexander, H.</p>
        <p class="info">Score: 26.0, Published: 2023-07-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.30.551153' target='https://doi.org/10.1101/2023.07.30.551153'> 10.1101/2023.07.30.551153</a></p>
        <p class="abstract">Meta-omics has become commonplace in the study of microbial eukaryotes. The explosion of available data has inspired large-scale analyses, including species or taxonomic group distribution mapping, gene catalog construction, and inference on the functional roles and activities of microbial eukaryotes in situ. However, genome and transcriptome databases are prone to misannotation biases, and meta-omic inventories may have no recoverable taxonomic annotation for more than half of assembled contigs or predicted proteins. Direct mapping solely to organisms of interest might introduce a problematic misattribution bias, while full databases can annotate any cataloged organism but may be imbalanced between taxa. Here, we explore the potential pitfalls of common approaches to taxonomic annotation of protistan meta-omic datasets. We argue that ongoing curation of genetic resources is critical in accurately annotating protists in situ in meta-omic datasets. Moreover, we propose that precise taxonomic annotation of meta-omic data is a clustering problem rather than a feasible alignment problem. We show that taxonomic membership of sequence clusters demonstrates more accurate estimated community composition than returning exact sequence labels, and overlap between clusters can address database shortcomings. Clustering approaches can be applied to diverse environments while continuing to exploit the wealth of annotation data collated in databases, and database selection and evaluation is a critical part of correctly annotating protistan taxonomy in environmental datasets. We re-analyze three environmental datasets at three levels of taxonomic hierarchy in order to illustrate the critical importance of both database completeness and curation in enabling accurate environmental interpretation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.29.551094">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.29.551094" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.29.551094">
        <p class="paperTitle">An interbacterial lipase toxin with an unprecedented reverse domain arrangement defines a new class of type VII secretion system effector</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.29.551094" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.29.551094" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Garrett, S. R.; Mietrach, N.; Deme, J.; Bitzer, A.; Yang, Y.; Ulhuq, F. R.; Kretschmer, D.; Heilbronner, S.; Smith, T. K.; Lea, S. M.; Palmer, T.</p>
        <p class="info">Score: 21.4, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.29.551094' target='https://doi.org/10.1101/2023.07.29.551094'> 10.1101/2023.07.29.551094</a></p>
        <p class="abstract">The type VII protein secretion system (T7SS) is found in many Gram-positive bacteria and in pathogenic mycobacteria. All T7SS substrate proteins described to date share a common helical domain architecture at the N-terminus that typically interacts with other helical partner proteins, forming a composite signal sequence for targeting to the T7SS. The C-terminal domains are functionally diverse and in Gram-positive bacteria such as Staphylococcus aureus often specify toxic anti-bacterial activity. Here we describe the first example of a new class of T7 substrate, TslA, that has an unexpected reverse domain organisation. TslA is widely found across Bacillota including Staphylococcus, Enterococcus and Listeria. We show that the S. aureus TslA N-terminal domain is a phospholipase A with anti-staphylococcal activity that is neutralised by the immunity lipoprotein TilA. Two small helical partner proteins, TlaA1 and TlaA2 are essential for T7-dependent secretion of TslA and at least one of these interacts with the TslA C-terminal domain to form a helical stack. Cryo-EM analysis of purified TslA complexes indicate that they share structural similarity with canonical T7 substrates. Our findings suggest that the T7SS has the extraordinary feature of recognising a secretion signal present at either end of a substrate.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.551436">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.551436" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.551436">
        <p class="paperTitle">The host RNA polymerase II C-terminal domain is the anchor for replication of the influenza virus genome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.551436" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.551436" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krischuns, T.; Arragain, B.; Isel, C.; Paisant, S.; Budt, M.; Wolff, T.; Cusack, S.; Naffakh, N.</p>
        <p class="info">Score: 18.5, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.551436' target='https://doi.org/10.1101/2023.08.01.551436'> 10.1101/2023.08.01.551436</a></p>
        <p class="abstract">The current model is that the influenza virus polymerase (FluPol) binds either to host RNA polymerase II (RNAP II) or to the acidic nuclear phosphoprotein 32 (ANP32), which drives its conformation and activity towards transcription or replication of the viral genome, respectively. Here, we provide evidence that the FluPol-RNAP II binding interface has a so far overlooked function for replication of the viral genome. Using a combination of cell-based and in vitro approaches, we show that the RNAP II C-terminal-domain, jointly with ANP32, enhances FluPol replication activity and we propose a model in which the host RNAP II is the anchor for transcription and replication of the viral genome. Our data open new perspectives on the spatial coupling of viral transcription and replication and the coordinated balance between these two activities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.550633">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.550633" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.550633">
        <p class="paperTitle">The Feather Epithelium Contributes to the Dissemination and Ecology of clade 2.3.4.4b H5 High Pathogenicity Avian Influenza Virus in Ducks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.550633" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.550633" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gaide, N.; Filaire, F.; Bertran, K.; Crispo, M.; Dirat, M.; Secula, A.; Foret-Lucas, C.; Payre, B.; Perlas, A.; Cantero, G.; Majo, N.; Soubies, S.; GUERIN, J.-L.</p>
        <p class="info">Score: 18.5, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.550633' target='https://doi.org/10.1101/2023.07.26.550633'> 10.1101/2023.07.26.550633</a></p>
        <p class="abstract">Immature feathers are known replication sites for high pathogenicity avian influenza viruses (HPAIVs) in poultry. However, it is unclear whether feathers play an active role in viral transmission. This study aims to investigate the contribution of the feather epithelium to the dissemination of clade 2.3.4.4b goose/Guangdong/1996 lineage H5 HPAIVs in the environment, based on natural and experimental infections of domestic ducks. During the 2016-22 outbreaks, H5 HPAIVs exhibited persistent and marked feather epitheliotropism in naturally infected commercial ducks. Infection of feathers resulted in epithelial necrosis, disruption, and the production and release of infectious virions. Viral and feather antigens colocalized in dust samples obtained from poultry barns housing naturally infected birds. In summary, the feather epithelium contributes to viral replication, and it is a likely source of environmental infectious material. This underestimated excretion route could greatly impact the ecology of HPAIVs, facilitating airborne and preening-related infections within a flock, and promoting prolonged viral infectivity and long-distance viral transmission between poultry farms.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=112 SRC=&#34;FIGDIR/small/550633v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (54K):
org.highwire.dtl.DTLVardef@16f6b24org.highwire.dtl.DTLVardef@1e33c82org.highwire.dtl.DTLVardef@199b745org.highwire.dtl.DTLVardef@cf2024_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.551500">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.551500" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.551500">
        <p class="paperTitle">Recombinant Rotavirus Expressing the Glycosylated S1 Protein of SARS-CoV-2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.551500" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.551500" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Philip, A. A.; Hu, S.; Patton, J. T.</p>
        <p class="info">Score: 17.0, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.551500' target='https://doi.org/10.1101/2023.08.01.551500'> 10.1101/2023.08.01.551500</a></p>
        <p class="abstract">Reverse genetic systems have been used to introduce heterologous sequences into the rotavirus segmented double-stranded (ds)RNA genome, enabling the generation of recombinant viruses that express foreign proteins and possibly serve as vaccine vectors. Notably, insertion of SARS-CoV-2 sequences into the segment 7 (NSP3) RNA of simian SA11 rotavirus was previously shown to result in the production of recombinant viruses that efficiently expressed the N-terminal domain (NTD) and the receptor-binding domain (RBD) of the S1 region of the SARS-CoV-2 spike protein. However, efforts to generate a similar recombinant (r) SA11 virus that efficiently expressed full-length S1 were less successful. In this study, we describe modifications to the S1-coding cassette inserted in the segment 7 RNA that allowed recovery of second-generation rSA11 viruses that efficiently expressed the [~]120-kDa S1 protein. The [~]120-kDa S1 products were shown to be glycosylated, based on treatment with endoglycosidase H, which reduced the protein to a size of [~]80 kDa. Co-pulldown assays demonstrated that the [~]120-kDa S1 proteins had affinity for the human ACE2 receptor. Although all the second-generation rSA11 viruses expressed glycosylated S1 with affinity for the ACE receptor, only the S1 product of one virus (rSA11/S1f) was appropriately recognized by anti-S1 antibody, suggesting the rSA11/S1f virus expressed an authentic form of S1. Probably due to the presence of FLAG tags on their S1 signal peptides, the S1 products of the other viruses (rSA11/3fS1 and rSA11/3fS1-His) may have undergone defective glycosylation, impeding antibody binding. In summary, these results indicate that recombinant rotaviruses can serve as expression vectors of foreign glycosylated proteins, raising the possibility of generating rotavirus-based vaccines that can induce protective immune responses against enteric and mucosal viruses with glycosylated capsid components, including SARS-CoV-2.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550811">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550811" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550811">
        <p class="paperTitle">Butyrate Protects against SARS-CoV-2-induced Tissue Damage in Golden Hamsters.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550811" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550811" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yu, H.; Yuan, L.; Yan, Z.; Zhou, M.; Ye, J.; Wu, K.; Chen, W.; Chen, R.; Xia, N.-S.; Guan, Y.; Zhu, H.</p>
        <p class="info">Score: 18.7, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550811' target='https://doi.org/10.1101/2023.07.27.550811'> 10.1101/2023.07.27.550811</a></p>
        <p class="abstract">Butyrate, produced by gut microbe during dietary fiber fermentation, plays anti-inflammatory and antioxidant effects in chronic inflammation diseases, yet it remains to be explored whether butyrate has protective effects against viral infections. Here, we demonstrated that butyrate alleviated tissue injury in severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)-infected golden hamsters with supplementation of butyrate before and during the infection. Butyrate-treated hamsters showed augmentation of type I interferon (IFN) response and activation of endothelial cells without exaggerated inflammation. In addition, butyrate regulated redox homeostasis by enhancing the activity of superoxide dismutase (SOD) to inhibit excessive apoptotic cell death. Therefore, butyrate exhibited an effective prevention against SARS-CoV-2 by upregulating antiviral immune responses and promoting cell survival.

IMPORTANCESince SARS-CoV-2 has caused severe disease characterized by acute respiratory distress syndrome (ARDS) in humans, it is essential to develop therapeutics based on relieving such severe clinical symptoms. Current therapy strategies mainly focus on individuals who have COVID-19, however, there is still a strong need for prevention and treatment of SARS-CoV-2 infection. This study showed that butyrate, a bacterial metabolite, improved the response of SARS-CoV-2-infected hamsters by reducing immunopathology caused by impaired antiviral defenses and inhibiting excessive apoptosis through reduction in oxidative stress.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
