<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="An infection and pathogenesis mouse model of SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR)
Authors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.
Score: 4266.1, Published: 2024-01-21 DOI: 10.1101/2024.01.03.574008
SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) is highly attenuated, but can cause mortality in a specifically designed human ACE2-transgenic mouse model, making it an invaluable surrogate model for evaluating the efficacy of drugs and vaccines against SARS-CoV-2.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="An infection and pathogenesis mouse model of SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR)
Authors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.
Score: 4266.1, Published: 2024-01-21 DOI: 10.1101/2024.01.03.574008
SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) is highly attenuated, but can cause mortality in a specifically designed human ACE2-transgenic mouse model, making it an invaluable surrogate model for evaluating the efficacy of drugs and vaccines against SARS-CoV-2." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-28T10:37:59+00:00" />
<meta property="article:modified_time" content="2024-01-28T10:37:59+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="An infection and pathogenesis mouse model of SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR)
Authors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.
Score: 4266.1, Published: 2024-01-21 DOI: 10.1101/2024.01.03.574008
SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) is highly attenuated, but can cause mortality in a specifically designed human ACE2-transgenic mouse model, making it an invaluable surrogate model for evaluating the efficacy of drugs and vaccines against SARS-CoV-2."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "An infection and pathogenesis mouse model of SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR)\nAuthors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.\nScore: 4266.1, Published: 2024-01-21 DOI: 10.1101/2024.01.03.574008\nSARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) is highly attenuated, but can cause mortality in a specifically designed human ACE2-transgenic mouse model, making it an invaluable surrogate model for evaluating the efficacy of drugs and vaccines against SARS-CoV-2.",
  "keywords": [
    
  ],
  "articleBody": " An infection and pathogenesis mouse model of SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR)\nAuthors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.\nScore: 4266.1, Published: 2024-01-21 DOI: 10.1101/2024.01.03.574008\nSARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) is highly attenuated, but can cause mortality in a specifically designed human ACE2-transgenic mouse model, making it an invaluable surrogate model for evaluating the efficacy of drugs and vaccines against SARS-CoV-2.\nViroid-like colonists of human microbiomes\nAuthors: Zheludev, I. N.; Edgar, R. C.; Galiano, M. J. L.; De la Pena, M.; Babaian, A.; Bhatt, A. S.; Fire, A. Z.\nScore: 117.5, Published: 2024-01-21 DOI: 10.1101/2024.01.20.576352\nHere, we describe the \"Obelisks,\" a previously unrecognised class of viroid-like elements that we first identified in human gut metatranscriptomic data. \"Obelisks\" share several properties: (i) apparently circular RNA [~]1kb genome assemblies, (ii) predicted rod-like secondary structures encompassing the entire genome, and (iii) open reading frames coding for a novel protein superfamily, which we call the \"Oblins\". We find that Obelisks form their own distinct phylogenetic group with no detectable sequence or structural similarity to known biological agents. Further, Obelisks are prevalent in tested human microbiome metatranscriptomes with representatives detected in [~]7% of analysed stool metatranscriptomes (29/440) and in [~]50% of analysed oral metatranscriptomes (17/32). Obelisk compositions appear to differ between the anatomic sites and are capable of persisting in individuals, with continued presence over \u003e300 days observed in one case. Large scale searches identified 29,959 Obelisks (clustered at 90% nucleotide identity), with examples from all seven continents and in diverse ecological niches. From this search, a subset of Obelisks are identified to code for Obelisk-specific variants of the hammerhead type-III self-cleaving ribozyme. Lastly, we identified one case of a bacterial species (Streptococcus sanguinis) in which a subset of defined laboratory strains harboured a specific Obelisk RNA population. As such, Obelisks comprise a class of diverse RNAs that have colonised, and gone unnoticed in, human, and global microbiomes.\nTemperature-dependent Spike-ACE2 interaction of Omicron subvariants is associated with viral transmission\nAuthors: Benlarbi, M.; Ding, S.; Belanger, E.; Tauzin, A.; Poujol, R.; Medjahed, H.; El Ferri, O.; Bo, Y.; Bourassa, C.; Hussin, J.; Fafard, J.; Pazgier, M.; Levade, I.; Abrams, C.; Cote, M.; Finzi, A.\nScore: 62.9, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576353\nThe continued evolution of SARS-CoV-2 requires persistent monitoring of its subvariants. Omicron subvariants are responsible for the vast majority of SARS-CoV-2 infections worldwide, with XBB and BA.2.86 sublineages representing more than 90% of circulating strains as of January 2024. In this study, we characterized the functional properties of Spike glycoproteins from BA.2.75, CH.1.1, DV.7.1, BA.4/5, BQ.1.1, XBB, XBB.1, XBB.1.16, XBB.1.5, FD.1.1, EG.5.1, HK.3 BA.2.86 and JN.1. We tested their capacity to evade plasma-mediated recognition and neutralization, ACE2 binding, their susceptibility to cold inactivation, Spike processing, as well as the impact of temperature on Spike-ACE2 interaction. We found that compared to the early wild-type (D614G) strain, most Omicron subvariants Spike glycoproteins evolved to escape recognition and neutralization by plasma from individuals who received a fifth dose of bivalent (BA.1 or BA.4/5) mRNA vaccine and improve ACE2 binding, particularly at low temperatures. Moreover, BA.2.86 had the best affinity for ACE2 at all temperatures tested. We found that Omicron subvariants Spike processing is associated with their susceptibility to cold inactivation. Intriguingly, we found that Spike-ACE2 binding at low temperature was significantly associated with growth rates of Omicron subvariants in humans. Overall, we report that Spikes from newly emerged Omicron subvariants are relatively more stable and resistant to plasma-mediated neutralization, present improved affinity for ACE2 which is associated, particularly at low temperatures, with their growth rates.\nViral entry is a weak barrier to zoonosis\nAuthors: Dufloo, J.; Andreu-Moreno, I.; Valero-Rello, A.; Sanjuan, R.\nScore: 58.2, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576693\nRecent advances in viral metagenomics have led to the discovery of many mammalian viruses, but experimental tests to determine whether they pose a threat to humans are largely lacking. A first step for a virus to cross the species barrier is to penetrate host cells. Here, we use gene synthesis and viral pseudotyping to experimentally test the ability of viral receptor-binding proteins (RBPs) from \u003e100 enveloped RNA viruses to mediate entry into human cells. Analysis of thousands of RBP-cell pairs demonstrated such ability for most viruses, with significant variation among the 14 viral families studied. Comparison of RBP-mediated infectivity with cellular gene expression data showed that viral entry is often not limited by the presence of a receptor and revealed the contribution of additional host factors. Our results prove the weakness of interspecies barriers at the early stages of infection and identify molecular interactions that enable viral zoonosis.\nBirth of new protein folds and functions in the virome\nAuthors: Nomburg, J.; Price, N.; Doudna, J. A.\nScore: 39.0, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576744\nRapid virus evolution generates proteins essential to infectivity and replication but with unknown function due to extreme sequence divergence1. Using a database of 67,715 newly predicted protein structures from 4,463 eukaryotic viral species, we found that 62% of viral proteins are evolutionarily young and lack homologs in the Alphafold database2,3. Among the 38% of more ancient viral proteins, many have non-viral structural homologs that revealed surprising similarities between human pathogens and their eukaryotic hosts. Structural comparisons suggested putative functions for \u003e25% of unannotated viral proteins, including those with roles in the evasion of innate immunity. In particular, RNA ligase T- (ligT) like phosphodiesterases were found to resemble phage-encoded proteins that hydrolyze the host immune-activating cyclic dinucleotides 33 and 23 cyclic G-A monophosphate (cGAMP). Experimental analysis showed that ligT homologs encoded by avian poxviruses likewise hydrolyze 23 cGAMP, showing that ligT-mediated targeting of cGAMP is an evolutionarily conserved mechanism of immune evasion present in both bacteriophage and eukaryotic viruses. Together, the viral protein structural database and analytics presented here afford new opportunities to identify mechanisms of virus-host interactions that are common across the virome.\nHalofilins as Emerging Bactofilin Families of Archaeal Cell Shape Plasticity Orchestrators\nAuthors: Curtis, Z.; Escudeiro, P.; Mallon, J. L.; Leland, O.; Rados, T.; Dodge, A.; Andre, K.; Kwak, J.; Yun, K.; Pohlschroder, M.; Alva, V.; Bisson, A.\nScore: 23.3, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576759\nBactofilins are rigid, non-polar bacterial cytoskeletal filaments that link cellular processes to specific curvatures of the cytoplasmic membrane. Although homologs of bactofilins have been identified in archaea and eukaryotes, functional studies have remained confined to bacterial systems. Here, we characterized representatives of two families of archaeal bactofilins from the pleomorphic archaeon Haloferax volcanii, halofilin A (HalA) and halofilin B (HalB). Unlike bacterial bactofilins, HalA polymerizes into polar filaments in vivo at positive membrane curvature, whereas HalB forms more static foci and accumulates in areas of local negative curvatures on the outer cell surface. Combining gene deletions, super-resolution, and single-cell microscopy showed that halofilins are critical in maintaining H. volcanii cell integrity during shape transition from disk (sessile) to rod (motile). Morphological defects in {Delta}halA primarily affected rod-shaped cells from accumulating highly positive curvatures. Conversely, disk-shaped cells were exclusively affected by halB deletion, showing a decrease in positive and negative curvatures, resulting in flatter cells. Furthermore, while {Delta}halA and {Delta}halB cells displayed lower cell division placement precision, morphological defects arose predominantly during the disk-to-rod shape remodeling. We propose halofilins provide mechanical scaffolding, dynamically coupling the cytoplasmic membrane and the S-layer. We speculate that HalA filaments support rods under low S-layer lipidation (flexible, fast membrane diffusion). In contrast, HalB connects the S-layer to negative curvatures in disks under high lipidation levels (rigid, slow membrane diffusion). Significance StatementHow have cells evolved molecular machineries to support the mechanical and morphological requirements of microbial behavior? The biochemistry and biophysics of cell envelopes and their associated cytoskeleton are key to understanding such adaptations. Archaea are strategically positioned in the Tree of Life, bridging the molecular and cellular biology of bacteria and eukaryotes. Our work investigated the archaeal shapeshift between two morphological cell types and the role of two new cytoskeletal proteins in mechanical scaffolding during development. Specifically, halofilins are the first identified archaeal cell shape factors to be demonstrated to function at the onset of cell-type shifting. Broadly, our model offers potential for new synthetic cell designs and bioinspired materials that are both minimalist and modular.\nExtracellular ATP is a signalling molecule in bacteria\nAuthors: Tronnet, S.; Pandey, V.; Lloret-Berrocal, M.; Perez-del-Pozo, M.; Soderholm, N.; Hernandez-Ortego, C.; Billker, O.; Nordstrom, A.; Puhar, A.\nScore: 20.4, Published: 2024-01-25 DOI: 10.1101/2024.01.24.577065\nIn animals and plants extracellular ATP (eATP) functions as signalling molecule and regulates the immune response. During inflammation intestinal bacteria are exposed to elevated eATP originating from the mucosa. Whether bacteria respond to eATP is unclear. Here we show that non-pathogenic Escherichia coli responds to eATP at physiologically relevant concentrations by modifying its transcriptional and metabolic landscape. The use of a promoter library showed that the response to eATP is time-, dose- and medium-dependent. Genes related to lipid, amino acid, or vitamin metabolism were regulated. Metabolomics showed that eATP triggers enrichment of molecules with bioactive properties on the host or bacteria. Combined genome-scale modelling highlighted the global metabolic modifications. Moreover, eATP altered the sensitivity to antibiotics and antimicrobial peptides. Finally, in pathogens eATP controls the expression of virulence and fitness factors. Our results indicate that eATP is a signalling molecule in prokaryotes which globally regulates physiology, antimicrobial resistance, and virulence.\nComparative genomics reveals intra and inter species variation in the pathogenic fungus Batrachochytrium dendrobatidis\nAuthors: Yacoub, M. N.; Stajich, J. E.\nScore: 12.8, Published: 2024-01-25 DOI: 10.1101/2024.01.24.576925\nThe Global Pandemic Lineage (GPL) of the amphibian pathogen Batrachochytrium dendrobatidis (Bd) has been described as a main driver of amphibian extinctions on nearly every continent. Near complete genome of three Bd-GPL strains have enabled studies of the pathogen but the genomic features that set Bd-GPL apart from other Bd lineages is not well understood due to a lack of high-quality genome assemblies and annotations from other lineages. We used long-read DNA sequencing to assemble high-quality genomes of three Bd-BRAZIL isolates and one non-pathogen outgroup species Polyrhizophydium stewartii (Ps) strain JEL0888, and compared these to genomes of previously sequenced Bd-GPL strains. The Bd-BRAZIL assemblies range in size between 22.0 and 26.1 Mb and encode 8495-8620 protein-coding genes for each strain. Our pan-genome analysis provided insight into shared and lineage-specific gene content. The core genome of Bd consists of 6278 conserved gene families, with 202 Bd-BRAZIL and 172 Bd-GPL specific gene families. We discovered gene copy number variation in pathogenicity gene families between Bd-BRAZIL and Bd-GPL strains though none were consistently expanded in Bd-GPL or Bd-BRAZIL strains. Comparison within the Batrachochytrium genus and two closely related non-pathogenic saprophytic chytrids identified variation in sequence and protein domain counts. We further test these new Bd-BRAZIL genomes to assess their utility as reference genomes for transcriptome alignment and analysis. Our analysis examines the genomic variation between strains in Bd-BRAZIL and Bd-GPL and offers insights into the application of these genomes as reference genomes for future studies.\nGut environmental factors explain variations in the gut microbiome composition and metabolism within and between healthy adults\nAuthors: Prochazkova, N.; Laursen, M. F.; La Barbera, G.; Tsekitsidi, E.; Joergensen, M. S.; Rasmussen, M. A.; Raes, J.; Dragsted, L. O.; Licht, T. R.; Roager, H. M.\nScore: 12.4, Published: 2024-01-23 DOI: 10.1101/2024.01.23.574598\nThe human gut microbiome is highly personal. However, the contribution of the gut environment to variations in the gut microbiome remains elusive. Here, we profiled the gut microbiome composition and metabolism over 9 consecutive days in 61 healthy adults and assessed gut environmental factors including segmental transit time and pH using a wireless motility capsule. Day-to-day fluctuations in gut environmental factors as well as segmental transit time and pH varied substantially between individuals. The gut environment explained more variations in gut microbiome and urine metabolome than dietary macronutrients or personal characteristics. Finally, we identified coffee-derived metabolites to be negatively correlated with small intestinal transit time and several microbial metabolites to be associated with colonic transit time including urinary proteolytic markers, faecal short-chain fatty acids, and breath methane. Our work suggests that the gut environment is key for understanding the individuality of the human gut microbiome composition and metabolism.\nResolution of ribosomal stalling by ABCF ATPases YfmR and YkpA/YbiT\nAuthors: Takada, H.; Fujiwara, K.; Atkinson, G. C.; Chiba, S.; Hauryliuk, V.\nScore: 12.1, Published: 2024-01-26 DOI: 10.1101/2024.01.25.577322\nEfficiency of protein synthesis on the ribosome is strongly affected by the amino acid composition of the assembled amino acid chain. Challenging sequences include proline-rich motifs as well as highly positively and negatively charged amino acid stretches. Members of the F subfamily of ABC ATPases (ABCFs) have been long hypothesised to promote translation of such problematic motifs. In this study we have applied genetics and reporter-based assays to characterise the four housekeeping ABCF ATPases of Bacillus subtilis: YdiF, YfmM, YfmR/Uup and YkpA/YbiT. We show that YfmR cooperates with the translation factor EF-P that promotes translation of Pro-rich motifs. Simultaneous loss of both YfmR and EF-P results in a dramatic growth defect. Surprisingly, this growth defect can be largely suppressed though overexpression of an EF-P variant lacking the otherwise crucial 5-amino-pentanolylated residue K32. Using in vivo reporter assays, we show that overexpression of YfmR can alleviate ribosomal stalling on Asp-Pro motifs. Finally, we demonstrate that YkpA/YbiT promotes translation of positively and negatively charged motifs but is inactive in resolving ribosomal stalls on proline-rich stretches. Collectively, our results provide insights into the function of ABCF translation factors in modulating protein synthesis in B. subtilis.\n",
  "wordCount" : "2230",
  "inLanguage": "en",
  "datePublished": "2024-01-28T10:37:59Z",
  "dateModified": "2024-01-28T10:37:59Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 28, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574008">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574008" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574008">
        <p class="paperTitle">An infection and pathogenesis mouse model of SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574008" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574008" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.</p>
        <p class="info">Score: 4266.1, Published: 2024-01-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574008' target='https://doi.org/10.1101/2024.01.03.574008'> 10.1101/2024.01.03.574008</a></p>
        <p class="abstract">SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) is highly attenuated, but can cause mortality in a specifically designed human ACE2-transgenic mouse model, making it an invaluable surrogate model for evaluating the efficacy of drugs and vaccines against SARS-CoV-2.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576352">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576352" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576352">
        <p class="paperTitle">Viroid-like colonists of human microbiomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576352" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576352" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zheludev, I. N.; Edgar, R. C.; Galiano, M. J. L.; De la Pena, M.; Babaian, A.; Bhatt, A. S.; Fire, A. Z.</p>
        <p class="info">Score: 117.5, Published: 2024-01-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576352' target='https://doi.org/10.1101/2024.01.20.576352'> 10.1101/2024.01.20.576352</a></p>
        <p class="abstract">Here, we describe the &#34;Obelisks,&#34; a previously unrecognised class of viroid-like elements that we first identified in human gut metatranscriptomic data. &#34;Obelisks&#34; share several properties: (i) apparently circular RNA [~]1kb genome assemblies, (ii) predicted rod-like secondary structures encompassing the entire genome, and (iii) open reading frames coding for a novel protein superfamily, which we call the &#34;Oblins&#34;. We find that Obelisks form their own distinct phylogenetic group with no detectable sequence or structural similarity to known biological agents. Further, Obelisks are prevalent in tested human microbiome metatranscriptomes with representatives detected in [~]7% of analysed stool metatranscriptomes (29/440) and in [~]50% of analysed oral metatranscriptomes (17/32). Obelisk compositions appear to differ between the anatomic sites and are capable of persisting in individuals, with continued presence over &gt;300 days observed in one case. Large scale searches identified 29,959 Obelisks (clustered at 90% nucleotide identity), with examples from all seven continents and in diverse ecological niches. From this search, a subset of Obelisks are identified to code for Obelisk-specific variants of the hammerhead type-III self-cleaving ribozyme. Lastly, we identified one case of a bacterial species (Streptococcus sanguinis) in which a subset of defined laboratory strains harboured a specific Obelisk RNA population. As such, Obelisks comprise a class of diverse RNAs that have colonised, and gone unnoticed in, human, and global microbiomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576353">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576353" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576353">
        <p class="paperTitle">Temperature-dependent Spike-ACE2 interaction of Omicron subvariants is associated with viral transmission</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576353" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576353" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Benlarbi, M.; Ding, S.; Belanger, E.; Tauzin, A.; Poujol, R.; Medjahed, H.; El Ferri, O.; Bo, Y.; Bourassa, C.; Hussin, J.; Fafard, J.; Pazgier, M.; Levade, I.; Abrams, C.; Cote, M.; Finzi, A.</p>
        <p class="info">Score: 62.9, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576353' target='https://doi.org/10.1101/2024.01.20.576353'> 10.1101/2024.01.20.576353</a></p>
        <p class="abstract">The continued evolution of SARS-CoV-2 requires persistent monitoring of its subvariants. Omicron subvariants are responsible for the vast majority of SARS-CoV-2 infections worldwide, with XBB and BA.2.86 sublineages representing more than 90% of circulating strains as of January 2024. In this study, we characterized the functional properties of Spike glycoproteins from BA.2.75, CH.1.1, DV.7.1, BA.4/5, BQ.1.1, XBB, XBB.1, XBB.1.16, XBB.1.5, FD.1.1, EG.5.1, HK.3 BA.2.86 and JN.1. We tested their capacity to evade plasma-mediated recognition and neutralization, ACE2 binding, their susceptibility to cold inactivation, Spike processing, as well as the impact of temperature on Spike-ACE2 interaction. We found that compared to the early wild-type (D614G) strain, most Omicron subvariants Spike glycoproteins evolved to escape recognition and neutralization by plasma from individuals who received a fifth dose of bivalent (BA.1 or BA.4/5) mRNA vaccine and improve ACE2 binding, particularly at low temperatures. Moreover, BA.2.86 had the best affinity for ACE2 at all temperatures tested. We found that Omicron subvariants Spike processing is associated with their susceptibility to cold inactivation. Intriguingly, we found that Spike-ACE2 binding at low temperature was significantly associated with growth rates of Omicron subvariants in humans. Overall, we report that Spikes from newly emerged Omicron subvariants are relatively more stable and resistant to plasma-mediated neutralization, present improved affinity for ACE2 which is associated, particularly at low temperatures, with their growth rates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.576693">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.576693" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.576693">
        <p class="paperTitle">Viral entry is a weak barrier to zoonosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.576693" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.576693" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dufloo, J.; Andreu-Moreno, I.; Valero-Rello, A.; Sanjuan, R.</p>
        <p class="info">Score: 58.2, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.576693' target='https://doi.org/10.1101/2024.01.22.576693'> 10.1101/2024.01.22.576693</a></p>
        <p class="abstract">Recent advances in viral metagenomics have led to the discovery of many mammalian viruses, but experimental tests to determine whether they pose a threat to humans are largely lacking. A first step for a virus to cross the species barrier is to penetrate host cells. Here, we use gene synthesis and viral pseudotyping to experimentally test the ability of viral receptor-binding proteins (RBPs) from &gt;100 enveloped RNA viruses to mediate entry into human cells. Analysis of thousands of RBP-cell pairs demonstrated such ability for most viruses, with significant variation among the 14 viral families studied. Comparison of RBP-mediated infectivity with cellular gene expression data showed that viral entry is often not limited by the presence of a receptor and revealed the contribution of additional host factors. Our results prove the weakness of interspecies barriers at the early stages of infection and identify molecular interactions that enable viral zoonosis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.576744">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.576744" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.576744">
        <p class="paperTitle">Birth of new protein folds and functions in the virome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.576744" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.576744" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nomburg, J.; Price, N.; Doudna, J. A.</p>
        <p class="info">Score: 39.0, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.576744' target='https://doi.org/10.1101/2024.01.22.576744'> 10.1101/2024.01.22.576744</a></p>
        <p class="abstract">Rapid virus evolution generates proteins essential to infectivity and replication but with unknown function due to extreme sequence divergence1. Using a database of 67,715 newly predicted protein structures from 4,463 eukaryotic viral species, we found that 62% of viral proteins are evolutionarily young and lack homologs in the Alphafold database2,3. Among the 38% of more ancient viral proteins, many have non-viral structural homologs that revealed surprising similarities between human pathogens and their eukaryotic hosts. Structural comparisons suggested putative functions for &gt;25% of unannotated viral proteins, including those with roles in the evasion of innate immunity. In particular, RNA ligase T- (ligT) like phosphodiesterases were found to resemble phage-encoded proteins that hydrolyze the host immune-activating cyclic dinucleotides 33 and 23 cyclic G-A monophosphate (cGAMP). Experimental analysis showed that ligT homologs encoded by avian poxviruses likewise hydrolyze 23 cGAMP, showing that ligT-mediated targeting of cGAMP is an evolutionarily conserved mechanism of immune evasion present in both bacteriophage and eukaryotic viruses. Together, the viral protein structural database and analytics presented here afford new opportunities to identify mechanisms of virus-host interactions that are common across the virome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.576759">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.576759" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.576759">
        <p class="paperTitle">Halofilins as Emerging Bactofilin Families of Archaeal Cell Shape Plasticity Orchestrators</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.576759" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.576759" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Curtis, Z.; Escudeiro, P.; Mallon, J. L.; Leland, O.; Rados, T.; Dodge, A.; Andre, K.; Kwak, J.; Yun, K.; Pohlschroder, M.; Alva, V.; Bisson, A.</p>
        <p class="info">Score: 23.3, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.576759' target='https://doi.org/10.1101/2024.01.22.576759'> 10.1101/2024.01.22.576759</a></p>
        <p class="abstract">Bactofilins are rigid, non-polar bacterial cytoskeletal filaments that link cellular processes to specific curvatures of the cytoplasmic membrane. Although homologs of bactofilins have been identified in archaea and eukaryotes, functional studies have remained confined to bacterial systems. Here, we characterized representatives of two families of archaeal bactofilins from the pleomorphic archaeon Haloferax volcanii, halofilin A (HalA) and halofilin B (HalB). Unlike bacterial bactofilins, HalA polymerizes into polar filaments in vivo at positive membrane curvature, whereas HalB forms more static foci and accumulates in areas of local negative curvatures on the outer cell surface. Combining gene deletions, super-resolution, and single-cell microscopy showed that halofilins are critical in maintaining H. volcanii cell integrity during shape transition from disk (sessile) to rod (motile). Morphological defects in {Delta}halA primarily affected rod-shaped cells from accumulating highly positive curvatures. Conversely, disk-shaped cells were exclusively affected by halB deletion, showing a decrease in positive and negative curvatures, resulting in flatter cells. Furthermore, while {Delta}halA and {Delta}halB cells displayed lower cell division placement precision, morphological defects arose predominantly during the disk-to-rod shape remodeling. We propose halofilins provide mechanical scaffolding, dynamically coupling the cytoplasmic membrane and the S-layer. We speculate that HalA filaments support rods under low S-layer lipidation (flexible, fast membrane diffusion). In contrast, HalB connects the S-layer to negative curvatures in disks under high lipidation levels (rigid, slow membrane diffusion).

Significance StatementHow have cells evolved molecular machineries to support the mechanical and morphological requirements of microbial behavior? The biochemistry and biophysics of cell envelopes and their associated cytoskeleton are key to understanding such adaptations. Archaea are strategically positioned in the Tree of Life, bridging the molecular and cellular biology of bacteria and eukaryotes. Our work investigated the archaeal shapeshift between two morphological cell types and the role of two new cytoskeletal proteins in mechanical scaffolding during development. Specifically, halofilins are the first identified archaeal cell shape factors to be demonstrated to function at the onset of cell-type shifting. Broadly, our model offers potential for new synthetic cell designs and bioinspired materials that are both minimalist and modular.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.577065">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.577065" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.577065">
        <p class="paperTitle">Extracellular ATP is a signalling molecule in bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.577065" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.577065" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tronnet, S.; Pandey, V.; Lloret-Berrocal, M.; Perez-del-Pozo, M.; Soderholm, N.; Hernandez-Ortego, C.; Billker, O.; Nordstrom, A.; Puhar, A.</p>
        <p class="info">Score: 20.4, Published: 2024-01-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.577065' target='https://doi.org/10.1101/2024.01.24.577065'> 10.1101/2024.01.24.577065</a></p>
        <p class="abstract">In animals and plants extracellular ATP (eATP) functions as signalling molecule and regulates the immune response. During inflammation intestinal bacteria are exposed to elevated eATP originating from the mucosa. Whether bacteria respond to eATP is unclear. Here we show that non-pathogenic Escherichia coli responds to eATP at physiologically relevant concentrations by modifying its transcriptional and metabolic landscape. The use of a promoter library showed that the response to eATP is time-, dose- and medium-dependent. Genes related to lipid, amino acid, or vitamin metabolism were regulated. Metabolomics showed that eATP triggers enrichment of molecules with bioactive properties on the host or bacteria. Combined genome-scale modelling highlighted the global metabolic modifications. Moreover, eATP altered the sensitivity to antibiotics and antimicrobial peptides. Finally, in pathogens eATP controls the expression of virulence and fitness factors. Our results indicate that eATP is a signalling molecule in prokaryotes which globally regulates physiology, antimicrobial resistance, and virulence.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.576925">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.576925" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.576925">
        <p class="paperTitle">Comparative genomics reveals intra and inter species variation in the pathogenic fungus Batrachochytrium dendrobatidis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.576925" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.576925" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yacoub, M. N.; Stajich, J. E.</p>
        <p class="info">Score: 12.8, Published: 2024-01-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.576925' target='https://doi.org/10.1101/2024.01.24.576925'> 10.1101/2024.01.24.576925</a></p>
        <p class="abstract">The Global Pandemic Lineage (GPL) of the amphibian pathogen Batrachochytrium dendrobatidis (Bd) has been described as a main driver of amphibian extinctions on nearly every continent. Near complete genome of three Bd-GPL strains have enabled studies of the pathogen but the genomic features that set Bd-GPL apart from other Bd lineages is not well understood due to a lack of high-quality genome assemblies and annotations from other lineages. We used long-read DNA sequencing to assemble high-quality genomes of three Bd-BRAZIL isolates and one non-pathogen outgroup species Polyrhizophydium stewartii (Ps) strain JEL0888, and compared these to genomes of previously sequenced Bd-GPL strains. The Bd-BRAZIL assemblies range in size between 22.0 and 26.1 Mb and encode 8495-8620 protein-coding genes for each strain. Our pan-genome analysis provided insight into shared and lineage-specific gene content. The core genome of Bd consists of 6278 conserved gene families, with 202 Bd-BRAZIL and 172 Bd-GPL specific gene families. We discovered gene copy number variation in pathogenicity gene families between Bd-BRAZIL and Bd-GPL strains though none were consistently expanded in Bd-GPL or Bd-BRAZIL strains. Comparison within the Batrachochytrium genus and two closely related non-pathogenic saprophytic chytrids identified variation in sequence and protein domain counts. We further test these new Bd-BRAZIL genomes to assess their utility as reference genomes for transcriptome alignment and analysis. Our analysis examines the genomic variation between strains in Bd-BRAZIL and Bd-GPL and offers insights into the application of these genomes as reference genomes for future studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.23.574598">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.23.574598" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.23.574598">
        <p class="paperTitle">Gut environmental factors explain variations in the gut microbiome composition and metabolism within and between healthy adults</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.23.574598" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.23.574598" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Prochazkova, N.; Laursen, M. F.; La Barbera, G.; Tsekitsidi, E.; Joergensen, M. S.; Rasmussen, M. A.; Raes, J.; Dragsted, L. O.; Licht, T. R.; Roager, H. M.</p>
        <p class="info">Score: 12.4, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.23.574598' target='https://doi.org/10.1101/2024.01.23.574598'> 10.1101/2024.01.23.574598</a></p>
        <p class="abstract">The human gut microbiome is highly personal. However, the contribution of the gut environment to variations in the gut microbiome remains elusive. Here, we profiled the gut microbiome composition and metabolism over 9 consecutive days in 61 healthy adults and assessed gut environmental factors including segmental transit time and pH using a wireless motility capsule. Day-to-day fluctuations in gut environmental factors as well as segmental transit time and pH varied substantially between individuals. The gut environment explained more variations in gut microbiome and urine metabolome than dietary macronutrients or personal characteristics. Finally, we identified coffee-derived metabolites to be negatively correlated with small intestinal transit time and several microbial metabolites to be associated with colonic transit time including urinary proteolytic markers, faecal short-chain fatty acids, and breath methane. Our work suggests that the gut environment is key for understanding the individuality of the human gut microbiome composition and metabolism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.25.577322">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.25.577322" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.25.577322">
        <p class="paperTitle">Resolution of ribosomal stalling by ABCF ATPases YfmR and YkpA/YbiT</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.25.577322" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.25.577322" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Takada, H.; Fujiwara, K.; Atkinson, G. C.; Chiba, S.; Hauryliuk, V.</p>
        <p class="info">Score: 12.1, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.25.577322' target='https://doi.org/10.1101/2024.01.25.577322'> 10.1101/2024.01.25.577322</a></p>
        <p class="abstract">Efficiency of protein synthesis on the ribosome is strongly affected by the amino acid composition of the assembled amino acid chain. Challenging sequences include proline-rich motifs as well as highly positively and negatively charged amino acid stretches. Members of the F subfamily of ABC ATPases (ABCFs) have been long hypothesised to promote translation of such problematic motifs. In this study we have applied genetics and reporter-based assays to characterise the four housekeeping ABCF ATPases of Bacillus subtilis: YdiF, YfmM, YfmR/Uup and YkpA/YbiT. We show that YfmR cooperates with the translation factor EF-P that promotes translation of Pro-rich motifs. Simultaneous loss of both YfmR and EF-P results in a dramatic growth defect. Surprisingly, this growth defect can be largely suppressed though overexpression of an EF-P variant lacking the otherwise crucial 5-amino-pentanolylated residue K32. Using in vivo reporter assays, we show that overexpression of YfmR can alleviate ribosomal stalling on Asp-Pro motifs. Finally, we demonstrate that YkpA/YbiT promotes translation of positively and negatively charged motifs but is inactive in resolving ribosomal stalls on proline-rich stretches. Collectively, our results provide insights into the function of ABCF translation factors in modulating protein synthesis in B. subtilis.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
