<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Characterization of an EG.5.1 clinical isolate in vitro and in vivo
Authors: Uraki, R.; Kiso, M.; Iwatsuki-Horimoto, K.; Yamayoshi, S.; Ito, M.; Chiba, S.; Sakai-Tagawa, Y.; Imai, M.; Kashima, Y.; Koga, M.; Fuwa, N.; Okumura, N.; Hojo, M.; Iwamoto, N.; Kato, H.; Nakajima, H.; Ohmagari, N.; Yotsuyanagi, H.; Suzuki, Y.; Kawaoka, Y.
Score: 52.3, Published: 2023-09-01 DOI: 10.1101/2023.08.31.555819
EG.5.1 is a subvariant of the SARS-CoV-2 Omicron XBB variant that is rapidly increasing in prevalence worldwide.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Characterization of an EG.5.1 clinical isolate in vitro and in vivo
Authors: Uraki, R.; Kiso, M.; Iwatsuki-Horimoto, K.; Yamayoshi, S.; Ito, M.; Chiba, S.; Sakai-Tagawa, Y.; Imai, M.; Kashima, Y.; Koga, M.; Fuwa, N.; Okumura, N.; Hojo, M.; Iwamoto, N.; Kato, H.; Nakajima, H.; Ohmagari, N.; Yotsuyanagi, H.; Suzuki, Y.; Kawaoka, Y.
Score: 52.3, Published: 2023-09-01 DOI: 10.1101/2023.08.31.555819
EG.5.1 is a subvariant of the SARS-CoV-2 Omicron XBB variant that is rapidly increasing in prevalence worldwide." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-03T10:37:18+00:00" />
<meta property="article:modified_time" content="2023-09-03T10:37:18+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Characterization of an EG.5.1 clinical isolate in vitro and in vivo
Authors: Uraki, R.; Kiso, M.; Iwatsuki-Horimoto, K.; Yamayoshi, S.; Ito, M.; Chiba, S.; Sakai-Tagawa, Y.; Imai, M.; Kashima, Y.; Koga, M.; Fuwa, N.; Okumura, N.; Hojo, M.; Iwamoto, N.; Kato, H.; Nakajima, H.; Ohmagari, N.; Yotsuyanagi, H.; Suzuki, Y.; Kawaoka, Y.
Score: 52.3, Published: 2023-09-01 DOI: 10.1101/2023.08.31.555819
EG.5.1 is a subvariant of the SARS-CoV-2 Omicron XBB variant that is rapidly increasing in prevalence worldwide."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Characterization of an EG.5.1 clinical isolate in vitro and in vivo\nAuthors: Uraki, R.; Kiso, M.; Iwatsuki-Horimoto, K.; Yamayoshi, S.; Ito, M.; Chiba, S.; Sakai-Tagawa, Y.; Imai, M.; Kashima, Y.; Koga, M.; Fuwa, N.; Okumura, N.; Hojo, M.; Iwamoto, N.; Kato, H.; Nakajima, H.; Ohmagari, N.; Yotsuyanagi, H.; Suzuki, Y.; Kawaoka, Y.\nScore: 52.3, Published: 2023-09-01 DOI: 10.1101/2023.08.31.555819\nEG.5.1 is a subvariant of the SARS-CoV-2 Omicron XBB variant that is rapidly increasing in prevalence worldwide.",
  "keywords": [
    
  ],
  "articleBody": " Characterization of an EG.5.1 clinical isolate in vitro and in vivo\nAuthors: Uraki, R.; Kiso, M.; Iwatsuki-Horimoto, K.; Yamayoshi, S.; Ito, M.; Chiba, S.; Sakai-Tagawa, Y.; Imai, M.; Kashima, Y.; Koga, M.; Fuwa, N.; Okumura, N.; Hojo, M.; Iwamoto, N.; Kato, H.; Nakajima, H.; Ohmagari, N.; Yotsuyanagi, H.; Suzuki, Y.; Kawaoka, Y.\nScore: 52.3, Published: 2023-09-01 DOI: 10.1101/2023.08.31.555819\nEG.5.1 is a subvariant of the SARS-CoV-2 Omicron XBB variant that is rapidly increasing in prevalence worldwide. EG.5.1 has additional substitutions in its spike protein (namely, Q52H and F456L) compared with XBB.1.5. However, the pathogenicity, transmissibility, and immune evasion properties of clinical isolates of EG.5.1 are largely unknown. In this study, we used wild-type Syrian hamsters to investigate the replicative ability, pathogenicity, and transmissibility of a clinical EG.5.1 isolate. Our data show that there are no obvious differences in growth ability and pathogenicity between EG.5.1 and XBB.1.5, and both EG.5.1 and XBB.1.5 are attenuated compared to a Delta variant isolate. We also found that EG.5.1 is transmitted more efficiently between hamsters compared with XBB.1.5. In addition, unlike XBB.1.5, we detected EG.5.1 virus in the lungs of four of six exposed hamsters, suggesting that the virus tropism of EG.5.1 is different from that of XBB.1.5 after airborne transmission. Finally, we assessed the neutralizing ability of plasma from convalescent individuals and found that the neutralizing activity against EG.5.1 was slightly, but significantly, lower than that against XBB.1.5 or XBB.1.9.2. This suggests that EG.5.1 effectively evades humoral immunity and that the amino acid differences in the S protein of EG.5.1 compared with that of XBB.1.5 or XBB.1.9.2 (i.e., Q52H, R158G, and F456L) alter the antigenicity of EG.5.1. Our data suggest that the increased transmissibility and altered antigenicity of EG.5.1 may be driving its increasing prevalence over XBB.1.5 in the human population.\nCommunity ecology of phages on a clonal bacterial host\nAuthors: Pyenson, N. C.; Leeks, A.; Nweke, O.; Goldford, J.; Turner, P.; Foster, K. R.; Sanchez, A.\nScore: 36.4, Published: 2023-09-01 DOI: 10.1101/2023.09.01.555216\nBacteriophages are the most abundant and diverse biological entities on Earth, yet the ecological mechanisms that sustain this extraordinary diversity remain unclear. Here, we have discovered a general mechanism that allows phage diversity to outstrip the diversity of their bacterial hosts. We assembled and passaged dozens of diverse phage communities on a single, non-coevolving strain of Escherichia coli until the phage communities reached equilibrium. In all cases, we found multiple phage species coexisted stably, despite competition for a single, clonal host population. Coexistence within these communities was supported through host phenotypic heterogeneity, whereby phages specialized on cells adopting different growth phenotypes. Our experiments reveal a rich community ecology of bacteriophages where multiple phage species coexist and interact at the scale of a single bacterial host.\nAntiviral efficacy of the SARS-CoV-2 XBB breakthrough infection sera against Omicron subvariants including EG.5\nAuthors: Kaku, Y.; Kosugi, Y.; Uriu, K.; Ito, J.; Kuramochi, J.; Sadamasu, K.; Yoshimura, K.; Asakura, H.; Nagashima, M.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Sato, K.\nScore: 1038.1, Published: 2023-08-08 DOI: 10.1101/2023.08.08.552415\nAs of July 2023, EG.5.1 (a.k.a. XBB.1.9.2.5.1), a XBB subvariant bearing the S:Q52H and S:F456L substitutions, alongside the S:F486P substitution (Figure S1A), has rapidly spread in some countries. On July 19, 2023, the WHO classified EG.5 as a variant under monitoring. First, we showed that EG.5.1 exhibits a higher effective reproduction number compared with XBB.1.5, XBB.1.16, and its parental lineage (XBB.1.9.2), suggesting that EG.5.1 will spread globally and outcompete these XBB subvariants in the near future. We then addressed whether EG.5.1 evades from the antiviral effect of the humoral immunity induced by breakthrough infection (BTI) of XBB subvariants and performed a neutralization assay using XBB BTI sera. However, the 50% neutralization titer (NT50) of XBB BTI sera against EG.5.1 was comparable to those against XBB.1.5/1.9.2 and XBB.1.16. Moreover, the sensitivity of EG.5.1 to convalescent sera of XBB.1- and XBB.1.5-infected hamsters was similar to those of XBB.1.5/1.9 and XBB.1.16. These results suggest that the increased Re of EG.5.1 is attributed to neither increased infectivity nor immune evasion from XBB BTI, and the emergence and spread of EG.5 is driven by the other pressures. We previously demonstrated that Omicron BTI cannot efficiently induce antiviral humoral immunity against the variant infected. In fact, the NT50s of the BTI sera of Omicron BA.1, BA.2, and BA.5 against the variant infected were 3.0-, 2.2-, and 3.4-fold lower than that against the ancestral B.1.1 variant, respectively. However, strikingly, we found that the NT50 of the BTI sera of XBB1.5/1.9 and XBB.1.16 against the variant infected were 8.7- and 8.3-fold lower than that against the B.1.1 variant. These results suggest that XBB BTI cannot efficiently induce antiviral humoral immunity against XBB subvariants. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=147 SRC=\"FIGDIR/small/552415v1_figs1.gif\" ALT=\"Figure 1\"\u003e View larger version (39K): org.highwire.dtl.DTLVardef@f95376org.highwire.dtl.DTLVardef@d66fa8org.highwire.dtl.DTLVardef@3c8841org.highwire.dtl.DTLVardef@15824c_HPS_FORMAT_FIGEXP M_FIG O_FLOATNOFigure S1.C_FLOATNO Virological features of EG.5.1 and XBB BTI (A) Frequency of mutations of interest in the representative XBB sublineages. Only mutations with a frequency \u003e0.5 in at least one but not all the representative sublineages are shown. Note that the S proteins of XBB.1.5 and XBB.1.9.2 are identical. (B) Estimated epidemic dynamics of the representative XBB sublineages in countries where \u003e50 sequences of EG.5.1, XBB.1.5, XBB.1.9.2, and XBB.1.16 were detected from April 1, 2023 to July 13, 2023. Countries are ordered according to the number of detected sequences of EG.5.1. Line, posterior mean; ribbon, 95% Bayesian confidence interval. The dynamics for EG.5.1 is highlighted by a red arrowhead. (C) Estimated relative Re of the representative XBB sublineages in the six countries. The relative Re of XBB.1.5 is set to 1 (horizontal dashed line). Violin, posterior distribution; dot, posterior mean; line, 95% Bayesian confidence interval. (D) Lentivirus-based pseudovirus assay. HOS-ACE2-TMPRSS2 cells were infected with pseudoviruses bearing each S protein. The amount of input virus was normalized to the amount of HIV-1 p24 capsid protein. The percentage infectivity of XBB.1.5/1.9.2, XBB.1.5/1.9.2+Q52H, XBB.1.5/1.9.2+F456L, and EG.5.1 compared to that of XBB.1.5/1.9.2 are shown. The horizontal dash line indicates the mean value of the percentage infectivity of the XBB.1.5/1.9.2. Assays were performed in quadruplicate. The presented data are expressed as the average {+/-} SD. Each dot indicates the result of an individual replicate. (E-G) Neutralization assay. Assays were performed with pseudoviruses harboring the S proteins of B.1.1, BA.1, BA.2, BA.5, BQ.1.1, XBB.1, XBB.1.5/1.9.2, XBB.1.16, EG.5.1, XBB.1.5/1.9.2+Q52H, and XBB.1.5/1.9.2+F456L. The following sera were used: convalescent sera from fully vaccinated individuals who had been infected with XBB.1.5 (one 3-dose vaccinated. 1 donor in total), XBB.1.9 (one 3-dose vaccinated donor, one 4-dose vaccinated donor and one 5-dose vaccinated donor. 3 donors in total), and XBB.1.16 (one 2-dose vaccinated donor, two 3-dose vaccinated donors, and one 4-dose vaccinated donor. 4 donors in total) (E); sera from hamster infected with XBB.1 (left) or XBB.1.5 (right) (F); and convalescent sera from fully vaccinated individuals who had been infected with BA.1 (thirteen 2-dose vaccinated, 13 donors in total) (left)1, BA.2 (nine 2-dose vaccinated and four 3-dose vaccinated donors. 13 donors in total) (middle)19, and BA.5 (one 2-dose vaccinated, thirteen 3-dose vaccinated donors, and one 4-dose vaccinated. 15 donors in total) (right)19 (G). Each dot indicates the result of an individual replicate. Assays for each serum sample were performed in triplicate to determine the 50% neutralization titer (NT50). Each dot represents one NT50 value, and the geometric mean and 95% confidence interval are shown. The number in parenthesis indicates the mean of NT50 values. The horizontal dash line indicates the detection limit (120-fold). In D, statistically significant differences (**, P \u003c 0.001, ***, P \u003c 0.0001) versus XBB.1.5/1.9.2 were determined by two-sided Students t tests. Blue asterisks indicate decreased percentage of infectivity. In E and G, statistically significant differences versus B.1.1 were determined by two-sided Wilcoxon signed-rank tests. The fold change between B.1.1 and the variant indicated is shown in red. Background information on the convalescent donors is summarized in Table S1. In F, statistically significant differences (*, P \u003c 0.01, **, P \u003c 0.001) between B.1.1 and XBB.1 (left) or XBB.1.5/1.9.2 (right) were determined by two-sided Wilcoxon signed-rank tests and indicated with asterisks. Red asterisks indicate decreased NT50s. C_FIG\nSARS-CoV-2 hijacks fragile X mental retardation proteins for efficient infection\nAuthors: Garvanska, D.; Alvarado, R. E.; Mundt, F.; Nilsson, E.; Duel, J.; Coscia, F.; Lindqvist, R.; Lokugamage, K.; Johnson, B.; Plante, J. A.; Morris, D. R.; Vu, M. N.; Estes, L.; McLeland, A.; Walker, J.; Crocquet-Valdes, P.; Lopez-Mendez, B.; Plante, K.; Walker, D. H.; Weisser, M. B.; Overby, A. K.; Mann, M.; Menachery, V. D.; Nilsson, J.\nScore: 17.6, Published: 2023-09-01 DOI: 10.1101/2023.09.01.555899\nViruses interact with numerous host factors to facilitate viral replication and to dampen antiviral defense mechanisms. We currently have a limited mechanistic understanding of how SARS-CoV-2 binds host factors and the functional role of these interactions. Here, we uncover a novel interaction between the viral NSP3 protein and the fragile X mental retardation proteins (FMRPs: FMR1 and FXR1-2). SARS-CoV-2 NSP3 mutant viruses preventing FMRP binding have attenuated replication in vitro and have delayed disease onset in vivo. We show that a unique peptide motif in NSP3 binds directly to the two central KH domains of FMRPs and that this interaction is disrupted by the I304N mutation found in a patient with fragile X syndrome. NSP3 binding to FMRPs disrupts their interaction with the stress granule component UBAP2L through direct competition with a peptide motif in UBAP2L to prevent FMRP incorporation into stress granules. Collectively, our results provide novel insight into how SARS-CoV-2 hijacks host cell proteins for efficient infection and provides molecular insight to the possible underlying molecular defects in fragile X syndrome.\nAntibody Neutralization of Emerging SARS-CoV-2: EG.5.1 and XBC.1.6\nAuthors: Wang, Q.; Guo, Y.; Zhang, R. M.; Ho, J.; Mohri, H.; Valdez, R.; Manthei, D. M.; Gordon, A.; Liu, L.; Ho, D. D.\nScore: 32.0, Published: 2023-08-25 DOI: 10.1101/2023.08.21.553968\nSARS-CoV-2 variants EG.5.1 and XBC.1.6 have recently emerged, attracting increased attention due to their rapid expansion globally and in Australia, respectively. EG.5.1 evolved from Omicron subvariant XBB.1.9, harboring additional Q52H and F456L spike substitutions. The F456L mutation is located within the epitopes of many class-1 monoclonal antibodies (mAbs) directed to the receptor-binding domain (RBD), raising concerns about further antibody evasion. XBC.1.6, a descendant of a Delta-BA.2 recombinant, carries 15 additional spike mutations. The extent to which antibody evasion contributes to the growth advantage of XBC.1.6 in Australia remains to be determined. To assess the antibody evasion properties of the emergent variants, we conducted pseudovirus neutralization assays using sera from individuals who received three doses of COVID-19 mRNA monovalent vaccines plus one dose of a BA.5 bivalent vaccine, as well as from patients with BQ or XBB breakthrough infection. The assays were also performed using a panel of 14 mAbs that retained neutralizing activity against prior XBB subvariants. Our data suggested that EG.5.1 was slightly but significantly more resistant (\u003c 2-fold) to neutralization by BQ and XBB breakthrough sera than XBB.1.16, which is known to be antigenically similar to XBB.1.5. Moreover, the F456L mutation in EG.5.1 conferred heightened resistance to certain RBD class-1 mAbs. In contrast, XBC.1.6 was more sensitive to neutralization by sera and mAbs than the XBB subvariants. Notably, XBB breakthrough sera retained only weak neutralization activity against XBB subvariants. In summary, EG.5.1 and XBC.1.6 exhibited distinct antibody evasion properties. The recent global expansion of EG.5.1 might be attributable, in part, to its enhanced neutralization resistance. That XBB breakthrough infections did not elicit a robust antibody neutralization response against XBB subvariants is indicative of immunological imprinting. The high prevalence of XBC.1.6 in Australia is not due to enhanced antibody evasion.\nDissecting the neurotropism and neurovirulence of MPXV using human stem cell-based models\nAuthors: Bauer, L.; Giussani, S.; Palazzi, N.; Zare, F.; Colombo, E.; Pinci, F.; Leijten, L.; Smeenk, H.; Embregts, C. W. E.; Silva, M.; Spoor, J. K. H.; Dirven, C.; Gao, Z.; Bolleboom, A.; Verstrepen, B. E.; Schuele, L.; de Vrij, F. M. S.; Kushner, S. A.; Oude Munnink, B. B.; Davila-Velderrain, J.; van Riel, D.; Harschnitz, O.\nScore: 16.6, Published: 2023-08-28 DOI: 10.1101/2023.08.25.554849\nMpox, formerly known as monkeypox, is a zoonotic illness of international concern that can lead to severe disease including neurological sequelae. However, it remains unclear what the neurotropism of monkeypox virus (MPXV) is and how MPXV infection leads to neurological deficits. Here, we determined the neurotropism and neurovirulence of MPXV using human pluripotent stem cell- (hPSC)-derived neural stem cells, astrocytes, cortical neurons, and microglia together with ex vivo human brain tissue. We found that MPXV infects and replicates more efficiently in astrocytes and microglia compared to cortical neurons, which unlike glial cells showed activation of distinct antiviral programs that may confer differential susceptibility to MPXV. Ex vivo infection of human brain tissue confirmed the susceptibility of astrocytes to MPXV infection, which also had the strongest disease-associated changes. Molecular pathway analyses revealed induction of cellular senescence and a senescence-associated secretory phenotype upon MPXV infection in astrocytes. Finally, we demonstrated that antiviral treatment using tecovirimat inhibits MPXV replication and prevents virus-induced senescence in hPSC-derived astrocytes. Altogether, leveraging hPSC-derived brain cells, we reveal MPXV-induced cell type-specific effects at the molecular and cellular level, which provide important insights into the neuropathogenesis of MPXV infection.\nHIV-1 binds dynein directly to hijack microtubule transport machinery\nAuthors: Badieyan, S.; Lichon, D.; Andreas, M. P.; Gillies, J. P.; Peng, W.; Shi, J.; DeSantis, M. E.; Aiken, C. R.; Böcking, T.; Giessen, T. W.; Campbell, E. M.; Cianfrocco, M. A.\nScore: 15.6, Published: 2023-08-29 DOI: 10.1101/2023.08.29.555335\nHIV-1 uses the microtubule cytoskeleton to reach the host cell nucleus during replication, yet the molecular basis for microtubule-dependent HIV-1 motility is poorly understood. Using in vitro reconstitution biochemistry and single-molecule imaging, we found that HIV-1 binds to the retrograde microtubule-associated motor, dynein, directly and not via a cargo adaptor, as has been previously suggested. We found dyneins tail domain binds to a novel interface on the HIV-1 capsid lattice. Further, we use an engineered encapsulin system to demonstrate that multiple dynein motors tethered to rigid cargoes, like HIV-1 capsids, display reduced motility, distinct from the behavior of multiple motors on membraneous cargoes. Our results provide a new model of HIV-1 trafficking wherein HIV-1 hijacks the dynein transport machinery for microtubule motility, establishing a new model of viral trafficking by directly co-opting host dynein motors.\nSequential early-life viral infections modulate the microbiota and adaptive immune responses to systemic and mucosal vaccination\nAuthors: Li, Y.; Molleston, J. M.; Kim, A. H.; Ingle, H.; Aggarwal, S.; Nolan, L.; Hassan, A.; Foster, L.; Diamond, M.; Baldridge, M. T.\nScore: 15.6, Published: 2023-09-01 DOI: 10.1101/2023.08.31.555772\nIncreasing evidence points to the microbial exposome as a critical factor in maturing and shaping the host immune system, thereby influencing responses to immune challenges such as infections or vaccines. To investigate the effect of early-life viral exposures on immune development and vaccine responses, we inoculated mice with six distinct viral pathogens in sequence beginning in the neonatal period, and then evaluated their immune signatures before and after intramuscular or intranasal vaccination against SARS-CoV-2. Sequential viral infection drove profound changes in all aspects of the immune system, including increasing circulating leukocytes, altering innate and adaptive immune cell lineages in tissues, and markedly influencing serum cytokine and total antibody levels. Beyond these immune responses changes, these exposures also modulated the composition of the endogenous intestinal microbiota. Although sequentially-infected mice exhibited increased systemic immune activation and T cell responses after intramuscular and intranasal SARS-CoV-2 immunization, we observed decreased vaccine-induced antibody responses in these animals. These results suggest that early-life viral exposures are sufficient to diminish antibody responses to vaccination in mice, and highlight their potential importance of considering prior microbial exposures when investigating vaccine responses.\nDisruption of Aldehyde Dehydrogenase 2 protects against bacterial infection\nAuthors: Berry, S.; Espich, S.; Thuong, N. T. T.; Chang, X.; Dorajoo, R.; Khor, C. C.; Heng, C. K.; Yuan, J.-M.; Fox, D.; Anaya-Sanchez, A.; Tenney, L.; Chang, C. J.; Kotov, D. I.; Vance, R. E.; Dunstan, S. J.; Darwin, K. H.; Stanley, S. A.\nScore: 27.2, Published: 2023-08-26 DOI: 10.1101/2023.08.24.554661\nThe ALDH2*2 (rs671) allele is one of the most common genetic mutations in humans, yet the positive evolutionary selective pressure to maintain this mutation is unknown, despite its association with adverse health outcomes. ALDH2 is responsible for the detoxification of metabolically produced aldehydes, including lipid-peroxidation end products derived from inflammation. Here, we demonstrate that host-derived aldehydes 4-hydroxynonenal (4HNE), malondialdehyde (MDA), and formaldehyde (FA), all of which are metabolized by ALDH2, are directly toxic to the bacterial pathogens Mycobacterium tuberculosis and Francisella tularensis at physiological levels. We find that Aldh2 expression in macrophages is decreased upon immune stimulation, and that bone marrow-derived macrophages from Aldh2-/- mice contain elevated aldehydes relative to wild-type mice. Macrophages deficient for Aldh2 exhibited enhanced control of Francisella infection. Finally, mice lacking Aldh2 demonstrated increased resistance to pulmonary infection by M. tuberculosis, including in a hypersusceptible model of tuberculosis, and were also resistant to Francisella infection. We hypothesize that the absence of ALDH2 contributes to the hosts ability to control infection by pathogens such as M. tuberculosis and F. tularensis, and that host-derived aldehydes act as antimicrobial factors during intracellular bacterial infections. One sentence summaryAldehydes produced by host cells contribute to the control of bacterial infections.\nA mutant fitness compendium in Bifidobacteria reveals molecular determinants of colonization and host-microbe interactions\nAuthors: Shiver, A. L.; Sun, J.; Culver, R.; Violette, A.; Wynter, C.; Nieckarz, M.; Mattiello, S. P.; Sekhon, P. K.; Friess, L.; Carlson, H. K.; Wong, D.; Higginbottom, S. K.; Weglarz, M.; Wang, W.; Knapp, B. D.; Guiberson, E.; Sanchez, J.; Huang, P.-H.; Garcia, P. A.; Buie, C. R.; Good, B. H.; DeFelice, B.; Cava, F.; Scaria, J.; Sonnenburg, J.; Van Sinderen, D.; Deutschbauer, A. M.; Huang, K. C.\nScore: 13.9, Published: 2023-08-29 DOI: 10.1101/2023.08.29.555234\nBifidobacteria commonly represent a dominant constituent of human gut microbiomes during infancy, influencing nutrition, immune development, and resistance to infection. Despite interest as a probiotic therapy, predicting the nutritional requirements and health-promoting effects of Bifidobacteria is challenging due to major knowledge gaps. To overcome these deficiencies, we used large-scale genetics to create a compendium of mutant fitness in Bifidobacterium breve (Bb). We generated a high density, randomly barcoded transposon insertion pool in Bb, and used this pool to determine Bb fitness requirements during colonization of germ-free mice and chickens with multiple diets and in response to hundreds of in vitro perturbations. To enable mechanistic investigation, we constructed an ordered collection of insertion strains covering 1462 genes. We leveraged these tools to improve models of metabolic pathways, reveal unexpected host- and diet-specific requirements for colonization, and connect the production of immunomodulatory molecules to growth benefits. These resources will greatly reduce the barrier to future investigations of this important beneficial microbe.\n",
  "wordCount" : "3096",
  "inLanguage": "en",
  "datePublished": "2023-09-03T10:37:18Z",
  "dateModified": "2023-09-03T10:37:18Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta"><span>updated on September 3, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.31.555819">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.31.555819" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.31.555819">
        <p class="paperTitle">Characterization of an EG.5.1 clinical isolate in vitro and in vivo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.31.555819" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.31.555819" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Uraki, R.; Kiso, M.; Iwatsuki-Horimoto, K.; Yamayoshi, S.; Ito, M.; Chiba, S.; Sakai-Tagawa, Y.; Imai, M.; Kashima, Y.; Koga, M.; Fuwa, N.; Okumura, N.; Hojo, M.; Iwamoto, N.; Kato, H.; Nakajima, H.; Ohmagari, N.; Yotsuyanagi, H.; Suzuki, Y.; Kawaoka, Y.</p>
        <p class="info">Score: 52.3, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.31.555819' target='https://doi.org/10.1101/2023.08.31.555819'> 10.1101/2023.08.31.555819</a></p>
        <p class="abstract">EG.5.1 is a subvariant of the SARS-CoV-2 Omicron XBB variant that is rapidly increasing in prevalence worldwide. EG.5.1 has additional substitutions in its spike protein (namely, Q52H and F456L) compared with XBB.1.5. However, the pathogenicity, transmissibility, and immune evasion properties of clinical isolates of EG.5.1 are largely unknown. In this study, we used wild-type Syrian hamsters to investigate the replicative ability, pathogenicity, and transmissibility of a clinical EG.5.1 isolate. Our data show that there are no obvious differences in growth ability and pathogenicity between EG.5.1 and XBB.1.5, and both EG.5.1 and XBB.1.5 are attenuated compared to a Delta variant isolate. We also found that EG.5.1 is transmitted more efficiently between hamsters compared with XBB.1.5. In addition, unlike XBB.1.5, we detected EG.5.1 virus in the lungs of four of six exposed hamsters, suggesting that the virus tropism of EG.5.1 is different from that of XBB.1.5 after airborne transmission. Finally, we assessed the neutralizing ability of plasma from convalescent individuals and found that the neutralizing activity against EG.5.1 was slightly, but significantly, lower than that against XBB.1.5 or XBB.1.9.2. This suggests that EG.5.1 effectively evades humoral immunity and that the amino acid differences in the S protein of EG.5.1 compared with that of XBB.1.5 or XBB.1.9.2 (i.e., Q52H, R158G, and F456L) alter the antigenicity of EG.5.1. Our data suggest that the increased transmissibility and altered antigenicity of EG.5.1 may be driving its increasing prevalence over XBB.1.5 in the human population.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555216">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555216" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555216">
        <p class="paperTitle">Community ecology of phages on a clonal bacterial host</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555216" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555216" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pyenson, N. C.; Leeks, A.; Nweke, O.; Goldford, J.; Turner, P.; Foster, K. R.; Sanchez, A.</p>
        <p class="info">Score: 36.4, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555216' target='https://doi.org/10.1101/2023.09.01.555216'> 10.1101/2023.09.01.555216</a></p>
        <p class="abstract">Bacteriophages are the most abundant and diverse biological entities on Earth, yet the ecological mechanisms that sustain this extraordinary diversity remain unclear. Here, we have discovered a general mechanism that allows phage diversity to outstrip the diversity of their bacterial hosts. We assembled and passaged dozens of diverse phage communities on a single, non-coevolving strain of Escherichia coli until the phage communities reached equilibrium. In all cases, we found multiple phage species coexisted stably, despite competition for a single, clonal host population. Coexistence within these communities was supported through host phenotypic heterogeneity, whereby phages specialized on cells adopting different growth phenotypes. Our experiments reveal a rich community ecology of bacteriophages where multiple phage species coexist and interact at the scale of a single bacterial host.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552415">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552415" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552415">
        <p class="paperTitle">Antiviral efficacy of the SARS-CoV-2 XBB breakthrough infection sera against Omicron subvariants including EG.5</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552415" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552415" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kaku, Y.; Kosugi, Y.; Uriu, K.; Ito, J.; Kuramochi, J.; Sadamasu, K.; Yoshimura, K.; Asakura, H.; Nagashima, M.; The Genotype to Phenotype Japan (G2P-Japan) Consortium,  ; Sato, K.</p>
        <p class="info">Score: 1038.1, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552415' target='https://doi.org/10.1101/2023.08.08.552415'> 10.1101/2023.08.08.552415</a></p>
        <p class="abstract">As of July 2023, EG.5.1 (a.k.a. XBB.1.9.2.5.1), a XBB subvariant bearing the S:Q52H and S:F456L substitutions, alongside the S:F486P substitution (Figure S1A), has rapidly spread in some countries. On July 19, 2023, the WHO classified EG.5 as a variant under monitoring. First, we showed that EG.5.1 exhibits a higher effective reproduction number compared with XBB.1.5, XBB.1.16, and its parental lineage (XBB.1.9.2), suggesting that EG.5.1 will spread globally and outcompete these XBB subvariants in the near future. We then addressed whether EG.5.1 evades from the antiviral effect of the humoral immunity induced by breakthrough infection (BTI) of XBB subvariants and performed a neutralization assay using XBB BTI sera. However, the 50% neutralization titer (NT50) of XBB BTI sera against EG.5.1 was comparable to those against XBB.1.5/1.9.2 and XBB.1.16. Moreover, the sensitivity of EG.5.1 to convalescent sera of XBB.1- and XBB.1.5-infected hamsters was similar to those of XBB.1.5/1.9 and XBB.1.16. These results suggest that the increased Re of EG.5.1 is attributed to neither increased infectivity nor immune evasion from XBB BTI, and the emergence and spread of EG.5 is driven by the other pressures. We previously demonstrated that Omicron BTI cannot efficiently induce antiviral humoral immunity against the variant infected. In fact, the NT50s of the BTI sera of Omicron BA.1, BA.2, and BA.5 against the variant infected were 3.0-, 2.2-, and 3.4-fold lower than that against the ancestral B.1.1 variant, respectively. However, strikingly, we found that the NT50 of the BTI sera of XBB1.5/1.9 and XBB.1.16 against the variant infected were 8.7- and 8.3-fold lower than that against the B.1.1 variant. These results suggest that XBB BTI cannot efficiently induce antiviral humoral immunity against XBB subvariants.

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=147 SRC=&#34;FIGDIR/small/552415v1_figs1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (39K):
org.highwire.dtl.DTLVardef@f95376org.highwire.dtl.DTLVardef@d66fa8org.highwire.dtl.DTLVardef@3c8841org.highwire.dtl.DTLVardef@15824c_HPS_FORMAT_FIGEXP  M_FIG O_FLOATNOFigure S1.C_FLOATNO Virological features of EG.5.1 and XBB BTI (A) Frequency of mutations of interest in the representative XBB sublineages. Only mutations with a frequency &gt;0.5 in at least one but not all the representative sublineages are shown. Note that the S proteins of XBB.1.5 and XBB.1.9.2 are identical.

(B) Estimated epidemic dynamics of the representative XBB sublineages in countries where &gt;50 sequences of EG.5.1, XBB.1.5, XBB.1.9.2, and XBB.1.16 were detected from April 1, 2023 to July 13, 2023. Countries are ordered according to the number of detected sequences of EG.5.1. Line, posterior mean; ribbon, 95% Bayesian confidence interval. The dynamics for EG.5.1 is highlighted by a red arrowhead.

(C) Estimated relative Re of the representative XBB sublineages in the six countries. The relative Re of XBB.1.5 is set to 1 (horizontal dashed line). Violin, posterior distribution; dot, posterior mean; line, 95% Bayesian confidence interval.

(D) Lentivirus-based pseudovirus assay. HOS-ACE2-TMPRSS2 cells were infected with pseudoviruses bearing each S protein. The amount of input virus was normalized to the amount of HIV-1 p24 capsid protein. The percentage infectivity of XBB.1.5/1.9.2, XBB.1.5/1.9.2&#43;Q52H, XBB.1.5/1.9.2&#43;F456L, and EG.5.1 compared to that of XBB.1.5/1.9.2 are shown. The horizontal dash line indicates the mean value of the percentage infectivity of the XBB.1.5/1.9.2. Assays were performed in quadruplicate. The presented data are expressed as the average {&#43;/-} SD. Each dot indicates the result of an individual replicate.

(E-G) Neutralization assay. Assays were performed with pseudoviruses harboring the S proteins of B.1.1, BA.1, BA.2, BA.5, BQ.1.1, XBB.1, XBB.1.5/1.9.2, XBB.1.16, EG.5.1, XBB.1.5/1.9.2&#43;Q52H, and XBB.1.5/1.9.2&#43;F456L. The following sera were used: convalescent sera from fully vaccinated individuals who had been infected with XBB.1.5 (one 3-dose vaccinated. 1 donor in total), XBB.1.9 (one 3-dose vaccinated donor, one 4-dose vaccinated donor and one 5-dose vaccinated donor. 3 donors in total), and XBB.1.16 (one 2-dose vaccinated donor, two 3-dose vaccinated donors, and one 4-dose vaccinated donor. 4 donors in total) (E); sera from hamster infected with XBB.1 (left) or XBB.1.5 (right) (F); and convalescent sera from fully vaccinated individuals who had been infected with BA.1 (thirteen 2-dose vaccinated, 13 donors in total) (left)1, BA.2 (nine 2-dose vaccinated and four 3-dose vaccinated donors. 13 donors in total) (middle)19, and BA.5 (one 2-dose vaccinated, thirteen 3-dose vaccinated donors, and one 4-dose vaccinated. 15 donors in total) (right)19 (G). Each dot indicates the result of an individual replicate. Assays for each serum sample were performed in triplicate to determine the 50% neutralization titer (NT50). Each dot represents one NT50 value, and the geometric mean and 95% confidence interval are shown. The number in parenthesis indicates the mean of NT50 values. The horizontal dash line indicates the detection limit (120-fold).

In D, statistically significant differences (**, P &lt; 0.001, ***, P &lt; 0.0001) versus XBB.1.5/1.9.2 were determined by two-sided Students t tests. Blue asterisks indicate decreased percentage of infectivity.

In E and G, statistically significant differences versus B.1.1 were determined by two-sided Wilcoxon signed-rank tests. The fold change between B.1.1 and the variant indicated is shown in red. Background information on the convalescent donors is summarized in Table S1.

In F, statistically significant differences (*, P &lt; 0.01, **, P &lt; 0.001) between B.1.1 and XBB.1 (left) or XBB.1.5/1.9.2 (right) were determined by two-sided Wilcoxon signed-rank tests and indicated with asterisks. Red asterisks indicate decreased NT50s.

C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555899">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555899" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555899">
        <p class="paperTitle">SARS-CoV-2 hijacks fragile X mental retardation proteins for efficient infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555899" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555899" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Garvanska, D.; Alvarado, R. E.; Mundt, F.; Nilsson, E.; Duel, J.; Coscia, F.; Lindqvist, R.; Lokugamage, K.; Johnson, B.; Plante, J. A.; Morris, D. R.; Vu, M. N.; Estes, L.; McLeland, A.; Walker, J.; Crocquet-Valdes, P.; Lopez-Mendez, B.; Plante, K.; Walker, D. H.; Weisser, M. B.; Overby, A. K.; Mann, M.; Menachery, V. D.; Nilsson, J.</p>
        <p class="info">Score: 17.6, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555899' target='https://doi.org/10.1101/2023.09.01.555899'> 10.1101/2023.09.01.555899</a></p>
        <p class="abstract">Viruses interact with numerous host factors to facilitate viral replication and to dampen antiviral defense mechanisms. We currently have a limited mechanistic understanding of how SARS-CoV-2 binds host factors and the functional role of these interactions. Here, we uncover a novel interaction between the viral NSP3 protein and the fragile X mental retardation proteins (FMRPs: FMR1 and FXR1-2). SARS-CoV-2 NSP3 mutant viruses preventing FMRP binding have attenuated replication in vitro and have delayed disease onset in vivo. We show that a unique peptide motif in NSP3 binds directly to the two central KH domains of FMRPs and that this interaction is disrupted by the I304N mutation found in a patient with fragile X syndrome. NSP3 binding to FMRPs disrupts their interaction with the stress granule component UBAP2L through direct competition with a peptide motif in UBAP2L to prevent FMRP incorporation into stress granules. Collectively, our results provide novel insight into how SARS-CoV-2 hijacks host cell proteins for efficient infection and provides molecular insight to the possible underlying molecular defects in fragile X syndrome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.553968">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.553968" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.553968">
        <p class="paperTitle">Antibody Neutralization of Emerging SARS-CoV-2: EG.5.1 and XBC.1.6</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.553968" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.553968" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Q.; Guo, Y.; Zhang, R. M.; Ho, J.; Mohri, H.; Valdez, R.; Manthei, D. M.; Gordon, A.; Liu, L.; Ho, D. D.</p>
        <p class="info">Score: 32.0, Published: 2023-08-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.553968' target='https://doi.org/10.1101/2023.08.21.553968'> 10.1101/2023.08.21.553968</a></p>
        <p class="abstract">SARS-CoV-2 variants EG.5.1 and XBC.1.6 have recently emerged, attracting increased attention due to their rapid expansion globally and in Australia, respectively. EG.5.1 evolved from Omicron subvariant XBB.1.9, harboring additional Q52H and F456L spike substitutions. The F456L mutation is located within the epitopes of many class-1 monoclonal antibodies (mAbs) directed to the receptor-binding domain (RBD), raising concerns about further antibody evasion. XBC.1.6, a descendant of a Delta-BA.2 recombinant, carries 15 additional spike mutations. The extent to which antibody evasion contributes to the growth advantage of XBC.1.6 in Australia remains to be determined. To assess the antibody evasion properties of the emergent variants, we conducted pseudovirus neutralization assays using sera from individuals who received three doses of COVID-19 mRNA monovalent vaccines plus one dose of a BA.5 bivalent vaccine, as well as from patients with BQ or XBB breakthrough infection. The assays were also performed using a panel of 14 mAbs that retained neutralizing activity against prior XBB subvariants. Our data suggested that EG.5.1 was slightly but significantly more resistant (&lt; 2-fold) to neutralization by BQ and XBB breakthrough sera than XBB.1.16, which is known to be antigenically similar to XBB.1.5. Moreover, the F456L mutation in EG.5.1 conferred heightened resistance to certain RBD class-1 mAbs. In contrast, XBC.1.6 was more sensitive to neutralization by sera and mAbs than the XBB subvariants. Notably, XBB breakthrough sera retained only weak neutralization activity against XBB subvariants. In summary, EG.5.1 and XBC.1.6 exhibited distinct antibody evasion properties. The recent global expansion of EG.5.1 might be attributable, in part, to its enhanced neutralization resistance. That XBB breakthrough infections did not elicit a robust antibody neutralization response against XBB subvariants is indicative of immunological imprinting. The high prevalence of XBC.1.6 in Australia is not due to enhanced antibody evasion.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.554849">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.554849" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.554849">
        <p class="paperTitle">Dissecting the neurotropism and neurovirulence of MPXV using human stem cell-based models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.554849" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.554849" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bauer, L.; Giussani, S.; Palazzi, N.; Zare, F.; Colombo, E.; Pinci, F.; Leijten, L.; Smeenk, H.; Embregts, C. W. E.; Silva, M.; Spoor, J. K. H.; Dirven, C.; Gao, Z.; Bolleboom, A.; Verstrepen, B. E.; Schuele, L.; de Vrij, F. M. S.; Kushner, S. A.; Oude Munnink, B. B.; Davila-Velderrain, J.; van Riel, D.; Harschnitz, O.</p>
        <p class="info">Score: 16.6, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.554849' target='https://doi.org/10.1101/2023.08.25.554849'> 10.1101/2023.08.25.554849</a></p>
        <p class="abstract">Mpox, formerly known as monkeypox, is a zoonotic illness of international concern that can lead to severe disease including neurological sequelae. However, it remains unclear what the neurotropism of monkeypox virus (MPXV) is and how MPXV infection leads to neurological deficits. Here, we determined the neurotropism and neurovirulence of MPXV using human pluripotent stem cell- (hPSC)-derived neural stem cells, astrocytes, cortical neurons, and microglia together with ex vivo human brain tissue. We found that MPXV infects and replicates more efficiently in astrocytes and microglia compared to cortical neurons, which unlike glial cells showed activation of distinct antiviral programs that may confer differential susceptibility to MPXV. Ex vivo infection of human brain tissue confirmed the susceptibility of astrocytes to MPXV infection, which also had the strongest disease-associated changes. Molecular pathway analyses revealed induction of cellular senescence and a senescence-associated secretory phenotype upon MPXV infection in astrocytes. Finally, we demonstrated that antiviral treatment using tecovirimat inhibits MPXV replication and prevents virus-induced senescence in hPSC-derived astrocytes. Altogether, leveraging hPSC-derived brain cells, we reveal MPXV-induced cell type-specific effects at the molecular and cellular level, which provide important insights into the neuropathogenesis of MPXV infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.555335">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.555335" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.555335">
        <p class="paperTitle">HIV-1 binds dynein directly to hijack microtubule transport machinery</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.555335" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.555335" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Badieyan, S.; Lichon, D.; Andreas, M. P.; Gillies, J. P.; Peng, W.; Shi, J.; DeSantis, M. E.; Aiken, C. R.; Böcking, T.; Giessen, T. W.; Campbell, E. M.; Cianfrocco, M. A.</p>
        <p class="info">Score: 15.6, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.555335' target='https://doi.org/10.1101/2023.08.29.555335'> 10.1101/2023.08.29.555335</a></p>
        <p class="abstract">HIV-1 uses the microtubule cytoskeleton to reach the host cell nucleus during replication, yet the molecular basis for microtubule-dependent HIV-1 motility is poorly understood. Using in vitro reconstitution biochemistry and single-molecule imaging, we found that HIV-1 binds to the retrograde microtubule-associated motor, dynein, directly and not via a cargo adaptor, as has been previously suggested. We found dyneins tail domain binds to a novel interface on the HIV-1 capsid lattice. Further, we use an engineered encapsulin system to demonstrate that multiple dynein motors tethered to rigid cargoes, like HIV-1 capsids, display reduced motility, distinct from the behavior of multiple motors on membraneous cargoes. Our results provide a new model of HIV-1 trafficking wherein HIV-1  hijacks the dynein transport machinery for microtubule motility, establishing a new model of viral trafficking by directly co-opting host dynein motors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.31.555772">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.31.555772" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.31.555772">
        <p class="paperTitle">Sequential early-life viral infections modulate the microbiota and adaptive immune responses to systemic and mucosal vaccination</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.31.555772" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.31.555772" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, Y.; Molleston, J. M.; Kim, A. H.; Ingle, H.; Aggarwal, S.; Nolan, L.; Hassan, A.; Foster, L.; Diamond, M.; Baldridge, M. T.</p>
        <p class="info">Score: 15.6, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.31.555772' target='https://doi.org/10.1101/2023.08.31.555772'> 10.1101/2023.08.31.555772</a></p>
        <p class="abstract">Increasing evidence points to the microbial exposome as a critical factor in maturing and shaping the host immune system, thereby influencing responses to immune challenges such as infections or vaccines. To investigate the effect of early-life viral exposures on immune development and vaccine responses, we inoculated mice with six distinct viral pathogens in sequence beginning in the neonatal period, and then evaluated their immune signatures before and after intramuscular or intranasal vaccination against SARS-CoV-2. Sequential viral infection drove profound changes in all aspects of the immune system, including increasing circulating leukocytes, altering innate and adaptive immune cell lineages in tissues, and markedly influencing serum cytokine and total antibody levels. Beyond these immune responses changes, these exposures also modulated the composition of the endogenous intestinal microbiota. Although sequentially-infected mice exhibited increased systemic immune activation and T cell responses after intramuscular and intranasal SARS-CoV-2 immunization, we observed decreased vaccine-induced antibody responses in these animals. These results suggest that early-life viral exposures are sufficient to diminish antibody responses to vaccination in mice, and highlight their potential importance of considering prior microbial exposures when investigating vaccine responses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554661">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554661" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554661">
        <p class="paperTitle">Disruption of Aldehyde Dehydrogenase 2 protects against bacterial infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554661" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554661" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Berry, S.; Espich, S.; Thuong, N. T. T.; Chang, X.; Dorajoo, R.; Khor, C. C.; Heng, C. K.; Yuan, J.-M.; Fox, D.; Anaya-Sanchez, A.; Tenney, L.; Chang, C. J.; Kotov, D. I.; Vance, R. E.; Dunstan, S. J.; Darwin, K. H.; Stanley, S. A.</p>
        <p class="info">Score: 27.2, Published: 2023-08-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554661' target='https://doi.org/10.1101/2023.08.24.554661'> 10.1101/2023.08.24.554661</a></p>
        <p class="abstract">The ALDH2*2 (rs671) allele is one of the most common genetic mutations in humans, yet the positive evolutionary selective pressure to maintain this mutation is unknown, despite its association with adverse health outcomes. ALDH2 is responsible for the detoxification of metabolically produced aldehydes, including lipid-peroxidation end products derived from inflammation. Here, we demonstrate that host-derived aldehydes 4-hydroxynonenal (4HNE), malondialdehyde (MDA), and formaldehyde (FA), all of which are metabolized by ALDH2, are directly toxic to the bacterial pathogens Mycobacterium tuberculosis and Francisella tularensis at physiological levels. We find that Aldh2 expression in macrophages is decreased upon immune stimulation, and that bone marrow-derived macrophages from Aldh2-/- mice contain elevated aldehydes relative to wild-type mice. Macrophages deficient for Aldh2 exhibited enhanced control of Francisella infection. Finally, mice lacking Aldh2 demonstrated increased resistance to pulmonary infection by M. tuberculosis, including in a hypersusceptible model of tuberculosis, and were also resistant to Francisella infection. We hypothesize that the absence of ALDH2 contributes to the hosts ability to control infection by pathogens such as M. tuberculosis and F. tularensis, and that host-derived aldehydes act as antimicrobial factors during intracellular bacterial infections.

One sentence summaryAldehydes produced by host cells contribute to the control of bacterial infections.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.555234">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.555234" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.555234">
        <p class="paperTitle">A mutant fitness compendium in Bifidobacteria reveals molecular determinants of colonization and host-microbe interactions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.555234" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.555234" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shiver, A. L.; Sun, J.; Culver, R.; Violette, A.; Wynter, C.; Nieckarz, M.; Mattiello, S. P.; Sekhon, P. K.; Friess, L.; Carlson, H. K.; Wong, D.; Higginbottom, S. K.; Weglarz, M.; Wang, W.; Knapp, B. D.; Guiberson, E.; Sanchez, J.; Huang, P.-H.; Garcia, P. A.; Buie, C. R.; Good, B. H.; DeFelice, B.; Cava, F.; Scaria, J.; Sonnenburg, J.; Van Sinderen, D.; Deutschbauer, A. M.; Huang, K. C.</p>
        <p class="info">Score: 13.9, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.555234' target='https://doi.org/10.1101/2023.08.29.555234'> 10.1101/2023.08.29.555234</a></p>
        <p class="abstract">Bifidobacteria commonly represent a dominant constituent of human gut microbiomes during infancy, influencing nutrition, immune development, and resistance to infection. Despite interest as a probiotic therapy, predicting the nutritional requirements and health-promoting effects of Bifidobacteria is challenging due to major knowledge gaps. To overcome these deficiencies, we used large-scale genetics to create a compendium of mutant fitness in Bifidobacterium breve (Bb). We generated a high density, randomly barcoded transposon insertion pool in Bb, and used this pool to determine Bb fitness requirements during colonization of germ-free mice and chickens with multiple diets and in response to hundreds of in vitro perturbations. To enable mechanistic investigation, we constructed an ordered collection of insertion strains covering 1462 genes. We leveraged these tools to improve models of metabolic pathways, reveal unexpected host- and diet-specific requirements for colonization, and connect the production of immunomodulatory molecules to growth benefits. These resources will greatly reduce the barrier to future investigations of this important beneficial microbe.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
