<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="AMAPEC: accurate antimicrobial activity prediction for fungal effector proteins
Authors: Mesny, F.; Thomma, B. P.
Score: 46.6, Published: 2024-01-04 DOI: 10.1101/2024.01.04.574150
Fungi typically occur in environments where numerous and diverse other microbes occur as well, often resulting in fierce competition for nutrients and habitat. To support fungal fitness in these environments, they evolved various mechanisms that mediate direct antagonism towards niche competitors. Among these, the secretion of proteins with antimicrobial activities has been reported in fungi with diverse lifestyles.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="AMAPEC: accurate antimicrobial activity prediction for fungal effector proteins
Authors: Mesny, F.; Thomma, B. P.
Score: 46.6, Published: 2024-01-04 DOI: 10.1101/2024.01.04.574150
Fungi typically occur in environments where numerous and diverse other microbes occur as well, often resulting in fierce competition for nutrients and habitat. To support fungal fitness in these environments, they evolved various mechanisms that mediate direct antagonism towards niche competitors. Among these, the secretion of proteins with antimicrobial activities has been reported in fungi with diverse lifestyles." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-07T10:38:59+00:00" />
<meta property="article:modified_time" content="2024-01-07T10:38:59+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="AMAPEC: accurate antimicrobial activity prediction for fungal effector proteins
Authors: Mesny, F.; Thomma, B. P.
Score: 46.6, Published: 2024-01-04 DOI: 10.1101/2024.01.04.574150
Fungi typically occur in environments where numerous and diverse other microbes occur as well, often resulting in fierce competition for nutrients and habitat. To support fungal fitness in these environments, they evolved various mechanisms that mediate direct antagonism towards niche competitors. Among these, the secretion of proteins with antimicrobial activities has been reported in fungi with diverse lifestyles."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "AMAPEC: accurate antimicrobial activity prediction for fungal effector proteins\nAuthors: Mesny, F.; Thomma, B. P.\nScore: 46.6, Published: 2024-01-04 DOI: 10.1101/2024.01.04.574150\nFungi typically occur in environments where numerous and diverse other microbes occur as well, often resulting in fierce competition for nutrients and habitat. To support fungal fitness in these environments, they evolved various mechanisms that mediate direct antagonism towards niche competitors. Among these, the secretion of proteins with antimicrobial activities has been reported in fungi with diverse lifestyles.",
  "keywords": [
    
  ],
  "articleBody": " AMAPEC: accurate antimicrobial activity prediction for fungal effector proteins\nAuthors: Mesny, F.; Thomma, B. P.\nScore: 46.6, Published: 2024-01-04 DOI: 10.1101/2024.01.04.574150\nFungi typically occur in environments where numerous and diverse other microbes occur as well, often resulting in fierce competition for nutrients and habitat. To support fungal fitness in these environments, they evolved various mechanisms that mediate direct antagonism towards niche competitors. Among these, the secretion of proteins with antimicrobial activities has been reported in fungi with diverse lifestyles. Recently, several plant-associated fungi were shown to rely on the secretion of antimicrobial effector proteins to antagonize certain members of plant hosts microbiota and to successfully colonize plant tissues. Some of these effectors do not share homology with known antimicrobials and represent novel antibiotics. Accordingly, the occurrence and conservation of proteinaceous antimicrobials throughout the fungal tree of life remains enigmatic. Here we present a computational approach to annotate candidate antimicrobial effectors in fungal secretomes based on protein physicochemical properties. After curating a set of proteins that were experimentally verified to display antimicrobial activity and a set of proteins that lack such activity, we trained a machine learning classifier on properties of protein sequences and predicted structures. This predictor performs particularly well on fungal proteins (R2=0.89) according to our validations and is delivered as a software package named AMAPEC, dedicated to antimicrobial activity prediction for effector candidates. We subsequently used this novel software to predict antimicrobial effector catalogs in three phylogenetically distant fungi with distinct lifestyles, revealing relatively large catalogs of candidate antimicrobials for each of the three fungi, and suggesting a broad occurrence of such proteins throughout the fungal kingdom. Thus, AMAPEC is a unique method to uncover antimicrobials in fungal secretomes that are often sparsely functionally annotated, and may assist biological interpretations during omic analyses. It is freely available at https://github.com/fantin-mesny/amapec.\nLethal Infection of Human ACE2-Transgenic Mice Caused by SARS-CoV-2-related Pangolin Coronavirus GX_P2V(short_3UTR)\nAuthors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.\nScore: 43.6, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574008\nSARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) can cause 100% mortality in human ACE2-transgenic mice, potentially attributable to late-stage brain infection. This underscores a spillover risk of GX_P2V into humans and provides a unique model for understanding the pathogenic mechanisms of SARS-CoV-2-related viruses.\nTIR domains produce histidine-ADPR conjugates as immune signaling molecules in bacteria\nAuthors: Sabonis, D.; Avraham, C.; Lu, A.; Herbst, E.; Silanskas, A.; Leavitt, A.; Yirmiya, E.; Zaremba, M.; Amitai, G.; Kranzusch, P. J.; Sorek, R.; Tamulaitiene, G.\nScore: 35.9, Published: 2024-01-03 DOI: 10.1101/2024.01.03.573942\nTIR domains are central components of pattern recognition immune proteins across all domains of life. In both bacteria and plants, TIR-domain proteins were shown to recognize pathogen invasion and then produce immune signaling molecules exclusively comprising nucleotide moieties. Here we show that the TIR domain protein of the type II Thoeris defense system in bacteria produces a unique signaling molecule comprising the amino acid histidine conjugated to ADP-ribose (His-ADPR). His-ADPR is generated in response to phage infection and activates the cognate Thoeris effector by binding a Macro domain located at the C-terminus of the effector protein. By determining the crystal structure of a ligand-bound Macro domain, we describe the structural basis for His-ADPR recognition. Our analyses furthermore reveal a family of phage proteins that bind and sequester His-ADPR signaling molecules, allowing phages to evade TIR- mediated immunity. These data demonstrate diversity in bacterial TIR signaling and reveal a new class of TIR-derived immune signaling molecules combining nucleotide and amino acid moieties.\nVirological characteristics of the SARS-CoV-2 JN.1 variant\nAuthors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Zahradnik, J.; Ito, J.; Sato, K.\nScore: 774.1, Published: 2023-12-09 DOI: 10.1101/2023.12.08.570782\nThe SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG.5.1 and HK.3. Comparing to XBB and BA.2, BA.2.86 carries more than 30 mutations in the spike (S) protein, indicating a high potential for immune evasion. BA.2.86 has evolved and its descendant, JN.1 (BA.2.86.1.1), emerged in late 2023. JN.1 harbors S:L455S and three mutations in non-S proteins. S:L455S is a hallmark mutation of JN.1: we have recently shown that HK.3 and other \"FLip\" variants carry S:L455F, which contributes to increased transmissibility and immune escape ability compared to the parental EG.5.1 variant. Here, we investigated the virological properties of JN.1.\nVaginal Lactobacillus fatty acid response mechanisms reveal a novel strategy for bacterial vaginosis treatment\nAuthors: Zhu, M.; Frank, M. W.; Radka, C. D.; Jeanfavre, S.; Tse, M. W.; Avila Pacheco, J.; Pierce, K.; Deik, A.; Xhu, J.; Hussain, S.; Hussain, F. A.; Xulu, N.; Khan, N.; Pillay, V.; Dong, K. L.; Ndung'u, T.; Clish, C. B.; Rock, C. O.; Blainey, P. C.; Bloom, S. M.; Kwon, D. S.\nScore: 20.5, Published: 2023-12-30 DOI: 10.1101/2023.12.30.573720\nBacterial vaginosis (BV), a common syndrome characterized by Lactobacillus-deficient vaginal microbiota, is associated with adverse health outcomes. BV often recurs after standard antibiotic therapy in part because antibiotics promote microbiota dominance by Lactobacillus iners instead of Lactobacillus crispatus, which has more beneficial health associations. Strategies to promote L. crispatus and inhibit L. iners are thus needed. We show that oleic acid (OA) and similar long-chain fatty acids simultaneously inhibit L. iners and enhance L. crispatus growth. These phenotypes require OA-inducible genes conserved in L. crispatus and related species, including an oleate hydratase (ohyA) and putative fatty acid efflux pump (farE). FarE mediates OA resistance, while OhyA is robustly active in the human vaginal microbiota and sequesters OA in a derivative form that only ohyA-harboring organisms can exploit. Finally, OA promotes L. crispatus dominance more effectively than antibiotics in an in vitro model of BV, suggesting a novel approach for treatment.\nVirion morphology and on-virus spike protein structures of diverse SARS-CoV-2 variants\nAuthors: Ke, Z.; Peacock, T. P.; Brown, J. C.; Sheppard, C. M.; Croll, T. I.; Kotecha, A.; Goldhill, D. H.; Barclay, W. S.; Briggs, J. A.\nScore: 28.8, Published: 2023-12-22 DOI: 10.1101/2023.12.21.572824\nThe evolution of SARS-CoV-2 variants with increased fitness has been accompanied by structural changes in the spike (S) proteins that are the major target for the adaptive immune response. Single-particle cryo-EM analysis of soluble S from SARS-CoV-2 variants has revealed this structural adaptation at high-resolution. The analysis of S trimers in situ on intact virions has the potential to provide more functionally relevant insights into S structure and virion morphology. Here, we characterized B.1, Alpha, Beta, Gamma, Delta, Kappa, and Mu variants by cryo-electron microscopy and tomography, assessing S cleavage, virion morphology, S incorporation, \"in-situ\" high-resolution S structures and the range of S conformational states. We found no evidence for adaptive changes in virion morphology, but describe multiple different positions in the S protein where amino acid changes alter local protein structure. Considered together, our data is consistent with a model where amino acid changes at multiple positions from the top to the base of the spike cause structural changes that can modulate the conformational dynamics of S.\nSequence-based modelling of bacterial genomes enables accurate antibiotic resistance prediction\nAuthors: Wiatrak, M.; Weimann, A.; Dinan, A. M.; Brbic, M.; Floto, R. A.\nScore: 15.4, Published: 2024-01-03 DOI: 10.1101/2024.01.03.574022\nAbstractRapid detection of antibiotic-resistant bacteria and understanding the mecha- nisms underlying antimicrobial resistance (AMR) are major unsolved problems that pose significant threats to global public health. However, existing methods for predicting antibiotic resistance from genomic sequence data have had lim- ited success due to their inability to model epistatic effects and generalize to novel variants. Here, we present GeneBac, a deep learning method for predicting antibiotic resistance from DNA sequence through the integration of interactions between genes. We apply GeneBac to two distinct bacterial species and show that it can successfully predict the minimum inhibitory concentration (MIC) of multiple antibiotics. We use the WHO Mycobacterium tuberculosis mutation cat- alogue to demonstrate that GeneBac accurately predicts the effects of different variants, including novel variants that have not been observed during training. GeneBac is a modular framework which can be applied to a number of tasks including gene expression prediction, resistant gene identification and strain clus- tering. We leverage this modularity to transfer learn from the transcriptomic data to improve performance on the MIC prediction task.\nInterspecies secreted surfactants induce emergent motility in Pseudomonas aeruginosa\nAuthors: Warrell, D. L.; Zarrella, T. M.; Machalek, C.; Khare, A.\nScore: 12.0, Published: 2024-01-04 DOI: 10.1101/2024.01.03.573969\nIn most natural environments, bacteria live in polymicrobial communities where secreted molecules from neighboring species alter bacterial behaviors including motility, but such interactions are understudied. Pseudomonas aeruginosa is a motile opportunistic pathogen that exists in diverse multispecies environments such as the soil and is frequently found in human wound and respiratory tract co-infections with other bacteria including Staphylococcus aureus. Here we show that P. aeruginosa can co-opt secreted molecules from other species for flagellar-based emergent motility. We found that in the presence of exoproducts from S. aureus and other bacteria, P. aeruginosa switched from swarming to an alternative form of motility where it slid and spread on semi-solid surfaces, and it showed similar behavior on hard agar where it was otherwise unable to move. Surfactants from these species were required for the P. aeruginosa emergent motility, and it was also induced by the addition of numerous exogenous biological and synthetic surfactants. Mutant analysis indicated that this motility was similar to a previously described mucin-based P. aeruginosa motility, surfing, albeit with divergent regulation. Thus, our study establishes a major P. aeruginosa surfing-like emergent motility where secreted surfactants from the host as well as neighboring bacterial and interkingdom species serve as public goods, allowing P. aeruginosa to access different environmental niches.\nCRISPR-repressed toxin-antitoxin provides population-level immunity against diverse anti-CRISPR elements\nAuthors: Li, M.; shu, x.; Wang, R.; Li, Z.; Xue, Q.; Liu, C.; Cheng, F.; Zhao, H.; Wang, J.; Liu, J.; Hu, C.; Li, J.; Ouyang, S.\nScore: 11.3, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574328\nProkaryotic CRISPR-Cas systems are highly vulnerable to phage-encoded anti-CRISPR (Acr) factors. How CRISPR-Cas systems protect themselves remains unclear. Here, we uncovered a broad-spectrum anti-anti-CRISPR strategy involving a phage-derived toxic protein. Transcription of this toxin is normally reppressed by the CRISPR-Cas effector, but is activated to halt cell division when the effector is inhibited by any anti-CRISPR proteins or RNAs. We showed that this abortive infection-like effect efficiently expels Acr elements from bacterial population. Furthermore, we exploited this anti-anti-CRISPR mechanism to develop a screening method for specific Acr candidates for a CRISPR-Cas system, and successfully identified two distinct Acr proteins that enhance the binding of CRISPR effector to non-target DNA. Our data highlight the broad-spectrum role of CRISPR-repressed toxins in counteracting various types of Acr factors, which illuminates that the regulatory function of CRISPR-Cas confers host cells herd immunity against Acr-encoding genetic invaders, no matter they are CRISPR-targeted or not.\nPrecision Genome Editing Unveils a Breakthrough in Reversing Antibiotic Resistance: CRISPR/Cas9 Targeting of Multi-Drug Resistance Genes in Methicillin-Resistant Staphylococcus aureus\nAuthors: Ates, A.; Tastan, C.; Ermertcan, S.\nScore: 11.1, Published: 2024-01-01 DOI: 10.1101/2023.12.31.573511\nAntibiotic resistance poses a global health crisis, limiting the efficacy of available therapeutic agents and hindering the development of new antibiotics. The pharmaceutical industrys waning interest in antibiotic production further exacerbates this challenge. Addressing antibiotic resistance demands innovative solutions. Here, we explore the application of CRISPR-Cas-based antimicrobials as a pioneering approach to combat multidrug resistance. Our study focuses on the methicillin-resistant clinical Staphylococcus aureus (MRSA), a significant clinical threat. Utilizing CRISPR/Cas9 technology, we aimed to concurrently target methicillin (mecA), gentamicin (aacA), and ciprofloxacin (grlA, grlB) resistance genes, thereby altering the resistance profile and enhancing sensitivity to antibiotics. We engineered CRISPR plasmids containing sgRNAs specific to the target regions, which were subsequently electroporated into MRSA strains. Real-time Polymerase Chain Reaction (RT-PCR) assessed changes in resistance gene expression, while disk diffusion and broth microdilution methods determined alterations in resistance status. Western blotting analyzed changes in PBP2a expression, and Sanger sequence analysis confirmed mutations in target regions. Results revealed a statistically significant 1.5-fold decrease in mecA gene expression, 5.5-fold decrease in grlA gene, 6-fold decrease in grlB gene, and 4-fold decrease in aacA gene compared to the wild type strain, as determined by RT-PCR. Antibiotic susceptibility tests demonstrated the suppression of mecA, grlA, grlB, and aacA genes, resulting in the reversal of resistance to beta-lactam, quinolone, and aminoglycoside antibiotics. Western blot analysis showed 70% decrease in PBP2a expression, indicating the breakdown of beta-lactam resistance. Sanger sequence analysis confirmed point mutations in grlB and aacA genes, along with three-base mutations in grlA and mecA genes. Our findings underscore the potential of CRISPR/Cas9 technology to mitigate antibiotic resistance, offering a transformative strategy to restore the efficacy of existing antibiotics in the face of multidrug-resistant pathogens.\n",
  "wordCount" : "2121",
  "inLanguage": "en",
  "datePublished": "2024-01-07T10:38:59Z",
  "dateModified": "2024-01-07T10:38:59Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 7, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.574150">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.574150" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.574150">
        <p class="paperTitle">AMAPEC: accurate antimicrobial activity prediction for fungal effector proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.574150" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.574150" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mesny, F.; Thomma, B. P.</p>
        <p class="info">Score: 46.6, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.574150' target='https://doi.org/10.1101/2024.01.04.574150'> 10.1101/2024.01.04.574150</a></p>
        <p class="abstract">Fungi typically occur in environments where numerous and diverse other microbes occur as well, often resulting in fierce competition for nutrients and habitat. To support fungal fitness in these environments, they evolved various mechanisms that mediate direct antagonism towards niche competitors. Among these, the secretion of proteins with antimicrobial activities has been reported in fungi with diverse lifestyles. Recently, several plant-associated fungi were shown to rely on the secretion of antimicrobial effector proteins to antagonize certain members of plant hosts microbiota and to successfully colonize plant tissues. Some of these effectors do not share homology with known antimicrobials and represent novel antibiotics. Accordingly, the occurrence and conservation of proteinaceous antimicrobials throughout the fungal tree of life remains enigmatic. Here we present a computational approach to annotate candidate antimicrobial effectors in fungal secretomes based on protein physicochemical properties. After curating a set of proteins that were experimentally verified to display antimicrobial activity and a set of proteins that lack such activity, we trained a machine learning classifier on properties of protein sequences and predicted structures. This predictor performs particularly well on fungal proteins (R2=0.89) according to our validations and is delivered as a software package named AMAPEC, dedicated to antimicrobial activity prediction for effector candidates. We subsequently used this novel software to predict antimicrobial effector catalogs in three phylogenetically distant fungi with distinct lifestyles, revealing relatively large catalogs of candidate antimicrobials for each of the three fungi, and suggesting a broad occurrence of such proteins throughout the fungal kingdom. Thus, AMAPEC is a unique method to uncover antimicrobials in fungal secretomes that are often sparsely functionally annotated, and may assist biological interpretations during omic analyses. It is freely available at https://github.com/fantin-mesny/amapec.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574008">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574008" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574008">
        <p class="paperTitle">Lethal Infection of Human ACE2-Transgenic Mice Caused by SARS-CoV-2-related Pangolin Coronavirus GX_P2V(short_3UTR)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574008" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574008" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.</p>
        <p class="info">Score: 43.6, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574008' target='https://doi.org/10.1101/2024.01.03.574008'> 10.1101/2024.01.03.574008</a></p>
        <p class="abstract">SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) can cause 100% mortality in human ACE2-transgenic mice, potentially attributable to late-stage brain infection. This underscores a spillover risk of GX_P2V into humans and provides a unique model for understanding the pathogenic mechanisms of SARS-CoV-2-related viruses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.573942">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.573942" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.573942">
        <p class="paperTitle">TIR domains produce histidine-ADPR conjugates as immune signaling molecules in bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.573942" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.573942" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sabonis, D.; Avraham, C.; Lu, A.; Herbst, E.; Silanskas, A.; Leavitt, A.; Yirmiya, E.; Zaremba, M.; Amitai, G.; Kranzusch, P. J.; Sorek, R.; Tamulaitiene, G.</p>
        <p class="info">Score: 35.9, Published: 2024-01-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.573942' target='https://doi.org/10.1101/2024.01.03.573942'> 10.1101/2024.01.03.573942</a></p>
        <p class="abstract">TIR domains are central components of pattern recognition immune proteins across all domains of life. In both bacteria and plants, TIR-domain proteins were shown to recognize pathogen invasion and then produce immune signaling molecules exclusively comprising nucleotide moieties. Here we show that the TIR domain protein of the type II Thoeris defense system in bacteria produces a unique signaling molecule comprising the amino acid histidine conjugated to ADP-ribose (His-ADPR). His-ADPR is generated in response to phage infection and activates the cognate Thoeris effector by binding a Macro domain located at the C-terminus of the effector protein. By determining the crystal structure of a ligand-bound Macro domain, we describe the structural basis for His-ADPR recognition. Our analyses furthermore reveal a family of phage proteins that bind and sequester His-ADPR signaling molecules, allowing phages to evade TIR- mediated immunity. These data demonstrate diversity in bacterial TIR signaling and reveal a new class of TIR-derived immune signaling molecules combining nucleotide and amino acid moieties.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.570782">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.570782" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.570782">
        <p class="paperTitle">Virological characteristics of the SARS-CoV-2 JN.1 variant</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.570782" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.570782" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium,  ; Zahradnik, J.; Ito, J.; Sato, K.</p>
        <p class="info">Score: 774.1, Published: 2023-12-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.570782' target='https://doi.org/10.1101/2023.12.08.570782'> 10.1101/2023.12.08.570782</a></p>
        <p class="abstract">The SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG.5.1 and HK.3. Comparing to XBB and BA.2, BA.2.86 carries more than 30 mutations in the spike (S) protein, indicating a high potential for immune evasion. BA.2.86 has evolved and its descendant, JN.1 (BA.2.86.1.1), emerged in late 2023. JN.1 harbors S:L455S and three mutations in non-S proteins. S:L455S is a hallmark mutation of JN.1: we have recently shown that HK.3 and other &#34;FLip&#34; variants carry S:L455F, which contributes to increased transmissibility and immune escape ability compared to the parental EG.5.1 variant. Here, we investigated the virological properties of JN.1.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.30.573720">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.30.573720" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.30.573720">
        <p class="paperTitle">Vaginal Lactobacillus fatty acid response mechanisms reveal a novel strategy for bacterial vaginosis treatment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.30.573720" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.30.573720" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhu, M.; Frank, M. W.; Radka, C. D.; Jeanfavre, S.; Tse, M. W.; Avila Pacheco, J.; Pierce, K.; Deik, A.; Xhu, J.; Hussain, S.; Hussain, F. A.; Xulu, N.; Khan, N.; Pillay, V.; Dong, K. L.; Ndung&#39;u, T.; Clish, C. B.; Rock, C. O.; Blainey, P. C.; Bloom, S. M.; Kwon, D. S.</p>
        <p class="info">Score: 20.5, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.30.573720' target='https://doi.org/10.1101/2023.12.30.573720'> 10.1101/2023.12.30.573720</a></p>
        <p class="abstract">Bacterial vaginosis (BV), a common syndrome characterized by Lactobacillus-deficient vaginal microbiota, is associated with adverse health outcomes. BV often recurs after standard antibiotic therapy in part because antibiotics promote microbiota dominance by Lactobacillus iners instead of Lactobacillus crispatus, which has more beneficial health associations. Strategies to promote L. crispatus and inhibit L. iners are thus needed. We show that oleic acid (OA) and similar long-chain fatty acids simultaneously inhibit L. iners and enhance L. crispatus growth. These phenotypes require OA-inducible genes conserved in L. crispatus and related species, including an oleate hydratase (ohyA) and putative fatty acid efflux pump (farE). FarE mediates OA resistance, while OhyA is robustly active in the human vaginal microbiota and sequesters OA in a derivative form that only ohyA-harboring organisms can exploit. Finally, OA promotes L. crispatus dominance more effectively than antibiotics in an in vitro model of BV, suggesting a novel approach for treatment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572824">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572824" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572824">
        <p class="paperTitle">Virion morphology and on-virus spike protein structures of diverse SARS-CoV-2 variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572824" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572824" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ke, Z.; Peacock, T. P.; Brown, J. C.; Sheppard, C. M.; Croll, T. I.; Kotecha, A.; Goldhill, D. H.; Barclay, W. S.; Briggs, J. A.</p>
        <p class="info">Score: 28.8, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572824' target='https://doi.org/10.1101/2023.12.21.572824'> 10.1101/2023.12.21.572824</a></p>
        <p class="abstract">The evolution of SARS-CoV-2 variants with increased fitness has been accompanied by structural changes in the spike (S) proteins that are the major target for the adaptive immune response. Single-particle cryo-EM analysis of soluble S from SARS-CoV-2 variants has revealed this structural adaptation at high-resolution. The analysis of S trimers in situ on intact virions has the potential to provide more functionally relevant insights into S structure and virion morphology. Here, we characterized B.1, Alpha, Beta, Gamma, Delta, Kappa, and Mu variants by cryo-electron microscopy and tomography, assessing S cleavage, virion morphology, S incorporation, &#34;in-situ&#34; high-resolution S structures and the range of S conformational states. We found no evidence for adaptive changes in virion morphology, but describe multiple different positions in the S protein where amino acid changes alter local protein structure. Considered together, our data is consistent with a model where amino acid changes at multiple positions from the top to the base of the spike cause structural changes that can modulate the conformational dynamics of S.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574022">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574022" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574022">
        <p class="paperTitle">Sequence-based modelling of bacterial genomes enables accurate antibiotic resistance prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574022" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574022" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wiatrak, M.; Weimann, A.; Dinan, A. M.; Brbic, M.; Floto, R. A.</p>
        <p class="info">Score: 15.4, Published: 2024-01-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574022' target='https://doi.org/10.1101/2024.01.03.574022'> 10.1101/2024.01.03.574022</a></p>
        <p class="abstract">AbstractRapid detection of antibiotic-resistant bacteria and understanding the mecha- nisms underlying antimicrobial resistance (AMR) are major unsolved problems that pose significant threats to global public health. However, existing methods for predicting antibiotic resistance from genomic sequence data have had lim- ited success due to their inability to model epistatic effects and generalize to novel variants. Here, we present GeneBac, a deep learning method for predicting antibiotic resistance from DNA sequence through the integration of interactions between genes. We apply GeneBac to two distinct bacterial species and show that it can successfully predict the minimum inhibitory concentration (MIC) of multiple antibiotics. We use the WHO Mycobacterium tuberculosis mutation cat- alogue to demonstrate that GeneBac accurately predicts the effects of different variants, including novel variants that have not been observed during training. GeneBac is a modular framework which can be applied to a number of tasks including gene expression prediction, resistant gene identification and strain clus- tering. We leverage this modularity to transfer learn from the transcriptomic data to improve performance on the MIC prediction task.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.573969">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.573969" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.573969">
        <p class="paperTitle">Interspecies secreted surfactants induce emergent motility in Pseudomonas aeruginosa</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.573969" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.573969" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Warrell, D. L.; Zarrella, T. M.; Machalek, C.; Khare, A.</p>
        <p class="info">Score: 12.0, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.573969' target='https://doi.org/10.1101/2024.01.03.573969'> 10.1101/2024.01.03.573969</a></p>
        <p class="abstract">In most natural environments, bacteria live in polymicrobial communities where secreted molecules from neighboring species alter bacterial behaviors including motility, but such interactions are understudied. Pseudomonas aeruginosa is a motile opportunistic pathogen that exists in diverse multispecies environments such as the soil and is frequently found in human wound and respiratory tract co-infections with other bacteria including Staphylococcus aureus. Here we show that P. aeruginosa can co-opt secreted molecules from other species for flagellar-based emergent motility. We found that in the presence of exoproducts from S. aureus and other bacteria, P. aeruginosa switched from swarming to an alternative form of motility where it slid and spread on semi-solid surfaces, and it showed similar behavior on hard agar where it was otherwise unable to move. Surfactants from these species were required for the P. aeruginosa emergent motility, and it was also induced by the addition of numerous exogenous biological and synthetic surfactants. Mutant analysis indicated that this motility was similar to a previously described mucin-based P. aeruginosa motility,  surfing, albeit with divergent regulation. Thus, our study establishes a major P. aeruginosa surfing-like emergent motility where secreted surfactants from the host as well as neighboring bacterial and interkingdom species serve as public goods, allowing P. aeruginosa to access different environmental niches.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574328">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574328" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574328">
        <p class="paperTitle">CRISPR-repressed toxin-antitoxin provides population-level immunity against diverse anti-CRISPR elements</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574328" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574328" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, M.; shu, x.; Wang, R.; Li, Z.; Xue, Q.; Liu, C.; Cheng, F.; Zhao, H.; Wang, J.; Liu, J.; Hu, C.; Li, J.; Ouyang, S.</p>
        <p class="info">Score: 11.3, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574328' target='https://doi.org/10.1101/2024.01.05.574328'> 10.1101/2024.01.05.574328</a></p>
        <p class="abstract">Prokaryotic CRISPR-Cas systems are highly vulnerable to phage-encoded anti-CRISPR (Acr) factors. How CRISPR-Cas systems protect themselves remains unclear. Here, we uncovered a broad-spectrum anti-anti-CRISPR strategy involving a phage-derived toxic protein. Transcription of this toxin is normally reppressed by the CRISPR-Cas effector, but is activated to halt cell division when the effector is inhibited by any anti-CRISPR proteins or RNAs. We showed that this abortive infection-like effect efficiently expels Acr elements from bacterial population. Furthermore, we exploited this anti-anti-CRISPR mechanism to develop a screening method for specific Acr candidates for a CRISPR-Cas system, and successfully identified two distinct Acr proteins that enhance the binding of CRISPR effector to non-target DNA. Our data highlight the broad-spectrum role of CRISPR-repressed toxins in counteracting various types of Acr factors, which illuminates that the regulatory function of CRISPR-Cas confers host cells herd immunity against Acr-encoding genetic invaders, no matter they are CRISPR-targeted or not.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.31.573511">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.31.573511" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.31.573511">
        <p class="paperTitle">Precision Genome Editing Unveils a Breakthrough in Reversing Antibiotic Resistance: CRISPR/Cas9 Targeting of Multi-Drug Resistance Genes in Methicillin-Resistant Staphylococcus aureus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.31.573511" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.31.573511" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ates, A.; Tastan, C.; Ermertcan, S.</p>
        <p class="info">Score: 11.1, Published: 2024-01-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.31.573511' target='https://doi.org/10.1101/2023.12.31.573511'> 10.1101/2023.12.31.573511</a></p>
        <p class="abstract">Antibiotic resistance poses a global health crisis, limiting the efficacy of available therapeutic agents and hindering the development of new antibiotics. The pharmaceutical industrys waning interest in antibiotic production further exacerbates this challenge. Addressing antibiotic resistance demands innovative solutions. Here, we explore the application of CRISPR-Cas-based antimicrobials as a pioneering approach to combat multidrug resistance. Our study focuses on the methicillin-resistant clinical Staphylococcus aureus (MRSA), a significant clinical threat. Utilizing CRISPR/Cas9 technology, we aimed to concurrently target methicillin (mecA), gentamicin (aacA), and ciprofloxacin (grlA, grlB) resistance genes, thereby altering the resistance profile and enhancing sensitivity to antibiotics. We engineered CRISPR plasmids containing sgRNAs specific to the target regions, which were subsequently electroporated into MRSA strains. Real-time Polymerase Chain Reaction (RT-PCR) assessed changes in resistance gene expression, while disk diffusion and broth microdilution methods determined alterations in resistance status. Western blotting analyzed changes in PBP2a expression, and Sanger sequence analysis confirmed mutations in target regions. Results revealed a statistically significant 1.5-fold decrease in mecA gene expression, 5.5-fold decrease in grlA gene, 6-fold decrease in grlB gene, and 4-fold decrease in aacA gene compared to the wild type strain, as determined by RT-PCR. Antibiotic susceptibility tests demonstrated the suppression of mecA, grlA, grlB, and aacA genes, resulting in the reversal of resistance to beta-lactam, quinolone, and aminoglycoside antibiotics. Western blot analysis showed 70% decrease in PBP2a expression, indicating the breakdown of beta-lactam resistance. Sanger sequence analysis confirmed point mutations in grlB and aacA genes, along with three-base mutations in grlA and mecA genes. Our findings underscore the potential of CRISPR/Cas9 technology to mitigate antibiotic resistance, offering a transformative strategy to restore the efficacy of existing antibiotics in the face of multidrug-resistant pathogens.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
