<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="SARS-CoV-2 infects neurons, astrocytes, choroid plexus epithelial cells and pericytes of the human central nervous system
Authors: Haverty, R.; McCormack, J.; Evans, C.; Purves, K.; O&#39;Reilly, S.; Gautier, V. W.; Rochfort, K.; Fabre, A.; Fletcher, N.
Score: 143.6, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568132
SARS-CoV-2, the coronavirus responsible for the COVID-19 pandemic, is associated with a range of neurological manifestations including haemorrhage, thrombosis and ischaemic necrosis and encephalitits. However, the mechanism by which this occurs is unclear.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="SARS-CoV-2 infects neurons, astrocytes, choroid plexus epithelial cells and pericytes of the human central nervous system
Authors: Haverty, R.; McCormack, J.; Evans, C.; Purves, K.; O&#39;Reilly, S.; Gautier, V. W.; Rochfort, K.; Fabre, A.; Fletcher, N.
Score: 143.6, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568132
SARS-CoV-2, the coronavirus responsible for the COVID-19 pandemic, is associated with a range of neurological manifestations including haemorrhage, thrombosis and ischaemic necrosis and encephalitits. However, the mechanism by which this occurs is unclear." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-29T10:40:14+00:00" />
<meta property="article:modified_time" content="2023-11-29T10:40:14+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="SARS-CoV-2 infects neurons, astrocytes, choroid plexus epithelial cells and pericytes of the human central nervous system
Authors: Haverty, R.; McCormack, J.; Evans, C.; Purves, K.; O&#39;Reilly, S.; Gautier, V. W.; Rochfort, K.; Fabre, A.; Fletcher, N.
Score: 143.6, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568132
SARS-CoV-2, the coronavirus responsible for the COVID-19 pandemic, is associated with a range of neurological manifestations including haemorrhage, thrombosis and ischaemic necrosis and encephalitits. However, the mechanism by which this occurs is unclear."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "SARS-CoV-2 infects neurons, astrocytes, choroid plexus epithelial cells and pericytes of the human central nervous system\nAuthors: Haverty, R.; McCormack, J.; Evans, C.; Purves, K.; O\u0026#39;Reilly, S.; Gautier, V. W.; Rochfort, K.; Fabre, A.; Fletcher, N.\nScore: 143.6, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568132\nSARS-CoV-2, the coronavirus responsible for the COVID-19 pandemic, is associated with a range of neurological manifestations including haemorrhage, thrombosis and ischaemic necrosis and encephalitits. However, the mechanism by which this occurs is unclear.",
  "keywords": [
    
  ],
  "articleBody": " SARS-CoV-2 infects neurons, astrocytes, choroid plexus epithelial cells and pericytes of the human central nervous system\nAuthors: Haverty, R.; McCormack, J.; Evans, C.; Purves, K.; O'Reilly, S.; Gautier, V. W.; Rochfort, K.; Fabre, A.; Fletcher, N.\nScore: 143.6, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568132\nSARS-CoV-2, the coronavirus responsible for the COVID-19 pandemic, is associated with a range of neurological manifestations including haemorrhage, thrombosis and ischaemic necrosis and encephalitits. However, the mechanism by which this occurs is unclear. Neurological disease associated with SARS-CoV-2 infection has been proposed to occur following direct infection of the central nervous system and/or indirect sequelae as a result of peripheral inflammation. We profiled ACE2 and TMPRSS2 in brain tissue from five healthy human donors, and observed expression of these proteins in astrocytes, neurons and choroid plexus epithelium within frontal cortex and medulla. Primary human astrocytes, neurons and choroid plexus epithelial cells supported productive SARS-CoV-2 infection in an ACE2- dependent manner. Infected cells supported the full viral lifecycle, releasing infectious virus particles. In contrast, primary brain microvascular endothelial cells, pericytes and microglia were refractory to SARS-CoV-2 infection. These data support a model whereby SARS-CoV-2 is neurotropic, and this may in part explain the neurological sequelae of infection. ImportanceA subset of patients with COVID-19 develop neurological symptoms, but the underlying cause is poorly understood. We observed that cells within normal human brain express the SARS-CoV-2 entry factors ACE-2 and TMPRRS2, with expression mainly observed within astrocytes, neurons and choroid plexus epithelium. Primary human astrocytes, neurons and choroid plexus epithelial cells cultured in vitro supported the full SARS-CoV-2 life cycle with a range of SARS-CoV-2 variants. This study demonstrates that cells of the human central nervous system express SARS-CoV-2 entry factors in vivo and support viral infection in vitro, thus supporting a model where neurological symptoms seen in some COVID-19 patients may be as a result of direct viral infection of the central nervous system. Furthermore, these data highlight the importance of investigating the ability of therapeutics to clear virus from this potential reservoir of infection.\nHuman microbiota is a reservoir of SARS-CoV-2 advantageous mutations\nAuthors: Cao, B.; Wang, X.; Yin, W.; Gao, Z.; Xia, B.\nScore: 152.7, Published: 2023-11-17 DOI: 10.1101/2023.11.16.567485\nSARS-CoV-2 mutations are rapidly emerging, in particular advantageous mutations in the spike (S) protein, which either increase transmissibility or lead to immune escape, are posing a major challenge to pandemic prevention and treatment. However, how the virus acquires a high number of advantageous mutations in a short time remains a mystery. Here, we show that the human microbiota may contribute to mutations in variants of concern (VOCs). We identified a mutation and adjacent 6 amino acids (aa) in a viral mutation fragment (VMF) and searched for homologous fragments (HFs) in the National Center for Biotechnology Information (NCBI) database. Among the approximate 8000 HFs obtained, 61 mutations in S and other outer membrane proteins were found in bacteria, accounting for 62% of all mutation sources, which is a 12-fold higher than the natural variable proportion. Approximately 70% of these bacterial species belong to the human microbiota, are primarily distributed in the gut or lung and exhibit a composition pattern similar to that of COVID-19 patients. Importantly, SARS- CoV-2 RNA-dependent RNA polymerase (RdRp) replicates corresponding bacterial mRNAs harboring mutations, producing chimeric RNAs. Collectively, SARS-CoV-2 may acquire mutations from the human microbiota, resulting in alterations in the binding sites or antigenic determinants of the original virus. Our study sheds light on the evolving mutational mechanisms of SARS-CoV-2.\nPredicting phage-bacteria interactions at the strain level from genomes\nAuthors: Gaborieau, B.; Vaysset, H.; Tesson, F.; Charachon, I.; Dib, N.; Bernier, J.; Dequidt, T.; Georjon, H.; Clermont, O.; Hersen, P.; Debarbieux, L.; Ricard, J.-D.; Denamur, E.; Bernheim, A.\nScore: 64.1, Published: 2023-11-22 DOI: 10.1101/2023.11.22.567924\nPredicting how phages can selectively infect specific bacterial strains holds promise for developing novel approaches to combat bacterial infections and better understanding microbial ecology. Experimental studies on phage-bacteria interactions have been mostly focusing on a few model organisms to understand the molecular mechanisms which makes a particular bacterial strain susceptible to a given phage. However, both bacteria and phages are extremely diverse in natural contexts. How well the concepts learned from well-established experimental models generalize to a broad diversity of what is encountered in the wild is currently unknown. Recent advances in genomics allow to identify traits involved in phage-host specificity, implying that these traits could be utilized for the prediction of such interactions. Here, we show that we could predict outcomes of most phage-bacteria interactions at the strain level in Escherichia natural isolates based solely on genomic data. First, we established a dataset of experimental outcomes of phage-bacteria interactions of 403 natural, phylogenetically diverse, Escherichia strains to 96 bacteriophages matched with fully sequenced and genomically characterized strains and phages. To predict these interactions, we set out to define genomic traits with predictive power. We show that most interactions in our dataset can be explained by adsorption factors as opposed to antiphage systems which play a marginal role. We then trained predictive algorithms to pinpoint which interactions could be accurately predicted and where future research should focus on. Finally, we show the application of such predictions by establishing a pipeline to recommend tailored phage cocktails to target pathogenic strains from their genomes only and show higher efficiency of tailored cocktails on a collection of 100 pathogenic E. coli isolates. Altogether, this work provides quantitative insights into understanding phage-host specificity at the strain level and paves the way for the use of predictive algorithms in phage therapy.\nDistinct evolution of SARS-CoV-2 Omicron XBB and BA.2.86 lineages combining increased fitness and antibody evasion\nAuthors: Planas, D.; Staropoli, I.; Michel, V.; Lemoine, F.; Donati, F.; Prot, M.; Porrot, F.; Benhassine, F. G.; Jeyarajah, B.; Brisebarre, A.; Dehan, O.; Bolland, W.; Hubert, M.; Buchreiser, J.; Vanhoucke, T.; Rosenbaum, P.; Veyer, D.; Pere, H.; Lina, B.; Assant, S. T.; Hocqueloux, L.; Prazuck, T.; Loriere, E. S.; Schwartz, O.\nScore: 47.9, Published: 2023-11-21 DOI: 10.1101/2023.11.20.567873\nThe unceasing SARS-CoV-2 circulation in an immune population led to the continuous emergence of new viral sublineages. Here, we isolated and characterized XBF, XBB.1, XBB.1.5, XBB.1.9.1, XBB.1.16.1, EG.5.1.1, EG.5.1.3 and BA.2.86.1 variants, that represented \u003e80% of circulating strains as of November 2023. These XBB subvariants carry few recurrent mutations in the spike, whereas BA.2.86.1 harbors \u003e30 additional mutations. We compared their fitness in culture, sensitivity to antivirals and sera from vaccinees. These variants replicated in IGROV-1 and no longer in VeroE6 cells. They were not markedly fusogenic. BA.2.86.1 displayed the strongest binding to ACE2. They potently infected nasal epithelial cells, with EG.5.1.3 exhibiting the highest fitness. Nirmatrelvir, Remdesivir and Molnupiravir remained active, whereas Sotrovimab lost efficacy against BA.2.86.1. Neutralizing antibody (NAb) responses from vaccinees and BA.1/BA.2-infected individuals were 7 to 21-fold lower compared to BA.1, without major differences between variants. A breakthrough XBB infection enhanced NAb responses, particularly against XBB variants. Thus, while distinct, the evolution trajectory of these variants combines increased fitness and antibody evasion.\nThermal and pigment characterization of environmental fungi in the urban heat island of Baltimore City\nAuthors: Smith, D. F. Q.; Faiez, T. S.; Bencomo, A.; Casadevall, A.\nScore: 40.0, Published: 2023-11-10 DOI: 10.1101/2023.11.10.566554\nOne of the major barriers of fungal infections of mammals is the inability to grow and/or survive at mammalian body temperature, typically around 37{degrees}C. This has provided mammals an advantage over fungi. However, environmental fungi may soon adapt to persist at higher temperatures, consistent with mammalian body temperature, due to thermal selection pressures imposed by climate change, global warming, and increased frequency of extreme heat events. Consequently, there is a need for more updated information about the thermal tolerance range of fungi near humans, such as in urban areas. The heat island effect suggests that cities are up to 8{degrees}C warmer than their suburban counterparts because of increased heat production, asphalt coatings and reduced greenspace among other factors, and it is more common in lower income and marginalized urban communities. Thus, urban centers are at increased risk for the emergence of heat tolerant fungi. In this study, we developed a methodology to collect and archive fungal isolates from sidewalk and soil samples in both warmer and cooler neighborhoods in Baltimore, Maryland. We demonstrate a novel methodology for fungal sample collection from sidewalks, employing the use of standardized and commercially available taffy. Analysis of fungal isolates collected from warmer neighborhoods revealed greater thermal tolerance and lower pigmentation, suggesting local adaptation to heat. Lower pigmentation in hotter areas is consistent with the notion that fungi use pigmentation to help regulate their temperature. Further, we identified the robust presence of the polyextremotolerant fungus Aureobasidium pullalans from the warmest neighborhood in Baltimore, further showing that the extreme conditions of cities can drive proliferation of extremotolerant fungi. This study develops new techniques for environmental fungal collection and provides insight on the fungal census in an urban setting that can inform future work to study how urban environments may drive stress/thermotolerance in fungi, which could alter fungal interactions with humans and impact human health.\nGiant genes are rare but implicated in cell wall degradation by predatory bacteria\nAuthors: West-Roberts, J. A.; Valentin Alvarado, L. E.; Mullen, S.; Sachdeva, R.; Smith, J.; Hug, L. A.; Gregoire, D.; Liu, W.; Lin, T.-Y.; Husain, G.; Amano, Y.; Ly, L.; Banfield, J.\nScore: 32.3, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568195\nAcross the tree of life, gene lengths vary, but most are no more than a few thousand base pairs in length. The largest protein often reported is the [~]40,000 aa eukaryotic Titin. Even larger proteins may occur in the rapidly expanding set of metagenome-derived sequences, but their existence may be obscured by assembly fragmentation. Here, we leverage genome curation to complete metagenome-derived sequences that encode predicted proteins of up to 85,804 aa. Overall, the findings illuminate a huge knowledge gap related to giant proteins. Although predicted proteins of \u003e30,000 aa occur in bacterial phyla such as Firmicutes and Actinobacteria, they are most common in ca. Omnitrophota, ultra small bacteria that adopt predatory lifestyles. All full length giant genes encode numerous transmembrane regions and most encode divergent secA DEAD helicase domains. In silico structural prediction of protein subregions was required to identify domains in unannotated protein segments, and revealed putative domains implicated in attachment and carbohydrate degradation. Many giant genes in new complete and near-complete Omnitrophota genomes occur in close proximity to genes homologous to type II secretion systems as well as carbohydrate import systems. This, in combination with the domain content, suggests that many bacterial giant proteins enable prey adhesion and cell wall digestion during bacterial predation.\nDetection and spread of high pathogenicity avian influenza virus H5N1 in the Antarctic Region\nAuthors: Bennison, A.; Byrne, A. M.; Reid, S. M.; Lynton-Jenkins, J. G.; Mollett, B.; De Sliva, D.; Peers-Dent, J.; Finlayson, K. E.; Hall, R.; Blockley, F.; Blyth, M.; Falchieri, M.; Fowler, Z.; Fitzcharles, E.; Brown, I. H.; James, J.; Banyard, A. C.\nScore: 30.4, Published: 2023-11-24 DOI: 10.1101/2023.11.23.568045\nThe Antarctic is the only major geographical region in which high pathogenicity avian influenza virus (HPAIV) has never previously been detected. The current panzootic of H5N1 HPAIV has decimated wild bird populations across Europe, North America and South America. Here we report on the emergence of clade 2.3.4.4b H5N1 HPAIV in the Antarctic and sub-Antarctic regions of South Georgia and the Falkland Islands respectively. We initially detected H5N1 HPAIV in samples collected from brown skuas at Bird Island, South Georgia on 8th October 2023. Since this detection, increased mortalities were observed in brown skuas, kelp gulls, elephant seals and fur seal at multiple sites across South Georgia. We confirmed H5N1 HPAIV in multiple brown skuas and kelp gulls across four different sampling locations in South Georgia. Simultaneously, we also confirmed H5N1 HPAIV in a southern fulmar in the Falkland Islands. Genetic assessment of the virus indicates spread from South America, likely through movement of migratory birds. Here we describe the emergence, species impact and genetic composition of the virus and propose both introductory routes and potential long-term impact on avian and mammalian species across the Antarctic region.\nComprehensive contact tracing during an outbreak of alpha-variant SARS-CoV-2 in a rural community reveals less viral genomic diversity and higher household secondary attack rates than expected\nAuthors: Sivertsen, A.; Mortensen, N.; Solem, U.; Valen, E.; Bullita, M. F.; Wensaas, K.-A.; Litleskare, S.; Rortveit, G.; Grewal, H. M. S.; Ulvestad, E.\nScore: 43.7, Published: 2023-11-21 DOI: 10.1101/2023.11.17.567570\nSequencing of SARS-CoV-2 genomes throughout the COVID-19 pandemic has generated a wealth of data on viral evolution across populations, but only a few studies have so far explored SARS-CoV-2 evolution across transmission networks of tens to hundreds of persons. Here, we couple data from SARS-CoV-2 sequencing with contact tracing data from an outbreak with a single origin in a rural Norwegian community where samples from all exposed persons were collected prospectively. A total of 134 nasopharyngeal samples were positive by PCR. Among the 121 retrievable genomes, 81 were identical to the genome of the introductor, thus demonstrating that genomics offers limited additional value to manual contact-tracing. In the cases where mutations were discovered, five small genetic clusters were identified. We observed a household secondary attack rate of 67%, with 92% of household members infected among households with secondary transmission, suggesting that SARS-CoV-2 introduction into large families are likely to affect all household members. ImportanceIn outbreak investigations, obtaining a full overview of infected individuals within a population is seldom acheived. We here present an example of just that, where a single introduction of B1.1.7 SARS-CoV-2 within a rural community allowed for tracing of the virus, from an introductor via dissemination through larger gatherings, into households. The outbreak occurred before widespread vaccination, allowing for a \"natural\" outbreak development with community lock-down. We show through sequencing that the virus can infect up to five consecutive persons without gaining mutations, thereby showing that contact tracing seems more important than sequencing for local outbreak investigations. We also show how families with small children are less likely to contain spread to all family members if SARS-CoV-2 enters the household either by a child or a caregiver, as isolation of the primary infected is difficult in such scenarios.\nβ-lactamase expression induces collateral sensitivity in Escherichia coli\nAuthors: Herencias, C.; Alvaro-Llorente, L.; Ramiro-Martinez, P.; Munoz-Cazalla, A.; DeLaFuente, J.; Castillo-Polo, J. A.; Canton, R.; San Millan, A.; Rodriguez-Beltran, J.\nScore: 27.4, Published: 2023-11-22 DOI: 10.1101/2023.11.22.568265\nMajor antibiotic groups are losing effectiveness due to the uncontrollable spread of antimicrobial resistance (AMR) genes. Among these, {beta}-lactam resistance genes -encoding {beta}-lactamases- stand as the most common resistance mechanism in Enterobacterales due to their frequent association with mobile genetic elements. In this context, novel approaches that counter mobile AMR are urgently needed. Collateral sensitivity (CS) occurs when the acquisition of resistance to one antibiotic increases susceptibility to another antibiotic and can be exploited to selectively eliminate AMR. However, most CS networks described so far emerge as a consequence of chromosomal mutations and cannot be leveraged to tackle mobile AMR. Here, we dissected the CS response elicited by the acquisition of a prevalent antibiotic resistance plasmid to reveal that the expression of the {beta}-lactamase blaOXA-48 induces CS to colistin and azithromycin. We next showed that expression of other clinically relevant mobile {beta}-lactamases produces similar CS responses in multiple, phylogenetically unrelated E. coli strains. Finally, by combining experiments with surveillance data comprising thousands of antibiotic susceptibility tests, we showed that {beta}-lactamase-induced CS is pervasive within Enterobacterales. These results highlight that the physiological side-effects of {beta}-lactamases can be leveraged therapeutically, paving the way for the rational design of specific therapies to block mobile AMR or at least counteract their effects.\nAsgard archaea modulate potential methanogenesis substrates in wetland soil\nAuthors: Valentin-Alvarado, L. E.; Appler, K. E.; De Anda, V.; Schoelmerich, M. C.; West-Roberts, J.; Kivenson, V.; Crits-Christoph, A.; Ly, L.; Sachdeva, R.; Savage, D. F.; Baker, B. J.; Banfield, J. F.\nScore: 27.0, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568159\nThe roles of Asgard archaea in eukaryogenesis and marine biogeochemical cycles are well studied, yet their contributions in soil ecosystems are unknown. Of particular interest are Asgard archaeal contributions to methane cycling in wetland soils. To investigate this, we reconstructed two complete genomes for soil-associated Atabeyarchaeia, a new Asgard lineage, and the first complete genome of Freyarchaeia, and defined their metabolism in situ. Metatranscriptomics highlights high expression of [NiFe]-hydrogenases, pyruvate oxidation and carbon fixation via the Wood-Ljungdahl pathway genes. Also highly expressed are genes encoding enzymes for amino acid metabolism, anaerobic aldehyde oxidation, hydrogen peroxide detoxification and glycerol and carbohydrate breakdown to acetate and formate. Overall, soil-associated Asgard archaea are predicted to be non-methanogenic acetogens, likely impacting reservoirs of substrates for methane production in terrestrial ecosystems. One-Sentence SummaryComplete genomes of Asgard archaea, coupled with metatranscriptomic data, indicate roles in production and consumption of carbon compounds that are known to serve as substrates for methane production in wetlands.\n",
  "wordCount" : "2795",
  "inLanguage": "en",
  "datePublished": "2023-11-29T10:40:14Z",
  "dateModified": "2023-11-29T10:40:14Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 29, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568132">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568132" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568132">
        <p class="paperTitle">SARS-CoV-2 infects neurons, astrocytes, choroid plexus epithelial cells and pericytes of the human central nervous system</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568132" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568132" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Haverty, R.; McCormack, J.; Evans, C.; Purves, K.; O&#39;Reilly, S.; Gautier, V. W.; Rochfort, K.; Fabre, A.; Fletcher, N.</p>
        <p class="info">Score: 143.6, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568132' target='https://doi.org/10.1101/2023.11.21.568132'> 10.1101/2023.11.21.568132</a></p>
        <p class="abstract">SARS-CoV-2, the coronavirus responsible for the COVID-19 pandemic, is associated with a range of neurological manifestations including haemorrhage, thrombosis and ischaemic necrosis and encephalitits. However, the mechanism by which this occurs is unclear. Neurological disease associated with SARS-CoV-2 infection has been proposed to occur following direct infection of the central nervous system and/or indirect sequelae as a result of peripheral inflammation. We profiled ACE2 and TMPRSS2 in brain tissue from five healthy human donors, and observed expression of these proteins in astrocytes, neurons and choroid plexus epithelium within frontal cortex and medulla. Primary human astrocytes, neurons and choroid plexus epithelial cells supported productive SARS-CoV-2 infection in an ACE2- dependent manner. Infected cells supported the full viral lifecycle, releasing infectious virus particles. In contrast, primary brain microvascular endothelial cells, pericytes and microglia were refractory to SARS-CoV-2 infection. These data support a model whereby SARS-CoV-2 is neurotropic, and this may in part explain the neurological sequelae of infection.

ImportanceA subset of patients with COVID-19 develop neurological symptoms, but the underlying cause is poorly understood. We observed that cells within normal human brain express the SARS-CoV-2 entry factors ACE-2 and TMPRRS2, with expression mainly observed within astrocytes, neurons and choroid plexus epithelium. Primary human astrocytes, neurons and choroid plexus epithelial cells cultured in vitro supported the full SARS-CoV-2 life cycle with a range of SARS-CoV-2 variants. This study demonstrates that cells of the human central nervous system express SARS-CoV-2 entry factors in vivo and support viral infection in vitro, thus supporting a model where neurological symptoms seen in some COVID-19 patients may be as a result of direct viral infection of the central nervous system. Furthermore, these data highlight the importance of investigating the ability of therapeutics to clear virus from this potential reservoir of infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.567485">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.567485" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.567485">
        <p class="paperTitle">Human microbiota is a reservoir of SARS-CoV-2 advantageous mutations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.567485" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.567485" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cao, B.; Wang, X.; Yin, W.; Gao, Z.; Xia, B.</p>
        <p class="info">Score: 152.7, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.567485' target='https://doi.org/10.1101/2023.11.16.567485'> 10.1101/2023.11.16.567485</a></p>
        <p class="abstract">SARS-CoV-2 mutations are rapidly emerging, in particular advantageous mutations in the spike (S) protein, which either increase transmissibility or lead to immune escape, are posing a major challenge to pandemic prevention and treatment. However, how the virus acquires a high number of advantageous mutations in a short time remains a mystery. Here, we show that the human microbiota may contribute to mutations in variants of concern (VOCs). We identified a mutation and adjacent 6 amino acids (aa) in a viral mutation fragment (VMF) and searched for homologous fragments (HFs) in the National Center for Biotechnology Information (NCBI) database. Among the approximate 8000 HFs obtained, 61 mutations in S and other outer membrane proteins were found in bacteria, accounting for 62% of all mutation sources, which is a 12-fold higher than the natural variable proportion. Approximately 70% of these bacterial species belong to the human microbiota, are primarily distributed in the gut or lung and exhibit a composition pattern similar to that of COVID-19 patients. Importantly, SARS- CoV-2 RNA-dependent RNA polymerase (RdRp) replicates corresponding bacterial mRNAs harboring mutations, producing chimeric RNAs. Collectively, SARS-CoV-2 may acquire mutations from the human microbiota, resulting in alterations in the binding sites or antigenic determinants of the original virus. Our study sheds light on the evolving mutational mechanisms of SARS-CoV-2.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.567924">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.567924" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.567924">
        <p class="paperTitle">Predicting phage-bacteria interactions at the strain level from genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.567924" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.567924" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gaborieau, B.; Vaysset, H.; Tesson, F.; Charachon, I.; Dib, N.; Bernier, J.; Dequidt, T.; Georjon, H.; Clermont, O.; Hersen, P.; Debarbieux, L.; Ricard, J.-D.; Denamur, E.; Bernheim, A.</p>
        <p class="info">Score: 64.1, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.567924' target='https://doi.org/10.1101/2023.11.22.567924'> 10.1101/2023.11.22.567924</a></p>
        <p class="abstract">Predicting how phages can selectively infect specific bacterial strains holds promise for developing novel approaches to combat bacterial infections and better understanding microbial ecology. Experimental studies on phage-bacteria interactions have been mostly focusing on a few model organisms to understand the molecular mechanisms which makes a particular bacterial strain susceptible to a given phage. However, both bacteria and phages are extremely diverse in natural contexts. How well the concepts learned from well-established experimental models generalize to a broad diversity of what is encountered in the wild is currently unknown. Recent advances in genomics allow to identify traits involved in phage-host specificity, implying that these traits could be utilized for the prediction of such interactions. Here, we show that we could predict outcomes of most phage-bacteria interactions at the strain level in Escherichia natural isolates based solely on genomic data. First, we established a dataset of experimental outcomes of phage-bacteria interactions of 403 natural, phylogenetically diverse, Escherichia strains to 96 bacteriophages matched with fully sequenced and genomically characterized strains and phages. To predict these interactions, we set out to define genomic traits with predictive power. We show that most interactions in our dataset can be explained by adsorption factors as opposed to antiphage systems which play a marginal role. We then trained predictive algorithms to pinpoint which interactions could be accurately predicted and where future research should focus on. Finally, we show the application of such predictions by establishing a pipeline to recommend tailored phage cocktails to target pathogenic strains from their genomes only and show higher efficiency of tailored cocktails on a collection of 100 pathogenic E. coli isolates. Altogether, this work provides quantitative insights into understanding phage-host specificity at the strain level and paves the way for the use of predictive algorithms in phage therapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567873">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567873" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567873">
        <p class="paperTitle">Distinct evolution of SARS-CoV-2 Omicron XBB and BA.2.86 lineages combining increased fitness and antibody evasion</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567873" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567873" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Planas, D.; Staropoli, I.; Michel, V.; Lemoine, F.; Donati, F.; Prot, M.; Porrot, F.; Benhassine, F. G.; Jeyarajah, B.; Brisebarre, A.; Dehan, O.; Bolland, W.; Hubert, M.; Buchreiser, J.; Vanhoucke, T.; Rosenbaum, P.; Veyer, D.; Pere, H.; Lina, B.; Assant, S. T.; Hocqueloux, L.; Prazuck, T.; Loriere, E. S.; Schwartz, O.</p>
        <p class="info">Score: 47.9, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567873' target='https://doi.org/10.1101/2023.11.20.567873'> 10.1101/2023.11.20.567873</a></p>
        <p class="abstract">The unceasing SARS-CoV-2 circulation in an immune population led to the continuous emergence of new viral sublineages. Here, we isolated and characterized XBF, XBB.1, XBB.1.5, XBB.1.9.1, XBB.1.16.1, EG.5.1.1, EG.5.1.3 and BA.2.86.1 variants, that represented &gt;80% of circulating strains as of November 2023. These XBB subvariants carry few recurrent mutations in the spike, whereas BA.2.86.1 harbors &gt;30 additional mutations. We compared their fitness in culture, sensitivity to antivirals and sera from vaccinees. These variants replicated in IGROV-1 and no longer in VeroE6 cells. They were not markedly fusogenic. BA.2.86.1 displayed the strongest binding to ACE2. They potently infected nasal epithelial cells, with EG.5.1.3 exhibiting the highest fitness. Nirmatrelvir, Remdesivir and Molnupiravir remained active, whereas Sotrovimab lost efficacy against BA.2.86.1. Neutralizing antibody (NAb) responses from vaccinees and BA.1/BA.2-infected individuals were 7 to 21-fold lower compared to BA.1, without major differences between variants. A breakthrough XBB infection enhanced NAb responses, particularly against XBB variants. Thus, while distinct, the evolution trajectory of these variants combines increased fitness and antibody evasion.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.566554">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.566554" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.566554">
        <p class="paperTitle">Thermal and pigment characterization of environmental fungi in the urban heat island of Baltimore City</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.566554" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.566554" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smith, D. F. Q.; Faiez, T. S.; Bencomo, A.; Casadevall, A.</p>
        <p class="info">Score: 40.0, Published: 2023-11-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.566554' target='https://doi.org/10.1101/2023.11.10.566554'> 10.1101/2023.11.10.566554</a></p>
        <p class="abstract">One of the major barriers of fungal infections of mammals is the inability to grow and/or survive at mammalian body temperature, typically around 37{degrees}C. This has provided mammals an advantage over fungi. However, environmental fungi may soon adapt to persist at higher temperatures, consistent with mammalian body temperature, due to thermal selection pressures imposed by climate change, global warming, and increased frequency of extreme heat events. Consequently, there is a need for more updated information about the thermal tolerance range of fungi near humans, such as in urban areas. The heat island effect suggests that cities are up to 8{degrees}C warmer than their suburban counterparts because of increased heat production, asphalt coatings and reduced greenspace among other factors, and it is more common in lower income and marginalized urban communities. Thus, urban centers are at increased risk for the emergence of heat tolerant fungi. In this study, we developed a methodology to collect and archive fungal isolates from sidewalk and soil samples in both warmer and cooler neighborhoods in Baltimore, Maryland. We demonstrate a novel methodology for fungal sample collection from sidewalks, employing the use of standardized and commercially available taffy. Analysis of fungal isolates collected from warmer neighborhoods revealed greater thermal tolerance and lower pigmentation, suggesting local adaptation to heat. Lower pigmentation in hotter areas is consistent with the notion that fungi use pigmentation to help regulate their temperature. Further, we identified the robust presence of the polyextremotolerant fungus Aureobasidium pullalans from the warmest neighborhood in Baltimore, further showing that the extreme conditions of cities can drive proliferation of extremotolerant fungi. This study develops new techniques for environmental fungal collection and provides insight on the fungal census in an urban setting that can inform future work to study how urban environments may drive stress/thermotolerance in fungi, which could alter fungal interactions with humans and impact human health.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568195">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568195" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568195">
        <p class="paperTitle">Giant genes are rare but implicated in cell wall degradation by predatory bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568195" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568195" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: West-Roberts, J. A.; Valentin Alvarado, L. E.; Mullen, S.; Sachdeva, R.; Smith, J.; Hug, L. A.; Gregoire, D.; Liu, W.; Lin, T.-Y.; Husain, G.; Amano, Y.; Ly, L.; Banfield, J.</p>
        <p class="info">Score: 32.3, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568195' target='https://doi.org/10.1101/2023.11.21.568195'> 10.1101/2023.11.21.568195</a></p>
        <p class="abstract">Across the tree of life, gene lengths vary, but most are no more than a few thousand base pairs in length. The largest protein often reported is the [~]40,000 aa eukaryotic Titin. Even larger proteins may occur in the rapidly expanding set of metagenome-derived sequences, but their existence may be obscured by assembly fragmentation. Here, we leverage genome curation to complete metagenome-derived sequences that encode predicted proteins of up to 85,804 aa. Overall, the findings illuminate a huge knowledge gap related to giant proteins. Although predicted proteins of &gt;30,000 aa occur in bacterial phyla such as Firmicutes and Actinobacteria, they are most common in ca. Omnitrophota, ultra small bacteria that adopt predatory lifestyles. All full length giant genes encode numerous transmembrane regions and most encode divergent secA DEAD helicase domains. In silico structural prediction of protein subregions was required to identify domains in unannotated protein segments, and revealed putative domains implicated in attachment and carbohydrate degradation. Many giant genes in new complete and near-complete Omnitrophota genomes occur in close proximity to genes homologous to type II secretion systems as well as carbohydrate import systems. This, in combination with the domain content, suggests that many bacterial giant proteins enable prey adhesion and cell wall digestion during bacterial predation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.568045">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.568045" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.568045">
        <p class="paperTitle">Detection and spread of high pathogenicity avian influenza virus H5N1 in the Antarctic Region</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.568045" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.568045" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bennison, A.; Byrne, A. M.; Reid, S. M.; Lynton-Jenkins, J. G.; Mollett, B.; De Sliva, D.; Peers-Dent, J.; Finlayson, K. E.; Hall, R.; Blockley, F.; Blyth, M.; Falchieri, M.; Fowler, Z.; Fitzcharles, E.; Brown, I. H.; James, J.; Banyard, A. C.</p>
        <p class="info">Score: 30.4, Published: 2023-11-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.568045' target='https://doi.org/10.1101/2023.11.23.568045'> 10.1101/2023.11.23.568045</a></p>
        <p class="abstract">The Antarctic is the only major geographical region in which high pathogenicity avian influenza virus (HPAIV) has never previously been detected. The current panzootic of H5N1 HPAIV has decimated wild bird populations across Europe, North America and South America. Here we report on the emergence of clade 2.3.4.4b H5N1 HPAIV in the Antarctic and sub-Antarctic regions of South Georgia and the Falkland Islands respectively. We initially detected H5N1 HPAIV in samples collected from brown skuas at Bird Island, South Georgia on 8th October 2023. Since this detection, increased mortalities were observed in brown skuas, kelp gulls, elephant seals and fur seal at multiple sites across South Georgia. We confirmed H5N1 HPAIV in multiple brown skuas and kelp gulls across four different sampling locations in South Georgia. Simultaneously, we also confirmed H5N1 HPAIV in a southern fulmar in the Falkland Islands. Genetic assessment of the virus indicates spread from South America, likely through movement of migratory birds. Here we describe the emergence, species impact and genetic composition of the virus and propose both introductory routes and potential long-term impact on avian and mammalian species across the Antarctic region.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.17.567570">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.17.567570" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.17.567570">
        <p class="paperTitle">Comprehensive contact tracing during an outbreak of alpha-variant SARS-CoV-2 in a rural community reveals less viral genomic diversity and higher household secondary attack rates than expected</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.17.567570" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.17.567570" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sivertsen, A.; Mortensen, N.; Solem, U.; Valen, E.; Bullita, M. F.; Wensaas, K.-A.; Litleskare, S.; Rortveit, G.; Grewal, H. M. S.; Ulvestad, E.</p>
        <p class="info">Score: 43.7, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.17.567570' target='https://doi.org/10.1101/2023.11.17.567570'> 10.1101/2023.11.17.567570</a></p>
        <p class="abstract">Sequencing of SARS-CoV-2 genomes throughout the COVID-19 pandemic has generated a wealth of data on viral evolution across populations, but only a few studies have so far explored SARS-CoV-2 evolution across transmission networks of tens to hundreds of persons. Here, we couple data from SARS-CoV-2 sequencing with contact tracing data from an outbreak with a single origin in a rural Norwegian community where samples from all exposed persons were collected prospectively. A total of 134 nasopharyngeal samples were positive by PCR. Among the 121 retrievable genomes, 81 were identical to the genome of the introductor, thus demonstrating that genomics offers limited additional value to manual contact-tracing. In the cases where mutations were discovered, five small genetic clusters were identified. We observed a household secondary attack rate of 67%, with 92% of household members infected among households with secondary transmission, suggesting that SARS-CoV-2 introduction into large families are likely to affect all household members.

ImportanceIn outbreak investigations, obtaining a full overview of infected individuals within a population is seldom acheived. We here present an example of just that, where a single introduction of B1.1.7 SARS-CoV-2 within a rural community allowed for tracing of the virus, from an introductor via dissemination through larger gatherings, into households. The outbreak occurred before widespread vaccination, allowing for a &#34;natural&#34; outbreak development with community lock-down. We show through sequencing that the virus can infect up to five consecutive persons without gaining mutations, thereby showing that contact tracing seems more important than sequencing for local outbreak investigations. We also show how families with small children are less likely to contain spread to all family members if SARS-CoV-2 enters the household either by a child or a caregiver, as isolation of the primary infected is difficult in such scenarios.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.568265">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.568265" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.568265">
        <p class="paperTitle">β-lactamase expression induces collateral sensitivity in Escherichia coli</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.568265" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.568265" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Herencias, C.; Alvaro-Llorente, L.; Ramiro-Martinez, P.; Munoz-Cazalla, A.; DeLaFuente, J.; Castillo-Polo, J. A.; Canton, R.; San Millan, A.; Rodriguez-Beltran, J.</p>
        <p class="info">Score: 27.4, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.568265' target='https://doi.org/10.1101/2023.11.22.568265'> 10.1101/2023.11.22.568265</a></p>
        <p class="abstract">Major antibiotic groups are losing effectiveness due to the uncontrollable spread of antimicrobial resistance (AMR) genes. Among these, {beta}-lactam resistance genes -encoding {beta}-lactamases- stand as the most common resistance mechanism in Enterobacterales due to their frequent association with mobile genetic elements. In this context, novel approaches that counter mobile AMR are urgently needed. Collateral sensitivity (CS) occurs when the acquisition of resistance to one antibiotic increases susceptibility to another antibiotic and can be exploited to selectively eliminate AMR. However, most CS networks described so far emerge as a consequence of chromosomal mutations and cannot be leveraged to tackle mobile AMR. Here, we dissected the CS response elicited by the acquisition of a prevalent antibiotic resistance plasmid to reveal that the expression of the {beta}-lactamase blaOXA-48 induces CS to colistin and azithromycin. We next showed that expression of other clinically relevant mobile {beta}-lactamases produces similar CS responses in multiple, phylogenetically unrelated E. coli strains. Finally, by combining experiments with surveillance data comprising thousands of antibiotic susceptibility tests, we showed that {beta}-lactamase-induced CS is pervasive within Enterobacterales. These results highlight that the physiological side-effects of {beta}-lactamases can be leveraged therapeutically, paving the way for the rational design of specific therapies to block mobile AMR or at least counteract their effects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568159">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568159" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568159">
        <p class="paperTitle">Asgard archaea modulate potential methanogenesis substrates in wetland soil</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568159" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568159" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Valentin-Alvarado, L. E.; Appler, K. E.; De Anda, V.; Schoelmerich, M. C.; West-Roberts, J.; Kivenson, V.; Crits-Christoph, A.; Ly, L.; Sachdeva, R.; Savage, D. F.; Baker, B. J.; Banfield, J. F.</p>
        <p class="info">Score: 27.0, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568159' target='https://doi.org/10.1101/2023.11.21.568159'> 10.1101/2023.11.21.568159</a></p>
        <p class="abstract">The roles of Asgard archaea in eukaryogenesis and marine biogeochemical cycles are well studied, yet their contributions in soil ecosystems are unknown. Of particular interest are Asgard archaeal contributions to methane cycling in wetland soils. To investigate this, we reconstructed two complete genomes for soil-associated Atabeyarchaeia, a new Asgard lineage, and the first complete genome of Freyarchaeia, and defined their metabolism in situ. Metatranscriptomics highlights high expression of [NiFe]-hydrogenases, pyruvate oxidation and carbon fixation via the Wood-Ljungdahl pathway genes. Also highly expressed are genes encoding enzymes for amino acid metabolism, anaerobic aldehyde oxidation, hydrogen peroxide detoxification and glycerol and carbohydrate breakdown to acetate and formate. Overall, soil-associated Asgard archaea are predicted to be non-methanogenic acetogens, likely impacting reservoirs of substrates for methane production in terrestrial ecosystems.

One-Sentence SummaryComplete genomes of Asgard archaea, coupled with metatranscriptomic data, indicate roles in production and consumption of carbon compounds that are known to serve as substrates for methane production in wetlands.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
