<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="ORBIT for E. coli: Kilobase-scale oligonucleotide recombineering at high throughput and high efficiency
Authors: Saunders, S. H.; Ahmed, A. M.
Score: 40.5, Published: 2023-07-11 DOI: 10.1101/2023.06.28.546561
Microbiology and synthetic biology depend on reverse genetic approaches to manipulate bacterial genomes; however, existing methods require molecular biology to generate genomic homology, suffer from low efficiency, and are not easily scaled to high throughput applications. To overcome these limitations, we developed a system for creating kilobase-scale genomic modifications that uses DNA oligonucleotides to direct the integration of a non-replicating plasmid.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="ORBIT for E. coli: Kilobase-scale oligonucleotide recombineering at high throughput and high efficiency
Authors: Saunders, S. H.; Ahmed, A. M.
Score: 40.5, Published: 2023-07-11 DOI: 10.1101/2023.06.28.546561
Microbiology and synthetic biology depend on reverse genetic approaches to manipulate bacterial genomes; however, existing methods require molecular biology to generate genomic homology, suffer from low efficiency, and are not easily scaled to high throughput applications. To overcome these limitations, we developed a system for creating kilobase-scale genomic modifications that uses DNA oligonucleotides to direct the integration of a non-replicating plasmid." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-16T10:39:40+00:00" />
<meta property="article:modified_time" content="2023-07-16T10:39:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="ORBIT for E. coli: Kilobase-scale oligonucleotide recombineering at high throughput and high efficiency
Authors: Saunders, S. H.; Ahmed, A. M.
Score: 40.5, Published: 2023-07-11 DOI: 10.1101/2023.06.28.546561
Microbiology and synthetic biology depend on reverse genetic approaches to manipulate bacterial genomes; however, existing methods require molecular biology to generate genomic homology, suffer from low efficiency, and are not easily scaled to high throughput applications. To overcome these limitations, we developed a system for creating kilobase-scale genomic modifications that uses DNA oligonucleotides to direct the integration of a non-replicating plasmid."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "ORBIT for E. coli: Kilobase-scale oligonucleotide recombineering at high throughput and high efficiency\nAuthors: Saunders, S. H.; Ahmed, A. M.\nScore: 40.5, Published: 2023-07-11 DOI: 10.1101/2023.06.28.546561\nMicrobiology and synthetic biology depend on reverse genetic approaches to manipulate bacterial genomes; however, existing methods require molecular biology to generate genomic homology, suffer from low efficiency, and are not easily scaled to high throughput applications. To overcome these limitations, we developed a system for creating kilobase-scale genomic modifications that uses DNA oligonucleotides to direct the integration of a non-replicating plasmid.",
  "keywords": [
    
  ],
  "articleBody": " ORBIT for E. coli: Kilobase-scale oligonucleotide recombineering at high throughput and high efficiency\nAuthors: Saunders, S. H.; Ahmed, A. M.\nScore: 40.5, Published: 2023-07-11 DOI: 10.1101/2023.06.28.546561\nMicrobiology and synthetic biology depend on reverse genetic approaches to manipulate bacterial genomes; however, existing methods require molecular biology to generate genomic homology, suffer from low efficiency, and are not easily scaled to high throughput applications. To overcome these limitations, we developed a system for creating kilobase-scale genomic modifications that uses DNA oligonucleotides to direct the integration of a non-replicating plasmid. This method, Oligonucleotide Recombineering followed by Bxb-1 Integrase Targeting (ORBIT) was pioneered in Mycobacteria, and here we adapt and expand it for E. coli. Our redesigned plasmid toolkit achieved nearly 1000x higher efficiency than {lambda} Red recombination and enabled precise, stable knockouts (\u003c134 kb) and integrations (\u003c11 kb) of various sizes. Additionally, we constructed multi-mutants (double and triple) in a single transformation, using orthogonal attachment sites. At high throughput, we used pools of targeting oligonucleotides to knock out nearly all known transcription factor and small RNA genes, yielding accurate, genome-wide, single mutant libraries. By counting genomic barcodes, we also show ORBIT libraries can scale to thousands of unique members (\u003e30k). This work demonstrates that ORBIT for E. coli is a flexible reverse genetic system that facilitates rapid construction of complex strains and readily scales to create sophisticated mutant libraries.\nFirst Eurasian cases of SARS-CoV-2 seropositivity in a free-ranging urban population of wild fallow deer\nAuthors: Purves, K.; Brown, H.; Haverty, R.; Ryan, A.; Griffin, L.; McCormack, J.; O'Reilly, S. R.; Mallon, P. W.; Gautier, V.; Cassidy, J. P.; Fabre, A.; Carr, M. J.; Gonzalez, G.; Ciuti, S.; Fletcher, N.\nScore: 64.6, Published: 2023-07-07 DOI: 10.1101/2023.07.07.547941\nSevere acute respiratory syndrome coronavirus-2 (SARS-CoV-2) infects wildlife. Recent studies highlighted that variants of concern (VOC) may expand into novel animal reservoirs with the potential for reverse zoonosis. North American white-tailed deer are the only deer species in which SARS-CoV-2 has been documented, raising the question whether further reservoir species exist as new VOC emerge. Here, we report the first cases of deer SARS-CoV-2 seropositivity in Eurasia, in a city population of fallow deer in Dublin, Ireland. Deer were seronegative in 2020 (circulating variant in humans: Alpha), one animal was seropositive in 2021 (Delta variant), and 57% of animals tested in 2022 were seropositive (Omicron variant). Ex vivo, a clinical isolate of Omicron BA.1 infected fallow deer precision cut lung slice type-2 pneumocytes, also a major target of infection in human lungs. Our findings suggest a change in host tropism as new variants emerged in the human reservoir, highlighting the importance of continued wildlife disease monitoring and limiting human-wildlife contacts. Teaser: This study is the first report of SARS-CoV-2 seropositivity in fallow deer, highlighting expansion of viral variants into new host reservoirs.\nBridges instead of boats? The Mla system of diderm Firmicute Veillonella parvula reveals an ancestral transenvelope core of phospholipid trafficking\nAuthors: BELOIN, C.; Grasekamp, K. P.; Beaud, B.; Taib, N.; Audrain, B.; Bardiaux, B.; Rossez, Y.; IZADI-PRUNEYRE, N.; Lejeune, M.; Trivelli, X.; Chouit, Z.; Guerardel, Y.; GHIGO, J.-M.; Gribaldo, S.\nScore: 35.8, Published: 2023-06-30 DOI: 10.1101/2023.06.30.547184\nDespite extensive characterisation of envelope biogenesis systems in diderm bacteria, glycerophospholipid (GPL) trafficking remains poorly understood, and has only been studied in a handful of model species. Within the Proteobacteria, the maintenance of lipid asymmetry (Mla) system facilitates retrograde GPL trafficking via six proteins, MlaA-F. GPLs are extracted from the outer leaflet of the outer membrane by the lipoprotein MlaA which associates with porin trimers, then shipped through the periplasmic space by the chaperone MlaC, which finally delivers GPLs to the inner membrane complex formed by MlaBDEF. Here, we investigate GPL trafficking in Veillonella parvula, a diderm member of the Firmicutes which encodes an Mla system devoid of MlaA and MlaC. V. parvula {Delta}mla mutants display phenotypes characteristic of disrupted lipid asymmetry such as hypervesiculation and detergent hypersensitivity, and lipid content analysis from outer membrane vesicles reveals an enrichment for the major lipid component phosphatidylethanolamine. Interestingly, suppressor analysis identifies mutations in tamB that rescue detergent hypersensitivity and hypervesiculation of {Delta}mla strains, supporting the involvement of these two systems in antagonistic GPL trafficking functions across diverse bacterial lineages. A combination of structural modeling and subcellular localisation assays shows that MlaDVp is longer than in classical diderm models and forms a transenvelope bridge, encoding both an inner membrane-localised MCE domain and an outer membrane {beta}-barrel. These results strongly suggest that V. parvula possesses a minimal Mla system for GPL trafficking, replacing the need for chaperones and outer membrane lipoproteins by directly connecting the two membranes. Finally, phylogenomic analysis indicates that this MlaEFD self-contained architecture is widely distributed in diderm bacteria and most likely represents the ancestral functional core of the Mla system, which subsequently increased in complexity in Proteobacteria and closely related phyla following the emergence of MlaABC. Our work broadens the diversity of current models of GPL trafficking in diderm bacteria, challenging the paradigm set by classical models and shedding light on the evolution of a crucial system in the biogenesis and maintenance of the bacterial outer membrane.\nA side-by-side comparison of peptide-delivered antisense antibiotics employing different nucleotide mimics\nAuthors: Ghosh, C.; Popella, L.; Dhamodharan, V.; Jung, J.; Barquist, L.; Hoebartner, C.; Vogel, J.\nScore: 24.3, Published: 2023-07-11 DOI: 10.1101/2023.07.11.548539\nAntisense oligomer (ASO)-based antibiotics that target mRNAs of essential bacterial genes have great potential for counteracting antimicrobial resistance and for precision microbiome editing. To date, the development of such antisense antibiotics has primarily focused on using phosphorodiamidate morpholino (PMO) and peptide nucleic acid (PNA) backbones, largely ignoring the growing number of chemical modalities that have spurred the success of ASO-based human therapy. Here, we directly compare the activities of seven chemically distinct 10mer ASOs, all designed to target the essential gene acpP upon delivery with a KFF-peptide carrier into Salmonella. Our systematic analysis of PNA, PMO, phosphorothioate-modified DNA (PTO), 2-methylated RNA (RNA-OMe), 2-methoxyethylated RNA (RNA-MOE), 2-fluorinated RNA (RNA-F) and 2-4-locked RNA (LNA) is based on a variety of in vitro and in vivo methods to evaluate ASO uptake, target pairing and inhibition of bacterial growth. Our data show that only PNA and PMO are efficiently delivered by the KFF peptide into Salmonella to inhibit bacterial growth. Nevertheless, the strong target binding affinity and in vitro translational repression activity of LNA and RNA-MOE make them promising modalities for antisense antibiotics that will require the identification of an effective carrier.\nLC3B conjugation machinery promotes autophagy-independent HIV-1 entry in CD4+ T lymphocytes\nAuthors: Pradel, B.; Deffieu, M. S.; Robert-Hebmann, V.; Cantaloube, G.; Faure, M.; Chazal, N.; Gaudin, R.; Espert, L.\nScore: 20.4, Published: 2023-07-11 DOI: 10.1101/2023.07.11.548555\nHIV-1 entry into CD4+ T lymphocytes relies on the viral and cellular membranes fusion, leading to viral capsid delivery in the cytoplasm of target cells. The conjugation of ATG8/LC3B protein, process referred to as ATG8ylation and mainly studied in the context of autophagy, occurs transiently in the early stages of the HIV-1 replication cycle in CD4+ T lymphocytes. Despite numerous studies investigating the interplays of HIV-1 with autophagy machinery, the impact of ATG8ylation in the early stages of HIV-1 infection remains unknown. Here we found that HIV-1 exposure leads to the rapid enrichment of LC3B towards the target cell plasma membrane, in close proximity with the incoming viral particles. Furthermore, we demonstrated that ATG8ylation is a key event that facilitates HIV-1 fusion with target CD4+ T cells. Interestingly, this effect is independent of the canonical autophagy pathway as ATG13 silencing does not prevent HIV-1 entry. Together, our results provide an unconventional role of LC3B conjugation subverted by HIV-1 to achieve a critical early step of its replication cycle. TeaserHIV-1 induces LC3B enrichment towards its target cell entry site and uses the conjugation of this protein to favor its entry step.\nDual Membrane-Sspanning Anti-Sigma Factors Regulate Vesiculation in Gut Bacteroidota\nAuthors: Pardue, E. J.; Sartorio, M.; Jana, B.; Scott, N.; Beatty, W.; Ortiz-Marquez, J.; van Opijnen, T.; Hsu, F.-F.; Potter, R. F.; Feldman, M.\nScore: 18.9, Published: 2023-07-13 DOI: 10.1101/2023.07.13.548920\nBacteroidota are abundant members of the human gut microbiota that shape the enteric landscape by modulating host immunity and degrading dietary- and host-derived glycans. These processes are at least partially mediated by Outer Membrane Vesicles (OMVs). In this work, we developed a high-throughput screen to identify genes required for OMV biogenesis and its regulation in Bacteroides thetaiotaomicron (Bt). Our screening led us to the identification of a novel family of Dual Membrane-spanning Anti-sigma factors (Dma), which regulate OMV biogenesis in Bt. We employed molecular and multiomic analyses to demonstrate that deletion of Dma1, the founding member of the Dma family, results in hypervesiculation by modulating the expression of NigD1, which belongs to a family of uncharacterized proteins found throughout Bacteroidota. Dma1 has an unprecedented domain organization: it contains a C-terminal {beta}-barrel embedded in the OM; its N-terminal domain interacts with its cognate sigma factor in the cytoplasm, and both domains are tethered via an intrinsically disordered region that traverses the periplasm. Phylogenetic analyses reveal that the Dma family is a unique feature of Bacteroidota. This study provides the first mechanistic insights into the regulation of OMV biogenesis in human gut bacteria\nThe distribution of fitness effects of plasmid pOXA-48 in clinical enterobacteria\nAuthors: Fernandez-Calvet, A.; Toribio-Celestino, L.; Alonso-del Valle, A.; Sastre-Dominguez, J.; Valdes-Chiara, P.; San Millan, A.; DelaFuente, J.\nScore: 14.9, Published: 2023-07-11 DOI: 10.1101/2023.07.11.548518\nAntimicrobial resistance (AMR) in bacteria is a major public health problem. The main route for AMR acquisition in clinically important bacteria is the horizontal transfer of plasmids carrying resistance genes. AMR plasmids allow bacteria to survive antibiotics, but they also entail physiological alterations in the host cell. Multiple studies over the last years indicate that these alterations can translate into a fitness cost when antibiotics are absent. However, due to technical limitations, most of these studies are based on analysing new associations between plasmids and bacteria generated in vitro, and we know very little about the effects of plasmids in their native bacterial hosts. In this study, we used a CRISPR-Cas9-tool to selectively cure plasmids from clinical enterobacteria to overcome this limitation. Using this approach, we were able to study the fitness effects of the carbapenem resistance plasmid pOXA-48 in 35 pOXA-48-carrying isolates recovered from hospitalised patients. Our results revealed that pOXA-48 produces variable effects across the collection of wild type enterobacterial strains naturally carrying the plasmid, ranging from fitness costs to fitness benefits. Importantly, the plasmid was only associated with a significant fitness reduction in 4 out of 35 clones, and produced no significant changes in fitness in the great majority of isolates. Our results suggest that plasmids produce neutral fitness effects in most native bacterial hosts, helping to explain the great prevalence of plasmids in natural microbial communities.\nNECing goes: flexibility of the herpesvirus nuclear egress complex\nAuthors: Prazak, V.; Mironova, Y.; Vasishtan, D.; Hagen, C.; Laugks, U.; Jensen, Y.; Sanders, S.; Heumann, J. M.; Bosse, J. B.; Klupp, B.; Mettenleiter, T. C.; Grange, M.; Grünewald, K.\nScore: 21.1, Published: 2023-07-07 DOI: 10.1101/2023.07.07.547920\nThe nuclear egress complex (NEC) allows herpesvirus capsids to escape from the nucleus without breaking the nuclear envelope barrier. It assembles into a lattice on the inner nuclear membrane enveloping newly assembled nucleocapsids, which bud into the perinuclear space. The primary virion envelope subsequently fuses with the outer nuclear membrane, releasing the capsid into the cytosol. Here we interrogated the NEC in the context of intact cells infected with pseudorabies or herpes simplex virus using focused-ion beam milling and electron cryo- tomography. We determined the structure of NEC in different conformations and show that it consists of a flexible hexameric lattice that generates curvature through a combination of ordered and disordered domains. After interrogating the intermediate stages of capsid formation, we show that capsid vertex binding may initiate envelopment but does not directly induce curvature formation. These data and many examples of the intermediate stages of nuclear egress paint a detailed holistic view of a versatile transport system.\nAlternative cell entry mechanisms for SARS-CoV-2 and multiple animal viruses\nAuthors: Ojha, R.; Jiang, A.; Mantyla, E.; Modhiran, N.; Witte, R.; Gaudin, A.; De Zanetti, L.; Gormal, R.; Vihinen-Ranta, M.; Mercer, J.; Suomalainen, M.; Greber, U. F.; Yamauchi, Y.; Yves-Lozach, P.; Helenius, A.; Vapalahti, O.; Young, P.; Watterson, D.; Meunier, F. A.; Joensuu, M.; Balistreri, G.\nScore: 16.0, Published: 2023-07-11 DOI: 10.1101/2023.07.02.547368\nThe cell entry mechanism of SARS-CoV-2, the causative agent of the COVID-19 pandemic, is not fully understood. Most animal viruses hijack cellular endocytic pathways as an entry route into the cell. Here, we show that in cells that do not express serine proteases such as TMPRSS2, genetic depletion of all dynamin isoforms blocked the uptake and strongly reduced infection with SARS-CoV-2 and its variant Delta. However, increasing the viral loads partially and dose-dependently restored infection via a thus far uncharacterized entry mechanism. Ultrastructural analysis by electron microscopy showed that this dynamin-independent endocytic processes appeared as 150-200 nm non-coated invaginations and was efficiently used by numerous mammalian viruses, including alphaviruses, influenza, vesicular stomatitis, bunya, adeno, vaccinia, and rhinovirus. Both the dynamin-dependent and dynamin-independent infection of SARS-CoV-2 required a functional actin cytoskeleton. In contrast, the alphavirus Semliki Forest virus, which is smaller in diameter, required actin only for the dynamin-independent entry. The presence of TMPRSS2 protease rescued SARS-CoV-2 infection in the absence of dynamins. Collectively, these results indicate that some viruses such as canine parvovirus and SARS-CoV-2 mainly rely on dynamin for endocytosis-dependent infection, while other viruses can efficiently bypass this requirement harnessing an alternative infection entry route dependent on actin.\nTranscriptome profiling of type VI secretion system core gene tssM mutant of Xanthomonas perforans highlights regulators controlling diverse functions ranging from virulence to metabolism\nAuthors: Ramamoorthy, S.; Pena, M.; Ghosh, P.; Liao, Y.-Y.; Paret, M. L.; Jones, J. B.; Potnis, N.\nScore: 8.1, Published: 2023-07-13 DOI: 10.1101/2023.07.13.548846\nType VI secretion system (T6SS) is a contact dependent contractile nano-weapon in Gram-negative bacteria that fires effector molecules directly into prokaryotic and eukaryotic cells aiding in manipulation of the host and killing of competitors in complex niches. In plant pathogenic xanthomonads, T6SS has been demonstrated to play diverse roles in individual pathosystems. However, the regulatory circuit involved in mediating biological functions carried out by T6SS are still elusive. To bridge this knowledge gap, we conducted an in vitro transcriptome screen using plant apoplast mimicking minimal medium, XVM2 medium, to decipher the effect of tssM deletion, a core gene belonging to T6SS-cluster i3*, on the regulation of gene expression in Xanthomonas perforans strain AL65. Transcriptomic data revealed that a total of 277 and 525 genes were upregulated, while 307 and 392 genes were downregulated in the mutant strain post 8 and 16 hours of growth in XVM2 medium. The transcript abundance of several genes associated with flagellum and pilus biogenesis as well as type III secretion system were downregulated in the mutant strain. Deletion of tssM of cluster-i3* resulted in upregulation of several T6SS genes belonging to cluster-i3*** and genes involved in biofilm and cell wall biogenesis. Transcription regulators like rpoN, Pho regulon, rpoE and csrA were identified to be upregulated in the mutant strain. Our results suggest that T6SS modulates the expression of global regulators like csrA, rpoN and pho regulons triggering a signaling cascade and co-ordinates the expression of suite of virulence factors, stress response genes and metabolic genes.\n",
  "wordCount" : "2547",
  "inLanguage": "en",
  "datePublished": "2023-07-16T10:39:40Z",
  "dateModified": "2023-07-16T10:39:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta"><span>updated on July 16, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546561">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546561" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546561">
        <p class="paperTitle">ORBIT for E. coli: Kilobase-scale oligonucleotide recombineering at high throughput and high efficiency</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546561" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546561" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Saunders, S. H.; Ahmed, A. M.</p>
        <p class="info">Score: 40.5, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546561' target='https://doi.org/10.1101/2023.06.28.546561'> 10.1101/2023.06.28.546561</a></p>
        <p class="abstract">Microbiology and synthetic biology depend on reverse genetic approaches to manipulate bacterial genomes; however, existing methods require molecular biology to generate genomic homology, suffer from low efficiency, and are not easily scaled to high throughput applications. To overcome these limitations, we developed a system for creating kilobase-scale genomic modifications that uses DNA oligonucleotides to direct the integration of a non-replicating plasmid. This method, Oligonucleotide Recombineering followed by Bxb-1 Integrase Targeting (ORBIT) was pioneered in Mycobacteria, and here we adapt and expand it for E. coli. Our redesigned plasmid toolkit achieved nearly 1000x higher efficiency than {lambda} Red recombination and enabled precise, stable knockouts (&lt;134 kb) and integrations (&lt;11 kb) of various sizes. Additionally, we constructed multi-mutants (double and triple) in a single transformation, using orthogonal attachment sites. At high throughput, we used pools of targeting oligonucleotides to knock out nearly all known transcription factor and small RNA genes, yielding accurate, genome-wide, single mutant libraries. By counting genomic barcodes, we also show ORBIT libraries can scale to thousands of unique members (&gt;30k). This work demonstrates that ORBIT for E. coli is a flexible reverse genetic system that facilitates rapid construction of complex strains and readily scales to create sophisticated mutant libraries.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.547941">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.547941" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.547941">
        <p class="paperTitle">First Eurasian cases of SARS-CoV-2 seropositivity in a free-ranging urban population of wild fallow deer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.547941" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.547941" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Purves, K.; Brown, H.; Haverty, R.; Ryan, A.; Griffin, L.; McCormack, J.; O&#39;Reilly, S. R.; Mallon, P. W.; Gautier, V.; Cassidy, J. P.; Fabre, A.; Carr, M. J.; Gonzalez, G.; Ciuti, S.; Fletcher, N.</p>
        <p class="info">Score: 64.6, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.547941' target='https://doi.org/10.1101/2023.07.07.547941'> 10.1101/2023.07.07.547941</a></p>
        <p class="abstract">Severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) infects wildlife. Recent studies highlighted that variants of concern (VOC) may expand into novel animal reservoirs with the potential for reverse zoonosis. North American white-tailed deer are the only deer species in which SARS-CoV-2 has been documented, raising the question whether further reservoir species exist as new VOC emerge. Here, we report the first cases of deer SARS-CoV-2 seropositivity in Eurasia, in a city population of fallow deer in Dublin, Ireland. Deer were seronegative in 2020 (circulating variant in humans: Alpha), one animal was seropositive in 2021 (Delta variant), and 57% of animals tested in 2022 were seropositive (Omicron variant). Ex vivo, a clinical isolate of Omicron BA.1 infected fallow deer precision cut lung slice type-2 pneumocytes, also a major target of infection in human lungs. Our findings suggest a change in host tropism as new variants emerged in the human reservoir, highlighting the importance of continued wildlife disease monitoring and limiting human-wildlife contacts.

Teaser: This study is the first report of SARS-CoV-2 seropositivity in fallow deer, highlighting expansion of viral variants into new host reservoirs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547184">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547184" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547184">
        <p class="paperTitle">Bridges instead of boats? The Mla system of diderm Firmicute Veillonella parvula reveals an ancestral transenvelope core of phospholipid trafficking</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547184" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547184" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: BELOIN, C.; Grasekamp, K. P.; Beaud, B.; Taib, N.; Audrain, B.; Bardiaux, B.; Rossez, Y.; IZADI-PRUNEYRE, N.; Lejeune, M.; Trivelli, X.; Chouit, Z.; Guerardel, Y.; GHIGO, J.-M.; Gribaldo, S.</p>
        <p class="info">Score: 35.8, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547184' target='https://doi.org/10.1101/2023.06.30.547184'> 10.1101/2023.06.30.547184</a></p>
        <p class="abstract">Despite extensive characterisation of envelope biogenesis systems in diderm bacteria, glycerophospholipid (GPL) trafficking remains poorly understood, and has only been studied in a handful of model species. Within the Proteobacteria, the maintenance of lipid asymmetry (Mla) system facilitates retrograde GPL trafficking via six proteins, MlaA-F. GPLs are extracted from the outer leaflet of the outer membrane by the lipoprotein MlaA which associates with porin trimers, then shipped through the periplasmic space by the chaperone MlaC, which finally delivers GPLs to the inner membrane complex formed by MlaBDEF. Here, we investigate GPL trafficking in Veillonella parvula, a diderm member of the Firmicutes which encodes an Mla system devoid of MlaA and MlaC. V. parvula {Delta}mla mutants display phenotypes characteristic of disrupted lipid asymmetry such as hypervesiculation and detergent hypersensitivity, and lipid content analysis from outer membrane vesicles reveals an enrichment for the major lipid component phosphatidylethanolamine. Interestingly, suppressor analysis identifies mutations in tamB that rescue detergent hypersensitivity and hypervesiculation of {Delta}mla strains, supporting the involvement of these two systems in antagonistic GPL trafficking functions across diverse bacterial lineages. A combination of structural modeling and subcellular localisation assays shows that MlaDVp is longer than in classical diderm models and forms a transenvelope bridge, encoding both an inner membrane-localised MCE domain and an outer membrane {beta}-barrel. These results strongly suggest that V. parvula possesses a minimal Mla system for GPL trafficking, replacing the need for chaperones and outer membrane lipoproteins by directly connecting the two membranes. Finally, phylogenomic analysis indicates that this MlaEFD self-contained architecture is widely distributed in diderm bacteria and most likely represents the ancestral functional core of the Mla system, which subsequently increased in complexity in Proteobacteria and closely related phyla following the emergence of MlaABC. Our work broadens the diversity of current models of GPL trafficking in diderm bacteria, challenging the paradigm set by classical models and shedding light on the evolution of a crucial system in the biogenesis and maintenance of the bacterial outer membrane.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.548539">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.548539" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.548539">
        <p class="paperTitle">A side-by-side comparison of peptide-delivered antisense antibiotics employing different nucleotide mimics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.548539" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.548539" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ghosh, C.; Popella, L.; Dhamodharan, V.; Jung, J.; Barquist, L.; Hoebartner, C.; Vogel, J.</p>
        <p class="info">Score: 24.3, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.548539' target='https://doi.org/10.1101/2023.07.11.548539'> 10.1101/2023.07.11.548539</a></p>
        <p class="abstract">Antisense oligomer (ASO)-based antibiotics that target mRNAs of essential bacterial genes have great potential for counteracting antimicrobial resistance and for precision microbiome editing. To date, the development of such antisense antibiotics has primarily focused on using phosphorodiamidate morpholino (PMO) and peptide nucleic acid (PNA) backbones, largely ignoring the growing number of chemical modalities that have spurred the success of ASO-based human therapy. Here, we directly compare the activities of seven chemically distinct 10mer ASOs, all designed to target the essential gene acpP upon delivery with a KFF-peptide carrier into Salmonella. Our systematic analysis of PNA, PMO, phosphorothioate-modified DNA (PTO), 2-methylated RNA (RNA-OMe), 2-methoxyethylated RNA (RNA-MOE), 2-fluorinated RNA (RNA-F) and 2-4-locked RNA (LNA) is based on a variety of in vitro and in vivo methods to evaluate ASO uptake, target pairing and inhibition of bacterial growth. Our data show that only PNA and PMO are efficiently delivered by the KFF peptide into Salmonella to inhibit bacterial growth. Nevertheless, the strong target binding affinity and in vitro translational repression activity of LNA and RNA-MOE make them promising modalities for antisense antibiotics that will require the identification of an effective carrier.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.548555">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.548555" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.548555">
        <p class="paperTitle">LC3B conjugation machinery promotes autophagy-independent HIV-1 entry in CD4&#43; T lymphocytes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.548555" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.548555" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pradel, B.; Deffieu, M. S.; Robert-Hebmann, V.; Cantaloube, G.; Faure, M.; Chazal, N.; Gaudin, R.; Espert, L.</p>
        <p class="info">Score: 20.4, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.548555' target='https://doi.org/10.1101/2023.07.11.548555'> 10.1101/2023.07.11.548555</a></p>
        <p class="abstract">HIV-1 entry into CD4&#43; T lymphocytes relies on the viral and cellular membranes fusion, leading to viral capsid delivery in the cytoplasm of target cells. The conjugation of ATG8/LC3B protein, process referred to as ATG8ylation and mainly studied in the context of autophagy, occurs transiently in the early stages of the HIV-1 replication cycle in CD4&#43; T lymphocytes. Despite numerous studies investigating the interplays of HIV-1 with autophagy machinery, the impact of ATG8ylation in the early stages of HIV-1 infection remains unknown. Here we found that HIV-1 exposure leads to the rapid enrichment of LC3B towards the target cell plasma membrane, in close proximity with the incoming viral particles. Furthermore, we demonstrated that ATG8ylation is a key event that facilitates HIV-1 fusion with target CD4&#43; T cells. Interestingly, this effect is independent of the canonical autophagy pathway as ATG13 silencing does not prevent HIV-1 entry. Together, our results provide an unconventional role of LC3B conjugation subverted by HIV-1 to achieve a critical early step of its replication cycle.

TeaserHIV-1 induces LC3B enrichment towards its target cell entry site and uses the conjugation of this protein to favor its entry step.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.548920">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.548920" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.548920">
        <p class="paperTitle">Dual Membrane-Sspanning Anti-Sigma Factors Regulate Vesiculation in Gut Bacteroidota</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.548920" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.548920" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pardue, E. J.; Sartorio, M.; Jana, B.; Scott, N.; Beatty, W.; Ortiz-Marquez, J.; van Opijnen, T.; Hsu, F.-F.; Potter, R. F.; Feldman, M.</p>
        <p class="info">Score: 18.9, Published: 2023-07-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.548920' target='https://doi.org/10.1101/2023.07.13.548920'> 10.1101/2023.07.13.548920</a></p>
        <p class="abstract">Bacteroidota are abundant members of the human gut microbiota that shape the enteric landscape by modulating host immunity and degrading dietary- and host-derived glycans. These processes are at least partially mediated by Outer Membrane Vesicles (OMVs). In this work, we developed a high-throughput screen to identify genes required for OMV biogenesis and its regulation in Bacteroides thetaiotaomicron (Bt). Our screening led us to the identification of a novel family of Dual Membrane-spanning Anti-sigma factors (Dma), which regulate OMV biogenesis in Bt. We employed molecular and multiomic analyses to demonstrate that deletion of Dma1, the founding member of the Dma family, results in hypervesiculation by modulating the expression of NigD1, which belongs to a family of uncharacterized proteins found throughout Bacteroidota. Dma1 has an unprecedented domain organization: it contains a C-terminal {beta}-barrel embedded in the OM; its N-terminal domain interacts with its cognate sigma factor in the cytoplasm, and both domains are tethered via an intrinsically disordered region that traverses the periplasm. Phylogenetic analyses reveal that the Dma family is a unique feature of Bacteroidota. This study provides the first mechanistic insights into the regulation of OMV biogenesis in human gut bacteria</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.548518">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.548518" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.548518">
        <p class="paperTitle">The distribution of fitness effects of plasmid pOXA-48 in clinical enterobacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.548518" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.548518" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fernandez-Calvet, A.; Toribio-Celestino, L.; Alonso-del Valle, A.; Sastre-Dominguez, J.; Valdes-Chiara, P.; San Millan, A.; DelaFuente, J.</p>
        <p class="info">Score: 14.9, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.548518' target='https://doi.org/10.1101/2023.07.11.548518'> 10.1101/2023.07.11.548518</a></p>
        <p class="abstract">Antimicrobial resistance (AMR) in bacteria is a major public health problem. The main route for AMR acquisition in clinically important bacteria is the horizontal transfer of plasmids carrying resistance genes. AMR plasmids allow bacteria to survive antibiotics, but they also entail physiological alterations in the host cell. Multiple studies over the last years indicate that these alterations can translate into a fitness cost when antibiotics are absent. However, due to technical limitations, most of these studies are based on analysing new associations between plasmids and bacteria generated in vitro, and we know very little about the effects of plasmids in their native bacterial hosts. In this study, we used a CRISPR-Cas9-tool to selectively cure plasmids from clinical enterobacteria to overcome this limitation. Using this approach, we were able to study the fitness effects of the carbapenem resistance plasmid pOXA-48 in 35 pOXA-48-carrying isolates recovered from hospitalised patients. Our results revealed that pOXA-48 produces variable effects across the collection of wild type enterobacterial strains naturally carrying the plasmid, ranging from fitness costs to fitness benefits. Importantly, the plasmid was only associated with a significant fitness reduction in 4 out of 35 clones, and produced no significant changes in fitness in the great majority of isolates. Our results suggest that plasmids produce neutral fitness effects in most native bacterial hosts, helping to explain the great prevalence of plasmids in natural microbial communities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.547920">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.547920" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.547920">
        <p class="paperTitle">NECing goes: flexibility of the herpesvirus nuclear egress complex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.547920" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.547920" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Prazak, V.; Mironova, Y.; Vasishtan, D.; Hagen, C.; Laugks, U.; Jensen, Y.; Sanders, S.; Heumann, J. M.; Bosse, J. B.; Klupp, B.; Mettenleiter, T. C.; Grange, M.; Grünewald, K.</p>
        <p class="info">Score: 21.1, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.547920' target='https://doi.org/10.1101/2023.07.07.547920'> 10.1101/2023.07.07.547920</a></p>
        <p class="abstract">The nuclear egress complex (NEC) allows herpesvirus capsids to escape from the nucleus without breaking the nuclear envelope barrier. It assembles into a lattice on the inner nuclear membrane enveloping newly assembled nucleocapsids, which bud into the perinuclear space. The primary virion envelope subsequently fuses with the outer nuclear membrane, releasing the capsid into the cytosol. Here we interrogated the NEC in the context of intact cells infected with pseudorabies or herpes simplex virus using focused-ion beam milling and electron cryo- tomography. We determined the structure of NEC in different conformations and show that it consists of a flexible hexameric lattice that generates curvature through a combination of ordered and disordered domains. After interrogating the intermediate stages of capsid formation, we show that capsid vertex binding may initiate envelopment but does not directly induce curvature formation. These data and many examples of the intermediate stages of nuclear egress paint a detailed holistic view of a versatile transport system.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.02.547368">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.02.547368" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.02.547368">
        <p class="paperTitle">Alternative cell entry mechanisms for SARS-CoV-2 and multiple animal viruses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.02.547368" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.02.547368" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ojha, R.; Jiang, A.; Mantyla, E.; Modhiran, N.; Witte, R.; Gaudin, A.; De Zanetti, L.; Gormal, R.; Vihinen-Ranta, M.; Mercer, J.; Suomalainen, M.; Greber, U. F.; Yamauchi, Y.; Yves-Lozach, P.; Helenius, A.; Vapalahti, O.; Young, P.; Watterson, D.; Meunier, F. A.; Joensuu, M.; Balistreri, G.</p>
        <p class="info">Score: 16.0, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.02.547368' target='https://doi.org/10.1101/2023.07.02.547368'> 10.1101/2023.07.02.547368</a></p>
        <p class="abstract">The cell entry mechanism of SARS-CoV-2, the causative agent of the COVID-19 pandemic, is not fully understood. Most animal viruses hijack cellular endocytic pathways as an entry route into the cell. Here, we show that in cells that do not express serine proteases such as TMPRSS2, genetic depletion of all dynamin isoforms blocked the uptake and strongly reduced infection with SARS-CoV-2 and its variant Delta. However, increasing the viral loads partially and dose-dependently restored infection via a thus far uncharacterized entry mechanism. Ultrastructural analysis by electron microscopy showed that this dynamin-independent endocytic processes appeared as 150-200 nm non-coated invaginations and was efficiently used by numerous mammalian viruses, including alphaviruses, influenza, vesicular stomatitis, bunya, adeno, vaccinia, and rhinovirus. Both the dynamin-dependent and dynamin-independent infection of SARS-CoV-2 required a functional actin cytoskeleton. In contrast, the alphavirus Semliki Forest virus, which is smaller in diameter, required actin only for the dynamin-independent entry. The presence of TMPRSS2 protease rescued SARS-CoV-2 infection in the absence of dynamins. Collectively, these results indicate that some viruses such as canine parvovirus and SARS-CoV-2 mainly rely on dynamin for endocytosis-dependent infection, while other viruses can efficiently bypass this requirement harnessing an alternative infection entry route dependent on actin.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.548846">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.548846" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.548846">
        <p class="paperTitle">Transcriptome profiling of type VI secretion system core gene tssM mutant of Xanthomonas perforans highlights regulators controlling diverse functions ranging from virulence to metabolism</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.548846" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.548846" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ramamoorthy, S.; Pena, M.; Ghosh, P.; Liao, Y.-Y.; Paret, M. L.; Jones, J. B.; Potnis, N.</p>
        <p class="info">Score: 8.1, Published: 2023-07-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.548846' target='https://doi.org/10.1101/2023.07.13.548846'> 10.1101/2023.07.13.548846</a></p>
        <p class="abstract">Type VI secretion system (T6SS) is a contact dependent contractile nano-weapon in Gram-negative bacteria that fires effector molecules directly into prokaryotic and eukaryotic cells aiding in manipulation of the host and killing of competitors in complex niches. In plant pathogenic xanthomonads, T6SS has been demonstrated to play diverse roles in individual pathosystems. However, the regulatory circuit involved in mediating biological functions carried out by T6SS are still elusive. To bridge this knowledge gap, we conducted an in vitro transcriptome screen using plant apoplast mimicking minimal medium, XVM2 medium, to decipher the effect of tssM deletion, a core gene belonging to T6SS-cluster i3*, on the regulation of gene expression in Xanthomonas perforans strain AL65. Transcriptomic data revealed that a total of 277 and 525 genes were upregulated, while 307 and 392 genes were downregulated in the mutant strain post 8 and 16 hours of growth in XVM2 medium. The transcript abundance of several genes associated with flagellum and pilus biogenesis as well as type III secretion system were downregulated in the mutant strain. Deletion of tssM of cluster-i3* resulted in upregulation of several T6SS genes belonging to cluster-i3*** and genes involved in biofilm and cell wall biogenesis. Transcription regulators like rpoN, Pho regulon, rpoE and csrA were identified to be upregulated in the mutant strain. Our results suggest that T6SS modulates the expression of global regulators like csrA, rpoN and pho regulons triggering a signaling cascade and co-ordinates the expression of suite of virulence factors, stress response genes and metabolic genes.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
