<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Observations of Elemental Composition of Enceladus Consistent with Generalized Models of Theoretical Ecosystems
Authors: Muratore, D.; Walker, S. I.; Graham, H.; House, C. H.; Kempes, C. P.
Score: 333.6, Published: 2023-10-29 DOI: 10.1101/2023.10.29.564608
Recent analysis of data from the Cassini Cosmic Dust Analyzer confirm geochemical modeling work that shows that the ocean of Enceladus contains considerable quantities of inorganic phosphorus as well as ammonium [55]. Technological advancement in flight instrumentation will continue to yield increasingly detailed data about the relative elemental and molecular composition of life detection candidates.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Observations of Elemental Composition of Enceladus Consistent with Generalized Models of Theoretical Ecosystems
Authors: Muratore, D.; Walker, S. I.; Graham, H.; House, C. H.; Kempes, C. P.
Score: 333.6, Published: 2023-10-29 DOI: 10.1101/2023.10.29.564608
Recent analysis of data from the Cassini Cosmic Dust Analyzer confirm geochemical modeling work that shows that the ocean of Enceladus contains considerable quantities of inorganic phosphorus as well as ammonium [55]. Technological advancement in flight instrumentation will continue to yield increasingly detailed data about the relative elemental and molecular composition of life detection candidates." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-12T10:37:43+00:00" />
<meta property="article:modified_time" content="2023-11-12T10:37:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Observations of Elemental Composition of Enceladus Consistent with Generalized Models of Theoretical Ecosystems
Authors: Muratore, D.; Walker, S. I.; Graham, H.; House, C. H.; Kempes, C. P.
Score: 333.6, Published: 2023-10-29 DOI: 10.1101/2023.10.29.564608
Recent analysis of data from the Cassini Cosmic Dust Analyzer confirm geochemical modeling work that shows that the ocean of Enceladus contains considerable quantities of inorganic phosphorus as well as ammonium [55]. Technological advancement in flight instrumentation will continue to yield increasingly detailed data about the relative elemental and molecular composition of life detection candidates."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Observations of Elemental Composition of Enceladus Consistent with Generalized Models of Theoretical Ecosystems\nAuthors: Muratore, D.; Walker, S. I.; Graham, H.; House, C. H.; Kempes, C. P.\nScore: 333.6, Published: 2023-10-29 DOI: 10.1101/2023.10.29.564608\nRecent analysis of data from the Cassini Cosmic Dust Analyzer confirm geochemical modeling work that shows that the ocean of Enceladus contains considerable quantities of inorganic phosphorus as well as ammonium [55]. Technological advancement in flight instrumentation will continue to yield increasingly detailed data about the relative elemental and molecular composition of life detection candidates.",
  "keywords": [
    
  ],
  "articleBody": " Observations of Elemental Composition of Enceladus Consistent with Generalized Models of Theoretical Ecosystems\nAuthors: Muratore, D.; Walker, S. I.; Graham, H.; House, C. H.; Kempes, C. P.\nScore: 333.6, Published: 2023-10-29 DOI: 10.1101/2023.10.29.564608\nRecent analysis of data from the Cassini Cosmic Dust Analyzer confirm geochemical modeling work that shows that the ocean of Enceladus contains considerable quantities of inorganic phosphorus as well as ammonium [55]. Technological advancement in flight instrumentation will continue to yield increasingly detailed data about the relative elemental and molecular composition of life detection candidates. Apart from speculating about threshold concentrations of bioactive compounds to support ecosystems, metabolic and ecological theory can provide a powerful interpretative lens to assess whether extraterrestrial environments are compatible with living ecosystems. Using multiple levels of ecological analysis, spanning from assuming strictly Earth-like organismal physiology to more agnostic understandings of putative biochemistries, we compare the proposed N:P stoichiometries of the Enceledus ocean to possible ecologies. We use chemostat models to predict potentially supportable biomass concentrations under different energy and matter flux regimes, macromolecular allometric theory to compare hypothetical biomass N:P ratios to possible environmental N:P supply ratios, and finally take a data-driven biogeochemical approach to predict possible biomass N:P ratios from the observed dissolved ratios. All three of our modeling approaches suggest marginal compatibility of an ecosystem with the ranges of dissolved N and P concentrations in the Enceledean ocean. Based on our analysis, we suggest two main priorities for further research into terrestrial analogs to improve our ability to interpret geochemical ratios as a life detection instrument.\nEmergence and spread of feline infectious peritonitis due to a highly pathogenic canine/feline recombinant coronavirus\nAuthors: Atippa, C.; Warr, A. S.; Epaminondas, D.; O'Shea, M.; Fletcher, S. L.; Malbon, A.; Lyraki, M.; Hammond, R.; Hardas, A.; Zanti, A.; Loukaidou, S.; Gentil, M.; Gunn-Moore, D.; Mazeri, S.; Tait-Burkard, C.\nScore: 84.4, Published: 2023-11-10 DOI: 10.1101/2023.11.08.566182\nCross-species transmission of coronaviruses (CoVs) poses a serious threat to both animal and human health. Whilst the large RNA genome of CoVs shows relatively low mutation rates, recombination within genera is frequently observed and demonstrated. Companion animals are often overlooked in the transmission cycle of viral diseases; however, the close relationship of feline (FCoV) and canine CoV (CCoV) to human hCoV-229E, as well as their susceptibility to SARS-CoV-2 highlight their importance in potential transmission cycles. Whilst recombination between CCoV and FCoV of a large fragment spanning orf1b to M has been previously described, here we report the emergence of a novel, highly pathogenic FCoV-CCoV recombinant responsible for a rapidly spreading outbreak of feline infectious peritonitis (FIP), originating in Cyprus. The recombination, spanning spike, shows 97% sequence identity to the pantropic canine coronavirus CB/05. Infection is spreading fast and infecting cats of all ages. Development of FIP appears rapid and likely non-reliant on biotype switch. High sequence identity of isolates from cats in different districts of the island is strongly supportive of direct transmission. A deletion and several amino acid changes in spike, particularly the receptor binding domain, compared to other FCoV-2s, indicate changes to receptor binding and likely cell tropism.\nVirological characteristics of the SARS-CoV-2 BA.2.86 variant\nAuthors: Tamura, T.; Mizuma, K.; Nasser, H.; Deguchi, S.; Padilla-Blanco, M.; Uriu, K.; Tolentino, J. E. M.; Tsujino, S.; Suzuki, R.; Kojima, I.; Nao, N.; Shimizu, R.; Jonathan, M.; Kosugi, Y.; Guo, Z.; Hinay, A. A.; Putri, O.; Kim, Y.; Tanaka, Y. L.; Asakura, H.; Nagashima, M.; Sadamasu, K.; Yoshimura, K.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Saito, A.; Ito, J.; Irie, T.; Zahradnik, J.; Ikeda, T.; Takayama, K.; Matsuno, K.; Fukuhara, T.; Sato, K.\nScore: 331.3, Published: 2023-11-03 DOI: 10.1101/2023.11.02.565304\nIn late 2023, a lineage of SARS-CoV-2 emerged and was named the BA.2.86 variant. BA.2.86 is phylogenetically distinct from other Omicron sublineages identified so far, displaying an accumulation of over 30 amino acid mutations in its spike protein. Here, we performed multiscale investigations to reveal the virological characteristics of the BA.2.86 variant. Our epidemic dynamics modeling suggested that the relative reproduction number of BA.2.86 is significantly higher than that of EG.5.1. Experimental studies showed that four clinically-available antivirals were effective against BA.2.86. Although the fusogenicity of BA.2.86 spike is similar to that of the parental BA.2 spike, the intrinsic pathogenicity of BA.2.86 in hamsters was significantly lower than that of BA.2. Since the growth kinetics of BA.2.86 is significantly lower than that of BA.2 in both in vitro cell cultures and in vivo, it is suggested that the attenuated pathogenicity of BA.2.86 is due to its decreased replication capacity.\nAn acidophilic fungus is integral to prey digestion in a carnivorous plant\nAuthors: Sun, P.-F.; Lu, M. R.; Liu, Y.-C.; Lin, Y.-f.; Hoh, D. Z.; Ke, H.-M.; Wang, I.-F.; Lu, M.-Y. J.; Kirschner, R.; Lin, Y.-C. J.; Chen, Y.-L.; Tsai, I. J.\nScore: 57.0, Published: 2023-11-08 DOI: 10.1101/2023.11.07.566145\nCarnivorous plant leaves, such as those of the spoon-leaved sundew Drosera spatulata, secrete mucilage which hosts microorganisms potentially aiding in prey digestion. We characterised the mucilage microbial communities and identified the acidophilic fungus Acrodontium crateriforme as the ecologically dominant species. The fungus grows and sporulates on sundew glands as its preferred acidic environment. We show that the A. crateriforme has a reduced genome similar to that of other symbiotic fungi. Based on the transcriptomes when encountering prey insects, we revealed a high degree of genes co-option in each species during fungus-plant coexistence and digestion. Expression patterns of the holobiont during digestion further revealed synergistic effects in several gene families including fungal aspartic and sedolisin peptidases, facilitating the digestion of sundew's prey, as well as transporters and dose-dependent responses in plant genes involved in jasmonate signalling pathway. This study establishes that botanical carnivory is defined by multidimensional adaptations correlated with interspecies interactions.\nDiscovery and characterization of the first known biological lanthanide chelator\nAuthors: Zytnick, A. M.; Gutenthaler, S. M.; Aron, A. T.; Reitz, Z. L.; Phi, M. T.; Good, N. M.; Petras, D.; Daumann, L. J.; Martinez-Gomez, N. C.\nScore: 59.0, Published: 2023-11-10 DOI: 10.1101/2022.01.19.476857\nMany bacteria secrete metallophores, low-molecular weight organic compounds that bind ions with high selectivity and affinity, in order to access essential metals from the environment.1 Previous work has elucidated the structures and biosynthetic machinery of metallophores specific for iron, zinc, nickel, molybdenum, and copper.1 No lanthanide-specific metallophore has been discovered despite the knowledge that lanthanide metals (Ln) have been revealed to be essential cofactors for certain alcohol dehydrogenases across a diverse range of phyla.2 Here, we report the biosynthetic machinery, the structure, and the physiological relevance of the first known lanthanophore, methylolanthanin. The structure of methylolanthanin exhibits a unique 4-hydroxybenzoate moiety which has not previously been described in other metallophores. We find that production of methylolanthanin is required for normal levels of Ln accumulation in the methylotrophic bacterium Methylobacterium extorquens AM1, while overexpression of the molecule greatly increases bioaccumulation. Our results provide a clearer understanding of how Ln-utilizing bacteria sense, scavenge, and store Ln; essential processes in the environment where Ln are poorly bioavailable. Beyond Ln, we anticipate our study to be a starting point for understanding how organisms acquire other f-block metals, the actinides.3 More broadly, the discovery of a lanthanophore opens doors for study of how biosynthetic gene clusters are repurposed for new functions, how metallophores acquire their metal specificity, and the complex relationship between metal homeostasis and fitness.\nSynergistic anti-phage activity of bacterial defence systems\nAuthors: Wu, Y.; Garushyants, S. K.; van den Hurk, A.; Aparicio-Maldonado, C.; Kushwaha, S. K.; King, C. M.; Ou, Y.; Todeschini, T. C.; Clokie, M. R. J.; Millard, A. M.; Gencay, Y. E.; Koonin, E. V.; Nobrega, F. L.\nScore: 62.3, Published: 2023-11-06 DOI: 10.1101/2022.08.21.504612\nBacterial defence against phage predation involves diverse defence systems acting both individually and concurrently, yet their interactions remain poorly understood. We investigated \u003e100 defence systems in 42,925 bacterial genomes and identified numerous instances of their non-random co-occurrence and mutual exclusion. For several pairs of defence systems significantly co-occurring in Escherichia coli strains, we demonstrate synergistic anti-phage activity, including tmn synergising with defence systems containing sensory switch ATPase domains. Some of the defence system pairs that are mutually exclusive in E. coli significantly co-occur in other taxa, and one tested mutually exclusive pair showed synergy. These findings demonstrate compatibility and synergy between defence systems, allowing bacteria to adopt flexible strategies for phage defence, driven by specific environmental pressures. The evolution of bacterial immune repertoires seems to be shaped largely by selection for resistance to host-specific phages rather than by negative epistasis.\nSensory kinase KdpD is a tandem serine histidine kinase controlling K+ pump KdpFABC on the translational and post-transcriptional level\nAuthors: Silberberg, J. M.; Ketter, S.; Boehm, P. J.; Jordan, K.; Wittenberg, M.; Grass, J.; Haenelt, I.\nScore: 18.8, Published: 2023-11-09 DOI: 10.1101/2023.11.09.566405\nTwo-component systems (TCSs), consisting of a histidine kinase (HK) and a response regulator, serve signal transduction in bacteria, often regulating transcription in response to environmental stimuli. Here, we identify a tandem serine histidine kinase function for KdpD, previously described as a HK of the TCS KdpDE, which controls production of K+ pump KdpFABC. We show that KdpD additionally mediates an inhibitory serine phosphorylation of KdpFABC at high K+ levels, using not its C-terminal HK domain but an N-terminal atypical serine kinase (ASK) domain. Sequence analysis of KdpDs from different species highlights that some KdpDs comprise solely ASK and Usp domains. We show that, while Escherichia coli KdpD's ASK responds directly to K+ levels, a shorter version from Deinococcus geothermalis is controlled by second messenger cyclic di-AMP. Our findings add to the growing functional diversity of sensor kinases while simultaneously expanding the framework for regulatory mechanisms in bacterial K+ homeostasis.\nA CRISPR interference system for the nitrogen-fixing bacterium Azotobacter vinelandii\nAuthors: Russell, S. J.; Garcia, A. K.; Kacar, B.\nScore: 17.5, Published: 2023-11-08 DOI: 10.1101/2023.11.07.565814\nA grand challenge for the next century can be found in mitigating the effects of changing climate through bioengineering solutions. Biological nitrogen fixation, the globally consequential, nitrogenase-catalyzed reduction of atmospheric nitrogen to bioavailable ammonia, is a particularly vital area of focus. Nitrogen fixation engineering relies upon extensive understanding of underlying genetics in microbial models, including the broadly utilized gammaproteobacterium, Azotobacter vinelandii (A. vinelandii). Here we report the first CRISPR interference (CRISPRi) system for targeted gene silencing in A. vinelandii that integrates genomically via site-specific transposon insertion. We demonstrate that CRISPRi can repress transcription of an essential nitrogen fixation gene by [~]60%. Further, we show that nitrogenase genes are suitably expressed from the transposon insertion site, indicating that CRISPRi and engineered nitrogen fixation genes can be co-integrated for combinatorial studies of gene expression and engineering. Our established CRISPRi system extends the utility of A. vinelandii and will aid efforts to engineer microbial nitrogen fixation for desired purposes. For Table of Contents Only O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=102 SRC=\"FIGDIR/small/565814v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (24K): org.highwire.dtl.DTLVardef@62667forg.highwire.dtl.DTLVardef@3735b0org.highwire.dtl.DTLVardef@a5e415org.highwire.dtl.DTLVardef@1214f3e_HPS_FORMAT_FIGEXP M_FIG C_FIG\nMultidrug resistance plasmids commonly reprogramme expression of metabolic genes in Escherichia coli\nAuthors: Hall, R. J.; Snaith, A. E.; Thomas, M.; Brockhurst, M.; McNally, A.\nScore: 13.2, Published: 2023-11-06 DOI: 10.1101/2023.11.06.565804\nMultidrug resistant Escherichia coli is a leading cause of global mortality. Transfer of plasmids carrying genes encoding beta-lactamases, carbapenamases, and colistin resistance genes between lineages is driving the rising rates of hard to treat nosocomial and community infections. Multidrug resistance (MDR) plasmid acquisition commonly causes transcriptional disruption, and whilst a number of studies have shown strain-specific fitness and transcriptional effects of an MDR plasmid across diverse bacterial lineages, fewer studies have compared impacts of different MDR plasmids in a common bacterial host. As such, our ability to predict which MDR plasmids are the most likely to be maintained and spread in bacterial populations is limited. Here, we introduced eight diverse MDR plasmids encoding resistances against a range of clinically important antibiotics into E. coli K-12 MG1655 and measured their fitness costs and transcriptional impacts. The scale of the transcriptional responses varied substantially between plasmids, ranging from \u003e650 to \u003c20 chromosomal genes being differentially expressed. However, neither the scale of regulatory disruption nor the plasmid size correlated with the magnitude of the plasmid fitness cost, which also varied between plasmids. The identities of differentially expressed genes varied among plasmids, although expression of certain metabolic genes and functions were convergently affected by multiple plasmids, including the downregulation of genes involved in L-methionine transport and metabolism. Our data show the complexity of interaction between host genetic background and plasmid genetic background in determining the impact of MDR plasmid acquisition on E. coli.\nCas13-based amplification-free quantification of aberrant RNAs during influenza virus infection\nAuthors: Lamb, C.; Pitre, E.; Elshina, E.; Rigby, C.; Bisht, K.; Jalal, H.; Myhrvold, C.; te Velthuis, A. J. W.\nScore: 14.3, Published: 2023-11-03 DOI: 10.1101/2023.11.03.565460\nInfluenza A virus RNA synthesis produces full-length and aberrant RNA molecules, which include defective viral genomes (DVG) and mini viral RNAs (mvRNA). Sequencing approaches have shown that several hundred unique aberrant RNA species may be present during infection, and that they can vary in size, segment origin, and sequence. Moreover, a subset of aberrant RNA molecules can bind and activate host pathogen receptor retinoic acid-inducible gene I (RIG-I), leading to innate immune signaling and the expression of type I and III interferons. Understanding the kinetics and distribution of these immunostimulatory aberrant RNA sequences is important for modeling the outcomes of IAV infection. We here first show that reverse transcription and PCR steps can yield imperfect aberrant RNA quantification data in a sequence-dependent manner. Next, we developed an amplification-free LbuCas13a-based detection method to quantify mvRNA amplification kinetics and subcellular distributions. We show that our assay can quantify the copy numbers of 10 specific mvRNA sequences in total RNA from cell culture, animal tissue or clinical nasopharyngeal swab extracts. In addition, we find kinetic and distribution differences between immunostimulatory and non-immunostimulatory mvRNAs, as well as mvRNAs derived from different segments, during infection. Overall, our results reveal a hitherto hidden diversity in the behavior of IAV mvRNAs and they suggest that their production is linked to replication of the individual viral segments. Cas13 is therefore a valuable new tool in our repertoire for investigating the impact of aberrant RNAs on RNA virus infection.\n",
  "wordCount" : "2341",
  "inLanguage": "en",
  "datePublished": "2023-11-12T10:37:43Z",
  "dateModified": "2023-11-12T10:37:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 12, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.29.564608">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.29.564608" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.29.564608">
        <p class="paperTitle">Observations of Elemental Composition of Enceladus Consistent with Generalized Models of Theoretical Ecosystems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.29.564608" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.29.564608" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Muratore, D.; Walker, S. I.; Graham, H.; House, C. H.; Kempes, C. P.</p>
        <p class="info">Score: 333.6, Published: 2023-10-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.29.564608' target='https://doi.org/10.1101/2023.10.29.564608'> 10.1101/2023.10.29.564608</a></p>
        <p class="abstract">Recent analysis of data from the Cassini Cosmic Dust Analyzer confirm geochemical modeling work that shows that the ocean of Enceladus contains considerable quantities of inorganic phosphorus as well as ammonium [55]. Technological advancement in flight instrumentation will continue to yield increasingly detailed data about the relative elemental and molecular composition of life detection candidates. Apart from speculating about threshold concentrations of bioactive compounds to support ecosystems, metabolic and ecological theory can provide a powerful interpretative lens to assess whether extraterrestrial environments are compatible with living ecosystems. Using multiple levels of ecological analysis, spanning from assuming strictly Earth-like organismal physiology to more agnostic understandings of putative biochemistries, we compare the proposed N:P stoichiometries of the Enceledus ocean to possible ecologies. We use chemostat models to predict potentially supportable biomass concentrations under different energy and matter flux regimes, macromolecular allometric theory to compare hypothetical biomass N:P ratios to possible environmental N:P supply ratios, and finally take a data-driven biogeochemical approach to predict possible biomass N:P ratios from the observed dissolved ratios. All three of our modeling approaches suggest marginal compatibility of an ecosystem with the ranges of dissolved N and P concentrations in the Enceledean ocean. Based on our analysis, we suggest two main priorities for further research into terrestrial analogs to improve our ability to interpret geochemical ratios as a life detection instrument.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.08.566182">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.08.566182" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.08.566182">
        <p class="paperTitle">Emergence and spread of feline infectious peritonitis due to a highly pathogenic canine/feline recombinant coronavirus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.08.566182" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.08.566182" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Atippa, C.; Warr, A. S.; Epaminondas, D.; O&#39;Shea, M.; Fletcher, S. L.; Malbon, A.; Lyraki, M.; Hammond, R.; Hardas, A.; Zanti, A.; Loukaidou, S.; Gentil, M.; Gunn-Moore, D.; Mazeri, S.; Tait-Burkard, C.</p>
        <p class="info">Score: 84.4, Published: 2023-11-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.08.566182' target='https://doi.org/10.1101/2023.11.08.566182'> 10.1101/2023.11.08.566182</a></p>
        <p class="abstract">Cross-species transmission of coronaviruses (CoVs) poses a serious threat to both animal and human health. Whilst the large RNA genome of CoVs shows relatively low mutation rates, recombination within genera is frequently observed and demonstrated. Companion animals are often overlooked in the transmission cycle of viral diseases; however, the close relationship of feline (FCoV) and canine CoV (CCoV) to human hCoV-229E, as well as their susceptibility to SARS-CoV-2 highlight their importance in potential transmission cycles. Whilst recombination between CCoV and FCoV of a large fragment spanning orf1b to M has been previously described, here we report the emergence of a novel, highly pathogenic FCoV-CCoV recombinant responsible for a rapidly spreading outbreak of feline infectious peritonitis (FIP), originating in Cyprus. The recombination, spanning spike, shows 97% sequence identity to the pantropic canine coronavirus CB/05. Infection is spreading fast and infecting cats of all ages. Development of FIP appears rapid and likely non-reliant on biotype switch. High sequence identity of isolates from cats in different districts of the island is strongly supportive of direct transmission. A deletion and several amino acid changes in spike, particularly the receptor binding domain, compared to other FCoV-2s, indicate changes to receptor binding and likely cell tropism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.565304">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.565304" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.565304">
        <p class="paperTitle">Virological characteristics of the SARS-CoV-2 BA.2.86 variant</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.565304" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.565304" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tamura, T.; Mizuma, K.; Nasser, H.; Deguchi, S.; Padilla-Blanco, M.; Uriu, K.; Tolentino, J. E. M.; Tsujino, S.; Suzuki, R.; Kojima, I.; Nao, N.; Shimizu, R.; Jonathan, M.; Kosugi, Y.; Guo, Z.; Hinay, A. A.; Putri, O.; Kim, Y.; Tanaka, Y. L.; Asakura, H.; Nagashima, M.; Sadamasu, K.; Yoshimura, K.; The Genotype to Phenotype Japan (G2P-Japan) Consortium,  ; Saito, A.; Ito, J.; Irie, T.; Zahradnik, J.; Ikeda, T.; Takayama, K.; Matsuno, K.; Fukuhara, T.; Sato, K.</p>
        <p class="info">Score: 331.3, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.565304' target='https://doi.org/10.1101/2023.11.02.565304'> 10.1101/2023.11.02.565304</a></p>
        <p class="abstract">In late 2023, a lineage of SARS-CoV-2 emerged and was named the BA.2.86 variant. BA.2.86 is phylogenetically distinct from other Omicron sublineages identified so far, displaying an accumulation of over 30 amino acid mutations in its spike protein. Here, we performed multiscale investigations to reveal the virological characteristics of the BA.2.86 variant. Our epidemic dynamics modeling suggested that the relative reproduction number of BA.2.86 is significantly higher than that of EG.5.1. Experimental studies showed that four clinically-available antivirals were effective against BA.2.86. Although the fusogenicity of BA.2.86 spike is similar to that of the parental BA.2 spike, the intrinsic pathogenicity of BA.2.86 in hamsters was significantly lower than that of BA.2. Since the growth kinetics of BA.2.86 is significantly lower than that of BA.2 in both in vitro cell cultures and in vivo, it is suggested that the attenuated pathogenicity of BA.2.86 is due to its decreased replication capacity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.566145">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.566145" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.566145">
        <p class="paperTitle">An acidophilic fungus is integral to prey digestion in a carnivorous plant</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.566145" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.566145" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sun, P.-F.; Lu, M. R.; Liu, Y.-C.; Lin, Y.-f.; Hoh, D. Z.; Ke, H.-M.; Wang, I.-F.; Lu, M.-Y. J.; Kirschner, R.; Lin, Y.-C. J.; Chen, Y.-L.; Tsai, I. J.</p>
        <p class="info">Score: 57.0, Published: 2023-11-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.566145' target='https://doi.org/10.1101/2023.11.07.566145'> 10.1101/2023.11.07.566145</a></p>
        <p class="abstract">Carnivorous plant leaves, such as those of the spoon-leaved sundew Drosera spatulata, secrete mucilage which hosts microorganisms potentially aiding in prey digestion. We characterised the mucilage microbial communities and identified the acidophilic fungus Acrodontium crateriforme as the ecologically dominant species. The fungus grows and sporulates on sundew glands as its preferred acidic environment. We show that the A. crateriforme has a reduced genome similar to that of other symbiotic fungi. Based on the transcriptomes when encountering prey insects, we revealed a high degree of genes co-option in each species during fungus-plant coexistence and digestion. Expression patterns of the holobiont during digestion further revealed synergistic effects in several gene families including fungal aspartic and sedolisin peptidases, facilitating the digestion of sundew&#39;s prey, as well as transporters and dose-dependent responses in plant genes involved in jasmonate signalling pathway. This study establishes that botanical carnivory is defined by multidimensional adaptations correlated with interspecies interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.01.19.476857">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.01.19.476857" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.01.19.476857">
        <p class="paperTitle">Discovery and characterization of the first known biological lanthanide chelator</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.01.19.476857" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.01.19.476857" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zytnick, A. M.; Gutenthaler, S. M.; Aron, A. T.; Reitz, Z. L.; Phi, M. T.; Good, N. M.; Petras, D.; Daumann, L. J.; Martinez-Gomez, N. C.</p>
        <p class="info">Score: 59.0, Published: 2023-11-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.01.19.476857' target='https://doi.org/10.1101/2022.01.19.476857'> 10.1101/2022.01.19.476857</a></p>
        <p class="abstract">Many bacteria secrete metallophores, low-molecular weight organic compounds that bind ions with high selectivity and affinity, in order to access essential metals from the environment.1 Previous work has elucidated the structures and biosynthetic machinery of metallophores specific for iron, zinc, nickel, molybdenum, and copper.1 No lanthanide-specific metallophore has been discovered despite the knowledge that lanthanide metals (Ln) have been revealed to be essential cofactors for certain alcohol dehydrogenases across a diverse range of phyla.2 Here, we report the biosynthetic machinery, the structure, and the physiological relevance of the first known lanthanophore, methylolanthanin. The structure of methylolanthanin exhibits a unique 4-hydroxybenzoate moiety which has not previously been described in other metallophores. We find that production of methylolanthanin is required for normal levels of Ln accumulation in the methylotrophic bacterium Methylobacterium extorquens AM1, while overexpression of the molecule greatly increases bioaccumulation. Our results provide a clearer understanding of how Ln-utilizing bacteria sense, scavenge, and store Ln; essential processes in the environment where Ln are poorly bioavailable. Beyond Ln, we anticipate our study to be a starting point for understanding how organisms acquire other f-block metals, the actinides.3 More broadly, the discovery of a lanthanophore opens doors for study of how biosynthetic gene clusters are repurposed for new functions, how metallophores acquire their metal specificity, and the complex relationship between metal homeostasis and fitness.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.08.21.504612">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.08.21.504612" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.08.21.504612">
        <p class="paperTitle">Synergistic anti-phage activity of bacterial defence systems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.08.21.504612" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.08.21.504612" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wu, Y.; Garushyants, S. K.; van den Hurk, A.; Aparicio-Maldonado, C.; Kushwaha, S. K.; King, C. M.; Ou, Y.; Todeschini, T. C.; Clokie, M. R. J.; Millard, A. M.; Gencay, Y. E.; Koonin, E. V.; Nobrega, F. L.</p>
        <p class="info">Score: 62.3, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.08.21.504612' target='https://doi.org/10.1101/2022.08.21.504612'> 10.1101/2022.08.21.504612</a></p>
        <p class="abstract">Bacterial defence against phage predation involves diverse defence systems acting both individually and concurrently, yet their interactions remain poorly understood. We investigated &gt;100 defence systems in 42,925 bacterial genomes and identified numerous instances of their non-random co-occurrence and mutual exclusion. For several pairs of defence systems significantly co-occurring in Escherichia coli strains, we demonstrate synergistic anti-phage activity, including tmn synergising with defence systems containing sensory switch ATPase domains. Some of the defence system pairs that are mutually exclusive in E. coli significantly co-occur in other taxa, and one tested mutually exclusive pair showed synergy. These findings demonstrate compatibility and synergy between defence systems, allowing bacteria to adopt flexible strategies for phage defence, driven by specific environmental pressures. The evolution of bacterial immune repertoires seems to be shaped largely by selection for resistance to host-specific phages rather than by negative epistasis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.566405">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.566405" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.566405">
        <p class="paperTitle">Sensory kinase KdpD is a tandem serine histidine kinase controlling K&#43; pump KdpFABC on the translational and post-transcriptional level</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.566405" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.566405" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Silberberg, J. M.; Ketter, S.; Boehm, P. J.; Jordan, K.; Wittenberg, M.; Grass, J.; Haenelt, I.</p>
        <p class="info">Score: 18.8, Published: 2023-11-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.566405' target='https://doi.org/10.1101/2023.11.09.566405'> 10.1101/2023.11.09.566405</a></p>
        <p class="abstract">Two-component systems (TCSs), consisting of a histidine kinase (HK) and a response regulator, serve signal transduction in bacteria, often regulating transcription in response to environmental stimuli. Here, we identify a tandem serine histidine kinase function for KdpD, previously described as a HK of the TCS KdpDE, which controls production of K&#43; pump KdpFABC. We show that KdpD additionally mediates an inhibitory serine phosphorylation of KdpFABC at high K&#43; levels, using not its C-terminal HK domain but an N-terminal atypical serine kinase (ASK) domain. Sequence analysis of KdpDs from different species highlights that some KdpDs comprise solely ASK and Usp domains. We show that, while Escherichia coli KdpD&#39;s ASK responds directly to K&#43; levels, a shorter version from Deinococcus geothermalis is controlled by second messenger cyclic di-AMP. Our findings add to the growing functional diversity of sensor kinases while simultaneously expanding the framework for regulatory mechanisms in bacterial K&#43; homeostasis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.565814">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.565814" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.565814">
        <p class="paperTitle">A CRISPR interference system for the nitrogen-fixing bacterium Azotobacter vinelandii</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.565814" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.565814" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Russell, S. J.; Garcia, A. K.; Kacar, B.</p>
        <p class="info">Score: 17.5, Published: 2023-11-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.565814' target='https://doi.org/10.1101/2023.11.07.565814'> 10.1101/2023.11.07.565814</a></p>
        <p class="abstract">A grand challenge for the next century can be found in mitigating the effects of changing climate through bioengineering solutions. Biological nitrogen fixation, the globally consequential, nitrogenase-catalyzed reduction of atmospheric nitrogen to bioavailable ammonia, is a particularly vital area of focus. Nitrogen fixation engineering relies upon extensive understanding of underlying genetics in microbial models, including the broadly utilized gammaproteobacterium, Azotobacter vinelandii (A. vinelandii). Here we report the first CRISPR interference (CRISPRi) system for targeted gene silencing in A. vinelandii that integrates genomically via site-specific transposon insertion. We demonstrate that CRISPRi can repress transcription of an essential nitrogen fixation gene by [~]60%. Further, we show that nitrogenase genes are suitably expressed from the transposon insertion site, indicating that CRISPRi and engineered nitrogen fixation genes can be co-integrated for combinatorial studies of gene expression and engineering. Our established CRISPRi system extends the utility of A. vinelandii and will aid efforts to engineer microbial nitrogen fixation for desired purposes.

For Table of Contents Only

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=102 SRC=&#34;FIGDIR/small/565814v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (24K):
org.highwire.dtl.DTLVardef@62667forg.highwire.dtl.DTLVardef@3735b0org.highwire.dtl.DTLVardef@a5e415org.highwire.dtl.DTLVardef@1214f3e_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565804">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565804" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565804">
        <p class="paperTitle">Multidrug resistance plasmids commonly reprogramme expression of metabolic genes in Escherichia coli</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565804" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565804" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hall, R. J.; Snaith, A. E.; Thomas, M.; Brockhurst, M.; McNally, A.</p>
        <p class="info">Score: 13.2, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565804' target='https://doi.org/10.1101/2023.11.06.565804'> 10.1101/2023.11.06.565804</a></p>
        <p class="abstract">Multidrug resistant Escherichia coli is a leading cause of global mortality. Transfer of plasmids carrying genes encoding beta-lactamases, carbapenamases, and colistin resistance genes between lineages is driving the rising rates of hard to treat nosocomial and community infections. Multidrug resistance (MDR) plasmid acquisition commonly causes transcriptional disruption, and whilst a number of studies have shown strain-specific fitness and transcriptional effects of an MDR plasmid across diverse bacterial lineages, fewer studies have compared impacts of different MDR plasmids in a common bacterial host. As such, our ability to predict which MDR plasmids are the most likely to be maintained and spread in bacterial populations is limited. Here, we introduced eight diverse MDR plasmids encoding resistances against a range of clinically important antibiotics into E. coli K-12 MG1655 and measured their fitness costs and transcriptional impacts. The scale of the transcriptional responses varied substantially between plasmids, ranging from &gt;650 to &lt;20 chromosomal genes being differentially expressed. However, neither the scale of regulatory disruption nor the plasmid size correlated with the magnitude of the plasmid fitness cost, which also varied between plasmids. The identities of differentially expressed genes varied among plasmids, although expression of certain metabolic genes and functions were convergently affected by multiple plasmids, including the downregulation of genes involved in L-methionine transport and metabolism. Our data show the complexity of interaction between host genetic background and plasmid genetic background in determining the impact of MDR plasmid acquisition on E. coli.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.03.565460">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.03.565460" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.03.565460">
        <p class="paperTitle">Cas13-based amplification-free quantification of aberrant RNAs during influenza virus infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.03.565460" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.03.565460" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lamb, C.; Pitre, E.; Elshina, E.; Rigby, C.; Bisht, K.; Jalal, H.; Myhrvold, C.; te Velthuis, A. J. W.</p>
        <p class="info">Score: 14.3, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.03.565460' target='https://doi.org/10.1101/2023.11.03.565460'> 10.1101/2023.11.03.565460</a></p>
        <p class="abstract">Influenza A virus RNA synthesis produces full-length and aberrant RNA molecules, which include defective viral genomes (DVG) and mini viral RNAs (mvRNA). Sequencing approaches have shown that several hundred unique aberrant RNA species may be present during infection, and that they can vary in size, segment origin, and sequence. Moreover, a subset of aberrant RNA molecules can bind and activate host pathogen receptor retinoic acid-inducible gene I (RIG-I), leading to innate immune signaling and the expression of type I and III interferons. Understanding the kinetics and distribution of these immunostimulatory aberrant RNA sequences is important for modeling the outcomes of IAV infection. We here first show that reverse transcription and PCR steps can yield imperfect aberrant RNA quantification data in a sequence-dependent manner. Next, we developed an amplification-free LbuCas13a-based detection method to quantify mvRNA amplification kinetics and subcellular distributions. We show that our assay can quantify the copy numbers of 10 specific mvRNA sequences in total RNA from cell culture, animal tissue or clinical nasopharyngeal swab extracts. In addition, we find kinetic and distribution differences between immunostimulatory and non-immunostimulatory mvRNAs, as well as mvRNAs derived from different segments, during infection. Overall, our results reveal a hitherto hidden diversity in the behavior of IAV mvRNAs and they suggest that their production is linked to replication of the individual viral segments. Cas13 is therefore a valuable new tool in our repertoire for investigating the impact of aberrant RNAs on RNA virus infection.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
