<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="SARS-CoV-2 spike glycosylation affects function and neutralization sensitivity
Authors: Zhang, F.; Schmidt, F.; Muecksch, F.; Wang, Z.; Gazumyan, A.; Nussenzweig, M. C.; Gaebler, C.; Caskey, M.; Hatziioannou, T.; Bieniasz, P. D.
Score: 81.1, Published: 2023-06-30 DOI: 10.1101/2023.06.30.547241
The glycosylation of viral envelope proteins can play important roles in virus biology and immune evasion. The spike (S) glycoprotein of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) includes 22 N-linked glycosylation sequons and 17 O-linked glycosites.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="SARS-CoV-2 spike glycosylation affects function and neutralization sensitivity
Authors: Zhang, F.; Schmidt, F.; Muecksch, F.; Wang, Z.; Gazumyan, A.; Nussenzweig, M. C.; Gaebler, C.; Caskey, M.; Hatziioannou, T.; Bieniasz, P. D.
Score: 81.1, Published: 2023-06-30 DOI: 10.1101/2023.06.30.547241
The glycosylation of viral envelope proteins can play important roles in virus biology and immune evasion. The spike (S) glycoprotein of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) includes 22 N-linked glycosylation sequons and 17 O-linked glycosites." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-09T10:40:17+00:00" />
<meta property="article:modified_time" content="2023-07-09T10:40:17+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="SARS-CoV-2 spike glycosylation affects function and neutralization sensitivity
Authors: Zhang, F.; Schmidt, F.; Muecksch, F.; Wang, Z.; Gazumyan, A.; Nussenzweig, M. C.; Gaebler, C.; Caskey, M.; Hatziioannou, T.; Bieniasz, P. D.
Score: 81.1, Published: 2023-06-30 DOI: 10.1101/2023.06.30.547241
The glycosylation of viral envelope proteins can play important roles in virus biology and immune evasion. The spike (S) glycoprotein of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) includes 22 N-linked glycosylation sequons and 17 O-linked glycosites."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "SARS-CoV-2 spike glycosylation affects function and neutralization sensitivity\nAuthors: Zhang, F.; Schmidt, F.; Muecksch, F.; Wang, Z.; Gazumyan, A.; Nussenzweig, M. C.; Gaebler, C.; Caskey, M.; Hatziioannou, T.; Bieniasz, P. D.\nScore: 81.1, Published: 2023-06-30 DOI: 10.1101/2023.06.30.547241\nThe glycosylation of viral envelope proteins can play important roles in virus biology and immune evasion. The spike (S) glycoprotein of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) includes 22 N-linked glycosylation sequons and 17 O-linked glycosites.",
  "keywords": [
    
  ],
  "articleBody": " SARS-CoV-2 spike glycosylation affects function and neutralization sensitivity\nAuthors: Zhang, F.; Schmidt, F.; Muecksch, F.; Wang, Z.; Gazumyan, A.; Nussenzweig, M. C.; Gaebler, C.; Caskey, M.; Hatziioannou, T.; Bieniasz, P. D.\nScore: 81.1, Published: 2023-06-30 DOI: 10.1101/2023.06.30.547241\nThe glycosylation of viral envelope proteins can play important roles in virus biology and immune evasion. The spike (S) glycoprotein of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) includes 22 N-linked glycosylation sequons and 17 O-linked glycosites. Here, we investigated the effect of individual glycosylation sites on SARS-CoV-2 S function in pseudotyped virus infection assays and on sensitivity to monoclonal and polyclonal neutralizing antibodies. In most cases, removal of individual glycosylation sites decreased the infectiousness of the pseudotyped virus. For glycosylation mutants in the N-terminal domain (NTD) and the receptor binding domain (RBD), reduction in pseudotype infectivity was predicted by a commensurate reduction in the level of virion-incorporated spike protein. Notably, the presence of a glycan at position N343 within the RBD had diverse effects on neutralization by RBD-specific monoclonal antibodies (mAbs) cloned from convalescent individuals. The N343 glycan reduced overall sensitivity to polyclonal antibodies in plasma from COVID-19 convalescent individuals, suggesting a role for SARS-CoV-2 spike glycosylation in immune evasion. However, vaccination of convalescent individuals produced neutralizing activity that was resilient to the inhibitory effect of the N343 glycan.\nFirst Eurasian cases of SARS-CoV-2 seropositivity in a free-ranging urban population of wild fallow deer\nAuthors: Purves, K.; Brown, H.; Haverty, R.; Ryan, A.; Griffin, L.; McCormack, J.; O'Reilly, S. R.; Mallon, P. W.; Gautier, V.; Cassidy, J. P.; Fabre, A.; Carr, M. J.; Gonzalez, G.; Ciuti, S.; Fletcher, N.\nScore: 47.1, Published: 2023-07-07 DOI: 10.1101/2023.07.07.547941\nSevere acute respiratory syndrome coronavirus-2 (SARS-CoV-2) infects wildlife. Recent studies highlighted that variants of concern (VOC) may expand into novel animal reservoirs with the potential for reverse zoonosis. North American white-tailed deer are the only deer species in which SARS-CoV-2 has been documented, raising the question whether further reservoir species exist as new VOC emerge. Here, we report the first cases of deer SARS-CoV-2 seropositivity in Eurasia, in a city population of fallow deer in Dublin, Ireland. Deer were seronegative in 2020 (circulating variant in humans: Alpha), one animal was seropositive in 2021 (Delta variant), and 57% of animals tested in 2022 were seropositive (Omicron variant). Ex vivo, a clinical isolate of Omicron BA.1 infected fallow deer precision cut lung slice type-2 pneumocytes, also a major target of infection in human lungs. Our findings suggest a change in host tropism as new variants emerged in the human reservoir, highlighting the importance of continued wildlife disease monitoring and limiting human-wildlife contacts.\nDistinct groups of RNA viruses associated with thermoacidophilic bacteria\nAuthors: Urayama, S.-i.; Fukudome, A.; Hirai, M.; Okumura, T.; Nishimura, Y.; Takaki, Y.; Kurosawa, N.; Koonin, E. V.; Krupovic, M.; Nunoura, T.\nScore: 41.9, Published: 2023-07-03 DOI: 10.1101/2023.07.02.547447\nRecent massive metatranscriptome mining substantially expanded the diversity of the bacterial RNA virome, suggesting that additional groups of riboviruses infecting bacterial hosts remain to be discovered. We employed full length double-stranded (ds) RNA sequencing for identification of riboviruses associated with microbial consortia dominated by bacteria and archaea in acidic hot springs in Japan. Whole sequences of two groups of multisegmented riboviruses genomes were obtained. One group, which we denoted hot spring riboviruses (HsRV), consists of unusual viruses with distinct RNA-dependent RNA polymerases (RdRPs) that seem to be intermediates between typical ribovirus RdRPs and viral reverse transcriptases. We also identified viruses encoding HsRV-like RdRPs in moderate aquatic environments, including marine water, river sediments and salt marsh, indicating that this previously overlooked ribovirus group is not restricted to the extreme ecosystem. The HsRV-like viruses are candidates for a distinct phylum or even kingdom within the viral realm Riboviria. The second group, denoted hot spring partiti-like viruses (HsPV), is a distinct branch within the family Partitiviridae. All genome segments in both these groups of viruses display the organization typical of bacterial riboviruses, where multiple open reading frames encoding individual proteins are preceded by ribosome-binding sites. Together with the identification in bacteria-dominated habitats, this genome architecture indicates that riboviruses of these distinct groups infect thermoacidophilic bacterial hosts.\nHoming in on the rare virosphere reveals the native host of giant viruses\nAuthors: Fromm, A.; Hevroni, G.; Vincent, F.; Schatz, D.; Martinez-Gutierrez, C. A.; Aylward, F. O.; Vardi, A.\nScore: 27.8, Published: 2023-06-27 DOI: 10.1101/2023.06.27.546645\nGiant viruses (phylum Nucleocytoviricota) are globally distributed in aquatic ecosystems1,2. They play major roles as evolutionary drivers of eukaryotic plankton3 and regulators of global biogeochemical cycles4. Recent metagenomic studies have significantly expanded the known diversity of marine giant viruses1,5-7, but we still lack fundamental knowledge about their native hosts, thereby hindering our understanding of their lifecycle and ecological importance. Here, we aim to discover the native hosts of giant viruses using a novel, sensitive single-cell metatranscriptomic approach. By applying this approach to natural plankton communities, we unraveled an active viral infection of several giant viruses, from multiple lineages, and identified their native hosts. We identify a rare lineage of giant virus (Imitervirales-07) infecting a minute population of protists (class Katablepharidaceae) and revealed the prevalence of highly expressed viral-encoded cell-fate regulation genes in infected cells. Further examination of this host-virus dynamics in a temporal resolution suggested this giant virus controls its host population demise. Our results demonstrate how single-cell metatranscriptomics is a sensitive approach for pairing viruses with their authentic hosts and studying their ecological significance in a culture-independent manner in the marine environment.\nAmazing structural diversity of giant virus-like particles in forest soil\nAuthors: Fischer, M. G.; Mersdorf, U.; Blanchard, J. L.\nScore: 121.7, Published: 2023-06-30 DOI: 10.1101/2023.06.30.546935\nLarge DNA viruses of the phylum Nucleocytoviricota infect diverse eukaryotic hosts from protists to humans, with profound consequences for aquatic and terrestrial ecosystems. While nucleocytoviruses are known to be highly diverse in metagenomes, knowledge of their capsid structures is restricted to a few characterized representatives. Here, we visualize giant virus-like particles (VLPs, diameter \u003e0.2 {micro}m) directly from the environment using transmission electron microscopy. We found that Harvard Forest soils contain a higher diversity of giant VLP morphotypes than all hitherto isolated giant viruses combined. These included VLPs with icosahedral capsid symmetry, ovoid shapes similar to pandoraviruses, and bacilliform shapes that may represent novel viruses. We discovered giant icosahedral capsids with structural modifications that had not been described before including tubular appendages, modified vertices, tails, and capsids consisting of multiple layers or internal channels. Many giant VLPs were covered with fibers of varying lengths, thicknesses, densities, and terminal structures. These findings imply that giant viruses employ a much wider array of capsid structures and mechanisms to interact with their host cells than is currently known. We also found diverse tailed bacteriophages and filamentous VLPs, as well as ultra-small cells. Our study offers a first glimpse of the vast diversity of unexplored viral structures in soil and reinforces the potential of transmission electron microscopy for fundamental discoveries in environmental microbiology.\nBam complex associated proteins in Escherichia coli are functionally linked to peptidoglycan biosynthesis, membrane fluidity and DNA replication\nAuthors: Bryant, J. A.; Staunton, K. A.; Doherty, H. M.; Alao, M. B.; Ma, X.; Morcinek-Orlowska, J.; Goodall, E. C. A.; Gray, J.; Milner, M.; Cole, J. A.; de Cogan, F.; Knowles, T. J.; Glinkowska, M. K.; Moradigaravand, D.; Henderson, I. R.; Banzhaf, M.\nScore: 22.9, Published: 2023-07-05 DOI: 10.1101/2023.07.05.547807\nBiogenesis of the bacterial outer membrane is key to survival and antibiotic resistance. Central to this is the beta-barrel assembly machine (Bam) complex and its associated chaperones, which are responsible for outer membrane protein (OMP) transport and insertion. The Escherichia coli Bam complex consists of two essential subunits, BamA and BamD, and three non-essential lipoproteins, BamB, BamC and BamE. Optimal Bam function is further dependent on the non-essential chaperones DegP, Skp and SurA. Despite intensive study, the specific function of these non-essential Bam-associated proteins remains unknown. Here, we analysed knockout strains for each gene by phenotypic screening, conservation analysis and high-throughput genetics. We reveal that Bam activity is affected by outer membrane lipid composition and that enterobacterial common antigen is essential in the absence of the chaperone SurA. We also show that components of peptidoglycan are conditionally essential with Bam accessory lipoproteins and that DNA replication is perturbed in the absence of BamB. Together, our data indicates potential mechanisms for coordination of OMP biogenesis with processes such as LPS and peptidoglycan biogenesis, and DNA replication.\nPredicting bacterial interaction outcomes from monoculture growth and supernatant assays\nAuthors: Schmitz, D. A.; Wechsler, T.; Mignot, I.; Kuemmerli, R.\nScore: 17.9, Published: 2023-07-03 DOI: 10.1101/2023.07.03.547502\nHow to derive principles of community dynamics and stability is a central question in microbial ecology. To answer this, bottom-up experimental approaches, in which a small number of bacterial species are mixed, have become popular. However, experimental setups are typically limited because species are difficult to distinguish in mixed cultures and co-culture experiments are labor-intensive. Here, we use a community of four bacterial species to show that information from monoculture growth and inhibitory effects caused by secreted compounds can be combined to reliably predict pairwise species interactions in co-cultures. Specifically, integrative parameters from growth curves allow to build a competitive rank order, which is then adjusted using inhibitory compound effects from supernatant assays. While our procedure worked for two media examined, we observed differences in species rank orders between media. We further used computer simulations, parameterized with our empirical data, to show that higher-order species interactions largely follow the dynamics predicted from pairwise interactions with one important exception. The impact of inhibitory compounds was reduced in higher-order communities because these compounds are spread across multiple species, thereby diluting their effects. Altogether, our results lead to the formulation of three simple rules of how monoculture growth and supernatant assay data can be combined to establish competitive species rank orders in bacterial communities.\nInterrogation of RNA-protein interaction dynamics in bacterial growth\nAuthors: Monti, M.; Herman, R.; Davey, K.; Dawson, C. S.; Ule, J.; Thomas, G. H.; Willis, A. E.; Lilley, K. S.; Villanueva, E.\nScore: 15.3, Published: 2023-07-03 DOI: 10.1101/2023.07.03.547468\nCharacterising RNA-protein interaction dynamics is fundamental to understand how bacteria respond to their environment. In this study, we have analysed the dynamics of 91% of the Escherichia coli expressed proteome and the RNA-interaction properties of 271 RNA-binding proteins (RBPs) at different growth phases. We find that 68% of RBPs differentially bind RNA across growth phases and characterise 17 previously unannotated proteins as bacterial RBPs including YfiF, a ncRNA-binding protein. While these new RBPs are mostly present in Proteobacteria, two of them have human orthologs in the form of mitochondrial proteins associated with rare metabolic disorders. Moreover, we reveal novel RBP functions for proteins such as the chaperone HtpG, a new stationary phase tRNA-binding protein. Altogether, we provide the first dynamic RBPome of a bacterium, showcasing how this approach can reveal the function of uncharacterised proteins, and identify critical RNA-protein interactions for cell growth which could inform new antimicrobial therapies.\nYeast diversity in open agave fermentations across Mexico\nAuthors: Gallegos-Gasillas, P.; Garcia-Ortega, L. F.; Espinosa-Cantu, A.; Torres-Lagunes, C. G.; Avelar-Rivas, J. A.; Cano-Ricardez, A.; Garcia-Acero, A. M.; Ruiz-Castro, S.; Flores-Barraza, M.; Castillo, A.; Gonzalez-Zozaya, F.; Delgado-Lemus, A.; Molina-Freaner, F.; Jacques-Hernandez, C.; Hernandez-Lopez, A.; Delaye, L.; Aguirre-Dugua, X.; Kirchmayr, M. R.; Morales, L.; Mancera, E.; DeLuna, A.\nScore: 15.2, Published: 2023-07-03 DOI: 10.1101/2023.07.02.547337\nYeasts are a diverse group of fungal microorganisms that are widely used to produce fermented foods and beverages. In Mexico, open fermentations are used to obtain spirits from agave plants. Despite the prevalence of this traditional practice throughout the country, yeasts have only been isolated and studied from a limited number of distilleries. To systematically describe the diversity of yeast species from open agave fermentations, here we generate the YMX.1.0 culture collection by isolating 4,524 strains from 68 sites in diverse climatic, geographical, and biological contexts. We used MALDI-TOF mass spectrometry for taxonomic classification and a subset of the strains was verified by ITS and D1/D2 sequencing. The most abundant species isolated from all producing regions were Saccharomyces cerevisiae, Pichia kudriavzevii, Pichia manshurica, and Kluyveromyces marxianus. Despite the great diversity of environmental conditions and production practices the composition of yeast communities remained largely homogeneous throughout locations and fermentation stages, even if less abundant but commonly occurring yeasts were considered. Furthermore, ITS and D1/D2 sequencing revealed two candidate new species of Saccharomycetales. To explore the intraspecific variation of the yeasts from agave fermentations, we conducted genome sequencing on four isolates of the non-conventional yeast Kazachstania humilis. The genomes of these four strains were considerably distinct from other genomes of the same species, suggesting that they belong to a different population. Our work contributes to the understanding and conservation of an open fermentation system of great cultural and economic importance, providing a valuable resource to study the biology and genetic diversity of microorganisms living at the interface of natural and human-associated environments. TAKE AWAYO_LIWe isolated and identified 4,524 yeast strains from open agave fermentations in Mexico. C_LIO_LIYeast communities remained largely homogeneous throughout diverse locations. C_LIO_LIKazachstania humilis genomes differed significantly from isolates in other regions of the world. C_LIO_LIWe report two candidate new species related to the Pichia clade. C_LI\nCovalent macrocyclic proteasome inhibitors mitigate resistance in Plasmodium falciparum\nAuthors: Bennett, J. M.; Ward, K. E.; Muir, R.; Kabeche, S.; Yoo, E.; Yeo, T.; Lam, G.; Zhang, H.; Almaliti, J.; Berger, G.; Faucher, F. F.; Lin, G.; Gerwick, W. H.; Yeh, E.; Fidock, D. A.; Bogyo, M.\nScore: 15.0, Published: 2023-07-04 DOI: 10.1101/2023.07.03.547579\nThe Plasmodium proteasome is a promising antimalarial drug target due to its essential role in all parasite lifecycle stages. Furthermore, proteasome inhibitors have synergistic effects when combined with current first-line artemisinins. Linear peptides that covalently inhibit the proteasome are effective at killing parasites and have a low propensity for inducing resistance. However, these scaffolds generally suffer from poor pharmacokinetics and bioavailability. Here we describe the development of covalent, irreversible macrocyclic inhibitors of the P. falciparum proteasome. We identified compounds with excellent potency and low cytotoxicity, however, the first generation suffered from poor microsomal stability. Further optimization of an existing macrocyclic scaffold resulted in an irreversible covalent inhibitor carrying a vinyl sulfone electrophile that retained high potency, low cytotoxicity, and had acceptable metabolic stability. Importantly, unlike the parent reversible inhibitor that selected for multiple mutations in the proteasome, with one resulting in a 5,000-fold loss of potency, the irreversible analog only showed a 5-fold loss in potency for any single point mutation. Furthermore, an epoxyketone analog of the same scaffold retained potency against a panel of known proteasome mutants. These results confirm that macrocycles are optimal scaffolds to target the malarial proteasome and that the use of a covalent electrophile can greatly reduce the ability of the parasite to generate drug resistance mutations.\n",
  "wordCount" : "2444",
  "inLanguage": "en",
  "datePublished": "2023-07-09T10:40:17Z",
  "dateModified": "2023-07-09T10:40:17Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta"><span>updated on July 9, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547241">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547241" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547241">
        <p class="paperTitle">SARS-CoV-2 spike glycosylation affects function and neutralization sensitivity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547241" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547241" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, F.; Schmidt, F.; Muecksch, F.; Wang, Z.; Gazumyan, A.; Nussenzweig, M. C.; Gaebler, C.; Caskey, M.; Hatziioannou, T.; Bieniasz, P. D.</p>
        <p class="info">Score: 81.1, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547241' target='https://doi.org/10.1101/2023.06.30.547241'> 10.1101/2023.06.30.547241</a></p>
        <p class="abstract">The glycosylation of viral envelope proteins can play important roles in virus biology and immune evasion. The spike (S) glycoprotein of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) includes 22 N-linked glycosylation sequons and 17 O-linked glycosites. Here, we investigated the effect of individual glycosylation sites on SARS-CoV-2 S function in pseudotyped virus infection assays and on sensitivity to monoclonal and polyclonal neutralizing antibodies. In most cases, removal of individual glycosylation sites decreased the infectiousness of the pseudotyped virus. For glycosylation mutants in the N-terminal domain (NTD) and the receptor binding domain (RBD), reduction in pseudotype infectivity was predicted by a commensurate reduction in the level of virion-incorporated spike protein. Notably, the presence of a glycan at position N343 within the RBD had diverse effects on neutralization by RBD-specific monoclonal antibodies (mAbs) cloned from convalescent individuals. The N343 glycan reduced overall sensitivity to polyclonal antibodies in plasma from COVID-19 convalescent individuals, suggesting a role for SARS-CoV-2 spike glycosylation in immune evasion. However, vaccination of convalescent individuals produced neutralizing activity that was resilient to the inhibitory effect of the N343 glycan.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.547941">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.547941" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.547941">
        <p class="paperTitle">First Eurasian cases of SARS-CoV-2 seropositivity in a free-ranging urban population of wild fallow deer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.547941" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.547941" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Purves, K.; Brown, H.; Haverty, R.; Ryan, A.; Griffin, L.; McCormack, J.; O&#39;Reilly, S. R.; Mallon, P. W.; Gautier, V.; Cassidy, J. P.; Fabre, A.; Carr, M. J.; Gonzalez, G.; Ciuti, S.; Fletcher, N.</p>
        <p class="info">Score: 47.1, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.547941' target='https://doi.org/10.1101/2023.07.07.547941'> 10.1101/2023.07.07.547941</a></p>
        <p class="abstract">Severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) infects wildlife. Recent studies highlighted that variants of concern (VOC) may expand into novel animal reservoirs with the potential for reverse zoonosis. North American white-tailed deer are the only deer species in which SARS-CoV-2 has been documented, raising the question whether further reservoir species exist as new VOC emerge. Here, we report the first cases of deer SARS-CoV-2 seropositivity in Eurasia, in a city population of fallow deer in Dublin, Ireland. Deer were seronegative in 2020 (circulating variant in humans: Alpha), one animal was seropositive in 2021 (Delta variant), and 57% of animals tested in 2022 were seropositive (Omicron variant). Ex vivo, a clinical isolate of Omicron BA.1 infected fallow deer precision cut lung slice type-2 pneumocytes, also a major target of infection in human lungs. Our findings suggest a change in host tropism as new variants emerged in the human reservoir, highlighting the importance of continued wildlife disease monitoring and limiting human-wildlife contacts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.02.547447">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.02.547447" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.02.547447">
        <p class="paperTitle">Distinct groups of RNA viruses associated with thermoacidophilic bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.02.547447" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.02.547447" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Urayama, S.-i.; Fukudome, A.; Hirai, M.; Okumura, T.; Nishimura, Y.; Takaki, Y.; Kurosawa, N.; Koonin, E. V.; Krupovic, M.; Nunoura, T.</p>
        <p class="info">Score: 41.9, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.02.547447' target='https://doi.org/10.1101/2023.07.02.547447'> 10.1101/2023.07.02.547447</a></p>
        <p class="abstract">Recent massive metatranscriptome mining substantially expanded the diversity of the bacterial RNA virome, suggesting that additional groups of riboviruses infecting bacterial hosts remain to be discovered. We employed full length double-stranded (ds) RNA sequencing for identification of riboviruses associated with microbial consortia dominated by bacteria and archaea in acidic hot springs in Japan. Whole sequences of two groups of multisegmented riboviruses genomes were obtained. One group, which we denoted hot spring riboviruses (HsRV), consists of unusual viruses with distinct RNA-dependent RNA polymerases (RdRPs) that seem to be intermediates between typical ribovirus RdRPs and viral reverse transcriptases. We also identified viruses encoding HsRV-like RdRPs in moderate aquatic environments, including marine water, river sediments and salt marsh, indicating that this previously overlooked ribovirus group is not restricted to the extreme ecosystem. The HsRV-like viruses are candidates for a distinct phylum or even kingdom within the viral realm Riboviria. The second group, denoted hot spring partiti-like viruses (HsPV), is a distinct branch within the family Partitiviridae. All genome segments in both these groups of viruses display the organization typical of bacterial riboviruses, where multiple open reading frames encoding individual proteins are preceded by ribosome-binding sites. Together with the identification in bacteria-dominated habitats, this genome architecture indicates that riboviruses of these distinct groups infect thermoacidophilic bacterial hosts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546645">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546645" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546645">
        <p class="paperTitle">Homing in on the rare virosphere reveals the native host of giant viruses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546645" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546645" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fromm, A.; Hevroni, G.; Vincent, F.; Schatz, D.; Martinez-Gutierrez, C. A.; Aylward, F. O.; Vardi, A.</p>
        <p class="info">Score: 27.8, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546645' target='https://doi.org/10.1101/2023.06.27.546645'> 10.1101/2023.06.27.546645</a></p>
        <p class="abstract">Giant viruses (phylum Nucleocytoviricota) are globally distributed in aquatic ecosystems1,2. They play major roles as evolutionary drivers of eukaryotic plankton3 and regulators of global biogeochemical cycles4. Recent metagenomic studies have significantly expanded the known diversity of marine giant viruses1,5-7, but we still lack fundamental knowledge about their native hosts, thereby hindering our understanding of their lifecycle and ecological importance. Here, we aim to discover the native hosts of giant viruses using a novel, sensitive single-cell metatranscriptomic approach. By applying this approach to natural plankton communities, we unraveled an active viral infection of several giant viruses, from multiple lineages, and identified their native hosts. We identify a rare lineage of giant virus (Imitervirales-07) infecting a minute population of protists (class Katablepharidaceae) and revealed the prevalence of highly expressed viral-encoded cell-fate regulation genes in infected cells. Further examination of this host-virus dynamics in a temporal resolution suggested this giant virus controls its host population demise. Our results demonstrate how single-cell metatranscriptomics is a sensitive approach for pairing viruses with their authentic hosts and studying their ecological significance in a culture-independent manner in the marine environment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.546935">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.546935" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.546935">
        <p class="paperTitle">Amazing structural diversity of giant virus-like particles in forest soil</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.546935" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.546935" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fischer, M. G.; Mersdorf, U.; Blanchard, J. L.</p>
        <p class="info">Score: 121.7, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.546935' target='https://doi.org/10.1101/2023.06.30.546935'> 10.1101/2023.06.30.546935</a></p>
        <p class="abstract">Large DNA viruses of the phylum Nucleocytoviricota infect diverse eukaryotic hosts from protists to humans, with profound consequences for aquatic and terrestrial ecosystems. While nucleocytoviruses are known to be highly diverse in metagenomes, knowledge of their capsid structures is restricted to a few characterized representatives. Here, we visualize giant virus-like particles (VLPs, diameter &gt;0.2 {micro}m) directly from the environment using transmission electron microscopy. We found that Harvard Forest soils contain a higher diversity of giant VLP morphotypes than all hitherto isolated giant viruses combined. These included VLPs with icosahedral capsid symmetry, ovoid shapes similar to pandoraviruses, and bacilliform shapes that may represent novel viruses. We discovered giant icosahedral capsids with structural modifications that had not been described before including tubular appendages, modified vertices, tails, and capsids consisting of multiple layers or internal channels. Many giant VLPs were covered with fibers of varying lengths, thicknesses, densities, and terminal structures. These findings imply that giant viruses employ a much wider array of capsid structures and mechanisms to interact with their host cells than is currently known. We also found diverse tailed bacteriophages and filamentous VLPs, as well as ultra-small cells. Our study offers a first glimpse of the vast diversity of unexplored viral structures in soil and reinforces the potential of transmission electron microscopy for fundamental discoveries in environmental microbiology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.547807">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.547807" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.547807">
        <p class="paperTitle">Bam complex associated proteins in Escherichia coli are functionally linked to peptidoglycan biosynthesis, membrane fluidity and DNA replication</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.547807" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.547807" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bryant, J. A.; Staunton, K. A.; Doherty, H. M.; Alao, M. B.; Ma, X.; Morcinek-Orlowska, J.; Goodall, E. C. A.; Gray, J.; Milner, M.; Cole, J. A.; de Cogan, F.; Knowles, T. J.; Glinkowska, M. K.; Moradigaravand, D.; Henderson, I. R.; Banzhaf, M.</p>
        <p class="info">Score: 22.9, Published: 2023-07-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.547807' target='https://doi.org/10.1101/2023.07.05.547807'> 10.1101/2023.07.05.547807</a></p>
        <p class="abstract">Biogenesis of the bacterial outer membrane is key to survival and antibiotic resistance. Central to this is the beta-barrel assembly machine (Bam) complex and its associated chaperones, which are responsible for outer membrane protein (OMP) transport and insertion. The Escherichia coli Bam complex consists of two essential subunits, BamA and BamD, and three non-essential lipoproteins, BamB, BamC and BamE. Optimal Bam function is further dependent on the non-essential chaperones DegP, Skp and SurA. Despite intensive study, the specific function of these non-essential Bam-associated proteins remains unknown. Here, we analysed knockout strains for each gene by phenotypic screening, conservation analysis and high-throughput genetics. We reveal that Bam activity is affected by outer membrane lipid composition and that enterobacterial common antigen is essential in the absence of the chaperone SurA. We also show that components of peptidoglycan are conditionally essential with Bam accessory lipoproteins and that DNA replication is perturbed in the absence of BamB. Together, our data indicates potential mechanisms for coordination of OMP biogenesis with processes such as LPS and peptidoglycan biogenesis, and DNA replication.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547502">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547502" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547502">
        <p class="paperTitle">Predicting bacterial interaction outcomes from monoculture growth and supernatant assays</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547502" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547502" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schmitz, D. A.; Wechsler, T.; Mignot, I.; Kuemmerli, R.</p>
        <p class="info">Score: 17.9, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547502' target='https://doi.org/10.1101/2023.07.03.547502'> 10.1101/2023.07.03.547502</a></p>
        <p class="abstract">How to derive principles of community dynamics and stability is a central question in microbial ecology. To answer this, bottom-up experimental approaches, in which a small number of bacterial species are mixed, have become popular. However, experimental setups are typically limited because species are difficult to distinguish in mixed cultures and co-culture experiments are labor-intensive. Here, we use a community of four bacterial species to show that information from monoculture growth and inhibitory effects caused by secreted compounds can be combined to reliably predict pairwise species interactions in co-cultures. Specifically, integrative parameters from growth curves allow to build a competitive rank order, which is then adjusted using inhibitory compound effects from supernatant assays. While our procedure worked for two media examined, we observed differences in species rank orders between media. We further used computer simulations, parameterized with our empirical data, to show that higher-order species interactions largely follow the dynamics predicted from pairwise interactions with one important exception. The impact of inhibitory compounds was reduced in higher-order communities because these compounds are spread across multiple species, thereby diluting their effects. Altogether, our results lead to the formulation of three simple rules of how monoculture growth and supernatant assay data can be combined to establish competitive species rank orders in bacterial communities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547468">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547468" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547468">
        <p class="paperTitle">Interrogation of RNA-protein interaction dynamics in bacterial growth</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547468" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547468" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Monti, M.; Herman, R.; Davey, K.; Dawson, C. S.; Ule, J.; Thomas, G. H.; Willis, A. E.; Lilley, K. S.; Villanueva, E.</p>
        <p class="info">Score: 15.3, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547468' target='https://doi.org/10.1101/2023.07.03.547468'> 10.1101/2023.07.03.547468</a></p>
        <p class="abstract">Characterising RNA-protein interaction dynamics is fundamental to understand how bacteria respond to their environment. In this study, we have analysed the dynamics of 91% of the Escherichia coli expressed proteome and the RNA-interaction properties of 271 RNA-binding proteins (RBPs) at different growth phases. We find that 68% of RBPs differentially bind RNA across growth phases and characterise 17 previously unannotated proteins as bacterial RBPs including YfiF, a ncRNA-binding protein. While these new RBPs are mostly present in Proteobacteria, two of them have human orthologs in the form of mitochondrial proteins associated with rare metabolic disorders. Moreover, we reveal novel RBP functions for proteins such as the chaperone HtpG, a new stationary phase tRNA-binding protein. Altogether, we provide the first dynamic RBPome of a bacterium, showcasing how this approach can reveal the function of uncharacterised proteins, and identify critical RNA-protein interactions for cell growth which could inform new antimicrobial therapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.02.547337">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.02.547337" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.02.547337">
        <p class="paperTitle">Yeast diversity in open agave fermentations across Mexico</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.02.547337" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.02.547337" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gallegos-Gasillas, P.; Garcia-Ortega, L. F.; Espinosa-Cantu, A.; Torres-Lagunes, C. G.; Avelar-Rivas, J. A.; Cano-Ricardez, A.; Garcia-Acero, A. M.; Ruiz-Castro, S.; Flores-Barraza, M.; Castillo, A.; Gonzalez-Zozaya, F.; Delgado-Lemus, A.; Molina-Freaner, F.; Jacques-Hernandez, C.; Hernandez-Lopez, A.; Delaye, L.; Aguirre-Dugua, X.; Kirchmayr, M. R.; Morales, L.; Mancera, E.; DeLuna, A.</p>
        <p class="info">Score: 15.2, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.02.547337' target='https://doi.org/10.1101/2023.07.02.547337'> 10.1101/2023.07.02.547337</a></p>
        <p class="abstract">Yeasts are a diverse group of fungal microorganisms that are widely used to produce fermented foods and beverages. In Mexico, open fermentations are used to obtain spirits from agave plants. Despite the prevalence of this traditional practice throughout the country, yeasts have only been isolated and studied from a limited number of distilleries. To systematically describe the diversity of yeast species from open agave fermentations, here we generate the YMX.1.0 culture collection by isolating 4,524 strains from 68 sites in diverse climatic, geographical, and biological contexts. We used MALDI-TOF mass spectrometry for taxonomic classification and a subset of the strains was verified by ITS and D1/D2 sequencing. The most abundant species isolated from all producing regions were Saccharomyces cerevisiae, Pichia kudriavzevii, Pichia manshurica, and Kluyveromyces marxianus. Despite the great diversity of environmental conditions and production practices the composition of yeast communities remained largely homogeneous throughout locations and fermentation stages, even if less abundant but commonly occurring yeasts were considered. Furthermore, ITS and D1/D2 sequencing revealed two candidate new species of Saccharomycetales. To explore the intraspecific variation of the yeasts from agave fermentations, we conducted genome sequencing on four isolates of the non-conventional yeast Kazachstania humilis. The genomes of these four strains were considerably distinct from other genomes of the same species, suggesting that they belong to a different population. Our work contributes to the understanding and conservation of an open fermentation system of great cultural and economic importance, providing a valuable resource to study the biology and genetic diversity of microorganisms living at the interface of natural and human-associated environments.

TAKE AWAYO_LIWe isolated and identified 4,524 yeast strains from open agave fermentations in Mexico.
C_LIO_LIYeast communities remained largely homogeneous throughout diverse locations.
C_LIO_LIKazachstania humilis genomes differed significantly from isolates in other regions of the world.
C_LIO_LIWe report two candidate new species related to the Pichia clade.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547579">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547579" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547579">
        <p class="paperTitle">Covalent macrocyclic proteasome inhibitors mitigate resistance in Plasmodium falciparum</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547579" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547579" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bennett, J. M.; Ward, K. E.; Muir, R.; Kabeche, S.; Yoo, E.; Yeo, T.; Lam, G.; Zhang, H.; Almaliti, J.; Berger, G.; Faucher, F. F.; Lin, G.; Gerwick, W. H.; Yeh, E.; Fidock, D. A.; Bogyo, M.</p>
        <p class="info">Score: 15.0, Published: 2023-07-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547579' target='https://doi.org/10.1101/2023.07.03.547579'> 10.1101/2023.07.03.547579</a></p>
        <p class="abstract">The Plasmodium proteasome is a promising antimalarial drug target due to its essential role in all parasite lifecycle stages. Furthermore, proteasome inhibitors have synergistic effects when combined with current first-line artemisinins. Linear peptides that covalently inhibit the proteasome are effective at killing parasites and have a low propensity for inducing resistance. However, these scaffolds generally suffer from poor pharmacokinetics and bioavailability. Here we describe the development of covalent, irreversible macrocyclic inhibitors of the P. falciparum proteasome. We identified compounds with excellent potency and low cytotoxicity, however, the first generation suffered from poor microsomal stability. Further optimization of an existing macrocyclic scaffold resulted in an irreversible covalent inhibitor carrying a vinyl sulfone electrophile that retained high potency, low cytotoxicity, and had acceptable metabolic stability. Importantly, unlike the parent reversible inhibitor that selected for multiple mutations in the proteasome, with one resulting in a 5,000-fold loss of potency, the irreversible analog only showed a 5-fold loss in potency for any single point mutation. Furthermore, an epoxyketone analog of the same scaffold retained potency against a panel of known proteasome mutants. These results confirm that macrocycles are optimal scaffolds to target the malarial proteasome and that the use of a covalent electrophile can greatly reduce the ability of the parasite to generate drug resistance mutations.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
