<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="High-resolution profiling of bacterial and fungal communities using pangenome-informed taxon-specific amplicons and long-read sequencing
Authors: Stalder, L.; Maurhofer, M.; Croll, D.
Score: 29.6, Published: 2023-07-18 DOI: 10.1101/2023.07.17.549274
AO_SCPLOWBSTRACTC_SCPLOWHigh-throughput sequencing technologies have greatly advanced our understanding of microbiomes, but resolving microbial communities at species and strain levels remains challenging. In this study, we present a pipeline for designing, multiplexing, and sequencing highly polymorphic taxon-specific amplicons using PacBio circular consensus sequencing. We focus on the wheat microbiome as a proof-of-principle and demonstrate unprecedented resolution for the wheat-associated Pseudomonas microbiome and the ubiquitous fungal pathogen Zymoseptoria tritici.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="High-resolution profiling of bacterial and fungal communities using pangenome-informed taxon-specific amplicons and long-read sequencing
Authors: Stalder, L.; Maurhofer, M.; Croll, D.
Score: 29.6, Published: 2023-07-18 DOI: 10.1101/2023.07.17.549274
AO_SCPLOWBSTRACTC_SCPLOWHigh-throughput sequencing technologies have greatly advanced our understanding of microbiomes, but resolving microbial communities at species and strain levels remains challenging. In this study, we present a pipeline for designing, multiplexing, and sequencing highly polymorphic taxon-specific amplicons using PacBio circular consensus sequencing. We focus on the wheat microbiome as a proof-of-principle and demonstrate unprecedented resolution for the wheat-associated Pseudomonas microbiome and the ubiquitous fungal pathogen Zymoseptoria tritici." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-23T10:36:34+00:00" />
<meta property="article:modified_time" content="2023-07-23T10:36:34+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="High-resolution profiling of bacterial and fungal communities using pangenome-informed taxon-specific amplicons and long-read sequencing
Authors: Stalder, L.; Maurhofer, M.; Croll, D.
Score: 29.6, Published: 2023-07-18 DOI: 10.1101/2023.07.17.549274
AO_SCPLOWBSTRACTC_SCPLOWHigh-throughput sequencing technologies have greatly advanced our understanding of microbiomes, but resolving microbial communities at species and strain levels remains challenging. In this study, we present a pipeline for designing, multiplexing, and sequencing highly polymorphic taxon-specific amplicons using PacBio circular consensus sequencing. We focus on the wheat microbiome as a proof-of-principle and demonstrate unprecedented resolution for the wheat-associated Pseudomonas microbiome and the ubiquitous fungal pathogen Zymoseptoria tritici."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "High-resolution profiling of bacterial and fungal communities using pangenome-informed taxon-specific amplicons and long-read sequencing\nAuthors: Stalder, L.; Maurhofer, M.; Croll, D.\nScore: 29.6, Published: 2023-07-18 DOI: 10.1101/2023.07.17.549274\nAO_SCPLOWBSTRACTC_SCPLOWHigh-throughput sequencing technologies have greatly advanced our understanding of microbiomes, but resolving microbial communities at species and strain levels remains challenging. In this study, we present a pipeline for designing, multiplexing, and sequencing highly polymorphic taxon-specific amplicons using PacBio circular consensus sequencing. We focus on the wheat microbiome as a proof-of-principle and demonstrate unprecedented resolution for the wheat-associated Pseudomonas microbiome and the ubiquitous fungal pathogen Zymoseptoria tritici.",
  "keywords": [
    
  ],
  "articleBody": " High-resolution profiling of bacterial and fungal communities using pangenome-informed taxon-specific amplicons and long-read sequencing\nAuthors: Stalder, L.; Maurhofer, M.; Croll, D.\nScore: 29.6, Published: 2023-07-18 DOI: 10.1101/2023.07.17.549274\nAO_SCPLOWBSTRACTC_SCPLOWHigh-throughput sequencing technologies have greatly advanced our understanding of microbiomes, but resolving microbial communities at species and strain levels remains challenging. In this study, we present a pipeline for designing, multiplexing, and sequencing highly polymorphic taxon-specific amplicons using PacBio circular consensus sequencing. We focus on the wheat microbiome as a proof-of-principle and demonstrate unprecedented resolution for the wheat-associated Pseudomonas microbiome and the ubiquitous fungal pathogen Zymoseptoria tritici. Our approach achieves an order of magnitude higher phylogenetic resolution compared to existing ribosomal amplicons. We show that the designed amplicons accurately capture species and strain diversity outperforming full-length 16S and ITS amplicons. We track microbial communities in the wheat phyllosphere across time and space to establish fine-grained species and strain-specific dynamics. The high-resolution profiling provided by taxon-specific amplicons opens new avenues for studying microbial community dynamics in complex environments. Moreover, the cost-effectiveness and scalability of our approach make it applicable to diverse ecological studies. Overall, our work demonstrates the potential of long-read amplicon sequencing to overcome limitations in phylogenetic resolution and enhance our understanding of microbiome diversity and dynamics.\nPhage inducible chromosomal minimalist island (PICMI), a family of satellites of marine virulent phages\nAuthors: Barcia-Cruz, R.; Goudenege, D.; Moura de Sousa, J. A.; Piel, D.; Marbouty, M.; Rocha, E.; Le Roux, F.\nScore: 24.0, Published: 2023-07-18 DOI: 10.1101/2023.07.18.549517\nPhage satellites are genetic elements that hijack the phage machinery for their own dissemination. However, only few phage satellites have been characterized, and mechanisms by which they influence microbial evolution in nature are unclear. Here we identify a new family of satellites, the Phage Inducible Chromosomal Minimalist Island (PICMI), which is broadly distributed in the marine bacteria Vibrionaceae. PICMI is characterized by reduced gene content, does not encode genes for capsid remodeling and packages its DNA as a concatemer. PICMI is integrated in the bacterial host genome at the end of the fis regulator and encodes three core proteins necessary for excision and replication. PICMI is dependent on virulent phage particles to spread to other bacteria and confers host protection from other competitive phages, without interfering with its helper phage. The discovery of PICMI strongly suggests that phages, including virulent ones, play important roles for mobility of phage defense elements.\nPolysaccharide breakdown products drive degradation-dispersal cycles of foraging bacteria through changes in metabolism and motility\nAuthors: Stubbusch, A. K. M.; Keegstra, J. M.; Schwartzman, J.; Pontrelli, S.; Clerc, E. E.; Stocker, R.; Magnabosco, C.; Schubert, O. T.; Ackermann, M.; D'Souza, G. G.\nScore: 24.0, Published: 2023-07-14 DOI: 10.1101/2023.07.14.548877\nMost of Earths biomass is composed of polysaccharides. During biomass decomposition, polysaccharides are degraded by heterotrophic bacteria as a nutrient and energy source and are thereby partly remineralized into CO2. As polysaccharides are heterogeneously distributed in nature, following the colonization and degradation of a polysaccharide hotspot the cells need to reach new polysaccharide hotspots. Even though these degradation-dispersal cycles are an integral part in the global carbon cycle, we know little about how cells alternate between degradation and motility, and which signal triggers this behavioral switch. Here, we studied the growth of the marine bacterium Vibrio cyclitrophicus ZF270 on the abundant marine polysaccharide alginate. We used microfluidics-coupled time-lapse microscopy to analyze motility and growth of individual cells, and RNA sequencing to study associated changes in gene expression. Single cells grow at reduced rate on alginate until they form large groups that cooperatively break down the polymer. Exposing cell groups to digested alginate accelerates cell growth and changes the expression of genes involved in alginate degradation and catabolism, central metabolism, ribosomal biosynthesis, and transport. However, exposure to digested alginate also triggers cells to become motile and disperse from cell groups, proportionally increasing with the group size before the nutrient switch, accompanied by high expression of genes involved in flagellar assembly, chemotaxis, and quorum sensing. The motile cells chemotax toward alginate hotspots, likely enabling cells to find new polysaccharide hotspots. Overall, our findings reveal the cellular mechanisms underlying bacterial degradation-dispersal cycles that drive remineralization in natural environments. ImportancePolysaccharides, also known as glycans, are the most abundant form of biomass on Earth and understanding how they are degraded by microorganisms is essential for our understanding of the global carbon cycle and the storage and release of CO2 by natural systems. Although group formation is a common strategy used by bacterial cells to degrade ubiquitous polymeric growth substrates in nature, where nutrient hotspots are heterogeneously distributed, little is known about how cells prepare for dispersal from an exhausted nutrient source and re-initiate degradation of new nutrient patches. By quantifying growth, motility and chemotaxis of individual cells and comparing gene expression changes when populations were exposed to either polysaccharides or their degradation products in the form of digested polysaccharides, we show that bacterial cells alter their behavior when they experience a shift from polymeric to digested polysaccharides: After cells form groups during growth on polymers, the exposure to degradation products made cells motile, enabling dispersal from sessile cell groups and - guided by chemotaxis - movement towards new polysaccharide hotspots. Our study sheds light on the cellular processes that drive bacterial growth and behavior during carbon remineralization, an important driver of CO2 release from biomass in natural systems.\nOral immunization with rVSV bivalent vaccine elicits protective immune responses, including ADCC, against both SARS-CoV-2 and Influenza A viruses\nAuthors: Ouyang, M. J.; Ao, Z.; Olukitibi, T.; Lawrynuik, P.; Shieh, C.; Kung, S.; Fowke, K.; Kobasa, D.; Yao, X.\nScore: 19.4, Published: 2023-07-15 DOI: 10.1101/2023.07.14.549076\nCOVID-19 and influenza both cause enormous disease burdens, and vaccines are the primary measures for their control. Since these viral diseases are transmitted through the mucosal surface of the respiratory tract, developing an effective and convenient mucosal vaccine should be a high priority. We previously reported a recombinant vesicular stomatitis virus (rVSV)-based bivalent vaccine (v-EM2/SP{Delta}C1Delta) that protects animals from both SARS-CoV-2 and influenza viruses via intramuscular and intranasal immunization. Here, we further investigated the immune response induced by oral immunization with this vaccine and its protective efficacy in mice. The results demonstrated that the oral cavity delivery, like the intranasal route, elicited strong and protective systemic immune responses against SARS-CoV-2 and influenza A virus. This included high levels of neutralizing antibodies (NAbs) against SARS-CoV-2, as well as strong anti-SARS-CoV-2 spike protein (SP) antibody-dependent cellular cytotoxicity (ADCC) and anti-influenza M2 ADCC responses in mice sera. Furthermore, it provided efficient protection against challenge with influenza H1N1 virus in a mouse model, with a 100% survival rate and a significant low lung viral load of influenza virus. All these findings provide substantial evidence for the effectiveness of oral immunization with the rVSV bivalent vaccine.\nCross-species transmission and PB2 mammalian adaptations of highly pathogenic avian influenza A/H5N1 viruses in Chile\nAuthors: Pardo Roa, C.; Nelson, M.; Ariyama, N.; Aguayo, C.; Almonacid, L.; Munoz, G.; Navarro, C.; Avila, C.; Ulloa, M.; Reyes, R.; Fuentes Luppichini, E.; Mathieu, C.; Vergara, R.; Gonzalez, A.; Gonzalez, C. G.; Araya, H.; Fernandez, J.; Fasce, R.; Johow, M.; Medina, R. A.; Neira-Ramirez, V.\nScore: 22.8, Published: 2023-06-30 DOI: 10.1101/2023.06.30.547205\nH5N1 highly pathogenic avian influenza viruses (HPAIV) emerged in wild birds in Chile in December 2022 and spilled over into poultry, marine mammals, and one human. Between December 9, 2022 - March 14, 2023, a coordinated government/academic response detected HPAIV by real-time RT-PCR in 8.5% (412/4735) of samples from 23 avian and 3 mammal orders. Whole-genome sequences obtained from 77 birds and 8 marine mammals revealed that all Chilean H5N1 viruses belong to lineage 2.3.4.4b and cluster monophyletically with viruses from Peru, indicating a single introduction from North America into Peru/Chile. Mammalian adaptations were identified in the PB2 segment: D701N in two sea lions, one human, and one shorebird, and Q591K in the human and one sea lion. Minor variant analysis revealed that D701N was present in 52.9 - 70.9% of sequence reads, indicating the presence of both genotypes within hosts. Further surveillance of spillover events is warranted to assess the emergence and potential onward transmission of mammalian adapted H5N1 HPAIV in South America.\nA Taxonomically-informed Mass Spectrometry Search Tool for Microbial Metabolomics Data\nAuthors: Zuffa, S.; Schmid, R.; Bauermeister, A.; P. Gomes, P. W.; Caraballo-Rodriguez, A. M.; El Abiead, Y.; Aron, A. T.; Gentry, E. C.; Zemlin, J.; Meehan, M. J.; Avalon, N. E.; Cichewicz, R. H.; Buzun, E.; Carrillo Terrazas, M.; Hsu, C.-Y.; Oles, R.; Vasquez Ayala, A.; Zhao, J.; Chu, H.; Kuijpers, M. C. M.; Jackrel, S. L.; Tugizimana, F.; Nephali, L. P.; Dubery, I. A.; Madala, N. E.; Moreira, E. A.; Costa-Lotufo, L. V.; Lopes, N. P.; Rezende-Teixeira, P.; Jimenez, P. C.; Rimal, B.; Patterson, A. D.; Traxler, M. F.; Pessotti, R. d. C.; Alvarado-Villalobos, D.; Tamayo-Castillo, G.; Chaverri, P.; Escud\nScore: 15.3, Published: 2023-07-20 DOI: 10.1101/2023.07.20.549584\nMicrobeMASST, a taxonomically-informed mass spectrometry (MS) search tool, tackles limited microbial metabolite annotation in untargeted metabolomics experiments. Leveraging a curated database of \u003e60,000 microbial monocultures, users can search known and unknown MS/MS spectra and link them to their respective microbial producers via MS/MS fragmentation patterns. Identification of microbial-derived metabolites and relative producers, without a priori knowledge, will vastly enhance the understanding of microorganisms' role in ecology and human health.\nInflammasomes primarily restrict cytosolic Salmonella replication within human macrophages\nAuthors: Egan, M. S.; O'Rourke, E. A.; Mageswaran, S. K.; Zuo, B.; Martynyuk, I.; Demissie, T.; Hunter, E. N.; Bass, A. R.; Chang, Y.-W.; Brodsky, I. E.; Shin, S.\nScore: 14.7, Published: 2023-07-18 DOI: 10.1101/2023.07.17.549348\nSalmonella enterica serovar Typhimurium is a facultative intracellular pathogen that utilizes its type III secretion systems (T3SSs) to inject virulence factors into the host cell and colonize the host. In turn, a subset of cytosolic immune receptors respond to T3SS ligands by forming multimeric signaling complexes called inflammasomes, which activate caspases that induce interleukin-1 (IL-1) family cytokine release and an inflammatory form of cell death called pyroptosis. Human macrophages mount a multifaceted inflammasome response to Salmonella infection that ultimately restricts intracellular bacterial replication. However, how inflammasomes restrict Salmonella replication remains unknown. We find that caspase-1 is essential for mediating inflammasome responses to Salmonella and subsequent restriction of bacterial replication within human macrophages, with caspase-4 contributing as well. We also demonstrate that the downstream pore-forming protein gasdermin D (GSDMD) and ninjurin-1 (NINJ1), a mediator of terminal cell lysis, play a role in controlling Salmonella replication in human macrophages. Notably, in the absence of inflammasome responses, we observed hyperreplication of Salmonella within the cytosol of infected cells, and we also observed increased bacterial replication within vacuoles, suggesting that inflammasomes control Salmonella replication primarily within the cytosol and also within vacuoles. These findings reveal that inflammatory caspases and pyroptotic factors mediate inflammasome responses that restrict the subcellular localization of intracellular Salmonella replication within human macrophages.\nA host AAA-ATPase exhibits bacteriolytic activity for clearance of microbial infection\nAuthors: Ghosh, S.; Roy, S.; Baid, N.; Das, U. K.; Hajra, D.; Menon, S.; Sahil, M.; Shaw, S.; Rakshit, S.; Rajmani, R. S.; Adicherla, H.; Mondal, J.; Chakravortty, D.; Banerjee, A.\nScore: 14.6, Published: 2023-07-18 DOI: 10.1101/2023.07.18.549519\nAn array of cytosol guarding factors impede bacterial invasion and preserve cellular sterility. Amongst them, proteasomal degradation of ubiquitinated pathogens has emerged as a critical mechanism ensuring cytosolic sanctity. But smaller size of the proteasomal barrel and its inability to extract membrane-bound proteins, questions this paradigm. Our study unveiled a unique mechanical force-based strategy, employed by VCP/p97, an AAA-ATPase, which by extracting ubiquitinated bacterial surface proteins, eliminates pathogens. Using molecular dynamic simulation along with in-vitro and ex-vivo experiments, we demonstrated that p97s segregase activity is central to its bactericidal effect. Assisted by cofactors NPLOC4 and UFD1, p97 causes extensive lysis of phylogenetically diverse microbes and triggers leaching of bacterial cytosolic contents. In-vivo, p97 abrogated bacterial proliferation in host tissues and protected animals from lethal infections. Overall, we unravelled a distinct innate immune function of p97, which is critical for host protection against bacterial infections.\nThe prevalence of motility within the human oral microbiota\nAuthors: Rocha, S. T.; Shah, D. D.; Zhu, Q.; Shrivastava, A.\nScore: 13.0, Published: 2023-07-17 DOI: 10.1101/2023.07.17.549387\nThe human oral and nasal microbiota contains approximately 770 cultivable bacterial species. More than 2000 genome sequences of these bacteria can be found in the expanded Human Oral Microbiome Database (eHOMD). We developed HOMDscrape, a freely available Python software tool to programmatically retrieve and process amino acid sequences and sequence identifiers from BLAST results acquired from the eHOMD website. Using the data obtained through HOMDscrape, the phylogeny of proteins involved in bacterial flagellar motility, Type 4 pilus driven twitching motility, and Type 9 Secretion system (T9SS) driven gliding motility was constructed. A comprehensive phylogenetic analysis was conducted for all components of the rotary T9SS, a machinery responsible for secreting various enzymes, virulence factors, and enabling bacterial gliding motility. Results revealed that the T9SS outer membrane {beta}-barrel protein SprA of human oral microbes underwent horizontal evolution. Overall, we catalog motile microbes that inhabit the human oral microbiota and document their evolutionary connections. These results will serve as a guide for further studies exploring the impact of motility on shaping of the human oral microbiota.\nGlycan-specific IgM is critical for human immunity to Staphylococcus aureus\nAuthors: Hendriks, A.; Kerkman, P. F.; Varkila, M. R.; Haitsma-Mulier, J. L.; Ali, S.; ten Doesschate, T.; van der Vaart, T. W.; de Haas, C. J. C.; Aerts, P. C.; Cremer, O. L.; Bonten, M. J.; Nizet, V.; Liu, G. Y.; Codee, J. D. C.; Rooijakkers, S. H.; van Strijp, J. A.; van Sorge, N. M.\nScore: 14.2, Published: 2023-07-16 DOI: 10.1101/2023.07.14.548956\nStaphylococcus aureus is a major human pathogen but the immune factors that protect against it remain elusive. In particular, high opsonic IgG titers achieved in preclinical S. aureus animal immunization studies have consistently failed to translate to protection in human clinical trials. Here, we investigated the antibody responses to a conserved surface glycan, Wall Teichoic Acid (WTA). IgM and IgG antibodies specific to WTA were universally present in plasma from healthy individuals. Functionally, WTA-specific IgM outperformed IgG in opsonophagocytic killing of S. aureus and conferred passive protection against S. aureus infection in vivo. In the clinical setting, WTA-specific IgM responses, but not IgG responses, were significantly lower in S. aureus bacteremia patients compared to healthy individuals, correlated with mortality risk and showed impaired bacterial opsonization. Our findings can guide risk stratification of hospitalized patients and inform future design of antibody-based therapies and vaccines against serious S. aureus infection.\n",
  "wordCount" : "2405",
  "inLanguage": "en",
  "datePublished": "2023-07-23T10:36:34Z",
  "dateModified": "2023-07-23T10:36:34Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta"><span>updated on July 23, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549274">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549274" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549274">
        <p class="paperTitle">High-resolution profiling of bacterial and fungal communities using pangenome-informed taxon-specific amplicons and long-read sequencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549274" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549274" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stalder, L.; Maurhofer, M.; Croll, D.</p>
        <p class="info">Score: 29.6, Published: 2023-07-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549274' target='https://doi.org/10.1101/2023.07.17.549274'> 10.1101/2023.07.17.549274</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWHigh-throughput sequencing technologies have greatly advanced our understanding of microbiomes, but resolving microbial communities at species and strain levels remains challenging. In this study, we present a pipeline for designing, multiplexing, and sequencing highly polymorphic taxon-specific amplicons using PacBio circular consensus sequencing. We focus on the wheat microbiome as a proof-of-principle and demonstrate unprecedented resolution for the wheat-associated Pseudomonas microbiome and the ubiquitous fungal pathogen Zymoseptoria tritici. Our approach achieves an order of magnitude higher phylogenetic resolution compared to existing ribosomal amplicons. We show that the designed amplicons accurately capture species and strain diversity outperforming full-length 16S and ITS amplicons. We track microbial communities in the wheat phyllosphere across time and space to establish fine-grained species and strain-specific dynamics. The high-resolution profiling provided by taxon-specific amplicons opens new avenues for studying microbial community dynamics in complex environments. Moreover, the cost-effectiveness and scalability of our approach make it applicable to diverse ecological studies. Overall, our work demonstrates the potential of long-read amplicon sequencing to overcome limitations in phylogenetic resolution and enhance our understanding of microbiome diversity and dynamics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549517">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549517" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549517">
        <p class="paperTitle">Phage inducible chromosomal minimalist island (PICMI), a family of satellites of marine virulent phages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549517" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549517" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Barcia-Cruz, R.; Goudenege, D.; Moura de Sousa, J. A.; Piel, D.; Marbouty, M.; Rocha, E.; Le Roux, F.</p>
        <p class="info">Score: 24.0, Published: 2023-07-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549517' target='https://doi.org/10.1101/2023.07.18.549517'> 10.1101/2023.07.18.549517</a></p>
        <p class="abstract">Phage satellites are genetic elements that hijack the phage machinery for their own dissemination. However, only few phage satellites have been characterized, and mechanisms by which they influence microbial evolution in nature are unclear. Here we identify a new family of satellites, the Phage Inducible Chromosomal Minimalist Island (PICMI), which is broadly distributed in the marine bacteria Vibrionaceae. PICMI is characterized by reduced gene content, does not encode genes for capsid remodeling and packages its DNA as a concatemer. PICMI is integrated in the bacterial host genome at the end of the fis regulator and encodes three core proteins necessary for excision and replication. PICMI is dependent on virulent phage particles to spread to other bacteria and confers host protection from other competitive phages, without interfering with its helper phage. The discovery of PICMI strongly suggests that phages, including virulent ones, play important roles for mobility of phage defense elements.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.548877">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.548877" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.548877">
        <p class="paperTitle">Polysaccharide breakdown products drive degradation-dispersal cycles of foraging bacteria through changes in metabolism and motility</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.548877" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.548877" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stubbusch, A. K. M.; Keegstra, J. M.; Schwartzman, J.; Pontrelli, S.; Clerc, E. E.; Stocker, R.; Magnabosco, C.; Schubert, O. T.; Ackermann, M.; D&#39;Souza, G. G.</p>
        <p class="info">Score: 24.0, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.548877' target='https://doi.org/10.1101/2023.07.14.548877'> 10.1101/2023.07.14.548877</a></p>
        <p class="abstract">Most of Earths biomass is composed of polysaccharides. During biomass decomposition, polysaccharides are degraded by heterotrophic bacteria as a nutrient and energy source and are thereby partly remineralized into CO2. As polysaccharides are heterogeneously distributed in nature, following the colonization and degradation of a polysaccharide hotspot the cells need to reach new polysaccharide hotspots. Even though these degradation-dispersal cycles are an integral part in the global carbon cycle, we know little about how cells alternate between degradation and motility, and which signal triggers this behavioral switch. Here, we studied the growth of the marine bacterium Vibrio cyclitrophicus ZF270 on the abundant marine polysaccharide alginate. We used microfluidics-coupled time-lapse microscopy to analyze motility and growth of individual cells, and RNA sequencing to study associated changes in gene expression. Single cells grow at reduced rate on alginate until they form large groups that cooperatively break down the polymer. Exposing cell groups to digested alginate accelerates cell growth and changes the expression of genes involved in alginate degradation and catabolism, central metabolism, ribosomal biosynthesis, and transport. However, exposure to digested alginate also triggers cells to become motile and disperse from cell groups, proportionally increasing with the group size before the nutrient switch, accompanied by high expression of genes involved in flagellar assembly, chemotaxis, and quorum sensing. The motile cells chemotax toward alginate hotspots, likely enabling cells to find new polysaccharide hotspots. Overall, our findings reveal the cellular mechanisms underlying bacterial degradation-dispersal cycles that drive remineralization in natural environments.

ImportancePolysaccharides, also known as glycans, are the most abundant form of biomass on Earth and understanding how they are degraded by microorganisms is essential for our understanding of the global carbon cycle and the storage and release of CO2 by natural systems. Although group formation is a common strategy used by bacterial cells to degrade ubiquitous polymeric growth substrates in nature, where nutrient hotspots are heterogeneously distributed, little is known about how cells prepare for dispersal from an exhausted nutrient source and re-initiate degradation of new nutrient patches. By quantifying growth, motility and chemotaxis of individual cells and comparing gene expression changes when populations were exposed to either polysaccharides or their degradation products in the form of digested polysaccharides, we show that bacterial cells alter their behavior when they experience a shift from polymeric to digested polysaccharides: After cells form groups during growth on polymers, the exposure to degradation products made cells motile, enabling dispersal from sessile cell groups and - guided by chemotaxis - movement towards new polysaccharide hotspots. Our study sheds light on the cellular processes that drive bacterial growth and behavior during carbon remineralization, an important driver of CO2 release from biomass in natural systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.549076">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.549076" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.549076">
        <p class="paperTitle">Oral immunization with rVSV bivalent vaccine elicits protective immune responses, including ADCC, against both SARS-CoV-2 and Influenza A viruses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.549076" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.549076" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ouyang, M. J.; Ao, Z.; Olukitibi, T.; Lawrynuik, P.; Shieh, C.; Kung, S.; Fowke, K.; Kobasa, D.; Yao, X.</p>
        <p class="info">Score: 19.4, Published: 2023-07-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.549076' target='https://doi.org/10.1101/2023.07.14.549076'> 10.1101/2023.07.14.549076</a></p>
        <p class="abstract">COVID-19 and influenza both cause enormous disease burdens, and vaccines are the primary measures for their control. Since these viral diseases are transmitted through the mucosal surface of the respiratory tract, developing an effective and convenient mucosal vaccine should be a high priority. We previously reported a recombinant vesicular stomatitis virus (rVSV)-based bivalent vaccine (v-EM2/SP{Delta}C1Delta) that protects animals from both SARS-CoV-2 and influenza viruses via intramuscular and intranasal immunization. Here, we further investigated the immune response induced by oral immunization with this vaccine and its protective efficacy in mice. The results demonstrated that the oral cavity delivery, like the intranasal route, elicited strong and protective systemic immune responses against SARS-CoV-2 and influenza A virus. This included high levels of neutralizing antibodies (NAbs) against SARS-CoV-2, as well as strong anti-SARS-CoV-2 spike protein (SP) antibody-dependent cellular cytotoxicity (ADCC) and anti-influenza M2 ADCC responses in mice sera. Furthermore, it provided efficient protection against challenge with influenza H1N1 virus in a mouse model, with a 100% survival rate and a significant low lung viral load of influenza virus. All these findings provide substantial evidence for the effectiveness of oral immunization with the rVSV bivalent vaccine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547205">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547205" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547205">
        <p class="paperTitle">Cross-species transmission and PB2 mammalian adaptations of highly pathogenic avian influenza A/H5N1 viruses in Chile</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547205" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547205" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pardo Roa, C.; Nelson, M.; Ariyama, N.; Aguayo, C.; Almonacid, L.; Munoz, G.; Navarro, C.; Avila, C.; Ulloa, M.; Reyes, R.; Fuentes Luppichini, E.; Mathieu, C.; Vergara, R.; Gonzalez, A.; Gonzalez, C. G.; Araya, H.; Fernandez, J.; Fasce, R.; Johow, M.; Medina, R. A.; Neira-Ramirez, V.</p>
        <p class="info">Score: 22.8, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547205' target='https://doi.org/10.1101/2023.06.30.547205'> 10.1101/2023.06.30.547205</a></p>
        <p class="abstract">H5N1 highly pathogenic avian influenza viruses (HPAIV) emerged in wild birds in Chile in December 2022 and spilled over into poultry, marine mammals, and one human. Between December 9, 2022 - March 14, 2023, a coordinated government/academic response detected HPAIV by real-time RT-PCR in 8.5% (412/4735) of samples from 23 avian and 3 mammal orders. Whole-genome sequences obtained from 77 birds and 8 marine mammals revealed that all Chilean H5N1 viruses belong to lineage 2.3.4.4b and cluster monophyletically with viruses from Peru, indicating a single introduction from North America into Peru/Chile. Mammalian adaptations were identified in the PB2 segment: D701N in two sea lions, one human, and one shorebird, and Q591K in the human and one sea lion. Minor variant analysis revealed that D701N was present in 52.9 - 70.9% of sequence reads, indicating the presence of both genotypes within hosts. Further surveillance of spillover events is warranted to assess the emergence and potential onward transmission of mammalian adapted H5N1 HPAIV in South America.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549584">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549584" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549584">
        <p class="paperTitle">A Taxonomically-informed Mass Spectrometry Search Tool for Microbial Metabolomics Data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549584" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549584" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zuffa, S.; Schmid, R.; Bauermeister, A.; P. Gomes, P. W.; Caraballo-Rodriguez, A. M.; El Abiead, Y.; Aron, A. T.; Gentry, E. C.; Zemlin, J.; Meehan, M. J.; Avalon, N. E.; Cichewicz, R. H.; Buzun, E.; Carrillo Terrazas, M.; Hsu, C.-Y.; Oles, R.; Vasquez Ayala, A.; Zhao, J.; Chu, H.; Kuijpers, M. C. M.; Jackrel, S. L.; Tugizimana, F.; Nephali, L. P.; Dubery, I. A.; Madala, N. E.; Moreira, E. A.; Costa-Lotufo, L. V.; Lopes, N. P.; Rezende-Teixeira, P.; Jimenez, P. C.; Rimal, B.; Patterson, A. D.; Traxler, M. F.; Pessotti, R. d. C.; Alvarado-Villalobos, D.; Tamayo-Castillo, G.; Chaverri, P.; Escud</p>
        <p class="info">Score: 15.3, Published: 2023-07-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549584' target='https://doi.org/10.1101/2023.07.20.549584'> 10.1101/2023.07.20.549584</a></p>
        <p class="abstract">MicrobeMASST, a taxonomically-informed mass spectrometry (MS) search tool, tackles limited microbial metabolite annotation in untargeted metabolomics experiments. Leveraging a curated database of &gt;60,000 microbial monocultures, users can search known and unknown MS/MS spectra and link them to their respective microbial producers via MS/MS fragmentation patterns. Identification of microbial-derived metabolites and relative producers, without a priori knowledge, will vastly enhance the understanding of microorganisms&#39; role in ecology and human health.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549348">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549348" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549348">
        <p class="paperTitle">Inflammasomes primarily restrict cytosolic Salmonella replication within human macrophages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549348" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549348" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Egan, M. S.; O&#39;Rourke, E. A.; Mageswaran, S. K.; Zuo, B.; Martynyuk, I.; Demissie, T.; Hunter, E. N.; Bass, A. R.; Chang, Y.-W.; Brodsky, I. E.; Shin, S.</p>
        <p class="info">Score: 14.7, Published: 2023-07-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549348' target='https://doi.org/10.1101/2023.07.17.549348'> 10.1101/2023.07.17.549348</a></p>
        <p class="abstract">Salmonella enterica serovar Typhimurium is a facultative intracellular pathogen that utilizes its type III secretion systems (T3SSs) to inject virulence factors into the host cell and colonize the host. In turn, a subset of cytosolic immune receptors respond to T3SS ligands by forming multimeric signaling complexes called inflammasomes, which activate caspases that induce interleukin-1 (IL-1) family cytokine release and an inflammatory form of cell death called pyroptosis. Human macrophages mount a multifaceted inflammasome response to Salmonella infection that ultimately restricts intracellular bacterial replication. However, how inflammasomes restrict Salmonella replication remains unknown. We find that caspase-1 is essential for mediating inflammasome responses to Salmonella and subsequent restriction of bacterial replication within human macrophages, with caspase-4 contributing as well. We also demonstrate that the downstream pore-forming protein gasdermin D (GSDMD) and ninjurin-1 (NINJ1), a mediator of terminal cell lysis, play a role in controlling Salmonella replication in human macrophages. Notably, in the absence of inflammasome responses, we observed hyperreplication of Salmonella within the cytosol of infected cells, and we also observed increased bacterial replication within vacuoles, suggesting that inflammasomes control Salmonella replication primarily within the cytosol and also within vacuoles. These findings reveal that inflammatory caspases and pyroptotic factors mediate inflammasome responses that restrict the subcellular localization of intracellular Salmonella replication within human macrophages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549519">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549519" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549519">
        <p class="paperTitle">A host AAA-ATPase exhibits bacteriolytic activity for clearance of microbial infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549519" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549519" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ghosh, S.; Roy, S.; Baid, N.; Das, U. K.; Hajra, D.; Menon, S.; Sahil, M.; Shaw, S.; Rakshit, S.; Rajmani, R. S.; Adicherla, H.; Mondal, J.; Chakravortty, D.; Banerjee, A.</p>
        <p class="info">Score: 14.6, Published: 2023-07-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549519' target='https://doi.org/10.1101/2023.07.18.549519'> 10.1101/2023.07.18.549519</a></p>
        <p class="abstract">An array of cytosol guarding factors impede bacterial invasion and preserve cellular sterility. Amongst them, proteasomal degradation of ubiquitinated pathogens has emerged as a critical mechanism ensuring cytosolic sanctity. But smaller size of the proteasomal barrel and its inability to extract membrane-bound proteins, questions this paradigm. Our study unveiled a unique mechanical force-based strategy, employed by VCP/p97, an AAA-ATPase, which by extracting ubiquitinated bacterial surface proteins, eliminates pathogens. Using molecular dynamic simulation along with in-vitro and ex-vivo experiments, we demonstrated that p97s segregase activity is central to its bactericidal effect. Assisted by cofactors NPLOC4 and UFD1, p97 causes extensive lysis of phylogenetically diverse microbes and triggers leaching of bacterial cytosolic contents. In-vivo, p97 abrogated bacterial proliferation in host tissues and protected animals from lethal infections. Overall, we unravelled a distinct innate immune function of p97, which is critical for host protection against bacterial infections.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549387">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549387" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549387">
        <p class="paperTitle">The prevalence of motility within the human oral microbiota</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549387" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549387" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rocha, S. T.; Shah, D. D.; Zhu, Q.; Shrivastava, A.</p>
        <p class="info">Score: 13.0, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549387' target='https://doi.org/10.1101/2023.07.17.549387'> 10.1101/2023.07.17.549387</a></p>
        <p class="abstract">The human oral and nasal microbiota contains approximately 770 cultivable bacterial species. More than 2000 genome sequences of these bacteria can be found in the expanded Human Oral Microbiome Database (eHOMD). We developed HOMDscrape, a freely available Python software tool to programmatically retrieve and process amino acid sequences and sequence identifiers from BLAST results acquired from the eHOMD website. Using the data obtained through HOMDscrape, the phylogeny of proteins involved in bacterial flagellar motility, Type 4 pilus driven twitching motility, and Type 9 Secretion system (T9SS) driven gliding motility was constructed. A comprehensive phylogenetic analysis was conducted for all components of the rotary T9SS, a machinery responsible for secreting various enzymes, virulence factors, and enabling bacterial gliding motility. Results revealed that the T9SS outer membrane {beta}-barrel protein SprA of human oral microbes underwent horizontal evolution. Overall, we catalog motile microbes that inhabit the human oral microbiota and document their evolutionary connections. These results will serve as a guide for further studies exploring the impact of motility on shaping of the human oral microbiota.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.548956">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.548956" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.548956">
        <p class="paperTitle">Glycan-specific IgM is critical for human immunity to Staphylococcus aureus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.548956" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.548956" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hendriks, A.; Kerkman, P. F.; Varkila, M. R.; Haitsma-Mulier, J. L.; Ali, S.; ten Doesschate, T.; van der Vaart, T. W.; de Haas, C. J. C.; Aerts, P. C.; Cremer, O. L.; Bonten, M. J.; Nizet, V.; Liu, G. Y.; Codee, J. D. C.; Rooijakkers, S. H.; van Strijp, J. A.; van Sorge, N. M.</p>
        <p class="info">Score: 14.2, Published: 2023-07-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.548956' target='https://doi.org/10.1101/2023.07.14.548956'> 10.1101/2023.07.14.548956</a></p>
        <p class="abstract">Staphylococcus aureus is a major human pathogen but the immune factors that protect against it remain elusive. In particular, high opsonic IgG titers achieved in preclinical S. aureus animal immunization studies have consistently failed to translate to protection in human clinical trials. Here, we investigated the antibody responses to a conserved surface glycan, Wall Teichoic Acid (WTA). IgM and IgG antibodies specific to WTA were universally present in plasma from healthy individuals. Functionally, WTA-specific IgM outperformed IgG in opsonophagocytic killing of S. aureus and conferred passive protection against S. aureus infection in vivo. In the clinical setting, WTA-specific IgM responses, but not IgG responses, were significantly lower in S. aureus bacteremia patients compared to healthy individuals, correlated with mortality risk and showed impaired bacterial opsonization. Our findings can guide risk stratification of hospitalized patients and inform future design of antibody-based therapies and vaccines against serious S. aureus infection.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
