<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Persistence of an infectious form of SARS-CoV-2 post protease inhibitor treatment of permissive cells in vitro
Authors: Nair, M. S.; Luck, M. I.; Huang, Y.; Sabo, Y.; Ho, D. D.
Score: 68.0, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572655
Reports have described SARS-CoV-2 rebound in COVID-19 patients treated with nirmatrelvir, a 3CL protease inhibitor. The cause remains a mystery, although drug resistance, re-infection, and lack of adequate immune responses have been excluded. We now present virologic findings that provide a clue to the cause of viral rebound, which occurs in [~]20% of the treated cases.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Persistence of an infectious form of SARS-CoV-2 post protease inhibitor treatment of permissive cells in vitro
Authors: Nair, M. S.; Luck, M. I.; Huang, Y.; Sabo, Y.; Ho, D. D.
Score: 68.0, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572655
Reports have described SARS-CoV-2 rebound in COVID-19 patients treated with nirmatrelvir, a 3CL protease inhibitor. The cause remains a mystery, although drug resistance, re-infection, and lack of adequate immune responses have been excluded. We now present virologic findings that provide a clue to the cause of viral rebound, which occurs in [~]20% of the treated cases." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-31T10:38:19+00:00" />
<meta property="article:modified_time" content="2023-12-31T10:38:19+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Persistence of an infectious form of SARS-CoV-2 post protease inhibitor treatment of permissive cells in vitro
Authors: Nair, M. S.; Luck, M. I.; Huang, Y.; Sabo, Y.; Ho, D. D.
Score: 68.0, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572655
Reports have described SARS-CoV-2 rebound in COVID-19 patients treated with nirmatrelvir, a 3CL protease inhibitor. The cause remains a mystery, although drug resistance, re-infection, and lack of adequate immune responses have been excluded. We now present virologic findings that provide a clue to the cause of viral rebound, which occurs in [~]20% of the treated cases."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Persistence of an infectious form of SARS-CoV-2 post protease inhibitor treatment of permissive cells in vitro\nAuthors: Nair, M. S.; Luck, M. I.; Huang, Y.; Sabo, Y.; Ho, D. D.\nScore: 68.0, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572655\nReports have described SARS-CoV-2 rebound in COVID-19 patients treated with nirmatrelvir, a 3CL protease inhibitor. The cause remains a mystery, although drug resistance, re-infection, and lack of adequate immune responses have been excluded. We now present virologic findings that provide a clue to the cause of viral rebound, which occurs in [~]20% of the treated cases.",
  "keywords": [
    
  ],
  "articleBody": " Persistence of an infectious form of SARS-CoV-2 post protease inhibitor treatment of permissive cells in vitro\nAuthors: Nair, M. S.; Luck, M. I.; Huang, Y.; Sabo, Y.; Ho, D. D.\nScore: 68.0, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572655\nReports have described SARS-CoV-2 rebound in COVID-19 patients treated with nirmatrelvir, a 3CL protease inhibitor. The cause remains a mystery, although drug resistance, re-infection, and lack of adequate immune responses have been excluded. We now present virologic findings that provide a clue to the cause of viral rebound, which occurs in [~]20% of the treated cases. The persistence of an intermediary form of infectious SARS-CoV-2 was experimentally documented in vitro after treatment with nirmatrelvir or another 3CL protease inhibitor, but not with a polymerase inhibitor, remdesivir. This infectious intermediate decayed slowly with a half-life of [~]1 day, suggesting that its persistence could outlive the treatment course to re-ignited SARS-CoV-2 infection as the drug is eliminated. Additional studies are needed to define the nature of this viral intermediate, but our findings point to a particular direction for future investigation and offer a specific treatment recommendation that should be tested clinically.\nExtensive variation and strain-specificity in dengue virus susceptibility among African Aedes aegypti populations\nAuthors: Dabo, S.; Henrion-Lacritick, A.; Lecuyer, A.; Jiolle, D.; Paupy, C.; Ayala, D.; da Veiga Leal, S.; Badolo, A.; Vega-Rua, A.; Sylla, M.; Akorli, J.; Otoo, S.; Lutomiah, J.; Sang, R.; Mutebi, J.-P.; Saleh, M.-C.; Rose, N.; McBride, C.; Lambrechts, L.\nScore: 39.7, Published: 2023-12-14 DOI: 10.1101/2023.12.14.571617\nAfrican populations of the mosquito Aedes aegypti are usually considered less susceptible to infection by human-pathogenic flaviviruses than globally invasive populations found outside Africa. Although this contrast has been well documented for Zika virus (ZIKV), it is unclear to what extent it is true for dengue virus (DENV), the most prevalent flavivirus of humans. Addressing this question is complicated by substantial genetic diversity among DENV strains, most notably in the form of four genetic types (DENV1 to DENV4), that can lead to genetically specific interactions with mosquito populations. Here, we carried out a continent-wide survey of DENV susceptibility using a panel of field-derived Ae. aegypti colonies from across the African range of the species and a colony from Guadeloupe, French West Indies as non-African reference. We found considerable variation in the ability of African Ae. aegypti populations to acquire and replicate a panel of six DENV strains spanning the four DENV types. Although African Ae. aegypti populations were generally less susceptible than the reference non-African population from Guadeloupe, in several instances some African populations were equally or more susceptible than the Guadeloupe population. Moreover, the relative level of susceptibility between African mosquito populations depended on the DENV strain, indicating genetically specific interactions. We conclude that unlike ZIKV susceptibility, there is no clear-cut dichotomy in DENV susceptibility between African and non-African Ae. aegypti. DENV susceptibility of African Ae. aegypti populations is highly heterogeneous and largely governed by the specific pairing of mosquito population and DENV strain. Author summaryAfrican populations of the mosquito Aedes aegypti are usually thought to be less likely to get infected by flaviviruses compared to Ae. aegypti mosquitoes found outside Africa. While this has been well-demonstrated for Zika virus, it is not clear if the same is true for dengue virus, which is the most common flavivirus in humans. Studying this is complicated by the strain diversity of dengue virus, including four main genetic types, potentially causing different interactions. In this study, we compared several mosquito populations and found that, in general, African mosquitoes were less likely to get infected by dengue virus compared to mosquitoes from outside Africa. However, in some cases, African mosquitoes were just as or even more likely to get infected. The specific strain of dengue virus also influenced how likely African mosquitoes were to get infected, showing that the relationship between African mosquitoes and dengue virus is complex.\nVirological characteristics of the SARS-CoV-2 JN.1 variant\nAuthors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Zahradnik, J.; Ito, J.; Sato, K.\nScore: 763.1, Published: 2023-12-09 DOI: 10.1101/2023.12.08.570782\nThe SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG.5.1 and HK.3. Comparing to XBB and BA.2, BA.2.86 carries more than 30 mutations in the spike (S) protein, indicating a high potential for immune evasion. BA.2.86 has evolved and its descendant, JN.1 (BA.2.86.1.1), emerged in late 2023. JN.1 harbors S:L455S and three mutations in non-S proteins. S:L455S is a hallmark mutation of JN.1: we have recently shown that HK.3 and other \"FLip\" variants carry S:L455F, which contributes to increased transmissibility and immune escape ability compared to the parental EG.5.1 variant. Here, we investigated the virological properties of JN.1.\nNovel nitrifying symbiont lineages are vertically inherited and widespread in marine sponges\nAuthors: Glasl, B.; Luter, H. M.; Damjanovic, K.; Kitzinger, K.; Mueller, A. J.; Mahler, L.; Engelberts, J. P.; Rix, L.; Osvatic, J. T.; Hausmann, B.; Seneca, J.; Daims, H.; Pjevac, P.; Wagner, M.\nScore: 12.3, Published: 2023-12-23 DOI: 10.1101/2023.12.23.573102\nAmmonia-oxidising archaea and nitrite-oxidising bacteria are common members of marine sponge microbiomes. They derive energy for carbon fixation and growth from nitrification - the oxidation of ammonia to nitrite and further to nitrate - and are proposed to play essential roles in the carbon and nitrogen cycling of sponge holobionts. In this study, we characterise two novel nitrifying symbiont lineages, Ca. Nitrosokoinonia and Ca. Nitrosymbion in the marine sponge Coscinoderma matthewsi using a combination of molecular tools, single-cell imaging techniques, and physiological rate measurements. Both represent a new genus in the ammonia-oxidising archaeal class Nitrososphaeria and the nitrite-oxidising bacterial order Nitrospirales, respectively. Furthermore, we show that larvae of this viviparous sponge are densely colonised by representatives of Ca. Nitrosokoinonia and Ca. Nitrosymbion indicating vertical transmission. In adults, the representatives of both symbiont genera are located extracellularly in the mesohyl. Comparative metagenome analyses and physiological data suggest that ammonia-oxidising archaeal symbionts of the genus Ca. Nitrosokoinonia strongly rely on endogenously produced nitrogenous compounds (i.e., ammonium, urea, nitriles/cyanides, and creatinine) rather than on exogenous ammonium sources taken up by the sponge. Additionally, the nitrite-oxidising bacterial symbionts Ca. Nitrosymbion may reciprocally support the ammonia-oxidisers with ammonia via the utilisation of sponge-derived urea and cyanate. Interestingly, comparative analyses of published environmental 16S rRNA amplicon data revealed that Ca. Nitrosokoinonia and Ca. Nitrosymbion are widely distributed and predominantly associated with marine sponges and corals, suggesting a broad relevance of our findings.\nQuantitative measurement of microbial growth rate with Raman microspectroscopy\nAuthors: Caro, T. A.; Kashyap, S.; Brown, G.; Chen, C.; Kopf, S.; Templeton, A.\nScore: 12.0, Published: 2023-12-17 DOI: 10.1101/2023.12.16.571966\nRates of microbial activity and growth are fundamental to understanding environmental geochemistry and ecology. However, measuring the heterogeneity of microbial activity at the single-cell level, especially within complex populations and environmental matrices, remains a forefront challenge. Stable Isotope Probing (SIP) is a standard method for assessing microbial activity and involves measuring the incorporation of an isotopically labeled compound into microbial biomass. Here, we assess the utility of Raman microspectroscopy as a SIP technique, specifically focusing on the measurement of deuterium (2H), a tracer of microbial biomass production. We generate calibrations of microbial biomass 2H values and find that Raman microspectroscopy reliably quantifies 2H incorporation ranging between 0 and 40 at. %. Applying the results of this calibration to a SIP model, we explicitly parameterize the factors controlling microbial growth quantification, demonstrating how Raman-SIP can measure the growth of microorganisms with doubling times ranging from hours to years. Furthermore, we correlatively compare our Raman-derived measurements with those of nanoscale secondary ion mass spectrometry (nanoSIMS) to compare the relative strengths of nanoSIMS- and Raman-based SIP approaches. We find that Raman microspectroscopy is a robust, accessible methodology that can readily differentiate and quantify the growth of individual microbial cells in complex samples.\nTime-resolved transcriptomics reveal a mechanism of host niche defense: beneficial root endophytes deploy a host-protective antimicrobial GH18-CBM5 chitinase\nAuthors: Eichfeld, R.; Mahdi, L. K.; De Quattro, C.; Armbruster, L.; Endeshaw, A. B.; Miyauchi, S.; Hellmann, M. J.; Cord-Landwehr, S.; Grigoriev, I.; Peterson, D.; Singan, V.; Lail, K.; Savage, E.; Ng, V.; Langen, G.; Moerschbacher, B.; Zuccaro, A.\nScore: 9.9, Published: 2023-12-29 DOI: 10.1101/2023.12.29.572992\nAssociations between plants and beneficial root-endophytic fungi enhance plant performance by improving nutrient uptake, abiotic stress tolerance and disease resistance. To successfully colonize different host plants and defend their host niche against competing microbes, but also to cooperate with beneficial bacterial members of the microbiota, root endophytes such as Sebacinales secrete a multitude of tightly regulated effector-proteins and carbohydrate-active enzymes. However, the functions, specificity, and regulation of these proteins remain poorly understood. In this study, we employ time-resolved transcriptomics to analyse the gene expression profiles of two Sebacinales members interacting with organisms from different kingdoms of life. We identified crucial genes for plant colonization and intermicrobial competition, including a fungal GH18-CBM5 chitinase specifically upregulated in response to the phytopathogenic fungus Bipolaris sorokiniana. This chitinase protects the plant hosts against the pathogen, reducing fungal biomass and disease symptoms in barley and Arabidopsis thaliana. Our findings shed light on interaction partner specific gene expression in Sebacinales endophytes, with potential applications in enhancing plant health and resilience.\nSustained 3D isotropic imaging of subcellular dynamics using adaptive VCD light-field microscopy 2.0\nAuthors: Zhu, L.; Sun, J.; Yi, C.; Zhang, M.; Niu, G.; Tang, J.; Zhang, Y.; Li, D.; Fei, P.; Chen, L.; He, M.; Zhang, Y.; Huang, Y.; Zheng, C.; Wang, J.\nScore: 16.4, Published: 2023-12-26 DOI: 10.1101/2023.03.15.532876\nLong-term and high-spatiotemporal-resolution 3D imaging of living cells remains an unmet challenge for super-resolution microscopy, owing to the noticeable phototoxicity and limited scanning speed. While emerging light-field microscopy can mitigate this issue through threedimensionally capturing biological dynamics with merely single snapshot, it suffers from suboptimal resolution insufficient for resolving subcellular structures. Here we propose an adaptive light-field microscopy (VCD-LFM 2.0) with new optics-aware and adaptive-tuning deep learning strategies (VCD 2.0) capable for highly-generalizable light-field reconstruction of diverse subcellular dynamics. VCD-LFM 2.0 accomplishes isotropic spatial resolution down to 120 nm without compromising superior temporal resolution and low phototoxicity. With the development of a compact light-field add-on and an open-source reconstruction software, we realize rapid (up to 333 volumes per second), sustained (up to 60 hours) 3D super-resolution imaging of diverse intracellular dynamics simply on a retrofitted inverted microscope. With VCD-LFM 2.0 approach, we quantitatively analyze both the instantaneous processes of lysosome-mitochondrial interaction and long-term evolution of mitochondrial across two complete cell cycles.\nSubstrate complexity buffers negative interactions in a synthetic microbial community of leaf litter degraders.\nAuthors: Abdoli, P.; Vulin, C.; Lepiz, M.; Chase, A. B.; Weihe, C.; Rodriguez-Verdugo, A.\nScore: 6.1, Published: 2023-12-24 DOI: 10.1101/2023.12.23.573222\nMicrobes associated with leaf litter, the top layer of soil, collectively decompose organic matter such as plant polysaccharides, and respire carbon dioxide, regulating the land-atmosphere fluxes of carbon. Therefore, it is crucial to understand the processes limiting biopolymer degradation and their influences on soil community properties. For example, it is still unclear how substrate complexity - defined as the structure of the saccharide and the amount of external processing by extracellular enzymes - influences species interactions and species coexistence. Here, we tested the hypothesis that growth on monosaccharides (i.e., xylose) promotes negative interactions through resource competition, and growth on polysaccharides (i.e., xylan) promotes neutral or positive interactions through resource partitioning or synergism among extracellular enzymes. We assembled a three-species community of leaf litter-degrading bacteria isolated from a grassland site in Southern California. In the polysaccharide xylan, pairs of species stably coexisted and grew equally in co-culture and in monoculture. Conversely, in the monosaccharide xylose, competitive exclusion and negative interactions prevailed. These pairwise dynamics remained consistent in a three-species community: all three species coexisted in xylan, while only two species coexisted in xylose. A mathematical model parameterized from single-species growth behaviors showed that in xylose these dynamics could be explained by resource competition. Instead, the resource competition model could not predict the coexistence patterns in xylan. Overall, our study shows that substrate complexity influences species interactions and patterns of coexistence in a synthetic microbial community of leaf litter degraders that can serve as a model for studying carbon cycling and climate change. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=112 SRC=\"FIGDIR/small/573222v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (35K): org.highwire.dtl.DTLVardef@364b81org.highwire.dtl.DTLVardef@970a77org.highwire.dtl.DTLVardef@303a1eorg.highwire.dtl.DTLVardef@37a87e_HPS_FORMAT_FIGEXP M_FIG C_FIG\nCyclo(Pro-Tyr) elicits conserved cellular damage in fungi by targeting the ATPase Pma1 in plasma membrane domains\nAuthors: Vela-Corcia, D.; Hierrezuelo, J.; Perez-Lorente, A. I.; Stincone, P.; Pakkir Shah, A. K.; Grelard, A.; de Vicente, A.; Perez-Garcia, A.; Loquet, A.; Petras, D.; Romero, D.\nScore: 6.0, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573620\nBioactive metabolites play a crucial role in shaping interactions among diverse organisms. In this study, we identified cyclo(Pro-Tyr), a metabolite produced by Bacillus velezensis, as a potent inhibitor of Botrytis cinerea and Caenorhabditis elegans, two potential cohabitant eukaryotic organisms. Based on our investigation, cyclo(Pro-Tyr) disrupts plasma membrane polarization, induces oxidative stress and increases membrane fluidity, which compromises fungal membrane integrity. These cytological and physiological changes induced by cyclo(Pro-Tyr) may be triggered by the destabilization of membrane microdomains containing the [H+]ATPase Pma1. In response to cyclo(Pro-Tyr) stress, fungal cells activate a transcriptomic and metabolomic response, which primarily involves lipid metabolism and Reactive Oxygen Species (ROS) detoxification, to mitigate membrane damage. This similar response occurs in the nematode C. elegans, suggesting that cyclo(Pro-Tyr) universally targets eukaryotic cellular membranes.\nVirion morphology and on-virus spike protein structures of diverse SARS-CoV-2 variants\nAuthors: Ke, Z.; Peacock, T. P.; Brown, J. C.; Sheppard, C. M.; Croll, T. I.; Kotecha, A.; Goldhill, D. H.; Barclay, W. S.; Briggs, J. A.\nScore: 17.0, Published: 2023-12-22 DOI: 10.1101/2023.12.21.572824\nThe evolution of SARS-CoV-2 variants with increased fitness has been accompanied by structural changes in the spike (S) proteins that are the major target for the adaptive immune response. Single-particle cryo-EM analysis of soluble S from SARS-CoV-2 variants has revealed this structural adaptation at high-resolution. The analysis of S trimers in situ on intact virions has the potential to provide more functionally relevant insights into S structure and virion morphology. Here, we characterized B.1, Alpha, Beta, Gamma, Delta, Kappa, and Mu variants by cryo-electron microscopy and tomography, assessing S cleavage, virion morphology, S incorporation, \"in-situ\" high-resolution S structures and the range of S conformational states. We found no evidence for adaptive changes in virion morphology, but describe multiple different positions in the S protein where amino acid changes alter local protein structure. Considered together, our data is consistent with a model where amino acid changes at multiple positions from the top to the base of the spike cause structural changes that can modulate the conformational dynamics of S.\n",
  "wordCount" : "2440",
  "inLanguage": "en",
  "datePublished": "2023-12-31T10:38:19Z",
  "dateModified": "2023-12-31T10:38:19Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 31, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572655">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572655" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572655">
        <p class="paperTitle">Persistence of an infectious form of SARS-CoV-2 post protease inhibitor treatment of permissive cells in vitro</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572655" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572655" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nair, M. S.; Luck, M. I.; Huang, Y.; Sabo, Y.; Ho, D. D.</p>
        <p class="info">Score: 68.0, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572655' target='https://doi.org/10.1101/2023.12.20.572655'> 10.1101/2023.12.20.572655</a></p>
        <p class="abstract">Reports have described SARS-CoV-2 rebound in COVID-19 patients treated with nirmatrelvir, a 3CL protease inhibitor. The cause remains a mystery, although drug resistance, re-infection, and lack of adequate immune responses have been excluded. We now present virologic findings that provide a clue to the cause of viral rebound, which occurs in [~]20% of the treated cases. The persistence of an intermediary form of infectious SARS-CoV-2 was experimentally documented in vitro after treatment with nirmatrelvir or another 3CL protease inhibitor, but not with a polymerase inhibitor, remdesivir. This infectious intermediate decayed slowly with a half-life of [~]1 day, suggesting that its persistence could outlive the treatment course to re-ignited SARS-CoV-2 infection as the drug is eliminated. Additional studies are needed to define the nature of this viral intermediate, but our findings point to a particular direction for future investigation and offer a specific treatment recommendation that should be tested clinically.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.571617">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.571617" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.571617">
        <p class="paperTitle">Extensive variation and strain-specificity in dengue virus susceptibility among African Aedes aegypti populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.571617" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.571617" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dabo, S.; Henrion-Lacritick, A.; Lecuyer, A.; Jiolle, D.; Paupy, C.; Ayala, D.; da Veiga Leal, S.; Badolo, A.; Vega-Rua, A.; Sylla, M.; Akorli, J.; Otoo, S.; Lutomiah, J.; Sang, R.; Mutebi, J.-P.; Saleh, M.-C.; Rose, N.; McBride, C.; Lambrechts, L.</p>
        <p class="info">Score: 39.7, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.571617' target='https://doi.org/10.1101/2023.12.14.571617'> 10.1101/2023.12.14.571617</a></p>
        <p class="abstract">African populations of the mosquito Aedes aegypti are usually considered less susceptible to infection by human-pathogenic flaviviruses than globally invasive populations found outside Africa. Although this contrast has been well documented for Zika virus (ZIKV), it is unclear to what extent it is true for dengue virus (DENV), the most prevalent flavivirus of humans. Addressing this question is complicated by substantial genetic diversity among DENV strains, most notably in the form of four genetic types (DENV1 to DENV4), that can lead to genetically specific interactions with mosquito populations. Here, we carried out a continent-wide survey of DENV susceptibility using a panel of field-derived Ae. aegypti colonies from across the African range of the species and a colony from Guadeloupe, French West Indies as non-African reference. We found considerable variation in the ability of African Ae. aegypti populations to acquire and replicate a panel of six DENV strains spanning the four DENV types. Although African Ae. aegypti populations were generally less susceptible than the reference non-African population from Guadeloupe, in several instances some African populations were equally or more susceptible than the Guadeloupe population. Moreover, the relative level of susceptibility between African mosquito populations depended on the DENV strain, indicating genetically specific interactions. We conclude that unlike ZIKV susceptibility, there is no clear-cut dichotomy in DENV susceptibility between African and non-African Ae. aegypti. DENV susceptibility of African Ae. aegypti populations is highly heterogeneous and largely governed by the specific pairing of mosquito population and DENV strain.

Author summaryAfrican populations of the mosquito Aedes aegypti are usually thought to be less likely to get infected by flaviviruses compared to Ae. aegypti mosquitoes found outside Africa. While this has been well-demonstrated for Zika virus, it is not clear if the same is true for dengue virus, which is the most common flavivirus in humans. Studying this is complicated by the strain diversity of dengue virus, including four main genetic types, potentially causing different interactions. In this study, we compared several mosquito populations and found that, in general, African mosquitoes were less likely to get infected by dengue virus compared to mosquitoes from outside Africa. However, in some cases, African mosquitoes were just as or even more likely to get infected. The specific strain of dengue virus also influenced how likely African mosquitoes were to get infected, showing that the relationship between African mosquitoes and dengue virus is complex.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.570782">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.570782" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.570782">
        <p class="paperTitle">Virological characteristics of the SARS-CoV-2 JN.1 variant</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.570782" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.570782" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium,  ; Zahradnik, J.; Ito, J.; Sato, K.</p>
        <p class="info">Score: 763.1, Published: 2023-12-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.570782' target='https://doi.org/10.1101/2023.12.08.570782'> 10.1101/2023.12.08.570782</a></p>
        <p class="abstract">The SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG.5.1 and HK.3. Comparing to XBB and BA.2, BA.2.86 carries more than 30 mutations in the spike (S) protein, indicating a high potential for immune evasion. BA.2.86 has evolved and its descendant, JN.1 (BA.2.86.1.1), emerged in late 2023. JN.1 harbors S:L455S and three mutations in non-S proteins. S:L455S is a hallmark mutation of JN.1: we have recently shown that HK.3 and other &#34;FLip&#34; variants carry S:L455F, which contributes to increased transmissibility and immune escape ability compared to the parental EG.5.1 variant. Here, we investigated the virological properties of JN.1.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.573102">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.573102" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.573102">
        <p class="paperTitle">Novel nitrifying symbiont lineages are vertically inherited and widespread in marine sponges</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.573102" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.573102" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Glasl, B.; Luter, H. M.; Damjanovic, K.; Kitzinger, K.; Mueller, A. J.; Mahler, L.; Engelberts, J. P.; Rix, L.; Osvatic, J. T.; Hausmann, B.; Seneca, J.; Daims, H.; Pjevac, P.; Wagner, M.</p>
        <p class="info">Score: 12.3, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.573102' target='https://doi.org/10.1101/2023.12.23.573102'> 10.1101/2023.12.23.573102</a></p>
        <p class="abstract">Ammonia-oxidising archaea and nitrite-oxidising bacteria are common members of marine sponge microbiomes. They derive energy for carbon fixation and growth from nitrification - the oxidation of ammonia to nitrite and further to nitrate - and are proposed to play essential roles in the carbon and nitrogen cycling of sponge holobionts. In this study, we characterise two novel nitrifying symbiont lineages,  Ca. Nitrosokoinonia and  Ca. Nitrosymbion in the marine sponge Coscinoderma matthewsi using a combination of molecular tools, single-cell imaging techniques, and physiological rate measurements. Both represent a new genus in the ammonia-oxidising archaeal class Nitrososphaeria and the nitrite-oxidising bacterial order Nitrospirales, respectively. Furthermore, we show that larvae of this viviparous sponge are densely colonised by representatives of  Ca. Nitrosokoinonia and  Ca. Nitrosymbion indicating vertical transmission. In adults, the representatives of both symbiont genera are located extracellularly in the mesohyl. Comparative metagenome analyses and physiological data suggest that ammonia-oxidising archaeal symbionts of the genus Ca. Nitrosokoinonia strongly rely on endogenously produced nitrogenous compounds (i.e., ammonium, urea, nitriles/cyanides, and creatinine) rather than on exogenous ammonium sources taken up by the sponge. Additionally, the nitrite-oxidising bacterial symbionts Ca. Nitrosymbion may reciprocally support the ammonia-oxidisers with ammonia via the utilisation of sponge-derived urea and cyanate. Interestingly, comparative analyses of published environmental 16S rRNA amplicon data revealed that  Ca. Nitrosokoinonia and  Ca. Nitrosymbion are widely distributed and predominantly associated with marine sponges and corals, suggesting a broad relevance of our findings.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.16.571966">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.16.571966" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.16.571966">
        <p class="paperTitle">Quantitative measurement of microbial growth rate with Raman microspectroscopy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.16.571966" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.16.571966" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Caro, T. A.; Kashyap, S.; Brown, G.; Chen, C.; Kopf, S.; Templeton, A.</p>
        <p class="info">Score: 12.0, Published: 2023-12-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.16.571966' target='https://doi.org/10.1101/2023.12.16.571966'> 10.1101/2023.12.16.571966</a></p>
        <p class="abstract">Rates of microbial activity and growth are fundamental to understanding environmental geochemistry and ecology. However, measuring the heterogeneity of microbial activity at the single-cell level, especially within complex populations and environmental matrices, remains a forefront challenge. Stable Isotope Probing (SIP) is a standard method for assessing microbial activity and involves measuring the incorporation of an isotopically labeled compound into microbial biomass. Here, we assess the utility of Raman microspectroscopy as a SIP technique, specifically focusing on the measurement of deuterium (2H), a tracer of microbial biomass production. We generate calibrations of microbial biomass 2H values and find that Raman microspectroscopy reliably quantifies 2H incorporation ranging between 0 and 40 at. %. Applying the results of this calibration to a SIP model, we explicitly parameterize the factors controlling microbial growth quantification, demonstrating how Raman-SIP can measure the growth of microorganisms with doubling times ranging from hours to years. Furthermore, we correlatively compare our Raman-derived measurements with those of nanoscale secondary ion mass spectrometry (nanoSIMS) to compare the relative strengths of nanoSIMS- and Raman-based SIP approaches. We find that Raman microspectroscopy is a robust, accessible methodology that can readily differentiate and quantify the growth of individual microbial cells in complex samples.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.572992">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.572992" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.572992">
        <p class="paperTitle">Time-resolved transcriptomics reveal a mechanism of host niche defense: beneficial root endophytes deploy a host-protective antimicrobial GH18-CBM5 chitinase</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.572992" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.572992" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eichfeld, R.; Mahdi, L. K.; De Quattro, C.; Armbruster, L.; Endeshaw, A. B.; Miyauchi, S.; Hellmann, M. J.; Cord-Landwehr, S.; Grigoriev, I.; Peterson, D.; Singan, V.; Lail, K.; Savage, E.; Ng, V.; Langen, G.; Moerschbacher, B.; Zuccaro, A.</p>
        <p class="info">Score: 9.9, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.572992' target='https://doi.org/10.1101/2023.12.29.572992'> 10.1101/2023.12.29.572992</a></p>
        <p class="abstract">Associations between plants and beneficial root-endophytic fungi enhance plant performance by improving nutrient uptake, abiotic stress tolerance and disease resistance. To successfully colonize different host plants and defend their host niche against competing microbes, but also to cooperate with beneficial bacterial members of the microbiota, root endophytes such as Sebacinales secrete a multitude of tightly regulated effector-proteins and carbohydrate-active enzymes. However, the functions, specificity, and regulation of these proteins remain poorly understood. In this study, we employ time-resolved transcriptomics to analyse the gene expression profiles of two Sebacinales members interacting with organisms from different kingdoms of life. We identified crucial genes for plant colonization and intermicrobial competition, including a fungal GH18-CBM5 chitinase specifically upregulated in response to the phytopathogenic fungus Bipolaris sorokiniana. This chitinase protects the plant hosts against the pathogen, reducing fungal biomass and disease symptoms in barley and Arabidopsis thaliana. Our findings shed light on interaction partner specific gene expression in Sebacinales endophytes, with potential applications in enhancing plant health and resilience.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.03.15.532876">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.03.15.532876" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.03.15.532876">
        <p class="paperTitle">Sustained 3D isotropic imaging of subcellular dynamics using adaptive VCD light-field microscopy 2.0</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.03.15.532876" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.03.15.532876" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhu, L.; Sun, J.; Yi, C.; Zhang, M.; Niu, G.; Tang, J.; Zhang, Y.; Li, D.; Fei, P.; Chen, L.; He, M.; Zhang, Y.; Huang, Y.; Zheng, C.; Wang, J.</p>
        <p class="info">Score: 16.4, Published: 2023-12-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.03.15.532876' target='https://doi.org/10.1101/2023.03.15.532876'> 10.1101/2023.03.15.532876</a></p>
        <p class="abstract">Long-term and high-spatiotemporal-resolution 3D imaging of living cells remains an unmet challenge for super-resolution microscopy, owing to the noticeable phototoxicity and limited scanning speed. While emerging light-field microscopy can mitigate this issue through threedimensionally capturing biological dynamics with merely single snapshot, it suffers from suboptimal resolution insufficient for resolving subcellular structures. Here we propose an adaptive light-field microscopy (VCD-LFM 2.0) with new optics-aware and adaptive-tuning deep learning strategies (VCD 2.0) capable for highly-generalizable light-field reconstruction of diverse subcellular dynamics. VCD-LFM 2.0 accomplishes isotropic spatial resolution down to 120 nm without compromising superior temporal resolution and low phototoxicity. With the development of a compact light-field add-on and an open-source reconstruction software, we realize rapid (up to 333 volumes per second), sustained (up to 60 hours) 3D super-resolution imaging of diverse intracellular dynamics simply on a retrofitted inverted microscope. With VCD-LFM 2.0 approach, we quantitatively analyze both the instantaneous processes of lysosome-mitochondrial interaction and long-term evolution of mitochondrial across two complete cell cycles.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.573222">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.573222" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.573222">
        <p class="paperTitle">Substrate complexity buffers negative interactions in a synthetic microbial community of leaf litter degraders.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.573222" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.573222" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Abdoli, P.; Vulin, C.; Lepiz, M.; Chase, A. B.; Weihe, C.; Rodriguez-Verdugo, A.</p>
        <p class="info">Score: 6.1, Published: 2023-12-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.573222' target='https://doi.org/10.1101/2023.12.23.573222'> 10.1101/2023.12.23.573222</a></p>
        <p class="abstract">Microbes associated with leaf litter, the top layer of soil, collectively decompose organic matter such as plant polysaccharides, and respire carbon dioxide, regulating the land-atmosphere fluxes of carbon. Therefore, it is crucial to understand the processes limiting biopolymer degradation and their influences on soil community properties. For example, it is still unclear how substrate complexity - defined as the structure of the saccharide and the amount of external processing by extracellular enzymes - influences species interactions and species coexistence. Here, we tested the hypothesis that growth on monosaccharides (i.e., xylose) promotes negative interactions through resource competition, and growth on polysaccharides (i.e., xylan) promotes neutral or positive interactions through resource partitioning or synergism among extracellular enzymes. We assembled a three-species community of leaf litter-degrading bacteria isolated from a grassland site in Southern California. In the polysaccharide xylan, pairs of species stably coexisted and grew equally in co-culture and in monoculture. Conversely, in the monosaccharide xylose, competitive exclusion and negative interactions prevailed. These pairwise dynamics remained consistent in a three-species community: all three species coexisted in xylan, while only two species coexisted in xylose. A mathematical model parameterized from single-species growth behaviors showed that in xylose these dynamics could be explained by resource competition. Instead, the resource competition model could not predict the coexistence patterns in xylan. Overall, our study shows that substrate complexity influences species interactions and patterns of coexistence in a synthetic microbial community of leaf litter degraders that can serve as a model for studying carbon cycling and climate change.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=112 SRC=&#34;FIGDIR/small/573222v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (35K):
org.highwire.dtl.DTLVardef@364b81org.highwire.dtl.DTLVardef@970a77org.highwire.dtl.DTLVardef@303a1eorg.highwire.dtl.DTLVardef@37a87e_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.573620">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.573620" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.573620">
        <p class="paperTitle">Cyclo(Pro-Tyr) elicits conserved cellular damage in fungi by targeting the ATPase Pma1 in plasma membrane domains</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.573620" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.573620" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vela-Corcia, D.; Hierrezuelo, J.; Perez-Lorente, A. I.; Stincone, P.; Pakkir Shah, A. K.; Grelard, A.; de Vicente, A.; Perez-Garcia, A.; Loquet, A.; Petras, D.; Romero, D.</p>
        <p class="info">Score: 6.0, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.573620' target='https://doi.org/10.1101/2023.12.29.573620'> 10.1101/2023.12.29.573620</a></p>
        <p class="abstract">Bioactive metabolites play a crucial role in shaping interactions among diverse organisms. In this study, we identified cyclo(Pro-Tyr), a metabolite produced by Bacillus velezensis, as a potent inhibitor of Botrytis cinerea and Caenorhabditis elegans, two potential cohabitant eukaryotic organisms. Based on our investigation, cyclo(Pro-Tyr) disrupts plasma membrane polarization, induces oxidative stress and increases membrane fluidity, which compromises fungal membrane integrity. These cytological and physiological changes induced by cyclo(Pro-Tyr) may be triggered by the destabilization of membrane microdomains containing the [H&#43;]ATPase Pma1. In response to cyclo(Pro-Tyr) stress, fungal cells activate a transcriptomic and metabolomic response, which primarily involves lipid metabolism and Reactive Oxygen Species (ROS) detoxification, to mitigate membrane damage. This similar response occurs in the nematode C. elegans, suggesting that cyclo(Pro-Tyr) universally targets eukaryotic cellular membranes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572824">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572824" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572824">
        <p class="paperTitle">Virion morphology and on-virus spike protein structures of diverse SARS-CoV-2 variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572824" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572824" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ke, Z.; Peacock, T. P.; Brown, J. C.; Sheppard, C. M.; Croll, T. I.; Kotecha, A.; Goldhill, D. H.; Barclay, W. S.; Briggs, J. A.</p>
        <p class="info">Score: 17.0, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572824' target='https://doi.org/10.1101/2023.12.21.572824'> 10.1101/2023.12.21.572824</a></p>
        <p class="abstract">The evolution of SARS-CoV-2 variants with increased fitness has been accompanied by structural changes in the spike (S) proteins that are the major target for the adaptive immune response. Single-particle cryo-EM analysis of soluble S from SARS-CoV-2 variants has revealed this structural adaptation at high-resolution. The analysis of S trimers in situ on intact virions has the potential to provide more functionally relevant insights into S structure and virion morphology. Here, we characterized B.1, Alpha, Beta, Gamma, Delta, Kappa, and Mu variants by cryo-electron microscopy and tomography, assessing S cleavage, virion morphology, S incorporation, &#34;in-situ&#34; high-resolution S structures and the range of S conformational states. We found no evidence for adaptive changes in virion morphology, but describe multiple different positions in the S protein where amino acid changes alter local protein structure. Considered together, our data is consistent with a model where amino acid changes at multiple positions from the top to the base of the spike cause structural changes that can modulate the conformational dynamics of S.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
