<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Viroid-like colonists of human microbiomes
Authors: Zheludev, I. N.; Edgar, R. C.; Galiano, M. J. L.; De la Pena, M.; Babaian, A.; Bhatt, A. S.; Fire, A. Z.
Score: 801.3, Published: 2024-01-21 DOI: 10.1101/2024.01.20.576352
Here, we describe the &#34;Obelisks,&#34; a previously unrecognised class of viroid-like elements that we first identified in human gut metatranscriptomic data. &#34;Obelisks&#34; share several properties: (i) apparently circular RNA [~]1kb genome assemblies, (ii) predicted rod-like secondary structures encompassing the entire genome, and (iii) open reading frames coding for a novel protein superfamily, which we call the &#34;Oblins&#34;.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Viroid-like colonists of human microbiomes
Authors: Zheludev, I. N.; Edgar, R. C.; Galiano, M. J. L.; De la Pena, M.; Babaian, A.; Bhatt, A. S.; Fire, A. Z.
Score: 801.3, Published: 2024-01-21 DOI: 10.1101/2024.01.20.576352
Here, we describe the &#34;Obelisks,&#34; a previously unrecognised class of viroid-like elements that we first identified in human gut metatranscriptomic data. &#34;Obelisks&#34; share several properties: (i) apparently circular RNA [~]1kb genome assemblies, (ii) predicted rod-like secondary structures encompassing the entire genome, and (iii) open reading frames coding for a novel protein superfamily, which we call the &#34;Oblins&#34;." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-11T10:38:55+00:00" />
<meta property="article:modified_time" content="2024-02-11T10:38:55+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Viroid-like colonists of human microbiomes
Authors: Zheludev, I. N.; Edgar, R. C.; Galiano, M. J. L.; De la Pena, M.; Babaian, A.; Bhatt, A. S.; Fire, A. Z.
Score: 801.3, Published: 2024-01-21 DOI: 10.1101/2024.01.20.576352
Here, we describe the &#34;Obelisks,&#34; a previously unrecognised class of viroid-like elements that we first identified in human gut metatranscriptomic data. &#34;Obelisks&#34; share several properties: (i) apparently circular RNA [~]1kb genome assemblies, (ii) predicted rod-like secondary structures encompassing the entire genome, and (iii) open reading frames coding for a novel protein superfamily, which we call the &#34;Oblins&#34;."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Viroid-like colonists of human microbiomes\nAuthors: Zheludev, I. N.; Edgar, R. C.; Galiano, M. J. L.; De la Pena, M.; Babaian, A.; Bhatt, A. S.; Fire, A. Z.\nScore: 801.3, Published: 2024-01-21 DOI: 10.1101/2024.01.20.576352\nHere, we describe the \u0026#34;Obelisks,\u0026#34; a previously unrecognised class of viroid-like elements that we first identified in human gut metatranscriptomic data. \u0026#34;Obelisks\u0026#34; share several properties: (i) apparently circular RNA [~]1kb genome assemblies, (ii) predicted rod-like secondary structures encompassing the entire genome, and (iii) open reading frames coding for a novel protein superfamily, which we call the \u0026#34;Oblins\u0026#34;.",
  "keywords": [
    
  ],
  "articleBody": " Viroid-like colonists of human microbiomes\nAuthors: Zheludev, I. N.; Edgar, R. C.; Galiano, M. J. L.; De la Pena, M.; Babaian, A.; Bhatt, A. S.; Fire, A. Z.\nScore: 801.3, Published: 2024-01-21 DOI: 10.1101/2024.01.20.576352\nHere, we describe the \"Obelisks,\" a previously unrecognised class of viroid-like elements that we first identified in human gut metatranscriptomic data. \"Obelisks\" share several properties: (i) apparently circular RNA [~]1kb genome assemblies, (ii) predicted rod-like secondary structures encompassing the entire genome, and (iii) open reading frames coding for a novel protein superfamily, which we call the \"Oblins\". We find that Obelisks form their own distinct phylogenetic group with no detectable sequence or structural similarity to known biological agents. Further, Obelisks are prevalent in tested human microbiome metatranscriptomes with representatives detected in [~]7% of analysed stool metatranscriptomes (29/440) and in [~]50% of analysed oral metatranscriptomes (17/32). Obelisk compositions appear to differ between the anatomic sites and are capable of persisting in individuals, with continued presence over \u003e300 days observed in one case. Large scale searches identified 29,959 Obelisks (clustered at 90% nucleotide identity), with examples from all seven continents and in diverse ecological niches. From this search, a subset of Obelisks are identified to code for Obelisk-specific variants of the hammerhead type-III self-cleaving ribozyme. Lastly, we identified one case of a bacterial species (Streptococcus sanguinis) in which a subset of defined laboratory strains harboured a specific Obelisk RNA population. As such, Obelisks comprise a class of diverse RNAs that have colonised, and gone unnoticed in, human, and global microbiomes.\nViral interference between severe acute respiratory syndrome coronavirus 2 and influenza A viruses\nAuthors: Gilbert-girard, S.; Piret, J.; Carbonneau, J.; Henaut, M.; Goyette, N.; Boivin, G.\nScore: 61.3, Published: 2024-02-05 DOI: 10.1101/2024.02.02.578538\nSome respiratory viruses can cause a viral interference through the activation of the interferon (IFN) pathway that reduces the replication of another virus. Epidemiological studies of coinfections between SARS-CoV-2 and other respiratory viruses have been hampered by non-pharmaceutical measures applied to mitigate the spread of SARS-CoV-2 during the COVID-19 pandemic. With the ease of these interventions, SARS-CoV-2 and influenza A viruses can now co-circulate. It is thus of prime importance to characterize their interactions. In this work, we investigated viral interference effects between an Omicron variant and a contemporary influenza A/H3N2 strain, in comparison with an ancestral SARS-CoV-2 strain and the 2009 pandemic influenza A/H1N1 virus. We infected nasal human airway epitheliums with SARS-CoV-2 and influenza, either simultaneously or 24 h apart. Viral load was measured by RT-qPCR and IFN-/{beta}/{lambda}1/{lambda}2 proteins were quantified by immunoassay. Expression of four interferon-stimulated genes (ISGs; OAS1/IFITM3/ISG15/MxA) was also measured by RT-droplet digital PCR. Additionally, susceptibility of each virus to IFN-/{beta}/{lambda}2 recombinant proteins was determined. Our results showed that influenza A, and especially A/H3N2, interfered with both SARS-CoV-2 viruses, but that SARS-CoV-2 only interfered with A/H1N1. Consistently with these results, influenza, and particularly the A/H3N2 strain, caused a higher production of IFN proteins and expression of ISGs than SARS-CoV-2. The IFN production induced by SARS-CoV-2 was marginal and its presence during coinfections with influenza was associated with a reduced IFN response. All viruses were susceptible to exogenous IFNs, with the ancestral SARS-CoV-2 and Omicron being less susceptible to type I and type III IFNs, respectively. Thus, influenza A causes a viral interference towards SARS-CoV-2 most likely through an IFN response. The opposite is not necessarily true, and a concurrent infection with both viruses leads to a lower IFN response. Taken together, these results help us to understand how SARS-CoV-2 interacts with another major respiratory pathogen. Author summaryDuring the COVID-19 pandemic, non-pharmaceutical measures were able to reduce the spread of SARS-CoV-2 and most respiratory viruses. Since the ease of these measures, SARS-CoV-2 variants and other viruses, such as influenza A, have started to co-circulate and can now infect a same host and interact with each other. These interactions can lead to attenuated or aggravated infections and can affect the timing of epidemics. Therefore, it is very important to elucidate how the new SARS-CoV-2 interacts with other viruses to better predict their implications in human health and their epidemic activity. Our work contributes to better understand these interactions using viruses that have likely co-circulated after lifting mitigation interventions, i.e., SARS-CoV-2 Omicron variant and a contemporary influenza A/H3N2 strain. We studied how each virus may affect the other virus growth and how these interactions were associated with the innate immune response of the host. We found that a prior infection with influenza A can decrease the growth of SARS-CoV-2 while the latter reduces the innate immune response. Our results help to understand the interplay between SARS-CoV-2 and influenza A in the host and may improve mathematical models predicting epidemics.\nA 20-year time-series of a freshwater lake reveals seasonal dynamics and environmental drivers of viral diversity, ecology, and evolution\nAuthors: Zhou, Z.; Tran, P. Q.; Martin, C.; Rohwer, R. R.; Baker, B.; McMahon, K.; Anantharaman, K.\nScore: 43.2, Published: 2024-02-06 DOI: 10.1101/2024.02.06.579183\nLong-term ecological studies are powerful tools to investigate microbiomes and ecosystem change but have mostly ignored viruses. Here, we leveraged a 20-year time-series of a freshwater lake to characterize 1.3 million viral genomes over time, seasonality, and environmental factors. We identified 578 auxiliary metabolic gene (AMG) clusters representing over 150,000 AMGs, the most abundant of which, including psbA for photosynthesis, pmoC for methane oxidation, and katG for hydrogen peroxide decomposition, were consistently represented in viruses across decades and seasons. We observed positive associations and niche differentiation between virus-host pairs during seasonal change including in keystone taxa, Cyanobacteria, methanotrophs, and Nanopelagicales. Environmental constraints, specifically inorganic carbon and ammonium influenced viral abundances over time, and highlighted roles of viruses in both \"top-down\" and \"bottom-up\" interactions. Key evolutionary processes shaping gene and genome-wide selection included favored fitness genes, reduced genomic heterogeneity, and dominant sub-populations carrying specific genes. Overall, our study advances understanding of diversity, ecological dynamics, and evolutionary trajectories of viruses in Earths microbiomes and ecosystems.\nMapping glycoprotein structure reveals defining events in the evolution of the Flaviviridae\nAuthors: Mifsud, J. C. O.; Lytras, S.; Oliver, M. R.; Toon, K.; Costa, V. A.; Holmes, E. C.; Grove, J.\nScore: 39.2, Published: 2024-02-06 DOI: 10.1101/2024.02.06.579159\nViral glycoproteins drive membrane fusion in enveloped viruses and determine host range, tissue tropism and pathogenesis. Despite their importance, there is a fragmentary understanding of glycoproteins within the Flaviviridae; for many species the glycoproteins have not yet been identified, for others, such as the hepaciviruses, the molecular mechanisms of membrane fusion remain uncharacterised. Here, we combine comprehensive phylogenetic analyses with systematic protein structure prediction to survey glycoproteins across the entire Flaviviridae. We discover class-II fusion systems, homologous to the orthoflavivirus E glycoprotein, in most species, including highly-divergent jingmenviruses and large genome flaviviruses. However, the E1E2 glycoproteins of the hepaci-, pegi- and pestiviruses are structurally distinct, may represent a novel class of fusion mechanism, and are strictly associated with infection of vertebrate hosts. By mapping glycoprotein distribution onto the underlying phylogeny we reveal a complex history of evolutionary events that have shaped the diverse virology and ecology of the Flaviviridae.\nAn infection and pathogenesis mouse model of SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR)\nAuthors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.\nScore: 4435.0, Published: 2024-01-21 DOI: 10.1101/2024.01.03.574008\nSARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) is highly attenuated, but can cause mortality in a specifically designed human ACE2-transgenic mouse model, making it an invaluable surrogate model for evaluating the efficacy of drugs and vaccines against SARS-CoV-2.\nDeep mutational scanning reveals functional constraints and antigenic variability of Lassa virus glycoprotein complex\nAuthors: Carr, C. R.; Crawford, K. H.; Murphy, M.; Galloway, J. G.; Haddox, H. K.; Matsen, F. A.; Andersen, K. G.; King, N. P.; Bloom, J. D.\nScore: 20.4, Published: 2024-02-06 DOI: 10.1101/2024.02.05.579020\nLassa virus is estimated to cause thousands of human deaths per year, primarily due to spillovers from its natural host, Mastomys rodents. Efforts to create vaccines and antibody therapeutics must account for the evolutionary variability of Lassa viruss glycoprotein complex (GPC), which mediates viral entry into cells and is the target of neutralizing antibodies. To map the evolutionary space accessible to GPC, we use pseudovirus deep mutational scanning to measure how nearly all GPC amino-acid mutations affect cell entry and antibody neutralization. Our experiments define functional constraints throughout GPC. We quantify how GPC mutations affect neutralization by a panel of monoclonal antibodies and show that all antibodies are escaped by mutations that exist among natural Lassa virus lineages. Overall, our work describes a biosafety-level-2 method to elucidate the mutational space accessible to GPC and shows how prospective characterization of antigenic variation could aid design of therapeutics and vaccines.\nYraN is a helicase-associated nuclease fostering extended recombination events by natural transformation\nAuthors: Hardy, L.; Plantade, J.; Morales, V.; Mazzamurro, F.; Rocha, E. P. C.; Polard, P.; Charpentier, X.\nScore: 18.6, Published: 2024-02-09 DOI: 10.1101/2024.02.06.579203\nNatural transformation is a widespread molecular pathway of horizontal gene transfer involving the uptake and recombination of exogenous DNA. Exogenous DNA follows a pathway involving genes sequentially required for its capture, internalization, protection, and recombination with the chromosome. Most of these genes were identified through the isolation of transformation-defective mutants and/or based on their expression preceding natural transformation. Yet, genes required for key steps of the pathway remain elusive. We sought to identify any missing component by comparing Tn-seq data obtained in two distantly-related transformable diderm species, the human pathogen Legionella pneumophila and the cyanobacterium Synechococcus elongatus. We identified yraN, a widespread and highly conserved gene of unknown function required for natural transformation. We provide evidence that YraN is a nuclease associated with the ComM helicase, which cooperate to process the D-loop formed by the invasion of the transforming DNA in the chromosomal DNA strands. We propose a model in which cleavage of the displaced strand by YraN can promote the recombination of transforming DNA, leading to extended recombination events. The identification of this YraN/ComM nuclease/helicase system supports the hypothesis that bacteria possess a conserved pathway for the transport and recombination of exogenous DNA.\nDe novo mutations mediate phenotypic switching in an opportunistic human lung pathogen\nAuthors: Poret, A. J.; Schaefers, M. M.; Merakou, C.; Mansour, K. E.; Lagoudas, G. K.; Cross, A. R.; Goldberg, J. B.; Kishony, R.; Uluer, A. Z.; McAdam, A. J.; Blainey, P. C.; Vargas, S. O.; Lieberman, T. D.; Priebe, G. P.\nScore: 16.3, Published: 2024-02-06 DOI: 10.1101/2024.02.06.579193\nBacteria evolving within human hosts encounter selective tradeoffs that render mutations adaptive in one context and deleterious in another. Here, we report that the cystic fibrosis-associated pathogen Burkholderia dolosa overcomes in-human selective tradeoffs by acquiring successive point mutations that alternate phenotypes. We sequenced the whole genomes of 931 respiratory isolates from two recently infected patients and an epidemiologically-linked, chronically-infected patient. These isolates are contextualized using 112 historical genomes from the same outbreak strain. Within both newly infected patients, diverse parallel mutations that disrupt O-antigen expression quickly arose, comprising 29% and 63% of their B. dolosa communities by 3 years. The selection for loss of O-antigen starkly contrasts with our previous observation of parallel O-antigen-restoring mutations after many years of chronic infection in the historical outbreak. Experimental characterization revealed that O-antigen loss increases uptake in immune cells while decreasing competitiveness in the mouse lung. We propose that the balance of these pressures, and thus whether O-antigen expression is advantageous, depends on tissue localization and infection duration. These results suggest that mutation-driven alternation during infection may be more frequent than appreciated and is underestimated without dense temporal sampling.\nStratification of viral shedding patterns in saliva of COVID-19 patients\nAuthors: Park, H.; Raiki, Y.; Iwanami, S.; Kim, K.; Ejima, K.; Nakamura, N.; Aihara, K.; Miyazaki, Y.; Umeyama, T.; Miyazawa, K.; Morita, T.; Watashi, K.; Brooke, C. B.; Ke, R.; Iwami, S.; Miyazaki, T.\nScore: 17.0, Published: 2024-02-02 DOI: 10.1101/2024.01.30.578034\nLiving with COVID-19 requires continued vigilance against the spread and emergence of variants of concern (VOCs). Rapid and accurate saliva diagnostic testing, alongside basic public health responses, is a viable option contributing to effective transmission control. Nevertheless, our knowledge regarding the dynamics of SARS-CoV-2 infection in saliva is not as advanced as our understanding of the respiratory tract. Here we analyzed longitudinal viral load data of SARS-CoV-2 in saliva samples from 144 patients with mild COVID-19 (a combination of our collected data and published data). Using a mathematical model, we successfully stratified infection dynamics into three distinct groups with clear patterns of viral shedding: viral shedding durations in the three groups were 11.5 days (95% CI: 10.6 to 12.4), 17.4 days (16.6 to 18.2), and 30.0 days (28.1 to 31.8), respectively. Surprisingly, this stratified grouping remained unexplained despite our analysis of 47 types of clinical data, including basic demographic information, clinical symptoms, results of blood tests, and vital signs. Additionally, we quantified the expression levels of 92 micro-RNAs in a subset of saliva samples, but these also failed to explain the observed stratification, although the mir-1846 level may have been weakly correlated with peak viral load. Our study provides insights into SARS-CoV-2 infection dynamics in saliva, highlighting the challenges in predicting the duration of viral shedding without indicators that directly reflect an individuals immune response, such as antibody induction. Given the significant individual heterogeneity in the kinetics of saliva viral shedding, identifying biomarker(s) for viral shedding patterns will be crucial for improving public health interventions in the era of living with COVID-19.\nSingleM and Sandpiper: Robust microbial taxonomic profiles from metagenomic data\nAuthors: Woodcroft, B. J.; Aroney, S. T. N.; Zhao, R.; Cunningham, M.; Mitchell, J. A. M.; Blackall, L.; Tyson, G. W.\nScore: 55.4, Published: 2024-01-31 DOI: 10.1101/2024.01.30.578060\nDetermining the taxonomy and relative abundance of microorganisms in metagenomic data is a foundational problem in microbial ecology. To address the limitations of existing approaches, we developed SingleM, which estimates community composition using conserved regions within universal marker genes. SingleM accurately profiles complex communities of known microbial species, and is the only tool that detects species without genomic representation, even those representing novel phyla. Given SingleMs computational efficiency, we applied it to 248,559 publicly available metagenomes and show that the vast majority of samples from marine, freshwater, sediment and soil environments are dominated by novel species lacking genomic representation (median relative abundance 75.0%). SingleM also provides a way to identify metagenomes for the recovery of novel metagenome-assembled genomes from lineages of interest, and can incorporate user-recovered genomes into its reference database to improve profiling resolution. Quantifying the full diversity of Bacteria and Archaea in metagenomic data shows that microbial genome databases are far from saturated.\n",
  "wordCount" : "2413",
  "inLanguage": "en",
  "datePublished": "2024-02-11T10:38:55Z",
  "dateModified": "2024-02-11T10:38:55Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 11, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576352">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576352" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576352">
        <p class="paperTitle">Viroid-like colonists of human microbiomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576352" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576352" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zheludev, I. N.; Edgar, R. C.; Galiano, M. J. L.; De la Pena, M.; Babaian, A.; Bhatt, A. S.; Fire, A. Z.</p>
        <p class="info">Score: 801.3, Published: 2024-01-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576352' target='https://doi.org/10.1101/2024.01.20.576352'> 10.1101/2024.01.20.576352</a></p>
        <p class="abstract">Here, we describe the &#34;Obelisks,&#34; a previously unrecognised class of viroid-like elements that we first identified in human gut metatranscriptomic data. &#34;Obelisks&#34; share several properties: (i) apparently circular RNA [~]1kb genome assemblies, (ii) predicted rod-like secondary structures encompassing the entire genome, and (iii) open reading frames coding for a novel protein superfamily, which we call the &#34;Oblins&#34;. We find that Obelisks form their own distinct phylogenetic group with no detectable sequence or structural similarity to known biological agents. Further, Obelisks are prevalent in tested human microbiome metatranscriptomes with representatives detected in [~]7% of analysed stool metatranscriptomes (29/440) and in [~]50% of analysed oral metatranscriptomes (17/32). Obelisk compositions appear to differ between the anatomic sites and are capable of persisting in individuals, with continued presence over &gt;300 days observed in one case. Large scale searches identified 29,959 Obelisks (clustered at 90% nucleotide identity), with examples from all seven continents and in diverse ecological niches. From this search, a subset of Obelisks are identified to code for Obelisk-specific variants of the hammerhead type-III self-cleaving ribozyme. Lastly, we identified one case of a bacterial species (Streptococcus sanguinis) in which a subset of defined laboratory strains harboured a specific Obelisk RNA population. As such, Obelisks comprise a class of diverse RNAs that have colonised, and gone unnoticed in, human, and global microbiomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.02.578538">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.02.578538" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.02.578538">
        <p class="paperTitle">Viral interference between severe acute respiratory syndrome coronavirus 2 and influenza A viruses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.02.578538" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.02.578538" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gilbert-girard, S.; Piret, J.; Carbonneau, J.; Henaut, M.; Goyette, N.; Boivin, G.</p>
        <p class="info">Score: 61.3, Published: 2024-02-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.02.578538' target='https://doi.org/10.1101/2024.02.02.578538'> 10.1101/2024.02.02.578538</a></p>
        <p class="abstract">Some respiratory viruses can cause a viral interference through the activation of the interferon (IFN) pathway that reduces the replication of another virus. Epidemiological studies of coinfections between SARS-CoV-2 and other respiratory viruses have been hampered by non-pharmaceutical measures applied to mitigate the spread of SARS-CoV-2 during the COVID-19 pandemic. With the ease of these interventions, SARS-CoV-2 and influenza A viruses can now co-circulate. It is thus of prime importance to characterize their interactions. In this work, we investigated viral interference effects between an Omicron variant and a contemporary influenza A/H3N2 strain, in comparison with an ancestral SARS-CoV-2 strain and the 2009 pandemic influenza A/H1N1 virus. We infected nasal human airway epitheliums with SARS-CoV-2 and influenza, either simultaneously or 24 h apart. Viral load was measured by RT-qPCR and IFN-/{beta}/{lambda}1/{lambda}2 proteins were quantified by immunoassay. Expression of four interferon-stimulated genes (ISGs; OAS1/IFITM3/ISG15/MxA) was also measured by RT-droplet digital PCR. Additionally, susceptibility of each virus to IFN-/{beta}/{lambda}2 recombinant proteins was determined. Our results showed that influenza A, and especially A/H3N2, interfered with both SARS-CoV-2 viruses, but that SARS-CoV-2 only interfered with A/H1N1. Consistently with these results, influenza, and particularly the A/H3N2 strain, caused a higher production of IFN proteins and expression of ISGs than SARS-CoV-2. The IFN production induced by SARS-CoV-2 was marginal and its presence during coinfections with influenza was associated with a reduced IFN response. All viruses were susceptible to exogenous IFNs, with the ancestral SARS-CoV-2 and Omicron being less susceptible to type I and type III IFNs, respectively. Thus, influenza A causes a viral interference towards SARS-CoV-2 most likely through an IFN response. The opposite is not necessarily true, and a concurrent infection with both viruses leads to a lower IFN response. Taken together, these results help us to understand how SARS-CoV-2 interacts with another major respiratory pathogen.

Author summaryDuring the COVID-19 pandemic, non-pharmaceutical measures were able to reduce the spread of SARS-CoV-2 and most respiratory viruses. Since the ease of these measures, SARS-CoV-2 variants and other viruses, such as influenza A, have started to co-circulate and can now infect a same host and interact with each other. These interactions can lead to attenuated or aggravated infections and can affect the timing of epidemics. Therefore, it is very important to elucidate how the new SARS-CoV-2 interacts with other viruses to better predict their implications in human health and their epidemic activity. Our work contributes to better understand these interactions using viruses that have likely co-circulated after lifting mitigation interventions, i.e., SARS-CoV-2 Omicron variant and a contemporary influenza A/H3N2 strain. We studied how each virus may affect the other virus growth and how these interactions were associated with the innate immune response of the host. We found that a prior infection with influenza A can decrease the growth of SARS-CoV-2 while the latter reduces the innate immune response. Our results help to understand the interplay between SARS-CoV-2 and influenza A in the host and may improve mathematical models predicting epidemics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579183">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579183" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579183">
        <p class="paperTitle">A 20-year time-series of a freshwater lake reveals seasonal dynamics and environmental drivers of viral diversity, ecology, and evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579183" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579183" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhou, Z.; Tran, P. Q.; Martin, C.; Rohwer, R. R.; Baker, B.; McMahon, K.; Anantharaman, K.</p>
        <p class="info">Score: 43.2, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579183' target='https://doi.org/10.1101/2024.02.06.579183'> 10.1101/2024.02.06.579183</a></p>
        <p class="abstract">Long-term ecological studies are powerful tools to investigate microbiomes and ecosystem change but have mostly ignored viruses. Here, we leveraged a 20-year time-series of a freshwater lake to characterize 1.3 million viral genomes over time, seasonality, and environmental factors. We identified 578 auxiliary metabolic gene (AMG) clusters representing over 150,000 AMGs, the most abundant of which, including psbA for photosynthesis, pmoC for methane oxidation, and katG for hydrogen peroxide decomposition, were consistently represented in viruses across decades and seasons. We observed positive associations and niche differentiation between virus-host pairs during seasonal change including in keystone taxa, Cyanobacteria, methanotrophs, and Nanopelagicales. Environmental constraints, specifically inorganic carbon and ammonium influenced viral abundances over time, and highlighted roles of viruses in both &#34;top-down&#34; and &#34;bottom-up&#34; interactions. Key evolutionary processes shaping gene and genome-wide selection included favored fitness genes, reduced genomic heterogeneity, and dominant sub-populations carrying specific genes. Overall, our study advances understanding of diversity, ecological dynamics, and evolutionary trajectories of viruses in Earths microbiomes and ecosystems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579159">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579159" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579159">
        <p class="paperTitle">Mapping glycoprotein structure reveals defining events in the evolution of the Flaviviridae</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579159" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579159" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mifsud, J. C. O.; Lytras, S.; Oliver, M. R.; Toon, K.; Costa, V. A.; Holmes, E. C.; Grove, J.</p>
        <p class="info">Score: 39.2, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579159' target='https://doi.org/10.1101/2024.02.06.579159'> 10.1101/2024.02.06.579159</a></p>
        <p class="abstract">Viral glycoproteins drive membrane fusion in enveloped viruses and determine host range, tissue tropism and pathogenesis. Despite their importance, there is a fragmentary understanding of glycoproteins within the Flaviviridae; for many species the glycoproteins have not yet been identified, for others, such as the hepaciviruses, the molecular mechanisms of membrane fusion remain uncharacterised. Here, we combine comprehensive phylogenetic analyses with systematic protein structure prediction to survey glycoproteins across the entire Flaviviridae. We discover class-II fusion systems, homologous to the orthoflavivirus E glycoprotein, in most species, including highly-divergent jingmenviruses and large genome flaviviruses. However, the E1E2 glycoproteins of the hepaci-, pegi- and pestiviruses are structurally distinct, may represent a novel class of fusion mechanism, and are strictly associated with infection of vertebrate hosts. By mapping glycoprotein distribution onto the underlying phylogeny we reveal a complex history of evolutionary events that have shaped the diverse virology and ecology of the Flaviviridae.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574008">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574008" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574008">
        <p class="paperTitle">An infection and pathogenesis mouse model of SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574008" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574008" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.</p>
        <p class="info">Score: 4435.0, Published: 2024-01-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574008' target='https://doi.org/10.1101/2024.01.03.574008'> 10.1101/2024.01.03.574008</a></p>
        <p class="abstract">SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) is highly attenuated, but can cause mortality in a specifically designed human ACE2-transgenic mouse model, making it an invaluable surrogate model for evaluating the efficacy of drugs and vaccines against SARS-CoV-2.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.05.579020">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.05.579020" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.05.579020">
        <p class="paperTitle">Deep mutational scanning reveals functional constraints and antigenic variability of Lassa virus glycoprotein complex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.05.579020" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.05.579020" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carr, C. R.; Crawford, K. H.; Murphy, M.; Galloway, J. G.; Haddox, H. K.; Matsen, F. A.; Andersen, K. G.; King, N. P.; Bloom, J. D.</p>
        <p class="info">Score: 20.4, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.05.579020' target='https://doi.org/10.1101/2024.02.05.579020'> 10.1101/2024.02.05.579020</a></p>
        <p class="abstract">Lassa virus is estimated to cause thousands of human deaths per year, primarily due to spillovers from its natural host, Mastomys rodents. Efforts to create vaccines and antibody therapeutics must account for the evolutionary variability of Lassa viruss glycoprotein complex (GPC), which mediates viral entry into cells and is the target of neutralizing antibodies. To map the evolutionary space accessible to GPC, we use pseudovirus deep mutational scanning to measure how nearly all GPC amino-acid mutations affect cell entry and antibody neutralization. Our experiments define functional constraints throughout GPC. We quantify how GPC mutations affect neutralization by a panel of monoclonal antibodies and show that all antibodies are escaped by mutations that exist among natural Lassa virus lineages. Overall, our work describes a biosafety-level-2 method to elucidate the mutational space accessible to GPC and shows how prospective characterization of antigenic variation could aid design of therapeutics and vaccines.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579203">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579203" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579203">
        <p class="paperTitle">YraN is a helicase-associated nuclease fostering extended recombination events by natural transformation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579203" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579203" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hardy, L.; Plantade, J.; Morales, V.; Mazzamurro, F.; Rocha, E. P. C.; Polard, P.; Charpentier, X.</p>
        <p class="info">Score: 18.6, Published: 2024-02-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579203' target='https://doi.org/10.1101/2024.02.06.579203'> 10.1101/2024.02.06.579203</a></p>
        <p class="abstract">Natural transformation is a widespread molecular pathway of horizontal gene transfer involving the uptake and recombination of exogenous DNA. Exogenous DNA follows a pathway involving genes sequentially required for its capture, internalization, protection, and recombination with the chromosome. Most of these genes were identified through the isolation of transformation-defective mutants and/or based on their expression preceding natural transformation. Yet, genes required for key steps of the pathway remain elusive. We sought to identify any missing component by comparing Tn-seq data obtained in two distantly-related transformable diderm species, the human pathogen Legionella pneumophila and the cyanobacterium Synechococcus elongatus. We identified yraN, a widespread and highly conserved gene of unknown function required for natural transformation. We provide evidence that YraN is a nuclease associated with the ComM helicase, which cooperate to process the D-loop formed by the invasion of the transforming DNA in the chromosomal DNA strands. We propose a model in which cleavage of the displaced strand by YraN can promote the recombination of transforming DNA, leading to extended recombination events. The identification of this YraN/ComM nuclease/helicase system supports the hypothesis that bacteria possess a conserved pathway for the transport and recombination of exogenous DNA.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579193">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579193" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579193">
        <p class="paperTitle">De novo mutations mediate phenotypic switching in an opportunistic human lung pathogen</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579193" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579193" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Poret, A. J.; Schaefers, M. M.; Merakou, C.; Mansour, K. E.; Lagoudas, G. K.; Cross, A. R.; Goldberg, J. B.; Kishony, R.; Uluer, A. Z.; McAdam, A. J.; Blainey, P. C.; Vargas, S. O.; Lieberman, T. D.; Priebe, G. P.</p>
        <p class="info">Score: 16.3, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579193' target='https://doi.org/10.1101/2024.02.06.579193'> 10.1101/2024.02.06.579193</a></p>
        <p class="abstract">Bacteria evolving within human hosts encounter selective tradeoffs that render mutations adaptive in one context and deleterious in another. Here, we report that the cystic fibrosis-associated pathogen Burkholderia dolosa overcomes in-human selective tradeoffs by acquiring successive point mutations that alternate phenotypes. We sequenced the whole genomes of 931 respiratory isolates from two recently infected patients and an epidemiologically-linked, chronically-infected patient. These isolates are contextualized using 112 historical genomes from the same outbreak strain. Within both newly infected patients, diverse parallel mutations that disrupt O-antigen expression quickly arose, comprising 29% and 63% of their B. dolosa communities by 3 years. The selection for loss of O-antigen starkly contrasts with our previous observation of parallel O-antigen-restoring mutations after many years of chronic infection in the historical outbreak. Experimental characterization revealed that O-antigen loss increases uptake in immune cells while decreasing competitiveness in the mouse lung. We propose that the balance of these pressures, and thus whether O-antigen expression is advantageous, depends on tissue localization and infection duration. These results suggest that mutation-driven alternation during infection may be more frequent than appreciated and is underestimated without dense temporal sampling.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.30.578034">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.30.578034" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.30.578034">
        <p class="paperTitle">Stratification of viral shedding patterns in saliva of COVID-19 patients</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.30.578034" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.30.578034" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Park, H.; Raiki, Y.; Iwanami, S.; Kim, K.; Ejima, K.; Nakamura, N.; Aihara, K.; Miyazaki, Y.; Umeyama, T.; Miyazawa, K.; Morita, T.; Watashi, K.; Brooke, C. B.; Ke, R.; Iwami, S.; Miyazaki, T.</p>
        <p class="info">Score: 17.0, Published: 2024-02-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.30.578034' target='https://doi.org/10.1101/2024.01.30.578034'> 10.1101/2024.01.30.578034</a></p>
        <p class="abstract">Living with COVID-19 requires continued vigilance against the spread and emergence of variants of concern (VOCs). Rapid and accurate saliva diagnostic testing, alongside basic public health responses, is a viable option contributing to effective transmission control. Nevertheless, our knowledge regarding the dynamics of SARS-CoV-2 infection in saliva is not as advanced as our understanding of the respiratory tract. Here we analyzed longitudinal viral load data of SARS-CoV-2 in saliva samples from 144 patients with mild COVID-19 (a combination of our collected data and published data). Using a mathematical model, we successfully stratified infection dynamics into three distinct groups with clear patterns of viral shedding: viral shedding durations in the three groups were 11.5 days (95% CI: 10.6 to 12.4), 17.4 days (16.6 to 18.2), and 30.0 days (28.1 to 31.8), respectively. Surprisingly, this stratified grouping remained unexplained despite our analysis of 47 types of clinical data, including basic demographic information, clinical symptoms, results of blood tests, and vital signs. Additionally, we quantified the expression levels of 92 micro-RNAs in a subset of saliva samples, but these also failed to explain the observed stratification, although the mir-1846 level may have been weakly correlated with peak viral load. Our study provides insights into SARS-CoV-2 infection dynamics in saliva, highlighting the challenges in predicting the duration of viral shedding without indicators that directly reflect an individuals immune response, such as antibody induction. Given the significant individual heterogeneity in the kinetics of saliva viral shedding, identifying biomarker(s) for viral shedding patterns will be crucial for improving public health interventions in the era of living with COVID-19.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.30.578060">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.30.578060" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.30.578060">
        <p class="paperTitle">SingleM and Sandpiper: Robust microbial taxonomic profiles from metagenomic data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.30.578060" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.30.578060" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Woodcroft, B. J.; Aroney, S. T. N.; Zhao, R.; Cunningham, M.; Mitchell, J. A. M.; Blackall, L.; Tyson, G. W.</p>
        <p class="info">Score: 55.4, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.30.578060' target='https://doi.org/10.1101/2024.01.30.578060'> 10.1101/2024.01.30.578060</a></p>
        <p class="abstract">Determining the taxonomy and relative abundance of microorganisms in metagenomic data is a foundational problem in microbial ecology. To address the limitations of existing approaches, we developed  SingleM, which estimates community composition using conserved regions within universal marker genes. SingleM accurately profiles complex communities of known microbial species, and is the only tool that detects species without genomic representation, even those representing novel phyla. Given SingleMs computational efficiency, we applied it to 248,559 publicly available metagenomes and show that the vast majority of samples from marine, freshwater, sediment and soil environments are dominated by novel species lacking genomic representation (median relative abundance 75.0%). SingleM also provides a way to identify metagenomes for the recovery of novel metagenome-assembled genomes from lineages of interest, and can incorporate user-recovered genomes into its reference database to improve profiling resolution. Quantifying the full diversity of Bacteria and Archaea in metagenomic data shows that microbial genome databases are far from saturated.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
