<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Virological characteristics of the SARS-CoV-2 JN.1 variant
Authors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Zahradnik, J.; Ito, J.; Sato, K.
Score: 757.9, Published: 2023-12-09 DOI: 10.1101/2023.12.08.570782
The SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Virological characteristics of the SARS-CoV-2 JN.1 variant
Authors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Zahradnik, J.; Ito, J.; Sato, K.
Score: 757.9, Published: 2023-12-09 DOI: 10.1101/2023.12.08.570782
The SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-24T10:38:02+00:00" />
<meta property="article:modified_time" content="2023-12-24T10:38:02+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Virological characteristics of the SARS-CoV-2 JN.1 variant
Authors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Zahradnik, J.; Ito, J.; Sato, K.
Score: 757.9, Published: 2023-12-09 DOI: 10.1101/2023.12.08.570782
The SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Virological characteristics of the SARS-CoV-2 JN.1 variant\nAuthors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Zahradnik, J.; Ito, J.; Sato, K.\nScore: 757.9, Published: 2023-12-09 DOI: 10.1101/2023.12.08.570782\nThe SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG.",
  "keywords": [
    
  ],
  "articleBody": " Virological characteristics of the SARS-CoV-2 JN.1 variant\nAuthors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Zahradnik, J.; Ito, J.; Sato, K.\nScore: 757.9, Published: 2023-12-09 DOI: 10.1101/2023.12.08.570782\nThe SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG.5.1 and HK.3. Comparing to XBB and BA.2, BA.2.86 carries more than 30 mutations in the spike (S) protein, indicating a high potential for immune evasion. BA.2.86 has evolved and its descendant, JN.1 (BA.2.86.1.1), emerged in late 2023. JN.1 harbors S:L455S and three mutations in non-S proteins. S:L455S is a hallmark mutation of JN.1: we have recently shown that HK.3 and other \"FLip\" variants carry S:L455F, which contributes to increased transmissibility and immune escape ability compared to the parental EG.5.1 variant. Here, we investigated the virological properties of JN.1.\nAge-dependent heterogeneity in the antigenic effects of mutations to influenza hemagglutinin\nAuthors: Welsh, F. C.; Eguia, R. T.; Lee, J. M.; Haddox, H. K.; Galloway, J.; Vinh, N. V.; Loes, A. N.; Huddleston, J.; Yu, T. C.; Le, M. Q.; Nhat, N. T.; Thanh, N. T. L.; Greninger, A. L.; Chu, H. Y.; Englund, J. A.; Bedford, T.; Matsen, F. A.; Boni, M. F.; Bloom, J. D.\nScore: 78.5, Published: 2023-12-12 DOI: 10.1101/2023.12.12.571235\nABSTRACTHuman influenza virus evolves to escape neutralization by polyclonal antibodies. However, we have a limited understanding of how the antigenic effects of viral mutations vary across the human population, and how this heterogeneity affects virus evolution. Here we use deep mutational scanning to map how mutations to the hemagglutinin (HA) proteins of the A/Hong Kong/45/2019 (H3N2) and A/Perth/16/2009 (H3N2) strains affect neutralization by serum from individuals of a variety of ages. The effects of HA mutations on serum neutralization differ across age groups in ways that can be partially rationalized in terms of exposure histories. Mutations that fixed in influenza variants after 2020 cause the greatest escape from sera from younger individuals. Overall, these results demonstrate that influenza faces distinct antigenic selection regimes from different age groups, and suggest approaches to understand how this heterogeneous selection shapes viral evolution.\nInterferon signaling in the nasal epithelium distinguishes among lethal and common cold respiratory viruses and is critical for viral clearance\nAuthors: Otter, C. J.; Renner, D. M.; Fausto, A.; Tan, L. H.; Cohen, N. A.; Weiss, S. R.\nScore: 70.7, Published: 2023-12-19 DOI: 10.1101/2023.12.18.571720\nAll respiratory viruses establish primary infections in the nasal epithelium, where efficient innate immune induction may prevent dissemination to the lower airway and thus minimize pathogenesis. Human coronaviruses (HCoVs) cause a range of pathologies, but the host and viral determinants of disease during common cold versus lethal HCoV infections are poorly understood. We model the initial site of infection using primary nasal epithelial cells cultured at air-liquid interface (ALI). HCoV-229E, HCoV-NL63 and human rhinovirus-16 are common cold-associated viruses that exhibit unique features in this model: early induction of antiviral interferon (IFN) signaling, IFN-mediated viral clearance, and preferential replication at nasal airway temperature (33{degrees}C) which confers muted host IFN responses. In contrast, lethal SARS-CoV-2 and MERS-CoV encode antagonist proteins that prevent IFN-mediated clearance in nasal cultures. Our study identifies features shared among common cold-associated viruses, highlighting nasal innate immune responses as predictive of infection outcomes and nasally-directed IFNs as potential therapeutics.\nEngineered Skin Microbiome Reduces Mosquito Attraction to Mice\nAuthors: Liu, F.; Coutinho-Abreu, I. V.; Raban, R.; Nguyen, T.; Dimas, A.; Merriman, J.; Akbari, O.\nScore: 24.4, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572663\nThe skin microbiome plays a pivotal role in the production of attractive cues detected by mosquitoes. Here we leveraged recent advances in genetic engineering to significantly reduce the production of L-(+)-lactic acid as a strategy to reduce mosquito attraction to the highly prominent skin commensals Staphylococcus epidermidis and Corynebacterium amycolatum. Engraftment of these engineered bacteria onto the skin of mice reduced mosquito attraction and feeding for up to 11 uninterrupted days, which is considerably longer than the several hours of protection conferred by the leading chemical repellent DEET. Taken together, our findings demonstrate engineering the skin microbiome to reduce attractive volatiles represents an innovative untapped strategy to reduce vector attraction, preventing bites, and pathogen transmission setting the stage for new classes of long-lasting microbiome-based repellent products.\nTemperature impacts SARS-CoV-2 spike fusogenicity and evolution\nAuthors: Dufloo, J.; Sanjuan, R.\nScore: 23.3, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572501\nSARS-CoV-2 infects both the upper and lower respiratory tracts, which are characterized by different temperatures (33{degrees}C and 37{degrees}C, respectively). In addition, fever is a common COVID-19 symptom. SARS-CoV-2 has been shown to replicate more efficiently at low temperatures but the effect of temperature on different viral proteins remains poorly understood. Here, we investigate how temperature affects the SARS-CoV-2 spike function and evolution. We first observed that rising temperature from 33{degrees}C to 37{degrees}C or 39{degrees}C increased spike-mediated cell-cell fusion. We then experimentally evolved a recombinant vesicular stomatitis virus expressing the SARS-CoV-2 spike at these different temperatures. We found that spike-mediated cell-cell fusion was maintained during evolution at 39{degrees}C, but was lost in a high proportion of viruses evolved at 33{degrees}C or 37{degrees}C. Consistently, sequencing of the spikes evolved at 33{degrees}C or 37{degrees}C revealed the accumulation of mutations around the furin cleavage site, a region that determines cell-cell fusion, whereas this did not occur in spikes evolved at 39{degrees}C. Finally, using site-directed mutagenesis, we found that disruption of the furin cleavage site had a temperature-dependent effect on spike-induced cell-cell fusion and viral fitness. Our results suggest that variations in body temperature may affect the activity and diversification of the SARS-CoV-2 spike. ImportanceWhen it infects humans, SARS-CoV-2 is exposed to different temperaures (e.g. replication site, fever...). Temperature has been shown to strongly impact SARS-CoV-2 replication but how it affects the activity and evolution of the spike protein remains poorly understood. Here, we first show that high temperatures increase the SARS-CoV-2 spike fusogenicity. Then, we demonstrate that the evolution of the spike activity and variants depends on temperature. Finally, we show that the functional effect of specific spike mutations is temperature-dependent. Overall, our results suggest that temperature may be a factor influencing the activity and adapatation of the SARS-CoV-2 spike in vivo, which will help understanding viral tropism, pathogenesis, and evolution.\nMetabolism of ʟ -arabinose converges with virulence regulation to promote enteric pathogen fitness\nAuthors: Cottam, C.; White, R. T.; Beck, L. C.; Stewart, C. J.; Beatson, S. A.; Lowe, E. C.; Grinter, R.; Connolly, J. P.\nScore: 18.3, Published: 2023-12-22 DOI: 10.1101/2023.12.21.572775\nVirulence and metabolism are often interlinked to control the expression of essential colonisation factors in response to host-associated signals. Here, we identified a novel transporter of the dietary monosaccharide -arabinose that is widely encoded by the zoonotic pathogen enterohaemorrhagic Escherichia coli (EHEC), required for full competitive fitness in the mouse gut and highly expressed during human infection. Accordingly, we discovered that -arabinose induces expression of the EHEC type 3 secretion system, enhancing its ability to attach to host cells, and that the underlying mechanism is dependent on products of its catabolism rather than the sensing of -arabinose as a signal. Finally, using the murine pathogen Citrobacter rodentium, we show that -arabinose metabolism provides a fitness benefit during enteric infection via virulence factor regulation, as opposed to supporting pathogen growth. This study describes an intrinsic mechanism of integrating central sugar metabolism with virulence regulation and highlights the unexpected impact that nutrient utilisation can have in enteric pathogens.\nVirus-encoded glycosyltransferases hypermodify DNA with diverse glycans\nAuthors: Pyle, J. D.; Lund, S. R.; O'Toole, K. H.; Saleh, L.\nScore: 18.0, Published: 2023-12-22 DOI: 10.1101/2023.12.21.572611\nEnzymatic modification of DNA nucleobases can coordinate gene expression, protection from nucleases, or mutagenesis. We recently discovered a new clade of phage-specific cytosine methyltransferase (MT) and 5-methylpyrimidine dioxygenase (5mYOX, e.g., TET) enzymes that produce 5-hydroxymethylcytosine (5hmC) as a precursor for additional post-replicative enzymatic hypermodifications on viral genomes. Here, we identify phage MT- and 5mYOX-dependent glycosyltransferase (GT) enzymes that catalyze linkage of diverse glycans directly onto 5hmC reactive nucleobase substrates. Using targeted bioinformatic mining of the phage metavirome databases, we discovered thousands of new biosynthetic gene clusters (BGCs) containing enzymes with predicted roles in cytosine sugar hypermodification. We developed a pathway reassembly platform for high-throughput functional screening of GT-containing BGCs, relying on the endogenous E. coli metabolome as a substrate pool. We successfully reconstituted a subset of phage BGCs and isolated novel and highly diverse sugar modifications appended to 5hmC, including mono-, di-, or tri-saccharide moieties comprised of hexose, N-acetylhexosamine or heptose sugars. Structural predictions and sugar product analyses suggest that phage GTs are related to host lipopolysaccharide, teichoic acid, and other small molecule biosynthesis enzymes and have been repurposed for DNA substrates. An expanded metagenomic search revealed hypermodification BGCs within gene neighborhoods containing phage structural proteins and putative genome defense systems. These findings enrich our knowledge of secondary modifications on DNA and the origins of corresponding sugar writer enzymes. Post-replicative cytosine hypermodification by virus-encoded GTs is discussed in the context of genome defense, DNA partitioning and virion assembly, and host-pathogen co-evolution.\nMIMt: A curated 16S rRNA reference database with less redundancy and higher accuracy at species-level identification\nAuthors: Cabezas Rodriguez, M. P.; Fonseca, N. A.; Munoz-Merida, A.\nScore: 20.7, Published: 2023-12-15 DOI: 10.1101/2023.12.15.571851\nMotivationAccurate determination and quantification of the taxonomic composition of microbial communities, especially at the species level, is one of the major issues in metagenomics. This is primarily due to the limitations of commonly used 16S rRNA reference databases, which either contain a lot of redundancy, or a high percentage of sequences with missing taxonomic information. The use of these incomplete or biased databases may lead to erroneous identifications and, thus, to erroneous conclusions regarding the ecological role and importance of those microorganisms in the ecosystem. ResultsThe current study presents MIMt, a new 16S rRNA database for archaea and bacterias identification, encompassing 39 940 sequences, all precisely identified at species level. MIMt aims to be updated at least once a year to include all new sequenced species. We evaluated MIMt against Greengenes, RDP, GTDB and SILVA in terms of sequence distribution and accuracy of taxonomic assignments. Our results showed that MIMt contains less redundancy, and despite being five to 85 times smaller in size than existing databases, outperforms them in completeness and taxonomic accuracy, enabling more precise assignments at lower taxonomic ranks and thus, significantly improving species-level identification. Availability and ImplementationMIMt is freely available for non-commercial purposes at https://mimt.bu.biopolis.pt\nVirion morphology and on-virus spike protein structures of diverse SARS-CoV-2 variants\nAuthors: Ke, Z.; Peacock, T. P.; Brown, J. C.; Sheppard, C. M.; Croll, T. I.; Kotecha, A.; Goldhill, D. H.; Barclay, W. S.; Briggs, J. A.\nScore: 13.4, Published: 2023-12-22 DOI: 10.1101/2023.12.21.572824\nThe evolution of SARS-CoV-2 variants with increased fitness has been accompanied by structural changes in the spike (S) proteins that are the major target for the adaptive immune response. Single-particle cryo-EM analysis of soluble S from SARS-CoV-2 variants has revealed this structural adaptation at high-resolution. The analysis of S trimers in situ on intact virions has the potential to provide more functionally relevant insights into S structure and virion morphology. Here, we characterized B.1, Alpha, Beta, Gamma, Delta, Kappa, and Mu variants by cryo-electron microscopy and tomography, assessing S cleavage, virion morphology, S incorporation, \"in-situ\" high-resolution S structures and the range of S conformational states. We found no evidence for adaptive changes in virion morphology, but describe multiple different positions in the S protein where amino acid changes alter local protein structure. Considered together, our data is consistent with a model where amino acid changes at multiple positions from the top to the base of the spike cause structural changes that can modulate the conformational dynamics of S.\nRNA-guided RNA silencing by an Asgard archaeal Argonaute\nAuthors: Bastiaanssen, C.; Ugarte, P. B.; Kim, K.; Feng, Y.; Finocchio, G.; Anzelon, T. A.; Kostlbacher, S.; Tamarit, D.; Ettema, T. J. G.; Jinek, M.; MacRae, I. J.; Joo, C.; Swarts, D. C.; Wu, F.\nScore: 52.6, Published: 2023-12-15 DOI: 10.1101/2023.12.14.571608\nEukaryotic Argonaute proteins achieve gene repression and defense against viruses and transposons by RNA-guided RNA silencing. By contrast, known prokaryotic Argonautes adopt single-stranded DNA as guides and/or targets, leaving the evolutionary origin of RNA-guided RNA silencing elusive. Here, we show an evolutionary expansion of Asgard archaeal Argonautes (asAgos), including the discovery of HrAgo1 from the Lokiarchaeon Candidatus Harpocratesius repetitus that shares a common origin with eukaryotic PIWI proteins. HrAgo1 exhibits RNA-guided RNA cleavage in vitro and RNA silencing in human cells. The cryo-EM structure of HrAgo1 combined with quantitative single-molecule experiments reveals that HrAgo1 possesses hybrid structural features and target binding modes bridging those of the eukaryotic AGO and PIWI clades. Finally, genomic evidence suggests that eukaryotic Dicer-like processing of double-stranded RNA likely emerged as a mechanism of generating guide RNA for asAgos prior to eukaryogenesis. Our study provides new insights into the evolutionary origin and plasticity of Argonaute-based RNA silencing.\n",
  "wordCount" : "2152",
  "inLanguage": "en",
  "datePublished": "2023-12-24T10:38:02Z",
  "dateModified": "2023-12-24T10:38:02Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 24, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.570782">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.570782" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.570782">
        <p class="paperTitle">Virological characteristics of the SARS-CoV-2 JN.1 variant</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.570782" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.570782" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium,  ; Zahradnik, J.; Ito, J.; Sato, K.</p>
        <p class="info">Score: 757.9, Published: 2023-12-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.570782' target='https://doi.org/10.1101/2023.12.08.570782'> 10.1101/2023.12.08.570782</a></p>
        <p class="abstract">The SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG.5.1 and HK.3. Comparing to XBB and BA.2, BA.2.86 carries more than 30 mutations in the spike (S) protein, indicating a high potential for immune evasion. BA.2.86 has evolved and its descendant, JN.1 (BA.2.86.1.1), emerged in late 2023. JN.1 harbors S:L455S and three mutations in non-S proteins. S:L455S is a hallmark mutation of JN.1: we have recently shown that HK.3 and other &#34;FLip&#34; variants carry S:L455F, which contributes to increased transmissibility and immune escape ability compared to the parental EG.5.1 variant. Here, we investigated the virological properties of JN.1.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.12.571235">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.12.571235" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.12.571235">
        <p class="paperTitle">Age-dependent heterogeneity in the antigenic effects of mutations to influenza hemagglutinin</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.12.571235" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.12.571235" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Welsh, F. C.; Eguia, R. T.; Lee, J. M.; Haddox, H. K.; Galloway, J.; Vinh, N. V.; Loes, A. N.; Huddleston, J.; Yu, T. C.; Le, M. Q.; Nhat, N. T.; Thanh, N. T. L.; Greninger, A. L.; Chu, H. Y.; Englund, J. A.; Bedford, T.; Matsen, F. A.; Boni, M. F.; Bloom, J. D.</p>
        <p class="info">Score: 78.5, Published: 2023-12-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.12.571235' target='https://doi.org/10.1101/2023.12.12.571235'> 10.1101/2023.12.12.571235</a></p>
        <p class="abstract">ABSTRACTHuman influenza virus evolves to escape neutralization by polyclonal antibodies. However, we have a limited understanding of how the antigenic effects of viral mutations vary across the human population, and how this heterogeneity affects virus evolution. Here we use deep mutational scanning to map how mutations to the hemagglutinin (HA) proteins of the A/Hong Kong/45/2019 (H3N2) and A/Perth/16/2009 (H3N2) strains affect neutralization by serum from individuals of a variety of ages. The effects of HA mutations on serum neutralization differ across age groups in ways that can be partially rationalized in terms of exposure histories. Mutations that fixed in influenza variants after 2020 cause the greatest escape from sera from younger individuals. Overall, these results demonstrate that influenza faces distinct antigenic selection regimes from different age groups, and suggest approaches to understand how this heterogeneous selection shapes viral evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.571720">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.571720" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.571720">
        <p class="paperTitle">Interferon signaling in the nasal epithelium distinguishes among lethal and common cold respiratory viruses and is critical for viral clearance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.571720" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.571720" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Otter, C. J.; Renner, D. M.; Fausto, A.; Tan, L. H.; Cohen, N. A.; Weiss, S. R.</p>
        <p class="info">Score: 70.7, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.571720' target='https://doi.org/10.1101/2023.12.18.571720'> 10.1101/2023.12.18.571720</a></p>
        <p class="abstract">All respiratory viruses establish primary infections in the nasal epithelium, where efficient innate immune induction may prevent dissemination to the lower airway and thus minimize pathogenesis. Human coronaviruses (HCoVs) cause a range of pathologies, but the host and viral determinants of disease during common cold versus lethal HCoV infections are poorly understood. We model the initial site of infection using primary nasal epithelial cells cultured at air-liquid interface (ALI). HCoV-229E, HCoV-NL63 and human rhinovirus-16 are common cold-associated viruses that exhibit unique features in this model: early induction of antiviral interferon (IFN) signaling, IFN-mediated viral clearance, and preferential replication at nasal airway temperature (33{degrees}C) which confers muted host IFN responses. In contrast, lethal SARS-CoV-2 and MERS-CoV encode antagonist proteins that prevent IFN-mediated clearance in nasal cultures. Our study identifies features shared among common cold-associated viruses, highlighting nasal innate immune responses as predictive of infection outcomes and nasally-directed IFNs as potential therapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572663">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572663" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572663">
        <p class="paperTitle">Engineered Skin Microbiome Reduces Mosquito Attraction to Mice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572663" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572663" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, F.; Coutinho-Abreu, I. V.; Raban, R.; Nguyen, T.; Dimas, A.; Merriman, J.; Akbari, O.</p>
        <p class="info">Score: 24.4, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572663' target='https://doi.org/10.1101/2023.12.20.572663'> 10.1101/2023.12.20.572663</a></p>
        <p class="abstract">The skin microbiome plays a pivotal role in the production of attractive cues detected by mosquitoes. Here we leveraged recent advances in genetic engineering to significantly reduce the production of L-(&#43;)-lactic acid as a strategy to reduce mosquito attraction to the highly prominent skin commensals Staphylococcus epidermidis and Corynebacterium amycolatum. Engraftment of these engineered bacteria onto the skin of mice reduced mosquito attraction and feeding for up to 11 uninterrupted days, which is considerably longer than the several hours of protection conferred by the leading chemical repellent DEET. Taken together, our findings demonstrate engineering the skin microbiome to reduce attractive volatiles represents an innovative untapped strategy to reduce vector attraction, preventing bites, and pathogen transmission setting the stage for new classes of long-lasting microbiome-based repellent products.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572501">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572501" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572501">
        <p class="paperTitle">Temperature impacts SARS-CoV-2 spike fusogenicity and evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572501" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572501" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dufloo, J.; Sanjuan, R.</p>
        <p class="info">Score: 23.3, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572501' target='https://doi.org/10.1101/2023.12.20.572501'> 10.1101/2023.12.20.572501</a></p>
        <p class="abstract">SARS-CoV-2 infects both the upper and lower respiratory tracts, which are characterized by different temperatures (33{degrees}C and 37{degrees}C, respectively). In addition, fever is a common COVID-19 symptom. SARS-CoV-2 has been shown to replicate more efficiently at low temperatures but the effect of temperature on different viral proteins remains poorly understood. Here, we investigate how temperature affects the SARS-CoV-2 spike function and evolution. We first observed that rising temperature from 33{degrees}C to 37{degrees}C or 39{degrees}C increased spike-mediated cell-cell fusion. We then experimentally evolved a recombinant vesicular stomatitis virus expressing the SARS-CoV-2 spike at these different temperatures. We found that spike-mediated cell-cell fusion was maintained during evolution at 39{degrees}C, but was lost in a high proportion of viruses evolved at 33{degrees}C or 37{degrees}C. Consistently, sequencing of the spikes evolved at 33{degrees}C or 37{degrees}C revealed the accumulation of mutations around the furin cleavage site, a region that determines cell-cell fusion, whereas this did not occur in spikes evolved at 39{degrees}C. Finally, using site-directed mutagenesis, we found that disruption of the furin cleavage site had a temperature-dependent effect on spike-induced cell-cell fusion and viral fitness. Our results suggest that variations in body temperature may affect the activity and diversification of the SARS-CoV-2 spike.

ImportanceWhen it infects humans, SARS-CoV-2 is exposed to different temperaures (e.g. replication site, fever...). Temperature has been shown to strongly impact SARS-CoV-2 replication but how it affects the activity and evolution of the spike protein remains poorly understood. Here, we first show that high temperatures increase the SARS-CoV-2 spike fusogenicity. Then, we demonstrate that the evolution of the spike activity and variants depends on temperature. Finally, we show that the functional effect of specific spike mutations is temperature-dependent. Overall, our results suggest that temperature may be a factor influencing the activity and adapatation of the SARS-CoV-2 spike in vivo, which will help understanding viral tropism, pathogenesis, and evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572775">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572775" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572775">
        <p class="paperTitle">Metabolism of ʟ -arabinose converges with virulence regulation to promote enteric pathogen fitness</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572775" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572775" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cottam, C.; White, R. T.; Beck, L. C.; Stewart, C. J.; Beatson, S. A.; Lowe, E. C.; Grinter, R.; Connolly, J. P.</p>
        <p class="info">Score: 18.3, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572775' target='https://doi.org/10.1101/2023.12.21.572775'> 10.1101/2023.12.21.572775</a></p>
        <p class="abstract">Virulence and metabolism are often interlinked to control the expression of essential colonisation factors in response to host-associated signals. Here, we identified a novel transporter of the dietary monosaccharide -arabinose that is widely encoded by the zoonotic pathogen enterohaemorrhagic Escherichia coli (EHEC), required for full competitive fitness in the mouse gut and highly expressed during human infection. Accordingly, we discovered that -arabinose induces expression of the EHEC type 3 secretion system, enhancing its ability to attach to host cells, and that the underlying mechanism is dependent on products of its catabolism rather than the sensing of -arabinose as a signal. Finally, using the murine pathogen Citrobacter rodentium, we show that -arabinose metabolism provides a fitness benefit during enteric infection via virulence factor regulation, as opposed to supporting pathogen growth. This study describes an intrinsic mechanism of integrating central sugar metabolism with virulence regulation and highlights the unexpected impact that nutrient utilisation can have in enteric pathogens.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572611">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572611" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572611">
        <p class="paperTitle">Virus-encoded glycosyltransferases hypermodify DNA with diverse glycans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572611" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572611" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pyle, J. D.; Lund, S. R.; O&#39;Toole, K. H.; Saleh, L.</p>
        <p class="info">Score: 18.0, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572611' target='https://doi.org/10.1101/2023.12.21.572611'> 10.1101/2023.12.21.572611</a></p>
        <p class="abstract">Enzymatic modification of DNA nucleobases can coordinate gene expression, protection from nucleases, or mutagenesis. We recently discovered a new clade of phage-specific cytosine methyltransferase (MT) and 5-methylpyrimidine dioxygenase (5mYOX, e.g., TET) enzymes that produce 5-hydroxymethylcytosine (5hmC) as a precursor for additional post-replicative enzymatic hypermodifications on viral genomes. Here, we identify phage MT- and 5mYOX-dependent glycosyltransferase (GT) enzymes that catalyze linkage of diverse glycans directly onto 5hmC reactive nucleobase substrates. Using targeted bioinformatic mining of the phage metavirome databases, we discovered thousands of new biosynthetic gene clusters (BGCs) containing enzymes with predicted roles in cytosine sugar hypermodification. We developed a pathway reassembly platform for high-throughput functional screening of GT-containing BGCs, relying on the endogenous E. coli metabolome as a substrate pool. We successfully reconstituted a subset of phage BGCs and isolated novel and highly diverse sugar modifications appended to 5hmC, including mono-, di-, or tri-saccharide moieties comprised of hexose, N-acetylhexosamine or heptose sugars. Structural predictions and sugar product analyses suggest that phage GTs are related to host lipopolysaccharide, teichoic acid, and other small molecule biosynthesis enzymes and have been repurposed for DNA substrates. An expanded metagenomic search revealed hypermodification BGCs within gene neighborhoods containing phage structural proteins and putative genome defense systems. These findings enrich our knowledge of secondary modifications on DNA and the origins of corresponding sugar writer enzymes. Post-replicative cytosine hypermodification by virus-encoded GTs is discussed in the context of genome defense, DNA partitioning and virion assembly, and host-pathogen co-evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.571851">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.571851" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.571851">
        <p class="paperTitle">MIMt: A curated 16S rRNA reference database with less redundancy and higher accuracy at species-level identification</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.571851" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.571851" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cabezas Rodriguez, M. P.; Fonseca, N. A.; Munoz-Merida, A.</p>
        <p class="info">Score: 20.7, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.571851' target='https://doi.org/10.1101/2023.12.15.571851'> 10.1101/2023.12.15.571851</a></p>
        <p class="abstract">MotivationAccurate determination and quantification of the taxonomic composition of microbial communities, especially at the species level, is one of the major issues in metagenomics. This is primarily due to the limitations of commonly used 16S rRNA reference databases, which either contain a lot of redundancy, or a high percentage of sequences with missing taxonomic information. The use of these incomplete or biased databases may lead to erroneous identifications and, thus, to erroneous conclusions regarding the ecological role and importance of those microorganisms in the ecosystem.

ResultsThe current study presents MIMt, a new 16S rRNA database for archaea and bacterias identification, encompassing 39 940 sequences, all precisely identified at species level. MIMt aims to be updated at least once a year to include all new sequenced species. We evaluated MIMt against Greengenes, RDP, GTDB and SILVA in terms of sequence distribution and accuracy of taxonomic assignments. Our results showed that MIMt contains less redundancy, and despite being five to 85 times smaller in size than existing databases, outperforms them in completeness and taxonomic accuracy, enabling more precise assignments at lower taxonomic ranks and thus, significantly improving species-level identification.

Availability and ImplementationMIMt is freely available for non-commercial purposes at https://mimt.bu.biopolis.pt</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572824">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572824" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572824">
        <p class="paperTitle">Virion morphology and on-virus spike protein structures of diverse SARS-CoV-2 variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572824" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572824" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ke, Z.; Peacock, T. P.; Brown, J. C.; Sheppard, C. M.; Croll, T. I.; Kotecha, A.; Goldhill, D. H.; Barclay, W. S.; Briggs, J. A.</p>
        <p class="info">Score: 13.4, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572824' target='https://doi.org/10.1101/2023.12.21.572824'> 10.1101/2023.12.21.572824</a></p>
        <p class="abstract">The evolution of SARS-CoV-2 variants with increased fitness has been accompanied by structural changes in the spike (S) proteins that are the major target for the adaptive immune response. Single-particle cryo-EM analysis of soluble S from SARS-CoV-2 variants has revealed this structural adaptation at high-resolution. The analysis of S trimers in situ on intact virions has the potential to provide more functionally relevant insights into S structure and virion morphology. Here, we characterized B.1, Alpha, Beta, Gamma, Delta, Kappa, and Mu variants by cryo-electron microscopy and tomography, assessing S cleavage, virion morphology, S incorporation, &#34;in-situ&#34; high-resolution S structures and the range of S conformational states. We found no evidence for adaptive changes in virion morphology, but describe multiple different positions in the S protein where amino acid changes alter local protein structure. Considered together, our data is consistent with a model where amino acid changes at multiple positions from the top to the base of the spike cause structural changes that can modulate the conformational dynamics of S.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.571608">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.571608" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.571608">
        <p class="paperTitle">RNA-guided RNA silencing by an Asgard archaeal Argonaute</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.571608" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.571608" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bastiaanssen, C.; Ugarte, P. B.; Kim, K.; Feng, Y.; Finocchio, G.; Anzelon, T. A.; Kostlbacher, S.; Tamarit, D.; Ettema, T. J. G.; Jinek, M.; MacRae, I. J.; Joo, C.; Swarts, D. C.; Wu, F.</p>
        <p class="info">Score: 52.6, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.571608' target='https://doi.org/10.1101/2023.12.14.571608'> 10.1101/2023.12.14.571608</a></p>
        <p class="abstract">Eukaryotic Argonaute proteins achieve gene repression and defense against viruses and transposons by RNA-guided RNA silencing. By contrast, known prokaryotic Argonautes adopt single-stranded DNA as guides and/or targets, leaving the evolutionary origin of RNA-guided RNA silencing elusive. Here, we show an evolutionary expansion of Asgard archaeal Argonautes (asAgos), including the discovery of HrAgo1 from the Lokiarchaeon  Candidatus Harpocratesius repetitus that shares a common origin with eukaryotic PIWI proteins. HrAgo1 exhibits RNA-guided RNA cleavage in vitro and RNA silencing in human cells. The cryo-EM structure of HrAgo1 combined with quantitative single-molecule experiments reveals that HrAgo1 possesses hybrid structural features and target binding modes bridging those of the eukaryotic AGO and PIWI clades. Finally, genomic evidence suggests that eukaryotic Dicer-like processing of double-stranded RNA likely emerged as a mechanism of generating guide RNA for asAgos prior to eukaryogenesis. Our study provides new insights into the evolutionary origin and plasticity of Argonaute-based RNA silencing.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
