<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Virological characteristics of the SARS-CoV-2 BA.2.86 variant
Authors: Tamura, T.; Mizuma, K.; Nasser, H.; Deguchi, S.; Padilla-Blanco, M.; Uriu, K.; Tolentino, J. E. M.; Tsujino, S.; Suzuki, R.; Kojima, I.; Nao, N.; Shimizu, R.; Jonathan, M.; Kosugi, Y.; Guo, Z.; Hinay, A. A.; Putri, O.; Kim, Y.; Tanaka, Y. L.; Asakura, H.; Nagashima, M.; Sadamasu, K.; Yoshimura, K.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Saito, A.; Ito, J.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Virological characteristics of the SARS-CoV-2 BA.2.86 variant
Authors: Tamura, T.; Mizuma, K.; Nasser, H.; Deguchi, S.; Padilla-Blanco, M.; Uriu, K.; Tolentino, J. E. M.; Tsujino, S.; Suzuki, R.; Kojima, I.; Nao, N.; Shimizu, R.; Jonathan, M.; Kosugi, Y.; Guo, Z.; Hinay, A. A.; Putri, O.; Kim, Y.; Tanaka, Y. L.; Asakura, H.; Nagashima, M.; Sadamasu, K.; Yoshimura, K.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Saito, A.; Ito, J." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-05T10:37:37+00:00" />
<meta property="article:modified_time" content="2023-11-05T10:37:37+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Virological characteristics of the SARS-CoV-2 BA.2.86 variant
Authors: Tamura, T.; Mizuma, K.; Nasser, H.; Deguchi, S.; Padilla-Blanco, M.; Uriu, K.; Tolentino, J. E. M.; Tsujino, S.; Suzuki, R.; Kojima, I.; Nao, N.; Shimizu, R.; Jonathan, M.; Kosugi, Y.; Guo, Z.; Hinay, A. A.; Putri, O.; Kim, Y.; Tanaka, Y. L.; Asakura, H.; Nagashima, M.; Sadamasu, K.; Yoshimura, K.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Saito, A.; Ito, J."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Virological characteristics of the SARS-CoV-2 BA.2.86 variant\nAuthors: Tamura, T.; Mizuma, K.; Nasser, H.; Deguchi, S.; Padilla-Blanco, M.; Uriu, K.; Tolentino, J. E. M.; Tsujino, S.; Suzuki, R.; Kojima, I.; Nao, N.; Shimizu, R.; Jonathan, M.; Kosugi, Y.; Guo, Z.; Hinay, A. A.; Putri, O.; Kim, Y.; Tanaka, Y. L.; Asakura, H.; Nagashima, M.; Sadamasu, K.; Yoshimura, K.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Saito, A.; Ito, J.",
  "keywords": [
    
  ],
  "articleBody": " Virological characteristics of the SARS-CoV-2 BA.2.86 variant\nAuthors: Tamura, T.; Mizuma, K.; Nasser, H.; Deguchi, S.; Padilla-Blanco, M.; Uriu, K.; Tolentino, J. E. M.; Tsujino, S.; Suzuki, R.; Kojima, I.; Nao, N.; Shimizu, R.; Jonathan, M.; Kosugi, Y.; Guo, Z.; Hinay, A. A.; Putri, O.; Kim, Y.; Tanaka, Y. L.; Asakura, H.; Nagashima, M.; Sadamasu, K.; Yoshimura, K.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Saito, A.; Ito, J.; Irie, T.; Zahradnik, J.; Ikeda, T.; Takayama, K.; Matsuno, K.; Fukuhara, T.; Sato, K.\nScore: 262.6, Published: 2023-11-03 DOI: 10.1101/2023.11.02.565304\nIn late 2023, a lineage of SARS-CoV-2 emerged and was named the BA.2.86 variant. BA.2.86 is phylogenetically distinct from other Omicron sublineages identified so far, displaying an accumulation of over 30 amino acid mutations in its spike protein. Here, we performed multiscale investigations to reveal the virological characteristics of the BA.2.86 variant. Our epidemic dynamics modeling suggested that the relative reproduction number of BA.2.86 is significantly higher than that of EG.5.1. Experimental studies showed that four clinically-available antivirals were effective against BA.2.86. Although the fusogenicity of BA.2.86 spike is similar to that of the parental BA.2 spike, the intrinsic pathogenicity of BA.2.86 in hamsters was significantly lower than that of BA.2. Since the growth kinetics of BA.2.86 is significantly lower than that of BA.2 in both in vitro cell cultures and in vivo, it is suggested that the attenuated pathogenicity of BA.2.86 is due to its decreased replication capacity.\nSARS-CoV-2 infection leads to sustained testicular injury and functional impairments in K18 hACE2 mice\nAuthors: Giannakopoulos, S.; Ward, M. A.; Bakse, J.; Pak, J. H.; Nerurkar, V. R.; Tallquist, M. D.; Verma, S.\nScore: 106.4, Published: 2023-11-01 DOI: 10.1101/2023.10.31.565042\nCompromised male reproductive health is one of the symptoms of long COVID with a decrease in male fertility markers including testosterone levels and sperm count for months in recovering patients. However, the long-term impact of SARS-CoV-2 infection on testicular injury and underlying mechanisms remains unknown. We previously demonstrated a disrupted tissue architecture with no evidence of virus replication in the testis during the acute stage of the disease in K18-hACE2 mice. Here, we systematically delineate the consequences of SARS-CoV-2 infection on the testis injury and function both during the acute stage of the disease and up to 4 weeks after recovery in survivor K18-hACE2 mice. The gross morphological defects included sloughing of healthy spermatids and spermatocytes into the lumen, lack of lumen, and increase in apoptotic cells that sustained for at least 2 weeks after infection. Testis injury correlated with systemic and testicular inflammation, and infiltration of immune cells in the interstitial space and seminiferous tubules. Transcriptomic analysis identified dysregulation of key pathways associated with testicular immune homeostasis, spermatogenesis, and cell death at the symptomatic stage and immediately after recovery. Further, a significant reduction in testosterone levels was associated with transient reduction in sperm count and mouse fertility. Most of the testicular impairments except testosterone levels were resolved within 4 weeks, which is almost one spermatogenesis cycle in mice. These findings provide much-needed mechanistic insights beyond our current understanding of testicular pathogenesis, suggesting that recovering COVID-19 patients should be closely monitored to rescue the pathophysiological effects on male reproductive health.\nWidespread fungal-bacterial competition for magnesium enhances antibiotic resistance\nAuthors: Hsieh, Y.-Y. P.; Sun, W. W.; Young, J. M.; Cheung, R.; Hogan, D. A.; Dandekar, A. A.; Malik, H. S.\nScore: 25.2, Published: 2023-10-26 DOI: 10.1101/2023.10.25.563990\nFungi and bacteria coexist in many polymicrobial communities, yet the molecular basis of their interactions remains poorly understood. Using unbiased genomic approaches, we discover that the fungus Candida albicans sequesters essential Mg2+ ions from the bacterium Pseudomonas aeruginosa. In turn, the bacterium competes using a Mg2+ transporter, MgtA. We show that Mg2+ sequestration by fungi is a general mechanism of antagonism against gram-negative bacteria. But the resultant Mg2+ limitation enhances bacterial resistance to polymyxin antibiotics like colistin, which target gram-negative bacterial membranes. Experimental evolution reveals that bacteria in co-culture with fungi become phenotypically, but not genetically, resistant to colistin; antifungal treatment renders resistant bacteria from co-cultures to become colistin-sensitive. Fungal-bacterial nutritional competition may thus profoundly impact treatments of polymicrobial infections with antibiotics of last resort. One Sentence SummaryMagnesium sequestration by fungi lowers bacterial fitness but enhances antibiotic resistance.\nA global survey of small RNA interactors identifies KhpA and KhpB as major RNA-binding proteins in Fusobacterium nucleatum\nAuthors: Zhu, Y.; Ponath, F.; Cosi, V.; Vogel, J.\nScore: 22.0, Published: 2023-10-30 DOI: 10.1101/2023.10.30.564711\nABSTRACTThe common oral microbe Fusobacterium nucleatum has recently drawn attention after it was found to colonize tumors throughout the human body. Fusobacteria are also interesting study systems for bacterial RNA biology as these early-branching species encode many small noncoding RNAs (sRNAs) but lack homologs of the common RNA-binding proteins (RBPs) CsrA, Hfq and ProQ. To search for alternate sRNA-associated RBPs in F. nucleatum, we performed a systematic mass spectrometry analysis of proteins that co-purified with 19 different sRNAs. This approach revealed strong enrichment of the KH domain proteins KhpA and KhpB with nearly all tested sRNAs, including the {sigma}E-dependent sRNA FoxI, a regulator of several envelope proteins. KhpA/B act as a dimer to bind sRNAs with low micromolar affinity and influence the stability of several of their target transcripts. Transcriptome studies combined with biochemical and genetic analyses suggest that KhpA/B have several physiological functions, including being required for ethanolamine utilization. Our RBP search and the discovery of KhpA and KhpB as major RBPs in F. nucleatum are important first steps in identifying key players of post-transcriptional control at the root of the bacterial phylogenetic tree.\nCyanobacteria newly isolated from marine volcanic seeps display rapid sinking and robust, high density growth\nAuthors: Schubert, M. G.; Tang, T.-C.; Goodchild-Michelman, I. M.; Ryon, K. A.; Henriksen, J. R.; Chavkin, T.; Wu, Y.; Miettinen, T. P.; Van Wychen, S.; Dahlin, L.; Spatafora, D.; Turco, G.; Guarnieri, M.; Manalis, S. R.; Kowitz, J.; Dhir, R.; Quatrini, P.; Mason, C. E.; Church, G. M.; Milazzo, M.; Tierney, B. T.\nScore: 19.9, Published: 2023-10-30 DOI: 10.1101/2023.10.30.564770\nCyanobacteria are photosynthetic organisms that play important roles in carbon cycling as well as promising bioproduction chassis. Here, we isolate two novel cyanobacteria, UTEX 3221 and UTEX 3222, from a unique marine environment with naturally elevated CO2. We describe complete genome sequences for both isolates and, focusing on UTEX 3222 due to its planktonic growth in liquid, characterize biotechnologically-relevant growth and biomass characteristics. UTEX 3222 outpaces other fast-growing model strains on solid medium. It can double every 2.35 hours in a liquid medium and grows to high density (\u003e31g/L biomass dry weight) in batch culture, nearly double that of Synechococcus sp. PCC 11901, whose high-density growth was recently reported. In addition, UTEX 3222 sinks readily, settling more quickly than other fast-growing strains, suggesting improved de-watering of UTEX 3222 biomass. This settling behavior can be explained in part by larger cell volume. These traits may make UTEX 3222 a compelling choice for photosynthetic bioproduction from CO2. Overall, we find that bio-prospecting in environments with naturally elevated CO2 may uncover novel CO2-metabolizing organisms with unique characteristics.\nA virally encoded high resolution screen of cytomegalovirus host dependencies\nAuthors: Finkel, Y.; Nachshon, A.; Aharon, E.; Arazi, T.; Simonovsky, E.; Dobesova, M.; Gluck, A.; Fisher, T.; Stanton, R. J.; Schwartz, M.; Stern-Ginossar, N.\nScore: 18.8, Published: 2023-10-30 DOI: 10.1101/2023.10.30.564793\nGenetic screens have transformed our ability to interrogate cellular factor requirements in infection, yet current approaches are limited in their sensitivity, biased towards early stages of infection and provide only simplistic phenotypic information which is often based on infected cell survival. Here, by engineering human cytomegalovirus to express sgRNA libraries directly from the viral genome, we developed a sensitive, versatile, viral centric approach that allows profiling of different stages along viral infection in a pooled format. Using this approach, which we termed VECOS (Virus Encoded CRISPR-based direct readOut Screening system), we identified hundreds of novel host dependency and restriction factors and quantified their direct effects on viral genome replication, viral particle secretion and infectiousness of secreted particles, providing a multi-dimensional perspective on viral-host interactions. These high resolution measurements reveal that perturbations that alter late stages in HCMV life cycle mostly regulate HCMV particle quality rather than quantity, defining correct virion assembly as a critical stage that is heavily reliant on viral-host interactions. Overall, VECOS facilitates systematic high resolution dissection of human proteins role along the infection cycle, providing a roadmap for in-depth dissection of host-herpesvirus interactions.\nViral genomic features predict orthopoxvirus reservoir hosts\nAuthors: Tseng, K. K.; Koehler, H.; Becker, D. J.; Gibb, R.; Carlson, C. J.; Fernandez, M. d. P.; Seifert, S. N.\nScore: 16.4, Published: 2023-10-27 DOI: 10.1101/2023.10.26.564211\nOrthopoxviruses (OPVs), including the causative agents of smallpox and mpox have led to devastating outbreaks in human populations worldwide. However, the discontinuation of smallpox vaccination, which also provides cross-protection against related OPVs, has diminished global immunity to OPVs more broadly. We apply machine learning models incorporating both host ecological and viral genomic features to predict likely reservoirs of OPVs. We demonstrate that incorporating viral genomic features in addition to host ecological traits enhanced the accuracy of potential OPV host predictions, highlighting the importance of host-virus molecular interactions in predicting potential host species. We identify hotspots for geographic regions rich with potential OPV hosts in parts of southeast Asia, equatorial Africa, and the Amazon, revealing high overlap between regions predicted to have a high number of potential OPV host species and those with the lowest smallpox vaccination coverage, indicating a heightened risk for the emergence or establishment of zoonotic OPVs. Our findings can be used to target wildlife surveillance, particularly related to concerns about mpox establishment beyond its historical range.\nIdentification of novel tail-anchored membrane proteins integrated by the bacterial twin-arginine translocase\nAuthors: Gallego-Parrilla, J. J.; SEVERI, E.; Chandra, G.; Palmer, T.\nScore: 12.4, Published: 2023-11-02 DOI: 10.1101/2023.11.02.564894\nThe twin arginine protein transport (Tat) system exports folded proteins across the cytoplasmic membranes of prokaryotes and the energy transducing membranes of plant thylakoids and mitochondria. Proteins are targeted to the Tat machinery by N-terminal signal peptides with a conserved twin arginine motif, and some substrates are exported as heterodimers where the signal peptide is present on one of the partner proteins. A subset of Tat substrates is found in the membrane. Tat-dependent membrane proteins usually have large globular domains and a single transmembrane helix present at the N- or C-terminus. Five Tat substrates that have C-terminal transmembrane helices have previously been characterised in the model bacterium Escherichia coli. Each of these is an iron-sulphur cluster-containing protein involved in electron transfer from hydrogen or formate. Here we have undertaken a bioinformatic search to identify further tail-anchored Tat substrates encoded in bacterial genomes. Our analysis has revealed additional tail-anchored iron-sulphur proteins associated in modules with either a b-type cytochrome or a quinol oxidase. We also identified further candidate tail-anchored Tat substrates, particularly among members of the actinobacterial genus, that are not predicted to contain cofactors. Using reporter assays we show experimentally that six of these have both N-terminal Tat signal peptides and C-terminal transmembrane helices. The newly-identified proteins include a carboxypeptidase and a predicted protease, and four sortase substrates for which membrane integration is a pre-requisite for covalent attachment to the cell wall.\nQueuosine biosynthetic enzyme, QueE moonlights as a cell division regulator\nAuthors: Adeleye, S. A.; Yadavalli, S. S.\nScore: 11.0, Published: 2023-11-01 DOI: 10.1101/2023.10.31.565030\nIn many organisms, stress responses to adverse environments can trigger secondary functions of certain proteins by altering protein levels, localization, activity, or interaction partners. Escherichia coli cells respond to the presence of specific cationic antimicrobial peptides by strongly activating the PhoQ/PhoP two-component signaling system, which regulates genes important for growth under this stress. As part of this pathway, a biosynthetic enzyme called QueE, which catalyzes a step in the formation of queuosine (Q) tRNA modification is upregulated. When cellular QueE levels are high, it co-localizes with the central cell division protein FtsZ at the septal site, blocking division and resulting in filamentous growth. Here we show that QueE affects cell size in a dose-dependent manner. Using alanine scanning mutagenesis of amino acids in the catalytic active site, we pinpoint particular residues in QueE that contribute distinctly to each of its functions - Q biosynthesis or regulation of cell division, establishing QueE as a moonlighting protein. We further show that QueE orthologs from enterobacteria like Salmonella typhimurium and Klebsiella pneumoniae also cause filamentation in these organisms, but the more distant counterparts from Pseudomonas aeruginosa and Bacillus subtilis lack this ability. By comparative analysis of E. coli QueE with distant orthologs, we elucidate a unique region in this protein that is responsible for QueEs secondary function as a cell division regulator. A dual-function protein like QueE is an exception to the conventional model of one gene, one enzyme, one function, which has divergent roles across a range of fundamental cellular processes including RNA modification and translation to cell division and stress response.\nCell surface architecture of the cultivated DPANN archaeon Nanobdella aerobiophila\nAuthors: Kato, S.; Tahara, Y. O.; Nishimura, Y.; Uematsu, K.; Arai, T.; Nakane, D.; Ihara, A.; Nishizaka, T.; Iwasaki, W.; Itoh, T.; Miyata, M.; Ohkuma, M.\nScore: 10.2, Published: 2023-10-28 DOI: 10.1101/2023.10.27.564410\nThe DPANN archaeal clade includes obligately ectosymbiotic species. Their cell surfaces potentially play an important role in the symbiotic interaction between the ectosymbionts and their hosts. However, little is known about the mechanism of the ectosymbiosis. Here, we show cell surface structures of the cultivated DPANN archaeon Nanobdella aerobiophila strain MJ1T and its host Metallosphaera sedula strain MJ1HA, using a variety of electron microscopy techniques, i.e., negative-staining transmission electron microscopy (TEM), quick-freeze deep-etch (QFDE) TEM, and 3D electron tomography. The thickness, unit size, and lattice symmetry of the S-layer of strain MJ1T were different from those of the host archaeon strain MJ1HA. Genomic and transcriptomic analyses highlighted the most highly expressed MJ1T gene for a putative S-layer protein with multiple glycosylation sites and immunoglobulin-like folds, which has no sequence homology to known S-layer proteins. In addition, genes for putative pectin lyase- or lectin-like extracellular proteins, which are potentially involved in symbiotic interaction, were found in the MJ1T genome based on in silico 3D protein structure prediction. Live cell imaging at the optimum growth temperature of 65{degrees}C indicated that cell complexes of strains MJ1T and MJ1HA were motile, but sole MJ1T cells were not. Taken together, we propose a model of the symbiotic interaction and cell cycle of Nanobdella aerobiophila. ImportanceDPANN archaea are widely distributed in a variety of natural and artificial environments, and may play a considerable role in the microbial ecosystem. All of the cultivated DPANN archaea so far need host organisms for their growth, i.e., obligately ectosymbiotic. However, the mechanism of the ectosymbiosis by DPANN archaea is largely unknown. To this end, we performed a comprehensive analysis of the cultivated DPANN archaeon, Nanobdella aerobiophila, using electron microscopy, live cell imaging, transcriptomics, and genomics including 3D protein structure prediction. Based on the results, we propose a reasonable model of the symbiotic interaction and cell cycle of Nanobdella aerobiophila, which will enhance our understanding of the enigmatic physiology and ecological significance of DPANN archaea.\n",
  "wordCount" : "2505",
  "inLanguage": "en",
  "datePublished": "2023-11-05T10:37:37Z",
  "dateModified": "2023-11-05T10:37:37Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 5, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.565304">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.565304" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.565304">
        <p class="paperTitle">Virological characteristics of the SARS-CoV-2 BA.2.86 variant</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.565304" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.565304" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tamura, T.; Mizuma, K.; Nasser, H.; Deguchi, S.; Padilla-Blanco, M.; Uriu, K.; Tolentino, J. E. M.; Tsujino, S.; Suzuki, R.; Kojima, I.; Nao, N.; Shimizu, R.; Jonathan, M.; Kosugi, Y.; Guo, Z.; Hinay, A. A.; Putri, O.; Kim, Y.; Tanaka, Y. L.; Asakura, H.; Nagashima, M.; Sadamasu, K.; Yoshimura, K.; The Genotype to Phenotype Japan (G2P-Japan) Consortium,  ; Saito, A.; Ito, J.; Irie, T.; Zahradnik, J.; Ikeda, T.; Takayama, K.; Matsuno, K.; Fukuhara, T.; Sato, K.</p>
        <p class="info">Score: 262.6, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.565304' target='https://doi.org/10.1101/2023.11.02.565304'> 10.1101/2023.11.02.565304</a></p>
        <p class="abstract">In late 2023, a lineage of SARS-CoV-2 emerged and was named the BA.2.86 variant. BA.2.86 is phylogenetically distinct from other Omicron sublineages identified so far, displaying an accumulation of over 30 amino acid mutations in its spike protein. Here, we performed multiscale investigations to reveal the virological characteristics of the BA.2.86 variant. Our epidemic dynamics modeling suggested that the relative reproduction number of BA.2.86 is significantly higher than that of EG.5.1. Experimental studies showed that four clinically-available antivirals were effective against BA.2.86. Although the fusogenicity of BA.2.86 spike is similar to that of the parental BA.2 spike, the intrinsic pathogenicity of BA.2.86 in hamsters was significantly lower than that of BA.2. Since the growth kinetics of BA.2.86 is significantly lower than that of BA.2 in both in vitro cell cultures and in vivo, it is suggested that the attenuated pathogenicity of BA.2.86 is due to its decreased replication capacity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.565042">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.565042" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.565042">
        <p class="paperTitle">SARS-CoV-2 infection leads to sustained testicular injury and functional impairments in K18 hACE2 mice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.565042" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.565042" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Giannakopoulos, S.; Ward, M. A.; Bakse, J.; Pak, J. H.; Nerurkar, V. R.; Tallquist, M. D.; Verma, S.</p>
        <p class="info">Score: 106.4, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.565042' target='https://doi.org/10.1101/2023.10.31.565042'> 10.1101/2023.10.31.565042</a></p>
        <p class="abstract">Compromised male reproductive health is one of the symptoms of long COVID with a decrease in male fertility markers including testosterone levels and sperm count for months in recovering patients. However, the long-term impact of SARS-CoV-2 infection on testicular injury and underlying mechanisms remains unknown. We previously demonstrated a disrupted tissue architecture with no evidence of virus replication in the testis during the acute stage of the disease in K18-hACE2 mice. Here, we systematically delineate the consequences of SARS-CoV-2 infection on the testis injury and function both during the acute stage of the disease and up to 4 weeks after recovery in survivor K18-hACE2 mice. The gross morphological defects included sloughing of healthy spermatids and spermatocytes into the lumen, lack of lumen, and increase in apoptotic cells that sustained for at least 2 weeks after infection. Testis injury correlated with systemic and testicular inflammation, and infiltration of immune cells in the interstitial space and seminiferous tubules. Transcriptomic analysis identified dysregulation of key pathways associated with testicular immune homeostasis, spermatogenesis, and cell death at the symptomatic stage and immediately after recovery. Further, a significant reduction in testosterone levels was associated with transient reduction in sperm count and mouse fertility. Most of the testicular impairments except testosterone levels were resolved within 4 weeks, which is almost one spermatogenesis cycle in mice. These findings provide much-needed mechanistic insights beyond our current understanding of testicular pathogenesis, suggesting that recovering COVID-19 patients should be closely monitored to rescue the pathophysiological effects on male reproductive health.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.563990">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.563990" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.563990">
        <p class="paperTitle">Widespread fungal-bacterial competition for magnesium enhances antibiotic resistance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.563990" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.563990" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hsieh, Y.-Y. P.; Sun, W. W.; Young, J. M.; Cheung, R.; Hogan, D. A.; Dandekar, A. A.; Malik, H. S.</p>
        <p class="info">Score: 25.2, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.563990' target='https://doi.org/10.1101/2023.10.25.563990'> 10.1101/2023.10.25.563990</a></p>
        <p class="abstract">Fungi and bacteria coexist in many polymicrobial communities, yet the molecular basis of their interactions remains poorly understood. Using unbiased genomic approaches, we discover that the fungus Candida albicans sequesters essential Mg2&#43; ions from the bacterium Pseudomonas aeruginosa. In turn, the bacterium competes using a Mg2&#43; transporter, MgtA. We show that Mg2&#43; sequestration by fungi is a general mechanism of antagonism against gram-negative bacteria. But the resultant Mg2&#43; limitation enhances bacterial resistance to polymyxin antibiotics like colistin, which target gram-negative bacterial membranes. Experimental evolution reveals that bacteria in co-culture with fungi become phenotypically, but not genetically, resistant to colistin; antifungal treatment renders resistant bacteria from co-cultures to become colistin-sensitive. Fungal-bacterial nutritional competition may thus profoundly impact treatments of polymicrobial infections with antibiotics of last resort.

One Sentence SummaryMagnesium sequestration by fungi lowers bacterial fitness but enhances antibiotic resistance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564711">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564711" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564711">
        <p class="paperTitle">A global survey of small RNA interactors identifies KhpA and KhpB as major RNA-binding proteins in Fusobacterium nucleatum</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564711" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564711" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhu, Y.; Ponath, F.; Cosi, V.; Vogel, J.</p>
        <p class="info">Score: 22.0, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564711' target='https://doi.org/10.1101/2023.10.30.564711'> 10.1101/2023.10.30.564711</a></p>
        <p class="abstract">ABSTRACTThe common oral microbe Fusobacterium nucleatum has recently drawn attention after it was found to colonize tumors throughout the human body. Fusobacteria are also interesting study systems for bacterial RNA biology as these early-branching species encode many small noncoding RNAs (sRNAs) but lack homologs of the common RNA-binding proteins (RBPs) CsrA, Hfq and ProQ. To search for alternate sRNA-associated RBPs in F. nucleatum, we performed a systematic mass spectrometry analysis of proteins that co-purified with 19 different sRNAs. This approach revealed strong enrichment of the KH domain proteins KhpA and KhpB with nearly all tested sRNAs, including the {sigma}E-dependent sRNA FoxI, a regulator of several envelope proteins. KhpA/B act as a dimer to bind sRNAs with low micromolar affinity and influence the stability of several of their target transcripts. Transcriptome studies combined with biochemical and genetic analyses suggest that KhpA/B have several physiological functions, including being required for ethanolamine utilization. Our RBP search and the discovery of KhpA and KhpB as major RBPs in F. nucleatum are important first steps in identifying key players of post-transcriptional control at the root of the bacterial phylogenetic tree.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564770">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564770" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564770">
        <p class="paperTitle">Cyanobacteria newly isolated from marine volcanic seeps display rapid sinking and robust, high density growth</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564770" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564770" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schubert, M. G.; Tang, T.-C.; Goodchild-Michelman, I. M.; Ryon, K. A.; Henriksen, J. R.; Chavkin, T.; Wu, Y.; Miettinen, T. P.; Van Wychen, S.; Dahlin, L.; Spatafora, D.; Turco, G.; Guarnieri, M.; Manalis, S. R.; Kowitz, J.; Dhir, R.; Quatrini, P.; Mason, C. E.; Church, G. M.; Milazzo, M.; Tierney, B. T.</p>
        <p class="info">Score: 19.9, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564770' target='https://doi.org/10.1101/2023.10.30.564770'> 10.1101/2023.10.30.564770</a></p>
        <p class="abstract">Cyanobacteria are photosynthetic organisms that play important roles in carbon cycling as well as promising bioproduction chassis. Here, we isolate two novel cyanobacteria, UTEX 3221 and UTEX 3222, from a unique marine environment with naturally elevated CO2. We describe complete genome sequences for both isolates and, focusing on UTEX 3222 due to its planktonic growth in liquid, characterize biotechnologically-relevant growth and biomass characteristics. UTEX 3222 outpaces other fast-growing model strains on solid medium. It can double every 2.35 hours in a liquid medium and grows to high density (&gt;31g/L biomass dry weight) in batch culture, nearly double that of Synechococcus sp. PCC 11901, whose high-density growth was recently reported. In addition, UTEX 3222 sinks readily, settling more quickly than other fast-growing strains, suggesting improved de-watering of UTEX 3222 biomass. This settling behavior can be explained in part by larger cell volume. These traits may make UTEX 3222 a compelling choice for photosynthetic bioproduction from CO2. Overall, we find that bio-prospecting in environments with naturally elevated CO2 may uncover novel CO2-metabolizing organisms with unique characteristics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564793">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564793" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564793">
        <p class="paperTitle">A virally encoded high resolution screen of cytomegalovirus host dependencies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564793" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564793" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Finkel, Y.; Nachshon, A.; Aharon, E.; Arazi, T.; Simonovsky, E.; Dobesova, M.; Gluck, A.; Fisher, T.; Stanton, R. J.; Schwartz, M.; Stern-Ginossar, N.</p>
        <p class="info">Score: 18.8, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564793' target='https://doi.org/10.1101/2023.10.30.564793'> 10.1101/2023.10.30.564793</a></p>
        <p class="abstract">Genetic screens have transformed our ability to interrogate cellular factor requirements in infection, yet current approaches are limited in their sensitivity, biased towards early stages of infection and provide only simplistic phenotypic information which is often based on infected cell survival. Here, by engineering human cytomegalovirus to express sgRNA libraries directly from the viral genome, we developed a sensitive, versatile, viral centric approach that allows profiling of different stages along viral infection in a pooled format. Using this approach, which we termed VECOS (Virus Encoded CRISPR-based direct readOut Screening system), we identified hundreds of novel host dependency and restriction factors and quantified their direct effects on viral genome replication, viral particle secretion and infectiousness of secreted particles, providing a multi-dimensional perspective on viral-host interactions. These high resolution measurements reveal that perturbations that alter late stages in HCMV life cycle mostly regulate HCMV particle quality rather than quantity, defining correct virion assembly as a critical stage that is heavily reliant on viral-host interactions. Overall, VECOS facilitates systematic high resolution dissection of human proteins role along the infection cycle, providing a roadmap for in-depth dissection of host-herpesvirus interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564211">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564211" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564211">
        <p class="paperTitle">Viral genomic features predict orthopoxvirus reservoir hosts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564211" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564211" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tseng, K. K.; Koehler, H.; Becker, D. J.; Gibb, R.; Carlson, C. J.; Fernandez, M. d. P.; Seifert, S. N.</p>
        <p class="info">Score: 16.4, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564211' target='https://doi.org/10.1101/2023.10.26.564211'> 10.1101/2023.10.26.564211</a></p>
        <p class="abstract">Orthopoxviruses (OPVs), including the causative agents of smallpox and mpox have led to devastating outbreaks in human populations worldwide. However, the discontinuation of smallpox vaccination, which also provides cross-protection against related OPVs, has diminished global immunity to OPVs more broadly. We apply machine learning models incorporating both host ecological and viral genomic features to predict likely reservoirs of OPVs. We demonstrate that incorporating viral genomic features in addition to host ecological traits enhanced the accuracy of potential OPV host predictions, highlighting the importance of host-virus molecular interactions in predicting potential host species. We identify hotspots for geographic regions rich with potential OPV hosts in parts of southeast Asia, equatorial Africa, and the Amazon, revealing high overlap between regions predicted to have a high number of potential OPV host species and those with the lowest smallpox vaccination coverage, indicating a heightened risk for the emergence or establishment of zoonotic OPVs. Our findings can be used to target wildlife surveillance, particularly related to concerns about mpox establishment beyond its historical range.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.564894">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.564894" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.564894">
        <p class="paperTitle">Identification of novel tail-anchored membrane proteins integrated by the bacterial twin-arginine translocase</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.564894" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.564894" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gallego-Parrilla, J. J.; SEVERI, E.; Chandra, G.; Palmer, T.</p>
        <p class="info">Score: 12.4, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.564894' target='https://doi.org/10.1101/2023.11.02.564894'> 10.1101/2023.11.02.564894</a></p>
        <p class="abstract">The twin arginine protein transport (Tat) system exports folded proteins across the cytoplasmic membranes of prokaryotes and the energy transducing membranes of plant thylakoids and mitochondria. Proteins are targeted to the Tat machinery by N-terminal signal peptides with a conserved twin arginine motif, and some substrates are exported as heterodimers where the signal peptide is present on one of the partner proteins. A subset of Tat substrates is found in the membrane. Tat-dependent membrane proteins usually have large globular domains and a single transmembrane helix present at the N- or C-terminus. Five Tat substrates that have C-terminal transmembrane helices have previously been characterised in the model bacterium Escherichia coli. Each of these is an iron-sulphur cluster-containing protein involved in electron transfer from hydrogen or formate. Here we have undertaken a bioinformatic search to identify further tail-anchored Tat substrates encoded in bacterial genomes. Our analysis has revealed additional tail-anchored iron-sulphur proteins associated in modules with either a b-type cytochrome or a quinol oxidase. We also identified further candidate tail-anchored Tat substrates, particularly among members of the actinobacterial genus, that are not predicted to contain cofactors. Using reporter assays we show experimentally that six of these have both N-terminal Tat signal peptides and C-terminal transmembrane helices. The newly-identified proteins include a carboxypeptidase and a predicted protease, and four sortase substrates for which membrane integration is a pre-requisite for covalent attachment to the cell wall.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.565030">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.565030" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.565030">
        <p class="paperTitle">Queuosine biosynthetic enzyme, QueE moonlights as a cell division regulator</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.565030" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.565030" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Adeleye, S. A.; Yadavalli, S. S.</p>
        <p class="info">Score: 11.0, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.565030' target='https://doi.org/10.1101/2023.10.31.565030'> 10.1101/2023.10.31.565030</a></p>
        <p class="abstract">In many organisms, stress responses to adverse environments can trigger secondary functions of certain proteins by altering protein levels, localization, activity, or interaction partners. Escherichia coli cells respond to the presence of specific cationic antimicrobial peptides by strongly activating the PhoQ/PhoP two-component signaling system, which regulates genes important for growth under this stress. As part of this pathway, a biosynthetic enzyme called QueE, which catalyzes a step in the formation of queuosine (Q) tRNA modification is upregulated. When cellular QueE levels are high, it co-localizes with the central cell division protein FtsZ at the septal site, blocking division and resulting in filamentous growth. Here we show that QueE affects cell size in a dose-dependent manner. Using alanine scanning mutagenesis of amino acids in the catalytic active site, we pinpoint particular residues in QueE that contribute distinctly to each of its functions - Q biosynthesis or regulation of cell division, establishing QueE as a moonlighting protein. We further show that QueE orthologs from enterobacteria like Salmonella typhimurium and Klebsiella pneumoniae also cause filamentation in these organisms, but the more distant counterparts from Pseudomonas aeruginosa and Bacillus subtilis lack this ability. By comparative analysis of E. coli QueE with distant orthologs, we elucidate a unique region in this protein that is responsible for QueEs secondary function as a cell division regulator. A dual-function protein like QueE is an exception to the conventional model of one gene, one enzyme, one function, which has divergent roles across a range of fundamental cellular processes including RNA modification and translation to cell division and stress response.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564410">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564410" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564410">
        <p class="paperTitle">Cell surface architecture of the cultivated DPANN archaeon Nanobdella aerobiophila</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564410" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564410" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kato, S.; Tahara, Y. O.; Nishimura, Y.; Uematsu, K.; Arai, T.; Nakane, D.; Ihara, A.; Nishizaka, T.; Iwasaki, W.; Itoh, T.; Miyata, M.; Ohkuma, M.</p>
        <p class="info">Score: 10.2, Published: 2023-10-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564410' target='https://doi.org/10.1101/2023.10.27.564410'> 10.1101/2023.10.27.564410</a></p>
        <p class="abstract">The DPANN archaeal clade includes obligately ectosymbiotic species. Their cell surfaces potentially play an important role in the symbiotic interaction between the ectosymbionts and their hosts. However, little is known about the mechanism of the ectosymbiosis. Here, we show cell surface structures of the cultivated DPANN archaeon Nanobdella aerobiophila strain MJ1T and its host Metallosphaera sedula strain MJ1HA, using a variety of electron microscopy techniques, i.e., negative-staining transmission electron microscopy (TEM), quick-freeze deep-etch (QFDE) TEM, and 3D electron tomography. The thickness, unit size, and lattice symmetry of the S-layer of strain MJ1T were different from those of the host archaeon strain MJ1HA. Genomic and transcriptomic analyses highlighted the most highly expressed MJ1T gene for a putative S-layer protein with multiple glycosylation sites and immunoglobulin-like folds, which has no sequence homology to known S-layer proteins. In addition, genes for putative pectin lyase- or lectin-like extracellular proteins, which are potentially involved in symbiotic interaction, were found in the MJ1T genome based on in silico 3D protein structure prediction. Live cell imaging at the optimum growth temperature of 65{degrees}C indicated that cell complexes of strains MJ1T and MJ1HA were motile, but sole MJ1T cells were not. Taken together, we propose a model of the symbiotic interaction and cell cycle of Nanobdella aerobiophila.

ImportanceDPANN archaea are widely distributed in a variety of natural and artificial environments, and may play a considerable role in the microbial ecosystem. All of the cultivated DPANN archaea so far need host organisms for their growth, i.e., obligately ectosymbiotic. However, the mechanism of the ectosymbiosis by DPANN archaea is largely unknown. To this end, we performed a comprehensive analysis of the cultivated DPANN archaeon, Nanobdella aerobiophila, using electron microscopy, live cell imaging, transcriptomics, and genomics including 3D protein structure prediction. Based on the results, we propose a reasonable model of the symbiotic interaction and cell cycle of Nanobdella aerobiophila, which will enhance our understanding of the enigmatic physiology and ecological significance of DPANN archaea.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
