<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Virological characteristics of the SARS-CoV-2 Omicron EG.5.1 variant
Authors: Tsujino, S.; Deguchi, S.; Nomai, T.; Padilla-Blanco, M.; Plianchaisuk, A.; Wang, L.; Begum, M. M.; Uriu, K.; Mizuma, K.; Nao, N.; Kojima, I.; Tsubo, T.; Li, J.; Matsumura, Y.; Nagao, M.; Oda, Y.; Tsuda, M.; Anraku, Y.; Kita, S.; Yajima, H.; Tabata, K.; Guo, Z.; Hinay, A. A.; Yoshimatsu, K.; Yamamoto, Y.; Nagamoto, Y.; Asakura, H.; Nagashima, M.; Sadamasu, K.; Yoshimura, K.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Virological characteristics of the SARS-CoV-2 Omicron EG.5.1 variant
Authors: Tsujino, S.; Deguchi, S.; Nomai, T.; Padilla-Blanco, M.; Plianchaisuk, A.; Wang, L.; Begum, M. M.; Uriu, K.; Mizuma, K.; Nao, N.; Kojima, I.; Tsubo, T.; Li, J.; Matsumura, Y.; Nagao, M.; Oda, Y.; Tsuda, M.; Anraku, Y.; Kita, S.; Yajima, H.; Tabata, K.; Guo, Z.; Hinay, A. A.; Yoshimatsu, K.; Yamamoto, Y.; Nagamoto, Y.; Asakura, H.; Nagashima, M.; Sadamasu, K.; Yoshimura, K." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-25T10:39:04+00:00" />
<meta property="article:modified_time" content="2023-10-25T10:39:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Virological characteristics of the SARS-CoV-2 Omicron EG.5.1 variant
Authors: Tsujino, S.; Deguchi, S.; Nomai, T.; Padilla-Blanco, M.; Plianchaisuk, A.; Wang, L.; Begum, M. M.; Uriu, K.; Mizuma, K.; Nao, N.; Kojima, I.; Tsubo, T.; Li, J.; Matsumura, Y.; Nagao, M.; Oda, Y.; Tsuda, M.; Anraku, Y.; Kita, S.; Yajima, H.; Tabata, K.; Guo, Z.; Hinay, A. A.; Yoshimatsu, K.; Yamamoto, Y.; Nagamoto, Y.; Asakura, H.; Nagashima, M.; Sadamasu, K.; Yoshimura, K."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Virological characteristics of the SARS-CoV-2 Omicron EG.5.1 variant\nAuthors: Tsujino, S.; Deguchi, S.; Nomai, T.; Padilla-Blanco, M.; Plianchaisuk, A.; Wang, L.; Begum, M. M.; Uriu, K.; Mizuma, K.; Nao, N.; Kojima, I.; Tsubo, T.; Li, J.; Matsumura, Y.; Nagao, M.; Oda, Y.; Tsuda, M.; Anraku, Y.; Kita, S.; Yajima, H.; Tabata, K.; Guo, Z.; Hinay, A. A.; Yoshimatsu, K.; Yamamoto, Y.; Nagamoto, Y.; Asakura, H.; Nagashima, M.; Sadamasu, K.; Yoshimura, K.",
  "keywords": [
    
  ],
  "articleBody": " Virological characteristics of the SARS-CoV-2 Omicron EG.5.1 variant\nAuthors: Tsujino, S.; Deguchi, S.; Nomai, T.; Padilla-Blanco, M.; Plianchaisuk, A.; Wang, L.; Begum, M. M.; Uriu, K.; Mizuma, K.; Nao, N.; Kojima, I.; Tsubo, T.; Li, J.; Matsumura, Y.; Nagao, M.; Oda, Y.; Tsuda, M.; Anraku, Y.; Kita, S.; Yajima, H.; Tabata, K.; Guo, Z.; Hinay, A. A.; Yoshimatsu, K.; Yamamoto, Y.; Nagamoto, Y.; Asakura, H.; Nagashima, M.; Sadamasu, K.; Yoshimura, K.; Nasser, H.; Jonathan, M.; Putri, O.; Kim, Y.; Chen, L.; Suzuki, R.; Tamura, T.; Maenaka, K.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Irie, T.; Matsuno, K.; Tanaka, S.; Ito, J.; Ikeda, T.; Takayama, K.; Za\nScore: 146.9, Published: 2023-10-21 DOI: 10.1101/2023.10.19.563209\nIn middle-late 2023, a sublineage of SARS-CoV-2 Omicron XBB, EG.5.1 (a progeny of XBB.1.9.2), is spreading rapidly around the world. Here, we performed multiscale investigations to reveal virological features of newly emerging EG.5.1 variant. Our phylogenetic-epidemic dynamics modeling suggested that two hallmark substitutions of EG.5.1, S:F456L and ORF9b:I5T, are critical to the increased viral fitness. Experimental investigations addressing the growth kinetics, sensitivity to clinically available antivirals, fusogenicity and pathogenicity of EG.5.1 suggested that the virological features of EG.5.1 is comparable to that of XBB.1.5. However, the cryo-electron microscopy reveals the structural difference between the spike proteins of EG.5.1 and XBB.1.5. We further assessed the impact of ORF9b:I5T on viral features, but it was almost negligible at least in our experimental setup. Our multiscale investigations provide the knowledge for understanding of the evolution trait of newly emerging pathogenic viruses in the human population.\nKatG catalase deficiency confers bedaquiline hyper-susceptibility to isoniazid resistant Mycobacterium tuberculosis\nAuthors: Ofori-Anyinam, N.; Hamblin, M.; Coldren, M. L.; Li, B.; Mereddy, G.; Shaikh, M.; Shah, A.; Ranu, N.; Lu, S.; Blainey, P. C.; Ma, S.; Collins, J. J.; Yang, J. H.\nScore: 15.6, Published: 2023-10-17 DOI: 10.1101/2023.10.17.562707\nMultidrug-resistant tuberculosis (MDR-TB) is a growing source of global mortality and threatens global control of tuberculosis (TB) disease. The diarylquinoline bedaquiline (BDQ) recently emerged as a highly efficacious drug against MDR-TB, defined as resistance to the first-line drugs isoniazid (INH) and rifampin. INH resistance is primarily caused by loss-of-function mutations in the catalase KatG, but mechanisms underlying BDQs efficacy against MDR-TB remain unknown. Here we employ a systems biology approach to investigate BDQ hyper-susceptibility in INH-resistant Mycobacterium tuberculosis. We found hyper-susceptibility to BDQ in INH-resistant cells is due to several physiological changes induced by KatG deficiency, including increased susceptibility to reactive oxygen species and DNA damage, remodeling of transcriptional programs, and metabolic repression of folate biosynthesis. We demonstrate BDQ hyper-susceptibility is common in INH-resistant clinical isolates. Collectively, these results highlight how altered bacterial physiology can impact drug efficacy in drug-resistant bacteria.\nPhages Produce Persisters\nAuthors: Fernandez-Garcia, L.; Kirigo, J.; Huelgas-Mendez, D.; Tomas, M.; Garcia-Contreras, R.; Wood, T. K.\nScore: 15.3, Published: 2023-10-17 DOI: 10.1101/2023.10.17.562728\nBacteria primarily encounter stress, and, arguably, their greatest threats are phages. It is often assumed that those bacteria that escape phage attack have mutated; however, another possibility is that a subpopulation forms the dormant persister state, in a manner similar to that demonstrated for bacterial cells undergoing nutritive, oxidative, and antibiotic stress. Persister cells do not undergo mutation and survive lethal conditions by ceasing growth transiently. Slower growth and dormancy play a key physiological role as they allow host phage defense systems more time to clear the phage infection. Here we investigated how bacteria survive lytic phage infection by isolating surviving cells from the plaques of T2, T4, and lambda (cI mutant) virulent phages. We found that bacteria in plaques can escape phage attack both by mutation (i.e., become resistant) and without mutation (i.e., become persistent). Specifically, whereas T4-resistant and lambda-resistant bacteria with over a 100,000-fold less sensitivity were isolated from plaques with obvious genetic mutations (e.g., causing mucoidy), cells were also found after T2 infection that undergo no significant mutation, retain wild-type phage sensitivity, and survive lethal doses of antibiotics. Corroborating this, adding T2 phage to persister cells resulted in 137,000-fold more survival compared to that of addition to exponentially-growing cells. Phage treatments with Klebsiella pneumonia and Pseudomonas aeruginosa also generated persister cells. Hence, along with resistant strains, bacteria also form persister cells during phage infection.\nExploring archaeogenetic studies of dental calculus to shed light on past human migrations in Oceania\nAuthors: Velsko, I. M.; Fagernas, Z.; Tromp, M.; Bedford, S.; Buckley, H.; Clark, G.; Dudgeon, J.; Flexner, J. L.; Leal-Tamarii, A.; Lewis, C.; Lisa, M.-S.; Nagele, K.; Ozga, A.; Rohrlach, A. B.; Posth, C.; Shing, R.; Spriggs, M.; Willie, E.; Valentin, F.; Warinner, C.\nScore: 13.1, Published: 2023-10-19 DOI: 10.1101/2023.10.18.563027\nThe Pacific islands have experienced multiple waves of human migrations, providing a case study for exploring the potential of using the microbiome to study human migration. We performed a metagenomic study of archaeological dental calculus from 103 ancient individuals, originating from 12 Pacific islands and spanning a time range of [~]3000 years. Oral microbiome DNA preservation in calculus is far higher than that of human DNA in archaeological bone from the Pacific, and comparable to that seen in calculus from temperate regions. Variation in the microbial community composition was minimally driven by time period and geography within the Pacific, while comparison with samples from Europe, Africa, and Asia reveal the microbial communities of Pacific calculus samples to be distinctive. Phylogenies of individual bacterial species in Pacific calculus reflect geography. Archaeological dental calculus shows potential to yield information about past human migrations, complementing studies of the human genome.\nMicrotubule inner proteins of Plasmodium are essential for transmission of malaria parasites\nAuthors: Hentzschel, F.; Binder, A. M.; Dorner, L. P.; Herzel, L.; Nuglish, F.; Sema, M.; Aguirre-Botero, M. C.; Cyrklaff, M.; Funaya, C.; Frischknecht, F.\nScore: 11.2, Published: 2023-10-19 DOI: 10.1101/2023.10.19.562943\nMicrotubule inner proteins, MIPs, are microtubule associated proteins that bind to tubulin from the luminal side. MIPs can be found in axonemes to stabilize flagellar beat or within cytoplasmic microtubules. Plasmodium spp. are the causative agents of malaria that feature different forms across a complex life cycle with both unique and divergent microtubule-based arrays. Here we investigate the role of four MIPs in a rodent malaria parasite for their role in transmission to and from the mosquito. We show by single and double gene deletions that SPM1 and TrxL1, MIPs associated with the subpellicular microtubules are dispensable for transmission from the vertebrate host to the mosquito and back. In contrast, FAP20 and FAP52, MIPs associated with the axonemes of gametes, are essential for transmission to mosquitoes but only if both genes are deleted. In the absence of both, FAP20 and FAP52 the B-tubule of the axoneme partly detaches from the A-tubule resulting in the deficiency of axonemal beating and hence gamete formation and egress. Our data suggest that a high level of redundancy ensures microtubule stability in the transmissive stages of Plasmodium, which is important for parasite transmission.\nGlobal regulation via modulation of ribosome pausing by EttA\nAuthors: Ousalem, F.; Ngo, S.; Oiffer, T.; Omairi-Nasser, A.; Monlezun, L.; Boel, G.\nScore: 10.7, Published: 2023-10-17 DOI: 10.1101/2023.10.17.562674\nHaving multiple rounds of translation of the same mRNA is a key feature of the Central Dogma that creates substantial dynamic complexities along with opportunities for regulation related to ribosome pausing and stalling at specific sequences1-5. The molecular mechanisms controlling these critical processes and the principles guiding their evolution remain among the least understood facets of cellular physiology. We herein use a combination of genetic, genomic, physiological, and biochemical methods to show that regulation of ribosome pausing at specific amino acid sequences can produce [~]2-fold changes in protein expression levels that have a strong influence on cell growth and therefore evolutionary fitness. We demonstrate, both in vivo and in vitro, that the ABCF protein EttA6,7 directly controls the translation of mRNAs coding for a subset of enzymes in the tricarboxylic acid (TCA) cycle and its glyoxylate shunt, which dramatically modulates growth in some chemical environments. It also modulates expression of specific proteins involved in metabolically related physiological and stress-response pathways. These regulatory activities are mediated by EttA rescuing ribosomes paused at specific patterns of negatively charged residues within the first 30 amino acids of nascent proteins. The previously established dependency of EttAs activity on ADP:ATP ratio6,7 can modulate these effects based on cellular energy status. We thus establish a new global regulatory paradigm based on sequence-specific modulation of translational pausing.\nSoil fungi remain active and invest in storage compounds during drought independent of future climate conditions\nAuthors: Canarini, A.; Fuchslueger, L.; Schnecker, J.; Metze, D.; Nelson, D.; Kahmen, A.; Watzka, M.; Poetsch, E. M.; Schaumberger, A.; Bahn, M.; Richter, A.\nScore: 8.3, Published: 2023-10-23 DOI: 10.1101/2023.10.23.563577\nMicrobial growth is central to soil carbon cycling. However, how microbial communities grow under climate change is still largely unexplored. In an experiment simulating future climate conditions (increased atmospheric CO2 and temperature) and drought, we traced 2H or 18O applied via water-vapor exchange into fatty acids or DNA, respectively, allowing to measure community- and group-level adjustments in soil microbial physiology (replication, storage product synthesis, and carbon use efficiency, CUE). We show, that while overall community-level growth decreased by half during drought, fungal growth remained stable demonstrating an astonishing resistance of fungal activity against soil moisture changes. In addition, fungal investment into storage triglycerides increased more than five-fold under drought. CUE (the balance between anabolism and catabolism) was unaffected by drought but decreased in future climate conditions. Our results highlight that accounting for different growth strategies can foster our understanding of soil microbial contribution to C cycling and feedback to climate change.\nAnti-CRISPR proteins trigger a burst of CRISPR-Cas9 expression that enhances phage defense\nAuthors: Workman, R. E.; Stoltzfus, M. J.; Keith, N. C.; Euler, C. W.; Bondy-Denomy, J.; Modell, J. W.\nScore: 8.0, Published: 2023-10-22 DOI: 10.1101/2023.10.22.562561\nCRISPR-Cas immune systems provide bacteria with adaptive immunity against bacteriophages, but they are often transcriptionally downregulated to mitigate autoimmunity. In some cases, CRISPR-Cas expression increases in response to a phage infection, but the mechanisms of induction are largely unknown, and it is unclear whether induction occurs strongly and quickly enough to benefit the bacterial host. In S. pyogenes, Cas9 is both an immune effector and autorepressor of CRISPR-Cas expression. Here, we show that phage-encoded anti-CRISPR proteins relieve Cas9 autorepression and trigger a rapid increase in CRISPR-Cas levels during a single phage infective cycle. As a result, fewer cells succumb to lysis leading to a striking survival benefit after multiple rounds of infection. CRISPR-Cas induction also reduces lysogeny, thereby limiting a route for horizontal gene transfer. Altogether, we show that Cas9 is not only a CRISPR-Cas effector and repressor, but also a phage sensor that can mount an anti-anti-CRISPR transcriptional response.\nPrevalence and diversity of TAL effector-like proteins in fungal endosymbiotic Mycetohabitans spp.\nAuthors: Carpenter, S. C. D.; Bogdanove, A. J.; Stajich, J. E.; Uehling, J.; Lovett, B.; Kasson, M.; Carter, M. E.\nScore: 8.5, Published: 2023-10-16 DOI: 10.1101/2023.10.16.562584\nEndofungal Mycetohabitans (formerly Burkholderia) spp. rely on a type III secretion system to deliver mostly unidentified effector proteins when colonizing their host fungus, Rhizopus microsporus. The one known secreted effector family from Mycetohabitans consists of homologs of transcription activator-like (TAL) effectors, which are used by plant pathogenic Xanthomonas and Ralstonia spp. to activate host genes that promote disease. These Burkholderia TAL-like (Btl) proteins bind corresponding specific DNA sequences in a predictable manner, but their impact on transcription and their genomic target(s) in the fungus are not yet known. Recent characterization of two Btl proteins (Btl19-13 and MTAL1/Btl21-1), each from a different Mycetohabitans species, revealed different phenotypes in Rhizopus, underscoring the need to assess the sequence and functional diversity of Btl proteins. We sequenced and assembled nine Mycetohabitans spp. genomes using long-read PacBio technology. All assemblies contained fragments of btl genes, and most had intact copies. We then mined fungal-bacterial metagenomes assembled as part of the ZygoLife project. This analysis showed that btl genes are present across diverse Mycetohabitans strains from Mucoromycota fungal hosts yet vary in sequences and predicted DNA binding specificity. Phylogenetic analysis revealed distinct clades of Btl proteins and suggested that Mycetohabitans might contain more species than previously recognized. Within our data set, Btl proteins were more conserved across Mycetohabitans rhizoxinica strains than across Mycetohabitans endofungorum, but there was also evidence of greater overall strain diversity within the latter clade. Overall, the results suggest that Btl proteins contribute to bacterial-fungal symbioses in myriad ways. ImportanceMany Mucoromycota fungi harbor endosymbiotic bacteria, including Rhizopus spp. that are food fermenters and pathogens of plants and immunocompromised people. Rhizopus microsporus has endofungal Mycetohabitans (formerly Burkholderia) spp. that deploy proteins related to DNA-binding transcription activator-like effectors of plant pathogens, which enter plant nuclei and activate disease susceptibility genes. One Burkholderia TAL-like (Btl) protein, in M. endofungorum, enhances fungal membrane stress tolerance. Another, from M. rhizoxinica, promotes bacterial colonization of the fungus. By sequencing isolated bacteria and mining fungal holobiont sequences, we found Btl proteins in diverse Mycetohabitans strains, varying in DNA binding specificity, thus in potential host targets. Btl proteins were more conserved within M. rhizoxinica suggesting distinctions among the two named species. The results suggest that Btl proteins contribute to symbiosis in diverse ways, providing insight into effector evolution and arguing for functional characterization of additional Btl proteins to understand establishment and maintenance of these important fungal-bacterial interactions.\nTransertion and cell geometry organize the Escherichia coli nucleoid during rapid growth\nAuthors: Spahn, C.; Middlemiss, S.; Gomez-de-Mariscal, E.; Henriques, R.; Bode, H. B.; Holden, S.; Heilemann, M.\nScore: 27.5, Published: 2023-10-19 DOI: 10.1101/2023.10.16.562172\nAO_SCPLOWBSTRACTC_SCPLOWBacterial chromosomes are spatiotemporally organized and sensitive to environmental changes. However, the mechanisms underlying chromosome configuration and reorganization are currently not fully understood. Using 3D single-molecule localization microscopy and live-cell imaging, we show that the Escherichia coli nucleoid adopts a condensed, membrane-associated configuration during rapid growth. To study the influence of different biosynthetic processes on nucleoid morphology and positioning, we recorded multi-colour super-resolution images during drug treatment. After developing analysis routines for confocal and super-resolution images, we captured highly resolved snapshots which revealed the complete loss of the membrane-bound state of the nucleoid within 10 minutes of halting transcription and translation. This indicates an active role of transertion (coupled transcription, translation and membrane insertion) in nucleoid organization. In contrast, cell wall synthesis inhibition only affects nucleoid organization during morphological changes. Further, we provide evidence that the E. coli nucleoid spatially correlates with MreB in unperturbed E. coli cells, while this correlation diminishes in cells with changed cell geometry or upon inhibition of protein biosynthesis. Replication inhibition experiments, as well as multi-drug treatments highlight the role of entropic effects and transcription in nucleoid condensation and positioning. In summary, we provide experimental evidence for transertion as a principal organiser of the bacterial nucleoid, and show that an altered metabolic state and antibiotic treatment both lead to major changes in the degree of transertion and overall spatial organization of the nucleoid. Our high-resolution characterization reveals dynamics of antibiotic action and provides tools to quantify bacterial chromosome organization. This does not only provide valuable insights into the role of transertion, but can also be applied to study other cell-biological processes.\n",
  "wordCount" : "2502",
  "inLanguage": "en",
  "datePublished": "2023-10-25T10:39:04Z",
  "dateModified": "2023-10-25T10:39:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta"><span>updated on October 25, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.19.563209">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.19.563209" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.19.563209">
        <p class="paperTitle">Virological characteristics of the SARS-CoV-2 Omicron EG.5.1 variant</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.19.563209" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.19.563209" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tsujino, S.; Deguchi, S.; Nomai, T.; Padilla-Blanco, M.; Plianchaisuk, A.; Wang, L.; Begum, M. M.; Uriu, K.; Mizuma, K.; Nao, N.; Kojima, I.; Tsubo, T.; Li, J.; Matsumura, Y.; Nagao, M.; Oda, Y.; Tsuda, M.; Anraku, Y.; Kita, S.; Yajima, H.; Tabata, K.; Guo, Z.; Hinay, A. A.; Yoshimatsu, K.; Yamamoto, Y.; Nagamoto, Y.; Asakura, H.; Nagashima, M.; Sadamasu, K.; Yoshimura, K.; Nasser, H.; Jonathan, M.; Putri, O.; Kim, Y.; Chen, L.; Suzuki, R.; Tamura, T.; Maenaka, K.; The Genotype to Phenotype Japan (G2P-Japan) Consortium,  ; Irie, T.; Matsuno, K.; Tanaka, S.; Ito, J.; Ikeda, T.; Takayama, K.; Za</p>
        <p class="info">Score: 146.9, Published: 2023-10-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.19.563209' target='https://doi.org/10.1101/2023.10.19.563209'> 10.1101/2023.10.19.563209</a></p>
        <p class="abstract">In middle-late 2023, a sublineage of SARS-CoV-2 Omicron XBB, EG.5.1 (a progeny of XBB.1.9.2), is spreading rapidly around the world. Here, we performed multiscale investigations to reveal virological features of newly emerging EG.5.1 variant. Our phylogenetic-epidemic dynamics modeling suggested that two hallmark substitutions of EG.5.1, S:F456L and ORF9b:I5T, are critical to the increased viral fitness. Experimental investigations addressing the growth kinetics, sensitivity to clinically available antivirals, fusogenicity and pathogenicity of EG.5.1 suggested that the virological features of EG.5.1 is comparable to that of XBB.1.5. However, the cryo-electron microscopy reveals the structural difference between the spike proteins of EG.5.1 and XBB.1.5. We further assessed the impact of ORF9b:I5T on viral features, but it was almost negligible at least in our experimental setup. Our multiscale investigations provide the knowledge for understanding of the evolution trait of newly emerging pathogenic viruses in the human population.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562707">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562707" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562707">
        <p class="paperTitle">KatG catalase deficiency confers bedaquiline hyper-susceptibility to isoniazid resistant Mycobacterium tuberculosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562707" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562707" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ofori-Anyinam, N.; Hamblin, M.; Coldren, M. L.; Li, B.; Mereddy, G.; Shaikh, M.; Shah, A.; Ranu, N.; Lu, S.; Blainey, P. C.; Ma, S.; Collins, J. J.; Yang, J. H.</p>
        <p class="info">Score: 15.6, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562707' target='https://doi.org/10.1101/2023.10.17.562707'> 10.1101/2023.10.17.562707</a></p>
        <p class="abstract">Multidrug-resistant tuberculosis (MDR-TB) is a growing source of global mortality and threatens global control of tuberculosis (TB) disease. The diarylquinoline bedaquiline (BDQ) recently emerged as a highly efficacious drug against MDR-TB, defined as resistance to the first-line drugs isoniazid (INH) and rifampin. INH resistance is primarily caused by loss-of-function mutations in the catalase KatG, but mechanisms underlying BDQs efficacy against MDR-TB remain unknown. Here we employ a systems biology approach to investigate BDQ hyper-susceptibility in INH-resistant Mycobacterium tuberculosis. We found hyper-susceptibility to BDQ in INH-resistant cells is due to several physiological changes induced by KatG deficiency, including increased susceptibility to reactive oxygen species and DNA damage, remodeling of transcriptional programs, and metabolic repression of folate biosynthesis. We demonstrate BDQ hyper-susceptibility is common in INH-resistant clinical isolates. Collectively, these results highlight how altered bacterial physiology can impact drug efficacy in drug-resistant bacteria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562728">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562728" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562728">
        <p class="paperTitle">Phages Produce Persisters</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562728" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562728" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fernandez-Garcia, L.; Kirigo, J.; Huelgas-Mendez, D.; Tomas, M.; Garcia-Contreras, R.; Wood, T. K.</p>
        <p class="info">Score: 15.3, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562728' target='https://doi.org/10.1101/2023.10.17.562728'> 10.1101/2023.10.17.562728</a></p>
        <p class="abstract">Bacteria primarily encounter stress, and, arguably, their greatest threats are phages. It is often assumed that those bacteria that escape phage attack have mutated; however, another possibility is that a subpopulation forms the dormant persister state, in a manner similar to that demonstrated for bacterial cells undergoing nutritive, oxidative, and antibiotic stress. Persister cells do not undergo mutation and survive lethal conditions by ceasing growth transiently. Slower growth and dormancy play a key physiological role as they allow host phage defense systems more time to clear the phage infection. Here we investigated how bacteria survive lytic phage infection by isolating surviving cells from the plaques of T2, T4, and lambda (cI mutant) virulent phages. We found that bacteria in plaques can escape phage attack both by mutation (i.e., become resistant) and without mutation (i.e., become persistent). Specifically, whereas T4-resistant and lambda-resistant bacteria with over a 100,000-fold less sensitivity were isolated from plaques with obvious genetic mutations (e.g., causing mucoidy), cells were also found after T2 infection that undergo no significant mutation, retain wild-type phage sensitivity, and survive lethal doses of antibiotics. Corroborating this, adding T2 phage to persister cells resulted in 137,000-fold more survival compared to that of addition to exponentially-growing cells. Phage treatments with Klebsiella pneumonia and Pseudomonas aeruginosa also generated persister cells. Hence, along with resistant strains, bacteria also form persister cells during phage infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.18.563027">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.18.563027" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.18.563027">
        <p class="paperTitle">Exploring archaeogenetic studies of dental calculus to shed light on past human migrations in Oceania</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.18.563027" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.18.563027" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Velsko, I. M.; Fagernas, Z.; Tromp, M.; Bedford, S.; Buckley, H.; Clark, G.; Dudgeon, J.; Flexner, J. L.; Leal-Tamarii, A.; Lewis, C.; Lisa, M.-S.; Nagele, K.; Ozga, A.; Rohrlach, A. B.; Posth, C.; Shing, R.; Spriggs, M.; Willie, E.; Valentin, F.; Warinner, C.</p>
        <p class="info">Score: 13.1, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.18.563027' target='https://doi.org/10.1101/2023.10.18.563027'> 10.1101/2023.10.18.563027</a></p>
        <p class="abstract">The Pacific islands have experienced multiple waves of human migrations, providing a case study for exploring the potential of using the microbiome to study human migration. We performed a metagenomic study of archaeological dental calculus from 103 ancient individuals, originating from 12 Pacific islands and spanning a time range of [~]3000 years. Oral microbiome DNA preservation in calculus is far higher than that of human DNA in archaeological bone from the Pacific, and comparable to that seen in calculus from temperate regions. Variation in the microbial community composition was minimally driven by time period and geography within the Pacific, while comparison with samples from Europe, Africa, and Asia reveal the microbial communities of Pacific calculus samples to be distinctive. Phylogenies of individual bacterial species in Pacific calculus reflect geography. Archaeological dental calculus shows potential to yield information about past human migrations, complementing studies of the human genome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.19.562943">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.19.562943" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.19.562943">
        <p class="paperTitle">Microtubule inner proteins of Plasmodium are essential for transmission of malaria parasites</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.19.562943" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.19.562943" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hentzschel, F.; Binder, A. M.; Dorner, L. P.; Herzel, L.; Nuglish, F.; Sema, M.; Aguirre-Botero, M. C.; Cyrklaff, M.; Funaya, C.; Frischknecht, F.</p>
        <p class="info">Score: 11.2, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.19.562943' target='https://doi.org/10.1101/2023.10.19.562943'> 10.1101/2023.10.19.562943</a></p>
        <p class="abstract">Microtubule inner proteins, MIPs, are microtubule associated proteins that bind to tubulin from the luminal side. MIPs can be found in axonemes to stabilize flagellar beat or within cytoplasmic microtubules. Plasmodium spp. are the causative agents of malaria that feature different forms across a complex life cycle with both unique and divergent microtubule-based arrays. Here we investigate the role of four MIPs in a rodent malaria parasite for their role in transmission to and from the mosquito. We show by single and double gene deletions that SPM1 and TrxL1, MIPs associated with the subpellicular microtubules are dispensable for transmission from the vertebrate host to the mosquito and back. In contrast, FAP20 and FAP52, MIPs associated with the axonemes of gametes, are essential for transmission to mosquitoes but only if both genes are deleted. In the absence of both, FAP20 and FAP52 the B-tubule of the axoneme partly detaches from the A-tubule resulting in the deficiency of axonemal beating and hence gamete formation and egress. Our data suggest that a high level of redundancy ensures microtubule stability in the transmissive stages of Plasmodium, which is important for parasite transmission.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562674">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562674" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562674">
        <p class="paperTitle">Global regulation via modulation of ribosome pausing by EttA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562674" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562674" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ousalem, F.; Ngo, S.; Oiffer, T.; Omairi-Nasser, A.; Monlezun, L.; Boel, G.</p>
        <p class="info">Score: 10.7, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562674' target='https://doi.org/10.1101/2023.10.17.562674'> 10.1101/2023.10.17.562674</a></p>
        <p class="abstract">Having multiple rounds of translation of the same mRNA is a key feature of the Central Dogma that creates substantial dynamic complexities along with opportunities for regulation related to ribosome pausing and stalling at specific sequences1-5. The molecular mechanisms controlling these critical processes and the principles guiding their evolution remain among the least understood facets of cellular physiology. We herein use a combination of genetic, genomic, physiological, and biochemical methods to show that regulation of ribosome pausing at specific amino acid sequences can produce [~]2-fold changes in protein expression levels that have a strong influence on cell growth and therefore evolutionary fitness. We demonstrate, both in vivo and in vitro, that the ABCF protein EttA6,7 directly controls the translation of mRNAs coding for a subset of enzymes in the tricarboxylic acid (TCA) cycle and its glyoxylate shunt, which dramatically modulates growth in some chemical environments. It also modulates expression of specific proteins involved in metabolically related physiological and stress-response pathways. These regulatory activities are mediated by EttA rescuing ribosomes paused at specific patterns of negatively charged residues within the first 30 amino acids of nascent proteins. The previously established dependency of EttAs activity on ADP:ATP ratio6,7 can modulate these effects based on cellular energy status. We thus establish a new global regulatory paradigm based on sequence-specific modulation of translational pausing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563577">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563577" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563577">
        <p class="paperTitle">Soil fungi remain active and invest in storage compounds during drought independent of future climate conditions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563577" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563577" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Canarini, A.; Fuchslueger, L.; Schnecker, J.; Metze, D.; Nelson, D.; Kahmen, A.; Watzka, M.; Poetsch, E. M.; Schaumberger, A.; Bahn, M.; Richter, A.</p>
        <p class="info">Score: 8.3, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563577' target='https://doi.org/10.1101/2023.10.23.563577'> 10.1101/2023.10.23.563577</a></p>
        <p class="abstract">Microbial growth is central to soil carbon cycling. However, how microbial communities grow under climate change is still largely unexplored. In an experiment simulating future climate conditions (increased atmospheric CO2 and temperature) and drought, we traced 2H or 18O applied via water-vapor exchange into fatty acids or DNA, respectively, allowing to measure community- and group-level adjustments in soil microbial physiology (replication, storage product synthesis, and carbon use efficiency, CUE). We show, that while overall community-level growth decreased by half during drought, fungal growth remained stable demonstrating an astonishing resistance of fungal activity against soil moisture changes. In addition, fungal investment into storage triglycerides increased more than five-fold under drought. CUE (the balance between anabolism and catabolism) was unaffected by drought but decreased in future climate conditions. Our results highlight that accounting for different growth strategies can foster our understanding of soil microbial contribution to C cycling and feedback to climate change.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.22.562561">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.22.562561" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.22.562561">
        <p class="paperTitle">Anti-CRISPR proteins trigger a burst of CRISPR-Cas9 expression that enhances phage defense</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.22.562561" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.22.562561" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Workman, R. E.; Stoltzfus, M. J.; Keith, N. C.; Euler, C. W.; Bondy-Denomy, J.; Modell, J. W.</p>
        <p class="info">Score: 8.0, Published: 2023-10-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.22.562561' target='https://doi.org/10.1101/2023.10.22.562561'> 10.1101/2023.10.22.562561</a></p>
        <p class="abstract">CRISPR-Cas immune systems provide bacteria with adaptive immunity against bacteriophages, but they are often transcriptionally downregulated to mitigate autoimmunity. In some cases, CRISPR-Cas expression increases in response to a phage infection, but the mechanisms of induction are largely unknown, and it is unclear whether induction occurs strongly and quickly enough to benefit the bacterial host. In S. pyogenes, Cas9 is both an immune effector and autorepressor of CRISPR-Cas expression. Here, we show that phage-encoded anti-CRISPR proteins relieve Cas9 autorepression and trigger a rapid increase in CRISPR-Cas levels during a single phage infective cycle. As a result, fewer cells succumb to lysis leading to a striking survival benefit after multiple rounds of infection. CRISPR-Cas induction also reduces lysogeny, thereby limiting a route for horizontal gene transfer. Altogether, we show that Cas9 is not only a CRISPR-Cas effector and repressor, but also a phage sensor that can mount an anti-anti-CRISPR transcriptional response.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.16.562584">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.16.562584" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.16.562584">
        <p class="paperTitle">Prevalence and diversity of TAL effector-like proteins in fungal endosymbiotic Mycetohabitans spp.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.16.562584" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.16.562584" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carpenter, S. C. D.; Bogdanove, A. J.; Stajich, J. E.; Uehling, J.; Lovett, B.; Kasson, M.; Carter, M. E.</p>
        <p class="info">Score: 8.5, Published: 2023-10-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.16.562584' target='https://doi.org/10.1101/2023.10.16.562584'> 10.1101/2023.10.16.562584</a></p>
        <p class="abstract">Endofungal Mycetohabitans (formerly Burkholderia) spp. rely on a type III secretion system to deliver mostly unidentified effector proteins when colonizing their host fungus, Rhizopus microsporus. The one known secreted effector family from Mycetohabitans consists of homologs of transcription activator-like (TAL) effectors, which are used by plant pathogenic Xanthomonas and Ralstonia spp. to activate host genes that promote disease. These  Burkholderia TAL-like (Btl) proteins bind corresponding specific DNA sequences in a predictable manner, but their impact on transcription and their genomic target(s) in the fungus are not yet known. Recent characterization of two Btl proteins (Btl19-13 and MTAL1/Btl21-1), each from a different Mycetohabitans species, revealed different phenotypes in Rhizopus, underscoring the need to assess the sequence and functional diversity of Btl proteins. We sequenced and assembled nine Mycetohabitans spp. genomes using long-read PacBio technology. All assemblies contained fragments of btl genes, and most had intact copies. We then mined fungal-bacterial metagenomes assembled as part of the ZygoLife project. This analysis showed that btl genes are present across diverse Mycetohabitans strains from Mucoromycota fungal hosts yet vary in sequences and predicted DNA binding specificity. Phylogenetic analysis revealed distinct clades of Btl proteins and suggested that Mycetohabitans might contain more species than previously recognized. Within our data set, Btl proteins were more conserved across Mycetohabitans rhizoxinica strains than across Mycetohabitans endofungorum, but there was also evidence of greater overall strain diversity within the latter clade. Overall, the results suggest that Btl proteins contribute to bacterial-fungal symbioses in myriad ways.

ImportanceMany Mucoromycota fungi harbor endosymbiotic bacteria, including Rhizopus spp. that are food fermenters and pathogens of plants and immunocompromised people. Rhizopus microsporus has endofungal Mycetohabitans (formerly Burkholderia) spp. that deploy proteins related to DNA-binding  transcription activator-like effectors of plant pathogens, which enter plant nuclei and activate disease susceptibility genes. One  Burkholderia TAL-like (Btl) protein, in M. endofungorum, enhances fungal membrane stress tolerance. Another, from M. rhizoxinica, promotes bacterial colonization of the fungus. By sequencing isolated bacteria and mining fungal holobiont sequences, we found Btl proteins in diverse Mycetohabitans strains, varying in DNA binding specificity, thus in potential host targets. Btl proteins were more conserved within M. rhizoxinica suggesting distinctions among the two named species. The results suggest that Btl proteins contribute to symbiosis in diverse ways, providing insight into effector evolution and arguing for functional characterization of additional Btl proteins to understand establishment and maintenance of these important fungal-bacterial interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.16.562172">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.16.562172" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.16.562172">
        <p class="paperTitle">Transertion and cell geometry organize the Escherichia coli nucleoid during rapid growth</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.16.562172" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.16.562172" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Spahn, C.; Middlemiss, S.; Gomez-de-Mariscal, E.; Henriques, R.; Bode, H. B.; Holden, S.; Heilemann, M.</p>
        <p class="info">Score: 27.5, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.16.562172' target='https://doi.org/10.1101/2023.10.16.562172'> 10.1101/2023.10.16.562172</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWBacterial chromosomes are spatiotemporally organized and sensitive to environmental changes. However, the mechanisms underlying chromosome configuration and reorganization are currently not fully understood. Using 3D single-molecule localization microscopy and live-cell imaging, we show that the Escherichia coli nucleoid adopts a condensed, membrane-associated configuration during rapid growth. To study the influence of different biosynthetic processes on nucleoid morphology and positioning, we recorded multi-colour super-resolution images during drug treatment. After developing analysis routines for confocal and super-resolution images, we captured highly resolved snapshots which revealed the complete loss of the membrane-bound state of the nucleoid within 10 minutes of halting transcription and translation. This indicates an active role of transertion (coupled transcription, translation and membrane insertion) in nucleoid organization. In contrast, cell wall synthesis inhibition only affects nucleoid organization during morphological changes. Further, we provide evidence that the E. coli nucleoid spatially correlates with MreB in unperturbed E. coli cells, while this correlation diminishes in cells with changed cell geometry or upon inhibition of protein biosynthesis. Replication inhibition experiments, as well as multi-drug treatments highlight the role of entropic effects and transcription in nucleoid condensation and positioning. In summary, we provide experimental evidence for transertion as a principal organiser of the bacterial nucleoid, and show that an altered metabolic state and antibiotic treatment both lead to major changes in the degree of transertion and overall spatial organization of the nucleoid. Our high-resolution characterization reveals dynamics of antibiotic action and provides tools to quantify bacterial chromosome organization. This does not only provide valuable insights into the role of transertion, but can also be applied to study other cell-biological processes.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
