<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Virological characteristics of the SARS-CoV-2 Omicron HK.3 variant harboring the &#34;FLip&#34; substitution
Authors: Kosugi, Y.; Plianchaisuk, A.; Putri, O.; Uriu, K.; Kaku, Y.; Hinay, A. A.; Chen, L.; Kuramochi, J.; Sadamasu, K.; Yoshimura, K.; Asakura, H.; Nagashima, M.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Ito, J.; Sato, K.
Score: 180.6, Published: 2023-11-15 DOI: 10.1101/2023.11.14.566985
In November 2023, SARS-CoV-2 XBB descendants, including EG.5.1 (XBB.1.9.2.5.1), the currently predominant lineage, are circulating worldwide according to Nextstrain.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Virological characteristics of the SARS-CoV-2 Omicron HK.3 variant harboring the &#34;FLip&#34; substitution
Authors: Kosugi, Y.; Plianchaisuk, A.; Putri, O.; Uriu, K.; Kaku, Y.; Hinay, A. A.; Chen, L.; Kuramochi, J.; Sadamasu, K.; Yoshimura, K.; Asakura, H.; Nagashima, M.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Ito, J.; Sato, K.
Score: 180.6, Published: 2023-11-15 DOI: 10.1101/2023.11.14.566985
In November 2023, SARS-CoV-2 XBB descendants, including EG.5.1 (XBB.1.9.2.5.1), the currently predominant lineage, are circulating worldwide according to Nextstrain." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-19T10:38:34+00:00" />
<meta property="article:modified_time" content="2023-11-19T10:38:34+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Virological characteristics of the SARS-CoV-2 Omicron HK.3 variant harboring the &#34;FLip&#34; substitution
Authors: Kosugi, Y.; Plianchaisuk, A.; Putri, O.; Uriu, K.; Kaku, Y.; Hinay, A. A.; Chen, L.; Kuramochi, J.; Sadamasu, K.; Yoshimura, K.; Asakura, H.; Nagashima, M.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Ito, J.; Sato, K.
Score: 180.6, Published: 2023-11-15 DOI: 10.1101/2023.11.14.566985
In November 2023, SARS-CoV-2 XBB descendants, including EG.5.1 (XBB.1.9.2.5.1), the currently predominant lineage, are circulating worldwide according to Nextstrain."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Virological characteristics of the SARS-CoV-2 Omicron HK.3 variant harboring the \u0026#34;FLip\u0026#34; substitution\nAuthors: Kosugi, Y.; Plianchaisuk, A.; Putri, O.; Uriu, K.; Kaku, Y.; Hinay, A. A.; Chen, L.; Kuramochi, J.; Sadamasu, K.; Yoshimura, K.; Asakura, H.; Nagashima, M.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Ito, J.; Sato, K.\nScore: 180.6, Published: 2023-11-15 DOI: 10.1101/2023.11.14.566985\nIn November 2023, SARS-CoV-2 XBB descendants, including EG.5.1 (XBB.1.9.2.5.1), the currently predominant lineage, are circulating worldwide according to Nextstrain.",
  "keywords": [
    
  ],
  "articleBody": " Virological characteristics of the SARS-CoV-2 Omicron HK.3 variant harboring the \"FLip\" substitution\nAuthors: Kosugi, Y.; Plianchaisuk, A.; Putri, O.; Uriu, K.; Kaku, Y.; Hinay, A. A.; Chen, L.; Kuramochi, J.; Sadamasu, K.; Yoshimura, K.; Asakura, H.; Nagashima, M.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Ito, J.; Sato, K.\nScore: 180.6, Published: 2023-11-15 DOI: 10.1101/2023.11.14.566985\nIn November 2023, SARS-CoV-2 XBB descendants, including EG.5.1 (XBB.1.9.2.5.1), the currently predominant lineage, are circulating worldwide according to Nextstrain. EG.5.1 has a characteristic amino acid substitution in the spike protein (S), S:F456L, which contributes to its escape from humoral immunity. EG.5.1 has further evolved, and its descendant lineage harboring S:L455F (i.e., EG.5.1+S:L455F) emerged and was named HK.3 (XBB.1.9.2.5.1.1.3). HK.3 was initially discovered in East Asia and is rapidly spreading worldwide. Notably, the XBB subvariants bearing both S:L455F and S:F456L substitutions, including HK.3, are called the \"FLip\" variants. These FLip variants, such as JG.3 (XBB.1.9.2.5.1.3.3), JF.1 (XBB.1.16.6.1) and GK.3 (XBB.1.5.70.3), have emerged convergently, suggesting that the acquisition of these two substitutions confers a growth advantage to XBB in the human population. Here, we investigated the virological properties of HK.3 as a representative of the FLip variants.\nEmergence and spread of feline infectious peritonitis due to a highly pathogenic canine/feline recombinant coronavirus\nAuthors: Atippa, C.; Warr, A. S.; Epaminondas, D.; O'Shea, M.; Fletcher, S. L.; Malbon, A.; Lyraki, M.; Hammond, R.; Hardas, A.; Zanti, A.; Loukaidou, S.; Gentil, M.; Gunn-Moore, D.; Mazeri, S.; Tait-Burkard, C.\nScore: 206.7, Published: 2023-11-10 DOI: 10.1101/2023.11.08.566182\nCross-species transmission of coronaviruses (CoVs) poses a serious threat to both animal and human health1-3. Whilst the large RNA genome of CoVs shows relatively low mutation rates, recombination within genera is frequently observed and demonstrated4-7. Companion animals are often overlooked in the transmission cycle of viral diseases; however, the close relationship of feline (FCoV) and canine CoV (CCoV) to human hCoV-229E5,8, as well as their susceptibility to SARS-CoV-29 highlight their importance in potential transmission cycles. Whilst recombination between CCoV and FCoV of a large fragment spanning orf1b to M has been previously described5,10, here we report the emergence of a novel, highly pathogenic FCoV-CCoV recombinant responsible for a rapidly spreading outbreak of feline infectious peritonitis (FIP), originating in Cyprus11. The recombination, spanning spike, shows 97% sequence identity to the pantropic canine coronavirus CB/05. Infection is spreading fast and infecting cats of all ages. Development of FIP appears rapid and likely non-reliant on biotype switch12. High sequence identity of isolates from cats in different districts of the island is strongly supportive of direct transmission. A deletion and several amino acid changes in spike, particularly the receptor binding domain, compared to other FCoV-2s, indicate changes to receptor binding and likely cell tropism.\nHuman microbiota is a reservoir of SARS-CoV-2 advantageous mutations\nAuthors: Cao, B.; Wang, X.; Yin, W.; Gao, Z.; Xia, B.\nScore: 68.2, Published: 2023-11-17 DOI: 10.1101/2023.11.16.567485\nSARS-CoV-2 mutations are rapidly emerging, in particular advantageous mutations in the spike (S) protein, which either increase transmissibility or lead to immune escape, are posing a major challenge to pandemic prevention and treatment. However, how the virus acquires a high number of advantageous mutations in a short time remains a mystery. Here, we show that the human microbiota may contribute to mutations in variants of concern (VOCs). We identified a mutation and adjacent 6 amino acids (aa) in a viral mutation fragment (VMF) and searched for homologous fragments (HFs) in the National Center for Biotechnology Information (NCBI) database. Among the approximate 8000 HFs obtained, 61 mutations in S and other outer membrane proteins were found in bacteria, accounting for 62% of all mutation sources, which is a 12-fold higher than the natural variable proportion. Approximately 70% of these bacterial species belong to the human microbiota, are primarily distributed in the gut or lung and exhibit a composition pattern similar to that of COVID-19 patients. Importantly, SARS-CoV-2 RNA-dependent RNA polymerase (RdRp) replicates corresponding bacterial mRNAs harboring mutations, producing chimeric RNAs. Collectively, SARS-CoV-2 may acquire mutations from the human microbiota, resulting in alterations in the binding sites or antigenic determinants of the original virus. Our study sheds light on the evolving mutational mechanisms of SARS-CoV-2.\nFull-spike deep mutational scanning helps predict the evolutionary success of SARS-CoV-2 clades\nAuthors: Dadonaite, B.; Brown, J.; McMahon, T. E.; Farrell, A. G.; Asarnow, D.; Stewart, C.; Logue, J.; Murrell, B.; Chu, H. Y.; Veesler, D.; Bloom, J. D.\nScore: 60.7, Published: 2023-11-14 DOI: 10.1101/2023.11.13.566961\nSARS-CoV-2 variants acquire mutations in spike that promote immune evasion and impact other properties that contribute to viral fitness such as ACE2 receptor binding and cell entry. Knowledge of how mutations affect these spike phenotypes can provide insight into the current and potential future evolution of the virus. Here we use pseudovirus deep mutational scanning to measure how \u003e9,000 mutations across the full XBB.1.5 and BA.2 spikes affect ACE2 binding, cell entry, or escape from human sera. We find that mutations outside the receptor-binding domain (RBD) have meaningfully impacted ACE2 binding during SARS-CoV-2 evolution. We also measure how mutations to the XBB.1.5 spike affect neutralization by serum from individuals who recently had SARS-CoV-2 infections. The strongest serum escape mutations are in the RBD at sites 357, 420, 440, 456, and 473--however, the antigenic impacts of these mutations vary across individuals. We also identify strong escape mutations outside the RBD; however many of them decrease ACE2 binding, suggesting they act by modulating RBD conformation. Notably, the growth rates of human SARS-CoV-2 clades can be explained in substantial part by the measured effects of mutations on spike phenotypes, suggesting our data could enable better prediction of viral evolution.\nThe SARS-CoV-2 Omicron sub-variant BA.2.86 is attenuated in hamsters\nAuthors: Herder, V.; Mendonca, D. C.; Upfold, N.; Furnon, W.; Kerr, K.; Ilia, G.; Allan, J.; Sigal, A.; Patel, A. H.; Palmarini, M.\nScore: 58.7, Published: 2023-11-12 DOI: 10.1101/2023.11.10.566576\nSARS-CoV-2 variants have emerged throughout the COVID-19 pandemic. There is a need to risk-assess newly emerged variants in near \"real-time\" to estimate their potential threat to public health. The recently emerged Omicron sub-variant BA.2.86 raised concerns as it carries a high number of mutations compared to its predecessors. Here, we assessed the virulence of BA.2.86 in hamsters. We compared the pathogenesis of BA.2.86 and BA.2.75, as the latter is one of the most virulent Omicron sub-variants in this animal model. Using digital pathology pipelines, we quantified the extent of pulmonary lesions measuring T cell and macrophage infiltrates, in addition to alveolar epithelial hyperplasia. We also assessed body weight loss, clinical symptoms, virus load in oropharyngeal swabs, and virus replication in the respiratory tract. Our data show that BA.2.86 displays an attenuated phenotype in hamsters, suggesting that it poses no greater risk to public health than its parental Omicron sub-variants. Article summary lineThe newly emerged Omicron sub-variant BA.2.86 is attenuated in hamsters.\nMicrobeMod: A computational toolkit for identifying prokaryotic methylation and restriction-modification with nanopore sequencing\nAuthors: Crits-Christoph, A.; Kang, S. C.; Lee, H. H.; Ostrov, N.\nScore: 33.4, Published: 2023-11-14 DOI: 10.1101/2023.11.13.566931\nBacteria and archaea use restriction-modification (R-M) systems to distinguish self from foreign DNA by methylating their genomes with DNA methyltransferases with diverse sequence specificities, and these immunity systems often vary at the strain level. Identifying active methylation patterns and R-M systems can reveal barriers to the introduction of recombinant DNA or phage infection. Here, we present the computational MicrobeMod toolkit for identifying 5mC and 6mA methylation sequence motifs and R-M systems in bacterial genomes using nanopore sequencing of native DNA. We benchmark this approach on a set of reference E. coli strains expressing methyltransferases with known specificities. We then applied these analyses to 31 diverse bacterial and archaeal organisms to reveal the methylation patterns of strains with previously unexplored epigenetics, finding that prokaryotic 5-methylcytosine may be more common than previously reported. In summary, MicrobeMod can rapidly reveal new epigenetics within a prokaryotic genome sequenced with Oxford Nanopore R10.4.1 flow cells at sequencing depths as low as 10x and only requires native DNA. This toolkit can be used to advance fundamental knowledge of bacterial methylation and guide strategies to overcome R-M barriers of genetic tractability in non-model microbes.\nAn acidophilic fungus is integral to prey digestion in a carnivorous plant\nAuthors: Sun, P.-F.; Lu, M. R.; Liu, Y.-C.; Lin, Y.-f.; Hoh, D. Z.; Ke, H.-M.; Wang, I.-F.; Lu, M.-Y. J.; Kirschner, R.; Lin, Y.-C. J.; Chen, Y.-L.; Tsai, I. J.\nScore: 88.1, Published: 2023-11-08 DOI: 10.1101/2023.11.07.566145\nCarnivorous plant leaves, such as those of the spoon-leaved sundew Drosera spatulata, secrete mucilage which hosts microorganisms potentially aiding in prey digestion. We characterised the mucilage microbial communities and identified the acidophilic fungus Acrodontium crateriforme as the ecologically dominant species. The fungus grows and sporulates on sundew glands as its preferred acidic environment. We show that the A. crateriforme has a reduced genome similar to that of other symbiotic fungi. Based on the transcriptomes when encountering prey insects, we revealed a high degree of genes co-option in each species during fungus-plant coexistence and digestion. Expression patterns of the holobiont during digestion further revealed synergistic effects in several gene families including fungal aspartic and sedolisin peptidases, facilitating the digestion of sundews prey, as well as transporters and dose-dependent responses in plant genes involved in jasmonate signalling pathway. This study establishes that botanical carnivory is defined by multidimensional adaptations correlated with interspecies interactions.\nEvolution of a small phage protein confers resistance to antiphage defense in Enterococcus faecalis\nAuthors: Johnson, C. N.; Bullen, N. P.; Andersen, S. E.; Arya, G.; Marotta, S. R.; Whitney, J. C.; Duerkop, B. A.\nScore: 23.9, Published: 2023-11-16 DOI: 10.1101/2023.11.16.567456\nThe prevalence of multidrug resistant (MDR) bacterial infections continues to rise as the development of new antibiotics needed to combat these infections remains stagnant. MDR enterococci, which are a common cause of hospital-acquired infections, are emerging as one of the major contributors to this crisis. A potential therapeutic approach for combating MDR enterococci is bacteriophage (phage) therapy, which entails the use of lytic viruses to infect and kill pathogenic bacteria. While phages that lyse some strains of MDR enterococci have been identified, other strains display high levels of phage resistance and the mechanisms underlying this resistance are unknown. Here, we use a CRISPR interference (CRISPRi) screen to identify a genetic locus found on a mobilizable plasmid from vancomycin-resistant Enterococcus faecalis involved in phage resistance. This locus encodes a putative serine recombinase followed by a Type IV restriction enzyme (TIV-RE) and we show that this enzyme is sufficient to restrict the replication of the lytic phage in E. faecalis. We further find that phages can evolve to overcome restriction by acquiring a missense mutation in a novel TIV-RE inhibitor protein encoded by many enterococcal phages. We show that this inhibitor, which we have named anti-restriction-factor A (arfA), directly binds to and inactivates diverse TIV-REs. Overall, our findings significantly advance our understanding of phage defense in drug-resistant E. faecalis and provide mechanistic insight into how phages can evolve to overcome antiphage defense systems.\nThe Parkinson's drug entacapone disrupts gut microbiome homeostasis via iron sequestration\nAuthors: Pereira, F. C.; Ge, X.; Kristensen, J. M.; Kirkegaard, R. H.; Maritsch, K.; Zu, Y.; Decorte, M.; Hausmann, B.; Berry, D.; Wasmund, K.; Schintlmeister, A.; Boettcher, T.; Cheng, J.-X.; Wagner, M.\nScore: 21.4, Published: 2023-11-13 DOI: 10.1101/2023.11.12.566429\nIncreasing evidence shows that many human-targeted drugs alter the gut microbiome, leading to implications for host health. However, much less is known about the mechanisms by which drugs target the microbiome and how drugs affect microbial function. Here we combined quantitative microbiome profiling, long-read metagenomics, stable isotope probing and single cell chemical imaging to investigate the impact of two widely prescribed nervous system targeted drugs on the gut microbiome. Ex vivo supplementation of physiologically relevant concentrations of entacapone or loxapine succinate to faecal samples significantly impacted the abundance of up to one third of the microbial species present. Importantly, we demonstrate that the impact of these drugs on microbial metabolism is much more pronounced than their impact on abundances, with low concentrations of drugs reducing the activity, but not the abundance of key microbiome members like Bacteroides, Ruminococcus or Clostridium species. We further demonstrate that entacapone impacts the microbiome due to its ability to complex and deplete available iron, and that microbial growth can be rescued by replenishing levels of microbiota-accessible iron. Remarkably, entacapone-induced iron starvation selected for iron-scavenging organisms carrying antimicrobial resistance and virulence genes. Collectively, our study unveils the impact of two under-investigated drugs on whole microbiomes and identifies metal sequestration as a mechanism of drug-induced microbiome disturbance.\nChromosomal Integrons are Genetically and Functionally Isolated Units of Genomes\nAuthors: Blanco, P.; Trigo da Roza, F.; Toribio-Celestino, L.; Garcia-Pastor, L.; Caselli, N.; Ojeda, F.; Darracq, B.; Vergara, E.; San Millan, A.; Skovgaard, O.; Mazel, D.; Loot, C.; Escudero, J. A.\nScore: 18.6, Published: 2023-11-17 DOI: 10.1101/2023.11.17.567518\nIntegrons are genetic elements that increase the evolvability of bacteria by capturing new genes and stockpiling them in arrays. Sedentary chromosomal integrons(SCIs), can be massive and highly stabilized structures encoding hundreds of genes, whose function remains generally unknown. SCIs have co-evolved with the host for aeons and are highly intertwined with their physiology from a mechanistic point of view. But, paradoxically, other aspects, like their variable content and location within the genome, suggest a high genetic and functional independence. In this work, we have explored the connection of SCIs to their host genome using as a model the Superintegron (SI), a 179-cassette long SCI in the genome of Vibrio cholerae N16961. We have relocated and deleted the SI using SeqDelTA, a novel method that allows to counteract the strong stabilization conferred by toxin-antitoxin systems within the array. We have characterized in depth the impact in V. cholerae's physiology, measuring fitness, chromosome replication dynamics, persistence, transcriptomics, phenomics and virulence. The deletion of the SI did not produce detectable effects in any condition, proving that -despite millions of years of co-evolution-, SCIs are genetically and functionally isolated units of genomes.\n",
  "wordCount" : "2306",
  "inLanguage": "en",
  "datePublished": "2023-11-19T10:38:34Z",
  "dateModified": "2023-11-19T10:38:34Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 19, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.14.566985">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.14.566985" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.14.566985">
        <p class="paperTitle">Virological characteristics of the SARS-CoV-2 Omicron HK.3 variant harboring the &#34;FLip&#34; substitution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.14.566985" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.14.566985" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kosugi, Y.; Plianchaisuk, A.; Putri, O.; Uriu, K.; Kaku, Y.; Hinay, A. A.; Chen, L.; Kuramochi, J.; Sadamasu, K.; Yoshimura, K.; Asakura, H.; Nagashima, M.; The Genotype to Phenotype Japan (G2P-Japan) Consortium,  ; Ito, J.; Sato, K.</p>
        <p class="info">Score: 180.6, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.14.566985' target='https://doi.org/10.1101/2023.11.14.566985'> 10.1101/2023.11.14.566985</a></p>
        <p class="abstract">In November 2023, SARS-CoV-2 XBB descendants, including EG.5.1 (XBB.1.9.2.5.1), the currently predominant lineage, are circulating worldwide according to Nextstrain. EG.5.1 has a characteristic amino acid substitution in the spike protein (S), S:F456L, which contributes to its escape from humoral immunity. EG.5.1 has further evolved, and its descendant lineage harboring S:L455F (i.e., EG.5.1&#43;S:L455F) emerged and was named HK.3 (XBB.1.9.2.5.1.1.3). HK.3 was initially discovered in East Asia and is rapidly spreading worldwide. Notably, the XBB subvariants bearing both S:L455F and S:F456L substitutions, including HK.3, are called the &#34;FLip&#34; variants. These FLip variants, such as JG.3 (XBB.1.9.2.5.1.3.3), JF.1 (XBB.1.16.6.1) and GK.3 (XBB.1.5.70.3), have emerged convergently, suggesting that the acquisition of these two substitutions confers a growth advantage to XBB in the human population. Here, we investigated the virological properties of HK.3 as a representative of the FLip variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.08.566182">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.08.566182" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.08.566182">
        <p class="paperTitle">Emergence and spread of feline infectious peritonitis due to a highly pathogenic canine/feline recombinant coronavirus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.08.566182" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.08.566182" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Atippa, C.; Warr, A. S.; Epaminondas, D.; O&#39;Shea, M.; Fletcher, S. L.; Malbon, A.; Lyraki, M.; Hammond, R.; Hardas, A.; Zanti, A.; Loukaidou, S.; Gentil, M.; Gunn-Moore, D.; Mazeri, S.; Tait-Burkard, C.</p>
        <p class="info">Score: 206.7, Published: 2023-11-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.08.566182' target='https://doi.org/10.1101/2023.11.08.566182'> 10.1101/2023.11.08.566182</a></p>
        <p class="abstract">Cross-species transmission of coronaviruses (CoVs) poses a serious threat to both animal and human health1-3. Whilst the large RNA genome of CoVs shows relatively low mutation rates, recombination within genera is frequently observed and demonstrated4-7. Companion animals are often overlooked in the transmission cycle of viral diseases; however, the close relationship of feline (FCoV) and canine CoV (CCoV) to human hCoV-229E5,8, as well as their susceptibility to SARS-CoV-29 highlight their importance in potential transmission cycles. Whilst recombination between CCoV and FCoV of a large fragment spanning orf1b to M has been previously described5,10, here we report the emergence of a novel, highly pathogenic FCoV-CCoV recombinant responsible for a rapidly spreading outbreak of feline infectious peritonitis (FIP), originating in Cyprus11. The recombination, spanning spike, shows 97% sequence identity to the pantropic canine coronavirus CB/05. Infection is spreading fast and infecting cats of all ages. Development of FIP appears rapid and likely non-reliant on biotype switch12. High sequence identity of isolates from cats in different districts of the island is strongly supportive of direct transmission. A deletion and several amino acid changes in spike, particularly the receptor binding domain, compared to other FCoV-2s, indicate changes to receptor binding and likely cell tropism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.567485">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.567485" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.567485">
        <p class="paperTitle">Human microbiota is a reservoir of SARS-CoV-2 advantageous mutations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.567485" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.567485" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cao, B.; Wang, X.; Yin, W.; Gao, Z.; Xia, B.</p>
        <p class="info">Score: 68.2, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.567485' target='https://doi.org/10.1101/2023.11.16.567485'> 10.1101/2023.11.16.567485</a></p>
        <p class="abstract">SARS-CoV-2 mutations are rapidly emerging, in particular advantageous mutations in the spike (S) protein, which either increase transmissibility or lead to immune escape, are posing a major challenge to pandemic prevention and treatment. However, how the virus acquires a high number of advantageous mutations in a short time remains a mystery. Here, we show that the human microbiota may contribute to mutations in variants of concern (VOCs). We identified a mutation and adjacent 6 amino acids (aa) in a viral mutation fragment (VMF) and searched for homologous fragments (HFs) in the National Center for Biotechnology Information (NCBI) database. Among the approximate 8000 HFs obtained, 61 mutations in S and other outer membrane proteins were found in bacteria, accounting for 62% of all mutation sources, which is a 12-fold higher than the natural variable proportion. Approximately 70% of these bacterial species belong to the human microbiota, are primarily distributed in the gut or lung and exhibit a composition pattern similar to that of COVID-19 patients. Importantly, SARS-CoV-2 RNA-dependent RNA polymerase (RdRp) replicates corresponding bacterial mRNAs harboring mutations, producing chimeric RNAs. Collectively, SARS-CoV-2 may acquire mutations from the human microbiota, resulting in alterations in the binding sites or antigenic determinants of the original virus. Our study sheds light on the evolving mutational mechanisms of SARS-CoV-2.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566961">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566961" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566961">
        <p class="paperTitle">Full-spike deep mutational scanning helps predict the evolutionary success of SARS-CoV-2 clades</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566961" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566961" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dadonaite, B.; Brown, J.; McMahon, T. E.; Farrell, A. G.; Asarnow, D.; Stewart, C.; Logue, J.; Murrell, B.; Chu, H. Y.; Veesler, D.; Bloom, J. D.</p>
        <p class="info">Score: 60.7, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566961' target='https://doi.org/10.1101/2023.11.13.566961'> 10.1101/2023.11.13.566961</a></p>
        <p class="abstract">SARS-CoV-2 variants acquire mutations in spike that promote immune evasion and impact other properties that contribute to viral fitness such as ACE2 receptor binding and cell entry. Knowledge of how mutations affect these spike phenotypes can provide insight into the current and potential future evolution of the virus. Here we use pseudovirus deep mutational scanning to measure how &gt;9,000 mutations across the full XBB.1.5 and BA.2 spikes affect ACE2 binding, cell entry, or escape from human sera. We find that mutations outside the receptor-binding domain (RBD) have meaningfully impacted ACE2 binding during SARS-CoV-2 evolution. We also measure how mutations to the XBB.1.5 spike affect neutralization by serum from individuals who recently had SARS-CoV-2 infections. The strongest serum escape mutations are in the RBD at sites 357, 420, 440, 456, and 473--however, the antigenic impacts of these mutations vary across individuals. We also identify strong escape mutations outside the RBD; however many of them decrease ACE2 binding, suggesting they act by modulating RBD conformation. Notably, the growth rates of human SARS-CoV-2 clades can be explained in substantial part by the measured effects of mutations on spike phenotypes, suggesting our data could enable better prediction of viral evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.566576">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.566576" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.566576">
        <p class="paperTitle">The SARS-CoV-2 Omicron sub-variant BA.2.86 is attenuated in hamsters</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.566576" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.566576" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Herder, V.; Mendonca, D. C.; Upfold, N.; Furnon, W.; Kerr, K.; Ilia, G.; Allan, J.; Sigal, A.; Patel, A. H.; Palmarini, M.</p>
        <p class="info">Score: 58.7, Published: 2023-11-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.566576' target='https://doi.org/10.1101/2023.11.10.566576'> 10.1101/2023.11.10.566576</a></p>
        <p class="abstract">SARS-CoV-2 variants have emerged throughout the COVID-19 pandemic. There is a need to risk-assess newly emerged variants in near &#34;real-time&#34; to estimate their potential threat to public health. The recently emerged Omicron sub-variant BA.2.86 raised concerns as it carries a high number of mutations compared to its predecessors. Here, we assessed the virulence of BA.2.86 in hamsters. We compared the pathogenesis of BA.2.86 and BA.2.75, as the latter is one of the most virulent Omicron sub-variants in this animal model. Using digital pathology pipelines, we quantified the extent of pulmonary lesions measuring T cell and macrophage infiltrates, in addition to alveolar epithelial hyperplasia. We also assessed body weight loss, clinical symptoms, virus load in oropharyngeal swabs, and virus replication in the respiratory tract. Our data show that BA.2.86 displays an attenuated phenotype in hamsters, suggesting that it poses no greater risk to public health than its parental Omicron sub-variants.

Article summary lineThe newly emerged Omicron sub-variant BA.2.86 is attenuated in hamsters.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566931">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566931" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566931">
        <p class="paperTitle">MicrobeMod: A computational toolkit for identifying prokaryotic methylation and restriction-modification with nanopore sequencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566931" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566931" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Crits-Christoph, A.; Kang, S. C.; Lee, H. H.; Ostrov, N.</p>
        <p class="info">Score: 33.4, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566931' target='https://doi.org/10.1101/2023.11.13.566931'> 10.1101/2023.11.13.566931</a></p>
        <p class="abstract">Bacteria and archaea use restriction-modification (R-M) systems to distinguish self from foreign DNA by methylating their genomes with DNA methyltransferases with diverse sequence specificities, and these immunity systems often vary at the strain level. Identifying active methylation patterns and R-M systems can reveal barriers to the introduction of recombinant DNA or phage infection. Here, we present the computational MicrobeMod toolkit for identifying 5mC and 6mA methylation sequence motifs and R-M systems in bacterial genomes using nanopore sequencing of native DNA. We benchmark this approach on a set of reference E. coli strains expressing methyltransferases with known specificities. We then applied these analyses to 31 diverse bacterial and archaeal organisms to reveal the methylation patterns of strains with previously unexplored epigenetics, finding that prokaryotic 5-methylcytosine may be more common than previously reported. In summary, MicrobeMod can rapidly reveal new epigenetics within a prokaryotic genome sequenced with Oxford Nanopore R10.4.1 flow cells at sequencing depths as low as 10x and only requires native DNA. This toolkit can be used to advance fundamental knowledge of bacterial methylation and guide strategies to overcome R-M barriers of genetic tractability in non-model microbes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.566145">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.566145" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.566145">
        <p class="paperTitle">An acidophilic fungus is integral to prey digestion in a carnivorous plant</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.566145" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.566145" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sun, P.-F.; Lu, M. R.; Liu, Y.-C.; Lin, Y.-f.; Hoh, D. Z.; Ke, H.-M.; Wang, I.-F.; Lu, M.-Y. J.; Kirschner, R.; Lin, Y.-C. J.; Chen, Y.-L.; Tsai, I. J.</p>
        <p class="info">Score: 88.1, Published: 2023-11-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.566145' target='https://doi.org/10.1101/2023.11.07.566145'> 10.1101/2023.11.07.566145</a></p>
        <p class="abstract">Carnivorous plant leaves, such as those of the spoon-leaved sundew Drosera spatulata, secrete mucilage which hosts microorganisms potentially aiding in prey digestion. We characterised the mucilage microbial communities and identified the acidophilic fungus Acrodontium crateriforme as the ecologically dominant species. The fungus grows and sporulates on sundew glands as its preferred acidic environment. We show that the A. crateriforme has a reduced genome similar to that of other symbiotic fungi. Based on the transcriptomes when encountering prey insects, we revealed a high degree of genes co-option in each species during fungus-plant coexistence and digestion. Expression patterns of the holobiont during digestion further revealed synergistic effects in several gene families including fungal aspartic and sedolisin peptidases, facilitating the digestion of sundews prey, as well as transporters and dose-dependent responses in plant genes involved in jasmonate signalling pathway. This study establishes that botanical carnivory is defined by multidimensional adaptations correlated with interspecies interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.567456">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.567456" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.567456">
        <p class="paperTitle">Evolution of a small phage protein confers resistance to antiphage defense in Enterococcus faecalis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.567456" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.567456" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Johnson, C. N.; Bullen, N. P.; Andersen, S. E.; Arya, G.; Marotta, S. R.; Whitney, J. C.; Duerkop, B. A.</p>
        <p class="info">Score: 23.9, Published: 2023-11-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.567456' target='https://doi.org/10.1101/2023.11.16.567456'> 10.1101/2023.11.16.567456</a></p>
        <p class="abstract">The prevalence of multidrug resistant (MDR) bacterial infections continues to rise as the development of new antibiotics needed to combat these infections remains stagnant. MDR enterococci, which are a common cause of hospital-acquired infections, are emerging as one of the major contributors to this crisis. A potential therapeutic approach for combating MDR enterococci is bacteriophage (phage) therapy, which entails the use of lytic viruses to infect and kill pathogenic bacteria. While phages that lyse some strains of MDR enterococci have been identified, other strains display high levels of phage resistance and the mechanisms underlying this resistance are unknown. Here, we use a CRISPR interference (CRISPRi) screen to identify a genetic locus found on a mobilizable plasmid from vancomycin-resistant Enterococcus faecalis involved in phage resistance. This locus encodes a putative serine recombinase followed by a Type IV restriction enzyme (TIV-RE) and we show that this enzyme is sufficient to restrict the replication of the lytic phage in E. faecalis. We further find that phages can evolve to overcome restriction by acquiring a missense mutation in a novel TIV-RE inhibitor protein encoded by many enterococcal phages. We show that this inhibitor, which we have named anti-restriction-factor A (arfA), directly binds to and inactivates diverse TIV-REs. Overall, our findings significantly advance our understanding of phage defense in drug-resistant E. faecalis and provide mechanistic insight into how phages can evolve to overcome antiphage defense systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.12.566429">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.12.566429" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.12.566429">
        <p class="paperTitle">The Parkinson&#39;s drug entacapone disrupts gut microbiome homeostasis via iron sequestration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.12.566429" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.12.566429" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pereira, F. C.; Ge, X.; Kristensen, J. M.; Kirkegaard, R. H.; Maritsch, K.; Zu, Y.; Decorte, M.; Hausmann, B.; Berry, D.; Wasmund, K.; Schintlmeister, A.; Boettcher, T.; Cheng, J.-X.; Wagner, M.</p>
        <p class="info">Score: 21.4, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.12.566429' target='https://doi.org/10.1101/2023.11.12.566429'> 10.1101/2023.11.12.566429</a></p>
        <p class="abstract">Increasing evidence shows that many human-targeted drugs alter the gut microbiome, leading to implications for host health. However, much less is known about the mechanisms by which drugs target the microbiome and how drugs affect microbial function. Here we combined quantitative microbiome profiling, long-read metagenomics, stable isotope probing and single cell chemical imaging to investigate the impact of two widely prescribed nervous system targeted drugs on the gut microbiome. Ex vivo supplementation of physiologically relevant concentrations of entacapone or loxapine succinate to faecal samples significantly impacted the abundance of up to one third of the microbial species present. Importantly, we demonstrate that the impact of these drugs on microbial metabolism is much more pronounced than their impact on abundances, with low concentrations of drugs reducing the activity, but not the abundance of key microbiome members like Bacteroides, Ruminococcus or Clostridium species. We further demonstrate that entacapone impacts the microbiome due to its ability to complex and deplete available iron, and that microbial growth can be rescued by replenishing levels of microbiota-accessible iron. Remarkably, entacapone-induced iron starvation selected for iron-scavenging organisms carrying antimicrobial resistance and virulence genes. Collectively, our study unveils the impact of two under-investigated drugs on whole microbiomes and identifies metal sequestration as a mechanism of drug-induced microbiome disturbance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.17.567518">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.17.567518" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.17.567518">
        <p class="paperTitle">Chromosomal Integrons are Genetically and Functionally Isolated Units of Genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.17.567518" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.17.567518" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Blanco, P.; Trigo da Roza, F.; Toribio-Celestino, L.; Garcia-Pastor, L.; Caselli, N.; Ojeda, F.; Darracq, B.; Vergara, E.; San Millan, A.; Skovgaard, O.; Mazel, D.; Loot, C.; Escudero, J. A.</p>
        <p class="info">Score: 18.6, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.17.567518' target='https://doi.org/10.1101/2023.11.17.567518'> 10.1101/2023.11.17.567518</a></p>
        <p class="abstract">Integrons are genetic elements that increase the evolvability of bacteria by capturing new genes and stockpiling them in arrays. Sedentary chromosomal integrons(SCIs), can be massive and highly stabilized structures encoding hundreds of genes, whose function remains generally unknown. SCIs have co-evolved with the host for aeons and are highly intertwined with their physiology from a mechanistic point of view. But, paradoxically, other aspects, like their variable content and location within the genome, suggest a high genetic and functional independence. In this work, we have explored the connection of SCIs to their host genome using as a model the Superintegron (SI), a 179-cassette long SCI in the genome of Vibrio cholerae N16961. We have relocated and deleted the SI using SeqDelTA, a novel method that allows to counteract the strong stabilization conferred by toxin-antitoxin systems within the array. We have characterized in depth the impact in V. cholerae&#39;s physiology, measuring fitness, chromosome replication dynamics, persistence, transcriptomics, phenomics and virulence. The deletion of the SI did not produce detectable effects in any condition, proving that -despite millions of years of co-evolution-, SCIs are genetically and functionally isolated units of genomes.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
