<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Antiviral humoral immunity against SARS-CoV-2 Omicron subvariants induced by XBB.1.5 monovalent vaccine in infection-naive and XBB-infected individuals
Authors: Kosugi, Y.; Kaku, Y.; Hinay, A. A.; Guo, Z.; Uriu, K.; Kihara, M.; Saito, F.; Uwamino, Y.; Kuramochi, J.; Shirakawa, K.; Takaori-Kondo, A.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Sato, K.
Score: 52.1, Published: 2023-11-30 DOI: 10.1101/2023.11.29.569330
To control infection with SARS-CoV-2 Omicron XBB subvariants, the XBB.1.5 monovalent mRNA vaccine has been available since September 2023.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Antiviral humoral immunity against SARS-CoV-2 Omicron subvariants induced by XBB.1.5 monovalent vaccine in infection-naive and XBB-infected individuals
Authors: Kosugi, Y.; Kaku, Y.; Hinay, A. A.; Guo, Z.; Uriu, K.; Kihara, M.; Saito, F.; Uwamino, Y.; Kuramochi, J.; Shirakawa, K.; Takaori-Kondo, A.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Sato, K.
Score: 52.1, Published: 2023-11-30 DOI: 10.1101/2023.11.29.569330
To control infection with SARS-CoV-2 Omicron XBB subvariants, the XBB.1.5 monovalent mRNA vaccine has been available since September 2023." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-06T10:40:04+00:00" />
<meta property="article:modified_time" content="2023-12-06T10:40:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Antiviral humoral immunity against SARS-CoV-2 Omicron subvariants induced by XBB.1.5 monovalent vaccine in infection-naive and XBB-infected individuals
Authors: Kosugi, Y.; Kaku, Y.; Hinay, A. A.; Guo, Z.; Uriu, K.; Kihara, M.; Saito, F.; Uwamino, Y.; Kuramochi, J.; Shirakawa, K.; Takaori-Kondo, A.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Sato, K.
Score: 52.1, Published: 2023-11-30 DOI: 10.1101/2023.11.29.569330
To control infection with SARS-CoV-2 Omicron XBB subvariants, the XBB.1.5 monovalent mRNA vaccine has been available since September 2023."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Antiviral humoral immunity against SARS-CoV-2 Omicron subvariants induced by XBB.1.5 monovalent vaccine in infection-naive and XBB-infected individuals\nAuthors: Kosugi, Y.; Kaku, Y.; Hinay, A. A.; Guo, Z.; Uriu, K.; Kihara, M.; Saito, F.; Uwamino, Y.; Kuramochi, J.; Shirakawa, K.; Takaori-Kondo, A.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Sato, K.\nScore: 52.1, Published: 2023-11-30 DOI: 10.1101/2023.11.29.569330\nTo control infection with SARS-CoV-2 Omicron XBB subvariants, the XBB.1.5 monovalent mRNA vaccine has been available since September 2023.",
  "keywords": [
    
  ],
  "articleBody": " Antiviral humoral immunity against SARS-CoV-2 Omicron subvariants induced by XBB.1.5 monovalent vaccine in infection-naive and XBB-infected individuals\nAuthors: Kosugi, Y.; Kaku, Y.; Hinay, A. A.; Guo, Z.; Uriu, K.; Kihara, M.; Saito, F.; Uwamino, Y.; Kuramochi, J.; Shirakawa, K.; Takaori-Kondo, A.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Sato, K.\nScore: 52.1, Published: 2023-11-30 DOI: 10.1101/2023.11.29.569330\nTo control infection with SARS-CoV-2 Omicron XBB subvariants, the XBB.1.5 monovalent mRNA vaccine has been available since September 2023. However, we have found that natural infection with XBB subvariants, including XBB.1.5, does not efficiently induce humoral immunity against the infecting XBB subvariants. These observations raise the possibility that the XBB.1.5 monovalent vaccine may not be able to efficiently induce humoral immunity against emerging SARS-CoV-2 variants, including a variety of XBB subvariants (XBB.1.5, XBB.1.16, XBB.2.3, EG.5.1 and HK.3) as well as BA.2.86. To address this possibility, we collected two types of sera from individuals vaccinated with the XBB.1.5 vaccine; those who had not been previously infected with SARS-CoV-2 and those who had been infected with XBB subvariants prior to XBB.1.5 vaccination. We collected sera before and 3-4 weeks after vaccination, and then performed a neutralization assay using these sera and pseudoviruses.\nSpatial segregation and aging of metabolic processes underlie phenotypic heterogeneity in mycobacteria\nAuthors: Gwin, C. M.; Gupta, K. R.; Lu, Y.; Shao, L.; Rego, E. H.\nScore: 41.3, Published: 2023-12-02 DOI: 10.1101/2023.12.01.569614\nIndividual cells within clonal populations of mycobacteria vary in size, growth rate, and antibiotic susceptibility. Heterogeneity is, in part, determined by LamA, a protein found exclusively in mycobacteria. LamA localizes to sites of new cell wall synthesis where it recruits proteins important for polar growth and establishing asymmetry. Here, we report that in addition to this function, LamA interacts with complexes involved in oxidative phosphorylation (OXPHOS) at a subcellular location distinct from cell wall synthesis. Importantly, heterogeneity depends on a unique extension of the mycobacterial ATP synthase, and LamA mediates the coupling between ATP production and cell growth in single cells. Strikingly, as single cells age, concentrations of proteins important for oxidative phosphorylation become less abundant, and older cells rely less on oxidative phosphorylation for growth. Together, our data reveal that central metabolism is spatially organized within a single mycobacterium and varies within a genetically identical population of mycobacteria. Designing therapeutic regimens to account for this heterogeneity may help to treat mycobacterial infections faster and more completely.\nHigh defense system expression broadens protection range at the cost of increased autoimmunity\nAuthors: Aframian, N.; Omer-Bendori, S.; Hen, T.; Guler, P.; Eldar, A.\nScore: 31.6, Published: 2023-11-30 DOI: 10.1101/2023.11.30.569366\nThe evolutionary arms race between bacteria and their phage viruses has given rise to elaborate anti-phage defense mechanisms. Major advances have been made in revealing the molecular details underlying diverse defense systems, but general principles and constraints are largely unkown. Defense systems are often tested against a diverse set of phages, revealing widely varying protection ranges. While these disparities are usually attributed to differences in mechanism, here we show that increasing expression of defense systems can greatly enhance their protection range. This holds true for disparate mechanisms, and is achieved by circumventing phage strategies for overcoming bacterial defense. However, increased defense system expression comes with a heavy cost of autoimmunity. Therefore, the expression level of defense systems controls a tradeoff between protection range on the one hand and autoimmunity on the other. We discuss how this tradeoff may drive the regulation of defense systems expression and the acquisition of multiple systems within the same genome.\nEmergence and spread of feline infectious peritonitis due to a highly pathogenic canine/feline recombinant coronavirus\nAuthors: Atippa, C.; Warr, A. S.; Epaminondas, D.; O'Shea, M.; Fletcher, S. L.; Malbon, A.; Lyraki, M.; Hammond, R.; Hardas, A.; Zanti, A.; Loukaidou, S.; Gentil, M.; Gunn-Moore, D.; Mazeri, S.; Tait-Burkard, C.\nScore: 254.6, Published: 2023-11-10 DOI: 10.1101/2023.11.08.566182\nCross-species transmission of coronaviruses (CoVs) poses a serious threat to both animal and human health1-3. Whilst the large RNA genome of CoVs shows relatively low mutation rates, recombination within genera is frequently observed and demonstrated4-7. Companion animals are often overlooked in the transmission cycle of viral diseases; however, the close relationship of feline (FCoV) and canine CoV (CCoV) to human hCoV-229E5,8, as well as their susceptibility to SARS-CoV-29 highlight their importance in potential transmission cycles. Whilst recombination between CCoV and FCoV of a large fragment spanning orf1b to M has been previously described5,10, here we report the emergence of a novel, highly pathogenic FCoV-CCoV recombinant responsible for a rapidly spreading outbreak of feline infectious peritonitis (FIP), originating in Cyprus11. The recombination, spanning spike, shows 97% sequence identity to the pantropic canine coronavirus CB/05. Infection is spreading fast and infecting cats of all ages. Development of FIP appears rapid and likely non-reliant on biotype switch12. High sequence identity of isolates from cats in different districts of the island is strongly supportive of direct transmission. A deletion and several amino acid changes in spike, particularly the receptor binding domain, compared to other FCoV-2s, indicate changes to receptor binding and likely cell tropism.\nDNA methylation by CcrM contributes to genome maintenance in the Agrobacterium tumefaciens plant pathogen\nAuthors: Martin, S.; Fournes, F.; Ambrosini, G.; Iseli, C.; BOJKOWSKA, K.; Marquis, J.; Guex, N.; Collier, J.\nScore: 26.4, Published: 2023-11-28 DOI: 10.1101/2023.11.28.569018\nThe cell cycle-regulated DNA methyltransferase CcrM is conserved in most Alphaproteobacteria, but its role in bacteria with complex or multicentric genomes remains unexplored. Here, we compare the methylome, the transcriptome and the phenotypes of wild-type and CcrM-depleted Agrobacterium tumefaciens cells with a dicentric genome with two essential replication origins. We find that DNA methylation has a pleiotropic impact on motility, biofilm formation and viability. Remarkably, CcrM promotes the expression of the repABCCh2 operon, encoding proteins required for replication initiation/partitioning at ori2, and inhibits gcrA, encoding a conserved global cell cycle regulator. Imaging ori1 and ori2 in live cells, we show that replication from ori2 is often delayed in cells with a hypo-methylated genome, while ori2 over-initiates in cells with a hyper-methylated genome. We thus propose that methylation by CcrM stimulates RepABC-dependent chromosomal origins, uncovering a novel and original connection between CcrM-dependent DNA methylation and genome maintenance in an Alphaproteobacterial pathogen.\nInteractions between the gut microbiome, dietary restriction, and aging in genetically diverse mice\nAuthors: Litichevskiy, L.; Considine, M.; Gill, J.; Shandar, V.; Cox, T. O.; Descamps, H. C.; Wright, K. M.; Amses, K. R.; Dohnalova, L.; Liou, M. J.; Tetlak, M.; Galindo-Fiallos, M. R.; Wong, A. C.; Lundgren, P.; Kim, J.; Uhr, G. T.; Rahman, R. J.; Mason, S.; Merenstein, C.; Bushman, F. D.; Raj, A.; Harding, F.; Chen, Z.; Prateek, G. V.; Mullis, M.; Deighan, A. G.; Robinson, L.; Tanes, C.; Bittinger, K.; Chakraborty, M.; Bhatt, A. S.; Li, H.; Barnett, I.; Davenport, E. R.; Broman, K. W.; Cohen, R. L.; Botstein, D.; Freund, A.; Di Francesco, A.; Churchill, G. A.; Li, M.; Thaiss, C. A.\nScore: 25.2, Published: 2023-11-29 DOI: 10.1101/2023.11.28.568137\nThe intestinal microbiome changes with age, but the causes and consequences of microbiome aging remain unclear. Furthermore, the gut microbiome has been proposed to mediate the benefit of lifespan- extending interventions such as dietary restriction, but this hypothesis warrants further exploration. Here, by analyzing 2997 metagenomes collected longitudinally from 913 deeply phenotyped, genetically diverse mice, we provide new insights into the interplay between the microbiome, aging, dietary restriction, host genetics, and a wide range of health parameters. First, we find that microbiome uniqueness increases with age across datasets and species. Moreover, age-associated changes are better explained by cumulative exposure to stochastic events (neutral theory) than by the influence of an aging host (selection theory). Second, we unexpectedly find that the majority of microbiome features are significantly heritable and that the amount of variation explained by host genetics is as large as that of aging and dietary restriction. Third, we find that the intensity of dietary restriction parallels the extent of microbiome changes and that dietary restriction does not rejuvenate the microbiome. Lastly, we find that the microbiome is significantly associated with multiple health parameters -- including body composition, immune parameters, and frailty -- but not with lifespan. In summary, this large and multifaceted study sheds light on the factors influencing the microbiome and aspects of host physiology modulated by the microbiome.\nConvergent evolution in toxin detection and resistance provides evidence for conserved bacterial-fungal interactions\nAuthors: Dolan, S. K.; Duong, A. T.; Whiteley, M.\nScore: 21.9, Published: 2023-11-28 DOI: 10.1101/2023.11.27.568923\nMicrobes rarely exist in isolation, and instead form complex polymicrobial communities. As a result, microbes have developed intricate offensive and defensive strategies that enhance their fitness in these complex communities. Thus, identifying and understanding the molecular mechanisms controlling polymicrobial interactions is critical for understanding the function of microbial communities. In this study, we show that the Gram-negative opportunistic human pathogen Pseudomonas aeruginosa, which frequently causes infection alongside a plethora of other microbes including fungi, encodes a genetic network which can detect, and defend against gliotoxin, a potent, disulfide-containing antimicrobial produced by the ubiquitous filamentous fungus Aspergillus fumigatus. We show that gliotoxin exposure disrupts P. aeruginosa zinc homeostasis, leading to transcriptional activation of a gene encoding a previously uncharacterized dithiol oxidase (DnoP), which detoxifies gliotoxin and structurally related toxins. While the enzymatic activity of DnoP is identical to that used by A. fumigatus to protect itself against gliotoxin, DnoP shares little homology to the A. fumigatus gliotoxin resistance protein. Thus, DnoP and its transcriptional induction by low zinc represent an example of both convergent evolution of toxin defense and environmental cue sensing across kingdoms. Collectively, these data support disulfide-containing natural products as mediators of inter-kingdom interactions and provide evidence that P. aeruginosa has evolved to survive exposure to these molecules in the natural environment. Significance statementBacteria and fungi frequently exist as complex, polymicrobial communities during infection. Reconstructing ecological structure in the laboratory is challenging and, consequently, the precise molecular mechanisms which underpin microbial interactions remain elusive. Using a pre-clinical model that mimics the cystic fibrosis lung, we discovered that the bacterium Pseudomonas aeruginosa detects and defends against a disulfide-containing toxin produced by the fungus Aspergillus fumigatus. In an example of both convergent evolution of toxin defense and environmental cue sensing across kingdoms, we discovered that these organisms use the same cue to produce/sense this toxin, and the same enzymatic mechanism to protect against toxicity. This discovery of convergent evolution provides strong evidence for P. aeruginosa exposure to microbially-produced disulfide-containing toxins in natural environments.\nIntracellular Ebola Virus nucleocapsid assembly revealed by in situ cryo-electron tomography.\nAuthors: Watanabe, R.; Zyla, D.; Parekh, D.; Hong, C.; Jones, Y.; Schendel, S. L.; Wan, W.; Castillon, G.; Saphire, E. O.\nScore: 19.8, Published: 2023-12-02 DOI: 10.1101/2023.12.01.569666\nFiloviruses, including Ebola and Marburg viruses, are a family of highly lethal viruses that cause severe hemorrhagic fever in humans with near annual outbreaks. Antiviral options are limited, and broad-spectrum antivirals are needed. The core of the filamentous virion is formed by the nucleocapsid, a helical structure in which the polymerized nucleoprotein (NP) assembles on the RNA genome and is surrounded by viral proteins VP24 and VP351,2. The mechanism by which these proteins assemble to form nucleocapsids inside infected cells is unclear, and the identity of the outer half of nucleocapsid density remains unassigned. Using cryo-electron tomography, we revealed the assembly process of Ebola virus in both cells transfected with the viral proteins and cells infected with a biologically contained model Ebola virus. We obtained a fully assembled intracellular nucleocapsid-like structure at 9 [A] that allows assignment of previously unassigned densities and presents the first complete model of the intracellular Ebola nucleocapsid. Our model reveals a previously unknown, third layer of additional copies of NP in complex with VP35. In this outer layer, the N-terminal domain of VP35 maintains NP in a monomeric form, and the rest of VP35 appears to participate in an intra-rung nucleocapsid interaction. The C-terminal region of NP lies outside this third outer layer, allowing localization and recruitment of additional viral proteins and connection of the nucleocapsid to the matrix beneath the virus envelope. Comparison of this in-cell nucleocapsid to previous in-virion structures reveals that the nucleocapsid helix condenses upon incorporation into the virion. Molecular interfaces responsible for nucleocapsid assembly are conserved among filoviruses, offering potential target sites for broadly effective antivirals. The architecture and assembly dynamics of Ebola virus nucleocapsids revealed here uncover a unique filovirus nucleocapsid assembly and a condensation mechanism that protect viral genome integrity in the virion.\nGenomic plasticity is a blueprint of diversity in Salmonella lineages\nAuthors: Krishnakant Kushwaha, S.; Anand, A.; Wu, Y.; Leonardo Avila, H.; Sicheritz-Ponten, T.; Millard, A.; Amol Marathe, S.; Nobrega, F. L.\nScore: 17.9, Published: 2023-12-02 DOI: 10.1101/2023.12.02.569618\nThe flexible genome, organised in regions of genomic plasticity (RGP), serves as a potent facilitator of the dynamic evolution of bacterial genomes through gene acquisition and loss. Here we explored the genomic plasticity across Salmonella lineages, revealing a purposeful, non-random integration pattern of pathogenicity-related gene clusters into specific RGP. Noteworthy examples include the correlation between the type I-E CRISPR-Cas system, gold tolerance, and specific RGP. The scattered prevalence of RGP across Salmonella lineages profoundly shapes the pathogenicity makeup of Salmonella strains. The preferences of RGP seem guided by conserved flanking genes that likely share regulatory and functional coordination. For example, metal resistance genes are predominant in RGP positioned near stress resistance genes, indicating a regulatory network to efficiently counter stressors. Additionally, we observed that different plasmid incompatibility types and prophage genera carry distinct pathogenicity genes. Similar to RGP, their distribution across Salmonella lineages plays a critical role in defining pathogenicity. By uncovering these intricate connections among gene clusters, RGP, mobile genetic elements, and pathogenic attributes, our study offers novel insights into the evolutionary trajectory of Salmonella and holds promise for predicting future adaptations and developing targeted interventions against infections.\nDistinct dynamics and proximity networks of hub proteins at the prey-invading cell pole in a predatory bacterium\nAuthors: Remy, O.; Santin, Y. G.; Jonckheere, V.; Tesseur, C.; Kaljevic, J.; Van Damme, P.; Laloux, G.\nScore: 17.2, Published: 2023-11-29 DOI: 10.1101/2023.11.29.569176\nIn bacteria, cell poles function as subcellular compartments where proteins localize during specific lifecycle stages, orchestrated by polar \"hub\" proteins. Whereas most described bacteria inherit an \"old\" pole from the mother cell and a \"new\" pole from cell division, polarizing cells at birth, non-binary division poses challenges for establishing cell polarity, particularly for daughter cells inheriting only new poles. We investigated polarity dynamics in the obligate predatory bacterium Bdellovibrio bacteriovorus, proliferating through filamentous growth followed by non-binary division within prey bacteria. Monitoring the subcellular localization of two proteins known as polar hubs in other species, RomR and DivIVA, revealed RomR as an early polarity marker in B. bacteriovorus. RomR already marks the future anterior poles of the progeny during the predators growth phase, in a define time window closely following the onset of divisome assembly and the end of chromosome segregation. In contrast to RomRs stable unipolar localization in the progeny, DivIVA exhibits a dynamic pole-to-pole localization. This behaviour changes shortly before division of the elongated predator cell, where DivIVA accumulates at all septa and both poles. In vivo protein interaction networks for DivIVA and RomR, mapped through endogenous miniTurbo-based proximity labeling, further underscore their distinct roles in cell polarization and the importance of the anterior \"invasive\" cell pole in prey-predator interactions. Our work emphasizes the strict spatiotemporal coordination of cellular processes underlying B. bacteriovorus proliferation, offering insights into the subcellular organization of bacteria with filamentous growth and non-binary division.\n",
  "wordCount" : "2565",
  "inLanguage": "en",
  "datePublished": "2023-12-06T10:40:04Z",
  "dateModified": "2023-12-06T10:40:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 6, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569330">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569330" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569330">
        <p class="paperTitle">Antiviral humoral immunity against SARS-CoV-2 Omicron subvariants induced by XBB.1.5 monovalent vaccine in infection-naive and XBB-infected individuals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569330" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569330" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kosugi, Y.; Kaku, Y.; Hinay, A. A.; Guo, Z.; Uriu, K.; Kihara, M.; Saito, F.; Uwamino, Y.; Kuramochi, J.; Shirakawa, K.; Takaori-Kondo, A.; The Genotype to Phenotype Japan (G2P-Japan) Consortium,  ; Sato, K.</p>
        <p class="info">Score: 52.1, Published: 2023-11-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569330' target='https://doi.org/10.1101/2023.11.29.569330'> 10.1101/2023.11.29.569330</a></p>
        <p class="abstract">To control infection with SARS-CoV-2 Omicron XBB subvariants, the XBB.1.5 monovalent mRNA vaccine has been available since September 2023. However, we have found that natural infection with XBB subvariants, including XBB.1.5, does not efficiently induce humoral immunity against the infecting XBB subvariants. These observations raise the possibility that the XBB.1.5 monovalent vaccine may not be able to efficiently induce humoral immunity against emerging SARS-CoV-2 variants, including a variety of XBB subvariants (XBB.1.5, XBB.1.16, XBB.2.3, EG.5.1 and HK.3) as well as BA.2.86. To address this possibility, we collected two types of sera from individuals vaccinated with the XBB.1.5 vaccine; those who had not been previously infected with SARS-CoV-2 and those who had been infected with XBB subvariants prior to XBB.1.5 vaccination. We collected sera before and 3-4 weeks after vaccination, and then performed a neutralization assay using these sera and pseudoviruses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569614">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569614" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569614">
        <p class="paperTitle">Spatial segregation and aging of metabolic processes underlie phenotypic heterogeneity in mycobacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569614" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569614" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gwin, C. M.; Gupta, K. R.; Lu, Y.; Shao, L.; Rego, E. H.</p>
        <p class="info">Score: 41.3, Published: 2023-12-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569614' target='https://doi.org/10.1101/2023.12.01.569614'> 10.1101/2023.12.01.569614</a></p>
        <p class="abstract">Individual cells within clonal populations of mycobacteria vary in size, growth rate, and antibiotic susceptibility. Heterogeneity is, in part, determined by LamA, a protein found exclusively in mycobacteria. LamA localizes to sites of new cell wall synthesis where it recruits proteins important for polar growth and establishing asymmetry. Here, we report that in addition to this function, LamA interacts with complexes involved in oxidative phosphorylation (OXPHOS) at a subcellular location distinct from cell wall synthesis. Importantly, heterogeneity depends on a unique extension of the mycobacterial ATP synthase, and LamA mediates the coupling between ATP production and cell growth in single cells. Strikingly, as single cells age, concentrations of proteins important for oxidative phosphorylation become less abundant, and older cells rely less on oxidative phosphorylation for growth. Together, our data reveal that central metabolism is spatially organized within a single mycobacterium and varies within a genetically identical population of mycobacteria. Designing therapeutic regimens to account for this heterogeneity may help to treat mycobacterial infections faster and more completely.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569366">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569366" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569366">
        <p class="paperTitle">High defense system expression broadens protection range at the cost of increased autoimmunity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569366" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569366" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aframian, N.; Omer-Bendori, S.; Hen, T.; Guler, P.; Eldar, A.</p>
        <p class="info">Score: 31.6, Published: 2023-11-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569366' target='https://doi.org/10.1101/2023.11.30.569366'> 10.1101/2023.11.30.569366</a></p>
        <p class="abstract">The evolutionary arms race between bacteria and their phage viruses has given rise to elaborate anti-phage defense mechanisms. Major advances have been made in revealing the molecular details underlying diverse defense systems, but general principles and constraints are largely unkown. Defense systems are often tested against a diverse set of phages, revealing widely varying protection ranges. While these disparities are usually attributed to differences in mechanism, here we show that increasing expression of defense systems can greatly enhance their protection range. This holds true for disparate mechanisms, and is achieved by circumventing phage strategies for overcoming bacterial defense. However, increased defense system expression comes with a heavy cost of autoimmunity. Therefore, the expression level of defense systems controls a tradeoff between protection range on the one hand and autoimmunity on the other. We discuss how this tradeoff may drive the regulation of defense systems expression and the acquisition of multiple systems within the same genome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.08.566182">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.08.566182" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.08.566182">
        <p class="paperTitle">Emergence and spread of feline infectious peritonitis due to a highly pathogenic canine/feline recombinant coronavirus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.08.566182" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.08.566182" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Atippa, C.; Warr, A. S.; Epaminondas, D.; O&#39;Shea, M.; Fletcher, S. L.; Malbon, A.; Lyraki, M.; Hammond, R.; Hardas, A.; Zanti, A.; Loukaidou, S.; Gentil, M.; Gunn-Moore, D.; Mazeri, S.; Tait-Burkard, C.</p>
        <p class="info">Score: 254.6, Published: 2023-11-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.08.566182' target='https://doi.org/10.1101/2023.11.08.566182'> 10.1101/2023.11.08.566182</a></p>
        <p class="abstract">Cross-species transmission of coronaviruses (CoVs) poses a serious threat to both animal and human health1-3. Whilst the large RNA genome of CoVs shows relatively low mutation rates, recombination within genera is frequently observed and demonstrated4-7. Companion animals are often overlooked in the transmission cycle of viral diseases; however, the close relationship of feline (FCoV) and canine CoV (CCoV) to human hCoV-229E5,8, as well as their susceptibility to SARS-CoV-29 highlight their importance in potential transmission cycles. Whilst recombination between CCoV and FCoV of a large fragment spanning orf1b to M has been previously described5,10, here we report the emergence of a novel, highly pathogenic FCoV-CCoV recombinant responsible for a rapidly spreading outbreak of feline infectious peritonitis (FIP), originating in Cyprus11. The recombination, spanning spike, shows 97% sequence identity to the pantropic canine coronavirus CB/05. Infection is spreading fast and infecting cats of all ages. Development of FIP appears rapid and likely non-reliant on biotype switch12. High sequence identity of isolates from cats in different districts of the island is strongly supportive of direct transmission. A deletion and several amino acid changes in spike, particularly the receptor binding domain, compared to other FCoV-2s, indicate changes to receptor binding and likely cell tropism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.569018">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.569018" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.569018">
        <p class="paperTitle">DNA methylation by CcrM contributes to genome maintenance in the Agrobacterium tumefaciens plant pathogen</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.569018" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.569018" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martin, S.; Fournes, F.; Ambrosini, G.; Iseli, C.; BOJKOWSKA, K.; Marquis, J.; Guex, N.; Collier, J.</p>
        <p class="info">Score: 26.4, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.569018' target='https://doi.org/10.1101/2023.11.28.569018'> 10.1101/2023.11.28.569018</a></p>
        <p class="abstract">The cell cycle-regulated DNA methyltransferase CcrM is conserved in most Alphaproteobacteria, but its role in bacteria with complex or multicentric genomes remains unexplored. Here, we compare the methylome, the transcriptome and the phenotypes of wild-type and CcrM-depleted Agrobacterium tumefaciens cells with a dicentric genome with two essential replication origins. We find that DNA methylation has a pleiotropic impact on motility, biofilm formation and viability. Remarkably, CcrM promotes the expression of the repABCCh2 operon, encoding proteins required for replication initiation/partitioning at ori2, and inhibits gcrA, encoding a conserved global cell cycle regulator. Imaging ori1 and ori2 in live cells, we show that replication from ori2 is often delayed in cells with a hypo-methylated genome, while ori2 over-initiates in cells with a hyper-methylated genome. We thus propose that methylation by CcrM stimulates RepABC-dependent chromosomal origins, uncovering a novel and original connection between CcrM-dependent DNA methylation and genome maintenance in an Alphaproteobacterial pathogen.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.568137">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.568137" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.568137">
        <p class="paperTitle">Interactions between the gut microbiome, dietary restriction, and aging in genetically diverse mice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.568137" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.568137" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Litichevskiy, L.; Considine, M.; Gill, J.; Shandar, V.; Cox, T. O.; Descamps, H. C.; Wright, K. M.; Amses, K. R.; Dohnalova, L.; Liou, M. J.; Tetlak, M.; Galindo-Fiallos, M. R.; Wong, A. C.; Lundgren, P.; Kim, J.; Uhr, G. T.; Rahman, R. J.; Mason, S.; Merenstein, C.; Bushman, F. D.; Raj, A.; Harding, F.; Chen, Z.; Prateek, G. V.; Mullis, M.; Deighan, A. G.; Robinson, L.; Tanes, C.; Bittinger, K.; Chakraborty, M.; Bhatt, A. S.; Li, H.; Barnett, I.; Davenport, E. R.; Broman, K. W.; Cohen, R. L.; Botstein, D.; Freund, A.; Di Francesco, A.; Churchill, G. A.; Li, M.; Thaiss, C. A.</p>
        <p class="info">Score: 25.2, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.568137' target='https://doi.org/10.1101/2023.11.28.568137'> 10.1101/2023.11.28.568137</a></p>
        <p class="abstract">The intestinal microbiome changes with age, but the causes and consequences of microbiome aging remain unclear. Furthermore, the gut microbiome has been proposed to mediate the benefit of lifespan- extending interventions such as dietary restriction, but this hypothesis warrants further exploration. Here, by analyzing 2997 metagenomes collected longitudinally from 913 deeply phenotyped, genetically diverse mice, we provide new insights into the interplay between the microbiome, aging, dietary restriction, host genetics, and a wide range of health parameters. First, we find that microbiome uniqueness increases with age across datasets and species. Moreover, age-associated changes are better explained by cumulative exposure to stochastic events (neutral theory) than by the influence of an aging host (selection theory). Second, we unexpectedly find that the majority of microbiome features are significantly heritable and that the amount of variation explained by host genetics is as large as that of aging and dietary restriction. Third, we find that the intensity of dietary restriction parallels the extent of microbiome changes and that dietary restriction does not rejuvenate the microbiome. Lastly, we find that the microbiome is significantly associated with multiple health parameters -- including body composition, immune parameters, and frailty -- but not with lifespan. In summary, this large and multifaceted study sheds light on the factors influencing the microbiome and aspects of host physiology modulated by the microbiome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.568923">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.568923" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.568923">
        <p class="paperTitle">Convergent evolution in toxin detection and resistance provides evidence for conserved bacterial-fungal interactions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.568923" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.568923" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dolan, S. K.; Duong, A. T.; Whiteley, M.</p>
        <p class="info">Score: 21.9, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.568923' target='https://doi.org/10.1101/2023.11.27.568923'> 10.1101/2023.11.27.568923</a></p>
        <p class="abstract">Microbes rarely exist in isolation, and instead form complex polymicrobial communities. As a result, microbes have developed intricate offensive and defensive strategies that enhance their fitness in these complex communities. Thus, identifying and understanding the molecular mechanisms controlling polymicrobial interactions is critical for understanding the function of microbial communities. In this study, we show that the Gram-negative opportunistic human pathogen Pseudomonas aeruginosa, which frequently causes infection alongside a plethora of other microbes including fungi, encodes a genetic network which can detect, and defend against gliotoxin, a potent, disulfide-containing antimicrobial produced by the ubiquitous filamentous fungus Aspergillus fumigatus. We show that gliotoxin exposure disrupts P. aeruginosa zinc homeostasis, leading to transcriptional activation of a gene encoding a previously uncharacterized dithiol oxidase (DnoP), which detoxifies gliotoxin and structurally related toxins. While the enzymatic activity of DnoP is identical to that used by A. fumigatus to protect itself against gliotoxin, DnoP shares little homology to the A. fumigatus gliotoxin resistance protein. Thus, DnoP and its transcriptional induction by low zinc represent an example of both convergent evolution of toxin defense and environmental cue sensing across kingdoms. Collectively, these data support disulfide-containing natural products as mediators of inter-kingdom interactions and provide evidence that P. aeruginosa has evolved to survive exposure to these molecules in the natural environment.

Significance statementBacteria and fungi frequently exist as complex, polymicrobial communities during infection. Reconstructing ecological structure in the laboratory is challenging and, consequently, the precise molecular mechanisms which underpin microbial interactions remain elusive. Using a pre-clinical model that mimics the cystic fibrosis lung, we discovered that the bacterium Pseudomonas aeruginosa detects and defends against a disulfide-containing toxin produced by the fungus Aspergillus fumigatus. In an example of both convergent evolution of toxin defense and environmental cue sensing across kingdoms, we discovered that these organisms use the same cue to produce/sense this toxin, and the same enzymatic mechanism to protect against toxicity. This discovery of convergent evolution provides strong evidence for P. aeruginosa exposure to microbially-produced disulfide-containing toxins in natural environments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569666">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569666" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569666">
        <p class="paperTitle">Intracellular Ebola Virus nucleocapsid assembly revealed by in situ cryo-electron tomography.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569666" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569666" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Watanabe, R.; Zyla, D.; Parekh, D.; Hong, C.; Jones, Y.; Schendel, S. L.; Wan, W.; Castillon, G.; Saphire, E. O.</p>
        <p class="info">Score: 19.8, Published: 2023-12-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569666' target='https://doi.org/10.1101/2023.12.01.569666'> 10.1101/2023.12.01.569666</a></p>
        <p class="abstract">Filoviruses, including Ebola and Marburg viruses, are a family of highly lethal viruses that cause severe hemorrhagic fever in humans with near annual outbreaks. Antiviral options are limited, and broad-spectrum antivirals are needed. The core of the filamentous virion is formed by the nucleocapsid, a helical structure in which the polymerized nucleoprotein (NP) assembles on the RNA genome and is surrounded by viral proteins VP24 and VP351,2. The mechanism by which these proteins assemble to form nucleocapsids inside infected cells is unclear, and the identity of the outer half of nucleocapsid density remains unassigned. Using cryo-electron tomography, we revealed the assembly process of Ebola virus in both cells transfected with the viral proteins and cells infected with a biologically contained model Ebola virus. We obtained a fully assembled intracellular nucleocapsid-like structure at 9 [A] that allows assignment of previously unassigned densities and presents the first complete model of the intracellular Ebola nucleocapsid. Our model reveals a previously unknown, third layer of additional copies of NP in complex with VP35. In this outer layer, the N-terminal domain of VP35 maintains NP in a monomeric form, and the rest of VP35 appears to participate in an intra-rung nucleocapsid interaction. The C-terminal region of NP lies outside this third outer layer, allowing localization and recruitment of additional viral proteins and connection of the nucleocapsid to the matrix beneath the virus envelope. Comparison of this in-cell nucleocapsid to previous in-virion structures reveals that the nucleocapsid helix condenses upon incorporation into the virion. Molecular interfaces responsible for nucleocapsid assembly are conserved among filoviruses, offering potential target sites for broadly effective antivirals. The architecture and assembly dynamics of Ebola virus nucleocapsids revealed here uncover a unique filovirus nucleocapsid assembly and a condensation mechanism that protect viral genome integrity in the virion.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.02.569618">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.02.569618" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.02.569618">
        <p class="paperTitle">Genomic plasticity is a blueprint of diversity in Salmonella lineages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.02.569618" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.02.569618" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krishnakant Kushwaha, S.; Anand, A.; Wu, Y.; Leonardo Avila, H.; Sicheritz-Ponten, T.; Millard, A.; Amol Marathe, S.; Nobrega, F. L.</p>
        <p class="info">Score: 17.9, Published: 2023-12-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.02.569618' target='https://doi.org/10.1101/2023.12.02.569618'> 10.1101/2023.12.02.569618</a></p>
        <p class="abstract">The flexible genome, organised in regions of genomic plasticity (RGP), serves as a potent facilitator of the dynamic evolution of bacterial genomes through gene acquisition and loss. Here we explored the genomic plasticity across Salmonella lineages, revealing a purposeful, non-random integration pattern of pathogenicity-related gene clusters into specific RGP. Noteworthy examples include the correlation between the type I-E CRISPR-Cas system, gold tolerance, and specific RGP. The scattered prevalence of RGP across Salmonella lineages profoundly shapes the pathogenicity makeup of Salmonella strains. The preferences of RGP seem guided by conserved flanking genes that likely share regulatory and functional coordination. For example, metal resistance genes are predominant in RGP positioned near stress resistance genes, indicating a regulatory network to efficiently counter stressors. Additionally, we observed that different plasmid incompatibility types and prophage genera carry distinct pathogenicity genes. Similar to RGP, their distribution across Salmonella lineages plays a critical role in defining pathogenicity. By uncovering these intricate connections among gene clusters, RGP, mobile genetic elements, and pathogenic attributes, our study offers novel insights into the evolutionary trajectory of Salmonella and holds promise for predicting future adaptations and developing targeted interventions against infections.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569176">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569176" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569176">
        <p class="paperTitle">Distinct dynamics and proximity networks of hub proteins at the prey-invading cell pole in a predatory bacterium</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569176" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569176" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Remy, O.; Santin, Y. G.; Jonckheere, V.; Tesseur, C.; Kaljevic, J.; Van Damme, P.; Laloux, G.</p>
        <p class="info">Score: 17.2, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569176' target='https://doi.org/10.1101/2023.11.29.569176'> 10.1101/2023.11.29.569176</a></p>
        <p class="abstract">In bacteria, cell poles function as subcellular compartments where proteins localize during specific lifecycle stages, orchestrated by polar &#34;hub&#34; proteins. Whereas most described bacteria inherit an &#34;old&#34; pole from the mother cell and a &#34;new&#34; pole from cell division, polarizing cells at birth, non-binary division poses challenges for establishing cell polarity, particularly for daughter cells inheriting only new poles. We investigated polarity dynamics in the obligate predatory bacterium Bdellovibrio bacteriovorus, proliferating through filamentous growth followed by non-binary division within prey bacteria. Monitoring the subcellular localization of two proteins known as polar hubs in other species, RomR and DivIVA, revealed RomR as an early polarity marker in B. bacteriovorus. RomR already marks the future anterior poles of the progeny during the predators growth phase, in a define time window closely following the onset of divisome assembly and the end of chromosome segregation. In contrast to RomRs stable unipolar localization in the progeny, DivIVA exhibits a dynamic pole-to-pole localization. This behaviour changes shortly before division of the elongated predator cell, where DivIVA accumulates at all septa and both poles. In vivo protein interaction networks for DivIVA and RomR, mapped through endogenous miniTurbo-based proximity labeling, further underscore their distinct roles in cell polarization and the importance of the anterior &#34;invasive&#34; cell pole in prey-predator interactions. Our work emphasizes the strict spatiotemporal coordination of cellular processes underlying B. bacteriovorus proliferation, offering insights into the subcellular organization of bacteria with filamentous growth and non-binary division.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
