<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="First Eurasian cases of SARS-CoV-2 seropositivity in a free-ranging urban population of wild fallow deer
Authors: Purves, K.; Brown, H.; Haverty, R.; Ryan, A.; Griffin, L.; McCormack, J.; O&#39;Reilly, S. R.; Mallon, P. W.; Gautier, V.; Cassidy, J. P.; Fabre, A.; Carr, M. J.; Gonzalez, G.; Ciuti, S.; Fletcher, N.
Score: 65.4, Published: 2023-07-07 DOI: 10.1101/2023.07.07.547941
Severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) infects wildlife. Recent studies highlighted that variants of concern (VOC) may expand into novel animal reservoirs with the potential for reverse zoonosis.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="First Eurasian cases of SARS-CoV-2 seropositivity in a free-ranging urban population of wild fallow deer
Authors: Purves, K.; Brown, H.; Haverty, R.; Ryan, A.; Griffin, L.; McCormack, J.; O&#39;Reilly, S. R.; Mallon, P. W.; Gautier, V.; Cassidy, J. P.; Fabre, A.; Carr, M. J.; Gonzalez, G.; Ciuti, S.; Fletcher, N.
Score: 65.4, Published: 2023-07-07 DOI: 10.1101/2023.07.07.547941
Severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) infects wildlife. Recent studies highlighted that variants of concern (VOC) may expand into novel animal reservoirs with the potential for reverse zoonosis." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-12T10:41:10+00:00" />
<meta property="article:modified_time" content="2023-07-12T10:41:10+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="First Eurasian cases of SARS-CoV-2 seropositivity in a free-ranging urban population of wild fallow deer
Authors: Purves, K.; Brown, H.; Haverty, R.; Ryan, A.; Griffin, L.; McCormack, J.; O&#39;Reilly, S. R.; Mallon, P. W.; Gautier, V.; Cassidy, J. P.; Fabre, A.; Carr, M. J.; Gonzalez, G.; Ciuti, S.; Fletcher, N.
Score: 65.4, Published: 2023-07-07 DOI: 10.1101/2023.07.07.547941
Severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) infects wildlife. Recent studies highlighted that variants of concern (VOC) may expand into novel animal reservoirs with the potential for reverse zoonosis."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "First Eurasian cases of SARS-CoV-2 seropositivity in a free-ranging urban population of wild fallow deer\nAuthors: Purves, K.; Brown, H.; Haverty, R.; Ryan, A.; Griffin, L.; McCormack, J.; O\u0026#39;Reilly, S. R.; Mallon, P. W.; Gautier, V.; Cassidy, J. P.; Fabre, A.; Carr, M. J.; Gonzalez, G.; Ciuti, S.; Fletcher, N.\nScore: 65.4, Published: 2023-07-07 DOI: 10.1101/2023.07.07.547941\nSevere acute respiratory syndrome coronavirus-2 (SARS-CoV-2) infects wildlife. Recent studies highlighted that variants of concern (VOC) may expand into novel animal reservoirs with the potential for reverse zoonosis.",
  "keywords": [
    
  ],
  "articleBody": " First Eurasian cases of SARS-CoV-2 seropositivity in a free-ranging urban population of wild fallow deer\nAuthors: Purves, K.; Brown, H.; Haverty, R.; Ryan, A.; Griffin, L.; McCormack, J.; O'Reilly, S. R.; Mallon, P. W.; Gautier, V.; Cassidy, J. P.; Fabre, A.; Carr, M. J.; Gonzalez, G.; Ciuti, S.; Fletcher, N.\nScore: 65.4, Published: 2023-07-07 DOI: 10.1101/2023.07.07.547941\nSevere acute respiratory syndrome coronavirus-2 (SARS-CoV-2) infects wildlife. Recent studies highlighted that variants of concern (VOC) may expand into novel animal reservoirs with the potential for reverse zoonosis. North American white-tailed deer are the only deer species in which SARS-CoV-2 has been documented, raising the question whether further reservoir species exist as new VOC emerge. Here, we report the first cases of deer SARS-CoV-2 seropositivity in Eurasia, in a city population of fallow deer in Dublin, Ireland. Deer were seronegative in 2020 (circulating variant in humans: Alpha), one animal was seropositive in 2021 (Delta variant), and 57% of animals tested in 2022 were seropositive (Omicron variant). Ex vivo, a clinical isolate of Omicron BA.1 infected fallow deer precision cut lung slice type-2 pneumocytes, also a major target of infection in human lungs. Our findings suggest a change in host tropism as new variants emerged in the human reservoir, highlighting the importance of continued wildlife disease monitoring and limiting human-wildlife contacts.\nBam complex associated proteins in Escherichia coli are functionally linked to peptidoglycan biosynthesis, membrane fluidity and DNA replication\nAuthors: Bryant, J. A.; Staunton, K. A.; Doherty, H. M.; Alao, M. B.; Ma, X.; Morcinek-Orlowska, J.; Goodall, E. C. A.; Gray, J.; Milner, M.; Cole, J. A.; de Cogan, F.; Knowles, T. J.; Glinkowska, M. K.; Moradigaravand, D.; Henderson, I. R.; Banzhaf, M.\nScore: 22.9, Published: 2023-07-05 DOI: 10.1101/2023.07.05.547807\nBiogenesis of the bacterial outer membrane is key to survival and antibiotic resistance. Central to this is the {beta}-barrel assembly machine (Bam) complex and its associated chaperones, which are responsible for outer membrane protein (OMP) transport and insertion. The Escherichia coli Bam complex consists of two essential subunits, BamA and BamD, and three non-essential lipoproteins, BamB, BamC and BamE. Optimal Bam function is further dependent on the non-essential chaperones DegP, Skp and SurA. Despite intensive study, the specific function of these non-essential Bam-associated proteins remains unknown. Here, we analysed knockout strains for each gene by phenotypic screening, conservation analysis and high-throughput genetics. We reveal that Bam activity is affected by outer membrane lipid composition and that enterobacterial common antigen is essential in the absence of the chaperone SurA. We also show that components of peptidoglycan are conditionally essential with Bam accessory lipoproteins and that DNA replication is perturbed in the absence of BamB. Together, our data indicates potential mechanisms for coordination of OMP biogenesis with processes such as LPS and peptidoglycan biogenesis, and DNA replication.\nNECing goes: flexibility of the herpesvirus nuclear egress complex\nAuthors: Prazak, V.; Mironova, Y.; Vasishtan, D.; Hagen, C.; Laugks, U.; Jensen, Y.; Sanders, S.; Heumann, J. M.; Bosse, J. B.; Klupp, B.; Mettenleiter, T. C.; Grange, M.; Grünewald, K.\nScore: 20.7, Published: 2023-07-07 DOI: 10.1101/2023.07.07.547920\nThe nuclear egress complex (NEC) allows herpesvirus capsids to escape from the nucleus without breaking the nuclear envelope barrier. It assembles into a lattice on the inner nuclear membrane enveloping newly assembled nucleocapsids, which bud into the perinuclear space. The primary virion envelope subsequently fuses with the outer nuclear membrane, releasing the capsid into the cytosol. Here we interrogated the NEC in the context of intact cells infected with pseudorabies or herpes simplex virus using focused-ion beam milling and electron cryo-tomography. We determined the structure of NEC in different conformations and show that it consists of a flexible hexameric lattice that generates curvature through a combination of ordered and disordered domains. After interrogating the intermediate stages of capsid formation, we show that capsid vertex binding may initiate envelopment but does not directly induce curvature formation. These data and many examples of the intermediate stages of nuclear egress paint a detailed holistic view of a versatile transport system.\nBridges instead of boats? The Mla system of diderm Firmicute Veillonella parvula reveals an ancestral transenvelope core of phospholipid trafficking\nAuthors: BELOIN, C.; Grasekamp, K. P.; Beaud, B.; Taib, N.; Audrain, B.; Bardiaux, B.; Rossez, Y.; IZADI-PRUNEYRE, N.; Lejeune, M.; Trivelli, X.; Chouit, Z.; Guerardel, Y.; GHIGO, J.-M.; Gribaldo, S.\nScore: 21.2, Published: 2023-06-30 DOI: 10.1101/2023.06.30.547184\nDespite extensive characterisation of envelope biogenesis systems in diderm bacteria, glycerophospholipid (GPL) trafficking remains poorly understood, and has only been studied in a handful of model species. Within the Proteobacteria, the maintenance of lipid asymmetry (Mla) system facilitates retrograde GPL trafficking via six proteins, MlaA-F. GPLs are extracted from the outer leaflet of the outer membrane by the lipoprotein MlaA which associates with porin trimers, then shipped through the periplasmic space by the chaperone MlaC, which finally delivers GPLs to the inner membrane complex formed by MlaBDEF. Here, we investigate GPL trafficking in Veillonella parvula, a diderm member of the Firmicutes which encodes an Mla system devoid of MlaA and MlaC. V. parvula {Delta}mla mutants display phenotypes characteristic of disrupted lipid asymmetry such as hypervesiculation and detergent hypersensitivity, and lipid content analysis from outer membrane vesicles reveals an enrichment for the major lipid component phosphatidylethanolamine. Interestingly, suppressor analysis identifies mutations in tamB that rescue detergent hypersensitivity and hypervesiculation of {Delta}mla strains, supporting the involvement of these two systems in antagonistic GPL trafficking functions across diverse bacterial lineages. A combination of structural modeling and subcellular localisation assays shows that MlaDVp is longer than in classical diderm models and forms a transenvelope bridge, encoding both an inner membrane-localised MCE domain and an outer membrane {beta}-barrel. These results strongly suggest that V. parvula possesses a minimal Mla system for GPL trafficking, replacing the need for chaperones and outer membrane lipoproteins by directly connecting the two membranes. Finally, phylogenomic analysis indicates that this MlaEFD self-contained architecture is widely distributed in diderm bacteria and most likely represents the ancestral functional core of the Mla system, which subsequently increased in complexity in Proteobacteria and closely related phyla following the emergence of MlaABC. Our work broadens the diversity of current models of GPL trafficking in diderm bacteria, challenging the paradigm set by classical models and shedding light on the evolution of a crucial system in the biogenesis and maintenance of the bacterial outer membrane.\nPredicting bacterial interaction outcomes from monoculture growth and supernatant assays\nAuthors: Schmitz, D. A.; Wechsler, T.; Mignot, I.; Kuemmerli, R.\nScore: 17.9, Published: 2023-07-03 DOI: 10.1101/2023.07.03.547502\nHow to derive principles of community dynamics and stability is a central question in microbial ecology. To answer this, bottom-up experimental approaches, in which a small number of bacterial species are mixed, have become popular. However, experimental setups are typically limited because species are difficult to distinguish in mixed cultures and co-culture experiments are labor-intensive. Here, we use a community of four bacterial species to show that information from monoculture growth and inhibitory effects caused by secreted compounds can be combined to reliably predict pairwise species interactions in co-cultures. Specifically, integrative parameters from growth curves allow to build a competitive rank order, which is then adjusted using inhibitory compound effects from supernatant assays. While our procedure worked for two media examined, we observed differences in species rank orders between media. We further used computer simulations, parameterized with our empirical data, to show that higher-order species interactions largely follow the dynamics predicted from pairwise interactions with one important exception. The impact of inhibitory compounds was reduced in higher-order communities because these compounds are spread across multiple species, thereby diluting their effects. Altogether, our results lead to the formulation of three simple rules of how monoculture growth and supernatant assay data can be combined to establish competitive species rank orders in bacterial communities.\nA novel tamanavirus (Flaviviridae) of the European common frog (Rana temporaria) encodes a divergent class 1b XRN1-resistant RNA element.\nAuthors: Parry, R. H.; Slonchak, A.; Campbell, L. J.; Newton, N.; Debat, H.; Gifford, R. J.; Khromykh, A.\nScore: 15.3, Published: 2023-07-06 DOI: 10.1101/2023.07.06.547906\nFlavivirids are small, enveloped, positive-sense RNA viruses from the Flaviviridae family with genomes between [~]9-13kb. Metatranscriptomic analyses of metazoan organisms have revealed a diversity of flavivirus-like or flavivirid viral sequences in fish and marine invertebrate groups. To date, however, no flavivirus-like or flavivirid has been identified in amphibians. To remedy this, we investigated the virome of the European common frog (Rana temporaria) in the United Kingdom, utilising high-throughput sequencing at six catch locations. De novo assembly revealed a coding-complete virus contig of a novel flavivirid [~]11.2kb in length. The virus encodes a single open reading frame of 3456 amino acids and 5 and 3 untranslated regions (UTRs) of 227 and 666nt, respectively. We named this virus Rana tamanavirus (RaTV), as BLASTp analysis of the polyprotein showed the closest relationships to Tamana bat virus (TABV) and Cyclopterus lumpus virus from Pteronotus parnellii and Cyclopterus lumpus, respectively. Phylogenetic analysis of the RaTV polyprotein compared to Flavivirus and Flavivirus-like members indicated that RaTV was sufficiently divergent and basal to the vertebrate Tamanavirus clade. In addition to the Mitcham strain, partial but divergent RaTV, 95.64-97.39% pairwise nucleotide identity, were also obtained from the Poole and Deal samples, indicating that RaTV is widespread in UK frog samples. Bioinformatic analyses of putative secondary structures in the 3'-UTR of RaTV indicated a potential exoribonuclease-resistant RNA (xrRNA) structure identified in flaviviruses and TABV. To examine this biochemically, we conducted an in vitro XRN1 digestion assay showing that RaTV likely forms a divergent but functionally homologous XRN1-resistant xrRNA.\nA conserved antigen induces respiratory Th17-mediated serotype-independent protection against pneumococcal superinfection\nAuthors: Liu, X.; Van Maele, L.; Matarazzo, L.; Soulard, D.; Alves Duarte da Silva, V.; de Bakker, V.; Denereaz, J.; Bock, F. P.; Taschner, M.; Gruber, S.; Nizet, V.; Sirard, J.-C.; Veening, J.-W.\nScore: 13.9, Published: 2023-07-07 DOI: 10.1101/2023.07.07.548156\nSeveral vaccines targeting bacterial pathogens show reduced efficacy in the context of intercurrent viral infection indicating a new vaccinology approach is required to protect against such superinfections. To find antigens for the human pathogen Streptococcus pneumoniae that are effective following influenza infection, we performed CRISPRi-seq in a murine model of superinfection and identified the highly conserved lafB gene as virulence factor. We show that LafB is a membrane-associated, intracellular protein that catalyzes the formation of galactosyl-glucosyl-diacylglycerol, a glycolipid we show is important for cell wall homeostasis. Respiratory vaccination with recombinant LafB, in contrast to subcutaneous vaccination, was highly protective against all serotypes in a murine model. In contrast to standard pneumococcal capsule-based conjugate vaccines, protection did not require LafB-specific antibodies but was dependent on airway CD4+ T helper 17 cells. Healthy human individuals can elicit LafB-specific immune responses, suggesting its merit as a universal pneumococcal vaccine antigen that remains effective following influenza infection.\nThe evolutionary history of hepaciviruses\nAuthors: Li, Y.; Ghafari, M.; Holbrook, A. J.; Boonen, I.; Amor, N. M. S.; Catalano, S.; Webster, J. P.; Li, Y.; Li, H.; Vergote, V.; Maes, P.; Chong, Y. L.; Laudisoit, A.; Baelo, P.; Ngoy, S.; Mbalitini, S. G.; Gembu, G.-C.; Akawa, P. M.; Gouy de Bellocq, J.; Leirs, H.; Verheyen, E.; Pybus, O. G.; Katzourakis, A.; Alagaili, A.; Gryseels, S.; Li, Y.; Suchard, M. A.; Bletsa, M.; Lemey, P.\nScore: 12.5, Published: 2023-06-30 DOI: 10.1101/2023.06.30.547218\nIn the search for natural reservoirs of hepatitis C virus (HCV), a broad diversity of non-human viruses within the Hepacivirus genus has been uncovered. However, the evolutionary dynamics that shaped the diversity and timescale of hepaciviruses evolution remain elusive. To gain further insights into the origins and evolution of this genus, we screened a large dataset of wild mammal samples (n = 1,672) from Africa and Asia, and generated 34 full-length hepacivirus genomes. Phylogenetic analysis of these data together with publicly available genomes emphasizes the importance of rodents as hepacivirus hosts and we identify 13 rodent species and 3 rodent genera (in Cricetidae and Muridae families) as novel hosts of hepaciviruses. Through co-phylogenetic analyses, we demonstrate that hepacivirus diversity has been affected by cross-species transmission events against the backdrop of detectable signal of virus-host co-divergence in the deep evolutionary history. Using a Bayesian phylogenetic multidimensional scaling approach, we explore the extent to which host relatedness and geographic distances have structured present-day hepacivirus diversity. Our results provide evidence for a substantial structuring of mammalian hepacivirus diversity by host as well as geography, with a somewhat more irregular diffusion process in geographic space. Finally, using a mechanistic model that accounts for substitution saturation, we provide the first formal estimates of the timescale of hepacivirus evolution and estimate the origin of the genus to be about 22 million years ago. Our results offer a comprehensive overview of the micro- and macroevolutionary processes that have shaped hepacivirus diversity and enhance our understanding of the long-term evolution of the Hepacivirus genus. SignificanceSince the discovery of Hepatitis C virus, the search for animal virus homologues has gained significant traction, opening up new opportunities to study their origins and long-term evolutionary dynamics. Capitalizing on a large-scale screening of wild mammals, and genomic sequencing, we expand the novel rodent host range of hepaciviruses and document further virus diversity. We infer a significant influence of frequent cross-species transmission as well as some signal for virus-host co-divergence, and find comparative host and geographic structure. We also provide the first formal estimates of the timescale of hepaciviruses indicating an origin of about 22 million years ago. Our study offers new insights in hepacivirus evolutionary dynamics with broadly applicable methods that can support future research in virus evolution.\nYeast-based heterologous production of the Colletochlorin family of fungal secondary metabolites\nAuthors: Geistodt-Kiener, A.; Totozafy, J. C.; Le Goff, G.; Vergne, J.; Sakai, K.; Ouazzani, J.; Mouille, G.; Viaud, M.; O'Connell, R. J.; Dallery, J.-F.\nScore: 8.7, Published: 2023-07-05 DOI: 10.1101/2023.07.05.547564\nTranscriptomic studies have revealed that fungal pathogens of plants activate the expression of numerous biosynthetic gene clusters (BGC) exclusively when in presence of a living host plant. The identification and structural elucidation of the corresponding secondary metabolites remain challenging. Here we adapted a polycistronic vector for efficient, seamless and cost-effective cloning of biosynthetic genes using in vivo assembly (also called transformation-assisted recombination) directly in Escherichia coli followed by heterologous expression in Saccharomyces cerevisiae. Two vectors were generated with different auto-inducible yeast promoters and selection markers. The effectiveness of these vectors was validated with fluorescent proteins. As a proof-of-principle, we applied our approach to the Colletochlorin family of molecules. These polyketide secondary metabolites were known from the phytopathogenic fungus Colletotrichum higginsianum but had never been linked to their biosynthetic genes. Considering the requirement for an halogenase, and by applying comparative genomics, we identified a BGC putatively involved in the biosynthesis of Colletochlorins in C. higginsianum. Following the expression of those genes in S. cerevisiae, we could identify the presence of the precursor Orsellinic acid, Colletochlorins and their non-chlorinated counterparts, the Colletorins. In conclusion, the polycistronic vectors described herein were adapted for the host S. cerevisiae and allowed to link the Colletochlorin compound family to their corresponding biosynthetic genes. This system will now enable the production and purification of infection-specific secondary metabolites of fungal phytopathogens. More widely, this system could be applied to any fungal BGC of interest.\nColonization during a key developmental window reveals microbiota-dependent shifts in growth and immunity during undernutrition\nAuthors: Serrano Matos, Y. A.; Cano, J.; Shafiq, H.; Williams, C.; Sunny, J.; Cowardin, C. A.\nScore: 7.5, Published: 2023-07-07 DOI: 10.1101/2023.07.07.547849\nChildhood undernutrition is a major global health challenge with devastating lifelong consequences. Linear growth stunting due to undernutrition has been linked to poor outcomes, and mothers who experience stunting are more likely to give birth to stunted children. Murine models that capture the intergenerational and multifactorial nature of undernutrition are critical to understanding the underlying biology of this disorder. Here we report a gnotobiotic mouse model of undernutrition using microbiota from human infants with healthy or stunted growth trajectories. Intergenerational transmission of microbiota from parents to offspring leads to the development of growth and immune features of undernutrition and enteropathy, including reduced linear growth, intestinal villus blunting and accumulation of intraepithelial lymphocytes. In contrast, colonization after weaning reduces sensitivity to detect changes driven by distinct microbial communities. Overall, these results suggest intergenerational colonization is a useful approach with which to investigate microbiota-dependent growth and immunity in early life.\n",
  "wordCount" : "2695",
  "inLanguage": "en",
  "datePublished": "2023-07-12T10:41:10Z",
  "dateModified": "2023-07-12T10:41:10Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta"><span>updated on July 12, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.547941">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.547941" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.547941">
        <p class="paperTitle">First Eurasian cases of SARS-CoV-2 seropositivity in a free-ranging urban population of wild fallow deer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.547941" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.547941" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Purves, K.; Brown, H.; Haverty, R.; Ryan, A.; Griffin, L.; McCormack, J.; O&#39;Reilly, S. R.; Mallon, P. W.; Gautier, V.; Cassidy, J. P.; Fabre, A.; Carr, M. J.; Gonzalez, G.; Ciuti, S.; Fletcher, N.</p>
        <p class="info">Score: 65.4, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.547941' target='https://doi.org/10.1101/2023.07.07.547941'> 10.1101/2023.07.07.547941</a></p>
        <p class="abstract">Severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) infects wildlife. Recent studies highlighted that variants of concern (VOC) may expand into novel animal reservoirs with the potential for reverse zoonosis. North American white-tailed deer are the only deer species in which SARS-CoV-2 has been documented, raising the question whether further reservoir species exist as new VOC emerge. Here, we report the first cases of deer SARS-CoV-2 seropositivity in Eurasia, in a city population of fallow deer in Dublin, Ireland. Deer were seronegative in 2020 (circulating variant in humans: Alpha), one animal was seropositive in 2021 (Delta variant), and 57% of animals tested in 2022 were seropositive (Omicron variant). Ex vivo, a clinical isolate of Omicron BA.1 infected fallow deer precision cut lung slice type-2 pneumocytes, also a major target of infection in human lungs. Our findings suggest a change in host tropism as new variants emerged in the human reservoir, highlighting the importance of continued wildlife disease monitoring and limiting human-wildlife contacts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.547807">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.547807" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.547807">
        <p class="paperTitle">Bam complex associated proteins in Escherichia coli are functionally linked to peptidoglycan biosynthesis, membrane fluidity and DNA replication</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.547807" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.547807" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bryant, J. A.; Staunton, K. A.; Doherty, H. M.; Alao, M. B.; Ma, X.; Morcinek-Orlowska, J.; Goodall, E. C. A.; Gray, J.; Milner, M.; Cole, J. A.; de Cogan, F.; Knowles, T. J.; Glinkowska, M. K.; Moradigaravand, D.; Henderson, I. R.; Banzhaf, M.</p>
        <p class="info">Score: 22.9, Published: 2023-07-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.547807' target='https://doi.org/10.1101/2023.07.05.547807'> 10.1101/2023.07.05.547807</a></p>
        <p class="abstract">Biogenesis of the bacterial outer membrane is key to survival and antibiotic resistance. Central to this is the {beta}-barrel assembly machine (Bam) complex and its associated chaperones, which are responsible for outer membrane protein (OMP) transport and insertion. The Escherichia coli Bam complex consists of two essential subunits, BamA and BamD, and three non-essential lipoproteins, BamB, BamC and BamE. Optimal Bam function is further dependent on the non-essential chaperones DegP, Skp and SurA. Despite intensive study, the specific function of these non-essential Bam-associated proteins remains unknown. Here, we analysed knockout strains for each gene by phenotypic screening, conservation analysis and high-throughput genetics. We reveal that Bam activity is affected by outer membrane lipid composition and that enterobacterial common antigen is essential in the absence of the chaperone SurA. We also show that components of peptidoglycan are conditionally essential with Bam accessory lipoproteins and that DNA replication is perturbed in the absence of BamB. Together, our data indicates potential mechanisms for coordination of OMP biogenesis with processes such as LPS and peptidoglycan biogenesis, and DNA replication.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.547920">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.547920" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.547920">
        <p class="paperTitle">NECing goes: flexibility of the herpesvirus nuclear egress complex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.547920" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.547920" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Prazak, V.; Mironova, Y.; Vasishtan, D.; Hagen, C.; Laugks, U.; Jensen, Y.; Sanders, S.; Heumann, J. M.; Bosse, J. B.; Klupp, B.; Mettenleiter, T. C.; Grange, M.; Grünewald, K.</p>
        <p class="info">Score: 20.7, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.547920' target='https://doi.org/10.1101/2023.07.07.547920'> 10.1101/2023.07.07.547920</a></p>
        <p class="abstract">The nuclear egress complex (NEC) allows herpesvirus capsids to escape from the nucleus without breaking the nuclear envelope barrier. It assembles into a lattice on the inner nuclear membrane enveloping newly assembled nucleocapsids, which bud into the perinuclear space. The primary virion envelope subsequently fuses with the outer nuclear membrane, releasing the capsid into the cytosol. Here we interrogated the NEC in the context of intact cells infected with pseudorabies or herpes simplex virus using focused-ion beam milling and electron cryo-tomography. We determined the structure of NEC in different conformations and show that it consists of a flexible hexameric lattice that generates curvature through a combination of ordered and disordered domains. After interrogating the intermediate stages of capsid formation, we show that capsid vertex binding may initiate envelopment but does not directly induce curvature formation. These data and many examples of the intermediate stages of nuclear egress paint a detailed holistic view of a versatile transport system.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547184">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547184" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547184">
        <p class="paperTitle">Bridges instead of boats? The Mla system of diderm Firmicute Veillonella parvula reveals an ancestral transenvelope core of phospholipid trafficking</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547184" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547184" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: BELOIN, C.; Grasekamp, K. P.; Beaud, B.; Taib, N.; Audrain, B.; Bardiaux, B.; Rossez, Y.; IZADI-PRUNEYRE, N.; Lejeune, M.; Trivelli, X.; Chouit, Z.; Guerardel, Y.; GHIGO, J.-M.; Gribaldo, S.</p>
        <p class="info">Score: 21.2, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547184' target='https://doi.org/10.1101/2023.06.30.547184'> 10.1101/2023.06.30.547184</a></p>
        <p class="abstract">Despite extensive characterisation of envelope biogenesis systems in diderm bacteria, glycerophospholipid (GPL) trafficking remains poorly understood, and has only been studied in a handful of model species. Within the Proteobacteria, the maintenance of lipid asymmetry (Mla) system facilitates retrograde GPL trafficking via six proteins, MlaA-F. GPLs are extracted from the outer leaflet of the outer membrane by the lipoprotein MlaA which associates with porin trimers, then shipped through the periplasmic space by the chaperone MlaC, which finally delivers GPLs to the inner membrane complex formed by MlaBDEF. Here, we investigate GPL trafficking in Veillonella parvula, a diderm member of the Firmicutes which encodes an Mla system devoid of MlaA and MlaC. V. parvula {Delta}mla mutants display phenotypes characteristic of disrupted lipid asymmetry such as hypervesiculation and detergent hypersensitivity, and lipid content analysis from outer membrane vesicles reveals an enrichment for the major lipid component phosphatidylethanolamine. Interestingly, suppressor analysis identifies mutations in tamB that rescue detergent hypersensitivity and hypervesiculation of {Delta}mla strains, supporting the involvement of these two systems in antagonistic GPL trafficking functions across diverse bacterial lineages. A combination of structural modeling and subcellular localisation assays shows that MlaDVp is longer than in classical diderm models and forms a transenvelope bridge, encoding both an inner membrane-localised MCE domain and an outer membrane {beta}-barrel. These results strongly suggest that V. parvula possesses a minimal Mla system for GPL trafficking, replacing the need for chaperones and outer membrane lipoproteins by directly connecting the two membranes. Finally, phylogenomic analysis indicates that this MlaEFD self-contained architecture is widely distributed in diderm bacteria and most likely represents the ancestral functional core of the Mla system, which subsequently increased in complexity in Proteobacteria and closely related phyla following the emergence of MlaABC. Our work broadens the diversity of current models of GPL trafficking in diderm bacteria, challenging the paradigm set by classical models and shedding light on the evolution of a crucial system in the biogenesis and maintenance of the bacterial outer membrane.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547502">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547502" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547502">
        <p class="paperTitle">Predicting bacterial interaction outcomes from monoculture growth and supernatant assays</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547502" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547502" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schmitz, D. A.; Wechsler, T.; Mignot, I.; Kuemmerli, R.</p>
        <p class="info">Score: 17.9, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547502' target='https://doi.org/10.1101/2023.07.03.547502'> 10.1101/2023.07.03.547502</a></p>
        <p class="abstract">How to derive principles of community dynamics and stability is a central question in microbial ecology. To answer this, bottom-up experimental approaches, in which a small number of bacterial species are mixed, have become popular. However, experimental setups are typically limited because species are difficult to distinguish in mixed cultures and co-culture experiments are labor-intensive. Here, we use a community of four bacterial species to show that information from monoculture growth and inhibitory effects caused by secreted compounds can be combined to reliably predict pairwise species interactions in co-cultures. Specifically, integrative parameters from growth curves allow to build a competitive rank order, which is then adjusted using inhibitory compound effects from supernatant assays. While our procedure worked for two media examined, we observed differences in species rank orders between media. We further used computer simulations, parameterized with our empirical data, to show that higher-order species interactions largely follow the dynamics predicted from pairwise interactions with one important exception. The impact of inhibitory compounds was reduced in higher-order communities because these compounds are spread across multiple species, thereby diluting their effects. Altogether, our results lead to the formulation of three simple rules of how monoculture growth and supernatant assay data can be combined to establish competitive species rank orders in bacterial communities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.547906">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.547906" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.547906">
        <p class="paperTitle">A novel tamanavirus (Flaviviridae) of the European common frog (Rana temporaria) encodes a divergent class 1b XRN1-resistant RNA element.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.547906" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.547906" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Parry, R. H.; Slonchak, A.; Campbell, L. J.; Newton, N.; Debat, H.; Gifford, R. J.; Khromykh, A.</p>
        <p class="info">Score: 15.3, Published: 2023-07-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.547906' target='https://doi.org/10.1101/2023.07.06.547906'> 10.1101/2023.07.06.547906</a></p>
        <p class="abstract">Flavivirids are small, enveloped, positive-sense RNA viruses from the Flaviviridae family with genomes between [~]9-13kb. Metatranscriptomic analyses of metazoan organisms have revealed a diversity of flavivirus-like or flavivirid viral sequences in fish and marine invertebrate groups. To date, however, no flavivirus-like or flavivirid has been identified in amphibians. To remedy this, we investigated the virome of the European common frog (Rana temporaria) in the United Kingdom, utilising high-throughput sequencing at six catch locations. De novo assembly revealed a coding-complete virus contig of a novel flavivirid [~]11.2kb in length. The virus encodes a single open reading frame of 3456 amino acids and 5 and 3 untranslated regions (UTRs) of 227 and 666nt, respectively. We named this virus Rana tamanavirus (RaTV), as BLASTp analysis of the polyprotein showed the closest relationships to Tamana bat virus (TABV) and Cyclopterus lumpus virus from Pteronotus parnellii and Cyclopterus lumpus, respectively. Phylogenetic analysis of the RaTV polyprotein compared to Flavivirus and Flavivirus-like members indicated that RaTV was sufficiently divergent and basal to the vertebrate Tamanavirus clade. In addition to the Mitcham strain, partial but divergent RaTV, 95.64-97.39% pairwise nucleotide identity, were also obtained from the Poole and Deal samples, indicating that RaTV is widespread in UK frog samples. Bioinformatic analyses of putative secondary structures in the 3&#39;-UTR of RaTV indicated a potential exoribonuclease-resistant RNA (xrRNA) structure identified in flaviviruses and TABV. To examine this biochemically, we conducted an in vitro XRN1 digestion assay showing that RaTV likely forms a divergent but functionally homologous XRN1-resistant xrRNA.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548156">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548156" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548156">
        <p class="paperTitle">A conserved antigen induces respiratory Th17-mediated serotype-independent protection against pneumococcal superinfection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548156" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548156" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, X.; Van Maele, L.; Matarazzo, L.; Soulard, D.; Alves Duarte da Silva, V.; de Bakker, V.; Denereaz, J.; Bock, F. P.; Taschner, M.; Gruber, S.; Nizet, V.; Sirard, J.-C.; Veening, J.-W.</p>
        <p class="info">Score: 13.9, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548156' target='https://doi.org/10.1101/2023.07.07.548156'> 10.1101/2023.07.07.548156</a></p>
        <p class="abstract">Several vaccines targeting bacterial pathogens show reduced efficacy in the context of intercurrent viral infection indicating a new vaccinology approach is required to protect against such superinfections. To find antigens for the human pathogen Streptococcus pneumoniae that are effective following influenza infection, we performed CRISPRi-seq in a murine model of superinfection and identified the highly conserved lafB gene as virulence factor. We show that LafB is a membrane-associated, intracellular protein that catalyzes the formation of galactosyl-glucosyl-diacylglycerol, a glycolipid we show is important for cell wall homeostasis. Respiratory vaccination with recombinant LafB, in contrast to subcutaneous vaccination, was highly protective against all serotypes in a murine model. In contrast to standard pneumococcal capsule-based conjugate vaccines, protection did not require LafB-specific antibodies but was dependent on airway CD4&#43; T helper 17 cells. Healthy human individuals can elicit LafB-specific immune responses, suggesting its merit as a universal pneumococcal vaccine antigen that remains effective following influenza infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547218">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547218" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547218">
        <p class="paperTitle">The evolutionary history of hepaciviruses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547218" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547218" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, Y.; Ghafari, M.; Holbrook, A. J.; Boonen, I.; Amor, N. M. S.; Catalano, S.; Webster, J. P.; Li, Y.; Li, H.; Vergote, V.; Maes, P.; Chong, Y. L.; Laudisoit, A.; Baelo, P.; Ngoy, S.; Mbalitini, S. G.; Gembu, G.-C.; Akawa, P. M.; Gouy de Bellocq, J.; Leirs, H.; Verheyen, E.; Pybus, O. G.; Katzourakis, A.; Alagaili, A.; Gryseels, S.; Li, Y.; Suchard, M. A.; Bletsa, M.; Lemey, P.</p>
        <p class="info">Score: 12.5, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547218' target='https://doi.org/10.1101/2023.06.30.547218'> 10.1101/2023.06.30.547218</a></p>
        <p class="abstract">In the search for natural reservoirs of hepatitis C virus (HCV), a broad diversity of non-human viruses within the Hepacivirus genus has been uncovered. However, the evolutionary dynamics that shaped the diversity and timescale of hepaciviruses evolution remain elusive. To gain further insights into the origins and evolution of this genus, we screened a large dataset of wild mammal samples (n = 1,672) from Africa and Asia, and generated 34 full-length hepacivirus genomes. Phylogenetic analysis of these data together with publicly available genomes emphasizes the importance of rodents as hepacivirus hosts and we identify 13 rodent species and 3 rodent genera (in Cricetidae and Muridae families) as novel hosts of hepaciviruses. Through co-phylogenetic analyses, we demonstrate that hepacivirus diversity has been affected by cross-species transmission events against the backdrop of detectable signal of virus-host co-divergence in the deep evolutionary history. Using a Bayesian phylogenetic multidimensional scaling approach, we explore the extent to which host relatedness and geographic distances have structured present-day hepacivirus diversity. Our results provide evidence for a substantial structuring of mammalian hepacivirus diversity by host as well as geography, with a somewhat more irregular diffusion process in geographic space. Finally, using a mechanistic model that accounts for substitution saturation, we provide the first formal estimates of the timescale of hepacivirus evolution and estimate the origin of the genus to be about 22 million years ago. Our results offer a comprehensive overview of the micro- and macroevolutionary processes that have shaped hepacivirus diversity and enhance our understanding of the long-term evolution of the Hepacivirus genus.

SignificanceSince the discovery of Hepatitis C virus, the search for animal virus homologues has gained significant traction, opening up new opportunities to study their origins and long-term evolutionary dynamics. Capitalizing on a large-scale screening of wild mammals, and genomic sequencing, we expand the novel rodent host range of hepaciviruses and document further virus diversity. We infer a significant influence of frequent cross-species transmission as well as some signal for virus-host co-divergence, and find comparative host and geographic structure. We also provide the first formal estimates of the timescale of hepaciviruses indicating an origin of about 22 million years ago. Our study offers new insights in hepacivirus evolutionary dynamics with broadly applicable methods that can support future research in virus evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.547564">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.547564" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.547564">
        <p class="paperTitle">Yeast-based heterologous production of the Colletochlorin family of fungal secondary metabolites</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.547564" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.547564" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Geistodt-Kiener, A.; Totozafy, J. C.; Le Goff, G.; Vergne, J.; Sakai, K.; Ouazzani, J.; Mouille, G.; Viaud, M.; O&#39;Connell, R. J.; Dallery, J.-F.</p>
        <p class="info">Score: 8.7, Published: 2023-07-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.547564' target='https://doi.org/10.1101/2023.07.05.547564'> 10.1101/2023.07.05.547564</a></p>
        <p class="abstract">Transcriptomic studies have revealed that fungal pathogens of plants activate the expression of numerous biosynthetic gene clusters (BGC) exclusively when in presence of a living host plant. The identification and structural elucidation of the corresponding secondary metabolites remain challenging. Here we adapted a polycistronic vector for efficient, seamless and cost-effective cloning of biosynthetic genes using in vivo assembly (also called transformation-assisted recombination) directly in Escherichia coli followed by heterologous expression in Saccharomyces cerevisiae. Two vectors were generated with different auto-inducible yeast promoters and selection markers. The effectiveness of these vectors was validated with fluorescent proteins. As a proof-of-principle, we applied our approach to the Colletochlorin family of molecules. These polyketide secondary metabolites were known from the phytopathogenic fungus Colletotrichum higginsianum but had never been linked to their biosynthetic genes. Considering the requirement for an halogenase, and by applying comparative genomics, we identified a BGC putatively involved in the biosynthesis of Colletochlorins in C. higginsianum. Following the expression of those genes in S. cerevisiae, we could identify the presence of the precursor Orsellinic acid, Colletochlorins and their non-chlorinated counterparts, the Colletorins. In conclusion, the polycistronic vectors described herein were adapted for the host S. cerevisiae and allowed to link the Colletochlorin compound family to their corresponding biosynthetic genes. This system will now enable the production and purification of infection-specific secondary metabolites of fungal phytopathogens. More widely, this system could be applied to any fungal BGC of interest.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.547849">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.547849" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.547849">
        <p class="paperTitle">Colonization during a key developmental window reveals microbiota-dependent shifts in growth and immunity during undernutrition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.547849" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.547849" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Serrano Matos, Y. A.; Cano, J.; Shafiq, H.; Williams, C.; Sunny, J.; Cowardin, C. A.</p>
        <p class="info">Score: 7.5, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.547849' target='https://doi.org/10.1101/2023.07.07.547849'> 10.1101/2023.07.07.547849</a></p>
        <p class="abstract">Childhood undernutrition is a major global health challenge with devastating lifelong consequences. Linear growth stunting due to undernutrition has been linked to poor outcomes, and mothers who experience stunting are more likely to give birth to stunted children. Murine models that capture the intergenerational and multifactorial nature of undernutrition are critical to understanding the underlying biology of this disorder. Here we report a gnotobiotic mouse model of undernutrition using microbiota from human infants with healthy or stunted growth trajectories. Intergenerational transmission of microbiota from parents to offspring leads to the development of growth and immune features of undernutrition and enteropathy, including reduced linear growth, intestinal villus blunting and accumulation of intraepithelial lymphocytes. In contrast, colonization after weaning reduces sensitivity to detect changes driven by distinct microbial communities. Overall, these results suggest intergenerational colonization is a useful approach with which to investigate microbiota-dependent growth and immunity in early life.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
