<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="An infection and pathogenesis mouse model of SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR)
Authors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.
Score: 4150.5, Published: 2024-01-21 DOI: 10.1101/2024.01.03.574008
SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) is highly attenuated, but can cause mortality in a specifically designed human ACE2-transgenic mouse model, making it an invaluable surrogate model for evaluating the efficacy of drugs and vaccines against SARS-CoV-2.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="An infection and pathogenesis mouse model of SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR)
Authors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.
Score: 4150.5, Published: 2024-01-21 DOI: 10.1101/2024.01.03.574008
SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) is highly attenuated, but can cause mortality in a specifically designed human ACE2-transgenic mouse model, making it an invaluable surrogate model for evaluating the efficacy of drugs and vaccines against SARS-CoV-2." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-24T10:40:04+00:00" />
<meta property="article:modified_time" content="2024-01-24T10:40:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="An infection and pathogenesis mouse model of SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR)
Authors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.
Score: 4150.5, Published: 2024-01-21 DOI: 10.1101/2024.01.03.574008
SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) is highly attenuated, but can cause mortality in a specifically designed human ACE2-transgenic mouse model, making it an invaluable surrogate model for evaluating the efficacy of drugs and vaccines against SARS-CoV-2."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "An infection and pathogenesis mouse model of SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR)\nAuthors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.\nScore: 4150.5, Published: 2024-01-21 DOI: 10.1101/2024.01.03.574008\nSARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) is highly attenuated, but can cause mortality in a specifically designed human ACE2-transgenic mouse model, making it an invaluable surrogate model for evaluating the efficacy of drugs and vaccines against SARS-CoV-2.",
  "keywords": [
    
  ],
  "articleBody": " An infection and pathogenesis mouse model of SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR)\nAuthors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.\nScore: 4150.5, Published: 2024-01-21 DOI: 10.1101/2024.01.03.574008\nSARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) is highly attenuated, but can cause mortality in a specifically designed human ACE2-transgenic mouse model, making it an invaluable surrogate model for evaluating the efficacy of drugs and vaccines against SARS-CoV-2.\nViroid-like colonists of human microbiomes\nAuthors: Zheludev, I. N.; Edgar, R. C.; Galiano, M. J. L.; De la Pena, M.; Babaian, A.; Bhatt, A. S.; Fire, A. Z.\nScore: 77.8, Published: 2024-01-21 DOI: 10.1101/2024.01.20.576352\nHere, we describe the \"Obelisks,\" a previously unrecognised class of viroid-like elements that we first identified in human gut metatranscriptomic data. \"Obelisks\" share several properties: (i) apparently circular RNA [~]1kb genome assemblies, (ii) predicted rod-like secondary structures encompassing the entire genome, and (iii) open reading frames coding for a novel protein superfamily, which we call the \"Oblins\". We find that Obelisks form their own distinct phylogenetic group with no detectable sequence or structural similarity to known biological agents. Further, Obelisks are prevalent in tested human microbiome metatranscriptomes with representatives detected in [~]7% of analysed stool metatranscriptomes (29/440) and in [~]50% of analysed oral metatranscriptomes (17/32). Obelisk compositions appear to differ between the anatomic sites and are capable of persisting in individuals, with continued presence over \u003e300 days observed in one case. Large scale searches identified 29,959 Obelisks (clustered at 90% nucleotide identity), with examples from all seven continents and in diverse ecological niches. From this search, a subset of Obelisks are identified to code for Obelisk-specific variants of the hammerhead type-III self-cleaving ribozyme. Lastly, we identified one case of a bacterial species (Streptococcus sanguinis) in which a subset of defined laboratory strains harboured a specific Obelisk RNA population. As such, Obelisks comprise a class of diverse RNAs that have colonised, and gone unnoticed in, human, and global microbiomes.\nA treasure trove of 1,034 actinomycete genomes\nAuthors: Jorgensen, T. S.; Mohite, O.; Sterndorff, E. B.; Alvarez-Arevalo, M.; Blin, K.; Booth, T. J.; Charusanti, P.; Faurdal, D.; Hansen, T. O.; Nuhamunada, M. J.; Mourched, A.-S.; Palsson, B. O.; Weber, T.\nScore: 47.1, Published: 2024-01-16 DOI: 10.1101/2024.01.16.574955\nFilamentous Actinobacteria, recently renamed Actinomycetia, are the most prolific source of microbial bioactive natural products. Studies on biosynthetic gene clusters benefit from or require chromosome-level assemblies. Here, we provide DNA sequences from more than 1,000 isolates: 881 complete genomes and 153 near-complete genomes, representing 28 genera and 389 species, including 244 likely novel species. All genomes are from filamentous isolates of the class Actinomycetia from the NBC culture collection. The largest genus is Streptomyces with 886 genomes including 742 complete assemblies. We use this data to show that analysis of complete genomes can bring biological understanding not previously derived from more fragmented sequences or less systematic datasets. We document the central and structured location of core genes and distal location of specialized metabolite biosynthetic gene clusters and duplicate core genes on the linear Streptomyces chromosome, and analyze the content and length of the terminal inverted repeats which are characteristic for Streptomyces. We then analyze the diversity of trans-AT polyketide synthase biosynthetic gene clusters, which encodes the machinery of a biotechnologically highly interesting compound class. These insights have both ecological and biotechnological implications in understanding the importance of high quality genomic resources and the complex role synteny plays in Actinomycetia biology.\nTemperature-dependent Spike-ACE2 interaction of Omicron subvariants is associated with viral transmission\nAuthors: Benlarbi, M.; Ding, S.; Belanger, E.; Tauzin, A.; Poujol, R.; Medjahed, H.; El Ferri, O.; Bo, Y.; Bourassa, C.; Hussin, J.; Fafard, J.; Pazgier, M.; Levade, I.; Abrams, C.; Cote, M.; Finzi, A.\nScore: 43.2, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576353\nThe continued evolution of SARS-CoV-2 requires persistent monitoring of its subvariants. Omicron subvariants are responsible for the vast majority of SARS-CoV-2 infections worldwide, with XBB and BA.2.86 sublineages representing more than 90% of circulating strains as of January 2024. In this study, we characterized the functional properties of Spike glycoproteins from BA.2.75, CH.1.1, DV.7.1, BA.4/5, BQ.1.1, XBB, XBB.1, XBB.1.16, XBB.1.5, FD.1.1, EG.5.1, HK.3 BA.2.86 and JN.1. We tested their capacity to evade plasma-mediated recognition and neutralization, ACE2 binding, their susceptibility to cold inactivation, Spike processing, as well as the impact of temperature on Spike-ACE2 interaction. We found that compared to the early wild-type (D614G) strain, most Omicron subvariants Spike glycoproteins evolved to escape recognition and neutralization by plasma from individuals who received a fifth dose of bivalent (BA.1 or BA.4/5) mRNA vaccine and improve ACE2 binding, particularly at low temperatures. Moreover, BA.2.86 had the best affinity for ACE2 at all temperatures tested. We found that Omicron subvariants Spike processing is associated with their susceptibility to cold inactivation. Intriguingly, we found that Spike-ACE2 binding at low temperature was significantly associated with growth rates of Omicron subvariants in humans. Overall, we report that Spikes from newly emerged Omicron subvariants are relatively more stable and resistant to plasma-mediated neutralization, present improved affinity for ACE2 which is associated, particularly at low temperatures, with their growth rates.\nBirth of new protein folds and functions in the virome\nAuthors: Nomburg, J.; Price, N.; Doudna, J. A.\nScore: 24.5, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576744\nRapid virus evolution generates proteins essential to infectivity and replication but with unknown function due to extreme sequence divergence. Using a database of 67,715 newly predicted protein structures from 4,463 eukaryotic viral species, we found that 62% of viral proteins are evolutionarily young and lack homologs in the Alphafold database. Among the 38% of more ancient viral proteins, many have non-viral structural homologs that revealed surprising similarities between human pathogens and their eukaryotic hosts. Structural comparisons suggested putative functions for \u003e25% of unannotated viral proteins, including those with roles in the evasion of innate immunity. In particular, RNA ligase T- (ligT) like phosphodiesterases were found to resemble phage-encoded proteins that hydrolyze the host immune-activating cyclic dinucleotides 3'3' and 2'3' cyclic G-A monophosphate (cGAMP). Experimental analysis showed that ligT homologs encoded by avian poxviruses likewise hydrolyze 2'3' cGAMP, showing that ligT-mediated targeting of cGAMP is an evolutionarily conserved mechanism of immune evasion present in both bacteriophage and eukaryotic viruses. Together, the viral protein structural database and analytics presented here afford new opportunities to identify mechanisms of virus-host interactions that are common across the virome.\nViral entry is a weak barrier to zoonosis\nAuthors: Dufloo, J.; Andreu-Moreno, I.; Valero-Rello, A.; Sanjuan, R.\nScore: 22.4, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576693\nRecent advances in viral metagenomics have led to the discovery of many mammalian viruses, but experimental tests to determine whether they pose a threat to humans are largely lacking. A first step for a virus to cross the species barrier is to penetrate host cells. Here, we use gene synthesis and viral pseudotyping to experimentally test the ability of viral receptor-binding proteins (RBPs) from \u003e100 enveloped RNA viruses to mediate entry into human cells. Analysis of thousands of RBP-cell pairs demonstrated such ability for most viruses, with significant variation among the 14 viral families studied. Comparison of RBP-mediated infectivity with cellular gene expression data showed that viral entry is often not limited by the presence of a receptor and revealed the contribution of additional host factors. Our results prove the weakness of interspecies barriers at the early stages of infection and identify molecular interactions that enable viral zoonosis.\nHalofilins as Emerging Bactofilin Families of Archaeal Cell Shape Plasticity Orchestrators\nAuthors: Curtis, Z.; Escudeiro, P.; Mallon, J. L.; Leland, O.; Rados, T.; Dodge, A.; Andre, K.; Kwak, J.; Yun, K.; Pohlschroder, M.; Alva, V.; Bisson, A.\nScore: 18.4, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576759\nBactofilins are rigid, non-polar bacterial cytoskeletal filaments that link cellular processes to specific curvatures of the cytoplasmic membrane. Although homologs of bactofilins have been identified in archaea and eukaryotes, functional studies have remained confined to bacterial systems. Here, we characterized representatives of two families of archaeal bactofilins from the pleomorphic archaeon Haloferax volcanii, halofilin A (HalA) and halofilin B (HalB). Unlike bacterial bactofilins, HalA polymerizes into polar filaments in vivo at positive membrane curvature, whereas HalB forms more static foci and accumulates in areas of local negative curvatures on the outer cell surface. Combining gene deletions, super-resolution, and single-cell microscopy showed that halofilins are critical in maintaining H. volcanii cell integrity during shape transition from disk (sessile) to rod (motile). Morphological defects in {Delta}halA primarily affected rod-shaped cells from accumulating highly positive curvatures. Conversely, disk-shaped cells were exclusively affected by halB deletion, showing a decrease in positive and negative curvatures, resulting in flatter cells. Furthermore, while {Delta}halA and {Delta}halB cells displayed lower cell division placement precision, morphological defects arose predominantly during the disk-to-rod shape remodeling. We propose halofilins provide mechanical scaffolding, dynamically coupling the cytoplasmic membrane and the S-layer. We speculate that HalA filaments support rods under low S-layer lipidation (flexible, fast membrane diffusion). In contrast, HalB connects the S-layer to negative curvatures in disks under high lipidation levels (rigid, slow membrane diffusion).\nNRX-101 (D-Cycloserine + Lurasidone) , a Qualified Infectious Disease Product, is Active Against Drug-Resistant Urinary Pathogens In Vitro\nAuthors: Sapko, M. T.; Manyak, M.; Panicucci, R.; Javitt, J. C.\nScore: 58.5, Published: 2024-01-14 DOI: 10.1101/2024.01.14.575572\nBackgroundD-Cycloserine (DCS) is a broad-spectrum antibiotic that is currently FDA-approved to treat tuberculosis (TB) disease and urinary tract infection. Despite numerous reports showing good clinical efficacy, DCS fell out of favor as a UTI treatment because of its propensity to cause side effects. NRX-101, a fixed dose combination of DCS and lurasidone, has been awarded Qualified Infectious Disease Product and Fast Track Designation by the US Food and Drug Administration and is being developed for various CNS indications because of its unique synergistic effect; each component mitigates side effects of the other. MethodsIn this study, we tested NRX-101 against the urinary tract pathogens E. coli, P. aeruginosa, K. pneumoniae, and A. baumannii in Mueller Hinton broth (caMHB) and artificial urine media (AUM). Several strains were multidrug resistant. Test compounds were serially diluted in broth/media. Minimum inhibitory concentration (MIC) was defined as the lowest concentration of test compound at which no bacterial growth was observed. ResultsDCS exhibited antibacterial efficacy against all strains tested while lurasidone did not appreciably affect the antibacterial action of DCS in vitro. In AUM, the MICs ranged from 128 to 512 mcg/ml for both DCS and NRX-101. In caMHB, MICs ranged from 8 to 1024 mcg/ml for NRX-101 and 32 to 512 mcg/ml for DCS alone. ConclusionsOur data confirm that DCS as antibacterial activity against reference and drug-resistant urinary pathogens. Furthermore, lurasidone does not interfere with DCSs anti-microbial action in vitro. These results support the clinical development of NRX-101 as a treatment for complicated urinary tract infection.\nNitrous oxide respiration in acidophilic methanotrophs\nAuthors: Awala, S. I.; Gwak, J.-H.; Kim, Y.; Jung, M.-Y.; Dunfield, P. F.; Wagner, M.; Rhee, S.-K.\nScore: 13.9, Published: 2024-01-16 DOI: 10.1101/2024.01.15.574570\nMethanotrophic bacteria mitigate methane (CH4) emissions from natural environments. Although aerobic methanotrophs are considered strict aerobes, they are often highly abundant in extremely hypoxic and even anoxic environments. Despite the presence of denitrification genes, it remains to be verified whether denitrification contributes to their growth. Here, we revealed that two acidophilic methanotrophs encoding N2O reductase (clade I and type II nosZ, respectively): Methylocella tundrae T4 and Methylacidiphilum caldifontis IT6, respired N2O and grew anaerobically on diverse non-methane substrates, including methanol, C-C substrates, and hydrogen. However, NO3 - and NO2 - could be reduced during methanol oxidation in Methylocella tundrae T4 and Methylocella silvestris BL2 without significantly increasing cell biomass. The lack of growth on methanol + NO3- or NO2- was likely due to the production of toxic reactive nitrogen species and C1 metabolites. However, the oxidation of pyruvate, a C3 electron donor, combined with NO3- or NO2- reduction resulted in anaerobic growth of Methylocella tundrae T4 and Methylocella silvestris BL2. In the extreme acidophile, Methylacidiphilum caldifontis IT6, N2O respiration supported cell growth at an extremely acidic pH of 2.0. In Methylocella tundrae T4, simultaneous consumption of N2O and CH4 was observed in suboxic conditions, both in microrespirometry and growth experiments, indicating the robustness of its N2O reductase activity in the presence of O2. Furthermore, CH4 oxidation per O2 reduced in O2-limiting conditions increased when N2O was added, indicating that cells of T4 can direct more O2 towards methane monooxygenase when respiring N2O as a terminal electron acceptor. Upregulation of nosZ and distinct repertories of methanol dehydrogenase-encoding genes (XoxF- and MxaFI-type) in Methylocella tundrae T4 cells grown anaerobically on methanol with N2O as the sole electron acceptor indicated adaptation mechanisms to anoxia. Our findings demonstrate that some methanotrophs can respire N2O independently or in tandem with O2, significantly expanding their potential ecological niche and paving the way for enhanced growth and survival in dynamic environments. This metabolic capability has application potential for simultaneously mitigating the emissions of the key greenhouse gases, CO2, CH4, and N2O, from natural and engineered environments.\nGut environmental factors explain variations in the gut microbiome composition and metabolism within and between healthy adults\nAuthors: Prochazkova, N.; Laursen, M. F.; La Barbera, G.; Tsekitsidi, E.; Joergensen, M. S.; Rasmussen, M. A.; Raes, J.; Dragsted, L. O.; Licht, T. R.; Roager, H. M.\nScore: 12.7, Published: 2024-01-23 DOI: 10.1101/2024.01.23.574598\nThe human gut microbiome is highly personal. However, the contribution of the gut environment to variations in the gut microbiome remains elusive. Here, we profiled the gut microbiome composition and metabolism over 9 consecutive days in 61 healthy adults and assessed gut environmental factors including segmental transit time and pH using a wireless motility capsule. Day-to-day fluctuations in gut environmental factors as well as segmental transit time and pH varied substantially between individuals. The gut environment explained more variations in gut microbiome and urine metabolome than dietary macronutrients or personal characteristics. Finally, we identified coffee-derived metabolites to be negatively correlated with small intestinal transit time and several microbial metabolites to be associated with colonic transit time including urinary proteolytic markers, faecal short-chain fatty acids, and breath methane. Our work suggests that the gut environment is key for understanding the individuality of the human gut microbiome composition and metabolism.\n",
  "wordCount" : "2340",
  "inLanguage": "en",
  "datePublished": "2024-01-24T10:40:04Z",
  "dateModified": "2024-01-24T10:40:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 24, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574008">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574008" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574008">
        <p class="paperTitle">An infection and pathogenesis mouse model of SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574008" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574008" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.</p>
        <p class="info">Score: 4150.5, Published: 2024-01-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574008' target='https://doi.org/10.1101/2024.01.03.574008'> 10.1101/2024.01.03.574008</a></p>
        <p class="abstract">SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) is highly attenuated, but can cause mortality in a specifically designed human ACE2-transgenic mouse model, making it an invaluable surrogate model for evaluating the efficacy of drugs and vaccines against SARS-CoV-2.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576352">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576352" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576352">
        <p class="paperTitle">Viroid-like colonists of human microbiomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576352" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576352" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zheludev, I. N.; Edgar, R. C.; Galiano, M. J. L.; De la Pena, M.; Babaian, A.; Bhatt, A. S.; Fire, A. Z.</p>
        <p class="info">Score: 77.8, Published: 2024-01-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576352' target='https://doi.org/10.1101/2024.01.20.576352'> 10.1101/2024.01.20.576352</a></p>
        <p class="abstract">Here, we describe the &#34;Obelisks,&#34; a previously unrecognised class of viroid-like elements that we first identified in human gut metatranscriptomic data. &#34;Obelisks&#34; share several properties: (i) apparently circular RNA [~]1kb genome assemblies, (ii) predicted rod-like secondary structures encompassing the entire genome, and (iii) open reading frames coding for a novel protein superfamily, which we call the &#34;Oblins&#34;. We find that Obelisks form their own distinct phylogenetic group with no detectable sequence or structural similarity to known biological agents. Further, Obelisks are prevalent in tested human microbiome metatranscriptomes with representatives detected in [~]7% of analysed stool metatranscriptomes (29/440) and in [~]50% of analysed oral metatranscriptomes (17/32). Obelisk compositions appear to differ between the anatomic sites and are capable of persisting in individuals, with continued presence over &gt;300 days observed in one case. Large scale searches identified 29,959 Obelisks (clustered at 90% nucleotide identity), with examples from all seven continents and in diverse ecological niches. From this search, a subset of Obelisks are identified to code for Obelisk-specific variants of the hammerhead type-III self-cleaving ribozyme. Lastly, we identified one case of a bacterial species (Streptococcus sanguinis) in which a subset of defined laboratory strains harboured a specific Obelisk RNA population. As such, Obelisks comprise a class of diverse RNAs that have colonised, and gone unnoticed in, human, and global microbiomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.16.574955">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.16.574955" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.16.574955">
        <p class="paperTitle">A treasure trove of 1,034 actinomycete genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.16.574955" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.16.574955" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jorgensen, T. S.; Mohite, O.; Sterndorff, E. B.; Alvarez-Arevalo, M.; Blin, K.; Booth, T. J.; Charusanti, P.; Faurdal, D.; Hansen, T. O.; Nuhamunada, M. J.; Mourched, A.-S.; Palsson, B. O.; Weber, T.</p>
        <p class="info">Score: 47.1, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.16.574955' target='https://doi.org/10.1101/2024.01.16.574955'> 10.1101/2024.01.16.574955</a></p>
        <p class="abstract">Filamentous Actinobacteria, recently renamed Actinomycetia, are the most prolific source of microbial bioactive natural products. Studies on biosynthetic gene clusters benefit from or require chromosome-level assemblies. Here, we provide DNA sequences from more than 1,000 isolates: 881 complete genomes and 153 near-complete genomes, representing 28 genera and 389 species, including 244 likely novel species. All genomes are from filamentous isolates of the class Actinomycetia from the NBC culture collection. The largest genus is Streptomyces with 886 genomes including 742 complete assemblies. We use this data to show that analysis of complete genomes can bring biological understanding not previously derived from more fragmented sequences or less systematic datasets. We document the central and structured location of core genes and distal location of specialized metabolite biosynthetic gene clusters and duplicate core genes on the linear Streptomyces chromosome, and analyze the content and length of the terminal inverted repeats which are characteristic for Streptomyces. We then analyze the diversity of trans-AT polyketide synthase biosynthetic gene clusters, which encodes the machinery of a biotechnologically highly interesting compound class. These insights have both ecological and biotechnological implications in understanding the importance of high quality genomic resources and the complex role synteny plays in Actinomycetia biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576353">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576353" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576353">
        <p class="paperTitle">Temperature-dependent Spike-ACE2 interaction of Omicron subvariants is associated with viral transmission</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576353" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576353" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Benlarbi, M.; Ding, S.; Belanger, E.; Tauzin, A.; Poujol, R.; Medjahed, H.; El Ferri, O.; Bo, Y.; Bourassa, C.; Hussin, J.; Fafard, J.; Pazgier, M.; Levade, I.; Abrams, C.; Cote, M.; Finzi, A.</p>
        <p class="info">Score: 43.2, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576353' target='https://doi.org/10.1101/2024.01.20.576353'> 10.1101/2024.01.20.576353</a></p>
        <p class="abstract">The continued evolution of SARS-CoV-2 requires persistent monitoring of its subvariants. Omicron subvariants are responsible for the vast majority of SARS-CoV-2 infections worldwide, with XBB and BA.2.86 sublineages representing more than 90% of circulating strains as of January 2024. In this study, we characterized the functional properties of Spike glycoproteins from BA.2.75, CH.1.1, DV.7.1, BA.4/5, BQ.1.1, XBB, XBB.1, XBB.1.16, XBB.1.5, FD.1.1, EG.5.1, HK.3 BA.2.86 and JN.1. We tested their capacity to evade plasma-mediated recognition and neutralization, ACE2 binding, their susceptibility to cold inactivation, Spike processing, as well as the impact of temperature on Spike-ACE2 interaction. We found that compared to the early wild-type (D614G) strain, most Omicron subvariants Spike glycoproteins evolved to escape recognition and neutralization by plasma from individuals who received a fifth dose of bivalent (BA.1 or BA.4/5) mRNA vaccine and improve ACE2 binding, particularly at low temperatures. Moreover, BA.2.86 had the best affinity for ACE2 at all temperatures tested. We found that Omicron subvariants Spike processing is associated with their susceptibility to cold inactivation. Intriguingly, we found that Spike-ACE2 binding at low temperature was significantly associated with growth rates of Omicron subvariants in humans. Overall, we report that Spikes from newly emerged Omicron subvariants are relatively more stable and resistant to plasma-mediated neutralization, present improved affinity for ACE2 which is associated, particularly at low temperatures, with their growth rates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.576744">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.576744" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.576744">
        <p class="paperTitle">Birth of new protein folds and functions in the virome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.576744" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.576744" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nomburg, J.; Price, N.; Doudna, J. A.</p>
        <p class="info">Score: 24.5, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.576744' target='https://doi.org/10.1101/2024.01.22.576744'> 10.1101/2024.01.22.576744</a></p>
        <p class="abstract">Rapid virus evolution generates proteins essential to infectivity and replication but with unknown function due to extreme sequence divergence. Using a database of 67,715 newly predicted protein structures from 4,463 eukaryotic viral species, we found that 62% of viral proteins are evolutionarily young and lack homologs in the Alphafold database. Among the 38% of more ancient viral proteins, many have non-viral structural homologs that revealed surprising similarities between human pathogens and their eukaryotic hosts. Structural comparisons suggested putative functions for &gt;25% of unannotated viral proteins, including those with roles in the evasion of innate immunity. In particular, RNA ligase T- (ligT) like phosphodiesterases were found to resemble phage-encoded proteins that hydrolyze the host immune-activating cyclic dinucleotides 3&#39;3&#39; and 2&#39;3&#39; cyclic G-A monophosphate (cGAMP). Experimental analysis showed that ligT homologs encoded by avian poxviruses likewise hydrolyze 2&#39;3&#39; cGAMP, showing that ligT-mediated targeting of cGAMP is an evolutionarily conserved mechanism of immune evasion present in both bacteriophage and eukaryotic viruses. Together, the viral protein structural database and analytics presented here afford new opportunities to identify mechanisms of virus-host interactions that are common across the virome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.576693">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.576693" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.576693">
        <p class="paperTitle">Viral entry is a weak barrier to zoonosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.576693" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.576693" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dufloo, J.; Andreu-Moreno, I.; Valero-Rello, A.; Sanjuan, R.</p>
        <p class="info">Score: 22.4, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.576693' target='https://doi.org/10.1101/2024.01.22.576693'> 10.1101/2024.01.22.576693</a></p>
        <p class="abstract">Recent advances in viral metagenomics have led to the discovery of many mammalian viruses, but experimental tests to determine whether they pose a threat to humans are largely lacking. A first step for a virus to cross the species barrier is to penetrate host cells. Here, we use gene synthesis and viral pseudotyping to experimentally test the ability of viral receptor-binding proteins (RBPs) from &gt;100 enveloped RNA viruses to mediate entry into human cells. Analysis of thousands of RBP-cell pairs demonstrated such ability for most viruses, with significant variation among the 14 viral families studied. Comparison of RBP-mediated infectivity with cellular gene expression data showed that viral entry is often not limited by the presence of a receptor and revealed the contribution of additional host factors. Our results prove the weakness of interspecies barriers at the early stages of infection and identify molecular interactions that enable viral zoonosis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.576759">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.576759" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.576759">
        <p class="paperTitle">Halofilins as Emerging Bactofilin Families of Archaeal Cell Shape Plasticity Orchestrators</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.576759" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.576759" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Curtis, Z.; Escudeiro, P.; Mallon, J. L.; Leland, O.; Rados, T.; Dodge, A.; Andre, K.; Kwak, J.; Yun, K.; Pohlschroder, M.; Alva, V.; Bisson, A.</p>
        <p class="info">Score: 18.4, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.576759' target='https://doi.org/10.1101/2024.01.22.576759'> 10.1101/2024.01.22.576759</a></p>
        <p class="abstract">Bactofilins are rigid, non-polar bacterial cytoskeletal filaments that link cellular processes to specific curvatures of the cytoplasmic membrane. Although homologs of bactofilins have been identified in archaea and eukaryotes, functional studies have remained confined to bacterial systems. Here, we characterized representatives of two families of archaeal bactofilins from the pleomorphic archaeon Haloferax volcanii, halofilin A (HalA) and halofilin B (HalB). Unlike bacterial bactofilins, HalA polymerizes into polar filaments in vivo at positive membrane curvature, whereas HalB forms more static foci and accumulates in areas of local negative curvatures on the outer cell surface. Combining gene deletions, super-resolution, and single-cell microscopy showed that halofilins are critical in maintaining H. volcanii cell integrity during shape transition from disk (sessile) to rod (motile). Morphological defects in {Delta}halA primarily affected rod-shaped cells from accumulating highly positive curvatures. Conversely, disk-shaped cells were exclusively affected by halB deletion, showing a decrease in positive and negative curvatures, resulting in flatter cells. Furthermore, while {Delta}halA and {Delta}halB cells displayed lower cell division placement precision, morphological defects arose predominantly during the disk-to-rod shape remodeling. We propose halofilins provide mechanical scaffolding, dynamically coupling the cytoplasmic membrane and the S-layer. We speculate that HalA filaments support rods under low S-layer lipidation (flexible, fast membrane diffusion). In contrast, HalB connects the S-layer to negative curvatures in disks under high lipidation levels (rigid, slow membrane diffusion).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.14.575572">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.14.575572" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.14.575572">
        <p class="paperTitle">NRX-101 (D-Cycloserine &#43; Lurasidone) , a Qualified Infectious Disease Product, is Active Against Drug-Resistant Urinary Pathogens In Vitro</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.14.575572" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.14.575572" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sapko, M. T.; Manyak, M.; Panicucci, R.; Javitt, J. C.</p>
        <p class="info">Score: 58.5, Published: 2024-01-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.14.575572' target='https://doi.org/10.1101/2024.01.14.575572'> 10.1101/2024.01.14.575572</a></p>
        <p class="abstract">BackgroundD-Cycloserine (DCS) is a broad-spectrum antibiotic that is currently FDA-approved to treat tuberculosis (TB) disease and urinary tract infection. Despite numerous reports showing good clinical efficacy, DCS fell out of favor as a UTI treatment because of its propensity to cause side effects. NRX-101, a fixed dose combination of DCS and lurasidone, has been awarded Qualified Infectious Disease Product and Fast Track Designation by the US Food and Drug Administration and is being developed for various CNS indications because of its unique synergistic effect; each component mitigates side effects of the other.

MethodsIn this study, we tested NRX-101 against the urinary tract pathogens E. coli, P. aeruginosa, K. pneumoniae, and A. baumannii in Mueller Hinton broth (caMHB) and artificial urine media (AUM). Several strains were multidrug resistant. Test compounds were serially diluted in broth/media. Minimum inhibitory concentration (MIC) was defined as the lowest concentration of test compound at which no bacterial growth was observed.

ResultsDCS exhibited antibacterial efficacy against all strains tested while lurasidone did not appreciably affect the antibacterial action of DCS in vitro. In AUM, the MICs ranged from 128 to 512 mcg/ml for both DCS and NRX-101. In caMHB, MICs ranged from 8 to 1024 mcg/ml for NRX-101 and 32 to 512 mcg/ml for DCS alone.

ConclusionsOur data confirm that DCS as antibacterial activity against reference and drug-resistant urinary pathogens. Furthermore, lurasidone does not interfere with DCSs anti-microbial action in vitro. These results support the clinical development of NRX-101 as a treatment for complicated urinary tract infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.574570">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.574570" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.574570">
        <p class="paperTitle">Nitrous oxide respiration in acidophilic methanotrophs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.574570" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.574570" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Awala, S. I.; Gwak, J.-H.; Kim, Y.; Jung, M.-Y.; Dunfield, P. F.; Wagner, M.; Rhee, S.-K.</p>
        <p class="info">Score: 13.9, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.574570' target='https://doi.org/10.1101/2024.01.15.574570'> 10.1101/2024.01.15.574570</a></p>
        <p class="abstract">Methanotrophic bacteria mitigate methane (CH4) emissions from natural environments. Although aerobic methanotrophs are considered strict aerobes, they are often highly abundant in extremely hypoxic and even anoxic environments. Despite the presence of denitrification genes, it remains to be verified whether denitrification contributes to their growth. Here, we revealed that two acidophilic methanotrophs encoding N2O reductase (clade I and type II nosZ, respectively): Methylocella tundrae T4 and Methylacidiphilum caldifontis IT6, respired N2O and grew anaerobically on diverse non-methane substrates, including methanol, C-C substrates, and hydrogen. However, NO3 - and NO2 - could be reduced during methanol oxidation in Methylocella tundrae T4 and Methylocella silvestris BL2 without significantly increasing cell biomass. The lack of growth on methanol &#43; NO3- or NO2- was likely due to the production of toxic reactive nitrogen species and C1 metabolites. However, the oxidation of pyruvate, a C3 electron donor, combined with NO3- or NO2- reduction resulted in anaerobic growth of Methylocella tundrae T4 and Methylocella silvestris BL2. In the extreme acidophile, Methylacidiphilum caldifontis IT6, N2O respiration supported cell growth at an extremely acidic pH of 2.0. In Methylocella tundrae T4, simultaneous consumption of N2O and CH4 was observed in suboxic conditions, both in microrespirometry and growth experiments, indicating the robustness of its N2O reductase activity in the presence of O2. Furthermore, CH4 oxidation per O2 reduced in O2-limiting conditions increased when N2O was added, indicating that cells of T4 can direct more O2 towards methane monooxygenase when respiring N2O as a terminal electron acceptor. Upregulation of nosZ and distinct repertories of methanol dehydrogenase-encoding genes (XoxF- and MxaFI-type) in Methylocella tundrae T4 cells grown anaerobically on methanol with N2O as the sole electron acceptor indicated adaptation mechanisms to anoxia. Our findings demonstrate that some methanotrophs can respire N2O independently or in tandem with O2, significantly expanding their potential ecological niche and paving the way for enhanced growth and survival in dynamic environments. This metabolic capability has application potential for simultaneously mitigating the emissions of the key greenhouse gases, CO2, CH4, and N2O, from natural and engineered environments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.23.574598">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.23.574598" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.23.574598">
        <p class="paperTitle">Gut environmental factors explain variations in the gut microbiome composition and metabolism within and between healthy adults</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.23.574598" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.23.574598" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Prochazkova, N.; Laursen, M. F.; La Barbera, G.; Tsekitsidi, E.; Joergensen, M. S.; Rasmussen, M. A.; Raes, J.; Dragsted, L. O.; Licht, T. R.; Roager, H. M.</p>
        <p class="info">Score: 12.7, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.23.574598' target='https://doi.org/10.1101/2024.01.23.574598'> 10.1101/2024.01.23.574598</a></p>
        <p class="abstract">The human gut microbiome is highly personal. However, the contribution of the gut environment to variations in the gut microbiome remains elusive. Here, we profiled the gut microbiome composition and metabolism over 9 consecutive days in 61 healthy adults and assessed gut environmental factors including segmental transit time and pH using a wireless motility capsule. Day-to-day fluctuations in gut environmental factors as well as segmental transit time and pH varied substantially between individuals. The gut environment explained more variations in gut microbiome and urine metabolome than dietary macronutrients or personal characteristics. Finally, we identified coffee-derived metabolites to be negatively correlated with small intestinal transit time and several microbial metabolites to be associated with colonic transit time including urinary proteolytic markers, faecal short-chain fatty acids, and breath methane. Our work suggests that the gut environment is key for understanding the individuality of the human gut microbiome composition and metabolism.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
