<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="SARS-CoV-2 serosurvey of healthy, privately owned cats presenting to a New York City animal hospital in the early phase of the COVID-19 pandemic (2020-2021)
Authors: Choi, A.; Stout, A. E.; Rollins, A.; Wang, K.; Guo, Q.; Jaimes, J. A.; Kennedy, M.; Wagner, B.; Whittaker, G. R.
Score: 241.6, Published: 2024-02-15 DOI: 10.1101/2024.02.13.580068
Both domestic and non-domestic cats are now established to be susceptible to infection by SARS-CoV-2, the cause of the ongoing COVID-19 pandemic.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="SARS-CoV-2 serosurvey of healthy, privately owned cats presenting to a New York City animal hospital in the early phase of the COVID-19 pandemic (2020-2021)
Authors: Choi, A.; Stout, A. E.; Rollins, A.; Wang, K.; Guo, Q.; Jaimes, J. A.; Kennedy, M.; Wagner, B.; Whittaker, G. R.
Score: 241.6, Published: 2024-02-15 DOI: 10.1101/2024.02.13.580068
Both domestic and non-domestic cats are now established to be susceptible to infection by SARS-CoV-2, the cause of the ongoing COVID-19 pandemic." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-18T10:38:00+00:00" />
<meta property="article:modified_time" content="2024-02-18T10:38:00+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="SARS-CoV-2 serosurvey of healthy, privately owned cats presenting to a New York City animal hospital in the early phase of the COVID-19 pandemic (2020-2021)
Authors: Choi, A.; Stout, A. E.; Rollins, A.; Wang, K.; Guo, Q.; Jaimes, J. A.; Kennedy, M.; Wagner, B.; Whittaker, G. R.
Score: 241.6, Published: 2024-02-15 DOI: 10.1101/2024.02.13.580068
Both domestic and non-domestic cats are now established to be susceptible to infection by SARS-CoV-2, the cause of the ongoing COVID-19 pandemic."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "SARS-CoV-2 serosurvey of healthy, privately owned cats presenting to a New York City animal hospital in the early phase of the COVID-19 pandemic (2020-2021)\nAuthors: Choi, A.; Stout, A. E.; Rollins, A.; Wang, K.; Guo, Q.; Jaimes, J. A.; Kennedy, M.; Wagner, B.; Whittaker, G. R.\nScore: 241.6, Published: 2024-02-15 DOI: 10.1101/2024.02.13.580068\nBoth domestic and non-domestic cats are now established to be susceptible to infection by SARS-CoV-2, the cause of the ongoing COVID-19 pandemic.",
  "keywords": [
    
  ],
  "articleBody": " SARS-CoV-2 serosurvey of healthy, privately owned cats presenting to a New York City animal hospital in the early phase of the COVID-19 pandemic (2020-2021)\nAuthors: Choi, A.; Stout, A. E.; Rollins, A.; Wang, K.; Guo, Q.; Jaimes, J. A.; Kennedy, M.; Wagner, B.; Whittaker, G. R.\nScore: 241.6, Published: 2024-02-15 DOI: 10.1101/2024.02.13.580068\nBoth domestic and non-domestic cats are now established to be susceptible to infection by SARS-CoV-2, the cause of the ongoing COVID-19 pandemic. While serious disease in cats may occur in some instances, the majority of infections appear to be subclinical. Differing prevalence data for SARS-CoV-2 infection of cats have been reported, and are highly context-dependent. Here, we report a retrospective serological survey of cats presented to an animal practice in New York City, located in close proximity to a large medical center that treated the first wave of COVID-19 patients in the US in the Spring of 2020. We sampled 79, mostly indoor, cats between June 2020 to May 2021, the early part of which time the community was under a strict public health lock-down. Using a highly sensitive and specific fluorescent bead-based multiplex assay, we found an overall prevalence of 13/79 (16%) serologically-positive animals for the study period; however, cats sampled in the Fall of 2020 had a confirmed positive prevalence of 44%. For SARS-CoV-2 seropositive cats, we performed viral neutralization test with live SARS-CoV-2 to additionally confirm presence of SARS-CoV-2 specific antibodies. Of the thirteen seropositive cats, 7/13 (54%) were also positive by virus neutralization, and 2 of seropositive cats had previously documented respiratory signs, with high neutralization titers of 1:1024 and 1:4096; overall however, there was no statistically significant association of SARS-CoV-2 seropositivity with respiratory signs, or with breed, sex or age of the animals. Follow up sampling of cats, while limited in scope, showed that positive serological titers were maintained over time. In comparison, we found an overall confirmed positive prevalence of 51% for feline coronavirus (FCoV), an endemic virus of cats, with 30% confirmed negative for FCoV. We demonstrate the impact of SARS-CoV in a defined feline population during the first wave of SARS-CoV-2 infection of humans, and suggest that human-cat transmission was substantial in our study group. Our data provide a new context for SARS-CoV-2 transmission events across species.\nRobust Neutralization of SARS-CoV-2 Variants Including JN.1 and BA.2.87.1 by Trivalent XBB Vaccine-Induced Antibodies\nAuthors: Wang, X.; Jiang, S.; Ma, W.; Li, C.; Liu, C.; Xie, F.; Zhu, J.; Zhan, Y.; Jiang, S.; Li, M.; Zhang, Y.; Wang, P.\nScore: 188.9, Published: 2024-02-16 DOI: 10.1101/2024.02.16.580615\nNewly emerged SARS-CoV-2 variants like JN.1, and more recently, the hypermutated BA.2.87.1, have raised global concern. We recruited two groups of participants who had BA.5/BF.7 breakthrough infection post three doses of inactivated vaccines: one group experienced subsequent XBB reinfection, while the other received the XBB-containing trivalent WSK-V102C vaccine. Our comparative analysis of their serum neutralization activities revealed that the WSK-V102C vaccine induced stronger antibody responses against a wide range of variants, notably including JN.1 and the highly escaped BA.2.87.1. Furthermore, our investigation into specific mutations revealed that fragment deletions in NTD significantly contribute to the immune evasion of the BA.2.87.1 variant. Our findings emphasize the necessity for ongoing vaccine development and adaptation to address the dynamic nature of SARS-CoV-2 variants.\nViroid-like colonists of human microbiomes\nAuthors: Zheludev, I. N.; Edgar, R. C.; Galiano, M. J. L.; De la Pena, M.; Babaian, A.; Bhatt, A. S.; Fire, A. Z.\nScore: 850.9, Published: 2024-01-21 DOI: 10.1101/2024.01.20.576352\nHere, we describe the \"Obelisks,\" a previously unrecognised class of viroid-like elements that we first identified in human gut metatranscriptomic data. \"Obelisks\" share several properties: (i) apparently circular RNA [~]1kb genome assemblies, (ii) predicted rod-like secondary structures encompassing the entire genome, and (iii) open reading frames coding for a novel protein superfamily, which we call the \"Oblins\". We find that Obelisks form their own distinct phylogenetic group with no detectable sequence or structural similarity to known biological agents. Further, Obelisks are prevalent in tested human microbiome metatranscriptomes with representatives detected in [~]7% of analysed stool metatranscriptomes (29/440) and in [~]50% of analysed oral metatranscriptomes (17/32). Obelisk compositions appear to differ between the anatomic sites and are capable of persisting in individuals, with continued presence over \u003e300 days observed in one case. Large scale searches identified 29,959 Obelisks (clustered at 90% nucleotide identity), with examples from all seven continents and in diverse ecological niches. From this search, a subset of Obelisks are identified to code for Obelisk-specific variants of the hammerhead type-III self-cleaving ribozyme. Lastly, we identified one case of a bacterial species (Streptococcus sanguinis) in which a subset of defined laboratory strains harboured a specific Obelisk RNA population. As such, Obelisks comprise a class of diverse RNAs that have colonised, and gone unnoticed in, human, and global microbiomes.\nPhages reconstitute NAD+ to counter bacterial immunity\nAuthors: Osterman, I.; Samra, H.; Rousset, F.; Loseva, E.; Itkin, M.; Malitsky, S.; Yirmiya, E.; Millman, A.; Sorek, R.\nScore: 59.4, Published: 2024-02-13 DOI: 10.1101/2024.02.11.579819\nBacteria defend against phage infection via a variety of antiphage defense systems. Many defense systems were recently shown to deplete cellular nicotinamide adenine dinucleotide (NAD+) in response to infection, by breaking NAD+ to ADP-ribose (ADPR) and nicotinamide. It was demonstrated that NAD+ depletion during infection deprives the phage from this essential molecule and impedes phage replication. Here we show that a substantial fraction of phages possess enzymatic pathways allowing reconstitution of NAD+ from its degradation products in infected cells. We describe NAD+ reconstitution pathway 1 (NARP1), a two-step pathway in which one enzyme phosphorylates ADPR to generate ADPR-pyrophosphate (ADPR-PP), and the second enzyme conjugates ADPR- PP and nicotinamide to generate NAD+. Phages encoding the NARP1 pathway can overcome a diverse set of defense systems, including Thoeris, DSR1, DSR2, SIR2-HerA, and SEFIR, all of which deplete NAD+ as part of their defensive mechanism. Phylogenetic analyses show that NARP1 is primarily encoded on phage genomes, suggesting a phage- specific function in countering bacterial defenses. A second pathway, NARP2, allows phages to overcome bacterial defenses by building NAD+ via metabolites different than ADPR-PP. Our findings report a unique immune evasion strategy where viruses rebuild molecules depleted by defense systems, thus overcoming host immunity.\nMolecular switching of a DNA-sliding clamp to a repressor mediates long-range gene silencing\nAuthors: McLean, T. C.; Balaguer-Perez, F.; Chandanani, J.; Thomas, C. M.; Aicart-Ramos, C.; Burick, S.; Olinares, P. D. B.; Gobbato, G.; Mundy, J. E. A.; Chait, B. T.; Lawson, D. M.; Darst, S. A.; Campbell, E. A.; Moreno-Herrero, F.; Le, T. B. K.\nScore: 33.8, Published: 2024-02-16 DOI: 10.1101/2024.02.16.579611\nLong-range gene regulation is rare in bacteria and is confined to the classical DNA looping model. Here, we use a combination of biophysical approaches, including X-ray crystallography and single-molecule analysis, to show that long-range gene silencing on the plasmid RK2, a source of multidrug resistance across diverse Gram-negative bacteria, is achieved cooperatively by a DNA-sliding clamp, KorB, and a clamp-locking protein, KorA. We find that KorB is a CTPase clamp that can entrap and slide along DNA to reach distal target promoters. We resolved the tripartite crystal structure of a KorB-KorA-DNA co-complex, revealing that KorA latches KorB into a closed-clamp state. KorA thus stimulates repression by stalling KorB sliding at target promoters to occlude RNA polymerase holoenzymes. Altogether, our findings explain the mechanistic basis for KorB role-switching from a DNA-sliding clamp to a co-repressor, and provide a new paradigm for the long-range regulation of gene expression.\nH-NS is a bacterial transposon capture protein\nAuthors: Cooper, C.; Legood, S.; Wheat, R. L.; Forrest, D.; Sharma, P.; Haycocks, J. R.; Grainger, D. C.\nScore: 28.8, Published: 2024-02-16 DOI: 10.1101/2024.02.16.580519\nThe histone-like nucleoid structuring (H-NS) protein is a DNA binding factor, found in {gamma}-proteobacteria, with functional equivalents in diverse microbes. Universally, such proteins are understood to silence transcription of horizontally acquired genes. Here, we identify transposon capture as a major overlooked function of H-NS. Using genome scale approaches, we show that H NS bound chromosomal regions are transposition \"hotspots\". Since H-NS often interacts with pathogenicity islands, such targeting creates clinically relevant phenotypic diversity. For example, in Acinetobacter baumannii, we identify altered motility, biofilm formation, and interactions with the human immune system. Transposon capture is mediated by the DNA bridging activity of H-NS and, if absent, more ubiquitous transposition results. Consequently, transcribed and essential genes are disrupted. Hence, H NS directs transposition to favour evolutionary outcomes useful for the host cell.\nA new archaeal virus that suppresses the transcription of host immunity genes\nAuthors: Turgeman-Grott, I.; Golan, N.; Neri, U.; Naki, D.; Altman-Price, N.; Eizenshtein, K.; Choudhary, D. K.; Levy, R.; Papke, R. T.; Himani, H.; Reshef, L.; Gophna, U.\nScore: 28.6, Published: 2024-02-12 DOI: 10.1101/2024.02.12.579488\nIn some extreme environments, archaeal cells have been shown to have chronic viral infections, and such infections are well-tolerated by the hosts and may potentially protect against more lethal infections by lytic viruses. We have discovered that a natural Haloferax strain (48N), which is closely related to the model organism Haloferax volcanii, is chronically-infected by a lemon-shaped virus, which we could purify from the medium. The chronic infection by this virus, which we named LSV-48N, is never cleared, despite the multiple defense systems of the host that include CRISPR-Cas, and two CBASS systems. Curing 48N of its virus by genetic engineering, led to radical changes in the gene expression profile of 48N and a dramatic improvement in its growth rate. Remarkably, the cured 48N is the fastest-growing haloarchaeon reported to date, with a generation time of approximately 1 hour at 45{degrees}C compared to the typical 2.5 hours of H. volcanii or its infected isogen, and faster than any known haloarchaeon. The virus subverts host defenses by reducing their transcription and interfering with the CRISPR spacer acquisition machinery. Our results suggest that the slow growth of many halophilic archaea could be due to the effects of proviruses within their genomes that consume resources and alter the gene expression of their hosts.\nThe Cas10 nuclease activity relieves host dormancy to facilitate spacer acquisition and retention during type III-A CRISPR immunity\nAuthors: Aviram, N.; Shilton, A.; Lyn, N.; Reis, B. S.; Brivanlou, A.; Marraffini, L. A.\nScore: 26.8, Published: 2024-02-12 DOI: 10.1101/2024.02.11.579731\nA hallmark of CRISPR immunity is the acquisition of short viral DNA sequences, known as spacers, that are transcribed into guide RNAs to recognize complementary sequences. The staphylococcal type III-A CRISPR-Cas system uses guide RNAs to locate viral transcripts and start a response that displays two mechanisms of immunity. When immunity is triggered by an early-expressed phage RNA, degradation of viral ssDNA can cure the host from infection. In contrast, when the RNA guide targets a late-expressed transcript, defense requires the activity of Csm6, a non-specific RNase. Here we show that Csm6 triggers a growth arrest of the host that provides immunity at the population level which hinders viral propagation to allow the replication of non-infected cells. We demonstrate that this mechanism leads to defense against not only the target phage but also other viruses present in the population that fail to replicate in the arrested cells. On the other hand, dormancy limits the acquisition and retention of spacers that trigger it. We found that the ssDNase activity of type III-A systems is required for the re-growth of a subset of the arrested cells, presumably through the degradation of the phage DNA, ending target transcription and inactivating the immune response. Altogether, our work reveals a built-in mechanism within type III-A CRISPR-Cas systems that allows the exit from dormancy needed for the subsistence of spacers that provide broad-spectrum immunity.\nComplex carbohydrate utilization by gut bacteria modulates host food preference\nAuthors: Yu, K. B.; Son, C.; Chandra, A.; Paramo, J.; Novoselov, A.; Ozcan, E.; Kazmi, S.; Lum, G.; Lopez-Romero, A.; Lynch, J.; Hsiao, E. Y.\nScore: 22.7, Published: 2024-02-14 DOI: 10.1101/2024.02.13.580152\nThe gut microbiota interacts directly with dietary nutrients and has the ability to modify host feeding behavior, but the underlying mechanisms remain poorly understood. Select gut bacteria digest complex carbohydrates that are non-digestible by the host and liberate metabolites that serve as additional energy sources and pleiotropic signaling molecules. Here we use a gnotobiotic mouse model to examine how differential fructose polysaccharide metabolism by commensal gut bacteria influences host preference for diets containing these carbohydrates. Bacteroides thetaiotaomicron and Bacteroides ovatus selectively ferment fructans with different glycosidic linkages: B. thetaiotaomicron ferments levan with {beta}2-6 linkages, whereas B. ovatus ferments inulin with {beta}2-1 linkages. Since inulin and levan are both fructose polymers, inulin and levan diet have similar perceptual salience to mice. We find that mice colonized with B. thetaiotaomicron prefer the non-fermentable inulin diet, while mice colonized with B. ovatus prefer the non-fermentable levan diet. Knockout of bacterial fructan utilization genes abrogates this preference, whereas swapping the fermentation ability of B. thetaiotaomicron to inulin confers host preference for the levan diet. Bacterial fructan fermentation and host behavioral preference for the non-fermentable fructan are associated with increased neuronal activation in the arcuate nucleus of the hypothalamus, a key brain region for appetite regulation. These results reveal that selective nutrient metabolism by gut bacteria contributes to host associative learning of dietary preference, and further informs fundamental understanding of the biological determinants of food choice.\nImmune escape of Omicron lineages BA.1, BA.2, BA.5.1, BQ.1, XBB.1.5, EG.5.1 and JN.1.1 after vaccination, infection and hybrid immunity\nAuthors: Bekliz, M.; Essaidi-Laziosi, M.; Adea, K.; Hosszu-Fellous, K.; Alvarez, C.; Bellon, M.; Sattonnet-Roche, P.; Puhach, O.; Dbeissi, D.; Zaballa, M. E.; Stringhini, S.; Guessous, I.; Vetter, P.; Eberhardt, C. S.; Kaiser, L.; Eckerle, I.\nScore: 21.9, Published: 2024-02-15 DOI: 10.1101/2024.02.14.579654\nSince their emergence in late 2021, SARS-CoV-2 Omicron replaced earlier variants of concern and marked a new phase in the SARS-CoV-2 pandemic. Until the end of 2023, Omicron lineages continue to circulate and continue to evolve, with new lineages causing infection waves throughout 2022 and 2023. In the population, this leads to a complex immunological exposure background, characterized by immunity derived through vaccination, in the 5th year of the pandemic in the majority of individuals followed by at least one or even multiple infections or only natural infection in individuals that did not receive a vaccine. In this study, we use eight authentic SARS-CoV-2 isolates (ancestral lineage B.1 and the seven Omicron lineages BA.1, BA.2, BA.5.1, BQ.1, XBB.1.5, EG.5.1 and JN.1.1) in a live virus neutralization assay to study immune escape in 97 human sera or plasma of different immunological backgrounds (vaccination, hybrid immunity due to one or two natural infections and natural infection without vaccination in children and adults). We showed a gradually increasing immune escape after vaccination and hybrid immunity in from B.1 to BA.1/BA.2 to BA.5.1 to BQ.1 to XBB.1.5 to EG.5.1, but remarkably, no more enhanced immune escape of JN.1.1 compared to EG.5.1, with the latter two showing almost identical neutralization titers in individuals with hybrid immunity due to one or more infections. In vaccinated but never infected individuals, neutralization was markedly reduced or completely lost for XBB.1.5., EG.5.1 and JN.1.1, while in those with hybrid immunity, titers were reduced but almost all sera still showed some degree of neutralization. After a single infection without vaccination, reduced or complete loss of neutralization occurred for BQ.1, XBB.1.5, EG.5.1 and JN.1.1 compared to BA.1/BA.2. Furthermore, we observed that, although absolute titers differed between groups, the pattern of immune escape between the variants remains comparable across groups, with strongest loss of neutralization for BQ.1, XBB.1.5, EG.5.1 and JN.1.1 was observed across the different immunological backgrounds. Our results show gradually increasing antibody escape of evolving Omicron lineages over the last two years of Omicron circulation until variant EG.5.1, but not anymore for the currently dominant lineages JN.1.1, suggesting other mechanisms than immune escape to be behind the rapid global emergence of JN.1.\n",
  "wordCount" : "2611",
  "inLanguage": "en",
  "datePublished": "2024-02-18T10:38:00Z",
  "dateModified": "2024-02-18T10:38:00Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 18, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.13.580068">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.13.580068" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.13.580068">
        <p class="paperTitle">SARS-CoV-2 serosurvey of healthy, privately owned cats presenting to a New York City animal hospital in the early phase of the COVID-19 pandemic (2020-2021)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.13.580068" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.13.580068" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Choi, A.; Stout, A. E.; Rollins, A.; Wang, K.; Guo, Q.; Jaimes, J. A.; Kennedy, M.; Wagner, B.; Whittaker, G. R.</p>
        <p class="info">Score: 241.6, Published: 2024-02-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.13.580068' target='https://doi.org/10.1101/2024.02.13.580068'> 10.1101/2024.02.13.580068</a></p>
        <p class="abstract">Both domestic and non-domestic cats are now established to be susceptible to infection by SARS-CoV-2, the cause of the ongoing COVID-19 pandemic. While serious disease in cats may occur in some instances, the majority of infections appear to be subclinical. Differing prevalence data for SARS-CoV-2 infection of cats have been reported, and are highly context-dependent. Here, we report a retrospective serological survey of cats presented to an animal practice in New York City, located in close proximity to a large medical center that treated the first wave of COVID-19 patients in the US in the Spring of 2020. We sampled 79, mostly indoor, cats between June 2020 to May 2021, the early part of which time the community was under a strict public health lock-down. Using a highly sensitive and specific fluorescent bead-based multiplex assay, we found an overall prevalence of 13/79 (16%) serologically-positive animals for the study period; however, cats sampled in the Fall of 2020 had a confirmed positive prevalence of 44%. For SARS-CoV-2 seropositive cats, we performed viral neutralization test with live SARS-CoV-2 to additionally confirm presence of SARS-CoV-2 specific antibodies. Of the thirteen seropositive cats, 7/13 (54%) were also positive by virus neutralization, and 2 of seropositive cats had previously documented respiratory signs, with high neutralization titers of 1:1024 and 1:4096; overall however, there was no statistically significant association of SARS-CoV-2 seropositivity with respiratory signs, or with breed, sex or age of the animals. Follow up sampling of cats, while limited in scope, showed that positive serological titers were maintained over time. In comparison, we found an overall confirmed positive prevalence of 51% for feline coronavirus (FCoV), an endemic virus of cats, with 30% confirmed negative for FCoV. We demonstrate the impact of SARS-CoV in a defined feline population during the first wave of SARS-CoV-2 infection of humans, and suggest that human-cat transmission was substantial in our study group. Our data provide a new context for SARS-CoV-2 transmission events across species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.16.580615">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.16.580615" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.16.580615">
        <p class="paperTitle">Robust Neutralization of SARS-CoV-2 Variants Including JN.1 and BA.2.87.1 by Trivalent XBB Vaccine-Induced Antibodies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.16.580615" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.16.580615" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, X.; Jiang, S.; Ma, W.; Li, C.; Liu, C.; Xie, F.; Zhu, J.; Zhan, Y.; Jiang, S.; Li, M.; Zhang, Y.; Wang, P.</p>
        <p class="info">Score: 188.9, Published: 2024-02-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.16.580615' target='https://doi.org/10.1101/2024.02.16.580615'> 10.1101/2024.02.16.580615</a></p>
        <p class="abstract">Newly emerged SARS-CoV-2 variants like JN.1, and more recently, the hypermutated BA.2.87.1, have raised global concern. We recruited two groups of participants who had BA.5/BF.7 breakthrough infection post three doses of inactivated vaccines: one group experienced subsequent XBB reinfection, while the other received the XBB-containing trivalent WSK-V102C vaccine. Our comparative analysis of their serum neutralization activities revealed that the WSK-V102C vaccine induced stronger antibody responses against a wide range of variants, notably including JN.1 and the highly escaped BA.2.87.1. Furthermore, our investigation into specific mutations revealed that fragment deletions in NTD significantly contribute to the immune evasion of the BA.2.87.1 variant. Our findings emphasize the necessity for ongoing vaccine development and adaptation to address the dynamic nature of SARS-CoV-2 variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576352">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576352" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576352">
        <p class="paperTitle">Viroid-like colonists of human microbiomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576352" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576352" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zheludev, I. N.; Edgar, R. C.; Galiano, M. J. L.; De la Pena, M.; Babaian, A.; Bhatt, A. S.; Fire, A. Z.</p>
        <p class="info">Score: 850.9, Published: 2024-01-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576352' target='https://doi.org/10.1101/2024.01.20.576352'> 10.1101/2024.01.20.576352</a></p>
        <p class="abstract">Here, we describe the &#34;Obelisks,&#34; a previously unrecognised class of viroid-like elements that we first identified in human gut metatranscriptomic data. &#34;Obelisks&#34; share several properties: (i) apparently circular RNA [~]1kb genome assemblies, (ii) predicted rod-like secondary structures encompassing the entire genome, and (iii) open reading frames coding for a novel protein superfamily, which we call the &#34;Oblins&#34;. We find that Obelisks form their own distinct phylogenetic group with no detectable sequence or structural similarity to known biological agents. Further, Obelisks are prevalent in tested human microbiome metatranscriptomes with representatives detected in [~]7% of analysed stool metatranscriptomes (29/440) and in [~]50% of analysed oral metatranscriptomes (17/32). Obelisk compositions appear to differ between the anatomic sites and are capable of persisting in individuals, with continued presence over &gt;300 days observed in one case. Large scale searches identified 29,959 Obelisks (clustered at 90% nucleotide identity), with examples from all seven continents and in diverse ecological niches. From this search, a subset of Obelisks are identified to code for Obelisk-specific variants of the hammerhead type-III self-cleaving ribozyme. Lastly, we identified one case of a bacterial species (Streptococcus sanguinis) in which a subset of defined laboratory strains harboured a specific Obelisk RNA population. As such, Obelisks comprise a class of diverse RNAs that have colonised, and gone unnoticed in, human, and global microbiomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.11.579819">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.11.579819" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.11.579819">
        <p class="paperTitle">Phages reconstitute NAD&#43; to counter bacterial immunity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.11.579819" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.11.579819" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Osterman, I.; Samra, H.; Rousset, F.; Loseva, E.; Itkin, M.; Malitsky, S.; Yirmiya, E.; Millman, A.; Sorek, R.</p>
        <p class="info">Score: 59.4, Published: 2024-02-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.11.579819' target='https://doi.org/10.1101/2024.02.11.579819'> 10.1101/2024.02.11.579819</a></p>
        <p class="abstract">Bacteria defend against phage infection via a variety of antiphage defense systems. Many defense systems were recently shown to deplete cellular nicotinamide adenine dinucleotide (NAD&#43;) in response to infection, by breaking NAD&#43; to ADP-ribose (ADPR) and nicotinamide. It was demonstrated that NAD&#43; depletion during infection deprives the phage from this essential molecule and impedes phage replication. Here we show that a substantial fraction of phages possess enzymatic pathways allowing reconstitution of NAD&#43; from its degradation products in infected cells. We describe NAD&#43; reconstitution pathway 1 (NARP1), a two-step pathway in which one enzyme phosphorylates ADPR to generate ADPR-pyrophosphate (ADPR-PP), and the second enzyme conjugates ADPR- PP and nicotinamide to generate NAD&#43;. Phages encoding the NARP1 pathway can overcome a diverse set of defense systems, including Thoeris, DSR1, DSR2, SIR2-HerA, and SEFIR, all of which deplete NAD&#43; as part of their defensive mechanism. Phylogenetic analyses show that NARP1 is primarily encoded on phage genomes, suggesting a phage- specific function in countering bacterial defenses. A second pathway, NARP2, allows phages to overcome bacterial defenses by building NAD&#43; via metabolites different than ADPR-PP. Our findings report a unique immune evasion strategy where viruses rebuild molecules depleted by defense systems, thus overcoming host immunity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.16.579611">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.16.579611" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.16.579611">
        <p class="paperTitle">Molecular switching of a DNA-sliding clamp to a repressor mediates long-range gene silencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.16.579611" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.16.579611" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McLean, T. C.; Balaguer-Perez, F.; Chandanani, J.; Thomas, C. M.; Aicart-Ramos, C.; Burick, S.; Olinares, P. D. B.; Gobbato, G.; Mundy, J. E. A.; Chait, B. T.; Lawson, D. M.; Darst, S. A.; Campbell, E. A.; Moreno-Herrero, F.; Le, T. B. K.</p>
        <p class="info">Score: 33.8, Published: 2024-02-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.16.579611' target='https://doi.org/10.1101/2024.02.16.579611'> 10.1101/2024.02.16.579611</a></p>
        <p class="abstract">Long-range gene regulation is rare in bacteria and is confined to the classical DNA looping model. Here, we use a combination of biophysical approaches, including X-ray crystallography and single-molecule analysis, to show that long-range gene silencing on the plasmid RK2, a source of multidrug resistance across diverse Gram-negative bacteria, is achieved cooperatively by a DNA-sliding clamp, KorB, and a clamp-locking protein, KorA. We find that KorB is a CTPase clamp that can entrap and slide along DNA to reach distal target promoters. We resolved the tripartite crystal structure of a KorB-KorA-DNA co-complex, revealing that KorA latches KorB into a closed-clamp state. KorA thus stimulates repression by stalling KorB sliding at target promoters to occlude RNA polymerase holoenzymes. Altogether, our findings explain the mechanistic basis for KorB role-switching from a DNA-sliding clamp to a co-repressor, and provide a new paradigm for the long-range regulation of gene expression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.16.580519">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.16.580519" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.16.580519">
        <p class="paperTitle">H-NS is a bacterial transposon capture protein</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.16.580519" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.16.580519" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cooper, C.; Legood, S.; Wheat, R. L.; Forrest, D.; Sharma, P.; Haycocks, J. R.; Grainger, D. C.</p>
        <p class="info">Score: 28.8, Published: 2024-02-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.16.580519' target='https://doi.org/10.1101/2024.02.16.580519'> 10.1101/2024.02.16.580519</a></p>
        <p class="abstract">The histone-like nucleoid structuring (H-NS) protein is a DNA binding factor, found in {gamma}-proteobacteria, with functional equivalents in diverse microbes. Universally, such proteins are understood to silence transcription of horizontally acquired genes. Here, we identify transposon capture as a major overlooked function of H-NS. Using genome scale approaches, we show that H NS bound chromosomal regions are transposition &#34;hotspots&#34;. Since H-NS often interacts with pathogenicity islands, such targeting creates clinically relevant phenotypic diversity. For example, in Acinetobacter baumannii, we identify altered motility, biofilm formation, and interactions with the human immune system. Transposon capture is mediated by the DNA bridging activity of H-NS and, if absent, more ubiquitous transposition results. Consequently, transcribed and essential genes are disrupted. Hence, H NS directs transposition to favour evolutionary outcomes useful for the host cell.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.12.579488">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.12.579488" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.12.579488">
        <p class="paperTitle">A new archaeal virus that suppresses the transcription of host immunity genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.12.579488" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.12.579488" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Turgeman-Grott, I.; Golan, N.; Neri, U.; Naki, D.; Altman-Price, N.; Eizenshtein, K.; Choudhary, D. K.; Levy, R.; Papke, R. T.; Himani, H.; Reshef, L.; Gophna, U.</p>
        <p class="info">Score: 28.6, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.12.579488' target='https://doi.org/10.1101/2024.02.12.579488'> 10.1101/2024.02.12.579488</a></p>
        <p class="abstract">In some extreme environments, archaeal cells have been shown to have chronic viral infections, and such infections are well-tolerated by the hosts and may potentially protect against more lethal infections by lytic viruses. We have discovered that a natural Haloferax strain (48N), which is closely related to the model organism Haloferax volcanii, is chronically-infected by a lemon-shaped virus, which we could purify from the medium. The chronic infection by this virus, which we named LSV-48N, is never cleared, despite the multiple defense systems of the host that include CRISPR-Cas, and two CBASS systems. Curing 48N of its virus by genetic engineering, led to radical changes in the gene expression profile of 48N and a dramatic improvement in its growth rate. Remarkably, the cured 48N is the fastest-growing haloarchaeon reported to date, with a generation time of approximately 1 hour at 45{degrees}C compared to the typical 2.5 hours of H. volcanii or its infected isogen, and faster than any known haloarchaeon. The virus subverts host defenses by reducing their transcription and interfering with the CRISPR spacer acquisition machinery. Our results suggest that the slow growth of many halophilic archaea could be due to the effects of proviruses within their genomes that consume resources and alter the gene expression of their hosts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.11.579731">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.11.579731" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.11.579731">
        <p class="paperTitle">The Cas10 nuclease activity relieves host dormancy to facilitate spacer acquisition and retention during type III-A CRISPR immunity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.11.579731" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.11.579731" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aviram, N.; Shilton, A.; Lyn, N.; Reis, B. S.; Brivanlou, A.; Marraffini, L. A.</p>
        <p class="info">Score: 26.8, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.11.579731' target='https://doi.org/10.1101/2024.02.11.579731'> 10.1101/2024.02.11.579731</a></p>
        <p class="abstract">A hallmark of CRISPR immunity is the acquisition of short viral DNA sequences, known as spacers, that are transcribed into guide RNAs to recognize complementary sequences. The staphylococcal type III-A CRISPR-Cas system uses guide RNAs to locate viral transcripts and start a response that displays two mechanisms of immunity. When immunity is triggered by an early-expressed phage RNA, degradation of viral ssDNA can cure the host from infection. In contrast, when the RNA guide targets a late-expressed transcript, defense requires the activity of Csm6, a non-specific RNase. Here we show that Csm6 triggers a growth arrest of the host that provides immunity at the population level which hinders viral propagation to allow the replication of non-infected cells. We demonstrate that this mechanism leads to defense against not only the target phage but also other viruses present in the population that fail to replicate in the arrested cells. On the other hand, dormancy limits the acquisition and retention of spacers that trigger it. We found that the ssDNase activity of type III-A systems is required for the re-growth of a subset of the arrested cells, presumably through the degradation of the phage DNA, ending target transcription and inactivating the immune response. Altogether, our work reveals a built-in mechanism within type III-A CRISPR-Cas systems that allows the exit from dormancy needed for the subsistence of spacers that provide broad-spectrum immunity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.13.580152">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.13.580152" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.13.580152">
        <p class="paperTitle">Complex carbohydrate utilization by gut bacteria modulates host food preference</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.13.580152" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.13.580152" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yu, K. B.; Son, C.; Chandra, A.; Paramo, J.; Novoselov, A.; Ozcan, E.; Kazmi, S.; Lum, G.; Lopez-Romero, A.; Lynch, J.; Hsiao, E. Y.</p>
        <p class="info">Score: 22.7, Published: 2024-02-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.13.580152' target='https://doi.org/10.1101/2024.02.13.580152'> 10.1101/2024.02.13.580152</a></p>
        <p class="abstract">The gut microbiota interacts directly with dietary nutrients and has the ability to modify host feeding behavior, but the underlying mechanisms remain poorly understood. Select gut bacteria digest complex carbohydrates that are non-digestible by the host and liberate metabolites that serve as additional energy sources and pleiotropic signaling molecules. Here we use a gnotobiotic mouse model to examine how differential fructose polysaccharide metabolism by commensal gut bacteria influences host preference for diets containing these carbohydrates. Bacteroides thetaiotaomicron and Bacteroides ovatus selectively ferment fructans with different glycosidic linkages: B. thetaiotaomicron ferments levan with {beta}2-6 linkages, whereas B. ovatus ferments inulin with {beta}2-1 linkages. Since inulin and levan are both fructose polymers, inulin and levan diet have similar perceptual salience to mice. We find that mice colonized with B. thetaiotaomicron prefer the non-fermentable inulin diet, while mice colonized with B. ovatus prefer the non-fermentable levan diet. Knockout of bacterial fructan utilization genes abrogates this preference, whereas swapping the fermentation ability of B. thetaiotaomicron to inulin confers host preference for the levan diet. Bacterial fructan fermentation and host behavioral preference for the non-fermentable fructan are associated with increased neuronal activation in the arcuate nucleus of the hypothalamus, a key brain region for appetite regulation. These results reveal that selective nutrient metabolism by gut bacteria contributes to host associative learning of dietary preference, and further informs fundamental understanding of the biological determinants of food choice.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.14.579654">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.14.579654" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.14.579654">
        <p class="paperTitle">Immune escape of Omicron lineages BA.1, BA.2, BA.5.1, BQ.1, XBB.1.5, EG.5.1 and JN.1.1 after vaccination, infection and hybrid immunity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.14.579654" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.14.579654" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bekliz, M.; Essaidi-Laziosi, M.; Adea, K.; Hosszu-Fellous, K.; Alvarez, C.; Bellon, M.; Sattonnet-Roche, P.; Puhach, O.; Dbeissi, D.; Zaballa, M. E.; Stringhini, S.; Guessous, I.; Vetter, P.; Eberhardt, C. S.; Kaiser, L.; Eckerle, I.</p>
        <p class="info">Score: 21.9, Published: 2024-02-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.14.579654' target='https://doi.org/10.1101/2024.02.14.579654'> 10.1101/2024.02.14.579654</a></p>
        <p class="abstract">Since their emergence in late 2021, SARS-CoV-2 Omicron replaced earlier variants of concern and marked a new phase in the SARS-CoV-2 pandemic. Until the end of 2023, Omicron lineages continue to circulate and continue to evolve, with new lineages causing infection waves throughout 2022 and 2023. In the population, this leads to a complex immunological exposure background, characterized by immunity derived through vaccination, in the 5th year of the pandemic in the majority of individuals followed by at least one or even multiple infections or only natural infection in individuals that did not receive a vaccine. In this study, we use eight authentic SARS-CoV-2 isolates (ancestral lineage B.1 and the seven Omicron lineages BA.1, BA.2, BA.5.1, BQ.1, XBB.1.5, EG.5.1 and JN.1.1) in a live virus neutralization assay to study immune escape in 97 human sera or plasma of different immunological backgrounds (vaccination, hybrid immunity due to one or two natural infections and natural infection without vaccination in children and adults). We showed a gradually increasing immune escape after vaccination and hybrid immunity in from B.1 to BA.1/BA.2 to BA.5.1 to BQ.1 to XBB.1.5 to EG.5.1, but remarkably, no more enhanced immune escape of JN.1.1 compared to EG.5.1, with the latter two showing almost identical neutralization titers in individuals with hybrid immunity due to one or more infections. In vaccinated but never infected individuals, neutralization was markedly reduced or completely lost for XBB.1.5., EG.5.1 and JN.1.1, while in those with hybrid immunity, titers were reduced but almost all sera still showed some degree of neutralization. After a single infection without vaccination, reduced or complete loss of neutralization occurred for BQ.1, XBB.1.5, EG.5.1 and JN.1.1 compared to BA.1/BA.2. Furthermore, we observed that, although absolute titers differed between groups, the pattern of immune escape between the variants remains comparable across groups, with strongest loss of neutralization for BQ.1, XBB.1.5, EG.5.1 and JN.1.1 was observed across the different immunological backgrounds. Our results show gradually increasing antibody escape of evolving Omicron lineages over the last two years of Omicron circulation until variant EG.5.1, but not anymore for the currently dominant lineages JN.1.1, suggesting other mechanisms than immune escape to be behind the rapid global emergence of JN.1.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
