<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Integration of 168,000 samples reveals global patterns of the human gut microbiome
Authors: Abdill, R. J.; Graham, S. P.; Rubinetti, V.; Albert, F. W.; Greene, C. S.; Davis, S.; Blekhman, R.
Score: 79.6, Published: 2023-10-11 DOI: 10.1101/2023.10.11.560955
Understanding the factors that shape variation in the human microbiome is a major goal of research in biology. While other genomics fields have used large, pre-compiled compendia to extract systematic insights requiring otherwise impractical sample sizes, there has been no comparable resource for the 16S rRNA sequencing data commonly used to quantify microbiome composition.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Integration of 168,000 samples reveals global patterns of the human gut microbiome
Authors: Abdill, R. J.; Graham, S. P.; Rubinetti, V.; Albert, F. W.; Greene, C. S.; Davis, S.; Blekhman, R.
Score: 79.6, Published: 2023-10-11 DOI: 10.1101/2023.10.11.560955
Understanding the factors that shape variation in the human microbiome is a major goal of research in biology. While other genomics fields have used large, pre-compiled compendia to extract systematic insights requiring otherwise impractical sample sizes, there has been no comparable resource for the 16S rRNA sequencing data commonly used to quantify microbiome composition." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-15T10:38:01+00:00" />
<meta property="article:modified_time" content="2023-10-15T10:38:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Integration of 168,000 samples reveals global patterns of the human gut microbiome
Authors: Abdill, R. J.; Graham, S. P.; Rubinetti, V.; Albert, F. W.; Greene, C. S.; Davis, S.; Blekhman, R.
Score: 79.6, Published: 2023-10-11 DOI: 10.1101/2023.10.11.560955
Understanding the factors that shape variation in the human microbiome is a major goal of research in biology. While other genomics fields have used large, pre-compiled compendia to extract systematic insights requiring otherwise impractical sample sizes, there has been no comparable resource for the 16S rRNA sequencing data commonly used to quantify microbiome composition."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Integration of 168,000 samples reveals global patterns of the human gut microbiome\nAuthors: Abdill, R. J.; Graham, S. P.; Rubinetti, V.; Albert, F. W.; Greene, C. S.; Davis, S.; Blekhman, R.\nScore: 79.6, Published: 2023-10-11 DOI: 10.1101/2023.10.11.560955\nUnderstanding the factors that shape variation in the human microbiome is a major goal of research in biology. While other genomics fields have used large, pre-compiled compendia to extract systematic insights requiring otherwise impractical sample sizes, there has been no comparable resource for the 16S rRNA sequencing data commonly used to quantify microbiome composition.",
  "keywords": [
    
  ],
  "articleBody": " Integration of 168,000 samples reveals global patterns of the human gut microbiome\nAuthors: Abdill, R. J.; Graham, S. P.; Rubinetti, V.; Albert, F. W.; Greene, C. S.; Davis, S.; Blekhman, R.\nScore: 79.6, Published: 2023-10-11 DOI: 10.1101/2023.10.11.560955\nUnderstanding the factors that shape variation in the human microbiome is a major goal of research in biology. While other genomics fields have used large, pre-compiled compendia to extract systematic insights requiring otherwise impractical sample sizes, there has been no comparable resource for the 16S rRNA sequencing data commonly used to quantify microbiome composition. To help close this gap, we have assembled a set of 168,484 publicly available human gut microbiome samples, processed with a single pipeline and combined into the largest unified microbiome dataset to date. We use this resource, which is freely available at microbiomap.org, to shed light on global variation in the human gut microbiome. We find that Firmicutes, particularly Bacilli and Clostridia, are almost universally present in the human gut. At the same time, the relative abundance of the 65 most common microbial genera differ between at least two world regions. We also show that gut microbiomes in undersampled world regions, such as Central and Southern Asia, differ significantly from the more thoroughly characterized microbiomes of Europe and Northern America. Moreover, humans in these overlooked regions likely harbor hundreds of taxa that have not yet been discovered due to this undersampling, highlighting the need for diversity in microbiome studies. We anticipate that this new compendium can serve the community and enable advanced applied and methodological research.\nGenetically refactored Agrobacterium-mediated transformation\nAuthors: Thompson, M.; Kirkpatrick, L. D.; Geiselman, G. M.; Waldburger, L. M.; Pearson, A. N.; Szarzanowicz, M.; Vuu, K. M.; Markel, K.; Hummel, N. F.; Suazo, D. D.; Tahmin, C.; Cui, R.; Liu, S.; Cevallos, J.; Pannu, H.; Liu, D.; Gin, J. W.; Chen, Y.; Petzold, C. J.; Gladden, J.; Keasling, J. D.; Chang, J. H.; Weisberg, A. J.; Shih, P. M.\nScore: 29.4, Published: 2023-10-13 DOI: 10.1101/2023.10.13.561914\nMembers of Agrobacterium are costly plant pathogens while also essential tools for plant transformation. Though Agrobacterium-mediated transformation (AMT) has been heavily studied, its polygenic nature and its complex transcriptional regulation make identifying the genetic basis of transformational efficiency difficult through traditional genetic and bioinformatic approaches. Here we use a bottom-up synthetic approach to systematically refactor the tumor-inducing plasmid, wherein the majority of AMT machine components are encoded, into a minimal set of genes capable of plant and fungal transformation that is both controllable and orthogonal to its environment. We demonstrate that engineered vectors can be transferred to new heterologous bacteria, enabling them to transform plants. Our reductionist approach demonstrates how bottom-up engineering can be used to dissect and elucidate the genetic underpinnings of complex biological traits, and may lead to the development of strains of bacteria more capable of transforming recalcitrant plant species of societal importance.\nWidespread prevalence of a post-translational modification in activation of an essential bacterial DNA damage response\nAuthors: Kamat, A.; Tran, N.; Sharda, M.; Sontakke, N.; Le, T. B.; Badrinarayanan, A.\nScore: 25.7, Published: 2023-10-10 DOI: 10.1101/2023.10.09.561495\nDNA methylation plays central roles in diverse cellular processes, ranging from error-correction during replication to regulation of bacterial defense mechanisms. Nevertheless, certain aberrant methylation modifications can have lethal consequences. The mechanisms by which bacteria detect and respond to such damage remain incompletely understood. Here, we discover a highly conserved but previously uncharacterized transcription factor (Cada2), which orchestrates a methylation-dependent adaptive response in Caulobacter. This response operates independently of the SOS response, governs the expression of genes crucial for direct repair, and is essential for surviving methylation-induced damage. Our molecular investigation of Cada2 reveals a cysteine methylation-dependent post-translational modification and mode of action distinct from its E. coli counterpart, a trait conserved across all bacteria harboring a Cada2-like homolog instead. Extending across the bacterial kingdom, our findings support the notion of divergence and co-evolution of adaptive response transcription factors and their corresponding sequence-specific DNA motifs. Despite this diversity, the ubiquitous prevalence of adaptive response regulators underscores the significance of a transcriptional switch, mediated by methylation post-translational modification, in driving a specific and essential bacterial DNA damage response.\nA comparison of short-read, HiFi long-read, and hybrid strategies for genome-resolved metagenomics\nAuthors: Eisenhofer, R.; Nesme, J.; Santos-Bay, L.; Koziol, A.; Sorensen, S. J.; Alberdi, A.; Aizpurua, O.\nScore: 24.2, Published: 2023-10-06 DOI: 10.1101/2023.10.04.560907\nShotgun metagenomics enables the reconstruction of complex microbial communities at a high level of detail. Such an approach can be conducted using both short-read and long-read sequencing data, as well as a combination of both. To assess the pros and cons of these different approaches, we used 22 faecal DNA extracts collected weekly for 11 weeks from two respective lab mice to study seven performance metrics over four combinations of sequencing depth and technology: i) 20 Gbp of Illumina short-read data, ii) 40 Gbp of short-read data, iii) 20 Gbp of PacBio HiFi long-read data, and iv) 40 Gbp of hybrid (20 Gbp of short-read + 20 Gbp of long-read) data. No strategy was best for all metrics, but instead, each one excelled across different metrics. The long-read approach yielded the best assembly statistics, with the highest N50 and lowest number of contigs. The 40 Gbp short-read approach yielded the highest number of refined bins. Finally, the hybrid approach yielded the longest assemblies, and the highest mapping rate to the bacterial genomes. Our results suggest that while long-read sequencing significantly improves the quality of reconstructed bacterial genomes, it is more expensive and requires deeper sequencing than short-read approaches to recover a comparable amount of reconstructed genomes. The most optimal strategy is study-specific, and depends on how researchers assess the tradeoff between the quantity and quality of recovered genomes. ImportanceOur understanding of microbial communities is limited by the technologies we employ. Here, we test several different DNA sequencing techniques to better understand the pros and cons of each. Long read DNA sequencing allowed for the reconstruction of higher quality and even complete microbial genomes, however, the cost was greater than commonly used short-read DNA sequencing. We suggest researchers consider the trade-offs between each method and decide based on the goals of their research question/s.\nCultivating efficiency: High-throughput growth analysis of anaerobic bacteria in compact microplate readers.\nAuthors: Snoeyenbos-West, O. L. O.; Guerrero, C. R.; Valencia, M.; Carini, P.\nScore: 19.1, Published: 2023-10-10 DOI: 10.1101/2023.10.10.561742\nAnaerobic microbes play crucial roles in environmental processes, industry, and human health. Traditional methods for monitoring the growth of anaerobes, including plate counts or subsampling broth cultures for optical density measurements, are time and resource intensive. The advent of microplate readers revolutionized bacterial growth studies by enabling high-throughput and real-time monitoring of microbial growth kinetics but their use in anaerobic microbiology has remained limited. Here, we present a workflow for using small-footprint microplate readers and the Growthcurver R package to analyze the kinetic growth metrics of anaerobic bacteria. We benchmarked the small-footprint Cerillo Stratus microplate reader against a BioTek Synergy HTX microplate reader in aerobic conditions using Escherichia coli DSM 28618 cultures. The growth rates and carrying capacities obtained from the two readers were statistically indistinguishable. However, the area under the logistic curve was significantly higher in cultures monitored by the Stratus reader. We used the Stratus to quantify the growth responses of anaerobically grown E. coli and Clostridium bolteae DSM 29485 to different doses of the toxin sodium arsenite. The growth of E. coli and C. bolteae was sensitive to arsenite doses of 1.3 M and 0.4 M, respectively. Complete inhibition of growth was achieved at 38 M arsenite for C. bolteae, and 338 M in E. coli. These results show that the Stratus performs similarly to a leading brand of microplate reader and can be reliably used in anaerobic conditions. We discuss the advantages of the small format microplate readers and our experiences with the Stratus. Importance statementWe present a workflow that facilitates the production and analysis of growth curves for anaerobic microbes using small-footprint microplate readers and an R script. This workflow is a cost and space-effective solution to most high-throughput solutions for collecting growth data from anaerobic microbes. This technology can be used for applications in which high-throughput would advance discovery, including microbial isolation, bioprospecting, co-culturing, host-microbe interactions, and drug/toxin-microbial interactions.\nThe cytoplasmic phosphate level has a central regulatory role in the phosphate starvation response of Caulobacter crescentus\nAuthors: Billini, M.; Hoffmann, T.; Kuhn, J.; Bremer, E.; Thanbichler, M.\nScore: 15.8, Published: 2023-10-07 DOI: 10.1101/2023.10.07.561339\nIn bacteria, the availability of environmental inorganic phosphate is typically sensed by the conserved PhoR-PhoB two-component signal transduction pathway, which is thought to use the flux through the PstSCAB phosphate transporter as a readout of the extracellular phosphate level to control phosphate-responsive genes. While the sensing of environmental phosphate is well-investigated, the regulatory effects of cytoplasmic phosphate are unclear. Here, we disentangle the physiological and transcriptional responses of Caulobacter crescentus to changes in the environmental and cytoplasmic phosphate levels by uncoupling phosphate uptake from the activity of the PstSCAB system, using an additional, heterologously produced phosphate transporter. This approach reveals a two-pronged response of C. crescentus to phosphate limitation, in which PhoR-PhoB signaling mostly facilitates the utilization of alternative phosphate sources, whereas the cytoplasmic phosphate level controls the morphological and physiological adaptation of cells to growth under global phosphate limitation. These findings open the door to a comprehensive understanding of phosphate signaling in bacteria.\nAminoglycoside heteroresistance in Enterobacter cloacae is driven by the cell envelope stress response.\nAuthors: Choi, A. J.; Bennison, D. J.; Kulkarni, E.; Sun, H.; Li, H.; Bradshaw, J.; Yeap, H. W.; Mishra, V.; Crespo-Puig, A.; Davies, F.; Sriskandan, S.; Shenoy, A. R.\nScore: 13.3, Published: 2023-10-11 DOI: 10.1101/2023.10.11.561937\nEnterobacter cloacae is a Gram-negative nosocomial pathogen of the ESKAPE priority group with increasing multi-drug resistance via the acquisition of resistance plasmids. However, E. cloacae can also display phenotypic antimicrobial resistance, such as heteroresistance or persistence. Here we report that E. cloacae ATCC 13047 and six strains isolated from patients with blood infections display heteroresistance or persistence to aminoglycosides. E. cloacae heteroresistance is transient, accompanied with formation of \"petite\" colonies and increased MIC against gentamicin and other aminoglycosides used in the clinic, but not other antibiotic classes. To explore the underlying mechanisms, we performed RNA sequencing of heteroresistant bacteria, which revealed global gene-expression changes and a signature of the CpxRA cell envelope stress response. Deletion of the cpxRA two-component system abrogated aminoglycoside heteroresistance and petite colony formation, pointing to its indispensable role in phenotypic resistance. The introduction of a constitutively active allele of cpxA led to high aminoglycoside MICs, consistent with cell envelope stress driving these behaviours in E. cloacae. Cell envelope stress can be caused by environmental cues, including heavy metals. Indeed, bacterial exposure to copper increased gentamicin MIC in the wild type, but not the {Delta}cpxRA mutant. Moreover, copper exposure also elevated the gentamicin MICs of bloodstream isolates, suggesting that CpxRA- and copper-dependent aminoglycoside resistance is broadly conserved in E. cloacae strains. Altogether, we establish that E. cloacae relies on transcriptional reprogramming via the envelope stress response pathway for transient resistance to a major class of frontline antibiotic.\nAustralian terrestrial environments harbour extensive RNA virus diversity\nAuthors: Sadiq, S.; Harvey, E.; Mifsud, J. C.; Minasny, B.; McBratney, A. B.; Pozza, L. E.; Mahar, J. E.; Holmes, E. C.\nScore: 12.7, Published: 2023-10-10 DOI: 10.1101/2023.10.09.561620\nAustralia is home to a diverse range of unique native fauna and flora. To address whether Australian ecosystems also harbour unique viruses, we performed meta-transcriptomic sequencing of 16 farmland and sediment samples taken from the east and west coasts of Australia. We identified 2,562 putatively novel viruses across 15 orders, the vast majority of which belonged to the microbe-associated phylum Lenarviricota. In many orders, the novel viruses identified here comprised entirely new clades, such as the Nodamuvirales and Ghabrivirales. Novel viruses also fell between established genera or families, such as in the Cystoviridae and Picornavirales, while highly divergent lineages were identified in the Martellivirales and Ghabrivirales. Viral abundance and alpha diversity were influenced by sampling site, soil type and land use, but not by depth from the surface. In sum, Australian soils and sediments are home to remarkable viral diversity, reflecting the biodiversity of local fauna and flora.\nUsing population dynamics to count bacteriophages and their lysogens\nAuthors: Geng, Y.; Nguyen, T. V. P.; Homaee, E.; Golding, I.\nScore: 14.3, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561271\nTraditional assays for counting bacteriophages and their lysogens are labor-intensive and highly perturbative to the host cells. Here, we present a high-throughput infection method where all steps--cell growth, viral encounters, and post-infection recovery--take place in a microplate reader, and the growth dynamics of the infected culture are measured continuously using the optical density (OD). We find that the post-infection dynamics are reproducible and interpretable. In particular, the OD at which the culture lyses scales linearly with the logarithm of the initial phage concentration, providing a way of measuring phage numbers in unknown samples over nine decades and down to single-phage sensitivity. Interpreting the measured dynamics using a mathematical model for the coupled kinetics of phages and bacteria further allows us to infer the rates of viral encounters and cell lysis. Adding a single step of antibiotic selection provides the ability to measure the rate of host lysogenization. To demonstrate the application of our assay, we characterized the effect of bacterial growth rate on the propensity of lambda phage to lysogenize E. coli. When infected by a single phage, the probability of lysogenization is found to decrease approximately exponentially with the host growth rate. In growing, but not in stationary, cells, the propensity to lysogenize increases ~50-fold when multiple phages co-infect the cell. These findings illuminate how host physiology feeds into the lysis/lysogeny decision circuit, and demonstrate the utility of high-throughput infection to interrogating phage-host interactions.\nTemperature Influences Commensal-Pathogen Dynamics in a Nasal Epithelial Cell Co-culture Model\nAuthors: Huffines, J. T.; Boone, R. L.; Kiedrowski, M. R.\nScore: 10.5, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561218\nChronic rhinosinusitis (CRS) is an inflammatory disease of the paranasal sinuses, and microbial dysbiosis associated with CRS is thought to be a key driver of host inflammation that contributes to disease progression. Staphylococcus aureus is a common upper respiratory tract (URT) pathobiont that is associated with higher carriage rates in CRS populations, where S. aureus secreted toxins can be identified in CRS tissue samples. Although many genera of bacteria colonize the URT, relatively few account for the majority of sequencing reads. These include S. aureus, as well as several species belonging to the genus Corynebacterium, including Corynebacterium propinquum and Corynebacterium pseudodiphtheriticum, which are observed at high relative abundance in the URT of healthy individuals. Studies have examined the bacterial interactions between the major microbionts of the URT and S. aureus, but few have done so in the context of a healthy versus diseased URT environment. Here, we examine the role of temperature in commensal, pathogen, and epithelial dynamics using an air-liquid interface cell culture model mimicking the nasal epithelial environment. The healthy URT temperature changes from the nares to the nasopharynx and is altered during disease. Temperatures representative of the healthy URT increase persistence and aggregate formation of commensal C. propinquum and C. pseudodiphtheriticum, reduce S. aureus growth, and lower epithelial cytotoxicity compared to higher temperatures correlating with the diseased CRS sinus. Dual-species colonization revealed species-specific interactions between commensal Corynebacterium species and S. aureus dependent on temperature. Our findings suggest that URT mucosal temperature plays a significant role in mediating polymicrobial and host-bacterial interactions that may exacerbate microbial dysbiosis found in chronic URT disease. IMPORTANCEChronic rhinosinusitis is a complex inflammatory disease with a significant healthcare burden. Although presence of S. aureus and microbial dysbiosis are considered mediators of inflammation in CRS, no studies have examined the influence of temperature on S. aureus interactions with the nasal epithelium and the dominant genus of the healthy URT, Corynebacterium. Interactions between Corynebacterium species and S. aureus have been documented in several studies, but none to date have examined how environmental changes in the URT may alter their interactions with the epithelium or each other. This study utilizes a polarized epithelial cell culture model at air-liquid interface to study the colonization and spatial dynamics of S. aureus and clinical isolates of Corynebacterium from people with CRS to characterize the role temperature has in single-and dual-species dynamics on the nasal epithelium.\n",
  "wordCount" : "2712",
  "inLanguage": "en",
  "datePublished": "2023-10-15T10:38:01Z",
  "dateModified": "2023-10-15T10:38:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta"><span>updated on October 15, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.11.560955">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.11.560955" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.11.560955">
        <p class="paperTitle">Integration of 168,000 samples reveals global patterns of the human gut microbiome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.11.560955" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.11.560955" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Abdill, R. J.; Graham, S. P.; Rubinetti, V.; Albert, F. W.; Greene, C. S.; Davis, S.; Blekhman, R.</p>
        <p class="info">Score: 79.6, Published: 2023-10-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.11.560955' target='https://doi.org/10.1101/2023.10.11.560955'> 10.1101/2023.10.11.560955</a></p>
        <p class="abstract">Understanding the factors that shape variation in the human microbiome is a major goal of research in biology. While other genomics fields have used large, pre-compiled compendia to extract systematic insights requiring otherwise impractical sample sizes, there has been no comparable resource for the 16S rRNA sequencing data commonly used to quantify microbiome composition. To help close this gap, we have assembled a set of 168,484 publicly available human gut microbiome samples, processed with a single pipeline and combined into the largest unified microbiome dataset to date. We use this resource, which is freely available at microbiomap.org, to shed light on global variation in the human gut microbiome. We find that Firmicutes, particularly Bacilli and Clostridia, are almost universally present in the human gut. At the same time, the relative abundance of the 65 most common microbial genera differ between at least two world regions. We also show that gut microbiomes in undersampled world regions, such as Central and Southern Asia, differ significantly from the more thoroughly characterized microbiomes of Europe and Northern America. Moreover, humans in these overlooked regions likely harbor hundreds of taxa that have not yet been discovered due to this undersampling, highlighting the need for diversity in microbiome studies. We anticipate that this new compendium can serve the community and enable advanced applied and methodological research.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.561914">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.561914" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.561914">
        <p class="paperTitle">Genetically refactored Agrobacterium-mediated transformation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.561914" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.561914" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Thompson, M.; Kirkpatrick, L. D.; Geiselman, G. M.; Waldburger, L. M.; Pearson, A. N.; Szarzanowicz, M.; Vuu, K. M.; Markel, K.; Hummel, N. F.; Suazo, D. D.; Tahmin, C.; Cui, R.; Liu, S.; Cevallos, J.; Pannu, H.; Liu, D.; Gin, J. W.; Chen, Y.; Petzold, C. J.; Gladden, J.; Keasling, J. D.; Chang, J. H.; Weisberg, A. J.; Shih, P. M.</p>
        <p class="info">Score: 29.4, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.561914' target='https://doi.org/10.1101/2023.10.13.561914'> 10.1101/2023.10.13.561914</a></p>
        <p class="abstract">Members of Agrobacterium are costly plant pathogens while also essential tools for plant transformation. Though Agrobacterium-mediated transformation (AMT) has been heavily studied, its polygenic nature and its complex transcriptional regulation make identifying the genetic basis of transformational efficiency difficult through traditional genetic and bioinformatic approaches. Here we use a bottom-up synthetic approach to systematically refactor the tumor-inducing plasmid, wherein the majority of AMT machine components are encoded, into a minimal set of genes capable of plant and fungal transformation that is both controllable and orthogonal to its environment. We demonstrate that engineered vectors can be transferred to new heterologous bacteria, enabling them to transform plants. Our reductionist approach demonstrates how bottom-up engineering can be used to dissect and elucidate the genetic underpinnings of complex biological traits, and may lead to the development of strains of bacteria more capable of transforming recalcitrant plant species of societal importance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561495">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561495" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561495">
        <p class="paperTitle">Widespread prevalence of a post-translational modification in activation of an essential bacterial DNA damage response</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561495" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561495" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kamat, A.; Tran, N.; Sharda, M.; Sontakke, N.; Le, T. B.; Badrinarayanan, A.</p>
        <p class="info">Score: 25.7, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561495' target='https://doi.org/10.1101/2023.10.09.561495'> 10.1101/2023.10.09.561495</a></p>
        <p class="abstract">DNA methylation plays central roles in diverse cellular processes, ranging from error-correction during replication to regulation of bacterial defense mechanisms. Nevertheless, certain aberrant methylation modifications can have lethal consequences. The mechanisms by which bacteria detect and respond to such damage remain incompletely understood. Here, we discover a highly conserved but previously uncharacterized transcription factor (Cada2), which orchestrates a methylation-dependent adaptive response in Caulobacter. This response operates independently of the SOS response, governs the expression of genes crucial for direct repair, and is essential for surviving methylation-induced damage. Our molecular investigation of Cada2 reveals a cysteine methylation-dependent post-translational modification and mode of action distinct from its E. coli counterpart, a trait conserved across all bacteria harboring a Cada2-like homolog instead. Extending across the bacterial kingdom, our findings support the notion of divergence and co-evolution of adaptive response transcription factors and their corresponding sequence-specific DNA motifs. Despite this diversity, the ubiquitous prevalence of adaptive response regulators underscores the significance of a transcriptional switch, mediated by methylation post-translational modification, in driving a specific and essential bacterial DNA damage response.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560907">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560907" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560907">
        <p class="paperTitle">A comparison of short-read, HiFi long-read, and hybrid strategies for genome-resolved metagenomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560907" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560907" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eisenhofer, R.; Nesme, J.; Santos-Bay, L.; Koziol, A.; Sorensen, S. J.; Alberdi, A.; Aizpurua, O.</p>
        <p class="info">Score: 24.2, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560907' target='https://doi.org/10.1101/2023.10.04.560907'> 10.1101/2023.10.04.560907</a></p>
        <p class="abstract">Shotgun metagenomics enables the reconstruction of complex microbial communities at a high level of detail. Such an approach can be conducted using both short-read and long-read sequencing data, as well as a combination of both. To assess the pros and cons of these different approaches, we used 22 faecal DNA extracts collected weekly for 11 weeks from two respective lab mice to study seven performance metrics over four combinations of sequencing depth and technology: i) 20 Gbp of Illumina short-read data, ii) 40 Gbp of short-read data, iii) 20 Gbp of PacBio HiFi long-read data, and iv) 40 Gbp of hybrid (20 Gbp of short-read &#43; 20 Gbp of long-read) data. No strategy was best for all metrics, but instead, each one excelled across different metrics. The long-read approach yielded the best assembly statistics, with the highest N50 and lowest number of contigs. The 40 Gbp short-read approach yielded the highest number of refined bins. Finally, the hybrid approach yielded the longest assemblies, and the highest mapping rate to the bacterial genomes. Our results suggest that while long-read sequencing significantly improves the quality of reconstructed bacterial genomes, it is more expensive and requires deeper sequencing than short-read approaches to recover a comparable amount of reconstructed genomes. The most optimal strategy is study-specific, and depends on how researchers assess the tradeoff between the quantity and quality of recovered genomes.

ImportanceOur understanding of microbial communities is limited by the technologies we employ. Here, we test several different DNA sequencing techniques to better understand the pros and cons of each. Long read DNA sequencing allowed for the reconstruction of higher quality and even complete microbial genomes, however, the cost was greater than commonly used short-read DNA sequencing. We suggest researchers consider the trade-offs between each method and decide based on the goals of their research question/s.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561742">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561742" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561742">
        <p class="paperTitle">Cultivating efficiency: High-throughput growth analysis of anaerobic bacteria in compact microplate readers.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561742" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561742" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Snoeyenbos-West, O. L. O.; Guerrero, C. R.; Valencia, M.; Carini, P.</p>
        <p class="info">Score: 19.1, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561742' target='https://doi.org/10.1101/2023.10.10.561742'> 10.1101/2023.10.10.561742</a></p>
        <p class="abstract">Anaerobic microbes play crucial roles in environmental processes, industry, and human health. Traditional methods for monitoring the growth of anaerobes, including plate counts or subsampling broth cultures for optical density measurements, are time and resource intensive. The advent of microplate readers revolutionized bacterial growth studies by enabling high-throughput and real-time monitoring of microbial growth kinetics but their use in anaerobic microbiology has remained limited. Here, we present a workflow for using small-footprint microplate readers and the Growthcurver R package to analyze the kinetic growth metrics of anaerobic bacteria. We benchmarked the small-footprint Cerillo Stratus microplate reader against a BioTek Synergy HTX microplate reader in aerobic conditions using Escherichia coli DSM 28618 cultures. The growth rates and carrying capacities obtained from the two readers were statistically indistinguishable. However, the area under the logistic curve was significantly higher in cultures monitored by the Stratus reader. We used the Stratus to quantify the growth responses of anaerobically grown E. coli and Clostridium bolteae DSM 29485 to different doses of the toxin sodium arsenite. The growth of E. coli and C. bolteae was sensitive to arsenite doses of 1.3 M and 0.4 M, respectively. Complete inhibition of growth was achieved at 38 M arsenite for C. bolteae, and 338 M in E. coli. These results show that the Stratus performs similarly to a leading brand of microplate reader and can be reliably used in anaerobic conditions. We discuss the advantages of the small format microplate readers and our experiences with the Stratus.

Importance statementWe present a workflow that facilitates the production and analysis of growth curves for anaerobic microbes using small-footprint microplate readers and an R script. This workflow is a cost and space-effective solution to most high-throughput solutions for collecting growth data from anaerobic microbes. This technology can be used for applications in which high-throughput would advance discovery, including microbial isolation, bioprospecting, co-culturing, host-microbe interactions, and drug/toxin-microbial interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.07.561339">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.07.561339" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.07.561339">
        <p class="paperTitle">The cytoplasmic phosphate level has a central regulatory role in the phosphate starvation response of Caulobacter crescentus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.07.561339" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.07.561339" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Billini, M.; Hoffmann, T.; Kuhn, J.; Bremer, E.; Thanbichler, M.</p>
        <p class="info">Score: 15.8, Published: 2023-10-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.07.561339' target='https://doi.org/10.1101/2023.10.07.561339'> 10.1101/2023.10.07.561339</a></p>
        <p class="abstract">In bacteria, the availability of environmental inorganic phosphate is typically sensed by the conserved PhoR-PhoB two-component signal transduction pathway, which is thought to use the flux through the PstSCAB phosphate transporter as a readout of the extracellular phosphate level to control phosphate-responsive genes. While the sensing of environmental phosphate is well-investigated, the regulatory effects of cytoplasmic phosphate are unclear. Here, we disentangle the physiological and transcriptional responses of Caulobacter crescentus to changes in the environmental and cytoplasmic phosphate levels by uncoupling phosphate uptake from the activity of the PstSCAB system, using an additional, heterologously produced phosphate transporter. This approach reveals a two-pronged response of C. crescentus to phosphate limitation, in which PhoR-PhoB signaling mostly facilitates the utilization of alternative phosphate sources, whereas the cytoplasmic phosphate level controls the morphological and physiological adaptation of cells to growth under global phosphate limitation. These findings open the door to a comprehensive understanding of phosphate signaling in bacteria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.11.561937">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.11.561937" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.11.561937">
        <p class="paperTitle">Aminoglycoside heteroresistance in Enterobacter cloacae is driven by the cell envelope stress response.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.11.561937" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.11.561937" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Choi, A. J.; Bennison, D. J.; Kulkarni, E.; Sun, H.; Li, H.; Bradshaw, J.; Yeap, H. W.; Mishra, V.; Crespo-Puig, A.; Davies, F.; Sriskandan, S.; Shenoy, A. R.</p>
        <p class="info">Score: 13.3, Published: 2023-10-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.11.561937' target='https://doi.org/10.1101/2023.10.11.561937'> 10.1101/2023.10.11.561937</a></p>
        <p class="abstract">Enterobacter cloacae is a Gram-negative nosocomial pathogen of the ESKAPE priority group with increasing multi-drug resistance via the acquisition of resistance plasmids. However, E. cloacae can also display phenotypic antimicrobial resistance, such as heteroresistance or persistence. Here we report that E. cloacae ATCC 13047 and six strains isolated from patients with blood infections display heteroresistance or persistence to aminoglycosides. E. cloacae heteroresistance is transient, accompanied with formation of &#34;petite&#34; colonies and increased MIC against gentamicin and other aminoglycosides used in the clinic, but not other antibiotic classes. To explore the underlying mechanisms, we performed RNA sequencing of heteroresistant bacteria, which revealed global gene-expression changes and a signature of the CpxRA cell envelope stress response. Deletion of the cpxRA two-component system abrogated aminoglycoside heteroresistance and petite colony formation, pointing to its indispensable role in phenotypic resistance. The introduction of a constitutively active allele of cpxA led to high aminoglycoside MICs, consistent with cell envelope stress driving these behaviours in E. cloacae. Cell envelope stress can be caused by environmental cues, including heavy metals. Indeed, bacterial exposure to copper increased gentamicin MIC in the wild type, but not the {Delta}cpxRA mutant. Moreover, copper exposure also elevated the gentamicin MICs of bloodstream isolates, suggesting that CpxRA- and copper-dependent aminoglycoside resistance is broadly conserved in E. cloacae strains. Altogether, we establish that E. cloacae relies on transcriptional reprogramming via the envelope stress response pathway for transient resistance to a major class of frontline antibiotic.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561620">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561620" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561620">
        <p class="paperTitle">Australian terrestrial environments harbour extensive RNA virus diversity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561620" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561620" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sadiq, S.; Harvey, E.; Mifsud, J. C.; Minasny, B.; McBratney, A. B.; Pozza, L. E.; Mahar, J. E.; Holmes, E. C.</p>
        <p class="info">Score: 12.7, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561620' target='https://doi.org/10.1101/2023.10.09.561620'> 10.1101/2023.10.09.561620</a></p>
        <p class="abstract">Australia is home to a diverse range of unique native fauna and flora. To address whether Australian ecosystems also harbour unique viruses, we performed meta-transcriptomic sequencing of 16 farmland and sediment samples taken from the east and west coasts of Australia. We identified 2,562 putatively novel viruses across 15 orders, the vast majority of which belonged to the microbe-associated phylum Lenarviricota. In many orders, the novel viruses identified here comprised entirely new clades, such as the Nodamuvirales and Ghabrivirales. Novel viruses also fell between established genera or families, such as in the Cystoviridae and Picornavirales, while highly divergent lineages were identified in the Martellivirales and Ghabrivirales. Viral abundance and alpha diversity were influenced by sampling site, soil type and land use, but not by depth from the surface. In sum, Australian soils and sediments are home to remarkable viral diversity, reflecting the biodiversity of local fauna and flora.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561271">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561271" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561271">
        <p class="paperTitle">Using population dynamics to count bacteriophages and their lysogens</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561271" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561271" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Geng, Y.; Nguyen, T. V. P.; Homaee, E.; Golding, I.</p>
        <p class="info">Score: 14.3, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561271' target='https://doi.org/10.1101/2023.10.06.561271'> 10.1101/2023.10.06.561271</a></p>
        <p class="abstract">Traditional assays for counting bacteriophages and their lysogens are labor-intensive and highly perturbative to the host cells. Here, we present a high-throughput infection method where all steps--cell growth, viral encounters, and post-infection recovery--take place in a microplate reader, and the growth dynamics of the infected culture are measured continuously using the optical density (OD). We find that the post-infection dynamics are reproducible and interpretable. In particular, the OD at which the culture lyses scales linearly with the logarithm of the initial phage concentration, providing a way of measuring phage numbers in unknown samples over nine decades and down to single-phage sensitivity. Interpreting the measured dynamics using a mathematical model for the coupled kinetics of phages and bacteria further allows us to infer the rates of viral encounters and cell lysis. Adding a single step of antibiotic selection provides the ability to measure the rate of host lysogenization. To demonstrate the application of our assay, we characterized the effect of bacterial growth rate on the propensity of lambda phage to lysogenize E. coli. When infected by a single phage, the probability of lysogenization is found to decrease approximately exponentially with the host growth rate. In growing, but not in stationary, cells, the propensity to lysogenize increases ~50-fold when multiple phages co-infect the cell. These findings illuminate how host physiology feeds into the lysis/lysogeny decision circuit, and demonstrate the utility of high-throughput infection to interrogating phage-host interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561218">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561218" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561218">
        <p class="paperTitle">Temperature Influences Commensal-Pathogen Dynamics in a Nasal Epithelial Cell Co-culture Model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561218" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561218" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Huffines, J. T.; Boone, R. L.; Kiedrowski, M. R.</p>
        <p class="info">Score: 10.5, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561218' target='https://doi.org/10.1101/2023.10.06.561218'> 10.1101/2023.10.06.561218</a></p>
        <p class="abstract">Chronic rhinosinusitis (CRS) is an inflammatory disease of the paranasal sinuses, and microbial dysbiosis associated with CRS is thought to be a key driver of host inflammation that contributes to disease progression. Staphylococcus aureus is a common upper respiratory tract (URT) pathobiont that is associated with higher carriage rates in CRS populations, where S. aureus secreted toxins can be identified in CRS tissue samples. Although many genera of bacteria colonize the URT, relatively few account for the majority of sequencing reads. These include S. aureus, as well as several species belonging to the genus Corynebacterium, including Corynebacterium propinquum and Corynebacterium pseudodiphtheriticum, which are observed at high relative abundance in the URT of healthy individuals. Studies have examined the bacterial interactions between the major microbionts of the URT and S. aureus, but few have done so in the context of a healthy versus diseased URT environment. Here, we examine the role of temperature in commensal, pathogen, and epithelial dynamics using an air-liquid interface cell culture model mimicking the nasal epithelial environment. The healthy URT temperature changes from the nares to the nasopharynx and is altered during disease. Temperatures representative of the healthy URT increase persistence and aggregate formation of commensal C. propinquum and C. pseudodiphtheriticum, reduce S. aureus growth, and lower epithelial cytotoxicity compared to higher temperatures correlating with the diseased CRS sinus. Dual-species colonization revealed species-specific interactions between commensal Corynebacterium species and S. aureus dependent on temperature. Our findings suggest that URT mucosal temperature plays a significant role in mediating polymicrobial and host-bacterial interactions that may exacerbate microbial dysbiosis found in chronic URT disease.

IMPORTANCEChronic rhinosinusitis is a complex inflammatory disease with a significant healthcare burden. Although presence of S. aureus and microbial dysbiosis are considered mediators of inflammation in CRS, no studies have examined the influence of temperature on S. aureus interactions with the nasal epithelium and the dominant genus of the healthy URT, Corynebacterium. Interactions between Corynebacterium species and S. aureus have been documented in several studies, but none to date have examined how environmental changes in the URT may alter their interactions with the epithelium or each other. This study utilizes a polarized epithelial cell culture model at air-liquid interface to study the colonization and spatial dynamics of S. aureus and clinical isolates of Corynebacterium from people with CRS to characterize the role temperature has in single-and dual-species dynamics on the nasal epithelium.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
