<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Amazing structural diversity of giant virus-like particles in forest soil
Authors: Fischer, M. G.; Mersdorf, U.; Blanchard, J. L.
Score: 96.8, Published: 2023-06-30 DOI: 10.1101/2023.06.30.546935
Large DNA viruses of the phylum Nucleocytoviricota infect diverse eukaryotic hosts from protists to humans, with profound consequences for aquatic and terrestrial ecosystems. While nucleocytoviruses are known to be highly diverse in metagenomes, knowledge of their capsid structures is restricted to a few characterized representatives.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Amazing structural diversity of giant virus-like particles in forest soil
Authors: Fischer, M. G.; Mersdorf, U.; Blanchard, J. L.
Score: 96.8, Published: 2023-06-30 DOI: 10.1101/2023.06.30.546935
Large DNA viruses of the phylum Nucleocytoviricota infect diverse eukaryotic hosts from protists to humans, with profound consequences for aquatic and terrestrial ecosystems. While nucleocytoviruses are known to be highly diverse in metagenomes, knowledge of their capsid structures is restricted to a few characterized representatives." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-02T10:39:32+00:00" />
<meta property="article:modified_time" content="2023-07-02T10:39:32+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Amazing structural diversity of giant virus-like particles in forest soil
Authors: Fischer, M. G.; Mersdorf, U.; Blanchard, J. L.
Score: 96.8, Published: 2023-06-30 DOI: 10.1101/2023.06.30.546935
Large DNA viruses of the phylum Nucleocytoviricota infect diverse eukaryotic hosts from protists to humans, with profound consequences for aquatic and terrestrial ecosystems. While nucleocytoviruses are known to be highly diverse in metagenomes, knowledge of their capsid structures is restricted to a few characterized representatives."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Amazing structural diversity of giant virus-like particles in forest soil\nAuthors: Fischer, M. G.; Mersdorf, U.; Blanchard, J. L.\nScore: 96.8, Published: 2023-06-30 DOI: 10.1101/2023.06.30.546935\nLarge DNA viruses of the phylum Nucleocytoviricota infect diverse eukaryotic hosts from protists to humans, with profound consequences for aquatic and terrestrial ecosystems. While nucleocytoviruses are known to be highly diverse in metagenomes, knowledge of their capsid structures is restricted to a few characterized representatives.",
  "keywords": [
    
  ],
  "articleBody": " Amazing structural diversity of giant virus-like particles in forest soil\nAuthors: Fischer, M. G.; Mersdorf, U.; Blanchard, J. L.\nScore: 96.8, Published: 2023-06-30 DOI: 10.1101/2023.06.30.546935\nLarge DNA viruses of the phylum Nucleocytoviricota infect diverse eukaryotic hosts from protists to humans, with profound consequences for aquatic and terrestrial ecosystems. While nucleocytoviruses are known to be highly diverse in metagenomes, knowledge of their capsid structures is restricted to a few characterized representatives. Here, we visualize giant virus-like particles (VLPs, diameter \u003e0.2 m) directly from the environment using transmission electron microscopy. We found that Harvard Forest soils contain a higher diversity of giant VLP morphotypes than all hitherto isolated giant viruses combined. These included VLPs with icosahedral capsid symmetry, ovoid shapes similar to pandoraviruses, and bacilliform shapes that may represent novel viruses. We discovered giant icosahedral capsids with structural modifications that had not been described before including tubular appendages, modified vertices, tails, and capsids consisting of multiple layers or internal channels. Many giant VLPs were covered with fibers of varying lengths, thicknesses, densities, and terminal structures. These findings imply that giant viruses employ a much wider array of capsid structures and mechanisms to interact with their host cells than is currently known. We also found diverse tailed bacteriophages and filamentous VLPs, as well as ultra-small cells. Our study offers a first glimpse of the vast diversity of unexplored viral structures in soil and reinforces the potential of transmission electron microscopy for fundamental discoveries in environmental microbiology.\nType IV-A3 CRISPR-Cas systems drive inter-plasmid conflicts by acquiring spacers in trans\nAuthors: Benz, F.; Camara-Wilpert, S.; Russel, J.; Wandera, K. G.; Cepaite, R.; Ares-Arroyo, M.; Gomes-Filho, J. V.; Englert, F.; Kuehn, J.; Gloor, S.; Cuenod, A.; Aguila-Sans, M.; Maccario, L.; Egli, A.; Randau, L.; Pausch, P.; Rocha, E.; Beisel, C. L.; Madsen, J. S.; Bikard, D.; Hall, A. R.; Soerensen, S. J.; Pinilla-Redondo, R.\nScore: 78.1, Published: 2023-06-23 DOI: 10.1101/2023.06.23.546257\nType IV-A CRISPR-Cas systems are primarily encoded on plasmids and form multi-subunit ribonucleoprotein complexes with unknown biological functions. In contrast to other CRISPR-Cas types, they lack the archetypical CRISPR acquisition module and encode a DinG helicase instead of a nuclease component. Type IV-A3 systems are carried by large conjugative plasmids that often harbor multiple antibiotic-resistance genes. Although their CRISPR array contents suggest a role in inter-plasmid conflicts, this function and the underlying mechanisms have remained unexplored. Here, we demonstrate that a plasmid-encoded type IV-A3 CRISPR-Cas system co-opts the type I-E adaptation machinery from its clinical Klebsiella pneumoniae host to update its CRISPR array. Furthermore, we demonstrate that robust interference of conjugative plasmids and phages is elicited through CRISPR RNA-dependent transcriptional repression. By targeting plasmid core functions, type IV-A3 can prevent the uptake of incoming plasmids, limit their horizontal transfer, and destabilize co-residing plasmids, altogether supporting type IV-A3s involvement in plasmid competition. Collectively, our findings shed light on the molecular mechanisms and ecological function of type IV-A3 systems and have broad implications for understanding and countering the spread of antibiotic resistance in clinically relevant strains.\nWaste not, want not: Revisiting the analysis that called into question the practice of rarefaction\nAuthors: Schloss, P. D.\nScore: 65.1, Published: 2023-06-26 DOI: 10.1101/2023.06.23.546312\nIn 2014, McMurdie and Holmes published the provocatively titled, \"Waste not, want not: why rarefying microbiome data is inadmissible\". The claims of their study have significantly altered how microbiome researchers control for the unavoidable uneven sequencing depths that are inherent in modern 16S rRNA gene sequencing. Confusion over the distinction between the definitions of rarefying and rarefaction continues to cloud the interpretation of their results. More importantly, the authors made a variety of choices in designing and analyzing their simulations that were potentially problematic. I identified 11 factors that could have compromised the results of the original study. I reproduced the original simulation results and assessed the impact of those 11 factors on the underlying conclusion that rarefying data is inadmissible. Throughout, the design of the original study made choices that caused rarefying and rarefaction to appear to perform worse than they truly did. Most important were the approaches used to assess ecological distances, the removal of samples with low sequencing depth, and not accounting for conditions where sequencing effort is confounded with treatment group. Although the original study criticized rarefying for the arbitrary removal of valid data, repeatedly rarefying data many times (i.e., rarefaction) incorporates all the data. In contrast, it is the removal of rare taxa that would appear to remove valid data. Overall, I show that rarefaction is the most robust approach to control for uneven sequencing effort when considered across a variety of alpha and beta diversity metrics. ImportanceOver the past 10 years, the best method for normalizing the sequencing depth of samples characterized by 16S rRNA gene sequencing has been contentious. An often cited paper by McMurdie and Holmes forcefully argued that rarefying the number of sequence counts was \"inadmissible\" and should not be employed. However, I identified a number of problems with the design of their simulations and analysis that could compromise their results. In fact, when I reproduced and expanded upon their analysis, it was clear that rarefaction was actually the most robust approach for controlling for uneven sequencing effort across samples. Rarefaction limits the rate of falsely detecting and rejecting differences between treatment groups. Far from being \"inadmissible\", rarefaction is a valuable tool for analyzing microbiome sequence data.\nSARS-CoV-2 spike glycosylation affects function and neutralization sensitivity\nAuthors: Zhang, F.; Schmidt, F.; Muecksch, F.; Wang, Z.; Gazumyan, A.; Nussenzweig, M. C.; Gaebler, C.; Caskey, M.; Hatziioannou, T.; Bieniasz, P. D.\nScore: 52.5, Published: 2023-06-30 DOI: 10.1101/2023.06.30.547241\nThe glycosylation of viral envelope proteins can play important roles in virus biology and immune evasion. The spike (S) glycoprotein of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) includes 22 N-linked glycosylation sequons and 17 O-linked glycosites. Here, we investigated the effect of individual glycosylation sites on SARS-CoV-2 S function in pseudotyped virus infection assays and on sensitivity to monoclonal and polyclonal neutralizing antibodies. In most cases, removal of individual glycosylation sites decreased the infectiousness of the pseudotyped virus. For glycosylation mutants in the N-terminal domain (NTD) and the receptor binding domain (RBD), reduction in pseudotype infectivity was predicted by a commensurate reduction in the level of virion-incorporated spike protein. Notably, the presence of a glycan at position N343 within the RBD had diverse effects on neutralization by RBD-specific monoclonal antibodies (mAbs) cloned from convalescent individuals. The N343 glycan reduced overall sensitivity to polyclonal antibodies in plasma from COVID-19 convalescent individuals, suggesting a role for SARS-CoV-2 spike glycosylation in immune evasion. However, vaccination of convalescent individuals produced neutralizing activity that was resilient to the inhibitory effect of the N343 glycan.\nEukaryotic antiviral immune proteins arose via convergence, horizontal transfer, and ancient inheritance\nAuthors: Culbertson, E. M.; Levin, T. C.\nScore: 51.6, Published: 2023-06-27 DOI: 10.1101/2023.06.27.546753\nAnimals use a variety of cell-autonomous innate immune proteins to detect viral infections and prevent replication. Recent studies have discovered that a subset of mammalian antiviral proteins have homology to anti-phage defense proteins in bacteria, implying that there are aspects of innate immunity that are shared across the Tree of Life. While the majority of these studies have focused on characterizing the diversity and biochemical functions of the bacterial proteins, the evolutionary relationships between animal and bacterial proteins are less clear. This ambiguity is partly due to the long evolutionary distances separating animal and bacterial proteins, which obscures their relationships. Here, we tackle this problem for three innate immune families (CD-NTases [including cGAS], STINGs, and Viperins) by deeply sampling protein diversity across eukaryotes. We find that Viperins are truly ancient immune proteins, likely inherited since the last eukaryotic common ancestor and possibly longer. In contrast, we find other immune proteins that arose via at least five independent events of horizontal gene transfer (HGT) from bacteria. Two of these events allowed algae to acquire new bacterial viperins, while three more HGT events gave rise to distinct superfamilies of eukaryotic CD-NTases: the OAS superfamily, which is widespread across eukaryotes; the Mab21 superfamily (containing cGAS) which has diversified via a series of animal-specific duplications, and a previously undefined eSMODS superfamily, which more closely resembles bacterial CD-NTases. Finally, we found that cGAS and STING proteins have substantially different histories, with STINGs arising via convergent domain shuffling in bacteria and eukaryotes. Overall, our findings paint a picture of eukaryotic innate immunity as highly dynamic, where eukaryotes build upon their ancient antiviral repertoires through the reuse of protein domains and by repeatedly sampling a rich reservoir of bacterial anti-phage genes.\nDiverse viral cas genes antagonize CRISPR immunity\nAuthors: Katz, M. A.; Sawyer, E. M.; Kozlova, A.; Williams, M. C.; Margolis, S. R.; Oriolt, L.; Johnson, M.; Bondy-Denomy, J.; Meeske, A. J.\nScore: 45.7, Published: 2023-06-24 DOI: 10.1101/2023.06.24.545427\nProkaryotic CRISPR-Cas immunity is subverted via anti-CRISPRs (Acrs), small proteins that inhibit Cas protein activities when expressed during the phage lytic cycle or from resident prophages or plasmids. CRISPR-Cas defenses are classified into 6 types and 33 subtypes, which employ a diverse suite of Cas effectors and differ in their mechanisms of interference. As Acrs often work via binding to a cognate Cas protein, inhibition is almost always limited to a single CRISPR type. Furthermore, while acr genes are frequently organized together in phage-associated gene clusters, how such inhibitors initially evolve has remained unclear. Here we have investigated the Acr content and inhibition specificity of a collection of Listeria isolates, which naturally harbor four diverse CRISPR-Cas systems (types I-B, II-A, II-C, and VI-A). We observed widespread antagonism of CRISPR, which we traced to 12 novel and 4 known Acr gene families encoded on endogenous mobile genetic elements. Among these were two Acrs that possess sequence homology to type I-B Cas proteins and assemble into a defective interference complex. Surprisingly, an additional type I-B Cas homolog did not affect type I immunity, but instead inhibited the RNA-targeting type VI CRISPR system through sequestration of crRNA. By probing the IMGVR database of viral genomes, we detected abundant orphan cas genes located within putative anti-defense gene clusters. We experimentally verified the Acr activity of one viral cas gene, a particularly broad-spectrum cas3 homolog that inhibits type I-B, II-A, and VI-A CRISPR immunity. Our observations provide direct evidence of Acr evolution via cas gene co-option, and new genes with potential for broad-spectrum control of genome editing technologies.\nAntiviral Type III CRISPR signalling via conjugation of ATP and AdoMet\nAuthors: Chi, H.; Hoikkala, V.; Gruschow, S.; Graham, S.; Shirran, S.; White, M. F.\nScore: 36.4, Published: 2023-06-27 DOI: 10.1101/2023.06.26.546636\nCRISPR systems are widespread in the prokaryotic world, providing adaptive immunity against mobile genetic elements (MGE) 1, 2. Type III CRISPR systems, with the signature gene cas10, use CRISPR RNA (crRNA) to detect non-self RNA, activating the enzymatic Cas10 subunit to defend the cell against MGE either directly, via the integral HD nuclease domain 3-5 or indirectly, via synthesis of cyclic oligonucleotide (cOA) second messengers to activate diverse ancillary effectors 6-9. A subset of type III CRISPR systems encode an uncharacterised CorA-family membrane protein and an associated NrN family phosphodiesterase predicted to function in antiviral defence. Here, we demonstrate that the CorA associated type III-B (Cmr) CRISPR system from Bacteroides fragilis provides immunity against MGE when expressed inE. coli. However, B. fragilis Cmr does not synthesise cOA species on activation, instead generating a previously undescribed sigalling molecule, SAM-AMP (3-adenylyl-AdoMet) by conjugating ATP to S-adenosyl methionine via a phosphodiester bond. Once synthesised, SAM-AMP binds to the CorA effector, presumably leading to cell death by disruption of the membrane integrity. SAM-AMP is degraded by CRISPR associated phosphodiesterases or a SAM-AMP lyase, providing an \"off switch\" analogous to cOA specific ring nucleases 10. SAM-AMP thus represents a new class of second messenger for antiviral signalling, which may function in different roles in diverse cellular contexts.\nVirome analysis of New Zealand's bats reveals cross-species viral transmission among the Coronaviridae\nAuthors: Waller, S. J.; Tortosa, P.; Thurley, T.; O'Donnell, C.; Jackson, R.; Dennis, G.; Grimwood, R. M.; Holmes, E. C.; McInnes, K.; Geoghegan, J. L.\nScore: 49.7, Published: 2023-06-20 DOI: 10.1101/2023.06.19.545647\n1.The lesser short-tailed bat (Mystacina tuberculata) and the long-tailed bat (Chalinolobus tuberculatus) are Aotearoa New Zealands only native extant terrestrial mammals and are believed to have migrated from Australia. Long-tailed bats arrived in New Zealand an estimated two million years ago and are closely related to other Australian bat species. Lesser short-tailed bats, in contrast, are the only extant species within the Mystacinidae and are estimated to have been living in isolation in New Zealand for the past 16-18 million years. Throughout this period of isolation, lesser short-tailed bats have become one of the most terrestrial bats in the world. Through a metatranscriptomic analysis of guano samples from eight locations across New Zealand we aimed to characterise the viromes of New Zealands bats and determine whether viruses have jumped between these species over the past two million years. High viral richness was observed among long-tailed bats with viruses spanning seven different viral families. In contrast, no bat-specific viruses were identified in lesser short-tailed bats. Both bat species harboured an abundance of likely dietary- and environmental-associated viruses. We also identified alphacoronaviruses in long-tailed bat guano that had previously been identified in lesser short-tailed bats, suggesting that these viruses had jumped the species barrier after long-tailed bats migrated to New Zealand. Of note, an alphacoronavirus species discovered here possessed a complete genome of only 22,416 nucleotides with entire deletions or truncations of several non-structural proteins, thereby representing what is possibly the shortest genome within the Coronaviridae identified to date. Overall, this study has revealed a diverse range of novel viruses harboured by New Zealands only native terrestrial mammals, in turn expanding our understanding of bat viral dynamics and evolution globally.\nRarefaction is currently the best approach to control for uneven sequencing effort in amplicon sequence analyses\nAuthors: Schloss, P. D.\nScore: 56.8, Published: 2023-06-23 DOI: 10.1101/2023.06.23.546313\nConsidering it is common to find as much as 100-fold variation in the number of 16S rRNA gene sequences found across samples in a study, researchers need to control for the effect of uneven sequencing effort. How to do this has become a contentious question. Some have argued that rarefying or rarefaction is \"inadmissible\" because it omits valid data. A number of alternative approaches have been developed to normalize and rescale the data that purport to be invariant to the number of observations. I generated community distributions based on 12 published datasets where I was able to assess the ability of multiple methods to control for uneven sequencing effort. Rarefaction was the only method that could control for variation in uneven sequencing effort when measuring commonly used alpha and beta diversity metrics. Next, I compared the false detection rate and power to detect true differences between simulated communities with a known effect size using various alpha and beta diversity metrics. Although all methods of controlling for uneven sequencing effort had an acceptable false detection rate when samples were randomly assigned to two treatment groups, rarefaction was consistently able to control for differences in sequencing effort when sequencing depth was confounded with treatment group. Finally, the statistical power to detect differences in alpha and beta diversity metrics was consistently the highest when using rarefaction. These simulations underscore the importance of using rarefaction to normalize the number of sequences across samples in amplicon sequencing analyses. ImportanceSequencing 16S rRNA gene fragments has become a fundamental tool for understanding the diversity of microbial communities and the factors that affect their diversity. Due to technical challenges, it is common to observe wide variation in the number of sequences that are collected from different samples within the same study. However, the diversity metrics used by microbial ecologists are sensitive to differences in sequencing effort. Therefore, tools are needed to control for the uneven levels of sequencing. This simulation-based analysis shows that despite a longstanding controversy, rarefaction is the most robust approach to control for uneven sequencing effort. The controversy started because of confusion over the definition of rarefaction and violation of assumptions that are made by methods that have been borrowed from other fields. Microbial ecologists should use rarefaction.\nConstruction and Characterisation of a Structured, Tuneable, and Transparent 3D Culture Platform for Soil Bacteria\nAuthors: Rooney, L. M.; Dupuy, L. X.; Hoskisson, P. A.; McConnell, G.\nScore: 24.5, Published: 2023-06-28 DOI: 10.1101/2023.06.28.546105\nWe have developed a tuneable workflow for the study of soil microbes in an imitative 3D soil environment that is compatible with routine and advanced optical imaging, is chemically customisable, and is reliably refractive index matched based on the metabolic profile of the study organism. We demonstrate our transparent soil pipeline with two representative soil organisms, Bacillus subtilis and Streptomyces coelicolor, and visualise their colonisation behaviours using fluorescence microscopy and mesoscopy. This spatially structured, 3D approach to microbial culture has the potential to further study the behaviour of other difficult-to-culture bacteria in conditions matching their native environment and could be expanded to study microbial interactions, such as interaction, competition, and warfare.\n",
  "wordCount" : "2822",
  "inLanguage": "en",
  "datePublished": "2023-07-02T10:39:32Z",
  "dateModified": "2023-07-02T10:39:32Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta"><span>updated on July 2, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.546935">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.546935" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.546935">
        <p class="paperTitle">Amazing structural diversity of giant virus-like particles in forest soil</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.546935" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.546935" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fischer, M. G.; Mersdorf, U.; Blanchard, J. L.</p>
        <p class="info">Score: 96.8, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.546935' target='https://doi.org/10.1101/2023.06.30.546935'> 10.1101/2023.06.30.546935</a></p>
        <p class="abstract">Large DNA viruses of the phylum Nucleocytoviricota infect diverse eukaryotic hosts from protists to humans, with profound consequences for aquatic and terrestrial ecosystems. While nucleocytoviruses are known to be highly diverse in metagenomes, knowledge of their capsid structures is restricted to a few characterized representatives. Here, we visualize giant virus-like particles (VLPs, diameter &gt;0.2 m) directly from the environment using transmission electron microscopy. We found that Harvard Forest soils contain a higher diversity of giant VLP morphotypes than all hitherto isolated giant viruses combined. These included VLPs with icosahedral capsid symmetry, ovoid shapes similar to pandoraviruses, and bacilliform shapes that may represent novel viruses. We discovered giant icosahedral capsids with structural modifications that had not been described before including tubular appendages, modified vertices, tails, and capsids consisting of multiple layers or internal channels. Many giant VLPs were covered with fibers of varying lengths, thicknesses, densities, and terminal structures. These findings imply that giant viruses employ a much wider array of capsid structures and mechanisms to interact with their host cells than is currently known. We also found diverse tailed bacteriophages and filamentous VLPs, as well as ultra-small cells. Our study offers a first glimpse of the vast diversity of unexplored viral structures in soil and reinforces the potential of transmission electron microscopy for fundamental discoveries in environmental microbiology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546257">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546257" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546257">
        <p class="paperTitle">Type IV-A3 CRISPR-Cas systems drive inter-plasmid conflicts by acquiring spacers in trans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546257" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546257" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Benz, F.; Camara-Wilpert, S.; Russel, J.; Wandera, K. G.; Cepaite, R.; Ares-Arroyo, M.; Gomes-Filho, J. V.; Englert, F.; Kuehn, J.; Gloor, S.; Cuenod, A.; Aguila-Sans, M.; Maccario, L.; Egli, A.; Randau, L.; Pausch, P.; Rocha, E.; Beisel, C. L.; Madsen, J. S.; Bikard, D.; Hall, A. R.; Soerensen, S. J.; Pinilla-Redondo, R.</p>
        <p class="info">Score: 78.1, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546257' target='https://doi.org/10.1101/2023.06.23.546257'> 10.1101/2023.06.23.546257</a></p>
        <p class="abstract">Type IV-A CRISPR-Cas systems are primarily encoded on plasmids and form multi-subunit ribonucleoprotein complexes with unknown biological functions. In contrast to other CRISPR-Cas types, they lack the archetypical CRISPR acquisition module and encode a DinG helicase instead of a nuclease component. Type IV-A3 systems are carried by large conjugative plasmids that often harbor multiple antibiotic-resistance genes. Although their CRISPR array contents suggest a role in inter-plasmid conflicts, this function and the underlying mechanisms have remained unexplored. Here, we demonstrate that a plasmid-encoded type IV-A3 CRISPR-Cas system co-opts the type I-E adaptation machinery from its clinical Klebsiella pneumoniae host to update its CRISPR array. Furthermore, we demonstrate that robust interference of conjugative plasmids and phages is elicited through CRISPR RNA-dependent transcriptional repression. By targeting plasmid core functions, type IV-A3 can prevent the uptake of incoming plasmids, limit their horizontal transfer, and destabilize co-residing plasmids, altogether supporting type IV-A3s involvement in plasmid competition. Collectively, our findings shed light on the molecular mechanisms and ecological function of type IV-A3 systems and have broad implications for understanding and countering the spread of antibiotic resistance in clinically relevant strains.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546312">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546312" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546312">
        <p class="paperTitle">Waste not, want not: Revisiting the analysis that called into question the practice of rarefaction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546312" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546312" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schloss, P. D.</p>
        <p class="info">Score: 65.1, Published: 2023-06-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546312' target='https://doi.org/10.1101/2023.06.23.546312'> 10.1101/2023.06.23.546312</a></p>
        <p class="abstract">In 2014, McMurdie and Holmes published the provocatively titled, &#34;Waste not, want not: why rarefying microbiome data is inadmissible&#34;. The claims of their study have significantly altered how microbiome researchers control for the unavoidable uneven sequencing depths that are inherent in modern 16S rRNA gene sequencing. Confusion over the distinction between the definitions of rarefying and rarefaction continues to cloud the interpretation of their results. More importantly, the authors made a variety of choices in designing and analyzing their simulations that were potentially problematic. I identified 11 factors that could have compromised the results of the original study. I reproduced the original simulation results and assessed the impact of those 11 factors on the underlying conclusion that rarefying data is inadmissible. Throughout, the design of the original study made choices that caused rarefying and rarefaction to appear to perform worse than they truly did. Most important were the approaches used to assess ecological distances, the removal of samples with low sequencing depth, and not accounting for conditions where sequencing effort is confounded with treatment group. Although the original study criticized rarefying for the arbitrary removal of valid data, repeatedly rarefying data many times (i.e., rarefaction) incorporates all the data. In contrast, it is the removal of rare taxa that would appear to remove valid data. Overall, I show that rarefaction is the most robust approach to control for uneven sequencing effort when considered across a variety of alpha and beta diversity metrics.

ImportanceOver the past 10 years, the best method for normalizing the sequencing depth of samples characterized by 16S rRNA gene sequencing has been contentious. An often cited paper by McMurdie and Holmes forcefully argued that rarefying the number of sequence counts was &#34;inadmissible&#34; and should not be employed. However, I identified a number of problems with the design of their simulations and analysis that could compromise their results. In fact, when I reproduced and expanded upon their analysis, it was clear that rarefaction was actually the most robust approach for controlling for uneven sequencing effort across samples. Rarefaction limits the rate of falsely detecting and rejecting differences between treatment groups. Far from being &#34;inadmissible&#34;, rarefaction is a valuable tool for analyzing microbiome sequence data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547241">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547241" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547241">
        <p class="paperTitle">SARS-CoV-2 spike glycosylation affects function and neutralization sensitivity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547241" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547241" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, F.; Schmidt, F.; Muecksch, F.; Wang, Z.; Gazumyan, A.; Nussenzweig, M. C.; Gaebler, C.; Caskey, M.; Hatziioannou, T.; Bieniasz, P. D.</p>
        <p class="info">Score: 52.5, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547241' target='https://doi.org/10.1101/2023.06.30.547241'> 10.1101/2023.06.30.547241</a></p>
        <p class="abstract">The glycosylation of viral envelope proteins can play important roles in virus biology and immune evasion. The spike (S) glycoprotein of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) includes 22 N-linked glycosylation sequons and 17 O-linked glycosites. Here, we investigated the effect of individual glycosylation sites on SARS-CoV-2 S function in pseudotyped virus infection assays and on sensitivity to monoclonal and polyclonal neutralizing antibodies. In most cases, removal of individual glycosylation sites decreased the infectiousness of the pseudotyped virus. For glycosylation mutants in the N-terminal domain (NTD) and the receptor binding domain (RBD), reduction in pseudotype infectivity was predicted by a commensurate reduction in the level of virion-incorporated spike protein. Notably, the presence of a glycan at position N343 within the RBD had diverse effects on neutralization by RBD-specific monoclonal antibodies (mAbs) cloned from convalescent individuals. The N343 glycan reduced overall sensitivity to polyclonal antibodies in plasma from COVID-19 convalescent individuals, suggesting a role for SARS-CoV-2 spike glycosylation in immune evasion. However, vaccination of convalescent individuals produced neutralizing activity that was resilient to the inhibitory effect of the N343 glycan.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546753">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546753" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546753">
        <p class="paperTitle">Eukaryotic antiviral immune proteins arose via convergence, horizontal transfer, and ancient inheritance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546753" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546753" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Culbertson, E. M.; Levin, T. C.</p>
        <p class="info">Score: 51.6, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546753' target='https://doi.org/10.1101/2023.06.27.546753'> 10.1101/2023.06.27.546753</a></p>
        <p class="abstract">Animals use a variety of cell-autonomous innate immune proteins to detect viral infections and prevent replication. Recent studies have discovered that a subset of mammalian antiviral proteins have homology to anti-phage defense proteins in bacteria, implying that there are aspects of innate immunity that are shared across the Tree of Life. While the majority of these studies have focused on characterizing the diversity and biochemical functions of the bacterial proteins, the evolutionary relationships between animal and bacterial proteins are less clear. This ambiguity is partly due to the long evolutionary distances separating animal and bacterial proteins, which obscures their relationships. Here, we tackle this problem for three innate immune families (CD-NTases [including cGAS], STINGs, and Viperins) by deeply sampling protein diversity across eukaryotes. We find that Viperins are truly ancient immune proteins, likely inherited since the last eukaryotic common ancestor and possibly longer. In contrast, we find other immune proteins that arose via at least five independent events of horizontal gene transfer (HGT) from bacteria. Two of these events allowed algae to acquire new bacterial viperins, while three more HGT events gave rise to distinct superfamilies of eukaryotic CD-NTases: the OAS superfamily, which is widespread across eukaryotes; the Mab21 superfamily (containing cGAS) which has diversified via a series of animal-specific duplications, and a previously undefined eSMODS superfamily, which more closely resembles bacterial CD-NTases. Finally, we found that cGAS and STING proteins have substantially different histories, with STINGs arising via convergent domain shuffling in bacteria and eukaryotes. Overall, our findings paint a picture of eukaryotic innate immunity as highly dynamic, where eukaryotes build upon their ancient antiviral repertoires through the reuse of protein domains and by repeatedly sampling a rich reservoir of bacterial anti-phage genes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.24.545427">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.24.545427" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.24.545427">
        <p class="paperTitle">Diverse viral cas genes antagonize CRISPR immunity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.24.545427" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.24.545427" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Katz, M. A.; Sawyer, E. M.; Kozlova, A.; Williams, M. C.; Margolis, S. R.; Oriolt, L.; Johnson, M.; Bondy-Denomy, J.; Meeske, A. J.</p>
        <p class="info">Score: 45.7, Published: 2023-06-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.24.545427' target='https://doi.org/10.1101/2023.06.24.545427'> 10.1101/2023.06.24.545427</a></p>
        <p class="abstract">Prokaryotic CRISPR-Cas immunity is subverted via anti-CRISPRs (Acrs), small proteins that inhibit Cas protein activities when expressed during the phage lytic cycle or from resident prophages or plasmids. CRISPR-Cas defenses are classified into 6 types and 33 subtypes, which employ a diverse suite of Cas effectors and differ in their mechanisms of interference. As Acrs often work via binding to a cognate Cas protein, inhibition is almost always limited to a single CRISPR type. Furthermore, while acr genes are frequently organized together in phage-associated gene clusters, how such inhibitors initially evolve has remained unclear. Here we have investigated the Acr content and inhibition specificity of a collection of Listeria isolates, which naturally harbor four diverse CRISPR-Cas systems (types I-B, II-A, II-C, and VI-A). We observed widespread antagonism of CRISPR, which we traced to 12 novel and 4 known Acr gene families encoded on endogenous mobile genetic elements. Among these were two Acrs that possess sequence homology to type I-B Cas proteins and assemble into a defective interference complex. Surprisingly, an additional type I-B Cas homolog did not affect type I immunity, but instead inhibited the RNA-targeting type VI CRISPR system through sequestration of crRNA. By probing the IMGVR database of viral genomes, we detected abundant orphan cas genes located within putative anti-defense gene clusters. We experimentally verified the Acr activity of one viral cas gene, a particularly broad-spectrum cas3 homolog that inhibits type I-B, II-A, and VI-A CRISPR immunity. Our observations provide direct evidence of Acr evolution via cas gene co-option, and new genes with potential for broad-spectrum control of genome editing technologies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546636">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546636" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546636">
        <p class="paperTitle">Antiviral Type III CRISPR signalling via conjugation of ATP and AdoMet</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546636" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546636" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chi, H.; Hoikkala, V.; Gruschow, S.; Graham, S.; Shirran, S.; White, M. F.</p>
        <p class="info">Score: 36.4, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546636' target='https://doi.org/10.1101/2023.06.26.546636'> 10.1101/2023.06.26.546636</a></p>
        <p class="abstract">CRISPR systems are widespread in the prokaryotic world, providing adaptive immunity against mobile genetic elements (MGE) 1, 2. Type III CRISPR systems, with the signature gene cas10, use CRISPR RNA (crRNA) to detect non-self RNA, activating the enzymatic Cas10 subunit to defend the cell against MGE either directly, via the integral HD nuclease domain 3-5 or indirectly, via synthesis of cyclic oligonucleotide (cOA) second messengers to activate diverse ancillary effectors 6-9. A subset of type III CRISPR systems encode an uncharacterised CorA-family membrane protein and an associated NrN family phosphodiesterase predicted to function in antiviral defence. Here, we demonstrate that the CorA associated type III-B (Cmr) CRISPR system from Bacteroides fragilis provides immunity against MGE when expressed inE. coli. However, B. fragilis Cmr does not synthesise cOA species on activation, instead generating a previously undescribed sigalling molecule, SAM-AMP (3-adenylyl-AdoMet) by conjugating ATP to S-adenosyl methionine via a phosphodiester bond. Once synthesised, SAM-AMP binds to the CorA effector, presumably leading to cell death by disruption of the membrane integrity. SAM-AMP is degraded by CRISPR associated phosphodiesterases or a SAM-AMP lyase, providing an &#34;off switch&#34; analogous to cOA specific ring nucleases 10. SAM-AMP thus represents a new class of second messenger for antiviral signalling, which may function in different roles in diverse cellular contexts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.545647">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.545647" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.545647">
        <p class="paperTitle">Virome analysis of New Zealand&#39;s bats reveals cross-species viral transmission among the Coronaviridae</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.545647" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.545647" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Waller, S. J.; Tortosa, P.; Thurley, T.; O&#39;Donnell, C.; Jackson, R.; Dennis, G.; Grimwood, R. M.; Holmes, E. C.; McInnes, K.; Geoghegan, J. L.</p>
        <p class="info">Score: 49.7, Published: 2023-06-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.545647' target='https://doi.org/10.1101/2023.06.19.545647'> 10.1101/2023.06.19.545647</a></p>
        <p class="abstract">1.The lesser short-tailed bat (Mystacina tuberculata) and the long-tailed bat (Chalinolobus tuberculatus) are Aotearoa New Zealands only native extant terrestrial mammals and are believed to have migrated from Australia. Long-tailed bats arrived in New Zealand an estimated two million years ago and are closely related to other Australian bat species. Lesser short-tailed bats, in contrast, are the only extant species within the Mystacinidae and are estimated to have been living in isolation in New Zealand for the past 16-18 million years. Throughout this period of isolation, lesser short-tailed bats have become one of the most terrestrial bats in the world.

Through a metatranscriptomic analysis of guano samples from eight locations across New Zealand we aimed to characterise the viromes of New Zealands bats and determine whether viruses have jumped between these species over the past two million years. High viral richness was observed among long-tailed bats with viruses spanning seven different viral families. In contrast, no bat-specific viruses were identified in lesser short-tailed bats. Both bat species harboured an abundance of likely dietary- and environmental-associated viruses. We also identified alphacoronaviruses in long-tailed bat guano that had previously been identified in lesser short-tailed bats, suggesting that these viruses had jumped the species barrier after long-tailed bats migrated to New Zealand. Of note, an alphacoronavirus species discovered here possessed a complete genome of only 22,416 nucleotides with entire deletions or truncations of several non-structural proteins, thereby representing what is possibly the shortest genome within the Coronaviridae identified to date. Overall, this study has revealed a diverse range of novel viruses harboured by New Zealands only native terrestrial mammals, in turn expanding our understanding of bat viral dynamics and evolution globally.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546313">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546313" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546313">
        <p class="paperTitle">Rarefaction is currently the best approach to control for uneven sequencing effort in amplicon sequence analyses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546313" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546313" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schloss, P. D.</p>
        <p class="info">Score: 56.8, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546313' target='https://doi.org/10.1101/2023.06.23.546313'> 10.1101/2023.06.23.546313</a></p>
        <p class="abstract">Considering it is common to find as much as 100-fold variation in the number of 16S rRNA gene sequences found across samples in a study, researchers need to control for the effect of uneven sequencing effort. How to do this has become a contentious question. Some have argued that rarefying or rarefaction is &#34;inadmissible&#34; because it omits valid data. A number of alternative approaches have been developed to normalize and rescale the data that purport to be invariant to the number of observations. I generated community distributions based on 12 published datasets where I was able to assess the ability of multiple methods to control for uneven sequencing effort. Rarefaction was the only method that could control for variation in uneven sequencing effort when measuring commonly used alpha and beta diversity metrics. Next, I compared the false detection rate and power to detect true differences between simulated communities with a known effect size using various alpha and beta diversity metrics. Although all methods of controlling for uneven sequencing effort had an acceptable false detection rate when samples were randomly assigned to two treatment groups, rarefaction was consistently able to control for differences in sequencing effort when sequencing depth was confounded with treatment group. Finally, the statistical power to detect differences in alpha and beta diversity metrics was consistently the highest when using rarefaction. These simulations underscore the importance of using rarefaction to normalize the number of sequences across samples in amplicon sequencing analyses.

ImportanceSequencing 16S rRNA gene fragments has become a fundamental tool for understanding the diversity of microbial communities and the factors that affect their diversity. Due to technical challenges, it is common to observe wide variation in the number of sequences that are collected from different samples within the same study. However, the diversity metrics used by microbial ecologists are sensitive to differences in sequencing effort. Therefore, tools are needed to control for the uneven levels of sequencing. This simulation-based analysis shows that despite a longstanding controversy, rarefaction is the most robust approach to control for uneven sequencing effort. The controversy started because of confusion over the definition of rarefaction and violation of assumptions that are made by methods that have been borrowed from other fields. Microbial ecologists should use rarefaction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546105">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546105" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546105">
        <p class="paperTitle">Construction and Characterisation of a Structured, Tuneable, and Transparent 3D Culture Platform for Soil Bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546105" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546105" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rooney, L. M.; Dupuy, L. X.; Hoskisson, P. A.; McConnell, G.</p>
        <p class="info">Score: 24.5, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546105' target='https://doi.org/10.1101/2023.06.28.546105'> 10.1101/2023.06.28.546105</a></p>
        <p class="abstract">We have developed a tuneable workflow for the study of soil microbes in an imitative 3D soil environment that is compatible with routine and advanced optical imaging, is chemically customisable, and is reliably refractive index matched based on the metabolic profile of the study organism. We demonstrate our transparent soil pipeline with two representative soil organisms, Bacillus subtilis and Streptomyces coelicolor, and visualise their colonisation behaviours using fluorescence microscopy and mesoscopy. This spatially structured, 3D approach to microbial culture has the potential to further study the behaviour of other difficult-to-culture bacteria in conditions matching their native environment and could be expanded to study microbial interactions, such as interaction, competition, and warfare.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
