<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="SARS-CoV-2 Neutralizing Antibodies Following a Second BA.5 Bivalent Booster
Authors: Wang, Q.; Bowen, A.; Ho, J.; Zhang, R.; Valdez, R.; Stoneman, E.; Gordon, A.; Lihong, L.; Ho, D. D.
Score: 300.6, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553148
Bivalent COVID-19 mRNA vaccines expressing both the ancestral D614G and Omicron BA.5 spike proteins were introduced in August 2022 with the goal of broadening immunity to emerging SARS-CoV-2 Omicron subvariants. Subsequent studies on bivalent boosters found neutralizing antibody responses similar to boosters with the original monovalent vaccine, likely the result of immunological imprinting.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="SARS-CoV-2 Neutralizing Antibodies Following a Second BA.5 Bivalent Booster
Authors: Wang, Q.; Bowen, A.; Ho, J.; Zhang, R.; Valdez, R.; Stoneman, E.; Gordon, A.; Lihong, L.; Ho, D. D.
Score: 300.6, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553148
Bivalent COVID-19 mRNA vaccines expressing both the ancestral D614G and Omicron BA.5 spike proteins were introduced in August 2022 with the goal of broadening immunity to emerging SARS-CoV-2 Omicron subvariants. Subsequent studies on bivalent boosters found neutralizing antibody responses similar to boosters with the original monovalent vaccine, likely the result of immunological imprinting." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-20T10:35:54+00:00" />
<meta property="article:modified_time" content="2023-08-20T10:35:54+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="SARS-CoV-2 Neutralizing Antibodies Following a Second BA.5 Bivalent Booster
Authors: Wang, Q.; Bowen, A.; Ho, J.; Zhang, R.; Valdez, R.; Stoneman, E.; Gordon, A.; Lihong, L.; Ho, D. D.
Score: 300.6, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553148
Bivalent COVID-19 mRNA vaccines expressing both the ancestral D614G and Omicron BA.5 spike proteins were introduced in August 2022 with the goal of broadening immunity to emerging SARS-CoV-2 Omicron subvariants. Subsequent studies on bivalent boosters found neutralizing antibody responses similar to boosters with the original monovalent vaccine, likely the result of immunological imprinting."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "SARS-CoV-2 Neutralizing Antibodies Following a Second BA.5 Bivalent Booster\nAuthors: Wang, Q.; Bowen, A.; Ho, J.; Zhang, R.; Valdez, R.; Stoneman, E.; Gordon, A.; Lihong, L.; Ho, D. D.\nScore: 300.6, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553148\nBivalent COVID-19 mRNA vaccines expressing both the ancestral D614G and Omicron BA.5 spike proteins were introduced in August 2022 with the goal of broadening immunity to emerging SARS-CoV-2 Omicron subvariants. Subsequent studies on bivalent boosters found neutralizing antibody responses similar to boosters with the original monovalent vaccine, likely the result of immunological imprinting.",
  "keywords": [
    
  ],
  "articleBody": " SARS-CoV-2 Neutralizing Antibodies Following a Second BA.5 Bivalent Booster\nAuthors: Wang, Q.; Bowen, A.; Ho, J.; Zhang, R.; Valdez, R.; Stoneman, E.; Gordon, A.; Lihong, L.; Ho, D. D.\nScore: 300.6, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553148\nBivalent COVID-19 mRNA vaccines expressing both the ancestral D614G and Omicron BA.5 spike proteins were introduced in August 2022 with the goal of broadening immunity to emerging SARS-CoV-2 Omicron subvariants. Subsequent studies on bivalent boosters found neutralizing antibody responses similar to boosters with the original monovalent vaccine, likely the result of immunological imprinting. Guidelines allow for administration of a second bivalent booster in high-risk groups, but it remains unknown whether this would broaden antibody responses. To address this question, we assessed longitudinal serum SARS-CoV-2-neutralizing titers in 18 elderly immunocompetent individuals (mean age 69) following a fourth monovalent booster and two BA.5 bivalent booster vaccines using pseudovirus neutralization assays against D614G, Omicron BA.5, and Omicron XBB.1.5. There was a small but significant increase in peak neutralizing antibody responses against Omicron BA.5 and XBB.1.5 following the first bivalent booster, but no significant increase in peak titers following the second bivalent booster. Omicron-specific neutralizing titers remained low after both doses of the BA.5 bivalent booster. Our results suggest that a second dose of the BA.5 bivalent booster is not sufficient to broaden antibody responses and to overcome immunological imprinting. A monovalent vaccine targeting only the spike of the recently dominant SARS-CoV-2 may mitigate the \"back boosting\" associated with the \"original antigenic sin.\"\nAntiviral efficacy of the SARS-CoV-2 XBB breakthrough infection sera against Omicron subvariants including EG.5\nAuthors: Kaku, Y.; Kosugi, Y.; Uriu, K.; Ito, J.; Kuramochi, J.; Sadamasu, K.; Yoshimura, K.; Asakura, H.; Nagashima, M.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Sato, K.\nScore: 1042.0, Published: 2023-08-08 DOI: 10.1101/2023.08.08.552415\nAs of July 2023, EG.5.1 (a.k.a. XBB.1.9.2.5.1), a XBB subvariant bearing the S:Q52H and S:F456L substitutions, alongside the S:F486P substitution (Figure S1A), has rapidly spread in some countries. On July 19, 2023, the WHO classified EG.5 as a variant under monitoring. First, we showed that EG.5.1 exhibits a higher effective reproduction number compared with XBB.1.5, XBB.1.16, and its parental lineage (XBB.1.9.2), suggesting that EG.5.1 will spread globally and outcompete these XBB subvariants in the near future. We then addressed whether EG.5.1 evades from the antiviral effect of the humoral immunity induced by breakthrough infection (BTI) of XBB subvariants and performed a neutralization assay using XBB BTI sera. However, the 50% neutralization titer (NT50) of XBB BTI sera against EG.5.1 was comparable to those against XBB.1.5/1.9.2 and XBB.1.16. Moreover, the sensitivity of EG.5.1 to convalescent sera of XBB.1- and XBB.1.5-infected hamsters was similar to those of XBB.1.5/1.9 and XBB.1.16. These results suggest that the increased Re of EG.5.1 is attributed to neither increased infectivity nor immune evasion from XBB BTI, and the emergence and spread of EG.5 is driven by the other pressures. We previously demonstrated that Omicron BTI cannot efficiently induce antiviral humoral immunity against the variant infected. In fact, the NT50s of the BTI sera of Omicron BA.1, BA.2, and BA.5 against the variant infected were 3.0-, 2.2-, and 3.4-fold lower than that against the ancestral B.1.1 variant, respectively. However, strikingly, we found that the NT50 of the BTI sera of XBB1.5/1.9 and XBB.1.16 against the variant infected were 8.7- and 8.3-fold lower than that against the B.1.1 variant. These results suggest that XBB BTI cannot efficiently induce antiviral humoral immunity against XBB subvariants. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=147 SRC=\"FIGDIR/small/552415v1_figs1.gif\" ALT=\"Figure 1\"\u003e View larger version (39K): org.highwire.dtl.DTLVardef@f95376org.highwire.dtl.DTLVardef@d66fa8org.highwire.dtl.DTLVardef@3c8841org.highwire.dtl.DTLVardef@15824c_HPS_FORMAT_FIGEXP M_FIG O_FLOATNOFigure S1.C_FLOATNO Virological features of EG.5.1 and XBB BTI (A) Frequency of mutations of interest in the representative XBB sublineages. Only mutations with a frequency \u003e0.5 in at least one but not all the representative sublineages are shown. Note that the S proteins of XBB.1.5 and XBB.1.9.2 are identical. (B) Estimated epidemic dynamics of the representative XBB sublineages in countries where \u003e50 sequences of EG.5.1, XBB.1.5, XBB.1.9.2, and XBB.1.16 were detected from April 1, 2023 to July 13, 2023. Countries are ordered according to the number of detected sequences of EG.5.1. Line, posterior mean; ribbon, 95% Bayesian confidence interval. The dynamics for EG.5.1 is highlighted by a red arrowhead. (C) Estimated relative Re of the representative XBB sublineages in the six countries. The relative Re of XBB.1.5 is set to 1 (horizontal dashed line). Violin, posterior distribution; dot, posterior mean; line, 95% Bayesian confidence interval. (D) Lentivirus-based pseudovirus assay. HOS-ACE2-TMPRSS2 cells were infected with pseudoviruses bearing each S protein. The amount of input virus was normalized to the amount of HIV-1 p24 capsid protein. The percentage infectivity of XBB.1.5/1.9.2, XBB.1.5/1.9.2+Q52H, XBB.1.5/1.9.2+F456L, and EG.5.1 compared to that of XBB.1.5/1.9.2 are shown. The horizontal dash line indicates the mean value of the percentage infectivity of the XBB.1.5/1.9.2. Assays were performed in quadruplicate. The presented data are expressed as the average {+/-} SD. Each dot indicates the result of an individual replicate. (E-G) Neutralization assay. Assays were performed with pseudoviruses harboring the S proteins of B.1.1, BA.1, BA.2, BA.5, BQ.1.1, XBB.1, XBB.1.5/1.9.2, XBB.1.16, EG.5.1, XBB.1.5/1.9.2+Q52H, and XBB.1.5/1.9.2+F456L. The following sera were used: convalescent sera from fully vaccinated individuals who had been infected with XBB.1.5 (one 3-dose vaccinated. 1 donor in total), XBB.1.9 (one 3-dose vaccinated donor, one 4-dose vaccinated donor and one 5-dose vaccinated donor. 3 donors in total), and XBB.1.16 (one 2-dose vaccinated donor, two 3-dose vaccinated donors, and one 4-dose vaccinated donor. 4 donors in total) (E); sera from hamster infected with XBB.1 (left) or XBB.1.5 (right) (F); and convalescent sera from fully vaccinated individuals who had been infected with BA.1 (thirteen 2-dose vaccinated, 13 donors in total) (left)1, BA.2 (nine 2-dose vaccinated and four 3-dose vaccinated donors. 13 donors in total) (middle)19, and BA.5 (one 2-dose vaccinated, thirteen 3-dose vaccinated donors, and one 4-dose vaccinated. 15 donors in total) (right)19 (G). Each dot indicates the result of an individual replicate. Assays for each serum sample were performed in triplicate to determine the 50% neutralization titer (NT50). Each dot represents one NT50 value, and the geometric mean and 95% confidence interval are shown. The number in parenthesis indicates the mean of NT50 values. The horizontal dash line indicates the detection limit (120-fold). In D, statistically significant differences (**, P \u003c 0.001, ***, P \u003c 0.0001) versus XBB.1.5/1.9.2 were determined by two-sided Students t tests. Blue asterisks indicate decreased percentage of infectivity. In E and G, statistically significant differences versus B.1.1 were determined by two-sided Wilcoxon signed-rank tests. The fold change between B.1.1 and the variant indicated is shown in red. Background information on the convalescent donors is summarized in Table S1. In F, statistically significant differences (*, P \u003c 0.01, **, P \u003c 0.001) between B.1.1 and XBB.1 (left) or XBB.1.5/1.9.2 (right) were determined by two-sided Wilcoxon signed-rank tests and indicated with asterisks. Red asterisks indicate decreased NT50s. C_FIG\nImpact of age and sex on neuroinflammation following SARS-CoV-2 infection in a murine model\nAuthors: D Krishna, V.; Chang, A.; Korthas, H.; Var, S. R.; Low, W. C.; Li, L.; Cheeran, M. C.-J.\nScore: 74.2, Published: 2023-08-14 DOI: 10.1101/2023.08.11.552998\nSevere Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), the etiological agent for the worldwide COVID-19 pandemic, is known to infect people of all ages and both sexes. Senior populations have the greatest risk of severe disease, and sexual dimorphism in clinical outcomes has been reported in COVID-19. SARS-CoV-2 infection in humans can cause damage to multiple organ systems, including the brain. Neurological symptoms are widely observed in patients with COVID-19, with many survivors suffering from persistent neurological and cognitive impairment, potentially accelerating Alzheimers disease. The present study aims to investigate the impact of age and sex on the neuroinflammatory response to SARS-CoV-2 infection using a mouse model. Wild-type C57BL/6 mice were inoculated, by intranasal route, with SARS-CoV-2 lineage B.1.351 variant known to infect mice. Older animals and in particular males exhibited a significantly greater weight loss starting at 4 dpi. In addition, male animals exhibited higher viral RNA loads and higher titers of infectious virus in the lung, which was particularly evident in males at 16 months of age. Notably, no viral RNA was detected in the brains of infected mice, regardless of age or sex. Nevertheless, expression of IL-6, TNF-, and CCL-2 in the lung and brain was increased with viral infection. An unbiased brain RNA-seq/transcriptomic analysis showed that SARS-CoV-2 infection caused significant changes in gene expression profiles in the brain, with innate immunity, defense response to virus, cerebravascular and neuronal functions, as the major molecular networks affected. The data presented in this study show that SARS-CoV-2 infection triggers a neuroinflammatory response despite the lack of detectable virus in the brain. Age and sex have a modifying effect on this pathogenic process. Aberrant activation of innate immune response, disruption of blood-brain barrier and endothelial cell integrity, and supression of neuronal activity and axonogenesis underlie the impact of SARS-CoV-2 infection on the brain. Understanding the role of these affected pathways in SARS-CoV-2 pathogenesis helps identify appropriate points of therapeutic interventions to alleviate neurological dysfunction observed during COVID-19.\nToxinome - The Bacterial Protein Toxin Database\nAuthors: Danov, A.; Segev, O.; Bograd, A.; Ben Eliyahu, Y.; Dotan, N.; Kaplan, T.; Levy, A.\nScore: 57.5, Published: 2023-08-12 DOI: 10.1101/2023.08.12.553073\nProtein toxins are key molecular weapons in biology that are used to attack neighboring cells. Bacteria use protein toxins to kill or inhibit growth of prokaryotic and eukaryotic cells using various modes of action that target essential cellular components. The toxins are responsible for shaping microbiomes in different habitats, for abortive phage infection, and for severe infectious diseases of animals and plants. Although several toxin databases have been developed, each one is devoted to a specific toxin family and they encompass a relatively small number of toxins. Antimicrobial toxins are often accompanied by antitoxins (or immunity proteins) that neutralize the cognate toxins. Here, we combined toxins and antitoxins from many resources and created Toxinome, a comprehensive and updated bacterial protein toxin database. Toxinome includes a total of 1,483,028 toxins and 491,345 antitoxins encoded in 59,475 bacterial genomes across the tree of life. We identified a depletion of toxin and antitoxin genes in bacteria that dwell in extreme temperatures. We defined 5,161 unique Toxin Islands within phylogenetically diverse bacterial genomes, which are loci dense in toxin and antitoxin genes. By focusing on the unannotated genes within these islands, we characterized a number of these genes as toxins or antitoxins. Finally, we developed an interactive Toxinome website (http://toxinome.pythonanywhere.com) that allows searching and downloading of our database. The Toxinome resource will be useful to the large research community interested in bacterial toxins and can guide toxin discovery and function elucidation, and infectious disease diagnosis and treatment. ImportanceMicrobes use protein toxins as important tools to attack neighboring cells, microbial or eukaryotic, and for self-killing when attacked by viruses. These toxins work by different mechanisms to inhibit cell growth or kill cells. Microbes also use antitoxin proteins to neutralize the toxin activities. Here, we developed a comprehensive database called Toxinome of nearly two million toxin and antitoxins that are encoded in 59,475 bacterial genomes. We described the distribution of bacterial toxins and identified that they are depleted from bacteria that live in hot and cold temperatures. We find 5,161 cases in which toxins and antitoxins are densely clustered in bacterial genomes and termed these areas \"Toxin Islands\". The Toxinome database is a useful resource for anyone interested in toxin biology and evolution, and it can guide discovery of new toxins.\nComprehensive Assessment of the Intrinsic Pancreatic Microbiome\nAuthors: Eckhoff, A. M.; Fletcher, A. A.; Kelly, M. S.; Dohlman, A.; McIntyre, C. A.; Shen, X.; Iyer, M. K.; Nussbaum, D. P.; Allen, P. J.\nScore: 25.9, Published: 2023-08-12 DOI: 10.1101/2023.08.12.553074\nBackgroundSmall studies in pancreatic ductal adenocarcinoma (PDAC) and intraductal papillary mucinous neoplasm (IPMN) have suggested that intra-pancreatic microbial dysbiosis may drive malignant transformation. We sought to comprehensively profile tissue and cyst fluid in patients with benign, precancerous, and cancerous conditions of the pancreas to characterize the intrinsic pancreatic microbiome. MethodsPancreatic samples were collected at the time of resection from 109 patients. Samples included tumor tissue (control, n=20; IPMN, n=20; PDAC, n=19) and pancreatic cyst fluid (IPMN, n=30; SCA, n=10; MCN, n=10). Assessment of bacterial DNA by quantitative PCR and 16S ribosomal RNA gene sequencing was performed. Downstream analyses determined the relative abundances of individual taxa between groups and compared intergroup diversity. Whole-genome sequencing data from 140 patients with PDAC in the National Cancer Institutes Clinical Proteomic Tumor Analysis Consortium (CPTAC) were analyzed to validate findings. ResultsSequencing of pancreatic tissue yielded few microbial reads regardless of diagnosis, and analysis of pancreatic tissue showed no difference in the abundance and composition of bacterial taxa between normal pancreas, IPMN, or PDAC groups. Low-grade dysplasia (LGD) and high-grade dysplasia (HGD) IPMN were characterized by low bacterial abundances with no difference in tissue composition and a slight increase in Pseudomonas and Sediminibacterium in HGD cyst fluid. Decontamination analysis using the CPTAC database confirmed a low-biomass, low-diversity intrinsic pancreatic microbiome that did not differ by pathology. ConclusionsOur analysis of the pancreatic microbiome demonstrated very low intrinsic biomass that is relatively conserved across diverse neoplastic conditions and thus unlikely to drive malignant transformation. Significance of this studyO_ST_ABSWhat is already known on this subject?C_ST_ABSO_LIMicrobial colonization, infection, and dysbiosis have been implicated in the oncogenesis of various gastrointestinal tumors (e.g.: Helicobacter pylori in gastric cancer, hepatitis B and C in hepatocellular carcinoma, colon dysbiosis in colon cancer progression). C_LIO_LIFew studies have analyzed the intrinsic pancreatic microbiome, and these have produced conflicting results regarding microbial presence and alterations associated with malignant disease. C_LIO_LIIPMN is a precursor lesion to pancreatic cancer that represents a whole gland defect without an established driver event, and microbiome changes have been implicated as a possible etiology of cyst formation and dysplastic progression. C_LI What are the new findings?O_LIA low-biomass, low-diversity intrinsic pancreatic microbiome is present in both pancreatic tissue and cyst fluid. C_LIO_LIThis intrinsic pancreatic microbiome does not differ in terms of abundance, composition, or diversity between patients with PDAC, IPMN, or other benign conditions of the pancreas. C_LI How might it impact clinical practice in the foreseeable future?O_LIMicrobiome dysbiosis does not appear to be a driver of malignant degeneration of IPMN, and further research is needed to identify drivers of oncogenesis in order for possible chemoprevention strategies to be developed. C_LI\nInterkingdom interactions shape the fungal microbiome of mosquitoes\nAuthors: Hegde, S.; Khanipov, K.; Hornett, E. A.; Nilyanimit, P.; Pimenova, M.; Saldana, M. A.; Becker, C. d.; Golovko, G.; Hughes, G.\nScore: 12.2, Published: 2023-08-12 DOI: 10.1101/2023.08.11.552965\nThe mosquito microbiome is an important modulator of vector competence and vectoral capacity. Unlike the extensively studied bacterial microbiome, fungal communities in the mosquito microbiome (mycobiome) remain largely unexplored. To work towards getting an improved understanding of the fungi associated with mosquitoes, we sequenced the mycobiome of three field-collected and laboratory-reared mosquito species (Aedes albopictus, Aedes aegypti, and Culex quinquefasciatus). Our analysis showed both environment and host species were contributing to the diversity of the fungal microbiome of mosquitoes. When comparing species, Ae. albopictus possessed a higher number of diverse fungal taxa than Cx. quinquefasciatus, while strikingly less than 1% of reads from Ae. aegypti samples were fungal. Fungal reads from Ae. aegypti were \u003c1% even after inhibiting host amplification using a PNA blocker, indicating that this species lacked a significant fungal microbiome that was amplified using this sequencing approach. Using a mono-association mosquito infection model, we confirmed that mosquito-derived fungal isolates colonize and for Aedes mosquitoes, support growth and development at comparable rates to their bacterial counterparts. Strikingly, native bacterial taxa isolated from mosquitoes impeded the colonization of symbiotic fungi in Ae. aegypti suggesting interkingdom interactions shape fungal microbiome communities. Collectively, this study adds to our understanding of the fungal microbiome of different mosquito species, that these fungal microbes support growth and development, and highlights that microbial interactions underpin fungal colonization of these medically relevent species.\nMicrobial eukaryotic predation pressure and biomass at deep-sea hydrothermal vents: Implications for deep-sea carbon cycling\nAuthors: Hu, S. K.; Anderson, R. E.; Pachiadaki, M. G.; Edgcomb, V. P.; Serres, M. H.; Sylva, S. P.; German, C. R.; Seewald, J. S.; Lang, S. Q.; Huber, J. A.\nScore: 12.2, Published: 2023-08-11 DOI: 10.1101/2023.08.11.552852\nDeep-sea hydrothermal vent geochemistry shapes the foundation of the microbial food web by fueling chemolithoautotrophic microbial activity. Microbial eukaryotes (or protists) play a critical role in hydrothermal vent food webs as consumers, hosts of symbiotic bacteria, and as a nutritional source to higher trophic levels. We measured cell abundances and predation pressures of vent-associated microbial eukaryotes in low temperature diffuse hydrothermal fluids at the Von Damm and Piccard vent fields along the Mid-Cayman Rise in the Western Caribbean Sea. We present findings from experiments performed under in situ pressure that show higher cell abundances and grazing rates compared to those done at 1 atmosphere (shipboard ambient pressure); this trend was attributed to the impact of depressurization on cell integrity. A relationship between protistan grazing rate, prey cell abundance, and temperature of end member hydrothermal vent fluid was observed at both vent fields, regardless of experimental approach. The quantification of protistan biomass and grazing pressure shows that hydrothermally-fueled microbial food webs play a significant role in the broader deep-sea carbon budget by contributing to local carbon export and supply of nutrient resources to the deep ocean.\nGoblet cell invasion promotes breaching of respiratory epithelia by an opportunistic human pathogen\nAuthors: Swart, A. L.; Laventie, B. J.; Suetterlin, R.; Junne, T.; Yu, X.; Karagkiozi, E.; Okujava, R.; Jenal, U.\nScore: 11.4, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553119\nWhile commensal bacteria generally respect natural barriers of the human body, pathogens are able to breach epithelia, invade deeper tissue layers and cause life-threatening infections. Pseudomonas aeruginosa, an opportunistic human pathogen, is a leading cause of severe hospital-acquired pneumonia, with mortality rates as high as 50% in mechanically ventilated patients1-3. Effective colonization and breaching of lung mucosa are hallmarks of P. aeruginosa pathogenesis4. Although virulence factors and behavioral strategies of P. aeruginosa have been described5,6, it has remained unclear how this pathogen disseminates on functional mucosal surfaces, how it avoids mucociliary clearance and how it invades the tissue barrier. Using fully differentiated human lung epithelia, we demonstrate that P. aeruginosa efficiently spreads on the apical tissue surface before it breaches epithelia by specifically invading mucus secreting goblet cells. Internalization leads to host cell death and expulsion and the formation of ruptures of the epithelial barrier. Rupture sites are rapidly colonized by extracellular bacteria through active chemotaxis, leading to increasing tissue damage and successful pathogen translocation to the unprotected basolateral side of the epithelium. We show that cell invasion is promoted by two Type-6 toxin secretion systems (T6SS), while Type-3 (T3SS) mediates cell death of infected goblet cells. T3SS mutants invade goblet cells normally, but internalized bacteria fail to trigger goblet cell expulsion and instead show unrestrained intracellular replication. While the effective shedding of infected host cells reveals potent tissue protection mechanisms, the discovery of an intracellular lifestyle of P. aeruginosa in human lung epithelia provides new entry points into investigating the intersection of antibiotic and immune mechanisms during lung infections. By demonstrating that P. aeruginosa uses a combination of specific virulence factors and collective behavior to invade goblet cells and breach the lung tissue barrier from within, these studies reveal novel mechanisms underlying lung infection dynamics under physiological conditions.\nCell division machinery drives cell-specific gene activation during bacterial differentiation\nAuthors: Chareyre, S.; Li, X.; Anjuwon-Foster, B. R.; Clifford, S.; Brogan, A.; Su, Y.; Shroff, H.; Ramamurthi, K. S.\nScore: 18.2, Published: 2023-08-10 DOI: 10.1101/2023.08.10.552768\nWhen faced with starvation, the bacterium Bacillus subtilis transforms itself into a dormant cell type called a \"spore\". Sporulation initiates with an asymmetric division event, which requires the relocation of the core divisome components FtsA and FtsZ, after which the sigma factor {sigma}F is exclusively activated in the smaller daughter cell. Compartment specific activation of {sigma}F requires the SpoIIE phosphatase, which displays a biased localization on one side of the asymmetric division septum and associates with the structural protein DivIVA, but the mechanism by which this preferential localization is achieved is unclear. Here, we isolated a variant of DivIVA that indiscriminately activates {sigma}F in both daughter cells due to promiscuous localization of SpoIIE, which was corrected by overproduction of FtsA and FtsZ. We propose that a unique feature of the sporulation septum, defined by the cell division machinery, drives the asymmetric localization of DivIVA and SpoIIE to trigger the initiation of the sporulation program.\nDefense mechanism of a bacterial retron supramolecular assembly\nAuthors: Wang, Y.; Wang, C.; Guan, Z.; Cao, J.; Xu, J.; Wang, S.; Cui, Y.; Wang, Q.; Chen, Y.; Zhang, D.; Sun, M.; Tao, P.; Zou, T.\nScore: 10.0, Published: 2023-08-16 DOI: 10.1101/2023.08.16.553469\nRetrons are a class of multigene antiphage defense system typically consisting of a retron reverse transcriptase, a non-coding RNA, and a cognate effector. Although the triggers for several retron systems have been discovered recently, the full picture of how retron systems sense invading phages and mediate defense remains to be elucidated. Here, we focus on the retron Ec86 defense system and report its modes of activation and action. We identified a phage-encoded DNA cytosine methyltransferase (Dcm) as the trigger of the Ec86 system and show that Ec86 senses msDNA methylation and becomes activated. We further determined the structure of a tripartite retron Ec86 supramolecular assembly, which is primed for activation by Dcm, and demonstrated that the activated system confers defense through depletion of nucleoside derivatives. These findings emphasize the role of retrons being a second line of defense and highlight an emerging theme of anti-phage defense through supramolecular complex assemblies.\n",
  "wordCount" : "3591",
  "inLanguage": "en",
  "datePublished": "2023-08-20T10:35:54Z",
  "dateModified": "2023-08-20T10:35:54Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta"><span>updated on August 20, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.13.553148">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.13.553148" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.13.553148">
        <p class="paperTitle">SARS-CoV-2 Neutralizing Antibodies Following a Second BA.5 Bivalent Booster</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.13.553148" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.13.553148" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Q.; Bowen, A.; Ho, J.; Zhang, R.; Valdez, R.; Stoneman, E.; Gordon, A.; Lihong, L.; Ho, D. D.</p>
        <p class="info">Score: 300.6, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.13.553148' target='https://doi.org/10.1101/2023.08.13.553148'> 10.1101/2023.08.13.553148</a></p>
        <p class="abstract">Bivalent COVID-19 mRNA vaccines expressing both the ancestral D614G and Omicron BA.5 spike proteins were introduced in August 2022 with the goal of broadening immunity to emerging SARS-CoV-2 Omicron subvariants. Subsequent studies on bivalent boosters found neutralizing antibody responses similar to boosters with the original monovalent vaccine, likely the result of immunological imprinting. Guidelines allow for administration of a second bivalent booster in high-risk groups, but it remains unknown whether this would broaden antibody responses. To address this question, we assessed longitudinal serum SARS-CoV-2-neutralizing titers in 18 elderly immunocompetent individuals (mean age 69) following a fourth monovalent booster and two BA.5 bivalent booster vaccines using pseudovirus neutralization assays against D614G, Omicron BA.5, and Omicron XBB.1.5. There was a small but significant increase in peak neutralizing antibody responses against Omicron BA.5 and XBB.1.5 following the first bivalent booster, but no significant increase in peak titers following the second bivalent booster. Omicron-specific neutralizing titers remained low after both doses of the BA.5 bivalent booster. Our results suggest that a second dose of the BA.5 bivalent booster is not sufficient to broaden antibody responses and to overcome immunological imprinting. A monovalent vaccine targeting only the spike of the recently dominant SARS-CoV-2 may mitigate the &#34;back boosting&#34; associated with the &#34;original antigenic sin.&#34;</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552415">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552415" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552415">
        <p class="paperTitle">Antiviral efficacy of the SARS-CoV-2 XBB breakthrough infection sera against Omicron subvariants including EG.5</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552415" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552415" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kaku, Y.; Kosugi, Y.; Uriu, K.; Ito, J.; Kuramochi, J.; Sadamasu, K.; Yoshimura, K.; Asakura, H.; Nagashima, M.; The Genotype to Phenotype Japan (G2P-Japan) Consortium,  ; Sato, K.</p>
        <p class="info">Score: 1042.0, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552415' target='https://doi.org/10.1101/2023.08.08.552415'> 10.1101/2023.08.08.552415</a></p>
        <p class="abstract">As of July 2023, EG.5.1 (a.k.a. XBB.1.9.2.5.1), a XBB subvariant bearing the S:Q52H and S:F456L substitutions, alongside the S:F486P substitution (Figure S1A), has rapidly spread in some countries. On July 19, 2023, the WHO classified EG.5 as a variant under monitoring. First, we showed that EG.5.1 exhibits a higher effective reproduction number compared with XBB.1.5, XBB.1.16, and its parental lineage (XBB.1.9.2), suggesting that EG.5.1 will spread globally and outcompete these XBB subvariants in the near future. We then addressed whether EG.5.1 evades from the antiviral effect of the humoral immunity induced by breakthrough infection (BTI) of XBB subvariants and performed a neutralization assay using XBB BTI sera. However, the 50% neutralization titer (NT50) of XBB BTI sera against EG.5.1 was comparable to those against XBB.1.5/1.9.2 and XBB.1.16. Moreover, the sensitivity of EG.5.1 to convalescent sera of XBB.1- and XBB.1.5-infected hamsters was similar to those of XBB.1.5/1.9 and XBB.1.16. These results suggest that the increased Re of EG.5.1 is attributed to neither increased infectivity nor immune evasion from XBB BTI, and the emergence and spread of EG.5 is driven by the other pressures. We previously demonstrated that Omicron BTI cannot efficiently induce antiviral humoral immunity against the variant infected. In fact, the NT50s of the BTI sera of Omicron BA.1, BA.2, and BA.5 against the variant infected were 3.0-, 2.2-, and 3.4-fold lower than that against the ancestral B.1.1 variant, respectively. However, strikingly, we found that the NT50 of the BTI sera of XBB1.5/1.9 and XBB.1.16 against the variant infected were 8.7- and 8.3-fold lower than that against the B.1.1 variant. These results suggest that XBB BTI cannot efficiently induce antiviral humoral immunity against XBB subvariants.

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=147 SRC=&#34;FIGDIR/small/552415v1_figs1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (39K):
org.highwire.dtl.DTLVardef@f95376org.highwire.dtl.DTLVardef@d66fa8org.highwire.dtl.DTLVardef@3c8841org.highwire.dtl.DTLVardef@15824c_HPS_FORMAT_FIGEXP  M_FIG O_FLOATNOFigure S1.C_FLOATNO Virological features of EG.5.1 and XBB BTI (A) Frequency of mutations of interest in the representative XBB sublineages. Only mutations with a frequency &gt;0.5 in at least one but not all the representative sublineages are shown. Note that the S proteins of XBB.1.5 and XBB.1.9.2 are identical.

(B) Estimated epidemic dynamics of the representative XBB sublineages in countries where &gt;50 sequences of EG.5.1, XBB.1.5, XBB.1.9.2, and XBB.1.16 were detected from April 1, 2023 to July 13, 2023. Countries are ordered according to the number of detected sequences of EG.5.1. Line, posterior mean; ribbon, 95% Bayesian confidence interval. The dynamics for EG.5.1 is highlighted by a red arrowhead.

(C) Estimated relative Re of the representative XBB sublineages in the six countries. The relative Re of XBB.1.5 is set to 1 (horizontal dashed line). Violin, posterior distribution; dot, posterior mean; line, 95% Bayesian confidence interval.

(D) Lentivirus-based pseudovirus assay. HOS-ACE2-TMPRSS2 cells were infected with pseudoviruses bearing each S protein. The amount of input virus was normalized to the amount of HIV-1 p24 capsid protein. The percentage infectivity of XBB.1.5/1.9.2, XBB.1.5/1.9.2&#43;Q52H, XBB.1.5/1.9.2&#43;F456L, and EG.5.1 compared to that of XBB.1.5/1.9.2 are shown. The horizontal dash line indicates the mean value of the percentage infectivity of the XBB.1.5/1.9.2. Assays were performed in quadruplicate. The presented data are expressed as the average {&#43;/-} SD. Each dot indicates the result of an individual replicate.

(E-G) Neutralization assay. Assays were performed with pseudoviruses harboring the S proteins of B.1.1, BA.1, BA.2, BA.5, BQ.1.1, XBB.1, XBB.1.5/1.9.2, XBB.1.16, EG.5.1, XBB.1.5/1.9.2&#43;Q52H, and XBB.1.5/1.9.2&#43;F456L. The following sera were used: convalescent sera from fully vaccinated individuals who had been infected with XBB.1.5 (one 3-dose vaccinated. 1 donor in total), XBB.1.9 (one 3-dose vaccinated donor, one 4-dose vaccinated donor and one 5-dose vaccinated donor. 3 donors in total), and XBB.1.16 (one 2-dose vaccinated donor, two 3-dose vaccinated donors, and one 4-dose vaccinated donor. 4 donors in total) (E); sera from hamster infected with XBB.1 (left) or XBB.1.5 (right) (F); and convalescent sera from fully vaccinated individuals who had been infected with BA.1 (thirteen 2-dose vaccinated, 13 donors in total) (left)1, BA.2 (nine 2-dose vaccinated and four 3-dose vaccinated donors. 13 donors in total) (middle)19, and BA.5 (one 2-dose vaccinated, thirteen 3-dose vaccinated donors, and one 4-dose vaccinated. 15 donors in total) (right)19 (G). Each dot indicates the result of an individual replicate. Assays for each serum sample were performed in triplicate to determine the 50% neutralization titer (NT50). Each dot represents one NT50 value, and the geometric mean and 95% confidence interval are shown. The number in parenthesis indicates the mean of NT50 values. The horizontal dash line indicates the detection limit (120-fold).

In D, statistically significant differences (**, P &lt; 0.001, ***, P &lt; 0.0001) versus XBB.1.5/1.9.2 were determined by two-sided Students t tests. Blue asterisks indicate decreased percentage of infectivity.

In E and G, statistically significant differences versus B.1.1 were determined by two-sided Wilcoxon signed-rank tests. The fold change between B.1.1 and the variant indicated is shown in red. Background information on the convalescent donors is summarized in Table S1.

In F, statistically significant differences (*, P &lt; 0.01, **, P &lt; 0.001) between B.1.1 and XBB.1 (left) or XBB.1.5/1.9.2 (right) were determined by two-sided Wilcoxon signed-rank tests and indicated with asterisks. Red asterisks indicate decreased NT50s.

C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.552998">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.552998" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.552998">
        <p class="paperTitle">Impact of age and sex on neuroinflammation following SARS-CoV-2 infection in a murine model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.552998" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.552998" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: D Krishna, V.; Chang, A.; Korthas, H.; Var, S. R.; Low, W. C.; Li, L.; Cheeran, M. C.-J.</p>
        <p class="info">Score: 74.2, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.552998' target='https://doi.org/10.1101/2023.08.11.552998'> 10.1101/2023.08.11.552998</a></p>
        <p class="abstract">Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), the etiological agent for the worldwide COVID-19 pandemic, is known to infect people of all ages and both sexes. Senior populations have the greatest risk of severe disease, and sexual dimorphism in clinical outcomes has been reported in COVID-19. SARS-CoV-2 infection in humans can cause damage to multiple organ systems, including the brain. Neurological symptoms are widely observed in patients with COVID-19, with many survivors suffering from persistent neurological and cognitive impairment, potentially accelerating Alzheimers disease. The present study aims to investigate the impact of age and sex on the neuroinflammatory response to SARS-CoV-2 infection using a mouse model. Wild-type C57BL/6 mice were inoculated, by intranasal route, with SARS-CoV-2 lineage B.1.351 variant known to infect mice. Older animals and in particular males exhibited a significantly greater weight loss starting at 4 dpi. In addition, male animals exhibited higher viral RNA loads and higher titers of infectious virus in the lung, which was particularly evident in males at 16 months of age. Notably, no viral RNA was detected in the brains of infected mice, regardless of age or sex. Nevertheless, expression of IL-6, TNF-, and CCL-2 in the lung and brain was increased with viral infection. An unbiased brain RNA-seq/transcriptomic analysis showed that SARS-CoV-2 infection caused significant changes in gene expression profiles in the brain, with innate immunity, defense response to virus, cerebravascular and neuronal functions, as the major molecular networks affected. The data presented in this study show that SARS-CoV-2 infection triggers a neuroinflammatory response despite the lack of detectable virus in the brain. Age and sex have a modifying effect on this pathogenic process. Aberrant activation of innate immune response, disruption of blood-brain barrier and endothelial cell integrity, and supression of neuronal activity and axonogenesis underlie the impact of SARS-CoV-2 infection on the brain. Understanding the role of these affected pathways in SARS-CoV-2 pathogenesis helps identify appropriate points of therapeutic interventions to alleviate neurological dysfunction observed during COVID-19.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.12.553073">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.12.553073" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.12.553073">
        <p class="paperTitle">Toxinome - The Bacterial Protein Toxin Database</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.12.553073" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.12.553073" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Danov, A.; Segev, O.; Bograd, A.; Ben Eliyahu, Y.; Dotan, N.; Kaplan, T.; Levy, A.</p>
        <p class="info">Score: 57.5, Published: 2023-08-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.12.553073' target='https://doi.org/10.1101/2023.08.12.553073'> 10.1101/2023.08.12.553073</a></p>
        <p class="abstract">Protein toxins are key molecular weapons in biology that are used to attack neighboring cells. Bacteria use protein toxins to kill or inhibit growth of prokaryotic and eukaryotic cells using various modes of action that target essential cellular components. The toxins are responsible for shaping microbiomes in different habitats, for abortive phage infection, and for severe infectious diseases of animals and plants. Although several toxin databases have been developed, each one is devoted to a specific toxin family and they encompass a relatively small number of toxins. Antimicrobial toxins are often accompanied by antitoxins (or immunity proteins) that neutralize the cognate toxins. Here, we combined toxins and antitoxins from many resources and created Toxinome, a comprehensive and updated bacterial protein toxin database. Toxinome includes a total of 1,483,028 toxins and 491,345 antitoxins encoded in 59,475 bacterial genomes across the tree of life. We identified a depletion of toxin and antitoxin genes in bacteria that dwell in extreme temperatures. We defined 5,161 unique Toxin Islands within phylogenetically diverse bacterial genomes, which are loci dense in toxin and antitoxin genes. By focusing on the unannotated genes within these islands, we characterized a number of these genes as toxins or antitoxins. Finally, we developed an interactive Toxinome website (http://toxinome.pythonanywhere.com) that allows searching and downloading of our database. The Toxinome resource will be useful to the large research community interested in bacterial toxins and can guide toxin discovery and function elucidation, and infectious disease diagnosis and treatment.

ImportanceMicrobes use protein toxins as important tools to attack neighboring cells, microbial or eukaryotic, and for self-killing when attacked by viruses. These toxins work by different mechanisms to inhibit cell growth or kill cells. Microbes also use antitoxin proteins to neutralize the toxin activities. Here, we developed a comprehensive database called Toxinome of nearly two million toxin and antitoxins that are encoded in 59,475 bacterial genomes. We described the distribution of bacterial toxins and identified that they are depleted from bacteria that live in hot and cold temperatures. We find 5,161 cases in which toxins and antitoxins are densely clustered in bacterial genomes and termed these areas &#34;Toxin Islands&#34;. The Toxinome database is a useful resource for anyone interested in toxin biology and evolution, and it can guide discovery of new toxins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.12.553074">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.12.553074" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.12.553074">
        <p class="paperTitle">Comprehensive Assessment of the Intrinsic Pancreatic Microbiome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.12.553074" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.12.553074" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eckhoff, A. M.; Fletcher, A. A.; Kelly, M. S.; Dohlman, A.; McIntyre, C. A.; Shen, X.; Iyer, M. K.; Nussbaum, D. P.; Allen, P. J.</p>
        <p class="info">Score: 25.9, Published: 2023-08-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.12.553074' target='https://doi.org/10.1101/2023.08.12.553074'> 10.1101/2023.08.12.553074</a></p>
        <p class="abstract">BackgroundSmall studies in pancreatic ductal adenocarcinoma (PDAC) and intraductal papillary mucinous neoplasm (IPMN) have suggested that intra-pancreatic microbial dysbiosis may drive malignant transformation. We sought to comprehensively profile tissue and cyst fluid in patients with benign, precancerous, and cancerous conditions of the pancreas to characterize the intrinsic pancreatic microbiome.

MethodsPancreatic samples were collected at the time of resection from 109 patients. Samples included tumor tissue (control, n=20; IPMN, n=20; PDAC, n=19) and pancreatic cyst fluid (IPMN, n=30; SCA, n=10; MCN, n=10). Assessment of bacterial DNA by quantitative PCR and 16S ribosomal RNA gene sequencing was performed. Downstream analyses determined the relative abundances of individual taxa between groups and compared intergroup diversity. Whole-genome sequencing data from 140 patients with PDAC in the National Cancer Institutes Clinical Proteomic Tumor Analysis Consortium (CPTAC) were analyzed to validate findings.

ResultsSequencing of pancreatic tissue yielded few microbial reads regardless of diagnosis, and analysis of pancreatic tissue showed no difference in the abundance and composition of bacterial taxa between normal pancreas, IPMN, or PDAC groups. Low-grade dysplasia (LGD) and high-grade dysplasia (HGD) IPMN were characterized by low bacterial abundances with no difference in tissue composition and a slight increase in Pseudomonas and Sediminibacterium in HGD cyst fluid. Decontamination analysis using the CPTAC database confirmed a low-biomass, low-diversity intrinsic pancreatic microbiome that did not differ by pathology.

ConclusionsOur analysis of the pancreatic microbiome demonstrated very low intrinsic biomass that is relatively conserved across diverse neoplastic conditions and thus unlikely to drive malignant transformation.

Significance of this studyO_ST_ABSWhat is already known on this subject?C_ST_ABSO_LIMicrobial colonization, infection, and dysbiosis have been implicated in the oncogenesis of various gastrointestinal tumors (e.g.: Helicobacter pylori in gastric cancer, hepatitis B and C in hepatocellular carcinoma, colon dysbiosis in colon cancer progression).
C_LIO_LIFew studies have analyzed the intrinsic pancreatic microbiome, and these have produced conflicting results regarding microbial presence and alterations associated with malignant disease.
C_LIO_LIIPMN is a precursor lesion to pancreatic cancer that represents a whole gland defect without an established driver event, and microbiome changes have been implicated as a possible etiology of cyst formation and dysplastic progression.
C_LI

What are the new findings?O_LIA low-biomass, low-diversity intrinsic pancreatic microbiome is present in both pancreatic tissue and cyst fluid.
C_LIO_LIThis intrinsic pancreatic microbiome does not differ in terms of abundance, composition, or diversity between patients with PDAC, IPMN, or other benign conditions of the pancreas.
C_LI

How might it impact clinical practice in the foreseeable future?O_LIMicrobiome dysbiosis does not appear to be a driver of malignant degeneration of IPMN, and further research is needed to identify drivers of oncogenesis in order for possible chemoprevention strategies to be developed.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.552965">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.552965" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.552965">
        <p class="paperTitle">Interkingdom interactions shape the fungal microbiome of mosquitoes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.552965" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.552965" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hegde, S.; Khanipov, K.; Hornett, E. A.; Nilyanimit, P.; Pimenova, M.; Saldana, M. A.; Becker, C. d.; Golovko, G.; Hughes, G.</p>
        <p class="info">Score: 12.2, Published: 2023-08-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.552965' target='https://doi.org/10.1101/2023.08.11.552965'> 10.1101/2023.08.11.552965</a></p>
        <p class="abstract">The mosquito microbiome is an important modulator of vector competence and vectoral capacity. Unlike the extensively studied bacterial microbiome, fungal communities in the mosquito microbiome (mycobiome) remain largely unexplored. To work towards getting an improved understanding of the fungi associated with mosquitoes, we sequenced the mycobiome of three field-collected and laboratory-reared mosquito species (Aedes albopictus, Aedes aegypti, and Culex quinquefasciatus). Our analysis showed both environment and host species were contributing to the diversity of the fungal microbiome of mosquitoes. When comparing species, Ae. albopictus possessed a higher number of diverse fungal taxa than Cx. quinquefasciatus, while strikingly less than 1% of reads from Ae. aegypti samples were fungal. Fungal reads from Ae. aegypti were &lt;1% even after inhibiting host amplification using a PNA blocker, indicating that this species lacked a significant fungal microbiome that was amplified using this sequencing approach. Using a mono-association mosquito infection model, we confirmed that mosquito-derived fungal isolates colonize and for Aedes mosquitoes, support growth and development at comparable rates to their bacterial counterparts. Strikingly, native bacterial taxa isolated from mosquitoes impeded the colonization of symbiotic fungi in Ae. aegypti suggesting interkingdom interactions shape fungal microbiome communities. Collectively, this study adds to our understanding of the fungal microbiome of different mosquito species, that these fungal microbes support growth and development, and highlights that microbial interactions underpin fungal colonization of these medically relevent species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.552852">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.552852" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.552852">
        <p class="paperTitle">Microbial eukaryotic predation pressure and biomass at deep-sea hydrothermal vents: Implications for deep-sea carbon cycling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.552852" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.552852" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hu, S. K.; Anderson, R. E.; Pachiadaki, M. G.; Edgcomb, V. P.; Serres, M. H.; Sylva, S. P.; German, C. R.; Seewald, J. S.; Lang, S. Q.; Huber, J. A.</p>
        <p class="info">Score: 12.2, Published: 2023-08-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.552852' target='https://doi.org/10.1101/2023.08.11.552852'> 10.1101/2023.08.11.552852</a></p>
        <p class="abstract">Deep-sea hydrothermal vent geochemistry shapes the foundation of the microbial food web by fueling chemolithoautotrophic microbial activity. Microbial eukaryotes (or protists) play a critical role in hydrothermal vent food webs as consumers, hosts of symbiotic bacteria, and as a nutritional source to higher trophic levels. We measured cell abundances and predation pressures of vent-associated microbial eukaryotes in low temperature diffuse hydrothermal fluids at the Von Damm and Piccard vent fields along the Mid-Cayman Rise in the Western Caribbean Sea. We present findings from experiments performed under in situ pressure that show higher cell abundances and grazing rates compared to those done at 1 atmosphere (shipboard ambient pressure); this trend was attributed to the impact of depressurization on cell integrity. A relationship between protistan grazing rate, prey cell abundance, and temperature of end member hydrothermal vent fluid was observed at both vent fields, regardless of experimental approach. The quantification of protistan biomass and grazing pressure shows that hydrothermally-fueled microbial food webs play a significant role in the broader deep-sea carbon budget by contributing to local carbon export and supply of nutrient resources to the deep ocean.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.13.553119">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.13.553119" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.13.553119">
        <p class="paperTitle">Goblet cell invasion promotes breaching of respiratory epithelia by an opportunistic human pathogen</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.13.553119" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.13.553119" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Swart, A. L.; Laventie, B. J.; Suetterlin, R.; Junne, T.; Yu, X.; Karagkiozi, E.; Okujava, R.; Jenal, U.</p>
        <p class="info">Score: 11.4, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.13.553119' target='https://doi.org/10.1101/2023.08.13.553119'> 10.1101/2023.08.13.553119</a></p>
        <p class="abstract">While commensal bacteria generally respect natural barriers of the human body, pathogens are able to breach epithelia, invade deeper tissue layers and cause life-threatening infections. Pseudomonas aeruginosa, an opportunistic human pathogen, is a leading cause of severe hospital-acquired pneumonia, with mortality rates as high as 50% in mechanically ventilated patients1-3. Effective colonization and breaching of lung mucosa are hallmarks of P. aeruginosa pathogenesis4. Although virulence factors and behavioral strategies of P. aeruginosa have been described5,6, it has remained unclear how this pathogen disseminates on functional mucosal surfaces, how it avoids mucociliary clearance and how it invades the tissue barrier. Using fully differentiated human lung epithelia, we demonstrate that P. aeruginosa efficiently spreads on the apical tissue surface before it breaches epithelia by specifically invading mucus secreting goblet cells. Internalization leads to host cell death and expulsion and the formation of ruptures of the epithelial barrier. Rupture sites are rapidly colonized by extracellular bacteria through active chemotaxis, leading to increasing tissue damage and successful pathogen translocation to the unprotected basolateral side of the epithelium. We show that cell invasion is promoted by two Type-6 toxin secretion systems (T6SS), while Type-3 (T3SS) mediates cell death of infected goblet cells. T3SS mutants invade goblet cells normally, but internalized bacteria fail to trigger goblet cell expulsion and instead show unrestrained intracellular replication. While the effective shedding of infected host cells reveals potent tissue protection mechanisms, the discovery of an intracellular lifestyle of P. aeruginosa in human lung epithelia provides new entry points into investigating the intersection of antibiotic and immune mechanisms during lung infections. By demonstrating that P. aeruginosa uses a combination of specific virulence factors and collective behavior to invade goblet cells and breach the lung tissue barrier from within, these studies reveal novel mechanisms underlying lung infection dynamics under physiological conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552768">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552768" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552768">
        <p class="paperTitle">Cell division machinery drives cell-specific gene activation during bacterial differentiation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552768" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552768" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chareyre, S.; Li, X.; Anjuwon-Foster, B. R.; Clifford, S.; Brogan, A.; Su, Y.; Shroff, H.; Ramamurthi, K. S.</p>
        <p class="info">Score: 18.2, Published: 2023-08-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552768' target='https://doi.org/10.1101/2023.08.10.552768'> 10.1101/2023.08.10.552768</a></p>
        <p class="abstract">When faced with starvation, the bacterium Bacillus subtilis transforms itself into a dormant cell type called a &#34;spore&#34;. Sporulation initiates with an asymmetric division event, which requires the relocation of the core divisome components FtsA and FtsZ, after which the sigma factor {sigma}F is exclusively activated in the smaller daughter cell. Compartment specific activation of {sigma}F requires the SpoIIE phosphatase, which displays a biased localization on one side of the asymmetric division septum and associates with the structural protein DivIVA, but the mechanism by which this preferential localization is achieved is unclear. Here, we isolated a variant of DivIVA that indiscriminately activates {sigma}F in both daughter cells due to promiscuous localization of SpoIIE, which was corrected by overproduction of FtsA and FtsZ. We propose that a unique feature of the sporulation septum, defined by the cell division machinery, drives the asymmetric localization of DivIVA and SpoIIE to trigger the initiation of the sporulation program.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.553469">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.553469" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.553469">
        <p class="paperTitle">Defense mechanism of a bacterial retron supramolecular assembly</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.553469" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.553469" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Y.; Wang, C.; Guan, Z.; Cao, J.; Xu, J.; Wang, S.; Cui, Y.; Wang, Q.; Chen, Y.; Zhang, D.; Sun, M.; Tao, P.; Zou, T.</p>
        <p class="info">Score: 10.0, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.553469' target='https://doi.org/10.1101/2023.08.16.553469'> 10.1101/2023.08.16.553469</a></p>
        <p class="abstract">Retrons are a class of multigene antiphage defense system typically consisting of a retron reverse transcriptase, a non-coding RNA, and a cognate effector. Although the triggers for several retron systems have been discovered recently, the full picture of how retron systems sense invading phages and mediate defense remains to be elucidated. Here, we focus on the retron Ec86 defense system and report its modes of activation and action. We identified a phage-encoded DNA cytosine methyltransferase (Dcm) as the trigger of the Ec86 system and show that Ec86 senses msDNA methylation and becomes activated. We further determined the structure of a tripartite retron Ec86 supramolecular assembly, which is primed for activation by Dcm, and demonstrated that the activated system confers defense through depletion of nucleoside derivatives. These findings emphasize the role of retrons being a second line of defense and highlight an emerging theme of anti-phage defense through supramolecular complex assemblies.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
