<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Virological characteristics of the SARS-CoV-2 JN.1 variant
Authors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Zahradnik, J.; Ito, J.; Sato, K.
Score: 739.8, Published: 2023-12-09 DOI: 10.1101/2023.12.08.570782
The SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Virological characteristics of the SARS-CoV-2 JN.1 variant
Authors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Zahradnik, J.; Ito, J.; Sato, K.
Score: 739.8, Published: 2023-12-09 DOI: 10.1101/2023.12.08.570782
The SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-20T10:36:27+00:00" />
<meta property="article:modified_time" content="2023-12-20T10:36:27+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Virological characteristics of the SARS-CoV-2 JN.1 variant
Authors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Zahradnik, J.; Ito, J.; Sato, K.
Score: 739.8, Published: 2023-12-09 DOI: 10.1101/2023.12.08.570782
The SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Virological characteristics of the SARS-CoV-2 JN.1 variant\nAuthors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Zahradnik, J.; Ito, J.; Sato, K.\nScore: 739.8, Published: 2023-12-09 DOI: 10.1101/2023.12.08.570782\nThe SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG.",
  "keywords": [
    
  ],
  "articleBody": " Virological characteristics of the SARS-CoV-2 JN.1 variant\nAuthors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Zahradnik, J.; Ito, J.; Sato, K.\nScore: 739.8, Published: 2023-12-09 DOI: 10.1101/2023.12.08.570782\nThe SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG.5.1 and HK.3. Comparing to XBB and BA.2, BA.2.86 carries more than 30 mutations in the spike (S) protein, indicating a high potential for immune evasion. BA.2.86 has evolved and its descendant, JN.1 (BA.2.86.1.1), emerged in late 2023. JN.1 harbors S:L455S and three mutations in non-S proteins. S:L455S is a hallmark mutation of JN.1: we have recently shown that HK.3 and other \"FLip\" variants carry S:L455F, which contributes to increased transmissibility and immune escape ability compared to the parental EG.5.1 variant. Here, we investigated the virological properties of JN.1.\nAge-dependent heterogeneity in the antigenic effects of mutations to influenza hemagglutinin\nAuthors: Welsh, F. C.; Eguia, R. T.; Lee, J. M.; Haddox, H. K.; Galloway, J.; Vinh, N. V.; Loes, A. N.; Huddleston, J.; Yu, T. C.; Le, M. Q.; Nhat, N. T.; Thanh, N. T. L.; Greninger, A. L.; Chu, H. Y.; Englund, J. A.; Bedford, T.; Matsen, F. A.; Boni, M. F.; Bloom, J. D.\nScore: 75.6, Published: 2023-12-12 DOI: 10.1101/2023.12.12.571235\nABSTRACTHuman influenza virus evolves to escape neutralization by polyclonal antibodies. However, we have a limited understanding of how the antigenic effects of viral mutations vary across the human population, and how this heterogeneity affects virus evolution. Here we use deep mutational scanning to map how mutations to the hemagglutinin (HA) proteins of the A/Hong Kong/45/2019 (H3N2) and A/Perth/16/2009 (H3N2) strains affect neutralization by serum from individuals of a variety of ages. The effects of HA mutations on serum neutralization differ across age groups in ways that can be partially rationalized in terms of exposure histories. Mutations that fixed in influenza variants after 2020 cause the greatest escape from sera from younger individuals. Overall, these results demonstrate that influenza faces distinct antigenic selection regimes from different age groups, and suggest approaches to understand how this heterogeneous selection shapes viral evolution.\nRNA-guided RNA silencing by an Asgard archaeal Argonaute\nAuthors: Bastiaanssen, C.; Ugarte, P. B.; Kim, K.; Feng, Y.; Finocchio, G.; Anzelon, T. A.; Kostlbacher, S.; Tamarit, D.; Ettema, T. J. G.; Jinek, M.; MacRae, I. J.; Joo, C.; Swarts, D. C.; Wu, F.\nScore: 48.2, Published: 2023-12-15 DOI: 10.1101/2023.12.14.571608\nEukaryotic Argonaute proteins achieve gene repression and defense against viruses and transposons by RNA-guided RNA silencing. By contrast, known prokaryotic Argonautes adopt single-stranded DNA as guides and/or targets, leaving the evolutionary origin of RNA-guided RNA silencing elusive. Here, we show an evolutionary expansion of Asgard archaeal Argonautes (asAgos), including the discovery of HrAgo1 from the Lokiarchaeon Candidatus Harpocratesius repetitus that shares a common origin with eukaryotic PIWI proteins. HrAgo1 exhibits RNA-guided RNA cleavage in vitro and RNA silencing in human cells. The cryo-EM structure of HrAgo1 combined with quantitative single-molecule experiments reveals that HrAgo1 possesses hybrid structural features and target binding modes bridging those of the eukaryotic AGO and PIWI clades. Finally, genomic evidence suggests that eukaryotic Dicer-like processing of double-stranded RNA likely emerged as a mechanism of generating guide RNA for asAgos prior to eukaryogenesis. Our study provides new insights into the evolutionary origin and plasticity of Argonaute-based RNA silencing.\nThe Natural Products Discovery Center: Release of the First 8490 Sequenced Strains for Exploring Actinobacteria Biosynthetic Diversity\nAuthors: Kalkreuter, E.; Kautsar, S. A.; Yang, D.; Teijaro, C. N.; Bader, C. D.; Fluegel, L. L.; Davis, C. M.; Simpson, J. R.; Steele, A. D.; Gui, C.; Meng, S.; Li, G.; Viehrig, K.; Ye, F.; Su, P.; Kiefer, A. F.; Lauterbach, L.; Nichols, A.; Cepeda, A. J.; Yan, W.; Fan, B.; Jiang, Y.; Adhikari, A.; Zheng, C.-J.; Shen, B.\nScore: 28.0, Published: 2023-12-15 DOI: 10.1101/2023.12.14.571759\nActinobacteria, the bacterial phylum most renowned for natural product discovery, has been established as a valuable source for drug discovery and biotechnology but is underrepresented within accessible genome and strain collections. Herein, we introduce the Natural Products Discovery Center (NPDC), featuring 122,449 strains assembled over seven decades, the genomes of the first 8490 NPDC strains (7142 Actinobacteria), and the online NPDC Portal making both strains and genomes publicly available. A comparative survey of RefSeq and NPDC Actinobacteria highlights the taxonomic and biosynthetic diversity within the NPDC collection, including three new genera, hundreds of new species, and [~]7000 non-RefSeq gene cluster families. Selected examples demonstrate how the NPDC Portals strain metadata, genomes, and biosynthetic gene clusters can be leveraged using genome mining approaches. Our findings underscore the ongoing significance of Actinobacteria in natural product discovery, and the NPDC serves as an unparalleled resource for both Actinobacteria strains and genomes.\nExtensive variation and strain-specificity in dengue virus susceptibility among African Aedes aegypti populations\nAuthors: Dabo, S.; Henrion-Lacritick, A.; Lecuyer, A.; Jiolle, D.; Paupy, C.; Ayala, D.; da Veiga Leal, S.; Badolo, A.; Vega-Rua, A.; Sylla, M.; Akorli, J.; Otoo, S.; Lutomiah, J.; Sang, R.; Mutebi, J.-P.; Saleh, M.-C.; Rose, N.; McBride, C.; Lambrechts, L.\nScore: 24.5, Published: 2023-12-14 DOI: 10.1101/2023.12.14.571617\nAfrican populations of the mosquito Aedes aegypti are usually considered less susceptible to infection by human-pathogenic flaviviruses than globally invasive populations found outside Africa. Although this contrast has been well documented for Zika virus (ZIKV), it is unclear to what extent it is true for dengue virus (DENV), the most prevalent flavivirus of humans. Addressing this question is complicated by substantial genetic diversity among DENV strains, most notably in the form of four genetic types (DENV1 to DENV4), that can lead to genetically specific interactions with mosquito populations. Here, we carried out a continent-wide survey of DENV susceptibility using a panel of field-derived Ae. aegypti colonies from across the African range of the species and a colony from Guadeloupe, French West Indies as non-African reference. We found considerable variation in the ability of African Ae. aegypti populations to acquire and replicate a panel of six DENV strains spanning the four DENV types. Although African Ae. aegypti populations were generally less susceptible than the reference non-African population from Guadeloupe, in several instances some African populations were equally or more susceptible than the Guadeloupe population. Moreover, the relative level of susceptibility between African mosquito populations depended on the DENV strain, indicating genetically specific interactions. We conclude that unlike ZIKV susceptibility, there is no clear-cut dichotomy in DENV susceptibility between African and non-African Ae. aegypti. DENV susceptibility of African Ae. aegypti populations is highly heterogeneous and largely governed by the specific pairing of mosquito population and DENV strain. Author summaryAfrican populations of the mosquito Aedes aegypti are usually thought to be less likely to get infected by flaviviruses compared to Ae. aegypti mosquitoes found outside Africa. While this has been well-demonstrated for Zika virus, it is not clear if the same is true for dengue virus, which is the most common flavivirus in humans. Studying this is complicated by the strain diversity of dengue virus, including four main genetic types, potentially causing different interactions. In this study, we compared several mosquito populations and found that, in general, African mosquitoes were less likely to get infected by dengue virus compared to mosquitoes from outside Africa. However, in some cases, African mosquitoes were just as or even more likely to get infected. The specific strain of dengue virus also influenced how likely African mosquitoes were to get infected, showing that the relationship between African mosquitoes and dengue virus is complex.\nMost azole antifungal resistance mutations in the drug target provide cross-resistance and carry no intrinsic fitness cost\nAuthors: Bedard, C.; Gagnon-Arsenault, I.; Boisvert, J.; Plante, S.; Dube, A. K.; Pageau, A.; Sharma, J.; Maroc, L.; Shapiro, R. S.; Landry, C. R.\nScore: 19.8, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571438\nAzole antifungals are among the most frequently used drugs to treat fungal infections. Amino acid substitutions in and around the binding site of the azole target Erg11 (Cyp51) are a common resistance mechanism in pathogenic yeasts such as Candida albicans. How many and which mutations confer resistance, and at what cost, is however largely unknown. Here, we measure the impact of nearly 4,000 amino acid variants of the Erg11 ligand binding pocket on the susceptibility to six medical azoles. We find that a large fraction of amino acid substitutions lead to resistance (33%), most resistance mutations confer cross-resistance to two or more azoles (88%) and most importantly, only a handful of resistance mutations show a significant fitness cost in the absence of drug (9%). Our results reveal that selection for azole resistance can arise through a large set of mutations and this will likely lead to azole pan-resistance, with very little evolutionary compromise. Such a resource will help inform treatment choices in clinical settings and guide the development of new drugs.\nMIMt: A curated 16S rRNA reference database with less redundancy and higher accuracy at species-level identification\nAuthors: Cabezas Rodriguez, M. P.; Fonseca, N. A.; Munoz-Merida, A.\nScore: 19.1, Published: 2023-12-15 DOI: 10.1101/2023.12.15.571851\nMotivationAccurate determination and quantification of the taxonomic composition of microbial communities, especially at the species level, is one of the major issues in metagenomics. This is primarily due to the limitations of commonly used 16S rRNA reference databases, which either contain a lot of redundancy, or a high percentage of sequences with missing taxonomic information. The use of these incomplete or biased databases may lead to erroneous identifications and, thus, to erroneous conclusions regarding the ecological role and importance of those microorganisms in the ecosystem. ResultsThe current study presents MIMt, a new 16S rRNA database for archaea and bacterias identification, encompassing 39 940 sequences, all precisely identified at species level. MIMt aims to be updated at least once a year to include all new sequenced species. We evaluated MIMt against Greengenes, RDP, GTDB and SILVA in terms of sequence distribution and accuracy of taxonomic assignments. Our results showed that MIMt contains less redundancy, and despite being five to 85 times smaller in size than existing databases, outperforms them in completeness and taxonomic accuracy, enabling more precise assignments at lower taxonomic ranks and thus, significantly improving species-level identification. Availability and ImplementationMIMt is freely available for non-commercial purposes at https://mimt.bu.biopolis.pt\nDetection and spread of high pathogenicity avian influenza virus H5N1 in the Antarctic Region\nAuthors: Bennison, A.; Byrne, A. M.; Reid, S. M.; Lynton-Jenkins, J. G.; Mollett, B.; De Sliva, D.; Peers-Dent, J.; Finlayson, K. E.; Hall, R.; Blockley, F.; Blyth, M.; Falchieri, M.; Fowler, Z.; Fitzcharles, E.; Brown, I. H.; James, J.; Banyard, A. C.\nScore: 96.8, Published: 2023-11-24 DOI: 10.1101/2023.11.23.568045\nThe Antarctic is the only major geographical region in which high pathogenicity avian influenza virus (HPAIV) has never previously been detected. The current panzootic of H5N1 HPAIV has decimated wild bird populations across Europe, North America and South America. Here we report on the emergence of clade 2.3.4.4b H5N1 HPAIV in the Antarctic and sub-Antarctic regions of South Georgia and the Falkland Islands respectively. We initially detected H5N1 HPAIV in samples collected from brown skuas at Bird Island, South Georgia on 8th October 2023. Since this detection, increased mortalities were observed in brown skuas, kelp gulls, elephant seals and fur seal at multiple sites across South Georgia. We confirmed H5N1 HPAIV in multiple brown skuas and kelp gulls across four different sampling locations in South Georgia. Simultaneously, we also confirmed H5N1 HPAIV in a southern fulmar in the Falkland Islands. Genetic assessment of the virus indicates spread from South America, likely through movement of migratory birds. Here we describe the emergence, species impact and genetic composition of the virus and propose both introductory routes and potential long-term impact on avian and mammalian species across the Antarctic region.\nPan-azole- and multi-fungicide-resistant Aspergillus fumigatus is widespread in the United States\nAuthors: Celia-Sanchez, B.; Mangum, B.; Gomez, L. L.; Wang, C.; Shuman, B.; Brewer, M. T.; Momany, M.\nScore: 17.1, Published: 2023-12-15 DOI: 10.1101/2023.12.14.571763\nAspergillus fumigatus is an important global fungal pathogen of humans. Azole drugs are among the most effective treatments for A. fumigatus infection. Azoles are also widely used in agriculture as fungicides against fungal pathogens of crops. Azole-resistant A. fumigatus has been increasing in Europe and Asia for two decades where clinical resistance is thought to be driven by agricultural use of azole fungicides. The most prevalent mechanisms of azole resistance in A. fumigatus are tandem repeats (TR) in the cyp51A promoter coupled with mutations in the coding region which result in resistance to multiple azole drugs (pan-azole resistance). Azole-resistant A. fumigatus has been isolated from patients in the United States (U.S.), but little is known about its environmental distribution. To better understand the distribution of azole-resistant A. fumigatus in the U.S., we collected isolates from agricultural sites in 8 states and tested 202 isolates for sensitivity to azoles. We found azole-resistant A. fumigatus in agricultural environments in 7 states showing that it is widespread in the U.S. We sequenced environmental isolates representing the range of U.S. sample sites and compared them with publicly available environmental worldwide isolates in phylogenetic, principal component, and ADMIXTURE analyses. We found worldwide isolates fell into three clades and that TR-based pan-azole resistance was largely in a single clade that was strongly associated with resistance to multiple agricultural fungicides. We also found high levels of gene flow with clear recombination between two clades highlighting the potential for azole-resistance to continue spreading in the U.S. ImportanceAspergillus fumigatus is a fungal pathogen of humans that causes over 250,000 invasive infections each year. It is found in soils, plant debris and compost. Azoles are the first line of defense antifungal drugs against A. fumigatus. Azoles are also used as agricultural fungicides to combat other fungi that attack plants. Azole-resistant A. fumigatus has been a problem in Europe and Asia for twenty years and has recently been reported in patients in the U.S. Until this study we didnt know much about azole-resistant A. fumigatus in agricultural settings in the U.S. In this study we isolated azole-resistant A. fumigatus from multiple states and compared it to isolates from around the world. We show that A. fumigatus that is resistant to azoles and to other strictly agricultural fungicides is widespread in the U.S.\nA broad antibody class engages the influenza virus hemagglutinin head at its stem interface\nAuthors: Simmons, H. C.; Finney, J.; Kotaki, R.; Adachi, Y.; Moseman, A. P.; Watanabe, A.; Song, S.; Robinson-McCarthy, L. R.; Le Sage, V.; Kuraoka, M.; Moseman, E. A.; Kelsoe, G.; Takahashi, Y.; McCarthy, K. R.\nScore: 16.8, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571543\nInfluenza infection and vaccination impart strain-specific immunity that fails to protect against both seasonal antigenic variants and the next pandemic. However, antibodies directed to conserved sites can confer broad protection. We identify and characterize a class of human antibodies that engage a previously undescribed, conserved, epitope on the influenza hemagglutinin protein (HA). Prototype antibody S8V1-157 binds at the normally occluded interface between the HA head and stem. Antibodies to this HA head-stem interface epitope are non-neutralizing in vitro but protect against lethal infection in mice. Their breadth of binding extends across most influenza A serotypes and seasonal human variants. Antibodies to the head-stem interface epitope are present at low frequency in the memory B cell populations of multiple donors. The immunogenicity of the epitope warrants its consideration for inclusion in improved or \"universal\" influenza vaccines.\n",
  "wordCount" : "2560",
  "inLanguage": "en",
  "datePublished": "2023-12-20T10:36:27Z",
  "dateModified": "2023-12-20T10:36:27Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 20, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.570782">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.570782" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.570782">
        <p class="paperTitle">Virological characteristics of the SARS-CoV-2 JN.1 variant</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.570782" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.570782" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium,  ; Zahradnik, J.; Ito, J.; Sato, K.</p>
        <p class="info">Score: 739.8, Published: 2023-12-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.570782' target='https://doi.org/10.1101/2023.12.08.570782'> 10.1101/2023.12.08.570782</a></p>
        <p class="abstract">The SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG.5.1 and HK.3. Comparing to XBB and BA.2, BA.2.86 carries more than 30 mutations in the spike (S) protein, indicating a high potential for immune evasion. BA.2.86 has evolved and its descendant, JN.1 (BA.2.86.1.1), emerged in late 2023. JN.1 harbors S:L455S and three mutations in non-S proteins. S:L455S is a hallmark mutation of JN.1: we have recently shown that HK.3 and other &#34;FLip&#34; variants carry S:L455F, which contributes to increased transmissibility and immune escape ability compared to the parental EG.5.1 variant. Here, we investigated the virological properties of JN.1.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.12.571235">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.12.571235" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.12.571235">
        <p class="paperTitle">Age-dependent heterogeneity in the antigenic effects of mutations to influenza hemagglutinin</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.12.571235" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.12.571235" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Welsh, F. C.; Eguia, R. T.; Lee, J. M.; Haddox, H. K.; Galloway, J.; Vinh, N. V.; Loes, A. N.; Huddleston, J.; Yu, T. C.; Le, M. Q.; Nhat, N. T.; Thanh, N. T. L.; Greninger, A. L.; Chu, H. Y.; Englund, J. A.; Bedford, T.; Matsen, F. A.; Boni, M. F.; Bloom, J. D.</p>
        <p class="info">Score: 75.6, Published: 2023-12-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.12.571235' target='https://doi.org/10.1101/2023.12.12.571235'> 10.1101/2023.12.12.571235</a></p>
        <p class="abstract">ABSTRACTHuman influenza virus evolves to escape neutralization by polyclonal antibodies. However, we have a limited understanding of how the antigenic effects of viral mutations vary across the human population, and how this heterogeneity affects virus evolution. Here we use deep mutational scanning to map how mutations to the hemagglutinin (HA) proteins of the A/Hong Kong/45/2019 (H3N2) and A/Perth/16/2009 (H3N2) strains affect neutralization by serum from individuals of a variety of ages. The effects of HA mutations on serum neutralization differ across age groups in ways that can be partially rationalized in terms of exposure histories. Mutations that fixed in influenza variants after 2020 cause the greatest escape from sera from younger individuals. Overall, these results demonstrate that influenza faces distinct antigenic selection regimes from different age groups, and suggest approaches to understand how this heterogeneous selection shapes viral evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.571608">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.571608" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.571608">
        <p class="paperTitle">RNA-guided RNA silencing by an Asgard archaeal Argonaute</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.571608" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.571608" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bastiaanssen, C.; Ugarte, P. B.; Kim, K.; Feng, Y.; Finocchio, G.; Anzelon, T. A.; Kostlbacher, S.; Tamarit, D.; Ettema, T. J. G.; Jinek, M.; MacRae, I. J.; Joo, C.; Swarts, D. C.; Wu, F.</p>
        <p class="info">Score: 48.2, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.571608' target='https://doi.org/10.1101/2023.12.14.571608'> 10.1101/2023.12.14.571608</a></p>
        <p class="abstract">Eukaryotic Argonaute proteins achieve gene repression and defense against viruses and transposons by RNA-guided RNA silencing. By contrast, known prokaryotic Argonautes adopt single-stranded DNA as guides and/or targets, leaving the evolutionary origin of RNA-guided RNA silencing elusive. Here, we show an evolutionary expansion of Asgard archaeal Argonautes (asAgos), including the discovery of HrAgo1 from the Lokiarchaeon  Candidatus Harpocratesius repetitus that shares a common origin with eukaryotic PIWI proteins. HrAgo1 exhibits RNA-guided RNA cleavage in vitro and RNA silencing in human cells. The cryo-EM structure of HrAgo1 combined with quantitative single-molecule experiments reveals that HrAgo1 possesses hybrid structural features and target binding modes bridging those of the eukaryotic AGO and PIWI clades. Finally, genomic evidence suggests that eukaryotic Dicer-like processing of double-stranded RNA likely emerged as a mechanism of generating guide RNA for asAgos prior to eukaryogenesis. Our study provides new insights into the evolutionary origin and plasticity of Argonaute-based RNA silencing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.571759">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.571759" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.571759">
        <p class="paperTitle">The Natural Products Discovery Center: Release of the First 8490 Sequenced Strains for Exploring Actinobacteria Biosynthetic Diversity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.571759" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.571759" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kalkreuter, E.; Kautsar, S. A.; Yang, D.; Teijaro, C. N.; Bader, C. D.; Fluegel, L. L.; Davis, C. M.; Simpson, J. R.; Steele, A. D.; Gui, C.; Meng, S.; Li, G.; Viehrig, K.; Ye, F.; Su, P.; Kiefer, A. F.; Lauterbach, L.; Nichols, A.; Cepeda, A. J.; Yan, W.; Fan, B.; Jiang, Y.; Adhikari, A.; Zheng, C.-J.; Shen, B.</p>
        <p class="info">Score: 28.0, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.571759' target='https://doi.org/10.1101/2023.12.14.571759'> 10.1101/2023.12.14.571759</a></p>
        <p class="abstract">Actinobacteria, the bacterial phylum most renowned for natural product discovery, has been established as a valuable source for drug discovery and biotechnology but is underrepresented within accessible genome and strain collections. Herein, we introduce the Natural Products Discovery Center (NPDC), featuring 122,449 strains assembled over seven decades, the genomes of the first 8490 NPDC strains (7142 Actinobacteria), and the online NPDC Portal making both strains and genomes publicly available. A comparative survey of RefSeq and NPDC Actinobacteria highlights the taxonomic and biosynthetic diversity within the NPDC collection, including three new genera, hundreds of new species, and [~]7000 non-RefSeq gene cluster families. Selected examples demonstrate how the NPDC Portals strain metadata, genomes, and biosynthetic gene clusters can be leveraged using genome mining approaches. Our findings underscore the ongoing significance of Actinobacteria in natural product discovery, and the NPDC serves as an unparalleled resource for both Actinobacteria strains and genomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.571617">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.571617" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.571617">
        <p class="paperTitle">Extensive variation and strain-specificity in dengue virus susceptibility among African Aedes aegypti populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.571617" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.571617" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dabo, S.; Henrion-Lacritick, A.; Lecuyer, A.; Jiolle, D.; Paupy, C.; Ayala, D.; da Veiga Leal, S.; Badolo, A.; Vega-Rua, A.; Sylla, M.; Akorli, J.; Otoo, S.; Lutomiah, J.; Sang, R.; Mutebi, J.-P.; Saleh, M.-C.; Rose, N.; McBride, C.; Lambrechts, L.</p>
        <p class="info">Score: 24.5, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.571617' target='https://doi.org/10.1101/2023.12.14.571617'> 10.1101/2023.12.14.571617</a></p>
        <p class="abstract">African populations of the mosquito Aedes aegypti are usually considered less susceptible to infection by human-pathogenic flaviviruses than globally invasive populations found outside Africa. Although this contrast has been well documented for Zika virus (ZIKV), it is unclear to what extent it is true for dengue virus (DENV), the most prevalent flavivirus of humans. Addressing this question is complicated by substantial genetic diversity among DENV strains, most notably in the form of four genetic types (DENV1 to DENV4), that can lead to genetically specific interactions with mosquito populations. Here, we carried out a continent-wide survey of DENV susceptibility using a panel of field-derived Ae. aegypti colonies from across the African range of the species and a colony from Guadeloupe, French West Indies as non-African reference. We found considerable variation in the ability of African Ae. aegypti populations to acquire and replicate a panel of six DENV strains spanning the four DENV types. Although African Ae. aegypti populations were generally less susceptible than the reference non-African population from Guadeloupe, in several instances some African populations were equally or more susceptible than the Guadeloupe population. Moreover, the relative level of susceptibility between African mosquito populations depended on the DENV strain, indicating genetically specific interactions. We conclude that unlike ZIKV susceptibility, there is no clear-cut dichotomy in DENV susceptibility between African and non-African Ae. aegypti. DENV susceptibility of African Ae. aegypti populations is highly heterogeneous and largely governed by the specific pairing of mosquito population and DENV strain.

Author summaryAfrican populations of the mosquito Aedes aegypti are usually thought to be less likely to get infected by flaviviruses compared to Ae. aegypti mosquitoes found outside Africa. While this has been well-demonstrated for Zika virus, it is not clear if the same is true for dengue virus, which is the most common flavivirus in humans. Studying this is complicated by the strain diversity of dengue virus, including four main genetic types, potentially causing different interactions. In this study, we compared several mosquito populations and found that, in general, African mosquitoes were less likely to get infected by dengue virus compared to mosquitoes from outside Africa. However, in some cases, African mosquitoes were just as or even more likely to get infected. The specific strain of dengue virus also influenced how likely African mosquitoes were to get infected, showing that the relationship between African mosquitoes and dengue virus is complex.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571438">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571438" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571438">
        <p class="paperTitle">Most azole antifungal resistance mutations in the drug target provide cross-resistance and carry no intrinsic fitness cost</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571438" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571438" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bedard, C.; Gagnon-Arsenault, I.; Boisvert, J.; Plante, S.; Dube, A. K.; Pageau, A.; Sharma, J.; Maroc, L.; Shapiro, R. S.; Landry, C. R.</p>
        <p class="info">Score: 19.8, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571438' target='https://doi.org/10.1101/2023.12.13.571438'> 10.1101/2023.12.13.571438</a></p>
        <p class="abstract">Azole antifungals are among the most frequently used drugs to treat fungal infections. Amino acid substitutions in and around the binding site of the azole target Erg11 (Cyp51) are a common resistance mechanism in pathogenic yeasts such as Candida albicans. How many and which mutations confer resistance, and at what cost, is however largely unknown. Here, we measure the impact of nearly 4,000 amino acid variants of the Erg11 ligand binding pocket on the susceptibility to six medical azoles. We find that a large fraction of amino acid substitutions lead to resistance (33%), most resistance mutations confer cross-resistance to two or more azoles (88%) and most importantly, only a handful of resistance mutations show a significant fitness cost in the absence of drug (9%). Our results reveal that selection for azole resistance can arise through a large set of mutations and this will likely lead to azole pan-resistance, with very little evolutionary compromise. Such a resource will help inform treatment choices in clinical settings and guide the development of new drugs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.571851">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.571851" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.571851">
        <p class="paperTitle">MIMt: A curated 16S rRNA reference database with less redundancy and higher accuracy at species-level identification</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.571851" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.571851" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cabezas Rodriguez, M. P.; Fonseca, N. A.; Munoz-Merida, A.</p>
        <p class="info">Score: 19.1, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.571851' target='https://doi.org/10.1101/2023.12.15.571851'> 10.1101/2023.12.15.571851</a></p>
        <p class="abstract">MotivationAccurate determination and quantification of the taxonomic composition of microbial communities, especially at the species level, is one of the major issues in metagenomics. This is primarily due to the limitations of commonly used 16S rRNA reference databases, which either contain a lot of redundancy, or a high percentage of sequences with missing taxonomic information. The use of these incomplete or biased databases may lead to erroneous identifications and, thus, to erroneous conclusions regarding the ecological role and importance of those microorganisms in the ecosystem.

ResultsThe current study presents MIMt, a new 16S rRNA database for archaea and bacterias identification, encompassing 39 940 sequences, all precisely identified at species level. MIMt aims to be updated at least once a year to include all new sequenced species. We evaluated MIMt against Greengenes, RDP, GTDB and SILVA in terms of sequence distribution and accuracy of taxonomic assignments. Our results showed that MIMt contains less redundancy, and despite being five to 85 times smaller in size than existing databases, outperforms them in completeness and taxonomic accuracy, enabling more precise assignments at lower taxonomic ranks and thus, significantly improving species-level identification.

Availability and ImplementationMIMt is freely available for non-commercial purposes at https://mimt.bu.biopolis.pt</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.568045">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.568045" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.568045">
        <p class="paperTitle">Detection and spread of high pathogenicity avian influenza virus H5N1 in the Antarctic Region</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.568045" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.568045" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bennison, A.; Byrne, A. M.; Reid, S. M.; Lynton-Jenkins, J. G.; Mollett, B.; De Sliva, D.; Peers-Dent, J.; Finlayson, K. E.; Hall, R.; Blockley, F.; Blyth, M.; Falchieri, M.; Fowler, Z.; Fitzcharles, E.; Brown, I. H.; James, J.; Banyard, A. C.</p>
        <p class="info">Score: 96.8, Published: 2023-11-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.568045' target='https://doi.org/10.1101/2023.11.23.568045'> 10.1101/2023.11.23.568045</a></p>
        <p class="abstract">The Antarctic is the only major geographical region in which high pathogenicity avian influenza virus (HPAIV) has never previously been detected. The current panzootic of H5N1 HPAIV has decimated wild bird populations across Europe, North America and South America. Here we report on the emergence of clade 2.3.4.4b H5N1 HPAIV in the Antarctic and sub-Antarctic regions of South Georgia and the Falkland Islands respectively. We initially detected H5N1 HPAIV in samples collected from brown skuas at Bird Island, South Georgia on 8th October 2023. Since this detection, increased mortalities were observed in brown skuas, kelp gulls, elephant seals and fur seal at multiple sites across South Georgia. We confirmed H5N1 HPAIV in multiple brown skuas and kelp gulls across four different sampling locations in South Georgia. Simultaneously, we also confirmed H5N1 HPAIV in a southern fulmar in the Falkland Islands. Genetic assessment of the virus indicates spread from South America, likely through movement of migratory birds. Here we describe the emergence, species impact and genetic composition of the virus and propose both introductory routes and potential long-term impact on avian and mammalian species across the Antarctic region.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.571763">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.571763" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.571763">
        <p class="paperTitle">Pan-azole- and multi-fungicide-resistant Aspergillus fumigatus is widespread in the United States</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.571763" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.571763" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Celia-Sanchez, B.; Mangum, B.; Gomez, L. L.; Wang, C.; Shuman, B.; Brewer, M. T.; Momany, M.</p>
        <p class="info">Score: 17.1, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.571763' target='https://doi.org/10.1101/2023.12.14.571763'> 10.1101/2023.12.14.571763</a></p>
        <p class="abstract">Aspergillus fumigatus is an important global fungal pathogen of humans. Azole drugs are among the most effective treatments for A. fumigatus infection. Azoles are also widely used in agriculture as fungicides against fungal pathogens of crops. Azole-resistant A. fumigatus has been increasing in Europe and Asia for two decades where clinical resistance is thought to be driven by agricultural use of azole fungicides. The most prevalent mechanisms of azole resistance in A. fumigatus are tandem repeats (TR) in the cyp51A promoter coupled with mutations in the coding region which result in resistance to multiple azole drugs (pan-azole resistance). Azole-resistant A. fumigatus has been isolated from patients in the United States (U.S.), but little is known about its environmental distribution. To better understand the distribution of azole-resistant A. fumigatus in the U.S., we collected isolates from agricultural sites in 8 states and tested 202 isolates for sensitivity to azoles. We found azole-resistant A. fumigatus in agricultural environments in 7 states showing that it is widespread in the U.S. We sequenced environmental isolates representing the range of U.S. sample sites and compared them with publicly available environmental worldwide isolates in phylogenetic, principal component, and ADMIXTURE analyses. We found worldwide isolates fell into three clades and that TR-based pan-azole resistance was largely in a single clade that was strongly associated with resistance to multiple agricultural fungicides. We also found high levels of gene flow with clear recombination between two clades highlighting the potential for azole-resistance to continue spreading in the U.S.

ImportanceAspergillus fumigatus is a fungal pathogen of humans that causes over 250,000 invasive infections each year. It is found in soils, plant debris and compost. Azoles are the first line of defense antifungal drugs against A. fumigatus. Azoles are also used as agricultural fungicides to combat other fungi that attack plants. Azole-resistant A. fumigatus has been a problem in Europe and Asia for twenty years and has recently been reported in patients in the U.S. Until this study we didnt know much about azole-resistant A. fumigatus in agricultural settings in the U.S. In this study we isolated azole-resistant A. fumigatus from multiple states and compared it to isolates from around the world. We show that A. fumigatus that is resistant to azoles and to other strictly agricultural fungicides is widespread in the U.S.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571543">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571543" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571543">
        <p class="paperTitle">A broad antibody class engages the influenza virus hemagglutinin head at its stem interface</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571543" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571543" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Simmons, H. C.; Finney, J.; Kotaki, R.; Adachi, Y.; Moseman, A. P.; Watanabe, A.; Song, S.; Robinson-McCarthy, L. R.; Le Sage, V.; Kuraoka, M.; Moseman, E. A.; Kelsoe, G.; Takahashi, Y.; McCarthy, K. R.</p>
        <p class="info">Score: 16.8, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571543' target='https://doi.org/10.1101/2023.12.13.571543'> 10.1101/2023.12.13.571543</a></p>
        <p class="abstract">Influenza infection and vaccination impart strain-specific immunity that fails to protect against both seasonal antigenic variants and the next pandemic. However, antibodies directed to conserved sites can confer broad protection. We identify and characterize a class of human antibodies that engage a previously undescribed, conserved, epitope on the influenza hemagglutinin protein (HA). Prototype antibody S8V1-157 binds at the normally occluded interface between the HA head and stem. Antibodies to this HA head-stem interface epitope are non-neutralizing in vitro but protect against lethal infection in mice. Their breadth of binding extends across most influenza A serotypes and seasonal human variants. Antibodies to the head-stem interface epitope are present at low frequency in the memory B cell populations of multiple donors. The immunogenicity of the epitope warrants its consideration for inclusion in improved or &#34;universal&#34; influenza vaccines.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
