<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Lethal Infection of Human ACE2-Transgenic Mice Caused by SARS-CoV-2-related Pangolin Coronavirus GX_P2V(short_3UTR)
Authors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.
Score: 1787.1, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574008
SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) can cause 100% mortality in human ACE2-transgenic mice, potentially attributable to late-stage brain infection. This underscores a spillover risk of GX_P2V into humans and provides a unique model for understanding the pathogenic mechanisms of SARS-CoV-2-related viruses.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Lethal Infection of Human ACE2-Transgenic Mice Caused by SARS-CoV-2-related Pangolin Coronavirus GX_P2V(short_3UTR)
Authors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.
Score: 1787.1, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574008
SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) can cause 100% mortality in human ACE2-transgenic mice, potentially attributable to late-stage brain infection. This underscores a spillover risk of GX_P2V into humans and provides a unique model for understanding the pathogenic mechanisms of SARS-CoV-2-related viruses." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-14T10:38:40+00:00" />
<meta property="article:modified_time" content="2024-01-14T10:38:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Lethal Infection of Human ACE2-Transgenic Mice Caused by SARS-CoV-2-related Pangolin Coronavirus GX_P2V(short_3UTR)
Authors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.
Score: 1787.1, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574008
SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) can cause 100% mortality in human ACE2-transgenic mice, potentially attributable to late-stage brain infection. This underscores a spillover risk of GX_P2V into humans and provides a unique model for understanding the pathogenic mechanisms of SARS-CoV-2-related viruses."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Lethal Infection of Human ACE2-Transgenic Mice Caused by SARS-CoV-2-related Pangolin Coronavirus GX_P2V(short_3UTR)\nAuthors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.\nScore: 1787.1, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574008\nSARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) can cause 100% mortality in human ACE2-transgenic mice, potentially attributable to late-stage brain infection. This underscores a spillover risk of GX_P2V into humans and provides a unique model for understanding the pathogenic mechanisms of SARS-CoV-2-related viruses.",
  "keywords": [
    
  ],
  "articleBody": " Lethal Infection of Human ACE2-Transgenic Mice Caused by SARS-CoV-2-related Pangolin Coronavirus GX_P2V(short_3UTR)\nAuthors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.\nScore: 1787.1, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574008\nSARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) can cause 100% mortality in human ACE2-transgenic mice, potentially attributable to late-stage brain infection. This underscores a spillover risk of GX_P2V into humans and provides a unique model for understanding the pathogenic mechanisms of SARS-CoV-2-related viruses.\nRapid Emergence and Evolution of SARS-CoV-2 Variants in Advanced HIV Infection\nAuthors: Ko, S. H.; Radecki, P.; Belinky, F.; Bhiman, J.; Meiring, S.; Kleynhans, J.; Amoako, D.; Guerra Canedo, V.; Lucas, M.; Kekana, D.; Martinson, N.; Lebina, L.; Everatt, J.; Tempia, S.; Bylund, T.; Rawi, R.; Kwong, P.; Wolter, N.; von Gottberg, A.; Cohen, C.; Boritz, E. A.\nScore: 77.6, Published: 2024-01-06 DOI: 10.1101/2024.01.05.574420\nPrevious studies have linked the evolution of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) genetic variants to persistent infections in people with immunocompromising conditions1-4, but the evolutionary processes underlying these observations are incompletely understood. Here we used high-throughput, single-genome amplification and sequencing (HT-SGS) to obtain up to [~]103 SARS-CoV-2 spike gene sequences in each of 184 respiratory samples from 22 people with HIV (PWH) and 25 people without HIV (PWOH). Twelve of 22 PWH had advanced HIV infection, defined by peripheral blood CD4 T cell counts (i.e., CD4 counts) \u003c200 cells/L. In PWOH and PWH with CD4 counts [\u0026ge;]200 cells/L, most single-genome spike sequences in each person matched one haplotype that predominated throughout the infection. By contrast, people with advanced HIV showed elevated intra-host spike diversity with a median of 46 haplotypes per person (IQR 14-114). Higher intra-host spike diversity immediately after COVID-19 symptom onset predicted longer SARS-CoV-2 RNA shedding among PWH, and intra-host spike diversity at this timepoint was significantly higher in people with advanced HIV than in PWOH. Composition of spike sequence populations in people with advanced HIV fluctuated rapidly over time, with founder sequences often replaced by groups of new haplotypes. These population-level changes were associated with a high total burden of intra-host mutations and positive selection at functionally important residues. In several cases, delayed emergence of detectable serum binding to spike was associated with positive selection for presumptive antibody-escape mutations. Taken together, our findings show remarkable intra-host genetic diversity of SARS-CoV-2 in advanced HIV infection and suggest that adaptive intra-host SARS-CoV-2 evolution in this setting may contribute to the emergence of new variants of concern (VOCs).\nPreferential apical infection of intestinal cell monolayers by SARS-CoV-2 is associated with damage to cellular barrier integrity: Implications for the physiopathology of COVID-19\nAuthors: Clemence, G.; Jeffrey, A.; Jonatane, A.; Mege, J.-L.; Omar Osman, I.; Devaux, C. A.\nScore: 58.1, Published: 2024-01-09 DOI: 10.1101/2024.01.08.574642\nSARS-CoV-2 can infect different organs, including the intestine. In Caco-2 intestinal cell line, SARS-CoV-2 modulates the ACE2 receptor expression and affects the expression of molecules involved in intercellular junctions. To further explore the possibility that the intestinal epithelium serves as an alternative infection route for SARS-CoV-2, we used a model of polarised intestinal cell monolayers grown on the polycarbonate membrane of Transwell inserts, inoculated with the virus either in the upper or lower chamber of culture. In both polarised Caco-2 cell monolayers and co-culture Caco-2/HT29 cell monolayer, apical SARS-CoV-2 inoculation was found to be much more effective in establishing infection than basolateral inoculation. In addition, apical SARS-CoV-2 infection triggers monolayer degeneration, as shown by histological examination, measurement of trans-epithelial electronic resistance, and cell adhesion molecule expression. During this process, the infectious viruses reach the lower chamber, suggesting either a transcytosis mechanism from the apical side to the basolateral side of cells, a paracellular trafficking of the virus after damage to intercellular junctions in the epithelial barrier, or both. Taken together, these data highlight a preferential tropism of SARS-CoV-2 for the apical side of the human intestinal tract and suggests that infection via the intestinal lumen leads to a systemic infection.\nTransient RNA structures underlie highly pathogenic avian influenza virus genesis\nAuthors: Funk, M.; Spronken, M. I.; Bestebroer, T. M.; de Bruin, A. C. M.; Gultyaev, A. P.; Fouchier, R. A. M.; te Velthuis, A. J. W.; Richard, M.\nScore: 30.7, Published: 2024-01-11 DOI: 10.1101/2024.01.11.574333\nHighly pathogenic avian influenza viruses (HPAIVs) cause severe disease and high fatality in poultry1. They emerge exclusively from H5 and H7 low pathogenic avian influenza viruses (LPAIVs)2. Although insertion of a furin-cleavable multibasic cleavage site (MBCS) in the hemagglutinin gene was identified decades ago as the genetic basis for LPAIV-to-HPAIV transition3,4, the exact mechanisms underlying said insertion have remained unknown. Here we used an innovative combination of bioinformatic models to predict RNA structures forming around the influenza virus RNA polymerase during replication, and circular sequencing5 to reliably detect nucleotide insertions. We show that transient H5 hemagglutinin RNA structures predicted to trap the polymerase on purine-rich sequences drive nucleotide insertions characteristic of MBCSs, providing the first strong empirical evidence of RNA structure involvement in MBCS acquisition. Insertion frequencies at the H5 cleavage site were strongly affected by substitutions in flanking genomic regions altering predicted transient RNA structures. Introduction of H5-like cleavage site sequences and structures into an H6 hemagglutinin resulted in MBCS-yielding insertions never observed before in H6 viruses. Our results demonstrate that nucleotide insertions that underlie H5 HPAIV emergence result from a previously unknown RNA-structure-driven diversity-generating mechanism, which could be shared with other RNA viruses.\nQuantifying the daily harvest of fermentation products from the human gut microbiota\nAuthors: Arnoldini, M.; Sharma, R.; Moresi, C.; Chure, G.; Chabbey, J.; Slack, E.; Cremer, J.\nScore: 31.7, Published: 2024-01-06 DOI: 10.1101/2024.01.05.573977\nFermentation products released by the gut microbiota provide energy and important regulatory functions for the host. Yet, little quantitative information is available on the metabolite exchange between the microbiota and the human host, and thus the effective doses of fermentation products. Here, we introduce an integrative framework combining experimental characterization of major gut bacteria with a quantitative analysis of human digestive physiology to put numbers on this exchange and its dependence on diet and microbiota composition. From the carbohydrates fueling microbiota growth, we find most carbon ends up in fermentation products which are largely utilized by the host. This harvest varies strongly with diet, from between 100-700 mmol/day within the US population to up to 1300 for the Hadza people of Tanzania. Accordingly, fermentation products cover between 1.8% and 12.1% of the daily energy demand of human hosts, substantially less than the 21% estimated for laboratory mice. In contrast, microbiota composition has little impact on the total daily harvest but determines the harvest of specific fermentation products. Butyrate, known for promoting epithelial health, shows the biggest variation. Our framework thus identifies and quantifies major factors driving the metabolic and signaling interactions between microbiota and host, crucial to mechanistically dissect the role of the microbiota in health and disease.\nCharacterization and genomic analysis of the Lyme disease spirochete bacteriophage phiBB-1\nAuthors: Faith, D. R.; Kinnersley, M.; Brooks, D. M.; Drecktrah, D.; Hall, L. S.; Luo, E.; Santiago-Frangos, A.; Wachter, J.; Samuels, S.; Secor, P. R.\nScore: 19.0, Published: 2024-01-13 DOI: 10.1101/2024.01.08.574763\nLyme disease is a tick-borne infection caused by the spirochete Borrelia (Borreliella) burgdorferi. Borrelia species have highly fragmented genomes composed of a linear chromosome and a constellation of linear and circular plasmids that encode diverse outer membrane lipoproteins, which facilitate movement of the spirochete between its tick vector and a vertebrate host in an enzootic cycle. The B. burgdorferi genome shows evidence of horizontal transfer between strains, but the mechanisms remain poorly defined. Almost all Lyme disease spirochetes are infected by 32-kp circular plasmid (cp32) prophages that undergo lytic replication and produce infectious virions called phiBB-1. In the laboratory, phiBB-1 transduces cp32s and shuttle vectors between spirochetes. However, the extent that phiBB-1 participates in horizontal gene transfer between Lyme disease spirochetes is not known. Here, we use proteomics and long-read sequencing to characterize phiBB-1 virions and the genetic material they package. Our studies reveal that phiBB-1 packages linear cp32s via a headful mechanism. We identify the cp32 pac region and show that plasmids containing the cp32 pac region are preferentially packaged into phiBB-1 virions. Additionally, we find phiBB-1 packages fragments of the linear chromosome and other plasmids including lp54, cp26, and others. Furthermore, sequencing of phiBB-1 packaged DNA allowed us to resolve the covalently closed hairpin telomeres for the linear B. burgdorferi chromosome and most linear plasmids in strain CA-11.2A. Collectively, our results shed light on the biology of the ubiquitous phiBB-1 phage and further implicates phiBB-1 in the generalized transduction of diverse genes between spirochetes and the maintenance of genetic diversity in Lyme disease spirochetes.\nBroadly conserved FlgV controls flagellar assembly and Borrelia burgdorferi dissemination in mice\nAuthors: Zamba-Campero, M.; Soliman, D.; Yu, H.; Lasseter, A. G.; Chang, Y.-Y.; Liu, J.; Aravind, L.; Jewett, M. W.; Storz, G.; Adams, P. P.\nScore: 18.8, Published: 2024-01-09 DOI: 10.1101/2024.01.09.574855\nFlagella propel pathogens through their environments yet are expensive to synthesize and are immunogenic. Thus, complex hierarchical regulatory networks control flagellar gene expression. Spirochetes are highly motile bacteria, but peculiarly in the Lyme spirochete Borrelia burgdorferi, the archetypal flagellar regulator {sigma}28 is absent. We rediscovered gene bb0268 in B. burgdorferi as flgV, a broadly-conserved gene in the flagellar superoperon alongside {sigma}28 in many Spirochaetes, Firmicutes and other phyla, with distant homologs in Epsilonproteobacteria. We found that B. burgdorferi FlgV is localized within flagellar motors. B. burgdorferi lacking flgV construct fewer and shorter flagellar filaments and are defective in cell division and motility. During the enzootic cycle, B. burgdorferi lacking flgV survive and replicate in Ixodes ticks but are attenuated for dissemination and infection in mice. Our work defines infection timepoints when spirochete motility is most crucial and implicates FlgV as a broadly distributed structural flagellar component that modulates flagellar assembly.\nRobust control of replication initiation in the absence of DnaA-ATP {leftrightharpoons} DnaA-ADP regulatory elements in Escherichia coli\nAuthors: Boesen, T.; Charbon, G.; Fu, H.; Jensen, C.; Sandler, M.; Li, D.; Jun, S.; Lobner-Olesen, A.\nScore: 19.2, Published: 2024-01-11 DOI: 10.1101/2022.09.08.507175\nInvestigating a long-standing conceptual question in bacterial physiology, we examine why DnaA, the bacterial master replication initiator protein, exists in both ATP and ADP forms, despite only the ATP form being essential for initiation. We engineered the {Delta}4 Escherichia coli strain, devoid of all known external elements facilitating the DnaA-ATP/ADP conversion, and found that these cells display nearly wild-type behaviors under non-overlapping replication cycles. However, during rapid growth with overlapping cycles, {Delta}4 cells exhibit initiation instability. This aligns with our model predictions, suggesting that the intrinsic ATPase activity of DnaA alone is sufficient for robust initiation control in E. coli and the DnaA-ATP/ADP conversion regulatory elements extend the robustness to multifork replication, indicating an evolutionary adaptation. Moreover, our experiments revealed constant DnaA concentrations during steady-state cell elongation in both wild-type and {Delta}4 cells. These insights not only advance our understanding of bacterial cell-cycle regulation and DnaA, but also highlight a fundamental divergence from eukaryotic cell-cycle controls, emphasizing protein copy-number sensing in bacteria versus programmed protein concentration oscillations in eukaryotes.\nVariant-specific interactions at the plasma membrane: Heparan sulfate's impact on SARS-CoV-2 binding kinetics\nAuthors: Conca, D. V.; Bano, F.; von Wiren, J.; Scherrer, L.; Svirelis, J.; Thorsteinsson, K.; Dahlin, A.; Bally, M.\nScore: 14.6, Published: 2024-01-10 DOI: 10.1101/2024.01.10.574981\nThe worldwide spread of SARS-CoV-2 has been characterised by the emergence of several variants of concern (VOCs) presenting an increasing number of mutations in the viral genome. The spike glycoprotein, responsible for engaging the viral receptor ACE2, exhibits the highest density of mutations, suggesting an ongoing evolution to optimize viral entry. However, previous studies focussed on isolated molecular interactions, neglecting the intricate composition of the plasma membrane and the interplay between viral attachment factors. Our study explores the role of avidity and of the complexity of the plasma membrane composition in modulating the virus-host binding kinetics during the early stages of viral entry for the original Wuhan strain and three VOCs: Omicron BA.1, Delta, and Alpha. We employ fluorescent liposomes decorated with spike from several VOCs as virion mimics in single-particle tracking studies on native supported lipid bilayers derived from pulmonary Calu-3 cells. Our findings reveal an increase in the affinity of the multivalent bond to the cell surface for Omicron driven by an increased association rate. We show that heparan sulfate (HS), a sulfated glycosaminoglycan commonly expressed on cells plasma membrane, plays a central role in modulating the interaction with the cell surface and we observe a shift in its role from screening the interaction with ACE2 in early VOCs to an important binding factor for Omicron. This is caused by a [~]10-fold increase in Omicrons affinity to HS compared to the original Wuhan strain, as shown using atomic force microscopy-based single-molecule force spectroscopy. Our results show the importance of coreceptors, particularly HS, and membrane complexity in the modulation of the attachment in SARS-CoV-2 VOCs. We highlight a transition in the variants attachment strategy towards the use of HS as an initial docking site, which likely plays a role in shaping Omicrons tropism towards infection of the upper airways, milder symptoms, and higher transmissibility.\nAMAPEC: accurate antimicrobial activity prediction for fungal effector proteins\nAuthors: Mesny, F.; Thomma, B. P.\nScore: 56.9, Published: 2024-01-04 DOI: 10.1101/2024.01.04.574150\nFungi typically occur in environments where numerous and diverse other microbes occur as well, often resulting in fierce competition for nutrients and habitat. To support fungal fitness in these environments, they evolved various mechanisms that mediate direct antagonism towards niche competitors. Among these, the secretion of proteins with antimicrobial activities has been reported in fungi with diverse lifestyles. Recently, several plant-associated fungi were shown to rely on the secretion of antimicrobial effector proteins to antagonize certain members of plant hosts microbiota and to successfully colonize plant tissues. Some of these effectors do not share homology with known antimicrobials and represent novel antibiotics. Accordingly, the occurrence and conservation of proteinaceous antimicrobials throughout the fungal tree of life remains enigmatic. Here we present a computational approach to annotate candidate antimicrobial effectors in fungal secretomes based on protein physicochemical properties. After curating a set of proteins that were experimentally verified to display antimicrobial activity and a set of proteins that lack such activity, we trained a machine learning classifier on properties of protein sequences and predicted structures. This predictor performs particularly well on fungal proteins (R2=0.89) according to our validations and is delivered as a software package named AMAPEC, dedicated to antimicrobial activity prediction for effector candidates. We subsequently used this novel software to predict antimicrobial effector catalogs in three phylogenetically distant fungi with distinct lifestyles, revealing relatively large catalogs of candidate antimicrobials for each of the three fungi, and suggesting a broad occurrence of such proteins throughout the fungal kingdom. Thus, AMAPEC is a unique method to uncover antimicrobials in fungal secretomes that are often sparsely functionally annotated, and may assist biological interpretations during omic analyses. It is freely available at https://github.com/fantin-mesny/amapec.\n",
  "wordCount" : "2479",
  "inLanguage": "en",
  "datePublished": "2024-01-14T10:38:40Z",
  "dateModified": "2024-01-14T10:38:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 14, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574008">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574008" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574008">
        <p class="paperTitle">Lethal Infection of Human ACE2-Transgenic Mice Caused by SARS-CoV-2-related Pangolin Coronavirus GX_P2V(short_3UTR)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574008" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574008" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.</p>
        <p class="info">Score: 1787.1, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574008' target='https://doi.org/10.1101/2024.01.03.574008'> 10.1101/2024.01.03.574008</a></p>
        <p class="abstract">SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) can cause 100% mortality in human ACE2-transgenic mice, potentially attributable to late-stage brain infection. This underscores a spillover risk of GX_P2V into humans and provides a unique model for understanding the pathogenic mechanisms of SARS-CoV-2-related viruses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574420">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574420" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574420">
        <p class="paperTitle">Rapid Emergence and Evolution of SARS-CoV-2 Variants in Advanced HIV Infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574420" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574420" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ko, S. H.; Radecki, P.; Belinky, F.; Bhiman, J.; Meiring, S.; Kleynhans, J.; Amoako, D.; Guerra Canedo, V.; Lucas, M.; Kekana, D.; Martinson, N.; Lebina, L.; Everatt, J.; Tempia, S.; Bylund, T.; Rawi, R.; Kwong, P.; Wolter, N.; von Gottberg, A.; Cohen, C.; Boritz, E. A.</p>
        <p class="info">Score: 77.6, Published: 2024-01-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574420' target='https://doi.org/10.1101/2024.01.05.574420'> 10.1101/2024.01.05.574420</a></p>
        <p class="abstract">Previous studies have linked the evolution of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) genetic variants to persistent infections in people with immunocompromising conditions1-4, but the evolutionary processes underlying these observations are incompletely understood. Here we used high-throughput, single-genome amplification and sequencing (HT-SGS) to obtain up to [~]103 SARS-CoV-2 spike gene sequences in each of 184 respiratory samples from 22 people with HIV (PWH) and 25 people without HIV (PWOH). Twelve of 22 PWH had advanced HIV infection, defined by peripheral blood CD4 T cell counts (i.e., CD4 counts) &lt;200 cells/L. In PWOH and PWH with CD4 counts [&amp;ge;]200 cells/L, most single-genome spike sequences in each person matched one haplotype that predominated throughout the infection. By contrast, people with advanced HIV showed elevated intra-host spike diversity with a median of 46 haplotypes per person (IQR 14-114). Higher intra-host spike diversity immediately after COVID-19 symptom onset predicted longer SARS-CoV-2 RNA shedding among PWH, and intra-host spike diversity at this timepoint was significantly higher in people with advanced HIV than in PWOH. Composition of spike sequence populations in people with advanced HIV fluctuated rapidly over time, with founder sequences often replaced by groups of new haplotypes. These population-level changes were associated with a high total burden of intra-host mutations and positive selection at functionally important residues. In several cases, delayed emergence of detectable serum binding to spike was associated with positive selection for presumptive antibody-escape mutations. Taken together, our findings show remarkable intra-host genetic diversity of SARS-CoV-2 in advanced HIV infection and suggest that adaptive intra-host SARS-CoV-2 evolution in this setting may contribute to the emergence of new variants of concern (VOCs).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.574642">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.574642" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.574642">
        <p class="paperTitle">Preferential apical infection of intestinal cell monolayers by SARS-CoV-2 is associated with damage to cellular barrier integrity: Implications for the physiopathology of COVID-19</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.574642" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.574642" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Clemence, G.; Jeffrey, A.; Jonatane, A.; Mege, J.-L.; Omar Osman, I.; Devaux, C. A.</p>
        <p class="info">Score: 58.1, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.574642' target='https://doi.org/10.1101/2024.01.08.574642'> 10.1101/2024.01.08.574642</a></p>
        <p class="abstract">SARS-CoV-2 can infect different organs, including the intestine. In Caco-2 intestinal cell line, SARS-CoV-2 modulates the ACE2 receptor expression and affects the expression of molecules involved in intercellular junctions. To further explore the possibility that the intestinal epithelium serves as an alternative infection route for SARS-CoV-2, we used a model of polarised intestinal cell monolayers grown on the polycarbonate membrane of Transwell inserts, inoculated with the virus either in the upper or lower chamber of culture. In both polarised Caco-2 cell monolayers and co-culture Caco-2/HT29 cell monolayer, apical SARS-CoV-2 inoculation was found to be much more effective in establishing infection than basolateral inoculation. In addition, apical SARS-CoV-2 infection triggers monolayer degeneration, as shown by histological examination, measurement of trans-epithelial electronic resistance, and cell adhesion molecule expression. During this process, the infectious viruses reach the lower chamber, suggesting either a transcytosis mechanism from the apical side to the basolateral side of cells, a paracellular trafficking of the virus after damage to intercellular junctions in the epithelial barrier, or both. Taken together, these data highlight a preferential tropism of SARS-CoV-2 for the apical side of the human intestinal tract and suggests that infection via the intestinal lumen leads to a systemic infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.574333">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.574333" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.574333">
        <p class="paperTitle">Transient RNA structures underlie highly pathogenic avian influenza virus genesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.574333" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.574333" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Funk, M.; Spronken, M. I.; Bestebroer, T. M.; de Bruin, A. C. M.; Gultyaev, A. P.; Fouchier, R. A. M.; te Velthuis, A. J. W.; Richard, M.</p>
        <p class="info">Score: 30.7, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.574333' target='https://doi.org/10.1101/2024.01.11.574333'> 10.1101/2024.01.11.574333</a></p>
        <p class="abstract">Highly pathogenic avian influenza viruses (HPAIVs) cause severe disease and high fatality in poultry1. They emerge exclusively from H5 and H7 low pathogenic avian influenza viruses (LPAIVs)2. Although insertion of a furin-cleavable multibasic cleavage site (MBCS) in the hemagglutinin gene was identified decades ago as the genetic basis for LPAIV-to-HPAIV transition3,4, the exact mechanisms underlying said insertion have remained unknown. Here we used an innovative combination of bioinformatic models to predict RNA structures forming around the influenza virus RNA polymerase during replication, and circular sequencing5 to reliably detect nucleotide insertions. We show that transient H5 hemagglutinin RNA structures predicted to trap the polymerase on purine-rich sequences drive nucleotide insertions characteristic of MBCSs, providing the first strong empirical evidence of RNA structure involvement in MBCS acquisition. Insertion frequencies at the H5 cleavage site were strongly affected by substitutions in flanking genomic regions altering predicted transient RNA structures. Introduction of H5-like cleavage site sequences and structures into an H6 hemagglutinin resulted in MBCS-yielding insertions never observed before in H6 viruses. Our results demonstrate that nucleotide insertions that underlie H5 HPAIV emergence result from a previously unknown RNA-structure-driven diversity-generating mechanism, which could be shared with other RNA viruses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.573977">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.573977" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.573977">
        <p class="paperTitle">Quantifying the daily harvest of fermentation products from the human gut microbiota</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.573977" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.573977" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Arnoldini, M.; Sharma, R.; Moresi, C.; Chure, G.; Chabbey, J.; Slack, E.; Cremer, J.</p>
        <p class="info">Score: 31.7, Published: 2024-01-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.573977' target='https://doi.org/10.1101/2024.01.05.573977'> 10.1101/2024.01.05.573977</a></p>
        <p class="abstract">Fermentation products released by the gut microbiota provide energy and important regulatory functions for the host. Yet, little quantitative information is available on the metabolite exchange between the microbiota and the human host, and thus the effective doses of fermentation products. Here, we introduce an integrative framework combining experimental characterization of major gut bacteria with a quantitative analysis of human digestive physiology to put numbers on this exchange and its dependence on diet and microbiota composition. From the carbohydrates fueling microbiota growth, we find most carbon ends up in fermentation products which are largely utilized by the host. This harvest varies strongly with diet, from between 100-700 mmol/day within the US population to up to 1300 for the Hadza people of Tanzania. Accordingly, fermentation products cover between 1.8% and 12.1% of the daily energy demand of human hosts, substantially less than the 21% estimated for laboratory mice. In contrast, microbiota composition has little impact on the total daily harvest but determines the harvest of specific fermentation products. Butyrate, known for promoting epithelial health, shows the biggest variation. Our framework thus identifies and quantifies major factors driving the metabolic and signaling interactions between microbiota and host, crucial to mechanistically dissect the role of the microbiota in health and disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.574763">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.574763" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.574763">
        <p class="paperTitle">Characterization and genomic analysis of the Lyme disease spirochete bacteriophage phiBB-1</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.574763" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.574763" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Faith, D. R.; Kinnersley, M.; Brooks, D. M.; Drecktrah, D.; Hall, L. S.; Luo, E.; Santiago-Frangos, A.; Wachter, J.; Samuels, S.; Secor, P. R.</p>
        <p class="info">Score: 19.0, Published: 2024-01-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.574763' target='https://doi.org/10.1101/2024.01.08.574763'> 10.1101/2024.01.08.574763</a></p>
        <p class="abstract">Lyme disease is a tick-borne infection caused by the spirochete Borrelia (Borreliella) burgdorferi. Borrelia species have highly fragmented genomes composed of a linear chromosome and a constellation of linear and circular plasmids that encode diverse outer membrane lipoproteins, which facilitate movement of the spirochete between its tick vector and a vertebrate host in an enzootic cycle. The B. burgdorferi genome shows evidence of horizontal transfer between strains, but the mechanisms remain poorly defined. Almost all Lyme disease spirochetes are infected by 32-kp circular plasmid (cp32) prophages that undergo lytic replication and produce infectious virions called phiBB-1. In the laboratory, phiBB-1 transduces cp32s and shuttle vectors between spirochetes. However, the extent that phiBB-1 participates in horizontal gene transfer between Lyme disease spirochetes is not known. Here, we use proteomics and long-read sequencing to characterize phiBB-1 virions and the genetic material they package. Our studies reveal that phiBB-1 packages linear cp32s via a headful mechanism. We identify the cp32 pac region and show that plasmids containing the cp32 pac region are preferentially packaged into phiBB-1 virions. Additionally, we find phiBB-1 packages fragments of the linear chromosome and other plasmids including lp54, cp26, and others. Furthermore, sequencing of phiBB-1 packaged DNA allowed us to resolve the covalently closed hairpin telomeres for the linear B. burgdorferi chromosome and most linear plasmids in strain CA-11.2A. Collectively, our results shed light on the biology of the ubiquitous phiBB-1 phage and further implicates phiBB-1 in the generalized transduction of diverse genes between spirochetes and the maintenance of genetic diversity in Lyme disease spirochetes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.574855">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.574855" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.574855">
        <p class="paperTitle">Broadly conserved FlgV controls flagellar assembly and Borrelia burgdorferi dissemination in mice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.574855" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.574855" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zamba-Campero, M.; Soliman, D.; Yu, H.; Lasseter, A. G.; Chang, Y.-Y.; Liu, J.; Aravind, L.; Jewett, M. W.; Storz, G.; Adams, P. P.</p>
        <p class="info">Score: 18.8, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.574855' target='https://doi.org/10.1101/2024.01.09.574855'> 10.1101/2024.01.09.574855</a></p>
        <p class="abstract">Flagella propel pathogens through their environments yet are expensive to synthesize and are immunogenic. Thus, complex hierarchical regulatory networks control flagellar gene expression. Spirochetes are highly motile bacteria, but peculiarly in the Lyme spirochete Borrelia burgdorferi, the archetypal flagellar regulator {sigma}28 is absent. We rediscovered gene bb0268 in B. burgdorferi as flgV, a broadly-conserved gene in the flagellar superoperon alongside {sigma}28 in many Spirochaetes, Firmicutes and other phyla, with distant homologs in Epsilonproteobacteria. We found that B. burgdorferi FlgV is localized within flagellar motors. B. burgdorferi lacking flgV construct fewer and shorter flagellar filaments and are defective in cell division and motility. During the enzootic cycle, B. burgdorferi lacking flgV survive and replicate in Ixodes ticks but are attenuated for dissemination and infection in mice. Our work defines infection timepoints when spirochete motility is most crucial and implicates FlgV as a broadly distributed structural flagellar component that modulates flagellar assembly.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.09.08.507175">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.09.08.507175" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.09.08.507175">
        <p class="paperTitle">Robust control of replication initiation in the absence of DnaA-ATP {leftrightharpoons} DnaA-ADP regulatory elements in Escherichia coli</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.09.08.507175" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.09.08.507175" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Boesen, T.; Charbon, G.; Fu, H.; Jensen, C.; Sandler, M.; Li, D.; Jun, S.; Lobner-Olesen, A.</p>
        <p class="info">Score: 19.2, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.09.08.507175' target='https://doi.org/10.1101/2022.09.08.507175'> 10.1101/2022.09.08.507175</a></p>
        <p class="abstract">Investigating a long-standing conceptual question in bacterial physiology, we examine why DnaA, the bacterial master replication initiator protein, exists in both ATP and ADP forms, despite only the ATP form being essential for initiation. We engineered the {Delta}4 Escherichia coli strain, devoid of all known external elements facilitating the DnaA-ATP/ADP conversion, and found that these cells display nearly wild-type behaviors under non-overlapping replication cycles. However, during rapid growth with overlapping cycles, {Delta}4 cells exhibit initiation instability. This aligns with our model predictions, suggesting that the intrinsic ATPase activity of DnaA alone is sufficient for robust initiation control in E. coli and the DnaA-ATP/ADP conversion regulatory elements extend the robustness to multifork replication, indicating an evolutionary adaptation. Moreover, our experiments revealed constant DnaA concentrations during steady-state cell elongation in both wild-type and {Delta}4 cells. These insights not only advance our understanding of bacterial cell-cycle regulation and DnaA, but also highlight a fundamental divergence from eukaryotic cell-cycle controls, emphasizing protein copy-number sensing in bacteria versus programmed protein concentration oscillations in eukaryotes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.574981">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.574981" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.574981">
        <p class="paperTitle">Variant-specific interactions at the plasma membrane: Heparan sulfate&#39;s impact on SARS-CoV-2 binding kinetics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.574981" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.574981" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Conca, D. V.; Bano, F.; von Wiren, J.; Scherrer, L.; Svirelis, J.; Thorsteinsson, K.; Dahlin, A.; Bally, M.</p>
        <p class="info">Score: 14.6, Published: 2024-01-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.574981' target='https://doi.org/10.1101/2024.01.10.574981'> 10.1101/2024.01.10.574981</a></p>
        <p class="abstract">The worldwide spread of SARS-CoV-2 has been characterised by the emergence of several variants of concern (VOCs) presenting an increasing number of mutations in the viral genome. The spike glycoprotein, responsible for engaging the viral receptor ACE2, exhibits the highest density of mutations, suggesting an ongoing evolution to optimize viral entry. However, previous studies focussed on isolated molecular interactions, neglecting the intricate composition of the plasma membrane and the interplay between viral attachment factors. Our study explores the role of avidity and of the complexity of the plasma membrane composition in modulating the virus-host binding kinetics during the early stages of viral entry for the original Wuhan strain and three VOCs: Omicron BA.1, Delta, and Alpha. We employ fluorescent liposomes decorated with spike from several VOCs as virion mimics in single-particle tracking studies on native supported lipid bilayers derived from pulmonary Calu-3 cells. Our findings reveal an increase in the affinity of the multivalent bond to the cell surface for Omicron driven by an increased association rate. We show that heparan sulfate (HS), a sulfated glycosaminoglycan commonly expressed on cells plasma membrane, plays a central role in modulating the interaction with the cell surface and we observe a shift in its role from screening the interaction with ACE2 in early VOCs to an important binding factor for Omicron. This is caused by a [~]10-fold increase in Omicrons affinity to HS compared to the original Wuhan strain, as shown using atomic force microscopy-based single-molecule force spectroscopy. Our results show the importance of coreceptors, particularly HS, and membrane complexity in the modulation of the attachment in SARS-CoV-2 VOCs. We highlight a transition in the variants attachment strategy towards the use of HS as an initial docking site, which likely plays a role in shaping Omicrons tropism towards infection of the upper airways, milder symptoms, and higher transmissibility.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.574150">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.574150" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.574150">
        <p class="paperTitle">AMAPEC: accurate antimicrobial activity prediction for fungal effector proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.574150" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.574150" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mesny, F.; Thomma, B. P.</p>
        <p class="info">Score: 56.9, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.574150' target='https://doi.org/10.1101/2024.01.04.574150'> 10.1101/2024.01.04.574150</a></p>
        <p class="abstract">Fungi typically occur in environments where numerous and diverse other microbes occur as well, often resulting in fierce competition for nutrients and habitat. To support fungal fitness in these environments, they evolved various mechanisms that mediate direct antagonism towards niche competitors. Among these, the secretion of proteins with antimicrobial activities has been reported in fungi with diverse lifestyles. Recently, several plant-associated fungi were shown to rely on the secretion of antimicrobial effector proteins to antagonize certain members of plant hosts microbiota and to successfully colonize plant tissues. Some of these effectors do not share homology with known antimicrobials and represent novel antibiotics. Accordingly, the occurrence and conservation of proteinaceous antimicrobials throughout the fungal tree of life remains enigmatic. Here we present a computational approach to annotate candidate antimicrobial effectors in fungal secretomes based on protein physicochemical properties. After curating a set of proteins that were experimentally verified to display antimicrobial activity and a set of proteins that lack such activity, we trained a machine learning classifier on properties of protein sequences and predicted structures. This predictor performs particularly well on fungal proteins (R2=0.89) according to our validations and is delivered as a software package named AMAPEC, dedicated to antimicrobial activity prediction for effector candidates. We subsequently used this novel software to predict antimicrobial effector catalogs in three phylogenetically distant fungi with distinct lifestyles, revealing relatively large catalogs of candidate antimicrobials for each of the three fungi, and suggesting a broad occurrence of such proteins throughout the fungal kingdom. Thus, AMAPEC is a unique method to uncover antimicrobials in fungal secretomes that are often sparsely functionally annotated, and may assist biological interpretations during omic analyses. It is freely available at https://github.com/fantin-mesny/amapec.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
