<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="SARS-CoV-2 Neutralizing Antibodies Following a Second BA.5 Bivalent Booster
Authors: Wang, Q.; Bowen, A.; Ho, J.; Zhang, R.; Valdez, R.; Stoneman, E.; Gordon, A.; Lihong, L.; Ho, D. D.
Score: 339.0, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553148
Bivalent COVID-19 mRNA vaccines expressing both the ancestral D614G and Omicron BA.5 spike proteins were introduced in August 2022 with the goal of broadening immunity to emerging SARS-CoV-2 Omicron subvariants. Subsequent studies on bivalent boosters found neutralizing antibody responses similar to boosters with the original monovalent vaccine, likely the result of immunological imprinting.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="SARS-CoV-2 Neutralizing Antibodies Following a Second BA.5 Bivalent Booster
Authors: Wang, Q.; Bowen, A.; Ho, J.; Zhang, R.; Valdez, R.; Stoneman, E.; Gordon, A.; Lihong, L.; Ho, D. D.
Score: 339.0, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553148
Bivalent COVID-19 mRNA vaccines expressing both the ancestral D614G and Omicron BA.5 spike proteins were introduced in August 2022 with the goal of broadening immunity to emerging SARS-CoV-2 Omicron subvariants. Subsequent studies on bivalent boosters found neutralizing antibody responses similar to boosters with the original monovalent vaccine, likely the result of immunological imprinting." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-23T10:37:56+00:00" />
<meta property="article:modified_time" content="2023-08-23T10:37:56+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="SARS-CoV-2 Neutralizing Antibodies Following a Second BA.5 Bivalent Booster
Authors: Wang, Q.; Bowen, A.; Ho, J.; Zhang, R.; Valdez, R.; Stoneman, E.; Gordon, A.; Lihong, L.; Ho, D. D.
Score: 339.0, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553148
Bivalent COVID-19 mRNA vaccines expressing both the ancestral D614G and Omicron BA.5 spike proteins were introduced in August 2022 with the goal of broadening immunity to emerging SARS-CoV-2 Omicron subvariants. Subsequent studies on bivalent boosters found neutralizing antibody responses similar to boosters with the original monovalent vaccine, likely the result of immunological imprinting."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "SARS-CoV-2 Neutralizing Antibodies Following a Second BA.5 Bivalent Booster\nAuthors: Wang, Q.; Bowen, A.; Ho, J.; Zhang, R.; Valdez, R.; Stoneman, E.; Gordon, A.; Lihong, L.; Ho, D. D.\nScore: 339.0, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553148\nBivalent COVID-19 mRNA vaccines expressing both the ancestral D614G and Omicron BA.5 spike proteins were introduced in August 2022 with the goal of broadening immunity to emerging SARS-CoV-2 Omicron subvariants. Subsequent studies on bivalent boosters found neutralizing antibody responses similar to boosters with the original monovalent vaccine, likely the result of immunological imprinting.",
  "keywords": [
    
  ],
  "articleBody": " SARS-CoV-2 Neutralizing Antibodies Following a Second BA.5 Bivalent Booster\nAuthors: Wang, Q.; Bowen, A.; Ho, J.; Zhang, R.; Valdez, R.; Stoneman, E.; Gordon, A.; Lihong, L.; Ho, D. D.\nScore: 339.0, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553148\nBivalent COVID-19 mRNA vaccines expressing both the ancestral D614G and Omicron BA.5 spike proteins were introduced in August 2022 with the goal of broadening immunity to emerging SARS-CoV-2 Omicron subvariants. Subsequent studies on bivalent boosters found neutralizing antibody responses similar to boosters with the original monovalent vaccine, likely the result of immunological imprinting. Guidelines allow for administration of a second bivalent booster in high-risk groups, but it remains unknown whether this would broaden antibody responses. To address this question, we assessed longitudinal serum SARS-CoV-2-neutralizing titers in 18 elderly immunocompetent individuals (mean age 69) following a fourth monovalent booster and two BA.5 bivalent booster vaccines using pseudovirus neutralization assays against D614G, Omicron BA.5, and Omicron XBB.1.5. There was a small but significant increase in peak neutralizing antibody responses against Omicron BA.5 and XBB.1.5 following the first bivalent booster, but no significant increase in peak titers following the second bivalent booster. Omicron-specific neutralizing titers remained low after both doses of the BA.5 bivalent booster. Our results suggest that a second dose of the BA.5 bivalent booster is not sufficient to broaden antibody responses and to overcome immunological imprinting. A monovalent vaccine targeting only the spike of the recently dominant SARS-CoV-2 may mitigate the \"back boosting\" associated with the \"original antigenic sin.\"\nTemperate gut phages are prevalent, diverse, and predominantly inactive\nAuthors: Dahlman, S.; Avellaneda-Franco, L.; Kett, C.; Subedi, D.; Young, R. B.; Gould, J. A.; Rutten, E. L.; Gilliver, E. L.; Turkington, C. J.; Nezam-Abadi, N.; Grasis, J. A.; Lyras, D.; Edwards, R. A.; Forster, S. C.; Barr, J. J.\nScore: 58.1, Published: 2023-08-18 DOI: 10.1101/2023.08.17.553642\nLarge-scale metagenomic and data mining efforts have uncovered an expansive diversity of bacteriophages (phages) within the human gut1-3. These insights include broader phage populational dynamics such as temporal stability4, interindividual uniqueness5,6 and potential associations to specific disease states7,8. However, the functional understanding of phage-host interactions and their impacts within this complex ecosystem have been limited due to a lack of cultured isolates for experimental validation. Here we characterise 125 active prophages originating from 252 diverse human gut bacterial isolates using seven different induction conditions to substantially expand the experimentally validated temperate phage-host pairs originating from the human gut. Importantly, only 17% of computationally predicted prophages were induced with common induction agents and these exhibited distinct gene patterns compared to non-induced predictions. Active Bacteroidota prophages were among the most prevalent members of the gut virome, with extensive use of diversity generating retroelements and exhibiting broad host ranges. Moreover, active polylysogeny was present in 52% of studied gut lysogens and led to coordinated prophage induction across diverse conditions. This study represents a substantial expansion of experimentally validated gut prophages, providing key insights into their diversity and genetics, including a genetic pathway for prophage domestication and demonstration that differential induction was complex and influenced by divergent prophage integration sites. More broadly, it highlights the importance of experimental validation alongside genomic based computational prediction to enable further functional understanding of these commensal viruses within the human gut.\nImpact of age and sex on neuroinflammation following SARS-CoV-2 infection in a murine model\nAuthors: D Krishna, V.; Chang, A.; Korthas, H.; Var, S. R.; Low, W. C.; Li, L.; Cheeran, M. C.-J.\nScore: 80.9, Published: 2023-08-14 DOI: 10.1101/2023.08.11.552998\nSevere Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), the etiological agent for the worldwide COVID-19 pandemic, is known to infect people of all ages and both sexes. Senior populations have the greatest risk of severe disease, and sexual dimorphism in clinical outcomes has been reported in COVID-19. SARS-CoV-2 infection in humans can cause damage to multiple organ systems, including the brain. Neurological symptoms are widely observed in patients with COVID-19, with many survivors suffering from persistent neurological and cognitive impairment, potentially accelerating Alzheimers disease. The present study aims to investigate the impact of age and sex on the neuroinflammatory response to SARS-CoV-2 infection using a mouse model. Wild-type C57BL/6 mice were inoculated, by intranasal route, with SARS-CoV-2 lineage B.1.351 variant known to infect mice. Older animals and in particular males exhibited a significantly greater weight loss starting at 4 dpi. In addition, male animals exhibited higher viral RNA loads and higher titers of infectious virus in the lung, which was particularly evident in males at 16 months of age. Notably, no viral RNA was detected in the brains of infected mice, regardless of age or sex. Nevertheless, expression of IL-6, TNF-, and CCL-2 in the lung and brain was increased with viral infection. An unbiased brain RNA-seq/transcriptomic analysis showed that SARS-CoV-2 infection caused significant changes in gene expression profiles in the brain, with innate immunity, defense response to virus, cerebravascular and neuronal functions, as the major molecular networks affected. The data presented in this study show that SARS-CoV-2 infection triggers a neuroinflammatory response despite the lack of detectable virus in the brain. Age and sex have a modifying effect on this pathogenic process. Aberrant activation of innate immune response, disruption of blood-brain barrier and endothelial cell integrity, and supression of neuronal activity and axonogenesis underlie the impact of SARS-CoV-2 infection on the brain. Understanding the role of these affected pathways in SARS-CoV-2 pathogenesis helps identify appropriate points of therapeutic interventions to alleviate neurological dysfunction observed during COVID-19.\nAntiviral efficacy of the SARS-CoV-2 XBB breakthrough infection sera against Omicron subvariants including EG.5\nAuthors: Kaku, Y.; Kosugi, Y.; Uriu, K.; Ito, J.; Kuramochi, J.; Sadamasu, K.; Yoshimura, K.; Asakura, H.; Nagashima, M.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Sato, K.\nScore: 1041.8, Published: 2023-08-08 DOI: 10.1101/2023.08.08.552415\nAs of July 2023, EG.5.1 (a.k.a. XBB.1.9.2.5.1), a XBB subvariant bearing the S:Q52H and S:F456L substitutions, alongside the S:F486P substitution (Figure S1A), has rapidly spread in some countries. On July 19, 2023, the WHO classified EG.5 as a variant under monitoring. First, we showed that EG.5.1 exhibits a higher effective reproduction number compared with XBB.1.5, XBB.1.16, and its parental lineage (XBB.1.9.2), suggesting that EG.5.1 will spread globally and outcompete these XBB subvariants in the near future. We then addressed whether EG.5.1 evades from the antiviral effect of the humoral immunity induced by breakthrough infection (BTI) of XBB subvariants and performed a neutralization assay using XBB BTI sera. However, the 50% neutralization titer (NT50) of XBB BTI sera against EG.5.1 was comparable to those against XBB.1.5/1.9.2 and XBB.1.16. Moreover, the sensitivity of EG.5.1 to convalescent sera of XBB.1- and XBB.1.5-infected hamsters was similar to those of XBB.1.5/1.9 and XBB.1.16. These results suggest that the increased Re of EG.5.1 is attributed to neither increased infectivity nor immune evasion from XBB BTI, and the emergence and spread of EG.5 is driven by the other pressures. We previously demonstrated that Omicron BTI cannot efficiently induce antiviral humoral immunity against the variant infected. In fact, the NT50s of the BTI sera of Omicron BA.1, BA.2, and BA.5 against the variant infected were 3.0-, 2.2-, and 3.4-fold lower than that against the ancestral B.1.1 variant, respectively. However, strikingly, we found that the NT50 of the BTI sera of XBB1.5/1.9 and XBB.1.16 against the variant infected were 8.7- and 8.3-fold lower than that against the B.1.1 variant. These results suggest that XBB BTI cannot efficiently induce antiviral humoral immunity against XBB subvariants. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=147 SRC=\"FIGDIR/small/552415v1_figs1.gif\" ALT=\"Figure 1\"\u003e View larger version (39K): org.highwire.dtl.DTLVardef@f95376org.highwire.dtl.DTLVardef@d66fa8org.highwire.dtl.DTLVardef@3c8841org.highwire.dtl.DTLVardef@15824c_HPS_FORMAT_FIGEXP M_FIG O_FLOATNOFigure S1.C_FLOATNO Virological features of EG.5.1 and XBB BTI (A) Frequency of mutations of interest in the representative XBB sublineages. Only mutations with a frequency \u003e0.5 in at least one but not all the representative sublineages are shown. Note that the S proteins of XBB.1.5 and XBB.1.9.2 are identical. (B) Estimated epidemic dynamics of the representative XBB sublineages in countries where \u003e50 sequences of EG.5.1, XBB.1.5, XBB.1.9.2, and XBB.1.16 were detected from April 1, 2023 to July 13, 2023. Countries are ordered according to the number of detected sequences of EG.5.1. Line, posterior mean; ribbon, 95% Bayesian confidence interval. The dynamics for EG.5.1 is highlighted by a red arrowhead. (C) Estimated relative Re of the representative XBB sublineages in the six countries. The relative Re of XBB.1.5 is set to 1 (horizontal dashed line). Violin, posterior distribution; dot, posterior mean; line, 95% Bayesian confidence interval. (D) Lentivirus-based pseudovirus assay. HOS-ACE2-TMPRSS2 cells were infected with pseudoviruses bearing each S protein. The amount of input virus was normalized to the amount of HIV-1 p24 capsid protein. The percentage infectivity of XBB.1.5/1.9.2, XBB.1.5/1.9.2+Q52H, XBB.1.5/1.9.2+F456L, and EG.5.1 compared to that of XBB.1.5/1.9.2 are shown. The horizontal dash line indicates the mean value of the percentage infectivity of the XBB.1.5/1.9.2. Assays were performed in quadruplicate. The presented data are expressed as the average {+/-} SD. Each dot indicates the result of an individual replicate. (E-G) Neutralization assay. Assays were performed with pseudoviruses harboring the S proteins of B.1.1, BA.1, BA.2, BA.5, BQ.1.1, XBB.1, XBB.1.5/1.9.2, XBB.1.16, EG.5.1, XBB.1.5/1.9.2+Q52H, and XBB.1.5/1.9.2+F456L. The following sera were used: convalescent sera from fully vaccinated individuals who had been infected with XBB.1.5 (one 3-dose vaccinated. 1 donor in total), XBB.1.9 (one 3-dose vaccinated donor, one 4-dose vaccinated donor and one 5-dose vaccinated donor. 3 donors in total), and XBB.1.16 (one 2-dose vaccinated donor, two 3-dose vaccinated donors, and one 4-dose vaccinated donor. 4 donors in total) (E); sera from hamster infected with XBB.1 (left) or XBB.1.5 (right) (F); and convalescent sera from fully vaccinated individuals who had been infected with BA.1 (thirteen 2-dose vaccinated, 13 donors in total) (left)1, BA.2 (nine 2-dose vaccinated and four 3-dose vaccinated donors. 13 donors in total) (middle)19, and BA.5 (one 2-dose vaccinated, thirteen 3-dose vaccinated donors, and one 4-dose vaccinated. 15 donors in total) (right)19 (G). Each dot indicates the result of an individual replicate. Assays for each serum sample were performed in triplicate to determine the 50% neutralization titer (NT50). Each dot represents one NT50 value, and the geometric mean and 95% confidence interval are shown. The number in parenthesis indicates the mean of NT50 values. The horizontal dash line indicates the detection limit (120-fold). In D, statistically significant differences (**, P \u003c 0.001, ***, P \u003c 0.0001) versus XBB.1.5/1.9.2 were determined by two-sided Students t tests. Blue asterisks indicate decreased percentage of infectivity. In E and G, statistically significant differences versus B.1.1 were determined by two-sided Wilcoxon signed-rank tests. The fold change between B.1.1 and the variant indicated is shown in red. Background information on the convalescent donors is summarized in Table S1. In F, statistically significant differences (*, P \u003c 0.01, **, P \u003c 0.001) between B.1.1 and XBB.1 (left) or XBB.1.5/1.9.2 (right) were determined by two-sided Wilcoxon signed-rank tests and indicated with asterisks. Red asterisks indicate decreased NT50s. C_FIG\nVirological characteristics of the SARS-CoV-2 XBB.1.5 variant\nAuthors: Tamura, T.; Irie, T.; Deguchi, S.; Yajima, H.; Tsuda, M.; Nasser, H.; Mizuma, K.; Plianchaisuk, A.; Suzuki, S.; Uriu, K.; Begum, M. M.; Shimizu, R.; Jonathan, M.; Suzuki, R.; Kondo, T.; Ito, H.; Kamiyama, A.; Yoshimatsu, K.; Shofa, M.; Hashimoto, R.; Anraku, Y.; Kimura, K. T.; Kita, S.; Sasaki, J.; Sasaki-Tabata, K.; Maenaka, K.; Nao, N.; Wang, L.; Oda, Y.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Ikeda, T.; Saito, A.; Matsuno, K.; Ito, J.; Tanaka, S.; Sato, K.; Hashiguchi, T.; Takayama, K.; Fukuhara, T.\nScore: 35.7, Published: 2023-08-17 DOI: 10.1101/2023.08.16.553332\nCirculation of SARS-CoV-2 Omicron XBB has resulted in the emergence of XBB.1.5, a new Variant of Interest. Our phylogenetic analysis suggests that XBB.1.5 evolved from XBB.1 by acquiring the F486P spike (S) mutation, subsequent to the acquisition of a nonsense mutation in ORF8. Neutralization assays showed similar abilities of immune escape between XBB.1.5 and XBB.1. We determined the structural basis for the interaction between human ACE2 and the S protein of XBB.1.5, showing similar overall structures between the S proteins of XBB.1 and XBB.1.5. The intrinsic pathogenicity of XBB.1.5 in hamsters is lower than that of XBB.1. Importantly, we found that the ORF8 nonsense mutation of XBB.1.5 resulted in impairment of MHC expression. In vivo experiments using recombinant viruses revealed that the XBB.1.5 mutations are involved with reduced virulence of XBB.1.5. Together, these data suggest that the mutations in ORF8 and S could enhance spreading of XBB.1.5 in humans.\nNon-antibiotic pharmaceuticals exhibit toxicity against Escherichia coli at environmentallyrelevant concentrations with no evolution of cross-resistance to antibiotics\nAuthors: Hall, R. J.; Snaith, A. E.; Element, S.; Moran, R. A.; Smith, H.; Cummins, E. A.; Bottery, M. J.; Chowdhury, K. F.; Sareen, D.; Ahmad, I.; Blair, J. M. A.; Carter, L. J.; McNally, A.\nScore: 19.7, Published: 2023-08-22 DOI: 10.1101/2023.08.21.554069\nAntimicrobial resistance can arise in the natural environment via prolonged exposure to the effluent surrounding manufacturing facilities. These facilities also produce non-antibiotic pharmaceuticals, and the effect of these on the surrounding microbial communities is less clear; whether they have inherent toxicity, or whether long-term exposure might select for cross-resistance to antibiotics. To this end, we screened four non-antibiotic pharmaceuticals (acetaminophen, ibuprofen, propranolol, metformin) and titanium dioxide for toxicity against Escherichia coli K-12 MG1655 and conducted a 30 day selection experiment to assess the effect of long-term exposure. All compounds reduced the maximum optical density reached by E. coli at a range of concentrations including one of environmental relevance, with transcriptome analysis identifying upregulated genes related to stress response and multidrug efflux in response ibuprofen treatment. The non-antibiotic pharmaceuticals did not select for significant genetic changes following a 30 day exposure, and no evidence of selection for cross-resistance to antibiotics was observed for population evolved in the presence of ibuprofen in spite of the differential gene expression after exposure to this compound. This work suggests that these non-antibiotic pharmaceuticals, at environmental concentrations, do not select for cross-resistance to antibiotics in E. coli.\nDefense mechanism of a bacterial retron supramolecular assembly\nAuthors: Wang, Y.; Wang, C.; Guan, Z.; Cao, J.; Xu, J.; Wang, S.; Cui, Y.; Wang, Q.; Chen, Y.; Zhang, D.; Sun, M.; Tao, P.; Zou, T.\nScore: 10.0, Published: 2023-08-16 DOI: 10.1101/2023.08.16.553469\nRetrons are a class of multigene antiphage defense system typically consisting of a retron reverse transcriptase, a non-coding RNA, and a cognate effector. Although the triggers for several retron systems have been discovered recently, the full picture of how retron systems sense invading phages and mediate defense remains to be elucidated. Here, we focus on the retron Ec86 defense system and report its modes of activation and action. We identified a phage-encoded DNA cytosine methyltransferase (Dcm) as the trigger of the Ec86 system and show that Ec86 senses msDNA methylation and becomes activated. We further determined the structure of a tripartite retron Ec86 supramolecular assembly, which is primed for activation by Dcm, and demonstrated that the activated system confers defense through depletion of nucleoside derivatives. These findings emphasize the role of retrons being a second line of defense and highlight an emerging theme of anti-phage defense through supramolecular complex assemblies.\nSpace and epigenetic inheritance determine inter-individual differences in siderophore gene expression in bacterial colonies\nAuthors: Mridha, S.; Wechsler, T.; Kuemmerli, R.\nScore: 9.7, Published: 2023-08-21 DOI: 10.1101/2023.08.21.554085\nHeterogeneity in gene expression among cells in clonal groups is common in bacteria. Albeit ubiquitous, it often remains unclear what the sources of variation are and whether variation has functional significance. Here, we tracked the expression of genes involved in the synthesis of iron-chelating siderophores (pyoverdine and pyochelin) in individual cells of the bacterium Pseudomonas aeruginosa during colony growth on surfaces using time-lapse fluorescence microscopy, to explore potential sources and functions of cellular heterogeneity. Regarding sources, we found that the physical position of cells within the colony and epigenetic gene expression inheritance from mother to daughter cells significantly contributed to cellular heterogeneity. In contrast, cell pole age and cellular lifespan had no effect. Regarding functions, our results indicate that cells optimize their siderophore investment strategies (pyoverdine vs. pyochelin) along a gradient from the centre to the edge of the colony. Moreover, we found evidence that cell lineages with above-average siderophore investment increase the fitness of cell lineages with below-average investment through cooperative sharing of secreted siderophores. Altogether, our study highlights that single-cell experiments combined with automated image and cell-tracking analyses can identify sources of heterogeneity and yield adaptive explanations for gene expression variation among clonal bacterial cells.\nMother-Infant Gut Viruses and their Bacterial Hosts: Transmission Patterns and Dynamics during Pregnancy and Early Life\nAuthors: Garmaeva, S.; Sinha, T.; Gulyaeva, A.; Kuzub, N.; Spreckels, J. E.; Andreu-Sanchez, S.; Gacesa, R.; Vich Vila, A.; Brushett, S.; Kruk, M.; Dekens, J.; Sikkema, J.; Kuipers, F.; Shkoporov, A.; Hill, C.; Scherjon, S.; Wijmenga, C.; Fu, J.; Kurilshikov, A.; Zhernakova, A.\nScore: 9.6, Published: 2023-08-21 DOI: 10.1101/2023.08.21.554108\nEarly development of the gut ecosystem is crucial for lifelong health. While infant gut bacterial communities have been studied extensively, the infant gut virome remains under-explored. We longitudinally assessed the composition of gut viruses and their bacterial hosts in 322 total metagenomes and 205 metaviromes from 30 mothers during and after pregnancy and from their 32 infants during their first year of life. While the maternal gut virome composition remained stable during late pregnancy and after birth, the infant gut virome was dynamic in the first year of life and contained a higher abundance of active temperate phages compared to the maternal gut viromes. The infant gut virome composition was also influenced by infant feeding mode and place of delivery. Lastly, we provide evidence of viral-bacterial strains co-transmission from mothers to infants, demonstrating that infants acquire some of their virome from their mother's gut.\nMicrobial eukaryotic predation pressure and biomass at deep-sea hydrothermal vents: Implications for deep-sea carbon cycling\nAuthors: Hu, S. K.; Anderson, R. E.; Pachiadaki, M. G.; Edgcomb, V. P.; Serres, M. H.; Sylva, S. P.; German, C. R.; Seewald, J. S.; Lang, S. Q.; Huber, J. A.\nScore: 12.2, Published: 2023-08-11 DOI: 10.1101/2023.08.11.552852\nDeep-sea hydrothermal vent geochemistry shapes the foundation of the microbial food web by fueling chemolithoautotrophic microbial activity. Microbial eukaryotes (or protists) play a critical role in hydrothermal vent food webs as consumers, hosts of symbiotic bacteria, and as a nutritional source to higher trophic levels. We measured cell abundances and predation pressures of vent-associated microbial eukaryotes in low temperature diffuse hydrothermal fluids at the Von Damm and Piccard vent fields along the Mid-Cayman Rise in the Western Caribbean Sea. We present findings from experiments performed under in situ pressure that show higher cell abundances and grazing rates compared to those done at 1 atmosphere (shipboard ambient pressure); this trend was attributed to the impact of depressurization on cell integrity. A relationship between protistan grazing rate, prey cell abundance, and temperature of end member hydrothermal vent fluid was observed at both vent fields, regardless of experimental approach. The quantification of protistan biomass and grazing pressure shows that hydrothermally-fueled microbial food webs play a significant role in the broader deep-sea carbon budget by contributing to local carbon export and supply of nutrient resources to the deep ocean.\n",
  "wordCount" : "3142",
  "inLanguage": "en",
  "datePublished": "2023-08-23T10:37:56Z",
  "dateModified": "2023-08-23T10:37:56Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta"><span>updated on August 23, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.13.553148">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.13.553148" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.13.553148">
        <p class="paperTitle">SARS-CoV-2 Neutralizing Antibodies Following a Second BA.5 Bivalent Booster</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.13.553148" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.13.553148" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Q.; Bowen, A.; Ho, J.; Zhang, R.; Valdez, R.; Stoneman, E.; Gordon, A.; Lihong, L.; Ho, D. D.</p>
        <p class="info">Score: 339.0, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.13.553148' target='https://doi.org/10.1101/2023.08.13.553148'> 10.1101/2023.08.13.553148</a></p>
        <p class="abstract">Bivalent COVID-19 mRNA vaccines expressing both the ancestral D614G and Omicron BA.5 spike proteins were introduced in August 2022 with the goal of broadening immunity to emerging SARS-CoV-2 Omicron subvariants. Subsequent studies on bivalent boosters found neutralizing antibody responses similar to boosters with the original monovalent vaccine, likely the result of immunological imprinting. Guidelines allow for administration of a second bivalent booster in high-risk groups, but it remains unknown whether this would broaden antibody responses. To address this question, we assessed longitudinal serum SARS-CoV-2-neutralizing titers in 18 elderly immunocompetent individuals (mean age 69) following a fourth monovalent booster and two BA.5 bivalent booster vaccines using pseudovirus neutralization assays against D614G, Omicron BA.5, and Omicron XBB.1.5. There was a small but significant increase in peak neutralizing antibody responses against Omicron BA.5 and XBB.1.5 following the first bivalent booster, but no significant increase in peak titers following the second bivalent booster. Omicron-specific neutralizing titers remained low after both doses of the BA.5 bivalent booster. Our results suggest that a second dose of the BA.5 bivalent booster is not sufficient to broaden antibody responses and to overcome immunological imprinting. A monovalent vaccine targeting only the spike of the recently dominant SARS-CoV-2 may mitigate the &#34;back boosting&#34; associated with the &#34;original antigenic sin.&#34;</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.17.553642">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.17.553642" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.17.553642">
        <p class="paperTitle">Temperate gut phages are prevalent, diverse, and predominantly inactive</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.17.553642" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.17.553642" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dahlman, S.; Avellaneda-Franco, L.; Kett, C.; Subedi, D.; Young, R. B.; Gould, J. A.; Rutten, E. L.; Gilliver, E. L.; Turkington, C. J.; Nezam-Abadi, N.; Grasis, J. A.; Lyras, D.; Edwards, R. A.; Forster, S. C.; Barr, J. J.</p>
        <p class="info">Score: 58.1, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.17.553642' target='https://doi.org/10.1101/2023.08.17.553642'> 10.1101/2023.08.17.553642</a></p>
        <p class="abstract">Large-scale metagenomic and data mining efforts have uncovered an expansive diversity of bacteriophages (phages) within the human gut1-3. These insights include broader phage populational dynamics such as temporal stability4, interindividual uniqueness5,6 and potential associations to specific disease states7,8. However, the functional understanding of phage-host interactions and their impacts within this complex ecosystem have been limited due to a lack of cultured isolates for experimental validation. Here we characterise 125 active prophages originating from 252 diverse human gut bacterial isolates using seven different induction conditions to substantially expand the experimentally validated temperate phage-host pairs originating from the human gut. Importantly, only 17% of computationally predicted prophages were induced with common induction agents and these exhibited distinct gene patterns compared to non-induced predictions. Active Bacteroidota prophages were among the most prevalent members of the gut virome, with extensive use of diversity generating retroelements and exhibiting broad host ranges. Moreover, active polylysogeny was present in 52% of studied gut lysogens and led to coordinated prophage induction across diverse conditions. This study represents a substantial expansion of experimentally validated gut prophages, providing key insights into their diversity and genetics, including a genetic pathway for prophage domestication and demonstration that differential induction was complex and influenced by divergent prophage integration sites. More broadly, it highlights the importance of experimental validation alongside genomic based computational prediction to enable further functional understanding of these commensal viruses within the human gut.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.552998">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.552998" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.552998">
        <p class="paperTitle">Impact of age and sex on neuroinflammation following SARS-CoV-2 infection in a murine model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.552998" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.552998" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: D Krishna, V.; Chang, A.; Korthas, H.; Var, S. R.; Low, W. C.; Li, L.; Cheeran, M. C.-J.</p>
        <p class="info">Score: 80.9, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.552998' target='https://doi.org/10.1101/2023.08.11.552998'> 10.1101/2023.08.11.552998</a></p>
        <p class="abstract">Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), the etiological agent for the worldwide COVID-19 pandemic, is known to infect people of all ages and both sexes. Senior populations have the greatest risk of severe disease, and sexual dimorphism in clinical outcomes has been reported in COVID-19. SARS-CoV-2 infection in humans can cause damage to multiple organ systems, including the brain. Neurological symptoms are widely observed in patients with COVID-19, with many survivors suffering from persistent neurological and cognitive impairment, potentially accelerating Alzheimers disease. The present study aims to investigate the impact of age and sex on the neuroinflammatory response to SARS-CoV-2 infection using a mouse model. Wild-type C57BL/6 mice were inoculated, by intranasal route, with SARS-CoV-2 lineage B.1.351 variant known to infect mice. Older animals and in particular males exhibited a significantly greater weight loss starting at 4 dpi. In addition, male animals exhibited higher viral RNA loads and higher titers of infectious virus in the lung, which was particularly evident in males at 16 months of age. Notably, no viral RNA was detected in the brains of infected mice, regardless of age or sex. Nevertheless, expression of IL-6, TNF-, and CCL-2 in the lung and brain was increased with viral infection. An unbiased brain RNA-seq/transcriptomic analysis showed that SARS-CoV-2 infection caused significant changes in gene expression profiles in the brain, with innate immunity, defense response to virus, cerebravascular and neuronal functions, as the major molecular networks affected. The data presented in this study show that SARS-CoV-2 infection triggers a neuroinflammatory response despite the lack of detectable virus in the brain. Age and sex have a modifying effect on this pathogenic process. Aberrant activation of innate immune response, disruption of blood-brain barrier and endothelial cell integrity, and supression of neuronal activity and axonogenesis underlie the impact of SARS-CoV-2 infection on the brain. Understanding the role of these affected pathways in SARS-CoV-2 pathogenesis helps identify appropriate points of therapeutic interventions to alleviate neurological dysfunction observed during COVID-19.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552415">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552415" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552415">
        <p class="paperTitle">Antiviral efficacy of the SARS-CoV-2 XBB breakthrough infection sera against Omicron subvariants including EG.5</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552415" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552415" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kaku, Y.; Kosugi, Y.; Uriu, K.; Ito, J.; Kuramochi, J.; Sadamasu, K.; Yoshimura, K.; Asakura, H.; Nagashima, M.; The Genotype to Phenotype Japan (G2P-Japan) Consortium,  ; Sato, K.</p>
        <p class="info">Score: 1041.8, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552415' target='https://doi.org/10.1101/2023.08.08.552415'> 10.1101/2023.08.08.552415</a></p>
        <p class="abstract">As of July 2023, EG.5.1 (a.k.a. XBB.1.9.2.5.1), a XBB subvariant bearing the S:Q52H and S:F456L substitutions, alongside the S:F486P substitution (Figure S1A), has rapidly spread in some countries. On July 19, 2023, the WHO classified EG.5 as a variant under monitoring. First, we showed that EG.5.1 exhibits a higher effective reproduction number compared with XBB.1.5, XBB.1.16, and its parental lineage (XBB.1.9.2), suggesting that EG.5.1 will spread globally and outcompete these XBB subvariants in the near future. We then addressed whether EG.5.1 evades from the antiviral effect of the humoral immunity induced by breakthrough infection (BTI) of XBB subvariants and performed a neutralization assay using XBB BTI sera. However, the 50% neutralization titer (NT50) of XBB BTI sera against EG.5.1 was comparable to those against XBB.1.5/1.9.2 and XBB.1.16. Moreover, the sensitivity of EG.5.1 to convalescent sera of XBB.1- and XBB.1.5-infected hamsters was similar to those of XBB.1.5/1.9 and XBB.1.16. These results suggest that the increased Re of EG.5.1 is attributed to neither increased infectivity nor immune evasion from XBB BTI, and the emergence and spread of EG.5 is driven by the other pressures. We previously demonstrated that Omicron BTI cannot efficiently induce antiviral humoral immunity against the variant infected. In fact, the NT50s of the BTI sera of Omicron BA.1, BA.2, and BA.5 against the variant infected were 3.0-, 2.2-, and 3.4-fold lower than that against the ancestral B.1.1 variant, respectively. However, strikingly, we found that the NT50 of the BTI sera of XBB1.5/1.9 and XBB.1.16 against the variant infected were 8.7- and 8.3-fold lower than that against the B.1.1 variant. These results suggest that XBB BTI cannot efficiently induce antiviral humoral immunity against XBB subvariants.

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=147 SRC=&#34;FIGDIR/small/552415v1_figs1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (39K):
org.highwire.dtl.DTLVardef@f95376org.highwire.dtl.DTLVardef@d66fa8org.highwire.dtl.DTLVardef@3c8841org.highwire.dtl.DTLVardef@15824c_HPS_FORMAT_FIGEXP  M_FIG O_FLOATNOFigure S1.C_FLOATNO Virological features of EG.5.1 and XBB BTI (A) Frequency of mutations of interest in the representative XBB sublineages. Only mutations with a frequency &gt;0.5 in at least one but not all the representative sublineages are shown. Note that the S proteins of XBB.1.5 and XBB.1.9.2 are identical.

(B) Estimated epidemic dynamics of the representative XBB sublineages in countries where &gt;50 sequences of EG.5.1, XBB.1.5, XBB.1.9.2, and XBB.1.16 were detected from April 1, 2023 to July 13, 2023. Countries are ordered according to the number of detected sequences of EG.5.1. Line, posterior mean; ribbon, 95% Bayesian confidence interval. The dynamics for EG.5.1 is highlighted by a red arrowhead.

(C) Estimated relative Re of the representative XBB sublineages in the six countries. The relative Re of XBB.1.5 is set to 1 (horizontal dashed line). Violin, posterior distribution; dot, posterior mean; line, 95% Bayesian confidence interval.

(D) Lentivirus-based pseudovirus assay. HOS-ACE2-TMPRSS2 cells were infected with pseudoviruses bearing each S protein. The amount of input virus was normalized to the amount of HIV-1 p24 capsid protein. The percentage infectivity of XBB.1.5/1.9.2, XBB.1.5/1.9.2&#43;Q52H, XBB.1.5/1.9.2&#43;F456L, and EG.5.1 compared to that of XBB.1.5/1.9.2 are shown. The horizontal dash line indicates the mean value of the percentage infectivity of the XBB.1.5/1.9.2. Assays were performed in quadruplicate. The presented data are expressed as the average {&#43;/-} SD. Each dot indicates the result of an individual replicate.

(E-G) Neutralization assay. Assays were performed with pseudoviruses harboring the S proteins of B.1.1, BA.1, BA.2, BA.5, BQ.1.1, XBB.1, XBB.1.5/1.9.2, XBB.1.16, EG.5.1, XBB.1.5/1.9.2&#43;Q52H, and XBB.1.5/1.9.2&#43;F456L. The following sera were used: convalescent sera from fully vaccinated individuals who had been infected with XBB.1.5 (one 3-dose vaccinated. 1 donor in total), XBB.1.9 (one 3-dose vaccinated donor, one 4-dose vaccinated donor and one 5-dose vaccinated donor. 3 donors in total), and XBB.1.16 (one 2-dose vaccinated donor, two 3-dose vaccinated donors, and one 4-dose vaccinated donor. 4 donors in total) (E); sera from hamster infected with XBB.1 (left) or XBB.1.5 (right) (F); and convalescent sera from fully vaccinated individuals who had been infected with BA.1 (thirteen 2-dose vaccinated, 13 donors in total) (left)1, BA.2 (nine 2-dose vaccinated and four 3-dose vaccinated donors. 13 donors in total) (middle)19, and BA.5 (one 2-dose vaccinated, thirteen 3-dose vaccinated donors, and one 4-dose vaccinated. 15 donors in total) (right)19 (G). Each dot indicates the result of an individual replicate. Assays for each serum sample were performed in triplicate to determine the 50% neutralization titer (NT50). Each dot represents one NT50 value, and the geometric mean and 95% confidence interval are shown. The number in parenthesis indicates the mean of NT50 values. The horizontal dash line indicates the detection limit (120-fold).

In D, statistically significant differences (**, P &lt; 0.001, ***, P &lt; 0.0001) versus XBB.1.5/1.9.2 were determined by two-sided Students t tests. Blue asterisks indicate decreased percentage of infectivity.

In E and G, statistically significant differences versus B.1.1 were determined by two-sided Wilcoxon signed-rank tests. The fold change between B.1.1 and the variant indicated is shown in red. Background information on the convalescent donors is summarized in Table S1.

In F, statistically significant differences (*, P &lt; 0.01, **, P &lt; 0.001) between B.1.1 and XBB.1 (left) or XBB.1.5/1.9.2 (right) were determined by two-sided Wilcoxon signed-rank tests and indicated with asterisks. Red asterisks indicate decreased NT50s.

C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.553332">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.553332" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.553332">
        <p class="paperTitle">Virological characteristics of the SARS-CoV-2 XBB.1.5 variant</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.553332" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.553332" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tamura, T.; Irie, T.; Deguchi, S.; Yajima, H.; Tsuda, M.; Nasser, H.; Mizuma, K.; Plianchaisuk, A.; Suzuki, S.; Uriu, K.; Begum, M. M.; Shimizu, R.; Jonathan, M.; Suzuki, R.; Kondo, T.; Ito, H.; Kamiyama, A.; Yoshimatsu, K.; Shofa, M.; Hashimoto, R.; Anraku, Y.; Kimura, K. T.; Kita, S.; Sasaki, J.; Sasaki-Tabata, K.; Maenaka, K.; Nao, N.; Wang, L.; Oda, Y.; The Genotype to Phenotype Japan (G2P-Japan) Consortium,  ; Ikeda, T.; Saito, A.; Matsuno, K.; Ito, J.; Tanaka, S.; Sato, K.; Hashiguchi, T.; Takayama, K.; Fukuhara, T.</p>
        <p class="info">Score: 35.7, Published: 2023-08-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.553332' target='https://doi.org/10.1101/2023.08.16.553332'> 10.1101/2023.08.16.553332</a></p>
        <p class="abstract">Circulation of SARS-CoV-2 Omicron XBB has resulted in the emergence of XBB.1.5, a new Variant of Interest. Our phylogenetic analysis suggests that XBB.1.5 evolved from XBB.1 by acquiring the F486P spike (S) mutation, subsequent to the acquisition of a nonsense mutation in ORF8. Neutralization assays showed similar abilities of immune escape between XBB.1.5 and XBB.1. We determined the structural basis for the interaction between human ACE2 and the S protein of XBB.1.5, showing similar overall structures between the S proteins of XBB.1 and XBB.1.5. The intrinsic pathogenicity of XBB.1.5 in hamsters is lower than that of XBB.1. Importantly, we found that the ORF8 nonsense mutation of XBB.1.5 resulted in impairment of MHC expression. In vivo experiments using recombinant viruses revealed that the XBB.1.5 mutations are involved with reduced virulence of XBB.1.5. Together, these data suggest that the mutations in ORF8 and S could enhance spreading of XBB.1.5 in humans.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554069">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554069" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554069">
        <p class="paperTitle">Non-antibiotic pharmaceuticals exhibit toxicity against Escherichia coli at environmentallyrelevant concentrations with no evolution of cross-resistance to antibiotics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554069" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554069" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hall, R. J.; Snaith, A. E.; Element, S.; Moran, R. A.; Smith, H.; Cummins, E. A.; Bottery, M. J.; Chowdhury, K. F.; Sareen, D.; Ahmad, I.; Blair, J. M. A.; Carter, L. J.; McNally, A.</p>
        <p class="info">Score: 19.7, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554069' target='https://doi.org/10.1101/2023.08.21.554069'> 10.1101/2023.08.21.554069</a></p>
        <p class="abstract">Antimicrobial resistance can arise in the natural environment via prolonged exposure to the effluent surrounding manufacturing facilities. These facilities also produce non-antibiotic pharmaceuticals, and the effect of these on the surrounding microbial communities is less clear; whether they have inherent toxicity, or whether long-term exposure might select for cross-resistance to antibiotics. To this end, we screened four non-antibiotic pharmaceuticals (acetaminophen, ibuprofen, propranolol, metformin) and titanium dioxide for toxicity against Escherichia coli K-12 MG1655 and conducted a 30 day selection experiment to assess the effect of long-term exposure. All compounds reduced the maximum optical density reached by E. coli at a range of concentrations including one of environmental relevance, with transcriptome analysis identifying upregulated genes related to stress response and multidrug efflux in response ibuprofen treatment. The non-antibiotic pharmaceuticals did not select for significant genetic changes following a 30 day exposure, and no evidence of selection for cross-resistance to antibiotics was observed for population evolved in the presence of ibuprofen in spite of the differential gene expression after exposure to this compound. This work suggests that these non-antibiotic pharmaceuticals, at environmental concentrations, do not select for cross-resistance to antibiotics in E. coli.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.553469">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.553469" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.553469">
        <p class="paperTitle">Defense mechanism of a bacterial retron supramolecular assembly</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.553469" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.553469" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Y.; Wang, C.; Guan, Z.; Cao, J.; Xu, J.; Wang, S.; Cui, Y.; Wang, Q.; Chen, Y.; Zhang, D.; Sun, M.; Tao, P.; Zou, T.</p>
        <p class="info">Score: 10.0, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.553469' target='https://doi.org/10.1101/2023.08.16.553469'> 10.1101/2023.08.16.553469</a></p>
        <p class="abstract">Retrons are a class of multigene antiphage defense system typically consisting of a retron reverse transcriptase, a non-coding RNA, and a cognate effector. Although the triggers for several retron systems have been discovered recently, the full picture of how retron systems sense invading phages and mediate defense remains to be elucidated. Here, we focus on the retron Ec86 defense system and report its modes of activation and action. We identified a phage-encoded DNA cytosine methyltransferase (Dcm) as the trigger of the Ec86 system and show that Ec86 senses msDNA methylation and becomes activated. We further determined the structure of a tripartite retron Ec86 supramolecular assembly, which is primed for activation by Dcm, and demonstrated that the activated system confers defense through depletion of nucleoside derivatives. These findings emphasize the role of retrons being a second line of defense and highlight an emerging theme of anti-phage defense through supramolecular complex assemblies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554085">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554085" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554085">
        <p class="paperTitle">Space and epigenetic inheritance determine inter-individual differences in siderophore gene expression in bacterial colonies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554085" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554085" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mridha, S.; Wechsler, T.; Kuemmerli, R.</p>
        <p class="info">Score: 9.7, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554085' target='https://doi.org/10.1101/2023.08.21.554085'> 10.1101/2023.08.21.554085</a></p>
        <p class="abstract">Heterogeneity in gene expression among cells in clonal groups is common in bacteria. Albeit ubiquitous, it often remains unclear what the sources of variation are and whether variation has functional significance. Here, we tracked the expression of genes involved in the synthesis of iron-chelating siderophores (pyoverdine and pyochelin) in individual cells of the bacterium Pseudomonas aeruginosa during colony growth on surfaces using time-lapse fluorescence microscopy, to explore potential sources and functions of cellular heterogeneity. Regarding sources, we found that the physical position of cells within the colony and epigenetic gene expression inheritance from mother to daughter cells significantly contributed to cellular heterogeneity. In contrast, cell pole age and cellular lifespan had no effect. Regarding functions, our results indicate that cells optimize their siderophore investment strategies (pyoverdine vs. pyochelin) along a gradient from the centre to the edge of the colony. Moreover, we found evidence that cell lineages with above-average siderophore investment increase the fitness of cell lineages with below-average investment through cooperative sharing of secreted siderophores. Altogether, our study highlights that single-cell experiments combined with automated image and cell-tracking analyses can identify sources of heterogeneity and yield adaptive explanations for gene expression variation among clonal bacterial cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554108">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554108" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554108">
        <p class="paperTitle">Mother-Infant Gut Viruses and their Bacterial Hosts: Transmission Patterns and Dynamics during Pregnancy and Early Life</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554108" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554108" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Garmaeva, S.; Sinha, T.; Gulyaeva, A.; Kuzub, N.; Spreckels, J. E.; Andreu-Sanchez, S.; Gacesa, R.; Vich Vila, A.; Brushett, S.; Kruk, M.; Dekens, J.; Sikkema, J.; Kuipers, F.; Shkoporov, A.; Hill, C.; Scherjon, S.; Wijmenga, C.; Fu, J.; Kurilshikov, A.; Zhernakova, A.</p>
        <p class="info">Score: 9.6, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554108' target='https://doi.org/10.1101/2023.08.21.554108'> 10.1101/2023.08.21.554108</a></p>
        <p class="abstract">Early development of the gut ecosystem is crucial for lifelong health. While infant gut bacterial communities have been studied extensively, the infant gut virome remains under-explored. We longitudinally assessed the composition of gut viruses and their bacterial hosts in 322 total metagenomes and 205 metaviromes from 30 mothers during and after pregnancy and from their 32 infants during their first year of life. While the maternal gut virome composition remained stable during late pregnancy and after birth, the infant gut virome was dynamic in the first year of life and contained a higher abundance of active temperate phages compared to the maternal gut viromes. The infant gut virome composition was also influenced by infant feeding mode and place of delivery. Lastly, we provide evidence of viral-bacterial strains co-transmission from mothers to infants, demonstrating that infants acquire some of their virome from their mother&#39;s gut.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.552852">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.552852" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.552852">
        <p class="paperTitle">Microbial eukaryotic predation pressure and biomass at deep-sea hydrothermal vents: Implications for deep-sea carbon cycling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.552852" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.552852" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hu, S. K.; Anderson, R. E.; Pachiadaki, M. G.; Edgcomb, V. P.; Serres, M. H.; Sylva, S. P.; German, C. R.; Seewald, J. S.; Lang, S. Q.; Huber, J. A.</p>
        <p class="info">Score: 12.2, Published: 2023-08-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.552852' target='https://doi.org/10.1101/2023.08.11.552852'> 10.1101/2023.08.11.552852</a></p>
        <p class="abstract">Deep-sea hydrothermal vent geochemistry shapes the foundation of the microbial food web by fueling chemolithoautotrophic microbial activity. Microbial eukaryotes (or protists) play a critical role in hydrothermal vent food webs as consumers, hosts of symbiotic bacteria, and as a nutritional source to higher trophic levels. We measured cell abundances and predation pressures of vent-associated microbial eukaryotes in low temperature diffuse hydrothermal fluids at the Von Damm and Piccard vent fields along the Mid-Cayman Rise in the Western Caribbean Sea. We present findings from experiments performed under in situ pressure that show higher cell abundances and grazing rates compared to those done at 1 atmosphere (shipboard ambient pressure); this trend was attributed to the impact of depressurization on cell integrity. A relationship between protistan grazing rate, prey cell abundance, and temperature of end member hydrothermal vent fluid was observed at both vent fields, regardless of experimental approach. The quantification of protistan biomass and grazing pressure shows that hydrothermally-fueled microbial food webs play a significant role in the broader deep-sea carbon budget by contributing to local carbon export and supply of nutrient resources to the deep ocean.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
