<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="TMPRSS2 activation of Omicron lineage Spike glycoproteins is regulated by TMPRSS2 cleavage of ACE2
Authors: Aggarwal, A.; Fichter, C.; Milogiannakis, V.; Akerman, A.; Ison, T.; Ruiz Silva, M.; Esneau, C.; Bartlett, N.; Burrell, L.; Patel, S.; Churchill, M.; Angelovich, T.; Parry, R.; Sng, J. D.; Neely, G. G.; Moreno, C. L.; Loo, L.; Kelleher, A. D.; Brilot, F.; Khromykh, A.; Turville, S. G.
Score: 65.1, Published: 2023-09-22 DOI: 10.1101/2023.09.22.558930">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="TMPRSS2 activation of Omicron lineage Spike glycoproteins is regulated by TMPRSS2 cleavage of ACE2
Authors: Aggarwal, A.; Fichter, C.; Milogiannakis, V.; Akerman, A.; Ison, T.; Ruiz Silva, M.; Esneau, C.; Bartlett, N.; Burrell, L.; Patel, S.; Churchill, M.; Angelovich, T.; Parry, R.; Sng, J. D.; Neely, G. G.; Moreno, C. L.; Loo, L.; Kelleher, A. D.; Brilot, F.; Khromykh, A.; Turville, S. G.
Score: 65.1, Published: 2023-09-22 DOI: 10.1101/2023.09.22.558930" />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-27T10:39:11+00:00" />
<meta property="article:modified_time" content="2023-09-27T10:39:11+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="TMPRSS2 activation of Omicron lineage Spike glycoproteins is regulated by TMPRSS2 cleavage of ACE2
Authors: Aggarwal, A.; Fichter, C.; Milogiannakis, V.; Akerman, A.; Ison, T.; Ruiz Silva, M.; Esneau, C.; Bartlett, N.; Burrell, L.; Patel, S.; Churchill, M.; Angelovich, T.; Parry, R.; Sng, J. D.; Neely, G. G.; Moreno, C. L.; Loo, L.; Kelleher, A. D.; Brilot, F.; Khromykh, A.; Turville, S. G.
Score: 65.1, Published: 2023-09-22 DOI: 10.1101/2023.09.22.558930"/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "TMPRSS2 activation of Omicron lineage Spike glycoproteins is regulated by TMPRSS2 cleavage of ACE2\nAuthors: Aggarwal, A.; Fichter, C.; Milogiannakis, V.; Akerman, A.; Ison, T.; Ruiz Silva, M.; Esneau, C.; Bartlett, N.; Burrell, L.; Patel, S.; Churchill, M.; Angelovich, T.; Parry, R.; Sng, J. D.; Neely, G. G.; Moreno, C. L.; Loo, L.; Kelleher, A. D.; Brilot, F.; Khromykh, A.; Turville, S. G.\nScore: 65.1, Published: 2023-09-22 DOI: 10.1101/2023.09.22.558930",
  "keywords": [
    
  ],
  "articleBody": " TMPRSS2 activation of Omicron lineage Spike glycoproteins is regulated by TMPRSS2 cleavage of ACE2\nAuthors: Aggarwal, A.; Fichter, C.; Milogiannakis, V.; Akerman, A.; Ison, T.; Ruiz Silva, M.; Esneau, C.; Bartlett, N.; Burrell, L.; Patel, S.; Churchill, M.; Angelovich, T.; Parry, R.; Sng, J. D.; Neely, G. G.; Moreno, C. L.; Loo, L.; Kelleher, A. D.; Brilot, F.; Khromykh, A.; Turville, S. G.\nScore: 65.1, Published: 2023-09-22 DOI: 10.1101/2023.09.22.558930\nContinued high levels spread of SARS-CoV-2 globally enabled accumulation of changes within the Spike glycoprotein, leading to resistance to neutralising antibodies and concomitant changes to entry requirements that increased viral transmission fitness. Herein, we demonstrate a significant change in ACE2 and TMPRSS2 use by primary SARS-CoV-2 isolates that occurred upon arrival of Omicron lineages. Mechanistically we show this shift to be a function of two distinct ACE2 pools based on cleavage or non-cleavage of ACE2 by TMPRSS2 activity. In engineered cells overexpressing ACE2 and TMPRSS2, ACE2 was cleaved by TMPRSS2 and this led to either augmentation or progressive attenuation of pre-Omicron and Omicron lineages, respectfully. In contrast, TMPRSS2 resistant ACE2 restored infectivity across all Omicron lineages through enabling ACE2 binding that facilitated TMPRSS2 spike activation. Therefore, our data support the tropism shift of Omicron lineages to be a function of evolution towards the use of uncleaved pools of ACE2 with the latter consistent with its role as a chaperone for many tissue specific amino acid transport proteins. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=99 SRC=\"FIGDIR/small/558930v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (26K): org.highwire.dtl.DTLVardef@885f4eorg.highwire.dtl.DTLVardef@12937b4org.highwire.dtl.DTLVardef@1082bd7org.highwire.dtl.DTLVardef@1766e55_HPS_FORMAT_FIGEXP M_FIG O_FLOATNOGraphical AbstractC_FLOATNO Cleaved ACE2 pool model and evolution of SARS CoV-2 Tropism. A. \u0026B. ACE2 cleaved pool model to reconcile the evolving entry requirements of SARS CoV-2 and changes in viral tropism in vivo. A. As observed for SARS CoV-1 and early SARS CoV-2 lineages, cleavage of ACE2 by TMPRSS2 augments cellular entry by enabling TMPRSS2 Spike S2 activation to proceed. In primary lower respiratory this would proceed to finely regulate ACE2 as part of the Renin Angiogensin System (RAAS). In other settings where ACE2 is not cleaved by TMPRSS2, entry still proceeds. In both ACE2 pools, TMPRSS2 activation can proceed (\"on\" conformation) B. For Omicron lineages, cleavage of ACE2 by TMPRSS2 is an \"off\" switch for latter TMPRSS2 Spike S2 activation. Whilst this supports attenuation in tissue where ACE2 is cleavage sensitive due to its role in RAAS (e.g. lung), replication in other tissue has been observed to proceed. Independent of RAAS, ACE2 can also act as a chaperone for several tissue specific amino acid transporters which engage at and around the TMPRSS2 cleavage site. To mimic lack of ACE2 cleavage in this setting, we mutated the ACE2-TMPRSS2 cleavage site and revealed settings where ACE2 is not subject to TMPRSS2 cleavage is a \"on\" switch for latter TMPRSS2 Spike S2 activation. C. Through generating engineered lines that were biased for TMPRSS2 cleavage or expressed ACE2 that was TMPRSS2 resistant, we mapped the evolution of ACE2 and TMPRSS2 use across the first three years of the pandemic. In early clades, TMPRSS2 cleavage of ACE2 augmented ACE2/TMPRSS2 viral entry. This augmentation steadily decayed and led to attenuation upon the arrival of omicron BA.1 with continued attenuation observed now in XBB omicron lineages. Over the same time, TMPRSS2 cleavage resistant ACE2 has supported TMPRSS2 activation entry across all SARS CoV-2 lineages equally. C_FIG\nGenome-wide Characterization of Diverse Bacteriophages Enabled by RNA-Binding CRISPRi\nAuthors: Adler, B. A.; Al-Shimary, M. J.; Patel, J. R.; Ambruster, E.; Colognori, D.; Charles, E. J.; Miller, K. V.; Lahiri, A.; Trinidad, M.; Boger, R.; Nomburg, J.; Beurnier, S.; Cui, M. L.; Barrangou, R.; Mutalik, V. K.; Schoeniger, J. S.; Pogliano, J. A.; Savage, D. F.; Doudna, J. A.; Cress, B. F.\nScore: 71.0, Published: 2023-09-18 DOI: 10.1101/2023.09.18.558157\nBacteriophages constitute one of the largest sources of unknown gene content in the biosphere. Even for well-studied model phages, robust experimental approaches to identify and study their essential genes remain elusive. We uncover and exploit the conserved vulnerability of the phage transcriptome to facilitate genome-wide protein expression knockdown via programmable RNA-binding protein dRfxCas13d (CRISPRi-ART) across diverse phages and their host. Establishing the first broad-spectrum phage functional genomics platform, we predict over 90 essential genes across four phage genomes, a third of which have no known function. These results highlight hidden infection strategies encoded in the most abundant biological entities on earth and provide a facile platform to study them.\nA phage nucleus-associated RNA-binding protein is required for jumbo phage infection\nAuthors: Enustun, E.; Armbruster, E. G.; Lee, J.; Zhang, S.; Yee, B. A.; Gu, Y.; Deep, A.; Naritomi, J. T.; Liang, Q.; Aigner, S.; Adler, B. A.; Cress, B. F.; Doudna, J. A.; Chaikeeratisak, V.; Cleveland, D. W.; Ghassemian, M.; Yeo, G. W.; Pogliano, J.; Corbett, K. D.\nScore: 21.1, Published: 2023-09-22 DOI: 10.1101/2023.09.22.559000\nLarge-genome bacteriophages (jumbo phages) of the Chimalliviridae family assemble a nucleus-like compartment bounded by a protein shell that protects the replicating phage genome from host-encoded restriction enzymes and CRISPR/Cas nucleases. While the nuclear shell provides broad protection against host nucleases, it necessitates transport of mRNA out of the nucleus-like compartment for translation by host ribosomes, and transport of specific proteins into the nucleus-like compartment to support DNA replication and mRNA transcription. Here we identify a conserved phage nuclear shell-associated protein that we term Chimallin C (ChmC), which adopts a nucleic acid-binding fold, binds RNA with high affinity in vitro, and binds phage mRNAs in infected cells. ChmC also forms phase-separated condensates with RNA in vitro. Targeted knockdown of ChmC using mRNA-targeting dCas13d halts infections at an early stage. Taken together, our data suggest that the conserved ChmC protein acts as a chaperone for phage mRNAs, potentially stabilizing these mRNAs and driving their translocation through the nuclear shell to promote translation and infection progression.\nEvolutionary dynamics of proteinaceous MAMPs reveals intrabacterial antagonism of plant immune perception\nAuthors: Stevens, D. M.; Moreno-Perez, A.; Weisberg, A. J.; Ramsing, C.; Fliegmann, J.; Zhang, N.; Madrigal, M.; Martin, G.; Steinbrenner, A.; Felix, G.; Coaker, G.\nScore: 19.8, Published: 2023-09-21 DOI: 10.1101/2023.09.21.558511\nPlants and animals detect biomolecules termed Microbe-Associated Molecular Patterns (MAMPs) and induce immune responses. The impact of diverse multi-copy MAMPs on immune induction is unknown. We characterized the evolutionary trajectories of five proteinaceous MAMPs and the effect of copy number and sequence variation on plant immune outcomes. From 4,228 plant-associated bacterial genomes, 34,262 MAMPs were identified. Natural variation was constricted, enabling characterization of immune perception across thousands of bacteria. We characterized the immunogenic outcomes of two MAMPs with different evolutionary trajectories the Elongation Factor Tu (EF-Tu) epitope elf18 and Cold Shock Protein (CSP) epitope csp22. EF-Tu is a single-copy gene and 90% of elf18 epitopes are immunogenic. Conversely, CSPs are multi-copy with 99.8% of bacteria carrying at least one immunogenic CSP and only 54% of csp22 epitopes are immunogenic. We uncovered a new mechanism for immune evasion, intrabacterial antagonism, from a non-immunogenic actinobacterial CSP that blocks perception of immunogenic forms.\nA Parasite Odyssey: An RNA virus concealed in Toxoplasma gondii\nAuthors: Gupta, P.; Hiller, A.; Chowdhury, J.; Lim, D.; Lim, D.; Babaian, A.; Pereira, L.; Morales, A.\nScore: 31.2, Published: 2023-09-18 DOI: 10.1101/2023.09.17.558162\nWe are in the midst of the \"Platinum Age of Virus Discovery\", an era characterized by the exponential growth in the discovery of virus biodiversity. In humans (or any animal) there are more viruses than simply those directly infecting the animal cells. Viruses infect all species constituting the microbiome: including bacteria, fungi, and unicellular parasites. Thus the complexity of possible interactions between host, microbe, and viruses is unfathomable. To understand this network we must employ computationally assisted virology as a means of analyzing and interpreting the millions of available samples to decipher the myriad of ways in which viruses may intersect human health. From a global screen for human neuron-associated viruses, we identify a novel narnavirus Achaeavirus odysseus (Ao) which likely infects the neurotropic parasite Toxoplasma gondii. Previously, several parasitic protozoan viruses (PPVs) have been mechanistically established as triggers of host innate responses, and here we present in silico evidence that Ao is a plausible pro-inflammatory factor in mouse and human cells infected by T. gondii. T. gondii infects billions of people worldwide, yet the prognosis of toxoplasmosis disease is highly variable, and PPVs like Ao could function as a hitherto undescribed hypervirulence factor. More broadly, we explore phylogenetically proximal viruses to Ao and discover 18 new species of Achaeavirus, all found in vertebrate transcriptome and metatranscriptomes. While the Narnavirus samples making up this genus-like clade are derived from sheep, goat, bat, rabbit, chicken, and pigeon, the presence of virus is strongly predictive of parasitic (Apicomplexa) nucleic acid co-occurrence, supporting that these are a clade of parasite-infecting viruses. This is a computational proof-of-concept study in which we rapidly analyze millions of datasets from which we distil a mechanistically, ecologically, and phylogenetically refined hypothesis. We predict this highly diverged Ao RNA virus is biologically a T. gondii infection, and that Ao, and other viruses like it, will modulate this disease which afflicts billions worldwide.\nPhage-induced efflux down-regulation boosts antibiotic efficacy\nAuthors: Kraus, S.; Lapinska, U.; Chawla, K.; Baker, E.; Attrill, E.; ONeill, P.; Farbos, A.; Jeffries, A.; Galyov, E.; Korbsrisate, S.; Barnes, K.; Harding, S.; Tsaneva-Atanasova, K.; Blaskovich, M.; Pagliara, S.\nScore: 14.6, Published: 2023-09-22 DOI: 10.1101/2023.09.21.558807\nThe interactions between a virus and its host vary in space and time and are affected by the presence of molecules that alter the physiology of either the host or the virus. Determining the dynamics of these interactions is paramount for predicting the fate of bacterial and phage populations and for designing rational phage-antibiotic therapies. We study the interactions between stationary phase Burkholderia thailandensis and the phage {Phi}Bp-AMP1. Although heterogeneous genetic resistance to phage rapidly emerges in B. thailandensis, the presence of phage enhances the efficacy of three major antibiotic classes, the quinolones, the beta-lactams and the tetracyclines, but antagonizes tetrahydrofolate synthesis inhibitors. Enhanced antibiotic efficacy is underpinned by reduced antibiotic efflux in the presence of phage. This new phage-antibiotic therapy allows for eradication of stationary phase bacteria, whilst requiring reduced antibiotic concentrations, which is crucial for treating infections in sites where it is difficult to achieve high antibiotic concentrations.\nA conserved cell-pole determinant organizes proper polar flagellum formation\nAuthors: Arroyo-Perez, E. E.; Hook, J. C.; Alvarado, A.; Wimmi, S.; Glatter, T.; Thormann, K. M.; Ringgaard, S.\nScore: 14.2, Published: 2023-09-21 DOI: 10.1101/2023.09.20.558563\nThe coordination of cell cycle progression and flagellar synthesis is a complex process in motile bacteria. In {gamma}-proteobacteria, the localization of the flagellum to the cell pole is mediated by the SRP-type GTPase FlhF. However, the mechanism of action of FlhF, and its relationship with the cell pole landmark protein HubP remain unclear. In this study, we discovered a novel protein called FipA that is required for normal FlhF activity and function in polar flagellar synthesis. We demonstrated that membrane-localized FipA interacts with FlhF and is required for normal flagellar synthesis in Vibrio parahaemolyticus, Pseudomonas putida, and Shewanella putrefaciens, and it does so independently of the polar localization mediated by HubP. FipA exhibits a dynamic localization pattern and is present at the designated pole before flagellar synthesis begins, suggesting its role in licensing flagellar formation. This discovery provides insight into a new pathway for regulating flagellum synthesis and coordinating cellular organization in bacteria that rely on polar flagellation and FlhF-dependent localization.\nThe lactonase BxdA mediates metabolic adaptation of maize root bacteria to benzoxazinoids\nAuthors: Thoenen, L.; Kreuzer, M.; Florean, M.; Mateo, P.; Zuest, T.; Giroud, C.; Rouyer, L.; Gfeller, V.; Notter, M.; Knoch, E.; Hapfelmeier, S.; Becker, C.; Schandry, N.; Robert, C.; Koellner, T.; Bruggmann, R.; Erb, M.; Schlaeppi, K.\nScore: 13.1, Published: 2023-09-23 DOI: 10.1101/2023.09.22.559061\nRoot exudates contain secondary metabolites that affect the plant's root microbiome. How microbes cope with these bioactive compounds, and how this ability shapes root microbiomes remain largely unknown. We investigated how maize root bacteria metabolise benzoxazinoids, the main specialised metabolites of maize. Diverse and abundant bacteria metabolised the major compound (6-methoxy-benzoxazolin-2-one, MBOA) in the maize rhizosphere to 2-amino-7-methoxyphenoxazin-3-one (AMPO). By contrast, bacteria isolated from Arabidopsis, which does not produce benzoxazinoids, were unable to metabolise MBOA. Among Microbacteria strains, this differential metabolisation allowed to identify a conserved gene cluster containing the lactonase bxdA. BxdA converts MBOA to AMPO in vitro and we show that this capacity provided bacteria a growth benefit under carbon-limiting conditions. Together these results reveal that maize root bacteria - through BxdA - are metabolically adapted to the benzoxazinoids of their host. We propose that metabolic adaptation to plant-specialised compounds shapes root bacterial communities across the plant kingdom.\nNeutralization Escape by SARS-CoV-2 Omicron Subvariant BA.2.86\nAuthors: Lasrado, N.; Collier, A.-r.; Hachmann, N.; Miller, J.; Rowe, M.; Schonberg, E.; Rodrigues, S.; LaPiana, A.; Patio, R.; Anand, T.; Fisher, J.; Mazurek, C.; Guan, R.; Wagh, K.; Theiler, J.; Korber, B.; Barouch, D. H.\nScore: 824.2, Published: 2023-09-05 DOI: 10.1101/2023.09.04.556272\nThe continued evolution of SARS-CoV-2 may lead to evasion of vaccine immunity and natural immunity. A highly mutated Omicron variant BA.2.86 has recently been identified with over 30 amino acid changes in Spike compared with BA.2 and XBB.1.5. As of September 4, 2023, BA.2.86 has been identified in 37 sequences from 10 countries, which is likely an underestimate due to limited surveillance. The ability of BA.2.86 to evade NAbs compared with other currently circulating Omicron variants remains unknown. Our data show that NAb responses to BA.2.86 were lower than to BA.2 but were comparable or slightly higher than to the current circulating recombinant variants XBB.1.5, XBB.1.16, EG.5, EG.5.1, and FL.1.5.1.\nBenchmarking long-read sequencing strategies for obtaining ASV-resolved rRNA operons from environmental microeukaryotes\nAuthors: Overgaard, C. K.; Jamy, M.; Radutoiu, S.; Burki, F.; Dueholm, M. K. D.\nScore: 12.1, Published: 2023-09-22 DOI: 10.1101/2023.09.21.558928\nThe use of short-read metabarcoding for classifying microeukaryotes is challenged by the lack of comprehensive 18S rRNA reference databases. While recent advances in high-throughput long-read sequencing provide the potential to greatly increase the phylogenetic coverage of these databases, the performance of different sequencing technologies and subsequent bioinformatics processing remain to be evaluated, primarily because of the absence of well-defined eukaryotic mock communities. To address this challenge, we created a eukaryotic rRNA operons clone-library and turned it into a precisely defined synthetic eukaryotic mock community. This mock community was then used to evaluate the performance of three long-read sequencing techniques (PacBio HiFi, and Nanopore UMI with/without clonal pre-amplification) and three tools for resolving amplicons sequence variants (ASVs) (Uchime3, Unoise3, and DADA2). We investigated the sensitivity of the sequencing techniques based on the number of detected mock taxa, and the accuracy of the different ASV-calling tools with a specific focus on the presence of chimera among the final rRNA operon ASVs. Based on our findings, we provide recommendations and best practice protocols for how to cost-effectively obtain essential error-free rRNA operons in high-throughput. An agricultural soil sample was used to demonstrate that the sequencing and bioinformatic results from the mock community also translates to highly diverse natural samples.\n",
  "wordCount" : "2483",
  "inLanguage": "en",
  "datePublished": "2023-09-27T10:39:11Z",
  "dateModified": "2023-09-27T10:39:11Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta"><span>updated on September 27, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.558930">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.558930" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.558930">
        <p class="paperTitle">TMPRSS2 activation of Omicron lineage Spike glycoproteins is regulated by TMPRSS2 cleavage of ACE2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.558930" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.558930" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aggarwal, A.; Fichter, C.; Milogiannakis, V.; Akerman, A.; Ison, T.; Ruiz Silva, M.; Esneau, C.; Bartlett, N.; Burrell, L.; Patel, S.; Churchill, M.; Angelovich, T.; Parry, R.; Sng, J. D.; Neely, G. G.; Moreno, C. L.; Loo, L.; Kelleher, A. D.; Brilot, F.; Khromykh, A.; Turville, S. G.</p>
        <p class="info">Score: 65.1, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.558930' target='https://doi.org/10.1101/2023.09.22.558930'> 10.1101/2023.09.22.558930</a></p>
        <p class="abstract">Continued high levels spread of SARS-CoV-2 globally enabled accumulation of changes within the Spike glycoprotein, leading to resistance to neutralising antibodies and concomitant changes to entry requirements that increased viral transmission fitness. Herein, we demonstrate a significant change in ACE2 and TMPRSS2 use by primary SARS-CoV-2 isolates that occurred upon arrival of Omicron lineages. Mechanistically we show this shift to be a function of two distinct ACE2 pools based on cleavage or non-cleavage of ACE2 by TMPRSS2 activity. In engineered cells overexpressing ACE2 and TMPRSS2, ACE2 was cleaved by TMPRSS2 and this led to either augmentation or progressive attenuation of pre-Omicron and Omicron lineages, respectfully. In contrast, TMPRSS2 resistant ACE2 restored infectivity across all Omicron lineages through enabling ACE2 binding that facilitated TMPRSS2 spike activation. Therefore, our data support the tropism shift of Omicron lineages to be a function of evolution towards the use of uncleaved pools of ACE2 with the latter consistent with its role as a chaperone for many tissue specific amino acid transport proteins.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=99 SRC=&#34;FIGDIR/small/558930v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (26K):
org.highwire.dtl.DTLVardef@885f4eorg.highwire.dtl.DTLVardef@12937b4org.highwire.dtl.DTLVardef@1082bd7org.highwire.dtl.DTLVardef@1766e55_HPS_FORMAT_FIGEXP  M_FIG O_FLOATNOGraphical AbstractC_FLOATNO Cleaved ACE2 pool model and evolution of SARS CoV-2 Tropism.

A. &amp;B. ACE2 cleaved pool model to reconcile the evolving entry requirements of SARS CoV-2 and changes in viral tropism in vivo. A. As observed for SARS CoV-1 and early SARS CoV-2 lineages, cleavage of ACE2 by TMPRSS2 augments cellular entry by enabling TMPRSS2 Spike S2 activation to proceed. In primary lower respiratory this would proceed to finely regulate ACE2 as part of the Renin Angiogensin System (RAAS). In other settings where ACE2 is not cleaved by TMPRSS2, entry still proceeds. In both ACE2 pools, TMPRSS2 activation can proceed (&#34;on&#34; conformation) B. For Omicron lineages, cleavage of ACE2 by TMPRSS2 is an &#34;off&#34; switch for latter TMPRSS2 Spike S2 activation. Whilst this supports attenuation in tissue where ACE2 is cleavage sensitive due to its role in RAAS (e.g. lung), replication in other tissue has been observed to proceed. Independent of RAAS, ACE2 can also act as a chaperone for several tissue specific amino acid transporters which engage at and around the TMPRSS2 cleavage site. To mimic lack of ACE2 cleavage in this setting, we mutated the ACE2-TMPRSS2 cleavage site and revealed settings where ACE2 is not subject to TMPRSS2 cleavage is a &#34;on&#34; switch for latter TMPRSS2 Spike S2 activation. C. Through generating engineered lines that were biased for TMPRSS2 cleavage or expressed ACE2 that was TMPRSS2 resistant, we mapped the evolution of ACE2 and TMPRSS2 use across the first three years of the pandemic. In early clades, TMPRSS2 cleavage of ACE2 augmented ACE2/TMPRSS2 viral entry. This augmentation steadily decayed and led to attenuation upon the arrival of omicron BA.1 with continued attenuation observed now in XBB omicron lineages. Over the same time, TMPRSS2 cleavage resistant ACE2 has supported TMPRSS2 activation entry across all SARS CoV-2 lineages equally.

C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558157">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558157" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558157">
        <p class="paperTitle">Genome-wide Characterization of Diverse Bacteriophages Enabled by RNA-Binding CRISPRi</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558157" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558157" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Adler, B. A.; Al-Shimary, M. J.; Patel, J. R.; Ambruster, E.; Colognori, D.; Charles, E. J.; Miller, K. V.; Lahiri, A.; Trinidad, M.; Boger, R.; Nomburg, J.; Beurnier, S.; Cui, M. L.; Barrangou, R.; Mutalik, V. K.; Schoeniger, J. S.; Pogliano, J. A.; Savage, D. F.; Doudna, J. A.; Cress, B. F.</p>
        <p class="info">Score: 71.0, Published: 2023-09-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558157' target='https://doi.org/10.1101/2023.09.18.558157'> 10.1101/2023.09.18.558157</a></p>
        <p class="abstract">Bacteriophages constitute one of the largest sources of unknown gene content in the biosphere. Even for well-studied model phages, robust experimental approaches to identify and study their essential genes remain elusive. We uncover and exploit the conserved vulnerability of the phage transcriptome to facilitate genome-wide protein expression knockdown via programmable RNA-binding protein dRfxCas13d (CRISPRi-ART) across diverse phages and their host. Establishing the first broad-spectrum phage functional genomics platform, we predict over 90 essential genes across four phage genomes, a third of which have no known function. These results highlight hidden infection strategies encoded in the most abundant biological entities on earth and provide a facile platform to study them.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.559000">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.559000" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.559000">
        <p class="paperTitle">A phage nucleus-associated RNA-binding protein is required for jumbo phage infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.559000" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.559000" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Enustun, E.; Armbruster, E. G.; Lee, J.; Zhang, S.; Yee, B. A.; Gu, Y.; Deep, A.; Naritomi, J. T.; Liang, Q.; Aigner, S.; Adler, B. A.; Cress, B. F.; Doudna, J. A.; Chaikeeratisak, V.; Cleveland, D. W.; Ghassemian, M.; Yeo, G. W.; Pogliano, J.; Corbett, K. D.</p>
        <p class="info">Score: 21.1, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.559000' target='https://doi.org/10.1101/2023.09.22.559000'> 10.1101/2023.09.22.559000</a></p>
        <p class="abstract">Large-genome bacteriophages (jumbo phages) of the Chimalliviridae family assemble a nucleus-like compartment bounded by a protein shell that protects the replicating phage genome from host-encoded restriction enzymes and CRISPR/Cas nucleases. While the nuclear shell provides broad protection against host nucleases, it necessitates transport of mRNA out of the nucleus-like compartment for translation by host ribosomes, and transport of specific proteins into the nucleus-like compartment to support DNA replication and mRNA transcription. Here we identify a conserved phage nuclear shell-associated protein that we term Chimallin C (ChmC), which adopts a nucleic acid-binding fold, binds RNA with high affinity in vitro, and binds phage mRNAs in infected cells. ChmC also forms phase-separated condensates with RNA in vitro. Targeted knockdown of ChmC using mRNA-targeting dCas13d halts infections at an early stage. Taken together, our data suggest that the conserved ChmC protein acts as a chaperone for phage mRNAs, potentially stabilizing these mRNAs and driving their translocation through the nuclear shell to promote translation and infection progression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558511">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558511" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558511">
        <p class="paperTitle">Evolutionary dynamics of proteinaceous MAMPs reveals intrabacterial antagonism of plant immune perception</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558511" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558511" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stevens, D. M.; Moreno-Perez, A.; Weisberg, A. J.; Ramsing, C.; Fliegmann, J.; Zhang, N.; Madrigal, M.; Martin, G.; Steinbrenner, A.; Felix, G.; Coaker, G.</p>
        <p class="info">Score: 19.8, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558511' target='https://doi.org/10.1101/2023.09.21.558511'> 10.1101/2023.09.21.558511</a></p>
        <p class="abstract">Plants and animals detect biomolecules termed Microbe-Associated Molecular Patterns (MAMPs) and induce immune responses. The impact of diverse multi-copy MAMPs on immune induction is unknown. We characterized the evolutionary trajectories of five proteinaceous MAMPs and the effect of copy number and sequence variation on plant immune outcomes. From 4,228 plant-associated bacterial genomes, 34,262 MAMPs were identified. Natural variation was constricted, enabling characterization of immune perception across thousands of bacteria. We characterized the immunogenic outcomes of two MAMPs with different evolutionary trajectories the Elongation Factor Tu (EF-Tu) epitope elf18 and Cold Shock Protein (CSP) epitope csp22. EF-Tu is a single-copy gene and 90% of elf18 epitopes are immunogenic. Conversely, CSPs are multi-copy with 99.8% of bacteria carrying at least one immunogenic CSP and only 54% of csp22 epitopes are immunogenic. We uncovered a new mechanism for immune evasion, intrabacterial antagonism, from a non-immunogenic actinobacterial CSP that blocks perception of immunogenic forms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.17.558162">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.17.558162" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.17.558162">
        <p class="paperTitle">A Parasite Odyssey: An RNA virus concealed in Toxoplasma gondii</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.17.558162" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.17.558162" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gupta, P.; Hiller, A.; Chowdhury, J.; Lim, D.; Lim, D.; Babaian, A.; Pereira, L.; Morales, A.</p>
        <p class="info">Score: 31.2, Published: 2023-09-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.17.558162' target='https://doi.org/10.1101/2023.09.17.558162'> 10.1101/2023.09.17.558162</a></p>
        <p class="abstract">We are in the midst of the &#34;Platinum Age of Virus Discovery&#34;, an era characterized by the exponential growth in the discovery of virus biodiversity. In humans (or any animal) there are more viruses than simply those directly infecting the animal cells. Viruses infect all species constituting the microbiome: including bacteria, fungi, and unicellular parasites. Thus the complexity of possible interactions between host, microbe, and viruses is unfathomable. To understand this network we must employ computationally assisted virology as a means of analyzing and interpreting the millions of available samples to decipher the myriad of ways in which viruses may intersect human health.

From a global screen for human neuron-associated viruses, we identify a novel narnavirus Achaeavirus odysseus (Ao) which likely infects the neurotropic parasite Toxoplasma gondii. Previously, several parasitic protozoan viruses (PPVs) have been mechanistically established as triggers of host innate responses, and here we present in silico evidence that Ao is a plausible pro-inflammatory factor in mouse and human cells infected by T. gondii. T. gondii infects billions of people worldwide, yet the prognosis of toxoplasmosis disease is highly variable, and PPVs like Ao could function as a hitherto undescribed hypervirulence factor. More broadly, we explore phylogenetically proximal viruses to Ao and discover 18 new species of Achaeavirus, all found in vertebrate transcriptome and metatranscriptomes. While the Narnavirus samples making up this genus-like clade are derived from sheep, goat, bat, rabbit, chicken, and pigeon, the presence of virus is strongly predictive of parasitic (Apicomplexa) nucleic acid co-occurrence, supporting that these are a clade of parasite-infecting viruses.

This is a computational proof-of-concept study in which we rapidly analyze millions of datasets from which we distil a mechanistically, ecologically, and phylogenetically refined hypothesis. We predict this highly diverged Ao RNA virus is biologically a T. gondii infection, and that Ao, and other viruses like it, will modulate this disease which afflicts billions worldwide.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558807">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558807" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558807">
        <p class="paperTitle">Phage-induced efflux down-regulation boosts antibiotic efficacy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558807" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558807" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kraus, S.; Lapinska, U.; Chawla, K.; Baker, E.; Attrill, E.; ONeill, P.; Farbos, A.; Jeffries, A.; Galyov, E.; Korbsrisate, S.; Barnes, K.; Harding, S.; Tsaneva-Atanasova, K.; Blaskovich, M.; Pagliara, S.</p>
        <p class="info">Score: 14.6, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558807' target='https://doi.org/10.1101/2023.09.21.558807'> 10.1101/2023.09.21.558807</a></p>
        <p class="abstract">The interactions between a virus and its host vary in space and time and are affected by the presence of molecules that alter the physiology of either the host or the virus. Determining the dynamics of these interactions is paramount for predicting the fate of bacterial and phage populations and for designing rational phage-antibiotic therapies. We study the interactions between stationary phase Burkholderia thailandensis and the phage {Phi}Bp-AMP1. Although heterogeneous genetic resistance to phage rapidly emerges in B. thailandensis, the presence of phage enhances the efficacy of three major antibiotic classes, the quinolones, the beta-lactams and the tetracyclines, but antagonizes tetrahydrofolate synthesis inhibitors. Enhanced antibiotic efficacy is underpinned by reduced antibiotic efflux in the presence of phage. This new phage-antibiotic therapy allows for eradication of stationary phase bacteria, whilst requiring reduced antibiotic concentrations, which is crucial for treating infections in sites where it is difficult to achieve high antibiotic concentrations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.558563">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.558563" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.558563">
        <p class="paperTitle">A conserved cell-pole determinant organizes proper polar flagellum formation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.558563" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.558563" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Arroyo-Perez, E. E.; Hook, J. C.; Alvarado, A.; Wimmi, S.; Glatter, T.; Thormann, K. M.; Ringgaard, S.</p>
        <p class="info">Score: 14.2, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.558563' target='https://doi.org/10.1101/2023.09.20.558563'> 10.1101/2023.09.20.558563</a></p>
        <p class="abstract">The coordination of cell cycle progression and flagellar synthesis is a complex process in motile bacteria. In {gamma}-proteobacteria, the localization of the flagellum to the cell pole is mediated by the SRP-type GTPase FlhF. However, the mechanism of action of FlhF, and its relationship with the cell pole landmark protein HubP remain unclear. In this study, we discovered a novel protein called FipA that is required for normal FlhF activity and function in polar flagellar synthesis. We demonstrated that membrane-localized FipA interacts with FlhF and is required for normal flagellar synthesis in Vibrio parahaemolyticus, Pseudomonas putida, and Shewanella putrefaciens, and it does so independently of the polar localization mediated by HubP. FipA exhibits a dynamic localization pattern and is present at the designated pole before flagellar synthesis begins, suggesting its role in licensing flagellar formation. This discovery provides insight into a new pathway for regulating flagellum synthesis and coordinating cellular organization in bacteria that rely on polar flagellation and FlhF-dependent localization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.559061">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.559061" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.559061">
        <p class="paperTitle">The lactonase BxdA mediates metabolic adaptation of maize root bacteria to benzoxazinoids</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.559061" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.559061" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Thoenen, L.; Kreuzer, M.; Florean, M.; Mateo, P.; Zuest, T.; Giroud, C.; Rouyer, L.; Gfeller, V.; Notter, M.; Knoch, E.; Hapfelmeier, S.; Becker, C.; Schandry, N.; Robert, C.; Koellner, T.; Bruggmann, R.; Erb, M.; Schlaeppi, K.</p>
        <p class="info">Score: 13.1, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.559061' target='https://doi.org/10.1101/2023.09.22.559061'> 10.1101/2023.09.22.559061</a></p>
        <p class="abstract">Root exudates contain secondary metabolites that affect the plant&#39;s root microbiome. How microbes cope with these bioactive compounds, and how this ability shapes root microbiomes remain largely unknown. We investigated how maize root bacteria metabolise benzoxazinoids, the main specialised metabolites of maize. Diverse and abundant bacteria metabolised the major compound (6-methoxy-benzoxazolin-2-one, MBOA) in the maize rhizosphere to 2-amino-7-methoxyphenoxazin-3-one (AMPO). By contrast, bacteria isolated from Arabidopsis, which does not produce benzoxazinoids, were unable to metabolise MBOA. Among Microbacteria strains, this differential metabolisation allowed to identify a conserved gene cluster containing the lactonase bxdA. BxdA converts MBOA to AMPO in vitro and we show that this capacity provided bacteria a growth benefit under carbon-limiting conditions. Together these results reveal that maize root bacteria - through BxdA - are metabolically adapted to the benzoxazinoids of their host. We propose that metabolic adaptation to plant-specialised compounds shapes root bacterial communities across the plant kingdom.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.04.556272">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.04.556272" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.04.556272">
        <p class="paperTitle">Neutralization Escape by SARS-CoV-2 Omicron Subvariant BA.2.86</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.04.556272" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.04.556272" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lasrado, N.; Collier, A.-r.; Hachmann, N.; Miller, J.; Rowe, M.; Schonberg, E.; Rodrigues, S.; LaPiana, A.; Patio, R.; Anand, T.; Fisher, J.; Mazurek, C.; Guan, R.; Wagh, K.; Theiler, J.; Korber, B.; Barouch, D. H.</p>
        <p class="info">Score: 824.2, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.04.556272' target='https://doi.org/10.1101/2023.09.04.556272'> 10.1101/2023.09.04.556272</a></p>
        <p class="abstract">The continued evolution of SARS-CoV-2 may lead to evasion of vaccine immunity and natural immunity. A highly mutated Omicron variant BA.2.86 has recently been identified with over 30 amino acid changes in Spike compared with BA.2 and XBB.1.5. As of September 4, 2023, BA.2.86 has been identified in 37 sequences from 10 countries, which is likely an underestimate due to limited surveillance. The ability of BA.2.86 to evade NAbs compared with other currently circulating Omicron variants remains unknown. Our data show that NAb responses to BA.2.86 were lower than to BA.2 but were comparable or slightly higher than to the current circulating recombinant variants XBB.1.5, XBB.1.16, EG.5, EG.5.1, and FL.1.5.1.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558928">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558928" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558928">
        <p class="paperTitle">Benchmarking long-read sequencing strategies for obtaining ASV-resolved rRNA operons from environmental microeukaryotes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558928" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558928" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Overgaard, C. K.; Jamy, M.; Radutoiu, S.; Burki, F.; Dueholm, M. K. D.</p>
        <p class="info">Score: 12.1, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558928' target='https://doi.org/10.1101/2023.09.21.558928'> 10.1101/2023.09.21.558928</a></p>
        <p class="abstract">The use of short-read metabarcoding for classifying microeukaryotes is challenged by the lack of comprehensive 18S rRNA reference databases. While recent advances in high-throughput long-read sequencing provide the potential to greatly increase the phylogenetic coverage of these databases, the performance of different sequencing technologies and subsequent bioinformatics processing remain to be evaluated, primarily because of the absence of well-defined eukaryotic mock communities. To address this challenge, we created a eukaryotic rRNA operons clone-library and turned it into a precisely defined synthetic eukaryotic mock community. This mock community was then used to evaluate the performance of three long-read sequencing techniques (PacBio HiFi, and Nanopore UMI with/without clonal pre-amplification) and three tools for resolving amplicons sequence variants (ASVs) (Uchime3, Unoise3, and DADA2). We investigated the sensitivity of the sequencing techniques based on the number of detected mock taxa, and the accuracy of the different ASV-calling tools with a specific focus on the presence of chimera among the final rRNA operon ASVs. Based on our findings, we provide recommendations and best practice protocols for how to cost-effectively obtain essential error-free rRNA operons in high-throughput. An agricultural soil sample was used to demonstrate that the sequencing and bioinformatic results from the mock community also translates to highly diverse natural samples.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
