<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Viroid-like colonists of human microbiomes
Authors: Zheludev, I. N.; Edgar, R. C.; Galiano, M. J. L.; De la Pena, M.; Babaian, A.; Bhatt, A. S.; Fire, A. Z.
Score: 385.7, Published: 2024-01-21 DOI: 10.1101/2024.01.20.576352
Here, we describe the &#34;Obelisks,&#34; a previously unrecognised class of viroid-like elements that we first identified in human gut metatranscriptomic data. &#34;Obelisks&#34; share several properties: (i) apparently circular RNA [~]1kb genome assemblies, (ii) predicted rod-like secondary structures encompassing the entire genome, and (iii) open reading frames coding for a novel protein superfamily, which we call the &#34;Oblins&#34;.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Viroid-like colonists of human microbiomes
Authors: Zheludev, I. N.; Edgar, R. C.; Galiano, M. J. L.; De la Pena, M.; Babaian, A.; Bhatt, A. S.; Fire, A. Z.
Score: 385.7, Published: 2024-01-21 DOI: 10.1101/2024.01.20.576352
Here, we describe the &#34;Obelisks,&#34; a previously unrecognised class of viroid-like elements that we first identified in human gut metatranscriptomic data. &#34;Obelisks&#34; share several properties: (i) apparently circular RNA [~]1kb genome assemblies, (ii) predicted rod-like secondary structures encompassing the entire genome, and (iii) open reading frames coding for a novel protein superfamily, which we call the &#34;Oblins&#34;." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-31T10:38:28+00:00" />
<meta property="article:modified_time" content="2024-01-31T10:38:28+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Viroid-like colonists of human microbiomes
Authors: Zheludev, I. N.; Edgar, R. C.; Galiano, M. J. L.; De la Pena, M.; Babaian, A.; Bhatt, A. S.; Fire, A. Z.
Score: 385.7, Published: 2024-01-21 DOI: 10.1101/2024.01.20.576352
Here, we describe the &#34;Obelisks,&#34; a previously unrecognised class of viroid-like elements that we first identified in human gut metatranscriptomic data. &#34;Obelisks&#34; share several properties: (i) apparently circular RNA [~]1kb genome assemblies, (ii) predicted rod-like secondary structures encompassing the entire genome, and (iii) open reading frames coding for a novel protein superfamily, which we call the &#34;Oblins&#34;."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Viroid-like colonists of human microbiomes\nAuthors: Zheludev, I. N.; Edgar, R. C.; Galiano, M. J. L.; De la Pena, M.; Babaian, A.; Bhatt, A. S.; Fire, A. Z.\nScore: 385.7, Published: 2024-01-21 DOI: 10.1101/2024.01.20.576352\nHere, we describe the \u0026#34;Obelisks,\u0026#34; a previously unrecognised class of viroid-like elements that we first identified in human gut metatranscriptomic data. \u0026#34;Obelisks\u0026#34; share several properties: (i) apparently circular RNA [~]1kb genome assemblies, (ii) predicted rod-like secondary structures encompassing the entire genome, and (iii) open reading frames coding for a novel protein superfamily, which we call the \u0026#34;Oblins\u0026#34;.",
  "keywords": [
    
  ],
  "articleBody": " Viroid-like colonists of human microbiomes\nAuthors: Zheludev, I. N.; Edgar, R. C.; Galiano, M. J. L.; De la Pena, M.; Babaian, A.; Bhatt, A. S.; Fire, A. Z.\nScore: 385.7, Published: 2024-01-21 DOI: 10.1101/2024.01.20.576352\nHere, we describe the \"Obelisks,\" a previously unrecognised class of viroid-like elements that we first identified in human gut metatranscriptomic data. \"Obelisks\" share several properties: (i) apparently circular RNA [~]1kb genome assemblies, (ii) predicted rod-like secondary structures encompassing the entire genome, and (iii) open reading frames coding for a novel protein superfamily, which we call the \"Oblins\". We find that Obelisks form their own distinct phylogenetic group with no detectable sequence or structural similarity to known biological agents. Further, Obelisks are prevalent in tested human microbiome metatranscriptomes with representatives detected in [~]7% of analysed stool metatranscriptomes (29/440) and in [~]50% of analysed oral metatranscriptomes (17/32). Obelisk compositions appear to differ between the anatomic sites and are capable of persisting in individuals, with continued presence over \u003e300 days observed in one case. Large scale searches identified 29,959 Obelisks (clustered at 90% nucleotide identity), with examples from all seven continents and in diverse ecological niches. From this search, a subset of Obelisks are identified to code for Obelisk-specific variants of the hammerhead type-III self-cleaving ribozyme. Lastly, we identified one case of a bacterial species (Streptococcus sanguinis) in which a subset of defined laboratory strains harboured a specific Obelisk RNA population. As such, Obelisks comprise a class of diverse RNAs that have colonised, and gone unnoticed in, human, and global microbiomes.\nAn infection and pathogenesis mouse model of SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR)\nAuthors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.\nScore: 4291.9, Published: 2024-01-21 DOI: 10.1101/2024.01.03.574008\nSARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) is highly attenuated, but can cause mortality in a specifically designed human ACE2-transgenic mouse model, making it an invaluable surrogate model for evaluating the efficacy of drugs and vaccines against SARS-CoV-2.\nViral entry is a weak barrier to zoonosis\nAuthors: Dufloo, J.; Andreu-Moreno, I.; Valero-Rello, A.; Sanjuan, R.\nScore: 75.8, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576693\nRecent advances in viral metagenomics have led to the discovery of many mammalian viruses, but experimental tests to determine whether they pose a threat to humans are largely lacking. A first step for a virus to cross the species barrier is to penetrate host cells. Here, we use gene synthesis and viral pseudotyping to experimentally test the ability of viral receptor-binding proteins (RBPs) from \u003e100 enveloped RNA viruses to mediate entry into human cells. Analysis of thousands of RBP-cell pairs demonstrated such ability for most viruses, with significant variation among the 14 viral families studied. Comparison of RBP-mediated infectivity with cellular gene expression data showed that viral entry is often not limited by the presence of a receptor and revealed the contribution of additional host factors. Our results prove the weakness of interspecies barriers at the early stages of infection and identify molecular interactions that enable viral zoonosis.\nA Murine Model of Post-acute Neurological Sequelae Following SARS-CoV-2 Variant Infection\nAuthors: Singh, A.; Adam, A.; Aditi, A.; Peng, B.-H.; Yu, X.; Zou, J.; Kulkarni, V. V.; Kan, P.; Jiang, W.; Shi, P.-Y.; Samir, P.; Cisneros, I.; Wang, T.\nScore: 71.3, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574064\nViral variant is one known risk factor associated with post-acute sequelae of COVID-19 (PASC), yet the pathogenesis is largely unknown. Here, we studied SARS-CoV-2 Delta variant-induced PASC in K18-hACE2 mice. The virus replicated productively, induced robust inflammatory responses in lung and brain tissues, and caused weight loss and mortality during the acute infection. Longitudinal behavior studies in surviving mice up to 4 months post-acute infection revealed persistent abnormalities in neuropsychiatric state and motor behaviors, while reflex and sensory functions recovered over time. Surviving mice showed no detectable viral RNA in the brain and minimal neuroinflammation post-acute infection. Transcriptome analysis revealed persistent activation of immune pathways, including humoral responses, complement, and phagocytosis, and reduced levels of genes associated with ataxia telangiectasia, impaired cognitive function and memory recall, and neuronal dysfunction and degeneration. Furthermore, surviving mice maintained potent T helper 1 prone cellular immune responses and high neutralizing antibodies against Delta and Omicron variants in the periphery for months post-acute infection. Overall, infection in K18-hACE2 mice recapitulates the persistent clinical symptoms reported in long COVID patients and may be useful for future assessment of the efficacy of vaccines and therapeutics against SARS-CoV-2 variants.\nTemperature-dependent Spike-ACE2 interaction of Omicron subvariants is associated with viral transmission\nAuthors: Benlarbi, M.; Ding, S.; Belanger, E.; Tauzin, A.; Poujol, R.; Medjahed, H.; El Ferri, O.; Bo, Y.; Bourassa, C.; Hussin, J.; Fafard, J.; Pazgier, M.; Levade, I.; Abrams, C.; Cote, M.; Finzi, A.\nScore: 62.9, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576353\nThe continued evolution of SARS-CoV-2 requires persistent monitoring of its subvariants. Omicron subvariants are responsible for the vast majority of SARS-CoV-2 infections worldwide, with XBB and BA.2.86 sublineages representing more than 90% of circulating strains as of January 2024. In this study, we characterized the functional properties of Spike glycoproteins from BA.2.75, CH.1.1, DV.7.1, BA.4/5, BQ.1.1, XBB, XBB.1, XBB.1.16, XBB.1.5, FD.1.1, EG.5.1, HK.3 BA.2.86 and JN.1. We tested their capacity to evade plasma-mediated recognition and neutralization, ACE2 binding, their susceptibility to cold inactivation, Spike processing, as well as the impact of temperature on Spike-ACE2 interaction. We found that compared to the early wild-type (D614G) strain, most Omicron subvariants Spike glycoproteins evolved to escape recognition and neutralization by plasma from individuals who received a fifth dose of bivalent (BA.1 or BA.4/5) mRNA vaccine and improve ACE2 binding, particularly at low temperatures. Moreover, BA.2.86 had the best affinity for ACE2 at all temperatures tested. We found that Omicron subvariants Spike processing is associated with their susceptibility to cold inactivation. Intriguingly, we found that Spike-ACE2 binding at low temperature was significantly associated with growth rates of Omicron subvariants in humans. Overall, we report that Spikes from newly emerged Omicron subvariants are relatively more stable and resistant to plasma-mediated neutralization, present improved affinity for ACE2 which is associated, particularly at low temperatures, with their growth rates.\nAssembly, stability, and dynamics of the infant gut microbiome are linked to bacterial strains and functions in mother's milk\nAuthors: Allert, M.; Ferretti, P.; Johnson, K.; Heisel, T.; Gonia, S.; Knights, D.; Fields, D. A.; Albert, F. W.; Demerath, E. W.; Gale, C. A.; Blekhman, R.\nScore: 58.4, Published: 2024-01-28 DOI: 10.1101/2024.01.28.577594\nThe establishment of the gut microbiome in early life is critical for healthy infant development. Although human milk is recommended as the sole source of nutrition for the human infant, little is known about how variation in milk composition, and especially the milk microbiome, shapes the microbial communities in the infant gut. Here, we quantified the similarity between the maternal milk and the infant gut microbiome using 507 metagenomic samples collected from 195 mother-infant pairs at one, three, and six months postpartum. We found that the microbial taxonomic overlap between milk and the infant gut was driven by bifidobacteria, in particular by B. longum. Infant stool samples dominated by B. longum also showed higher temporal stability compared to samples dominated by other species. We identified two instances of strain sharing between maternal milk and the infant gut, one involving a commensal (B. longum ) and one a pathobiont (K. pneumoniae). In addition, strain sharing between unrelated infants was higher among infants born at the same hospital compared to infants born in different hospitals, suggesting a potential role of the hospital environment in shaping the infant gut microbiome composition. The infant gut microbiome at one month compared to six months of age was enriched in metabolic pathways associated with de-novo molecule biosynthesis, suggesting that early colonisers might be more versatile and metabolically independent compared to later colonizers. Lastly, we found a significant overlap in antimicrobial resistance genes carriage between the mother's milk and their infant's gut microbiome. Taken together, our results suggest that the human milk microbiome has an important role in the assembly, composition, and stability of the infant gut microbiome.\nMechanism of bacterial predation via ixotrophy\nAuthors: Lien, Y.-W.; Amendola, D.; Lee, K. S.; Bartlau, N.; Xu, J.; Furusawa, G.; Polz, M. F.; Stocker, R.; Weiss, G. L.; PIlhofer, M.\nScore: 58.0, Published: 2024-01-29 DOI: 10.1101/2024.01.29.577165\nPredation allows bacteria to access alternative substrates in low-nutrient conditions. Ixotrophy has been proposed as a predatory lifestyle of multicellular filamentous bacteria in aquatic environments; however, the molecular mechanism remains unknown. Here we uncover by a multidisciplinary approach that ixotrophy requires the interplay of multiple cellular machineries and a regulatory mechanism. Attacker-prey contacts are established by gliding motility and extracellular grappling hook-like structures that bind prey flagella. Cryo-electron microscopy identifies the grappling hooks as a heptameric assembly of a Type 9 Secretion System substrate. Cryo-electron tomography and functional assays show that killing is mediated by puncturing of the prey cell using a Type 6 Secretion System, possibly triggered by extracellular antennae. Single-cell analyses with stable isotope-labeled prey demonstrate that prey components are taken up by the attacker. Depending on nutrient availability, ixotrophy is switched off by endogenous Insertion Sequence Elements and re-activated through their excision. A marine metagenomic time series provides evidence for coupled dynamics of ixotrophic bacteria and their prey. Our study reveals the complex mechanism of a conserved microbial predatory lifestyle and indicates the need for its regulation in conditions where the expression of costly pathways is dispensable.\nBirth of new protein folds and functions in the virome\nAuthors: Nomburg, J.; Price, N.; Doudna, J. A.\nScore: 47.8, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576744\nRapid virus evolution generates proteins essential to infectivity and replication but with unknown function due to extreme sequence divergence1. Using a database of 67,715 newly predicted protein structures from 4,463 eukaryotic viral species, we found that 62% of viral proteins are evolutionarily young and lack homologs in the Alphafold database2,3. Among the 38% of more ancient viral proteins, many have non-viral structural homologs that revealed surprising similarities between human pathogens and their eukaryotic hosts. Structural comparisons suggested putative functions for \u003e25% of unannotated viral proteins, including those with roles in the evasion of innate immunity. In particular, RNA ligase T- (ligT) like phosphodiesterases were found to resemble phage-encoded proteins that hydrolyze the host immune-activating cyclic dinucleotides 33 and 23 cyclic G-A monophosphate (cGAMP). Experimental analysis showed that ligT homologs encoded by avian poxviruses likewise hydrolyze 23 cGAMP, showing that ligT-mediated targeting of cGAMP is an evolutionarily conserved mechanism of immune evasion present in both bacteriophage and eukaryotic viruses. Together, the viral protein structural database and analytics presented here afford new opportunities to identify mechanisms of virus-host interactions that are common across the virome.\nMultiple layers of innate immune response antagonism of SARS-CoV-2\nAuthors: Zhou, F.; Periasamy, S.; Jackson, N.; Cheng, W. S.; Soto Acosta, R.; Ilinykh, P.; Ye, C.; Chauhan, S.; Nudelman, G.; Zaslavsky, E.; Widen, S.; Martinez-Sobrido, L.; Sealfon, S.; Bukreyev, A.\nScore: 26.6, Published: 2024-01-29 DOI: 10.1101/2024.01.29.577695\nSeveral SARS-CoV-2 proteins have been shown to counteract the host innate immune response, mostly using in vitro protein expression, which may not fully reflect their role in the context of viral infection. In addition, while each viral protein was characterized in a different experimental system, their relative contribution in immunosuppression remains unclear. Here we used a SARS-CoV-2 bacterial artificial chromosome with en passant mutagenesis to recover a panel of twelve infectious recombinant SARS-CoV-2 viruses, each with mutations in either NSP1, NSP2, NSP3, NSP6, NSP12, NSP13, NSP14, NSP15, NSP16, ORF3a, ORF6 or ORF8. We used the interferon-stimulated response element (ISRE)-driven luciferase assay in 293T-ACE2/TMPRSS2 cells to test the panel, demonstrating that mutations in many proteins, especially in NSP1 and NSP15, increased the type I interferon response relative to the parental wild-type virus. RNA-seq analysis of mutant-virus infected Calu-3 cells showed that the mutations in NSP1 or NSP15 lead to higher expression of multiple genes involved in innate immune response, cytokine-mediated signaling and regulation of lymphocyte proliferation. Furthermore, mutations in either NSP1 or NSP15 resulted in a greater maturation of human monocyte-derived dendritic cells in vitro. Infection of K18 hACE2 transgenic mice with either NSP1 or NSP15 mutated viruses demonstrated attentuated respiratory tract replication. Analysis of lung immune cells from infected mice by single-cell RNA-seq identified 15 populations of major myeloid and lymphoid cells with changes in the pattern of their activation associated with viral infection. The effects of mutations in NSP1 or NSP15 on these responses are consistent with differences in the immunosuppressive mechanisms utilized by the two proteins. Overall, these data demonstrate different and redundant mechanisms of innate immune antagonism by SARS-CoV-2 and suppression of activation of antigen presenting cells and T and B lymphocytes mediated by multiple viral proteins.\nExtracellular ATP is a signalling molecule in bacteria\nAuthors: Tronnet, S.; Pandey, V.; Lloret-Berrocal, M.; Perez-del-Pozo, M.; Soderholm, N.; Hernandez-Ortego, C.; Billker, O.; Nordstrom, A.; Puhar, A.\nScore: 21.4, Published: 2024-01-25 DOI: 10.1101/2024.01.24.577065\nIn animals and plants extracellular ATP (eATP) functions as signalling molecule and regulates the immune response. During inflammation intestinal bacteria are exposed to elevated eATP originating from the mucosa. Whether bacteria respond to eATP is unclear. Here we show that non-pathogenic Escherichia coli responds to eATP at physiologically relevant concentrations by modifying its transcriptional and metabolic landscape. The use of a promoter library showed that the response to eATP is time-, dose- and medium-dependent. Genes related to lipid, amino acid, or vitamin metabolism were regulated. Metabolomics showed that eATP triggers enrichment of molecules with bioactive properties on the host or bacteria. Combined genome-scale modelling highlighted the global metabolic modifications. Moreover, eATP altered the sensitivity to antibiotics and antimicrobial peptides. Finally, in pathogens eATP controls the expression of virulence and fitness factors. Our results indicate that eATP is a signalling molecule in prokaryotes which globally regulates physiology, antimicrobial resistance, and virulence.\n",
  "wordCount" : "2280",
  "inLanguage": "en",
  "datePublished": "2024-01-31T10:38:28Z",
  "dateModified": "2024-01-31T10:38:28Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 31, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576352">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576352" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576352">
        <p class="paperTitle">Viroid-like colonists of human microbiomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576352" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576352" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zheludev, I. N.; Edgar, R. C.; Galiano, M. J. L.; De la Pena, M.; Babaian, A.; Bhatt, A. S.; Fire, A. Z.</p>
        <p class="info">Score: 385.7, Published: 2024-01-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576352' target='https://doi.org/10.1101/2024.01.20.576352'> 10.1101/2024.01.20.576352</a></p>
        <p class="abstract">Here, we describe the &#34;Obelisks,&#34; a previously unrecognised class of viroid-like elements that we first identified in human gut metatranscriptomic data. &#34;Obelisks&#34; share several properties: (i) apparently circular RNA [~]1kb genome assemblies, (ii) predicted rod-like secondary structures encompassing the entire genome, and (iii) open reading frames coding for a novel protein superfamily, which we call the &#34;Oblins&#34;. We find that Obelisks form their own distinct phylogenetic group with no detectable sequence or structural similarity to known biological agents. Further, Obelisks are prevalent in tested human microbiome metatranscriptomes with representatives detected in [~]7% of analysed stool metatranscriptomes (29/440) and in [~]50% of analysed oral metatranscriptomes (17/32). Obelisk compositions appear to differ between the anatomic sites and are capable of persisting in individuals, with continued presence over &gt;300 days observed in one case. Large scale searches identified 29,959 Obelisks (clustered at 90% nucleotide identity), with examples from all seven continents and in diverse ecological niches. From this search, a subset of Obelisks are identified to code for Obelisk-specific variants of the hammerhead type-III self-cleaving ribozyme. Lastly, we identified one case of a bacterial species (Streptococcus sanguinis) in which a subset of defined laboratory strains harboured a specific Obelisk RNA population. As such, Obelisks comprise a class of diverse RNAs that have colonised, and gone unnoticed in, human, and global microbiomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574008">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574008" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574008">
        <p class="paperTitle">An infection and pathogenesis mouse model of SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574008" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574008" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.</p>
        <p class="info">Score: 4291.9, Published: 2024-01-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574008' target='https://doi.org/10.1101/2024.01.03.574008'> 10.1101/2024.01.03.574008</a></p>
        <p class="abstract">SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) is highly attenuated, but can cause mortality in a specifically designed human ACE2-transgenic mouse model, making it an invaluable surrogate model for evaluating the efficacy of drugs and vaccines against SARS-CoV-2.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.576693">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.576693" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.576693">
        <p class="paperTitle">Viral entry is a weak barrier to zoonosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.576693" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.576693" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dufloo, J.; Andreu-Moreno, I.; Valero-Rello, A.; Sanjuan, R.</p>
        <p class="info">Score: 75.8, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.576693' target='https://doi.org/10.1101/2024.01.22.576693'> 10.1101/2024.01.22.576693</a></p>
        <p class="abstract">Recent advances in viral metagenomics have led to the discovery of many mammalian viruses, but experimental tests to determine whether they pose a threat to humans are largely lacking. A first step for a virus to cross the species barrier is to penetrate host cells. Here, we use gene synthesis and viral pseudotyping to experimentally test the ability of viral receptor-binding proteins (RBPs) from &gt;100 enveloped RNA viruses to mediate entry into human cells. Analysis of thousands of RBP-cell pairs demonstrated such ability for most viruses, with significant variation among the 14 viral families studied. Comparison of RBP-mediated infectivity with cellular gene expression data showed that viral entry is often not limited by the presence of a receptor and revealed the contribution of additional host factors. Our results prove the weakness of interspecies barriers at the early stages of infection and identify molecular interactions that enable viral zoonosis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574064">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574064" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574064">
        <p class="paperTitle">A Murine Model of Post-acute Neurological Sequelae Following SARS-CoV-2 Variant Infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574064" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574064" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Singh, A.; Adam, A.; Aditi, A.; Peng, B.-H.; Yu, X.; Zou, J.; Kulkarni, V. V.; Kan, P.; Jiang, W.; Shi, P.-Y.; Samir, P.; Cisneros, I.; Wang, T.</p>
        <p class="info">Score: 71.3, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574064' target='https://doi.org/10.1101/2024.01.03.574064'> 10.1101/2024.01.03.574064</a></p>
        <p class="abstract">Viral variant is one known risk factor associated with post-acute sequelae of COVID-19 (PASC), yet the pathogenesis is largely unknown. Here, we studied SARS-CoV-2 Delta variant-induced PASC in K18-hACE2 mice. The virus replicated productively, induced robust inflammatory responses in lung and brain tissues, and caused weight loss and mortality during the acute infection. Longitudinal behavior studies in surviving mice up to 4 months post-acute infection revealed persistent abnormalities in neuropsychiatric state and motor behaviors, while reflex and sensory functions recovered over time. Surviving mice showed no detectable viral RNA in the brain and minimal neuroinflammation post-acute infection. Transcriptome analysis revealed persistent activation of immune pathways, including humoral responses, complement, and phagocytosis, and reduced levels of genes associated with ataxia telangiectasia, impaired cognitive function and memory recall, and neuronal dysfunction and degeneration. Furthermore, surviving mice maintained potent T helper 1 prone cellular immune responses and high neutralizing antibodies against Delta and Omicron variants in the periphery for months post-acute infection. Overall, infection in K18-hACE2 mice recapitulates the persistent clinical symptoms reported in long COVID patients and may be useful for future assessment of the efficacy of vaccines and therapeutics against SARS-CoV-2 variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576353">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576353" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576353">
        <p class="paperTitle">Temperature-dependent Spike-ACE2 interaction of Omicron subvariants is associated with viral transmission</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576353" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576353" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Benlarbi, M.; Ding, S.; Belanger, E.; Tauzin, A.; Poujol, R.; Medjahed, H.; El Ferri, O.; Bo, Y.; Bourassa, C.; Hussin, J.; Fafard, J.; Pazgier, M.; Levade, I.; Abrams, C.; Cote, M.; Finzi, A.</p>
        <p class="info">Score: 62.9, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576353' target='https://doi.org/10.1101/2024.01.20.576353'> 10.1101/2024.01.20.576353</a></p>
        <p class="abstract">The continued evolution of SARS-CoV-2 requires persistent monitoring of its subvariants. Omicron subvariants are responsible for the vast majority of SARS-CoV-2 infections worldwide, with XBB and BA.2.86 sublineages representing more than 90% of circulating strains as of January 2024. In this study, we characterized the functional properties of Spike glycoproteins from BA.2.75, CH.1.1, DV.7.1, BA.4/5, BQ.1.1, XBB, XBB.1, XBB.1.16, XBB.1.5, FD.1.1, EG.5.1, HK.3 BA.2.86 and JN.1. We tested their capacity to evade plasma-mediated recognition and neutralization, ACE2 binding, their susceptibility to cold inactivation, Spike processing, as well as the impact of temperature on Spike-ACE2 interaction. We found that compared to the early wild-type (D614G) strain, most Omicron subvariants Spike glycoproteins evolved to escape recognition and neutralization by plasma from individuals who received a fifth dose of bivalent (BA.1 or BA.4/5) mRNA vaccine and improve ACE2 binding, particularly at low temperatures. Moreover, BA.2.86 had the best affinity for ACE2 at all temperatures tested. We found that Omicron subvariants Spike processing is associated with their susceptibility to cold inactivation. Intriguingly, we found that Spike-ACE2 binding at low temperature was significantly associated with growth rates of Omicron subvariants in humans. Overall, we report that Spikes from newly emerged Omicron subvariants are relatively more stable and resistant to plasma-mediated neutralization, present improved affinity for ACE2 which is associated, particularly at low temperatures, with their growth rates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.28.577594">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.28.577594" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.28.577594">
        <p class="paperTitle">Assembly, stability, and dynamics of the infant gut microbiome are linked to bacterial strains and functions in mother&#39;s milk</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.28.577594" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.28.577594" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Allert, M.; Ferretti, P.; Johnson, K.; Heisel, T.; Gonia, S.; Knights, D.; Fields, D. A.; Albert, F. W.; Demerath, E. W.; Gale, C. A.; Blekhman, R.</p>
        <p class="info">Score: 58.4, Published: 2024-01-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.28.577594' target='https://doi.org/10.1101/2024.01.28.577594'> 10.1101/2024.01.28.577594</a></p>
        <p class="abstract">The establishment of the gut microbiome in early life is critical for healthy infant development. Although human milk is recommended as the sole source of nutrition for the human infant, little is known about how variation in milk composition, and especially the milk microbiome, shapes the microbial communities in the infant gut. Here, we quantified the similarity between the maternal milk and the infant gut microbiome using 507 metagenomic samples collected from 195 mother-infant pairs at one, three, and six months postpartum. We found that the microbial taxonomic overlap between milk and the infant gut was driven by bifidobacteria, in particular by B. longum. Infant stool samples dominated by B. longum also showed higher temporal stability compared to samples dominated by other species. We identified two instances of strain sharing between maternal milk and the infant gut, one involving a commensal (B. longum ) and one a pathobiont (K. pneumoniae). In addition, strain sharing between unrelated infants was higher among infants born at the same hospital compared to infants born in different hospitals, suggesting a potential role of the hospital environment in shaping the infant gut microbiome composition. The infant gut microbiome at one month compared to six months of age was enriched in metabolic pathways associated with de-novo molecule biosynthesis, suggesting that early colonisers might be more versatile and metabolically independent compared to later colonizers. Lastly, we found a significant overlap in antimicrobial resistance genes carriage between the mother&#39;s milk and their infant&#39;s gut microbiome. Taken together, our results suggest that the human milk microbiome has an important role in the assembly, composition, and stability of the infant gut microbiome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577165">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577165" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577165">
        <p class="paperTitle">Mechanism of bacterial predation via ixotrophy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577165" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577165" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lien, Y.-W.; Amendola, D.; Lee, K. S.; Bartlau, N.; Xu, J.; Furusawa, G.; Polz, M. F.; Stocker, R.; Weiss, G. L.; PIlhofer, M.</p>
        <p class="info">Score: 58.0, Published: 2024-01-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577165' target='https://doi.org/10.1101/2024.01.29.577165'> 10.1101/2024.01.29.577165</a></p>
        <p class="abstract">Predation allows bacteria to access alternative substrates in low-nutrient conditions. Ixotrophy has been proposed as a predatory lifestyle of multicellular filamentous bacteria in aquatic environments; however, the molecular mechanism remains unknown. Here we uncover by a multidisciplinary approach that ixotrophy requires the interplay of multiple cellular machineries and a regulatory mechanism. Attacker-prey contacts are established by gliding motility and extracellular grappling hook-like structures that bind prey flagella. Cryo-electron microscopy identifies the grappling hooks as a heptameric assembly of a Type 9 Secretion System substrate. Cryo-electron tomography and functional assays show that killing is mediated by puncturing of the prey cell using a Type 6 Secretion System, possibly triggered by extracellular antennae. Single-cell analyses with stable isotope-labeled prey demonstrate that prey components are taken up by the attacker. Depending on nutrient availability, ixotrophy is switched off by endogenous Insertion Sequence Elements and re-activated through their excision. A marine metagenomic time series provides evidence for coupled dynamics of ixotrophic bacteria and their prey. Our study reveals the complex mechanism of a conserved microbial predatory lifestyle and indicates the need for its regulation in conditions where the expression of costly pathways is dispensable.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.576744">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.576744" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.576744">
        <p class="paperTitle">Birth of new protein folds and functions in the virome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.576744" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.576744" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nomburg, J.; Price, N.; Doudna, J. A.</p>
        <p class="info">Score: 47.8, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.576744' target='https://doi.org/10.1101/2024.01.22.576744'> 10.1101/2024.01.22.576744</a></p>
        <p class="abstract">Rapid virus evolution generates proteins essential to infectivity and replication but with unknown function due to extreme sequence divergence1. Using a database of 67,715 newly predicted protein structures from 4,463 eukaryotic viral species, we found that 62% of viral proteins are evolutionarily young and lack homologs in the Alphafold database2,3. Among the 38% of more ancient viral proteins, many have non-viral structural homologs that revealed surprising similarities between human pathogens and their eukaryotic hosts. Structural comparisons suggested putative functions for &gt;25% of unannotated viral proteins, including those with roles in the evasion of innate immunity. In particular, RNA ligase T- (ligT) like phosphodiesterases were found to resemble phage-encoded proteins that hydrolyze the host immune-activating cyclic dinucleotides 33 and 23 cyclic G-A monophosphate (cGAMP). Experimental analysis showed that ligT homologs encoded by avian poxviruses likewise hydrolyze 23 cGAMP, showing that ligT-mediated targeting of cGAMP is an evolutionarily conserved mechanism of immune evasion present in both bacteriophage and eukaryotic viruses. Together, the viral protein structural database and analytics presented here afford new opportunities to identify mechanisms of virus-host interactions that are common across the virome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577695">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577695" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577695">
        <p class="paperTitle">Multiple layers of innate immune response antagonism of SARS-CoV-2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577695" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577695" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhou, F.; Periasamy, S.; Jackson, N.; Cheng, W. S.; Soto Acosta, R.; Ilinykh, P.; Ye, C.; Chauhan, S.; Nudelman, G.; Zaslavsky, E.; Widen, S.; Martinez-Sobrido, L.; Sealfon, S.; Bukreyev, A.</p>
        <p class="info">Score: 26.6, Published: 2024-01-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577695' target='https://doi.org/10.1101/2024.01.29.577695'> 10.1101/2024.01.29.577695</a></p>
        <p class="abstract">Several SARS-CoV-2 proteins have been shown to counteract the host innate immune response, mostly using in vitro protein expression, which may not fully reflect their role in the context of viral infection. In addition, while each viral protein was characterized in a different experimental system, their relative contribution in immunosuppression remains unclear. Here we used a SARS-CoV-2 bacterial artificial chromosome with en passant mutagenesis to recover a panel of twelve infectious recombinant SARS-CoV-2 viruses, each with mutations in either NSP1, NSP2, NSP3, NSP6, NSP12, NSP13, NSP14, NSP15, NSP16, ORF3a, ORF6 or ORF8. We used the interferon-stimulated response element (ISRE)-driven luciferase assay in 293T-ACE2/TMPRSS2 cells to test the panel, demonstrating that mutations in many proteins, especially in NSP1 and NSP15, increased the type I interferon response relative to the parental wild-type virus. RNA-seq analysis of mutant-virus infected Calu-3 cells showed that the mutations in NSP1 or NSP15 lead to higher expression of multiple genes involved in innate immune response, cytokine-mediated signaling and regulation of lymphocyte proliferation. Furthermore, mutations in either NSP1 or NSP15 resulted in a greater maturation of human monocyte-derived dendritic cells in vitro. Infection of K18 hACE2 transgenic mice with either NSP1 or NSP15 mutated viruses demonstrated attentuated respiratory tract replication. Analysis of lung immune cells from infected mice by single-cell RNA-seq identified 15 populations of major myeloid and lymphoid cells with changes in the pattern of their activation associated with viral infection. The effects of mutations in NSP1 or NSP15 on these responses are consistent with differences in the immunosuppressive mechanisms utilized by the two proteins. Overall, these data demonstrate different and redundant mechanisms of innate immune antagonism by SARS-CoV-2 and suppression of activation of antigen presenting cells and T and B lymphocytes mediated by multiple viral proteins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.577065">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.577065" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.577065">
        <p class="paperTitle">Extracellular ATP is a signalling molecule in bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.577065" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.577065" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tronnet, S.; Pandey, V.; Lloret-Berrocal, M.; Perez-del-Pozo, M.; Soderholm, N.; Hernandez-Ortego, C.; Billker, O.; Nordstrom, A.; Puhar, A.</p>
        <p class="info">Score: 21.4, Published: 2024-01-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.577065' target='https://doi.org/10.1101/2024.01.24.577065'> 10.1101/2024.01.24.577065</a></p>
        <p class="abstract">In animals and plants extracellular ATP (eATP) functions as signalling molecule and regulates the immune response. During inflammation intestinal bacteria are exposed to elevated eATP originating from the mucosa. Whether bacteria respond to eATP is unclear. Here we show that non-pathogenic Escherichia coli responds to eATP at physiologically relevant concentrations by modifying its transcriptional and metabolic landscape. The use of a promoter library showed that the response to eATP is time-, dose- and medium-dependent. Genes related to lipid, amino acid, or vitamin metabolism were regulated. Metabolomics showed that eATP triggers enrichment of molecules with bioactive properties on the host or bacteria. Combined genome-scale modelling highlighted the global metabolic modifications. Moreover, eATP altered the sensitivity to antibiotics and antimicrobial peptides. Finally, in pathogens eATP controls the expression of virulence and fitness factors. Our results indicate that eATP is a signalling molecule in prokaryotes which globally regulates physiology, antimicrobial resistance, and virulence.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
