<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Lethal Infection of Human ACE2-Transgenic Mice Caused by SARS-CoV-2-related Pangolin Coronavirus GX_P2V(short_3UTR)
Authors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.
Score: 4033.2, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574008
SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) can cause 100% mortality in human ACE2-transgenic mice, potentially attributable to late-stage brain infection. This underscores a spillover risk of GX_P2V into humans and provides a unique model for understanding the pathogenic mechanisms of SARS-CoV-2-related viruses.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Lethal Infection of Human ACE2-Transgenic Mice Caused by SARS-CoV-2-related Pangolin Coronavirus GX_P2V(short_3UTR)
Authors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.
Score: 4033.2, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574008
SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) can cause 100% mortality in human ACE2-transgenic mice, potentially attributable to late-stage brain infection. This underscores a spillover risk of GX_P2V into humans and provides a unique model for understanding the pathogenic mechanisms of SARS-CoV-2-related viruses." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-21T10:39:08+00:00" />
<meta property="article:modified_time" content="2024-01-21T10:39:08+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Lethal Infection of Human ACE2-Transgenic Mice Caused by SARS-CoV-2-related Pangolin Coronavirus GX_P2V(short_3UTR)
Authors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.
Score: 4033.2, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574008
SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) can cause 100% mortality in human ACE2-transgenic mice, potentially attributable to late-stage brain infection. This underscores a spillover risk of GX_P2V into humans and provides a unique model for understanding the pathogenic mechanisms of SARS-CoV-2-related viruses."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Lethal Infection of Human ACE2-Transgenic Mice Caused by SARS-CoV-2-related Pangolin Coronavirus GX_P2V(short_3UTR)\nAuthors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.\nScore: 4033.2, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574008\nSARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) can cause 100% mortality in human ACE2-transgenic mice, potentially attributable to late-stage brain infection. This underscores a spillover risk of GX_P2V into humans and provides a unique model for understanding the pathogenic mechanisms of SARS-CoV-2-related viruses.",
  "keywords": [
    
  ],
  "articleBody": " Lethal Infection of Human ACE2-Transgenic Mice Caused by SARS-CoV-2-related Pangolin Coronavirus GX_P2V(short_3UTR)\nAuthors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.\nScore: 4033.2, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574008\nSARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) can cause 100% mortality in human ACE2-transgenic mice, potentially attributable to late-stage brain infection. This underscores a spillover risk of GX_P2V into humans and provides a unique model for understanding the pathogenic mechanisms of SARS-CoV-2-related viruses.\nNRX-101 (D-Cycloserine + Lurasidone) , a Qualified Infectious Disease Product, is Active Against Drug-Resistant Urinary Pathogens In Vitro\nAuthors: Sapko, M. T.; Manyak, M.; Panicucci, R.; Javitt, J. C.\nScore: 58.5, Published: 2024-01-14 DOI: 10.1101/2024.01.14.575572\nBackgroundD-Cycloserine (DCS) is a broad-spectrum antibiotic that is currently FDA-approved to treat tuberculosis (TB) disease and urinary tract infection. Despite numerous reports showing good clinical efficacy, DCS fell out of favor as a UTI treatment because of its propensity to cause side effects. NRX-101, a fixed dose combination of DCS and lurasidone, has been awarded Qualified Infectious Disease Product and Fast Track Designation by the US Food and Drug Administration and is being developed for various CNS indications because of its unique synergistic effect; each component mitigates side effects of the other. MethodsIn this study, we tested NRX-101 against the urinary tract pathogens E. coli, P. aeruginosa, K. pneumoniae, and A. baumannii in Mueller Hinton broth (caMHB) and artificial urine media (AUM). Several strains were multidrug resistant. Test compounds were serially diluted in broth/media. Minimum inhibitory concentration (MIC) was defined as the lowest concentration of test compound at which no bacterial growth was observed. ResultsDCS exhibited antibacterial efficacy against all strains tested while lurasidone did not appreciably affect the antibacterial action of DCS in vitro. In AUM, the MICs ranged from 128 to 512 mcg/ml for both DCS and NRX-101. In caMHB, MICs ranged from 8 to 1024 mcg/ml for NRX-101 and 32 to 512 mcg/ml for DCS alone. ConclusionsOur data confirm that DCS as antibacterial activity against reference and drug-resistant urinary pathogens. Furthermore, lurasidone does not interfere with DCSs anti-microbial action in vitro. These results support the clinical development of NRX-101 as a treatment for complicated urinary tract infection.\nA treasure trove of 1,034 actinomycete genomes\nAuthors: Jorgensen, T. S.; Mohite, O.; Sterndorff, E. B.; Alvarez-Arevalo, M.; Blin, K.; Booth, T. J.; Charusanti, P.; Faurdal, D.; Hansen, T. O.; Nuhamunada, M. J.; Mourched, A.-S.; Palsson, B. O.; Weber, T.\nScore: 45.1, Published: 2024-01-16 DOI: 10.1101/2024.01.16.574955\nFilamentous Actinobacteria, recently renamed Actinomycetia, are the most prolific source of microbial bioactive natural products. Studies on biosynthetic gene clusters benefit from or require chromosome-level assemblies. Here, we provide DNA sequences from more than 1,000 isolates: 881 complete genomes and 153 near-complete genomes, representing 28 genera and 389 species, including 244 likely novel species. All genomes are from filamentous isolates of the class Actinomycetia from the NBC culture collection. The largest genus is Streptomyces with 886 genomes including 742 complete assemblies. We use this data to show that analysis of complete genomes can bring biological understanding not previously derived from more fragmented sequences or less systematic datasets. We document the central and structured location of core genes and distal location of specialized metabolite biosynthetic gene clusters and duplicate core genes on the linear Streptomyces chromosome, and analyze the content and length of the terminal inverted repeats which are characteristic for Streptomyces. We then analyze the diversity of trans-AT polyketide synthase biosynthetic gene clusters, which encodes the machinery of a biotechnologically highly interesting compound class. These insights have both ecological and biotechnological implications in understanding the importance of high quality genomic resources and the complex role synteny plays in Actinomycetia biology.\nSuperiority of chromosomal compared to plasmid-encoded compensatory mutations\nAuthors: Wright, R. C. T.; Wood, J.; Bottery, M. J.; Muddiman, K. J.; Paterson, S.; Harrison, E.; Brockhurst, M. A.; Hall, J. P. J.\nScore: 13.9, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575717\nPlasmids are important vectors of horizontal gene transfer in microbial communities but can impose a burden on the bacteria that carry them. Such plasmid fitness costs are thought to arise principally from conflicts between chromosomal- and plasmid-encoded molecular machineries, and thus can be ameliorated by compensatory mutations (CMs) that reduce or resolve the underlying causes. CMs can arise on plasmids (i.e. plaCM) or on chromosomes (i.e. chrCM), with contrasting predicted effects upon plasmid success and subsequent gene transfer because plaCM can also reduce fitness costs in plasmid recipients, whereas chrCM can potentially ameliorate multiple distinct plasmids. Here, we develop theory and a novel experimental system to directly compare the ecological effects of plaCM and chrCM that arose during evolution experiments between Pseudomonas fluorescens SBW25 and its sympatric mercury resistance megaplasmid pQBR57. We show that while plaCM was predicted to succeed under a broader range of parameters in mathematical models, experimentally chrCM dominated under all conditions, including those with numerous recipients, due to a more efficacious mechanism of compensation, and advantages arising from transmission of costly plasmids to competitors (plasmid weaponisation). We show analytically the presence of a mixed Rock-Paper-Scissors regime for plaCM, driven by trade-offs with horizontal transmission, that explains the observed failure of plaCM to dominate even in competition against an uncompensated plasmid. Our results reveal broader implications of plasmid-bacterial evolution for plasmid ecology, demonstrating the importance of compensatory mutations for resistance gene spread. One consequence of the superiority of chrCM over plaCM is the likely emergence in microbial communities of compensated bacteria that can act as hubs for plasmid accumulation and dissemination.\nNitrous oxide respiration in acidophilic methanotrophs\nAuthors: Awala, S. I.; Gwak, J.-H.; Kim, Y.; Jung, M.-Y.; Dunfield, P. F.; Wagner, M.; Rhee, S.-K.\nScore: 12.4, Published: 2024-01-16 DOI: 10.1101/2024.01.15.574570\nMethanotrophic bacteria mitigate methane (CH4) emissions from natural environments. Although aerobic methanotrophs are considered strict aerobes, they are often highly abundant in extremely hypoxic and even anoxic environments. Despite the presence of denitrification genes, it remains to be verified whether denitrification contributes to their growth. Here, we revealed that two acidophilic methanotrophs encoding N2O reductase (clade I and type II nosZ, respectively): Methylocella tundrae T4 and Methylacidiphilum caldifontis IT6, respired N2O and grew anaerobically on diverse non-methane substrates, including methanol, C-C substrates, and hydrogen. However, NO3 - and NO2 - could be reduced during methanol oxidation in Methylocella tundrae T4 and Methylocella silvestris BL2 without significantly increasing cell biomass. The lack of growth on methanol + NO3- or NO2- was likely due to the production of toxic reactive nitrogen species and C1 metabolites. However, the oxidation of pyruvate, a C3 electron donor, combined with NO3- or NO2- reduction resulted in anaerobic growth of Methylocella tundrae T4 and Methylocella silvestris BL2. In the extreme acidophile, Methylacidiphilum caldifontis IT6, N2O respiration supported cell growth at an extremely acidic pH of 2.0. In Methylocella tundrae T4, simultaneous consumption of N2O and CH4 was observed in suboxic conditions, both in microrespirometry and growth experiments, indicating the robustness of its N2O reductase activity in the presence of O2. Furthermore, CH4 oxidation per O2 reduced in O2-limiting conditions increased when N2O was added, indicating that cells of T4 can direct more O2 towards methane monooxygenase when respiring N2O as a terminal electron acceptor. Upregulation of nosZ and distinct repertories of methanol dehydrogenase-encoding genes (XoxF- and MxaFI-type) in Methylocella tundrae T4 cells grown anaerobically on methanol with N2O as the sole electron acceptor indicated adaptation mechanisms to anoxia. Our findings demonstrate that some methanotrophs can respire N2O independently or in tandem with O2, significantly expanding their potential ecological niche and paving the way for enhanced growth and survival in dynamic environments. This metabolic capability has application potential for simultaneously mitigating the emissions of the key greenhouse gases, CO2, CH4, and N2O, from natural and engineered environments.\nOn the ecology of Acinetobacter baumannii - jet stream rider and opportunist by nature\nAuthors: Wilharm, G.; Skiebe, E.; Lopinska, A.; Higgins, P. G.; Weber, K.; Schaudinn, C.; Neugebauer, C.; Goerlitz, K.; Meimers, G.; Rizova, Y.; Blaschke, U.; Heider, C.; Cuny, C.; Drewes, S.; Heuser, E.; Jeske, K.; Jacob, J.; Ulrich, R. G.; Bochenski, M.; Kasprzak, M.; Burda, E.; Cieplinski, M.; Kaluga, I.; Jankowiak, L.; Aguirre, J. I.; Lopez-Garcia, A.; Hoefle, U.; Jagiello, Z.; Tobolka, M.; Janic, B.; Zielinski, P.; Kaminski, M.; Frisch, J.; Siekiera, J.; Wendel, A. F.; Brauner, P.; Jaeckel, U.; Kaatz, M.; Mueller, S.; Luebke-Becker, A.; Wieler, L. H.; von Wachsmann, J.; Thrukonda, L.; Helal, M.; E\nScore: 11.5, Published: 2024-01-15 DOI: 10.1101/2024.01.15.572815\nThe natural reservoirs of the nosocomial pathogen Acinetobacter baumannii are not well defined. We previously identified white storks as a model system to study the ecology of A. baumannii. Having screened more than 1,300 white stork nestlings over a period of six years across different regions of Poland and Germany (overall isolation rate of [~]29.5%), including food chain analyses and environmental samplings, we come up with a detailed picture of the dynamics and diversity of A. baumannii in their natural habitats. Adult storks, rather than being stably colonized with strains of A. baumannii which are successively transferred to their offspring, instead initially encounter these bacteria while foraging. Among their common food sources, consisting of earthworms, small mammals, and insects, we identified earthworms as a potential source of A. baumannii, but more so the associated soil as well as plant roots. Through this, hotspot soil and compost habitats were identified which enable population dynamics to be studied over the course of the year. We demonstrate that sterilized plant material is rapidly colonized by airborne A. baumannii suggesting they patrol to search for novel habitats, being opportunist by nature. The prevalence of A. baumannii exhibited a strong seasonality and peaked during summer. The strains we collected in Poland and Germany represent more than 50% of the worldwide known diversity in terms of the intrinsic OXA-51-like {beta}-lactamase. A set of [~]400 genomes was determined and compared to a diverse set of publicly available genomes. Our pan-genome estimate of the species ([~]51,000 unique genes) more than doubles the amount proposed by previous studies. Core-genome based phylogenetic analyses illustrated numerous links between wildlife isolates and hospital strains, including ancient as well as recent intercontinental transfer. Our data further suggest massive radiation within the species early after its emergence, matching with human activity during the Neolithic. Deforestation in particular seemed to set the stage for this bloom as we found that forests do not provide conducive conditions for the proliferation of A. baumannii. In contrast, wet and nutrient-rich soil alongside rivers sampled during the summer can yield an isolation rate of [~]30%. Linking published work on the interaction between A. baumannii and fungi and on aspergillosis as a major cause of mortality in white stork nestlings to our findings, we hypothesized that fungi and A. baumannii share a long history of coevolution. Interaction studies revealed the capability of A. baumannii to adhere to fungal spores and to suppress spore germination. Taken together, the intrinsic resistance endowment and potential to acquire antibiotic resistance can be explained by coevolution with antibiotic-producing fungi and other microorganisms within soil, and resistance to desiccation stress and radiation can be interpreted in the light of intercontinental hitchhiking through fungal spores. Originality - SignificanceThe ecology of the nosocomial pathogen Acinetobacter baumannii remains poorly understood outside the hospital. Here, we present the most comprehensive study on its environmental biology to date, after having collected more than 1,450 independent isolates of which around 400 were whole genome-sequenced. This study more than doubles the size of the pan-genome of the species, illustrating both the diversity of our collection and the bias of previous work, but also the bottleneck for the establishment of lineages within the hospital environment. We reached isolation rates of about 30% both in white stork (Ciconia ciconia) nestlings and in soil samples when considering for sampling all preferences of A. baumannii we uncovered. Thus, it is now possible to study the ecology and evolution of A. baumannii in nature at an unprecedented temporal and spatial resolution. We describe the worldwide spread of A. baumannii lineages in nature as an ancient phenomenon that even surpasses that of human-associated bacteria in magnitude. This is likely due to airborne spread, putatively facilitated by association with fungal spores. We propose that A. baumannii is an opportunist by nature, using airborne patrolling to rapidly enter new suitable habitats consisting of organic matter in early stages of decomposition. Our collective data suggest that A. baumannii, early after its speciation, went through massive radiation during the Neolithic, likely due to deforestation, settlement and farming producing numerous favorable habitats. Their natural lifestyle, which requires rapid adaptability to various habitats as well as tolerance to desiccation, radiation and antibiotic stress, perfectly predispose these opportunistic pathogens to establish within the hospital setting. Comparison of genomes from environmental and clinical isolates will now enable studies of the adaptive evolution of environmental bacteria towards multidrug-resistant opportunistic pathogens.\nA scaleable inducible knockout systemfor studying essential gene function inthe malaria parasite\nAuthors: Ramaprasad, A.; Blackman, M.\nScore: 10.6, Published: 2024-01-15 DOI: 10.1101/2024.01.14.575607\nThe malaria parasite needs nearly half of its genes to propagate normally within red blood cells. Inducible ways to interfere with gene expression like the DiCre-lox system is necessary to study the function of these essential genes. However, the existing DiCre-lox strategy is not well-suited to be deployed at scale to study several genes simultaneously. To overcome this, we have developed SHIFTiKO (frameshift-based trackable inducible knockout), a novel scaleable strategy that uses short, easy-to-construct, barcoded repair templates to insert loxP sites around short regions in the target genes. Induced DiCre-mediated excision of the flanked region causes a frameshift mutation resulting in genetic ablation of gene function. Dual DNA barcodes inserted into each mutant enables verification of successful modification and induced excision at each locus and collective phenotyping of the mutants, not only across multiple replication cycles to assess growth fitness but also within a single cycle to identify the specific phenotypic impairment they exhibit. As a proof of concept, we have applied SHIFTiKO to screen the functions of malarial rhomboid proteases, successfully identifying their blood stage-specific essentiality. SHIFTiKO, thus offers a powerful platform to conduct inducible phenotypic screens to study essential gene function at scale in the malaria parasite.\nDivisome minimization shows that FtsZ and SepF can form an active Z-ring, and reveals BraB as a new cell division influencing protein in Bacillus subtilis\nAuthors: Celik Gulsoy, I.; Saaki, T.; Wenzel, M.; Syvertsson, S.; Morimoto, T.; Hamoen, L. W.\nScore: 10.3, Published: 2024-01-13 DOI: 10.1101/2024.01.12.575403\nOne of the aims of synthetic biology is the construction of a minimal cell. Since bacteria are the simplest life forms, they are the preferred blueprint for such a cell. In this study, we tried to find the minimal set of proteins required for division of a cell wall containing cell, using the bacterial model system Bacillus subtilis. Bacterial cytokinesis begins with polymerization of the tubulin homologue FtsZ at midcell into the Z-ring, which recruits the late cell division protein that synthesize the division septum. Assembly of FtsZ is carefully regulated, involving a dozen conserved cell division proteins. These proteins are not essential, but removing more than one is in many cases lethal. We made use of known suppressor mutations to find a gene deletion route that eventually enabled us to remove eight conserved cell division proteins: ZapA, MinC, MinJ, UgtP, ClpX, Noc, EzrA and FtsA. Only FtsZ and its membrane anchor SepF appeared to be required for Z-ring formation. Interestingly, SepF is also the FtsZ anchor in archaea, and both proteins may date back to the Last Universal Common Ancestor (LUCA). Curiously, viability was not greatly affected by the multiple deletions, although the frequency of cell division was considerably reduced. However, genome sequencing exposed the accumulation of several suppressor mutations, and revealed an unexpected cell division regulation function for the branched chain amino acid transporter BraB. The implications of these findings for the role of SepF in cell division, and the construction of a minimal cell division machinery are discussed.\nA new protocol for multispecies bacterial infections in zebrafish and their monitoring through automated image analysis\nAuthors: Schmitz, D. A.; Wechsler, T.; Li, H. B.; Menze, B.; Kuemmerli, R.\nScore: 10.1, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575759\nThe zebrafish Danio rerio has become a popular model host to explore disease pathology caused by infectious agents. A main advantage is its transparency at an early age, which enables live imaging of infection dynamics. While multispecies infections are common in patients, the zebrafish model is rarely used to study them, although the model would be ideal for investigating pathogen-pathogen and pathogen-host interactions. This may be due to the absence of an established multispecies infection protocol for a defined organ and the lack of suitable image analysis pipelines for automated image processing. To address these issues, we developed a protocol for establishing and tracking single and multispecies bacterial infections in the inner ear structure (otic vesicle) of the zebrafish by imaging. Subsequently, we generated an image analysis pipeline that involved deep learning for the automated segmentation of the otic vesicle, and scripts for the quantification of pathogen frequencies through fluorescence intensity measures. We used Pseudomonas aeruginosa, Acinetobacter baumannii, and Klebsiella pneumoniae, three of the difficult-to-treat ESKAPE pathogens, to show that our infection protocol and image analysis pipeline work both for single pathogens and pairwise pathogen combinations. Thus, our protocols provide a comprehensive toolbox for studying single and multispecies infections in real-time in zebrafish.\nTesting microbial biomining from asteroidal material onboard the International Space Station\nAuthors: Santomartino, R.; Rodriguez Blanco, G.; Gudgeon, A.; Hafner, J. H.; Stirpe, A.; Waterfall, M.; Cayzer, N.; Pichevin, L.; Calder, G.; Birkenfeld, K. R.; Waajen, A. C.; McLaughlin, S.; Mariani, A.; Balsamo, M.; Neri, G.; Eades, L. J.; Cockell, C.\nScore: 10.0, Published: 2024-01-14 DOI: 10.1101/2024.01.13.575412\nExpanding human space exploration beyond Earths orbit necessitates efficient technologies for self-sustainable acquisition of local resources to overcome unviable resupply missions from Earth. Potential source of materials are asteroids, some of which contain valuable metals, such as platinum group elements. The BioAsteroid experiment, performed onboard the International Space Station, tested the use of microorganisms (bacteria and fungi) to carry out mining of useful elements from asteroidal material (L-chondrite) under microgravity, in support of a long-term human presence in space. The fungus Penicillium simplicissimum, enhanced the mean release of palladium, platinum and other elements from the meteorite material in microgravity, compared to non-biological leaching. However, there was large variability in the results. For many elements, non-biological leaching under microgravity was enhanced compared to terrestrial gravity, while bioleaching was unaffected. Metabolomics results revealed clear patterns that highlight the influence of space conditions on the microbial metabolism, particularly for P. simplicissimum. We identified the presence of carboxylic acids, and molecules of potential biomining and pharmaceutical interest, enhanced in microgravity. These results show a non-trivial effect of microgravity on bioleaching, highlighting the requirement of an optimal combination of microorganism(s), rock substrate, and conditions for successful biomining, both in space and Earth.\n",
  "wordCount" : "3139",
  "inLanguage": "en",
  "datePublished": "2024-01-21T10:39:08Z",
  "dateModified": "2024-01-21T10:39:08Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 21, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574008">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574008" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574008">
        <p class="paperTitle">Lethal Infection of Human ACE2-Transgenic Mice Caused by SARS-CoV-2-related Pangolin Coronavirus GX_P2V(short_3UTR)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574008" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574008" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.</p>
        <p class="info">Score: 4033.2, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574008' target='https://doi.org/10.1101/2024.01.03.574008'> 10.1101/2024.01.03.574008</a></p>
        <p class="abstract">SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) can cause 100% mortality in human ACE2-transgenic mice, potentially attributable to late-stage brain infection. This underscores a spillover risk of GX_P2V into humans and provides a unique model for understanding the pathogenic mechanisms of SARS-CoV-2-related viruses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.14.575572">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.14.575572" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.14.575572">
        <p class="paperTitle">NRX-101 (D-Cycloserine &#43; Lurasidone) , a Qualified Infectious Disease Product, is Active Against Drug-Resistant Urinary Pathogens In Vitro</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.14.575572" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.14.575572" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sapko, M. T.; Manyak, M.; Panicucci, R.; Javitt, J. C.</p>
        <p class="info">Score: 58.5, Published: 2024-01-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.14.575572' target='https://doi.org/10.1101/2024.01.14.575572'> 10.1101/2024.01.14.575572</a></p>
        <p class="abstract">BackgroundD-Cycloserine (DCS) is a broad-spectrum antibiotic that is currently FDA-approved to treat tuberculosis (TB) disease and urinary tract infection. Despite numerous reports showing good clinical efficacy, DCS fell out of favor as a UTI treatment because of its propensity to cause side effects. NRX-101, a fixed dose combination of DCS and lurasidone, has been awarded Qualified Infectious Disease Product and Fast Track Designation by the US Food and Drug Administration and is being developed for various CNS indications because of its unique synergistic effect; each component mitigates side effects of the other.

MethodsIn this study, we tested NRX-101 against the urinary tract pathogens E. coli, P. aeruginosa, K. pneumoniae, and A. baumannii in Mueller Hinton broth (caMHB) and artificial urine media (AUM). Several strains were multidrug resistant. Test compounds were serially diluted in broth/media. Minimum inhibitory concentration (MIC) was defined as the lowest concentration of test compound at which no bacterial growth was observed.

ResultsDCS exhibited antibacterial efficacy against all strains tested while lurasidone did not appreciably affect the antibacterial action of DCS in vitro. In AUM, the MICs ranged from 128 to 512 mcg/ml for both DCS and NRX-101. In caMHB, MICs ranged from 8 to 1024 mcg/ml for NRX-101 and 32 to 512 mcg/ml for DCS alone.

ConclusionsOur data confirm that DCS as antibacterial activity against reference and drug-resistant urinary pathogens. Furthermore, lurasidone does not interfere with DCSs anti-microbial action in vitro. These results support the clinical development of NRX-101 as a treatment for complicated urinary tract infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.16.574955">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.16.574955" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.16.574955">
        <p class="paperTitle">A treasure trove of 1,034 actinomycete genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.16.574955" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.16.574955" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jorgensen, T. S.; Mohite, O.; Sterndorff, E. B.; Alvarez-Arevalo, M.; Blin, K.; Booth, T. J.; Charusanti, P.; Faurdal, D.; Hansen, T. O.; Nuhamunada, M. J.; Mourched, A.-S.; Palsson, B. O.; Weber, T.</p>
        <p class="info">Score: 45.1, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.16.574955' target='https://doi.org/10.1101/2024.01.16.574955'> 10.1101/2024.01.16.574955</a></p>
        <p class="abstract">Filamentous Actinobacteria, recently renamed Actinomycetia, are the most prolific source of microbial bioactive natural products. Studies on biosynthetic gene clusters benefit from or require chromosome-level assemblies. Here, we provide DNA sequences from more than 1,000 isolates: 881 complete genomes and 153 near-complete genomes, representing 28 genera and 389 species, including 244 likely novel species. All genomes are from filamentous isolates of the class Actinomycetia from the NBC culture collection. The largest genus is Streptomyces with 886 genomes including 742 complete assemblies. We use this data to show that analysis of complete genomes can bring biological understanding not previously derived from more fragmented sequences or less systematic datasets. We document the central and structured location of core genes and distal location of specialized metabolite biosynthetic gene clusters and duplicate core genes on the linear Streptomyces chromosome, and analyze the content and length of the terminal inverted repeats which are characteristic for Streptomyces. We then analyze the diversity of trans-AT polyketide synthase biosynthetic gene clusters, which encodes the machinery of a biotechnologically highly interesting compound class. These insights have both ecological and biotechnological implications in understanding the importance of high quality genomic resources and the complex role synteny plays in Actinomycetia biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575717">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575717" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575717">
        <p class="paperTitle">Superiority of chromosomal compared to plasmid-encoded compensatory mutations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575717" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575717" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wright, R. C. T.; Wood, J.; Bottery, M. J.; Muddiman, K. J.; Paterson, S.; Harrison, E.; Brockhurst, M. A.; Hall, J. P. J.</p>
        <p class="info">Score: 13.9, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575717' target='https://doi.org/10.1101/2024.01.15.575717'> 10.1101/2024.01.15.575717</a></p>
        <p class="abstract">Plasmids are important vectors of horizontal gene transfer in microbial communities but can impose a burden on the bacteria that carry them. Such plasmid fitness costs are thought to arise principally from conflicts between chromosomal- and plasmid-encoded molecular machineries, and thus can be ameliorated by compensatory mutations (CMs) that reduce or resolve the underlying causes. CMs can arise on plasmids (i.e. plaCM) or on chromosomes (i.e. chrCM), with contrasting predicted effects upon plasmid success and subsequent gene transfer because plaCM can also reduce fitness costs in plasmid recipients, whereas chrCM can potentially ameliorate multiple distinct plasmids. Here, we develop theory and a novel experimental system to directly compare the ecological effects of plaCM and chrCM that arose during evolution experiments between Pseudomonas fluorescens SBW25 and its sympatric mercury resistance megaplasmid pQBR57. We show that while plaCM was predicted to succeed under a broader range of parameters in mathematical models, experimentally chrCM dominated under all conditions, including those with numerous recipients, due to a more efficacious mechanism of compensation, and advantages arising from transmission of costly plasmids to competitors (plasmid  weaponisation). We show analytically the presence of a mixed Rock-Paper-Scissors regime for plaCM, driven by trade-offs with horizontal transmission, that explains the observed failure of plaCM to dominate even in competition against an uncompensated plasmid. Our results reveal broader implications of plasmid-bacterial evolution for plasmid ecology, demonstrating the importance of compensatory mutations for resistance gene spread. One consequence of the superiority of chrCM over plaCM is the likely emergence in microbial communities of compensated bacteria that can act as  hubs for plasmid accumulation and dissemination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.574570">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.574570" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.574570">
        <p class="paperTitle">Nitrous oxide respiration in acidophilic methanotrophs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.574570" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.574570" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Awala, S. I.; Gwak, J.-H.; Kim, Y.; Jung, M.-Y.; Dunfield, P. F.; Wagner, M.; Rhee, S.-K.</p>
        <p class="info">Score: 12.4, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.574570' target='https://doi.org/10.1101/2024.01.15.574570'> 10.1101/2024.01.15.574570</a></p>
        <p class="abstract">Methanotrophic bacteria mitigate methane (CH4) emissions from natural environments. Although aerobic methanotrophs are considered strict aerobes, they are often highly abundant in extremely hypoxic and even anoxic environments. Despite the presence of denitrification genes, it remains to be verified whether denitrification contributes to their growth. Here, we revealed that two acidophilic methanotrophs encoding N2O reductase (clade I and type II nosZ, respectively): Methylocella tundrae T4 and Methylacidiphilum caldifontis IT6, respired N2O and grew anaerobically on diverse non-methane substrates, including methanol, C-C substrates, and hydrogen. However, NO3 - and NO2 - could be reduced during methanol oxidation in Methylocella tundrae T4 and Methylocella silvestris BL2 without significantly increasing cell biomass. The lack of growth on methanol &#43; NO3- or NO2- was likely due to the production of toxic reactive nitrogen species and C1 metabolites. However, the oxidation of pyruvate, a C3 electron donor, combined with NO3- or NO2- reduction resulted in anaerobic growth of Methylocella tundrae T4 and Methylocella silvestris BL2. In the extreme acidophile, Methylacidiphilum caldifontis IT6, N2O respiration supported cell growth at an extremely acidic pH of 2.0. In Methylocella tundrae T4, simultaneous consumption of N2O and CH4 was observed in suboxic conditions, both in microrespirometry and growth experiments, indicating the robustness of its N2O reductase activity in the presence of O2. Furthermore, CH4 oxidation per O2 reduced in O2-limiting conditions increased when N2O was added, indicating that cells of T4 can direct more O2 towards methane monooxygenase when respiring N2O as a terminal electron acceptor. Upregulation of nosZ and distinct repertories of methanol dehydrogenase-encoding genes (XoxF- and MxaFI-type) in Methylocella tundrae T4 cells grown anaerobically on methanol with N2O as the sole electron acceptor indicated adaptation mechanisms to anoxia. Our findings demonstrate that some methanotrophs can respire N2O independently or in tandem with O2, significantly expanding their potential ecological niche and paving the way for enhanced growth and survival in dynamic environments. This metabolic capability has application potential for simultaneously mitigating the emissions of the key greenhouse gases, CO2, CH4, and N2O, from natural and engineered environments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.572815">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.572815" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.572815">
        <p class="paperTitle">On the ecology of Acinetobacter baumannii - jet stream rider and opportunist by nature</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.572815" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.572815" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wilharm, G.; Skiebe, E.; Lopinska, A.; Higgins, P. G.; Weber, K.; Schaudinn, C.; Neugebauer, C.; Goerlitz, K.; Meimers, G.; Rizova, Y.; Blaschke, U.; Heider, C.; Cuny, C.; Drewes, S.; Heuser, E.; Jeske, K.; Jacob, J.; Ulrich, R. G.; Bochenski, M.; Kasprzak, M.; Burda, E.; Cieplinski, M.; Kaluga, I.; Jankowiak, L.; Aguirre, J. I.; Lopez-Garcia, A.; Hoefle, U.; Jagiello, Z.; Tobolka, M.; Janic, B.; Zielinski, P.; Kaminski, M.; Frisch, J.; Siekiera, J.; Wendel, A. F.; Brauner, P.; Jaeckel, U.; Kaatz, M.; Mueller, S.; Luebke-Becker, A.; Wieler, L. H.; von Wachsmann, J.; Thrukonda, L.; Helal, M.; E</p>
        <p class="info">Score: 11.5, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.572815' target='https://doi.org/10.1101/2024.01.15.572815'> 10.1101/2024.01.15.572815</a></p>
        <p class="abstract">The natural reservoirs of the nosocomial pathogen Acinetobacter baumannii are not well defined. We previously identified white storks as a model system to study the ecology of A. baumannii. Having screened more than 1,300 white stork nestlings over a period of six years across different regions of Poland and Germany (overall isolation rate of [~]29.5%), including food chain analyses and environmental samplings, we come up with a detailed picture of the dynamics and diversity of A. baumannii in their natural habitats. Adult storks, rather than being stably colonized with strains of A. baumannii which are successively transferred to their offspring, instead initially encounter these bacteria while foraging. Among their common food sources, consisting of earthworms, small mammals, and insects, we identified earthworms as a potential source of A. baumannii, but more so the associated soil as well as plant roots. Through this, hotspot soil and compost habitats were identified which enable population dynamics to be studied over the course of the year. We demonstrate that sterilized plant material is rapidly colonized by airborne A. baumannii suggesting they patrol to search for novel habitats, being opportunist by nature. The prevalence of A. baumannii exhibited a strong seasonality and peaked during summer. The strains we collected in Poland and Germany represent more than 50% of the worldwide known diversity in terms of the intrinsic OXA-51-like {beta}-lactamase. A set of [~]400 genomes was determined and compared to a diverse set of publicly available genomes. Our pan-genome estimate of the species ([~]51,000 unique genes) more than doubles the amount proposed by previous studies. Core-genome based phylogenetic analyses illustrated numerous links between wildlife isolates and hospital strains, including ancient as well as recent intercontinental transfer. Our data further suggest massive radiation within the species early after its emergence, matching with human activity during the Neolithic. Deforestation in particular seemed to set the stage for this bloom as we found that forests do not provide conducive conditions for the proliferation of A. baumannii. In contrast, wet and nutrient-rich soil alongside rivers sampled during the summer can yield an isolation rate of [~]30%. Linking published work on the interaction between A. baumannii and fungi and on aspergillosis as a major cause of mortality in white stork nestlings to our findings, we hypothesized that fungi and A. baumannii share a long history of coevolution. Interaction studies revealed the capability of A. baumannii to adhere to fungal spores and to suppress spore germination. Taken together, the intrinsic resistance endowment and potential to acquire antibiotic resistance can be explained by coevolution with antibiotic-producing fungi and other microorganisms within soil, and resistance to desiccation stress and radiation can be interpreted in the light of intercontinental hitchhiking through fungal spores.

Originality - SignificanceThe ecology of the nosocomial pathogen Acinetobacter baumannii remains poorly understood outside the hospital. Here, we present the most comprehensive study on its environmental biology to date, after having collected more than 1,450 independent isolates of which around 400 were whole genome-sequenced. This study more than doubles the size of the pan-genome of the species, illustrating both the diversity of our collection and the bias of previous work, but also the bottleneck for the establishment of lineages within the hospital environment. We reached isolation rates of about 30% both in white stork (Ciconia ciconia) nestlings and in soil samples when considering for sampling all preferences of A. baumannii we uncovered. Thus, it is now possible to study the ecology and evolution of A. baumannii in nature at an unprecedented temporal and spatial resolution. We describe the worldwide spread of A. baumannii lineages in nature as an ancient phenomenon that even surpasses that of human-associated bacteria in magnitude. This is likely due to airborne spread, putatively facilitated by association with fungal spores. We propose that A. baumannii is an opportunist by nature, using airborne patrolling to rapidly enter new suitable habitats consisting of organic matter in early stages of decomposition. Our collective data suggest that A. baumannii, early after its speciation, went through massive radiation during the Neolithic, likely due to deforestation, settlement and farming producing numerous favorable habitats. Their natural lifestyle, which requires rapid adaptability to various habitats as well as tolerance to desiccation, radiation and antibiotic stress, perfectly predispose these opportunistic pathogens to establish within the hospital setting. Comparison of genomes from environmental and clinical isolates will now enable studies of the adaptive evolution of environmental bacteria towards multidrug-resistant opportunistic pathogens.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.14.575607">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.14.575607" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.14.575607">
        <p class="paperTitle">A scaleable inducible knockout systemfor studying essential gene function inthe malaria parasite</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.14.575607" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.14.575607" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ramaprasad, A.; Blackman, M.</p>
        <p class="info">Score: 10.6, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.14.575607' target='https://doi.org/10.1101/2024.01.14.575607'> 10.1101/2024.01.14.575607</a></p>
        <p class="abstract">The malaria parasite needs nearly half of its genes to propagate normally within red blood cells. Inducible ways to interfere with gene expression like the DiCre-lox system is necessary to study the function of these essential genes. However, the existing DiCre-lox strategy is not well-suited to be deployed at scale to study several genes simultaneously. To overcome this, we have developed SHIFTiKO (frameshift-based trackable inducible knockout), a novel scaleable strategy that uses short, easy-to-construct, barcoded repair templates to insert loxP sites around short regions in the target genes. Induced DiCre-mediated excision of the flanked region causes a frameshift mutation resulting in genetic ablation of gene function. Dual DNA barcodes inserted into each mutant enables verification of successful modification and induced excision at each locus and collective phenotyping of the mutants, not only across multiple replication cycles to assess growth fitness but also within a single cycle to identify the specific phenotypic impairment they exhibit. As a proof of concept, we have applied SHIFTiKO to screen the functions of malarial rhomboid proteases, successfully identifying their blood stage-specific essentiality. SHIFTiKO, thus offers a powerful platform to conduct inducible phenotypic screens to study essential gene function at scale in the malaria parasite.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.575403">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.575403" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.575403">
        <p class="paperTitle">Divisome minimization shows that FtsZ and SepF can form an active Z-ring, and reveals BraB as a new cell division influencing protein in Bacillus subtilis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.575403" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.575403" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Celik Gulsoy, I.; Saaki, T.; Wenzel, M.; Syvertsson, S.; Morimoto, T.; Hamoen, L. W.</p>
        <p class="info">Score: 10.3, Published: 2024-01-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.575403' target='https://doi.org/10.1101/2024.01.12.575403'> 10.1101/2024.01.12.575403</a></p>
        <p class="abstract">One of the aims of synthetic biology is the construction of a minimal cell. Since bacteria are the simplest life forms, they are the preferred blueprint for such a cell. In this study, we tried to find the minimal set of proteins required for division of a cell wall containing cell, using the bacterial model system Bacillus subtilis. Bacterial cytokinesis begins with polymerization of the tubulin homologue FtsZ at midcell into the Z-ring, which recruits the late cell division protein that synthesize the division septum. Assembly of FtsZ is carefully regulated, involving a dozen conserved cell division proteins. These proteins are not essential, but removing more than one is in many cases lethal. We made use of known suppressor mutations to find a gene deletion route that eventually enabled us to remove eight conserved cell division proteins: ZapA, MinC, MinJ, UgtP, ClpX, Noc, EzrA and FtsA. Only FtsZ and its membrane anchor SepF appeared to be required for Z-ring formation. Interestingly, SepF is also the FtsZ anchor in archaea, and both proteins may date back to the Last Universal Common Ancestor (LUCA). Curiously, viability was not greatly affected by the multiple deletions, although the frequency of cell division was considerably reduced. However, genome sequencing exposed the accumulation of several suppressor mutations, and revealed an unexpected cell division regulation function for the branched chain amino acid transporter BraB. The implications of these findings for the role of SepF in cell division, and the construction of a minimal cell division machinery are discussed.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575759">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575759" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575759">
        <p class="paperTitle">A new protocol for multispecies bacterial infections in zebrafish and their monitoring through automated image analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575759" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575759" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schmitz, D. A.; Wechsler, T.; Li, H. B.; Menze, B.; Kuemmerli, R.</p>
        <p class="info">Score: 10.1, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575759' target='https://doi.org/10.1101/2024.01.15.575759'> 10.1101/2024.01.15.575759</a></p>
        <p class="abstract">The zebrafish Danio rerio has become a popular model host to explore disease pathology caused by infectious agents. A main advantage is its transparency at an early age, which enables live imaging of infection dynamics. While multispecies infections are common in patients, the zebrafish model is rarely used to study them, although the model would be ideal for investigating pathogen-pathogen and pathogen-host interactions. This may be due to the absence of an established multispecies infection protocol for a defined organ and the lack of suitable image analysis pipelines for automated image processing. To address these issues, we developed a protocol for establishing and tracking single and multispecies bacterial infections in the inner ear structure (otic vesicle) of the zebrafish by imaging. Subsequently, we generated an image analysis pipeline that involved deep learning for the automated segmentation of the otic vesicle, and scripts for the quantification of pathogen frequencies through fluorescence intensity measures. We used Pseudomonas aeruginosa, Acinetobacter baumannii, and Klebsiella pneumoniae, three of the difficult-to-treat ESKAPE pathogens, to show that our infection protocol and image analysis pipeline work both for single pathogens and pairwise pathogen combinations. Thus, our protocols provide a comprehensive toolbox for studying single and multispecies infections in real-time in zebrafish.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.13.575412">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.13.575412" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.13.575412">
        <p class="paperTitle">Testing microbial biomining from asteroidal material onboard the International Space Station</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.13.575412" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.13.575412" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Santomartino, R.; Rodriguez Blanco, G.; Gudgeon, A.; Hafner, J. H.; Stirpe, A.; Waterfall, M.; Cayzer, N.; Pichevin, L.; Calder, G.; Birkenfeld, K. R.; Waajen, A. C.; McLaughlin, S.; Mariani, A.; Balsamo, M.; Neri, G.; Eades, L. J.; Cockell, C.</p>
        <p class="info">Score: 10.0, Published: 2024-01-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.13.575412' target='https://doi.org/10.1101/2024.01.13.575412'> 10.1101/2024.01.13.575412</a></p>
        <p class="abstract">Expanding human space exploration beyond Earths orbit necessitates efficient technologies for self-sustainable acquisition of local resources to overcome unviable resupply missions from Earth. Potential source of materials are asteroids, some of which contain valuable metals, such as platinum group elements.

The BioAsteroid experiment, performed onboard the International Space Station, tested the use of microorganisms (bacteria and fungi) to carry out mining of useful elements from asteroidal material (L-chondrite) under microgravity, in support of a long-term human presence in space. The fungus Penicillium simplicissimum, enhanced the mean release of palladium, platinum and other elements from the meteorite material in microgravity, compared to non-biological leaching. However, there was large variability in the results. For many elements, non-biological leaching under microgravity was enhanced compared to terrestrial gravity, while bioleaching was unaffected. Metabolomics results revealed clear patterns that highlight the influence of space conditions on the microbial metabolism, particularly for P. simplicissimum. We identified the presence of carboxylic acids, and molecules of potential biomining and pharmaceutical interest, enhanced in microgravity.

These results show a non-trivial effect of microgravity on bioleaching, highlighting the requirement of an optimal combination of microorganism(s), rock substrate, and conditions for successful biomining, both in space and Earth.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
