<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="SARS-CoV-2 Neutralizing Antibodies Following a Second BA.5 Bivalent Booster
Authors: Wang, Q.; Bowen, A.; Ho, J.; Zhang, R.; Valdez, R.; Stoneman, E.; Gordon, A.; Lihong, L.; Ho, D. D.
Score: 355.7, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553148
Bivalent COVID-19 mRNA vaccines expressing both the ancestral D614G and Omicron BA.5 spike proteins were introduced in August 2022 with the goal of broadening immunity to emerging SARS-CoV-2 Omicron subvariants. Subsequent studies on bivalent boosters found neutralizing antibody responses similar to boosters with the original monovalent vaccine, likely the result of immunological imprinting.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="SARS-CoV-2 Neutralizing Antibodies Following a Second BA.5 Bivalent Booster
Authors: Wang, Q.; Bowen, A.; Ho, J.; Zhang, R.; Valdez, R.; Stoneman, E.; Gordon, A.; Lihong, L.; Ho, D. D.
Score: 355.7, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553148
Bivalent COVID-19 mRNA vaccines expressing both the ancestral D614G and Omicron BA.5 spike proteins were introduced in August 2022 with the goal of broadening immunity to emerging SARS-CoV-2 Omicron subvariants. Subsequent studies on bivalent boosters found neutralizing antibody responses similar to boosters with the original monovalent vaccine, likely the result of immunological imprinting." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-30T10:38:23+00:00" />
<meta property="article:modified_time" content="2023-08-30T10:38:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="SARS-CoV-2 Neutralizing Antibodies Following a Second BA.5 Bivalent Booster
Authors: Wang, Q.; Bowen, A.; Ho, J.; Zhang, R.; Valdez, R.; Stoneman, E.; Gordon, A.; Lihong, L.; Ho, D. D.
Score: 355.7, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553148
Bivalent COVID-19 mRNA vaccines expressing both the ancestral D614G and Omicron BA.5 spike proteins were introduced in August 2022 with the goal of broadening immunity to emerging SARS-CoV-2 Omicron subvariants. Subsequent studies on bivalent boosters found neutralizing antibody responses similar to boosters with the original monovalent vaccine, likely the result of immunological imprinting."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "SARS-CoV-2 Neutralizing Antibodies Following a Second BA.5 Bivalent Booster\nAuthors: Wang, Q.; Bowen, A.; Ho, J.; Zhang, R.; Valdez, R.; Stoneman, E.; Gordon, A.; Lihong, L.; Ho, D. D.\nScore: 355.7, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553148\nBivalent COVID-19 mRNA vaccines expressing both the ancestral D614G and Omicron BA.5 spike proteins were introduced in August 2022 with the goal of broadening immunity to emerging SARS-CoV-2 Omicron subvariants. Subsequent studies on bivalent boosters found neutralizing antibody responses similar to boosters with the original monovalent vaccine, likely the result of immunological imprinting.",
  "keywords": [
    
  ],
  "articleBody": " SARS-CoV-2 Neutralizing Antibodies Following a Second BA.5 Bivalent Booster\nAuthors: Wang, Q.; Bowen, A.; Ho, J.; Zhang, R.; Valdez, R.; Stoneman, E.; Gordon, A.; Lihong, L.; Ho, D. D.\nScore: 355.7, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553148\nBivalent COVID-19 mRNA vaccines expressing both the ancestral D614G and Omicron BA.5 spike proteins were introduced in August 2022 with the goal of broadening immunity to emerging SARS-CoV-2 Omicron subvariants. Subsequent studies on bivalent boosters found neutralizing antibody responses similar to boosters with the original monovalent vaccine, likely the result of immunological imprinting. Guidelines allow for administration of a second bivalent booster in high-risk groups, but it remains unknown whether this would broaden antibody responses. To address this question, we assessed longitudinal serum SARS-CoV-2-neutralizing titers in 18 elderly immunocompetent individuals (mean age 69) following a fourth monovalent booster and two BA.5 bivalent booster vaccines using pseudovirus neutralization assays against D614G, Omicron BA.5, and Omicron XBB.1.5. There was a small but significant increase in peak neutralizing antibody responses against Omicron BA.5 and XBB.1.5 following the first bivalent booster, but no significant increase in peak titers following the second bivalent booster. Omicron-specific neutralizing titers remained low after both doses of the BA.5 bivalent booster. Our results suggest that a second dose of the BA.5 bivalent booster is not sufficient to broaden antibody responses and to overcome immunological imprinting. A monovalent vaccine targeting only the spike of the recently dominant SARS-CoV-2 may mitigate the \"back boosting\" associated with the \"original antigenic sin.\"\nAntibody Neutralization of Emerging SARS-CoV-2: EG.5.1 and XBC.1.6\nAuthors: Wang, Q.; Guo, Y.; Zhang, R. M.; Ho, J.; Mohri, H.; Valdez, R.; Manthei, D. M.; Gordon, A.; Liu, L.; Ho, D. D.\nScore: 32.3, Published: 2023-08-25 DOI: 10.1101/2023.08.21.553968\nSARS-CoV-2 variants EG.5.1 and XBC.1.6 have recently emerged, attracting increased attention due to their rapid expansion globally and in Australia, respectively. EG.5.1 evolved from Omicron subvariant XBB.1.9, harboring additional Q52H and F456L spike substitutions. The F456L mutation is located within the epitopes of many class-1 monoclonal antibodies (mAbs) directed to the receptor-binding domain (RBD), raising concerns about further antibody evasion. XBC.1.6, a descendant of a Delta-BA.2 recombinant, carries 15 additional spike mutations. The extent to which antibody evasion contributes to the growth advantage of XBC.1.6 in Australia remains to be determined. To assess the antibody evasion properties of the emergent variants, we conducted pseudovirus neutralization assays using sera from individuals who received three doses of COVID-19 mRNA monovalent vaccines plus one dose of a BA.5 bivalent vaccine, as well as from patients with BQ or XBB breakthrough infection. The assays were also performed using a panel of 14 mAbs that retained neutralizing activity against prior XBB subvariants. Our data suggested that EG.5.1 was slightly but significantly more resistant (\u003c 2-fold) to neutralization by BQ and XBB breakthrough sera than XBB.1.16, which is known to be antigenically similar to XBB.1.5. Moreover, the F456L mutation in EG.5.1 conferred heightened resistance to certain RBD class-1 mAbs. In contrast, XBC.1.6 was more sensitive to neutralization by sera and mAbs than the XBB subvariants. Notably, XBB breakthrough sera retained only weak neutralization activity against XBB subvariants. In summary, EG.5.1 and XBC.1.6 exhibited distinct antibody evasion properties. The recent global expansion of EG.5.1 might be attributable, in part, to its enhanced neutralization resistance. That XBB breakthrough infections did not elicit a robust antibody neutralization response against XBB subvariants is indicative of immunological imprinting. The high prevalence of XBC.1.6 in Australia is not due to enhanced antibody evasion.\nDisruption of Aldehyde Dehydrogenase 2 protects against bacterial infection\nAuthors: Berry, S.; Espich, S.; Thuong, N. T. T.; Chang, X.; Dorajoo, R.; Khor, C. C.; Heng, C. K.; Yuan, J.-M.; Fox, D.; Anaya-Sanchez, A.; Tenney, L.; Chang, C. J.; Kotov, D. I.; Vance, R. E.; Dunstan, S. J.; Darwin, K. H.; Stanley, S. A.\nScore: 26.7, Published: 2023-08-26 DOI: 10.1101/2023.08.24.554661\nThe ALDH2*2 (rs671) allele is one of the most common genetic mutations in humans, yet the positive evolutionary selective pressure to maintain this mutation is unknown, despite its association with adverse health outcomes. ALDH2 is responsible for the detoxification of metabolically produced aldehydes, including lipid-peroxidation end products derived from inflammation. Here, we demonstrate that host-derived aldehydes 4-hydroxynonenal (4HNE), malondialdehyde (MDA), and formaldehyde (FA), all of which are metabolized by ALDH2, are directly toxic to the bacterial pathogens Mycobacterium tuberculosis and Francisella tularensis at physiological levels. We find that Aldh2 expression in macrophages is decreased upon immune stimulation, and that bone marrow-derived macrophages from Aldh2-/- mice contain elevated aldehydes relative to wild-type mice. Macrophages deficient for Aldh2 exhibited enhanced control of Francisella infection. Finally, mice lacking Aldh2 demonstrated increased resistance to pulmonary infection by M. tuberculosis, including in a hypersusceptible model of tuberculosis, and were also resistant to Francisella infection. We hypothesize that the absence of ALDH2 contributes to the hosts ability to control infection by pathogens such as M. tuberculosis and F. tularensis, and that host-derived aldehydes act as antimicrobial factors during intracellular bacterial infections. One sentence summaryAldehydes produced by host cells contribute to the control of bacterial infections.\nMiDAS 5: Global diversity of bacteria and archaea in anaerobic digesters\nAuthors: Dueholm, M. K. D.; Andersen, K. S.; Petersen, A.-K. C.; Rudkjoebing, V.; MiDAS Global Consortium for Anaerobic Digesters, ; Nielsen, P. H.\nScore: 17.0, Published: 2023-08-24 DOI: 10.1101/2023.08.24.554448\nAnaerobic digestion represents a key biotechnology for the transformation of organic waste into renewable energy (biogas) and relies on complex microbial communities that work in concert to degrade the complex substrates into methane and carbon dioxide. Here, we sequenced more than half a million high-quality, full-length 16S rRNA gene sequences from 285 full-scale anaerobic digesters (ADs) across the world to expand our knowledge about diversity and function of the bacteria and archaea in ADs. The sequences were processed into full-length 16S rRNA amplicon sequence variants (FL-ASVs), which were added to the MiDAS 4 database for bacteria and archaea in wastewater treatment systems to create MiDAS 5. The expansion of the MiDAS database significantly increased the coverage for bacteria and archaea in ADs worldwide, leading to an improved rate of genus and species-level classification. Using MiDAS 5, we carried out an amplicon-based, global-scale microbial community profiling of the sampled ADs using three common sets of primers targeting different regions of the 16S rRNA gene in bacteria and/or archaea. We revealed how environmental conditions and biogeography shape the AD microbiota. We also identify core and conditionally rare or abundant taxa, encompassing less than 1000 genera and species. These represent 83-99% and 10-64% of the accumulated read abundance respectively, across samples depending on the amplicon primers used. Finally, we examined the global diversity of functional groups with known importance for the anaerobic digestion process. Our online global MiDAS Field Guide presents the data generated in this study and summarizes present knowledge about all taxa.\nDissecting the neurotropism and neurovirulence of MPXV using human stem cell-based models\nAuthors: Bauer, L.; Giussani, S.; Palazzi, N.; Zare, F.; Colombo, E.; Pinci, F.; Leijten, L.; Smeenk, H.; Embregts, C. W. E.; Silva, M.; Spoor, J. K. H.; Dirven, C.; Gao, Z.; Bolleboom, A.; Verstrepen, B. E.; Schuele, L.; de Vrij, F. M. S.; Kushner, S. A.; Oude Munnink, B. B.; Davila-Velderrain, J.; van Riel, D.; Harschnitz, O.\nScore: 14.9, Published: 2023-08-28 DOI: 10.1101/2023.08.25.554849\nMpox, formerly known as monkeypox, is a zoonotic illness of international concern that can lead to severe disease including neurological sequelae. However, it remains unclear what the neurotropism of monkeypox virus (MPXV) is and how MPXV infection leads to neurological deficits. Here, we determined the neurotropism and neurovirulence of MPXV using human pluripotent stem cell- (hPSC)-derived neural stem cells, astrocytes, cortical neurons, and microglia together with ex vivo human brain tissue. We found that MPXV infects and replicates more efficiently in astrocytes and microglia compared to cortical neurons, which unlike glial cells showed activation of distinct antiviral programs that may confer differential susceptibility to MPXV. Ex vivo infection of human brain tissue confirmed the susceptibility of astrocytes to MPXV infection, which also had the strongest disease-associated changes. Molecular pathway analyses revealed induction of cellular senescence and a senescence-associated secretory phenotype upon MPXV infection in astrocytes. Finally, we demonstrated that antiviral treatment using tecovirimat inhibits MPXV replication and prevents virus-induced senescence in hPSC-derived astrocytes. Altogether, leveraging hPSC-derived brain cells, we reveal MPXV-induced cell type-specific effects at the molecular and cellular level, which provide important insights into the neuropathogenesis of MPXV infection.\nInhibition of PQS signaling by the Pf bacteriophage protein PfsE enhances viral replication in Pseudomonas aeruginosa\nAuthors: Schwartzkopf, C. M.; Taylor, V. L.; Groleau, M.-C.; Faith, D. R.; Schmidt, A. K.; Lamma, T. L.; Brooks, D. M.; Deziel, E.; Maxwell, K. L.; Secor, P. R.\nScore: 14.5, Published: 2023-08-26 DOI: 10.1101/2023.08.25.554831\nQuorum sensing, a bacterial signaling system that coordinates group behaviors as a function of cell density, plays an important role in regulating viral (phage) defense mechanisms in bacteria. The opportunistic pathogen Pseudomonas aeruginosa is a model system for the study of quorum sensing. P. aeruginosa is also frequently infected by Pf prophages that integrate into the host chromosome. Upon induction, Pf phages suppress host quorum sensing systems; however, the physiological relevance and mechanism of suppression are unknown. Here, we identify the Pf phage protein PfsE as an inhibitor of Pseudomonas Quinolone Signal (PQS) quorum sensing. PfsE binds to the host protein PqsA, which is essential for the biosynthesis of the PQS signaling molecule. Inhibition of PqsA increases the replication efficiency of Pf virions when infecting a new host and when the Pf prophage switches from lysogenic replication to active virion replication. In addition to inhibiting PQS signaling, our prior work demonstrates that PfsE also binds to PilC and inhibits type IV pili extension, protecting P. aeruginosa from infection by type IV pili-dependent phages. Overall, this work suggests that the simultaneous inhibition of PQS signaling and type IV pili by PfsE may be a viral strategy to suppress host defenses to promote Pf replication while at the same time protecting the susceptible host from competing phages. Abbreviated summaryQuorum sensing regulates phage defense in Pseudomonas aeruginosa. The Pf phage protein PfsE inhibits PQS-mediated quorum sensing by binding to the host enzyme PqsA, while also protecting against type IV pili-dependent phage infection. This dual inhibition strategy promotes Pf replication and safeguards the host from competing phages.\nA ubiquitous mobile genetic element disarms a bacterial antagonist of the gut microbiota\nAuthors: Sheahan, M. L.; Coyne, M. J.; Flores, K.; Garcia-Bayona, L.; Chatzidaki-Livanis, M.; Sundararajan, A.; Holst, A. Q.; Barquera, B.; Comstock, L. E.\nScore: 13.0, Published: 2023-08-26 DOI: 10.1101/2023.08.25.553775\nDNA transfer is ubiquitous in the gut microbiota, especially among species of Bacteroidales. In silico analyses have revealed hundreds of mobile genetic elements shared between these species, yet little is known about the phenotypes they encode, their effects on fitness, or pleiotropic consequences for the recipient's genome. Here, we show that acquisition of a ubiquitous integrative and conjugative element encoding an antagonistic system shuts down the native contact-dependent antagonistic system of . Despite inactivating the native antagonism system, mobile element acquisition increases fitness of the transconjugant over its progenitor by arming it with a new weapon. This DNA transfer causes the strain to change allegiances so that it no longer targets ecosystem members containing the same element yet is armed for communal defense.\nVirological characteristics of the SARS-CoV-2 XBB.1.5 variant\nAuthors: Tamura, T.; Irie, T.; Deguchi, S.; Yajima, H.; Tsuda, M.; Nasser, H.; Mizuma, K.; Plianchaisuk, A.; Suzuki, S.; Uriu, K.; Begum, M. M.; Shimizu, R.; Jonathan, M.; Suzuki, R.; Kondo, T.; Ito, H.; Kamiyama, A.; Yoshimatsu, K.; Shofa, M.; Hashimoto, R.; Anraku, Y.; Kimura, K. T.; Kita, S.; Sasaki, J.; Sasaki-Tabata, K.; Maenaka, K.; Nao, N.; Wang, L.; Oda, Y.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Ikeda, T.; Saito, A.; Matsuno, K.; Ito, J.; Tanaka, S.; Sato, K.; Hashiguchi, T.; Takayama, K.; Fukuhara, T.\nScore: 38.2, Published: 2023-08-17 DOI: 10.1101/2023.08.16.553332\nCirculation of SARS-CoV-2 Omicron XBB has resulted in the emergence of XBB.1.5, a new Variant of Interest. Our phylogenetic analysis suggests that XBB.1.5 evolved from XBB.1 by acquiring the F486P spike (S) mutation, subsequent to the acquisition of a nonsense mutation in ORF8. Neutralization assays showed similar abilities of immune escape between XBB.1.5 and XBB.1. We determined the structural basis for the interaction between human ACE2 and the S protein of XBB.1.5, showing similar overall structures between the S proteins of XBB.1 and XBB.1.5. The intrinsic pathogenicity of XBB.1.5 in hamsters is lower than that of XBB.1. Importantly, we found that the ORF8 nonsense mutation of XBB.1.5 resulted in impairment of MHC expression. In vivo experiments using recombinant viruses revealed that the XBB.1.5 mutations are involved with reduced virulence of XBB.1.5. Together, these data suggest that the mutations in ORF8 and S could enhance spreading of XBB.1.5 in humans.\nInnate immune control of influenza virus interspecies adaptation\nAuthors: Denz, P. J.; Speaks, S.; Kenney, A. D.; Eddy, A. C.; Papa, J. L.; Roettger, J.; Scace, S. C.; Hemann, E. A.; Forero, A.; Webby, R. J.; Bowman, A. S.; Yount, J. S.\nScore: 11.8, Published: 2023-08-23 DOI: 10.1101/2023.08.23.554491\nInfluenza virus pandemics are caused by viruses from animal reservoirs that adapt to efficiently infect and replicate in human hosts. Here, we investigated whether Interferon-Induced Transmembrane Protein 3 (IFITM3), a host antiviral factor with known human deficiencies, plays a role in interspecies virus infection and adaptation. We found that IFITM3-deficient mice and human cells could be infected with low doses of avian influenza viruses that failed to infect WT counterparts, identifying a new role for IFITM3 in controlling the minimum infectious viral dose threshold. Remarkably, influenza viruses passaged through Ifitm3-/- mice exhibited enhanced host adaptation, a result that was distinct from passaging in mice deficient for interferon signaling, which caused virus attenuation. Our data demonstrate that IFITM3 deficiency uniquely facilitates zoonotic influenza virus infections and subsequent adaptation, implicating IFITM3 deficiencies in the human population as a vulnerability for emergence of new pandemic viruses.\nAntiviral efficacy of the SARS-CoV-2 XBB breakthrough infection sera against Omicron subvariants including EG.5\nAuthors: Kaku, Y.; Kosugi, Y.; Uriu, K.; Ito, J.; Kuramochi, J.; Sadamasu, K.; Yoshimura, K.; Asakura, H.; Nagashima, M.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Sato, K.\nScore: 1038.6, Published: 2023-08-08 DOI: 10.1101/2023.08.08.552415\nAs of July 2023, EG.5.1 (a.k.a. XBB.1.9.2.5.1), a XBB subvariant bearing the S:Q52H and S:F456L substitutions, alongside the S:F486P substitution (Figure S1A), has rapidly spread in some countries. On July 19, 2023, the WHO classified EG.5 as a variant under monitoring. First, we showed that EG.5.1 exhibits a higher effective reproduction number compared with XBB.1.5, XBB.1.16, and its parental lineage (XBB.1.9.2), suggesting that EG.5.1 will spread globally and outcompete these XBB subvariants in the near future. We then addressed whether EG.5.1 evades from the antiviral effect of the humoral immunity induced by breakthrough infection (BTI) of XBB subvariants and performed a neutralization assay using XBB BTI sera. However, the 50% neutralization titer (NT50) of XBB BTI sera against EG.5.1 was comparable to those against XBB.1.5/1.9.2 and XBB.1.16. Moreover, the sensitivity of EG.5.1 to convalescent sera of XBB.1- and XBB.1.5-infected hamsters was similar to those of XBB.1.5/1.9 and XBB.1.16. These results suggest that the increased Re of EG.5.1 is attributed to neither increased infectivity nor immune evasion from XBB BTI, and the emergence and spread of EG.5 is driven by the other pressures. We previously demonstrated that Omicron BTI cannot efficiently induce antiviral humoral immunity against the variant infected. In fact, the NT50s of the BTI sera of Omicron BA.1, BA.2, and BA.5 against the variant infected were 3.0-, 2.2-, and 3.4-fold lower than that against the ancestral B.1.1 variant, respectively. However, strikingly, we found that the NT50 of the BTI sera of XBB1.5/1.9 and XBB.1.16 against the variant infected were 8.7- and 8.3-fold lower than that against the B.1.1 variant. These results suggest that XBB BTI cannot efficiently induce antiviral humoral immunity against XBB subvariants. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=147 SRC=\"FIGDIR/small/552415v1_figs1.gif\" ALT=\"Figure 1\"\u003e View larger version (39K): org.highwire.dtl.DTLVardef@f95376org.highwire.dtl.DTLVardef@d66fa8org.highwire.dtl.DTLVardef@3c8841org.highwire.dtl.DTLVardef@15824c_HPS_FORMAT_FIGEXP M_FIG O_FLOATNOFigure S1.C_FLOATNO Virological features of EG.5.1 and XBB BTI (A) Frequency of mutations of interest in the representative XBB sublineages. Only mutations with a frequency \u003e0.5 in at least one but not all the representative sublineages are shown. Note that the S proteins of XBB.1.5 and XBB.1.9.2 are identical. (B) Estimated epidemic dynamics of the representative XBB sublineages in countries where \u003e50 sequences of EG.5.1, XBB.1.5, XBB.1.9.2, and XBB.1.16 were detected from April 1, 2023 to July 13, 2023. Countries are ordered according to the number of detected sequences of EG.5.1. Line, posterior mean; ribbon, 95% Bayesian confidence interval. The dynamics for EG.5.1 is highlighted by a red arrowhead. (C) Estimated relative Re of the representative XBB sublineages in the six countries. The relative Re of XBB.1.5 is set to 1 (horizontal dashed line). Violin, posterior distribution; dot, posterior mean; line, 95% Bayesian confidence interval. (D) Lentivirus-based pseudovirus assay. HOS-ACE2-TMPRSS2 cells were infected with pseudoviruses bearing each S protein. The amount of input virus was normalized to the amount of HIV-1 p24 capsid protein. The percentage infectivity of XBB.1.5/1.9.2, XBB.1.5/1.9.2+Q52H, XBB.1.5/1.9.2+F456L, and EG.5.1 compared to that of XBB.1.5/1.9.2 are shown. The horizontal dash line indicates the mean value of the percentage infectivity of the XBB.1.5/1.9.2. Assays were performed in quadruplicate. The presented data are expressed as the average {+/-} SD. Each dot indicates the result of an individual replicate. (E-G) Neutralization assay. Assays were performed with pseudoviruses harboring the S proteins of B.1.1, BA.1, BA.2, BA.5, BQ.1.1, XBB.1, XBB.1.5/1.9.2, XBB.1.16, EG.5.1, XBB.1.5/1.9.2+Q52H, and XBB.1.5/1.9.2+F456L. The following sera were used: convalescent sera from fully vaccinated individuals who had been infected with XBB.1.5 (one 3-dose vaccinated. 1 donor in total), XBB.1.9 (one 3-dose vaccinated donor, one 4-dose vaccinated donor and one 5-dose vaccinated donor. 3 donors in total), and XBB.1.16 (one 2-dose vaccinated donor, two 3-dose vaccinated donors, and one 4-dose vaccinated donor. 4 donors in total) (E); sera from hamster infected with XBB.1 (left) or XBB.1.5 (right) (F); and convalescent sera from fully vaccinated individuals who had been infected with BA.1 (thirteen 2-dose vaccinated, 13 donors in total) (left)1, BA.2 (nine 2-dose vaccinated and four 3-dose vaccinated donors. 13 donors in total) (middle)19, and BA.5 (one 2-dose vaccinated, thirteen 3-dose vaccinated donors, and one 4-dose vaccinated. 15 donors in total) (right)19 (G). Each dot indicates the result of an individual replicate. Assays for each serum sample were performed in triplicate to determine the 50% neutralization titer (NT50). Each dot represents one NT50 value, and the geometric mean and 95% confidence interval are shown. The number in parenthesis indicates the mean of NT50 values. The horizontal dash line indicates the detection limit (120-fold). In D, statistically significant differences (**, P \u003c 0.001, ***, P \u003c 0.0001) versus XBB.1.5/1.9.2 were determined by two-sided Students t tests. Blue asterisks indicate decreased percentage of infectivity. In E and G, statistically significant differences versus B.1.1 were determined by two-sided Wilcoxon signed-rank tests. The fold change between B.1.1 and the variant indicated is shown in red. Background information on the convalescent donors is summarized in Table S1. In F, statistically significant differences (*, P \u003c 0.01, **, P \u003c 0.001) between B.1.1 and XBB.1 (left) or XBB.1.5/1.9.2 (right) were determined by two-sided Wilcoxon signed-rank tests and indicated with asterisks. Red asterisks indicate decreased NT50s. C_FIG\n",
  "wordCount" : "3195",
  "inLanguage": "en",
  "datePublished": "2023-08-30T10:38:23Z",
  "dateModified": "2023-08-30T10:38:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta"><span>updated on August 30, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.13.553148">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.13.553148" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.13.553148">
        <p class="paperTitle">SARS-CoV-2 Neutralizing Antibodies Following a Second BA.5 Bivalent Booster</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.13.553148" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.13.553148" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Q.; Bowen, A.; Ho, J.; Zhang, R.; Valdez, R.; Stoneman, E.; Gordon, A.; Lihong, L.; Ho, D. D.</p>
        <p class="info">Score: 355.7, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.13.553148' target='https://doi.org/10.1101/2023.08.13.553148'> 10.1101/2023.08.13.553148</a></p>
        <p class="abstract">Bivalent COVID-19 mRNA vaccines expressing both the ancestral D614G and Omicron BA.5 spike proteins were introduced in August 2022 with the goal of broadening immunity to emerging SARS-CoV-2 Omicron subvariants. Subsequent studies on bivalent boosters found neutralizing antibody responses similar to boosters with the original monovalent vaccine, likely the result of immunological imprinting. Guidelines allow for administration of a second bivalent booster in high-risk groups, but it remains unknown whether this would broaden antibody responses. To address this question, we assessed longitudinal serum SARS-CoV-2-neutralizing titers in 18 elderly immunocompetent individuals (mean age 69) following a fourth monovalent booster and two BA.5 bivalent booster vaccines using pseudovirus neutralization assays against D614G, Omicron BA.5, and Omicron XBB.1.5. There was a small but significant increase in peak neutralizing antibody responses against Omicron BA.5 and XBB.1.5 following the first bivalent booster, but no significant increase in peak titers following the second bivalent booster. Omicron-specific neutralizing titers remained low after both doses of the BA.5 bivalent booster. Our results suggest that a second dose of the BA.5 bivalent booster is not sufficient to broaden antibody responses and to overcome immunological imprinting. A monovalent vaccine targeting only the spike of the recently dominant SARS-CoV-2 may mitigate the &#34;back boosting&#34; associated with the &#34;original antigenic sin.&#34;</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.553968">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.553968" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.553968">
        <p class="paperTitle">Antibody Neutralization of Emerging SARS-CoV-2: EG.5.1 and XBC.1.6</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.553968" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.553968" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Q.; Guo, Y.; Zhang, R. M.; Ho, J.; Mohri, H.; Valdez, R.; Manthei, D. M.; Gordon, A.; Liu, L.; Ho, D. D.</p>
        <p class="info">Score: 32.3, Published: 2023-08-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.553968' target='https://doi.org/10.1101/2023.08.21.553968'> 10.1101/2023.08.21.553968</a></p>
        <p class="abstract">SARS-CoV-2 variants EG.5.1 and XBC.1.6 have recently emerged, attracting increased attention due to their rapid expansion globally and in Australia, respectively. EG.5.1 evolved from Omicron subvariant XBB.1.9, harboring additional Q52H and F456L spike substitutions. The F456L mutation is located within the epitopes of many class-1 monoclonal antibodies (mAbs) directed to the receptor-binding domain (RBD), raising concerns about further antibody evasion. XBC.1.6, a descendant of a Delta-BA.2 recombinant, carries 15 additional spike mutations. The extent to which antibody evasion contributes to the growth advantage of XBC.1.6 in Australia remains to be determined. To assess the antibody evasion properties of the emergent variants, we conducted pseudovirus neutralization assays using sera from individuals who received three doses of COVID-19 mRNA monovalent vaccines plus one dose of a BA.5 bivalent vaccine, as well as from patients with BQ or XBB breakthrough infection. The assays were also performed using a panel of 14 mAbs that retained neutralizing activity against prior XBB subvariants. Our data suggested that EG.5.1 was slightly but significantly more resistant (&lt; 2-fold) to neutralization by BQ and XBB breakthrough sera than XBB.1.16, which is known to be antigenically similar to XBB.1.5. Moreover, the F456L mutation in EG.5.1 conferred heightened resistance to certain RBD class-1 mAbs. In contrast, XBC.1.6 was more sensitive to neutralization by sera and mAbs than the XBB subvariants. Notably, XBB breakthrough sera retained only weak neutralization activity against XBB subvariants. In summary, EG.5.1 and XBC.1.6 exhibited distinct antibody evasion properties. The recent global expansion of EG.5.1 might be attributable, in part, to its enhanced neutralization resistance. That XBB breakthrough infections did not elicit a robust antibody neutralization response against XBB subvariants is indicative of immunological imprinting. The high prevalence of XBC.1.6 in Australia is not due to enhanced antibody evasion.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554661">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554661" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554661">
        <p class="paperTitle">Disruption of Aldehyde Dehydrogenase 2 protects against bacterial infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554661" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554661" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Berry, S.; Espich, S.; Thuong, N. T. T.; Chang, X.; Dorajoo, R.; Khor, C. C.; Heng, C. K.; Yuan, J.-M.; Fox, D.; Anaya-Sanchez, A.; Tenney, L.; Chang, C. J.; Kotov, D. I.; Vance, R. E.; Dunstan, S. J.; Darwin, K. H.; Stanley, S. A.</p>
        <p class="info">Score: 26.7, Published: 2023-08-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554661' target='https://doi.org/10.1101/2023.08.24.554661'> 10.1101/2023.08.24.554661</a></p>
        <p class="abstract">The ALDH2*2 (rs671) allele is one of the most common genetic mutations in humans, yet the positive evolutionary selective pressure to maintain this mutation is unknown, despite its association with adverse health outcomes. ALDH2 is responsible for the detoxification of metabolically produced aldehydes, including lipid-peroxidation end products derived from inflammation. Here, we demonstrate that host-derived aldehydes 4-hydroxynonenal (4HNE), malondialdehyde (MDA), and formaldehyde (FA), all of which are metabolized by ALDH2, are directly toxic to the bacterial pathogens Mycobacterium tuberculosis and Francisella tularensis at physiological levels. We find that Aldh2 expression in macrophages is decreased upon immune stimulation, and that bone marrow-derived macrophages from Aldh2-/- mice contain elevated aldehydes relative to wild-type mice. Macrophages deficient for Aldh2 exhibited enhanced control of Francisella infection. Finally, mice lacking Aldh2 demonstrated increased resistance to pulmonary infection by M. tuberculosis, including in a hypersusceptible model of tuberculosis, and were also resistant to Francisella infection. We hypothesize that the absence of ALDH2 contributes to the hosts ability to control infection by pathogens such as M. tuberculosis and F. tularensis, and that host-derived aldehydes act as antimicrobial factors during intracellular bacterial infections.

One sentence summaryAldehydes produced by host cells contribute to the control of bacterial infections.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554448">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554448" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554448">
        <p class="paperTitle">MiDAS 5: Global diversity of bacteria and archaea in anaerobic digesters</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554448" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554448" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dueholm, M. K. D.; Andersen, K. S.; Petersen, A.-K. C.; Rudkjoebing, V.; MiDAS Global Consortium for Anaerobic Digesters,  ; Nielsen, P. H.</p>
        <p class="info">Score: 17.0, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554448' target='https://doi.org/10.1101/2023.08.24.554448'> 10.1101/2023.08.24.554448</a></p>
        <p class="abstract">Anaerobic digestion represents a key biotechnology for the transformation of organic waste into renewable energy (biogas) and relies on complex microbial communities that work in concert to degrade the complex substrates into methane and carbon dioxide. Here, we sequenced more than half a million high-quality, full-length 16S rRNA gene sequences from 285 full-scale anaerobic digesters (ADs) across the world to expand our knowledge about diversity and function of the bacteria and archaea in ADs. The sequences were processed into full-length 16S rRNA amplicon sequence variants (FL-ASVs), which were added to the MiDAS 4 database for bacteria and archaea in wastewater treatment systems to create  MiDAS 5. The expansion of the MiDAS database significantly increased the coverage for bacteria and archaea in ADs worldwide, leading to an improved rate of genus and species-level classification. Using MiDAS 5, we carried out an amplicon-based, global-scale microbial community profiling of the sampled ADs using three common sets of primers targeting different regions of the 16S rRNA gene in bacteria and/or archaea. We revealed how environmental conditions and biogeography shape the AD microbiota. We also identify core and conditionally rare or abundant taxa, encompassing less than 1000 genera and species. These represent 83-99% and 10-64% of the accumulated read abundance respectively, across samples depending on the amplicon primers used. Finally, we examined the global diversity of functional groups with known importance for the anaerobic digestion process. Our online global MiDAS Field Guide presents the data generated in this study and summarizes present knowledge about all taxa.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.554849">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.554849" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.554849">
        <p class="paperTitle">Dissecting the neurotropism and neurovirulence of MPXV using human stem cell-based models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.554849" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.554849" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bauer, L.; Giussani, S.; Palazzi, N.; Zare, F.; Colombo, E.; Pinci, F.; Leijten, L.; Smeenk, H.; Embregts, C. W. E.; Silva, M.; Spoor, J. K. H.; Dirven, C.; Gao, Z.; Bolleboom, A.; Verstrepen, B. E.; Schuele, L.; de Vrij, F. M. S.; Kushner, S. A.; Oude Munnink, B. B.; Davila-Velderrain, J.; van Riel, D.; Harschnitz, O.</p>
        <p class="info">Score: 14.9, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.554849' target='https://doi.org/10.1101/2023.08.25.554849'> 10.1101/2023.08.25.554849</a></p>
        <p class="abstract">Mpox, formerly known as monkeypox, is a zoonotic illness of international concern that can lead to severe disease including neurological sequelae. However, it remains unclear what the neurotropism of monkeypox virus (MPXV) is and how MPXV infection leads to neurological deficits. Here, we determined the neurotropism and neurovirulence of MPXV using human pluripotent stem cell- (hPSC)-derived neural stem cells, astrocytes, cortical neurons, and microglia together with ex vivo human brain tissue. We found that MPXV infects and replicates more efficiently in astrocytes and microglia compared to cortical neurons, which unlike glial cells showed activation of distinct antiviral programs that may confer differential susceptibility to MPXV. Ex vivo infection of human brain tissue confirmed the susceptibility of astrocytes to MPXV infection, which also had the strongest disease-associated changes. Molecular pathway analyses revealed induction of cellular senescence and a senescence-associated secretory phenotype upon MPXV infection in astrocytes. Finally, we demonstrated that antiviral treatment using tecovirimat inhibits MPXV replication and prevents virus-induced senescence in hPSC-derived astrocytes. Altogether, leveraging hPSC-derived brain cells, we reveal MPXV-induced cell type-specific effects at the molecular and cellular level, which provide important insights into the neuropathogenesis of MPXV infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.554831">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.554831" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.554831">
        <p class="paperTitle">Inhibition of PQS signaling by the Pf bacteriophage protein PfsE enhances viral replication in Pseudomonas aeruginosa</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.554831" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.554831" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schwartzkopf, C. M.; Taylor, V. L.; Groleau, M.-C.; Faith, D. R.; Schmidt, A. K.; Lamma, T. L.; Brooks, D. M.; Deziel, E.; Maxwell, K. L.; Secor, P. R.</p>
        <p class="info">Score: 14.5, Published: 2023-08-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.554831' target='https://doi.org/10.1101/2023.08.25.554831'> 10.1101/2023.08.25.554831</a></p>
        <p class="abstract">Quorum sensing, a bacterial signaling system that coordinates group behaviors as a function of cell density, plays an important role in regulating viral (phage) defense mechanisms in bacteria. The opportunistic pathogen Pseudomonas aeruginosa is a model system for the study of quorum sensing. P. aeruginosa is also frequently infected by Pf prophages that integrate into the host chromosome. Upon induction, Pf phages suppress host quorum sensing systems; however, the physiological relevance and mechanism of suppression are unknown. Here, we identify the Pf phage protein PfsE as an inhibitor of Pseudomonas Quinolone Signal (PQS) quorum sensing. PfsE binds to the host protein PqsA, which is essential for the biosynthesis of the PQS signaling molecule. Inhibition of PqsA increases the replication efficiency of Pf virions when infecting a new host and when the Pf prophage switches from lysogenic replication to active virion replication. In addition to inhibiting PQS signaling, our prior work demonstrates that PfsE also binds to PilC and inhibits type IV pili extension, protecting P. aeruginosa from infection by type IV pili-dependent phages. Overall, this work suggests that the simultaneous inhibition of PQS signaling and type IV pili by PfsE may be a viral strategy to suppress host defenses to promote Pf replication while at the same time protecting the susceptible host from competing phages.

Abbreviated summaryQuorum sensing regulates phage defense in Pseudomonas aeruginosa. The Pf phage protein PfsE inhibits PQS-mediated quorum sensing by binding to the host enzyme PqsA, while also protecting against type IV pili-dependent phage infection. This dual inhibition strategy promotes Pf replication and safeguards the host from competing phages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.553775">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.553775" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.553775">
        <p class="paperTitle">A ubiquitous mobile genetic element disarms a bacterial antagonist of the gut microbiota</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.553775" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.553775" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sheahan, M. L.; Coyne, M. J.; Flores, K.; Garcia-Bayona, L.; Chatzidaki-Livanis, M.; Sundararajan, A.; Holst, A. Q.; Barquera, B.; Comstock, L. E.</p>
        <p class="info">Score: 13.0, Published: 2023-08-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.553775' target='https://doi.org/10.1101/2023.08.25.553775'> 10.1101/2023.08.25.553775</a></p>
        <p class="abstract">DNA transfer is ubiquitous in the gut microbiota, especially among species of Bacteroidales. In silico analyses have revealed hundreds of mobile genetic elements shared between these species, yet little is known about the phenotypes they encode, their effects on fitness, or pleiotropic consequences for the recipient&#39;s genome. Here, we show that acquisition of a ubiquitous integrative and conjugative element encoding an antagonistic system shuts down the native contact-dependent antagonistic system of &lt;Bacteroides fragilis&gt;. Despite inactivating the native antagonism system, mobile element acquisition increases fitness of the &lt;B. fragilis&gt; transconjugant over its progenitor by arming it with a new weapon. This DNA transfer causes the strain to change allegiances so that it no longer targets ecosystem members containing the same element yet is armed for communal defense.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.553332">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.553332" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.553332">
        <p class="paperTitle">Virological characteristics of the SARS-CoV-2 XBB.1.5 variant</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.553332" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.553332" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tamura, T.; Irie, T.; Deguchi, S.; Yajima, H.; Tsuda, M.; Nasser, H.; Mizuma, K.; Plianchaisuk, A.; Suzuki, S.; Uriu, K.; Begum, M. M.; Shimizu, R.; Jonathan, M.; Suzuki, R.; Kondo, T.; Ito, H.; Kamiyama, A.; Yoshimatsu, K.; Shofa, M.; Hashimoto, R.; Anraku, Y.; Kimura, K. T.; Kita, S.; Sasaki, J.; Sasaki-Tabata, K.; Maenaka, K.; Nao, N.; Wang, L.; Oda, Y.; The Genotype to Phenotype Japan (G2P-Japan) Consortium,  ; Ikeda, T.; Saito, A.; Matsuno, K.; Ito, J.; Tanaka, S.; Sato, K.; Hashiguchi, T.; Takayama, K.; Fukuhara, T.</p>
        <p class="info">Score: 38.2, Published: 2023-08-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.553332' target='https://doi.org/10.1101/2023.08.16.553332'> 10.1101/2023.08.16.553332</a></p>
        <p class="abstract">Circulation of SARS-CoV-2 Omicron XBB has resulted in the emergence of XBB.1.5, a new Variant of Interest. Our phylogenetic analysis suggests that XBB.1.5 evolved from XBB.1 by acquiring the F486P spike (S) mutation, subsequent to the acquisition of a nonsense mutation in ORF8. Neutralization assays showed similar abilities of immune escape between XBB.1.5 and XBB.1. We determined the structural basis for the interaction between human ACE2 and the S protein of XBB.1.5, showing similar overall structures between the S proteins of XBB.1 and XBB.1.5. The intrinsic pathogenicity of XBB.1.5 in hamsters is lower than that of XBB.1. Importantly, we found that the ORF8 nonsense mutation of XBB.1.5 resulted in impairment of MHC expression. In vivo experiments using recombinant viruses revealed that the XBB.1.5 mutations are involved with reduced virulence of XBB.1.5. Together, these data suggest that the mutations in ORF8 and S could enhance spreading of XBB.1.5 in humans.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554491">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554491" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554491">
        <p class="paperTitle">Innate immune control of influenza virus interspecies adaptation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554491" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554491" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Denz, P. J.; Speaks, S.; Kenney, A. D.; Eddy, A. C.; Papa, J. L.; Roettger, J.; Scace, S. C.; Hemann, E. A.; Forero, A.; Webby, R. J.; Bowman, A. S.; Yount, J. S.</p>
        <p class="info">Score: 11.8, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554491' target='https://doi.org/10.1101/2023.08.23.554491'> 10.1101/2023.08.23.554491</a></p>
        <p class="abstract">Influenza virus pandemics are caused by viruses from animal reservoirs that adapt to efficiently infect and replicate in human hosts. Here, we investigated whether Interferon-Induced Transmembrane Protein 3 (IFITM3), a host antiviral factor with known human deficiencies, plays a role in interspecies virus infection and adaptation. We found that IFITM3-deficient mice and human cells could be infected with low doses of avian influenza viruses that failed to infect WT counterparts, identifying a new role for IFITM3 in controlling the minimum infectious viral dose threshold. Remarkably, influenza viruses passaged through Ifitm3-/- mice exhibited enhanced host adaptation, a result that was distinct from passaging in mice deficient for interferon signaling, which caused virus attenuation. Our data demonstrate that IFITM3 deficiency uniquely facilitates zoonotic influenza virus infections and subsequent adaptation, implicating IFITM3 deficiencies in the human population as a vulnerability for emergence of new pandemic viruses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552415">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552415" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552415">
        <p class="paperTitle">Antiviral efficacy of the SARS-CoV-2 XBB breakthrough infection sera against Omicron subvariants including EG.5</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552415" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552415" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kaku, Y.; Kosugi, Y.; Uriu, K.; Ito, J.; Kuramochi, J.; Sadamasu, K.; Yoshimura, K.; Asakura, H.; Nagashima, M.; The Genotype to Phenotype Japan (G2P-Japan) Consortium,  ; Sato, K.</p>
        <p class="info">Score: 1038.6, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552415' target='https://doi.org/10.1101/2023.08.08.552415'> 10.1101/2023.08.08.552415</a></p>
        <p class="abstract">As of July 2023, EG.5.1 (a.k.a. XBB.1.9.2.5.1), a XBB subvariant bearing the S:Q52H and S:F456L substitutions, alongside the S:F486P substitution (Figure S1A), has rapidly spread in some countries. On July 19, 2023, the WHO classified EG.5 as a variant under monitoring. First, we showed that EG.5.1 exhibits a higher effective reproduction number compared with XBB.1.5, XBB.1.16, and its parental lineage (XBB.1.9.2), suggesting that EG.5.1 will spread globally and outcompete these XBB subvariants in the near future. We then addressed whether EG.5.1 evades from the antiviral effect of the humoral immunity induced by breakthrough infection (BTI) of XBB subvariants and performed a neutralization assay using XBB BTI sera. However, the 50% neutralization titer (NT50) of XBB BTI sera against EG.5.1 was comparable to those against XBB.1.5/1.9.2 and XBB.1.16. Moreover, the sensitivity of EG.5.1 to convalescent sera of XBB.1- and XBB.1.5-infected hamsters was similar to those of XBB.1.5/1.9 and XBB.1.16. These results suggest that the increased Re of EG.5.1 is attributed to neither increased infectivity nor immune evasion from XBB BTI, and the emergence and spread of EG.5 is driven by the other pressures. We previously demonstrated that Omicron BTI cannot efficiently induce antiviral humoral immunity against the variant infected. In fact, the NT50s of the BTI sera of Omicron BA.1, BA.2, and BA.5 against the variant infected were 3.0-, 2.2-, and 3.4-fold lower than that against the ancestral B.1.1 variant, respectively. However, strikingly, we found that the NT50 of the BTI sera of XBB1.5/1.9 and XBB.1.16 against the variant infected were 8.7- and 8.3-fold lower than that against the B.1.1 variant. These results suggest that XBB BTI cannot efficiently induce antiviral humoral immunity against XBB subvariants.

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=147 SRC=&#34;FIGDIR/small/552415v1_figs1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (39K):
org.highwire.dtl.DTLVardef@f95376org.highwire.dtl.DTLVardef@d66fa8org.highwire.dtl.DTLVardef@3c8841org.highwire.dtl.DTLVardef@15824c_HPS_FORMAT_FIGEXP  M_FIG O_FLOATNOFigure S1.C_FLOATNO Virological features of EG.5.1 and XBB BTI (A) Frequency of mutations of interest in the representative XBB sublineages. Only mutations with a frequency &gt;0.5 in at least one but not all the representative sublineages are shown. Note that the S proteins of XBB.1.5 and XBB.1.9.2 are identical.

(B) Estimated epidemic dynamics of the representative XBB sublineages in countries where &gt;50 sequences of EG.5.1, XBB.1.5, XBB.1.9.2, and XBB.1.16 were detected from April 1, 2023 to July 13, 2023. Countries are ordered according to the number of detected sequences of EG.5.1. Line, posterior mean; ribbon, 95% Bayesian confidence interval. The dynamics for EG.5.1 is highlighted by a red arrowhead.

(C) Estimated relative Re of the representative XBB sublineages in the six countries. The relative Re of XBB.1.5 is set to 1 (horizontal dashed line). Violin, posterior distribution; dot, posterior mean; line, 95% Bayesian confidence interval.

(D) Lentivirus-based pseudovirus assay. HOS-ACE2-TMPRSS2 cells were infected with pseudoviruses bearing each S protein. The amount of input virus was normalized to the amount of HIV-1 p24 capsid protein. The percentage infectivity of XBB.1.5/1.9.2, XBB.1.5/1.9.2&#43;Q52H, XBB.1.5/1.9.2&#43;F456L, and EG.5.1 compared to that of XBB.1.5/1.9.2 are shown. The horizontal dash line indicates the mean value of the percentage infectivity of the XBB.1.5/1.9.2. Assays were performed in quadruplicate. The presented data are expressed as the average {&#43;/-} SD. Each dot indicates the result of an individual replicate.

(E-G) Neutralization assay. Assays were performed with pseudoviruses harboring the S proteins of B.1.1, BA.1, BA.2, BA.5, BQ.1.1, XBB.1, XBB.1.5/1.9.2, XBB.1.16, EG.5.1, XBB.1.5/1.9.2&#43;Q52H, and XBB.1.5/1.9.2&#43;F456L. The following sera were used: convalescent sera from fully vaccinated individuals who had been infected with XBB.1.5 (one 3-dose vaccinated. 1 donor in total), XBB.1.9 (one 3-dose vaccinated donor, one 4-dose vaccinated donor and one 5-dose vaccinated donor. 3 donors in total), and XBB.1.16 (one 2-dose vaccinated donor, two 3-dose vaccinated donors, and one 4-dose vaccinated donor. 4 donors in total) (E); sera from hamster infected with XBB.1 (left) or XBB.1.5 (right) (F); and convalescent sera from fully vaccinated individuals who had been infected with BA.1 (thirteen 2-dose vaccinated, 13 donors in total) (left)1, BA.2 (nine 2-dose vaccinated and four 3-dose vaccinated donors. 13 donors in total) (middle)19, and BA.5 (one 2-dose vaccinated, thirteen 3-dose vaccinated donors, and one 4-dose vaccinated. 15 donors in total) (right)19 (G). Each dot indicates the result of an individual replicate. Assays for each serum sample were performed in triplicate to determine the 50% neutralization titer (NT50). Each dot represents one NT50 value, and the geometric mean and 95% confidence interval are shown. The number in parenthesis indicates the mean of NT50 values. The horizontal dash line indicates the detection limit (120-fold).

In D, statistically significant differences (**, P &lt; 0.001, ***, P &lt; 0.0001) versus XBB.1.5/1.9.2 were determined by two-sided Students t tests. Blue asterisks indicate decreased percentage of infectivity.

In E and G, statistically significant differences versus B.1.1 were determined by two-sided Wilcoxon signed-rank tests. The fold change between B.1.1 and the variant indicated is shown in red. Background information on the convalescent donors is summarized in Table S1.

In F, statistically significant differences (*, P &lt; 0.01, **, P &lt; 0.001) between B.1.1 and XBB.1 (left) or XBB.1.5/1.9.2 (right) were determined by two-sided Wilcoxon signed-rank tests and indicated with asterisks. Red asterisks indicate decreased NT50s.

C_FIG</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
