<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Escherichia coli plasmidome maps the game of clones
Authors: Arredondo-Alonso, S.; Pontinen, A. K.; Gama, J.; Gladstone, R. A.; Harms, K.; Tonkin-Hill, G.; Thorpe, H. A.; Simonsen, G. S.; Samuelsen, O.; Johnsen, P. J.; Corander, J.
Score: 43.4, Published: 2023-10-14 DOI: 10.1101/2023.10.14.562336
Escherichia coli is a major human pathogen and the most widely studied microbe in history, but its extrachromosomal elements known as plasmids remain poorly delineated. Here we used long-read technology to high-resolution sequence the entire plasmidome and the corresponding host chromosomes from a longitudinal survey covering two decades and over 2,000 E.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Escherichia coli plasmidome maps the game of clones
Authors: Arredondo-Alonso, S.; Pontinen, A. K.; Gama, J.; Gladstone, R. A.; Harms, K.; Tonkin-Hill, G.; Thorpe, H. A.; Simonsen, G. S.; Samuelsen, O.; Johnsen, P. J.; Corander, J.
Score: 43.4, Published: 2023-10-14 DOI: 10.1101/2023.10.14.562336
Escherichia coli is a major human pathogen and the most widely studied microbe in history, but its extrachromosomal elements known as plasmids remain poorly delineated. Here we used long-read technology to high-resolution sequence the entire plasmidome and the corresponding host chromosomes from a longitudinal survey covering two decades and over 2,000 E." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-22T10:37:23+00:00" />
<meta property="article:modified_time" content="2023-10-22T10:37:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Escherichia coli plasmidome maps the game of clones
Authors: Arredondo-Alonso, S.; Pontinen, A. K.; Gama, J.; Gladstone, R. A.; Harms, K.; Tonkin-Hill, G.; Thorpe, H. A.; Simonsen, G. S.; Samuelsen, O.; Johnsen, P. J.; Corander, J.
Score: 43.4, Published: 2023-10-14 DOI: 10.1101/2023.10.14.562336
Escherichia coli is a major human pathogen and the most widely studied microbe in history, but its extrachromosomal elements known as plasmids remain poorly delineated. Here we used long-read technology to high-resolution sequence the entire plasmidome and the corresponding host chromosomes from a longitudinal survey covering two decades and over 2,000 E."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Escherichia coli plasmidome maps the game of clones\nAuthors: Arredondo-Alonso, S.; Pontinen, A. K.; Gama, J.; Gladstone, R. A.; Harms, K.; Tonkin-Hill, G.; Thorpe, H. A.; Simonsen, G. S.; Samuelsen, O.; Johnsen, P. J.; Corander, J.\nScore: 43.4, Published: 2023-10-14 DOI: 10.1101/2023.10.14.562336\nEscherichia coli is a major human pathogen and the most widely studied microbe in history, but its extrachromosomal elements known as plasmids remain poorly delineated. Here we used long-read technology to high-resolution sequence the entire plasmidome and the corresponding host chromosomes from a longitudinal survey covering two decades and over 2,000 E.",
  "keywords": [
    
  ],
  "articleBody": " Escherichia coli plasmidome maps the game of clones\nAuthors: Arredondo-Alonso, S.; Pontinen, A. K.; Gama, J.; Gladstone, R. A.; Harms, K.; Tonkin-Hill, G.; Thorpe, H. A.; Simonsen, G. S.; Samuelsen, O.; Johnsen, P. J.; Corander, J.\nScore: 43.4, Published: 2023-10-14 DOI: 10.1101/2023.10.14.562336\nEscherichia coli is a major human pathogen and the most widely studied microbe in history, but its extrachromosomal elements known as plasmids remain poorly delineated. Here we used long-read technology to high-resolution sequence the entire plasmidome and the corresponding host chromosomes from a longitudinal survey covering two decades and over 2,000 E. coli isolates. Separation of chromosomal and extrachromosomal DNA enabled us to reconstruct co-evolutionary trajectories of host lineages and their plasmids on a population-wide scale, demonstrating that plasmid evolution is markedly constrained contrary to the established dogma. We find that some plasmids have persisted in lineages for centuries, and provide a high-resolution map of recent vertical and horizontal evolutionary events in plasmids with key antibiotic resistance, competition and virulence determinants. We present genomic evidence of both chromosomal and plasmid-driven success strategies that represent convergent phenotypic evolution in distant lineages, and use in vitro experiments to verify the importance of bacteriocin-producing plasmids for clone success. Our study has general implications for understanding plasmid biology and bacterial evolutionary strategies, and informing development of future interventions against pathogens.\nLong-term dynamics of the 'Serratia marcescens complex' in the hospital-built environment\nAuthors: Aracil-Gisbert, S.; Fernandez-de-Bobadilla, M. D.; Guerra-Pinto, N.; Serrano-Calleja, S.; Perez-Cobas, A. E.; Soriano, C.; De Pablo, R.; Lanza, V. F.; Perez-Viso, B.; Reuter, S.; Hasman, H.; Canton, R.; Baquero, F.; Coque, T. M.\nScore: 33.3, Published: 2023-10-15 DOI: 10.1101/2023.10.15.562376\nSerratia marcescens is an opportunistic pathogen historically associated with abrupt, unpredictable, and severe outbreaks in hospital intensive care units (ICUs) and, more recently, with the spread of acquired genes encoding carbapenem resistance. However, the population biology and ecology of S. marcescens in the hospital ecosystem is still poorly understood. Here, we combine epidemiological information of 1417 Serratia isolates collected from the sinks of a large ICU ward that underwent significant demographic and operational changes (2019-2020) and 99 non-redundant outbreak/non-outbreak isolates from the same hospital (2003-2019), with genomic data of 165 isolates. We first report hospital sinks as reservoirs of heterogeneous and coexistent populations of the S. marcescens complex (SMC). A novel SMC clade congruent with the recently described Serratia nevei species is predominant, exhibiting chromosomal AmpC {beta}-lactamase with an unusual basal expression in contrast to one of the major features of S. marcescens. Persistent Serratia sink strains are identical to those involved in clonal and polyclonal outbreaks of VIM-1 and OXA-48 producers since at least 2017. The \"source-sink\" dynamics of SMC populations able to acquire the highly conserved plasmids such as IncL carrying blaVIM-1 or blaOXA-48 offer novel insights that might improve interventions to control outbreaks and treat Serratia infections in high-risk hospital areas.\nTransertion and cell geometry organize the Escherichia coli nucleoid during rapid growth\nAuthors: Spahn, C.; Middlemiss, S.; Gomez-de-Mariscal, E.; Henriques, R.; Bode, H. B.; Holden, S.; Heilemann, M.\nScore: 27.5, Published: 2023-10-19 DOI: 10.1101/2023.10.16.562172\nABSTRACTUsing 3D single-molecule localization microscopy and live-cell imaging, we show that the Escherichia coli nucleoid adopts a condensed, membrane-associated configuration during rapid growth. To study the influence of different biosynthetic processes on nucleoid morphology and positioning, we recorded multi-colour super-resolution images during drug treatment. After developing analysis routines for confocal and super-resolution images, we captured highly resolved snapshots which revealed the complete loss of the membrane-bound state of the nucleoid within 10 minutes of halting transcription and translation. This indicates an active role of transertion (coupled transcription, translation and membrane insertion) in nucleoid organization. In contrast, cell wall synthesis inhibition only affects nucleoid organization during morphological changes. Further, we provide evidence that the E. coli nucleoid spatially correlates with MreB in unperturbed E. coli cells, while this correlation diminishes in cells with changed cell geometry or upon inhibition of protein biosynthesis. Replication inhibition experiments, as well as multi-drug treatments highlight the role of entropic effects and transcription in nucleoid condensation and positioning. In summary, we provide experimental evidence for transertion as a principal organiser of the bacterial nucleoid, and show that an altered metabolic state and antibiotic treatment both lead to major changes in the degree of transertion and overall spatial organization of the nucleoid. Our high-resolution characterization reveals dynamics of antibiotic action and provides tools to quantify bacterial chromosome organization. This does not provide valuable insights into the role of transertion, but can also be applied to study other cell- biological processes.\nGenetically refactored Agrobacterium-mediated transformation\nAuthors: Thompson, M.; Kirkpatrick, L. D.; Geiselman, G. M.; Waldburger, L. M.; Pearson, A. N.; Szarzanowicz, M.; Vuu, K. M.; Markel, K.; Hummel, N. F.; Suazo, D. D.; Tahmin, C.; Cui, R.; Liu, S.; Cevallos, J.; Pannu, H.; Liu, D.; Gin, J. W.; Chen, Y.; Petzold, C. J.; Gladden, J.; Keasling, J. D.; Chang, J. H.; Weisberg, A. J.; Shih, P. M.\nScore: 37.3, Published: 2023-10-13 DOI: 10.1101/2023.10.13.561914\nMembers of Agrobacterium are costly plant pathogens while also essential tools for plant transformation. Though Agrobacterium-mediated transformation (AMT) has been heavily studied, its polygenic nature and its complex transcriptional regulation make identifying the genetic basis of transformational efficiency difficult through traditional genetic and bioinformatic approaches. Here we use a bottom-up synthetic approach to systematically refactor the tumor-inducing plasmid, wherein the majority of AMT machine components are encoded, into a minimal set of genes capable of plant and fungal transformation that is both controllable and orthogonal to its environment. We demonstrate that engineered vectors can be transferred to new heterologous bacteria, enabling them to transform plants. Our reductionist approach demonstrates how bottom-up engineering can be used to dissect and elucidate the genetic underpinnings of complex biological traits, and may lead to the development of strains of bacteria more capable of transforming recalcitrant plant species of societal importance.\nPhages Produce Persisters\nAuthors: Fernandez-Garcia, L.; Kirigo, J.; Huelgas-Mendez, D.; Tomas, M.; Garcia-Contreras, R.; Wood, T. K.\nScore: 15.3, Published: 2023-10-17 DOI: 10.1101/2023.10.17.562728\nBacteria primarily encounter stress, and, arguably, their greatest threats are phages. It is often assumed that those bacteria that escape phage attack have mutated; however, another possibility is that a subpopulation forms the dormant persister state, in a manner similar to that demonstrated for bacterial cells undergoing nutritive, oxidative, and antibiotic stress. Persister cells do not undergo mutation and survive lethal conditions by ceasing growth transiently. Slower growth and dormancy play a key physiological role as they allow host phage defense systems more time to clear the phage infection. Here we investigated how bacteria survive lytic phage infection by isolating surviving cells from the plaques of T2, T4, and lambda (cI mutant) virulent phages. We found that bacteria in plaques can escape phage attack both by mutation (i.e., become resistant) and without mutation (i.e., become persistent). Specifically, whereas T4-resistant and lambda-resistant bacteria with over a 100,000-fold less sensitivity were isolated from plaques with obvious genetic mutations (e.g., causing mucoidy), cells were also found after T2 infection that undergo no significant mutation, retain wild-type phage sensitivity, and survive lethal doses of antibiotics. Corroborating this, adding T2 phage to persister cells resulted in 137,000-fold more survival compared to that of addition to exponentially-growing cells. Phage treatments with Klebsiella pneumonia and Pseudomonas aeruginosa also generated persister cells. Hence, along with resistant strains, bacteria also form persister cells during phage infection.\nKatG catalase deficiency confers bedaquiline hyper-susceptibility to isoniazid resistant Mycobacterium tuberculosis\nAuthors: Ofori-Anyinam, N.; Hamblin, M.; Coldren, M. L.; Li, B.; Mereddy, G.; Shaikh, M.; Shah, A.; Ranu, N.; Lu, S.; Blainey, P. C.; Ma, S.; Collins, J. J.; Yang, J. H.\nScore: 13.7, Published: 2023-10-17 DOI: 10.1101/2023.10.17.562707\nMultidrug-resistant tuberculosis (MDR-TB) is a growing source of global mortality and threatens global control of tuberculosis (TB) disease. The diarylquinoline bedaquiline (BDQ) recently emerged as a highly efficacious drug against MDR-TB, defined as resistance to the first-line drugs isoniazid (INH) and rifampin. INH resistance is primarily caused by loss-of-function mutations in the catalase KatG, but mechanisms underlying BDQs efficacy against MDR-TB remain unknown. Here we employ a systems biology approach to investigate BDQ hyper-susceptibility in INH-resistant Mycobacterium tuberculosis. We found hyper-susceptibility to BDQ in INH-resistant cells is due to several physiological changes induced by KatG deficiency, including increased susceptibility to reactive oxygen species and DNA damage, remodeling of transcriptional programs, and metabolic repression of folate biosynthesis. We demonstrate BDQ hyper-susceptibility is common in INH-resistant clinical isolates. Collectively, these results highlight how altered bacterial physiology can impact drug efficacy in drug-resistant bacteria.\nIntegration of 168,000 samples reveals global patterns of the human gut microbiome\nAuthors: Abdill, R. J.; Graham, S. P.; Rubinetti, V.; Albert, F. W.; Greene, C. S.; Davis, S.; Blekhman, R.\nScore: 88.1, Published: 2023-10-11 DOI: 10.1101/2023.10.11.560955\nUnderstanding the factors that shape variation in the human microbiome is a major goal of research in biology. While other genomics fields have used large, pre-compiled compendia to extract systematic insights requiring otherwise impractical sample sizes, there has been no comparable resource for the 16S rRNA sequencing data commonly used to quantify microbiome composition. To help close this gap, we have assembled a set of 168,484 publicly available human gut microbiome samples, processed with a single pipeline and combined into the largest unified microbiome dataset to date. We use this resource, which is freely available at microbiomap.org, to shed light on global variation in the human gut microbiome. We find that Firmicutes, particularly Bacilli and Clostridia, are almost universally present in the human gut. At the same time, the relative abundance of the 65 most common microbial genera differ between at least two world regions. We also show that gut microbiomes in undersampled world regions, such as Central and Southern Asia, differ significantly from the more thoroughly characterized microbiomes of Europe and Northern America. Moreover, humans in these overlooked regions likely harbor hundreds of taxa that have not yet been discovered due to this undersampling, highlighting the need for diversity in microbiome studies. We anticipate that this new compendium can serve the community and enable advanced applied and methodological research.\nMicrotubule inner proteins of Plasmodium are essential for transmission of malaria parasites\nAuthors: Hentzschel, F.; Binder, A. M.; Dorner, L. P.; Herzel, L.; Nuglish, F.; Sema, M.; Aguirre-Botero, M. C.; Cyrklaff, M.; Funaya, C.; Frischknecht, F.\nScore: 11.2, Published: 2023-10-19 DOI: 10.1101/2023.10.19.562943\nMicrotubule inner proteins, MIPs, are microtubule associated proteins that bind to tubulin from the luminal side. MIPs can be found in axonemes to stabilize flagellar beat or within cytoplasmic microtubules. Plasmodium spp. are the causative agents of malaria that feature different forms across a complex life cycle with both unique and divergent microtubule-based arrays. Here we investigate the role of four MIPs in a rodent malaria parasite for their role in transmission to and from the mosquito. We show by single and double gene deletions that SPM1 and TrxL1, MIPs associated with the subpellicular microtubules are dispensable for transmission from the vertebrate host to the mosquito and back. In contrast, FAP20 and FAP52, MIPs associated with the axonemes of gametes, are essential for transmission to mosquitoes but only if both genes are deleted. In the absence of both, FAP20 and FAP52 the B-tubule of the axoneme partly detaches from the A-tubule resulting in the deficiency of axonemal beating and hence gamete formation and egress. Our data suggest that a high level of redundancy ensures microtubule stability in the transmissive stages of Plasmodium, which is important for parasite transmission.\nExploring archaeogenetic studies of dental calculus to shed light on past human migrations in Oceania\nAuthors: Velsko, I. M.; Fagernas, Z.; Tromp, M.; Bedford, S.; Buckley, H.; Clark, G.; Dudgeon, J.; Flexner, J. L.; Leal-Tamarii, A.; Lewis, C.; Lisa, M.-S.; Nagele, K.; Ozga, A.; Rohrlach, A. B.; Posth, C.; Shing, R.; Spriggs, M.; Willie, E.; Valentin, F.; Warinner, C.\nScore: 10.8, Published: 2023-10-19 DOI: 10.1101/2023.10.18.563027\nThe Pacific islands have experienced multiple waves of human migrations, providing a case study for exploring the potential of using the microbiome to study human migration. We performed a metagenomic study of archaeological dental calculus from 103 ancient individuals, originating from 12 Pacific islands and spanning a time range of ~3000 years. Oral microbiome DNA preservation in calculus is far higher than that of human DNA in archaeological bone from the Pacific, and comparable to that seen in calculus from temperate regions. Variation in the microbial community composition was minimally driven by time period and geography within the Pacific, while comparison with samples from Europe, Africa, and Asia reveal the microbial communities of Pacific calculus samples to be distinctive. Phylogenies of individual bacterial species in Pacific calculus reflect geography. Archaeological dental calculus shows potential to yield information about past human migrations, complementing studies of the human genome.\nGlobal regulation via modulation of ribosome pausing by EttA\nAuthors: Ousalem, F.; Ngo, S.; Oiffer, T.; Omairi-Nasser, A.; Monlezun, L.; Boel, G.\nScore: 10.7, Published: 2023-10-17 DOI: 10.1101/2023.10.17.562674\nHaving multiple rounds of translation of the same mRNA is a key feature of the Central Dogma that creates substantial dynamic complexities along with opportunities for regulation related to ribosome pausing and stalling at specific sequences1-5. The molecular mechanisms controlling these critical processes and the principles guiding their evolution remain among the least understood facets of cellular physiology. We herein use a combination of genetic, genomic, physiological, and biochemical methods to show that regulation of ribosome pausing at specific amino acid sequences can produce [~]2-fold changes in protein expression levels that have a strong influence on cell growth and therefore evolutionary fitness. We demonstrate, both in vivo and in vitro, that the ABCF protein EttA6,7 directly controls the translation of mRNAs coding for a subset of enzymes in the tricarboxylic acid (TCA) cycle and its glyoxylate shunt, which dramatically modulates growth in some chemical environments. It also modulates expression of specific proteins involved in metabolically related physiological and stress-response pathways. These regulatory activities are mediated by EttA rescuing ribosomes paused at specific patterns of negatively charged residues within the first 30 amino acids of nascent proteins. The previously established dependency of EttAs activity on ADP:ATP ratio6,7 can modulate these effects based on cellular energy status. We thus establish a new global regulatory paradigm based on sequence-specific modulation of translational pausing.\n",
  "wordCount" : "2354",
  "inLanguage": "en",
  "datePublished": "2023-10-22T10:37:23Z",
  "dateModified": "2023-10-22T10:37:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta"><span>updated on October 22, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.14.562336">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.14.562336" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.14.562336">
        <p class="paperTitle">Escherichia coli plasmidome maps the game of clones</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.14.562336" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.14.562336" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Arredondo-Alonso, S.; Pontinen, A. K.; Gama, J.; Gladstone, R. A.; Harms, K.; Tonkin-Hill, G.; Thorpe, H. A.; Simonsen, G. S.; Samuelsen, O.; Johnsen, P. J.; Corander, J.</p>
        <p class="info">Score: 43.4, Published: 2023-10-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.14.562336' target='https://doi.org/10.1101/2023.10.14.562336'> 10.1101/2023.10.14.562336</a></p>
        <p class="abstract">Escherichia coli is a major human pathogen and the most widely studied microbe in history, but its extrachromosomal elements known as plasmids remain poorly delineated. Here we used long-read technology to high-resolution sequence the entire plasmidome and the corresponding host chromosomes from a longitudinal survey covering two decades and over 2,000 E. coli isolates. Separation of chromosomal and extrachromosomal DNA enabled us to reconstruct co-evolutionary trajectories of host lineages and their plasmids on a population-wide scale, demonstrating that plasmid evolution is markedly constrained contrary to the established dogma. We find that some plasmids have persisted in lineages for centuries, and provide a high-resolution map of recent vertical and horizontal evolutionary events in plasmids with key antibiotic resistance, competition and virulence determinants. We present genomic evidence of both chromosomal and plasmid-driven success strategies that represent convergent phenotypic evolution in distant lineages, and use in vitro experiments to verify the importance of bacteriocin-producing plasmids for clone success. Our study has general implications for understanding plasmid biology and bacterial evolutionary strategies, and informing development of future interventions against pathogens.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.15.562376">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.15.562376" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.15.562376">
        <p class="paperTitle">Long-term dynamics of the &#39;Serratia marcescens complex&#39; in the hospital-built environment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.15.562376" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.15.562376" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aracil-Gisbert, S.; Fernandez-de-Bobadilla, M. D.; Guerra-Pinto, N.; Serrano-Calleja, S.; Perez-Cobas, A. E.; Soriano, C.; De Pablo, R.; Lanza, V. F.; Perez-Viso, B.; Reuter, S.; Hasman, H.; Canton, R.; Baquero, F.; Coque, T. M.</p>
        <p class="info">Score: 33.3, Published: 2023-10-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.15.562376' target='https://doi.org/10.1101/2023.10.15.562376'> 10.1101/2023.10.15.562376</a></p>
        <p class="abstract">Serratia marcescens is an opportunistic pathogen historically associated with abrupt, unpredictable, and severe outbreaks in hospital intensive care units (ICUs) and, more recently, with the spread of acquired genes encoding carbapenem resistance. However, the population biology and ecology of S. marcescens in the hospital ecosystem is still poorly understood. Here, we combine epidemiological information of 1417 Serratia isolates collected from the sinks of a large ICU ward that underwent significant demographic and operational changes (2019-2020) and 99 non-redundant outbreak/non-outbreak isolates from the same hospital (2003-2019), with genomic data of 165 isolates. We first report hospital sinks as reservoirs of heterogeneous and coexistent populations of the S. marcescens complex (SMC). A novel SMC clade congruent with the recently described Serratia nevei species is predominant, exhibiting chromosomal AmpC {beta}-lactamase with an unusual basal expression in contrast to one of the major features of S. marcescens. Persistent Serratia sink strains are identical to those involved in clonal and polyclonal outbreaks of VIM-1 and OXA-48 producers since at least 2017. The &#34;source-sink&#34; dynamics of SMC populations able to acquire the highly conserved plasmids such as IncL carrying blaVIM-1 or blaOXA-48 offer novel insights that might improve interventions to control outbreaks and treat Serratia infections in high-risk hospital areas.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.16.562172">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.16.562172" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.16.562172">
        <p class="paperTitle">Transertion and cell geometry organize the Escherichia coli nucleoid during rapid growth</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.16.562172" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.16.562172" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Spahn, C.; Middlemiss, S.; Gomez-de-Mariscal, E.; Henriques, R.; Bode, H. B.; Holden, S.; Heilemann, M.</p>
        <p class="info">Score: 27.5, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.16.562172' target='https://doi.org/10.1101/2023.10.16.562172'> 10.1101/2023.10.16.562172</a></p>
        <p class="abstract">ABSTRACTUsing 3D single-molecule localization microscopy and live-cell imaging, we show that the Escherichia coli nucleoid adopts a condensed, membrane-associated configuration during rapid growth. To study the influence of different biosynthetic processes on nucleoid morphology and positioning, we recorded multi-colour super-resolution images during drug treatment. After developing analysis routines for confocal and super-resolution images, we captured highly resolved snapshots which revealed the complete loss of the membrane-bound state of the nucleoid within 10 minutes of halting transcription and translation. This indicates an active role of transertion (coupled transcription, translation and membrane insertion) in nucleoid organization. In contrast, cell wall synthesis inhibition only affects nucleoid organization during morphological changes. Further, we provide evidence that the E. coli nucleoid spatially correlates with MreB in unperturbed E. coli cells, while this correlation diminishes in cells with changed cell geometry or upon inhibition of protein biosynthesis. Replication inhibition experiments, as well as multi-drug treatments highlight the role of entropic effects and transcription in nucleoid condensation and positioning. In summary, we provide experimental evidence for transertion as a principal organiser of the bacterial nucleoid, and show that an altered metabolic state and antibiotic treatment both lead to major changes in the degree of transertion and overall spatial organization of the nucleoid. Our high-resolution characterization reveals dynamics of antibiotic action and provides tools to quantify bacterial chromosome organization. This does not provide valuable insights into the role of transertion, but can also be applied to study other cell- biological processes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.561914">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.561914" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.561914">
        <p class="paperTitle">Genetically refactored Agrobacterium-mediated transformation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.561914" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.561914" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Thompson, M.; Kirkpatrick, L. D.; Geiselman, G. M.; Waldburger, L. M.; Pearson, A. N.; Szarzanowicz, M.; Vuu, K. M.; Markel, K.; Hummel, N. F.; Suazo, D. D.; Tahmin, C.; Cui, R.; Liu, S.; Cevallos, J.; Pannu, H.; Liu, D.; Gin, J. W.; Chen, Y.; Petzold, C. J.; Gladden, J.; Keasling, J. D.; Chang, J. H.; Weisberg, A. J.; Shih, P. M.</p>
        <p class="info">Score: 37.3, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.561914' target='https://doi.org/10.1101/2023.10.13.561914'> 10.1101/2023.10.13.561914</a></p>
        <p class="abstract">Members of Agrobacterium are costly plant pathogens while also essential tools for plant transformation. Though Agrobacterium-mediated transformation (AMT) has been heavily studied, its polygenic nature and its complex transcriptional regulation make identifying the genetic basis of transformational efficiency difficult through traditional genetic and bioinformatic approaches. Here we use a bottom-up synthetic approach to systematically refactor the tumor-inducing plasmid, wherein the majority of AMT machine components are encoded, into a minimal set of genes capable of plant and fungal transformation that is both controllable and orthogonal to its environment. We demonstrate that engineered vectors can be transferred to new heterologous bacteria, enabling them to transform plants. Our reductionist approach demonstrates how bottom-up engineering can be used to dissect and elucidate the genetic underpinnings of complex biological traits, and may lead to the development of strains of bacteria more capable of transforming recalcitrant plant species of societal importance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562728">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562728" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562728">
        <p class="paperTitle">Phages Produce Persisters</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562728" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562728" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fernandez-Garcia, L.; Kirigo, J.; Huelgas-Mendez, D.; Tomas, M.; Garcia-Contreras, R.; Wood, T. K.</p>
        <p class="info">Score: 15.3, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562728' target='https://doi.org/10.1101/2023.10.17.562728'> 10.1101/2023.10.17.562728</a></p>
        <p class="abstract">Bacteria primarily encounter stress, and, arguably, their greatest threats are phages. It is often assumed that those bacteria that escape phage attack have mutated; however, another possibility is that a subpopulation forms the dormant persister state, in a manner similar to that demonstrated for bacterial cells undergoing nutritive, oxidative, and antibiotic stress. Persister cells do not undergo mutation and survive lethal conditions by ceasing growth transiently. Slower growth and dormancy play a key physiological role as they allow host phage defense systems more time to clear the phage infection. Here we investigated how bacteria survive lytic phage infection by isolating surviving cells from the plaques of T2, T4, and lambda (cI mutant) virulent phages. We found that bacteria in plaques can escape phage attack both by mutation (i.e., become resistant) and without mutation (i.e., become persistent). Specifically, whereas T4-resistant and lambda-resistant bacteria with over a 100,000-fold less sensitivity were isolated from plaques with obvious genetic mutations (e.g., causing mucoidy), cells were also found after T2 infection that undergo no significant mutation, retain wild-type phage sensitivity, and survive lethal doses of antibiotics. Corroborating this, adding T2 phage to persister cells resulted in 137,000-fold more survival compared to that of addition to exponentially-growing cells. Phage treatments with Klebsiella pneumonia and Pseudomonas aeruginosa also generated persister cells. Hence, along with resistant strains, bacteria also form persister cells during phage infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562707">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562707" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562707">
        <p class="paperTitle">KatG catalase deficiency confers bedaquiline hyper-susceptibility to isoniazid resistant Mycobacterium tuberculosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562707" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562707" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ofori-Anyinam, N.; Hamblin, M.; Coldren, M. L.; Li, B.; Mereddy, G.; Shaikh, M.; Shah, A.; Ranu, N.; Lu, S.; Blainey, P. C.; Ma, S.; Collins, J. J.; Yang, J. H.</p>
        <p class="info">Score: 13.7, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562707' target='https://doi.org/10.1101/2023.10.17.562707'> 10.1101/2023.10.17.562707</a></p>
        <p class="abstract">Multidrug-resistant tuberculosis (MDR-TB) is a growing source of global mortality and threatens global control of tuberculosis (TB) disease. The diarylquinoline bedaquiline (BDQ) recently emerged as a highly efficacious drug against MDR-TB, defined as resistance to the first-line drugs isoniazid (INH) and rifampin. INH resistance is primarily caused by loss-of-function mutations in the catalase KatG, but mechanisms underlying BDQs efficacy against MDR-TB remain unknown. Here we employ a systems biology approach to investigate BDQ hyper-susceptibility in INH-resistant Mycobacterium tuberculosis. We found hyper-susceptibility to BDQ in INH-resistant cells is due to several physiological changes induced by KatG deficiency, including increased susceptibility to reactive oxygen species and DNA damage, remodeling of transcriptional programs, and metabolic repression of folate biosynthesis. We demonstrate BDQ hyper-susceptibility is common in INH-resistant clinical isolates. Collectively, these results highlight how altered bacterial physiology can impact drug efficacy in drug-resistant bacteria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.11.560955">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.11.560955" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.11.560955">
        <p class="paperTitle">Integration of 168,000 samples reveals global patterns of the human gut microbiome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.11.560955" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.11.560955" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Abdill, R. J.; Graham, S. P.; Rubinetti, V.; Albert, F. W.; Greene, C. S.; Davis, S.; Blekhman, R.</p>
        <p class="info">Score: 88.1, Published: 2023-10-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.11.560955' target='https://doi.org/10.1101/2023.10.11.560955'> 10.1101/2023.10.11.560955</a></p>
        <p class="abstract">Understanding the factors that shape variation in the human microbiome is a major goal of research in biology. While other genomics fields have used large, pre-compiled compendia to extract systematic insights requiring otherwise impractical sample sizes, there has been no comparable resource for the 16S rRNA sequencing data commonly used to quantify microbiome composition. To help close this gap, we have assembled a set of 168,484 publicly available human gut microbiome samples, processed with a single pipeline and combined into the largest unified microbiome dataset to date. We use this resource, which is freely available at microbiomap.org, to shed light on global variation in the human gut microbiome. We find that Firmicutes, particularly Bacilli and Clostridia, are almost universally present in the human gut. At the same time, the relative abundance of the 65 most common microbial genera differ between at least two world regions. We also show that gut microbiomes in undersampled world regions, such as Central and Southern Asia, differ significantly from the more thoroughly characterized microbiomes of Europe and Northern America. Moreover, humans in these overlooked regions likely harbor hundreds of taxa that have not yet been discovered due to this undersampling, highlighting the need for diversity in microbiome studies. We anticipate that this new compendium can serve the community and enable advanced applied and methodological research.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.19.562943">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.19.562943" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.19.562943">
        <p class="paperTitle">Microtubule inner proteins of Plasmodium are essential for transmission of malaria parasites</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.19.562943" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.19.562943" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hentzschel, F.; Binder, A. M.; Dorner, L. P.; Herzel, L.; Nuglish, F.; Sema, M.; Aguirre-Botero, M. C.; Cyrklaff, M.; Funaya, C.; Frischknecht, F.</p>
        <p class="info">Score: 11.2, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.19.562943' target='https://doi.org/10.1101/2023.10.19.562943'> 10.1101/2023.10.19.562943</a></p>
        <p class="abstract">Microtubule inner proteins, MIPs, are microtubule associated proteins that bind to tubulin from the luminal side. MIPs can be found in axonemes to stabilize flagellar beat or within cytoplasmic microtubules. Plasmodium spp. are the causative agents of malaria that feature different forms across a complex life cycle with both unique and divergent microtubule-based arrays. Here we investigate the role of four MIPs in a rodent malaria parasite for their role in transmission to and from the mosquito. We show by single and double gene deletions that SPM1 and TrxL1, MIPs associated with the subpellicular microtubules are dispensable for transmission from the vertebrate host to the mosquito and back. In contrast, FAP20 and FAP52, MIPs associated with the axonemes of gametes, are essential for transmission to mosquitoes but only if both genes are deleted. In the absence of both, FAP20 and FAP52 the B-tubule of the axoneme partly detaches from the A-tubule resulting in the deficiency of axonemal beating and hence gamete formation and egress. Our data suggest that a high level of redundancy ensures microtubule stability in the transmissive stages of Plasmodium, which is important for parasite transmission.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.18.563027">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.18.563027" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.18.563027">
        <p class="paperTitle">Exploring archaeogenetic studies of dental calculus to shed light on past human migrations in Oceania</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.18.563027" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.18.563027" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Velsko, I. M.; Fagernas, Z.; Tromp, M.; Bedford, S.; Buckley, H.; Clark, G.; Dudgeon, J.; Flexner, J. L.; Leal-Tamarii, A.; Lewis, C.; Lisa, M.-S.; Nagele, K.; Ozga, A.; Rohrlach, A. B.; Posth, C.; Shing, R.; Spriggs, M.; Willie, E.; Valentin, F.; Warinner, C.</p>
        <p class="info">Score: 10.8, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.18.563027' target='https://doi.org/10.1101/2023.10.18.563027'> 10.1101/2023.10.18.563027</a></p>
        <p class="abstract">The Pacific islands have experienced multiple waves of human migrations, providing a case study for exploring the potential of using the microbiome to study human migration. We performed a metagenomic study of archaeological dental calculus from 103 ancient individuals, originating from 12 Pacific islands and spanning a time range of ~3000 years. Oral microbiome DNA preservation in calculus is far higher than that of human DNA in archaeological bone from the Pacific, and comparable to that seen in calculus from temperate regions. Variation in the microbial community composition was minimally driven by time period and geography within the Pacific, while comparison with samples from Europe, Africa, and Asia reveal the microbial communities of Pacific calculus samples to be distinctive. Phylogenies of individual bacterial species in Pacific calculus reflect geography. Archaeological dental calculus shows potential to yield information about past human migrations, complementing studies of the human genome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562674">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562674" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562674">
        <p class="paperTitle">Global regulation via modulation of ribosome pausing by EttA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562674" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562674" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ousalem, F.; Ngo, S.; Oiffer, T.; Omairi-Nasser, A.; Monlezun, L.; Boel, G.</p>
        <p class="info">Score: 10.7, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562674' target='https://doi.org/10.1101/2023.10.17.562674'> 10.1101/2023.10.17.562674</a></p>
        <p class="abstract">Having multiple rounds of translation of the same mRNA is a key feature of the Central Dogma that creates substantial dynamic complexities along with opportunities for regulation related to ribosome pausing and stalling at specific sequences1-5. The molecular mechanisms controlling these critical processes and the principles guiding their evolution remain among the least understood facets of cellular physiology. We herein use a combination of genetic, genomic, physiological, and biochemical methods to show that regulation of ribosome pausing at specific amino acid sequences can produce [~]2-fold changes in protein expression levels that have a strong influence on cell growth and therefore evolutionary fitness. We demonstrate, both in vivo and in vitro, that the ABCF protein EttA6,7 directly controls the translation of mRNAs coding for a subset of enzymes in the tricarboxylic acid (TCA) cycle and its glyoxylate shunt, which dramatically modulates growth in some chemical environments. It also modulates expression of specific proteins involved in metabolically related physiological and stress-response pathways. These regulatory activities are mediated by EttA rescuing ribosomes paused at specific patterns of negatively charged residues within the first 30 amino acids of nascent proteins. The previously established dependency of EttAs activity on ADP:ATP ratio6,7 can modulate these effects based on cellular energy status. We thus establish a new global regulatory paradigm based on sequence-specific modulation of translational pausing.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
