<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Virological characteristics of the SARS-CoV-2 JN.1 variant
Authors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Zahradnik, J.; Ito, J.; Sato, K.
Score: 613.3, Published: 2023-12-09 DOI: 10.1101/2023.12.08.570782
The SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Virological characteristics of the SARS-CoV-2 JN.1 variant
Authors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Zahradnik, J.; Ito, J.; Sato, K.
Score: 613.3, Published: 2023-12-09 DOI: 10.1101/2023.12.08.570782
The SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-13T10:39:57+00:00" />
<meta property="article:modified_time" content="2023-12-13T10:39:57+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Virological characteristics of the SARS-CoV-2 JN.1 variant
Authors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Zahradnik, J.; Ito, J.; Sato, K.
Score: 613.3, Published: 2023-12-09 DOI: 10.1101/2023.12.08.570782
The SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Virological characteristics of the SARS-CoV-2 JN.1 variant\nAuthors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Zahradnik, J.; Ito, J.; Sato, K.\nScore: 613.3, Published: 2023-12-09 DOI: 10.1101/2023.12.08.570782\nThe SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG.",
  "keywords": [
    
  ],
  "articleBody": " Virological characteristics of the SARS-CoV-2 JN.1 variant\nAuthors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Zahradnik, J.; Ito, J.; Sato, K.\nScore: 613.3, Published: 2023-12-09 DOI: 10.1101/2023.12.08.570782\nThe SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG.5.1 and HK.3. Comparing to XBB and BA.2, BA.2.86 carries more than 30 mutations in the spike (S) protein, indicating a high potential for immune evasion. BA.2.86 has evolved and its descendant, JN.1 (BA.2.86.1.1), emerged in late 2023. JN.1 harbors S:L455S and three mutations in non-S proteins. S:L455S is a hallmark mutation of JN.1: we have recently shown that HK.3 and other \"FLip\" variants carry S:L455F, which contributes to increased transmissibility and immune escape ability compared to the parental EG.5.1 variant. Here, we investigated the virological properties of JN.1.\nSARS-CoV-2 infection is associated with intestinal permeability, systemic inflammation, and microbial dysbiosis in hospitalized COVID-19 patients\nAuthors: Basting, C. M.; Langat, R.; Broedlow, C. A.; Guerrero, C.; Bold, T. D.; Bailey, M.; Velez, A.; Schroeder, T.; Short-Miller, J.; Cromarty, R.; Mayer, Z.; Southern, P. J.; Schacker, T. W.; Safo, S. E.; Bramante, C. T.; Tignanelli, C. J.; Schifanella, L.; Klatt, N. R.\nScore: 237.1, Published: 2023-12-08 DOI: 10.1101/2023.12.07.570670\nCoronavirus disease 2019 (COVID-19) and associated severity has been linked to uncontrolled inflammation and may be associated with changes in the microbiome of mucosal sites including the gastrointestinal tract and oral cavity. These sites play an important role in host-microbe homeostasis and disruption of epithelial barrier integrity during COVID-19 may potentially lead to exacerbated inflammation and immune dysfunction. Outcomes in COVID-19 are highly disparate, ranging from asymptomatic to fatal, and the impact of microbial dysbiosis on disease severity is unclear. Here, we obtained plasma, rectal swabs, oropharyngeal swabs, and nasal swabs from 86 patients hospitalized with COVID-19 and 12 healthy volunteers. We performed 16S rRNA sequencing to characterize the microbial communities in the mucosal swabs and measured circulating cytokines, markers of gut barrier integrity, and fatty acids in the plasma samples. We compared these plasma concentrations and microbiomes between healthy volunteers and the COVID-19 patients who had survived or unfortunately died by the end of study enrollment, and between severe disease and healthy controls, as well as performed a correlation analysis between plasma variables and bacterial abundances. The rectal swabs of COVID-19 patients had reduced abundances of several commensal bacteria including Faecalibacterium prausnitsii, and an increased abundance of the opportunistic pathogens Eggerthella lenta and Hungatella hathewayi. Furthermore, the oral pathogen Scardovia wiggsiae was more abundant in the oropharyngeal swabs of COVID-19 patients who died. The abundance of both H. hathewayi and S. wiggsiae correlated with circulating inflammatory markers including IL-6, highlighting the possible role of the microbiome in COVID-19 severity, and providing potential therapeutic targets for managing COVID-19.\nModel cyanobacterial consortia reveal a consistent core microbiome independent of inoculation source or cyanobacterial host species\nAuthors: Kust, A.; Zorz, J.; Paniker, C. C.; Bouma-Gregson, K.; Krishnappa, N.; Banfield, J. F.; Diamond, S.\nScore: 43.1, Published: 2023-12-10 DOI: 10.1101/2023.12.09.570939\nCyanobacteria are central to biogeochemical cycling, climate change, and eutrophication. While they readily develop associations with environmental microorganisms, the question of whether they consistently recruit specific microbiomes remains unresolved. Here, we established in vitro cyanobacterial consortia by inoculating five different cyanobacterial strains with microbiomes from three freshwater environments to determine if similar non-cyanobacterial organisms were recruited into stable cyanobacteria-based communities. Using amplicon and genome-resolved metagenomic methods we determined that stable cyanobacterial communities largely share the same taxa, regardless of the host cyanobacterial species or inoculation source. Specifically, we identified a 25 species core microbiome, which was significantly enriched in functions for micronutrient biosynthesis, metabolite transport/exchange, and anoxygenic photosynthesis. Furthermore, core species were enriched in putative mobile genetic elements, including two large (\u003e 100 kb) circularized and curated plasmids, encoding functions that may support cyanobacteria-heterotroph symbioses. The consistency observed across these systems indicate that cyanobacterial consortia maintained under similar conditions converge to a predictable structure, and support a core microbiome with taxonomic and functional similarity to core microbiomes of other phototroph-heterotroph symbiotic assemblages.\nDetection and spread of high pathogenicity avian influenza virus H5N1 in the Antarctic Region\nAuthors: Bennison, A.; Byrne, A. M.; Reid, S. M.; Lynton-Jenkins, J. G.; Mollett, B.; De Sliva, D.; Peers-Dent, J.; Finlayson, K. E.; Hall, R.; Blockley, F.; Blyth, M.; Falchieri, M.; Fowler, Z.; Fitzcharles, E.; Brown, I. H.; James, J.; Banyard, A. C.\nScore: 77.0, Published: 2023-11-24 DOI: 10.1101/2023.11.23.568045\nThe Antarctic is the only major geographical region in which high pathogenicity avian influenza virus (HPAIV) has never previously been detected. The current panzootic of H5N1 HPAIV has decimated wild bird populations across Europe, North America and South America. Here we report on the emergence of clade 2.3.4.4b H5N1 HPAIV in the Antarctic and sub-Antarctic regions of South Georgia and the Falkland Islands respectively. We initially detected H5N1 HPAIV in samples collected from brown skuas at Bird Island, South Georgia on 8th October 2023. Since this detection, increased mortalities were observed in brown skuas, kelp gulls, elephant seals and fur seal at multiple sites across South Georgia. We confirmed H5N1 HPAIV in multiple brown skuas and kelp gulls across four different sampling locations in South Georgia. Simultaneously, we also confirmed H5N1 HPAIV in a southern fulmar in the Falkland Islands. Genetic assessment of the virus indicates spread from South America, likely through movement of migratory birds. Here we describe the emergence, species impact and genetic composition of the virus and propose both introductory routes and potential long-term impact on avian and mammalian species across the Antarctic region.\nGiant genes are rare but implicated in cell wall degradation by predatory bacteria\nAuthors: West-Roberts, J. A.; Valentin Alvarado, L. E.; Mullen, S.; Sachdeva, R.; Smith, J.; Hug, L. A.; Gregoire, D.; Liu, W.; Lin, T.-Y.; Husain, G.; Amano, Y.; Ly, L.; Banfield, J.\nScore: 66.1, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568195\nAcross the tree of life, gene lengths vary, but most are no more than a few thousand base pairs in length. The largest protein often reported is the [~]40,000 aa eukaryotic Titin. Even larger proteins may occur in the rapidly expanding set of metagenome-derived sequences, but their existence may be obscured by assembly fragmentation. Here, we leverage genome curation to complete metagenome-derived sequences that encode predicted proteins of up to 85,804 aa. Overall, the findings illuminate a huge knowledge gap related to giant proteins. Although predicted proteins of \u003e30,000 aa occur in bacterial phyla such as Firmicutes and Actinobacteria, they are most common in ca. Omnitrophota, ultra small bacteria that adopt predatory lifestyles. All full length giant genes encode numerous transmembrane regions and most encode divergent secA DEAD helicase domains. In silico structural prediction of protein subregions was required to identify domains in unannotated protein segments, and revealed putative domains implicated in attachment and carbohydrate degradation. Many giant genes in new complete and near-complete Omnitrophota genomes occur in close proximity to genes homologous to type II secretion systems as well as carbohydrate import systems. This, in combination with the domain content, suggests that many bacterial giant proteins enable prey adhesion and cell wall digestion during bacterial predation.\nDistinct evolution of SARS-CoV-2 Omicron XBB and BA.2.86 lineages combining increased fitness and antibody evasion\nAuthors: Planas, D.; Staropoli, I.; Michel, V.; Lemoine, F.; Donati, F.; Prot, M.; Porrot, F.; Benhassine, F. G.; Jeyarajah, B.; Brisebarre, A.; Dehan, O.; Avon, L.; Bolland, W.; Hubert, M.; Buchreiser, J.; Vanhoucke, T.; Rosenbaum, P.; Veyer, D.; Pere, H.; Lina, B.; Assant, S. T.; Hocqueloux, L.; Prazuck, T.; Loriere, E. S.; Schwartz, O.\nScore: 76.0, Published: 2023-12-06 DOI: 10.1101/2023.11.20.567873\nThe unceasing circulation of SARS-CoV-2 leads to the continuous emergence of novel viral sublineages. Here, we isolated and characterized XBB.1, XBB.1.5, XBB.1.9.1, XBB.1.16.1, EG.5.1.1, EG.5.1.3, XBF, BA.2.86.1 and JN.1 variants, representing \u003e80% of circulating variants in November 2023. The XBB subvariants carry few but recurrent mutations in the spike, whereas BA.2.86.1 and JN.1 harbor \u003e30 additional changes. These variants replicated in IGROV-1 but no longer in Vero E6 and were not markedly fusogenic. They potently infected nasal epithelial cells, with EG.5.1.3 exhibiting the highest fitness. Antivirals remained active. Neutralizing antibody (NAb) responses from vaccinees and BA.1/BA.2-infected individuals were markedly lower compared to BA.1, without major differences between variants. An XBB breakthrough infection enhanced NAb responses against both XBB and BA.2.86 variants. JN.1 displayed lower affinity to ACE2 and higher immune evasion properties compared to BA.2.86.1. Thus, while distinct, the evolutionary trajectory of these variants combines increased fitness and antibody evasion.\nUmbrella toxin particles produced by Streptomyces block mycelial growth of competing species\nAuthors: Zhao, Q.; Bertolli, S.; Park, Y.-J.; Tan, Y.; Cutler, K. J.; Srinivas, P.; Asfahl, K. L.; Fonseca Garcia, C.; Gallagher, L. A.; Li, Y.; Wang, Y.; Coleman-Derr, D.; DiMaio, F.; Zhang, D.; Peterson, S. B.; Veesler, D.; Mougous, J. D.\nScore: 24.9, Published: 2023-12-10 DOI: 10.1101/2023.12.09.570830\nThe Streptomyces are a genus of ubiquitous soil bacteria from which the majority of clinically utilized antibiotics derive. The production of these antibacterial molecules reflects the relentless competition Streptomyces engage in with other bacteria, including other Streptomyces species. Here we show that in addition to small molecule antibiotics, Streptomyces produce and secrete antibacterial protein complexes that feature a large, degenerate repeat-containing polymorphic toxin protein. A cryo-EM structure of these particles reveals an extended stalk topped by a ringed crown comprising the toxin repeats scaffolding five lectin-tipped spokes, leading to our naming them umbrella particles. S. coelicolor encodes three umbrella particles with distinct toxin and lectin composition, and supernatant containing these toxins specifically and potently inhibits the growth of select Streptomyces species from among a diverse collection of bacteria screened. For one target, S. griseus, we find inhibition relies on a single toxin and that intoxication manifests as rapid cessation of vegetative mycelial growth. Our data show that Streptomyces umbrella particles mediate competition between vegetative mycelia of related species, a function distinct from small molecule antibiotics, which are produced at the onset of reproductive growth and act broadly. Sequence analyses suggest this role of umbrella particles extends beyond Streptomyces, as we find umbrella loci in nearly one-thousand species across Actinobacteria.\nElevated mutation rates in the multi-azole resistant Aspergillus fumigatus cladedrives rapid evolution of antifungal resistance\nAuthors: Bottery, M. J.; van Rhijn, N. J.; Chown, H.; Rhodes, J. L.; Celia-Sanchez, B. N.; Brewer, M. T.; Momany, M.; Fisher, M. C.; Knight, C. G.; Bromley, M. J.\nScore: 21.2, Published: 2023-12-05 DOI: 10.1101/2023.12.05.570068\nThe evolution of antifungal resistance is an emerging global threat. Particularly concerning is the widespread occurrence of azole resistance within Aspergillus fumigatus, a globally ubiquitous environmental mould that causes over 1 million life-threatening invasive infections in humans each year. It is increasingly evident that the environmental use of azoles has led to selective sweeps across multiple genomic loci resulting in the rapid expansion of a genetically distinct cluster of genotypes (clade A) that results in resistance to clinically deployed azoles. Isolates within this cluster are more likely to be cross resistant to agricultural antifungals with unrelated modes of action suggesting they may be adapting rapidly to antifungal challenge. Here we show that this cluster is not only multi-azole resistant but has increased propensity to develop resistance to new antifungals because of variants in the DNA mismatch repair system. A variant in msh6 is found almost exclusively within clade A, occurs in 88% of multi-azole resistant isolates harbouring the canonical cyp51A azole resistance allelic variant TR34/L98H, and is globally distributed. Naturally occurring isolates with this msh6 variant display a 4 to 9-times higher rate of mutation, leading to an increased propensity to evolve resistance to current and next generation antifungals. We argue that pervasive environmental use of fungicides creates selective arenas whereby genotypes of A. fumigatus with increased adaptive capability thrive in the face of strong directional selection, leading to the genesis and amplification of antifungal resistance. These results help explain the pronounced clustering of multiple independent resistance mechanisms within the mutable clade A. Our findings further suggest that resistance to next generation antifungals is more likely to emerge within organisms that are already multi-azole resistant, posing a major problem due to the prospect of dual use of novel antifungals in clinical and agricultural settings.\nModulation of Salmonella virulence by a novel SPI-2 injectisome effector that interacts with the dystrophin-associated protein complex.\nAuthors: Yu, X.-J.; Xie, H. X.; Li, Y.; Liu, M.; Hou, R.; Predeus, A.; Perez-Sepulveda, B. M.; Hinton, J.; Holden, D. W.; Thurston, T. L. M.\nScore: 19.9, Published: 2023-12-08 DOI: 10.1101/2023.12.07.570645\nThe injectisome encoded by Salmonella pathogenicity island 2 (SPI-2) had been thought to translocate 28 effectors. Here, we used a proteomic approach to characterise the secretome of a clinical strain of invasive non-typhoidal Salmonella enterica serovar Enteritidis, that had been mutated to cause hyper-secretion of the SPI-2 injectisome effectors. Along with many known effectors, we discovered the novel SseM protein. sseM is widely distributed between the five subspecies of Salmonella enterica, is found in many clinically-relevant serovars, and is co-transcribed with pipB2, a SPI-2 effector gene. Translocation of SseM required a functional SPI-2 injectisome. Following expression in human cells, SseM interacted with five components of the dystrophin-associated protein complex (DAPC), namely {beta}-2-syntrophin, utrophin/ dystrophin, -catulin, -dystrobrevin and {beta}-dystrobrevin. The interaction between SseM and {beta}-2-syntrophin and -dystrobrevin was verified in S. Typhimurium-infected cells and relied on the PDZ domain of {beta}-2-syntrophin and a sequence corresponding to a PDZ-binding motif (PBM) in SseM. A {Delta}sseM mutant strain had a small competitive advantage over the wild-type strain in the S. Typhimurium/mouse model of systemic disease. This phenotype was complemented by a plasmid expressing wild type SseM from S. Typhimurium or S. Enteritidis and was dependent on the PBM of SseM. Therefore, a PBM within a Salmonella effector mediates interactions with the DAPC and modulates systemic growth of bacteria in mice. ImportanceIn Salmonella enterica, the injectisome machinery encoded by Salmonella pathogenicity island 2 (SPI-2) is conserved among the five subspecies and delivers proteins (effectors) into host cells that are required for Salmonella virulence. The identification and functional characterisation of SPI-2 injectisome effectors advances our understanding of the interplay between Salmonella and its host(s). Using an optimised method for preparing secreted proteins and a clinical isolate of the invasive non-typhoidal (iNTS) Salmonella enterica serovar Enteritidis strain D24359, we identified 22 known SPI-2 injectisome effectors and one new effector - SseM. SseM modulates bacterial growth during murine infection and has a sequence corresponding to a PDZ-binding motif that is essential for interaction with the PDZ-containing host protein {beta}-2-syntrophin and other components of the dystrophin-associated protein complex (DAPC). To our knowledge, SseM is unique among Salmonella effectors in containing a functional PDZ-binding motif and is the first bacterial protein to target the DAPC.\nSelective lipid recruitment by an archaeal DPANN symbiont from its host\nAuthors: Ding, S.; Hamm, J. N.; Bale, N. J.; Sinninghe Damste, J. S.; Spang, A.\nScore: 17.3, Published: 2023-12-08 DOI: 10.1101/2023.12.06.570485\nThe symbiont Ca. Nanohaloarchaeum antarcticus is obligately dependent on its host Halorubrum lacusprofundi for lipids and other metabolites due to its lack of certain biosynthetic genes. However, it remains unclear which specific lipids or metabolites are acquired from its host, and how the host responds to infection. Here, we explored the lipidome dynamics of the Ca. Nha. antarcticus - Hrr. lacusprofundi symbiotic relationship during co-cultivation. By using a comprehensive untargeted lipidomic methodology, our study reveals that Ca. Nha. antarcticus selectively recruits 110 lipid species from its host, i.e. nearly two-thirds of the total number of host lipids. Lipid profiles of co-cultures displayed shifts in abundances of bacterioruberins and menaquinones and changes in degree of bilayer-forming glycerolipid unsaturation. This likely results in increased membrane fluidity and improved resistance to membrane disruptions, consistent with compensation for higher metabolic load and mechanical stress on host membranes when in contact with Ca. Nha. antarcticus cells. Notably, our findings differ from previous observations of other DPANN symbiont-host systems, where no differences in lipidome composition were reported. Altogether, our work emphasizes the strength of employing untargeted lipidomics approaches to provide details into the dynamics underlying a DPANN symbiont-host system.\n",
  "wordCount" : "2681",
  "inLanguage": "en",
  "datePublished": "2023-12-13T10:39:57Z",
  "dateModified": "2023-12-13T10:39:57Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 13, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.570782">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.570782" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.570782">
        <p class="paperTitle">Virological characteristics of the SARS-CoV-2 JN.1 variant</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.570782" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.570782" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium,  ; Zahradnik, J.; Ito, J.; Sato, K.</p>
        <p class="info">Score: 613.3, Published: 2023-12-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.570782' target='https://doi.org/10.1101/2023.12.08.570782'> 10.1101/2023.12.08.570782</a></p>
        <p class="abstract">The SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG.5.1 and HK.3. Comparing to XBB and BA.2, BA.2.86 carries more than 30 mutations in the spike (S) protein, indicating a high potential for immune evasion. BA.2.86 has evolved and its descendant, JN.1 (BA.2.86.1.1), emerged in late 2023. JN.1 harbors S:L455S and three mutations in non-S proteins. S:L455S is a hallmark mutation of JN.1: we have recently shown that HK.3 and other &#34;FLip&#34; variants carry S:L455F, which contributes to increased transmissibility and immune escape ability compared to the parental EG.5.1 variant. Here, we investigated the virological properties of JN.1.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570670">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570670" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570670">
        <p class="paperTitle">SARS-CoV-2 infection is associated with intestinal permeability, systemic inflammation, and microbial dysbiosis in hospitalized COVID-19 patients</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570670" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570670" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Basting, C. M.; Langat, R.; Broedlow, C. A.; Guerrero, C.; Bold, T. D.; Bailey, M.; Velez, A.; Schroeder, T.; Short-Miller, J.; Cromarty, R.; Mayer, Z.; Southern, P. J.; Schacker, T. W.; Safo, S. E.; Bramante, C. T.; Tignanelli, C. J.; Schifanella, L.; Klatt, N. R.</p>
        <p class="info">Score: 237.1, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570670' target='https://doi.org/10.1101/2023.12.07.570670'> 10.1101/2023.12.07.570670</a></p>
        <p class="abstract">Coronavirus disease 2019 (COVID-19) and associated severity has been linked to uncontrolled inflammation and may be associated with changes in the microbiome of mucosal sites including the gastrointestinal tract and oral cavity. These sites play an important role in host-microbe homeostasis and disruption of epithelial barrier integrity during COVID-19 may potentially lead to exacerbated inflammation and immune dysfunction. Outcomes in COVID-19 are highly disparate, ranging from asymptomatic to fatal, and the impact of microbial dysbiosis on disease severity is unclear. Here, we obtained plasma, rectal swabs, oropharyngeal swabs, and nasal swabs from 86 patients hospitalized with COVID-19 and 12 healthy volunteers. We performed 16S rRNA sequencing to characterize the microbial communities in the mucosal swabs and measured circulating cytokines, markers of gut barrier integrity, and fatty acids in the plasma samples. We compared these plasma concentrations and microbiomes between healthy volunteers and the COVID-19 patients who had survived or unfortunately died by the end of study enrollment, and between severe disease and healthy controls, as well as performed a correlation analysis between plasma variables and bacterial abundances. The rectal swabs of COVID-19 patients had reduced abundances of several commensal bacteria including Faecalibacterium prausnitsii, and an increased abundance of the opportunistic pathogens Eggerthella lenta and Hungatella hathewayi. Furthermore, the oral pathogen Scardovia wiggsiae was more abundant in the oropharyngeal swabs of COVID-19 patients who died. The abundance of both H. hathewayi and S. wiggsiae correlated with circulating inflammatory markers including IL-6, highlighting the possible role of the microbiome in COVID-19 severity, and providing potential therapeutic targets for managing COVID-19.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.09.570939">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.09.570939" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.09.570939">
        <p class="paperTitle">Model cyanobacterial consortia reveal a consistent core microbiome independent of inoculation source or cyanobacterial host species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.09.570939" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.09.570939" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kust, A.; Zorz, J.; Paniker, C. C.; Bouma-Gregson, K.; Krishnappa, N.; Banfield, J. F.; Diamond, S.</p>
        <p class="info">Score: 43.1, Published: 2023-12-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.09.570939' target='https://doi.org/10.1101/2023.12.09.570939'> 10.1101/2023.12.09.570939</a></p>
        <p class="abstract">Cyanobacteria are central to biogeochemical cycling, climate change, and eutrophication. While they readily develop associations with environmental microorganisms, the question of whether they consistently recruit specific microbiomes remains unresolved. Here, we established in vitro cyanobacterial consortia by inoculating five different cyanobacterial strains with microbiomes from three freshwater environments to determine if similar non-cyanobacterial organisms were recruited into stable cyanobacteria-based communities. Using amplicon and genome-resolved metagenomic methods we determined that stable cyanobacterial communities largely share the same taxa, regardless of the host cyanobacterial species or inoculation source. Specifically, we identified a 25 species core microbiome, which was significantly enriched in functions for micronutrient biosynthesis, metabolite transport/exchange, and anoxygenic photosynthesis. Furthermore, core species were enriched in putative mobile genetic elements, including two large (&gt; 100 kb) circularized and curated plasmids, encoding functions that may support cyanobacteria-heterotroph symbioses. The consistency observed across these systems indicate that cyanobacterial consortia maintained under similar conditions converge to a predictable structure, and support a core microbiome with taxonomic and functional similarity to core microbiomes of other phototroph-heterotroph symbiotic assemblages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.568045">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.568045" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.568045">
        <p class="paperTitle">Detection and spread of high pathogenicity avian influenza virus H5N1 in the Antarctic Region</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.568045" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.568045" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bennison, A.; Byrne, A. M.; Reid, S. M.; Lynton-Jenkins, J. G.; Mollett, B.; De Sliva, D.; Peers-Dent, J.; Finlayson, K. E.; Hall, R.; Blockley, F.; Blyth, M.; Falchieri, M.; Fowler, Z.; Fitzcharles, E.; Brown, I. H.; James, J.; Banyard, A. C.</p>
        <p class="info">Score: 77.0, Published: 2023-11-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.568045' target='https://doi.org/10.1101/2023.11.23.568045'> 10.1101/2023.11.23.568045</a></p>
        <p class="abstract">The Antarctic is the only major geographical region in which high pathogenicity avian influenza virus (HPAIV) has never previously been detected. The current panzootic of H5N1 HPAIV has decimated wild bird populations across Europe, North America and South America. Here we report on the emergence of clade 2.3.4.4b H5N1 HPAIV in the Antarctic and sub-Antarctic regions of South Georgia and the Falkland Islands respectively. We initially detected H5N1 HPAIV in samples collected from brown skuas at Bird Island, South Georgia on 8th October 2023. Since this detection, increased mortalities were observed in brown skuas, kelp gulls, elephant seals and fur seal at multiple sites across South Georgia. We confirmed H5N1 HPAIV in multiple brown skuas and kelp gulls across four different sampling locations in South Georgia. Simultaneously, we also confirmed H5N1 HPAIV in a southern fulmar in the Falkland Islands. Genetic assessment of the virus indicates spread from South America, likely through movement of migratory birds. Here we describe the emergence, species impact and genetic composition of the virus and propose both introductory routes and potential long-term impact on avian and mammalian species across the Antarctic region.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568195">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568195" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568195">
        <p class="paperTitle">Giant genes are rare but implicated in cell wall degradation by predatory bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568195" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568195" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: West-Roberts, J. A.; Valentin Alvarado, L. E.; Mullen, S.; Sachdeva, R.; Smith, J.; Hug, L. A.; Gregoire, D.; Liu, W.; Lin, T.-Y.; Husain, G.; Amano, Y.; Ly, L.; Banfield, J.</p>
        <p class="info">Score: 66.1, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568195' target='https://doi.org/10.1101/2023.11.21.568195'> 10.1101/2023.11.21.568195</a></p>
        <p class="abstract">Across the tree of life, gene lengths vary, but most are no more than a few thousand base pairs in length. The largest protein often reported is the [~]40,000 aa eukaryotic Titin. Even larger proteins may occur in the rapidly expanding set of metagenome-derived sequences, but their existence may be obscured by assembly fragmentation. Here, we leverage genome curation to complete metagenome-derived sequences that encode predicted proteins of up to 85,804 aa. Overall, the findings illuminate a huge knowledge gap related to giant proteins. Although predicted proteins of &gt;30,000 aa occur in bacterial phyla such as Firmicutes and Actinobacteria, they are most common in ca. Omnitrophota, ultra small bacteria that adopt predatory lifestyles. All full length giant genes encode numerous transmembrane regions and most encode divergent secA DEAD helicase domains. In silico structural prediction of protein subregions was required to identify domains in unannotated protein segments, and revealed putative domains implicated in attachment and carbohydrate degradation. Many giant genes in new complete and near-complete Omnitrophota genomes occur in close proximity to genes homologous to type II secretion systems as well as carbohydrate import systems. This, in combination with the domain content, suggests that many bacterial giant proteins enable prey adhesion and cell wall digestion during bacterial predation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567873">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567873" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567873">
        <p class="paperTitle">Distinct evolution of SARS-CoV-2 Omicron XBB and BA.2.86 lineages combining increased fitness and antibody evasion</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567873" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567873" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Planas, D.; Staropoli, I.; Michel, V.; Lemoine, F.; Donati, F.; Prot, M.; Porrot, F.; Benhassine, F. G.; Jeyarajah, B.; Brisebarre, A.; Dehan, O.; Avon, L.; Bolland, W.; Hubert, M.; Buchreiser, J.; Vanhoucke, T.; Rosenbaum, P.; Veyer, D.; Pere, H.; Lina, B.; Assant, S. T.; Hocqueloux, L.; Prazuck, T.; Loriere, E. S.; Schwartz, O.</p>
        <p class="info">Score: 76.0, Published: 2023-12-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567873' target='https://doi.org/10.1101/2023.11.20.567873'> 10.1101/2023.11.20.567873</a></p>
        <p class="abstract">The unceasing circulation of SARS-CoV-2 leads to the continuous emergence of novel viral sublineages. Here, we isolated and characterized XBB.1, XBB.1.5, XBB.1.9.1, XBB.1.16.1, EG.5.1.1, EG.5.1.3, XBF, BA.2.86.1 and JN.1 variants, representing &gt;80% of circulating variants in November 2023. The XBB subvariants carry few but recurrent mutations in the spike, whereas BA.2.86.1 and JN.1 harbor &gt;30 additional changes. These variants replicated in IGROV-1 but no longer in Vero E6 and were not markedly fusogenic. They potently infected nasal epithelial cells, with EG.5.1.3 exhibiting the highest fitness. Antivirals remained active. Neutralizing antibody (NAb) responses from vaccinees and BA.1/BA.2-infected individuals were markedly lower compared to BA.1, without major differences between variants. An XBB breakthrough infection enhanced NAb responses against both XBB and BA.2.86 variants. JN.1 displayed lower affinity to ACE2 and higher immune evasion properties compared to BA.2.86.1. Thus, while distinct, the evolutionary trajectory of these variants combines increased fitness and antibody evasion.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.09.570830">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.09.570830" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.09.570830">
        <p class="paperTitle">Umbrella toxin particles produced by Streptomyces block mycelial growth of competing species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.09.570830" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.09.570830" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhao, Q.; Bertolli, S.; Park, Y.-J.; Tan, Y.; Cutler, K. J.; Srinivas, P.; Asfahl, K. L.; Fonseca Garcia, C.; Gallagher, L. A.; Li, Y.; Wang, Y.; Coleman-Derr, D.; DiMaio, F.; Zhang, D.; Peterson, S. B.; Veesler, D.; Mougous, J. D.</p>
        <p class="info">Score: 24.9, Published: 2023-12-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.09.570830' target='https://doi.org/10.1101/2023.12.09.570830'> 10.1101/2023.12.09.570830</a></p>
        <p class="abstract">The Streptomyces are a genus of ubiquitous soil bacteria from which the majority of clinically utilized antibiotics derive. The production of these antibacterial molecules reflects the relentless competition Streptomyces engage in with other bacteria, including other Streptomyces species. Here we show that in addition to small molecule antibiotics, Streptomyces produce and secrete antibacterial protein complexes that feature a large, degenerate repeat-containing polymorphic toxin protein. A cryo-EM structure of these particles reveals an extended stalk topped by a ringed crown comprising the toxin repeats scaffolding five lectin-tipped spokes, leading to our naming them umbrella particles. S. coelicolor encodes three umbrella particles with distinct toxin and lectin composition, and supernatant containing these toxins specifically and potently inhibits the growth of select Streptomyces species from among a diverse collection of bacteria screened. For one target, S. griseus, we find inhibition relies on a single toxin and that intoxication manifests as rapid cessation of vegetative mycelial growth. Our data show that Streptomyces umbrella particles mediate competition between vegetative mycelia of related species, a function distinct from small molecule antibiotics, which are produced at the onset of reproductive growth and act broadly. Sequence analyses suggest this role of umbrella particles extends beyond Streptomyces, as we find umbrella loci in nearly one-thousand species across Actinobacteria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.570068">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.570068" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.570068">
        <p class="paperTitle">Elevated mutation rates in the multi-azole resistant Aspergillus fumigatus cladedrives rapid evolution of antifungal resistance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.570068" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.570068" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bottery, M. J.; van Rhijn, N. J.; Chown, H.; Rhodes, J. L.; Celia-Sanchez, B. N.; Brewer, M. T.; Momany, M.; Fisher, M. C.; Knight, C. G.; Bromley, M. J.</p>
        <p class="info">Score: 21.2, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.570068' target='https://doi.org/10.1101/2023.12.05.570068'> 10.1101/2023.12.05.570068</a></p>
        <p class="abstract">The evolution of antifungal resistance is an emerging global threat. Particularly concerning is the widespread occurrence of azole resistance within Aspergillus fumigatus, a globally ubiquitous environmental mould that causes over 1 million life-threatening invasive infections in humans each year. It is increasingly evident that the environmental use of azoles has led to selective sweeps across multiple genomic loci resulting in the rapid expansion of a genetically distinct cluster of genotypes (clade A) that results in resistance to clinically deployed azoles. Isolates within this cluster are more likely to be cross resistant to agricultural antifungals with unrelated modes of action suggesting they may be adapting rapidly to antifungal challenge. Here we show that this cluster is not only multi-azole resistant but has increased propensity to develop resistance to new antifungals because of variants in the DNA mismatch repair system. A variant in msh6 is found almost exclusively within clade A, occurs in 88% of multi-azole resistant isolates harbouring the canonical cyp51A azole resistance allelic variant TR34/L98H, and is globally distributed. Naturally occurring isolates with this msh6 variant display a 4 to 9-times higher rate of mutation, leading to an increased propensity to evolve resistance to current and next generation antifungals. We argue that pervasive environmental use of fungicides creates selective arenas whereby genotypes of A. fumigatus with increased adaptive capability thrive in the face of strong directional selection, leading to the genesis and amplification of antifungal resistance. These results help explain the pronounced clustering of multiple independent resistance mechanisms within the mutable clade A. Our findings further suggest that resistance to next generation antifungals is more likely to emerge within organisms that are already multi-azole resistant, posing a major problem due to the prospect of dual use of novel antifungals in clinical and agricultural settings.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570645">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570645" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570645">
        <p class="paperTitle">Modulation of Salmonella virulence by a novel SPI-2 injectisome effector that interacts with the dystrophin-associated protein complex.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570645" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570645" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yu, X.-J.; Xie, H. X.; Li, Y.; Liu, M.; Hou, R.; Predeus, A.; Perez-Sepulveda, B. M.; Hinton, J.; Holden, D. W.; Thurston, T. L. M.</p>
        <p class="info">Score: 19.9, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570645' target='https://doi.org/10.1101/2023.12.07.570645'> 10.1101/2023.12.07.570645</a></p>
        <p class="abstract">The injectisome encoded by Salmonella pathogenicity island 2 (SPI-2) had been thought to translocate 28 effectors. Here, we used a proteomic approach to characterise the secretome of a clinical strain of invasive non-typhoidal Salmonella enterica serovar Enteritidis, that had been mutated to cause hyper-secretion of the SPI-2 injectisome effectors. Along with many known effectors, we discovered the novel SseM protein. sseM is widely distributed between the five subspecies of Salmonella enterica, is found in many clinically-relevant serovars, and is co-transcribed with pipB2, a SPI-2 effector gene. Translocation of SseM required a functional SPI-2 injectisome. Following expression in human cells, SseM interacted with five components of the dystrophin-associated protein complex (DAPC), namely {beta}-2-syntrophin, utrophin/ dystrophin, -catulin, -dystrobrevin and {beta}-dystrobrevin. The interaction between SseM and {beta}-2-syntrophin and -dystrobrevin was verified in S. Typhimurium-infected cells and relied on the PDZ domain of {beta}-2-syntrophin and a sequence corresponding to a PDZ-binding motif (PBM) in SseM. A {Delta}sseM mutant strain had a small competitive advantage over the wild-type strain in the S. Typhimurium/mouse model of systemic disease. This phenotype was complemented by a plasmid expressing wild type SseM from S. Typhimurium or S. Enteritidis and was dependent on the PBM of SseM. Therefore, a PBM within a Salmonella effector mediates interactions with the DAPC and modulates systemic growth of bacteria in mice.

ImportanceIn Salmonella enterica, the injectisome machinery encoded by Salmonella pathogenicity island 2 (SPI-2) is conserved among the five subspecies and delivers proteins (effectors) into host cells that are required for Salmonella virulence. The identification and functional characterisation of SPI-2 injectisome effectors advances our understanding of the interplay between Salmonella and its host(s). Using an optimised method for preparing secreted proteins and a clinical isolate of the invasive non-typhoidal (iNTS) Salmonella enterica serovar Enteritidis strain D24359, we identified 22 known SPI-2 injectisome effectors and one new effector - SseM. SseM modulates bacterial growth during murine infection and has a sequence corresponding to a PDZ-binding motif that is essential for interaction with the PDZ-containing host protein {beta}-2-syntrophin and other components of the dystrophin-associated protein complex (DAPC). To our knowledge, SseM is unique among Salmonella effectors in containing a functional PDZ-binding motif and is the first bacterial protein to target the DAPC.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.06.570485">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.06.570485" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.06.570485">
        <p class="paperTitle">Selective lipid recruitment by an archaeal DPANN symbiont from its host</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.06.570485" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.06.570485" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ding, S.; Hamm, J. N.; Bale, N. J.; Sinninghe Damste, J. S.; Spang, A.</p>
        <p class="info">Score: 17.3, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.06.570485' target='https://doi.org/10.1101/2023.12.06.570485'> 10.1101/2023.12.06.570485</a></p>
        <p class="abstract">The symbiont Ca. Nanohaloarchaeum antarcticus is obligately dependent on its host Halorubrum lacusprofundi for lipids and other metabolites due to its lack of certain biosynthetic genes. However, it remains unclear which specific lipids or metabolites are acquired from its host, and how the host responds to infection. Here, we explored the lipidome dynamics of the Ca. Nha. antarcticus - Hrr. lacusprofundi symbiotic relationship during co-cultivation. By using a comprehensive untargeted lipidomic methodology, our study reveals that Ca. Nha. antarcticus selectively recruits 110 lipid species from its host, i.e. nearly two-thirds of the total number of host lipids. Lipid profiles of co-cultures displayed shifts in abundances of bacterioruberins and menaquinones and changes in degree of bilayer-forming glycerolipid unsaturation. This likely results in increased membrane fluidity and improved resistance to membrane disruptions, consistent with compensation for higher metabolic load and mechanical stress on host membranes when in contact with Ca. Nha. antarcticus cells. Notably, our findings differ from previous observations of other DPANN symbiont-host systems, where no differences in lipidome composition were reported. Altogether, our work emphasizes the strength of employing untargeted lipidomics approaches to provide details into the dynamics underlying a DPANN symbiont-host system.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
