<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="SARS-CoV-2 Neutralizing Antibodies Following a Second BA.5 Bivalent Booster
Authors: Wang, Q.; Bowen, A.; Ho, J.; Zhang, R.; Valdez, R.; Stoneman, E.; Gordon, A.; Lihong, L.; Ho, D. D.
Score: 355.5, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553148
Bivalent COVID-19 mRNA vaccines expressing both the ancestral D614G and Omicron BA.5 spike proteins were introduced in August 2022 with the goal of broadening immunity to emerging SARS-CoV-2 Omicron subvariants. Subsequent studies on bivalent boosters found neutralizing antibody responses similar to boosters with the original monovalent vaccine, likely the result of immunological imprinting.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="SARS-CoV-2 Neutralizing Antibodies Following a Second BA.5 Bivalent Booster
Authors: Wang, Q.; Bowen, A.; Ho, J.; Zhang, R.; Valdez, R.; Stoneman, E.; Gordon, A.; Lihong, L.; Ho, D. D.
Score: 355.5, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553148
Bivalent COVID-19 mRNA vaccines expressing both the ancestral D614G and Omicron BA.5 spike proteins were introduced in August 2022 with the goal of broadening immunity to emerging SARS-CoV-2 Omicron subvariants. Subsequent studies on bivalent boosters found neutralizing antibody responses similar to boosters with the original monovalent vaccine, likely the result of immunological imprinting." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-27T10:37:02+00:00" />
<meta property="article:modified_time" content="2023-08-27T10:37:02+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="SARS-CoV-2 Neutralizing Antibodies Following a Second BA.5 Bivalent Booster
Authors: Wang, Q.; Bowen, A.; Ho, J.; Zhang, R.; Valdez, R.; Stoneman, E.; Gordon, A.; Lihong, L.; Ho, D. D.
Score: 355.5, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553148
Bivalent COVID-19 mRNA vaccines expressing both the ancestral D614G and Omicron BA.5 spike proteins were introduced in August 2022 with the goal of broadening immunity to emerging SARS-CoV-2 Omicron subvariants. Subsequent studies on bivalent boosters found neutralizing antibody responses similar to boosters with the original monovalent vaccine, likely the result of immunological imprinting."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "SARS-CoV-2 Neutralizing Antibodies Following a Second BA.5 Bivalent Booster\nAuthors: Wang, Q.; Bowen, A.; Ho, J.; Zhang, R.; Valdez, R.; Stoneman, E.; Gordon, A.; Lihong, L.; Ho, D. D.\nScore: 355.5, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553148\nBivalent COVID-19 mRNA vaccines expressing both the ancestral D614G and Omicron BA.5 spike proteins were introduced in August 2022 with the goal of broadening immunity to emerging SARS-CoV-2 Omicron subvariants. Subsequent studies on bivalent boosters found neutralizing antibody responses similar to boosters with the original monovalent vaccine, likely the result of immunological imprinting.",
  "keywords": [
    
  ],
  "articleBody": " SARS-CoV-2 Neutralizing Antibodies Following a Second BA.5 Bivalent Booster\nAuthors: Wang, Q.; Bowen, A.; Ho, J.; Zhang, R.; Valdez, R.; Stoneman, E.; Gordon, A.; Lihong, L.; Ho, D. D.\nScore: 355.5, Published: 2023-08-14 DOI: 10.1101/2023.08.13.553148\nBivalent COVID-19 mRNA vaccines expressing both the ancestral D614G and Omicron BA.5 spike proteins were introduced in August 2022 with the goal of broadening immunity to emerging SARS-CoV-2 Omicron subvariants. Subsequent studies on bivalent boosters found neutralizing antibody responses similar to boosters with the original monovalent vaccine, likely the result of immunological imprinting. Guidelines allow for administration of a second bivalent booster in high-risk groups, but it remains unknown whether this would broaden antibody responses. To address this question, we assessed longitudinal serum SARS-CoV-2-neutralizing titers in 18 elderly immunocompetent individuals (mean age 69) following a fourth monovalent booster and two BA.5 bivalent booster vaccines using pseudovirus neutralization assays against D614G, Omicron BA.5, and Omicron XBB.1.5. There was a small but significant increase in peak neutralizing antibody responses against Omicron BA.5 and XBB.1.5 following the first bivalent booster, but no significant increase in peak titers following the second bivalent booster. Omicron-specific neutralizing titers remained low after both doses of the BA.5 bivalent booster. Our results suggest that a second dose of the BA.5 bivalent booster is not sufficient to broaden antibody responses and to overcome immunological imprinting. A monovalent vaccine targeting only the spike of the recently dominant SARS-CoV-2 may mitigate the \"back boosting\" associated with the \"original antigenic sin.\"\nTemperate gut phages are prevalent, diverse, and predominantly inactive\nAuthors: Dahlman, S.; Avellaneda-Franco, L.; Kett, C.; Subedi, D.; Young, R. B.; Gould, J. A.; Rutten, E. L.; Gilliver, E. L.; Turkington, C. J.; Nezam-Abadi, N.; Grasis, J. A.; Lyras, D.; Edwards, R. A.; Forster, S. C.; Barr, J. J.\nScore: 60.6, Published: 2023-08-18 DOI: 10.1101/2023.08.17.553642\nLarge-scale metagenomic and data mining efforts have uncovered an expansive diversity of bacteriophages (phages) within the human gut1-3. These insights include broader phage populational dynamics such as temporal stability4, interindividual uniqueness5,6 and potential associations to specific disease states7,8. However, the functional understanding of phage-host interactions and their impacts within this complex ecosystem have been limited due to a lack of cultured isolates for experimental validation. Here we characterise 125 active prophages originating from 252 diverse human gut bacterial isolates using seven different induction conditions to substantially expand the experimentally validated temperate phage-host pairs originating from the human gut. Importantly, only 17% of computationally predicted prophages were induced with common induction agents and these exhibited distinct gene patterns compared to non-induced predictions. Active Bacteroidota prophages were among the most prevalent members of the gut virome, with extensive use of diversity generating retroelements and exhibiting broad host ranges. Moreover, active polylysogeny was present in 52% of studied gut lysogens and led to coordinated prophage induction across diverse conditions. This study represents a substantial expansion of experimentally validated gut prophages, providing key insights into their diversity and genetics, including a genetic pathway for prophage domestication and demonstration that differential induction was complex and influenced by divergent prophage integration sites. More broadly, it highlights the importance of experimental validation alongside genomic based computational prediction to enable further functional understanding of these commensal viruses within the human gut.\nVirological characteristics of the SARS-CoV-2 XBB.1.5 variant\nAuthors: Tamura, T.; Irie, T.; Deguchi, S.; Yajima, H.; Tsuda, M.; Nasser, H.; Mizuma, K.; Plianchaisuk, A.; Suzuki, S.; Uriu, K.; Begum, M. M.; Shimizu, R.; Jonathan, M.; Suzuki, R.; Kondo, T.; Ito, H.; Kamiyama, A.; Yoshimatsu, K.; Shofa, M.; Hashimoto, R.; Anraku, Y.; Kimura, K. T.; Kita, S.; Sasaki, J.; Sasaki-Tabata, K.; Maenaka, K.; Nao, N.; Wang, L.; Oda, Y.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Ikeda, T.; Saito, A.; Matsuno, K.; Ito, J.; Tanaka, S.; Sato, K.; Hashiguchi, T.; Takayama, K.; Fukuhara, T.\nScore: 37.8, Published: 2023-08-17 DOI: 10.1101/2023.08.16.553332\nCirculation of SARS-CoV-2 Omicron XBB has resulted in the emergence of XBB.1.5, a new Variant of Interest. Our phylogenetic analysis suggests that XBB.1.5 evolved from XBB.1 by acquiring the F486P spike (S) mutation, subsequent to the acquisition of a nonsense mutation in ORF8. Neutralization assays showed similar abilities of immune escape between XBB.1.5 and XBB.1. We determined the structural basis for the interaction between human ACE2 and the S protein of XBB.1.5, showing similar overall structures between the S proteins of XBB.1 and XBB.1.5. The intrinsic pathogenicity of XBB.1.5 in hamsters is lower than that of XBB.1. Importantly, we found that the ORF8 nonsense mutation of XBB.1.5 resulted in impairment of MHC expression. In vivo experiments using recombinant viruses revealed that the XBB.1.5 mutations are involved with reduced virulence of XBB.1.5. Together, these data suggest that the mutations in ORF8 and S could enhance spreading of XBB.1.5 in humans.\nNon-antibiotic pharmaceuticals exhibit toxicity against Escherichia coli at environmentallyrelevant concentrations with no evolution of cross-resistance to antibiotics\nAuthors: Hall, R. J.; Snaith, A. E.; Element, S.; Moran, R. A.; Smith, H.; Cummins, E. A.; Bottery, M. J.; Chowdhury, K. F.; Sareen, D.; Ahmad, I.; Blair, J. M. A.; Carter, L. J.; McNally, A.\nScore: 20.3, Published: 2023-08-22 DOI: 10.1101/2023.08.21.554069\nAntimicrobial resistance can arise in the natural environment via prolonged exposure to the effluent surrounding manufacturing facilities. These facilities also produce non-antibiotic pharmaceuticals, and the effect of these on the surrounding microbial communities is less clear; whether they have inherent toxicity, or whether long-term exposure might select for cross-resistance to antibiotics. To this end, we screened four non-antibiotic pharmaceuticals (acetaminophen, ibuprofen, propranolol, met formin) and titanium dioxide for toxicity against Escherichia coli K-12 MG1655 and conducted a 30 day selection experiment to assess the effect of long-term exposure. All compounds reduced the maximum optical density reached by E. coli at a range of concentrations including one of environmental relevance, with transcriptome analysis identifying upregulated genes related to stress response and multidrug efflux in response ibuprofen treatment. The non-antibiotic pharmaceuticals did not select for significant genetic changes following a 30 day exposure, and no evidence of selection for cross-resistance to antibiotics was observed for population evolved in the presence of ibuprofen in spite of the differential gene expression after exposure to this compound. This work suggests that these non-antibiotic pharmaceuticals, at environmental concentrations, do not select for cross-resistance to antibiotics in E. coli.\nMother-Infant Gut Viruses and their Bacterial Hosts: Transmission Patterns and Dynamics during Pregnancy and Early Life\nAuthors: Garmaeva, S.; Sinha, T.; Gulyaeva, A.; Kuzub, N.; Spreckels, J. E.; Andreu-Sanchez, S.; Gacesa, R.; Vich Vila, A.; Brushett, S.; Kruk, M.; Dekens, J.; Sikkema, J.; Kuipers, F.; Shkoporov, A.; Hill, C.; Scherjon, S.; Wijmenga, C.; Fu, J.; Kurilshikov, A.; Zhernakova, A.\nScore: 16.9, Published: 2023-08-21 DOI: 10.1101/2023.08.21.554108\nEarly development of the gut ecosystem is crucial for lifelong health. While infant gut bacterial communities have been studied extensively, the infant gut virome remains under-explored. We longitudinally assessed the composition of gut viruses and their bacterial hosts in 322 total metagenomes and 205 metaviromes from 30 mothers during and after pregnancy and from their 32 infants during their first year of life. While the maternal gut virome composition remained stable during late pregnancy and after birth, the infant gut virome was dynamic in the first year of life and contained a higher abundance of active temperate phages compared to the maternal gut viromes. The infant gut virome composition was also influenced by infant feeding mode and place of delivery. Lastly, we provide evidence of viral-bacterial strains co-transmission from mothers to infants, demonstrating that infants acquire some of their virome from their mothers gut. Highlights- Longitudinal characterisation of the gut microbiome and virome in 30 mothers during pregnancy, at birth and 3 months after birth and in 32 infants from birth across the first year of life. - The maternal gut bacteriome changes from the first to the second trimester and then remains stable through birth and the first 3 months after birth. - The maternal gut virome remains stable during late pregnancy, birth and the first 3 months after birth. - The infant gut virome is highly dynamic during the first year of life and is shaped by infant feeding mode and place of delivery. - The infant gut harbours more temperate bacteriophages than the maternal gut, but their relative abundance decreases with increasing infant age. - Gut viral strains and their bacterial host strains are co-transmitted from mothers to their infants. - Gut viral strains are transferred from mother to infant around birth directly or via transfer of their bacterial hosts followed by the induction of prophages.\nDisruption of Aldehyde Dehydrogenase 2 protects against bacterial infection\nAuthors: Berry, S.; Espich, S.; Thuong, N. T. T.; Chang, X.; Dorajoo, R.; Khor, C. C.; Heng, C. K.; Yuan, J.-M.; Fox, D.; Anaya-Sanchez, A.; Tenney, L.; Chang, C. J.; Kotov, D. I.; Vance, R. E.; Dunstan, S. J.; Darwin, K. H.; Stanley, S. A.\nScore: 16.0, Published: 2023-08-26 DOI: 10.1101/2023.08.24.554661\nThe ALDH2*2 (rs671) allele is one of the most common genetic mutations in humans, yet the positive evolutionary selective pressure to maintain this mutation is unknown, despite its association with adverse health outcomes. ALDH2 is responsible for the detoxification of metabolically produced aldehydes, including lipid-peroxidation end products derived from inflammation. Here, we demonstrate that host-derived aldehydes 4-hydroxynonenal (4HNE), malondialdehyde (MDA), and formaldehyde (FA), all of which are metabolized by ALDH2, are directly toxic to the bacterial pathogens Mycobacterium tuberculosis and Francisella tularensis at physiological levels. We find that Aldh2 expression in macrophages is decreased upon immune stimulation, and that bone marrow-derived macrophages from Aldh2-/- mice contain elevated aldehydes relative to wild-type mice. Macrophages deficient for Aldh2 exhibited enhanced control of Francisella infection. Finally, mice lacking Aldh2 demonstrated increased resistance to pulmonary infection by M. tuberculosis, including in a hypersusceptible model of tuberculosis, and were also resistant to Francisella infection. We hypothesize that the absence of ALDH2 contributes to the hosts ability to control infection by pathogens such as M. tuberculosis and F. tularensis, and that host-derived aldehydes act as antimicrobial factors during intracellular bacterial infections.\nAntibody Neutralization of Emerging SARS-CoV-2: EG.5.1 and XBC.1.6\nAuthors: Wang, Q.; Guo, Y.; Zhang, R. M.; Ho, J.; Mohri, H.; Valdez, R.; Manthei, D. M.; Gordon, A.; Liu, L.; Ho, D. D.\nScore: 15.9, Published: 2023-08-25 DOI: 10.1101/2023.08.21.553968\nSARS-CoV-2 variants EG.5.1 and XBC.1.6 have recently emerged, attracting increased attention due to their rapid expansion globally and in Australia, respectively. EG.5.1 evolved from Omicron subvariant XBB.1.9, harboring additional Q52H and F456L spike substitutions. The F456L mutation is located within the epitopes of many class-1 monoclonal antibodies (mAbs) directed to the receptor-binding domain (RBD), raising concerns about further antibody evasion. XBC.1.6, a descendant of a Delta-BA.2 recombinant, carries 15 additional spike mutations. The extent to which antibody evasion contributes to the growth advantage of XBC.1.6 in Australia remains to be determined. To assess the antibody evasion properties of the emergent variants, we conducted pseudovirus neutralization assays using sera from individuals who received three doses of COVID-19 mRNA monovalent vaccines plus one dose of a BA.5 bivalent vaccine, as well as from patients with BQ or XBB breakthrough infection. The assays were also performed using a panel of 14 mAbs that retained neutralizing activity against prior XBB subvariants. Our data suggested that EG.5.1 was slightly but significantly more resistant (\u003c 2-fold) to neutralization by BQ and XBB breakthrough sera than XBB.1.16, which is known to be antigenically similar to XBB.1.5. Moreover, the F456L mutation in EG.5.1 conferred heightened resistance to certain RBD class-1 mAbs. In contrast, XBC.1.6 was more sensitive to neutralization by sera and mAbs than the XBB subvariants. Notably, XBB breakthrough sera retained only weak neutralization activity against XBB subvariants. In summary, EG.5.1 and XBC.1.6 exhibited distinct antibody evasion properties. The recent global expansion of EG.5.1 might be attributable, in part, to its enhanced neutralization resistance. That XBB breakthrough infections did not elicit a robust antibody neutralization response against XBB subvariants is indicative of immunological imprinting. The high prevalence of XBC.1.6 in Australia is not due to enhanced antibody evasion.\nMiDAS 5: Global diversity of bacteria and archaea in anaerobic digesters\nAuthors: Dueholm, M. K. D.; Andersen, K. S.; Petersen, A.-K. C.; Rudkjoebing, V.; MiDAS Global Consortium for Anaerobic Digesters, ; Nielsen, P. H.\nScore: 12.0, Published: 2023-08-24 DOI: 10.1101/2023.08.24.554448\nAnaerobic digestion represents a key biotechnology for the transformation of organic waste into renewable energy (biogas) and relies on complex microbial communities that work in concert to degrade the complex substrates into methane and carbon dioxide. Here, we sequenced more than half a million high-quality, full-length 16S rRNA gene sequences from 285 full-scale anaerobic digesters (ADs) across the world to expand our knowledge about diversity and function of the bacteria and archaea in ADs. The sequences were processed into full-length 16S rRNA amplicon sequence variants (FL-ASVs), which were added to the MiDAS 4 database for bacteria and archaea in wastewater treatment systems to create MiDAS 5. The expansion of the MiDAS database significantly increased the coverage for bacteria and archaea in ADs worldwide, leading to an improved rate of genus and species-level classification. Using MiDAS 5, we carried out an amplicon-based, global-scale microbial community profiling of the sampled ADs using three common sets of primers targeting different regions of the 16S rRNA gene in bacteria and/or archaea. We revealed how environmental conditions and biogeography shape the AD microbiota. We also identify core and conditionally rare or abundant taxa, encompassing less than 1000 genera and species. These represent 83-99% and 10-64% of the accumulated read abundance respectively, across samples depending on the amplicon primers used. Finally, we examined the global diversity of functional groups with known importance for the anaerobic digestion process. Our online global MiDAS Field Guide presents the data generated in this study and summarizes present knowledge about all taxa.\nInnate immune control of influenza virus interspecies adaptation\nAuthors: Denz, P. J.; Speaks, S.; Kenney, A. D.; Eddy, A. C.; Papa, J. L.; Roettger, J.; Scace, S. C.; Hemann, E. A.; Forero, A.; Webby, R. J.; Bowman, A. S.; Yount, J. S.\nScore: 11.3, Published: 2023-08-23 DOI: 10.1101/2023.08.23.554491\nInfluenza virus pandemics are caused by viruses from animal reservoirs that adapt to efficiently infect and replicate in human hosts. Here, we investigated whether Interferon-Induced Transmembrane Protein 3 (IFITM3), a host antiviral factor with known human deficiencies, plays a role in interspecies virus infection and adaptation. We found that IFITM3-deficient mice and human cells could be infected with low doses of avian influenza viruses that failed to infect WT counterparts, identifying a new role for IFITM3 in controlling the minimum infectious viral dose threshold. Remarkably, influenza viruses passaged through Ifitm3-/- mice exhibited enhanced host adaptation, a result that was distinct from passaging in mice deficient for interferon signaling, which caused virus attenuation. Our data demonstrate that IFITM3 deficiency uniquely facilitates zoonotic influenza virus infections and subsequent adaptation, implicating IFITM3 deficiencies in the human population as a vulnerability for emergence of new pandemic viruses.\nSpace and epigenetic inheritance determine inter-individual differences in siderophore gene expression in bacterial colonies\nAuthors: Mridha, S.; Wechsler, T.; Kuemmerli, R.\nScore: 10.2, Published: 2023-08-21 DOI: 10.1101/2023.08.21.554085\nHeterogeneity in gene expression among cells in clonal groups is common in bacteria. Albeit ubiquitous, it often remains unclear what the sources of variation are and whether variation has functional significance. Here, we tracked the expression of genes involved in the synthesis of iron-chelating siderophores (pyoverdine and pyochelin) in individual cells of the bacterium Pseudomonas aeruginosa during colony growth on surfaces using time-lapse fluorescence microscopy, to explore potential sources and functions of cellular heterogeneity. Regarding sources, we found that the physical position of cells within the colony and epigenetic gene expression inheritance from mother to daughter cells significantly contributed to cellular heterogeneity. In contrast, cell pole age and cellular lifespan had no effect. Regarding functions, our results indicate that cells optimize their siderophore investment strategies (pyoverdine vs. pyochelin) along a gradient from the centre to the edge of the colony. Moreover, we found evidence that cell lineages with above-average siderophore investment increase the fitness of cell lineages with below-average investment through cooperative sharing of secreted siderophores. Altogether, our study highlights that single-cell experiments combined with automated image and cell-tracking analyses can identify sources of heterogeneity and yield adaptive explanations for gene expression variation among clonal bacterial cells.\n",
  "wordCount" : "2686",
  "inLanguage": "en",
  "datePublished": "2023-08-27T10:37:02Z",
  "dateModified": "2023-08-27T10:37:02Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta"><span>updated on August 27, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.13.553148">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.13.553148" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.13.553148">
        <p class="paperTitle">SARS-CoV-2 Neutralizing Antibodies Following a Second BA.5 Bivalent Booster</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.13.553148" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.13.553148" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Q.; Bowen, A.; Ho, J.; Zhang, R.; Valdez, R.; Stoneman, E.; Gordon, A.; Lihong, L.; Ho, D. D.</p>
        <p class="info">Score: 355.5, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.13.553148' target='https://doi.org/10.1101/2023.08.13.553148'> 10.1101/2023.08.13.553148</a></p>
        <p class="abstract">Bivalent COVID-19 mRNA vaccines expressing both the ancestral D614G and Omicron BA.5 spike proteins were introduced in August 2022 with the goal of broadening immunity to emerging SARS-CoV-2 Omicron subvariants. Subsequent studies on bivalent boosters found neutralizing antibody responses similar to boosters with the original monovalent vaccine, likely the result of immunological imprinting. Guidelines allow for administration of a second bivalent booster in high-risk groups, but it remains unknown whether this would broaden antibody responses. To address this question, we assessed longitudinal serum SARS-CoV-2-neutralizing titers in 18 elderly immunocompetent individuals (mean age 69) following a fourth monovalent booster and two BA.5 bivalent booster vaccines using pseudovirus neutralization assays against D614G, Omicron BA.5, and Omicron XBB.1.5. There was a small but significant increase in peak neutralizing antibody responses against Omicron BA.5 and XBB.1.5 following the first bivalent booster, but no significant increase in peak titers following the second bivalent booster. Omicron-specific neutralizing titers remained low after both doses of the BA.5 bivalent booster. Our results suggest that a second dose of the BA.5 bivalent booster is not sufficient to broaden antibody responses and to overcome immunological imprinting. A monovalent vaccine targeting only the spike of the recently dominant SARS-CoV-2 may mitigate the &#34;back boosting&#34; associated with the &#34;original antigenic sin.&#34;</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.17.553642">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.17.553642" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.17.553642">
        <p class="paperTitle">Temperate gut phages are prevalent, diverse, and predominantly inactive</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.17.553642" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.17.553642" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dahlman, S.; Avellaneda-Franco, L.; Kett, C.; Subedi, D.; Young, R. B.; Gould, J. A.; Rutten, E. L.; Gilliver, E. L.; Turkington, C. J.; Nezam-Abadi, N.; Grasis, J. A.; Lyras, D.; Edwards, R. A.; Forster, S. C.; Barr, J. J.</p>
        <p class="info">Score: 60.6, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.17.553642' target='https://doi.org/10.1101/2023.08.17.553642'> 10.1101/2023.08.17.553642</a></p>
        <p class="abstract">Large-scale metagenomic and data mining efforts have uncovered an expansive diversity of bacteriophages (phages) within the human gut1-3. These insights include broader phage populational dynamics such as temporal stability4, interindividual uniqueness5,6 and potential associations to specific disease states7,8. However, the functional understanding of phage-host interactions and their impacts within this complex ecosystem have been limited due to a lack of cultured isolates for experimental validation. Here we characterise 125 active prophages originating from 252 diverse human gut bacterial isolates using seven different induction conditions to substantially expand the experimentally validated temperate phage-host pairs originating from the human gut. Importantly, only 17% of computationally predicted prophages were induced with common induction agents and these exhibited distinct gene patterns compared to non-induced predictions. Active Bacteroidota prophages were among the most prevalent members of the gut virome, with extensive use of diversity generating retroelements and exhibiting broad host ranges. Moreover, active polylysogeny was present in 52% of studied gut lysogens and led to coordinated prophage induction across diverse conditions. This study represents a substantial expansion of experimentally validated gut prophages, providing key insights into their diversity and genetics, including a genetic pathway for prophage domestication and demonstration that differential induction was complex and influenced by divergent prophage integration sites. More broadly, it highlights the importance of experimental validation alongside genomic based computational prediction to enable further functional understanding of these commensal viruses within the human gut.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.553332">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.553332" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.553332">
        <p class="paperTitle">Virological characteristics of the SARS-CoV-2 XBB.1.5 variant</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.553332" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.553332" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tamura, T.; Irie, T.; Deguchi, S.; Yajima, H.; Tsuda, M.; Nasser, H.; Mizuma, K.; Plianchaisuk, A.; Suzuki, S.; Uriu, K.; Begum, M. M.; Shimizu, R.; Jonathan, M.; Suzuki, R.; Kondo, T.; Ito, H.; Kamiyama, A.; Yoshimatsu, K.; Shofa, M.; Hashimoto, R.; Anraku, Y.; Kimura, K. T.; Kita, S.; Sasaki, J.; Sasaki-Tabata, K.; Maenaka, K.; Nao, N.; Wang, L.; Oda, Y.; The Genotype to Phenotype Japan (G2P-Japan) Consortium,  ; Ikeda, T.; Saito, A.; Matsuno, K.; Ito, J.; Tanaka, S.; Sato, K.; Hashiguchi, T.; Takayama, K.; Fukuhara, T.</p>
        <p class="info">Score: 37.8, Published: 2023-08-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.553332' target='https://doi.org/10.1101/2023.08.16.553332'> 10.1101/2023.08.16.553332</a></p>
        <p class="abstract">Circulation of SARS-CoV-2 Omicron XBB has resulted in the emergence of XBB.1.5, a new Variant of Interest. Our phylogenetic analysis suggests that XBB.1.5 evolved from XBB.1 by acquiring the F486P spike (S) mutation, subsequent to the acquisition of a nonsense mutation in ORF8. Neutralization assays showed similar abilities of immune escape between XBB.1.5 and XBB.1. We determined the structural basis for the interaction between human ACE2 and the S protein of XBB.1.5, showing similar overall structures between the S proteins of XBB.1 and XBB.1.5. The intrinsic pathogenicity of XBB.1.5 in hamsters is lower than that of XBB.1. Importantly, we found that the ORF8 nonsense mutation of XBB.1.5 resulted in impairment of MHC expression. In vivo experiments using recombinant viruses revealed that the XBB.1.5 mutations are involved with reduced virulence of XBB.1.5. Together, these data suggest that the mutations in ORF8 and S could enhance spreading of XBB.1.5 in humans.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554069">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554069" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554069">
        <p class="paperTitle">Non-antibiotic pharmaceuticals exhibit toxicity against Escherichia coli at environmentallyrelevant concentrations with no evolution of cross-resistance to antibiotics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554069" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554069" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hall, R. J.; Snaith, A. E.; Element, S.; Moran, R. A.; Smith, H.; Cummins, E. A.; Bottery, M. J.; Chowdhury, K. F.; Sareen, D.; Ahmad, I.; Blair, J. M. A.; Carter, L. J.; McNally, A.</p>
        <p class="info">Score: 20.3, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554069' target='https://doi.org/10.1101/2023.08.21.554069'> 10.1101/2023.08.21.554069</a></p>
        <p class="abstract">Antimicrobial resistance can arise in the natural environment via prolonged exposure to the effluent surrounding manufacturing facilities. These facilities also produce non-antibiotic pharmaceuticals, and the effect of these on the surrounding microbial communities is less clear; whether they have inherent toxicity, or whether long-term exposure might select for cross-resistance to antibiotics. To this end, we screened four non-antibiotic pharmaceuticals (acetaminophen, ibuprofen, propranolol, met formin) and titanium dioxide for toxicity against Escherichia coli K-12 MG1655 and conducted a 30 day selection experiment to assess the effect of long-term exposure. All compounds reduced the maximum optical density reached by E. coli at a range of concentrations including one of environmental relevance, with transcriptome analysis identifying upregulated genes related to stress response and multidrug efflux in response ibuprofen treatment. The non-antibiotic pharmaceuticals did not select for significant genetic changes following a 30 day exposure, and no evidence of selection for cross-resistance to antibiotics was observed for population evolved in the presence of ibuprofen in spite of the differential gene expression after exposure to this compound. This work suggests that these non-antibiotic pharmaceuticals, at environmental concentrations, do not select for cross-resistance to antibiotics in E. coli.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554108">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554108" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554108">
        <p class="paperTitle">Mother-Infant Gut Viruses and their Bacterial Hosts: Transmission Patterns and Dynamics during Pregnancy and Early Life</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554108" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554108" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Garmaeva, S.; Sinha, T.; Gulyaeva, A.; Kuzub, N.; Spreckels, J. E.; Andreu-Sanchez, S.; Gacesa, R.; Vich Vila, A.; Brushett, S.; Kruk, M.; Dekens, J.; Sikkema, J.; Kuipers, F.; Shkoporov, A.; Hill, C.; Scherjon, S.; Wijmenga, C.; Fu, J.; Kurilshikov, A.; Zhernakova, A.</p>
        <p class="info">Score: 16.9, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554108' target='https://doi.org/10.1101/2023.08.21.554108'> 10.1101/2023.08.21.554108</a></p>
        <p class="abstract">Early development of the gut ecosystem is crucial for lifelong health. While infant gut bacterial communities have been studied extensively, the infant gut virome remains under-explored. We longitudinally assessed the composition of gut viruses and their bacterial hosts in 322 total metagenomes and 205 metaviromes from 30 mothers during and after pregnancy and from their 32 infants during their first year of life. While the maternal gut virome composition remained stable during late pregnancy and after birth, the infant gut virome was dynamic in the first year of life and contained a higher abundance of active temperate phages compared to the maternal gut viromes. The infant gut virome composition was also influenced by infant feeding mode and place of delivery. Lastly, we provide evidence of viral-bacterial strains co-transmission from mothers to infants, demonstrating that infants acquire some of their virome from their mothers gut.

Highlights- Longitudinal characterisation of the gut microbiome and virome in 30 mothers during pregnancy, at birth and 3 months after birth and in 32 infants from birth across the first year of life.
- The maternal gut bacteriome changes from the first to the second trimester and then remains stable through birth and the first 3 months after birth.
- The maternal gut virome remains stable during late pregnancy, birth and the first 3 months after birth.
- The infant gut virome is highly dynamic during the first year of life and is shaped by infant feeding mode and place of delivery.
- The infant gut harbours more temperate bacteriophages than the maternal gut, but their relative abundance decreases with increasing infant age.
- Gut viral strains and their bacterial host strains are co-transmitted from mothers to their infants.
- Gut viral strains are transferred from mother to infant around birth directly or via transfer of their bacterial hosts followed by the induction of prophages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554661">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554661" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554661">
        <p class="paperTitle">Disruption of Aldehyde Dehydrogenase 2 protects against bacterial infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554661" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554661" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Berry, S.; Espich, S.; Thuong, N. T. T.; Chang, X.; Dorajoo, R.; Khor, C. C.; Heng, C. K.; Yuan, J.-M.; Fox, D.; Anaya-Sanchez, A.; Tenney, L.; Chang, C. J.; Kotov, D. I.; Vance, R. E.; Dunstan, S. J.; Darwin, K. H.; Stanley, S. A.</p>
        <p class="info">Score: 16.0, Published: 2023-08-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554661' target='https://doi.org/10.1101/2023.08.24.554661'> 10.1101/2023.08.24.554661</a></p>
        <p class="abstract">The ALDH2*2 (rs671) allele is one of the most common genetic mutations in humans, yet the positive evolutionary selective pressure to maintain this mutation is unknown, despite its association with adverse health outcomes. ALDH2 is responsible for the detoxification of metabolically produced aldehydes, including lipid-peroxidation end products derived from inflammation. Here, we demonstrate that host-derived aldehydes 4-hydroxynonenal (4HNE), malondialdehyde (MDA), and formaldehyde (FA), all of which are metabolized by ALDH2, are directly toxic to the bacterial pathogens Mycobacterium tuberculosis and Francisella tularensis at physiological levels. We find that Aldh2 expression in macrophages is decreased upon immune stimulation, and that bone marrow-derived macrophages from Aldh2-/- mice contain elevated aldehydes relative to wild-type mice. Macrophages deficient for Aldh2 exhibited enhanced control of Francisella infection. Finally, mice lacking Aldh2 demonstrated increased resistance to pulmonary infection by M. tuberculosis, including in a hypersusceptible model of tuberculosis, and were also resistant to Francisella infection. We hypothesize that the absence of ALDH2 contributes to the hosts ability to control infection by pathogens such as M. tuberculosis and F. tularensis, and that host-derived aldehydes act as antimicrobial factors during intracellular bacterial infections.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.553968">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.553968" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.553968">
        <p class="paperTitle">Antibody Neutralization of Emerging SARS-CoV-2: EG.5.1 and XBC.1.6</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.553968" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.553968" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Q.; Guo, Y.; Zhang, R. M.; Ho, J.; Mohri, H.; Valdez, R.; Manthei, D. M.; Gordon, A.; Liu, L.; Ho, D. D.</p>
        <p class="info">Score: 15.9, Published: 2023-08-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.553968' target='https://doi.org/10.1101/2023.08.21.553968'> 10.1101/2023.08.21.553968</a></p>
        <p class="abstract">SARS-CoV-2 variants EG.5.1 and XBC.1.6 have recently emerged, attracting increased attention due to their rapid expansion globally and in Australia, respectively. EG.5.1 evolved from Omicron subvariant XBB.1.9, harboring additional Q52H and F456L spike substitutions. The F456L mutation is located within the epitopes of many class-1 monoclonal antibodies (mAbs) directed to the receptor-binding domain (RBD), raising concerns about further antibody evasion. XBC.1.6, a descendant of a Delta-BA.2 recombinant, carries 15 additional spike mutations. The extent to which antibody evasion contributes to the growth advantage of XBC.1.6 in Australia remains to be determined. To assess the antibody evasion properties of the emergent variants, we conducted pseudovirus neutralization assays using sera from individuals who received three doses of COVID-19 mRNA monovalent vaccines plus one dose of a BA.5 bivalent vaccine, as well as from patients with BQ or XBB breakthrough infection. The assays were also performed using a panel of 14 mAbs that retained neutralizing activity against prior XBB subvariants. Our data suggested that EG.5.1 was slightly but significantly more resistant (&lt; 2-fold) to neutralization by BQ and XBB breakthrough sera than XBB.1.16, which is known to be antigenically similar to XBB.1.5. Moreover, the F456L mutation in EG.5.1 conferred heightened resistance to certain RBD class-1 mAbs. In contrast, XBC.1.6 was more sensitive to neutralization by sera and mAbs than the XBB subvariants. Notably, XBB breakthrough sera retained only weak neutralization activity against XBB subvariants. In summary, EG.5.1 and XBC.1.6 exhibited distinct antibody evasion properties. The recent global expansion of EG.5.1 might be attributable, in part, to its enhanced neutralization resistance. That XBB breakthrough infections did not elicit a robust antibody neutralization response against XBB subvariants is indicative of immunological imprinting. The high prevalence of XBC.1.6 in Australia is not due to enhanced antibody evasion.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554448">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554448" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554448">
        <p class="paperTitle">MiDAS 5: Global diversity of bacteria and archaea in anaerobic digesters</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554448" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554448" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dueholm, M. K. D.; Andersen, K. S.; Petersen, A.-K. C.; Rudkjoebing, V.; MiDAS Global Consortium for Anaerobic Digesters,  ; Nielsen, P. H.</p>
        <p class="info">Score: 12.0, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554448' target='https://doi.org/10.1101/2023.08.24.554448'> 10.1101/2023.08.24.554448</a></p>
        <p class="abstract">Anaerobic digestion represents a key biotechnology for the transformation of organic waste into renewable energy (biogas) and relies on complex microbial communities that work in concert to degrade the complex substrates into methane and carbon dioxide. Here, we sequenced more than half a million high-quality, full-length 16S rRNA gene sequences from 285 full-scale anaerobic digesters (ADs) across the world to expand our knowledge about diversity and function of the bacteria and archaea in ADs. The sequences were processed into full-length 16S rRNA amplicon sequence variants (FL-ASVs), which were added to the MiDAS 4 database for bacteria and archaea in wastewater treatment systems to create MiDAS 5. The expansion of the MiDAS database significantly increased the coverage for bacteria and archaea in ADs worldwide, leading to an improved rate of genus and species-level classification. Using MiDAS 5, we carried out an amplicon-based, global-scale microbial community profiling of the sampled ADs using three common sets of primers targeting different regions of the 16S rRNA gene in bacteria and/or archaea. We revealed how environmental conditions and biogeography shape the AD microbiota. We also identify core and conditionally rare or abundant taxa, encompassing less than 1000 genera and species. These represent 83-99% and 10-64% of the accumulated read abundance respectively, across samples depending on the amplicon primers used. Finally, we examined the global diversity of functional groups with known importance for the anaerobic digestion process. Our online global MiDAS Field Guide presents the data generated in this study and summarizes present knowledge about all taxa.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554491">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554491" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554491">
        <p class="paperTitle">Innate immune control of influenza virus interspecies adaptation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554491" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554491" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Denz, P. J.; Speaks, S.; Kenney, A. D.; Eddy, A. C.; Papa, J. L.; Roettger, J.; Scace, S. C.; Hemann, E. A.; Forero, A.; Webby, R. J.; Bowman, A. S.; Yount, J. S.</p>
        <p class="info">Score: 11.3, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554491' target='https://doi.org/10.1101/2023.08.23.554491'> 10.1101/2023.08.23.554491</a></p>
        <p class="abstract">Influenza virus pandemics are caused by viruses from animal reservoirs that adapt to efficiently infect and replicate in human hosts. Here, we investigated whether Interferon-Induced Transmembrane Protein 3 (IFITM3), a host antiviral factor with known human deficiencies, plays a role in interspecies virus infection and adaptation. We found that IFITM3-deficient mice and human cells could be infected with low doses of avian influenza viruses that failed to infect WT counterparts, identifying a new role for IFITM3 in controlling the minimum infectious viral dose threshold. Remarkably, influenza viruses passaged through Ifitm3-/- mice exhibited enhanced host adaptation, a result that was distinct from passaging in mice deficient for interferon signaling, which caused virus attenuation. Our data demonstrate that IFITM3 deficiency uniquely facilitates zoonotic influenza virus infections and subsequent adaptation, implicating IFITM3 deficiencies in the human population as a vulnerability for emergence of new pandemic viruses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554085">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554085" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554085">
        <p class="paperTitle">Space and epigenetic inheritance determine inter-individual differences in siderophore gene expression in bacterial colonies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554085" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554085" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mridha, S.; Wechsler, T.; Kuemmerli, R.</p>
        <p class="info">Score: 10.2, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554085' target='https://doi.org/10.1101/2023.08.21.554085'> 10.1101/2023.08.21.554085</a></p>
        <p class="abstract">Heterogeneity in gene expression among cells in clonal groups is common in bacteria. Albeit ubiquitous, it often remains unclear what the sources of variation are and whether variation has functional significance. Here, we tracked the expression of genes involved in the synthesis of iron-chelating siderophores (pyoverdine and pyochelin) in individual cells of the bacterium Pseudomonas aeruginosa during colony growth on surfaces using time-lapse fluorescence microscopy, to explore potential sources and functions of cellular heterogeneity. Regarding sources, we found that the physical position of cells within the colony and epigenetic gene expression inheritance from mother to daughter cells significantly contributed to cellular heterogeneity. In contrast, cell pole age and cellular lifespan had no effect. Regarding functions, our results indicate that cells optimize their siderophore investment strategies (pyoverdine vs. pyochelin) along a gradient from the centre to the edge of the colony. Moreover, we found evidence that cell lineages with above-average siderophore investment increase the fitness of cell lineages with below-average investment through cooperative sharing of secreted siderophores. Altogether, our study highlights that single-cell experiments combined with automated image and cell-tracking analyses can identify sources of heterogeneity and yield adaptive explanations for gene expression variation among clonal bacterial cells.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
