<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Antiviral efficacy of the SARS-CoV-2 XBB breakthrough infection sera against Omicron subvariants including EG.5
Authors: Kaku, Y.; Kosugi, Y.; Uriu, K.; Ito, J.; Kuramochi, J.; Sadamasu, K.; Yoshimura, K.; Asakura, H.; Nagashima, M.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Sato, K.
Score: 940.4, Published: 2023-08-08 DOI: 10.1101/2023.08.08.552415
As of July 2023, EG.5.1 (a.k.a. XBB.1.9.2.5.1), a XBB subvariant bearing the S:Q52H and S:F456L substitutions, alongside the S:F486P substitution (Figure S1A), has rapidly spread in some countries.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Antiviral efficacy of the SARS-CoV-2 XBB breakthrough infection sera against Omicron subvariants including EG.5
Authors: Kaku, Y.; Kosugi, Y.; Uriu, K.; Ito, J.; Kuramochi, J.; Sadamasu, K.; Yoshimura, K.; Asakura, H.; Nagashima, M.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Sato, K.
Score: 940.4, Published: 2023-08-08 DOI: 10.1101/2023.08.08.552415
As of July 2023, EG.5.1 (a.k.a. XBB.1.9.2.5.1), a XBB subvariant bearing the S:Q52H and S:F456L substitutions, alongside the S:F486P substitution (Figure S1A), has rapidly spread in some countries." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-13T10:36:01+00:00" />
<meta property="article:modified_time" content="2023-08-13T10:36:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Antiviral efficacy of the SARS-CoV-2 XBB breakthrough infection sera against Omicron subvariants including EG.5
Authors: Kaku, Y.; Kosugi, Y.; Uriu, K.; Ito, J.; Kuramochi, J.; Sadamasu, K.; Yoshimura, K.; Asakura, H.; Nagashima, M.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Sato, K.
Score: 940.4, Published: 2023-08-08 DOI: 10.1101/2023.08.08.552415
As of July 2023, EG.5.1 (a.k.a. XBB.1.9.2.5.1), a XBB subvariant bearing the S:Q52H and S:F456L substitutions, alongside the S:F486P substitution (Figure S1A), has rapidly spread in some countries."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Antiviral efficacy of the SARS-CoV-2 XBB breakthrough infection sera against Omicron subvariants including EG.5\nAuthors: Kaku, Y.; Kosugi, Y.; Uriu, K.; Ito, J.; Kuramochi, J.; Sadamasu, K.; Yoshimura, K.; Asakura, H.; Nagashima, M.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Sato, K.\nScore: 940.4, Published: 2023-08-08 DOI: 10.1101/2023.08.08.552415\nAs of July 2023, EG.5.1 (a.k.a. XBB.1.9.2.5.1), a XBB subvariant bearing the S:Q52H and S:F456L substitutions, alongside the S:F486P substitution (Figure S1A), has rapidly spread in some countries.",
  "keywords": [
    
  ],
  "articleBody": " Antiviral efficacy of the SARS-CoV-2 XBB breakthrough infection sera against Omicron subvariants including EG.5\nAuthors: Kaku, Y.; Kosugi, Y.; Uriu, K.; Ito, J.; Kuramochi, J.; Sadamasu, K.; Yoshimura, K.; Asakura, H.; Nagashima, M.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Sato, K.\nScore: 940.4, Published: 2023-08-08 DOI: 10.1101/2023.08.08.552415\nAs of July 2023, EG.5.1 (a.k.a. XBB.1.9.2.5.1), a XBB subvariant bearing the S:Q52H and S:F456L substitutions, alongside the S:F486P substitution (Figure S1A), has rapidly spread in some countries. On July 19, 2023, the WHO classified EG.5 as a variant under monitoring. First, we showed that EG.5.1 exhibits a higher effective reproduction number compared with XBB.1.5, XBB.1.16, and its parental lineage (XBB.1.9.2), suggesting that EG.5.1 will spread globally and outcompete these XBB subvariants in the near future. We then addressed whether EG.5.1 evades from the antiviral effect of the humoral immunity induced by breakthrough infection (BTI) of XBB subvariants and performed a neutralization assay using XBB BTI sera. However, the 50% neutralization titer (NT50) of XBB BTI sera against EG.5.1 was comparable to those against XBB.1.5/1.9.2 and XBB.1.16. Moreover, the sensitivity of EG.5.1 to convalescent sera of XBB.1- and XBB.1.5-infected hamsters was similar to those of XBB.1.5/1.9 and XBB.1.16. These results suggest that the increased Re of EG.5.1 is attributed to neither increased infectivity nor immune evasion from XBB BTI, and the emergence and spread of EG.5 is driven by the other pressures. We previously demonstrated that Omicron BTI cannot efficiently induce antiviral humoral immunity against the variant infected. In fact, the NT50s of the BTI sera of Omicron BA.1, BA.2, and BA.5 against the variant infected were 3.0-, 2.2-, and 3.4-fold lower than that against the ancestral B.1.1 variant, respectively. However, strikingly, we found that the NT50 of the BTI sera of XBB1.5/1.9 and XBB.1.16 against the variant infected were 8.7- and 8.3-fold lower than that against the B.1.1 variant. These results suggest that XBB BTI cannot efficiently induce antiviral humoral immunity against XBB subvariants. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=147 SRC=\"FIGDIR/small/552415v1_figs1.gif\" ALT=\"Figure 1\"\u003e View larger version (39K): org.highwire.dtl.DTLVardef@150feeforg.highwire.dtl.DTLVardef@9130e0org.highwire.dtl.DTLVardef@1fd6eeorg.highwire.dtl.DTLVardef@57ce67_HPS_FORMAT_FIGEXP M_FIG O_FLOATNOFigure S1.C_FLOATNO Virological features of EG.5.1 and XBB BTI (A) Frequency of mutations of interest in the representative XBB sublineages. Only mutations with a frequency \u003e0.5 in at least one but not all the representative sublineages are shown. Note that the S proteins of XBB.1.5 and XBB.1.9.2 are identical. (B) Estimated epidemic dynamics of the representative XBB sublineages in countries where \u003e50 sequences of EG.5.1, XBB.1.5, XBB.1.9.2, and XBB.1.16 were detected from April 1, 2023 to July 13, 2023. Countries are ordered according to the number of detected sequences of EG.5.1. Line, posterior mean; ribbon, 95% Bayesian confidence interval. The dynamics for EG.5.1 is highlighted by a red arrowhead. (C) Estimated relative Re of the representative XBB sublineages in the six countries. The relative Re of XBB.1.5 is set to 1 (horizontal dashed line). Violin, posterior distribution; dot, posterior mean; line, 95% Bayesian confidence interval. (D) Lentivirus-based pseudovirus assay. HOS-ACE2-TMPRSS2 cells were infected with pseudoviruses bearing each S protein. The amount of input virus was normalized to the amount of HIV-1 p24 capsid protein. The percentage infectivity of XBB.1.5/1.9.2, XBB.1.5/1.9.2+Q52H, XBB.1.5/1.9.2+F456L, and EG.5.1 compared to that of XBB.1.5/1.9.2 are shown. The horizontal dash line indicates the mean value of the percentage infectivity of the XBB.1.5/1.9.2. Assays were performed in quadruplicate. The presented data are expressed as the average {+/-} SD. Each dot indicates the result of an individual replicate. (E-G) Neutralization assay. Assays were performed with pseudoviruses harboring the S proteins of B.1.1, BA.1, BA.2, BA.5, BQ.1.1, XBB.1, XBB.1.5/1.9.2, XBB.1.16, EG.5.1, XBB.1.5/1.9.2+Q52H, and XBB.1.5/1.9.2+F456L. The following sera were used: convalescent sera from fully vaccinated individuals who had been infected with XBB.1.5 (one 3-dose vaccinated. 1 donor in total), XBB.1.9 (one 3-dose vaccinated donor, one 4-dose vaccinated donor and one 5-dose vaccinated donor. 3 donors in total), and XBB.1.16 (one 2-dose vaccinated donor, two 3-dose vaccinated donors, and one 4-dose vaccinated donor. 4 donors in total) (E); sera from hamster infected with XBB.1 (left) or XBB.1.5 (right) (F); and convalescent sera from fully vaccinated individuals who had been infected with BA.1 (thirteen 2-dose vaccinated, 13 donors in total) (left)1, BA.2 (nine 2-dose vaccinated and four 3-dose vaccinated donors. 13 donors in total) (middle)19, and BA.5 (one 2-dose vaccinated, thirteen 3-dose vaccinated donors, and one 4-dose vaccinated. 15 donors in total) (right)19 (G). Each dot indicates the result of an individual replicate. Assays for each serum sample were performed in triplicate to determine the 50% neutralization titer (NT50). Each dot represents one NT50 value, and the geometric mean and 95% confidence interval are shown. The number in parenthesis indicates the mean of NT50 values. The horizontal dash line indicates the detection limit (120-fold). In D, statistically significant differences (**, P \u003c 0.001, ***, P \u003c 0.0001) versus XBB.1.5/1.9.2 were determined by two-sided Students t tests. Blue asterisks indicate decreased percentage of infectivity. In E and G, statistically significant differences versus B.1.1 were determined by two-sided Wilcoxon signed-rank tests. The fold change between B.1.1 and the variant indicated is shown in red. Background information on the convalescent donors is summarized in Table S1. In F, statistically significant differences (*, P \u003c 0.01, **, P \u003c 0.001) between B.1.1 and XBB.1 (left) or XBB.1.5/1.9.2 (right) were determined by two-sided Wilcoxon signed-rank tests and indicated with asterisks. Red asterisks indicate decreased NT50s. C_FIG\nPhenotyping the virulence of SARS-CoV-2 variants in hamsters by digital pathology and machine learning\nAuthors: Meehan, G. R.; Herder, V.; Allan, J.; Huang, X.; Kerr, K.; Mendonca, D. C.; Ilia, G.; Wright, D.; Nomikou, K.; Gu, Q.; Arias, S. M.; De Lorenzo, G.; Cowton, V.; Upfold, N.; Palmalux, N.; Brown, J. C.; Barclay, W.; Filipe, A. D. S.; Furnon, W.; Patel, A.; Palmarini, M.\nScore: 919.4, Published: 2023-08-01 DOI: 10.1101/2023.08.01.551417\nSARS-CoV-2 has continued to evolve throughout the COVID-19 pandemic, giving rise to multiple variants of concern (VOCs) with different biological properties. As the pandemic progresses, it will be essential to test in near real time the potential of any new emerging variant to cause severe disease. BA.1 (Omicron) was shown to be attenuated compared to the previous VOCs like Delta, but it is possible that newly emerging variants may regain a virulent phenotype. Hamsters have been proven to be an exceedingly good model for SARS-CoV-2 pathogenesis. Here, we aimed to develop robust quantitative pipelines to assess the virulence of SARS-CoV-2 variants in hamsters. We used various approaches including RNAseq, RNA in situ hybridization, immunohistochemistry, and digital pathology, including software assisted whole section imaging and downstream automatic analyses enhanced by machine learning, to develop methods to assess and quantify virus-induced pulmonary lesions in an unbiased manner. Initially, we used Delta and Omicron to develop our experimental pipelines. We then assessed the virulence of recent Omicron sub-lineages including BA.5, XBB, BQ.1.18, BA.2 and BA.2.75. We show that in experimentally infected hamsters, accurate quantification of alveolar epithelial hyperplasia and macrophage infiltrates represent robust markers for assessing the extent of virus-induced pulmonary pathology, and hence virus virulence. In addition, using these pipelines, we could reveal how some Omicron sub-lineages (e.g., BA.2.75) have regained virulence compared to the original BA.1. Finally, to maximise the utility of the digital pathology pipelines reported in our study, we developed an online repository containing representative whole organ histopathology sections that can be visualised at variable magnifications (https://covid-atlas.cvr.gla.ac.uk). Overall, this pipeline can provide unbiased and invaluable data for rapidly assessing newly emerging variants and their potential to cause severe disease.\nEpigenetic liquid biopsies reveal elevated vascular endothelial cell turnover and erythropoiesis in asymptomatic COVID-19 patients\nAuthors: Ben Ami, R.; Loyfer, N.; Cohn, E.; Fialkoff, G.; Sharkia, I.; Bogot, N.; Kochan, D.; Kalak, G.; Jarjoui, A.; Chen-Shuali, C.; Azulai, H.; Barhoum, H.; Arish, N.; Greenberger, M. M.; Velleman, D.; Kurd, R.; Chetrit, E. B.; Bohm, D.; Wolak, T.; Quteineh, A.; Cann, G.; Glaser, B.; Friedman, N.; Kaplan, T.; Shemer, R.; Rokach, A.; Dor, Y.\nScore: 344.9, Published: 2023-08-01 DOI: 10.1101/2023.07.28.550957\nThe full spectrum of tissues affected by SARS-CoV-2 infection is crucial for deciphering the heterogenous clinical course of COVID-19. Here, we analyzed DNA methylation and histone modification patterns in circulating chromatin to assess cell type-specific turnover in severe and asymptomatic COVID-19 patients, in relation to clinical outcome. Patients with severe COVID-19 had a massive elevation of circulating cell-free DNA (cfDNA) levels, which originated in lung epithelial cells, cardiomyocytes, vascular endothelial cells and erythroblasts, suggesting increased cell death or turnover in these tissues. The immune response to infection was reflected by elevated B cell and monocyte/macrophage cfDNA levels, and by evidence of an interferon response in cells prior to cfDNA release. Strikingly, monocyte/macrophage cfDNA levels (but not monocyte counts), as well as lung epithelium cfDNA and vascular endothelial cfDNA, predicted clinical deterioration and duration of hospitalization. Asymptomatic patients had elevated levels of immune-derived cfDNA but did not show evidence of pulmonary or cardiac damage. Surprisingly, these patients showed elevated levels of vascular endothelial cell and erythroblast cfDNA, suggesting that sub-clinical vascular and erythrocyte turnover are universal features of COVID-19, independent of disease severity. Epigenetic liquid biopsies provide non-invasive means of monitoring COVID-19 patients, and reveal sub-clinical vascular damage and red blood cell turnover.\nAnalyses of cell wall synthesis in Clostridioides difficile reveal a diversification in cell division mechanisms in endospore-forming bacteria\nAuthors: Shrestha, S.; Taib, N.; Gribaldo, S.; Shen, A.\nScore: 31.7, Published: 2023-08-08 DOI: 10.1101/2023.08.06.552200\nCurrent models of bacterial cell division assume that the core synthases of the multiprotein divisome complex, FtsW-FtsI, are the primary drivers of septal peptidoglycan (PG) synthesis. These enzymes are typically encoded in the highly conserved division and cell wall (dcw) cluster and are considered to be universally essential for cell division. Here, we combine bioinformatics analyses with functional characterization in the pathogen Clostridioides difficile to show that dcw-encoded PG synthases have undergone a surprising specialization in the sole endospore-forming phylum, Firmicutes, to fulfill sporulation-specific roles. We describe a novel role for these enzymes in synthesizing septal PG during the sporulation-specific mode of cell division in C. difficile. Although these enzymes are directly regulated by canonical divisome components during this process, dcw-encoded PG synthases and their divisome regulators are unexpectedly dispensable for cell division during normal growth. Instead, C. difficile uses its sole bifunctional class A penicillin-binding protein (aPBP) to drive cell division, revealing a previously unreported role for this class of PG synthases as the core divisome enzyme. Collectively, our findings reveal how the emergence of endosporulation in the Firmicutes phylum was a key driver for the functional repurposing of an otherwise universally conserved cellular process such as cell division. Moreover, they indicate that C. difficile, and likely other clostridia, assemble a divisome that differs markedly from previously studied bacteria, thus representing an attractive, unique target for therapeutic purposes.\nInfant gut bacteriophage strain persistence during the first three years of life\nAuthors: Lou, Y. C.; Chen, L.; Borges, A. L.; West-Roberts, J. A.; Firek, B. A.; Morowitz, M.; Banfield, J.\nScore: 24.7, Published: 2023-08-08 DOI: 10.1101/2023.08.07.552389\nBacteriophages are key components of gut microbiomes, yet the phage colonization process in the infant gut remains uncertain. Here, we established a large phage sequence database and used strain-resolved analyses to investigate phage succession in infants throughout the first three years of life. Analysis of 819 fecal metagenomes collected from 28 full-term and 24 preterm infants and their mothers revealed that early-life phageome richness increased over time and reached adult-like complexity by age three. Approximately 9% of early phage colonizers, mostly maternally transmitted and infecting Bacteroides, persisted for three years and were more prevalent in full-term than in preterm infants. Although rare, phages with stop codon reassignment were more likely to persist than non-recoded phages and generally displayed an increase in in-frame re-assigned stop codons over three years. Overall, maternal seeding, stop codon reassignment, host CRISPR-Cas locus prevalence, and diverse phage populations contribute to stable viral colonization.\nExtracellular vesicles are the main contributor to the non-viral protected extracellular sequence space\nAuthors: Luecking, D.; Mercier, C.; Alarcon-Schumacher, T.; Erdmann, S.\nScore: 17.7, Published: 2023-08-07 DOI: 10.1101/2023.08.07.552246\nMarine environmental virus metagenomes, commonly referred to as viromes, are typically generated by physically separating virus-like particles (VLPs) from the microbial fraction based on their size and mass. However, most methods used to purify VLPs, enrich extracellular vesicles (EVs) and gene transfer agents (GTAs) simultaneously. Consequently, the sequence space traditionally referred to as a virome contains host-associated sequences, transported via EVs or GTAs. We therefore propose to call the genetic material isolated from size-fractionated (0.22 {micro}m) and DNase-treated samples protected environmental DNA (peDNA). This sequence space contains viral genomes, DNA transduced by viruses and DNA transported in EVs and GTAs. Since there is no genetic signature for peDNA transported in EVs, GTAs and virus particles, we rely on the successful removal of contaminating remaining cellular and free DNA when analyzing peDNA. Using marine samples collected from the North Sea, we generated a thoroughly purified peDNA dataset and developed a bioinformatic pipeline to determine the potential origin of the purified DNA. This pipeline was applied to our dataset as well as existing global marine viromes. Through this pipeline, we identified known GTA and EV producers, as well as organisms with actively transducing proviruses as the source of the peDNA, thus confirming the reliability of our approach. Additionally, we identified novel and widespread EV producers, and found quantitative evidence suggesting that EV-mediated gene transfer plays a significant role in driving horizontal gene transfer (HGT) in the worlds oceans.\nEco-evolutionary dynamics of massive, parallel bacteriophage outbreaks in compost communities\nAuthors: Meijer, J.; Skiadas, P.; Rainey, P. B.; Hogeweg, P.; Dutilh, B. E.\nScore: 22.9, Published: 2023-07-31 DOI: 10.1101/2023.07.31.550844\nBacteriophages are important drivers of microbial ecosystems, but their influence and dynamics in terrestrial biomes remain poorly understood compared to aquatic and host-associated systems. To investigate this, we analyzed shotgun metagenomics datasets from ten compost-derived microbial communities propagated over 48 weeks. We found that the communities clustered into two distinct types consisting of hundreds of microbial genera, and in one community type identified Theomophage, a lytic bacteriophage representing a new Schitoviridae subfamily, which accounted for up to 74.3% of the total community metagenome, indicating massive viral outbreaks. We tracked molecular evolution of Theomophage and found that isolated communities were dominated by a single strain that showed little molecular evolution during outbreaks. However, when experimental manipulation allowed phages to migrate between communities, we observed transient coexistence of strains followed by genomic recombination that underpinned replacement of the ancestral strains. Additionally, when Theomophage colonized mesocosms where it was originally absent, new mutations evolved that fixed and spread to other communities. Our study describes the largest bacteriophage outbreak reported to date and reveals the spatial and temporal scales at which terrestrial bacteriophage microdiversity evolves. It also demonstrates that mixing of viral communities, which may be frequent in natural systems, promotes rapid bacteriophage evolution. Significance StatementTerrestrial viral ecology and evolution is an active research area, but current knowledge on soil viruses still lags behind that of other biomes, and the terrestrial microbiome harbors untapped viral diversity. This study describes parallel, massive outbreaks of a novel bacteriophage in a 48- week evolution experiment with compost-derived microbial communities. The unprecedented abundance of this bacteriophage highlights the importance of accounting for viral sequences - which may be challenging to identify and annotate - when analyzing community sequence data. Moreover, we show how dispersal accelerates the dynamics of molecular evolution on a timescale of weeks. These results advance understanding of the spatiotemporal scales at which bacteriophage eco-evolutionary dynamics play out in terrestrial biomes.\nSARS-CoV-2 infection of human pluripotent stem cell-derived vascular cells reveals smooth muscle cells as key mediators of vascular pathology during infection\nAuthors: Richards, A.; Khalil, A.; Friesen, M.; Whitfield, T.; Lungjangwa, T.; Gehrke, L.; Mooney, D.; Jaenisch, R.\nScore: 15.6, Published: 2023-08-07 DOI: 10.1101/2023.08.06.552160\nAlthough respiratory symptoms are the most prevalent disease manifestation of infection by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), nearly 20% of hospitalized patients are at risk for thromboembolic events1. This prothrombotic state is considered a key factor in the increased risk of stroke, which has been observed clinically during both acute infection and long after symptoms have cleared2. Here we developed a model of SARS-CoV-2 infection using human-induced pluripotent stem cell-derived endothelial cells, pericytes, and smooth muscle cells to recapitulate the vascular pathology associated with SARS-CoV-2 exposure. Our results demonstrate that perivascular cells, particularly smooth muscle cells (SMCs), are a specifically susceptible vascular target for SARS-CoV-2 infection. Utilizing RNA sequencing, we characterized the transcriptomic changes accompanying SARS-CoV-2 infection of SMCs, and endothelial cells (ECs). We observed that infected human SMCs shift to a pro-inflammatory state and increase the expression of key mediators of the coagulation cascade. Further, we showed human ECs exposed to the secretome of infected SMCs produce hemostatic factors that can contribute to vascular dysfunction, despite not being susceptible to direct infection. The findings here recapitulate observations from patient sera in human COVID-19 patients and provide mechanistic insight into the unique vascular implications of SARS-CoV-2 infection at a cellular level.\nToxinome - The Bacterial Protein Toxin Database\nAuthors: Danov, A.; Segev, O.; Bograd, A.; Ben Eliyahu, Y.; Dotan, N.; Kaplan, T.; Levy, A.\nScore: 13.1, Published: 2023-08-12 DOI: 10.1101/2023.08.12.553073\nProtein toxins are key molecular weapons in biology that are used to attack neighboring cells. Bacteria use protein toxins to kill or inhibit growth of prokaryotic and eukaryotic cells using various modes of action that target essential cellular components. The toxins are responsible for shaping microbiomes in different habitats, for abortive phage infection, and for severe infectious diseases of animals and plants. Although several toxin databases have been developed, each one is devoted to a specific toxin family and they encompass a relatively small number of toxins. Antimicrobial toxins are often accompanied by antitoxins (or immunity proteins) that neutralize the cognate toxins. Here, we combined toxins and antitoxins from many resources and created Toxinome, a comprehensive and updated bacterial protein toxin database. Toxinome includes a total of 1,483,028 toxins and 491,345 antitoxins encoded in 59,475 bacterial genomes across the tree of life. We identified a depletion of toxin and antitoxin genes in bacteria that dwell in extreme temperatures. We defined 5,161 unique Toxin Islands within phylogenetically diverse bacterial genomes, which are loci dense in toxin and antitoxin genes. By focusing on the unannotated genes within these islands, we characterized a number of these genes as toxins or antitoxins. Finally, we developed an interactive Toxinome website (http://toxinome.pythonanywhere.com) that allows searching and downloading of our database. The Toxinome resource will be useful to the large research community interested in bacterial toxins and can guide toxin discovery and function elucidation, and infectious disease diagnosis and treatment.\nA bacterial sialidase mediates early life colonization by a pioneering gut commensal\nAuthors: Buzun, E.; Hsu, C.-Y.; Sejane, K.; Oles, R. E.; Ayala, A. V.; Loomis, L. R.; Zhao, J.; Rossitto, L.-A.; McGrosso, D.; Gonzalez, D. J.; Bode, L.; Chu, H.\nScore: 12.8, Published: 2023-08-09 DOI: 10.1101/2023.08.08.552477\nThe early microbial colonization of the gastrointestinal tract can lead to long-term impacts in development and overall human health. Keystone species, including Bacteroides spp., play a crucial role in maintaining the structure, diversity, and function of the intestinal ecosystem. However, the process by which a defined and resilient community is curated and maintained during early life remains inadequately understood. Here, we show that a single sialidase, NanH, in Bacteroides fragilis mediates stable occupancy of the intestinal mucosa and regulates the commensal colonization program during the first weeks of life. This program is triggered by sialylated glycans, including those found in human milk oligosaccharides and intestinal mucus. After examining the dynamics between pioneer gut Bacteroides species in the murine gut, we discovered that NanH enables vertical transmission from dams to pups and promotes B. fragilis dominance during early life. Furthermore, we demonstrate that NanH facilitates commensal resilience and recovery after antibiotic treatment in a defined microbial community. Collectively, our study reveals a co-evolutionary mechanism between the host and the microbiota mediated through host-derived glycans to promote stable intestinal colonization.\n",
  "wordCount" : "3314",
  "inLanguage": "en",
  "datePublished": "2023-08-13T10:36:01Z",
  "dateModified": "2023-08-13T10:36:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta"><span>updated on August 13, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552415">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552415" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552415">
        <p class="paperTitle">Antiviral efficacy of the SARS-CoV-2 XBB breakthrough infection sera against Omicron subvariants including EG.5</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552415" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552415" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kaku, Y.; Kosugi, Y.; Uriu, K.; Ito, J.; Kuramochi, J.; Sadamasu, K.; Yoshimura, K.; Asakura, H.; Nagashima, M.; The Genotype to Phenotype Japan (G2P-Japan) Consortium,  ; Sato, K.</p>
        <p class="info">Score: 940.4, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552415' target='https://doi.org/10.1101/2023.08.08.552415'> 10.1101/2023.08.08.552415</a></p>
        <p class="abstract">As of July 2023, EG.5.1 (a.k.a. XBB.1.9.2.5.1), a XBB subvariant bearing the S:Q52H and S:F456L substitutions, alongside the S:F486P substitution (Figure S1A), has rapidly spread in some countries. On July 19, 2023, the WHO classified EG.5 as a variant under monitoring. First, we showed that EG.5.1 exhibits a higher effective reproduction number compared with XBB.1.5, XBB.1.16, and its parental lineage (XBB.1.9.2), suggesting that EG.5.1 will spread globally and outcompete these XBB subvariants in the near future. We then addressed whether EG.5.1 evades from the antiviral effect of the humoral immunity induced by breakthrough infection (BTI) of XBB subvariants and performed a neutralization assay using XBB BTI sera. However, the 50% neutralization titer (NT50) of XBB BTI sera against EG.5.1 was comparable to those against XBB.1.5/1.9.2 and XBB.1.16. Moreover, the sensitivity of EG.5.1 to convalescent sera of XBB.1- and XBB.1.5-infected hamsters was similar to those of XBB.1.5/1.9 and XBB.1.16. These results suggest that the increased Re of EG.5.1 is attributed to neither increased infectivity nor immune evasion from XBB BTI, and the emergence and spread of EG.5 is driven by the other pressures. We previously demonstrated that Omicron BTI cannot efficiently induce antiviral humoral immunity against the variant infected. In fact, the NT50s of the BTI sera of Omicron BA.1, BA.2, and BA.5 against the variant infected were 3.0-, 2.2-, and 3.4-fold lower than that against the ancestral B.1.1 variant, respectively. However, strikingly, we found that the NT50 of the BTI sera of XBB1.5/1.9 and XBB.1.16 against the variant infected were 8.7- and 8.3-fold lower than that against the B.1.1 variant. These results suggest that XBB BTI cannot efficiently induce antiviral humoral immunity against XBB subvariants.

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=147 SRC=&#34;FIGDIR/small/552415v1_figs1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (39K):
org.highwire.dtl.DTLVardef@150feeforg.highwire.dtl.DTLVardef@9130e0org.highwire.dtl.DTLVardef@1fd6eeorg.highwire.dtl.DTLVardef@57ce67_HPS_FORMAT_FIGEXP  M_FIG O_FLOATNOFigure S1.C_FLOATNO Virological features of EG.5.1 and XBB BTI (A) Frequency of mutations of interest in the representative XBB sublineages. Only mutations with a frequency &gt;0.5 in at least one but not all the representative sublineages are shown. Note that the S proteins of XBB.1.5 and XBB.1.9.2 are identical.

(B) Estimated epidemic dynamics of the representative XBB sublineages in countries where &gt;50 sequences of EG.5.1, XBB.1.5, XBB.1.9.2, and XBB.1.16 were detected from April 1, 2023 to July 13, 2023. Countries are ordered according to the number of detected sequences of EG.5.1. Line, posterior mean; ribbon, 95% Bayesian confidence interval. The dynamics for EG.5.1 is highlighted by a red arrowhead.

(C) Estimated relative Re of the representative XBB sublineages in the six countries. The relative Re of XBB.1.5 is set to 1 (horizontal dashed line). Violin, posterior distribution; dot, posterior mean; line, 95% Bayesian confidence interval.

(D) Lentivirus-based pseudovirus assay. HOS-ACE2-TMPRSS2 cells were infected with pseudoviruses bearing each S protein. The amount of input virus was normalized to the amount of HIV-1 p24 capsid protein. The percentage infectivity of XBB.1.5/1.9.2, XBB.1.5/1.9.2&#43;Q52H, XBB.1.5/1.9.2&#43;F456L, and EG.5.1 compared to that of XBB.1.5/1.9.2 are shown. The horizontal dash line indicates the mean value of the percentage infectivity of the XBB.1.5/1.9.2. Assays were performed in quadruplicate. The presented data are expressed as the average {&#43;/-} SD. Each dot indicates the result of an individual replicate.

(E-G) Neutralization assay. Assays were performed with pseudoviruses harboring the S proteins of B.1.1, BA.1, BA.2, BA.5, BQ.1.1, XBB.1, XBB.1.5/1.9.2, XBB.1.16, EG.5.1, XBB.1.5/1.9.2&#43;Q52H, and XBB.1.5/1.9.2&#43;F456L. The following sera were used: convalescent sera from fully vaccinated individuals who had been infected with XBB.1.5 (one 3-dose vaccinated. 1 donor in total), XBB.1.9 (one 3-dose vaccinated donor, one 4-dose vaccinated donor and one 5-dose vaccinated donor. 3 donors in total), and XBB.1.16 (one 2-dose vaccinated donor, two 3-dose vaccinated donors, and one 4-dose vaccinated donor. 4 donors in total) (E); sera from hamster infected with XBB.1 (left) or XBB.1.5 (right) (F); and convalescent sera from fully vaccinated individuals who had been infected with BA.1 (thirteen 2-dose vaccinated, 13 donors in total) (left)1, BA.2 (nine 2-dose vaccinated and four 3-dose vaccinated donors. 13 donors in total) (middle)19, and BA.5 (one 2-dose vaccinated, thirteen 3-dose vaccinated donors, and one 4-dose vaccinated. 15 donors in total) (right)19 (G). Each dot indicates the result of an individual replicate. Assays for each serum sample were performed in triplicate to determine the 50% neutralization titer (NT50). Each dot represents one NT50 value, and the geometric mean and 95% confidence interval are shown. The number in parenthesis indicates the mean of NT50 values. The horizontal dash line indicates the detection limit (120-fold).

In D, statistically significant differences (**, P &lt; 0.001, ***, P &lt; 0.0001) versus XBB.1.5/1.9.2 were determined by two-sided Students t tests. Blue asterisks indicate decreased percentage of infectivity.

In E and G, statistically significant differences versus B.1.1 were determined by two-sided Wilcoxon signed-rank tests. The fold change between B.1.1 and the variant indicated is shown in red. Background information on the convalescent donors is summarized in Table S1.

In F, statistically significant differences (*, P &lt; 0.01, **, P &lt; 0.001) between B.1.1 and XBB.1 (left) or XBB.1.5/1.9.2 (right) were determined by two-sided Wilcoxon signed-rank tests and indicated with asterisks. Red asterisks indicate decreased NT50s.

C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.551417">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.551417" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.551417">
        <p class="paperTitle">Phenotyping the virulence of SARS-CoV-2 variants in hamsters by digital pathology and machine learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.551417" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.551417" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Meehan, G. R.; Herder, V.; Allan, J.; Huang, X.; Kerr, K.; Mendonca, D. C.; Ilia, G.; Wright, D.; Nomikou, K.; Gu, Q.; Arias, S. M.; De Lorenzo, G.; Cowton, V.; Upfold, N.; Palmalux, N.; Brown, J. C.; Barclay, W.; Filipe, A. D. S.; Furnon, W.; Patel, A.; Palmarini, M.</p>
        <p class="info">Score: 919.4, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.551417' target='https://doi.org/10.1101/2023.08.01.551417'> 10.1101/2023.08.01.551417</a></p>
        <p class="abstract">SARS-CoV-2 has continued to evolve throughout the COVID-19 pandemic, giving rise to multiple variants of concern (VOCs) with different biological properties. As the pandemic progresses, it will be essential to test in near real time the potential of any new emerging variant to cause severe disease. BA.1 (Omicron) was shown to be attenuated compared to the previous VOCs like Delta, but it is possible that newly emerging variants may regain a virulent phenotype. Hamsters have been proven to be an exceedingly good model for SARS-CoV-2 pathogenesis. Here, we aimed to develop robust quantitative pipelines to assess the virulence of SARS-CoV-2 variants in hamsters. We used various approaches including RNAseq, RNA in situ hybridization, immunohistochemistry, and digital pathology, including software assisted whole section imaging and downstream automatic analyses enhanced by machine learning, to develop methods to assess and quantify virus-induced pulmonary lesions in an unbiased manner. Initially, we used Delta and Omicron to develop our experimental pipelines. We then assessed the virulence of recent Omicron sub-lineages including BA.5, XBB, BQ.1.18, BA.2 and BA.2.75. We show that in experimentally infected hamsters, accurate quantification of alveolar epithelial hyperplasia and macrophage infiltrates represent robust markers for assessing the extent of virus-induced pulmonary pathology, and hence virus virulence. In addition, using these pipelines, we could reveal how some Omicron sub-lineages (e.g., BA.2.75) have regained virulence compared to the original BA.1. Finally, to maximise the utility of the digital pathology pipelines reported in our study, we developed an online repository containing representative whole organ histopathology sections that can be visualised at variable magnifications (https://covid-atlas.cvr.gla.ac.uk). Overall, this pipeline can provide unbiased and invaluable data for rapidly assessing newly emerging variants and their potential to cause severe disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.550957">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.550957" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.550957">
        <p class="paperTitle">Epigenetic liquid biopsies reveal elevated vascular endothelial cell turnover and erythropoiesis in asymptomatic COVID-19 patients</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.550957" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.550957" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ben Ami, R.; Loyfer, N.; Cohn, E.; Fialkoff, G.; Sharkia, I.; Bogot, N.; Kochan, D.; Kalak, G.; Jarjoui, A.; Chen-Shuali, C.; Azulai, H.; Barhoum, H.; Arish, N.; Greenberger, M. M.; Velleman, D.; Kurd, R.; Chetrit, E. B.; Bohm, D.; Wolak, T.; Quteineh, A.; Cann, G.; Glaser, B.; Friedman, N.; Kaplan, T.; Shemer, R.; Rokach, A.; Dor, Y.</p>
        <p class="info">Score: 344.9, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.550957' target='https://doi.org/10.1101/2023.07.28.550957'> 10.1101/2023.07.28.550957</a></p>
        <p class="abstract">The full spectrum of tissues affected by SARS-CoV-2 infection is crucial for deciphering the heterogenous clinical course of COVID-19. Here, we analyzed DNA methylation and histone modification patterns in circulating chromatin to assess cell type-specific turnover in severe and asymptomatic COVID-19 patients, in relation to clinical outcome. Patients with severe COVID-19 had a massive elevation of circulating cell-free DNA (cfDNA) levels, which originated in lung epithelial cells, cardiomyocytes, vascular endothelial cells and erythroblasts, suggesting increased cell death or turnover in these tissues. The immune response to infection was reflected by elevated B cell and monocyte/macrophage cfDNA levels, and by evidence of an interferon response in cells prior to cfDNA release. Strikingly, monocyte/macrophage cfDNA levels (but not monocyte counts), as well as lung epithelium cfDNA and vascular endothelial cfDNA, predicted clinical deterioration and duration of hospitalization. Asymptomatic patients had elevated levels of immune-derived cfDNA but did not show evidence of pulmonary or cardiac damage. Surprisingly, these patients showed elevated levels of vascular endothelial cell and erythroblast cfDNA, suggesting that sub-clinical vascular and erythrocyte turnover are universal features of COVID-19, independent of disease severity. Epigenetic liquid biopsies provide non-invasive means of monitoring COVID-19 patients, and reveal sub-clinical vascular damage and red blood cell turnover.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.06.552200">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.06.552200" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.06.552200">
        <p class="paperTitle">Analyses of cell wall synthesis in Clostridioides difficile reveal a diversification in cell division mechanisms in endospore-forming bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.06.552200" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.06.552200" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shrestha, S.; Taib, N.; Gribaldo, S.; Shen, A.</p>
        <p class="info">Score: 31.7, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.06.552200' target='https://doi.org/10.1101/2023.08.06.552200'> 10.1101/2023.08.06.552200</a></p>
        <p class="abstract">Current models of bacterial cell division assume that the core synthases of the multiprotein divisome complex, FtsW-FtsI, are the primary drivers of septal peptidoglycan (PG) synthesis. These enzymes are typically encoded in the highly conserved division and cell wall (dcw) cluster and are considered to be universally essential for cell division. Here, we combine bioinformatics analyses with functional characterization in the pathogen Clostridioides difficile to show that dcw-encoded PG synthases have undergone a surprising specialization in the sole endospore-forming phylum, Firmicutes, to fulfill sporulation-specific roles. We describe a novel role for these enzymes in synthesizing septal PG during the sporulation-specific mode of cell division in C. difficile. Although these enzymes are directly regulated by canonical divisome components during this process, dcw-encoded PG synthases and their divisome regulators are unexpectedly dispensable for cell division during normal growth. Instead, C. difficile uses its sole bifunctional class A penicillin-binding protein (aPBP) to drive cell division, revealing a previously unreported role for this class of PG synthases as the core divisome enzyme. Collectively, our findings reveal how the emergence of endosporulation in the Firmicutes phylum was a key driver for the functional repurposing of an otherwise universally conserved cellular process such as cell division. Moreover, they indicate that C. difficile, and likely other clostridia, assemble a divisome that differs markedly from previously studied bacteria, thus representing an attractive, unique target for therapeutic purposes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552389">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552389" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552389">
        <p class="paperTitle">Infant gut bacteriophage strain persistence during the first three years of life</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552389" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552389" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lou, Y. C.; Chen, L.; Borges, A. L.; West-Roberts, J. A.; Firek, B. A.; Morowitz, M.; Banfield, J.</p>
        <p class="info">Score: 24.7, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552389' target='https://doi.org/10.1101/2023.08.07.552389'> 10.1101/2023.08.07.552389</a></p>
        <p class="abstract">Bacteriophages are key components of gut microbiomes, yet the phage colonization process in the infant gut remains uncertain. Here, we established a large phage sequence database and used strain-resolved analyses to investigate phage succession in infants throughout the first three years of life. Analysis of 819 fecal metagenomes collected from 28 full-term and 24 preterm infants and their mothers revealed that early-life phageome richness increased over time and reached adult-like complexity by age three. Approximately 9% of early phage colonizers, mostly maternally transmitted and infecting Bacteroides, persisted for three years and were more prevalent in full-term than in preterm infants. Although rare, phages with stop codon reassignment were more likely to persist than non-recoded phages and generally displayed an increase in in-frame re-assigned stop codons over three years. Overall, maternal seeding, stop codon reassignment, host CRISPR-Cas locus prevalence, and diverse phage populations contribute to stable viral colonization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552246">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552246" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552246">
        <p class="paperTitle">Extracellular vesicles are the main contributor to the non-viral protected extracellular sequence space</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552246" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552246" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Luecking, D.; Mercier, C.; Alarcon-Schumacher, T.; Erdmann, S.</p>
        <p class="info">Score: 17.7, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552246' target='https://doi.org/10.1101/2023.08.07.552246'> 10.1101/2023.08.07.552246</a></p>
        <p class="abstract">Marine environmental virus metagenomes, commonly referred to as viromes, are typically generated by physically separating virus-like particles (VLPs) from the microbial fraction based on their size and mass. However, most methods used to purify VLPs, enrich extracellular vesicles (EVs) and gene transfer agents (GTAs) simultaneously. Consequently, the sequence space traditionally referred to as a virome contains host-associated sequences, transported via EVs or GTAs. We therefore propose to call the genetic material isolated from size-fractionated (0.22 {micro}m) and DNase-treated samples protected environmental DNA (peDNA). This sequence space contains viral genomes, DNA transduced by viruses and DNA transported in EVs and GTAs. Since there is no genetic signature for peDNA transported in EVs, GTAs and virus particles, we rely on the successful removal of contaminating remaining cellular and free DNA when analyzing peDNA. Using marine samples collected from the North Sea, we generated a thoroughly purified peDNA dataset and developed a bioinformatic pipeline to determine the potential origin of the purified DNA. This pipeline was applied to our dataset as well as existing global marine viromes. Through this pipeline, we identified known GTA and EV producers, as well as organisms with actively transducing proviruses as the source of the peDNA, thus confirming the reliability of our approach. Additionally, we identified novel and widespread EV producers, and found quantitative evidence suggesting that EV-mediated gene transfer plays a significant role in driving horizontal gene transfer (HGT) in the worlds oceans.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.550844">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.550844" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.550844">
        <p class="paperTitle">Eco-evolutionary dynamics of massive, parallel bacteriophage outbreaks in compost communities</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.550844" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.550844" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Meijer, J.; Skiadas, P.; Rainey, P. B.; Hogeweg, P.; Dutilh, B. E.</p>
        <p class="info">Score: 22.9, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.550844' target='https://doi.org/10.1101/2023.07.31.550844'> 10.1101/2023.07.31.550844</a></p>
        <p class="abstract">Bacteriophages are important drivers of microbial ecosystems, but their influence and dynamics in terrestrial biomes remain poorly understood compared to aquatic and host-associated systems. To investigate this, we analyzed shotgun metagenomics datasets from ten compost-derived microbial communities propagated over 48 weeks. We found that the communities clustered into two distinct types consisting of hundreds of microbial genera, and in one community type identified Theomophage, a lytic bacteriophage representing a new Schitoviridae subfamily, which accounted for up to 74.3% of the total community metagenome, indicating massive viral outbreaks. We tracked molecular evolution of Theomophage and found that isolated communities were dominated by a single strain that showed little molecular evolution during outbreaks. However, when experimental manipulation allowed phages to migrate between communities, we observed transient coexistence of strains followed by genomic recombination that underpinned replacement of the ancestral strains. Additionally, when Theomophage colonized mesocosms where it was originally absent, new mutations evolved that fixed and spread to other communities. Our study describes the largest bacteriophage outbreak reported to date and reveals the spatial and temporal scales at which terrestrial bacteriophage microdiversity evolves. It also demonstrates that mixing of viral communities, which may be frequent in natural systems, promotes rapid bacteriophage evolution.

Significance StatementTerrestrial viral ecology and evolution is an active research area, but current knowledge on soil viruses still lags behind that of other biomes, and the terrestrial microbiome harbors untapped viral diversity. This study describes parallel, massive outbreaks of a novel bacteriophage in a 48- week evolution experiment with compost-derived microbial communities. The unprecedented abundance of this bacteriophage highlights the importance of accounting for viral sequences - which may be challenging to identify and annotate - when analyzing community sequence data. Moreover, we show how dispersal accelerates the dynamics of molecular evolution on a timescale of weeks. These results advance understanding of the spatiotemporal scales at which bacteriophage eco-evolutionary dynamics play out in terrestrial biomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.06.552160">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.06.552160" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.06.552160">
        <p class="paperTitle">SARS-CoV-2 infection of human pluripotent stem cell-derived vascular cells reveals smooth muscle cells as key mediators of vascular pathology during infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.06.552160" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.06.552160" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Richards, A.; Khalil, A.; Friesen, M.; Whitfield, T.; Lungjangwa, T.; Gehrke, L.; Mooney, D.; Jaenisch, R.</p>
        <p class="info">Score: 15.6, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.06.552160' target='https://doi.org/10.1101/2023.08.06.552160'> 10.1101/2023.08.06.552160</a></p>
        <p class="abstract">Although respiratory symptoms are the most prevalent disease manifestation of infection by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), nearly 20% of hospitalized patients are at risk for thromboembolic events1. This prothrombotic state is considered a key factor in the increased risk of stroke, which has been observed clinically during both acute infection and long after symptoms have cleared2. Here we developed a model of SARS-CoV-2 infection using human-induced pluripotent stem cell-derived endothelial cells, pericytes, and smooth muscle cells to recapitulate the vascular pathology associated with SARS-CoV-2 exposure. Our results demonstrate that perivascular cells, particularly smooth muscle cells (SMCs), are a specifically susceptible vascular target for SARS-CoV-2 infection. Utilizing RNA sequencing, we characterized the transcriptomic changes accompanying SARS-CoV-2 infection of SMCs, and endothelial cells (ECs). We observed that infected human SMCs shift to a pro-inflammatory state and increase the expression of key mediators of the coagulation cascade. Further, we showed human ECs exposed to the secretome of infected SMCs produce hemostatic factors that can contribute to vascular dysfunction, despite not being susceptible to direct infection. The findings here recapitulate observations from patient sera in human COVID-19 patients and provide mechanistic insight into the unique vascular implications of SARS-CoV-2 infection at a cellular level.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.12.553073">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.12.553073" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.12.553073">
        <p class="paperTitle">Toxinome - The Bacterial Protein Toxin Database</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.12.553073" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.12.553073" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Danov, A.; Segev, O.; Bograd, A.; Ben Eliyahu, Y.; Dotan, N.; Kaplan, T.; Levy, A.</p>
        <p class="info">Score: 13.1, Published: 2023-08-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.12.553073' target='https://doi.org/10.1101/2023.08.12.553073'> 10.1101/2023.08.12.553073</a></p>
        <p class="abstract">Protein toxins are key molecular weapons in biology that are used to attack neighboring cells. Bacteria use protein toxins to kill or inhibit growth of prokaryotic and eukaryotic cells using various modes of action that target essential cellular components. The toxins are responsible for shaping microbiomes in different habitats, for abortive phage infection, and for severe infectious diseases of animals and plants. Although several toxin databases have been developed, each one is devoted to a specific toxin family and they encompass a relatively small number of toxins. Antimicrobial toxins are often accompanied by antitoxins (or immunity proteins) that neutralize the cognate toxins. Here, we combined toxins and antitoxins from many resources and created Toxinome, a comprehensive and updated bacterial protein toxin database. Toxinome includes a total of 1,483,028 toxins and 491,345 antitoxins encoded in 59,475 bacterial genomes across the tree of life. We identified a depletion of toxin and antitoxin genes in bacteria that dwell in extreme temperatures. We defined 5,161 unique Toxin Islands within phylogenetically diverse bacterial genomes, which are loci dense in toxin and antitoxin genes. By focusing on the unannotated genes within these islands, we characterized a number of these genes as toxins or antitoxins. Finally, we developed an interactive Toxinome website (http://toxinome.pythonanywhere.com) that allows searching and downloading of our database. The Toxinome resource will be useful to the large research community interested in bacterial toxins and can guide toxin discovery and function elucidation, and infectious disease diagnosis and treatment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552477">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552477" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552477">
        <p class="paperTitle">A bacterial sialidase mediates early life colonization by a pioneering gut commensal</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552477" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552477" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Buzun, E.; Hsu, C.-Y.; Sejane, K.; Oles, R. E.; Ayala, A. V.; Loomis, L. R.; Zhao, J.; Rossitto, L.-A.; McGrosso, D.; Gonzalez, D. J.; Bode, L.; Chu, H.</p>
        <p class="info">Score: 12.8, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552477' target='https://doi.org/10.1101/2023.08.08.552477'> 10.1101/2023.08.08.552477</a></p>
        <p class="abstract">The early microbial colonization of the gastrointestinal tract can lead to long-term impacts in development and overall human health. Keystone species, including Bacteroides spp., play a crucial role in maintaining the structure, diversity, and function of the intestinal ecosystem. However, the process by which a defined and resilient community is curated and maintained during early life remains inadequately understood. Here, we show that a single sialidase, NanH, in Bacteroides fragilis mediates stable occupancy of the intestinal mucosa and regulates the commensal colonization program during the first weeks of life. This program is triggered by sialylated glycans, including those found in human milk oligosaccharides and intestinal mucus. After examining the dynamics between pioneer gut Bacteroides species in the murine gut, we discovered that NanH enables vertical transmission from dams to pups and promotes B. fragilis dominance during early life. Furthermore, we demonstrate that NanH facilitates commensal resilience and recovery after antibiotic treatment in a defined microbial community. Collectively, our study reveals a co-evolutionary mechanism between the host and the microbiota mediated through host-derived glycans to promote stable intestinal colonization.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
