<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Amazing structural diversity of giant virus-like particles in forest soil
Authors: Fischer, M. G.; Mersdorf, U.; Blanchard, J. L.
Score: 118.6, Published: 2023-06-30 DOI: 10.1101/2023.06.30.546935
Large DNA viruses of the phylum Nucleocytoviricota infect diverse eukaryotic hosts from protists to humans, with profound consequences for aquatic and terrestrial ecosystems. While nucleocytoviruses are known to be highly diverse in metagenomes, knowledge of their capsid structures is restricted to a few characterized representatives.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Amazing structural diversity of giant virus-like particles in forest soil
Authors: Fischer, M. G.; Mersdorf, U.; Blanchard, J. L.
Score: 118.6, Published: 2023-06-30 DOI: 10.1101/2023.06.30.546935
Large DNA viruses of the phylum Nucleocytoviricota infect diverse eukaryotic hosts from protists to humans, with profound consequences for aquatic and terrestrial ecosystems. While nucleocytoviruses are known to be highly diverse in metagenomes, knowledge of their capsid structures is restricted to a few characterized representatives." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-05T10:42:43+00:00" />
<meta property="article:modified_time" content="2023-07-05T10:42:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Amazing structural diversity of giant virus-like particles in forest soil
Authors: Fischer, M. G.; Mersdorf, U.; Blanchard, J. L.
Score: 118.6, Published: 2023-06-30 DOI: 10.1101/2023.06.30.546935
Large DNA viruses of the phylum Nucleocytoviricota infect diverse eukaryotic hosts from protists to humans, with profound consequences for aquatic and terrestrial ecosystems. While nucleocytoviruses are known to be highly diverse in metagenomes, knowledge of their capsid structures is restricted to a few characterized representatives."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Amazing structural diversity of giant virus-like particles in forest soil\nAuthors: Fischer, M. G.; Mersdorf, U.; Blanchard, J. L.\nScore: 118.6, Published: 2023-06-30 DOI: 10.1101/2023.06.30.546935\nLarge DNA viruses of the phylum Nucleocytoviricota infect diverse eukaryotic hosts from protists to humans, with profound consequences for aquatic and terrestrial ecosystems. While nucleocytoviruses are known to be highly diverse in metagenomes, knowledge of their capsid structures is restricted to a few characterized representatives.",
  "keywords": [
    
  ],
  "articleBody": " Amazing structural diversity of giant virus-like particles in forest soil\nAuthors: Fischer, M. G.; Mersdorf, U.; Blanchard, J. L.\nScore: 118.6, Published: 2023-06-30 DOI: 10.1101/2023.06.30.546935\nLarge DNA viruses of the phylum Nucleocytoviricota infect diverse eukaryotic hosts from protists to humans, with profound consequences for aquatic and terrestrial ecosystems. While nucleocytoviruses are known to be highly diverse in metagenomes, knowledge of their capsid structures is restricted to a few characterized representatives. Here, we visualize giant virus-like particles (VLPs, diameter \u003e0.2 m) directly from the environment using transmission electron microscopy. We found that Harvard Forest soils contain a higher diversity of giant VLP morphotypes than all hitherto isolated giant viruses combined. These included VLPs with icosahedral capsid symmetry, ovoid shapes similar to pandoraviruses, and bacilliform shapes that may represent novel viruses. We discovered giant icosahedral capsids with structural modifications that had not been described before including tubular appendages, modified vertices, tails, and capsids consisting of multiple layers or internal channels. Many giant VLPs were covered with fibers of varying lengths, thicknesses, densities, and terminal structures. These findings imply that giant viruses employ a much wider array of capsid structures and mechanisms to interact with their host cells than is currently known. We also found diverse tailed bacteriophages and filamentous VLPs, as well as ultra-small cells. Our study offers a first glimpse of the vast diversity of unexplored viral structures in soil and reinforces the potential of transmission electron microscopy for fundamental discoveries in environmental microbiology.\nSARS-CoV-2 spike glycosylation affects function and neutralization sensitivity\nAuthors: Zhang, F.; Schmidt, F.; Muecksch, F.; Wang, Z.; Gazumyan, A.; Nussenzweig, M. C.; Gaebler, C.; Caskey, M.; Hatziioannou, T.; Bieniasz, P. D.\nScore: 79.4, Published: 2023-06-30 DOI: 10.1101/2023.06.30.547241\nThe glycosylation of viral envelope proteins can play important roles in virus biology and immune evasion. The spike (S) glycoprotein of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) includes 22 N-linked glycosylation sequons and 17 O-linked glycosites. Here, we investigated the effect of individual glycosylation sites on SARS-CoV-2 S function in pseudotyped virus infection assays and on sensitivity to monoclonal and polyclonal neutralizing antibodies. In most cases, removal of individual glycosylation sites decreased the infectiousness of the pseudotyped virus. For glycosylation mutants in the N-terminal domain (NTD) and the receptor binding domain (RBD), reduction in pseudotype infectivity was predicted by a commensurate reduction in the level of virion-incorporated spike protein. Notably, the presence of a glycan at position N343 within the RBD had diverse effects on neutralization by RBD-specific monoclonal antibodies (mAbs) cloned from convalescent individuals. The N343 glycan reduced overall sensitivity to polyclonal antibodies in plasma from COVID-19 convalescent individuals, suggesting a role for SARS-CoV-2 spike glycosylation in immune evasion. However, vaccination of convalescent individuals produced neutralizing activity that was resilient to the inhibitory effect of the N343 glycan.\nEukaryotic antiviral immune proteins arose via convergence, horizontal transfer, and ancient inheritance\nAuthors: Culbertson, E. M.; Levin, T. C.\nScore: 51.6, Published: 2023-06-27 DOI: 10.1101/2023.06.27.546753\nAnimals use a variety of cell-autonomous innate immune proteins to detect viral infections and prevent replication. Recent studies have discovered that a subset of mammalian antiviral proteins have homology to anti-phage defense proteins in bacteria, implying that there are aspects of innate immunity that are shared across the Tree of Life. While the majority of these studies have focused on characterizing the diversity and biochemical functions of the bacterial proteins, the evolutionary relationships between animal and bacterial proteins are less clear. This ambiguity is partly due to the long evolutionary distances separating animal and bacterial proteins, which obscures their relationships. Here, we tackle this problem for three innate immune families (CD-NTases [including cGAS], STINGs, and Viperins) by deeply sampling protein diversity across eukaryotes. We find that Viperins are truly ancient immune proteins, likely inherited since the last eukaryotic common ancestor and possibly longer. In contrast, we find other immune proteins that arose via at least five independent events of horizontal gene transfer (HGT) from bacteria. Two of these events allowed algae to acquire new bacterial viperins, while three more HGT events gave rise to distinct superfamilies of eukaryotic CD-NTases: the OAS superfamily, which is widespread across eukaryotes; the Mab21 superfamily (containing cGAS) which has diversified via a series of animal-specific duplications, and a previously undefined eSMODS superfamily, which more closely resembles bacterial CD-NTases. Finally, we found that cGAS and STING proteins have substantially different histories, with STINGs arising via convergent domain shuffling in bacteria and eukaryotes. Overall, our findings paint a picture of eukaryotic innate immunity as highly dynamic, where eukaryotes build upon their ancient antiviral repertoires through the reuse of protein domains and by repeatedly sampling a rich reservoir of bacterial anti-phage genes.\nDistinct groups of RNA viruses associated with thermoacidophilic bacteria\nAuthors: Urayama, S.-i.; Fukudome, A.; Hirai, M.; Okumura, T.; Nishimura, Y.; Takaki, Y.; Kurosawa, N.; Koonin, E. V.; Krupovic, M.; Nunoura, T.\nScore: 40.8, Published: 2023-07-03 DOI: 10.1101/2023.07.02.547447\nRecent massive metatranscriptome mining substantially expanded the diversity of the bacterial RNA virome, suggesting that additional groups of riboviruses infecting bacterial hosts remain to be discovered. We employed full length double-stranded (ds) RNA sequencing for identification of riboviruses associated with microbial consortia dominated by bacteria and archaea in acidic hot springs in Japan. Whole sequences of two groups of multisegmented riboviruses genomes were obtained. One group, which we denoted hot spring riboviruses (HsRV), consists of unusual viruses with distinct RNA-dependent RNA polymerases (RdRPs) that seem to be intermediates between typical ribovirus RdRPs and viral reverse transcriptases. We also identified viruses encoding HsRV-like RdRPs in moderate aquatic environments, including marine water, river sediments and salt marsh, indicating that this previously overlooked ribovirus group is not restricted to the extreme ecosystem. The HsRV-like viruses are candidates for a distinct phylum or even kingdom within the viral realm Riboviria. The second group, denoted hot spring partiti-like viruses (HsPV), is a distinct branch within the family Partitiviridae. All genome segments in both these groups of viruses display the organization typical of bacterial riboviruses, where multiple open reading frames encoding individual proteins are preceded by ribosome-binding sites. Together with the identification in bacteria-dominated habitats, this genome architecture indicates that riboviruses of these distinct groups infect thermoacidophilic bacterial hosts.\nAntiviral Type III CRISPR signalling via conjugation of ATP and AdoMet\nAuthors: Chi, H.; Hoikkala, V.; Gruschow, S.; Graham, S.; Shirran, S.; White, M. F.\nScore: 36.9, Published: 2023-06-27 DOI: 10.1101/2023.06.26.546636\nCRISPR systems are widespread in the prokaryotic world, providing adaptive immunity against mobile genetic elements (MGE) 1, 2. Type III CRISPR systems, with the signature gene cas10, use CRISPR RNA (crRNA) to detect non-self RNA, activating the enzymatic Cas10 subunit to defend the cell against MGE either directly, via the integral HD nuclease domain 3-5 or indirectly, via synthesis of cyclic oligonucleotide (cOA) second messengers to activate diverse ancillary effectors 6-9. A subset of type III CRISPR systems encode an uncharacterised CorA-family membrane protein and an associated NrN family phosphodiesterase predicted to function in antiviral defence. Here, we demonstrate that the CorA associated type III-B (Cmr) CRISPR system from Bacteroides fragilis provides immunity against MGE when expressed inE. coli. However, B. fragilis Cmr does not synthesise cOA species on activation, instead generating a previously undescribed sigalling molecule, SAM-AMP (3-adenylyl-AdoMet) by conjugating ATP to S-adenosyl methionine via a phosphodiester bond. Once synthesised, SAM-AMP binds to the CorA effector, presumably leading to cell death by disruption of the membrane integrity. SAM-AMP is degraded by CRISPR associated phosphodiesterases or a SAM-AMP lyase, providing an \"off switch\" analogous to cOA specific ring nucleases 10. SAM-AMP thus represents a new class of second messenger for antiviral signalling, which may function in different roles in diverse cellular contexts.\nHoming in on the rare virosphere reveals the native host of giant viruses\nAuthors: Fromm, A.; Hevroni, G.; Vincent, F.; Schatz, D.; Martinez-Gutierrez, C. A.; Aylward, F. O.; Vardi, A.\nScore: 27.6, Published: 2023-06-27 DOI: 10.1101/2023.06.27.546645\nGiant viruses (phylum Nucleocytoviricota) are globally distributed in aquatic ecosystems1,2. They play major roles as evolutionary drivers of eukaryotic plankton3 and regulators of global biogeochemical cycles4. Recent metagenomic studies have significantly expanded the known diversity of marine giant viruses1,5-7, but we still lack fundamental knowledge about their native hosts, thereby hindering our understanding of their lifecycle and ecological importance. Here, we aim to discover the native hosts of giant viruses using a novel, sensitive single-cell metatranscriptomic approach. By applying this approach to natural plankton communities, we unraveled an active viral infection of several giant viruses, from multiple lineages, and identified their native hosts. We identify a rare lineage of giant virus (Imitervirales-07) infecting a minute population of protists (class Katablepharidaceae) and revealed the prevalence of highly expressed viral-encoded cell-fate regulation genes in infected cells. Further examination of this host-virus dynamics in a temporal resolution suggested this giant virus controls its host population demise. Our results demonstrate how single-cell metatranscriptomics is a sensitive approach for pairing viruses with their authentic hosts and studying their ecological significance in a culture-independent manner in the marine environment.\nConstruction and Characterisation of a Structured, Tuneable, and Transparent 3D Culture Platform for Soil Bacteria\nAuthors: Rooney, L. M.; Dupuy, L. X.; Hoskisson, P. A.; McConnell, G.\nScore: 24.5, Published: 2023-06-28 DOI: 10.1101/2023.06.28.546105\n2.We have developed a tuneable workflow for the study of soil microbes in an imitative 3D soil environment that is compatible with routine and advanced optical imaging, is chemically customisable, and is reliably refractive index matched based on the metabolic profile of the study organism. We demonstrate our transparent soil pipeline with two representative soil organisms, Bacillus subtilis and Streptomyces coelicolor, and visualise their colonisation behaviours using fluorescence microscopy and mesoscopy. This spatially structured, 3D approach to microbial culture has the potential to further study the behaviour of other difficult-to-culture bacteria in conditions matching their native environment and could be expanded to study microbial interactions, such as interaction, competition, and warfare. 3. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=179 SRC=\"FIGDIR/small/546105v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (45K): org.highwire.dtl.DTLVardef@1c472d6org.highwire.dtl.DTLVardef@4f5713org.highwire.dtl.DTLVardef@ddbf53org.highwire.dtl.DTLVardef@3c2230_HPS_FORMAT_FIGEXP M_FIG C_FIG A step-by-step method for creating a tailored 3D culture medium for study of soil microbes. The complete workflow can be split into three parts: Growth and observation, metabolic profiling to provide a stable refractive index matching solution, and production of the 3D soil environment. The 3D culture scaffold was created by cryomilling Nafion resin pellets and size filtration. Chemical processing altered the surface chemistry of Nafion particles and facilitated nutrient binding by titration of a defined liquid culture medium. Metabolic profiling determined non-metabolisable sugars and provided an inert refractive index matching substrate, which was added to the final nutrient titration. Inoculation and growth of the test strain allowed for downstream assessment of colonisation behaviours and community dynamics in situ by, for example, optical microscopy.\nA one-track model for spatiotemporal coordination of Bacillus subtilis septal cell wall synthesis\nAuthors: Whitley, K. D.; Grimshaw, J.; Karinou, E.; Stansfeld, P. J.; Holden, S.\nScore: 23.7, Published: 2023-06-29 DOI: 10.1101/2023.06.29.547024\nBacterial cell division requires synthesis of a septal peptidoglycan (sPG) wall across the middle of the cell. This is accomplished by the divisome synthesis complex in coordination with numerous other division proteins--such as the essential tubulin homolog FtsZ--but the molecular mechanism of its spatiotemporal regulation remains unclear. Here, we investigate the dynamics of sPG synthesis in the model Gram-positive bacterium Bacillus subtilis using live-cell single-molecule imaging of the divisome transpeptidase PBP2B. In contrast to previous models for division, we show that there is a single population of processively-moving PBP2B molecules whose motion is driven by peptidoglycan synthesis and is not associated with FtsZ treadmilling. However, although the motions of PBP2B and FtsZ are asynchronous, we demonstrate that processive PBP2B motion is regulated by FtsZ treadmilling. Additionally, we provide evidence that the divisome synthesis complex is multimeric. Our results support a new model for division in B. subtilis where a multimeric synthesis complex follows a single track dependent on sPG synthesis whose activity is regulated by FtsZ treadmilling.\nBacterial SEAL domains undergo autoproteolysis and function in regulated intramembrane proteolysis\nAuthors: Brogan, A. P.; Habib, C.; Hobbs, S. J.; Kranzusch, P. J.; Rudner, D. Z.\nScore: 22.5, Published: 2023-06-27 DOI: 10.1101/2023.06.27.546760\nGram-positive bacteria use SigI/RsgI-family sigma factor/anti-sigma factor pairs to sense and respond to cell wall defects and plant polysaccharides. In Bacillus subtilis this signal transduction pathway involves regulated intramembrane proteolysis (RIP) of the membrane-anchored anti-sigma factor RsgI. However, unlike most RIP signaling pathways, site-1 cleavage of RsgI on the extracytoplasmic side of the membrane is constitutive and the cleavage products remain stably associated, preventing intramembrane proteolysis. The regulated step in this pathway is their dissociation, which is hypothesized to involve mechanical force. Release of the ectodomain enables intramembrane cleavage by the RasP site-2 protease and activation of SigI. The constitutive site-1 protease has not been identified for any RsgI homolog. Here, we report that RsgIs extracytoplasmic domain has structural and functional similarities to eukaryotic SEA domains that undergo autoproteolysis and have been implicated in mechanotransduction. We show that site-1 proteolysis in B. subtilis and Clostridial RsgI family members is mediated by enzyme-independent autoproteolysis of these SEA-like (SEAL) domains. Importantly, the site of proteolysis enables retention of the ectodomain through an undisrupted {beta}-sheet that spans the two cleavage products. Autoproteolysis can be abrogated by relief of conformational strain in the scissile loop, in a mechanism analogous to eukaryotic SEA domains. Collectively, our data support the model that RsgI-SigI signaling is mediated by mechanotransduction in a manner that has striking parallels with eukaryotic mechanotransducive signaling pathways. SIGNIFICANCESEA domains are broadly conserved among eukaryotes but absent in bacteria. They are present on diverse membrane-anchored proteins some of which have been implicated in mechanotransducive signaling pathways. Many of these domains have been found to undergo autoproteolysis and remain noncovalently associated following cleavage. Their dissociation requires mechanical force. Here, we identify a family of bacterial SEA-like (SEAL) domains that arose independently from their eukaryotic counterparts but have structural and functional similarities. We show these SEAL domains autocleave and the cleavage products remain stably associated. Importantly, these domains are present on membrane-anchored anti-sigma factors that have been implicated in mechanotransduction pathways analogous to those in eukaryotes. Our findings suggest that bacterial and eukaryotic signaling systems have evolved a similar mechanism to transduce mechanical stimuli across the lipid bilayer.\nProcessive movement of Staphylococcus aureus essential septal peptidoglycan synthases is independent of FtsZ treadmilling and drives cell constriction\nAuthors: Schaeper, S.; Brito, A. D.; Saraiva, B. M.; Squyres, G. R.; Holmes, M. J.; Garner, E. C.; Hensel, Z.; Henriques, R.; Pinho, M. G.\nScore: 22.0, Published: 2023-06-29 DOI: 10.1101/2023.06.29.547026\nBacterial cell division is mediated by the tubulin-homolog FtsZ, which recruits peptidoglycan (PG) synthesis enzymes to the division site. Septal PG synthases promote inward growth of the division septum, but the mechanisms governing the spatiotemporal regulation of these enzymes are poorly understood. Recent studies on various organisms have proposed different models for the relationship between the movement and activity of septum-specific PG synthases and FtsZ treadmilling. Here, we studied the movement dynamics of conserved cell division proteins relative to the rates of septum constriction and FtsZ treadmilling in the Gram-positive pathogen Staphylococcus aureus. The septal PG synthesis enzyme complex FtsW/PBP1 and its putative activator protein, DivIB, moved processively, around the division site, with the same velocity. Impairing FtsZ treadmilling did not affect FtsW and DivIB velocities or septum constriction rates. Contrarily, inhibition of PG synthesis slowed down or completely stopped both septum constriction and the directional movement of FtsW/PBP1 and DivIB. Our findings support a model for S. aureus in which a single population of processively moving FtsW/PBP1 remains associated with DivIB to drive cell constriction independently of treadmilling FtsZ filaments.\n",
  "wordCount" : "2578",
  "inLanguage": "en",
  "datePublished": "2023-07-05T10:42:43Z",
  "dateModified": "2023-07-05T10:42:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta"><span>updated on July 5, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.546935">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.546935" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.546935">
        <p class="paperTitle">Amazing structural diversity of giant virus-like particles in forest soil</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.546935" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.546935" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fischer, M. G.; Mersdorf, U.; Blanchard, J. L.</p>
        <p class="info">Score: 118.6, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.546935' target='https://doi.org/10.1101/2023.06.30.546935'> 10.1101/2023.06.30.546935</a></p>
        <p class="abstract">Large DNA viruses of the phylum Nucleocytoviricota infect diverse eukaryotic hosts from protists to humans, with profound consequences for aquatic and terrestrial ecosystems. While nucleocytoviruses are known to be highly diverse in metagenomes, knowledge of their capsid structures is restricted to a few characterized representatives. Here, we visualize giant virus-like particles (VLPs, diameter &gt;0.2 m) directly from the environment using transmission electron microscopy. We found that Harvard Forest soils contain a higher diversity of giant VLP morphotypes than all hitherto isolated giant viruses combined. These included VLPs with icosahedral capsid symmetry, ovoid shapes similar to pandoraviruses, and bacilliform shapes that may represent novel viruses. We discovered giant icosahedral capsids with structural modifications that had not been described before including tubular appendages, modified vertices, tails, and capsids consisting of multiple layers or internal channels. Many giant VLPs were covered with fibers of varying lengths, thicknesses, densities, and terminal structures. These findings imply that giant viruses employ a much wider array of capsid structures and mechanisms to interact with their host cells than is currently known. We also found diverse tailed bacteriophages and filamentous VLPs, as well as ultra-small cells. Our study offers a first glimpse of the vast diversity of unexplored viral structures in soil and reinforces the potential of transmission electron microscopy for fundamental discoveries in environmental microbiology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547241">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547241" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547241">
        <p class="paperTitle">SARS-CoV-2 spike glycosylation affects function and neutralization sensitivity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547241" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547241" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, F.; Schmidt, F.; Muecksch, F.; Wang, Z.; Gazumyan, A.; Nussenzweig, M. C.; Gaebler, C.; Caskey, M.; Hatziioannou, T.; Bieniasz, P. D.</p>
        <p class="info">Score: 79.4, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547241' target='https://doi.org/10.1101/2023.06.30.547241'> 10.1101/2023.06.30.547241</a></p>
        <p class="abstract">The glycosylation of viral envelope proteins can play important roles in virus biology and immune evasion. The spike (S) glycoprotein of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) includes 22 N-linked glycosylation sequons and 17 O-linked glycosites. Here, we investigated the effect of individual glycosylation sites on SARS-CoV-2 S function in pseudotyped virus infection assays and on sensitivity to monoclonal and polyclonal neutralizing antibodies. In most cases, removal of individual glycosylation sites decreased the infectiousness of the pseudotyped virus. For glycosylation mutants in the N-terminal domain (NTD) and the receptor binding domain (RBD), reduction in pseudotype infectivity was predicted by a commensurate reduction in the level of virion-incorporated spike protein. Notably, the presence of a glycan at position N343 within the RBD had diverse effects on neutralization by RBD-specific monoclonal antibodies (mAbs) cloned from convalescent individuals. The N343 glycan reduced overall sensitivity to polyclonal antibodies in plasma from COVID-19 convalescent individuals, suggesting a role for SARS-CoV-2 spike glycosylation in immune evasion. However, vaccination of convalescent individuals produced neutralizing activity that was resilient to the inhibitory effect of the N343 glycan.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546753">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546753" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546753">
        <p class="paperTitle">Eukaryotic antiviral immune proteins arose via convergence, horizontal transfer, and ancient inheritance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546753" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546753" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Culbertson, E. M.; Levin, T. C.</p>
        <p class="info">Score: 51.6, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546753' target='https://doi.org/10.1101/2023.06.27.546753'> 10.1101/2023.06.27.546753</a></p>
        <p class="abstract">Animals use a variety of cell-autonomous innate immune proteins to detect viral infections and prevent replication. Recent studies have discovered that a subset of mammalian antiviral proteins have homology to anti-phage defense proteins in bacteria, implying that there are aspects of innate immunity that are shared across the Tree of Life. While the majority of these studies have focused on characterizing the diversity and biochemical functions of the bacterial proteins, the evolutionary relationships between animal and bacterial proteins are less clear. This ambiguity is partly due to the long evolutionary distances separating animal and bacterial proteins, which obscures their relationships. Here, we tackle this problem for three innate immune families (CD-NTases [including cGAS], STINGs, and Viperins) by deeply sampling protein diversity across eukaryotes. We find that Viperins are truly ancient immune proteins, likely inherited since the last eukaryotic common ancestor and possibly longer. In contrast, we find other immune proteins that arose via at least five independent events of horizontal gene transfer (HGT) from bacteria. Two of these events allowed algae to acquire new bacterial viperins, while three more HGT events gave rise to distinct superfamilies of eukaryotic CD-NTases: the OAS superfamily, which is widespread across eukaryotes; the Mab21 superfamily (containing cGAS) which has diversified via a series of animal-specific duplications, and a previously undefined eSMODS superfamily, which more closely resembles bacterial CD-NTases. Finally, we found that cGAS and STING proteins have substantially different histories, with STINGs arising via convergent domain shuffling in bacteria and eukaryotes. Overall, our findings paint a picture of eukaryotic innate immunity as highly dynamic, where eukaryotes build upon their ancient antiviral repertoires through the reuse of protein domains and by repeatedly sampling a rich reservoir of bacterial anti-phage genes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.02.547447">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.02.547447" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.02.547447">
        <p class="paperTitle">Distinct groups of RNA viruses associated with thermoacidophilic bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.02.547447" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.02.547447" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Urayama, S.-i.; Fukudome, A.; Hirai, M.; Okumura, T.; Nishimura, Y.; Takaki, Y.; Kurosawa, N.; Koonin, E. V.; Krupovic, M.; Nunoura, T.</p>
        <p class="info">Score: 40.8, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.02.547447' target='https://doi.org/10.1101/2023.07.02.547447'> 10.1101/2023.07.02.547447</a></p>
        <p class="abstract">Recent massive metatranscriptome mining substantially expanded the diversity of the bacterial RNA virome, suggesting that additional groups of riboviruses infecting bacterial hosts remain to be discovered. We employed full length double-stranded (ds) RNA sequencing for identification of riboviruses associated with microbial consortia dominated by bacteria and archaea in acidic hot springs in Japan. Whole sequences of two groups of multisegmented riboviruses genomes were obtained. One group, which we denoted hot spring riboviruses (HsRV), consists of unusual viruses with distinct RNA-dependent RNA polymerases (RdRPs) that seem to be intermediates between typical ribovirus RdRPs and viral reverse transcriptases. We also identified viruses encoding HsRV-like RdRPs in moderate aquatic environments, including marine water, river sediments and salt marsh, indicating that this previously overlooked ribovirus group is not restricted to the extreme ecosystem. The HsRV-like viruses are candidates for a distinct phylum or even kingdom within the viral realm Riboviria. The second group, denoted hot spring partiti-like viruses (HsPV), is a distinct branch within the family Partitiviridae. All genome segments in both these groups of viruses display the organization typical of bacterial riboviruses, where multiple open reading frames encoding individual proteins are preceded by ribosome-binding sites. Together with the identification in bacteria-dominated habitats, this genome architecture indicates that riboviruses of these distinct groups infect thermoacidophilic bacterial hosts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546636">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546636" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546636">
        <p class="paperTitle">Antiviral Type III CRISPR signalling via conjugation of ATP and AdoMet</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546636" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546636" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chi, H.; Hoikkala, V.; Gruschow, S.; Graham, S.; Shirran, S.; White, M. F.</p>
        <p class="info">Score: 36.9, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546636' target='https://doi.org/10.1101/2023.06.26.546636'> 10.1101/2023.06.26.546636</a></p>
        <p class="abstract">CRISPR systems are widespread in the prokaryotic world, providing adaptive immunity against mobile genetic elements (MGE) 1, 2. Type III CRISPR systems, with the signature gene cas10, use CRISPR RNA (crRNA) to detect non-self RNA, activating the enzymatic Cas10 subunit to defend the cell against MGE either directly, via the integral HD nuclease domain 3-5 or indirectly, via synthesis of cyclic oligonucleotide (cOA) second messengers to activate diverse ancillary effectors 6-9. A subset of type III CRISPR systems encode an uncharacterised CorA-family membrane protein and an associated NrN family phosphodiesterase predicted to function in antiviral defence. Here, we demonstrate that the CorA associated type III-B (Cmr) CRISPR system from Bacteroides fragilis provides immunity against MGE when expressed inE. coli. However, B. fragilis Cmr does not synthesise cOA species on activation, instead generating a previously undescribed sigalling molecule, SAM-AMP (3-adenylyl-AdoMet) by conjugating ATP to S-adenosyl methionine via a phosphodiester bond. Once synthesised, SAM-AMP binds to the CorA effector, presumably leading to cell death by disruption of the membrane integrity. SAM-AMP is degraded by CRISPR associated phosphodiesterases or a SAM-AMP lyase, providing an &#34;off switch&#34; analogous to cOA specific ring nucleases 10. SAM-AMP thus represents a new class of second messenger for antiviral signalling, which may function in different roles in diverse cellular contexts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546645">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546645" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546645">
        <p class="paperTitle">Homing in on the rare virosphere reveals the native host of giant viruses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546645" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546645" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fromm, A.; Hevroni, G.; Vincent, F.; Schatz, D.; Martinez-Gutierrez, C. A.; Aylward, F. O.; Vardi, A.</p>
        <p class="info">Score: 27.6, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546645' target='https://doi.org/10.1101/2023.06.27.546645'> 10.1101/2023.06.27.546645</a></p>
        <p class="abstract">Giant viruses (phylum Nucleocytoviricota) are globally distributed in aquatic ecosystems1,2. They play major roles as evolutionary drivers of eukaryotic plankton3 and regulators of global biogeochemical cycles4. Recent metagenomic studies have significantly expanded the known diversity of marine giant viruses1,5-7, but we still lack fundamental knowledge about their native hosts, thereby hindering our understanding of their lifecycle and ecological importance. Here, we aim to discover the native hosts of giant viruses using a novel, sensitive single-cell metatranscriptomic approach. By applying this approach to natural plankton communities, we unraveled an active viral infection of several giant viruses, from multiple lineages, and identified their native hosts. We identify a rare lineage of giant virus (Imitervirales-07) infecting a minute population of protists (class Katablepharidaceae) and revealed the prevalence of highly expressed viral-encoded cell-fate regulation genes in infected cells. Further examination of this host-virus dynamics in a temporal resolution suggested this giant virus controls its host population demise. Our results demonstrate how single-cell metatranscriptomics is a sensitive approach for pairing viruses with their authentic hosts and studying their ecological significance in a culture-independent manner in the marine environment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546105">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546105" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546105">
        <p class="paperTitle">Construction and Characterisation of a Structured, Tuneable, and Transparent 3D Culture Platform for Soil Bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546105" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546105" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rooney, L. M.; Dupuy, L. X.; Hoskisson, P. A.; McConnell, G.</p>
        <p class="info">Score: 24.5, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546105' target='https://doi.org/10.1101/2023.06.28.546105'> 10.1101/2023.06.28.546105</a></p>
        <p class="abstract">2.We have developed a tuneable workflow for the study of soil microbes in an imitative 3D soil environment that is compatible with routine and advanced optical imaging, is chemically customisable, and is reliably refractive index matched based on the metabolic profile of the study organism. We demonstrate our transparent soil pipeline with two representative soil organisms, Bacillus subtilis and Streptomyces coelicolor, and visualise their colonisation behaviours using fluorescence microscopy and mesoscopy. This spatially structured, 3D approach to microbial culture has the potential to further study the behaviour of other difficult-to-culture bacteria in conditions matching their native environment and could be expanded to study microbial interactions, such as interaction, competition, and warfare.

3. Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=179 SRC=&#34;FIGDIR/small/546105v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (45K):
org.highwire.dtl.DTLVardef@1c472d6org.highwire.dtl.DTLVardef@4f5713org.highwire.dtl.DTLVardef@ddbf53org.highwire.dtl.DTLVardef@3c2230_HPS_FORMAT_FIGEXP  M_FIG C_FIG A step-by-step method for creating a tailored 3D culture medium for study of soil microbes.

The complete workflow can be split into three parts: Growth and observation, metabolic profiling to provide a stable refractive index matching solution, and production of the 3D soil environment. The 3D culture scaffold was created by cryomilling Nafion resin pellets and size filtration. Chemical processing altered the surface chemistry of Nafion particles and facilitated nutrient binding by titration of a defined liquid culture medium. Metabolic profiling determined non-metabolisable sugars and provided an inert refractive index matching substrate, which was added to the final nutrient titration. Inoculation and growth of the test strain allowed for downstream assessment of colonisation behaviours and community dynamics in situ by, for example, optical microscopy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547024">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547024" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547024">
        <p class="paperTitle">A one-track model for spatiotemporal coordination of Bacillus subtilis septal cell wall synthesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547024" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547024" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Whitley, K. D.; Grimshaw, J.; Karinou, E.; Stansfeld, P. J.; Holden, S.</p>
        <p class="info">Score: 23.7, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547024' target='https://doi.org/10.1101/2023.06.29.547024'> 10.1101/2023.06.29.547024</a></p>
        <p class="abstract">Bacterial cell division requires synthesis of a septal peptidoglycan (sPG) wall across the middle of the cell. This is accomplished by the divisome synthesis complex in coordination with numerous other division proteins--such as the essential tubulin homolog FtsZ--but the molecular mechanism of its spatiotemporal regulation remains unclear. Here, we investigate the dynamics of sPG synthesis in the model Gram-positive bacterium Bacillus subtilis using live-cell single-molecule imaging of the divisome transpeptidase PBP2B. In contrast to previous models for division, we show that there is a single population of processively-moving PBP2B molecules whose motion is driven by peptidoglycan synthesis and is not associated with FtsZ treadmilling. However, although the motions of PBP2B and FtsZ are asynchronous, we demonstrate that processive PBP2B motion is regulated by FtsZ treadmilling. Additionally, we provide evidence that the divisome synthesis complex is multimeric. Our results support a new model for division in B. subtilis where a multimeric synthesis complex follows a single track dependent on sPG synthesis whose activity is regulated by FtsZ treadmilling.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546760">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546760" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546760">
        <p class="paperTitle">Bacterial SEAL domains undergo autoproteolysis and function in regulated intramembrane proteolysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546760" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546760" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brogan, A. P.; Habib, C.; Hobbs, S. J.; Kranzusch, P. J.; Rudner, D. Z.</p>
        <p class="info">Score: 22.5, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546760' target='https://doi.org/10.1101/2023.06.27.546760'> 10.1101/2023.06.27.546760</a></p>
        <p class="abstract">Gram-positive bacteria use SigI/RsgI-family sigma factor/anti-sigma factor pairs to sense and respond to cell wall defects and plant polysaccharides. In Bacillus subtilis this signal transduction pathway involves regulated intramembrane proteolysis (RIP) of the membrane-anchored anti-sigma factor RsgI. However, unlike most RIP signaling pathways, site-1 cleavage of RsgI on the extracytoplasmic side of the membrane is constitutive and the cleavage products remain stably associated, preventing intramembrane proteolysis. The regulated step in this pathway is their dissociation, which is hypothesized to involve mechanical force. Release of the ectodomain enables intramembrane cleavage by the RasP site-2 protease and activation of SigI. The constitutive site-1 protease has not been identified for any RsgI homolog. Here, we report that RsgIs extracytoplasmic domain has structural and functional similarities to eukaryotic SEA domains that undergo autoproteolysis and have been implicated in mechanotransduction. We show that site-1 proteolysis in B. subtilis and Clostridial RsgI family members is mediated by enzyme-independent autoproteolysis of these SEA-like (SEAL) domains. Importantly, the site of proteolysis enables retention of the ectodomain through an undisrupted {beta}-sheet that spans the two cleavage products. Autoproteolysis can be abrogated by relief of conformational strain in the scissile loop, in a mechanism analogous to eukaryotic SEA domains. Collectively, our data support the model that RsgI-SigI signaling is mediated by mechanotransduction in a manner that has striking parallels with eukaryotic mechanotransducive signaling pathways.

SIGNIFICANCESEA domains are broadly conserved among eukaryotes but absent in bacteria. They are present on diverse membrane-anchored proteins some of which have been implicated in mechanotransducive signaling pathways. Many of these domains have been found to undergo autoproteolysis and remain noncovalently associated following cleavage. Their dissociation requires mechanical force. Here, we identify a family of bacterial SEA-like (SEAL) domains that arose independently from their eukaryotic counterparts but have structural and functional similarities. We show these SEAL domains autocleave and the cleavage products remain stably associated. Importantly, these domains are present on membrane-anchored anti-sigma factors that have been implicated in mechanotransduction pathways analogous to those in eukaryotes. Our findings suggest that bacterial and eukaryotic signaling systems have evolved a similar mechanism to transduce mechanical stimuli across the lipid bilayer.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547026">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547026" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547026">
        <p class="paperTitle">Processive movement of Staphylococcus aureus essential septal peptidoglycan synthases is independent of FtsZ treadmilling and drives cell constriction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547026" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547026" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schaeper, S.; Brito, A. D.; Saraiva, B. M.; Squyres, G. R.; Holmes, M. J.; Garner, E. C.; Hensel, Z.; Henriques, R.; Pinho, M. G.</p>
        <p class="info">Score: 22.0, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547026' target='https://doi.org/10.1101/2023.06.29.547026'> 10.1101/2023.06.29.547026</a></p>
        <p class="abstract">Bacterial cell division is mediated by the tubulin-homolog FtsZ, which recruits peptidoglycan (PG) synthesis enzymes to the division site. Septal PG synthases promote inward growth of the division septum, but the mechanisms governing the spatiotemporal regulation of these enzymes are poorly understood. Recent studies on various organisms have proposed different models for the relationship between the movement and activity of septum-specific PG synthases and FtsZ treadmilling. Here, we studied the movement dynamics of conserved cell division proteins relative to the rates of septum constriction and FtsZ treadmilling in the Gram-positive pathogen Staphylococcus aureus. The septal PG synthesis enzyme complex FtsW/PBP1 and its putative activator protein, DivIB, moved processively, around the division site, with the same velocity. Impairing FtsZ treadmilling did not affect FtsW and DivIB velocities or septum constriction rates. Contrarily, inhibition of PG synthesis slowed down or completely stopped both septum constriction and the directional movement of FtsW/PBP1 and DivIB. Our findings support a model for S. aureus in which a single population of processively moving FtsW/PBP1 remains associated with DivIB to drive cell constriction independently of treadmilling FtsZ filaments.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
