<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Virological characteristics of the SARS-CoV-2 Omicron EG.5.1 variant
Authors: Tsujino, S.; Deguchi, S.; Nomai, T.; Padilla-Blanco, M.; Plianchaisuk, A.; Wang, L.; Begum, M. M.; Uriu, K.; Mizuma, K.; Nao, N.; Kojima, I.; Tsubo, T.; Li, J.; Matsumura, Y.; Nagao, M.; Oda, Y.; Tsuda, M.; Anraku, Y.; Kita, S.; Yajima, H.; Tabata, K.; Guo, Z.; Hinay, A. A.; Yoshimatsu, K.; Yamamoto, Y.; Nagamoto, Y.; Asakura, H.; Nagashima, M.; Sadamasu, K.; Yoshimura, K.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Virological characteristics of the SARS-CoV-2 Omicron EG.5.1 variant
Authors: Tsujino, S.; Deguchi, S.; Nomai, T.; Padilla-Blanco, M.; Plianchaisuk, A.; Wang, L.; Begum, M. M.; Uriu, K.; Mizuma, K.; Nao, N.; Kojima, I.; Tsubo, T.; Li, J.; Matsumura, Y.; Nagao, M.; Oda, Y.; Tsuda, M.; Anraku, Y.; Kita, S.; Yajima, H.; Tabata, K.; Guo, Z.; Hinay, A. A.; Yoshimatsu, K.; Yamamoto, Y.; Nagamoto, Y.; Asakura, H.; Nagashima, M.; Sadamasu, K.; Yoshimura, K." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-29T10:37:40+00:00" />
<meta property="article:modified_time" content="2023-10-29T10:37:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Virological characteristics of the SARS-CoV-2 Omicron EG.5.1 variant
Authors: Tsujino, S.; Deguchi, S.; Nomai, T.; Padilla-Blanco, M.; Plianchaisuk, A.; Wang, L.; Begum, M. M.; Uriu, K.; Mizuma, K.; Nao, N.; Kojima, I.; Tsubo, T.; Li, J.; Matsumura, Y.; Nagao, M.; Oda, Y.; Tsuda, M.; Anraku, Y.; Kita, S.; Yajima, H.; Tabata, K.; Guo, Z.; Hinay, A. A.; Yoshimatsu, K.; Yamamoto, Y.; Nagamoto, Y.; Asakura, H.; Nagashima, M.; Sadamasu, K.; Yoshimura, K."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Virological characteristics of the SARS-CoV-2 Omicron EG.5.1 variant\nAuthors: Tsujino, S.; Deguchi, S.; Nomai, T.; Padilla-Blanco, M.; Plianchaisuk, A.; Wang, L.; Begum, M. M.; Uriu, K.; Mizuma, K.; Nao, N.; Kojima, I.; Tsubo, T.; Li, J.; Matsumura, Y.; Nagao, M.; Oda, Y.; Tsuda, M.; Anraku, Y.; Kita, S.; Yajima, H.; Tabata, K.; Guo, Z.; Hinay, A. A.; Yoshimatsu, K.; Yamamoto, Y.; Nagamoto, Y.; Asakura, H.; Nagashima, M.; Sadamasu, K.; Yoshimura, K.",
  "keywords": [
    
  ],
  "articleBody": " Virological characteristics of the SARS-CoV-2 Omicron EG.5.1 variant\nAuthors: Tsujino, S.; Deguchi, S.; Nomai, T.; Padilla-Blanco, M.; Plianchaisuk, A.; Wang, L.; Begum, M. M.; Uriu, K.; Mizuma, K.; Nao, N.; Kojima, I.; Tsubo, T.; Li, J.; Matsumura, Y.; Nagao, M.; Oda, Y.; Tsuda, M.; Anraku, Y.; Kita, S.; Yajima, H.; Tabata, K.; Guo, Z.; Hinay, A. A.; Yoshimatsu, K.; Yamamoto, Y.; Nagamoto, Y.; Asakura, H.; Nagashima, M.; Sadamasu, K.; Yoshimura, K.; Nasser, H.; Jonathan, M.; Putri, O.; Kim, Y.; Chen, L.; Suzuki, R.; Tamura, T.; Maenaka, K.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Irie, T.; Matsuno, K.; Tanaka, S.; Ito, J.; Ikeda, T.; Takayama, K.; Za\nScore: 152.6, Published: 2023-10-21 DOI: 10.1101/2023.10.19.563209\nIn middle-late 2023, a sublineage of SARS-CoV-2 Omicron XBB, EG.5.1 (a progeny of XBB.1.9.2), is spreading rapidly around the world. Here, we performed multiscale investigations to reveal virological features of newly emerging EG.5.1 variant. Our phylogenetic-epidemic dynamics modeling suggested that two hallmark substitutions of EG.5.1, S:F456L and ORF9b:I5T, are critical to the increased viral fitness. Experimental investigations addressing the growth kinetics, sensitivity to clinically available antivirals, fusogenicity and pathogenicity of EG.5.1 suggested that the virological features of EG.5.1 is comparable to that of XBB.1.5. However, the cryo-electron microscopy reveals the structural difference between the spike proteins of EG.5.1 and XBB.1.5. We further assessed the impact of ORF9b:I5T on viral features, but it was almost negligible at least in our experimental setup. Our multiscale investigations provide the knowledge for understanding of the evolution trait of newly emerging pathogenic viruses in the human population.\nLifecycle of a predatory bacterium vampirizing its prey through the cell envelope and S-layer\nAuthors: Santin, Y. G.; Sogues, A.; Bourigault, Y.; Remaut, H. K.; Laloux, G.\nScore: 58.7, Published: 2023-10-26 DOI: 10.1101/2023.10.25.563945\nPredatory bacteria feed upon and kill other bacteria in various natural environments. Obligate epibiotic predators like Bdellovibrio exovorus consume their prey whilst remaining attached to the outside of the prey. How these predators achieve epibiotic feeding through the prey cell envelope has not been explored previously. Whereas the S-layer is the only proposed defensive structure against predatory bacteria, it remains unclear how this thin outer layer of the envelope might prevent epibiotic attacks. Similarly, the lifecycle of B. exovorus during the predator-prey interaction is poorly understood, with current models suggesting a binary division. Here we imaged the entire predatory lifecycle of B. exovorus and the fate of its Caulobacter crescentus prey by time-lapse microscopy and cryo-electron microscopy to monitor predator attack, growth and division and assess the impact of the S-layer on epibiotic predation. Our data reveal that B. exovorus uses non-binary division in a novel proliferation pattern that mainly generates three progenies. Moreover, we found that B. exovorus predates regardless of the presence of an S-layer, calling for revisiting its protective role against predators. Finally, our results indicate that epibiotic predation relies on the establishment of a secured junction between the prey and predator outer membranes, which must be resolved unilaterally to maintain cellular integrity of the predator departing from the prey surface.\nThe mycobacterial glycoside hydrolase LamH enables capsular arabinomannan release and stimulates growth\nAuthors: Franklin, A.; Layton, A. J.; Mize, T.; Salgueiro, V. C.; Sullivan, R.; Benedict, S. T.; Gurcha, S. S.; Anso, I.; Besra, G. S.; Banzhaf, M.; Lovering, A. L.; Williams, S. J.; Guerin, M. E.; Scott, N. E.; Prados-Rosales, R.; Lowe, E. C.; Moynihan, P. J.\nScore: 20.7, Published: 2023-10-26 DOI: 10.1101/2023.10.26.563968\nMycobacterial glycolipids are important cell envelope structures that drive host-pathogen interactions. Arguably, the most important amongst these are lipoarabinomannan (LAM) and its precursor, lipomannan (LM); both are trafficked out of the bacterium to the host via completely unknown mechanisms. An important class of exported LM/LAM is the capsular derivative of these molecules which is devoid of its lipid anchor. Here, we describe the identification of LamH, a glycoside hydrolase (GH) family 76 enzyme that specifically cleaves LM/LAM, driving its export to the capsule releasing its phosphatidyl-myo-inositol mannoside lipid anchor. Unexpectedly, we found that the catalytic activity of this enzyme is important for efficient exit from stationary phase cultures where arabinomannan acts as a signal for growth phase transition. Finally, we demonstrate that LamH is important for Mycobacterium tuberculosis survival in macrophages. These data provide a new framework for understanding the biological role of LAM in mycobacteria.\nA non-canonical Arp2/3 complex is essential for Plasmodium DNA segregation and transmission of malaria.\nAuthors: Hentzschel, F.; Jewanski, D.; Sokolowski, Y.; Agarwal, P.; Kraeft, A.; Hildenbrand, K.; Dorner, L. P.; Singer, M.; Frischknecht, F.; Marti, M.\nScore: 20.0, Published: 2023-10-26 DOI: 10.1101/2023.10.25.563799\nThe malaria-causing parasite Plasmodium has a complex life cycle involving both vertebrate and mosquito hosts. Sexual stages or gametocytes are the only stage competent for transmission to mosquitoes. Formation of flagellated male gametes from gametocytes requires rapid rounds of genome replication. Here we discovered a non-canonical Plasmodium actin-related protein 2/3 (Arp2/3) complex essential for DNA segregation during male gametogenesis. Plasmodium Arp2/3 dynamically localizes within the nucleus to the endomitotic spindles and interacts with a kinetochore protein. Deletion of key Arp2/3 subunits or interfering with actin polymerisation leads to the formation of sub-haploid male gametes and a complete block in transmission through delayed developmental arrest at the oocyst stage. Our work identified an evolutionary divergent protein complex in malaria parasites that offers potential targets for transmission-blocking interventions.\nInteractions between pili affect the outcome of bacterial competition driven by the type VI secretion system\nAuthors: Otto, S. B.; Servajean, R.; Lemopoulos, A.; Bitbol, A.-F.; Blokesch, M.\nScore: 19.4, Published: 2023-10-26 DOI: 10.1101/2023.10.25.564063\nThe bacterial type VI secretion system (T6SS) is a widespread, kin-discriminatory weapon capable of shaping microbial communities. Due to the system's dependency on contact, cellular interactions can lead to either competition or kin protection. Cell-to-cell contact is often accomplished via surface-exposed type IV pili (T4P). In Vibrio cholerae, these T4P facilitate specific interactions when the bacteria colonize natural chitinous surfaces. However, it has remained unclear whether and, if so, how these interactions affect the bacterium's T6SS-mediated killing. In this study, we demonstrate that pilus-mediated interactions can be harnessed to reduce the population of V. cholerae under liquid growth conditions in a T6SS-dependent manner. We also show that the naturally occurring diversity of pili determines the likelihood of cell-to-cell contact and, consequently, the extent of T6SS-mediated competition. To determine the factors that enable or hinder the T6SS's targeted reduction of competitors carrying pili, we developed a physics-grounded computational model for autoaggregation. Collectively, our research demonstrates that T4P involved in cell-to-cell contact can impose a selective burden when V. cholerae encounters non-kin cells that possess an active T6SS. Additionally, our study underscores the significance of T4P diversity in protecting closely related individuals from T6SS attacks through autoaggregation and spatial segregation.\nTranscription-driven DNA supercoiling activates bacterial chromatin at a distance\nAuthors: Figueroa-Bossi, N.; Fernandez-Fernandez, R.; Kerboriou, P.; Bouloc, P.; Casadesus, J.; Sanchez-Romero, M.-A.; Bossi, L.\nScore: 18.9, Published: 2023-10-26 DOI: 10.1101/2023.10.26.564199\nIn all living cells, genomic DNA is compacted through interactions with dedicated proteins and/or the formation of plectonemic coils. In bacteria, DNA compaction is achieved dynamically, coordinated with dense and constantly changing transcriptional activity. H-NS, a major bacterial nucleoid structuring protein, is of special interest due to its interplay with RNA polymerase. H-NS:DNA nucleoprotein filaments inhibit transcription initiation by RNA polymerase. However, the discovery that genes silenced by H-NS can be activated by transcription originating from neighboring regions has suggested that elongating RNA polymerases can disassemble H-NS:DNA filaments. In this study, we present evidence that transcription-induced counter-silencing does not require transcription to reach the silenced gene; rather, it exerts its effect at a distance. Counter-silencing is suppressed by introducing a DNA gyrase binding site within the intervening segment, suggesting that the long-range effect results from transcription-driven positive DNA supercoils diffusing toward the silenced gene. We propose a model wherein H-NS:DNA complexes form in vivo on negatively supercoiled DNA, with H-NS bridging the two arms of the plectoneme. Rotational diffusion of positive supercoils generated by neighbouring transcription will cause the H-NS-bound negatively-supercoiled plectoneme to unroll disrupting the H-NS bridges and releasing H-NS.\nNo evidence for highly pathogenic avian influenza virus H5N1 (clade 2.3.4.4b) in the Antarctic region during the austral summer 2022/23\nAuthors: Lisovski, S.; Gunther, A.; Dewar, M.; Anley, D.; Arce, R.; Ballard, G.; Belliure, J.; Boulinier, T.; Bennison, A.; Cary, C.; Catry, P.; Clessin, A.; Connan, M.; Cox, A.; Christina, J.; Elrod, M.; Ferreiro, I.; Fowler, Z.; Gamble, A.; Hurtado, J.; Jongsomjit, D.; Lejeune, M.; Lescroel, A.; Li, A.; Morandini, V.; Moratorio, G.; Militao, T.; Moreno, P.; Perbolianachis, P.; Pennycook, J.; Schmidt, A. E.; Smyth, L.; Soutullo, A.; Tornos, J.; Varsani, A.; Herzschuh, U.; Beer, M.; Wille, M.\nScore: 17.1, Published: 2023-10-24 DOI: 10.1101/2023.10.24.563692\nThe current highly pathogenic avian influenza H5N1 panzootic has profound impacts on wild birds. Herein, we compiled H5N1 surveillance from Antarctica and Sub-Antarctic Islands to ascertain whether HPAI was present in this region. Observations and HPAI testing indicated no incursion of the virus during the austral summer 2022/23.\nPhage-mediated resolution of genetic conflict alters the evolutionary trajectory of Pseudomonas aeruginosa lysogens\nAuthors: Suttenfield, L. C.; Rapti, Z.; Steinlein, A. C.; Vera, J. C.; Kim, T.; Whitaker, R. J.\nScore: 14.4, Published: 2023-10-26 DOI: 10.1101/2023.10.24.563318\nThe opportunistic human pathogen Pseudomonas aeruginosa is naturally infected by a large class of temperate, transposable, Mu-like phages. Phage escape leads to imperfect CRISPR matches between host spacers and viral protospacers, which fails to degrade phage genomes. We examined the genotypic and phenotypic diversity of P. aeruginosa PA14 populations as they resolve CRISPR autoimmunity, mediated by an imperfect CRISPR match to the Mu-like phage DMS3 prophage, and show that the lysogen evolution is distinct from the uninfected PA14 strain in the same environment. After 12 days of evolution, we measured a decrease in spontaneous induction in both exponential and stationary phase growth. Co-existing variation in spontaneous induction rates in exponential phase corresponded to a difference in the mutational modes of CRISPR self-targeting resolution, mediated either by host mutation or phage transposition. Multiple mutational modes to resolve genetic conflict between host and phage resulted in coexistence in evolved populations of single lysogens that maintained immunity to other phages and polylysogens that have lost immunity completely. None of the evolved lysogens share mutations with the uninfected PA14 strain evolving in culture. This work provides more insight into the nature of viral evolution in single ancestor lysogen populations like the CF lung, and highlights a new dimension to the role of lysogenic phages on the evolution of their hosts.\nAnti-CRISPR proteins trigger a burst of CRISPR-Cas9 expression that enhances phage defense\nAuthors: Workman, R. E.; Stoltzfus, M. J.; Keith, N. C.; Euler, C. W.; Bondy-Denomy, J.; Modell, J. W.\nScore: 13.7, Published: 2023-10-22 DOI: 10.1101/2023.10.22.562561\nCRISPR-Cas immune systems provide bacteria with adaptive immunity against bacteriophages, but they are often transcriptionally downregulated to mitigate autoimmunity. In some cases, CRISPR-Cas expression increases in response to a phage infection, but the mechanisms of induction are largely unknown, and it is unclear whether induction occurs strongly and quickly enough to benefit the bacterial host. In S. pyogenes, Cas9 is both an immune effector and autorepressor of CRISPR-Cas expression. Here, we show that phage-encoded anti-CRISPR proteins relieve Cas9 autorepression and trigger a rapid increase in CRISPR-Cas levels during a single phage infective cycle. As a result, fewer cells succumb to lysis leading to a striking survival benefit after multiple rounds of infection. CRISPR-Cas induction also reduces lysogeny, thereby limiting a route for horizontal gene transfer. Altogether, we show that Cas9 is not only a CRISPR-Cas effector and repressor, but also a phage sensor that can mount an anti-anti- CRISPR transcriptional response.\nSoil fungi remain active and invest in storage compounds during drought independent of future climate conditions\nAuthors: Canarini, A.; Fuchslueger, L.; Schnecker, J.; Metze, D.; Nelson, D.; Kahmen, A.; Watzka, M.; Poetsch, E. M.; Schaumberger, A.; Bahn, M.; Richter, A.\nScore: 9.2, Published: 2023-10-23 DOI: 10.1101/2023.10.23.563577\nMicrobial growth is central to soil carbon cycling. However, how microbial communities grow under climate change is still largely unexplored. In an experiment simulating future climate conditions (increased atmospheric CO2 and temperature) and drought, we traced 2H or 18O applied via water-vapor exchange into fatty acids or DNA, respectively, allowing to measure community- and group-level adjustments in soil microbial physiology (replication, storage product synthesis, and carbon use efficiency, CUE). We show, that while overall community-level growth decreased by half during drought, fungal growth remained stable demonstrating an astonishing resistance of fungal activity against soil moisture changes. In addition, fungal investment into storage triglycerides increased more than five-fold under drought. CUE (the balance between anabolism and catabolism) was unaffected by drought but decreased in future climate conditions. Our results highlight that accounting for different growth strategies can foster our understanding of soil microbial contribution to C cycling and feedback to climate change.\n",
  "wordCount" : "2122",
  "inLanguage": "en",
  "datePublished": "2023-10-29T10:37:40Z",
  "dateModified": "2023-10-29T10:37:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta"><span>updated on October 29, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.19.563209">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.19.563209" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.19.563209">
        <p class="paperTitle">Virological characteristics of the SARS-CoV-2 Omicron EG.5.1 variant</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.19.563209" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.19.563209" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tsujino, S.; Deguchi, S.; Nomai, T.; Padilla-Blanco, M.; Plianchaisuk, A.; Wang, L.; Begum, M. M.; Uriu, K.; Mizuma, K.; Nao, N.; Kojima, I.; Tsubo, T.; Li, J.; Matsumura, Y.; Nagao, M.; Oda, Y.; Tsuda, M.; Anraku, Y.; Kita, S.; Yajima, H.; Tabata, K.; Guo, Z.; Hinay, A. A.; Yoshimatsu, K.; Yamamoto, Y.; Nagamoto, Y.; Asakura, H.; Nagashima, M.; Sadamasu, K.; Yoshimura, K.; Nasser, H.; Jonathan, M.; Putri, O.; Kim, Y.; Chen, L.; Suzuki, R.; Tamura, T.; Maenaka, K.; The Genotype to Phenotype Japan (G2P-Japan) Consortium,  ; Irie, T.; Matsuno, K.; Tanaka, S.; Ito, J.; Ikeda, T.; Takayama, K.; Za</p>
        <p class="info">Score: 152.6, Published: 2023-10-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.19.563209' target='https://doi.org/10.1101/2023.10.19.563209'> 10.1101/2023.10.19.563209</a></p>
        <p class="abstract">In middle-late 2023, a sublineage of SARS-CoV-2 Omicron XBB, EG.5.1 (a progeny of XBB.1.9.2), is spreading rapidly around the world. Here, we performed multiscale investigations to reveal virological features of newly emerging EG.5.1 variant. Our phylogenetic-epidemic dynamics modeling suggested that two hallmark substitutions of EG.5.1, S:F456L and ORF9b:I5T, are critical to the increased viral fitness. Experimental investigations addressing the growth kinetics, sensitivity to clinically available antivirals, fusogenicity and pathogenicity of EG.5.1 suggested that the virological features of EG.5.1 is comparable to that of XBB.1.5. However, the cryo-electron microscopy reveals the structural difference between the spike proteins of EG.5.1 and XBB.1.5. We further assessed the impact of ORF9b:I5T on viral features, but it was almost negligible at least in our experimental setup. Our multiscale investigations provide the knowledge for understanding of the evolution trait of newly emerging pathogenic viruses in the human population.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.563945">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.563945" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.563945">
        <p class="paperTitle">Lifecycle of a predatory bacterium vampirizing its prey through the cell envelope and S-layer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.563945" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.563945" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Santin, Y. G.; Sogues, A.; Bourigault, Y.; Remaut, H. K.; Laloux, G.</p>
        <p class="info">Score: 58.7, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.563945' target='https://doi.org/10.1101/2023.10.25.563945'> 10.1101/2023.10.25.563945</a></p>
        <p class="abstract">Predatory bacteria feed upon and kill other bacteria in various natural environments. Obligate epibiotic predators like Bdellovibrio exovorus consume their prey whilst remaining attached to the outside of the prey. How these predators achieve epibiotic feeding through the prey cell envelope has not been explored previously. Whereas the S-layer is the only proposed defensive structure against predatory bacteria, it remains unclear how this thin outer layer of the envelope might prevent epibiotic attacks. Similarly, the lifecycle of B. exovorus during the predator-prey interaction is poorly understood, with current models suggesting a binary division. Here we imaged the entire predatory lifecycle of B. exovorus and the fate of its Caulobacter crescentus prey by time-lapse microscopy and cryo-electron microscopy to monitor predator attack, growth and division and assess the impact of the S-layer on epibiotic predation. Our data reveal that B. exovorus uses non-binary division in a novel proliferation pattern that mainly generates three progenies. Moreover, we found that B. exovorus predates regardless of the presence of an S-layer, calling for revisiting its protective role against predators. Finally, our results indicate that epibiotic predation relies on the establishment of a secured junction between the prey and predator outer membranes, which must be resolved unilaterally to maintain cellular integrity of the predator departing from the prey surface.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.563968">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.563968" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.563968">
        <p class="paperTitle">The mycobacterial glycoside hydrolase LamH enables capsular arabinomannan release and stimulates growth</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.563968" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.563968" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Franklin, A.; Layton, A. J.; Mize, T.; Salgueiro, V. C.; Sullivan, R.; Benedict, S. T.; Gurcha, S. S.; Anso, I.; Besra, G. S.; Banzhaf, M.; Lovering, A. L.; Williams, S. J.; Guerin, M. E.; Scott, N. E.; Prados-Rosales, R.; Lowe, E. C.; Moynihan, P. J.</p>
        <p class="info">Score: 20.7, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.563968' target='https://doi.org/10.1101/2023.10.26.563968'> 10.1101/2023.10.26.563968</a></p>
        <p class="abstract">Mycobacterial glycolipids are important cell envelope structures that drive host-pathogen interactions. Arguably, the most important amongst these are lipoarabinomannan (LAM) and its precursor, lipomannan (LM); both are trafficked out of the bacterium to the host via completely unknown mechanisms. An important class of exported LM/LAM is the capsular derivative of these molecules which is devoid of its lipid anchor. Here, we describe the identification of LamH, a glycoside hydrolase (GH) family 76 enzyme that specifically cleaves LM/LAM, driving its export to the capsule releasing its phosphatidyl-myo-inositol mannoside lipid anchor. Unexpectedly, we found that the catalytic activity of this enzyme is important for efficient exit from stationary phase cultures where arabinomannan acts as a signal for growth phase transition. Finally, we demonstrate that LamH is important for Mycobacterium tuberculosis survival in macrophages. These data provide a new framework for understanding the biological role of LAM in mycobacteria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.563799">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.563799" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.563799">
        <p class="paperTitle">A non-canonical Arp2/3 complex is essential for Plasmodium DNA segregation and transmission of malaria.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.563799" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.563799" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hentzschel, F.; Jewanski, D.; Sokolowski, Y.; Agarwal, P.; Kraeft, A.; Hildenbrand, K.; Dorner, L. P.; Singer, M.; Frischknecht, F.; Marti, M.</p>
        <p class="info">Score: 20.0, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.563799' target='https://doi.org/10.1101/2023.10.25.563799'> 10.1101/2023.10.25.563799</a></p>
        <p class="abstract">The malaria-causing parasite Plasmodium has a complex life cycle involving both vertebrate and mosquito hosts. Sexual stages or gametocytes are the only stage competent for transmission to mosquitoes. Formation of flagellated male gametes from gametocytes requires rapid rounds of genome replication. Here we discovered a non-canonical Plasmodium actin-related protein 2/3 (Arp2/3) complex essential for DNA segregation during male gametogenesis. Plasmodium Arp2/3 dynamically localizes within the nucleus to the endomitotic spindles and interacts with a kinetochore protein. Deletion of key Arp2/3 subunits or interfering with actin polymerisation leads to the formation of sub-haploid male gametes and a complete block in transmission through delayed developmental arrest at the oocyst stage. Our work identified an evolutionary divergent protein complex in malaria parasites that offers potential targets for transmission-blocking interventions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.564063">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.564063" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.564063">
        <p class="paperTitle">Interactions between pili affect the outcome of bacterial competition driven by the type VI secretion system</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.564063" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.564063" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Otto, S. B.; Servajean, R.; Lemopoulos, A.; Bitbol, A.-F.; Blokesch, M.</p>
        <p class="info">Score: 19.4, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.564063' target='https://doi.org/10.1101/2023.10.25.564063'> 10.1101/2023.10.25.564063</a></p>
        <p class="abstract">The bacterial type VI secretion system (T6SS) is a widespread, kin-discriminatory weapon capable of shaping microbial communities. Due to the system&#39;s dependency on contact, cellular interactions can lead to either competition or kin protection. Cell-to-cell contact is often accomplished via surface-exposed type IV pili (T4P). In Vibrio cholerae, these T4P facilitate specific interactions when the bacteria colonize natural chitinous surfaces. However, it has remained unclear whether and, if so, how these interactions affect the bacterium&#39;s T6SS-mediated killing. In this study, we demonstrate that pilus-mediated interactions can be harnessed to reduce the population of V. cholerae under liquid growth conditions in a T6SS-dependent manner. We also show that the naturally occurring diversity of pili determines the likelihood of cell-to-cell contact and, consequently, the extent of T6SS-mediated competition. To determine the factors that enable or hinder the T6SS&#39;s targeted reduction of competitors carrying pili, we developed a physics-grounded computational model for autoaggregation. Collectively, our research demonstrates that T4P involved in cell-to-cell contact can impose a selective burden when V. cholerae encounters non-kin cells that possess an active T6SS. Additionally, our study underscores the significance of T4P diversity in protecting closely related individuals from T6SS attacks through autoaggregation and spatial segregation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564199">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564199" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564199">
        <p class="paperTitle">Transcription-driven DNA supercoiling activates bacterial chromatin at a distance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564199" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564199" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Figueroa-Bossi, N.; Fernandez-Fernandez, R.; Kerboriou, P.; Bouloc, P.; Casadesus, J.; Sanchez-Romero, M.-A.; Bossi, L.</p>
        <p class="info">Score: 18.9, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564199' target='https://doi.org/10.1101/2023.10.26.564199'> 10.1101/2023.10.26.564199</a></p>
        <p class="abstract">In all living cells, genomic DNA is compacted through interactions with dedicated proteins and/or the formation of plectonemic coils. In bacteria, DNA compaction is achieved dynamically, coordinated with dense and constantly changing transcriptional activity. H-NS, a major bacterial nucleoid structuring protein, is of special interest due to its interplay with RNA polymerase. H-NS:DNA nucleoprotein filaments inhibit transcription initiation by RNA polymerase. However, the discovery that genes silenced by H-NS can be activated by transcription originating from neighboring regions has suggested that elongating RNA polymerases can disassemble H-NS:DNA filaments. In this study, we present evidence that transcription-induced counter-silencing does not require transcription to reach the silenced gene; rather, it exerts its effect at a distance. Counter-silencing is suppressed by introducing a DNA gyrase binding site within the intervening segment, suggesting that the long-range effect results from transcription-driven positive DNA supercoils diffusing toward the silenced gene. We propose a model wherein H-NS:DNA complexes form in vivo on negatively supercoiled DNA, with H-NS bridging the two arms of the plectoneme. Rotational diffusion of positive supercoils generated by neighbouring transcription will cause the H-NS-bound negatively-supercoiled plectoneme to unroll disrupting the H-NS bridges and releasing H-NS.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563692">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563692" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563692">
        <p class="paperTitle">No evidence for highly pathogenic avian influenza virus H5N1 (clade 2.3.4.4b) in the Antarctic region during the austral summer 2022/23</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563692" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563692" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lisovski, S.; Gunther, A.; Dewar, M.; Anley, D.; Arce, R.; Ballard, G.; Belliure, J.; Boulinier, T.; Bennison, A.; Cary, C.; Catry, P.; Clessin, A.; Connan, M.; Cox, A.; Christina, J.; Elrod, M.; Ferreiro, I.; Fowler, Z.; Gamble, A.; Hurtado, J.; Jongsomjit, D.; Lejeune, M.; Lescroel, A.; Li, A.; Morandini, V.; Moratorio, G.; Militao, T.; Moreno, P.; Perbolianachis, P.; Pennycook, J.; Schmidt, A. E.; Smyth, L.; Soutullo, A.; Tornos, J.; Varsani, A.; Herzschuh, U.; Beer, M.; Wille, M.</p>
        <p class="info">Score: 17.1, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563692' target='https://doi.org/10.1101/2023.10.24.563692'> 10.1101/2023.10.24.563692</a></p>
        <p class="abstract">The current highly pathogenic avian influenza H5N1 panzootic has profound impacts on wild birds. Herein, we compiled H5N1 surveillance from Antarctica and Sub-Antarctic Islands to ascertain whether HPAI was present in this region. Observations and HPAI testing indicated no incursion of the virus during the austral summer 2022/23.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563318">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563318" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563318">
        <p class="paperTitle">Phage-mediated resolution of genetic conflict alters the evolutionary trajectory of Pseudomonas aeruginosa lysogens</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563318" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563318" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Suttenfield, L. C.; Rapti, Z.; Steinlein, A. C.; Vera, J. C.; Kim, T.; Whitaker, R. J.</p>
        <p class="info">Score: 14.4, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563318' target='https://doi.org/10.1101/2023.10.24.563318'> 10.1101/2023.10.24.563318</a></p>
        <p class="abstract">The opportunistic human pathogen Pseudomonas aeruginosa is naturally infected by a large class of temperate, transposable, Mu-like phages. Phage escape leads to imperfect CRISPR matches between host spacers and viral protospacers, which fails to degrade phage genomes. We examined the genotypic and phenotypic diversity of P. aeruginosa PA14 populations as they resolve CRISPR autoimmunity, mediated by an imperfect CRISPR match to the Mu-like phage DMS3 prophage, and show that the lysogen evolution is distinct from the uninfected PA14 strain in the same environment. After 12 days of evolution, we measured a decrease in spontaneous induction in both exponential and stationary phase growth. Co-existing variation in spontaneous induction rates in exponential phase corresponded to a difference in the mutational modes of CRISPR self-targeting resolution, mediated either by host mutation or phage transposition. Multiple mutational modes to resolve genetic conflict between host and phage resulted in coexistence in evolved populations of single lysogens that maintained immunity to other phages and polylysogens that have lost immunity completely. None of the evolved lysogens share mutations with the uninfected PA14 strain evolving in culture. This work provides more insight into the nature of viral evolution in single ancestor lysogen populations like the CF lung, and highlights a new dimension to the role of lysogenic phages on the evolution of their hosts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.22.562561">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.22.562561" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.22.562561">
        <p class="paperTitle">Anti-CRISPR proteins trigger a burst of CRISPR-Cas9 expression that enhances phage defense</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.22.562561" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.22.562561" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Workman, R. E.; Stoltzfus, M. J.; Keith, N. C.; Euler, C. W.; Bondy-Denomy, J.; Modell, J. W.</p>
        <p class="info">Score: 13.7, Published: 2023-10-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.22.562561' target='https://doi.org/10.1101/2023.10.22.562561'> 10.1101/2023.10.22.562561</a></p>
        <p class="abstract">CRISPR-Cas immune systems provide bacteria with adaptive immunity against bacteriophages, but they are often transcriptionally downregulated to mitigate autoimmunity. In some cases, CRISPR-Cas expression increases in response to a phage infection, but the mechanisms of induction are largely unknown, and it is unclear whether induction occurs strongly and quickly enough to benefit the bacterial host. In S. pyogenes, Cas9 is both an immune effector and autorepressor of CRISPR-Cas expression. Here, we show that phage-encoded anti-CRISPR proteins relieve Cas9 autorepression and trigger a rapid increase in CRISPR-Cas levels during a single phage infective cycle. As a result, fewer cells succumb to lysis leading to a striking survival benefit after multiple rounds of infection. CRISPR-Cas induction also reduces lysogeny, thereby limiting a route for horizontal gene transfer. Altogether, we show that Cas9 is not only a CRISPR-Cas effector and repressor, but also a phage sensor that can mount an anti-anti- CRISPR transcriptional response.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563577">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563577" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563577">
        <p class="paperTitle">Soil fungi remain active and invest in storage compounds during drought independent of future climate conditions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563577" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563577" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Canarini, A.; Fuchslueger, L.; Schnecker, J.; Metze, D.; Nelson, D.; Kahmen, A.; Watzka, M.; Poetsch, E. M.; Schaumberger, A.; Bahn, M.; Richter, A.</p>
        <p class="info">Score: 9.2, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563577' target='https://doi.org/10.1101/2023.10.23.563577'> 10.1101/2023.10.23.563577</a></p>
        <p class="abstract">Microbial growth is central to soil carbon cycling. However, how microbial communities grow under climate change is still largely unexplored. In an experiment simulating future climate conditions (increased atmospheric CO2 and temperature) and drought, we traced 2H or 18O applied via water-vapor exchange into fatty acids or DNA, respectively, allowing to measure community- and group-level adjustments in soil microbial physiology (replication, storage product synthesis, and carbon use efficiency, CUE). We show, that while overall community-level growth decreased by half during drought, fungal growth remained stable demonstrating an astonishing resistance of fungal activity against soil moisture changes. In addition, fungal investment into storage triglycerides increased more than five-fold under drought. CUE (the balance between anabolism and catabolism) was unaffected by drought but decreased in future climate conditions. Our results highlight that accounting for different growth strategies can foster our understanding of soil microbial contribution to C cycling and feedback to climate change.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
