<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Neutralization Escape by SARS-CoV-2 Omicron Subvariant BA.2.86
Authors: Lasrado, N.; Collier, A.-r.; Hachmann, N.; Miller, J.; Rowe, M.; Schonberg, E.; Rodrigues, S.; LaPiana, A.; Patio, R.; Anand, T.; Fisher, J.; Mazurek, C.; Guan, R.; Wagh, K.; Theiler, J.; Korber, B.; Barouch, D. H.
Score: 796.9, Published: 2023-09-05 DOI: 10.1101/2023.09.04.556272
The continued evolution of SARS-CoV-2 may lead to evasion of vaccine immunity and natural immunity. A highly mutated Omicron variant BA.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Neutralization Escape by SARS-CoV-2 Omicron Subvariant BA.2.86
Authors: Lasrado, N.; Collier, A.-r.; Hachmann, N.; Miller, J.; Rowe, M.; Schonberg, E.; Rodrigues, S.; LaPiana, A.; Patio, R.; Anand, T.; Fisher, J.; Mazurek, C.; Guan, R.; Wagh, K.; Theiler, J.; Korber, B.; Barouch, D. H.
Score: 796.9, Published: 2023-09-05 DOI: 10.1101/2023.09.04.556272
The continued evolution of SARS-CoV-2 may lead to evasion of vaccine immunity and natural immunity. A highly mutated Omicron variant BA." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-17T10:36:50+00:00" />
<meta property="article:modified_time" content="2023-09-17T10:36:50+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Neutralization Escape by SARS-CoV-2 Omicron Subvariant BA.2.86
Authors: Lasrado, N.; Collier, A.-r.; Hachmann, N.; Miller, J.; Rowe, M.; Schonberg, E.; Rodrigues, S.; LaPiana, A.; Patio, R.; Anand, T.; Fisher, J.; Mazurek, C.; Guan, R.; Wagh, K.; Theiler, J.; Korber, B.; Barouch, D. H.
Score: 796.9, Published: 2023-09-05 DOI: 10.1101/2023.09.04.556272
The continued evolution of SARS-CoV-2 may lead to evasion of vaccine immunity and natural immunity. A highly mutated Omicron variant BA."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Neutralization Escape by SARS-CoV-2 Omicron Subvariant BA.2.86\nAuthors: Lasrado, N.; Collier, A.-r.; Hachmann, N.; Miller, J.; Rowe, M.; Schonberg, E.; Rodrigues, S.; LaPiana, A.; Patio, R.; Anand, T.; Fisher, J.; Mazurek, C.; Guan, R.; Wagh, K.; Theiler, J.; Korber, B.; Barouch, D. H.\nScore: 796.9, Published: 2023-09-05 DOI: 10.1101/2023.09.04.556272\nThe continued evolution of SARS-CoV-2 may lead to evasion of vaccine immunity and natural immunity. A highly mutated Omicron variant BA.",
  "keywords": [
    
  ],
  "articleBody": " Neutralization Escape by SARS-CoV-2 Omicron Subvariant BA.2.86\nAuthors: Lasrado, N.; Collier, A.-r.; Hachmann, N.; Miller, J.; Rowe, M.; Schonberg, E.; Rodrigues, S.; LaPiana, A.; Patio, R.; Anand, T.; Fisher, J.; Mazurek, C.; Guan, R.; Wagh, K.; Theiler, J.; Korber, B.; Barouch, D. H.\nScore: 796.9, Published: 2023-09-05 DOI: 10.1101/2023.09.04.556272\nThe continued evolution of SARS-CoV-2 may lead to evasion of vaccine immunity and natural immunity. A highly mutated Omicron variant BA.2.86 has recently been identified with over 30 amino acid changes in Spike compared with BA.2 and XBB.1.5. As of September 4, 2023, BA.2.86 has been identified in 37 sequences from 10 countries, which is likely an underestimate due to limited surveillance. The ability of BA.2.86 to evade NAbs compared with other currently circulating Omicron variants remains unknown. Our data show that NAb responses to BA.2.86 were lower than to BA.2 but were comparable or slightly higher than to the current circulating recombinant variants XBB.1.5, XBB.1.16, EG.5, EG.5.1, and FL.1.5.1.\nTransmissibility, infectivity, and immune resistance of the SARS-CoV-2 BA.2.86 variant\nAuthors: Uriu, K.; Ito, J.; Kosugi, Y.; Tanaka, Y. L.; Mugita, Y.; Guo, Z.; Hinay, A. A.; Putri, O.; Kim, Y.; Shimizu, R.; Begum, M. M.; Jonathan, M.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Saito, A.; Ikeda, T.; Sato, K.\nScore: 345.5, Published: 2023-09-07 DOI: 10.1101/2023.09.07.556636\nIn September 2023, the SARS-CoV-2 XBB descendants, such as XBB.1.5 and EG.5.1 (originally XBB.1.9.2.5.1), are predominantly circulating worldwide. Unexpectedly, however, a lineage distinct from XBB was identified and named BA.2.86 on August 14, 2023. Notably, BA.2.86 bears more than 30 mutations in the spike (S) protein when compared to XBB and the parental BA.2, and many of them are assumed to be associated with immune evasion. Although the number of reported cases is low (68 sequences have been reported as of 7 September 2023), BA.2.86 has been detected in several continents (Europe, North America and Africa), suggesting that this variant may be spreading silently worldwide. On 17 August 2023, the WHO designated BA.2.86 as a variant under monitoring. Here we show evidence suggesting that BA.2.86 potentially has greater fitness than current circulating XBB variants including EG.5.1. The pseudovirus assay showed that the infectivity of BA.2.86 was significantly lower than that of B.1.1 and EG.5.1, suggesting that the increased fitness of BA.2.86 is not due to the increased infectivity. We then performed a neutralization assay using XBB breakthrough infection sera to address whether BA.2.86 evades the antiviral effect of the humoral immunity induced by XBB subvariants. The 50% neutralization titer of XBB BTI sera against BA.2.86 was significantly (1.6-fold) lower than those against EG.5.1. The sera obtained from individuals vaccinated with 3rd-dose monovalent, 4th-dose monovalent, BA.1 bivalent, and BA.5 bivalent mRNA vaccines exhibited very little or no antiviral effects against BA.2.86. Moreover, the three monoclonal antibodies (Bebtelovimab, Sotrovimab and Cilgavimab), which worked against the parental BA.2, did not exhibit antiviral effects against BA.2.86. These results suggest that BA.2.86 is one of the most highly immune evasive variants ever.\nLess neutralization evasion of SARS-CoV-2 BA.2.86 than XBB sublineages and CH.1.1\nAuthors: Hu, Y.; Zou, J.; Kurhade, C.; Deng, X.; Chang, H. C.; Kim, D. K.; Shi, P.-Y.; Ren, P.; Xie, X.\nScore: 76.5, Published: 2023-09-11 DOI: 10.1101/2023.09.10.557047\nThe highly mutated BA.2.86, with over 30 spike protein mutations in comparison to Omicron BA.2 and XBB.1.5 variants, has raised concerns about its potential to evade COVID-19 vaccination or prior SARS-CoV-2 infection-elicited immunity. In this study, we employ a live SARS-CoV-2 neutralization assay to compare the neutralization evasion ability of BA.2.86 with other emerged SARS-CoV-2 subvariants, including BA.2-derived CH.1.1, Delta-Omicron recombinant XBC.1.6, and XBB descendants XBB.1.5, XBB.1.16, XBB.2.3, EG.5.1 and FL.1.5.1. Our results show that BA.2.86 is less neutralization evasive than XBB sublineages. Among all the tested variants, CH.1.1 exhibits the greatest neutralization evasion. In comparison to XBB.1.5, the more recent XBB descendants, particularly EG.5.1 and FL.1.5.1, display increased resistance to neutralization induced by parental COVID-19 mRNA vaccine and a BA.5-Bivalent-booster. In contrast, XBC.1.6 shows a slight reduction but remains comparable sensitivity to neutralization when compared to BA.5. Furthermore, a recent XBB.1.5-breakthrough infection significantly enhances the breadth and potency of cross-neutralization. These findings reinforce the expectation that the upcoming XBB.1.5 mRNA vaccine would likely boost the neutralization of currently circulating variants, while also underscoring the critical importance of ongoing surveillance to monitor the evolution and immune evasion potential of SARS-CoV-2 variants.\nAsgard archaea defense systems and their roles in the origin of immunity in eukaryotes\nAuthors: Leao, P.; Little, M. E.; Appler, K. E.; Sahaya, D.; Aguilar-Pine, E.; Currie, K.; Finkelstein, I. J.; De Anda, V.; Baker, B. J.\nScore: 74.7, Published: 2023-09-13 DOI: 10.1101/2023.09.13.557551\nImmune systems are integral to survival against viral infection. Recently, dozens of new anti-viral systems have been characterized in bacteria1. Some of these systems are present in eukaryotes and appear to have originated in prokaryotes. However, little is known about these defense mechanisms in archaea. Here, we identified 2,610 complete defense systems in archaea related to eukaryotes, the Asgardarchaeota2. These comprise 89 unique systems, including argonaute, NLR, mokosh, viperin, lassamu, and CBASS. Asgard viperin (asVip) and argonaute (asAgo) proteins are present at high frequencies compared to bacteria and have structural homology to eukaryotes. Phylogenetic analyses revealed asVips are ancestral eukaryotic proteins. Heterologous expression of asVips in bacteria, including the lineage closest to eukaryotes, Hodarchaeales, showed robust anti-phage activity. Eukaryotic-and bacterial-argonaute proteins appear to have originated in the Asgardarchaeota, and have ancient structural characteristics. AsAgos appear to have argonaute-PIWI domains which are key components of the RNA interference (RNAi) in eukaryotes. Characterization of hundreds of new defense systems in the Asgardarchaeota revealed these archaea played important roles in the innovation of viral protection in eukaryotes. Given their relationship to eukaryotes, these defense systems may have applications in biomedicine and biotechnology.\nImmune Evasion, Infectivity, and Fusogenicity of SARS-CoV-2 Omicron BA.2.86 and FLip Variants\nAuthors: Qu, P.; Xu, K.; Faraone, J. N.; Goodarzi, N.; Zheng, Y.-M.; Carlin, C.; Bednash, J. S.; Horowitz, J. C.; Mallampalli, R. K.; Saif, L. J.; Oltz, E. M.; Jones, D.; Gumina, R. J.; Liu, S.-L.\nScore: 66.9, Published: 2023-09-12 DOI: 10.1101/2023.09.11.557206\nEvolution of SARS-CoV-2 requires the reassessment of current vaccine measures. Here, we characterized BA.2.86 and the XBB-lineage variant FLip by investigating their neutralization alongside D614G, BA.1, BA.2, BA.4/5, XBB.1.5, and EG.5.1 by sera from 3-dose vaccinated and bivalent vaccinated healthcare workers, XBB.1.5-wave infected first responders, and monoclonal antibody (mAb) S309. We assessed the biology of the variant Spikes by measuring viral infectivity and membrane fusogenicity. BA.2.86 is less immune evasive compared to FLip and other XBB variants, consistent with antigenic distances. Importantly, distinct from XBB variants, mAb S309 was unable to neutralize BA.2.86, likely due to a D339H mutation based on modeling. BA.2.86 had relatively high fusogenicity and infectivity in CaLu-3 cells but low fusion and infectivity in 293T-ACE2 cells compared to some XBB variants, suggesting a potentially differences conformational stability of BA.2.86 Spike. Overall, our study underscores the importance of SARS-CoV-2 variant surveillance and the need for updated COVID-19 vaccines.\nMultivariate comparison of taxonomic, chemical and technical data from 80 full-scale an-aerobic digester-related systems\nAuthors: Otto, P.; Puchol-Royo, R.; Ortega-Legarreta, A.; Tanner, K.; Tideman, J.; de Vries, S.-J.; Pascual, J.; Porcar, M.; Latorre-Perez, A.; Abendroth, C.\nScore: 67.2, Published: 2023-09-08 DOI: 10.1101/2023.09.08.556802\nThis study represents one of the most comprehensive characterisations of the anaerobic digestion (AD) microbiome with 80 samples from 45 different large-scale reactors in three coun-tries. Technical, chemical and taxonomic data was thoroughly collected, analyzed and correlated to identify the main drivers of AD processes. Our results showed that MBA03, Proteiniphilum, a member of Dethiobacteraceae, and Caldicoprobacter were present in all the samples, while Meth-anosarcina was the most abundant and prevalent archaea. Two distinct bacterial clusters were iden-tified by correlating microbial abundances. One was correlated with hydrogenotrophic and the other with acetoclastic methanogenesis. Organic acids, ammonia, nitrogen, COD and the trace el-ements Fe, Mo, and the macro nutrient P had the greatest impact on AD microbiomes. Temperature, reactor type and substrate also influenced the formation of a specialized microbial community. Overall, this work sheds light on the microbial key players involved in AD and evaluates how they are affected by technical and chemical parameters. HighlightsO_LIGeneration of a holistic dataset of chemical, taxonomic and technical parameters of 80 large-scale anaerobic digestion systems. C_LIO_LIIdentification of a core microbiome comprising MBA03, Proteiniphilum, an uncultured or-ganism from the Dethiobacteraceae family, and the Caldicoprobacter family. C_LIO_LIIdentification of the main influencing parameters that determine the occurrence and non-occurrence of specific genera. C_LIO_LICorrelation of bacterial taxa with hydrogenotrophic and/or acetoclastic archaea. C_LI\nAn obligate aerobe adapts to hypoxia by hybridising fermentation with carbon storage\nAuthors: Gillett, D. L.; Hutchinson, T.; Mudaliyar, M.; Watts, T. D.; Wong, W. W.; Locop, J.; Jimenez, L.; Hanchapola, I.; Lee, H.-C.; Tanuwidjaya, E.; Steele, J. R.; Schittenhelm, R. B.; Barlow, C.; Grinter, R.; Ghosal, D.; Cook, P.; Greening, C.\nScore: 23.6, Published: 2023-09-12 DOI: 10.1101/2023.09.11.557286\nIn soil ecosystems, obligately aerobic bacteria survive oxygen deprivation (hypoxia) by entering non-replicative persistent states. Little is known about how these bacteria rewire their metabolism to stay viable in these states. The model obligate aerobe Mycobacterium smegmatis maintains redox homeostasis during hypoxia by mediating fermentative hydrogen production. However, the fate of organic carbon during fermentation, and the associated remodeling of carbon metabolism, is unresolved. Here we systematically profiled the metabolism of M. smegmatis during aerobic growth, hypoxic persistence, and the transition between these states. Using differential isotope labelling, and paired metabolomics and proteomics, we observed rerouting of central carbon metabolism through the pentose phosphate pathway and Entner-Doudoroff pathway during hypoxia. We show that M. smegmatis excretes high levels of hydrogen concurrently with upregulating triacylglyceride synthases and accumulating glycerides as carbon stores. Using electron cryotomography (cryo-ET), we observed the presence of large spheroid structures consistent with the appearance of lipid droplets. Thus, in contrast to obligately and facultative anaerobic fermentative bacteria, M. smegmatis stores rather than excretes organic carbon during hypoxia. This novel hybrid metabolism likely provides a competitive advantage in resource-variable environments by allowing M. smegmatis to simultaneously dispose excess reductant during hypoxia and maintain carbon stores to rapidly resume growth upon reoxygenation.\nMultiple evolutionary pathways lead to vancomycin resistance in Clostridioides difficile\nAuthors: Buddle, J. E.; Wright, R. C. T.; Turner, C. E.; Chaudhuri, R. R.; Brockhurst, M. A.; Fagan, R. P.\nScore: 23.1, Published: 2023-09-15 DOI: 10.1101/2023.09.15.557922\nClostridioides difficile is an important human pathogen, for which there are very limited treatment options, primarily the glycopeptide antibiotic vancomycin. In recent years vancomycin resistance has emerged as a serious problem in several Gram positive pathogens, but high level resistance has yet to be reported for C. difficile, although it is not known if this is due to constraints upon resistance evolution in this species. Here we show that resistance to vancomycin can evolve rapidly under ramping selection but is accompanied by severe fitness costs and pleiotropic trade-offs, including sporulation defects that would be expected to severely impact transmission. We identified two distinct pathways to resistance, both of which are predicted to result in changes to the muropeptide terminal D-Ala-D-Ala that is the primary target of vancomycin. One of these pathways involves a previously uncharacterised D,D-carboxypeptidase, expression of which is controlled by a dedicated two-component signal transduction system. Our findings suggest that while C. difficile is capable of evolving high-level vancomycin resistance, this outcome may be limited clinically due to pleiotropic effects on key pathogenicity trains. Moreover, our data provide a mutational roadmap to inform genomic surveillance.\nPfMORC protein regulates chromatin accessibility and transcriptional repression in the human malaria parasite, P. falciparum.\nAuthors: Chahine, Z. M.; Gupta, M.; Lenz, T.; Hollin, T.; Abel, S.; Banks, C.; Saraf, A.; Prudhomme, J.; Florens, L.; Le Roch, K. G.\nScore: 19.9, Published: 2023-09-11 DOI: 10.1101/2023.09.11.557253\nThe environmental challenges the human malaria parasite, Plasmodium falciparum, faces during its progression into its various lifecycle stages warrant the use of effective and highly regulated access to chromatin for transcriptional regulation. Microrchidia (MORC) proteins have been implicated in DNA compaction and gene silencing across plant and animal kingdoms. Accumulating evidence has shed light into the role MORC protein plays as a transcriptional switch in apicomplexan parasites. In this study, using CRISPR/Cas9 genome editing tool along with complementary molecular and genomics approaches, we demonstrate that PfMORC not only modulates chromatin structure and heterochromatin formation throughout the parasite erythrocytic cycle, but is also essential to the parasite survival. Chromatin immunoprecipitation followed by deep sequencing (ChIP-seq) experiments suggest that PfMORC binds to not only sub-telomeric regions and genes involved in antigenic variation but is also most likely a key modulator of stage transition. Protein knockdown experiments followed by chromatin conformation capture (Hi-C) studies indicate that downregulation of PfMORC induces the collapse of the parasite heterochromatin structure leading to its death. All together these findings confirm that PfMORC plays a crucial role in chromatin structure and gene regulation, validating this factor as a strong candidate for novel antimalarial strategies.\nGenomic analysis of Coccomyxa viridis, a common low-abundance alga associated with lichen symbioses\nAuthors: Tagirdzhanova, G.; Scharnagl, K.; Yan, X.; Talbot, N. J.\nScore: 17.1, Published: 2023-09-13 DOI: 10.1101/2023.09.13.557537\nLichen symbiosis is centered around a relationship between a fungus and a photosynthetic microbe, usually a green alga. In addition to their main photosynthetic partner (the photobiont), lichen symbioses can contain additional algae present in low abundance. The biology of these algae and the way they interact with the rest of lichen symbionts remains largely unknown. Here we present the first genome sequence of a non-photobiont lichen-associated alga. Coccomyxa viridis was unexpectedly found in 12% of publicly available lichen metagenomes. With few exceptions, members of the Coccomyxa viridis clade occur in lichens as non-photobionts, potentially growing in thalli endophytically. The 45.7 Mbp genome of C. viridis was assembled into 18 near chromosome-level contigs, making it one of the most contiguous genomic assemblies for any lichen-associated algae. Comparing the C. viridis genome to its close relatives revealed the presence of traits associated with the lichen lifestyle. The genome of C. viridis provides a new resource for exploring the evolution of the lichen symbiosis, and how symbiotic lifestyles shaped evolution in green algae.\n",
  "wordCount" : "2343",
  "inLanguage": "en",
  "datePublished": "2023-09-17T10:36:50Z",
  "dateModified": "2023-09-17T10:36:50Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta"><span>updated on September 17, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.04.556272">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.04.556272" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.04.556272">
        <p class="paperTitle">Neutralization Escape by SARS-CoV-2 Omicron Subvariant BA.2.86</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.04.556272" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.04.556272" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lasrado, N.; Collier, A.-r.; Hachmann, N.; Miller, J.; Rowe, M.; Schonberg, E.; Rodrigues, S.; LaPiana, A.; Patio, R.; Anand, T.; Fisher, J.; Mazurek, C.; Guan, R.; Wagh, K.; Theiler, J.; Korber, B.; Barouch, D. H.</p>
        <p class="info">Score: 796.9, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.04.556272' target='https://doi.org/10.1101/2023.09.04.556272'> 10.1101/2023.09.04.556272</a></p>
        <p class="abstract">The continued evolution of SARS-CoV-2 may lead to evasion of vaccine immunity and natural immunity. A highly mutated Omicron variant BA.2.86 has recently been identified with over 30 amino acid changes in Spike compared with BA.2 and XBB.1.5. As of September 4, 2023, BA.2.86 has been identified in 37 sequences from 10 countries, which is likely an underestimate due to limited surveillance. The ability of BA.2.86 to evade NAbs compared with other currently circulating Omicron variants remains unknown. Our data show that NAb responses to BA.2.86 were lower than to BA.2 but were comparable or slightly higher than to the current circulating recombinant variants XBB.1.5, XBB.1.16, EG.5, EG.5.1, and FL.1.5.1.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556636">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556636" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556636">
        <p class="paperTitle">Transmissibility, infectivity, and immune resistance of the SARS-CoV-2 BA.2.86 variant</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556636" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556636" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Uriu, K.; Ito, J.; Kosugi, Y.; Tanaka, Y. L.; Mugita, Y.; Guo, Z.; Hinay, A. A.; Putri, O.; Kim, Y.; Shimizu, R.; Begum, M. M.; Jonathan, M.; The Genotype to Phenotype Japan (G2P-Japan) Consortium,  ; Saito, A.; Ikeda, T.; Sato, K.</p>
        <p class="info">Score: 345.5, Published: 2023-09-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556636' target='https://doi.org/10.1101/2023.09.07.556636'> 10.1101/2023.09.07.556636</a></p>
        <p class="abstract">In September 2023, the SARS-CoV-2 XBB descendants, such as XBB.1.5 and EG.5.1 (originally XBB.1.9.2.5.1), are predominantly circulating worldwide. Unexpectedly, however, a lineage distinct from XBB was identified and named BA.2.86 on August 14, 2023. Notably, BA.2.86 bears more than 30 mutations in the spike (S) protein when compared to XBB and the parental BA.2, and many of them are assumed to be associated with immune evasion. Although the number of reported cases is low (68 sequences have been reported as of 7 September 2023), BA.2.86 has been detected in several continents (Europe, North America and Africa), suggesting that this variant may be spreading silently worldwide. On 17 August 2023, the WHO designated BA.2.86 as a variant under monitoring. Here we show evidence suggesting that BA.2.86 potentially has greater fitness than current circulating XBB variants including EG.5.1. The pseudovirus assay showed that the infectivity of BA.2.86 was significantly lower than that of B.1.1 and EG.5.1, suggesting that the increased fitness of BA.2.86 is not due to the increased infectivity. We then performed a neutralization assay using XBB breakthrough infection sera to address whether BA.2.86 evades the antiviral effect of the humoral immunity induced by XBB subvariants. The 50% neutralization titer of XBB BTI sera against BA.2.86 was significantly (1.6-fold) lower than those against EG.5.1. The sera obtained from individuals vaccinated with 3rd-dose monovalent, 4th-dose monovalent, BA.1 bivalent, and BA.5 bivalent mRNA vaccines exhibited very little or no antiviral effects against BA.2.86. Moreover, the three monoclonal antibodies (Bebtelovimab, Sotrovimab and Cilgavimab), which worked against the parental BA.2, did not exhibit antiviral effects against BA.2.86. These results suggest that BA.2.86 is one of the most highly immune evasive variants ever.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.10.557047">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.10.557047" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.10.557047">
        <p class="paperTitle">Less neutralization evasion of SARS-CoV-2 BA.2.86 than XBB sublineages and CH.1.1</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.10.557047" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.10.557047" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hu, Y.; Zou, J.; Kurhade, C.; Deng, X.; Chang, H. C.; Kim, D. K.; Shi, P.-Y.; Ren, P.; Xie, X.</p>
        <p class="info">Score: 76.5, Published: 2023-09-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.10.557047' target='https://doi.org/10.1101/2023.09.10.557047'> 10.1101/2023.09.10.557047</a></p>
        <p class="abstract">The highly mutated BA.2.86, with over 30 spike protein mutations in comparison to Omicron BA.2 and XBB.1.5 variants, has raised concerns about its potential to evade COVID-19 vaccination or prior SARS-CoV-2 infection-elicited immunity. In this study, we employ a live SARS-CoV-2 neutralization assay to compare the neutralization evasion ability of BA.2.86 with other emerged SARS-CoV-2 subvariants, including BA.2-derived CH.1.1, Delta-Omicron recombinant XBC.1.6, and XBB descendants XBB.1.5, XBB.1.16, XBB.2.3, EG.5.1 and FL.1.5.1. Our results show that BA.2.86 is less neutralization evasive than XBB sublineages. Among all the tested variants, CH.1.1 exhibits the greatest neutralization evasion. In comparison to XBB.1.5, the more recent XBB descendants, particularly EG.5.1 and FL.1.5.1, display increased resistance to neutralization induced by parental COVID-19 mRNA vaccine and a BA.5-Bivalent-booster. In contrast, XBC.1.6 shows a slight reduction but remains comparable sensitivity to neutralization when compared to BA.5. Furthermore, a recent XBB.1.5-breakthrough infection significantly enhances the breadth and potency of cross-neutralization. These findings reinforce the expectation that the upcoming XBB.1.5 mRNA vaccine would likely boost the neutralization of currently circulating variants, while also underscoring the critical importance of ongoing surveillance to monitor the evolution and immune evasion potential of SARS-CoV-2 variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.13.557551">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.13.557551" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.13.557551">
        <p class="paperTitle">Asgard archaea defense systems and their roles in the origin of immunity in eukaryotes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.13.557551" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.13.557551" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Leao, P.; Little, M. E.; Appler, K. E.; Sahaya, D.; Aguilar-Pine, E.; Currie, K.; Finkelstein, I. J.; De Anda, V.; Baker, B. J.</p>
        <p class="info">Score: 74.7, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.13.557551' target='https://doi.org/10.1101/2023.09.13.557551'> 10.1101/2023.09.13.557551</a></p>
        <p class="abstract">Immune systems are integral to survival against viral infection. Recently, dozens of new anti-viral systems have been characterized in bacteria1. Some of these systems are present in eukaryotes and appear to have originated in prokaryotes. However, little is known about these defense mechanisms in archaea. Here, we identified 2,610 complete defense systems in archaea related to eukaryotes, the Asgardarchaeota2. These comprise 89 unique systems, including argonaute, NLR, mokosh, viperin, lassamu, and CBASS. Asgard viperin (asVip) and argonaute (asAgo) proteins are present at high frequencies compared to bacteria and have structural homology to eukaryotes. Phylogenetic analyses revealed asVips are ancestral eukaryotic proteins. Heterologous expression of asVips in bacteria, including the lineage closest to eukaryotes, Hodarchaeales, showed robust anti-phage activity. Eukaryotic-and bacterial-argonaute proteins appear to have originated in the Asgardarchaeota, and have ancient structural characteristics. AsAgos appear to have argonaute-PIWI domains which are key components of the RNA interference (RNAi) in eukaryotes. Characterization of hundreds of new defense systems in the Asgardarchaeota revealed these archaea played important roles in the innovation of viral protection in eukaryotes. Given their relationship to eukaryotes, these defense systems may have applications in biomedicine and biotechnology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557206">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557206" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557206">
        <p class="paperTitle">Immune Evasion, Infectivity, and Fusogenicity of SARS-CoV-2 Omicron BA.2.86 and FLip Variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557206" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557206" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Qu, P.; Xu, K.; Faraone, J. N.; Goodarzi, N.; Zheng, Y.-M.; Carlin, C.; Bednash, J. S.; Horowitz, J. C.; Mallampalli, R. K.; Saif, L. J.; Oltz, E. M.; Jones, D.; Gumina, R. J.; Liu, S.-L.</p>
        <p class="info">Score: 66.9, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557206' target='https://doi.org/10.1101/2023.09.11.557206'> 10.1101/2023.09.11.557206</a></p>
        <p class="abstract">Evolution of SARS-CoV-2 requires the reassessment of current vaccine measures. Here, we characterized BA.2.86 and the XBB-lineage variant FLip by investigating their neutralization alongside D614G, BA.1, BA.2, BA.4/5, XBB.1.5, and EG.5.1 by sera from 3-dose vaccinated and bivalent vaccinated healthcare workers, XBB.1.5-wave infected first responders, and monoclonal antibody (mAb) S309. We assessed the biology of the variant Spikes by measuring viral infectivity and membrane fusogenicity. BA.2.86 is less immune evasive compared to FLip and other XBB variants, consistent with antigenic distances. Importantly, distinct from XBB variants, mAb S309 was unable to neutralize BA.2.86, likely due to a D339H mutation based on modeling. BA.2.86 had relatively high fusogenicity and infectivity in CaLu-3 cells but low fusion and infectivity in 293T-ACE2 cells compared to some XBB variants, suggesting a potentially differences conformational stability of BA.2.86 Spike. Overall, our study underscores the importance of SARS-CoV-2 variant surveillance and the need for updated COVID-19 vaccines.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556802">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556802" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556802">
        <p class="paperTitle">Multivariate comparison of taxonomic, chemical and technical data from 80 full-scale an-aerobic digester-related systems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556802" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556802" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Otto, P.; Puchol-Royo, R.; Ortega-Legarreta, A.; Tanner, K.; Tideman, J.; de Vries, S.-J.; Pascual, J.; Porcar, M.; Latorre-Perez, A.; Abendroth, C.</p>
        <p class="info">Score: 67.2, Published: 2023-09-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556802' target='https://doi.org/10.1101/2023.09.08.556802'> 10.1101/2023.09.08.556802</a></p>
        <p class="abstract">This study represents one of the most comprehensive characterisations of the anaerobic digestion (AD) microbiome with 80 samples from 45 different large-scale reactors in three coun-tries. Technical, chemical and taxonomic data was thoroughly collected, analyzed and correlated to identify the main drivers of AD processes. Our results showed that MBA03, Proteiniphilum, a member of Dethiobacteraceae, and Caldicoprobacter were present in all the samples, while Meth-anosarcina was the most abundant and prevalent archaea. Two distinct bacterial clusters were iden-tified by correlating microbial abundances. One was correlated with hydrogenotrophic and the other with acetoclastic methanogenesis. Organic acids, ammonia, nitrogen, COD and the trace el-ements Fe, Mo, and the macro nutrient P had the greatest impact on AD microbiomes. Temperature, reactor type and substrate also influenced the formation of a specialized microbial community. Overall, this work sheds light on the microbial key players involved in AD and evaluates how they are affected by technical and chemical parameters.

HighlightsO_LIGeneration of a holistic dataset of chemical, taxonomic and technical parameters of 80 large-scale anaerobic digestion systems.
C_LIO_LIIdentification of a core microbiome comprising MBA03, Proteiniphilum, an uncultured or-ganism from the Dethiobacteraceae family, and the Caldicoprobacter family.
C_LIO_LIIdentification of the main influencing parameters that determine the occurrence and non-occurrence of specific genera.
C_LIO_LICorrelation of bacterial taxa with hydrogenotrophic and/or acetoclastic archaea.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557286">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557286" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557286">
        <p class="paperTitle">An obligate aerobe adapts to hypoxia by hybridising fermentation with carbon storage</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557286" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557286" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gillett, D. L.; Hutchinson, T.; Mudaliyar, M.; Watts, T. D.; Wong, W. W.; Locop, J.; Jimenez, L.; Hanchapola, I.; Lee, H.-C.; Tanuwidjaya, E.; Steele, J. R.; Schittenhelm, R. B.; Barlow, C.; Grinter, R.; Ghosal, D.; Cook, P.; Greening, C.</p>
        <p class="info">Score: 23.6, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557286' target='https://doi.org/10.1101/2023.09.11.557286'> 10.1101/2023.09.11.557286</a></p>
        <p class="abstract">In soil ecosystems, obligately aerobic bacteria survive oxygen deprivation (hypoxia) by entering non-replicative persistent states. Little is known about how these bacteria rewire their metabolism to stay viable in these states. The model obligate aerobe Mycobacterium smegmatis maintains redox homeostasis during hypoxia by mediating fermentative hydrogen production. However, the fate of organic carbon during fermentation, and the associated remodeling of carbon metabolism, is unresolved. Here we systematically profiled the metabolism of M. smegmatis during aerobic growth, hypoxic persistence, and the transition between these states. Using differential isotope labelling, and paired metabolomics and proteomics, we observed rerouting of central carbon metabolism through the pentose phosphate pathway and Entner-Doudoroff pathway during hypoxia. We show that M. smegmatis excretes high levels of hydrogen concurrently with upregulating triacylglyceride synthases and accumulating glycerides as carbon stores. Using electron cryotomography (cryo-ET), we observed the presence of large spheroid structures consistent with the appearance of lipid droplets. Thus, in contrast to obligately and facultative anaerobic fermentative bacteria, M. smegmatis stores rather than excretes organic carbon during hypoxia. This novel hybrid metabolism likely provides a competitive advantage in resource-variable environments by allowing M. smegmatis to simultaneously dispose excess reductant during hypoxia and maintain carbon stores to rapidly resume growth upon reoxygenation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.15.557922">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.15.557922" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.15.557922">
        <p class="paperTitle">Multiple evolutionary pathways lead to vancomycin resistance in Clostridioides difficile</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.15.557922" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.15.557922" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Buddle, J. E.; Wright, R. C. T.; Turner, C. E.; Chaudhuri, R. R.; Brockhurst, M. A.; Fagan, R. P.</p>
        <p class="info">Score: 23.1, Published: 2023-09-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.15.557922' target='https://doi.org/10.1101/2023.09.15.557922'> 10.1101/2023.09.15.557922</a></p>
        <p class="abstract">Clostridioides difficile is an important human pathogen, for which there are very limited treatment options, primarily the glycopeptide antibiotic vancomycin. In recent years vancomycin resistance has emerged as a serious problem in several Gram positive pathogens, but high level resistance has yet to be reported for C. difficile, although it is not known if this is due to constraints upon resistance evolution in this species. Here we show that resistance to vancomycin can evolve rapidly under ramping selection but is accompanied by severe fitness costs and pleiotropic trade-offs, including sporulation defects that would be expected to severely impact transmission. We identified two distinct pathways to resistance, both of which are predicted to result in changes to the muropeptide terminal D-Ala-D-Ala that is the primary target of vancomycin. One of these pathways involves a previously uncharacterised D,D-carboxypeptidase, expression of which is controlled by a dedicated two-component signal transduction system. Our findings suggest that while C. difficile is capable of evolving high-level vancomycin resistance, this outcome may be limited clinically due to pleiotropic effects on key pathogenicity trains. Moreover, our data provide a mutational roadmap to inform genomic surveillance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557253">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557253" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557253">
        <p class="paperTitle">PfMORC protein regulates chromatin accessibility and transcriptional repression in the human malaria parasite, P. falciparum.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557253" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557253" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chahine, Z. M.; Gupta, M.; Lenz, T.; Hollin, T.; Abel, S.; Banks, C.; Saraf, A.; Prudhomme, J.; Florens, L.; Le Roch, K. G.</p>
        <p class="info">Score: 19.9, Published: 2023-09-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557253' target='https://doi.org/10.1101/2023.09.11.557253'> 10.1101/2023.09.11.557253</a></p>
        <p class="abstract">The environmental challenges the human malaria parasite, Plasmodium falciparum, faces during its progression into its various lifecycle stages warrant the use of effective and highly regulated access to chromatin for transcriptional regulation. Microrchidia (MORC) proteins have been implicated in DNA compaction and gene silencing across plant and animal kingdoms. Accumulating evidence has shed light into the role MORC protein plays as a transcriptional switch in apicomplexan parasites. In this study, using CRISPR/Cas9 genome editing tool along with complementary molecular and genomics approaches, we demonstrate that PfMORC not only modulates chromatin structure and heterochromatin formation throughout the parasite erythrocytic cycle, but is also essential to the parasite survival. Chromatin immunoprecipitation followed by deep sequencing (ChIP-seq) experiments suggest that PfMORC binds to not only sub-telomeric regions and genes involved in antigenic variation but is also most likely a key modulator of stage transition. Protein knockdown experiments followed by chromatin conformation capture (Hi-C) studies indicate that downregulation of PfMORC induces the collapse of the parasite heterochromatin structure leading to its death. All together these findings confirm that PfMORC plays a crucial role in chromatin structure and gene regulation, validating this factor as a strong candidate for novel antimalarial strategies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.13.557537">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.13.557537" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.13.557537">
        <p class="paperTitle">Genomic analysis of Coccomyxa viridis, a common low-abundance alga associated with lichen symbioses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.13.557537" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.13.557537" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tagirdzhanova, G.; Scharnagl, K.; Yan, X.; Talbot, N. J.</p>
        <p class="info">Score: 17.1, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.13.557537' target='https://doi.org/10.1101/2023.09.13.557537'> 10.1101/2023.09.13.557537</a></p>
        <p class="abstract">Lichen symbiosis is centered around a relationship between a fungus and a photosynthetic microbe, usually a green alga. In addition to their main photosynthetic partner (the photobiont), lichen symbioses can contain additional algae present in low abundance. The biology of these algae and the way they interact with the rest of lichen symbionts remains largely unknown. Here we present the first genome sequence of a non-photobiont lichen-associated alga. Coccomyxa viridis was unexpectedly found in 12% of publicly available lichen metagenomes. With few exceptions, members of the Coccomyxa viridis clade occur in lichens as non-photobionts, potentially growing in thalli endophytically. The 45.7 Mbp genome of C. viridis was assembled into 18 near chromosome-level contigs, making it one of the most contiguous genomic assemblies for any lichen-associated algae. Comparing the C. viridis genome to its close relatives revealed the presence of traits associated with the lichen lifestyle. The genome of C. viridis provides a new resource for exploring the evolution of the lichen symbiosis, and how symbiotic lifestyles shaped evolution in green algae.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
