<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Virological characteristics of the SARS-CoV-2 JN.1 variant
Authors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Zahradnik, J.; Ito, J.; Sato, K.
Score: 190.7, Published: 2023-12-09 DOI: 10.1101/2023.12.08.570782
The SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Virological characteristics of the SARS-CoV-2 JN.1 variant
Authors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Zahradnik, J.; Ito, J.; Sato, K.
Score: 190.7, Published: 2023-12-09 DOI: 10.1101/2023.12.08.570782
The SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-10T10:37:48+00:00" />
<meta property="article:modified_time" content="2023-12-10T10:37:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Virological characteristics of the SARS-CoV-2 JN.1 variant
Authors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Zahradnik, J.; Ito, J.; Sato, K.
Score: 190.7, Published: 2023-12-09 DOI: 10.1101/2023.12.08.570782
The SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Virological characteristics of the SARS-CoV-2 JN.1 variant\nAuthors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Zahradnik, J.; Ito, J.; Sato, K.\nScore: 190.7, Published: 2023-12-09 DOI: 10.1101/2023.12.08.570782\nThe SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG.",
  "keywords": [
    
  ],
  "articleBody": " Virological characteristics of the SARS-CoV-2 JN.1 variant\nAuthors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Zahradnik, J.; Ito, J.; Sato, K.\nScore: 190.7, Published: 2023-12-09 DOI: 10.1101/2023.12.08.570782\nThe SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG.5.1 and HK.3. Comparing to XBB and BA.2, BA.2.86 carries more than 30 mutations in the spike (S) protein, indicating a high potential for immune evasion. BA.2.86 has evolved and its descendant, JN.1 (BA.2.86.1.1), emerged in late 2023. JN.1 harbors S:L455S and three mutations in non-S proteins. S:L455S is a hallmark mutation of JN.1: we have recently shown that HK.3 and other \"FLip\" variants carry S:L455F, which contributes to increased transmissibility and immune escape ability compared to the parental EG.5.1 variant. Here, we investigated the virological properties of JN.1.\nSARS-CoV-2 infection is associated with intestinal permeability, systemic inflammation, and microbial dysbiosis in hospitalized COVID-19 patients\nAuthors: Basting, C. M.; Langat, R.; Broedlow, C. A.; Guerrero, C.; Bold, T. D.; Bailey, M.; Velez, A.; Schroeder, T.; Short-Miller, J.; Cromarty, R.; Mayer, Z.; Southern, P. J.; Schacker, T. W.; Safo, S. E.; Bramante, C. T.; Tignanelli, C. J.; Schifanella, L.; Klatt, N. R.\nScore: 169.9, Published: 2023-12-08 DOI: 10.1101/2023.12.07.570670\nCoronavirus disease 2019 (COVID-19) and associated severity has been linked to uncontrolled inflammation and may be associated with changes in the microbiome of mucosal sites including the gastrointestinal tract and oral cavity. These sites play an important role in host-microbe homeostasis and disruption of epithelial barrier integrity during COVID-19 may potentially lead to exacerbated inflammation and immune dysfunction. Outcomes in COVID-19 are highly disparate, ranging from asymptomatic to fatal, and the impact of microbial dysbiosis on disease severity is unclear. Here, we obtained plasma, rectal swabs, oropharyngeal swabs, and nasal swabs from 86 patients hospitalized with COVID-19 and 12 healthy volunteers. We performed 16S rRNA sequencing to characterize the microbial communities in the mucosal swabs and measured circulating cytokines, markers of gut barrier integrity, and fatty acids in the plasma samples. We compared these plasma concentrations and microbiomes between healthy volunteers and the COVID-19 patients who had survived or unfortunately died by the end of study enrollment, and between severe disease and healthy controls, as well as performed a correlation analysis between plasma variables and bacterial abundances. The rectal swabs of COVID-19 patients had reduced abundances of several commensal bacteria including Faecalibacterium prausnitsii, and an increased abundance of the opportunistic pathogens Eggerthella lenta and Hungatella hathewayi. Furthermore, the oral pathogen Scardovia wiggsiae was more abundant in the oropharyngeal swabs of COVID-19 patients who died. The abundance of both H. hathewayi and S. wiggsiae correlated with circulating inflammatory markers including IL-6, highlighting the possible role of the microbiome in COVID-19 severity, and providing potential therapeutic targets for managing COVID-19.\nDetection and spread of high pathogenicity avian influenza virus H5N1 in the Antarctic Region\nAuthors: Bennison, A.; Byrne, A. M.; Reid, S. M.; Lynton-Jenkins, J. G.; Mollett, B.; De Sliva, D.; Peers-Dent, J.; Finlayson, K. E.; Hall, R.; Blockley, F.; Blyth, M.; Falchieri, M.; Fowler, Z.; Fitzcharles, E.; Brown, I. H.; James, J.; Banyard, A. C.\nScore: 77.8, Published: 2023-11-24 DOI: 10.1101/2023.11.23.568045\nThe Antarctic is the only major geographical region in which high pathogenicity avian influenza virus (HPAIV) has never previously been detected. The current panzootic of H5N1 HPAIV has decimated wild bird populations across Europe, North America and South America. Here we report on the emergence of clade 2.3.4.4b H5N1 HPAIV in the Antarctic and sub-Antarctic regions of South Georgia and the Falkland Islands respectively. We initially detected H5N1 HPAIV in samples collected from brown skuas at Bird Island, South Georgia on 8th October 2023. Since this detection, increased mortalities were observed in brown skuas, kelp gulls, elephant seals and fur seal at multiple sites across South Georgia. We confirmed H5N1 HPAIV in multiple brown skuas and kelp gulls across four different sampling locations in South Georgia. Simultaneously, we also confirmed H5N1 HPAIV in a southern fulmar in the Falkland Islands. Genetic assessment of the virus indicates spread from South America, likely through movement of migratory birds. Here we describe the emergence, species impact and genetic composition of the virus and propose both introductory routes and potential long-term impact on avian and mammalian species across the Antarctic region.\nSpatial segregation and aging of metabolic processes underlie phenotypic heterogeneity in mycobacteria\nAuthors: Gwin, C. M.; Gupta, K. R.; Lu, Y.; Shao, L.; Rego, E. H.\nScore: 43.0, Published: 2023-12-02 DOI: 10.1101/2023.12.01.569614\nIndividual cells within clonal populations of mycobacteria vary in size, growth rate, and antibiotic susceptibility. Heterogeneity is, in part, determined by LamA, a protein found exclusively in mycobacteria. LamA localizes to sites of new cell wall synthesis where it recruits proteins important for polar growth and establishing asymmetry. Here, we report that in addition to this function, LamA interacts with complexes involved in oxidative phosphorylation (OXPHOS) at a subcellular location distinct from cell wall synthesis. Importantly, heterogeneity depends on a unique extension of the mycobacterial ATP synthase, and LamA mediates the coupling between ATP production and cell growth in single cells. Strikingly, as single cells age, concentrations of proteins important for oxidative phosphorylation become less abundant, and older cells rely less on oxidative phosphorylation for growth. Together, our data reveal that central metabolism is spatially organized within a single mycobacterium and varies within a genetically identical population of mycobacteria. Designing therapeutic regimens to account for this heterogeneity may help to treat mycobacterial infections faster and more completely.\nDistinct evolution of SARS-CoV-2 Omicron XBB and BA.2.86 lineages combining increased fitness and antibody evasion\nAuthors: Planas, D.; Staropoli, I.; Michel, V.; Lemoine, F.; Donati, F.; Prot, M.; Porrot, F.; Benhassine, F. G.; Jeyarajah, B.; Brisebarre, A.; Dehan, O.; Avon, L.; Bolland, W.; Hubert, M.; Buchreiser, J.; Vanhoucke, T.; Rosenbaum, P.; Veyer, D.; Pere, H.; Lina, B.; Assant, S. T.; Hocqueloux, L.; Prazuck, T.; Loriere, E. S.; Schwartz, O.\nScore: 75.4, Published: 2023-12-06 DOI: 10.1101/2023.11.20.567873\nThe unceasing circulation of SARS-CoV-2 leads to the continuous emergence of novel viral sublineages. Here, we isolated and characterized XBB.1, XBB.1.5, XBB.1.9.1, XBB.1.16.1, EG.5.1.1, EG.5.1.3, XBF, BA.2.86.1 and JN.1 variants, representing \u003e80% of circulating variants in November 2023. The XBB subvariants carry few but recurrent mutations in the spike, whereas BA.2.86.1 and JN.1 harbor \u003e30 additional changes. These variants replicated in IGROV-1 but no longer in Vero E6 and were not markedly fusogenic. They potently infected nasal epithelial cells, with EG.5.1.3 exhibiting the highest fitness. Antivirals remained active. Neutralizing antibody (NAb) responses from vaccinees and BA.1/BA.2-infected individuals were markedly lower compared to BA.1, without major differences between variants. An XBB breakthrough infection enhanced NAb responses against both XBB and BA.2.86 variants. JN.1 displayed lower affinity to ACE2 and higher immune evasion properties compared to BA.2.86.1. Thus, while distinct, the evolutionary trajectory of these variants combines increased fitness and antibody evasion.\nGiant genes are rare but implicated in cell wall degradation by predatory bacteria\nAuthors: West-Roberts, J. A.; Valentin Alvarado, L. E.; Mullen, S.; Sachdeva, R.; Smith, J.; Hug, L. A.; Gregoire, D.; Liu, W.; Lin, T.-Y.; Husain, G.; Amano, Y.; Ly, L.; Banfield, J.\nScore: 55.7, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568195\nAcross the tree of life, gene lengths vary, but most are no more than a few thousand base pairs in length. The largest protein often reported is the [~]40,000 aa eukaryotic Titin. Even larger proteins may occur in the rapidly expanding set of metagenome-derived sequences, but their existence may be obscured by assembly fragmentation. Here, we leverage genome curation to complete metagenome-derived sequences that encode predicted proteins of up to 85,804 aa. Overall, the findings illuminate a huge knowledge gap related to giant proteins. Although predicted proteins of \u003e30,000 aa occur in bacterial phyla such as Firmicutes and Actinobacteria, they are most common in ca. Omnitrophota, ultra small bacteria that adopt predatory lifestyles. All full length giant genes encode numerous transmembrane regions and most encode divergent secA DEAD helicase domains. In silico structural prediction of protein subregions was required to identify domains in unannotated protein segments, and revealed putative domains implicated in attachment and carbohydrate degradation. Many giant genes in new complete and near-complete Omnitrophota genomes occur in close proximity to genes homologous to type II secretion systems as well as carbohydrate import systems. This, in combination with the domain content, suggests that many bacterial giant proteins enable prey adhesion and cell wall digestion during bacterial predation.\nElevated mutation rates in the multi-azole resistant Aspergillus fumigatus cladedrives rapid evolution of antifungal resistance\nAuthors: Bottery, M. J.; van Rhijn, N. J.; Chown, H.; Rhodes, J. L.; Celia-Sanchez, B. N.; Brewer, M. T.; Momany, M.; Fisher, M. C.; Knight, C. G.; Bromley, M. J.\nScore: 21.2, Published: 2023-12-05 DOI: 10.1101/2023.12.05.570068\nThe evolution of antifungal resistance is an emerging global threat. Particularly concerning is the widespread occurrence of azole resistance within Aspergillus fumigatus, a globally ubiquitous environmental mould that causes over 1 million life-threatening invasive infections in humans each year. It is increasingly evident that the environmental use of azoles has led to selective sweeps across multiple genomic loci resulting in the rapid expansion of a genetically distinct cluster of genotypes (clade A) that results in resistance to clinically deployed azoles. Isolates within this cluster are more likely to be cross resistant to agricultural antifungals with unrelated modes of action suggesting they may be adapting rapidly to antifungal challenge. Here we show that this cluster is not only multi-azole resistant but has increased propensity to develop resistance to new antifungals because of variants in the DNA mismatch repair system. A variant in msh6 is found almost exclusively within clade A, occurs in 88% of multi-azole resistant isolates harbouring the canonical cyp51A azole resistance allelic variant TR34/L98H, and is globally distributed. Naturally occurring isolates with this msh6 variant display a 4 to 9-times higher rate of mutation, leading to an increased propensity to evolve resistance to current and next generation antifungals. We argue that pervasive environmental use of fungicides creates selective arenas whereby genotypes of A. fumigatus with increased adaptive capability thrive in the face of strong directional selection, leading to the genesis and amplification of antifungal resistance. These results help explain the pronounced clustering of multiple independent resistance mechanisms within the mutable clade A. Our findings further suggest that resistance to next generation antifungals is more likely to emerge within organisms that are already multi-azole resistant, posing a major problem due to the prospect of dual use of novel antifungals in clinical and agricultural settings.\nIntracellular Ebola Virus nucleocapsid assembly revealed by in situ cryo-electron tomography.\nAuthors: Watanabe, R.; Zyla, D.; Parekh, D.; Hong, C.; Jones, Y.; Schendel, S. L.; Wan, W.; Castillon, G.; Saphire, E. O.\nScore: 19.8, Published: 2023-12-02 DOI: 10.1101/2023.12.01.569666\nFiloviruses, including Ebola and Marburg viruses, are a family of highly lethal viruses that cause severe hemorrhagic fever in humans with near annual outbreaks. Antiviral options are limited, and broad-spectrum antivirals are needed. The core of the filamentous virion is formed by the nucleocapsid, a helical structure in which the polymerized nucleoprotein (NP) assembles on the RNA genome and is surrounded by viral proteins VP24 and VP351,2. The mechanism by which these proteins assemble to form nucleocapsids inside infected cells is unclear, and the identity of the outer half of nucleocapsid density remains unassigned. Using cryo-electron tomography, we revealed the assembly process of Ebola virus in both cells transfected with the viral proteins and cells infected with a biologically contained model Ebola virus. We obtained a fully assembled intracellular nucleocapsid-like structure at 9 [A] that allows assignment of previously unassigned densities and presents the first complete model of the intracellular Ebola nucleocapsid. Our model reveals a previously unknown, third layer of additional copies of NP in complex with VP35. In this outer layer, the N-terminal domain of VP35 maintains NP in a monomeric form, and the rest of VP35 appears to participate in an intra-rung nucleocapsid interaction. The C-terminal region of NP lies outside this third outer layer, allowing localization and recruitment of additional viral proteins and connection of the nucleocapsid to the matrix beneath the virus envelope. Comparison of this in-cell nucleocapsid to previous in-virion structures reveals that the nucleocapsid helix condenses upon incorporation into the virion. Molecular interfaces responsible for nucleocapsid assembly are conserved among filoviruses, offering potential target sites for broadly effective antivirals. The architecture and assembly dynamics of Ebola virus nucleocapsids revealed here uncover a unique filovirus nucleocapsid assembly and a condensation mechanism that protect viral genome integrity in the virion.\nGenomic plasticity is a blueprint of diversity in Salmonella lineages\nAuthors: Krishnakant Kushwaha, S.; Anand, A.; Wu, Y.; Leonardo Avila, H.; Sicheritz-Ponten, T.; Millard, A.; Amol Marathe, S.; Nobrega, F. L.\nScore: 18.5, Published: 2023-12-02 DOI: 10.1101/2023.12.02.569618\nThe flexible genome, organised in regions of genomic plasticity (RGP), serves as a potent facilitator of the dynamic evolution of bacterial genomes through gene acquisition and loss. Here we explored the genomic plasticity across Salmonella lineages, revealing a purposeful, non-random integration pattern of pathogenicity-related gene clusters into specific RGP. Noteworthy examples include the correlation between the type I-E CRISPR-Cas system, gold tolerance, and specific RGP. The scattered prevalence of RGP across Salmonella lineages profoundly shapes the pathogenicity makeup of Salmonella strains. The preferences of RGP seem guided by conserved flanking genes that likely share regulatory and functional coordination. For example, metal resistance genes are predominant in RGP positioned near stress resistance genes, indicating a regulatory network to efficiently counter stressors. Additionally, we observed that different plasmid incompatibility types and prophage genera carry distinct pathogenicity genes. Similar to RGP, their distribution across Salmonella lineages plays a critical role in defining pathogenicity. By uncovering these intricate connections among gene clusters, RGP, mobile genetic elements, and pathogenic attributes, our study offers novel insights into the evolutionary trajectory of Salmonella and holds promise for predicting future adaptations and developing targeted interventions against infections.\nAlternative splicing expands the antiviral IFITM repertoire in Chinese horseshoe bats\nAuthors: Mak, N.; Zhang, D.; Li, X.; Rahman, K.; Datta, S. A. K.; Taylor, J.; Liu, J.; Shi, Z.; Temperton, N.; Irving, A. T.; Compton, A. A.; Sloan, R. D.\nScore: 17.0, Published: 2023-12-04 DOI: 10.1101/2023.12.04.569605\nThe interferon response is shaped by the evolutionary arms race between hosts and the pathogens they carry. The human interferon-induced transmembrane protein (IFITM) family consists of three antiviral IFITM genes that arose by gene duplication, they restrict virus entry and are key players of the interferon response. Yet, little is known about IFITMs in other mammals. Here, we identified an IFITM gene in Chinese horseshoe bat, a natural host of SARS-coronaviruses, that is alternatively spliced to produce two IFITM isoforms. These bat IFITMs have conserved structures in vitro and differential antiviral activities against influenza A virus and coronaviruses including SARS- and MERS-coronavirus. In parallel with human IFITM1-3, the bat IFITM isoforms localize to distinct cellular compartments. Further analysis of IFITM repertoires in 205 mammals reveals that alternative splicing is a ubiquitous strategy for IFITM diversification, albeit less widely adopted than gene duplication. These findings showcase an example of convergent evolution where species-specific selection pressures led to expansion of the IFITM family through multiple means, underscoring the importance of IFITM diversity as a component of innate immunity.\n",
  "wordCount" : "2543",
  "inLanguage": "en",
  "datePublished": "2023-12-10T10:37:48Z",
  "dateModified": "2023-12-10T10:37:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 10, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.570782">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.570782" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.570782">
        <p class="paperTitle">Virological characteristics of the SARS-CoV-2 JN.1 variant</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.570782" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.570782" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kaku, Y.; Okumura, K.; Padilla-Blanco, M.; Kosugi, Y.; Uriu, K.; Hinay, A. A.; Chen, L.; Plianchaisuk, A.; Kobiyama, K.; Ishii, K. J.; The Genotype to Phenotype Japan (G2P-Japan) Consortium,  ; Zahradnik, J.; Ito, J.; Sato, K.</p>
        <p class="info">Score: 190.7, Published: 2023-12-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.570782' target='https://doi.org/10.1101/2023.12.08.570782'> 10.1101/2023.12.08.570782</a></p>
        <p class="abstract">The SARS-CoV-2 BA.2.86 lineage, first identified in August 2023, is phylogenetically distinct from the currently circulating SARS-CoV-2 Omicron XBB lineages, including EG.5.1 and HK.3. Comparing to XBB and BA.2, BA.2.86 carries more than 30 mutations in the spike (S) protein, indicating a high potential for immune evasion. BA.2.86 has evolved and its descendant, JN.1 (BA.2.86.1.1), emerged in late 2023. JN.1 harbors S:L455S and three mutations in non-S proteins. S:L455S is a hallmark mutation of JN.1: we have recently shown that HK.3 and other &#34;FLip&#34; variants carry S:L455F, which contributes to increased transmissibility and immune escape ability compared to the parental EG.5.1 variant. Here, we investigated the virological properties of JN.1.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570670">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570670" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570670">
        <p class="paperTitle">SARS-CoV-2 infection is associated with intestinal permeability, systemic inflammation, and microbial dysbiosis in hospitalized COVID-19 patients</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570670" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570670" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Basting, C. M.; Langat, R.; Broedlow, C. A.; Guerrero, C.; Bold, T. D.; Bailey, M.; Velez, A.; Schroeder, T.; Short-Miller, J.; Cromarty, R.; Mayer, Z.; Southern, P. J.; Schacker, T. W.; Safo, S. E.; Bramante, C. T.; Tignanelli, C. J.; Schifanella, L.; Klatt, N. R.</p>
        <p class="info">Score: 169.9, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570670' target='https://doi.org/10.1101/2023.12.07.570670'> 10.1101/2023.12.07.570670</a></p>
        <p class="abstract">Coronavirus disease 2019 (COVID-19) and associated severity has been linked to uncontrolled inflammation and may be associated with changes in the microbiome of mucosal sites including the gastrointestinal tract and oral cavity. These sites play an important role in host-microbe homeostasis and disruption of epithelial barrier integrity during COVID-19 may potentially lead to exacerbated inflammation and immune dysfunction. Outcomes in COVID-19 are highly disparate, ranging from asymptomatic to fatal, and the impact of microbial dysbiosis on disease severity is unclear. Here, we obtained plasma, rectal swabs, oropharyngeal swabs, and nasal swabs from 86 patients hospitalized with COVID-19 and 12 healthy volunteers. We performed 16S rRNA sequencing to characterize the microbial communities in the mucosal swabs and measured circulating cytokines, markers of gut barrier integrity, and fatty acids in the plasma samples. We compared these plasma concentrations and microbiomes between healthy volunteers and the COVID-19 patients who had survived or unfortunately died by the end of study enrollment, and between severe disease and healthy controls, as well as performed a correlation analysis between plasma variables and bacterial abundances. The rectal swabs of COVID-19 patients had reduced abundances of several commensal bacteria including Faecalibacterium prausnitsii, and an increased abundance of the opportunistic pathogens Eggerthella lenta and Hungatella hathewayi. Furthermore, the oral pathogen Scardovia wiggsiae was more abundant in the oropharyngeal swabs of COVID-19 patients who died. The abundance of both H. hathewayi and S. wiggsiae correlated with circulating inflammatory markers including IL-6, highlighting the possible role of the microbiome in COVID-19 severity, and providing potential therapeutic targets for managing COVID-19.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.568045">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.568045" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.568045">
        <p class="paperTitle">Detection and spread of high pathogenicity avian influenza virus H5N1 in the Antarctic Region</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.568045" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.568045" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bennison, A.; Byrne, A. M.; Reid, S. M.; Lynton-Jenkins, J. G.; Mollett, B.; De Sliva, D.; Peers-Dent, J.; Finlayson, K. E.; Hall, R.; Blockley, F.; Blyth, M.; Falchieri, M.; Fowler, Z.; Fitzcharles, E.; Brown, I. H.; James, J.; Banyard, A. C.</p>
        <p class="info">Score: 77.8, Published: 2023-11-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.568045' target='https://doi.org/10.1101/2023.11.23.568045'> 10.1101/2023.11.23.568045</a></p>
        <p class="abstract">The Antarctic is the only major geographical region in which high pathogenicity avian influenza virus (HPAIV) has never previously been detected. The current panzootic of H5N1 HPAIV has decimated wild bird populations across Europe, North America and South America. Here we report on the emergence of clade 2.3.4.4b H5N1 HPAIV in the Antarctic and sub-Antarctic regions of South Georgia and the Falkland Islands respectively. We initially detected H5N1 HPAIV in samples collected from brown skuas at Bird Island, South Georgia on 8th October 2023. Since this detection, increased mortalities were observed in brown skuas, kelp gulls, elephant seals and fur seal at multiple sites across South Georgia. We confirmed H5N1 HPAIV in multiple brown skuas and kelp gulls across four different sampling locations in South Georgia. Simultaneously, we also confirmed H5N1 HPAIV in a southern fulmar in the Falkland Islands. Genetic assessment of the virus indicates spread from South America, likely through movement of migratory birds. Here we describe the emergence, species impact and genetic composition of the virus and propose both introductory routes and potential long-term impact on avian and mammalian species across the Antarctic region.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569614">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569614" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569614">
        <p class="paperTitle">Spatial segregation and aging of metabolic processes underlie phenotypic heterogeneity in mycobacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569614" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569614" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gwin, C. M.; Gupta, K. R.; Lu, Y.; Shao, L.; Rego, E. H.</p>
        <p class="info">Score: 43.0, Published: 2023-12-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569614' target='https://doi.org/10.1101/2023.12.01.569614'> 10.1101/2023.12.01.569614</a></p>
        <p class="abstract">Individual cells within clonal populations of mycobacteria vary in size, growth rate, and antibiotic susceptibility. Heterogeneity is, in part, determined by LamA, a protein found exclusively in mycobacteria. LamA localizes to sites of new cell wall synthesis where it recruits proteins important for polar growth and establishing asymmetry. Here, we report that in addition to this function, LamA interacts with complexes involved in oxidative phosphorylation (OXPHOS) at a subcellular location distinct from cell wall synthesis. Importantly, heterogeneity depends on a unique extension of the mycobacterial ATP synthase, and LamA mediates the coupling between ATP production and cell growth in single cells. Strikingly, as single cells age, concentrations of proteins important for oxidative phosphorylation become less abundant, and older cells rely less on oxidative phosphorylation for growth. Together, our data reveal that central metabolism is spatially organized within a single mycobacterium and varies within a genetically identical population of mycobacteria. Designing therapeutic regimens to account for this heterogeneity may help to treat mycobacterial infections faster and more completely.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567873">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567873" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567873">
        <p class="paperTitle">Distinct evolution of SARS-CoV-2 Omicron XBB and BA.2.86 lineages combining increased fitness and antibody evasion</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567873" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567873" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Planas, D.; Staropoli, I.; Michel, V.; Lemoine, F.; Donati, F.; Prot, M.; Porrot, F.; Benhassine, F. G.; Jeyarajah, B.; Brisebarre, A.; Dehan, O.; Avon, L.; Bolland, W.; Hubert, M.; Buchreiser, J.; Vanhoucke, T.; Rosenbaum, P.; Veyer, D.; Pere, H.; Lina, B.; Assant, S. T.; Hocqueloux, L.; Prazuck, T.; Loriere, E. S.; Schwartz, O.</p>
        <p class="info">Score: 75.4, Published: 2023-12-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567873' target='https://doi.org/10.1101/2023.11.20.567873'> 10.1101/2023.11.20.567873</a></p>
        <p class="abstract">The unceasing circulation of SARS-CoV-2 leads to the continuous emergence of novel viral sublineages. Here, we isolated and characterized XBB.1, XBB.1.5, XBB.1.9.1, XBB.1.16.1, EG.5.1.1, EG.5.1.3, XBF, BA.2.86.1 and JN.1 variants, representing &gt;80% of circulating variants in November 2023. The XBB subvariants carry few but recurrent mutations in the spike, whereas BA.2.86.1 and JN.1 harbor &gt;30 additional changes. These variants replicated in IGROV-1 but no longer in Vero E6 and were not markedly fusogenic. They potently infected nasal epithelial cells, with EG.5.1.3 exhibiting the highest fitness. Antivirals remained active. Neutralizing antibody (NAb) responses from vaccinees and BA.1/BA.2-infected individuals were markedly lower compared to BA.1, without major differences between variants. An XBB breakthrough infection enhanced NAb responses against both XBB and BA.2.86 variants. JN.1 displayed lower affinity to ACE2 and higher immune evasion properties compared to BA.2.86.1. Thus, while distinct, the evolutionary trajectory of these variants combines increased fitness and antibody evasion.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568195">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568195" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568195">
        <p class="paperTitle">Giant genes are rare but implicated in cell wall degradation by predatory bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568195" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568195" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: West-Roberts, J. A.; Valentin Alvarado, L. E.; Mullen, S.; Sachdeva, R.; Smith, J.; Hug, L. A.; Gregoire, D.; Liu, W.; Lin, T.-Y.; Husain, G.; Amano, Y.; Ly, L.; Banfield, J.</p>
        <p class="info">Score: 55.7, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568195' target='https://doi.org/10.1101/2023.11.21.568195'> 10.1101/2023.11.21.568195</a></p>
        <p class="abstract">Across the tree of life, gene lengths vary, but most are no more than a few thousand base pairs in length. The largest protein often reported is the [~]40,000 aa eukaryotic Titin. Even larger proteins may occur in the rapidly expanding set of metagenome-derived sequences, but their existence may be obscured by assembly fragmentation. Here, we leverage genome curation to complete metagenome-derived sequences that encode predicted proteins of up to 85,804 aa. Overall, the findings illuminate a huge knowledge gap related to giant proteins. Although predicted proteins of &gt;30,000 aa occur in bacterial phyla such as Firmicutes and Actinobacteria, they are most common in ca. Omnitrophota, ultra small bacteria that adopt predatory lifestyles. All full length giant genes encode numerous transmembrane regions and most encode divergent secA DEAD helicase domains. In silico structural prediction of protein subregions was required to identify domains in unannotated protein segments, and revealed putative domains implicated in attachment and carbohydrate degradation. Many giant genes in new complete and near-complete Omnitrophota genomes occur in close proximity to genes homologous to type II secretion systems as well as carbohydrate import systems. This, in combination with the domain content, suggests that many bacterial giant proteins enable prey adhesion and cell wall digestion during bacterial predation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.570068">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.570068" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.570068">
        <p class="paperTitle">Elevated mutation rates in the multi-azole resistant Aspergillus fumigatus cladedrives rapid evolution of antifungal resistance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.570068" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.570068" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bottery, M. J.; van Rhijn, N. J.; Chown, H.; Rhodes, J. L.; Celia-Sanchez, B. N.; Brewer, M. T.; Momany, M.; Fisher, M. C.; Knight, C. G.; Bromley, M. J.</p>
        <p class="info">Score: 21.2, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.570068' target='https://doi.org/10.1101/2023.12.05.570068'> 10.1101/2023.12.05.570068</a></p>
        <p class="abstract">The evolution of antifungal resistance is an emerging global threat. Particularly concerning is the widespread occurrence of azole resistance within Aspergillus fumigatus, a globally ubiquitous environmental mould that causes over 1 million life-threatening invasive infections in humans each year. It is increasingly evident that the environmental use of azoles has led to selective sweeps across multiple genomic loci resulting in the rapid expansion of a genetically distinct cluster of genotypes (clade A) that results in resistance to clinically deployed azoles. Isolates within this cluster are more likely to be cross resistant to agricultural antifungals with unrelated modes of action suggesting they may be adapting rapidly to antifungal challenge. Here we show that this cluster is not only multi-azole resistant but has increased propensity to develop resistance to new antifungals because of variants in the DNA mismatch repair system. A variant in msh6 is found almost exclusively within clade A, occurs in 88% of multi-azole resistant isolates harbouring the canonical cyp51A azole resistance allelic variant TR34/L98H, and is globally distributed. Naturally occurring isolates with this msh6 variant display a 4 to 9-times higher rate of mutation, leading to an increased propensity to evolve resistance to current and next generation antifungals. We argue that pervasive environmental use of fungicides creates selective arenas whereby genotypes of A. fumigatus with increased adaptive capability thrive in the face of strong directional selection, leading to the genesis and amplification of antifungal resistance. These results help explain the pronounced clustering of multiple independent resistance mechanisms within the mutable clade A. Our findings further suggest that resistance to next generation antifungals is more likely to emerge within organisms that are already multi-azole resistant, posing a major problem due to the prospect of dual use of novel antifungals in clinical and agricultural settings.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569666">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569666" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569666">
        <p class="paperTitle">Intracellular Ebola Virus nucleocapsid assembly revealed by in situ cryo-electron tomography.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569666" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569666" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Watanabe, R.; Zyla, D.; Parekh, D.; Hong, C.; Jones, Y.; Schendel, S. L.; Wan, W.; Castillon, G.; Saphire, E. O.</p>
        <p class="info">Score: 19.8, Published: 2023-12-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569666' target='https://doi.org/10.1101/2023.12.01.569666'> 10.1101/2023.12.01.569666</a></p>
        <p class="abstract">Filoviruses, including Ebola and Marburg viruses, are a family of highly lethal viruses that cause severe hemorrhagic fever in humans with near annual outbreaks. Antiviral options are limited, and broad-spectrum antivirals are needed. The core of the filamentous virion is formed by the nucleocapsid, a helical structure in which the polymerized nucleoprotein (NP) assembles on the RNA genome and is surrounded by viral proteins VP24 and VP351,2. The mechanism by which these proteins assemble to form nucleocapsids inside infected cells is unclear, and the identity of the outer half of nucleocapsid density remains unassigned. Using cryo-electron tomography, we revealed the assembly process of Ebola virus in both cells transfected with the viral proteins and cells infected with a biologically contained model Ebola virus. We obtained a fully assembled intracellular nucleocapsid-like structure at 9 [A] that allows assignment of previously unassigned densities and presents the first complete model of the intracellular Ebola nucleocapsid. Our model reveals a previously unknown, third layer of additional copies of NP in complex with VP35. In this outer layer, the N-terminal domain of VP35 maintains NP in a monomeric form, and the rest of VP35 appears to participate in an intra-rung nucleocapsid interaction. The C-terminal region of NP lies outside this third outer layer, allowing localization and recruitment of additional viral proteins and connection of the nucleocapsid to the matrix beneath the virus envelope. Comparison of this in-cell nucleocapsid to previous in-virion structures reveals that the nucleocapsid helix condenses upon incorporation into the virion. Molecular interfaces responsible for nucleocapsid assembly are conserved among filoviruses, offering potential target sites for broadly effective antivirals. The architecture and assembly dynamics of Ebola virus nucleocapsids revealed here uncover a unique filovirus nucleocapsid assembly and a condensation mechanism that protect viral genome integrity in the virion.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.02.569618">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.02.569618" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.02.569618">
        <p class="paperTitle">Genomic plasticity is a blueprint of diversity in Salmonella lineages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.02.569618" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.02.569618" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krishnakant Kushwaha, S.; Anand, A.; Wu, Y.; Leonardo Avila, H.; Sicheritz-Ponten, T.; Millard, A.; Amol Marathe, S.; Nobrega, F. L.</p>
        <p class="info">Score: 18.5, Published: 2023-12-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.02.569618' target='https://doi.org/10.1101/2023.12.02.569618'> 10.1101/2023.12.02.569618</a></p>
        <p class="abstract">The flexible genome, organised in regions of genomic plasticity (RGP), serves as a potent facilitator of the dynamic evolution of bacterial genomes through gene acquisition and loss. Here we explored the genomic plasticity across Salmonella lineages, revealing a purposeful, non-random integration pattern of pathogenicity-related gene clusters into specific RGP. Noteworthy examples include the correlation between the type I-E CRISPR-Cas system, gold tolerance, and specific RGP. The scattered prevalence of RGP across Salmonella lineages profoundly shapes the pathogenicity makeup of Salmonella strains. The preferences of RGP seem guided by conserved flanking genes that likely share regulatory and functional coordination. For example, metal resistance genes are predominant in RGP positioned near stress resistance genes, indicating a regulatory network to efficiently counter stressors. Additionally, we observed that different plasmid incompatibility types and prophage genera carry distinct pathogenicity genes. Similar to RGP, their distribution across Salmonella lineages plays a critical role in defining pathogenicity. By uncovering these intricate connections among gene clusters, RGP, mobile genetic elements, and pathogenic attributes, our study offers novel insights into the evolutionary trajectory of Salmonella and holds promise for predicting future adaptations and developing targeted interventions against infections.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.04.569605">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.04.569605" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.04.569605">
        <p class="paperTitle">Alternative splicing expands the antiviral IFITM repertoire in Chinese horseshoe bats</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.04.569605" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.04.569605" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mak, N.; Zhang, D.; Li, X.; Rahman, K.; Datta, S. A. K.; Taylor, J.; Liu, J.; Shi, Z.; Temperton, N.; Irving, A. T.; Compton, A. A.; Sloan, R. D.</p>
        <p class="info">Score: 17.0, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.04.569605' target='https://doi.org/10.1101/2023.12.04.569605'> 10.1101/2023.12.04.569605</a></p>
        <p class="abstract">The interferon response is shaped by the evolutionary arms race between hosts and the pathogens they carry. The human interferon-induced transmembrane protein (IFITM) family consists of three antiviral IFITM genes that arose by gene duplication, they restrict virus entry and are key players of the interferon response. Yet, little is known about IFITMs in other mammals. Here, we identified an IFITM gene in Chinese horseshoe bat, a natural host of SARS-coronaviruses, that is alternatively spliced to produce two IFITM isoforms. These bat IFITMs have conserved structures in vitro and differential antiviral activities against influenza A virus and coronaviruses including SARS- and MERS-coronavirus. In parallel with human IFITM1-3, the bat IFITM isoforms localize to distinct cellular compartments. Further analysis of IFITM repertoires in 205 mammals reveals that alternative splicing is a ubiquitous strategy for IFITM diversification, albeit less widely adopted than gene duplication. These findings showcase an example of convergent evolution where species-specific selection pressures led to expansion of the IFITM family through multiple means, underscoring the importance of IFITM diversity as a component of innate immunity.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
