<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Lethal Infection of Human ACE2-Transgenic Mice Caused by SARS-CoV-2-related Pangolin Coronavirus GX_P2V(short_3UTR)
Authors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.
Score: 1039.7, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574008
SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) can cause 100% mortality in human ACE2-transgenic mice, potentially attributable to late-stage brain infection. This underscores a spillover risk of GX_P2V into humans and provides a unique model for understanding the pathogenic mechanisms of SARS-CoV-2-related viruses.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Lethal Infection of Human ACE2-Transgenic Mice Caused by SARS-CoV-2-related Pangolin Coronavirus GX_P2V(short_3UTR)
Authors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.
Score: 1039.7, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574008
SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) can cause 100% mortality in human ACE2-transgenic mice, potentially attributable to late-stage brain infection. This underscores a spillover risk of GX_P2V into humans and provides a unique model for understanding the pathogenic mechanisms of SARS-CoV-2-related viruses." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-10T10:40:05+00:00" />
<meta property="article:modified_time" content="2024-01-10T10:40:05+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Lethal Infection of Human ACE2-Transgenic Mice Caused by SARS-CoV-2-related Pangolin Coronavirus GX_P2V(short_3UTR)
Authors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.
Score: 1039.7, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574008
SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) can cause 100% mortality in human ACE2-transgenic mice, potentially attributable to late-stage brain infection. This underscores a spillover risk of GX_P2V into humans and provides a unique model for understanding the pathogenic mechanisms of SARS-CoV-2-related viruses."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Lethal Infection of Human ACE2-Transgenic Mice Caused by SARS-CoV-2-related Pangolin Coronavirus GX_P2V(short_3UTR)\nAuthors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.\nScore: 1039.7, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574008\nSARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) can cause 100% mortality in human ACE2-transgenic mice, potentially attributable to late-stage brain infection. This underscores a spillover risk of GX_P2V into humans and provides a unique model for understanding the pathogenic mechanisms of SARS-CoV-2-related viruses.",
  "keywords": [
    
  ],
  "articleBody": " Lethal Infection of Human ACE2-Transgenic Mice Caused by SARS-CoV-2-related Pangolin Coronavirus GX_P2V(short_3UTR)\nAuthors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.\nScore: 1039.7, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574008\nSARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) can cause 100% mortality in human ACE2-transgenic mice, potentially attributable to late-stage brain infection. This underscores a spillover risk of GX_P2V into humans and provides a unique model for understanding the pathogenic mechanisms of SARS-CoV-2-related viruses.\nRapid Emergence and Evolution of SARS-CoV-2 Variants in Advanced HIV Infection\nAuthors: Ko, S. H.; Radecki, P.; Belinky, F.; Bhiman, J.; Meiring, S.; Kleynhans, J.; Amoako, D.; Guerra Canedo, V.; Lucas, M.; Kekana, D.; Martinson, N.; Lebina, L.; Everatt, J.; Tempia, S.; Bylund, T.; Rawi, R.; Kwong, P.; Wolter, N.; von Gottberg, A.; Cohen, C.; Boritz, E. A.\nScore: 77.7, Published: 2024-01-06 DOI: 10.1101/2024.01.05.574420\nPrevious studies have linked the evolution of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) genetic variants to persistent infections in people with immunocompromising conditions1-4, but the evolutionary processes underlying these observations are incompletely understood. Here we used high-throughput, single-genome amplification and sequencing (HT-SGS) to obtain up to [~]103 SARS-CoV-2 spike gene sequences in each of 184 respiratory samples from 22 people with HIV (PWH) and 25 people without HIV (PWOH). Twelve of 22 PWH had advanced HIV infection, defined by peripheral blood CD4 T cell counts (i.e., CD4 counts) \u003c200 cells/L. In PWOH and PWH with CD4 counts [\u0026ge;]200 cells/L, most single-genome spike sequences in each person matched one haplotype that predominated throughout the infection. By contrast, people with advanced HIV showed elevated intra-host spike diversity with a median of 46 haplotypes per person (IQR 14-114). Higher intra-host spike diversity immediately after COVID-19 symptom onset predicted longer SARS-CoV-2 RNA shedding among PWH, and intra-host spike diversity at this timepoint was significantly higher in people with advanced HIV than in PWOH. Composition of spike sequence populations in people with advanced HIV fluctuated rapidly over time, with founder sequences often replaced by groups of new haplotypes. These population-level changes were associated with a high total burden of intra-host mutations and positive selection at functionally important residues. In several cases, delayed emergence of detectable serum binding to spike was associated with positive selection for presumptive antibody-escape mutations. Taken together, our findings show remarkable intra-host genetic diversity of SARS-CoV-2 in advanced HIV infection and suggest that adaptive intra-host SARS-CoV-2 evolution in this setting may contribute to the emergence of new variants of concern (VOCs).\nAMAPEC: accurate antimicrobial activity prediction for fungal effector proteins\nAuthors: Mesny, F.; Thomma, B. P.\nScore: 54.6, Published: 2024-01-04 DOI: 10.1101/2024.01.04.574150\nFungi typically occur in environments where numerous and diverse other microbes occur as well, often resulting in fierce competition for nutrients and habitat. To support fungal fitness in these environments, they evolved various mechanisms that mediate direct antagonism towards niche competitors. Among these, the secretion of proteins with antimicrobial activities has been reported in fungi with diverse lifestyles. Recently, several plant-associated fungi were shown to rely on the secretion of antimicrobial effector proteins to antagonize certain members of plant hosts microbiota and to successfully colonize plant tissues. Some of these effectors do not share homology with known antimicrobials and represent novel antibiotics. Accordingly, the occurrence and conservation of proteinaceous antimicrobials throughout the fungal tree of life remains enigmatic. Here we present a computational approach to annotate candidate antimicrobial effectors in fungal secretomes based on protein physicochemical properties. After curating a set of proteins that were experimentally verified to display antimicrobial activity and a set of proteins that lack such activity, we trained a machine learning classifier on properties of protein sequences and predicted structures. This predictor performs particularly well on fungal proteins (R2=0.89) according to our validations and is delivered as a software package named AMAPEC, dedicated to antimicrobial activity prediction for effector candidates. We subsequently used this novel software to predict antimicrobial effector catalogs in three phylogenetically distant fungi with distinct lifestyles, revealing relatively large catalogs of candidate antimicrobials for each of the three fungi, and suggesting a broad occurrence of such proteins throughout the fungal kingdom. Thus, AMAPEC is a unique method to uncover antimicrobials in fungal secretomes that are often sparsely functionally annotated, and may assist biological interpretations during omic analyses. It is freely available at https://github.com/fantin-mesny/amapec.\nTIR domains produce histidine-ADPR conjugates as immune signaling molecules in bacteria\nAuthors: Sabonis, D.; Avraham, C.; Lu, A.; Herbst, E.; Silanskas, A.; Leavitt, A.; Yirmiya, E.; Zaremba, M.; Amitai, G.; Kranzusch, P. J.; Sorek, R.; Tamulaitiene, G.\nScore: 38.6, Published: 2024-01-03 DOI: 10.1101/2024.01.03.573942\nTIR domains are central components of pattern recognition immune proteins across all domains of life. In both bacteria and plants, TIR-domain proteins were shown to recognize pathogen invasion and then produce immune signaling molecules exclusively comprising nucleotide moieties. Here we show that the TIR domain protein of the type II Thoeris defense system in bacteria produces a unique signaling molecule comprising the amino acid histidine conjugated to ADP-ribose (His-ADPR). His-ADPR is generated in response to phage infection and activates the cognate Thoeris effector by binding a Macro domain located at the C-terminus of the effector protein. By determining the crystal structure of a ligand-bound Macro domain, we describe the structural basis for His-ADPR recognition. Our analyses furthermore reveal a family of phage proteins that bind and sequester His-ADPR signaling molecules, allowing phages to evade TIR- mediated immunity. These data demonstrate diversity in bacterial TIR signaling and reveal a new class of TIR-derived immune signaling molecules combining nucleotide and amino acid moieties.\nQuantifying the daily harvest of fermentation products from the human gut microbiota\nAuthors: Arnoldini, M.; Sharma, R.; Moresi, C.; Chure, G.; Chabbey, J.; Slack, E.; Cremer, J.\nScore: 31.2, Published: 2024-01-06 DOI: 10.1101/2024.01.05.573977\nFermentation products released by the gut microbiota provide energy and important regulatory functions for the host. Yet, little quantitative information is available on the metabolite exchange between the microbiota and the human host, and thus the effective doses of fermentation products. Here, we introduce an integrative framework combining experimental characterization of major gut bacteria with a quantitative analysis of human digestive physiology to put numbers on this exchange and its dependence on diet and microbiota composition. From the carbohydrates fueling microbiota growth, we find most carbon ends up in fermentation products which are largely utilized by the host. This harvest varies strongly with diet, from between 100-700 mmol/day within the US population to up to 1300 for the Hadza people of Tanzania. Accordingly, fermentation products cover between 1.8% and 12.1% of the daily energy demand of human hosts, substantially less than the 21% estimated for laboratory mice. In contrast, microbiota composition has little impact on the total daily harvest but determines the harvest of specific fermentation products. Butyrate, known for promoting epithelial health, shows the biggest variation. Our framework thus identifies and quantifies major factors driving the metabolic and signaling interactions between microbiota and host, crucial to mechanistically dissect the role of the microbiota in health and disease.\nPreferential apical infection of intestinal cell monolayers by SARS-CoV-2 is associated with damage to cellular barrier integrity: Implications for the physiopathology of COVID-19\nAuthors: Clemence, G.; Jeffrey, A.; Jonatane, A.; Mege, J.-L.; Omar Osman, I.; Devaux, C. A.\nScore: 25.0, Published: 2024-01-09 DOI: 10.1101/2024.01.08.574642\nSARS-CoV-2 can infect different organs, including the intestine. In Caco-2 intestinal cell line, SARS-CoV-2 modulates the ACE2 receptor expression and affects the expression of molecules involved in intercellular junctions. To further explore the possibility that the intestinal epithelium serves as an alternative infection route for SARS-CoV-2, we used a model of polarised intestinal cell monolayers grown on the polycarbonate membrane of Transwell inserts, inoculated with the virus either in the upper or lower chamber of culture. In both polarised Caco-2 cell monolayers and co-culture Caco-2/HT29 cell monolayer, apical SARS-CoV-2 inoculation was found to be much more effective in establishing infection than basolateral inoculation. In addition, apical SARS-CoV-2 infection triggers monolayer degeneration, as shown by histological examination, measurement of trans-epithelial electronic resistance, and cell adhesion molecule expression. During this process, the infectious viruses reach the lower chamber, suggesting either a transcytosis mechanism from the apical side to the basolateral side of cells, a paracellular trafficking of the virus after damage to intercellular junctions in the epithelial barrier, or both. Taken together, these data highlight a preferential tropism of SARS-CoV-2 for the apical side of the human intestinal tract and suggests that infection via the intestinal lumen leads to a systemic infection.\nVaginal Lactobacillus fatty acid response mechanisms reveal a novel strategy for bacterial vaginosis treatment\nAuthors: Zhu, M.; Frank, M. W.; Radka, C. D.; Jeanfavre, S.; Tse, M. W.; Avila Pacheco, J.; Pierce, K.; Deik, A.; Xhu, J.; Hussain, S.; Hussain, F. A.; Xulu, N.; Khan, N.; Pillay, V.; Dong, K. L.; Ndung'u, T.; Clish, C. B.; Rock, C. O.; Blainey, P. C.; Bloom, S. M.; Kwon, D. S.\nScore: 21.0, Published: 2023-12-30 DOI: 10.1101/2023.12.30.573720\nBacterial vaginosis (BV), a common syndrome characterized by Lactobacillus-deficient vaginal microbiota, is associated with adverse health outcomes. BV often recurs after standard antibiotic therapy in part because antibiotics promote microbiota dominance by Lactobacillus iners instead of Lactobacillus crispatus, which has more beneficial health associations. Strategies to promote L. crispatus and inhibit L. iners are thus needed. We show that oleic acid (OA) and similar long-chain fatty acids simultaneously inhibit L. iners and enhance L. crispatus growth. These phenotypes require OA-inducible genes conserved in L. crispatus and related species, including an oleate hydratase (ohyA) and putative fatty acid efflux pump (farE). FarE mediates OA resistance, while OhyA is robustly active in the human vaginal microbiota and sequesters OA in a derivative form that only ohyA-harboring organisms can exploit. Finally, OA promotes L. crispatus dominance more effectively than antibiotics in an in vitro model of BV, suggesting a novel approach for treatment.\nCharacterization and genomic analysis of the Lyme disease spirochete bacteriophage phiBB-1\nAuthors: Faith, D. R.; Kinnersley, M.; Brooks, D. M.; Drecktrah, D.; Hall, L. S.; Luo, E.; Santiago-Frangos, A.; Wachter, J.; Samuels, S.; Secor, P. R.\nScore: 17.5, Published: 2024-01-09 DOI: 10.1101/2024.01.08.574763\nLyme disease is a tick-borne infection caused by the spirochete Borrelia (Borreliella) burgdorferi. Borrelia species have highly fragmented genomes composed of a linear chromosome and a constellation of linear and circular plasmids that encode diverse outer membrane lipoproteins, which facilitate movement of the spirochete between its tick vector and a vertebrate host in an enzootic cycle. The B. burgdorferi genome shows evidence of horizontal transfer between strains, but the mechanisms remain poorly defined. Almost all Lyme disease spirochetes are infected by 32-kp circular plasmid (cp32) prophages that undergo lytic replication and produce infectious virions called phiBB-1. In the laboratory, phiBB-1 transduces cp32s and shuttle vectors between spirochetes. However, the extent that phiBB-1 participates in horizontal gene transfer between Lyme disease spirochetes is not known. Here, we use proteomics and long-read sequencing to characterize phiBB-1 virions and the genetic material they package. Our studies reveal that phiBB-1 packages linear cp32s via a headful mechanism. We identify the cp32 pac region and show that plasmids containing the cp32 pac region are preferentially packaged into phiBB-1 virions. Additionally, we find phiBB-1 packages fragments of the linear chromosome and other plasmids including lp54, cp26, and others. Furthermore, sequencing of phiBB-1 packaged DNA allowed us to resolve the covalently closed hairpin telomeres for the linear B. burgdorferi chromosome and most linear plasmids in strain CA-11.2A. Collectively, our results shed light on the biology of the ubiquitous phiBB-1 phage and further implicates phiBB-1 in the generalized transduction of diverse genes between spirochetes and the maintenance of genetic diversity in Lyme disease spirochetes.\nSequence-based modelling of bacterial genomes enables accurate antibiotic resistance prediction\nAuthors: Wiatrak, M.; Weimann, A.; Dinan, A. M.; Brbic, M.; Floto, R. A.\nScore: 15.4, Published: 2024-01-03 DOI: 10.1101/2024.01.03.574022\nAbstractRapid detection of antibiotic-resistant bacteria and understanding the mecha- nisms underlying antimicrobial resistance (AMR) are major unsolved problems that pose significant threats to global public health. However, existing methods for predicting antibiotic resistance from genomic sequence data have had lim- ited success due to their inability to model epistatic effects and generalize to novel variants. Here, we present GeneBac, a deep learning method for predicting antibiotic resistance from DNA sequence through the integration of interactions between genes. We apply GeneBac to two distinct bacterial species and show that it can successfully predict the minimum inhibitory concentration (MIC) of multiple antibiotics. We use the WHO Mycobacterium tuberculosis mutation cat- alogue to demonstrate that GeneBac accurately predicts the effects of different variants, including novel variants that have not been observed during training. GeneBac is a modular framework which can be applied to a number of tasks including gene expression prediction, resistant gene identification and strain clus- tering. We leverage this modularity to transfer learn from the transcriptomic data to improve performance on the MIC prediction task.\nCRISPR-repressed toxin-antitoxin provides population-level immunity against diverse anti-CRISPR elements\nAuthors: Li, M.; shu, x.; Wang, R.; Li, Z.; Xue, Q.; Liu, C.; Cheng, F.; Zhao, H.; Wang, J.; Liu, J.; Hu, C.; Li, J.; Ouyang, S.\nScore: 12.9, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574328\nProkaryotic CRISPR-Cas systems are highly vulnerable to phage-encoded anti-CRISPR (Acr) factors. How CRISPR-Cas systems protect themselves remains unclear. Here, we uncovered a broad-spectrum anti-anti-CRISPR strategy involving a phage-derived toxic protein. Transcription of this toxin is normally reppressed by the CRISPR-Cas effector, but is activated to halt cell division when the effector is inhibited by any anti-CRISPR proteins or RNAs. We showed that this abortive infection-like effect efficiently expels Acr elements from bacterial population. Furthermore, we exploited this anti-anti-CRISPR mechanism to develop a screening method for specific Acr candidates for a CRISPR-Cas system, and successfully identified two distinct Acr proteins that enhance the binding of CRISPR effector to non-target DNA. Our data highlight the broad-spectrum role of CRISPR-repressed toxins in counteracting various types of Acr factors, which illuminates that the regulatory function of CRISPR-Cas confers host cells herd immunity against Acr-encoding genetic invaders, no matter they are CRISPR-targeted or not.\n",
  "wordCount" : "2318",
  "inLanguage": "en",
  "datePublished": "2024-01-10T10:40:05Z",
  "dateModified": "2024-01-10T10:40:05Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 10, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574008">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574008" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574008">
        <p class="paperTitle">Lethal Infection of Human ACE2-Transgenic Mice Caused by SARS-CoV-2-related Pangolin Coronavirus GX_P2V(short_3UTR)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574008" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574008" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wei, L.; Liu, S.; Lu, S.; Luo, S.; An, X.; Fan, H.; Chen, W.; Li, E.; Tong, Y.; Song, L.</p>
        <p class="info">Score: 1039.7, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574008' target='https://doi.org/10.1101/2024.01.03.574008'> 10.1101/2024.01.03.574008</a></p>
        <p class="abstract">SARS-CoV-2-related pangolin coronavirus GX_P2V(short_3UTR) can cause 100% mortality in human ACE2-transgenic mice, potentially attributable to late-stage brain infection. This underscores a spillover risk of GX_P2V into humans and provides a unique model for understanding the pathogenic mechanisms of SARS-CoV-2-related viruses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574420">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574420" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574420">
        <p class="paperTitle">Rapid Emergence and Evolution of SARS-CoV-2 Variants in Advanced HIV Infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574420" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574420" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ko, S. H.; Radecki, P.; Belinky, F.; Bhiman, J.; Meiring, S.; Kleynhans, J.; Amoako, D.; Guerra Canedo, V.; Lucas, M.; Kekana, D.; Martinson, N.; Lebina, L.; Everatt, J.; Tempia, S.; Bylund, T.; Rawi, R.; Kwong, P.; Wolter, N.; von Gottberg, A.; Cohen, C.; Boritz, E. A.</p>
        <p class="info">Score: 77.7, Published: 2024-01-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574420' target='https://doi.org/10.1101/2024.01.05.574420'> 10.1101/2024.01.05.574420</a></p>
        <p class="abstract">Previous studies have linked the evolution of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) genetic variants to persistent infections in people with immunocompromising conditions1-4, but the evolutionary processes underlying these observations are incompletely understood. Here we used high-throughput, single-genome amplification and sequencing (HT-SGS) to obtain up to [~]103 SARS-CoV-2 spike gene sequences in each of 184 respiratory samples from 22 people with HIV (PWH) and 25 people without HIV (PWOH). Twelve of 22 PWH had advanced HIV infection, defined by peripheral blood CD4 T cell counts (i.e., CD4 counts) &lt;200 cells/L. In PWOH and PWH with CD4 counts [&amp;ge;]200 cells/L, most single-genome spike sequences in each person matched one haplotype that predominated throughout the infection. By contrast, people with advanced HIV showed elevated intra-host spike diversity with a median of 46 haplotypes per person (IQR 14-114). Higher intra-host spike diversity immediately after COVID-19 symptom onset predicted longer SARS-CoV-2 RNA shedding among PWH, and intra-host spike diversity at this timepoint was significantly higher in people with advanced HIV than in PWOH. Composition of spike sequence populations in people with advanced HIV fluctuated rapidly over time, with founder sequences often replaced by groups of new haplotypes. These population-level changes were associated with a high total burden of intra-host mutations and positive selection at functionally important residues. In several cases, delayed emergence of detectable serum binding to spike was associated with positive selection for presumptive antibody-escape mutations. Taken together, our findings show remarkable intra-host genetic diversity of SARS-CoV-2 in advanced HIV infection and suggest that adaptive intra-host SARS-CoV-2 evolution in this setting may contribute to the emergence of new variants of concern (VOCs).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.574150">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.574150" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.574150">
        <p class="paperTitle">AMAPEC: accurate antimicrobial activity prediction for fungal effector proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.574150" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.574150" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mesny, F.; Thomma, B. P.</p>
        <p class="info">Score: 54.6, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.574150' target='https://doi.org/10.1101/2024.01.04.574150'> 10.1101/2024.01.04.574150</a></p>
        <p class="abstract">Fungi typically occur in environments where numerous and diverse other microbes occur as well, often resulting in fierce competition for nutrients and habitat. To support fungal fitness in these environments, they evolved various mechanisms that mediate direct antagonism towards niche competitors. Among these, the secretion of proteins with antimicrobial activities has been reported in fungi with diverse lifestyles. Recently, several plant-associated fungi were shown to rely on the secretion of antimicrobial effector proteins to antagonize certain members of plant hosts microbiota and to successfully colonize plant tissues. Some of these effectors do not share homology with known antimicrobials and represent novel antibiotics. Accordingly, the occurrence and conservation of proteinaceous antimicrobials throughout the fungal tree of life remains enigmatic. Here we present a computational approach to annotate candidate antimicrobial effectors in fungal secretomes based on protein physicochemical properties. After curating a set of proteins that were experimentally verified to display antimicrobial activity and a set of proteins that lack such activity, we trained a machine learning classifier on properties of protein sequences and predicted structures. This predictor performs particularly well on fungal proteins (R2=0.89) according to our validations and is delivered as a software package named AMAPEC, dedicated to antimicrobial activity prediction for effector candidates. We subsequently used this novel software to predict antimicrobial effector catalogs in three phylogenetically distant fungi with distinct lifestyles, revealing relatively large catalogs of candidate antimicrobials for each of the three fungi, and suggesting a broad occurrence of such proteins throughout the fungal kingdom. Thus, AMAPEC is a unique method to uncover antimicrobials in fungal secretomes that are often sparsely functionally annotated, and may assist biological interpretations during omic analyses. It is freely available at https://github.com/fantin-mesny/amapec.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.573942">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.573942" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.573942">
        <p class="paperTitle">TIR domains produce histidine-ADPR conjugates as immune signaling molecules in bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.573942" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.573942" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sabonis, D.; Avraham, C.; Lu, A.; Herbst, E.; Silanskas, A.; Leavitt, A.; Yirmiya, E.; Zaremba, M.; Amitai, G.; Kranzusch, P. J.; Sorek, R.; Tamulaitiene, G.</p>
        <p class="info">Score: 38.6, Published: 2024-01-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.573942' target='https://doi.org/10.1101/2024.01.03.573942'> 10.1101/2024.01.03.573942</a></p>
        <p class="abstract">TIR domains are central components of pattern recognition immune proteins across all domains of life. In both bacteria and plants, TIR-domain proteins were shown to recognize pathogen invasion and then produce immune signaling molecules exclusively comprising nucleotide moieties. Here we show that the TIR domain protein of the type II Thoeris defense system in bacteria produces a unique signaling molecule comprising the amino acid histidine conjugated to ADP-ribose (His-ADPR). His-ADPR is generated in response to phage infection and activates the cognate Thoeris effector by binding a Macro domain located at the C-terminus of the effector protein. By determining the crystal structure of a ligand-bound Macro domain, we describe the structural basis for His-ADPR recognition. Our analyses furthermore reveal a family of phage proteins that bind and sequester His-ADPR signaling molecules, allowing phages to evade TIR- mediated immunity. These data demonstrate diversity in bacterial TIR signaling and reveal a new class of TIR-derived immune signaling molecules combining nucleotide and amino acid moieties.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.573977">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.573977" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.573977">
        <p class="paperTitle">Quantifying the daily harvest of fermentation products from the human gut microbiota</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.573977" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.573977" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Arnoldini, M.; Sharma, R.; Moresi, C.; Chure, G.; Chabbey, J.; Slack, E.; Cremer, J.</p>
        <p class="info">Score: 31.2, Published: 2024-01-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.573977' target='https://doi.org/10.1101/2024.01.05.573977'> 10.1101/2024.01.05.573977</a></p>
        <p class="abstract">Fermentation products released by the gut microbiota provide energy and important regulatory functions for the host. Yet, little quantitative information is available on the metabolite exchange between the microbiota and the human host, and thus the effective doses of fermentation products. Here, we introduce an integrative framework combining experimental characterization of major gut bacteria with a quantitative analysis of human digestive physiology to put numbers on this exchange and its dependence on diet and microbiota composition. From the carbohydrates fueling microbiota growth, we find most carbon ends up in fermentation products which are largely utilized by the host. This harvest varies strongly with diet, from between 100-700 mmol/day within the US population to up to 1300 for the Hadza people of Tanzania. Accordingly, fermentation products cover between 1.8% and 12.1% of the daily energy demand of human hosts, substantially less than the 21% estimated for laboratory mice. In contrast, microbiota composition has little impact on the total daily harvest but determines the harvest of specific fermentation products. Butyrate, known for promoting epithelial health, shows the biggest variation. Our framework thus identifies and quantifies major factors driving the metabolic and signaling interactions between microbiota and host, crucial to mechanistically dissect the role of the microbiota in health and disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.574642">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.574642" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.574642">
        <p class="paperTitle">Preferential apical infection of intestinal cell monolayers by SARS-CoV-2 is associated with damage to cellular barrier integrity: Implications for the physiopathology of COVID-19</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.574642" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.574642" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Clemence, G.; Jeffrey, A.; Jonatane, A.; Mege, J.-L.; Omar Osman, I.; Devaux, C. A.</p>
        <p class="info">Score: 25.0, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.574642' target='https://doi.org/10.1101/2024.01.08.574642'> 10.1101/2024.01.08.574642</a></p>
        <p class="abstract">SARS-CoV-2 can infect different organs, including the intestine. In Caco-2 intestinal cell line, SARS-CoV-2 modulates the ACE2 receptor expression and affects the expression of molecules involved in intercellular junctions. To further explore the possibility that the intestinal epithelium serves as an alternative infection route for SARS-CoV-2, we used a model of polarised intestinal cell monolayers grown on the polycarbonate membrane of Transwell inserts, inoculated with the virus either in the upper or lower chamber of culture. In both polarised Caco-2 cell monolayers and co-culture Caco-2/HT29 cell monolayer, apical SARS-CoV-2 inoculation was found to be much more effective in establishing infection than basolateral inoculation. In addition, apical SARS-CoV-2 infection triggers monolayer degeneration, as shown by histological examination, measurement of trans-epithelial electronic resistance, and cell adhesion molecule expression. During this process, the infectious viruses reach the lower chamber, suggesting either a transcytosis mechanism from the apical side to the basolateral side of cells, a paracellular trafficking of the virus after damage to intercellular junctions in the epithelial barrier, or both. Taken together, these data highlight a preferential tropism of SARS-CoV-2 for the apical side of the human intestinal tract and suggests that infection via the intestinal lumen leads to a systemic infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.30.573720">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.30.573720" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.30.573720">
        <p class="paperTitle">Vaginal Lactobacillus fatty acid response mechanisms reveal a novel strategy for bacterial vaginosis treatment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.30.573720" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.30.573720" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhu, M.; Frank, M. W.; Radka, C. D.; Jeanfavre, S.; Tse, M. W.; Avila Pacheco, J.; Pierce, K.; Deik, A.; Xhu, J.; Hussain, S.; Hussain, F. A.; Xulu, N.; Khan, N.; Pillay, V.; Dong, K. L.; Ndung&#39;u, T.; Clish, C. B.; Rock, C. O.; Blainey, P. C.; Bloom, S. M.; Kwon, D. S.</p>
        <p class="info">Score: 21.0, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.30.573720' target='https://doi.org/10.1101/2023.12.30.573720'> 10.1101/2023.12.30.573720</a></p>
        <p class="abstract">Bacterial vaginosis (BV), a common syndrome characterized by Lactobacillus-deficient vaginal microbiota, is associated with adverse health outcomes. BV often recurs after standard antibiotic therapy in part because antibiotics promote microbiota dominance by Lactobacillus iners instead of Lactobacillus crispatus, which has more beneficial health associations. Strategies to promote L. crispatus and inhibit L. iners are thus needed. We show that oleic acid (OA) and similar long-chain fatty acids simultaneously inhibit L. iners and enhance L. crispatus growth. These phenotypes require OA-inducible genes conserved in L. crispatus and related species, including an oleate hydratase (ohyA) and putative fatty acid efflux pump (farE). FarE mediates OA resistance, while OhyA is robustly active in the human vaginal microbiota and sequesters OA in a derivative form that only ohyA-harboring organisms can exploit. Finally, OA promotes L. crispatus dominance more effectively than antibiotics in an in vitro model of BV, suggesting a novel approach for treatment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.574763">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.574763" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.574763">
        <p class="paperTitle">Characterization and genomic analysis of the Lyme disease spirochete bacteriophage phiBB-1</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.574763" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.574763" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Faith, D. R.; Kinnersley, M.; Brooks, D. M.; Drecktrah, D.; Hall, L. S.; Luo, E.; Santiago-Frangos, A.; Wachter, J.; Samuels, S.; Secor, P. R.</p>
        <p class="info">Score: 17.5, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.574763' target='https://doi.org/10.1101/2024.01.08.574763'> 10.1101/2024.01.08.574763</a></p>
        <p class="abstract">Lyme disease is a tick-borne infection caused by the spirochete Borrelia (Borreliella) burgdorferi. Borrelia species have highly fragmented genomes composed of a linear chromosome and a constellation of linear and circular plasmids that encode diverse outer membrane lipoproteins, which facilitate movement of the spirochete between its tick vector and a vertebrate host in an enzootic cycle. The B. burgdorferi genome shows evidence of horizontal transfer between strains, but the mechanisms remain poorly defined. Almost all Lyme disease spirochetes are infected by 32-kp circular plasmid (cp32) prophages that undergo lytic replication and produce infectious virions called phiBB-1. In the laboratory, phiBB-1 transduces cp32s and shuttle vectors between spirochetes. However, the extent that phiBB-1 participates in horizontal gene transfer between Lyme disease spirochetes is not known. Here, we use proteomics and long-read sequencing to characterize phiBB-1 virions and the genetic material they package. Our studies reveal that phiBB-1 packages linear cp32s via a headful mechanism. We identify the cp32 pac region and show that plasmids containing the cp32 pac region are preferentially packaged into phiBB-1 virions. Additionally, we find phiBB-1 packages fragments of the linear chromosome and other plasmids including lp54, cp26, and others. Furthermore, sequencing of phiBB-1 packaged DNA allowed us to resolve the covalently closed hairpin telomeres for the linear B. burgdorferi chromosome and most linear plasmids in strain CA-11.2A. Collectively, our results shed light on the biology of the ubiquitous phiBB-1 phage and further implicates phiBB-1 in the generalized transduction of diverse genes between spirochetes and the maintenance of genetic diversity in Lyme disease spirochetes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574022">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574022" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574022">
        <p class="paperTitle">Sequence-based modelling of bacterial genomes enables accurate antibiotic resistance prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574022" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574022" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wiatrak, M.; Weimann, A.; Dinan, A. M.; Brbic, M.; Floto, R. A.</p>
        <p class="info">Score: 15.4, Published: 2024-01-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574022' target='https://doi.org/10.1101/2024.01.03.574022'> 10.1101/2024.01.03.574022</a></p>
        <p class="abstract">AbstractRapid detection of antibiotic-resistant bacteria and understanding the mecha- nisms underlying antimicrobial resistance (AMR) are major unsolved problems that pose significant threats to global public health. However, existing methods for predicting antibiotic resistance from genomic sequence data have had lim- ited success due to their inability to model epistatic effects and generalize to novel variants. Here, we present GeneBac, a deep learning method for predicting antibiotic resistance from DNA sequence through the integration of interactions between genes. We apply GeneBac to two distinct bacterial species and show that it can successfully predict the minimum inhibitory concentration (MIC) of multiple antibiotics. We use the WHO Mycobacterium tuberculosis mutation cat- alogue to demonstrate that GeneBac accurately predicts the effects of different variants, including novel variants that have not been observed during training. GeneBac is a modular framework which can be applied to a number of tasks including gene expression prediction, resistant gene identification and strain clus- tering. We leverage this modularity to transfer learn from the transcriptomic data to improve performance on the MIC prediction task.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574328">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574328" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574328">
        <p class="paperTitle">CRISPR-repressed toxin-antitoxin provides population-level immunity against diverse anti-CRISPR elements</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574328" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574328" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, M.; shu, x.; Wang, R.; Li, Z.; Xue, Q.; Liu, C.; Cheng, F.; Zhao, H.; Wang, J.; Liu, J.; Hu, C.; Li, J.; Ouyang, S.</p>
        <p class="info">Score: 12.9, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574328' target='https://doi.org/10.1101/2024.01.05.574328'> 10.1101/2024.01.05.574328</a></p>
        <p class="abstract">Prokaryotic CRISPR-Cas systems are highly vulnerable to phage-encoded anti-CRISPR (Acr) factors. How CRISPR-Cas systems protect themselves remains unclear. Here, we uncovered a broad-spectrum anti-anti-CRISPR strategy involving a phage-derived toxic protein. Transcription of this toxin is normally reppressed by the CRISPR-Cas effector, but is activated to halt cell division when the effector is inhibited by any anti-CRISPR proteins or RNAs. We showed that this abortive infection-like effect efficiently expels Acr elements from bacterial population. Furthermore, we exploited this anti-anti-CRISPR mechanism to develop a screening method for specific Acr candidates for a CRISPR-Cas system, and successfully identified two distinct Acr proteins that enhance the binding of CRISPR effector to non-target DNA. Our data highlight the broad-spectrum role of CRISPR-repressed toxins in counteracting various types of Acr factors, which illuminates that the regulatory function of CRISPR-Cas confers host cells herd immunity against Acr-encoding genetic invaders, no matter they are CRISPR-targeted or not.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
