<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Neutralization Escape by SARS-CoV-2 Omicron Subvariant BA.2.86
Authors: Lasrado, N.; Collier, A.-r.; Hachmann, N.; Miller, J.; Rowe, M.; Schonberg, E.; Rodrigues, S.; LaPiana, A.; Patio, R.; Anand, T.; Fisher, J.; Mazurek, C.; Guan, R.; Wagh, K.; Theiler, J.; Korber, B.; Barouch, D. H.
Score: 539.3, Published: 2023-09-05 DOI: 10.1101/2023.09.04.556272
The continued evolution of SARS-CoV-2 may lead to evasion of vaccine immunity and natural immunity. A highly mutated Omicron variant BA.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Neutralization Escape by SARS-CoV-2 Omicron Subvariant BA.2.86
Authors: Lasrado, N.; Collier, A.-r.; Hachmann, N.; Miller, J.; Rowe, M.; Schonberg, E.; Rodrigues, S.; LaPiana, A.; Patio, R.; Anand, T.; Fisher, J.; Mazurek, C.; Guan, R.; Wagh, K.; Theiler, J.; Korber, B.; Barouch, D. H.
Score: 539.3, Published: 2023-09-05 DOI: 10.1101/2023.09.04.556272
The continued evolution of SARS-CoV-2 may lead to evasion of vaccine immunity and natural immunity. A highly mutated Omicron variant BA." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-10T10:36:49+00:00" />
<meta property="article:modified_time" content="2023-09-10T10:36:49+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Neutralization Escape by SARS-CoV-2 Omicron Subvariant BA.2.86
Authors: Lasrado, N.; Collier, A.-r.; Hachmann, N.; Miller, J.; Rowe, M.; Schonberg, E.; Rodrigues, S.; LaPiana, A.; Patio, R.; Anand, T.; Fisher, J.; Mazurek, C.; Guan, R.; Wagh, K.; Theiler, J.; Korber, B.; Barouch, D. H.
Score: 539.3, Published: 2023-09-05 DOI: 10.1101/2023.09.04.556272
The continued evolution of SARS-CoV-2 may lead to evasion of vaccine immunity and natural immunity. A highly mutated Omicron variant BA."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Neutralization Escape by SARS-CoV-2 Omicron Subvariant BA.2.86\nAuthors: Lasrado, N.; Collier, A.-r.; Hachmann, N.; Miller, J.; Rowe, M.; Schonberg, E.; Rodrigues, S.; LaPiana, A.; Patio, R.; Anand, T.; Fisher, J.; Mazurek, C.; Guan, R.; Wagh, K.; Theiler, J.; Korber, B.; Barouch, D. H.\nScore: 539.3, Published: 2023-09-05 DOI: 10.1101/2023.09.04.556272\nThe continued evolution of SARS-CoV-2 may lead to evasion of vaccine immunity and natural immunity. A highly mutated Omicron variant BA.",
  "keywords": [
    
  ],
  "articleBody": " Neutralization Escape by SARS-CoV-2 Omicron Subvariant BA.2.86\nAuthors: Lasrado, N.; Collier, A.-r.; Hachmann, N.; Miller, J.; Rowe, M.; Schonberg, E.; Rodrigues, S.; LaPiana, A.; Patio, R.; Anand, T.; Fisher, J.; Mazurek, C.; Guan, R.; Wagh, K.; Theiler, J.; Korber, B.; Barouch, D. H.\nScore: 539.3, Published: 2023-09-05 DOI: 10.1101/2023.09.04.556272\nThe continued evolution of SARS-CoV-2 may lead to evasion of vaccine immunity and natural immunity. A highly mutated Omicron variant BA.2.86 has recently been identified with over 30 amino acid changes in Spike compared with BA.2 and XBB.1.5. As of September 4, 2023, BA.2.86 has been identified in 37 sequences from 10 countries, which is likely an underestimate due to limited surveillance. The ability of BA.2.86 to evade NAbs compared with other currently circulating Omicron variants remains unknown. Our data show that NAb responses to BA.2.86 were lower than to BA.2 but were comparable or slightly higher than to the current circulating recombinant variants XBB.1.5, XBB.1.16, EG.5, EG.5.1, and FL.1.5.1.\nTransmissibility, infectivity, and immune resistance of the SARS-CoV-2 BA.2.86 variant\nAuthors: Uriu, K.; Ito, J.; Kosugi, Y.; Tanaka, Y. L.; Mugita, Y.; Guo, Z.; Hinay, A. A.; Putri, O.; Kim, Y.; Shimizu, R.; Begum, M. M.; Jonathan, M.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Saito, A.; Ikeda, T.; Sato, K.\nScore: 287.5, Published: 2023-09-07 DOI: 10.1101/2023.09.07.556636\nIn September 2023, the SARS-CoV-2 XBB descendants, such as XBB.1.5 and EG.5.1 (originally XBB.1.9.2.5.1), are predominantly circulating worldwide. Unexpectedly, however, a lineage distinct from XBB was identified and named BA.2.86 on August 14, 2023. Notably, BA.2.86 bears more than 30 mutations in the spike (S) protein when compared to XBB and the parental BA.2, and many of them are assumed to be associated with immune evasion. Although the number of reported cases is low (68 sequences have been reported as of 7 September 2023), BA.2.86 has been detected in several continents (Europe, North America and Africa), suggesting that this variant may be spreading silently worldwide. On 17 August 2023, the WHO designated BA.2.86 as a variant under monitoring. Here we show evidence suggesting that BA.2.86 potentially has greater fitness than current circulating XBB variants including EG.5.1. The pseudovirus assay showed that the infectivity of BA.2.86 was significantly lower than that of B.1.1 and EG.5.1, suggesting that the increased fitness of BA.2.86 is not due to the increased infectivity. We then performed a neutralization assay using XBB breakthrough infection sera to address whether BA.2.86 evades the antiviral effect of the humoral immunity induced XBB subvariants. The 50% neutralization titer of XBB BTI sera against BA.2.86 was significantly (1.4-fold) lower than those against EG.5.1. The sera obtained from individuals vaccinated with 3rd-dose monovalent, 4th-dose monovalent, 4th-dose BA.1 bivalent, and 4th-dose BA.5 bivalent mRNA vaccines exhibited very little or no antiviral effects against BA.2.86. Moreover, the three monoclonal antibodies (Bebtelovimab, Sotrovimab and Tixagevimab), which worked against the parental BA.2, did not exhibit antiviral effects against BA.2.86. These results suggest that BA.2.86 is one of the most highly immune evasive variants ever.\nCharacterization of an EG.5.1 clinical isolate in vitro and in vivo\nAuthors: Uraki, R.; Kiso, M.; Iwatsuki-Horimoto, K.; Yamayoshi, S.; Ito, M.; Chiba, S.; Sakai-Tagawa, Y.; Imai, M.; Kashima, Y.; Koga, M.; Fuwa, N.; Okumura, N.; Hojo, M.; Iwamoto, N.; Kato, H.; Nakajima, H.; Ohmagari, N.; Yotsuyanagi, H.; Suzuki, Y.; Kawaoka, Y.\nScore: 271.1, Published: 2023-09-01 DOI: 10.1101/2023.08.31.555819\nEG.5.1 is a subvariant of the SARS-CoV-2 Omicron XBB variant that is rapidly increasing in prevalence worldwide. EG.5.1 has additional substitutions in its spike protein (namely, Q52H and F456L) compared with XBB.1.5. However, the pathogenicity, transmissibility, and immune evasion properties of clinical isolates of EG.5.1 are largely unknown. In this study, we used wild-type Syrian hamsters to investigate the replicative ability, pathogenicity, and transmissibility of a clinical EG.5.1 isolate. Our data show that there are no obvious differences in growth ability and pathogenicity between EG.5.1 and XBB.1.5, and both EG.5.1 and XBB.1.5 are attenuated compared to a Delta variant isolate. We also found that EG.5.1 is transmitted more efficiently between hamsters compared with XBB.1.5. In addition, unlike XBB.1.5, we detected EG.5.1 virus in the lungs of four of six exposed hamsters, suggesting that the virus tropism of EG.5.1 is different from that of XBB.1.5 after airborne transmission. Finally, we assessed the neutralizing ability of plasma from convalescent individuals and found that the neutralizing activity against EG.5.1 was slightly, but significantly, lower than that against XBB.1.5 or XBB.1.9.2. This suggests that EG.5.1 effectively evades humoral immunity and that the amino acid differences in the S protein of EG.5.1 compared with that of XBB.1.5 or XBB.1.9.2 (i.e., Q52H, R158G, and F456L) alter the antigenicity of EG.5.1. Our data suggest that the increased transmissibility and altered antigenicity of EG.5.1 may be driving its increasing prevalence over XBB.1.5 in the human population.\nBacteria conjugate ubiquitin-like proteins to interfere with phage assembly\nAuthors: Hör, J.; Wolf, S. G.; Sorek, R.\nScore: 86.3, Published: 2023-09-04 DOI: 10.1101/2023.09.04.556158\nMultiple immune pathways in humans conjugate ubiquitin-like proteins to virus and host molecules as a means of antiviral defense. Here we studied an anti-phage defense system in bacteria, comprising a ubiquitin-like protein, ubiquitin-conjugating enzymes E1 and E2, and a deubiquitinase. We show that during phage infection, this system specifically conjugates the ubiquitin-like protein to the phage central tail fiber, a protein at the tip of the tail that is essential for tail assembly as well as for recognition of the target host receptor. Following infection, cells encoding this defense system release a mixture of partially assembled, tailless phage particles, and fully assembled phages in which the central tail fiber is obstructed by the covalently attached ubiquitin-like protein. These phages exhibit severely impaired infectivity, explaining how the defense system protects the bacterial population from the spread of phage infection. Our findings demonstrate that conjugation of ubiquitin-like proteins is an antiviral strategy conserved across the tree of life.\nCommunity ecology of phages on a clonal bacterial host\nAuthors: Pyenson, N. C.; Leeks, A.; Nweke, O.; Goldford, J.; Turner, P.; Foster, K. R.; Sanchez, A.\nScore: 70.7, Published: 2023-09-01 DOI: 10.1101/2023.09.01.555216\nBacteriophages are the most abundant and diverse biological entities on Earth, yet the ecological mechanisms that sustain this extraordinary diversity remain unclear. Here, we have discovered a general mechanism that allows phage diversity to outstrip the diversity of their bacterial hosts. We assembled and passaged dozens of diverse phage communities on a single, non-coevolving strain of Escherichia coli until the phage communities reached equilibrium. In all cases, we found multiple phage species coexisted stably, despite competition for a single, clonal host population. Coexistence within these communities was supported through host phenotypic heterogeneity, whereby phages specialized on cells adopting different growth phenotypes. Our experiments reveal a rich community ecology of bacteriophages where multiple phage species coexist and interact at the scale of a single bacterial host. One-Sentence SummaryDiverse communities of phages emerge stably and consistently on a clonal bacterial host, enabled by phenotypic heterogeneity.\nProtein coopted from a phage restriction system dictates orthogonal cell division plane selection in Staphylococcus aureus\nAuthors: Ramos-Leon, F.; Anjuwon-Foster, B. R.; Anantharaman, V.; Ferreira, C. N.; Ibrahim, A. M.; Tai, C.-H.; Missiakas, D. M.; Camberg, J.; Aravind, L.; Ramamurthi, K. S.\nScore: 38.1, Published: 2023-09-03 DOI: 10.1101/2023.09.03.556088\nThe spherical bacterium Staphylococcus aureus, a leading cause of nosocomial infections, undergoes binary fission by dividing in two alternating orthogonal planes, but the mechanism by which S. aureus correctly selects the next cell division plane is not known. To identify cell division placement factors, we performed a chemical genetic screen that revealed a gene which we termed pcdA. We show that PcdA is a member of the McrB family of AAA+ NTPases that has undergone structural changes and a concomitant functional shift from a restriction enzyme subunit to an early cell division protein. PcdA directly interacts with the tubulin-like central divisome component FtsZ and localizes to future cell division sites before membrane invagination initiates. This parallels the action of another McrB family protein, CTTNBP2, which stabilizes microtubules in animals. We show that PcdA also interacts with the structural protein DivIVA and propose that the DivIVA/PcdA complex recruits unpolymerized FtsZ to assemble along the proper cell division plane. Deletion of pcdA conferred abnormal, non-orthogonal division plane selection, increased sensitivity to cell wall-targeting antibiotics, and reduced virulence in a murine infection model. Targeting PcdA could therefore highlight a treatment strategy for combatting antibiotic-resistant strains of S. aureus.\nSequential early-life viral infections modulate the microbiota and adaptive immune responses to systemic and mucosal vaccination\nAuthors: Li, Y.; Molleston, J. M.; Kim, A. H.; Ingle, H.; Aggarwal, S.; Nolan, L.; Hassan, A.; Foster, L.; Diamond, M.; Baldridge, M. T.\nScore: 41.2, Published: 2023-09-01 DOI: 10.1101/2023.08.31.555772\nIncreasing evidence points to the microbial exposome as a critical factor in maturing and shaping the host immune system, thereby influencing responses to immune challenges such as infections or vaccines. To investigate the effect of early-life viral exposures on immune development and vaccine responses, we inoculated mice with six distinct viral pathogens in sequence beginning in the neonatal period, and then evaluated their immune signatures before and after intramuscular or intranasal vaccination against SARS-CoV-2. Sequential viral infection drove profound changes in all aspects of the immune system, including increasing circulating leukocytes, altering innate and adaptive immune cell lineages in tissues, and markedly influencing serum cytokine and total antibody levels. Beyond these immune responses changes, these exposures also modulated the composition of the endogenous intestinal microbiota. Although sequentially-infected mice exhibited increased systemic immune activation and T cell responses after intramuscular and intranasal SARS-CoV-2 immunization, we observed decreased vaccine-induced antibody responses in these animals. These results suggest that early-life viral exposures are sufficient to diminish antibody responses to vaccination in mice, and highlight their potential importance of considering prior microbial exposures when investigating vaccine responses.\nSARS-CoV-2 hijacks fragile X mental retardation proteins for efficient infection\nAuthors: Garvanska, D.; Alvarado, R. E.; Mundt, F.; Nilsson, E.; Duel, J.; Coscia, F.; Lindqvist, R.; Lokugamage, K.; Johnson, B.; Plante, J. A.; Morris, D. R.; Vu, M. N.; Estes, L.; McLeland, A.; Walker, J.; Crocquet-Valdes, P.; Lopez-Mendez, B.; Plante, K.; Walker, D. H.; Weisser, M. B.; Overby, A. K.; Mann, M.; Menachery, V. D.; Nilsson, J.\nScore: 32.4, Published: 2023-09-01 DOI: 10.1101/2023.09.01.555899\nViruses interact with numerous host factors to facilitate viral replication and to dampen antiviral defense mechanisms. We currently have a limited mechanistic understanding of how SARS-CoV-2 binds host factors and the functional role of these interactions. Here, we uncover a novel interaction between the viral NSP3 protein and the fragile X mental retardation proteins (FMRPs: FMR1 and FXR1-2). SARS-CoV-2 NSP3 mutant viruses preventing FMRP binding have attenuated replication in vitro and have delayed disease onset in vivo. We show that a unique peptide motif in NSP3 binds directly to the two central KH domains of FMRPs and that this interaction is disrupted by the I304N mutation found in a patient with fragile X syndrome. NSP3 binding to FMRPs disrupts their interaction with the stress granule component UBAP2L through direct competition with a peptide motif in UBAP2L to prevent FMRP incorporation into stress granules. Collectively, our results provide novel insight into how SARS-CoV-2 hijacks host cell proteins for efficient infection and provides molecular insight to the possible underlying molecular defects in fragile X syndrome.\nCo-Occurrence Network Analysis Reveals The Alterations Of The Skin Microbiome And Metabolome In Atopic Dermatitis Patients\nAuthors: Portal Gomes, P. W.; Mannochio-Russo, H.; Mao, J.; Zhao, H. N.; Tipton, C. D.; Ancira, J.; Dorrestein, P. C.; Li, M.\nScore: 27.2, Published: 2023-08-18 DOI: 10.1101/2023.08.17.553735\nSkin microbiome can be altered in patients with Atopic Dermatitis (AD). An understanding of the changes from healthy to atopic skin can help develop new targets for better treatments and identify specific microbial or molecular biomarkers. This study investigates the skin microbiome and metabolome of healthy subjects and lesion (ADL) and non-lesion (ADNL) of AD patients by 16S rRNA gene sequencing and mass spectrometry, respectively. Samples from AD patients showed alterations in the diversity and composition of the skin microbiome. Staphylococcus species, especially S. aureus, were significantly increased in the ADL group. Metabolomic profiles were also different between the groups. Dipeptide-derived are more abundant in ADL, which may be related to skin inflammation. Co-occurrence network analysis was applied to integrate the microbiome and metabolomics data and revealed higher co-occurrence of metabolites and bacteria in healthy and ADNL compared to ADL. S. aureus co-occurred with dipeptide-derived in ADL, while phytosphingosine-derived compounds showed co-occurrences with commensal bacteria, e.g. Paracoccus sp., Pseudomonas sp., Prevotella bivia, Lactobacillus iners, Anaerococcus sp., Micrococcus sp., Corynebacterium ureicelerivorans, Corynebacterium massiliense, Streptococcus thermophilus, and Roseomonas mucosa, in healthy and ADNL groups. Therefore, these findings provide valuable insights into how AD affects the human skin metabolome and microbiome. ImportanceThis study provides valuable insight into changes in the skin microbiome and associated metabolomic profiles. It also identifies new therapeutic targets that may be useful for developing personalized treatments for individuals with atopic dermatitis based on their unique skin microbiome and metabolic profiles.\nComputational exploration of the global microbiome for antibiotic discovery\nAuthors: Celio Dias, S.-J.; Marcelo D. T., T.; Yiqian, D.; Alvaro, R. d. R.; Thomas S.B., S.; Hui, C.; Anthony, F.; Kuhn, M.; Chengkai, Z.; Amy, H.; Jelena, S.; Anna, V.; Xing-Ming, Z.; Peer, B.; Jaime, H.-C.; Cesar, d. l. F.-N.; Luis Pedro, C.\nScore: 20.9, Published: 2023-08-31 DOI: 10.1101/2023.08.31.555663\nNovel antibiotics are urgently needed to combat the antibiotic-resistance crisis. We present a machine learning-based approach to predict prokaryotic antimicrobial peptides (AMPs) by leveraging a vast dataset of 63,410 metagenomes and 87,920 microbial genomes. This led to the creation of AMPSphere, a comprehensive catalog comprising 863,498 non-redundant peptides, the majority of which were previously unknown. We observed that AMP production varies by habitat, with animal-associated samples displaying the highest proportion of AMPs compared to other habitats. Furthermore, within different human-associated microbiota, strain-level differences were evident. To validate our predictions, we synthesized and experimentally tested 50 AMPs, demonstrating their efficacy against clinically relevant drug-resistant pathogens both in vitro and in vivo. These AMPs exhibited antibacterial activity by targeting the bacterial membrane. Additionally, AMPSphere provides valuable insights into the evolutionary origins of peptides. In conclusion, our approach identified AMP sequences within prokaryotic microbiomes, opening up new avenues for the discovery of antibiotics.\n",
  "wordCount" : "2357",
  "inLanguage": "en",
  "datePublished": "2023-09-10T10:36:49Z",
  "dateModified": "2023-09-10T10:36:49Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta"><span>updated on September 10, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.04.556272">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.04.556272" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.04.556272">
        <p class="paperTitle">Neutralization Escape by SARS-CoV-2 Omicron Subvariant BA.2.86</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.04.556272" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.04.556272" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lasrado, N.; Collier, A.-r.; Hachmann, N.; Miller, J.; Rowe, M.; Schonberg, E.; Rodrigues, S.; LaPiana, A.; Patio, R.; Anand, T.; Fisher, J.; Mazurek, C.; Guan, R.; Wagh, K.; Theiler, J.; Korber, B.; Barouch, D. H.</p>
        <p class="info">Score: 539.3, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.04.556272' target='https://doi.org/10.1101/2023.09.04.556272'> 10.1101/2023.09.04.556272</a></p>
        <p class="abstract">The continued evolution of SARS-CoV-2 may lead to evasion of vaccine immunity and natural immunity. A highly mutated Omicron variant BA.2.86 has recently been identified with over 30 amino acid changes in Spike compared with BA.2 and XBB.1.5. As of September 4, 2023, BA.2.86 has been identified in 37 sequences from 10 countries, which is likely an underestimate due to limited surveillance. The ability of BA.2.86 to evade NAbs compared with other currently circulating Omicron variants remains unknown. Our data show that NAb responses to BA.2.86 were lower than to BA.2 but were comparable or slightly higher than to the current circulating recombinant variants XBB.1.5, XBB.1.16, EG.5, EG.5.1, and FL.1.5.1.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556636">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556636" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556636">
        <p class="paperTitle">Transmissibility, infectivity, and immune resistance of the SARS-CoV-2 BA.2.86 variant</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556636" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556636" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Uriu, K.; Ito, J.; Kosugi, Y.; Tanaka, Y. L.; Mugita, Y.; Guo, Z.; Hinay, A. A.; Putri, O.; Kim, Y.; Shimizu, R.; Begum, M. M.; Jonathan, M.; The Genotype to Phenotype Japan (G2P-Japan) Consortium,  ; Saito, A.; Ikeda, T.; Sato, K.</p>
        <p class="info">Score: 287.5, Published: 2023-09-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556636' target='https://doi.org/10.1101/2023.09.07.556636'> 10.1101/2023.09.07.556636</a></p>
        <p class="abstract">In September 2023, the SARS-CoV-2 XBB descendants, such as XBB.1.5 and EG.5.1 (originally XBB.1.9.2.5.1), are predominantly circulating worldwide. Unexpectedly, however, a lineage distinct from XBB was identified and named BA.2.86 on August 14, 2023. Notably, BA.2.86 bears more than 30 mutations in the spike (S) protein when compared to XBB and the parental BA.2, and many of them are assumed to be associated with immune evasion. Although the number of reported cases is low (68 sequences have been reported as of 7 September 2023), BA.2.86 has been detected in several continents (Europe, North America and Africa), suggesting that this variant may be spreading silently worldwide. On 17 August 2023, the WHO designated BA.2.86 as a variant under monitoring. Here we show evidence suggesting that BA.2.86 potentially has greater fitness than current circulating XBB variants including EG.5.1. The pseudovirus assay showed that the infectivity of BA.2.86 was significantly lower than that of B.1.1 and EG.5.1, suggesting that the increased fitness of BA.2.86 is not due to the increased infectivity. We then performed a neutralization assay using XBB breakthrough infection sera to address whether BA.2.86 evades the antiviral effect of the humoral immunity induced XBB subvariants. The 50% neutralization titer of XBB BTI sera against BA.2.86 was significantly (1.4-fold) lower than those against EG.5.1. The sera obtained from individuals vaccinated with 3rd-dose monovalent, 4th-dose monovalent, 4th-dose BA.1 bivalent, and 4th-dose BA.5 bivalent mRNA vaccines exhibited very little or no antiviral effects against BA.2.86. Moreover, the three monoclonal antibodies (Bebtelovimab, Sotrovimab and Tixagevimab), which worked against the parental BA.2, did not exhibit antiviral effects against BA.2.86. These results suggest that BA.2.86 is one of the most highly immune evasive variants ever.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.31.555819">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.31.555819" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.31.555819">
        <p class="paperTitle">Characterization of an EG.5.1 clinical isolate in vitro and in vivo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.31.555819" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.31.555819" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Uraki, R.; Kiso, M.; Iwatsuki-Horimoto, K.; Yamayoshi, S.; Ito, M.; Chiba, S.; Sakai-Tagawa, Y.; Imai, M.; Kashima, Y.; Koga, M.; Fuwa, N.; Okumura, N.; Hojo, M.; Iwamoto, N.; Kato, H.; Nakajima, H.; Ohmagari, N.; Yotsuyanagi, H.; Suzuki, Y.; Kawaoka, Y.</p>
        <p class="info">Score: 271.1, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.31.555819' target='https://doi.org/10.1101/2023.08.31.555819'> 10.1101/2023.08.31.555819</a></p>
        <p class="abstract">EG.5.1 is a subvariant of the SARS-CoV-2 Omicron XBB variant that is rapidly increasing in prevalence worldwide. EG.5.1 has additional substitutions in its spike protein (namely, Q52H and F456L) compared with XBB.1.5. However, the pathogenicity, transmissibility, and immune evasion properties of clinical isolates of EG.5.1 are largely unknown.

In this study, we used wild-type Syrian hamsters to investigate the replicative ability, pathogenicity, and transmissibility of a clinical EG.5.1 isolate. Our data show that there are no obvious differences in growth ability and pathogenicity between EG.5.1 and XBB.1.5, and both EG.5.1 and XBB.1.5 are attenuated compared to a Delta variant isolate.

We also found that EG.5.1 is transmitted more efficiently between hamsters compared with XBB.1.5. In addition, unlike XBB.1.5, we detected EG.5.1 virus in the lungs of four of six exposed hamsters, suggesting that the virus tropism of EG.5.1 is different from that of XBB.1.5 after airborne transmission.

Finally, we assessed the neutralizing ability of plasma from convalescent individuals and found that the neutralizing activity against EG.5.1 was slightly, but significantly, lower than that against XBB.1.5 or XBB.1.9.2. This suggests that EG.5.1 effectively evades humoral immunity and that the amino acid differences in the S protein of EG.5.1 compared with that of XBB.1.5 or XBB.1.9.2 (i.e., Q52H, R158G, and F456L) alter the antigenicity of EG.5.1.

Our data suggest that the increased transmissibility and altered antigenicity of EG.5.1 may be driving its increasing prevalence over XBB.1.5 in the human population.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.04.556158">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.04.556158" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.04.556158">
        <p class="paperTitle">Bacteria conjugate ubiquitin-like proteins to interfere with phage assembly</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.04.556158" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.04.556158" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hör, J.; Wolf, S. G.; Sorek, R.</p>
        <p class="info">Score: 86.3, Published: 2023-09-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.04.556158' target='https://doi.org/10.1101/2023.09.04.556158'> 10.1101/2023.09.04.556158</a></p>
        <p class="abstract">Multiple immune pathways in humans conjugate ubiquitin-like proteins to virus and host molecules as a means of antiviral defense. Here we studied an anti-phage defense system in bacteria, comprising a ubiquitin-like protein, ubiquitin-conjugating enzymes E1 and E2, and a deubiquitinase. We show that during phage infection, this system specifically conjugates the ubiquitin-like protein to the phage central tail fiber, a protein at the tip of the tail that is essential for tail assembly as well as for recognition of the target host receptor. Following infection, cells encoding this defense system release a mixture of partially assembled, tailless phage particles, and fully assembled phages in which the central tail fiber is obstructed by the covalently attached ubiquitin-like protein. These phages exhibit severely impaired infectivity, explaining how the defense system protects the bacterial population from the spread of phage infection. Our findings demonstrate that conjugation of ubiquitin-like proteins is an antiviral strategy conserved across the tree of life.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555216">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555216" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555216">
        <p class="paperTitle">Community ecology of phages on a clonal bacterial host</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555216" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555216" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pyenson, N. C.; Leeks, A.; Nweke, O.; Goldford, J.; Turner, P.; Foster, K. R.; Sanchez, A.</p>
        <p class="info">Score: 70.7, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555216' target='https://doi.org/10.1101/2023.09.01.555216'> 10.1101/2023.09.01.555216</a></p>
        <p class="abstract">Bacteriophages are the most abundant and diverse biological entities on Earth, yet the ecological mechanisms that sustain this extraordinary diversity remain unclear. Here, we have discovered a general mechanism that allows phage diversity to outstrip the diversity of their bacterial hosts. We assembled and passaged dozens of diverse phage communities on a single, non-coevolving strain of Escherichia coli until the phage communities reached equilibrium. In all cases, we found multiple phage species coexisted stably, despite competition for a single, clonal host population. Coexistence within these communities was supported through host phenotypic heterogeneity, whereby phages specialized on cells adopting different growth phenotypes. Our experiments reveal a rich community ecology of bacteriophages where multiple phage species coexist and interact at the scale of a single bacterial host.

One-Sentence SummaryDiverse communities of phages emerge stably and consistently on a clonal bacterial host, enabled by phenotypic heterogeneity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.03.556088">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.03.556088" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.03.556088">
        <p class="paperTitle">Protein coopted from a phage restriction system dictates orthogonal cell division plane selection in Staphylococcus aureus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.03.556088" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.03.556088" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ramos-Leon, F.; Anjuwon-Foster, B. R.; Anantharaman, V.; Ferreira, C. N.; Ibrahim, A. M.; Tai, C.-H.; Missiakas, D. M.; Camberg, J.; Aravind, L.; Ramamurthi, K. S.</p>
        <p class="info">Score: 38.1, Published: 2023-09-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.03.556088' target='https://doi.org/10.1101/2023.09.03.556088'> 10.1101/2023.09.03.556088</a></p>
        <p class="abstract">The spherical bacterium Staphylococcus aureus, a leading cause of nosocomial infections, undergoes binary fission by dividing in two alternating orthogonal planes, but the mechanism by which S. aureus correctly selects the next cell division plane is not known. To identify cell division placement factors, we performed a chemical genetic screen that revealed a gene which we termed pcdA. We show that PcdA is a member of the McrB family of AAA&#43; NTPases that has undergone structural changes and a concomitant functional shift from a restriction enzyme subunit to an early cell division protein. PcdA directly interacts with the tubulin-like central divisome component FtsZ and localizes to future cell division sites before membrane invagination initiates. This parallels the action of another McrB family protein, CTTNBP2, which stabilizes microtubules in animals. We show that PcdA also interacts with the structural protein DivIVA and propose that the DivIVA/PcdA complex recruits unpolymerized FtsZ to assemble along the proper cell division plane. Deletion of pcdA conferred abnormal, non-orthogonal division plane selection, increased sensitivity to cell wall-targeting antibiotics, and reduced virulence in a murine infection model. Targeting PcdA could therefore highlight a treatment strategy for combatting antibiotic-resistant strains of S. aureus.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.31.555772">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.31.555772" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.31.555772">
        <p class="paperTitle">Sequential early-life viral infections modulate the microbiota and adaptive immune responses to systemic and mucosal vaccination</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.31.555772" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.31.555772" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, Y.; Molleston, J. M.; Kim, A. H.; Ingle, H.; Aggarwal, S.; Nolan, L.; Hassan, A.; Foster, L.; Diamond, M.; Baldridge, M. T.</p>
        <p class="info">Score: 41.2, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.31.555772' target='https://doi.org/10.1101/2023.08.31.555772'> 10.1101/2023.08.31.555772</a></p>
        <p class="abstract">Increasing evidence points to the microbial exposome as a critical factor in maturing and shaping the host immune system, thereby influencing responses to immune challenges such as infections or vaccines. To investigate the effect of early-life viral exposures on immune development and vaccine responses, we inoculated mice with six distinct viral pathogens in sequence beginning in the neonatal period, and then evaluated their immune signatures before and after intramuscular or intranasal vaccination against SARS-CoV-2. Sequential viral infection drove profound changes in all aspects of the immune system, including increasing circulating leukocytes, altering innate and adaptive immune cell lineages in tissues, and markedly influencing serum cytokine and total antibody levels. Beyond these immune responses changes, these exposures also modulated the composition of the endogenous intestinal microbiota. Although sequentially-infected mice exhibited increased systemic immune activation and T cell responses after intramuscular and intranasal SARS-CoV-2 immunization, we observed decreased vaccine-induced antibody responses in these animals. These results suggest that early-life viral exposures are sufficient to diminish antibody responses to vaccination in mice, and highlight their potential importance of considering prior microbial exposures when investigating vaccine responses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555899">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555899" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555899">
        <p class="paperTitle">SARS-CoV-2 hijacks fragile X mental retardation proteins for efficient infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555899" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555899" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Garvanska, D.; Alvarado, R. E.; Mundt, F.; Nilsson, E.; Duel, J.; Coscia, F.; Lindqvist, R.; Lokugamage, K.; Johnson, B.; Plante, J. A.; Morris, D. R.; Vu, M. N.; Estes, L.; McLeland, A.; Walker, J.; Crocquet-Valdes, P.; Lopez-Mendez, B.; Plante, K.; Walker, D. H.; Weisser, M. B.; Overby, A. K.; Mann, M.; Menachery, V. D.; Nilsson, J.</p>
        <p class="info">Score: 32.4, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555899' target='https://doi.org/10.1101/2023.09.01.555899'> 10.1101/2023.09.01.555899</a></p>
        <p class="abstract">Viruses interact with numerous host factors to facilitate viral replication and to dampen antiviral defense mechanisms. We currently have a limited mechanistic understanding of how SARS-CoV-2 binds host factors and the functional role of these interactions. Here, we uncover a novel interaction between the viral NSP3 protein and the fragile X mental retardation proteins (FMRPs: FMR1 and FXR1-2). SARS-CoV-2 NSP3 mutant viruses preventing FMRP binding have attenuated replication in vitro and have delayed disease onset in vivo. We show that a unique peptide motif in NSP3 binds directly to the two central KH domains of FMRPs and that this interaction is disrupted by the I304N mutation found in a patient with fragile X syndrome. NSP3 binding to FMRPs disrupts their interaction with the stress granule component UBAP2L through direct competition with a peptide motif in UBAP2L to prevent FMRP incorporation into stress granules. Collectively, our results provide novel insight into how SARS-CoV-2 hijacks host cell proteins for efficient infection and provides molecular insight to the possible underlying molecular defects in fragile X syndrome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.17.553735">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.17.553735" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.17.553735">
        <p class="paperTitle">Co-Occurrence Network Analysis Reveals The Alterations Of The Skin Microbiome And Metabolome In Atopic Dermatitis Patients</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.17.553735" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.17.553735" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Portal Gomes, P. W.; Mannochio-Russo, H.; Mao, J.; Zhao, H. N.; Tipton, C. D.; Ancira, J.; Dorrestein, P. C.; Li, M.</p>
        <p class="info">Score: 27.2, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.17.553735' target='https://doi.org/10.1101/2023.08.17.553735'> 10.1101/2023.08.17.553735</a></p>
        <p class="abstract">Skin microbiome can be altered in patients with Atopic Dermatitis (AD). An understanding of the changes from healthy to atopic skin can help develop new targets for better treatments and identify specific microbial or molecular biomarkers. This study investigates the skin microbiome and metabolome of healthy subjects and lesion (ADL) and non-lesion (ADNL) of AD patients by 16S rRNA gene sequencing and mass spectrometry, respectively. Samples from AD patients showed alterations in the diversity and composition of the skin microbiome. Staphylococcus species, especially S. aureus, were significantly increased in the ADL group. Metabolomic profiles were also different between the groups. Dipeptide-derived are more abundant in ADL, which may be related to skin inflammation. Co-occurrence network analysis was applied to integrate the microbiome and metabolomics data and revealed higher co-occurrence of metabolites and bacteria in healthy and ADNL compared to ADL. S. aureus co-occurred with dipeptide-derived in ADL, while phytosphingosine-derived compounds showed co-occurrences with commensal bacteria, e.g. Paracoccus sp., Pseudomonas sp., Prevotella bivia, Lactobacillus iners, Anaerococcus sp., Micrococcus sp., Corynebacterium ureicelerivorans, Corynebacterium massiliense, Streptococcus thermophilus, and Roseomonas mucosa, in healthy and ADNL groups. Therefore, these findings provide valuable insights into how AD affects the human skin metabolome and microbiome.

ImportanceThis study provides valuable insight into changes in the skin microbiome and associated metabolomic profiles. It also identifies new therapeutic targets that may be useful for developing personalized treatments for individuals with atopic dermatitis based on their unique skin microbiome and metabolic profiles.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.31.555663">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.31.555663" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.31.555663">
        <p class="paperTitle">Computational exploration of the global microbiome for antibiotic discovery</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.31.555663" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.31.555663" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Celio Dias, S.-J.; Marcelo D. T., T.; Yiqian, D.; Alvaro, R. d. R.; Thomas S.B., S.; Hui, C.; Anthony, F.; Kuhn, M.; Chengkai, Z.; Amy, H.; Jelena, S.; Anna, V.; Xing-Ming, Z.; Peer, B.; Jaime, H.-C.; Cesar, d. l. F.-N.; Luis Pedro, C.</p>
        <p class="info">Score: 20.9, Published: 2023-08-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.31.555663' target='https://doi.org/10.1101/2023.08.31.555663'> 10.1101/2023.08.31.555663</a></p>
        <p class="abstract">Novel antibiotics are urgently needed to combat the antibiotic-resistance crisis. We present a machine learning-based approach to predict prokaryotic antimicrobial peptides (AMPs) by leveraging a vast dataset of 63,410 metagenomes and 87,920 microbial genomes. This led to the creation of AMPSphere, a comprehensive catalog comprising 863,498 non-redundant peptides, the majority of which were previously unknown. We observed that AMP production varies by habitat, with animal-associated samples displaying the highest proportion of AMPs compared to other habitats. Furthermore, within different human-associated microbiota, strain-level differences were evident. To validate our predictions, we synthesized and experimentally tested 50 AMPs, demonstrating their efficacy against clinically relevant drug-resistant pathogens both in vitro and in vivo. These AMPs exhibited antibacterial activity by targeting the bacterial membrane. Additionally, AMPSphere provides valuable insights into the evolutionary origins of peptides. In conclusion, our approach identified AMP sequences within prokaryotic microbiomes, opening up new avenues for the discovery of antibiotics.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
