<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>microbiology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>microbiology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Lifecycle of a predatory bacterium vampirizing its prey through the cell envelope and S-layer
Authors: Santin, Y. G.; Sogues, A.; Bourigault, Y.; Remaut, H. K.; Laloux, G.
Score: 61.1, Published: 2023-10-26 DOI: 10.1101/2023.10.25.563945
Predatory bacteria feed upon and kill other bacteria in various natural environments. Obligate epibiotic predators like Bdellovibrio exovorus consume their prey whilst remaining attached to the outside of the prey. How these predators achieve epibiotic feeding through the prey cell envelope has not been explored previously.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/microbiology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="microbiology" />
<meta property="og:description" content="Lifecycle of a predatory bacterium vampirizing its prey through the cell envelope and S-layer
Authors: Santin, Y. G.; Sogues, A.; Bourigault, Y.; Remaut, H. K.; Laloux, G.
Score: 61.1, Published: 2023-10-26 DOI: 10.1101/2023.10.25.563945
Predatory bacteria feed upon and kill other bacteria in various natural environments. Obligate epibiotic predators like Bdellovibrio exovorus consume their prey whilst remaining attached to the outside of the prey. How these predators achieve epibiotic feeding through the prey cell envelope has not been explored previously." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/microbiology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-01T10:38:40+00:00" />
<meta property="article:modified_time" content="2023-11-01T10:38:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="microbiology"/>
<meta name="twitter:description" content="Lifecycle of a predatory bacterium vampirizing its prey through the cell envelope and S-layer
Authors: Santin, Y. G.; Sogues, A.; Bourigault, Y.; Remaut, H. K.; Laloux, G.
Score: 61.1, Published: 2023-10-26 DOI: 10.1101/2023.10.25.563945
Predatory bacteria feed upon and kill other bacteria in various natural environments. Obligate epibiotic predators like Bdellovibrio exovorus consume their prey whilst remaining attached to the outside of the prey. How these predators achieve epibiotic feeding through the prey cell envelope has not been explored previously."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "microbiology",
      "item": "https://trxiv.yorks0n.com/posts/microbiology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "microbiology",
  "name": "microbiology",
  "description": "Lifecycle of a predatory bacterium vampirizing its prey through the cell envelope and S-layer\nAuthors: Santin, Y. G.; Sogues, A.; Bourigault, Y.; Remaut, H. K.; Laloux, G.\nScore: 61.1, Published: 2023-10-26 DOI: 10.1101/2023.10.25.563945\nPredatory bacteria feed upon and kill other bacteria in various natural environments. Obligate epibiotic predators like Bdellovibrio exovorus consume their prey whilst remaining attached to the outside of the prey. How these predators achieve epibiotic feeding through the prey cell envelope has not been explored previously.",
  "keywords": [
    
  ],
  "articleBody": " Lifecycle of a predatory bacterium vampirizing its prey through the cell envelope and S-layer\nAuthors: Santin, Y. G.; Sogues, A.; Bourigault, Y.; Remaut, H. K.; Laloux, G.\nScore: 61.1, Published: 2023-10-26 DOI: 10.1101/2023.10.25.563945\nPredatory bacteria feed upon and kill other bacteria in various natural environments. Obligate epibiotic predators like Bdellovibrio exovorus consume their prey whilst remaining attached to the outside of the prey. How these predators achieve epibiotic feeding through the prey cell envelope has not been explored previously. Whereas the S-layer is the only proposed defensive structure against predatory bacteria, it remains unclear how this thin outer layer of the envelope might prevent epibiotic attacks. Similarly, the lifecycle of B. exovorus during the predator-prey interaction is poorly understood, with current models suggesting a binary division. Here we imaged the entire predatory lifecycle of B. exovorus and the fate of its Caulobacter crescentus prey by time-lapse microscopy and cryo-electron microscopy to monitor predator attack, growth and division and assess the impact of the S-layer on epibiotic predation. Our data reveal that B. exovorus uses non-binary division in a novel proliferation pattern that mainly generates three progenies. Moreover, we found that B. exovorus predates regardless of the presence of an S-layer, calling for revisiting its protective role against predators. Finally, our results indicate that epibiotic predation relies on the establishment of a secured junction between the prey and predator outer membranes, which must be resolved unilaterally to maintain cellular integrity of the predator departing from the prey surface.\nWidespread fungal-bacterial competition for magnesium enhances antibiotic resistance\nAuthors: Hsieh, Y.-Y. P.; Sun, W. W.; Young, J. M.; Cheung, R.; Hogan, D. A.; Dandekar, A. A.; Malik, H. S.\nScore: 22.3, Published: 2023-10-26 DOI: 10.1101/2023.10.25.563990\nFungi and bacteria coexist in many polymicrobial communities, yet the molecular basis of their interactions remains poorly understood. Using unbiased genomic approaches, we discover that the fungus Candida albicans sequesters essential Mg2+ ions from the bacterium Pseudomonas aeruginosa. In turn, the bacterium competes using a Mg2+ transporter, MgtA. We show that Mg2+ sequestration by fungi is a general mechanism of antagonism against gram-negative bacteria. But the resultant Mg2+ limitation enhances bacterial resistance to polymyxin antibiotics like colistin, which target gram-negative bacterial membranes. Experimental evolution reveals that bacteria in co-culture with fungi become phenotypically, but not genetically, resistant to colistin; antifungal treatment renders resistant bacteria from co-cultures to become colistin-sensitive. Fungal-bacterial nutritional competition may thus profoundly impact treatments of polymicrobial infections with antibiotics of last resort. One Sentence SummaryMagnesium sequestration by fungi lowers bacterial fitness but enhances antibiotic resistance.\nInteractions between pili affect the outcome of bacterial competition driven by the type VI secretion system\nAuthors: Otto, S. B.; Servajean, R.; Lemopoulos, A.; Bitbol, A.-F.; Blokesch, M.\nScore: 20.9, Published: 2023-10-26 DOI: 10.1101/2023.10.25.564063\nThe bacterial type VI secretion system (T6SS) is a widespread, kin-discriminatory weapon capable of shaping microbial communities. Due to the systems dependency on contact, cellular interactions can lead to either competition or kin protection. Cell-to-cell contact is often accomplished via surface-exposed type IV pili (T4P). In Vibrio cholerae, these T4P facilitate specific interactions when the bacteria colonize natural chitinous surfaces. However, it has remained unclear whether and, if so, how these interactions affect the bacteriums T6SS-mediated killing. In this study, we demonstrate that pilus-mediated interactions can be harnessed to reduce the population of V. cholerae under liquid growth conditions in a T6SS-dependent manner. We also show that the naturally occurring diversity of pili determines the likelihood of cell-to-cell contact and, consequently, the extent of T6SS-mediated competition. To determine the factors that enable or hinder the T6SSs targeted reduction of competitors carrying pili, we developed a physics-grounded computational model for autoaggregation. Collectively, our research demonstrates that T4P involved in cell-to-cell contact can impose a selective burden when V. cholerae encounters non-kin cells that possess an active T6SS. Additionally, our study underscores the significance of T4P diversity in protecting closely related individuals from T6SS attacks through autoaggregation and spatial segregation.\nThe mycobacterial glycoside hydrolase LamH enables capsular arabinomannan release and stimulates growth\nAuthors: Franklin, A.; Layton, A. J.; Mize, T.; Salgueiro, V. C.; Sullivan, R.; Benedict, S. T.; Gurcha, S. S.; Anso, I.; Besra, G. S.; Banzhaf, M.; Lovering, A. L.; Williams, S. J.; Guerin, M. E.; Scott, N. E.; Prados-Rosales, R.; Lowe, E. C.; Moynihan, P. J.\nScore: 20.7, Published: 2023-10-26 DOI: 10.1101/2023.10.26.563968\nMycobacterial glycolipids are important cell envelope structures that drive host-pathogen interactions. Arguably, the most important amongst these are lipoarabinomannan (LAM) and its precursor, lipomannan (LM), which are both trafficked out of the bacterium to the host via unknown mechanisms. An important class of exported LM/LAM is the capsular derivative of these molecules which is devoid of its lipid anchor. Here, we describe the identification of a glycoside hydrolase family 76 enzyme that we term LamH which specifically cleaves -1,6-mannoside linkages within LM and LAM, driving its export to the capsule releasing its phosphatidyl-myo-inositol mannoside lipid anchor. Unexpectedly, we found that the catalytic activity of this enzyme is important for efficient exit from stationary phase cultures where arabinomannan acts as a signal for growth phase transition. Finally, we demonstrate that LamH is important for Mycobacterium tuberculosis survival in macrophages. These data provide a new framework for understanding the biological role of LAM in mycobacteria.\nA non-canonical Arp2/3 complex is essential for Plasmodium DNA segregation and transmission of malaria.\nAuthors: Hentzschel, F.; Jewanski, D.; Sokolowski, Y.; Agarwal, P.; Kraeft, A.; Hildenbrand, K.; Dorner, L. P.; Singer, M.; Frischknecht, F.; Marti, M.\nScore: 20.0, Published: 2023-10-26 DOI: 10.1101/2023.10.25.563799\nThe malaria-causing parasite Plasmodium has a complex life cycle involving both vertebrate and mosquito hosts. Sexual stages or gametocytes are the only stage competent for transmission to mosquitoes. Formation of flagellated male gametes from gametocytes requires rapid rounds of genome replication. Here we discovered a non-canonical Plasmodium actin-related protein 2/3 (Arp2/3) complex essential for DNA segregation during male gametogenesis. Plasmodium Arp2/3 dynamically localizes within the nucleus to the endomitotic spindles and interacts with a kinetochore protein. Deletion of key Arp2/3 subunits or interfering with actin polymerisation leads to the formation of sub-haploid male gametes and a complete block in transmission through delayed developmental arrest at the oocyst stage. Our work identified an evolutionary divergent protein complex in malaria parasites that offers potential targets for transmission-blocking interventions.\nTranscription-driven DNA supercoiling activates bacterial chromatin at a distance\nAuthors: Figueroa-Bossi, N.; Fernandez-Fernandez, R.; Kerboriou, P.; Bouloc, P.; Casadesus, J.; Sanchez-Romero, M.-A.; Bossi, L.\nScore: 19.6, Published: 2023-10-26 DOI: 10.1101/2023.10.26.564199\nIn all living cells, genomic DNA is compacted through interactions with dedicated proteins and/or the formation of plectonemic coils. In bacteria, DNA compaction is achieved dynamically, coordinated with dense and constantly changing transcriptional activity. H-NS, a major bacterial nucleoid structuring protein, is of special interest due to its interplay with RNA polymerase. H-NS:DNA nucleoprotein filaments inhibit transcription initiation by RNA polymerase. However, the discovery that genes silenced by H-NS can be activated by transcription originating from neighboring regions has suggested that elongating RNA polymerases can disassemble H-NS:DNA filaments. In this study, we present evidence that transcription-induced counter-silencing does not require transcription to reach the silenced gene; rather, it exerts its effect at a distance. Counter-silencing is suppressed by introducing a DNA gyrase binding site within the intervening segment, suggesting that the long-range effect results from transcription-driven positive DNA supercoils diffusing toward the silenced gene. We propose a model wherein H-NS:DNA complexes form in vivo on negaXvely supercoiled DNA, with H-NS bridging the two arms of the plectoneme. Rotational diffusion of positive supercoils generated by neighbouring transcription will cause the H-NS-bound negatively-supercoiled plectoneme to \"unroll\" disrupting the H-NS bridges and releasing H-NS.\nNo evidence for highly pathogenic avian influenza virus H5N1 (clade 2.3.4.4b) in the Antarctic region during the austral summer 2022/23\nAuthors: Lisovski, S.; Gunther, A.; Dewar, M.; Anley, D.; Arce, R.; Ballard, G.; Belliure, J.; Boulinier, T.; Bennison, A.; Cary, C.; Catry, P.; Clessin, A.; Connan, M.; Cox, A.; Christina, J.; Elrod, M.; Ferreiro, I.; Fowler, Z.; Gamble, A.; Hurtado, J.; Jongsomjit, D.; Lejeune, M.; Lescroel, A.; Li, A.; Morandini, V.; Moratorio, G.; Militao, T.; Moreno, P.; Perbolianachis, P.; Pennycook, J.; Schmidt, A. E.; Smyth, L.; Soutullo, A.; Tornos, J.; Varsani, A.; Herzschuh, U.; Beer, M.; Wille, M.\nScore: 17.1, Published: 2023-10-24 DOI: 10.1101/2023.10.24.563692\nThe current highly pathogenic avian influenza H5N1 panzootic has profound impacts on wild birds. Herein, we compiled H5N1 surveillance from Antarctica and Sub-Antarctic Islands to ascertain whether HPAI was present in this region. Observations and HPAI testing indicated no incursion of the virus during the austral summer 2022/23.\nViral genomic features predict orthopoxvirus reservoir hosts\nAuthors: Tseng, K. K.; Koehler, H.; Becker, D. J.; Gibb, R.; Carlson, C. J.; Fernandez, M. d. P.; Seifert, S. N.\nScore: 15.5, Published: 2023-10-27 DOI: 10.1101/2023.10.26.564211\nOrthopoxviruses (OPVs), including the causative agents of smallpox and mpox have led to devastating outbreaks in human populations worldwide. However, the discontinuation of smallpox vaccination, which also provides cross-protection against related OPVs, has diminished global immunity to OPVs more broadly. We apply machine learning models incorporating both host ecological and viral genomic features to predict likely reservoirs of OPVs. We demonstrate that incorporating viral genomic features in addition to host ecological traits enhanced the accuracy of potential OPV host predictions, highlighting the importance of host-virus molecular interactions in predicting potential host species. We identify hotspots for geographic regions rich with potential OPV hosts in parts of southeast Asia, equatorial Africa, and the Amazon, revealing high overlap between regions predicted to have a high number of potential OPV host species and those with the lowest smallpox vaccination coverage, indicating a heightened risk for the emergence or establishment of zoonotic OPVs. Our findings can be used to target wildlife surveillance, particularly related to concerns about mpox establishment beyond its historical range.\nCyanobacteria newly isolated from marine volcanic seeps display rapid sinking and robust, high density growth\nAuthors: Schubert, M. G.; Tang, T.-C.; Goodchild-Michelman, I. M.; Ryon, K. A.; Henriksen, J. R.; Chavkin, T.; Wu, Y.; Miettinen, T. P.; Van Wychen, S.; Dahlin, L.; Spatafora, D.; Turco, G.; Guarnieri, M.; Manalis, S. R.; Kowitz, J.; Dhir, R.; Quatrini, P.; Mason, C. E.; Church, G. M.; Milazzo, M.; Tierney, B. T.\nScore: 14.8, Published: 2023-10-30 DOI: 10.1101/2023.10.30.564770\nCyanobacteria are photosynthetic organisms that play important roles in carbon cycling as well as promising bioproduction chassis. Here, we isolate two novel cyanobacteria, UTEX 3221 and UTEX 3222, from a unique marine environment with naturally elevated CO2. We describe complete genome sequences for both isolates and, focusing on UTEX 3222 due to its planktonic growth in liquid, characterize biotechnologically-relevant growth and biomass characteristics. UTEX 3222 outpaces other fast-growing model strains on solid medium. It can double every 2.35 hours in a liquid medium and grows to high density (\u003e31g/L biomass dry weight) in batch culture, nearly double that of Synechococcus sp. PCC 11901, whose high-density growth was recently reported. In addition, UTEX 3222 sinks readily, settling more quickly than other fast-growing strains, suggesting improved de-watering of UTEX 3222 biomass. This settling behavior can be explained in part by larger cell volume. These traits may make UTEX 3222 a compelling choice for photosynthetic bioproduction from CO2. Overall, we find that bio-prospecting in environments with naturally elevated CO2 may uncover novel CO2-metabolizing organisms with unique characteristics.\nVirological characteristics of the SARS-CoV-2 Omicron EG.5.1 variant\nAuthors: Tsujino, S.; Deguchi, S.; Nomai, T.; Padilla-Blanco, M.; Plianchaisuk, A.; Wang, L.; Begum, M. M.; Uriu, K.; Mizuma, K.; Nao, N.; Kojima, I.; Tsubo, T.; Li, J.; Matsumura, Y.; Nagao, M.; Oda, Y.; Tsuda, M.; Anraku, Y.; Kita, S.; Yajima, H.; Tabata, K.; Guo, Z.; Hinay, A. A.; Yoshimatsu, K.; Yamamoto, Y.; Nagamoto, Y.; Asakura, H.; Nagashima, M.; Sadamasu, K.; Yoshimura, K.; Nasser, H.; Jonathan, M.; Putri, O.; Kim, Y.; Chen, L.; Suzuki, R.; Tamura, T.; Maenaka, K.; The Genotype to Phenotype Japan (G2P-Japan) Consortium, ; Irie, T.; Matsuno, K.; Tanaka, S.; Ito, J.; Ikeda, T.; Takayama, K.; Za\nScore: 151.6, Published: 2023-10-21 DOI: 10.1101/2023.10.19.563209\nIn middle-late 2023, a sublineage of SARS-CoV-2 Omicron XBB, EG.5.1 (a progeny of XBB.1.9.2), is spreading rapidly around the world. Here, we performed multiscale investigations to reveal virological features of newly emerging EG.5.1 variant. Our phylogenetic-epidemic dynamics modeling suggested that two hallmark substitutions of EG.5.1, S:F456L and ORF9b:I5T, are critical to the increased viral fitness. Experimental investigations addressing the growth kinetics, sensitivity to clinically available antivirals, fusogenicity and pathogenicity of EG.5.1 suggested that the virological features of EG.5.1 is comparable to that of XBB.1.5. However, the cryo-electron microscopy reveals the structural difference between the spike proteins of EG.5.1 and XBB.1.5. We further assessed the impact of ORF9b:I5T on viral features, but it was almost negligible at least in our experimental setup. Our multiscale investigations provide the knowledge for understanding of the evolution trait of newly emerging pathogenic viruses in the human population.\n",
  "wordCount" : "2117",
  "inLanguage": "en",
  "datePublished": "2023-11-01T10:38:40Z",
  "dateModified": "2023-11-01T10:38:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/microbiology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      microbiology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 1, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.563945">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.563945" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.563945">
        <p class="paperTitle">Lifecycle of a predatory bacterium vampirizing its prey through the cell envelope and S-layer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.563945" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.563945" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Santin, Y. G.; Sogues, A.; Bourigault, Y.; Remaut, H. K.; Laloux, G.</p>
        <p class="info">Score: 61.1, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.563945' target='https://doi.org/10.1101/2023.10.25.563945'> 10.1101/2023.10.25.563945</a></p>
        <p class="abstract">Predatory bacteria feed upon and kill other bacteria in various natural environments. Obligate epibiotic predators like Bdellovibrio exovorus consume their prey whilst remaining attached to the outside of the prey. How these predators achieve epibiotic feeding through the prey cell envelope has not been explored previously. Whereas the S-layer is the only proposed defensive structure against predatory bacteria, it remains unclear how this thin outer layer of the envelope might prevent epibiotic attacks. Similarly, the lifecycle of B. exovorus during the predator-prey interaction is poorly understood, with current models suggesting a binary division. Here we imaged the entire predatory lifecycle of B. exovorus and the fate of its Caulobacter crescentus prey by time-lapse microscopy and cryo-electron microscopy to monitor predator attack, growth and division and assess the impact of the S-layer on epibiotic predation. Our data reveal that B. exovorus uses non-binary division in a novel proliferation pattern that mainly generates three progenies. Moreover, we found that B. exovorus predates regardless of the presence of an S-layer, calling for revisiting its protective role against predators. Finally, our results indicate that epibiotic predation relies on the establishment of a secured junction between the prey and predator outer membranes, which must be resolved unilaterally to maintain cellular integrity of the predator departing from the prey surface.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.563990">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.563990" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.563990">
        <p class="paperTitle">Widespread fungal-bacterial competition for magnesium enhances antibiotic resistance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.563990" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.563990" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hsieh, Y.-Y. P.; Sun, W. W.; Young, J. M.; Cheung, R.; Hogan, D. A.; Dandekar, A. A.; Malik, H. S.</p>
        <p class="info">Score: 22.3, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.563990' target='https://doi.org/10.1101/2023.10.25.563990'> 10.1101/2023.10.25.563990</a></p>
        <p class="abstract">Fungi and bacteria coexist in many polymicrobial communities, yet the molecular basis of their interactions remains poorly understood. Using unbiased genomic approaches, we discover that the fungus Candida albicans sequesters essential Mg2&#43; ions from the bacterium Pseudomonas aeruginosa. In turn, the bacterium competes using a Mg2&#43; transporter, MgtA. We show that Mg2&#43; sequestration by fungi is a general mechanism of antagonism against gram-negative bacteria. But the resultant Mg2&#43; limitation enhances bacterial resistance to polymyxin antibiotics like colistin, which target gram-negative bacterial membranes. Experimental evolution reveals that bacteria in co-culture with fungi become phenotypically, but not genetically, resistant to colistin; antifungal treatment renders resistant bacteria from co-cultures to become colistin-sensitive. Fungal-bacterial nutritional competition may thus profoundly impact treatments of polymicrobial infections with antibiotics of last resort.

One Sentence SummaryMagnesium sequestration by fungi lowers bacterial fitness but enhances antibiotic resistance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.564063">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.564063" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.564063">
        <p class="paperTitle">Interactions between pili affect the outcome of bacterial competition driven by the type VI secretion system</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.564063" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.564063" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Otto, S. B.; Servajean, R.; Lemopoulos, A.; Bitbol, A.-F.; Blokesch, M.</p>
        <p class="info">Score: 20.9, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.564063' target='https://doi.org/10.1101/2023.10.25.564063'> 10.1101/2023.10.25.564063</a></p>
        <p class="abstract">The bacterial type VI secretion system (T6SS) is a widespread, kin-discriminatory weapon capable of shaping microbial communities. Due to the systems dependency on contact, cellular interactions can lead to either competition or kin protection. Cell-to-cell contact is often accomplished via surface-exposed type IV pili (T4P). In Vibrio cholerae, these T4P facilitate specific interactions when the bacteria colonize natural chitinous surfaces. However, it has remained unclear whether and, if so, how these interactions affect the bacteriums T6SS-mediated killing. In this study, we demonstrate that pilus-mediated interactions can be harnessed to reduce the population of V. cholerae under liquid growth conditions in a T6SS-dependent manner. We also show that the naturally occurring diversity of pili determines the likelihood of cell-to-cell contact and, consequently, the extent of T6SS-mediated competition. To determine the factors that enable or hinder the T6SSs targeted reduction of competitors carrying pili, we developed a physics-grounded computational model for autoaggregation. Collectively, our research demonstrates that T4P involved in cell-to-cell contact can impose a selective burden when V. cholerae encounters non-kin cells that possess an active T6SS. Additionally, our study underscores the significance of T4P diversity in protecting closely related individuals from T6SS attacks through autoaggregation and spatial segregation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.563968">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.563968" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.563968">
        <p class="paperTitle">The mycobacterial glycoside hydrolase LamH enables capsular arabinomannan release and stimulates growth</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.563968" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.563968" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Franklin, A.; Layton, A. J.; Mize, T.; Salgueiro, V. C.; Sullivan, R.; Benedict, S. T.; Gurcha, S. S.; Anso, I.; Besra, G. S.; Banzhaf, M.; Lovering, A. L.; Williams, S. J.; Guerin, M. E.; Scott, N. E.; Prados-Rosales, R.; Lowe, E. C.; Moynihan, P. J.</p>
        <p class="info">Score: 20.7, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.563968' target='https://doi.org/10.1101/2023.10.26.563968'> 10.1101/2023.10.26.563968</a></p>
        <p class="abstract">Mycobacterial glycolipids are important cell envelope structures that drive host-pathogen interactions. Arguably, the most important amongst these are lipoarabinomannan (LAM) and its precursor, lipomannan (LM), which are both trafficked out of the bacterium to the host via unknown mechanisms. An important class of exported LM/LAM is the capsular derivative of these molecules which is devoid of its lipid anchor. Here, we describe the identification of a glycoside hydrolase family 76 enzyme that we term LamH which specifically cleaves -1,6-mannoside linkages within LM and LAM, driving its export to the capsule releasing its phosphatidyl-myo-inositol mannoside lipid anchor. Unexpectedly, we found that the catalytic activity of this enzyme is important for efficient exit from stationary phase cultures where arabinomannan acts as a signal for growth phase transition. Finally, we demonstrate that LamH is important for Mycobacterium tuberculosis survival in macrophages. These data provide a new framework for understanding the biological role of LAM in mycobacteria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.563799">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.563799" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.563799">
        <p class="paperTitle">A non-canonical Arp2/3 complex is essential for Plasmodium DNA segregation and transmission of malaria.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.563799" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.563799" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hentzschel, F.; Jewanski, D.; Sokolowski, Y.; Agarwal, P.; Kraeft, A.; Hildenbrand, K.; Dorner, L. P.; Singer, M.; Frischknecht, F.; Marti, M.</p>
        <p class="info">Score: 20.0, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.563799' target='https://doi.org/10.1101/2023.10.25.563799'> 10.1101/2023.10.25.563799</a></p>
        <p class="abstract">The malaria-causing parasite Plasmodium has a complex life cycle involving both vertebrate and mosquito hosts. Sexual stages or gametocytes are the only stage competent for transmission to mosquitoes. Formation of flagellated male gametes from gametocytes requires rapid rounds of genome replication. Here we discovered a non-canonical Plasmodium actin-related protein 2/3 (Arp2/3) complex essential for DNA segregation during male gametogenesis. Plasmodium Arp2/3 dynamically localizes within the nucleus to the endomitotic spindles and interacts with a kinetochore protein. Deletion of key Arp2/3 subunits or interfering with actin polymerisation leads to the formation of sub-haploid male gametes and a complete block in transmission through delayed developmental arrest at the oocyst stage. Our work identified an evolutionary divergent protein complex in malaria parasites that offers potential targets for transmission-blocking interventions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564199">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564199" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564199">
        <p class="paperTitle">Transcription-driven DNA supercoiling activates bacterial chromatin at a distance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564199" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564199" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Figueroa-Bossi, N.; Fernandez-Fernandez, R.; Kerboriou, P.; Bouloc, P.; Casadesus, J.; Sanchez-Romero, M.-A.; Bossi, L.</p>
        <p class="info">Score: 19.6, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564199' target='https://doi.org/10.1101/2023.10.26.564199'> 10.1101/2023.10.26.564199</a></p>
        <p class="abstract">In all living cells, genomic DNA is compacted through interactions with dedicated proteins and/or the formation of plectonemic coils. In bacteria, DNA compaction is achieved dynamically, coordinated with dense and constantly changing transcriptional activity. H-NS, a major bacterial nucleoid structuring protein, is of special interest due to its interplay with RNA polymerase. H-NS:DNA nucleoprotein filaments inhibit transcription initiation by RNA polymerase. However, the discovery that genes silenced by H-NS can be activated by transcription originating from neighboring regions has suggested that elongating RNA polymerases can disassemble H-NS:DNA filaments. In this study, we present evidence that transcription-induced counter-silencing does not require transcription to reach the silenced gene; rather, it exerts its effect at a distance. Counter-silencing is suppressed by introducing a DNA gyrase binding site within the intervening segment, suggesting that the long-range effect results from transcription-driven positive DNA supercoils diffusing toward the silenced gene. We propose a model wherein H-NS:DNA complexes form in vivo on negaXvely supercoiled DNA, with H-NS bridging the two arms of the plectoneme. Rotational diffusion of positive supercoils generated by neighbouring transcription will cause the H-NS-bound negatively-supercoiled plectoneme to &#34;unroll&#34; disrupting the H-NS bridges and releasing H-NS.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563692">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563692" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563692">
        <p class="paperTitle">No evidence for highly pathogenic avian influenza virus H5N1 (clade 2.3.4.4b) in the Antarctic region during the austral summer 2022/23</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563692" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563692" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lisovski, S.; Gunther, A.; Dewar, M.; Anley, D.; Arce, R.; Ballard, G.; Belliure, J.; Boulinier, T.; Bennison, A.; Cary, C.; Catry, P.; Clessin, A.; Connan, M.; Cox, A.; Christina, J.; Elrod, M.; Ferreiro, I.; Fowler, Z.; Gamble, A.; Hurtado, J.; Jongsomjit, D.; Lejeune, M.; Lescroel, A.; Li, A.; Morandini, V.; Moratorio, G.; Militao, T.; Moreno, P.; Perbolianachis, P.; Pennycook, J.; Schmidt, A. E.; Smyth, L.; Soutullo, A.; Tornos, J.; Varsani, A.; Herzschuh, U.; Beer, M.; Wille, M.</p>
        <p class="info">Score: 17.1, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563692' target='https://doi.org/10.1101/2023.10.24.563692'> 10.1101/2023.10.24.563692</a></p>
        <p class="abstract">The current highly pathogenic avian influenza H5N1 panzootic has profound impacts on wild birds. Herein, we compiled H5N1 surveillance from Antarctica and Sub-Antarctic Islands to ascertain whether HPAI was present in this region. Observations and HPAI testing indicated no incursion of the virus during the austral summer 2022/23.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564211">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564211" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564211">
        <p class="paperTitle">Viral genomic features predict orthopoxvirus reservoir hosts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564211" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564211" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tseng, K. K.; Koehler, H.; Becker, D. J.; Gibb, R.; Carlson, C. J.; Fernandez, M. d. P.; Seifert, S. N.</p>
        <p class="info">Score: 15.5, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564211' target='https://doi.org/10.1101/2023.10.26.564211'> 10.1101/2023.10.26.564211</a></p>
        <p class="abstract">Orthopoxviruses (OPVs), including the causative agents of smallpox and mpox have led to devastating outbreaks in human populations worldwide. However, the discontinuation of smallpox vaccination, which also provides cross-protection against related OPVs, has diminished global immunity to OPVs more broadly. We apply machine learning models incorporating both host ecological and viral genomic features to predict likely reservoirs of OPVs. We demonstrate that incorporating viral genomic features in addition to host ecological traits enhanced the accuracy of potential OPV host predictions, highlighting the importance of host-virus molecular interactions in predicting potential host species. We identify hotspots for geographic regions rich with potential OPV hosts in parts of southeast Asia, equatorial Africa, and the Amazon, revealing high overlap between regions predicted to have a high number of potential OPV host species and those with the lowest smallpox vaccination coverage, indicating a heightened risk for the emergence or establishment of zoonotic OPVs. Our findings can be used to target wildlife surveillance, particularly related to concerns about mpox establishment beyond its historical range.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564770">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564770" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564770">
        <p class="paperTitle">Cyanobacteria newly isolated from marine volcanic seeps display rapid sinking and robust, high density growth</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564770" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564770" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schubert, M. G.; Tang, T.-C.; Goodchild-Michelman, I. M.; Ryon, K. A.; Henriksen, J. R.; Chavkin, T.; Wu, Y.; Miettinen, T. P.; Van Wychen, S.; Dahlin, L.; Spatafora, D.; Turco, G.; Guarnieri, M.; Manalis, S. R.; Kowitz, J.; Dhir, R.; Quatrini, P.; Mason, C. E.; Church, G. M.; Milazzo, M.; Tierney, B. T.</p>
        <p class="info">Score: 14.8, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564770' target='https://doi.org/10.1101/2023.10.30.564770'> 10.1101/2023.10.30.564770</a></p>
        <p class="abstract">Cyanobacteria are photosynthetic organisms that play important roles in carbon cycling as well as promising bioproduction chassis. Here, we isolate two novel cyanobacteria, UTEX 3221 and UTEX 3222, from a unique marine environment with naturally elevated CO2. We describe complete genome sequences for both isolates and, focusing on UTEX 3222 due to its planktonic growth in liquid, characterize biotechnologically-relevant growth and biomass characteristics. UTEX 3222 outpaces other fast-growing model strains on solid medium. It can double every 2.35 hours in a liquid medium and grows to high density (&gt;31g/L biomass dry weight) in batch culture, nearly double that of Synechococcus sp. PCC 11901, whose high-density growth was recently reported. In addition, UTEX 3222 sinks readily, settling more quickly than other fast-growing strains, suggesting improved de-watering of UTEX 3222 biomass. This settling behavior can be explained in part by larger cell volume. These traits may make UTEX 3222 a compelling choice for photosynthetic bioproduction from CO2. Overall, we find that bio-prospecting in environments with naturally elevated CO2 may uncover novel CO2-metabolizing organisms with unique characteristics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.19.563209">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.19.563209" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.19.563209">
        <p class="paperTitle">Virological characteristics of the SARS-CoV-2 Omicron EG.5.1 variant</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.19.563209" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.19.563209" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tsujino, S.; Deguchi, S.; Nomai, T.; Padilla-Blanco, M.; Plianchaisuk, A.; Wang, L.; Begum, M. M.; Uriu, K.; Mizuma, K.; Nao, N.; Kojima, I.; Tsubo, T.; Li, J.; Matsumura, Y.; Nagao, M.; Oda, Y.; Tsuda, M.; Anraku, Y.; Kita, S.; Yajima, H.; Tabata, K.; Guo, Z.; Hinay, A. A.; Yoshimatsu, K.; Yamamoto, Y.; Nagamoto, Y.; Asakura, H.; Nagashima, M.; Sadamasu, K.; Yoshimura, K.; Nasser, H.; Jonathan, M.; Putri, O.; Kim, Y.; Chen, L.; Suzuki, R.; Tamura, T.; Maenaka, K.; The Genotype to Phenotype Japan (G2P-Japan) Consortium,  ; Irie, T.; Matsuno, K.; Tanaka, S.; Ito, J.; Ikeda, T.; Takayama, K.; Za</p>
        <p class="info">Score: 151.6, Published: 2023-10-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.19.563209' target='https://doi.org/10.1101/2023.10.19.563209'> 10.1101/2023.10.19.563209</a></p>
        <p class="abstract">In middle-late 2023, a sublineage of SARS-CoV-2 Omicron XBB, EG.5.1 (a progeny of XBB.1.9.2), is spreading rapidly around the world. Here, we performed multiscale investigations to reveal virological features of newly emerging EG.5.1 variant. Our phylogenetic-epidemic dynamics modeling suggested that two hallmark substitutions of EG.5.1, S:F456L and ORF9b:I5T, are critical to the increased viral fitness. Experimental investigations addressing the growth kinetics, sensitivity to clinically available antivirals, fusogenicity and pathogenicity of EG.5.1 suggested that the virological features of EG.5.1 is comparable to that of XBB.1.5. However, the cryo-electron microscopy reveals the structural difference between the spike proteins of EG.5.1 and XBB.1.5. We further assessed the impact of ORF9b:I5T on viral features, but it was almost negligible at least in our experimental setup. Our multiscale investigations provide the knowledge for understanding of the evolution trait of newly emerging pathogenic viruses in the human population.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
