<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>neurology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>neurology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Lesions causing Alice in Wonderland Syndrome map to a common brain network linking body and size perception
Authors: Friedrich, M. U.; Baughan, E. C.; Kletenik, I.; Younger, E.; Zhao, C. W.; Howard, C.; Ferguson, M. A.; Chen, A.; Zeller, D.; Piervincenzi, C.; Tommasin, S.; Pantano, P.; Blanke, O.; Prasad, S.; Nielsen, J.; Fox, M. D.
Score: 32.0, Published: 2024-01-18 DOI: 10.1101/2024.01.17.24301332
BackgroundIn Lewis Carrolls 1865 novel &#34;Alices Adventures in Wonderland&#34;, the protagonist experiences distortions in the size of her body and those of others.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/neurology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="neurology" />
<meta property="og:description" content="Lesions causing Alice in Wonderland Syndrome map to a common brain network linking body and size perception
Authors: Friedrich, M. U.; Baughan, E. C.; Kletenik, I.; Younger, E.; Zhao, C. W.; Howard, C.; Ferguson, M. A.; Chen, A.; Zeller, D.; Piervincenzi, C.; Tommasin, S.; Pantano, P.; Blanke, O.; Prasad, S.; Nielsen, J.; Fox, M. D.
Score: 32.0, Published: 2024-01-18 DOI: 10.1101/2024.01.17.24301332
BackgroundIn Lewis Carrolls 1865 novel &#34;Alices Adventures in Wonderland&#34;, the protagonist experiences distortions in the size of her body and those of others." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/neurology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-11T10:38:55+00:00" />
<meta property="article:modified_time" content="2024-02-11T10:38:55+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="neurology"/>
<meta name="twitter:description" content="Lesions causing Alice in Wonderland Syndrome map to a common brain network linking body and size perception
Authors: Friedrich, M. U.; Baughan, E. C.; Kletenik, I.; Younger, E.; Zhao, C. W.; Howard, C.; Ferguson, M. A.; Chen, A.; Zeller, D.; Piervincenzi, C.; Tommasin, S.; Pantano, P.; Blanke, O.; Prasad, S.; Nielsen, J.; Fox, M. D.
Score: 32.0, Published: 2024-01-18 DOI: 10.1101/2024.01.17.24301332
BackgroundIn Lewis Carrolls 1865 novel &#34;Alices Adventures in Wonderland&#34;, the protagonist experiences distortions in the size of her body and those of others."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "neurology",
      "item": "https://trxiv.yorks0n.com/posts/neurology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "neurology",
  "name": "neurology",
  "description": "Lesions causing Alice in Wonderland Syndrome map to a common brain network linking body and size perception\nAuthors: Friedrich, M. U.; Baughan, E. C.; Kletenik, I.; Younger, E.; Zhao, C. W.; Howard, C.; Ferguson, M. A.; Chen, A.; Zeller, D.; Piervincenzi, C.; Tommasin, S.; Pantano, P.; Blanke, O.; Prasad, S.; Nielsen, J.; Fox, M. D.\nScore: 32.0, Published: 2024-01-18 DOI: 10.1101/2024.01.17.24301332\nBackgroundIn Lewis Carrolls 1865 novel \u0026#34;Alices Adventures in Wonderland\u0026#34;, the protagonist experiences distortions in the size of her body and those of others.",
  "keywords": [
    
  ],
  "articleBody": " Lesions causing Alice in Wonderland Syndrome map to a common brain network linking body and size perception\nAuthors: Friedrich, M. U.; Baughan, E. C.; Kletenik, I.; Younger, E.; Zhao, C. W.; Howard, C.; Ferguson, M. A.; Chen, A.; Zeller, D.; Piervincenzi, C.; Tommasin, S.; Pantano, P.; Blanke, O.; Prasad, S.; Nielsen, J.; Fox, M. D.\nScore: 32.0, Published: 2024-01-18 DOI: 10.1101/2024.01.17.24301332\nBackgroundIn Lewis Carrolls 1865 novel \"Alices Adventures in Wonderland\", the protagonist experiences distortions in the size of her body and those of others. This fiction becomes reality in neurological patients with Alice in Wonderland Syndrome (AIWS). Brain lesions causing AIWS may offer unique insights into the syndromes elusive neuroanatomy. MethodsA systematic literature search identified 37 cases of lesion-induced AIWS. Lesion locations were mapped onto a brain atlas and functional connectivity between each lesion location and other brain regions was estimated using resting-state fMRI data from 1000 healthy subjects. Connections common to AIWS lesions were identified and compared to connections from 1073 lesions associated with 25 other neuropsychiatric disorders. Alignment between this lesion-derived AIWS network and neuroimaging findings from patients with AIWS due to other etiologies was assessed. ResultsAlthough AIWS lesions occurred in many different brain locations, these lesions fell within a specific, functionally connected brain network. This network was defined by connectivity to the right extrastriate body area, a brain region selectively activated by viewing body parts, and the inferior parietal cortex, a brain region involved in processing of size and scale. This connectivity pattern was specific to AIWS when compared to lesions causing other neuropsychiatric disorders and aligned with neuroimaging findings in patients with AIWS from other etiologies. ConclusionLesions causing AIWS fall within a specific brain network defined by connectivity to two distinct brain regions, one region involved in body perception and another in processing of size and scale.\nVAL-1221 FOR THE TREATMENT OF PATIENTS WITH LAFORA DISEASE: STUDY PROTOCOL FOR A SINGLE-ARM, OPEN-LABEL CLINICAL TRIAL\nAuthors: Muccioli, L.; Vignatelli, L.; Tappata, M.; Mazzone, S.; Zenesini, C.; DEFEAT-LD study group, ; Armstrong, D.; Michelucci, R.; Bisulli, F.\nScore: 2.0, Published: 2024-02-03 DOI: 10.1101/2024.02.01.24302141\nIntroductionLafora Disease (LD) is an ultrarare fatal progressive myoclonic epilepsy, causing drug-resistant epilepsy, myoclonus, and psychomotor deterioration. LD is caused by mutations in EPM2A or NHLRC1, which lead to the accumulation of polyglucosans in the brain and neurodegeneration. There are no approved treatments for LD. VAL-1221 is a fusion protein comprised of the Fab portion of a cell-penetrating antibody and recombinant human acid alpha glucosidase, and has demonstrated an ability to clear polyglucosans. We hypothesize that intravenous infusion of VAL-1221 might be able to degrade cerebral polyglucosans and stabilize or improve disease outcomes. The aim of this study is to assess the safety and preliminary efficacy of VAL-1221 in patients with LD. Methods and analysisThe study is a phase 2, single-arm, open-label, baseline-controlled clinical trial which will be conducted in a single investigational study center in Italy, namely the sponsor \"IRCCS Istituto delle Scienze Neurologiche di Bologna - Azienda USL di Bologna\". The study will enroll 6 genetically-confirmed patients with mid- to late stage LD. The global duration of the study for each participant will be 18 months, including screening period, open-label treatment (12 months), and follow-up period. VAL-1221 20 mg/kg will be administered as an intravenous infusion every week for 3 weeks, then every other week. Patients will undergo full clinical assessments at baseline, at an intermediate and at the end-of-treatment visit. The primary objective is to evaluate the safety. The exploratory efficacy endpoints will be related to epilepsy, neuropsychological and motor functions, global assessment and disease burden, in addition to biomarkers. Statistical analyses will be primarily descriptive. Ethics and disseminationThe study protocol was approved by the local ethics committee (number 232-2023-FARM-AUSLBO-23020, 22-Mar-2023). The results of this study will be disseminated by the investigators through presentations at international scientific conferences and reported in peer-reviewed scientific journals. Trial RegistrationEuropean Union Clinical Trials Register (EudraCT number 2023-000185-34). ARTICLE SUMMARYO_ST_ABSStrengths And Limitations Of This StudyC_ST_ABSO_LIThis was the first approved study protocol developed for a potential disease-modifying drug in Lafora disease C_LIO_LIThe study will be open label due to the serious and life-threatening nature of the disease and its rarity C_LIO_LIGiven the lack of a control group in this study, the exploratory endpoints observed will be baseline-controlled and compared with natural history studies C_LIO_LIThe rationale for VAL-1221 dose and schedule comes from experience in Pompe disease clinical trials and expanded access programs C_LI\nExplainable artificial intelligence identifies an AQP4 polymorphism-based risk score associated with brain amyloid burden\nAuthors: Beer, S.; Elmenhorst, D.; Bischof, G. N.; Ramirez, A.; Bauer, A.; Drzezga, A.; for the Alzheimers Disease Neuroimaging Initiative, Score: 1.2, Published: 2024-02-06 DOI: 10.1101/2024.02.05.24302223\nAquaporin-4 (AQP4) is an integral component of the glymphatic system, today considered a crucial pathway for removing brain interstitial solutes like amyloid-{beta} (A{beta}). Evidence exists that genetic variation of AQP4 impacts A{beta} clearance, clinical outcome in Alzheimers disease as well as sleep measures. We examined whether a risk score calculated from several AQP4 single-nucleotide polymorphisms (SNPs) is related to A{beta} neuropathology in older cognitively unimpaired individuals. We used a machine learning approach with decision tree ensembles and explainable artificial intelligence (AI) to extract information on synergistic effects of AQP4 SNPs on brain amyloid burden from the Alzheimers Disease Neuroimaging Initiative (ADNI) cohort. From this information, we formulated a sex-specific AQP4 SNP-based risk score and evaluated it on the basis of data from the screening process of the Anti-Amyloid Treatment in Asymptomatic Alzheimers (A4) study. We found in both cohorts significant associations of the risk score with brain amyloid burden as well as amyloid positivity. The results support the hypothesis of an involvement of the glymphatic system, and particularly AQP4, in brain amyloid aggregation pathology. They suggest also that different AQP4 SNPs exert a synergistic effect on the build-up of brain amyloid burden.\nIdentification of a novel mosaic MTOR variant in purified neuronal DNA from depth electrodes in a patient with focal cortical dysplasia\nAuthors: Klein, K. M.; Mascarenhas, R.; Merrikh, D.; Khanbabaei, M.; Maroilley, T.; Kaur, N.; Liu, Y.; Soule, T.; Manalo, M. K.; Tamura, G.; Jacobs, J.; Hader, W.; Pfeffer, G.; Tarailo-Graovac, M.\nScore: 1.2, Published: 2024-01-20 DOI: 10.1101/2024.01.18.24301006\nBackgroundRecent studies have identified brain somatic variants as a cause of focal epilepsy. These studies relied on resected tissue from epilepsy surgery which is not available in most patients. The use of trace tissue adherent to depth electrodes used for stereo electroencephalography (stereo EEG) has been proposed as an alternative but is hampered by the low cell quality and contamination by non-brain cells. Here, we use our improved depth electrode harvesting technique that purifies neuronal nuclei to achieve molecular diagnosis in a patient with focal cortical dysplasia (FCD). MethodsDepth electrode tips were collected, pooled by brain region and seizure onset zone, nuclei isolated and sorted using fluorescence-activated nuclei sorting (FANS). Somatic DNA was amplified from neuronal and astrocyte nuclei using primary template amplification followed by exome sequencing of neuronal DNA from the affected pool, unaffected pool, and saliva. The identified variant was validated using droplet digital PCR. ResultsAn adolescent male with drug-resistant genetic-structural epilepsy due to left anterior insula FCD had daily focal aware seizures. Stereo EEG confirmed seizure onset in the left anterior insula. The two anterior insula electrodes were combined as the affected pool and three frontal electrodes as the unaffected pool. FANS isolated 140 neuronal nuclei from the affected and 245 neuronal nuclei from the unaffected pool. A novel somatic missense MTOR variant (p.Leu489Met, CADD score 23.7) was identified in the affected neuronal sample. Droplet digital PCR confirmed a mosaic gradient (VAF 0.78% in affected neuronal sample, variant was absent in all other samples). ConclusionsOur finding confirms that harvesting neuronal DNA from depth electrodes followed by molecular analysis to identify brain somatic variants is feasible. Our novel method represents a significant improvement compared to the previous method by focusing the analysis on high quality cells of the cell type of interest.\n",
  "wordCount" : "1319",
  "inLanguage": "en",
  "datePublished": "2024-02-11T10:38:55Z",
  "dateModified": "2024-02-11T10:38:55Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/neurology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      neurology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 11, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.24301332">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.24301332" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.24301332">
        <p class="paperTitle">Lesions causing Alice in Wonderland Syndrome map to a common brain network linking body and size perception</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.24301332" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.24301332" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Friedrich, M. U.; Baughan, E. C.; Kletenik, I.; Younger, E.; Zhao, C. W.; Howard, C.; Ferguson, M. A.; Chen, A.; Zeller, D.; Piervincenzi, C.; Tommasin, S.; Pantano, P.; Blanke, O.; Prasad, S.; Nielsen, J.; Fox, M. D.</p>
        <p class="info">Score: 32.0, Published: 2024-01-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.24301332' target='https://doi.org/10.1101/2024.01.17.24301332'> 10.1101/2024.01.17.24301332</a></p>
        <p class="abstract">BackgroundIn Lewis Carrolls 1865 novel &#34;Alices Adventures in Wonderland&#34;, the protagonist experiences distortions in the size of her body and those of others. This fiction becomes reality in neurological patients with Alice in Wonderland Syndrome (AIWS). Brain lesions causing AIWS may offer unique insights into the syndromes elusive neuroanatomy.

MethodsA systematic literature search identified 37 cases of lesion-induced AIWS. Lesion locations were mapped onto a brain atlas and functional connectivity between each lesion location and other brain regions was estimated using resting-state fMRI data from 1000 healthy subjects. Connections common to AIWS lesions were identified and compared to connections from 1073 lesions associated with 25 other neuropsychiatric disorders. Alignment between this lesion-derived AIWS network and neuroimaging findings from patients with AIWS due to other etiologies was assessed.

ResultsAlthough AIWS lesions occurred in many different brain locations, these lesions fell within a specific, functionally connected brain network. This network was defined by connectivity to the right extrastriate body area, a brain region selectively activated by viewing body parts, and the inferior parietal cortex, a brain region involved in processing of size and scale. This connectivity pattern was specific to AIWS when compared to lesions causing other neuropsychiatric disorders and aligned with neuroimaging findings in patients with AIWS from other etiologies.

ConclusionLesions causing AIWS fall within a specific brain network defined by connectivity to two distinct brain regions, one region involved in body perception and another in processing of size and scale.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.24302141">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.24302141" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.24302141">
        <p class="paperTitle">VAL-1221 FOR THE TREATMENT OF PATIENTS WITH LAFORA DISEASE: STUDY PROTOCOL FOR A SINGLE-ARM, OPEN-LABEL CLINICAL TRIAL</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.24302141" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.24302141" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Muccioli, L.; Vignatelli, L.; Tappata, M.; Mazzone, S.; Zenesini, C.; DEFEAT-LD study group,  ; Armstrong, D.; Michelucci, R.; Bisulli, F.</p>
        <p class="info">Score: 2.0, Published: 2024-02-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.24302141' target='https://doi.org/10.1101/2024.02.01.24302141'> 10.1101/2024.02.01.24302141</a></p>
        <p class="abstract">IntroductionLafora Disease (LD) is an ultrarare fatal progressive myoclonic epilepsy, causing drug-resistant epilepsy, myoclonus, and psychomotor deterioration. LD is caused by mutations in EPM2A or NHLRC1, which lead to the accumulation of polyglucosans in the brain and neurodegeneration. There are no approved treatments for LD. VAL-1221 is a fusion protein comprised of the Fab portion of a cell-penetrating antibody and recombinant human acid alpha glucosidase, and has demonstrated an ability to clear polyglucosans. We hypothesize that intravenous infusion of VAL-1221 might be able to degrade cerebral polyglucosans and stabilize or improve disease outcomes. The aim of this study is to assess the safety and preliminary efficacy of VAL-1221 in patients with LD.

Methods and analysisThe study is a phase 2, single-arm, open-label, baseline-controlled clinical trial which will be conducted in a single investigational study center in Italy, namely the sponsor &#34;IRCCS Istituto delle Scienze Neurologiche di Bologna - Azienda USL di Bologna&#34;. The study will enroll 6 genetically-confirmed patients with mid- to late stage LD. The global duration of the study for each participant will be 18 months, including screening period, open-label treatment (12 months), and follow-up period. VAL-1221 20 mg/kg will be administered as an intravenous infusion every week for 3 weeks, then every other week. Patients will undergo full clinical assessments at baseline, at an intermediate and at the end-of-treatment visit. The primary objective is to evaluate the safety. The exploratory efficacy endpoints will be related to epilepsy, neuropsychological and motor functions, global assessment and disease burden, in addition to biomarkers. Statistical analyses will be primarily descriptive.

Ethics and disseminationThe study protocol was approved by the local ethics committee (number 232-2023-FARM-AUSLBO-23020, 22-Mar-2023). The results of this study will be disseminated by the investigators through presentations at international scientific conferences and reported in peer-reviewed scientific journals.

Trial RegistrationEuropean Union Clinical Trials Register (EudraCT number 2023-000185-34).

ARTICLE SUMMARYO_ST_ABSStrengths And Limitations Of This StudyC_ST_ABSO_LIThis was the first approved study protocol developed for a potential disease-modifying drug in Lafora disease
C_LIO_LIThe study will be open label due to the serious and life-threatening nature of the disease and its rarity
C_LIO_LIGiven the lack of a control group in this study, the exploratory endpoints observed will be baseline-controlled and compared with natural history studies
C_LIO_LIThe rationale for VAL-1221 dose and schedule comes from experience in Pompe disease clinical trials and expanded access programs
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.05.24302223">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.05.24302223" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.05.24302223">
        <p class="paperTitle">Explainable artificial intelligence identifies an AQP4 polymorphism-based risk score associated with brain amyloid burden</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.05.24302223" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.05.24302223" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Beer, S.; Elmenhorst, D.; Bischof, G. N.; Ramirez, A.; Bauer, A.; Drzezga, A.; for the Alzheimers Disease Neuroimaging Initiative,  </p>
        <p class="info">Score: 1.2, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.05.24302223' target='https://doi.org/10.1101/2024.02.05.24302223'> 10.1101/2024.02.05.24302223</a></p>
        <p class="abstract">Aquaporin-4 (AQP4) is an integral component of the glymphatic system, today considered a crucial pathway for removing brain interstitial solutes like amyloid-{beta} (A{beta}). Evidence exists that genetic variation of AQP4 impacts A{beta} clearance, clinical outcome in Alzheimers disease as well as sleep measures. We examined whether a risk score calculated from several AQP4 single-nucleotide polymorphisms (SNPs) is related to A{beta} neuropathology in older cognitively unimpaired individuals. We used a machine learning approach with decision tree ensembles and explainable artificial intelligence (AI) to extract information on synergistic effects of AQP4 SNPs on brain amyloid burden from the Alzheimers Disease Neuroimaging Initiative (ADNI) cohort. From this information, we formulated a sex-specific AQP4 SNP-based risk score and evaluated it on the basis of data from the screening process of the Anti-Amyloid Treatment in Asymptomatic Alzheimers (A4) study. We found in both cohorts significant associations of the risk score with brain amyloid burden as well as amyloid positivity. The results support the hypothesis of an involvement of the glymphatic system, and particularly AQP4, in brain amyloid aggregation pathology. They suggest also that different AQP4 SNPs exert a synergistic effect on the build-up of brain amyloid burden.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.24301006">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.24301006" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.24301006">
        <p class="paperTitle">Identification of a novel mosaic MTOR variant in purified neuronal DNA from depth electrodes in a patient with focal cortical dysplasia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.24301006" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.24301006" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Klein, K. M.; Mascarenhas, R.; Merrikh, D.; Khanbabaei, M.; Maroilley, T.; Kaur, N.; Liu, Y.; Soule, T.; Manalo, M. K.; Tamura, G.; Jacobs, J.; Hader, W.; Pfeffer, G.; Tarailo-Graovac, M.</p>
        <p class="info">Score: 1.2, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.24301006' target='https://doi.org/10.1101/2024.01.18.24301006'> 10.1101/2024.01.18.24301006</a></p>
        <p class="abstract">BackgroundRecent studies have identified brain somatic variants as a cause of focal epilepsy. These studies relied on resected tissue from epilepsy surgery which is not available in most patients. The use of trace tissue adherent to depth electrodes used for stereo electroencephalography (stereo EEG) has been proposed as an alternative but is hampered by the low cell quality and contamination by non-brain cells. Here, we use our improved depth electrode harvesting technique that purifies neuronal nuclei to achieve molecular diagnosis in a patient with focal cortical dysplasia (FCD).

MethodsDepth electrode tips were collected, pooled by brain region and seizure onset zone, nuclei isolated and sorted using fluorescence-activated nuclei sorting (FANS). Somatic DNA was amplified from neuronal and astrocyte nuclei using primary template amplification followed by exome sequencing of neuronal DNA from the affected pool, unaffected pool, and saliva. The identified variant was validated using droplet digital PCR.

ResultsAn adolescent male with drug-resistant genetic-structural epilepsy due to left anterior insula FCD had daily focal aware seizures. Stereo EEG confirmed seizure onset in the left anterior insula. The two anterior insula electrodes were combined as the affected pool and three frontal electrodes as the unaffected pool. FANS isolated 140 neuronal nuclei from the affected and 245 neuronal nuclei from the unaffected pool. A novel somatic missense MTOR variant (p.Leu489Met, CADD score 23.7) was identified in the affected neuronal sample. Droplet digital PCR confirmed a mosaic gradient (VAF 0.78% in affected neuronal sample, variant was absent in all other samples).

ConclusionsOur finding confirms that harvesting neuronal DNA from depth electrodes followed by molecular analysis to identify brain somatic variants is feasible. Our novel method represents a significant improvement compared to the previous method by focusing the analysis on high quality cells of the cell type of interest.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
