<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>neurology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>neurology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A pathogenic variant in RAB32 causes autosomal dominant Parkinsons disease and activates LRRK2 kinase
Authors: Gustavsson, E. K.; Follett, J.; Trinh, J.; Barodia, S. K.; Real, R.; Liu, Z.; Grant-Peters, M.; Fox, J. D.; Appel-Cresswell, S.; Stoessl, J. A.; Rajput, A.; Rajput, A. H.; Auer, R.; Tilney, R.; Sturm, M.; Haack, T. B.; Lesage, S.; Tesson, C.; Brice, A.; Vilarino-Gueell, C.; Ryten, M.; Goldberg, M. S.; West, A. B.; Hu, M.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/neurology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="neurology" />
<meta property="og:description" content="A pathogenic variant in RAB32 causes autosomal dominant Parkinsons disease and activates LRRK2 kinase
Authors: Gustavsson, E. K.; Follett, J.; Trinh, J.; Barodia, S. K.; Real, R.; Liu, Z.; Grant-Peters, M.; Fox, J. D.; Appel-Cresswell, S.; Stoessl, J. A.; Rajput, A.; Rajput, A. H.; Auer, R.; Tilney, R.; Sturm, M.; Haack, T. B.; Lesage, S.; Tesson, C.; Brice, A.; Vilarino-Gueell, C.; Ryten, M.; Goldberg, M. S.; West, A. B.; Hu, M." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/neurology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-21T10:39:08+00:00" />
<meta property="article:modified_time" content="2024-01-21T10:39:08+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="neurology"/>
<meta name="twitter:description" content="A pathogenic variant in RAB32 causes autosomal dominant Parkinsons disease and activates LRRK2 kinase
Authors: Gustavsson, E. K.; Follett, J.; Trinh, J.; Barodia, S. K.; Real, R.; Liu, Z.; Grant-Peters, M.; Fox, J. D.; Appel-Cresswell, S.; Stoessl, J. A.; Rajput, A.; Rajput, A. H.; Auer, R.; Tilney, R.; Sturm, M.; Haack, T. B.; Lesage, S.; Tesson, C.; Brice, A.; Vilarino-Gueell, C.; Ryten, M.; Goldberg, M. S.; West, A. B.; Hu, M."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "neurology",
      "item": "https://trxiv.yorks0n.com/posts/neurology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "neurology",
  "name": "neurology",
  "description": "A pathogenic variant in RAB32 causes autosomal dominant Parkinsons disease and activates LRRK2 kinase\nAuthors: Gustavsson, E. K.; Follett, J.; Trinh, J.; Barodia, S. K.; Real, R.; Liu, Z.; Grant-Peters, M.; Fox, J. D.; Appel-Cresswell, S.; Stoessl, J. A.; Rajput, A.; Rajput, A. H.; Auer, R.; Tilney, R.; Sturm, M.; Haack, T. B.; Lesage, S.; Tesson, C.; Brice, A.; Vilarino-Gueell, C.; Ryten, M.; Goldberg, M. S.; West, A. B.; Hu, M.",
  "keywords": [
    
  ],
  "articleBody": " A pathogenic variant in RAB32 causes autosomal dominant Parkinsons disease and activates LRRK2 kinase\nAuthors: Gustavsson, E. K.; Follett, J.; Trinh, J.; Barodia, S. K.; Real, R.; Liu, Z.; Grant-Peters, M.; Fox, J. D.; Appel-Cresswell, S.; Stoessl, J. A.; Rajput, A.; Rajput, A. H.; Auer, R.; Tilney, R.; Sturm, M.; Haack, T. B.; Lesage, S.; Tesson, C.; Brice, A.; Vilarino-Gueell, C.; Ryten, M.; Goldberg, M. S.; West, A. B.; Hu, M. T.; Morris, H. R.; Sharma, M.; Gan-Or, Z.; Samanci, B.; Lis, P.; Tocino, T.; Amouri, R.; Ben Sassi, S.; Hentati, F.; Tonelli, F.; Alessi, D. R.; Farrer, M. J.\nScore: 9.9, Published: 2024-01-18 DOI: 10.1101/2024.01.17.24300927\nBackground Parkinson disease (PD) is a progressive neurodegenerative disorder. Mendelian forms have revealed multiple genes, with a notable emphasis on membrane trafficking; RAB GTPases play an important role in PD as a subset are both regulators and substrates of LRRK2 protein kinase. To explore the role of RAB GTPases in PD, we undertook a comprehensive examination of their genetic variability in familial PD. Methods Affected probands from 130 multi-incident PD families underwent whole-exome sequencing and genotyping, Potential pathogenic variants in 61 RAB GTPases were genotyped in relatives to assess disease segregation. These variants were also genotyped in a larger case-control series, totaling 3,078 individuals (2,734 with PD). The single most significant finding was subsequently validated within genetic data (6,043 with PD). Clinical and pathologic findings were summarized for gene-identified patients, and haplotypes were constructed. In parallel, wild-type and mutant RAB GTPase structural variation, protein interactions, and resultant enzyme activities were assessed. Findings We found RAB32 c.213C\u003eG (Ser71Arg) to co-segregate with autosomal dominant parkinsonism in three multi-incident families. RAB32 Ser71Arg was also significantly associated with PD in case-control samples: genotyping and database searches identified thirteen more patients with the same variant that was absent in unaffected controls. Notably, RAB32 Ser71Arg heterozygotes share a common haplotype. At autopsy, one patient had sparse neurofibrillary tangle pathology in the midbrain and thalamus, without Lewy body pathology. In transfected cells the RAB32 Arg71 was twice as potent as Ser71 wild type to activate LRRK2 kinase. Interpretation Our study provides unequivocal evidence to implicate RAB32 Ser71Arg in PD. Functional analysis demonstrates LRRK2 kinase activation. We provide a mechanistic explanation to expand and unify the etiopathogenesis of monogenic PD. Funding National Institutes of Health, the Canada Excellence Research Chairs program, Aligning Science Across Parkinson s, the Michael J. Fox Foundation for Parkinsons Research, and the UK Medical Research Council.\nPeripheral innate immunophenotype in neurodegenerative disease: blood-based profiles and links to survival\nAuthors: Strauss, A.; Swann, P.; Kigar, S.; Christou, R.; Yarkoni, N. S.; Murley, A. G.; Chouliaras, L.; Savulich, G.; Bevan-Jones, R.; Surendranathan, A.; O'Brien, J.; Rowe, J.; Malpetti, M.\nScore: 6.7, Published: 2024-01-17 DOI: 10.1101/2024.01.16.24301348\nThe innate immune system plays an integral role in the progression of many neurodegenerative diseases. In addition to central innate immune cells (e.g. cerebral microglia), peripheral innate immune cells (e.g. blood monocytes, natural killer cells, and dendritic cells) may also differ in these conditions. However, the characterization of peripheral innate immune cell types across different neurodegenerative diseases remains incomplete. This study aimed to characterize peripheral innate immune profiles using flow cytometry for immunophenotyping of peripheral blood mononuclear cells, in n=148 people with Alzheimers disease (AD), Frontotemporal Dementia (FTD), Corticobasal syndrome (CBS), Progressive Supranuclear Palsy (PSP), Lewy Body Disease (LBD) as compared to n=37 healthy controls. To compare groups, we used Principal Component Analysis and multivariate Dissimilarity analysis across 19 innate immune cell types. We identified pro-inflammatory profiles that significantly differ between patients with all-cause dementia and healthy controls, with some significant differences between groups. Regression analysis confirmed that time to death following the blood test correlated with the individuals immune profile weighting, positively to TREM2+ and nonclassical monocytes and negatively to classical monocytes. Taken together, these results describe transdiagnostic peripheral immune profiles and highlight the link between prognosis and the monocyte cellular subdivision and function (as measured by surface protein expression). The results suggest that blood-derived innate immune profiles can inform sub-populations of cells relevant for specific neurodegenerative diseases that are significantly linked to accelerated disease progression and worse survival outcomes across diagnoses. Blood-based innate immune profiles may contribute to enhanced precision medicine approaches dementia, helping to identify and monitor therapeutic targets and stratify patients for candidate immunotherapies.\nAlzheimer's Disease biological PET staging using plasma p217+tau\nAuthors: Feizpour, A.; Dore, V.; Krishnadas, N.; Bourgeat, P.; Doecke, J. D.; Saad, Z.; Triana-Baltzer, G.; Laws, S. M.; Shishegar, R.; Huang, K.; Fowler, C.; Ward, L.; Masters, C. L.; Mejan-Fripp, J.; Kolb, H. C.; Villemagne, V. L.; Rowe, C. C.\nScore: 3.8, Published: 2024-01-13 DOI: 10.1101/2024.01.11.24301180\nBackgroundPlasma phospho-tau biomarkers, such as p217+tau, excel at identifying Alzheimers Disease (AD) neuropathology. However, questions remain regarding their capacity to inform AD biological PET stages at group level and maintain the same precision at individual patient level. MethodParticipants included 248 cognitively unimpaired (CU) and 227 cognitively impaired (CI) individuals, with Janssen plasma p217+tau Simoa(R) assay, 18F-NAV4694 A{beta} PET (A) and 18F-MK6240 tau PET (T) data. Biological PET stages were defined based on the draft NIA-AA Revised Criteria (July 2023): Initial (A+T-), Early (A+TMTL+), Intermediate (A+TMOD+), and Advanced (A+THIGH+). We used thresholds for A+ of 25 Centiloid and for THIGH of 80 Centaur (2.68 SUVRtemporo-parietal). Adding an A-T- stage for comparison, we assessed the performance of p217+tau in discriminating between these stages at the group level using Receiver Operating Characteristic (ROC) analysis and at the individual level using logistic regression. ResultsPlasma p217+tau concentrations increased across the stages, with significant differences between them, except for the Initial and Early stages. Screening for Advanced (vs. lower stages), combined Intermediate/Advanced (vs. lower stages), or all stages (vs. A-T-), p217+tau showed good group-level discriminations (AUC 0.91, 0.92 and 0.92; CI only: AUC 0.83, 0.89, 0.93, respectively). At the individual level, the likelihood of PET stage vs. p217+tau level showed good discrimination of A-T- vs any A+ stage and of combined Intermediate/Advanced disease stage vs lower stages in the CI. ConclusionIn addition to accurately screening for A+ individuals, plasma p217+tau shows promise for separating persons with either Intermediate or Advanced stage AD from those at a lower stage, providing prognostic information and informing better selection for trials and disease modifying therapies.\nLesions causing Alice in Wonderland Syndrome map to a common brain network linking body and size perception\nAuthors: Friedrich, M. U.; Baughan, E. C.; Kletenik, I.; Younger, E.; Zhao, C. W.; Howard, C.; Ferguson, M. A.; Chen, A.; Zeller, D.; Piervincenzi, C.; Tommasin, S.; Pantano, P.; Blanke, O.; Prasad, S.; Nielsen, J.; Fox, M. D.\nScore: 2.5, Published: 2024-01-18 DOI: 10.1101/2024.01.17.24301332\nBackground: In Lewis Carroll s 1865 novel \"Alice s Adventures in Wonderland\", the protagonist experiences distortions in the size of her body and those of others. This fiction becomes reality in neurological patients with Alice in Wonderland Syndrome (AIWS). Brain lesions causing AIWS may offer unique insights into the syndrome s elusive neuroanatomy. Methods: A systematic literature search identified 37 cases of lesion-induced AIWS. Lesion locations were mapped onto a brain atlas and functional connectivity between each lesion location and other brain regions was estimated using resting-state fMRI data from 1000 healthy subjects. Connections common to AIWS lesions were identified and compared to connections from 1073 lesions associated with 25 other neuropsychiatric disorders. Alignment between this lesion-derived AIWS network and neuroimaging findings from patients with AIWS due to other etiologies was assessed. Results: Although AIWS lesions occurred in many different brain locations, these lesions fell within a specific, functionally connected brain network. This network was defined by connectivity to the right extrastriate body area, a brain region selectively activated by viewing body parts, and the inferior parietal cortex, a brain region involved in processing of size and scale. This connectivity pattern was specific to AIWS when compared to lesions causing other neuropsychiatric disorders and aligned with neuroimaging findings in patients with AIWS from other etiologies. Conclusion: Lesions causing AIWS fall within a specific brain network defined by connectivity to two distinct brain regions, one region involved in body perception and another in processing of size and scale.\nFunctional connectivity of stimulus-evoked brain responses to natural speech in post-stroke aphasia.\nAuthors: Mehraram, R.; De Clercq, P.; Kries, J.; Vandermosten, M.; Francart, T.\nScore: 2.5, Published: 2024-01-16 DOI: 10.1101/2024.01.15.24301324\nOne out of three stroke-patients develop language processing impairment known as aphasia. The need for ecological validity of the existing diagnostic tools motivates research on biomarkers, such as stimulus-evoked brain responses. With the aim of enhancing the physiological interpretation of the latter, we used EEG to investigate how functional brain network patterns associated with the neural response to natural speech are affected in persons with post-stroke chronic aphasia. EEG was recorded from 24 healthy controls and 40 persons with aphasia while they listened to a story. Stimulus-evoked brain responses at all scalp regions were measured as neural envelope tracking in the delta (0.5-4 Hz), theta (4-8 Hz) and low-gamma bands (30-49 Hz) using mutual information. Functional connectivity between neural-tracking signals was measured, and the Network-Based Statistics toolbox was used to: 1) assess the added value of the neural tracking vs EEG time series, 2) test between-group differences and 3) investigate any association with language performance in aphasia. Graph theory was also used to investigate topological alterations in aphasia. Functional connectivity was higher when assessed from neural tracking compared to EEG time series. Persons with aphasia showed weaker low-gamma-band left-hemispheric connectivity, and graph theory-based results showed a greater network segregation and higher region-specific node strength. Aphasia also exhibited a correlation between delta-band connectivity within the left pre-frontal region and language performance. We demonstrated the added value of neural tracking when investigating functional brain networks associated with natural speech processing in post-stroke aphasia. Its higher sensitivity to language-related brain circuits favours its use as informative biomarker for the assessment of aphasia.\nSomatic mutations in Alzheimer-associated tetraploid neurons\nAuthors: Lopez-Sanchez, N.; Rabano, A.; Frade, J. M.\nScore: 1.4, Published: 2024-01-15 DOI: 10.1101/2024.01.13.24301214\nAn early pathological process affecting the brain of Alzheimers disease (AD) is the reactivation of the cell cycle in neurons followed by somatic neuronal tetraploidization (NT). NT also increases with age, and ageing has been shown to correlate with the accumulation of DNA somatic mutations. In this study, we have evaluated the presence of somatic mutations including single nucleotide variants (SNVs) and indels in genomic DNA from tetraploid neurons obtained from the parietal cortex of AD patients, compared with diploid neurons from control individuals and AD patients. Here we show that, in contrast to somatic indels, the proportion of somatic SNVs (sSNVs) significantly increases in the exome of tetraploid neurons, having increased levels of T to C (A to G) transitions, a type of mutation that is associated with oxidative stress. This finding correlates with the over-representation of sSNVs in genes involved in oxidative stress response and DNA repair, suggesting that these alterations exacerbate oxidative DNA damage in tetraploid neurons. sSNVs affecting cancer-related (CR) genes showed a greater molecular pathogenicity score compared with those from a random sample of genes. We propose that neuronal tetraploidy is stochastically triggered through a CR mechanism in neurons whose DNA repair genes become mutated in an oxidative stress scenario. This mechanism likely participates in the etiology of AD.\nAnti-CD20 antibodies in relapsing multiple sclerosis: Protocol for a systematic review and network meta-analysis\nAuthors: Navarro, C. E.; Cardenas-Robledo, S.; Macea, J.; Ortiz-Cano, N. A.; Florez, I. D.\nScore: 0.8, Published: 2024-01-15 DOI: 10.1101/2024.01.12.24301256\nBackgroundMultiple sclerosis is a chronic disease of the central nervous system characterized by autoimmune demyelination. Various immunomodulatory medications are available for its treatment, among which anti-CD20 monoclonal antibodies have demonstrated superior efficacy compared to other disease-modifying therapies. However, there is a lack of direct comparison between the four available anti-CD20 monoclonal antibodies. Therefore, this study aims to systematically assess the relative efficacy and safety of anti-CD20 monoclonal antibodies for treating relapsing multiple sclerosis. Materials and MethodsWe will conduct a systematic review of phase IIb and phase III clinical trials of anti-CD20 monoclonal antibodies for the treatment of relapsing multiple sclerosis, following the Preferred Reporting Items for Systematic Reviews and Meta-Analysis guidelines. A comprehensive review of the Cochrane Central Register of Controlled Trials; MEDLINE; Embase; ClinicalTrials; International Clinical Trials Registry Platform; OpenGray; and MedRxiv will be performed. Two independent reviewers will select titles, abstracts, and eligible full texts to execute the data extraction. The risk of bias will be assessed with the Cochrane RoB 2 tool. Data on pre-specified outcomes will be analyzed for the selected articles using a random-effects network meta-analysis with a frequentist framework. Summary statistics, along with 95% confidence intervals will be presented, and the effectiveness and safety of each intervention will be ranked using the surface under the cumulative ranking curve. A missing data and a sensitivity analysis will be conducted. We will use the GRADE approach to assess the certainty of the direct, indirect, and network estimate for all outcomes. Results and ConclusionThe results will provide evidence of the relative efficacy and safety of anti-CD20 monoclonal antibodies for the treatment of relapsing multiple sclerosis. Systematic review registrationPROSPERO registration number CRD42023437996\nData leakage in deep learning studies of translational EEG\nAuthors: Brookshire, G.; Kasper, J.; Blauch, N.; Wu, Y.; Glatt, R.; Merrill, D. A.; Gerrol, S.; Yoder, K. J.; Quirk, C.; Lucero, C.\nScore: 0.5, Published: 2024-01-17 DOI: 10.1101/2024.01.16.24301366\nA growing number of studies use deep neural networks (DNNs) to identify diseases from recordings of brain activity. DNN studies of electroencephalography (EEG) typically use cross-validation to test how accurately a model can predict the disease state of held-out test data. In these studies, segments of EEG data are often randomly assigned to the training or test sets. As a consequence, data from individual subjects appears in both training and test data. Could high test-set accuracy reflect leakage from subject-specific representations, rather than patterns that identify a disease? We address this question by testing the performance of DNN classifiers using segment-based holdout (where EEG segments from one subject can appear in both the training and test sets), and comparing this to their performance using subject-based holdout (where individual subjects data appears exclusively in either the training set or the test set). We compare segment-based and subject-based holdout in two EEG datasets: one classifying Alzheimers disease, and the other classifying epileptic seizures. In both datasets, we find that performance on previously-unseen subjects is strongly overestimated when models are trained using segment-based holdout. Next, we survey the literature and find that the majority of translational DNN-EEG studies use segment-based holdout, and therefore overestimate model performance on new subjects. In a hospital or doctors office, clinicians need to diagnose new patients whose data was not used in training the model; segment-based holdout, therefore, does not reflect the real-world performance of a translational DNN model. When evaluating how DNNs could be used for medical diagnosis, models must be tested on subjects whose data was not included in the training set.\n",
  "wordCount" : "2476",
  "inLanguage": "en",
  "datePublished": "2024-01-21T10:39:08Z",
  "dateModified": "2024-01-21T10:39:08Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/neurology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      neurology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 21, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.24300927">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.24300927" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.24300927">
        <p class="paperTitle">A pathogenic variant in RAB32 causes autosomal dominant Parkinsons disease and activates LRRK2 kinase</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.24300927" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.24300927" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gustavsson, E. K.; Follett, J.; Trinh, J.; Barodia, S. K.; Real, R.; Liu, Z.; Grant-Peters, M.; Fox, J. D.; Appel-Cresswell, S.; Stoessl, J. A.; Rajput, A.; Rajput, A. H.; Auer, R.; Tilney, R.; Sturm, M.; Haack, T. B.; Lesage, S.; Tesson, C.; Brice, A.; Vilarino-Gueell, C.; Ryten, M.; Goldberg, M. S.; West, A. B.; Hu, M. T.; Morris, H. R.; Sharma, M.; Gan-Or, Z.; Samanci, B.; Lis, P.; Tocino, T.; Amouri, R.; Ben Sassi, S.; Hentati, F.; Tonelli, F.; Alessi, D. R.; Farrer, M. J.</p>
        <p class="info">Score: 9.9, Published: 2024-01-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.24300927' target='https://doi.org/10.1101/2024.01.17.24300927'> 10.1101/2024.01.17.24300927</a></p>
        <p class="abstract">Background Parkinson disease (PD) is a progressive neurodegenerative disorder. Mendelian forms have revealed multiple genes, with a notable emphasis on membrane trafficking; RAB GTPases play an important role in PD as a subset are both regulators and substrates of LRRK2 protein kinase. To explore the role of RAB GTPases in PD, we undertook a comprehensive examination of their genetic variability in familial PD. Methods Affected probands from 130 multi-incident PD families underwent whole-exome sequencing and genotyping, Potential pathogenic variants in 61 RAB GTPases were genotyped in relatives to assess disease segregation. These variants were also genotyped in a larger case-control series, totaling 3,078 individuals (2,734 with PD). The single most significant finding was subsequently validated within genetic data (6,043 with PD). Clinical and pathologic findings were summarized for gene-identified patients, and haplotypes were constructed. In parallel, wild-type and mutant RAB GTPase structural variation, protein interactions, and resultant enzyme activities were assessed. Findings We found RAB32 c.213C&gt;G (Ser71Arg) to co-segregate with autosomal dominant parkinsonism in three multi-incident families. RAB32 Ser71Arg was also significantly associated with PD in case-control samples: genotyping and database searches identified thirteen more patients with the same variant that was absent in unaffected controls. Notably, RAB32 Ser71Arg heterozygotes share a common haplotype. At autopsy, one patient had sparse neurofibrillary tangle pathology in the midbrain and thalamus, without Lewy body pathology. In transfected cells the RAB32 Arg71 was twice as potent as Ser71 wild type to activate LRRK2 kinase. Interpretation Our study provides unequivocal evidence to implicate RAB32 Ser71Arg in PD. Functional analysis demonstrates LRRK2 kinase activation. We provide a mechanistic explanation to expand and unify the etiopathogenesis of monogenic PD. Funding National Institutes of Health, the Canada Excellence Research Chairs program, Aligning Science Across Parkinson s, the Michael J. Fox Foundation for Parkinsons Research, and the UK Medical Research Council.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.16.24301348">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.16.24301348" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.16.24301348">
        <p class="paperTitle">Peripheral innate immunophenotype in neurodegenerative disease: blood-based profiles and links to survival</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.16.24301348" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.16.24301348" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Strauss, A.; Swann, P.; Kigar, S.; Christou, R.; Yarkoni, N. S.; Murley, A. G.; Chouliaras, L.; Savulich, G.; Bevan-Jones, R.; Surendranathan, A.; O&#39;Brien, J.; Rowe, J.; Malpetti, M.</p>
        <p class="info">Score: 6.7, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.16.24301348' target='https://doi.org/10.1101/2024.01.16.24301348'> 10.1101/2024.01.16.24301348</a></p>
        <p class="abstract">The innate immune system plays an integral role in the progression of many neurodegenerative diseases. In addition to central innate immune cells (e.g. cerebral microglia), peripheral innate immune cells (e.g. blood monocytes, natural killer cells, and dendritic cells) may also differ in these conditions. However, the characterization of peripheral innate immune cell types across different neurodegenerative diseases remains incomplete. This study aimed to characterize peripheral innate immune profiles using flow cytometry for immunophenotyping of peripheral blood mononuclear cells, in n=148 people with Alzheimers disease (AD), Frontotemporal Dementia (FTD), Corticobasal syndrome (CBS), Progressive Supranuclear Palsy (PSP), Lewy Body Disease (LBD) as compared to n=37 healthy controls. To compare groups, we used Principal Component Analysis and multivariate Dissimilarity analysis across 19 innate immune cell types. We identified pro-inflammatory profiles that significantly differ between patients with all-cause dementia and healthy controls, with some significant differences between groups. Regression analysis confirmed that time to death following the blood test correlated with the individuals immune profile weighting, positively to TREM2&#43; and nonclassical monocytes and negatively to classical monocytes. Taken together, these results describe transdiagnostic peripheral immune profiles and highlight the link between prognosis and the monocyte cellular subdivision and function (as measured by surface protein expression). The results suggest that blood-derived innate immune profiles can inform sub-populations of cells relevant for specific neurodegenerative diseases that are significantly linked to accelerated disease progression and worse survival outcomes across diagnoses. Blood-based innate immune profiles may contribute to enhanced precision medicine approaches dementia, helping to identify and monitor therapeutic targets and stratify patients for candidate immunotherapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.24301180">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.24301180" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.24301180">
        <p class="paperTitle">Alzheimer&#39;s Disease biological PET staging using plasma p217&#43;tau</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.24301180" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.24301180" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Feizpour, A.; Dore, V.; Krishnadas, N.; Bourgeat, P.; Doecke, J. D.; Saad, Z.; Triana-Baltzer, G.; Laws, S. M.; Shishegar, R.; Huang, K.; Fowler, C.; Ward, L.; Masters, C. L.; Mejan-Fripp, J.; Kolb, H. C.; Villemagne, V. L.; Rowe, C. C.</p>
        <p class="info">Score: 3.8, Published: 2024-01-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.24301180' target='https://doi.org/10.1101/2024.01.11.24301180'> 10.1101/2024.01.11.24301180</a></p>
        <p class="abstract">BackgroundPlasma phospho-tau biomarkers, such as p217&#43;tau, excel at identifying Alzheimers Disease (AD) neuropathology. However, questions remain regarding their capacity to inform AD biological PET stages at group level and maintain the same precision at individual patient level.

MethodParticipants included 248 cognitively unimpaired (CU) and 227 cognitively impaired (CI) individuals, with Janssen plasma p217&#43;tau Simoa(R) assay, 18F-NAV4694 A{beta} PET (A) and 18F-MK6240 tau PET (T) data. Biological PET stages were defined based on the draft NIA-AA Revised Criteria (July 2023): Initial (A&#43;T-), Early (A&#43;TMTL&#43;), Intermediate (A&#43;TMOD&#43;), and Advanced (A&#43;THIGH&#43;). We used thresholds for A&#43; of 25 Centiloid and for THIGH of 80 Centaur (2.68 SUVRtemporo-parietal). Adding an A-T- stage for comparison, we assessed the performance of p217&#43;tau in discriminating between these stages at the group level using Receiver Operating Characteristic (ROC) analysis and at the individual level using logistic regression.

ResultsPlasma p217&#43;tau concentrations increased across the stages, with significant differences between them, except for the Initial and Early stages. Screening for Advanced (vs. lower stages), combined Intermediate/Advanced (vs. lower stages), or all stages (vs. A-T-), p217&#43;tau showed good group-level discriminations (AUC 0.91, 0.92 and 0.92; CI only: AUC 0.83, 0.89, 0.93, respectively). At the individual level, the likelihood of PET stage vs. p217&#43;tau level showed good discrimination of A-T- vs any A&#43; stage and of combined Intermediate/Advanced disease stage vs lower stages in the CI.

ConclusionIn addition to accurately screening for A&#43; individuals, plasma p217&#43;tau shows promise for separating persons with either Intermediate or Advanced stage AD from those at a lower stage, providing prognostic information and informing better selection for trials and disease modifying therapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.24301332">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.24301332" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.24301332">
        <p class="paperTitle">Lesions causing Alice in Wonderland Syndrome map to a common brain network linking body and size perception</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.24301332" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.24301332" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Friedrich, M. U.; Baughan, E. C.; Kletenik, I.; Younger, E.; Zhao, C. W.; Howard, C.; Ferguson, M. A.; Chen, A.; Zeller, D.; Piervincenzi, C.; Tommasin, S.; Pantano, P.; Blanke, O.; Prasad, S.; Nielsen, J.; Fox, M. D.</p>
        <p class="info">Score: 2.5, Published: 2024-01-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.24301332' target='https://doi.org/10.1101/2024.01.17.24301332'> 10.1101/2024.01.17.24301332</a></p>
        <p class="abstract">Background: In Lewis Carroll s 1865 novel &#34;Alice s Adventures in Wonderland&#34;, the protagonist experiences distortions in the size of her body and those of others. This fiction becomes reality in neurological patients with Alice in Wonderland Syndrome (AIWS). Brain lesions causing AIWS may offer unique insights into the syndrome s elusive neuroanatomy. Methods: A systematic literature search identified 37 cases of lesion-induced AIWS. Lesion locations were mapped onto a brain atlas and functional connectivity between each lesion location and other brain regions was estimated using resting-state fMRI data from 1000 healthy subjects. Connections common to AIWS lesions were identified and compared to connections from 1073 lesions associated with 25 other neuropsychiatric disorders. Alignment between this lesion-derived AIWS network and neuroimaging findings from patients with AIWS due to other etiologies was assessed. Results: Although AIWS lesions occurred in many different brain locations, these lesions fell within a specific, functionally connected brain network. This network was defined by connectivity to the right extrastriate body area, a brain region selectively activated by viewing body parts, and the inferior parietal cortex, a brain region involved in processing of size and scale. This connectivity pattern was specific to AIWS when compared to lesions causing other neuropsychiatric disorders and aligned with neuroimaging findings in patients with AIWS from other etiologies. Conclusion: Lesions causing AIWS fall within a specific brain network defined by connectivity to two distinct brain regions, one region involved in body perception and another in processing of size and scale.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.24301324">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.24301324" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.24301324">
        <p class="paperTitle">Functional connectivity of stimulus-evoked brain responses to natural speech in post-stroke aphasia.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.24301324" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.24301324" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mehraram, R.; De Clercq, P.; Kries, J.; Vandermosten, M.; Francart, T.</p>
        <p class="info">Score: 2.5, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.24301324' target='https://doi.org/10.1101/2024.01.15.24301324'> 10.1101/2024.01.15.24301324</a></p>
        <p class="abstract">One out of three stroke-patients develop language processing impairment known as aphasia. The need for ecological validity of the existing diagnostic tools motivates research on biomarkers, such as stimulus-evoked brain responses. With the aim of enhancing the physiological interpretation of the latter, we used EEG to investigate how functional brain network patterns associated with the neural response to natural speech are affected in persons with post-stroke chronic aphasia.

EEG was recorded from 24 healthy controls and 40 persons with aphasia while they listened to a story. Stimulus-evoked brain responses at all scalp regions were measured as neural envelope tracking in the delta (0.5-4 Hz), theta (4-8 Hz) and low-gamma bands (30-49 Hz) using mutual information. Functional connectivity between neural-tracking signals was measured, and the Network-Based Statistics toolbox was used to: 1) assess the added value of the neural tracking vs EEG time series, 2) test between-group differences and 3) investigate any association with language performance in aphasia. Graph theory was also used to investigate topological alterations in aphasia.

Functional connectivity was higher when assessed from neural tracking compared to EEG time series. Persons with aphasia showed weaker low-gamma-band left-hemispheric connectivity, and graph theory-based results showed a greater network segregation and higher region-specific node strength. Aphasia also exhibited a correlation between delta-band connectivity within the left pre-frontal region and language performance.

We demonstrated the added value of neural tracking when investigating functional brain networks associated with natural speech processing in post-stroke aphasia. Its higher sensitivity to language-related brain circuits favours its use as informative biomarker for the assessment of aphasia.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.13.24301214">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.13.24301214" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.13.24301214">
        <p class="paperTitle">Somatic mutations in Alzheimer-associated tetraploid neurons</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.13.24301214" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.13.24301214" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lopez-Sanchez, N.; Rabano, A.; Frade, J. M.</p>
        <p class="info">Score: 1.4, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.13.24301214' target='https://doi.org/10.1101/2024.01.13.24301214'> 10.1101/2024.01.13.24301214</a></p>
        <p class="abstract">An early pathological process affecting the brain of Alzheimers disease (AD) is the reactivation of the cell cycle in neurons followed by somatic neuronal tetraploidization (NT). NT also increases with age, and ageing has been shown to correlate with the accumulation of DNA somatic mutations. In this study, we have evaluated the presence of somatic mutations including single nucleotide variants (SNVs) and indels in genomic DNA from tetraploid neurons obtained from the parietal cortex of AD patients, compared with diploid neurons from control individuals and AD patients. Here we show that, in contrast to somatic indels, the proportion of somatic SNVs (sSNVs) significantly increases in the exome of tetraploid neurons, having increased levels of T to C (A to G) transitions, a type of mutation that is associated with oxidative stress. This finding correlates with the over-representation of sSNVs in genes involved in oxidative stress response and DNA repair, suggesting that these alterations exacerbate oxidative DNA damage in tetraploid neurons. sSNVs affecting cancer-related (CR) genes showed a greater molecular pathogenicity score compared with those from a random sample of genes. We propose that neuronal tetraploidy is stochastically triggered through a CR mechanism in neurons whose DNA repair genes become mutated in an oxidative stress scenario. This mechanism likely participates in the etiology of AD.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.24301256">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.24301256" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.24301256">
        <p class="paperTitle">Anti-CD20 antibodies in relapsing multiple sclerosis: Protocol for a systematic review and network meta-analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.24301256" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.24301256" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Navarro, C. E.; Cardenas-Robledo, S.; Macea, J.; Ortiz-Cano, N. A.; Florez, I. D.</p>
        <p class="info">Score: 0.8, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.24301256' target='https://doi.org/10.1101/2024.01.12.24301256'> 10.1101/2024.01.12.24301256</a></p>
        <p class="abstract">BackgroundMultiple sclerosis is a chronic disease of the central nervous system characterized by autoimmune demyelination. Various immunomodulatory medications are available for its treatment, among which anti-CD20 monoclonal antibodies have demonstrated superior efficacy compared to other disease-modifying therapies. However, there is a lack of direct comparison between the four available anti-CD20 monoclonal antibodies. Therefore, this study aims to systematically assess the relative efficacy and safety of anti-CD20 monoclonal antibodies for treating relapsing multiple sclerosis.

Materials and MethodsWe will conduct a systematic review of phase IIb and phase III clinical trials of anti-CD20 monoclonal antibodies for the treatment of relapsing multiple sclerosis, following the Preferred Reporting Items for Systematic Reviews and Meta-Analysis guidelines. A comprehensive review of the Cochrane Central Register of Controlled Trials; MEDLINE; Embase; ClinicalTrials; International Clinical Trials Registry Platform; OpenGray; and MedRxiv will be performed. Two independent reviewers will select titles, abstracts, and eligible full texts to execute the data extraction. The risk of bias will be assessed with the Cochrane RoB 2 tool. Data on pre-specified outcomes will be analyzed for the selected articles using a random-effects network meta-analysis with a frequentist framework. Summary statistics, along with 95% confidence intervals will be presented, and the effectiveness and safety of each intervention will be ranked using the surface under the cumulative ranking curve. A missing data and a sensitivity analysis will be conducted. We will use the GRADE approach to assess the certainty of the direct, indirect, and network estimate for all outcomes.

Results and ConclusionThe results will provide evidence of the relative efficacy and safety of anti-CD20 monoclonal antibodies for the treatment of relapsing multiple sclerosis.

Systematic review registrationPROSPERO registration number CRD42023437996</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.16.24301366">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.16.24301366" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.16.24301366">
        <p class="paperTitle">Data leakage in deep learning studies of translational EEG</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.16.24301366" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.16.24301366" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brookshire, G.; Kasper, J.; Blauch, N.; Wu, Y.; Glatt, R.; Merrill, D. A.; Gerrol, S.; Yoder, K. J.; Quirk, C.; Lucero, C.</p>
        <p class="info">Score: 0.5, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.16.24301366' target='https://doi.org/10.1101/2024.01.16.24301366'> 10.1101/2024.01.16.24301366</a></p>
        <p class="abstract">A growing number of studies use deep neural networks (DNNs) to identify diseases from recordings of brain activity. DNN studies of electroencephalography (EEG) typically use cross-validation to test how accurately a model can predict the disease state of held-out test data. In these studies, segments of EEG data are often randomly assigned to the training or test sets. As a consequence, data from individual subjects appears in both training and test data. Could high test-set accuracy reflect leakage from subject-specific representations, rather than patterns that identify a disease? We address this question by testing the performance of DNN classifiers using segment-based holdout (where EEG segments from one subject can appear in both the training and test sets), and comparing this to their performance using subject-based holdout (where individual subjects data appears exclusively in either the training set or the test set). We compare segment-based and subject-based holdout in two EEG datasets: one classifying Alzheimers disease, and the other classifying epileptic seizures. In both datasets, we find that performance on previously-unseen subjects is strongly overestimated when models are trained using segment-based holdout. Next, we survey the literature and find that the majority of translational DNN-EEG studies use segment-based holdout, and therefore overestimate model performance on new subjects. In a hospital or doctors office, clinicians need to diagnose new patients whose data was not used in training the model; segment-based holdout, therefore, does not reflect the real-world performance of a translational DNN model. When evaluating how DNNs could be used for medical diagnosis, models must be tested on subjects whose data was not included in the training set.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
