<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>oncology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>oncology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Long-read RNA sequencing redefines the clear cell renal cell carcinoma transcriptome and reveals novel genes and transcripts associated with disease recurrence and immune evasion
Authors: Lee, J.; Snell, E. A.; Brown, J.; Banks, R. E.; Turner, D. J.; Vasudev, N. S.; Lagos, D.
Score: 13.6, Published: 2023-09-08 DOI: 10.1101/2023.09.08.23295204
BackgroundLong-read direct RNA sequencing (DRS) and PCR cDNA sequencing (PCS) of tumour samples could lead to discovery of novel transcript isoforms, novel genes, and transcriptomic co-dependencies missed by conventional short-read sequencing.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/oncology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="oncology" />
<meta property="og:description" content="Long-read RNA sequencing redefines the clear cell renal cell carcinoma transcriptome and reveals novel genes and transcripts associated with disease recurrence and immune evasion
Authors: Lee, J.; Snell, E. A.; Brown, J.; Banks, R. E.; Turner, D. J.; Vasudev, N. S.; Lagos, D.
Score: 13.6, Published: 2023-09-08 DOI: 10.1101/2023.09.08.23295204
BackgroundLong-read direct RNA sequencing (DRS) and PCR cDNA sequencing (PCS) of tumour samples could lead to discovery of novel transcript isoforms, novel genes, and transcriptomic co-dependencies missed by conventional short-read sequencing." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/oncology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-13T10:39:03+00:00" />
<meta property="article:modified_time" content="2023-09-13T10:39:03+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="oncology"/>
<meta name="twitter:description" content="Long-read RNA sequencing redefines the clear cell renal cell carcinoma transcriptome and reveals novel genes and transcripts associated with disease recurrence and immune evasion
Authors: Lee, J.; Snell, E. A.; Brown, J.; Banks, R. E.; Turner, D. J.; Vasudev, N. S.; Lagos, D.
Score: 13.6, Published: 2023-09-08 DOI: 10.1101/2023.09.08.23295204
BackgroundLong-read direct RNA sequencing (DRS) and PCR cDNA sequencing (PCS) of tumour samples could lead to discovery of novel transcript isoforms, novel genes, and transcriptomic co-dependencies missed by conventional short-read sequencing."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "oncology",
      "item": "https://trxiv.yorks0n.com/posts/oncology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "oncology",
  "name": "oncology",
  "description": "Long-read RNA sequencing redefines the clear cell renal cell carcinoma transcriptome and reveals novel genes and transcripts associated with disease recurrence and immune evasion\nAuthors: Lee, J.; Snell, E. A.; Brown, J.; Banks, R. E.; Turner, D. J.; Vasudev, N. S.; Lagos, D.\nScore: 13.6, Published: 2023-09-08 DOI: 10.1101/2023.09.08.23295204\nBackgroundLong-read direct RNA sequencing (DRS) and PCR cDNA sequencing (PCS) of tumour samples could lead to discovery of novel transcript isoforms, novel genes, and transcriptomic co-dependencies missed by conventional short-read sequencing.",
  "keywords": [
    
  ],
  "articleBody": " Long-read RNA sequencing redefines the clear cell renal cell carcinoma transcriptome and reveals novel genes and transcripts associated with disease recurrence and immune evasion\nAuthors: Lee, J.; Snell, E. A.; Brown, J.; Banks, R. E.; Turner, D. J.; Vasudev, N. S.; Lagos, D.\nScore: 13.6, Published: 2023-09-08 DOI: 10.1101/2023.09.08.23295204\nBackgroundLong-read direct RNA sequencing (DRS) and PCR cDNA sequencing (PCS) of tumour samples could lead to discovery of novel transcript isoforms, novel genes, and transcriptomic co-dependencies missed by conventional short-read sequencing. However, only a handful of reports using DRS or PCS in cancer exist with no direct comparison between the two methods. Clear cell renal cell carcinoma (ccRCC) is the most common form of kidney cancer. Following primary tumour resection approximately 30% of patients experience disease recurrence. Long-read RNA sequencing has not been applied to kidney cancer. Methods12 primary ccRCC archival tumours (discovery cohort), 6 from patients who went on to relapse, were analysed by Oxford Nanopore DRS and PCS. Results were validated in an independent cohort of 20 patients and compared to DRS analysis of RCC4 cells. ResultsDRS and PCS were successfully performed achieving high read length, with PCS achieving higher sequencing depth. Differentially expressed gene sets in patients who went on to relapse were determined with good overlap between DRS and PCS. Deconvolution analysis showed a loss of immune infiltrate in primary tumours of patients who relapse and revealed the CD8+ T cell exhaustion marker TOX as a novel recurrence-associated gene. Notably, novel transcript analysis revealed more than 10,000 uncharacterised candidate novel transcripts detected by both methods and in a ccRCC cell line in vitro. This allowed the definition of the full exonic structure of ccRCC-associated splice variants, including variants of MVK and HPCAL1. Remarkably, this also revealed a novel sPD-L1 transcript encoding for the soluble version of the protein with a longer 3UTR and lower stability in ccRCC cells than the annotated transcript. Levels of sPD-L1 transcripts are unchanged in primary tumours that go on to relapse, whereas membrane PD-L1 shows a trend towards down-regulation. Finally, both methods identified 414 novel genes, also detected in ccRCC cells in vitro, including a novel non-coding gene over-expressed in patients who relapse. ConclusionsPCS and DRS can be used in tumour samples to uncover substantial yet unmapped features underpinning the plasticity and instability of cancer transcriptomes which are linked to disease progression and immune evasion.\nEarly Morning Checkpoint Inhibitor Infusion and Overall Survival of Patients with Metastatic Cancer: an In-depth Chronotherapeutic Study\nAuthors: Catozzi, S.; Assaad, S.; Delrieu, L.; Favier, B.; Dumas, E.; Hamy, A.-S.; Latouche, A.; Crochet, H.; Blay, J.-Y.; Mullaert, J.; Ballesta, A.; Heudel, P.\nScore: 5.3, Published: 2023-09-06 DOI: 10.1101/2023.09.06.23295010\nIntroductionRecent retrospective studies suggest potential large patients benefit through proper timing of immune checkpoint inhibitors (ICI). The association between ICI treatment timing and patient survival, neoplastic response and toxicities was investigated, together with interactions with performance status (PS) and sex. MethodsA cohort of patients with metastatic or locally advanced solid tumors, who received pembrolizumab, nivolumab, atezolizumab, durvalumab, or avelumab, alone or with concomitant chemotherapy, between November 2015 and March 2021, at the Centre Leon Berard (France), was retrospectively studied. Results361 patients were investigated (80% non-small cell lung cancer patients, mean [SD] age: 63 [11] years, 39% of women, 83% PS0-1 at first infusion, 19% received concomitant chemotherapy). ICI were administered from 07:25 to 17:21 and optimal morning/afternoon cut-off was 11:37. Morning infusions were associated with increased OS as compared to afternoon (median 30.3 vs 15.9 months, p=0.0024; HR 1.56 [1.17-2.1], p=0.003). A strong PS-timing interaction was found (PS0-1 patients, HR=1.53 [1.10-2.12], p=0.011; PS2-3 patients, HR=0.50 [0.25-0.97], p=0.042). Morning PS0-1 patients displayed increased OS (median 36.7 vs 21.3 months, p=0.023), partial/complete response rate (58% vs 41%, p=0.027), and grade1-3 toxicities (49% vs 34%, p=0.028). Mortality risk ratio between infusions at worst time-of-day, estimated at 13:36 [12:48-14:23], and in early morning was 5.5-fold ([2-20], p=0.008). Timing differences in toxicities resulted significant only in female patients (women vs men: p\u003c0.001 vs 0.4). ConclusionsEarly morning ICI infusion was associated with increased OS, response, and toxicities in patients with PS0-1 as compared to later infusions within the day. Prospective randomized trials are needed to confirm this retrospective study.\nGenomic Profiling to Contextualize the Results of Intervention for High-Risk Smoldering Myeloma\nAuthors: Kazandjian, D.; Diamond, B.; Papadimitriou, M.; Hill, E.; Sklavenitis-Pistofidis, R.; Ziccheddu, B.; Blaney, P.; Chojnacka, M.; Durante, M.; Maclachlan, K.; Young, R.; Usmani, S.; Davies, F.; Getz, G.; Ghobrial, I.; Korde, N.; Morgan, G.; Maura, F.; Landgren, O.\nScore: 12.0, Published: 2023-08-31 DOI: 10.1101/2023.08.30.23294483\nEarly intervention for High-Risk Smoldering Multiple Myeloma (HR-SMM) achieves deeper and more prolonged responses compared to Newly Diagnosed (ND) MM. It is unclear if beneficial outcomes of interventional studies in HR-SMM are due to treatment of less complex, susceptible disease or inaccuracy in clinical definition of cases entered. Here, to gain greater biologic insight into treatment outcomes, we performed the first whole genome sequencing analysis of treated HR-SMM for 27 patients treated with carfilzomib, lenalidomide, and dexamethasone and lenalidomide maintenance (NCT01572480). Genomic features were pooled with another contemporary HR-SMM interventional study (E-PRISM; NCT02279394) and compared to those of NDMM. We reveal that across interventional cohorts, the genomic landscape of HR-SMM is uniformly simple as compared to NDMM counterparts, with fewer inactivation events of tumor suppressor genes, fewer RAS pathway mutations, lower frequency of MYC disruption, and lower APOBEC contribution. The absence of these genomic events parallels that of indolent precursor conditions with low chance of progression, possibly explaining the overall superior outcomes across these trials. However, there remains a subgroup of patients harboring genomic complexity for whom early intervention with potent triplet therapy fails to sustain response and who experience resistant, progressive disease. Overall, these results suggest that clinical risk scores do not effectively discriminate between genomically indolent and aggressive disease. Furthermore, our study supports the use of genomics to contextualize the advantage of early intervention in SMM and to consider novel approaches for those with the most aggressive precursor states. Key PointsTreated clinical high-risk smoldering multiple myeloma is genomically heterogeneous but is mostly less complex than multiple myeloma counterparts. A small subgroup of high-risk genomic features is associated with disease progression despite early intervention with triplet therapy.\n",
  "wordCount" : "1029",
  "inLanguage": "en",
  "datePublished": "2023-09-13T10:39:03Z",
  "dateModified": "2023-09-13T10:39:03Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/oncology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      oncology
    </h1>
    <div class="post-meta"><span>updated on September 13, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.23295204">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.23295204" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.23295204">
        <p class="paperTitle">Long-read RNA sequencing redefines the clear cell renal cell carcinoma transcriptome and reveals novel genes and transcripts associated with disease recurrence and immune evasion</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.23295204" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.23295204" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, J.; Snell, E. A.; Brown, J.; Banks, R. E.; Turner, D. J.; Vasudev, N. S.; Lagos, D.</p>
        <p class="info">Score: 13.6, Published: 2023-09-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.23295204' target='https://doi.org/10.1101/2023.09.08.23295204'> 10.1101/2023.09.08.23295204</a></p>
        <p class="abstract">BackgroundLong-read direct RNA sequencing (DRS) and PCR cDNA sequencing (PCS) of tumour samples could lead to discovery of novel transcript isoforms, novel genes, and transcriptomic co-dependencies missed by conventional short-read sequencing. However, only a handful of reports using DRS or PCS in cancer exist with no direct comparison between the two methods. Clear cell renal cell carcinoma (ccRCC) is the most common form of kidney cancer. Following primary tumour resection approximately 30% of patients experience disease recurrence. Long-read RNA sequencing has not been applied to kidney cancer.

Methods12 primary ccRCC archival tumours (discovery cohort), 6 from patients who went on to relapse, were analysed by Oxford Nanopore DRS and PCS. Results were validated in an independent cohort of 20 patients and compared to DRS analysis of RCC4 cells.

ResultsDRS and PCS were successfully performed achieving high read length, with PCS achieving higher sequencing depth. Differentially expressed gene sets in patients who went on to relapse were determined with good overlap between DRS and PCS. Deconvolution analysis showed a loss of immune infiltrate in primary tumours of patients who relapse and revealed the CD8&#43; T cell exhaustion marker TOX as a novel recurrence-associated gene. Notably, novel transcript analysis revealed more than 10,000 uncharacterised candidate novel transcripts detected by both methods and in a ccRCC cell line in vitro. This allowed the definition of the full exonic structure of ccRCC-associated splice variants, including variants of MVK and HPCAL1. Remarkably, this also revealed a novel sPD-L1 transcript encoding for the soluble version of the protein with a longer 3UTR and lower stability in ccRCC cells than the annotated transcript. Levels of sPD-L1 transcripts are unchanged in primary tumours that go on to relapse, whereas membrane PD-L1 shows a trend towards down-regulation. Finally, both methods identified 414 novel genes, also detected in ccRCC cells in vitro, including a novel non-coding gene over-expressed in patients who relapse.

ConclusionsPCS and DRS can be used in tumour samples to uncover substantial yet unmapped features underpinning the plasticity and instability of cancer transcriptomes which are linked to disease progression and immune evasion.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.23295010">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.23295010" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.23295010">
        <p class="paperTitle">Early Morning Checkpoint Inhibitor Infusion and Overall Survival of Patients with Metastatic Cancer: an In-depth Chronotherapeutic Study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.23295010" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.23295010" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Catozzi, S.; Assaad, S.; Delrieu, L.; Favier, B.; Dumas, E.; Hamy, A.-S.; Latouche, A.; Crochet, H.; Blay, J.-Y.; Mullaert, J.; Ballesta, A.; Heudel, P.</p>
        <p class="info">Score: 5.3, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.23295010' target='https://doi.org/10.1101/2023.09.06.23295010'> 10.1101/2023.09.06.23295010</a></p>
        <p class="abstract">IntroductionRecent retrospective studies suggest potential large patients benefit through proper timing of immune checkpoint inhibitors (ICI). The association between ICI treatment timing and patient survival, neoplastic response and toxicities was investigated, together with interactions with performance status (PS) and sex.

MethodsA cohort of patients with metastatic or locally advanced solid tumors, who received pembrolizumab, nivolumab, atezolizumab, durvalumab, or avelumab, alone or with concomitant chemotherapy, between November 2015 and March 2021, at the Centre Leon Berard (France), was retrospectively studied.

Results361 patients were investigated (80% non-small cell lung cancer patients, mean [SD] age: 63 [11] years, 39% of women, 83% PS0-1 at first infusion, 19% received concomitant chemotherapy). ICI were administered from 07:25 to 17:21 and optimal morning/afternoon cut-off was 11:37. Morning infusions were associated with increased OS as compared to afternoon (median 30.3 vs 15.9 months, p=0.0024; HR 1.56 [1.17-2.1], p=0.003). A strong PS-timing interaction was found (PS0-1 patients, HR=1.53 [1.10-2.12], p=0.011; PS2-3 patients, HR=0.50 [0.25-0.97], p=0.042). Morning PS0-1 patients displayed increased OS (median 36.7 vs 21.3 months, p=0.023), partial/complete response rate (58% vs 41%, p=0.027), and grade1-3 toxicities (49% vs 34%, p=0.028). Mortality risk ratio between infusions at worst time-of-day, estimated at 13:36 [12:48-14:23], and in early morning was 5.5-fold ([2-20], p=0.008). Timing differences in toxicities resulted significant only in female patients (women vs men: p&lt;0.001 vs 0.4).

ConclusionsEarly morning ICI infusion was associated with increased OS, response, and toxicities in patients with PS0-1 as compared to later infusions within the day. Prospective randomized trials are needed to confirm this retrospective study.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.23294483">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.23294483" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.23294483">
        <p class="paperTitle">Genomic Profiling to Contextualize the Results of Intervention for High-Risk Smoldering Myeloma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.23294483" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.23294483" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kazandjian, D.; Diamond, B.; Papadimitriou, M.; Hill, E.; Sklavenitis-Pistofidis, R.; Ziccheddu, B.; Blaney, P.; Chojnacka, M.; Durante, M.; Maclachlan, K.; Young, R.; Usmani, S.; Davies, F.; Getz, G.; Ghobrial, I.; Korde, N.; Morgan, G.; Maura, F.; Landgren, O.</p>
        <p class="info">Score: 12.0, Published: 2023-08-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.23294483' target='https://doi.org/10.1101/2023.08.30.23294483'> 10.1101/2023.08.30.23294483</a></p>
        <p class="abstract">Early intervention for High-Risk Smoldering Multiple Myeloma (HR-SMM) achieves deeper and more prolonged responses compared to Newly Diagnosed (ND) MM. It is unclear if beneficial outcomes of interventional studies in HR-SMM are due to treatment of less complex, susceptible disease or inaccuracy in clinical definition of cases entered. Here, to gain greater biologic insight into treatment outcomes, we performed the first whole genome sequencing analysis of treated HR-SMM for 27 patients treated with carfilzomib, lenalidomide, and dexamethasone and lenalidomide maintenance (NCT01572480). Genomic features were pooled with another contemporary HR-SMM interventional study (E-PRISM; NCT02279394) and compared to those of NDMM. We reveal that across interventional cohorts, the genomic landscape of HR-SMM is uniformly simple as compared to NDMM counterparts, with fewer inactivation events of tumor suppressor genes, fewer RAS pathway mutations, lower frequency of MYC disruption, and lower APOBEC contribution. The absence of these genomic events parallels that of indolent precursor conditions with low chance of progression, possibly explaining the overall superior outcomes across these trials. However, there remains a subgroup of patients harboring genomic complexity for whom early intervention with potent triplet therapy fails to sustain response and who experience resistant, progressive disease. Overall, these results suggest that clinical risk scores do not effectively discriminate between genomically indolent and aggressive disease. Furthermore, our study supports the use of genomics to contextualize the advantage of early intervention in SMM and to consider novel approaches for those with the most aggressive precursor states.

Key PointsTreated clinical high-risk smoldering multiple myeloma is genomically heterogeneous but is mostly less complex than multiple myeloma counterparts.

A small subgroup of high-risk genomic features is associated with disease progression despite early intervention with triplet therapy.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
