<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>oncology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>oncology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Mapping inherited genetic variation with opposite effects on autoimmune disease and cancer identifies candidate drug targets associated with the anti-tumor immune response
Authors: Chen, J.; Epstein, M. P.; Schildkraut, J. M.; Kar, S. P.
Score: 8.7, Published: 2023-12-28 DOI: 10.1101/2023.12.23.23300491
BackgroundGermline alleles near genes that encode certain immune checkpoints (CTLA4, CD200) are associated with autoimmune/autoinflammatory disease and cancer but in opposite directions. This motivates a systematic search for additional germline alleles which demonstrate this pattern with the aim of identifying potential cancer immunotherapeutic targets using human genetic evidence.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/oncology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="oncology" />
<meta property="og:description" content="Mapping inherited genetic variation with opposite effects on autoimmune disease and cancer identifies candidate drug targets associated with the anti-tumor immune response
Authors: Chen, J.; Epstein, M. P.; Schildkraut, J. M.; Kar, S. P.
Score: 8.7, Published: 2023-12-28 DOI: 10.1101/2023.12.23.23300491
BackgroundGermline alleles near genes that encode certain immune checkpoints (CTLA4, CD200) are associated with autoimmune/autoinflammatory disease and cancer but in opposite directions. This motivates a systematic search for additional germline alleles which demonstrate this pattern with the aim of identifying potential cancer immunotherapeutic targets using human genetic evidence." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/oncology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-07T10:38:59+00:00" />
<meta property="article:modified_time" content="2024-01-07T10:38:59+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="oncology"/>
<meta name="twitter:description" content="Mapping inherited genetic variation with opposite effects on autoimmune disease and cancer identifies candidate drug targets associated with the anti-tumor immune response
Authors: Chen, J.; Epstein, M. P.; Schildkraut, J. M.; Kar, S. P.
Score: 8.7, Published: 2023-12-28 DOI: 10.1101/2023.12.23.23300491
BackgroundGermline alleles near genes that encode certain immune checkpoints (CTLA4, CD200) are associated with autoimmune/autoinflammatory disease and cancer but in opposite directions. This motivates a systematic search for additional germline alleles which demonstrate this pattern with the aim of identifying potential cancer immunotherapeutic targets using human genetic evidence."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "oncology",
      "item": "https://trxiv.yorks0n.com/posts/oncology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "oncology",
  "name": "oncology",
  "description": "Mapping inherited genetic variation with opposite effects on autoimmune disease and cancer identifies candidate drug targets associated with the anti-tumor immune response\nAuthors: Chen, J.; Epstein, M. P.; Schildkraut, J. M.; Kar, S. P.\nScore: 8.7, Published: 2023-12-28 DOI: 10.1101/2023.12.23.23300491\nBackgroundGermline alleles near genes that encode certain immune checkpoints (CTLA4, CD200) are associated with autoimmune/autoinflammatory disease and cancer but in opposite directions. This motivates a systematic search for additional germline alleles which demonstrate this pattern with the aim of identifying potential cancer immunotherapeutic targets using human genetic evidence.",
  "keywords": [
    
  ],
  "articleBody": " Mapping inherited genetic variation with opposite effects on autoimmune disease and cancer identifies candidate drug targets associated with the anti-tumor immune response\nAuthors: Chen, J.; Epstein, M. P.; Schildkraut, J. M.; Kar, S. P.\nScore: 8.7, Published: 2023-12-28 DOI: 10.1101/2023.12.23.23300491\nBackgroundGermline alleles near genes that encode certain immune checkpoints (CTLA4, CD200) are associated with autoimmune/autoinflammatory disease and cancer but in opposite directions. This motivates a systematic search for additional germline alleles which demonstrate this pattern with the aim of identifying potential cancer immunotherapeutic targets using human genetic evidence. MethodsPairwise fixed effect cross-disorder meta-analyses combining genome-wide association studies (GWAS) for breast, prostate, ovarian and endometrial cancers (240,540 cases/317,000 controls) and seven autoimmune/autoinflammatory diseases (112,631 cases/895,386 controls) coupled with in silico follow-up. To ensure detection of alleles with opposite effects on cancer and autoimmune/autoinflammatory disease, the signs on the beta coefficients in the autoimmune/autoinflammatory GWAS were reversed prior to meta-analyses. ResultsMeta-analyses followed by linkage disequilibrium clumping identified 312 unique, independent lead variants with Pmeta\u003c5x10-8 associated with at least one of the cancer types at Pcancer\u003c10-3 and one of the autoimmune/autoinflammatory diseases at Pauto\u003c10-3. At each lead variant, the allele that conferred autoimmune/autoinflammatory disease risk was protective for cancer. Mapping each lead variant to its nearest gene as its putative functional target and focusing on genes with established immunological effects implicated 32 of the nearest genes. Tumor bulk RNA-Seq data highlighted that the tumor expression of 5/32 genes (IRF1, IKZF1, SPI1, SH2B3, LAT) were each strongly correlated (Spearmans {rho}\u003e0.5) with at least one intra-tumor T/myeloid cell infiltration marker (CD4, CD8A, CD11B, CD45) in every one of the cancer types. Tumor single-cell RNA-Seq data from all cancer types showed that the five genes were more likely to be expressed in intra-tumor immune versus malignant cells. The five lead SNPs corresponding to these genes were linked to them via expression quantitative trait locus mechanisms and at least one additional line of functional evidence. Proteins encoded by the genes were predicted to be druggable. ConclusionWe provide population-scale germline genetic and functional genomic evidence to support further evaluation of the proteins encoded by IRF1, IKZF1, SPI1, SH2B3, and LAT as possible targets for cancer immunotherapy.\nInvestigative needle core biopsies for multi-omics in Glioblastoma\nAuthors: Yu, K. K.; Basu, S.; Bacquer, G.; Ahn, R.; Gantchev, J.; Jindal, S.; Regan, M. S.; Abou-Mrad, Z.; Prabhu, M. C.; Williams, M. J.; D'Souza, A. D.; Malinowski, S. W.; Hopland, K.; Elhanati, Y.; Stopka, S. A.; Stortchevoi, A.; He, Z.; Sun, J.; Chen, Y.; Espejo, A.; Chow, K. H.; Yerrum, S.; Kao, P.-L.; Parker Kerrigan, B.; Norberg, L.; Nielsen, D.; Puduvalli, V. K.; Huse, J.; Beroukhim, R.; Kim, Y. S. B.; Goswami, S.; Boire, A.; Frisken, S.; Cima, M. J.; Holdoff, M.; Lucas, C.-H. G.; Bettegowda, C.; Levine, S. S.; Bale, T. A.; Brennan, C.; Reardon, D. A.; Lang, F. F.; Chiocca, E. A.; Ligon, K. L.;\nScore: 1.0, Published: 2023-12-31 DOI: 10.1101/2023.12.29.23300541\nGlioblastoma (GBM) is a primary brain cancer with an abysmal prognosis and few effective therapies. The ability to investigate the tumor microenvironment before and during treatment would greatly enhance both understanding of disease response and progression, as well as the delivery and impact of therapeutics. Stereotactic biopsies are a routine surgical procedure performed primarily for diagnostic histopathologic purposes. The role of investigative biopsies - tissue sampling for the purpose of understanding tumor microenvironmental responses to treatment using integrated multi-modal molecular analyses ( Multi-omics\") has yet to be defined. Secondly, it is unknown whether comparatively small tissue samples from brain biopsies can yield sufficient information with such methods. Here we adapt stereotactic needle core biopsy tissue in two separate patients. In the first patient with recurrent GBM we performed highly resolved multi-omics analysis methods including single cell RNA sequencing, spatial-transcriptomics, metabolomics, proteomics, phosphoproteomics, T-cell clonotype analysis, and MHC Class I immunopeptidomics from biopsy tissue that was obtained from a single procedure. In a second patient we analyzed multi-regional core biopsies to decipher spatial and genomic variance. We also investigated the utility of stereotactic biopsies as a method for generating patient derived xenograft models in a separate patient cohort. Dataset integration across modalities showed good correspondence between spatial modalities, highlighted immune cell associated metabolic pathways and revealed poor correlation between RNA expression and the tumor MHC Class I immunopeptidome. In conclusion, stereotactic needle biopsy cores are of sufficient quality to generate multi-omics data, provide data rich insight into a patients disease process and tumor immune microenvironment and can be of value in evaluating treatment responses. One sentence summaryIntegrative multi-omics analysis of stereotactic needle core biopsies in glioblastoma\nMulti-omics profiling with untargeted proteomics for blood-based early detection of lung cancer\nAuthors: Koh, B.; Liu, M.; Almonte, R.; Ariad, D.; Bundalian, G.; Chan, J.; Choi, J.; Chou, W.-F.; Cuaresma, R.; Hoedt, E.; Hopper, L.; Hu, Y.; Jain, A.; Khaledian, E.; Khin, T.; Kokate, A.; Lee, J.-Y.; Leung, S.; Lin, C.-H.; Marispini, M.; Malekpour, H.; Mora, M.; Mudaliar, N.; Nouri Golmaei, S.; Ramaiah, M.; Ramaswamy, S.; Ranjan, P.; Shu, G.; Spiro, P.; Ta, B.; Vitko, D.; Waiss, J.; Yanagihara, Z.; Zawada, R.; Zeng, J. Y.; Zhang, S.; Yee, J.; Blume, J. E.; Belthangady, C.; Wilcox, B.; Ma, P.\nScore: 0.5, Published: 2024-01-04 DOI: 10.1101/2024.01.03.24300798\nBlood-based approaches to detect early-stage cancer provide an opportunity to improve survival rates for lung cancer, the most lethal cancer world-wide. Multiple approaches for blood-based cancer detection using molecular analytes derived from individual omics (cell-free DNA, RNA transcripts, proteins, metabolites) have been developed and tested, generally showing significantly lower sensitivity for early-stage versus late-stage cancer. We hypothesized that an approach using multiple types of molecular analytes, including broad and untargeted coverage of proteins, could identify biomarkers that more directly reveal changes in gene expression and molecular phenotype in response to carcinogenesis to potentially improve detection of early-stage lung cancer. To that end, we designed and conducted one of the largest multi-omics, observational studies to date, enrolling 2513 case and control subjects. Multi-omics profiling detected 113,671 peptides corresponding to 8385 protein groups, 219,729 RNA transcripts, 71,756 RNA introns, and 1801 metabolites across all subject samples. We then developed a machine learning-based classifier for lung cancer detection comprising 682 of these multi-omics analytes. This multi-omics classifier demonstrated 89%, 80%, and 98-100% sensitivity for all-stage, stage I, and stage III-IV lung cancer, respectively, at 89% specificity in a validation set. The application of a multi-omics platform for discovery of blood-based disease biomarkers, including proteins and complementary molecular analytes, enables the noninvasive detection of early-stage lung cancer with the potential for downstaging at initial diagnosis and the improvement of clinical outcomes.\nImmune checkpoint inhibitor response in sarcomas associates with immune infiltrates and increased expression of transposable elements and viral response pathways\nAuthors: Nacev, B. A.; Bradic, M.; Woo, H. J.; Richards, A. L.; Kelly, C. M.; Dickson, M. A.; Gounder, M. M.; Keohan, M. L.; Chi, P.; Movva, S.; Maki, R.; Slotkin, E. K.; Rosenbaum, E.; Avutu, V.; Chan, J. E.; Banks, L.; Adamson, T.; Singer, S.; Antonescu, C. R.; Tap, W. D.; Donoghue, M. T.; D'Angelo, S. P.\nScore: 0.5, Published: 2024-01-03 DOI: 10.1101/2024.01.02.24300710\nResponse to immune checkpoint inhibition (ICI) in sarcoma is overall low and heterogeneous. Understanding determinants of ICI outcomes may improve efficacy and patient selection. One potential mechanism is epigenetic de-repression of transposable elements (TEs), which stimulates antitumor immunity. Here, we used transcriptomic data to assign immune-hot versus immune-cold status to 67 pre-treatment biopsies of sarcomas from patients treated on ICI trials. Progression-free survival and overall response was superior in the immune-hot group. Expression of TEs and epigenetic regulators significantly predicted immune-hot status in a regression model in which specific TE subfamilies and IKZF1, a chromatin-interacting transcription factor, were significantly contributory. TE and IKZF1 expression positively correlated with tumor immune infiltrates, inflammatory pathways, and clinical outcomes. Key findings were confirmed in a validation cohort (n=190). This work suggests that TE and IKZF1 expression warrant investigation as predictive biomarkers for ICI response and as therapeutic targets in sarcomas.\nMolecular landscapes of oral cancers of unknown etiology\nAuthors: Deneuve, S.; Fervers, B.; Senkin, S.; Bouaoun, L.; Perol, O.; Chavanel, B.; Lu, L.; Coste, I.; Renno, T.; Zavadil, J.; Virard, F.\nScore: 1.0, Published: 2023-12-17 DOI: 10.1101/2023.12.15.23299866\nThe incidence of the mobile tongue cancer in young patients has been rising. This oral cancer (OC) type has no identified risk factors (NIRF), no established molecular markers and is not yet recognized as a distinct clinical entity. To understand this emerging malignancy, we innovatively analyzed the public head and neck cancer multi-omics data. We identified mutational signatures that successfully stratified 307 OC and 109 laryngeal cancer cases according to their clinico-pathological characteristics. The NIRF OCs exhibited significantly increased activities of endogenous clock-like and APOBEC-associated mutagenesis, alongside specific cancer driver gene mutations, distinct methylome patterns and prominent antimicrobial transcriptomic responses. Furthermore, we show that mutational signature SBS16 in OCs reflects the combined effects of alcohol drinking and tobacco smoking. Our study characterizes the unique disease histories and molecular programs of the NIRF OCs revealing that this emerging cancer subtype is likely driven by increased endogenous mutagenesis correlated with responses to microbial insults.\n",
  "wordCount" : "1506",
  "inLanguage": "en",
  "datePublished": "2024-01-07T10:38:59Z",
  "dateModified": "2024-01-07T10:38:59Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/oncology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      oncology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 7, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.23300491">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.23300491" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.23300491">
        <p class="paperTitle">Mapping inherited genetic variation with opposite effects on autoimmune disease and cancer identifies candidate drug targets associated with the anti-tumor immune response</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.23300491" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.23300491" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, J.; Epstein, M. P.; Schildkraut, J. M.; Kar, S. P.</p>
        <p class="info">Score: 8.7, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.23300491' target='https://doi.org/10.1101/2023.12.23.23300491'> 10.1101/2023.12.23.23300491</a></p>
        <p class="abstract">BackgroundGermline alleles near genes that encode certain immune checkpoints (CTLA4, CD200) are associated with autoimmune/autoinflammatory disease and cancer but in opposite directions. This motivates a systematic search for additional germline alleles which demonstrate this pattern with the aim of identifying potential cancer immunotherapeutic targets using human genetic evidence.

MethodsPairwise fixed effect cross-disorder meta-analyses combining genome-wide association studies (GWAS) for breast, prostate, ovarian and endometrial cancers (240,540 cases/317,000 controls) and seven autoimmune/autoinflammatory diseases (112,631 cases/895,386 controls) coupled with in silico follow-up. To ensure detection of alleles with opposite effects on cancer and autoimmune/autoinflammatory disease, the signs on the beta coefficients in the autoimmune/autoinflammatory GWAS were reversed prior to meta-analyses.

ResultsMeta-analyses followed by linkage disequilibrium clumping identified 312 unique, independent lead variants with Pmeta&lt;5x10-8 associated with at least one of the cancer types at Pcancer&lt;10-3 and one of the autoimmune/autoinflammatory diseases at Pauto&lt;10-3. At each lead variant, the allele that conferred autoimmune/autoinflammatory disease risk was protective for cancer. Mapping each lead variant to its nearest gene as its putative functional target and focusing on genes with established immunological effects implicated 32 of the nearest genes. Tumor bulk RNA-Seq data highlighted that the tumor expression of 5/32 genes (IRF1, IKZF1, SPI1, SH2B3, LAT) were each strongly correlated (Spearmans {rho}&gt;0.5) with at least one intra-tumor T/myeloid cell infiltration marker (CD4, CD8A, CD11B, CD45) in every one of the cancer types. Tumor single-cell RNA-Seq data from all cancer types showed that the five genes were more likely to be expressed in intra-tumor immune versus malignant cells. The five lead SNPs corresponding to these genes were linked to them via expression quantitative trait locus mechanisms and at least one additional line of functional evidence. Proteins encoded by the genes were predicted to be druggable.

ConclusionWe provide population-scale germline genetic and functional genomic evidence to support further evaluation of the proteins encoded by IRF1, IKZF1, SPI1, SH2B3, and LAT as possible targets for cancer immunotherapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.23300541">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.23300541" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.23300541">
        <p class="paperTitle">Investigative needle core biopsies for multi-omics in Glioblastoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.23300541" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.23300541" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yu, K. K.; Basu, S.; Bacquer, G.; Ahn, R.; Gantchev, J.; Jindal, S.; Regan, M. S.; Abou-Mrad, Z.; Prabhu, M. C.; Williams, M. J.; D&#39;Souza, A. D.; Malinowski, S. W.; Hopland, K.; Elhanati, Y.; Stopka, S. A.; Stortchevoi, A.; He, Z.; Sun, J.; Chen, Y.; Espejo, A.; Chow, K. H.; Yerrum, S.; Kao, P.-L.; Parker Kerrigan, B.; Norberg, L.; Nielsen, D.; Puduvalli, V. K.; Huse, J.; Beroukhim, R.; Kim, Y. S. B.; Goswami, S.; Boire, A.; Frisken, S.; Cima, M. J.; Holdoff, M.; Lucas, C.-H. G.; Bettegowda, C.; Levine, S. S.; Bale, T. A.; Brennan, C.; Reardon, D. A.; Lang, F. F.; Chiocca, E. A.; Ligon, K. L.;</p>
        <p class="info">Score: 1.0, Published: 2023-12-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.23300541' target='https://doi.org/10.1101/2023.12.29.23300541'> 10.1101/2023.12.29.23300541</a></p>
        <p class="abstract">Glioblastoma (GBM) is a primary brain cancer with an abysmal prognosis and few effective therapies. The ability to investigate the tumor microenvironment before and during treatment would greatly enhance both understanding of disease response and progression, as well as the delivery and impact of therapeutics. Stereotactic biopsies are a routine surgical procedure performed primarily for diagnostic histopathologic purposes. The role of investigative biopsies - tissue sampling for the purpose of understanding tumor microenvironmental responses to treatment using integrated multi-modal molecular analyses ( Multi-omics&#34;) has yet to be defined. Secondly, it is unknown whether comparatively small tissue samples from brain biopsies can yield sufficient information with such methods. Here we adapt stereotactic needle core biopsy tissue in two separate patients. In the first patient with recurrent GBM we performed highly resolved multi-omics analysis methods including single cell RNA sequencing, spatial-transcriptomics, metabolomics, proteomics, phosphoproteomics, T-cell clonotype analysis, and MHC Class I immunopeptidomics from biopsy tissue that was obtained from a single procedure. In a second patient we analyzed multi-regional core biopsies to decipher spatial and genomic variance. We also investigated the utility of stereotactic biopsies as a method for generating patient derived xenograft models in a separate patient cohort. Dataset integration across modalities showed good correspondence between spatial modalities, highlighted immune cell associated metabolic pathways and revealed poor correlation between RNA expression and the tumor MHC Class I immunopeptidome. In conclusion, stereotactic needle biopsy cores are of sufficient quality to generate multi-omics data, provide data rich insight into a patients disease process and tumor immune microenvironment and can be of value in evaluating treatment responses.

One sentence summaryIntegrative multi-omics analysis of stereotactic needle core biopsies in glioblastoma</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.24300798">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.24300798" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.24300798">
        <p class="paperTitle">Multi-omics profiling with untargeted proteomics for blood-based early detection of lung cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.24300798" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.24300798" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Koh, B.; Liu, M.; Almonte, R.; Ariad, D.; Bundalian, G.; Chan, J.; Choi, J.; Chou, W.-F.; Cuaresma, R.; Hoedt, E.; Hopper, L.; Hu, Y.; Jain, A.; Khaledian, E.; Khin, T.; Kokate, A.; Lee, J.-Y.; Leung, S.; Lin, C.-H.; Marispini, M.; Malekpour, H.; Mora, M.; Mudaliar, N.; Nouri Golmaei, S.; Ramaiah, M.; Ramaswamy, S.; Ranjan, P.; Shu, G.; Spiro, P.; Ta, B.; Vitko, D.; Waiss, J.; Yanagihara, Z.; Zawada, R.; Zeng, J. Y.; Zhang, S.; Yee, J.; Blume, J. E.; Belthangady, C.; Wilcox, B.; Ma, P.</p>
        <p class="info">Score: 0.5, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.24300798' target='https://doi.org/10.1101/2024.01.03.24300798'> 10.1101/2024.01.03.24300798</a></p>
        <p class="abstract">Blood-based approaches to detect early-stage cancer provide an opportunity to improve survival rates for lung cancer, the most lethal cancer world-wide. Multiple approaches for blood-based cancer detection using molecular analytes derived from individual  omics (cell-free DNA, RNA transcripts, proteins, metabolites) have been developed and tested, generally showing significantly lower sensitivity for early-stage versus late-stage cancer. We hypothesized that an approach using multiple types of molecular analytes, including broad and untargeted coverage of proteins, could identify biomarkers that more directly reveal changes in gene expression and molecular phenotype in response to carcinogenesis to potentially improve detection of early-stage lung cancer. To that end, we designed and conducted one of the largest multi-omics, observational studies to date, enrolling 2513 case and control subjects. Multi-omics profiling detected 113,671 peptides corresponding to 8385 protein groups, 219,729 RNA transcripts, 71,756 RNA introns, and 1801 metabolites across all subject samples. We then developed a machine learning-based classifier for lung cancer detection comprising 682 of these multi-omics analytes. This multi-omics classifier demonstrated 89%, 80%, and 98-100% sensitivity for all-stage, stage I, and stage III-IV lung cancer, respectively, at 89% specificity in a validation set. The application of a multi-omics platform for discovery of blood-based disease biomarkers, including proteins and complementary molecular analytes, enables the noninvasive detection of early-stage lung cancer with the potential for downstaging at initial diagnosis and the improvement of clinical outcomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.24300710">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.24300710" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.24300710">
        <p class="paperTitle">Immune checkpoint inhibitor response in sarcomas associates with immune infiltrates and increased expression of transposable elements and viral response pathways</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.24300710" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.24300710" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nacev, B. A.; Bradic, M.; Woo, H. J.; Richards, A. L.; Kelly, C. M.; Dickson, M. A.; Gounder, M. M.; Keohan, M. L.; Chi, P.; Movva, S.; Maki, R.; Slotkin, E. K.; Rosenbaum, E.; Avutu, V.; Chan, J. E.; Banks, L.; Adamson, T.; Singer, S.; Antonescu, C. R.; Tap, W. D.; Donoghue, M. T.; D&#39;Angelo, S. P.</p>
        <p class="info">Score: 0.5, Published: 2024-01-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.24300710' target='https://doi.org/10.1101/2024.01.02.24300710'> 10.1101/2024.01.02.24300710</a></p>
        <p class="abstract">Response to immune checkpoint inhibition (ICI) in sarcoma is overall low and heterogeneous. Understanding determinants of ICI outcomes may improve efficacy and patient selection. One potential mechanism is epigenetic de-repression of transposable elements (TEs), which stimulates antitumor immunity. Here, we used transcriptomic data to assign immune-hot versus immune-cold status to 67 pre-treatment biopsies of sarcomas from patients treated on ICI trials. Progression-free survival and overall response was superior in the immune-hot group. Expression of TEs and epigenetic regulators significantly predicted immune-hot status in a regression model in which specific TE subfamilies and IKZF1, a chromatin-interacting transcription factor, were significantly contributory. TE and IKZF1 expression positively correlated with tumor immune infiltrates, inflammatory pathways, and clinical outcomes. Key findings were confirmed in a validation cohort (n=190). This work suggests that TE and IKZF1 expression warrant investigation as predictive biomarkers for ICI response and as therapeutic targets in sarcomas.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.23299866">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.23299866" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.23299866">
        <p class="paperTitle">Molecular landscapes of oral cancers of unknown etiology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.23299866" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.23299866" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Deneuve, S.; Fervers, B.; Senkin, S.; Bouaoun, L.; Perol, O.; Chavanel, B.; Lu, L.; Coste, I.; Renno, T.; Zavadil, J.; Virard, F.</p>
        <p class="info">Score: 1.0, Published: 2023-12-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.23299866' target='https://doi.org/10.1101/2023.12.15.23299866'> 10.1101/2023.12.15.23299866</a></p>
        <p class="abstract">The incidence of the mobile tongue cancer in young patients has been rising. This oral cancer (OC) type has no identified risk factors (NIRF), no established molecular markers and is not yet recognized as a distinct clinical entity.

To understand this emerging malignancy, we innovatively analyzed the public head and neck cancer multi-omics data. We identified mutational signatures that successfully stratified 307 OC and 109 laryngeal cancer cases according to their clinico-pathological characteristics. The NIRF OCs exhibited significantly increased activities of endogenous clock-like and APOBEC-associated mutagenesis, alongside specific cancer driver gene mutations, distinct methylome patterns and prominent antimicrobial transcriptomic responses. Furthermore, we show that mutational signature SBS16 in OCs reflects the combined effects of alcohol drinking and tobacco smoking.

Our study characterizes the unique disease histories and molecular programs of the NIRF OCs revealing that this emerging cancer subtype is likely driven by increased endogenous mutagenesis correlated with responses to microbial insults.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
