<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>oncology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>oncology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="DNA mismatch repair gene variant classification: evaluating the utility of somatic mutations and mismatch repair deficient colonic crypts and endometrial glands
Authors: Walker, R.; Mahmood, K.; Como, J.; Clendenning, M.; Joo, J. E.; Georgeson, P.; Joseland, S.; Preston, S. G.; Pope, B. J.; Austin, R.; Bojadzieva, J.; Campbell, A.; Chan, J. M.; Edwards, E.; Gleeson, M.; Goodwin, A.; Harris, M. T.; Ip, E.; Kirk, J.; Mansour, J.; Marfan, H.; Nichols, C.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/oncology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="oncology" />
<meta property="og:description" content="DNA mismatch repair gene variant classification: evaluating the utility of somatic mutations and mismatch repair deficient colonic crypts and endometrial glands
Authors: Walker, R.; Mahmood, K.; Como, J.; Clendenning, M.; Joo, J. E.; Georgeson, P.; Joseland, S.; Preston, S. G.; Pope, B. J.; Austin, R.; Bojadzieva, J.; Campbell, A.; Chan, J. M.; Edwards, E.; Gleeson, M.; Goodwin, A.; Harris, M. T.; Ip, E.; Kirk, J.; Mansour, J.; Marfan, H.; Nichols, C." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/oncology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-08T10:37:16+00:00" />
<meta property="article:modified_time" content="2023-10-08T10:37:16+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="oncology"/>
<meta name="twitter:description" content="DNA mismatch repair gene variant classification: evaluating the utility of somatic mutations and mismatch repair deficient colonic crypts and endometrial glands
Authors: Walker, R.; Mahmood, K.; Como, J.; Clendenning, M.; Joo, J. E.; Georgeson, P.; Joseland, S.; Preston, S. G.; Pope, B. J.; Austin, R.; Bojadzieva, J.; Campbell, A.; Chan, J. M.; Edwards, E.; Gleeson, M.; Goodwin, A.; Harris, M. T.; Ip, E.; Kirk, J.; Mansour, J.; Marfan, H.; Nichols, C."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "oncology",
      "item": "https://trxiv.yorks0n.com/posts/oncology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "oncology",
  "name": "oncology",
  "description": "DNA mismatch repair gene variant classification: evaluating the utility of somatic mutations and mismatch repair deficient colonic crypts and endometrial glands\nAuthors: Walker, R.; Mahmood, K.; Como, J.; Clendenning, M.; Joo, J. E.; Georgeson, P.; Joseland, S.; Preston, S. G.; Pope, B. J.; Austin, R.; Bojadzieva, J.; Campbell, A.; Chan, J. M.; Edwards, E.; Gleeson, M.; Goodwin, A.; Harris, M. T.; Ip, E.; Kirk, J.; Mansour, J.; Marfan, H.; Nichols, C.",
  "keywords": [
    
  ],
  "articleBody": " DNA mismatch repair gene variant classification: evaluating the utility of somatic mutations and mismatch repair deficient colonic crypts and endometrial glands\nAuthors: Walker, R.; Mahmood, K.; Como, J.; Clendenning, M.; Joo, J. E.; Georgeson, P.; Joseland, S.; Preston, S. G.; Pope, B. J.; Austin, R.; Bojadzieva, J.; Campbell, A.; Chan, J. M.; Edwards, E.; Gleeson, M.; Goodwin, A.; Harris, M. T.; Ip, E.; Kirk, J.; Mansour, J.; Marfan, H.; Nichols, C.; Pachter, N.; Ragunathan, A.; Spigelman, A.; Susman, R.; Christie, M.; Jenkins, M. A.; Pai, R. K.; Rosty, C.; Macrae, F. A.; Winship, I. M.; Buchanan, D. D.\nScore: 4.6, Published: 2023-09-27 DOI: 10.1101/2023.09.26.23295173\nGermline pathogenic variants in the DNA mismatch repair (MMR) genes (Lynch syndrome) predispose to colorectal (CRC) and endometrial (EC) cancer. Lynch syndrome specific tumor features were evaluated for their ability to support the ACMG/InSiGHT framework in classifying variants of uncertain clinical significance (VUS) in the MMR genes. Twenty-eight CRC or EC tumors from 25 VUS carriers (6xMLH1, 9xMSH2, 6xMSH6, 4xPMS2), underwent targeted tumor sequencing for the presence of microsatellite instability/MMR-deficiency (MSI-H/dMMR) status and identification of a somatic MMR mutation (second hit). Immunohistochemical testing for the presence of dMMR crypts/glands in normal tissue was also performed. The ACMG/InSiGHT framework reclassified 7/25 (28%) VUS to likely pathogenic (LP), three (12%) to benign/likely benign, and 15 (60%) VUS remained unchanged. For the seven re-classified LP variants comprising nine tumors, tumor sequencing confirmed MSI-H/dMMR (8/9, 88.9%) and a second hit (7/9, 77.8%). Of these LP reclassified variants where normal tissue was available, the presence of a dMMR crypt/gland was found in 2/4 (50%). Furthermore, a dMMR endometrial gland in a carrier of an MSH2 exon 1-6 duplication provided further support for upgrade of this VUS to LP. Our study confirmed that identifying these Lynch syndrome features can improve MMR variant classification, enabling optimal clinical care. Simple SummaryLynch syndrome is caused by germline pathogenic variants in the DNA mismatch repair (MMR) genes predisposing carriers to colorectal and endometrial cancer. Genetic testing for Lynch syndrome, in the form of multigene panel testing, frequently identifies variants of uncertain clinical significance (VUS). These VUS have limited clinical actionability and create uncertainty for patients and clinicians regarding their risk of cancer. In this study, we tested carriers of germline VUS for features consistent with Lynch syndrome, namely 1) tumor microsatellite instability/MMR-deficiency, 2) the presence of a somatic second hit in the MMR gene harboring the VUS by tumor sequencing and 3) the presence of MMR-deficiency in normal colonic mucosa crypts or normal endometrial glands. Our findings showed that microsatellite instability/MMR-deficiency status and somatic second hits were consistent with MMR variant classifications as determined by the ACMG/InSiGHT framework. In addition to this, the presence of MMR-deficient crypts/glands were consistent with pathogenic variant classification.\nDeep Learning Model for Tumor Type Prediction using Targeted Clinical Genomic Sequencing Data\nAuthors: Darmofal, M.; Suman, S.; Atwal, G.; Chen, J.-F.; Chang, J. C.; Toomey, M.; Vakiani, E.; Varghese, A. M.; Balakrishnan Rema, A.; Syed, A.; Schultz, N.; Berger, M.; Morris, Q.\nScore: 16.8, Published: 2023-09-10 DOI: 10.1101/2023.09.08.23295131\nTumor type guides clinical treatment decisions in cancer, but histology-based diagnosis remains challenging. Genomic alterations are highly diagnostic of tumor type, and tumor type classifiers trained on genomic features have been explored, but the most accurate methods are not clinically feasible, relying on features derived from whole genome sequencing (WGS), or predicting across limited cancer types. We use genomic features from a dataset of 39,787 solid tumors sequenced using a clinical targeted cancer gene panel to develop Genome-Derived-Diagnosis Ensemble (GDD-ENS): a hyperparameter ensemble for classifying tumor type using deep neural networks. GDD-ENS achieves 93% accuracy for high-confidence predictions across 38 cancer types, rivalling performance of WGS-based methods. GDD-ENS can also guide diagnoses on rare type and cancers of unknown primary, and incorporate patient-specific clinical information for improved predictions. Overall, integrating GDD-ENS into prospective clinical sequencing workflows has enabled clinically-relevant tumor type predictions to guide treatment decisions in real time.\n",
  "wordCount" : "655",
  "inLanguage": "en",
  "datePublished": "2023-10-08T10:37:16Z",
  "dateModified": "2023-10-08T10:37:16Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/oncology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      oncology
    </h1>
    <div class="post-meta"><span>updated on October 8, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.23295173">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.23295173" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.23295173">
        <p class="paperTitle">DNA mismatch repair gene variant classification: evaluating the utility of somatic mutations and mismatch repair deficient colonic crypts and endometrial glands</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.23295173" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.23295173" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Walker, R.; Mahmood, K.; Como, J.; Clendenning, M.; Joo, J. E.; Georgeson, P.; Joseland, S.; Preston, S. G.; Pope, B. J.; Austin, R.; Bojadzieva, J.; Campbell, A.; Chan, J. M.; Edwards, E.; Gleeson, M.; Goodwin, A.; Harris, M. T.; Ip, E.; Kirk, J.; Mansour, J.; Marfan, H.; Nichols, C.; Pachter, N.; Ragunathan, A.; Spigelman, A.; Susman, R.; Christie, M.; Jenkins, M. A.; Pai, R. K.; Rosty, C.; Macrae, F. A.; Winship, I. M.; Buchanan, D. D.</p>
        <p class="info">Score: 4.6, Published: 2023-09-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.23295173' target='https://doi.org/10.1101/2023.09.26.23295173'> 10.1101/2023.09.26.23295173</a></p>
        <p class="abstract">Germline pathogenic variants in the DNA mismatch repair (MMR) genes (Lynch syndrome) predispose to colorectal (CRC) and endometrial (EC) cancer. Lynch syndrome specific tumor features were evaluated for their ability to support the ACMG/InSiGHT framework in classifying variants of uncertain clinical significance (VUS) in the MMR genes. Twenty-eight CRC or EC tumors from 25 VUS carriers (6xMLH1, 9xMSH2, 6xMSH6, 4xPMS2), underwent targeted tumor sequencing for the presence of microsatellite instability/MMR-deficiency (MSI-H/dMMR) status and identification of a somatic MMR mutation (second hit). Immunohistochemical testing for the presence of dMMR crypts/glands in normal tissue was also performed. The ACMG/InSiGHT framework reclassified 7/25 (28%) VUS to likely pathogenic (LP), three (12%) to benign/likely benign, and 15 (60%) VUS remained unchanged. For the seven re-classified LP variants comprising nine tumors, tumor sequencing confirmed MSI-H/dMMR (8/9, 88.9%) and a second hit (7/9, 77.8%). Of these LP reclassified variants where normal tissue was available, the presence of a dMMR crypt/gland was found in 2/4 (50%). Furthermore, a dMMR endometrial gland in a carrier of an MSH2 exon 1-6 duplication provided further support for upgrade of this VUS to LP. Our study confirmed that identifying these Lynch syndrome features can improve MMR variant classification, enabling optimal clinical care.

Simple SummaryLynch syndrome is caused by germline pathogenic variants in the DNA mismatch repair (MMR) genes predisposing carriers to colorectal and endometrial cancer. Genetic testing for Lynch syndrome, in the form of multigene panel testing, frequently identifies variants of uncertain clinical significance (VUS). These VUS have limited clinical actionability and create uncertainty for patients and clinicians regarding their risk of cancer. In this study, we tested carriers of germline VUS for features consistent with Lynch syndrome, namely 1) tumor microsatellite instability/MMR-deficiency, 2) the presence of a somatic second hit in the MMR gene harboring the VUS by tumor sequencing and 3) the presence of MMR-deficiency in normal colonic mucosa crypts or normal endometrial glands. Our findings showed that microsatellite instability/MMR-deficiency status and somatic second hits were consistent with MMR variant classifications as determined by the ACMG/InSiGHT framework. In addition to this, the presence of MMR-deficient crypts/glands were consistent with pathogenic variant classification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.23295131">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.23295131" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.23295131">
        <p class="paperTitle">Deep Learning Model for Tumor Type Prediction using Targeted Clinical Genomic Sequencing Data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.23295131" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.23295131" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Darmofal, M.; Suman, S.; Atwal, G.; Chen, J.-F.; Chang, J. C.; Toomey, M.; Vakiani, E.; Varghese, A. M.; Balakrishnan Rema, A.; Syed, A.; Schultz, N.; Berger, M.; Morris, Q.</p>
        <p class="info">Score: 16.8, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.23295131' target='https://doi.org/10.1101/2023.09.08.23295131'> 10.1101/2023.09.08.23295131</a></p>
        <p class="abstract">Tumor type guides clinical treatment decisions in cancer, but histology-based diagnosis remains challenging. Genomic alterations are highly diagnostic of tumor type, and tumor type classifiers trained on genomic features have been explored, but the most accurate methods are not clinically feasible, relying on features derived from whole genome sequencing (WGS), or predicting across limited cancer types. We use genomic features from a dataset of 39,787 solid tumors sequenced using a clinical targeted cancer gene panel to develop Genome-Derived-Diagnosis Ensemble (GDD-ENS): a hyperparameter ensemble for classifying tumor type using deep neural networks. GDD-ENS achieves 93% accuracy for high-confidence predictions across 38 cancer types, rivalling performance of WGS-based methods. GDD-ENS can also guide diagnoses on rare type and cancers of unknown primary, and incorporate patient-specific clinical information for improved predictions. Overall, integrating GDD-ENS into prospective clinical sequencing workflows has enabled clinically-relevant tumor type predictions to guide treatment decisions in real time.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
