<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>oncology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>oncology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="An updated PREDICT breast cancer prognostic model including the benefits and harms of radiotherapy
Authors: Grootes, I.; Wishart, G. C.; Pharoah, P. D. P.
Score: 6.8, Published: 2023-07-23 DOI: 10.1101/2023.07.18.23292777
BackgroundPredict Breast (www.breast.predict.nhs.uk) is an online prognostication and treatment benefit tool for early invasive breast cancer. However, the most recent version of PREDICT Breast (v2.2) was based on data for breast cancer cases diagnosed from 1999 to 2003 and did not incorporate the benefits of radiotherapy or the harms associated with theray.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/oncology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="oncology" />
<meta property="og:description" content="An updated PREDICT breast cancer prognostic model including the benefits and harms of radiotherapy
Authors: Grootes, I.; Wishart, G. C.; Pharoah, P. D. P.
Score: 6.8, Published: 2023-07-23 DOI: 10.1101/2023.07.18.23292777
BackgroundPredict Breast (www.breast.predict.nhs.uk) is an online prognostication and treatment benefit tool for early invasive breast cancer. However, the most recent version of PREDICT Breast (v2.2) was based on data for breast cancer cases diagnosed from 1999 to 2003 and did not incorporate the benefits of radiotherapy or the harms associated with theray." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/oncology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-26T10:37:30+00:00" />
<meta property="article:modified_time" content="2023-07-26T10:37:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="oncology"/>
<meta name="twitter:description" content="An updated PREDICT breast cancer prognostic model including the benefits and harms of radiotherapy
Authors: Grootes, I.; Wishart, G. C.; Pharoah, P. D. P.
Score: 6.8, Published: 2023-07-23 DOI: 10.1101/2023.07.18.23292777
BackgroundPredict Breast (www.breast.predict.nhs.uk) is an online prognostication and treatment benefit tool for early invasive breast cancer. However, the most recent version of PREDICT Breast (v2.2) was based on data for breast cancer cases diagnosed from 1999 to 2003 and did not incorporate the benefits of radiotherapy or the harms associated with theray."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "oncology",
      "item": "https://trxiv.yorks0n.com/posts/oncology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "oncology",
  "name": "oncology",
  "description": "An updated PREDICT breast cancer prognostic model including the benefits and harms of radiotherapy\nAuthors: Grootes, I.; Wishart, G. C.; Pharoah, P. D. P.\nScore: 6.8, Published: 2023-07-23 DOI: 10.1101/2023.07.18.23292777\nBackgroundPredict Breast (www.breast.predict.nhs.uk) is an online prognostication and treatment benefit tool for early invasive breast cancer. However, the most recent version of PREDICT Breast (v2.2) was based on data for breast cancer cases diagnosed from 1999 to 2003 and did not incorporate the benefits of radiotherapy or the harms associated with theray.",
  "keywords": [
    
  ],
  "articleBody": " An updated PREDICT breast cancer prognostic model including the benefits and harms of radiotherapy\nAuthors: Grootes, I.; Wishart, G. C.; Pharoah, P. D. P.\nScore: 6.8, Published: 2023-07-23 DOI: 10.1101/2023.07.18.23292777\nBackgroundPredict Breast (www.breast.predict.nhs.uk) is an online prognostication and treatment benefit tool for early invasive breast cancer. However, the most recent version of PREDICT Breast (v2.2) was based on data for breast cancer cases diagnosed from 1999 to 2003 and did not incorporate the benefits of radiotherapy or the harms associated with theray. Since then, there has been a substantial improvement in the outcomes for breast cancer cases. The aim of this study was to update PREDICT Breast to ensure that the underlying model is appropriate for contemporary patients. MethodsData from 4,644 ER-negative and 30,830 ER-positive breast cancer cases diagnosed from 2000 to 2017 in the region served by the Eastern Cancer Registry were used for model development. Multivariable fractional polynomials in a Cox proportional hazards framework were used to estimate the prognostic effects of year of diagnosis, age at diagnosis, tumour size, tumour grade and number of positive nodes and to compute the baseline hazard functions. Separate models were developed for ER-positive and ER-negative disease. Data on 32,408 breast cancer patients from the West Midlands Cancer Registry and from 100,551 breast cancer cases from the other English Cancer Registries combined were used to determine the discriminative power, calibration, and reclassification of the new version of PREDICT Breast (v3.0). ResultsThe new model (v3.0) was well-calibrated; predicted numbers of 5-, 10- and 15-year breast cancer deaths were within 10 per cent of the observed number in both model development and model validation data sets. In contrast, PREDICT Breast v2.2 was found to substantially over-predict the number of deaths. Discrimination was also good: The AUC for 15-year breast cancer survival was 0. 824 in the model development data, 0.809 in the West Midlands data set and 0.846 in the data set for the other registries. There figures were slightly better than those for PREDICT Breast v2.2 ConclusionIncorporating the prognostic effect of year of diagnosis, updating the prognostic effects of all risk factors and amending the baseline hazard functions have led to an improvement of model performance of PREDICT Breast. The new model will be implemented in the online tool which should lead to more accurate absolute treatment benefit predictions for individual patients.\nAssociation of Genetic Ancestry with Molecular Tumor Profiles in Colorectal Cancer\nAuthors: Rhead, B.; Hein, D.; Pouliot, Y.; Guinney, J.; De La Vega, F. M.; Sanford, N.\nScore: 2.5, Published: 2023-07-12 DOI: 10.1101/2023.07.12.23292571\nPurposePrior research in the molecular correlates of disparities in incidence and outcomes of colorectal cancer (CRC) by race and ethnicity have typically used self-reported or observed categories, which can be missing or inaccurate. Furthermore, race and ethnicity do not always capture genetic similarity well, particularly in admixed populations. To overcome these limitations, we examined associations of CRC tumor molecular profiles with genetic ancestry. Experimental DesignSequencing was performed with the Tempus xT NGS 648-gene panel and whole exome capture RNA-Seq for 8,454 CRC patients. Genetic ancestry proportions were estimated for five continental groups, Africa (AFR), Americas (AMR), East Asia (EAS), Europe (EUR), and South Asia (SAS), using ancestry informative markers. We assessed association of genetic ancestry proportions and genetic ancestry-imputed race and ethnicity categories with somatic mutations in relevant CRC genes and in expression profiles, including consensus molecular subtypes (CMS). ResultsIncreased AFR ancestry was associated with higher odds of somatic mutations in APC, KRAS and PIK3CA and lower odds of BRAF mutations. Additionally, increased EAS ancestry was associated with lower odds of mutations in KRAS, EUR with higher odds in BRAF, and the Hispanic/Latino category with lower odds in BRAF. Greater AFR ancestry and the non-Hispanic Black category were associated with higher than expected CMS3, while patients in the Hispanic/Latino category had higher indeterminate CMS. ConclusionsUse of genetic ancestry enables identification of molecular differences in CRC tumor mutation frequencies and gene expression that may underlie observed differences by race and ethnicity, and suggests that subtype classifications such as CMS may benefit from more diversity in representation. Translational RelevanceThis study demonstrates differences in somatic mutation patterns and gene expression in CRC by genetic ancestry that have not previously been reported in studies using race and ethnicity categories. Increased AFR genetic ancestry was associated with higher odds of APC, KRAS, and PIK3CA mutations and CMS3 tumors, and lower odds of BRAF mutations. Increased EAS genetic ancestry correlated with lower odds of mutations in KRAS. Furthermore, the increased odds of indeterminate CMS tumors in the imputed Hispanic/Latino category suggests that more diverse representation could reduce disparities in the applicability of disease subtype models. Our findings demonstrate the importance of using genetic ancestry in addition to race and ethnicity in CRC, and the need to enroll diverse patients into genomic research, clinical trials, and tumor registries.\nValidation of Automated Visual Evaluation (AVE) on Smartphone Images for Cervical Cancer Screening in a Prospective Study in Zambia\nAuthors: Hu, L.; Mwanahamuntu, M. H.; Sahasrabuddhe, V. V.; Barrett, C.; Horning, M. P.; Shah, I.; Laverriere, Z.; Banik, D.; Ji, Y.; Shibemba, A. L.; Chisele, S.; Munalula, M. K.; Kaunga, F.; Musonda, F.; Musonda, F.; Malyangu, E.; Hariharan, K. M.; Parham, G. P.\nScore: 1.1, Published: 2023-07-23 DOI: 10.1101/2023.07.19.23292888\nI.ObjectivesVisual inspection with acetic acid (VIA) is a low-cost approach for cervical cancer screening in most low- and middle-income countries (LMICs) but lacks objectivity and requires sustained training and quality assurance. We developed, trained, and validated an artificial-intelligence-based \"Automated Visual Evaluation\" (AVE) tool that can be adapted to run on smartphones to assess smartphone-captured images of the cervix and identify precancerous lesions, helping augment performance of VIA. DesignProspective study. SettinEight public health facilities in Zambia. Participants8,204 women aged 25-55. InterventionsCervical images captured on commonly used low-cost smartphone models were matched with key clinical information including human immunodeficiency virus (HIV) and human papillomavirus (HPV) status, plus histopathology analysis (where applicable), to develop and train an AVE algorithm and evaluate its performance for use as a primary screen and triage test for women who are HPV positive. Main outcome measuresArea under the receiver operating curve (AUC); sensitivity; specificity. ResultsAs a general population screening for cervical precancerous lesions, AVE identified cases of cervical precancerous and cancerous (CIN2+) lesions with high performance (AUC = 0.91, 95% confidence interval [CI] = 0.89 to 0.93), which translates to a sensitivity of 85% (95% CI = 81% to 90%) and specificity of 86% (95% CI = 84% to 88%) based on maximizing the Youdens index. This represents a considerable improvement over VIA, which a meta-analysis by the World Health Organization (WHO) estimates to have sensitivity of 66% and specificity of 87%. For women living with HIV, the AUC of AVE was 0.91 (95% CI = 0.88 to 0.93), and among those testing positive for high-risk HPV types, the AUC was 0.87 (95% CI = 0.83 to 0.91). ConclusionsThese results demonstrate the feasibility of utilizing AVE on images captured using a commonly available smartphone and support our transition to clinical evaluation of AVEs sensitivity, specificity, feasibility, and acceptability across a broader range of settings.\n",
  "wordCount" : "1178",
  "inLanguage": "en",
  "datePublished": "2023-07-26T10:37:30Z",
  "dateModified": "2023-07-26T10:37:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/oncology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      oncology
    </h1>
    <div class="post-meta"><span>updated on July 26, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.23292777">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.23292777" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.23292777">
        <p class="paperTitle">An updated PREDICT breast cancer prognostic model including the benefits and harms of radiotherapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.23292777" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.23292777" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Grootes, I.; Wishart, G. C.; Pharoah, P. D. P.</p>
        <p class="info">Score: 6.8, Published: 2023-07-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.23292777' target='https://doi.org/10.1101/2023.07.18.23292777'> 10.1101/2023.07.18.23292777</a></p>
        <p class="abstract">BackgroundPredict Breast (www.breast.predict.nhs.uk) is an online prognostication and treatment benefit tool for early invasive breast cancer. However, the most recent version of PREDICT Breast (v2.2) was based on data for breast cancer cases diagnosed from 1999 to 2003 and did not incorporate the benefits of radiotherapy or the harms associated with theray. Since then, there has been a substantial improvement in the outcomes for breast cancer cases. The aim of this study was to update PREDICT Breast to ensure that the underlying model is appropriate for contemporary patients.

MethodsData from 4,644 ER-negative and 30,830 ER-positive breast cancer cases diagnosed from 2000 to 2017 in the region served by the Eastern Cancer Registry were used for model development. Multivariable fractional polynomials in a Cox proportional hazards framework were used to estimate the prognostic effects of year of diagnosis, age at diagnosis, tumour size, tumour grade and number of positive nodes and to compute the baseline hazard functions. Separate models were developed for ER-positive and ER-negative disease. Data on 32,408 breast cancer patients from the West Midlands Cancer Registry and from 100,551 breast cancer cases from the other English Cancer Registries combined were used to determine the discriminative power, calibration, and reclassification of the new version of PREDICT Breast (v3.0).

ResultsThe new model (v3.0) was well-calibrated; predicted numbers of 5-, 10- and 15-year breast cancer deaths were within 10 per cent of the observed number in both model development and model validation data sets. In contrast, PREDICT Breast v2.2 was found to substantially over-predict the number of deaths. Discrimination was also good: The AUC for 15-year breast cancer survival was 0. 824 in the model development data, 0.809 in the West Midlands data set and 0.846 in the data set for the other registries. There figures were slightly better than those for PREDICT Breast v2.2

ConclusionIncorporating the prognostic effect of year of diagnosis, updating the prognostic effects of all risk factors and amending the baseline hazard functions have led to an improvement of model performance of PREDICT Breast. The new model will be implemented in the online tool which should lead to more accurate absolute treatment benefit predictions for individual patients.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.12.23292571">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.12.23292571" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.12.23292571">
        <p class="paperTitle">Association of Genetic Ancestry with Molecular Tumor Profiles in Colorectal Cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.12.23292571" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.12.23292571" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rhead, B.; Hein, D.; Pouliot, Y.; Guinney, J.; De La Vega, F. M.; Sanford, N.</p>
        <p class="info">Score: 2.5, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.12.23292571' target='https://doi.org/10.1101/2023.07.12.23292571'> 10.1101/2023.07.12.23292571</a></p>
        <p class="abstract">PurposePrior research in the molecular correlates of disparities in incidence and outcomes of colorectal cancer (CRC) by race and ethnicity have typically used self-reported or observed categories, which can be missing or inaccurate. Furthermore, race and ethnicity do not always capture genetic similarity well, particularly in admixed populations. To overcome these limitations, we examined associations of CRC tumor molecular profiles with genetic ancestry.

Experimental DesignSequencing was performed with the Tempus xT NGS 648-gene panel and whole exome capture RNA-Seq for 8,454 CRC patients. Genetic ancestry proportions were estimated for five continental groups, Africa (AFR), Americas (AMR), East Asia (EAS), Europe (EUR), and South Asia (SAS), using ancestry informative markers. We assessed association of genetic ancestry proportions and genetic ancestry-imputed race and ethnicity categories with somatic mutations in relevant CRC genes and in expression profiles, including consensus molecular subtypes (CMS).

ResultsIncreased AFR ancestry was associated with higher odds of somatic mutations in APC, KRAS and PIK3CA and lower odds of BRAF mutations. Additionally, increased EAS ancestry was associated with lower odds of mutations in KRAS, EUR with higher odds in BRAF, and the Hispanic/Latino category with lower odds in BRAF. Greater AFR ancestry and the non-Hispanic Black category were associated with higher than expected CMS3, while patients in the Hispanic/Latino category had higher indeterminate CMS.

ConclusionsUse of genetic ancestry enables identification of molecular differences in CRC tumor mutation frequencies and gene expression that may underlie observed differences by race and ethnicity, and suggests that subtype classifications such as CMS may benefit from more diversity in representation.

Translational RelevanceThis study demonstrates differences in somatic mutation patterns and gene expression in CRC by genetic ancestry that have not previously been reported in studies using race and ethnicity categories. Increased AFR genetic ancestry was associated with higher odds of APC, KRAS, and PIK3CA mutations and CMS3 tumors, and lower odds of BRAF mutations. Increased EAS genetic ancestry correlated with lower odds of mutations in KRAS. Furthermore, the increased odds of indeterminate CMS tumors in the imputed Hispanic/Latino category suggests that more diverse representation could reduce disparities in the applicability of disease subtype models. Our findings demonstrate the importance of using genetic ancestry in addition to race and ethnicity in CRC, and the need to enroll diverse patients into genomic research, clinical trials, and tumor registries.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.23292888">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.23292888" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.23292888">
        <p class="paperTitle">Validation of Automated Visual Evaluation (AVE) on Smartphone Images for Cervical Cancer Screening in a Prospective Study in Zambia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.23292888" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.23292888" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hu, L.; Mwanahamuntu, M. H.; Sahasrabuddhe, V. V.; Barrett, C.; Horning, M. P.; Shah, I.; Laverriere, Z.; Banik, D.; Ji, Y.; Shibemba, A. L.; Chisele, S.; Munalula, M. K.; Kaunga, F.; Musonda, F.; Musonda, F.; Malyangu, E.; Hariharan, K. M.; Parham, G. P.</p>
        <p class="info">Score: 1.1, Published: 2023-07-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.23292888' target='https://doi.org/10.1101/2023.07.19.23292888'> 10.1101/2023.07.19.23292888</a></p>
        <p class="abstract">I.ObjectivesVisual inspection with acetic acid (VIA) is a low-cost approach for cervical cancer screening in most low- and middle-income countries (LMICs) but lacks objectivity and requires sustained training and quality assurance. We developed, trained, and validated an artificial-intelligence-based &#34;Automated Visual Evaluation&#34; (AVE) tool that can be adapted to run on smartphones to assess smartphone-captured images of the cervix and identify precancerous lesions, helping augment performance of VIA.

DesignProspective study.

SettinEight public health facilities in Zambia.

Participants8,204 women aged 25-55.

InterventionsCervical images captured on commonly used low-cost smartphone models were matched with key clinical information including human immunodeficiency virus (HIV) and human papillomavirus (HPV) status, plus histopathology analysis (where applicable), to develop and train an AVE algorithm and evaluate its performance for use as a primary screen and triage test for women who are HPV positive.

Main outcome measuresArea under the receiver operating curve (AUC); sensitivity; specificity.

ResultsAs a general population screening for cervical precancerous lesions, AVE identified cases of cervical precancerous and cancerous (CIN2&#43;) lesions with high performance (AUC = 0.91, 95% confidence interval [CI] = 0.89 to 0.93), which translates to a sensitivity of 85% (95% CI = 81% to 90%) and specificity of 86% (95% CI = 84% to 88%) based on maximizing the Youdens index. This represents a considerable improvement over VIA, which a meta-analysis by the World Health Organization (WHO) estimates to have sensitivity of 66% and specificity of 87%. For women living with HIV, the AUC of AVE was 0.91 (95% CI = 0.88 to 0.93), and among those testing positive for high-risk HPV types, the AUC was 0.87 (95% CI = 0.83 to 0.91).

ConclusionsThese results demonstrate the feasibility of utilizing AVE on images captured using a commonly available smartphone and support our transition to clinical evaluation of AVEs sensitivity, specificity, feasibility, and acceptability across a broader range of settings.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
