<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>oncology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>oncology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Deep Learning Model for Tumor Type Prediction using Targeted Clinical Genomic Sequencing Data
Authors: Darmofal, M.; Suman, S.; Atwal, G.; Chen, J.-F.; Chang, J. C.; Toomey, M.; Vakiani, E.; Varghese, A. M.; Balakrishnan Rema, A.; Syed, A.; Schultz, N.; Berger, M.; Morris, Q.
Score: 16.1, Published: 2023-09-10 DOI: 10.1101/2023.09.08.23295131
Tumor type guides clinical treatment decisions in cancer, but histology-based diagnosis remains challenging. Genomic alterations are highly diagnostic of tumor type, and tumor type classifiers trained on genomic features have been explored, but the most accurate methods are not clinically feasible, relying on features derived from whole genome sequencing (WGS), or predicting across limited cancer types.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/oncology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="oncology" />
<meta property="og:description" content="Deep Learning Model for Tumor Type Prediction using Targeted Clinical Genomic Sequencing Data
Authors: Darmofal, M.; Suman, S.; Atwal, G.; Chen, J.-F.; Chang, J. C.; Toomey, M.; Vakiani, E.; Varghese, A. M.; Balakrishnan Rema, A.; Syed, A.; Schultz, N.; Berger, M.; Morris, Q.
Score: 16.1, Published: 2023-09-10 DOI: 10.1101/2023.09.08.23295131
Tumor type guides clinical treatment decisions in cancer, but histology-based diagnosis remains challenging. Genomic alterations are highly diagnostic of tumor type, and tumor type classifiers trained on genomic features have been explored, but the most accurate methods are not clinically feasible, relying on features derived from whole genome sequencing (WGS), or predicting across limited cancer types." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/oncology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-17T10:36:50+00:00" />
<meta property="article:modified_time" content="2023-09-17T10:36:50+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="oncology"/>
<meta name="twitter:description" content="Deep Learning Model for Tumor Type Prediction using Targeted Clinical Genomic Sequencing Data
Authors: Darmofal, M.; Suman, S.; Atwal, G.; Chen, J.-F.; Chang, J. C.; Toomey, M.; Vakiani, E.; Varghese, A. M.; Balakrishnan Rema, A.; Syed, A.; Schultz, N.; Berger, M.; Morris, Q.
Score: 16.1, Published: 2023-09-10 DOI: 10.1101/2023.09.08.23295131
Tumor type guides clinical treatment decisions in cancer, but histology-based diagnosis remains challenging. Genomic alterations are highly diagnostic of tumor type, and tumor type classifiers trained on genomic features have been explored, but the most accurate methods are not clinically feasible, relying on features derived from whole genome sequencing (WGS), or predicting across limited cancer types."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "oncology",
      "item": "https://trxiv.yorks0n.com/posts/oncology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "oncology",
  "name": "oncology",
  "description": "Deep Learning Model for Tumor Type Prediction using Targeted Clinical Genomic Sequencing Data\nAuthors: Darmofal, M.; Suman, S.; Atwal, G.; Chen, J.-F.; Chang, J. C.; Toomey, M.; Vakiani, E.; Varghese, A. M.; Balakrishnan Rema, A.; Syed, A.; Schultz, N.; Berger, M.; Morris, Q.\nScore: 16.1, Published: 2023-09-10 DOI: 10.1101/2023.09.08.23295131\nTumor type guides clinical treatment decisions in cancer, but histology-based diagnosis remains challenging. Genomic alterations are highly diagnostic of tumor type, and tumor type classifiers trained on genomic features have been explored, but the most accurate methods are not clinically feasible, relying on features derived from whole genome sequencing (WGS), or predicting across limited cancer types.",
  "keywords": [
    
  ],
  "articleBody": " Deep Learning Model for Tumor Type Prediction using Targeted Clinical Genomic Sequencing Data\nAuthors: Darmofal, M.; Suman, S.; Atwal, G.; Chen, J.-F.; Chang, J. C.; Toomey, M.; Vakiani, E.; Varghese, A. M.; Balakrishnan Rema, A.; Syed, A.; Schultz, N.; Berger, M.; Morris, Q.\nScore: 16.1, Published: 2023-09-10 DOI: 10.1101/2023.09.08.23295131\nTumor type guides clinical treatment decisions in cancer, but histology-based diagnosis remains challenging. Genomic alterations are highly diagnostic of tumor type, and tumor type classifiers trained on genomic features have been explored, but the most accurate methods are not clinically feasible, relying on features derived from whole genome sequencing (WGS), or predicting across limited cancer types. We use genomic features from a dataset of 39,787 solid tumors sequenced using a clinical targeted cancer gene panel to develop Genome-Derived-Diagnosis Ensemble (GDD-ENS): a hyperparameter ensemble for classifying tumor type using deep neural networks. GDD-ENS achieves 93% accuracy for high-confidence predictions across 38 cancer types, rivalling performance of WGS-based methods. GDD-ENS can also guide diagnoses on rare type and cancers of unknown primary, and incorporate patient-specific clinical information for improved predictions. Overall, integrating GDD-ENS into prospective clinical sequencing workflows has enabled clinically-relevant tumor type predictions to guide treatment decisions in real time.\nLong-read RNA sequencing redefines the clear cell renal cell carcinoma transcriptome and reveals novel genes and transcripts associated with disease recurrence and immune evasion\nAuthors: Lee, J.; Snell, E. A.; Brown, J.; Banks, R. E.; Turner, D. J.; Vasudev, N. S.; Lagos, D.\nScore: 13.6, Published: 2023-09-08 DOI: 10.1101/2023.09.08.23295204\nBackgroundLong-read direct RNA sequencing (DRS) and PCR cDNA sequencing (PCS) of tumour samples could lead to discovery of novel transcript isoforms, novel genes, and transcriptomic co-dependencies missed by conventional short-read sequencing. However, only a handful of reports using DRS or PCS in cancer exist with no direct comparison between the two methods. Clear cell renal cell carcinoma (ccRCC) is the most common form of kidney cancer. Following primary tumour resection approximately 30% of patients experience disease recurrence. Long-read RNA sequencing has not been applied to kidney cancer. Methods12 primary ccRCC archival tumours (discovery cohort), 6 from patients who went on to relapse, were analysed by Oxford Nanopore DRS and PCS. Results were validated in an independent cohort of 20 patients and compared to DRS analysis of RCC4 cells. ResultsDRS and PCS were successfully performed achieving high read length, with PCS achieving higher sequencing depth. Differentially expressed gene sets in patients who went on to relapse were determined with good overlap between DRS and PCS. Deconvolution analysis showed a loss of immune infiltrate in primary tumours of patients who relapse and revealed the CD8+ T cell exhaustion marker TOX as a novel recurrence-associated gene. Notably, novel transcript analysis revealed more than 10,000 uncharacterised candidate novel transcripts detected by both methods and in a ccRCC cell line in vitro. This allowed the definition of the full exonic structure of ccRCC-associated splice variants, including variants of MVK and HPCAL1. Remarkably, this also revealed a novel sPD-L1 transcript encoding for the soluble version of the protein with a longer 3UTR and lower stability in ccRCC cells than the annotated transcript. Levels of sPD-L1 transcripts are unchanged in primary tumours that go on to relapse, whereas membrane PD-L1 shows a trend towards down-regulation. Finally, both methods identified 414 novel genes, also detected in ccRCC cells in vitro, including a novel non-coding gene over-expressed in patients who relapse. ConclusionsPCS and DRS can be used in tumour samples to uncover substantial yet unmapped features underpinning the plasticity and instability of cancer transcriptomes which are linked to disease progression and immune evasion.\nA clinicogenomic model including GARD predicts outcome for radiation treated patients with HPV+ oropharyngeal squamous cell carcinoma\nAuthors: Ho, E.; DeCecco, L.; Cavalieri, S.; Sedor, G.; Hoebers, F.; Brakenhoff, R.; Scheckenbach, K.; Poli, T.; Yang, K.; Scarborough, J.; Campbell, S.; Koyfman, S.; Eschrich, S. A.; Caudell, J.; Kattan, M. W.; Licitra, L.; Torres-Roca, J.; Scott, J. G.\nScore: 10.5, Published: 2023-09-14 DOI: 10.1101/2023.09.14.23295538\nBackground: Treatment decision-making in oropharyngeal squamous cell carcinoma (OPSCC) includes clinical stage, HPV status, and smoking history. Despite improvements in staging with separation of HPV-positive and -negative OPSCC in AJCC 8th edition (AJCC8), patients are largely treated with a uniform approach, with recent efforts focused on de-intensification in low-risk patients. We have previously shown, in a pooled analysis, that the genomic adjusted radiation dose (GARD) is predictive of radiation treatment benefit and can be used to guide RT dose selection. We hypothesize that GARD can be used to predict overall survival (OS) in HPV-positive OPSCC patients treated with radiotherapy (RT). Methods: Gene expression profiles (Affymetrix Clariom D) were analyzed for 234 formalin-fixed paraffin-embedded samples from HPV-positive OPSCC patients within an international, multi-institutional, prospective/retrospective observational study including patients with AJCC 7th edition stage III-IVb. GARD, a measure of the treatment effect of RT, was calculated for each patient as previously described. In total, 191 patients received primary RT definitive treatment (chemoradiation or RT alone, and 43 patients received post-operative RT. Two RT dose fractionations were utilized for primary RT cases (70 Gy in 35 fractions or 69.96 Gy in 33 fractions). Median RT dose was 70 Gy (range 50.88-74) for primary RT definitive cases and 66 Gy (range 44-70) for post-operative RT cases. The median follow up was 46.2 months (95% CI, 33.5-63.1). Cox proportional hazards analyses were performed with GARD as both a continuous and dichotomous variable and time-dependent ROC analyses compared the performance of GARD with the NRG clinical nomogram for overall survival. Results: Despite uniform radiation dose utilization, GARD showed significant heterogeneity (range 30-110), reflecting the underlying genomic differences in the cohort. On multivariable analysis, each unit increase in GARD was associated with an improvement in OS (HR = 0.951 (0.911, 0.993), p = 0.023) compared to AJCC8 (HR = 1.999 (0.791, 5.047)), p = 0.143). ROC analysis for GARD at 36 months yielded an AUC of 80.6 (69.4, 91.9) compared with an AUC of 73.6 (55.4, 91.7) for the NRG clinical nomogram. GARD[\u0026ge;]64.2 was associated with improved OS (HR = 0.280 (0.100, 0.781), p = 0.015). In a virtual trial, GARD predicts that uniform RT dose de-escalation results in overall inferior OS but proposes two separate genomic strategies where selective RT dose de-escalation in GARD-selected populations results in clinical equipoise. Conclusions: In this multi-institutional cohort of patients with HPV-positive OPSCC, GARD predicts OS as a continuous variable, outperforms the NRG nomogram and provides a novel genomic strategy to modern clinical trial design. We propose that GARD, which provides the first opportunity for genomic guided personalization of radiation dose, should be incorporated in the diagnostic workup of HPV-positive OPSCC patients.\nDecoding biological age from face photographs using deep learning\nAuthors: Zalay, O.; Bontempi, D.; Bitterman, D. S.; Birkbak, N.; Shyr, D.; Haugg, F.; Qian, J. M.; Roberts, H.; Perni, S.; Prudente, V.; Pai, S.; Dekker, A.; Haibe-Kains, B.; Guthier, C.; Balboni, T.; Warren, L.; Krishan, M.; Kann, B. H.; Swanton, C.; De Ruysscher, D.; Mak, R. H.; Aerts, H. J.\nScore: 7.2, Published: 2023-09-12 DOI: 10.1101/2023.09.12.23295132\nBecause humans age at different rates, a persons physical appearance may yield insights into their biological age and physiological health more reliably than their chronological age. In medicine, however, appearance is incorporated into medical judgments in a subjective and non-standardized fashion. In this study, we developed and validated FaceAge, a deep learning system to estimate biological age from easily obtainable and low-cost face photographs. FaceAge was trained on data from 58,851 healthy individuals, and clinical utility was evaluated on data from 6,196 patients with cancer diagnoses from two institutions in the United States and The Netherlands. To assess the prognostic relevance of FaceAge estimation, we performed Kaplan Meier survival analysis. To test a relevant clinical application of FaceAge, we assessed the performance of FaceAge in end-of-life patients with metastatic cancer who received palliative treatment by incorporating FaceAge into clinical prediction models. We found that, on average, cancer patients look older than their chronological age, and looking older is correlated with worse overall survival. FaceAge demonstrated significant independent prognostic performance in a range of cancer types and stages. We found that FaceAge can improve physicians survival predictions in incurable patients receiving palliative treatments, highlighting the clinical utility of the algorithm to support end-of-life decision-making. FaceAge was also significantly associated with molecular mechanisms of senescence through gene analysis, while age was not. These findings may extend to diseases beyond cancer, motivating using deep learning algorithms to translate a patients visual appearance into objective, quantitative, and clinically useful measures.\nThe Clinical Significance of CEA, CA19-9, and CA125 in Management of Appendiceal Adenocarcinoma\nAuthors: Yousef, A.; Yousef, M.; Zeineddine, M.; More, A.; Chowdhury, S.; Knafl, M.; Edelkamp, P.; Ito, I.; Gu, Y.; Pattalachinti, V.; Naini, Z. A.; Zeineddine, F.; Peterson, J.; Alfaro, K.; Foo, W. C.; Jin, J.; Bhutiani, N.; Higbie, V.; Scally, C.; Kee, B.; Kopetz, S.; Goldstein, D.; Uppal, A.; G. White, M.; Helmink, B.; Fournier, K.; Raghav, K.; Taggart, M.; J. Overman, M.; Shen, J. P.\nScore: 1.2, Published: 2023-09-11 DOI: 10.1101/2023.09.10.23295319\nImportanceSerum tumor markers CEA, CA19-9, \u0026 CA125 have been useful in the management of gastrointestinal and gynecological cancers, however there is limited information regarding their utility in patients with appendiceal adenocarcinoma. ObjectiveAssessing the association of serum tumor markers (CEA, CA19-9, and CA125) with clinical outcomes, pathologic, and molecular features in patients with appendiceal adenocarcinoma. DesignThis is a retrospective study with results reported in 2023. The median follow-up time was 43 months. SettingSingle tertiary care comprehensive cancer center. ParticipantsUnder an approved Institutional Review Board protocol, the Palantir Foundry software system was used to query the MD Anderson internal patient database to identify patients with a diagnosis of appendiceal adenocarcinoma and at least one tumor marker measured at MD Anderson between 2016 and 2023. ResultsA total of 1,338 patients with appendiceal adenocarcinoma were included, with a median age of 56.5 years. The majority of the patients had metastatic disease (80.7%). CEA was elevated in more than half of the patients tested (56%), while CA19-9 and CA125 were elevated in 34% and 27%, respectively. Individually, elevation of CEA, CA19-9, or CA125 were associated with worse 5-year survival; 82% vs 95%, 84% vs 92%, and 69% vs 93% elevated vs normal for CEA, CA19-9, and CA125 respectively (all p\u003c0.0001). Quantitative evaluation of tumor markers increased prognostic ability. Patients with highly elevated (top 10th percentile) CEA, CA19-9 or CA125 had markedly worse survival with 5-year survival rates of 59%, 64%, and 57%, respectively (HR vs. normal : 9.8, 6.0, 7.6, all p\u003c0.0001). Although metastatic tumors had higher levels of all tumor markers, when restricting survival analysis to 1080 patients with metastatic disease elevated CEA, CA19-9 or CA125 were all still associated worse survival (HR vs. normal : 3.4, 1.8, 3.9, p\u003c0.0001 for CEA and CA125, p=0.0019 for CA19-9). Interestingly tumor grade was not associated with CEA or CA19-9 level, while CA-125 was slightly higher in high relative to low-grade tumors (18.3 vs. 15.0, p=0.0009). Multivariable analysis identified an incremental increase in the risk of death with an increase in the number of elevated tumor markers, with a 11-fold increased risk of death in patients with all three tumor markers elevated relative to those with none elevated. Mutation in KRAS and GNAS were associated with significantly higher levels of CEA and CA19-9. ConclusionsThese findings demonstrate the utility of measuring CEA, CA19-9, and CA125 in the management of appendiceal adenocarcinoma. Given their prognostic value, all three biomarkers should be included in the initial workup of patients diagnosed with appendiceal adenocarcinoma. Key PointsO_ST_ABSQuestionC_ST_ABSCan serum tumor markers CEA, CA19-9, or CA125 be useful in management of patients with appendiceal adenocarcinoma? FindingsIn this single institution retrospective cohort study, elevation of CEA, CA19-9, or CA125 were associated with significantly worse 5-year survival; 82% vs 95%, 84% vs 92%, and 69% vs 93% elevated vs normal respectively. Moreover, quantitative evaluation of tumor markers increased prognostic ability. Further analysis identified an incremental increase in the risk of death with an increase in the number of elevated tumor markers, with a 11-fold increased risk of death in patients with all three tumor markers elevated relative to those with none elevated. MeaningGiven their prognostic value, all three biomarkers should be included in the initial workup of patients diagnosed with appendiceal adenocarcinoma.\n",
  "wordCount" : "2029",
  "inLanguage": "en",
  "datePublished": "2023-09-17T10:36:50Z",
  "dateModified": "2023-09-17T10:36:50Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/oncology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      oncology
    </h1>
    <div class="post-meta"><span>updated on September 17, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.23295131">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.23295131" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.23295131">
        <p class="paperTitle">Deep Learning Model for Tumor Type Prediction using Targeted Clinical Genomic Sequencing Data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.23295131" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.23295131" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Darmofal, M.; Suman, S.; Atwal, G.; Chen, J.-F.; Chang, J. C.; Toomey, M.; Vakiani, E.; Varghese, A. M.; Balakrishnan Rema, A.; Syed, A.; Schultz, N.; Berger, M.; Morris, Q.</p>
        <p class="info">Score: 16.1, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.23295131' target='https://doi.org/10.1101/2023.09.08.23295131'> 10.1101/2023.09.08.23295131</a></p>
        <p class="abstract">Tumor type guides clinical treatment decisions in cancer, but histology-based diagnosis remains challenging. Genomic alterations are highly diagnostic of tumor type, and tumor type classifiers trained on genomic features have been explored, but the most accurate methods are not clinically feasible, relying on features derived from whole genome sequencing (WGS), or predicting across limited cancer types. We use genomic features from a dataset of 39,787 solid tumors sequenced using a clinical targeted cancer gene panel to develop Genome-Derived-Diagnosis Ensemble (GDD-ENS): a hyperparameter ensemble for classifying tumor type using deep neural networks. GDD-ENS achieves 93% accuracy for high-confidence predictions across 38 cancer types, rivalling performance of WGS-based methods. GDD-ENS can also guide diagnoses on rare type and cancers of unknown primary, and incorporate patient-specific clinical information for improved predictions. Overall, integrating GDD-ENS into prospective clinical sequencing workflows has enabled clinically-relevant tumor type predictions to guide treatment decisions in real time.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.23295204">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.23295204" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.23295204">
        <p class="paperTitle">Long-read RNA sequencing redefines the clear cell renal cell carcinoma transcriptome and reveals novel genes and transcripts associated with disease recurrence and immune evasion</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.23295204" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.23295204" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, J.; Snell, E. A.; Brown, J.; Banks, R. E.; Turner, D. J.; Vasudev, N. S.; Lagos, D.</p>
        <p class="info">Score: 13.6, Published: 2023-09-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.23295204' target='https://doi.org/10.1101/2023.09.08.23295204'> 10.1101/2023.09.08.23295204</a></p>
        <p class="abstract">BackgroundLong-read direct RNA sequencing (DRS) and PCR cDNA sequencing (PCS) of tumour samples could lead to discovery of novel transcript isoforms, novel genes, and transcriptomic co-dependencies missed by conventional short-read sequencing. However, only a handful of reports using DRS or PCS in cancer exist with no direct comparison between the two methods. Clear cell renal cell carcinoma (ccRCC) is the most common form of kidney cancer. Following primary tumour resection approximately 30% of patients experience disease recurrence. Long-read RNA sequencing has not been applied to kidney cancer.

Methods12 primary ccRCC archival tumours (discovery cohort), 6 from patients who went on to relapse, were analysed by Oxford Nanopore DRS and PCS. Results were validated in an independent cohort of 20 patients and compared to DRS analysis of RCC4 cells.

ResultsDRS and PCS were successfully performed achieving high read length, with PCS achieving higher sequencing depth. Differentially expressed gene sets in patients who went on to relapse were determined with good overlap between DRS and PCS. Deconvolution analysis showed a loss of immune infiltrate in primary tumours of patients who relapse and revealed the CD8&#43; T cell exhaustion marker TOX as a novel recurrence-associated gene. Notably, novel transcript analysis revealed more than 10,000 uncharacterised candidate novel transcripts detected by both methods and in a ccRCC cell line in vitro. This allowed the definition of the full exonic structure of ccRCC-associated splice variants, including variants of MVK and HPCAL1. Remarkably, this also revealed a novel sPD-L1 transcript encoding for the soluble version of the protein with a longer 3UTR and lower stability in ccRCC cells than the annotated transcript. Levels of sPD-L1 transcripts are unchanged in primary tumours that go on to relapse, whereas membrane PD-L1 shows a trend towards down-regulation. Finally, both methods identified 414 novel genes, also detected in ccRCC cells in vitro, including a novel non-coding gene over-expressed in patients who relapse.

ConclusionsPCS and DRS can be used in tumour samples to uncover substantial yet unmapped features underpinning the plasticity and instability of cancer transcriptomes which are linked to disease progression and immune evasion.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.14.23295538">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.14.23295538" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.14.23295538">
        <p class="paperTitle">A clinicogenomic model including GARD predicts outcome for radiation treated patients with HPV&#43; oropharyngeal squamous cell carcinoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.14.23295538" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.14.23295538" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ho, E.; DeCecco, L.; Cavalieri, S.; Sedor, G.; Hoebers, F.; Brakenhoff, R.; Scheckenbach, K.; Poli, T.; Yang, K.; Scarborough, J.; Campbell, S.; Koyfman, S.; Eschrich, S. A.; Caudell, J.; Kattan, M. W.; Licitra, L.; Torres-Roca, J.; Scott, J. G.</p>
        <p class="info">Score: 10.5, Published: 2023-09-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.14.23295538' target='https://doi.org/10.1101/2023.09.14.23295538'> 10.1101/2023.09.14.23295538</a></p>
        <p class="abstract">Background: Treatment decision-making in oropharyngeal squamous cell carcinoma (OPSCC) includes clinical stage, HPV status, and smoking history. Despite improvements in staging with separation of HPV-positive and -negative OPSCC in AJCC 8th edition (AJCC8), patients are largely treated with a uniform approach, with recent efforts focused on de-intensification in low-risk patients. We have previously shown, in a pooled analysis, that the genomic adjusted radiation dose (GARD) is predictive of radiation treatment benefit and can be used to guide RT dose selection. We hypothesize that GARD can be used to predict overall survival (OS) in HPV-positive OPSCC patients treated with radiotherapy (RT). Methods: Gene expression profiles (Affymetrix Clariom D) were analyzed for 234 formalin-fixed paraffin-embedded samples from HPV-positive OPSCC patients within an international, multi-institutional, prospective/retrospective observational study including patients with AJCC 7th edition stage III-IVb. GARD, a measure of the treatment effect of RT, was calculated for each patient as previously described. In total, 191 patients received primary RT definitive treatment (chemoradiation or RT alone, and 43 patients received post-operative RT. Two RT dose fractionations were utilized for primary RT cases (70 Gy in 35 fractions or 69.96 Gy in 33 fractions). Median RT dose was 70 Gy (range 50.88-74) for primary RT definitive cases and 66 Gy (range 44-70) for post-operative RT cases. The median follow up was 46.2 months (95% CI, 33.5-63.1). Cox proportional hazards analyses were performed with GARD as both a continuous and dichotomous variable and time-dependent ROC analyses compared the performance of GARD with the NRG clinical nomogram for overall survival. Results: Despite uniform radiation dose utilization, GARD showed significant heterogeneity (range 30-110), reflecting the underlying genomic differences in the cohort. On multivariable analysis, each unit increase in GARD was associated with an improvement in OS (HR = 0.951 (0.911, 0.993), p = 0.023) compared to AJCC8 (HR = 1.999 (0.791, 5.047)), p = 0.143). ROC analysis for GARD at 36 months yielded an AUC of 80.6 (69.4, 91.9) compared with an AUC of 73.6 (55.4, 91.7) for the NRG clinical nomogram. GARD[&amp;ge;]64.2 was associated with improved OS (HR = 0.280 (0.100, 0.781), p = 0.015). In a virtual trial, GARD predicts that uniform RT dose de-escalation results in overall inferior OS but proposes two separate genomic strategies where selective RT dose de-escalation in GARD-selected populations results in clinical equipoise. Conclusions: In this multi-institutional cohort of patients with HPV-positive OPSCC, GARD predicts OS as a continuous variable, outperforms the NRG nomogram and provides a novel genomic strategy to modern clinical trial design. We propose that GARD, which provides the first opportunity for genomic guided personalization of radiation dose, should be incorporated in the diagnostic workup of HPV-positive OPSCC patients.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.23295132">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.23295132" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.23295132">
        <p class="paperTitle">Decoding biological age from face photographs using deep learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.23295132" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.23295132" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zalay, O.; Bontempi, D.; Bitterman, D. S.; Birkbak, N.; Shyr, D.; Haugg, F.; Qian, J. M.; Roberts, H.; Perni, S.; Prudente, V.; Pai, S.; Dekker, A.; Haibe-Kains, B.; Guthier, C.; Balboni, T.; Warren, L.; Krishan, M.; Kann, B. H.; Swanton, C.; De Ruysscher, D.; Mak, R. H.; Aerts, H. J.</p>
        <p class="info">Score: 7.2, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.23295132' target='https://doi.org/10.1101/2023.09.12.23295132'> 10.1101/2023.09.12.23295132</a></p>
        <p class="abstract">Because humans age at different rates, a persons physical appearance may yield insights into their biological age and physiological health more reliably than their chronological age. In medicine, however, appearance is incorporated into medical judgments in a subjective and non-standardized fashion. In this study, we developed and validated FaceAge, a deep learning system to estimate biological age from easily obtainable and low-cost face photographs. FaceAge was trained on data from 58,851 healthy individuals, and clinical utility was evaluated on data from 6,196 patients with cancer diagnoses from two institutions in the United States and The Netherlands. To assess the prognostic relevance of FaceAge estimation, we performed Kaplan Meier survival analysis. To test a relevant clinical application of FaceAge, we assessed the performance of FaceAge in end-of-life patients with metastatic cancer who received palliative treatment by incorporating FaceAge into clinical prediction models. We found that, on average, cancer patients look older than their chronological age, and looking older is correlated with worse overall survival. FaceAge demonstrated significant independent prognostic performance in a range of cancer types and stages. We found that FaceAge can improve physicians survival predictions in incurable patients receiving palliative treatments, highlighting the clinical utility of the algorithm to support end-of-life decision-making. FaceAge was also significantly associated with molecular mechanisms of senescence through gene analysis, while age was not. These findings may extend to diseases beyond cancer, motivating using deep learning algorithms to translate a patients visual appearance into objective, quantitative, and clinically useful measures.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.10.23295319">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.10.23295319" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.10.23295319">
        <p class="paperTitle">The Clinical Significance of CEA, CA19-9, and CA125 in Management of Appendiceal Adenocarcinoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.10.23295319" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.10.23295319" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yousef, A.; Yousef, M.; Zeineddine, M.; More, A.; Chowdhury, S.; Knafl, M.; Edelkamp, P.; Ito, I.; Gu, Y.; Pattalachinti, V.; Naini, Z. A.; Zeineddine, F.; Peterson, J.; Alfaro, K.; Foo, W. C.; Jin, J.; Bhutiani, N.; Higbie, V.; Scally, C.; Kee, B.; Kopetz, S.; Goldstein, D.; Uppal, A.; G. White, M.; Helmink, B.; Fournier, K.; Raghav, K.; Taggart, M.; J. Overman, M.; Shen, J. P.</p>
        <p class="info">Score: 1.2, Published: 2023-09-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.10.23295319' target='https://doi.org/10.1101/2023.09.10.23295319'> 10.1101/2023.09.10.23295319</a></p>
        <p class="abstract">ImportanceSerum tumor markers CEA, CA19-9, &amp; CA125 have been useful in the management of gastrointestinal and gynecological cancers, however there is limited information regarding their utility in patients with appendiceal adenocarcinoma.

ObjectiveAssessing the association of serum tumor markers (CEA, CA19-9, and CA125) with clinical outcomes, pathologic, and molecular features in patients with appendiceal adenocarcinoma.

DesignThis is a retrospective study with results reported in 2023. The median follow-up time was 43 months.

SettingSingle tertiary care comprehensive cancer center.

ParticipantsUnder an approved Institutional Review Board protocol, the Palantir Foundry software system was used to query the MD Anderson internal patient database to identify patients with a diagnosis of appendiceal adenocarcinoma and at least one tumor marker measured at MD Anderson between 2016 and 2023.

ResultsA total of 1,338 patients with appendiceal adenocarcinoma were included, with a median age of 56.5 years. The majority of the patients had metastatic disease (80.7%). CEA was elevated in more than half of the patients tested (56%), while CA19-9 and CA125 were elevated in 34% and 27%, respectively. Individually, elevation of CEA, CA19-9, or CA125 were associated with worse 5-year survival; 82% vs 95%, 84% vs 92%, and 69% vs 93% elevated vs normal for CEA, CA19-9, and CA125 respectively (all p&lt;0.0001). Quantitative evaluation of tumor markers increased prognostic ability. Patients with highly elevated (top 10th percentile) CEA, CA19-9 or CA125 had markedly worse survival with 5-year survival rates of 59%, 64%, and 57%, respectively (HR vs. normal : 9.8, 6.0, 7.6, all p&lt;0.0001). Although metastatic tumors had higher levels of all tumor markers, when restricting survival analysis to 1080 patients with metastatic disease elevated CEA, CA19-9 or CA125 were all still associated worse survival (HR vs. normal : 3.4, 1.8, 3.9, p&lt;0.0001 for CEA and CA125, p=0.0019 for CA19-9). Interestingly tumor grade was not associated with CEA or CA19-9 level, while CA-125 was slightly higher in high relative to low-grade tumors (18.3 vs. 15.0, p=0.0009). Multivariable analysis identified an incremental increase in the risk of death with an increase in the number of elevated tumor markers, with a 11-fold increased risk of death in patients with all three tumor markers elevated relative to those with none elevated. Mutation in KRAS and GNAS were associated with significantly higher levels of CEA and CA19-9.

ConclusionsThese findings demonstrate the utility of measuring CEA, CA19-9, and CA125 in the management of appendiceal adenocarcinoma. Given their prognostic value, all three biomarkers should be included in the initial workup of patients diagnosed with appendiceal adenocarcinoma.

Key PointsO_ST_ABSQuestionC_ST_ABSCan serum tumor markers CEA, CA19-9, or CA125 be useful in management of patients with appendiceal adenocarcinoma?

FindingsIn this single institution retrospective cohort study, elevation of CEA, CA19-9, or CA125 were associated with significantly worse 5-year survival; 82% vs 95%, 84% vs 92%, and 69% vs 93% elevated vs normal respectively. Moreover, quantitative evaluation of tumor markers increased prognostic ability. Further analysis identified an incremental increase in the risk of death with an increase in the number of elevated tumor markers, with a 11-fold increased risk of death in patients with all three tumor markers elevated relative to those with none elevated.

MeaningGiven their prognostic value, all three biomarkers should be included in the initial workup of patients diagnosed with appendiceal adenocarcinoma.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
