<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>oncology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>oncology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A Video Intervention to Improve Patient Understanding of Tumor Genomic Testing in Patients with Cancer
Authors: Veney, D.; Wei, L.; Toland, A. E.; Presley, C. J.; Hampel, H.; Padamsee, T. J.; Lee, C. N.; Irvin, W. J.; Bishop, M.; Kim, J.; Hovick, S. R.; Senter, L.; Stover, D.
Score: 6.8, Published: 2023-12-05 DOI: 10.1101/2023.12.05.23299443
BackgroundTumor genomic testing (TGT) has become standard-of-care for most patients with advanced/metastatic cancer. Despite established guidelines, patient education prior to TGT is variable or frequently omitted.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/oncology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="oncology" />
<meta property="og:description" content="A Video Intervention to Improve Patient Understanding of Tumor Genomic Testing in Patients with Cancer
Authors: Veney, D.; Wei, L.; Toland, A. E.; Presley, C. J.; Hampel, H.; Padamsee, T. J.; Lee, C. N.; Irvin, W. J.; Bishop, M.; Kim, J.; Hovick, S. R.; Senter, L.; Stover, D.
Score: 6.8, Published: 2023-12-05 DOI: 10.1101/2023.12.05.23299443
BackgroundTumor genomic testing (TGT) has become standard-of-care for most patients with advanced/metastatic cancer. Despite established guidelines, patient education prior to TGT is variable or frequently omitted." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/oncology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-10T10:37:48+00:00" />
<meta property="article:modified_time" content="2023-12-10T10:37:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="oncology"/>
<meta name="twitter:description" content="A Video Intervention to Improve Patient Understanding of Tumor Genomic Testing in Patients with Cancer
Authors: Veney, D.; Wei, L.; Toland, A. E.; Presley, C. J.; Hampel, H.; Padamsee, T. J.; Lee, C. N.; Irvin, W. J.; Bishop, M.; Kim, J.; Hovick, S. R.; Senter, L.; Stover, D.
Score: 6.8, Published: 2023-12-05 DOI: 10.1101/2023.12.05.23299443
BackgroundTumor genomic testing (TGT) has become standard-of-care for most patients with advanced/metastatic cancer. Despite established guidelines, patient education prior to TGT is variable or frequently omitted."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "oncology",
      "item": "https://trxiv.yorks0n.com/posts/oncology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "oncology",
  "name": "oncology",
  "description": "A Video Intervention to Improve Patient Understanding of Tumor Genomic Testing in Patients with Cancer\nAuthors: Veney, D.; Wei, L.; Toland, A. E.; Presley, C. J.; Hampel, H.; Padamsee, T. J.; Lee, C. N.; Irvin, W. J.; Bishop, M.; Kim, J.; Hovick, S. R.; Senter, L.; Stover, D.\nScore: 6.8, Published: 2023-12-05 DOI: 10.1101/2023.12.05.23299443\nBackgroundTumor genomic testing (TGT) has become standard-of-care for most patients with advanced/metastatic cancer. Despite established guidelines, patient education prior to TGT is variable or frequently omitted.",
  "keywords": [
    
  ],
  "articleBody": " A Video Intervention to Improve Patient Understanding of Tumor Genomic Testing in Patients with Cancer\nAuthors: Veney, D.; Wei, L.; Toland, A. E.; Presley, C. J.; Hampel, H.; Padamsee, T. J.; Lee, C. N.; Irvin, W. J.; Bishop, M.; Kim, J.; Hovick, S. R.; Senter, L.; Stover, D.\nScore: 6.8, Published: 2023-12-05 DOI: 10.1101/2023.12.05.23299443\nBackgroundTumor genomic testing (TGT) has become standard-of-care for most patients with advanced/metastatic cancer. Despite established guidelines, patient education prior to TGT is variable or frequently omitted. The purpose of this study was to evaluate the impact of a concise (3-4 minute) video for patient education prior to TGT. MethodsBased on a quality improvement cycle, an animated video was created to be applicable to any cancer type, incorporating culturally diverse images, available in English and Spanish. Patients undergoing standard-of care TGT were enrolled at a tertiary academic institution and completed validated survey instruments immediately prior to video viewing (T1) and immediately post-viewing (T2). Instruments included: 1) 10-question objective genomic knowledge/understanding; 2) 10-question video message-specific knowledge/recall; 3) 11-question Trust in Physician/Provider; 4) attitudes regarding TGT. The primary objective was change in outcomes from before to after the video was assessed with Wilcoxon signed rank test. ResultsFrom April 2022 to May 2023, a total of 150 participants were enrolled (MBC n=53, LC n=38, OC n=59). For the primary endpoint, there was a significant increase in video message-specific knowledge (median 10 point increase; p\u003c0.0001) with no significant change in genomic knowledge/understanding (p=0.89) or Trust in Physician/Provider (p=0.59). Results for five questions significantly improved, including the likelihood of TGT impact on treatment decision, incidental germline findings, and cost of testing. Improvement in video message-specific knowledge was consistent across demographic groups, including age, income, and education. Individuals with less educational attainment had had greater improvement from before to after video viewing. ConclusionsA concise, 3-4 minute, broadly applicable video incorporating culturally diverse images administered prior to TGT significantly improved video message-specific knowledge across all demographic groups. This resource is publicly available at http://www.tumor-testing.com, with a goal to efficiently educate and empower patients regarding TGT while addressing guidelines within the flow of clinical practice. Clinical Trial RegistrationClinicalTrials.gov NCT05215769\nIdentification of clinically-relevant genetic alterations in uveal melanoma using RNA sequencing\nAuthors: Nell, R. J.; Versluis, M.; Cats, D.; Mei, H.; Verdijk, R. M.; Kroes, W. G. M.; Luyten, G. P. M.; Jager, M. J.; Van der Velden, P. A.\nScore: 1.9, Published: 2023-12-05 DOI: 10.1101/2023.12.03.23299340\nIntroductionUveal melanoma is a lethal intraocular tumour, in which the presence of certain genetic alterations correlates with the risk of metastatic dissemination and patient survival. RNA data is typically used to transcriptionally characterise tumours and their micro-environment. In this study, we tested the detectability of all key genetic alterations in uveal melanoma from RNA sequencing data. MethodsCohort-wide gene expression profiling was used to classify tumours at the transcriptional level. In individual samples, copy number alterations affecting chromosomes 3 and 8q were analysed by measuring expressed allelic imbalances of heterozygous common single nucleotide polymorphisms. Mutations in GNAQ, GNA11, CYSLTR2, PLCB4, BAP1, SF3B1 and EIF1AX were identified by screening of hotspot regions and by evaluating their transcriptional effects. All findings were cross-validated with DNA-derived data in a training cohort of 80 primary uveal melanomas studied by The Cancer Genome Atlas (TCGA) initiative, and in five prospectively analysed cases from our institution. ResultsUnsupervised gene expression profiling strongly correlated to the presence of chromosome 3 alterations, but was not reliable in identifying other (clinically-)relevant genetic alterations. However, the presence of both chromosome 3 and 8q copy number alterations could be successfully inferred from expressed allelic imbalances in most tumours. The majority of mutations were adequately recognised at the RNA level by their nucleotide changes (all genes), alternative splicing around the mutant position (BAP1) and transcriptome-wide aberrant splice junction usage (SF3B1). Notably, in the TCGA cohort we detected previously unreported mutations in BAP1 (n=3) and EIF1AX (n=5), that were missed by the original DNA sequencing. In our prospective cohort, all mutations and copy number alterations were successfully identified at the RNA level by combining the described approaches. ConclusionIn addition to providing gene expression levels and profiles, RNA from uveal melanomas presents insights into the expressed tumour genotype and its phenotypic consequences. Such complete analysis of transcriptional data may augment or even substitute current DNA-based approaches, and has potential applicability in both research and clinical practice.\nEvolutionary dynamics of 1,976 lymphoid malignancies predict clinical outcome\nAuthors: Gabbutt, C.; Duran-Ferrer, M.; Grant, H.; Mallo, D.; Nadeu, F.; Househam, J.; Villamor, N.; Krali, O.; Nordlund, J.; Zenz, T.; Campo, E.; Lopez-Guillermo, A.; Fitzgibbon, J.; Barnes, C. P.; Shibata, D.; Martin-Subero, J. I.; Graham, T. A.\nScore: 55.5, Published: 2023-11-20 DOI: 10.1101/2023.11.10.23298336\nCancer development, progression, and response to treatment are evolutionary processes, but characterising the evolutionary dynamics at sufficient scale to be clinically-meaningful has remained challenging. Here, we develop a new methodology called EVOFLUx, based upon natural DNA methylation barcodes fluctuating over time, that quantitatively infers evolutionary dynamics using only a bulk tumour methylation profile as input. We apply EVOFLUx to 1,976 well-characterised lymphoid cancer samples spanning a broad spectrum of diseases and show that tumour growth rates, malignancy age and epimutation rates vary by orders of magnitude across disease types. We measure that subclonal selection occurs only infrequently within bulk samples and detect occasional examples of multiple independent primary tumours. Clinically, we observe that tumour growth rates are higher in more aggressive disease subtypes, and in two series of chronic lymphocytic leukaemia patients, evolutionary histories are independent prognostic factors. Phylogenetic analyses of longitudinal CLL samples using EVOFLUx detect the seeds of future Richter transformation many decades prior to presentation. We provide orthogonal verification of EVOFLUx inferences using additional genetic and clinical data. Collectively, we show how widely-available, low-cost bulk DNA methylation data precisely measures cancer evolutionary dynamics, and provides new insights into cancer biology and clinical behaviour.\nA three-gene expression score for predicting clinical benefit to anti-PD-1 blockade in advanced renal cell carcinoma\nAuthors: Betancor, Y. Z.; Ferreiro-Pantin, M.; Anido-Herranz, U.; Fuentes-Losada, M.; Leon-Mateos, L.; Garcia-Acuna, S. M.; Vaamonde-Rodriguez, V.; Garcia-Pinel, B.; Cebey-Lopez, V.; Villaverde-Viano, R.; Kotrulev, M.; Fernandez-Diaz, N.; Gomez-Tourino, I.; Fernandez-Baltar, C.; Garcia-Gonzalez, J.; C. Tubio, J. M.; Lopez-Lopez, R.; Ruiz-Banobre, J.\nScore: 1.2, Published: 2023-12-04 DOI: 10.1101/2023.12.03.23299342\nBackgroundIn the advanced renal cell carcinoma (RCC) scenario there are no consistent biomarkers to predict the clinical benefit patients derived from immune checkpoint blockade. We conducted a retrospective study in order to develop and to validate a gene expression score for predicting clinical benefit to nivolumab in the advanced clear cell RCC setting and to characterize its underlying clinical, molecular, and immune features. MethodsThis is a post hoc pooled analysis of 311 patients with available clinical, molecular, and immune tumor data from the CheckMate-009, -010 and -025 trials. Efficacy endpoints were overall survival (OS), disease control rate, and overall response rate. Survival estimates were calculated by the Kaplan-Meier method, and groups were compared with the log-rank test. The Cox proportional hazards regression model was used to evaluate factors independently associated with OS. Factors associated with disease control and response were tested with logistic regression. Biomarker-treatment interaction was evaluated with the likelihood ratio test (LRT). ResultsFirst, a three-gene expression score (3GES) with prognostic value for OS integrating HMGA1, NUP62, and ARHGAP42 transcripts was developed in a cohort of patients treated with nivolumab. Favorable 3GES risk category was significantly associated with a better OS in univariable (HR = 0.32, 95% CI 0.21 - 0.48, P\u003c0.001) and multivariable (HR = 0.36, 95% CI 0.24 - 0.56, P\u003c0.001) analyses. Consistent and significant correlation was found with disease control (P=0.066) and response (P=0.002). The 3GES prognostic value was validated in the TCGA-KIRC cohort (HR=0.35, P\u003c0.001). Next, the predictive value for nivolumab was confirmed in a set of patients from the CheckMate-025 trial (LRT P\u003c0.001). ConclusionsIn accRCC, 3GES is not only an independent prognostic factor for OS but also a positive predictive biomarker for nivolumab. The predictive value deserves further validation in other retrospective and large-scale, randomized prospective studies.\n",
  "wordCount" : "1325",
  "inLanguage": "en",
  "datePublished": "2023-12-10T10:37:48Z",
  "dateModified": "2023-12-10T10:37:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/oncology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      oncology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 10, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.23299443">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.23299443" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.23299443">
        <p class="paperTitle">A Video Intervention to Improve Patient Understanding of Tumor Genomic Testing in Patients with Cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.23299443" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.23299443" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Veney, D.; Wei, L.; Toland, A. E.; Presley, C. J.; Hampel, H.; Padamsee, T. J.; Lee, C. N.; Irvin, W. J.; Bishop, M.; Kim, J.; Hovick, S. R.; Senter, L.; Stover, D.</p>
        <p class="info">Score: 6.8, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.23299443' target='https://doi.org/10.1101/2023.12.05.23299443'> 10.1101/2023.12.05.23299443</a></p>
        <p class="abstract">BackgroundTumor genomic testing (TGT) has become standard-of-care for most patients with advanced/metastatic cancer. Despite established guidelines, patient education prior to TGT is variable or frequently omitted. The purpose of this study was to evaluate the impact of a concise (3-4 minute) video for patient education prior to TGT.

MethodsBased on a quality improvement cycle, an animated video was created to be applicable to any cancer type, incorporating culturally diverse images, available in English and Spanish. Patients undergoing standard-of care TGT were enrolled at a tertiary academic institution and completed validated survey instruments immediately prior to video viewing (T1) and immediately post-viewing (T2). Instruments included: 1) 10-question objective genomic knowledge/understanding; 2) 10-question video message-specific knowledge/recall; 3) 11-question Trust in Physician/Provider; 4) attitudes regarding TGT. The primary objective was change in outcomes from before to after the video was assessed with Wilcoxon signed rank test.

ResultsFrom April 2022 to May 2023, a total of 150 participants were enrolled (MBC n=53, LC n=38, OC n=59). For the primary endpoint, there was a significant increase in video message-specific knowledge (median 10 point increase; p&lt;0.0001) with no significant change in genomic knowledge/understanding (p=0.89) or Trust in Physician/Provider (p=0.59). Results for five questions significantly improved, including the likelihood of TGT impact on treatment decision, incidental germline findings, and cost of testing. Improvement in video message-specific knowledge was consistent across demographic groups, including age, income, and education. Individuals with less educational attainment had had greater improvement from before to after video viewing.

ConclusionsA concise, 3-4 minute, broadly applicable video incorporating culturally diverse images administered prior to TGT significantly improved video message-specific knowledge across all demographic groups. This resource is publicly available at http://www.tumor-testing.com, with a goal to efficiently educate and empower patients regarding TGT while addressing guidelines within the flow of clinical practice.

Clinical Trial RegistrationClinicalTrials.gov NCT05215769</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.03.23299340">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.03.23299340" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.03.23299340">
        <p class="paperTitle">Identification of clinically-relevant genetic alterations in uveal melanoma using RNA sequencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.03.23299340" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.03.23299340" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nell, R. J.; Versluis, M.; Cats, D.; Mei, H.; Verdijk, R. M.; Kroes, W. G. M.; Luyten, G. P. M.; Jager, M. J.; Van der Velden, P. A.</p>
        <p class="info">Score: 1.9, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.03.23299340' target='https://doi.org/10.1101/2023.12.03.23299340'> 10.1101/2023.12.03.23299340</a></p>
        <p class="abstract">IntroductionUveal melanoma is a lethal intraocular tumour, in which the presence of certain genetic alterations correlates with the risk of metastatic dissemination and patient survival. RNA data is typically used to transcriptionally characterise tumours and their micro-environment. In this study, we tested the detectability of all key genetic alterations in uveal melanoma from RNA sequencing data.

MethodsCohort-wide gene expression profiling was used to classify tumours at the transcriptional level. In individual samples, copy number alterations affecting chromosomes 3 and 8q were analysed by measuring expressed allelic imbalances of heterozygous common single nucleotide polymorphisms. Mutations in GNAQ, GNA11, CYSLTR2, PLCB4, BAP1, SF3B1 and EIF1AX were identified by screening of hotspot regions and by evaluating their transcriptional effects. All findings were cross-validated with DNA-derived data in a training cohort of 80 primary uveal melanomas studied by The Cancer Genome Atlas (TCGA) initiative, and in five prospectively analysed cases from our institution.

ResultsUnsupervised gene expression profiling strongly correlated to the presence of chromosome 3 alterations, but was not reliable in identifying other (clinically-)relevant genetic alterations. However, the presence of both chromosome 3 and 8q copy number alterations could be successfully inferred from expressed allelic imbalances in most tumours. The majority of mutations were adequately recognised at the RNA level by their nucleotide changes (all genes), alternative splicing around the mutant position (BAP1) and transcriptome-wide aberrant splice junction usage (SF3B1). Notably, in the TCGA cohort we detected previously unreported mutations in BAP1 (n=3) and EIF1AX (n=5), that were missed by the original DNA sequencing. In our prospective cohort, all mutations and copy number alterations were successfully identified at the RNA level by combining the described approaches.

ConclusionIn addition to providing gene expression levels and profiles, RNA from uveal melanomas presents insights into the expressed tumour genotype and its phenotypic consequences. Such complete analysis of transcriptional data may augment or even substitute current DNA-based approaches, and has potential applicability in both research and clinical practice.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.23298336">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.23298336" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.23298336">
        <p class="paperTitle">Evolutionary dynamics of 1,976 lymphoid malignancies predict clinical outcome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.23298336" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.23298336" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gabbutt, C.; Duran-Ferrer, M.; Grant, H.; Mallo, D.; Nadeu, F.; Househam, J.; Villamor, N.; Krali, O.; Nordlund, J.; Zenz, T.; Campo, E.; Lopez-Guillermo, A.; Fitzgibbon, J.; Barnes, C. P.; Shibata, D.; Martin-Subero, J. I.; Graham, T. A.</p>
        <p class="info">Score: 55.5, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.23298336' target='https://doi.org/10.1101/2023.11.10.23298336'> 10.1101/2023.11.10.23298336</a></p>
        <p class="abstract">Cancer development, progression, and response to treatment are evolutionary processes, but characterising the evolutionary dynamics at sufficient scale to be clinically-meaningful has remained challenging. Here, we develop a new methodology called EVOFLUx, based upon natural DNA methylation barcodes fluctuating over time, that quantitatively infers evolutionary dynamics using only a bulk tumour methylation profile as input. We apply EVOFLUx to 1,976 well-characterised lymphoid cancer samples spanning a broad spectrum of diseases and show that tumour growth rates, malignancy age and epimutation rates vary by orders of magnitude across disease types. We measure that subclonal selection occurs only infrequently within bulk samples and detect occasional examples of multiple independent primary tumours. Clinically, we observe that tumour growth rates are higher in more aggressive disease subtypes, and in two series of chronic lymphocytic leukaemia patients, evolutionary histories are independent prognostic factors. Phylogenetic analyses of longitudinal CLL samples using EVOFLUx detect the seeds of future Richter transformation many decades prior to presentation. We provide orthogonal verification of EVOFLUx inferences using additional genetic and clinical data. Collectively, we show how widely-available, low-cost bulk DNA methylation data precisely measures cancer evolutionary dynamics, and provides new insights into cancer biology and clinical behaviour.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.03.23299342">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.03.23299342" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.03.23299342">
        <p class="paperTitle">A three-gene expression score for predicting clinical benefit to anti-PD-1 blockade in advanced renal cell carcinoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.03.23299342" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.03.23299342" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Betancor, Y. Z.; Ferreiro-Pantin, M.; Anido-Herranz, U.; Fuentes-Losada, M.; Leon-Mateos, L.; Garcia-Acuna, S. M.; Vaamonde-Rodriguez, V.; Garcia-Pinel, B.; Cebey-Lopez, V.; Villaverde-Viano, R.; Kotrulev, M.; Fernandez-Diaz, N.; Gomez-Tourino, I.; Fernandez-Baltar, C.; Garcia-Gonzalez, J.; C. Tubio, J. M.; Lopez-Lopez, R.; Ruiz-Banobre, J.</p>
        <p class="info">Score: 1.2, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.03.23299342' target='https://doi.org/10.1101/2023.12.03.23299342'> 10.1101/2023.12.03.23299342</a></p>
        <p class="abstract">BackgroundIn the advanced renal cell carcinoma (RCC) scenario there are no consistent biomarkers to predict the clinical benefit patients derived from immune checkpoint blockade. We conducted a retrospective study in order to develop and to validate a gene expression score for predicting clinical benefit to nivolumab in the advanced clear cell RCC setting and to characterize its underlying clinical, molecular, and immune features.

MethodsThis is a post hoc pooled analysis of 311 patients with available clinical, molecular, and immune tumor data from the CheckMate-009, -010 and -025 trials. Efficacy endpoints were overall survival (OS), disease control rate, and overall response rate. Survival estimates were calculated by the Kaplan-Meier method, and groups were compared with the log-rank test. The Cox proportional hazards regression model was used to evaluate factors independently associated with OS. Factors associated with disease control and response were tested with logistic regression. Biomarker-treatment interaction was evaluated with the likelihood ratio test (LRT).

ResultsFirst, a three-gene expression score (3GES) with prognostic value for OS integrating HMGA1, NUP62, and ARHGAP42 transcripts was developed in a cohort of patients treated with nivolumab. Favorable 3GES risk category was significantly associated with a better OS in univariable (HR = 0.32, 95% CI 0.21 - 0.48, P&lt;0.001) and multivariable (HR = 0.36, 95% CI 0.24 - 0.56, P&lt;0.001) analyses. Consistent and significant correlation was found with disease control (P=0.066) and response (P=0.002). The 3GES prognostic value was validated in the TCGA-KIRC cohort (HR=0.35, P&lt;0.001). Next, the predictive value for nivolumab was confirmed in a set of patients from the CheckMate-025 trial (LRT P&lt;0.001).

ConclusionsIn accRCC, 3GES is not only an independent prognostic factor for OS but also a positive predictive biomarker for nivolumab. The predictive value deserves further validation in other retrospective and large-scale, randomized prospective studies.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
