<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>oncology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>oncology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Evolutionary dynamics of 1,976 lymphoid malignancies predict clinical outcome
Authors: Gabbutt, C.; Duran-Ferrer, M.; Grant, H.; Mallo, D.; Nadeu, F.; Househam, J.; Villamor, N.; Krali, O.; Nordlund, J.; Zenz, T.; Campo, E.; Lopez-Guillermo, A.; Fitzgibbon, J.; Barnes, C. P.; Shibata, D.; Martin-Subero, J. I.; Graham, T. A.
Score: 42.8, Published: 2023-11-10 DOI: 10.1101/2023.11.10.23298336
Cancer development, progression, and response to treatment are evolutionary processes, but characterising the evolutionary dynamics at sufficient scale to be clinically-meaningful has remained challenging.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/oncology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="oncology" />
<meta property="og:description" content="Evolutionary dynamics of 1,976 lymphoid malignancies predict clinical outcome
Authors: Gabbutt, C.; Duran-Ferrer, M.; Grant, H.; Mallo, D.; Nadeu, F.; Househam, J.; Villamor, N.; Krali, O.; Nordlund, J.; Zenz, T.; Campo, E.; Lopez-Guillermo, A.; Fitzgibbon, J.; Barnes, C. P.; Shibata, D.; Martin-Subero, J. I.; Graham, T. A.
Score: 42.8, Published: 2023-11-10 DOI: 10.1101/2023.11.10.23298336
Cancer development, progression, and response to treatment are evolutionary processes, but characterising the evolutionary dynamics at sufficient scale to be clinically-meaningful has remained challenging." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/oncology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-19T10:38:34+00:00" />
<meta property="article:modified_time" content="2023-11-19T10:38:34+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="oncology"/>
<meta name="twitter:description" content="Evolutionary dynamics of 1,976 lymphoid malignancies predict clinical outcome
Authors: Gabbutt, C.; Duran-Ferrer, M.; Grant, H.; Mallo, D.; Nadeu, F.; Househam, J.; Villamor, N.; Krali, O.; Nordlund, J.; Zenz, T.; Campo, E.; Lopez-Guillermo, A.; Fitzgibbon, J.; Barnes, C. P.; Shibata, D.; Martin-Subero, J. I.; Graham, T. A.
Score: 42.8, Published: 2023-11-10 DOI: 10.1101/2023.11.10.23298336
Cancer development, progression, and response to treatment are evolutionary processes, but characterising the evolutionary dynamics at sufficient scale to be clinically-meaningful has remained challenging."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "oncology",
      "item": "https://trxiv.yorks0n.com/posts/oncology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "oncology",
  "name": "oncology",
  "description": "Evolutionary dynamics of 1,976 lymphoid malignancies predict clinical outcome\nAuthors: Gabbutt, C.; Duran-Ferrer, M.; Grant, H.; Mallo, D.; Nadeu, F.; Househam, J.; Villamor, N.; Krali, O.; Nordlund, J.; Zenz, T.; Campo, E.; Lopez-Guillermo, A.; Fitzgibbon, J.; Barnes, C. P.; Shibata, D.; Martin-Subero, J. I.; Graham, T. A.\nScore: 42.8, Published: 2023-11-10 DOI: 10.1101/2023.11.10.23298336\nCancer development, progression, and response to treatment are evolutionary processes, but characterising the evolutionary dynamics at sufficient scale to be clinically-meaningful has remained challenging.",
  "keywords": [
    
  ],
  "articleBody": " Evolutionary dynamics of 1,976 lymphoid malignancies predict clinical outcome\nAuthors: Gabbutt, C.; Duran-Ferrer, M.; Grant, H.; Mallo, D.; Nadeu, F.; Househam, J.; Villamor, N.; Krali, O.; Nordlund, J.; Zenz, T.; Campo, E.; Lopez-Guillermo, A.; Fitzgibbon, J.; Barnes, C. P.; Shibata, D.; Martin-Subero, J. I.; Graham, T. A.\nScore: 42.8, Published: 2023-11-10 DOI: 10.1101/2023.11.10.23298336\nCancer development, progression, and response to treatment are evolutionary processes, but characterising the evolutionary dynamics at sufficient scale to be clinically-meaningful has remained challenging. Here, we develop a new methodology called EVOFLUx, based upon natural DNA methylation barcodes fluctuating over time, that quantitatively infers evolutionary dynamics using only a bulk tumour methylation profile as input. We apply EVOFLUx to 1,976 well-characterised lymphoid cancer samples spanning a broad spectrum of diseases and show that tumour growth rates, malignancy age and epimutation rates vary by orders of magnitude across disease types. We measure that subclonal selection occurs only infrequently within bulk samples and detect occasional examples of multiple independent primary tumours. Clinically, we observe that tumour growth rates are higher in more aggressive disease subtypes, and in two series of chronic lymphocytic leukaemia patients, evolutionary histories are independent prognostic factors. Phylogenetic analyses of longitudinal CLL samples using EVOFLUx detect the seeds of future Richter transformation many decades prior to presentation. We provide orthogonal verification of EVOFLUx inferences using additional genetic and clinical data. Collectively, we show how widely-available, low-cost bulk DNA methylation data precisely measures cancer evolutionary dynamics, and provides new insights into cancer biology and clinical behaviour.\nAn elevated rate of whole-genome duplication events in cancers from Black patients\nAuthors: Brown, L. M.; Hagenson, R. A.; Sheltzer, J. M.\nScore: 34.1, Published: 2023-11-10 DOI: 10.1101/2023.11.10.23298349\nIn the United States, Black individuals have higher rates of cancer mortality than any other racial or ethnic group. The sources of these significant racial disparities are not fully understood, and may include social, environmental, and genetic factors that influence cancer onset, diagnosis, and treatment. Here, we examined genomic data from several large-scale cancer patient cohorts to search for racial associations in chromosome copy number alterations. We found that tumors from Black patients were significantly more likely to exhibit whole-genome duplications (WGDs), a genomic event that enhances metastasis and aggressive disease, compared to tumors from white patients. Among patients with WGD-positive cancers, there was no significant difference in survival between Black and white patients, suggesting that the increased incidence of WGD events could contribute to the disparities in patient outcome. Genomic analysis identified several somatic alterations associated with WGD events that were consistent between Black and white populations, indicating that the increase in WGD events may be driven by environmental or epigenetic factors in Black patients. In total, these findings identify a class of genomic alterations that may influence racial disparities in cancer patient outcome. As cancers that have undergone WGD events exhibit unique genetic vulnerabilities, therapies that selectively target WGD-positive cancers may be particularly effective at treating aggressive malignancies in Black patients.\nRewiring of cortical glucose metabolism fuels human brain cancer growth\nAuthors: Scott, A. J.; Mittal, A.; Meghdadi, B.; Palavalasa, S.; Achreja, A.; O'Brien, A.; Kothari, A. U.; Zhou, W.; Xu, J.; Lin, A.; Wilder-Romans, K.; Edwards, D. M.; Wu, Z.; Feng, J.; Andren, A. C.; Zhang, L.; Tarnal, V.; Redic, K. A.; Qi, N.; Fischer, J.; Yang, E.; Regan, M. S.; Stopko, S. A.; Baquer, G.; Lawrence, T. S.; Venneti, S.; Agar, N. Y. R.; Lyssiotis, C. A.; Al-Holou, W. N.; Nagrath, D.; Wahl, D. R.\nScore: 3.9, Published: 2023-10-25 DOI: 10.1101/2023.10.24.23297489\nThe brain avidly consumes glucose to fuel neurophysiology. Cancers of the brain, such as glioblastoma (GBM), lose aspects of normal biology and gain the ability to proliferate and invade healthy tissue. How brain cancers rewire glucose utilization to fuel these processes is poorly understood. Here we perform infusions of 13C-labeled glucose into patients and mice with brain cancer to define the metabolic fates of glucose-derived carbon in tumor and cortex. By combining these measurements with quantitative metabolic flux analysis, we find that human cortex funnels glucose-derived carbons towards physiologic processes including TCA cycle oxidation and neurotransmitter synthesis. In contrast, brain cancers downregulate these physiologic processes, scavenge alternative carbon sources from the environment, and instead use glucose-derived carbons to produce molecules needed for proliferation and invasion. Targeting this metabolic rewiring in mice through dietary modulation selectively alters GBM metabolism and slows tumor growth. SignificanceThis study is the first to directly measure biosynthetic flux in both glioma and cortical tissue in human brain cancer patients. Brain tumors rewire glucose carbon utilization away from oxidation and neurotransmitter production towards biosynthesis to fuel growth. Blocking these metabolic adaptations with dietary interventions slows brain cancer growth with minimal effects on cortical metabolism.\nClinical validity and utility of circulating tumor DNA (ctDNA) testing in advanced non-small cell lung cancer (aNSCLC): a systematic literature review and meta-analysis\nAuthors: Chen, C.; Douglas, M. P.; Ragavan, M. V.; Phillips, K. A.; Jansen, J. P.\nScore: 2.2, Published: 2023-10-28 DOI: 10.1101/2023.10.27.23297657\nPurposeCirculating tumor DNA (ctDNA) testing has become a promising tool to guide first-line (1L) targeted treatment for advanced non-small cell lung cancer (aNSCLC). This study aims to estimate the clinical validity (CV) and clinical utility (CU) of ctDNA-based next-generation sequencing (NGS) for oncogenic driver mutations to inform 1L treatment decisions in aNSCLC through a systematic literature review and meta-analysis. MethodsA systematic literature search was conducted in PubMed/MEDLINE and Embase to identify randomized control trials or observational studies reporting CV/CU on ctDNA testing in patients with aNSCLC. Meta-analyses were performed using bivariate random-effects models to estimate pooled sensitivity and specificity. Progression-free/overall survival (PFS/OS) was summarized for CU studies. ResultsEighteen studies were identified: 17 CV only, 2 CU only, and 1 both. Thirteen studies were included for the meta-analysis on multi-gene detection. The overall sensitivity and specificity for ctDNA detection of any mutation were 0.69 (95% CI, 0.63-0.74) and 0.99 (95% CI, 0.97-1.00) respectively. However, sensitivity varied greatly by driver gene, ranging from 0.29 (95% CI, 0.13-0.53) for ROS1 to 0.77 (95% CI, 0.63-0.86) for KRAS. Two studies compared PFS with ctDNA versus tissue-based testing followed by 1L targeted therapy found no significant differences. One study reported OS curves on ctDNA-matched and tissue-matched therapies but no hazard ratios were provided. ConclusionctDNA testing demonstrated an overall acceptable diagnostic accuracy in aNSCLC patients, however, sensitivity varied greatly by driver mutation. Further research is needed, especially for uncommon driver mutations, to better understand the CU of ctDNA testing in guiding targeted treatments for aNSCLC.\n",
  "wordCount" : "1072",
  "inLanguage": "en",
  "datePublished": "2023-11-19T10:38:34Z",
  "dateModified": "2023-11-19T10:38:34Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/oncology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      oncology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 19, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.23298336">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.23298336" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.23298336">
        <p class="paperTitle">Evolutionary dynamics of 1,976 lymphoid malignancies predict clinical outcome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.23298336" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.23298336" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gabbutt, C.; Duran-Ferrer, M.; Grant, H.; Mallo, D.; Nadeu, F.; Househam, J.; Villamor, N.; Krali, O.; Nordlund, J.; Zenz, T.; Campo, E.; Lopez-Guillermo, A.; Fitzgibbon, J.; Barnes, C. P.; Shibata, D.; Martin-Subero, J. I.; Graham, T. A.</p>
        <p class="info">Score: 42.8, Published: 2023-11-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.23298336' target='https://doi.org/10.1101/2023.11.10.23298336'> 10.1101/2023.11.10.23298336</a></p>
        <p class="abstract">Cancer development, progression, and response to treatment are evolutionary processes, but characterising the evolutionary dynamics at sufficient scale to be clinically-meaningful has remained challenging. Here, we develop a new methodology called EVOFLUx, based upon natural DNA methylation barcodes fluctuating over time, that quantitatively infers evolutionary dynamics using only a bulk tumour methylation profile as input. We apply EVOFLUx to 1,976 well-characterised lymphoid cancer samples spanning a broad spectrum of diseases and show that tumour growth rates, malignancy age and epimutation rates vary by orders of magnitude across disease types. We measure that subclonal selection occurs only infrequently within bulk samples and detect occasional examples of multiple independent primary tumours. Clinically, we observe that tumour growth rates are higher in more aggressive disease subtypes, and in two series of chronic lymphocytic leukaemia patients, evolutionary histories are independent prognostic factors. Phylogenetic analyses of longitudinal CLL samples using EVOFLUx detect the seeds of future Richter transformation many decades prior to presentation. We provide orthogonal verification of EVOFLUx inferences using additional genetic and clinical data. Collectively, we show how widely-available, low-cost bulk DNA methylation data precisely measures cancer evolutionary dynamics, and provides new insights into cancer biology and clinical behaviour.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.23298349">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.23298349" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.23298349">
        <p class="paperTitle">An elevated rate of whole-genome duplication events in cancers from Black patients</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.23298349" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.23298349" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brown, L. M.; Hagenson, R. A.; Sheltzer, J. M.</p>
        <p class="info">Score: 34.1, Published: 2023-11-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.23298349' target='https://doi.org/10.1101/2023.11.10.23298349'> 10.1101/2023.11.10.23298349</a></p>
        <p class="abstract">In the United States, Black individuals have higher rates of cancer mortality than any other racial or ethnic group. The sources of these significant racial disparities are not fully understood, and may include social, environmental, and genetic factors that influence cancer onset, diagnosis, and treatment. Here, we examined genomic data from several large-scale cancer patient cohorts to search for racial associations in chromosome copy number alterations. We found that tumors from Black patients were significantly more likely to exhibit whole-genome duplications (WGDs), a genomic event that enhances metastasis and aggressive disease, compared to tumors from white patients. Among patients with WGD-positive cancers, there was no significant difference in survival between Black and white patients, suggesting that the increased incidence of WGD events could contribute to the disparities in patient outcome. Genomic analysis identified several somatic alterations associated with WGD events that were consistent between Black and white populations, indicating that the increase in WGD events may be driven by environmental or epigenetic factors in Black patients. In total, these findings identify a class of genomic alterations that may influence racial disparities in cancer patient outcome. As cancers that have undergone WGD events exhibit unique genetic vulnerabilities, therapies that selectively target WGD-positive cancers may be particularly effective at treating aggressive malignancies in Black patients.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.23297489">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.23297489" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.23297489">
        <p class="paperTitle">Rewiring of cortical glucose metabolism fuels human brain cancer growth</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.23297489" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.23297489" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Scott, A. J.; Mittal, A.; Meghdadi, B.; Palavalasa, S.; Achreja, A.; O&#39;Brien, A.; Kothari, A. U.; Zhou, W.; Xu, J.; Lin, A.; Wilder-Romans, K.; Edwards, D. M.; Wu, Z.; Feng, J.; Andren, A. C.; Zhang, L.; Tarnal, V.; Redic, K. A.; Qi, N.; Fischer, J.; Yang, E.; Regan, M. S.; Stopko, S. A.; Baquer, G.; Lawrence, T. S.; Venneti, S.; Agar, N. Y. R.; Lyssiotis, C. A.; Al-Holou, W. N.; Nagrath, D.; Wahl, D. R.</p>
        <p class="info">Score: 3.9, Published: 2023-10-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.23297489' target='https://doi.org/10.1101/2023.10.24.23297489'> 10.1101/2023.10.24.23297489</a></p>
        <p class="abstract">The brain avidly consumes glucose to fuel neurophysiology. Cancers of the brain, such as glioblastoma (GBM), lose aspects of normal biology and gain the ability to proliferate and invade healthy tissue. How brain cancers rewire glucose utilization to fuel these processes is poorly understood. Here we perform infusions of 13C-labeled glucose into patients and mice with brain cancer to define the metabolic fates of glucose-derived carbon in tumor and cortex. By combining these measurements with quantitative metabolic flux analysis, we find that human cortex funnels glucose-derived carbons towards physiologic processes including TCA cycle oxidation and neurotransmitter synthesis. In contrast, brain cancers downregulate these physiologic processes, scavenge alternative carbon sources from the environment, and instead use glucose-derived carbons to produce molecules needed for proliferation and invasion. Targeting this metabolic rewiring in mice through dietary modulation selectively alters GBM metabolism and slows tumor growth.

SignificanceThis study is the first to directly measure biosynthetic flux in both glioma and cortical tissue in human brain cancer patients. Brain tumors rewire glucose carbon utilization away from oxidation and neurotransmitter production towards biosynthesis to fuel growth. Blocking these metabolic adaptations with dietary interventions slows brain cancer growth with minimal effects on cortical metabolism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.23297657">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.23297657" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.23297657">
        <p class="paperTitle">Clinical validity and utility of circulating tumor DNA (ctDNA) testing in advanced non-small cell lung cancer (aNSCLC): a systematic literature review and meta-analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.23297657" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.23297657" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, C.; Douglas, M. P.; Ragavan, M. V.; Phillips, K. A.; Jansen, J. P.</p>
        <p class="info">Score: 2.2, Published: 2023-10-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.23297657' target='https://doi.org/10.1101/2023.10.27.23297657'> 10.1101/2023.10.27.23297657</a></p>
        <p class="abstract">PurposeCirculating tumor DNA (ctDNA) testing has become a promising tool to guide first-line (1L) targeted treatment for advanced non-small cell lung cancer (aNSCLC). This study aims to estimate the clinical validity (CV) and clinical utility (CU) of ctDNA-based next-generation sequencing (NGS) for oncogenic driver mutations to inform 1L treatment decisions in aNSCLC through a systematic literature review and meta-analysis.

MethodsA systematic literature search was conducted in PubMed/MEDLINE and Embase to identify randomized control trials or observational studies reporting CV/CU on ctDNA testing in patients with aNSCLC. Meta-analyses were performed using bivariate random-effects models to estimate pooled sensitivity and specificity. Progression-free/overall survival (PFS/OS) was summarized for CU studies.

ResultsEighteen studies were identified: 17 CV only, 2 CU only, and 1 both. Thirteen studies were included for the meta-analysis on multi-gene detection. The overall sensitivity and specificity for ctDNA detection of any mutation were 0.69 (95% CI, 0.63-0.74) and 0.99 (95% CI, 0.97-1.00) respectively. However, sensitivity varied greatly by driver gene, ranging from 0.29 (95% CI, 0.13-0.53) for ROS1 to 0.77 (95% CI, 0.63-0.86) for KRAS. Two studies compared PFS with ctDNA versus tissue-based testing followed by 1L targeted therapy found no significant differences. One study reported OS curves on ctDNA-matched and tissue-matched therapies but no hazard ratios were provided.

ConclusionctDNA testing demonstrated an overall acceptable diagnostic accuracy in aNSCLC patients, however, sensitivity varied greatly by driver mutation. Further research is needed, especially for uncommon driver mutations, to better understand the CU of ctDNA testing in guiding targeted treatments for aNSCLC.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
