<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>oncology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>oncology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Decoding pan-cancer treatment outcomes using multimodal real-world data and explainable artificial intelligence
Authors: Keyl, J.; Keyl, P.; Montavon, G.; Hosch, R.; Brehmer, A.; Mochmann, L.; Jurmeister, P.; Dernbach, G.; Kim, M.; Koitka, S.; Bauer, S.; Bechrakis, N.; Forsting, M.; Fuehrer-Sakel, D.; Glas, M.; Gruenwald, V.; Hadaschik, B.; Haubold, J.; Herrmann, K.; Kasper, S.; Kimmig, R.; Lang, S.; Rassaf, T.; Roesch, A.; Schadendorf, D.; Siveke, J. T.; Stuschke, M.; Sure, U.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/oncology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="oncology" />
<meta property="og:description" content="Decoding pan-cancer treatment outcomes using multimodal real-world data and explainable artificial intelligence
Authors: Keyl, J.; Keyl, P.; Montavon, G.; Hosch, R.; Brehmer, A.; Mochmann, L.; Jurmeister, P.; Dernbach, G.; Kim, M.; Koitka, S.; Bauer, S.; Bechrakis, N.; Forsting, M.; Fuehrer-Sakel, D.; Glas, M.; Gruenwald, V.; Hadaschik, B.; Haubold, J.; Herrmann, K.; Kasper, S.; Kimmig, R.; Lang, S.; Rassaf, T.; Roesch, A.; Schadendorf, D.; Siveke, J. T.; Stuschke, M.; Sure, U." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/oncology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-22T10:37:23+00:00" />
<meta property="article:modified_time" content="2023-10-22T10:37:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="oncology"/>
<meta name="twitter:description" content="Decoding pan-cancer treatment outcomes using multimodal real-world data and explainable artificial intelligence
Authors: Keyl, J.; Keyl, P.; Montavon, G.; Hosch, R.; Brehmer, A.; Mochmann, L.; Jurmeister, P.; Dernbach, G.; Kim, M.; Koitka, S.; Bauer, S.; Bechrakis, N.; Forsting, M.; Fuehrer-Sakel, D.; Glas, M.; Gruenwald, V.; Hadaschik, B.; Haubold, J.; Herrmann, K.; Kasper, S.; Kimmig, R.; Lang, S.; Rassaf, T.; Roesch, A.; Schadendorf, D.; Siveke, J. T.; Stuschke, M.; Sure, U."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "oncology",
      "item": "https://trxiv.yorks0n.com/posts/oncology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "oncology",
  "name": "oncology",
  "description": "Decoding pan-cancer treatment outcomes using multimodal real-world data and explainable artificial intelligence\nAuthors: Keyl, J.; Keyl, P.; Montavon, G.; Hosch, R.; Brehmer, A.; Mochmann, L.; Jurmeister, P.; Dernbach, G.; Kim, M.; Koitka, S.; Bauer, S.; Bechrakis, N.; Forsting, M.; Fuehrer-Sakel, D.; Glas, M.; Gruenwald, V.; Hadaschik, B.; Haubold, J.; Herrmann, K.; Kasper, S.; Kimmig, R.; Lang, S.; Rassaf, T.; Roesch, A.; Schadendorf, D.; Siveke, J. T.; Stuschke, M.; Sure, U.",
  "keywords": [
    
  ],
  "articleBody": " Decoding pan-cancer treatment outcomes using multimodal real-world data and explainable artificial intelligence\nAuthors: Keyl, J.; Keyl, P.; Montavon, G.; Hosch, R.; Brehmer, A.; Mochmann, L.; Jurmeister, P.; Dernbach, G.; Kim, M.; Koitka, S.; Bauer, S.; Bechrakis, N.; Forsting, M.; Fuehrer-Sakel, D.; Glas, M.; Gruenwald, V.; Hadaschik, B.; Haubold, J.; Herrmann, K.; Kasper, S.; Kimmig, R.; Lang, S.; Rassaf, T.; Roesch, A.; Schadendorf, D.; Siveke, J. T.; Stuschke, M.; Sure, U.; Totzeck, M.; Welt, A.; Wiesweg, M.; Baba, H. A.; Nensa, F.; Egger, J.; Mueller, K.-R.; Schuler, M.; Klauschen, F.; Kleesiek, J.\nScore: 3.0, Published: 2023-10-19 DOI: 10.1101/2023.10.12.23296873\nDespite advances in precision oncology, clinical decision-making still relies on limited parameters and expert knowledge. To address this limitation, we combined multimodal real-world data and explainable artificial intelligence (xAI) to introduce novel AI-derived (AID) markers for clinical decision support. We used deep learning to model the outcome of 15,726 patients across 38 solid cancer entities based on 350 markers, including clinical records, image-derived body compositions, and mutational tumor profiles. xAI determined the prognostic contribution of each clinical marker at the patient level and identified 114 key markers that accounted for 90% of the neural networks decision process. Moreover, xAI enabled us to uncover 1,373 prognostic interactions between markers. Our approach was validated in an independent cohort of 3,288 lung cancer patients from a US nationwide electronic health record-derived database. These results show the potential of xAI to transform the assessment of clinical parameters and enable personalized, data-driven cancer care.\nThe tumor multi-omic landscape of endometrial cancers developed on a germline genetic background of adiposity\nAuthors: Richenberg, G.; Francis, A.; Owen, C. N.; Gray, V.; Robinson, T.; Gabriel, A. A. G.; Lawrenson, K.; Crosbie, E. J.; Schildkraut, J. M.; Mckay, J. D.; Gaunt, T. R.; Relton, C. L.; Vincent, E. E.; Kar, S. P.\nScore: 7.5, Published: 2023-10-10 DOI: 10.1101/2023.10.09.23296765\nHigh body mass index (BMI) is a causal risk factor for endometrial cancer but the tumor molecular mechanisms affected by adiposity and their therapeutic relevance remain poorly understood. Here we characterize the tumor multi-omic landscape of endometrial cancers that have developed on a background of lifelong germline genetic exposure to elevated BMI. We built a polygenic score (PGS) for BMI in women using data on independent, genome-wide significant variants associated with adult BMI in 434,794 women. We performed germline (blood) genotype quality control and imputation on data from 354 endometrial cancer cases from The Cancer Genome Atlas (TCGA). We assigned each case in this TCGA cohort their genetically predicted life-course BMI based on the BMI PGS. Multivariable generalized linear models adjusted for age, stage, microsatellite status and genetic principal components were used to test for associations between the BMI germline PGS and endometrial cancer tumor genome-wide genomic, transcriptomic, proteomic, epigenomic and immune traits in TCGA. High BMI germline PGS was associated with (i) upregulated tumor gene expression in the IL6-JAK-STAT3 pathway (FDR=4.2x10-7); (ii) increased estimated intra-tumor activated mast cell infiltration (FDR=0.008); (iii) increased single base substitution (SBS) mutational signatures 1 (FDR=0.03) and 5 (FDR=0.09) and decreased SBS13 (FDR=0.09), implicating age-related and APOBEC mutagenesis, respectively; and (iv) decreased tumor EGFR protein expression (FDR=0.07). Alterations in IL6-JAK-STAT3 signaling gene and EGFR protein expression were, in turn, significantly associated with both overall survival and progression-free interval. Thus, we integrated germline and somatic data using a novel study design to identify associations between genetically predicted lifelong exposure to higher BMI and potentially actionable endometrial cancer tumor molecular features. These associations inform our understanding of how high BMI may influence the development and progression of this cancer, impacting endometrial tumor biology and clinical outcomes.\nUnderstanding the genetic epidemiology of hereditary breast cancer in India using whole genome data from 1029 healthy individuals\nAuthors: Vatsyayan, A.; Mathur, P.; Bhoyar, R. C.; Imran, M.; Senthivel, V.; Divakar, M. K.; Mishra, A.; Jolly, B.; Sivasubbu, S.; Scaria, V.\nScore: 0.8, Published: 2023-10-21 DOI: 10.1101/2023.10.20.23297296\nAim: Breast cancer is the most highly reported cancer in India as well as globally (Globocan 2020). Genetic testing could help tackle the increasing cancer burden by enabling carriers of pathogenic variants obtain an early diagnosis through increased surveillance, and help guide treatment, and in some cases enable prevention. However, accurate interpretation of variant pathogenicity must be established in a population-specific manner to ensure effective use of genetic testing. Further, since BRCA1 and BRCA2 are importance breast cancer genes, even rare variants must be studied for their potential effect on the disease. Materials and Methods: We query the IndiGen data obtained from whole genome sequencing of 1029 Indian individuals, and perform variant classification of all reported BRCA variants according to the gold-standard ACMG/AMP guidelines to establish disease epidemiology. We further implement machine learning approaches to classify all reported non-benign variants, and create a ready-reference of variants classified in a manner close to ACMG guidelines at scale. Results: We determined the genetic prevalence to be the following: 1 in nearly 341 individuals for BRCA1, and 1 in nearly 170 individuals for BRCA2 are likely to be carriers of pathogenic mutations. Overall, 1 in nearly 114 individuals are likely to be carriers of pathogenic BRCA mutations. Further, using the brca-NOVUS tool, we classified 1,54,045 genetic variants across 18 population sets and 4 large variant repositories as either pathogenic or benign. Conclusion: The high population prevalence indicates a greater need of studying genetic variants linked with breast cancer in an Indian population specific manner. To the best of our knowledge, this is the first and most comprehensive population-scale genetic epidemiological study of BRCA-linked breast cancer variants reported from India.\nUnderstanding the variant landscape, and genetic epidemiology of Multiple Endocrine Neoplasia in India\nAuthors: Vatsyayan, A.; Bhardwaj, J.; Agrawal, S. J.; Saikia, B. J.; VR, A.; Senthivel, V.; Trehan, S.; Pandhare, K.; Scaria, V.\nScore: 0.8, Published: 2023-10-21 DOI: 10.1101/2023.10.20.23297293\nAim: Multiple Endocrine Neoplasia (MEN) is a familial cancer syndrome that encompasses several different types of endocrine tumors. It has three main types, namely MEN1, MEN2 and MEN4 that may or may not overlap phenotypically, but are caused by genetic mutations in three different genes, namely RET, MEN1 and CDKN1B respectively. Genetic testing for effective diagnosis, improved prognosis, and treatment is recommended as part of of clinical practice guidelines, which makes establishment of accurate pathogenicity classification of variants across the three genes essential. However, few resources offer such classification, especially in a population specific manner. Materials and Methods: Using the gold-standard ACMG/AMP guidelines for variant classification, we have systematically classified variants reported across the RET, MEN1 and CDKN1B genes reported in the IndiGen dataset, and established the genetic epidemiology of MEN in the Indian population. We have additionally classified variants from ClinVar and Mastermind, and made all variant classifications freely accessible in the form of a database called MAPVar. Results: We have established the genetic prevalence of MEN in the Indian population to be the following: 1 in nearly 146 individuals are likely carriers of pathogenic RET mutations, and 1 in nearly 514 individuals are likely carriers of MEN1 pathogenic mutations. Overall, 1 in nearly 114 individuals are expected to be carriers of MEN-linked pathogenic mutations in the Indian population. We have compiled ACMG-classified variants from three large datasets to create an exhaustive compendium of MEN-linked variants called MEN-Associated Pathogenic Variants (MAPVar). The database is available at: https://clingen.igib.res.in/MAPVar/ Conclusion: Our work establishes that MEN is prevalent disorder in India, with MEN2 variants being the most reported of the three types. This indicates the need of more genomic studies of MEN variants to establish a more comprehensive variant landscape specific to Indian populations. Additionally, genetic testing is an effective tool used against MEN. Our resource MAPVar offers an exhaustive resource of ACMG-classified MEN variants, that can act as a ready resource to aid in interpretation of genetic testing results, as well as to better understanding genetic variants in clinical as well as research settings.\n",
  "wordCount" : "1301",
  "inLanguage": "en",
  "datePublished": "2023-10-22T10:37:23Z",
  "dateModified": "2023-10-22T10:37:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/oncology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      oncology
    </h1>
    <div class="post-meta"><span>updated on October 22, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.12.23296873">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.12.23296873" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.12.23296873">
        <p class="paperTitle">Decoding pan-cancer treatment outcomes using multimodal real-world data and explainable artificial intelligence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.12.23296873" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.12.23296873" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Keyl, J.; Keyl, P.; Montavon, G.; Hosch, R.; Brehmer, A.; Mochmann, L.; Jurmeister, P.; Dernbach, G.; Kim, M.; Koitka, S.; Bauer, S.; Bechrakis, N.; Forsting, M.; Fuehrer-Sakel, D.; Glas, M.; Gruenwald, V.; Hadaschik, B.; Haubold, J.; Herrmann, K.; Kasper, S.; Kimmig, R.; Lang, S.; Rassaf, T.; Roesch, A.; Schadendorf, D.; Siveke, J. T.; Stuschke, M.; Sure, U.; Totzeck, M.; Welt, A.; Wiesweg, M.; Baba, H. A.; Nensa, F.; Egger, J.; Mueller, K.-R.; Schuler, M.; Klauschen, F.; Kleesiek, J.</p>
        <p class="info">Score: 3.0, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.12.23296873' target='https://doi.org/10.1101/2023.10.12.23296873'> 10.1101/2023.10.12.23296873</a></p>
        <p class="abstract">Despite advances in precision oncology, clinical decision-making still relies on limited parameters and expert knowledge. To address this limitation, we combined multimodal real-world data and explainable artificial intelligence (xAI) to introduce novel AI-derived (AID) markers for clinical decision support.

We used deep learning to model the outcome of 15,726 patients across 38 solid cancer entities based on 350 markers, including clinical records, image-derived body compositions, and mutational tumor profiles. xAI determined the prognostic contribution of each clinical marker at the patient level and identified 114 key markers that accounted for 90% of the neural networks decision process. Moreover, xAI enabled us to uncover 1,373 prognostic interactions between markers. Our approach was validated in an independent cohort of 3,288 lung cancer patients from a US nationwide electronic health record-derived database.

These results show the potential of xAI to transform the assessment of clinical parameters and enable personalized, data-driven cancer care.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.23296765">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.23296765" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.23296765">
        <p class="paperTitle">The tumor multi-omic landscape of endometrial cancers developed on a germline genetic background of adiposity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.23296765" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.23296765" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Richenberg, G.; Francis, A.; Owen, C. N.; Gray, V.; Robinson, T.; Gabriel, A. A. G.; Lawrenson, K.; Crosbie, E. J.; Schildkraut, J. M.; Mckay, J. D.; Gaunt, T. R.; Relton, C. L.; Vincent, E. E.; Kar, S. P.</p>
        <p class="info">Score: 7.5, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.23296765' target='https://doi.org/10.1101/2023.10.09.23296765'> 10.1101/2023.10.09.23296765</a></p>
        <p class="abstract">High body mass index (BMI) is a causal risk factor for endometrial cancer but the tumor molecular mechanisms affected by adiposity and their therapeutic relevance remain poorly understood. Here we characterize the tumor multi-omic landscape of endometrial cancers that have developed on a background of lifelong germline genetic exposure to elevated BMI. We built a polygenic score (PGS) for BMI in women using data on independent, genome-wide significant variants associated with adult BMI in 434,794 women. We performed germline (blood) genotype quality control and imputation on data from 354 endometrial cancer cases from The Cancer Genome Atlas (TCGA). We assigned each case in this TCGA cohort their genetically predicted life-course BMI based on the BMI PGS. Multivariable generalized linear models adjusted for age, stage, microsatellite status and genetic principal components were used to test for associations between the BMI germline PGS and endometrial cancer tumor genome-wide genomic, transcriptomic, proteomic, epigenomic and immune traits in TCGA. High BMI germline PGS was associated with (i) upregulated tumor gene expression in the IL6-JAK-STAT3 pathway (FDR=4.2x10-7); (ii) increased estimated intra-tumor activated mast cell infiltration (FDR=0.008); (iii) increased single base substitution (SBS) mutational signatures 1 (FDR=0.03) and 5 (FDR=0.09) and decreased SBS13 (FDR=0.09), implicating age-related and APOBEC mutagenesis, respectively; and (iv) decreased tumor EGFR protein expression (FDR=0.07). Alterations in IL6-JAK-STAT3 signaling gene and EGFR protein expression were, in turn, significantly associated with both overall survival and progression-free interval. Thus, we integrated germline and somatic data using a novel study design to identify associations between genetically predicted lifelong exposure to higher BMI and potentially actionable endometrial cancer tumor molecular features. These associations inform our understanding of how high BMI may influence the development and progression of this cancer, impacting endometrial tumor biology and clinical outcomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.23297296">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.23297296" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.23297296">
        <p class="paperTitle">Understanding the genetic epidemiology of hereditary breast cancer in India using whole genome data from 1029 healthy individuals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.23297296" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.23297296" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vatsyayan, A.; Mathur, P.; Bhoyar, R. C.; Imran, M.; Senthivel, V.; Divakar, M. K.; Mishra, A.; Jolly, B.; Sivasubbu, S.; Scaria, V.</p>
        <p class="info">Score: 0.8, Published: 2023-10-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.23297296' target='https://doi.org/10.1101/2023.10.20.23297296'> 10.1101/2023.10.20.23297296</a></p>
        <p class="abstract">Aim: Breast cancer is the most highly reported cancer in India as well as globally (Globocan 2020). Genetic testing could help tackle the increasing cancer burden by enabling carriers of pathogenic variants obtain an early diagnosis through increased surveillance, and help guide treatment, and in some cases enable prevention. However, accurate interpretation of variant pathogenicity must be established in a population-specific manner to ensure effective use of genetic testing. Further, since BRCA1 and BRCA2 are importance breast cancer genes, even rare variants must be studied for their potential effect on the disease. Materials and Methods: We query the IndiGen data obtained from whole genome sequencing of 1029 Indian individuals, and perform variant classification of all reported BRCA variants according to the gold-standard ACMG/AMP guidelines to establish disease epidemiology. We further implement machine learning approaches to classify all reported non-benign variants, and create a ready-reference of variants classified in a manner close to ACMG guidelines at scale. Results: We determined the genetic prevalence to be the following: 1 in nearly 341 individuals for BRCA1, and 1 in nearly 170 individuals for BRCA2 are likely to be carriers of pathogenic mutations. Overall, 1 in nearly 114 individuals are likely to be carriers of pathogenic BRCA mutations. Further, using the brca-NOVUS tool, we classified 1,54,045 genetic variants across 18 population sets and 4 large variant repositories as either pathogenic or benign. Conclusion: The high population prevalence indicates a greater need of studying genetic variants linked with breast cancer in an Indian population specific manner. To the best of our knowledge, this is the first and most comprehensive population-scale genetic epidemiological study of BRCA-linked breast cancer variants reported from India.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.23297293">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.23297293" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.23297293">
        <p class="paperTitle">Understanding the variant landscape, and genetic epidemiology of Multiple Endocrine Neoplasia in India</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.23297293" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.23297293" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vatsyayan, A.; Bhardwaj, J.; Agrawal, S. J.; Saikia, B. J.; VR, A.; Senthivel, V.; Trehan, S.; Pandhare, K.; Scaria, V.</p>
        <p class="info">Score: 0.8, Published: 2023-10-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.23297293' target='https://doi.org/10.1101/2023.10.20.23297293'> 10.1101/2023.10.20.23297293</a></p>
        <p class="abstract">Aim: Multiple Endocrine Neoplasia (MEN) is a familial cancer syndrome that encompasses several different types of endocrine tumors. It has three main types, namely MEN1, MEN2 and MEN4 that may or may not overlap phenotypically, but are caused by genetic mutations in three different genes, namely RET, MEN1 and CDKN1B respectively. Genetic testing for effective diagnosis, improved prognosis, and treatment is recommended as part of of clinical practice guidelines, which makes establishment of accurate pathogenicity classification of variants across the three genes essential. However, few resources offer such classification, especially in a population specific manner. Materials and Methods: Using the gold-standard ACMG/AMP guidelines for variant classification, we have systematically classified variants reported across the RET, MEN1 and CDKN1B genes reported in the IndiGen dataset, and established the genetic epidemiology of MEN in the Indian population. We have additionally classified variants from ClinVar and Mastermind, and made all variant classifications freely accessible in the form of a database called MAPVar. Results: We have established the genetic prevalence of MEN in the Indian population to be the following: 1 in nearly 146 individuals are likely carriers of pathogenic RET mutations, and 1 in nearly 514 individuals are likely carriers of MEN1 pathogenic mutations. Overall, 1 in nearly 114 individuals are expected to be carriers of MEN-linked pathogenic mutations in the Indian population. We have compiled ACMG-classified variants from three large datasets to create an exhaustive compendium of MEN-linked variants called MEN-Associated Pathogenic Variants (MAPVar). The database is available at: https://clingen.igib.res.in/MAPVar/ Conclusion: Our work establishes that MEN is prevalent disorder in India, with MEN2 variants being the most reported of the three types. This indicates the need of more genomic studies of MEN variants to establish a more comprehensive variant landscape specific to Indian populations. Additionally, genetic testing is an effective tool used against MEN. Our resource MAPVar offers an exhaustive resource of ACMG-classified MEN variants, that can act as a ready resource to aid in interpretation of genetic testing results, as well as to better understanding genetic variants in clinical as well as research settings.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
