<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>oncology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>oncology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Spatial characterization and stratification of colorectal adenomas by Deep Visual Proteomics
Authors: Kabatnik, S.; Post, F.; Drici, L.; Snejbjerg Bartels, A.; Strauss, M. T.; Zheng, X.; Madsen, G. I.; Mund, A.; Rosenberger, F. A.; Moreira, J. M. A.; Mann, M.
Score: 15.3, Published: 2023-11-27 DOI: 10.1101/2023.11.25.23298987
Colorectal adenomas (CRA) are precursor lesions of the colon that may progress to adenocarcinomas, with patients currently categorized into risk groups by morphological features.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/oncology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="oncology" />
<meta property="og:description" content="Spatial characterization and stratification of colorectal adenomas by Deep Visual Proteomics
Authors: Kabatnik, S.; Post, F.; Drici, L.; Snejbjerg Bartels, A.; Strauss, M. T.; Zheng, X.; Madsen, G. I.; Mund, A.; Rosenberger, F. A.; Moreira, J. M. A.; Mann, M.
Score: 15.3, Published: 2023-11-27 DOI: 10.1101/2023.11.25.23298987
Colorectal adenomas (CRA) are precursor lesions of the colon that may progress to adenocarcinomas, with patients currently categorized into risk groups by morphological features." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/oncology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-03T10:38:01+00:00" />
<meta property="article:modified_time" content="2023-12-03T10:38:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="oncology"/>
<meta name="twitter:description" content="Spatial characterization and stratification of colorectal adenomas by Deep Visual Proteomics
Authors: Kabatnik, S.; Post, F.; Drici, L.; Snejbjerg Bartels, A.; Strauss, M. T.; Zheng, X.; Madsen, G. I.; Mund, A.; Rosenberger, F. A.; Moreira, J. M. A.; Mann, M.
Score: 15.3, Published: 2023-11-27 DOI: 10.1101/2023.11.25.23298987
Colorectal adenomas (CRA) are precursor lesions of the colon that may progress to adenocarcinomas, with patients currently categorized into risk groups by morphological features."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "oncology",
      "item": "https://trxiv.yorks0n.com/posts/oncology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "oncology",
  "name": "oncology",
  "description": "Spatial characterization and stratification of colorectal adenomas by Deep Visual Proteomics\nAuthors: Kabatnik, S.; Post, F.; Drici, L.; Snejbjerg Bartels, A.; Strauss, M. T.; Zheng, X.; Madsen, G. I.; Mund, A.; Rosenberger, F. A.; Moreira, J. M. A.; Mann, M.\nScore: 15.3, Published: 2023-11-27 DOI: 10.1101/2023.11.25.23298987\nColorectal adenomas (CRA) are precursor lesions of the colon that may progress to adenocarcinomas, with patients currently categorized into risk groups by morphological features.",
  "keywords": [
    
  ],
  "articleBody": " Spatial characterization and stratification of colorectal adenomas by Deep Visual Proteomics\nAuthors: Kabatnik, S.; Post, F.; Drici, L.; Snejbjerg Bartels, A.; Strauss, M. T.; Zheng, X.; Madsen, G. I.; Mund, A.; Rosenberger, F. A.; Moreira, J. M. A.; Mann, M.\nScore: 15.3, Published: 2023-11-27 DOI: 10.1101/2023.11.25.23298987\nColorectal adenomas (CRA) are precursor lesions of the colon that may progress to adenocarcinomas, with patients currently categorized into risk groups by morphological features. We aimed to establish a molecular feature-based risk allocation framework towards improved patient stratification. Deep Visual Proteomics (DVP) is a novel approach that combines image-based artificial intelligence with automated microdissection and ultra-high sensitive mass spectrometry. Here we used DVP on formalin-fixed, paraffin-embedded (FFPE) CRA tissues from nine patients, immunohistologically stained for Caudal-type homeobox 2 (CDX2), a protein implicated in colorectal cancer, enabling the characterization of cellular heterogeneity within distinct tissue regions and across patients. DVP seamlessly integrated with current pathology workflows and equipment, identifying DMBT1, MARCKS and CD99 as correlated with disease recurrence history, making them potential markers of risk stratification. DVP uncovered a metabolic switch towards anaerobic glycolysis in areas of high dysplasia, which was specific for cells with high CDX2 expression. Our findings underscore the potential of spatial proteomics to refine early-stage detection and contribute to personalized patient management strategies and provided novel insights into metabolic reprogramming.\nEvolutionary dynamics of 1,976 lymphoid malignancies predict clinical outcome\nAuthors: Gabbutt, C.; Duran-Ferrer, M.; Grant, H.; Mallo, D.; Nadeu, F.; Househam, J.; Villamor, N.; Krali, O.; Nordlund, J.; Zenz, T.; Campo, E.; Lopez-Guillermo, A.; Fitzgibbon, J.; Barnes, C. P.; Shibata, D.; Martin-Subero, J. I.; Graham, T. A.\nScore: 54.1, Published: 2023-11-20 DOI: 10.1101/2023.11.10.23298336\nCancer development, progression, and response to treatment are evolutionary processes, but characterising the evolutionary dynamics at sufficient scale to be clinically-meaningful has remained challenging. Here, we develop a new methodology called EVOFLUx, based upon natural DNA methylation barcodes fluctuating over time, that quantitatively infers evolutionary dynamics using only a bulk tumour methylation profile as input. We apply EVOFLUx to 1,976 well-characterised lymphoid cancer samples spanning a broad spectrum of diseases and show that tumour growth rates, malignancy age and epimutation rates vary by orders of magnitude across disease types. We measure that subclonal selection occurs only infrequently within bulk samples and detect occasional examples of multiple independent primary tumours. Clinically, we observe that tumour growth rates are higher in more aggressive disease subtypes, and in two series of chronic lymphocytic leukaemia patients, evolutionary histories are independent prognostic factors. Phylogenetic analyses of longitudinal CLL samples using EVOFLUx detect the seeds of future Richter transformation many decades prior to presentation. We provide orthogonal verification of EVOFLUx inferences using additional genetic and clinical data. Collectively, we show how widely-available, low-cost bulk DNA methylation data precisely measures cancer evolutionary dynamics, and provides new insights into cancer biology and clinical behaviour.\nVariability of mutational signatures is a footprint of carcinogens\nAuthors: Morrison, M. L.; Mange, L.; Senkin, S.; Rosenberg, N. A.; Foll, M.; Fernandez-Cuesta, L.; Alcala, N.\nScore: 3.9, Published: 2023-11-24 DOI: 10.1101/2023.11.23.23298821\nUnderstanding the genomic impact of carcinogens is fundamental to cancer biology and prevention. However, recent coordinated efforts to detect such fingerprints have been largely unsuccessful, challenging the paradigm that carcinogens induce identifiable mutational signatures. Here we introduce a new method based on statistics from population genetics, signature variability analysis (SVA), that elucidates both the diversity of tumorcausing processes and the heterogeneity of population carcinogen exposure. When we use SVA to re-analyze four prominent studies commonly cited as evidence of nonmutagenic carcinogens, we find that tumors induced by environmental carcinogens do possess mutational signature patterns that distinguish them from spontaneous tumors, even if a specific mutational signature cannot be detected. We find that, across cancers, organs, and model organisms, carcinogen exposure generally increases both the diversity of mutational signatures within a tumor and the homogeneity of signature activity across subjects. Importantly, we show that this increase in signature diversity, far from being a background effect, is associated with the geographic incidence of cancer and can facilitate the acquisition of cancer driver mutations. Our results both encourage a re-examination of the genomic impact of numerous substances and introduce new tools for the analysis of the genomic effects of other substances, potentially influencing carcinogen classifications and cancer prevention policies.\nAn elevated rate of whole-genome duplication events in cancers from Black patients\nAuthors: Brown, L. M.; Hagenson, R. A.; Sheltzer, J. M.\nScore: 36.1, Published: 2023-11-10 DOI: 10.1101/2023.11.10.23298349\nIn the United States, Black individuals have higher rates of cancer mortality than any other racial or ethnic group. The sources of these significant racial disparities are not fully understood, and may include social, environmental, and genetic factors that influence cancer onset, diagnosis, and treatment. Here, we examined genomic data from several large-scale cancer patient cohorts to search for racial associations in chromosome copy number alterations. We found that tumors from Black patients were significantly more likely to exhibit whole-genome duplications (WGDs), a genomic event that enhances metastasis and aggressive disease, compared to tumors from white patients. Among patients with WGD-positive cancers, there was no significant difference in survival between Black and white patients, suggesting that the increased incidence of WGD events could contribute to the disparities in patient outcome. Genomic analysis identified several somatic alterations associated with WGD events that were consistent between Black and white populations, indicating that the increase in WGD events may be driven by environmental or epigenetic factors in Black patients. In total, these findings identify a class of genomic alterations that may influence racial disparities in cancer patient outcome. As cancers that have undergone WGD events exhibit unique genetic vulnerabilities, therapies that selectively target WGD-positive cancers may be particularly effective at treating aggressive malignancies in Black patients.\nImpact of disruptions in antigen processing and presentation machinery on sarcoma\nAuthors: Renne, S. L.; Sama', L.; Kumar, S.; Mintemur, O.; Ruspi, L.; Santori, I.; Sicoli, F.; Bertuzzi, A.; Laffi, A.; Bonometti, A.; Colombo, P.; D'Amato, V.; Bressan, A.; Scorsetti, M.; Terracciano, L.; Navarria, P.; D'Incalci, M.; Quagliuolo, V.; Pasqualini, F.; Grizzi, F.; Cananzi, F. C. M.\nScore: 1.0, Published: 2023-12-01 DOI: 10.1101/2023.12.01.23299064\nBackground: The antigen processing machinery (APM) plays a critical role in generating tumor-specific antigens that can be recognized and targeted by the immune system. The status of the APM in sarcomas is not well characterized. Methods: We investigated 126 patients with 8 types of bone and soft tissue sarcoma operated between 2001-2021. Tissue microarrays mapped 11 specific areas in each case. The presence/absence of APM protein was determined through immunohistochemistry. Bayesian networks were used. Results: All investigated sarcomas had some defects in APM. The least damaged component was HLA Class I subunit {beta}2-microglobulin and HLA Class II. The proteasome LMP10 subunit was defective in leiomyosarcoma (LMS), myxoid liposarcoma (MLPS), and dedifferentiated liposarcoma (DDLPS), while MHC I transporting unit TAP2 was altered in undifferentiated pleomorphic sarcoma (UPS), gastrointestinal stromal tumor (GIST), and chordoma (CH). Among different neoplastic areas, high-grade areas showed different patterns of expression compared to high lymphocytic infiltrate areas. Heterogeneity at the patient level was also observed. Loss of any APM component was prognostic of distant metastasis (DM) for LMS and DDLPS and of overall survival (OS) for LMS. Conclusion: Sarcomas exhibit a high degree of defects in APM components, with differences among histotypes and tumoral areas. The most commonly altered APM components were HLA Class I subunit {beta}2-microglobulin, HLA Class I subunit (HC10), and MHC I transporting unit TAP2. The loss of APM components was prognostic of DM and OS and clinically relevant for LMS and DDLPS. This study explores sarcoma molecular mechanisms, enriching personalized therapeutic approaches.\n",
  "wordCount" : "1259",
  "inLanguage": "en",
  "datePublished": "2023-12-03T10:38:01Z",
  "dateModified": "2023-12-03T10:38:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/oncology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      oncology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 3, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.25.23298987">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.25.23298987" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.25.23298987">
        <p class="paperTitle">Spatial characterization and stratification of colorectal adenomas by Deep Visual Proteomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.25.23298987" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.25.23298987" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kabatnik, S.; Post, F.; Drici, L.; Snejbjerg Bartels, A.; Strauss, M. T.; Zheng, X.; Madsen, G. I.; Mund, A.; Rosenberger, F. A.; Moreira, J. M. A.; Mann, M.</p>
        <p class="info">Score: 15.3, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.25.23298987' target='https://doi.org/10.1101/2023.11.25.23298987'> 10.1101/2023.11.25.23298987</a></p>
        <p class="abstract">Colorectal adenomas (CRA) are precursor lesions of the colon that may progress to adenocarcinomas, with patients currently categorized into risk groups by morphological features. We aimed to establish a molecular feature-based risk allocation framework towards improved patient stratification. Deep Visual Proteomics (DVP) is a novel approach that combines image-based artificial intelligence with automated microdissection and ultra-high sensitive mass spectrometry. Here we used DVP on formalin-fixed, paraffin-embedded (FFPE) CRA tissues from nine patients, immunohistologically stained for Caudal-type homeobox 2 (CDX2), a protein implicated in colorectal cancer, enabling the characterization of cellular heterogeneity within distinct tissue regions and across patients. DVP seamlessly integrated with current pathology workflows and equipment, identifying DMBT1, MARCKS and CD99 as correlated with disease recurrence history, making them potential markers of risk stratification. DVP uncovered a metabolic switch towards anaerobic glycolysis in areas of high dysplasia, which was specific for cells with high CDX2 expression. Our findings underscore the potential of spatial proteomics to refine early-stage detection and contribute to personalized patient management strategies and provided novel insights into metabolic reprogramming.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.23298336">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.23298336" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.23298336">
        <p class="paperTitle">Evolutionary dynamics of 1,976 lymphoid malignancies predict clinical outcome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.23298336" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.23298336" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gabbutt, C.; Duran-Ferrer, M.; Grant, H.; Mallo, D.; Nadeu, F.; Househam, J.; Villamor, N.; Krali, O.; Nordlund, J.; Zenz, T.; Campo, E.; Lopez-Guillermo, A.; Fitzgibbon, J.; Barnes, C. P.; Shibata, D.; Martin-Subero, J. I.; Graham, T. A.</p>
        <p class="info">Score: 54.1, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.23298336' target='https://doi.org/10.1101/2023.11.10.23298336'> 10.1101/2023.11.10.23298336</a></p>
        <p class="abstract">Cancer development, progression, and response to treatment are evolutionary processes, but characterising the evolutionary dynamics at sufficient scale to be clinically-meaningful has remained challenging. Here, we develop a new methodology called EVOFLUx, based upon natural DNA methylation barcodes fluctuating over time, that quantitatively infers evolutionary dynamics using only a bulk tumour methylation profile as input. We apply EVOFLUx to 1,976 well-characterised lymphoid cancer samples spanning a broad spectrum of diseases and show that tumour growth rates, malignancy age and epimutation rates vary by orders of magnitude across disease types. We measure that subclonal selection occurs only infrequently within bulk samples and detect occasional examples of multiple independent primary tumours. Clinically, we observe that tumour growth rates are higher in more aggressive disease subtypes, and in two series of chronic lymphocytic leukaemia patients, evolutionary histories are independent prognostic factors. Phylogenetic analyses of longitudinal CLL samples using EVOFLUx detect the seeds of future Richter transformation many decades prior to presentation. We provide orthogonal verification of EVOFLUx inferences using additional genetic and clinical data. Collectively, we show how widely-available, low-cost bulk DNA methylation data precisely measures cancer evolutionary dynamics, and provides new insights into cancer biology and clinical behaviour.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.23298821">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.23298821" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.23298821">
        <p class="paperTitle">Variability of mutational signatures is a footprint of carcinogens</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.23298821" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.23298821" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Morrison, M. L.; Mange, L.; Senkin, S.; Rosenberg, N. A.; Foll, M.; Fernandez-Cuesta, L.; Alcala, N.</p>
        <p class="info">Score: 3.9, Published: 2023-11-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.23298821' target='https://doi.org/10.1101/2023.11.23.23298821'> 10.1101/2023.11.23.23298821</a></p>
        <p class="abstract">Understanding the genomic impact of carcinogens is fundamental to cancer biology and prevention. However, recent coordinated efforts to detect such fingerprints have been largely unsuccessful, challenging the paradigm that carcinogens induce identifiable mutational signatures. Here we introduce a new method based on statistics from population genetics, signature variability analysis (SVA), that elucidates both the diversity of tumorcausing processes and the heterogeneity of population carcinogen exposure. When we use SVA to re-analyze four prominent studies commonly cited as evidence of nonmutagenic carcinogens, we find that tumors induced by environmental carcinogens do possess mutational signature patterns that distinguish them from spontaneous tumors, even if a specific mutational signature cannot be detected. We find that, across cancers, organs, and model organisms, carcinogen exposure generally increases both the diversity of mutational signatures within a tumor and the homogeneity of signature activity across subjects. Importantly, we show that this increase in signature diversity, far from being a background effect, is associated with the geographic incidence of cancer and can facilitate the acquisition of cancer driver mutations. Our results both encourage a re-examination of the genomic impact of numerous substances and introduce new tools for the analysis of the genomic effects of other substances, potentially influencing carcinogen classifications and cancer prevention policies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.23298349">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.23298349" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.23298349">
        <p class="paperTitle">An elevated rate of whole-genome duplication events in cancers from Black patients</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.23298349" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.23298349" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brown, L. M.; Hagenson, R. A.; Sheltzer, J. M.</p>
        <p class="info">Score: 36.1, Published: 2023-11-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.23298349' target='https://doi.org/10.1101/2023.11.10.23298349'> 10.1101/2023.11.10.23298349</a></p>
        <p class="abstract">In the United States, Black individuals have higher rates of cancer mortality than any other racial or ethnic group. The sources of these significant racial disparities are not fully understood, and may include social, environmental, and genetic factors that influence cancer onset, diagnosis, and treatment. Here, we examined genomic data from several large-scale cancer patient cohorts to search for racial associations in chromosome copy number alterations. We found that tumors from Black patients were significantly more likely to exhibit whole-genome duplications (WGDs), a genomic event that enhances metastasis and aggressive disease, compared to tumors from white patients. Among patients with WGD-positive cancers, there was no significant difference in survival between Black and white patients, suggesting that the increased incidence of WGD events could contribute to the disparities in patient outcome. Genomic analysis identified several somatic alterations associated with WGD events that were consistent between Black and white populations, indicating that the increase in WGD events may be driven by environmental or epigenetic factors in Black patients. In total, these findings identify a class of genomic alterations that may influence racial disparities in cancer patient outcome. As cancers that have undergone WGD events exhibit unique genetic vulnerabilities, therapies that selectively target WGD-positive cancers may be particularly effective at treating aggressive malignancies in Black patients.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.23299064">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.23299064" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.23299064">
        <p class="paperTitle">Impact of disruptions in antigen processing and presentation machinery on sarcoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.23299064" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.23299064" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Renne, S. L.; Sama&#39;, L.; Kumar, S.; Mintemur, O.; Ruspi, L.; Santori, I.; Sicoli, F.; Bertuzzi, A.; Laffi, A.; Bonometti, A.; Colombo, P.; D&#39;Amato, V.; Bressan, A.; Scorsetti, M.; Terracciano, L.; Navarria, P.; D&#39;Incalci, M.; Quagliuolo, V.; Pasqualini, F.; Grizzi, F.; Cananzi, F. C. M.</p>
        <p class="info">Score: 1.0, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.23299064' target='https://doi.org/10.1101/2023.12.01.23299064'> 10.1101/2023.12.01.23299064</a></p>
        <p class="abstract">Background: The antigen processing machinery (APM) plays a critical role in generating tumor-specific antigens that can be recognized and targeted by the immune system. The status of the APM in sarcomas is not well characterized. Methods: We investigated 126 patients with 8 types of bone and soft tissue sarcoma operated between 2001-2021. Tissue microarrays mapped 11 specific areas in each case. The presence/absence of APM protein was determined through immunohistochemistry. Bayesian networks were used. Results: All investigated sarcomas had some defects in APM. The least damaged component was HLA Class I subunit {beta}2-microglobulin and HLA Class II. The proteasome LMP10 subunit was defective in leiomyosarcoma (LMS), myxoid liposarcoma (MLPS), and dedifferentiated liposarcoma (DDLPS), while MHC I transporting unit TAP2 was altered in undifferentiated pleomorphic sarcoma (UPS), gastrointestinal stromal tumor (GIST), and chordoma (CH). Among different neoplastic areas, high-grade areas showed different patterns of expression compared to high lymphocytic infiltrate areas. Heterogeneity at the patient level was also observed. Loss of any APM component was prognostic of distant metastasis (DM) for LMS and DDLPS and of overall survival (OS) for LMS. Conclusion: Sarcomas exhibit a high degree of defects in APM components, with differences among histotypes and tumoral areas. The most commonly altered APM components were HLA Class I subunit {beta}2-microglobulin, HLA Class I subunit  (HC10), and MHC I transporting unit TAP2. The loss of APM components was prognostic of DM and OS and clinically relevant for LMS and DDLPS. This study explores sarcoma molecular mechanisms, enriching personalized therapeutic approaches.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
