<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>oncology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>oncology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Mapping inherited genetic variation with opposite effects on autoimmune disease and cancer identifies candidate drug targets associated with the anti-tumor immune response
Authors: Chen, J.; Epstein, M. P.; Schildkraut, J. M.; Kar, S. P.
Score: 7.8, Published: 2023-12-28 DOI: 10.1101/2023.12.23.23300491
BackgroundGermline alleles near genes that encode certain immune checkpoints (CTLA4, CD200) are associated with autoimmune/autoinflammatory disease and cancer but in opposite directions. This motivates a systematic search for additional germline alleles which demonstrate this pattern with the aim of identifying potential cancer immunotherapeutic targets using human genetic evidence.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/oncology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="oncology" />
<meta property="og:description" content="Mapping inherited genetic variation with opposite effects on autoimmune disease and cancer identifies candidate drug targets associated with the anti-tumor immune response
Authors: Chen, J.; Epstein, M. P.; Schildkraut, J. M.; Kar, S. P.
Score: 7.8, Published: 2023-12-28 DOI: 10.1101/2023.12.23.23300491
BackgroundGermline alleles near genes that encode certain immune checkpoints (CTLA4, CD200) are associated with autoimmune/autoinflammatory disease and cancer but in opposite directions. This motivates a systematic search for additional germline alleles which demonstrate this pattern with the aim of identifying potential cancer immunotherapeutic targets using human genetic evidence." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/oncology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-03T10:39:43+00:00" />
<meta property="article:modified_time" content="2024-01-03T10:39:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="oncology"/>
<meta name="twitter:description" content="Mapping inherited genetic variation with opposite effects on autoimmune disease and cancer identifies candidate drug targets associated with the anti-tumor immune response
Authors: Chen, J.; Epstein, M. P.; Schildkraut, J. M.; Kar, S. P.
Score: 7.8, Published: 2023-12-28 DOI: 10.1101/2023.12.23.23300491
BackgroundGermline alleles near genes that encode certain immune checkpoints (CTLA4, CD200) are associated with autoimmune/autoinflammatory disease and cancer but in opposite directions. This motivates a systematic search for additional germline alleles which demonstrate this pattern with the aim of identifying potential cancer immunotherapeutic targets using human genetic evidence."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "oncology",
      "item": "https://trxiv.yorks0n.com/posts/oncology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "oncology",
  "name": "oncology",
  "description": "Mapping inherited genetic variation with opposite effects on autoimmune disease and cancer identifies candidate drug targets associated with the anti-tumor immune response\nAuthors: Chen, J.; Epstein, M. P.; Schildkraut, J. M.; Kar, S. P.\nScore: 7.8, Published: 2023-12-28 DOI: 10.1101/2023.12.23.23300491\nBackgroundGermline alleles near genes that encode certain immune checkpoints (CTLA4, CD200) are associated with autoimmune/autoinflammatory disease and cancer but in opposite directions. This motivates a systematic search for additional germline alleles which demonstrate this pattern with the aim of identifying potential cancer immunotherapeutic targets using human genetic evidence.",
  "keywords": [
    
  ],
  "articleBody": " Mapping inherited genetic variation with opposite effects on autoimmune disease and cancer identifies candidate drug targets associated with the anti-tumor immune response\nAuthors: Chen, J.; Epstein, M. P.; Schildkraut, J. M.; Kar, S. P.\nScore: 7.8, Published: 2023-12-28 DOI: 10.1101/2023.12.23.23300491\nBackgroundGermline alleles near genes that encode certain immune checkpoints (CTLA4, CD200) are associated with autoimmune/autoinflammatory disease and cancer but in opposite directions. This motivates a systematic search for additional germline alleles which demonstrate this pattern with the aim of identifying potential cancer immunotherapeutic targets using human genetic evidence. MethodsPairwise fixed effect cross-disorder meta-analyses combining genome-wide association studies (GWAS) for breast, prostate, ovarian and endometrial cancers (240,540 cases/317,000 controls) and seven autoimmune/autoinflammatory diseases (112,631 cases/895,386 controls) coupled with in silico follow-up. To ensure detection of alleles with opposite effects on cancer and autoimmune/autoinflammatory disease, the signs on the beta coefficients in the autoimmune/autoinflammatory GWAS were reversed prior to meta-analyses. ResultsMeta-analyses followed by linkage disequilibrium clumping identified 312 unique, independent lead variants with Pmeta\u003c5x10-8 associated with at least one of the cancer types at Pcancer\u003c10-3 and one of the autoimmune/autoinflammatory diseases at Pauto\u003c10-3. At each lead variant, the allele that conferred autoimmune/autoinflammatory disease risk was protective for cancer. Mapping each lead variant to its nearest gene as its putative functional target and focusing on genes with established immunological effects implicated 32 of the nearest genes. Tumor bulk RNA-Seq data highlighted that the tumor expression of 5/32 genes (IRF1, IKZF1, SPI1, SH2B3, LAT) were each strongly correlated (Spearmans {rho}\u003e0.5) with at least one intra-tumor T/myeloid cell infiltration marker (CD4, CD8A, CD11B, CD45) in every one of the cancer types. Tumor single-cell RNA-Seq data from all cancer types showed that the five genes were more likely to be expressed in intra-tumor immune versus malignant cells. The five lead SNPs corresponding to these genes were linked to them via expression quantitative trait locus mechanisms and at least one additional line of functional evidence. Proteins encoded by the genes were predicted to be druggable. ConclusionWe provide population-scale germline genetic and functional genomic evidence to support further evaluation of the proteins encoded by IRF1, IKZF1, SPI1, SH2B3, and LAT as possible targets for cancer immunotherapy.\nInvestigative needle core biopsies for multi-omics in Glioblastoma\nAuthors: Yu, K. K.; Basu, S.; Bacquer, G.; Ahn, R.; Gantchev, J.; Jindal, S.; Regan, M. S.; Abou-Mrad, Z.; Prabhu, M. C.; Williams, M. J.; D'Souza, A. D.; Malinowski, S. W.; Hopland, K.; Elhanati, Y.; Stopka, S. A.; Stortchevoi, A.; He, Z.; Sun, J.; Chen, Y.; Espejo, A.; Chow, K. H.; Yerrum, S.; Kao, P.-L.; Parker Kerrigan, B.; Norberg, L.; Nielsen, D.; Puduvalli, V. K.; Huse, J.; Beroukhim, R.; Kim, Y. S. B.; Goswami, S.; Boire, A.; Frisken, S.; Cima, M. J.; Holdoff, M.; Lucas, C.-H. G.; Bettegowda, C.; Levine, S. S.; Bale, T. A.; Brennan, C.; Reardon, D. A.; Lang, F. F.; Chiocca, E. A.; Ligon, K. L.;\nScore: 1.0, Published: 2023-12-31 DOI: 10.1101/2023.12.29.23300541\nGlioblastoma (GBM) is a primary brain cancer with an abysmal prognosis and few effective therapies. The ability to investigate the tumor microenvironment before and during treatment would greatly enhance both understanding of disease response and progression, as well as the delivery and impact of therapeutics. Stereotactic biopsies are a routine surgical procedure performed primarily for diagnostic histopathologic purposes. The role of investigative biopsies - tissue sampling for the purpose of understanding tumor microenvironmental responses to treatment using integrated multi-modal molecular analyses ( Multi-omics\") has yet to be defined. Secondly, it is unknown whether comparatively small tissue samples from brain biopsies can yield sufficient information with such methods. Here we adapt stereotactic needle core biopsy tissue in two separate patients. In the first patient with recurrent GBM we performed highly resolved multi-omics analysis methods including single cell RNA sequencing, spatial-transcriptomics, metabolomics, proteomics, phosphoproteomics, T-cell clonotype analysis, and MHC Class I immunopeptidomics from biopsy tissue that was obtained from a single procedure. In a second patient we analyzed multi-regional core biopsies to decipher spatial and genomic variance. We also investigated the utility of stereotactic biopsies as a method for generating patient derived xenograft models in a separate patient cohort. Dataset integration across modalities showed good correspondence between spatial modalities, highlighted immune cell associated metabolic pathways and revealed poor correlation between RNA expression and the tumor MHC Class I immunopeptidome. In conclusion, stereotactic needle biopsy cores are of sufficient quality to generate multi-omics data, provide data rich insight into a patients disease process and tumor immune microenvironment and can be of value in evaluating treatment responses. One sentence summaryIntegrative multi-omics analysis of stereotactic needle core biopsies in glioblastoma\nUnraveling the clinicopathological and molecular changes induced by neoadjuvant chemotherapy and endocrine therapy in hormone receptor-positive/HER2-low and HER2 0 breast cancer\nAuthors: Schettini, F.; Nucera, S.; Braso-Maristany, F.; De Santo, I.; Pascual, T.; Bergamino, M.; Galvan, P.; Conte, B.; Segui, E.; Garcia Fructuoso, I.; Gomez Bravo, R.; Rivera, P.; Belen Rodriguez, A.; Martinez Saez, O.; Ganau, S.; Sanfeliu, E.; Gonzalez Farre, B.; Vidal, M.; Adamo, B.; Cebrecos, I.; Mension, E.; Oses, G.; Jares, P.; Vidal Sicart, S.; Molla, M.; Munoz, M.; Prat, A.\nScore: 0.5, Published: 2023-12-29 DOI: 10.1101/2023.12.27.23299114\nBackgroundThe characterization and comparison of gene expression (GE) and intrinsic subtypes (IS) changes induced by neoadjuvant chemotherapy (NACT) and endocrine therapy (NET) in hormone receptor-positive(HR+)/HER2-low vs. HR+/HER2-0 breast cancer (BC) has not been conducted so far. Most evidence on the association of HER2 status with pathologic responses and prognosis in HR+/HER2-negative BC is controversial and restricted to NACT-treated disease. Similarly, a temporal heterogeneity in HER2 status has been described only with NACT. MethodsWe retrospectively recruited a consecutive cohort of 186 patients with stage I-IIIB HR+/HER2-negative BC treated with neoadjuvant therapy (NAT). Available diagnostic biopsies and surgical samples were characterized for main pathological features, PAM50 intrinsic subtypes (IS) and risk-of-relapse (ROR)-P score, and GE. Associations with pathologic complete response (pCR), residual cancer burden (RCB)-0/I, event-free survival (EFS) and overall survival (OS) based on HER2 status were assessed. Pre/post pathologic/molecular changes were analyzed in matched samples. ResultsThe HER2-low (62.9%) and HER2-0 (37.1%) cohorts did not differ significantly in main baseline features, treatments administered, breast conserving surgery (BCS), pCR and RCB-0/I rates, EFS and OS. NAT induced, regardless of HER2 status, a significant reduction of ER/PgR and Ki67, a downregulation of PAM50 proliferation- and luminal-related genes/signatures, an upregulation of selected immune genes and a shift towards less aggressive IS and lower ROR-P. Moreover, 25% of HER2-0 changed to HER2-low and 34% HER2-low became HER2-0. HER2 shifts were significant after NACT (p\u003c0.001), not NET (p=0.063), with consistent ERBB2 mRNA level dynamics. HER2 changes were not associated to EFS/OS. ConclusionsHER2 status changes after NAT in [~]1/4 of cases, mostly after NACT. Targeted adjuvant strategies should be investigated accordingly. Molecular downstaging with current chemo/endocrine agents and immunotherapy should not rely on HER2 immunohistochemical levels in HR+/HER2-negative BC. Instead, HER2-low-targeted approaches should be explored to pursue more effective and/or less toxic dimensional downstaging. HighlightsO_LIHormone receptor-positive (HR+)/HER2-low and HER2-0 breast cancer (BC) showed similar post-neoadjuvant surgical outcomes. C_LIO_LINeoadjuvant therapy (NAT) induced a shift towards less aggressive subtypes and ROR-P classes regardless of HER2 status. C_LIO_LIAll NAT strategies induced a downregulation of proliferation- and luminal biology-related genes, regardless of HER2 status. C_LIO_LINAT induced changes in HER2 status, with a discordance rate of 34% and HER2-low showing higher instability than HER2-0. C_LIO_LIHER2 status at baseline, after surgery and its dynamics were not significantly associated to long-term outcomes. C_LI\nIdentification of a serum proteomic biomarker panel using diagnosis specific ensemble learning and symptoms for early pancreatic cancer detection\nAuthors: Ney, A.; Nene, N. R.; Sedlak, E.; Acedo, P.; Blyuss, O.; Whitwell, H. J.; Costello, E.; Gentry-Maharaj, A.; Williams, N. R.; Menon, U.; Fusai, G. K.; Zaikin, A.; Pereira, S. P.\nScore: 0.5, Published: 2023-12-28 DOI: 10.1101/2023.12.24.23300505\nBACKGROUNDThe grim (\u003c10% 5-year) survival rates for pancreatic ductal adenocarcinoma (PDAC) are attributed to its complex intrinsic biology and most often late-stage detection. The overlap of symptoms with benign gastrointestinal conditions in early stage further complicates timely detection. The suboptimal diagnostic performance of carbohydrate antigen (CA) 19-9 and elevation in benign hyperbilirubinaemia undermine its reliability, leaving a notable absence of accurate diagnostic biomarkers. Using a selected patient cohort with benign pancreatic and biliary tract conditions we aimed to develop a biomarker signature capable of distinguishing patients with non-specific yet concerning clinical presentations, from those with PDAC. METHODS539 patient serum samples collected under the Accelerated Diagnosis of neuro Endocrine and Pancreatic TumourS (ADEPTS) study (benign disease controls and PDACs) and the UK Collaborative Trial of Ovarian Cancer Screening (UKCTOCS, healthy controls) were screened using the Olink Oncology II panel, supplemented with five in-house markers. 16 specialized base-learner classifiers were stacked to select and enhance biomarker performances and robustness in blinded samples. Each base-learner was constructed through cross-validation and recursive feature elimination in a discovery set comprising approximately two thirds of the ADEPTS and UKCTOCS samples and contrasted specific diagnosis with PDAC. RESULTSThe signature which was developed using diagnosis-specific ensemble learning demonstrated predictive capabilities outperforming CA19-9 and individual biomarkers in both discovery and validation sets. An AUC of 0.98 (95% CI 0.98 - 0.99) and sensitivity of 0.99 (95% CI 0.98 - 1) at 90% specificity was achieved with the ensemble method, which was significantly larger than the AUC of 0.79 (95% CI 0.66 - 0.91) and sensitivity 0.67 (95% CI 0.50 - 0.83), also at 90% specificity, for CA19- 9, in the discovery set (p=0.0016 and p=0.00050, respectively). During ensemble signature validation, an AUC of 0.95 (95% CI 0.91 - 0.99), sensitivity 0.86 (95% CI 0.68 - 1), was attained compared to an AUC of 0.80 (95% CI 0.66 - 0.93), sensitivity 0.65 (95% CI 0.48 - 0.56) at 90% specificity for CA19-9 alone (p=0.0082 and p=0.024, respectively). When validated only on the benign disease controls and PDACs collected from ADEPTS, the diagnostic-specific signature achieved an AUC of 0.96 (95% CI 0.92 - 0.99), sensitivity 0.82 (95% CI 0.64 - 0.95) at 90% specificity, which was still significantly higher than the performance for CA19-9 taken as a single predictor, AUC of 0.79 (95% CI 0.64-0.93) and sensitivity of 0.18 (95% CI 0.03 - 0.69) (p= 0.013 and p=0.0055, respectively). CONCLUSIONOur ensemble modelling technique outperformed CA19-9, individual biomarkers and prevailing algorithms in distinguishing patients with non-specific but concerning symptoms from those with PDAC, with implications for improving its early detection in individuals at risk.\n",
  "wordCount" : "1712",
  "inLanguage": "en",
  "datePublished": "2024-01-03T10:39:43Z",
  "dateModified": "2024-01-03T10:39:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/oncology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      oncology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 3, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.23300491">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.23300491" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.23300491">
        <p class="paperTitle">Mapping inherited genetic variation with opposite effects on autoimmune disease and cancer identifies candidate drug targets associated with the anti-tumor immune response</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.23300491" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.23300491" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, J.; Epstein, M. P.; Schildkraut, J. M.; Kar, S. P.</p>
        <p class="info">Score: 7.8, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.23300491' target='https://doi.org/10.1101/2023.12.23.23300491'> 10.1101/2023.12.23.23300491</a></p>
        <p class="abstract">BackgroundGermline alleles near genes that encode certain immune checkpoints (CTLA4, CD200) are associated with autoimmune/autoinflammatory disease and cancer but in opposite directions. This motivates a systematic search for additional germline alleles which demonstrate this pattern with the aim of identifying potential cancer immunotherapeutic targets using human genetic evidence.

MethodsPairwise fixed effect cross-disorder meta-analyses combining genome-wide association studies (GWAS) for breast, prostate, ovarian and endometrial cancers (240,540 cases/317,000 controls) and seven autoimmune/autoinflammatory diseases (112,631 cases/895,386 controls) coupled with in silico follow-up. To ensure detection of alleles with opposite effects on cancer and autoimmune/autoinflammatory disease, the signs on the beta coefficients in the autoimmune/autoinflammatory GWAS were reversed prior to meta-analyses.

ResultsMeta-analyses followed by linkage disequilibrium clumping identified 312 unique, independent lead variants with Pmeta&lt;5x10-8 associated with at least one of the cancer types at Pcancer&lt;10-3 and one of the autoimmune/autoinflammatory diseases at Pauto&lt;10-3. At each lead variant, the allele that conferred autoimmune/autoinflammatory disease risk was protective for cancer. Mapping each lead variant to its nearest gene as its putative functional target and focusing on genes with established immunological effects implicated 32 of the nearest genes. Tumor bulk RNA-Seq data highlighted that the tumor expression of 5/32 genes (IRF1, IKZF1, SPI1, SH2B3, LAT) were each strongly correlated (Spearmans {rho}&gt;0.5) with at least one intra-tumor T/myeloid cell infiltration marker (CD4, CD8A, CD11B, CD45) in every one of the cancer types. Tumor single-cell RNA-Seq data from all cancer types showed that the five genes were more likely to be expressed in intra-tumor immune versus malignant cells. The five lead SNPs corresponding to these genes were linked to them via expression quantitative trait locus mechanisms and at least one additional line of functional evidence. Proteins encoded by the genes were predicted to be druggable.

ConclusionWe provide population-scale germline genetic and functional genomic evidence to support further evaluation of the proteins encoded by IRF1, IKZF1, SPI1, SH2B3, and LAT as possible targets for cancer immunotherapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.23300541">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.23300541" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.23300541">
        <p class="paperTitle">Investigative needle core biopsies for multi-omics in Glioblastoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.23300541" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.23300541" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yu, K. K.; Basu, S.; Bacquer, G.; Ahn, R.; Gantchev, J.; Jindal, S.; Regan, M. S.; Abou-Mrad, Z.; Prabhu, M. C.; Williams, M. J.; D&#39;Souza, A. D.; Malinowski, S. W.; Hopland, K.; Elhanati, Y.; Stopka, S. A.; Stortchevoi, A.; He, Z.; Sun, J.; Chen, Y.; Espejo, A.; Chow, K. H.; Yerrum, S.; Kao, P.-L.; Parker Kerrigan, B.; Norberg, L.; Nielsen, D.; Puduvalli, V. K.; Huse, J.; Beroukhim, R.; Kim, Y. S. B.; Goswami, S.; Boire, A.; Frisken, S.; Cima, M. J.; Holdoff, M.; Lucas, C.-H. G.; Bettegowda, C.; Levine, S. S.; Bale, T. A.; Brennan, C.; Reardon, D. A.; Lang, F. F.; Chiocca, E. A.; Ligon, K. L.;</p>
        <p class="info">Score: 1.0, Published: 2023-12-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.23300541' target='https://doi.org/10.1101/2023.12.29.23300541'> 10.1101/2023.12.29.23300541</a></p>
        <p class="abstract">Glioblastoma (GBM) is a primary brain cancer with an abysmal prognosis and few effective therapies. The ability to investigate the tumor microenvironment before and during treatment would greatly enhance both understanding of disease response and progression, as well as the delivery and impact of therapeutics. Stereotactic biopsies are a routine surgical procedure performed primarily for diagnostic histopathologic purposes. The role of investigative biopsies - tissue sampling for the purpose of understanding tumor microenvironmental responses to treatment using integrated multi-modal molecular analyses ( Multi-omics&#34;) has yet to be defined. Secondly, it is unknown whether comparatively small tissue samples from brain biopsies can yield sufficient information with such methods. Here we adapt stereotactic needle core biopsy tissue in two separate patients. In the first patient with recurrent GBM we performed highly resolved multi-omics analysis methods including single cell RNA sequencing, spatial-transcriptomics, metabolomics, proteomics, phosphoproteomics, T-cell clonotype analysis, and MHC Class I immunopeptidomics from biopsy tissue that was obtained from a single procedure. In a second patient we analyzed multi-regional core biopsies to decipher spatial and genomic variance. We also investigated the utility of stereotactic biopsies as a method for generating patient derived xenograft models in a separate patient cohort. Dataset integration across modalities showed good correspondence between spatial modalities, highlighted immune cell associated metabolic pathways and revealed poor correlation between RNA expression and the tumor MHC Class I immunopeptidome. In conclusion, stereotactic needle biopsy cores are of sufficient quality to generate multi-omics data, provide data rich insight into a patients disease process and tumor immune microenvironment and can be of value in evaluating treatment responses.

One sentence summaryIntegrative multi-omics analysis of stereotactic needle core biopsies in glioblastoma</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.27.23299114">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.27.23299114" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.27.23299114">
        <p class="paperTitle">Unraveling the clinicopathological and molecular changes induced by neoadjuvant chemotherapy and endocrine therapy in hormone receptor-positive/HER2-low and HER2 0 breast cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.27.23299114" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.27.23299114" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schettini, F.; Nucera, S.; Braso-Maristany, F.; De Santo, I.; Pascual, T.; Bergamino, M.; Galvan, P.; Conte, B.; Segui, E.; Garcia Fructuoso, I.; Gomez Bravo, R.; Rivera, P.; Belen Rodriguez, A.; Martinez Saez, O.; Ganau, S.; Sanfeliu, E.; Gonzalez Farre, B.; Vidal, M.; Adamo, B.; Cebrecos, I.; Mension, E.; Oses, G.; Jares, P.; Vidal Sicart, S.; Molla, M.; Munoz, M.; Prat, A.</p>
        <p class="info">Score: 0.5, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.27.23299114' target='https://doi.org/10.1101/2023.12.27.23299114'> 10.1101/2023.12.27.23299114</a></p>
        <p class="abstract">BackgroundThe characterization and comparison of gene expression (GE) and intrinsic subtypes (IS) changes induced by neoadjuvant chemotherapy (NACT) and endocrine therapy (NET) in hormone receptor-positive(HR&#43;)/HER2-low vs. HR&#43;/HER2-0 breast cancer (BC) has not been conducted so far. Most evidence on the association of HER2 status with pathologic responses and prognosis in HR&#43;/HER2-negative BC is controversial and restricted to NACT-treated disease. Similarly, a temporal heterogeneity in HER2 status has been described only with NACT.

MethodsWe retrospectively recruited a consecutive cohort of 186 patients with stage I-IIIB HR&#43;/HER2-negative BC treated with neoadjuvant therapy (NAT). Available diagnostic biopsies and surgical samples were characterized for main pathological features, PAM50 intrinsic subtypes (IS) and risk-of-relapse (ROR)-P score, and GE. Associations with pathologic complete response (pCR), residual cancer burden (RCB)-0/I, event-free survival (EFS) and overall survival (OS) based on HER2 status were assessed. Pre/post pathologic/molecular changes were analyzed in matched samples.

ResultsThe HER2-low (62.9%) and HER2-0 (37.1%) cohorts did not differ significantly in main baseline features, treatments administered, breast conserving surgery (BCS), pCR and RCB-0/I rates, EFS and OS. NAT induced, regardless of HER2 status, a significant reduction of ER/PgR and Ki67, a downregulation of PAM50 proliferation- and luminal-related genes/signatures, an upregulation of selected immune genes and a shift towards less aggressive IS and lower ROR-P. Moreover, 25% of HER2-0 changed to HER2-low and 34% HER2-low became HER2-0. HER2 shifts were significant after NACT (p&lt;0.001), not NET (p=0.063), with consistent ERBB2 mRNA level dynamics. HER2 changes were not associated to EFS/OS.

ConclusionsHER2 status changes after NAT in [~]1/4 of cases, mostly after NACT. Targeted adjuvant strategies should be investigated accordingly. Molecular downstaging with current chemo/endocrine agents and immunotherapy should not rely on HER2 immunohistochemical levels in HR&#43;/HER2-negative BC. Instead, HER2-low-targeted approaches should be explored to pursue more effective and/or less toxic dimensional downstaging.

HighlightsO_LIHormone receptor-positive (HR&#43;)/HER2-low and HER2-0 breast cancer (BC) showed similar post-neoadjuvant surgical outcomes.
C_LIO_LINeoadjuvant therapy (NAT) induced a shift towards less aggressive subtypes and ROR-P classes regardless of HER2 status.
C_LIO_LIAll NAT strategies induced a downregulation of proliferation- and luminal biology-related genes, regardless of HER2 status.
C_LIO_LINAT induced changes in HER2 status, with a discordance rate of 34% and HER2-low showing higher instability than HER2-0.
C_LIO_LIHER2 status at baseline, after surgery and its dynamics were not significantly associated to long-term outcomes.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.24.23300505">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.24.23300505" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.24.23300505">
        <p class="paperTitle">Identification of a serum proteomic biomarker panel using diagnosis specific ensemble learning and symptoms for early pancreatic cancer detection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.24.23300505" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.24.23300505" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ney, A.; Nene, N. R.; Sedlak, E.; Acedo, P.; Blyuss, O.; Whitwell, H. J.; Costello, E.; Gentry-Maharaj, A.; Williams, N. R.; Menon, U.; Fusai, G. K.; Zaikin, A.; Pereira, S. P.</p>
        <p class="info">Score: 0.5, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.24.23300505' target='https://doi.org/10.1101/2023.12.24.23300505'> 10.1101/2023.12.24.23300505</a></p>
        <p class="abstract">BACKGROUNDThe grim (&lt;10% 5-year) survival rates for pancreatic ductal adenocarcinoma (PDAC) are attributed to its complex intrinsic biology and most often late-stage detection. The overlap of symptoms with benign gastrointestinal conditions in early stage further complicates timely detection. The suboptimal diagnostic performance of carbohydrate antigen (CA) 19-9 and elevation in benign hyperbilirubinaemia undermine its reliability, leaving a notable absence of accurate diagnostic biomarkers. Using a selected patient cohort with benign pancreatic and biliary tract conditions we aimed to develop a biomarker signature capable of distinguishing patients with non-specific yet concerning clinical presentations, from those with PDAC.

METHODS539 patient serum samples collected under the Accelerated Diagnosis of neuro Endocrine and Pancreatic TumourS (ADEPTS) study (benign disease controls and PDACs) and the UK Collaborative Trial of Ovarian Cancer Screening (UKCTOCS, healthy controls) were screened using the Olink Oncology II panel, supplemented with five in-house markers. 16 specialized base-learner classifiers were stacked to select and enhance biomarker performances and robustness in blinded samples. Each base-learner was constructed through cross-validation and recursive feature elimination in a discovery set comprising approximately two thirds of the ADEPTS and UKCTOCS samples and contrasted specific diagnosis with PDAC.

RESULTSThe signature which was developed using diagnosis-specific ensemble learning demonstrated predictive capabilities outperforming CA19-9 and individual biomarkers in both discovery and validation sets. An AUC of 0.98 (95% CI 0.98 - 0.99) and sensitivity of 0.99 (95% CI 0.98 - 1) at 90% specificity was achieved with the ensemble method, which was significantly larger than the AUC of 0.79 (95% CI 0.66 - 0.91) and sensitivity 0.67 (95% CI 0.50 - 0.83), also at 90% specificity, for CA19- 9, in the discovery set (p=0.0016 and p=0.00050, respectively). During ensemble signature validation, an AUC of 0.95 (95% CI 0.91 - 0.99), sensitivity 0.86 (95% CI 0.68 - 1), was attained compared to an AUC of 0.80 (95% CI 0.66 - 0.93), sensitivity 0.65 (95% CI 0.48 - 0.56) at 90% specificity for CA19-9 alone (p=0.0082 and p=0.024, respectively). When validated only on the benign disease controls and PDACs collected from ADEPTS, the diagnostic-specific signature achieved an AUC of 0.96 (95% CI 0.92 - 0.99), sensitivity 0.82 (95% CI 0.64 - 0.95) at 90% specificity, which was still significantly higher than the performance for CA19-9 taken as a single predictor, AUC of 0.79 (95% CI 0.64-0.93) and sensitivity of 0.18 (95% CI 0.03 - 0.69) (p= 0.013 and p=0.0055, respectively).

CONCLUSIONOur ensemble modelling technique outperformed CA19-9, individual biomarkers and prevailing algorithms in distinguishing patients with non-specific but concerning symptoms from those with PDAC, with implications for improving its early detection in individuals at risk.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
