<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>oncology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>oncology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Variant-Specific Landscape of Mutual Exclusivity Among BRAF, EGFR, and KRAS Oncogenes in Human Cancer
Authors: Vaeyens, F.; Hetzel, J.-P.; Mernberger, M.; Eggermont, C.; Olsen, C.; Maes, K.; Vlaeminck, J.; Hes, F.; Pichler, M.; Giron, P.; Timofeev, O.; Noeparast, M.
Score: 2.5, Published: 2023-10-22 DOI: 10.1101/2023.10.21.23297089
In this cross-sectional study, we report the findings of our investigation into the mutual exclusivity (ME) and co-occurrence (CO) patterns of BRAF, KRAS, and EGFR mutations in human cancer.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/oncology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="oncology" />
<meta property="og:description" content="Variant-Specific Landscape of Mutual Exclusivity Among BRAF, EGFR, and KRAS Oncogenes in Human Cancer
Authors: Vaeyens, F.; Hetzel, J.-P.; Mernberger, M.; Eggermont, C.; Olsen, C.; Maes, K.; Vlaeminck, J.; Hes, F.; Pichler, M.; Giron, P.; Timofeev, O.; Noeparast, M.
Score: 2.5, Published: 2023-10-22 DOI: 10.1101/2023.10.21.23297089
In this cross-sectional study, we report the findings of our investigation into the mutual exclusivity (ME) and co-occurrence (CO) patterns of BRAF, KRAS, and EGFR mutations in human cancer." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/oncology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-25T10:39:04+00:00" />
<meta property="article:modified_time" content="2023-10-25T10:39:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="oncology"/>
<meta name="twitter:description" content="Variant-Specific Landscape of Mutual Exclusivity Among BRAF, EGFR, and KRAS Oncogenes in Human Cancer
Authors: Vaeyens, F.; Hetzel, J.-P.; Mernberger, M.; Eggermont, C.; Olsen, C.; Maes, K.; Vlaeminck, J.; Hes, F.; Pichler, M.; Giron, P.; Timofeev, O.; Noeparast, M.
Score: 2.5, Published: 2023-10-22 DOI: 10.1101/2023.10.21.23297089
In this cross-sectional study, we report the findings of our investigation into the mutual exclusivity (ME) and co-occurrence (CO) patterns of BRAF, KRAS, and EGFR mutations in human cancer."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "oncology",
      "item": "https://trxiv.yorks0n.com/posts/oncology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "oncology",
  "name": "oncology",
  "description": "Variant-Specific Landscape of Mutual Exclusivity Among BRAF, EGFR, and KRAS Oncogenes in Human Cancer\nAuthors: Vaeyens, F.; Hetzel, J.-P.; Mernberger, M.; Eggermont, C.; Olsen, C.; Maes, K.; Vlaeminck, J.; Hes, F.; Pichler, M.; Giron, P.; Timofeev, O.; Noeparast, M.\nScore: 2.5, Published: 2023-10-22 DOI: 10.1101/2023.10.21.23297089\nIn this cross-sectional study, we report the findings of our investigation into the mutual exclusivity (ME) and co-occurrence (CO) patterns of BRAF, KRAS, and EGFR mutations in human cancer.",
  "keywords": [
    
  ],
  "articleBody": " Variant-Specific Landscape of Mutual Exclusivity Among BRAF, EGFR, and KRAS Oncogenes in Human Cancer\nAuthors: Vaeyens, F.; Hetzel, J.-P.; Mernberger, M.; Eggermont, C.; Olsen, C.; Maes, K.; Vlaeminck, J.; Hes, F.; Pichler, M.; Giron, P.; Timofeev, O.; Noeparast, M.\nScore: 2.5, Published: 2023-10-22 DOI: 10.1101/2023.10.21.23297089\nIn this cross-sectional study, we report the findings of our investigation into the mutual exclusivity (ME) and co-occurrence (CO) patterns of BRAF, KRAS, and EGFR mutations in human cancer. Our analysis acknowledges previously overlooked mutational subtypes with distinct clinical implications. Creating an automated R framework, we analyzed mutation data from 64807 unique cBioPortal samples, 1570 cell lines, and 2714 unique Belgian cancer samples. Consistently, across all three datasets, we observe that co-occurrence is less likely among class I BRAF, Hydrolysis KRAS, and Classical-like EGFR mutations. Bilateral variant-assigned CO matrices uncover novel inter-class and inter-type CO and ME scenarios, encompassing conventional and atypical mutations. Besides Class I BRAF, various mutation classes exhibit diverse CO patterns, justifying the need to refine mutational classifications. We provide a variant-specific database for precision oncology showcasing ME among three actionable oncogenes. These findings may guide the discovery of novel synthetically lethal interactions for targeted cancer therapy\nUnderstanding the genetic epidemiology of hereditary breast cancer in India using whole genome data from 1029 healthy individuals\nAuthors: Vatsyayan, A.; Mathur, P.; Bhoyar, R. C.; Imran, M.; Senthivel, V.; Divakar, M. K.; Mishra, A.; Jolly, B.; Sivasubbu, S.; Scaria, V.\nScore: 1.2, Published: 2023-10-21 DOI: 10.1101/2023.10.20.23297296\nAim: Breast cancer is the most highly reported cancer in India as well as globally (Globocan 2020). Genetic testing could help tackle the increasing cancer burden by enabling carriers of pathogenic variants obtain an early diagnosis through increased surveillance, and help guide treatment, and in some cases enable prevention. However, accurate interpretation of variant pathogenicity must be established in a population-specific manner to ensure effective use of genetic testing. Further, since BRCA1 and BRCA2 are importance breast cancer genes, even rare variants must be studied for their potential effect on the disease. Materials and Methods: We query the IndiGen data obtained from whole genome sequencing of 1029 Indian individuals, and perform variant classification of all reported BRCA variants according to the gold-standard ACMG/AMP guidelines to establish disease epidemiology. We further implement machine learning approaches to classify all reported non-benign variants, and create a ready-reference of variants classified in a manner close to ACMG guidelines at scale. Results: We determined the genetic prevalence to be the following: 1 in nearly 341 individuals for BRCA1, and 1 in nearly 170 individuals for BRCA2 are likely to be carriers of pathogenic mutations. Overall, 1 in nearly 114 individuals are likely to be carriers of pathogenic BRCA mutations. Further, using the brca-NOVUS tool, we classified 1,54,045 genetic variants across 18 population sets and 4 large variant repositories as either pathogenic or benign. Conclusion: The high population prevalence indicates a greater need of studying genetic variants linked with breast cancer in an Indian population specific manner. To the best of our knowledge, this is the first and most comprehensive population-scale genetic epidemiological study of BRCA-linked breast cancer variants reported from India.\nUnderstanding the variant landscape, and genetic epidemiology of Multiple Endocrine Neoplasia in India\nAuthors: Vatsyayan, A.; Bhardwaj, J.; Agrawal, S. J.; Saikia, B. J.; VR, A.; Senthivel, V.; Trehan, S.; Pandhare, K.; Scaria, V.\nScore: 1.0, Published: 2023-10-21 DOI: 10.1101/2023.10.20.23297293\nAimMultiple Endocrine Neoplasia (MEN) is a familial cancer syndrome that encompasses several different types of endocrine tumors. It has three main types, namely MEN1, MEN2 and MEN4 that may or may not overlap phenotypically, but are caused by genetic mutations in three different genes, namely RET, MEN1 and CDKN1B respectively. Genetic testing for effective diagnosis, improved prognosis, and treatment is recommended as part of of clinical practice guidelines, which makes establishment of accurate pathogenicity classification of variants across the three genes essential. However, few resources offer such classification, especially in a population specific manner. Materials and MethodsUsing the gold-standard ACMG/AMP guidelines for variant classification, we have systematically classified variants reported across the RET, MEN1 and CDKN1B genes reported in the IndiGen dataset, and established the genetic epidemiology of MEN in the Indian population. We have additionally classified variants from ClinVar and Mastermind, and made all variant classifications freely accessible in the form of a database called MAPVar. ResultsWe have established the genetic prevalence of MEN in the Indian population to be the following: 1 in nearly 146 individuals are likely carriers of pathogenic RET mutations, and 1 in nearly 514 individuals are likely carriers of MEN1 pathogenic mutations. Overall, 1 in nearly 114 individuals are expected to be carriers of MEN-linked pathogenic mutations in the Indian population. We have compiled ACMG-classified variants from three large datasets to create an exhaustive compendium of MEN-linked variants called MEN-Associated Pathogenic Variants (MAPVar). The database is available at: https://clingen.igib.res.in/MAPVar/ ConclusionOur work establishes that MEN is prevalent disorder in India, with MEN2 variants being the most reported of the three types. This indicates the need of more genomic studies of MEN variants to establish a more comprehensive variant landscape specific to Indian populations. Additionally, genetic testing is an effective tool used against MEN. Our resource MAPVar offers an exhaustive resource of ACMG-classified MEN variants, that can act as a ready resource to aid in interpretation of genetic testing results, as well as to better understanding genetic variants in clinical as well as research settings.\ncirculating tumor DNA in the immediate post-operative setting\nAuthors: Efthymiou, V.; Queenan, N.; Haas, M.; Naegele, S.; Goss, D.; Faden, D.\nScore: 1.2, Published: 2023-10-02 DOI: 10.1101/2023.09.30.23296390\nBackgroundCirculating tumor DNA (ctDNA) has emerged as an accurate real-time biomarker of disease status across most solid tumor types. Most studies evaluating the utility of ctDNA have focused on time points weeks to months after surgery, which for many cancer types, is significantly later than decision-making time points for adjuvant treatment. In this systematic review, we summarize the state of the literature on the feasibility of using ctDNA as a biomarker in the immediate postoperative period. MethodsWe performed a systematic review evaluating the early kinetics, defined here as three days, of ctDNA in patients who underwent curative-intent surgery across several cancer types. ResultsAmong the 2057 studies identified, we evaluated eight cohort studies with ctDNA levels measured within the first three days after surgery. Across six different cancer types, all studies showed an increased risk of cancer recurrence in patients with a positive early postoperative ctDNA level. DiscussionWhile ctDNA clearance kinetics appear to vary based on tumor type, across all studies-detectable ctDNA after surgery was predictive of recurrence, suggesting early post-operative timepoints could be feasibly used for determining minimal residual disease. However, larger studies need to be performed to better understand the precise kinetics of ctDNA clearance across different cancer types as well as to determine optimal postoperative time points. SynopsisThis systematic review analyzed the use of ctDNA as a biomarker for minimal residual disease detection in the early postoperative setting and found that ctDNA detection within three days after surgery is associated with an increased risk of recurrence.\n",
  "wordCount" : "1172",
  "inLanguage": "en",
  "datePublished": "2023-10-25T10:39:04Z",
  "dateModified": "2023-10-25T10:39:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/oncology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      oncology
    </h1>
    <div class="post-meta"><span>updated on October 25, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.21.23297089">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.21.23297089" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.21.23297089">
        <p class="paperTitle">Variant-Specific Landscape of Mutual Exclusivity Among BRAF, EGFR, and KRAS Oncogenes in Human Cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.21.23297089" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.21.23297089" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vaeyens, F.; Hetzel, J.-P.; Mernberger, M.; Eggermont, C.; Olsen, C.; Maes, K.; Vlaeminck, J.; Hes, F.; Pichler, M.; Giron, P.; Timofeev, O.; Noeparast, M.</p>
        <p class="info">Score: 2.5, Published: 2023-10-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.21.23297089' target='https://doi.org/10.1101/2023.10.21.23297089'> 10.1101/2023.10.21.23297089</a></p>
        <p class="abstract">In this cross-sectional study, we report the findings of our investigation into the mutual exclusivity (ME) and co-occurrence (CO) patterns of BRAF, KRAS, and EGFR mutations in human cancer. Our analysis acknowledges previously overlooked mutational subtypes with distinct clinical implications. Creating an automated R framework, we analyzed mutation data from 64807 unique cBioPortal samples, 1570 cell lines, and 2714 unique Belgian cancer samples. Consistently, across all three datasets, we observe that co-occurrence is less likely among class I BRAF, Hydrolysis KRAS, and Classical-like EGFR mutations. Bilateral variant-assigned CO matrices uncover novel inter-class and inter-type CO and ME scenarios, encompassing conventional and atypical mutations. Besides Class I BRAF, various mutation classes exhibit diverse CO patterns, justifying the need to refine mutational classifications. We provide a variant-specific database for precision oncology showcasing ME among three actionable oncogenes. These findings may guide the discovery of novel synthetically lethal interactions for targeted cancer therapy</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.23297296">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.23297296" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.23297296">
        <p class="paperTitle">Understanding the genetic epidemiology of hereditary breast cancer in India using whole genome data from 1029 healthy individuals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.23297296" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.23297296" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vatsyayan, A.; Mathur, P.; Bhoyar, R. C.; Imran, M.; Senthivel, V.; Divakar, M. K.; Mishra, A.; Jolly, B.; Sivasubbu, S.; Scaria, V.</p>
        <p class="info">Score: 1.2, Published: 2023-10-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.23297296' target='https://doi.org/10.1101/2023.10.20.23297296'> 10.1101/2023.10.20.23297296</a></p>
        <p class="abstract">Aim: Breast cancer is the most highly reported cancer in India as well as globally (Globocan 2020). Genetic testing could help tackle the increasing cancer burden by enabling carriers of pathogenic variants obtain an early diagnosis through increased surveillance, and help guide treatment, and in some cases enable prevention. However, accurate interpretation of variant pathogenicity must be established in a population-specific manner to ensure effective use of genetic testing. Further, since BRCA1 and BRCA2 are importance breast cancer genes, even rare variants must be studied for their potential effect on the disease. Materials and Methods: We query the IndiGen data obtained from whole genome sequencing of 1029 Indian individuals, and perform variant classification of all reported BRCA variants according to the gold-standard ACMG/AMP guidelines to establish disease epidemiology. We further implement machine learning approaches to classify all reported non-benign variants, and create a ready-reference of variants classified in a manner close to ACMG guidelines at scale. Results: We determined the genetic prevalence to be the following: 1 in nearly 341 individuals for BRCA1, and 1 in nearly 170 individuals for BRCA2 are likely to be carriers of pathogenic mutations. Overall, 1 in nearly 114 individuals are likely to be carriers of pathogenic BRCA mutations. Further, using the brca-NOVUS tool, we classified 1,54,045 genetic variants across 18 population sets and 4 large variant repositories as either pathogenic or benign. Conclusion: The high population prevalence indicates a greater need of studying genetic variants linked with breast cancer in an Indian population specific manner. To the best of our knowledge, this is the first and most comprehensive population-scale genetic epidemiological study of BRCA-linked breast cancer variants reported from India.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.23297293">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.23297293" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.23297293">
        <p class="paperTitle">Understanding the variant landscape, and genetic epidemiology of Multiple Endocrine Neoplasia in India</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.23297293" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.23297293" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vatsyayan, A.; Bhardwaj, J.; Agrawal, S. J.; Saikia, B. J.; VR, A.; Senthivel, V.; Trehan, S.; Pandhare, K.; Scaria, V.</p>
        <p class="info">Score: 1.0, Published: 2023-10-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.23297293' target='https://doi.org/10.1101/2023.10.20.23297293'> 10.1101/2023.10.20.23297293</a></p>
        <p class="abstract">AimMultiple Endocrine Neoplasia (MEN) is a familial cancer syndrome that encompasses several different types of endocrine tumors. It has three main types, namely MEN1, MEN2 and MEN4 that may or may not overlap phenotypically, but are caused by genetic mutations in three different genes, namely RET, MEN1 and CDKN1B respectively. Genetic testing for effective diagnosis, improved prognosis, and treatment is recommended as part of of clinical practice guidelines, which makes establishment of accurate pathogenicity classification of variants across the three genes essential. However, few resources offer such classification, especially in a population specific manner.

Materials and MethodsUsing the gold-standard ACMG/AMP guidelines for variant classification, we have systematically classified variants reported across the RET, MEN1 and CDKN1B genes reported in the IndiGen dataset, and established the genetic epidemiology of MEN in the Indian population. We have additionally classified variants from ClinVar and Mastermind, and made all variant classifications freely accessible in the form of a database called MAPVar.

ResultsWe have established the genetic prevalence of MEN in the Indian population to be the following: 1 in nearly 146 individuals are likely carriers of pathogenic RET mutations, and 1 in nearly 514 individuals are likely carriers of MEN1 pathogenic mutations. Overall, 1 in nearly 114 individuals are expected to be carriers of MEN-linked pathogenic mutations in the Indian population.

We have compiled ACMG-classified variants from three large datasets to create an exhaustive compendium of MEN-linked variants called MEN-Associated Pathogenic Variants (MAPVar). The database is available at: https://clingen.igib.res.in/MAPVar/

ConclusionOur work establishes that MEN is prevalent disorder in India, with MEN2 variants being the most reported of the three types. This indicates the need of more genomic studies of MEN variants to establish a more comprehensive variant landscape specific to Indian populations.

Additionally, genetic testing is an effective tool used against MEN. Our resource MAPVar offers an exhaustive resource of ACMG-classified MEN variants, that can act as a ready resource to aid in interpretation of genetic testing results, as well as to better understanding genetic variants in clinical as well as research settings.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.30.23296390">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.30.23296390" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.30.23296390">
        <p class="paperTitle">circulating tumor DNA in the immediate post-operative setting</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.30.23296390" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.30.23296390" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Efthymiou, V.; Queenan, N.; Haas, M.; Naegele, S.; Goss, D.; Faden, D.</p>
        <p class="info">Score: 1.2, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.30.23296390' target='https://doi.org/10.1101/2023.09.30.23296390'> 10.1101/2023.09.30.23296390</a></p>
        <p class="abstract">BackgroundCirculating tumor DNA (ctDNA) has emerged as an accurate real-time biomarker of disease status across most solid tumor types. Most studies evaluating the utility of ctDNA have focused on time points weeks to months after surgery, which for many cancer types, is significantly later than decision-making time points for adjuvant treatment. In this systematic review, we summarize the state of the literature on the feasibility of using ctDNA as a biomarker in the immediate postoperative period.

MethodsWe performed a systematic review evaluating the early kinetics, defined here as three days, of ctDNA in patients who underwent curative-intent surgery across several cancer types.

ResultsAmong the 2057 studies identified, we evaluated eight cohort studies with ctDNA levels measured within the first three days after surgery. Across six different cancer types, all studies showed an increased risk of cancer recurrence in patients with a positive early postoperative ctDNA level.

DiscussionWhile ctDNA clearance kinetics appear to vary based on tumor type, across all studies-detectable ctDNA after surgery was predictive of recurrence, suggesting early post-operative timepoints could be feasibly used for determining minimal residual disease. However, larger studies need to be performed to better understand the precise kinetics of ctDNA clearance across different cancer types as well as to determine optimal postoperative time points.

SynopsisThis systematic review analyzed the use of ctDNA as a biomarker for minimal residual disease detection in the early postoperative setting and found that ctDNA detection within three days after surgery is associated with an increased risk of recurrence.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
