<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>oncology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>oncology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Multi-omics profiling with untargeted proteomics for blood-based early detection of lung cancer
Authors: Koh, B.; Liu, M.; Almonte, R.; Ariad, D.; Bundalian, G.; Chan, J.; Choi, J.; Chou, W.-F.; Cuaresma, R.; Hoedt, E.; Hopper, L.; Hu, Y.; Jain, A.; Khaledian, E.; Khin, T.; Kokate, A.; Lee, J.-Y.; Leung, S.; Lin, C.-H.; Marispini, M.; Malekpour, H.; Mora, M.; Mudaliar, N.; Nouri Golmaei, S.; Ramaiah, M.; Ramaswamy, S.; Ranjan, P.; Shu, G.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/oncology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="oncology" />
<meta property="og:description" content="Multi-omics profiling with untargeted proteomics for blood-based early detection of lung cancer
Authors: Koh, B.; Liu, M.; Almonte, R.; Ariad, D.; Bundalian, G.; Chan, J.; Choi, J.; Chou, W.-F.; Cuaresma, R.; Hoedt, E.; Hopper, L.; Hu, Y.; Jain, A.; Khaledian, E.; Khin, T.; Kokate, A.; Lee, J.-Y.; Leung, S.; Lin, C.-H.; Marispini, M.; Malekpour, H.; Mora, M.; Mudaliar, N.; Nouri Golmaei, S.; Ramaiah, M.; Ramaswamy, S.; Ranjan, P.; Shu, G." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/oncology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-14T10:38:40+00:00" />
<meta property="article:modified_time" content="2024-01-14T10:38:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="oncology"/>
<meta name="twitter:description" content="Multi-omics profiling with untargeted proteomics for blood-based early detection of lung cancer
Authors: Koh, B.; Liu, M.; Almonte, R.; Ariad, D.; Bundalian, G.; Chan, J.; Choi, J.; Chou, W.-F.; Cuaresma, R.; Hoedt, E.; Hopper, L.; Hu, Y.; Jain, A.; Khaledian, E.; Khin, T.; Kokate, A.; Lee, J.-Y.; Leung, S.; Lin, C.-H.; Marispini, M.; Malekpour, H.; Mora, M.; Mudaliar, N.; Nouri Golmaei, S.; Ramaiah, M.; Ramaswamy, S.; Ranjan, P.; Shu, G."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "oncology",
      "item": "https://trxiv.yorks0n.com/posts/oncology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "oncology",
  "name": "oncology",
  "description": "Multi-omics profiling with untargeted proteomics for blood-based early detection of lung cancer\nAuthors: Koh, B.; Liu, M.; Almonte, R.; Ariad, D.; Bundalian, G.; Chan, J.; Choi, J.; Chou, W.-F.; Cuaresma, R.; Hoedt, E.; Hopper, L.; Hu, Y.; Jain, A.; Khaledian, E.; Khin, T.; Kokate, A.; Lee, J.-Y.; Leung, S.; Lin, C.-H.; Marispini, M.; Malekpour, H.; Mora, M.; Mudaliar, N.; Nouri Golmaei, S.; Ramaiah, M.; Ramaswamy, S.; Ranjan, P.; Shu, G.",
  "keywords": [
    
  ],
  "articleBody": " Multi-omics profiling with untargeted proteomics for blood-based early detection of lung cancer\nAuthors: Koh, B.; Liu, M.; Almonte, R.; Ariad, D.; Bundalian, G.; Chan, J.; Choi, J.; Chou, W.-F.; Cuaresma, R.; Hoedt, E.; Hopper, L.; Hu, Y.; Jain, A.; Khaledian, E.; Khin, T.; Kokate, A.; Lee, J.-Y.; Leung, S.; Lin, C.-H.; Marispini, M.; Malekpour, H.; Mora, M.; Mudaliar, N.; Nouri Golmaei, S.; Ramaiah, M.; Ramaswamy, S.; Ranjan, P.; Shu, G.; Spiro, P.; Ta, B.; Vitko, D.; Waiss, J.; Yanagihara, Z.; Zawada, R.; Zeng, J. Y.; Zhang, S.; Yee, J.; Blume, J. E.; Belthangady, C.; Wilcox, B.; Ma, P.\nScore: 29.5, Published: 2024-01-04 DOI: 10.1101/2024.01.03.24300798\nBlood-based approaches to detect early-stage cancer provide an opportunity to improve survival rates for lung cancer, the most lethal cancer world-wide. Multiple approaches for blood-based cancer detection using molecular analytes derived from individual omics (cell-free DNA, RNA transcripts, proteins, metabolites) have been developed and tested, generally showing significantly lower sensitivity for early-stage versus late-stage cancer. We hypothesized that an approach using multiple types of molecular analytes, including broad and untargeted coverage of proteins, could identify biomarkers that more directly reveal changes in gene expression and molecular phenotype in response to carcinogenesis to potentially improve detection of early-stage lung cancer. To that end, we designed and conducted one of the largest multi-omics, observational studies to date, enrolling 2513 case and control subjects. Multi-omics profiling detected 113,671 peptides corresponding to 8385 protein groups, 219,729 RNA transcripts, 71,756 RNA introns, and 1801 metabolites across all subject samples. We then developed a machine learning-based classifier for lung cancer detection comprising 682 of these multi-omics analytes. This multi-omics classifier demonstrated 89%, 80%, and 98-100% sensitivity for all-stage, stage I, and stage III-IV lung cancer, respectively, at 89% specificity in a validation set. The application of a multi-omics platform for discovery of blood-based disease biomarkers, including proteins and complementary molecular analytes, enables the noninvasive detection of early-stage lung cancer with the potential for downstaging at initial diagnosis and the improvement of clinical outcomes.\nObesity shapes selection for driver mutations in cancer\nAuthors: Tang, C.; Castillon, V. J.; Waters, M.; Fong, C.; Park, T.; Boscenco, S.; Kim, S.; Schultz, N.; Ostrovnaya, I.; Gusev, A.; Jee, J.; Reznik, E.\nScore: 10.3, Published: 2024-01-11 DOI: 10.1101/2024.01.10.24301114\nObesity is a leading risk factor for cancer, but whether obesity is linked to specific genomic subtypes of cancer is unknown. Here, we examined the relationship between obesity and tumor genotype in two large clinicogenomic corpora. Obesity was associated with specific driver mutations in lung adenocarcinoma, endometrial carcinoma, and cancers of unknown primary, independent of clinical covariates and genetic ancestry. Obesity is therefore a putative driver of etiologic heterogeneity across cancers.\nMachine learning enabled prediction of digital biomarkers from whole slide histopathology images\nAuthors: McCaw, Z. R.; Shcherbina, A.; Shah, Y.; Huang, D.; Elliott, S.; Szabo, P. M.; Dulken, B.; Holland, S.; Tagari, P.; Light, D.; Koller, D.; Probert, C.\nScore: 7.3, Published: 2024-01-09 DOI: 10.1101/2024.01.06.24300926\nCurrent predictive biomarkers generally leverage technologies such as immunohis-tochemistry or genetic analysis, which may require specialized equipment, be time-intensive to deploy, or incur human error. In this paper, we present an alternative approach for the development and deployment of a class of predictive biomarkers, leveraging deep learning on digital images of hematoxylin and eosin (H\u0026E)-stained biopsy samples to simultaneously predict a range of molecular factors that are relevant to treatment selection and response. Our framework begins with the training of a pan-solid tumor H\u0026E foundation model, which can generate a universal featurization of H\u0026E-stained tissue images. This featurization becomes the input to machine learning models that perform multi-target, pan-cancer imputation. For a set of 352 drug targets, we show the ability to predict with high accuracy: copy number amplifications, target RNA expression, and an RNA-derived \"amplification signature\" that captures the transcriptional consequences of an amplification event. We facilitate exploratory analyses by making broad predictions initially. Having identified the subset of biomarkers relevant to a patient population of interest, we develop specialized machine learning models, built on the same foundational featurization, which achieve even higher performance for key biomarkers in tumor types of interest. Moreover, our models are robust, generalizing with minimal loss of performance across different patient populations. By generating imputations from tile-level featurizations, we enable spatial overlays of molecular annotations on top of whole-slide images. These annotation maps provide a clear means of interpreting the histological correlates of our models predictions, and align with features identified by expert pathologist review. Overall, our work demonstrates a flexible and scalable framework for imputing molecular measurements from H\u0026E, providing a generalizable approach to the development and deployment of predictive biomarkers for targeted therapeutics in cancer.\nRedefining Clinical Hyperprogression: the Incidence, Clinical Implications, and Risk Factors of Hyperprogression in Non-Small Cell Lung Cancer Treated with Immunotherapy\nAuthors: Djunadi, T. A.; Oh, Y. J.; Lee, J.; Yu, J.; Chung, L. I.-Y.; Lee, Y.; Kim, L.; Hong, T.; Lee, S.; Shah, Z.; Park, J. H.; Yoon, S. M.; Chae, Y. K.\nScore: 1.2, Published: 2024-01-05 DOI: 10.1101/2024.01.04.23300116\nIntroductionImmune checkpoint inhibitors (ICIs) may be associated with hyperprogressive disease (HPD). However, there is currently no standardized definition of HPD, with its risk factors and clinical implications remaining unclear. We investigated HPD in lung cancer patients undergoing immunotherapy, aiming to redefine HPD, identify risk factors, and assess its impact on survival. MethodsClinical and radiologic data from 121 non-small cell lung cancer (NSCLC) patients with 136 immunotherapy cases were reviewed retrospectively. Three HPD definitions (Champiat et al., HPDc; Saada-Bouzid et al., HPDs; and Ferrara et al., HPDf) were employed. Additionally, all new measurable lesions on the post-treatment CT scan were incorporated in measuring the sum of longest diameters (SLD) to define modified HPD (mHPD). ResultsAmong the 121 patients, 4 (3.3%) had HPDc, 11 (9.1%) had HPDs, and none had HPDf. Adding all new measurable lesions increased HPD incidence by 5%-10% across definitions. Multivariate analysis revealed significantly lower progression-free survival (PFS) and overall survival (OS) for patients with HPDc (HR 5.25, p=0.001; HR 3.75, p=0.015) and HPDs (HR 3.74, p\u003c0.001; HR 3.46, p\u003c0.001) compared to those without. Patients with mHPD showed similarly poor survival outcomes as HPD patients. Liver metastasis at diagnosis was associated with HPDs, and a high tumor burden correlated with HPDc. ConclusionsThe incidence and risk factors of HPD varied with different definitions, but mHPD identified more cases with poor outcomes. This comprehensive approach may enhance the identification of at-risk patients and lead to a better understanding of HPD in lung cancer during immunotherapy.\nA high throughput blood-based assay for the early detection of pancreatic cancer\nAuthors: Montoya Mira, J. L.; Quentel, A.; Patel, R. K.; Keith, D.; Minnier, J.; David, L.; Esener, S. C.; Sears, R. C.; Lopez, C. D.; Sheppard, B. C.; Demirci, U.; Wong, M. H.; Fischer, J. M.\nScore: 0.8, Published: 2024-01-13 DOI: 10.1101/2024.01.12.24301220\nPancreatic ductal adenocarcinoma (PDAC) remains one of the deadliest cancers due in part to the cancer being diagnosed is at a late stage when effective treatment options are limited. Early detection of PDAC via liquid biopsy would revolutionize survival from the disease. To address the lack of effective non-invasive detection assays for PDAC, we developed a protease activity-based assay using a magnetic nanosensor (PAC-MANN). The PAC-MANN assay leverages protease activity in blood to amplify the signal of the target-probe based sensor. An initial screening revealed that the PAC-MANN assay could reliably differentiate patients with PDAC from healthy subjects and patients at high risk of PDAC. Finally, in two cohorts: training (n=145) and blinded validation (n=72), we demonstrated that the PAC-MANN assay had high specificity (86%) and sensitivity (78%) for detection of PDAC compared to healthy subjects. This performance was enhanced when combined with the current standard of care assay, CA19-9 (100% specificity, 84% sensitivity). Our results demonstrate a novel assay that is rapid, high-throughput, and requires low specimen volume, which may not only improve cancer detection but could be useful for monitoring of at-risk patients and could be deployed in low resource settings.\n",
  "wordCount" : "1320",
  "inLanguage": "en",
  "datePublished": "2024-01-14T10:38:40Z",
  "dateModified": "2024-01-14T10:38:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/oncology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      oncology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 14, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.24300798">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.24300798" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.24300798">
        <p class="paperTitle">Multi-omics profiling with untargeted proteomics for blood-based early detection of lung cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.24300798" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.24300798" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Koh, B.; Liu, M.; Almonte, R.; Ariad, D.; Bundalian, G.; Chan, J.; Choi, J.; Chou, W.-F.; Cuaresma, R.; Hoedt, E.; Hopper, L.; Hu, Y.; Jain, A.; Khaledian, E.; Khin, T.; Kokate, A.; Lee, J.-Y.; Leung, S.; Lin, C.-H.; Marispini, M.; Malekpour, H.; Mora, M.; Mudaliar, N.; Nouri Golmaei, S.; Ramaiah, M.; Ramaswamy, S.; Ranjan, P.; Shu, G.; Spiro, P.; Ta, B.; Vitko, D.; Waiss, J.; Yanagihara, Z.; Zawada, R.; Zeng, J. Y.; Zhang, S.; Yee, J.; Blume, J. E.; Belthangady, C.; Wilcox, B.; Ma, P.</p>
        <p class="info">Score: 29.5, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.24300798' target='https://doi.org/10.1101/2024.01.03.24300798'> 10.1101/2024.01.03.24300798</a></p>
        <p class="abstract">Blood-based approaches to detect early-stage cancer provide an opportunity to improve survival rates for lung cancer, the most lethal cancer world-wide. Multiple approaches for blood-based cancer detection using molecular analytes derived from individual  omics (cell-free DNA, RNA transcripts, proteins, metabolites) have been developed and tested, generally showing significantly lower sensitivity for early-stage versus late-stage cancer. We hypothesized that an approach using multiple types of molecular analytes, including broad and untargeted coverage of proteins, could identify biomarkers that more directly reveal changes in gene expression and molecular phenotype in response to carcinogenesis to potentially improve detection of early-stage lung cancer. To that end, we designed and conducted one of the largest multi-omics, observational studies to date, enrolling 2513 case and control subjects. Multi-omics profiling detected 113,671 peptides corresponding to 8385 protein groups, 219,729 RNA transcripts, 71,756 RNA introns, and 1801 metabolites across all subject samples. We then developed a machine learning-based classifier for lung cancer detection comprising 682 of these multi-omics analytes. This multi-omics classifier demonstrated 89%, 80%, and 98-100% sensitivity for all-stage, stage I, and stage III-IV lung cancer, respectively, at 89% specificity in a validation set. The application of a multi-omics platform for discovery of blood-based disease biomarkers, including proteins and complementary molecular analytes, enables the noninvasive detection of early-stage lung cancer with the potential for downstaging at initial diagnosis and the improvement of clinical outcomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.24301114">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.24301114" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.24301114">
        <p class="paperTitle">Obesity shapes selection for driver mutations in cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.24301114" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.24301114" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tang, C.; Castillon, V. J.; Waters, M.; Fong, C.; Park, T.; Boscenco, S.; Kim, S.; Schultz, N.; Ostrovnaya, I.; Gusev, A.; Jee, J.; Reznik, E.</p>
        <p class="info">Score: 10.3, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.24301114' target='https://doi.org/10.1101/2024.01.10.24301114'> 10.1101/2024.01.10.24301114</a></p>
        <p class="abstract">Obesity is a leading risk factor for cancer, but whether obesity is linked to specific genomic subtypes of cancer is unknown. Here, we examined the relationship between obesity and tumor genotype in two large clinicogenomic corpora. Obesity was associated with specific driver mutations in lung adenocarcinoma, endometrial carcinoma, and cancers of unknown primary, independent of clinical covariates and genetic ancestry. Obesity is therefore a putative driver of etiologic heterogeneity across cancers.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.06.24300926">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.06.24300926" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.06.24300926">
        <p class="paperTitle">Machine learning enabled prediction of digital biomarkers from whole slide histopathology images</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.06.24300926" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.06.24300926" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McCaw, Z. R.; Shcherbina, A.; Shah, Y.; Huang, D.; Elliott, S.; Szabo, P. M.; Dulken, B.; Holland, S.; Tagari, P.; Light, D.; Koller, D.; Probert, C.</p>
        <p class="info">Score: 7.3, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.06.24300926' target='https://doi.org/10.1101/2024.01.06.24300926'> 10.1101/2024.01.06.24300926</a></p>
        <p class="abstract">Current predictive biomarkers generally leverage technologies such as immunohis-tochemistry or genetic analysis, which may require specialized equipment, be time-intensive to deploy, or incur human error. In this paper, we present an alternative approach for the development and deployment of a class of predictive biomarkers, leveraging deep learning on digital images of hematoxylin and eosin (H&amp;E)-stained biopsy samples to simultaneously predict a range of molecular factors that are relevant to treatment selection and response. Our framework begins with the training of a pan-solid tumor H&amp;E foundation model, which can generate a universal featurization of H&amp;E-stained tissue images. This featurization becomes the input to machine learning models that perform multi-target, pan-cancer imputation. For a set of 352 drug targets, we show the ability to predict with high accuracy: copy number amplifications, target RNA expression, and an RNA-derived &#34;amplification signature&#34; that captures the transcriptional consequences of an amplification event. We facilitate exploratory analyses by making broad predictions initially. Having identified the subset of biomarkers relevant to a patient population of interest, we develop specialized machine learning models, built on the same foundational featurization, which achieve even higher performance for key biomarkers in tumor types of interest. Moreover, our models are robust, generalizing with minimal loss of performance across different patient populations. By generating imputations from tile-level featurizations, we enable spatial overlays of molecular annotations on top of whole-slide images. These annotation maps provide a clear means of interpreting the histological correlates of our models predictions, and align with features identified by expert pathologist review. Overall, our work demonstrates a flexible and scalable framework for imputing molecular measurements from H&amp;E, providing a generalizable approach to the development and deployment of predictive biomarkers for targeted therapeutics in cancer.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.23300116">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.23300116" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.23300116">
        <p class="paperTitle">Redefining Clinical Hyperprogression: the Incidence, Clinical Implications, and Risk Factors of Hyperprogression in Non-Small Cell Lung Cancer Treated with Immunotherapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.23300116" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.23300116" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Djunadi, T. A.; Oh, Y. J.; Lee, J.; Yu, J.; Chung, L. I.-Y.; Lee, Y.; Kim, L.; Hong, T.; Lee, S.; Shah, Z.; Park, J. H.; Yoon, S. M.; Chae, Y. K.</p>
        <p class="info">Score: 1.2, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.23300116' target='https://doi.org/10.1101/2024.01.04.23300116'> 10.1101/2024.01.04.23300116</a></p>
        <p class="abstract">IntroductionImmune checkpoint inhibitors (ICIs) may be associated with hyperprogressive disease (HPD). However, there is currently no standardized definition of HPD, with its risk factors and clinical implications remaining unclear. We investigated HPD in lung cancer patients undergoing immunotherapy, aiming to redefine HPD, identify risk factors, and assess its impact on survival.

MethodsClinical and radiologic data from 121 non-small cell lung cancer (NSCLC) patients with 136 immunotherapy cases were reviewed retrospectively. Three HPD definitions (Champiat et al., HPDc; Saada-Bouzid et al., HPDs; and Ferrara et al., HPDf) were employed. Additionally, all new measurable lesions on the post-treatment CT scan were incorporated in measuring the sum of longest diameters (SLD) to define modified HPD (mHPD).

ResultsAmong the 121 patients, 4 (3.3%) had HPDc, 11 (9.1%) had HPDs, and none had HPDf. Adding all new measurable lesions increased HPD incidence by 5%-10% across definitions. Multivariate analysis revealed significantly lower progression-free survival (PFS) and overall survival (OS) for patients with HPDc (HR 5.25, p=0.001; HR 3.75, p=0.015) and HPDs (HR 3.74, p&lt;0.001; HR 3.46, p&lt;0.001) compared to those without. Patients with mHPD showed similarly poor survival outcomes as HPD patients. Liver metastasis at diagnosis was associated with HPDs, and a high tumor burden correlated with HPDc.

ConclusionsThe incidence and risk factors of HPD varied with different definitions, but mHPD identified more cases with poor outcomes. This comprehensive approach may enhance the identification of at-risk patients and lead to a better understanding of HPD in lung cancer during immunotherapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.24301220">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.24301220" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.24301220">
        <p class="paperTitle">A high throughput blood-based assay for the early detection of pancreatic cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.24301220" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.24301220" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Montoya Mira, J. L.; Quentel, A.; Patel, R. K.; Keith, D.; Minnier, J.; David, L.; Esener, S. C.; Sears, R. C.; Lopez, C. D.; Sheppard, B. C.; Demirci, U.; Wong, M. H.; Fischer, J. M.</p>
        <p class="info">Score: 0.8, Published: 2024-01-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.24301220' target='https://doi.org/10.1101/2024.01.12.24301220'> 10.1101/2024.01.12.24301220</a></p>
        <p class="abstract">Pancreatic ductal adenocarcinoma (PDAC) remains one of the deadliest cancers due in part to the cancer being diagnosed is at a late stage when effective treatment options are limited. Early detection of PDAC via liquid biopsy would revolutionize survival from the disease. To address the lack of effective non-invasive detection assays for PDAC, we developed a protease activity-based assay using a magnetic nanosensor (PAC-MANN). The PAC-MANN assay leverages protease activity in blood to amplify the signal of the target-probe based sensor. An initial screening revealed that the PAC-MANN assay could reliably differentiate patients with PDAC from healthy subjects and patients at high risk of PDAC. Finally, in two cohorts: training (n=145) and blinded validation (n=72), we demonstrated that the PAC-MANN assay had high specificity (86%) and sensitivity (78%) for detection of PDAC compared to healthy subjects. This performance was enhanced when combined with the current standard of care assay, CA19-9 (100% specificity, 84% sensitivity). Our results demonstrate a novel assay that is rapid, high-throughput, and requires low specimen volume, which may not only improve cancer detection but could be useful for monitoring of at-risk patients and could be deployed in low resource settings.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
