<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>oncology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>oncology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Non-invasive multi-cancer diagnosis using DNA hypomethylation of LINE-1 retrotransposons
Authors: Michel, M.; Heidary, M.; Mechri, A.; Da Silva, K.; Gorse, M.; von Grafenstein, K.; Hego, C.; Rampanou, A.; Lamy, C.; Kamal, M.; Le Tourneau, C.; Sene, M.; Bieche, I.; Reyes, C.; Gentien, D.; Stern, M.-H.; Lantz, O.; Cabel, L.; Pierga, J.-Y.; Bidard, F.-C.; Azencott, C.-A.; Proudhon, C.
Score: 3.0, Published: 2024-01-23 DOI: 10.1101/2024.01.20.23288905
The detection of circulating tumor DNA, which allows non-invasive tumor molecular profiling and disease follow-up, promises optimal and individualized management of patients with cancer.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/oncology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="oncology" />
<meta property="og:description" content="Non-invasive multi-cancer diagnosis using DNA hypomethylation of LINE-1 retrotransposons
Authors: Michel, M.; Heidary, M.; Mechri, A.; Da Silva, K.; Gorse, M.; von Grafenstein, K.; Hego, C.; Rampanou, A.; Lamy, C.; Kamal, M.; Le Tourneau, C.; Sene, M.; Bieche, I.; Reyes, C.; Gentien, D.; Stern, M.-H.; Lantz, O.; Cabel, L.; Pierga, J.-Y.; Bidard, F.-C.; Azencott, C.-A.; Proudhon, C.
Score: 3.0, Published: 2024-01-23 DOI: 10.1101/2024.01.20.23288905
The detection of circulating tumor DNA, which allows non-invasive tumor molecular profiling and disease follow-up, promises optimal and individualized management of patients with cancer." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/oncology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-31T10:38:28+00:00" />
<meta property="article:modified_time" content="2024-01-31T10:38:28+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="oncology"/>
<meta name="twitter:description" content="Non-invasive multi-cancer diagnosis using DNA hypomethylation of LINE-1 retrotransposons
Authors: Michel, M.; Heidary, M.; Mechri, A.; Da Silva, K.; Gorse, M.; von Grafenstein, K.; Hego, C.; Rampanou, A.; Lamy, C.; Kamal, M.; Le Tourneau, C.; Sene, M.; Bieche, I.; Reyes, C.; Gentien, D.; Stern, M.-H.; Lantz, O.; Cabel, L.; Pierga, J.-Y.; Bidard, F.-C.; Azencott, C.-A.; Proudhon, C.
Score: 3.0, Published: 2024-01-23 DOI: 10.1101/2024.01.20.23288905
The detection of circulating tumor DNA, which allows non-invasive tumor molecular profiling and disease follow-up, promises optimal and individualized management of patients with cancer."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "oncology",
      "item": "https://trxiv.yorks0n.com/posts/oncology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "oncology",
  "name": "oncology",
  "description": "Non-invasive multi-cancer diagnosis using DNA hypomethylation of LINE-1 retrotransposons\nAuthors: Michel, M.; Heidary, M.; Mechri, A.; Da Silva, K.; Gorse, M.; von Grafenstein, K.; Hego, C.; Rampanou, A.; Lamy, C.; Kamal, M.; Le Tourneau, C.; Sene, M.; Bieche, I.; Reyes, C.; Gentien, D.; Stern, M.-H.; Lantz, O.; Cabel, L.; Pierga, J.-Y.; Bidard, F.-C.; Azencott, C.-A.; Proudhon, C.\nScore: 3.0, Published: 2024-01-23 DOI: 10.1101/2024.01.20.23288905\nThe detection of circulating tumor DNA, which allows non-invasive tumor molecular profiling and disease follow-up, promises optimal and individualized management of patients with cancer.",
  "keywords": [
    
  ],
  "articleBody": " Non-invasive multi-cancer diagnosis using DNA hypomethylation of LINE-1 retrotransposons\nAuthors: Michel, M.; Heidary, M.; Mechri, A.; Da Silva, K.; Gorse, M.; von Grafenstein, K.; Hego, C.; Rampanou, A.; Lamy, C.; Kamal, M.; Le Tourneau, C.; Sene, M.; Bieche, I.; Reyes, C.; Gentien, D.; Stern, M.-H.; Lantz, O.; Cabel, L.; Pierga, J.-Y.; Bidard, F.-C.; Azencott, C.-A.; Proudhon, C.\nScore: 3.0, Published: 2024-01-23 DOI: 10.1101/2024.01.20.23288905\nThe detection of circulating tumor DNA, which allows non-invasive tumor molecular profiling and disease follow-up, promises optimal and individualized management of patients with cancer. However, detecting small fractions of tumor DNA released when the tumor burden is reduced remains a challenge. We implemented a new highly sensitive strategy to detect base-pair resolution methylation patterns from plasma DNA and assessed the potential of hypomethylation of LINE-1 retrotransposons as a non-invasive multi-cancer detection biomarker. Resulting machine learning-based classifiers showed powerful correct classification rates discriminating healthy and tumor plasmas from 6 types of cancers in two independent cohorts (AUC = 88% to 100%, N = 747). This should lead to the development of more efficient non-invasive diagnostic tests adapted to all cancer patients, based on the universality of these factors. One-Sentence SummaryLINE-1 retrotransposons hypomethylation is a sensitive and specific biomarker to detect multiple forms of cancer non-invasively.\nImmunotherapy and Cannabis: A Harmful Drug Interaction or Reefer Madness?\nAuthors: Piper, B. J.; Tian, M.; Saini, P.; Higazy, A.; Graham, J.; Carbe, C. J.; Bordonaro, M.\nScore: 2.2, Published: 2024-01-30 DOI: 10.1101/2024.01.26.24301817\nA retrospective (N=140) and a prospective (N=102) observational Israeli study by Bar-Sela and colleagues about cannabis potentially adversely impacting the response to immunotherapy have together been cited 191 times including by clinical practice guidelines. There have also been re-ports on PubPeer outlining unverifiable information in their statistics and numerous discrepan-cies calculating percentages. This report attempted to replicate the data-analysis including non-parametric statistics. Table 1 of the corrected prospective report contained 22 p-values but only one (4.5%) could be verified, despite the authors being transparent about the N and statistics employed. Cannabis users were significantly (p \u003c .0025) younger than non-users but this was not reported in the retrospective report. There were also errors in percentage calculations (e.g. 13/34 reported as 22.0% instead of 38.2%). Overall, these observational investigations, and especially the prospective, appear to contain gross inaccuracies which could impact the statistical decisions (i.e. significant findings reported as non-significant or vice-versa). Although it is mechanistically plausible that cannabis could have immunosuppressive effects which inhibit the response to immunotherapy, these two reports should be viewed cautiously. Larger prospective studies of this purported drug interaction that account for potential confounds (e.g. greater nicotine smok-ing among cannabis users) may be warranted.\nMathematical modeling suggests improved clinical outcomes of second-generation PARP inhibitors with reduced toxicity\nAuthors: Kozlowska, E.; Haltia, U.-M.; Puszynski, K.; Farkkila, A.\nScore: 1.8, Published: 2024-01-30 DOI: 10.1101/2024.01.29.24301918\nRecent clinical trials have shown the remarkable benefits of maintenance treatment with Poly-ADP Ribose Polymerase inhibitors, particularly in patients with tumors deficient in homologous recombination DNA repair. However, the large Phase III clinical trials are limited in their ability to evaluate various dosages and treatment durations, and the effects of toxicities and treatment interruptions on clinical outcomes remain unknown. In this study, we developed a computational framework for virtual clinical trials taking into account both tumor dynamics (growth selection and evolution of resistance mechanisms) and clinico-pharmacological features (pharmacokinetics, dosing, hematological toxicity, dose interruptions, and reductions). Our model replicates the clinical outcomes observed in the landmark SOLO-1 clinical trial. The branching process model revealed a heterogeneous patient population with distinct tumor dynamics, highlighting fitness and selection pressure as determinants of a poor response. Through a virtual trial approach, we show that managing toxicity via treatment interruptions or dose reductions does not compromise the clinical benefits of the treatment. Importantly, we present evidence that further reduction of hematological toxicity could significantly improve clinical outcomes in first-line PARPi maintenance treatment in ovarian cancer.\nUse of tryptic peptide MALDI mass spectrometry imaging to identify the spatial proteomic landscape of colorectal cancer liver metastases.\nAuthors: Li, C. M. Y.; Briggs, M.; Lee, Y. R.; Tin, T.; Young, C.; Pierides, J.; Kaur, G.; Drew, P.; Maddern, G.; Hoffman, P.; Klinger-Hoffman, M.; Fenix, K.\nScore: 1.0, Published: 2024-01-25 DOI: 10.1101/2024.01.24.24301748\nColorectal cancer (CRC) is the second leading cause of cancer-related deaths worldwide. CRC liver metastases (CRLM) are often resistant to conventional treatments, with high rates of recurrence. Therefore, it is crucial to identify biomarkers for CRLM patients that predict cancer progression. This study utilised matrix-assisted laser desorption/ionisation mass spectrometry imaging (MALDI-MSI) in combination with liquid chromatography-tandem mass spectrometry (LC-MS/MS) to spatially map the CRLM tumour proteome. CRLM tissue microarrays (TMAs) of 84 patients were analysed using tryptic peptide MALDI-MSI to spatially monitor peptide abundances across CRLM tissues. Abundance of peptides was compared between tumour vs stroma, male vs female and across three groups of patients based on overall survival (0-3 years, 4-6 years, and 7+ years). Peptides were then characterised and matched using LC-MS/MS. A total of 471 potential peptides were identified by MALDI-MSI. Our results show that two unidentified m/z values (1589.876 and 1092.727) had significantly higher intensities in tumours compared to stroma. Ten m/z values were identified to have correlation with biological sex. Survival analysis identified three peptides (Histone H4, Hemoglobin subunit alpha, and Inosine-5-monophosphate dehydrogenase 2) and two unidentified m/z values (1305.840 and 1661.060) that were significantly higher in patients with shorter survival (0-3 years relative to 4-6 years and 7+ years). This is the first study using MALDI-MSI, combined with LC-MS/MS, on a large cohort of CRLM patients to identify the spatial proteome in this malignancy. Further, we identify several protein candidates that may be suitable for drug targeting or for future prognostic biomarker development.\nMulti-omics profiling with untargeted proteomics for blood-based early detection of lung cancer\nAuthors: Koh, B.; Liu, M.; Almonte, R.; Ariad, D.; Bundalian, G.; Chan, J.; Choi, J.; Chou, W.-F.; Cuaresma, R.; Hoedt, E.; Hopper, L.; Hu, Y.; Jain, A.; Khaledian, E.; Khin, T.; Kokate, A.; Lee, J.-Y.; Leung, S.; Lin, C.-H.; Marispini, M.; Malekpour, H.; Mora, M.; Mudaliar, N.; Nouri Golmaei, S.; Ramaiah, M.; Ramaswamy, S.; Ranjan, P.; Shu, G.; Spiro, P.; Ta, B.; Vitko, D.; Waiss, J.; Yanagihara, Z.; Zawada, R.; Zeng, J. Y.; Zhang, S.; Yee, J.; Blume, J. E.; Belthangady, C.; Wilcox, B.; Ma, P.\nScore: 30.5, Published: 2024-01-04 DOI: 10.1101/2024.01.03.24300798\nBlood-based approaches to detect early-stage cancer provide an opportunity to improve survival rates for lung cancer, the most lethal cancer world-wide. Multiple approaches for blood-based cancer detection using molecular analytes derived from individual omics (cell-free DNA, RNA transcripts, proteins, metabolites) have been developed and tested, generally showing significantly lower sensitivity for early-stage versus late-stage cancer. We hypothesized that an approach using multiple types of molecular analytes, including broad and untargeted coverage of proteins, could identify biomarkers that more directly reveal changes in gene expression and molecular phenotype in response to carcinogenesis to potentially improve detection of early-stage lung cancer. To that end, we designed and conducted one of the largest multi-omics, observational studies to date, enrolling 2513 case and control subjects. Multi-omics profiling detected 113,671 peptides corresponding to 8385 protein groups, 219,729 RNA transcripts, 71,756 RNA introns, and 1801 metabolites across all subject samples. We then developed a machine learning-based classifier for lung cancer detection comprising 682 of these multi-omics analytes. This multi-omics classifier demonstrated 89%, 80%, and 98-100% sensitivity for all-stage, stage I, and stage III-IV lung cancer, respectively, at 89% specificity in a validation set. The application of a multi-omics platform for discovery of blood-based disease biomarkers, including proteins and complementary molecular analytes, enables the noninvasive detection of early-stage lung cancer with the potential for downstaging at initial diagnosis and the improvement of clinical outcomes.\n",
  "wordCount" : "1279",
  "inLanguage": "en",
  "datePublished": "2024-01-31T10:38:28Z",
  "dateModified": "2024-01-31T10:38:28Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/oncology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      oncology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 31, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.23288905">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.23288905" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.23288905">
        <p class="paperTitle">Non-invasive multi-cancer diagnosis using DNA hypomethylation of LINE-1 retrotransposons</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.23288905" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.23288905" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Michel, M.; Heidary, M.; Mechri, A.; Da Silva, K.; Gorse, M.; von Grafenstein, K.; Hego, C.; Rampanou, A.; Lamy, C.; Kamal, M.; Le Tourneau, C.; Sene, M.; Bieche, I.; Reyes, C.; Gentien, D.; Stern, M.-H.; Lantz, O.; Cabel, L.; Pierga, J.-Y.; Bidard, F.-C.; Azencott, C.-A.; Proudhon, C.</p>
        <p class="info">Score: 3.0, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.23288905' target='https://doi.org/10.1101/2024.01.20.23288905'> 10.1101/2024.01.20.23288905</a></p>
        <p class="abstract">The detection of circulating tumor DNA, which allows non-invasive tumor molecular profiling and disease follow-up, promises optimal and individualized management of patients with cancer. However, detecting small fractions of tumor DNA released when the tumor burden is reduced remains a challenge. We implemented a new highly sensitive strategy to detect base-pair resolution methylation patterns from plasma DNA and assessed the potential of hypomethylation of LINE-1 retrotransposons as a non-invasive multi-cancer detection biomarker. Resulting machine learning-based classifiers showed powerful correct classification rates discriminating healthy and tumor plasmas from 6 types of cancers in two independent cohorts (AUC = 88% to 100%, N = 747). This should lead to the development of more efficient non-invasive diagnostic tests adapted to all cancer patients, based on the universality of these factors.

One-Sentence SummaryLINE-1 retrotransposons hypomethylation is a sensitive and specific biomarker to detect multiple forms of cancer non-invasively.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.26.24301817">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.26.24301817" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.26.24301817">
        <p class="paperTitle">Immunotherapy and Cannabis: A Harmful Drug Interaction or Reefer Madness?</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.26.24301817" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.26.24301817" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Piper, B. J.; Tian, M.; Saini, P.; Higazy, A.; Graham, J.; Carbe, C. J.; Bordonaro, M.</p>
        <p class="info">Score: 2.2, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.26.24301817' target='https://doi.org/10.1101/2024.01.26.24301817'> 10.1101/2024.01.26.24301817</a></p>
        <p class="abstract">A retrospective (N=140) and a prospective (N=102) observational Israeli study by Bar-Sela and colleagues about cannabis potentially adversely impacting the response to immunotherapy have together been cited 191 times including by clinical practice guidelines. There have also been re-ports on PubPeer outlining unverifiable information in their statistics and numerous discrepan-cies calculating percentages. This report attempted to replicate the data-analysis including non-parametric statistics. Table 1 of the corrected prospective report contained 22 p-values but only one (4.5%) could be verified, despite the authors being transparent about the N and statistics employed. Cannabis users were significantly (p &lt; .0025) younger than non-users but this was not reported in the retrospective report. There were also errors in percentage calculations (e.g. 13/34 reported as 22.0% instead of 38.2%). Overall, these observational investigations, and especially the prospective, appear to contain gross inaccuracies which could impact the statistical decisions (i.e. significant findings reported as non-significant or vice-versa). Although it is mechanistically plausible that cannabis could have immunosuppressive effects which inhibit the response to immunotherapy, these two reports should be viewed cautiously. Larger prospective studies of this purported drug interaction that account for potential confounds (e.g. greater nicotine smok-ing among cannabis users) may be warranted.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.24301918">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.24301918" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.24301918">
        <p class="paperTitle">Mathematical modeling suggests improved clinical outcomes of second-generation PARP inhibitors with reduced toxicity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.24301918" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.24301918" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kozlowska, E.; Haltia, U.-M.; Puszynski, K.; Farkkila, A.</p>
        <p class="info">Score: 1.8, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.24301918' target='https://doi.org/10.1101/2024.01.29.24301918'> 10.1101/2024.01.29.24301918</a></p>
        <p class="abstract">Recent clinical trials have shown the remarkable benefits of maintenance treatment with Poly-ADP Ribose Polymerase inhibitors, particularly in patients with tumors deficient in homologous recombination DNA repair. However, the large Phase III clinical trials are limited in their ability to evaluate various dosages and treatment durations, and the effects of toxicities and treatment interruptions on clinical outcomes remain unknown. In this study, we developed a computational framework for virtual clinical trials taking into account both tumor dynamics (growth selection and evolution of resistance mechanisms) and clinico-pharmacological features (pharmacokinetics, dosing, hematological toxicity, dose interruptions, and reductions). Our model replicates the clinical outcomes observed in the landmark SOLO-1 clinical trial. The branching process model revealed a heterogeneous patient population with distinct tumor dynamics, highlighting fitness and selection pressure as determinants of a poor response. Through a virtual trial approach, we show that managing toxicity via treatment interruptions or dose reductions does not compromise the clinical benefits of the treatment. Importantly, we present evidence that further reduction of hematological toxicity could significantly improve clinical outcomes in first-line PARPi maintenance treatment in ovarian cancer.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.24301748">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.24301748" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.24301748">
        <p class="paperTitle">Use of tryptic peptide MALDI mass spectrometry imaging to identify the spatial proteomic landscape of colorectal cancer liver metastases.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.24301748" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.24301748" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, C. M. Y.; Briggs, M.; Lee, Y. R.; Tin, T.; Young, C.; Pierides, J.; Kaur, G.; Drew, P.; Maddern, G.; Hoffman, P.; Klinger-Hoffman, M.; Fenix, K.</p>
        <p class="info">Score: 1.0, Published: 2024-01-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.24301748' target='https://doi.org/10.1101/2024.01.24.24301748'> 10.1101/2024.01.24.24301748</a></p>
        <p class="abstract">Colorectal cancer (CRC) is the second leading cause of cancer-related deaths worldwide. CRC liver metastases (CRLM) are often resistant to conventional treatments, with high rates of recurrence. Therefore, it is crucial to identify biomarkers for CRLM patients that predict cancer progression. This study utilised matrix-assisted laser desorption/ionisation mass spectrometry imaging (MALDI-MSI) in combination with liquid chromatography-tandem mass spectrometry (LC-MS/MS) to spatially map the CRLM tumour proteome. CRLM tissue microarrays (TMAs) of 84 patients were analysed using tryptic peptide MALDI-MSI to spatially monitor peptide abundances across CRLM tissues. Abundance of peptides was compared between tumour vs stroma, male vs female and across three groups of patients based on overall survival (0-3 years, 4-6 years, and 7&#43; years). Peptides were then characterised and matched using LC-MS/MS. A total of 471 potential peptides were identified by MALDI-MSI. Our results show that two unidentified m/z values (1589.876 and 1092.727) had significantly higher intensities in tumours compared to stroma. Ten m/z values were identified to have correlation with biological sex. Survival analysis identified three peptides (Histone H4, Hemoglobin subunit alpha, and Inosine-5-monophosphate dehydrogenase 2) and two unidentified m/z values (1305.840 and 1661.060) that were significantly higher in patients with shorter survival (0-3 years relative to 4-6 years and 7&#43; years). This is the first study using MALDI-MSI, combined with LC-MS/MS, on a large cohort of CRLM patients to identify the spatial proteome in this malignancy. Further, we identify several protein candidates that may be suitable for drug targeting or for future prognostic biomarker development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.24300798">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.24300798" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.24300798">
        <p class="paperTitle">Multi-omics profiling with untargeted proteomics for blood-based early detection of lung cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.24300798" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.24300798" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Koh, B.; Liu, M.; Almonte, R.; Ariad, D.; Bundalian, G.; Chan, J.; Choi, J.; Chou, W.-F.; Cuaresma, R.; Hoedt, E.; Hopper, L.; Hu, Y.; Jain, A.; Khaledian, E.; Khin, T.; Kokate, A.; Lee, J.-Y.; Leung, S.; Lin, C.-H.; Marispini, M.; Malekpour, H.; Mora, M.; Mudaliar, N.; Nouri Golmaei, S.; Ramaiah, M.; Ramaswamy, S.; Ranjan, P.; Shu, G.; Spiro, P.; Ta, B.; Vitko, D.; Waiss, J.; Yanagihara, Z.; Zawada, R.; Zeng, J. Y.; Zhang, S.; Yee, J.; Blume, J. E.; Belthangady, C.; Wilcox, B.; Ma, P.</p>
        <p class="info">Score: 30.5, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.24300798' target='https://doi.org/10.1101/2024.01.03.24300798'> 10.1101/2024.01.03.24300798</a></p>
        <p class="abstract">Blood-based approaches to detect early-stage cancer provide an opportunity to improve survival rates for lung cancer, the most lethal cancer world-wide. Multiple approaches for blood-based cancer detection using molecular analytes derived from individual  omics (cell-free DNA, RNA transcripts, proteins, metabolites) have been developed and tested, generally showing significantly lower sensitivity for early-stage versus late-stage cancer. We hypothesized that an approach using multiple types of molecular analytes, including broad and untargeted coverage of proteins, could identify biomarkers that more directly reveal changes in gene expression and molecular phenotype in response to carcinogenesis to potentially improve detection of early-stage lung cancer. To that end, we designed and conducted one of the largest multi-omics, observational studies to date, enrolling 2513 case and control subjects. Multi-omics profiling detected 113,671 peptides corresponding to 8385 protein groups, 219,729 RNA transcripts, 71,756 RNA introns, and 1801 metabolites across all subject samples. We then developed a machine learning-based classifier for lung cancer detection comprising 682 of these multi-omics analytes. This multi-omics classifier demonstrated 89%, 80%, and 98-100% sensitivity for all-stage, stage I, and stage III-IV lung cancer, respectively, at 89% specificity in a validation set. The application of a multi-omics platform for discovery of blood-based disease biomarkers, including proteins and complementary molecular analytes, enables the noninvasive detection of early-stage lung cancer with the potential for downstaging at initial diagnosis and the improvement of clinical outcomes.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
