<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cell biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cell biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Psychosocial experiences are associated with human brain mitochondrial biology
Authors: Trumpff, C.; Monzel, A.; Sandi, C.; Menon, V.; Klein, H.-U.; Fujita, M.; Lee, A. J.; Petyuk, V. A.; Hurst, C.; Duong, D. A.; Seyfried, N.; Wingo, A.; Wingo, T. S.; Wang, Y.; Thambisetty, M.; Ferrucci, L.; Bennett, D. A.; De Jager, P.; Picard, M.
Score: 49.8, Published: 2023-10-06 DOI: 10.1101/2023.10.06.559575
Psychosocial experiences affect brain health and aging trajectories, but the molecular pathways underlying these associations remain unclear.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cell-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cell biology" />
<meta property="og:description" content="Psychosocial experiences are associated with human brain mitochondrial biology
Authors: Trumpff, C.; Monzel, A.; Sandi, C.; Menon, V.; Klein, H.-U.; Fujita, M.; Lee, A. J.; Petyuk, V. A.; Hurst, C.; Duong, D. A.; Seyfried, N.; Wingo, A.; Wingo, T. S.; Wang, Y.; Thambisetty, M.; Ferrucci, L.; Bennett, D. A.; De Jager, P.; Picard, M.
Score: 49.8, Published: 2023-10-06 DOI: 10.1101/2023.10.06.559575
Psychosocial experiences affect brain health and aging trajectories, but the molecular pathways underlying these associations remain unclear." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cell-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-22T10:37:23+00:00" />
<meta property="article:modified_time" content="2023-10-22T10:37:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cell biology"/>
<meta name="twitter:description" content="Psychosocial experiences are associated with human brain mitochondrial biology
Authors: Trumpff, C.; Monzel, A.; Sandi, C.; Menon, V.; Klein, H.-U.; Fujita, M.; Lee, A. J.; Petyuk, V. A.; Hurst, C.; Duong, D. A.; Seyfried, N.; Wingo, A.; Wingo, T. S.; Wang, Y.; Thambisetty, M.; Ferrucci, L.; Bennett, D. A.; De Jager, P.; Picard, M.
Score: 49.8, Published: 2023-10-06 DOI: 10.1101/2023.10.06.559575
Psychosocial experiences affect brain health and aging trajectories, but the molecular pathways underlying these associations remain unclear."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cell biology",
      "item": "https://trxiv.yorks0n.com/posts/cell-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cell biology",
  "name": "cell biology",
  "description": "Psychosocial experiences are associated with human brain mitochondrial biology\nAuthors: Trumpff, C.; Monzel, A.; Sandi, C.; Menon, V.; Klein, H.-U.; Fujita, M.; Lee, A. J.; Petyuk, V. A.; Hurst, C.; Duong, D. A.; Seyfried, N.; Wingo, A.; Wingo, T. S.; Wang, Y.; Thambisetty, M.; Ferrucci, L.; Bennett, D. A.; De Jager, P.; Picard, M.\nScore: 49.8, Published: 2023-10-06 DOI: 10.1101/2023.10.06.559575\nPsychosocial experiences affect brain health and aging trajectories, but the molecular pathways underlying these associations remain unclear.",
  "keywords": [
    
  ],
  "articleBody": " Psychosocial experiences are associated with human brain mitochondrial biology\nAuthors: Trumpff, C.; Monzel, A.; Sandi, C.; Menon, V.; Klein, H.-U.; Fujita, M.; Lee, A. J.; Petyuk, V. A.; Hurst, C.; Duong, D. A.; Seyfried, N.; Wingo, A.; Wingo, T. S.; Wang, Y.; Thambisetty, M.; Ferrucci, L.; Bennett, D. A.; De Jager, P.; Picard, M.\nScore: 49.8, Published: 2023-10-06 DOI: 10.1101/2023.10.06.559575\nPsychosocial experiences affect brain health and aging trajectories, but the molecular pathways underlying these associations remain unclear. Normal brain function relies heavily on energy transformation by mitochondria oxidative phosphorylation (OxPhos), and two main lines of evidence bi-directionally link mitochondria as both targets and drivers of psychosocial experiences. On the one hand, chronic stress exposure and possibly mood states alter multiple aspects of mitochondrial biology; and on the other hand, functional variations in mitochondrial OxPhos capacity alter social behavior, stress reactivity, and mood. However, knowledge on whether positive or negative psychosocial exposures and experiences are linked to mitochondrial biology in the human brain is currently unknown. By combining longitudinal antemortem assessments of psychosocial factors with postmortem brain (dorsolateral prefrontal cortex) proteomics in older adults, we found that positive experiences (e.g. higher well-being) are linked to greater abundance of the mitochondrial OxPhos machinery, whereas negative experiences (e.g. higher negative mood) are linked to lower OxPhos protein content. Combined, psychosocial factors explained 18% of the variance in the abundance of OxPhos complex I, the primary biochemical entry point that energizes brain mitochondria. To increase the sensitivity of our approach, we next interrogated mitochondrial psychobiological associations in specific neuronal and non-neuronal brain cells with single-nucleus RNA sequencing. These results revealed strong cell type specific associations, particularly between positive psychosocial experiences and molecular mitochondrial phenotypes in glial cells, whereas neurons tended to show opposite associations. Accordingly, in bulk transcriptomic analyses where all cells are pooled, these RNA-based associations were masked. Thus, our results highlight the likely underestimation of effect sizes in bulk brain tissues, and document novel cell type specific mitochondrial psychobiological associations in the human brain. Cell type specific mitochondrial recalibrations represent a potential psychobiological pathway linking positive and negative psychosocial experiences to human brain biology. Significance statementPsychosocial experiences predict health trajectories, but the underlying mechanism remains unclear. We found that positive psychosocial experiences are linked to greater abundance of the mitochondrial energy transformation machinery, whereas negative experiences are linked to lower abundance. Overall, we found that psychosocial experiences explain 18% of the variance in abundance of complex I proteins, the main entry point of the mitochondrial oxidative phosphorylation (OxPhos) system. At single-cell resolution using single nucleus transcriptomics, positive psychosocial experiences were particularly related to glial cell mitochondrial phenotypes. Opposite associations between glial cells and neurons were naturally masked in bulk transcriptomic analyses. Our results suggest that mitochondrial recalibrations in specific brain cell types may represent a potential psychobiological pathway linking psychosocial experiences to human brain health.\nRecombinant biosensors for multiplex and super-resolution imaging of phosphoinositides\nAuthors: Maib, H.; Adarska, P.; Hunton, R.; Vines, J.; Strutt, D.; Bottanelli, F.; Murray, D. H.\nScore: 45.8, Published: 2023-10-19 DOI: 10.1101/2023.10.18.562882\nPhosphoinositides are a small family of phospholipids, acting as signalling hubs and key regulators of cellular function. Detecting their subcellular distribution is crucial to gain insights into membrane organisation and is most commonly done by over-expression of biosensors. However, this leads to perturbations of phosphoinositide signalling and is challenging in systems that cannot be transfected. Here, we present a toolkit for the reliable, fast, multiplex, and super-resolution detection of all 8 phosphoinositides using a unifying staining approach for fixed cells and tissue, based on recombinant biosensors with self-labelling SNAP tags. These recombinant biosensors are highly specific, and reliably visualise the subcellular distributions of phosphoinositides across scales, ranging from 2D or 3D cell culture to Drosophila tissue. Using stimulated emission depletion (STED) microscopy, we reveal the nanoscale organisation of PI(3)P on endosomes and PI(4)P on the Golgi and confirm the preservation of subcellular membranes. Multiplex staining enables the investigation of phosphoinositide conversions and reveals an unexpected presence of residual PI(3,5)P2 positive membranes in swollen lysosomes following PIKfyve inhibition. This approach enables the versatile, high-resolution visualisation of multiple phosphoinositide species in an unprecedented manner.\nGenetic transformation of the frog-killing chytrid fungus Batrachochytrium dendrobatidis\nAuthors: Kalinka, E.; Brody, S.; Swafford, A.; Medina, E.; Fritz-Laylin, L.\nScore: 27.6, Published: 2023-10-17 DOI: 10.1101/2023.10.17.561934\nBatrachochytrium dendrobatidis (Bd), a causative agent of chytridiomycosis, is decimating amphibian populations around the world. Bd belongs to the chytrid lineage, a group of early-diverging fungi that are widely used to study fungal evolution. Like all chytrids, Bd develops from a motile form into a sessile, growth form, a transition that involves drastic changes in its cytoskeletal architecture. Efforts to study Bd cell biology, development, and pathogenicity have been limited by the lack of genetic tools with which to test hypotheses about underlying molecular mechanisms. Here, we report the development of a transient genetic transformation system for Bd. We used electroporation to deliver exogenous DNA into Bd cells and detected transgene expression for up to three generations under both heterologous and native promoters. We also adapted the transformation protocol for selection using an antibiotic resistance marker. Finally, we used this system to express fluorescent protein fusions and, as a proof of concept, expressed a genetically encoded probe for the actin cytoskeleton. Using live-cell imaging, we visualized the distribution and dynamics of polymerized actin at each stage of the Bd life cycle, as well as during key developmental transitions. This transformation system allows, for the first time, direct testing of key hypotheses regarding mechanisms of Bd pathogenesis. This technology also paves the way for answering fundamental questions of chytrid cell, developmental, and evolutionary biology.\nFailed reprogramming of transformed cells due to induction of apoptosis and senescence impairs tumor progression in lung cancer\nAuthors: Pedrosa, P.; Zang, Z.; Macias, D.; Ge, J.; Denholm, M.; Dyas, A.; Nunez-Quintela, V.; Estevez-Souto, V.; Lado-Fernandez, P.; Gonzalez, P.; Gomez, M.; Martin, J. E.; Da Silva-Alvarez, S.; Munoz-Espin, D.; Collado, M.\nScore: 21.2, Published: 2023-10-20 DOI: 10.1101/2023.10.19.563086\nCell reprogramming to pluripotency applied to the study of cancer has identified transformation and pluripotency as two independent and incompatible cell fates. A detailed knowledge of the relationship between transformation and reprogramming could lead to the identification of new vulnerabilities and therapeutic targets in cancer. Here, we explore this interplay and find that OSKM expression limits tumor cell growth by inducing apoptosis and senescence. We identify Oct4 and Klf4 as the main individual reprogramming factors responsible for this effect. Mechanistically, the induction of cell cycle inhibitor p21 downstream of the reprogramming factors acts as mediator of cell death and senescence. Using a variety of in vivo systems, including allografts, orthotopic transplantation and KRAS-driven lung cancer mouse models, we demonstrate that OSKM expression impairs tumor growth and reduces tumor burden.\nPrecise immunofluorescence canceling for highly multiplexed imaging capturing specific cell state\nAuthors: Tomimatsu, K.; Fujii, T.; Bise, R.; Hosoda, K.; Taniguchi, Y.; Ochiai, H.; Ohishi, H.; Ando, K.; Minami, R.; Taro, T.; Tanaka, K.; Mori, S.; Harada, A.; Maehara, K.; Nagasaki, M.; Uchida, S.; Kimura, H.; Narita, M.; Ohkawa, Y.\nScore: 16.6, Published: 2023-10-18 DOI: 10.1101/2023.10.17.561810\nCell states are regulated by the response of signaling pathways to receptor ligand-binding and inter-cellular interactions. High-resolution imaging has been attempted to explore the dynamics of these processes and, recently, multiplexed imaging has profiled cell states by achieving comprehensive acquisition of spatial protein information from cells. However, resolution is still compromised when visualizing activated signals. Here we developed Precise Emission Canceling Antibody (PECAb) that have cleavable fluorescent labeling. PECAbs allow high-resolution sequential imaging using hundreds of antibodies and allow reconstruction of the spatiotemporal dynamics of signaling pathways. Additionally, combining this approach with seq-smFISH can effectively classify cells and identify their signal activation states in human tissue. Overall, the PECAb system can serve as a comprehensive platform for analyzing complex cell processes.\nT cells Use Focal Adhesions to Pull Themselves Through Confined Environments\nAuthors: Caillier, A.; Oleksyn, D.; Fowell, D. J.; Miller, J.; Oakes, P. W.\nScore: 16.3, Published: 2023-10-19 DOI: 10.1101/2023.10.16.562587\nImmune cells are highly dynamic and able to migrate through environments with diverse biochemical and mechanical composition. Their migration has classically been defined as amoeboid under the assumption that it is integrin-independent. Here we show that activated primary Th1 T cells require both confinement and extracellular matrix protein to migrate efficiently. This migration is mediated through small and dynamic focal adhesions that are composed of the same proteins associated with canonical mesenchymal focal adhesions, such as integrins, talin, and vinculin. These focal adhesions, furthermore, localize to sites of contractile traction stresses, enabling T cells to pull themselves through confined spaces. Finally, we show that Th1 T cell preferentially follows tracks of other T cells, suggesting that these adhesions are modifying the extracellular matrix to provide additional environmental guidance cues. These results demonstrate not only that the boundaries between amoeboid and mesenchymal migration modes are ambiguous, but that integrin-mediated adhesions play a key role in T cell motility. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=175 SRC=\"FIGDIR/small/562587v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (78K): org.highwire.dtl.DTLVardef@cdf4e3org.highwire.dtl.DTLVardef@1888037org.highwire.dtl.DTLVardef@99c8cdorg.highwire.dtl.DTLVardef@4e133e_HPS_FORMAT_FIGEXP M_FIG C_FIG\nRepression of mRNA translation initiation by GIGYF1 via blocking the eIF3-eIF4G1 interaction\nAuthors: Choi, J.-H.; Luo, J.; Hesketh, G. G.; Guo, S.; Pistofidis, A.; Ladak, R. J.; An, Y.; Alain, T.; Schmeing, M.; Gingras, A.-C.; Duchaine, T.; Zhang, X.; Sonenberg, N.; Jafarnejad, S. M.\nScore: 15.5, Published: 2023-10-15 DOI: 10.1101/2023.10.14.562322\nViruses commonly interfere with the function of the eukaryotic translation initiation factor 4G1 (eIF4G1), a pivotal factor in the recruitment of the eIF3 complex and ribosome to the mRNA. This results in the inhibition of general host protein synthesis and redirecting ribosomes toward viral mRNAs. Certain viruses also selectively repress the translation of mRNAs involved in the host antiviral response. GIGYF2 and its interacting cap-binding protein 4EHP enable the transcript-specific repression of mRNA translation mediated by microRNAs and RNA-binding proteins (RBPs). RNA viruses, such as SARS-CoV-2, exploit the GIGYF2/4EHP complex to selectively repress the translation of transcripts such as Ifnb1 mRNA, which encodes the antiviral cytokine Interferon {beta} (IFN-{beta}). Herein, we reveal that GIGYF1, a paralogue of GIGYF2, robustly represses cellular mRNA translation through a distinct mechanism independent of 4EHP. Upon recruitment to a target mRNA by RBPs, the C-terminal region of GIGYF1 binds to subunits of eIF3 at the interaction interface of eIF3-eIF4G1. This disrupts binding of eIF3 to eIF4G1, resulting in mRNA-specific translational repression. This mechanism exerts profound influences on the host cells response to viral infection. Depletion of GIGYF1 induces a robust immune response by derepressing Ifnb1 mRNA translation. Overall, our study highlights a unique mechanism of translational regulation by GIGYF1 that involves sequestering eIF3 and abrogating its binding to eIF4G1. This mechanism can be utilized by RBPs that interact with GIGYF1 to specifically repress the translation of their target mRNAs, significantly affecting critical biological processes, including host-pathogen interactions.\nChromosome size-dependent polar ejection force impairs mammalian mitotic error correction\nAuthors: Chong, M. K.; Rosas Salvans, M. K.; Tran, V.; Dumont, S.\nScore: 15.3, Published: 2023-10-18 DOI: 10.1101/2023.10.16.562637\nAccurate chromosome segregation requires sister kinetochores to biorient, attaching to opposite spindle poles. To this end, the mammalian kinetochore destabilizes incorrect attachments and stabilizes correct ones, but how it discriminates between these is not yet clear. Here, we test the model that kinetochore tension is the stabilizing cue and ask how chromosome size impacts that model. We live image PtK2 cells, with just 14 chromosomes, widely ranging in size, and find that long chromosomes align at the metaphase plate later than short chromosomes. Enriching for errors and imaging error correction live, we show that long chromosomes exhibit a specific delay in correcting attachments. Using chromokinesin overexpression and laser ablation to perturb polar ejection forces, we find that chromosome size and force on arms determine alignment order. Thus, we propose a model where increased force on long chromosomes can falsely stabilize incorrect attachments, delaying their biorientation. As such, long chromosomes may require compensatory mechanisms for correcting errors to avoid chromosomal instability.\nThe H3.3 K36M oncohistone disrupts the establishment of epigenetic memory through loss of DNA methylation\nAuthors: Sinha, J.; Nickels, J. F.; Thurm, A. R.; Ludwig, C. H.; Archibald, B. N.; Hinks, M. M.; Wan, J.; Fang, D.; Bintu, L.\nScore: 13.2, Published: 2023-10-14 DOI: 10.1101/2023.10.13.562147\nHistone H3.3 is frequently mutated in cancers, with the lysine 36 to methionine mutation (K36M) being a hallmark of chondroblastomas. While it is known that H3.3K36M changes the cellular epigenetic landscape, it remains unclear how it affects the dynamics of gene expression. Here, we use a synthetic reporter to measure the effect of H3.3K36M on silencing and epigenetic memory after recruitment of KRAB: a member of the largest class of human repressors, commonly used in synthetic biology, and associated with H3K9me3. We find that H3.3K36M, which decreases H3K36 methylation, leads to a decrease in epigenetic memory and promoter methylation weeks after KRAB release. We propose a new model for establishment and maintenance of epigenetic memory, where H3K36 methylation is necessary to convert H3K9me3 domains into DNA methylation for stable epigenetic memory. Our quantitative model can inform oncogenic mechanisms and guide development of epigenetic editing tools.\nCENP-E initiates chromosome congression by opposing Aurora kinases to promote end-on microtubule attachments close to centrosomes\nAuthors: Vukusic, K.; Tolic, I. M.\nScore: 12.3, Published: 2023-10-20 DOI: 10.1101/2023.10.19.563150\nChromosome congression to the spindle equator, which recurrently fails in cancer cells leading to aneuploidy, requires the kinetochore motor protein CENP-E/kinesin-7. Current models state that chromosomes congress by CENP-E-driven gliding along spindle microtubules. Here, we show that CENP-E is not essential for the congression movement, but for initiating movement through the formation of end-on attachments. We reached this conclusion by large-scale live-cell imaging under varying activity of CENP-E using lattice light-sheet microscopy. We found that congression without CENP-E is delayed by the activity of centriole-dependent Aurora A and outer-kinetochore-localized Aurora B kinases, which limit the formation of end-on attachments. CENP-E bypasses this inhibition by causing a decrease in Aurora B activity and in phosphorylation of its outer-kinetochore targets. Chromosomally unstable cancer cells rely on CENP-E more than non-cancer cells, leading to segregation errors specifically in cancer cells with prolonged mitosis, thereby exposing a mitotic timing-dependent vulnerability of cancer cells.\n",
  "wordCount" : "2348",
  "inLanguage": "en",
  "datePublished": "2023-10-22T10:37:23Z",
  "dateModified": "2023-10-22T10:37:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cell-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cell biology
    </h1>
    <div class="post-meta"><span>updated on October 22, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.559575">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.559575" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.559575">
        <p class="paperTitle">Psychosocial experiences are associated with human brain mitochondrial biology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.559575" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.559575" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Trumpff, C.; Monzel, A.; Sandi, C.; Menon, V.; Klein, H.-U.; Fujita, M.; Lee, A. J.; Petyuk, V. A.; Hurst, C.; Duong, D. A.; Seyfried, N.; Wingo, A.; Wingo, T. S.; Wang, Y.; Thambisetty, M.; Ferrucci, L.; Bennett, D. A.; De Jager, P.; Picard, M.</p>
        <p class="info">Score: 49.8, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.559575' target='https://doi.org/10.1101/2023.10.06.559575'> 10.1101/2023.10.06.559575</a></p>
        <p class="abstract">Psychosocial experiences affect brain health and aging trajectories, but the molecular pathways underlying these associations remain unclear. Normal brain function relies heavily on energy transformation by mitochondria oxidative phosphorylation (OxPhos), and two main lines of evidence bi-directionally link mitochondria as both targets and drivers of psychosocial experiences. On the one hand, chronic stress exposure and possibly mood states alter multiple aspects of mitochondrial biology; and on the other hand, functional variations in mitochondrial OxPhos capacity alter social behavior, stress reactivity, and mood. However, knowledge on whether positive or negative psychosocial exposures and experiences are linked to mitochondrial biology in the human brain is currently unknown. By combining longitudinal antemortem assessments of psychosocial factors with postmortem brain (dorsolateral prefrontal cortex) proteomics in older adults, we found that positive experiences (e.g. higher well-being) are linked to greater abundance of the mitochondrial OxPhos machinery, whereas negative experiences (e.g. higher negative mood) are linked to lower OxPhos protein content. Combined, psychosocial factors explained 18% of the variance in the abundance of OxPhos complex I, the primary biochemical entry point that energizes brain mitochondria. To increase the sensitivity of our approach, we next interrogated mitochondrial psychobiological associations in specific neuronal and non-neuronal brain cells with single-nucleus RNA sequencing. These results revealed strong cell type specific associations, particularly between positive psychosocial experiences and molecular mitochondrial phenotypes in glial cells, whereas neurons tended to show opposite associations. Accordingly, in bulk transcriptomic analyses where all cells are pooled, these RNA-based associations were masked. Thus, our results highlight the likely underestimation of effect sizes in bulk brain tissues, and document novel cell type specific mitochondrial psychobiological associations in the human brain. Cell type specific mitochondrial recalibrations represent a potential psychobiological pathway linking positive and negative psychosocial experiences to human brain biology.

Significance statementPsychosocial experiences predict health trajectories, but the underlying mechanism remains unclear. We found that positive psychosocial experiences are linked to greater abundance of the mitochondrial energy transformation machinery, whereas negative experiences are linked to lower abundance. Overall, we found that psychosocial experiences explain 18% of the variance in abundance of complex I proteins, the main entry point of the mitochondrial oxidative phosphorylation (OxPhos) system. At single-cell resolution using single nucleus transcriptomics, positive psychosocial experiences were particularly related to glial cell mitochondrial phenotypes. Opposite associations between glial cells and neurons were naturally masked in bulk transcriptomic analyses. Our results suggest that mitochondrial recalibrations in specific brain cell types may represent a potential psychobiological pathway linking psychosocial experiences to human brain health.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.18.562882">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.18.562882" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.18.562882">
        <p class="paperTitle">Recombinant biosensors for multiplex and super-resolution imaging of phosphoinositides</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.18.562882" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.18.562882" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Maib, H.; Adarska, P.; Hunton, R.; Vines, J.; Strutt, D.; Bottanelli, F.; Murray, D. H.</p>
        <p class="info">Score: 45.8, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.18.562882' target='https://doi.org/10.1101/2023.10.18.562882'> 10.1101/2023.10.18.562882</a></p>
        <p class="abstract">Phosphoinositides are a small family of phospholipids, acting as signalling hubs and key regulators of cellular function. Detecting their subcellular distribution is crucial to gain insights into membrane organisation and is most commonly done by over-expression of biosensors. However, this leads to perturbations of phosphoinositide signalling and is challenging in systems that cannot be transfected. Here, we present a toolkit for the reliable, fast, multiplex, and super-resolution detection of all 8 phosphoinositides using a unifying staining approach for fixed cells and tissue, based on recombinant biosensors with self-labelling SNAP tags. These recombinant biosensors are highly specific, and reliably visualise the subcellular distributions of phosphoinositides across scales, ranging from 2D or 3D cell culture to Drosophila tissue. Using stimulated emission depletion (STED) microscopy, we reveal the nanoscale organisation of PI(3)P on endosomes and PI(4)P on the Golgi and confirm the preservation of subcellular membranes. Multiplex staining enables the investigation of phosphoinositide conversions and reveals an unexpected presence of residual PI(3,5)P2 positive membranes in swollen lysosomes following PIKfyve inhibition. This approach enables the versatile, high-resolution visualisation of multiple phosphoinositide species in an unprecedented manner.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.561934">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.561934" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.561934">
        <p class="paperTitle">Genetic transformation of the frog-killing chytrid fungus Batrachochytrium dendrobatidis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.561934" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.561934" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kalinka, E.; Brody, S.; Swafford, A.; Medina, E.; Fritz-Laylin, L.</p>
        <p class="info">Score: 27.6, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.561934' target='https://doi.org/10.1101/2023.10.17.561934'> 10.1101/2023.10.17.561934</a></p>
        <p class="abstract">Batrachochytrium dendrobatidis (Bd), a causative agent of chytridiomycosis, is decimating amphibian populations around the world. Bd belongs to the chytrid lineage, a group of early-diverging fungi that are widely used to study fungal evolution. Like all chytrids, Bd develops from a motile form into a sessile, growth form, a transition that involves drastic changes in its cytoskeletal architecture. Efforts to study Bd cell biology, development, and pathogenicity have been limited by the lack of genetic tools with which to test hypotheses about underlying molecular mechanisms. Here, we report the development of a transient genetic transformation system for Bd. We used electroporation to deliver exogenous DNA into Bd cells and detected transgene expression for up to three generations under both heterologous and native promoters. We also adapted the transformation protocol for selection using an antibiotic resistance marker. Finally, we used this system to express fluorescent protein fusions and, as a proof of concept, expressed a genetically encoded probe for the actin cytoskeleton. Using live-cell imaging, we visualized the distribution and dynamics of polymerized actin at each stage of the Bd life cycle, as well as during key developmental transitions. This transformation system allows, for the first time, direct testing of key hypotheses regarding mechanisms of Bd pathogenesis. This technology also paves the way for answering fundamental questions of chytrid cell, developmental, and evolutionary biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.19.563086">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.19.563086" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.19.563086">
        <p class="paperTitle">Failed reprogramming of transformed cells due to induction of apoptosis and senescence impairs tumor progression in lung cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.19.563086" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.19.563086" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pedrosa, P.; Zang, Z.; Macias, D.; Ge, J.; Denholm, M.; Dyas, A.; Nunez-Quintela, V.; Estevez-Souto, V.; Lado-Fernandez, P.; Gonzalez, P.; Gomez, M.; Martin, J. E.; Da Silva-Alvarez, S.; Munoz-Espin, D.; Collado, M.</p>
        <p class="info">Score: 21.2, Published: 2023-10-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.19.563086' target='https://doi.org/10.1101/2023.10.19.563086'> 10.1101/2023.10.19.563086</a></p>
        <p class="abstract">Cell reprogramming to pluripotency applied to the study of cancer has identified transformation and pluripotency as two independent and incompatible cell fates. A detailed knowledge of the relationship between transformation and reprogramming could lead to the identification of new vulnerabilities and therapeutic targets in cancer. Here, we explore this interplay and find that OSKM expression limits tumor cell growth by inducing apoptosis and senescence. We identify Oct4 and Klf4 as the main individual reprogramming factors responsible for this effect. Mechanistically, the induction of cell cycle inhibitor p21 downstream of the reprogramming factors acts as mediator of cell death and senescence. Using a variety of in vivo systems, including allografts, orthotopic transplantation and KRAS-driven lung cancer mouse models, we demonstrate that OSKM expression impairs tumor growth and reduces tumor burden.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.561810">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.561810" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.561810">
        <p class="paperTitle">Precise immunofluorescence canceling for highly multiplexed imaging capturing specific cell state</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.561810" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.561810" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tomimatsu, K.; Fujii, T.; Bise, R.; Hosoda, K.; Taniguchi, Y.; Ochiai, H.; Ohishi, H.; Ando, K.; Minami, R.; Taro, T.; Tanaka, K.; Mori, S.; Harada, A.; Maehara, K.; Nagasaki, M.; Uchida, S.; Kimura, H.; Narita, M.; Ohkawa, Y.</p>
        <p class="info">Score: 16.6, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.561810' target='https://doi.org/10.1101/2023.10.17.561810'> 10.1101/2023.10.17.561810</a></p>
        <p class="abstract">Cell states are regulated by the response of signaling pathways to receptor ligand-binding and inter-cellular interactions. High-resolution imaging has been attempted to explore the dynamics of these processes and, recently, multiplexed imaging has profiled cell states by achieving comprehensive acquisition of spatial protein information from cells. However, resolution is still compromised when visualizing activated signals. Here we developed Precise Emission Canceling Antibody (PECAb) that have cleavable fluorescent labeling. PECAbs allow high-resolution sequential imaging using hundreds of antibodies and allow reconstruction of the spatiotemporal dynamics of signaling pathways. Additionally, combining this approach with seq-smFISH can effectively classify cells and identify their signal activation states in human tissue. Overall, the PECAb system can serve as a comprehensive platform for analyzing complex cell processes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.16.562587">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.16.562587" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.16.562587">
        <p class="paperTitle">T cells Use Focal Adhesions to Pull Themselves Through Confined Environments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.16.562587" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.16.562587" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Caillier, A.; Oleksyn, D.; Fowell, D. J.; Miller, J.; Oakes, P. W.</p>
        <p class="info">Score: 16.3, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.16.562587' target='https://doi.org/10.1101/2023.10.16.562587'> 10.1101/2023.10.16.562587</a></p>
        <p class="abstract">Immune cells are highly dynamic and able to migrate through environments with diverse biochemical and mechanical composition. Their migration has classically been defined as amoeboid under the assumption that it is integrin-independent. Here we show that activated primary Th1 T cells require both confinement and extracellular matrix protein to migrate efficiently. This migration is mediated through small and dynamic focal adhesions that are composed of the same proteins associated with canonical mesenchymal focal adhesions, such as integrins, talin, and vinculin. These focal adhesions, furthermore, localize to sites of contractile traction stresses, enabling T cells to pull themselves through confined spaces. Finally, we show that Th1 T cell preferentially follows tracks of other T cells, suggesting that these adhesions are modifying the extracellular matrix to provide additional environmental guidance cues. These results demonstrate not only that the boundaries between amoeboid and mesenchymal migration modes are ambiguous, but that integrin-mediated adhesions play a key role in T cell motility.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=175 SRC=&#34;FIGDIR/small/562587v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (78K):
org.highwire.dtl.DTLVardef@cdf4e3org.highwire.dtl.DTLVardef@1888037org.highwire.dtl.DTLVardef@99c8cdorg.highwire.dtl.DTLVardef@4e133e_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.14.562322">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.14.562322" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.14.562322">
        <p class="paperTitle">Repression of mRNA translation initiation by GIGYF1 via blocking the eIF3-eIF4G1 interaction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.14.562322" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.14.562322" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Choi, J.-H.; Luo, J.; Hesketh, G. G.; Guo, S.; Pistofidis, A.; Ladak, R. J.; An, Y.; Alain, T.; Schmeing, M.; Gingras, A.-C.; Duchaine, T.; Zhang, X.; Sonenberg, N.; Jafarnejad, S. M.</p>
        <p class="info">Score: 15.5, Published: 2023-10-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.14.562322' target='https://doi.org/10.1101/2023.10.14.562322'> 10.1101/2023.10.14.562322</a></p>
        <p class="abstract">Viruses commonly interfere with the function of the eukaryotic translation initiation factor 4G1 (eIF4G1), a pivotal factor in the recruitment of the eIF3 complex and ribosome to the mRNA. This results in the inhibition of general host protein synthesis and redirecting ribosomes toward viral mRNAs. Certain viruses also selectively repress the translation of mRNAs involved in the host antiviral response. GIGYF2 and its interacting cap-binding protein 4EHP enable the transcript-specific repression of mRNA translation mediated by microRNAs and RNA-binding proteins (RBPs). RNA viruses, such as SARS-CoV-2, exploit the GIGYF2/4EHP complex to selectively repress the translation of transcripts such as Ifnb1 mRNA, which encodes the antiviral cytokine Interferon {beta} (IFN-{beta}). Herein, we reveal that GIGYF1, a paralogue of GIGYF2, robustly represses cellular mRNA translation through a distinct mechanism independent of 4EHP. Upon recruitment to a target mRNA by RBPs, the C-terminal region of GIGYF1 binds to subunits of eIF3 at the interaction interface of eIF3-eIF4G1. This disrupts binding of eIF3 to eIF4G1, resulting in mRNA-specific translational repression. This mechanism exerts profound influences on the host cells response to viral infection. Depletion of GIGYF1 induces a robust immune response by derepressing Ifnb1 mRNA translation. Overall, our study highlights a unique mechanism of translational regulation by GIGYF1 that involves sequestering eIF3 and abrogating its binding to eIF4G1. This mechanism can be utilized by RBPs that interact with GIGYF1 to specifically repress the translation of their target mRNAs, significantly affecting critical biological processes, including host-pathogen interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.16.562637">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.16.562637" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.16.562637">
        <p class="paperTitle">Chromosome size-dependent polar ejection force impairs mammalian mitotic error correction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.16.562637" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.16.562637" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chong, M. K.; Rosas Salvans, M. K.; Tran, V.; Dumont, S.</p>
        <p class="info">Score: 15.3, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.16.562637' target='https://doi.org/10.1101/2023.10.16.562637'> 10.1101/2023.10.16.562637</a></p>
        <p class="abstract">Accurate chromosome segregation requires sister kinetochores to biorient, attaching to opposite spindle poles. To this end, the mammalian kinetochore destabilizes incorrect attachments and stabilizes correct ones, but how it discriminates between these is not yet clear. Here, we test the model that kinetochore tension is the stabilizing cue and ask how chromosome size impacts that model. We live image PtK2 cells, with just 14 chromosomes, widely ranging in size, and find that long chromosomes align at the metaphase plate later than short chromosomes. Enriching for errors and imaging error correction live, we show that long chromosomes exhibit a specific delay in correcting attachments. Using chromokinesin overexpression and laser ablation to perturb polar ejection forces, we find that chromosome size and force on arms determine alignment order. Thus, we propose a model where increased force on long chromosomes can falsely stabilize incorrect attachments, delaying their biorientation. As such, long chromosomes may require compensatory mechanisms for correcting errors to avoid chromosomal instability.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562147">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562147" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562147">
        <p class="paperTitle">The H3.3 K36M oncohistone disrupts the establishment of epigenetic memory through loss of DNA methylation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562147" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562147" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sinha, J.; Nickels, J. F.; Thurm, A. R.; Ludwig, C. H.; Archibald, B. N.; Hinks, M. M.; Wan, J.; Fang, D.; Bintu, L.</p>
        <p class="info">Score: 13.2, Published: 2023-10-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562147' target='https://doi.org/10.1101/2023.10.13.562147'> 10.1101/2023.10.13.562147</a></p>
        <p class="abstract">Histone H3.3 is frequently mutated in cancers, with the lysine 36 to methionine mutation (K36M) being a hallmark of chondroblastomas. While it is known that H3.3K36M changes the cellular epigenetic landscape, it remains unclear how it affects the dynamics of gene expression. Here, we use a synthetic reporter to measure the effect of H3.3K36M on silencing and epigenetic memory after recruitment of KRAB: a member of the largest class of human repressors, commonly used in synthetic biology, and associated with H3K9me3. We find that H3.3K36M, which decreases H3K36 methylation, leads to a decrease in epigenetic memory and promoter methylation weeks after KRAB release. We propose a new model for establishment and maintenance of epigenetic memory, where H3K36 methylation is necessary to convert H3K9me3 domains into DNA methylation for stable epigenetic memory. Our quantitative model can inform oncogenic mechanisms and guide development of epigenetic editing tools.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.19.563150">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.19.563150" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.19.563150">
        <p class="paperTitle">CENP-E initiates chromosome congression by opposing Aurora kinases to promote end-on microtubule attachments close to centrosomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.19.563150" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.19.563150" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vukusic, K.; Tolic, I. M.</p>
        <p class="info">Score: 12.3, Published: 2023-10-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.19.563150' target='https://doi.org/10.1101/2023.10.19.563150'> 10.1101/2023.10.19.563150</a></p>
        <p class="abstract">Chromosome congression to the spindle equator, which recurrently fails in cancer cells leading to aneuploidy, requires the kinetochore motor protein CENP-E/kinesin-7. Current models state that chromosomes congress by CENP-E-driven gliding along spindle microtubules. Here, we show that CENP-E is not essential for the congression movement, but for initiating movement through the formation of end-on attachments. We reached this conclusion by large-scale live-cell imaging under varying activity of CENP-E using lattice light-sheet microscopy. We found that congression without CENP-E is delayed by the activity of centriole-dependent Aurora A and outer-kinetochore-localized Aurora B kinases, which limit the formation of end-on attachments. CENP-E bypasses this inhibition by causing a decrease in Aurora B activity and in phosphorylation of its outer-kinetochore targets. Chromosomally unstable cancer cells rely on CENP-E more than non-cancer cells, leading to segregation errors specifically in cancer cells with prolonged mitosis, thereby exposing a mitotic timing-dependent vulnerability of cancer cells.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
