<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cell biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cell biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="CZ CELLxGENE Discover: A single-cell data platform for scalable exploration, analysis and modeling of aggregated data
Authors: CZI Single-Cell Biology, ; Abdulla, S.; Aevermann, B.; Assis, P.; Badajoz, S.; Bell, S. M.; Bezzi, E.; Cakir, B.; Chaffer, J.; Chambers, S.; Cherry, J. M.; Chi, T.; Chien, J.; Dorman, L.; Gloria, N.; Garcia-Nieto, P.; Hastie, M.; Hegeman, D.; Hilton, J.; Huang, T.; Infeld, A.; Istrate, A.-M.; Jelic, I.; Katsuya, K.; Kim, Y.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cell-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cell biology" />
<meta property="og:description" content="CZ CELLxGENE Discover: A single-cell data platform for scalable exploration, analysis and modeling of aggregated data
Authors: CZI Single-Cell Biology, ; Abdulla, S.; Aevermann, B.; Assis, P.; Badajoz, S.; Bell, S. M.; Bezzi, E.; Cakir, B.; Chaffer, J.; Chambers, S.; Cherry, J. M.; Chi, T.; Chien, J.; Dorman, L.; Gloria, N.; Garcia-Nieto, P.; Hastie, M.; Hegeman, D.; Hilton, J.; Huang, T.; Infeld, A.; Istrate, A.-M.; Jelic, I.; Katsuya, K.; Kim, Y." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cell-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-08T10:39:23+00:00" />
<meta property="article:modified_time" content="2023-11-08T10:39:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cell biology"/>
<meta name="twitter:description" content="CZ CELLxGENE Discover: A single-cell data platform for scalable exploration, analysis and modeling of aggregated data
Authors: CZI Single-Cell Biology, ; Abdulla, S.; Aevermann, B.; Assis, P.; Badajoz, S.; Bell, S. M.; Bezzi, E.; Cakir, B.; Chaffer, J.; Chambers, S.; Cherry, J. M.; Chi, T.; Chien, J.; Dorman, L.; Gloria, N.; Garcia-Nieto, P.; Hastie, M.; Hegeman, D.; Hilton, J.; Huang, T.; Infeld, A.; Istrate, A.-M.; Jelic, I.; Katsuya, K.; Kim, Y."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cell biology",
      "item": "https://trxiv.yorks0n.com/posts/cell-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cell biology",
  "name": "cell biology",
  "description": "CZ CELLxGENE Discover: A single-cell data platform for scalable exploration, analysis and modeling of aggregated data\nAuthors: CZI Single-Cell Biology, ; Abdulla, S.; Aevermann, B.; Assis, P.; Badajoz, S.; Bell, S. M.; Bezzi, E.; Cakir, B.; Chaffer, J.; Chambers, S.; Cherry, J. M.; Chi, T.; Chien, J.; Dorman, L.; Gloria, N.; Garcia-Nieto, P.; Hastie, M.; Hegeman, D.; Hilton, J.; Huang, T.; Infeld, A.; Istrate, A.-M.; Jelic, I.; Katsuya, K.; Kim, Y.",
  "keywords": [
    
  ],
  "articleBody": " CZ CELLxGENE Discover: A single-cell data platform for scalable exploration, analysis and modeling of aggregated data\nAuthors: CZI Single-Cell Biology, ; Abdulla, S.; Aevermann, B.; Assis, P.; Badajoz, S.; Bell, S. M.; Bezzi, E.; Cakir, B.; Chaffer, J.; Chambers, S.; Cherry, J. M.; Chi, T.; Chien, J.; Dorman, L.; Gloria, N.; Garcia-Nieto, P.; Hastie, M.; Hegeman, D.; Hilton, J.; Huang, T.; Infeld, A.; Istrate, A.-M.; Jelic, I.; Katsuya, K.; Kim, Y. J.; Liang, K.; Lin, M.; Lombardo, M.; Marshall, B.; Martin, B.; McDade, F.; Megill, C.; Patel, N.; Predeus, A.; Raymor, B.; Robatmili, B.; Rogers, D.; Rutherford, E.; Sadgat, D.; Shin, A.; Small, C.; Smith, T.; Sridharan, P.; Tarashansky, A.; Tavares, N.; Thomas, H.; Tolop\nScore: 48.3, Published: 2023-11-02 DOI: 10.1101/2023.10.30.563174\nHundreds of millions of single cells have been analyzed to date using high throughput transcriptomic methods, thanks to technological advances driving the increasingly rapid generation of single-cell data. This provides an exciting opportunity for unlocking new insights into health and disease, made possible by meta-analysis that span diverse datasets building on recent advances in large language models and other machine learning approaches. Despite the promise of these and emerging analytical tools for analyzing large amounts of data, a major challenge remains the sheer number of datasets and inconsistent format, data models and accessibility. Many datasets are available via unique portals platforms that often lack interoperability. Here, we present CZ CellxGene Discover (cellxgene.cziscience.com), a data platform that provides curated and interoperable data. This single-cell data resource, available via a free-to-use online data portal, hosts a growing corpus of community contributed data that spans more than 50 million unique cells. Curated, standardized, and associated with consistent cell-level metadata, this collection of interoperable single-cell transcriptomic data is the largest of its kind. A suite of tools and features enables accessibility and reusability of the data via both computational and visual interfaces to allow researchers to rapidly explore individual datasets and perform cross-corpus analysis. This functionality is enabling meta-analyses of tens of millions of cells across studies and tissues and providing global views of human cells at the resolution of single cells.\nNanometer-scale views of visual cortex reveal anatomical features of primary cilia poised to detect synaptic spillover\nAuthors: Ott, C. M.; Torres, R.; Kuan, T.-S.; Kuan, A. T.; Buchanan, J.; Elabbady, L.; Seshamani, S.; Bodor, A. L.; Collman, F. C.; Bock, D. D.; Lee, W.-C. A.; Macarico da Costa, N.; Lippincott-Schwartz, J.\nScore: 36.6, Published: 2023-11-06 DOI: 10.1101/2023.10.31.564838\nA primary cilium is a thin membrane-bound extension off a cell surface that contains receptors for perceiving and transmitting signals that modulate cell state and activity. While many cell types have a primary cilium, little is known about primary cilia in the brain, where they are less accessible than cilia on cultured cells or epithelial tissues and protrude from cell bodies into a deep, dense network of glial and neuronal processes. Here, we investigated cilia frequency, internal structure, shape, and position in large, high-resolution transmission electron microscopy volumes of mouse primary visual cortex. Cilia extended from the cell bodies of nearly all excitatory and inhibitory neurons, astrocytes, and oligodendrocyte precursor cells (OPCs), but were absent from oligodendrocytes and microglia. Structural comparisons revealed that the membrane structure at the base of the cilium and the microtubule organization differed between neurons and glia. OPC cilia were distinct in that they were the shortest and contained pervasive internal vesicles only occasionally observed in neuron and astrocyte cilia. Investigating cilia-proximal features revealed that many cilia were directly adjacent to synapses, suggesting cilia are well poised to encounter locally released signaling molecules. The internal anatomy, including microtubule changes and centriole location, defined key structural features including cilium placement and shape. Together, the anatomical insights both within and around neuron and glia cilia provide new insights into cilia formation and function across cell types in the brain.\nA STING-CASM-GABARAP Pathway Activates LRRK2 at Lysosomes\nAuthors: Bentley-DeSousa, A.; Ferguson, S.\nScore: 33.3, Published: 2023-11-01 DOI: 10.1101/2023.10.31.564602\nMutations that increase LRRK2 kinase activity have been linked to Parkinsons disease and Crohns disease. LRRK2 is also activated by lysosome damage evoked by chemical and pathogenic stimuli. However, the endogenous cellular mechanisms that control LRRK2 kinase activity are not well understood. In this study, we identify signaling through Stimulator of Interferon Genes (STING) as an upstream activator of LRRK2. This LRRK2 activation occurs via the Conjugation of ATG8 to Single Membranes (CASM) pathway. We furthermore establish that multiple chemical stimuli that perturb lysosomal homeostasis also converge on CASM to activate LRRK2. Although CASM mediates the lipidation of multiple ATG8 protein family members, LRRK2 lysosome recruitment and kinase activation is highly dependent on an interaction with the GABARAP member of this family. Collectively these results define a pathway that integrates multiple stimuli at lysosomes to control the kinase activity of LRRK2. Aberrant activation this pathway may be of relevance in both Parkinsons and Crohns diseases.\nTunneling nanotubes enable intercellular transfer in zebrafish embryos\nAuthors: Korenkova, O.; Liu, S.; Prlesi, I.; Pepe, A.; Albadri, S.; Del Bene, F.; Zurzolo, C.\nScore: 29.9, Published: 2023-10-12 DOI: 10.1101/2023.10.12.562005\nTunneling nanotubes (TNTs) are thin intercellular connections facilitating the transport of diverse cargoes, ranging from ions to organelles. While TNT studies have predominantly been conducted in cell cultures, the existence of open-ended TNTs within live organisms remains unverified. Despite the observation of intercellular connections during embryonic development across various species, their functional role has not been confirmed. In this study, we performed mosaic labeling of gastrula cells in zebrafish embryos to demonstrate the coexistence of TNT-like structures alongside other cellular protrusions. These embryonic TNT-like connections exhibited similar morphology to TNTs described in cell culture, appeared to have similar formation mechanisms and could be induced by Eps8 overexpression and CK666 treatment. Most notably, to classify them as TNTs, we demonstrated their capability to transfer both soluble cargoes and organelles, which is a defining feature of open-ended TNTs. This study marks the first demonstration of functional TNTs in a living embryo.\nAn immunohistochemical atlas of necroptotic pathway expression\nAuthors: Chiou, S.; Al-Ani, A.; Pan, Y.; Patel, K.; Kong, I.; Whitehead, L.; Light, A.; Young, S.; Barrios, M.; Sargeant, C.; Rajasekhar, P.; Zhu, L.; Hempel, A.; Lin, A.; Rickard, J.; Hall, C.; Gangatirkar, P.; Yip, R.; Cawthorne, W.; Jacobsen, A.; Horne, C.; Ioannidis, L.; Hansen, D.; Day, J.; Wicks, I.; Law, C. W.; Ritchie, M. E.; Bowden, R.; Hildebrand, J.; O'Reilly, L.; Silke, J.; Giulino-Roth, L.; Tsui, E.; Rogers, K.; Hawkins, E.; Christensen, B.; Murphy, J. M.; Samson, A. L.\nScore: 18.1, Published: 2023-11-02 DOI: 10.1101/2023.10.31.565039\nNecroptosis is a lytic form of regulated cell death reported to contribute to inflammatory diseases of the gut, skin and lung, as well as ischemic-reperfusion injuries of the kidney, heart and brain. However, precise identification of the cells and tissues that undergo necroptotic cell death in vivo has proven challenging in the absence of robust reagents and protocols for immunohistochemical detection. Here, we provide automated immunohistochemistry protocols to detect core necroptosis regulators - Caspase-8, RIPK1, RIPK3 and MLKL - in formalin-fixed mouse and human tissues. We observed surprising heterogeneity in protein expression within tissues, whereby short-lived immune barrier cells were replete with necroptotic effectors, whereas long-lived cells exhibited reduced RIPK3 or MLKL expression. Local changes in the expression of necroptotic effectors were observed in response to insults such as sterile inflammation, dysbiosis or immune challenge, consistent with necroptosis being dysregulated in disease contexts. This toolbox and methods will facilitate precise localisation and evaluation of necroptotic signaling in vivo.\ncellPLATO: an unsupervised method for identifying cell behaviour in heterogeneous cell trajectory data\nAuthors: Shannon, M. J.; Eisman, S. E.; Lowe, A. R.; Sloan, T.; Mace, E. M.\nScore: 18.8, Published: 2023-11-05 DOI: 10.1101/2023.10.28.564355\nAdvances in imaging, cell segmentation, and cell tracking now routinely produce microscopy datasets of a size and complexity comparable to transcriptomics or proteomics. New tools are required to process this phenomics type data. Cell PLasticity Analysis TOol (cellPLATO) is a Python-based analysis software designed for measurement and classification of diverse cell behaviours based on clustering of parameters of cell morphology and motility. cellPLATO is used after segmentation and tracking of cells from live cell microscopy data. The tool extracts morphological and motility metrics from each cell per timepoint, before being using them to segregate cells into behavioural subtypes with dimensionality reduction. Resultant cell tracks have a behavioural ID for each cell per timepoint corresponding to their changing behaviour over time in a sequence. Similarity analysis allows the grouping of behavioural sequences into discrete trajectories with assigned IDs. Trajectories and underlying behaviours generate a phenotypic fingerprint for each experimental condition, and representative cells are mathematically identified and graphically displayed for human understanding of each subtype. Here, we use cellPLATO to investigate the role of IL-15 in modulating NK cell migration on ICAM-1 or VCAM-1. We find 8 behavioural subsets of NK cells based on their shape and migration dynamics, and 4 trajectories of behaviour. Therefore, using cellPLATO we show that IL-15 increases plasticity between cell migration behaviours and that different integrin ligands induce different forms of NK cell migration.\nFABP7 Progenitors are a Targetable Metabolic Root in the BRCA1 Breast\nAuthors: McCloskey, C. W.; Zhang, B.; Waas, M.; Abazari, G.; Vyas, F.; Aliar, K.; Tharmapalan, P.; Kuttanamkuzhi, A.; Narala, S.; Cruikshank, J.; Hofer, S. O. P.; Jackson, H. W.; Kislinger, T.; Berman, H. K.; Khokha, R.\nScore: 16.9, Published: 2023-11-04 DOI: 10.1101/2023.11.02.565360\nIt has been nearly 3 decades since the discovery of the BRCA1/2 genes and their link to breast cancer risk, with prophylactic mastectomy remaining the primary management option for these high-risk mutation carriers. The current paucity of interception strategies is due to undefined, targetable cancer precursor populations in the high-risk breast. Despite known cellular alterations in the BRCA1 breast, epithelial populations at the root of unwarranted cell state transitions remain unresolved. Here, we identify a root progenitor population that is dysregulated in BRCA1 carriers stemming from the metabolic role of BRCA1. This fatty-acid binding protein 7 (FABP7) expressing luminal progenitor population is spatially confined to the mammary ducts, has enhanced clonogenic capacity, and is the predicted origin of mixed basal-luminal differentiation in the BRCA1 but not BRCA2 breast. We show global H3K27 acetylation is reduced within ductal FABP7 cells in BRCA1 carriers in situ, linking to a non-canonical metabolic role of BRCA1 in regulating acetyl-CoA pools and de novo fatty acid synthesis. We demonstrate FABP7 progenitor capacity is preferentially ablated in BRCA1 carriers through inhibition of fatty acid metabolism using an FDA-approved fatty acid synthase (FASN) inhibitor. This study lays the foundation for metabolic control of breast progenitor dynamics to mitigate breast cancer risk in the BRCA1 breast.\nThe role of the AP-1 adaptor complex in outgoing and incoming membrane traffic\nAuthors: Robinson, M. S.; Antrobus, R.; Sanger, A.; Davies, A. K.; Gershlick, D. C.\nScore: 12.3, Published: 2023-11-04 DOI: 10.1101/2023.11.02.565271\nThe AP-1 adaptor complex is found in all eukaryotes, but it has been implicated in different pathways in different organisms. To look directly at AP-1 function, we generated stably transduced HeLa cells co-expressing tagged AP-1 and various tagged membrane proteins. Live cell imaging showed that AP-1 is recruited onto tubular carriers trafficking from the Golgi apparatus to the plasma membrane, as well as onto transferrin-containing early/recycling endosomes. Analysis of single AP-1 vesicles showed that they are a heterogeneous population, which start to sequester cargo by 30 minutes after exit from the ER. Vesicle capture showed that AP-1 vesicles contain transmembrane proteins found at the TGN and early/recycling endosomes, as well as lysosomal hydrolases, but very little of the anterograde adaptor GGA2. Together, our results support a model in which AP-1 retrieves proteins from post-Golgi compartments back to the TGN, analogous to COPI's role in the early secretory pathway. We propose that this is the function of AP-1 in all eukaryotes.\nChemically induced proximity reveals mechanotransduction of a meiotic checkpoint at the nuclear envelope\nAuthors: Liu, C.; Dernburg, A. F.\nScore: 12.1, Published: 2023-10-28 DOI: 10.1101/2023.10.28.564506\nSuccessful sexual reproduction relies on robust quality control during meiosis. Assembly of the synaptonemal complex between homologous chromosomes (synapsis) regulates meiotic recombination and is crucial for accurate chromosome segregation in most eukaryotes. Synapsis defects can trigger cell cycle delays and, in some cases, apoptosis. Here, by developing and deploying a new chemically induced proximity system, we iden-tify key players in this quality control pathway in Caenorhabditis elegans. We find that persistence of the Polo-like kinase PLK-2 at pairing centers, specialized chromosome regions that interact with the nuclear envelope to promote homolog pairing and synapsis, induces apoptosis of oocytes by phosphorylating and destabilizing the nuclear lamina. Unexpectedly, we find that a mechanosensitive Piezo1/PEZO-1 channel localizes to the nuclear envelope and is required to transduce this signal to promote apoptosis. Thus, mechanosensitive ion channels play essential roles in detecting nuclear events and triggering apoptosis during gamete production. One-sentence summaryDestabilization of the nuclear lamina triggers Piezo-dependent germline apoptosis.\nPhase Separation of a Novel form of Euchromatic Histone Methyltransferase1 (EHMT1N/C) into cytoplasmic RNA viral Inclusion bodies facilitates their coalescence, thereby enhancing viral replication\nAuthors: Biligiri, K. K.; Sharma, N. R.; Mohanty, A.; Sarkar, D. P.; Vemula, P.; Rampalli, S.\nScore: 8.9, Published: 2023-11-02 DOI: 10.1101/2023.11.01.564969\nProtein lysine methyltransferases (PKMTs) methylate histone and non-histone proteins to regulate biological outcomes such as development and disease including viral infection. While PKMTs have been extensively studied for modulating the antiviral responses via host gene regulation, their role in methylation of proteins encoded by viruses and its impact on host-pathogen interactions remain poorly understood. In this study, we discovered a distinct nucleo-cytoplasmic form of Euchromatic Histone Methyltransferase1(EHMT1N/C), a PKMT, that phase separates into viral inclusion bodies (IBs) upon cytoplasmic RNA-virus infection (Sendai Virus). EHMT1N/C interacts with cytoplasmic EHMT2 and methylates SeV-Nucleoprotein upon infection. Elevated nucleoprotein methylation during infection correlated with coalescence of small IBs into large mature platforms for efficient replication. Inhibition of EHMT activity by pharmacological inhibitors or genetic depletion of EHMT1N/C reduced the size of IBs with a concomitant reduction in replication. Since IB formation is conserved among all cytoplasmic RNA-viruses, our study will have strong implications in understanding the mechanisms regulating IB formation, discerning RNA viral pathogenesis and designing therapeutic strategies.\n",
  "wordCount" : "2347",
  "inLanguage": "en",
  "datePublished": "2023-11-08T10:39:23Z",
  "dateModified": "2023-11-08T10:39:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cell-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cell biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 8, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.563174">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.563174" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.563174">
        <p class="paperTitle">CZ CELLxGENE Discover: A single-cell data platform for scalable exploration, analysis and modeling of aggregated data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.563174" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.563174" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: CZI Single-Cell Biology,  ; Abdulla, S.; Aevermann, B.; Assis, P.; Badajoz, S.; Bell, S. M.; Bezzi, E.; Cakir, B.; Chaffer, J.; Chambers, S.; Cherry, J. M.; Chi, T.; Chien, J.; Dorman, L.; Gloria, N.; Garcia-Nieto, P.; Hastie, M.; Hegeman, D.; Hilton, J.; Huang, T.; Infeld, A.; Istrate, A.-M.; Jelic, I.; Katsuya, K.; Kim, Y. J.; Liang, K.; Lin, M.; Lombardo, M.; Marshall, B.; Martin, B.; McDade, F.; Megill, C.; Patel, N.; Predeus, A.; Raymor, B.; Robatmili, B.; Rogers, D.; Rutherford, E.; Sadgat, D.; Shin, A.; Small, C.; Smith, T.; Sridharan, P.; Tarashansky, A.; Tavares, N.; Thomas, H.; Tolop</p>
        <p class="info">Score: 48.3, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.563174' target='https://doi.org/10.1101/2023.10.30.563174'> 10.1101/2023.10.30.563174</a></p>
        <p class="abstract">Hundreds of millions of single cells have been analyzed to date using high throughput transcriptomic methods, thanks to technological advances driving the increasingly rapid generation of single-cell data. This provides an exciting opportunity for unlocking new insights into health and disease, made possible by meta-analysis that span diverse datasets building on recent advances in large language models and other machine learning approaches. Despite the promise of these and emerging analytical tools for analyzing large amounts of data, a major challenge remains the sheer number of datasets and inconsistent format, data models and accessibility. Many datasets are available via unique portals platforms that often lack interoperability. Here, we present CZ CellxGene Discover (cellxgene.cziscience.com), a data platform that provides curated and interoperable data. This single-cell data resource, available via a free-to-use online data portal, hosts a growing corpus of community contributed data that spans more than 50 million unique cells. Curated, standardized, and associated with consistent cell-level metadata, this collection of interoperable single-cell transcriptomic data is the largest of its kind. A suite of tools and features enables accessibility and reusability of the data via both computational and visual interfaces to allow researchers to rapidly explore individual datasets and perform cross-corpus analysis. This functionality is enabling meta-analyses of tens of millions of cells across studies and tissues and providing global views of human cells at the resolution of single cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.564838">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.564838" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.564838">
        <p class="paperTitle">Nanometer-scale views of visual cortex reveal anatomical features of primary cilia poised to detect synaptic spillover</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.564838" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.564838" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ott, C. M.; Torres, R.; Kuan, T.-S.; Kuan, A. T.; Buchanan, J.; Elabbady, L.; Seshamani, S.; Bodor, A. L.; Collman, F. C.; Bock, D. D.; Lee, W.-C. A.; Macarico da Costa, N.; Lippincott-Schwartz, J.</p>
        <p class="info">Score: 36.6, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.564838' target='https://doi.org/10.1101/2023.10.31.564838'> 10.1101/2023.10.31.564838</a></p>
        <p class="abstract">A primary cilium is a thin membrane-bound extension off a cell surface that contains receptors for perceiving and transmitting signals that modulate cell state and activity. While many cell types have a primary cilium, little is known about primary cilia in the brain, where they are less accessible than cilia on cultured cells or epithelial tissues and protrude from cell bodies into a deep, dense network of glial and neuronal processes. Here, we investigated cilia frequency, internal structure, shape, and position in large, high-resolution transmission electron microscopy volumes of mouse primary visual cortex. Cilia extended from the cell bodies of nearly all excitatory and inhibitory neurons, astrocytes, and oligodendrocyte precursor cells (OPCs), but were absent from oligodendrocytes and microglia. Structural comparisons revealed that the membrane structure at the base of the cilium and the microtubule organization differed between neurons and glia. OPC cilia were distinct in that they were the shortest and contained pervasive internal vesicles only occasionally observed in neuron and astrocyte cilia. Investigating cilia-proximal features revealed that many cilia were directly adjacent to synapses, suggesting cilia are well poised to encounter locally released signaling molecules. The internal anatomy, including microtubule changes and centriole location, defined key structural features including cilium placement and shape. Together, the anatomical insights both within and around neuron and glia cilia provide new insights into cilia formation and function across cell types in the brain.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.564602">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.564602" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.564602">
        <p class="paperTitle">A STING-CASM-GABARAP Pathway Activates LRRK2 at Lysosomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.564602" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.564602" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bentley-DeSousa, A.; Ferguson, S.</p>
        <p class="info">Score: 33.3, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.564602' target='https://doi.org/10.1101/2023.10.31.564602'> 10.1101/2023.10.31.564602</a></p>
        <p class="abstract">Mutations that increase LRRK2 kinase activity have been linked to Parkinsons disease and Crohns disease. LRRK2 is also activated by lysosome damage evoked by chemical and pathogenic stimuli. However, the endogenous cellular mechanisms that control LRRK2 kinase activity are not well understood. In this study, we identify signaling through Stimulator of Interferon Genes (STING) as an upstream activator of LRRK2. This LRRK2 activation occurs via the Conjugation of ATG8 to Single Membranes (CASM) pathway. We furthermore establish that multiple chemical stimuli that perturb lysosomal homeostasis also converge on CASM to activate LRRK2. Although CASM mediates the lipidation of multiple ATG8 protein family members, LRRK2 lysosome recruitment and kinase activation is highly dependent on an interaction with the GABARAP member of this family. Collectively these results define a pathway that integrates multiple stimuli at lysosomes to control the kinase activity of LRRK2. Aberrant activation this pathway may be of relevance in both Parkinsons and Crohns diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.12.562005">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.12.562005" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.12.562005">
        <p class="paperTitle">Tunneling nanotubes enable intercellular transfer in zebrafish embryos</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.12.562005" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.12.562005" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Korenkova, O.; Liu, S.; Prlesi, I.; Pepe, A.; Albadri, S.; Del Bene, F.; Zurzolo, C.</p>
        <p class="info">Score: 29.9, Published: 2023-10-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.12.562005' target='https://doi.org/10.1101/2023.10.12.562005'> 10.1101/2023.10.12.562005</a></p>
        <p class="abstract">Tunneling nanotubes (TNTs) are thin intercellular connections facilitating the transport of diverse cargoes, ranging from ions to organelles. While TNT studies have predominantly been conducted in cell cultures, the existence of open-ended TNTs within live organisms remains unverified. Despite the observation of intercellular connections during embryonic development across various species, their functional role has not been confirmed. In this study, we performed mosaic labeling of gastrula cells in zebrafish embryos to demonstrate the coexistence of TNT-like structures alongside other cellular protrusions. These embryonic TNT-like connections exhibited similar morphology to TNTs described in cell culture, appeared to have similar formation mechanisms and could be induced by Eps8 overexpression and CK666 treatment. Most notably, to classify them as TNTs, we demonstrated their capability to transfer both soluble cargoes and organelles, which is a defining feature of open-ended TNTs. This study marks the first demonstration of functional TNTs in a living embryo.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.565039">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.565039" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.565039">
        <p class="paperTitle">An immunohistochemical atlas of necroptotic pathway expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.565039" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.565039" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chiou, S.; Al-Ani, A.; Pan, Y.; Patel, K.; Kong, I.; Whitehead, L.; Light, A.; Young, S.; Barrios, M.; Sargeant, C.; Rajasekhar, P.; Zhu, L.; Hempel, A.; Lin, A.; Rickard, J.; Hall, C.; Gangatirkar, P.; Yip, R.; Cawthorne, W.; Jacobsen, A.; Horne, C.; Ioannidis, L.; Hansen, D.; Day, J.; Wicks, I.; Law, C. W.; Ritchie, M. E.; Bowden, R.; Hildebrand, J.; O&#39;Reilly, L.; Silke, J.; Giulino-Roth, L.; Tsui, E.; Rogers, K.; Hawkins, E.; Christensen, B.; Murphy, J. M.; Samson, A. L.</p>
        <p class="info">Score: 18.1, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.565039' target='https://doi.org/10.1101/2023.10.31.565039'> 10.1101/2023.10.31.565039</a></p>
        <p class="abstract">Necroptosis is a lytic form of regulated cell death reported to contribute to inflammatory diseases of the gut, skin and lung, as well as ischemic-reperfusion injuries of the kidney, heart and brain. However, precise identification of the cells and tissues that undergo necroptotic cell death in vivo has proven challenging in the absence of robust reagents and protocols for immunohistochemical detection. Here, we provide automated immunohistochemistry protocols to detect core necroptosis regulators - Caspase-8, RIPK1, RIPK3 and MLKL - in formalin-fixed mouse and human tissues. We observed surprising heterogeneity in protein expression within tissues, whereby short-lived immune barrier cells were replete with necroptotic effectors, whereas long-lived cells exhibited reduced RIPK3 or MLKL expression. Local changes in the expression of necroptotic effectors were observed in response to insults such as sterile inflammation, dysbiosis or immune challenge, consistent with necroptosis being dysregulated in disease contexts. This toolbox and methods will facilitate precise localisation and evaluation of necroptotic signaling in vivo.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.28.564355">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.28.564355" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.28.564355">
        <p class="paperTitle">cellPLATO: an unsupervised method for identifying cell behaviour in heterogeneous cell trajectory data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.28.564355" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.28.564355" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shannon, M. J.; Eisman, S. E.; Lowe, A. R.; Sloan, T.; Mace, E. M.</p>
        <p class="info">Score: 18.8, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.28.564355' target='https://doi.org/10.1101/2023.10.28.564355'> 10.1101/2023.10.28.564355</a></p>
        <p class="abstract">Advances in imaging, cell segmentation, and cell tracking now routinely produce microscopy datasets of a size and complexity comparable to transcriptomics or proteomics. New tools are required to process this phenomics type data. Cell PLasticity Analysis TOol (cellPLATO) is a Python-based analysis software designed for measurement and classification of diverse cell behaviours based on clustering of parameters of cell morphology and motility. cellPLATO is used after segmentation and tracking of cells from live cell microscopy data. The tool extracts morphological and motility metrics from each cell per timepoint, before being using them to segregate cells into behavioural subtypes with dimensionality reduction. Resultant cell tracks have a behavioural ID for each cell per timepoint corresponding to their changing behaviour over time in a sequence. Similarity analysis allows the grouping of behavioural sequences into discrete trajectories with assigned IDs. Trajectories and underlying behaviours generate a phenotypic fingerprint for each experimental condition, and representative cells are mathematically identified and graphically displayed for human understanding of each subtype. Here, we use cellPLATO to investigate the role of IL-15 in modulating NK cell migration on ICAM-1 or VCAM-1. We find 8 behavioural subsets of NK cells based on their shape and migration dynamics, and 4 trajectories of behaviour. Therefore, using cellPLATO we show that IL-15 increases plasticity between cell migration behaviours and that different integrin ligands induce different forms of NK cell migration.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.565360">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.565360" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.565360">
        <p class="paperTitle">FABP7 Progenitors are a Targetable Metabolic Root in the BRCA1 Breast</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.565360" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.565360" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McCloskey, C. W.; Zhang, B.; Waas, M.; Abazari, G.; Vyas, F.; Aliar, K.; Tharmapalan, P.; Kuttanamkuzhi, A.; Narala, S.; Cruikshank, J.; Hofer, S. O. P.; Jackson, H. W.; Kislinger, T.; Berman, H. K.; Khokha, R.</p>
        <p class="info">Score: 16.9, Published: 2023-11-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.565360' target='https://doi.org/10.1101/2023.11.02.565360'> 10.1101/2023.11.02.565360</a></p>
        <p class="abstract">It has been nearly 3 decades since the discovery of the BRCA1/2 genes and their link to breast cancer risk, with prophylactic mastectomy remaining the primary management option for these high-risk mutation carriers. The current paucity of interception strategies is due to undefined, targetable cancer precursor populations in the high-risk breast. Despite known cellular alterations in the BRCA1 breast, epithelial populations at the root of unwarranted cell state transitions remain unresolved. Here, we identify a root progenitor population that is dysregulated in BRCA1 carriers stemming from the metabolic role of BRCA1. This fatty-acid binding protein 7 (FABP7) expressing luminal progenitor population is spatially confined to the mammary ducts, has enhanced clonogenic capacity, and is the predicted origin of mixed basal-luminal differentiation in the BRCA1 but not BRCA2 breast. We show global H3K27 acetylation is reduced within ductal FABP7 cells in BRCA1 carriers in situ, linking to a non-canonical metabolic role of BRCA1 in regulating acetyl-CoA pools and de novo fatty acid synthesis. We demonstrate FABP7 progenitor capacity is preferentially ablated in BRCA1 carriers through inhibition of fatty acid metabolism using an FDA-approved fatty acid synthase (FASN) inhibitor. This study lays the foundation for metabolic control of breast progenitor dynamics to mitigate breast cancer risk in the BRCA1 breast.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.565271">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.565271" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.565271">
        <p class="paperTitle">The role of the AP-1 adaptor complex in outgoing and incoming membrane traffic</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.565271" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.565271" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Robinson, M. S.; Antrobus, R.; Sanger, A.; Davies, A. K.; Gershlick, D. C.</p>
        <p class="info">Score: 12.3, Published: 2023-11-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.565271' target='https://doi.org/10.1101/2023.11.02.565271'> 10.1101/2023.11.02.565271</a></p>
        <p class="abstract">The AP-1 adaptor complex is found in all eukaryotes, but it has been implicated in different pathways in different organisms. To look directly at AP-1 function, we generated stably transduced HeLa cells co-expressing tagged AP-1 and various tagged membrane proteins. Live cell imaging showed that AP-1 is recruited onto tubular carriers trafficking from the Golgi apparatus to the plasma membrane, as well as onto transferrin-containing early/recycling endosomes. Analysis of single AP-1 vesicles showed that they are a heterogeneous population, which start to sequester cargo by 30 minutes after exit from the ER. Vesicle capture showed that AP-1 vesicles contain transmembrane proteins found at the TGN and early/recycling endosomes, as well as lysosomal hydrolases, but very little of the anterograde adaptor GGA2. Together, our results support a model in which AP-1 retrieves proteins from post-Golgi compartments back to the TGN, analogous to COPI&#39;s role in the early secretory pathway. We propose that this is the function of AP-1 in all eukaryotes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.28.564506">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.28.564506" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.28.564506">
        <p class="paperTitle">Chemically induced proximity reveals mechanotransduction of a meiotic checkpoint at the nuclear envelope</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.28.564506" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.28.564506" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, C.; Dernburg, A. F.</p>
        <p class="info">Score: 12.1, Published: 2023-10-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.28.564506' target='https://doi.org/10.1101/2023.10.28.564506'> 10.1101/2023.10.28.564506</a></p>
        <p class="abstract">Successful sexual reproduction relies on robust quality control during meiosis. Assembly of the synaptonemal complex between homologous chromosomes (synapsis) regulates meiotic recombination and is crucial for accurate chromosome segregation in most eukaryotes. Synapsis defects can trigger cell cycle delays and, in some cases, apoptosis. Here, by developing and deploying a new chemically induced proximity system, we iden-tify key players in this quality control pathway in Caenorhabditis elegans. We find that persistence of the Polo-like kinase PLK-2 at pairing centers, specialized chromosome regions that interact with the nuclear envelope to promote homolog pairing and synapsis, induces apoptosis of oocytes by phosphorylating and destabilizing the nuclear lamina. Unexpectedly, we find that a mechanosensitive Piezo1/PEZO-1 channel localizes to the nuclear envelope and is required to transduce this signal to promote apoptosis. Thus, mechanosensitive ion channels play essential roles in detecting nuclear events and triggering apoptosis during gamete production.

One-sentence summaryDestabilization of the nuclear lamina triggers Piezo-dependent germline apoptosis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.564969">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.564969" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.564969">
        <p class="paperTitle">Phase Separation of a Novel form of Euchromatic Histone Methyltransferase1 (EHMT1N/C) into cytoplasmic RNA viral Inclusion bodies facilitates their coalescence, thereby enhancing viral replication</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.564969" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.564969" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Biligiri, K. K.; Sharma, N. R.; Mohanty, A.; Sarkar, D. P.; Vemula, P.; Rampalli, S.</p>
        <p class="info">Score: 8.9, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.564969' target='https://doi.org/10.1101/2023.11.01.564969'> 10.1101/2023.11.01.564969</a></p>
        <p class="abstract">Protein lysine methyltransferases (PKMTs) methylate histone and non-histone proteins to regulate biological outcomes such as development and disease including viral infection. While PKMTs have been extensively studied for modulating the antiviral responses via host gene regulation, their role in methylation of proteins encoded by viruses and its impact on host-pathogen interactions remain poorly understood. In this study, we discovered a distinct nucleo-cytoplasmic form of Euchromatic Histone Methyltransferase1(EHMT1N/C), a PKMT, that phase separates into viral inclusion bodies (IBs) upon cytoplasmic RNA-virus infection (Sendai Virus). EHMT1N/C interacts with cytoplasmic EHMT2 and methylates SeV-Nucleoprotein upon infection. Elevated nucleoprotein methylation during infection correlated with coalescence of small IBs into large mature platforms for efficient replication. Inhibition of EHMT activity by pharmacological inhibitors or genetic depletion of EHMT1N/C reduced the size of IBs with a concomitant reduction in replication. Since IB formation is conserved among all cytoplasmic RNA-viruses, our study will have strong implications in understanding the mechanisms regulating IB formation, discerning RNA viral pathogenesis and designing therapeutic strategies.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
