<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cell biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cell biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Self-extinguishing relay waves enable homeostaticcontrol of human neutrophil swarming
Authors: Strickland, J.; Pan, D.; Godfrey, C.; Kim, J. S.; Hopke, A.; Degrange, M.; Villavicencio, B.; Mansour, M. K.; Zerbe, C. S.; Irimia, D.; Amir, A.; Weiner, O. D.
Score: 313.3, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546744
Neutrophils exhibit self-amplified swarming to sites of injury and infection. How swarming is controlled to ensure the proper level of neutrophil recruitment is unknown. Using an ex vivo model of infection, we find that human neutrophils use active relay to generate multiple pulsatile waves of swarming signals.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cell-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cell biology" />
<meta property="og:description" content="Self-extinguishing relay waves enable homeostaticcontrol of human neutrophil swarming
Authors: Strickland, J.; Pan, D.; Godfrey, C.; Kim, J. S.; Hopke, A.; Degrange, M.; Villavicencio, B.; Mansour, M. K.; Zerbe, C. S.; Irimia, D.; Amir, A.; Weiner, O. D.
Score: 313.3, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546744
Neutrophils exhibit self-amplified swarming to sites of injury and infection. How swarming is controlled to ensure the proper level of neutrophil recruitment is unknown. Using an ex vivo model of infection, we find that human neutrophils use active relay to generate multiple pulsatile waves of swarming signals." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cell-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-02T10:39:32+00:00" />
<meta property="article:modified_time" content="2023-07-02T10:39:32+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cell biology"/>
<meta name="twitter:description" content="Self-extinguishing relay waves enable homeostaticcontrol of human neutrophil swarming
Authors: Strickland, J.; Pan, D.; Godfrey, C.; Kim, J. S.; Hopke, A.; Degrange, M.; Villavicencio, B.; Mansour, M. K.; Zerbe, C. S.; Irimia, D.; Amir, A.; Weiner, O. D.
Score: 313.3, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546744
Neutrophils exhibit self-amplified swarming to sites of injury and infection. How swarming is controlled to ensure the proper level of neutrophil recruitment is unknown. Using an ex vivo model of infection, we find that human neutrophils use active relay to generate multiple pulsatile waves of swarming signals."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cell biology",
      "item": "https://trxiv.yorks0n.com/posts/cell-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cell biology",
  "name": "cell biology",
  "description": "Self-extinguishing relay waves enable homeostaticcontrol of human neutrophil swarming\nAuthors: Strickland, J.; Pan, D.; Godfrey, C.; Kim, J. S.; Hopke, A.; Degrange, M.; Villavicencio, B.; Mansour, M. K.; Zerbe, C. S.; Irimia, D.; Amir, A.; Weiner, O. D.\nScore: 313.3, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546744\nNeutrophils exhibit self-amplified swarming to sites of injury and infection. How swarming is controlled to ensure the proper level of neutrophil recruitment is unknown. Using an ex vivo model of infection, we find that human neutrophils use active relay to generate multiple pulsatile waves of swarming signals.",
  "keywords": [
    
  ],
  "articleBody": " Self-extinguishing relay waves enable homeostaticcontrol of human neutrophil swarming\nAuthors: Strickland, J.; Pan, D.; Godfrey, C.; Kim, J. S.; Hopke, A.; Degrange, M.; Villavicencio, B.; Mansour, M. K.; Zerbe, C. S.; Irimia, D.; Amir, A.; Weiner, O. D.\nScore: 313.3, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546744\nNeutrophils exhibit self-amplified swarming to sites of injury and infection. How swarming is controlled to ensure the proper level of neutrophil recruitment is unknown. Using an ex vivo model of infection, we find that human neutrophils use active relay to generate multiple pulsatile waves of swarming signals. Unlike classic active relay systems such as action potentials, neutrophil swarming relay waves are self-extinguishing, limiting the spatial range of cell recruitment. We identify an NADPH-oxidase-based negative feedback loop that is needed for this self-extinguishing behavior. Through this circuit, neutrophils adjust the number and size of swarming waves for homeostatic levels of cell recruitment over a wide range of initial cell densities. We link a broken homeostat to neutrophil over-recruitment in the context of human chronic granulomatous disease.\nTREX reveals proteins that bind to specific RNA regions in living cells\nAuthors: Dodel, M.; Guiducci, G.; Dermit, M.; Krishnamurthy, S.; Stojic, L.; Mardakheh, F. K.\nScore: 36.9, Published: 2023-06-30 DOI: 10.1101/2023.06.30.547259\nDifferent regions of RNA molecules can often engage in specific interactions with distinct RNA-binding proteins (RBPs), giving rise to diverse modalities of RNA regulation and function. However, there are currently no methods for unbiased identification of RBPs that interact with specific RNA regions in living cells under endogenous settings. Here, we introduce TREX (Targeted RNase H-mediated extraction of crosslinked RBPs), a highly sensitive approach for identifying proteins that directly bind to specific RNA regions in living cells. We demonstrate that TREX outperforms existing methods in identifying known interactors of U1 snRNA, and reveals endogenous region-specific interactors of NORAD lncRNA. Using TREX, we generated a comprehensive region-by-region interactome for 45S rRNA, uncovering both established and novel interactions that regulate ribosome biogenesis. With its applicability to any RNA in any cell-type, TREX is the first RNA-centric tool for unbiased positional mapping of endogenous RNA-protein interactions in living cells.\nCombinatorial selective ER-phagy remodels the ER during neurogenesis\nAuthors: Hoyer, M. J.; Smith, I. R.; Paoli, J. C.; Jiang, Y.; Paulo, J. A.; Harper, W.\nScore: 28.8, Published: 2023-06-26 DOI: 10.1101/2023.06.26.546565\nThe endoplasmic reticulum (ER) has a vast proteomic landscape to perform many diverse functions including protein and lipid synthesis, calcium ion flux, and inter-organelle communication. The ER proteome is remodeled in part through membrane-embedded receptors linking ER to degradative autophagy machinery (selective ER-phagy)1, 2. A refined tubular ER network3, 4 is formed in neurons within highly polarized dendrites and axons5, 6. Autophagy-deficient neurons in vivo display axonal ER accumulation within synaptic ER boutons,7 and the ER-phagy receptor FAM134B has been genetically linked with human sensory and autonomic neuropathy8, 9. However, mechanisms, including receptor selectivity, that define ER remodeling by autophagy in neurons are limited. Here, we combine a genetically tractable induced neuron (iNeuron) system for monitoring extensive ER remodeling during differentiation with proteomic and computational tools to create a quantitative landscape of ER proteome remodeling via selective autophagy. Through analysis of single and combinatorial ER-phagy receptor mutants, we delineate the extent to which each receptor contributes to both magnitude and selectivity of ER clearance via autophagy for individual ER protein cargos. We define specific subsets of ER curvature-shaping proteins or lumenal proteins as preferred clients for distinct receptors. Using spatial sensors and flux reporters, we demonstrate receptor-specific autophagic capture of ER in axons, which correlates with aberrant ER accumulation in axons of ER-phagy receptor or autophagy-deficient neurons. This molecular inventory of ER proteome remodeling and versatile genetic toolkit provides a quantitative framework for understanding contributions of individual ER-phagy receptors for reshaping ER during cell state transitions.\nSingle cell-resolution in situ sequencing elucidates spatial dynamics of multiple sclerosis lesion and disease evolution\nAuthors: Langseth, C. M.; Kukanja, P.; Rodriguez-Kirby, L. A. R.; Agirre, E.; Raman, A.; Yokota, C.; Avenel, C.; Tiklova, K.; Guerreiro-Cacais, A. O.; Olsson, T.; Hilscher, M. M.; Nilsson, M.; Castelo-Branco, G.\nScore: 23.2, Published: 2023-06-30 DOI: 10.1101/2023.06.29.547074\nMultiple sclerosis (MS) is a neurological disease characterised by multifocal lesions and smouldering pathology. While single-cell analyses have provided insights into neuropathology, cellular processes underlying MS remain poorly understood. We modelled the cellular dynamics of MS by examining temporal and regional rates of disease progression in the experimental autoimmune encephalomyelitis (EAE) mouse model. By performing single-cell spatial expression profiling using In situ sequencing, we annotated disease neighbourhoods during lesion evolution and found centrifugal propagation of active lesions. We demonstrated that disease-associated (DA) glia are dynamic and induced independently of lesions, preceding their formation. Single-cell spatial analysis of human archival MS spinal cord confirmed differential distribution of DA-glia, enabled deconvolution of active and inactive lesions into sub-compartments, and identification of new lesion areas. By establishing a spatial resource of mouse and human MS neuropathology at a single-cell resolution, our study unveils the intricate cellular dynamics underlying MS disease evolution.\nC-terminal amides mark proteins for degradation via SCF/FBXO31\nAuthors: Muhar, M.; Farnung, J.; Hofmann, R.; Cernakova, M.; Sidiropoulos, N. D.; Bode, J. W.; Corn, J. E.\nScore: 22.8, Published: 2023-06-30 DOI: 10.1101/2023.06.29.547030\nDuring normal cellular homeostasis unfolded and mis-localized proteins are recognized and removed, preventing the build-up of toxic byproducts. When protein homeostasis is perturbed during aging, neurodegeneration or cellular stress, proteins can accumulate several forms of chemical damage through reactive metabolites. Such modifications have been proposed to trigger the selective removal of chemically marked proteins; however, discovering modifications sufficient to induce protein degradation has remained challenging. Using a semi-synthetic chemical biology approach coupled to cellular assays, we found that C-terminal amide-bearing proteins (CTAPs) are rapidly cleared from human cells. A CRISPR screen identified the SCF/FBXO31 ubiquitin ligase as a reader of C-terminal amides, which ubiquitylates CTAPs for subsequent proteasomal degradation. A conserved binding pocket enables FBXO31 to bind almost any C-terminal peptide bearing an amide while retaining exquisite selectivity over non-modified clients. This mechanism facilitates binding and turnover of endogenous CTAPs that are formed following oxidative stress. A dominant human mutation found in neurodevelopmental disorders switches CTAP recognition, such that non-amidated neosubstrates are now degraded and FBXO31 becomes markedly toxic. We propose that CTAPs may represent the vanguard of a largely unexplored class of modified amino acid degrons that could provide a general strategy for selective yet broad surveillance of chemically damaged proteins.\nThe Tubulin Nano-Code: a protofilament-specific pattern of tubulin post-translational modifications regulates ciliary beating mechanics\nAuthors: Alvarez Viar, G.; Klena, N.; Martino, F.; Nievergelt, A. P.; Pigino, G.\nScore: 18.1, Published: 2023-06-28 DOI: 10.1101/2023.06.28.546853\nControl of ciliary beating is crucial for motility and signaling in eukaryotic cells and requires spatially restricted interactions between axonemal proteins and specific protofilaments within the ciliary microtubules. How these interactions are regulated remains poorly understood, but increasing evidence indicates that tubulin post-translational modifications (tPTMs) are required for proper ciliary motility. The Tubulin Code refers to the concept that tPTMs can modulate the function of individual microtubules in cells. Here we use a combination of immuno-cryo-electron tomography, expansion microscopy and mutant analysis to show that, in motile cilia, tubulin glycylation and polyglutamylation form mutually exclusive protofilament-specific nano-patterns at sub-microtubular scale. We show that these two nano-patterns are consistent with the distributions of axonemal dyneins and nexin-dynein regulatory complexes, respectively, and are required for their regulation during ciliary beating. Our discovery of a tubulin nano-code in cilia highlights the need of higher-resolution studies also in other cellular compartments to understand the molecular role of tPTMs.\nEndogenous tagging using split mNeonGreen in human iPSCs for live imaging studies\nAuthors: Husser, M. C.; Pham, N. P.; Law, C.; Araujo, F. R. B.; Martin, V. J. J.; Piekny, A.\nScore: 17.6, Published: 2023-06-28 DOI: 10.1101/2023.06.28.546942\nEndogenous tags have become invaluable tools to visualize and study native proteins in live cells. However, generating human cell lines carrying endogenous tags is difficult due to the low efficiency of homology-directed repair. Recently, an engineered split mNeonGreen protein was used to generate a large-scale endogenous tag library in HEK293 cells. Using split mNeonGreen for large-scale endogenous tagging in human iPSCs would open the door to studying protein function in healthy cells and across differentiated cell types. We engineered an iPS cell line to express the large fragment of the split mNeonGreen protein (mNG21-10) and showed that it enables fast and efficient endogenous tagging of proteins with the short fragment (mNG211). We also demonstrate that neural network-based image restoration enables live imaging studies of highly dynamic cellular processes such as cytokinesis in iPSCs. This work represents the first step towards a genome-wide endogenous tag library in human stem cells.\nThermosensitivity of translation underlies the mammalian nocturnal-diurnal switch\nAuthors: Beale, A. D.; Rzechorzek, N. M.; Mihut, A.; Zeng, A.; Smyllie, N. J.; Pilorz, V.; Richardson, R.; Bertlesen, M. F.; James, N. R.; Fazal, S. V.; Voysey, Z.; Pelletier, J.; Crosby, P.; Peak-Chew, S. Y.; Lancaster, M. A.; Hut, R. A.; O'Neill, J. S.\nScore: 18.0, Published: 2023-06-22 DOI: 10.1101/2023.06.22.546020\nEarly mammals were nocturnal until the Cretaceous-Paleogene extinction facilitated their rapid expansion into daytime niches. Diurnality subsequently evolved multiple times, independently, but the mechanisms facilitating this switch are unknown. We found that physiological daily temperature shifts oppositely affect circadian clock rhythms in nocturnal versus diurnal mammals. This occurs through a cell-intrinsic signal inverter, mediated by global differences in protein phosphorylation, and effected at the level of bulk protein synthesis rates, with diurnal translation rate being less thermosensitive than nocturnal. Perturbations that reduce translational initiation or mTOR activity are sufficient to trigger the nocturnal-to-diurnal switch at the cellular, tissue, and organismal scale. Our results suggest a convergent selection pressure to attain diurnality by reducing the effect of temperature-dependent changes in protein synthesis on circadian clocks. One sentence summaryRecalibrating the thermosensitivity of protein synthesis drives daytime-selective activity in mammals.\nSpecific prostaglandins are produced in the migratory cells and the surrounding substrate to promote Drosophila border cell migration\nAuthors: Mellentine, S. Q.; Ramsey, A. S.; Li, J.; Brown, H. N.; Tootle, T.\nScore: 7.5, Published: 2023-06-26 DOI: 10.1101/2023.06.23.546291\nA key regulator of collective cell migration is prostaglandin (PG) signaling. However, it remains largely unclear whether PGs act within the migratory cells or their microenvironment to promote migration. Here we use Drosophila border cell migration as a model to uncover the cell-specific roles of two PGs in collective migration. Prior work shows PG signaling is required for on-time migration and cluster cohesion. We find that the PGE2 synthase cPGES is required in the substrate, while the PGF2 synthase Akr1B is required in the border cells for on-time migration. Akr1B acts in both the border cells and their substrate to regulate cluster cohesion. One means by which Akr1B regulates border cell migration is by promoting integrin-based adhesions. Additionally, Akr1B limits myosin activity, and thereby cellular stiffness, in the border cells, whereas cPGES limits myosin activity in both the border cells and their substrate. Together these data reveal that two PGs, PGE2 and PGF2, produced in different locations, play key roles in promoting border cell migration. These PGs likely have similar migratory versus microenvironment roles in other collective cell migrations.\nA Dual Color Pax7 and Myf5 In Vivo Reporter to Investigate Muscle Stem Cell Heterogeneity in Regeneration and Aging\nAuthors: Ancel, S.; Michaud, J.; Sizzano, F.; Tauzin, L.; Oliveira, M.; Migliavacca, E.; Dammone, G.; Karaz, S.; Sanchez-Garcia, J. L.; Metairon, S.; Jacot, G.; Bentzinger, C. F.; Feige, J. N.; Stuelsatz, P.\nScore: 15.8, Published: 2023-06-29 DOI: 10.1101/2023.06.19.545587\nIncreasing evidence suggests heterogeneity in the muscle stem cell (MuSC) pool. In particular, a rare subset of Pax7 positive MuSCs that has never expressed the myogenic regulatory factor Myf5 has enhanced self-renewal and engraftment characteristics. However, the scarcity and limited availability of protein markers make the characterization of these cells challenging. We describe the generation of StemRep reporter mice allowing to monitor Pax7 and Myf5 protein based on equimolar levels of dual nuclear fluorescence. High levels of Pax7 protein and low levels of Myf5 delineate a deeply quiescent MuSC subpopulation with distinct molecular signatures and dynamics of activation, proliferation, and commitment. Aging decreases the number of these cells, and skews the MuSC pool towards Myf5-High cells with impaired quiescence. Altogether, we describe a novel deeply quiescent MuSC subpopulation whose maintenance is impaired in old muscles, and establish the StemRep line as a versatile tool to study quiescence and MuSC heterogeneity.\n",
  "wordCount" : "2052",
  "inLanguage": "en",
  "datePublished": "2023-07-02T10:39:32Z",
  "dateModified": "2023-07-02T10:39:32Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cell-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cell biology
    </h1>
    <div class="post-meta"><span>updated on July 2, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546744">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546744" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546744">
        <p class="paperTitle">Self-extinguishing relay waves enable homeostaticcontrol of human neutrophil swarming</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546744" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546744" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Strickland, J.; Pan, D.; Godfrey, C.; Kim, J. S.; Hopke, A.; Degrange, M.; Villavicencio, B.; Mansour, M. K.; Zerbe, C. S.; Irimia, D.; Amir, A.; Weiner, O. D.</p>
        <p class="info">Score: 313.3, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546744' target='https://doi.org/10.1101/2023.06.27.546744'> 10.1101/2023.06.27.546744</a></p>
        <p class="abstract">Neutrophils exhibit self-amplified swarming to sites of injury and infection. How swarming is controlled to ensure the proper level of neutrophil recruitment is unknown. Using an ex vivo model of infection, we find that human neutrophils use active relay to generate multiple pulsatile waves of swarming signals. Unlike classic active relay systems such as action potentials, neutrophil swarming relay waves are self-extinguishing, limiting the spatial range of cell recruitment. We identify an NADPH-oxidase-based negative feedback loop that is needed for this self-extinguishing behavior. Through this circuit, neutrophils adjust the number and size of swarming waves for homeostatic levels of cell recruitment over a wide range of initial cell densities. We link a broken homeostat to neutrophil over-recruitment in the context of human chronic granulomatous disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547259">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547259" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547259">
        <p class="paperTitle">TREX reveals proteins that bind to specific RNA regions in living cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547259" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547259" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dodel, M.; Guiducci, G.; Dermit, M.; Krishnamurthy, S.; Stojic, L.; Mardakheh, F. K.</p>
        <p class="info">Score: 36.9, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547259' target='https://doi.org/10.1101/2023.06.30.547259'> 10.1101/2023.06.30.547259</a></p>
        <p class="abstract">Different regions of RNA molecules can often engage in specific interactions with distinct RNA-binding proteins (RBPs), giving rise to diverse modalities of RNA regulation and function. However, there are currently no methods for unbiased identification of RBPs that interact with specific RNA regions in living cells under endogenous settings. Here, we introduce TREX (Targeted RNase H-mediated extraction of crosslinked RBPs), a highly sensitive approach for identifying proteins that directly bind to specific RNA regions in living cells. We demonstrate that TREX outperforms existing methods in identifying known interactors of U1 snRNA, and reveals endogenous region-specific interactors of NORAD lncRNA. Using TREX, we generated a comprehensive region-by-region interactome for 45S rRNA, uncovering both established and novel interactions that regulate ribosome biogenesis. With its applicability to any RNA in any cell-type, TREX is the first RNA-centric tool for unbiased positional mapping of endogenous RNA-protein interactions in living cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546565">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546565" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546565">
        <p class="paperTitle">Combinatorial selective ER-phagy remodels the ER during neurogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546565" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546565" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hoyer, M. J.; Smith, I. R.; Paoli, J. C.; Jiang, Y.; Paulo, J. A.; Harper, W.</p>
        <p class="info">Score: 28.8, Published: 2023-06-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546565' target='https://doi.org/10.1101/2023.06.26.546565'> 10.1101/2023.06.26.546565</a></p>
        <p class="abstract">The endoplasmic reticulum (ER) has a vast proteomic landscape to perform many diverse functions including protein and lipid synthesis, calcium ion flux, and inter-organelle communication. The ER proteome is remodeled in part through membrane-embedded receptors linking ER to degradative autophagy machinery (selective ER-phagy)1, 2. A refined tubular ER network3, 4 is formed in neurons within highly polarized dendrites and axons5, 6. Autophagy-deficient neurons in vivo display axonal ER accumulation within synaptic ER boutons,7 and the ER-phagy receptor FAM134B has been genetically linked with human sensory and autonomic neuropathy8, 9. However, mechanisms, including receptor selectivity, that define ER remodeling by autophagy in neurons are limited. Here, we combine a genetically tractable induced neuron (iNeuron) system for monitoring extensive ER remodeling during differentiation with proteomic and computational tools to create a quantitative landscape of ER proteome remodeling via selective autophagy. Through analysis of single and combinatorial ER-phagy receptor mutants, we delineate the extent to which each receptor contributes to both magnitude and selectivity of ER clearance via autophagy for individual ER protein cargos. We define specific subsets of ER curvature-shaping proteins or lumenal proteins as preferred clients for distinct receptors. Using spatial sensors and flux reporters, we demonstrate receptor-specific autophagic capture of ER in axons, which correlates with aberrant ER accumulation in axons of ER-phagy receptor or autophagy-deficient neurons. This molecular inventory of ER proteome remodeling and versatile genetic toolkit provides a quantitative framework for understanding contributions of individual ER-phagy receptors for reshaping ER during cell state transitions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547074">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547074" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547074">
        <p class="paperTitle">Single cell-resolution in situ sequencing elucidates spatial dynamics of multiple sclerosis lesion and disease evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547074" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547074" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Langseth, C. M.; Kukanja, P.; Rodriguez-Kirby, L. A. R.; Agirre, E.; Raman, A.; Yokota, C.; Avenel, C.; Tiklova, K.; Guerreiro-Cacais, A. O.; Olsson, T.; Hilscher, M. M.; Nilsson, M.; Castelo-Branco, G.</p>
        <p class="info">Score: 23.2, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547074' target='https://doi.org/10.1101/2023.06.29.547074'> 10.1101/2023.06.29.547074</a></p>
        <p class="abstract">Multiple sclerosis (MS) is a neurological disease characterised by multifocal lesions and smouldering pathology. While single-cell analyses have provided insights into neuropathology, cellular processes underlying MS remain poorly understood. We modelled the cellular dynamics of MS by examining temporal and regional rates of disease progression in the experimental autoimmune encephalomyelitis (EAE) mouse model. By performing single-cell spatial expression profiling using In situ sequencing, we annotated disease neighbourhoods during lesion evolution and found centrifugal propagation of active lesions. We demonstrated that disease-associated (DA) glia are dynamic and induced independently of lesions, preceding their formation. Single-cell spatial analysis of human archival MS spinal cord confirmed differential distribution of DA-glia, enabled deconvolution of active and inactive lesions into sub-compartments, and identification of new lesion areas. By establishing a spatial resource of mouse and human MS neuropathology at a single-cell resolution, our study unveils the intricate cellular dynamics underlying MS disease evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547030">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547030" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547030">
        <p class="paperTitle">C-terminal amides mark proteins for degradation via SCF/FBXO31</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547030" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547030" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Muhar, M.; Farnung, J.; Hofmann, R.; Cernakova, M.; Sidiropoulos, N. D.; Bode, J. W.; Corn, J. E.</p>
        <p class="info">Score: 22.8, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547030' target='https://doi.org/10.1101/2023.06.29.547030'> 10.1101/2023.06.29.547030</a></p>
        <p class="abstract">During normal cellular homeostasis unfolded and mis-localized proteins are recognized and removed, preventing the build-up of toxic byproducts. When protein homeostasis is perturbed during aging, neurodegeneration or cellular stress, proteins can accumulate several forms of chemical damage through reactive metabolites. Such modifications have been proposed to trigger the selective removal of chemically marked proteins; however, discovering modifications sufficient to induce protein degradation has remained challenging. Using a semi-synthetic chemical biology approach coupled to cellular assays, we found that C-terminal amide-bearing proteins (CTAPs) are rapidly cleared from human cells. A CRISPR screen identified the SCF/FBXO31 ubiquitin ligase as a reader of C-terminal amides, which ubiquitylates CTAPs for subsequent proteasomal degradation. A conserved binding pocket enables FBXO31 to bind almost any C-terminal peptide bearing an amide while retaining exquisite selectivity over non-modified clients. This mechanism facilitates binding and turnover of endogenous CTAPs that are formed following oxidative stress. A dominant human mutation found in neurodevelopmental disorders switches CTAP recognition, such that non-amidated neosubstrates are now degraded and FBXO31 becomes markedly toxic. We propose that CTAPs may represent the vanguard of a largely unexplored class of modified amino acid degrons that could provide a general strategy for selective yet broad surveillance of chemically damaged proteins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546853">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546853" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546853">
        <p class="paperTitle">The Tubulin Nano-Code: a protofilament-specific pattern of tubulin post-translational modifications regulates ciliary beating mechanics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546853" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546853" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alvarez Viar, G.; Klena, N.; Martino, F.; Nievergelt, A. P.; Pigino, G.</p>
        <p class="info">Score: 18.1, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546853' target='https://doi.org/10.1101/2023.06.28.546853'> 10.1101/2023.06.28.546853</a></p>
        <p class="abstract">Control of ciliary beating is crucial for motility and signaling in eukaryotic cells and requires spatially restricted interactions between axonemal proteins and specific protofilaments within the ciliary microtubules. How these interactions are regulated remains poorly understood, but increasing evidence indicates that tubulin post-translational modifications (tPTMs) are required for proper ciliary motility. The Tubulin Code refers to the concept that tPTMs can modulate the function of individual microtubules in cells. Here we use a combination of immuno-cryo-electron tomography, expansion microscopy and mutant analysis to show that, in motile cilia, tubulin glycylation and polyglutamylation form mutually exclusive protofilament-specific nano-patterns at sub-microtubular scale. We show that these two nano-patterns are consistent with the distributions of axonemal dyneins and nexin-dynein regulatory complexes, respectively, and are required for their regulation during ciliary beating. Our discovery of a tubulin nano-code in cilia highlights the need of higher-resolution studies also in other cellular compartments to understand the molecular role of tPTMs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546942">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546942" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546942">
        <p class="paperTitle">Endogenous tagging using split mNeonGreen in human iPSCs for live imaging studies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546942" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546942" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Husser, M. C.; Pham, N. P.; Law, C.; Araujo, F. R. B.; Martin, V. J. J.; Piekny, A.</p>
        <p class="info">Score: 17.6, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546942' target='https://doi.org/10.1101/2023.06.28.546942'> 10.1101/2023.06.28.546942</a></p>
        <p class="abstract">Endogenous tags have become invaluable tools to visualize and study native proteins in live cells. However, generating human cell lines carrying endogenous tags is difficult due to the low efficiency of homology-directed repair. Recently, an engineered split mNeonGreen protein was used to generate a large-scale endogenous tag library in HEK293 cells. Using split mNeonGreen for large-scale endogenous tagging in human iPSCs would open the door to studying protein function in healthy cells and across differentiated cell types. We engineered an iPS cell line to express the large fragment of the split mNeonGreen protein (mNG21-10) and showed that it enables fast and efficient endogenous tagging of proteins with the short fragment (mNG211). We also demonstrate that neural network-based image restoration enables live imaging studies of highly dynamic cellular processes such as cytokinesis in iPSCs. This work represents the first step towards a genome-wide endogenous tag library in human stem cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.22.546020">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.22.546020" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.22.546020">
        <p class="paperTitle">Thermosensitivity of translation underlies the mammalian nocturnal-diurnal switch</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.22.546020" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.22.546020" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Beale, A. D.; Rzechorzek, N. M.; Mihut, A.; Zeng, A.; Smyllie, N. J.; Pilorz, V.; Richardson, R.; Bertlesen, M. F.; James, N. R.; Fazal, S. V.; Voysey, Z.; Pelletier, J.; Crosby, P.; Peak-Chew, S. Y.; Lancaster, M. A.; Hut, R. A.; O&#39;Neill, J. S.</p>
        <p class="info">Score: 18.0, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.22.546020' target='https://doi.org/10.1101/2023.06.22.546020'> 10.1101/2023.06.22.546020</a></p>
        <p class="abstract">Early mammals were nocturnal until the Cretaceous-Paleogene extinction facilitated their rapid expansion into daytime niches. Diurnality subsequently evolved multiple times, independently, but the mechanisms facilitating this switch are unknown. We found that physiological daily temperature shifts oppositely affect circadian clock rhythms in nocturnal versus diurnal mammals. This occurs through a cell-intrinsic signal inverter, mediated by global differences in protein phosphorylation, and effected at the level of bulk protein synthesis rates, with diurnal translation rate being less thermosensitive than nocturnal. Perturbations that reduce translational initiation or mTOR activity are sufficient to trigger the nocturnal-to-diurnal switch at the cellular, tissue, and organismal scale. Our results suggest a convergent selection pressure to attain diurnality by reducing the effect of temperature-dependent changes in protein synthesis on circadian clocks.

One sentence summaryRecalibrating the thermosensitivity of protein synthesis drives daytime-selective activity in mammals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546291">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546291" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546291">
        <p class="paperTitle">Specific prostaglandins are produced in the migratory cells and the surrounding substrate to promote Drosophila border cell migration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546291" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546291" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mellentine, S. Q.; Ramsey, A. S.; Li, J.; Brown, H. N.; Tootle, T.</p>
        <p class="info">Score: 7.5, Published: 2023-06-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546291' target='https://doi.org/10.1101/2023.06.23.546291'> 10.1101/2023.06.23.546291</a></p>
        <p class="abstract">A key regulator of collective cell migration is prostaglandin (PG) signaling. However, it remains largely unclear whether PGs act within the migratory cells or their microenvironment to promote migration. Here we use Drosophila border cell migration as a model to uncover the cell-specific roles of two PGs in collective migration. Prior work shows PG signaling is required for on-time migration and cluster cohesion. We find that the PGE2 synthase cPGES is required in the substrate, while the PGF2 synthase Akr1B is required in the border cells for on-time migration. Akr1B acts in both the border cells and their substrate to regulate cluster cohesion. One means by which Akr1B regulates border cell migration is by promoting integrin-based adhesions. Additionally, Akr1B limits myosin activity, and thereby cellular stiffness, in the border cells, whereas cPGES limits myosin activity in both the border cells and their substrate. Together these data reveal that two PGs, PGE2 and PGF2, produced in different locations, play key roles in promoting border cell migration. These PGs likely have similar migratory versus microenvironment roles in other collective cell migrations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.545587">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.545587" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.545587">
        <p class="paperTitle">A Dual Color Pax7 and Myf5 In Vivo Reporter to Investigate Muscle Stem Cell Heterogeneity in Regeneration and Aging</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.545587" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.545587" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ancel, S.; Michaud, J.; Sizzano, F.; Tauzin, L.; Oliveira, M.; Migliavacca, E.; Dammone, G.; Karaz, S.; Sanchez-Garcia, J. L.; Metairon, S.; Jacot, G.; Bentzinger, C. F.; Feige, J. N.; Stuelsatz, P.</p>
        <p class="info">Score: 15.8, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.545587' target='https://doi.org/10.1101/2023.06.19.545587'> 10.1101/2023.06.19.545587</a></p>
        <p class="abstract">Increasing evidence suggests heterogeneity in the muscle stem cell (MuSC) pool. In particular, a rare subset of Pax7 positive MuSCs that has never expressed the myogenic regulatory factor Myf5 has enhanced self-renewal and engraftment characteristics. However, the scarcity and limited availability of protein markers make the characterization of these cells challenging. We describe the generation of StemRep reporter mice allowing to monitor Pax7 and Myf5 protein based on equimolar levels of dual nuclear fluorescence. High levels of Pax7 protein and low levels of Myf5 delineate a deeply quiescent MuSC subpopulation with distinct molecular signatures and dynamics of activation, proliferation, and commitment. Aging decreases the number of these cells, and skews the MuSC pool towards Myf5-High cells with impaired quiescence. Altogether, we describe a novel deeply quiescent MuSC subpopulation whose maintenance is impaired in old muscles, and establish the StemRep line as a versatile tool to study quiescence and MuSC heterogeneity.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
