<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cell biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cell biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Self-extinguishing relay waves enable homeostaticcontrol of human neutrophil swarming
Authors: Strickland, J.; Pan, D.; Godfrey, C.; Kim, J. S.; Hopke, A.; Degrange, M.; Villavicencio, B.; Mansour, M. K.; Zerbe, C. S.; Irimia, D.; Amir, A.; Weiner, O. D.
Score: 542.7, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546744
Neutrophils exhibit self-amplified swarming to sites of injury and infection. How swarming is controlled to ensure the proper level of neutrophil recruitment is unknown. Using an ex vivo model of infection, we find that human neutrophils use active relay to generate multiple pulsatile waves of swarming signals.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cell-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cell biology" />
<meta property="og:description" content="Self-extinguishing relay waves enable homeostaticcontrol of human neutrophil swarming
Authors: Strickland, J.; Pan, D.; Godfrey, C.; Kim, J. S.; Hopke, A.; Degrange, M.; Villavicencio, B.; Mansour, M. K.; Zerbe, C. S.; Irimia, D.; Amir, A.; Weiner, O. D.
Score: 542.7, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546744
Neutrophils exhibit self-amplified swarming to sites of injury and infection. How swarming is controlled to ensure the proper level of neutrophil recruitment is unknown. Using an ex vivo model of infection, we find that human neutrophils use active relay to generate multiple pulsatile waves of swarming signals." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cell-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-05T10:42:43+00:00" />
<meta property="article:modified_time" content="2023-07-05T10:42:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cell biology"/>
<meta name="twitter:description" content="Self-extinguishing relay waves enable homeostaticcontrol of human neutrophil swarming
Authors: Strickland, J.; Pan, D.; Godfrey, C.; Kim, J. S.; Hopke, A.; Degrange, M.; Villavicencio, B.; Mansour, M. K.; Zerbe, C. S.; Irimia, D.; Amir, A.; Weiner, O. D.
Score: 542.7, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546744
Neutrophils exhibit self-amplified swarming to sites of injury and infection. How swarming is controlled to ensure the proper level of neutrophil recruitment is unknown. Using an ex vivo model of infection, we find that human neutrophils use active relay to generate multiple pulsatile waves of swarming signals."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cell biology",
      "item": "https://trxiv.yorks0n.com/posts/cell-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cell biology",
  "name": "cell biology",
  "description": "Self-extinguishing relay waves enable homeostaticcontrol of human neutrophil swarming\nAuthors: Strickland, J.; Pan, D.; Godfrey, C.; Kim, J. S.; Hopke, A.; Degrange, M.; Villavicencio, B.; Mansour, M. K.; Zerbe, C. S.; Irimia, D.; Amir, A.; Weiner, O. D.\nScore: 542.7, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546744\nNeutrophils exhibit self-amplified swarming to sites of injury and infection. How swarming is controlled to ensure the proper level of neutrophil recruitment is unknown. Using an ex vivo model of infection, we find that human neutrophils use active relay to generate multiple pulsatile waves of swarming signals.",
  "keywords": [
    
  ],
  "articleBody": " Self-extinguishing relay waves enable homeostaticcontrol of human neutrophil swarming\nAuthors: Strickland, J.; Pan, D.; Godfrey, C.; Kim, J. S.; Hopke, A.; Degrange, M.; Villavicencio, B.; Mansour, M. K.; Zerbe, C. S.; Irimia, D.; Amir, A.; Weiner, O. D.\nScore: 542.7, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546744\nNeutrophils exhibit self-amplified swarming to sites of injury and infection. How swarming is controlled to ensure the proper level of neutrophil recruitment is unknown. Using an ex vivo model of infection, we find that human neutrophils use active relay to generate multiple pulsatile waves of swarming signals. Unlike classic active relay systems such as action potentials, neutrophil swarming relay waves are self-extinguishing, limiting the spatial range of cell recruitment. We identify an NADPH-oxidase-based negative feedback loop that is needed for this self-extinguishing behavior. Through this circuit, neutrophils adjust the number and size of swarming waves for homeostatic levels of cell recruitment over a wide range of initial cell densities. We link a broken homeostat to neutrophil over-recruitment in the context of human chronic granulomatous disease.\nTREX reveals proteins that bind to specific RNA regions in living cells\nAuthors: Dodel, M.; Guiducci, G.; Dermit, M.; Krishnamurthy, S.; Stojic, L.; Mardakheh, F. K.\nScore: 59.7, Published: 2023-06-30 DOI: 10.1101/2023.06.30.547259\nDifferent regions of RNA molecules can often engage in specific interactions with distinct RNA-binding proteins (RBPs), giving rise to diverse modalities of RNA regulation and function. However, there are currently no methods for unbiased identification of RBPs that interact with specific RNA regions in living cells under endogenous settings. Here, we introduce TREX (Targeted RNase H-mediated extraction of crosslinked RBPs), a highly sensitive approach for identifying proteins that directly bind to specific RNA regions in living cells. We demonstrate that TREX outperforms existing methods in identifying known interactors of U1 snRNA, and reveals endogenous region-specific interactors of NORAD lncRNA. Using TREX, we generated a comprehensive region-by-region interactome for 45S rRNA, uncovering both established and novel interactions that regulate ribosome biogenesis. With its applicability to any RNA in any cell-type, TREX is the first RNA-centric tool for unbiased positional mapping of endogenous RNA-protein interactions in living cells.\nDynein and dynactin move long-range but are delivered separately to the axon tip\nAuthors: Fellows, A. D.; Bruntraeger, M.; Burgold, T.; Bassett, A. R.; Carter, A. P.\nScore: 43.6, Published: 2023-07-03 DOI: 10.1101/2023.07.03.547521\nThe microtubule motor dynein drives retrograde transport from the axon tip back to the cell body and is essential for neuronal survival. Its function requires its cofactor dynactin and regulators LIS1 and NDEL1. However, it is unclear if all dynein components can travel along the axon, how far they move and whether they do so together. Here, we use neuron-inducible (NGN2-OPTi-OX) human-stem-cell lines to endogenously tag dynein components and track them under a near single molecule regime. In the retrograde direction dynein and dynactin can move the entire \u003e 500 um length of the axon in one go. Furthermore, LIS1 and NDEL1 also undergo long-distance movement, despite being mainly implicated with initiation of transport. Intriguingly, in the anterograde direction dynein/LIS1 move faster than dynactin/NDEL1 implying they can travel on different cargos. Therefore, neurons ensure efficient transport by keeping dynein/dynactin on cargos over long distances, but keeping them separate until their function is required.\nSingle cell-resolution in situ sequencing elucidates spatial dynamics of multiple sclerosis lesion and disease evolution\nAuthors: Langseth, C. M.; Kukanja, P.; Rodriguez-Kirby, L. A. R.; Agirre, E.; Raman, A.; Yokota, C.; Avenel, C.; Tiklova, K.; Guerreiro-Cacais, A. O.; Olsson, T.; Hilscher, M. M.; Nilsson, M.; Castelo-Branco, G.\nScore: 25.6, Published: 2023-06-30 DOI: 10.1101/2023.06.29.547074\nMultiple sclerosis (MS) is a neurological disease characterised by multifocal lesions and smouldering pathology. While single-cell analyses have provided insights into neuropathology, cellular processes underlying MS remain poorly understood. We modelled the cellular dynamics of MS by examining temporal and regional rates of disease progression in the experimental autoimmune encephalomyelitis (EAE) mouse model. By performing single-cell spatial expression profiling using In situ sequencing, we annotated disease neighbourhoods during lesion evolution and found centrifugal propagation of active lesions. We demonstrated that disease-associated (DA) glia are dynamic and induced independently of lesions, preceding their formation. Single-cell spatial analysis of human archival MS spinal cord confirmed differential distribution of DA-glia, enabled deconvolution of active and inactive lesions into sub-compartments, and identification of new lesion areas. By establishing a spatial resource of mouse and human MS neuropathology at a single-cell resolution, our study unveils the intricate cellular dynamics underlying MS disease evolution.\nThe Tubulin Nano-Code: a protofilament-specific pattern of tubulin post-translational modifications regulates ciliary beating mechanics\nAuthors: Alvarez Viar, G.; Klena, N.; Martino, F.; Nievergelt, A. P.; Pigino, G.\nScore: 24.2, Published: 2023-06-28 DOI: 10.1101/2023.06.28.546853\nControl of ciliary beating is crucial for motility and signaling in eukaryotic cells and requires spatially restricted interactions between axonemal proteins and specific protofilaments within the ciliary microtubules. How these interactions are regulated remains poorly understood, but increasing evidence indicates that tubulin post-translational modifications (tPTMs) are required for proper ciliary motility. The Tubulin Code refers to the concept that tPTMs can modulate the function of individual microtubules in cells. Here we use a combination of immuno-cryo-electron tomography, expansion microscopy and mutant analysis to show that, in motile cilia, tubulin glycylation and polyglutamylation form mutually exclusive protofilament-specific nano-patterns at sub-microtubular scale. We show that these two nano-patterns are consistent with the distributions of axonemal dyneins and nexin-dynein regulatory complexes, respectively, and are required for their regulation during ciliary beating. Our discovery of a tubulin nano-code in cilia highlights the need of higher-resolution studies also in other cellular compartments to understand the molecular role of tPTMs. One-Sentence SummaryTubulin post-translational modifications form nanopatterns at sub-microtubular scale that enable individual protofilaments to perform specific functions Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=126 SRC=\"FIGDIR/small/546853v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (62K): org.highwire.dtl.DTLVardef@fb5e4corg.highwire.dtl.DTLVardef@ec0987org.highwire.dtl.DTLVardef@65819dorg.highwire.dtl.DTLVardef@20a517_HPS_FORMAT_FIGEXP M_FIG C_FIG\nC-terminal amides mark proteins for degradation via SCF/FBXO31\nAuthors: Muhar, M.; Farnung, J.; Hofmann, R.; Cernakova, M.; Sidiropoulos, N. D.; Bode, J. W.; Corn, J. E.\nScore: 22.8, Published: 2023-06-30 DOI: 10.1101/2023.06.29.547030\nDuring normal cellular homeostasis unfolded and mis-localized proteins are recognized and removed, preventing the build-up of toxic byproducts1. When protein homeostasis is perturbed during aging, neurodegeneration or cellular stress, proteins can accumulate several forms of chemical damage through reactive metabolites2, 3. Such modifications have been proposed to trigger the selective removal of chemically marked proteins3-6; however, discovering modifications sufficient to induce protein degradation has remained challenging. Using a semi-synthetic chemical biology approach coupled to cellular assays, we found that C-terminal amide-bearing proteins (CTAPs) are rapidly cleared from human cells. A CRISPR screen identified the SCF/FBXO31 ubiquitin ligase as a reader of C-terminal amides, which ubiquitylates CTAPs for subsequent proteasomal degradation. A conserved binding pocket enables FBXO31 to bind almost any C-terminal peptide bearing an amide while retaining exquisite selectivity over non-modified clients. This mechanism facilitates binding and turnover of endogenous CTAPs that are formed following oxidative stress. A dominant human mutation found in neurodevelopmental disorders switches CTAP recognition, such that non-amidated neosubstrates are now degraded and FBXO31 becomes markedly toxic. We propose that CTAPs may represent the vanguard of a largely unexplored class of modified amino acid degrons that could provide a general strategy for selective yet broad surveillance of chemically damaged proteins.\nEndogenous tagging using split mNeonGreen in human iPSCs for live imaging studies\nAuthors: Husser, M. C.; Pham, N. P.; Law, C.; Araujo, F. R. B.; Martin, V. J. J.; Piekny, A.\nScore: 19.5, Published: 2023-06-28 DOI: 10.1101/2023.06.28.546942\nEndogenous tags have become invaluable tools to visualize and study native proteins in live cells. However, generating human cell lines carrying endogenous tags is difficult due to the low efficiency of homology-directed repair. Recently, an engineered split mNeonGreen protein was used to generate a large-scale endogenous tag library in HEK293 cells. Using split mNeonGreen for large-scale endogenous tagging in human iPSCs would open the door to studying protein function in healthy cells and across differentiated cell types. We engineered an iPS cell line to express the large fragment of the split mNeonGreen protein (mNG21-10) and showed that it enables fast and efficient endogenous tagging of proteins with the short fragment (mNG211). We also demonstrate that neural network-based image restoration enables live imaging studies of highly dynamic cellular processes such as cytokinesis in iPSCs. This work represents the first step towards a genome-wide endogenous tag library in human stem cells.\nRaman micro-spectroscopy reveals the spatial distribution of fumarate in cells and tissues.\nAuthors: Kamp, M.; Surmacki, J.; Segarra Mondejar, M.; Young, T.; Chrabaszcz, K.; Joud, F.; Zecchini, V.; Speed, A.; Frezza, C.; Bohndiek, S.\nScore: 15.6, Published: 2023-07-02 DOI: 10.1101/2023.07.02.547209\nAberrantly accumulated metabolites such as fumarate elicit intra- and inter-cellular prooncogenic cascades, yet current methods to measure them require sample perturbation or disruption and lack spatio-temporal resolution, limiting our ability to fully characterize their function and distribution in cells and within a tissue. Raman spectroscopy (RS) is a powerful bio-analytical tool that directly characterizes the chemical composition of a sample based solely on the optical fingerprint of vibrational modes. Here, we show for the first time that RS can directly detect fumarate in living cells in vivo and animal tissues ex vivo. Using the observed linear relationship between Raman scattered intensity and fumarate concentration, we demonstrate that RS can distinguish between Fumarate hydratase (Fh1)-deficient and Fh1-proficient cells based on their fumarate concentration. Moreover, RS reveals the spatial compartmentalization of fumarate within cellular organelles: consistent with disruptive methods, in Fh1-deficient cells we observe the highest fumarate concentration (37 +/- 19 mM) in the mitochondria, where the TCA cycle operates, followed by the cytoplasm (24 +/- 13 mM) and then the nucleus (9 +/- 6 mM). Finally, we apply RS to tissues from an inducible mouse model of FH loss in the kidney, demonstrating that RS can accurately classify FH status in these tissues. These results suggest that RS could be adopted as a valuable tool for small molecule metabolic imaging, enabling in situ dynamic evaluation of fumarate compartmentalization.\nMatrix stiffening promotes perinuclear clustering of mitochondria\nAuthors: Daga, P.; Thurakkal, B.; Rawal, S.; Das, T.\nScore: 7.1, Published: 2023-06-30 DOI: 10.1101/2023.06.29.547150\nMechanical cues from the tissue microenvironment, such as the stiffness of the extracellular matrix, modulate cellular forms and functions. As numerous studies have shown, this modulation depends on the stiffness-dependent remodeling of cytoskeletal elements. In contrast, very little is known about how the intracellular organelles such as mitochondria respond to matrix stiffness and whether their form, function, and localization change accordingly. Here, we performed an extensive quantitative characterization of mitochondrial morphology, subcellular localization, dynamics and membrane tension on soft and stiff matrices. This characterization revealed that while matrix stiffness affected all these aspects, matrix stiffening most distinctively led to an increased perinuclear clustering of mitochondria. Subsequently, we could identify the matrix stiffness-sensitive perinuclear localization of filamin as the key factor dictating this perinuclear clustering. Photo-conversion labeling and fluorescent recovery after photobleaching experiments revealed that perinuclear and peripheral mitochondrial populations differed in their motility on the soft matrix but surprisingly they did not show any difference on the stiff matrix. Finally, perinuclear mitochondrial clustering appeared to be crucial for priming human mesenchymal stem cells towards osteogenesis on the stiff matrix. Taken together, we elucidate a dependence of mitochondrial localization on matrix stiffness, which possibly enables a cell to adapt to its microenvironment.\nCombinatorial selective ER-phagy remodels the ER during neurogenesis\nAuthors: Hoyer, M. J.; Smith, I. R.; Paoli, J. C.; Jiang, Y.; Paulo, J. A.; Harper, W.\nScore: 28.8, Published: 2023-06-26 DOI: 10.1101/2023.06.26.546565\nThe endoplasmic reticulum (ER) has a vast proteomic landscape to perform many diverse functions including protein and lipid synthesis, calcium ion flux, and inter-organelle communication. The ER proteome is remodeled in part through membrane-embedded receptors linking ER to degradative autophagy machinery (selective ER-phagy)1, 2. A refined tubular ER network3, 4 is formed in neurons within highly polarized dendrites and axons5, 6. Autophagy-deficient neurons in vivo display axonal ER accumulation within synaptic ER boutons,7 and the ER-phagy receptor FAM134B has been genetically linked with human sensory and autonomic neuropathy8, 9. However, mechanisms, including receptor selectivity, that define ER remodeling by autophagy in neurons are limited. Here, we combine a genetically tractable induced neuron (iNeuron) system for monitoring extensive ER remodeling during differentiation with proteomic and computational tools to create a quantitative landscape of ER proteome remodeling via selective autophagy. Through analysis of single and combinatorial ER-phagy receptor mutants, we delineate the extent to which each receptor contributes to both magnitude and selectivity of ER clearance via autophagy for individual ER protein cargos. We define specific subsets of ER curvature-shaping proteins or lumenal proteins as preferred clients for distinct receptors. Using spatial sensors and flux reporters, we demonstrate receptor-specific autophagic capture of ER in axons, which correlates with aberrant ER accumulation in axons of ER-phagy receptor or autophagy-deficient neurons. This molecular inventory of ER proteome remodeling and versatile genetic toolkit provides a quantitative framework for understanding contributions of individual ER-phagy receptors for reshaping ER during cell state transitions.\n",
  "wordCount" : "2142",
  "inLanguage": "en",
  "datePublished": "2023-07-05T10:42:43Z",
  "dateModified": "2023-07-05T10:42:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cell-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cell biology
    </h1>
    <div class="post-meta"><span>updated on July 5, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546744">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546744" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546744">
        <p class="paperTitle">Self-extinguishing relay waves enable homeostaticcontrol of human neutrophil swarming</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546744" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546744" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Strickland, J.; Pan, D.; Godfrey, C.; Kim, J. S.; Hopke, A.; Degrange, M.; Villavicencio, B.; Mansour, M. K.; Zerbe, C. S.; Irimia, D.; Amir, A.; Weiner, O. D.</p>
        <p class="info">Score: 542.7, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546744' target='https://doi.org/10.1101/2023.06.27.546744'> 10.1101/2023.06.27.546744</a></p>
        <p class="abstract">Neutrophils exhibit self-amplified swarming to sites of injury and infection. How swarming is controlled to ensure the proper level of neutrophil recruitment is unknown. Using an ex vivo model of infection, we find that human neutrophils use active relay to generate multiple pulsatile waves of swarming signals. Unlike classic active relay systems such as action potentials, neutrophil swarming relay waves are self-extinguishing, limiting the spatial range of cell recruitment. We identify an NADPH-oxidase-based negative feedback loop that is needed for this self-extinguishing behavior. Through this circuit, neutrophils adjust the number and size of swarming waves for homeostatic levels of cell recruitment over a wide range of initial cell densities. We link a broken homeostat to neutrophil over-recruitment in the context of human chronic granulomatous disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547259">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547259" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547259">
        <p class="paperTitle">TREX reveals proteins that bind to specific RNA regions in living cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547259" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547259" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dodel, M.; Guiducci, G.; Dermit, M.; Krishnamurthy, S.; Stojic, L.; Mardakheh, F. K.</p>
        <p class="info">Score: 59.7, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547259' target='https://doi.org/10.1101/2023.06.30.547259'> 10.1101/2023.06.30.547259</a></p>
        <p class="abstract">Different regions of RNA molecules can often engage in specific interactions with distinct RNA-binding proteins (RBPs), giving rise to diverse modalities of RNA regulation and function. However, there are currently no methods for unbiased identification of RBPs that interact with specific RNA regions in living cells under endogenous settings. Here, we introduce TREX (Targeted RNase H-mediated extraction of crosslinked RBPs), a highly sensitive approach for identifying proteins that directly bind to specific RNA regions in living cells. We demonstrate that TREX outperforms existing methods in identifying known interactors of U1 snRNA, and reveals endogenous region-specific interactors of NORAD lncRNA. Using TREX, we generated a comprehensive region-by-region interactome for 45S rRNA, uncovering both established and novel interactions that regulate ribosome biogenesis. With its applicability to any RNA in any cell-type, TREX is the first RNA-centric tool for unbiased positional mapping of endogenous RNA-protein interactions in living cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547521">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547521" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547521">
        <p class="paperTitle">Dynein and dynactin move long-range but are delivered separately to the axon tip</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547521" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547521" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fellows, A. D.; Bruntraeger, M.; Burgold, T.; Bassett, A. R.; Carter, A. P.</p>
        <p class="info">Score: 43.6, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547521' target='https://doi.org/10.1101/2023.07.03.547521'> 10.1101/2023.07.03.547521</a></p>
        <p class="abstract">The microtubule motor dynein drives retrograde transport from the axon tip back to the cell body and is essential for neuronal survival. Its function requires its cofactor dynactin and regulators LIS1 and NDEL1. However, it is unclear if all dynein components can travel along the axon, how far they move and whether they do so together. Here, we use neuron-inducible (NGN2-OPTi-OX) human-stem-cell lines to endogenously tag dynein components and track them under a near single molecule regime. In the retrograde direction dynein and dynactin can move the entire &gt; 500 um length of the axon in one go. Furthermore, LIS1 and NDEL1 also undergo long-distance movement, despite being mainly implicated with initiation of transport. Intriguingly, in the anterograde direction dynein/LIS1 move faster than dynactin/NDEL1 implying they can travel on different cargos. Therefore, neurons ensure efficient transport by keeping dynein/dynactin on cargos over long distances, but keeping them separate until their function is required.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547074">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547074" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547074">
        <p class="paperTitle">Single cell-resolution in situ sequencing elucidates spatial dynamics of multiple sclerosis lesion and disease evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547074" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547074" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Langseth, C. M.; Kukanja, P.; Rodriguez-Kirby, L. A. R.; Agirre, E.; Raman, A.; Yokota, C.; Avenel, C.; Tiklova, K.; Guerreiro-Cacais, A. O.; Olsson, T.; Hilscher, M. M.; Nilsson, M.; Castelo-Branco, G.</p>
        <p class="info">Score: 25.6, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547074' target='https://doi.org/10.1101/2023.06.29.547074'> 10.1101/2023.06.29.547074</a></p>
        <p class="abstract">Multiple sclerosis (MS) is a neurological disease characterised by multifocal lesions and smouldering pathology. While single-cell analyses have provided insights into neuropathology, cellular processes underlying MS remain poorly understood. We modelled the cellular dynamics of MS by examining temporal and regional rates of disease progression in the experimental autoimmune encephalomyelitis (EAE) mouse model. By performing single-cell spatial expression profiling using In situ sequencing, we annotated disease neighbourhoods during lesion evolution and found centrifugal propagation of active lesions. We demonstrated that disease-associated (DA) glia are dynamic and induced independently of lesions, preceding their formation. Single-cell spatial analysis of human archival MS spinal cord confirmed differential distribution of DA-glia, enabled deconvolution of active and inactive lesions into sub-compartments, and identification of new lesion areas. By establishing a spatial resource of mouse and human MS neuropathology at a single-cell resolution, our study unveils the intricate cellular dynamics underlying MS disease evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546853">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546853" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546853">
        <p class="paperTitle">The Tubulin Nano-Code: a protofilament-specific pattern of tubulin post-translational modifications regulates ciliary beating mechanics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546853" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546853" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alvarez Viar, G.; Klena, N.; Martino, F.; Nievergelt, A. P.; Pigino, G.</p>
        <p class="info">Score: 24.2, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546853' target='https://doi.org/10.1101/2023.06.28.546853'> 10.1101/2023.06.28.546853</a></p>
        <p class="abstract">Control of ciliary beating is crucial for motility and signaling in eukaryotic cells and requires spatially restricted interactions between axonemal proteins and specific protofilaments within the ciliary microtubules. How these interactions are regulated remains poorly understood, but increasing evidence indicates that tubulin post-translational modifications (tPTMs) are required for proper ciliary motility. The Tubulin Code refers to the concept that tPTMs can modulate the function of individual microtubules in cells. Here we use a combination of immuno-cryo-electron tomography, expansion microscopy and mutant analysis to show that, in motile cilia, tubulin glycylation and polyglutamylation form mutually exclusive protofilament-specific nano-patterns at sub-microtubular scale. We show that these two nano-patterns are consistent with the distributions of axonemal dyneins and nexin-dynein regulatory complexes, respectively, and are required for their regulation during ciliary beating. Our discovery of a tubulin nano-code in cilia highlights the need of higher-resolution studies also in other cellular compartments to understand the molecular role of tPTMs.

One-Sentence SummaryTubulin post-translational modifications form nanopatterns at sub-microtubular scale that enable individual protofilaments to perform specific functions

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=126 SRC=&#34;FIGDIR/small/546853v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (62K):
org.highwire.dtl.DTLVardef@fb5e4corg.highwire.dtl.DTLVardef@ec0987org.highwire.dtl.DTLVardef@65819dorg.highwire.dtl.DTLVardef@20a517_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547030">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547030" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547030">
        <p class="paperTitle">C-terminal amides mark proteins for degradation via SCF/FBXO31</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547030" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547030" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Muhar, M.; Farnung, J.; Hofmann, R.; Cernakova, M.; Sidiropoulos, N. D.; Bode, J. W.; Corn, J. E.</p>
        <p class="info">Score: 22.8, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547030' target='https://doi.org/10.1101/2023.06.29.547030'> 10.1101/2023.06.29.547030</a></p>
        <p class="abstract">During normal cellular homeostasis unfolded and mis-localized proteins are recognized and removed, preventing the build-up of toxic byproducts1. When protein homeostasis is perturbed during aging, neurodegeneration or cellular stress, proteins can accumulate several forms of chemical damage through reactive metabolites2, 3. Such modifications have been proposed to trigger the selective removal of chemically marked proteins3-6; however, discovering modifications sufficient to induce protein degradation has remained challenging. Using a semi-synthetic chemical biology approach coupled to cellular assays, we found that C-terminal amide-bearing proteins (CTAPs) are rapidly cleared from human cells. A CRISPR screen identified the SCF/FBXO31 ubiquitin ligase as a reader of C-terminal amides, which ubiquitylates CTAPs for subsequent proteasomal degradation. A conserved binding pocket enables FBXO31 to bind almost any C-terminal peptide bearing an amide while retaining exquisite selectivity over non-modified clients. This mechanism facilitates binding and turnover of endogenous CTAPs that are formed following oxidative stress. A dominant human mutation found in neurodevelopmental disorders switches CTAP recognition, such that non-amidated neosubstrates are now degraded and FBXO31 becomes markedly toxic. We propose that CTAPs may represent the vanguard of a largely unexplored class of modified amino acid degrons that could provide a general strategy for selective yet broad surveillance of chemically damaged proteins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546942">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546942" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546942">
        <p class="paperTitle">Endogenous tagging using split mNeonGreen in human iPSCs for live imaging studies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546942" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546942" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Husser, M. C.; Pham, N. P.; Law, C.; Araujo, F. R. B.; Martin, V. J. J.; Piekny, A.</p>
        <p class="info">Score: 19.5, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546942' target='https://doi.org/10.1101/2023.06.28.546942'> 10.1101/2023.06.28.546942</a></p>
        <p class="abstract">Endogenous tags have become invaluable tools to visualize and study native proteins in live cells. However, generating human cell lines carrying endogenous tags is difficult due to the low efficiency of homology-directed repair. Recently, an engineered split mNeonGreen protein was used to generate a large-scale endogenous tag library in HEK293 cells. Using split mNeonGreen for large-scale endogenous tagging in human iPSCs would open the door to studying protein function in healthy cells and across differentiated cell types. We engineered an iPS cell line to express the large fragment of the split mNeonGreen protein (mNG21-10) and showed that it enables fast and efficient endogenous tagging of proteins with the short fragment (mNG211). We also demonstrate that neural network-based image restoration enables live imaging studies of highly dynamic cellular processes such as cytokinesis in iPSCs. This work represents the first step towards a genome-wide endogenous tag library in human stem cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.02.547209">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.02.547209" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.02.547209">
        <p class="paperTitle">Raman micro-spectroscopy reveals the spatial distribution of fumarate in cells and tissues.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.02.547209" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.02.547209" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kamp, M.; Surmacki, J.; Segarra Mondejar, M.; Young, T.; Chrabaszcz, K.; Joud, F.; Zecchini, V.; Speed, A.; Frezza, C.; Bohndiek, S.</p>
        <p class="info">Score: 15.6, Published: 2023-07-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.02.547209' target='https://doi.org/10.1101/2023.07.02.547209'> 10.1101/2023.07.02.547209</a></p>
        <p class="abstract">Aberrantly accumulated metabolites such as fumarate elicit intra- and inter-cellular prooncogenic cascades, yet current methods to measure them require sample perturbation or disruption and lack spatio-temporal resolution, limiting our ability to fully characterize their function and distribution in cells and within a tissue. Raman spectroscopy (RS) is a powerful bio-analytical tool that directly characterizes the chemical composition of a sample based solely on the optical fingerprint of vibrational modes. Here, we show for the first time that RS can directly detect fumarate in living cells in vivo and animal tissues ex vivo. Using the observed linear relationship between Raman scattered intensity and fumarate concentration, we demonstrate that RS can distinguish between Fumarate hydratase (Fh1)-deficient and Fh1-proficient cells based on their fumarate concentration. Moreover, RS reveals the spatial compartmentalization of fumarate within cellular organelles: consistent with disruptive methods, in Fh1-deficient cells we observe the highest fumarate concentration (37 &#43;/- 19 mM) in the mitochondria, where the TCA cycle operates, followed by the cytoplasm (24 &#43;/- 13 mM) and then the nucleus (9 &#43;/- 6 mM). Finally, we apply RS to tissues from an inducible mouse model of FH loss in the kidney, demonstrating that RS can accurately classify FH status in these tissues. These results suggest that RS could be adopted as a valuable tool for small molecule metabolic imaging, enabling in situ dynamic evaluation of fumarate compartmentalization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547150">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547150" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547150">
        <p class="paperTitle">Matrix stiffening promotes perinuclear clustering of mitochondria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547150" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547150" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Daga, P.; Thurakkal, B.; Rawal, S.; Das, T.</p>
        <p class="info">Score: 7.1, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547150' target='https://doi.org/10.1101/2023.06.29.547150'> 10.1101/2023.06.29.547150</a></p>
        <p class="abstract">Mechanical cues from the tissue microenvironment, such as the stiffness of the extracellular matrix, modulate cellular forms and functions. As numerous studies have shown, this modulation depends on the stiffness-dependent remodeling of cytoskeletal elements. In contrast, very little is known about how the intracellular organelles such as mitochondria respond to matrix stiffness and whether their form, function, and localization change accordingly. Here, we performed an extensive quantitative characterization of mitochondrial morphology, subcellular localization, dynamics and membrane tension on soft and stiff matrices. This characterization revealed that while matrix stiffness affected all these aspects, matrix stiffening most distinctively led to an increased perinuclear clustering of mitochondria. Subsequently, we could identify the matrix stiffness-sensitive perinuclear localization of filamin as the key factor dictating this perinuclear clustering. Photo-conversion labeling and fluorescent recovery after photobleaching experiments revealed that perinuclear and peripheral mitochondrial populations differed in their motility on the soft matrix but surprisingly they did not show any difference on the stiff matrix. Finally, perinuclear mitochondrial clustering appeared to be crucial for priming human mesenchymal stem cells towards osteogenesis on the stiff matrix. Taken together, we elucidate a dependence of mitochondrial localization on matrix stiffness, which possibly enables a cell to adapt to its microenvironment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546565">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546565" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546565">
        <p class="paperTitle">Combinatorial selective ER-phagy remodels the ER during neurogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546565" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546565" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hoyer, M. J.; Smith, I. R.; Paoli, J. C.; Jiang, Y.; Paulo, J. A.; Harper, W.</p>
        <p class="info">Score: 28.8, Published: 2023-06-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546565' target='https://doi.org/10.1101/2023.06.26.546565'> 10.1101/2023.06.26.546565</a></p>
        <p class="abstract">The endoplasmic reticulum (ER) has a vast proteomic landscape to perform many diverse functions including protein and lipid synthesis, calcium ion flux, and inter-organelle communication. The ER proteome is remodeled in part through membrane-embedded receptors linking ER to degradative autophagy machinery (selective ER-phagy)1, 2. A refined tubular ER network3, 4 is formed in neurons within highly polarized dendrites and axons5, 6. Autophagy-deficient neurons in vivo display axonal ER accumulation within synaptic ER boutons,7 and the ER-phagy receptor FAM134B has been genetically linked with human sensory and autonomic neuropathy8, 9. However, mechanisms, including receptor selectivity, that define ER remodeling by autophagy in neurons are limited. Here, we combine a genetically tractable induced neuron (iNeuron) system for monitoring extensive ER remodeling during differentiation with proteomic and computational tools to create a quantitative landscape of ER proteome remodeling via selective autophagy. Through analysis of single and combinatorial ER-phagy receptor mutants, we delineate the extent to which each receptor contributes to both magnitude and selectivity of ER clearance via autophagy for individual ER protein cargos. We define specific subsets of ER curvature-shaping proteins or lumenal proteins as preferred clients for distinct receptors. Using spatial sensors and flux reporters, we demonstrate receptor-specific autophagic capture of ER in axons, which correlates with aberrant ER accumulation in axons of ER-phagy receptor or autophagy-deficient neurons. This molecular inventory of ER proteome remodeling and versatile genetic toolkit provides a quantitative framework for understanding contributions of individual ER-phagy receptors for reshaping ER during cell state transitions.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
