<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cell biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cell biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Self-extinguishing relay waves enable homeostaticcontrol of human neutrophil swarming
Authors: Strickland, J.; Pan, D.; Godfrey, C.; Kim, J. S.; Hopke, A.; Degrange, M.; Villavicencio, B.; Mansour, M. K.; Zerbe, C. S.; Irimia, D.; Amir, A.; Weiner, O. D.
Score: 563.8, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546744
Neutrophils exhibit self-amplified swarming to sites of injury and infection. How swarming is controlled to ensure the proper level of neutrophil recruitment is unknown. Using an ex vivo model of infection, we find that human neutrophils use active relay to generate multiple pulsatile waves of swarming signals.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cell-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cell biology" />
<meta property="og:description" content="Self-extinguishing relay waves enable homeostaticcontrol of human neutrophil swarming
Authors: Strickland, J.; Pan, D.; Godfrey, C.; Kim, J. S.; Hopke, A.; Degrange, M.; Villavicencio, B.; Mansour, M. K.; Zerbe, C. S.; Irimia, D.; Amir, A.; Weiner, O. D.
Score: 563.8, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546744
Neutrophils exhibit self-amplified swarming to sites of injury and infection. How swarming is controlled to ensure the proper level of neutrophil recruitment is unknown. Using an ex vivo model of infection, we find that human neutrophils use active relay to generate multiple pulsatile waves of swarming signals." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cell-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-09T10:40:17+00:00" />
<meta property="article:modified_time" content="2023-07-09T10:40:17+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cell biology"/>
<meta name="twitter:description" content="Self-extinguishing relay waves enable homeostaticcontrol of human neutrophil swarming
Authors: Strickland, J.; Pan, D.; Godfrey, C.; Kim, J. S.; Hopke, A.; Degrange, M.; Villavicencio, B.; Mansour, M. K.; Zerbe, C. S.; Irimia, D.; Amir, A.; Weiner, O. D.
Score: 563.8, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546744
Neutrophils exhibit self-amplified swarming to sites of injury and infection. How swarming is controlled to ensure the proper level of neutrophil recruitment is unknown. Using an ex vivo model of infection, we find that human neutrophils use active relay to generate multiple pulsatile waves of swarming signals."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cell biology",
      "item": "https://trxiv.yorks0n.com/posts/cell-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cell biology",
  "name": "cell biology",
  "description": "Self-extinguishing relay waves enable homeostaticcontrol of human neutrophil swarming\nAuthors: Strickland, J.; Pan, D.; Godfrey, C.; Kim, J. S.; Hopke, A.; Degrange, M.; Villavicencio, B.; Mansour, M. K.; Zerbe, C. S.; Irimia, D.; Amir, A.; Weiner, O. D.\nScore: 563.8, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546744\nNeutrophils exhibit self-amplified swarming to sites of injury and infection. How swarming is controlled to ensure the proper level of neutrophil recruitment is unknown. Using an ex vivo model of infection, we find that human neutrophils use active relay to generate multiple pulsatile waves of swarming signals.",
  "keywords": [
    
  ],
  "articleBody": " Self-extinguishing relay waves enable homeostaticcontrol of human neutrophil swarming\nAuthors: Strickland, J.; Pan, D.; Godfrey, C.; Kim, J. S.; Hopke, A.; Degrange, M.; Villavicencio, B.; Mansour, M. K.; Zerbe, C. S.; Irimia, D.; Amir, A.; Weiner, O. D.\nScore: 563.8, Published: 2023-06-28 DOI: 10.1101/2023.06.27.546744\nNeutrophils exhibit self-amplified swarming to sites of injury and infection. How swarming is controlled to ensure the proper level of neutrophil recruitment is unknown. Using an ex vivo model of infection, we find that human neutrophils use active relay to generate multiple pulsatile waves of swarming signals. Unlike classic active relay systems such as action potentials, neutrophil swarming relay waves are self-extinguishing, limiting the spatial range of cell recruitment. We identify an NADPH-oxidase-based negative feedback loop that is needed for this self-extinguishing behavior. Through this circuit, neutrophils adjust the number and size of swarming waves for homeostatic levels of cell recruitment over a wide range of initial cell densities. We link a broken homeostat to neutrophil over-recruitment in the context of human chronic granulomatous disease.\nDynein and dynactin move long-range but are delivered separately to the axon tip\nAuthors: Fellows, A. D.; Bruntraeger, M.; Burgold, T.; Bassett, A. R.; Carter, A. P.\nScore: 48.3, Published: 2023-07-03 DOI: 10.1101/2023.07.03.547521\nThe microtubule motor dynein drives retrograde transport from the axon tip back to the cell body and is essential for neuronal survival. Its function requires its cofactor dynactin and regulators LIS1 and NDEL1. However, it is unclear if all dynein components can travel along the axon, how far they move and whether they do so together. Here, we use neuron-inducible (NGN2-OPTi-OX) human-stem-cell lines to endogenously tag dynein components and track them under a near single molecule regime. In the retrograde direction dynein and dynactin can move the entire \u003e 500 {micro}m length of the axon in one go. Furthermore, LIS1 and NDEL1 also undergo long-distance movement, despite being mainly implicated with initiation of transport. Intriguingly, in the anterograde direction dynein/LIS1 move faster than dynactin/NDEL1 implying they can travel on different cargos. Therefore, neurons ensure efficient transport by keeping dynein/dynactin on cargos over long distances, but keeping them separate until their function is required.\nTREX reveals proteins that bind to specific RNA regions in living cells\nAuthors: Dodel, M.; Guiducci, G.; Dermit, M.; Krishnamurthy, S.; Stojic, L.; Mardakheh, F. K.\nScore: 60.8, Published: 2023-06-30 DOI: 10.1101/2023.06.30.547259\nDifferent regions of RNA molecules can often engage in specific interactions with distinct RNA-binding proteins (RBPs), giving rise to diverse modalities of RNA regulation and function. However, there are currently no methods for unbiased identification of RBPs that interact with specific RNA regions in living cells under endogenous settings. Here, we introduce TREX (Targeted RNase H-mediated extraction of crosslinked RBPs), a highly sensitive approach for identifying proteins that directly bind to specific RNA regions in living cells. We demonstrate that TREX outperforms existing methods in identifying known interactors of U1 snRNA, and reveals endogenous region-specific interactors of NORAD lncRNA. Using TREX, we generated a comprehensive region-by-region interactome for 45S rRNA, uncovering both established and novel interactions that regulate ribosome biogenesis. With its applicability to any RNA in any cell-type, TREX is the first RNA-centric tool for unbiased positional mapping of endogenous RNA-protein interactions in living cells.\nThe Tubulin Nano-Code: a protofilament-specific pattern of tubulin post-translational modifications regulates ciliary beating mechanics\nAuthors: Alvarez Viar, G.; Klena, N.; Martino, F.; Nievergelt, A. P.; Pigino, G.\nScore: 23.9, Published: 2023-06-28 DOI: 10.1101/2023.06.28.546853\nControl of ciliary beating is crucial for motility and signaling in eukaryotic cells and requires spatially restricted interactions between axonemal proteins and specific protofilaments within the ciliary microtubules. How these interactions are regulated remains poorly understood, but increasing evidence indicates that tubulin post-translational modifications (tPTMs) are required for proper ciliary motility. The Tubulin Code refers to the concept that tPTMs can modulate the function of individual microtubules in cells. Here we use a combination of immuno-cryo-electron tomography, expansion microscopy and mutant analysis to show that, in motile cilia, tubulin glycylation and polyglutamylation form mutually exclusive protofilament-specific nano-patterns at sub-microtubular scale. We show that these two nano-patterns are consistent with the distributions of axonemal dyneins and nexin-dynein regulatory complexes, respectively, and are required for their regulation during ciliary beating. Our discovery of a tubulin nano-code in cilia highlights the need of higher-resolution studies also in other cellular compartments to understand the molecular role of tPTMs. One-Sentence SummaryTubulin post-translational modifications form nanopatterns at sub-microtubular scale that enable individual protofilaments to perform specific functions Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=126 SRC=\"FIGDIR/small/546853v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (62K): org.highwire.dtl.DTLVardef@253704org.highwire.dtl.DTLVardef@13f0a8forg.highwire.dtl.DTLVardef@1ce673dorg.highwire.dtl.DTLVardef@1dc04e1_HPS_FORMAT_FIGEXP M_FIG C_FIG\nRaman micro-spectroscopy reveals the spatial distribution of fumarate in cells and tissues.\nAuthors: Kamp, M.; Surmacki, J.; Segarra Mondejar, M.; Young, T.; Chrabaszcz, K.; Joud, F.; Zecchini, V.; Speed, A.; Frezza, C.; Bohndiek, S.\nScore: 16.4, Published: 2023-07-02 DOI: 10.1101/2023.07.02.547209\nAberrantly accumulated metabolites such as fumarate elicit intra- and inter-cellular pro-oncogenic cascades, yet current methods to measure them require sample perturbation or disruption and lack spatio-temporal resolution, limiting our ability to fully characterize their function and distribution in cells and within a tissue. Raman spectroscopy (RS) is a powerful bio-analytical tool that directly characterizes the chemical composition of a sample based solely on the optical fingerprint of vibrational modes. Here, we show for the first time that RS can directly detect fumarate in living cells in vivo and animal tissues ex vivo. Using the observed linear relationship between Raman scattered intensity and fumarate concentration, we demonstrate that RS can distinguish between Fumarate hydratase (Fh1)-deficient and Fh1-proficient cells based on their fumarate concentration. Moreover, RS reveals the spatial compartmentalization of fumarate within cellular organelles: consistent with disruptive methods, in Fh1-deficient cells we observe the highest fumarate concentration (37 {+/-} 19 mM) in the mitochondria, where the TCA cycle operates, followed by the cytoplasm (24 {+/-} 13 mM) and then the nucleus (9 {+/-} 6 mM). Finally, we apply RS to tissues from an inducible mouse model of FH loss in the kidney, demonstrating that RS can accurately classify FH status in these tissues. These results suggest that RS could be adopted as a valuable tool for small molecule metabolic imaging, enabling in situ dynamic evaluation of fumarate compartmentalization.\nA role for Vps13-mediated lipid transfer at the ER-endosome contact site in ESCRT-mediated sorting\nAuthors: Suzuki, S.; West, M.; Zhang, Y.; Fan, J. S.; Roberts, R. T.; Odorizzi, G.; Emr, S. D.\nScore: 14.0, Published: 2023-07-07 DOI: 10.1101/2023.07.06.547997\nEndosomes are specialized organelles that function in the secretory and endocytic protein sorting pathways. Endocytosed cell surface receptors and transporters destined for lysosomal degradation are sorted into intralumenal vesicles (ILVs) at endosomes by Endosomal Sorting Complex Required for Transport (ESCRT) proteins. The endosomes (multivesicular bodies, MVBs) then fuse with the lysosome. During endosomal maturation, the number of ILVs increases, but the size of endosomes does not decrease despite consumption of the limiting membrane during ILV formation. Vesicle-mediated trafficking is thought to provide lipids to support MVB biogenesis. However, we have uncovered an unexpected contribution of a large bridge-like lipid transfer protein, Vps13, in this process. Here, we reveal that Vps13-mediated lipid transfer at ER-endosome contact sites is required for the ESCRT pathway. We propose that Vps13 may play a critical role in supplying lipids to the endosome, ensuring continuous ESCRT-mediated sorting during MVB formation.\nMetabolism regulates muscle stem cell self-renewal by connecting the microenvironment and histone acetylation\nAuthors: Ly, C. H.; Chung, J. D.; Nguyen, J. H.; Tian, L.; Schroeder, J.; Knaupp, A. S.; Su, S.; Trieu, J.; Salmi, T. M.; Zalcenstein, D.; Jabbari, J. S.; Boughton, B. A.; Cox, A. G.; Naik, S. H.; Polo, J. M.; Ritchie, M. E.; Lynch, G. S.; Ryall, J. G.\nScore: 11.3, Published: 2023-07-05 DOI: 10.1101/2023.07.04.547746\nSkeletal muscle contains a resident population of somatic stem cells capable of both self-renewal and differentiation. The signals that regulate this important decision have yet to be fully elucidated. Here we use metabolomics and mass spectrometry imaging (MSI) to identity a state of localized hyperglycaemia following skeletal muscle injury. We show that committed muscle progenitor cells exhibit an enrichment of glycolytic and TCA cycle genes and that extracellular monosaccharide availability regulates intracellular citrate levels and global histone acetylation. Muscle stem cells exposed to a reduced (or altered) monosaccharide environment demonstrate reduced global histone acetylation and transcription of myogenic determination factors (including myod1). Importantly, reduced monosaccharide availability was linked directly to increased rates of asymmetric division and muscle stem cell self-renewal in regenerating skeletal muscle. Our results reveal an important role for the extracellular metabolic environment in the decision to undergo self-renewal or myogenic commitment during skeletal muscle regeneration.\nEndogenous tagging using split mNeonGreen in human iPSCs for live imaging studies\nAuthors: Husser, M. C.; Pham, N. P.; Law, C.; Araujo, F. R. B.; Martin, V. J. J.; Piekny, A.\nScore: 26.6, Published: 2023-06-28 DOI: 10.1101/2023.06.28.546942\nEndogenous tags have become invaluable tools to visualize and study native proteins in live cells. However, generating human cell lines carrying endogenous tags is difficult due to the low efficiency of homology-directed repair. Recently, an engineered split mNeonGreen protein was used to generate a large-scale endogenous tag library in HEK293 cells. Using split mNeonGreen for large-scale endogenous tagging in human iPSCs would open the door to studying protein function in healthy cells and across differentiated cell types. We engineered an iPS cell line to express the large fragment of the split mNeonGreen protein (mNG21-10) and showed that it enables fast and efficient endogenous tagging of proteins with the short fragment (mNG211). We also demonstrate that neural network-based image restoration enables live imaging studies of highly dynamic cellular processes such as cytokinesis in iPSCs. This work represents the first step towards a genome-wide endogenous tag library in human stem cells.\nDirect recruitment of Mis18 to interphase spindle poles promotes CENP-A chromatin assembly\nAuthors: London, N.; Median-Pritchard, B.; Spanos, C.; Rappsilber, J.; Jeyaprakash, A. A.; Allshire, R.\nScore: 6.4, Published: 2023-07-04 DOI: 10.1101/2023.07.03.547485\nCENP-A chromatin specifies mammalian centromere identity, and its chaperone HJURP replenishes CENP-A when recruited by the Mis18 complex (Mis18C) via M18BP/KNL2 to CENP-C at kinetochores during interphase. However, the Mis18C recruitment mechanism remains unresolved in species lacking M18BP1, such as fission yeast. Fission yeast centromeres cluster at G2 spindle pole bodies (SPBs) when CENP-ACnp1 is replenished and where Mis18C also localizes. We show that SPBs play an unexpected role in concentrating Mis18C near centromeres through the recruitment of Mis18 by direct binding to the major SPB LInker of Nucleoskeleton and Cytoskeleton (LINC) complex component Sad1. Mis18 recruitment by Sad1 is important for CENP-ACnp1 chromatin establishment and acts in parallel with a CENP-C-mediated Mis18C recruitment pathway to maintain centromeric CENP-ACnp1, but is independent of Sad1-mediated centromere clustering. SPBs therefore provide a non-chromosomal scaffold for both Mis18C recruitment and centromere clustering during G2. This centromere-independent Mis18-SPB recruitment provides a mechanism that governs de novo CENP-ACnp1 chromatin assembly by the proximity of appropriate sequences to SPBs and highlights how nuclear spatial organization influences centromere identity.\nA Serum- and Feeder-Free System to Generate CD4 and Regulatory T Cells from Human iPSCs\nAuthors: Fong, H.; Mendel, M.; Jascur, J.; Najmi, L.; Kim, K.; Lew, G.; Garimalla, S.; Schock, S.; Hu, J.; Villegas, A.; Conway, A.; Fontenot, J.; Zompi, S.\nScore: 5.5, Published: 2023-07-02 DOI: 10.1101/2023.07.01.547333\niPSCs can serve as a renewable source of a consistent edited cell product, overcoming limitations of primary cells. While feeder-free generation of clinical grade iPSC-derived CD8 T cells has been achieved, differentiation of iPSC-derived CD4sp and regulatory T cells requires mouse stromal cells in an artificial thymic organoid. Here we report a serum- and feeder-free differentiation process suitable for large-scale production. Using an optimized concentration of PMA/Ionomycin, we generated iPSC-CD4sp T cells at high efficiency and converted them to Tregs using TGF{beta} and ATRA. Using zinc finger nucleases, we demonstrated high non-viral, targeted integration of an HLA-A2 CAR in iPSCs. iPSC-Tregs +/- HLA-A2 CAR phenotypically, transcriptionally and functionally resemble primary Tregs and suppress T cell proliferation in vitro. Our work is the first to demonstrate an iPSC-based platform amenable to manufacturing CD4 T cells to complement iPSC-CD8 oncology products and functional iPSC-Tregs to deliver Treg cell therapies at scale.\n",
  "wordCount" : "2006",
  "inLanguage": "en",
  "datePublished": "2023-07-09T10:40:17Z",
  "dateModified": "2023-07-09T10:40:17Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cell-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cell biology
    </h1>
    <div class="post-meta"><span>updated on July 9, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546744">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546744" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546744">
        <p class="paperTitle">Self-extinguishing relay waves enable homeostaticcontrol of human neutrophil swarming</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546744" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546744" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Strickland, J.; Pan, D.; Godfrey, C.; Kim, J. S.; Hopke, A.; Degrange, M.; Villavicencio, B.; Mansour, M. K.; Zerbe, C. S.; Irimia, D.; Amir, A.; Weiner, O. D.</p>
        <p class="info">Score: 563.8, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546744' target='https://doi.org/10.1101/2023.06.27.546744'> 10.1101/2023.06.27.546744</a></p>
        <p class="abstract">Neutrophils exhibit self-amplified swarming to sites of injury and infection. How swarming is controlled to ensure the proper level of neutrophil recruitment is unknown. Using an ex vivo model of infection, we find that human neutrophils use active relay to generate multiple pulsatile waves of swarming signals. Unlike classic active relay systems such as action potentials, neutrophil swarming relay waves are self-extinguishing, limiting the spatial range of cell recruitment. We identify an NADPH-oxidase-based negative feedback loop that is needed for this self-extinguishing behavior. Through this circuit, neutrophils adjust the number and size of swarming waves for homeostatic levels of cell recruitment over a wide range of initial cell densities. We link a broken homeostat to neutrophil over-recruitment in the context of human chronic granulomatous disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547521">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547521" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547521">
        <p class="paperTitle">Dynein and dynactin move long-range but are delivered separately to the axon tip</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547521" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547521" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fellows, A. D.; Bruntraeger, M.; Burgold, T.; Bassett, A. R.; Carter, A. P.</p>
        <p class="info">Score: 48.3, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547521' target='https://doi.org/10.1101/2023.07.03.547521'> 10.1101/2023.07.03.547521</a></p>
        <p class="abstract">The microtubule motor dynein drives retrograde transport from the axon tip back to the cell body and is essential for neuronal survival. Its function requires its cofactor dynactin and regulators LIS1 and NDEL1. However, it is unclear if all dynein components can travel along the axon, how far they move and whether they do so together. Here, we use neuron-inducible (NGN2-OPTi-OX) human-stem-cell lines to endogenously tag dynein components and track them under a near single molecule regime. In the retrograde direction dynein and dynactin can move the entire &gt; 500 {micro}m length of the axon in one go. Furthermore, LIS1 and NDEL1 also undergo long-distance movement, despite being mainly implicated with initiation of transport. Intriguingly, in the anterograde direction dynein/LIS1 move faster than dynactin/NDEL1 implying they can travel on different cargos. Therefore, neurons ensure efficient transport by keeping dynein/dynactin on cargos over long distances, but keeping them separate until their function is required.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547259">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547259" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547259">
        <p class="paperTitle">TREX reveals proteins that bind to specific RNA regions in living cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547259" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547259" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dodel, M.; Guiducci, G.; Dermit, M.; Krishnamurthy, S.; Stojic, L.; Mardakheh, F. K.</p>
        <p class="info">Score: 60.8, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547259' target='https://doi.org/10.1101/2023.06.30.547259'> 10.1101/2023.06.30.547259</a></p>
        <p class="abstract">Different regions of RNA molecules can often engage in specific interactions with distinct RNA-binding proteins (RBPs), giving rise to diverse modalities of RNA regulation and function. However, there are currently no methods for unbiased identification of RBPs that interact with specific RNA regions in living cells under endogenous settings. Here, we introduce TREX (Targeted RNase H-mediated extraction of crosslinked RBPs), a highly sensitive approach for identifying proteins that directly bind to specific RNA regions in living cells. We demonstrate that TREX outperforms existing methods in identifying known interactors of U1 snRNA, and reveals endogenous region-specific interactors of NORAD lncRNA. Using TREX, we generated a comprehensive region-by-region interactome for 45S rRNA, uncovering both established and novel interactions that regulate ribosome biogenesis. With its applicability to any RNA in any cell-type, TREX is the first RNA-centric tool for unbiased positional mapping of endogenous RNA-protein interactions in living cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546853">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546853" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546853">
        <p class="paperTitle">The Tubulin Nano-Code: a protofilament-specific pattern of tubulin post-translational modifications regulates ciliary beating mechanics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546853" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546853" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alvarez Viar, G.; Klena, N.; Martino, F.; Nievergelt, A. P.; Pigino, G.</p>
        <p class="info">Score: 23.9, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546853' target='https://doi.org/10.1101/2023.06.28.546853'> 10.1101/2023.06.28.546853</a></p>
        <p class="abstract">Control of ciliary beating is crucial for motility and signaling in eukaryotic cells and requires spatially restricted interactions between axonemal proteins and specific protofilaments within the ciliary microtubules. How these interactions are regulated remains poorly understood, but increasing evidence indicates that tubulin post-translational modifications (tPTMs) are required for proper ciliary motility. The Tubulin Code refers to the concept that tPTMs can modulate the function of individual microtubules in cells. Here we use a combination of immuno-cryo-electron tomography, expansion microscopy and mutant analysis to show that, in motile cilia, tubulin glycylation and polyglutamylation form mutually exclusive protofilament-specific nano-patterns at sub-microtubular scale. We show that these two nano-patterns are consistent with the distributions of axonemal dyneins and nexin-dynein regulatory complexes, respectively, and are required for their regulation during ciliary beating. Our discovery of a tubulin nano-code in cilia highlights the need of higher-resolution studies also in other cellular compartments to understand the molecular role of tPTMs.

One-Sentence SummaryTubulin post-translational modifications form nanopatterns at sub-microtubular scale that enable individual protofilaments to perform specific functions

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=126 SRC=&#34;FIGDIR/small/546853v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (62K):
org.highwire.dtl.DTLVardef@253704org.highwire.dtl.DTLVardef@13f0a8forg.highwire.dtl.DTLVardef@1ce673dorg.highwire.dtl.DTLVardef@1dc04e1_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.02.547209">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.02.547209" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.02.547209">
        <p class="paperTitle">Raman micro-spectroscopy reveals the spatial distribution of fumarate in cells and tissues.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.02.547209" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.02.547209" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kamp, M.; Surmacki, J.; Segarra Mondejar, M.; Young, T.; Chrabaszcz, K.; Joud, F.; Zecchini, V.; Speed, A.; Frezza, C.; Bohndiek, S.</p>
        <p class="info">Score: 16.4, Published: 2023-07-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.02.547209' target='https://doi.org/10.1101/2023.07.02.547209'> 10.1101/2023.07.02.547209</a></p>
        <p class="abstract">Aberrantly accumulated metabolites such as fumarate elicit intra- and inter-cellular pro-oncogenic cascades, yet current methods to measure them require sample perturbation or disruption and lack spatio-temporal resolution, limiting our ability to fully characterize their function and distribution in cells and within a tissue. Raman spectroscopy (RS) is a powerful bio-analytical tool that directly characterizes the chemical composition of a sample based solely on the optical fingerprint of vibrational modes. Here, we show for the first time that RS can directly detect fumarate in living cells in vivo and animal tissues ex vivo. Using the observed linear relationship between Raman scattered intensity and fumarate concentration, we demonstrate that RS can distinguish between Fumarate hydratase (Fh1)-deficient and Fh1-proficient cells based on their fumarate concentration. Moreover, RS reveals the spatial compartmentalization of fumarate within cellular organelles: consistent with disruptive methods, in Fh1-deficient cells we observe the highest fumarate concentration (37 {&#43;/-} 19 mM) in the mitochondria, where the TCA cycle operates, followed by the cytoplasm (24 {&#43;/-} 13 mM) and then the nucleus (9 {&#43;/-} 6 mM). Finally, we apply RS to tissues from an inducible mouse model of FH loss in the kidney, demonstrating that RS can accurately classify FH status in these tissues. These results suggest that RS could be adopted as a valuable tool for small molecule metabolic imaging, enabling in situ dynamic evaluation of fumarate compartmentalization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.547997">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.547997" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.547997">
        <p class="paperTitle">A role for Vps13-mediated lipid transfer at the ER-endosome contact site in ESCRT-mediated sorting</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.547997" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.547997" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Suzuki, S.; West, M.; Zhang, Y.; Fan, J. S.; Roberts, R. T.; Odorizzi, G.; Emr, S. D.</p>
        <p class="info">Score: 14.0, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.547997' target='https://doi.org/10.1101/2023.07.06.547997'> 10.1101/2023.07.06.547997</a></p>
        <p class="abstract">Endosomes are specialized organelles that function in the secretory and endocytic protein sorting pathways. Endocytosed cell surface receptors and transporters destined for lysosomal degradation are sorted into intralumenal vesicles (ILVs) at endosomes by Endosomal Sorting Complex Required for Transport (ESCRT) proteins. The endosomes (multivesicular bodies, MVBs) then fuse with the lysosome. During endosomal maturation, the number of ILVs increases, but the size of endosomes does not decrease despite consumption of the limiting membrane during ILV formation. Vesicle-mediated trafficking is thought to provide lipids to support MVB biogenesis. However, we have uncovered an unexpected contribution of a large bridge-like lipid transfer protein, Vps13, in this process. Here, we reveal that Vps13-mediated lipid transfer at ER-endosome contact sites is required for the ESCRT pathway. We propose that Vps13 may play a critical role in supplying lipids to the endosome, ensuring continuous ESCRT-mediated sorting during MVB formation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547746">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547746" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547746">
        <p class="paperTitle">Metabolism regulates muscle stem cell self-renewal by connecting the microenvironment and histone acetylation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547746" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547746" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ly, C. H.; Chung, J. D.; Nguyen, J. H.; Tian, L.; Schroeder, J.; Knaupp, A. S.; Su, S.; Trieu, J.; Salmi, T. M.; Zalcenstein, D.; Jabbari, J. S.; Boughton, B. A.; Cox, A. G.; Naik, S. H.; Polo, J. M.; Ritchie, M. E.; Lynch, G. S.; Ryall, J. G.</p>
        <p class="info">Score: 11.3, Published: 2023-07-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547746' target='https://doi.org/10.1101/2023.07.04.547746'> 10.1101/2023.07.04.547746</a></p>
        <p class="abstract">Skeletal muscle contains a resident population of somatic stem cells capable of both self-renewal and differentiation. The signals that regulate this important decision have yet to be fully elucidated. Here we use metabolomics and mass spectrometry imaging (MSI) to identity a state of localized hyperglycaemia following skeletal muscle injury. We show that committed muscle progenitor cells exhibit an enrichment of glycolytic and TCA cycle genes and that extracellular monosaccharide availability regulates intracellular citrate levels and global histone acetylation. Muscle stem cells exposed to a reduced (or altered) monosaccharide environment demonstrate reduced global histone acetylation and transcription of myogenic determination factors (including myod1). Importantly, reduced monosaccharide availability was linked directly to increased rates of asymmetric division and muscle stem cell self-renewal in regenerating skeletal muscle. Our results reveal an important role for the extracellular metabolic environment in the decision to undergo self-renewal or myogenic commitment during skeletal muscle regeneration.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546942">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546942" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546942">
        <p class="paperTitle">Endogenous tagging using split mNeonGreen in human iPSCs for live imaging studies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546942" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546942" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Husser, M. C.; Pham, N. P.; Law, C.; Araujo, F. R. B.; Martin, V. J. J.; Piekny, A.</p>
        <p class="info">Score: 26.6, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546942' target='https://doi.org/10.1101/2023.06.28.546942'> 10.1101/2023.06.28.546942</a></p>
        <p class="abstract">Endogenous tags have become invaluable tools to visualize and study native proteins in live cells. However, generating human cell lines carrying endogenous tags is difficult due to the low efficiency of homology-directed repair. Recently, an engineered split mNeonGreen protein was used to generate a large-scale endogenous tag library in HEK293 cells. Using split mNeonGreen for large-scale endogenous tagging in human iPSCs would open the door to studying protein function in healthy cells and across differentiated cell types. We engineered an iPS cell line to express the large fragment of the split mNeonGreen protein (mNG21-10) and showed that it enables fast and efficient endogenous tagging of proteins with the short fragment (mNG211). We also demonstrate that neural network-based image restoration enables live imaging studies of highly dynamic cellular processes such as cytokinesis in iPSCs. This work represents the first step towards a genome-wide endogenous tag library in human stem cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547485">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547485" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547485">
        <p class="paperTitle">Direct recruitment of Mis18 to interphase spindle poles promotes CENP-A chromatin assembly</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547485" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547485" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: London, N.; Median-Pritchard, B.; Spanos, C.; Rappsilber, J.; Jeyaprakash, A. A.; Allshire, R.</p>
        <p class="info">Score: 6.4, Published: 2023-07-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547485' target='https://doi.org/10.1101/2023.07.03.547485'> 10.1101/2023.07.03.547485</a></p>
        <p class="abstract">CENP-A chromatin specifies mammalian centromere identity, and its chaperone HJURP replenishes CENP-A when recruited by the Mis18 complex (Mis18C) via M18BP/KNL2 to CENP-C at kinetochores during interphase. However, the Mis18C recruitment mechanism remains unresolved in species lacking M18BP1, such as fission yeast. Fission yeast centromeres cluster at G2 spindle pole bodies (SPBs) when CENP-ACnp1 is replenished and where Mis18C also localizes. We show that SPBs play an unexpected role in concentrating Mis18C near centromeres through the recruitment of Mis18 by direct binding to the major SPB LInker of Nucleoskeleton and Cytoskeleton (LINC) complex component Sad1. Mis18 recruitment by Sad1 is important for CENP-ACnp1 chromatin establishment and acts in parallel with a CENP-C-mediated Mis18C recruitment pathway to maintain centromeric CENP-ACnp1, but is independent of Sad1-mediated centromere clustering. SPBs therefore provide a non-chromosomal scaffold for both Mis18C recruitment and centromere clustering during G2. This centromere-independent Mis18-SPB recruitment provides a mechanism that governs de novo CENP-ACnp1 chromatin assembly by the proximity of appropriate sequences to SPBs and highlights how nuclear spatial organization influences centromere identity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.01.547333">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.01.547333" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.01.547333">
        <p class="paperTitle">A Serum- and Feeder-Free System to Generate CD4 and Regulatory T Cells from Human iPSCs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.01.547333" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.01.547333" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fong, H.; Mendel, M.; Jascur, J.; Najmi, L.; Kim, K.; Lew, G.; Garimalla, S.; Schock, S.; Hu, J.; Villegas, A.; Conway, A.; Fontenot, J.; Zompi, S.</p>
        <p class="info">Score: 5.5, Published: 2023-07-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.01.547333' target='https://doi.org/10.1101/2023.07.01.547333'> 10.1101/2023.07.01.547333</a></p>
        <p class="abstract">iPSCs can serve as a renewable source of a consistent edited cell product, overcoming limitations of primary cells. While feeder-free generation of clinical grade iPSC-derived CD8 T cells has been achieved, differentiation of iPSC-derived CD4sp and regulatory T cells requires mouse stromal cells in an artificial thymic organoid. Here we report a serum- and feeder-free differentiation process suitable for large-scale production. Using an optimized concentration of PMA/Ionomycin, we generated iPSC-CD4sp T cells at high efficiency and converted them to Tregs using TGF{beta} and ATRA. Using zinc finger nucleases, we demonstrated high non-viral, targeted integration of an HLA-A2 CAR in iPSCs. iPSC-Tregs &#43;/- HLA-A2 CAR phenotypically, transcriptionally and functionally resemble primary Tregs and suppress T cell proliferation in vitro. Our work is the first to demonstrate an iPSC-based platform amenable to manufacturing CD4 T cells to complement iPSC-CD8 oncology products and functional iPSC-Tregs to deliver Treg cell therapies at scale.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
