<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cell biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cell biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Non-homologous end joining shapes the genomic rearrangement landscape of chromothripsis from mitotic errors
Authors: Hu, Q.; Espejo Valle-Inclan, J.; Dahiya, R.; Guyer, A.; Mazzagatti, A.; Maurais, E. G.; Engel, J. L.; Cortes-Ciriano, I.; Ly, P.
Score: 20.8, Published: 2023-08-11 DOI: 10.1101/2023.08.10.552800
Errors in mitosis can generate micronuclei that entrap mis-segregated chromosomes, which are susceptible to catastrophic fragmentation through a process termed chromothripsis. The reassembly of fragmented chromosomes by error-prone DNA double-strand break (DSB) repair generates a spectrum of simple and complex genomic rearrangements that are associated with human cancers and disorders.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cell-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cell biology" />
<meta property="og:description" content="Non-homologous end joining shapes the genomic rearrangement landscape of chromothripsis from mitotic errors
Authors: Hu, Q.; Espejo Valle-Inclan, J.; Dahiya, R.; Guyer, A.; Mazzagatti, A.; Maurais, E. G.; Engel, J. L.; Cortes-Ciriano, I.; Ly, P.
Score: 20.8, Published: 2023-08-11 DOI: 10.1101/2023.08.10.552800
Errors in mitosis can generate micronuclei that entrap mis-segregated chromosomes, which are susceptible to catastrophic fragmentation through a process termed chromothripsis. The reassembly of fragmented chromosomes by error-prone DNA double-strand break (DSB) repair generates a spectrum of simple and complex genomic rearrangements that are associated with human cancers and disorders." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cell-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-16T10:37:20+00:00" />
<meta property="article:modified_time" content="2023-08-16T10:37:20+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cell biology"/>
<meta name="twitter:description" content="Non-homologous end joining shapes the genomic rearrangement landscape of chromothripsis from mitotic errors
Authors: Hu, Q.; Espejo Valle-Inclan, J.; Dahiya, R.; Guyer, A.; Mazzagatti, A.; Maurais, E. G.; Engel, J. L.; Cortes-Ciriano, I.; Ly, P.
Score: 20.8, Published: 2023-08-11 DOI: 10.1101/2023.08.10.552800
Errors in mitosis can generate micronuclei that entrap mis-segregated chromosomes, which are susceptible to catastrophic fragmentation through a process termed chromothripsis. The reassembly of fragmented chromosomes by error-prone DNA double-strand break (DSB) repair generates a spectrum of simple and complex genomic rearrangements that are associated with human cancers and disorders."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cell biology",
      "item": "https://trxiv.yorks0n.com/posts/cell-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cell biology",
  "name": "cell biology",
  "description": "Non-homologous end joining shapes the genomic rearrangement landscape of chromothripsis from mitotic errors\nAuthors: Hu, Q.; Espejo Valle-Inclan, J.; Dahiya, R.; Guyer, A.; Mazzagatti, A.; Maurais, E. G.; Engel, J. L.; Cortes-Ciriano, I.; Ly, P.\nScore: 20.8, Published: 2023-08-11 DOI: 10.1101/2023.08.10.552800\nErrors in mitosis can generate micronuclei that entrap mis-segregated chromosomes, which are susceptible to catastrophic fragmentation through a process termed chromothripsis. The reassembly of fragmented chromosomes by error-prone DNA double-strand break (DSB) repair generates a spectrum of simple and complex genomic rearrangements that are associated with human cancers and disorders.",
  "keywords": [
    
  ],
  "articleBody": " Non-homologous end joining shapes the genomic rearrangement landscape of chromothripsis from mitotic errors\nAuthors: Hu, Q.; Espejo Valle-Inclan, J.; Dahiya, R.; Guyer, A.; Mazzagatti, A.; Maurais, E. G.; Engel, J. L.; Cortes-Ciriano, I.; Ly, P.\nScore: 20.8, Published: 2023-08-11 DOI: 10.1101/2023.08.10.552800\nErrors in mitosis can generate micronuclei that entrap mis-segregated chromosomes, which are susceptible to catastrophic fragmentation through a process termed chromothripsis. The reassembly of fragmented chromosomes by error-prone DNA double-strand break (DSB) repair generates a spectrum of simple and complex genomic rearrangements that are associated with human cancers and disorders. How specific DSB repair pathways recognize and process these lesions remains poorly understood. Here we used CRISPR/Cas9 to systematically inactivate distinct DSB processing or repair pathways and interrogated the rearrangement landscape of fragmented chromosomes from micronuclei. Deletion of canonical non-homologous end joining (NHEJ) components, including DNA-PKcs, LIG4, and XLF, substantially reduced the formation of complex rearrangements and shifted the rearrangement landscape toward simple alterations without the characteristic patterns of cancer-associated chromothripsis. Following reincorporation into the nucleus, fragmented chromosomes localize within micronuclei bodies (MN bodies) and undergo successful ligation by NHEJ within a single cell cycle. In the absence of NHEJ, chromosome fragments were rarely engaged by polymerase theta-mediated alternative end-joining or recombination-based mechanisms, resulting in delayed repair kinetics and persistent 53BP1-labeled MN bodies in the interphase nucleus. Prolonged DNA damage signaling from unrepaired fragments ultimately triggered cell cycle arrest. Thus, we provide evidence supporting NHEJ as the exclusive DSB repair pathway generating complex rearrangements following chromothripsis from mitotic errors.\nATG2A-mediated bridge-like lipid transport regulates lipid droplet accumulation\nAuthors: Korfhage, J. L.; Wan, N.; Elhan, H.; Kauffman, L.; Pineda, M. J.; Fuller, D. M.; Thiam, A. R.; Reinisch, K.; Melia, T.\nScore: 13.1, Published: 2023-08-14 DOI: 10.1101/2023.08.14.553257\nATG2 proteins facilitate bulk lipid transport between membranes. ATG2 is an essential autophagy protein, but ATG2 also localizes to lipid droplets (LDs), and genetic depletion of ATG2 increases LD numbers while impairing fatty acid transport from LDs to mitochondria. How ATG2 supports LD homeostasis and whether lipid transport regulates this homeostasis remains unknown. Here we demonstrate that ATG2 is preferentially recruited to phospholipid monolayers such as those surrounding LDs rather than to phospholipid bilayers. In vitro, ATG2 can drive phospholipid transport from artificial LDs with rates that correlate with the binding affinities, such that phospholipids are moved much more efficiently when one of the ATG2-interacting structures is an artificial LD. ATG2 is thought to exhibit bridge-like lipid transport, with lipids flowing across the protein between membranes. We mutated key amino acids within the bridge to form a transport-dead ATG2 mutant (TD-ATG2A) which we show specifically blocks bridge-like, but not shuttle-like, lipid transport in vitro. TD-ATG2A still localizes to LDs, but is unable to rescue LD accumulation in ATG2 knockout cells. Thus, ATG2 has a natural affinity for, and an enhanced activity upon LD surfaces and uses bridge-like lipid transport to support LD dynamics in cells.\nErythroid differentiation dependent interaction of VPS13A with XK at the plasma membrane of K562 cells\nAuthors: Amos, C.; Xu, P.; De Camilli, P.\nScore: 12.9, Published: 2023-08-10 DOI: 10.1101/2023.08.09.552634\nMutations of the bridge-like lipid transport protein VPS13A and of the lipid scramblase XK result in Chorea Acanthocytosis (ChAc) and McLeod syndrome (MLS) respectively, two similar conditions involving neurodegeneration and deformed erythrocytes (acanthocytes). VPS13A binds XK, suggesting a model in which VPS13A forms a lipid transport bridge between the ER and the plasma membrane (PM) where XK resides. However, studies of VPS13A in HeLa and COS7 cells showed that this protein localizes primarily at contacts of the ER with mitochondria. Overexpression of XK in these cells redistributed VPS13A to the biosynthetic XK pool in the ER but not to PM localized XK. Colocalization of VPS13A with XK at the PM was only observed if overexpressed XK harbored mutations that disengage its VPS13A binding site from an intramolecular interaction. As the acanthocytosis phenotype of ChAc and MLS suggests a role of the two proteins in cells of the erythroid lineage, we explored their localization in K562 cells, which differentiate into erythroblasts upon hemin addition. When tagged VPS13A was overexpressed in hemin treated K562 cells, robust formation of ER-PM contacts positive for VPS13A were observed and their formation was abolished in XK KO cells. ER-PM contacts positive for VPS13A were seldomly observed in undifferentiated K562 cells, in spite of the presence of XK in these cells at concentrations similar to those observed after differentiation. These findings reveal that the interaction of VPS13A with XK at ER-PM contacts requires a permissive state which depends upon cell type and/or functional state of the cell.\nAn adult clock component links circadian rhythms to pancreatic β-cell maturation\nAuthors: Montalvo, A. P.; Gruskin, Z. L.; Leduc, A.; Liu, M.; Gao, Z.; Ahn, J. H.; Straubhaar, J. R.; Slavov, N.; Alvarez-Dominguez, J. R.\nScore: 10.7, Published: 2023-08-12 DOI: 10.1101/2023.08.11.552890\nHow ubiquitous circadian clocks orchestrate tissue-specific outputs is not well understood. Pancreatic {beta} cell-autonomous clocks attune insulin secretion to daily energy cycles, and desynchrony from genetic or behavioral disruptions raises type 2 diabetes risk. We show that the transcription factor DEC1, a clock component induced in adult {beta} cells, coordinates their glucose responsiveness by synchronizing energy metabolism and secretory gene oscillations. Dec1-ablated mice develop lifelong hypo-insulinemic diabetes, despite normal islet formation and intact circadian Clock and Bmal1 activators. DEC1, but not CLOCK/BMAL1, binds maturity-linked genes that mediate respiratory metabolism and insulin exocytosis, and Dec1 loss disrupts their transcription synchrony. Accordingly, {beta}-cell Dec1 ablation causes hypo-insulinemia due to immature glucose responsiveness, dampening insulin rhythms. Thus, Dec1 links circadian clockwork to the {beta}-cell maturation process, aligning metabolism to diurnal energy cycles.\nThe actin binding protein profilin 1 is critical for mitochondria function\nAuthors: Read, T.-A.; Cisterna, B. A.; Skruber, K.; Ahmadieh, S.; Lindamood, H. L.; Vitriol, J.; Shi, Y.; Lefebvre, A. E. Y. T.; Black, J. B.; Butler, M. T.; Bear, J. E.; Cherezova, A.; Ilatovskaya, D. V.; Weintraub, N. L.; Vitriol, E. A.\nScore: 35.5, Published: 2023-08-08 DOI: 10.1101/2023.08.07.552354\nProfilin 1 (PFN1) is an actin binding protein that is vital for the polymerization of monomeric actin into filaments. Here we screened knockout cells for novel functions of PFN1 and discovered that mitophagy, a type of selective autophagy that removes defective or damaged mitochondria from the cell, was significantly upregulated in the absence of PFN1. Despite successful autophagosome formation and fusion with the lysosome, and activation of additional mitochondrial quality control pathways, PFN1 knockout cells still accumulate damaged, dysfunctional mitochondria. Subsequent imaging and functional assays showed that loss of PFN1 significantly affects mitochondria morphology, dynamics, and respiration. Further experiments revealed that PFN1 is located to the mitochondria matrix and is likely regulating mitochondria function from within rather than through polymerizing actin at the mitochondria surface. Finally, PFN1 mutants associated with amyotrophic lateral sclerosis (ALS) fail to rescue PFN1 knockout mitochondrial phenotypes and form aggregates within mitochondria, further perturbing them. Together, these results suggest a novel function for PFN1 in regulating mitochondria and identify a potential pathogenic mechanism of ALS-linked PFN1 variants.\nProjective oblique plane structured illumination microscopy\nAuthors: Chang, B.-J.; Shepherd, D. P.; Fiolka, R. P.\nScore: 9.2, Published: 2023-08-09 DOI: 10.1101/2023.08.08.552447\nStructured illumination microscopy (SIM) can double the spatial resolution of a fluorescence microscope and video rate live cell imaging in a two-dimensional format has been demonstrated. However, rapid implementations of 2D SIM typically only cover a narrow slice of the sample immediately at the coverslip, with most of the cellular volume out of reach. Here we implement oblique plane structured illumination microscopy (OPSIM) in a projection format to rapidly image an entire cell in a 2D SIM framework. As no mechanical scanning of the sample or objective is involved, this technique has the potential for rapid projection imaging with doubled resolution. We characterize the spatial resolution with fluorescent nanospheres, compare projection and 3D imaging using OPSIM and image mitochondria and ER dynamics across an entire cell at up to 2.7 Hz. To our knowledge, this represents the fastest whole cell SIM imaging to date.\nCyclin B3 is a dominant fast-acting cyclin that drives rapid early embryonic mitoses\nAuthors: Lara-Gonzalez, P.; Variyar, S.; Budrewicz, J.; Schlientz, A.; Varshney, N.; Bellaart, A.; Moghareh, S.; Nguyen, A. C. N.; Oegema, K.; Desai, A.\nScore: 8.6, Published: 2023-08-12 DOI: 10.1101/2023.08.11.553011\nIn many species, early embryonic mitoses proceed at a very rapid pace, but how this pace is achieved is not understood. Here we show that in the early C. elegans embryo, cyclin B3 is the dominant driver of rapid embryonic mitoses. Metazoans typically have three cyclin B isoforms that associate with and activate Cdk1 kinase to orchestrate mitotic events: the related cyclins B1 and B2 and the more divergent cyclin B3. We show that whereas embryos expressing cyclins B1 and B2 support slow mitosis (NEBD to Anaphase ~ 600s), the presence of cyclin B3 dominantly drives the ~3-fold faster mitosis observed in wildtype embryos. CYB-1/2-driven mitosis is longer than CYB-3-driven mitosis primarily because the progression of mitotic events itself is slower, rather than delayed anaphase onset due to activation of the spindle checkpoint or inhibitory phosphorylation of the anaphase activator CDC-20. Addition of cyclin B1 to cyclin B3-only mitosis introduces an ~60s delay between the completion of chromosome alignment and anaphase onset, which likely ensures segregation fidelity; this delay is mediated by inhibitory phosphorylation on CDC-20. Thus, the dominance of cyclin B3 in driving mitotic events, coupled to introduction of a short cyclin B1-dependent delay in anaphase onset, sets the rapid pace and ensures fidelity of mitoses in the early C. elegans embryo.\nGolgi membrane proteins YIPF3 and YIPF4 regulate turnover of the Golgi apparatus through autophagy\nAuthors: Kitta, S.; Kaminishi, T.; Higashi, M.; Shima, T.; Kosako, H.; Yoshimori, T.; Kuma, A.\nScore: 8.3, Published: 2023-08-10 DOI: 10.1101/2023.08.09.552599\nThe degradation of organelles by autophagy is essential for cellular homeostasis. The Golgi apparatus has recently been demonstrated to be degraded by autophagy, but little is known about how the Golgi is recognized by the autophagosome. Using quantitative proteomic analysis and two Golgiphagy-reporter systems that we developed, we found that five-transmembrane Golgi-resident proteins, YIPF3 and YIPF4, constitute a Golgiphagy receptor. The YIPF3-YIPF4 complex interacts with LC3B, GABARAP, and GABARAPL1 via the LIR motif in YIPF3, whose stability is dependent on YIPF4. Phosphorylation of the LIR in YIPF3 seems to be required for YIPF3-ATG8 interaction. Moreover, expression of the YIPF3 LIR mutant caused an elongated Golgi morphology, indicating the importance of Golgi turnover via selective autophagy. The reporter assays that we established will pave the way for future studies to obtain deeper insights into Golgiphagy.\nThe pore-forming apolipoprotein APOL7C drives phagosomal rupture and antigen cross-presentation by dendritic cells\nAuthors: Gonzales, G. A.; Huang, S.; Rajwani, J.; Wilkinson, L.; Nguyen, J. A.; Wood, C. M.; Moore, M.; Cedeno, E.; Sikdar, S.; McKenna, N.; Ebacher, V.; Rosin, N. L.; Carneiro, M. B.; Surewaard, B.; Peters, N. C.; Biernaskie, J.; Mahoney, D. J.; Yates, R. M.; Canton, J.\nScore: 8.2, Published: 2023-08-12 DOI: 10.1101/2023.08.11.553042\nType I conventional dendritic cells (cDC1s) are essential for the generation of protective cytotoxic T lymphocyte (CTL) responses against many types of viruses and tumours. They do so by internalizing antigens from virally infected or tumour cells and presenting them to CD8+ T cells in a process known as cross-presentation (XP). Despite the obvious biological importance of XP, the molecular mechanism(s) driving this process remain unclear. Here, we show that a cDC-specific pore-forming protein called apolipoprotein 7C (APOL7C) is upregulated in response to innate immune stimuli and is recruited to phagosomes. Strikingly, the association of APOL7C with phagosomes leads to phagosomal rupture, which in turn allows for the escape of engulfed protein antigens to the cytosol where they can be processed via the endogenous major histocompatibility complex (MHC) class I antigen processing pathway. We show that APOL7C recruitment to phagosomes is voltage-dependent and occurs in response to NADPH oxidase-induced depolarization of the phagosomal membrane. Our data indicate the presence of dedicated pore-forming apolipoproteins that mediate the delivery of phagocytosed proteins to the cytosol of activated cDC1s to facilitate MHC class I presentation of exogenous antigen and to regulate adaptive immunity.\nMulti-scale spatial mapping of cell populations across anatomical sites in healthy human skin and basal cell carcinoma\nAuthors: Ganier, C.; Mazin, P.; Herrera-Oropeza, G.; Du-Harpur, X.; Blakeley, M.; Gabriel, J.; Predeus, A.; Cakir, B.; Prete, M.; Harun, N.; Darrigrand, J.-F.; Haiser, A.; Wyles, S.; Shaw, T. J.; Teichmann, S. A.; Haniffa, M.; Watt, F. M.; Lynch, M. D.\nScore: 6.1, Published: 2023-08-09 DOI: 10.1101/2023.08.08.551504\nOur understanding of how human skin cells differ according to anatomical site and tumour formation is limited. To address this we have created a multi-scale spatial atlas of healthy skin and basal cell carcinoma (BCC), incorporating in vivo optical coherence tomography, single cell RNA sequencing, spatial global transcriptional profiling and in situ sequencing. Computational spatial deconvolution and projection revealed the localisation of distinct cell populations to specific tissue contexts. Although cell populations were conserved between healthy anatomical sites and in BCC, mesenchymal cell populations including fibroblasts and pericytes retained signatures of developmental origin. Spatial profiling and in silico lineage tracing support a hair follicle origin for BCC and demonstrate that cancer-associated fibroblasts are an expansion of a POSTN+ subpopulation associated with hair follicles in healthy skin. RGS5+ pericytes are also expanded in BCC suggesting a role in vascular remodelling. We propose that the identity of mesenchymal cell populations is regulated by signals emanating from adjacent structures and that these signals are repurposed to promote the expansion of skin cancer stroma. The resource we have created is publicly available in an interactive format for the research community. Significance statementSingle cells RNA sequencing has revolutionised cell biology, enabling high resolution analysis of cell types and states within human tissues. Here, we report a comprehensive spatial atlas of adult human skin across different anatomical sites and basal cell carcinoma (BCC) - the most common form of skin cancer - encompassing in vivo optical coherence tomography, single cell RNA sequencing, global spatial transcriptomic profiling and in situ sequencing. In combination these modalities have allowed us to assemble a comprehensive nuclear-resolution atlas of cellular identity in health and disease.\n",
  "wordCount" : "2344",
  "inLanguage": "en",
  "datePublished": "2023-08-16T10:37:20Z",
  "dateModified": "2023-08-16T10:37:20Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cell-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cell biology
    </h1>
    <div class="post-meta"><span>updated on August 16, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552800">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552800" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552800">
        <p class="paperTitle">Non-homologous end joining shapes the genomic rearrangement landscape of chromothripsis from mitotic errors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552800" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552800" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hu, Q.; Espejo Valle-Inclan, J.; Dahiya, R.; Guyer, A.; Mazzagatti, A.; Maurais, E. G.; Engel, J. L.; Cortes-Ciriano, I.; Ly, P.</p>
        <p class="info">Score: 20.8, Published: 2023-08-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552800' target='https://doi.org/10.1101/2023.08.10.552800'> 10.1101/2023.08.10.552800</a></p>
        <p class="abstract">Errors in mitosis can generate micronuclei that entrap mis-segregated chromosomes, which are susceptible to catastrophic fragmentation through a process termed chromothripsis. The reassembly of fragmented chromosomes by error-prone DNA double-strand break (DSB) repair generates a spectrum of simple and complex genomic rearrangements that are associated with human cancers and disorders. How specific DSB repair pathways recognize and process these lesions remains poorly understood. Here we used CRISPR/Cas9 to systematically inactivate distinct DSB processing or repair pathways and interrogated the rearrangement landscape of fragmented chromosomes from micronuclei. Deletion of canonical non-homologous end joining (NHEJ) components, including DNA-PKcs, LIG4, and XLF, substantially reduced the formation of complex rearrangements and shifted the rearrangement landscape toward simple alterations without the characteristic patterns of cancer-associated chromothripsis. Following reincorporation into the nucleus, fragmented chromosomes localize within micronuclei bodies (MN bodies) and undergo successful ligation by NHEJ within a single cell cycle. In the absence of NHEJ, chromosome fragments were rarely engaged by polymerase theta-mediated alternative end-joining or recombination-based mechanisms, resulting in delayed repair kinetics and persistent 53BP1-labeled MN bodies in the interphase nucleus. Prolonged DNA damage signaling from unrepaired fragments ultimately triggered cell cycle arrest. Thus, we provide evidence supporting NHEJ as the exclusive DSB repair pathway generating complex rearrangements following chromothripsis from mitotic errors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.14.553257">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.14.553257" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.14.553257">
        <p class="paperTitle">ATG2A-mediated bridge-like lipid transport regulates lipid droplet accumulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.14.553257" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.14.553257" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Korfhage, J. L.; Wan, N.; Elhan, H.; Kauffman, L.; Pineda, M. J.; Fuller, D. M.; Thiam, A. R.; Reinisch, K.; Melia, T.</p>
        <p class="info">Score: 13.1, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.14.553257' target='https://doi.org/10.1101/2023.08.14.553257'> 10.1101/2023.08.14.553257</a></p>
        <p class="abstract">ATG2 proteins facilitate bulk lipid transport between membranes. ATG2 is an essential autophagy protein, but ATG2 also localizes to lipid droplets (LDs), and genetic depletion of ATG2 increases LD numbers while impairing fatty acid transport from LDs to mitochondria. How ATG2 supports LD homeostasis and whether lipid transport regulates this homeostasis remains unknown. Here we demonstrate that ATG2 is preferentially recruited to phospholipid monolayers such as those surrounding LDs rather than to phospholipid bilayers. In vitro, ATG2 can drive phospholipid transport from artificial LDs with rates that correlate with the binding affinities, such that phospholipids are moved much more efficiently when one of the ATG2-interacting structures is an artificial LD. ATG2 is thought to exhibit bridge-like lipid transport, with lipids flowing across the protein between membranes. We mutated key amino acids within the bridge to form a transport-dead ATG2 mutant (TD-ATG2A) which we show specifically blocks bridge-like, but not shuttle-like, lipid transport in vitro. TD-ATG2A still localizes to LDs, but is unable to rescue LD accumulation in ATG2 knockout cells. Thus, ATG2 has a natural affinity for, and an enhanced activity upon LD surfaces and uses bridge-like lipid transport to support LD dynamics in cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.552634">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.552634" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.552634">
        <p class="paperTitle">Erythroid differentiation dependent interaction of VPS13A with XK at the plasma membrane of K562 cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.552634" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.552634" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Amos, C.; Xu, P.; De Camilli, P.</p>
        <p class="info">Score: 12.9, Published: 2023-08-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.552634' target='https://doi.org/10.1101/2023.08.09.552634'> 10.1101/2023.08.09.552634</a></p>
        <p class="abstract">Mutations of the bridge-like lipid transport protein VPS13A and of the lipid scramblase XK result in Chorea Acanthocytosis (ChAc) and McLeod syndrome (MLS) respectively, two similar conditions involving neurodegeneration and deformed erythrocytes (acanthocytes). VPS13A binds XK, suggesting a model in which VPS13A forms a lipid transport bridge between the ER and the plasma membrane (PM) where XK resides. However, studies of VPS13A in HeLa and COS7 cells showed that this protein localizes primarily at contacts of the ER with mitochondria. Overexpression of XK in these cells redistributed VPS13A to the biosynthetic XK pool in the ER but not to PM localized XK. Colocalization of VPS13A with XK at the PM was only observed if overexpressed XK harbored mutations that disengage its VPS13A binding site from an intramolecular interaction. As the acanthocytosis phenotype of ChAc and MLS suggests a role of the two proteins in cells of the erythroid lineage, we explored their localization in K562 cells, which differentiate into erythroblasts upon hemin addition. When tagged VPS13A was overexpressed in hemin treated K562 cells, robust formation of ER-PM contacts positive for VPS13A were observed and their formation was abolished in XK KO cells. ER-PM contacts positive for VPS13A were seldomly observed in undifferentiated K562 cells, in spite of the presence of XK in these cells at concentrations similar to those observed after differentiation. These findings reveal that the interaction of VPS13A with XK at ER-PM contacts requires a permissive state which depends upon cell type and/or functional state of the cell.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.552890">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.552890" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.552890">
        <p class="paperTitle">An adult clock component links circadian rhythms to pancreatic β-cell maturation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.552890" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.552890" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Montalvo, A. P.; Gruskin, Z. L.; Leduc, A.; Liu, M.; Gao, Z.; Ahn, J. H.; Straubhaar, J. R.; Slavov, N.; Alvarez-Dominguez, J. R.</p>
        <p class="info">Score: 10.7, Published: 2023-08-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.552890' target='https://doi.org/10.1101/2023.08.11.552890'> 10.1101/2023.08.11.552890</a></p>
        <p class="abstract">How ubiquitous circadian clocks orchestrate tissue-specific outputs is not well understood. Pancreatic {beta} cell-autonomous clocks attune insulin secretion to daily energy cycles, and desynchrony from genetic or behavioral disruptions raises type 2 diabetes risk. We show that the transcription factor DEC1, a clock component induced in adult {beta} cells, coordinates their glucose responsiveness by synchronizing energy metabolism and secretory gene oscillations. Dec1-ablated mice develop lifelong hypo-insulinemic diabetes, despite normal islet formation and intact circadian Clock and Bmal1 activators. DEC1, but not CLOCK/BMAL1, binds maturity-linked genes that mediate respiratory metabolism and insulin exocytosis, and Dec1 loss disrupts their transcription synchrony. Accordingly, {beta}-cell Dec1 ablation causes hypo-insulinemia due to immature glucose responsiveness, dampening insulin rhythms. Thus, Dec1 links circadian clockwork to the {beta}-cell maturation process, aligning metabolism to diurnal energy cycles.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552354">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552354" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552354">
        <p class="paperTitle">The actin binding protein profilin 1 is critical for mitochondria function</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552354" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552354" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Read, T.-A.; Cisterna, B. A.; Skruber, K.; Ahmadieh, S.; Lindamood, H. L.; Vitriol, J.; Shi, Y.; Lefebvre, A. E. Y. T.; Black, J. B.; Butler, M. T.; Bear, J. E.; Cherezova, A.; Ilatovskaya, D. V.; Weintraub, N. L.; Vitriol, E. A.</p>
        <p class="info">Score: 35.5, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552354' target='https://doi.org/10.1101/2023.08.07.552354'> 10.1101/2023.08.07.552354</a></p>
        <p class="abstract">Profilin 1 (PFN1) is an actin binding protein that is vital for the polymerization of monomeric actin into filaments. Here we screened knockout cells for novel functions of PFN1 and discovered that mitophagy, a type of selective autophagy that removes defective or damaged mitochondria from the cell, was significantly upregulated in the absence of PFN1. Despite successful autophagosome formation and fusion with the lysosome, and activation of additional mitochondrial quality control pathways, PFN1 knockout cells still accumulate damaged, dysfunctional mitochondria. Subsequent imaging and functional assays showed that loss of PFN1 significantly affects mitochondria morphology, dynamics, and respiration. Further experiments revealed that PFN1 is located to the mitochondria matrix and is likely regulating mitochondria function from within rather than through polymerizing actin at the mitochondria surface. Finally, PFN1 mutants associated with amyotrophic lateral sclerosis (ALS) fail to rescue PFN1 knockout mitochondrial phenotypes and form aggregates within mitochondria, further perturbing them. Together, these results suggest a novel function for PFN1 in regulating mitochondria and identify a potential pathogenic mechanism of ALS-linked PFN1 variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552447">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552447" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552447">
        <p class="paperTitle">Projective oblique plane structured illumination microscopy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552447" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552447" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chang, B.-J.; Shepherd, D. P.; Fiolka, R. P.</p>
        <p class="info">Score: 9.2, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552447' target='https://doi.org/10.1101/2023.08.08.552447'> 10.1101/2023.08.08.552447</a></p>
        <p class="abstract">Structured illumination microscopy (SIM) can double the spatial resolution of a fluorescence microscope and video rate live cell imaging in a two-dimensional format has been demonstrated. However, rapid implementations of 2D SIM typically only cover a narrow slice of the sample immediately at the coverslip, with most of the cellular volume out of reach. Here we implement oblique plane structured illumination microscopy (OPSIM) in a projection format to rapidly image an entire cell in a 2D SIM framework. As no mechanical scanning of the sample or objective is involved, this technique has the potential for rapid projection imaging with doubled resolution. We characterize the spatial resolution with fluorescent nanospheres, compare projection and 3D imaging using OPSIM and image mitochondria and ER dynamics across an entire cell at up to 2.7 Hz. To our knowledge, this represents the fastest whole cell SIM imaging to date.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.553011">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.553011" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.553011">
        <p class="paperTitle">Cyclin B3 is a dominant fast-acting cyclin that drives rapid early embryonic mitoses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.553011" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.553011" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lara-Gonzalez, P.; Variyar, S.; Budrewicz, J.; Schlientz, A.; Varshney, N.; Bellaart, A.; Moghareh, S.; Nguyen, A. C. N.; Oegema, K.; Desai, A.</p>
        <p class="info">Score: 8.6, Published: 2023-08-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.553011' target='https://doi.org/10.1101/2023.08.11.553011'> 10.1101/2023.08.11.553011</a></p>
        <p class="abstract">In many species, early embryonic mitoses proceed at a very rapid pace, but how this pace is achieved is not understood. Here we show that in the early C. elegans embryo, cyclin B3 is the dominant driver of rapid embryonic mitoses. Metazoans typically have three cyclin B isoforms that associate with and activate Cdk1 kinase to orchestrate mitotic events: the related cyclins B1 and B2 and the more divergent cyclin B3. We show that whereas embryos expressing cyclins B1 and B2 support slow mitosis (NEBD to Anaphase ~ 600s), the presence of cyclin B3 dominantly drives the ~3-fold faster mitosis observed in wildtype embryos. CYB-1/2-driven mitosis is longer than CYB-3-driven mitosis primarily because the progression of mitotic events itself is slower, rather than delayed anaphase onset due to activation of the spindle checkpoint or inhibitory phosphorylation of the anaphase activator CDC-20. Addition of cyclin B1 to cyclin B3-only mitosis introduces an ~60s delay between the completion of chromosome alignment and anaphase onset, which likely ensures segregation fidelity; this delay is mediated by inhibitory phosphorylation on CDC-20. Thus, the dominance of cyclin B3 in driving mitotic events, coupled to introduction of a short cyclin B1-dependent delay in anaphase onset, sets the rapid pace and ensures fidelity of mitoses in the early C. elegans embryo.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.552599">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.552599" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.552599">
        <p class="paperTitle">Golgi membrane proteins YIPF3 and YIPF4 regulate turnover of the Golgi apparatus through autophagy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.552599" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.552599" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kitta, S.; Kaminishi, T.; Higashi, M.; Shima, T.; Kosako, H.; Yoshimori, T.; Kuma, A.</p>
        <p class="info">Score: 8.3, Published: 2023-08-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.552599' target='https://doi.org/10.1101/2023.08.09.552599'> 10.1101/2023.08.09.552599</a></p>
        <p class="abstract">The degradation of organelles by autophagy is essential for cellular homeostasis. The Golgi apparatus has recently been demonstrated to be degraded by autophagy, but little is known about how the Golgi is recognized by the autophagosome. Using quantitative proteomic analysis and two Golgiphagy-reporter systems that we developed, we found that five-transmembrane Golgi-resident proteins, YIPF3 and YIPF4, constitute a Golgiphagy receptor. The YIPF3-YIPF4 complex interacts with LC3B, GABARAP, and GABARAPL1 via the LIR motif in YIPF3, whose stability is dependent on YIPF4. Phosphorylation of the LIR in YIPF3 seems to be required for YIPF3-ATG8 interaction. Moreover, expression of the YIPF3 LIR mutant caused an elongated Golgi morphology, indicating the importance of Golgi turnover via selective autophagy. The reporter assays that we established will pave the way for future studies to obtain deeper insights into Golgiphagy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.553042">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.553042" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.553042">
        <p class="paperTitle">The pore-forming apolipoprotein APOL7C drives phagosomal rupture and antigen cross-presentation by dendritic cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.553042" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.553042" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gonzales, G. A.; Huang, S.; Rajwani, J.; Wilkinson, L.; Nguyen, J. A.; Wood, C. M.; Moore, M.; Cedeno, E.; Sikdar, S.; McKenna, N.; Ebacher, V.; Rosin, N. L.; Carneiro, M. B.; Surewaard, B.; Peters, N. C.; Biernaskie, J.; Mahoney, D. J.; Yates, R. M.; Canton, J.</p>
        <p class="info">Score: 8.2, Published: 2023-08-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.553042' target='https://doi.org/10.1101/2023.08.11.553042'> 10.1101/2023.08.11.553042</a></p>
        <p class="abstract">Type I conventional dendritic cells (cDC1s) are essential for the generation of protective cytotoxic T lymphocyte (CTL) responses against many types of viruses and tumours. They do so by internalizing antigens from virally infected or tumour cells and presenting them to CD8&#43; T cells in a process known as cross-presentation (XP). Despite the obvious biological importance of XP, the molecular mechanism(s) driving this process remain unclear. Here, we show that a cDC-specific pore-forming protein called apolipoprotein 7C (APOL7C) is upregulated in response to innate immune stimuli and is recruited to phagosomes. Strikingly, the association of APOL7C with phagosomes leads to phagosomal rupture, which in turn allows for the escape of engulfed protein antigens to the cytosol where they can be processed via the endogenous major histocompatibility complex (MHC) class I antigen processing pathway. We show that APOL7C recruitment to phagosomes is voltage-dependent and occurs in response to NADPH oxidase-induced depolarization of the phagosomal membrane. Our data indicate the presence of dedicated pore-forming apolipoproteins that mediate the delivery of phagocytosed proteins to the cytosol of activated cDC1s to facilitate MHC class I presentation of exogenous antigen and to regulate adaptive immunity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.551504">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.551504" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.551504">
        <p class="paperTitle">Multi-scale spatial mapping of cell populations across anatomical sites in healthy human skin and basal cell carcinoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.551504" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.551504" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ganier, C.; Mazin, P.; Herrera-Oropeza, G.; Du-Harpur, X.; Blakeley, M.; Gabriel, J.; Predeus, A.; Cakir, B.; Prete, M.; Harun, N.; Darrigrand, J.-F.; Haiser, A.; Wyles, S.; Shaw, T. J.; Teichmann, S. A.; Haniffa, M.; Watt, F. M.; Lynch, M. D.</p>
        <p class="info">Score: 6.1, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.551504' target='https://doi.org/10.1101/2023.08.08.551504'> 10.1101/2023.08.08.551504</a></p>
        <p class="abstract">Our understanding of how human skin cells differ according to anatomical site and tumour formation is limited. To address this we have created a multi-scale spatial atlas of healthy skin and basal cell carcinoma (BCC), incorporating in vivo optical coherence tomography, single cell RNA sequencing, spatial global transcriptional profiling and in situ sequencing. Computational spatial deconvolution and projection revealed the localisation of distinct cell populations to specific tissue contexts. Although cell populations were conserved between healthy anatomical sites and in BCC, mesenchymal cell populations including fibroblasts and pericytes retained signatures of developmental origin. Spatial profiling and in silico lineage tracing support a hair follicle origin for BCC and demonstrate that cancer-associated fibroblasts are an expansion of a POSTN&#43; subpopulation associated with hair follicles in healthy skin. RGS5&#43; pericytes are also expanded in BCC suggesting a role in vascular remodelling. We propose that the identity of mesenchymal cell populations is regulated by signals emanating from adjacent structures and that these signals are repurposed to promote the expansion of skin cancer stroma. The resource we have created is publicly available in an interactive format for the research community.

Significance statementSingle cells RNA sequencing has revolutionised cell biology, enabling high resolution analysis of cell types and states within human tissues. Here, we report a comprehensive spatial atlas of adult human skin across different anatomical sites and basal cell carcinoma (BCC) - the most common form of skin cancer - encompassing in vivo optical coherence tomography, single cell RNA sequencing, global spatial transcriptomic profiling and in situ sequencing. In combination these modalities have allowed us to assemble a comprehensive nuclear-resolution atlas of cellular identity in health and disease.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
