<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cell biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cell biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Unveiling the ultrastructural landscape of native extracellular matrix via lift-out cryo-FIBSEM and cryo-ET
Authors: Zens, B.; Faessler, F.; Hansen, J. M.; Hauschild, R.; Datler, J.; Hodirnau, V.-V.; Zheden, V.; Alanko, J.; Sixt, M.; Schur, F. K.
Score: 43.3, Published: 2023-09-25 DOI: 10.1101/2023.09.25.559261
The extracellular matrix (ECM) is a highly hydrated, three-dimensional network composed of various macromolecules and signaling factors. It serves as a structural scaffold for cells and plays an essential role in the regulation of numerous cellular processes, including cell migration, adhesion, and proliferation.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cell-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cell biology" />
<meta property="og:description" content="Unveiling the ultrastructural landscape of native extracellular matrix via lift-out cryo-FIBSEM and cryo-ET
Authors: Zens, B.; Faessler, F.; Hansen, J. M.; Hauschild, R.; Datler, J.; Hodirnau, V.-V.; Zheden, V.; Alanko, J.; Sixt, M.; Schur, F. K.
Score: 43.3, Published: 2023-09-25 DOI: 10.1101/2023.09.25.559261
The extracellular matrix (ECM) is a highly hydrated, three-dimensional network composed of various macromolecules and signaling factors. It serves as a structural scaffold for cells and plays an essential role in the regulation of numerous cellular processes, including cell migration, adhesion, and proliferation." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cell-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-27T10:39:11+00:00" />
<meta property="article:modified_time" content="2023-09-27T10:39:11+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cell biology"/>
<meta name="twitter:description" content="Unveiling the ultrastructural landscape of native extracellular matrix via lift-out cryo-FIBSEM and cryo-ET
Authors: Zens, B.; Faessler, F.; Hansen, J. M.; Hauschild, R.; Datler, J.; Hodirnau, V.-V.; Zheden, V.; Alanko, J.; Sixt, M.; Schur, F. K.
Score: 43.3, Published: 2023-09-25 DOI: 10.1101/2023.09.25.559261
The extracellular matrix (ECM) is a highly hydrated, three-dimensional network composed of various macromolecules and signaling factors. It serves as a structural scaffold for cells and plays an essential role in the regulation of numerous cellular processes, including cell migration, adhesion, and proliferation."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cell biology",
      "item": "https://trxiv.yorks0n.com/posts/cell-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cell biology",
  "name": "cell biology",
  "description": "Unveiling the ultrastructural landscape of native extracellular matrix via lift-out cryo-FIBSEM and cryo-ET\nAuthors: Zens, B.; Faessler, F.; Hansen, J. M.; Hauschild, R.; Datler, J.; Hodirnau, V.-V.; Zheden, V.; Alanko, J.; Sixt, M.; Schur, F. K.\nScore: 43.3, Published: 2023-09-25 DOI: 10.1101/2023.09.25.559261\nThe extracellular matrix (ECM) is a highly hydrated, three-dimensional network composed of various macromolecules and signaling factors. It serves as a structural scaffold for cells and plays an essential role in the regulation of numerous cellular processes, including cell migration, adhesion, and proliferation.",
  "keywords": [
    
  ],
  "articleBody": " Unveiling the ultrastructural landscape of native extracellular matrix via lift-out cryo-FIBSEM and cryo-ET\nAuthors: Zens, B.; Faessler, F.; Hansen, J. M.; Hauschild, R.; Datler, J.; Hodirnau, V.-V.; Zheden, V.; Alanko, J.; Sixt, M.; Schur, F. K.\nScore: 43.3, Published: 2023-09-25 DOI: 10.1101/2023.09.25.559261\nThe extracellular matrix (ECM) is a highly hydrated, three-dimensional network composed of various macromolecules and signaling factors. It serves as a structural scaffold for cells and plays an essential role in the regulation of numerous cellular processes, including cell migration, adhesion, and proliferation. Despite its importance in metazoans, structural knowledge is rudimentary on how the components of the matrisome are secreted, remodeled, and interact with each other and with surrounding cells. Specifically, the exact molecular assembly of important ECM fibers, such as fibronectin fibrils, fibrillin microfibrils, or Collagen-VI filaments has remained enigmatic. This is largely due to methodological limitations in specimen preparation for conventional room-temperature electron microscopy (EM). To overcome these limitations, we have developed a cell culture-based 3D-ECM platform compatible with sample thinning by cryo-lift out focused ion beam (FIB) milling and cryo-electron tomography (cryo-ET). Our workflow involves the implementation of cell-derived matrices (CDMs) grown on EM grids, resulting in a highly adaptable and versatile tool to closely mimic ECM environments. This allows us to visualize native ECM and its components for the first time in their fully hydrated, cellular context. Our data reveals an intricate network of ECM fibers and their positioning with respect to matrix-secreting cells. In addition to D-spaced collagen fibers, we visualize previously unresolved fibrous structures, and an amorphous matrix co-assembling in proximity to ECM fibers and delineating the boundary between ECM and empty extra-cellular space. Intra- and extracellular granules presumably represent assembly intermediates of the ECM. Our results add to the structural atlas of the ECM and provide novel insights into ECM secretion, assembly and maintenance.\nSequential membrane- and protein-bound organelles compartmentalize genomes during phage infection\nAuthors: Armbruster, E.; Lee, J.; Hutchings, J.; VanderWal, A.; Enustun, E.; Adler, B.; Aindow, A.; Deep, A.; Rodriguez, Z.; Morgan, C.; Ghassemian, M.; Charles, E.; Cress, B.; Savage, D.; Doudna, J.; Pogliano, K.; Corbett, K.; Villa, E.; Pogliano, J.\nScore: 41.1, Published: 2023-09-21 DOI: 10.1101/2023.09.20.558163\nEukaryotic viruses assemble compartments required for genome replication, but no such organelles are known to be essential for prokaryotic viruses. Bacteriophages of the family Chimalliviridae sequester their genomes within a phagegenerated organelle, the phage nucleus, which is enclosed by a lattice of viral protein ChmA. Using the dRfxCas13d-based knockdown system CRISPRi-ART, we show that ChmA is essential for the E. coli phage Goslar life cycle. Without ChmA, infections are arrested at an early stage in which the injected phage genome is enclosed in a membrane-bound vesicle capable of gene expression but not DNA replication. Not only do we demonstrate that the phage nucleus is essential for genome replication, but we also show that the Chimalliviridae early phage infection (EPI) vesicle is a transcriptionally active, phage-generated organelle.\nThe anti-SARS-CoV-2 BNT162b2 vaccine suppresses mithramycin-induced erythroid differentiation and expression of embryo-fetal globin genes in human erythroleukemia K562 cells\nAuthors: Zurlo, M.; Gasparello, J.; Verona, M.; Papi, C.; Cosenza, L. C.; Finotti, A.; Marzaro, G.; Gambari, R.\nScore: 997.2, Published: 2023-09-07 DOI: 10.1101/2023.09.07.556634\nThe COVID-19 severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) the ongoing coronavirus disease 2019 (COVID-19) pandemic. The SARS-CoV-2 Spike protein (S-protein) plays an important role in the early phase of SARS-CoV2 infection through efficient interaction with ACE2. The S-protein is produced by RNA-based COVID-19 vaccines, and has been hypothesized to be responsible for damaging cells of several tissues and for some important side effects of RNA-based COVID-19 vaccines. The aim of this study was to verify the effect of the BNT162b2 vaccine on erythroid differentiation of the human K562 cell line, that has been in the past intensively studied as a model system mimicking some steps of erythropoiesis. We found that the BNT162b2 vaccine suppresses mithramycin-induced erythroid differentiation of K562 cells. Reverse-transcription-PCR and Western blotting assays demonstrated that suppression of erythroid differentiation was associated with sharp inhibition of the expression of -globin and {gamma}-globin mRNA accumulation. Inhibition of accumulation of {zeta}-globin and {varepsilon}-globin mRNAs was also observed. In addition, we provide in silico studies suggesting a direct interaction between SARS-CoV-2 Spike protein and Hb Portland, that is the major hemoglobin produced by K562 cells. This study thus provides information suggesting the need of great attention on possible alteration of hematopoietic parameters following SARS-CoV-2 infection and/or COVID-19 vaccination.\nControl of mitophagy initiation and progression by the TBK1 adaptors NAP1 and SINTBAD\nAuthors: Adriaenssens, E.; Nguyen, T. N.; Sawa-Makarska, J.; Khuu, G.; Schuschnig, M.; Shoebridge, S.; Watts, E. M.; Csalyi, K. D.; Padman, B. S.; Lazarou, M.; Martens, S.\nScore: 24.0, Published: 2023-09-25 DOI: 10.1101/2023.09.25.559255\nMitophagy preserves overall mitochondrial fitness by selectively targeting damaged mitochondria for degradation. The regulatory mechanisms that prevent PINK1/Parkin-dependent mitophagy and other selective autophagy pathways from overreacting while ensuring swift progression once initiated are largely elusive. Here, we demonstrate how the TBK1 adaptors NAP1 and SINTBAD restrict the initiation of OPTN-driven mitophagy by competing with OPTN for TBK1. Conversely, they promote the progression of NDP52-driven mitophagy by recruiting TBK1 to NDP52 and stabilizing its interaction with FIP200. Notably, OPTN emerges as the primary recruiter of TBK1 during mitophagy initiation, which in return boosts NDP52-mediated mitophagy. Our results thus define NAP1 and SINTBAD as cargo receptor rheostats, elevating the threshold for mitophagy initiation by OPTN while promoting the progression of the pathway once set in motion by supporting NDP52. These findings shed light on the cellular strategy to prevent pathway hyperactivity while still ensuring efficient progression.\nMapping the substrate landscape of protein phosphatase 2A catalytic subunit PPP2CA\nAuthors: Brewer, A.; Sathe, G.; Pflug, B. E.; Macartney, T.; Sapkota, G. P.\nScore: 19.3, Published: 2023-09-19 DOI: 10.1101/2023.09.19.558429\nProtein phosphatase 2A (PP2A) is an essential Ser/Thr phosphatase that regulates a plethora of cellular processes. PP2A operates as a holoenzyme complex, comprising one each of the scaffolding (A), regulatory (B) and catalytic (C) subunits. PPP2CA is the principal catalytic subunit of the PP2A holoenzyme complex. Although previous studies have reported many substrates of specific PP2A holoenzyme complexes, the full scope of PP2A substrates in cells remains to be defined. To address this, we generated HEK293 cells in which PPP2CA was homozygously knocked in with a dTAG, allowing for efficient and selective degradation of dTAG-PPP2CA with proteolysis-targeting chimeras (PROTACs) targeting the dTAG. By employing an unbiased global phospho-proteomic analysis, we identified 6,280 phospho-peptides corresponding to 2,204 proteins that showed a significant increase in abundance upon dTAG-PPP2CA degradation, implicating them as potential PPP2CA substrates. Among these, some were established PP2A substrates, while most were novel. Bioinformatic analyses revealed the involvement of the identified potential PPP2CA substrates in many cellular processes, including spliceosome function, the cell cycle, RNA transport and ubiquitin-mediated proteolysis. We show that a pSP/pTP motif is a predominant target for PPP2CA. We confirmed some of our phospho-proteomic data with immunoblotting, by utilising commercially available phospho-specific antibodies. We provide an in-depth atlas of potential PPP2CA substrates and establish targeted degradation as a robust tool to unveil phosphatase substrates in cells.\nRewiring of the phosphoproteome executes two meiotic divisions\nAuthors: Koch, L. B.; Spanos, C.; Kelly, V.; Ly, T.; Marston, A. L.\nScore: 13.5, Published: 2023-09-22 DOI: 10.1101/2023.09.22.559006\nThe cell cycle is ordered by a controlled network of kinases and phosphatases. To generate gametes via meiosis, two distinct and sequential chromosome segregation events occur without an intervening S phase. How canonical cell cycle controls are modified for meiosis is not well understood. Here, using highly synchronous budding yeast populations, we reveal how the global proteome and phosphoproteome changes during the meiotic divisions. While protein abundance changes are limited to key cell cycle regulators, dynamic phosphorylation changes are pervasive. Our data indicate that two waves of cyclin-dependent kinase and Polo (Cdc5Polo) kinase activity drive successive meiotic divisions. These two distinct waves of phosphorylation are ensured by the meiosis-specific Spo13Meikin protein, which rewires the phosphoproteome. Spo13Meikin binds to Cdc5Polo to promote phosphorylation of a subset of substrates in meiosis I containing a newly identified motif, which we define as the Spo13Meikin -Cdc5Polo consensus phosphorylation motif. Overall, our findings reveal that a master regulator of meiosis redirects the activity of a kinase to change the phosphorylation landscape and elicit a developmental cascade.\nOrganelle landscape analysis using a multi-parametric particle-based method\nAuthors: Kurikawa, Y.; Koyama-Honda, I.; Igarashi, K.; Tamura, N.; Koike, S.; Mizushima, N.\nScore: 10.5, Published: 2023-09-26 DOI: 10.1101/2023.09.25.559448\nOrganelles have unique structures and molecular compositions for their functions and have been classified accordingly. However, many organelles are heterogeneous and in the process of maturation and differentiation. Because traditional methods have a limited number of parameters and spatial resolution, they struggle to capture the heterogeneous landscapes of organelles. Here, we present a method for multi-parametric particle-based analysis of organelles. After disrupting cells, fluorescence microscopy images of organelle particles labeled with six to eight different organelle markers were obtained, and their multi-dimensional data were represented in intuitive two-dimensional UMAP (uniform manifold approximation and projection) spaces. This method enabled visualization of landscapes of seven major organelles as well as the transitional states of endocytic organelles directed to the recycling and degradation pathways. Furthermore, endoplasmic reticulum-mitochondria contact sites were detected in these maps. Our proposed method successfully detects a wide array of organelles simultaneously, enabling the analysis of heterogeneous organelle landscapes.\nA hematopoietic stem cell subset that retains memory of prior inflammatory stress accumulates in aging and clonal hematopoiesis\nAuthors: Zeng, A. G. X.; Nagree, M. S.; Jakobsen, N. A.; Shah, S.; Murison, A.; Cheong, J.-G.; Lim, I.; Jin, L.; Aguilar-Navarro, A. G.; Araujo, J.; Parris, D.; McLeod, J.; Kim, H.; Lee, H.; Zhang, L.; Boulanger, M.; Wagenblast, E.; Flores-Figueroa, E.; Wang, B.; Schwartz, G. W.; Josefowicz, S. Z.; Vyas, P.; Dick, J. E.; Xie, S. Z.\nScore: 30.5, Published: 2023-09-13 DOI: 10.1101/2023.09.11.557271\nHematopoietic stem cells (HSC) must persist through a lifetime of infections to ensure life-long blood production, but whether molecular adaptations following inflammatory stress recovery in HSC are linked to aging and clonal hematopoiesis (CH) are unclear. Here, we performed single cell (sc) Multiomics on human HSC isolated from a xenograft inflammation-recovery model. Two transcriptionally and epigenetically distinct HSC subsets expressing canonical HSC programs were identified. Only one showed scATACseq and scRNAseq changes after recovery from TNF or lipopolysaccharide treatment. This inflammatory memory HSC (HSC-iM) program is enriched in memory T cells and HSC from recovered COVID-19 patients. Importantly, HSC-iM accumulates with age and with CH. Overall, a human HSC subset that retains memory of prior inflammatory stress has implications towards HSC fitness and leukemia transformation.\nCytogenetic resource enables mechanistic resolution of changing trends in human pluripotent stem cell aberrations linked to feeder-free culture\nAuthors: Stavish, D.; Price, C. J.; Gelezauskaite, G.; Leonhard, K. A.; Taapken, S. M.; McIntire, E. M.; Laing, O.; James, B. M.; Riley, J. J.; Zerbib, J.; Baker, D.; Harding, A. L.; Jestice, L. H.; Eleveld, T. F.; Gillis, A. J. M.; Hillenius, S.; Looijenga, L. H. J.; Gokhale, P. J.; Ben-David, U. J.; Ludwig, T.; Barbaric, I.\nScore: 7.3, Published: 2023-09-21 DOI: 10.1101/2023.09.21.558777\nSince the first derivation of human pluripotent stem cells (hPSCs), the number of culture conditions has steadily increased, making hPSC culture more facile. Nonetheless, there remains the persistent issue of culture-acquired genetic changes, hampering the reproducibility of hPSC research and jeopardising their clinical use. Here, we utilised comprehensive karyotyping datasets from over 20,000 hPSC cultures sampled under different conditions to ascertain association of genetic changes with specific culture regimens. We found condition-dependent patterns of aberrations, with higher prevalence of chromosome 1q gains in recent years, associated with increased use of contemporary, feeder-free cultures. Mechanistically, we show the context-dependent selection of 1q variants is mainly driven by MDM4, a gene amplified in many cancers, located on chromosome 1q. To facilitate reproducibility of hPSC research and their safe clinical utility, we provide a unique hPSC karyotype resource for informing the risk assessment of genetic aberrations and developing strategies to suppress their occurrence.\nThe sterol transporter STARD3 transports sphingosine at ER-lysosome contact sites\nAuthors: Hempelmann, P.; Lolicato, F.; Graziadei, A.; Brown, R. D. R.; Spiegel, S.; Rappsilber, J.; Nickel, W.; Höglinger, D.; Jamecna, D.\nScore: 12.0, Published: 2023-09-18 DOI: 10.1101/2023.09.18.557036\nSphingolipids are important structural components of membranes. Additionally, simple sphingolipids such as sphingosine are highly bioactive and participate in complex subcellular signaling. Sphingolipid deregulation is associated with many severe diseases including diabetes, Parkinsons and cancer. Here, we focus on how sphingosine, generated from sphingolipid catabolism in late endosomes/lysosomes, is reintegrated into the biosynthetic machinery at the endoplasmic reticulum (ER). We characterized the sterol transporter STARD3 as a sphingosine transporter acting at lysosome-ER contact sites. Experiments featuring crosslinkable sphingosine probes, supported by unbiased molecular dynamics simulations, exposed how sphingosine binds to the lipid-binding domain of STARD3. Following the metabolic fate of pre-localized lysosomal sphingosine showed the importance of STARD3 and its actions at contact sites for the integration of sphingosine into ceramide in a cellular context. Our findings provide the first example of inter-organellar sphingosine transfer and pave the way for a better understanding of sphingolipid - sterol co-regulation.\n",
  "wordCount" : "2173",
  "inLanguage": "en",
  "datePublished": "2023-09-27T10:39:11Z",
  "dateModified": "2023-09-27T10:39:11Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cell-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cell biology
    </h1>
    <div class="post-meta"><span>updated on September 27, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559261">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559261" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559261">
        <p class="paperTitle">Unveiling the ultrastructural landscape of native extracellular matrix via lift-out cryo-FIBSEM and cryo-ET</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559261" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559261" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zens, B.; Faessler, F.; Hansen, J. M.; Hauschild, R.; Datler, J.; Hodirnau, V.-V.; Zheden, V.; Alanko, J.; Sixt, M.; Schur, F. K.</p>
        <p class="info">Score: 43.3, Published: 2023-09-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559261' target='https://doi.org/10.1101/2023.09.25.559261'> 10.1101/2023.09.25.559261</a></p>
        <p class="abstract">The extracellular matrix (ECM) is a highly hydrated, three-dimensional network composed of various macromolecules and signaling factors. It serves as a structural scaffold for cells and plays an essential role in the regulation of numerous cellular processes, including cell migration, adhesion, and proliferation. Despite its importance in metazoans, structural knowledge is rudimentary on how the components of the matrisome are secreted, remodeled, and interact with each other and with surrounding cells. Specifically, the exact molecular assembly of important ECM fibers, such as fibronectin fibrils, fibrillin microfibrils, or Collagen-VI filaments has remained enigmatic. This is largely due to methodological limitations in specimen preparation for conventional room-temperature electron microscopy (EM). To overcome these limitations, we have developed a cell culture-based 3D-ECM platform compatible with sample thinning by cryo-lift out focused ion beam (FIB) milling and cryo-electron tomography (cryo-ET). Our workflow involves the implementation of cell-derived matrices (CDMs) grown on EM grids, resulting in a highly adaptable and versatile tool to closely mimic ECM environments. This allows us to visualize native ECM and its components for the first time in their fully hydrated, cellular context. Our data reveals an intricate network of ECM fibers and their positioning with respect to matrix-secreting cells. In addition to D-spaced collagen fibers, we visualize previously unresolved fibrous structures, and an amorphous matrix co-assembling in proximity to ECM fibers and delineating the boundary between ECM and empty extra-cellular space. Intra- and extracellular granules presumably represent assembly intermediates of the ECM. Our results add to the structural atlas of the ECM and provide novel insights into ECM secretion, assembly and maintenance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.558163">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.558163" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.558163">
        <p class="paperTitle">Sequential membrane- and protein-bound organelles compartmentalize genomes during phage infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.558163" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.558163" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Armbruster, E.; Lee, J.; Hutchings, J.; VanderWal, A.; Enustun, E.; Adler, B.; Aindow, A.; Deep, A.; Rodriguez, Z.; Morgan, C.; Ghassemian, M.; Charles, E.; Cress, B.; Savage, D.; Doudna, J.; Pogliano, K.; Corbett, K.; Villa, E.; Pogliano, J.</p>
        <p class="info">Score: 41.1, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.558163' target='https://doi.org/10.1101/2023.09.20.558163'> 10.1101/2023.09.20.558163</a></p>
        <p class="abstract">Eukaryotic viruses assemble compartments required for genome replication, but no such organelles are known to be essential for prokaryotic viruses. Bacteriophages of the family Chimalliviridae sequester their genomes within a phagegenerated organelle, the phage nucleus, which is enclosed by a lattice of viral protein ChmA. Using the dRfxCas13d-based knockdown system CRISPRi-ART, we show that ChmA is essential for the E. coli phage Goslar life cycle. Without ChmA, infections are arrested at an early stage in which the injected phage genome is enclosed in a membrane-bound vesicle capable of gene expression but not DNA replication. Not only do we demonstrate that the phage nucleus is essential for genome replication, but we also show that the Chimalliviridae early phage infection (EPI) vesicle is a transcriptionally active, phage-generated organelle.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556634">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556634" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556634">
        <p class="paperTitle">The anti-SARS-CoV-2 BNT162b2 vaccine suppresses mithramycin-induced erythroid differentiation and expression of embryo-fetal globin genes in human erythroleukemia K562 cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556634" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556634" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zurlo, M.; Gasparello, J.; Verona, M.; Papi, C.; Cosenza, L. C.; Finotti, A.; Marzaro, G.; Gambari, R.</p>
        <p class="info">Score: 997.2, Published: 2023-09-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556634' target='https://doi.org/10.1101/2023.09.07.556634'> 10.1101/2023.09.07.556634</a></p>
        <p class="abstract">The COVID-19 severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) the ongoing coronavirus disease 2019 (COVID-19) pandemic. The SARS-CoV-2 Spike protein (S-protein) plays an important role in the early phase of SARS-CoV2 infection through efficient interaction with ACE2. The S-protein is produced by RNA-based COVID-19 vaccines, and has been hypothesized to be responsible for damaging cells of several tissues and for some important side effects of RNA-based COVID-19 vaccines. The aim of this study was to verify the effect of the BNT162b2 vaccine on erythroid differentiation of the human K562 cell line, that has been in the past intensively studied as a model system mimicking some steps of erythropoiesis. We found that the BNT162b2 vaccine suppresses mithramycin-induced erythroid differentiation of K562 cells. Reverse-transcription-PCR and Western blotting assays demonstrated that suppression of erythroid differentiation was associated with sharp inhibition of the expression of -globin and {gamma}-globin mRNA accumulation. Inhibition of accumulation of {zeta}-globin and {varepsilon}-globin mRNAs was also observed. In addition, we provide in silico studies suggesting a direct interaction between SARS-CoV-2 Spike protein and Hb Portland, that is the major hemoglobin produced by K562 cells. This study thus provides information suggesting the need of great attention on possible alteration of hematopoietic parameters following SARS-CoV-2 infection and/or COVID-19 vaccination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559255">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559255" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559255">
        <p class="paperTitle">Control of mitophagy initiation and progression by the TBK1 adaptors NAP1 and SINTBAD</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559255" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559255" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Adriaenssens, E.; Nguyen, T. N.; Sawa-Makarska, J.; Khuu, G.; Schuschnig, M.; Shoebridge, S.; Watts, E. M.; Csalyi, K. D.; Padman, B. S.; Lazarou, M.; Martens, S.</p>
        <p class="info">Score: 24.0, Published: 2023-09-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559255' target='https://doi.org/10.1101/2023.09.25.559255'> 10.1101/2023.09.25.559255</a></p>
        <p class="abstract">Mitophagy preserves overall mitochondrial fitness by selectively targeting damaged mitochondria for degradation. The regulatory mechanisms that prevent PINK1/Parkin-dependent mitophagy and other selective autophagy pathways from overreacting while ensuring swift progression once initiated are largely elusive. Here, we demonstrate how the TBK1 adaptors NAP1 and SINTBAD restrict the initiation of OPTN-driven mitophagy by competing with OPTN for TBK1. Conversely, they promote the progression of NDP52-driven mitophagy by recruiting TBK1 to NDP52 and stabilizing its interaction with FIP200. Notably, OPTN emerges as the primary recruiter of TBK1 during mitophagy initiation, which in return boosts NDP52-mediated mitophagy. Our results thus define NAP1 and SINTBAD as cargo receptor rheostats, elevating the threshold for mitophagy initiation by OPTN while promoting the progression of the pathway once set in motion by supporting NDP52. These findings shed light on the cellular strategy to prevent pathway hyperactivity while still ensuring efficient progression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.19.558429">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.19.558429" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.19.558429">
        <p class="paperTitle">Mapping the substrate landscape of protein phosphatase 2A catalytic subunit PPP2CA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.19.558429" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.19.558429" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brewer, A.; Sathe, G.; Pflug, B. E.; Macartney, T.; Sapkota, G. P.</p>
        <p class="info">Score: 19.3, Published: 2023-09-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.19.558429' target='https://doi.org/10.1101/2023.09.19.558429'> 10.1101/2023.09.19.558429</a></p>
        <p class="abstract">Protein phosphatase 2A (PP2A) is an essential Ser/Thr phosphatase that regulates a plethora of cellular processes. PP2A operates as a holoenzyme complex, comprising one each of the scaffolding (A), regulatory (B) and catalytic (C) subunits. PPP2CA is the principal catalytic subunit of the PP2A holoenzyme complex. Although previous studies have reported many substrates of specific PP2A holoenzyme complexes, the full scope of PP2A substrates in cells remains to be defined. To address this, we generated HEK293 cells in which PPP2CA was homozygously knocked in with a dTAG, allowing for efficient and selective degradation of dTAG-PPP2CA with proteolysis-targeting chimeras (PROTACs) targeting the dTAG. By employing an unbiased global phospho-proteomic analysis, we identified 6,280 phospho-peptides corresponding to 2,204 proteins that showed a significant increase in abundance upon dTAG-PPP2CA degradation, implicating them as potential PPP2CA substrates. Among these, some were established PP2A substrates, while most were novel. Bioinformatic analyses revealed the involvement of the identified potential PPP2CA substrates in many cellular processes, including spliceosome function, the cell cycle, RNA transport and ubiquitin-mediated proteolysis. We show that a pSP/pTP motif is a predominant target for PPP2CA. We confirmed some of our phospho-proteomic data with immunoblotting, by utilising commercially available phospho-specific antibodies. We provide an in-depth atlas of potential PPP2CA substrates and establish targeted degradation as a robust tool to unveil phosphatase substrates in cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.559006">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.559006" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.559006">
        <p class="paperTitle">Rewiring of the phosphoproteome executes two meiotic divisions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.559006" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.559006" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Koch, L. B.; Spanos, C.; Kelly, V.; Ly, T.; Marston, A. L.</p>
        <p class="info">Score: 13.5, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.559006' target='https://doi.org/10.1101/2023.09.22.559006'> 10.1101/2023.09.22.559006</a></p>
        <p class="abstract">The cell cycle is ordered by a controlled network of kinases and phosphatases. To generate gametes via meiosis, two distinct and sequential chromosome segregation events occur without an intervening S phase. How canonical cell cycle controls are modified for meiosis is not well understood. Here, using highly synchronous budding yeast populations, we reveal how the global proteome and phosphoproteome changes during the meiotic divisions. While protein abundance changes are limited to key cell cycle regulators, dynamic phosphorylation changes are pervasive. Our data indicate that two waves of cyclin-dependent kinase and Polo (Cdc5Polo) kinase activity drive successive meiotic divisions. These two distinct waves of phosphorylation are ensured by the meiosis-specific Spo13Meikin protein, which rewires the phosphoproteome. Spo13Meikin binds to Cdc5Polo to promote phosphorylation of a subset of substrates in meiosis I containing a newly identified motif, which we define as the Spo13Meikin -Cdc5Polo consensus phosphorylation motif. Overall, our findings reveal that a master regulator of meiosis redirects the activity of a kinase to change the phosphorylation landscape and elicit a developmental cascade.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559448">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559448" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559448">
        <p class="paperTitle">Organelle landscape analysis using a multi-parametric particle-based method</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559448" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559448" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kurikawa, Y.; Koyama-Honda, I.; Igarashi, K.; Tamura, N.; Koike, S.; Mizushima, N.</p>
        <p class="info">Score: 10.5, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559448' target='https://doi.org/10.1101/2023.09.25.559448'> 10.1101/2023.09.25.559448</a></p>
        <p class="abstract">Organelles have unique structures and molecular compositions for their functions and have been classified accordingly. However, many organelles are heterogeneous and in the process of maturation and differentiation. Because traditional methods have a limited number of parameters and spatial resolution, they struggle to capture the heterogeneous landscapes of organelles. Here, we present a method for multi-parametric particle-based analysis of organelles. After disrupting cells, fluorescence microscopy images of organelle particles labeled with six to eight different organelle markers were obtained, and their multi-dimensional data were represented in intuitive two-dimensional UMAP (uniform manifold approximation and projection) spaces. This method enabled visualization of landscapes of seven major organelles as well as the transitional states of endocytic organelles directed to the recycling and degradation pathways. Furthermore, endoplasmic reticulum-mitochondria contact sites were detected in these maps. Our proposed method successfully detects a wide array of organelles simultaneously, enabling the analysis of heterogeneous organelle landscapes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557271">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557271" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557271">
        <p class="paperTitle">A hematopoietic stem cell subset that retains memory of prior inflammatory stress accumulates in aging and clonal hematopoiesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557271" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557271" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zeng, A. G. X.; Nagree, M. S.; Jakobsen, N. A.; Shah, S.; Murison, A.; Cheong, J.-G.; Lim, I.; Jin, L.; Aguilar-Navarro, A. G.; Araujo, J.; Parris, D.; McLeod, J.; Kim, H.; Lee, H.; Zhang, L.; Boulanger, M.; Wagenblast, E.; Flores-Figueroa, E.; Wang, B.; Schwartz, G. W.; Josefowicz, S. Z.; Vyas, P.; Dick, J. E.; Xie, S. Z.</p>
        <p class="info">Score: 30.5, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557271' target='https://doi.org/10.1101/2023.09.11.557271'> 10.1101/2023.09.11.557271</a></p>
        <p class="abstract">Hematopoietic stem cells (HSC) must persist through a lifetime of infections to ensure life-long blood production, but whether molecular adaptations following inflammatory stress recovery in HSC are linked to aging and clonal hematopoiesis (CH) are unclear. Here, we performed single cell (sc) Multiomics on human HSC isolated from a xenograft inflammation-recovery model. Two transcriptionally and epigenetically distinct HSC subsets expressing canonical HSC programs were identified. Only one showed scATACseq and scRNAseq changes after recovery from TNF or lipopolysaccharide treatment. This inflammatory memory HSC (HSC-iM) program is enriched in memory T cells and HSC from recovered COVID-19 patients. Importantly, HSC-iM accumulates with age and with CH. Overall, a human HSC subset that retains memory of prior inflammatory stress has implications towards HSC fitness and leukemia transformation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558777">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558777" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558777">
        <p class="paperTitle">Cytogenetic resource enables mechanistic resolution of changing trends in human pluripotent stem cell aberrations linked to feeder-free culture</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558777" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558777" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stavish, D.; Price, C. J.; Gelezauskaite, G.; Leonhard, K. A.; Taapken, S. M.; McIntire, E. M.; Laing, O.; James, B. M.; Riley, J. J.; Zerbib, J.; Baker, D.; Harding, A. L.; Jestice, L. H.; Eleveld, T. F.; Gillis, A. J. M.; Hillenius, S.; Looijenga, L. H. J.; Gokhale, P. J.; Ben-David, U. J.; Ludwig, T.; Barbaric, I.</p>
        <p class="info">Score: 7.3, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558777' target='https://doi.org/10.1101/2023.09.21.558777'> 10.1101/2023.09.21.558777</a></p>
        <p class="abstract">Since the first derivation of human pluripotent stem cells (hPSCs), the number of culture conditions has steadily increased, making hPSC culture more facile. Nonetheless, there remains the persistent issue of culture-acquired genetic changes, hampering the reproducibility of hPSC research and jeopardising their clinical use. Here, we utilised comprehensive karyotyping datasets from over 20,000 hPSC cultures sampled under different conditions to ascertain association of genetic changes with specific culture regimens. We found condition-dependent patterns of aberrations, with higher prevalence of chromosome 1q gains in recent years, associated with increased use of contemporary, feeder-free cultures. Mechanistically, we show the context-dependent selection of 1q variants is mainly driven by MDM4, a gene amplified in many cancers, located on chromosome 1q. To facilitate reproducibility of hPSC research and their safe clinical utility, we provide a unique hPSC karyotype resource for informing the risk assessment of genetic aberrations and developing strategies to suppress their occurrence.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.557036">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.557036" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.557036">
        <p class="paperTitle">The sterol transporter STARD3 transports sphingosine at ER-lysosome contact sites</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.557036" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.557036" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hempelmann, P.; Lolicato, F.; Graziadei, A.; Brown, R. D. R.; Spiegel, S.; Rappsilber, J.; Nickel, W.; Höglinger, D.; Jamecna, D.</p>
        <p class="info">Score: 12.0, Published: 2023-09-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.557036' target='https://doi.org/10.1101/2023.09.18.557036'> 10.1101/2023.09.18.557036</a></p>
        <p class="abstract">Sphingolipids are important structural components of membranes. Additionally, simple sphingolipids such as sphingosine are highly bioactive and participate in complex subcellular signaling. Sphingolipid deregulation is associated with many severe diseases including diabetes, Parkinsons and cancer. Here, we focus on how sphingosine, generated from sphingolipid catabolism in late endosomes/lysosomes, is reintegrated into the biosynthetic machinery at the endoplasmic reticulum (ER). We characterized the sterol transporter STARD3 as a sphingosine transporter acting at lysosome-ER contact sites. Experiments featuring crosslinkable sphingosine probes, supported by unbiased molecular dynamics simulations, exposed how sphingosine binds to the lipid-binding domain of STARD3. Following the metabolic fate of pre-localized lysosomal sphingosine showed the importance of STARD3 and its actions at contact sites for the integration of sphingosine into ceramide in a cellular context. Our findings provide the first example of inter-organellar sphingosine transfer and pave the way for a better understanding of sphingolipid - sterol co-regulation.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
