<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cell biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cell biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A centromere-derived retroelement RNA localizes in cis and is a core element of the transcriptional landscape of Drosophila centromeres
Authors: Santinello, B.; Sun, R.; Amjad, A.; Hoyt, S. J.; Ouyang, L.; Courret, C.; Drennan, R.; Leo, L.; Larracuente, A. M.; Core, L. M.; O&#39;Neill, R. J.; Mellone, B. G.
Score: 25.2, Published: 2024-01-16 DOI: 10.1101/2024.01.14.574223
Centromeres are essential chromosomal landmarks that dictate the point of attachment between chromosomes and spindle microtubules during cell division.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cell-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cell biology" />
<meta property="og:description" content="A centromere-derived retroelement RNA localizes in cis and is a core element of the transcriptional landscape of Drosophila centromeres
Authors: Santinello, B.; Sun, R.; Amjad, A.; Hoyt, S. J.; Ouyang, L.; Courret, C.; Drennan, R.; Leo, L.; Larracuente, A. M.; Core, L. M.; O&#39;Neill, R. J.; Mellone, B. G.
Score: 25.2, Published: 2024-01-16 DOI: 10.1101/2024.01.14.574223
Centromeres are essential chromosomal landmarks that dictate the point of attachment between chromosomes and spindle microtubules during cell division." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cell-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-21T10:39:08+00:00" />
<meta property="article:modified_time" content="2024-01-21T10:39:08+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cell biology"/>
<meta name="twitter:description" content="A centromere-derived retroelement RNA localizes in cis and is a core element of the transcriptional landscape of Drosophila centromeres
Authors: Santinello, B.; Sun, R.; Amjad, A.; Hoyt, S. J.; Ouyang, L.; Courret, C.; Drennan, R.; Leo, L.; Larracuente, A. M.; Core, L. M.; O&#39;Neill, R. J.; Mellone, B. G.
Score: 25.2, Published: 2024-01-16 DOI: 10.1101/2024.01.14.574223
Centromeres are essential chromosomal landmarks that dictate the point of attachment between chromosomes and spindle microtubules during cell division."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cell biology",
      "item": "https://trxiv.yorks0n.com/posts/cell-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cell biology",
  "name": "cell biology",
  "description": "A centromere-derived retroelement RNA localizes in cis and is a core element of the transcriptional landscape of Drosophila centromeres\nAuthors: Santinello, B.; Sun, R.; Amjad, A.; Hoyt, S. J.; Ouyang, L.; Courret, C.; Drennan, R.; Leo, L.; Larracuente, A. M.; Core, L. M.; O\u0026#39;Neill, R. J.; Mellone, B. G.\nScore: 25.2, Published: 2024-01-16 DOI: 10.1101/2024.01.14.574223\nCentromeres are essential chromosomal landmarks that dictate the point of attachment between chromosomes and spindle microtubules during cell division.",
  "keywords": [
    
  ],
  "articleBody": " A centromere-derived retroelement RNA localizes in cis and is a core element of the transcriptional landscape of Drosophila centromeres\nAuthors: Santinello, B.; Sun, R.; Amjad, A.; Hoyt, S. J.; Ouyang, L.; Courret, C.; Drennan, R.; Leo, L.; Larracuente, A. M.; Core, L. M.; O'Neill, R. J.; Mellone, B. G.\nScore: 25.2, Published: 2024-01-16 DOI: 10.1101/2024.01.14.574223\nCentromeres are essential chromosomal landmarks that dictate the point of attachment between chromosomes and spindle microtubules during cell division. The stable transmission of the centromere site through generations is ensured by a unique chromatin containing the histone H3 variant CENP-A. Previous studies have highlighted the impact of transcription on promoting CENP-A deposition. However, the specific sequences undergoing this transcription and their contribution to centromere function in metazoan systems remain elusive. In this study, we unveil the centromeric transcriptional landscape and explore its correlation with CENP-A in D. melanogaster, currently the only in vivo model with assembled centromeres. We find that the centromere-enriched retroelement G2/Jockey-3 (hereafter referred to as Jockey-3) is a major driver of centromere transcription, producing RNAs that localize to all mitotic centromeres, with the Y centromere showing the most transcription. Taking advantage of the polymorphism of Jockey-3, we show that these RNAs remain associated with their cognate DNA sequences in cis. Using a LacI/lacO system to generate de novo centromeres, we find that Jockey-3 transcripts do not localize to ectopic sites, suggesting they are unlikely to function as non-coding RNAs with a structural role at centromeres. At de novo centromeres on the lacO array, the presence of CENP-A augments the detection of exogenous lacO-derived transcripts specifically in metaphase. We propose that Jockey-3 contributes to the epigenetic maintenance of the centromere by promoting chromatin transcription, while inserting in a region that permits its continuous transmission. Given the conservation of retroelements as centromere components across taxa, our findings have broad implications in understanding this widespread association.\nHigh-Throughput Identification of Calcium Regulated Proteins Across Diverse Proteomes\nAuthors: Locke, T. M.; Fields, R.; Gizinski, H.; Otto, G. M.; Berg, M. D.; Villen, J.; Sancak, Y.; Schweppe, D. K.\nScore: 21.1, Published: 2024-01-20 DOI: 10.1101/2024.01.18.575273\nCalcium ions play important roles in nearly every biological process, yet whole-proteome analysis of calcium effectors has been hindered by lack of high-throughput, unbiased, and quantitative methods to identify proteins-calcium engagement. To address this, we adapted protein thermostability assays in the budding yeast, human cells, and mouse mitochondria. Based on calcium-dependent thermostability, we identified 2884 putative calcium-regulated proteins across human, mouse, and yeast proteomes. These data revealed calcium engagement of novel signaling hubs and cellular processes, including metabolic enzymes and the spliceosome. Cross-species comparison of calcium-protein engagement and mutagenesis experiments identified residue-specific cation engagement, even within well-known EF-hand domains. Additionally, we found that the dienoyl-CoA reductase DECR1 binds calcium at physiologically-relevant concentrations with substrate-specific affinity, suggesting direct calcium regulation of mitochondrial fatty acid oxidation. These unbiased, proteomic analyses of calcium effectors establish a key resource to dissect cation engagement and its mechanistic effects across multiple species and diverse biological processes.\nNuclear pores as conduits for fluid flow during osmotic stress\nAuthors: Hoffmann, P. C.; Kim, H.; Obarska-Kosinska, A.; Kreysing, J. P.; Andino-Frydman, E.; Cruz-Leon, S.; Cernikova, L.; Kosinski, J.; Turonova, B.; Hummer, G.; Beck, M.\nScore: 20.3, Published: 2024-01-17 DOI: 10.1101/2024.01.17.575985\nChanging environmental conditions necessitate an immediate cellular adaptation to ensure survival. Dictyostelium discoideum, a bacteriovore slime mold present in the soil of most terrestrial ecosystems, is known for its ability to tolerate drastic changes in osmolarity. How the cells cope with the resulting mechanical stress remains understudied. Here we show that D. discoideum has extraordinarily elaborate and resilient nuclear pores that serve as conduits for massive fluid exchange between cytosol and nucleus. We capitalize on the unique properties of D. discoideum cells to quantify flow across the nuclear envelope that is necessitated by changing nuclear size in response to osmotic stress. Based on mathematical concepts adapted from hydrodynamics, we conceptualize this phenomenon as porous flow across nuclear pores. This type of fluid flow is distinct from the canonically characterized modes of nucleocytoplasmic transport, i.e. passive diffusion and active nuclear transport, because of its dependence on pressure. Our insights are relevant in any biological condition that necessitates rapid nuclear size changes, which includes metastasizing cancer cells squeezing through constrictions, migrating cells and differentiating tissues.\nStimulation of NCAM1-14.3.3.ζδ-derived Peptide Interaction Fuels Angiogenesis and Osteogenesis in Ageing\nAuthors: Kusumbe, A.; Yesin, T.; Liu, H.; Ding, Z.; Singh, A.; Zhang, Y.; Chen, J.\nScore: 13.4, Published: 2024-01-17 DOI: 10.1101/2024.01.16.575939\nThe skeletal structure and bone marrow endothelium collectively form a critical functional unit essential for bone development, health, and aging. At the core of osteogenesis and bone formation lies the dynamic process of angiogenesis. In this study, we reveal a potent new endogenous anabolic NCAM1-14.3.3.{zeta}{delta}-derived-Peptide interaction, which stimulates bone angiogenesis and osteogenesis during homeostasis, aging, and age-related bone diseases. Employing high-resolution imaging and inducible cell-specific mouse genetics, our results elucidate the pivotal role of the NCAM1-14.3.3.{zeta}{delta}-derived-Peptide interaction in driving the expansion of Clec14a+ angiogenic endothelial cells. Notably, Clec14a+ endothelial cells express key osteogenic factors. The NCAM1-14.3.3.{zeta}{delta}-derived-Peptide interaction in osteoblasts drives osteoblast differentiation, ultimately contributing to the genesis of new bone. Moreover, the NCAM1-14.3.3.{zeta}{delta}-derived-Peptide interaction leads to a reduction in bone resorption. In age-associated vascular and bone loss diseases, stimulating the NCAM1-14.3.3.{zeta}{delta}-derived-Peptide interaction not only promotes angiogenesis but also reverses bone loss. Consequently, harnessing the endogenous anabolic potential of the NCAM1-14.3.3.{zeta}{delta}-derived-Peptide interaction emerges as a promising therapeutic modality for managing age-related bone diseases.\nStructural basis of respiratory complexes adaptation to cold temperatures\nAuthors: Shin, Y.-C.; Latorre-Muro, P.; Djurabekova, A.; Zdorevskyi, O.; Bennett, C.; Burger, N.; Song, K.; Xu, C.; Sharma, V.; Liao, M.; Puigserver, P.\nScore: 13.4, Published: 2024-01-17 DOI: 10.1101/2024.01.16.575914\nIn response to cold, mammals activate brown fat for respiratory-dependent thermogenesis reliant on the electron transport chain (1, 2). Yet, the structural basis of respiratory complex adaptation to cold remains elusive. Herein we combined thermoregulatory physiology and cryo-EM to study endogenous respiratory supercomplexes exposed to different temperatures. A cold-induced conformation of CI:III2 (termed type 2) was identified with a [~]25{degrees} rotation of CIII2 around its inter-dimer axis, shortening inter-complex Q exchange space, and exhibiting different catalytic states which favor electron transfer. Large-scale supercomplex simulations in lipid membrane reveal how unique lipid-protein arrangements stabilize type 2 complexes to enhance catalytic activity. Together, our cryo-EM studies, multiscale simulations and biochemical analyses unveil the mechanisms and dynamics of respiratory adaptation at the structural and energetic level.\nMeasuring and modeling the dynamics of mitotic error correction\nAuthors: Ha, G.; Dieterle, P.; Shen, H.; Amir, A.; Needleman, D. J.\nScore: 13.0, Published: 2024-01-11 DOI: 10.1101/2024.01.10.574250\nWhen functioning properly, the mitotic spindle segregates an equal number of chromosomes to daughter cells with high fidelity. Over the course of spindle assembly, many initially erroneous attachments between kinetochores and microtubules are fixed through a process called error correction. Despite the importance of chromosome segregation errors in cancer and other diseases, there is a lack of methods to characterize the dynamics of error correction and how it can go wrong. Here, we present an experimental method and analysis framework to quantify chromosome segregation error correction in human tissue culture cells with live cell confocal imaging, timed premature anaphase, and automated counting of kinetochores after cell division. We find that errors decrease exponentially over time during spindle assembly. A coarse-grained model, in which errors are corrected in a chromosome autonomous manner at a constant rate, can quantitatively explain both the measured error correction dynamics and the distribution of anaphase onset times. We further validated our model using perturbations that destabilized microtubules and changed the initial configuration of chromosomal attachments. Taken together, this work provides a quantitative framework for understanding the dynamics of mitotic error correction.\nThe PCM scaffold enables RNA localization to centrosomes\nAuthors: Fang, J.; Tian, W.; Quintanilla, M. A.; Beach, J. R.; Lerit, D. A.\nScore: 9.5, Published: 2024-01-14 DOI: 10.1101/2024.01.13.575509\nAs microtubule-organizing centers, centrosomes direct assembly of the bipolar mitotic spindle required for chromosome segregation and genome stability. Centrosome activity requires the dynamic assembly of pericentriolar material (PCM), the composition and organization of which changes throughout the cell cycle. Recent studies highlight the conserved localization of several mRNAs encoded from centrosome-associated genes enriched at centrosomes, including Pericentrin-like protein (Plp) mRNA. However, relatively little is known about how RNAs localize to centrosomes and influence centrosome function. Here, we examine mechanisms underlying the subcellular localization of Plp mRNA. We find that Plp mRNA localization is puromycin-sensitive, and the Plp coding sequence is both necessary and sufficient for RNA localization, consistent with a co-translational transport mechanism. We identify regions within the Plp coding sequence that regulate Plp mRNA localization. Finally, we show that protein-protein interactions critical for elaboration of the PCM scaffold permit RNA localization to centrosomes. Taken together, these findings inform the mechanistic basis of Plp mRNA localization and lend insight into the oscillatory enrichment of RNA at centrosomes.\nA Multiscale approach reveals the molecular architecture of the autoinhibited kinesin KIF5A.\nAuthors: Carrington, G.; Fatima, U.; Caramujo, I.; Lewis, T.; Casas-Mao, D.; Peckham, M.\nScore: 9.3, Published: 2024-01-20 DOI: 10.1101/2024.01.18.576271\nKinesin-1 is a microtubule motor that transports cellular cargo along microtubules. KIF5A is one of three kinesin-1 isoforms in humans, all of which are autoinhibited by an interaction between the motor and an IAK motif in the proximal region of the C-terminal tail. The C-terminal tail of KIF5A is ~80 residues longer than the other two kinesin-1 isoforms (KIF5B and KIF5C) and it is unclear if it contributes to autoinhibition. Mutations in KIF5A cause neuronal diseases and could affect autoinhibition, as reported for a mutation that skips exon 27, altering its C-terminal sequence. Here, we combined negative-stain electron microscopy, crosslinking mass spectrometry (XL-MS) and AlphaFold2 structure prediction to determine the molecular architecture of the full-length autoinhibited KIF5A homodimer, in the absence of light chains. We show that KIF5A forms a compact, bent conformation, through a bend between coiled coils 2 and 3, around P687. XL-MS of WT KIF5A revealed extensive interactions between residues in the motor, between coiled-coil 1 and the motor, between coiled-coils 1 and 2, with coiled coils 3 and 4, and the proximal region of the C-terminal tail and the motor in the autoinhibited state, but not between the distal C-terminal region and the rest of the molecule. While negative stain electron microscopy of exon-27 KIF5A splice mutant showed the presence of autoinhibited molecules, XL-MS analysis suggested that its autoinhibited state is more labile. Our model offers a conceptual framework for understanding how mutations within the motor and stalk domain may affect motor activity.\nGamma protocadherins in vascular endothelial cells inhibit Klf2/4 to promote atherosclerosis\nAuthors: Joshi, D.; Coon, B. G.; Chakraborty, R.; Deng, H.; Fernandez Tussy, P.; Meredith, E.; Traylor, J. G.; Orr, A. W.; Fernandez-Hernando, C.; Schwartz, M. A.\nScore: 8.8, Published: 2024-01-17 DOI: 10.1101/2024.01.16.575958\nAtherosclerotic cardiovascular disease (ASCVD) is the leading cause of mortality worldwide1. Laminar shear stress (LSS) from blood flow in straight regions of arteries protects against ASCVD by upregulating the Klf2/4 anti-inflammatory program in endothelial cells (ECs)2-8. Conversely, disturbed shear stress (DSS) at curves or branches predisposes these regions to plaque formation9,10. We previously reported a whole genome CRISPR knockout screen11 that identified novel inducers of Klf2/4. Here we report suppressors of Klf2/4 and characterize one candidate, protocadherin gamma A9 (Pcdhga9), a member of the clustered protocadherin gene family12. Pcdhg deletion increases Klf2/4 levels in vitro and in vivo and suppresses inflammatory activation of ECs. Pcdhg suppresses Klf2/4 by inhibiting the Notch pathway via physical interaction of cleaved Notch1 intracellular domain (NICD Val1744) with nuclear Pcdhg C-terminal constant domain (CCD). Pcdhg inhibition by EC knockout (KO) or blocking antibody protects from atherosclerosis. Pcdhg is elevated in the arteries of human atherosclerosis. This study identifies a novel fundamental mechanism of EC resilience and therapeutic target for treating inflammatory vascular disease.\nAn unconventional regulatory circuitry involving Aurora B controls anaphase onset and error-free chromosome segregation in trypanosomes\nAuthors: Ballmer, D.; Lou, H. J.; Ishii, M.; Turk, B. E.; Akiyoshi, B.\nScore: 7.5, Published: 2024-01-20 DOI: 10.1101/2024.01.20.576407\nAccurate chromosome segregation during mitosis requires that all chromosomes establish stable bi-oriented attachments with the spindle apparatus. Kinetochores form the interface between chromosomes and spindle microtubules and as such are under tight control by complex regulatory circuitry. As part of the chromosomal passenger complex (CPC), the Aurora B kinase plays a central role within this circuitry by destabilizing improper kinetochore-microtubule attachments and relaying the attachment status to the spindle assembly checkpoint, a feedback control system that delays the onset of anaphase by inhibiting the anaphase-promoting complex/cyclosome. Intriguingly, Aurora B is conserved even in kinetoplastids, an evolutionarily divergent group of eukaryotes, whose kinetochores are composed of a unique set of structural and regulatory proteins. Kinetoplastids do not have a canonical spindle checkpoint and it remains unclear how their kinetochores are regulated to ensure the fidelity and timing of chromosome segregation. Here, we show in Trypanosoma brucei, the kinetoplastid parasite that causes African sleeping sickness, that inhibition of Aurora B using an analogue-sensitive approach arrests cells in metaphase, with a reduction in properly bi-oriented kinetochores. Aurora B phosphorylates several kinetochore proteins in vitro, including the N-terminal region of the divergent Bub1-like protein KKT14. Depletion of KKT14 partially overrides the cell cycle arrest caused by Aurora B inhibition, while overexpression of a non-phosphorylatable KKT14 protein results in a prominent delay in the metaphase-to-anaphase transition. Finally, we demonstrate using a nanobody-based system that re-targeting the catalytic module of the CPC to the outer kinetochore is sufficient to promote mitotic exit but causes massive chromosome mis-segregation in anaphase. Our results indicate that the CPC and KKT14 are involved in an unconventional pathway controlling mitotic exit and error-free chromosome segregation in trypanosomes.\n",
  "wordCount" : "2273",
  "inLanguage": "en",
  "datePublished": "2024-01-21T10:39:08Z",
  "dateModified": "2024-01-21T10:39:08Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cell-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cell biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 21, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.14.574223">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.14.574223" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.14.574223">
        <p class="paperTitle">A centromere-derived retroelement RNA localizes in cis and is a core element of the transcriptional landscape of Drosophila centromeres</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.14.574223" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.14.574223" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Santinello, B.; Sun, R.; Amjad, A.; Hoyt, S. J.; Ouyang, L.; Courret, C.; Drennan, R.; Leo, L.; Larracuente, A. M.; Core, L. M.; O&#39;Neill, R. J.; Mellone, B. G.</p>
        <p class="info">Score: 25.2, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.14.574223' target='https://doi.org/10.1101/2024.01.14.574223'> 10.1101/2024.01.14.574223</a></p>
        <p class="abstract">Centromeres are essential chromosomal landmarks that dictate the point of attachment between chromosomes and spindle microtubules during cell division. The stable transmission of the centromere site through generations is ensured by a unique chromatin containing the histone H3 variant CENP-A. Previous studies have highlighted the impact of transcription on promoting CENP-A deposition. However, the specific sequences undergoing this transcription and their contribution to centromere function in metazoan systems remain elusive. In this study, we unveil the centromeric transcriptional landscape and explore its correlation with CENP-A in D. melanogaster, currently the only in vivo model with assembled centromeres. We find that the centromere-enriched retroelement G2/Jockey-3 (hereafter referred to as Jockey-3) is a major driver of centromere transcription, producing RNAs that localize to all mitotic centromeres, with the Y centromere showing the most transcription. Taking advantage of the polymorphism of Jockey-3, we show that these RNAs remain associated with their cognate DNA sequences in cis. Using a LacI/lacO system to generate de novo centromeres, we find that Jockey-3 transcripts do not localize to ectopic sites, suggesting they are unlikely to function as non-coding RNAs with a structural role at centromeres. At de novo centromeres on the lacO array, the presence of CENP-A augments the detection of exogenous lacO-derived transcripts specifically in metaphase. We propose that Jockey-3 contributes to the epigenetic maintenance of the centromere by promoting chromatin transcription, while inserting in a region that permits its continuous transmission. Given the conservation of retroelements as centromere components across taxa, our findings have broad implications in understanding this widespread association.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.575273">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.575273" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.575273">
        <p class="paperTitle">High-Throughput Identification of Calcium Regulated Proteins Across Diverse Proteomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.575273" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.575273" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Locke, T. M.; Fields, R.; Gizinski, H.; Otto, G. M.; Berg, M. D.; Villen, J.; Sancak, Y.; Schweppe, D. K.</p>
        <p class="info">Score: 21.1, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.575273' target='https://doi.org/10.1101/2024.01.18.575273'> 10.1101/2024.01.18.575273</a></p>
        <p class="abstract">Calcium ions play important roles in nearly every biological process, yet whole-proteome analysis of calcium effectors has been hindered by lack of high-throughput, unbiased, and quantitative methods to identify proteins-calcium engagement. To address this, we adapted protein thermostability assays in the budding yeast, human cells, and mouse mitochondria. Based on calcium-dependent thermostability, we identified 2884 putative calcium-regulated proteins across human, mouse, and yeast proteomes. These data revealed calcium engagement of novel signaling hubs and cellular processes, including metabolic enzymes and the spliceosome. Cross-species comparison of calcium-protein engagement and mutagenesis experiments identified residue-specific cation engagement, even within well-known EF-hand domains. Additionally, we found that the dienoyl-CoA reductase DECR1 binds calcium at physiologically-relevant concentrations with substrate-specific affinity, suggesting direct calcium regulation of mitochondrial fatty acid oxidation. These unbiased, proteomic analyses of calcium effectors establish a key resource to dissect cation engagement and its mechanistic effects across multiple species and diverse biological processes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.575985">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.575985" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.575985">
        <p class="paperTitle">Nuclear pores as conduits for fluid flow during osmotic stress</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.575985" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.575985" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hoffmann, P. C.; Kim, H.; Obarska-Kosinska, A.; Kreysing, J. P.; Andino-Frydman, E.; Cruz-Leon, S.; Cernikova, L.; Kosinski, J.; Turonova, B.; Hummer, G.; Beck, M.</p>
        <p class="info">Score: 20.3, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.575985' target='https://doi.org/10.1101/2024.01.17.575985'> 10.1101/2024.01.17.575985</a></p>
        <p class="abstract">Changing environmental conditions necessitate an immediate cellular adaptation to ensure survival. Dictyostelium discoideum, a bacteriovore slime mold present in the soil of most terrestrial ecosystems, is known for its ability to tolerate drastic changes in osmolarity. How the cells cope with the resulting mechanical stress remains understudied. Here we show that D. discoideum has extraordinarily elaborate and resilient nuclear pores that serve as conduits for massive fluid exchange between cytosol and nucleus. We capitalize on the unique properties of D. discoideum cells to quantify flow across the nuclear envelope that is necessitated by changing nuclear size in response to osmotic stress. Based on mathematical concepts adapted from hydrodynamics, we conceptualize this phenomenon as porous flow across nuclear pores. This type of fluid flow is distinct from the canonically characterized modes of nucleocytoplasmic transport, i.e. passive diffusion and active nuclear transport, because of its dependence on pressure. Our insights are relevant in any biological condition that necessitates rapid nuclear size changes, which includes metastasizing cancer cells squeezing through constrictions, migrating cells and differentiating tissues.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.16.575939">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.16.575939" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.16.575939">
        <p class="paperTitle">Stimulation of NCAM1-14.3.3.ζδ-derived Peptide Interaction Fuels Angiogenesis and Osteogenesis in Ageing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.16.575939" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.16.575939" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kusumbe, A.; Yesin, T.; Liu, H.; Ding, Z.; Singh, A.; Zhang, Y.; Chen, J.</p>
        <p class="info">Score: 13.4, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.16.575939' target='https://doi.org/10.1101/2024.01.16.575939'> 10.1101/2024.01.16.575939</a></p>
        <p class="abstract">The skeletal structure and bone marrow endothelium collectively form a critical functional unit essential for bone development, health, and aging. At the core of osteogenesis and bone formation lies the dynamic process of angiogenesis. In this study, we reveal a potent new endogenous anabolic NCAM1-14.3.3.{zeta}{delta}-derived-Peptide interaction, which stimulates bone angiogenesis and osteogenesis during homeostasis, aging, and age-related bone diseases. Employing high-resolution imaging and inducible cell-specific mouse genetics, our results elucidate the pivotal role of the NCAM1-14.3.3.{zeta}{delta}-derived-Peptide interaction in driving the expansion of Clec14a&#43; angiogenic endothelial cells. Notably, Clec14a&#43; endothelial cells express key osteogenic factors. The NCAM1-14.3.3.{zeta}{delta}-derived-Peptide interaction in osteoblasts drives osteoblast differentiation, ultimately contributing to the genesis of new bone. Moreover, the NCAM1-14.3.3.{zeta}{delta}-derived-Peptide interaction leads to a reduction in bone resorption. In age-associated vascular and bone loss diseases, stimulating the NCAM1-14.3.3.{zeta}{delta}-derived-Peptide interaction not only promotes angiogenesis but also reverses bone loss. Consequently, harnessing the endogenous anabolic potential of the NCAM1-14.3.3.{zeta}{delta}-derived-Peptide interaction emerges as a promising therapeutic modality for managing age-related bone diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.16.575914">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.16.575914" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.16.575914">
        <p class="paperTitle">Structural basis of respiratory complexes adaptation to cold temperatures</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.16.575914" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.16.575914" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shin, Y.-C.; Latorre-Muro, P.; Djurabekova, A.; Zdorevskyi, O.; Bennett, C.; Burger, N.; Song, K.; Xu, C.; Sharma, V.; Liao, M.; Puigserver, P.</p>
        <p class="info">Score: 13.4, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.16.575914' target='https://doi.org/10.1101/2024.01.16.575914'> 10.1101/2024.01.16.575914</a></p>
        <p class="abstract">In response to cold, mammals activate brown fat for respiratory-dependent thermogenesis reliant on the electron transport chain (1, 2). Yet, the structural basis of respiratory complex adaptation to cold remains elusive. Herein we combined thermoregulatory physiology and cryo-EM to study endogenous respiratory supercomplexes exposed to different temperatures. A cold-induced conformation of CI:III2 (termed type 2) was identified with a [~]25{degrees} rotation of CIII2 around its inter-dimer axis, shortening inter-complex Q exchange space, and exhibiting different catalytic states which favor electron transfer. Large-scale supercomplex simulations in lipid membrane reveal how unique lipid-protein arrangements stabilize type 2 complexes to enhance catalytic activity. Together, our cryo-EM studies, multiscale simulations and biochemical analyses unveil the mechanisms and dynamics of respiratory adaptation at the structural and energetic level.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.574250">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.574250" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.574250">
        <p class="paperTitle">Measuring and modeling the dynamics of mitotic error correction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.574250" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.574250" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ha, G.; Dieterle, P.; Shen, H.; Amir, A.; Needleman, D. J.</p>
        <p class="info">Score: 13.0, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.574250' target='https://doi.org/10.1101/2024.01.10.574250'> 10.1101/2024.01.10.574250</a></p>
        <p class="abstract">When functioning properly, the mitotic spindle segregates an equal number of chromosomes to daughter cells with high fidelity. Over the course of spindle assembly, many initially erroneous attachments between kinetochores and microtubules are fixed through a process called error correction. Despite the importance of chromosome segregation errors in cancer and other diseases, there is a lack of methods to characterize the dynamics of error correction and how it can go wrong. Here, we present an experimental method and analysis framework to quantify chromosome segregation error correction in human tissue culture cells with live cell confocal imaging, timed premature anaphase, and automated counting of kinetochores after cell division. We find that errors decrease exponentially over time during spindle assembly. A coarse-grained model, in which errors are corrected in a chromosome autonomous manner at a constant rate, can quantitatively explain both the measured error correction dynamics and the distribution of anaphase onset times. We further validated our model using perturbations that destabilized microtubules and changed the initial configuration of chromosomal attachments. Taken together, this work provides a quantitative framework for understanding the dynamics of mitotic error correction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.13.575509">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.13.575509" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.13.575509">
        <p class="paperTitle">The PCM scaffold enables RNA localization to centrosomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.13.575509" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.13.575509" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fang, J.; Tian, W.; Quintanilla, M. A.; Beach, J. R.; Lerit, D. A.</p>
        <p class="info">Score: 9.5, Published: 2024-01-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.13.575509' target='https://doi.org/10.1101/2024.01.13.575509'> 10.1101/2024.01.13.575509</a></p>
        <p class="abstract">As microtubule-organizing centers, centrosomes direct assembly of the bipolar mitotic spindle required for chromosome segregation and genome stability. Centrosome activity requires the dynamic assembly of pericentriolar material (PCM), the composition and organization of which changes throughout the cell cycle. Recent studies highlight the conserved localization of several mRNAs encoded from centrosome-associated genes enriched at centrosomes, including Pericentrin-like protein (Plp) mRNA. However, relatively little is known about how RNAs localize to centrosomes and influence centrosome function. Here, we examine mechanisms underlying the subcellular localization of Plp mRNA. We find that Plp mRNA localization is puromycin-sensitive, and the Plp coding sequence is both necessary and sufficient for RNA localization, consistent with a co-translational transport mechanism. We identify regions within the Plp coding sequence that regulate Plp mRNA localization. Finally, we show that protein-protein interactions critical for elaboration of the PCM scaffold permit RNA localization to centrosomes. Taken together, these findings inform the mechanistic basis of Plp mRNA localization and lend insight into the oscillatory enrichment of RNA at centrosomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576271">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576271" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576271">
        <p class="paperTitle">A Multiscale approach reveals the molecular architecture of the autoinhibited kinesin KIF5A.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576271" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576271" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carrington, G.; Fatima, U.; Caramujo, I.; Lewis, T.; Casas-Mao, D.; Peckham, M.</p>
        <p class="info">Score: 9.3, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576271' target='https://doi.org/10.1101/2024.01.18.576271'> 10.1101/2024.01.18.576271</a></p>
        <p class="abstract">Kinesin-1 is a microtubule motor that transports cellular cargo along microtubules. KIF5A is one of three kinesin-1 isoforms in humans, all of which are autoinhibited by an interaction between the motor and an IAK motif in the proximal region of the C-terminal tail. The C-terminal tail of KIF5A is ~80 residues longer than the other two kinesin-1 isoforms (KIF5B and KIF5C) and it is unclear if it contributes to autoinhibition. Mutations in KIF5A cause neuronal diseases and could affect autoinhibition, as reported for a mutation that skips exon 27, altering its C-terminal sequence. Here, we combined negative-stain electron microscopy, crosslinking mass spectrometry (XL-MS) and AlphaFold2 structure prediction to determine the molecular architecture of the full-length autoinhibited KIF5A homodimer, in the absence of light chains. We show that KIF5A forms a compact, bent conformation, through a bend between coiled coils 2 and 3, around P687. XL-MS of WT KIF5A revealed extensive interactions between residues in the motor, between coiled-coil 1 and the motor, between coiled-coils 1 and 2, with coiled coils 3 and 4, and the proximal region of the C-terminal tail and the motor in the autoinhibited state, but not between the distal C-terminal region and the rest of the molecule. While negative stain electron microscopy of exon-27 KIF5A splice mutant showed the presence of autoinhibited molecules, XL-MS analysis suggested that its autoinhibited state is more labile. Our model offers a conceptual framework for understanding how mutations within the motor and stalk domain may affect motor activity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.16.575958">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.16.575958" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.16.575958">
        <p class="paperTitle">Gamma protocadherins in vascular endothelial cells inhibit Klf2/4 to promote atherosclerosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.16.575958" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.16.575958" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Joshi, D.; Coon, B. G.; Chakraborty, R.; Deng, H.; Fernandez Tussy, P.; Meredith, E.; Traylor, J. G.; Orr, A. W.; Fernandez-Hernando, C.; Schwartz, M. A.</p>
        <p class="info">Score: 8.8, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.16.575958' target='https://doi.org/10.1101/2024.01.16.575958'> 10.1101/2024.01.16.575958</a></p>
        <p class="abstract">Atherosclerotic cardiovascular disease (ASCVD) is the leading cause of mortality worldwide1. Laminar shear stress (LSS) from blood flow in straight regions of arteries protects against ASCVD by upregulating the Klf2/4 anti-inflammatory program in endothelial cells (ECs)2-8. Conversely, disturbed shear stress (DSS) at curves or branches predisposes these regions to plaque formation9,10. We previously reported a whole genome CRISPR knockout screen11 that identified novel inducers of Klf2/4. Here we report suppressors of Klf2/4 and characterize one candidate, protocadherin gamma A9 (Pcdhga9), a member of the clustered protocadherin gene family12. Pcdhg deletion increases Klf2/4 levels in vitro and in vivo and suppresses inflammatory activation of ECs. Pcdhg suppresses Klf2/4 by inhibiting the Notch pathway via physical interaction of cleaved Notch1 intracellular domain (NICD Val1744) with nuclear Pcdhg C-terminal constant domain (CCD). Pcdhg inhibition by EC knockout (KO) or blocking antibody protects from atherosclerosis. Pcdhg is elevated in the arteries of human atherosclerosis. This study identifies a novel fundamental mechanism of EC resilience and therapeutic target for treating inflammatory vascular disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576407">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576407" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576407">
        <p class="paperTitle">An unconventional regulatory circuitry involving Aurora B controls anaphase onset and error-free chromosome segregation in trypanosomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576407" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576407" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ballmer, D.; Lou, H. J.; Ishii, M.; Turk, B. E.; Akiyoshi, B.</p>
        <p class="info">Score: 7.5, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576407' target='https://doi.org/10.1101/2024.01.20.576407'> 10.1101/2024.01.20.576407</a></p>
        <p class="abstract">Accurate chromosome segregation during mitosis requires that all chromosomes establish stable bi-oriented attachments with the spindle apparatus. Kinetochores form the interface between chromosomes and spindle microtubules and as such are under tight control by complex regulatory circuitry. As part of the chromosomal passenger complex (CPC), the Aurora B kinase plays a central role within this circuitry by destabilizing improper kinetochore-microtubule attachments and relaying the attachment status to the spindle assembly checkpoint, a feedback control system that delays the onset of anaphase by inhibiting the anaphase-promoting complex/cyclosome. Intriguingly, Aurora B is conserved even in kinetoplastids, an evolutionarily divergent group of eukaryotes, whose kinetochores are composed of a unique set of structural and regulatory proteins. Kinetoplastids do not have a canonical spindle checkpoint and it remains unclear how their kinetochores are regulated to ensure the fidelity and timing of chromosome segregation. Here, we show in Trypanosoma brucei, the kinetoplastid parasite that causes African sleeping sickness, that inhibition of Aurora B using an analogue-sensitive approach arrests cells in metaphase, with a reduction in properly bi-oriented kinetochores. Aurora B phosphorylates several kinetochore proteins in vitro, including the N-terminal region of the divergent Bub1-like protein KKT14. Depletion of KKT14 partially overrides the cell cycle arrest caused by Aurora B inhibition, while overexpression of a non-phosphorylatable KKT14 protein results in a prominent delay in the metaphase-to-anaphase transition. Finally, we demonstrate using a nanobody-based system that re-targeting the catalytic module of the CPC to the outer kinetochore is sufficient to promote mitotic exit but causes massive chromosome mis-segregation in anaphase. Our results indicate that the CPC and KKT14 are involved in an unconventional pathway controlling mitotic exit and error-free chromosome segregation in trypanosomes.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
