<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cell biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cell biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Cryosectioning-enabled super-resolution microscopy for studying nuclear architecture at the single protein level
Authors: Stein, J.; Ericsson, M.; Nofal, M.; Magni, L.; Aufmkolk, S.; McMillan, R. B.; Breimann, L.; Herlihy, C. P.; Lee, S. D.; Willemin, A.; Wohlmann, J.; Arguedas-Jimenez, L.; Yin, P.; Pombo, A.; Church, G. M.; Wu, C.-t.
Score: 47.5, Published: 2024-02-05 DOI: 10.1101/2024.02.05.576943
DNA-PAINT combined with total Internal Reflection Fluorescence (TIRF) microscopy enables the highest localization precisions, down to single nanometers in thin biological samples, due to TIRFs unique method for optical sectioning and attaining high contrast.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cell-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cell biology" />
<meta property="og:description" content="Cryosectioning-enabled super-resolution microscopy for studying nuclear architecture at the single protein level
Authors: Stein, J.; Ericsson, M.; Nofal, M.; Magni, L.; Aufmkolk, S.; McMillan, R. B.; Breimann, L.; Herlihy, C. P.; Lee, S. D.; Willemin, A.; Wohlmann, J.; Arguedas-Jimenez, L.; Yin, P.; Pombo, A.; Church, G. M.; Wu, C.-t.
Score: 47.5, Published: 2024-02-05 DOI: 10.1101/2024.02.05.576943
DNA-PAINT combined with total Internal Reflection Fluorescence (TIRF) microscopy enables the highest localization precisions, down to single nanometers in thin biological samples, due to TIRFs unique method for optical sectioning and attaining high contrast." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cell-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-14T10:38:48+00:00" />
<meta property="article:modified_time" content="2024-02-14T10:38:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cell biology"/>
<meta name="twitter:description" content="Cryosectioning-enabled super-resolution microscopy for studying nuclear architecture at the single protein level
Authors: Stein, J.; Ericsson, M.; Nofal, M.; Magni, L.; Aufmkolk, S.; McMillan, R. B.; Breimann, L.; Herlihy, C. P.; Lee, S. D.; Willemin, A.; Wohlmann, J.; Arguedas-Jimenez, L.; Yin, P.; Pombo, A.; Church, G. M.; Wu, C.-t.
Score: 47.5, Published: 2024-02-05 DOI: 10.1101/2024.02.05.576943
DNA-PAINT combined with total Internal Reflection Fluorescence (TIRF) microscopy enables the highest localization precisions, down to single nanometers in thin biological samples, due to TIRFs unique method for optical sectioning and attaining high contrast."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cell biology",
      "item": "https://trxiv.yorks0n.com/posts/cell-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cell biology",
  "name": "cell biology",
  "description": "Cryosectioning-enabled super-resolution microscopy for studying nuclear architecture at the single protein level\nAuthors: Stein, J.; Ericsson, M.; Nofal, M.; Magni, L.; Aufmkolk, S.; McMillan, R. B.; Breimann, L.; Herlihy, C. P.; Lee, S. D.; Willemin, A.; Wohlmann, J.; Arguedas-Jimenez, L.; Yin, P.; Pombo, A.; Church, G. M.; Wu, C.-t.\nScore: 47.5, Published: 2024-02-05 DOI: 10.1101/2024.02.05.576943\nDNA-PAINT combined with total Internal Reflection Fluorescence (TIRF) microscopy enables the highest localization precisions, down to single nanometers in thin biological samples, due to TIRFs unique method for optical sectioning and attaining high contrast.",
  "keywords": [
    
  ],
  "articleBody": " Cryosectioning-enabled super-resolution microscopy for studying nuclear architecture at the single protein level\nAuthors: Stein, J.; Ericsson, M.; Nofal, M.; Magni, L.; Aufmkolk, S.; McMillan, R. B.; Breimann, L.; Herlihy, C. P.; Lee, S. D.; Willemin, A.; Wohlmann, J.; Arguedas-Jimenez, L.; Yin, P.; Pombo, A.; Church, G. M.; Wu, C.-t.\nScore: 47.5, Published: 2024-02-05 DOI: 10.1101/2024.02.05.576943\nDNA-PAINT combined with total Internal Reflection Fluorescence (TIRF) microscopy enables the highest localization precisions, down to single nanometers in thin biological samples, due to TIRFs unique method for optical sectioning and attaining high contrast. However, most cellular targets elude the accessible TIRF range close to the cover glass and thus require alternative imaging conditions, affecting resolution and image quality. Here, we address this limitation by applying ultrathin physical cryosectioning in combination with DNA-PAINT. With \"tomographic \u0026 kinetically-enhanced\" DNA-PAINT (tokPAINT), we demonstrate the imaging of nuclear proteins with sub-3 nanometer localization precision, advancing the quantitative study of nuclear organization within fixed cells and mouse tissues at the level of single antibodies. We believe that ultrathin sectioning combined with the versatility and multiplexing capabilities of DNA-PAINT will be a powerful addition to the toolbox of quantitative DNA-based super-resolution microscopy in intracellular structural analyses of proteins, RNA and DNA in situ.\nDissecting reprogramming heterogeneity at single-cell resolution using scTF-seq\nAuthors: Liu, W.; Saelens, W.; Rainer, P.; Biocanin, M.; Gardeux, V.; Gralak, A.; van Mierlo, G.; Russeil, J.; Liu, T.; Chen, W.; Deplancke, B.\nScore: 43.7, Published: 2024-02-02 DOI: 10.1101/2024.01.30.577921\nReprogramming approaches often produce heterogeneous cell fates and the mechanisms behind this heterogeneity are not well-understood. To address this gap, we developed scTF-seq, a technique inducing single-cell barcoded and doxycycline-inducible TF overexpression while quantifying TF dose-dependent transcriptomic changes. Applied to mouse embryonic multipotent stromal cells (MSCs), scTF-seq produced a gain-of-function atlas for 384 murine TFs. This atlas offers a valuable resource for gene regulation and reprogramming research, identifying key TFs governing MSC lineage differentiation, cell cycle control, and their interplay. Leveraging the single-cell resolution, we dissected reprogramming heterogeneity along dose and pseudotime. We thereby revealed TF dose-dependent and stochastic cell fate branching, unveiling gene expression signatures that enhance our understanding and prediction of reprogramming efficiency. scTF-seq also allowed us to classify TFs into four sensitivity classes based on dose response and determining features. Finally, in combinatorial scTF-seq, we observed that the same TF can exhibit both synergistic and antagonistic effects on another TF depending on its dose. In summary, scTF-seq provides a powerful tool for gaining mechanistic insights into how TFs determine cell states, while offering novel perspectives for cellular engineering strategies.\nGentle rhodamines for live-cell fluorescence microscopy\nAuthors: Liu, T.; Kompa, J.; Ling, J.; Lardon, N.; Zhang, Y.; Chen, J.; Reymond, L.; Chen, P.; Tran, M.; Yang, Z.; Zhang, H.; Liu, Y.; Pitsch, S.; Zou, P.; Wang, L.; Johnsson, K.; Chen, Z.\nScore: 13.7, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579089\nRhodamines have been continuously optimized in brightness, biocompatibility, and colors to fulfill the demands of modern bioimaging. However, the problem of phototoxicity caused by the excited fluorophore under long-term illumination has been largely neglected, hampering their use in time-lapse imaging. Here we introduce cyclooctatetraene (COT) conjugated rhodamines that span the visible spectrum and exhibit significantly reduced phototoxicity. We identified a general strategy for the generation of Gentle Rhodamines, which preserved their outstanding spectroscopic properties and cell permeability while showing an efficient reduction of singlet-oxygen formation and diminished cellular photodamage. Paradoxically, their photobleaching kinetics do not go hand in hand with reduced phototoxicity. By combining COT-conjugated spirocyclization motifs with targeting moieties, these gentle rhodamines compose a toolkit for time-lapse imaging of mitochondria, DNA, and actin and synergize with covalent and exchangeable HaloTag labeling of cellular proteins with less photodamage than their commonly used precursors. Taken together, the Gentle Rhodamines generally offer alleviated phototoxicity and allow advanced video recording applications, including voltage imaging.\nSingle-nucleus RNA sequencing reveals cellular and molecular dynamics of white and brown adipose tissue in a mouse model of type-2 diabetes\nAuthors: Gaynor, T. L.; Hsu, I.; Krstevski, C.; Farrugia, G. E.; Dona, M. S.; Cohen, C. D.; Drew, B. G.; Pinto, A. R.\nScore: 8.3, Published: 2024-02-07 DOI: 10.1101/2024.02.06.578860\nExcessive adipose tissue expansion is often linked with type-2 diabetes. Despite recent efforts mapping adipose tissue changes in obesity using single-cell omics, an understanding of cellular and gene expression changes in a model of type 2 diabetes, and the transcriptional circuitry controlling it, is still lacking. Here, we use single-nucleus RNA sequencing to analyse the remodelling of gonadal white and interscapular brown adipose tissue from female and male mice with or without diabetes. Analysis of 51,877 nuclei revealed altered phenotypes in every cell population in type 2 diabetes. This included an immunoregulatory response, and changes in extracellular matrix organisation, energy generation, and hormone stimuli. Key transcription factors were inferred as cell-specific and non-specific nodes controlling diabetes-linked phenotypes. Finally, female-to-male population heterogeneity and gene expression differences were observed. Here we provide a resource detailing how adipose tissue remodelling, and the molecular mechanisms governing it, may contribute to cardiometabolic disease.\nSubstrate recognition principles for the PP2A-B55 protein phosphatase\nAuthors: Kruse, T.; Garvanska, D.; Varga, J. K.; Garland, W.; McEwan, B.; Hein, J.; Weisser, M. B.; Puy, I. B.; Chan, C.; Parrila, P. S.; Lopez-Mendez, B.; Jeyaprakash, A. A.; Schueler-Furman, O.; Jensen, T. H.; Kettenbach, A.; Nilsson, J.\nScore: 8.1, Published: 2024-02-13 DOI: 10.1101/2024.02.10.579793\nThe PP2A-B55 phosphatase regulates a plethora of signaling pathways throughout eukaryotes. How PP2A-B55 selects its substrates presents a severe knowledge gap. By integrating AlphaFold modelling with comprehensive high resolution mutational scanning, we show that alpha helices in substrates bind B55 through an evolutionary conserved mechanism. Despite a large diversity in sequence and composition, these alpha helices share key amino acid determinants that engage discrete hydrophobic and electrostatic patches. Using deep learning protein design, we generate a specific and potent competitive peptide inhibitor of PP2A-B55 substrate interactions. With this inhibitor, we uncover that PP2A-B55 regulates the nuclear exosome targeting complex by binding to a helical recruitment module in RBM7. Collectively, our findings provide a framework for the understanding and interrogation of PP2A-B55 in health and disease.\nCytoplasmic protein-free mRNA induces stress granules by two G3BP1/2-dependent mechanisms\nAuthors: Ihn, S. J.; Farlam-Williams, L.; Palazzo, A. F.; Lee, H. O.\nScore: 7.9, Published: 2024-02-08 DOI: 10.1101/2024.02.07.578830\nStress granules are cytoplasmic membraneless organelles that sequester proteins and non-translating mRNAs in response to various stressors. To assess the contributions of mRNA and RNA-binding proteins to stress granule formation, we use microinjection to deliver protein-free mRNA into the cytoplasm in a controlled manner. We demonstrate that mRNAs trigger stress granule formation through two mechanisms that are enhanced by the presence of G3BP1 and G3BP2. Low concentrations of in vitro transcribed mRNA activated protein kinase R (PKR), leading to phosphorylation and inhibition of the eukaryotic translation initiation factor eIF2 and stress granule formation. This was inhibited by replacing uridine with pseudouridine in the mRNA or by treating it with RNase III, which cleaves double-stranded RNA. High concentrations of mRNA triggered stress granule formation by a mechanism that was independent of PKR and enhanced by G3BP1/2, highlighting the importance of both protein-free mRNA and RNA-binding proteins in stress granule formation. Graphical Abstract/Model O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=197 SRC=\"FIGDIR/small/578830v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (27K): org.highwire.dtl.DTLVardef@19dc771org.highwire.dtl.DTLVardef@190ce98org.highwire.dtl.DTLVardef@1eb7aaeorg.highwire.dtl.DTLVardef@c35f86_HPS_FORMAT_FIGEXP M_FIG C_FIG SummaryMicroinjected mRNA induces stress granules in mammalian cells by two G3BP1/2-dependent mechanisms: one requires the stress-sensing protein kinase PKR to phosphorylate the translation initiation factor eIF2, and the other is independent of PKR and phospho-eIF2 and acts when the cytoplasmic concentration of ribosome-free mRNA is increased acutely.\nNuclear pores safeguard the integrity of the nuclear envelope\nAuthors: Taniguchi, R.; Orniacki, C.; Kreysing, J. P.; Zila, V.; Zimmerli, C. E.; Boehm, S.; Turonova, B.; Kraeusslich, H.-G.; Doye, V.; Beck, M.\nScore: 25.2, Published: 2024-02-05 DOI: 10.1101/2024.02.05.578890\nNuclear pore complexes (NPCs) constitute giant channels within the nuclear envelope that mediate nucleocytoplasmic exchange. NPC diameter is thought to be regulated by nuclear envelope tension, but how such diameter changes are physiologically linked to cell differentiation, where mechanical properties of nuclei are remodeled and nuclear mechanosensing occurs, remains unstudied. Here we used cryo-electron tomography to show that NPCs dilate during differentiation of mouse embryonic stem cells into neural progenitors. In Nup133-deficient cells, which are known to display impaired neural differentiation, NPCs however fail to dilate. By analyzing the architectures of individual NPCs with template matching, we revealed that the Nup133-deficient NPCs are structurally heterogeneous and frequently disintegrate, resulting in the formation of large nuclear envelope openings. We propose that the elasticity of the NPC scaffold mechanically safeguards the nuclear envelope. Our studies provide a molecular explanation for how genetic perturbation of scaffolding components of macromolecular complexes causes tissue-specific phenotypes.\nSelective Translation Orchestrates Key Signaling Pathways in Primed Pluripotency\nAuthors: Okubo, C.; Nakamura, M.; Sato, M.; Shichino, Y.; Mito, M.; Takashima, Y.; Iwasaki, S.; Takahashi, K.\nScore: 6.3, Published: 2024-02-10 DOI: 10.1101/2024.02.09.579580\nPluripotent stem cell (PSC) identities, such as differentiation and infinite proliferation, have long been understood within the frameworks of transcription factor networks, epigenomes, and signal transduction, yet remain unclear and fragmented. Directing attention toward translational regulation, as a bridge between these events, promises to yield new insights into previously unexplained mechanisms. Our functional CRISPR interference screening-based approach revealed that EIF3D maintains primed pluripotency through selective translational regulation. The loss of EIF3D disrupts the balance of pluripotency-associated signaling pathways, impairing primed pluripotency. Moreover, we discovered that EIF3D ensures robust proliferation by controlling the translation of various p53 regulators, which maintain low p53 activity in the undifferentiated state. In this way, selective translation by EIF3D tunes the homeostasis of the primed pluripotency networks, ensuring the maintenance of an undifferentiated state with high proliferative potential. Therefore, this study establishes a paradigm for selective translational regulation as a defining feature of primed PSC identity.\nCa2+-driven cytoplasmic backflow secures spindle position in fertilized mouse eggs\nAuthors: Totsuka, T.; Ohsugi, M.\nScore: 6.5, Published: 2024-02-06 DOI: 10.1101/2024.02.05.578955\nAbstractFertilization triggers hours-long Ca2+ oscillations in mammalian eggs, but the effects of repeated Ca2+ surges remain unclear. Here, we investigate spindle dynamics and its relationship with cytoplasmic streaming in fertilized mouse eggs. The spindle, initially parallel to the plasma membrane, rotates vertically, in accordance with previously reported results using artificially activated eggs. Intriguingly, it transiently reverses its rotation direction in synchrony with Ca2+ oscillations, regardless of artificially altered frequency. This effect results from cytoplasmic streaming, initially moving from spindle to egg center, displaying a Ca2+-dependent backflow. Streaming also impacts spindle positioning, balancing spindle rotation and cortical localization maintenance. We provide evidence that Ca2+-dependent cortical myosin II activation causes actomyosin contraction, leading to transient streaming towards non-contracting actin cap regions overlaying chromosomes. Our findings underscore the role of Ca2+ oscillations in maintaining spindle position in fertilized eggs, thereby ensuring highly asymmetric division and preservation of maternal stores in zygotes.\nOncogenic STIL-mediated loss of BRCA1 functionality causes DNA damage and centrosome amplification\nAuthors: Sanghi, S.; Singh, P.\nScore: 6.1, Published: 2024-02-12 DOI: 10.1101/2024.02.11.579865\nCentrosomes are major microtubule organizing centers in many animal cells and could also act as a communicator between nuclear-cytoplasmic signaling during DNA damages. Therefore, DNA damages cause an increase in centrosome number, however the molecular linkers remain elusive. STIL (SCL/TAL1 Interrupting locus) is a core centrosome organizing protein, which is overexpressed in different cancer types. In this work, we identify a frequently reported heterozygous missense mutation in STIL, which maps to its N-terminal located CR1 region (S76L). The oncogenic STIL-S76L loses interaction with the DNA damage response protein BRCA1 and exhibits a decrease in BRCA1 nuclear localization, which causes DNA damage and enhanced centrosome number. Interestingly, exogenous BRCA1 can rescue these phenotypes of STIL-S76L, thus, establishing it as an effector for the oncogenic STIL functions. These amplified centrosomes tend to cluster in the M phase and cause pseudo bipolar spindle organization, which is a promising target for anti-cancer drugs.\n",
  "wordCount" : "1958",
  "inLanguage": "en",
  "datePublished": "2024-02-14T10:38:48Z",
  "dateModified": "2024-02-14T10:38:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cell-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cell biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 14, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.05.576943">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.05.576943" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.05.576943">
        <p class="paperTitle">Cryosectioning-enabled super-resolution microscopy for studying nuclear architecture at the single protein level</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.05.576943" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.05.576943" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stein, J.; Ericsson, M.; Nofal, M.; Magni, L.; Aufmkolk, S.; McMillan, R. B.; Breimann, L.; Herlihy, C. P.; Lee, S. D.; Willemin, A.; Wohlmann, J.; Arguedas-Jimenez, L.; Yin, P.; Pombo, A.; Church, G. M.; Wu, C.-t.</p>
        <p class="info">Score: 47.5, Published: 2024-02-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.05.576943' target='https://doi.org/10.1101/2024.02.05.576943'> 10.1101/2024.02.05.576943</a></p>
        <p class="abstract">DNA-PAINT combined with total Internal Reflection Fluorescence (TIRF) microscopy enables the highest localization precisions, down to single nanometers in thin biological samples, due to TIRFs unique method for optical sectioning and attaining high contrast. However, most cellular targets elude the accessible TIRF range close to the cover glass and thus require alternative imaging conditions, affecting resolution and image quality. Here, we address this limitation by applying ultrathin physical cryosectioning in combination with DNA-PAINT. With &#34;tomographic &amp; kinetically-enhanced&#34; DNA-PAINT (tokPAINT), we demonstrate the imaging of nuclear proteins with sub-3 nanometer localization precision, advancing the quantitative study of nuclear organization within fixed cells and mouse tissues at the level of single antibodies. We believe that ultrathin sectioning combined with the versatility and multiplexing capabilities of DNA-PAINT will be a powerful addition to the toolbox of quantitative DNA-based super-resolution microscopy in intracellular structural analyses of proteins, RNA and DNA in situ.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.30.577921">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.30.577921" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.30.577921">
        <p class="paperTitle">Dissecting reprogramming heterogeneity at single-cell resolution using scTF-seq</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.30.577921" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.30.577921" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, W.; Saelens, W.; Rainer, P.; Biocanin, M.; Gardeux, V.; Gralak, A.; van Mierlo, G.; Russeil, J.; Liu, T.; Chen, W.; Deplancke, B.</p>
        <p class="info">Score: 43.7, Published: 2024-02-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.30.577921' target='https://doi.org/10.1101/2024.01.30.577921'> 10.1101/2024.01.30.577921</a></p>
        <p class="abstract">Reprogramming approaches often produce heterogeneous cell fates and the mechanisms behind this heterogeneity are not well-understood. To address this gap, we developed scTF-seq, a technique inducing single-cell barcoded and doxycycline-inducible TF overexpression while quantifying TF dose-dependent transcriptomic changes. Applied to mouse embryonic multipotent stromal cells (MSCs), scTF-seq produced a gain-of-function atlas for 384 murine TFs. This atlas offers a valuable resource for gene regulation and reprogramming research, identifying key TFs governing MSC lineage differentiation, cell cycle control, and their interplay. Leveraging the single-cell resolution, we dissected reprogramming heterogeneity along dose and pseudotime. We thereby revealed TF dose-dependent and stochastic cell fate branching, unveiling gene expression signatures that enhance our understanding and prediction of reprogramming efficiency. scTF-seq also allowed us to classify TFs into four sensitivity classes based on dose response and determining features. Finally, in combinatorial scTF-seq, we observed that the same TF can exhibit both synergistic and antagonistic effects on another TF depending on its dose. In summary, scTF-seq provides a powerful tool for gaining mechanistic insights into how TFs determine cell states, while offering novel perspectives for cellular engineering strategies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579089">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579089" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579089">
        <p class="paperTitle">Gentle rhodamines for live-cell fluorescence microscopy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579089" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579089" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, T.; Kompa, J.; Ling, J.; Lardon, N.; Zhang, Y.; Chen, J.; Reymond, L.; Chen, P.; Tran, M.; Yang, Z.; Zhang, H.; Liu, Y.; Pitsch, S.; Zou, P.; Wang, L.; Johnsson, K.; Chen, Z.</p>
        <p class="info">Score: 13.7, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579089' target='https://doi.org/10.1101/2024.02.06.579089'> 10.1101/2024.02.06.579089</a></p>
        <p class="abstract">Rhodamines have been continuously optimized in brightness, biocompatibility, and colors to fulfill the demands of modern bioimaging. However, the problem of phototoxicity caused by the excited fluorophore under long-term illumination has been largely neglected, hampering their use in time-lapse imaging. Here we introduce cyclooctatetraene (COT) conjugated rhodamines that span the visible spectrum and exhibit significantly reduced phototoxicity. We identified a general strategy for the generation of Gentle Rhodamines, which preserved their outstanding spectroscopic properties and cell permeability while showing an efficient reduction of singlet-oxygen formation and diminished cellular photodamage. Paradoxically, their photobleaching kinetics do not go hand in hand with reduced phototoxicity. By combining COT-conjugated spirocyclization motifs with targeting moieties, these gentle rhodamines compose a toolkit for time-lapse imaging of mitochondria, DNA, and actin and synergize with covalent and exchangeable HaloTag labeling of cellular proteins with less photodamage than their commonly used precursors. Taken together, the Gentle Rhodamines generally offer alleviated phototoxicity and allow advanced video recording applications, including voltage imaging.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.578860">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.578860" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.578860">
        <p class="paperTitle">Single-nucleus RNA sequencing reveals cellular and molecular dynamics of white and brown adipose tissue in a mouse model of type-2 diabetes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.578860" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.578860" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gaynor, T. L.; Hsu, I.; Krstevski, C.; Farrugia, G. E.; Dona, M. S.; Cohen, C. D.; Drew, B. G.; Pinto, A. R.</p>
        <p class="info">Score: 8.3, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.578860' target='https://doi.org/10.1101/2024.02.06.578860'> 10.1101/2024.02.06.578860</a></p>
        <p class="abstract">Excessive adipose tissue expansion is often linked with type-2 diabetes. Despite recent efforts mapping adipose tissue changes in obesity using single-cell omics, an understanding of cellular and gene expression changes in a model of type 2 diabetes, and the transcriptional circuitry controlling it, is still lacking. Here, we use single-nucleus RNA sequencing to analyse the remodelling of gonadal white and interscapular brown adipose tissue from female and male mice with or without diabetes. Analysis of 51,877 nuclei revealed altered phenotypes in every cell population in type 2 diabetes. This included an immunoregulatory response, and changes in extracellular matrix organisation, energy generation, and hormone stimuli. Key transcription factors were inferred as cell-specific and non-specific nodes controlling diabetes-linked phenotypes. Finally, female-to-male population heterogeneity and gene expression differences were observed. Here we provide a resource detailing how adipose tissue remodelling, and the molecular mechanisms governing it, may contribute to cardiometabolic disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.10.579793">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.10.579793" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.10.579793">
        <p class="paperTitle">Substrate recognition principles for the PP2A-B55 protein phosphatase</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.10.579793" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.10.579793" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kruse, T.; Garvanska, D.; Varga, J. K.; Garland, W.; McEwan, B.; Hein, J.; Weisser, M. B.; Puy, I. B.; Chan, C.; Parrila, P. S.; Lopez-Mendez, B.; Jeyaprakash, A. A.; Schueler-Furman, O.; Jensen, T. H.; Kettenbach, A.; Nilsson, J.</p>
        <p class="info">Score: 8.1, Published: 2024-02-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.10.579793' target='https://doi.org/10.1101/2024.02.10.579793'> 10.1101/2024.02.10.579793</a></p>
        <p class="abstract">The PP2A-B55 phosphatase regulates a plethora of signaling pathways throughout eukaryotes. How PP2A-B55 selects its substrates presents a severe knowledge gap. By integrating AlphaFold modelling with comprehensive high resolution mutational scanning, we show that alpha helices in substrates bind B55 through an evolutionary conserved mechanism. Despite a large diversity in sequence and composition, these alpha helices share key amino acid determinants that engage discrete hydrophobic and electrostatic patches. Using deep learning protein design, we generate a specific and potent competitive peptide inhibitor of PP2A-B55 substrate interactions. With this inhibitor, we uncover that PP2A-B55 regulates the nuclear exosome targeting complex by binding to a helical recruitment module in RBM7. Collectively, our findings provide a framework for the understanding and interrogation of PP2A-B55 in health and disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.07.578830">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.07.578830" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.07.578830">
        <p class="paperTitle">Cytoplasmic protein-free mRNA induces stress granules by two G3BP1/2-dependent mechanisms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.07.578830" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.07.578830" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ihn, S. J.; Farlam-Williams, L.; Palazzo, A. F.; Lee, H. O.</p>
        <p class="info">Score: 7.9, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.07.578830' target='https://doi.org/10.1101/2024.02.07.578830'> 10.1101/2024.02.07.578830</a></p>
        <p class="abstract">Stress granules are cytoplasmic membraneless organelles that sequester proteins and non-translating mRNAs in response to various stressors. To assess the contributions of mRNA and RNA-binding proteins to stress granule formation, we use microinjection to deliver protein-free mRNA into the cytoplasm in a controlled manner. We demonstrate that mRNAs trigger stress granule formation through two mechanisms that are enhanced by the presence of G3BP1 and G3BP2. Low concentrations of in vitro transcribed mRNA activated protein kinase R (PKR), leading to phosphorylation and inhibition of the eukaryotic translation initiation factor eIF2 and stress granule formation. This was inhibited by replacing uridine with pseudouridine in the mRNA or by treating it with RNase III, which cleaves double-stranded RNA. High concentrations of mRNA triggered stress granule formation by a mechanism that was independent of PKR and enhanced by G3BP1/2, highlighting the importance of both protein-free mRNA and RNA-binding proteins in stress granule formation.

Graphical Abstract/Model

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=197 SRC=&#34;FIGDIR/small/578830v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (27K):
org.highwire.dtl.DTLVardef@19dc771org.highwire.dtl.DTLVardef@190ce98org.highwire.dtl.DTLVardef@1eb7aaeorg.highwire.dtl.DTLVardef@c35f86_HPS_FORMAT_FIGEXP  M_FIG C_FIG SummaryMicroinjected mRNA induces stress granules in mammalian cells by two G3BP1/2-dependent mechanisms: one requires the stress-sensing protein kinase PKR to phosphorylate the translation initiation factor eIF2, and the other is independent of PKR and phospho-eIF2 and acts when the cytoplasmic concentration of ribosome-free mRNA is increased acutely.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.05.578890">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.05.578890" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.05.578890">
        <p class="paperTitle">Nuclear pores safeguard the integrity of the nuclear envelope</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.05.578890" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.05.578890" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Taniguchi, R.; Orniacki, C.; Kreysing, J. P.; Zila, V.; Zimmerli, C. E.; Boehm, S.; Turonova, B.; Kraeusslich, H.-G.; Doye, V.; Beck, M.</p>
        <p class="info">Score: 25.2, Published: 2024-02-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.05.578890' target='https://doi.org/10.1101/2024.02.05.578890'> 10.1101/2024.02.05.578890</a></p>
        <p class="abstract">Nuclear pore complexes (NPCs) constitute giant channels within the nuclear envelope that mediate nucleocytoplasmic exchange. NPC diameter is thought to be regulated by nuclear envelope tension, but how such diameter changes are physiologically linked to cell differentiation, where mechanical properties of nuclei are remodeled and nuclear mechanosensing occurs, remains unstudied. Here we used cryo-electron tomography to show that NPCs dilate during differentiation of mouse embryonic stem cells into neural progenitors. In Nup133-deficient cells, which are known to display impaired neural differentiation, NPCs however fail to dilate. By analyzing the architectures of individual NPCs with template matching, we revealed that the Nup133-deficient NPCs are structurally heterogeneous and frequently disintegrate, resulting in the formation of large nuclear envelope openings. We propose that the elasticity of the NPC scaffold mechanically safeguards the nuclear envelope. Our studies provide a molecular explanation for how genetic perturbation of scaffolding components of macromolecular complexes causes tissue-specific phenotypes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579580">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579580" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579580">
        <p class="paperTitle">Selective Translation Orchestrates Key Signaling Pathways in Primed Pluripotency</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579580" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579580" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Okubo, C.; Nakamura, M.; Sato, M.; Shichino, Y.; Mito, M.; Takashima, Y.; Iwasaki, S.; Takahashi, K.</p>
        <p class="info">Score: 6.3, Published: 2024-02-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579580' target='https://doi.org/10.1101/2024.02.09.579580'> 10.1101/2024.02.09.579580</a></p>
        <p class="abstract">Pluripotent stem cell (PSC) identities, such as differentiation and infinite proliferation, have long been understood within the frameworks of transcription factor networks, epigenomes, and signal transduction, yet remain unclear and fragmented. Directing attention toward translational regulation, as a bridge between these events, promises to yield new insights into previously unexplained mechanisms. Our functional CRISPR interference screening-based approach revealed that EIF3D maintains primed pluripotency through selective translational regulation. The loss of EIF3D disrupts the balance of pluripotency-associated signaling pathways, impairing primed pluripotency. Moreover, we discovered that EIF3D ensures robust proliferation by controlling the translation of various p53 regulators, which maintain low p53 activity in the undifferentiated state. In this way, selective translation by EIF3D tunes the homeostasis of the primed pluripotency networks, ensuring the maintenance of an undifferentiated state with high proliferative potential. Therefore, this study establishes a paradigm for selective translational regulation as a defining feature of primed PSC identity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.05.578955">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.05.578955" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.05.578955">
        <p class="paperTitle">Ca2&#43;-driven cytoplasmic backflow secures spindle position in fertilized mouse eggs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.05.578955" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.05.578955" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Totsuka, T.; Ohsugi, M.</p>
        <p class="info">Score: 6.5, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.05.578955' target='https://doi.org/10.1101/2024.02.05.578955'> 10.1101/2024.02.05.578955</a></p>
        <p class="abstract">AbstractFertilization triggers hours-long Ca2&#43; oscillations in mammalian eggs, but the effects of repeated Ca2&#43; surges remain unclear. Here, we investigate spindle dynamics and its relationship with cytoplasmic streaming in fertilized mouse eggs. The spindle, initially parallel to the plasma membrane, rotates vertically, in accordance with previously reported results using artificially activated eggs. Intriguingly, it transiently reverses its rotation direction in synchrony with Ca2&#43; oscillations, regardless of artificially altered frequency. This effect results from cytoplasmic streaming, initially moving from spindle to egg center, displaying a Ca2&#43;-dependent backflow. Streaming also impacts spindle positioning, balancing spindle rotation and cortical localization maintenance. We provide evidence that Ca2&#43;-dependent cortical myosin II activation causes actomyosin contraction, leading to transient streaming towards non-contracting actin cap regions overlaying chromosomes. Our findings underscore the role of Ca2&#43; oscillations in maintaining spindle position in fertilized eggs, thereby ensuring highly asymmetric division and preservation of maternal stores in zygotes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.11.579865">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.11.579865" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.11.579865">
        <p class="paperTitle">Oncogenic STIL-mediated loss of BRCA1 functionality causes DNA damage and centrosome amplification</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.11.579865" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.11.579865" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sanghi, S.; Singh, P.</p>
        <p class="info">Score: 6.1, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.11.579865' target='https://doi.org/10.1101/2024.02.11.579865'> 10.1101/2024.02.11.579865</a></p>
        <p class="abstract">Centrosomes are major microtubule organizing centers in many animal cells and could also act as a communicator between nuclear-cytoplasmic signaling during DNA damages. Therefore, DNA damages cause an increase in centrosome number, however the molecular linkers remain elusive. STIL (SCL/TAL1 Interrupting locus) is a core centrosome organizing protein, which is overexpressed in different cancer types. In this work, we identify a frequently reported heterozygous missense mutation in STIL, which maps to its N-terminal located CR1 region (S76L). The oncogenic STIL-S76L loses interaction with the DNA damage response protein BRCA1 and exhibits a decrease in BRCA1 nuclear localization, which causes DNA damage and enhanced centrosome number. Interestingly, exogenous BRCA1 can rescue these phenotypes of STIL-S76L, thus, establishing it as an effector for the oncogenic STIL functions. These amplified centrosomes tend to cluster in the M phase and cause pseudo bipolar spindle organization, which is a promising target for anti-cancer drugs.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
