<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cell biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cell biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Psychosocial experiences are associated with human brain mitochondrial biology
Authors: Trumpff, C.; Monzel, A.; Sandi, C.; Menon, V.; Klein, H.-U.; Fujita, M.; Lee, A. J.; Petyuk, V. A.; Hurst, C.; Duong, D. A.; Seyfried, N.; Wingo, A.; Wingo, T. S.; Wang, Y.; Thambisetty, M.; Ferrucci, L.; Bennett, D. A.; De Jager, P.; Picard, M.
Score: 37.3, Published: 2023-10-06 DOI: 10.1101/2023.10.06.559575
Psychosocial experiences affect brain health and aging trajectories, but the molecular pathways underlying these associations remain unclear.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cell-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cell biology" />
<meta property="og:description" content="Psychosocial experiences are associated with human brain mitochondrial biology
Authors: Trumpff, C.; Monzel, A.; Sandi, C.; Menon, V.; Klein, H.-U.; Fujita, M.; Lee, A. J.; Petyuk, V. A.; Hurst, C.; Duong, D. A.; Seyfried, N.; Wingo, A.; Wingo, T. S.; Wang, Y.; Thambisetty, M.; Ferrucci, L.; Bennett, D. A.; De Jager, P.; Picard, M.
Score: 37.3, Published: 2023-10-06 DOI: 10.1101/2023.10.06.559575
Psychosocial experiences affect brain health and aging trajectories, but the molecular pathways underlying these associations remain unclear." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cell-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-18T10:38:41+00:00" />
<meta property="article:modified_time" content="2023-10-18T10:38:41+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cell biology"/>
<meta name="twitter:description" content="Psychosocial experiences are associated with human brain mitochondrial biology
Authors: Trumpff, C.; Monzel, A.; Sandi, C.; Menon, V.; Klein, H.-U.; Fujita, M.; Lee, A. J.; Petyuk, V. A.; Hurst, C.; Duong, D. A.; Seyfried, N.; Wingo, A.; Wingo, T. S.; Wang, Y.; Thambisetty, M.; Ferrucci, L.; Bennett, D. A.; De Jager, P.; Picard, M.
Score: 37.3, Published: 2023-10-06 DOI: 10.1101/2023.10.06.559575
Psychosocial experiences affect brain health and aging trajectories, but the molecular pathways underlying these associations remain unclear."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cell biology",
      "item": "https://trxiv.yorks0n.com/posts/cell-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cell biology",
  "name": "cell biology",
  "description": "Psychosocial experiences are associated with human brain mitochondrial biology\nAuthors: Trumpff, C.; Monzel, A.; Sandi, C.; Menon, V.; Klein, H.-U.; Fujita, M.; Lee, A. J.; Petyuk, V. A.; Hurst, C.; Duong, D. A.; Seyfried, N.; Wingo, A.; Wingo, T. S.; Wang, Y.; Thambisetty, M.; Ferrucci, L.; Bennett, D. A.; De Jager, P.; Picard, M.\nScore: 37.3, Published: 2023-10-06 DOI: 10.1101/2023.10.06.559575\nPsychosocial experiences affect brain health and aging trajectories, but the molecular pathways underlying these associations remain unclear.",
  "keywords": [
    
  ],
  "articleBody": " Psychosocial experiences are associated with human brain mitochondrial biology\nAuthors: Trumpff, C.; Monzel, A.; Sandi, C.; Menon, V.; Klein, H.-U.; Fujita, M.; Lee, A. J.; Petyuk, V. A.; Hurst, C.; Duong, D. A.; Seyfried, N.; Wingo, A.; Wingo, T. S.; Wang, Y.; Thambisetty, M.; Ferrucci, L.; Bennett, D. A.; De Jager, P.; Picard, M.\nScore: 37.3, Published: 2023-10-06 DOI: 10.1101/2023.10.06.559575\nPsychosocial experiences affect brain health and aging trajectories, but the molecular pathways underlying these associations remain unclear. Normal brain function relies heavily on energy transformation by mitochondria oxidative phosphorylation (OxPhos), and two main lines of evidence bi-directionally link mitochondria as both targets and drivers of psychosocial experiences. On the one hand, chronic stress exposure and possibly mood states alter multiple aspects of mitochondrial biology; and on the other hand, functional variations in mitochondrial OxPhos capacity alter social behavior, stress reactivity, and mood. However, knowledge on whether positive or negative psychosocial exposures and experiences are linked to mitochondrial biology in the human brain is currently unknown. By combining longitudinal antemortem assessments of psychosocial factors with postmortem brain (dorsolateral prefrontal cortex) proteomics in older adults, we found that positive experiences (e.g. higher well-being) are linked to greater abundance of the mitochondrial OxPhos machinery, whereas negative experiences (e.g. higher negative mood) are linked to lower OxPhos protein content. Combined, psychosocial factors explained 18% of the variance in the abundance of OxPhos complex I, the primary biochemical entry point that energizes brain mitochondria. To increase the sensitivity of our approach, we next interrogated mitochondrial psychobiological associations in specific neuronal and non-neuronal brain cells with single-nucleus RNA sequencing. These results revealed strong cell type specific associations, particularly between positive psychosocial experiences and molecular mitochondrial phenotypes in glial cells, whereas neurons tended to show opposite associations. Accordingly, in bulk transcriptomic analyses where all cells are pooled, these RNA-based associations were masked. Thus, our results highlight the likely underestimation of effect sizes in bulk brain tissues, and document novel cell type specific mitochondrial psychobiological associations in the human brain. Cell type specific mitochondrial recalibrations represent a potential psychobiological pathway linking positive and negative psychosocial experiences to human brain biology. Significance statementPsychosocial experiences predict health trajectories, but the underlying mechanism remains unclear. We found that positive psychosocial experiences are linked to greater abundance of the mitochondrial energy transformation machinery, whereas negative experiences are linked to lower abundance. Overall, we found that psychosocial experiences explain 18% of the variance in abundance of complex I proteins, the main entry point of the mitochondrial oxidative phosphorylation (OxPhos) system. At single-cell resolution using single nucleus transcriptomics, positive psychosocial experiences were particularly related to glial cell mitochondrial phenotypes. Opposite associations between glial cells and neurons were naturally masked in bulk transcriptomic analyses. Our results suggest that mitochondrial recalibrations in specific brain cell types may represent a potential psychobiological pathway linking psychosocial experiences to human brain health.\nGenetic transformation of the frog-killing chytrid fungus Batrachochytrium dendrobatidis\nAuthors: Kalinka, E.; Brody, S.; Swafford, A.; Medina, E.; Fritz-Laylin, L.\nScore: 23.8, Published: 2023-10-17 DOI: 10.1101/2023.10.17.561934\nBatrachochytrium dendrobatidis (Bd), a causative agent of chytridiomycosis, is decimating amphibian populations around the world. Bd belongs to the chytrid lineage, a group of early-diverging fungi that are widely used to study fungal evolution. Like all chytrids, Bd develops from a motile form into a sessile, growth form, a transition that involves drastic changes in its cytoskeletal architecture. Efforts to study Bd cell biology, development, and pathogenicity have been limited by the lack of genetic tools with which to test hypotheses about underlying molecular mechanisms. Here, we report the development of a transient genetic transformation system for Bd. We used electroporation to deliver exogenous DNA into Bd cells and detected transgene expression for up to three generations under both heterologous and native promoters. We also adapted the transformation protocol for selection using an antibiotic resistance marker. Finally, we used this system to express fluorescent protein fusions and, as a proof of concept, expressed a genetically encoded probe for the actin cytoskeleton. Using live-cell imaging, we visualized the distribution and dynamics of polymerized actin at each stage of the Bd life cycle, as well as during key developmental transitions. This transformation system allows, for the first time, direct testing of key hypotheses regarding mechanisms of Bd pathogenesis. This technology also paves the way for answering fundamental questions of chytrid cell, developmental, and evolutionary biology.\nRepression of mRNA translation initiation by GIGYF1 via blocking the eIF3-eIF4G1 interaction\nAuthors: Choi, J.-H.; Luo, J.; Hesketh, G. G.; Guo, S.; Pistofidis, A.; Ladak, R. J.; An, Y.; Alain, T.; Schmeing, M.; Gingras, A.-C.; Duchaine, T.; Zhang, X.; Sonenberg, N.; Jafarnejad, S. M.\nScore: 14.9, Published: 2023-10-15 DOI: 10.1101/2023.10.14.562322\nViruses commonly interfere with the function of the eukaryotic translation initiation factor 4G1 (eIF4G1), a pivotal factor in the recruitment of the eIF3 complex and ribosome to the mRNA. This results in the inhibition of general host protein synthesis and redirecting ribosomes toward viral mRNAs. Certain viruses also selectively repress the translation of mRNAs involved in the host antiviral response. GIGYF2 and its interacting cap-binding protein 4EHP enable the transcript-specific repression of mRNA translation mediated by microRNAs and RNA-binding proteins (RBPs). RNA viruses, such as SARS-CoV-2, exploit the GIGYF2/4EHP complex to selectively repress the translation of transcripts such as Ifnb1 mRNA, which encodes the antiviral cytokine Interferon {beta} (IFN-{beta}). Herein, we reveal that GIGYF1, a paralogue of GIGYF2, robustly represses cellular mRNA translation through a distinct mechanism independent of 4EHP. Upon recruitment to a target mRNA by RBPs, the C-terminal region of GIGYF1 binds to subunits of eIF3 at the interaction interface of eIF3-eIF4G1. This disrupts the recruitment of eIF3 to the mRNA by eIF4G1, resulting in mRNA-specific translational repression. This mechanism exerts profound influences on the host cell's response to viral infection. Depletion of GIGYF1 induces a robust immune response by derepressing Ifnb1 mRNA translation. Overall, our study highlights a unique mechanism of translational regulation by GIGYF1 that involves sequestering eIF3 and abrogating its binding to eIF4G1. This mechanism can be utilized by RBPs that interact with GIGYF1 to specifically repress the translation of their target mRNAs, significantly affecting critical biological processes, including host-pathogen interactions.\nThe H3.3 K36M oncohistone disrupts the establishment of epigenetic memory through loss of DNA methylation\nAuthors: Sinha, J.; Nickels, J. F.; Thurm, A. R.; Ludwig, C. H.; Archibald, B. N.; Hinks, M. M.; Wan, J.; Fang, D.; Bintu, L.\nScore: 11.4, Published: 2023-10-14 DOI: 10.1101/2023.10.13.562147\nHistone H3.3 is frequently mutated in cancers, with the lysine 36 to methionine mutation (K36M) being a hallmark of chondroblastomas. While it is known that H3.3K36M changes the cellular epigenetic landscape, it remains unclear how it affects the dynamics of gene expression. Here, we use a synthetic reporter to measure the effect of H3.3K36M on silencing and epigenetic memory after recruitment of KRAB: a member of the largest class of human repressors, commonly used in synthetic biology, and associated with H3K9me3. We find that H3.3K36M, which decreases H3K36 methylation, leads to a decrease in epigenetic memory and promoter methylation weeks after KRAB release. We propose a new model for establishment and maintenance of epigenetic memory, where H3K36 methylation is necessary to convert H3K9me3 domains into DNA methylation for stable epigenetic memory. Our quantitative model can inform oncogenic mechanisms and guide development of epigenetic editing tools.\nmTOR inhibits autophagy to facilitate cell swelling and rapid wound repair\nAuthors: Scepanovic, G.; Fernandez-Gonzalez, R.\nScore: 9.4, Published: 2023-10-10 DOI: 10.1101/2023.10.10.561758\nEmbryonic wounds repair rapidly, with no inflammation or scarring. Embryonic wound healing is driven by collective cell movements facilitated by the swelling of the cells adjacent to the wound. The mechanistic target of rapamycin complex 1 (mTORC1) is often associated with cell growth. We found that disrupting mTORC1 signalling prevented cell swelling and slowed down wound repair. Catabolic processes, such as autophagy, can inhibit cell growth. Using five-dimensional time-lapse microscopy, as well as pharmacological and genetic manipulations, we demonstrated that the number of autophagosomes decreased during wound repair, suggesting that autophagy must be tightly regulated for rapid wound healing. Quantitative image analysis showed that mTOR inhibition increased autophagy, and that activating autophagy prevented cell swelling and slowed down embryonic wound closure. Finally, reducing autophagy in embryos in which mTORC1 signalling was disrupted rescued rapid wound repair. Together, our results show that mTORC1 activation upon wounding negatively regulates autophagy, allowing cells to increase their volumes to facilitate rapid wound healing.\nCirculating senescent myeloid cells drive blood brain barrier breakdown and neurodegeneration\nAuthors: Wilk, C. M.; Cathomas, F.; Torok, O.; Le Berichel, J.; Park, M. D.; Heaton, G. R.; Hamon, P.; Troncoso, L.; Scull, B. P.; Dangoor, D.; Silvin, A.; Fleischmann, R.; Belabed, M.; Lin, H.; Taouli, E. M.; Boettcher, S.; Manz, M. G.; Kofler, J. K.; Yue, Z.; Lira, S. A.; Ginhoux, F.; Crary, J. F.; McClain, K. L.; Picarsic, J. L.; Russo, S. J.; Allen, C. E.; Merad, M.\nScore: 9.2, Published: 2023-10-11 DOI: 10.1101/2023.10.10.561744\nNeurodegenerative diseases (ND) are characterized by progressive loss of neuronal function. Mechanisms of ND pathogenesis are incompletely understood, hampering the development of effective therapies. Langerhans cell histiocytosis (LCH) is an inflammatory neoplastic disorder caused by hematopoietic progenitors expressing MAPK activating mutations that differentiate into senescent myeloid cells that drive lesion formation. Some patients with LCH subsequently develop progressive and incurable neurodegeneration (LCH-ND). Here, we show that LCH-ND is caused by myeloid cells that are clonal with peripheral LCH cells. We discovered that circulating BRAFV600E+ myeloid cells cause the breakdown of the blood-brain barrier (BBB), enhancing migration into the brain parenchyma where they differentiate into senescent, inflammatory CD11a+ macrophages that accumulate in the brainstem and cerebellum. Blocking MAPK activity and senescence programs reduced parenchymal infiltration, neuroinflammation, neuronal damage and improved neurological outcome in preclinical LCH-ND. MAPK activation and senescence programs in circulating myeloid cells represent novel and targetable mechanisms of ND.\nThe fatty liver disease-causing protein PNPLA3-I148M alters lipid droplet-Golgi dynamics\nAuthors: Sherman, D. J.; Liu, L.; Mamrosh, J. L.; Xie, J.; Ferbas, J.; Lomenick, B.; Ladinsky, M. S.; Verma, R.; Rulifson, I. C.; Deshaies, R. J.\nScore: 7.4, Published: 2023-10-14 DOI: 10.1101/2023.10.13.562302\nNon-alcoholic fatty liver disease (NAFLD), recently renamed metabolic dysfunction-associated steatotic liver disease (MASLD), is a progressive metabolic disorder that begins with aberrant triglyceride accumulation in the liver and can lead to cirrhosis and cancer. A common variant in the gene PNPLA3, encoding the protein PNPLA3-I148M, is the strongest known genetic risk factor for MASLD to date. Despite its discovery twenty years ago, the function of PNPLA3, and now the role of PNPLA3-I148M, remain unclear. In this study, we sought to dissect the biogenesis of PNPLA3 and PNPLA3-I148M and characterize changes induced by endogenous expression of the disease-causing variant. Contrary to bioinformatic predictions and prior studies with overexpressed proteins, we demonstrate here that PNPLA3 and PNPLA3-I148M are not endoplasmic reticulum-resident transmembrane proteins. To identify their intracellular associations, we generated a paired set of isogenic human hepatoma cells expressing PNPLA3 and PNPLA3-I148M at endogenous levels. Both proteins were enriched in lipid droplet, Golgi, and endosomal fractions. Purified PNPLA3 and PNPLA3-I148M proteins associated with phosphoinositides commonly found in these compartments. Despite a similar fractionation pattern as the wild-type variant, PNPLA3-I148M induced morphological changes in the Golgi apparatus, including increased lipid droplet-Golgi contact sites, which were also observed in I148M-expressing primary human patient hepatocytes. In addition to lipid droplet accumulation, PNPLA3-I148M expression caused significant proteomic and transcriptomic changes that resembled all stages of liver disease. Cumulatively, we validate an endogenous human cellular system for investigating PNPLA3-I148M biology and identify the Golgi apparatus as a central hub of PNPLA3-I148M-driven cellular change.\nProteomic Analysis Reveals a PLK1-Dependent G2/M Degradation Program and Links PKA-AKAP2 to Cell Cycle Control\nAuthors: Mouery, R. D.; Hsu, C.; Bonacci, T.; Bolhuis, D. L.; Wang, X.; Mills, C. A.; Toomer, E. D.; Canterbury, O. G.; Robertson, K. C.; Branigan, T. B.; Brown, N. G.; Herring, L. E.; Emanuele, M.\nScore: 6.5, Published: 2023-10-12 DOI: 10.1101/2023.10.11.561963\nTargeted protein degradation by the ubiquitin-proteasome system is an essential mechanism regulating cellular division. The kinase PLK1 coordinates protein degradation at the G2/M phase of the cell cycle by promoting the binding of substrates to the E3 ubiquitin ligase SCF{beta}TrCP. However, the magnitude to which PLK1 shapes the mitotic proteome has not been characterized. Combining deep, quantitative proteomics with pharmacologic PLK1 inhibition (PLK1i), we identified more than 200 proteins whose abundances were increased by PLK1i at G2/M. We validate many new PLK1-regulated proteins, including several substrates of the cell cycle E3 SCFCyclin F, demonstrating that PLK1 promotes proteolysis through at least two distinct SCF-family E3 ligases. Further, we found that the protein kinase A anchoring protein AKAP2 is cell cycle regulated and that its mitotic degradation is dependent on the PLK1/{beta}TrCP-signaling axis. Interactome analysis revealed that the strongest interactors of AKAP2 function in signaling networks regulating proliferation, including MAPK, AKT, and Hippo. Altogether, our data demonstrate that PLK1 coordinates a widespread program of protein breakdown at G2/M. We propose that dynamic proteolytic changes mediated by PLK1 integrate proliferative signals with the core cell cycle machinery during cell division. This has potential implications in malignancies where PLK1 is aberrantly regulated.\nSTING induces LUBAC-mediated synthesis of linear ubiquitin chains to stimulate innate immune signaling\nAuthors: Fischer, T. D.; Bunker, E. N.; Zhu, P.-P.; Le Guerroue, F.; Dominguez-Martin, E.; Scavone, F.; Cohen, R. E.; Yao, T.; Wang, Y.; Werner, A.; Youle, R. J.\nScore: 5.4, Published: 2023-10-16 DOI: 10.1101/2023.10.14.562349\nSTING activation by cyclic dinucleotides in mammals induces interferon- and NFkB -related gene expression, and the lipidation of LC3B at Golgi membranes. While mechanisms of the interferon response are well understood, the mechanisms of NFkB activation mediated by STING remain unclear. We report that STING activation induces K63- and M1-linked/linear ubiquitin chain formation at LC3B-associated Golgi membranes. Loss of the LUBAC E3 ubiquitin ligase prevents formation of linear, but not K63-linked ubiquitin chains or STING activation and inhibits STING-induced NFkB and IRF3-mediated signaling in monocytic THP1 cells. The proton channel activity of STING is also important for both K63 and linear ubiquitin chain formation, and NFkB- and interferon-related gene expression. Thus, LUBAC synthesis of linear ubiquitin chains regulates STING-mediated innate immune signaling.\nNanobodies against SARS-CoV-2 non-structural protein Nsp9 inhibit viral replication by targeting innate immunity\nAuthors: Venit, T.; Blavier, J.; Maseko, S. B.; Shu, S.; Espada, L.; Breunig, C.; Holthoff, H.-P.; Desbordes, S. C.; Lohse, M.; Esposito, G.; Twizere, J.-C.; Percipalle, P.\nScore: 5.2, Published: 2023-10-13 DOI: 10.1101/2023.10.12.561992\nNanobodies are emerging as critical tools for drug design. Several have been recently created to serve as inhibitors of SARS-Cov-2 entry in the host cell by targeting surface-exposed Spike protein. However, due to the high frequency of mutations that affect Spike, these nanobodies may not target it to their full potential and as a consequence, inhibition of viral entry may not be efficient. Here we have established a pipeline that instead targets highly conserved viral proteins that are made only after viral entry into the host cell when the SARS-Cov-2 RNA-based genome is translated. As proof of principle, we designed nanobodies against the SARS-CoV-2 non-structural protein Nsp9, required for viral genome replication. To find out if this strategy efficiently blocks viral replication, one of these anti-Nsp9 nanobodies, 2NSP23, previously characterized using immunoassays and NMR spectroscopy for epitope mapping, was encapsulated into lipid nanoparticles (LNP) as mRNA. We show that this nanobody, hereby referred to as LNP-mRNA- 2NSP23, is internalized and translated in HEK293 cells. We next infected HEK293-ACE2 cells with multiple SARS-CoV-2 variants and subjected them to LNP-mRNA-2NSP23 treatment. Analysis of total RNA isolated from infected cells treated or untreated with LNP-mRNA-2NSP23 using qPCR and RNA deep sequencing shows that the LNP-mRNA-2NSP23 nanobody protects HEK293-ACE2 cells and suppresses replication of several SARS-CoV-2 variants. These observations indicate that following translation, the nanobody 2NSP23 inhibits viral replication by targeting Nsp9 in living cells. We speculate that LNP-mRNA-2NSP23 may be translated into an innovative technology to generate novel antiviral drugs highly efficient across coronaviruses.\n",
  "wordCount" : "2635",
  "inLanguage": "en",
  "datePublished": "2023-10-18T10:38:41Z",
  "dateModified": "2023-10-18T10:38:41Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cell-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cell biology
    </h1>
    <div class="post-meta"><span>updated on October 18, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.559575">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.559575" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.559575">
        <p class="paperTitle">Psychosocial experiences are associated with human brain mitochondrial biology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.559575" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.559575" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Trumpff, C.; Monzel, A.; Sandi, C.; Menon, V.; Klein, H.-U.; Fujita, M.; Lee, A. J.; Petyuk, V. A.; Hurst, C.; Duong, D. A.; Seyfried, N.; Wingo, A.; Wingo, T. S.; Wang, Y.; Thambisetty, M.; Ferrucci, L.; Bennett, D. A.; De Jager, P.; Picard, M.</p>
        <p class="info">Score: 37.3, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.559575' target='https://doi.org/10.1101/2023.10.06.559575'> 10.1101/2023.10.06.559575</a></p>
        <p class="abstract">Psychosocial experiences affect brain health and aging trajectories, but the molecular pathways underlying these associations remain unclear. Normal brain function relies heavily on energy transformation by mitochondria oxidative phosphorylation (OxPhos), and two main lines of evidence bi-directionally link mitochondria as both targets and drivers of psychosocial experiences. On the one hand, chronic stress exposure and possibly mood states alter multiple aspects of mitochondrial biology; and on the other hand, functional variations in mitochondrial OxPhos capacity alter social behavior, stress reactivity, and mood. However, knowledge on whether positive or negative psychosocial exposures and experiences are linked to mitochondrial biology in the human brain is currently unknown. By combining longitudinal antemortem assessments of psychosocial factors with postmortem brain (dorsolateral prefrontal cortex) proteomics in older adults, we found that positive experiences (e.g. higher well-being) are linked to greater abundance of the mitochondrial OxPhos machinery, whereas negative experiences (e.g. higher negative mood) are linked to lower OxPhos protein content. Combined, psychosocial factors explained 18% of the variance in the abundance of OxPhos complex I, the primary biochemical entry point that energizes brain mitochondria. To increase the sensitivity of our approach, we next interrogated mitochondrial psychobiological associations in specific neuronal and non-neuronal brain cells with single-nucleus RNA sequencing. These results revealed strong cell type specific associations, particularly between positive psychosocial experiences and molecular mitochondrial phenotypes in glial cells, whereas neurons tended to show opposite associations. Accordingly, in bulk transcriptomic analyses where all cells are pooled, these RNA-based associations were masked. Thus, our results highlight the likely underestimation of effect sizes in bulk brain tissues, and document novel cell type specific mitochondrial psychobiological associations in the human brain. Cell type specific mitochondrial recalibrations represent a potential psychobiological pathway linking positive and negative psychosocial experiences to human brain biology.

Significance statementPsychosocial experiences predict health trajectories, but the underlying mechanism remains unclear. We found that positive psychosocial experiences are linked to greater abundance of the mitochondrial energy transformation machinery, whereas negative experiences are linked to lower abundance. Overall, we found that psychosocial experiences explain 18% of the variance in abundance of complex I proteins, the main entry point of the mitochondrial oxidative phosphorylation (OxPhos) system. At single-cell resolution using single nucleus transcriptomics, positive psychosocial experiences were particularly related to glial cell mitochondrial phenotypes. Opposite associations between glial cells and neurons were naturally masked in bulk transcriptomic analyses. Our results suggest that mitochondrial recalibrations in specific brain cell types may represent a potential psychobiological pathway linking psychosocial experiences to human brain health.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.561934">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.561934" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.561934">
        <p class="paperTitle">Genetic transformation of the frog-killing chytrid fungus Batrachochytrium dendrobatidis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.561934" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.561934" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kalinka, E.; Brody, S.; Swafford, A.; Medina, E.; Fritz-Laylin, L.</p>
        <p class="info">Score: 23.8, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.561934' target='https://doi.org/10.1101/2023.10.17.561934'> 10.1101/2023.10.17.561934</a></p>
        <p class="abstract">Batrachochytrium dendrobatidis (Bd), a causative agent of chytridiomycosis, is decimating amphibian populations around the world. Bd belongs to the chytrid lineage, a group of early-diverging fungi that are widely used to study fungal evolution. Like all chytrids, Bd develops from a motile form into a sessile, growth form, a transition that involves drastic changes in its cytoskeletal architecture. Efforts to study Bd cell biology, development, and pathogenicity have been limited by the lack of genetic tools with which to test hypotheses about underlying molecular mechanisms. Here, we report the development of a transient genetic transformation system for Bd. We used electroporation to deliver exogenous DNA into Bd cells and detected transgene expression for up to three generations under both heterologous and native promoters. We also adapted the transformation protocol for selection using an antibiotic resistance marker. Finally, we used this system to express fluorescent protein fusions and, as a proof of concept, expressed a genetically encoded probe for the actin cytoskeleton. Using live-cell imaging, we visualized the distribution and dynamics of polymerized actin at each stage of the Bd life cycle, as well as during key developmental transitions. This transformation system allows, for the first time, direct testing of key hypotheses regarding mechanisms of Bd pathogenesis. This technology also paves the way for answering fundamental questions of chytrid cell, developmental, and evolutionary biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.14.562322">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.14.562322" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.14.562322">
        <p class="paperTitle">Repression of mRNA translation initiation by GIGYF1 via blocking the eIF3-eIF4G1 interaction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.14.562322" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.14.562322" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Choi, J.-H.; Luo, J.; Hesketh, G. G.; Guo, S.; Pistofidis, A.; Ladak, R. J.; An, Y.; Alain, T.; Schmeing, M.; Gingras, A.-C.; Duchaine, T.; Zhang, X.; Sonenberg, N.; Jafarnejad, S. M.</p>
        <p class="info">Score: 14.9, Published: 2023-10-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.14.562322' target='https://doi.org/10.1101/2023.10.14.562322'> 10.1101/2023.10.14.562322</a></p>
        <p class="abstract">Viruses commonly interfere with the function of the eukaryotic translation initiation factor 4G1 (eIF4G1), a pivotal factor in the recruitment of the eIF3 complex and ribosome to the mRNA. This results in the inhibition of general host protein synthesis and redirecting ribosomes toward viral mRNAs. Certain viruses also selectively repress the translation of mRNAs involved in the host antiviral response. GIGYF2 and its interacting cap-binding protein 4EHP enable the transcript-specific repression of mRNA translation mediated by microRNAs and RNA-binding proteins (RBPs). RNA viruses, such as SARS-CoV-2, exploit the GIGYF2/4EHP complex to selectively repress the translation of transcripts such as Ifnb1 mRNA, which encodes the antiviral cytokine Interferon {beta} (IFN-{beta}). Herein, we reveal that GIGYF1, a paralogue of GIGYF2, robustly represses cellular mRNA translation through a distinct mechanism independent of 4EHP. Upon recruitment to a target mRNA by RBPs, the C-terminal region of GIGYF1 binds to subunits of eIF3 at the interaction interface of eIF3-eIF4G1. This disrupts the recruitment of eIF3 to the mRNA by eIF4G1, resulting in mRNA-specific translational repression. This mechanism exerts profound influences on the host cell&#39;s response to viral infection. Depletion of GIGYF1 induces a robust immune response by derepressing Ifnb1 mRNA translation. Overall, our study highlights a unique mechanism of translational regulation by GIGYF1 that involves sequestering eIF3 and abrogating its binding to eIF4G1. This mechanism can be utilized by RBPs that interact with GIGYF1 to specifically repress the translation of their target mRNAs, significantly affecting critical biological processes, including host-pathogen interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562147">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562147" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562147">
        <p class="paperTitle">The H3.3 K36M oncohistone disrupts the establishment of epigenetic memory through loss of DNA methylation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562147" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562147" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sinha, J.; Nickels, J. F.; Thurm, A. R.; Ludwig, C. H.; Archibald, B. N.; Hinks, M. M.; Wan, J.; Fang, D.; Bintu, L.</p>
        <p class="info">Score: 11.4, Published: 2023-10-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562147' target='https://doi.org/10.1101/2023.10.13.562147'> 10.1101/2023.10.13.562147</a></p>
        <p class="abstract">Histone H3.3 is frequently mutated in cancers, with the lysine 36 to methionine mutation (K36M) being a hallmark of chondroblastomas. While it is known that H3.3K36M changes the cellular epigenetic landscape, it remains unclear how it affects the dynamics of gene expression. Here, we use a synthetic reporter to measure the effect of H3.3K36M on silencing and epigenetic memory after recruitment of KRAB: a member of the largest class of human repressors, commonly used in synthetic biology, and associated with H3K9me3. We find that H3.3K36M, which decreases H3K36 methylation, leads to a decrease in epigenetic memory and promoter methylation weeks after KRAB release. We propose a new model for establishment and maintenance of epigenetic memory, where H3K36 methylation is necessary to convert H3K9me3 domains into DNA methylation for stable epigenetic memory. Our quantitative model can inform oncogenic mechanisms and guide development of epigenetic editing tools.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561758">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561758" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561758">
        <p class="paperTitle">mTOR inhibits autophagy to facilitate cell swelling and rapid wound repair</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561758" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561758" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Scepanovic, G.; Fernandez-Gonzalez, R.</p>
        <p class="info">Score: 9.4, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561758' target='https://doi.org/10.1101/2023.10.10.561758'> 10.1101/2023.10.10.561758</a></p>
        <p class="abstract">Embryonic wounds repair rapidly, with no inflammation or scarring. Embryonic wound healing is driven by collective cell movements facilitated by the swelling of the cells adjacent to the wound. The mechanistic target of rapamycin complex 1 (mTORC1) is often associated with cell growth. We found that disrupting mTORC1 signalling prevented cell swelling and slowed down wound repair. Catabolic processes, such as autophagy, can inhibit cell growth. Using five-dimensional time-lapse microscopy, as well as pharmacological and genetic manipulations, we demonstrated that the number of autophagosomes decreased during wound repair, suggesting that autophagy must be tightly regulated for rapid wound healing. Quantitative image analysis showed that mTOR inhibition increased autophagy, and that activating autophagy prevented cell swelling and slowed down embryonic wound closure. Finally, reducing autophagy in embryos in which mTORC1 signalling was disrupted rescued rapid wound repair. Together, our results show that mTORC1 activation upon wounding negatively regulates autophagy, allowing cells to increase their volumes to facilitate rapid wound healing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561744">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561744" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561744">
        <p class="paperTitle">Circulating senescent myeloid cells drive blood brain barrier breakdown and neurodegeneration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561744" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561744" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wilk, C. M.; Cathomas, F.; Torok, O.; Le Berichel, J.; Park, M. D.; Heaton, G. R.; Hamon, P.; Troncoso, L.; Scull, B. P.; Dangoor, D.; Silvin, A.; Fleischmann, R.; Belabed, M.; Lin, H.; Taouli, E. M.; Boettcher, S.; Manz, M. G.; Kofler, J. K.; Yue, Z.; Lira, S. A.; Ginhoux, F.; Crary, J. F.; McClain, K. L.; Picarsic, J. L.; Russo, S. J.; Allen, C. E.; Merad, M.</p>
        <p class="info">Score: 9.2, Published: 2023-10-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561744' target='https://doi.org/10.1101/2023.10.10.561744'> 10.1101/2023.10.10.561744</a></p>
        <p class="abstract">Neurodegenerative diseases (ND) are characterized by progressive loss of neuronal function. Mechanisms of ND pathogenesis are incompletely understood, hampering the development of effective therapies. Langerhans cell histiocytosis (LCH) is an inflammatory neoplastic disorder caused by hematopoietic progenitors expressing MAPK activating mutations that differentiate into senescent myeloid cells that drive lesion formation. Some patients with LCH subsequently develop progressive and incurable neurodegeneration (LCH-ND). Here, we show that LCH-ND is caused by myeloid cells that are clonal with peripheral LCH cells. We discovered that circulating BRAFV600E&#43; myeloid cells cause the breakdown of the blood-brain barrier (BBB), enhancing migration into the brain parenchyma where they differentiate into senescent, inflammatory CD11a&#43; macrophages that accumulate in the brainstem and cerebellum. Blocking MAPK activity and senescence programs reduced parenchymal infiltration, neuroinflammation, neuronal damage and improved neurological outcome in preclinical LCH-ND. MAPK activation and senescence programs in circulating myeloid cells represent novel and targetable mechanisms of ND.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562302">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562302" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562302">
        <p class="paperTitle">The fatty liver disease-causing protein PNPLA3-I148M alters lipid droplet-Golgi dynamics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562302" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562302" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sherman, D. J.; Liu, L.; Mamrosh, J. L.; Xie, J.; Ferbas, J.; Lomenick, B.; Ladinsky, M. S.; Verma, R.; Rulifson, I. C.; Deshaies, R. J.</p>
        <p class="info">Score: 7.4, Published: 2023-10-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562302' target='https://doi.org/10.1101/2023.10.13.562302'> 10.1101/2023.10.13.562302</a></p>
        <p class="abstract">Non-alcoholic fatty liver disease (NAFLD), recently renamed metabolic dysfunction-associated steatotic liver disease (MASLD), is a progressive metabolic disorder that begins with aberrant triglyceride accumulation in the liver and can lead to cirrhosis and cancer. A common variant in the gene PNPLA3, encoding the protein PNPLA3-I148M, is the strongest known genetic risk factor for MASLD to date. Despite its discovery twenty years ago, the function of PNPLA3, and now the role of PNPLA3-I148M, remain unclear. In this study, we sought to dissect the biogenesis of PNPLA3 and PNPLA3-I148M and characterize changes induced by endogenous expression of the disease-causing variant. Contrary to bioinformatic predictions and prior studies with overexpressed proteins, we demonstrate here that PNPLA3 and PNPLA3-I148M are not endoplasmic reticulum-resident transmembrane proteins. To identify their intracellular associations, we generated a paired set of isogenic human hepatoma cells expressing PNPLA3 and PNPLA3-I148M at endogenous levels. Both proteins were enriched in lipid droplet, Golgi, and endosomal fractions. Purified PNPLA3 and PNPLA3-I148M proteins associated with phosphoinositides commonly found in these compartments. Despite a similar fractionation pattern as the wild-type variant, PNPLA3-I148M induced morphological changes in the Golgi apparatus, including increased lipid droplet-Golgi contact sites, which were also observed in I148M-expressing primary human patient hepatocytes. In addition to lipid droplet accumulation, PNPLA3-I148M expression caused significant proteomic and transcriptomic changes that resembled all stages of liver disease. Cumulatively, we validate an endogenous human cellular system for investigating PNPLA3-I148M biology and identify the Golgi apparatus as a central hub of PNPLA3-I148M-driven cellular change.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.11.561963">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.11.561963" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.11.561963">
        <p class="paperTitle">Proteomic Analysis Reveals a PLK1-Dependent G2/M Degradation Program and Links PKA-AKAP2 to Cell Cycle Control</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.11.561963" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.11.561963" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mouery, R. D.; Hsu, C.; Bonacci, T.; Bolhuis, D. L.; Wang, X.; Mills, C. A.; Toomer, E. D.; Canterbury, O. G.; Robertson, K. C.; Branigan, T. B.; Brown, N. G.; Herring, L. E.; Emanuele, M.</p>
        <p class="info">Score: 6.5, Published: 2023-10-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.11.561963' target='https://doi.org/10.1101/2023.10.11.561963'> 10.1101/2023.10.11.561963</a></p>
        <p class="abstract">Targeted protein degradation by the ubiquitin-proteasome system is an essential mechanism regulating cellular division. The kinase PLK1 coordinates protein degradation at the G2/M phase of the cell cycle by promoting the binding of substrates to the E3 ubiquitin ligase SCF{beta}TrCP. However, the magnitude to which PLK1 shapes the mitotic proteome has not been characterized. Combining deep, quantitative proteomics with pharmacologic PLK1 inhibition (PLK1i), we identified more than 200 proteins whose abundances were increased by PLK1i at G2/M. We validate many new PLK1-regulated proteins, including several substrates of the cell cycle E3 SCFCyclin F, demonstrating that PLK1 promotes proteolysis through at least two distinct SCF-family E3 ligases. Further, we found that the protein kinase A anchoring protein AKAP2 is cell cycle regulated and that its mitotic degradation is dependent on the PLK1/{beta}TrCP-signaling axis. Interactome analysis revealed that the strongest interactors of AKAP2 function in signaling networks regulating proliferation, including MAPK, AKT, and Hippo. Altogether, our data demonstrate that PLK1 coordinates a widespread program of protein breakdown at G2/M. We propose that dynamic proteolytic changes mediated by PLK1 integrate proliferative signals with the core cell cycle machinery during cell division. This has potential implications in malignancies where PLK1 is aberrantly regulated.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.14.562349">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.14.562349" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.14.562349">
        <p class="paperTitle">STING induces LUBAC-mediated synthesis of linear ubiquitin chains to stimulate innate immune signaling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.14.562349" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.14.562349" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fischer, T. D.; Bunker, E. N.; Zhu, P.-P.; Le Guerroue, F.; Dominguez-Martin, E.; Scavone, F.; Cohen, R. E.; Yao, T.; Wang, Y.; Werner, A.; Youle, R. J.</p>
        <p class="info">Score: 5.4, Published: 2023-10-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.14.562349' target='https://doi.org/10.1101/2023.10.14.562349'> 10.1101/2023.10.14.562349</a></p>
        <p class="abstract">STING activation by cyclic dinucleotides in mammals induces interferon- and NFkB -related gene expression, and the lipidation of LC3B at Golgi membranes. While mechanisms of the interferon response are well understood, the mechanisms of NFkB activation mediated by STING remain unclear. We report that STING activation induces K63- and M1-linked/linear ubiquitin chain formation at LC3B-associated Golgi membranes. Loss of the LUBAC E3 ubiquitin ligase prevents formation of linear, but not K63-linked ubiquitin chains or STING activation and inhibits STING-induced NFkB and IRF3-mediated signaling in monocytic THP1 cells. The proton channel activity of STING is also important for both K63 and linear ubiquitin chain formation, and NFkB- and interferon-related gene expression. Thus, LUBAC synthesis of linear ubiquitin chains regulates STING-mediated innate immune signaling.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.12.561992">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.12.561992" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.12.561992">
        <p class="paperTitle">Nanobodies against SARS-CoV-2 non-structural protein Nsp9 inhibit viral replication by targeting innate immunity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.12.561992" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.12.561992" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Venit, T.; Blavier, J.; Maseko, S. B.; Shu, S.; Espada, L.; Breunig, C.; Holthoff, H.-P.; Desbordes, S. C.; Lohse, M.; Esposito, G.; Twizere, J.-C.; Percipalle, P.</p>
        <p class="info">Score: 5.2, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.12.561992' target='https://doi.org/10.1101/2023.10.12.561992'> 10.1101/2023.10.12.561992</a></p>
        <p class="abstract">Nanobodies are emerging as critical tools for drug design. Several have been recently created to serve as inhibitors of SARS-Cov-2 entry in the host cell by targeting surface-exposed Spike protein. However, due to the high frequency of mutations that affect Spike, these nanobodies may not target it to their full potential and as a consequence, inhibition of viral entry may not be efficient. Here we have established a pipeline that instead targets highly conserved viral proteins that are made only after viral entry into the host cell when the SARS-Cov-2 RNA-based genome is translated. As proof of principle, we designed nanobodies against the SARS-CoV-2 non-structural protein Nsp9, required for viral genome replication. To find out if this strategy efficiently blocks viral replication, one of these anti-Nsp9 nanobodies, 2NSP23, previously characterized using immunoassays and NMR spectroscopy for epitope mapping, was encapsulated into lipid nanoparticles (LNP) as mRNA. We show that this nanobody, hereby referred to as LNP-mRNA- 2NSP23, is internalized and translated in HEK293 cells. We next infected HEK293-ACE2 cells with multiple SARS-CoV-2 variants and subjected them to LNP-mRNA-2NSP23 treatment. Analysis of total RNA isolated from infected cells treated or untreated with LNP-mRNA-2NSP23 using qPCR and RNA deep sequencing shows that the LNP-mRNA-2NSP23 nanobody protects HEK293-ACE2 cells and suppresses replication of several SARS-CoV-2 variants. These observations indicate that following translation, the nanobody 2NSP23 inhibits viral replication by targeting Nsp9 in living cells. We speculate that LNP-mRNA-2NSP23 may be translated into an innovative technology to generate novel antiviral drugs highly efficient across coronaviruses.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
