<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cell biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cell biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Sulfolobus acidocaldarius adhesion pili power twitching motility in the absence of a dedicated retraction ATPase
Authors: Charles-Orszag, A.; van Wolferen, M.; Lord, S. J.; Albers, S.-V.; Mullins, D.
Score: 30.4, Published: 2023-08-04 DOI: 10.1101/2023.08.04.552066
Type IV pili are ancient and widespread filamentous organelles found in most bacterial and archaeal phyla where they support a wide range of functions, including substrate adhesion, DNA uptake, self aggregation, and cell motility. In most bacteria, PilT-family ATPases disassemble adhesion pili, causing them to rapidly retract and produce twitching motility, important for surface colonization.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cell-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cell biology" />
<meta property="og:description" content="Sulfolobus acidocaldarius adhesion pili power twitching motility in the absence of a dedicated retraction ATPase
Authors: Charles-Orszag, A.; van Wolferen, M.; Lord, S. J.; Albers, S.-V.; Mullins, D.
Score: 30.4, Published: 2023-08-04 DOI: 10.1101/2023.08.04.552066
Type IV pili are ancient and widespread filamentous organelles found in most bacterial and archaeal phyla where they support a wide range of functions, including substrate adhesion, DNA uptake, self aggregation, and cell motility. In most bacteria, PilT-family ATPases disassemble adhesion pili, causing them to rapidly retract and produce twitching motility, important for surface colonization." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cell-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-09T10:39:33+00:00" />
<meta property="article:modified_time" content="2023-08-09T10:39:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cell biology"/>
<meta name="twitter:description" content="Sulfolobus acidocaldarius adhesion pili power twitching motility in the absence of a dedicated retraction ATPase
Authors: Charles-Orszag, A.; van Wolferen, M.; Lord, S. J.; Albers, S.-V.; Mullins, D.
Score: 30.4, Published: 2023-08-04 DOI: 10.1101/2023.08.04.552066
Type IV pili are ancient and widespread filamentous organelles found in most bacterial and archaeal phyla where they support a wide range of functions, including substrate adhesion, DNA uptake, self aggregation, and cell motility. In most bacteria, PilT-family ATPases disassemble adhesion pili, causing them to rapidly retract and produce twitching motility, important for surface colonization."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cell biology",
      "item": "https://trxiv.yorks0n.com/posts/cell-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cell biology",
  "name": "cell biology",
  "description": "Sulfolobus acidocaldarius adhesion pili power twitching motility in the absence of a dedicated retraction ATPase\nAuthors: Charles-Orszag, A.; van Wolferen, M.; Lord, S. J.; Albers, S.-V.; Mullins, D.\nScore: 30.4, Published: 2023-08-04 DOI: 10.1101/2023.08.04.552066\nType IV pili are ancient and widespread filamentous organelles found in most bacterial and archaeal phyla where they support a wide range of functions, including substrate adhesion, DNA uptake, self aggregation, and cell motility. In most bacteria, PilT-family ATPases disassemble adhesion pili, causing them to rapidly retract and produce twitching motility, important for surface colonization.",
  "keywords": [
    
  ],
  "articleBody": " Sulfolobus acidocaldarius adhesion pili power twitching motility in the absence of a dedicated retraction ATPase\nAuthors: Charles-Orszag, A.; van Wolferen, M.; Lord, S. J.; Albers, S.-V.; Mullins, D.\nScore: 30.4, Published: 2023-08-04 DOI: 10.1101/2023.08.04.552066\nType IV pili are ancient and widespread filamentous organelles found in most bacterial and archaeal phyla where they support a wide range of functions, including substrate adhesion, DNA uptake, self aggregation, and cell motility. In most bacteria, PilT-family ATPases disassemble adhesion pili, causing them to rapidly retract and produce twitching motility, important for surface colonization. As archaea do not possess homologs of PilT, it was thought that archaeal pili cannot retract. Here, we employ live-cell imaging under native conditions (75{degrees}C and pH 2), together with automated single-cell tracking, high-temperature fluorescence imaging, and genetic manipulation to demonstrate that S. acidocaldarius exhibits bona fide twitching motility, and that this behavior depends specifically on retractable adhesion pili. Our results demonstrate that archaeal adhesion pili are capable of retraction in the absence of a PilT retraction ATPase and suggests that the ancestral type IV pilus machinery in the last universal common ancestor (LUCA) relied on such a bifunctional ATPase for both extension and retraction.\nThe actin binding protein profilin 1 is critical for mitochondria function\nAuthors: Read, T.-A.; Cisterna, B. A.; Skruber, K.; Ahmadieh, S.; Lindamood, H. L.; Vitriol, J.; Shi, Y.; Lefebvre, A. E. Y. T.; Black, J. B.; Butler, M. T.; Bear, J. E.; Cherezova, A.; Ilatovskaya, D. V.; Weintraub, N. L.; Vitriol, E. A.\nScore: 28.9, Published: 2023-08-08 DOI: 10.1101/2023.08.07.552354\nProfilin 1 (PFN1) is an actin binding protein that is vital for the polymerization of monomeric actin into filaments. Here we screened knockout cells for novel functions of PFN1 and discovered that mitophagy, a type of selective autophagy that removes defective or damaged mitochondria from the cell, was significantly upregulated in the absence of PFN1. Despite successful autophagosome formation and fusion with the lysosome, and activation of additional mitochondrial quality control pathways, PFN1 knockout cells still accumulate damaged, dysfunctional mitochondria. Subsequent imaging and functional assays showed that loss of PFN1 significantly affects mitochondria morphology, dynamics, and respiration. Further experiments revealed that PFN1 is located to the mitochondria matrix and is likely regulating mitochondria function from within rather than through polymerizing actin at the mitochondria surface. Finally, PFN1 mutants associated with amyotrophic lateral sclerosis (ALS) fail to rescue PFN1 knockout mitochondrial phenotypes and form aggregates within mitochondria, further perturbing them. Together, these results suggest a novel function for PFN1 in regulating mitochondria and identify a potential pathogenic mechanism of ALS-linked PFN1 variants loss and gain of toxic function effects.\nLactate activates the mitochondrial electron transport chain independent of its metabolism\nAuthors: Cai, X.; Ng, C. C.; Jones, O.; Fung, T. S.; Ryu, K.; Li, D.; Thompson, C. B.\nScore: 20.2, Published: 2023-08-04 DOI: 10.1101/2023.08.02.551712\nLactate has long been considered a cellular waste product. However, we found that as extracellular lactate accumulates, it also enters the mitochondrial matrix and stimulates mitochondrial electron transport chain (ETC) activity. The resulting increase in mitochondrial ATP synthesis suppresses glycolysis and increases the utilization of pyruvate and/or alternative respiratory substrates. The ability of lactate to increase oxidative phosphorylation does not depend on its metabolism. Both L- and D-lactate are effective at enhancing ETC activity and suppressing glycolysis. Furthermore, the selective induction of mitochondrial oxidative phosphorylation by unmetabolized D-lactate reversibly suppressed aerobic glycolysis in both cancer cell lines and proliferating primary cells in an ATP-dependent manner and enabled cell growth on respiratory-dependent bioenergetic substrates. In primary T cells, D-lactate enhanced cell proliferation and effector function. Together, these findings demonstrate that lactate is a critical regulator of the ability of mitochondrial oxidative phosphorylation to suppress glucose fermentation.\nStructure of the human outer kinetochore KMN network complex\nAuthors: Yatskevich, S.; Yang, J.; Bellini, D.; Zhang, Z.; Barford, D.\nScore: 18.6, Published: 2023-08-07 DOI: 10.1101/2023.08.07.552234\nFaithful chromosome segregation requires robust, load-bearing attachments of chromosomes to the mitotic spindle, a function accomplished by large macromolecular complexes termed kinetochores. In most eukaryotes, the constitutive centromere-associated network (CCAN) complex of the inner kinetochore recruits to centromeres the ten-subunit outer kinetochore KMN network, which comprises the KNL1C, MIS12C and NDC80C complexes. The KMN network directly attaches CCAN to microtubules through MIS12C and NDC80C. Here, we determined a high-resolution cryo-EM structure of the human KMN network. This showed an intricate and extensive assembly of KMN subunits, with the central MIS12C forming rigid interfaces with NDC80C and KNL1C. The redundancy and strength of inter-subunit connections explains how KMN withstands strong forces applied during chromosome segregation. We also observed that unphosphorylated MIS12C exists in an auto-inhibited state that suppresses its capacity to interact with CCAN. Ser100 and Ser109 of the N-terminal segment of the MIS12C subunit Dsn1, two key targets of Aurora B kinase, directly stabilize this auto-inhibition. Our work provides a molecular mechanism for how selectively relieving this auto-inhibition through Ser100 and Ser109 phosphorylation would restrict outer kinetochore assembly to functional centromeres during cell division.\nA force-sensitive mutation reveals a spindle assembly checkpoint-independent role for dynein in anaphase progression\nAuthors: Salvador-Garcia, D.; Jin, L.; Hensley, A.; Golcuk, M.; Gallaud, E.; Chaaban, S.; Port, F.; Vagnoni, A.; Planelles-Herrero, V. J.; McClintock, M. A.; Derivery, E.; Carter, A. P.; Giet, R.; Gur, M.; Yildiz, A.; Bullock, S. L.\nScore: 14.8, Published: 2023-08-04 DOI: 10.1101/2023.08.03.551815\nThe cytoplasmic dynein-1 (dynein) motor organizes cells by shaping microtubule networks and moving a large variety of cargoes along them. However, dyneins diverse roles complicate in vivo studies of its functions significantly. To address this issue, we have used gene editing to generate a series of missense mutations in Drosophila Dynein heavy chain (Dhc). We find that mutations associated with human neurological disease cause a range of defects in larval and adult flies, including impaired cargo trafficking in neurons. We also describe a novel mutation in the microtubule-binding domain (MTBD) of Dhc that, remarkably, causes metaphase arrest of mitotic spindles in the embryo but does not impair other dynein-dependent processes. We demonstrate that the mitotic arrest is independent of dyneins well-established roles in silencing the spindle assembly checkpoint. In vitro reconstitution and optical trapping assays reveal that the mutation only impairs the performance of dynein under load. In silico all-atom molecular dynamics simulations show that this effect correlates with increased flexibility of the MTBD, as well as an altered orientation of the stalk domain, with respect to the microtubule. Collectively, our data point to a novel role of dynein in anaphase progression that depends on the motor operating in a specific load regime. More broadly, our work illustrates how cytoskeletal transport processes can be dissected in vivo by manipulating mechanical properties of motors.\nDisordered C-terminal domain drives spatiotemporal confinement of RNAPII to enhance search for chromatin targets\nAuthors: Ling, Y. H.; Ye, J.; Yu, C.; Liang, C.; Park, G.; Corden, J.; Wu, C.\nScore: 13.1, Published: 2023-08-05 DOI: 10.1101/2023.07.31.551302\nEfficient gene expression demands RNA Polymerase II (RNAPII) to find chromatin targets precisely in space and time. How RNAPII manages this complex diffusive search in 3D nuclear space remains largely unknown. The disordered carboxy-terminal domain (CTD) of RNAPII, which is essential for recruiting transcription-associated proteins, forms phase-separated droplets in vitro, hinting at a potential role in modulating RNAPII dynamics. Here, we use single-molecule tracking and spatiotemporal mapping in living yeast to show that the CTD is responsible for confining RNAPII diffusion within a subnuclear region enriched for active genes, but without apparent phase separation into condensates. Both Mediator and global chromatin organization are required for sustaining RNAPII confinement. Remarkably, truncating the CTD disrupts RNAPII spatial confinement, prolongs target search, diminishes chromatin binding, impairs pre-initiation complex formation, and reduces transcription bursting. This study illuminates the pivotal role of the CTD in driving spatiotemporal confinement of RNAPII for efficient gene expression.\nLysosomal release of amino acids at ER three-way junctions regulates transmembrane and secretory protein mRNA translation\nAuthors: Choi, H.; Liao, Y.-C.; Yoon, Y. J.; Grimm, J. B.; Lavis, L.; Singer, R. H.; Lippincott-Schwartz, J.\nScore: 11.8, Published: 2023-08-03 DOI: 10.1101/2023.08.01.551382\nOne-third of the mammalian proteome is comprised of transmembrane and secretory proteins that are synthesized on endoplasmic reticulum (ER). Here, we investigate the spatial distribution and regulation of mRNAs encoding these membrane and secretory proteins (termed \"secretome\" mRNAs) through live cell, single molecule tracking to directly monitor the position and translation states of secretome mRNAs on ER and their relationship to other organelles. Notably, translation of secretome mRNAs occurred preferentially near lysosomes on ER marked by the ER junction-associated protein, Lunapark. Knockdown of Lunapark reduced the extent of secretome mRNA translation without affecting translation of other mRNAs. Less secretome mRNA translation also occurred when lysosome function was perturbed by raising lysosomal pH or inhibiting lysosomal proteases. Secretome mRNA translation near lysosomes was enhanced during amino acid deprivation. Addition of the integrated stress response inhibitor, ISRIB, reversed the translation inhibition seen in Lunapark knockdown cells, implying an eIF2 dependency. Altogether, these findings uncover a novel coordination between ER and lysosomes, in which local release of amino acids and other factors from ER-associated lysosomes patterns and regulates translation of mRNAs encoding secretory and membrane proteins.\nExtracellular filaments revealed by affinity capture cryo-electron tomography of lymphocytes\nAuthors: Engel, L.; Zaoralova, M.; Dunn, A. R.; Oliver, S. L.\nScore: 10.6, Published: 2023-08-07 DOI: 10.1101/2023.08.05.552110\nCryogenic-electron tomography (cryo-ET) has provided an unprecedented glimpse into the nanoscale architecture of cells by combining cryogenic preservation of biological structures with electron tomography. Micropatterning of extracellular matrix proteins is increasingly used as a method to prepare adherent cell types for cryo-ET as it promotes optimal positioning of cells and subcellular regions of interest for vitrification, cryo-focused ion beam (cryo-FIB) milling, and data acquisition. Here we demonstrate a micropatterning workflow for capturing minimally adherent cell types, human T-cells and Jurkat cells, for cryo-FIB and cryo-ET. Our affinity capture system facilitated the nanoscale imaging of Jurkat cells, revealing extracellular filamentous structures. It improved workflow efficiency by consistently producing grids with a sufficient number of wellpositioned cells for an entire cryo-FIB session. Affinity capture can be extended to facilitate high resolution imaging of other adherent and non-adherent cell types with cryo-ET.\nhGRAD - a versatile 'one-fits-all' system for the acute depletion of RNA binding proteins in nuclear condensates\nAuthors: Muller-McNicoll, M.; Zarnack, K.; McNicoll, F.; Keller, M.; Sliskovic, I.; Okuda, E. K.; Riegger, R. J.; Arnold, B.\nScore: 10.3, Published: 2023-08-04 DOI: 10.1101/2023.08.04.551933\nNuclear RNA binding proteins (RBPs) are difficult to study because they often belong to large protein families and form extensive networks of auto- and cross- regulation. They are highly abundant and often localize to condensates with a slow turnover, requiring long depletion times or knockouts that cannot distinguish between direct and indirect or compensatory effects. Here, we developed a system that is optimized for the rapid degradation of nuclear RBPs, called hGRAD. It comes as a 'one-fits-all' plasmid, and integration into any cell line that expresses endogenously GFP-tagged proteins allows an inducible, rapid and complete knockdown. We show that the nuclear RBPs SRSF3, SRSF5, SRRM2 and NONO are completely cleared from nuclear speckles and paraspeckles within two hours. hGRAD works in various cell types, is more efficient than other methods and does not require the expression of exogenous ubiquitin ligases. Combining SRSF5 hGRAD degradation with Nascent-seq uncovered highly dynamic transient transcript changes, compensatory mechanisms and that SRSF5 promotes transcript stability.\nSorting of secretory proteins at the trans-Golgi network by TGN46\nAuthors: Lujan, P.; Garcia-Cabau, C.; Wakana, Y.; Vera Lillo, J.; Rodilla-Ramirez, C.; Malhotra, V.; Salvatella, X.; Garcia-Parajo, M. F.; Campelo, F.\nScore: 22.4, Published: 2023-08-07 DOI: 10.1101/2022.04.20.488883\nSecretory proteins are sorted at the trans-Golgi network (TGN) for export into specific transport carriers. However, the molecular players involved in this fundamental process remain largely elusive. Here, we identified the transmembrane protein TGN46 as a receptor for the export of secretory cargo protein PAUF in CARTS - a class of protein kinase D-dependent TGN-to-plasma membrane carriers. We show that TGN46 is necessary for cargo sorting and loading into nascent carriers at the TGN. By combining quantitative fluorescence microscopy and mutagenesis approaches, we further discovered that the lumenal domain of TGN46 encodes for its cargo sorting function. In summary, our results define a cellular function of TGN46 in sorting secretory proteins for export from the TGN.\n",
  "wordCount" : "2030",
  "inLanguage": "en",
  "datePublished": "2023-08-09T10:39:33Z",
  "dateModified": "2023-08-09T10:39:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cell-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cell biology
    </h1>
    <div class="post-meta"><span>updated on August 9, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.552066">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.552066" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.552066">
        <p class="paperTitle">Sulfolobus acidocaldarius adhesion pili power twitching motility in the absence of a dedicated retraction ATPase</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.552066" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.552066" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Charles-Orszag, A.; van Wolferen, M.; Lord, S. J.; Albers, S.-V.; Mullins, D.</p>
        <p class="info">Score: 30.4, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.552066' target='https://doi.org/10.1101/2023.08.04.552066'> 10.1101/2023.08.04.552066</a></p>
        <p class="abstract">Type IV pili are ancient and widespread filamentous organelles found in most bacterial and archaeal phyla where they support a wide range of functions, including substrate adhesion, DNA uptake, self aggregation, and cell motility. In most bacteria, PilT-family ATPases disassemble adhesion pili, causing them to rapidly retract and produce twitching motility, important for surface colonization. As archaea do not possess homologs of PilT, it was thought that archaeal pili cannot retract. Here, we employ live-cell imaging under native conditions (75{degrees}C and pH 2), together with automated single-cell tracking, high-temperature fluorescence imaging, and genetic manipulation to demonstrate that S. acidocaldarius exhibits bona fide twitching motility, and that this behavior depends specifically on retractable adhesion pili. Our results demonstrate that archaeal adhesion pili are capable of retraction in the absence of a PilT retraction ATPase and suggests that the ancestral type IV pilus machinery in the last universal common ancestor (LUCA) relied on such a bifunctional ATPase for both extension and retraction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552354">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552354" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552354">
        <p class="paperTitle">The actin binding protein profilin 1 is critical for mitochondria function</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552354" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552354" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Read, T.-A.; Cisterna, B. A.; Skruber, K.; Ahmadieh, S.; Lindamood, H. L.; Vitriol, J.; Shi, Y.; Lefebvre, A. E. Y. T.; Black, J. B.; Butler, M. T.; Bear, J. E.; Cherezova, A.; Ilatovskaya, D. V.; Weintraub, N. L.; Vitriol, E. A.</p>
        <p class="info">Score: 28.9, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552354' target='https://doi.org/10.1101/2023.08.07.552354'> 10.1101/2023.08.07.552354</a></p>
        <p class="abstract">Profilin 1 (PFN1) is an actin binding protein that is vital for the polymerization of monomeric actin into filaments. Here we screened knockout cells for novel functions of PFN1 and discovered that mitophagy, a type of selective autophagy that removes defective or damaged mitochondria from the cell, was significantly upregulated in the absence of PFN1. Despite successful autophagosome formation and fusion with the lysosome, and activation of additional mitochondrial quality control pathways, PFN1 knockout cells still accumulate damaged, dysfunctional mitochondria. Subsequent imaging and functional assays showed that loss of PFN1 significantly affects mitochondria morphology, dynamics, and respiration. Further experiments revealed that PFN1 is located to the mitochondria matrix and is likely regulating mitochondria function from within rather than through polymerizing actin at the mitochondria surface. Finally, PFN1 mutants associated with amyotrophic lateral sclerosis (ALS) fail to rescue PFN1 knockout mitochondrial phenotypes and form aggregates within mitochondria, further perturbing them. Together, these results suggest a novel function for PFN1 in regulating mitochondria and identify a potential pathogenic mechanism of ALS-linked PFN1 variants loss and gain of toxic function effects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.551712">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.551712" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.551712">
        <p class="paperTitle">Lactate activates the mitochondrial electron transport chain independent of its metabolism</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.551712" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.551712" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cai, X.; Ng, C. C.; Jones, O.; Fung, T. S.; Ryu, K.; Li, D.; Thompson, C. B.</p>
        <p class="info">Score: 20.2, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.551712' target='https://doi.org/10.1101/2023.08.02.551712'> 10.1101/2023.08.02.551712</a></p>
        <p class="abstract">Lactate has long been considered a cellular waste product. However, we found that as extracellular lactate accumulates, it also enters the mitochondrial matrix and stimulates mitochondrial electron transport chain (ETC) activity. The resulting increase in mitochondrial ATP synthesis suppresses glycolysis and increases the utilization of pyruvate and/or alternative respiratory substrates. The ability of lactate to increase oxidative phosphorylation does not depend on its metabolism. Both L- and D-lactate are effective at enhancing ETC activity and suppressing glycolysis. Furthermore, the selective induction of mitochondrial oxidative phosphorylation by unmetabolized D-lactate reversibly suppressed aerobic glycolysis in both cancer cell lines and proliferating primary cells in an ATP-dependent manner and enabled cell growth on respiratory-dependent bioenergetic substrates. In primary T cells, D-lactate enhanced cell proliferation and effector function. Together, these findings demonstrate that lactate is a critical regulator of the ability of mitochondrial oxidative phosphorylation to suppress glucose fermentation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552234">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552234" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552234">
        <p class="paperTitle">Structure of the human outer kinetochore KMN network complex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552234" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552234" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yatskevich, S.; Yang, J.; Bellini, D.; Zhang, Z.; Barford, D.</p>
        <p class="info">Score: 18.6, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552234' target='https://doi.org/10.1101/2023.08.07.552234'> 10.1101/2023.08.07.552234</a></p>
        <p class="abstract">Faithful chromosome segregation requires robust, load-bearing attachments of chromosomes to the mitotic spindle, a function accomplished by large macromolecular complexes termed kinetochores. In most eukaryotes, the constitutive centromere-associated network (CCAN) complex of the inner kinetochore recruits to centromeres the ten-subunit outer kinetochore KMN network, which comprises the KNL1C, MIS12C and NDC80C complexes. The KMN network directly attaches CCAN to microtubules through MIS12C and NDC80C. Here, we determined a high-resolution cryo-EM structure of the human KMN network. This showed an intricate and extensive assembly of KMN subunits, with the central MIS12C forming rigid interfaces with NDC80C and KNL1C. The redundancy and strength of inter-subunit connections explains how KMN withstands strong forces applied during chromosome segregation. We also observed that unphosphorylated MIS12C exists in an auto-inhibited state that suppresses its capacity to interact with CCAN. Ser100 and Ser109 of the N-terminal segment of the MIS12C subunit Dsn1, two key targets of Aurora B kinase, directly stabilize this auto-inhibition. Our work provides a molecular mechanism for how selectively relieving this auto-inhibition through Ser100 and Ser109 phosphorylation would restrict outer kinetochore assembly to functional centromeres during cell division.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.03.551815">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.03.551815" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.03.551815">
        <p class="paperTitle">A force-sensitive mutation reveals a spindle assembly checkpoint-independent role for dynein in anaphase progression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.03.551815" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.03.551815" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Salvador-Garcia, D.; Jin, L.; Hensley, A.; Golcuk, M.; Gallaud, E.; Chaaban, S.; Port, F.; Vagnoni, A.; Planelles-Herrero, V. J.; McClintock, M. A.; Derivery, E.; Carter, A. P.; Giet, R.; Gur, M.; Yildiz, A.; Bullock, S. L.</p>
        <p class="info">Score: 14.8, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.03.551815' target='https://doi.org/10.1101/2023.08.03.551815'> 10.1101/2023.08.03.551815</a></p>
        <p class="abstract">The cytoplasmic dynein-1 (dynein) motor organizes cells by shaping microtubule networks and moving a large variety of cargoes along them. However, dyneins diverse roles complicate in vivo studies of its functions significantly. To address this issue, we have used gene editing to generate a series of missense mutations in Drosophila Dynein heavy chain (Dhc). We find that mutations associated with human neurological disease cause a range of defects in larval and adult flies, including impaired cargo trafficking in neurons. We also describe a novel mutation in the microtubule-binding domain (MTBD) of Dhc that, remarkably, causes metaphase arrest of mitotic spindles in the embryo but does not impair other dynein-dependent processes. We demonstrate that the mitotic arrest is independent of dyneins well-established roles in silencing the spindle assembly checkpoint. In vitro reconstitution and optical trapping assays reveal that the mutation only impairs the performance of dynein under load. In silico all-atom molecular dynamics simulations show that this effect correlates with increased flexibility of the MTBD, as well as an altered orientation of the stalk domain, with respect to the microtubule. Collectively, our data point to a novel role of dynein in anaphase progression that depends on the motor operating in a specific load regime. More broadly, our work illustrates how cytoskeletal transport processes can be dissected in vivo by manipulating mechanical properties of motors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.551302">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.551302" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.551302">
        <p class="paperTitle">Disordered C-terminal domain drives spatiotemporal confinement of RNAPII to enhance search for chromatin targets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.551302" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.551302" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ling, Y. H.; Ye, J.; Yu, C.; Liang, C.; Park, G.; Corden, J.; Wu, C.</p>
        <p class="info">Score: 13.1, Published: 2023-08-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.551302' target='https://doi.org/10.1101/2023.07.31.551302'> 10.1101/2023.07.31.551302</a></p>
        <p class="abstract">Efficient gene expression demands RNA Polymerase II (RNAPII) to find chromatin targets precisely in space and time. How RNAPII manages this complex diffusive search in 3D nuclear space remains largely unknown. The disordered carboxy-terminal domain (CTD) of RNAPII, which is essential for recruiting transcription-associated proteins, forms phase-separated droplets in vitro, hinting at a potential role in modulating RNAPII dynamics. Here, we use single-molecule tracking and spatiotemporal mapping in living yeast to show that the CTD is responsible for confining RNAPII diffusion within a subnuclear region enriched for active genes, but without apparent phase separation into condensates. Both Mediator and global chromatin organization are required for sustaining RNAPII confinement. Remarkably, truncating the CTD disrupts RNAPII spatial confinement, prolongs target search, diminishes chromatin binding, impairs pre-initiation complex formation, and reduces transcription bursting. This study illuminates the pivotal role of the CTD in driving spatiotemporal confinement of RNAPII for efficient gene expression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.551382">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.551382" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.551382">
        <p class="paperTitle">Lysosomal release of amino acids at ER three-way junctions regulates transmembrane and secretory protein mRNA translation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.551382" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.551382" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Choi, H.; Liao, Y.-C.; Yoon, Y. J.; Grimm, J. B.; Lavis, L.; Singer, R. H.; Lippincott-Schwartz, J.</p>
        <p class="info">Score: 11.8, Published: 2023-08-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.551382' target='https://doi.org/10.1101/2023.08.01.551382'> 10.1101/2023.08.01.551382</a></p>
        <p class="abstract">One-third of the mammalian proteome is comprised of transmembrane and secretory proteins that are synthesized on endoplasmic reticulum (ER). Here, we investigate the spatial distribution and regulation of mRNAs encoding these membrane and secretory proteins (termed &#34;secretome&#34; mRNAs) through live cell, single molecule tracking to directly monitor the position and translation states of secretome mRNAs on ER and their relationship to other organelles. Notably, translation of secretome mRNAs occurred preferentially near lysosomes on ER marked by the ER junction-associated protein, Lunapark. Knockdown of Lunapark reduced the extent of secretome mRNA translation without affecting translation of other mRNAs. Less secretome mRNA translation also occurred when lysosome function was perturbed by raising lysosomal pH or inhibiting lysosomal proteases. Secretome mRNA translation near lysosomes was enhanced during amino acid deprivation. Addition of the integrated stress response inhibitor, ISRIB, reversed the translation inhibition seen in Lunapark knockdown cells, implying an eIF2 dependency. Altogether, these findings uncover a novel coordination between ER and lysosomes, in which local release of amino acids and other factors from ER-associated lysosomes patterns and regulates translation of mRNAs encoding secretory and membrane proteins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.05.552110">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.05.552110" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.05.552110">
        <p class="paperTitle">Extracellular filaments revealed by affinity capture cryo-electron tomography of lymphocytes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.05.552110" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.05.552110" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Engel, L.; Zaoralova, M.; Dunn, A. R.; Oliver, S. L.</p>
        <p class="info">Score: 10.6, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.05.552110' target='https://doi.org/10.1101/2023.08.05.552110'> 10.1101/2023.08.05.552110</a></p>
        <p class="abstract">Cryogenic-electron tomography (cryo-ET) has provided an unprecedented glimpse into the nanoscale architecture of cells by combining cryogenic preservation of biological structures with electron tomography. Micropatterning of extracellular matrix proteins is increasingly used as a method to prepare adherent cell types for cryo-ET as it promotes optimal positioning of cells and subcellular regions of interest for vitrification, cryo-focused ion beam (cryo-FIB) milling, and data acquisition. Here we demonstrate a micropatterning workflow for capturing minimally adherent cell types, human T-cells and Jurkat cells, for cryo-FIB and cryo-ET. Our affinity capture system facilitated the nanoscale imaging of Jurkat cells, revealing extracellular filamentous structures. It improved workflow efficiency by consistently producing grids with a sufficient number of wellpositioned cells for an entire cryo-FIB session. Affinity capture can be extended to facilitate high resolution imaging of other adherent and non-adherent cell types with cryo-ET.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.551933">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.551933" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.551933">
        <p class="paperTitle">hGRAD - a versatile &#39;one-fits-all&#39; system for the acute depletion of RNA binding proteins in nuclear condensates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.551933" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.551933" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Muller-McNicoll, M.; Zarnack, K.; McNicoll, F.; Keller, M.; Sliskovic, I.; Okuda, E. K.; Riegger, R. J.; Arnold, B.</p>
        <p class="info">Score: 10.3, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.551933' target='https://doi.org/10.1101/2023.08.04.551933'> 10.1101/2023.08.04.551933</a></p>
        <p class="abstract">Nuclear RNA binding proteins (RBPs) are difficult to study because they often belong to large protein families and form extensive networks of auto- and cross- regulation. They are highly abundant and often localize to condensates with a slow turnover, requiring long depletion times or knockouts that cannot distinguish between direct and indirect or compensatory effects.

Here, we developed a system that is optimized for the rapid degradation of nuclear RBPs, called hGRAD. It comes as a &#39;one-fits-all&#39; plasmid, and integration into any cell line that expresses endogenously GFP-tagged proteins allows an inducible, rapid and complete knockdown. We show that the nuclear RBPs SRSF3, SRSF5, SRRM2 and NONO are completely cleared from nuclear speckles and paraspeckles within two hours. hGRAD works in various cell types, is more efficient than other methods and does not require the expression of exogenous ubiquitin ligases. Combining SRSF5 hGRAD degradation with Nascent-seq uncovered highly dynamic transient transcript changes, compensatory mechanisms and that SRSF5 promotes transcript stability.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.04.20.488883">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.04.20.488883" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.04.20.488883">
        <p class="paperTitle">Sorting of secretory proteins at the trans-Golgi network by TGN46</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.04.20.488883" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.04.20.488883" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lujan, P.; Garcia-Cabau, C.; Wakana, Y.; Vera Lillo, J.; Rodilla-Ramirez, C.; Malhotra, V.; Salvatella, X.; Garcia-Parajo, M. F.; Campelo, F.</p>
        <p class="info">Score: 22.4, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.04.20.488883' target='https://doi.org/10.1101/2022.04.20.488883'> 10.1101/2022.04.20.488883</a></p>
        <p class="abstract">Secretory proteins are sorted at the trans-Golgi network (TGN) for export into specific transport carriers. However, the molecular players involved in this fundamental process remain largely elusive. Here, we identified the transmembrane protein TGN46 as a receptor for the export of secretory cargo protein PAUF in CARTS - a class of protein kinase D-dependent TGN-to-plasma membrane carriers. We show that TGN46 is necessary for cargo sorting and loading into nascent carriers at the TGN. By combining quantitative fluorescence microscopy and mutagenesis approaches, we further discovered that the lumenal domain of TGN46 encodes for its cargo sorting function. In summary, our results define a cellular function of TGN46 in sorting secretory proteins for export from the TGN.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
