<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cell biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cell biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="The actin binding protein profilin 1 is critical for mitochondria function
Authors: Read, T.-A.; Cisterna, B. A.; Skruber, K.; Ahmadieh, S.; Lindamood, H. L.; Vitriol, J.; Shi, Y.; Lefebvre, A. E. Y. T.; Black, J. B.; Butler, M. T.; Bear, J. E.; Cherezova, A.; Ilatovskaya, D. V.; Weintraub, N. L.; Vitriol, E. A.
Score: 35.2, Published: 2023-08-08 DOI: 10.1101/2023.08.07.552354
Profilin 1 (PFN1) is an actin binding protein that is vital for the polymerization of monomeric actin into filaments.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cell-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cell biology" />
<meta property="og:description" content="The actin binding protein profilin 1 is critical for mitochondria function
Authors: Read, T.-A.; Cisterna, B. A.; Skruber, K.; Ahmadieh, S.; Lindamood, H. L.; Vitriol, J.; Shi, Y.; Lefebvre, A. E. Y. T.; Black, J. B.; Butler, M. T.; Bear, J. E.; Cherezova, A.; Ilatovskaya, D. V.; Weintraub, N. L.; Vitriol, E. A.
Score: 35.2, Published: 2023-08-08 DOI: 10.1101/2023.08.07.552354
Profilin 1 (PFN1) is an actin binding protein that is vital for the polymerization of monomeric actin into filaments." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cell-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-13T10:36:01+00:00" />
<meta property="article:modified_time" content="2023-08-13T10:36:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cell biology"/>
<meta name="twitter:description" content="The actin binding protein profilin 1 is critical for mitochondria function
Authors: Read, T.-A.; Cisterna, B. A.; Skruber, K.; Ahmadieh, S.; Lindamood, H. L.; Vitriol, J.; Shi, Y.; Lefebvre, A. E. Y. T.; Black, J. B.; Butler, M. T.; Bear, J. E.; Cherezova, A.; Ilatovskaya, D. V.; Weintraub, N. L.; Vitriol, E. A.
Score: 35.2, Published: 2023-08-08 DOI: 10.1101/2023.08.07.552354
Profilin 1 (PFN1) is an actin binding protein that is vital for the polymerization of monomeric actin into filaments."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cell biology",
      "item": "https://trxiv.yorks0n.com/posts/cell-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cell biology",
  "name": "cell biology",
  "description": "The actin binding protein profilin 1 is critical for mitochondria function\nAuthors: Read, T.-A.; Cisterna, B. A.; Skruber, K.; Ahmadieh, S.; Lindamood, H. L.; Vitriol, J.; Shi, Y.; Lefebvre, A. E. Y. T.; Black, J. B.; Butler, M. T.; Bear, J. E.; Cherezova, A.; Ilatovskaya, D. V.; Weintraub, N. L.; Vitriol, E. A.\nScore: 35.2, Published: 2023-08-08 DOI: 10.1101/2023.08.07.552354\nProfilin 1 (PFN1) is an actin binding protein that is vital for the polymerization of monomeric actin into filaments.",
  "keywords": [
    
  ],
  "articleBody": " The actin binding protein profilin 1 is critical for mitochondria function\nAuthors: Read, T.-A.; Cisterna, B. A.; Skruber, K.; Ahmadieh, S.; Lindamood, H. L.; Vitriol, J.; Shi, Y.; Lefebvre, A. E. Y. T.; Black, J. B.; Butler, M. T.; Bear, J. E.; Cherezova, A.; Ilatovskaya, D. V.; Weintraub, N. L.; Vitriol, E. A.\nScore: 35.2, Published: 2023-08-08 DOI: 10.1101/2023.08.07.552354\nProfilin 1 (PFN1) is an actin binding protein that is vital for the polymerization of monomeric actin into filaments. Here we screened knockout cells for novel functions of PFN1 and discovered that mitophagy, a type of selective autophagy that removes defective or damaged mitochondria from the cell, was significantly upregulated in the absence of PFN1. Despite successful autophagosome formation and fusion with the lysosome, and activation of additional mitochondrial quality control pathways, PFN1 knockout cells still accumulate damaged, dysfunctional mitochondria. Subsequent imaging and functional assays showed that loss of PFN1 significantly affects mitochondria morphology, dynamics, and respiration. Further experiments revealed that PFN1 is located to the mitochondria matrix and is likely regulating mitochondria function from within rather than through polymerizing actin at the mitochondria surface. Finally, PFN1 mutants associated with amyotrophic lateral sclerosis (ALS) fail to rescue PFN1 knockout mitochondrial phenotypes and form aggregates within mitochondria, further perturbing them. Together, these results suggest a novel function for PFN1 in regulating mitochondria and identify a potential pathogenic mechanism of ALS-linked PFN1 variants.\nSulfolobus acidocaldarius adhesion pili power twitching motility in the absence of a dedicated retraction ATPase\nAuthors: Charles-Orszag, A.; van Wolferen, M.; Lord, S. J.; Albers, S.-V.; Mullins, D.\nScore: 31.4, Published: 2023-08-04 DOI: 10.1101/2023.08.04.552066\nType IV pili are ancient and widespread filamentous organelles found in most bacterial and archaeal phyla where they support a wide range of functions, including substrate adhesion, DNA uptake, self aggregation, and cell motility. In most bacteria, PilT-family ATPases disassemble adhesion pili, causing them to rapidly retract and produce twitching motility, important for surface colonization. As archaea do not possess homologs of PilT, it was thought that archaeal pili cannot retract. Here, we employ live-cell imaging under native conditions (75{degrees}C and pH 2), together with automated single-cell tracking, high-temperature fluorescence imaging, and genetic manipulation to demonstrate that S. acidocaldarius exhibits bona fide twitching motility, and that this behavior depends specifically on retractable adhesion pili. Our results demonstrate that archaeal adhesion pili are capable of retraction in the absence of a PilT retraction ATPase and suggests that the ancestral type IV pilus machinery in the last universal common ancestor (LUCA) relied on such a bifunctional ATPase for both extension and retraction.\nStructure of the human outer kinetochore KMN network complex\nAuthors: Yatskevich, S.; Yang, J.; Bellini, D.; Zhang, Z.; Barford, D.\nScore: 19.4, Published: 2023-08-07 DOI: 10.1101/2023.08.07.552234\nFaithful chromosome segregation requires robust, load-bearing attachments of chromosomes to the mitotic spindle, a function accomplished by large macromolecular complexes termed kinetochores. In most eukaryotes, the constitutive centromere-associated network (CCAN) complex of the inner kinetochore recruits to centromeres the ten-subunit outer kinetochore KMN network, which comprises the KNL1C, MIS12C and NDC80C complexes. The KMN network directly attaches CCAN to microtubules through MIS12C and NDC80C. Here, we determined a high-resolution cryo-EM structure of the human KMN network. This showed an intricate and extensive assembly of KMN subunits, with the central MIS12C forming rigid interfaces with NDC80C and KNL1C. The redundancy and strength of inter-subunit connections explains how KMN withstands strong forces applied during chromosome segregation. We also observed that unphosphorylated MIS12C exists in an auto-inhibited state that suppresses its capacity to interact with CCAN. Ser100 and Ser109 of the N-terminal segment of the MIS12C subunit Dsn1, two key targets of Aurora B kinase, directly stabilize this auto-inhibition. Our work provides a molecular mechanism for how selectively relieving this auto-inhibition through Ser100 and Ser109 phosphorylation would restrict outer kinetochore assembly to functional centromeres during cell division.\nExtracellular filaments revealed by affinity capture cryo-electron tomography of lymphocytes\nAuthors: Engel, L.; Zaoralova, M.; Dunn, A. R.; Oliver, S. L.\nScore: 12.8, Published: 2023-08-07 DOI: 10.1101/2023.08.05.552110\nCryogenic-electron tomography (cryo-ET) has provided an unprecedented glimpse into the nanoscale architecture of cells by combining cryogenic preservation of biological structures with electron tomography. Micropatterning of extracellular matrix proteins is increasingly used as a method to prepare adherent cell types for cryo-ET as it promotes optimal positioning of cells and subcellular regions of interest for vitrification, cryo-focused ion beam (cryo-FIB) milling, and data acquisition. Here we demonstrate a micropatterning workflow for capturing minimally adherent cell types, human T-cells and Jurkat cells, for cryo-FIB and cryo-ET. Our affinity capture system facilitated the nanoscale imaging of Jurkat cells, revealing extracellular filamentous structures. It improved workflow efficiency by consistently producing grids with a sufficient number of wellpositioned cells for an entire cryo-FIB session. Affinity capture can be extended to facilitate high resolution imaging of other adherent and non-adherent cell types with cryo-ET.\nNon-homologous end joining shapes the genomic rearrangement landscape of chromothripsis from mitotic errors\nAuthors: Hu, Q.; Espejo Valle-Inclan, J.; Dahiya, R.; Guyer, A.; Mazzagatti, A.; Maurais, E. G.; Engel, J. L.; Cortes-Ciriano, I.; Ly, P.\nScore: 12.1, Published: 2023-08-11 DOI: 10.1101/2023.08.10.552800\nErrors in mitosis can generate micronuclei that entrap mis-segregated chromosomes, which are susceptible to catastrophic fragmentation through a process termed chromothripsis. The reassembly of fragmented chromosomes by error-prone DNA double-strand break (DSB) repair generates a spectrum of simple and complex genomic rearrangements that are associated with human cancers and disorders. How specific DSB repair pathways recognize and process these lesions remains poorly understood. Here we used CRISPR/Cas9 to systematically inactivate distinct DSB processing or repair pathways and interrogated the rearrangement landscape of fragmented chromosomes from micronuclei. Deletion of canonical non-homologous end joining (NHEJ) components, including DNA-PKcs, LIG4, and XLF, substantially reduced the formation of complex rearrangements and shifted the rearrangement landscape toward simple alterations without the characteristic patterns of cancer-associated chromothripsis. Following reincorporation into the nucleus, fragmented chromosomes localize within micronuclei bodies (MN bodies) and undergo successful ligation by NHEJ within a single cell cycle. In the absence of NHEJ, chromosome fragments were rarely engaged by polymerase theta-mediated alternative end-joining or recombination-based mechanisms, resulting in delayed repair kinetics and persistent 53BP1-labeled MN bodies in the interphase nucleus. Prolonged DNA damage signaling from unrepaired fragments ultimately triggered cell cycle arrest. Thus, we provide evidence supporting NHEJ as the exclusive DSB repair pathway generating complex rearrangements following chromothripsis from mitotic errors.\nSorting of secretory proteins at the trans-Golgi network by TGN46\nAuthors: Lujan, P.; Garcia-Cabau, C.; Wakana, Y.; Vera Lillo, J.; Rodilla-Ramirez, C.; Malhotra, V.; Salvatella, X.; Garcia-Parajo, M. F.; Campelo, F.\nScore: 22.8, Published: 2023-08-07 DOI: 10.1101/2022.04.20.488883\nSecretory proteins are sorted at the trans-Golgi network (TGN) for export into specific transport carriers. However, the molecular players involved in this fundamental process remain largely elusive. Here, we identified the transmembrane protein TGN46 as a receptor for the export of secretory cargo protein PAUF in CARTS - a class of protein kinase D-dependent TGN-to-plasma membrane carriers. We show that TGN46 is necessary for cargo sorting and loading into nascent carriers at the TGN. By combining quantitative fluorescence microscopy and mutagenesis approaches, we further discovered that the lumenal domain of TGN46 encodes for its cargo sorting function. In summary, our results define a cellular function of TGN46 in sorting secretory proteins for export from the TGN.\nProjective oblique plane structured illumination microscopy\nAuthors: Chang, B.-J.; Shepherd, D. P.; Fiolka, R. P.\nScore: 9.2, Published: 2023-08-09 DOI: 10.1101/2023.08.08.552447\nStructured illumination microscopy (SIM) can double the spatial resolution of a fluorescence microscope and video rate live cell imaging in a two-dimensional format has been demonstrated. However, rapid implementations of 2D SIM typically only cover a narrow slice of the sample immediately at the coverslip, with most of the cellular volume out of reach. Here we implement oblique plane structured illumination microscopy (OPSIM) in a projection format to rapidly image an entire cell in a 2D SIM framework. As no mechanical scanning of the sample or objective is involved, this technique has the potential for rapid projection imaging with doubled resolution. We characterize the spatial resolution with fluorescent nanospheres, compare projection and 3D imaging using OPSIM and image mitochondria and ER dynamics across an entire cell at up to 2.7 Hz. To our knowledge, this represents the fastest whole cell SIM imaging to date.\nLoss of HSC stemness identity is associated with exhaustion and hyporesponsiveness in GATA2 deficiency syndrome\nAuthors: Largeaud, L.; Fregona, V.; Jamrog, L.; Hamelle, C.; Dufrechou, S.; Prade, N.; Sellam, E.; Enfedaque, P.; Bayet, M.; Hebrard, S.; Bouttier, M.; Didier, C.; Delabesse, E.; Gerby, B.; Pasquet, M.; Broccardo, C.\nScore: 7.9, Published: 2023-08-07 DOI: 10.1101/2023.08.07.551811\nGermline GATA2 mutations lead to a syndrome involving both immunodeficiency and myeloid malignancies. Since GATA2 is a key player in hematopoietic initiation and development, we specify the impact of these germline mutations on hematopoietic homeostasis by generated a knock-in mouse model expressing the recurrent Gata2 R396Q missense mutation. These mice exhibit a hematopoietic stem and progenitor cell (HSPC) compartment profoundly impacted with increased HSC number, decreased self-renewal potential and inability to respond to acute inflammatory stimuli. Moreover, mutated HSPCs are predisposed to be hyporesponsive, as evidenced by lower interferon signaling and enrichment of inflammatory stress signatures. Furthermore, a Gata2 allelic specific expression results in a molecular and functional heterogeneity of the mutated Long Term-HSC population. Altogether, we highlight that Gata2 plays a crucial role in the ability of HSCs to perceive and respond to their environment, and that germline mutation contributes to the decline in HSC functionality.\nGolgi membrane proteins YIPF3 and YIPF4 regulate turnover of the Golgi apparatus through autophagy\nAuthors: Kitta, S.; Kaminishi, T.; Higashi, M.; Shima, T.; Kosako, H.; Yoshimori, T.; Kuma, A.\nScore: 7.7, Published: 2023-08-10 DOI: 10.1101/2023.08.09.552599\nThe degradation of organelles by autophagy is essential for cellular homeostasis. The Golgi apparatus has recently been demonstrated to be degraded by autophagy, but little is known about how the Golgi is recognized by the autophagosome. Using quantitative proteomic analysis and two Golgiphagy-reporter systems that we developed, we found that five-transmembrane Golgi-resident proteins, YIPF3 and YIPF4, constitute a Golgiphagy receptor. The YIPF3-YIPF4 complex interacts with LC3B, GABARAP, and GABARAPL1 via the LIR motif in YIPF3, whose stability is dependent on YIPF4. Phosphorylation of the LIR in YIPF3 seems to be required for YIPF3-ATG8 interaction. Moreover, expression of the YIPF3 LIR mutant caused an elongated Golgi morphology, indicating the importance of Golgi turnover via selective autophagy. The reporter assays that we established will pave the way for future studies to obtain deeper insights into Golgiphagy.\nLactate activates the mitochondrial electron transport chain independent of its metabolism\nAuthors: Cai, X.; Ng, C. C.; Jones, O.; Fung, T. S.; Ryu, K.; Li, D.; Thompson, C. B.\nScore: 21.4, Published: 2023-08-04 DOI: 10.1101/2023.08.02.551712\nLactate has long been considered a cellular waste product. However, we found that as extracellular lactate accumulates, it also enters the mitochondrial matrix and stimulates mitochondrial electron transport chain (ETC) activity. The resulting increase in mitochondrial ATP synthesis suppresses glycolysis and increases the utilization of pyruvate and/or alternative respiratory substrates. The ability of lactate to increase oxidative phosphorylation does not depend on its metabolism. Both L- and D-lactate are effective at enhancing ETC activity and suppressing glycolysis. Furthermore, the selective induction of mitochondrial oxidative phosphorylation by unmetabolized D-lactate reversibly suppressed aerobic glycolysis in both cancer cell lines and proliferating primary cells in an ATP-dependent manner and enabled cell growth on respiratory-dependent bioenergetic substrates. In primary T cells, D-lactate enhanced cell proliferation and effector function. Together, these findings demonstrate that lactate is a critical regulator of the ability of mitochondrial oxidative phosphorylation to suppress glucose fermentation.\n",
  "wordCount" : "1918",
  "inLanguage": "en",
  "datePublished": "2023-08-13T10:36:01Z",
  "dateModified": "2023-08-13T10:36:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cell-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cell biology
    </h1>
    <div class="post-meta"><span>updated on August 13, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552354">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552354" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552354">
        <p class="paperTitle">The actin binding protein profilin 1 is critical for mitochondria function</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552354" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552354" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Read, T.-A.; Cisterna, B. A.; Skruber, K.; Ahmadieh, S.; Lindamood, H. L.; Vitriol, J.; Shi, Y.; Lefebvre, A. E. Y. T.; Black, J. B.; Butler, M. T.; Bear, J. E.; Cherezova, A.; Ilatovskaya, D. V.; Weintraub, N. L.; Vitriol, E. A.</p>
        <p class="info">Score: 35.2, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552354' target='https://doi.org/10.1101/2023.08.07.552354'> 10.1101/2023.08.07.552354</a></p>
        <p class="abstract">Profilin 1 (PFN1) is an actin binding protein that is vital for the polymerization of monomeric actin into filaments. Here we screened knockout cells for novel functions of PFN1 and discovered that mitophagy, a type of selective autophagy that removes defective or damaged mitochondria from the cell, was significantly upregulated in the absence of PFN1. Despite successful autophagosome formation and fusion with the lysosome, and activation of additional mitochondrial quality control pathways, PFN1 knockout cells still accumulate damaged, dysfunctional mitochondria. Subsequent imaging and functional assays showed that loss of PFN1 significantly affects mitochondria morphology, dynamics, and respiration. Further experiments revealed that PFN1 is located to the mitochondria matrix and is likely regulating mitochondria function from within rather than through polymerizing actin at the mitochondria surface. Finally, PFN1 mutants associated with amyotrophic lateral sclerosis (ALS) fail to rescue PFN1 knockout mitochondrial phenotypes and form aggregates within mitochondria, further perturbing them. Together, these results suggest a novel function for PFN1 in regulating mitochondria and identify a potential pathogenic mechanism of ALS-linked PFN1 variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.552066">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.552066" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.552066">
        <p class="paperTitle">Sulfolobus acidocaldarius adhesion pili power twitching motility in the absence of a dedicated retraction ATPase</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.552066" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.552066" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Charles-Orszag, A.; van Wolferen, M.; Lord, S. J.; Albers, S.-V.; Mullins, D.</p>
        <p class="info">Score: 31.4, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.552066' target='https://doi.org/10.1101/2023.08.04.552066'> 10.1101/2023.08.04.552066</a></p>
        <p class="abstract">Type IV pili are ancient and widespread filamentous organelles found in most bacterial and archaeal phyla where they support a wide range of functions, including substrate adhesion, DNA uptake, self aggregation, and cell motility. In most bacteria, PilT-family ATPases disassemble adhesion pili, causing them to rapidly retract and produce twitching motility, important for surface colonization. As archaea do not possess homologs of PilT, it was thought that archaeal pili cannot retract. Here, we employ live-cell imaging under native conditions (75{degrees}C and pH 2), together with automated single-cell tracking, high-temperature fluorescence imaging, and genetic manipulation to demonstrate that S. acidocaldarius exhibits bona fide twitching motility, and that this behavior depends specifically on retractable adhesion pili. Our results demonstrate that archaeal adhesion pili are capable of retraction in the absence of a PilT retraction ATPase and suggests that the ancestral type IV pilus machinery in the last universal common ancestor (LUCA) relied on such a bifunctional ATPase for both extension and retraction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552234">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552234" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552234">
        <p class="paperTitle">Structure of the human outer kinetochore KMN network complex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552234" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552234" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yatskevich, S.; Yang, J.; Bellini, D.; Zhang, Z.; Barford, D.</p>
        <p class="info">Score: 19.4, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552234' target='https://doi.org/10.1101/2023.08.07.552234'> 10.1101/2023.08.07.552234</a></p>
        <p class="abstract">Faithful chromosome segregation requires robust, load-bearing attachments of chromosomes to the mitotic spindle, a function accomplished by large macromolecular complexes termed kinetochores. In most eukaryotes, the constitutive centromere-associated network (CCAN) complex of the inner kinetochore recruits to centromeres the ten-subunit outer kinetochore KMN network, which comprises the KNL1C, MIS12C and NDC80C complexes. The KMN network directly attaches CCAN to microtubules through MIS12C and NDC80C. Here, we determined a high-resolution cryo-EM structure of the human KMN network. This showed an intricate and extensive assembly of KMN subunits, with the central MIS12C forming rigid interfaces with NDC80C and KNL1C. The redundancy and strength of inter-subunit connections explains how KMN withstands strong forces applied during chromosome segregation. We also observed that unphosphorylated MIS12C exists in an auto-inhibited state that suppresses its capacity to interact with CCAN. Ser100 and Ser109 of the N-terminal segment of the MIS12C subunit Dsn1, two key targets of Aurora B kinase, directly stabilize this auto-inhibition. Our work provides a molecular mechanism for how selectively relieving this auto-inhibition through Ser100 and Ser109 phosphorylation would restrict outer kinetochore assembly to functional centromeres during cell division.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.05.552110">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.05.552110" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.05.552110">
        <p class="paperTitle">Extracellular filaments revealed by affinity capture cryo-electron tomography of lymphocytes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.05.552110" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.05.552110" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Engel, L.; Zaoralova, M.; Dunn, A. R.; Oliver, S. L.</p>
        <p class="info">Score: 12.8, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.05.552110' target='https://doi.org/10.1101/2023.08.05.552110'> 10.1101/2023.08.05.552110</a></p>
        <p class="abstract">Cryogenic-electron tomography (cryo-ET) has provided an unprecedented glimpse into the nanoscale architecture of cells by combining cryogenic preservation of biological structures with electron tomography. Micropatterning of extracellular matrix proteins is increasingly used as a method to prepare adherent cell types for cryo-ET as it promotes optimal positioning of cells and subcellular regions of interest for vitrification, cryo-focused ion beam (cryo-FIB) milling, and data acquisition. Here we demonstrate a micropatterning workflow for capturing minimally adherent cell types, human T-cells and Jurkat cells, for cryo-FIB and cryo-ET. Our affinity capture system facilitated the nanoscale imaging of Jurkat cells, revealing extracellular filamentous structures. It improved workflow efficiency by consistently producing grids with a sufficient number of wellpositioned cells for an entire cryo-FIB session. Affinity capture can be extended to facilitate high resolution imaging of other adherent and non-adherent cell types with cryo-ET.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552800">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552800" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552800">
        <p class="paperTitle">Non-homologous end joining shapes the genomic rearrangement landscape of chromothripsis from mitotic errors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552800" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552800" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hu, Q.; Espejo Valle-Inclan, J.; Dahiya, R.; Guyer, A.; Mazzagatti, A.; Maurais, E. G.; Engel, J. L.; Cortes-Ciriano, I.; Ly, P.</p>
        <p class="info">Score: 12.1, Published: 2023-08-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552800' target='https://doi.org/10.1101/2023.08.10.552800'> 10.1101/2023.08.10.552800</a></p>
        <p class="abstract">Errors in mitosis can generate micronuclei that entrap mis-segregated chromosomes, which are susceptible to catastrophic fragmentation through a process termed chromothripsis. The reassembly of fragmented chromosomes by error-prone DNA double-strand break (DSB) repair generates a spectrum of simple and complex genomic rearrangements that are associated with human cancers and disorders. How specific DSB repair pathways recognize and process these lesions remains poorly understood. Here we used CRISPR/Cas9 to systematically inactivate distinct DSB processing or repair pathways and interrogated the rearrangement landscape of fragmented chromosomes from micronuclei. Deletion of canonical non-homologous end joining (NHEJ) components, including DNA-PKcs, LIG4, and XLF, substantially reduced the formation of complex rearrangements and shifted the rearrangement landscape toward simple alterations without the characteristic patterns of cancer-associated chromothripsis. Following reincorporation into the nucleus, fragmented chromosomes localize within micronuclei bodies (MN bodies) and undergo successful ligation by NHEJ within a single cell cycle. In the absence of NHEJ, chromosome fragments were rarely engaged by polymerase theta-mediated alternative end-joining or recombination-based mechanisms, resulting in delayed repair kinetics and persistent 53BP1-labeled MN bodies in the interphase nucleus. Prolonged DNA damage signaling from unrepaired fragments ultimately triggered cell cycle arrest. Thus, we provide evidence supporting NHEJ as the exclusive DSB repair pathway generating complex rearrangements following chromothripsis from mitotic errors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.04.20.488883">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.04.20.488883" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.04.20.488883">
        <p class="paperTitle">Sorting of secretory proteins at the trans-Golgi network by TGN46</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.04.20.488883" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.04.20.488883" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lujan, P.; Garcia-Cabau, C.; Wakana, Y.; Vera Lillo, J.; Rodilla-Ramirez, C.; Malhotra, V.; Salvatella, X.; Garcia-Parajo, M. F.; Campelo, F.</p>
        <p class="info">Score: 22.8, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.04.20.488883' target='https://doi.org/10.1101/2022.04.20.488883'> 10.1101/2022.04.20.488883</a></p>
        <p class="abstract">Secretory proteins are sorted at the trans-Golgi network (TGN) for export into specific transport carriers. However, the molecular players involved in this fundamental process remain largely elusive. Here, we identified the transmembrane protein TGN46 as a receptor for the export of secretory cargo protein PAUF in CARTS - a class of protein kinase D-dependent TGN-to-plasma membrane carriers. We show that TGN46 is necessary for cargo sorting and loading into nascent carriers at the TGN. By combining quantitative fluorescence microscopy and mutagenesis approaches, we further discovered that the lumenal domain of TGN46 encodes for its cargo sorting function. In summary, our results define a cellular function of TGN46 in sorting secretory proteins for export from the TGN.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552447">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552447" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552447">
        <p class="paperTitle">Projective oblique plane structured illumination microscopy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552447" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552447" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chang, B.-J.; Shepherd, D. P.; Fiolka, R. P.</p>
        <p class="info">Score: 9.2, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552447' target='https://doi.org/10.1101/2023.08.08.552447'> 10.1101/2023.08.08.552447</a></p>
        <p class="abstract">Structured illumination microscopy (SIM) can double the spatial resolution of a fluorescence microscope and video rate live cell imaging in a two-dimensional format has been demonstrated. However, rapid implementations of 2D SIM typically only cover a narrow slice of the sample immediately at the coverslip, with most of the cellular volume out of reach. Here we implement oblique plane structured illumination microscopy (OPSIM) in a projection format to rapidly image an entire cell in a 2D SIM framework. As no mechanical scanning of the sample or objective is involved, this technique has the potential for rapid projection imaging with doubled resolution. We characterize the spatial resolution with fluorescent nanospheres, compare projection and 3D imaging using OPSIM and image mitochondria and ER dynamics across an entire cell at up to 2.7 Hz. To our knowledge, this represents the fastest whole cell SIM imaging to date.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.551811">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.551811" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.551811">
        <p class="paperTitle">Loss of HSC stemness identity is associated with exhaustion and hyporesponsiveness in GATA2 deficiency syndrome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.551811" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.551811" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Largeaud, L.; Fregona, V.; Jamrog, L.; Hamelle, C.; Dufrechou, S.; Prade, N.; Sellam, E.; Enfedaque, P.; Bayet, M.; Hebrard, S.; Bouttier, M.; Didier, C.; Delabesse, E.; Gerby, B.; Pasquet, M.; Broccardo, C.</p>
        <p class="info">Score: 7.9, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.551811' target='https://doi.org/10.1101/2023.08.07.551811'> 10.1101/2023.08.07.551811</a></p>
        <p class="abstract">Germline GATA2 mutations lead to a syndrome involving both immunodeficiency and myeloid malignancies. Since GATA2 is a key player in hematopoietic initiation and development, we specify the impact of these germline mutations on hematopoietic homeostasis by generated a knock-in mouse model expressing the recurrent Gata2 R396Q missense mutation. These mice exhibit a hematopoietic stem and progenitor cell (HSPC) compartment profoundly impacted with increased HSC number, decreased self-renewal potential and inability to respond to acute inflammatory stimuli. Moreover, mutated HSPCs are predisposed to be hyporesponsive, as evidenced by lower interferon signaling and enrichment of inflammatory stress signatures. Furthermore, a Gata2 allelic specific expression results in a molecular and functional heterogeneity of the mutated Long Term-HSC population. Altogether, we highlight that Gata2 plays a crucial role in the ability of HSCs to perceive and respond to their environment, and that germline mutation contributes to the decline in HSC functionality.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.552599">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.552599" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.552599">
        <p class="paperTitle">Golgi membrane proteins YIPF3 and YIPF4 regulate turnover of the Golgi apparatus through autophagy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.552599" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.552599" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kitta, S.; Kaminishi, T.; Higashi, M.; Shima, T.; Kosako, H.; Yoshimori, T.; Kuma, A.</p>
        <p class="info">Score: 7.7, Published: 2023-08-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.552599' target='https://doi.org/10.1101/2023.08.09.552599'> 10.1101/2023.08.09.552599</a></p>
        <p class="abstract">The degradation of organelles by autophagy is essential for cellular homeostasis. The Golgi apparatus has recently been demonstrated to be degraded by autophagy, but little is known about how the Golgi is recognized by the autophagosome. Using quantitative proteomic analysis and two Golgiphagy-reporter systems that we developed, we found that five-transmembrane Golgi-resident proteins, YIPF3 and YIPF4, constitute a Golgiphagy receptor. The YIPF3-YIPF4 complex interacts with LC3B, GABARAP, and GABARAPL1 via the LIR motif in YIPF3, whose stability is dependent on YIPF4. Phosphorylation of the LIR in YIPF3 seems to be required for YIPF3-ATG8 interaction. Moreover, expression of the YIPF3 LIR mutant caused an elongated Golgi morphology, indicating the importance of Golgi turnover via selective autophagy. The reporter assays that we established will pave the way for future studies to obtain deeper insights into Golgiphagy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.551712">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.551712" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.551712">
        <p class="paperTitle">Lactate activates the mitochondrial electron transport chain independent of its metabolism</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.551712" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.551712" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cai, X.; Ng, C. C.; Jones, O.; Fung, T. S.; Ryu, K.; Li, D.; Thompson, C. B.</p>
        <p class="info">Score: 21.4, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.551712' target='https://doi.org/10.1101/2023.08.02.551712'> 10.1101/2023.08.02.551712</a></p>
        <p class="abstract">Lactate has long been considered a cellular waste product. However, we found that as extracellular lactate accumulates, it also enters the mitochondrial matrix and stimulates mitochondrial electron transport chain (ETC) activity. The resulting increase in mitochondrial ATP synthesis suppresses glycolysis and increases the utilization of pyruvate and/or alternative respiratory substrates. The ability of lactate to increase oxidative phosphorylation does not depend on its metabolism. Both L- and D-lactate are effective at enhancing ETC activity and suppressing glycolysis. Furthermore, the selective induction of mitochondrial oxidative phosphorylation by unmetabolized D-lactate reversibly suppressed aerobic glycolysis in both cancer cell lines and proliferating primary cells in an ATP-dependent manner and enabled cell growth on respiratory-dependent bioenergetic substrates. In primary T cells, D-lactate enhanced cell proliferation and effector function. Together, these findings demonstrate that lactate is a critical regulator of the ability of mitochondrial oxidative phosphorylation to suppress glucose fermentation.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
