<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cell biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cell biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Measuring and modeling the dynamics of mitotic error correction
Authors: Ha, G.; Dieterle, P.; Shen, H.; Amir, A.; Needleman, D. J.
Score: 13.0, Published: 2024-01-11 DOI: 10.1101/2024.01.10.574250
When functioning properly, the mitotic spindle segregates an equal number of chromosomes to daughter cells with high fidelity. Over the course of spindle assembly, many initially erroneous attachments between kinetochores and microtubules are fixed through a process called error correction. Despite the importance of chromosome segregation errors in cancer and other diseases, there is a lack of methods to characterize the dynamics of error correction and how it can go wrong.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cell-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cell biology" />
<meta property="og:description" content="Measuring and modeling the dynamics of mitotic error correction
Authors: Ha, G.; Dieterle, P.; Shen, H.; Amir, A.; Needleman, D. J.
Score: 13.0, Published: 2024-01-11 DOI: 10.1101/2024.01.10.574250
When functioning properly, the mitotic spindle segregates an equal number of chromosomes to daughter cells with high fidelity. Over the course of spindle assembly, many initially erroneous attachments between kinetochores and microtubules are fixed through a process called error correction. Despite the importance of chromosome segregation errors in cancer and other diseases, there is a lack of methods to characterize the dynamics of error correction and how it can go wrong." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cell-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-17T10:40:22+00:00" />
<meta property="article:modified_time" content="2024-01-17T10:40:22+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cell biology"/>
<meta name="twitter:description" content="Measuring and modeling the dynamics of mitotic error correction
Authors: Ha, G.; Dieterle, P.; Shen, H.; Amir, A.; Needleman, D. J.
Score: 13.0, Published: 2024-01-11 DOI: 10.1101/2024.01.10.574250
When functioning properly, the mitotic spindle segregates an equal number of chromosomes to daughter cells with high fidelity. Over the course of spindle assembly, many initially erroneous attachments between kinetochores and microtubules are fixed through a process called error correction. Despite the importance of chromosome segregation errors in cancer and other diseases, there is a lack of methods to characterize the dynamics of error correction and how it can go wrong."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cell biology",
      "item": "https://trxiv.yorks0n.com/posts/cell-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cell biology",
  "name": "cell biology",
  "description": "Measuring and modeling the dynamics of mitotic error correction\nAuthors: Ha, G.; Dieterle, P.; Shen, H.; Amir, A.; Needleman, D. J.\nScore: 13.0, Published: 2024-01-11 DOI: 10.1101/2024.01.10.574250\nWhen functioning properly, the mitotic spindle segregates an equal number of chromosomes to daughter cells with high fidelity. Over the course of spindle assembly, many initially erroneous attachments between kinetochores and microtubules are fixed through a process called error correction. Despite the importance of chromosome segregation errors in cancer and other diseases, there is a lack of methods to characterize the dynamics of error correction and how it can go wrong.",
  "keywords": [
    
  ],
  "articleBody": " Measuring and modeling the dynamics of mitotic error correction\nAuthors: Ha, G.; Dieterle, P.; Shen, H.; Amir, A.; Needleman, D. J.\nScore: 13.0, Published: 2024-01-11 DOI: 10.1101/2024.01.10.574250\nWhen functioning properly, the mitotic spindle segregates an equal number of chromosomes to daughter cells with high fidelity. Over the course of spindle assembly, many initially erroneous attachments between kinetochores and microtubules are fixed through a process called error correction. Despite the importance of chromosome segregation errors in cancer and other diseases, there is a lack of methods to characterize the dynamics of error correction and how it can go wrong. Here, we present an experimental method and analysis framework to quantify chromosome segregation error correction in human tissue culture cells with live cell confocal imaging, timed premature anaphase, and automated counting of kinetochores after cell division. We find that errors decrease exponentially over time during spindle assembly. A coarse-grained model, in which errors are corrected in a chromosome autonomous manner at a constant rate, can quantitatively explain both the measured error correction dynamics and the distribution of anaphase onset times. We further validated our model using perturbations that destabilized microtubules and changed the initial configuration of chromosomal attachments. Taken together, this work provides a quantitative framework for understanding the dynamics of mitotic error correction.\nMechanism and regulation of cargo entry into the Commander recycling pathway\nAuthors: Butkovic, R.; Walker, A. P.; Healy, M. D.; McNally, K. E.; Liu, M.; Kato, K.; Collins, B. M.; Cullen, P. J.\nScore: 10.1, Published: 2024-01-11 DOI: 10.1101/2024.01.10.574988\nCommander is a multiprotein complex that orchestrates endosomal recycling of diverse integral cargo proteins and in humans is required for normal skeletal, brain, kidney, and cardiovascular development. While the structure of this complex has recently been described, the central question of how cargo proteins are selected for entry into the Commander recycling pathway remains unclear. Here using recombinant protein reconstitution and in silico predictions we identify the evolutionary conserved mechanism through which the unstructured carboxy-terminal tail of the integral protein adaptor sorting nexin-17 (SNX17) directly binds to the Retriever sub-complex of Commander. SNX17 adopts an autoinhibited conformation where its carboxy-terminal tail occupies the cargo binding groove. Competitive cargo binding overcomes this autoinhibition, promoting SNX17 endosomal residency and the release of the carboxy tail for Retriever association. Using molecular cell biology and high-resolution microscopy, we establish the central importance of SNX17-Retriever association in the handover of integrin and lipoprotein receptor cargoes into pre-existing endosomal retrieval sub-domains for entry into the recycling pathway. In describing the principal mechanism of cargo entry into the Commander recycling pathway we provide key insight into the function and regulation of this evolutionary conserved sorting complex.\nGeneration of systemic rabbit chimeras via Induced Pluripotent Stem Cells Reprogrammed with KLF2, ERAS, and PRMT6\nAuthors: Perold, F.; Pham, H.-T.; Pijoff, Y.; Doerflinger, N.; Rival-Gervier, S.; Moulin, A.; Jouneau, L.; Pain, B.; Joly, T.; Duranthon, V.; Afanassieff, M.; Savatier, P.; Beaujean, N.\nScore: 9.4, Published: 2024-01-11 DOI: 10.1101/2024.01.10.575048\nLittle is known about the molecular underpinnings of pluripotent stem cells (PSCs) ability to colonize the epiblast of preimplantation embryos and generate chimeras. In our study, using rabbit PSCs as a model system, we conducted unbiased screening of a cDNA library that encodes a panel of 36 pluripotency factors. From this screening, we identified KLF2, ERAS and PRMT6, whose overexpression confers the ability for self-renewal in a KOSR/FGF2-free culture medium supplemented with LIF, activin A, PKC and WNT inhibitors. The reprogrammed cells acquired transcriptomic and epigenetic features of naive pluripotency, including the reactivation of the 2nd X-chromosome. Leveraging these PSC lines, we determined the transcriptomic signature of embryonic colonization-competence, demonstrating transcriptional repression of genes involved in MAPK, WNT, HIPPO, and EPH signaling pathways, alongside the activation of genes involved in amino-acid metabolism, NF-kB signaling, and p53 pathway. Remarkably, a subset of reprogrammed cells, expressing CD75 at a high level, gained the ability to produce chimeric fetuses with a high contribution from PSCs in all lineages.\nThe PCM scaffold enables RNA localization to centrosomes\nAuthors: Fang, J.; Tian, W.; Quintanilla, M. A.; Beach, J. R.; Lerit, D. A.\nScore: 8.6, Published: 2024-01-14 DOI: 10.1101/2024.01.13.575509\nAs microtubule-organizing centers, centrosomes direct assembly of the bipolar mitotic spindle required for chromosome segregation and genome stability. Centrosome activity requires the dynamic assembly of pericentriolar material (PCM), the composition and organization of which changes throughout the cell cycle. Recent studies highlight the conserved localization of several mRNAs encoded from centrosome-associated genes enriched at centrosomes, including Pericentrin-like protein (Plp) mRNA. However, relatively little is known about how RNAs localize to centrosomes and influence centrosome function. Here, we examine mechanisms underlying the subcellular localization of Plp mRNA. We find that Plp mRNA localization is puromycin-sensitive, and the Plp coding sequence is both necessary and sufficient for RNA localization, consistent with a co-translational transport mechanism. We identify regions within the Plp coding sequence that regulate Plp mRNA localization. Finally, we show that protein-protein interactions critical for elaboration of the PCM scaffold permit RNA localization to centrosomes. Taken together, these findings inform the mechanistic basis of Plp mRNA localization and lend insight into the oscillatory enrichment of RNA at centrosomes.\nStarvation resistant cavefish reveal conserved mechanisms of starvation-induced hepatic lipotoxicity\nAuthors: Pozo-Morales, M.; Cobham, A. E.; Centola, C.; McKinney, M. C.; Liu, P.; Perazzolo, C.; Lefort, A.; Libert, F.; Bai, H.; Rohner, N.; Singh, S. P.\nScore: 8.3, Published: 2024-01-11 DOI: 10.1101/2024.01.10.574986\nStarvation causes the accumulation of lipid droplets in the liver, a somewhat counterintuitive phenomenon that is nevertheless conserved from flies to humans. Much like fatty liver resulting from overfeeding, hepatic lipid accumulation (steatosis) during undernourishment can lead to lipotoxicity and atrophy of the liver. Here, we found that while surface populations of Astyanax mexicanus undergo this evolutionarily conserved response to starvation, the starvation-resistant cavefish larvae of the same species do not display an accumulation of lipid droplets upon starvation. Moreover, cavefish are resistant to liver atrophy during starvation, providing a unique system to explore strategies for liver protection. Using comparative transcriptomics between zebrafish, surface fish, and cavefish, we identified the fatty acid transporter slc27a2a/fatp2 to be correlated with the development of fatty liver. Pharmacological inhibition of slc27a2a in zebrafish rescues steatosis and atrophy of the liver upon starvation. Further, down-regulation of FATP2 in drosophila larvae inhibits the development of starvation-induced steatosis, suggesting the evolutionary conserved importance of the gene in regulating fatty liver upon nutrition deprivation. Overall, our study identifies a conserved, druggable target to protect the liver from atrophy during starvation. One-Sentence SummaryCavefish evolved protection from starvation-induced liver damage through reduction of fatty acid uptake regulated by FATP2, a mechanism conserved through 400 million years of animal evolution.\nDetection of Fluorescent Protein Mechanical Switching in Cellulo\nAuthors: Shoyer, T. C.; Collins, K. L.; Ham, T. R.; Blanchard, A. T.; Malavade, J. N.; West, J. L.; Hoffman, B. D.\nScore: 7.8, Published: 2024-01-11 DOI: 10.1101/2024.01.10.575065\nABSTRACT/SUMMARYThe ability of cells to sense and respond to mechanical forces is critical in many physiological and pathological processes. However, the mechanisms by which forces affect protein function inside cells remain unclear. Motivated by in vitro demonstrations of fluorescent proteins (FPs) undergoing reversible mechanical switching of fluorescence, we investigated if force-sensitive changes in FP function could be visualized in cells. Guided by a computational model of FP mechanical switching, we develop a formalism for its detection in Forster resonance energy transfer (FRET)-based biosensors and demonstrate its occurrence in cellulo in a synthetic actin-crosslinker and the mechanical linker protein vinculin. We find that in cellulo mechanical switching is reversible and altered by manipulation of cellular force generation as well as force-sensitive bond dynamics of the biosensor. Together, this work describes a new framework for assessing FP mechanical stability and provides a means of probing force-sensitive protein function inside cells. MOTIVATIONThe ability of cells to sense mechanical forces is critical in developmental, physiological, and pathological processes. Cells sense mechanical cues via force-induced alterations in protein structure and function, but elucidation of the molecular mechanisms is hindered by the lack of approaches to directly probe the effect of forces on protein structure and function inside cells. Motivated by in vitro observations of reversible fluorescent protein mechanical switching, we developed an approach for detecting fluorescent protein mechanical switching in cellulo. This enables the visualization of force-sensitive protein function inside living cells.\nVariation of C-terminal domain governs RNA polymerase II genomic locations and alternative splicing in eukaryotic transcription\nAuthors: Zhang, Q.; Kim, W.; Panina, S.; Mayfield, J. E.; Portz, B.; Zhang, Y.\nScore: 8.8, Published: 2024-01-02 DOI: 10.1101/2024.01.01.573828\nThe C-terminal domain of RPB1 (CTD) orchestrates transcription by recruiting regulators to RNA Pol II upon phosphorylation. Recent insights highlight the pivotal role of CTD in driving condensate formation on gene loci. Yet, the molecular mechanism behind how CTD-mediated recruitment of transcriptional regulators influences condensates formation remains unclear. Our study unveils that phosphorylation reversibly dissolves phase separation induced by the unphosphorylated CTD. Phosphorylated CTD, upon specific association with transcription regulatory proteins, forms distinct condensates from unphosphorylated CTD. Function studies demonstrate CTD variants with diverse condensation properties in vitro exhibit difference in promoter binding and mRNA co-processing in cells. Notably, varying CTD lengths lead to alternative splicing outcomes impacting cellular growth, linking the evolution of CTD variation/length with the complexity of splicing from yeast to human. These findings provide compelling evidence for a model wherein post-translational modification enables the transition of functionally specialized condensates, highlighting a co-evolution link between CTD condensation and splicing.\nDeubiquitinating enzyme mutagenesis screens identify a USP43 driven HIF-1 transcriptional response\nAuthors: Pauzaite, T.; Wit, N.; Seear, R. V.; Nathan, J. A.\nScore: 4.8, Published: 2024-01-12 DOI: 10.1101/2024.01.10.574971\nThe ubiquitination and proteasome-mediated degradation of Hypoxia Inducible Factors (HIFs) is central to metazoan oxygen-sensing, but the involvement of deubiquitinating enzymes (DUBs) in HIF signalling is less clear. Here, using a bespoke DUBs sgRNA library we conduct CRISPR/Cas9 mutagenesis screens to determine how DUBs are involved in HIF signalling. Alongside defining DUBs involved in HIF activation or suppression, we identify USP43 as a DUB required for efficient activation of a HIF response. USP43 is hypoxia regulated and selectively associates with the HIF-1 isoform, and while USP43 does not alter HIF-1 stability, it facilitates HIF-1 nuclear accumulation and binding to its target genes. Mechanistically, USP43 associates with 14-3-3 proteins in a hypoxia and phosphorylation dependent manner to increase the nuclear pool of HIF-1. Together, our results unveil the DUB landscape in HIF signalling, and highlight the multifunctionality of DUBs, illustrating that they can provide important signalling functions alongside their catalytic roles.\nEvaluation of the efficacy of a full-spectrum medicinal cannabis plant extract with less than 0.3% Delta9-tetrahydrocannabinol in in vitro models of inflammation and excitotoxicity\nAuthors: Ross-Munro, E.; Isikgel, E.; Fleiss, B.\nScore: 3.6, Published: 2024-01-11 DOI: 10.1101/2024.01.10.575133\nThe rapid development of research on the therapeutic benefits of medicinal cannabis, in parallel with an increased understanding of the endocannabinoid system, has driven research of Cannabis sativa constituents for managing neurological conditions. While most studies have focused on the therapeutic potential of the major components of cannabis plant extract isolated or combined, limited research has explored the pharmacological benefits of whole cannabis plant extract. In this study, we investigated the potential anti-inflammatory and neuroprotective effects of NTI-164, a novel full-spectrum cannabis extract with negligible {Delta}9-tetrahydrocannabinol (THC), compared with cannabidiol (CBD) alone in BV-2 microglial and SHSY-5Y neuronal cells. The inflammation-induced upregulation of microglial inflammatory mediators, being tumour necrosis factor (TNF), granulocyte-macrophage colony-stimulating factor (GM-CSF), inducible nitric oxide synthase (iNOS), and Arginase-1 (Arg-1), were significantly attenuated by NTI-164. This immunomodulatory effect was not observed upon treatment with isolated CBD. Compared to CBD alone, NTI-164 prevented elevated mitochondrial activity while normalising cell numbers in immune-activated microglia cells. NTI-164 also promoted the proliferation of undifferentiated neurons and the survival of differentiated neurons under excitotoxic conditions. Overall, our work shows that the anti-inflammatory and neuroprotective effects of NTI-164 as a full-spectrum cannabis extract are enhanced relative to that of CBD alone, highlighting the potential therapeutic efficacy of NTI-164 for the treatment of neuropathologies such as autism spectrum disorder (ASD) and related neuropathologies. This study has further shown that understanding the synergistic effect of phytocannabinoids is integral to realising the therapeutic potential of full-spectrum cannabis extract to inform the design of botanical-derived treatments for managing neurological disorders.\nLive-cell fluorescence imaging and optogenetic control of PKA kinase activity in fission yeast Schizosaccharomyces pombe.\nAuthors: Sakai, K.; Aoki, K.; Goto, Y.\nScore: 3.2, Published: 2024-01-15 DOI: 10.1101/2024.01.14.575615\nThe cAMP-PKA signaling pathway plays a crucial role in sensing and responding to nutrient availability in the fission yeast Schizosaccharomyces pombe. This pathway monitors external glucose levels to control cell growth and sexual differentiation. However, the temporal dynamics of the cAMP-PKA pathway in response to external stimuli remains unclear mainly due to the lack of tools to quantitatively visualize the activity of the pathway. Here, we report the development of the kinase translocation reporter (KTR)-based biosensor spPKA-KTR1.0, which allows us to measure the dynamics of PKA activity in fission yeast cells. The spPKA-KTR1.0 is derived from the transcription factor Rst2, which translocates from the nucleus to the cytoplasm upon PKA activation. We found that spPKA-KTR1.0 translocates between the nucleus and cytoplasm in a cAMP-PKA pathway-dependent manner, indicating that the spPKA-KTR1.0 is a reliable indicator of the PKA activity in fission yeast cells. In addition, we implemented a system that simultaneously visualizes and manipulates the cAMP-PKA signaling dynamics by introducing bPAC, a photoactivatable adenylate cyclase, in combination with spPKA-KTR1.0. This system offers an opportunity for investigating the role of the signaling dynamics of the cAMP-PKA pathway in fission yeast cells with higher temporal resolution.\n",
  "wordCount" : "2250",
  "inLanguage": "en",
  "datePublished": "2024-01-17T10:40:22Z",
  "dateModified": "2024-01-17T10:40:22Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cell-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cell biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 17, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.574250">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.574250" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.574250">
        <p class="paperTitle">Measuring and modeling the dynamics of mitotic error correction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.574250" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.574250" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ha, G.; Dieterle, P.; Shen, H.; Amir, A.; Needleman, D. J.</p>
        <p class="info">Score: 13.0, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.574250' target='https://doi.org/10.1101/2024.01.10.574250'> 10.1101/2024.01.10.574250</a></p>
        <p class="abstract">When functioning properly, the mitotic spindle segregates an equal number of chromosomes to daughter cells with high fidelity. Over the course of spindle assembly, many initially erroneous attachments between kinetochores and microtubules are fixed through a process called error correction. Despite the importance of chromosome segregation errors in cancer and other diseases, there is a lack of methods to characterize the dynamics of error correction and how it can go wrong. Here, we present an experimental method and analysis framework to quantify chromosome segregation error correction in human tissue culture cells with live cell confocal imaging, timed premature anaphase, and automated counting of kinetochores after cell division. We find that errors decrease exponentially over time during spindle assembly. A coarse-grained model, in which errors are corrected in a chromosome autonomous manner at a constant rate, can quantitatively explain both the measured error correction dynamics and the distribution of anaphase onset times. We further validated our model using perturbations that destabilized microtubules and changed the initial configuration of chromosomal attachments. Taken together, this work provides a quantitative framework for understanding the dynamics of mitotic error correction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.574988">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.574988" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.574988">
        <p class="paperTitle">Mechanism and regulation of cargo entry into the Commander recycling pathway</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.574988" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.574988" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Butkovic, R.; Walker, A. P.; Healy, M. D.; McNally, K. E.; Liu, M.; Kato, K.; Collins, B. M.; Cullen, P. J.</p>
        <p class="info">Score: 10.1, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.574988' target='https://doi.org/10.1101/2024.01.10.574988'> 10.1101/2024.01.10.574988</a></p>
        <p class="abstract">Commander is a multiprotein complex that orchestrates endosomal recycling of diverse integral cargo proteins and in humans is required for normal skeletal, brain, kidney, and cardiovascular development. While the structure of this complex has recently been described, the central question of how cargo proteins are selected for entry into the Commander recycling pathway remains unclear. Here using recombinant protein reconstitution and in silico predictions we identify the evolutionary conserved mechanism through which the unstructured carboxy-terminal tail of the integral protein adaptor sorting nexin-17 (SNX17) directly binds to the Retriever sub-complex of Commander. SNX17 adopts an autoinhibited conformation where its carboxy-terminal tail occupies the cargo binding groove. Competitive cargo binding overcomes this autoinhibition, promoting SNX17 endosomal residency and the release of the carboxy tail for Retriever association. Using molecular cell biology and high-resolution microscopy, we establish the central importance of SNX17-Retriever association in the handover of integrin and lipoprotein receptor cargoes into pre-existing endosomal retrieval sub-domains for entry into the recycling pathway. In describing the principal mechanism of cargo entry into the Commander recycling pathway we provide key insight into the function and regulation of this evolutionary conserved sorting complex.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.575048">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.575048" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.575048">
        <p class="paperTitle">Generation of systemic rabbit chimeras via Induced Pluripotent Stem Cells Reprogrammed with KLF2, ERAS, and PRMT6</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.575048" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.575048" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Perold, F.; Pham, H.-T.; Pijoff, Y.; Doerflinger, N.; Rival-Gervier, S.; Moulin, A.; Jouneau, L.; Pain, B.; Joly, T.; Duranthon, V.; Afanassieff, M.; Savatier, P.; Beaujean, N.</p>
        <p class="info">Score: 9.4, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.575048' target='https://doi.org/10.1101/2024.01.10.575048'> 10.1101/2024.01.10.575048</a></p>
        <p class="abstract">Little is known about the molecular underpinnings of pluripotent stem cells (PSCs) ability to colonize the epiblast of preimplantation embryos and generate chimeras. In our study, using rabbit PSCs as a model system, we conducted unbiased screening of a cDNA library that encodes a panel of 36 pluripotency factors. From this screening, we identified KLF2, ERAS and PRMT6, whose overexpression confers the ability for self-renewal in a KOSR/FGF2-free culture medium supplemented with LIF, activin A, PKC and WNT inhibitors. The reprogrammed cells acquired transcriptomic and epigenetic features of naive pluripotency, including the reactivation of the 2nd X-chromosome. Leveraging these PSC lines, we determined the transcriptomic signature of embryonic colonization-competence, demonstrating transcriptional repression of genes involved in MAPK, WNT, HIPPO, and EPH signaling pathways, alongside the activation of genes involved in amino-acid metabolism, NF-kB signaling, and p53 pathway. Remarkably, a subset of reprogrammed cells, expressing CD75 at a high level, gained the ability to produce chimeric fetuses with a high contribution from PSCs in all lineages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.13.575509">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.13.575509" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.13.575509">
        <p class="paperTitle">The PCM scaffold enables RNA localization to centrosomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.13.575509" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.13.575509" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fang, J.; Tian, W.; Quintanilla, M. A.; Beach, J. R.; Lerit, D. A.</p>
        <p class="info">Score: 8.6, Published: 2024-01-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.13.575509' target='https://doi.org/10.1101/2024.01.13.575509'> 10.1101/2024.01.13.575509</a></p>
        <p class="abstract">As microtubule-organizing centers, centrosomes direct assembly of the bipolar mitotic spindle required for chromosome segregation and genome stability. Centrosome activity requires the dynamic assembly of pericentriolar material (PCM), the composition and organization of which changes throughout the cell cycle. Recent studies highlight the conserved localization of several mRNAs encoded from centrosome-associated genes enriched at centrosomes, including Pericentrin-like protein (Plp) mRNA. However, relatively little is known about how RNAs localize to centrosomes and influence centrosome function. Here, we examine mechanisms underlying the subcellular localization of Plp mRNA. We find that Plp mRNA localization is puromycin-sensitive, and the Plp coding sequence is both necessary and sufficient for RNA localization, consistent with a co-translational transport mechanism. We identify regions within the Plp coding sequence that regulate Plp mRNA localization. Finally, we show that protein-protein interactions critical for elaboration of the PCM scaffold permit RNA localization to centrosomes. Taken together, these findings inform the mechanistic basis of Plp mRNA localization and lend insight into the oscillatory enrichment of RNA at centrosomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.574986">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.574986" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.574986">
        <p class="paperTitle">Starvation resistant cavefish reveal conserved mechanisms of starvation-induced hepatic lipotoxicity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.574986" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.574986" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pozo-Morales, M.; Cobham, A. E.; Centola, C.; McKinney, M. C.; Liu, P.; Perazzolo, C.; Lefort, A.; Libert, F.; Bai, H.; Rohner, N.; Singh, S. P.</p>
        <p class="info">Score: 8.3, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.574986' target='https://doi.org/10.1101/2024.01.10.574986'> 10.1101/2024.01.10.574986</a></p>
        <p class="abstract">Starvation causes the accumulation of lipid droplets in the liver, a somewhat counterintuitive phenomenon that is nevertheless conserved from flies to humans. Much like fatty liver resulting from overfeeding, hepatic lipid accumulation (steatosis) during undernourishment can lead to lipotoxicity and atrophy of the liver. Here, we found that while surface populations of Astyanax mexicanus undergo this evolutionarily conserved response to starvation, the starvation-resistant cavefish larvae of the same species do not display an accumulation of lipid droplets upon starvation. Moreover, cavefish are resistant to liver atrophy during starvation, providing a unique system to explore strategies for liver protection. Using comparative transcriptomics between zebrafish, surface fish, and cavefish, we identified the fatty acid transporter slc27a2a/fatp2 to be correlated with the development of fatty liver. Pharmacological inhibition of slc27a2a in zebrafish rescues steatosis and atrophy of the liver upon starvation. Further, down-regulation of FATP2 in drosophila larvae inhibits the development of starvation-induced steatosis, suggesting the evolutionary conserved importance of the gene in regulating fatty liver upon nutrition deprivation. Overall, our study identifies a conserved, druggable target to protect the liver from atrophy during starvation.

One-Sentence SummaryCavefish evolved protection from starvation-induced liver damage through reduction of fatty acid uptake regulated by FATP2, a mechanism conserved through 400 million years of animal evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.575065">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.575065" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.575065">
        <p class="paperTitle">Detection of Fluorescent Protein Mechanical Switching in Cellulo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.575065" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.575065" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shoyer, T. C.; Collins, K. L.; Ham, T. R.; Blanchard, A. T.; Malavade, J. N.; West, J. L.; Hoffman, B. D.</p>
        <p class="info">Score: 7.8, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.575065' target='https://doi.org/10.1101/2024.01.10.575065'> 10.1101/2024.01.10.575065</a></p>
        <p class="abstract">ABSTRACT/SUMMARYThe ability of cells to sense and respond to mechanical forces is critical in many physiological and pathological processes. However, the mechanisms by which forces affect protein function inside cells remain unclear. Motivated by in vitro demonstrations of fluorescent proteins (FPs) undergoing reversible mechanical switching of fluorescence, we investigated if force-sensitive changes in FP function could be visualized in cells. Guided by a computational model of FP mechanical switching, we develop a formalism for its detection in Forster resonance energy transfer (FRET)-based biosensors and demonstrate its occurrence in cellulo in a synthetic actin-crosslinker and the mechanical linker protein vinculin. We find that in cellulo mechanical switching is reversible and altered by manipulation of cellular force generation as well as force-sensitive bond dynamics of the biosensor. Together, this work describes a new framework for assessing FP mechanical stability and provides a means of probing force-sensitive protein function inside cells.

MOTIVATIONThe ability of cells to sense mechanical forces is critical in developmental, physiological, and pathological processes. Cells sense mechanical cues via force-induced alterations in protein structure and function, but elucidation of the molecular mechanisms is hindered by the lack of approaches to directly probe the effect of forces on protein structure and function inside cells. Motivated by in vitro observations of reversible fluorescent protein mechanical switching, we developed an approach for detecting fluorescent protein mechanical switching in cellulo. This enables the visualization of force-sensitive protein function inside living cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.01.573828">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.01.573828" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.01.573828">
        <p class="paperTitle">Variation of C-terminal domain governs RNA polymerase II genomic locations and alternative splicing in eukaryotic transcription</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.01.573828" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.01.573828" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, Q.; Kim, W.; Panina, S.; Mayfield, J. E.; Portz, B.; Zhang, Y.</p>
        <p class="info">Score: 8.8, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.01.573828' target='https://doi.org/10.1101/2024.01.01.573828'> 10.1101/2024.01.01.573828</a></p>
        <p class="abstract">The C-terminal domain of RPB1 (CTD) orchestrates transcription by recruiting regulators to RNA Pol II upon phosphorylation. Recent insights highlight the pivotal role of CTD in driving condensate formation on gene loci. Yet, the molecular mechanism behind how CTD-mediated recruitment of transcriptional regulators influences condensates formation remains unclear. Our study unveils that phosphorylation reversibly dissolves phase separation induced by the unphosphorylated CTD. Phosphorylated CTD, upon specific association with transcription regulatory proteins, forms distinct condensates from unphosphorylated CTD. Function studies demonstrate CTD variants with diverse condensation properties in vitro exhibit difference in promoter binding and mRNA co-processing in cells. Notably, varying CTD lengths lead to alternative splicing outcomes impacting cellular growth, linking the evolution of CTD variation/length with the complexity of splicing from yeast to human. These findings provide compelling evidence for a model wherein post-translational modification enables the transition of functionally specialized condensates, highlighting a co-evolution link between CTD condensation and splicing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.574971">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.574971" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.574971">
        <p class="paperTitle">Deubiquitinating enzyme mutagenesis screens identify a USP43 driven HIF-1 transcriptional response</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.574971" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.574971" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pauzaite, T.; Wit, N.; Seear, R. V.; Nathan, J. A.</p>
        <p class="info">Score: 4.8, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.574971' target='https://doi.org/10.1101/2024.01.10.574971'> 10.1101/2024.01.10.574971</a></p>
        <p class="abstract">The ubiquitination and proteasome-mediated degradation of Hypoxia Inducible Factors (HIFs) is central to metazoan oxygen-sensing, but the involvement of deubiquitinating enzymes (DUBs) in HIF signalling is less clear. Here, using a bespoke DUBs sgRNA library we conduct CRISPR/Cas9 mutagenesis screens to determine how DUBs are involved in HIF signalling. Alongside defining DUBs involved in HIF activation or suppression, we identify USP43 as a DUB required for efficient activation of a HIF response. USP43 is hypoxia regulated and selectively associates with the HIF-1 isoform, and while USP43 does not alter HIF-1 stability, it facilitates HIF-1 nuclear accumulation and binding to its target genes. Mechanistically, USP43 associates with 14-3-3 proteins in a hypoxia and phosphorylation dependent manner to increase the nuclear pool of HIF-1. Together, our results unveil the DUB landscape in HIF signalling, and highlight the multifunctionality of DUBs, illustrating that they can provide important signalling functions alongside their catalytic roles.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.575133">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.575133" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.575133">
        <p class="paperTitle">Evaluation of the efficacy of a full-spectrum medicinal cannabis plant extract with less than 0.3% Delta9-tetrahydrocannabinol in in vitro models of inflammation and excitotoxicity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.575133" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.575133" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ross-Munro, E.; Isikgel, E.; Fleiss, B.</p>
        <p class="info">Score: 3.6, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.575133' target='https://doi.org/10.1101/2024.01.10.575133'> 10.1101/2024.01.10.575133</a></p>
        <p class="abstract">The rapid development of research on the therapeutic benefits of medicinal cannabis, in parallel with an increased understanding of the endocannabinoid system, has driven research of Cannabis sativa constituents for managing neurological conditions. While most studies have focused on the therapeutic potential of the major components of cannabis plant extract isolated or combined, limited research has explored the pharmacological benefits of whole cannabis plant extract. In this study, we investigated the potential anti-inflammatory and neuroprotective effects of NTI-164, a novel full-spectrum cannabis extract with negligible {Delta}9-tetrahydrocannabinol (THC), compared with cannabidiol (CBD) alone in BV-2 microglial and SHSY-5Y neuronal cells. The inflammation-induced upregulation of microglial inflammatory mediators, being tumour necrosis factor  (TNF), granulocyte-macrophage colony-stimulating factor (GM-CSF), inducible nitric oxide synthase (iNOS), and Arginase-1 (Arg-1), were significantly attenuated by NTI-164. This immunomodulatory effect was not observed upon treatment with isolated CBD. Compared to CBD alone, NTI-164 prevented elevated mitochondrial activity while normalising cell numbers in immune-activated microglia cells. NTI-164 also promoted the proliferation of undifferentiated neurons and the survival of differentiated neurons under excitotoxic conditions. Overall, our work shows that the anti-inflammatory and neuroprotective effects of NTI-164 as a full-spectrum cannabis extract are enhanced relative to that of CBD alone, highlighting the potential therapeutic efficacy of NTI-164 for the treatment of neuropathologies such as autism spectrum disorder (ASD) and related neuropathologies. This study has further shown that understanding the synergistic effect of phytocannabinoids is integral to realising the therapeutic potential of full-spectrum cannabis extract to inform the design of botanical-derived treatments for managing neurological disorders.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.14.575615">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.14.575615" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.14.575615">
        <p class="paperTitle">Live-cell fluorescence imaging and optogenetic control of PKA kinase activity in fission yeast Schizosaccharomyces pombe.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.14.575615" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.14.575615" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sakai, K.; Aoki, K.; Goto, Y.</p>
        <p class="info">Score: 3.2, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.14.575615' target='https://doi.org/10.1101/2024.01.14.575615'> 10.1101/2024.01.14.575615</a></p>
        <p class="abstract">The cAMP-PKA signaling pathway plays a crucial role in sensing and responding to nutrient availability in the fission yeast Schizosaccharomyces pombe. This pathway monitors external glucose levels to control cell growth and sexual differentiation. However, the temporal dynamics of the cAMP-PKA pathway in response to external stimuli remains unclear mainly due to the lack of tools to quantitatively visualize the activity of the pathway. Here, we report the development of the kinase translocation reporter (KTR)-based biosensor spPKA-KTR1.0, which allows us to measure the dynamics of PKA activity in fission yeast cells. The spPKA-KTR1.0 is derived from the transcription factor Rst2, which translocates from the nucleus to the cytoplasm upon PKA activation. We found that spPKA-KTR1.0 translocates between the nucleus and cytoplasm in a cAMP-PKA pathway-dependent manner, indicating that the spPKA-KTR1.0 is a reliable indicator of the PKA activity in fission yeast cells. In addition, we implemented a system that simultaneously visualizes and manipulates the cAMP-PKA signaling dynamics by introducing bPAC, a photoactivatable adenylate cyclase, in combination with spPKA-KTR1.0. This system offers an opportunity for investigating the role of the signaling dynamics of the cAMP-PKA pathway in fission yeast cells with higher temporal resolution.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
