<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cell biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cell biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Universal Cell Embeddings: A Foundation Model for Cell Biology
Authors: Rosen, Y.; Roohani, Y.; Agrawal, A.; Samotorcan, L.; Consortium, T. S.; Quake, S. R.; Leskovec, J.
Score: 66.8, Published: 2023-11-29 DOI: 10.1101/2023.11.28.568918
Developing a universal representation of cells which encompasses the tremendous molecular diversity of cell types within the human body and more generally, across species, would be transformative for cell biology. Recent work using single-cell transcriptomic approaches to create molecular definitions of cell types in the form of cell atlases has provided the necessary data for such an endeavor.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cell-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cell biology" />
<meta property="og:description" content="Universal Cell Embeddings: A Foundation Model for Cell Biology
Authors: Rosen, Y.; Roohani, Y.; Agrawal, A.; Samotorcan, L.; Consortium, T. S.; Quake, S. R.; Leskovec, J.
Score: 66.8, Published: 2023-11-29 DOI: 10.1101/2023.11.28.568918
Developing a universal representation of cells which encompasses the tremendous molecular diversity of cell types within the human body and more generally, across species, would be transformative for cell biology. Recent work using single-cell transcriptomic approaches to create molecular definitions of cell types in the form of cell atlases has provided the necessary data for such an endeavor." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cell-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-03T10:38:01+00:00" />
<meta property="article:modified_time" content="2023-12-03T10:38:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cell biology"/>
<meta name="twitter:description" content="Universal Cell Embeddings: A Foundation Model for Cell Biology
Authors: Rosen, Y.; Roohani, Y.; Agrawal, A.; Samotorcan, L.; Consortium, T. S.; Quake, S. R.; Leskovec, J.
Score: 66.8, Published: 2023-11-29 DOI: 10.1101/2023.11.28.568918
Developing a universal representation of cells which encompasses the tremendous molecular diversity of cell types within the human body and more generally, across species, would be transformative for cell biology. Recent work using single-cell transcriptomic approaches to create molecular definitions of cell types in the form of cell atlases has provided the necessary data for such an endeavor."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cell biology",
      "item": "https://trxiv.yorks0n.com/posts/cell-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cell biology",
  "name": "cell biology",
  "description": "Universal Cell Embeddings: A Foundation Model for Cell Biology\nAuthors: Rosen, Y.; Roohani, Y.; Agrawal, A.; Samotorcan, L.; Consortium, T. S.; Quake, S. R.; Leskovec, J.\nScore: 66.8, Published: 2023-11-29 DOI: 10.1101/2023.11.28.568918\nDeveloping a universal representation of cells which encompasses the tremendous molecular diversity of cell types within the human body and more generally, across species, would be transformative for cell biology. Recent work using single-cell transcriptomic approaches to create molecular definitions of cell types in the form of cell atlases has provided the necessary data for such an endeavor.",
  "keywords": [
    
  ],
  "articleBody": " Universal Cell Embeddings: A Foundation Model for Cell Biology\nAuthors: Rosen, Y.; Roohani, Y.; Agrawal, A.; Samotorcan, L.; Consortium, T. S.; Quake, S. R.; Leskovec, J.\nScore: 66.8, Published: 2023-11-29 DOI: 10.1101/2023.11.28.568918\nDeveloping a universal representation of cells which encompasses the tremendous molecular diversity of cell types within the human body and more generally, across species, would be transformative for cell biology. Recent work using single-cell transcriptomic approaches to create molecular definitions of cell types in the form of cell atlases has provided the necessary data for such an endeavor. Here, we present the Universal Cell Embedding (UCE) foundation model. UCE was trained on a corpus of cell atlas data from human and other species in a completely self-supervised way without any data annotations. UCE offers a unified biological latent space that can represent any cell, regardless of tissue or species. This universal cell embedding captures important biological variation despite the presence of experimental noise across diverse datasets. An important aspect of UCEs universality is that any new cell from any organism can be mapped to this embedding space with no additional data labeling, model training or fine-tuning. We applied UCE to create the Integrated Mega-scale Atlas, embedding 36 million cells, with more than 1,000 uniquely named cell types, from hundreds of experiments, dozens of tissues and eight species. We uncovered new insights about the organization of cell types and tissues within this universal cell embedding space, and leveraged it to infer function of newly discovered cell types. UCEs embedding space exhibits emergent behavior, uncovering new biology that it was never explicitly trained for, such as identifying developmental lineages and embedding data from novel species not included in the training set. Overall, by enabling a universal representation for every cell state and type, UCE provides a valuable tool for analysis, annotation and hypothesis generation as the scale and diversity of single cell datasets continues to grow.\nProliferation history and transcription factor levels drive direct conversion\nAuthors: Wang, N. B.; Lende-Dorn, B. A.; Adewumi, H. O.; Beitz, A. M.; Han, P.; O'Shea, T. M.; Galloway, K. E.\nScore: 55.7, Published: 2023-11-27 DOI: 10.1101/2023.11.26.568736\nThe sparse and stochastic nature of reprogramming has obscured our understanding of how transcription factors drive cells to new identities. To overcome this limit, we developed a compact, portable reprogramming system that increases direct conversion of fibroblasts to motor neurons by two orders of magnitude. We show that subpopulations with different reprogramming potentials are distinguishable by proliferation history. By controlling for proliferation history and titrating each transcription factor, we find that conversion correlates with levels of the pioneer transcription factor Ngn2, whereas conversion shows a biphasic response to Lhx3. Increasing the proliferation rate of adult human fibroblasts generates morphologically mature, induced motor neurons at high rates. Using compact, optimized, polycistronic cassettes, we generate motor neurons that graft with the murine central nervous system, demonstrating the potential for in vivo therapies. One Sentence SummaryUsing a systems and synthetic biology approach to study the molecular determinants of reprogramming, we find that proliferation history and transcription factor levels drive cell fate in direct conversion to motor neurons. HighlightsO_LIminimal high-efficiency cocktail allows systematic interrogation of the reprogramming process C_LIO_LIhistory provides a principal axis to distinguish transcription factors influence C_LIO_LIof the transcription factor cocktail impacts reprogramming efficiency and dynamics C_LIO_LIof individual transcription factors differentially influence the rate of reprogramming C_LIO_LIearly hyperproliferation increases direct conversion of adult human fibroblasts C_LIO_LIptimal cocktail allows neurotrophic factor-free reprogramming and in vivo grafting C_LI\nDynamic label-free analysis of SARS-CoV-2 infection reveals virus-induced subcellular remodeling.\nAuthors: Saunders, N.; Monel, B.; Cayet, N.; Archetti, L.; Moreno, H.; Jeanne, A.; Marguier, A.; Wai, T.; Schwartz, O.; Frechin, M.\nScore: 126.2, Published: 2023-11-17 DOI: 10.1101/2023.11.16.567378\nAssessing the impact of SARS-CoV-2 on organelle dynamics allows a better understanding of the mechanisms of viral replication. We combine label-free holo-tomographic microscopy (HTM) with Artificial Intelligence (AI) to visualize and quantify the subcellular changes triggered by SARS-CoV-2 infection. We study the dynamics of shape, position and dry mass of nucleoli, nuclei, lipid droplets (LD) and mitochondria within hundreds of single cells from early infection to syncytia formation and death. SARS-CoV-2 infection enlarges nucleoli, perturbs LD, changes mitochondrial shape and dry mass, and separates LD from mitochondria. We then used Bayesian statistics on organelle dry mass states to define organelle cross-regulation (OCR) networks and report modifications of OCR that are triggered by infection and syncytia formation. Our work highlights the subcellular remodeling induced by SARS-CoV-2 infection and provides a new AI-enhanced, label-free methodology to study in real-time the dynamics of cell populations and their content.\nBiomolecular condensates in fungi are tuned to function at specific temperatures\nAuthors: Stormo, B. M.; McLaughlin, G.; Frederick, L.; Jalihal, A.; Cole, S.; Seim, I.; Dietrich, F.; Gladfelter, A. S.\nScore: 40.1, Published: 2023-11-27 DOI: 10.1101/2023.11.27.568884\nTemperature can impact every reaction and molecular interaction essential to a cell. For organisms that cannot regulate their own temperature, a major challenge is how to adapt to temperatures that fluctuate unpredictability and on variable timescales. Biomolecular condensation offers a possible mechanism for encoding temperature-responsiveness and robustness into cell biochemistry and organization. To explore this idea, we examined temperature adaptation in a filamentous-growing fungus called Ashbya gossypii that engages biomolecular condensates containing the RNA-binding protein Whi3 to regulate mitosis and morphogenesis. We collected wild isolates of Ashbya that originate in different climates and found that mitotic asynchrony and polarized growth, which are known to be controlled by the condensation of Whi3, are temperature sensitive. Sequence analysis in the wild strains revealed changes to specific domains within Whi3 known to be important in condensate formation. Using an in vitro condensate reconstitution assay we found that temperature impacts the relative abundance of protein to RNA within condensates and that this directly impacts the material properties of the droplets. Finally, we found that exchanging Whi3 genes between warm and cold isolates was sufficient to rescue some, but not all, condensate-related phenotypes. Together these data demonstrate that material properties of Whi3 condensates are temperature sensitive, that these properties are important for function, and that sequence optimizes properties for a given climate.\nCryoET reveals actin filaments within platelet microtubules\nAuthors: Tsuji, C.; Bradshaw, M.; Allen, M.; Jackson, M. L.; Mantell, J.; Borucu, U.; Poole, A. W.; Verkade, P.; Hers, I.; Paul, D. M.; Dodding, M.\nScore: 34.0, Published: 2023-11-25 DOI: 10.1101/2023.11.24.568450\nCrosstalk between the actin and microtubule cytoskeletons is essential for many cellular processes. Recent studies have shown that microtubules and F-actin can assemble to form a composite structure where F-actin occupies the microtubule lumen. Whether these cytoskeletal hybrids exist in physiological settings and how they are formed is unclear. Here, we show that the short-crossover Class I actin filament previously identified inside microtubules in human HAP1 cells is cofilin-bound F-actin. Lumenal F-actin can be reconstituted in vitro, but cofilin is not essential. Moreover, actin filaments with both cofilin-bound and canonical morphologies reside within human platelet microtubules under physiological conditions. We propose that stress placed upon the microtubule network during motor-driven microtubule looping and sliding may facilitate the incorporation of actin into microtubules.\nSec7 regulatory domains scaffold autoinhibited and active conformations\nAuthors: Brownfield, B. A.; Richardson, B. C.; Halaby, S. L.; Fromme, J. C.\nScore: 20.1, Published: 2023-11-22 DOI: 10.1101/2023.11.22.568272\nThe late stages of Golgi maturation involve a series of sequential trafficking events in which cargo-laden vesicles are produced and targeted to multiple distinct subcellular destinations. Each of these vesicle biogenesis events requires activation of an Arf GTPase by the Sec7/BIG guanine nucleotide exchange factor (GEF). Sec7 localization and activity is regulated by autoinhibition, positive feedback, and interaction with other GTPases. Although these mechanisms have been characterized biochemically, we lack a clear picture of how GEF localization and activity is modulated by these signals. Here we report the cryoEM structure of full-length Sec7 in its autoinhibited form, revealing the architecture of its multiple regulatory domains. We use functional experiments to determine the basis for autoinhibition and use structural predictions to produce a model for an active conformation of the GEF that is supported empirically. This study therefore elucidates the conformational transition that Sec7 undergoes to become active on the organelle membrane surface.\nHigh resolution long-read telomere sequencing reveals dynamic mechanisms in aging and cancer\nAuthors: Schmidt, T. T.; Tyer, C.; Rughani, P.; Haggblom, C.; Jones, J. R.; Dai, X.; Frazer, K. A.; Gage, F. H.; Juul, S.; Hickey, S.; Karlseder, J.\nScore: 19.3, Published: 2023-11-29 DOI: 10.1101/2023.11.28.569082\nTelomeres are the protective nucleoprotein structures at the end of linear eukaryotic chromosomes. Telomeres repetitive nature and length have traditionally challenged the precise assessment of the composition and length of individual human telomeres. Here, we present Telo-seq to resolve bulk, chromosome arm-specific and allele-specific human telomere lengths using Oxford Nanopore Technologies native long-read sequencing. Telo-seq resolves telomere shortening in five population doubling increments and reveals intrasample, chromosome arm-specific, allele-specific telomere length heterogeneity. Telo-seq can reliably discriminate between telomerase- and ALT-positive cancer cell lines. Thus, Telo-seq is a novel tool to study telomere biology during development, aging, and cancer at unprecedented resolution.\nSelective activation of intracellular β1AR using a spatially restricted antagonist\nAuthors: Liccardo, F.; Morstein, J.; Lin, T.-Y.; Pampel, J.; Shokat, K.; Irannejad, R.\nScore: 16.1, Published: 2023-11-22 DOI: 10.1101/2023.11.22.568314\nG-protein-coupled receptors (GPCRs) regulate several physiological and pathological processes and represent the target of approximately 30% of FDA-approved drugs. GPCR-mediated signaling was thought to occur exclusively at the plasma membrane. However, recent studies have unveiled their presence and function at subcellular membrane compartments. There is a growing interest in studying compartmentalized signaling of GPCRs. This requires development of novel tools to separate GPCRs signaling at the plasma membrane from the ones initiated at intracellular compartments. We took advantage of the structural and pharmacological information available for {beta}1-adrenergic receptor ({beta}1AR), an exemplary GPCR that functions at subcellular compartments, and rationally designed spatially restricted antagonists. We generated a cell impermeable {beta}1AR antagonist by conjugating a suitable pharmacophore to a sulfonate-containing fluorophore. This cell-impermeable antagonist only inhibited {beta}1AR on the plasma membrane. In contrast, a cell permeable {beta}1AR agonist containing a non-sulfonated fluorophore, efficiently inhibited both the plasma membrane and Golgi pools of {beta}1ARs. Furthermore, the cell impermeable antagonist selectively inhibited the phosphorylation of downstream effectors of PKA proximal to the plasma membrane in adult cardiomyocytes while {beta}1AR intracellular pool remained active. Our tools offer promising avenues for investigating compartmentalized {beta}1AR signaling in various context, potentially advancing our understanding of {beta}1AR-mediated cellular responses in health and disease. They also offer a general strategy to study compartmentalized signaling for other GPCRs in various biological systems.\nA fluorogenic complementation tool kit for interrogating lipid droplet-organelle interaction\nAuthors: Li, X.; Gamuyao, R.; Wu, M.-L.; Cho, W. J.; Kurtz, N. B.; King, S. V.; Petersen, R. A.; Stabley, D. R.; Lindow, C.; Climer, L.; Shirinifard, A.; Ferrara, F.; Throm, R. E.; Robinson, C. G.; Carisey, A.; Tebo, A. G.; Chang, C.-L.\nScore: 13.3, Published: 2023-11-29 DOI: 10.1101/2023.11.29.569289\nContact sites between lipid droplets and other organelles are essential for cellular lipid and energy homeostasis. Detection of these contact sites at nanometer scale over time in living cells is challenging. Here, we developed a tool kit for detecting contact sites based on Fluorogen- Activated Bimolecular complementation at CONtact sites, FABCON, using a reversible, low affinity split fluorescent protein, splitFAST. FABCON labels contact sites with minimal perturbation to organelle interaction. Via FABCON, we quantitatively demonstrated that endoplasmic reticulum (ER)- and mitochondria (mito)-lipid droplet contact sites are dynamic foci in distinct metabolic conditions, such as during lipid droplet biogenesis and consumption. An automated analysis pipeline further classified individual contact sites into distinct subgroups based on size, likely reflecting differential regulation and function. Moreover, FABCON is generalizable to visualize a repertoire of organelle contact sites including ER-mito. Altogether, FABCON reveals insights into the dynamic regulation of lipid droplet-organelle contact sites and generates new hypotheses for further mechanistical interrogation during metabolic switch.\nExploring the molecular composition of the multipass translocon in its native membrane environment\nAuthors: Gemmer, M.; Chaillet, M.; Forster, F.\nScore: 11.9, Published: 2023-11-29 DOI: 10.1101/2023.11.28.569136\nMultispanning membrane proteins are inserted into the endoplasmic reticulum membrane by the ribosome-bound multipass translocon machinery. Based on cryo-electron tomography and extensive subtomogram analysis, we reveal the composition and arrangement of multipass translocon components in their native membrane environment. The intramembrane chaperone complex PAT and the translocon associated protein (TRAP) complex associate substoichiometrically with the multipass translocon in a translation-dependent manner. While PAT is preferentially recruited to active complexes, TRAP primarily associates with inactive translocons. The subtomogram average of the TRAP-multipass translocon reveals intermolecular contacts between the luminal domains of TRAP and an unknown subunit of the BOS complex. AlphaFold modeling suggests this protein is NOMO, bridging the luminal domains of nicalin and TRAP. Collectively, our results visualize the interplay of accessory factors associated with multipass membrane protein biogenesis under near-native conditions.\n",
  "wordCount" : "2123",
  "inLanguage": "en",
  "datePublished": "2023-12-03T10:38:01Z",
  "dateModified": "2023-12-03T10:38:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cell-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cell biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 3, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.568918">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.568918" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.568918">
        <p class="paperTitle">Universal Cell Embeddings: A Foundation Model for Cell Biology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.568918" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.568918" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rosen, Y.; Roohani, Y.; Agrawal, A.; Samotorcan, L.; Consortium, T. S.; Quake, S. R.; Leskovec, J.</p>
        <p class="info">Score: 66.8, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.568918' target='https://doi.org/10.1101/2023.11.28.568918'> 10.1101/2023.11.28.568918</a></p>
        <p class="abstract">Developing a universal representation of cells which encompasses the tremendous molecular diversity of cell types within the human body and more generally, across species, would be transformative for cell biology. Recent work using single-cell transcriptomic approaches to create molecular definitions of cell types in the form of cell atlases has provided the necessary data for such an endeavor. Here, we present the Universal Cell Embedding (UCE) foundation model. UCE was trained on a corpus of cell atlas data from human and other species in a completely self-supervised way without any data annotations. UCE offers a unified biological latent space that can represent any cell, regardless of tissue or species. This universal cell embedding captures important biological variation despite the presence of experimental noise across diverse datasets. An important aspect of UCEs universality is that any new cell from any organism can be mapped to this embedding space with no additional data labeling, model training or fine-tuning. We applied UCE to create the Integrated Mega-scale Atlas, embedding 36 million cells, with more than 1,000 uniquely named cell types, from hundreds of experiments, dozens of tissues and eight species. We uncovered new insights about the organization of cell types and tissues within this universal cell embedding space, and leveraged it to infer function of newly discovered cell types. UCEs embedding space exhibits emergent behavior, uncovering new biology that it was never explicitly trained for, such as identifying developmental lineages and embedding data from novel species not included in the training set. Overall, by enabling a universal representation for every cell state and type, UCE provides a valuable tool for analysis, annotation and hypothesis generation as the scale and diversity of single cell datasets continues to grow.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.26.568736">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.26.568736" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.26.568736">
        <p class="paperTitle">Proliferation history and transcription factor levels drive direct conversion</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.26.568736" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.26.568736" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, N. B.; Lende-Dorn, B. A.; Adewumi, H. O.; Beitz, A. M.; Han, P.; O&#39;Shea, T. M.; Galloway, K. E.</p>
        <p class="info">Score: 55.7, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.26.568736' target='https://doi.org/10.1101/2023.11.26.568736'> 10.1101/2023.11.26.568736</a></p>
        <p class="abstract">The sparse and stochastic nature of reprogramming has obscured our understanding of how transcription factors drive cells to new identities. To overcome this limit, we developed a compact, portable reprogramming system that increases direct conversion of fibroblasts to motor neurons by two orders of magnitude. We show that subpopulations with different reprogramming potentials are distinguishable by proliferation history. By controlling for proliferation history and titrating each transcription factor, we find that conversion correlates with levels of the pioneer transcription factor Ngn2, whereas conversion shows a biphasic response to Lhx3. Increasing the proliferation rate of adult human fibroblasts generates morphologically mature, induced motor neurons at high rates. Using compact, optimized, polycistronic cassettes, we generate motor neurons that graft with the murine central nervous system, demonstrating the potential for in vivo therapies.

One Sentence SummaryUsing a systems and synthetic biology approach to study the molecular determinants of reprogramming, we find that proliferation history and transcription factor levels drive cell fate in direct conversion to motor neurons.

HighlightsO_LIminimal high-efficiency cocktail allows systematic interrogation of the reprogramming process
C_LIO_LIhistory provides a principal axis to distinguish transcription factors influence
C_LIO_LIof the transcription factor cocktail impacts reprogramming efficiency and dynamics
C_LIO_LIof individual transcription factors differentially influence the rate of reprogramming
C_LIO_LIearly hyperproliferation increases direct conversion of adult human fibroblasts
C_LIO_LIptimal cocktail allows neurotrophic factor-free reprogramming and in vivo grafting
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.567378">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.567378" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.567378">
        <p class="paperTitle">Dynamic label-free analysis of SARS-CoV-2 infection reveals virus-induced subcellular remodeling.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.567378" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.567378" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Saunders, N.; Monel, B.; Cayet, N.; Archetti, L.; Moreno, H.; Jeanne, A.; Marguier, A.; Wai, T.; Schwartz, O.; Frechin, M.</p>
        <p class="info">Score: 126.2, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.567378' target='https://doi.org/10.1101/2023.11.16.567378'> 10.1101/2023.11.16.567378</a></p>
        <p class="abstract">Assessing the impact of SARS-CoV-2 on organelle dynamics allows a better understanding of the mechanisms of viral replication. We combine label-free holo-tomographic microscopy (HTM) with Artificial Intelligence (AI) to visualize and quantify the subcellular changes triggered by SARS-CoV-2 infection. We study the dynamics of shape, position and dry mass of nucleoli, nuclei, lipid droplets (LD) and mitochondria within hundreds of single cells from early infection to syncytia formation and death. SARS-CoV-2 infection enlarges nucleoli, perturbs LD, changes mitochondrial shape and dry mass, and separates LD from mitochondria. We then used Bayesian statistics on organelle dry mass states to define organelle cross-regulation (OCR) networks and report modifications of OCR that are triggered by infection and syncytia formation. Our work highlights the subcellular remodeling induced by SARS-CoV-2 infection and provides a new AI-enhanced, label-free methodology to study in real-time the dynamics of cell populations and their content.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.568884">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.568884" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.568884">
        <p class="paperTitle">Biomolecular condensates in fungi are tuned to function at specific temperatures</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.568884" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.568884" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stormo, B. M.; McLaughlin, G.; Frederick, L.; Jalihal, A.; Cole, S.; Seim, I.; Dietrich, F.; Gladfelter, A. S.</p>
        <p class="info">Score: 40.1, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.568884' target='https://doi.org/10.1101/2023.11.27.568884'> 10.1101/2023.11.27.568884</a></p>
        <p class="abstract">Temperature can impact every reaction and molecular interaction essential to a cell. For organisms that cannot regulate their own temperature, a major challenge is how to adapt to temperatures that fluctuate unpredictability and on variable timescales. Biomolecular condensation offers a possible mechanism for encoding temperature-responsiveness and robustness into cell biochemistry and organization. To explore this idea, we examined temperature adaptation in a filamentous-growing fungus called Ashbya gossypii that engages biomolecular condensates containing the RNA-binding protein Whi3 to regulate mitosis and morphogenesis. We collected wild isolates of Ashbya that originate in different climates and found that mitotic asynchrony and polarized growth, which are known to be controlled by the condensation of Whi3, are temperature sensitive. Sequence analysis in the wild strains revealed changes to specific domains within Whi3 known to be important in condensate formation. Using an in vitro condensate reconstitution assay we found that temperature impacts the relative abundance of protein to RNA within condensates and that this directly impacts the material properties of the droplets. Finally, we found that exchanging Whi3 genes between warm and cold isolates was sufficient to rescue some, but not all, condensate-related phenotypes. Together these data demonstrate that material properties of Whi3 condensates are temperature sensitive, that these properties are important for function, and that sequence optimizes properties for a given climate.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.24.568450">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.24.568450" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.24.568450">
        <p class="paperTitle">CryoET reveals actin filaments within platelet microtubules</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.24.568450" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.24.568450" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tsuji, C.; Bradshaw, M.; Allen, M.; Jackson, M. L.; Mantell, J.; Borucu, U.; Poole, A. W.; Verkade, P.; Hers, I.; Paul, D. M.; Dodding, M.</p>
        <p class="info">Score: 34.0, Published: 2023-11-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.24.568450' target='https://doi.org/10.1101/2023.11.24.568450'> 10.1101/2023.11.24.568450</a></p>
        <p class="abstract">Crosstalk between the actin and microtubule cytoskeletons is essential for many cellular processes. Recent studies have shown that microtubules and F-actin can assemble to form a composite structure where F-actin occupies the microtubule lumen. Whether these cytoskeletal hybrids exist in physiological settings and how they are formed is unclear. Here, we show that the short-crossover Class I actin filament previously identified inside microtubules in human HAP1 cells is cofilin-bound F-actin. Lumenal F-actin can be reconstituted in vitro, but cofilin is not essential. Moreover, actin filaments with both cofilin-bound and canonical morphologies reside within human platelet microtubules under physiological conditions. We propose that stress placed upon the microtubule network during motor-driven microtubule looping and sliding may facilitate the incorporation of actin into microtubules.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.568272">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.568272" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.568272">
        <p class="paperTitle">Sec7 regulatory domains scaffold autoinhibited and active conformations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.568272" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.568272" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brownfield, B. A.; Richardson, B. C.; Halaby, S. L.; Fromme, J. C.</p>
        <p class="info">Score: 20.1, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.568272' target='https://doi.org/10.1101/2023.11.22.568272'> 10.1101/2023.11.22.568272</a></p>
        <p class="abstract">The late stages of Golgi maturation involve a series of sequential trafficking events in which cargo-laden vesicles are produced and targeted to multiple distinct subcellular destinations. Each of these vesicle biogenesis events requires activation of an Arf GTPase by the Sec7/BIG guanine nucleotide exchange factor (GEF). Sec7 localization and activity is regulated by autoinhibition, positive feedback, and interaction with other GTPases. Although these mechanisms have been characterized biochemically, we lack a clear picture of how GEF localization and activity is modulated by these signals. Here we report the cryoEM structure of full-length Sec7 in its autoinhibited form, revealing the architecture of its multiple regulatory domains. We use functional experiments to determine the basis for autoinhibition and use structural predictions to produce a model for an active conformation of the GEF that is supported empirically. This study therefore elucidates the conformational transition that Sec7 undergoes to become active on the organelle membrane surface.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.569082">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.569082" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.569082">
        <p class="paperTitle">High resolution long-read telomere sequencing reveals dynamic mechanisms in aging and cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.569082" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.569082" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schmidt, T. T.; Tyer, C.; Rughani, P.; Haggblom, C.; Jones, J. R.; Dai, X.; Frazer, K. A.; Gage, F. H.; Juul, S.; Hickey, S.; Karlseder, J.</p>
        <p class="info">Score: 19.3, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.569082' target='https://doi.org/10.1101/2023.11.28.569082'> 10.1101/2023.11.28.569082</a></p>
        <p class="abstract">Telomeres are the protective nucleoprotein structures at the end of linear eukaryotic chromosomes. Telomeres repetitive nature and length have traditionally challenged the precise assessment of the composition and length of individual human telomeres. Here, we present Telo-seq to resolve bulk, chromosome arm-specific and allele-specific human telomere lengths using Oxford Nanopore Technologies native long-read sequencing. Telo-seq resolves telomere shortening in five population doubling increments and reveals intrasample, chromosome arm-specific, allele-specific telomere length heterogeneity. Telo-seq can reliably discriminate between telomerase- and ALT-positive cancer cell lines. Thus, Telo-seq is a novel tool to study telomere biology during development, aging, and cancer at unprecedented resolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.568314">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.568314" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.568314">
        <p class="paperTitle">Selective activation of intracellular β1AR using a spatially restricted antagonist</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.568314" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.568314" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liccardo, F.; Morstein, J.; Lin, T.-Y.; Pampel, J.; Shokat, K.; Irannejad, R.</p>
        <p class="info">Score: 16.1, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.568314' target='https://doi.org/10.1101/2023.11.22.568314'> 10.1101/2023.11.22.568314</a></p>
        <p class="abstract">G-protein-coupled receptors (GPCRs) regulate several physiological and pathological processes and represent the target of approximately 30% of FDA-approved drugs. GPCR-mediated signaling was thought to occur exclusively at the plasma membrane. However, recent studies have unveiled their presence and function at subcellular membrane compartments. There is a growing interest in studying compartmentalized signaling of GPCRs. This requires development of novel tools to separate GPCRs signaling at the plasma membrane from the ones initiated at intracellular compartments. We took advantage of the structural and pharmacological information available for {beta}1-adrenergic receptor ({beta}1AR), an exemplary GPCR that functions at subcellular compartments, and rationally designed spatially restricted antagonists. We generated a cell impermeable {beta}1AR antagonist by conjugating a suitable pharmacophore to a sulfonate-containing fluorophore. This cell-impermeable antagonist only inhibited {beta}1AR on the plasma membrane. In contrast, a cell permeable {beta}1AR agonist containing a non-sulfonated fluorophore, efficiently inhibited both the plasma membrane and Golgi pools of {beta}1ARs. Furthermore, the cell impermeable antagonist selectively inhibited the phosphorylation of downstream effectors of PKA proximal to the plasma membrane in adult cardiomyocytes while {beta}1AR intracellular pool remained active. Our tools offer promising avenues for investigating compartmentalized {beta}1AR signaling in various context, potentially advancing our understanding of {beta}1AR-mediated cellular responses in health and disease. They also offer a general strategy to study compartmentalized signaling for other GPCRs in various biological systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569289">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569289" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569289">
        <p class="paperTitle">A fluorogenic complementation tool kit for interrogating lipid droplet-organelle interaction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569289" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569289" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, X.; Gamuyao, R.; Wu, M.-L.; Cho, W. J.; Kurtz, N. B.; King, S. V.; Petersen, R. A.; Stabley, D. R.; Lindow, C.; Climer, L.; Shirinifard, A.; Ferrara, F.; Throm, R. E.; Robinson, C. G.; Carisey, A.; Tebo, A. G.; Chang, C.-L.</p>
        <p class="info">Score: 13.3, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569289' target='https://doi.org/10.1101/2023.11.29.569289'> 10.1101/2023.11.29.569289</a></p>
        <p class="abstract">Contact sites between lipid droplets and other organelles are essential for cellular lipid and energy homeostasis. Detection of these contact sites at nanometer scale over time in living cells is challenging. Here, we developed a tool kit for detecting contact sites based on Fluorogen- Activated Bimolecular complementation at CONtact sites, FABCON, using a reversible, low affinity split fluorescent protein, splitFAST. FABCON labels contact sites with minimal perturbation to organelle interaction. Via FABCON, we quantitatively demonstrated that endoplasmic reticulum (ER)- and mitochondria (mito)-lipid droplet contact sites are dynamic foci in distinct metabolic conditions, such as during lipid droplet biogenesis and consumption. An automated analysis pipeline further classified individual contact sites into distinct subgroups based on size, likely reflecting differential regulation and function. Moreover, FABCON is generalizable to visualize a repertoire of organelle contact sites including ER-mito. Altogether, FABCON reveals insights into the dynamic regulation of lipid droplet-organelle contact sites and generates new hypotheses for further mechanistical interrogation during metabolic switch.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.569136">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.569136" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.569136">
        <p class="paperTitle">Exploring the molecular composition of the multipass translocon in its native membrane environment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.569136" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.569136" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gemmer, M.; Chaillet, M.; Forster, F.</p>
        <p class="info">Score: 11.9, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.569136' target='https://doi.org/10.1101/2023.11.28.569136'> 10.1101/2023.11.28.569136</a></p>
        <p class="abstract">Multispanning membrane proteins are inserted into the endoplasmic reticulum membrane by the ribosome-bound multipass translocon machinery. Based on cryo-electron tomography and extensive subtomogram analysis, we reveal the composition and arrangement of multipass translocon components in their native membrane environment. The intramembrane chaperone complex PAT and the translocon associated protein (TRAP) complex associate substoichiometrically with the multipass translocon in a translation-dependent manner. While PAT is preferentially recruited to active complexes, TRAP primarily associates with inactive translocons. The subtomogram average of the TRAP-multipass translocon reveals intermolecular contacts between the luminal domains of TRAP and an unknown subunit of the BOS complex. AlphaFold modeling suggests this protein is NOMO, bridging the luminal domains of nicalin and TRAP. Collectively, our results visualize the interplay of accessory factors associated with multipass membrane protein biogenesis under near-native conditions.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
