<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cell biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cell biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Human cytomegalovirus deploys molecular mimicry to recruit VPS4A to sites of virus assembly
Authors: Butt, B. G.; Fischer, D.; Rep, A. R.; Schauflinger, M.; Read, C.; Bock, T.; Hirner, M.; Graham, S. C.; von Einem, J.
Score: 17.6, Published: 2024-01-04 DOI: 10.1101/2024.01.04.572781
The AAA-type ATPase VPS4 is recruited by proteins of the endosomal sorting complex required for transport III (ESCRT-III) to catalyse membrane constriction and membrane fission. VPS4A accumulates at the cytoplasmic viral assembly complex (cVAC) of cells infected with human cytomegalovirus (HCMV), the site where nascent virus particles obtain their membrane envelope.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cell-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cell biology" />
<meta property="og:description" content="Human cytomegalovirus deploys molecular mimicry to recruit VPS4A to sites of virus assembly
Authors: Butt, B. G.; Fischer, D.; Rep, A. R.; Schauflinger, M.; Read, C.; Bock, T.; Hirner, M.; Graham, S. C.; von Einem, J.
Score: 17.6, Published: 2024-01-04 DOI: 10.1101/2024.01.04.572781
The AAA-type ATPase VPS4 is recruited by proteins of the endosomal sorting complex required for transport III (ESCRT-III) to catalyse membrane constriction and membrane fission. VPS4A accumulates at the cytoplasmic viral assembly complex (cVAC) of cells infected with human cytomegalovirus (HCMV), the site where nascent virus particles obtain their membrane envelope." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cell-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-14T10:38:40+00:00" />
<meta property="article:modified_time" content="2024-01-14T10:38:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cell biology"/>
<meta name="twitter:description" content="Human cytomegalovirus deploys molecular mimicry to recruit VPS4A to sites of virus assembly
Authors: Butt, B. G.; Fischer, D.; Rep, A. R.; Schauflinger, M.; Read, C.; Bock, T.; Hirner, M.; Graham, S. C.; von Einem, J.
Score: 17.6, Published: 2024-01-04 DOI: 10.1101/2024.01.04.572781
The AAA-type ATPase VPS4 is recruited by proteins of the endosomal sorting complex required for transport III (ESCRT-III) to catalyse membrane constriction and membrane fission. VPS4A accumulates at the cytoplasmic viral assembly complex (cVAC) of cells infected with human cytomegalovirus (HCMV), the site where nascent virus particles obtain their membrane envelope."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cell biology",
      "item": "https://trxiv.yorks0n.com/posts/cell-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cell biology",
  "name": "cell biology",
  "description": "Human cytomegalovirus deploys molecular mimicry to recruit VPS4A to sites of virus assembly\nAuthors: Butt, B. G.; Fischer, D.; Rep, A. R.; Schauflinger, M.; Read, C.; Bock, T.; Hirner, M.; Graham, S. C.; von Einem, J.\nScore: 17.6, Published: 2024-01-04 DOI: 10.1101/2024.01.04.572781\nThe AAA-type ATPase VPS4 is recruited by proteins of the endosomal sorting complex required for transport III (ESCRT-III) to catalyse membrane constriction and membrane fission. VPS4A accumulates at the cytoplasmic viral assembly complex (cVAC) of cells infected with human cytomegalovirus (HCMV), the site where nascent virus particles obtain their membrane envelope.",
  "keywords": [
    
  ],
  "articleBody": " Human cytomegalovirus deploys molecular mimicry to recruit VPS4A to sites of virus assembly\nAuthors: Butt, B. G.; Fischer, D.; Rep, A. R.; Schauflinger, M.; Read, C.; Bock, T.; Hirner, M.; Graham, S. C.; von Einem, J.\nScore: 17.6, Published: 2024-01-04 DOI: 10.1101/2024.01.04.572781\nThe AAA-type ATPase VPS4 is recruited by proteins of the endosomal sorting complex required for transport III (ESCRT-III) to catalyse membrane constriction and membrane fission. VPS4A accumulates at the cytoplasmic viral assembly complex (cVAC) of cells infected with human cytomegalovirus (HCMV), the site where nascent virus particles obtain their membrane envelope. Here we show that VPS4A is recruited to the cVAC via interaction with pUL71. Sequence analysis, deep-learning structure prediction, molecular dynamics and mutagenic analysis identify a short peptide motif in the C-terminal region of pUL71 that is necessary and sufficient for the interaction with VPS4A. This motif is predicted to bind the same groove of the N-terminal VPS4A Microtubule-Interacting and Trafficking (MIT) domain as the Type 2 MIT-Interacting Motif (MIM2) of cellular ESCRT-III components, and this viral MIM2-like motif (vMIM2) is conserved across {beta}-herpesvirus pUL71 homologues. However, recruitment of VPS4A by pUL71 is dispensable for HCMV morphogenesis or replication and the function of the conserved vMIM2 during infection remains enigmatic. VPS4-recruitment via a vMIM2 represents a previously unknown mechanism of molecular mimicry in viruses, extending previous observations that herpesviruses encode proteins with structural and functional homology to cellular ESCRT-III components.\nShaping up genomes: Prokaryotic roots and eukaryotic diversification of SMC complexes\nAuthors: van Hooff, J. J. E.; Raas, M. W. D.; Tromer, E. C.; Eme, L.\nScore: 15.7, Published: 2024-01-08 DOI: 10.1101/2024.01.07.573240\nAcross the tree of life, SMC complexes organize, segregate and regulate DNA. In contrast to prokaryotes, which often possess one SMC complex, eukaryotes usually have four such complexes. This expanded set is involved in managing the considerably larger nuclear genomes of eukaryotes, which are distributed across multiple chromosomes. Despite their essential functions, SMC complexes exhibit variations across model eukaryotes, suggesting an even greater variety of these complexes that remains unexplored. Here, we aimed to uncover the diversity and evolution of SMC complexes across eukaryotes, and their deeper, prokaryotic evolutionary origins. For this, we conducted in-depth comparative genomic and phylogenetic analyses of SMC complexes. We show that the last eukaryotic common ancestor (LECA) likely had fully-fledged versions of all four complexes and that the condensin II complex was later lost at least 30 times in eukaryotes. We report evidence that proteins previously designated as functional analogs in various model organisms (e.g., Sororin, Securin, Nse5 and Nse6) are in fact genuine orthologs. Finally, we traced the prokaryotic origins of these complexes and propose that a single SMC complex duplicated in an early archaeon. Altogether, we provide a comprehensive overview of eukaryotic SMC complex diversity and evolution, both addressing and generating questions about their functioning in ancestral and contemporary organisms.\nGeneration of systemic rabbit chimeras via Induced Pluripotent Stem Cells Reprogrammed with KLF2, ERAS, and PRMT6\nAuthors: Perold, F.; Pham, H.-T.; Pijoff, Y.; Doerflinger, N.; Rival-Gervier, S.; Moulin, A.; Jouneau, L.; Pain, B.; Joly, T.; Duranthon, V.; Afanassieff, M.; Savatier, P.; Beaujean, N.\nScore: 8.6, Published: 2024-01-11 DOI: 10.1101/2024.01.10.575048\nLittle is known about the molecular underpinnings of pluripotent stem cells (PSCs) ability to colonize the epiblast of preimplantation embryos and generate chimeras. In our study, using rabbit PSCs as a model system, we conducted unbiased screening of a cDNA library that encodes a panel of 36 pluripotency factors. From this screening, we identified KLF2, ERAS and PRMT6, whose overexpression confers the ability for self-renewal in a KOSR/FGF2-free culture medium supplemented with LIF, activin A, PKC and WNT inhibitors. The reprogrammed cells acquired transcriptomic and epigenetic features of naive pluripotency, including the reactivation of the 2nd X-chromosome. Leveraging these PSC lines, we determined the transcriptomic signature of embryonic colonization-competence, demonstrating transcriptional repression of genes involved in MAPK, WNT, HIPPO, and EPH signaling pathways, alongside the activation of genes involved in amino-acid metabolism, NF-kB signaling, and p53 pathway. Remarkably, a subset of reprogrammed cells, expressing CD75 at a high level, gained the ability to produce chimeric fetuses with a high contribution from PSCs in all lineages.\nMechanism and regulation of cargo entry into the Commander recycling pathway\nAuthors: Butkovic, R.; Walker, A. P.; Healy, M. D.; McNally, K. E.; Liu, M.; Kato, K.; Collins, B. M.; Cullen, P. J.\nScore: 7.4, Published: 2024-01-11 DOI: 10.1101/2024.01.10.574988\nCommander is a multiprotein complex that orchestrates endosomal recycling of diverse integral cargo proteins and in humans is required for normal skeletal, brain, kidney, and cardiovascular development. While the structure of this complex has recently been described, the central question of how cargo proteins are selected for entry into the Commander recycling pathway remains unclear. Here using recombinant protein reconstitution and in silico predictions we identify the evolutionary conserved mechanism through which the unstructured carboxy-terminal tail of the integral protein adaptor sorting nexin-17 (SNX17) directly binds to the Retriever sub-complex of Commander. SNX17 adopts an autoinhibited conformation where its carboxy-terminal tail occupies the cargo binding groove. Competitive cargo binding overcomes this autoinhibition, promoting SNX17 endosomal residency and the release of the carboxy tail for Retriever association. Using molecular cell biology and high-resolution microscopy, we establish the central importance of SNX17-Retriever association in the handover of integrin and lipoprotein receptor cargoes into pre-existing endosomal retrieval sub-domains for entry into the recycling pathway. In describing the principal mechanism of cargo entry into the Commander recycling pathway we provide key insight into the function and regulation of this evolutionary conserved sorting complex.\nDetection of Fluorescent Protein Mechanical Switching in Cellulo\nAuthors: Shoyer, T. C.; Collins, K. L.; Ham, T. R.; Blanchard, A. T.; Malavade, J. N.; West, J. L.; Hoffman, B. D.\nScore: 6.4, Published: 2024-01-11 DOI: 10.1101/2024.01.10.575065\nThe ability of cells to sense and respond to mechanical forces is critical in many physiological and pathological processes. However, the mechanisms by which forces affect protein function inside cells remain unclear. Motivated by in vitro demonstrations of fluorescent proteins (FPs) undergoing reversible mechanical switching of fluorescence, we investigated if force-sensitive changes in FP function could be visualized in cells. Guided by a computational model of FP mechanical switching, we develop a formalism for its detection in Foerster resonance energy transfer (FRET)-based biosensors and demonstrate its occurrence in cellulo in a synthetic actin-crosslinker and the mechanical linker protein vinculin. We find that in cellulo mechanical switching is reversible and altered by manipulation of cellular force generation as well as force-sensitive bond dynamics of the biosensor. Together, this work describes a new framework for assessing FP mechanical stability and provides a means of probing force-sensitive protein function inside cells.\nDeubiquitinating enzyme mutagenesis screens identify a USP43 driven HIF-1 transcriptional response\nAuthors: Pauzaite, T.; Wit, N.; Seear, R. V.; Nathan, J. A.\nScore: 4.8, Published: 2024-01-12 DOI: 10.1101/2024.01.10.574971\nThe ubiquitination and proteasome-mediated degradation of Hypoxia Inducible Factors (HIFs) is central to metazoan oxygen-sensing, but the involvement of deubiquitinating enzymes (DUBs) in HIF signalling is less clear. Here, using a bespoke DUBs sgRNA library we conduct CRISPR/Cas9 mutagenesis screens to determine how DUBs are involved in HIF signalling. Alongside defining DUBs involved in HIF activation or suppression, we identify USP43 as a DUB required for efficient activation of a HIF response. USP43 is hypoxia regulated and selectively associates with the HIF-1alpha isoform, and while USP43 does not alter HIF-1alpha stability, it facilitates HIF-1 nuclear accumulation and binding to its target genes. Mechanistically, USP43 associates with 14-3-3 proteins in a hypoxia and phosphorylation dependent manner to increase the nuclear pool of HIF-1. Together, our results unveil the DUB landscape in HIF signalling, and highlight the multifunctionality of DUBs, illustrating that they can provide important signalling functions alongside their catalytic roles.\nTDP-43 pathology links innate and adaptive immunity in amyotrophic lateral sclerosis\nAuthors: Evangelista, B. A.; Ragusa, J. V.; Pellegrino, K.; Wu, Y.; Quiroga-Barber, I.; Cahalan, S. R.; Arooji, O. K.; Madren, J. A.; Schroeter, S.; Cozzarin, J.; Xie, L.; Chen, X.; White, K. K.; Ezzell, J. A.; Iannone, M. A.; Cohen, S.; Traub, R. E.; Li, X.; Bedlack, R.; Phanstiel, D. H.; Meeker, R.; Stanley, N.; Cohen, T. J.\nScore: 4.6, Published: 2024-01-10 DOI: 10.1101/2024.01.07.574541\nAmyotrophic lateral sclerosis is the most common fatal motor neuron disease. Approximately 90% of ALS patients exhibit pathology of the master RNA regulator, Transactive Response DNA Binding protein (TDP-43). Despite the prevalence TDP-43 pathology in ALS motor neurons, recent findings suggest immune dysfunction is a determinant of disease progression in patients. Whether TDP-43 pathology elicits disease-modifying immune responses in ALS remains underexplored. In this study, we demonstrate that TDP-43 pathology is internalized by antigen presenting cells, causes vesicle rupture, and leads to innate and adaptive immune cell activation. Using a multiplex imaging platform, we observed interactions between innate and adaptive immune cells near TDP-43 pathological lesions in ALS brain. We used a mass cytometry-based whole-blood stimulation assay to provide evidence that ALS patient peripheral immune cells exhibit responses to TDP-43 aggregates. Taken together, this study provides a novel link between TDP-43 pathology and ALS immune dysfunction, and further highlights the translational and diagnostic implications of monitoring and manipulating the ALS immune response.\nPharmacologically inducing regenerative cardiac cells by small molecule drugs\nAuthors: Zhou, W.; He, K.; Wang, C.; Wang, P.; Wang, D.; Wang, B.; Geng, H.; Lian, H.; Ma, T.; Nie, Y.; Ding, S.\nScore: 11.0, Published: 2023-12-23 DOI: 10.1101/2023.10.24.563872\nIn contrast to lower organisms, adult mammals cannot regenerate damaged hearts through de-differentiation of cardiomyocytes (CMs) into cells with regenerative capacity. Development of an enabling condition to induce such naturally unavailable cells with potential to proliferate and differentiate into CMs, i.e., regenerative cardiac cells (RCCs), in mammals will provide new insights and tools for heart regeneration. Here, a two-compound combination (2C), CHIR99021 and A-485, was identified to robustly induce RCCs from human embryonic stem cell (hESC) derived TNNT2+ CMs in vitro, which was confirmed by lineage tracing experiments. Functional analyses revealed that RCCs expressed a spectrum of genes essential for cardiogenesis and exhibited potential to become functional CMs, endothelial cells (ECs) and smooth muscle cells (SMCs). Consistent with the results in human cells, 2C-induced generation of RCCs were also observed in neonatal rats CMs in vitro and in vivo. Remarkably, administration of 2C can induce RCCs in adult mouse hearts and significantly improve survival and heart function in the mice undergoing myocardial infarction. Mechanistically, CHIR99021 is indispensable for transcriptional and epigenetic activation of genes essential for RCC, whereas A-485 mainly function to epigenetically down-regulate H3K27Ac and particularly H3K9Ac in CMs. Their combination specifically enhances both H3K27Ac and H3K9Ac on RCC genes, facilitating the establishment of RCC state dedifferentiated from CMs. Therefore, our findings demonstrated the feasibility and revealed the mechanisms of pharmacological induction of RCCs from endogenous CMs, which could offer a promising regenerative strategy to repair injured hearts.\nStarvation resistant cavefish reveal conserved mechanisms of starvation-induced hepatic lipotoxicity\nAuthors: Pozo-Morales, M.; Cobham, A. E.; Centola, C.; McKinney, M. C.; Liu, P.; Perazzolo, C.; Lefort, A.; Libert, F.; Bai, H.; Rohner, N.; Singh, S. P.\nScore: 3.2, Published: 2024-01-11 DOI: 10.1101/2024.01.10.574986\nStarvation causes the accumulation of lipid droplets in the liver, a somewhat counterintuitive phenomenon that is nevertheless conserved from flies to humans. Much like fatty liver resulting from overfeeding, hepatic lipid accumulation (steatosis) during undernourishment can lead to lipotoxicity and atrophy of the liver. Here, we found that while surface populations of Astyanax mexicanus undergo this evolutionarily conserved response to starvation, the starvation-resistant cavefish larvae of the same species do not display an accumulation of lipid droplets upon starvation. Moreover, cavefish are resistant to liver atrophy during starvation, providing a unique system to explore strategies for liver protection. Using comparative transcriptomics between zebrafish, surface fish, and cavefish, we identified the fatty acid transporter slc27a2a/fatp2 to be correlated with the development of fatty liver. Pharmacological inhibition of slc27a2a in zebrafish rescues steatosis and atrophy of the liver upon starvation. Further, down-regulation of FATP2 in drosophila larvae inhibits the development of starvation-induced steatosis, suggesting the evolutionary conserved importance of the gene in regulating fatty liver upon nutrition deprivation. Overall, our study identifies a conserved, druggable target to protect the liver from atrophy during starvation.\nAn internally controlled system to study microtubule network diversification links tubulin evolution to the use of distinct microtubule regulators\nAuthors: Kennard, A. S.; Velle, K. B.; Ranjan, R.; Schulz, D.; Fritz-Laylin, L.\nScore: 3.1, Published: 2024-01-09 DOI: 10.1101/2024.01.08.573270\nDiverse eukaryotic cells assemble microtubule networks that vary in structure and composition. While we understand how cells build microtubule networks with specialized functions, we do not know how microtubule networks diversify across deep evolutionary timescales. This problem has remained unresolved because most organisms use shared pools of tubulins for multiple networks, making it impossible to trace the evolution of any single network. In contrast, the amoeboflagellate Naegleria uses distinct tubulin genes to build distinct microtubule networks: while Naegleria builds flagella from conserved tubulins during differentiation, it uses divergent tubulins to build its mitotic spindle. This genetic separation makes for an internally controlled system to study independent microtubule networks in a single organismal and genomic context. To explore the evolution of these microtubule networks, we identified conserved microtubule binding proteins and used transcriptional profiling of mitosis and differentiation to determine which are upregulated during the assembly of each network. Surprisingly, most microtubule binding proteins are upregulated during only one process, suggesting that Naegleria uses distinct component pools to specialize its microtubule networks. Furthermore, the divergent residues of mitotic tubulins tend to fall within the binding sites of differentiation-specific microtubule regulators, suggesting that interactions between microtubules and their binding proteins constrain tubulin sequence diversification. We therefore propose a model for cytoskeletal evolution in which pools of microtubule network components constrain and guide the diversification of the entire network, so that the evolution of tubulin is inextricably linked to that of its binding partners.\n",
  "wordCount" : "2293",
  "inLanguage": "en",
  "datePublished": "2024-01-14T10:38:40Z",
  "dateModified": "2024-01-14T10:38:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cell-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cell biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 14, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.572781">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.572781" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.572781">
        <p class="paperTitle">Human cytomegalovirus deploys molecular mimicry to recruit VPS4A to sites of virus assembly</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.572781" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.572781" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Butt, B. G.; Fischer, D.; Rep, A. R.; Schauflinger, M.; Read, C.; Bock, T.; Hirner, M.; Graham, S. C.; von Einem, J.</p>
        <p class="info">Score: 17.6, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.572781' target='https://doi.org/10.1101/2024.01.04.572781'> 10.1101/2024.01.04.572781</a></p>
        <p class="abstract">The AAA-type ATPase VPS4 is recruited by proteins of the endosomal sorting complex required for transport III (ESCRT-III) to catalyse membrane constriction and membrane fission. VPS4A accumulates at the cytoplasmic viral assembly complex (cVAC) of cells infected with human cytomegalovirus (HCMV), the site where nascent virus particles obtain their membrane envelope. Here we show that VPS4A is recruited to the cVAC via interaction with pUL71. Sequence analysis, deep-learning structure prediction, molecular dynamics and mutagenic analysis identify a short peptide motif in the C-terminal region of pUL71 that is necessary and sufficient for the interaction with VPS4A. This motif is predicted to bind the same groove of the N-terminal VPS4A Microtubule-Interacting and Trafficking (MIT) domain as the Type 2 MIT-Interacting Motif (MIM2) of cellular ESCRT-III components, and this viral MIM2-like motif (vMIM2) is conserved across {beta}-herpesvirus pUL71 homologues. However, recruitment of VPS4A by pUL71 is dispensable for HCMV morphogenesis or replication and the function of the conserved vMIM2 during infection remains enigmatic. VPS4-recruitment via a vMIM2 represents a previously unknown mechanism of molecular mimicry in viruses, extending previous observations that herpesviruses encode proteins with structural and functional homology to cellular ESCRT-III components.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.07.573240">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.07.573240" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.07.573240">
        <p class="paperTitle">Shaping up genomes: Prokaryotic roots and eukaryotic diversification of SMC complexes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.07.573240" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.07.573240" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: van Hooff, J. J. E.; Raas, M. W. D.; Tromer, E. C.; Eme, L.</p>
        <p class="info">Score: 15.7, Published: 2024-01-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.07.573240' target='https://doi.org/10.1101/2024.01.07.573240'> 10.1101/2024.01.07.573240</a></p>
        <p class="abstract">Across the tree of life, SMC complexes organize, segregate and regulate DNA. In contrast to prokaryotes, which often possess one SMC complex, eukaryotes usually have four such complexes. This expanded set is involved in managing the considerably larger nuclear genomes of eukaryotes, which are distributed across multiple chromosomes. Despite their essential functions, SMC complexes exhibit variations across model eukaryotes, suggesting an even greater variety of these complexes that remains unexplored. Here, we aimed to uncover the diversity and evolution of SMC complexes across eukaryotes, and their deeper, prokaryotic evolutionary origins. For this, we conducted in-depth comparative genomic and phylogenetic analyses of SMC complexes. We show that the last eukaryotic common ancestor (LECA) likely had fully-fledged versions of all four complexes and that the condensin II complex was later lost at least 30 times in eukaryotes. We report evidence that proteins previously designated as functional analogs in various model organisms (e.g., Sororin, Securin, Nse5 and Nse6) are in fact genuine orthologs. Finally, we traced the prokaryotic origins of these complexes and propose that a single SMC complex duplicated in an early archaeon. Altogether, we provide a comprehensive overview of eukaryotic SMC complex diversity and evolution, both addressing and generating questions about their functioning in ancestral and contemporary organisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.575048">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.575048" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.575048">
        <p class="paperTitle">Generation of systemic rabbit chimeras via Induced Pluripotent Stem Cells Reprogrammed with KLF2, ERAS, and PRMT6</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.575048" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.575048" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Perold, F.; Pham, H.-T.; Pijoff, Y.; Doerflinger, N.; Rival-Gervier, S.; Moulin, A.; Jouneau, L.; Pain, B.; Joly, T.; Duranthon, V.; Afanassieff, M.; Savatier, P.; Beaujean, N.</p>
        <p class="info">Score: 8.6, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.575048' target='https://doi.org/10.1101/2024.01.10.575048'> 10.1101/2024.01.10.575048</a></p>
        <p class="abstract">Little is known about the molecular underpinnings of pluripotent stem cells (PSCs) ability to colonize the epiblast of preimplantation embryos and generate chimeras. In our study, using rabbit PSCs as a model system, we conducted unbiased screening of a cDNA library that encodes a panel of 36 pluripotency factors. From this screening, we identified KLF2, ERAS and PRMT6, whose overexpression confers the ability for self-renewal in a KOSR/FGF2-free culture medium supplemented with LIF, activin A, PKC and WNT inhibitors. The reprogrammed cells acquired transcriptomic and epigenetic features of naive pluripotency, including the reactivation of the 2nd X-chromosome. Leveraging these PSC lines, we determined the transcriptomic signature of embryonic colonization-competence, demonstrating transcriptional repression of genes involved in MAPK, WNT, HIPPO, and EPH signaling pathways, alongside the activation of genes involved in amino-acid metabolism, NF-kB signaling, and p53 pathway. Remarkably, a subset of reprogrammed cells, expressing CD75 at a high level, gained the ability to produce chimeric fetuses with a high contribution from PSCs in all lineages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.574988">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.574988" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.574988">
        <p class="paperTitle">Mechanism and regulation of cargo entry into the Commander recycling pathway</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.574988" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.574988" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Butkovic, R.; Walker, A. P.; Healy, M. D.; McNally, K. E.; Liu, M.; Kato, K.; Collins, B. M.; Cullen, P. J.</p>
        <p class="info">Score: 7.4, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.574988' target='https://doi.org/10.1101/2024.01.10.574988'> 10.1101/2024.01.10.574988</a></p>
        <p class="abstract">Commander is a multiprotein complex that orchestrates endosomal recycling of diverse integral cargo proteins and in humans is required for normal skeletal, brain, kidney, and cardiovascular development. While the structure of this complex has recently been described, the central question of how cargo proteins are selected for entry into the Commander recycling pathway remains unclear. Here using recombinant protein reconstitution and in silico predictions we identify the evolutionary conserved mechanism through which the unstructured carboxy-terminal tail of the integral protein adaptor sorting nexin-17 (SNX17) directly binds to the Retriever sub-complex of Commander. SNX17 adopts an autoinhibited conformation where its carboxy-terminal tail occupies the cargo binding groove. Competitive cargo binding overcomes this autoinhibition, promoting SNX17 endosomal residency and the release of the carboxy tail for Retriever association. Using molecular cell biology and high-resolution microscopy, we establish the central importance of SNX17-Retriever association in the handover of integrin and lipoprotein receptor cargoes into pre-existing endosomal retrieval sub-domains for entry into the recycling pathway. In describing the principal mechanism of cargo entry into the Commander recycling pathway we provide key insight into the function and regulation of this evolutionary conserved sorting complex.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.575065">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.575065" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.575065">
        <p class="paperTitle">Detection of Fluorescent Protein Mechanical Switching in Cellulo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.575065" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.575065" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shoyer, T. C.; Collins, K. L.; Ham, T. R.; Blanchard, A. T.; Malavade, J. N.; West, J. L.; Hoffman, B. D.</p>
        <p class="info">Score: 6.4, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.575065' target='https://doi.org/10.1101/2024.01.10.575065'> 10.1101/2024.01.10.575065</a></p>
        <p class="abstract">The ability of cells to sense and respond to mechanical forces is critical in many physiological and pathological processes. However, the mechanisms by which forces affect protein function inside cells remain unclear. Motivated by in vitro demonstrations of fluorescent proteins (FPs) undergoing reversible mechanical switching of fluorescence, we investigated if force-sensitive changes in FP function could be visualized in cells. Guided by a computational model of FP mechanical switching, we develop a formalism for its detection in Foerster resonance energy transfer (FRET)-based biosensors and demonstrate its occurrence in cellulo in a synthetic actin-crosslinker and the mechanical linker protein vinculin. We find that in cellulo mechanical switching is reversible and altered by manipulation of cellular force generation as well as force-sensitive bond dynamics of the biosensor. Together, this work describes a new framework for assessing FP mechanical stability and provides a means of probing force-sensitive protein function inside cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.574971">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.574971" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.574971">
        <p class="paperTitle">Deubiquitinating enzyme mutagenesis screens identify a USP43 driven HIF-1 transcriptional response</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.574971" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.574971" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pauzaite, T.; Wit, N.; Seear, R. V.; Nathan, J. A.</p>
        <p class="info">Score: 4.8, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.574971' target='https://doi.org/10.1101/2024.01.10.574971'> 10.1101/2024.01.10.574971</a></p>
        <p class="abstract">The ubiquitination and proteasome-mediated degradation of Hypoxia Inducible Factors (HIFs) is central to metazoan oxygen-sensing, but the involvement of deubiquitinating enzymes (DUBs) in HIF signalling is less clear. Here, using a bespoke DUBs sgRNA library we conduct CRISPR/Cas9 mutagenesis screens to determine how DUBs are involved in HIF signalling. Alongside defining DUBs involved in HIF activation or suppression, we identify USP43 as a DUB required for efficient activation of a HIF response. USP43 is hypoxia regulated and selectively associates with the HIF-1alpha isoform, and while USP43 does not alter HIF-1alpha stability, it facilitates HIF-1 nuclear accumulation and binding to its target genes. Mechanistically, USP43 associates with 14-3-3 proteins in a hypoxia and phosphorylation dependent manner to increase the nuclear pool of HIF-1. Together, our results unveil the DUB landscape in HIF signalling, and highlight the multifunctionality of DUBs, illustrating that they can provide important signalling functions alongside their catalytic roles.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.07.574541">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.07.574541" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.07.574541">
        <p class="paperTitle">TDP-43 pathology links innate and adaptive immunity in amyotrophic lateral sclerosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.07.574541" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.07.574541" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Evangelista, B. A.; Ragusa, J. V.; Pellegrino, K.; Wu, Y.; Quiroga-Barber, I.; Cahalan, S. R.; Arooji, O. K.; Madren, J. A.; Schroeter, S.; Cozzarin, J.; Xie, L.; Chen, X.; White, K. K.; Ezzell, J. A.; Iannone, M. A.; Cohen, S.; Traub, R. E.; Li, X.; Bedlack, R.; Phanstiel, D. H.; Meeker, R.; Stanley, N.; Cohen, T. J.</p>
        <p class="info">Score: 4.6, Published: 2024-01-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.07.574541' target='https://doi.org/10.1101/2024.01.07.574541'> 10.1101/2024.01.07.574541</a></p>
        <p class="abstract">Amyotrophic lateral sclerosis is the most common fatal motor neuron disease. Approximately 90% of ALS patients exhibit pathology of the master RNA regulator, Transactive Response DNA Binding protein (TDP-43). Despite the prevalence TDP-43 pathology in ALS motor neurons, recent findings suggest immune dysfunction is a determinant of disease progression in patients. Whether TDP-43 pathology elicits disease-modifying immune responses in ALS remains underexplored. In this study, we demonstrate that TDP-43 pathology is internalized by antigen presenting cells, causes vesicle rupture, and leads to innate and adaptive immune cell activation. Using a multiplex imaging platform, we observed interactions between innate and adaptive immune cells near TDP-43 pathological lesions in ALS brain. We used a mass cytometry-based whole-blood stimulation assay to provide evidence that ALS patient peripheral immune cells exhibit responses to TDP-43 aggregates. Taken together, this study provides a novel link between TDP-43 pathology and ALS immune dysfunction, and further highlights the translational and diagnostic implications of monitoring and manipulating the ALS immune response.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563872">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563872" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563872">
        <p class="paperTitle">Pharmacologically inducing regenerative cardiac cells by small molecule drugs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563872" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563872" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhou, W.; He, K.; Wang, C.; Wang, P.; Wang, D.; Wang, B.; Geng, H.; Lian, H.; Ma, T.; Nie, Y.; Ding, S.</p>
        <p class="info">Score: 11.0, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563872' target='https://doi.org/10.1101/2023.10.24.563872'> 10.1101/2023.10.24.563872</a></p>
        <p class="abstract">In contrast to lower organisms, adult mammals cannot regenerate damaged hearts through de-differentiation of cardiomyocytes (CMs) into cells with regenerative capacity. Development of an enabling condition to induce such naturally unavailable cells with potential to proliferate and differentiate into CMs, i.e., regenerative cardiac cells (RCCs), in mammals will provide new insights and tools for heart regeneration. Here, a two-compound combination (2C), CHIR99021 and A-485, was identified to robustly induce RCCs from human embryonic stem cell (hESC) derived TNNT2&#43; CMs in vitro, which was confirmed by lineage tracing experiments. Functional analyses revealed that RCCs expressed a spectrum of genes essential for cardiogenesis and exhibited potential to become functional CMs, endothelial cells (ECs) and smooth muscle cells (SMCs). Consistent with the results in human cells, 2C-induced generation of RCCs were also observed in neonatal rats CMs in vitro and in vivo. Remarkably, administration of 2C can induce RCCs in adult mouse hearts and significantly improve survival and heart function in the mice undergoing myocardial infarction. Mechanistically, CHIR99021 is indispensable for transcriptional and epigenetic activation of genes essential for RCC, whereas A-485 mainly function to epigenetically down-regulate H3K27Ac and particularly H3K9Ac in CMs. Their combination specifically enhances both H3K27Ac and H3K9Ac on RCC genes, facilitating the establishment of RCC state dedifferentiated from CMs. Therefore, our findings demonstrated the feasibility and revealed the mechanisms of pharmacological induction of RCCs from endogenous CMs, which could offer a promising regenerative strategy to repair injured hearts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.574986">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.574986" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.574986">
        <p class="paperTitle">Starvation resistant cavefish reveal conserved mechanisms of starvation-induced hepatic lipotoxicity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.574986" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.574986" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pozo-Morales, M.; Cobham, A. E.; Centola, C.; McKinney, M. C.; Liu, P.; Perazzolo, C.; Lefort, A.; Libert, F.; Bai, H.; Rohner, N.; Singh, S. P.</p>
        <p class="info">Score: 3.2, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.574986' target='https://doi.org/10.1101/2024.01.10.574986'> 10.1101/2024.01.10.574986</a></p>
        <p class="abstract">Starvation causes the accumulation of lipid droplets in the liver, a somewhat counterintuitive phenomenon that is nevertheless conserved from flies to humans. Much like fatty liver resulting from overfeeding, hepatic lipid accumulation (steatosis) during undernourishment can lead to lipotoxicity and atrophy of the liver. Here, we found that while surface populations of Astyanax mexicanus undergo this evolutionarily conserved response to starvation, the starvation-resistant cavefish larvae of the same species do not display an accumulation of lipid droplets upon starvation. Moreover, cavefish are resistant to liver atrophy during starvation, providing a unique system to explore strategies for liver protection. Using comparative transcriptomics between zebrafish, surface fish, and cavefish, we identified the fatty acid transporter slc27a2a/fatp2 to be correlated with the development of fatty liver. Pharmacological inhibition of slc27a2a in zebrafish rescues steatosis and atrophy of the liver upon starvation. Further, down-regulation of FATP2 in drosophila larvae inhibits the development of starvation-induced steatosis, suggesting the evolutionary conserved importance of the gene in regulating fatty liver upon nutrition deprivation. Overall, our study identifies a conserved, druggable target to protect the liver from atrophy during starvation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.573270">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.573270" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.573270">
        <p class="paperTitle">An internally controlled system to study microtubule network diversification links tubulin evolution to the use of distinct microtubule regulators</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.573270" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.573270" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kennard, A. S.; Velle, K. B.; Ranjan, R.; Schulz, D.; Fritz-Laylin, L.</p>
        <p class="info">Score: 3.1, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.573270' target='https://doi.org/10.1101/2024.01.08.573270'> 10.1101/2024.01.08.573270</a></p>
        <p class="abstract">Diverse eukaryotic cells assemble microtubule networks that vary in structure and composition. While we understand how cells build microtubule networks with specialized functions, we do not know how microtubule networks diversify across deep evolutionary timescales. This problem has remained unresolved because most organisms use shared pools of tubulins for multiple networks, making it impossible to trace the evolution of any single network. In contrast, the amoeboflagellate Naegleria uses distinct tubulin genes to build distinct microtubule networks: while Naegleria builds flagella from conserved tubulins during differentiation, it uses divergent tubulins to build its mitotic spindle. This genetic separation makes for an internally controlled system to study independent microtubule networks in a single organismal and genomic context. To explore the evolution of these microtubule networks, we identified conserved microtubule binding proteins and used transcriptional profiling of mitosis and differentiation to determine which are upregulated during the assembly of each network. Surprisingly, most microtubule binding proteins are upregulated during only one process, suggesting that Naegleria uses distinct component pools to specialize its microtubule networks. Furthermore, the divergent residues of mitotic tubulins tend to fall within the binding sites of differentiation-specific microtubule regulators, suggesting that interactions between microtubules and their binding proteins constrain tubulin sequence diversification. We therefore propose a model for cytoskeletal evolution in which pools of microtubule network components constrain and guide the diversification of the entire network, so that the evolution of tubulin is inextricably linked to that of its binding partners.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
