<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cell biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cell biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Universal Cell Embeddings: A Foundation Model for Cell Biology
Authors: Rosen, Y.; Roohani, Y.; Agrawal, A.; Samotorcan, L.; Consortium, T. S.; Quake, S. R.; Leskovec, J.
Score: 83.1, Published: 2023-11-29 DOI: 10.1101/2023.11.28.568918
Developing a universal representation of cells which encompasses the tremendous molecular diversity of cell types within the human body and more generally, across species, would be transformative for cell biology. Recent work using single-cell transcriptomic approaches to create molecular definitions of cell types in the form of cell atlases has provided the necessary data for such an endeavor.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cell-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cell biology" />
<meta property="og:description" content="Universal Cell Embeddings: A Foundation Model for Cell Biology
Authors: Rosen, Y.; Roohani, Y.; Agrawal, A.; Samotorcan, L.; Consortium, T. S.; Quake, S. R.; Leskovec, J.
Score: 83.1, Published: 2023-11-29 DOI: 10.1101/2023.11.28.568918
Developing a universal representation of cells which encompasses the tremendous molecular diversity of cell types within the human body and more generally, across species, would be transformative for cell biology. Recent work using single-cell transcriptomic approaches to create molecular definitions of cell types in the form of cell atlases has provided the necessary data for such an endeavor." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cell-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-10T10:37:48+00:00" />
<meta property="article:modified_time" content="2023-12-10T10:37:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cell biology"/>
<meta name="twitter:description" content="Universal Cell Embeddings: A Foundation Model for Cell Biology
Authors: Rosen, Y.; Roohani, Y.; Agrawal, A.; Samotorcan, L.; Consortium, T. S.; Quake, S. R.; Leskovec, J.
Score: 83.1, Published: 2023-11-29 DOI: 10.1101/2023.11.28.568918
Developing a universal representation of cells which encompasses the tremendous molecular diversity of cell types within the human body and more generally, across species, would be transformative for cell biology. Recent work using single-cell transcriptomic approaches to create molecular definitions of cell types in the form of cell atlases has provided the necessary data for such an endeavor."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cell biology",
      "item": "https://trxiv.yorks0n.com/posts/cell-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cell biology",
  "name": "cell biology",
  "description": "Universal Cell Embeddings: A Foundation Model for Cell Biology\nAuthors: Rosen, Y.; Roohani, Y.; Agrawal, A.; Samotorcan, L.; Consortium, T. S.; Quake, S. R.; Leskovec, J.\nScore: 83.1, Published: 2023-11-29 DOI: 10.1101/2023.11.28.568918\nDeveloping a universal representation of cells which encompasses the tremendous molecular diversity of cell types within the human body and more generally, across species, would be transformative for cell biology. Recent work using single-cell transcriptomic approaches to create molecular definitions of cell types in the form of cell atlases has provided the necessary data for such an endeavor.",
  "keywords": [
    
  ],
  "articleBody": " Universal Cell Embeddings: A Foundation Model for Cell Biology\nAuthors: Rosen, Y.; Roohani, Y.; Agrawal, A.; Samotorcan, L.; Consortium, T. S.; Quake, S. R.; Leskovec, J.\nScore: 83.1, Published: 2023-11-29 DOI: 10.1101/2023.11.28.568918\nDeveloping a universal representation of cells which encompasses the tremendous molecular diversity of cell types within the human body and more generally, across species, would be transformative for cell biology. Recent work using single-cell transcriptomic approaches to create molecular definitions of cell types in the form of cell atlases has provided the necessary data for such an endeavor. Here, we present the Universal Cell Embedding (UCE) foundation model. UCE was trained on a corpus of cell atlas data from human and other species in a completely self-supervised way without any data annotations. UCE offers a unified biological latent space that can represent any cell, regardless of tissue or species. This universal cell embedding captures important biological variation despite the presence of experimental noise across diverse datasets. An important aspect of UCEs universality is that any new cell from any organism can be mapped to this embedding space with no additional data labeling, model training or fine-tuning. We applied UCE to create the Integrated Mega-scale Atlas, embedding 36 million cells, with more than 1,000 uniquely named cell types, from hundreds of experiments, dozens of tissues and eight species. We uncovered new insights about the organization of cell types and tissues within this universal cell embedding space, and leveraged it to infer function of newly discovered cell types. UCEs embedding space exhibits emergent behavior, uncovering new biology that it was never explicitly trained for, such as identifying developmental lineages and embedding data from novel species not included in the training set. Overall, by enabling a universal representation for every cell state and type, UCE provides a valuable tool for analysis, annotation and hypothesis generation as the scale and diversity of single cell datasets continues to grow.\nEndoplasmic reticulum exit sites are segregated for secretion based on cargo size\nAuthors: Saxena, S.; Foresti, O.; Liu, A.; Androulaki, S.; Rodriguez, M. P.; Raote, I.; Cui, B.; Aridor, M.; Malhotra, V.\nScore: 18.5, Published: 2023-12-08 DOI: 10.1101/2023.12.07.570627\nTANGO1-family proteins (TANGO1, TANGO1S and cTAGE5) form stable complexes at the Endoplasmic Reticulum Exit Sites (ERES) and mediate export of bulky cargoes. The C-terminal proline rich domain (PRD) of these proteins binds Sec23A and affects COPII assembly at ERES. These PRD interactions were replaced with light-responsive domains to control the binding betweenTANGO1S-{Delta}PRD and Sec23A. TANGO1S{Delta}PRD was dispersed in the ER membrane but relocated rapidly, yet reversibly, topre-exiting ERES by binding to Sec23A upon light-activation. Prolonged binding of these two proteins concentrated ERES in the juxtanuclear region by a microtubule dependent process, blocked secretory cargo export and relocated ERGIC53 into the ER, but had limited impact on Golgi complex organization. Under these conditions, bulky collagen VII, and endogenous collagen I were collected at less than 47% of the stalled ERES, whereas small cargo molecules were halted uniformly across the ER, indicating that ERES differentially adapt to cargo size. We suggest these differences in cargo-accumulation at ERES permit cells to balance trafficking of cargoes of different sizes and optimize secretion.\nCLUH maintains functional mitochondria and translation in motoneuronal axons and prevents peripheral neuropathy\nAuthors: Zaninello, M.; Schlegel, T.; Nolte, H.; Rehman, P. M. U.; Savino, E.; Barth, E.; Wuestenberg, H.; Uddin, T.; Wolff, L.; Wirth, B.; Lehmann, H.; Cioni, J.-M.; Langer, T.; Rugarli, E.\nScore: 10.9, Published: 2023-12-03 DOI: 10.1101/2023.12.02.569712\nTransport and local translation of mRNAs in distal axonal compartments are essential for neuronal viability. Local synthesis of nuclear-encoded mitochondrial proteins protects mitochondria from damage during their long journey along the axon, however the regulatory factors involved are largely unknown. Here, we show that CLUH, a cytosolic protein that binds mRNAs encoding mitochondrial proteins, is essential for preventing axonal degeneration of spinal motoneurons and maintaining motor behavior in the mouse. We demonstrate that CLUH is enriched in the growth cone of developing spinal motoneurons and is required for their growth. The absence of CLUH affects the abundance of target mRNAs and the corresponding mitochondrial proteins more prominently in axons, leading to ATP deficits specifically in the growth cone. CLUH binds ribosomal subunits, translation initiation and ribosome recycling components, and preserves axonal translation. Overexpression of the ribosome recycling factor ABCE1 rescues the growth cone and translation defects in CLUH-deficient motoneurons. In conclusion, we demonstrate a role for CLUH in mitochondrial quality control and translational regulation in axons, which are essential for their development and long-term integrity and function.\nStructure, interaction, and nervous connectivity of beta cell primary cilia\nAuthors: Müller, A.; Klena, N.; Pang, S.; Garcia, L. E. G.; Sulaymankhil, D.; Topcheva, O.; Seliskar, M.; Mziaut, H.; Schöniger, E.; Friedland, D.; Kipke, N.; Kretschmar, S.; Münster, C.; Weitz, J.; Distler, M.; Kurth, T.; Schmidt, D.; Hess, H. F.; Xu, C. S.; Pigino, G.; Solimena, M.\nScore: 16.1, Published: 2023-12-09 DOI: 10.1101/2023.12.01.568979\nPrimary cilia are sensory organelles present in many cell types. Based on an array of microtubules termed axoneme they form a specialized membrane compartment partaking in various signaling processes. Primary cilia of pancreatic islet beta cells play a role in autocrine and paracrine signaling and are linked to diabetes. Yet, the structural basis for their functions is unclear. We present three-dimensional reconstructions of complete mouse and human beta cell cilia, revealing a disorganized 9+0 axoneme structure. Within the islet, cilia are spatially confined within deep ciliary pockets or squeezed into narrow extracellular spaces between adjacent cells. Beta and alpha cell cilia physically interact with neighboring islet cells pushing and strongly bending their plasma membranes. Furthermore, beta cells can contain multiple cilia that can meet with other islet cell cilia in the extracellular space. Additionally, beta cell cilia establish connections with islet-projecting nerves. These findings highlight the pivotal role of beta cell primary cilia in islet cell connectivity, pointing at their potential functional role in integrating islet intrinsic and extrinsic signals. These novel insights contribute to understanding their significance in health and diabetes.\nβ-H-Spectrin is a key component of an apical-medial hub of proteins during cell wedging in tube morphogenesis\nAuthors: Gillard, G.; Röper, K.\nScore: 8.3, Published: 2023-12-05 DOI: 10.1101/2023.12.05.570290\nCoordinated cell shape changes are a major driver of tissue morphogenesis during development, with apical constriction or wedging of groups of epithelial cells for instance leading to tissue bending in folding or budding processes. During the budding of the tubes of the salivary glands in the Drosophila embryo we previously identified a key interplay between the apical-medial actomyosin that drives apical constriction with the underlying longitudinal microtubule array. At this microtubule-actomyosin interface a hub of proteins accumulates: in addition to the microtubule-actin crosslinker Shot and the minus-end-binder Patronin, we identified two actin-crosslinkers, {beta}-H-Spectrin and Filamin, and the multi-PDZ protein Big bang as components of this apical-medial hub. Tissue-specific degradation of {beta}-H-Spectrin led to reduction of apical-medial Big bang, F-actin, Shot and Patronin and concomittant defects in apical constriction and tube morphogenesis. Residual Patronin still present in the apical-medial position was sufficient to assist microtubule reorganisation into the longitudinal array. In contrast to Patronin and Shot, neither {beta}-H-Spectrin nor Big bang required microtubules for their localisation. {beta}-H-Spectrin instead appeared to be recruited to the apical-medial domain via binding to phosphoinositides that accumulated here. Overexpression of a {beta}-H-Spectrin fragment containing its PH domain displaced endogenous {beta}-H-Spectrin from the apical-medial domain and led to strong morphogenetic defects. The interconnected hub therefore required the synergy of membrane-associated {beta}-H-Spectrin and microtubules and their respective interactors for its assembly and function in sustaining the apical constriction during tube invagination.\nMechanical positive feedback and biochemical negative feedback combine to generate complex contractile oscillations in cytokinesis\nAuthors: Werner, M. E.; Ray, D. D.; Breen, C.; Staddon, M. F.; Jug, F.; Banerjee, S.; Maddox, A. S.\nScore: 8.0, Published: 2023-12-03 DOI: 10.1101/2023.12.01.569672\nContractile force generation by the cortical actomyosin cytoskeleton is essential for a multitude of biological processes. The actomyosin cortex behaves as an active material that drives local and large-scale shape changes via cytoskeletal remodeling in response to biochemical cues and feedback loops. Cytokinesis is the essential cell division event during which a cortical actomyosin ring generates contractile force to change cell shape and separate two daughter cells. Our recent work with active gel theory predicts that actomyosin systems under the control of a biochemical oscillator and experiencing mechanical strain will exhibit complex spatiotemporal behavior, but cytokinetic contractility was thought to be kinetically simple. To test whether active materials in vivo exhibit spatiotemporally complex kinetics, we used 4-dimensional imaging with unprecedented temporal resolution and discovered sections of the cytokinetic cortex undergo periodic phases of acceleration and deceleration. Quantification of ingression speed oscillations revealed wide ranges of oscillation period and amplitude. In the cytokinetic ring, activity of the master regulator RhoA pulsed with a timescale of approximately 20 seconds, shorter than that reported for any other biological context. Contractility oscillated with 20-second periodicity and with much longer periods. A combination of in vivo and in silico approaches to modify mechanical feedback revealed that the period of contractile oscillation is prolonged as a function of the intensity of mechanical feedback. Effective local ring ingression is characterized by slower speed oscillations, likely due to increased local stresses and therefore mechanical feedback. Fast ingression also occurs where material turnover is high, in vivo and in silico. We propose that downstream of initiation by pulsed RhoA activity, mechanical positive feedback, including but not limited to material advection, extends the timescale of contractility beyond that of biochemical input and therefore makes it robust to fluctuations in activation. Circumferential propagation of contractility likely allows sustained contractility despite cytoskeletal remodeling necessary to recover from compaction. Our work demonstrates that while biochemical feedback loops afford systems responsiveness and robustness, mechanical feedback must also be considered to describe and understand the behaviors of active materials in vivo.\nGenomic and phenotypic stability of fusion-driven pediatric Ewing sarcoma cell lines\nAuthors: Kasan, M.; Siebenlist, J.; Sill, M.; Oellinger, R.; de Alava, E.; Surdez, D.; Dirksen, U.; Oehme, I.; Scotlandi, K.; Delattre, O.; Mueller-Nurasyid, M.; Rad, R.; Strauch, K.; Gruenewald, T. G. P.; Cidre-Aranaz, F.\nScore: 18.2, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567802\nHuman cancer cell lines are the mainstay of cancer research. Recent reports showed that highly mutated adult carcinoma cell lines (mainly HeLa and MCF-7) present striking diversity across laboratories and that long-term continuous culturing results in genomic/transcriptomic heterogeneity with strong phenotypical implications. This highlighted how despite human cell line models being powerful tools for cancer research, the findings derived from their use may present limitations in terms of reproducibility. However, to what extent these conclusions can be generalized to the majority of cancer cell lines remained unexplored. Here, we hypothesized that oligomutated pediatric sarcoma cell lines driven by a chimeric oncogenic transcription factor (COTF), such as Ewing sarcoma (EwS), were genetically and phenotypically more stable than the previously investigated (adult) carcinoma cell lines. A comprehensive molecular and phenotypic characterization of multiple EwS cell line strains in direct comparison to the HeLa and MCF-7 cell lines, together with a simultaneous analysis during 12 months of continuous cell culture showed that COTF-driven pediatric sarcoma cell line strains are genomically more stable than adult carcinoma strains, display remarkably stable and homogenous transcriptomes, and exhibit uniform and stable drug response. The analysis of multiple EwS cell lines subjected to long-term continuous culture conditions revealed that variable degrees of genomic/transcriptomic/phenotypic may be observed among COTF-driven cell lines, further exemplifying that the potential for reproducibility of in vitro scientific results may be rather understood as a spectrum, even within the same tumor entity.\nLeptomeningeal Neural Organoid (LMNO) Fusions as Models to Study Meninges-Brain Signaling\nAuthors: Jones, H. E.; Robertson, G. L.; Romero-Morales, A. I.; O'Rourke, R.; Siegenthaler, J. A.; Gama, V.\nScore: 9.0, Published: 2023-12-02 DOI: 10.1101/2023.12.01.569648\nNeural organoids derived from human induced pluripotent stem cells (iPSCs) provide a model to study the earliest stages of human brain development, including neurogenesis, neural differentiation, and synaptogenesis. However, neural organoids lack supportive tissues and some non-neural cell types that are key regulators of brain development. Neural organoids have instead been co-cultured with non-neural structures and cell types to promote their maturation and model interactions with neuronal cells. One structure that does not form de novo with neural organoids is the meninges, a tri-layered structure that surrounds the CNS and secretes key signaling molecules required for mammalian brain development. Most studies of meninges-brain signaling have been performed in mice or using two-dimensional (2D) cultures of human cells, the latter not recapitulating the architecture and cellular diversity of the tissue. To overcome this, we developed a co-culture system of neural organoids generated from human iPSCs fused with fetal leptomeninges from mice with fluorescently labeled meninges (Col1a1-GFP). These proof-of-concept studies test the stability of the different cell types in the leptomeninges (fibroblast and macrophage) and the fused brain organoid (progenitor and neuron), as well as the interface between the organoid and meningeal tissue. We test the longevity of the fusion pieces after 30 days and 60 days in culture, describe best practices for preparing the meninges sample prior to fusion, and examine the feasibility of single or multiple meninges pieces fused to a single organoid. We discuss potential uses of the current version of the LMNO fusion model and opportunities to improve the system.\nHigh resolution long-read telomere sequencing reveals dynamic mechanisms in aging and cancer\nAuthors: Schmidt, T. T.; Tyer, C.; Rughani, P.; Haggblom, C.; Jones, J. R.; Dai, X.; Frazer, K. A.; Gage, F. H.; Juul, S.; Hickey, S.; Karlseder, J.\nScore: 24.8, Published: 2023-11-29 DOI: 10.1101/2023.11.28.569082\nTelomeres are the protective nucleoprotein structures at the end of linear eukaryotic chromosomes. Telomeres repetitive nature and length have traditionally challenged the precise assessment of the composition and length of individual human telomeres. Here, we present Telo-seq to resolve bulk, chromosome arm-specific and allele-specific human telomere lengths using Oxford Nanopore Technologies native long-read sequencing. Telo-seq resolves telomere shortening in five population doubling increments and reveals intrasample, chromosome arm-specific, allele-specific telomere length heterogeneity. Telo-seq can reliably discriminate between telomerase- and ALT-positive cancer cell lines. Thus, Telo-seq is a novel tool to study telomere biology during development, aging, and cancer at unprecedented resolution.\nMitochondrial Apolipoprotein MIC26 is a metabolic rheostat regulating central cellular fuel pathways\nAuthors: Lubeck, M.; Naha, R.; Schaumkessel, Y.; Westhoff, P.; Stefanski, A.; Petzsch, P.; Stuehler, K.; Koehrer, K.; Weber, A. P. M.; Anand, R.; Reichert, A. S.; Kondadi, A. K.\nScore: 7.2, Published: 2023-12-03 DOI: 10.1101/2023.12.01.569567\nMitochondria play central roles in metabolism and metabolic disorders such as type 2 diabetes. MIC26, a MICOS complex subunit, was linked to diabetes and modulation of lipid metabolism. Yet, the functional role of MIC26 in regulating metabolism under hyperglycemia is not understood. We employed a multi-omics approach combined with functional assays using WT and MIC26 KO cells cultured in normoglycemia or hyperglycemia, mimicking altered nutrient availability. We show that MIC26 has an inhibitory role in glycolysis and cholesterol/lipid metabolism under normoglycemic conditions. Under hyperglycemia, this inhibitory role is reversed demonstrating that MIC26 is critical for metabolic adaptations. This is partially mediated by alterations of mitochondrial metabolite transporters. Furthermore, MIC26 deletion led to a major metabolic rewiring of glutamine utilization as well as oxidative phosphorylation. We propose that MIC26 acts as a metabolic rheostat, that modulates mitochondrial metabolite exchange via regulating mitochondrial cristae, allowing cells to cope with nutrient overload.\n",
  "wordCount" : "2547",
  "inLanguage": "en",
  "datePublished": "2023-12-10T10:37:48Z",
  "dateModified": "2023-12-10T10:37:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cell-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cell biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 10, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.568918">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.568918" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.568918">
        <p class="paperTitle">Universal Cell Embeddings: A Foundation Model for Cell Biology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.568918" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.568918" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rosen, Y.; Roohani, Y.; Agrawal, A.; Samotorcan, L.; Consortium, T. S.; Quake, S. R.; Leskovec, J.</p>
        <p class="info">Score: 83.1, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.568918' target='https://doi.org/10.1101/2023.11.28.568918'> 10.1101/2023.11.28.568918</a></p>
        <p class="abstract">Developing a universal representation of cells which encompasses the tremendous molecular diversity of cell types within the human body and more generally, across species, would be transformative for cell biology. Recent work using single-cell transcriptomic approaches to create molecular definitions of cell types in the form of cell atlases has provided the necessary data for such an endeavor. Here, we present the Universal Cell Embedding (UCE) foundation model. UCE was trained on a corpus of cell atlas data from human and other species in a completely self-supervised way without any data annotations. UCE offers a unified biological latent space that can represent any cell, regardless of tissue or species. This universal cell embedding captures important biological variation despite the presence of experimental noise across diverse datasets. An important aspect of UCEs universality is that any new cell from any organism can be mapped to this embedding space with no additional data labeling, model training or fine-tuning. We applied UCE to create the Integrated Mega-scale Atlas, embedding 36 million cells, with more than 1,000 uniquely named cell types, from hundreds of experiments, dozens of tissues and eight species. We uncovered new insights about the organization of cell types and tissues within this universal cell embedding space, and leveraged it to infer function of newly discovered cell types. UCEs embedding space exhibits emergent behavior, uncovering new biology that it was never explicitly trained for, such as identifying developmental lineages and embedding data from novel species not included in the training set. Overall, by enabling a universal representation for every cell state and type, UCE provides a valuable tool for analysis, annotation and hypothesis generation as the scale and diversity of single cell datasets continues to grow.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570627">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570627" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570627">
        <p class="paperTitle">Endoplasmic reticulum exit sites are segregated for secretion based on cargo size</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570627" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570627" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Saxena, S.; Foresti, O.; Liu, A.; Androulaki, S.; Rodriguez, M. P.; Raote, I.; Cui, B.; Aridor, M.; Malhotra, V.</p>
        <p class="info">Score: 18.5, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570627' target='https://doi.org/10.1101/2023.12.07.570627'> 10.1101/2023.12.07.570627</a></p>
        <p class="abstract">TANGO1-family proteins (TANGO1, TANGO1S and cTAGE5) form stable complexes at the Endoplasmic Reticulum Exit Sites (ERES) and mediate export of bulky cargoes. The C-terminal proline rich domain (PRD) of these proteins binds Sec23A and affects COPII assembly at ERES. These PRD interactions were replaced with light-responsive domains to control the binding betweenTANGO1S-{Delta}PRD and Sec23A. TANGO1S{Delta}PRD was dispersed in the ER membrane but relocated rapidly, yet reversibly, topre-exiting ERES by binding to Sec23A upon light-activation. Prolonged binding of these two proteins concentrated ERES in the juxtanuclear region by a microtubule dependent process, blocked secretory cargo export and relocated ERGIC53 into the ER, but had limited impact on Golgi complex organization. Under these conditions, bulky collagen VII, and endogenous collagen I were collected at less than 47% of the stalled ERES, whereas small cargo molecules were halted uniformly across the ER, indicating that ERES differentially adapt to cargo size. We suggest these differences in cargo-accumulation at ERES permit cells to balance trafficking of cargoes of different sizes and optimize secretion.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.02.569712">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.02.569712" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.02.569712">
        <p class="paperTitle">CLUH maintains functional mitochondria and translation in motoneuronal axons and prevents peripheral neuropathy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.02.569712" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.02.569712" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zaninello, M.; Schlegel, T.; Nolte, H.; Rehman, P. M. U.; Savino, E.; Barth, E.; Wuestenberg, H.; Uddin, T.; Wolff, L.; Wirth, B.; Lehmann, H.; Cioni, J.-M.; Langer, T.; Rugarli, E.</p>
        <p class="info">Score: 10.9, Published: 2023-12-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.02.569712' target='https://doi.org/10.1101/2023.12.02.569712'> 10.1101/2023.12.02.569712</a></p>
        <p class="abstract">Transport and local translation of mRNAs in distal axonal compartments are essential for neuronal viability. Local synthesis of nuclear-encoded mitochondrial proteins protects mitochondria from damage during their long journey along the axon, however the regulatory factors involved are largely unknown. Here, we show that CLUH, a cytosolic protein that binds mRNAs encoding mitochondrial proteins, is essential for preventing axonal degeneration of spinal motoneurons and maintaining motor behavior in the mouse. We demonstrate that CLUH is enriched in the growth cone of developing spinal motoneurons and is required for their growth. The absence of CLUH affects the abundance of target mRNAs and the corresponding mitochondrial proteins more prominently in axons, leading to ATP deficits specifically in the growth cone. CLUH binds ribosomal subunits, translation initiation and ribosome recycling components, and preserves axonal translation. Overexpression of the ribosome recycling factor ABCE1 rescues the growth cone and translation defects in CLUH-deficient motoneurons. In conclusion, we demonstrate a role for CLUH in mitochondrial quality control and translational regulation in axons, which are essential for their development and long-term integrity and function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.568979">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.568979" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.568979">
        <p class="paperTitle">Structure, interaction, and nervous connectivity of beta cell primary cilia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.568979" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.568979" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Müller, A.; Klena, N.; Pang, S.; Garcia, L. E. G.; Sulaymankhil, D.; Topcheva, O.; Seliskar, M.; Mziaut, H.; Schöniger, E.; Friedland, D.; Kipke, N.; Kretschmar, S.; Münster, C.; Weitz, J.; Distler, M.; Kurth, T.; Schmidt, D.; Hess, H. F.; Xu, C. S.; Pigino, G.; Solimena, M.</p>
        <p class="info">Score: 16.1, Published: 2023-12-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.568979' target='https://doi.org/10.1101/2023.12.01.568979'> 10.1101/2023.12.01.568979</a></p>
        <p class="abstract">Primary cilia are sensory organelles present in many cell types. Based on an array of microtubules termed axoneme they form a specialized membrane compartment partaking in various signaling processes. Primary cilia of pancreatic islet beta cells play a role in autocrine and paracrine signaling and are linked to diabetes. Yet, the structural basis for their functions is unclear. We present three-dimensional reconstructions of complete mouse and human beta cell cilia, revealing a disorganized 9&#43;0 axoneme structure. Within the islet, cilia are spatially confined within deep ciliary pockets or squeezed into narrow extracellular spaces between adjacent cells. Beta and alpha cell cilia physically interact with neighboring islet cells pushing and strongly bending their plasma membranes. Furthermore, beta cells can contain multiple cilia that can meet with other islet cell cilia in the extracellular space. Additionally, beta cell cilia establish connections with islet-projecting nerves. These findings highlight the pivotal role of beta cell primary cilia in islet cell connectivity, pointing at their potential functional role in integrating islet intrinsic and extrinsic signals. These novel insights contribute to understanding their significance in health and diabetes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.570290">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.570290" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.570290">
        <p class="paperTitle">β-H-Spectrin is a key component of an apical-medial hub of proteins during cell wedging in tube morphogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.570290" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.570290" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gillard, G.; Röper, K.</p>
        <p class="info">Score: 8.3, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.570290' target='https://doi.org/10.1101/2023.12.05.570290'> 10.1101/2023.12.05.570290</a></p>
        <p class="abstract">Coordinated cell shape changes are a major driver of tissue morphogenesis during development, with apical constriction or wedging of groups of epithelial cells for instance leading to tissue bending in folding or budding processes. During the budding of the tubes of the salivary glands in the Drosophila embryo we previously identified a key interplay between the apical-medial actomyosin that drives apical constriction with the underlying longitudinal microtubule array. At this microtubule-actomyosin interface a hub of proteins accumulates: in addition to the microtubule-actin crosslinker Shot and the minus-end-binder Patronin, we identified two actin-crosslinkers, {beta}-H-Spectrin and Filamin, and the multi-PDZ protein Big bang as components of this apical-medial hub. Tissue-specific degradation of {beta}-H-Spectrin led to reduction of apical-medial Big bang, F-actin, Shot and Patronin and concomittant defects in apical constriction and tube morphogenesis. Residual Patronin still present in the apical-medial position was sufficient to assist microtubule reorganisation into the longitudinal array. In contrast to Patronin and Shot, neither {beta}-H-Spectrin nor Big bang required microtubules for their localisation. {beta}-H-Spectrin instead appeared to be recruited to the apical-medial domain via binding to phosphoinositides that accumulated here. Overexpression of a {beta}-H-Spectrin fragment containing its PH domain displaced endogenous {beta}-H-Spectrin from the apical-medial domain and led to strong morphogenetic defects. The interconnected hub therefore required the synergy of membrane-associated {beta}-H-Spectrin and microtubules and their respective interactors for its assembly and function in sustaining the apical constriction during tube invagination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569672">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569672" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569672">
        <p class="paperTitle">Mechanical positive feedback and biochemical negative feedback combine to generate complex contractile oscillations in cytokinesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569672" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569672" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Werner, M. E.; Ray, D. D.; Breen, C.; Staddon, M. F.; Jug, F.; Banerjee, S.; Maddox, A. S.</p>
        <p class="info">Score: 8.0, Published: 2023-12-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569672' target='https://doi.org/10.1101/2023.12.01.569672'> 10.1101/2023.12.01.569672</a></p>
        <p class="abstract">Contractile force generation by the cortical actomyosin cytoskeleton is essential for a multitude of biological processes. The actomyosin cortex behaves as an active material that drives local and large-scale shape changes via cytoskeletal remodeling in response to biochemical cues and feedback loops. Cytokinesis is the essential cell division event during which a cortical actomyosin ring generates contractile force to change cell shape and separate two daughter cells. Our recent work with active gel theory predicts that actomyosin systems under the control of a biochemical oscillator and experiencing mechanical strain will exhibit complex spatiotemporal behavior, but cytokinetic contractility was thought to be kinetically simple. To test whether active materials in vivo exhibit spatiotemporally complex kinetics, we used 4-dimensional imaging with unprecedented temporal resolution and discovered sections of the cytokinetic cortex undergo periodic phases of acceleration and deceleration. Quantification of ingression speed oscillations revealed wide ranges of oscillation period and amplitude. In the cytokinetic ring, activity of the master regulator RhoA pulsed with a timescale of approximately 20 seconds, shorter than that reported for any other biological context. Contractility oscillated with 20-second periodicity and with much longer periods. A combination of in vivo and in silico approaches to modify mechanical feedback revealed that the period of contractile oscillation is prolonged as a function of the intensity of mechanical feedback. Effective local ring ingression is characterized by slower speed oscillations, likely due to increased local stresses and therefore mechanical feedback. Fast ingression also occurs where material turnover is high, in vivo and in silico. We propose that downstream of initiation by pulsed RhoA activity, mechanical positive feedback, including but not limited to material advection, extends the timescale of contractility beyond that of biochemical input and therefore makes it robust to fluctuations in activation. Circumferential propagation of contractility likely allows sustained contractility despite cytoskeletal remodeling necessary to recover from compaction. Our work demonstrates that while biochemical feedback loops afford systems responsiveness and robustness, mechanical feedback must also be considered to describe and understand the behaviors of active materials in vivo.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567802">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567802" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567802">
        <p class="paperTitle">Genomic and phenotypic stability of fusion-driven pediatric Ewing sarcoma cell lines</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567802" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567802" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kasan, M.; Siebenlist, J.; Sill, M.; Oellinger, R.; de Alava, E.; Surdez, D.; Dirksen, U.; Oehme, I.; Scotlandi, K.; Delattre, O.; Mueller-Nurasyid, M.; Rad, R.; Strauch, K.; Gruenewald, T. G. P.; Cidre-Aranaz, F.</p>
        <p class="info">Score: 18.2, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567802' target='https://doi.org/10.1101/2023.11.20.567802'> 10.1101/2023.11.20.567802</a></p>
        <p class="abstract">Human cancer cell lines are the mainstay of cancer research. Recent reports showed that highly mutated adult carcinoma cell lines (mainly HeLa and MCF-7) present striking diversity across laboratories and that long-term continuous culturing results in genomic/transcriptomic heterogeneity with strong phenotypical implications. This highlighted how despite human cell line models being powerful tools for cancer research, the findings derived from their use may present limitations in terms of reproducibility. However, to what extent these conclusions can be generalized to the majority of cancer cell lines remained unexplored. Here, we hypothesized that oligomutated pediatric sarcoma cell lines driven by a chimeric oncogenic transcription factor (COTF), such as Ewing sarcoma (EwS), were genetically and phenotypically more stable than the previously investigated (adult) carcinoma cell lines. A comprehensive molecular and phenotypic characterization of multiple EwS cell line strains in direct comparison to the HeLa and MCF-7 cell lines, together with a simultaneous analysis during 12 months of continuous cell culture showed that COTF-driven pediatric sarcoma cell line strains are genomically more stable than adult carcinoma strains, display remarkably stable and homogenous transcriptomes, and exhibit uniform and stable drug response. The analysis of multiple EwS cell lines subjected to long-term continuous culture conditions revealed that variable degrees of genomic/transcriptomic/phenotypic may be observed among COTF-driven cell lines, further exemplifying that the potential for reproducibility of in vitro scientific results may be rather understood as a spectrum, even within the same tumor entity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569648">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569648" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569648">
        <p class="paperTitle">Leptomeningeal Neural Organoid (LMNO) Fusions as Models to Study Meninges-Brain Signaling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569648" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569648" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jones, H. E.; Robertson, G. L.; Romero-Morales, A. I.; O&#39;Rourke, R.; Siegenthaler, J. A.; Gama, V.</p>
        <p class="info">Score: 9.0, Published: 2023-12-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569648' target='https://doi.org/10.1101/2023.12.01.569648'> 10.1101/2023.12.01.569648</a></p>
        <p class="abstract">Neural organoids derived from human induced pluripotent stem cells (iPSCs) provide a model to study the earliest stages of human brain development, including neurogenesis, neural differentiation, and synaptogenesis. However, neural organoids lack supportive tissues and some non-neural cell types that are key regulators of brain development. Neural organoids have instead been co-cultured with non-neural structures and cell types to promote their maturation and model interactions with neuronal cells. One structure that does not form de novo with neural organoids is the meninges, a tri-layered structure that surrounds the CNS and secretes key signaling molecules required for mammalian brain development. Most studies of meninges-brain signaling have been performed in mice or using two-dimensional (2D) cultures of human cells, the latter not recapitulating the architecture and cellular diversity of the tissue. To overcome this, we developed a co-culture system of neural organoids generated from human iPSCs fused with fetal leptomeninges from mice with fluorescently labeled meninges (Col1a1-GFP). These proof-of-concept studies test the stability of the different cell types in the leptomeninges (fibroblast and macrophage) and the fused brain organoid (progenitor and neuron), as well as the interface between the organoid and meningeal tissue. We test the longevity of the fusion pieces after 30 days and 60 days in culture, describe best practices for preparing the meninges sample prior to fusion, and examine the feasibility of single or multiple meninges pieces fused to a single organoid. We discuss potential uses of the current version of the LMNO fusion model and opportunities to improve the system.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.569082">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.569082" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.569082">
        <p class="paperTitle">High resolution long-read telomere sequencing reveals dynamic mechanisms in aging and cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.569082" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.569082" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schmidt, T. T.; Tyer, C.; Rughani, P.; Haggblom, C.; Jones, J. R.; Dai, X.; Frazer, K. A.; Gage, F. H.; Juul, S.; Hickey, S.; Karlseder, J.</p>
        <p class="info">Score: 24.8, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.569082' target='https://doi.org/10.1101/2023.11.28.569082'> 10.1101/2023.11.28.569082</a></p>
        <p class="abstract">Telomeres are the protective nucleoprotein structures at the end of linear eukaryotic chromosomes. Telomeres repetitive nature and length have traditionally challenged the precise assessment of the composition and length of individual human telomeres. Here, we present Telo-seq to resolve bulk, chromosome arm-specific and allele-specific human telomere lengths using Oxford Nanopore Technologies native long-read sequencing. Telo-seq resolves telomere shortening in five population doubling increments and reveals intrasample, chromosome arm-specific, allele-specific telomere length heterogeneity. Telo-seq can reliably discriminate between telomerase- and ALT-positive cancer cell lines. Thus, Telo-seq is a novel tool to study telomere biology during development, aging, and cancer at unprecedented resolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569567">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569567" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569567">
        <p class="paperTitle">Mitochondrial Apolipoprotein MIC26 is a metabolic rheostat regulating central cellular fuel pathways</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569567" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569567" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lubeck, M.; Naha, R.; Schaumkessel, Y.; Westhoff, P.; Stefanski, A.; Petzsch, P.; Stuehler, K.; Koehrer, K.; Weber, A. P. M.; Anand, R.; Reichert, A. S.; Kondadi, A. K.</p>
        <p class="info">Score: 7.2, Published: 2023-12-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569567' target='https://doi.org/10.1101/2023.12.01.569567'> 10.1101/2023.12.01.569567</a></p>
        <p class="abstract">Mitochondria play central roles in metabolism and metabolic disorders such as type 2 diabetes. MIC26, a MICOS complex subunit, was linked to diabetes and modulation of lipid metabolism. Yet, the functional role of MIC26 in regulating metabolism under hyperglycemia is not understood. We employed a multi-omics approach combined with functional assays using WT and MIC26 KO cells cultured in normoglycemia or hyperglycemia, mimicking altered nutrient availability. We show that MIC26 has an inhibitory role in glycolysis and cholesterol/lipid metabolism under normoglycemic conditions. Under hyperglycemia, this inhibitory role is reversed demonstrating that MIC26 is critical for metabolic adaptations. This is partially mediated by alterations of mitochondrial metabolite transporters. Furthermore, MIC26 deletion led to a major metabolic rewiring of glutamine utilization as well as oxidative phosphorylation. We propose that MIC26 acts as a metabolic  rheostat, that modulates mitochondrial metabolite exchange via regulating mitochondrial cristae, allowing cells to cope with nutrient overload.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
