<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cell biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cell biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="BioE3 enables the identification of bona fide targets of E3 ligases
Authors: Barroso-Gomila, O.; Merino-Cacho, L.; Muratore, V.; Perez, C.; Taibi, V.; Maspero, E.; Azkargorta, M.; Iloro, I.; Trulsson, F.; Vertegaal, A. C. O.; Mayor, U.; Elortza, F.; Polo, S.; Barrio, R.; Sutherland, J. D.
Score: 59.3, Published: 2023-08-19 DOI: 10.1101/2023.08.18.553883
The post-translational modification of proteins by ubiquitination is a highly regulated process that involves a dynamic, three-step enzymatic cascade, where more than 600 E3 ligases play a critical role in recognizing specific substrates for modification.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cell-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cell biology" />
<meta property="og:description" content="BioE3 enables the identification of bona fide targets of E3 ligases
Authors: Barroso-Gomila, O.; Merino-Cacho, L.; Muratore, V.; Perez, C.; Taibi, V.; Maspero, E.; Azkargorta, M.; Iloro, I.; Trulsson, F.; Vertegaal, A. C. O.; Mayor, U.; Elortza, F.; Polo, S.; Barrio, R.; Sutherland, J. D.
Score: 59.3, Published: 2023-08-19 DOI: 10.1101/2023.08.18.553883
The post-translational modification of proteins by ubiquitination is a highly regulated process that involves a dynamic, three-step enzymatic cascade, where more than 600 E3 ligases play a critical role in recognizing specific substrates for modification." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cell-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-27T10:37:02+00:00" />
<meta property="article:modified_time" content="2023-08-27T10:37:02+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cell biology"/>
<meta name="twitter:description" content="BioE3 enables the identification of bona fide targets of E3 ligases
Authors: Barroso-Gomila, O.; Merino-Cacho, L.; Muratore, V.; Perez, C.; Taibi, V.; Maspero, E.; Azkargorta, M.; Iloro, I.; Trulsson, F.; Vertegaal, A. C. O.; Mayor, U.; Elortza, F.; Polo, S.; Barrio, R.; Sutherland, J. D.
Score: 59.3, Published: 2023-08-19 DOI: 10.1101/2023.08.18.553883
The post-translational modification of proteins by ubiquitination is a highly regulated process that involves a dynamic, three-step enzymatic cascade, where more than 600 E3 ligases play a critical role in recognizing specific substrates for modification."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cell biology",
      "item": "https://trxiv.yorks0n.com/posts/cell-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cell biology",
  "name": "cell biology",
  "description": "BioE3 enables the identification of bona fide targets of E3 ligases\nAuthors: Barroso-Gomila, O.; Merino-Cacho, L.; Muratore, V.; Perez, C.; Taibi, V.; Maspero, E.; Azkargorta, M.; Iloro, I.; Trulsson, F.; Vertegaal, A. C. O.; Mayor, U.; Elortza, F.; Polo, S.; Barrio, R.; Sutherland, J. D.\nScore: 59.3, Published: 2023-08-19 DOI: 10.1101/2023.08.18.553883\nThe post-translational modification of proteins by ubiquitination is a highly regulated process that involves a dynamic, three-step enzymatic cascade, where more than 600 E3 ligases play a critical role in recognizing specific substrates for modification.",
  "keywords": [
    
  ],
  "articleBody": " BioE3 enables the identification of bona fide targets of E3 ligases\nAuthors: Barroso-Gomila, O.; Merino-Cacho, L.; Muratore, V.; Perez, C.; Taibi, V.; Maspero, E.; Azkargorta, M.; Iloro, I.; Trulsson, F.; Vertegaal, A. C. O.; Mayor, U.; Elortza, F.; Polo, S.; Barrio, R.; Sutherland, J. D.\nScore: 59.3, Published: 2023-08-19 DOI: 10.1101/2023.08.18.553883\nThe post-translational modification of proteins by ubiquitination is a highly regulated process that involves a dynamic, three-step enzymatic cascade, where more than 600 E3 ligases play a critical role in recognizing specific substrates for modification. Separating bona fide targets of E3s from E3-interacting proteins remains a major challenge in the field. In this study, we present BioE3, a novel approach for identifying substrates of ubiquitin-like (UbL) E3 ligases of interest. Using BirA-E3 ligase fusion proteins and bioUbLs, the method facilitates site-specific biotinylation of UbL-modified substrates of particular E3s for proteomic identification. We demonstrate that the BioE3 system can identify both known and novel targets of two RING-type ubiquitin E3 ligases: RNF4, known to be involved in DNA damage response and the regulation of PML nuclear bodies, and MIB1, implicated in endocytosis, autophagy, and centrosomal protein homeostasis. We further show the versatility of BioE3 by identifying targets of an organelle-specific E3 (MARCH5) and a relatively uncharacterized E3 (RNF214). Furthermore, we show that BioE3 works with HECT-type E3 ligases and identify novel targets of NEDD4 involved in vesicular trafficking. BioE3 is a powerful tool that enables identification of bona fide substrates of UbL E3 ligases and how they change with chemical perturbations. BioE3 may also be applicable for UbLs beyond Ub and SUMO, as well as other E3 ligase classes. The resulting knowledge can shed light on the regulation of cellular processes by the complex UbL network and provide information useful for strategies such as targeted protein degradation (TPD), advancing our understanding of fundamental biological mechanisms and their applications.\nComprehensive Interrogation of Synthetic Relationships in the Human DNA Damage Response\nAuthors: Fielden, J.; Siegner, S. M.; Gallagher, D. N.; Schroeder, M.; Dello Stritto, M. R.; Kobel, L.; Schlapansky, M.; Cejka, P.; Jost, M.; Corn, J. E.\nScore: 31.7, Published: 2023-08-19 DOI: 10.1101/2023.08.18.553865\nThe DNA damage response (DDR) is a multi-faceted network of pathways that preserves genome stability. Unraveling the complementary interplay between these pathways remains a challenge. Here, we comprehensively mapped genetic interactions for all core DDR genes using combinatorial CRISPRi screening. We discovered myriad new connections, including interactions between cancer genes and small molecule targets. We focused on two of the strongest interactions: FEN1/LIG1:WDR48 and FANCM:SMARCAL1. First, we found that WDR48 works with USP1 to restrain overactive translesion synthesis in FEN1/LIG1-deficient cells, and that a preclinical inhibitor of USP1 specifically kills FEN1-deficient cells. Second, we found that SMARCAL1 and FANCM suppress DNA double-strand break (DSB) formation at TA-rich repeats in late replicating regions that otherwise escape into mitosis and cause nuclear fragmentation. We present fundamental insights into genome maintenance processes and our dataset provides a springboard for mechanistic investigations into connections between DDR factors and suggests multiple interactions that could be exploited in cancer therapy.\nTriaging of -helical proteins to the mitochondrial outer membrane by distinct chaperone machinery based on substrate topology\nAuthors: Muthukumar, G.; Stevens, T. A.; Inglis, A. J.; Esantsi, T. K.; Saunders, R. A.; Schulte, F.; Voorhees, R. M.; Guna, A.; Weissman, J. S.\nScore: 20.4, Published: 2023-08-17 DOI: 10.1101/2023.08.16.553624\nMitochondrial outer membrane -helical proteins play critical roles in mitochondrial-cytoplasmic communication, but the rules governing the targeting and insertion of these biophysically diverse substrates remain unknown. Here, we first defined the complement of required mammalian biogenesis machinery through genome-wide CRISPRi screens using topologically distinct membrane proteins. Systematic analysis of nine identified factors across 21 diverse -helical substrates reveals that these components are organized into distinct targeting pathways which act on substrates based on their topology. NAC is required for efficient targeting of polytopic proteins whereas signal-anchored proteins require TTC1, a novel cytosolic chaperone which physically engages substrates. Biochemical and mutational studies reveal that TTC1 employs a conserved TPR domain and a hydrophobic groove in its C-terminal domain to support substrate solubilization and insertion into mitochondria. Thus, targeting of diverse mitochondrial membrane proteins is achieved through topological triaging in the cytosol using principles with similarities to ER membrane protein biogenesis systems.\nMitochondrial protein FKBP8 captures PDZD8 to form mitochondria-ER contacts\nAuthors: Nakamura, K.; Aoyama-Ishiwatari, S.; Nagao, T.; Paaran, M.; Obara, C. J.; Sakurai-Saito, Y.; Johnston, J.; Du, Y.; Suga, S.; Tsuboi, M.; Nakakido, M.; Tsumoto, K.; Kishi, Y.; Gotoh, Y.; Kwak, C.; Rhee, H.-W.; Seo, J. K.; Kosako, H.; Potter, C.; Carragher, B.; Lippincott-Schwartz, J.; Polleux, F.; Hirabayashi, Y.\nScore: 14.5, Published: 2023-08-22 DOI: 10.1101/2023.08.22.554218\nMitochondria-ER membrane contact sites (MERCS) represent a fundamental ultrastructural feature underlying unique biochemistry and physiology in all cells. The ER protein PDZD8 is required for the formation of MERCS in many cell types. PDZD8 tethering partner on the outer mitochondrial membrane (OMM) is unknown, limiting our understanding of MERCS formation and function. Here, by combining unbiased proximity proteomics, CRISPR-Cas9 endogenous protein tagging, Cryo-electron microscopy (Cryo-EM) tomography, and correlative light-EM (CLEM), we identified the OMM protein FKBP8 as the tethering partner of PDZD8 at MERCS. Single molecule tracking of PDZD8 revealed highly dynamic diffusion properties along the ER contrasting with capture at contacts between ER and mitochondria. Overexpression of FKBP8 was sufficient to recruit PDZD8 along the OMM and narrow the ER-OMM distance, whereas independent versus combined deletions of these two proteins demonstrated their interdependence for MERCS formation. Our results identify a novel molecular complex tethering ER- mitochondria membranes in mammalian cells.\nEndogenous BAX and BAK form mosaic rings of variable size and composition on apoptotic mitochondria\nAuthors: Schweighofer, S. V.; Jans, D. C.; Keller-Findeisen, J.; Folmeg, A.; Bates, M.; Jakobs, S.\nScore: 14.0, Published: 2023-08-19 DOI: 10.1101/2023.08.18.553869\nOne hallmark of apoptosis is the oligomerization of BAX and BAK to form a pore in the mitochondrial outer membrane, which mediates the release of pro-apoptotic intermembrane space proteins into the cytosol. Cells overexpressing BAX or BAK fusion proteins are a powerful model system to study the dynamics and localization of these proteins in cells. However, it is unclear whether overexpressed BAX and BAK form the same ultrastructural assemblies following the same temporal hierarchy as endogenously expressed proteins. Combining live- and fixed-cell STED super-resolution microscopy, we show that overexpression of BAK results in novel BAK structures, which are virtually absent in non-overexpressing apoptotic cells. We further demonstrate that in wildtype cells, BAK is recruited to apoptotic pores before BAX. Both proteins together form unordered, mosaic rings on apoptotic mitochondria in immortalized cell culture models as well as in human primary cells. In BAX- or BAK-single-knockout cells, the remaining protein is able to form rings independently. The heterogeneous nature of these rings corroborates the toroidal apoptotic pore model.\nTip60-mediated Rheb acetylation links palmitic acid with mTORC1 activation and insulin resistance\nAuthors: Zhao, Z.; Chen, Q.; Xiang, X.; Dai, W.; Fang, W.; Cui, K.; Li, B.; Liu, Q.; Liu, Y.; Shen, Y.; Li, Y.; Xu, W.; Mai, K.; Ai, Q.\nScore: 12.2, Published: 2023-08-19 DOI: 10.1101/2023.08.18.553816\nDifferences in dietary fatty acid saturation impact glucose homeostasis and insulin sensitivity in vertebrates. Excess dietary intake of saturated fatty acids (SFAs) induces glucose intolerance and metabolic disorders. In contrast, unsaturated fatty acids (UFAs) elicit beneficial effects on insulin sensitivity. However, it remains elusive how SFAs and UFAs signal differentially toward insulin signaling to influence glucose homeostasis. Here, using a croaker model, we report that dietary palmitic acid (PA), but not oleic acid or linoleic acid, leads to dysregulation of mTORC1 signaling which provokes systemic insulin resistance and glucose intolerance. Mechanistically, using croaker primary myocytes, mouse C2C12 myotubes and HEK293T cells, we show that PA-induced mTORC1 activation is dependent on mitochondrial fatty acid {beta} oxidation. Notably, PA profoundly elevates acetyl-CoA derived from mitochondrial fatty acid {beta} oxidation which intensifies Tip60-mediated Rheb acetylation. Subsequently, the induction of Rheb acetylation facilitates hyperactivation of mTORC1 which enhances serine phosphorylation of IRS1 and simultaneously inhibits transcription of IRS1 through impeding TFEB nuclear translocation, leading to impairment of insulin signaling. Furthermore, targeted abrogation of acetyl-CoA produced from fatty acid {beta} oxidation or Tip60-mediated Rheb acetylation by pharmacological inhibition and genetic knockdown rescues PA-induced insulin resistance. Collectively, this study reveals a conserved acetylation-dependent mechanistic insight for understanding the link between fatty acids and insulin resistance, which may provide a potential therapeutic avenue to intervene in the development of T2D.\nUbiquitin-driven protein condensation promotes clathrin-mediated endocytosis\nAuthors: Yuan, F.; Gollapudi, S.; Day, K.; Ashby, G.; Sangani, A.; Malady, B.; Wang, L.; Lafer, E. M.; Huibregtse, J.; Stachowiak, J.\nScore: 11.6, Published: 2023-08-22 DOI: 10.1101/2023.08.21.554139\nClathrin-mediated endocytosis is an essential cellular pathway that enables signaling and recycling of transmembrane proteins and lipids. During endocytosis, dozens of cytosolic proteins come together at the plasma membrane, assembling into a highly interconnected network that drives endocytic vesicle biogenesis. Recently, multiple labs have reported that early endocytic proteins form liquid-like condensates, which provide a flexible platform for the efficient assembly of endocytic vesicles. Given the importance of this network in the dynamics of endocytosis, how might cells regulate its stability? Many receptors and endocytic proteins are ubiquitylated, while early endocytic proteins such as Eps15 contain ubiquitin-interacting motifs. Therefore, we examined the influence of ubiquitin on the stability of the early endocytic protein network. In vitro, we found that recruitment of small amounts of polyubiquitin dramatically increased the stability of Eps15 condensates, suggesting that ubiquitylation could nucleate endocytic sites. In live cell imaging experiments, a version of Eps15 that lacked the ubiquitin-interacting motif failed to rescue defects in endocytic initiation created by Eps15 knockout. Furthermore, fusion of Eps15 to a deubiquitinase enzyme destabilized nascent endocytic sites within minutes. These results suggest that ubiquitylation drives assembly of the flexible protein network responsible for catalyzing endocytic events. More broadly, this work illustrates a biophysical mechanism by which ubiquitylated transmembrane proteins at the plasma membrane could regulate the efficiency of endocytic recycling.\nPhosphorylation-controlled cohesion of a nuclear condensate regulates mRNA retention\nAuthors: McIntyre, A. B. R.; Tschan, A.; Meyer, K.; Walser, S.; Rai, A. K.; Fujita, K.; Pelkmans, L.\nScore: 11.5, Published: 2023-08-22 DOI: 10.1101/2023.08.21.554101\nNuclear speckles are membraneless organelles that associate with active transcription sites and participate in post-transcriptional mRNA processing. During the cell cycle, nuclear speckles dissolve following phosphorylation of their protein components. Here, we identify the PP1 family as the phosphatases that counteract kinase-mediated dissolution. PP1 overexpression increases speckle cohesion and leads to retention of polyadenylated RNA within speckles and the nucleus. Using APEX2 proximity labeling combined with RNA-sequencing, we characterized the relationship between the cohesion of nuclear speckles and the recruitment of specific RNAs. We find that many transcripts are preferentially enriched within nuclear speckles compared to the nucleoplasm, particularly chromatin- and nucleus-associated transcripts. While total polyadenylated RNA retention increased with nuclear speckle cohesion, the ratios of most mRNA species to each other were constant, indicating non-selective, or proportional, retention. We then explored whether nuclear speckle cohesion changes in response to environmental perturbations associated with changes in kinase or phosphatase activity. We found that cellular responses to heat shock, oxidative stress, and hypoxia include changes to the cohesion of nuclear speckles and mRNA retention. Our results demonstrate that tuning the material properties of nuclear speckles provides a mechanism for the acute control of mRNA localization.\nRAD52 and ERCC6L/PICH have a compensatory relationship for genome stability in mitosis.\nAuthors: Osia, B. A.; Merkell, A.; Lopezcolorado, F. W.; Ping, X.; STARK, J. M.\nScore: 11.1, Published: 2023-08-23 DOI: 10.1101/2023.08.23.554522\nThe mammalian RAD52 protein is a DNA repair factor that has both strand annealing and recombination mediator activities, yet is dispensable for cell viability. To characterize genetic contexts that reveal dependence on RAD52 to sustain cell viability (i.e., synthetic lethal relationships), we performed genome-wide CRISPR knock-out screens. Subsequent secondary screening found that depletion of ERCC6L in RAD52-deficient cells causes reduced viability and elevated genome instability, measured as accumulation of 53BP1 into nuclear foci. Furthermore, loss of RAD52 causes elevated levels of anaphase ultrafine bridges marked by ERCC6L, and conversely depletion of ERCC6L causes elevated RAD52 foci both in prometaphase and interphase cells. These effects were enhanced with combination treatments using hydroxyurea and the topoisomerase II inhibitor ICRF-193, and the timing of these treatments are consistent with defects in addressing such stress in mitosis. Thus, loss of RAD52 appears to cause an increased reliance on ERCC6L in mitosis, and vice versa. Consistent with this notion, combined depletion of ERCC6L and disrupting G2/M progression via CDK1 inhibition causes a marked loss of viability in RAD52-deficient cells. We suggest that RAD52 and ERCC6L play compensatory roles in protecting genome stability in mitosis.\nCaloric restriction promotes beta cell longevity and delays aging and senescence by enhancing cell identity and homeostasis mechanisms.\nAuthors: Santos, C. d.; Shrestha, S.; Cottam, M. A.; Perkins, G.; Lev-Ram, V.; Roy, B.; Acree, C.; Kim, K.-Y.; Deerinck, T.; Cutler, M.; Dean, D.; Cartailler, J. P.; MacDonald, P. E.; Hetzer, M. W.; Ellisman, M. H.; Arrojo e Drigo, R.\nScore: 10.6, Published: 2023-08-24 DOI: 10.1101/2023.08.23.554369\nCaloric restriction (CR) extends organismal lifespan and health span by improving glucose homeostasis mechanisms. How CR affects organellar structure and function of pancreatic beta cells over the lifetime of the animal remains unknown. Here, we used single nucleus transcriptomics to show that CR increases the expression of genes for beta cell identity, protein processing, and organelle homeostasis. Gene regulatory network analysis link this transcriptional phenotype to transcription factors involved in beta cell identity (Mafa) and homeostasis (Atf6). Imaging metabolomics further demonstrates that CR beta cells are more energetically competent. In fact, high-resolution light and electron microscopy indicates that CR reduces beta cell mitophagy and increases mitochondria mass, increasing mitochondrial ATP generation. Finally, we show that long-term CR delays the onset of beta cell aging and senescence to promote longevity by reducing beta cell turnover. Therefore, CR could be a feasible approach to preserve compromised beta cells during aging and diabetes.\n",
  "wordCount" : "2307",
  "inLanguage": "en",
  "datePublished": "2023-08-27T10:37:02Z",
  "dateModified": "2023-08-27T10:37:02Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cell-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cell biology
    </h1>
    <div class="post-meta"><span>updated on August 27, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553883">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553883" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553883">
        <p class="paperTitle">BioE3 enables the identification of bona fide targets of E3 ligases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553883" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553883" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Barroso-Gomila, O.; Merino-Cacho, L.; Muratore, V.; Perez, C.; Taibi, V.; Maspero, E.; Azkargorta, M.; Iloro, I.; Trulsson, F.; Vertegaal, A. C. O.; Mayor, U.; Elortza, F.; Polo, S.; Barrio, R.; Sutherland, J. D.</p>
        <p class="info">Score: 59.3, Published: 2023-08-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553883' target='https://doi.org/10.1101/2023.08.18.553883'> 10.1101/2023.08.18.553883</a></p>
        <p class="abstract">The post-translational modification of proteins by ubiquitination is a highly regulated process that involves a dynamic, three-step enzymatic cascade, where more than 600 E3 ligases play a critical role in recognizing specific substrates for modification. Separating bona fide targets of E3s from E3-interacting proteins remains a major challenge in the field. In this study, we present BioE3, a novel approach for identifying substrates of ubiquitin-like (UbL) E3 ligases of interest. Using BirA-E3 ligase fusion proteins and bioUbLs, the method facilitates site-specific biotinylation of UbL-modified substrates of particular E3s for proteomic identification. We demonstrate that the BioE3 system can identify both known and novel targets of two RING-type ubiquitin E3 ligases: RNF4, known to be involved in DNA damage response and the regulation of PML nuclear bodies, and MIB1, implicated in endocytosis, autophagy, and centrosomal protein homeostasis. We further show the versatility of BioE3 by identifying targets of an organelle-specific E3 (MARCH5) and a relatively uncharacterized E3 (RNF214). Furthermore, we show that BioE3 works with HECT-type E3 ligases and identify novel targets of NEDD4 involved in vesicular trafficking. BioE3 is a powerful tool that enables identification of bona fide substrates of UbL E3 ligases and how they change with chemical perturbations. BioE3 may also be applicable for UbLs beyond Ub and SUMO, as well as other E3 ligase classes. The resulting knowledge can shed light on the regulation of cellular processes by the complex UbL network and provide information useful for strategies such as targeted protein degradation (TPD), advancing our understanding of fundamental biological mechanisms and their applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553865">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553865" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553865">
        <p class="paperTitle">Comprehensive Interrogation of Synthetic Relationships in the Human DNA Damage Response</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553865" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553865" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fielden, J.; Siegner, S. M.; Gallagher, D. N.; Schroeder, M.; Dello Stritto, M. R.; Kobel, L.; Schlapansky, M.; Cejka, P.; Jost, M.; Corn, J. E.</p>
        <p class="info">Score: 31.7, Published: 2023-08-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553865' target='https://doi.org/10.1101/2023.08.18.553865'> 10.1101/2023.08.18.553865</a></p>
        <p class="abstract">The DNA damage response (DDR) is a multi-faceted network of pathways that preserves genome stability. Unraveling the complementary interplay between these pathways remains a challenge. Here, we comprehensively mapped genetic interactions for all core DDR genes using combinatorial CRISPRi screening. We discovered myriad new connections, including interactions between cancer genes and small molecule targets. We focused on two of the strongest interactions: FEN1/LIG1:WDR48 and FANCM:SMARCAL1. First, we found that WDR48 works with USP1 to restrain overactive translesion synthesis in FEN1/LIG1-deficient cells, and that a preclinical inhibitor of USP1 specifically kills FEN1-deficient cells. Second, we found that SMARCAL1 and FANCM suppress DNA double-strand break (DSB) formation at TA-rich repeats in late replicating regions that otherwise escape into mitosis and cause nuclear fragmentation. We present fundamental insights into genome maintenance processes and our dataset provides a springboard for mechanistic investigations into connections between DDR factors and suggests multiple interactions that could be exploited in cancer therapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.553624">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.553624" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.553624">
        <p class="paperTitle">Triaging of -helical proteins to the mitochondrial outer membrane by distinct chaperone machinery based on substrate topology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.553624" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.553624" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Muthukumar, G.; Stevens, T. A.; Inglis, A. J.; Esantsi, T. K.; Saunders, R. A.; Schulte, F.; Voorhees, R. M.; Guna, A.; Weissman, J. S.</p>
        <p class="info">Score: 20.4, Published: 2023-08-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.553624' target='https://doi.org/10.1101/2023.08.16.553624'> 10.1101/2023.08.16.553624</a></p>
        <p class="abstract">Mitochondrial outer membrane -helical proteins play critical roles in mitochondrial-cytoplasmic communication, but the rules governing the targeting and insertion of these biophysically diverse substrates remain unknown. Here, we first defined the complement of required mammalian biogenesis machinery through genome-wide CRISPRi screens using topologically distinct membrane proteins. Systematic analysis of nine identified factors across 21 diverse -helical substrates reveals that these components are organized into distinct targeting pathways which act on substrates based on their topology. NAC is required for efficient targeting of polytopic proteins whereas signal-anchored proteins require TTC1, a novel cytosolic chaperone which physically engages substrates. Biochemical and mutational studies reveal that TTC1 employs a conserved TPR domain and a hydrophobic groove in its C-terminal domain to support substrate solubilization and insertion into mitochondria. Thus, targeting of diverse mitochondrial membrane proteins is achieved through topological triaging in the cytosol using principles with similarities to ER membrane protein biogenesis systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.554218">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.554218" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.554218">
        <p class="paperTitle">Mitochondrial protein FKBP8 captures PDZD8 to form mitochondria-ER contacts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.554218" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.554218" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nakamura, K.; Aoyama-Ishiwatari, S.; Nagao, T.; Paaran, M.; Obara, C. J.; Sakurai-Saito, Y.; Johnston, J.; Du, Y.; Suga, S.; Tsuboi, M.; Nakakido, M.; Tsumoto, K.; Kishi, Y.; Gotoh, Y.; Kwak, C.; Rhee, H.-W.; Seo, J. K.; Kosako, H.; Potter, C.; Carragher, B.; Lippincott-Schwartz, J.; Polleux, F.; Hirabayashi, Y.</p>
        <p class="info">Score: 14.5, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.554218' target='https://doi.org/10.1101/2023.08.22.554218'> 10.1101/2023.08.22.554218</a></p>
        <p class="abstract">Mitochondria-ER membrane contact sites (MERCS) represent a fundamental ultrastructural feature underlying unique biochemistry and physiology in all cells. The ER protein PDZD8 is required for the formation of MERCS in many cell types. PDZD8 tethering partner on the outer mitochondrial membrane (OMM) is unknown, limiting our understanding of MERCS formation and function. Here, by combining unbiased proximity proteomics, CRISPR-Cas9 endogenous protein tagging, Cryo-electron microscopy (Cryo-EM) tomography, and correlative light-EM (CLEM), we identified the OMM protein FKBP8 as the tethering partner of PDZD8 at MERCS. Single molecule tracking of PDZD8 revealed highly dynamic diffusion properties along the ER contrasting with capture at contacts between ER and mitochondria. Overexpression of FKBP8 was sufficient to recruit PDZD8 along the OMM and narrow the ER-OMM distance, whereas independent versus combined deletions of these two proteins demonstrated their interdependence for MERCS formation. Our results identify a novel molecular complex tethering ER- mitochondria membranes in mammalian cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553869">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553869" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553869">
        <p class="paperTitle">Endogenous BAX and BAK form mosaic rings of variable size and composition on apoptotic mitochondria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553869" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553869" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schweighofer, S. V.; Jans, D. C.; Keller-Findeisen, J.; Folmeg, A.; Bates, M.; Jakobs, S.</p>
        <p class="info">Score: 14.0, Published: 2023-08-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553869' target='https://doi.org/10.1101/2023.08.18.553869'> 10.1101/2023.08.18.553869</a></p>
        <p class="abstract">One hallmark of apoptosis is the oligomerization of BAX and BAK to form a pore in the mitochondrial outer membrane, which mediates the release of pro-apoptotic intermembrane space proteins into the cytosol. Cells overexpressing BAX or BAK fusion proteins are a powerful model system to study the dynamics and localization of these proteins in cells. However, it is unclear whether overexpressed BAX and BAK form the same ultrastructural assemblies following the same temporal hierarchy as endogenously expressed proteins. Combining live- and fixed-cell STED super-resolution microscopy, we show that overexpression of BAK results in novel BAK structures, which are virtually absent in non-overexpressing apoptotic cells. We further demonstrate that in wildtype cells, BAK is recruited to apoptotic pores before BAX. Both proteins together form unordered, mosaic rings on apoptotic mitochondria in immortalized cell culture models as well as in human primary cells. In BAX- or BAK-single-knockout cells, the remaining protein is able to form rings independently. The heterogeneous nature of these rings corroborates the toroidal apoptotic pore model.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553816">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553816" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553816">
        <p class="paperTitle">Tip60-mediated Rheb acetylation links palmitic acid with mTORC1 activation and insulin resistance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553816" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553816" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhao, Z.; Chen, Q.; Xiang, X.; Dai, W.; Fang, W.; Cui, K.; Li, B.; Liu, Q.; Liu, Y.; Shen, Y.; Li, Y.; Xu, W.; Mai, K.; Ai, Q.</p>
        <p class="info">Score: 12.2, Published: 2023-08-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553816' target='https://doi.org/10.1101/2023.08.18.553816'> 10.1101/2023.08.18.553816</a></p>
        <p class="abstract">Differences in dietary fatty acid saturation impact glucose homeostasis and insulin sensitivity in vertebrates. Excess dietary intake of saturated fatty acids (SFAs) induces glucose intolerance and metabolic disorders. In contrast, unsaturated fatty acids (UFAs) elicit beneficial effects on insulin sensitivity. However, it remains elusive how SFAs and UFAs signal differentially toward insulin signaling to influence glucose homeostasis. Here, using a croaker model, we report that dietary palmitic acid (PA), but not oleic acid or linoleic acid, leads to dysregulation of mTORC1 signaling which provokes systemic insulin resistance and glucose intolerance. Mechanistically, using croaker primary myocytes, mouse C2C12 myotubes and HEK293T cells, we show that PA-induced mTORC1 activation is dependent on mitochondrial fatty acid {beta} oxidation. Notably, PA profoundly elevates acetyl-CoA derived from mitochondrial fatty acid {beta} oxidation which intensifies Tip60-mediated Rheb acetylation. Subsequently, the induction of Rheb acetylation facilitates hyperactivation of mTORC1 which enhances serine phosphorylation of IRS1 and simultaneously inhibits transcription of IRS1 through impeding TFEB nuclear translocation, leading to impairment of insulin signaling. Furthermore, targeted abrogation of acetyl-CoA produced from fatty acid {beta} oxidation or Tip60-mediated Rheb acetylation by pharmacological inhibition and genetic knockdown rescues PA-induced insulin resistance. Collectively, this study reveals a conserved acetylation-dependent mechanistic insight for understanding the link between fatty acids and insulin resistance, which may provide a potential therapeutic avenue to intervene in the development of T2D.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554139">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554139" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554139">
        <p class="paperTitle">Ubiquitin-driven protein condensation promotes clathrin-mediated endocytosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554139" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554139" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yuan, F.; Gollapudi, S.; Day, K.; Ashby, G.; Sangani, A.; Malady, B.; Wang, L.; Lafer, E. M.; Huibregtse, J.; Stachowiak, J.</p>
        <p class="info">Score: 11.6, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554139' target='https://doi.org/10.1101/2023.08.21.554139'> 10.1101/2023.08.21.554139</a></p>
        <p class="abstract">Clathrin-mediated endocytosis is an essential cellular pathway that enables signaling and recycling of transmembrane proteins and lipids. During endocytosis, dozens of cytosolic proteins come together at the plasma membrane, assembling into a highly interconnected network that drives endocytic vesicle biogenesis. Recently, multiple labs have reported that early endocytic proteins form liquid-like condensates, which provide a flexible platform for the efficient assembly of endocytic vesicles. Given the importance of this network in the dynamics of endocytosis, how might cells regulate its stability? Many receptors and endocytic proteins are ubiquitylated, while early endocytic proteins such as Eps15 contain ubiquitin-interacting motifs. Therefore, we examined the influence of ubiquitin on the stability of the early endocytic protein network. In vitro, we found that recruitment of small amounts of polyubiquitin dramatically increased the stability of Eps15 condensates, suggesting that ubiquitylation could nucleate endocytic sites. In live cell imaging experiments, a version of Eps15 that lacked the ubiquitin-interacting motif failed to rescue defects in endocytic initiation created by Eps15 knockout. Furthermore, fusion of Eps15 to a deubiquitinase enzyme destabilized nascent endocytic sites within minutes. These results suggest that ubiquitylation drives assembly of the flexible protein network responsible for catalyzing endocytic events. More broadly, this work illustrates a biophysical mechanism by which ubiquitylated transmembrane proteins at the plasma membrane could regulate the efficiency of endocytic recycling.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554101">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554101" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554101">
        <p class="paperTitle">Phosphorylation-controlled cohesion of a nuclear condensate regulates mRNA retention</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554101" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554101" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McIntyre, A. B. R.; Tschan, A.; Meyer, K.; Walser, S.; Rai, A. K.; Fujita, K.; Pelkmans, L.</p>
        <p class="info">Score: 11.5, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554101' target='https://doi.org/10.1101/2023.08.21.554101'> 10.1101/2023.08.21.554101</a></p>
        <p class="abstract">Nuclear speckles are membraneless organelles that associate with active transcription sites and participate in post-transcriptional mRNA processing. During the cell cycle, nuclear speckles dissolve following phosphorylation of their protein components. Here, we identify the PP1 family as the phosphatases that counteract kinase-mediated dissolution. PP1 overexpression increases speckle cohesion and leads to retention of polyadenylated RNA within speckles and the nucleus. Using APEX2 proximity labeling combined with RNA-sequencing, we characterized the relationship between the cohesion of nuclear speckles and the recruitment of specific RNAs. We find that many transcripts are preferentially enriched within nuclear speckles compared to the nucleoplasm, particularly chromatin- and nucleus-associated transcripts. While total polyadenylated RNA retention increased with nuclear speckle cohesion, the ratios of most mRNA species to each other were constant, indicating non-selective, or proportional, retention. We then explored whether nuclear speckle cohesion changes in response to environmental perturbations associated with changes in kinase or phosphatase activity. We found that cellular responses to heat shock, oxidative stress, and hypoxia include changes to the cohesion of nuclear speckles and mRNA retention. Our results demonstrate that tuning the material properties of nuclear speckles provides a mechanism for the acute control of mRNA localization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554522">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554522" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554522">
        <p class="paperTitle">RAD52 and ERCC6L/PICH have a compensatory relationship for genome stability in mitosis.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554522" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554522" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Osia, B. A.; Merkell, A.; Lopezcolorado, F. W.; Ping, X.; STARK, J. M.</p>
        <p class="info">Score: 11.1, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554522' target='https://doi.org/10.1101/2023.08.23.554522'> 10.1101/2023.08.23.554522</a></p>
        <p class="abstract">The mammalian RAD52 protein is a DNA repair factor that has both strand annealing and recombination mediator activities, yet is dispensable for cell viability. To characterize genetic contexts that reveal dependence on RAD52 to sustain cell viability (i.e., synthetic lethal relationships), we performed genome-wide CRISPR knock-out screens. Subsequent secondary screening found that depletion of ERCC6L in RAD52-deficient cells causes reduced viability and elevated genome instability, measured as accumulation of 53BP1 into nuclear foci. Furthermore, loss of RAD52 causes elevated levels of anaphase ultrafine bridges marked by ERCC6L, and conversely depletion of ERCC6L causes elevated RAD52 foci both in prometaphase and interphase cells. These effects were enhanced with combination treatments using hydroxyurea and the topoisomerase II inhibitor ICRF-193, and the timing of these treatments are consistent with defects in addressing such stress in mitosis. Thus, loss of RAD52 appears to cause an increased reliance on ERCC6L in mitosis, and vice versa. Consistent with this notion, combined depletion of ERCC6L and disrupting G2/M progression via CDK1 inhibition causes a marked loss of viability in RAD52-deficient cells. We suggest that RAD52 and ERCC6L play compensatory roles in protecting genome stability in mitosis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554369">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554369" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554369">
        <p class="paperTitle">Caloric restriction promotes beta cell longevity and delays aging and senescence by enhancing cell identity and homeostasis mechanisms.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554369" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554369" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Santos, C. d.; Shrestha, S.; Cottam, M. A.; Perkins, G.; Lev-Ram, V.; Roy, B.; Acree, C.; Kim, K.-Y.; Deerinck, T.; Cutler, M.; Dean, D.; Cartailler, J. P.; MacDonald, P. E.; Hetzer, M. W.; Ellisman, M. H.; Arrojo e Drigo, R.</p>
        <p class="info">Score: 10.6, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554369' target='https://doi.org/10.1101/2023.08.23.554369'> 10.1101/2023.08.23.554369</a></p>
        <p class="abstract">Caloric restriction (CR) extends organismal lifespan and health span by improving glucose homeostasis mechanisms. How CR affects organellar structure and function of pancreatic beta cells over the lifetime of the animal remains unknown. Here, we used single nucleus transcriptomics to show that CR increases the expression of genes for beta cell identity, protein processing, and organelle homeostasis. Gene regulatory network analysis link this transcriptional phenotype to transcription factors involved in beta cell identity (Mafa) and homeostasis (Atf6). Imaging metabolomics further demonstrates that CR beta cells are more energetically competent. In fact, high-resolution light and electron microscopy indicates that CR reduces beta cell mitophagy and increases mitochondria mass, increasing mitochondrial ATP generation. Finally, we show that long-term CR delays the onset of beta cell aging and senescence to promote longevity by reducing beta cell turnover. Therefore, CR could be a feasible approach to preserve compromised beta cells during aging and diabetes.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
