<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>pathology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>pathology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Native ultrastructure of fresh human brain vitrified directly from autopsy revealed by cryo-electron tomography with cryo-plasma focused ion beam milling
Authors: Creekmore, B. C.; Kixmoeller, K.; Black, B. E.; Lee, E. B.; Chang, Y.-W.
Score: 26.6, Published: 2023-09-17 DOI: 10.1101/2023.09.13.557623
Ultrastructure of human brain tissue has traditionally been examined using electron microscopy (EM) following chemical fixation, staining, and mechanical sectioning, which limit attainable resolution and introduce artifacts. Alternatively, cryo-electron tomography (cryo-ET) offers the potential to image unfixed cellular samples at higher resolution while preserving their native structures, but it requires samples to be frozen free from crystalline ice and thin enough to image via transmission EM.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/pathology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="pathology" />
<meta property="og:description" content="Native ultrastructure of fresh human brain vitrified directly from autopsy revealed by cryo-electron tomography with cryo-plasma focused ion beam milling
Authors: Creekmore, B. C.; Kixmoeller, K.; Black, B. E.; Lee, E. B.; Chang, Y.-W.
Score: 26.6, Published: 2023-09-17 DOI: 10.1101/2023.09.13.557623
Ultrastructure of human brain tissue has traditionally been examined using electron microscopy (EM) following chemical fixation, staining, and mechanical sectioning, which limit attainable resolution and introduce artifacts. Alternatively, cryo-electron tomography (cryo-ET) offers the potential to image unfixed cellular samples at higher resolution while preserving their native structures, but it requires samples to be frozen free from crystalline ice and thin enough to image via transmission EM." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/pathology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-24T10:37:06+00:00" />
<meta property="article:modified_time" content="2023-09-24T10:37:06+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="pathology"/>
<meta name="twitter:description" content="Native ultrastructure of fresh human brain vitrified directly from autopsy revealed by cryo-electron tomography with cryo-plasma focused ion beam milling
Authors: Creekmore, B. C.; Kixmoeller, K.; Black, B. E.; Lee, E. B.; Chang, Y.-W.
Score: 26.6, Published: 2023-09-17 DOI: 10.1101/2023.09.13.557623
Ultrastructure of human brain tissue has traditionally been examined using electron microscopy (EM) following chemical fixation, staining, and mechanical sectioning, which limit attainable resolution and introduce artifacts. Alternatively, cryo-electron tomography (cryo-ET) offers the potential to image unfixed cellular samples at higher resolution while preserving their native structures, but it requires samples to be frozen free from crystalline ice and thin enough to image via transmission EM."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "pathology",
      "item": "https://trxiv.yorks0n.com/posts/pathology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "pathology",
  "name": "pathology",
  "description": "Native ultrastructure of fresh human brain vitrified directly from autopsy revealed by cryo-electron tomography with cryo-plasma focused ion beam milling\nAuthors: Creekmore, B. C.; Kixmoeller, K.; Black, B. E.; Lee, E. B.; Chang, Y.-W.\nScore: 26.6, Published: 2023-09-17 DOI: 10.1101/2023.09.13.557623\nUltrastructure of human brain tissue has traditionally been examined using electron microscopy (EM) following chemical fixation, staining, and mechanical sectioning, which limit attainable resolution and introduce artifacts. Alternatively, cryo-electron tomography (cryo-ET) offers the potential to image unfixed cellular samples at higher resolution while preserving their native structures, but it requires samples to be frozen free from crystalline ice and thin enough to image via transmission EM.",
  "keywords": [
    
  ],
  "articleBody": " Native ultrastructure of fresh human brain vitrified directly from autopsy revealed by cryo-electron tomography with cryo-plasma focused ion beam milling\nAuthors: Creekmore, B. C.; Kixmoeller, K.; Black, B. E.; Lee, E. B.; Chang, Y.-W.\nScore: 26.6, Published: 2023-09-17 DOI: 10.1101/2023.09.13.557623\nUltrastructure of human brain tissue has traditionally been examined using electron microscopy (EM) following chemical fixation, staining, and mechanical sectioning, which limit attainable resolution and introduce artifacts. Alternatively, cryo-electron tomography (cryo-ET) offers the potential to image unfixed cellular samples at higher resolution while preserving their native structures, but it requires samples to be frozen free from crystalline ice and thin enough to image via transmission EM. Due to these requirements, cryo-ET has yet to be employed to investigate the native ultrastructure of unfixed, never previously frozen human brain tissue. Here we present a method for generating lamellae in human brain tissue obtained at time of autopsy that can be imaged via cryo-ET. We vitrify the tissue directly on cryo-EM grids via plunge-freezing, as opposed to high pressure freezing which is generally used for thick samples. Following vitrification, we use xenon plasma focused ion beam (FIB) milling to generate lamellae directly on-grid. In comparison to gallium FIB, which is commonly used for biological samples, xenon plasma FIB is powerful enough to efficiently mill large volume samples, such as human brain tissue. Additionally, our approach allows for lamellae to be generated at variable depth inside the tissue as opposed to being limited to starting at the surface of the tissue. Lamellae generated in Alzheimers disease brain tissue and imaged by cryo-ET reveal intact subcellular structures including components of autophagy and potential tau fibrils. Furthermore, we visualize myelin revealing intact compact myelin and functional cytoplasmic expansions such as cytoplasmic channels and the inner tongue. From these images we also measure the dimensions of myelin membranes, providing insight into how myelin basic protein forces out oligodendrocyte cytoplasm to form compact myelin and tightly links intracellular polar head groups of the oligodendrocyte plasma membrane. This approach provides a first view of unfixed, never previously frozen human brain tissue prepared by cryo-plasma FIB milling and imaged at high resolution by cryo-ET.\nDigital holo-tomographic 3D maps of COVID-19 microclots in blood to assess disease severity\nAuthors: Bergaglio, T.; Synhaivska, O.; Nirmalraj, P.\nScore: 9.8, Published: 2023-09-12 DOI: 10.1101/2023.09.12.557318\nThe coronavirus disease 2019 (COVID-19) has impacted health globally. Cumulative evidence points to long-term effects of COVID-19 such as cardiovascular and cognitive disorders diagnosed in patients even after the recovery period. In particular, micrometer-sized blood clots and hyperactivated platelets have been identified as potential indicators of long COVID. Here we resolve individual microclot structures in platelet-rich plasma of donors with different subphenotypes of COVID-19 in a label-free manner, using 3D digital holo-tomographic microscopy (DHTM). Based on 3D refractive index (RI) tomograms, the size, dry mass, and prevalence of microclot composites were quantified and then parametrically differentiated from fibrin-rich microclots and platelet aggregates in the plasma of COVID-19 donors. Importantly, fewer microclots and platelet aggregates were detected in the plasma of healthy controls when compared to COVID-19 donors. Our work highlights the utility of integrating DHTM in clinical settings that may allow the detection of individuals at risk of developing microvascular thrombotic disorders and for monitoring the efficiency of prescribed treatments by screening plasma samples.\nTranscriptional Signatures of Hippocampal Tau Pathology in Primary Age-Related Tauopathy and Alzheimers Disease\nAuthors: Stein-O'Brien, G. L.; Palaganas, R.; Meyer, E. M.; Redding-Ochoa, J.; Pletnikova, O.; Guo, H.; Bell, W. R.; Troncoso, J. C.; Huganir, R. L.; Morris, M.\nScore: 2.6, Published: 2023-09-12 DOI: 10.1101/2023.09.12.23295440\nBackgroundTau pathology is common in age-related neurodegenerative diseases. Tau pathology in primary age-related tauopathy (PART) and in Alzheimers disease (AD) has a similar biochemical structure and anatomic distribution, which is distinct from tau pathology in other diseases. However, the molecular changes associated with intraneuronal tau pathology in PART and AD, and whether these changes are similar in the two diseases, is largely unexplored. MethodsUsing GeoMx spatial transcriptomics, mRNA was quantified in CA1 pyramidal neurons with tau pathology and adjacent neurons without tau pathology in 6 cases of PART and 6 cases of AD, and compared to 4 control cases without pathology. Transcriptional changes were analyzed for differential gene expression and for coordinated patterns of gene expression associated with both disease state and intraneuronal tau pathology. ResultsSynaptic gene changes and two novel gene expression signatures associated with intraneuronal tau were identified in PART and AD. Overall, gene expression changes associated with intraneuronal tau pathology were similar in PART and AD. Synaptic gene expression was decreased overall in neurons in AD and PART compared to control cases. However, this decrease was largely driven by neurons lacking tau pathology. Synaptic gene expression was increased in tau-positive neurons compared to tau-negative neurons in disease. Two novel gene expression signatures associated with intraneuronal tau were identified by examining coordinated patterns of gene expression. Genes in the up-regulated expression pattern were enriched in calcium regulation and synaptic function pathways, specifically in synaptic exocytosis. These synaptic gene changes and intraneuronal tau expression signatures were confirmed in a published transcriptional dataset of cortical neurons with tau pathology in AD. ConclusionsPART and AD show similar transcriptional changes associated with intraneuronal tau pathology in CA1 pyramidal neurons, raising the possibility of a mechanistic relationship between the tau pathology in the two diseases. Intraneuronal tau pathology was also associated with increased expression of genes associated with synaptic function and calcium regulation compared to tau-negative disease neurons. The findings highlight the power of molecular analysis stratified by pathology in neurodegenerative disease and provide novel insight into common molecular pathways associated with intraneuronal tau in PART and AD. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=90 SRC=\"FIGDIR/small/23295440v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (32K): org.highwire.dtl.DTLVardef@780133org.highwire.dtl.DTLVardef@96c1aaorg.highwire.dtl.DTLVardef@1848d89org.highwire.dtl.DTLVardef@1682d89_HPS_FORMAT_FIGEXP M_FIG C_FIG Created with BioRender.com (License GLSO).\niQC: machine-learning-driven prediction of surgical procedure uncovers systematic confounds of cancer whole slide images in specific medical centers\nAuthors: Schaumberg, A. J.; Lewis, M. S.; Nazarian, R.; Wadhwa, A.; Kane, N.; Turner, G.; Karnam, P.; Devineni, P.; Wolfe, N.; Kintner, R.; Rettig, M. B.; Knudsen, B. S.; Garraway, I. P.; Pyarajan, S.\nScore: 1.2, Published: 2023-09-22 DOI: 10.1101/2023.09.19.23295798\nProblem: The past decades have yielded an explosion of research using artificial intelligence for cancer detection and diagnosis in the field of computational pathology. Yet, an often unspoken assumption of this research is that a glass microscopy slide faithfully represents the underlying disease. Here we show systematic failure modes may dominate the slides digitized from a given medical center, such that neither the whole slide images nor the glass slides are suitable for rendering a diagnosis. Methods: We quantitatively define high quality data as a set of whole slide images where the type of surgery the patient received may be accurately predicted by an automated system such as ours, called \"iQC\". We find iQC accurately distinguished biopsies from nonbiopsies, e.g. prostatectomies or transurethral resections (TURPs, a.k.a. prostate chips), only when the data qualitatively appeared to be high quality, e.g. vibrant histopathology stains and minimal artifacts. Crucially, prostate needle biopsies appear as thin strands of tissue, whereas prostatectomies and TURPs appear as larger rectangular blocks of tissue. Therefore, when the data are of high quality, iQC (i) accurately classifies pixels as tissue, (ii) accurately generates statistics that describe the distribution of tissue in a slide, and (iii) accurately predicts surgical procedure. Results: While we do not control any medical center's protocols for making or storing slides, we developed the iQC tool to hold all medical centers and datasets to the same objective standard of quality. We validate this standard across five Veterans Affairs Medical Centers (VAMCs) and the Automated Gleason Grading Challenge (AGGC) 2022 public dataset. For our surgical procedure prediction task, we report an Area Under Receiver Operating Characteristic (AUROC) of 0.9966-1.000 at the VAMCs that consistently produce high quality data and AUROC of 0.9824 for the AGGC dataset. In contrast, we report an AUROC of 0.7115 at the VAMC that consistently produced poor quality data. An attending pathologist determined poor data quality was likely driven by faded histopathology stains and protocol differences among VAMCs. Specifically, this VAMC produced slides entirely by hand, whereas all other VAMCs leveraged automated methods to produce slides. Conclusion: Our surgical procedure prediction AUROC may be a quantitative indicator positively associated with high data quality at a medical center or for a specific dataset. To produce high quality data, we recommend producing slides using robotics or other forms of automation whenever possible. We recommend scanning slides digitally before the glass slide has time to develop signs of age, e.g faded stains and acrylamide bubbles. To our knowledge, iQC is the first automated system in computational pathology that validates data quality against objective evidence, e.g. surgical procedure data available in the EHR or LIMS, which requires zero efforts or annotations from anatomic pathologists.\n",
  "wordCount" : "1461",
  "inLanguage": "en",
  "datePublished": "2023-09-24T10:37:06Z",
  "dateModified": "2023-09-24T10:37:06Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/pathology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      pathology
    </h1>
    <div class="post-meta"><span>updated on September 24, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.13.557623">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.13.557623" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.13.557623">
        <p class="paperTitle">Native ultrastructure of fresh human brain vitrified directly from autopsy revealed by cryo-electron tomography with cryo-plasma focused ion beam milling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.13.557623" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.13.557623" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Creekmore, B. C.; Kixmoeller, K.; Black, B. E.; Lee, E. B.; Chang, Y.-W.</p>
        <p class="info">Score: 26.6, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.13.557623' target='https://doi.org/10.1101/2023.09.13.557623'> 10.1101/2023.09.13.557623</a></p>
        <p class="abstract">Ultrastructure of human brain tissue has traditionally been examined using electron microscopy (EM) following chemical fixation, staining, and mechanical sectioning, which limit attainable resolution and introduce artifacts. Alternatively, cryo-electron tomography (cryo-ET) offers the potential to image unfixed cellular samples at higher resolution while preserving their native structures, but it requires samples to be frozen free from crystalline ice and thin enough to image via transmission EM. Due to these requirements, cryo-ET has yet to be employed to investigate the native ultrastructure of unfixed, never previously frozen human brain tissue. Here we present a method for generating lamellae in human brain tissue obtained at time of autopsy that can be imaged via cryo-ET. We vitrify the tissue directly on cryo-EM grids via plunge-freezing, as opposed to high pressure freezing which is generally used for thick samples. Following vitrification, we use xenon plasma focused ion beam (FIB) milling to generate lamellae directly on-grid. In comparison to gallium FIB, which is commonly used for biological samples, xenon plasma FIB is powerful enough to efficiently mill large volume samples, such as human brain tissue. Additionally, our approach allows for lamellae to be generated at variable depth inside the tissue as opposed to being limited to starting at the surface of the tissue. Lamellae generated in Alzheimers disease brain tissue and imaged by cryo-ET reveal intact subcellular structures including components of autophagy and potential tau fibrils. Furthermore, we visualize myelin revealing intact compact myelin and functional cytoplasmic expansions such as cytoplasmic channels and the inner tongue. From these images we also measure the dimensions of myelin membranes, providing insight into how myelin basic protein forces out oligodendrocyte cytoplasm to form compact myelin and tightly links intracellular polar head groups of the oligodendrocyte plasma membrane. This approach provides a first view of unfixed, never previously frozen human brain tissue prepared by cryo-plasma FIB milling and imaged at high resolution by cryo-ET.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557318">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557318" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557318">
        <p class="paperTitle">Digital holo-tomographic 3D maps of COVID-19 microclots in blood to assess disease severity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557318" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557318" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bergaglio, T.; Synhaivska, O.; Nirmalraj, P.</p>
        <p class="info">Score: 9.8, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557318' target='https://doi.org/10.1101/2023.09.12.557318'> 10.1101/2023.09.12.557318</a></p>
        <p class="abstract">The coronavirus disease 2019 (COVID-19) has impacted health globally. Cumulative evidence points to long-term effects of COVID-19 such as cardiovascular and cognitive disorders diagnosed in patients even after the recovery period. In particular, micrometer-sized blood clots and hyperactivated platelets have been identified as potential indicators of long COVID. Here we resolve individual microclot structures in platelet-rich plasma of donors with different subphenotypes of COVID-19 in a label-free manner, using 3D digital holo-tomographic microscopy (DHTM). Based on 3D refractive index (RI) tomograms, the size, dry mass, and prevalence of microclot composites were quantified and then parametrically differentiated from fibrin-rich microclots and platelet aggregates in the plasma of COVID-19 donors. Importantly, fewer microclots and platelet aggregates were detected in the plasma of healthy controls when compared to COVID-19 donors. Our work highlights the utility of integrating DHTM in clinical settings that may allow the detection of individuals at risk of developing microvascular thrombotic disorders and for monitoring the efficiency of prescribed treatments by screening plasma samples.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.23295440">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.23295440" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.23295440">
        <p class="paperTitle">Transcriptional Signatures of Hippocampal Tau Pathology in Primary Age-Related Tauopathy and Alzheimers Disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.23295440" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.23295440" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stein-O&#39;Brien, G. L.; Palaganas, R.; Meyer, E. M.; Redding-Ochoa, J.; Pletnikova, O.; Guo, H.; Bell, W. R.; Troncoso, J. C.; Huganir, R. L.; Morris, M.</p>
        <p class="info">Score: 2.6, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.23295440' target='https://doi.org/10.1101/2023.09.12.23295440'> 10.1101/2023.09.12.23295440</a></p>
        <p class="abstract">BackgroundTau pathology is common in age-related neurodegenerative diseases. Tau pathology in primary age-related tauopathy (PART) and in Alzheimers disease (AD) has a similar biochemical structure and anatomic distribution, which is distinct from tau pathology in other diseases. However, the molecular changes associated with intraneuronal tau pathology in PART and AD, and whether these changes are similar in the two diseases, is largely unexplored.

MethodsUsing GeoMx spatial transcriptomics, mRNA was quantified in CA1 pyramidal neurons with tau pathology and adjacent neurons without tau pathology in 6 cases of PART and 6 cases of AD, and compared to 4 control cases without pathology. Transcriptional changes were analyzed for differential gene expression and for coordinated patterns of gene expression associated with both disease state and intraneuronal tau pathology.

ResultsSynaptic gene changes and two novel gene expression signatures associated with intraneuronal tau were identified in PART and AD. Overall, gene expression changes associated with intraneuronal tau pathology were similar in PART and AD. Synaptic gene expression was decreased overall in neurons in AD and PART compared to control cases. However, this decrease was largely driven by neurons lacking tau pathology. Synaptic gene expression was increased in tau-positive neurons compared to tau-negative neurons in disease. Two novel gene expression signatures associated with intraneuronal tau were identified by examining coordinated patterns of gene expression. Genes in the up-regulated expression pattern were enriched in calcium regulation and synaptic function pathways, specifically in synaptic exocytosis. These synaptic gene changes and intraneuronal tau expression signatures were confirmed in a published transcriptional dataset of cortical neurons with tau pathology in AD.

ConclusionsPART and AD show similar transcriptional changes associated with intraneuronal tau pathology in CA1 pyramidal neurons, raising the possibility of a mechanistic relationship between the tau pathology in the two diseases. Intraneuronal tau pathology was also associated with increased expression of genes associated with synaptic function and calcium regulation compared to tau-negative disease neurons. The findings highlight the power of molecular analysis stratified by pathology in neurodegenerative disease and provide novel insight into common molecular pathways associated with intraneuronal tau in PART and AD.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=90 SRC=&#34;FIGDIR/small/23295440v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (32K):
org.highwire.dtl.DTLVardef@780133org.highwire.dtl.DTLVardef@96c1aaorg.highwire.dtl.DTLVardef@1848d89org.highwire.dtl.DTLVardef@1682d89_HPS_FORMAT_FIGEXP  M_FIG C_FIG Created with BioRender.com (License GLSO).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.19.23295798">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.19.23295798" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.19.23295798">
        <p class="paperTitle">iQC: machine-learning-driven prediction of surgical procedure uncovers systematic confounds of cancer whole slide images in specific medical centers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.19.23295798" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.19.23295798" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schaumberg, A. J.; Lewis, M. S.; Nazarian, R.; Wadhwa, A.; Kane, N.; Turner, G.; Karnam, P.; Devineni, P.; Wolfe, N.; Kintner, R.; Rettig, M. B.; Knudsen, B. S.; Garraway, I. P.; Pyarajan, S.</p>
        <p class="info">Score: 1.2, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.19.23295798' target='https://doi.org/10.1101/2023.09.19.23295798'> 10.1101/2023.09.19.23295798</a></p>
        <p class="abstract">Problem: The past decades have yielded an explosion of research using artificial intelligence for cancer detection and diagnosis in the field of computational pathology. Yet, an often unspoken assumption of this research is that a glass microscopy slide faithfully represents the underlying disease. Here we show systematic failure modes may dominate the slides digitized from a given medical center, such that neither the whole slide images nor the glass slides are suitable for rendering a diagnosis. Methods: We quantitatively define high quality data as a set of whole slide images where the type of surgery the patient received may be accurately predicted by an automated system such as ours, called &#34;iQC&#34;. We find iQC accurately distinguished biopsies from nonbiopsies, e.g. prostatectomies or transurethral resections (TURPs, a.k.a. prostate chips), only when the data qualitatively appeared to be high quality, e.g. vibrant histopathology stains and minimal artifacts. Crucially, prostate needle biopsies appear as thin strands of tissue, whereas prostatectomies and TURPs appear as larger rectangular blocks of tissue. Therefore, when the data are of high quality, iQC (i) accurately classifies pixels as tissue, (ii) accurately generates statistics that describe the distribution of tissue in a slide, and (iii) accurately predicts surgical procedure. Results: While we do not control any medical center&#39;s protocols for making or storing slides, we developed the iQC tool to hold all medical centers and datasets to the same objective standard of quality. We validate this standard across five Veterans Affairs Medical Centers (VAMCs) and the Automated Gleason Grading Challenge (AGGC) 2022 public dataset. For our surgical procedure prediction task, we report an Area Under Receiver Operating Characteristic (AUROC) of 0.9966-1.000 at the VAMCs that consistently produce high quality data and AUROC of 0.9824 for the AGGC dataset. In contrast, we report an AUROC of 0.7115 at the VAMC that consistently produced poor quality data. An attending pathologist determined poor data quality was likely driven by faded histopathology stains and protocol differences among VAMCs. Specifically, this VAMC produced slides entirely by hand, whereas all other VAMCs leveraged automated methods to produce slides. Conclusion: Our surgical procedure prediction AUROC may be a quantitative indicator positively associated with high data quality at a medical center or for a specific dataset. To produce high quality data, we recommend producing slides using robotics or other forms of automation whenever possible. We recommend scanning slides digitally before the glass slide has time to develop signs of age, e.g faded stains and acrylamide bubbles. To our knowledge, iQC is the first automated system in computational pathology that validates data quality against objective evidence, e.g. surgical procedure data available in the EHR or LIMS, which requires zero efforts or annotations from anatomic pathologists.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
