<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>pathology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>pathology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="An end-to-end workflow for non-destructive 3D pathology
Authors: Bishop, K. W.; Barner, L. A. E.; Han, Q.; Baraznenok, E.; Lan, L.; Poudel, C.; Gao, G.; Serafin, R. B.; Chow, S. S. L.; Glaser, A. K.; Janowczyk, A.; Brenes, D.; Huang, H.; Miyasato, D.; True, L. D.; Kang, S.; Vaughan, J. C.; Liu, J. T. C.
Score: 119.3, Published: 2023-08-06 DOI: 10.1101/2023.08.03.551845
Recent advances in 3D pathology offer the ability to image orders-of-magnitude more tissue than conventional pathology while providing a volumetric context that is lacking with 2D tissue sections, all without requiring destructive tissue sectioning.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/pathology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="pathology" />
<meta property="og:description" content="An end-to-end workflow for non-destructive 3D pathology
Authors: Bishop, K. W.; Barner, L. A. E.; Han, Q.; Baraznenok, E.; Lan, L.; Poudel, C.; Gao, G.; Serafin, R. B.; Chow, S. S. L.; Glaser, A. K.; Janowczyk, A.; Brenes, D.; Huang, H.; Miyasato, D.; True, L. D.; Kang, S.; Vaughan, J. C.; Liu, J. T. C.
Score: 119.3, Published: 2023-08-06 DOI: 10.1101/2023.08.03.551845
Recent advances in 3D pathology offer the ability to image orders-of-magnitude more tissue than conventional pathology while providing a volumetric context that is lacking with 2D tissue sections, all without requiring destructive tissue sectioning." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/pathology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-09T10:39:33+00:00" />
<meta property="article:modified_time" content="2023-08-09T10:39:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="pathology"/>
<meta name="twitter:description" content="An end-to-end workflow for non-destructive 3D pathology
Authors: Bishop, K. W.; Barner, L. A. E.; Han, Q.; Baraznenok, E.; Lan, L.; Poudel, C.; Gao, G.; Serafin, R. B.; Chow, S. S. L.; Glaser, A. K.; Janowczyk, A.; Brenes, D.; Huang, H.; Miyasato, D.; True, L. D.; Kang, S.; Vaughan, J. C.; Liu, J. T. C.
Score: 119.3, Published: 2023-08-06 DOI: 10.1101/2023.08.03.551845
Recent advances in 3D pathology offer the ability to image orders-of-magnitude more tissue than conventional pathology while providing a volumetric context that is lacking with 2D tissue sections, all without requiring destructive tissue sectioning."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "pathology",
      "item": "https://trxiv.yorks0n.com/posts/pathology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "pathology",
  "name": "pathology",
  "description": "An end-to-end workflow for non-destructive 3D pathology\nAuthors: Bishop, K. W.; Barner, L. A. E.; Han, Q.; Baraznenok, E.; Lan, L.; Poudel, C.; Gao, G.; Serafin, R. B.; Chow, S. S. L.; Glaser, A. K.; Janowczyk, A.; Brenes, D.; Huang, H.; Miyasato, D.; True, L. D.; Kang, S.; Vaughan, J. C.; Liu, J. T. C.\nScore: 119.3, Published: 2023-08-06 DOI: 10.1101/2023.08.03.551845\nRecent advances in 3D pathology offer the ability to image orders-of-magnitude more tissue than conventional pathology while providing a volumetric context that is lacking with 2D tissue sections, all without requiring destructive tissue sectioning.",
  "keywords": [
    
  ],
  "articleBody": " An end-to-end workflow for non-destructive 3D pathology\nAuthors: Bishop, K. W.; Barner, L. A. E.; Han, Q.; Baraznenok, E.; Lan, L.; Poudel, C.; Gao, G.; Serafin, R. B.; Chow, S. S. L.; Glaser, A. K.; Janowczyk, A.; Brenes, D.; Huang, H.; Miyasato, D.; True, L. D.; Kang, S.; Vaughan, J. C.; Liu, J. T. C.\nScore: 119.3, Published: 2023-08-06 DOI: 10.1101/2023.08.03.551845\nRecent advances in 3D pathology offer the ability to image orders-of-magnitude more tissue than conventional pathology while providing a volumetric context that is lacking with 2D tissue sections, all without requiring destructive tissue sectioning. Generating high-quality 3D pathology datasets on a consistent basis is non-trivial, requiring careful attention to many details regarding tissue preparation, imaging, and data/image processing in an iterative process. Here we provide an end-to-end protocol covering all aspects of a 3D pathology workflow (using light-sheet microscopy as an illustrative imaging platform) with sufficient detail to perform well-controlled preclinical and clinical studies. While 3D pathology is compatible with diverse staining protocols and computationally generated color palettes for visual analysis, this protocol will focus on a fluorescent analog of hematoxylin and eosin (H\u0026E), which remains the most common stain for gold-standard diagnostic determinations. We present our guidelines for a broad range of end-users (e.g., biologists, clinical researchers, and engineers) in a simple tutorial format.\nConditional deletion of Ccl2 in smooth muscle cells does not reduce early atherosclerosis in mice\nAuthors: Gunnersen, S.; Shim, J. T.; Liu, F.; Tietge, U. J. F.; Sorensen, C. B.; Bentzon, J. F.\nScore: 1.5, Published: 2023-08-07 DOI: 10.1101/2023.08.06.552177\nBackground: C-C motif chemokine ligand 2 (CCL2) is a pro-inflammatory chemokine important for monocyte recruitment to the arterial wall and atherosclerotic plaques. Global knockout of Ccl2 reduces plaque formation and macrophage content in mice, but the importance of different plaque cell types in mediating this effect has not been resolved. Smooth muscle cells (SMCs) can adopt a potentially pro-inflammatory function with expression of CCL2. The present study aimed to test the hypothesis that SMC-secreted CCL2 is involved in early atherogenesis in mice. Methods: SMC-restricted Cre recombinase was activated at 6 weeks of age in mice with homozygous floxed or wildtype Ccl2 alleles. Separate experiments in mice lacking the Cre recombinase transgene were conducted to control for genetic background effects. Hypercholesterolemia and atherosclerosis were induced by a tail vein injection of recombinant adeno-associated virus (rAAV) encoding proprotein convertase subtilisin /kexin type 9 (PCSK9) and a high-fat diet for 12 weeks. Results: Unexpectedly, mice with SMC-specific Ccl2 deletion developed higher levels of plasma cholesterol and larger atherosclerotic plaques with more macrophages compared with wild-type littermates. When total cholesterol levels were incorporated into the statistical analysis, none of the effects on plaque development between groups remained significant. Importantly, changes in plasma cholesterol and atherosclerosis remained in mice lacking Cre recombinase indicating that they were not caused by SMC-specific CCL2 deletion but by effects of the floxed allele or passenger genes. Conclusions: SMC-specific deficiency of Ccl2 does not significantly affect plaque development in hypercholesterolemic mice.\nFlexible and Highly-Efficient Feature Perception for Molecular Traits Prediction via Self-interactive Deep Learning\nAuthors: Hu, Y.; Sirinukunwattana, K.; Li, B.; Gaitskell, K.; Bonnaffe, W.; Wojciechowska, M.; Wood, R.; Alham, N. K.; Malacrino, S.; Woodcock, D.; Verrill, C.; Ahmed, A.; Rittscher, J.\nScore: 1.0, Published: 2023-08-05 DOI: 10.1101/2023.07.30.23293391\nPredicting disease-related molecular traits from histomorphology brings great opportunities for precision medicine. Despite the rich information present in histopathological images, extracting fine-grained molecular features from standard whole slide images (WSI) is non-trivial. The task is further complicated by the lack of annotations for subtyping and contextual histomorphological features that might span multiple scales. This work proposes a novel multiple-instance learning (MIL) framework capable of WSI-based cancer morpho-molecular subtyping across scales. Our method, debuting as Inter-MIL, follows a weakly-supervised scheme. It enables the training of the patch-level encoder for WSI in a task-aware optimisation procedure, a step normally improbable in most existing MIL-based WSI analysis frameworks. We demonstrate that optimising the patch-level encoder is crucial to achieving high-quality fine-grained and tissue-level subtyping results and offers a significant improvement over task-agnostic encoders. Our approach deploys a pseudo-label propagation strategy to update the patch encoder iteratively, allowing discriminative subtype features to be learned. This mechanism also empowers extracting fine-grained attention within image tiles (the small patches), a task largely ignored in most existing weakly supervised-based frameworks. With Inter-MIL, we carried out four challenging cancer molecular subtyping tasks in the context of ovarian, colorectal, lung, and breast cancer. Extensive evaluation results show that Inter-MIL is a robust framework for cancer morpho-molecular subtyping with superior performance compared to several recently proposed methods, even in data-limited scenarios where the number of available training slides is less than 100. The iterative optimisation mechanism of Inter-MIL significantly improves the quality of the image features learned by the patch embedded and generally directs the attention map to areas that better align with experts interpretation, leading to the identification of more reliable histopathology biomarkers.\n",
  "wordCount" : "817",
  "inLanguage": "en",
  "datePublished": "2023-08-09T10:39:33Z",
  "dateModified": "2023-08-09T10:39:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/pathology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      pathology
    </h1>
    <div class="post-meta"><span>updated on August 9, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.03.551845">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.03.551845" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.03.551845">
        <p class="paperTitle">An end-to-end workflow for non-destructive 3D pathology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.03.551845" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.03.551845" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bishop, K. W.; Barner, L. A. E.; Han, Q.; Baraznenok, E.; Lan, L.; Poudel, C.; Gao, G.; Serafin, R. B.; Chow, S. S. L.; Glaser, A. K.; Janowczyk, A.; Brenes, D.; Huang, H.; Miyasato, D.; True, L. D.; Kang, S.; Vaughan, J. C.; Liu, J. T. C.</p>
        <p class="info">Score: 119.3, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.03.551845' target='https://doi.org/10.1101/2023.08.03.551845'> 10.1101/2023.08.03.551845</a></p>
        <p class="abstract">Recent advances in 3D pathology offer the ability to image orders-of-magnitude more tissue than conventional pathology while providing a volumetric context that is lacking with 2D tissue sections, all without requiring destructive tissue sectioning. Generating high-quality 3D pathology datasets on a consistent basis is non-trivial, requiring careful attention to many details regarding tissue preparation, imaging, and data/image processing in an iterative process. Here we provide an end-to-end protocol covering all aspects of a 3D pathology workflow (using light-sheet microscopy as an illustrative imaging platform) with sufficient detail to perform well-controlled preclinical and clinical studies. While 3D pathology is compatible with diverse staining protocols and computationally generated color palettes for visual analysis, this protocol will focus on a fluorescent analog of hematoxylin and eosin (H&amp;E), which remains the most common stain for gold-standard diagnostic determinations. We present our guidelines for a broad range of end-users (e.g., biologists, clinical researchers, and engineers) in a simple tutorial format.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.06.552177">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.06.552177" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.06.552177">
        <p class="paperTitle">Conditional deletion of Ccl2 in smooth muscle cells does not reduce early atherosclerosis in mice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.06.552177" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.06.552177" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gunnersen, S.; Shim, J. T.; Liu, F.; Tietge, U. J. F.; Sorensen, C. B.; Bentzon, J. F.</p>
        <p class="info">Score: 1.5, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.06.552177' target='https://doi.org/10.1101/2023.08.06.552177'> 10.1101/2023.08.06.552177</a></p>
        <p class="abstract">Background: C-C motif chemokine ligand 2 (CCL2) is a pro-inflammatory chemokine important for monocyte recruitment to the arterial wall and atherosclerotic plaques. Global knockout of Ccl2 reduces plaque formation and macrophage content in mice, but the importance of different plaque cell types in mediating this effect has not been resolved. Smooth muscle cells (SMCs) can adopt a potentially pro-inflammatory function with expression of CCL2. The present study aimed to test the hypothesis that SMC-secreted CCL2 is involved in early atherogenesis in mice. Methods: SMC-restricted Cre recombinase was activated at 6 weeks of age in mice with homozygous floxed or wildtype Ccl2 alleles. Separate experiments in mice lacking the Cre recombinase transgene were conducted to control for genetic background effects. Hypercholesterolemia and atherosclerosis were induced by a tail vein injection of recombinant adeno-associated virus (rAAV) encoding proprotein convertase subtilisin /kexin type 9 (PCSK9) and a high-fat diet for 12 weeks. Results: Unexpectedly, mice with SMC-specific Ccl2 deletion developed higher levels of plasma cholesterol and larger atherosclerotic plaques with more macrophages compared with wild-type littermates. When total cholesterol levels were incorporated into the statistical analysis, none of the effects on plaque development between groups remained significant. Importantly, changes in plasma cholesterol and atherosclerosis remained in mice lacking Cre recombinase indicating that they were not caused by SMC-specific CCL2 deletion but by effects of the floxed allele or passenger genes. Conclusions: SMC-specific deficiency of Ccl2 does not significantly affect plaque development in hypercholesterolemic mice.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.30.23293391">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.30.23293391" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.30.23293391">
        <p class="paperTitle">Flexible and Highly-Efficient Feature Perception for Molecular Traits Prediction via Self-interactive Deep Learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.30.23293391" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.30.23293391" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hu, Y.; Sirinukunwattana, K.; Li, B.; Gaitskell, K.; Bonnaffe, W.; Wojciechowska, M.; Wood, R.; Alham, N. K.; Malacrino, S.; Woodcock, D.; Verrill, C.; Ahmed, A.; Rittscher, J.</p>
        <p class="info">Score: 1.0, Published: 2023-08-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.30.23293391' target='https://doi.org/10.1101/2023.07.30.23293391'> 10.1101/2023.07.30.23293391</a></p>
        <p class="abstract">Predicting disease-related molecular traits from histomorphology brings great opportunities for precision medicine. Despite the rich information present in histopathological images, extracting fine-grained molecular features from standard whole slide images (WSI) is non-trivial. The task is further complicated by the lack of annotations for subtyping and contextual histomorphological features that might span multiple scales. This work proposes a novel multiple-instance learning (MIL) framework capable of WSI-based cancer morpho-molecular subtyping across scales. Our method, debuting as Inter-MIL, follows a weakly-supervised scheme. It enables the training of the patch-level encoder for WSI in a task-aware optimisation procedure, a step normally improbable in most existing MIL-based WSI analysis frameworks. We demonstrate that optimising the patch-level encoder is crucial to achieving high-quality fine-grained and tissue-level subtyping results and offers a significant improvement over task-agnostic encoders. Our approach deploys a pseudo-label propagation strategy to update the patch encoder iteratively, allowing discriminative subtype features to be learned. This mechanism also empowers extracting fine-grained attention within image tiles (the small patches), a task largely ignored in most existing weakly supervised-based frameworks. With Inter-MIL, we carried out four challenging cancer molecular subtyping tasks in the context of ovarian, colorectal, lung, and breast cancer. Extensive evaluation results show that Inter-MIL is a robust framework for cancer morpho-molecular subtyping with superior performance compared to several recently proposed methods, even in data-limited scenarios where the number of available training slides is less than 100. The iterative optimisation mechanism of Inter-MIL significantly improves the quality of the image features learned by the patch embedded and generally directs the attention map to areas that better align with experts interpretation, leading to the identification of more reliable histopathology biomarkers.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
