<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>pathology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>pathology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="PAH DEFICIENT PATHOLOGY IN HUMANIZED c.1066-11G&gt;A PHENYLKETONURIA MICE
Authors: Martinez-Pizarro, A.; Pico, S.; Lopez-Marquez, A.; Rodriguez-Lopez, C.; Montalvo, E.; Alvarez, M.; Castro, M.; Ramon-Maiques, S.; Perez, B.; Lucas, J. J.; Richard, E. M.; Desviat, L. R.
Score: 12.7, Published: 2023-11-05 DOI: 10.1101/2023.11.03.565447
We have generated using CRISPR/Cas9 technology a partially humanized mouse model of the neurometabolic disease phenylketonuria (PKU) carrying the prevalent splice intronic variant PAH c.1066-11G&gt;A. This variant creates a strong alternative splice accpetor site, leading to the inclusion of 9 nucleotides coding for a 3 amino acid insertion between Q355 and Y356 of the phenylalanine hydroxylase (PAH) protein.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/pathology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="pathology" />
<meta property="og:description" content="PAH DEFICIENT PATHOLOGY IN HUMANIZED c.1066-11G&gt;A PHENYLKETONURIA MICE
Authors: Martinez-Pizarro, A.; Pico, S.; Lopez-Marquez, A.; Rodriguez-Lopez, C.; Montalvo, E.; Alvarez, M.; Castro, M.; Ramon-Maiques, S.; Perez, B.; Lucas, J. J.; Richard, E. M.; Desviat, L. R.
Score: 12.7, Published: 2023-11-05 DOI: 10.1101/2023.11.03.565447
We have generated using CRISPR/Cas9 technology a partially humanized mouse model of the neurometabolic disease phenylketonuria (PKU) carrying the prevalent splice intronic variant PAH c.1066-11G&gt;A. This variant creates a strong alternative splice accpetor site, leading to the inclusion of 9 nucleotides coding for a 3 amino acid insertion between Q355 and Y356 of the phenylalanine hydroxylase (PAH) protein." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/pathology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-08T10:39:23+00:00" />
<meta property="article:modified_time" content="2023-11-08T10:39:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="pathology"/>
<meta name="twitter:description" content="PAH DEFICIENT PATHOLOGY IN HUMANIZED c.1066-11G&gt;A PHENYLKETONURIA MICE
Authors: Martinez-Pizarro, A.; Pico, S.; Lopez-Marquez, A.; Rodriguez-Lopez, C.; Montalvo, E.; Alvarez, M.; Castro, M.; Ramon-Maiques, S.; Perez, B.; Lucas, J. J.; Richard, E. M.; Desviat, L. R.
Score: 12.7, Published: 2023-11-05 DOI: 10.1101/2023.11.03.565447
We have generated using CRISPR/Cas9 technology a partially humanized mouse model of the neurometabolic disease phenylketonuria (PKU) carrying the prevalent splice intronic variant PAH c.1066-11G&gt;A. This variant creates a strong alternative splice accpetor site, leading to the inclusion of 9 nucleotides coding for a 3 amino acid insertion between Q355 and Y356 of the phenylalanine hydroxylase (PAH) protein."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "pathology",
      "item": "https://trxiv.yorks0n.com/posts/pathology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "pathology",
  "name": "pathology",
  "description": "PAH DEFICIENT PATHOLOGY IN HUMANIZED c.1066-11G\u0026gt;A PHENYLKETONURIA MICE\nAuthors: Martinez-Pizarro, A.; Pico, S.; Lopez-Marquez, A.; Rodriguez-Lopez, C.; Montalvo, E.; Alvarez, M.; Castro, M.; Ramon-Maiques, S.; Perez, B.; Lucas, J. J.; Richard, E. M.; Desviat, L. R.\nScore: 12.7, Published: 2023-11-05 DOI: 10.1101/2023.11.03.565447\nWe have generated using CRISPR/Cas9 technology a partially humanized mouse model of the neurometabolic disease phenylketonuria (PKU) carrying the prevalent splice intronic variant PAH c.1066-11G\u0026gt;A. This variant creates a strong alternative splice accpetor site, leading to the inclusion of 9 nucleotides coding for a 3 amino acid insertion between Q355 and Y356 of the phenylalanine hydroxylase (PAH) protein.",
  "keywords": [
    
  ],
  "articleBody": " PAH DEFICIENT PATHOLOGY IN HUMANIZED c.1066-11G\u003eA PHENYLKETONURIA MICE\nAuthors: Martinez-Pizarro, A.; Pico, S.; Lopez-Marquez, A.; Rodriguez-Lopez, C.; Montalvo, E.; Alvarez, M.; Castro, M.; Ramon-Maiques, S.; Perez, B.; Lucas, J. J.; Richard, E. M.; Desviat, L. R.\nScore: 12.7, Published: 2023-11-05 DOI: 10.1101/2023.11.03.565447\nWe have generated using CRISPR/Cas9 technology a partially humanized mouse model of the neurometabolic disease phenylketonuria (PKU) carrying the prevalent splice intronic variant PAH c.1066-11G\u003eA. This variant creates a strong alternative splice accpetor site, leading to the inclusion of 9 nucleotides coding for a 3 amino acid insertion between Q355 and Y356 of the phenylalanine hydroxylase (PAH) protein. Homozygous Pah c.1066-11A mice, with a partially humanized intron 10 sequence with the variant, accurately recapitulate the splicing defect and present almost undetectable hepatic PAH protein or activity. They exhibit fur hypopigmentation, lower brain and body weight and reduced survival. Blood and brain phenylalanine (Phe) levels are elevated, along with decreased tyrosine (Tyr), tryptophan (Trp) and monoamine neurotransmitters levels. Pah c.1066-11A mice present behavioral deficits, mainly hypoactivity and diminished social interaction, as well as locomotor deficiencies. Abnormal hind-limb clasping reflex and microophtalmia are observed. Immunohistochemistry analyses of GFAP and Iba1 markers revealed changes in the morphology of glial cells and a significant increase in GFAP and Iba1 staining signals. Western blot analyses in mouse liver lysates and in gene edited HepG2 cells with the variant showed reduced levels of chaperones DNAJC12 and HSC70/HSP70 and an increase in autophagy markers LAMP1 and LC3BII, suggesting that the mutant pQ355_Y356insGLQ PAH protein could aggregate complexed with DNAJC12 and HSP70 proteins, with the aggregates being processed by autophagy. This humanized model corresponding to a severe, classical PKU phenotype represents a useful tool for pathophysiology research in PKU and for novel therapies development.\nNew races with wider virulence indicate local evolution of Puccinia striiformis f. sp. tritici in South America\nAuthors: Riella, V.; Rodriguez-Algaba, J.; Garcia, R.; Pereira, F.; Silva, P.; Hovmoller, M.; German, S.\nScore: 5.0, Published: 2023-10-25 DOI: 10.1101/2023.10.23.563651\nWheat yellow (stripe) rust, caused by Puccinia striiformis f. sp. tritici (Pst), is one of the most devastating diseases of wheat worldwide. Pst populations are composed of multiple genetic groups, each carrying one or more races characterized by different avirulence/virulence combinations. Since the severe epidemics in 2017, yellow rust has become the most economically important wheat foliar disease in Uruguay. Evolution of virulence was investigated based on genotyping and race typing of a representative set of 27 Pst isolates collected from wheat fields in Uruguay between 2017 and 2021. Three genetic groups were identified, i.e., PstS7, PstS10 and PstS13, the latter being the most prevalent. Two races previously reported in Europe, Warrior (PstS7) and Benchmark (PstS10), were detected in four and two isolates, respectively. A third race known as Triticale2015 (PstS13), first detected in Europe in 2015 and in Argentina in 2017, was detected at several locations. Additional virulence to Yr3, Yr17, Yr25, Yr27 or Yr32 was detected in three new race variants within PstS13. The identification of these new races, which have not been reported outside South America, provides strong evidence of the local evolution of virulence in Pst during the recent epidemic years.\nQuality Control for Single Cell Analysis of High-plex Tissue Profiles using CyLinter\nAuthors: Baker, G. J.; Novikov, E.; Zhao, Z.; Vallius, T.; Davis, J. A.; Lin, J.-R.; Muhlich, J. L.; Mittendorf, E. A.; Santagata, S.; Guerriero, J. L.; Sorger, P. K.\nScore: 4.2, Published: 2023-11-01 DOI: 10.1101/2023.11.01.565120\nTumors are complex assemblies of cellular and acellular structures patterned on spatial scales from microns to centimeters. Study of these assemblies has advanced dramatically with the introduction of methods for highly multiplexed tissue imaging methods. These reveal the intensities and spatial distributions of 20-100 proteins in 103-107 cells per specimen in a preserved tissue microenvironment. Despite extensive work on extracting single-cell image data, all tissue images are afflicted by artifacts (e.g., folds, debris, antibody aggregates, optical effects, image processing errors) that arise from imperfections in specimen preparation, data acquisition, image assembly, and feature extraction. We show that artifacts dramatically impact single-cell data analysis, in extreme cases, preventing meaningful biological interpretation. We describe an interactive quality control software tool, CyLinter, that identifies and removes data associated with imaging artifacts. CyLinter greatly improves single-cell analysis, especially for archival specimens sectioned many years prior to data collection, including those from clinical trials.\nRecommendations For a Standardized Approach to Histopathologic Evaluation of Synovial Membrane in Murine Models of Experimental Osteoarthritis\nAuthors: Obeidat, A. M.; Kim, S. Y.; Burt, K. G.; Hu, B.; Li, J.; Ishihara, S.; Xiao, R.; Miller, R. E.; Little, C. B.; Malfait, A.-M.; Scanzello, C.\nScore: 4.8, Published: 2023-10-18 DOI: 10.1101/2023.10.14.562259\nBackgroundSynovial pathology has been linked to osteoarthritis (OA) pain in patients. Microscopic grading systems for synovial changes in human OA have been described, but a standardized approach for murine models of OA is needed. We sought to develop a reproducible approach and set of minimum recommendations for synovial histopathology in mouse models of OA. MethodsCoronal and sagittal sections from male mouse knee joints subjected to destabilization of medial meniscus (DMM) or partial meniscectomy (PMX) were collected as part of other studies. Stains included Hematoxylin and Eosin (H\u0026E), Toluidine Blue (T- Blue) and Safranin O/Fast Green (Saf-O). Four blinded readers graded pathological features (hyperplasia, cellularity, and fibrosis) at specific anatomic locations in the medial and lateral compartments. Inter-reader reliability of each feature was determined. ResultsThere was acceptable to very good agreement between raters. After DMM, increased hyperplasia and cellularity and a trend towards increased fibrosis were observed 6 weeks after DMM in the medial locations, and persisted up to 16 weeks. In the PMX model, cellularity and hyperplasia were evident in both medial and lateral compartments while fibrotic changes were largely seen on the medial side. Synovial changes were consistent from section to section in the mid-joint area mice. H\u0026E, T-blue, and Saf-O stains resulted in comparable reliability. ConclusionsTo allow for a standard evaluation that can be implemented and compared across labs and studies, we recommend using 3 readers to evaluate a minimum set of 3 pathological features at standardized anatomic areas. Pre-defining areas to be scored, and reliability for each pathologic feature should be considered.\nMulti-omics characterization of lymphedema-induced adipose tissue from breast cancer patients\nAuthors: Karaman, S.; Lehti, S.; Zhang, C.; Taskinen, M.-R.; Käkelä, R.; Mardinoglu, A.; Brorson, H.; Alitalo, K. K.; Kivelä, R.\nScore: 1.8, Published: 2023-11-06 DOI: 10.1101/2023.11.03.565593\nObjective: Secondary lymphedema (LE) following breast cancer surgery is a life-long complication, which currently has no cure. LE induces significant regional adipose tissue deposition, requiring liposuction as a treatment. Here, we aimed to elucidate the transcriptional, metabolomic, and lipidomic signature of the adipose tissue developed due to the surgery-induced LE in short- and long-term LE patients, and compared the transcriptomic landscape in LE to the obesity-induced adipose tissue. Methods: Adipose tissue biopsies were obtained from breast cancer-operated females with LE from the affected and non-affected arms (n=20 patients). To decipher molecular properties of the LE adipose tissue, we performed RNA sequencing, metabolomics, and lipidomics combined with bioinformatics analyses. Results: Integrative analysis of functional genomics revealed that inflammatory response, cell chemotaxis and angiogenesis were upregulated biological processes in the LE arm, indicating a sustained inflammation in the edematous adipose tissue, whereas, epidermal differentiation, cell-cell junction organization, water homeostasis and neurogenesis were, in turn, downregulated in the LE arm. Surprisingly, only few genes were found to be the same in the LE-induced and the obesity-induced adipose tissue expansion, indicating a different type of adipose tissue development in these two diseases. In metabolomics analysis, the concentration of a branched-chain amino acid valine was found to be reduced in the edematous arm together with downregulation of mRNA levels of its transporter SLC6A15. Lipidomics analyses did not show any significant differences between the diseased and healthy arm, suggesting that diet affects the lipid composition of the adipose tissue more than the LE. Conclusions: Our results provide a detailed molecular characterization of adipose tissue in secondary LE of breast cancer patients vs individuals with obesity. The results show distinct differences in transcriptomic signatures between LE patients vs individuals with obesity, but only minor differences in metabolome and lipidome between the diseased and the healthy arm.\nIntegrating cellular graph embeddings with tumor morphological features to predict in-silico spatial transcriptomics from H\u0026E images\nAuthors: Liu, K.; Prabhakar, V.\nScore: 1.5, Published: 2023-11-03 DOI: 10.1101/2023.10.31.565020\nSpatial transcriptomics allows precise RNA abundance measurement at high spatial resolution, linking cellular morphology with gene expression. We present a novel deep learning algorithm predicting local gene expression from histopathology images. Our approach employs a graph isomorphism neural network capturing cell-to-cell interactions in the tumor microenvironment and a Vision Transformer (CTransPath) for obtaining the tumor morphological features. Using a dataset of 30,612 spatially resolved gene expression profiles matched with histopathology images from 23 breast cancer patients, we identify 250 genes, including established breast cancer biomarkers, at a 100 {micro}m resolution. Additionally, we co-train our algorithm on spatial spot-level transcriptomics from 10x Visium breast cancer data along with another variant of our algorithm on TCGA-BRCA bulk RNA Seq. data, yielding mutual benefits and enhancing predictive accuracy on both these datasets. This work enables image-based screening for molecular biomarkers with spatial variation, promising breakthroughs in cancer research and diagnostics.\n",
  "wordCount" : "1532",
  "inLanguage": "en",
  "datePublished": "2023-11-08T10:39:23Z",
  "dateModified": "2023-11-08T10:39:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/pathology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      pathology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 8, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.03.565447">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.03.565447" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.03.565447">
        <p class="paperTitle">PAH DEFICIENT PATHOLOGY IN HUMANIZED c.1066-11G&gt;A PHENYLKETONURIA MICE</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.03.565447" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.03.565447" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martinez-Pizarro, A.; Pico, S.; Lopez-Marquez, A.; Rodriguez-Lopez, C.; Montalvo, E.; Alvarez, M.; Castro, M.; Ramon-Maiques, S.; Perez, B.; Lucas, J. J.; Richard, E. M.; Desviat, L. R.</p>
        <p class="info">Score: 12.7, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.03.565447' target='https://doi.org/10.1101/2023.11.03.565447'> 10.1101/2023.11.03.565447</a></p>
        <p class="abstract">We have generated using CRISPR/Cas9 technology a partially humanized mouse model of the neurometabolic disease phenylketonuria (PKU) carrying the prevalent splice intronic variant PAH c.1066-11G&gt;A. This variant creates a strong alternative splice accpetor site, leading to the inclusion of 9 nucleotides coding for a 3 amino acid insertion between Q355 and Y356 of the phenylalanine hydroxylase (PAH) protein. Homozygous Pah c.1066-11A mice, with a partially humanized intron 10 sequence with the variant, accurately recapitulate the splicing defect and present almost undetectable hepatic PAH protein or activity. They exhibit fur hypopigmentation, lower brain and body weight and reduced survival. Blood and brain phenylalanine (Phe) levels are elevated, along with decreased tyrosine (Tyr), tryptophan (Trp) and monoamine neurotransmitters levels. Pah c.1066-11A mice present behavioral deficits, mainly hypoactivity and diminished social interaction, as well as locomotor deficiencies. Abnormal hind-limb clasping reflex and microophtalmia are observed. Immunohistochemistry analyses of GFAP and Iba1 markers revealed changes in the morphology of glial cells and a significant increase in GFAP and Iba1 staining signals. Western blot analyses in mouse liver lysates and in gene edited HepG2 cells with the variant showed reduced levels of chaperones DNAJC12 and HSC70/HSP70 and an increase in autophagy markers LAMP1 and LC3BII, suggesting that the mutant pQ355_Y356insGLQ PAH protein could aggregate complexed with DNAJC12 and HSP70 proteins, with the aggregates being processed by autophagy. This humanized model corresponding to a severe, classical PKU phenotype represents a useful tool for pathophysiology research in PKU and for novel therapies development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563651">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563651" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563651">
        <p class="paperTitle">New races with wider virulence indicate local evolution of Puccinia striiformis f. sp. tritici in South America</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563651" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563651" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Riella, V.; Rodriguez-Algaba, J.; Garcia, R.; Pereira, F.; Silva, P.; Hovmoller, M.; German, S.</p>
        <p class="info">Score: 5.0, Published: 2023-10-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563651' target='https://doi.org/10.1101/2023.10.23.563651'> 10.1101/2023.10.23.563651</a></p>
        <p class="abstract">Wheat yellow (stripe) rust, caused by Puccinia striiformis f. sp. tritici (Pst), is one of the most devastating diseases of wheat worldwide. Pst populations are composed of multiple genetic groups, each carrying one or more races characterized by different avirulence/virulence combinations. Since the severe epidemics in 2017, yellow rust has become the most economically important wheat foliar disease in Uruguay. Evolution of virulence was investigated based on genotyping and race typing of a representative set of 27 Pst isolates collected from wheat fields in Uruguay between 2017 and 2021. Three genetic groups were identified, i.e., PstS7, PstS10 and PstS13, the latter being the most prevalent. Two races previously reported in Europe, Warrior (PstS7) and Benchmark (PstS10), were detected in four and two isolates, respectively. A third race known as Triticale2015 (PstS13), first detected in Europe in 2015 and in Argentina in 2017, was detected at several locations. Additional virulence to Yr3, Yr17, Yr25, Yr27 or Yr32 was detected in three new race variants within PstS13. The identification of these new races, which have not been reported outside South America, provides strong evidence of the local evolution of virulence in Pst during the recent epidemic years.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565120">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565120" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565120">
        <p class="paperTitle">Quality Control for Single Cell Analysis of High-plex Tissue Profiles using CyLinter</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565120" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565120" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Baker, G. J.; Novikov, E.; Zhao, Z.; Vallius, T.; Davis, J. A.; Lin, J.-R.; Muhlich, J. L.; Mittendorf, E. A.; Santagata, S.; Guerriero, J. L.; Sorger, P. K.</p>
        <p class="info">Score: 4.2, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565120' target='https://doi.org/10.1101/2023.11.01.565120'> 10.1101/2023.11.01.565120</a></p>
        <p class="abstract">Tumors are complex assemblies of cellular and acellular structures patterned on spatial scales from microns to centimeters. Study of these assemblies has advanced dramatically with the introduction of methods for highly multiplexed tissue imaging methods. These reveal the intensities and spatial distributions of 20-100 proteins in 103-107 cells per specimen in a preserved tissue microenvironment. Despite extensive work on extracting single-cell image data, all tissue images are afflicted by artifacts (e.g., folds, debris, antibody aggregates, optical effects, image processing errors) that arise from imperfections in specimen preparation, data acquisition, image assembly, and feature extraction. We show that artifacts dramatically impact single-cell data analysis, in extreme cases, preventing meaningful biological interpretation. We describe an interactive quality control software tool, CyLinter, that identifies and removes data associated with imaging artifacts. CyLinter greatly improves single-cell analysis, especially for archival specimens sectioned many years prior to data collection, including those from clinical trials.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.14.562259">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.14.562259" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.14.562259">
        <p class="paperTitle">Recommendations For a Standardized Approach to Histopathologic Evaluation of Synovial Membrane in Murine Models of Experimental Osteoarthritis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.14.562259" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.14.562259" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Obeidat, A. M.; Kim, S. Y.; Burt, K. G.; Hu, B.; Li, J.; Ishihara, S.; Xiao, R.; Miller, R. E.; Little, C. B.; Malfait, A.-M.; Scanzello, C.</p>
        <p class="info">Score: 4.8, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.14.562259' target='https://doi.org/10.1101/2023.10.14.562259'> 10.1101/2023.10.14.562259</a></p>
        <p class="abstract">BackgroundSynovial pathology has been linked to osteoarthritis (OA) pain in patients. Microscopic grading systems for synovial changes in human OA have been described, but a standardized approach for murine models of OA is needed. We sought to develop a reproducible approach and set of minimum recommendations for synovial histopathology in mouse models of OA.

MethodsCoronal and sagittal sections from male mouse knee joints subjected to destabilization of medial meniscus (DMM) or partial meniscectomy (PMX) were collected as part of other studies. Stains included Hematoxylin and Eosin (H&amp;E), Toluidine Blue (T- Blue) and Safranin O/Fast Green (Saf-O). Four blinded readers graded pathological features (hyperplasia, cellularity, and fibrosis) at specific anatomic locations in the medial and lateral compartments. Inter-reader reliability of each feature was determined.

ResultsThere was acceptable to very good agreement between raters. After DMM, increased hyperplasia and cellularity and a trend towards increased fibrosis were observed 6 weeks after DMM in the medial locations, and persisted up to 16 weeks. In the PMX model, cellularity and hyperplasia were evident in both medial and lateral compartments while fibrotic changes were largely seen on the medial side. Synovial changes were consistent from section to section in the mid-joint area mice. H&amp;E, T-blue, and Saf-O stains resulted in comparable reliability.

ConclusionsTo allow for a standard evaluation that can be implemented and compared across labs and studies, we recommend using 3 readers to evaluate a minimum set of 3 pathological features at standardized anatomic areas. Pre-defining areas to be scored, and reliability for each pathologic feature should be considered.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.03.565593">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.03.565593" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.03.565593">
        <p class="paperTitle">Multi-omics characterization of lymphedema-induced adipose tissue from breast cancer patients</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.03.565593" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.03.565593" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Karaman, S.; Lehti, S.; Zhang, C.; Taskinen, M.-R.; Käkelä, R.; Mardinoglu, A.; Brorson, H.; Alitalo, K. K.; Kivelä, R.</p>
        <p class="info">Score: 1.8, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.03.565593' target='https://doi.org/10.1101/2023.11.03.565593'> 10.1101/2023.11.03.565593</a></p>
        <p class="abstract">Objective: Secondary lymphedema (LE) following breast cancer surgery is a life-long complication, which currently has no cure. LE induces significant regional adipose tissue deposition, requiring liposuction as a treatment. Here, we aimed to elucidate the transcriptional, metabolomic, and lipidomic signature of the adipose tissue developed due to the surgery-induced LE in short- and long-term LE patients, and compared the transcriptomic landscape in LE to the obesity-induced adipose tissue. Methods: Adipose tissue biopsies were obtained from breast cancer-operated females with LE from the affected and non-affected arms (n=20 patients). To decipher molecular properties of the LE adipose tissue, we performed RNA sequencing, metabolomics, and lipidomics combined with bioinformatics analyses. Results: Integrative analysis of functional genomics revealed that inflammatory response, cell chemotaxis and angiogenesis were upregulated biological processes in the LE arm, indicating a sustained inflammation in the edematous adipose tissue, whereas, epidermal differentiation, cell-cell junction organization, water homeostasis and neurogenesis were, in turn, downregulated in the LE arm. Surprisingly, only few genes were found to be the same in the LE-induced and the obesity-induced adipose tissue expansion, indicating a different type of adipose tissue development in these two diseases. In metabolomics analysis, the concentration of a branched-chain amino acid valine was found to be reduced in the edematous arm together with downregulation of mRNA levels of its transporter SLC6A15. Lipidomics analyses did not show any significant differences between the diseased and healthy arm, suggesting that diet affects the lipid composition of the adipose tissue more than the LE. Conclusions: Our results provide a detailed molecular characterization of adipose tissue in secondary LE of breast cancer patients vs individuals with obesity. The results show distinct differences in transcriptomic signatures between LE patients vs individuals with obesity, but only minor differences in metabolome and lipidome between the diseased and the healthy arm.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.565020">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.565020" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.565020">
        <p class="paperTitle">Integrating cellular graph embeddings with tumor morphological features to predict in-silico spatial transcriptomics from H&amp;E images</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.565020" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.565020" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, K.; Prabhakar, V.</p>
        <p class="info">Score: 1.5, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.565020' target='https://doi.org/10.1101/2023.10.31.565020'> 10.1101/2023.10.31.565020</a></p>
        <p class="abstract">Spatial transcriptomics allows precise RNA abundance measurement at high spatial resolution, linking cellular morphology with gene expression. We present a novel deep learning algorithm predicting local gene expression from histopathology images. Our approach employs a graph isomorphism neural network capturing cell-to-cell interactions in the tumor microenvironment and a Vision Transformer (CTransPath) for obtaining the tumor morphological features. Using a dataset of 30,612 spatially resolved gene expression profiles matched with histopathology images from 23 breast cancer patients, we identify 250 genes, including established breast cancer biomarkers, at a 100 {micro}m resolution. Additionally, we co-train our algorithm on spatial spot-level transcriptomics from 10x Visium breast cancer data along with another variant of our algorithm on TCGA-BRCA bulk RNA Seq. data, yielding mutual benefits and enhancing predictive accuracy on both these datasets. This work enables image-based screening for molecular biomarkers with spatial variation, promising breakthroughs in cancer research and diagnostics.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
