<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>pathology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>pathology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="PAH DEFICIENT PATHOLOGY IN HUMANIZED c.1066-11G&gt;A PHENYLKETONURIA MICE
Authors: Martinez-Pizarro, A.; Pico, S.; Lopez-Marquez, A.; Rodriguez-Lopez, C.; Montalvo, E.; Alvarez, M.; Castro, M.; Ramon-Maiques, S.; Perez, B.; Lucas, J. J.; Richard, E. M.; Desviat, L. R.
Score: 12.7, Published: 2023-11-05 DOI: 10.1101/2023.11.03.565447
We have generated using CRISPR/Cas9 technology a partially humanized mouse model of the neurometabolic disease phenylketonuria (PKU) carrying the prevalent splice intronic variant PAH c.1066-11G&gt;A. This variant creates a strong alternative splice acceptor site, leading to the inclusion of 9 nucleotides coding for a 3 amino acid insertion between Q355 and Y356 of the phenylalanine hydroxylase (PAH) protein.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/pathology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="pathology" />
<meta property="og:description" content="PAH DEFICIENT PATHOLOGY IN HUMANIZED c.1066-11G&gt;A PHENYLKETONURIA MICE
Authors: Martinez-Pizarro, A.; Pico, S.; Lopez-Marquez, A.; Rodriguez-Lopez, C.; Montalvo, E.; Alvarez, M.; Castro, M.; Ramon-Maiques, S.; Perez, B.; Lucas, J. J.; Richard, E. M.; Desviat, L. R.
Score: 12.7, Published: 2023-11-05 DOI: 10.1101/2023.11.03.565447
We have generated using CRISPR/Cas9 technology a partially humanized mouse model of the neurometabolic disease phenylketonuria (PKU) carrying the prevalent splice intronic variant PAH c.1066-11G&gt;A. This variant creates a strong alternative splice acceptor site, leading to the inclusion of 9 nucleotides coding for a 3 amino acid insertion between Q355 and Y356 of the phenylalanine hydroxylase (PAH) protein." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/pathology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-15T10:40:26+00:00" />
<meta property="article:modified_time" content="2023-11-15T10:40:26+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="pathology"/>
<meta name="twitter:description" content="PAH DEFICIENT PATHOLOGY IN HUMANIZED c.1066-11G&gt;A PHENYLKETONURIA MICE
Authors: Martinez-Pizarro, A.; Pico, S.; Lopez-Marquez, A.; Rodriguez-Lopez, C.; Montalvo, E.; Alvarez, M.; Castro, M.; Ramon-Maiques, S.; Perez, B.; Lucas, J. J.; Richard, E. M.; Desviat, L. R.
Score: 12.7, Published: 2023-11-05 DOI: 10.1101/2023.11.03.565447
We have generated using CRISPR/Cas9 technology a partially humanized mouse model of the neurometabolic disease phenylketonuria (PKU) carrying the prevalent splice intronic variant PAH c.1066-11G&gt;A. This variant creates a strong alternative splice acceptor site, leading to the inclusion of 9 nucleotides coding for a 3 amino acid insertion between Q355 and Y356 of the phenylalanine hydroxylase (PAH) protein."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "pathology",
      "item": "https://trxiv.yorks0n.com/posts/pathology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "pathology",
  "name": "pathology",
  "description": "PAH DEFICIENT PATHOLOGY IN HUMANIZED c.1066-11G\u0026gt;A PHENYLKETONURIA MICE\nAuthors: Martinez-Pizarro, A.; Pico, S.; Lopez-Marquez, A.; Rodriguez-Lopez, C.; Montalvo, E.; Alvarez, M.; Castro, M.; Ramon-Maiques, S.; Perez, B.; Lucas, J. J.; Richard, E. M.; Desviat, L. R.\nScore: 12.7, Published: 2023-11-05 DOI: 10.1101/2023.11.03.565447\nWe have generated using CRISPR/Cas9 technology a partially humanized mouse model of the neurometabolic disease phenylketonuria (PKU) carrying the prevalent splice intronic variant PAH c.1066-11G\u0026gt;A. This variant creates a strong alternative splice acceptor site, leading to the inclusion of 9 nucleotides coding for a 3 amino acid insertion between Q355 and Y356 of the phenylalanine hydroxylase (PAH) protein.",
  "keywords": [
    
  ],
  "articleBody": " PAH DEFICIENT PATHOLOGY IN HUMANIZED c.1066-11G\u003eA PHENYLKETONURIA MICE\nAuthors: Martinez-Pizarro, A.; Pico, S.; Lopez-Marquez, A.; Rodriguez-Lopez, C.; Montalvo, E.; Alvarez, M.; Castro, M.; Ramon-Maiques, S.; Perez, B.; Lucas, J. J.; Richard, E. M.; Desviat, L. R.\nScore: 12.7, Published: 2023-11-05 DOI: 10.1101/2023.11.03.565447\nWe have generated using CRISPR/Cas9 technology a partially humanized mouse model of the neurometabolic disease phenylketonuria (PKU) carrying the prevalent splice intronic variant PAH c.1066-11G\u003eA. This variant creates a strong alternative splice acceptor site, leading to the inclusion of 9 nucleotides coding for a 3 amino acid insertion between Q355 and Y356 of the phenylalanine hydroxylase (PAH) protein. Homozygous Pah c.1066-11A mice, with a partially humanized intron 10 sequence with the variant, accurately recapitulate the splicing defect and present almost undetectable hepatic PAH protein or activity. They exhibit fur hypopigmentation, lower brain and body weight and reduced survival. Blood and brain phenylalanine (Phe) levels are elevated, along with decreased tyrosine (Tyr), tryptophan (Trp) and monoamine neurotransmitters levels. Pah c.1066-11A mice present behavioral deficits, mainly hypoactivity and diminished social interaction, as well as locomotor deficiencies. Abnormal hind-limb clasping reflex and microophtalmia are observed. Immunohistochemistry analyses of GFAP and Iba1 markers revealed changes in the morphology of glial cells and a significant increase in GFAP and Iba1 staining signals. Western blot analyses in mouse liver lysates and in gene edited HepG2 cells with the variant showed reduced levels of chaperones DNAJC12 and HSC70/HSP70 and an increase in autophagy markers LAMP1 and LC3BII, suggesting that the mutant pQ355_Y356insGLQ PAH protein could aggregate complexed with DNAJC12 and HSP70 proteins, with the aggregates being processed by autophagy. This humanized model corresponding to a severe, classical PKU phenotype represents a useful tool for pathophysiology research in PKU and for novel therapies development.\nA WIDESPREAD PICORNAVIRUS AFFECTS THE HAEMOCYTES OF THE NOBLE PEN SHELL (PINNA NOBILIS) LEADING TO IMMUNOSUPPRESSION\nAuthors: Carella, F.; Prado, P.; De Vico, G.; Palic, D.; Villari, G.; Garcia-March, J. R.; Tena-Mediadea, J.; Melendreras, E. C.; Gimenez-Casalduero, F.; Sigovini, M.; Aceto, S.\nScore: 1.2, Published: 2023-11-05 DOI: 10.1101/2023.11.05.565683\nThe widespread mass mortality of the noble pen shell (Pinna nobilis) has occurred in several Mediterranean countries in the past seven years. Single-stranded RNA virus affecting immune cells and leading to immune disfunction have been widely reported in human and animal species. Here we present data linking P. nobilis mass mortality events (MMEs) to haemocyte picornavirus (PV) infection. This study was performed on 30 specimens, from wild and captive populations. We sampled P. nobilis from two regions of Spain, Catalonia [24 animals] and Murcia [two animals]), and one region in Italy (Venice [four animals]). The low number of analyzed specimens was due to the scarcity of remaining individuals in the Mediterranean Sea. In 100% of our samples, ultrastructure revealed the presence of a virus (20nm diameter), capable of replicating within granulocytes and hyalinocytes, leading to the accumulation of complex vesicles of different dimensions within the cytoplasm. As the PV infection progressed, dead haemocytes, infectious exosomes, and budding of extracellular vesicles were visible, along with endocytic vesicles entering other cells. The THC (total haemocyte count) values observed in both captive (eight animals) (3.5 x 104 - 1.60 x105 ml-1 cells) and wild animals (14 samples) (1.90 - 2.42 x105 ml-1 cells) were lower than those reported before MMEs. Sequencing of P. nobilis (six animals) haemocyte cDNA libraries revealed the presence of two main sequences of Picornavirales, family Marnaviridae. The highest number of reads belonged to animals that exhibited active replication phases and abundant viral particles from Trasmission Electron Microscopy (TEM) observations. These sequences correspond to the genus Sogarnavirus - a picornavirus identified in the marine diatom Chaetoceros tenuissimus (named C. tenuissimus RNA virus type II). Real time PCR performed on the two most abundant RNA viruses previously identified by in silico analysis revealed positive results only for the sequences similar to C. tenuissimus RNA virus. These results may not be considered conclusive of picornavirus identification in noble pen shell haemocytes, and require further studies. Our findings suggest that picornavirus infection likely causes immunosuppression, making individuals prone to opportunistic infections which is a potential cause for the MMEs observed in the Mediterranean.\nPlatelet proteo-transcriptomic profiling validates mediators of thrombosis and proteostasis in patients with myeloproliferative neoplasms\nAuthors: Kelliher, S.; Gamba, S.; Weiss, L.; Shen, Z.; Marchetti, M.; Schieppati, F.; Scaife, C.; Madden, S.; Bennett, K.; Fortune, A.; Maung, S.; Fay, M.; Ni Ainle, F.; Maguire, P.; Falanga, A.; Kevane, B.; Krishnan, A.\nScore: 4.7, Published: 2023-10-26 DOI: 10.1101/2023.10.23.563619\nPatients with chronic Myeloproliferative Neoplasms (MPN) including polycythemia vera (PV) and essential thrombocythemia (ET) exhibit unique clinical features, such as a tendency toward thrombosis and hemorrhage, and risk of disease progression to secondary bone marrow fibrosis and/or acute leukemia. Although an increase in blood cell lineage counts (quantitative features) contribute to these morbid sequelae, the significant qualitative abnormalities of myeloid cells that contribute to vascular risk are not well understood. Here, we address this critical knowledge gap via a comprehensive and untargeted profiling of the platelet proteome in a large (n= 140) cohort of patients (from two independent sites) with an established diagnosis of PV and ET (and complement prior work on the MPN platelet transcriptome from a third site). We discover distinct MPN platelet protein expression and confirm key molecular impairments associated with proteostasis and thrombosis mechanisms of potential relevance to MPN pathology. Specifically, we validate expression of high-priority candidate markers from the platelet transcriptome at the platelet proteome (e.g., calreticulin (CALR), Fc gamma receptor (Fc{gamma}RIIA) and galectin-1 (LGALS1) pointing to their likely significance in the proinflammatory, prothrombotic and profibrotic phenotypes in patients with MPN. Together, our proteo-transcriptomic study identifies the peripherally-derived platelet molecular profile as a potential window into MPN pathophysiology and demonstrates the value of integrative multi-omic approaches in gaining a better understanding of the complex molecular dynamics of disease.\n",
  "wordCount" : "965",
  "inLanguage": "en",
  "datePublished": "2023-11-15T10:40:26Z",
  "dateModified": "2023-11-15T10:40:26Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/pathology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      pathology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 15, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.03.565447">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.03.565447" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.03.565447">
        <p class="paperTitle">PAH DEFICIENT PATHOLOGY IN HUMANIZED c.1066-11G&gt;A PHENYLKETONURIA MICE</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.03.565447" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.03.565447" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martinez-Pizarro, A.; Pico, S.; Lopez-Marquez, A.; Rodriguez-Lopez, C.; Montalvo, E.; Alvarez, M.; Castro, M.; Ramon-Maiques, S.; Perez, B.; Lucas, J. J.; Richard, E. M.; Desviat, L. R.</p>
        <p class="info">Score: 12.7, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.03.565447' target='https://doi.org/10.1101/2023.11.03.565447'> 10.1101/2023.11.03.565447</a></p>
        <p class="abstract">We have generated using CRISPR/Cas9 technology a partially humanized mouse model of the neurometabolic disease phenylketonuria (PKU) carrying the prevalent splice intronic variant PAH c.1066-11G&gt;A. This variant creates a strong alternative splice acceptor site, leading to the inclusion of 9 nucleotides coding for a 3 amino acid insertion between Q355 and Y356 of the phenylalanine hydroxylase (PAH) protein. Homozygous Pah c.1066-11A mice, with a partially humanized intron 10 sequence with the variant, accurately recapitulate the splicing defect and present almost undetectable hepatic PAH protein or activity. They exhibit fur hypopigmentation, lower brain and body weight and reduced survival. Blood and brain phenylalanine (Phe) levels are elevated, along with decreased tyrosine (Tyr), tryptophan (Trp) and monoamine neurotransmitters levels. Pah c.1066-11A mice present behavioral deficits, mainly hypoactivity and diminished social interaction, as well as locomotor deficiencies. Abnormal hind-limb clasping reflex and microophtalmia are observed. Immunohistochemistry analyses of GFAP and Iba1 markers revealed changes in the morphology of glial cells and a significant increase in GFAP and Iba1 staining signals. Western blot analyses in mouse liver lysates and in gene edited HepG2 cells with the variant showed reduced levels of chaperones DNAJC12 and HSC70/HSP70 and an increase in autophagy markers LAMP1 and LC3BII, suggesting that the mutant pQ355_Y356insGLQ PAH protein could aggregate complexed with DNAJC12 and HSP70 proteins, with the aggregates being processed by autophagy. This humanized model corresponding to a severe, classical PKU phenotype represents a useful tool for pathophysiology research in PKU and for novel therapies development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.05.565683">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.05.565683" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.05.565683">
        <p class="paperTitle">A WIDESPREAD PICORNAVIRUS AFFECTS THE HAEMOCYTES OF THE NOBLE PEN SHELL (PINNA NOBILIS) LEADING TO IMMUNOSUPPRESSION</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.05.565683" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.05.565683" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carella, F.; Prado, P.; De Vico, G.; Palic, D.; Villari, G.; Garcia-March, J. R.; Tena-Mediadea, J.; Melendreras, E. C.; Gimenez-Casalduero, F.; Sigovini, M.; Aceto, S.</p>
        <p class="info">Score: 1.2, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.05.565683' target='https://doi.org/10.1101/2023.11.05.565683'> 10.1101/2023.11.05.565683</a></p>
        <p class="abstract">The widespread mass mortality of the noble pen shell (Pinna nobilis) has occurred in several Mediterranean countries in the past seven years. Single-stranded RNA virus affecting immune cells and leading to immune disfunction have been widely reported in human and animal species. Here we present data linking P. nobilis mass mortality events (MMEs) to haemocyte picornavirus (PV) infection. This study was performed on 30 specimens, from wild and captive populations. We sampled P. nobilis from two regions of Spain, Catalonia [24 animals] and Murcia [two animals]), and one region in Italy (Venice [four animals]). The low number of analyzed specimens was due to the scarcity of remaining individuals in the Mediterranean Sea. In 100% of our samples, ultrastructure revealed the presence of a virus (20nm diameter), capable of replicating within granulocytes and hyalinocytes, leading to the accumulation of complex vesicles of different dimensions within the cytoplasm. As the PV infection progressed, dead haemocytes, infectious exosomes, and budding of extracellular vesicles were visible, along with endocytic vesicles entering other cells. The THC (total haemocyte count) values observed in both captive (eight animals) (3.5 x 104 - 1.60 x105 ml-1 cells) and wild animals (14 samples) (1.90 - 2.42 x105 ml-1 cells) were lower than those reported before MMEs. Sequencing of P. nobilis (six animals) haemocyte cDNA libraries revealed the presence of two main sequences of Picornavirales, family Marnaviridae. The highest number of reads belonged to animals that exhibited active replication phases and abundant viral particles from Trasmission Electron Microscopy (TEM) observations. These sequences correspond to the genus Sogarnavirus - a picornavirus identified in the marine diatom Chaetoceros tenuissimus (named C. tenuissimus RNA virus type II). Real time PCR performed on the two most abundant RNA viruses previously identified by in silico analysis revealed positive results only for the sequences similar to C. tenuissimus RNA virus. These results may not be considered conclusive of picornavirus identification in noble pen shell haemocytes, and require further studies. Our findings suggest that picornavirus infection likely causes immunosuppression, making individuals prone to opportunistic infections which is a potential cause for the MMEs observed in the Mediterranean.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563619">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563619" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563619">
        <p class="paperTitle">Platelet proteo-transcriptomic profiling validates mediators of thrombosis and proteostasis in patients with myeloproliferative neoplasms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563619" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563619" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kelliher, S.; Gamba, S.; Weiss, L.; Shen, Z.; Marchetti, M.; Schieppati, F.; Scaife, C.; Madden, S.; Bennett, K.; Fortune, A.; Maung, S.; Fay, M.; Ni Ainle, F.; Maguire, P.; Falanga, A.; Kevane, B.; Krishnan, A.</p>
        <p class="info">Score: 4.7, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563619' target='https://doi.org/10.1101/2023.10.23.563619'> 10.1101/2023.10.23.563619</a></p>
        <p class="abstract">Patients with chronic Myeloproliferative Neoplasms (MPN) including polycythemia vera (PV) and essential thrombocythemia (ET) exhibit unique clinical features, such as a tendency toward thrombosis and hemorrhage, and risk of disease progression to secondary bone marrow fibrosis and/or acute leukemia. Although an increase in blood cell lineage counts (quantitative features) contribute to these morbid sequelae, the significant qualitative abnormalities of myeloid cells that contribute to vascular risk are not well understood. Here, we address this critical knowledge gap via a comprehensive and untargeted profiling of the platelet proteome in a large (n= 140) cohort of patients (from two independent sites) with an established diagnosis of PV and ET (and complement prior work on the MPN platelet transcriptome from a third site). We discover distinct MPN platelet protein expression and confirm key molecular impairments associated with proteostasis and thrombosis mechanisms of potential relevance to MPN pathology. Specifically, we validate expression of high-priority candidate markers from the platelet transcriptome at the platelet proteome (e.g., calreticulin (CALR), Fc gamma receptor (Fc{gamma}RIIA) and galectin-1 (LGALS1) pointing to their likely significance in the proinflammatory, prothrombotic and profibrotic phenotypes in patients with MPN. Together, our proteo-transcriptomic study identifies the peripherally-derived platelet molecular profile as a potential window into MPN pathophysiology and demonstrates the value of integrative multi-omic approaches in gaining a better understanding of the complex molecular dynamics of disease.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
