<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="An Organism-Level Quantitative Flux Model of Mammalian Energy Metabolism
Authors: Yuan, B.; Inouye, K. E.; Hotamisligil, G.; Hui, S.
Score: 37.5, Published: 2024-02-12 DOI: 10.1101/2024.02.11.579776
Mammalian tissues feed on nutrients in the blood circulation. At the organism-level, mammalian energy metabolism comprises of oxidation, interconverting, storing and releasing of circulating nutrients. Though much is known about the individual processes and nutrients, a holistic and quantitative model describing these processes for all major circulating nutrients is lacking.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="An Organism-Level Quantitative Flux Model of Mammalian Energy Metabolism
Authors: Yuan, B.; Inouye, K. E.; Hotamisligil, G.; Hui, S.
Score: 37.5, Published: 2024-02-12 DOI: 10.1101/2024.02.11.579776
Mammalian tissues feed on nutrients in the blood circulation. At the organism-level, mammalian energy metabolism comprises of oxidation, interconverting, storing and releasing of circulating nutrients. Though much is known about the individual processes and nutrients, a holistic and quantitative model describing these processes for all major circulating nutrients is lacking." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-18T10:38:00+00:00" />
<meta property="article:modified_time" content="2024-02-18T10:38:00+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="An Organism-Level Quantitative Flux Model of Mammalian Energy Metabolism
Authors: Yuan, B.; Inouye, K. E.; Hotamisligil, G.; Hui, S.
Score: 37.5, Published: 2024-02-12 DOI: 10.1101/2024.02.11.579776
Mammalian tissues feed on nutrients in the blood circulation. At the organism-level, mammalian energy metabolism comprises of oxidation, interconverting, storing and releasing of circulating nutrients. Though much is known about the individual processes and nutrients, a holistic and quantitative model describing these processes for all major circulating nutrients is lacking."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "An Organism-Level Quantitative Flux Model of Mammalian Energy Metabolism\nAuthors: Yuan, B.; Inouye, K. E.; Hotamisligil, G.; Hui, S.\nScore: 37.5, Published: 2024-02-12 DOI: 10.1101/2024.02.11.579776\nMammalian tissues feed on nutrients in the blood circulation. At the organism-level, mammalian energy metabolism comprises of oxidation, interconverting, storing and releasing of circulating nutrients. Though much is known about the individual processes and nutrients, a holistic and quantitative model describing these processes for all major circulating nutrients is lacking.",
  "keywords": [
    
  ],
  "articleBody": " An Organism-Level Quantitative Flux Model of Mammalian Energy Metabolism\nAuthors: Yuan, B.; Inouye, K. E.; Hotamisligil, G.; Hui, S.\nScore: 37.5, Published: 2024-02-12 DOI: 10.1101/2024.02.11.579776\nMammalian tissues feed on nutrients in the blood circulation. At the organism-level, mammalian energy metabolism comprises of oxidation, interconverting, storing and releasing of circulating nutrients. Though much is known about the individual processes and nutrients, a holistic and quantitative model describing these processes for all major circulating nutrients is lacking. Here, by integrating isotope tracer infusion, mass spectrometry, and isotope gas analyzer measurement, we developed a framework to systematically quantify fluxes through these processes for 10 major circulating energy nutrients in mice, resulting in an organism-level quantitative flux model of energy metabolism. This model revealed in wildtype mice that circulating nutrients metabolic cycling fluxes are more dominant than their oxidation fluxes, with distinct partition between cycling and oxidation flux for individual circulating nutrients. Applications of this framework in obese mouse models showed on a per animal basis extensive elevation of metabolic cycling fluxes in ob/ob mice, but not in diet-induced obese mice. Thus, our framework describes quantitatively the functioning of energy metabolism at the organism-level, valuable for revealing new features of energy metabolism in physiological and disease conditions. HighlightsO_LIA flux model of energy metabolism integrating 13C labeling of metabolites and CO2 C_LIO_LICirculating nutrients have characteristic partition between oxidation and storing C_LIO_LICirculating nutrients total cycling flux outweighs their total oxidation flux C_LIO_LICycling fluxes are extensively elevated in ob/ob but not diet-induced obese mice C_LI Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=200 SRC=\"FIGDIR/small/579776v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (48K): org.highwire.dtl.DTLVardef@1ae3eb9org.highwire.dtl.DTLVardef@980a80org.highwire.dtl.DTLVardef@1d9682aorg.highwire.dtl.DTLVardef@45bf2c_HPS_FORMAT_FIGEXP M_FIG C_FIG\nSpatially Exploring RNA Biology in Archival Formalin-Fixed Paraffin-Embedded Tissues\nAuthors: Bai, Z.; Zhang, D.; Gao, Y.; Tao, B.; Bao, S.; Enninful, A.; Zhang, D.; Su, G.; Tian, X.; Zhang, N.; Xiao, Y.; Liu, Y.; Gerstein, M.; Li, M.; Xing, Y.; Lu, J.; Xu, M. L.; Fan, R.\nScore: 78.9, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579143\nSpatial transcriptomics has emerged as a powerful tool for dissecting spatial cellular heterogeneity but as of today is largely limited to gene expression analysis. Yet, the life of RNA molecules is multifaceted and dynamic, requiring spatial profiling of different RNA species throughout the life cycle to delve into the intricate RNA biology in complex tissues. Human disease-relevant tissues are commonly preserved as formalin-fixed and paraffin-embedded (FFPE) blocks, representing an important resource for human tissue specimens. The capability to spatially explore RNA biology in FFPE tissues holds transformative potential for human biology research and clinical histopathology. Here, we present Patho-DBiT combining in situ polyadenylation and deterministic barcoding for spatial full coverage transcriptome sequencing, tailored for probing the diverse landscape of RNA species even in clinically archived FFPE samples. It permits spatial co-profiling of gene expression and RNA processing, unveiling region-specific splicing isoforms, and high-sensitivity transcriptomic mapping of clinical tumor FFPE tissues stored for five years. Furthermore, genome-wide single nucleotide RNA variants can be captured to distinguish different malignant clones from non-malignant cells in human lymphomas. Patho-DBiT also maps microRNA-mRNA regulatory networks and RNA splicing dynamics, decoding their roles in spatial tumorigenesis trajectory. High resolution Patho-DBiT at the cellular level reveals a spatial neighborhood and traces the spatiotemporal kinetics driving tumor progression. Patho-DBiT stands poised as a valuable platform to unravel rich RNA biology in FFPE tissues to study human tissue biology and aid in clinical pathology evaluation.\nSemi-supervised Bayesian integration of multiple spatial proteomics datasets\nAuthors: Coleman, S. D.; Breckels, L.; Waller, R. F.; Lilley, K. S.; Wallace, C.; Kirk, P. D. W.; Crook, O. M.\nScore: 4.2, Published: 2024-02-12 DOI: 10.1101/2024.02.08.579519\nThe subcellular localisation of proteins is a key determinant of their function. High-throughput analyses of these localisations can be performed using mass spectrometry-based spatial proteomics, which enables us to examine the localisation and relocalisation of proteins. Furthermore, complementary data sources can provide additional sources of functional or localisation information. Examples include protein annotations and other high-throughput omic assays. Integrating these modalities can provide new insights as well as additional confidence in results, but existing approaches for integrative analyses of spatial proteomics datasets are limited in the types of data they can integrate and do not quantify uncertainty. Here we propose a semi-supervised Bayesian approach to integrate spatial proteomics datasets with other data sources, to improve the inference of protein sub-cellular localisation. We demonstrate our approach outperforms other transfer learning methods and has greater flexibility in the data it can model. To demonstrate the flexibility of our approach, we apply our method to integrate spatial proteomics data generated for the parasite Toxoplasma gondii with time-course gene expression data generated over its cell cycle. Our findings suggest that proteins linked to invasion organelles are associated with expression programs that peak at the end of the first cell-cycle. Furthermore, this integrative analysis divides the dense granule proteins into heterogeneous populations suggestive of potentially different functions. Our method is disseminated via the mdir R package available on the lead authors Github. Author summaryProteins are located in subcellular environments to ensure that they are near their interaction partners and occur in the correct biochemical environment to function. Where a protein is located can be determined from a number of data sources. To integrate diverse datasets together we develop an integrative Bayesian model to combine the information from several datasets in a principled manner. We learn how similar the dataset are as part of the modelling process and demonstrate the benefits of integrating mass-spectrometry based spatial proteomics data with timecourse gene-expression datasets.\nSelective enrichment of high-affinity clade II N2O-reducers in a mixed culture\nAuthors: Laureni, M.; Corbera Rubio, F.; Kim, D. D.; Browne, S.; Roothans, N.; Weissbrodt, D. G.; Olavarria, K. O.; de Jonge, N.; Yoon, S.; Pabst, M.; van Loosdrecht, M. C. M.\nScore: 4.1, Published: 2024-02-12 DOI: 10.1101/2024.02.09.579283\nMicroorganisms encoding for the N2O reductase (NosZ) are the only known biological sink of the potent greenhouse gas N2O, and are central to global N2O mitigation efforts. Yet, the ecological constraints selecting for different N2O-reducers strains and controlling the assembly of N2O-respiring communities remain largely unknown. Of particular biotechnological interest are clade II NosZ populations, which usually feature high N2O affinities and often lack other denitrification genes. Two planktonic N2O-respiring mixed cultures were enriched under limiting and excess dissolved N2O availability to assess the impact of substrate affinity and N2O cytotoxicity, respectively. Genome-resolved metaproteomics was used to infer the metabolism of the enriched populations. We show that clade II N2O-reducers outcompete clade I affiliates for N2O at sufficiently low sludge dilution rates (0.006 h-1), a scenario previously only theorized based on pure-cultures. Under N2O limitation, all enriched N2O-reducers encoded and expressed only clade II NosZ, while also possessing other denitrification genes. Two Azonexus and Thauera genera affiliates dominated the culture. We explain their coexistence with the genome-inferred metabolic exchange of cobalamin intermediates. Conversely, under excess N2O, clade I and II populations coexisted. Notably, the single dominant N2O-reducer (genus Azonexus) expressed most cobalamin biosynthesis marker genes, likely to contrast the continuous cobalamin inactivation by dissolved cytotoxic N2O concentrations (400 {micro}M). Ultimately, we demonstrate that the solids dilution rate controls the selection among NosZ clades, albeit the conditions selecting for genomes possessing the sole nosZ remain elusive. Additionally, we suggest the significance of N2O-cobalamin interactions in shaping the composition of N2O-respiring microbiomes.\nLarge-Scale Evaluation of Spatial Metabolomics Protocols and Technologies\nAuthors: Saharuka, V.; Vieira, L. M.; Stuart, L.; Ekelöf, M.; Molenaar, M. R.; Bailoni, A.; Ovchinnikova, K.; Soltwisch, J.; Bausbacher, T.; Jakob, D.; King, M.; Müller, M. A.; Oetjen, J.; Pace, C.; Pinto, F. E.; Strittmatter, N.; Velickovic, D.; Spengler, B.; Muddiman, D. C.; Liebeke, M.; Janfelt, C.; Goodwin, R.; Eberlin, L. S.; Anderton, C. R.; Hopf, C.; Dreisewerd, K.; Alexandrov, T.\nScore: 27.0, Published: 2024-01-31 DOI: 10.1101/2024.01.29.577354\nSpatial metabolomics using imaging mass spectrometry (MS) enables untargeted and label-free metabolite mapping in biological samples. Despite the range of available imaging MS protocols and technologies, our understanding of metabolite detection under specific conditions is limited due to sparse empirical data and predictive theories. Consequently, challenges persist in designing new experiments, and accurately annotating and interpreting data. In this study, we systematically measured the detectability of 172 biologically-relevant metabolites across common imaging MS protocols using custom reference samples. We evaluated 24 MALDI-imaging MS protocols for untargeted metabolomics, and demonstrated the applicability of our findings to complex biological samples through comparison with animal tissue data. We showcased the potential for extending our results to further analytes by predicting metabolite detectability based on molecular properties. Additionally, our interlaboratory comparison of 10 imaging MS technologies, including MALDI, DESI, and IR-MALDESI, showed extensive metabolite coverage and comparable results, underscoring the broad applicability of our findings within the imaging MS community. We share our results and data through a new interactive web application integrated with METASPACE. This resource offers an extensive catalogue of detectable metabolite ions, facilitating protocol selection, supporting data annotation, and benefiting future untargeted spatial metabolomics studies.\nChromatin structure and var2csa - a tango in regulation of var gene expression in the human malaria parasite Plasmodium falciparum?\nAuthors: Lenz, T.; Zhang, X.; Chakraborty, A.; Roayaei Ardakany, A.; Prudhomme, J.; Ay, F.; Deitsch, K.; Le Roch, K.\nScore: 1.2, Published: 2024-02-13 DOI: 10.1101/2024.02.13.580059\nOver the last few decades, novel methods have been developed to study how chromosome positioning within the nucleus may play a role in gene regulation. Adaptation of these methods in the human malaria parasite, Plasmodium falciparum, has recently led to the discovery that the three-dimensional structure of chromatin within the nucleus may be critical in controlling expression of virulence genes (var genes). Recent work has implicated an unusual, highly conserved var gene called var2csa in contributing to coordinated transcriptional switching, however how this gene functions in this capacity is unknown. To further understand how var2csa influences var gene switching, targeted DNA double-strand breaks (DSBs) within the sub-telomeric region of chromosome 12 were used to delete the gene and the surrounding chromosomal region. To characterize the changes in chromatin architecture stemming from this deletion and how these changes could affect var gene expression, we used a combination of RNA-seq, Chip-seq and Hi-C to pinpoint epigenetic and chromatin structural modifications in regions of differential gene expression. We observed a net gain of interactions in sub-telomeric regions and internal var gene regions following var2csa knockout, indicating an increase of tightly controlled heterochromatin structures. Our results suggest that disruption of var2csa results not only in changes in var gene transcriptional regulation but also a significant tightening of heterochromatin clusters thereby disrupting coordinated activation of var genes throughout the genome. Altogether our result confirms a strong link between the var2csa locus, chromatin structure and var gene expression. AUTHOR SUMMARYMalaria remains one of the deadliest parasite-borne diseases, causing not only over a half million deaths annually, but also infecting hundreds of millions more. Plasmodium falciparum, the protozoan parasite that is responsible for the most virulent form of human malaria, is transmitted to humans by infected female mosquitoes during a blood meal. Due to a growing resistance to all existing antimalarials, there is a need to identify novel targets to design new antimalarial strategies. Our research builds on the growing body of evidence that supports the role of genome organization or chromatin structure within the nucleus in controlling the parasite development as well as virulence factors designed to circumvent the host immune response. This study identifies genes and structural elements within the Plasmodium falciparum genome that are controlled, at least partially, by the expression of a single unique and highly conserved virulence gene.\nProteomic profiling of intracranial atherosclerotic plaque in the human brain\nAuthors: Hao, Q.; Wang, E.; Wang, J.; Wu, Z.; Crary, J. F.; Sharma, S.; Thorn, E. L.; Elahi, F.; Zhang, B.; Peng, J.\nScore: 1.2, Published: 2024-02-12 DOI: 10.1101/2024.02.10.579787\nBackgroundIntracranial atherosclerotic disease (ICAD) is one of the major causes of ischemic stroke and associated with high risk of stroke recurrence. There are no reliable and specific fluid biomarkers for ICAD, and little is known about the proteomic profiling of ICAD. In this study we aimed to explore the feasibility of applying proteomics technology to profile intracranial atherosclerotic plaques extracted from postmortem human brain arteries. MethodsEighteen segments (5-10mm in length) of major arteries from 10 postmortem brains were collected from the Mount Sinai Neuropathology Brain Bank. Among these segments, 5 had no evidence of atherosclerotic disease, and 13 had wall thickening or visible plaques with various degree of stenosis. Proteins were extracted from the vessel segments, quantified, and digested into peptides. Subsequently, the peptides underwent tandem mass tag (TMT) labeling, pooling, and analysis using two-dimensional liquid chromatography-tandem mass spectrometry (LC/LC-MS/MS). Protein identification and quantification were performed using the JUMP software. Differentially expressed proteins (DEPs) were defined as proteins with p.adj \u003c 0.05 and absolute log2 (fold change) \u003e log2 (1.2). ResultsA total of 7,492 unique proteins were detected, and 6,726 quantifiable proteins were retained for further analysis. Among these, 265 DEPs, spanning on 252 unique gene, were found to be associated with ICAD by comparing the arterial segments with vs those without atherosclerotic disease. The top 4 most significant DEPs include LONP1, RPS19, MRPL12 and SNU13. Among the top 50 DEPs, FADD, AIFM1 and PGK1 were associated with atherosclerotic disease or cardiovascular events in previous studies. Moreover, the previously reported proteins associated with atherosclerosis such as APCS, MMP12, CTSD were elevated in arterial segments with atherosclerotic changes. Furthermore, the up-regulation of APOE and LPL, the ICAD GWAS risk genes, was shown to be associated with the plaque severity. Finally, gene set enrichment analysis revealed the DEP signature is enriched for biological pathways such as chromatin structure, plasma lipoprotein, nucleosome, and protein-DNA complex, peroxide catabolic and metabolic processes, critical in ICAD pathology. ConclusionsDirect proteomic profiling of fresh-frozen intracranial artery samples by MS-based proteomic technology is a feasible approach to identify ICAD-associated proteins, which can be potential biomarker candidates for ICAD. Further plaque proteomic study in a larger sample size is warranted to uncover mechanistic insights into ICAD and discover novel biomarkers that may help to improve diagnosis and risk stratification in ICAD.\nA multiscale sensorimotor model of experience-dependent behavior in a minimal organism\nAuthors: Vidal-Saez, M. S.; Vilarroya, O.; Garcia-Ojalvo, J.\nScore: 1.2, Published: 2024-02-10 DOI: 10.1101/2024.02.07.579358\nTo survive in ever-changing environments, living organisms need to continuously combine the ongoing external inputs they receive, representing present conditions, with their dynamical internal state, which includes influences of past experiences. It is still unclear in general, however, (i) how this happens at the molecular and cellular levels, and (ii) how the corresponding molecular and cellular processes are integrated with the behavioral responses of the organism. Here we address these issues by modeling mathematically a particular behavioral paradigm in a minimal model organism, namely chemotaxis in the nematode C. elegans. Specifically, we use a long-standing collection of elegant experiments on salt chemotaxis in this animal, in which the migration direction varies depending on its previous experience. Our model integrates the molecular, cellular and organismal levels to reproduce the experimentally observed experience-dependent behavior. The model proposes specific molecular mechanisms for the encoding of current conditions and past experiences in key neurons associated with this response, predicting the behavior of various mutants associated with those molecular circuits.\nGlobal sensitivity analysis for investigation of bacterial physiology in complex media with FBA-PRCC\nAuthors: Sorokin, A.; Goryanin, I.\nScore: 0.8, Published: 2024-02-16 DOI: 10.1101/2024.02.14.580207\nIn this paper we propose the use of FBA-PRCC approach, which is based on sampling flux space and global sensitivity analysis (GSA), for analysis of the GEM FBA solution space in the nutrient-rich environment. We identify two new modes of interaction between species and metabolites: attraction and avoidance and propose the way of their use. We have shown that sensitivity coefficients provide additional information about behavior of the bacteria in the nutrient-rich environment in comparison to standard knockouts, FVA and CoPE-FBA analysis.\nPooled PPIseq: screening the SARS-CoV-2 and human interface with a scalable multiplexed protein-protein interaction assay platform\nAuthors: Miller, D.; Dziulko, A.; Levy, S.\nScore: 0.8, Published: 2024-02-14 DOI: 10.1101/2024.02.13.580123\nProtein-Protein Interactions (PPIs) are a key interface between virus and host, and these interactions are important to both viral reprogramming of the host and to host restriction of viral infection. In particular, viral-host PPI networks can be used to further our understanding of the molecular mechanisms of tissue specificity, host range, and virulence. At higher scales, viral-host PPI screening could also be used to screen for small-molecule antivirals that interfere with essential viral-host interactions, or to explore how the PPI networks between interacting viral and host genomes co-evolve. Current high-throughput PPI assays have screened entire viral-host PPI networks. However, these studies are time consuming, often require specialized equipment, and are difficult to further scale. Here, we develop methods that make larger-scale viral-host PPI screening more accessible. This approach combines the mDHFR split-tag reporter with the iSeq2 interaction-barcoding system to permit massively-multiplexed PPI quantification by simple pooled engineering of barcoded constructs, integration of these constructs into budding yeast, and fitness measurements by pooled cell competitions and barcode-sequencing. We applied this method to screen for PPIs between SARS-CoV-2 proteins and human proteins, screening in triplicate \u003e180,000 ORF-ORF combinations represented by \u003e1,000,000 barcoded lineages. Our results complement previous screens by identifying 74 putative PPIs, including interactions between ORF7A with the taste receptors TAS2R41 and TAS2R7, and between NSP4 with the transmembrane KDELR2 and KDELR3. We show that this PPI screening method is highly scalable, enabling larger studies aimed at generating a broad understanding of how viral effector proteins converge on cellular targets to effect replication.\n",
  "wordCount" : "2900",
  "inLanguage": "en",
  "datePublished": "2024-02-18T10:38:00Z",
  "dateModified": "2024-02-18T10:38:00Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 18, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.11.579776">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.11.579776" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.11.579776">
        <p class="paperTitle">An Organism-Level Quantitative Flux Model of Mammalian Energy Metabolism</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.11.579776" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.11.579776" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yuan, B.; Inouye, K. E.; Hotamisligil, G.; Hui, S.</p>
        <p class="info">Score: 37.5, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.11.579776' target='https://doi.org/10.1101/2024.02.11.579776'> 10.1101/2024.02.11.579776</a></p>
        <p class="abstract">Mammalian tissues feed on nutrients in the blood circulation. At the organism-level, mammalian energy metabolism comprises of oxidation, interconverting, storing and releasing of circulating nutrients. Though much is known about the individual processes and nutrients, a holistic and quantitative model describing these processes for all major circulating nutrients is lacking. Here, by integrating isotope tracer infusion, mass spectrometry, and isotope gas analyzer measurement, we developed a framework to systematically quantify fluxes through these processes for 10 major circulating energy nutrients in mice, resulting in an organism-level quantitative flux model of energy metabolism. This model revealed in wildtype mice that circulating nutrients metabolic cycling fluxes are more dominant than their oxidation fluxes, with distinct partition between cycling and oxidation flux for individual circulating nutrients. Applications of this framework in obese mouse models showed on a per animal basis extensive elevation of metabolic cycling fluxes in ob/ob mice, but not in diet-induced obese mice. Thus, our framework describes quantitatively the functioning of energy metabolism at the organism-level, valuable for revealing new features of energy metabolism in physiological and disease conditions.

HighlightsO_LIA flux model of energy metabolism integrating 13C labeling of metabolites and CO2
C_LIO_LICirculating nutrients have characteristic partition between oxidation and storing
C_LIO_LICirculating nutrients total cycling flux outweighs their total oxidation flux
C_LIO_LICycling fluxes are extensively elevated in ob/ob but not diet-induced obese mice
C_LI

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=200 SRC=&#34;FIGDIR/small/579776v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (48K):
org.highwire.dtl.DTLVardef@1ae3eb9org.highwire.dtl.DTLVardef@980a80org.highwire.dtl.DTLVardef@1d9682aorg.highwire.dtl.DTLVardef@45bf2c_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579143">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579143" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579143">
        <p class="paperTitle">Spatially Exploring RNA Biology in Archival Formalin-Fixed Paraffin-Embedded Tissues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579143" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579143" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bai, Z.; Zhang, D.; Gao, Y.; Tao, B.; Bao, S.; Enninful, A.; Zhang, D.; Su, G.; Tian, X.; Zhang, N.; Xiao, Y.; Liu, Y.; Gerstein, M.; Li, M.; Xing, Y.; Lu, J.; Xu, M. L.; Fan, R.</p>
        <p class="info">Score: 78.9, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579143' target='https://doi.org/10.1101/2024.02.06.579143'> 10.1101/2024.02.06.579143</a></p>
        <p class="abstract">Spatial transcriptomics has emerged as a powerful tool for dissecting spatial cellular heterogeneity but as of today is largely limited to gene expression analysis. Yet, the life of RNA molecules is multifaceted and dynamic, requiring spatial profiling of different RNA species throughout the life cycle to delve into the intricate RNA biology in complex tissues. Human disease-relevant tissues are commonly preserved as formalin-fixed and paraffin-embedded (FFPE) blocks, representing an important resource for human tissue specimens. The capability to spatially explore RNA biology in FFPE tissues holds transformative potential for human biology research and clinical histopathology. Here, we present Patho-DBiT combining in situ polyadenylation and deterministic barcoding for spatial full coverage transcriptome sequencing, tailored for probing the diverse landscape of RNA species even in clinically archived FFPE samples. It permits spatial co-profiling of gene expression and RNA processing, unveiling region-specific splicing isoforms, and high-sensitivity transcriptomic mapping of clinical tumor FFPE tissues stored for five years. Furthermore, genome-wide single nucleotide RNA variants can be captured to distinguish different malignant clones from non-malignant cells in human lymphomas. Patho-DBiT also maps microRNA-mRNA regulatory networks and RNA splicing dynamics, decoding their roles in spatial tumorigenesis trajectory. High resolution Patho-DBiT at the cellular level reveals a spatial neighborhood and traces the spatiotemporal kinetics driving tumor progression. Patho-DBiT stands poised as a valuable platform to unravel rich RNA biology in FFPE tissues to study human tissue biology and aid in clinical pathology evaluation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.08.579519">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.08.579519" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.08.579519">
        <p class="paperTitle">Semi-supervised Bayesian integration of multiple spatial proteomics datasets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.08.579519" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.08.579519" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Coleman, S. D.; Breckels, L.; Waller, R. F.; Lilley, K. S.; Wallace, C.; Kirk, P. D. W.; Crook, O. M.</p>
        <p class="info">Score: 4.2, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.08.579519' target='https://doi.org/10.1101/2024.02.08.579519'> 10.1101/2024.02.08.579519</a></p>
        <p class="abstract">The subcellular localisation of proteins is a key determinant of their function. High-throughput analyses of these localisations can be performed using mass spectrometry-based spatial proteomics, which enables us to examine the localisation and relocalisation of proteins. Furthermore, complementary data sources can provide additional sources of functional or localisation information. Examples include protein annotations and other high-throughput  omic assays. Integrating these modalities can provide new insights as well as additional confidence in results, but existing approaches for integrative analyses of spatial proteomics datasets are limited in the types of data they can integrate and do not quantify uncertainty. Here we propose a semi-supervised Bayesian approach to integrate spatial proteomics datasets with other data sources, to improve the inference of protein sub-cellular localisation. We demonstrate our approach outperforms other transfer learning methods and has greater flexibility in the data it can model. To demonstrate the flexibility of our approach, we apply our method to integrate spatial proteomics data generated for the parasite Toxoplasma gondii with time-course gene expression data generated over its cell cycle. Our findings suggest that proteins linked to invasion organelles are associated with expression programs that peak at the end of the first cell-cycle. Furthermore, this integrative analysis divides the dense granule proteins into heterogeneous populations suggestive of potentially different functions. Our method is disseminated via the mdir R package available on the lead authors Github.

Author summaryProteins are located in subcellular environments to ensure that they are near their interaction partners and occur in the correct biochemical environment to function. Where a protein is located can be determined from a number of data sources. To integrate diverse datasets together we develop an integrative Bayesian model to combine the information from several datasets in a principled manner. We learn how similar the dataset are as part of the modelling process and demonstrate the benefits of integrating mass-spectrometry based spatial proteomics data with timecourse gene-expression datasets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579283">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579283" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579283">
        <p class="paperTitle">Selective enrichment of high-affinity clade II N2O-reducers in a mixed culture</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579283" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579283" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Laureni, M.; Corbera Rubio, F.; Kim, D. D.; Browne, S.; Roothans, N.; Weissbrodt, D. G.; Olavarria, K. O.; de Jonge, N.; Yoon, S.; Pabst, M.; van Loosdrecht, M. C. M.</p>
        <p class="info">Score: 4.1, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579283' target='https://doi.org/10.1101/2024.02.09.579283'> 10.1101/2024.02.09.579283</a></p>
        <p class="abstract">Microorganisms encoding for the N2O reductase (NosZ) are the only known biological sink of the potent greenhouse gas N2O, and are central to global N2O mitigation efforts. Yet, the ecological constraints selecting for different N2O-reducers strains and controlling the assembly of N2O-respiring communities remain largely unknown. Of particular biotechnological interest are clade II NosZ populations, which usually feature high N2O affinities and often lack other denitrification genes. Two planktonic N2O-respiring mixed cultures were enriched under limiting and excess dissolved N2O availability to assess the impact of substrate affinity and N2O cytotoxicity, respectively. Genome-resolved metaproteomics was used to infer the metabolism of the enriched populations. We show that clade II N2O-reducers outcompete clade I affiliates for N2O at sufficiently low sludge dilution rates (0.006 h-1), a scenario previously only theorized based on pure-cultures. Under N2O limitation, all enriched N2O-reducers encoded and expressed only clade II NosZ, while also possessing other denitrification genes. Two Azonexus and Thauera genera affiliates dominated the culture. We explain their coexistence with the genome-inferred metabolic exchange of cobalamin intermediates. Conversely, under excess N2O, clade I and II populations coexisted. Notably, the single dominant N2O-reducer (genus Azonexus) expressed most cobalamin biosynthesis marker genes, likely to contrast the continuous cobalamin inactivation by dissolved cytotoxic N2O concentrations (400 {micro}M). Ultimately, we demonstrate that the solids dilution rate controls the selection among NosZ clades, albeit the conditions selecting for genomes possessing the sole nosZ remain elusive. Additionally, we suggest the significance of N2O-cobalamin interactions in shaping the composition of N2O-respiring microbiomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577354">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577354" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577354">
        <p class="paperTitle">Large-Scale Evaluation of Spatial Metabolomics Protocols and Technologies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577354" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577354" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Saharuka, V.; Vieira, L. M.; Stuart, L.; Ekelöf, M.; Molenaar, M. R.; Bailoni, A.; Ovchinnikova, K.; Soltwisch, J.; Bausbacher, T.; Jakob, D.; King, M.; Müller, M. A.; Oetjen, J.; Pace, C.; Pinto, F. E.; Strittmatter, N.; Velickovic, D.; Spengler, B.; Muddiman, D. C.; Liebeke, M.; Janfelt, C.; Goodwin, R.; Eberlin, L. S.; Anderton, C. R.; Hopf, C.; Dreisewerd, K.; Alexandrov, T.</p>
        <p class="info">Score: 27.0, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577354' target='https://doi.org/10.1101/2024.01.29.577354'> 10.1101/2024.01.29.577354</a></p>
        <p class="abstract">Spatial metabolomics using imaging mass spectrometry (MS) enables untargeted and label-free metabolite mapping in biological samples. Despite the range of available imaging MS protocols and technologies, our understanding of metabolite detection under specific conditions is limited due to sparse empirical data and predictive theories. Consequently, challenges persist in designing new experiments, and accurately annotating and interpreting data. In this study, we systematically measured the detectability of 172 biologically-relevant metabolites across common imaging MS protocols using custom reference samples. We evaluated 24 MALDI-imaging MS protocols for untargeted metabolomics, and demonstrated the applicability of our findings to complex biological samples through comparison with animal tissue data. We showcased the potential for extending our results to further analytes by predicting metabolite detectability based on molecular properties. Additionally, our interlaboratory comparison of 10 imaging MS technologies, including MALDI, DESI, and IR-MALDESI, showed extensive metabolite coverage and comparable results, underscoring the broad applicability of our findings within the imaging MS community. We share our results and data through a new interactive web application integrated with METASPACE. This resource offers an extensive catalogue of detectable metabolite ions, facilitating protocol selection, supporting data annotation, and benefiting future untargeted spatial metabolomics studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.13.580059">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.13.580059" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.13.580059">
        <p class="paperTitle">Chromatin structure and var2csa - a tango in regulation of var gene expression in the human malaria parasite Plasmodium falciparum?</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.13.580059" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.13.580059" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lenz, T.; Zhang, X.; Chakraborty, A.; Roayaei Ardakany, A.; Prudhomme, J.; Ay, F.; Deitsch, K.; Le Roch, K.</p>
        <p class="info">Score: 1.2, Published: 2024-02-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.13.580059' target='https://doi.org/10.1101/2024.02.13.580059'> 10.1101/2024.02.13.580059</a></p>
        <p class="abstract">Over the last few decades, novel methods have been developed to study how chromosome positioning within the nucleus may play a role in gene regulation. Adaptation of these methods in the human malaria parasite, Plasmodium falciparum, has recently led to the discovery that the three-dimensional structure of chromatin within the nucleus may be critical in controlling expression of virulence genes (var genes). Recent work has implicated an unusual, highly conserved var gene called var2csa in contributing to coordinated transcriptional switching, however how this gene functions in this capacity is unknown. To further understand how var2csa influences var gene switching, targeted DNA double-strand breaks (DSBs) within the sub-telomeric region of chromosome 12 were used to delete the gene and the surrounding chromosomal region. To characterize the changes in chromatin architecture stemming from this deletion and how these changes could affect var gene expression, we used a combination of RNA-seq, Chip-seq and Hi-C to pinpoint epigenetic and chromatin structural modifications in regions of differential gene expression. We observed a net gain of interactions in sub-telomeric regions and internal var gene regions following var2csa knockout, indicating an increase of tightly controlled heterochromatin structures. Our results suggest that disruption of var2csa results not only in changes in var gene transcriptional regulation but also a significant tightening of heterochromatin clusters thereby disrupting coordinated activation of var genes throughout the genome. Altogether our result confirms a strong link between the var2csa locus, chromatin structure and var gene expression.

AUTHOR SUMMARYMalaria remains one of the deadliest parasite-borne diseases, causing not only over a half million deaths annually, but also infecting hundreds of millions more. Plasmodium falciparum, the protozoan parasite that is responsible for the most virulent form of human malaria, is transmitted to humans by infected female mosquitoes during a blood meal. Due to a growing resistance to all existing antimalarials, there is a need to identify novel targets to design new antimalarial strategies. Our research builds on the growing body of evidence that supports the role of genome organization or chromatin structure within the nucleus in controlling the parasite development as well as virulence factors designed to circumvent the host immune response. This study identifies genes and structural elements within the Plasmodium falciparum genome that are controlled, at least partially, by the expression of a single unique and highly conserved virulence gene.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.10.579787">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.10.579787" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.10.579787">
        <p class="paperTitle">Proteomic profiling of intracranial atherosclerotic plaque in the human brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.10.579787" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.10.579787" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hao, Q.; Wang, E.; Wang, J.; Wu, Z.; Crary, J. F.; Sharma, S.; Thorn, E. L.; Elahi, F.; Zhang, B.; Peng, J.</p>
        <p class="info">Score: 1.2, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.10.579787' target='https://doi.org/10.1101/2024.02.10.579787'> 10.1101/2024.02.10.579787</a></p>
        <p class="abstract">BackgroundIntracranial atherosclerotic disease (ICAD) is one of the major causes of ischemic stroke and associated with high risk of stroke recurrence. There are no reliable and specific fluid biomarkers for ICAD, and little is known about the proteomic profiling of ICAD. In this study we aimed to explore the feasibility of applying proteomics technology to profile intracranial atherosclerotic plaques extracted from postmortem human brain arteries.

MethodsEighteen segments (5-10mm in length) of major arteries from 10 postmortem brains were collected from the Mount Sinai Neuropathology Brain Bank. Among these segments, 5 had no evidence of atherosclerotic disease, and 13 had wall thickening or visible plaques with various degree of stenosis. Proteins were extracted from the vessel segments, quantified, and digested into peptides. Subsequently, the peptides underwent tandem mass tag (TMT) labeling, pooling, and analysis using two-dimensional liquid chromatography-tandem mass spectrometry (LC/LC-MS/MS). Protein identification and quantification were performed using the JUMP software. Differentially expressed proteins (DEPs) were defined as proteins with p.adj &lt; 0.05 and absolute log2 (fold change) &gt; log2 (1.2).

ResultsA total of 7,492 unique proteins were detected, and 6,726 quantifiable proteins were retained for further analysis. Among these, 265 DEPs, spanning on 252 unique gene, were found to be associated with ICAD by comparing the arterial segments with vs those without atherosclerotic disease. The top 4 most significant DEPs include LONP1, RPS19, MRPL12 and SNU13. Among the top 50 DEPs, FADD, AIFM1 and PGK1 were associated with atherosclerotic disease or cardiovascular events in previous studies. Moreover, the previously reported proteins associated with atherosclerosis such as APCS, MMP12, CTSD were elevated in arterial segments with atherosclerotic changes. Furthermore, the up-regulation of APOE and LPL, the ICAD GWAS risk genes, was shown to be associated with the plaque severity. Finally, gene set enrichment analysis revealed the DEP signature is enriched for biological pathways such as chromatin structure, plasma lipoprotein, nucleosome, and protein-DNA complex, peroxide catabolic and metabolic processes, critical in ICAD pathology.

ConclusionsDirect proteomic profiling of fresh-frozen intracranial artery samples by MS-based proteomic technology is a feasible approach to identify ICAD-associated proteins, which can be potential biomarker candidates for ICAD. Further plaque proteomic study in a larger sample size is warranted to uncover mechanistic insights into ICAD and discover novel biomarkers that may help to improve diagnosis and risk stratification in ICAD.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.07.579358">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.07.579358" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.07.579358">
        <p class="paperTitle">A multiscale sensorimotor model of experience-dependent behavior in a minimal organism</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.07.579358" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.07.579358" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vidal-Saez, M. S.; Vilarroya, O.; Garcia-Ojalvo, J.</p>
        <p class="info">Score: 1.2, Published: 2024-02-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.07.579358' target='https://doi.org/10.1101/2024.02.07.579358'> 10.1101/2024.02.07.579358</a></p>
        <p class="abstract">To survive in ever-changing environments, living organisms need to continuously combine the ongoing external inputs they receive, representing present conditions, with their dynamical internal state, which includes influences of past experiences. It is still unclear in general, however, (i) how this happens at the molecular and cellular levels, and (ii) how the corresponding molecular and cellular processes are integrated with the behavioral responses of the organism. Here we address these issues by modeling mathematically a particular behavioral paradigm in a minimal model organism, namely chemotaxis in the nematode C. elegans. Specifically, we use a long-standing collection of elegant experiments on salt chemotaxis in this animal, in which the migration direction varies depending on its previous experience. Our model integrates the molecular, cellular and organismal levels to reproduce the experimentally observed experience-dependent behavior. The model proposes specific molecular mechanisms for the encoding of current conditions and past experiences in key neurons associated with this response, predicting the behavior of various mutants associated with those molecular circuits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.14.580207">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.14.580207" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.14.580207">
        <p class="paperTitle">Global sensitivity analysis for investigation of bacterial physiology in complex media with FBA-PRCC</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.14.580207" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.14.580207" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sorokin, A.; Goryanin, I.</p>
        <p class="info">Score: 0.8, Published: 2024-02-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.14.580207' target='https://doi.org/10.1101/2024.02.14.580207'> 10.1101/2024.02.14.580207</a></p>
        <p class="abstract">In this paper we propose the use of FBA-PRCC approach, which is based on sampling flux space and global sensitivity analysis (GSA), for analysis of the GEM FBA solution space in the nutrient-rich environment. We identify two new modes of interaction between species and metabolites: attraction and avoidance and propose the way of their use. We have shown that sensitivity coefficients provide additional information about behavior of the bacteria in the nutrient-rich environment in comparison to standard knockouts, FVA and CoPE-FBA analysis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.13.580123">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.13.580123" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.13.580123">
        <p class="paperTitle">Pooled PPIseq: screening the SARS-CoV-2 and human interface with a scalable multiplexed protein-protein interaction assay platform</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.13.580123" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.13.580123" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Miller, D.; Dziulko, A.; Levy, S.</p>
        <p class="info">Score: 0.8, Published: 2024-02-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.13.580123' target='https://doi.org/10.1101/2024.02.13.580123'> 10.1101/2024.02.13.580123</a></p>
        <p class="abstract">Protein-Protein Interactions (PPIs) are a key interface between virus and host, and these interactions are important to both viral reprogramming of the host and to host restriction of viral infection. In particular, viral-host PPI networks can be used to further our understanding of the molecular mechanisms of tissue specificity, host range, and virulence. At higher scales, viral-host PPI screening could also be used to screen for small-molecule antivirals that interfere with essential viral-host interactions, or to explore how the PPI networks between interacting viral and host genomes co-evolve. Current high-throughput PPI assays have screened entire viral-host PPI networks. However, these studies are time consuming, often require specialized equipment, and are difficult to further scale. Here, we develop methods that make larger-scale viral-host PPI screening more accessible. This approach combines the mDHFR split-tag reporter with the iSeq2 interaction-barcoding system to permit massively-multiplexed PPI quantification by simple pooled engineering of barcoded constructs, integration of these constructs into budding yeast, and fitness measurements by pooled cell competitions and barcode-sequencing. We applied this method to screen for PPIs between SARS-CoV-2 proteins and human proteins, screening in triplicate &gt;180,000 ORF-ORF combinations represented by &gt;1,000,000 barcoded lineages. Our results complement previous screens by identifying 74 putative PPIs, including interactions between ORF7A with the taste receptors TAS2R41 and TAS2R7, and between NSP4 with the transmembrane KDELR2 and KDELR3. We show that this PPI screening method is highly scalable, enabling larger studies aimed at generating a broad understanding of how viral effector proteins converge on cellular targets to effect replication.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
