<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="DrugMap: A quantitative pan-cancer analysis of cysteine ligandability
Authors: Takahashi, M.; Chong, H. B.; Zhang, S.; Lazarov, M.; Harry, S.; Maynard, M.; White, R.; Hilbert, B.; Gohar, M.; Ge, M.; Zhang, J.; Durr, B. R.; Kryukov, G.; Tsou, C.-C.; Brooijmans, N.; Alghali, A.; Rubio, K.; Vilanueva, A.; Harrison, D.; Koglin, A.-S.; Ojeda, S.; Karakyriakou, B.; Healy, A.; Assaad, J.; Makram, F.; Rachimin, I.; Khandelwal, N.; Tien, P.-C.; Popoola, G.; Chen, N.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="DrugMap: A quantitative pan-cancer analysis of cysteine ligandability
Authors: Takahashi, M.; Chong, H. B.; Zhang, S.; Lazarov, M.; Harry, S.; Maynard, M.; White, R.; Hilbert, B.; Gohar, M.; Ge, M.; Zhang, J.; Durr, B. R.; Kryukov, G.; Tsou, C.-C.; Brooijmans, N.; Alghali, A.; Rubio, K.; Vilanueva, A.; Harrison, D.; Koglin, A.-S.; Ojeda, S.; Karakyriakou, B.; Healy, A.; Assaad, J.; Makram, F.; Rachimin, I.; Khandelwal, N.; Tien, P.-C.; Popoola, G.; Chen, N." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-01T10:38:40+00:00" />
<meta property="article:modified_time" content="2023-11-01T10:38:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="DrugMap: A quantitative pan-cancer analysis of cysteine ligandability
Authors: Takahashi, M.; Chong, H. B.; Zhang, S.; Lazarov, M.; Harry, S.; Maynard, M.; White, R.; Hilbert, B.; Gohar, M.; Ge, M.; Zhang, J.; Durr, B. R.; Kryukov, G.; Tsou, C.-C.; Brooijmans, N.; Alghali, A.; Rubio, K.; Vilanueva, A.; Harrison, D.; Koglin, A.-S.; Ojeda, S.; Karakyriakou, B.; Healy, A.; Assaad, J.; Makram, F.; Rachimin, I.; Khandelwal, N.; Tien, P.-C.; Popoola, G.; Chen, N."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "DrugMap: A quantitative pan-cancer analysis of cysteine ligandability\nAuthors: Takahashi, M.; Chong, H. B.; Zhang, S.; Lazarov, M.; Harry, S.; Maynard, M.; White, R.; Hilbert, B.; Gohar, M.; Ge, M.; Zhang, J.; Durr, B. R.; Kryukov, G.; Tsou, C.-C.; Brooijmans, N.; Alghali, A.; Rubio, K.; Vilanueva, A.; Harrison, D.; Koglin, A.-S.; Ojeda, S.; Karakyriakou, B.; Healy, A.; Assaad, J.; Makram, F.; Rachimin, I.; Khandelwal, N.; Tien, P.-C.; Popoola, G.; Chen, N.",
  "keywords": [
    
  ],
  "articleBody": " DrugMap: A quantitative pan-cancer analysis of cysteine ligandability\nAuthors: Takahashi, M.; Chong, H. B.; Zhang, S.; Lazarov, M.; Harry, S.; Maynard, M.; White, R.; Hilbert, B.; Gohar, M.; Ge, M.; Zhang, J.; Durr, B. R.; Kryukov, G.; Tsou, C.-C.; Brooijmans, N.; Alghali, A.; Rubio, K.; Vilanueva, A.; Harrison, D.; Koglin, A.-S.; Ojeda, S.; Karakyriakou, B.; Healy, A.; Assaad, J.; Makram, F.; Rachimin, I.; Khandelwal, N.; Tien, P.-C.; Popoola, G.; Chen, N.; Vordermark, K.; Richter, M.; Patel, H.; Yang, T.-y.; Griesshaber, H.; Hosp, T.; van den Ouweland, S.; Hara, T.; Bussema, L.; Shi, L.; Rasmussen, M.; Domingues, A. C.; Lawless, A.; Fang, J.; Yoda, S.; Nguyen, L. P.; R\nScore: 27.9, Published: 2023-10-23 DOI: 10.1101/2023.10.20.563287\nCysteine-focused chemical proteomic platforms have accelerated the clinical development of covalent inhibitors of a wide-range of targets in cancer. However, how different oncogenic contexts influence cysteine targeting remains unknown. To address this question, we have developed DrugMap, an atlas of cysteine ligandability compiled across 416 cancer cell lines. We unexpectedly find that cysteine ligandability varies across cancer cell lines, and we attribute this to differences in cellular redox states, protein conformational changes, and genetic mutations. Leveraging these findings, we identify actionable cysteines in NF{kappa}B1 and SOX10 and develop corresponding covalent ligands that block the activity of these transcription factors. We demonstrate that the NF{kappa}B1 probe blocks DNA binding, whereas the SOX10 ligand increases SOX10-SOX10 interactions and disrupts melanoma transcriptional signaling. Our findings reveal heterogeneity in cysteine ligandability across cancers, pinpoint cell-intrinsic features driving cysteine targeting, and illustrate the use of covalent probes to disrupt oncogenic transcription factor activity.\nPhysiMeSS - A New PhysiCell Addon for Extracellular Matrix Modelling\nAuthors: Noel, V.; Ruscone, M.; Shuttleworth, R.; MacNamara, C. K.\nScore: 10.3, Published: 2023-10-27 DOI: 10.1101/2023.10.27.564365\nThe extracellular matrix is a complex assembly of macro-molecules, such as collagen fibres, which provides structural support for surrounding cells. In the context of cancer metastasis, it represents a barrier for the cells, that the migrating cells needs to degrade in order to leave the primary tumor and invade further tissues. Agent-based frameworks, such as PhysiCell, are often use to represent the spatial dynamics of tumor evolution. However, typically they only implement cells as agents, which are represented by either a circle (2D) or a sphere (3D). In order to accurately represent the extracellular matrix as a network of fibres, we require a new type of agent represented by a segment (2D) or a cylinder (3D). In this article, we present PhysiMeSS, an addon of PhysiCell, which introduces a new type of agent to describe fibres, and their physical interactions with cells and other fibres. PhysiMeSS implementation is publicly available at https://github.com/PhysiMeSS/PhysiMeSS, as well as in the official Physi-Cell repository. We also provide simple examples to describe the extended possibilities of this new framework. We hope that this tool will serve to tackle important biological questions such as diseases linked to dis-regulation of the extracellular matrix, or the processes leading to cancer metastasis.\nDevelopment of a genome scale metabolic model for the lager hybrid yeast S. pastorianus to understand evolution of metabolic pathways in industrial settings.\nAuthors: Timouma, S.; Balarezo-Cisneros, L. N.; Schwartz, J.-M.; Delneri, D.\nScore: 8.2, Published: 2023-10-28 DOI: 10.1101/2023.10.25.564032\nIn silico tools such as genome-scale metabolic models (GSMM) have shown to be powerful for metabolic engineering of microorganisms. Here, we created the iSP_1513 GSMM for the aneuploid hybrid S. pastorianus CBS1513 to allow top-down computational approaches to predict the evolution of metabolic pathways and to aid strain optimisation and media engineering in production processes. The iSP_1513 comprises 4062 reactions, 1808 alleles and 2747 metabolites, and takes into account the functional redundancy in the gene-protein-reaction rule caused by the presence of orthologous genes. Moreover, a universal algorithm to constrain GSMM reactions using transcriptome data was developed as a python library and enabled the integration of temperature as parameter. Essentiality datasets, growth data on various carbohydrates and volatile metabolites secretion were used to validate the model. Overall, the iSP_1513 GSMM represent an important step towards understanding the metabolic capabilities, evolutionary trajectories and adaptation potential of S. pastorianus in different industrial settings.\nLongitudinal single-cell data informs deterministic modelling of inflammatory bowel disease\nAuthors: Kilian, C.; Ulrich, H.; Zouboulis, V.; Sprezyna, P.; Schreiber, J.; Landsberger, T.; Büttner, M.; Biton, M.; Villablanca, E. J.; Huber, S.; Adlung, L.\nScore: 4.0, Published: 2023-10-30 DOI: 10.1101/2023.10.27.561846\nSingle-cell mRNA sequencing (scRNA-seq) allows deep molecular and cellular profiling of immunological processes. Longitudinal scRNA-seq datasets can be used for deterministic ordinary differential equation (ODE)-based modelling to mechanistically describe immune dynamics. Here, we derived changes in colonic cell type abundance during inflammatory bowel disease (IBD) from published longitudinal scRNA-seq data of a mouse model of colitis. We trained different ODE models, performed model selection and obtained a set of fully identifiable parameters describing experimental data from six different cell types. We then predicted the immune dynamics of a different mouse colitis protocol and confirmed these scRNA-seq-based predictions with our previously published single-cell-based flow cytometry data. We further hypothesised that the estimated model parameters reflect biological processes. We validated this prediction of cellular turnover rates with KI-67 staining and with gene expression information from the scRNA-seq data not used for model training. Finally, we tested the translational relevance of the model simulations by predicting genes indicative of treatment response in human IBD patients. The predictive power of IBD deterministic modelling from scRNA-seq data highlights its potential to advance our understanding of immune dynamics in health and disease.\nCoupled ribosome allocation and nutrient sensing lead to incoherent feedback and oscillatory response in bacterial growth.\nAuthors: Droghetti, R.; Firmano, V.; Tallarico, G.; Calabrese, L.; Grilli, J.; Cosentino Lagomarsino, M.\nScore: 3.4, Published: 2023-10-29 DOI: 10.1101/2023.10.25.563923\nCurrent theories describing bacterial growth physiology across environments have demonstrated an impressive predictive power, but they are typically phenomenological. Incorporating mechanistic details into these frameworks remains an open challenge that would greatly improve our ability to predict and control bacterial growth in varying environmental conditions. For example, the ``Flux Controlled Regulation'' (FCR) model is a reference out-of-equilibrium framework that links ribosome allocation to translation efficiency by means of a steady-state assumption. By making use of this assumption, this model does not account for ppGpp-mediated nutrient sensing and transcriptional regulation of ribosomal operons. In this study, we propose a simple model that integrates the FCR framework with a mechanistic description of three key components: (i) the amino-acid pool, (ii) ppGpp sensing of translation elongation rate, and (iii) transcriptional programming of protein allocation strategy by ppGpp-sensitive promoters. Our framework is fully coherent with observed steady-state growth laws and makes testable predictions for unobserved quantities. Furthermore, our theory predicts that under environmental changes the incoherent feedback between sensing and regulation leads to oscillatory relaxation towards new equilibria, a feature observed experimentally but not captured by previous phenomenological models.\nGlobal transcription regulation revealed from dynamical correlations in time-resolved single-cell RNA-sequencing\nAuthors: Volteras, D.; Shahrezaei, V.; Thomas, P.\nScore: 3.4, Published: 2023-10-27 DOI: 10.1101/2023.10.24.563709\nSingle-cell transcriptomics reveals significant variations in the transcriptional activity across cells. Yet, it remains challenging to identify mechanisms of transcription dynamics from static snapshots. It is thus still unknown what drives global transcription dynamics in single cells. We present a stochastic model of gene expression in growing and dividing cells that harnesses temporal dimensions of single-cell RNA-sequencing through metabolic labelling protocols and cell cycle reporters. We develop a parallel and highly scalable Approximate Bayesian Computation method that corrects for technical variation and accurately quantifies absolute burst frequency, burst size and degradation rate at a transcriptome-wide scale. Using Bayesian model selection, we reveal scaling between transcription rates and cell size and unveil waves of gene regulation across the cell cycle-dependent transcriptome. Our study shows that stochastic modelling of dynamical correlations identifies global mechanisms of transcription regulation. We illustrate how mechanistic modelling and amortised inference can efficiently integrate distributions across time, modalities and conditions. Our framework thus provides a versatile tool to uncover how global transcriptional dynamics is orchestrated genome-wide in single cells.\nIntegrative systems biology framework discovers common gene regulatory signatures in multiple mechanistically distinct inflammatory skin diseases\nAuthors: Mishra, B.; Athar, M.; Mukhtar, M. S.\nScore: 2.2, Published: 2023-10-28 DOI: 10.1101/2023.10.24.563852\nAim: More than 20% of the population across the world is affected by non-communicable inflammatory skin diseases including psoriasis, atopic dermatitis, hidradenitis suppurativa, rosacea, etc. Many of these chronic diseases are painful and debilitating with limited effective therapeutic interventions. However, recent advances in psoriasis treatment have improved the effectiveness and provide better management of the disease. Similar effective treatments are also needed for other chronic skin disorders regulated molecular pathogenesis. This study aims to identify common regulatory pathways and master regulators that regulate molecular pathogenesis. Methods: We designed an integrative systems biology framework to identify the significant regulators across several inflammatory skin diseases. We exploited the gene co-expression-, and protein-protein interaction-based networks to identify shared genes and pro components in different diseases with relevant functional implications. Further, we utilized network analytics to unravel the high-priority proteins, that play a crucial role in disease pathogenesis. Results: With conventional transcriptome analysis we identified 55 shared genes, which are enriched in several immune-associated pathways in eight inflammatory skin diseases. Our correlation-based co-expression networks identified the core co-expression networks and the integrative multi-omics interactomes providing core interactions across four of the eight diseases. Next, by implementing network analytics, we unraveled 55 high-value proteins as significant regulators in molecular pathogenesis. We believe that these significant regulators should be explored with critical experimental approaches to identify the putative drug targets for more effective treatments. As an example, we identified IKZF1 as a shared significant master regulator in three inflammatory skin diseases, which can serve as a putative drug target with known disease-derived molecules for developing efficacious combinatorial treatments for hidradenitis suppurativa, atopic dermatitis, and rosacea. Conclusion: The conventional transcriptome results identify huge complexity among inflammatory skin diseases, which demands a systems biology perspective for comprehensive understanding. The proposed framework is very modular, which can indicate a significant path of molecular mechanism-based drug development from complex transcriptomics data and other multi-omics data.\nPulsed stimuli entrain p53 to synchronize single cells and modulate cell-fate determination\nAuthors: Venkatachalapathy, H.; Yang, Z.; Azarin, S. M.; Sarkar, C. A.; Batchelor, E.\nScore: 2.1, Published: 2023-10-27 DOI: 10.1101/2023.10.24.563786\nThe dynamic expression pattern of the tumor suppressor p53 can have significant cell-to-cell heterogeneity, leading to variability in cellular stress responses and resulting in clinical consequences such as cancer chemoresistance. Using a combination of mathematical modeling, time-lapse fluorescence microscopy, and single-cell tracking, we exploit the oscillatory nature of the p53 response to DNA double-strand break (DSB) induction to reduce cellular heterogeneity by entraining p53 dynamics. Our study demonstrates that oscillatory p53 dynamics can be experimentally entrained over a wider range of DSB frequencies than predicted by an established computational model for this system. Non-intuitively, we determined that recapitulating this increased entrainment range required a modified model with a less robust oscillator and wider steady-state valley on the energy landscape. Further, we show that p53 entrainment can lead to altered expression dynamics of downstream targets responsible for cell fate in a manner dependent on target mRNA stability. Overall, this study exemplifies the potential of externally entraining p53 dynamics to reduce cellular variability and synchronize cell-fate responses to genotoxic agents.\nA robust ultrasensitive transcriptional switch in noisy cellular environments\nAuthors: Jeong, E. M.; Kim, J. K.\nScore: 11.3, Published: 2023-10-18 DOI: 10.1101/2023.08.15.553401\nUltrasensitive transcriptional switches enable sharp transitions between on and off states of transcription and are essential for cells to respond to environmental cues precisely. However, conventional switches, relying on direct repressor-DNA binding, are extremely sensitive to noise. Here, we discovered an alternative design combining three indirect transcriptional repression mechanisms, sequestration, blocking, and displacement, to generate a noise-resilient ultrasensitive switch. Although sequestration alone can generate an ultrasensitive switch, it remains sensitive to noise because the unintended transcriptional state induced by noise can persist for long periods. However, by jointly utilizing blocking and displacement, these noise-induced transitions can be rapidly restored to the original transcriptional state. Because this transcriptional switch is effective in noisy cellular contexts, it goes beyond previous synthetic transcriptional switches utilizing direct repression mechanisms, making it particularly valuable for robust synthetic system design. Our findings also provide insights into the evolution of robust ultrasensitive switches in real cells. Specifically, the concurrent use of seemingly redundant indirect repression mechanisms in diverse biological systems appears to be a strategy to achieve noise-resilience of ultrasensitive switches. GRAPHICAL ABSTRACT O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=175 SRC=\"FIGDIR/small/553401v2_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (28K): org.highwire.dtl.DTLVardef@e69a5aorg.highwire.dtl.DTLVardef@19faf13org.highwire.dtl.DTLVardef@9f3035org.highwire.dtl.DTLVardef@115ff7c_HPS_FORMAT_FIGEXP M_FIG C_FIG\nLogic-based mechanistic machine learning on high-content images reveals how drugs differentially regulate cardiac fibroblasts\nAuthors: Nelson, A. R.; Christiansen, S. L.; Naegle, K. M.; Saucerman, J. J.\nScore: 11.7, Published: 2023-10-23 DOI: 10.1101/2023.03.01.530599\nFibroblasts are essential regulators of extracellular matrix deposition following cardiac injury. These cells exhibit highly plastic responses in phenotype during fibrosis in response to environmental stimuli. Here, we test whether and how candidate anti-fibrotic drugs differentially regulate measures of cardiac fibroblast phenotype, which may help identify treatments for cardiac fibrosis. We conducted a high content microscopy screen of human cardiac fibroblasts treated with 13 clinically relevant drugs in the context of TGF{beta} and/or IL-1{beta}, measuring phenotype across 137 single-cell features. We used the phenotypic data from our high content imaging to train a logic-based mechanistic machine learning model (LogiMML) for fibroblast signaling. The model predicted how pirfenidone and Src inhibitor WH-4-023 reduce actin filament assembly and actin-myosin stress fiber formation, respectively. Validating the LogiMML model prediction that PI3K partially mediates the effects of Src inhibition, we found that PI3K inhibition reduces actin-myosin stress fiber formation and procollagen I production in human cardiac fibroblasts. In this study, we establish a modeling approach combining the strengths of logic-based network models and regularized regression models, apply this approach to predict mechanisms that mediate the differential effects of drugs on fibroblasts, revealing Src inhibition acting via PI3K as a potential therapy for cardiac fibrosis. SignificanceCardiac fibrosis is a dysregulation of the normal wound healing response, resulting in excessive scarring and cardiac dysfunction. As cardiac fibroblasts primarily regulate this process, we explored how candidate anti-fibrotic drugs alter the fibroblast phenotype. We identify a set of 137 phenotypic features that change in response to drug treatments. Using a new computational modeling approach termed logic-based mechanistic machine learning, we predict how pirfenidone and Src inhibition affect the regulation of the phenotypic features actin filament assembly and actin-myosin stress fiber formation. We also show that inhibition of PI3K reduces actin-myosin stress fiber formation and procollagen I production in human cardiac fibroblasts, supporting a role for PI3K as a mechanism by which Src inhibition may suppress fibrosis.\n",
  "wordCount" : "2429",
  "inLanguage": "en",
  "datePublished": "2023-11-01T10:38:40Z",
  "dateModified": "2023-11-01T10:38:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 1, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.563287">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.563287" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.563287">
        <p class="paperTitle">DrugMap: A quantitative pan-cancer analysis of cysteine ligandability</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.563287" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.563287" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Takahashi, M.; Chong, H. B.; Zhang, S.; Lazarov, M.; Harry, S.; Maynard, M.; White, R.; Hilbert, B.; Gohar, M.; Ge, M.; Zhang, J.; Durr, B. R.; Kryukov, G.; Tsou, C.-C.; Brooijmans, N.; Alghali, A.; Rubio, K.; Vilanueva, A.; Harrison, D.; Koglin, A.-S.; Ojeda, S.; Karakyriakou, B.; Healy, A.; Assaad, J.; Makram, F.; Rachimin, I.; Khandelwal, N.; Tien, P.-C.; Popoola, G.; Chen, N.; Vordermark, K.; Richter, M.; Patel, H.; Yang, T.-y.; Griesshaber, H.; Hosp, T.; van den Ouweland, S.; Hara, T.; Bussema, L.; Shi, L.; Rasmussen, M.; Domingues, A. C.; Lawless, A.; Fang, J.; Yoda, S.; Nguyen, L. P.; R</p>
        <p class="info">Score: 27.9, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.563287' target='https://doi.org/10.1101/2023.10.20.563287'> 10.1101/2023.10.20.563287</a></p>
        <p class="abstract">Cysteine-focused chemical proteomic platforms have accelerated the clinical development of covalent inhibitors of a wide-range of targets in cancer. However, how different oncogenic contexts influence cysteine targeting remains unknown. To address this question, we have developed DrugMap, an atlas of cysteine ligandability compiled across 416 cancer cell lines. We unexpectedly find that cysteine ligandability varies across cancer cell lines, and we attribute this to differences in cellular redox states, protein conformational changes, and genetic mutations. Leveraging these findings, we identify actionable cysteines in NF{kappa}B1 and SOX10 and develop corresponding covalent ligands that block the activity of these transcription factors. We demonstrate that the NF{kappa}B1 probe blocks DNA binding, whereas the SOX10 ligand increases SOX10-SOX10 interactions and disrupts melanoma transcriptional signaling. Our findings reveal heterogeneity in cysteine ligandability across cancers, pinpoint cell-intrinsic features driving cysteine targeting, and illustrate the use of covalent probes to disrupt oncogenic transcription factor activity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564365">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564365" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564365">
        <p class="paperTitle">PhysiMeSS - A New PhysiCell Addon for Extracellular Matrix Modelling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564365" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564365" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Noel, V.; Ruscone, M.; Shuttleworth, R.; MacNamara, C. K.</p>
        <p class="info">Score: 10.3, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564365' target='https://doi.org/10.1101/2023.10.27.564365'> 10.1101/2023.10.27.564365</a></p>
        <p class="abstract">The extracellular matrix is a complex assembly of macro-molecules, such as collagen fibres, which provides structural support for surrounding cells. In the context of cancer metastasis, it represents a barrier for the cells, that the migrating cells needs to degrade in order to leave the primary tumor and invade further tissues. Agent-based frameworks, such as PhysiCell, are often use to represent the spatial dynamics of tumor evolution. However, typically they only implement cells as agents, which are represented by either a circle (2D) or a sphere (3D). In order to accurately represent the extracellular matrix as a network of fibres, we require a new type of agent represented by a segment (2D) or a cylinder (3D).

In this article, we present PhysiMeSS, an addon of PhysiCell, which introduces a new type of agent to describe fibres, and their physical interactions with cells and other fibres. PhysiMeSS implementation is publicly available at https://github.com/PhysiMeSS/PhysiMeSS, as well as in the official Physi-Cell repository. We also provide simple examples to describe the extended possibilities of this new framework. We hope that this tool will serve to tackle important biological questions such as diseases linked to dis-regulation of the extracellular matrix, or the processes leading to cancer metastasis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.564032">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.564032" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.564032">
        <p class="paperTitle">Development of a genome scale metabolic model for the lager hybrid yeast S. pastorianus to understand evolution of metabolic pathways in industrial settings.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.564032" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.564032" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Timouma, S.; Balarezo-Cisneros, L. N.; Schwartz, J.-M.; Delneri, D.</p>
        <p class="info">Score: 8.2, Published: 2023-10-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.564032' target='https://doi.org/10.1101/2023.10.25.564032'> 10.1101/2023.10.25.564032</a></p>
        <p class="abstract">In silico tools such as genome-scale metabolic models (GSMM) have shown to be powerful for metabolic engineering of microorganisms. Here, we created the iSP_1513 GSMM for the aneuploid hybrid S. pastorianus CBS1513 to allow top-down computational approaches to predict the evolution of metabolic pathways and to aid strain optimisation and media engineering in production processes. The iSP_1513 comprises 4062 reactions, 1808 alleles and 2747 metabolites, and takes into account the functional redundancy in the gene-protein-reaction rule caused by the presence of orthologous genes. Moreover, a universal algorithm to constrain GSMM reactions using transcriptome data was developed as a python library and enabled the integration of temperature as parameter. Essentiality datasets, growth data on various carbohydrates and volatile metabolites secretion were used to validate the model. Overall, the iSP_1513 GSMM represent an important step towards understanding the metabolic capabilities, evolutionary trajectories and adaptation potential of S. pastorianus in different industrial settings.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.561846">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.561846" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.561846">
        <p class="paperTitle">Longitudinal single-cell data informs deterministic modelling of inflammatory bowel disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.561846" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.561846" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kilian, C.; Ulrich, H.; Zouboulis, V.; Sprezyna, P.; Schreiber, J.; Landsberger, T.; Büttner, M.; Biton, M.; Villablanca, E. J.; Huber, S.; Adlung, L.</p>
        <p class="info">Score: 4.0, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.561846' target='https://doi.org/10.1101/2023.10.27.561846'> 10.1101/2023.10.27.561846</a></p>
        <p class="abstract">Single-cell mRNA sequencing (scRNA-seq) allows deep molecular and cellular profiling of immunological processes. Longitudinal scRNA-seq datasets can be used for deterministic ordinary differential equation (ODE)-based modelling to mechanistically describe immune dynamics. Here, we derived changes in colonic cell type abundance during inflammatory bowel disease (IBD) from published longitudinal scRNA-seq data of a mouse model of colitis. We trained different ODE models, performed model selection and obtained a set of fully identifiable parameters describing experimental data from six different cell types. We then predicted the immune dynamics of a different mouse colitis protocol and confirmed these scRNA-seq-based predictions with our previously published single-cell-based flow cytometry data. We further hypothesised that the estimated model parameters reflect biological processes. We validated this prediction of cellular turnover rates with KI-67 staining and with gene expression information from the scRNA-seq data not used for model training. Finally, we tested the translational relevance of the model simulations by predicting genes indicative of treatment response in human IBD patients. The predictive power of IBD deterministic modelling from scRNA-seq data highlights its potential to advance our understanding of immune dynamics in health and disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.563923">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.563923" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.563923">
        <p class="paperTitle">Coupled ribosome allocation and nutrient sensing lead to incoherent feedback and oscillatory response in bacterial growth.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.563923" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.563923" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Droghetti, R.; Firmano, V.; Tallarico, G.; Calabrese, L.; Grilli, J.; Cosentino Lagomarsino, M.</p>
        <p class="info">Score: 3.4, Published: 2023-10-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.563923' target='https://doi.org/10.1101/2023.10.25.563923'> 10.1101/2023.10.25.563923</a></p>
        <p class="abstract">Current theories describing bacterial growth physiology across environments have demonstrated an impressive predictive power, but they are typically phenomenological. Incorporating mechanistic details into these frameworks remains an open challenge that would greatly improve our ability to predict and control bacterial growth in varying environmental conditions. For example, the ``Flux Controlled Regulation&#39;&#39; (FCR) model is a reference out-of-equilibrium framework that links ribosome allocation to translation efficiency by means of a steady-state assumption. By making use of this assumption, this model does not account for ppGpp-mediated nutrient sensing and transcriptional regulation of ribosomal operons. In this study, we propose a simple model that integrates the FCR framework with a mechanistic description of three key components: (i) the amino-acid pool, (ii) ppGpp sensing of translation elongation rate, and (iii) transcriptional programming of protein allocation strategy by ppGpp-sensitive promoters. Our framework is fully coherent with observed steady-state growth laws and makes testable predictions for unobserved quantities. Furthermore, our theory predicts that under environmental changes the incoherent feedback between sensing and regulation leads to oscillatory relaxation towards new equilibria, a feature observed experimentally but not captured by previous phenomenological models.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563709">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563709" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563709">
        <p class="paperTitle">Global transcription regulation revealed from dynamical correlations in time-resolved single-cell RNA-sequencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563709" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563709" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Volteras, D.; Shahrezaei, V.; Thomas, P.</p>
        <p class="info">Score: 3.4, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563709' target='https://doi.org/10.1101/2023.10.24.563709'> 10.1101/2023.10.24.563709</a></p>
        <p class="abstract">Single-cell transcriptomics reveals significant variations in the transcriptional activity across cells. Yet, it remains challenging to identify mechanisms of transcription dynamics from static snapshots. It is thus still unknown what drives global transcription dynamics in single cells. We present a stochastic model of gene expression in growing and dividing cells that harnesses temporal dimensions of single-cell RNA-sequencing through metabolic labelling protocols and cell cycle reporters. We develop a parallel and highly scalable Approximate Bayesian Computation method that corrects for technical variation and accurately quantifies absolute burst frequency, burst size and degradation rate at a transcriptome-wide scale. Using Bayesian model selection, we reveal scaling between transcription rates and cell size and unveil waves of gene regulation across the cell cycle-dependent transcriptome. Our study shows that stochastic modelling of dynamical correlations identifies global mechanisms of transcription regulation. We illustrate how mechanistic modelling and amortised inference can efficiently integrate distributions across time, modalities and conditions. Our framework thus provides a versatile tool to uncover how global transcriptional dynamics is orchestrated genome-wide in single cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563852">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563852" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563852">
        <p class="paperTitle">Integrative systems biology framework discovers common gene regulatory signatures in multiple mechanistically distinct inflammatory skin diseases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563852" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563852" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mishra, B.; Athar, M.; Mukhtar, M. S.</p>
        <p class="info">Score: 2.2, Published: 2023-10-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563852' target='https://doi.org/10.1101/2023.10.24.563852'> 10.1101/2023.10.24.563852</a></p>
        <p class="abstract">Aim: More than 20% of the population across the world is affected by non-communicable inflammatory skin diseases including psoriasis, atopic dermatitis, hidradenitis suppurativa, rosacea, etc. Many of these chronic diseases are painful and debilitating with limited effective therapeutic interventions. However, recent advances in psoriasis treatment have improved the effectiveness and provide better management of the disease. Similar effective treatments are also needed for other chronic skin disorders regulated molecular pathogenesis. This study aims to identify common regulatory pathways and master regulators that regulate molecular pathogenesis. Methods: We designed an integrative systems biology framework to identify the significant regulators across several inflammatory skin diseases. We exploited the gene co-expression-, and protein-protein interaction-based networks to identify shared genes and pro components in different diseases with relevant functional implications. Further, we utilized network analytics to unravel the high-priority proteins, that play a crucial role in disease pathogenesis. Results: With conventional transcriptome analysis we identified 55 shared genes, which are enriched in several immune-associated pathways in eight inflammatory skin diseases. Our correlation-based co-expression networks identified the core co-expression networks and the integrative multi-omics interactomes providing core interactions across four of the eight diseases. Next, by implementing network analytics, we unraveled 55 high-value proteins as significant regulators in molecular pathogenesis. We believe that these significant regulators should be explored with critical experimental approaches to identify the putative drug targets for more effective treatments. As an example, we identified IKZF1 as a shared significant master regulator in three inflammatory skin diseases, which can serve as a putative drug target with known disease-derived molecules for developing efficacious combinatorial treatments for hidradenitis suppurativa, atopic dermatitis, and rosacea. Conclusion: The conventional transcriptome results identify huge complexity among inflammatory skin diseases, which demands a systems biology perspective for comprehensive understanding. The proposed framework is very modular, which can indicate a significant path of molecular mechanism-based drug development from complex transcriptomics data and other multi-omics data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563786">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563786" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563786">
        <p class="paperTitle">Pulsed stimuli entrain p53 to synchronize single cells and modulate cell-fate determination</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563786" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563786" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Venkatachalapathy, H.; Yang, Z.; Azarin, S. M.; Sarkar, C. A.; Batchelor, E.</p>
        <p class="info">Score: 2.1, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563786' target='https://doi.org/10.1101/2023.10.24.563786'> 10.1101/2023.10.24.563786</a></p>
        <p class="abstract">The dynamic expression pattern of the tumor suppressor p53 can have significant cell-to-cell heterogeneity, leading to variability in cellular stress responses and resulting in clinical consequences such as cancer chemoresistance. Using a combination of mathematical modeling, time-lapse fluorescence microscopy, and single-cell tracking, we exploit the oscillatory nature of the p53 response to DNA double-strand break (DSB) induction to reduce cellular heterogeneity by entraining p53 dynamics. Our study demonstrates that oscillatory p53 dynamics can be experimentally entrained over a wider range of DSB frequencies than predicted by an established computational model for this system. Non-intuitively, we determined that recapitulating this increased entrainment range required a modified model with a less robust oscillator and wider steady-state valley on the energy landscape. Further, we show that p53 entrainment can lead to altered expression dynamics of downstream targets responsible for cell fate in a manner dependent on target mRNA stability. Overall, this study exemplifies the potential of externally entraining p53 dynamics to reduce cellular variability and synchronize cell-fate responses to genotoxic agents.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.15.553401">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.15.553401" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.15.553401">
        <p class="paperTitle">A robust ultrasensitive transcriptional switch in noisy cellular environments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.15.553401" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.15.553401" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jeong, E. M.; Kim, J. K.</p>
        <p class="info">Score: 11.3, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.15.553401' target='https://doi.org/10.1101/2023.08.15.553401'> 10.1101/2023.08.15.553401</a></p>
        <p class="abstract">Ultrasensitive transcriptional switches enable sharp transitions between on and off states of transcription and are essential for cells to respond to environmental cues precisely. However, conventional switches, relying on direct repressor-DNA binding, are extremely sensitive to noise. Here, we discovered an alternative design combining three indirect transcriptional repression mechanisms, sequestration, blocking, and displacement, to generate a noise-resilient ultrasensitive switch. Although sequestration alone can generate an ultrasensitive switch, it remains sensitive to noise because the unintended transcriptional state induced by noise can persist for long periods. However, by jointly utilizing blocking and displacement, these noise-induced transitions can be rapidly restored to the original transcriptional state. Because this transcriptional switch is effective in noisy cellular contexts, it goes beyond previous synthetic transcriptional switches utilizing direct repression mechanisms, making it particularly valuable for robust synthetic system design. Our findings also provide insights into the evolution of robust ultrasensitive switches in real cells. Specifically, the concurrent use of seemingly redundant indirect repression mechanisms in diverse biological systems appears to be a strategy to achieve noise-resilience of ultrasensitive switches.

GRAPHICAL ABSTRACT

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=175 SRC=&#34;FIGDIR/small/553401v2_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (28K):
org.highwire.dtl.DTLVardef@e69a5aorg.highwire.dtl.DTLVardef@19faf13org.highwire.dtl.DTLVardef@9f3035org.highwire.dtl.DTLVardef@115ff7c_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.03.01.530599">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.03.01.530599" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.03.01.530599">
        <p class="paperTitle">Logic-based mechanistic machine learning on high-content images reveals how drugs differentially regulate cardiac fibroblasts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.03.01.530599" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.03.01.530599" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nelson, A. R.; Christiansen, S. L.; Naegle, K. M.; Saucerman, J. J.</p>
        <p class="info">Score: 11.7, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.03.01.530599' target='https://doi.org/10.1101/2023.03.01.530599'> 10.1101/2023.03.01.530599</a></p>
        <p class="abstract">Fibroblasts are essential regulators of extracellular matrix deposition following cardiac injury. These cells exhibit highly plastic responses in phenotype during fibrosis in response to environmental stimuli. Here, we test whether and how candidate anti-fibrotic drugs differentially regulate measures of cardiac fibroblast phenotype, which may help identify treatments for cardiac fibrosis. We conducted a high content microscopy screen of human cardiac fibroblasts treated with 13 clinically relevant drugs in the context of TGF{beta} and/or IL-1{beta}, measuring phenotype across 137 single-cell features. We used the phenotypic data from our high content imaging to train a logic-based mechanistic machine learning model (LogiMML) for fibroblast signaling. The model predicted how pirfenidone and Src inhibitor WH-4-023 reduce actin filament assembly and actin-myosin stress fiber formation, respectively. Validating the LogiMML model prediction that PI3K partially mediates the effects of Src inhibition, we found that PI3K inhibition reduces actin-myosin stress fiber formation and procollagen I production in human cardiac fibroblasts. In this study, we establish a modeling approach combining the strengths of logic-based network models and regularized regression models, apply this approach to predict mechanisms that mediate the differential effects of drugs on fibroblasts, revealing Src inhibition acting via PI3K as a potential therapy for cardiac fibrosis.

SignificanceCardiac fibrosis is a dysregulation of the normal wound healing response, resulting in excessive scarring and cardiac dysfunction. As cardiac fibroblasts primarily regulate this process, we explored how candidate anti-fibrotic drugs alter the fibroblast phenotype. We identify a set of 137 phenotypic features that change in response to drug treatments. Using a new computational modeling approach termed logic-based mechanistic machine learning, we predict how pirfenidone and Src inhibition affect the regulation of the phenotypic features actin filament assembly and actin-myosin stress fiber formation. We also show that inhibition of PI3K reduces actin-myosin stress fiber formation and procollagen I production in human cardiac fibroblasts, supporting a role for PI3K as a mechanism by which Src inhibition may suppress fibrosis.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
