<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Statistical inference with a manifold-constrained RNA velocity model uncovers cell cycle speed modulations
Authors: Lederer, A. R.; Leonardi, M.; Talamanca, L.; Herrera, A.; Droin, C.; Khven, I.; Carvalho, H. J.; Valente, A.; Dominguez Mantes, A.; Mulet Arabi, P.; Pinello, L.; Naef, F.; La Manno, G.
Score: 40.1, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576093
Across a range of biological processes, cells undergo coordinated changes in gene expression, resulting in transcriptome dynamics that unfold within a low-dimensional manifold.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Statistical inference with a manifold-constrained RNA velocity model uncovers cell cycle speed modulations
Authors: Lederer, A. R.; Leonardi, M.; Talamanca, L.; Herrera, A.; Droin, C.; Khven, I.; Carvalho, H. J.; Valente, A.; Dominguez Mantes, A.; Mulet Arabi, P.; Pinello, L.; Naef, F.; La Manno, G.
Score: 40.1, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576093
Across a range of biological processes, cells undergo coordinated changes in gene expression, resulting in transcriptome dynamics that unfold within a low-dimensional manifold." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-28T10:37:59+00:00" />
<meta property="article:modified_time" content="2024-01-28T10:37:59+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Statistical inference with a manifold-constrained RNA velocity model uncovers cell cycle speed modulations
Authors: Lederer, A. R.; Leonardi, M.; Talamanca, L.; Herrera, A.; Droin, C.; Khven, I.; Carvalho, H. J.; Valente, A.; Dominguez Mantes, A.; Mulet Arabi, P.; Pinello, L.; Naef, F.; La Manno, G.
Score: 40.1, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576093
Across a range of biological processes, cells undergo coordinated changes in gene expression, resulting in transcriptome dynamics that unfold within a low-dimensional manifold."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Statistical inference with a manifold-constrained RNA velocity model uncovers cell cycle speed modulations\nAuthors: Lederer, A. R.; Leonardi, M.; Talamanca, L.; Herrera, A.; Droin, C.; Khven, I.; Carvalho, H. J.; Valente, A.; Dominguez Mantes, A.; Mulet Arabi, P.; Pinello, L.; Naef, F.; La Manno, G.\nScore: 40.1, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576093\nAcross a range of biological processes, cells undergo coordinated changes in gene expression, resulting in transcriptome dynamics that unfold within a low-dimensional manifold.",
  "keywords": [
    
  ],
  "articleBody": " Statistical inference with a manifold-constrained RNA velocity model uncovers cell cycle speed modulations\nAuthors: Lederer, A. R.; Leonardi, M.; Talamanca, L.; Herrera, A.; Droin, C.; Khven, I.; Carvalho, H. J.; Valente, A.; Dominguez Mantes, A.; Mulet Arabi, P.; Pinello, L.; Naef, F.; La Manno, G.\nScore: 40.1, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576093\nAcross a range of biological processes, cells undergo coordinated changes in gene expression, resulting in transcriptome dynamics that unfold within a low-dimensional manifold. Single-cell RNA-sequencing (scRNA-seq) only measures temporal snapshots of gene expression. However, information on the underlying low-dimensional dynamics can be extracted using RNA velocity, which models unspliced and spliced RNA abundances to estimate the rate of change of gene expression. Available RNA velocity algorithms can be fragile and rely on heuristics that lack statistical control. Moreover, the estimated vector field is not dynamically consistent with the traversed gene expression manifold. Here, we develop a generative model of RNA velocity and a Bayesian inference approach that solves these problems. Our model couples velocity field and manifold estimation in a reformulated, unified framework, so as to coherently identify the parameters of an autonomous dynamical system. Focusing on the cell cycle, we implemented VeloCycle to study gene regulation dynamics on one-dimensional periodic manifolds and validated using live-imaging its ability to infer actual cell cycle periods. We benchmarked RNA velocity inference with sensitivity analyses and demonstrated one- and multiple-sample testing. We also conducted Markov chain Monte Carlo inference on the model, uncovering key relationships between gene-specific kinetics and our gene-independent velocity estimate. Finally, we applied VeloCycle to in vivo samples and in vitro genome-wide Perturb-seq, revealing regionally-defined proliferation modes in neural progenitors and the effect of gene knockdowns on cell cycle speed. Ultimately, VeloCycle expands the scRNA-seq analysis toolkit with a modular and statistically rigorous RNA velocity inference framework.\nStructural determinants of co-translational protein complex assembly\nAuthors: Malik, S.; Venezian, J.; Lobov, A.; Heidenreich, M.; Garcia-Seisdedos, H.; Yeates, T. O.; Shiber, A.; Levy, E. D.\nScore: 23.0, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576408\nThe assembly of proteins into functional complexes is critical to lifes processes. While textbooks depict complex assembly as occurring between fully synthesized proteins, we know today that thousands of proteins in the human proteome assemble co-translationally during their synthesis. Why this process takes place, however, remains unknown. We show that co-translational assembly is governed by biophysical and structural characteristics of the protein complex, and involves mutually stabilized, intertwined subunits. Consequently, these subunits are also co-regulated across the central dogma, from transcription to protein degradation. Leveraging structural signatures with AlphaFold2-based predictions enables us to accurately predict co-translational assembly on a proteome-wide scale, which we validated by ribosome profiling, genetic perturbations, and smFISH experiments. Notably, the latter showed that co-translationally assembling subunits exhibit co-localized mRNAs. This work unveils a fundamental connection between protein structure and the translation process, highlighting the overarching impact of three-dimensional structure on gene expression, mRNA localization, and proteostasis. One Sentence SummaryProtein complexes with topologically intertwined subunits require co-translational assembly and synchronized proteostasis of subunits, with implications in protein stability, mRNA localization, and evolution. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=195 SRC=\"FIGDIR/small/576408v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (62K): org.highwire.dtl.DTLVardef@163d2d7org.highwire.dtl.DTLVardef@a73ca3org.highwire.dtl.DTLVardef@17e083corg.highwire.dtl.DTLVardef@14ea730_HPS_FORMAT_FIGEXP M_FIG C_FIG\nMoCHI: neural networks to fit interpretable models and quantify energies, energetic couplings, epistasis and allostery from deep mutational scanning data\nAuthors: Faure, A. J.; Lehner, B.\nScore: 10.9, Published: 2024-01-24 DOI: 10.1101/2024.01.21.575681\nThe massively parallel nature of deep mutational scanning (DMS) allows the quantification of the phenotypic effects of thousands of perturbations in a single experiment. We have developed MoCHI, a software tool that allows the parameterisation of arbitrarily complex models using DMS data. MoCHI simplifies the task of building custom models from measurements of mutant effects on any number of phenotypes. It allows the inference of free energy changes, as well as pairwise and higher-order interaction terms (energetic couplings) for specified biophysical models. When a suitable user-specified mechanistic model is not available, global nonlinearities (epistasis) can be estimated directly from the data. MoCHI also builds upon and leverages theory on ensemble (or background-averaged) epistasis to learn sparse predictive models that can incorporate higher-order epistatic terms and are informative of the genetic architecture of the underlying biological system. The combination of DMS and MoCHI allows biophysical measurements to be performed at scale, including the construction of complete allosteric maps of proteins. MoCHI is freely available (https://github.com/lehner-lab/MoCHI) and implemented as an easy-to-use python package relying on the PyTorch machine learning framework.\nEnhanced feature matching in single-cell proteomics characterizes response to IFN-gamma and reveals co-existence of different cell states\nAuthors: Krull, K. K.; Ali, S. A.; Krijgsveld, J.\nScore: 6.6, Published: 2024-01-10 DOI: 10.1101/2024.01.10.575010\nProteome analysis by data-independent acquisition (DIA) has become a powerful approach to obtain deep proteome coverage, and has gained recent traction for label-free analysis of single cells. However, optimal experimental design for DIA-based single-cell proteomics has not been fully explored, and performance metrics of subsequent data analysis tools remain to be evaluated. Therefore, we here present DIA-ME, a data analysis strategy that exploits the co-analysis of low-input samples with a so-called matching enhancer (ME) of higher input, to increase sensitivity, proteome coverage, and data completeness. We evaluate the matching specificity of DIA-ME by a two-proteome model, and demonstrate that false discovery and false transfer are maintained at low levels when using DIA-NN software, while preserving quantification accuracy. We apply DIA-ME to investigate the proteome response of U-2 OS cells to interferon gamma (IFN-{gamma}) in single cells, and recapitulate the time-resolved induction of IFN-{gamma} response proteins as observed in bulk material. Moreover, we observe co- and anti-correlating patterns of protein expression within the same cell, indicating mutually exclusive protein modules and the co-existence of different cell states. Collectively our data show that DIA-ME is a powerful, scalable, and easy-to- implement strategy for single-cell proteomics.\nHigh-content phenotypic analysis of a C. elegans recombinant inbred population identifies genetic and molecular regulators of lifespan\nAuthors: Gao, A. W.; El Alam, G. W.; Zhu, Y.; Li, W.; Katsyuba, E.; Sulc, J.; Li, T.; Li, X.; Overmyer, K. A.; Lalou, A.; Mouchiroud, L.; Bou Sleiman, M.; Cornaglia, M.; Morel, J.-D.; Houtkooper, R.; Coon, J.; Auwerx, J.\nScore: 2.8, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575638\nLifespan is influenced by complex interactions between genetic and environmental factors. Studying those factors in model organisms of a single genetic background limits their translational value for humans. Here, we mapped lifespan determinants in 85 genetically diverse C. elegans recombinant intercross advanced inbred lines (RIAILs). We assessed molecular profiles - transcriptome, proteome, and lipidome - and life-history traits, including lifespan, development, growth dynamics, and reproduction. RIAILs exhibited large variations in lifespan, which positively correlated with developmental time. Among the top candidates obtained from multi-omics data integration and QTL mapping, we validated known and novel longevity modulators, including rict-1, gfm-1 and mltn-1. We translated their relevance to humans using UK Biobank data and showed that variants in RICTOR and GFM1 are associated with an elevated risk of age-related heart disease, dementia, diabetes, kidney, and liver diseases. We organized our dataset as a resource (https://lisp-lms.shinyapps.io/RIAILs/) that allows interactive explorations for new longevity targets.\nProteome-scale tagging and functional screening in mammalian cells by ORFtag\nAuthors: Nemcko, F.; Himmelsbach, M.; Loubiere, V.; Yelagandula, R.; Pagani, M.; Brennecke, J.; Elling, U.; Stark, A.; Ameres, S. L.\nScore: 3.1, Published: 2024-01-17 DOI: 10.1101/2024.01.16.575827\nDetermining protein function in a systematic manner is a key goal of modern biology, but remains challenging with current approaches. Here, we present ORFtag, a versatile, cost-effective and highly efficient method for the massively-parallel tagging and functional interrogation of proteins at proteome scale. Using mouse embryonic stem cells, we showcase ORFtags utility through screens for transcriptional activators, repressors and post-transcriptional regulators. Each screen finds known and novel regulators, including long ORFs not accessible to other methods, revealing that Zfp574 is a highly selective transcriptional activator and that oncogenic fusions frequently function as transactivators.\nThe integration of network biology and pharmacophore modeling suggests repurposing Clindamycin as an inhibitor of pyroptosis via Caspase-1 blockage in tumor-associated macrophages\nAuthors: Weich, A.; Flamann, C.; Berges, J.; Singh, K. P.; Chambers, D.; Lai, X.; Wolkenhauer, O.; Berking, C.; Kroenke, G.; Gupta, S.; Bruns, H.; Vera Gonzalez, J.\nScore: 2.0, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576201\nBackgroundUveal melanoma (UM) is a highly malignant intraocular tumor with a poor prognosis and response to therapy, including immune checkpoint inhibitors (ICIs), after the onset of liver metastasis. The metastatic microenvironment contains high levels of tumor-associated macrophages (TAMs) that correlate positively with a worse patient prognosis. We hypothesized that one could increase the efficacy of ICIs in UM metastases by immunomodulating UM-associated macrophages. MethodsTo identify potential targets for the immunomodulation, we created a network-based representation of the biology of TAMs and employed (bulk and single-cell) differential gene expression analysis to obtain a regulatory core of UM macrophages-associated genes. We utilized selected targets for pharmacophore-based virtual screening against a library of FDA-approved chemical compounds, followed by refined flexible docking analysis. Finally, we ranked the interactions and selected one novel drug-target combination for in vitro validation. ResultsBased on the generated TAM-specific interaction network (3863 nodes, 9073 edges), we derived a UM macrophages-associated regulatory core (74 nodes, 286 edges). From the regulatory core genes, we selected eight potential targets for pharmacophore-based virtual screening (YBX1, GSTP1, NLRP3, ISG15, MYC, PTGS2, NFKB1, CASP1). Of 266 drug-target interactions screened, we identified the interaction between the antibiotic Clindamycin and Caspase-1 as a priority for experimental validation. Our in vitro validation experiments showed that Clindamycin specifically interferes with activated Caspase-1 and inhibits the secretion of IL-1{beta}, IL-18, and lactate dehydrogenase (LDH) in macrophages after stimulation. Our results suggest that repurposed Clindamycin could reduce pyroptosis in TAMs, a pro-inflammatory form of programmed immune cell death favouring tumor progression. ConclusionWe were able to predict a novel Clindamycin-Caspase-1 interaction that effectively blocks Caspase-1-mediated inflammasome activity and pyroptosis in vitro in macrophages. This interaction is a promising clinical immunomodulator of the tumor microenvironment for improving ICI responsivenss. This work demonstrates the power of combining network-based transcriptomic analysis with pharmacophore-guided screening for de novo drug-target repurposing. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=104 SRC=\"FIGDIR/small/576201v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (26K): org.highwire.dtl.DTLVardef@1fec0e0org.highwire.dtl.DTLVardef@28c182org.highwire.dtl.DTLVardef@1b85eb6org.highwire.dtl.DTLVardef@1521941_HPS_FORMAT_FIGEXP M_FIG C_FIG\nAutism genes converge on microtubule biology and RNA-binding proteins during excitatory neurogenesis\nAuthors: Sun, N.; Teyssier, N.; Wang, B.; Drake, S.; Seyler, M.; Zaltsman, Y.; Everitt, A.; Teerikorpi, N.; Willsey, H. R.; Goodarzi, H.; Tian, R.; Kampmann, M.; Willsey, J.\nScore: 17.9, Published: 2024-01-02 DOI: 10.1101/2023.12.22.573108\nRecent studies have identified over one hundred high-confidence (hc) autism spectrum disorder (ASD) genes. Systems biological and functional analyses on smaller subsets of these genes have consistently implicated excitatory neurogenesis. However, the extent to which the broader set of hcASD genes are involved in this process has not been explored systematically nor have the biological pathways underlying this convergence been identified. Here, we leveraged CROP-Seq to repress 87 hcASD genes in a human in vitro model of cortical neurogenesis. We identified 17 hcASD genes whose repression significantly alters developmental trajectory and results in a common cellular state characterized by disruptions in proliferation, differentiation, cell cycle, microtubule biology, and RNA-binding proteins (RBPs). We also characterized over 3,000 differentially expressed genes, 286 of which had expression profiles correlated with changes in developmental trajectory. Overall, we uncovered transcriptional disruptions downstream of hcASD gene perturbations, correlated these disruptions with distinct differentiation phenotypes, and reinforced neurogenesis, microtubule biology, and RBPs as convergent points of disruption in ASD.\nPredicting microbial genome-scale metabolic networks directly from 16S rRNA gene sequences\nAuthors: Oyas, O.; Kobel, C. M.; Vik, J. O.; Pope, P. B.\nScore: 1.8, Published: 2024-01-27 DOI: 10.1101/2024.01.26.576649\nGenome-scale metabolic models are key biotechnology tools that can predict metabolic capabilities and growth for an organism. In particular, these models have become indispensable for metabolic analysis of microbial species and communities such as the gut microbiomes of humans and other animals. Accurate microbial models can be built automatically from genomes, but many microbes have only been observed through sequencing of marker genes such as 16S rRNA and thus remain inaccessible to genome-scale modeling. To extend the scope of genome-scale metabolic models to microbes that lack genomic information, we trained an artificial neural network to build microbial models from numeric representations of 16S rRNA gene sequences. Specifically, we built models and extracted 16S rRNA gene sequences from more than 15,000 reference and representative microbial genomes, computed multiple sequence alignments and large language model embeddings for the 16S rRNA genes, and trained the neural network to predict metabolic reaction probabilities from sequences, alignments, or embeddings. Training was fast on a single graphics processing unit and trained networks predicted reaction probabilities accurately for unseen archaeal and bacterial sequences and species. This makes it possible to reconstruct microbial genome-scale metabolic networks from any 16S rRNA gene sequence and enables simulation of metabolism and growth for all observed microbial life.\nA rule-based multiscale model of hepatic stellate cell plasticity: critical role of the inactivation loop in fibrosis progression\nAuthors: Bougueon, M.; Legagneux, V.; Hazard, O.; Bomo, J.; Siegel, A.; Feret, J.; Theret, N.\nScore: 1.5, Published: 2024-01-26 DOI: 10.1101/2024.01.24.577157\nHepatic stellate cells (HSC) are the source of extracellular matrix (ECM) whose overproduction leads to fibrosis, a condition that impairs liver functions in chronic liver diseases. Understanding the dynamics of HSCs will provide insights needed to develop new therapeutic approaches. Few models of hepatic fibrosis have been proposed, and none of them include the heterogeneity of HSC phenotypes recently highlighted by single-cell RNA sequencing analyses. Here, we developed rule-based models to study HSC dynamics during fibrosis progression and reversion. We used the Kappa graph rewriting language, for which we used tokens and counters to overcome temporal explosion. HSCs are modeled as agents that present seven physiological cellular states and that interact with (TGF{beta}1) molecules which regulate HSC activation and the secretion of type I collagen, the main component of the ECM. Simulation studies revealed the critical role of the HSC inactivation process during fibrosis progression and reversion. While inactivation allows elimination of activated HSCs during reversion steps, reactivation loops of inactivated HSCs (iHSCs) are required to sustain fibrosis. Furthermore, we demonstrated the models sensitivity to (TGF{beta}1) parameters, suggesting its adaptability to a variety of pathophysiological conditions for which levels of (TGF{beta}1) production associated with the inflammatory response differ. Using new experimental data from a mouse model of CCl4-induced liver fibrosis, we validated the predicted ECM dynamics. Our model also predicts the accumulation of iHSCs during chronic liver disease. By analyzing RNA sequencing data from patients with non-alcoholic steatohepatitis (NASH) associated with liver fibrosis, we confirmed this accumulation, identifying iHSCs as novel markers of fibrosis progression. Overall, our study provides the first model of HSC dynamics in chronic liver disease that can be used to explore the regulatory role of iHSCs in liver homeostasis. Moreover our model can also be generalized to fibroblasts during repair and fibrosis in other tissues.\n",
  "wordCount" : "2454",
  "inLanguage": "en",
  "datePublished": "2024-01-28T10:37:59Z",
  "dateModified": "2024-01-28T10:37:59Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 28, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576093">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576093" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576093">
        <p class="paperTitle">Statistical inference with a manifold-constrained RNA velocity model uncovers cell cycle speed modulations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576093" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576093" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lederer, A. R.; Leonardi, M.; Talamanca, L.; Herrera, A.; Droin, C.; Khven, I.; Carvalho, H. J.; Valente, A.; Dominguez Mantes, A.; Mulet Arabi, P.; Pinello, L.; Naef, F.; La Manno, G.</p>
        <p class="info">Score: 40.1, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576093' target='https://doi.org/10.1101/2024.01.18.576093'> 10.1101/2024.01.18.576093</a></p>
        <p class="abstract">Across a range of biological processes, cells undergo coordinated changes in gene expression, resulting in transcriptome dynamics that unfold within a low-dimensional manifold. Single-cell RNA-sequencing (scRNA-seq) only measures temporal snapshots of gene expression. However, information on the underlying low-dimensional dynamics can be extracted using RNA velocity, which models unspliced and spliced RNA abundances to estimate the rate of change of gene expression. Available RNA velocity algorithms can be fragile and rely on heuristics that lack statistical control. Moreover, the estimated vector field is not dynamically consistent with the traversed gene expression manifold. Here, we develop a generative model of RNA velocity and a Bayesian inference approach that solves these problems. Our model couples velocity field and manifold estimation in a reformulated, unified framework, so as to coherently identify the parameters of an autonomous dynamical system. Focusing on the cell cycle, we implemented VeloCycle to study gene regulation dynamics on one-dimensional periodic manifolds and validated using live-imaging its ability to infer actual cell cycle periods. We benchmarked RNA velocity inference with sensitivity analyses and demonstrated one- and multiple-sample testing. We also conducted Markov chain Monte Carlo inference on the model, uncovering key relationships between gene-specific kinetics and our gene-independent velocity estimate. Finally, we applied VeloCycle to in vivo samples and in vitro genome-wide Perturb-seq, revealing regionally-defined proliferation modes in neural progenitors and the effect of gene knockdowns on cell cycle speed. Ultimately, VeloCycle expands the scRNA-seq analysis toolkit with a modular and statistically rigorous RNA velocity inference framework.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576408">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576408" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576408">
        <p class="paperTitle">Structural determinants of co-translational protein complex assembly</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576408" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576408" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Malik, S.; Venezian, J.; Lobov, A.; Heidenreich, M.; Garcia-Seisdedos, H.; Yeates, T. O.; Shiber, A.; Levy, E. D.</p>
        <p class="info">Score: 23.0, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576408' target='https://doi.org/10.1101/2024.01.20.576408'> 10.1101/2024.01.20.576408</a></p>
        <p class="abstract">The assembly of proteins into functional complexes is critical to lifes processes. While textbooks depict complex assembly as occurring between fully synthesized proteins, we know today that thousands of proteins in the human proteome assemble co-translationally during their synthesis. Why this process takes place, however, remains unknown. We show that co-translational assembly is governed by biophysical and structural characteristics of the protein complex, and involves mutually stabilized, intertwined subunits. Consequently, these subunits are also co-regulated across the central dogma, from transcription to protein degradation. Leveraging structural signatures with AlphaFold2-based predictions enables us to accurately predict co-translational assembly on a proteome-wide scale, which we validated by ribosome profiling, genetic perturbations, and smFISH experiments. Notably, the latter showed that co-translationally assembling subunits exhibit co-localized mRNAs. This work unveils a fundamental connection between protein structure and the translation process, highlighting the overarching impact of three-dimensional structure on gene expression, mRNA localization, and proteostasis.

One Sentence SummaryProtein complexes with topologically intertwined subunits require co-translational assembly and synchronized proteostasis of subunits, with implications in protein stability, mRNA localization, and evolution.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=195 SRC=&#34;FIGDIR/small/576408v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (62K):
org.highwire.dtl.DTLVardef@163d2d7org.highwire.dtl.DTLVardef@a73ca3org.highwire.dtl.DTLVardef@17e083corg.highwire.dtl.DTLVardef@14ea730_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.21.575681">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.21.575681" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.21.575681">
        <p class="paperTitle">MoCHI: neural networks to fit interpretable models and quantify energies, energetic couplings, epistasis and allostery from deep mutational scanning data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.21.575681" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.21.575681" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Faure, A. J.; Lehner, B.</p>
        <p class="info">Score: 10.9, Published: 2024-01-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.21.575681' target='https://doi.org/10.1101/2024.01.21.575681'> 10.1101/2024.01.21.575681</a></p>
        <p class="abstract">The massively parallel nature of deep mutational scanning (DMS) allows the quantification of the phenotypic effects of thousands of perturbations in a single experiment. We have developed MoCHI, a software tool that allows the parameterisation of arbitrarily complex models using DMS data. MoCHI simplifies the task of building custom models from measurements of mutant effects on any number of phenotypes. It allows the inference of free energy changes, as well as pairwise and higher-order interaction terms (energetic couplings) for specified biophysical models. When a suitable user-specified mechanistic model is not available, global nonlinearities (epistasis) can be estimated directly from the data. MoCHI also builds upon and leverages theory on ensemble (or background-averaged) epistasis to learn sparse predictive models that can incorporate higher-order epistatic terms and are informative of the genetic architecture of the underlying biological system. The combination of DMS and MoCHI allows biophysical measurements to be performed at scale, including the construction of complete allosteric maps of proteins. MoCHI is freely available (https://github.com/lehner-lab/MoCHI) and implemented as an easy-to-use python package relying on the PyTorch machine learning framework.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.575010">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.575010" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.575010">
        <p class="paperTitle">Enhanced feature matching in single-cell proteomics characterizes response to IFN-gamma and reveals co-existence of different cell states</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.575010" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.575010" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krull, K. K.; Ali, S. A.; Krijgsveld, J.</p>
        <p class="info">Score: 6.6, Published: 2024-01-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.575010' target='https://doi.org/10.1101/2024.01.10.575010'> 10.1101/2024.01.10.575010</a></p>
        <p class="abstract">Proteome analysis by data-independent acquisition (DIA) has become a powerful approach to obtain deep proteome coverage, and has gained recent traction for label-free analysis of single cells. However, optimal experimental design for DIA-based single-cell proteomics has not been fully explored, and performance metrics of subsequent data analysis tools remain to be evaluated. Therefore, we here present DIA-ME, a data analysis strategy that exploits the co-analysis of low-input samples with a so-called matching enhancer (ME) of higher input, to increase sensitivity, proteome coverage, and data completeness. We evaluate the matching specificity of DIA-ME by a two-proteome model, and demonstrate that false discovery and false transfer are maintained at low levels when using DIA-NN software, while preserving quantification accuracy. We apply DIA-ME to investigate the proteome response of U-2 OS cells to interferon gamma (IFN-{gamma}) in single cells, and recapitulate the time-resolved induction of IFN-{gamma} response proteins as observed in bulk material. Moreover, we observe co- and anti-correlating patterns of protein expression within the same cell, indicating mutually exclusive protein modules and the co-existence of different cell states. Collectively our data show that DIA-ME is a powerful, scalable, and easy-to- implement strategy for single-cell proteomics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575638">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575638" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575638">
        <p class="paperTitle">High-content phenotypic analysis of a C. elegans recombinant inbred population identifies genetic and molecular regulators of lifespan</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575638" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575638" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gao, A. W.; El Alam, G. W.; Zhu, Y.; Li, W.; Katsyuba, E.; Sulc, J.; Li, T.; Li, X.; Overmyer, K. A.; Lalou, A.; Mouchiroud, L.; Bou Sleiman, M.; Cornaglia, M.; Morel, J.-D.; Houtkooper, R.; Coon, J.; Auwerx, J.</p>
        <p class="info">Score: 2.8, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575638' target='https://doi.org/10.1101/2024.01.15.575638'> 10.1101/2024.01.15.575638</a></p>
        <p class="abstract">Lifespan is influenced by complex interactions between genetic and environmental factors. Studying those factors in model organisms of a single genetic background limits their translational value for humans. Here, we mapped lifespan determinants in 85 genetically diverse C. elegans recombinant intercross advanced inbred lines (RIAILs). We assessed molecular profiles - transcriptome, proteome, and lipidome - and life-history traits, including lifespan, development, growth dynamics, and reproduction. RIAILs exhibited large variations in lifespan, which positively correlated with developmental time. Among the top candidates obtained from multi-omics data integration and QTL mapping, we validated known and novel longevity modulators, including rict-1, gfm-1 and mltn-1. We translated their relevance to humans using UK Biobank data and showed that variants in RICTOR and GFM1 are associated with an elevated risk of age-related heart disease, dementia, diabetes, kidney, and liver diseases. We organized our dataset as a resource (https://lisp-lms.shinyapps.io/RIAILs/) that allows interactive explorations for new longevity targets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.16.575827">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.16.575827" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.16.575827">
        <p class="paperTitle">Proteome-scale tagging and functional screening in mammalian cells by ORFtag</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.16.575827" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.16.575827" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nemcko, F.; Himmelsbach, M.; Loubiere, V.; Yelagandula, R.; Pagani, M.; Brennecke, J.; Elling, U.; Stark, A.; Ameres, S. L.</p>
        <p class="info">Score: 3.1, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.16.575827' target='https://doi.org/10.1101/2024.01.16.575827'> 10.1101/2024.01.16.575827</a></p>
        <p class="abstract">Determining protein function in a systematic manner is a key goal of modern biology, but remains challenging with current approaches. Here, we present ORFtag, a versatile, cost-effective and highly efficient method for the massively-parallel tagging and functional interrogation of proteins at proteome scale. Using mouse embryonic stem cells, we showcase ORFtags utility through screens for transcriptional activators, repressors and post-transcriptional regulators. Each screen finds known and novel regulators, including long ORFs not accessible to other methods, revealing that Zfp574 is a highly selective transcriptional activator and that oncogenic fusions frequently function as transactivators.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576201">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576201" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576201">
        <p class="paperTitle">The integration of network biology and pharmacophore modeling suggests repurposing Clindamycin as an inhibitor of pyroptosis via Caspase-1 blockage in tumor-associated macrophages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576201" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576201" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Weich, A.; Flamann, C.; Berges, J.; Singh, K. P.; Chambers, D.; Lai, X.; Wolkenhauer, O.; Berking, C.; Kroenke, G.; Gupta, S.; Bruns, H.; Vera Gonzalez, J.</p>
        <p class="info">Score: 2.0, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576201' target='https://doi.org/10.1101/2024.01.18.576201'> 10.1101/2024.01.18.576201</a></p>
        <p class="abstract">BackgroundUveal melanoma (UM) is a highly malignant intraocular tumor with a poor prognosis and response to therapy, including immune checkpoint inhibitors (ICIs), after the onset of liver metastasis. The metastatic microenvironment contains high levels of tumor-associated macrophages (TAMs) that correlate positively with a worse patient prognosis. We hypothesized that one could increase the efficacy of ICIs in UM metastases by immunomodulating UM-associated macrophages.

MethodsTo identify potential targets for the immunomodulation, we created a network-based representation of the biology of TAMs and employed (bulk and single-cell) differential gene expression analysis to obtain a regulatory core of UM macrophages-associated genes. We utilized selected targets for pharmacophore-based virtual screening against a library of FDA-approved chemical compounds, followed by refined flexible docking analysis. Finally, we ranked the interactions and selected one novel drug-target combination for in vitro validation.

ResultsBased on the generated TAM-specific interaction network (3863 nodes, 9073 edges), we derived a UM macrophages-associated regulatory core (74 nodes, 286 edges). From the regulatory core genes, we selected eight potential targets for pharmacophore-based virtual screening (YBX1, GSTP1, NLRP3, ISG15, MYC, PTGS2, NFKB1, CASP1). Of 266 drug-target interactions screened, we identified the interaction between the antibiotic Clindamycin and Caspase-1 as a priority for experimental validation. Our in vitro validation experiments showed that Clindamycin specifically interferes with activated Caspase-1 and inhibits the secretion of IL-1{beta}, IL-18, and lactate dehydrogenase (LDH) in macrophages after stimulation. Our results suggest that repurposed Clindamycin could reduce pyroptosis in TAMs, a pro-inflammatory form of programmed immune cell death favouring tumor progression.

ConclusionWe were able to predict a novel Clindamycin-Caspase-1 interaction that effectively blocks Caspase-1-mediated inflammasome activity and pyroptosis in vitro in macrophages. This interaction is a promising clinical immunomodulator of the tumor microenvironment for improving ICI responsivenss. This work demonstrates the power of combining network-based transcriptomic analysis with pharmacophore-guided screening for de novo drug-target repurposing.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=104 SRC=&#34;FIGDIR/small/576201v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (26K):
org.highwire.dtl.DTLVardef@1fec0e0org.highwire.dtl.DTLVardef@28c182org.highwire.dtl.DTLVardef@1b85eb6org.highwire.dtl.DTLVardef@1521941_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.573108">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.573108" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.573108">
        <p class="paperTitle">Autism genes converge on microtubule biology and RNA-binding proteins during excitatory neurogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.573108" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.573108" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sun, N.; Teyssier, N.; Wang, B.; Drake, S.; Seyler, M.; Zaltsman, Y.; Everitt, A.; Teerikorpi, N.; Willsey, H. R.; Goodarzi, H.; Tian, R.; Kampmann, M.; Willsey, J.</p>
        <p class="info">Score: 17.9, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.573108' target='https://doi.org/10.1101/2023.12.22.573108'> 10.1101/2023.12.22.573108</a></p>
        <p class="abstract">Recent studies have identified over one hundred high-confidence (hc) autism spectrum disorder (ASD) genes. Systems biological and functional analyses on smaller subsets of these genes have consistently implicated excitatory neurogenesis. However, the extent to which the broader set of hcASD genes are involved in this process has not been explored systematically nor have the biological pathways underlying this convergence been identified. Here, we leveraged CROP-Seq to repress 87 hcASD genes in a human in vitro model of cortical neurogenesis. We identified 17 hcASD genes whose repression significantly alters developmental trajectory and results in a common cellular state characterized by disruptions in proliferation, differentiation, cell cycle, microtubule biology, and RNA-binding proteins (RBPs). We also characterized over 3,000 differentially expressed genes, 286 of which had expression profiles correlated with changes in developmental trajectory. Overall, we uncovered transcriptional disruptions downstream of hcASD gene perturbations, correlated these disruptions with distinct differentiation phenotypes, and reinforced neurogenesis, microtubule biology, and RBPs as convergent points of disruption in ASD.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.26.576649">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.26.576649" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.26.576649">
        <p class="paperTitle">Predicting microbial genome-scale metabolic networks directly from 16S rRNA gene sequences</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.26.576649" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.26.576649" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Oyas, O.; Kobel, C. M.; Vik, J. O.; Pope, P. B.</p>
        <p class="info">Score: 1.8, Published: 2024-01-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.26.576649' target='https://doi.org/10.1101/2024.01.26.576649'> 10.1101/2024.01.26.576649</a></p>
        <p class="abstract">Genome-scale metabolic models are key biotechnology tools that can predict metabolic capabilities and growth for an organism. In particular, these models have become indispensable for metabolic analysis of microbial species and communities such as the gut microbiomes of humans and other animals. Accurate microbial models can be built automatically from genomes, but many microbes have only been observed through sequencing of marker genes such as 16S rRNA and thus remain inaccessible to genome-scale modeling. To extend the scope of genome-scale metabolic models to microbes that lack genomic information, we trained an artificial neural network to build microbial models from numeric representations of 16S rRNA gene sequences. Specifically, we built models and extracted 16S rRNA gene sequences from more than 15,000 reference and representative microbial genomes, computed multiple sequence alignments and large language model embeddings for the 16S rRNA genes, and trained the neural network to predict metabolic reaction probabilities from sequences, alignments, or embeddings. Training was fast on a single graphics processing unit and trained networks predicted reaction probabilities accurately for unseen archaeal and bacterial sequences and species. This makes it possible to reconstruct microbial genome-scale metabolic networks from any 16S rRNA gene sequence and enables simulation of metabolism and growth for all observed microbial life.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.577157">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.577157" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.577157">
        <p class="paperTitle">A rule-based multiscale model of hepatic stellate cell plasticity: critical role of the inactivation loop in fibrosis progression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.577157" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.577157" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bougueon, M.; Legagneux, V.; Hazard, O.; Bomo, J.; Siegel, A.; Feret, J.; Theret, N.</p>
        <p class="info">Score: 1.5, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.577157' target='https://doi.org/10.1101/2024.01.24.577157'> 10.1101/2024.01.24.577157</a></p>
        <p class="abstract">Hepatic stellate cells (HSC) are the source of extracellular matrix (ECM) whose overproduction leads to fibrosis, a condition that impairs liver functions in chronic liver diseases. Understanding the dynamics of HSCs will provide insights needed to develop new therapeutic approaches. Few models of hepatic fibrosis have been proposed, and none of them include the heterogeneity of HSC phenotypes recently highlighted by single-cell RNA sequencing analyses. Here, we developed rule-based models to study HSC dynamics during fibrosis progression and reversion. We used the Kappa graph rewriting language, for which we used tokens and counters to overcome temporal explosion. HSCs are modeled as agents that present seven physiological cellular states and that interact with (TGF{beta}1) molecules which regulate HSC activation and the secretion of type I collagen, the main component of the ECM. Simulation studies revealed the critical role of the HSC inactivation process during fibrosis progression and reversion. While inactivation allows elimination of activated HSCs during reversion steps, reactivation loops of inactivated HSCs (iHSCs) are required to sustain fibrosis. Furthermore, we demonstrated the models sensitivity to (TGF{beta}1) parameters, suggesting its adaptability to a variety of pathophysiological conditions for which levels of (TGF{beta}1) production associated with the inflammatory response differ. Using new experimental data from a mouse model of CCl4-induced liver fibrosis, we validated the predicted ECM dynamics. Our model also predicts the accumulation of iHSCs during chronic liver disease. By analyzing RNA sequencing data from patients with non-alcoholic steatohepatitis (NASH) associated with liver fibrosis, we confirmed this accumulation, identifying iHSCs as novel markers of fibrosis progression. Overall, our study provides the first model of HSC dynamics in chronic liver disease that can be used to explore the regulatory role of iHSCs in liver homeostasis. Moreover our model can also be generalized to fibroblasts during repair and fibrosis in other tissues.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
