<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Self-inhibiting percolation and viral spreading in epithelial tissue
Authors: Xu, X.; Nielsen, B. F.; sneppen, k.
Score: 23.9, Published: 2023-12-13 DOI: 10.1101/2023.12.12.571279
SARS-CoV-2 induces delayed type-I/III interferon production, allowing it to escape the early innate immune response. The delay has been attributed to a deficiency in the ability of cells to sense viral replication upon infection, which in turn hampers activation of the antiviral state in bystander cells. Here, we introduce a cellular automaton model to investigate the spatiotemporal spreading of viral infection as a function of virus and host-dependent parameters.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Self-inhibiting percolation and viral spreading in epithelial tissue
Authors: Xu, X.; Nielsen, B. F.; sneppen, k.
Score: 23.9, Published: 2023-12-13 DOI: 10.1101/2023.12.12.571279
SARS-CoV-2 induces delayed type-I/III interferon production, allowing it to escape the early innate immune response. The delay has been attributed to a deficiency in the ability of cells to sense viral replication upon infection, which in turn hampers activation of the antiviral state in bystander cells. Here, we introduce a cellular automaton model to investigate the spatiotemporal spreading of viral infection as a function of virus and host-dependent parameters." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-20T10:36:27+00:00" />
<meta property="article:modified_time" content="2023-12-20T10:36:27+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Self-inhibiting percolation and viral spreading in epithelial tissue
Authors: Xu, X.; Nielsen, B. F.; sneppen, k.
Score: 23.9, Published: 2023-12-13 DOI: 10.1101/2023.12.12.571279
SARS-CoV-2 induces delayed type-I/III interferon production, allowing it to escape the early innate immune response. The delay has been attributed to a deficiency in the ability of cells to sense viral replication upon infection, which in turn hampers activation of the antiviral state in bystander cells. Here, we introduce a cellular automaton model to investigate the spatiotemporal spreading of viral infection as a function of virus and host-dependent parameters."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Self-inhibiting percolation and viral spreading in epithelial tissue\nAuthors: Xu, X.; Nielsen, B. F.; sneppen, k.\nScore: 23.9, Published: 2023-12-13 DOI: 10.1101/2023.12.12.571279\nSARS-CoV-2 induces delayed type-I/III interferon production, allowing it to escape the early innate immune response. The delay has been attributed to a deficiency in the ability of cells to sense viral replication upon infection, which in turn hampers activation of the antiviral state in bystander cells. Here, we introduce a cellular automaton model to investigate the spatiotemporal spreading of viral infection as a function of virus and host-dependent parameters.",
  "keywords": [
    
  ],
  "articleBody": " Self-inhibiting percolation and viral spreading in epithelial tissue\nAuthors: Xu, X.; Nielsen, B. F.; sneppen, k.\nScore: 23.9, Published: 2023-12-13 DOI: 10.1101/2023.12.12.571279\nSARS-CoV-2 induces delayed type-I/III interferon production, allowing it to escape the early innate immune response. The delay has been attributed to a deficiency in the ability of cells to sense viral replication upon infection, which in turn hampers activation of the antiviral state in bystander cells. Here, we introduce a cellular automaton model to investigate the spatiotemporal spreading of viral infection as a function of virus and host-dependent parameters. The model suggests that the considerable person-to-person heterogeneity in SARS-CoV-2 infections is a consequence of high sensitivity to slight variations in biological parameters near a critical threshold. It further suggests that within-host viral proliferation can be curtailed by the presence of remarkably few cells that are primed for IFN production. Thus the observed heterogeneity in defense readiness of cells reflects a remarkably cost-efficient strategy for protection.\nA distinctive evolution of alveolar T cell responses is associated with clinical outcomes in unvaccinated patients with SARS-CoV-2 pneumonia\nAuthors: Markov, N. S.; Ren, Z.; Senkow, K. J.; Grant, R. A.; Gao, C. A.; Malsin, E. S.; Sichizya, L.; Kihshen, H.; Helmin, K. A.; Jovisic, M.; Arnold, J. M.; Perez-Leonor, X. G.; Abdala-Valencia, H.; Swaminathan, S.; Nwaezeapu, J.; Kang, M.; Rasmussen, L.; Ozer, E. A.; Lorenzo-Redondo, R.; Hultquist, J. F.; Simons, L.; Rios-Guzman, E.; Misharin, A. V.; Wunderink, R. G.; Budinger, G. S.; Singer, B. D.; Morales-Nebreda, L.; The NU SCRIPT Investigators, Score: 6.8, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571479\nPathogen clearance and resolution of inflammation in patients with pneumonia require an effective local T cell response. Nevertheless, local T cell activation may drive lung injury, particularly during prolonged episodes of respiratory failure characteristic of severe SARS-CoV-2 pneumonia. While T cell responses in the peripheral blood are well described, the evolution of T cell phenotypes and molecular signatures in the distal lung of patients with severe pneumonia caused by SARS-CoV-2 or other pathogens is understudied. Accordingly, we serially obtained 432 bronchoalveolar lavage fluid samples from 273 patients with severe pneumonia and respiratory failure, including 74 unvaccinated patients with COVID-19, and performed flow cytometry, transcriptional, and T cell receptor profiling on sorted CD8+ and CD4+ T cell subsets. In patients with COVID-19 but not pneumonia secondary to other pathogens, we found that early and persistent enrichment in CD8+ and CD4+ T cell subsets correlated with survival to hospital discharge. Activation of interferon signaling pathways early after intubation for COVID-19 was associated with favorable outcomes, while activation of NF-{kappa}B-driven programs late in disease was associated with poor outcomes. Patients with SARS-CoV-2 pneumonia whose alveolar T cells preferentially targeted the Spike and Nucleocapsid proteins tended to experience more favorable outcomes than patients whose T cells predominantly targeted the ORF1ab polyprotein complex. These results suggest that in patients with severe SARS-CoV-2 pneumonia, alveolar T cell interferon responses targeting structural SARS-CoV-2 proteins characterize patients who recover, yet these responses progress to NF-{kappa}B activation against non-structural proteins in patients who go on to experience poor clinical outcomes.\nA foundational atlas of autism protein interactions reveals molecular convergence\nAuthors: Wang, B.; Vartak, R.; Zaltsman, Y.; Naing, Z. Z. C.; Hennick, K. M.; Polacco, B. J.; Bashir, A.; Eckhardt, M.; Bouhaddou, M.; Xu, J.; Sun, N.; Lasser, M.; Zhou, Y.; Guiley, K. Z.; Chan, U.; Kaye, J. A.; Khare, P.; Drake, S.; Drury, V.; Burke, D. F.; Gonzalez, S.; Alkhairy, S.; Morris, M.; Baum, T.; Krasnoff, R.; Wang, S.; Pham, P.; Arbalaez, J.; Pratt, D.; Chag, S.; Rolland, T.; Bourgeron, T.; Finkbeiner, S.; Bandyopadhay, S.; Ideker, T.; Beltrao, P.; Willsey, H. R.; Obernier, K.; Nowakowski, T. J.; Huttenhain, R.; State, M. W.; Willsey, A. J.; Krogan, N. J.\nScore: 57.1, Published: 2023-12-09 DOI: 10.1101/2023.12.03.569805\nDespite the identification of a large number of high-confidence (hc) autism spectrum disorder (ASD) risk genes, translation to viable treatment targets remains elusive. Using affinity purification-mass spectrometry and HEK293T cells, we constructed a protein-protein interaction (PPI) network involving 100 hcASD risk genes, revealing over 1,800 PPIs, the vast majority of which have not been previously described. Importantly, the interactors are highly expressed in the human brain and neural progenitor cells, and enriched for ASD, but not schizophrenia, genetic risk. The network is highly connected, converges on protein complexes, including those involved in neurogenesis, tubulin biology, transcriptional regulation, and chromatin modification. We also generated a PPI map using over 50 patient-derived missense variants to identify differential physical interactions. Using these data, we leveraged AlphaFold2 predictions to prioritize direct PPIs as well as specific variants for interrogation in Xenopus tropicalis and human forebrain organoids. One such mutation in the transcription factor FOXP1, which disrupts interaction with FOXP4, led to a reconfiguration of DNA binding sites and disruption of neurogenesis, with a particularly strong impact on deep layer cortical and subplate neurons,consistent with a large body of work in ASD. Overall, this work offers critical new insights into molecular mechanisms underlying ASD that results from large effect genetic variants and describes a powerful platform to develop and test novel therapeutic strategies for a wide range of genetically-defined conditions.\nBroad H3K4me3 Domain Is Associated with Spatial Coherence during Mammalian Embryonic Development\nAuthors: Cao, X.; Ma, T.; Fan, R.; Yuan, G.-C.\nScore: 11.2, Published: 2023-12-12 DOI: 10.1101/2023.12.11.570452\nIt is well known that the chromatin states play a major role in cell-fate decision and cell-identity maintenance; however, the spatial variation of chromatin states in situ remains poorly characterized. Here, by leveraging recently available spatial-CUT\u0026Tag data, we systematically characterized the global spatial organization of the H3K4me3 profiles in a mouse embryo. Our analysis identified a subset of genes with spatially coherent H3K4me3 patterns, which together delineate the tissue boundaries. The spatially coherent genes are strongly enriched with tissue-specific transcriptional regulators. Remarkably, their corresponding genomic loci are marked by broad H3K4me3 domains, which is distinct from the typical H3K4me3 signature. Spatial transition across tissue boundaries is associated with continuous shortening of the broad H3K4me3 domains as well as expansion of H3K27me3 domains. Our analysis reveals a strong connection between the genomic and spatial variation of chromatin states, which may play an important role in embryonic development.\nMachine Learning and Metabolic Model Guided CRISPRi Reveals a Central Role for Phosphoglycerate Mutase in Chlamydia trachomatis Persistence\nAuthors: Chowdhury, N. B.; Pokorzynski, N. D.; Rucks, E. A.; Ouellette, S. P.; CARABEO, R.; Saha, R.\nScore: 1.8, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572198\nUpon nutrient starvation, Chlamydia trachomatis serovar L2 (CTL) shifts from its normal growth to a non-replicating form, termed persistence. It is unclear if persistence is an adaptive response or lack of it. To understand that transcriptomics data were collected for nutrient-sufficient and nutrient-starved CTL. Applying machine learning approaches on transcriptomics data revealed a global transcriptomic rewiring of CTL under stress conditions without having any global stress regulator. This indicated that CTL stress response is due to lack of an adaptive response mechanism. To investigate the impact of this on CTL metabolism, we reconstructed a genome-scale metabolic model of CTL (iCTL278) and contextualized it with the collected transcriptomics data. Using the metabolic bottleneck analysis on contextualized iCTL278, we observed phosphoglycerate mutase (pgm) regulates the entry of CTL to the persistence. Later, pgm was found to have the highest thermodynamics driving force and lowest enzymatic cost. Furthermore, CRISPRi-driven knockdown of pgm and tryptophan starvation experiments revealed the importance of this gene in inducing persistence. Hence, this work, for the first time, introduced thermodynamics and enzyme-cost as tools to gain deeper understanding on CTL persistence.\nProfiling the proximal proteome of the mu opioid receptor identifies novel regulators of receptor signaling and trafficking\nAuthors: Polacco, B. J.; Lobingier, B. T.; Blythe, E. E.; Abreu, N.; Khare, P.; Howard, M. K.; Gonzalez-Hernandez, A. J.; Xu, J.; Li, Q.; Novy, B.; Naing, Z. Z. C.; Shoichet, B. K.; Coyote-Maestas, W.; Levitz, J.; Krogan, N. J.; Von Zastrow, M.; Huttenhain, R.\nScore: 29.4, Published: 2023-12-10 DOI: 10.1101/2022.03.28.486115\nThe mu opioid receptor (OR), a prototypic member of the large G protein-coupled receptor (GPCR) family, represents an important target of therapeutic and abused drugs. To date, most of our understanding of OR activity has focused on signal transducers and regulatory molecules including G proteins, GPCR kinases, and beta-arrestins. Yet it is clear that signaling through the OR is coordinated by additional proteins recruited into the proximal interaction network of the activated receptor, which have largely remained invisible given the lack of technologies to interrogate these networks systematically. Here, we implement a quantitative proteomics pipeline leveraging the chemical diversity of OR agonists and APEX-based proximity labeling to investigate the protein networks that underlie OR signaling. We leverage a novel computational framework to extract subcellular location, trafficking, and functional partners of GPCR activity from the proximity labeling datasets. Applying this unbiased, systematic approach to the OR, we demonstrate that opioid agonists exert differences in the OR proximal proteome mediated by endocytosis and subsequent endosomal sorting, exemplified by VPS35 and COMMD3. Moreover, we identify two novel OR network components, EYA4 and KCTD12, that are recruited into the receptor proximal network irrespective of the activating ligand and independent of receptor trafficking but based on receptor-triggered G protein activation. We provide functional evidence that these network components form a previously unrecognized buffering system for G protein activity which broadly modulates cellular GPCR signaling.\nA Mathematical Framework for Measuring and Tuning Tempo in Developmental Gene Regulatory Networks\nAuthors: Manser, C. L.; Perez-Carrasco, R.\nScore: 1.6, Published: 2023-12-19 DOI: 10.1101/2023.12.18.572181\nEmbryo development is a dynamic process governed by the regulation of timing and sequences of gene expression, which control the proper growth of the organism. While many genetic programs coordinating these sequences are common across species, the timescales of gene expression can vary significantly among different organisms. Currently, substantial experimental efforts are focused on identifying molecular mechanisms that control these temporal aspects. In contrast, the capacity of established mathematical models to incorporate tempo control while maintaining the same dynamical landscape remains less understood. This manuscript addresses this gap by developing a mathematical framework that links the functionality of developmental programs to the corresponding gene expression orbits (or landscapes). This unlocks the ability to find tempo differences as perturbations in the dynamical system that preserve its orbits. We demonstrate that this framework allows for the prediction of molecular mechanisms governing tempo, through both numerical and analytical methods. Our exploration includes two case studies: a generic network featuring coupled production and degradation, and the repressilator. In the latter, we illustrate how altering the dimerisation rates of transcription factors can decouple the tempo from the shape of the resulting orbits. The manuscript concludes by highlighting how the identification of orthogonal molecular mechanisms for tempo control can inform the design of circuits with specific orbits and tempos.\nSpatial Transcriptomic Characterization of Novel Pathologic Niches in IPF\nAuthors: Mayr, C. H.; Santacruz, D.; Jarosch, S.; Lempp, C.; Neubert, L.; Rath, B.; Kamp, J. C.; Jonigk, D.; Kuehnel, M.; Schlueter, H.; Doerr, J.; Dick, A.; Ramirez, F.; Thomas, M. J.\nScore: 1.5, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571464\nAn unmet medical need persists in Idiopathic Pulmonary fibrosis (IPF), for which treatments additional to anti-fibrotic therapy are needed. Single cell RNA sequencing (scRNA-seq) has advanced our understanding of IPF with cell type-specific insights but lacks cellular tissue context. Spatial transcriptomics addresses this by providing spatially resolved gene expression, enabling gene and cell type localization within the tissue environment. We profiled IPF and control patient lung tissue sections using spatial transcriptomics and combined the data with an atlas of integrated IPF scRNA-seq datasets. Through computational analysis, we identified three disease-associated pathologic niches with unique cellular composition / localization and analyzed their cell-cell communication. We identified the Fibrotic niche, comprising Myofibroblasts and Aberrant Basaloid cells, preferentially located around airways and close to the Airway Macrophage niche in the lumen, containing SPP1+ Macrophages. We also identified the Immune niche, distinct foci of lymphoid cells in fibrotic tissue, surrounded by remodeled endothelial vessels. TEASERSpatial transcriptomics localizes genes and cell types in the tissue and identifies pathological cellular niches in IPF and control lungs.\nSingle-cell imaging of protein dynamics of paralogs reveals mechanisms of gene retention\nAuthors: Dandage, R.; Papkov, M.; Greco, B. M.; Fishman, D.; Friesen, H.; Wang, K.; Styles, E.; Kraus, O.; Grys, B.; Boone, C.; Andrews, B. J.; Parts, L.; Kuzmin, E.\nScore: 12.1, Published: 2023-11-23 DOI: 10.1101/2023.11.23.568466\nGene duplication is common across the tree of life, including yeast and humans, and contributes to genomic robustness. In this study, we examined changes in the subcellular localization and abundance of proteins in response to the deletion of their paralogs originating from the whole-genome duplication event, which is a largely unexplored mechanism of functional divergence. We performed a systematic single-cell imaging analysis of protein dynamics and screened subcellular redistribution of proteins, capturing their localization and abundance changes, providing insight into forces determining paralog retention. Paralogs showed dependency, whereby proteins required their paralog to maintain their native abundance or localization, more often than compensation. Network feature analysis suggested the importance of functional redundancy and rewiring of protein and genetic interactions underlying redistribution response of paralogs. Translation of non-canonical protein isoform emerged as a novel compensatory mechanism. This study provides new insights into paralog retention and evolutionary forces that shape genomes.\nHigh-throughput and scalable single cell proteomics identifies over 5000 proteins per cell\nAuthors: Ye, Z.; Sabatier, P.; Hoeven, L.; Phlairaharn, T.; Hartlmayr, D.; Izaguirre, F.; Seth, A.; Joshi, H.; Bekker-Jensen, D.; Bache, N.; Olsen, J. V.\nScore: 55.5, Published: 2023-11-28 DOI: 10.1101/2023.11.27.568953\nThe emergence of mass spectrometry (MS)-based single-cell proteomics (SCP) promise to revolutionize the study of cellular biology and biomedicine by providing an unparalleled view of the proteome in individual cells. Despite its groundbreaking potential, SCP is nascent and faces challenges including limited sequence depth, throughput, and reproducibility, which have constrained its broader utility. This study introduces key methodological advances, which considerably improve the sensitivity, coverage and dependability of protein identification from single cells. We developed an almost lossless SCP workflow encompassing sample preparation to MS analysis, doubling the number of identified proteins from roughly 2000 to over 5000 in individual HeLa cells. A comprehensive evaluation of analytical software tools, alongside strict false discovery rate (FDR) controls solidified the reliability of our results. These enhancements also facilitated the direct detection of post-translational modifications (PTMs) in single cells, negating the need for enrichment and thereby simplifying the analytical process. Although throughput in MS remains a challenge, our study demonstrates the feasibility of processing up to 80 label-free SCP samples per day. Moreover, an optimized tissue dissociation buffer enabled effective single cell disaggregation of drug-treated cancer cell spheroids, refining the overall proteomic analysis. Our workflow sets a new benchmark in SCP for sensitivity and throughput, with broad applications ranging from the study of cellular development to disease progression and the identification of cell type-specific markers and therapeutic targets.\n",
  "wordCount" : "2457",
  "inLanguage": "en",
  "datePublished": "2023-12-20T10:36:27Z",
  "dateModified": "2023-12-20T10:36:27Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 20, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.12.571279">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.12.571279" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.12.571279">
        <p class="paperTitle">Self-inhibiting percolation and viral spreading in epithelial tissue</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.12.571279" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.12.571279" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xu, X.; Nielsen, B. F.; sneppen, k.</p>
        <p class="info">Score: 23.9, Published: 2023-12-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.12.571279' target='https://doi.org/10.1101/2023.12.12.571279'> 10.1101/2023.12.12.571279</a></p>
        <p class="abstract">SARS-CoV-2 induces delayed type-I/III interferon production, allowing it to escape the early innate immune response. The delay has been attributed to a deficiency in the ability of cells to sense viral replication upon infection, which in turn hampers activation of the antiviral state in bystander cells. Here, we introduce a cellular automaton model to investigate the spatiotemporal spreading of viral infection as a function of virus and host-dependent parameters. The model suggests that the considerable person-to-person heterogeneity in SARS-CoV-2 infections is a consequence of high sensitivity to slight variations in biological parameters near a critical threshold. It further suggests that within-host viral proliferation can be curtailed by the presence of remarkably few cells that are primed for IFN production. Thus the observed heterogeneity in defense readiness of cells reflects a remarkably cost-efficient strategy for protection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571479">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571479" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571479">
        <p class="paperTitle">A distinctive evolution of alveolar T cell responses is associated with clinical outcomes in unvaccinated patients with SARS-CoV-2 pneumonia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571479" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571479" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Markov, N. S.; Ren, Z.; Senkow, K. J.; Grant, R. A.; Gao, C. A.; Malsin, E. S.; Sichizya, L.; Kihshen, H.; Helmin, K. A.; Jovisic, M.; Arnold, J. M.; Perez-Leonor, X. G.; Abdala-Valencia, H.; Swaminathan, S.; Nwaezeapu, J.; Kang, M.; Rasmussen, L.; Ozer, E. A.; Lorenzo-Redondo, R.; Hultquist, J. F.; Simons, L.; Rios-Guzman, E.; Misharin, A. V.; Wunderink, R. G.; Budinger, G. S.; Singer, B. D.; Morales-Nebreda, L.; The NU SCRIPT Investigators,  </p>
        <p class="info">Score: 6.8, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571479' target='https://doi.org/10.1101/2023.12.13.571479'> 10.1101/2023.12.13.571479</a></p>
        <p class="abstract">Pathogen clearance and resolution of inflammation in patients with pneumonia require an effective local T cell response. Nevertheless, local T cell activation may drive lung injury, particularly during prolonged episodes of respiratory failure characteristic of severe SARS-CoV-2 pneumonia. While T cell responses in the peripheral blood are well described, the evolution of T cell phenotypes and molecular signatures in the distal lung of patients with severe pneumonia caused by SARS-CoV-2 or other pathogens is understudied. Accordingly, we serially obtained 432 bronchoalveolar lavage fluid samples from 273 patients with severe pneumonia and respiratory failure, including 74 unvaccinated patients with COVID-19, and performed flow cytometry, transcriptional, and T cell receptor profiling on sorted CD8&#43; and CD4&#43; T cell subsets. In patients with COVID-19 but not pneumonia secondary to other pathogens, we found that early and persistent enrichment in CD8&#43; and CD4&#43; T cell subsets correlated with survival to hospital discharge. Activation of interferon signaling pathways early after intubation for COVID-19 was associated with favorable outcomes, while activation of NF-{kappa}B-driven programs late in disease was associated with poor outcomes. Patients with SARS-CoV-2 pneumonia whose alveolar T cells preferentially targeted the Spike and Nucleocapsid proteins tended to experience more favorable outcomes than patients whose T cells predominantly targeted the ORF1ab polyprotein complex. These results suggest that in patients with severe SARS-CoV-2 pneumonia, alveolar T cell interferon responses targeting structural SARS-CoV-2 proteins characterize patients who recover, yet these responses progress to NF-{kappa}B activation against non-structural proteins in patients who go on to experience poor clinical outcomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.03.569805">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.03.569805" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.03.569805">
        <p class="paperTitle">A foundational atlas of autism protein interactions reveals molecular convergence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.03.569805" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.03.569805" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, B.; Vartak, R.; Zaltsman, Y.; Naing, Z. Z. C.; Hennick, K. M.; Polacco, B. J.; Bashir, A.; Eckhardt, M.; Bouhaddou, M.; Xu, J.; Sun, N.; Lasser, M.; Zhou, Y.; Guiley, K. Z.; Chan, U.; Kaye, J. A.; Khare, P.; Drake, S.; Drury, V.; Burke, D. F.; Gonzalez, S.; Alkhairy, S.; Morris, M.; Baum, T.; Krasnoff, R.; Wang, S.; Pham, P.; Arbalaez, J.; Pratt, D.; Chag, S.; Rolland, T.; Bourgeron, T.; Finkbeiner, S.; Bandyopadhay, S.; Ideker, T.; Beltrao, P.; Willsey, H. R.; Obernier, K.; Nowakowski, T. J.; Huttenhain, R.; State, M. W.; Willsey, A. J.; Krogan, N. J.</p>
        <p class="info">Score: 57.1, Published: 2023-12-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.03.569805' target='https://doi.org/10.1101/2023.12.03.569805'> 10.1101/2023.12.03.569805</a></p>
        <p class="abstract">Despite the identification of a large number of high-confidence (hc) autism spectrum disorder (ASD) risk genes, translation to viable treatment targets remains elusive. Using affinity purification-mass spectrometry and HEK293T cells, we constructed a protein-protein interaction (PPI) network involving 100 hcASD risk genes, revealing over 1,800 PPIs, the vast majority of which have not been previously described. Importantly, the interactors are highly expressed in the human brain and neural progenitor cells, and enriched for ASD, but not schizophrenia, genetic risk. The network is highly connected, converges on protein complexes, including those involved in neurogenesis, tubulin biology, transcriptional regulation, and chromatin modification. We also generated a PPI map using over 50 patient-derived missense variants to identify differential physical interactions. Using these data, we leveraged AlphaFold2 predictions to prioritize direct PPIs as well as specific variants for interrogation in Xenopus tropicalis and human forebrain organoids. One such mutation in the transcription factor FOXP1, which disrupts interaction with FOXP4, led to a reconfiguration of DNA binding sites and disruption of neurogenesis, with a particularly strong impact on deep layer cortical and subplate neurons,consistent with a large body of work in ASD. Overall, this work offers critical new insights into molecular mechanisms underlying ASD that results from large effect genetic variants and describes a powerful platform to develop and test novel therapeutic strategies for a wide range of genetically-defined conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.11.570452">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.11.570452" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.11.570452">
        <p class="paperTitle">Broad H3K4me3 Domain Is Associated with Spatial Coherence during Mammalian Embryonic Development</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.11.570452" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.11.570452" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cao, X.; Ma, T.; Fan, R.; Yuan, G.-C.</p>
        <p class="info">Score: 11.2, Published: 2023-12-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.11.570452' target='https://doi.org/10.1101/2023.12.11.570452'> 10.1101/2023.12.11.570452</a></p>
        <p class="abstract">It is well known that the chromatin states play a major role in cell-fate decision and cell-identity maintenance; however, the spatial variation of chromatin states in situ remains poorly characterized. Here, by leveraging recently available spatial-CUT&amp;Tag data, we systematically characterized the global spatial organization of the H3K4me3 profiles in a mouse embryo. Our analysis identified a subset of genes with spatially coherent H3K4me3 patterns, which together delineate the tissue boundaries. The spatially coherent genes are strongly enriched with tissue-specific transcriptional regulators. Remarkably, their corresponding genomic loci are marked by broad H3K4me3 domains, which is distinct from the typical H3K4me3 signature. Spatial transition across tissue boundaries is associated with continuous shortening of the broad H3K4me3 domains as well as expansion of H3K27me3 domains. Our analysis reveals a strong connection between the genomic and spatial variation of chromatin states, which may play an important role in embryonic development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572198">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572198" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572198">
        <p class="paperTitle">Machine Learning and Metabolic Model Guided CRISPRi Reveals a Central Role for Phosphoglycerate Mutase in Chlamydia trachomatis Persistence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572198" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572198" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chowdhury, N. B.; Pokorzynski, N. D.; Rucks, E. A.; Ouellette, S. P.; CARABEO, R.; Saha, R.</p>
        <p class="info">Score: 1.8, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572198' target='https://doi.org/10.1101/2023.12.18.572198'> 10.1101/2023.12.18.572198</a></p>
        <p class="abstract">Upon nutrient starvation, Chlamydia trachomatis serovar L2 (CTL) shifts from its normal growth to a non-replicating form, termed persistence. It is unclear if persistence is an adaptive response or lack of it. To understand that transcriptomics data were collected for nutrient-sufficient and nutrient-starved CTL. Applying machine learning approaches on transcriptomics data revealed a global transcriptomic rewiring of CTL under stress conditions without having any global stress regulator. This indicated that CTL stress response is due to lack of an adaptive response mechanism. To investigate the impact of this on CTL metabolism, we reconstructed a genome-scale metabolic model of CTL (iCTL278) and contextualized it with the collected transcriptomics data. Using the metabolic bottleneck analysis on contextualized iCTL278, we observed phosphoglycerate mutase (pgm) regulates the entry of CTL to the persistence. Later, pgm was found to have the highest thermodynamics driving force and lowest enzymatic cost. Furthermore, CRISPRi-driven knockdown of pgm and tryptophan starvation experiments revealed the importance of this gene in inducing persistence. Hence, this work, for the first time, introduced thermodynamics and enzyme-cost as tools to gain deeper understanding on CTL persistence.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.03.28.486115">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.03.28.486115" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.03.28.486115">
        <p class="paperTitle">Profiling the proximal proteome of the mu opioid receptor identifies novel regulators of receptor signaling and trafficking</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.03.28.486115" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.03.28.486115" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Polacco, B. J.; Lobingier, B. T.; Blythe, E. E.; Abreu, N.; Khare, P.; Howard, M. K.; Gonzalez-Hernandez, A. J.; Xu, J.; Li, Q.; Novy, B.; Naing, Z. Z. C.; Shoichet, B. K.; Coyote-Maestas, W.; Levitz, J.; Krogan, N. J.; Von Zastrow, M.; Huttenhain, R.</p>
        <p class="info">Score: 29.4, Published: 2023-12-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.03.28.486115' target='https://doi.org/10.1101/2022.03.28.486115'> 10.1101/2022.03.28.486115</a></p>
        <p class="abstract">The mu opioid receptor (OR), a prototypic member of the large G protein-coupled receptor (GPCR) family, represents an important target of therapeutic and abused drugs. To date, most of our understanding of OR activity has focused on signal transducers and regulatory molecules including G proteins, GPCR kinases, and beta-arrestins. Yet it is clear that signaling through the OR is coordinated by additional proteins recruited into the proximal interaction network of the activated receptor, which have largely remained invisible given the lack of technologies to interrogate these networks systematically.

Here, we implement a quantitative proteomics pipeline leveraging the chemical diversity of OR agonists and APEX-based proximity labeling to investigate the protein networks that underlie OR signaling. We leverage a novel computational framework to extract subcellular location, trafficking, and functional partners of GPCR activity from the proximity labeling datasets. Applying this unbiased, systematic approach to the OR, we demonstrate that opioid agonists exert differences in the OR proximal proteome mediated by endocytosis and subsequent endosomal sorting, exemplified by VPS35 and COMMD3. Moreover, we identify two novel OR network components, EYA4 and KCTD12, that are recruited into the receptor proximal network irrespective of the activating ligand and independent of receptor trafficking but based on receptor-triggered G protein activation. We provide functional evidence that these network components form a previously unrecognized buffering system for G protein activity which broadly modulates cellular GPCR signaling.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572181">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572181" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572181">
        <p class="paperTitle">A Mathematical Framework for Measuring and Tuning Tempo in Developmental Gene Regulatory Networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572181" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572181" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Manser, C. L.; Perez-Carrasco, R.</p>
        <p class="info">Score: 1.6, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572181' target='https://doi.org/10.1101/2023.12.18.572181'> 10.1101/2023.12.18.572181</a></p>
        <p class="abstract">Embryo development is a dynamic process governed by the regulation of timing and sequences of gene expression, which control the proper growth of the organism. While many genetic programs coordinating these sequences are common across species, the timescales of gene expression can vary significantly among different organisms. Currently, substantial experimental efforts are focused on identifying molecular mechanisms that control these temporal aspects. In contrast, the capacity of established mathematical models to incorporate tempo control while maintaining the same dynamical landscape remains less understood. This manuscript addresses this gap by developing a mathematical framework that links the functionality of developmental programs to the corresponding gene expression orbits (or landscapes). This unlocks the ability to find tempo differences as perturbations in the dynamical system that preserve its orbits. We demonstrate that this framework allows for the prediction of molecular mechanisms governing tempo, through both numerical and analytical methods. Our exploration includes two case studies: a generic network featuring coupled production and degradation, and the repressilator. In the latter, we illustrate how altering the dimerisation rates of transcription factors can decouple the tempo from the shape of the resulting orbits. The manuscript concludes by highlighting how the identification of orthogonal molecular mechanisms for tempo control can inform the design of circuits with specific orbits and tempos.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571464">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571464" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571464">
        <p class="paperTitle">Spatial Transcriptomic Characterization of Novel Pathologic Niches in IPF</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571464" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571464" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mayr, C. H.; Santacruz, D.; Jarosch, S.; Lempp, C.; Neubert, L.; Rath, B.; Kamp, J. C.; Jonigk, D.; Kuehnel, M.; Schlueter, H.; Doerr, J.; Dick, A.; Ramirez, F.; Thomas, M. J.</p>
        <p class="info">Score: 1.5, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571464' target='https://doi.org/10.1101/2023.12.13.571464'> 10.1101/2023.12.13.571464</a></p>
        <p class="abstract">An unmet medical need persists in Idiopathic Pulmonary fibrosis (IPF), for which treatments additional to anti-fibrotic therapy are needed. Single cell RNA sequencing (scRNA-seq) has advanced our understanding of IPF with cell type-specific insights but lacks cellular tissue context. Spatial transcriptomics addresses this by providing spatially resolved gene expression, enabling gene and cell type localization within the tissue environment. We profiled IPF and control patient lung tissue sections using spatial transcriptomics and combined the data with an atlas of integrated IPF scRNA-seq datasets. Through computational analysis, we identified three disease-associated pathologic niches with unique cellular composition / localization and analyzed their cell-cell communication. We identified the Fibrotic niche, comprising Myofibroblasts and Aberrant Basaloid cells, preferentially located around airways and close to the Airway Macrophage niche in the lumen, containing SPP1&#43; Macrophages. We also identified the Immune niche, distinct foci of lymphoid cells in fibrotic tissue, surrounded by remodeled endothelial vessels.

TEASERSpatial transcriptomics localizes genes and cell types in the tissue and identifies pathological cellular niches in IPF and control lungs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.568466">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.568466" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.568466">
        <p class="paperTitle">Single-cell imaging of protein dynamics of paralogs reveals mechanisms of gene retention</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.568466" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.568466" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dandage, R.; Papkov, M.; Greco, B. M.; Fishman, D.; Friesen, H.; Wang, K.; Styles, E.; Kraus, O.; Grys, B.; Boone, C.; Andrews, B. J.; Parts, L.; Kuzmin, E.</p>
        <p class="info">Score: 12.1, Published: 2023-11-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.568466' target='https://doi.org/10.1101/2023.11.23.568466'> 10.1101/2023.11.23.568466</a></p>
        <p class="abstract">Gene duplication is common across the tree of life, including yeast and humans, and contributes to genomic robustness. In this study, we examined changes in the subcellular localization and abundance of proteins in response to the deletion of their paralogs originating from the whole-genome duplication event, which is a largely unexplored mechanism of functional divergence. We performed a systematic single-cell imaging analysis of protein dynamics and screened subcellular redistribution of proteins, capturing their localization and abundance changes, providing insight into forces determining paralog retention. Paralogs showed dependency, whereby proteins required their paralog to maintain their native abundance or localization, more often than compensation. Network feature analysis suggested the importance of functional redundancy and rewiring of protein and genetic interactions underlying redistribution response of paralogs. Translation of non-canonical protein isoform emerged as a novel compensatory mechanism. This study provides new insights into paralog retention and evolutionary forces that shape genomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.568953">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.568953" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.568953">
        <p class="paperTitle">High-throughput and scalable single cell proteomics identifies over 5000 proteins per cell</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.568953" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.568953" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ye, Z.; Sabatier, P.; Hoeven, L.; Phlairaharn, T.; Hartlmayr, D.; Izaguirre, F.; Seth, A.; Joshi, H.; Bekker-Jensen, D.; Bache, N.; Olsen, J. V.</p>
        <p class="info">Score: 55.5, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.568953' target='https://doi.org/10.1101/2023.11.27.568953'> 10.1101/2023.11.27.568953</a></p>
        <p class="abstract">The emergence of mass spectrometry (MS)-based single-cell proteomics (SCP) promise to revolutionize the study of cellular biology and biomedicine by providing an unparalleled view of the proteome in individual cells. Despite its groundbreaking potential, SCP is nascent and faces challenges including limited sequence depth, throughput, and reproducibility, which have constrained its broader utility. This study introduces key methodological advances, which considerably improve the sensitivity, coverage and dependability of protein identification from single cells. We developed an almost lossless SCP workflow encompassing sample preparation to MS analysis, doubling the number of identified proteins from roughly 2000 to over 5000 in individual HeLa cells. A comprehensive evaluation of analytical software tools, alongside strict false discovery rate (FDR) controls solidified the reliability of our results. These enhancements also facilitated the direct detection of post-translational modifications (PTMs) in single cells, negating the need for enrichment and thereby simplifying the analytical process. Although throughput in MS remains a challenge, our study demonstrates the feasibility of processing up to 80 label-free SCP samples per day. Moreover, an optimized tissue dissociation buffer enabled effective single cell disaggregation of drug-treated cancer cell spheroids, refining the overall proteomic analysis. Our workflow sets a new benchmark in SCP for sensitivity and throughput, with broad applications ranging from the study of cellular development to disease progression and the identification of cell type-specific markers and therapeutic targets.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
