<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A foundational atlas of autism protein interactions reveals molecular convergence
Authors: Wang, B.; Vartak, R.; Zaltsman, Y.; Naing, Z. Z. C.; Hennick, K. M.; Polacco, B. J.; Bashir, A.; Eckhardt, M.; Bouhaddou, M.; Xu, J.; Sun, N.; Lasser, M.; Zhou, Y.; Guiley, K. Z.; Chan, U.; Kaye, J. A.; Khare, P.; Drake, S.; Drury, V.; Burke, D. F.; Gonzalez, S.; Alkhairy, S.; Morris, M.; Baum, T.; Krasnoff, R.; Wang, S.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="A foundational atlas of autism protein interactions reveals molecular convergence
Authors: Wang, B.; Vartak, R.; Zaltsman, Y.; Naing, Z. Z. C.; Hennick, K. M.; Polacco, B. J.; Bashir, A.; Eckhardt, M.; Bouhaddou, M.; Xu, J.; Sun, N.; Lasser, M.; Zhou, Y.; Guiley, K. Z.; Chan, U.; Kaye, J. A.; Khare, P.; Drake, S.; Drury, V.; Burke, D. F.; Gonzalez, S.; Alkhairy, S.; Morris, M.; Baum, T.; Krasnoff, R.; Wang, S." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-10T10:37:48+00:00" />
<meta property="article:modified_time" content="2023-12-10T10:37:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="A foundational atlas of autism protein interactions reveals molecular convergence
Authors: Wang, B.; Vartak, R.; Zaltsman, Y.; Naing, Z. Z. C.; Hennick, K. M.; Polacco, B. J.; Bashir, A.; Eckhardt, M.; Bouhaddou, M.; Xu, J.; Sun, N.; Lasser, M.; Zhou, Y.; Guiley, K. Z.; Chan, U.; Kaye, J. A.; Khare, P.; Drake, S.; Drury, V.; Burke, D. F.; Gonzalez, S.; Alkhairy, S.; Morris, M.; Baum, T.; Krasnoff, R.; Wang, S."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "A foundational atlas of autism protein interactions reveals molecular convergence\nAuthors: Wang, B.; Vartak, R.; Zaltsman, Y.; Naing, Z. Z. C.; Hennick, K. M.; Polacco, B. J.; Bashir, A.; Eckhardt, M.; Bouhaddou, M.; Xu, J.; Sun, N.; Lasser, M.; Zhou, Y.; Guiley, K. Z.; Chan, U.; Kaye, J. A.; Khare, P.; Drake, S.; Drury, V.; Burke, D. F.; Gonzalez, S.; Alkhairy, S.; Morris, M.; Baum, T.; Krasnoff, R.; Wang, S.",
  "keywords": [
    
  ],
  "articleBody": " A foundational atlas of autism protein interactions reveals molecular convergence\nAuthors: Wang, B.; Vartak, R.; Zaltsman, Y.; Naing, Z. Z. C.; Hennick, K. M.; Polacco, B. J.; Bashir, A.; Eckhardt, M.; Bouhaddou, M.; Xu, J.; Sun, N.; Lasser, M.; Zhou, Y.; Guiley, K. Z.; Chan, U.; Kaye, J. A.; Khare, P.; Drake, S.; Drury, V.; Burke, D. F.; Gonzalez, S.; Alkhairy, S.; Morris, M.; Baum, T.; Krasnoff, R.; Wang, S.; Pham, P.; Arbalaez, J.; Pratt, D.; Chag, S.; Rolland, T.; Bourgeron, T.; Finkbeiner, S.; Bandyopadhay, S.; Ideker, T.; Beltrao, P.; Willsey, H. R.; Obernier, K.; Nowakowski, T. J.; Huttenhain, R.; State, M. W.; Willsey, A. J.; Krogan, N. J.\nScore: 56.2, Published: 2023-12-09 DOI: 10.1101/2023.12.03.569805\nDespite the identification of a large number of high-confidence (hc) autism spectrum disorder (ASD) risk genes, translation to viable treatment targets remains elusive. Using affinity purification-mass spectrometry and HEK293T cells, we constructed a protein-protein interaction (PPI) network involving 100 hcASD risk genes, revealing over 1,800 PPIs, the vast majority of which have not been previously described. Importantly, the interactors are highly expressed in the human brain and neural progenitor cells, and enriched for ASD, but not schizophrenia, genetic risk. The network is highly connected, converges on protein complexes, including those involved in neurogenesis, tubulin biology, transcriptional regulation, and chromatin modification. We also generated a PPI map using over 50 patient-derived missense variants to identify differential physical interactions. Using these data, we leveraged AlphaFold2 predictions to prioritize direct PPIs as well as specific variants for interrogation in Xenopus tropicalis and human forebrain organoids. One such mutation in the transcription factor FOXP1, which disrupts interaction with FOXP4, led to a reconfiguration of DNA binding sites and disruption of neurogenesis, with a particularly strong impact on deep layer cortical and subplate neurons,consistent with a large body of work in ASD. Overall, this work offers critical new insights into molecular mechanisms underlying ASD that results from large effect genetic variants and describes a powerful platform to develop and test novel therapeutic strategies for a wide range of genetically-defined conditions.\nRemodeling of the protein ubiquitylation landscape in the aging vertebrate brain\nAuthors: Marino, A.; Di Fraia, D.; Panfilova, D.; Sahu, A. K.; Ori, A.\nScore: 18.9, Published: 2023-12-04 DOI: 10.1101/2023.12.02.569713\nPost-translational modifications (PTMs) regulate protein homeostasis and function. How aging affects the landscape of PTMs remains largely elusive. Here, we reveal changes in hundreds of protein ubiquitylation, acetylation, and phosphorylation sites in the aging brain of mice. We show that aging has a major impact on protein ubiquitylation and that 29% of the ubiquitylation sites are affected independently of protein abundance, indicating altered PTM stoichiometry. We found a subset of these sites to be also affected in the brain of the short-lived killifish Nothobranchius furzeri, highlighting a conserved aging phenotype. Furthermore, we estimated that over 35% of ubiquitylation changes observed in old mouse brains derive from partial proteasome inhibition, a well-established hallmark of brain aging. Our findings provide evidence of an evolutionarily conserved ubiquitylation signature of the aging brain and establish a causal link between proteasome inhibition and age-related remodeling of the ubiquitylome.\nAn automated and fast sample preparation workflow for laser microdissection guided ultrasensitive proteomics\nAuthors: Makhmut, A.; Qin, D.; Hartlmayr, D.; Seth, A.; Coscia, F.\nScore: 14.9, Published: 2023-12-07 DOI: 10.1101/2023.11.29.569257\nSpatial tissue proteomics integrating whole-slide imaging, laser microdissection and ultrasensitive mass spectrometry is a powerful approach to link cellular phenotypes to functional proteome states in (patho)physiology. To be applicable to large patient cohorts and low sample input amounts, including single-cell applications, loss-minimized and streamlined end-to-end workflows are key. We here introduce an automated sample preparation protocol for laser microdissected samples utilizing the cellenONE(R) robotic system, which has the capacity to process 192 samples in three hours. Following laser microdissection collection directly into the proteoCHIP LF 48 or EVO 96 chip, our optimized protocol facilitates lysis, formalin de-crosslinking and tryptic digest of low-input archival tissue samples. The seamless integration with the Evosep ONE LC system by centrifugation allows on-the-fly sample clean-up, particularly pertinent for laser microdissected workflows. We validate our method in human tonsil archival tissue, where we profile proteomes of spatially-defined B-cell, T-cell and epithelial microregions of 4,000 {micro}m2 to a depth of [~]2,000 proteins and with high cell type specificity. We finally provide detailed equipment templates and experimental guidelines for broad accessibility.\npyRBDome: A comprehensive computational platform for enhancing and interpreting RNA-binding proteome data\nAuthors: Chu, L.-C.; Christopoulou, N.; McCaughan, H.; Winterbourne, S.; Cazzola, D.; Wang, S.; Litvin, U.; Brunon, S.; Harker, P. J. B.; McNae, I.; Granneman, S.\nScore: 10.0, Published: 2023-12-08 DOI: 10.1101/2023.12.08.570608\nHigh-throughput proteomics approaches have revolutionised the identification of RNA-binding proteins (RBPome) and RNA-binding sequences (RBDome) across organisms. Many novel putative RNA-binding proteins (RBPs) were discovered, including those that lack recognisable RNA-binding domains. Yet the extent of noise, including false-positive identifications, associated with these methodologies is difficult to quantify as experimental approaches for validating the results are generally low throughput. To address this, we introduce pyRBDome, a pipeline for in-depth in silico enhancement of RNA-binding proteome data. It does so by comparing experimental results with RNA-binding site (RBS) predictions from several distinct machine learning tools and integrates high-resolution structural data of protein-RNA complexes when available. By providing a statistical evaluation of RBDome data, users can rapidly identify protein sequences from RBDome experiments most likely to be bona fide RNA-binders. Furthermore, by leveraging the predictions collated by pyRBDome, we have enhanced the sensitivity and specificity of RBS detection through training new ensemble machine learning models. We describe a pyRBDome analysis of a large human RBDome dataset and conducted a comparision with know structural data. These analyses reinforced the significance of stacking interactions in UV cross-linking protein-RNA interactions. Surprisingly, our analyses revealed two contrasting findings: While UV cross-linked amino acids were more likely to contain predicted RBSs, they infrequently bind RNA in high-resolution structures. Given the known limitations of structural data as benchmarks, these finding highlights the utility of pyRBDome as a valuable alternative approach for enhancing confidence in RBDome datasets. Finally, our comprehensive analysis of hundreds of (putative) RBPs offers a valuable resource for RBP enthusiasts.\nSignalling-state dependent drug-tolerance in head and neck squamous cell carcinoma.\nAuthors: Karjosukarso, D. W.; Dini, A.; Wingens, L. J. A.; Liu, R.; Joosten, L. A. B.; Bussink, J.; Mulder, K. W.\nScore: 8.4, Published: 2023-12-07 DOI: 10.1101/2023.12.05.570063\nIntratumor heterogeneity negatively impacts therapeutic response and patient prognosis. Besides the established role of genetic heterogeneity, non-genetic mechanisms of persistence to drug treatment are emerging. Here, we characterise cells selected for their persistence to control, epidermal growth factor inhibition (EGFRi), radiation and combined treatment from low passage head and neck squamous cell carcinoma (HNSCC) cultures. Using a panel of 70 (phospho-)specific DNA-conjugated antibodies we measured activities of 8 signalling pathways, self-renewal, differentiation, DNA damage and cell-cycle, in conjunction with the transcriptional output in single cells, using our RNA and Immuno-Detection (RAID) technology. Six recurrent transcriptional programs reflecting processes including proliferation, differentiation and metabolic activity, as well as protein-based signalling-states, were associated with drug persistence, while copy number variation inference indicated involvement of non-genetic tolerance mechanisms. Projecting RNA velocity onto the antibody-derived signalling-states suggested a key role for integrin-mediated focal-adhesion signalling in drug-persistence in our cell system. Using machine-learning we derived a core transcriptional signature connected to adhesion-based drug-persistence, which was predictive of poor prognosis in a TGCA HNSCC cohort (hazard-ratio 1.87, p\u003c10-5). Furthermore, functional analyses confirmed that cells expressing high levels of integrin alpha-6 (ITGA6) were tolerant to EGFRi treatment, and that forcing cells out of this cell-state through transient targeted inhibition of Focal Adhesion Kinase activity re-instated EGFRi sensitivity in drug persistent cells. Taken together, our single-cell multi-omics analysis identified an actionable adhesion-signalling mediated cell-state driving drug tolerance in HNSCC.\nEpistemic uncertainty challenges aging clock reliability in predicting rejuvenation effects\nAuthors: Kriukov, D.; Kuzmina, E. A.; Efimov, E.; Dylov, D. V.; Khrameeva, E.\nScore: 5.3, Published: 2023-12-04 DOI: 10.1101/2023.12.01.569529\nEpigenetic aging clocks have been widely used to validate rejuvenation effects during cellular reprogramming. However, these predictions are unfalsifiable, since the true biological age of reprogrammed cells remains inaccessible. We present a multifaceted analytical framework to consider rejuvenation predictions from the uncertainty perspective. We discover that DNA methylation profiles of reprogramming are not represented in the aging data used for clock training, which introduces high epistemic uncertainty in aging predictions. Moreover, predictions of different published clocks are poorly consistent with each other and suggest even zero or negative rejuvenation. We show that the high prediction uncertainty challenges the reliability of rejuvenation effects observed during in vitro reprogramming prior to pluripotency and throughout embryogenesis. Conversely, our method also reveals a significant age increase after in vivo reprogramming. We propose to include uncertainty estimation in future aging clocks to avoid the risk of misinterpreting the results of biological age prediction.\nDifferential proteome diversification in yeast populations: modes of short-term adaptation and fitness outcomes\nAuthors: Knafo, M.; Rezenman, S.; Tsigalnitski, I.; Nevo, R.; Reich, Z.; Kapon, R.\nScore: 5.1, Published: 2023-12-07 DOI: 10.1101/2023.11.28.568995\nShort-term proteomic adaptations serve as an initial line of defence, allowing populations to cope with environmental changes before long-term genetic alterations occur. Using a representative set of genes, we examined how stress affects gene expression variability for different types and levels of abiotic stresses and how this influences population-level adaptation. Our data reveal that, depending on the nature of the stress, two distinct modes of response can be employed. In one, the levels of most proteins vary between individuals, leading to varied fitness levels in the population. In the other, a more limited range of expression is seen, and fitness is more even. This suggests different levels of complexity and plasticity in adaptation to different types of stress.\nPredicting Phenotypic Traits Using a Massive RNA-seq Dataset\nAuthors: Hadish, J. A.; Honaas, L. A.; Ficklin, S. P.\nScore: 3.2, Published: 2023-12-07 DOI: 10.1101/2023.12.05.570195\nTranscriptomic data can be used to predict environmentally impacted phenotypic traits. This type of prediction is particularly useful for monitoring difficult-to-measure phenotypic traits and has become increasingly popular for monitoring high-value agricultural crops and in precision medicine. Despite this increase in popularity, little research has been done on how many samples are required for these models to be accurate, and which normalization should be used. Here we create a massive RNA-seq dataset from publicly available Arabidopsis thaliana data with corresponding measurements for age and tissue type. We use this dataset to determine how many samples are required for accurate model prediction and which normalization method is required. We find that Median Ratios Normalization significantly increases performance when predicting age. We also find that in the case of our dataset, only a few hundred samples are required to predict tissue types, and only a few thousand samples are necessary to accurately predict age. Researchers should consider these results when choosing the number of samples in a transcriptomic experiment and during data-processing.\nAn evolution-based framework for describing human gut bacteria\nAuthors: Doran, B. A.; Chen, R. Y.; Giba, H.; Behera, V.; Barat, B.; Sundararajan, A.; Lin, H.; Sidebottom, A.; Pamer, E. G.; Raman, A.\nScore: 3.0, Published: 2023-12-05 DOI: 10.1101/2023.12.04.569969\nThe human gut microbiome contains many bacterial strains of the same species ( strain-level variants). Describing strains in a biologically meaningful way rather than purely taxonomically is an important goal but challenging due to the genetic complexity of strain-level variation. Here, we measured patterns of co-evolution across \u003e7,000 strains spanning the bacterial tree-of-life. Using these patterns as a prior for studying hundreds of gut commensal strains that we isolated, sequenced, and metabolically profiled revealed widespread structure beneath the phylogenetic level of species. Defining strains by their co-evolutionary signatures enabled predicting their metabolic phenotypes and engineering consortia from strain genome content alone. Our findings demonstrate a biologically relevant organization to strain-level variation and motivate a new schema for describing bacterial strains based on their evolutionary history. One Sentence SummaryDescribing bacterial strains in the human gut by a statistical model that captures their evolutionary history provides insight into their biology.\nA bovine pulmosphere model and multiomics analyses identify a signature of early host response to Mycobacterium tuberculosis infection\nAuthors: Bhaskar, V.; Kumar, R.; Praharaj, M. R.; Gandham, S.; Maity, H. K.; Sarkar, U.; Dey, B.\nScore: 2.2, Published: 2023-12-08 DOI: 10.1101/2023.12.07.570553\nInteractions between the tubercle bacilli and lung cells during the early stages of tuberculosis (TB) are crucial for disease outcomes. Conventional 2D cell culture inadequately replicates the multicellular complexity of lungs. We introduce a 3D pulmosphere model for Mycobacterium tuberculosis infection in bovine systems, demonstrating through comprehensive transcriptome and proteome analyses that these 3D structures closely replicate the diverse cell populations and abundant extracellular matrix proteins, emphasizing their similarity to the in vivo pulmonary environment. While both avirulent BCG and virulent M. tuberculosis-infected pulmospheres exhibit commonalities in the upregulation of several host signaling pathways, distinct features such as upregulation of ECM receptors, neutrophil chemotaxis, interferon signaling, and RIG-1 signaling pathways characterize the unique early response to virulent M. tuberculosis. Moreover, a signature of seven genes/proteins, including IRF1, CCL5, CXCL8, CXCL10, ICAM1, COL17A1, and CFB, emerges as indicative of the early host response to M. tuberculosis infection. Overall, this study presents a superior ex vivo multicellular bovine pulmosphere TB model, with implications for discovering disease biomarkers, enabling high-throughput drug screening, and improving TB control strategies.\n",
  "wordCount" : "2203",
  "inLanguage": "en",
  "datePublished": "2023-12-10T10:37:48Z",
  "dateModified": "2023-12-10T10:37:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 10, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.03.569805">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.03.569805" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.03.569805">
        <p class="paperTitle">A foundational atlas of autism protein interactions reveals molecular convergence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.03.569805" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.03.569805" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, B.; Vartak, R.; Zaltsman, Y.; Naing, Z. Z. C.; Hennick, K. M.; Polacco, B. J.; Bashir, A.; Eckhardt, M.; Bouhaddou, M.; Xu, J.; Sun, N.; Lasser, M.; Zhou, Y.; Guiley, K. Z.; Chan, U.; Kaye, J. A.; Khare, P.; Drake, S.; Drury, V.; Burke, D. F.; Gonzalez, S.; Alkhairy, S.; Morris, M.; Baum, T.; Krasnoff, R.; Wang, S.; Pham, P.; Arbalaez, J.; Pratt, D.; Chag, S.; Rolland, T.; Bourgeron, T.; Finkbeiner, S.; Bandyopadhay, S.; Ideker, T.; Beltrao, P.; Willsey, H. R.; Obernier, K.; Nowakowski, T. J.; Huttenhain, R.; State, M. W.; Willsey, A. J.; Krogan, N. J.</p>
        <p class="info">Score: 56.2, Published: 2023-12-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.03.569805' target='https://doi.org/10.1101/2023.12.03.569805'> 10.1101/2023.12.03.569805</a></p>
        <p class="abstract">Despite the identification of a large number of high-confidence (hc) autism spectrum disorder (ASD) risk genes, translation to viable treatment targets remains elusive. Using affinity purification-mass spectrometry and HEK293T cells, we constructed a protein-protein interaction (PPI) network involving 100 hcASD risk genes, revealing over 1,800 PPIs, the vast majority of which have not been previously described. Importantly, the interactors are highly expressed in the human brain and neural progenitor cells, and enriched for ASD, but not schizophrenia, genetic risk. The network is highly connected, converges on protein complexes, including those involved in neurogenesis, tubulin biology, transcriptional regulation, and chromatin modification. We also generated a PPI map using over 50 patient-derived missense variants to identify differential physical interactions. Using these data, we leveraged AlphaFold2 predictions to prioritize direct PPIs as well as specific variants for interrogation in Xenopus tropicalis and human forebrain organoids. One such mutation in the transcription factor FOXP1, which disrupts interaction with FOXP4, led to a reconfiguration of DNA binding sites and disruption of neurogenesis, with a particularly strong impact on deep layer cortical and subplate neurons,consistent with a large body of work in ASD. Overall, this work offers critical new insights into molecular mechanisms underlying ASD that results from large effect genetic variants and describes a powerful platform to develop and test novel therapeutic strategies for a wide range of genetically-defined conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.02.569713">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.02.569713" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.02.569713">
        <p class="paperTitle">Remodeling of the protein ubiquitylation landscape in the aging vertebrate brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.02.569713" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.02.569713" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Marino, A.; Di Fraia, D.; Panfilova, D.; Sahu, A. K.; Ori, A.</p>
        <p class="info">Score: 18.9, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.02.569713' target='https://doi.org/10.1101/2023.12.02.569713'> 10.1101/2023.12.02.569713</a></p>
        <p class="abstract">Post-translational modifications (PTMs) regulate protein homeostasis and function. How aging affects the landscape of PTMs remains largely elusive. Here, we reveal changes in hundreds of protein ubiquitylation, acetylation, and phosphorylation sites in the aging brain of mice. We show that aging has a major impact on protein ubiquitylation and that 29% of the ubiquitylation sites are affected independently of protein abundance, indicating altered PTM stoichiometry. We found a subset of these sites to be also affected in the brain of the short-lived killifish Nothobranchius furzeri, highlighting a conserved aging phenotype. Furthermore, we estimated that over 35% of ubiquitylation changes observed in old mouse brains derive from partial proteasome inhibition, a well-established hallmark of brain aging. Our findings provide evidence of an evolutionarily conserved ubiquitylation signature of the aging brain and establish a causal link between proteasome inhibition and age-related remodeling of the ubiquitylome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569257">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569257" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569257">
        <p class="paperTitle">An automated and fast sample preparation workflow for laser microdissection guided ultrasensitive proteomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569257" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569257" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Makhmut, A.; Qin, D.; Hartlmayr, D.; Seth, A.; Coscia, F.</p>
        <p class="info">Score: 14.9, Published: 2023-12-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569257' target='https://doi.org/10.1101/2023.11.29.569257'> 10.1101/2023.11.29.569257</a></p>
        <p class="abstract">Spatial tissue proteomics integrating whole-slide imaging, laser microdissection and ultrasensitive mass spectrometry is a powerful approach to link cellular phenotypes to functional proteome states in (patho)physiology. To be applicable to large patient cohorts and low sample input amounts, including single-cell applications, loss-minimized and streamlined end-to-end workflows are key. We here introduce an automated sample preparation protocol for laser microdissected samples utilizing the cellenONE(R) robotic system, which has the capacity to process 192 samples in three hours. Following laser microdissection collection directly into the proteoCHIP LF 48 or EVO 96 chip, our optimized protocol facilitates lysis, formalin de-crosslinking and tryptic digest of low-input archival tissue samples. The seamless integration with the Evosep ONE LC system by centrifugation allows  on-the-fly sample clean-up, particularly pertinent for laser microdissected workflows. We validate our method in human tonsil archival tissue, where we profile proteomes of spatially-defined B-cell, T-cell and epithelial microregions of 4,000 {micro}m2 to a depth of [~]2,000 proteins and with high cell type specificity. We finally provide detailed equipment templates and experimental guidelines for broad accessibility.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.570608">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.570608" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.570608">
        <p class="paperTitle">pyRBDome: A comprehensive computational platform for enhancing and interpreting RNA-binding proteome data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.570608" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.570608" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chu, L.-C.; Christopoulou, N.; McCaughan, H.; Winterbourne, S.; Cazzola, D.; Wang, S.; Litvin, U.; Brunon, S.; Harker, P. J. B.; McNae, I.; Granneman, S.</p>
        <p class="info">Score: 10.0, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.570608' target='https://doi.org/10.1101/2023.12.08.570608'> 10.1101/2023.12.08.570608</a></p>
        <p class="abstract">High-throughput proteomics approaches have revolutionised the identification of RNA-binding proteins (RBPome) and RNA-binding sequences (RBDome) across organisms. Many novel putative RNA-binding proteins (RBPs) were discovered, including those that lack recognisable RNA-binding domains. Yet the extent of noise, including false-positive identifications, associated with these methodologies is difficult to quantify as experimental approaches for validating the results are generally low throughput. To address this, we introduce pyRBDome, a pipeline for in-depth in silico enhancement of RNA-binding proteome data. It does so by comparing experimental results with RNA-binding site (RBS) predictions from several distinct machine learning tools and integrates high-resolution structural data of protein-RNA complexes when available. By providing a statistical evaluation of RBDome data, users can rapidly identify protein sequences from RBDome experiments most likely to be bona fide RNA-binders. Furthermore, by leveraging the predictions collated by pyRBDome, we have enhanced the sensitivity and specificity of RBS detection through training new ensemble machine learning models. We describe a pyRBDome analysis of a large human RBDome dataset and conducted a comparision with know structural data. These analyses reinforced the significance of stacking interactions in UV cross-linking protein-RNA interactions. Surprisingly, our analyses revealed two contrasting findings: While UV cross-linked amino acids were more likely to contain predicted RBSs, they infrequently bind RNA in high-resolution structures. Given the known limitations of structural data as benchmarks, these finding highlights the utility of pyRBDome as a valuable alternative approach for enhancing confidence in RBDome datasets. Finally, our comprehensive analysis of hundreds of (putative) RBPs offers a valuable resource for RBP enthusiasts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.570063">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.570063" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.570063">
        <p class="paperTitle">Signalling-state dependent drug-tolerance in head and neck squamous cell carcinoma.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.570063" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.570063" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Karjosukarso, D. W.; Dini, A.; Wingens, L. J. A.; Liu, R.; Joosten, L. A. B.; Bussink, J.; Mulder, K. W.</p>
        <p class="info">Score: 8.4, Published: 2023-12-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.570063' target='https://doi.org/10.1101/2023.12.05.570063'> 10.1101/2023.12.05.570063</a></p>
        <p class="abstract">Intratumor heterogeneity negatively impacts therapeutic response and patient prognosis. Besides the established role of genetic heterogeneity, non-genetic mechanisms of persistence to drug treatment are emerging. Here, we characterise cells selected for their persistence to control, epidermal growth factor inhibition (EGFRi), radiation and combined treatment from low passage head and neck squamous cell carcinoma (HNSCC) cultures. Using a panel of 70 (phospho-)specific DNA-conjugated antibodies we measured activities of 8 signalling pathways, self-renewal, differentiation, DNA damage and cell-cycle, in conjunction with the transcriptional output in single cells, using our RNA and Immuno-Detection (RAID) technology. Six recurrent transcriptional programs reflecting processes including proliferation, differentiation and metabolic activity, as well as protein-based signalling-states, were associated with drug persistence, while copy number variation inference indicated involvement of non-genetic tolerance mechanisms. Projecting RNA velocity onto the antibody-derived signalling-states suggested a key role for integrin-mediated focal-adhesion signalling in drug-persistence in our cell system. Using machine-learning we derived a core transcriptional signature connected to adhesion-based drug-persistence, which was predictive of poor prognosis in a TGCA HNSCC cohort (hazard-ratio 1.87, p&lt;10-5). Furthermore, functional analyses confirmed that cells expressing high levels of integrin alpha-6 (ITGA6) were tolerant to EGFRi treatment, and that forcing cells out of this cell-state through transient targeted inhibition of Focal Adhesion Kinase activity re-instated EGFRi sensitivity in drug persistent cells. Taken together, our single-cell multi-omics analysis identified an actionable adhesion-signalling mediated cell-state driving drug tolerance in HNSCC.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569529">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569529" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569529">
        <p class="paperTitle">Epistemic uncertainty challenges aging clock reliability in predicting rejuvenation effects</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569529" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569529" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kriukov, D.; Kuzmina, E. A.; Efimov, E.; Dylov, D. V.; Khrameeva, E.</p>
        <p class="info">Score: 5.3, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569529' target='https://doi.org/10.1101/2023.12.01.569529'> 10.1101/2023.12.01.569529</a></p>
        <p class="abstract">Epigenetic aging clocks have been widely used to validate rejuvenation effects during cellular reprogramming. However, these predictions are unfalsifiable, since the true biological age of reprogrammed cells remains inaccessible. We present a multifaceted analytical framework to consider rejuvenation predictions from the uncertainty perspective. We discover that DNA methylation profiles of reprogramming are not represented in the aging data used for clock training, which introduces high epistemic uncertainty in aging predictions. Moreover, predictions of different published clocks are poorly consistent with each other and suggest even zero or negative rejuvenation. We show that the high prediction uncertainty challenges the reliability of rejuvenation effects observed during in vitro reprogramming prior to pluripotency and throughout embryogenesis. Conversely, our method also reveals a significant age increase after in vivo reprogramming. We propose to include uncertainty estimation in future aging clocks to avoid the risk of misinterpreting the results of biological age prediction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.568995">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.568995" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.568995">
        <p class="paperTitle">Differential proteome diversification in yeast populations: modes of short-term adaptation and fitness outcomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.568995" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.568995" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Knafo, M.; Rezenman, S.; Tsigalnitski, I.; Nevo, R.; Reich, Z.; Kapon, R.</p>
        <p class="info">Score: 5.1, Published: 2023-12-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.568995' target='https://doi.org/10.1101/2023.11.28.568995'> 10.1101/2023.11.28.568995</a></p>
        <p class="abstract">Short-term proteomic adaptations serve as an initial line of defence, allowing populations to cope with environmental changes before long-term genetic alterations occur. Using a representative set of genes, we examined how stress affects gene expression variability for different types and levels of abiotic stresses and how this influences population-level adaptation. Our data reveal that, depending on the nature of the stress, two distinct modes of response can be employed. In one, the levels of most proteins vary between individuals, leading to varied fitness levels in the population. In the other, a more limited range of expression is seen, and fitness is more even. This suggests different levels of complexity and plasticity in adaptation to different types of stress.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.570195">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.570195" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.570195">
        <p class="paperTitle">Predicting Phenotypic Traits Using a Massive RNA-seq Dataset</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.570195" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.570195" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hadish, J. A.; Honaas, L. A.; Ficklin, S. P.</p>
        <p class="info">Score: 3.2, Published: 2023-12-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.570195' target='https://doi.org/10.1101/2023.12.05.570195'> 10.1101/2023.12.05.570195</a></p>
        <p class="abstract">Transcriptomic data can be used to predict environmentally impacted phenotypic traits. This type of prediction is particularly useful for monitoring difficult-to-measure phenotypic traits and has become increasingly popular for monitoring high-value agricultural crops and in precision medicine. Despite this increase in popularity, little research has been done on how many samples are required for these models to be accurate, and which normalization should be used. Here we create a massive RNA-seq dataset from publicly available Arabidopsis thaliana data with corresponding measurements for age and tissue type. We use this dataset to determine how many samples are required for accurate model prediction and which normalization method is required. We find that Median Ratios Normalization significantly increases performance when predicting age. We also find that in the case of our dataset, only a few hundred samples are required to predict tissue types, and only a few thousand samples are necessary to accurately predict age. Researchers should consider these results when choosing the number of samples in a transcriptomic experiment and during data-processing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.04.569969">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.04.569969" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.04.569969">
        <p class="paperTitle">An evolution-based framework for describing human gut bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.04.569969" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.04.569969" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Doran, B. A.; Chen, R. Y.; Giba, H.; Behera, V.; Barat, B.; Sundararajan, A.; Lin, H.; Sidebottom, A.; Pamer, E. G.; Raman, A.</p>
        <p class="info">Score: 3.0, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.04.569969' target='https://doi.org/10.1101/2023.12.04.569969'> 10.1101/2023.12.04.569969</a></p>
        <p class="abstract">The human gut microbiome contains many bacterial strains of the same species ( strain-level variants). Describing strains in a biologically meaningful way rather than purely taxonomically is an important goal but challenging due to the genetic complexity of strain-level variation. Here, we measured patterns of co-evolution across &gt;7,000 strains spanning the bacterial tree-of-life. Using these patterns as a prior for studying hundreds of gut commensal strains that we isolated, sequenced, and metabolically profiled revealed widespread structure beneath the phylogenetic level of species. Defining strains by their co-evolutionary signatures enabled predicting their metabolic phenotypes and engineering consortia from strain genome content alone. Our findings demonstrate a biologically relevant organization to strain-level variation and motivate a new schema for describing bacterial strains based on their evolutionary history.

One Sentence SummaryDescribing bacterial strains in the human gut by a statistical model that captures their evolutionary history provides insight into their biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570553">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570553" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570553">
        <p class="paperTitle">A bovine pulmosphere model and multiomics analyses identify a signature of early host response to Mycobacterium tuberculosis infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570553" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570553" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bhaskar, V.; Kumar, R.; Praharaj, M. R.; Gandham, S.; Maity, H. K.; Sarkar, U.; Dey, B.</p>
        <p class="info">Score: 2.2, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570553' target='https://doi.org/10.1101/2023.12.07.570553'> 10.1101/2023.12.07.570553</a></p>
        <p class="abstract">Interactions between the tubercle bacilli and lung cells during the early stages of tuberculosis (TB) are crucial for disease outcomes. Conventional 2D cell culture inadequately replicates the multicellular complexity of lungs. We introduce a 3D pulmosphere model for Mycobacterium tuberculosis infection in bovine systems, demonstrating through comprehensive transcriptome and proteome analyses that these 3D structures closely replicate the diverse cell populations and abundant extracellular matrix proteins, emphasizing their similarity to the in vivo pulmonary environment. While both avirulent BCG and virulent M. tuberculosis-infected pulmospheres exhibit commonalities in the upregulation of several host signaling pathways, distinct features such as upregulation of ECM receptors, neutrophil chemotaxis, interferon signaling, and RIG-1 signaling pathways characterize the unique early response to virulent M. tuberculosis. Moreover, a signature of seven genes/proteins, including IRF1, CCL5, CXCL8, CXCL10, ICAM1, COL17A1, and CFB, emerges as indicative of the early host response to M. tuberculosis infection. Overall, this study presents a superior ex vivo multicellular bovine pulmosphere TB model, with implications for discovering disease biomarkers, enabling high-throughput drug screening, and improving TB control strategies.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
