<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Systematic discovery of protein interaction interfaces using AlphaFold and experimental validation
Authors: Lee, C. Y.; Hubrich, D.; Varga, J. K.; Schäfer, C.; Welzel, M.; Schumbera, E.; Đokic, M.; Strom, J. M.; Schönfeld, J.; Geist, J. L.; Polat, F.; Gibson, T. J.; Keller Valsecchi, C. I.; Kumar, M.; Schueler-Furman, O.; Luck, K.
Score: 46.1, Published: 2023-08-09 DOI: 10.1101/2023.08.07.552219
Structural resolution of protein interactions enables mechanistic and functional studies as well as interpretation of disease variants.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Systematic discovery of protein interaction interfaces using AlphaFold and experimental validation
Authors: Lee, C. Y.; Hubrich, D.; Varga, J. K.; Schäfer, C.; Welzel, M.; Schumbera, E.; Đokic, M.; Strom, J. M.; Schönfeld, J.; Geist, J. L.; Polat, F.; Gibson, T. J.; Keller Valsecchi, C. I.; Kumar, M.; Schueler-Furman, O.; Luck, K.
Score: 46.1, Published: 2023-08-09 DOI: 10.1101/2023.08.07.552219
Structural resolution of protein interactions enables mechanistic and functional studies as well as interpretation of disease variants." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-16T10:37:20+00:00" />
<meta property="article:modified_time" content="2023-08-16T10:37:20+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Systematic discovery of protein interaction interfaces using AlphaFold and experimental validation
Authors: Lee, C. Y.; Hubrich, D.; Varga, J. K.; Schäfer, C.; Welzel, M.; Schumbera, E.; Đokic, M.; Strom, J. M.; Schönfeld, J.; Geist, J. L.; Polat, F.; Gibson, T. J.; Keller Valsecchi, C. I.; Kumar, M.; Schueler-Furman, O.; Luck, K.
Score: 46.1, Published: 2023-08-09 DOI: 10.1101/2023.08.07.552219
Structural resolution of protein interactions enables mechanistic and functional studies as well as interpretation of disease variants."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Systematic discovery of protein interaction interfaces using AlphaFold and experimental validation\nAuthors: Lee, C. Y.; Hubrich, D.; Varga, J. K.; Schäfer, C.; Welzel, M.; Schumbera, E.; Đokic, M.; Strom, J. M.; Schönfeld, J.; Geist, J. L.; Polat, F.; Gibson, T. J.; Keller Valsecchi, C. I.; Kumar, M.; Schueler-Furman, O.; Luck, K.\nScore: 46.1, Published: 2023-08-09 DOI: 10.1101/2023.08.07.552219\nStructural resolution of protein interactions enables mechanistic and functional studies as well as interpretation of disease variants.",
  "keywords": [
    
  ],
  "articleBody": " Systematic discovery of protein interaction interfaces using AlphaFold and experimental validation\nAuthors: Lee, C. Y.; Hubrich, D.; Varga, J. K.; Schäfer, C.; Welzel, M.; Schumbera, E.; Đokic, M.; Strom, J. M.; Schönfeld, J.; Geist, J. L.; Polat, F.; Gibson, T. J.; Keller Valsecchi, C. I.; Kumar, M.; Schueler-Furman, O.; Luck, K.\nScore: 46.1, Published: 2023-08-09 DOI: 10.1101/2023.08.07.552219\nStructural resolution of protein interactions enables mechanistic and functional studies as well as interpretation of disease variants. However, structural data is still missing for most protein interactions because we lack computational and experimental tools at scale. We thoroughly assessed AlphaFold-Multimer accuracy for structure prediction of interactions involving folded domains binding to short linear motifs from the ELM database. The structure predictions were highly sensitive but not very specific when using small protein fragments. Sensitivity decreased substantially when using long protein fragments or full length proteins with intrinsically disordered regions. We delineated a fragmentation strategy to optimize sensitivity and applied it to interactions between proteins associated with neurodevelopmental disorders. This enabled prediction of highly confident and likely disease-related novel interfaces, but also resulted in many high scoring false positive predictions. Experiments supported predicted interfaces between CREBZF-HCFC1, FBXO23-STX1B, STX1B-VAMP2, ESRRG-PSMC5, PEX3-PEX19, PEX3-PEX16, and SNRPB-GIGYF1 providing novel molecular insights for diverse biological processes. Our work highlights exciting perspectives, but also reveals clear limitations and the need for future developments to maximize the power of Alphafold-Multimer for interface predictions.\nSystems-ecology designed bacterial consortium protects from severe Clostridioides difficile infection\nAuthors: Jenior, M. L.; Leslie, J. L.; Kolling, G. L.; Archbald-Pannone, L.; Powers, D. A.; Petri, W. A.; Papin, J. A.\nScore: 5.9, Published: 2023-08-10 DOI: 10.1101/2023.08.08.552483\nFecal Microbiota Transplant (FMT) is an emerging therapy that has had remarkable success in treatment and prevention of recurrent Clostridioides difficile infection (rCDI). FMT has recently been associated with adverse outcomes such as inadvertent transfer of antimicrobial resistance, necessitating development of more targeted bacteriotherapies. To address this challenge, we developed a novel systems biology pipeline to identify candidate probiotic strains that would be predicted to interrupt C. difficile pathogenesis. Utilizing metagenomic characterization of human FMT donor samples, we identified those metabolic pathways most associated with successful FMTs and reconstructed the metabolism of encoding species to simulate interactions with C. difficile. This analysis resulted in predictions of high levels of cross-feeding for amino acids in species most associated with FMT success. Guided by these in silico models, we assembled consortia of bacteria with increased amino acid cross-feeding which were then validated in vitro. We subsequently tested the consortia in a murine model of CDI, demonstrating total protection from severe CDI through decreased toxin levels, recovered gut microbiota, and increased intestinal eosinophils. These results support the novel framework that amino acid cross-feeding is likely a critical mechanism in the initial resolution of CDI by FMT. Importantly, we conclude that our predictive platform based on predicted and testable metabolic interactions between the microbiota and C. difficile led to a rationally designed biotherapeutic framework that may be extended to other enteric infections.\nTwo-way feedback between chromatin compaction and histone modification state explains S. cerevisiae heterochromatin bistability\nAuthors: Movilla Miangolarra, A.; Saxton, D. S.; Yan, Z.; Rine, J.; Howard, M.\nScore: 2.1, Published: 2023-08-15 DOI: 10.1101/2023.08.12.552948\nCompact chromatin is closely linked with gene silencing in part by sterically masking access to promoters, inhibiting transcription factor binding and preventing polymerase from efficiently transcribing a gene. Here, we propose a broader view: chromatin compaction can be both a cause and a consequence of the histone modification state, and this tight bidirectional interaction can underpin bistable transcriptional states. To test this theory, we developed a mathematical model for the dynamics of the HMR locus in S. cerevisiae, that incorporates activating histone modifications, silencing proteins and a dynamic, acetylation-dependent, three-dimensional locus size. Chromatin compaction enhances silencer protein binding, which in turn feeds back to remove activating histone modifications, leading to further compaction. The bistable output of the model was in good agreement with prior quantitative data, including switching rates from expressed to silent states, and vice versa, and protein binding levels within the locus. We then tested the model by predicting changes in switching rates as the genetic length of the locus was increased, which were then experimentally verified. This bidirectional feedback between chromatin compaction and the histone modification state may be an important regulatory mechanism at many loci.\nCURTAIN - A Unique Web-based tool for exploration and sharing of MS-based proteomics data\nAuthors: Phung, T. K.; Berndsen, K.; Phan, T. L.; Muqit, M. M.; Alessi, D. R.; Nirujogi, R. S.\nScore: 27.5, Published: 2023-07-25 DOI: 10.1101/2023.07.25.550405\nTo facilitate analysis and sharing of mass spectrometry (MS)-based proteomics data we created tools called CURTAIN (https://curtain.proteo.info) and CURTAIN-PTM (https://curtainptm.proteo.info). These enable the non-MS expert to interactively peruse volcano plots; deconvolute primary experimental data to individual replicates that can be visualized in bar charts or violin plots allowing statistical analysis; and export of plots in SVG format. They also permit assessment of experimental quality by correlation matrix and profile plot. Within CURTAIN, the user can analyze domain structure, AlphaFold predicted structure, reported interactors, relative expression, disease and pharmaceutical links, and mutagenesis information on all selected hits. Moreover, CURTAIN-PTM permits the comparison of all identified PTM sites on protein(s) of interest with PTM information contained within selected databases. For phosphorylation site analysis CURTAIN-PTM links with the kinase library to predict upstream kinases that phosphorylate sites of interest. We provide examples of the utility of CURTAIN and CURTAIN-PTM in analyzing how targeted degradation of the PPM1H Rab phosphatase that counteracts the Parkinsons LRRK2 kinase impacts cellular protein levels and phosphorylation sites. We reanalyzed a ubiquitylation dataset, characterizing the PINK1-Parkin pathway activation in primary neurons, revealing new data of interest not highlighted previously. CURTAIN and CURTAIN-PTM are free to use and open-source and will enable researchers to share and maximize the analysis and impact of their proteomics data. We advocate that differential expression proteomic data should be published containing a shareable CURTAIN web-link, allowing readers to better explore their data. Significance StatementTo enable non-experts to better share and explore mass spectrometry data, we have generated using open-source software, interactive tools termed CURTAIN and CURTAIN-PTM. These tools enable users to save their analysis sessions with a sharable unique web-link, enabling other researchers to visualize and further analyze these datasets. These links can also be reported in publications allowing readers to further survey the reported data. We discuss benefits for the research community of publishing proteomic data containing a shareable web-link.\nSpatial Metabolome Lipidome and Glycome from a Single brain Section\nAuthors: Clarke, H. A.; Ma, X.; shedlock, c. J.; medina, t.; Hawkinson, T. R.; Wu, L.; Robas, R. A.; Ravi, S.; Bizon, J. L. L.; Burke, S. N.; Abisambra, J. F.; Merritt, M. E.; Prentice, B. M.; Vander Kooi, C. W.; Gentry, m. S.; Chen, L.; sun, r. c.\nScore: 11.9, Published: 2023-07-25 DOI: 10.1101/2023.07.22.550155\nMetabolites, lipids, and glycans are fundamental biomolecules involved in complex biological systems. They are metabolically channeled through a myriad of pathways and molecular processes that define the physiology and pathology of an organism. Here, we present a blueprint for the simultaneous analysis of spatial metabolome, lipidome, and glycome from a single tissue section using mass spectrometry imaging. Complimenting an original experimental protocol, our workflow includes a computational framework called Spatial Augmented Multiomics Interface (Sami) that offers multiomics integration, high dimensionality clustering, spatial anatomical mapping with matched multiomics features, and metabolic pathway enrichment to providing unprecedented insights into the spatial distribution and interaction of these biomolecules in mammalian tissue biology.\nPrevalence of and gene regulatory constraints on transcriptional adaptation in single cells\nAuthors: Mellis, I. A.; Bodkin, N.; Goyal, Y.\nScore: 1.0, Published: 2023-08-15 DOI: 10.1101/2023.08.14.553318\nCells and tissues have a remarkable ability to adapt to genetic perturbations via a variety of molecular mechanisms. Nonsense-induced transcriptional compensation, a form of transcriptional adaptation, has recently emerged as one such mechanism, in which nonsense mutations in a gene can trigger upregulation of related genes, possibly conferring robustness at cellular and organismal levels. However, beyond a handful of developmental contexts and curated sets of genes, to date, no comprehensive genome-wide investigation of this behavior has been undertaken for mammalian cell types and contexts. Moreover, how the regulatory-level effects of inherently stochastic compensatory gene networks contribute to phenotypic penetrance in single cells remains unclear. Here we combine computational analysis of existing datasets with stochastic mathematical modeling and machine learning to uncover the widespread prevalence of transcriptional adaptation in mammalian systems and the diverse single-cell manifestations of minimal compensatory gene networks. Regulon gene expression analysis of a pooled single-cell genetic perturbation dataset recapitulates our model predictions. Our integrative approach uncovers several putative hits-genes demonstrating possible transcriptional adaptation-to follow up on experimentally, and provides a formal quantitative framework to test and refine models of transcriptional adaptation.\nEvaluation of Fendiline Treatment in VP40 System with Nucleation-Elongation Process: A Computational Model of Ebola Virus Matrix Protein Assembly\nAuthors: Liu, X.; Husby, M.; Stahelin, R. V.; Pienaar, E.\nScore: 2.2, Published: 2023-08-03 DOI: 10.1101/2023.08.03.551833\nEbola virus (EBOV) infection is threatening human health, especially in Central and West Africa. Limited clinical trials and the requirement of biosafety level-4 (BSL-4) laboratories hinders experimental work to advance our understanding of EBOV and evaluation of treatment. In this work, we use a computational model to study the assembly and budding process of EBOV and evaluate the effect of fendiline on these processes. Our results indicate that the assembly of VP40 filaments may follow the nucleation-elongation theory, as it is critical to maintain a pool of VP40 dimer for the maturation and production of virus-like particles (VLPs). We further find that the nucleation-elongation process can also be influenced by phosphatidylserine (PS), which can complicate the efficacy of fendiline, a drug that lowers cellular PS levels. We observe that fendiline may increase VLP production at earlier time points (24 h) and under low concentrations ([\u0026le;] 2 M). But this effect is transient and does not change the conclusion that fendiline generally decreases VLP production. We also conclude that fendiline can be more efficient at the stage of VLP budding relative to earlier phases. Combination therapy with a VLP budding step-targeted drug may further increase the treatment efficiency of fendiline. Finally, we also show that fendiline has higher efficacy when VP40 expression is high. While these are single-cell level results based on the VP40 system, it points out a potential way of fendiline application affecting EBOV assembly, which can be further tested in experimental studies with multiple EBOV proteins or live virus. ImportanceEBOV infection can cause deadly hemorrhagic fever, which has a mortality rate around 90% without treatment. The recent outbreaks in Uganda and Democratic Republic of the Congo illustrate its treat to human health. Though two antibody-based treatments are approved, mortality rates in the last outbreak is still higher than 30%. This can partly be due to the requirement of advanced medical facilities for current treatments. As a result, it is very important to develop and evaluate new therapies for EBOV infection, especially those can be easily applied in the developing world. The significance of our research is that we evaluate the potential treatment effect of fendiline on EBOV infection in the VP40 system with a computational approach, which both greatly saves time and lowers cost compared to traditional experimental studies, and provides innovative new tools to study viral protein dynamics.\nSystematic exploration of prokaryotic form I rubisco maximal carboxylation rates\nAuthors: de Pins, B.; Greenspoon, L.; Bar-On, Y. M.; Shamshoum, M.; Ben-Nissan, R.; Milshtein, E.; Davidi, D.; Sharon, I.; Mueller-Cajar, O.; Noor, E.; Milo, R.\nScore: 19.3, Published: 2023-07-27 DOI: 10.1101/2023.07.27.550689\nAutotrophy is the basis for complex life on Earth. Its physiological limits are largely constrained by rubisco - the enzyme that catalyzes almost all carbon fixation on the planet. Yet, with only a small fraction of rubisco diversity kinetically characterized so far, the underlying biological factors driving the evolution of fast rubiscos in nature remain unclear. We conducted a high-throughput kinetic characterization of over 100 prokaryotic form I rubiscos, the most ubiquitous group of rubisco sequences in nature, to uncover the determinants of rubiscos carboxylation velocity. We show that phototrophy and the presence of a carboxysome CO2 concentrating mechanism correlate with faster rubiscos (median rate difference of 3-5 fold). In contrast to previous assumptions, rubiscos originating from -cyanobacteria exhibit the highest carboxylation rates among form I enzymes ({approx}10 s-1 median versus \u003c7 s-1 in other groups). Our study ties biological and environmental insights associated with kinetic variation across rubiscos from nature.\n",
  "wordCount" : "2075",
  "inLanguage": "en",
  "datePublished": "2023-08-16T10:37:20Z",
  "dateModified": "2023-08-16T10:37:20Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta"><span>updated on August 16, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552219">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552219" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552219">
        <p class="paperTitle">Systematic discovery of protein interaction interfaces using AlphaFold and experimental validation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552219" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552219" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, C. Y.; Hubrich, D.; Varga, J. K.; Schäfer, C.; Welzel, M.; Schumbera, E.; Đokic, M.; Strom, J. M.; Schönfeld, J.; Geist, J. L.; Polat, F.; Gibson, T. J.; Keller Valsecchi, C. I.; Kumar, M.; Schueler-Furman, O.; Luck, K.</p>
        <p class="info">Score: 46.1, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552219' target='https://doi.org/10.1101/2023.08.07.552219'> 10.1101/2023.08.07.552219</a></p>
        <p class="abstract">Structural resolution of protein interactions enables mechanistic and functional studies as well as interpretation of disease variants. However, structural data is still missing for most protein interactions because we lack computational and experimental tools at scale. We thoroughly assessed AlphaFold-Multimer accuracy for structure prediction of interactions involving folded domains binding to short linear motifs from the ELM database. The structure predictions were highly sensitive but not very specific when using small protein fragments. Sensitivity decreased substantially when using long protein fragments or full length proteins with intrinsically disordered regions. We delineated a fragmentation strategy to optimize sensitivity and applied it to interactions between proteins associated with neurodevelopmental disorders. This enabled prediction of highly confident and likely disease-related novel interfaces, but also resulted in many high scoring false positive predictions. Experiments supported predicted interfaces between CREBZF-HCFC1, FBXO23-STX1B, STX1B-VAMP2, ESRRG-PSMC5, PEX3-PEX19, PEX3-PEX16, and SNRPB-GIGYF1 providing novel molecular insights for diverse biological processes. Our work highlights exciting perspectives, but also reveals clear limitations and the need for future developments to maximize the power of Alphafold-Multimer for interface predictions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552483">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552483" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552483">
        <p class="paperTitle">Systems-ecology designed bacterial consortium protects from severe Clostridioides difficile infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552483" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552483" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jenior, M. L.; Leslie, J. L.; Kolling, G. L.; Archbald-Pannone, L.; Powers, D. A.; Petri, W. A.; Papin, J. A.</p>
        <p class="info">Score: 5.9, Published: 2023-08-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552483' target='https://doi.org/10.1101/2023.08.08.552483'> 10.1101/2023.08.08.552483</a></p>
        <p class="abstract">Fecal Microbiota Transplant (FMT) is an emerging therapy that has had remarkable success in treatment and prevention of recurrent Clostridioides difficile infection (rCDI). FMT has recently been associated with adverse outcomes such as inadvertent transfer of antimicrobial resistance, necessitating development of more targeted bacteriotherapies. To address this challenge, we developed a novel systems biology pipeline to identify candidate probiotic strains that would be predicted to interrupt C. difficile pathogenesis. Utilizing metagenomic characterization of human FMT donor samples, we identified those metabolic pathways most associated with successful FMTs and reconstructed the metabolism of encoding species to simulate interactions with C. difficile. This analysis resulted in predictions of high levels of cross-feeding for amino acids in species most associated with FMT success. Guided by these in silico models, we assembled consortia of bacteria with increased amino acid cross-feeding which were then validated in vitro. We subsequently tested the consortia in a murine model of CDI, demonstrating total protection from severe CDI through decreased toxin levels, recovered gut microbiota, and increased intestinal eosinophils. These results support the novel framework that amino acid cross-feeding is likely a critical mechanism in the initial resolution of CDI by FMT. Importantly, we conclude that our predictive platform based on predicted and testable metabolic interactions between the microbiota and C. difficile led to a rationally designed biotherapeutic framework that may be extended to other enteric infections.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.12.552948">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.12.552948" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.12.552948">
        <p class="paperTitle">Two-way feedback between chromatin compaction and histone modification state explains S. cerevisiae heterochromatin bistability</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.12.552948" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.12.552948" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Movilla Miangolarra, A.; Saxton, D. S.; Yan, Z.; Rine, J.; Howard, M.</p>
        <p class="info">Score: 2.1, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.12.552948' target='https://doi.org/10.1101/2023.08.12.552948'> 10.1101/2023.08.12.552948</a></p>
        <p class="abstract">Compact chromatin is closely linked with gene silencing in part by sterically masking access to promoters, inhibiting transcription factor binding and preventing polymerase from efficiently transcribing a gene. Here, we propose a broader view: chromatin compaction can be both a cause and a consequence of the histone modification state, and this tight bidirectional interaction can underpin bistable transcriptional states. To test this theory, we developed a mathematical model for the dynamics of the HMR locus in S. cerevisiae, that incorporates activating histone modifications, silencing proteins and a dynamic, acetylation-dependent, three-dimensional locus size. Chromatin compaction enhances silencer protein binding, which in turn feeds back to remove activating histone modifications, leading to further compaction. The bistable output of the model was in good agreement with prior quantitative data, including switching rates from expressed to silent states, and vice versa, and protein binding levels within the locus. We then tested the model by predicting changes in switching rates as the genetic length of the locus was increased, which were then experimentally verified. This bidirectional feedback between chromatin compaction and the histone modification state may be an important regulatory mechanism at many loci.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550405">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550405" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550405">
        <p class="paperTitle">CURTAIN - A Unique Web-based tool for exploration and sharing of MS-based proteomics data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550405" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550405" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Phung, T. K.; Berndsen, K.; Phan, T. L.; Muqit, M. M.; Alessi, D. R.; Nirujogi, R. S.</p>
        <p class="info">Score: 27.5, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550405' target='https://doi.org/10.1101/2023.07.25.550405'> 10.1101/2023.07.25.550405</a></p>
        <p class="abstract">To facilitate analysis and sharing of mass spectrometry (MS)-based proteomics data we created tools called CURTAIN (https://curtain.proteo.info) and CURTAIN-PTM (https://curtainptm.proteo.info). These enable the non-MS expert to interactively peruse volcano plots; deconvolute primary experimental data to individual replicates that can be visualized in bar charts or violin plots allowing statistical analysis; and export of plots in SVG format. They also permit assessment of experimental quality by correlation matrix and profile plot. Within CURTAIN, the user can analyze domain structure, AlphaFold predicted structure, reported interactors, relative expression, disease and pharmaceutical links, and mutagenesis information on all selected hits. Moreover, CURTAIN-PTM permits the comparison of all identified PTM sites on protein(s) of interest with PTM information contained within selected databases. For phosphorylation site analysis CURTAIN-PTM links with the kinase library to predict upstream kinases that phosphorylate sites of interest. We provide examples of the utility of CURTAIN and CURTAIN-PTM in analyzing how targeted degradation of the PPM1H Rab phosphatase that counteracts the Parkinsons LRRK2 kinase impacts cellular protein levels and phosphorylation sites. We reanalyzed a ubiquitylation dataset, characterizing the PINK1-Parkin pathway activation in primary neurons, revealing new data of interest not highlighted previously. CURTAIN and CURTAIN-PTM are free to use and open-source and will enable researchers to share and maximize the analysis and impact of their proteomics data. We advocate that differential expression proteomic data should be published containing a shareable CURTAIN web-link, allowing readers to better explore their data.

Significance StatementTo enable non-experts to better share and explore mass spectrometry data, we have generated using open-source software, interactive tools termed CURTAIN and CURTAIN-PTM. These tools enable users to save their analysis sessions with a sharable unique web-link, enabling other researchers to visualize and further analyze these datasets. These links can also be reported in publications allowing readers to further survey the reported data. We discuss benefits for the research community of publishing proteomic data containing a shareable web-link.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.22.550155">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.22.550155" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.22.550155">
        <p class="paperTitle">Spatial Metabolome Lipidome and Glycome from a Single brain Section</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.22.550155" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.22.550155" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Clarke, H. A.; Ma, X.; shedlock, c. J.; medina, t.; Hawkinson, T. R.; Wu, L.; Robas, R. A.; Ravi, S.; Bizon, J. L. L.; Burke, S. N.; Abisambra, J. F.; Merritt, M. E.; Prentice, B. M.; Vander Kooi, C. W.; Gentry, m. S.; Chen, L.; sun, r. c.</p>
        <p class="info">Score: 11.9, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.22.550155' target='https://doi.org/10.1101/2023.07.22.550155'> 10.1101/2023.07.22.550155</a></p>
        <p class="abstract">Metabolites, lipids, and glycans are fundamental biomolecules involved in complex biological systems. They are metabolically channeled through a myriad of pathways and molecular processes that define the physiology and pathology of an organism. Here, we present a blueprint for the simultaneous analysis of spatial metabolome, lipidome, and glycome from a single tissue section using mass spectrometry imaging. Complimenting an original experimental protocol, our workflow includes a computational framework called Spatial Augmented Multiomics Interface (Sami) that offers multiomics integration, high dimensionality clustering, spatial anatomical mapping with matched multiomics features, and metabolic pathway enrichment to providing unprecedented insights into the spatial distribution and interaction of these biomolecules in mammalian tissue biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.14.553318">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.14.553318" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.14.553318">
        <p class="paperTitle">Prevalence of and gene regulatory constraints on transcriptional adaptation in single cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.14.553318" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.14.553318" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mellis, I. A.; Bodkin, N.; Goyal, Y.</p>
        <p class="info">Score: 1.0, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.14.553318' target='https://doi.org/10.1101/2023.08.14.553318'> 10.1101/2023.08.14.553318</a></p>
        <p class="abstract">Cells and tissues have a remarkable ability to adapt to genetic perturbations via a variety of molecular mechanisms. Nonsense-induced transcriptional compensation, a form of transcriptional adaptation, has recently emerged as one such mechanism, in which nonsense mutations in a gene can trigger upregulation of related genes, possibly conferring robustness at cellular and organismal levels. However, beyond a handful of developmental contexts and curated sets of genes, to date, no comprehensive genome-wide investigation of this behavior has been undertaken for mammalian cell types and contexts. Moreover, how the regulatory-level effects of inherently stochastic compensatory gene networks contribute to phenotypic penetrance in single cells remains unclear. Here we combine computational analysis of existing datasets with stochastic mathematical modeling and machine learning to uncover the widespread prevalence of transcriptional adaptation in mammalian systems and the diverse single-cell manifestations of minimal compensatory gene networks. Regulon gene expression analysis of a pooled single-cell genetic perturbation dataset recapitulates our model predictions. Our integrative approach uncovers several putative hits-genes demonstrating possible transcriptional adaptation-to follow up on experimentally, and provides a formal quantitative framework to test and refine models of transcriptional adaptation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.03.551833">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.03.551833" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.03.551833">
        <p class="paperTitle">Evaluation of Fendiline Treatment in VP40 System with Nucleation-Elongation Process: A Computational Model of Ebola Virus Matrix Protein Assembly</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.03.551833" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.03.551833" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, X.; Husby, M.; Stahelin, R. V.; Pienaar, E.</p>
        <p class="info">Score: 2.2, Published: 2023-08-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.03.551833' target='https://doi.org/10.1101/2023.08.03.551833'> 10.1101/2023.08.03.551833</a></p>
        <p class="abstract">Ebola virus (EBOV) infection is threatening human health, especially in Central and West Africa. Limited clinical trials and the requirement of biosafety level-4 (BSL-4) laboratories hinders experimental work to advance our understanding of EBOV and evaluation of treatment. In this work, we use a computational model to study the assembly and budding process of EBOV and evaluate the effect of fendiline on these processes. Our results indicate that the assembly of VP40 filaments may follow the nucleation-elongation theory, as it is critical to maintain a pool of VP40 dimer for the maturation and production of virus-like particles (VLPs). We further find that the nucleation-elongation process can also be influenced by phosphatidylserine (PS), which can complicate the efficacy of fendiline, a drug that lowers cellular PS levels. We observe that fendiline may increase VLP production at earlier time points (24 h) and under low concentrations ([&amp;le;] 2 M). But this effect is transient and does not change the conclusion that fendiline generally decreases VLP production. We also conclude that fendiline can be more efficient at the stage of VLP budding relative to earlier phases. Combination therapy with a VLP budding step-targeted drug may further increase the treatment efficiency of fendiline. Finally, we also show that fendiline has higher efficacy when VP40 expression is high. While these are single-cell level results based on the VP40 system, it points out a potential way of fendiline application affecting EBOV assembly, which can be further tested in experimental studies with multiple EBOV proteins or live virus.

ImportanceEBOV infection can cause deadly hemorrhagic fever, which has a mortality rate around 90% without treatment. The recent outbreaks in Uganda and Democratic Republic of the Congo illustrate its treat to human health. Though two antibody-based treatments are approved, mortality rates in the last outbreak is still higher than 30%. This can partly be due to the requirement of advanced medical facilities for current treatments. As a result, it is very important to develop and evaluate new therapies for EBOV infection, especially those can be easily applied in the developing world. The significance of our research is that we evaluate the potential treatment effect of fendiline on EBOV infection in the VP40 system with a computational approach, which both greatly saves time and lowers cost compared to traditional experimental studies, and provides innovative new tools to study viral protein dynamics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550689">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550689" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550689">
        <p class="paperTitle">Systematic exploration of prokaryotic form I rubisco maximal carboxylation rates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550689" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550689" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: de Pins, B.; Greenspoon, L.; Bar-On, Y. M.; Shamshoum, M.; Ben-Nissan, R.; Milshtein, E.; Davidi, D.; Sharon, I.; Mueller-Cajar, O.; Noor, E.; Milo, R.</p>
        <p class="info">Score: 19.3, Published: 2023-07-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550689' target='https://doi.org/10.1101/2023.07.27.550689'> 10.1101/2023.07.27.550689</a></p>
        <p class="abstract">Autotrophy is the basis for complex life on Earth. Its physiological limits are largely constrained by rubisco - the enzyme that catalyzes almost all carbon fixation on the planet. Yet, with only a small fraction of rubisco diversity kinetically characterized so far, the underlying biological factors driving the evolution of fast rubiscos in nature remain unclear. We conducted a high-throughput kinetic characterization of over 100 prokaryotic form I rubiscos, the most ubiquitous group of rubisco sequences in nature, to uncover the determinants of rubiscos carboxylation velocity. We show that phototrophy and the presence of a carboxysome CO2 concentrating mechanism correlate with faster rubiscos (median rate difference of 3-5 fold). In contrast to previous assumptions, rubiscos originating from -cyanobacteria exhibit the highest carboxylation rates among form I enzymes ({approx}10 s-1 median versus &lt;7 s-1 in other groups). Our study ties biological and environmental insights associated with kinetic variation across rubiscos from nature.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
