<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="LIANA&#43;: an all-in-one cell-cell communication framework
Authors: Dimitrov, D.; Schäfer, P. S. L.; Farr, E.; Rodriguez Mier, P.; Lobentanzer, S.; Dugourd, A.; Tanevski, J.; Ramirez Flores, R. O.; Saez-Rodriguez, J.
Score: 52.1, Published: 2023-08-21 DOI: 10.1101/2023.08.19.553863
The growing availability of single-cell and spatially-resolved transcriptomics has led to the rapidly growing popularity of methods to infer cell-cell communication. Many approaches have emerged, each capturing only a partial view of the complex landscape of cell-cell communication.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="LIANA&#43;: an all-in-one cell-cell communication framework
Authors: Dimitrov, D.; Schäfer, P. S. L.; Farr, E.; Rodriguez Mier, P.; Lobentanzer, S.; Dugourd, A.; Tanevski, J.; Ramirez Flores, R. O.; Saez-Rodriguez, J.
Score: 52.1, Published: 2023-08-21 DOI: 10.1101/2023.08.19.553863
The growing availability of single-cell and spatially-resolved transcriptomics has led to the rapidly growing popularity of methods to infer cell-cell communication. Many approaches have emerged, each capturing only a partial view of the complex landscape of cell-cell communication." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-27T10:37:02+00:00" />
<meta property="article:modified_time" content="2023-08-27T10:37:02+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="LIANA&#43;: an all-in-one cell-cell communication framework
Authors: Dimitrov, D.; Schäfer, P. S. L.; Farr, E.; Rodriguez Mier, P.; Lobentanzer, S.; Dugourd, A.; Tanevski, J.; Ramirez Flores, R. O.; Saez-Rodriguez, J.
Score: 52.1, Published: 2023-08-21 DOI: 10.1101/2023.08.19.553863
The growing availability of single-cell and spatially-resolved transcriptomics has led to the rapidly growing popularity of methods to infer cell-cell communication. Many approaches have emerged, each capturing only a partial view of the complex landscape of cell-cell communication."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "LIANA\u0026#43;: an all-in-one cell-cell communication framework\nAuthors: Dimitrov, D.; Schäfer, P. S. L.; Farr, E.; Rodriguez Mier, P.; Lobentanzer, S.; Dugourd, A.; Tanevski, J.; Ramirez Flores, R. O.; Saez-Rodriguez, J.\nScore: 52.1, Published: 2023-08-21 DOI: 10.1101/2023.08.19.553863\nThe growing availability of single-cell and spatially-resolved transcriptomics has led to the rapidly growing popularity of methods to infer cell-cell communication. Many approaches have emerged, each capturing only a partial view of the complex landscape of cell-cell communication.",
  "keywords": [
    
  ],
  "articleBody": " LIANA+: an all-in-one cell-cell communication framework\nAuthors: Dimitrov, D.; Schäfer, P. S. L.; Farr, E.; Rodriguez Mier, P.; Lobentanzer, S.; Dugourd, A.; Tanevski, J.; Ramirez Flores, R. O.; Saez-Rodriguez, J.\nScore: 52.1, Published: 2023-08-21 DOI: 10.1101/2023.08.19.553863\nThe growing availability of single-cell and spatially-resolved transcriptomics has led to the rapidly growing popularity of methods to infer cell-cell communication. Many approaches have emerged, each capturing only a partial view of the complex landscape of cell-cell communication. Here, we present LIANA+, a scalable framework to decode coordinated inter- and intracellular signalling events from single- and multi-condition datasets in both single-cell and spatially-resolved data. Beyond integrating and extending established methodologies and a rich knowledge base, LIANA+ enables novel analyses using diverse molecular mediators, including those measured in multi-omics data. Accessible as an open-source Python package at https://github.com/saezlab/liana-py, LIANA+ provides a comprehensive set of synergistic components to study cell-cell communication. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=124 SRC=\"FIGDIR/small/553863v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (53K): org.highwire.dtl.DTLVardef@1a38670org.highwire.dtl.DTLVardef@cf5210org.highwire.dtl.DTLVardef@114bfd5org.highwire.dtl.DTLVardef@13e594d_HPS_FORMAT_FIGEXP M_FIG C_FIG\nLarge-scale analysis of the integration of enhancer-enhancer signals by promoters\nAuthors: Martinez-Ara, M.; Comoglio, F.; van Steensel, B.\nScore: 24.7, Published: 2023-08-14 DOI: 10.1101/2023.08.11.552995\nGenes are often regulated by multiple enhancers. It is poorly understood how the individual enhancer activities are combined to control promoter activity. Anecdotal evidence has shown that enhancers can combine sub-additively, additively, synergistically, or redundantly. However, it is not clear which of these modes are more frequent in mammalian genomes. Here, we systematically tested how pairs of enhancers activate promoters using a three-way combinatorial reporter assay in mouse cells. By assaying about 69,000 enhancer-enhancer-promoter combinations we found that enhancer pairs generally combine near-additively. This behaviour was conserved across seven developmental promoters tested. Surprisingly, these promoters scale the enhancer signals approximately following a power-law, but the exponent of this response varies between promoters. A housekeeping promoter showed an overall different response to enhancer pairs, and a smaller dynamic range. Thus, our data indicate that enhancers mostly act additively, but promoters transform their collective effect non-linearly.\nSystematic discovery of protein interaction interfaces using AlphaFold and experimental validation\nAuthors: Lee, C. Y.; Hubrich, D.; Varga, J. K.; Schäfer, C.; Welzel, M.; Schumbera, E.; Đokic, M.; Strom, J. M.; Schönfeld, J.; Geist, J. L.; Polat, F.; Gibson, T. J.; Keller Valsecchi, C. I.; Kumar, M.; Schueler-Furman, O.; Luck, K.\nScore: 61.1, Published: 2023-08-09 DOI: 10.1101/2023.08.07.552219\nStructural resolution of protein interactions enables mechanistic and functional studies as well as interpretation of disease variants. However, structural data is still missing for most protein interactions because we lack computational and experimental tools at scale. We thoroughly assessed AlphaFold-Multimer accuracy for structure prediction of interactions involving folded domains binding to short linear motifs from the ELM database. The structure predictions were highly sensitive but not very specific when using small protein fragments. Sensitivity decreased substantially when using long protein fragments or full length proteins with intrinsically disordered regions. We delineated a fragmentation strategy to optimize sensitivity and applied it to interactions between proteins associated with neurodevelopmental disorders. This enabled prediction of highly confident and likely disease-related novel interfaces, but also resulted in many high scoring false positive predictions. Experiments supported predicted interfaces between CREBZF-HCFC1, FBXO23-STX1B, STX1B-VAMP2, ESRRG-PSMC5, PEX3-PEX19, PEX3-PEX16, and SNRPB-GIGYF1 providing novel molecular insights for diverse biological processes. Our work highlights exciting perspectives, but also reveals clear limitations and the need for future developments to maximize the power of Alphafold-Multimer for interface predictions.\nHistone mark age of human tissues and cells\nAuthors: de Lima Camillo, L. P.; Asif, M. H.; Horvath, S.; Larschan, E.; Singh, R.\nScore: 15.5, Published: 2023-08-23 DOI: 10.1101/2023.08.21.554165\nBackgroundAging involves intricate epigenetic changes, with histone modifications playing a pivotal role in dynamically regulating gene expression. Our research comprehensively analyzes seven key histone modifications across various tissues to understand their behavior during human aging and formulate age prediction models. ResultsThese histone-centric prediction models exhibit remarkable accuracy and resilience against experimental and artificial noise. They showcase comparable efficacy when compared with DNA methylation age predictors through simulation experiments. Intriguingly, our gene set enrichment analysis pinpoints vital developmental pathways crucial for age prediction. Unlike in DNA methylation age predictors, genes previously recognized in animal studies as integral to aging are amongst the most important features of our models. We also introduce a pan-histone-mark, pan-tissue age predictor that operates across multiple tissues and histone marks, reinforcing that age-related epigenetic markers are not restricted to particular histone modifications. ConclusionOur findings underscore the potential of histone marks in crafting robust age predictors and shed light on the intricate tapestry of epigenetic alterations in aging.\nOn standardization of controls in lifespan studies\nAuthors: Spiridonova, O.; Kriukov, D.; Nemirovich-Danchenko, N.; Peshkin, L.\nScore: 9.3, Published: 2023-08-18 DOI: 10.1101/2023.08.17.552381\nThe search for interventions to slow down and even reverse aging is a burgeoning field. The literature cites hundreds of supposedly beneficial pharmacological and genetic interventions in model organisms: mice, rats, flies and worms, where research into physiology is routinely accompanied by lifespan data. Naturally the negative results are more frequent, yet scientifically quite valuable if analyzed systematically. Yet, there is a strong \"discovery bias\", i.e. results of interventions which turn out not to be beneficial remain unpublished. Theoretically, all lifespan data is ripe for re-analysis: we could contrast the molecular targets and pathways across studies and help focus the further search for interventions. Alas, the results of most longevity studies are difficult to compare. This is in part because there are no clear, universally accepted standards for conducting such experiments or even for reporting such data. The situation is worsened by the fact that the authors often do not describe experimental conditions completely. As a result, works on longevity make up a set of precedents, each of which might be interesting in its own right, yet incoherent and incomparable. Here we point out specific issues and propose solutions for quality control by checking both inter- and intra-study consistency of lifespan data.\nReverse engineering morphogenesis through Bayesian optimization of physics-based models\nAuthors: Kumar, N.; Dowling, A.; Zartman, J. J.\nScore: 6.9, Published: 2023-08-22 DOI: 10.1101/2023.08.21.553928\nMorphogenetic programs direct the cell signaling and nonlinear mechanical interactions between multiple cell types and tissue layers to define organ shape and size. A key challenge for systems and synthetic biology is determining optimal combinations of intra- and inter-cellular interactions to predict an organs shape, size, and function. Physics-based mechanistic models that define the subcellular force distribution facilitate this, but it is extremely challenging to calibrate parameters in these models from data. To solve this inverse problem, we created a Bayesian optimization framework to determine the optimal cellular force distribution such that the predicted organ shapes match the desired organ shapes observed within the experimental imaging data. This integrative framework employs Gaussian Process Regression (GPR), a non-parametric kernel-based probabilistic machine learning modeling paradigm, to learn the mapping functions relating to the morphogenetic programs that generate and maintain the final organ shape. We calibrated and tested the method on cross-sections of Drosophila wing imaginal discs, a highly informative model organ system, to study mechanisms that regulate epithelial processes that range from development to cancer. As a specific test case, the parameter estimation framework successfully infers the underlying changes in core parameters needed to match simulation data with time series imaging data of wing discs perturbed with collagenase. Unexpectedly, the framework also identifies multiple distinct parameter sets that generate shapes similar to wild-type organ shapes. This platform enables an efficient, global sensitivity analysis to support the necessity of both actomyosin contractility and basal ECM stiffness to generate and maintain the curved shape of the wing imaginal disc. The optimization framework, combined with fixed tissue imaging, identified that Piezo, a mechanosensitive ion channel, impacts fold formation by regulating the apical-basal balance of actomyosin contractility and elasticity of ECM. This framework is extensible toward reverse-engineering the morphogenesis of any organ system and can be utilized in real-time control of complex multicellular systems.\nEvaluating single-cell variability in proteasomal decay\nAuthors: Das, S.; Singh, A.; Shah, P.\nScore: 5.1, Published: 2023-08-23 DOI: 10.1101/2023.08.22.554358\nGene expression is a stochastic process that leads to variability in mRNA and protein abundances even within an isogenic population of cells grown in the same environment. This variation, often called gene-expression noise, has typically been attributed to transcriptional and translational processes while ignoring the contributions of protein decay variability across cells. Here we estimate the single-cell protein decay rates of two degron GFPs in Saccharomyces cerevisiae using time-lapse microscopy. We find substantial cell-to-cell variability in the decay rates of the degron GFPs. We evaluate cellular features that explain the variability in the proteasomal decay and find that the amount of 20s catalytic beta subunit of the proteasome marginally explains the observed variability in the degron GFP half-lives. We propose alternate hypotheses that might explain the observed variability in the decay of the two degron GFPs. Overall, our study highlights the importance of studying the kinetics of the decay process at single-cell resolution and that decay rates vary at the single-cell level, and that the decay process is stochastic. A complex model of decay dynamics must be included when modeling stochastic gene expression to estimate gene expression noise.\nIntegration of Thermal Proteome Profiling with phosphoproteomic and transcriptomic data via mechanistic network models decodes the molecular response to PARP inhibition\nAuthors: Burtscher, M. L.; Gade, S.; Garrido-Rodriguez, M.; Rutkowska, A.; Werner, T.; Eberl, H. C.; Petretich, M.; Knopf, N.; Zirngibl, K.; Grandi, P.; Bergamini, G.; Bantscheff, M.; Faelth-Savitski, M.; Saez-Rodriguez, J.\nScore: 18.1, Published: 2023-08-16 DOI: 10.1101/2023.08.15.553354\nThe deregulation of complex diseases often spans multiple molecular processes. A multimodal functional characterization of these processes can shed light on the disease mechanisms and the effect of drugs. Thermal Proteome Profiling (TPP) is a mass-spectrometry based technique assessing changes in thermal protein stability that can serve as proxies of functional changes of the proteome. These unique insights of TPP can complement those obtained by other omics technologies. Here, we show how TPP can be integrated with phosphoproteomics and transcriptomics in a network-based approach using COSMOS, a framework for causal integration of multi-omics, to provide an integrated view of transcription factors, kinases and proteins with altered thermal stability. This allowed us to recover known mechanistic consequences of PARP inhibition in ovarian cancer cells on cell cycle and DNA damage response in detail and to uncover new insights into drug response mechanisms related to interferon and hippo signaling. We found that TPP complements the other omics data and allowed us to obtain a network model with higher coverage of the main underlying mechanisms. These results illustrate the added value of TPP, and more generally the power of network models to integrate the information provided by different omics technologies. We anticipate that this strategy can be used to broadly integrate functional proteomics with other omics to study complex molecular processes. Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=63 SRC=\"FIGDIR/small/553354v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (18K): org.highwire.dtl.DTLVardef@1c6b705org.highwire.dtl.DTLVardef@119b888org.highwire.dtl.DTLVardef@1a21a85org.highwire.dtl.DTLVardef@17b42d0_HPS_FORMAT_FIGEXP M_FIG C_FIG\nAn amortized approach to non-linear mixed-effects modeling based on neural posterior estimation\nAuthors: Arruda, J.; Schalte, Y.; Peiter, C.; Teplytska, O.; Jaehde, U.; Hasenauer, J.\nScore: 4.0, Published: 2023-08-23 DOI: 10.1101/2023.08.22.554273\nNon-linear mixed-effects models are a powerful tool for studying heterogeneous populations in various fields, including biology, medicine, economics, and engineering. However, fitting these models to data is computationally challenging if the description of individuals is complex and the population is large. To address this issue, we propose a novel machine learning-based approach: We exploit neural density estimation based on normalizing flows to approximate individual-specific posterior distributions in an amortized fashion, thereby allowing for an efficient inference of population parameters. Applying this approach to problems from cell biology and pharmacology, we demonstrate its scalability to large data sets in an unprecedented manner. Moreover, we show that it enables accurate uncertainty quantification and extends to stochastic models, which established methods, such as SAEM and FOCEI are unable to handle. Thus, our approach outperforms state-of-the-art methods and improves the analysis capabilities for heterogeneous populations.\nThe interplay between lumen pressure and cell proliferation determines organoid morphology in a multicellular phase field model\nAuthors: Tanida, S.; Fuji, K.; Lu, L.; Guyomar, T.; Lee, B. H.; Honigmann, A.; Grapin-Botton, A.; Riveline, D.; Nonomura, M.; Sano, M.; Hiraiwa, T.\nScore: 3.9, Published: 2023-08-18 DOI: 10.1101/2023.08.17.553655\nOrganoids are ideal systems to predict the phenotypes of organs. However, there is currently a lack of understanding regarding the generalized rules that enable use of simple cellular principles to make morphological predictions of entire organoids. Therefore, we employed a phase field model with the following basic components: the minimum conditions for the timing and volume of cell division, lumen nucleation rules, and lumenal pressure. Through our model, we could compute and generate a myriad of organoid phenotypes observed till date. We propose morphological indices necessary to characterize the shapes and construct phase diagrams and show their dependencies on proliferation time and lumen pressure. Additionally, we introduced the lumen-index parameter, which helped in examining the criteria to maintain organoids as spherical structures comprising a single layer of cells and enclosing an intact lumen. Finally, we predict a star-like organoid phenotype that did not undergo differentiation, suggesting that the volume constraint during cell division may determine the final phenotype. In summary, our approach provides researchers with guidelines to test the mechanisms of self-organization and predict the shape of organoid. Author summaryIn nature, a wide variety of organ morphologies are observed. Owing to the complexity of the process underlying the acquisition of organs morphology, it is challenging to investigate the mechanisms that lead to such variations. A promising approach to study these variations is the use of \"computational organoid\" study, which is the computational-based study of self-organizing shapes in multicellular assemblies and fluid-filled cavities called lumens that develop from a few proliferating cells. This study explores general mechanisms that dictate how various mechanical factors affect the growing self-organized multicellular assembly. We relied on computer simulations of the mathematical model called multicellular phase-field model with lumens and explored the mechanical factor effects, such as the lumen pressure while considering the time and volume conditions required for cell division. These simulations generated and categorized a wide range of organoid phenotypes based on the varying lumen pressure and cell division conditions. These phenotypes were characterized into seven distinct classes, based on the morphological index sets, including a cellular monolayer/multilayer surrounding single or multiple lumens and branch formation. These phenotypes were obtained without the assumption of differentiation. Our study elucidates the mechanisms underlying the organoid and organ formation with different shapes, thereby highlighting the significance of mechanical forces in shaping these complex biological structures.\n",
  "wordCount" : "2409",
  "inLanguage": "en",
  "datePublished": "2023-08-27T10:37:02Z",
  "dateModified": "2023-08-27T10:37:02Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta"><span>updated on August 27, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.19.553863">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.19.553863" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.19.553863">
        <p class="paperTitle">LIANA&#43;: an all-in-one cell-cell communication framework</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.19.553863" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.19.553863" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dimitrov, D.; Schäfer, P. S. L.; Farr, E.; Rodriguez Mier, P.; Lobentanzer, S.; Dugourd, A.; Tanevski, J.; Ramirez Flores, R. O.; Saez-Rodriguez, J.</p>
        <p class="info">Score: 52.1, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.19.553863' target='https://doi.org/10.1101/2023.08.19.553863'> 10.1101/2023.08.19.553863</a></p>
        <p class="abstract">The growing availability of single-cell and spatially-resolved transcriptomics has led to the rapidly growing popularity of methods to infer cell-cell communication. Many approaches have emerged, each capturing only a partial view of the complex landscape of cell-cell communication.

Here, we present LIANA&#43;, a scalable framework to decode coordinated inter- and intracellular signalling events from single- and multi-condition datasets in both single-cell and spatially-resolved data. Beyond integrating and extending established methodologies and a rich knowledge base, LIANA&#43; enables novel analyses using diverse molecular mediators, including those measured in multi-omics data. Accessible as an open-source Python package at https://github.com/saezlab/liana-py, LIANA&#43; provides a comprehensive set of synergistic components to study cell-cell communication.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=124 SRC=&#34;FIGDIR/small/553863v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (53K):
org.highwire.dtl.DTLVardef@1a38670org.highwire.dtl.DTLVardef@cf5210org.highwire.dtl.DTLVardef@114bfd5org.highwire.dtl.DTLVardef@13e594d_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.552995">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.552995" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.552995">
        <p class="paperTitle">Large-scale analysis of the integration of enhancer-enhancer signals by promoters</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.552995" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.552995" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martinez-Ara, M.; Comoglio, F.; van Steensel, B.</p>
        <p class="info">Score: 24.7, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.552995' target='https://doi.org/10.1101/2023.08.11.552995'> 10.1101/2023.08.11.552995</a></p>
        <p class="abstract">Genes are often regulated by multiple enhancers. It is poorly understood how the individual enhancer activities are combined to control promoter activity. Anecdotal evidence has shown that enhancers can combine sub-additively, additively, synergistically, or redundantly. However, it is not clear which of these modes are more frequent in mammalian genomes. Here, we systematically tested how pairs of enhancers activate promoters using a three-way combinatorial reporter assay in mouse cells. By assaying about 69,000 enhancer-enhancer-promoter combinations we found that enhancer pairs generally combine near-additively. This behaviour was conserved across seven developmental promoters tested. Surprisingly, these promoters scale the enhancer signals approximately following a power-law, but the exponent of this response varies between promoters. A housekeeping promoter showed an overall different response to enhancer pairs, and a smaller dynamic range. Thus, our data indicate that enhancers mostly act additively, but promoters transform their collective effect non-linearly.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552219">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552219" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552219">
        <p class="paperTitle">Systematic discovery of protein interaction interfaces using AlphaFold and experimental validation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552219" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552219" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, C. Y.; Hubrich, D.; Varga, J. K.; Schäfer, C.; Welzel, M.; Schumbera, E.; Đokic, M.; Strom, J. M.; Schönfeld, J.; Geist, J. L.; Polat, F.; Gibson, T. J.; Keller Valsecchi, C. I.; Kumar, M.; Schueler-Furman, O.; Luck, K.</p>
        <p class="info">Score: 61.1, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552219' target='https://doi.org/10.1101/2023.08.07.552219'> 10.1101/2023.08.07.552219</a></p>
        <p class="abstract">Structural resolution of protein interactions enables mechanistic and functional studies as well as interpretation of disease variants. However, structural data is still missing for most protein interactions because we lack computational and experimental tools at scale. We thoroughly assessed AlphaFold-Multimer accuracy for structure prediction of interactions involving folded domains binding to short linear motifs from the ELM database. The structure predictions were highly sensitive but not very specific when using small protein fragments. Sensitivity decreased substantially when using long protein fragments or full length proteins with intrinsically disordered regions. We delineated a fragmentation strategy to optimize sensitivity and applied it to interactions between proteins associated with neurodevelopmental disorders. This enabled prediction of highly confident and likely disease-related novel interfaces, but also resulted in many high scoring false positive predictions. Experiments supported predicted interfaces between CREBZF-HCFC1, FBXO23-STX1B, STX1B-VAMP2, ESRRG-PSMC5, PEX3-PEX19, PEX3-PEX16, and SNRPB-GIGYF1 providing novel molecular insights for diverse biological processes. Our work highlights exciting perspectives, but also reveals clear limitations and the need for future developments to maximize the power of Alphafold-Multimer for interface predictions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554165">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554165" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554165">
        <p class="paperTitle">Histone mark age of human tissues and cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554165" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554165" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: de Lima Camillo, L. P.; Asif, M. H.; Horvath, S.; Larschan, E.; Singh, R.</p>
        <p class="info">Score: 15.5, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554165' target='https://doi.org/10.1101/2023.08.21.554165'> 10.1101/2023.08.21.554165</a></p>
        <p class="abstract">BackgroundAging involves intricate epigenetic changes, with histone modifications playing a pivotal role in dynamically regulating gene expression. Our research comprehensively analyzes seven key histone modifications across various tissues to understand their behavior during human aging and formulate age prediction models.

ResultsThese histone-centric prediction models exhibit remarkable accuracy and resilience against experimental and artificial noise. They showcase comparable efficacy when compared with DNA methylation age predictors through simulation experiments. Intriguingly, our gene set enrichment analysis pinpoints vital developmental pathways crucial for age prediction. Unlike in DNA methylation age predictors, genes previously recognized in animal studies as integral to aging are amongst the most important features of our models. We also introduce a pan-histone-mark, pan-tissue age predictor that operates across multiple tissues and histone marks, reinforcing that age-related epigenetic markers are not restricted to particular histone modifications.

ConclusionOur findings underscore the potential of histone marks in crafting robust age predictors and shed light on the intricate tapestry of epigenetic alterations in aging.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.17.552381">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.17.552381" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.17.552381">
        <p class="paperTitle">On standardization of controls in lifespan studies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.17.552381" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.17.552381" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Spiridonova, O.; Kriukov, D.; Nemirovich-Danchenko, N.; Peshkin, L.</p>
        <p class="info">Score: 9.3, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.17.552381' target='https://doi.org/10.1101/2023.08.17.552381'> 10.1101/2023.08.17.552381</a></p>
        <p class="abstract">The search for interventions to slow down and even reverse aging is a burgeoning field. The literature cites hundreds of supposedly beneficial pharmacological and genetic interventions in model organisms: mice, rats, flies and worms, where research into physiology is routinely accompanied by lifespan data. Naturally the negative results are more frequent, yet scientifically quite valuable if analyzed systematically. Yet, there is a strong &#34;discovery bias&#34;, i.e. results of interventions which turn out not to be beneficial remain unpublished. Theoretically, all lifespan data is ripe for re-analysis: we could contrast the molecular targets and pathways across studies and help focus the further search for interventions. Alas, the results of most longevity studies are difficult to compare. This is in part because there are no clear, universally accepted standards for conducting such experiments or even for reporting such data. The situation is worsened by the fact that the authors often do not describe experimental conditions completely. As a result, works on longevity make up a set of precedents, each of which might be interesting in its own right, yet incoherent and incomparable. Here we point out specific issues and propose solutions for quality control by checking both inter- and intra-study consistency of lifespan data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.553928">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.553928" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.553928">
        <p class="paperTitle">Reverse engineering morphogenesis through Bayesian optimization of physics-based models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.553928" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.553928" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kumar, N.; Dowling, A.; Zartman, J. J.</p>
        <p class="info">Score: 6.9, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.553928' target='https://doi.org/10.1101/2023.08.21.553928'> 10.1101/2023.08.21.553928</a></p>
        <p class="abstract">Morphogenetic programs direct the cell signaling and nonlinear mechanical interactions between multiple cell types and tissue layers to define organ shape and size. A key challenge for systems and synthetic biology is determining optimal combinations of intra- and inter-cellular interactions to predict an organs shape, size, and function. Physics-based mechanistic models that define the subcellular force distribution facilitate this, but it is extremely challenging to calibrate parameters in these models from data. To solve this inverse problem, we created a Bayesian optimization framework to determine the optimal cellular force distribution such that the predicted organ shapes match the desired organ shapes observed within the experimental imaging data. This integrative framework employs Gaussian Process Regression (GPR), a non-parametric kernel-based probabilistic machine learning modeling paradigm, to learn the mapping functions relating to the morphogenetic programs that generate and maintain the final organ shape. We calibrated and tested the method on cross-sections of Drosophila wing imaginal discs, a highly informative model organ system, to study mechanisms that regulate epithelial processes that range from development to cancer. As a specific test case, the parameter estimation framework successfully infers the underlying changes in core parameters needed to match simulation data with time series imaging data of wing discs perturbed with collagenase. Unexpectedly, the framework also identifies multiple distinct parameter sets that generate shapes similar to wild-type organ shapes. This platform enables an efficient, global sensitivity analysis to support the necessity of both actomyosin contractility and basal ECM stiffness to generate and maintain the curved shape of the wing imaginal disc. The optimization framework, combined with fixed tissue imaging, identified that Piezo, a mechanosensitive ion channel, impacts fold formation by regulating the apical-basal balance of actomyosin contractility and elasticity of ECM. This framework is extensible toward reverse-engineering the morphogenesis of any organ system and can be utilized in real-time control of complex multicellular systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.554358">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.554358" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.554358">
        <p class="paperTitle">Evaluating single-cell variability in proteasomal decay</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.554358" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.554358" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Das, S.; Singh, A.; Shah, P.</p>
        <p class="info">Score: 5.1, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.554358' target='https://doi.org/10.1101/2023.08.22.554358'> 10.1101/2023.08.22.554358</a></p>
        <p class="abstract">Gene expression is a stochastic process that leads to variability in mRNA and protein abundances even within an isogenic population of cells grown in the same environment. This variation, often called gene-expression noise, has typically been attributed to transcriptional and translational processes while ignoring the contributions of protein decay variability across cells. Here we estimate the single-cell protein decay rates of two degron GFPs in Saccharomyces cerevisiae using time-lapse microscopy. We find substantial cell-to-cell variability in the decay rates of the degron GFPs. We evaluate cellular features that explain the variability in the proteasomal decay and find that the amount of 20s catalytic beta subunit of the proteasome marginally explains the observed variability in the degron GFP half-lives. We propose alternate hypotheses that might explain the observed variability in the decay of the two degron GFPs. Overall, our study highlights the importance of studying the kinetics of the decay process at single-cell resolution and that decay rates vary at the single-cell level, and that the decay process is stochastic. A complex model of decay dynamics must be included when modeling stochastic gene expression to estimate gene expression noise.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.15.553354">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.15.553354" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.15.553354">
        <p class="paperTitle">Integration of Thermal Proteome Profiling with phosphoproteomic and transcriptomic data via mechanistic network models decodes the molecular response to PARP inhibition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.15.553354" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.15.553354" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Burtscher, M. L.; Gade, S.; Garrido-Rodriguez, M.; Rutkowska, A.; Werner, T.; Eberl, H. C.; Petretich, M.; Knopf, N.; Zirngibl, K.; Grandi, P.; Bergamini, G.; Bantscheff, M.; Faelth-Savitski, M.; Saez-Rodriguez, J.</p>
        <p class="info">Score: 18.1, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.15.553354' target='https://doi.org/10.1101/2023.08.15.553354'> 10.1101/2023.08.15.553354</a></p>
        <p class="abstract">The deregulation of complex diseases often spans multiple molecular processes. A multimodal functional characterization of these processes can shed light on the disease mechanisms and the effect of drugs. Thermal Proteome Profiling (TPP) is a mass-spectrometry based technique assessing changes in thermal protein stability that can serve as proxies of functional changes of the proteome. These unique insights of TPP can complement those obtained by other omics technologies. Here, we show how TPP can be integrated with phosphoproteomics and transcriptomics in a network-based approach using COSMOS, a framework for causal integration of multi-omics, to provide an integrated view of transcription factors, kinases and proteins with altered thermal stability. This allowed us to recover known mechanistic consequences of PARP inhibition in ovarian cancer cells on cell cycle and DNA damage response in detail and to uncover new insights into drug response mechanisms related to interferon and hippo signaling. We found that TPP complements the other omics data and allowed us to obtain a network model with higher coverage of the main underlying mechanisms. These results illustrate the added value of TPP, and more generally the power of network models to integrate the information provided by different omics technologies. We anticipate that this strategy can be used to broadly integrate functional proteomics with other omics to study complex molecular processes.

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=63 SRC=&#34;FIGDIR/small/553354v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (18K):
org.highwire.dtl.DTLVardef@1c6b705org.highwire.dtl.DTLVardef@119b888org.highwire.dtl.DTLVardef@1a21a85org.highwire.dtl.DTLVardef@17b42d0_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.554273">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.554273" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.554273">
        <p class="paperTitle">An amortized approach to non-linear mixed-effects modeling based on neural posterior estimation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.554273" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.554273" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Arruda, J.; Schalte, Y.; Peiter, C.; Teplytska, O.; Jaehde, U.; Hasenauer, J.</p>
        <p class="info">Score: 4.0, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.554273' target='https://doi.org/10.1101/2023.08.22.554273'> 10.1101/2023.08.22.554273</a></p>
        <p class="abstract">Non-linear mixed-effects models are a powerful tool for studying heterogeneous populations in various fields, including biology, medicine, economics, and engineering. However, fitting these models to data is computationally challenging if the description of individuals is complex and the population is large. To address this issue, we propose a novel machine learning-based approach: We exploit neural density estimation based on normalizing flows to approximate individual-specific posterior distributions in an amortized fashion, thereby allowing for an efficient inference of population parameters. Applying this approach to problems from cell biology and pharmacology, we demonstrate its scalability to large data sets in an unprecedented manner. Moreover, we show that it enables accurate uncertainty quantification and extends to stochastic models, which established methods, such as SAEM and FOCEI are unable to handle. Thus, our approach outperforms state-of-the-art methods and improves the analysis capabilities for heterogeneous populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.17.553655">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.17.553655" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.17.553655">
        <p class="paperTitle">The interplay between lumen pressure and cell proliferation determines organoid morphology in a multicellular phase field model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.17.553655" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.17.553655" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tanida, S.; Fuji, K.; Lu, L.; Guyomar, T.; Lee, B. H.; Honigmann, A.; Grapin-Botton, A.; Riveline, D.; Nonomura, M.; Sano, M.; Hiraiwa, T.</p>
        <p class="info">Score: 3.9, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.17.553655' target='https://doi.org/10.1101/2023.08.17.553655'> 10.1101/2023.08.17.553655</a></p>
        <p class="abstract">Organoids are ideal systems to predict the phenotypes of organs. However, there is currently a lack of understanding regarding the generalized rules that enable use of simple cellular principles to make morphological predictions of entire organoids. Therefore, we employed a phase field model with the following basic components: the minimum conditions for the timing and volume of cell division, lumen nucleation rules, and lumenal pressure. Through our model, we could compute and generate a myriad of organoid phenotypes observed till date. We propose morphological indices necessary to characterize the shapes and construct phase diagrams and show their dependencies on proliferation time and lumen pressure. Additionally, we introduced the lumen-index parameter, which helped in examining the criteria to maintain organoids as spherical structures comprising a single layer of cells and enclosing an intact lumen. Finally, we predict a star-like organoid phenotype that did not undergo differentiation, suggesting that the volume constraint during cell division may determine the final phenotype. In summary, our approach provides researchers with guidelines to test the mechanisms of self-organization and predict the shape of organoid.

Author summaryIn nature, a wide variety of organ morphologies are observed. Owing to the complexity of the process underlying the acquisition of organs morphology, it is challenging to investigate the mechanisms that lead to such variations. A promising approach to study these variations is the use of &#34;computational organoid&#34; study, which is the computational-based study of self-organizing shapes in multicellular assemblies and fluid-filled cavities called lumens that develop from a few proliferating cells. This study explores general mechanisms that dictate how various mechanical factors affect the growing self-organized multicellular assembly. We relied on computer simulations of the mathematical model called multicellular phase-field model with lumens and explored the mechanical factor effects, such as the lumen pressure while considering the time and volume conditions required for cell division. These simulations generated and categorized a wide range of organoid phenotypes based on the varying lumen pressure and cell division conditions. These phenotypes were characterized into seven distinct classes, based on the morphological index sets, including a cellular monolayer/multilayer surrounding single or multiple lumens and branch formation. These phenotypes were obtained without the assumption of differentiation. Our study elucidates the mechanisms underlying the organoid and organ formation with different shapes, thereby highlighting the significance of mechanical forces in shaping these complex biological structures.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
