<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Spatially Exploring RNA Biology in Archival Formalin-Fixed Paraffin-Embedded Tissues
Authors: Bai, Z.; Zhang, D.; Gao, Y.; Tao, B.; Bao, S.; Enninful, A.; Zhang, D.; Su, G.; Tian, X.; Zhang, N.; Xiao, Y.; Liu, Y.; Gerstein, M.; Li, M.; Xing, Y.; Lu, J.; Xu, M. L.; Fan, R.
Score: 76.4, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579143
Spatial transcriptomics has emerged as a powerful tool for dissecting spatial cellular heterogeneity but as of today is largely limited to gene expression analysis.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Spatially Exploring RNA Biology in Archival Formalin-Fixed Paraffin-Embedded Tissues
Authors: Bai, Z.; Zhang, D.; Gao, Y.; Tao, B.; Bao, S.; Enninful, A.; Zhang, D.; Su, G.; Tian, X.; Zhang, N.; Xiao, Y.; Liu, Y.; Gerstein, M.; Li, M.; Xing, Y.; Lu, J.; Xu, M. L.; Fan, R.
Score: 76.4, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579143
Spatial transcriptomics has emerged as a powerful tool for dissecting spatial cellular heterogeneity but as of today is largely limited to gene expression analysis." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-14T10:38:48+00:00" />
<meta property="article:modified_time" content="2024-02-14T10:38:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Spatially Exploring RNA Biology in Archival Formalin-Fixed Paraffin-Embedded Tissues
Authors: Bai, Z.; Zhang, D.; Gao, Y.; Tao, B.; Bao, S.; Enninful, A.; Zhang, D.; Su, G.; Tian, X.; Zhang, N.; Xiao, Y.; Liu, Y.; Gerstein, M.; Li, M.; Xing, Y.; Lu, J.; Xu, M. L.; Fan, R.
Score: 76.4, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579143
Spatial transcriptomics has emerged as a powerful tool for dissecting spatial cellular heterogeneity but as of today is largely limited to gene expression analysis."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Spatially Exploring RNA Biology in Archival Formalin-Fixed Paraffin-Embedded Tissues\nAuthors: Bai, Z.; Zhang, D.; Gao, Y.; Tao, B.; Bao, S.; Enninful, A.; Zhang, D.; Su, G.; Tian, X.; Zhang, N.; Xiao, Y.; Liu, Y.; Gerstein, M.; Li, M.; Xing, Y.; Lu, J.; Xu, M. L.; Fan, R.\nScore: 76.4, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579143\nSpatial transcriptomics has emerged as a powerful tool for dissecting spatial cellular heterogeneity but as of today is largely limited to gene expression analysis.",
  "keywords": [
    
  ],
  "articleBody": " Spatially Exploring RNA Biology in Archival Formalin-Fixed Paraffin-Embedded Tissues\nAuthors: Bai, Z.; Zhang, D.; Gao, Y.; Tao, B.; Bao, S.; Enninful, A.; Zhang, D.; Su, G.; Tian, X.; Zhang, N.; Xiao, Y.; Liu, Y.; Gerstein, M.; Li, M.; Xing, Y.; Lu, J.; Xu, M. L.; Fan, R.\nScore: 76.4, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579143\nSpatial transcriptomics has emerged as a powerful tool for dissecting spatial cellular heterogeneity but as of today is largely limited to gene expression analysis. Yet, the life of RNA molecules is multifaceted and dynamic, requiring spatial profiling of different RNA species throughout the life cycle to delve into the intricate RNA biology in complex tissues. Human disease-relevant tissues are commonly preserved as formalin-fixed and paraffin-embedded (FFPE) blocks, representing an important resource for human tissue specimens. The capability to spatially explore RNA biology in FFPE tissues holds transformative potential for human biology research and clinical histopathology. Here, we present Patho-DBiT combining in situ polyadenylation and deterministic barcoding for spatial full coverage transcriptome sequencing, tailored for probing the diverse landscape of RNA species even in clinically archived FFPE samples. It permits spatial co-profiling of gene expression and RNA processing, unveiling region-specific splicing isoforms, and high-sensitivity transcriptomic mapping of clinical tumor FFPE tissues stored for five years. Furthermore, genome-wide single nucleotide RNA variants can be captured to distinguish different malignant clones from non-malignant cells in human lymphomas. Patho-DBiT also maps microRNA-mRNA regulatory networks and RNA splicing dynamics, decoding their roles in spatial tumorigenesis trajectory. High resolution Patho-DBiT at the cellular level reveals a spatial neighborhood and traces the spatiotemporal kinetics driving tumor progression. Patho-DBiT stands poised as a valuable platform to unravel rich RNA biology in FFPE tissues to study human tissue biology and aid in clinical pathology evaluation.\nAn Organism-Level Quantitative Flux Model of Mammalian Energy Metabolism\nAuthors: Yuan, B.; Inouye, K. E.; Hotamisligil, G.; Hui, S.\nScore: 28.9, Published: 2024-02-12 DOI: 10.1101/2024.02.11.579776\nMammalian tissues feed on nutrients in the blood circulation. At the organism-level, mammalian energy metabolism comprises of oxidation, interconverting, storing and releasing of circulating nutrients. Though much is known about the individual processes and nutrients, a holistic and quantitative model describing these processes for all major circulating nutrients is lacking. Here, by integrating isotope tracer infusion, mass spectrometry, and isotope gas analyzer measurement, we developed a framework to systematically quantify fluxes through these processes for 10 major circulating energy nutrients in mice, resulting in an organism-level quantitative flux model of energy metabolism. This model revealed in wildtype mice that circulating nutrients have more dominant metabolic cycling fluxes than their oxidation fluxes, with distinct partition between cycling and oxidation flux for individual circulating nutrients. Applications of this framework in obese mouse models showed on a per animal basis extensive elevation of metabolic cycling fluxes in ob/ob mice, but not in diet-induced obese mice. Thus, our framework describes quantitatively the functioning of energy metabolism at the organism-level, valuable for revealing new features of energy metabolism in physiological and disease conditions.\n1.4 min Plasma Proteome Profiling via Nanoparticle Protein Corona and Direct Infusion Mass Spectrometry\nAuthors: Jiang, Y.; Meyer, J. G.\nScore: 15.0, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579213\nNon-invasive detection of protein biomarkers in plasma is crucial for clinical purposes. Liquid chromatography mass spectrometry (LC-MS) is the gold standard technique for plasma proteome analysis, but despite recent advances, it remains limited by throughput, cost, and coverage. Here, we introduce a new hybrid method, which integrates direct infusion shotgun proteome analysis (DISPA) with nanoparticle (NP) protein coronas enrichment for high throughput and efficient plasma proteomic profiling. We realized over 280 protein identifications in 1.4 minutes collection time, which enables a potential throughput of approximately 1,000 samples daily. The identified proteins are involved in valuable pathways and 44 of the proteins are FDA approved biomarkers. The robustness and quantitative accuracy of this method were evaluated across multiple NPs and concentrations with a mean coefficient of variation at 17%. Moreover, different protein corona profiles were observed among various nanoparticles based on their distinct surface modifications, and all NP protein profiles exhibited deeper coverage and better quantification than neat plasma. Our streamlined workflow merges coverage and throughput with precise quantification, leveraging both DISPA and NP protein corona enrichments. This underscores the significant potential of DISPA when paired with NP sample preparation techniques for plasma proteome studies.\nLarge-Scale Evaluation of Spatial Metabolomics Protocols and Technologies\nAuthors: Saharuka, V.; Vieira, L. M.; Stuart, L.; Ekelöf, M.; Molenaar, M. R.; Bailoni, A.; Ovchinnikova, K.; Soltwisch, J.; Bausbacher, T.; Jakob, D.; King, M.; Müller, M. A.; Oetjen, J.; Pace, C.; Pinto, F. E.; Strittmatter, N.; Velickovic, D.; Spengler, B.; Muddiman, D. C.; Liebeke, M.; Janfelt, C.; Goodwin, R.; Eberlin, L. S.; Anderton, C. R.; Hopf, C.; Dreisewerd, K.; Alexandrov, T.\nScore: 25.6, Published: 2024-01-31 DOI: 10.1101/2024.01.29.577354\nSpatial metabolomics using imaging mass spectrometry (MS) enables untargeted and label-free metabolite mapping in biological samples. Despite the range of available imaging MS protocols and technologies, our understanding of metabolite detection under specific conditions is limited due to sparse empirical data and predictive theories. Consequently, challenges persist in designing new experiments, and accurately annotating and interpreting data. In this study, we systematically measured the detectability of 172 biologically-relevant metabolites across common imaging MS protocols using custom reference samples. We evaluated 24 MALDI-imaging MS protocols for untargeted metabolomics, and demonstrated the applicability of our findings to complex biological samples through comparison with animal tissue data. We showcased the potential for extending our results to further analytes by predicting metabolite detectability based on molecular properties. Additionally, our interlaboratory comparison of 10 imaging MS technologies, including MALDI, DESI, and IR-MALDESI, showed extensive metabolite coverage and comparable results, underscoring the broad applicability of our findings within the imaging MS community. We share our results and data through a new interactive web application integrated with METASPACE. This resource offers an extensive catalogue of detectable metabolite ions, facilitating protocol selection, supporting data annotation, and benefiting future untargeted spatial metabolomics studies.\nA foundational atlas of autism protein interactions reveals molecular convergence\nAuthors: Wang, B.; Vartak, R.; Zaltsman, Y.; Naing, Z. Z. C.; Hennick, K. M.; Polacco, B. J.; Bashir, A.; Eckhardt, M.; Bouhaddou, M.; Xu, J.; Sun, N.; Lasser, M.; Zhou, Y.; Guiley, K. Z.; Chan, U.; Kaye, J. A.; Khare, P.; Drake, S.; Drury, V.; Burke, D. F.; Gonzalez, S.; Alkhairy, S.; Morris, M.; Baum, T.; Krasnoff, R.; Wang, S.; Pham, P.; Arbalaez, J.; Pratt, D.; Chag, S.; Rolland, T.; Bourgeron, T.; Finkbeiner, S.; Bandyopadhay, S.; Ideker, T.; Beltrao, P.; Willsey, H. R.; Obernier, K.; Nowakowski, T. J.; Huttenhain, R.; State, M. W.; Willsey, A. J.; Krogan, N. J.\nScore: 60.7, Published: 2024-02-08 DOI: 10.1101/2023.12.03.569805\nTranslating high-confidence (hc) autism spectrum disorder (ASD) genes into viable treatment targets remains elusive. We constructed a foundational protein-protein interaction (PPI) network in HEK293T cells involving 100 hcASD risk genes, revealing over 1,800 PPIs (87% novel). Interactors, expressed in the human brain and enriched for ASD but not schizophrenia genetic risk, converged on protein complexes involved in neurogenesis, tubulin biology, transcriptional regulation, and chromatin modification. A PPI map of 54 patient-derived missense variants identified differential physical interactions, and we leveraged AlphaFold-Multimer predictions to prioritize direct PPIs and specific variants for interrogation in Xenopus tropicalis and human forebrain organoids. A mutation in the transcription factor FOXP1 led to reconfiguration of DNA binding sites and altered development of deep cortical layer neurons in forebrain organoids. This work offers new insights into molecular mechanisms underlying ASD and describes a powerful platform to develop and test therapeutic strategies for many genetically-defined conditions.\nPancreatic cancer mutationscape: revealing the link between modular restructuring and intervention efficacy amidst common mutations\nAuthors: Plaugher, D. R.; Murrugarra, D. R.\nScore: 7.3, Published: 2024-01-30 DOI: 10.1101/2024.01.27.577546\nAcross cancer types, the prognosis for pancreatic cancer (PC) is among the worst and options for treatment are limited. There is increasing evidence that biological systems, including PC, are modular in both structure and function. Complex biological signaling networks such as gene regulatory networks (GRNs) are proving to be composed of subcategories that are interconnected and hierarchically ranked. These networks contain highly dynamic processes that ultimately dictate cellular function over time. In this work, we use an established Boolean multicellular signaling network of PC to show that the variance in topological rankings of the most phenotypically influential modules implies a strong relationship between structure and function. We further show that induction of mutations alters the modular structure, which analogously influences the aggression and controllability of the disease in silico. We finally present evidence that the impact and location of mutations with respect to PC modular structure directly corresponds to the efficacy of single agent treatments in silico, because topologically deep mutations require deep targets for control.\nSemi-supervised Bayesian integration of multiple spatial proteomics datasets\nAuthors: Coleman, S. D.; Breckels, L.; Waller, R. F.; Lilley, K. S.; Wallace, C.; Kirk, P. D. W.; Crook, O. M.\nScore: 4.2, Published: 2024-02-12 DOI: 10.1101/2024.02.08.579519\nThe subcellular localisation of proteins is a key determinant of their function. High-throughput analyses of these localisations can be performed using mass spectrometry-based spatial proteomics, which enables us to examine the localisation and relocalisation of proteins. Furthermore, complementary data sources can provide additional sources of functional or localisation information. Examples include protein annotations and other high-throughput `omic assays. Integrating these modalities can provide new insights as well as additional confidence in results, but existing approaches for integrative analyses of spatial proteomics datasets are limited in the types of data they can integrate and do not quantify uncertainty. Here we propose a semi-supervised Bayesian approach to integrate spatial proteomics datasets with other data sources, to improve the inference of protein sub-cellular localisation. We demonstrate our approach outperforms other transfer-learning methods and has greater flexibility in the data it can model. To demonstrate the flexibility of our approach, we apply our method to integrate spatial proteomics data generated for the parasite Toxoplasma gondii with time-course gene expression data generated over its cell cycle. Our findings suggest that proteins linked to invasion organelles are associated with expression programs that peak at the end of the first cell-cycle. Furthermore, this integrative analysis divides the dense granule proteins into heterogeneous populations suggestive of potentially different functions. Our method is disseminated via the mdir R package available on the lead author's Github.\nFunctional genomic screening in Komagataella phaffii enabled by high-activity CRISPR-Cas9 library\nAuthors: Tafrishi, A.; Trivedi, V.; Xing, Z.; Li, M.; Mewalal, R.; Cutler, S.; Blaby, I.; Wheeldon, I.\nScore: 3.2, Published: 2024-02-09 DOI: 10.1101/2024.02.08.579509\nCRISPR-based high-throughput genome-wide loss-of-function screens are a valuable approach to functional genetics and strain engineering. The yeast Komagataella phaffii is a host of particular interest in the biopharmaceutical industry and as a metabolic engineering host for proteins and metabolites. Here, we design and validate a highly active 6-fold coverage genome-wide sgRNA library for this biotechnologically important yeast containing 30,848 active sgRNAs targeting over 99% of its coding sequences. Conducting fitness screens in the absence of functional non-homologous end joining (NHEJ), the dominant DNA repair mechanism in K. phaffii, provides a quantitative means to assess the activity of each sgRNA in the library. This approach allows for the experimental validation of each guides targeting activity, leading to more precise screening outcomes. We used this approach to conduct growth screens with glucose as the sole carbon source and identify essential genes. Comparative analysis of the called gene sets identified a core set of K. phaffii essential genes, many of which relate to protein production, secretion, and glycosylation. The high activity, genome-wide CRISPR library developed here enables functional genomic screening in K. phaffii, applied here to gene essentiality classification, and promises to enable other genetic screens. HighlightsO_LIDesigned and validated a high activity genome-wide CRISPR-Cas9 library for K. phaffii C_LIO_LIDisabling NHEJ DNA repair enables the generation of genome-wide guide activity profiles C_LIO_LIActivity-corrected fitness screens identify a high confidence set of essential genes in K. phaffii C_LIO_LIProtein production, secretion, and glycosylation pathways are essential in K. phaffii but not in other yeasts C_LI\nSelective enrichment of high-affinity clade II N2O-reducers in a mixed culture\nAuthors: Laureni, M.; Corbera Rubio, F.; Kim, D. D.; Browne, S.; Roothans, N.; Weissbrodt, D. G.; Olavarria, K. O.; de Jonge, N.; Yoon, S.; Pabst, M.; van Loosdrecht, M. C. M.\nScore: 2.8, Published: 2024-02-12 DOI: 10.1101/2024.02.09.579283\nMicroorganisms encoding for the N2O reductase (NosZ) are the only known biological sink of the potent greenhouse gas N2O, and are central to global N2O mitigation efforts. Yet, the ecological constraints selecting for different N2O-reducers strains and controlling the assembly of N2O-respiring communities remain largely unknown. Of particular biotechnological interest are clade II NosZ populations, which usually feature high N2O affinities and often lack other denitrification genes. Two planktonic N2O-respiring mixed cultures were enriched under limiting and excess dissolved N2O availability to assess the impact of substrate affinity and N2O cytotoxicity, respectively. Genome-resolved metaproteomics was used to infer the metabolism of the enriched populations. We show that clade II N2O-reducers outcompete clade I affiliates for N2O at sufficiently low sludge dilution rates (0.006 h-1), a scenario previously only theorized based on pure-cultures. Under N2O limitation, all enriched N2O reducers encoded and expressed only clade II NosZ, while also possessing other denitrification genes. Two Azonexus and Thauera genera affiliates dominated the culture. We explain their coexistence with the genome-inferred metabolic exchange of cobalamin intermediates. Conversely, under excess N2O, clade I and II populations coexisted. Notably, the single dominant N2O reducer (genus Azonexus) expressed most cobalamin biosynthesis marker genes, likely to contrast the continuous cobalamin inactivation by dissolved cytotoxic N2O concentrations (400 M). Ultimately, we demonstrate that the solids dilution rate controls the selection among NosZ clades, albeit the conditions selecting for genomes possessing the sole nosZ remain elusive. Additionally, we suggest the significance of N2O-cobalamin interactions in shaping the composition of N2O-respiring microbiomes.\nStructural determinants of co-translational protein complex assembly\nAuthors: Malik, S.; Venezian, J.; Lobov, A.; Heidenreich, M.; Garcia-Seisdedos, H.; Yeates, T. O.; Shiber, A.; Levy, E. D.\nScore: 29.7, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576408\nThe assembly of proteins into functional complexes is critical to lifes processes. While textbooks depict complex assembly as occurring between fully synthesized proteins, we know today that thousands of proteins in the human proteome assemble co-translationally during their synthesis. Why this process takes place, however, remains unknown. We show that co-translational assembly is governed by biophysical and structural characteristics of the protein complex, and involves mutually stabilized, intertwined subunits. Consequently, these subunits are also co-regulated across the central dogma, from transcription to protein degradation. Leveraging structural signatures with AlphaFold2-based predictions enables us to accurately predict co-translational assembly on a proteome-wide scale, which we validated by ribosome profiling, genetic perturbations, and smFISH experiments. Notably, the latter showed that co-translationally assembling subunits exhibit co-localized mRNAs. This work unveils a fundamental connection between protein structure and the translation process, highlighting the overarching impact of three-dimensional structure on gene expression, mRNA localization, and proteostasis. One Sentence SummaryProtein complexes with topologically intertwined subunits require co-translational assembly and synchronized proteostasis of subunits, with implications in protein stability, mRNA localization, and evolution. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=195 SRC=\"FIGDIR/small/576408v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (62K): org.highwire.dtl.DTLVardef@712a2org.highwire.dtl.DTLVardef@58894borg.highwire.dtl.DTLVardef@9db5c6org.highwire.dtl.DTLVardef@8eae4_HPS_FORMAT_FIGEXP M_FIG C_FIG\n",
  "wordCount" : "2505",
  "inLanguage": "en",
  "datePublished": "2024-02-14T10:38:48Z",
  "dateModified": "2024-02-14T10:38:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 14, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579143">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579143" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579143">
        <p class="paperTitle">Spatially Exploring RNA Biology in Archival Formalin-Fixed Paraffin-Embedded Tissues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579143" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579143" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bai, Z.; Zhang, D.; Gao, Y.; Tao, B.; Bao, S.; Enninful, A.; Zhang, D.; Su, G.; Tian, X.; Zhang, N.; Xiao, Y.; Liu, Y.; Gerstein, M.; Li, M.; Xing, Y.; Lu, J.; Xu, M. L.; Fan, R.</p>
        <p class="info">Score: 76.4, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579143' target='https://doi.org/10.1101/2024.02.06.579143'> 10.1101/2024.02.06.579143</a></p>
        <p class="abstract">Spatial transcriptomics has emerged as a powerful tool for dissecting spatial cellular heterogeneity but as of today is largely limited to gene expression analysis. Yet, the life of RNA molecules is multifaceted and dynamic, requiring spatial profiling of different RNA species throughout the life cycle to delve into the intricate RNA biology in complex tissues. Human disease-relevant tissues are commonly preserved as formalin-fixed and paraffin-embedded (FFPE) blocks, representing an important resource for human tissue specimens. The capability to spatially explore RNA biology in FFPE tissues holds transformative potential for human biology research and clinical histopathology. Here, we present Patho-DBiT combining in situ polyadenylation and deterministic barcoding for spatial full coverage transcriptome sequencing, tailored for probing the diverse landscape of RNA species even in clinically archived FFPE samples. It permits spatial co-profiling of gene expression and RNA processing, unveiling region-specific splicing isoforms, and high-sensitivity transcriptomic mapping of clinical tumor FFPE tissues stored for five years. Furthermore, genome-wide single nucleotide RNA variants can be captured to distinguish different malignant clones from non-malignant cells in human lymphomas. Patho-DBiT also maps microRNA-mRNA regulatory networks and RNA splicing dynamics, decoding their roles in spatial tumorigenesis trajectory. High resolution Patho-DBiT at the cellular level reveals a spatial neighborhood and traces the spatiotemporal kinetics driving tumor progression. Patho-DBiT stands poised as a valuable platform to unravel rich RNA biology in FFPE tissues to study human tissue biology and aid in clinical pathology evaluation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.11.579776">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.11.579776" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.11.579776">
        <p class="paperTitle">An Organism-Level Quantitative Flux Model of Mammalian Energy Metabolism</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.11.579776" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.11.579776" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yuan, B.; Inouye, K. E.; Hotamisligil, G.; Hui, S.</p>
        <p class="info">Score: 28.9, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.11.579776' target='https://doi.org/10.1101/2024.02.11.579776'> 10.1101/2024.02.11.579776</a></p>
        <p class="abstract">Mammalian tissues feed on nutrients in the blood circulation. At the organism-level, mammalian energy metabolism comprises of oxidation, interconverting, storing and releasing of circulating nutrients. Though much is known about the individual processes and nutrients, a holistic and quantitative model describing these processes for all major circulating nutrients is lacking. Here, by integrating isotope tracer infusion, mass spectrometry, and isotope gas analyzer measurement, we developed a framework to systematically quantify fluxes through these processes for 10 major circulating energy nutrients in mice, resulting in an organism-level quantitative flux model of energy metabolism. This model revealed in wildtype mice that circulating nutrients have more dominant metabolic cycling fluxes than their oxidation fluxes, with distinct partition between cycling and oxidation flux for individual circulating nutrients. Applications of this framework in obese mouse models showed on a per animal basis extensive elevation of metabolic cycling fluxes in ob/ob mice, but not in diet-induced obese mice. Thus, our framework describes quantitatively the functioning of energy metabolism at the organism-level, valuable for revealing new features of energy metabolism in physiological and disease conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579213">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579213" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579213">
        <p class="paperTitle">1.4 min Plasma Proteome Profiling via Nanoparticle Protein Corona and Direct Infusion Mass Spectrometry</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579213" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579213" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jiang, Y.; Meyer, J. G.</p>
        <p class="info">Score: 15.0, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579213' target='https://doi.org/10.1101/2024.02.06.579213'> 10.1101/2024.02.06.579213</a></p>
        <p class="abstract">Non-invasive detection of protein biomarkers in plasma is crucial for clinical purposes. Liquid chromatography mass spectrometry (LC-MS) is the gold standard technique for plasma proteome analysis, but despite recent advances, it remains limited by throughput, cost, and coverage. Here, we introduce a new hybrid method, which integrates direct infusion shotgun proteome analysis (DISPA) with nanoparticle (NP) protein coronas enrichment for high throughput and efficient plasma proteomic profiling. We realized over 280 protein identifications in 1.4 minutes collection time, which enables a potential throughput of approximately 1,000 samples daily. The identified proteins are involved in valuable pathways and 44 of the proteins are FDA approved biomarkers. The robustness and quantitative accuracy of this method were evaluated across multiple NPs and concentrations with a mean coefficient of variation at 17%. Moreover, different protein corona profiles were observed among various nanoparticles based on their distinct surface modifications, and all NP protein profiles exhibited deeper coverage and better quantification than neat plasma. Our streamlined workflow merges coverage and throughput with precise quantification, leveraging both DISPA and NP protein corona enrichments. This underscores the significant potential of DISPA when paired with NP sample preparation techniques for plasma proteome studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577354">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577354" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577354">
        <p class="paperTitle">Large-Scale Evaluation of Spatial Metabolomics Protocols and Technologies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577354" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577354" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Saharuka, V.; Vieira, L. M.; Stuart, L.; Ekelöf, M.; Molenaar, M. R.; Bailoni, A.; Ovchinnikova, K.; Soltwisch, J.; Bausbacher, T.; Jakob, D.; King, M.; Müller, M. A.; Oetjen, J.; Pace, C.; Pinto, F. E.; Strittmatter, N.; Velickovic, D.; Spengler, B.; Muddiman, D. C.; Liebeke, M.; Janfelt, C.; Goodwin, R.; Eberlin, L. S.; Anderton, C. R.; Hopf, C.; Dreisewerd, K.; Alexandrov, T.</p>
        <p class="info">Score: 25.6, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577354' target='https://doi.org/10.1101/2024.01.29.577354'> 10.1101/2024.01.29.577354</a></p>
        <p class="abstract">Spatial metabolomics using imaging mass spectrometry (MS) enables untargeted and label-free metabolite mapping in biological samples. Despite the range of available imaging MS protocols and technologies, our understanding of metabolite detection under specific conditions is limited due to sparse empirical data and predictive theories. Consequently, challenges persist in designing new experiments, and accurately annotating and interpreting data. In this study, we systematically measured the detectability of 172 biologically-relevant metabolites across common imaging MS protocols using custom reference samples. We evaluated 24 MALDI-imaging MS protocols for untargeted metabolomics, and demonstrated the applicability of our findings to complex biological samples through comparison with animal tissue data. We showcased the potential for extending our results to further analytes by predicting metabolite detectability based on molecular properties. Additionally, our interlaboratory comparison of 10 imaging MS technologies, including MALDI, DESI, and IR-MALDESI, showed extensive metabolite coverage and comparable results, underscoring the broad applicability of our findings within the imaging MS community. We share our results and data through a new interactive web application integrated with METASPACE. This resource offers an extensive catalogue of detectable metabolite ions, facilitating protocol selection, supporting data annotation, and benefiting future untargeted spatial metabolomics studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.03.569805">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.03.569805" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.03.569805">
        <p class="paperTitle">A foundational atlas of autism protein interactions reveals molecular convergence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.03.569805" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.03.569805" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, B.; Vartak, R.; Zaltsman, Y.; Naing, Z. Z. C.; Hennick, K. M.; Polacco, B. J.; Bashir, A.; Eckhardt, M.; Bouhaddou, M.; Xu, J.; Sun, N.; Lasser, M.; Zhou, Y.; Guiley, K. Z.; Chan, U.; Kaye, J. A.; Khare, P.; Drake, S.; Drury, V.; Burke, D. F.; Gonzalez, S.; Alkhairy, S.; Morris, M.; Baum, T.; Krasnoff, R.; Wang, S.; Pham, P.; Arbalaez, J.; Pratt, D.; Chag, S.; Rolland, T.; Bourgeron, T.; Finkbeiner, S.; Bandyopadhay, S.; Ideker, T.; Beltrao, P.; Willsey, H. R.; Obernier, K.; Nowakowski, T. J.; Huttenhain, R.; State, M. W.; Willsey, A. J.; Krogan, N. J.</p>
        <p class="info">Score: 60.7, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.03.569805' target='https://doi.org/10.1101/2023.12.03.569805'> 10.1101/2023.12.03.569805</a></p>
        <p class="abstract">Translating high-confidence (hc) autism spectrum disorder (ASD) genes into viable treatment targets remains elusive. We constructed a foundational protein-protein interaction (PPI) network in HEK293T cells involving 100 hcASD risk genes, revealing over 1,800 PPIs (87% novel). Interactors, expressed in the human brain and enriched for ASD but not schizophrenia genetic risk, converged on protein complexes involved in neurogenesis, tubulin biology, transcriptional regulation, and chromatin modification. A PPI map of 54 patient-derived missense variants identified differential physical interactions, and we leveraged AlphaFold-Multimer predictions to prioritize direct PPIs and specific variants for interrogation in Xenopus tropicalis and human forebrain organoids. A mutation in the transcription factor FOXP1 led to reconfiguration of DNA binding sites and altered development of deep cortical layer neurons in forebrain organoids. This work offers new insights into molecular mechanisms underlying ASD and describes a powerful platform to develop and test therapeutic strategies for many genetically-defined conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.27.577546">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.27.577546" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.27.577546">
        <p class="paperTitle">Pancreatic cancer mutationscape: revealing the link between modular restructuring and intervention efficacy amidst common mutations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.27.577546" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.27.577546" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Plaugher, D. R.; Murrugarra, D. R.</p>
        <p class="info">Score: 7.3, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.27.577546' target='https://doi.org/10.1101/2024.01.27.577546'> 10.1101/2024.01.27.577546</a></p>
        <p class="abstract">Across cancer types, the prognosis for pancreatic cancer (PC) is among the worst and options for treatment are limited. There is increasing evidence that biological systems, including PC, are modular in both structure and function. Complex biological signaling networks such as gene regulatory networks (GRNs) are proving to be composed of subcategories that are interconnected and hierarchically ranked. These networks contain highly dynamic processes that ultimately dictate cellular function over time. In this work, we use an established Boolean multicellular signaling network of PC to show that the variance in topological rankings of the most phenotypically influential modules implies a strong relationship between structure and function. We further show that induction of mutations alters the modular structure, which analogously influences the aggression and controllability of the disease in silico. We finally present evidence that the impact and location of mutations with respect to PC modular structure directly corresponds to the efficacy of single agent treatments in silico, because topologically deep mutations require deep targets for control.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.08.579519">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.08.579519" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.08.579519">
        <p class="paperTitle">Semi-supervised Bayesian integration of multiple spatial proteomics datasets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.08.579519" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.08.579519" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Coleman, S. D.; Breckels, L.; Waller, R. F.; Lilley, K. S.; Wallace, C.; Kirk, P. D. W.; Crook, O. M.</p>
        <p class="info">Score: 4.2, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.08.579519' target='https://doi.org/10.1101/2024.02.08.579519'> 10.1101/2024.02.08.579519</a></p>
        <p class="abstract">The subcellular localisation of proteins is a key determinant of their function. High-throughput analyses of these localisations can be performed using mass spectrometry-based spatial proteomics, which enables us to examine the localisation and relocalisation of proteins. Furthermore, complementary data sources can provide additional sources of functional or localisation information. Examples include protein annotations and other high-throughput `omic assays. Integrating these modalities can provide new insights as well as additional confidence in results, but existing approaches for integrative analyses of spatial proteomics datasets are limited in the types of data they can integrate and do not quantify uncertainty. Here we propose a semi-supervised Bayesian approach to integrate spatial proteomics datasets with other data sources, to improve the inference of protein sub-cellular localisation. We demonstrate our approach outperforms other transfer-learning methods and has greater flexibility in the data it can model. To demonstrate the flexibility of our approach, we apply our method to integrate spatial proteomics data generated for the parasite Toxoplasma gondii with time-course gene expression data generated over its cell cycle. Our findings suggest that proteins linked to invasion organelles are associated with expression programs that peak at the end of the first cell-cycle. Furthermore, this integrative analysis divides the dense granule proteins into heterogeneous populations suggestive of potentially different functions. Our method is disseminated via the mdir R package available on the lead author&#39;s Github.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.08.579509">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.08.579509" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.08.579509">
        <p class="paperTitle">Functional genomic screening in Komagataella phaffii enabled by high-activity CRISPR-Cas9 library</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.08.579509" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.08.579509" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tafrishi, A.; Trivedi, V.; Xing, Z.; Li, M.; Mewalal, R.; Cutler, S.; Blaby, I.; Wheeldon, I.</p>
        <p class="info">Score: 3.2, Published: 2024-02-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.08.579509' target='https://doi.org/10.1101/2024.02.08.579509'> 10.1101/2024.02.08.579509</a></p>
        <p class="abstract">CRISPR-based high-throughput genome-wide loss-of-function screens are a valuable approach to functional genetics and strain engineering. The yeast Komagataella phaffii is a host of particular interest in the biopharmaceutical industry and as a metabolic engineering host for proteins and metabolites. Here, we design and validate a highly active 6-fold coverage genome-wide sgRNA library for this biotechnologically important yeast containing 30,848 active sgRNAs targeting over 99% of its coding sequences. Conducting fitness screens in the absence of functional non-homologous end joining (NHEJ), the dominant DNA repair mechanism in K. phaffii, provides a quantitative means to assess the activity of each sgRNA in the library. This approach allows for the experimental validation of each guides targeting activity, leading to more precise screening outcomes. We used this approach to conduct growth screens with glucose as the sole carbon source and identify essential genes. Comparative analysis of the called gene sets identified a core set of K. phaffii essential genes, many of which relate to protein production, secretion, and glycosylation. The high activity, genome-wide CRISPR library developed here enables functional genomic screening in K. phaffii, applied here to gene essentiality classification, and promises to enable other genetic screens.

HighlightsO_LIDesigned and validated a high activity genome-wide CRISPR-Cas9 library for K. phaffii
C_LIO_LIDisabling NHEJ DNA repair enables the generation of genome-wide guide activity profiles
C_LIO_LIActivity-corrected fitness screens identify a high confidence set of essential genes in K. phaffii
C_LIO_LIProtein production, secretion, and glycosylation pathways are essential in K. phaffii but not in other yeasts
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579283">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579283" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579283">
        <p class="paperTitle">Selective enrichment of high-affinity clade II N2O-reducers in a mixed culture</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579283" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579283" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Laureni, M.; Corbera Rubio, F.; Kim, D. D.; Browne, S.; Roothans, N.; Weissbrodt, D. G.; Olavarria, K. O.; de Jonge, N.; Yoon, S.; Pabst, M.; van Loosdrecht, M. C. M.</p>
        <p class="info">Score: 2.8, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579283' target='https://doi.org/10.1101/2024.02.09.579283'> 10.1101/2024.02.09.579283</a></p>
        <p class="abstract">Microorganisms encoding for the N2O reductase (NosZ) are the only known biological sink of the potent greenhouse gas N2O, and are central to global N2O mitigation efforts. Yet, the ecological constraints selecting for different N2O-reducers strains and controlling the assembly of N2O-respiring communities remain largely unknown. Of particular biotechnological interest are clade II NosZ populations, which usually feature high N2O affinities and often lack other denitrification genes. Two planktonic N2O-respiring mixed cultures were enriched under limiting and excess dissolved N2O availability to assess the impact of substrate affinity and N2O cytotoxicity, respectively. Genome-resolved metaproteomics was used to infer the metabolism of the enriched populations. We show that clade II N2O-reducers outcompete clade I affiliates for N2O at sufficiently low sludge dilution rates (0.006 h-1), a scenario previously only theorized based on pure-cultures. Under N2O limitation, all enriched N2O reducers encoded and expressed only clade II NosZ, while also possessing other denitrification genes. Two Azonexus and Thauera genera affiliates dominated the culture. We explain their coexistence with the genome-inferred metabolic exchange of cobalamin intermediates. Conversely, under excess N2O, clade I and II populations coexisted. Notably, the single dominant N2O reducer (genus Azonexus) expressed most cobalamin biosynthesis marker genes, likely to contrast the continuous cobalamin inactivation by dissolved cytotoxic N2O concentrations (400 M). Ultimately, we demonstrate that the solids dilution rate controls the selection among NosZ clades, albeit the conditions selecting for genomes possessing the sole nosZ remain elusive. Additionally, we suggest the significance of N2O-cobalamin interactions in shaping the composition of N2O-respiring microbiomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576408">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576408" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576408">
        <p class="paperTitle">Structural determinants of co-translational protein complex assembly</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576408" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576408" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Malik, S.; Venezian, J.; Lobov, A.; Heidenreich, M.; Garcia-Seisdedos, H.; Yeates, T. O.; Shiber, A.; Levy, E. D.</p>
        <p class="info">Score: 29.7, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576408' target='https://doi.org/10.1101/2024.01.20.576408'> 10.1101/2024.01.20.576408</a></p>
        <p class="abstract">The assembly of proteins into functional complexes is critical to lifes processes. While textbooks depict complex assembly as occurring between fully synthesized proteins, we know today that thousands of proteins in the human proteome assemble co-translationally during their synthesis. Why this process takes place, however, remains unknown. We show that co-translational assembly is governed by biophysical and structural characteristics of the protein complex, and involves mutually stabilized, intertwined subunits. Consequently, these subunits are also co-regulated across the central dogma, from transcription to protein degradation. Leveraging structural signatures with AlphaFold2-based predictions enables us to accurately predict co-translational assembly on a proteome-wide scale, which we validated by ribosome profiling, genetic perturbations, and smFISH experiments. Notably, the latter showed that co-translationally assembling subunits exhibit co-localized mRNAs. This work unveils a fundamental connection between protein structure and the translation process, highlighting the overarching impact of three-dimensional structure on gene expression, mRNA localization, and proteostasis.

One Sentence SummaryProtein complexes with topologically intertwined subunits require co-translational assembly and synchronized proteostasis of subunits, with implications in protein stability, mRNA localization, and evolution.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=195 SRC=&#34;FIGDIR/small/576408v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (62K):
org.highwire.dtl.DTLVardef@712a2org.highwire.dtl.DTLVardef@58894borg.highwire.dtl.DTLVardef@9db5c6org.highwire.dtl.DTLVardef@8eae4_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
