<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Allometry of cell types in planarians by single cell transcriptomics
Authors: Emili, E.; Perez-Posada, A.; Christodoulou, M. D.; Solana, J.
Score: 63.6, Published: 2023-11-04 DOI: 10.1101/2023.11.01.565140
Allometry explores the relationship between an organisms body size and its various components, offering insights into ecology, physiology, metabolism, and disease. The cell is the basic unit of biological systems, and yet, the study of cell type allometry remains relatively unexplored. Single-cell RNA sequencing (scRNA-seq) provides a promising tool for investigating cell type allometry.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Allometry of cell types in planarians by single cell transcriptomics
Authors: Emili, E.; Perez-Posada, A.; Christodoulou, M. D.; Solana, J.
Score: 63.6, Published: 2023-11-04 DOI: 10.1101/2023.11.01.565140
Allometry explores the relationship between an organisms body size and its various components, offering insights into ecology, physiology, metabolism, and disease. The cell is the basic unit of biological systems, and yet, the study of cell type allometry remains relatively unexplored. Single-cell RNA sequencing (scRNA-seq) provides a promising tool for investigating cell type allometry." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-15T10:40:26+00:00" />
<meta property="article:modified_time" content="2023-11-15T10:40:26+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Allometry of cell types in planarians by single cell transcriptomics
Authors: Emili, E.; Perez-Posada, A.; Christodoulou, M. D.; Solana, J.
Score: 63.6, Published: 2023-11-04 DOI: 10.1101/2023.11.01.565140
Allometry explores the relationship between an organisms body size and its various components, offering insights into ecology, physiology, metabolism, and disease. The cell is the basic unit of biological systems, and yet, the study of cell type allometry remains relatively unexplored. Single-cell RNA sequencing (scRNA-seq) provides a promising tool for investigating cell type allometry."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Allometry of cell types in planarians by single cell transcriptomics\nAuthors: Emili, E.; Perez-Posada, A.; Christodoulou, M. D.; Solana, J.\nScore: 63.6, Published: 2023-11-04 DOI: 10.1101/2023.11.01.565140\nAllometry explores the relationship between an organisms body size and its various components, offering insights into ecology, physiology, metabolism, and disease. The cell is the basic unit of biological systems, and yet, the study of cell type allometry remains relatively unexplored. Single-cell RNA sequencing (scRNA-seq) provides a promising tool for investigating cell type allometry.",
  "keywords": [
    
  ],
  "articleBody": " Allometry of cell types in planarians by single cell transcriptomics\nAuthors: Emili, E.; Perez-Posada, A.; Christodoulou, M. D.; Solana, J.\nScore: 63.6, Published: 2023-11-04 DOI: 10.1101/2023.11.01.565140\nAllometry explores the relationship between an organisms body size and its various components, offering insights into ecology, physiology, metabolism, and disease. The cell is the basic unit of biological systems, and yet, the study of cell type allometry remains relatively unexplored. Single-cell RNA sequencing (scRNA-seq) provides a promising tool for investigating cell type allometry. Planarians, capable of growing and degrowing following allometric scaling rules, serve as an excellent model for such studies. We used scRNA-seq to examine cell type allometry in asexual planarians of different sizes, revealing that they consist of the same basic cell types but in varying proportions. Notably, the gut basal cells are the most responsive to changes in size, suggesting a role in energy storage. We capture the gene regulatory programs of distinct cell types in response to size. This research sheds light on the molecular and cellular aspects of cell type allometry in planarians and underscores the utility of scRNA-seq in such investigations.\nProteome-wide base editor screens to assess phosphorylation site functionality in high-throughput\nAuthors: Kennedy, P. H.; Deh Sheikh, A. A.; Balakar, M.; Jones, A. C.; Olive, M. E.; Hedge, M.; Matias, M. I.; Pirete, N.; Burt, R.; Levy, J.; Little, T.; Hogan, P. G.; Liu, D. R.; Doench, J. G.; Newton, A. C.; Gottschalk, R. A.; de Boer, C.; Alarcon, S.; Newby, G.; Myers, S. A.\nScore: 8.5, Published: 2023-11-14 DOI: 10.1101/2023.11.11.566649\nSignaling pathways that drive gene expression are typically depicted as having a dozen or so landmark phosphorylation and transcriptional events. In reality, thousands of dynamic post-translational modifications (PTMs) orchestrate nearly every cellular function, and we lack technologies to find causal links between these vast biochemical pathways and genetic circuits at scale. Here, we describe \"signaling-to-transcription network\" mapping through the development of PTM-centric base editing coupled to phenotypic screens, directed by temporally-resolved phosphoproteomics. Using T cell activation as a model, we observe hundreds of unstudied phosphorylation sites that modulate NFAT transcriptional activity. We identify the phosphorylation-mediated nuclear localization of PHLPP1 which promotes NFAT but inhibits NF{kappa}B activity. We also find that specific phosphosite mutants can alter gene expression in subtle yet distinct patterns, demonstrating the potential for fine-tuning transcriptional responses. Overall, base editor screening of PTM sites provides a powerful platform to dissect PTM function within signaling pathways.\nShort-term Pre-operative Methionine Restriction Induces Browning of Perivascular Adipose Tissue and Improves Vein Graft Remodeling in Mice\nAuthors: Kip, P.; Sluiter, T. J.; MacArthur, M. R.; Tao, M.; Jung, J.; Mitchell, S. J.; Kooijman, S.; Kruit, N.; Gorham, J.; Seidman, J.; Quax, P. H. A.; Aikawa, M.; Ozaki, C. K.; Mitchell, J. R.; De Vries, M. R.\nScore: 4.1, Published: 2023-11-04 DOI: 10.1101/2023.11.02.565269\nShort-term preoperative methionine restriction (MetR) shows promise as a translatable strategy to modulate the bodys response to surgical injury. Its application, however, to improve post-interventional vascular remodeling remains underexplored. Here, we find that MetR protects from arterial intimal hyperplasia in a focal stenosis model and adverse vascular remodeling after vein graft surgery. RNA sequencing reveals that MetR enhances the brown adipose tissue phenotype in arterial perivascular adipose tissue (PVAT) and induces it in venous PVAT. Specifically, PPAR- was highly upregulated in PVAT-adipocytes. Furthermore, MetR dampens the post-operative pro-inflammatory response to surgery in PVAT-macrophages in vivo and in vitro. This study shows for the first time that the detrimental effects of dysfunctional PVAT on vascular remodeling can be reversed by MetR, and identifies pathways involved in browning of PVAT. Furthermore, we demonstrate the potential of short-term pre-operative MetR as a simple intervention to ameliorate vascular remodeling after vascular surgery. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=109 SRC=\"FIGDIR/small/565269v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (38K): org.highwire.dtl.DTLVardef@1f69558org.highwire.dtl.DTLVardef@1c6cea5org.highwire.dtl.DTLVardef@128c3f6org.highwire.dtl.DTLVardef@1594d18_HPS_FORMAT_FIGEXP M_FIG C_FIG\nModel-based inference of a plant-specific dual role for HOPS in regulating guard cell vacuole fusion\nAuthors: Hodgens, C.; Flaherty, D.; Pullen, A.-M.; Khan, I.; English, N. J.; Gillan, L.; Rojas-Pierce, M.; Akpa, B.\nScore: 1.5, Published: 2023-11-09 DOI: 10.1101/2023.11.07.565947\nStomata are the pores on a leaf surface that regulate gas exchange. Each stoma consists of two guard cells whose movements regulate pore opening and thereby control CO2 fixation and water loss. Guard cell movements depend in part on the remodeling of vacuoles, which have been observed to change from a highly fragmented state to a fused morphology during stomata opening. This change in morphology requires a membrane fusion mechanism that responds rapidly to environmental signals, allowing plants to respond to diurnal and stress cues. With guard cell vacuoles being both large and responsive to external signals, stomata represent a unique system in which to delineate mechanisms of membrane fusion. Fusion of vacuole membranes is a highly conserved process in eukaryotes, with key roles played by two multi-subunit complexes: HOPS (homotypic fusion and vacuolar protein sorting) and SNARE (soluble NSF attachment protein receptor). HOPS is a vacuole tethering factor that is thought to chaperone SNAREs from apposing vacuole membranes into a fusion-competent complex capable of rearranging membranes. To resolve a counter-intuitive observation regarding the role of HOPS in regulating plant vacuole morphology, we derived a quantitative model of vacuole fusion dynamics and used it to generate testable predictions about HOPS-SNARE interactions. We derived our model by applying simulation-based inference to integrate prior knowledge about molecular interactions with limited, qualitative observations of emergent vacuole phenotypes. By constraining the model parameters to yield the emergent outcomes observed for stoma opening - as induced by two distinct chemical treatments - we predicted a dual role for HOPS and identified a stalled form of the SNARE complex that differs from phenomena reported in yeast. We predict that HOPS has contradictory actions at different points in the fusion signaling pathway, promoting the formation of SNARE complexes, but limiting their activity. Author summaryPlants \"breathe\" through pores in their leaves where each pore is formed by two specialized cells called guard cells. To open these pores, guard cells change in volume. This volume change is controlled by water-filled organelles called vacuoles that morph from multiple small entities to a few large ones capable of taking up more water to reshape the cell. Specialized proteins in vacuole membranes make this change happen by pulling vacuoles together until they fuse. Some of these proteins reside in membranes, but others must be drawn to the membrane from the cells cytoplasm. Specific lipid molecules in the membrane play an important role in recruiting those proteins to the vacuole membrane. We previously made an unexpected finding that removing this lipid induces plant vacuole fusion. To make sense of this observation, we used a mathematical model to piece together our knowledge of the proteins involved in this process and what we know about the chemical treatments that cause vacuoles to morph. Using computer simulations, we uncovered new rules about how molecules interact in membranes to accomplish the task of vacuole fusion in plants. We think the rules uncovered through mathematical modeling allow plants to respond quickly to environmental cues.\nDetermination of Metabolic Fluxes by Deep Learning of Isotope Labeling Patterns\nAuthors: Law, R. C.; O'Keeffe, S.; Nurwono, G.; Ki, R.; Lakhani, A.; Lai, P.-K.; Park, J. O.\nScore: 1.5, Published: 2023-11-08 DOI: 10.1101/2023.11.06.565907\nFluxomics offers a direct readout of metabolic state but relies on indirect measurement. Stable isotope tracers imprint flux-dependent isotope labeling patterns on metabolites we measure; however, the relationship between labeling patterns and fluxes remains elusive. Here we innovate a two-stage machine learning framework termed ML-Flux that streamlines metabolic flux quantitation from isotope tracing. We train machine learning models by simulating atom transitions across five universal metabolic models starting from 26 13C-glucose, 2H-glucose, and 13C-glutamine tracers within feasible flux space. ML-Flux employs deep-learning-based imputation to take variable measurements of labeling patterns as input and successive neural networks to convert the ensuing comprehensive labeling information into metabolic fluxes. Using ML-Flux with multi-isotope tracing, we obtain fluxes through central carbon metabolism that are comparable to those from a least-squares method but orders-of-magnitude faster. ML-Flux is deployed as a webtool to expand the accessibility of metabolic flux quantitation and afford actionable information on metabolism.\nIdentification and Pharmacological Targeting of Treatment-Resistant, Stem-like Breast Cancer Cells for Combination Therapy\nAuthors: Worley, J.; Noh, H.; You, D.; Turunen, M.; Ding, H.; Paull, E. O.; Griffen, A.; Grunn, A.; Zhang, M.; Guillan, K.; Bush, E.; Brosius, S.; Hibshoosh, H.; Mundi, P. S.; Sims, P.; Dalerba, P.; Dela Cruz, F. S.; Kung, A. L.; Califano, A.\nScore: 1.0, Published: 2023-11-10 DOI: 10.1101/2023.11.08.562798\nTumors frequently harbor isogenic yet epigenetically distinct subpopulations of multi-potent cells with high tumor-initiating potential--often called Cancer Stem-Like Cells (CSLCs). These can display preferential resistance to standard-of-care chemotherapy. Single-cell analyses can help elucidate Master Regulator (MR) proteins responsible for governing the transcriptional state of these cells, thus revealing complementary dependencies that may be leveraged via combination therapy. Interrogation of single-cell RNA sequencing profiles from seven metastatic breast cancer patients, using perturbational profiles of clinically relevant drugs, identified drugs predicted to invert the activity of MR proteins governing the transcriptional state of chemoresistant CSLCs, which were then validated by CROP-seq assays. The top drug, the anthelmintic albendazole, depleted this subpopulation in vivo without noticeable cytotoxicity. Moreover, sequential cycles of albendazole and paclitaxel--a commonly used chemotherapeutic --displayed significant synergy in a patient-derived xenograft (PDX) from a TNBC patient, suggesting that network-based approaches can help develop mechanism-based combinatorial therapies targeting complementary subpopulations.\n",
  "wordCount" : "1560",
  "inLanguage": "en",
  "datePublished": "2023-11-15T10:40:26Z",
  "dateModified": "2023-11-15T10:40:26Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 15, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565140">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565140" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565140">
        <p class="paperTitle">Allometry of cell types in planarians by single cell transcriptomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565140" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565140" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Emili, E.; Perez-Posada, A.; Christodoulou, M. D.; Solana, J.</p>
        <p class="info">Score: 63.6, Published: 2023-11-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565140' target='https://doi.org/10.1101/2023.11.01.565140'> 10.1101/2023.11.01.565140</a></p>
        <p class="abstract">Allometry explores the relationship between an organisms body size and its various components, offering insights into ecology, physiology, metabolism, and disease. The cell is the basic unit of biological systems, and yet, the study of cell type allometry remains relatively unexplored. Single-cell RNA sequencing (scRNA-seq) provides a promising tool for investigating cell type allometry. Planarians, capable of growing and degrowing following allometric scaling rules, serve as an excellent model for such studies. We used scRNA-seq to examine cell type allometry in asexual planarians of different sizes, revealing that they consist of the same basic cell types but in varying proportions. Notably, the gut basal cells are the most responsive to changes in size, suggesting a role in energy storage. We capture the gene regulatory programs of distinct cell types in response to size. This research sheds light on the molecular and cellular aspects of cell type allometry in planarians and underscores the utility of scRNA-seq in such investigations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.11.566649">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.11.566649" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.11.566649">
        <p class="paperTitle">Proteome-wide base editor screens to assess phosphorylation site functionality in high-throughput</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.11.566649" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.11.566649" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kennedy, P. H.; Deh Sheikh, A. A.; Balakar, M.; Jones, A. C.; Olive, M. E.; Hedge, M.; Matias, M. I.; Pirete, N.; Burt, R.; Levy, J.; Little, T.; Hogan, P. G.; Liu, D. R.; Doench, J. G.; Newton, A. C.; Gottschalk, R. A.; de Boer, C.; Alarcon, S.; Newby, G.; Myers, S. A.</p>
        <p class="info">Score: 8.5, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.11.566649' target='https://doi.org/10.1101/2023.11.11.566649'> 10.1101/2023.11.11.566649</a></p>
        <p class="abstract">Signaling pathways that drive gene expression are typically depicted as having a dozen or so landmark phosphorylation and transcriptional events. In reality, thousands of dynamic post-translational modifications (PTMs) orchestrate nearly every cellular function, and we lack technologies to find causal links between these vast biochemical pathways and genetic circuits at scale. Here, we describe &#34;signaling-to-transcription network&#34; mapping through the development of PTM-centric base editing coupled to phenotypic screens, directed by temporally-resolved phosphoproteomics. Using T cell activation as a model, we observe hundreds of unstudied phosphorylation sites that modulate NFAT transcriptional activity. We identify the phosphorylation-mediated nuclear localization of PHLPP1 which promotes NFAT but inhibits NF{kappa}B activity. We also find that specific phosphosite mutants can alter gene expression in subtle yet distinct patterns, demonstrating the potential for fine-tuning transcriptional responses. Overall, base editor screening of PTM sites provides a powerful platform to dissect PTM function within signaling pathways.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.565269">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.565269" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.565269">
        <p class="paperTitle">Short-term Pre-operative Methionine Restriction Induces Browning of Perivascular Adipose Tissue and Improves Vein Graft Remodeling in Mice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.565269" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.565269" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kip, P.; Sluiter, T. J.; MacArthur, M. R.; Tao, M.; Jung, J.; Mitchell, S. J.; Kooijman, S.; Kruit, N.; Gorham, J.; Seidman, J.; Quax, P. H. A.; Aikawa, M.; Ozaki, C. K.; Mitchell, J. R.; De Vries, M. R.</p>
        <p class="info">Score: 4.1, Published: 2023-11-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.565269' target='https://doi.org/10.1101/2023.11.02.565269'> 10.1101/2023.11.02.565269</a></p>
        <p class="abstract">Short-term preoperative methionine restriction (MetR) shows promise as a translatable strategy to modulate the bodys response to surgical injury. Its application, however, to improve post-interventional vascular remodeling remains underexplored. Here, we find that MetR protects from arterial intimal hyperplasia in a focal stenosis model and adverse vascular remodeling after vein graft surgery. RNA sequencing reveals that MetR enhances the brown adipose tissue phenotype in arterial perivascular adipose tissue (PVAT) and induces it in venous PVAT. Specifically, PPAR- was highly upregulated in PVAT-adipocytes. Furthermore, MetR dampens the post-operative pro-inflammatory response to surgery in PVAT-macrophages in vivo and in vitro. This study shows for the first time that the detrimental effects of dysfunctional PVAT on vascular remodeling can be reversed by MetR, and identifies pathways involved in browning of PVAT. Furthermore, we demonstrate the potential of short-term pre-operative MetR as a simple intervention to ameliorate vascular remodeling after vascular surgery.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=109 SRC=&#34;FIGDIR/small/565269v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (38K):
org.highwire.dtl.DTLVardef@1f69558org.highwire.dtl.DTLVardef@1c6cea5org.highwire.dtl.DTLVardef@128c3f6org.highwire.dtl.DTLVardef@1594d18_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.565947">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.565947" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.565947">
        <p class="paperTitle">Model-based inference of a plant-specific dual role for HOPS in regulating guard cell vacuole fusion</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.565947" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.565947" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hodgens, C.; Flaherty, D.; Pullen, A.-M.; Khan, I.; English, N. J.; Gillan, L.; Rojas-Pierce, M.; Akpa, B.</p>
        <p class="info">Score: 1.5, Published: 2023-11-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.565947' target='https://doi.org/10.1101/2023.11.07.565947'> 10.1101/2023.11.07.565947</a></p>
        <p class="abstract">Stomata are the pores on a leaf surface that regulate gas exchange. Each stoma consists of two guard cells whose movements regulate pore opening and thereby control CO2 fixation and water loss. Guard cell movements depend in part on the remodeling of vacuoles, which have been observed to change from a highly fragmented state to a fused morphology during stomata opening. This change in morphology requires a membrane fusion mechanism that responds rapidly to environmental signals, allowing plants to respond to diurnal and stress cues. With guard cell vacuoles being both large and responsive to external signals, stomata represent a unique system in which to delineate mechanisms of membrane fusion.

Fusion of vacuole membranes is a highly conserved process in eukaryotes, with key roles played by two multi-subunit complexes: HOPS (homotypic fusion and vacuolar protein sorting) and SNARE (soluble NSF attachment protein receptor). HOPS is a vacuole tethering factor that is thought to chaperone SNAREs from apposing vacuole membranes into a fusion-competent complex capable of rearranging membranes. To resolve a counter-intuitive observation regarding the role of HOPS in regulating plant vacuole morphology, we derived a quantitative model of vacuole fusion dynamics and used it to generate testable predictions about HOPS-SNARE interactions. We derived our model by applying simulation-based inference to integrate prior knowledge about molecular interactions with limited, qualitative observations of emergent vacuole phenotypes. By constraining the model parameters to yield the emergent outcomes observed for stoma opening - as induced by two distinct chemical treatments - we predicted a dual role for HOPS and identified a stalled form of the SNARE complex that differs from phenomena reported in yeast. We predict that HOPS has contradictory actions at different points in the fusion signaling pathway, promoting the formation of SNARE complexes, but limiting their activity.

Author summaryPlants &#34;breathe&#34; through pores in their leaves where each pore is formed by two specialized cells called guard cells. To open these pores, guard cells change in volume. This volume change is controlled by water-filled organelles called vacuoles that morph from multiple small entities to a few large ones capable of taking up more water to reshape the cell. Specialized proteins in vacuole membranes make this change happen by pulling vacuoles together until they fuse. Some of these proteins reside in membranes, but others must be drawn to the membrane from the cells cytoplasm. Specific lipid molecules in the membrane play an important role in recruiting those proteins to the vacuole membrane. We previously made an unexpected finding that removing this lipid induces plant vacuole fusion. To make sense of this observation, we used a mathematical model to piece together our knowledge of the proteins involved in this process and what we know about the chemical treatments that cause vacuoles to morph. Using computer simulations, we uncovered new rules about how molecules interact in membranes to accomplish the task of vacuole fusion in plants. We think the rules uncovered through mathematical modeling allow plants to respond quickly to environmental cues.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565907">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565907" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565907">
        <p class="paperTitle">Determination of Metabolic Fluxes by Deep Learning of Isotope Labeling Patterns</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565907" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565907" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Law, R. C.; O&#39;Keeffe, S.; Nurwono, G.; Ki, R.; Lakhani, A.; Lai, P.-K.; Park, J. O.</p>
        <p class="info">Score: 1.5, Published: 2023-11-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565907' target='https://doi.org/10.1101/2023.11.06.565907'> 10.1101/2023.11.06.565907</a></p>
        <p class="abstract">Fluxomics offers a direct readout of metabolic state but relies on indirect measurement. Stable isotope tracers imprint flux-dependent isotope labeling patterns on metabolites we measure; however, the relationship between labeling patterns and fluxes remains elusive. Here we innovate a two-stage machine learning framework termed ML-Flux that streamlines metabolic flux quantitation from isotope tracing. We train machine learning models by simulating atom transitions across five universal metabolic models starting from 26 13C-glucose, 2H-glucose, and 13C-glutamine tracers within feasible flux space. ML-Flux employs deep-learning-based imputation to take variable measurements of labeling patterns as input and successive neural networks to convert the ensuing comprehensive labeling information into metabolic fluxes. Using ML-Flux with multi-isotope tracing, we obtain fluxes through central carbon metabolism that are comparable to those from a least-squares method but orders-of-magnitude faster. ML-Flux is deployed as a webtool to expand the accessibility of metabolic flux quantitation and afford actionable information on metabolism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.08.562798">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.08.562798" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.08.562798">
        <p class="paperTitle">Identification and Pharmacological Targeting of Treatment-Resistant, Stem-like Breast Cancer Cells for Combination Therapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.08.562798" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.08.562798" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Worley, J.; Noh, H.; You, D.; Turunen, M.; Ding, H.; Paull, E. O.; Griffen, A.; Grunn, A.; Zhang, M.; Guillan, K.; Bush, E.; Brosius, S.; Hibshoosh, H.; Mundi, P. S.; Sims, P.; Dalerba, P.; Dela Cruz, F. S.; Kung, A. L.; Califano, A.</p>
        <p class="info">Score: 1.0, Published: 2023-11-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.08.562798' target='https://doi.org/10.1101/2023.11.08.562798'> 10.1101/2023.11.08.562798</a></p>
        <p class="abstract">Tumors frequently harbor isogenic yet epigenetically distinct subpopulations of multi-potent cells with high tumor-initiating potential--often called Cancer Stem-Like Cells (CSLCs). These can display preferential resistance to standard-of-care chemotherapy. Single-cell analyses can help elucidate Master Regulator (MR) proteins responsible for governing the transcriptional state of these cells, thus revealing complementary dependencies that may be leveraged via combination therapy. Interrogation of single-cell RNA sequencing profiles from seven metastatic breast cancer patients, using perturbational profiles of clinically relevant drugs, identified drugs predicted to invert the activity of MR proteins governing the transcriptional state of chemoresistant CSLCs, which were then validated by CROP-seq assays. The top drug, the anthelmintic albendazole, depleted this subpopulation in vivo without noticeable cytotoxicity. Moreover, sequential cycles of albendazole and paclitaxel--a commonly used chemotherapeutic --displayed significant synergy in a patient-derived xenograft (PDX) from a TNBC patient, suggesting that network-based approaches can help develop mechanism-based combinatorial therapies targeting complementary subpopulations.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
