<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A foundational atlas of autism protein interactions reveals molecular convergence
Authors: Wang, B.; Vartak, R.; Zaltsman, Y.; Naing, Z. Z. C.; Hennick, K. M.; Polacco, B. J.; Bashir, A.; Eckhardt, M.; Bouhaddou, M.; Xu, J.; Sun, N.; Lasser, M.; Zhou, Y.; Guiley, K. Z.; Chan, U.; Kaye, J. A.; Khare, P.; Drake, S.; Drury, V.; Burke, D. F.; Gonzalez, S.; Alkhairy, S.; Morris, M.; Baum, T.; Krasnoff, R.; Wang, S.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="A foundational atlas of autism protein interactions reveals molecular convergence
Authors: Wang, B.; Vartak, R.; Zaltsman, Y.; Naing, Z. Z. C.; Hennick, K. M.; Polacco, B. J.; Bashir, A.; Eckhardt, M.; Bouhaddou, M.; Xu, J.; Sun, N.; Lasser, M.; Zhou, Y.; Guiley, K. Z.; Chan, U.; Kaye, J. A.; Khare, P.; Drake, S.; Drury, V.; Burke, D. F.; Gonzalez, S.; Alkhairy, S.; Morris, M.; Baum, T.; Krasnoff, R.; Wang, S." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-24T10:38:02+00:00" />
<meta property="article:modified_time" content="2023-12-24T10:38:02+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="A foundational atlas of autism protein interactions reveals molecular convergence
Authors: Wang, B.; Vartak, R.; Zaltsman, Y.; Naing, Z. Z. C.; Hennick, K. M.; Polacco, B. J.; Bashir, A.; Eckhardt, M.; Bouhaddou, M.; Xu, J.; Sun, N.; Lasser, M.; Zhou, Y.; Guiley, K. Z.; Chan, U.; Kaye, J. A.; Khare, P.; Drake, S.; Drury, V.; Burke, D. F.; Gonzalez, S.; Alkhairy, S.; Morris, M.; Baum, T.; Krasnoff, R.; Wang, S."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "A foundational atlas of autism protein interactions reveals molecular convergence\nAuthors: Wang, B.; Vartak, R.; Zaltsman, Y.; Naing, Z. Z. C.; Hennick, K. M.; Polacco, B. J.; Bashir, A.; Eckhardt, M.; Bouhaddou, M.; Xu, J.; Sun, N.; Lasser, M.; Zhou, Y.; Guiley, K. Z.; Chan, U.; Kaye, J. A.; Khare, P.; Drake, S.; Drury, V.; Burke, D. F.; Gonzalez, S.; Alkhairy, S.; Morris, M.; Baum, T.; Krasnoff, R.; Wang, S.",
  "keywords": [
    
  ],
  "articleBody": " A foundational atlas of autism protein interactions reveals molecular convergence\nAuthors: Wang, B.; Vartak, R.; Zaltsman, Y.; Naing, Z. Z. C.; Hennick, K. M.; Polacco, B. J.; Bashir, A.; Eckhardt, M.; Bouhaddou, M.; Xu, J.; Sun, N.; Lasser, M.; Zhou, Y.; Guiley, K. Z.; Chan, U.; Kaye, J. A.; Khare, P.; Drake, S.; Drury, V.; Burke, D. F.; Gonzalez, S.; Alkhairy, S.; Morris, M.; Baum, T.; Krasnoff, R.; Wang, S.; Pham, P.; Arbalaez, J.; Pratt, D.; Chag, S.; Rolland, T.; Bourgeron, T.; Finkbeiner, S.; Bandyopadhay, S.; Ideker, T.; Beltrao, P.; Willsey, H. R.; Obernier, K.; Nowakowski, T. J.; Huttenhain, R.; State, M. W.; Willsey, A. J.; Krogan, N. J.\nScore: 57.8, Published: 2023-12-09 DOI: 10.1101/2023.12.03.569805\nTranslating high-confidence (hc) autism spectrum disorder (ASD) genes into viable treatment targets remains elusive. We constructed a foundational protein-protein interaction (PPI) network in HEK293T cells involving 100 hcASD risk genes, revealing over 1,800 PPIs (87% novel). Interactors, expressed in the human brain and enriched for ASD but not schizophrenia genetic risk, converged on protein complexes involved in neurogenesis, tubulin biology, transcriptional regulation, and chromatin modification. A PPI map of 54 patient-derived missense variants identified differential physical interactions, and we leveraged AlphaFold-Multimer predictions to prioritize direct PPIs and specific variants for interrogation in Xenopus tropicalis and human forebrain organoids. A mutation in the transcription factor FOXP1 led to reconfiguration of DNA binding sites and altered development of deep cortical layer neurons in forebrain organoids. This work offers new insights into molecular mechanisms underlying ASD and describes a powerful platform to develop and test therapeutic strategies for many genetically-defined conditions.\nA distinctive evolution of alveolar T cell responses is associated with clinical outcomes in unvaccinated patients with SARS-CoV-2 pneumonia\nAuthors: Markov, N. S.; Ren, Z.; Senkow, K. J.; Grant, R. A.; Gao, C. A.; Malsin, E. S.; Sichizya, L.; Kihshen, H.; Helmin, K. A.; Jovisic, M.; Arnold, J. M.; Perez-Leonor, X. G.; Abdala-Valencia, H.; Swaminathan, S.; Nwaezeapu, J.; Kang, M.; Rasmussen, L.; Ozer, E. A.; Lorenzo-Redondo, R.; Hultquist, J. F.; Simons, L.; Rios-Guzman, E.; Misharin, A. V.; Wunderink, R. G.; Budinger, G. S.; Singer, B. D.; Morales-Nebreda, L.; The NU SCRIPT Investigators, Score: 8.1, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571479\nPathogen clearance and resolution of inflammation in patients with pneumonia require an effective local T cell response. Nevertheless, local T cell activation may drive lung injury, particularly during prolonged episodes of respiratory failure characteristic of severe SARS-CoV-2 pneumonia. While T cell responses in the peripheral blood are well described, the evolution of T cell phenotypes and molecular signatures in the distal lung of patients with severe pneumonia caused by SARS-CoV-2 or other pathogens is understudied. Accordingly, we serially obtained 432 bronchoalveolar lavage fluid samples from 273 patients with severe pneumonia and respiratory failure, including 74 unvaccinated patients with COVID-19, and performed flow cytometry, transcriptional, and T cell receptor profiling on sorted CD8+ and CD4+ T cell subsets. In patients with COVID-19 but not pneumonia secondary to other pathogens, we found that early and persistent enrichment in CD8+ and CD4+ T cell subsets correlated with survival to hospital discharge. Activation of interferon signaling pathways early after intubation for COVID-19 was associated with favorable outcomes, while activation of NF-{kappa}B-driven programs late in disease was associated with poor outcomes. Patients with SARS-CoV-2 pneumonia whose alveolar T cells preferentially targeted the Spike and Nucleocapsid proteins tended to experience more favorable outcomes than patients whose T cells predominantly targeted the ORF1ab polyprotein complex. These results suggest that in patients with severe SARS-CoV-2 pneumonia, alveolar T cell interferon responses targeting structural SARS-CoV-2 proteins characterize patients who recover, yet these responses progress to NF-{kappa}B activation against non-structural proteins in patients who go on to experience poor clinical outcomes.\nMachine Learning and Metabolic Model Guided CRISPRi Reveals a Central Role for Phosphoglycerate Mutase in Chlamydia trachomatis Persistence\nAuthors: Chowdhury, N. B.; Pokorzynski, N. D.; Rucks, E. A.; Ouellette, S. P.; CARABEO, R.; Saha, R.\nScore: 3.6, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572198\nUpon nutrient starvation, Chlamydia trachomatis serovar L2 (CTL) shifts from its normal growth to a non-replicating form, termed persistence. It is unclear if persistence is an adaptive response or lack of it. To understand that transcriptomics data were collected for nutrient-sufficient and nutrient-starved CTL. Applying machine learning approaches on transcriptomics data revealed a global transcriptomic rewiring of CTL under stress conditions without having any global stress regulator. This indicated that CTLs stress response is due to lack of an adaptive response mechanism. To investigate the impact of this on CTL metabolism, we reconstructed a genome-scale metabolic model of CTL (iCTL278) and contextualized it with the collected transcriptomics data. Using the metabolic bottleneck analysis on contextualized iCTL278, we observed phosphoglycerate mutase (pgm) regulates the entry of CTL to the persistence. Later, pgm was found to have the highest thermodynamics driving force and lowest enzymatic cost. Furthermore, CRISPRi-driven knockdown of pgm and tryptophan starvation experiments revealed the importance of this gene in inducing persistence. Hence, this work, for the first time, introduced thermodynamics and enzyme-cost as tools to gain deeper understanding on CTL persistence.\nMechanistic computational modeling of monospecific and bispecific antibodies targeting interleukin-6/8 receptors\nAuthors: Ray, C. M.; Yang, H.; Spangler, J. B.; Mac Gabhann, F.\nScore: 3.4, Published: 2023-12-19 DOI: 10.1101/2023.12.18.570445\nThe spread of cancer from organ to organ (metastasis) is responsible for the vast majority of cancer deaths; however, most current anti-cancer drugs are designed to arrest or reverse tumor growth without directly addressing disease spread. It was recently discovered that tumor cell-secreted interleukin-6 (IL-6) and interleukin-8 (IL-8) synergize to enhance cancer metastasis in a cell-density dependent manner, and blockade of the IL-6 and IL-8 receptors (IL-6R and IL-8R) with a novel bispecific antibody, BS1, significantly reduced metastatic burden in multiple preclinical mouse models of cancer. Bispecific antibodies (BsAbs), which combine two different antigen-binding sites into one molecule, are a promising modality for drug development due to their enhanced avidity and dual targeting effects. However, while BsAbs have tremendous therapeutic potential, elucidating the mechanisms underlying their binding and inhibition will be critical for maximizing the efficacy of new BsAb treatments. Here, we describe a quantitative, computational model of the BS1 BsAb, exhibiting how modeling multivalent binding provides key insights into antibody affinity and avidity effects and can guide therapeutic design. We present detailed simulations of the monovalent and bivalent binding interactions between different antibody constructs and the IL-6 and IL-8 receptors to establish how antibody properties and system conditions impact the formation of binary (antibody-receptor) and ternary (receptor-antibody-receptor) complexes. Model results demonstrate how the balance of these complex types drives receptor inhibition, providing important and generalizable predictions for effective therapeutic design.\nKinAce: a web portal for exploring kinase-substrate interactions\nAuthors: Sekar, J. A. P.; Li, Y. C.; Schlessinger, A.; Pandey, G.\nScore: 3.0, Published: 2023-12-10 DOI: 10.1101/2023.12.08.570875\nInteractions between protein kinases and their substrates are critical for the modulation of complex signaling pathways. Currently, there is a large amount of information available about kinases and their substrates in disparate public databases. However, these data are difficult to interpret in the context of cellular systems, which can be facilitated by examining interactions among multiple proteins at once, such as the network of interactions that constitute a signaling pathway. We present KinAce, a user-friendly web portal that integrates and shares information about kinase-substrate interactions from multiple databases of post-translational modifications. KinAce enables the visual exploration of these interactions in systems contexts, such as pathways, domain families, and custom protein set inputs, in an interactive fashion. We expect KinAce to be useful as a knowledge discovery tool for kinase-substrate interactions, and the aggregated KinAce dataset to be useful for protein kinase studies and systems-level analyses. The portal is available at https://kinace.kinametrix.com/.\nSelf-inhibiting percolation and viral spreading in epithelial tissue\nAuthors: Xu, X.; Nielsen, B. F.; sneppen, k.\nScore: 24.1, Published: 2023-12-13 DOI: 10.1101/2023.12.12.571279\nSARS-CoV-2 induces delayed type-I/III interferon production, allowing it to escape the early innate immune response. The delay has been attributed to a deficiency in the ability of cells to sense viral replication upon infection, which in turn hampers activation of the antiviral state in bystander cells. Here, we introduce a cellular automaton model to investigate the spatiotemporal spreading of viral infection as a function of virus and host-dependent parameters. The model suggests that the considerable person-to-person heterogeneity in SARS-CoV-2 infections is a consequence of high sensitivity to slight variations in biological parameters near a critical threshold. It further suggests that within-host viral proliferation can be curtailed by the presence of remarkably few cells that are primed for IFN production. Thus the observed heterogeneity in defense readiness of cells reflects a remarkably cost-efficient strategy for protection.\nA Mathematical Framework for Measuring and Tuning Tempo in Developmental Gene Regulatory Networks\nAuthors: Manser, C. L.; Perez-Carrasco, R.\nScore: 1.6, Published: 2023-12-19 DOI: 10.1101/2023.12.18.572181\nEmbryo development is a dynamic process governed by the regulation of timing and sequences of gene expression, which control the proper growth of the organism. While many genetic programs coordinating these sequences are common across species, the timescales of gene expression can vary significantly among different organisms. Currently, substantial experimental efforts are focused on identifying molecular mechanisms that control these temporal aspects. In contrast, the capacity of established mathematical models to incorporate tempo control while maintaining the same dynamical landscape remains less understood. This manuscript addresses this gap by developing a mathematical framework that links the functionality of developmental programs to the corresponding gene expression orbits (or landscapes). This unlocks the ability to find tempo differences as perturbations in the dynamical system that preserve its orbits. We demonstrate that this framework allows for the prediction of molecular mechanisms governing tempo, through both numerical and analytical methods. Our exploration includes two case studies: a generic network featuring coupled production and degradation, and the repressilator. In the latter, we illustrate how altering the dimerisation rates of transcription factors can decouple the tempo from the shape of the resulting orbits. The manuscript concludes by highlighting how the identification of orthogonal molecular mechanisms for tempo control can inform the design of circuits with specific orbits and tempos.\nDifferential proteome diversification in yeast populations: modes of short-term adaptation and fitness outcomes\nAuthors: Knafo, M.; Rezenman, S.; Tsigalnitski, I.; Nevo, R.; Reich, Z.; Kapon, R.\nScore: 11.4, Published: 2023-12-07 DOI: 10.1101/2023.11.28.568995\nShort-term proteomic adaptations serve as an initial line of defence, allowing populations to cope with environmental changes before long-term genetic alterations occur. Using a representative set of genes, we examined how stress affects gene expression variability for different types and levels of abiotic stresses and how this influences population-level adaptation. Our data reveal that, depending on the nature of the stress, two distinct modes of response can be employed. In one, the levels of most proteins vary between individuals, leading to varied fitness levels in the population. In the other, a more limited range of expression is seen, and fitness is more even. This suggests different levels of complexity and plasticity in adaptation to different types of stress.\nFast and Deep Phosphoproteome Analysis with the Orbitrap Astral Mass Spectrometer\nAuthors: Lancaster, N. M.; Sinitcyn, P.; Forny, P.; Peters-Clarke, T. M.; Fecher, C.; Smith, A. J.; Shishkova, E.; Arrey, T. N.; Pashkova, A.; Robinson, M. L.; Arp, N.; Fan, J.; Hansen, J.; Galmozzi, A.; Serrano, L. R.; Westphall, M. S.; Stewart, H.; Hock, C.; Damoc, E.; Pagliarini, D. J.; Zabrouskov, V.; Coon, J. J.\nScore: 27.4, Published: 2023-12-14 DOI: 10.1101/2023.11.21.568149\nOwing to its roles in cellular signal transduction, protein phosphorylation plays critical roles in myriad cell processes. That said, detecting and quantifying protein phosphorylation has remained a challenge. We describe the use of a novel mass spectrometer (Orbitrap Astral) coupled with data-independent acquisition (DIA) to achieve rapid and deep analysis of human and mouse phosphoproteomes. With this method we map approximately 30,000 unique human phosphorylation sites within a half-hour of data collection. We applied this approach to generate a phosphoproteome multi-tissue atlas of the mouse. Altogether, we detected 81,120 unique phosphorylation sites within 12 hours of measurement. With this unique dataset, we examine the sequence and structural context of protein phosphorylation. Finally, we highlight the discovery potential of this resource with multiple examples of novel phosphorylation events relevant to mitochondrial and brain biology.\nHigh-throughput and scalable single cell proteomics identifies over 5000 proteins per cell\nAuthors: Ye, Z.; Sabatier, P.; Hoeven, L.; Phlairaharn, T.; Hartlmayr, D.; Izaguirre, F.; Seth, A.; Joshi, H.; Bekker-Jensen, D.; Bache, N.; Olsen, J. V.\nScore: 56.4, Published: 2023-11-28 DOI: 10.1101/2023.11.27.568953\nThe emergence of mass spectrometry (MS)-based single-cell proteomics (SCP) promise to revolutionize the study of cellular biology and biomedicine by providing an unparalleled view of the proteome in individual cells. Despite its groundbreaking potential, SCP is nascent and faces challenges including limited sequence depth, throughput, and reproducibility, which have constrained its broader utility. This study introduces key methodological advances, which considerably improve the sensitivity, coverage and dependability of protein identification from single cells. We developed an almost lossless SCP workflow encompassing sample preparation to MS analysis, doubling the number of identified proteins from roughly 2000 to over 5000 in individual HeLa cells. A comprehensive evaluation of analytical software tools, alongside strict false discovery rate (FDR) controls solidified the reliability of our results. These enhancements also facilitated the direct detection of post-translational modifications (PTMs) in single cells, negating the need for enrichment and thereby simplifying the analytical process. Although throughput in MS remains a challenge, our study demonstrates the feasibility of processing up to 80 label-free SCP samples per day. Moreover, an optimized tissue dissociation buffer enabled effective single cell disaggregation of drug-treated cancer cell spheroids, refining the overall proteomic analysis. Our workflow sets a new benchmark in SCP for sensitivity and throughput, with broad applications ranging from the study of cellular development to disease progression and the identification of cell type-specific markers and therapeutic targets.\n",
  "wordCount" : "2288",
  "inLanguage": "en",
  "datePublished": "2023-12-24T10:38:02Z",
  "dateModified": "2023-12-24T10:38:02Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 24, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.03.569805">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.03.569805" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.03.569805">
        <p class="paperTitle">A foundational atlas of autism protein interactions reveals molecular convergence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.03.569805" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.03.569805" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, B.; Vartak, R.; Zaltsman, Y.; Naing, Z. Z. C.; Hennick, K. M.; Polacco, B. J.; Bashir, A.; Eckhardt, M.; Bouhaddou, M.; Xu, J.; Sun, N.; Lasser, M.; Zhou, Y.; Guiley, K. Z.; Chan, U.; Kaye, J. A.; Khare, P.; Drake, S.; Drury, V.; Burke, D. F.; Gonzalez, S.; Alkhairy, S.; Morris, M.; Baum, T.; Krasnoff, R.; Wang, S.; Pham, P.; Arbalaez, J.; Pratt, D.; Chag, S.; Rolland, T.; Bourgeron, T.; Finkbeiner, S.; Bandyopadhay, S.; Ideker, T.; Beltrao, P.; Willsey, H. R.; Obernier, K.; Nowakowski, T. J.; Huttenhain, R.; State, M. W.; Willsey, A. J.; Krogan, N. J.</p>
        <p class="info">Score: 57.8, Published: 2023-12-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.03.569805' target='https://doi.org/10.1101/2023.12.03.569805'> 10.1101/2023.12.03.569805</a></p>
        <p class="abstract">Translating high-confidence (hc) autism spectrum disorder (ASD) genes into viable treatment targets remains elusive. We constructed a foundational protein-protein interaction (PPI) network in HEK293T cells involving 100 hcASD risk genes, revealing over 1,800 PPIs (87% novel). Interactors, expressed in the human brain and enriched for ASD but not schizophrenia genetic risk, converged on protein complexes involved in neurogenesis, tubulin biology, transcriptional regulation, and chromatin modification. A PPI map of 54 patient-derived missense variants identified differential physical interactions, and we leveraged AlphaFold-Multimer predictions to prioritize direct PPIs and specific variants for interrogation in Xenopus tropicalis and human forebrain organoids. A mutation in the transcription factor FOXP1 led to reconfiguration of DNA binding sites and altered development of deep cortical layer neurons in forebrain organoids. This work offers new insights into molecular mechanisms underlying ASD and describes a powerful platform to develop and test therapeutic strategies for many genetically-defined conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571479">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571479" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571479">
        <p class="paperTitle">A distinctive evolution of alveolar T cell responses is associated with clinical outcomes in unvaccinated patients with SARS-CoV-2 pneumonia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571479" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571479" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Markov, N. S.; Ren, Z.; Senkow, K. J.; Grant, R. A.; Gao, C. A.; Malsin, E. S.; Sichizya, L.; Kihshen, H.; Helmin, K. A.; Jovisic, M.; Arnold, J. M.; Perez-Leonor, X. G.; Abdala-Valencia, H.; Swaminathan, S.; Nwaezeapu, J.; Kang, M.; Rasmussen, L.; Ozer, E. A.; Lorenzo-Redondo, R.; Hultquist, J. F.; Simons, L.; Rios-Guzman, E.; Misharin, A. V.; Wunderink, R. G.; Budinger, G. S.; Singer, B. D.; Morales-Nebreda, L.; The NU SCRIPT Investigators,  </p>
        <p class="info">Score: 8.1, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571479' target='https://doi.org/10.1101/2023.12.13.571479'> 10.1101/2023.12.13.571479</a></p>
        <p class="abstract">Pathogen clearance and resolution of inflammation in patients with pneumonia require an effective local T cell response. Nevertheless, local T cell activation may drive lung injury, particularly during prolonged episodes of respiratory failure characteristic of severe SARS-CoV-2 pneumonia. While T cell responses in the peripheral blood are well described, the evolution of T cell phenotypes and molecular signatures in the distal lung of patients with severe pneumonia caused by SARS-CoV-2 or other pathogens is understudied. Accordingly, we serially obtained 432 bronchoalveolar lavage fluid samples from 273 patients with severe pneumonia and respiratory failure, including 74 unvaccinated patients with COVID-19, and performed flow cytometry, transcriptional, and T cell receptor profiling on sorted CD8&#43; and CD4&#43; T cell subsets. In patients with COVID-19 but not pneumonia secondary to other pathogens, we found that early and persistent enrichment in CD8&#43; and CD4&#43; T cell subsets correlated with survival to hospital discharge. Activation of interferon signaling pathways early after intubation for COVID-19 was associated with favorable outcomes, while activation of NF-{kappa}B-driven programs late in disease was associated with poor outcomes. Patients with SARS-CoV-2 pneumonia whose alveolar T cells preferentially targeted the Spike and Nucleocapsid proteins tended to experience more favorable outcomes than patients whose T cells predominantly targeted the ORF1ab polyprotein complex. These results suggest that in patients with severe SARS-CoV-2 pneumonia, alveolar T cell interferon responses targeting structural SARS-CoV-2 proteins characterize patients who recover, yet these responses progress to NF-{kappa}B activation against non-structural proteins in patients who go on to experience poor clinical outcomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572198">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572198" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572198">
        <p class="paperTitle">Machine Learning and Metabolic Model Guided CRISPRi Reveals a Central Role for Phosphoglycerate Mutase in Chlamydia trachomatis Persistence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572198" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572198" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chowdhury, N. B.; Pokorzynski, N. D.; Rucks, E. A.; Ouellette, S. P.; CARABEO, R.; Saha, R.</p>
        <p class="info">Score: 3.6, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572198' target='https://doi.org/10.1101/2023.12.18.572198'> 10.1101/2023.12.18.572198</a></p>
        <p class="abstract">Upon nutrient starvation, Chlamydia trachomatis serovar L2 (CTL) shifts from its normal growth to a non-replicating form, termed persistence. It is unclear if persistence is an adaptive response or lack of it. To understand that transcriptomics data were collected for nutrient-sufficient and nutrient-starved CTL. Applying machine learning approaches on transcriptomics data revealed a global transcriptomic rewiring of CTL under stress conditions without having any global stress regulator. This indicated that CTLs stress response is due to lack of an adaptive response mechanism. To investigate the impact of this on CTL metabolism, we reconstructed a genome-scale metabolic model of CTL (iCTL278) and contextualized it with the collected transcriptomics data. Using the metabolic bottleneck analysis on contextualized iCTL278, we observed phosphoglycerate mutase (pgm) regulates the entry of CTL to the persistence. Later, pgm was found to have the highest thermodynamics driving force and lowest enzymatic cost. Furthermore, CRISPRi-driven knockdown of pgm and tryptophan starvation experiments revealed the importance of this gene in inducing persistence. Hence, this work, for the first time, introduced thermodynamics and enzyme-cost as tools to gain deeper understanding on CTL persistence.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.570445">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.570445" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.570445">
        <p class="paperTitle">Mechanistic computational modeling of monospecific and bispecific antibodies targeting interleukin-6/8 receptors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.570445" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.570445" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ray, C. M.; Yang, H.; Spangler, J. B.; Mac Gabhann, F.</p>
        <p class="info">Score: 3.4, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.570445' target='https://doi.org/10.1101/2023.12.18.570445'> 10.1101/2023.12.18.570445</a></p>
        <p class="abstract">The spread of cancer from organ to organ (metastasis) is responsible for the vast majority of cancer deaths; however, most current anti-cancer drugs are designed to arrest or reverse tumor growth without directly addressing disease spread. It was recently discovered that tumor cell-secreted interleukin-6 (IL-6) and interleukin-8 (IL-8) synergize to enhance cancer metastasis in a cell-density dependent manner, and blockade of the IL-6 and IL-8 receptors (IL-6R and IL-8R) with a novel bispecific antibody, BS1, significantly reduced metastatic burden in multiple preclinical mouse models of cancer. Bispecific antibodies (BsAbs), which combine two different antigen-binding sites into one molecule, are a promising modality for drug development due to their enhanced avidity and dual targeting effects. However, while BsAbs have tremendous therapeutic potential, elucidating the mechanisms underlying their binding and inhibition will be critical for maximizing the efficacy of new BsAb treatments. Here, we describe a quantitative, computational model of the BS1 BsAb, exhibiting how modeling multivalent binding provides key insights into antibody affinity and avidity effects and can guide therapeutic design. We present detailed simulations of the monovalent and bivalent binding interactions between different antibody constructs and the IL-6 and IL-8 receptors to establish how antibody properties and system conditions impact the formation of binary (antibody-receptor) and ternary (receptor-antibody-receptor) complexes. Model results demonstrate how the balance of these complex types drives receptor inhibition, providing important and generalizable predictions for effective therapeutic design.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.570875">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.570875" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.570875">
        <p class="paperTitle">KinAce: a web portal for exploring kinase-substrate interactions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.570875" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.570875" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sekar, J. A. P.; Li, Y. C.; Schlessinger, A.; Pandey, G.</p>
        <p class="info">Score: 3.0, Published: 2023-12-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.570875' target='https://doi.org/10.1101/2023.12.08.570875'> 10.1101/2023.12.08.570875</a></p>
        <p class="abstract">Interactions between protein kinases and their substrates are critical for the modulation of complex signaling pathways. Currently, there is a large amount of information available about kinases and their substrates in disparate public databases. However, these data are difficult to interpret in the context of cellular systems, which can be facilitated by examining interactions among multiple proteins at once, such as the network of interactions that constitute a signaling pathway. We present KinAce, a user-friendly web portal that integrates and shares information about kinase-substrate interactions from multiple databases of post-translational modifications. KinAce enables the visual exploration of these interactions in systems contexts, such as pathways, domain families, and custom protein set inputs, in an interactive fashion. We expect KinAce to be useful as a knowledge discovery tool for kinase-substrate interactions, and the aggregated KinAce dataset to be useful for protein kinase studies and systems-level analyses. The portal is available at https://kinace.kinametrix.com/.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.12.571279">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.12.571279" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.12.571279">
        <p class="paperTitle">Self-inhibiting percolation and viral spreading in epithelial tissue</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.12.571279" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.12.571279" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xu, X.; Nielsen, B. F.; sneppen, k.</p>
        <p class="info">Score: 24.1, Published: 2023-12-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.12.571279' target='https://doi.org/10.1101/2023.12.12.571279'> 10.1101/2023.12.12.571279</a></p>
        <p class="abstract">SARS-CoV-2 induces delayed type-I/III interferon production, allowing it to escape the early innate immune response. The delay has been attributed to a deficiency in the ability of cells to sense viral replication upon infection, which in turn hampers activation of the antiviral state in bystander cells. Here, we introduce a cellular automaton model to investigate the spatiotemporal spreading of viral infection as a function of virus and host-dependent parameters. The model suggests that the considerable person-to-person heterogeneity in SARS-CoV-2 infections is a consequence of high sensitivity to slight variations in biological parameters near a critical threshold. It further suggests that within-host viral proliferation can be curtailed by the presence of remarkably few cells that are primed for IFN production. Thus the observed heterogeneity in defense readiness of cells reflects a remarkably cost-efficient strategy for protection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572181">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572181" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572181">
        <p class="paperTitle">A Mathematical Framework for Measuring and Tuning Tempo in Developmental Gene Regulatory Networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572181" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572181" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Manser, C. L.; Perez-Carrasco, R.</p>
        <p class="info">Score: 1.6, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572181' target='https://doi.org/10.1101/2023.12.18.572181'> 10.1101/2023.12.18.572181</a></p>
        <p class="abstract">Embryo development is a dynamic process governed by the regulation of timing and sequences of gene expression, which control the proper growth of the organism. While many genetic programs coordinating these sequences are common across species, the timescales of gene expression can vary significantly among different organisms. Currently, substantial experimental efforts are focused on identifying molecular mechanisms that control these temporal aspects. In contrast, the capacity of established mathematical models to incorporate tempo control while maintaining the same dynamical landscape remains less understood. This manuscript addresses this gap by developing a mathematical framework that links the functionality of developmental programs to the corresponding gene expression orbits (or landscapes). This unlocks the ability to find tempo differences as perturbations in the dynamical system that preserve its orbits. We demonstrate that this framework allows for the prediction of molecular mechanisms governing tempo, through both numerical and analytical methods. Our exploration includes two case studies: a generic network featuring coupled production and degradation, and the repressilator. In the latter, we illustrate how altering the dimerisation rates of transcription factors can decouple the tempo from the shape of the resulting orbits. The manuscript concludes by highlighting how the identification of orthogonal molecular mechanisms for tempo control can inform the design of circuits with specific orbits and tempos.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.568995">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.568995" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.568995">
        <p class="paperTitle">Differential proteome diversification in yeast populations: modes of short-term adaptation and fitness outcomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.568995" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.568995" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Knafo, M.; Rezenman, S.; Tsigalnitski, I.; Nevo, R.; Reich, Z.; Kapon, R.</p>
        <p class="info">Score: 11.4, Published: 2023-12-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.568995' target='https://doi.org/10.1101/2023.11.28.568995'> 10.1101/2023.11.28.568995</a></p>
        <p class="abstract">Short-term proteomic adaptations serve as an initial line of defence, allowing populations to cope with environmental changes before long-term genetic alterations occur. Using a representative set of genes, we examined how stress affects gene expression variability for different types and levels of abiotic stresses and how this influences population-level adaptation. Our data reveal that, depending on the nature of the stress, two distinct modes of response can be employed. In one, the levels of most proteins vary between individuals, leading to varied fitness levels in the population. In the other, a more limited range of expression is seen, and fitness is more even. This suggests different levels of complexity and plasticity in adaptation to different types of stress.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568149">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568149" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568149">
        <p class="paperTitle">Fast and Deep Phosphoproteome Analysis with the Orbitrap Astral Mass Spectrometer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568149" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568149" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lancaster, N. M.; Sinitcyn, P.; Forny, P.; Peters-Clarke, T. M.; Fecher, C.; Smith, A. J.; Shishkova, E.; Arrey, T. N.; Pashkova, A.; Robinson, M. L.; Arp, N.; Fan, J.; Hansen, J.; Galmozzi, A.; Serrano, L. R.; Westphall, M. S.; Stewart, H.; Hock, C.; Damoc, E.; Pagliarini, D. J.; Zabrouskov, V.; Coon, J. J.</p>
        <p class="info">Score: 27.4, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568149' target='https://doi.org/10.1101/2023.11.21.568149'> 10.1101/2023.11.21.568149</a></p>
        <p class="abstract">Owing to its roles in cellular signal transduction, protein phosphorylation plays critical roles in myriad cell processes. That said, detecting and quantifying protein phosphorylation has remained a challenge. We describe the use of a novel mass spectrometer (Orbitrap Astral) coupled with data-independent acquisition (DIA) to achieve rapid and deep analysis of human and mouse phosphoproteomes. With this method we map approximately 30,000 unique human phosphorylation sites within a half-hour of data collection. We applied this approach to generate a phosphoproteome multi-tissue atlas of the mouse. Altogether, we detected 81,120 unique phosphorylation sites within 12 hours of measurement. With this unique dataset, we examine the sequence and structural context of protein phosphorylation. Finally, we highlight the discovery potential of this resource with multiple examples of novel phosphorylation events relevant to mitochondrial and brain biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.568953">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.568953" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.568953">
        <p class="paperTitle">High-throughput and scalable single cell proteomics identifies over 5000 proteins per cell</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.568953" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.568953" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ye, Z.; Sabatier, P.; Hoeven, L.; Phlairaharn, T.; Hartlmayr, D.; Izaguirre, F.; Seth, A.; Joshi, H.; Bekker-Jensen, D.; Bache, N.; Olsen, J. V.</p>
        <p class="info">Score: 56.4, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.568953' target='https://doi.org/10.1101/2023.11.27.568953'> 10.1101/2023.11.27.568953</a></p>
        <p class="abstract">The emergence of mass spectrometry (MS)-based single-cell proteomics (SCP) promise to revolutionize the study of cellular biology and biomedicine by providing an unparalleled view of the proteome in individual cells. Despite its groundbreaking potential, SCP is nascent and faces challenges including limited sequence depth, throughput, and reproducibility, which have constrained its broader utility. This study introduces key methodological advances, which considerably improve the sensitivity, coverage and dependability of protein identification from single cells. We developed an almost lossless SCP workflow encompassing sample preparation to MS analysis, doubling the number of identified proteins from roughly 2000 to over 5000 in individual HeLa cells. A comprehensive evaluation of analytical software tools, alongside strict false discovery rate (FDR) controls solidified the reliability of our results. These enhancements also facilitated the direct detection of post-translational modifications (PTMs) in single cells, negating the need for enrichment and thereby simplifying the analytical process. Although throughput in MS remains a challenge, our study demonstrates the feasibility of processing up to 80 label-free SCP samples per day. Moreover, an optimized tissue dissociation buffer enabled effective single cell disaggregation of drug-treated cancer cell spheroids, refining the overall proteomic analysis. Our workflow sets a new benchmark in SCP for sensitivity and throughput, with broad applications ranging from the study of cellular development to disease progression and the identification of cell type-specific markers and therapeutic targets.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
