<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Principles of Computation by Competitive Protein Dimerization Networks
Authors: Parres-Gold, J.; Levine, M.; Emert, B.; Stuart, A.; Elowitz, M.
Score: 47.9, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564854
Many biological signaling pathways employ proteins that competitively dimerize in diverse combinations. These dimerization networks can perform biochemical computations, in which the concentrations of monomers (inputs) determine the concentrations of dimers (outputs). Despite their prevalence, little is known about the range of input-output computations that dimerization networks can perform (their &#34;expressivity&#34;) and how it depends on network size and connectivity.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Principles of Computation by Competitive Protein Dimerization Networks
Authors: Parres-Gold, J.; Levine, M.; Emert, B.; Stuart, A.; Elowitz, M.
Score: 47.9, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564854
Many biological signaling pathways employ proteins that competitively dimerize in diverse combinations. These dimerization networks can perform biochemical computations, in which the concentrations of monomers (inputs) determine the concentrations of dimers (outputs). Despite their prevalence, little is known about the range of input-output computations that dimerization networks can perform (their &#34;expressivity&#34;) and how it depends on network size and connectivity." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-05T10:37:37+00:00" />
<meta property="article:modified_time" content="2023-11-05T10:37:37+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Principles of Computation by Competitive Protein Dimerization Networks
Authors: Parres-Gold, J.; Levine, M.; Emert, B.; Stuart, A.; Elowitz, M.
Score: 47.9, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564854
Many biological signaling pathways employ proteins that competitively dimerize in diverse combinations. These dimerization networks can perform biochemical computations, in which the concentrations of monomers (inputs) determine the concentrations of dimers (outputs). Despite their prevalence, little is known about the range of input-output computations that dimerization networks can perform (their &#34;expressivity&#34;) and how it depends on network size and connectivity."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Principles of Computation by Competitive Protein Dimerization Networks\nAuthors: Parres-Gold, J.; Levine, M.; Emert, B.; Stuart, A.; Elowitz, M.\nScore: 47.9, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564854\nMany biological signaling pathways employ proteins that competitively dimerize in diverse combinations. These dimerization networks can perform biochemical computations, in which the concentrations of monomers (inputs) determine the concentrations of dimers (outputs). Despite their prevalence, little is known about the range of input-output computations that dimerization networks can perform (their \u0026#34;expressivity\u0026#34;) and how it depends on network size and connectivity.",
  "keywords": [
    
  ],
  "articleBody": " Principles of Computation by Competitive Protein Dimerization Networks\nAuthors: Parres-Gold, J.; Levine, M.; Emert, B.; Stuart, A.; Elowitz, M.\nScore: 47.9, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564854\nMany biological signaling pathways employ proteins that competitively dimerize in diverse combinations. These dimerization networks can perform biochemical computations, in which the concentrations of monomers (inputs) determine the concentrations of dimers (outputs). Despite their prevalence, little is known about the range of input-output computations that dimerization networks can perform (their \"expressivity\") and how it depends on network size and connectivity. Using a systematic computational approach, we demonstrate that even small dimerization networks (3-6 monomers) are expressive, performing diverse multi-input computations. Further, dimerization networks are versatile, performing different computations when their protein components are expressed at different levels, such as in different cell types. Remarkably, individual networks with random interaction affinities, when large enough (greater than or equal to 8 proteins), can perform nearly all (~90%) potential one-input network computations merely by tuning their monomer expression levels. Thus, even the simple process of competitive dimerization provides a powerful architecture for multi-input, cell-type-specific signal processing.\nSynthetic model ecosystem of 12 cryopreservable microbial species allowing for a noninvasive approach\nAuthors: Hosoda, K.; Seno, S.; Murakami, N.; Matsuda, H.; Osada, Y.; Kamiura, R.; Kondoh, M.\nScore: 29.2, Published: 2023-10-31 DOI: 10.1101/2020.10.23.351742\nSimultaneous understanding of both individual and ecosystem dynamics is crucial in an era marked by the degradation of ecosystem services. Herein, we present a high-throughput synthetic microcosm system comprising 12 functionally and phylogenetically diverse microbial species. These species are axenically culturable, cryopreservable, and can be measured noninvasively via microscopy, aided by machine learning. This system includes prokaryotic and eukaryotic producers and decomposers, and eukaryotic consumers to ensure functional redundancy. Our model system displayed both positive and negative interspecific interactions and higher-order interactions that surpassed the scope of any two-species interaction. Although complete species coexistence was not our primary objective, we identified several conditions under which at least one species from the producers, consumers, and decomposers groups, and one functionally redundant species, persisted for over six months. These conditions set the stage for detailed investigations in the future. Given its designability and experimental replicability, our model ecosystem offers a promising platform for deeper insights into both individual and ecosystem dynamics, including evolution and species interactions.\nDevelopment of a genome scale metabolic model for the lager hybrid yeast S. pastorianus to understand evolution of metabolic pathways in industrial settings.\nAuthors: Timouma, S.; Balarezo-Cisneros, L. N.; Schwartz, J.-M.; Delneri, D.\nScore: 9.8, Published: 2023-10-28 DOI: 10.1101/2023.10.25.564032\nIn silico tools such as genome-scale metabolic models (GSMM) have shown to be powerful for metabolic engineering of microorganisms. Here, we created the iSP_1513 GSMM for the aneuploid hybrid S. pastorianus CBS1513 to allow top-down computational approaches to predict the evolution of metabolic pathways and to aid strain optimisation and media engineering in production processes. The iSP_1513 comprises 4062 reactions, 1808 alleles and 2747 metabolites, and takes into account the functional redundancy in the gene-protein-reaction rule caused by the presence of orthologous genes. Moreover, a universal algorithm to constrain GSMM reactions using transcriptome data was developed as a python library and enabled the integration of temperature as parameter. Essentiality datasets, growth data on various carbohydrates and volatile metabolites secretion were used to validate the model. Overall, the iSP_1513 GSMM represent an important step towards understanding the metabolic capabilities, evolutionary trajectories and adaptation potential of S. pastorianus in different industrial settings.\nGlobal transcription regulation revealed from dynamical correlations in time-resolved single-cell RNA-sequencing\nAuthors: Volteras, D.; Shahrezaei, V.; Thomas, P.\nScore: 7.0, Published: 2023-10-27 DOI: 10.1101/2023.10.24.563709\nSingle-cell transcriptomics reveals significant variations in the transcriptional activity across cells. Yet, it remains challenging to identify mechanisms of transcription dynamics from static snapshots. It is thus still unknown what drives global transcription dynamics in single cells. We present a stochastic model of gene expression in growing and dividing cells that harnesses temporal dimensions of single-cell RNA-sequencing through metabolic labelling protocols and cell cycle reporters. We develop a parallel and highly scalable Approximate Bayesian Computation method that corrects for technical variation and accurately quantifies absolute burst frequency, burst size and degradation rate at a transcriptome-wide scale. Using Bayesian model selection, we reveal scaling between transcription rates and cell size and unveil waves of gene regulation across the cell cycle-dependent transcriptome. Our study shows that stochastic modelling of dynamical correlations identifies global mechanisms of transcription regulation. We illustrate how mechanistic modelling and amortised inference can efficiently integrate distributions across time, modalities and conditions. Our framework thus provides a versatile tool to uncover how global transcriptional dynamics is orchestrated genome-wide in single cells.\nStory of an infection: viral dynamics and host responses in the Caenorhabditis elegans-Orsay virus pathosystem\nAuthors: Castiglioni, V. G.; Olmo-Uceda, M. J.; Villena-Jimenez, A.; Munoz-Sanchez, J. C.; Elena, S. F.\nScore: 5.2, Published: 2023-11-03 DOI: 10.1101/2023.10.31.564947\nOrsay virus (OrV) is the only known natural virus affecting Caenorhabditis elegans, with minimal impact on the worms fitness due to its robust innate immune response. This study aimed to understand the interactions between C. elegans and OrV by tracking the infection progression during larval development. Four distinct stages of infection were identified based on viral load, with a peak in capsid-encoding RNA2 coinciding with the first signs of viral egression. Transcriptomic analysis revealed temporal changes in gene expression and functions induced by the infection. A specific set of up-regulated genes remained active throughout the infection, and genes correlated and anticorrelated with virus accumulation were identified. Responses to OrV mirrored reactions to other biotic stressors, distinguishing between virus-specific responses and broader immune responses. Additionally, mutants of early response genes and defense-related processes showed altered viral load progression, uncovering new players in the antiviral defense response.\nNeutral competition explains the clonal composition of neural organoids\nAuthors: Pflug, F. G.; Haendeler, S.; Esk, C.; Lindenhofer, D.; Knoblich, J. A.; von Haeseler, A.\nScore: 8.4, Published: 2023-10-08 DOI: 10.1101/2021.10.06.463206\nCerebral organoids model the development of the human brain and are an indispensable tool for studying neurodevelopment. Whole-organoid lineage tracing has revealed the number of progeny arising from each initial stem cell to be highly diverse, with lineage sizes ranging from one to more than 20,000 cells. This exceeds what can be explained by existing stochastic models of corticogenesis, indicating the existence of an additional source of stochasticity. We propose an explanation in terms of the SAN model in which this additional source of stochasticity is the survival time of a lineage within a long-lived population of symmetrically dividing cells under neutral competition. We demonstrate that this model explains the experimentally observed variability of lineage sizes and we derive a formula that captures the quantitative relationship between survival time and lineage size. Finally, we show that our model implies the existence of a regulatory mechanism to keeps the size of the symmetrically dividing cell population constant.\nMinimally Sufficient Experimental Design using Identifiability Analysis\nAuthors: Kareva, I.; Gevertz, J. L.\nScore: 8.6, Published: 2023-10-18 DOI: 10.1101/2023.10.14.562348\nMathematical models are increasingly being developed and calibrated in tandem with data collection, empowering scientists to intervene in real time based on quantitative model predictions. Well-designed experiments can help augment the predictive power of a mathematical model but the question of when to collect data to maximize its utility for a model is non-trivial. Here we define data as model-informative if it results in a unique parametrization, assessed through the lens of practical identifiability. The framework we propose identifies an optimal experimental design (how much data to collect and when to collect it) that ensures parameter identifiability (permitting confidence in model predictions), while minimizing experimental time and costs. We demonstrate the power of the method by applying it to a modified version of a classic site-of-action pharmacokinetic/pharmacodynamic model that describes distribution of a drug into the tumor microenvironment (TME), where its efficacy is dependent on the level of target occupancy in the TME. In this context, we identify a minimal set of time points when data needs to be collected that robustly ensures practical identifiability of model parameters. The proposed methodology can be applied broadly to any mathematical model, allowing for the identification of a minimally sufficient experimental design that collects the most informative data.\nExamination of the quantitative relationship between proteome- and transcriptome-level responses in larval guts of Frankliniella occidentalis infected with an orthotospovirus.\nAuthors: Han, J.; Rotenberg, D.\nScore: 4.8, Published: 2023-11-02 DOI: 10.1101/2023.10.31.564998\nThe western flower thrips, Frankliniella occidentalis, is the principal thrips vector of Orthotospovirus tomatomaculae (order Bunyavirales, family Tospoviridae), a devastating plant-pathogenic virus commonly referred to as tomato spotted wilt virus (TSWV). The larval gut is the gateway for virus transmission by F. occidentalis adults to plants. In a previous report, gut expression at the transcriptome-level was subtle but significant in response to TSWV in L1s. Since it has been well documented that the relationship between the expression of mRNA and associated protein products in eukaryotic cells is often discordant, we performed identical, replicated experiments to identify and quantify virus-responsive larval gut proteins (427) to expand our understanding of insect host response to TSWV. While we documented statistically-significant, positive correlations between mRNA and protein abundance for genes (4,192) expressed in first and second instar guts, there was virtually no alignment of individual genes identified to be differentially modulated by virus infection at the transcriptome and proteome levels. Predicted protein-protein interaction networks associated with clusters of co-expressed proteins revealed wide variation in correlation strength between protein and cognate transcript abundance, which appeared to be associated with the type of cellular processes, cellular compartments, and network connectivity represented by the proteins. In total, our findings indicate distinct and dynamic regulatory mechanisms of transcript and protein abundance (expression, modifications, and/or turnover) in gut tissues regardless of virus-infection. This study provides molecular candidates for future functional analysis of thrips vector competence, and underscores the necessity of examining complex virus-vector interactions at a systems level.\nLongitudinal single-cell data informs deterministic modelling of inflammatory bowel disease\nAuthors: Kilian, C.; Ulrich, H.; Zouboulis, V.; Sprezyna, P.; Schreiber, J.; Landsberger, T.; Büttner, M.; Biton, M.; Villablanca, E. J.; Huber, S.; Adlung, L.\nScore: 4.5, Published: 2023-10-30 DOI: 10.1101/2023.10.27.561846\nSingle-cell mRNA sequencing (scRNA-seq) allows deep molecular and cellular profiling of immunological processes. Longitudinal scRNA-seq datasets can be used for deterministic ordinary differential equation (ODE)-based modelling to mechanistically describe immune dynamics. Here, we derived longitudinal changes in the abundance of six colonic cell types during inflammatory bowel disease (IBD) from scRNA-seq data of a mouse model of colitis using ODE-based models. We then predicted the immune dynamics of a different mouse colitis protocol and confirmed these scRNA-seq-based predictions with our previously published single-cell-based flow cytometry data. We further hypothesised that the estimated model parameters reflect biological processes. We validated this prediction of cellular turnover rates with KI-67 staining and with gene expression information from the scRNA-seq data not used for model fitting. Finally, we tested the translational relevance of the model simulations by predicting genes indicative of treatment response in human IBD patients. The predictive power of IBD deterministic modelling from scRNA-seq data highlights its potential to advance our understanding of immune dynamics in health and disease.\nCoupled ribosome allocation and nutrient sensing lead to incoherent feedback and oscillatory response in bacterial growth.\nAuthors: Droghetti, R.; Firmano, V.; Tallarico, G.; Calabrese, L.; Grilli, J.; Cosentino Lagomarsino, M.\nScore: 3.4, Published: 2023-10-29 DOI: 10.1101/2023.10.25.563923\nCurrent theories describing bacterial growth physiology across environments have demonstrated an impressive predictive power, but they are typically phenomenological. Incorporating mechanistic details into these frameworks remains an open challenge that would greatly improve our ability to predict and control bacterial growth in varying environmental conditions. For example, the \"Flux Controlled Regulation\" (FCR) model is a reference out-of-equilibrium framework that links ribosome allocation to translation efficiency by means of a steady-state assumption. By making use of this assumption, this model does not account for ppGpp-mediated nutrient sensing and transcriptional regulation of ribosomal operons. In this study, we propose a simple model that integrates the FCR framework with a mechanistic description of three key components: (i) the amino-acid pool, (ii) ppGpp sensing of translation elongation rate, and (iii) transcriptional programming of protein allocation strategy by ppGpp-sensitive promoters. Our framework is fully coherent with observed steady-state growth laws and makes testable predictions for unobserved quantities. Furthermore, our theory predicts that under environmental changes the incoherent feedback between sensing and regulation leads to oscillatory relaxation towards new equilibria, a feature observed experimentally but not captured by previous phenomenological models. SIGNIFICANCE STATEMENTExisting theories that explain cellular growth by considering resource allocation do not provide mechanistic details connecting the nutrient precursors pools, the sensing process responsible for allocation and the allocation process itself. Instead, these theories view growth as an outcome of either flux balance or resource optimization, neglecting finer mechanistic aspects. An outstanding challenge is to describe mechanistically how resource allocation at the proteome level is implemented by regulatory circuits that can sense the environment and regulate protein synthesis accordingly. This study focuses on E. coli bacteria and proposes a theoretical frame-work based on contemporary formalisms that includes the description of the sensing and the regulation of protein synthesis at the mRNA level. Intriguingly, the interplay between sensing and regulatory architecture gives rise to oscillatory adaptation to environmental changes, which emerges in experiments but remains not well characterized.\n",
  "wordCount" : "2198",
  "inLanguage": "en",
  "datePublished": "2023-11-05T10:37:37Z",
  "dateModified": "2023-11-05T10:37:37Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 5, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564854">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564854" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564854">
        <p class="paperTitle">Principles of Computation by Competitive Protein Dimerization Networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564854" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564854" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Parres-Gold, J.; Levine, M.; Emert, B.; Stuart, A.; Elowitz, M.</p>
        <p class="info">Score: 47.9, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564854' target='https://doi.org/10.1101/2023.10.30.564854'> 10.1101/2023.10.30.564854</a></p>
        <p class="abstract">Many biological signaling pathways employ proteins that competitively dimerize in diverse combinations. These dimerization networks can perform biochemical computations, in which the concentrations of monomers (inputs) determine the concentrations of dimers (outputs). Despite their prevalence, little is known about the range of input-output computations that dimerization networks can perform (their &#34;expressivity&#34;) and how it depends on network size and connectivity. Using a systematic computational approach, we demonstrate that even small dimerization networks (3-6 monomers) are expressive, performing diverse multi-input computations. Further, dimerization networks are versatile, performing different computations when their protein components are expressed at different levels, such as in different cell types. Remarkably, individual networks with random interaction affinities, when large enough (greater than or equal to 8 proteins), can perform nearly all (~90%) potential one-input network computations merely by tuning their monomer expression levels. Thus, even the simple process of competitive dimerization provides a powerful architecture for multi-input, cell-type-specific signal processing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2020.10.23.351742">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2020.10.23.351742" aria-expanded="false" aria-controls="flush-collapse10.1101/2020.10.23.351742">
        <p class="paperTitle">Synthetic model ecosystem of 12 cryopreservable microbial species allowing for a noninvasive approach</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2020.10.23.351742" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2020.10.23.351742" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hosoda, K.; Seno, S.; Murakami, N.; Matsuda, H.; Osada, Y.; Kamiura, R.; Kondoh, M.</p>
        <p class="info">Score: 29.2, Published: 2023-10-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2020.10.23.351742' target='https://doi.org/10.1101/2020.10.23.351742'> 10.1101/2020.10.23.351742</a></p>
        <p class="abstract">Simultaneous understanding of both individual and ecosystem dynamics is crucial in an era marked by the degradation of ecosystem services. Herein, we present a high-throughput synthetic microcosm system comprising 12 functionally and phylogenetically diverse microbial species. These species are axenically culturable, cryopreservable, and can be measured noninvasively via microscopy, aided by machine learning. This system includes prokaryotic and eukaryotic producers and decomposers, and eukaryotic consumers to ensure functional redundancy. Our model system displayed both positive and negative interspecific interactions and higher-order interactions that surpassed the scope of any two-species interaction. Although complete species coexistence was not our primary objective, we identified several conditions under which at least one species from the producers, consumers, and decomposers groups, and one functionally redundant species, persisted for over six months. These conditions set the stage for detailed investigations in the future. Given its designability and experimental replicability, our model ecosystem offers a promising platform for deeper insights into both individual and ecosystem dynamics, including evolution and species interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.564032">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.564032" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.564032">
        <p class="paperTitle">Development of a genome scale metabolic model for the lager hybrid yeast S. pastorianus to understand evolution of metabolic pathways in industrial settings.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.564032" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.564032" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Timouma, S.; Balarezo-Cisneros, L. N.; Schwartz, J.-M.; Delneri, D.</p>
        <p class="info">Score: 9.8, Published: 2023-10-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.564032' target='https://doi.org/10.1101/2023.10.25.564032'> 10.1101/2023.10.25.564032</a></p>
        <p class="abstract">In silico tools such as genome-scale metabolic models (GSMM) have shown to be powerful for metabolic engineering of microorganisms. Here, we created the iSP_1513 GSMM for the aneuploid hybrid S. pastorianus CBS1513 to allow top-down computational approaches to predict the evolution of metabolic pathways and to aid strain optimisation and media engineering in production processes. The iSP_1513 comprises 4062 reactions, 1808 alleles and 2747 metabolites, and takes into account the functional redundancy in the gene-protein-reaction rule caused by the presence of orthologous genes. Moreover, a universal algorithm to constrain GSMM reactions using transcriptome data was developed as a python library and enabled the integration of temperature as parameter. Essentiality datasets, growth data on various carbohydrates and volatile metabolites secretion were used to validate the model. Overall, the iSP_1513 GSMM represent an important step towards understanding the metabolic capabilities, evolutionary trajectories and adaptation potential of S. pastorianus in different industrial settings.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563709">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563709" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563709">
        <p class="paperTitle">Global transcription regulation revealed from dynamical correlations in time-resolved single-cell RNA-sequencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563709" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563709" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Volteras, D.; Shahrezaei, V.; Thomas, P.</p>
        <p class="info">Score: 7.0, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563709' target='https://doi.org/10.1101/2023.10.24.563709'> 10.1101/2023.10.24.563709</a></p>
        <p class="abstract">Single-cell transcriptomics reveals significant variations in the transcriptional activity across cells. Yet, it remains challenging to identify mechanisms of transcription dynamics from static snapshots. It is thus still unknown what drives global transcription dynamics in single cells. We present a stochastic model of gene expression in growing and dividing cells that harnesses temporal dimensions of single-cell RNA-sequencing through metabolic labelling protocols and cell cycle reporters. We develop a parallel and highly scalable Approximate Bayesian Computation method that corrects for technical variation and accurately quantifies absolute burst frequency, burst size and degradation rate at a transcriptome-wide scale. Using Bayesian model selection, we reveal scaling between transcription rates and cell size and unveil waves of gene regulation across the cell cycle-dependent transcriptome. Our study shows that stochastic modelling of dynamical correlations identifies global mechanisms of transcription regulation. We illustrate how mechanistic modelling and amortised inference can efficiently integrate distributions across time, modalities and conditions. Our framework thus provides a versatile tool to uncover how global transcriptional dynamics is orchestrated genome-wide in single cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.564947">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.564947" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.564947">
        <p class="paperTitle">Story of an infection: viral dynamics and host responses in the Caenorhabditis elegans-Orsay virus pathosystem</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.564947" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.564947" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Castiglioni, V. G.; Olmo-Uceda, M. J.; Villena-Jimenez, A.; Munoz-Sanchez, J. C.; Elena, S. F.</p>
        <p class="info">Score: 5.2, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.564947' target='https://doi.org/10.1101/2023.10.31.564947'> 10.1101/2023.10.31.564947</a></p>
        <p class="abstract">Orsay virus (OrV) is the only known natural virus affecting Caenorhabditis elegans, with minimal impact on the worms fitness due to its robust innate immune response. This study aimed to understand the interactions between C. elegans and OrV by tracking the infection progression during larval development. Four distinct stages of infection were identified based on viral load, with a peak in capsid-encoding RNA2 coinciding with the first signs of viral egression. Transcriptomic analysis revealed temporal changes in gene expression and functions induced by the infection. A specific set of up-regulated genes remained active throughout the infection, and genes correlated and anticorrelated with virus accumulation were identified. Responses to OrV mirrored reactions to other biotic stressors, distinguishing between virus-specific responses and broader immune responses. Additionally, mutants of early response genes and defense-related processes showed altered viral load progression, uncovering new players in the antiviral defense response.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2021.10.06.463206">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2021.10.06.463206" aria-expanded="false" aria-controls="flush-collapse10.1101/2021.10.06.463206">
        <p class="paperTitle">Neutral competition explains the clonal composition of neural organoids</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2021.10.06.463206" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2021.10.06.463206" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pflug, F. G.; Haendeler, S.; Esk, C.; Lindenhofer, D.; Knoblich, J. A.; von Haeseler, A.</p>
        <p class="info">Score: 8.4, Published: 2023-10-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2021.10.06.463206' target='https://doi.org/10.1101/2021.10.06.463206'> 10.1101/2021.10.06.463206</a></p>
        <p class="abstract">Cerebral organoids model the development of the human brain and are an indispensable tool for studying neurodevelopment. Whole-organoid lineage tracing has revealed the number of progeny arising from each initial stem cell to be highly diverse, with lineage sizes ranging from one to more than 20,000 cells. This exceeds what can be explained by existing stochastic models of corticogenesis, indicating the existence of an additional source of stochasticity. We propose an explanation in terms of the SAN model in which this additional source of stochasticity is the survival time of a lineage within a long-lived population of symmetrically dividing cells under neutral competition. We demonstrate that this model explains the experimentally observed variability of lineage sizes and we derive a formula that captures the quantitative relationship between survival time and lineage size. Finally, we show that our model implies the existence of a regulatory mechanism to keeps the size of the symmetrically dividing cell population constant.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.14.562348">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.14.562348" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.14.562348">
        <p class="paperTitle">Minimally Sufficient Experimental Design using Identifiability Analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.14.562348" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.14.562348" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kareva, I.; Gevertz, J. L.</p>
        <p class="info">Score: 8.6, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.14.562348' target='https://doi.org/10.1101/2023.10.14.562348'> 10.1101/2023.10.14.562348</a></p>
        <p class="abstract">Mathematical models are increasingly being developed and calibrated in tandem with data collection, empowering scientists to intervene in real time based on quantitative model predictions. Well-designed experiments can help augment the predictive power of a mathematical model but the question of when to collect data to maximize its utility for a model is non-trivial. Here we define data as model-informative if it results in a unique parametrization, assessed through the lens of practical identifiability. The framework we propose identifies an optimal experimental design (how much data to collect and when to collect it) that ensures parameter identifiability (permitting confidence in model predictions), while minimizing experimental time and costs. We demonstrate the power of the method by applying it to a modified version of a classic site-of-action pharmacokinetic/pharmacodynamic model that describes distribution of a drug into the tumor microenvironment (TME), where its efficacy is dependent on the level of target occupancy in the TME. In this context, we identify a minimal set of time points when data needs to be collected that robustly ensures practical identifiability of model parameters. The proposed methodology can be applied broadly to any mathematical model, allowing for the identification of a minimally sufficient experimental design that collects the most informative data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.564998">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.564998" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.564998">
        <p class="paperTitle">Examination of the quantitative relationship between proteome- and transcriptome-level responses in larval guts of Frankliniella occidentalis infected with an orthotospovirus.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.564998" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.564998" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Han, J.; Rotenberg, D.</p>
        <p class="info">Score: 4.8, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.564998' target='https://doi.org/10.1101/2023.10.31.564998'> 10.1101/2023.10.31.564998</a></p>
        <p class="abstract">The western flower thrips, Frankliniella occidentalis, is the principal thrips vector of Orthotospovirus tomatomaculae (order Bunyavirales, family Tospoviridae), a devastating plant-pathogenic virus commonly referred to as tomato spotted wilt virus (TSWV). The larval gut is the gateway for virus transmission by F. occidentalis adults to plants. In a previous report, gut expression at the transcriptome-level was subtle but significant in response to TSWV in L1s. Since it has been well documented that the relationship between the expression of mRNA and associated protein products in eukaryotic cells is often discordant, we performed identical, replicated experiments to identify and quantify virus-responsive larval gut proteins (427) to expand our understanding of insect host response to TSWV. While we documented statistically-significant, positive correlations between mRNA and protein abundance for genes (4,192) expressed in first and second instar guts, there was virtually no alignment of individual genes identified to be differentially modulated by virus infection at the transcriptome and proteome levels. Predicted protein-protein interaction networks associated with clusters of co-expressed proteins revealed wide variation in correlation strength between protein and cognate transcript abundance, which appeared to be associated with the type of cellular processes, cellular compartments, and network connectivity represented by the proteins. In total, our findings indicate distinct and dynamic regulatory mechanisms of transcript and protein abundance (expression, modifications, and/or turnover) in gut tissues regardless of virus-infection. This study provides molecular candidates for future functional analysis of thrips vector competence, and underscores the necessity of examining complex virus-vector interactions at a systems level.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.561846">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.561846" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.561846">
        <p class="paperTitle">Longitudinal single-cell data informs deterministic modelling of inflammatory bowel disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.561846" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.561846" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kilian, C.; Ulrich, H.; Zouboulis, V.; Sprezyna, P.; Schreiber, J.; Landsberger, T.; Büttner, M.; Biton, M.; Villablanca, E. J.; Huber, S.; Adlung, L.</p>
        <p class="info">Score: 4.5, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.561846' target='https://doi.org/10.1101/2023.10.27.561846'> 10.1101/2023.10.27.561846</a></p>
        <p class="abstract">Single-cell mRNA sequencing (scRNA-seq) allows deep molecular and cellular profiling of immunological processes. Longitudinal scRNA-seq datasets can be used for deterministic ordinary differential equation (ODE)-based modelling to mechanistically describe immune dynamics. Here, we derived longitudinal changes in the abundance of six colonic cell types during inflammatory bowel disease (IBD) from scRNA-seq data of a mouse model of colitis using ODE-based models. We then predicted the immune dynamics of a different mouse colitis protocol and confirmed these scRNA-seq-based predictions with our previously published single-cell-based flow cytometry data. We further hypothesised that the estimated model parameters reflect biological processes. We validated this prediction of cellular turnover rates with KI-67 staining and with gene expression information from the scRNA-seq data not used for model fitting. Finally, we tested the translational relevance of the model simulations by predicting genes indicative of treatment response in human IBD patients. The predictive power of IBD deterministic modelling from scRNA-seq data highlights its potential to advance our understanding of immune dynamics in health and disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.563923">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.563923" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.563923">
        <p class="paperTitle">Coupled ribosome allocation and nutrient sensing lead to incoherent feedback and oscillatory response in bacterial growth.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.563923" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.563923" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Droghetti, R.; Firmano, V.; Tallarico, G.; Calabrese, L.; Grilli, J.; Cosentino Lagomarsino, M.</p>
        <p class="info">Score: 3.4, Published: 2023-10-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.563923' target='https://doi.org/10.1101/2023.10.25.563923'> 10.1101/2023.10.25.563923</a></p>
        <p class="abstract">Current theories describing bacterial growth physiology across environments have demonstrated an impressive predictive power, but they are typically phenomenological. Incorporating mechanistic details into these frameworks remains an open challenge that would greatly improve our ability to predict and control bacterial growth in varying environmental conditions. For example, the &#34;Flux Controlled Regulation&#34; (FCR) model is a reference out-of-equilibrium framework that links ribosome allocation to translation efficiency by means of a steady-state assumption. By making use of this assumption, this model does not account for ppGpp-mediated nutrient sensing and transcriptional regulation of ribosomal operons. In this study, we propose a simple model that integrates the FCR framework with a mechanistic description of three key components: (i) the amino-acid pool, (ii) ppGpp sensing of translation elongation rate, and (iii) transcriptional programming of protein allocation strategy by ppGpp-sensitive promoters. Our framework is fully coherent with observed steady-state growth laws and makes testable predictions for unobserved quantities. Furthermore, our theory predicts that under environmental changes the incoherent feedback between sensing and regulation leads to oscillatory relaxation towards new equilibria, a feature observed experimentally but not captured by previous phenomenological models.

SIGNIFICANCE STATEMENTExisting theories that explain cellular growth by considering resource allocation do not provide mechanistic details connecting the nutrient precursors pools, the sensing process responsible for allocation and the allocation process itself. Instead, these theories view growth as an outcome of either flux balance or resource optimization, neglecting finer mechanistic aspects. An outstanding challenge is to describe mechanistically how resource allocation at the proteome level is implemented by regulatory circuits that can sense the environment and regulate protein synthesis accordingly. This study focuses on E. coli bacteria and proposes a theoretical frame-work based on contemporary formalisms that includes the description of the sensing and the regulation of protein synthesis at the mRNA level. Intriguingly, the interplay between sensing and regulatory architecture gives rise to oscillatory adaptation to environmental changes, which emerges in experiments but remains not well characterized.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
