<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Efficient gene knockout and genetic interactions: the IN4MER CRISPR/Cas12a multiplex knockout platform
Authors: Esmaeili Anvar, N.; Lin, C.; Ma, X.; Wilson, L. L.; Steger, R.; Sangree, A. K.; Colic, M.; Wang, S. H.; Doench, J. G.; Hart, T.
Score: 32.1, Published: 2023-09-05 DOI: 10.1101/2023.01.03.522655
Genetic interactions mediate the emergence of phenotype from genotype, but initial technologies for combinatorial genetic perturbation in mammalian cells suffer from inefficiency and are challenging to scale.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Efficient gene knockout and genetic interactions: the IN4MER CRISPR/Cas12a multiplex knockout platform
Authors: Esmaeili Anvar, N.; Lin, C.; Ma, X.; Wilson, L. L.; Steger, R.; Sangree, A. K.; Colic, M.; Wang, S. H.; Doench, J. G.; Hart, T.
Score: 32.1, Published: 2023-09-05 DOI: 10.1101/2023.01.03.522655
Genetic interactions mediate the emergence of phenotype from genotype, but initial technologies for combinatorial genetic perturbation in mammalian cells suffer from inefficiency and are challenging to scale." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-10T10:36:49+00:00" />
<meta property="article:modified_time" content="2023-09-10T10:36:49+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Efficient gene knockout and genetic interactions: the IN4MER CRISPR/Cas12a multiplex knockout platform
Authors: Esmaeili Anvar, N.; Lin, C.; Ma, X.; Wilson, L. L.; Steger, R.; Sangree, A. K.; Colic, M.; Wang, S. H.; Doench, J. G.; Hart, T.
Score: 32.1, Published: 2023-09-05 DOI: 10.1101/2023.01.03.522655
Genetic interactions mediate the emergence of phenotype from genotype, but initial technologies for combinatorial genetic perturbation in mammalian cells suffer from inefficiency and are challenging to scale."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Efficient gene knockout and genetic interactions: the IN4MER CRISPR/Cas12a multiplex knockout platform\nAuthors: Esmaeili Anvar, N.; Lin, C.; Ma, X.; Wilson, L. L.; Steger, R.; Sangree, A. K.; Colic, M.; Wang, S. H.; Doench, J. G.; Hart, T.\nScore: 32.1, Published: 2023-09-05 DOI: 10.1101/2023.01.03.522655\nGenetic interactions mediate the emergence of phenotype from genotype, but initial technologies for combinatorial genetic perturbation in mammalian cells suffer from inefficiency and are challenging to scale.",
  "keywords": [
    
  ],
  "articleBody": " Efficient gene knockout and genetic interactions: the IN4MER CRISPR/Cas12a multiplex knockout platform\nAuthors: Esmaeili Anvar, N.; Lin, C.; Ma, X.; Wilson, L. L.; Steger, R.; Sangree, A. K.; Colic, M.; Wang, S. H.; Doench, J. G.; Hart, T.\nScore: 32.1, Published: 2023-09-05 DOI: 10.1101/2023.01.03.522655\nGenetic interactions mediate the emergence of phenotype from genotype, but initial technologies for combinatorial genetic perturbation in mammalian cells suffer from inefficiency and are challenging to scale. Recent focus on paralog synthetic lethality in cancer cells offers an opportunity to evaluate different approaches and improve on the state of the art. Here we report a meta-analysis of CRISPR genetic interactions screens, identifying a candidate set of background-independent paralog synthetic lethals, and find that the Cas12a platform provides superior sensitivity and assay replicability. We demonstrate that Cas12a can independently target up to four genes from a single guide array, and we build on this knowledge by constructing a genome-scale library that expresses arrays of four guides per clone, a platform we call in4mer. Our genome-scale human library, with only 49k clones, is substantially smaller than a typical CRISPR/Cas9 monogenic library while also targeting more than four thousand paralog pairs, triples, and quads. Proof of concept screens in four cell lines demonstrate discrimination of core and context-dependent essential genes similar to that of state-of-the-art CRISPR/Cas9 libraries, as well as detection of synthetic lethal and masking/buffering genetic interactions between paralogs of various family sizes, a capability not offered by any extant library. Importantly, the in4mer platform offers a fivefold reduction in the number of clones required to assay genetic interactions, dramatically improving the cost and effort required for these studies.\nCorrelation of myeloid-derived suppressor cell expansion with upregulated transposable elements in severe COVID-19 unveiled in single-cell RNA sequencing reanalysis\nAuthors: Farahmandnejad, M.; Mosadeghi, P.; Dorvash, M.; Sakhteman, A.; Faridi, P.; Negahdaripour, M.\nScore: 18.6, Published: 2023-09-05 DOI: 10.1101/2023.09.04.556192\nSome studies investigated the potential role of transposable elements (TEs) in COVID-19 pathogenesis and complications. However, to the best of our knowledge, there is no study to examine the possible association of TEs expression in cell functions and its potential role in COVID-19 immune response at the single-cell level. In this study, we reanalyzed single-cell RNA seq data of bronchoalveolar lavage (BAL) samples obtained from six severe COVID-19 patients and three healthy donors to assess the probable correlation of TE expression with the immune responses induced by the SARS-CoV-2 virus in COVID-19 patients. Our findings indicated that the expansion of myeloid-derived suppressor cells (MDSCs) may be a characteristic feature of COVID-19. Additionally, a significant increase in TEs expression in MDSCs was observed. This upregulation of TEs in COVID-19 may be linked to the adaptability of these cells in response to their microenvironments. Furthermore, it appears that the identification of overexpressed TEs by Pattern recognition receptors (PRRs) in MDSCs may enhance the suppressive capacity of these cells. Thus, this study emphasizes the crucial role of TEs in the functionality of MDSCs during COVID-19.\nExploiting fluctuations in gene expression to detect causal interactions between genes\nAuthors: Joly-Smith, E.; Talpur, M. M.; Allard, P.; Papazotos, F.; Potvin-Trottier, L.; Hilfinger, A.\nScore: 11.2, Published: 2023-09-04 DOI: 10.1101/2023.09.01.555799\nCharacterizing and manipulating cellular behaviour requires a mechanistic understanding of the causal interactions between cellular components. We present an approach that can detect causal interactions between genes without the need to perturb the physiological state of cells. This approach exploits naturally occurring cell-to-cell variability which is experimentally accessible from static population snapshots of genetically identical cells without the need to follow cells over time. Our main contribution is a simple mathematical relation that constrains the propagation of gene expression noise through biochemical reaction networks. This relation allows us to rigorously interpret fluctuation data even when only a small part of a complex gene regulatory process can be observed. This relation can be exploited to detect causal interactions by synthetically engineering a passive reporter of gene expression, akin to the established \"dual reporter assay\". While the focus of our contribution is theoretical, we also present an experimental proof-of-principle to illustrate the approach. Our data from synthetic gene regulatory networks in E. coli are not unequivocal but suggest that the method could prove useful in practice to identify causal interactions between genes from non-genetic cell-to-cell variability.\nApproximate Bayesian Computation for Inferring Waddington Landscapes from Single Cell Data\nAuthors: Liu, Y.; Zhang, S. Y.; Kleijn, I. T.; Stumpf, M. P. H.\nScore: 8.7, Published: 2023-09-05 DOI: 10.1101/2023.09.03.556134\nSingle cell technologies allow us to gain insights into cellular processes at unprecedented resolution. In stem cell and developmental biology snapshot data allows us to characterise how the transcriptional state of cells changes between successive cell types. Here we show how approximate Bayesian computation (ABC) can be employed to calibrate mathematical models against single cell data. In our simulation study we demonstrate the pivotal role of the adequate choice of distance measures appropriate for single cell data. We show that for good distance measures, notably optimal transport distances, we can infer parameters for mathematical models from simulated single cell data. We show that the ABC posteriors can be used to characterise parameter sensitivity and identify dependencies between different parameters, and to infer representations of the Waddington or epigenetic landscape, which forms a popular and interpretable representation of the developmental dynamics. In summary, these results pave the way for fitting mechanistic models of stem cell differentiation to single cell data.\nHuman subcutaneous and visceral adipocyte atlases uncover classical and specialized adipocytes and depot-specific patterns\nAuthors: Lazarescu, O.; Ziv Agam, M.; Haim, Y.; Hekselman, I.; Jubran, J.; Shneyour, A.; Kitsberg, D.; Levin, L.; Liberty, I. F.; Yoel, U.; Dukhno, O.; Adam, M.; Korner, A.; Murphy, R.; Bluher, M.; Habib, N.; Rudich, A.; Yeger-Lotem, E.\nScore: 8.7, Published: 2023-09-04 DOI: 10.1101/2023.09.04.555678\nHuman adipose depots are functionally distinct. Yet, recent single-nucleus RNA-sequencing (snRNA-seq) analyses largely uncovered overlapping/similar cell-type landscapes. We hypothesized that adipocytes subtypes, differentiation trajectories, and/or intercellular communication patterns could illuminate this depot similarity-difference gap. For this, we performed snRNA-seq of human subcutaneous and visceral adipose tissue. Whereas the majority of adipocytes in both depots were classical, namely enriched in lipid metabolism pathways, we also observed specialized adipocyte subtypes that were enriched in immune-related, extracellular matrix deposition (fibrosis), vascularization/angiogenesis, or ribosomal processes. Pseudo-temporal analysis suggested a developmental trajectory from adipose progenitor cells to classical adipocytes via specialized adipocytes, suggesting that the classical state stems from loss, rather than gain, of specialized functions. Lastly, intercellular communication routes were consistent with the different inflammatory tone of the two depots. Jointly, these findings provide a high-resolution view into the contribution of cellular composition, differentiation, and intercellular communication patterns to human fat depot differences.\nA hierarchy of metabolite exchanges in metabolic models of microbial species and communities\nAuthors: Wedmark, Y. K.; Vik, J. O.; Oyas, O.\nScore: 7.6, Published: 2023-09-07 DOI: 10.1101/2023.09.05.556413\nThe metabolic network of an organism can be analyzed as a constraint-based model (CBM). CBM analyses can be biased, optimizing an objective such as growth rate, or unbiased, attempting to describe the feasible flux space through pathway analysis or random flux sampling. Biased methods scale to genome-scale models (GEMs) and beyond, but unbiased methods suffer from combinatorial explosion. One promising approach to improve the scalability of unbiased analyses is to focus on cells' metabolite exchanges with each other and the environment rather than the internal details of their metabolic networks. However, methods for unbiased analysis of metabolite exchanges have not been evaluated or compared. Using CBMs ranging in size from core models to GEMs, we applied pathway enumeration and random flux sampling to metabolite exchanges in microbial species and a microbial community. We focused on pathway definitions that allow direct targeting of metabolite exchanges: elementary conversion modes (ECMs), elementary flux patterns (EFPs), and minimal pathways (MPs). Enumerating growth-supporting ECMs, EFPs, and MPs for the same models, we found that the MPs were always a subset of the ECMs and EFPs, specifically the subset of support-minimal pathways. The EFPs were always a subset of the ECMs, which in turn were always a subset of flux patterns from elementary flux modes (EFMs). We argue that this hierarchical relationship between pathway definitions holds more generally and that it helps explain the relative scalability of ECM, EFP, and MP enumeration. Metabolite exchange frequencies in enumerated pathways were similar across methods with a few specific exchanges explaining large differences in pathway counts. Thus, biological interpretation was largely robust to the choice of pathway definition. Random flux sampling scaled to GEMs and complemented pathway enumeration.\nA dynamic Boolean model of molecular and cellular interactions represents psoriasis development and predicts drug candidates.\nAuthors: Tsirvouli, E.; Noël, V.; Flobak, A.; Calzone, L.; Kuiper, M.\nScore: 5.0, Published: 2023-09-06 DOI: 10.1101/2023.09.03.556147\nPsoriasis is a chronic skin disease affecting 2-3% of the global population. Psoriasis arises from complex interactions between keratinocytes and immune cells, leading to uncontrolled inflammation, immune hyperactivation and perturbed keratinocyte life cycle. Although the latest generation of drugs have greatly improved psoriasis management, the disease remains incurable, and the substantial variability in treatment response calls for novel approaches to comprehend the intricate mechanisms underlying disease development and to discover potential drug targets. In this study, we present a multiscale population model that captures the dynamics of cell-specific phenotypes in psoriasis, integrating discrete logical formalism and population dynamics simulations. Through simulations and network metrics, we identify potential pairwise interventions as alternative treatment options. Specifically, The model predictions suggest that targeting neutrophil activation in conjunction with either PGE2 production or STAT3 signaling shows promise comparable to IL-17 inhibition, which is currently the most used treatment option for moderate and severe cases of psoriasis. Our findings underscore the significance of considering complex intercellular interactions and intracellular signaling cascades in psoriasis, and highlight the importance of computational approaches in unraveling complex biological systems for drug target identification.\nPTMNavigator: Interactive Visualization of Differentially Regulated Post-Translational Modifications in Cellular Signaling Pathways\nAuthors: Mueller, J.; Bayer, F. P.; Wilhelm, M.; Kuster, B.; The, M.\nScore: 2.0, Published: 2023-09-02 DOI: 10.1101/2023.08.31.555601\nPost-translational modifications (PTMs) play a governing role in regulating cellular signaling, fine-tuning protein function, and orchestrating complex biological processes. Despite their importance, the lack of comprehensive tools for studying PTMs from a pathway-centric perspective has limited our ability to understand how PTMs modulate cellular pathways on a molecular level. Here, we present PTMNavigator, a tool integrated into the ProteomicsDB platform, which offers an interactive interface for researchers to overlay experimental PTM data on top of [~]3000 well-curated pathway diagrams. This offers a comprehensive view of the intricate relationship between PTMs and signaling pathways. To demonstrate the utility of PTMNavigator, we applied it to two phosphoproteomics perturbation datasets. First, PTMNavigator enhanced pathway enrichment analysis by showing how the regulated peptides are distributed in the pathways with high enrichment scores. Second, it visualized how drug treatments result in a discernable flow of PTM-driven signaling within pathways. Third, PTMNavigator aided in extending existing pathways by uncovering previously unknown relationships between PTMs and pathway components. By facilitating the discovery of novel PTM-pathway interactions and enhancing our understanding of cellular signaling dynamics, PTMNavigator advances our knowledge of PTM biology and its implications in health and disease. One Sentence SummaryPTMNavigator is a software that overlays pathway diagrams with PTM proteomics data and allows their interactive exploration.\nAgent-based modeling of tumor-immune interactions reveals determinants of final tumor states\nAuthors: Ahmidouch, M.; Tangella, N.; Finley, S. D.\nScore: 1.2, Published: 2023-09-07 DOI: 10.1101/2023.09.06.556617\nInteractions between tumor and immune cells in the tumor microenvironment (TME) influence tumor growth and the tumor's response to treatment. Excitingly, this complex landscape of tumor-immune interactions can be studied using computational modeling. Mathematical oncology can provide quantitative insights into the TME, serving as a framework for understanding tumor dynamics. Here, we use an agent-based model to simulate the interactions among cancer cells, macrophages (naive, M1, and M2), and T cells (active CD8+ and inactive) in a 2D representation of the TME. Key diffusible factors, IL-4 and IFN-{gamma}, are also incorporated. We apply the model to predict how cell-specific properties influence tumor progression. The model predictions and analyses revealed the relationships between different cell populations and highlighted the importance of macrophages and T cells in shaping the TME. Thus, we quantify how components of the TME influence the final tumor state and the effects of macrophage-based therapies. The findings emphasize the significant role of computational models in unraveling the intricate dynamics of tumor-immune interactions and their potential for guiding the development of tailored immunotherapeutic strategies. This study provides a foundation for future investigations aiming to refine and expand the model, validate predictions experimentally, and pave the way for improved cancer treatments.\nSystems genetics analysis of human body fat distribution genes identifies Wnt signaling and mitochondrial activity in adipocytes\nAuthors: Reed, J. N.; Huang, J.; Li, Y.; Ma, L.; Banka, D.; Wabitsch, M.; Wang, T.; Ding, W.; Bjorkegren, J. L. M.; Civelek, M.\nScore: 1.2, Published: 2023-09-06 DOI: 10.1101/2023.09.06.556534\nBACKGROUND: Excess fat in the abdomen is a sexually dimorphic risk factor for cardio-metabolic disease. The relative storage between abdominal and lower-body subcutaneous adipose tissue depots is approximated by the waist-to-hip ratio adjusted for body mass index (WHRadjBMI). Genome-wide association studies (GWAS) identified 346 loci near 495 genes associated with WHRadjBMI. Most of these genes have unknown roles in fat distribution, but many are expressed and putatively act in adipose tissue. We aimed to identify novel sex- and depot-specific drivers of WHRadjBMI using a systems genetics approach. METHODS: We used two independent cohorts of adipose tissue gene expression with 362 - 444 males and 147 - 219 females, primarily of European ancestry. We constructed sex- and depot-specific Bayesian networks to model the gene-gene interactions from 8,492 adipose tissue genes. Key driver analysis identified genes that, in silico and putatively in vitro, regulate many others, including the 495 WHRadjBMI GWAS genes. Key driver gene function was determined by perturbing their expression in human subcutaneous pre-adipocytes using lenti-virus or siRNA. RESULTS: 51 - 119 key drivers in each network were replicated in both cohorts. We used single-cell expression data to select replicated key drivers expressed in adipocyte precursors and mature adipocytes, prioritized genes which have not been previously studied in adipose tissue, and used public human and mouse data to nominate 53 novel key driver genes (10 - 21 from each network) that may regulate fat distribution by altering adipocyte function. In other cell types, 23 of these genes are found in crucial adipocyte pathways: Wnt signaling or mitochondrial function. We selected seven genes whose expression is highly correlated with WHRadjBMI to further study their effects on adipogenesis/Wnt signaling (ANAPC2, PSME3, RSPO1, TYRO3) or mitochondrial function (C1QTNF3, MIGA1, PSME3, UBR1). Adipogenesis was inhibited in cells overexpressing ANAPC2 and RSPO1 compared to controls. RSPO1 results are consistent with a positive correlation between gene expression in the subcutaneous depot and WHRadjBMI, therefore lower relative storage in the subcutaneous depot. RSPO1 inhibited adipogenesis by increasing {beta}-catenin activation and Wnt-related transcription, thus repressing PPARG and CEBPA. PSME3 overexpression led to more adipogenesis than controls. In differentiated adipocytes, MIGA1 and UBR1 downregulation led to mitochondrial dysfunction, with lower oxygen consumption than controls; MIGA1 knockdown also lowered UCP1 expression. SUMMARY: ANAPC2, MIGA1, PSME3, RSPO1, and UBR1 affect adipocyte function and may drive body fat distribution.\n",
  "wordCount" : "2527",
  "inLanguage": "en",
  "datePublished": "2023-09-10T10:36:49Z",
  "dateModified": "2023-09-10T10:36:49Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta"><span>updated on September 10, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.01.03.522655">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.01.03.522655" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.01.03.522655">
        <p class="paperTitle">Efficient gene knockout and genetic interactions: the IN4MER CRISPR/Cas12a multiplex knockout platform</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.01.03.522655" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.01.03.522655" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Esmaeili Anvar, N.; Lin, C.; Ma, X.; Wilson, L. L.; Steger, R.; Sangree, A. K.; Colic, M.; Wang, S. H.; Doench, J. G.; Hart, T.</p>
        <p class="info">Score: 32.1, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.01.03.522655' target='https://doi.org/10.1101/2023.01.03.522655'> 10.1101/2023.01.03.522655</a></p>
        <p class="abstract">Genetic interactions mediate the emergence of phenotype from genotype, but initial technologies for combinatorial genetic perturbation in mammalian cells suffer from inefficiency and are challenging to scale. Recent focus on paralog synthetic lethality in cancer cells offers an opportunity to evaluate different approaches and improve on the state of the art. Here we report a meta-analysis of CRISPR genetic interactions screens, identifying a candidate set of background-independent paralog synthetic lethals, and find that the Cas12a platform provides superior sensitivity and assay replicability. We demonstrate that Cas12a can independently target up to four genes from a single guide array, and we build on this knowledge by constructing a genome-scale library that expresses arrays of four guides per clone, a platform we call  in4mer. Our genome-scale human library, with only 49k clones, is substantially smaller than a typical CRISPR/Cas9 monogenic library while also targeting more than four thousand paralog pairs, triples, and quads. Proof of concept screens in four cell lines demonstrate discrimination of core and context-dependent essential genes similar to that of state-of-the-art CRISPR/Cas9 libraries, as well as detection of synthetic lethal and masking/buffering genetic interactions between paralogs of various family sizes, a capability not offered by any extant library. Importantly, the in4mer platform offers a fivefold reduction in the number of clones required to assay genetic interactions, dramatically improving the cost and effort required for these studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.04.556192">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.04.556192" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.04.556192">
        <p class="paperTitle">Correlation of myeloid-derived suppressor cell expansion with upregulated transposable elements in severe COVID-19 unveiled in single-cell RNA sequencing reanalysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.04.556192" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.04.556192" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Farahmandnejad, M.; Mosadeghi, P.; Dorvash, M.; Sakhteman, A.; Faridi, P.; Negahdaripour, M.</p>
        <p class="info">Score: 18.6, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.04.556192' target='https://doi.org/10.1101/2023.09.04.556192'> 10.1101/2023.09.04.556192</a></p>
        <p class="abstract">Some studies investigated the potential role of transposable elements (TEs) in COVID-19 pathogenesis and complications. However, to the best of our knowledge, there is no study to examine the possible association of TEs expression in cell functions and its potential role in COVID-19 immune response at the single-cell level.

In this study, we reanalyzed single-cell RNA seq data of bronchoalveolar lavage (BAL) samples obtained from six severe COVID-19 patients and three healthy donors to assess the probable correlation of TE expression with the immune responses induced by the SARS-CoV-2 virus in COVID-19 patients.

Our findings indicated that the expansion of myeloid-derived suppressor cells (MDSCs) may be a characteristic feature of COVID-19. Additionally, a significant increase in TEs expression in MDSCs was observed. This upregulation of TEs in COVID-19 may be linked to the adaptability of these cells in response to their microenvironments. Furthermore, it appears that the identification of overexpressed TEs by Pattern recognition receptors (PRRs) in MDSCs may enhance the suppressive capacity of these cells. Thus, this study emphasizes the crucial role of TEs in the functionality of MDSCs during COVID-19.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555799">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555799" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555799">
        <p class="paperTitle">Exploiting fluctuations in gene expression to detect causal interactions between genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555799" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555799" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Joly-Smith, E.; Talpur, M. M.; Allard, P.; Papazotos, F.; Potvin-Trottier, L.; Hilfinger, A.</p>
        <p class="info">Score: 11.2, Published: 2023-09-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555799' target='https://doi.org/10.1101/2023.09.01.555799'> 10.1101/2023.09.01.555799</a></p>
        <p class="abstract">Characterizing and manipulating cellular behaviour requires a mechanistic understanding of the causal interactions between cellular components. We present an approach that can detect causal interactions between genes without the need to perturb the physiological state of cells. This approach exploits naturally occurring cell-to-cell variability which is experimentally accessible from static population snapshots of genetically identical cells without the need to follow cells over time. Our main contribution is a simple mathematical relation that constrains the propagation of gene expression noise through biochemical reaction networks. This relation allows us to rigorously interpret fluctuation data even when only a small part of a complex gene regulatory process can be observed. This relation can be exploited to detect causal interactions by synthetically engineering a passive reporter of gene expression, akin to the established &#34;dual reporter assay&#34;. While the focus of our contribution is theoretical, we also present an experimental proof-of-principle to illustrate the approach. Our data from synthetic gene regulatory networks in E. coli are not unequivocal but suggest that the method could prove useful in practice to identify causal interactions between genes from non-genetic cell-to-cell variability.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.03.556134">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.03.556134" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.03.556134">
        <p class="paperTitle">Approximate Bayesian Computation for Inferring Waddington Landscapes from Single Cell Data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.03.556134" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.03.556134" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, Y.; Zhang, S. Y.; Kleijn, I. T.; Stumpf, M. P. H.</p>
        <p class="info">Score: 8.7, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.03.556134' target='https://doi.org/10.1101/2023.09.03.556134'> 10.1101/2023.09.03.556134</a></p>
        <p class="abstract">Single cell technologies allow us to gain insights into cellular processes at unprecedented resolution. In stem cell and developmental biology snapshot data allows us to characterise how the transcriptional state of cells changes between successive cell types. Here we show how approximate Bayesian computation (ABC) can be employed to calibrate mathematical models against single cell data. In our simulation study we demonstrate the pivotal role of the adequate choice of distance measures appropriate for single cell data. We show that for good distance measures, notably optimal transport distances, we can infer parameters for mathematical models from simulated single cell data. We show that the ABC posteriors can be used to characterise parameter sensitivity and identify dependencies between different parameters, and to infer representations of the Waddington or epigenetic landscape, which forms a popular and interpretable representation of the developmental dynamics. In summary, these results pave the way for fitting mechanistic models of stem cell differentiation to single cell data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.04.555678">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.04.555678" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.04.555678">
        <p class="paperTitle">Human subcutaneous and visceral adipocyte atlases uncover classical and specialized adipocytes and depot-specific patterns</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.04.555678" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.04.555678" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lazarescu, O.; Ziv Agam, M.; Haim, Y.; Hekselman, I.; Jubran, J.; Shneyour, A.; Kitsberg, D.; Levin, L.; Liberty, I. F.; Yoel, U.; Dukhno, O.; Adam, M.; Korner, A.; Murphy, R.; Bluher, M.; Habib, N.; Rudich, A.; Yeger-Lotem, E.</p>
        <p class="info">Score: 8.7, Published: 2023-09-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.04.555678' target='https://doi.org/10.1101/2023.09.04.555678'> 10.1101/2023.09.04.555678</a></p>
        <p class="abstract">Human adipose depots are functionally distinct. Yet, recent single-nucleus RNA-sequencing (snRNA-seq) analyses largely uncovered overlapping/similar cell-type landscapes. We hypothesized that adipocytes subtypes, differentiation trajectories, and/or intercellular communication patterns could illuminate this depot similarity-difference gap. For this, we performed snRNA-seq of human subcutaneous and visceral adipose tissue. Whereas the majority of adipocytes in both depots were  classical, namely enriched in lipid metabolism pathways, we also observed  specialized adipocyte subtypes that were enriched in immune-related, extracellular matrix deposition (fibrosis), vascularization/angiogenesis, or ribosomal processes. Pseudo-temporal analysis suggested a developmental trajectory from adipose progenitor cells to classical adipocytes via specialized adipocytes, suggesting that the classical state stems from loss, rather than gain, of specialized functions. Lastly, intercellular communication routes were consistent with the different inflammatory tone of the two depots. Jointly, these findings provide a high-resolution view into the contribution of cellular composition, differentiation, and intercellular communication patterns to human fat depot differences.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.05.556413">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.05.556413" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.05.556413">
        <p class="paperTitle">A hierarchy of metabolite exchanges in metabolic models of microbial species and communities</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.05.556413" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.05.556413" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wedmark, Y. K.; Vik, J. O.; Oyas, O.</p>
        <p class="info">Score: 7.6, Published: 2023-09-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.05.556413' target='https://doi.org/10.1101/2023.09.05.556413'> 10.1101/2023.09.05.556413</a></p>
        <p class="abstract">The metabolic network of an organism can be analyzed as a constraint-based model (CBM). CBM analyses can be biased, optimizing an objective such as growth rate, or unbiased, attempting to describe the feasible flux space through pathway analysis or random flux sampling. Biased methods scale to genome-scale models (GEMs) and beyond, but unbiased methods suffer from combinatorial explosion. One promising approach to improve the scalability of unbiased analyses is to focus on cells&#39; metabolite exchanges with each other and the environment rather than the internal details of their metabolic networks. However, methods for unbiased analysis of metabolite exchanges have not been evaluated or compared. Using CBMs ranging in size from core models to GEMs, we applied pathway enumeration and random flux sampling to metabolite exchanges in microbial species and a microbial community. We focused on pathway definitions that allow direct targeting of metabolite exchanges: elementary conversion modes (ECMs), elementary flux patterns (EFPs), and minimal pathways (MPs). Enumerating growth-supporting ECMs, EFPs, and MPs for the same models, we found that the MPs were always a subset of the ECMs and EFPs, specifically the subset of support-minimal pathways. The EFPs were always a subset of the ECMs, which in turn were always a subset of flux patterns from elementary flux modes (EFMs). We argue that this hierarchical relationship between pathway definitions holds more generally and that it helps explain the relative scalability of ECM, EFP, and MP enumeration. Metabolite exchange frequencies in enumerated pathways were similar across methods with a few specific exchanges explaining large differences in pathway counts. Thus, biological interpretation was largely robust to the choice of pathway definition. Random flux sampling scaled to GEMs and complemented pathway enumeration.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.03.556147">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.03.556147" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.03.556147">
        <p class="paperTitle">A dynamic Boolean model of molecular and cellular interactions represents psoriasis development and predicts drug candidates.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.03.556147" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.03.556147" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tsirvouli, E.; Noël, V.; Flobak, A.; Calzone, L.; Kuiper, M.</p>
        <p class="info">Score: 5.0, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.03.556147' target='https://doi.org/10.1101/2023.09.03.556147'> 10.1101/2023.09.03.556147</a></p>
        <p class="abstract">Psoriasis is a chronic skin disease affecting 2-3% of the global population. Psoriasis arises from complex interactions between keratinocytes and immune cells, leading to uncontrolled inflammation, immune hyperactivation and perturbed keratinocyte life cycle. Although the latest generation of drugs have greatly improved psoriasis management, the disease remains incurable, and the substantial variability in treatment response calls for novel approaches to comprehend the intricate mechanisms underlying disease development and to discover potential drug targets. In this study, we present a multiscale population model that captures the dynamics of cell-specific phenotypes in psoriasis, integrating discrete logical formalism and population dynamics simulations. Through simulations and network metrics, we identify potential pairwise interventions as alternative treatment options. Specifically, The model predictions suggest that targeting neutrophil activation in conjunction with either PGE2 production or STAT3 signaling shows promise comparable to IL-17 inhibition, which is currently the most used treatment option for moderate and severe cases of psoriasis. Our findings underscore the significance of considering complex intercellular interactions and intracellular signaling cascades in psoriasis, and highlight the importance of computational approaches in unraveling complex biological systems for drug target identification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.31.555601">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.31.555601" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.31.555601">
        <p class="paperTitle">PTMNavigator: Interactive Visualization of Differentially Regulated Post-Translational Modifications in Cellular Signaling Pathways</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.31.555601" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.31.555601" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mueller, J.; Bayer, F. P.; Wilhelm, M.; Kuster, B.; The, M.</p>
        <p class="info">Score: 2.0, Published: 2023-09-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.31.555601' target='https://doi.org/10.1101/2023.08.31.555601'> 10.1101/2023.08.31.555601</a></p>
        <p class="abstract">Post-translational modifications (PTMs) play a governing role in regulating cellular signaling, fine-tuning protein function, and orchestrating complex biological processes. Despite their importance, the lack of comprehensive tools for studying PTMs from a pathway-centric perspective has limited our ability to understand how PTMs modulate cellular pathways on a molecular level. Here, we present PTMNavigator, a tool integrated into the ProteomicsDB platform, which offers an interactive interface for researchers to overlay experimental PTM data on top of [~]3000 well-curated pathway diagrams. This offers a comprehensive view of the intricate relationship between PTMs and signaling pathways. To demonstrate the utility of PTMNavigator, we applied it to two phosphoproteomics perturbation datasets. First, PTMNavigator enhanced pathway enrichment analysis by showing how the regulated peptides are distributed in the pathways with high enrichment scores. Second, it visualized how drug treatments result in a discernable flow of PTM-driven signaling within pathways. Third, PTMNavigator aided in extending existing pathways by uncovering previously unknown relationships between PTMs and pathway components. By facilitating the discovery of novel PTM-pathway interactions and enhancing our understanding of cellular signaling dynamics, PTMNavigator advances our knowledge of PTM biology and its implications in health and disease.

One Sentence SummaryPTMNavigator is a software that overlays pathway diagrams with PTM proteomics data and allows their interactive exploration.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.556617">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.556617" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.556617">
        <p class="paperTitle">Agent-based modeling of tumor-immune interactions reveals determinants of final tumor states</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.556617" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.556617" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ahmidouch, M.; Tangella, N.; Finley, S. D.</p>
        <p class="info">Score: 1.2, Published: 2023-09-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.556617' target='https://doi.org/10.1101/2023.09.06.556617'> 10.1101/2023.09.06.556617</a></p>
        <p class="abstract">Interactions between tumor and immune cells in the tumor microenvironment (TME) influence tumor growth and the tumor&#39;s response to treatment. Excitingly, this complex landscape of tumor-immune interactions can be studied using computational modeling. Mathematical oncology can provide quantitative insights into the TME, serving as a framework for understanding tumor dynamics. Here, we use an agent-based model to simulate the interactions among cancer cells, macrophages (naive, M1, and M2), and T cells (active CD8&#43; and inactive) in a 2D representation of the TME. Key diffusible factors, IL-4 and IFN-{gamma}, are also incorporated. We apply the model to predict how cell-specific properties influence tumor progression. The model predictions and analyses revealed the relationships between different cell populations and highlighted the importance of macrophages and T cells in shaping the TME. Thus, we quantify how components of the TME influence the final tumor state and the effects of macrophage-based therapies. The findings emphasize the significant role of computational models in unraveling the intricate dynamics of tumor-immune interactions and their potential for guiding the development of tailored immunotherapeutic strategies. This study provides a foundation for future investigations aiming to refine and expand the model, validate predictions experimentally, and pave the way for improved cancer treatments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.556534">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.556534" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.556534">
        <p class="paperTitle">Systems genetics analysis of human body fat distribution genes identifies Wnt signaling and mitochondrial activity in adipocytes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.556534" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.556534" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Reed, J. N.; Huang, J.; Li, Y.; Ma, L.; Banka, D.; Wabitsch, M.; Wang, T.; Ding, W.; Bjorkegren, J. L. M.; Civelek, M.</p>
        <p class="info">Score: 1.2, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.556534' target='https://doi.org/10.1101/2023.09.06.556534'> 10.1101/2023.09.06.556534</a></p>
        <p class="abstract">BACKGROUND: Excess fat in the abdomen is a sexually dimorphic risk factor for cardio-metabolic disease. The relative storage between abdominal and lower-body subcutaneous adipose tissue depots is approximated by the waist-to-hip ratio adjusted for body mass index (WHRadjBMI). Genome-wide association studies (GWAS) identified 346 loci near 495 genes associated with WHRadjBMI. Most of these genes have unknown roles in fat distribution, but many are expressed and putatively act in adipose tissue. We aimed to identify novel sex- and depot-specific drivers of WHRadjBMI using a systems genetics approach. METHODS: We used two independent cohorts of adipose tissue gene expression with 362 - 444 males and 147 - 219 females, primarily of European ancestry. We constructed sex- and depot-specific Bayesian networks to model the gene-gene interactions from 8,492 adipose tissue genes. Key driver analysis identified genes that, in silico and putatively in vitro, regulate many others, including the 495 WHRadjBMI GWAS genes. Key driver gene function was determined by perturbing their expression in human subcutaneous pre-adipocytes using lenti-virus or siRNA. RESULTS: 51 - 119 key drivers in each network were replicated in both cohorts. We used single-cell expression data to select replicated key drivers expressed in adipocyte precursors and mature adipocytes, prioritized genes which have not been previously studied in adipose tissue, and used public human and mouse data to nominate 53 novel key driver genes (10 - 21 from each network) that may regulate fat distribution by altering adipocyte function. In other cell types, 23 of these genes are found in crucial adipocyte pathways: Wnt signaling or mitochondrial function. We selected seven genes whose expression is highly correlated with WHRadjBMI to further study their effects on adipogenesis/Wnt signaling (ANAPC2, PSME3, RSPO1, TYRO3) or mitochondrial function (C1QTNF3, MIGA1, PSME3, UBR1). Adipogenesis was inhibited in cells overexpressing ANAPC2 and RSPO1 compared to controls. RSPO1 results are consistent with a positive correlation between gene expression in the subcutaneous depot and WHRadjBMI, therefore lower relative storage in the subcutaneous depot. RSPO1 inhibited adipogenesis by increasing {beta}-catenin activation and Wnt-related transcription, thus repressing PPARG and CEBPA. PSME3 overexpression led to more adipogenesis than controls. In differentiated adipocytes, MIGA1 and UBR1 downregulation led to mitochondrial dysfunction, with lower oxygen consumption than controls; MIGA1 knockdown also lowered UCP1 expression. SUMMARY: ANAPC2, MIGA1, PSME3, RSPO1, and UBR1 affect adipocyte function and may drive body fat distribution.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
