<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="The fitness cost of spurious phosphorylation
Authors: Bradley, D.; Hogrebe, A.; Dandage, R.; Dube, A. K.; Leutert, M.; Dionne, U.; Chang, A.; Villen, J.; Landry, C. R.
Score: 17.0, Published: 2023-10-10 DOI: 10.1101/2023.10.08.561337
The fidelity of signal transduction requires the binding of regulatory molecules to their cognate targets. However, the crowded cell interior risks off-target interactions between proteins that are functionally unrelated. How such off-target interactions impact fitness is not generally known, but quantifying this is required to understand the constraints faced by cell systems as they evolve.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="The fitness cost of spurious phosphorylation
Authors: Bradley, D.; Hogrebe, A.; Dandage, R.; Dube, A. K.; Leutert, M.; Dionne, U.; Chang, A.; Villen, J.; Landry, C. R.
Score: 17.0, Published: 2023-10-10 DOI: 10.1101/2023.10.08.561337
The fidelity of signal transduction requires the binding of regulatory molecules to their cognate targets. However, the crowded cell interior risks off-target interactions between proteins that are functionally unrelated. How such off-target interactions impact fitness is not generally known, but quantifying this is required to understand the constraints faced by cell systems as they evolve." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-15T10:38:01+00:00" />
<meta property="article:modified_time" content="2023-10-15T10:38:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="The fitness cost of spurious phosphorylation
Authors: Bradley, D.; Hogrebe, A.; Dandage, R.; Dube, A. K.; Leutert, M.; Dionne, U.; Chang, A.; Villen, J.; Landry, C. R.
Score: 17.0, Published: 2023-10-10 DOI: 10.1101/2023.10.08.561337
The fidelity of signal transduction requires the binding of regulatory molecules to their cognate targets. However, the crowded cell interior risks off-target interactions between proteins that are functionally unrelated. How such off-target interactions impact fitness is not generally known, but quantifying this is required to understand the constraints faced by cell systems as they evolve."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "The fitness cost of spurious phosphorylation\nAuthors: Bradley, D.; Hogrebe, A.; Dandage, R.; Dube, A. K.; Leutert, M.; Dionne, U.; Chang, A.; Villen, J.; Landry, C. R.\nScore: 17.0, Published: 2023-10-10 DOI: 10.1101/2023.10.08.561337\nThe fidelity of signal transduction requires the binding of regulatory molecules to their cognate targets. However, the crowded cell interior risks off-target interactions between proteins that are functionally unrelated. How such off-target interactions impact fitness is not generally known, but quantifying this is required to understand the constraints faced by cell systems as they evolve.",
  "keywords": [
    
  ],
  "articleBody": " The fitness cost of spurious phosphorylation\nAuthors: Bradley, D.; Hogrebe, A.; Dandage, R.; Dube, A. K.; Leutert, M.; Dionne, U.; Chang, A.; Villen, J.; Landry, C. R.\nScore: 17.0, Published: 2023-10-10 DOI: 10.1101/2023.10.08.561337\nThe fidelity of signal transduction requires the binding of regulatory molecules to their cognate targets. However, the crowded cell interior risks off-target interactions between proteins that are functionally unrelated. How such off-target interactions impact fitness is not generally known, but quantifying this is required to understand the constraints faced by cell systems as they evolve. Here, we use the model organism S. cerevisiae to inducibly express tyrosine kinases. Because yeast lacks bona fide tyrosine kinases, most of the resulting tyrosine phosphorylation is spurious. This provides a suitable system to measure the impact of artificial protein interactions on fitness. We engineered 44 yeast strains each expressing a tyrosine kinase, and quantitatively analysed their phosphoproteomes. This analysis resulted in [~]30,000 phosphosites mapping to [~]3,500 proteins. Examination of the fitness costs in each strain revealed a strong correlation between the number of spurious pY sites and decreased growth. Moreover, the analysis of pY effects on protein structure and on protein function revealed over 1000 pY events that we predict to be deleterious. However, we also find that a large number of the spurious pY sites have a negligible effect on fitness, possibly because of their low stoichiometry. This result is consistent with our evolutionary analyses demonstrating a lack of phosphotyrosine counter-selection in species with bona fide tyrosine kinases. Taken together, our results suggest that, alongside the risk for toxicity, the cell can tolerate a large degree of non-functional crosstalk as interaction networks evolve.\nSpatial heterogeneity in tumor adhesion qualifies collective cell migration\nAuthors: Prasanna, C. V. S.; Jolly, M. K.; Bhat, R.\nScore: 5.3, Published: 2023-10-06 DOI: 10.1101/2023.10.05.559967\nCollective cell migration, a canon of most invasive solid tumors, is an emergent property of the interactions between cancer cells and their surrounding extracellular matrix (ECM). However, tumor populations invariably consist of cells expressing variable levels of adhesive proteins that mediate such interactions, disallowing an intuitive understanding of how tumor invasiveness at a multicellular scale is influenced by spatial heterogeneity of cell-cell and cell-ECM adhesion. Here, we have used a Cellular Potts model-based multiscale computational framework that is constructed on the histopathological principles of glandular cancers. In earlier efforts on homogenous cancer cell populations, this framework revealed the relative ranges of interactions, including cell-cell and cell-ECM adhesion that drove collective, dispersed, and mixed multimodal migrations. Here, we constitute a tumor core of two separate cell subsets showing distinct intra-subset cell-cell or cell-ECM adhesion strengths. These two subsets of cells are arranged to varying extents of spatial intermingling, which we call the heterogeneity index (HI). Our simulations show that for a given two intra-subset cell-cell adhesions for two subsets of cells, low and high inter-subset cell adhesion favors migration of high HI and low HI intermingled populations, respectively. In addition, for the most explored values of cell-ECM adhesion strengths, populations with high HI values collectively migrate better than those with lower HI values. We then asked how spatial migration is regulated by progressively intermingled cellular subsets that were epithelial, i.e., showed high cell-cell but poor cell-ECM adhesion, and mesenchymal, i.e., with reversed adhesion strengths to the former. Here too, inter-subset adhesion plays an important role in contextualizing the proportionate relationship between HI and migration. We also observe an exception to this relationship for cases of heterogeneous cell-ECM adhesion where sub-maximal HI patterns with higher outer localization of cells with stronger ECM adhesion collectively migrate better than their relatively higher HI counterparts. Our simulations also reveal how adhesion heterogeneity qualifies migrative dynamics through collective cellular unjamming, when either cell-cell or -ECM adhesion type is varied but incorporates dispersion when both adhesion types are simultaneously altered.\nIntegrating microbial abundance time series with fermentation dynamics of the rumen microbiome via mathematical modelling\nAuthors: Davoudkhani, M.; Rubino, F.; Creevey, C. J.; Ahvenjarvi, S.; Bayat, A. R.; Tapio, I.; Belanche, A.; Munoz-Tamayo, R.\nScore: 5.5, Published: 2023-10-06 DOI: 10.1101/2023.10.05.560454\nThe rumen represents a dynamic microbial ecosystem where fermentation metabolites and microbial concentrations change over time in response to dietary changes. The integration of microbial genomic knowledge and dynamic modelling can enhance our system-level understanding of rumen ecosystems function. However, such an integration between dynamic models and rumen microbiota data is lacking. The objective of this work was to integrate rumen microbiota time series determined by 16S rRNA gene amplicon sequencing into a dynamic modelling framework to link microbial data to the dynamics of the volatile fatty acids (VFA) production during fermentation. For that, we used the theory of state observers to develop a model that estimates the dynamics of VFA from the data of microbial functional proxies associated with the specific production of each VFA. We determined the microbial proxies using CowPi to infer the functional potential of the rumen microbiota and extrapolate their functional modules from KEGG (Kyoto Encyclopedia of Genes and Genomes). The approach was challenged using data from an in vitro RUSITEC experiment and from an in vivo experiment with four cows. The model performance was evaluated by the coefficient of variation of the root mean square error (CRMSE). For the in vitro case study, the mean CVRMSE were 9.8% for acetate, 14% for butyrate and 14.5% for propionate. For the in vivo case study, the mean CVRMSE were 16.4% for acetate, 15.8% for butyrate and 19.8% for propionate. The mean CVRMSE for the VFA molar fractions were 3.1% for acetate, 3.8% for butyrate and 8.9% for propionate. Ours results show the promising application of state observers integrated with microbiota time series data for predicting rumen microbial metabolism.\nModelSEED v2: High-throughput genome-scale metabolic model reconstruction with enhanced energy biosynthesis pathway prediction\nAuthors: Faria, J. P.; Liu, F.; Edirisinghe, J. N.; Gupta, N.; Seaver, S. M. D.; Freiburger, A. P.; Zhang, Q.; Weisenhorn, P.; Conrad, N.; Zarecki, R.; Song, H.-S.; DeJongh, M.; Best, A. A.; Cottingham, R. W.; Arkin, A. P.; Henry, C. S.\nScore: 8.1, Published: 2023-10-06 DOI: 10.1101/2023.10.04.556561\nSince the release of ModelSEED in 2010, the systems biology research community has used the ModelSEED genome-scale metabolic model reconstruction pipeline to build over 200,000 draft metabolic reconstructions that support hundreds of publications. Here we describe the first comprehensive update to this reconstruction tool, with new features such as (i) a dramatically improved representation of energy metabolism, which ensures that models produce accurate amounts of ATP per mol of nutrient consumed; (ii) a new template for Archaea model reconstruction; and (iii) a significantly improved curation of all metabolic pathways with mappings to RAST subsystems annotations. We applied the improved pipeline to build and analyze core and genome-scale models for Archaea and Bacteria genomes in KEGG. The new ModelSEED pipeline generates larger models that possess more reactions and genes and require fewer gap-filled reactions. In addition, we see conserved patterns in the ATP biosynthesis mechanism across phylogeny, and identify clades where our understanding of energy biosynthesis is still poor. The ModelSEED v2 pipeline is currently available only as new reconstruction and gap-filling Apps in the KBase platform.\nComputational divergence analysis reveals the existence of regulatory degeneration and supports HDAC1 as a potential drug target for Alzheimer's disease\nAuthors: Wang, Q.; Zhao, Z.; Xu, H.; Lu, D.; Xia, J.; Zhang, W.; Meng, G.\nScore: 1.8, Published: 2023-10-08 DOI: 10.1101/2023.10.05.561015\nEpigenetic dysregulation has been widely reported in patients of Alzheimers disease (AD) and epigenetic drugs are gaining particular interest as a potential candidate therapy target. However, it is less clear how epigenetic dysregulation contributes to AD development. In this work, we performed regulatory divergence analysis using large-scale AD brain RNA-seq data and reported a widespread existence of regulatory degeneration among AD patients. It seems that transcription factor (TF)-mediated regulations get weakened or lost during AD development, resulting in disruption of normal neuronal function, especially including protein degradation, neuroinflammation, mitochondria and synaptic dysfunction. The regulatory degeneration burden (RDB) is well correlated with the detrimental clinical manifestations of AD patients. Studies of epigenetic marks, including histone modification, open chromatin accessibility and three TF binding sites supported the existence of regulatory degeneration. It suggested that epigenetic dysregulation contributed to regulatory degeneration, which also explained the consequence of epigenetic dysregulation. Among the epigenetic regulators, HDAC1 was proposed as a potential participator in such a process. Overall, our computational analysis suggested a novel causal mechanism of AD development and proposed HDAC1 as a drug target to treat AD.\nSmurfness-based two-phase model of ageing helps deconvolve the ageing transcriptional signature\nAuthors: Zane, F.; Bouzid, H.; Sosa Marmol, S.; Brazane, M.; Besse, S.; Molina, J. L.; Cansell, C.; Aprahamian, F.; Durand, S.; Ayache, J.; Antoniewski, C.; CARRE, C.; Todd, N.; Rera, M.\nScore: 12.4, Published: 2023-09-27 DOI: 10.1101/2022.11.22.517330\nAgeing is characterised at the molecular level by six transcriptional hallmarks of ageing, that are commonly described as progressively affected as time passes. By contrast, the Smurf assay separates high-and-constant-mortality risk individuals from healthy, zero-mortality risk individuals, based on increased intestinal permeability. Performing whole body total RNA sequencing, we found that Smurfness distinguishes transcriptional changes associated with chronological age from those associated with biological age. We show that transcriptional heterogeneity increases with chronological age in non-Smurf individuals preceding the other five hallmarks of ageing, that are specifically associated with the Smurf state. Using this approach, we also devise targeted pro-longevity genetic interventions delaying entry in the Smurf state. We anticipate that increased attention to the evolutionary conserved Smurf phenotype will bring about significant advances in our understanding of the mechanisms of ageing. Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=150 HEIGHT=200 SRC=\"FIGDIR/small/517330v5_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (35K): org.highwire.dtl.DTLVardef@139b16dorg.highwire.dtl.DTLVardef@af845aorg.highwire.dtl.DTLVardef@11a52eaorg.highwire.dtl.DTLVardef@139896b_HPS_FORMAT_FIGEXP M_FIG The two-phase model of ageing allows to study separately the effect of chronological and physiological age. (A) Classic approaches for studying ageing tend to consider it as a black box affecting all individuals progressively from birth to death. Instead, the Smurf phenotype shows that life can be divided into two consecutive phases separated by an abrupt transition. (B) All individuals undergo this transition at a different moment in their life, prior to death. This allows us to switch from population based approaches, comparing bulks of age-matched individuals through time, to individuals-centred approaches relying on direct access to their transition status. (C) Such paradigm shift shows that hallmarks of ageing long thought to progressively change with age are actually mostly affected in a growing proportion of Smurfs, allowing for the identification of the chain of events accompanying ageing and death from natural causes. (D) By studying the behaviour of the ageing transcriptome as a function of chronological age and Smurfness separately, we demonstrate that the progressively changing transcriptional ageing signature, as described in Frenk \u0026 Houseley (2018), is in fact the convolution changes accompanying chronological age signature (increased transcriptional noise) and changes associated with Smurfness (or biological age) signature (increased stress response and inflammation, decreased expression of ribosomal and mitochondrial genes). We also identified a hallmark partially associated with only old Smurfs (ATH5), suggesting that chronological age can affect, late in life, the Smurf response. C_FIG\nPreponderance of generalized chain functions in reconstructed Boolean models of biological networks\nAuthors: Mitra, S.; Sil, P.; Subbaroyan, A.; Martin, O. C.; Samal, A.\nScore: 1.2, Published: 2023-10-10 DOI: 10.1101/2023.10.08.561412\nBoolean networks (BNs) have been extensively used to model the dynamics of gene regulatory networks (GRNs) that underlie cellular decisions. The dynamics of BNs depend on the network architecture and regulatory logic rules (or Boolean functions (BFs)) associated with nodes, both of which have been shown to be far from random in large-scale studies of reconstructed Boolean models. At the level of the BFs, nested canalyzing functions (NCFs) have been shown to be strongly enriched in such GRN models. The central question we address here is whether that enrichment is due to certain sub-types of NCFs. To answer this, we build on one sub-type of NCFs, the chain functions (or chain-0 functions) proposed by Gat-Viks and Shamir. First, we propose 2 other sub-types of NCFs, namely, the class of chain-1 functions which is the dual of the class of chain-0 functions, and generalized chain functions, the union of the chain-0 and chain-1 types. Next, we find that the fraction of NCFs that are chain-0 functions decays exponentially with the number of inputs, and exhibits a fractal-like behaviour as a function of the bias for a fixed number of inputs. Moreover, we explain several of these observations analytically. Then, by analyzing 5990 BFs extracted from a large dataset of reconstructed Boolean models, and 2 other datasets, we find that generalized chain functions are significantly enriched within the NCFs. Lastly, we illustrate the severe restriction imposed by generalized chain functions compared to NCFs for 3 biological models and perform model selection on them using known relative stability constraints.\nVirus-host protein co-expression networks reveal temporal organization and strategies of viral infection\nAuthors: Guantes, R.; Aguirre, J.\nScore: 1.1, Published: 2023-10-10 DOI: 10.1101/2023.10.10.561729\nViral replication is a complex dynamical process involving the global remodelling of the host cellular machinery across several stages. In this study, we provide a unified view of the virus-host interaction at the proteome level reconstructing protein co-expression networks from quantitative temporal data of four large DNA viruses. We take advantage of a formal framework, the theory of interacting networks, to describe the viral infection as a dynamical system taking place on a network of networks where perturbations induced by viral proteins spread to hijack the host proteome for the virus benefit. Our methodology demonstrates how the viral replication cycle can be effectively examined as a complex interaction between protein networks, providing useful insights into the viral and host temporal organization and strategies, key protein nodes targeted by the virus and dynamical bottlenecks during the course of the infection.\nComputational modeling establishes mechanotransduction as a potent entrainment cue for the mammalian circadian clock\nAuthors: Francis, E. A.; Rangamani, P.\nScore: 1.0, Published: 2023-10-10 DOI: 10.1101/2023.10.09.561563\nMechanotransduction, which is the integration of mechanical signals from the cells external environment to changes in intracellular signaling, governs many cellular functions. Recent studies show that the mechanical state of the cell is also coupled to the cellular circadian clock. To investigate possible interactions between circadian rhythms and cellular mechanotransduction, we developed a computational model that integrates these two pathways. Combining previous models of mechan-otransduction and circadian clocks, we postulated that translocation of YAP/TAZ and MRTF into the nucleus leads to altered expression of circadian proteins. Simulations from our model predict that lower levels of cytoskeletal activity are associated with longer circadian oscillation periods and higher oscillation amplitudes, consistent with recent experimental observations. Furthermore, accumulation of YAP/TAZ and MRTF in the nucleus causes circadian oscillations to decay. These effects hold both at the single-cell level and within a population-level framework. Finally, we investigated the effects of mutations in YAP or lamin A, the latter of which lead to a class of diseases known as laminopathies. Oscillations in PER/CRY and BMAL1 are substantially weaker in populations of cells with in silico mutations in YAP and lamin A, suggesting that defects in mechanotransduction can disrupt the circadian clock in certain disease states. However, we find that normal oscillatory behavior can be rescued by lowering substrate stiffness, indicating a possible biological compensatory mechanism. Thus our study identifies that mechanotransduction could be a potent entrainment cue for cellular clocks and this crosstalk can be leveraged to rescue the circadian clock in disease states.\n",
  "wordCount" : "2557",
  "inLanguage": "en",
  "datePublished": "2023-10-15T10:38:01Z",
  "dateModified": "2023-10-15T10:38:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta"><span>updated on October 15, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.08.561337">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.08.561337" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.08.561337">
        <p class="paperTitle">The fitness cost of spurious phosphorylation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.08.561337" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.08.561337" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bradley, D.; Hogrebe, A.; Dandage, R.; Dube, A. K.; Leutert, M.; Dionne, U.; Chang, A.; Villen, J.; Landry, C. R.</p>
        <p class="info">Score: 17.0, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.08.561337' target='https://doi.org/10.1101/2023.10.08.561337'> 10.1101/2023.10.08.561337</a></p>
        <p class="abstract">The fidelity of signal transduction requires the binding of regulatory molecules to their cognate targets. However, the crowded cell interior risks off-target interactions between proteins that are functionally unrelated. How such off-target interactions impact fitness is not generally known, but quantifying this is required to understand the constraints faced by cell systems as they evolve. Here, we use the model organism S. cerevisiae to inducibly express tyrosine kinases. Because yeast lacks bona fide tyrosine kinases, most of the resulting tyrosine phosphorylation is spurious. This provides a suitable system to measure the impact of artificial protein interactions on fitness. We engineered 44 yeast strains each expressing a tyrosine kinase, and quantitatively analysed their phosphoproteomes. This analysis resulted in [~]30,000 phosphosites mapping to [~]3,500 proteins. Examination of the fitness costs in each strain revealed a strong correlation between the number of spurious pY sites and decreased growth. Moreover, the analysis of pY effects on protein structure and on protein function revealed over 1000 pY events that we predict to be deleterious. However, we also find that a large number of the spurious pY sites have a negligible effect on fitness, possibly because of their low stoichiometry. This result is consistent with our evolutionary analyses demonstrating a lack of phosphotyrosine counter-selection in species with bona fide tyrosine kinases. Taken together, our results suggest that, alongside the risk for toxicity, the cell can tolerate a large degree of non-functional crosstalk as interaction networks evolve.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.559967">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.559967" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.559967">
        <p class="paperTitle">Spatial heterogeneity in tumor adhesion qualifies collective cell migration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.559967" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.559967" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Prasanna, C. V. S.; Jolly, M. K.; Bhat, R.</p>
        <p class="info">Score: 5.3, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.559967' target='https://doi.org/10.1101/2023.10.05.559967'> 10.1101/2023.10.05.559967</a></p>
        <p class="abstract">Collective cell migration, a canon of most invasive solid tumors, is an emergent property of the interactions between cancer cells and their surrounding extracellular matrix (ECM). However, tumor populations invariably consist of cells expressing variable levels of adhesive proteins that mediate such interactions, disallowing an intuitive understanding of how tumor invasiveness at a multicellular scale is influenced by spatial heterogeneity of cell-cell and cell-ECM adhesion. Here, we have used a Cellular Potts model-based multiscale computational framework that is constructed on the histopathological principles of glandular cancers. In earlier efforts on homogenous cancer cell populations, this framework revealed the relative ranges of interactions, including cell-cell and cell-ECM adhesion that drove collective, dispersed, and mixed multimodal migrations. Here, we constitute a tumor core of two separate cell subsets showing distinct intra-subset cell-cell or cell-ECM adhesion strengths. These two subsets of cells are arranged to varying extents of spatial intermingling, which we call the heterogeneity index (HI). Our simulations show that for a given two intra-subset cell-cell adhesions for two subsets of cells, low and high inter-subset cell adhesion favors migration of high HI and low HI intermingled populations, respectively. In addition, for the most explored values of cell-ECM adhesion strengths, populations with high HI values collectively migrate better than those with lower HI values. We then asked how spatial migration is regulated by progressively intermingled cellular subsets that were epithelial, i.e., showed high cell-cell but poor cell-ECM adhesion, and mesenchymal, i.e., with reversed adhesion strengths to the former. Here too, inter-subset adhesion plays an important role in contextualizing the proportionate relationship between HI and migration. We also observe an exception to this relationship for cases of heterogeneous cell-ECM adhesion where sub-maximal HI patterns with higher outer localization of cells with stronger ECM adhesion collectively migrate better than their relatively higher HI counterparts. Our simulations also reveal how adhesion heterogeneity qualifies migrative dynamics through collective cellular unjamming, when either cell-cell or -ECM adhesion type is varied but incorporates dispersion when both adhesion types are simultaneously altered.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.560454">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.560454" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.560454">
        <p class="paperTitle">Integrating microbial abundance time series with fermentation dynamics of the rumen microbiome via mathematical modelling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.560454" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.560454" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Davoudkhani, M.; Rubino, F.; Creevey, C. J.; Ahvenjarvi, S.; Bayat, A. R.; Tapio, I.; Belanche, A.; Munoz-Tamayo, R.</p>
        <p class="info">Score: 5.5, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.560454' target='https://doi.org/10.1101/2023.10.05.560454'> 10.1101/2023.10.05.560454</a></p>
        <p class="abstract">The rumen represents a dynamic microbial ecosystem where fermentation metabolites and microbial concentrations change over time in response to dietary changes. The integration of microbial genomic knowledge and dynamic modelling can enhance our system-level understanding of rumen ecosystems function. However, such an integration between dynamic models and rumen microbiota data is lacking. The objective of this work was to integrate rumen microbiota time series determined by 16S rRNA gene amplicon sequencing into a dynamic modelling framework to link microbial data to the dynamics of the volatile fatty acids (VFA) production during fermentation. For that, we used the theory of state observers to develop a model that estimates the dynamics of VFA from the data of microbial functional proxies associated with the specific production of each VFA. We determined the microbial proxies using CowPi to infer the functional potential of the rumen microbiota and extrapolate their functional modules from KEGG (Kyoto Encyclopedia of Genes and Genomes). The approach was challenged using data from an in vitro RUSITEC experiment and from an in vivo experiment with four cows. The model performance was evaluated by the coefficient of variation of the root mean square error (CRMSE). For the in vitro case study, the mean CVRMSE were 9.8% for acetate, 14% for butyrate and 14.5% for propionate. For the in vivo case study, the mean CVRMSE were 16.4% for acetate, 15.8% for butyrate and 19.8% for propionate. The mean CVRMSE for the VFA molar fractions were 3.1% for acetate, 3.8% for butyrate and 8.9% for propionate. Ours results show the promising application of state observers integrated with microbiota time series data for predicting rumen microbial metabolism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.556561">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.556561" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.556561">
        <p class="paperTitle">ModelSEED v2: High-throughput genome-scale metabolic model reconstruction with enhanced energy biosynthesis pathway prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.556561" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.556561" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Faria, J. P.; Liu, F.; Edirisinghe, J. N.; Gupta, N.; Seaver, S. M. D.; Freiburger, A. P.; Zhang, Q.; Weisenhorn, P.; Conrad, N.; Zarecki, R.; Song, H.-S.; DeJongh, M.; Best, A. A.; Cottingham, R. W.; Arkin, A. P.; Henry, C. S.</p>
        <p class="info">Score: 8.1, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.556561' target='https://doi.org/10.1101/2023.10.04.556561'> 10.1101/2023.10.04.556561</a></p>
        <p class="abstract">Since the release of ModelSEED in 2010, the systems biology research community has used the ModelSEED genome-scale metabolic model reconstruction pipeline to build over 200,000 draft metabolic reconstructions that support hundreds of publications. Here we describe the first comprehensive update to this reconstruction tool, with new features such as (i) a dramatically improved representation of energy metabolism, which ensures that models produce accurate amounts of ATP per mol of nutrient consumed; (ii) a new template for Archaea model reconstruction; and (iii) a significantly improved curation of all metabolic pathways with mappings to RAST subsystems annotations. We applied the improved pipeline to build and analyze core and genome-scale models for Archaea and Bacteria genomes in KEGG. The new ModelSEED pipeline generates larger models that possess more reactions and genes and require fewer gap-filled reactions. In addition, we see conserved patterns in the ATP biosynthesis mechanism across phylogeny, and identify clades where our understanding of energy biosynthesis is still poor. The ModelSEED v2 pipeline is currently available only as new reconstruction and gap-filling Apps in the KBase platform.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.561015">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.561015" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.561015">
        <p class="paperTitle">Computational divergence analysis reveals the existence of regulatory degeneration and supports HDAC1 as a potential drug target for Alzheimer&#39;s disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.561015" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.561015" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Q.; Zhao, Z.; Xu, H.; Lu, D.; Xia, J.; Zhang, W.; Meng, G.</p>
        <p class="info">Score: 1.8, Published: 2023-10-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.561015' target='https://doi.org/10.1101/2023.10.05.561015'> 10.1101/2023.10.05.561015</a></p>
        <p class="abstract">Epigenetic dysregulation has been widely reported in patients of Alzheimers disease (AD) and epigenetic drugs are gaining particular interest as a potential candidate therapy target. However, it is less clear how epigenetic dysregulation contributes to AD development. In this work, we performed regulatory divergence analysis using large-scale AD brain RNA-seq data and reported a widespread existence of regulatory degeneration among AD patients. It seems that transcription factor (TF)-mediated regulations get weakened or lost during AD development, resulting in disruption of normal neuronal function, especially including protein degradation, neuroinflammation, mitochondria and synaptic dysfunction. The regulatory degeneration burden (RDB) is well correlated with the detrimental clinical manifestations of AD patients. Studies of epigenetic marks, including histone modification, open chromatin accessibility and three TF binding sites supported the existence of regulatory degeneration. It suggested that epigenetic dysregulation contributed to regulatory degeneration, which also explained the consequence of epigenetic dysregulation. Among the epigenetic regulators, HDAC1 was proposed as a potential participator in such a process. Overall, our computational analysis suggested a novel causal mechanism of AD development and proposed HDAC1 as a drug target to treat AD.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.11.22.517330">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.11.22.517330" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.11.22.517330">
        <p class="paperTitle">Smurfness-based two-phase model of ageing helps deconvolve the ageing transcriptional signature</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.11.22.517330" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.11.22.517330" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zane, F.; Bouzid, H.; Sosa Marmol, S.; Brazane, M.; Besse, S.; Molina, J. L.; Cansell, C.; Aprahamian, F.; Durand, S.; Ayache, J.; Antoniewski, C.; CARRE, C.; Todd, N.; Rera, M.</p>
        <p class="info">Score: 12.4, Published: 2023-09-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.11.22.517330' target='https://doi.org/10.1101/2022.11.22.517330'> 10.1101/2022.11.22.517330</a></p>
        <p class="abstract">Ageing is characterised at the molecular level by six transcriptional  hallmarks of ageing, that are commonly described as progressively affected as time passes. By contrast, the  Smurf assay separates high-and-constant-mortality risk individuals from healthy, zero-mortality risk individuals, based on increased intestinal permeability. Performing whole body total RNA sequencing, we found that Smurfness distinguishes transcriptional changes associated with chronological age from those associated with biological age. We show that transcriptional heterogeneity increases with chronological age in non-Smurf individuals preceding the other five hallmarks of ageing, that are specifically associated with the Smurf state. Using this approach, we also devise targeted pro-longevity genetic interventions delaying entry in the Smurf state. We anticipate that increased attention to the evolutionary conserved Smurf phenotype will bring about significant advances in our understanding of the mechanisms of ageing.

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=150 HEIGHT=200 SRC=&#34;FIGDIR/small/517330v5_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (35K):
org.highwire.dtl.DTLVardef@139b16dorg.highwire.dtl.DTLVardef@af845aorg.highwire.dtl.DTLVardef@11a52eaorg.highwire.dtl.DTLVardef@139896b_HPS_FORMAT_FIGEXP  M_FIG The two-phase model of ageing allows to study separately the effect of chronological and physiological age.

(A) Classic approaches for studying ageing tend to consider it as a black box affecting all individuals progressively from birth to death. Instead, the Smurf phenotype shows that life can be divided into two consecutive phases separated by an abrupt transition. (B) All individuals undergo this transition at a different moment in their life, prior to death. This allows us to switch from population based approaches, comparing bulks of age-matched individuals through time, to individuals-centred approaches relying on direct access to their transition status. (C) Such paradigm shift shows that hallmarks of ageing long thought to progressively change with age are actually mostly affected in a growing proportion of Smurfs, allowing for the identification of the chain of events accompanying ageing and death from natural causes. (D) By studying the behaviour of the ageing transcriptome as a function of chronological age and Smurfness separately, we demonstrate that the progressively changing transcriptional ageing signature, as described in Frenk &amp; Houseley (2018), is in fact the convolution changes accompanying chronological age signature (increased transcriptional noise) and changes associated with Smurfness (or biological age) signature (increased stress response and inflammation, decreased expression of ribosomal and mitochondrial genes). We also identified a hallmark partially associated with only old Smurfs (ATH5), suggesting that chronological age can affect, late in life, the Smurf response.

C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.08.561412">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.08.561412" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.08.561412">
        <p class="paperTitle">Preponderance of generalized chain functions in reconstructed Boolean models of biological networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.08.561412" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.08.561412" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mitra, S.; Sil, P.; Subbaroyan, A.; Martin, O. C.; Samal, A.</p>
        <p class="info">Score: 1.2, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.08.561412' target='https://doi.org/10.1101/2023.10.08.561412'> 10.1101/2023.10.08.561412</a></p>
        <p class="abstract">Boolean networks (BNs) have been extensively used to model the dynamics of gene regulatory networks (GRNs) that underlie cellular decisions. The dynamics of BNs depend on the network architecture and regulatory logic rules (or Boolean functions (BFs)) associated with nodes, both of which have been shown to be far from random in large-scale studies of reconstructed Boolean models. At the level of the BFs, nested canalyzing functions (NCFs) have been shown to be strongly enriched in such GRN models. The central question we address here is whether that enrichment is due to certain sub-types of NCFs. To answer this, we build on one sub-type of NCFs, the chain functions (or chain-0 functions) proposed by Gat-Viks and Shamir. First, we propose 2 other sub-types of NCFs, namely, the class of chain-1 functions which is the dual of the class of chain-0 functions, and generalized chain functions, the union of the chain-0 and chain-1 types. Next, we find that the fraction of NCFs that are chain-0 functions decays exponentially with the number of inputs, and exhibits a fractal-like behaviour as a function of the bias for a fixed number of inputs. Moreover, we explain several of these observations analytically. Then, by analyzing 5990 BFs extracted from a large dataset of reconstructed Boolean models, and 2 other datasets, we find that generalized chain functions are significantly enriched within the NCFs. Lastly, we illustrate the severe restriction imposed by generalized chain functions compared to NCFs for 3 biological models and perform model selection on them using known relative stability constraints.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561729">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561729" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561729">
        <p class="paperTitle">Virus-host protein co-expression networks reveal temporal organization and strategies of viral infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561729" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561729" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guantes, R.; Aguirre, J.</p>
        <p class="info">Score: 1.1, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561729' target='https://doi.org/10.1101/2023.10.10.561729'> 10.1101/2023.10.10.561729</a></p>
        <p class="abstract">Viral replication is a complex dynamical process involving the global remodelling of the host cellular machinery across several stages. In this study, we provide a unified view of the virus-host interaction at the proteome level reconstructing protein co-expression networks from quantitative temporal data of four large DNA viruses. We take advantage of a formal framework, the theory of interacting networks, to describe the viral infection as a dynamical system taking place on a network of networks where perturbations induced by viral proteins spread to hijack the host proteome for the virus benefit. Our methodology demonstrates how the viral replication cycle can be effectively examined as a complex interaction between protein networks, providing useful insights into the viral and host temporal organization and strategies, key protein nodes targeted by the virus and dynamical bottlenecks during the course of the infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561563">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561563" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561563">
        <p class="paperTitle">Computational modeling establishes mechanotransduction as a potent entrainment cue for the mammalian circadian clock</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561563" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561563" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Francis, E. A.; Rangamani, P.</p>
        <p class="info">Score: 1.0, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561563' target='https://doi.org/10.1101/2023.10.09.561563'> 10.1101/2023.10.09.561563</a></p>
        <p class="abstract">Mechanotransduction, which is the integration of mechanical signals from the cells external environment to changes in intracellular signaling, governs many cellular functions. Recent studies show that the mechanical state of the cell is also coupled to the cellular circadian clock. To investigate possible interactions between circadian rhythms and cellular mechanotransduction, we developed a computational model that integrates these two pathways. Combining previous models of mechan-otransduction and circadian clocks, we postulated that translocation of YAP/TAZ and MRTF into the nucleus leads to altered expression of circadian proteins. Simulations from our model predict that lower levels of cytoskeletal activity are associated with longer circadian oscillation periods and higher oscillation amplitudes, consistent with recent experimental observations. Furthermore, accumulation of YAP/TAZ and MRTF in the nucleus causes circadian oscillations to decay. These effects hold both at the single-cell level and within a population-level framework. Finally, we investigated the effects of mutations in YAP or lamin A, the latter of which lead to a class of diseases known as laminopathies. Oscillations in PER/CRY and BMAL1 are substantially weaker in populations of cells with in silico mutations in YAP and lamin A, suggesting that defects in mechanotransduction can disrupt the circadian clock in certain disease states. However, we find that normal oscillatory behavior can be rescued by lowering substrate stiffness, indicating a possible biological compensatory mechanism. Thus our study identifies that mechanotransduction could be a potent entrainment cue for cellular clocks and this crosstalk can be leveraged to rescue the circadian clock in disease states.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
