<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Does ribosome composition maximize growth rate? The role of RNA instability
Authors: Szeliova, D.; Müller, S.; Zanghellini, J.
Score: 15.0, Published: 2023-07-07 DOI: 10.1101/2023.07.07.548114
Ribosomes are protein synthesis machines that are central to cellular self-fabrication, and the synthesis time of a ribosome places an upper bound on growth rate. While most cellular enzymes are proteins, ribosomes consist of 1/3 protein and 2/3 RNA (in E. coli). Recent research suggests that ribosome composition arises from a trade-off between two &#34;autocatalytic loops&#34;, ribosomal protein and RNA polymerase synthesis, respectively.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Does ribosome composition maximize growth rate? The role of RNA instability
Authors: Szeliova, D.; Müller, S.; Zanghellini, J.
Score: 15.0, Published: 2023-07-07 DOI: 10.1101/2023.07.07.548114
Ribosomes are protein synthesis machines that are central to cellular self-fabrication, and the synthesis time of a ribosome places an upper bound on growth rate. While most cellular enzymes are proteins, ribosomes consist of 1/3 protein and 2/3 RNA (in E. coli). Recent research suggests that ribosome composition arises from a trade-off between two &#34;autocatalytic loops&#34;, ribosomal protein and RNA polymerase synthesis, respectively." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-16T10:39:40+00:00" />
<meta property="article:modified_time" content="2023-07-16T10:39:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Does ribosome composition maximize growth rate? The role of RNA instability
Authors: Szeliova, D.; Müller, S.; Zanghellini, J.
Score: 15.0, Published: 2023-07-07 DOI: 10.1101/2023.07.07.548114
Ribosomes are protein synthesis machines that are central to cellular self-fabrication, and the synthesis time of a ribosome places an upper bound on growth rate. While most cellular enzymes are proteins, ribosomes consist of 1/3 protein and 2/3 RNA (in E. coli). Recent research suggests that ribosome composition arises from a trade-off between two &#34;autocatalytic loops&#34;, ribosomal protein and RNA polymerase synthesis, respectively."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Does ribosome composition maximize growth rate? The role of RNA instability\nAuthors: Szeliova, D.; Müller, S.; Zanghellini, J.\nScore: 15.0, Published: 2023-07-07 DOI: 10.1101/2023.07.07.548114\nRibosomes are protein synthesis machines that are central to cellular self-fabrication, and the synthesis time of a ribosome places an upper bound on growth rate. While most cellular enzymes are proteins, ribosomes consist of 1/3 protein and 2/3 RNA (in E. coli). Recent research suggests that ribosome composition arises from a trade-off between two \u0026#34;autocatalytic loops\u0026#34;, ribosomal protein and RNA polymerase synthesis, respectively.",
  "keywords": [
    
  ],
  "articleBody": " Does ribosome composition maximize growth rate? The role of RNA instability\nAuthors: Szeliova, D.; Müller, S.; Zanghellini, J.\nScore: 15.0, Published: 2023-07-07 DOI: 10.1101/2023.07.07.548114\nRibosomes are protein synthesis machines that are central to cellular self-fabrication, and the synthesis time of a ribosome places an upper bound on growth rate. While most cellular enzymes are proteins, ribosomes consist of 1/3 protein and 2/3 RNA (in E. coli). Recent research suggests that ribosome composition arises from a trade-off between two \"autocatalytic loops\", ribosomal protein and RNA polymerase synthesis, respectively. In this work, we develop a (coarse-grained) mechanistic model of a self-fabricating cell, validate it under different growth conditions, and use resource balance analysis (RBA) to study maximum growth rate as a function of ribosome composition. Our model highlights the importance of RNA instability. If we neglect it, RNA synthesis is always \"cheaper\" than protein synthesis, leading to an RNA-only ribosome at maximum growth rate. To account for RNA turnover, we explore two scenarios regarding the activity of RNases. In (a) degradation is proportional to RNA content, whereas in (b) ribosomal proteins cooperatively mitigate RNA instability by protecting it from misfolding and subsequent degradation. In both cases, an increase in ribosomal protein content raises protein synthesis costs while reducing RNA turnover costs. This leads to a mixed ribosome composed of RNA and proteins. However, only in scenario (b), where we consider the cooperative protection of ribosomal RNA by proteins, our model predictions are in qualitative agreement with experimental data under different growth conditions. Our research offers new mechanistic insights into ribosome biogenesis and evolution. Furthermore, it paves the way for understanding the protein-rich ribosome composition found in archaea and mitochondria.\nThe protein interactome of the citrus Huanglongbing pathogen Candidatus Liberibacter asiaticus\nAuthors: Carter, E. W.; Guerra-Peraza, O.; Wang, N.\nScore: 12.6, Published: 2023-07-12 DOI: 10.1101/2023.07.10.548374\nCandidatus Liberibacter asiaticus (CLas) is the causal agent of the devastating citrus Huanglongbing (HLB) disease. Our understanding of the pathogenicity mechanism and biology of CLas remain limited because CLas has not been cultured in artificial media. CLas encodes 1136 proteins of which 415 have unknown functions. Since genetic studies of CLas genes with unknown functions are impossible, we utilized genome-wide protein-protein interactions (PPIs) yeast-two-hybrid (Y2H) assays to help solve the mystery. PPIs are fundamental to all cellular processes and machinery and instrumental in investigating uncharacterized proteins and inferring biological pathways. In total, 916 bait and 936 prey proteins were included in the three-phase screening, which identified 4245 interactions. The false positive rate of the Y2H assay was estimated to be 3.1%. Pull-down assays confirmed the robustness of our Y2H. The average interactions per node for CLas Y2H interactome were approximately 15.6, significantly higher than free-living bacteria, indicating genome reduction has led to a multi-function of proteins. PPIs provide clues for functions of 371 uncharacterized proteins of CLas. Forty HUB node proteins were identified which might play critical roles in CLas, including a quinone oxidoreductase and LysR that are known to protect bacteria against oxidative stress. This explains why CLas survives well in the phloem even though it triggers immune-mediated disease, systemic and chronic production of reactive oxygen species, and phloem cell death. This PPI database facilitates the investigation of CLas cellular biochemistry and physiology, functions of uncharacterized proteins, and pathogenicity mechanisms of the pathogen.\nProteome-wide copy-number estimation from transcriptomics\nAuthors: Sweatt, A. J.; Griffiths, C. D.; Paudel, B. B.; Janes, K. A.\nScore: 12.0, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548432\nProtein copy numbers constrain systems-level properties of regulatory networks, but absolute proteomic data remain scarce compared to transcriptomics obtained by RNA sequencing. We addressed this persistent gap by relating mRNA to protein statistically using best-available data from quantitative proteomics-transcriptomics for 4366 genes in 369 cell lines. The approach starts with a central estimate of protein copy number and hierarchically appends mRNA-protein and mRNA-mRNA dependencies to define an optimal gene-specific model that links mRNAs to protein. For dozens of independent cell lines and primary prostate samples, these protein inferences from mRNA outmatch stringent null models, a count-based protein-abundance repository, and empirical protein-to-mRNA ratios. The optimal mRNA-to-protein relationships capture biological processes along with hundreds of known protein-protein interaction complexes, suggesting mechanistic relationships are embedded. We use the method to estimate viral-receptor abundances of CD55-CXADR from human heart transcriptomes and build 1489 systems-biology models of coxsackievirus B3 infection susceptibility. When applied to 796 RNA sequencing profiles of breast cancer from The Cancer Genome Atlas, inferred copy-number estimates collectively reclassify 26% of Luminal A and 29% of Luminal B tumors. Protein-based reassignments strongly involve a pharmacologic target for luminal breast cancer (CDK4) and an -catenin that is often undetectable at the mRNA level (CTTNA2). Thus, by adopting a gene-centered perspective of mRNA-protein covariation across different biological contexts, we achieve accuracies comparable to the technical reproducibility limits of contemporary proteomics. The collection of gene-specific models is assembled as a web tool for users seeking mRNA-guided predictions of absolute protein abundance (http://janeslab.shinyapps.io/Pinferna). Significance statementIt is easier to quantify mRNA in cells than it is to quantify protein, but proteins are what execute most cellular functions. Even though protein is synthesized from mRNA in cells, relating a cellular quantity of mRNA to a quantity of protein is challenging. Here, we bring together quantitative measures of mRNA and protein for 4366 genes in 369 different cultured cell types to build equations that predict protein abundance from the abundance of mRNAs expressed. These equations capture facets of biological regulation and work better than existing alternatives that rely on consensus values or ratios. Since mRNA measurements are more widespread than protein, this study makes new analyses possible by protein estimation from mRNA.\nA systems-level analysis of the mutually antagonistic roles of RKIP and BACH1 in dynamics of cancer cell plasticity\nAuthors: Shyam, S.; R, S.; Sehgal, M.; Jolly, M. K.\nScore: 3.1, Published: 2023-07-08 DOI: 10.1101/2023.07.08.547338\nPhenotypic plasticity is a hallmark of cancer metastasis. Epithelial-mesenchymal transition (EMT) is an important axis of phenotypic plasticity. Raf kinase-B inhibitor protein (RKIP) and BTB and CNC homology 1 (BACH1) are two proteins reported to influence EMT. In breast cancer, they act antagonistically, but the exact nature of their roles in mediating EMT and associated other axes of plasticity remains unclear. Here, analysing transcriptomic data, we reveal their antagonistic trends in a pan-cancer manner, in terms of association with EMT, metabolic reprogramming and immune evasion via PD-L1. Next, we developed and simulated a mechanism-based gene regulatory network that captures how RKIP and BACH1 engage in feedback loops with drivers of EMT and stemness. We found that RKIP and BACH1 belong to two separate \"teams\" of players - while BACH1 belonged to the one that drove pro-EMT, stem-like and therapy-resistant cell-states, RKIP is a member of a team that enables pro-epithelial, less stem-like and therapy-sensitive phenotypes. Finally, we observed that low RKIP levels and concomitant upregulated BACH1 levels associated with worse clinical outcomes in many cancer types. Together, our systems-level analysis indicates that the emergent dynamics of underlying regulatory network underlie the antagonistic patterns of RKIP and BACH1 with various axes of cancer cell plasticity, as well as with patient survival data.\nUnveiling Hidden Connections in Omics Data via pyPARAGON: an Integrative Hybrid Approach for Disease Network Construction\nAuthors: Arici, K. M.; Tuncbag, N.\nScore: 2.0, Published: 2023-07-15 DOI: 10.1101/2023.07.13.547583\nOmics technologies are powerful tools for detecting dysregulated and altered signaling components in various contexts, encompassing disease states, patients, and drug-perturbations. Network inference or reconstruction algorithms play an integral role in the successful analysis and identification of causal relationships between omics hits. However, accurate representation of signaling networks and identification of context-specific interactions within sparse omics datasets in complex interactomes pose significant challenges in integrative approaches. To address these challenges, we present pyPARAGON (PAgeRAnk-flux on Graphlet-guided network for multi-Omic data integratioN), a novel tool that combines network propagation with graphlets. By leveraging network motifs instead of pairwise connections among proteins, pyPARAGON offers improved accuracy and reduces the inclusion of nonspecific interactions in signaling networks. Through comprehensive evaluations on benchmark cancer signaling pathways, we demonstrate that pyPARAGON outperforms state-of-the-art approaches in node propagation and edge inference. Furthermore, pyPARAGON exhibits promising performance in discovering cancer driver networks. Notably, we demonstrate its utility in network-based stratification of patient tumors by integrating phosphoproteomic data from 105 breast cancer tumors with the interactome, leading to the discovery of tumor-specific signaling pathways. Overall, the development and evaluation of pyPARAGON significantly contributes to the field as an effective tool for the analysis and integration of multi-omic data in the context of signaling networks. pyPARAGON is available at https://github.com/metunetlab/pyPARAGON.\nScaling, correlations, and artifacts in biological timeseries\nAuthors: Faerberg, D. F.; Gurarie, V.; Ruvinsky, I.\nScore: 2.0, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548439\nSO_SCPLOWUMMARYC_SCPLOWTime is a key variable in biology. Conceptual and methodological advances focused recent studies of timing in biological systems on monitoring trait dynamics in singled individuals. Here we contribute to this effort by analyzing general properties of individual timeseries. We make four broad claims. First, we show that for traits that range from behavior in animals, to growth in plants, to division timing in cell lineages, faster timeseries tend to be comprised of consistently shorter sub-stages, not a few unusually fast sub-stages. We demonstrate that this property constitutes a particular type of scaling that can be readily detected by a straightforward comparison of absolute and relative variability within timeseries data for any trait. We show that correlations within timeseries are necessary and sufficient for the type of scaling we describe and infer that the ubiquitous occurrence of scaling results from natural correlations within the continuous processes that govern trait dynamics. Second, failing to observe scaling indicates a break in correlation and, we argue, a switch in the mechanisms underlying the rate of trait change. Third, we caution that analyses of individual timeseries are highly susceptible to batch effects, but contend that it is possible to detect signal despite the artifactual correlations introduced by batching. Finally, we show that the timing of biological process that scale is overdispersed compared to matched but uncorrelated comparisons; this is an advantage in variable environments. Our results advance understanding of common properties of biological timeseries and offer practical tools for their further study.\nRobustness of biomolecular networks suggests functional modules far from the edge of chaos\nAuthors: Park, K. H.; Costa, F. X.; Rocha, L. M.; Albert, R.; Rozum, J. C.\nScore: 8.6, Published: 2023-07-02 DOI: 10.1101/2023.06.30.547297\nA common feature of complex systems is their ability to balance the flexibility needed to adapt to their environment with the rigidity required for robust function. It has been conjectured that living systems accomplish this by existing at the \"edge of chaos\", i.e., the critical boundary between ordered and disordered dynamics. Simple toy models of gene regulatory networks lend support to this idea, and mathematical tools developed for these toy models yield similar results when applied to experimentally-supported models of specific cellular regulatory mechanisms (functional modules). Here, however, we demonstrate that a deeper inspection of 72 experimentally-supported discrete dynamical models of functional modules reveals previously unobserved order in these systems on long time scales, suggesting greater rigidity in these systems than was previously conjectured. Our analysis relies on new measures that quantify the tendency of perturbations to spread through a discrete dynamical system. A benefit of our new approach is that it accounts for how system trajectories are mapped to phenotypes in practice. Because these measures are computationally expensive to estimate, existing tools were insufficient for the ensemble of models considered here. To simulate the tens of millions of trajectories required for convergence, we developed a multipurpose CUDA-based simulation tool, which we have made available as the open-source Python library cubewalkers. We find that in experimentally-supported models of biomolecular functional modules, perturbation propagation is more transitory than previously thought, and that even in cases where large perturbation cascades persist, their phenotypic effects are often minimal. Moreover, by examining the impact of update scheme on experimentally-supported models, we find evidence that stochasticity and desynchronization can lead to increased recovery from regulatory perturbation cascades in functional modules and uncover previously unreported population-level robustness to even timing perturbations in these systems. We identify specific biological mechanisms underlying these dynamical behaviors and highlight them in experimentally-supported regulatory networks from the systems biology literature. Based on novel measures and simulations, our results suggest that-contrary to current theory-functional modules of biological systems are ordered and far from the edge of chaos.\nLiebig's law of the minimum in the TGF-β pathway\nAuthors: Li, Y.; Deng, D.; Höfer, C. T.; Kim, J.; Heo, W. D.; Xu, Q.; Liu, X.; Zi, Z.\nScore: 1.4, Published: 2023-07-14 DOI: 10.1101/2023.07.10.548398\nCells use signaling pathways to sense and respond to their environments. The transforming growth factor-{beta} (TGF-{beta}) pathway produces context-specific responses. Here, we combined modeling and experimental analysis to study the dependence of the output of the TGF-{beta} pathway on the abundance of signaling molecules in the pathway. We showed that the TGF-{beta} pathway processes the variation of TGF-{beta} receptor abundance using Liebig's law of the minimum, meaning that the output-modifying factor is the signaling protein that is most limited, to determine signaling responses across cell types and in single cells. We found that the abundance of either the type I (TGFBR1) or type II (TGFBR2) TGF-{beta} receptor determined the responses of cancer cell lines, such that the receptor with relatively low abundance dictates the response. Furthermore, nuclear SMAD2 signaling correlated with the abundance of TGF-{beta} receptor in single cells depending on the relative expression levels of TGFBR1 and TGFBR2. A similar control principle could govern the heterogeneity of signaling responses in other signaling pathways.\n",
  "wordCount" : "2252",
  "inLanguage": "en",
  "datePublished": "2023-07-16T10:39:40Z",
  "dateModified": "2023-07-16T10:39:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta"><span>updated on July 16, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548114">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548114" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548114">
        <p class="paperTitle">Does ribosome composition maximize growth rate? The role of RNA instability</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548114" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548114" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Szeliova, D.; Müller, S.; Zanghellini, J.</p>
        <p class="info">Score: 15.0, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548114' target='https://doi.org/10.1101/2023.07.07.548114'> 10.1101/2023.07.07.548114</a></p>
        <p class="abstract">Ribosomes are protein synthesis machines that are central to cellular self-fabrication, and the synthesis time of a ribosome places an upper bound on growth rate. While most cellular enzymes are proteins, ribosomes consist of 1/3 protein and 2/3 RNA (in E. coli). Recent research suggests that ribosome composition arises from a trade-off between two &#34;autocatalytic loops&#34;, ribosomal protein and RNA polymerase synthesis, respectively.

In this work, we develop a (coarse-grained) mechanistic model of a self-fabricating cell, validate it under different growth conditions, and use resource balance analysis (RBA) to study maximum growth rate as a function of ribosome composition. Our model highlights the importance of RNA instability. If we neglect it, RNA synthesis is always &#34;cheaper&#34; than protein synthesis, leading to an RNA-only ribosome at maximum growth rate.

To account for RNA turnover, we explore two scenarios regarding the activity of RNases. In (a) degradation is proportional to RNA content, whereas in (b) ribosomal proteins cooperatively mitigate RNA instability by protecting it from misfolding and subsequent degradation. In both cases, an increase in ribosomal protein content raises protein synthesis costs while reducing RNA turnover costs. This leads to a mixed ribosome composed of RNA and proteins. However, only in scenario (b), where we consider the cooperative protection of ribosomal RNA by proteins, our model predictions are in qualitative agreement with experimental data under different growth conditions.

Our research offers new mechanistic insights into ribosome biogenesis and evolution. Furthermore, it paves the way for understanding the protein-rich ribosome composition found in archaea and mitochondria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548374">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548374" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548374">
        <p class="paperTitle">The protein interactome of the citrus Huanglongbing pathogen Candidatus Liberibacter asiaticus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548374" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548374" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carter, E. W.; Guerra-Peraza, O.; Wang, N.</p>
        <p class="info">Score: 12.6, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548374' target='https://doi.org/10.1101/2023.07.10.548374'> 10.1101/2023.07.10.548374</a></p>
        <p class="abstract">Candidatus Liberibacter asiaticus (CLas) is the causal agent of the devastating citrus Huanglongbing (HLB) disease. Our understanding of the pathogenicity mechanism and biology of CLas remain limited because CLas has not been cultured in artificial media. CLas encodes 1136 proteins of which 415 have unknown functions. Since genetic studies of CLas genes with unknown functions are impossible, we utilized genome-wide protein-protein interactions (PPIs) yeast-two-hybrid (Y2H) assays to help solve the mystery. PPIs are fundamental to all cellular processes and machinery and instrumental in investigating uncharacterized proteins and inferring biological pathways. In total, 916 bait and 936 prey proteins were included in the three-phase screening, which identified 4245 interactions. The false positive rate of the Y2H assay was estimated to be 3.1%. Pull-down assays confirmed the robustness of our Y2H. The average interactions per node for CLas Y2H interactome were approximately 15.6, significantly higher than free-living bacteria, indicating genome reduction has led to a multi-function of proteins. PPIs provide clues for functions of 371 uncharacterized proteins of CLas. Forty HUB node proteins were identified which might play critical roles in CLas, including a quinone oxidoreductase and LysR that are known to protect bacteria against oxidative stress. This explains why CLas survives well in the phloem even though it triggers immune-mediated disease, systemic and chronic production of reactive oxygen species, and phloem cell death. This PPI database facilitates the investigation of CLas cellular biochemistry and physiology, functions of uncharacterized proteins, and pathogenicity mechanisms of the pathogen.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548432">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548432" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548432">
        <p class="paperTitle">Proteome-wide copy-number estimation from transcriptomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548432" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548432" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sweatt, A. J.; Griffiths, C. D.; Paudel, B. B.; Janes, K. A.</p>
        <p class="info">Score: 12.0, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548432' target='https://doi.org/10.1101/2023.07.10.548432'> 10.1101/2023.07.10.548432</a></p>
        <p class="abstract">Protein copy numbers constrain systems-level properties of regulatory networks, but absolute proteomic data remain scarce compared to transcriptomics obtained by RNA sequencing. We addressed this persistent gap by relating mRNA to protein statistically using best-available data from quantitative proteomics-transcriptomics for 4366 genes in 369 cell lines. The approach starts with a central estimate of protein copy number and hierarchically appends mRNA-protein and mRNA-mRNA dependencies to define an optimal gene-specific model that links mRNAs to protein. For dozens of independent cell lines and primary prostate samples, these protein inferences from mRNA outmatch stringent null models, a count-based protein-abundance repository, and empirical protein-to-mRNA ratios. The optimal mRNA-to-protein relationships capture biological processes along with hundreds of known protein-protein interaction complexes, suggesting mechanistic relationships are embedded. We use the method to estimate viral-receptor abundances of CD55-CXADR from human heart transcriptomes and build 1489 systems-biology models of coxsackievirus B3 infection susceptibility. When applied to 796 RNA sequencing profiles of breast cancer from The Cancer Genome Atlas, inferred copy-number estimates collectively reclassify 26% of Luminal A and 29% of Luminal B tumors. Protein-based reassignments strongly involve a pharmacologic target for luminal breast cancer (CDK4) and an -catenin that is often undetectable at the mRNA level (CTTNA2). Thus, by adopting a gene-centered perspective of mRNA-protein covariation across different biological contexts, we achieve accuracies comparable to the technical reproducibility limits of contemporary proteomics. The collection of gene-specific models is assembled as a web tool for users seeking mRNA-guided predictions of absolute protein abundance (http://janeslab.shinyapps.io/Pinferna).

Significance statementIt is easier to quantify mRNA in cells than it is to quantify protein, but proteins are what execute most cellular functions. Even though protein is synthesized from mRNA in cells, relating a cellular quantity of mRNA to a quantity of protein is challenging. Here, we bring together quantitative measures of mRNA and protein for 4366 genes in 369 different cultured cell types to build equations that predict protein abundance from the abundance of mRNAs expressed. These equations capture facets of biological regulation and work better than existing alternatives that rely on consensus values or ratios. Since mRNA measurements are more widespread than protein, this study makes new analyses possible by protein estimation from mRNA.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.08.547338">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.08.547338" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.08.547338">
        <p class="paperTitle">A systems-level analysis of the mutually antagonistic roles of RKIP and BACH1 in dynamics of cancer cell plasticity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.08.547338" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.08.547338" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shyam, S.; R, S.; Sehgal, M.; Jolly, M. K.</p>
        <p class="info">Score: 3.1, Published: 2023-07-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.08.547338' target='https://doi.org/10.1101/2023.07.08.547338'> 10.1101/2023.07.08.547338</a></p>
        <p class="abstract">Phenotypic plasticity is a hallmark of cancer metastasis. Epithelial-mesenchymal transition (EMT) is an important axis of phenotypic plasticity. Raf kinase-B inhibitor protein (RKIP) and BTB and CNC homology 1 (BACH1) are two proteins reported to influence EMT. In breast cancer, they act antagonistically, but the exact nature of their roles in mediating EMT and associated other axes of plasticity remains unclear. Here, analysing transcriptomic data, we reveal their antagonistic trends in a pan-cancer manner, in terms of association with EMT, metabolic reprogramming and immune evasion via PD-L1. Next, we developed and simulated a mechanism-based gene regulatory network that captures how RKIP and BACH1 engage in feedback loops with drivers of EMT and stemness. We found that RKIP and BACH1 belong to two separate &#34;teams&#34; of players - while BACH1 belonged to the one that drove pro-EMT, stem-like and therapy-resistant cell-states, RKIP is a member of a team that enables pro-epithelial, less stem-like and therapy-sensitive phenotypes. Finally, we observed that low RKIP levels and concomitant upregulated BACH1 levels associated with worse clinical outcomes in many cancer types. Together, our systems-level analysis indicates that the emergent dynamics of underlying regulatory network underlie the antagonistic patterns of RKIP and BACH1 with various axes of cancer cell plasticity, as well as with patient survival data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.547583">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.547583" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.547583">
        <p class="paperTitle">Unveiling Hidden Connections in Omics Data via pyPARAGON: an Integrative Hybrid Approach for Disease Network Construction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.547583" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.547583" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Arici, K. M.; Tuncbag, N.</p>
        <p class="info">Score: 2.0, Published: 2023-07-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.547583' target='https://doi.org/10.1101/2023.07.13.547583'> 10.1101/2023.07.13.547583</a></p>
        <p class="abstract">Omics technologies are powerful tools for detecting dysregulated and altered signaling components in various contexts, encompassing disease states, patients, and drug-perturbations. Network inference or reconstruction algorithms play an integral role in the successful analysis and identification of causal relationships between omics hits. However, accurate representation of signaling networks and identification of context-specific interactions within sparse omics datasets in complex interactomes pose significant challenges in integrative approaches. To address these challenges, we present pyPARAGON (PAgeRAnk-flux on Graphlet-guided network for multi-Omic data integratioN), a novel tool that combines network propagation with graphlets. By leveraging network motifs instead of pairwise connections among proteins, pyPARAGON offers improved accuracy and reduces the inclusion of nonspecific interactions in signaling networks. Through comprehensive evaluations on benchmark cancer signaling pathways, we demonstrate that pyPARAGON outperforms state-of-the-art approaches in node propagation and edge inference. Furthermore, pyPARAGON exhibits promising performance in discovering cancer driver networks. Notably, we demonstrate its utility in network-based stratification of patient tumors by integrating phosphoproteomic data from 105 breast cancer tumors with the interactome, leading to the discovery of tumor-specific signaling pathways. Overall, the development and evaluation of pyPARAGON significantly contributes to the field as an effective tool for the analysis and integration of multi-omic data in the context of signaling networks. pyPARAGON is available at https://github.com/metunetlab/pyPARAGON.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548439">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548439" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548439">
        <p class="paperTitle">Scaling, correlations, and artifacts in biological timeseries</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548439" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548439" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Faerberg, D. F.; Gurarie, V.; Ruvinsky, I.</p>
        <p class="info">Score: 2.0, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548439' target='https://doi.org/10.1101/2023.07.10.548439'> 10.1101/2023.07.10.548439</a></p>
        <p class="abstract">SO_SCPLOWUMMARYC_SCPLOWTime is a key variable in biology. Conceptual and methodological advances focused recent studies of timing in biological systems on monitoring trait dynamics in singled individuals. Here we contribute to this effort by analyzing general properties of individual timeseries. We make four broad claims. First, we show that for traits that range from behavior in animals, to growth in plants, to division timing in cell lineages, faster timeseries tend to be comprised of consistently shorter sub-stages, not a few unusually fast sub-stages. We demonstrate that this property constitutes a particular type of scaling that can be readily detected by a straightforward comparison of absolute and relative variability within timeseries data for any trait. We show that correlations within timeseries are necessary and sufficient for the type of scaling we describe and infer that the ubiquitous occurrence of scaling results from natural correlations within the continuous processes that govern trait dynamics. Second, failing to observe scaling indicates a break in correlation and, we argue, a switch in the mechanisms underlying the rate of trait change. Third, we caution that analyses of individual timeseries are highly susceptible to batch effects, but contend that it is possible to detect signal despite the artifactual correlations introduced by batching. Finally, we show that the timing of biological process that scale is overdispersed compared to matched but uncorrelated comparisons; this is an advantage in variable environments. Our results advance understanding of common properties of biological timeseries and offer practical tools for their further study.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547297">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547297" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547297">
        <p class="paperTitle">Robustness of biomolecular networks suggests functional modules far from the edge of chaos</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547297" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547297" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Park, K. H.; Costa, F. X.; Rocha, L. M.; Albert, R.; Rozum, J. C.</p>
        <p class="info">Score: 8.6, Published: 2023-07-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547297' target='https://doi.org/10.1101/2023.06.30.547297'> 10.1101/2023.06.30.547297</a></p>
        <p class="abstract">A common feature of complex systems is their ability to balance the flexibility needed to adapt to their environment with the rigidity required for robust function. It has been conjectured that living systems accomplish this by existing at the &#34;edge of chaos&#34;, i.e., the critical boundary between ordered and disordered dynamics. Simple toy models of gene regulatory networks lend support to this idea, and mathematical tools developed for these toy models yield similar results when applied to experimentally-supported models of specific cellular regulatory mechanisms (functional modules). Here, however, we demonstrate that a deeper inspection of 72 experimentally-supported discrete dynamical models of functional modules reveals previously unobserved order in these systems on long time scales, suggesting greater rigidity in these systems than was previously conjectured. Our analysis relies on new measures that quantify the tendency of perturbations to spread through a discrete dynamical system. A benefit of our new approach is that it accounts for how system trajectories are mapped to phenotypes in practice. Because these measures are computationally expensive to estimate, existing tools were insufficient for the ensemble of models considered here. To simulate the tens of millions of trajectories required for convergence, we developed a multipurpose CUDA-based simulation tool, which we have made available as the open-source Python library cubewalkers. We find that in experimentally-supported models of biomolecular functional modules, perturbation propagation is more transitory than previously thought, and that even in cases where large perturbation cascades persist, their phenotypic effects are often minimal. Moreover, by examining the impact of update scheme on experimentally-supported models, we find evidence that stochasticity and desynchronization can lead to increased recovery from regulatory perturbation cascades in functional modules and uncover previously unreported population-level robustness to even timing perturbations in these systems. We identify specific biological mechanisms underlying these dynamical behaviors and highlight them in experimentally-supported regulatory networks from the systems biology literature. Based on novel measures and simulations, our results suggest that-contrary to current theory-functional modules of biological systems are ordered and far from the edge of chaos.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548398">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548398" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548398">
        <p class="paperTitle">Liebig&#39;s law of the minimum in the TGF-β pathway</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548398" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548398" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, Y.; Deng, D.; Höfer, C. T.; Kim, J.; Heo, W. D.; Xu, Q.; Liu, X.; Zi, Z.</p>
        <p class="info">Score: 1.4, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548398' target='https://doi.org/10.1101/2023.07.10.548398'> 10.1101/2023.07.10.548398</a></p>
        <p class="abstract">Cells use signaling pathways to sense and respond to their environments. The transforming growth factor-{beta} (TGF-{beta}) pathway produces context-specific responses. Here, we combined modeling and experimental analysis to study the dependence of the output of the TGF-{beta} pathway on the abundance of signaling molecules in the pathway. We showed that the TGF-{beta} pathway processes the variation of TGF-{beta} receptor abundance using Liebig&#39;s law of the minimum, meaning that the output-modifying factor is the signaling protein that is most limited, to determine signaling responses across cell types and in single cells. We found that the abundance of either the type I (TGFBR1) or type II (TGFBR2) TGF-{beta} receptor determined the responses of cancer cell lines, such that the receptor with relatively low abundance dictates the response. Furthermore, nuclear SMAD2 signaling correlated with the abundance of TGF-{beta} receptor in single cells depending on the relative expression levels of TGFBR1 and TGFBR2. A similar control principle could govern the heterogeneity of signaling responses in other signaling pathways.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
