<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Open-ST: High-resolution spatial transcriptomics in 3D
Authors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.
Score: 78.9, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572554
Spatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Open-ST: High-resolution spatial transcriptomics in 3D
Authors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.
Score: 78.9, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572554
Spatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-10T10:40:05+00:00" />
<meta property="article:modified_time" content="2024-01-10T10:40:05+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Open-ST: High-resolution spatial transcriptomics in 3D
Authors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.
Score: 78.9, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572554
Spatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Open-ST: High-resolution spatial transcriptomics in 3D\nAuthors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.\nScore: 78.9, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572554\nSpatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods.",
  "keywords": [
    
  ],
  "articleBody": " Open-ST: High-resolution spatial transcriptomics in 3D\nAuthors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.\nScore: 78.9, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572554\nSpatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods. Here, we report Open-ST, a sequencing-based, open-source experimental and computational resource to address these challenges and to study the molecular organization of tissues in 3D. In mouse brain, Open-ST captured transcripts at subcellular resolution and reconstructed cell types. In primary tumor and patient-matched healthy/metastatic lymph nodes, Open-ST captured the diversity of immune, stromal and tumor populations in space. Distinct cell states were organized around cell-cell communication hotspots in the tumor, but not the metastasis. Strikingly, the 3D reconstruction and multimodal analysis of the metastatic lymph node revealed spatially contiguous structures not visible in 2D and potential biomarkers precisely at the 3D tumor/lymph node boundary. We anticipate Open-ST to accelerate the identification of spatial molecular mechanisms in 2D and 3D.\nBroad H3K4me3 Domain Is Associated with Spatial Coherence during Mammalian Embryonic Development\nAuthors: Cao, X.; Ma, T.; Fan, R.; Yuan, G.-C.\nScore: 17.0, Published: 2023-12-12 DOI: 10.1101/2023.12.11.570452\nIt is well known that the chromatin states play a major role in cell-fate decision and cell-identity maintenance; however, the spatial variation of chromatin states in situ remains poorly characterized. Here, by leveraging recently available spatial-CUT\u0026Tag data, we systematically characterized the global spatial organization of the H3K4me3 profiles in a mouse embryo. Our analysis identified a subset of genes with spatially coherent H3K4me3 patterns, which together delineate the tissue boundaries. The spatially coherent genes are strongly enriched with tissue-specific transcriptional regulators. Remarkably, their corresponding genomic loci are marked by broad H3K4me3 domains, which is distinct from the typical H3K4me3 signature. Spatial transition across tissue boundaries is associated with continuous shortening of the broad H3K4me3 domains as well as expansion of H3K27me3 domains. Our analysis reveals a strong connection between the genomic and spatial variation of chromatin states, which may play an important role in embryonic development.\nAutism genes converge on microtubule biology and RNA-binding proteins during excitatory neurogenesis\nAuthors: Sun, N.; Teyssier, N.; Wang, B.; Drake, S.; Seyler, M.; Zaltsman, Y.; Everitt, A.; Teerikorpi, N.; Willsey, H. R.; Goodarzi, H.; Tian, R.; Kampmann, M.; Willsey, J.\nScore: 16.8, Published: 2024-01-02 DOI: 10.1101/2023.12.22.573108\nRecent studies have identified over one hundred high-confidence (hc) autism spectrum disorder (ASD) genes. Systems biological and functional analyses on smaller subsets of these genes have consistently implicated excitatory neurogenesis. However, the extent to which the broader set of hcASD genes are involved in this process has not been explored systematically nor have the biological pathways underlying this convergence been identified. Here, we leveraged CROP-Seq to repress 87 hcASD genes in a human in vitro model of cortical neurogenesis. We identified 17 hcASD genes whose repression significantly alters developmental trajectory and results in a common cellular state characterized by disruptions in proliferation, differentiation, cell cycle, microtubule biology, and RNA-binding proteins (RBPs). We also characterized over 3,000 differentially expressed genes, 286 of which had expression profiles correlated with changes in developmental trajectory. Overall, we uncovered transcriptional disruptions downstream of hcASD gene perturbations, correlated these disruptions with distinct differentiation phenotypes, and reinforced neurogenesis, microtubule biology, and RBPs as convergent points of disruption in ASD.\nHighly multiplexed, image-based pooled screens in primary cells and tissues with PerturbView\nAuthors: Kudo, T.; Meireles, A. M.; Moncada, R.; Chen, Y.; Wu, P.; Gould, J.; Hu, X.; Kornfeld, O.; Jesudason, R.; Foo, C.; Hockendorf, B.; Bravo, H. C.; Town, J. P.; Wei, R.; Rios, A.; Chandrasekar, V.; Heinlein, M.; Cai, S.; Lu, C. S.; Celen, C.; Kljavin, N.; Jiang, J.; Hleap, J. S.; Kayagaki, N.; Melo, F. d. S. e.; McGinnis, L.; Li, B.; Singh, A.; Garraway, L.; Rozenblatt-Rosen, O.; Regev, A.; Lubeck, E.\nScore: 49.3, Published: 2023-12-26 DOI: 10.1101/2023.12.26.573143\nOptical pooled screening (OPS) is a highly scalable method for linking image-based phenotypes with cellular perturbations. However, it has thus far been restricted to relatively low-plex phenotypic readouts in cancer cell lines in culture, due to limitations associated with in situ sequencing (ISS) of perturbation barcodes. Here, we developed PerturbView, an OPS technology that leverages in vitro transcription (IVT) to amplify barcodes prior to ISS, enabling screens with highly multiplexed phenotypic readouts across diverse systems, including primary cells and tissues. We demonstrate PerturbView in iPSC-derived neurons, primary immune cells, and tumor tissue sections from animal models. In a screen of immune signaling pathways in primary bone marrow-derived macrophages, PerturbView uncovered both known and novel regulators of NF{kappa}B signaling. Furthermore, we combined PerturbView with spatial transcriptomics in tissue sections from a mouse xenograft model, paving the way to in vivo screens with rich optical and transcriptomic phenotypes. PerturbView broadens the scope of OPS to a wide range of models and applications.\nTransient power-law behaviour following induction distinguishes between competing models of stochastic gene expression\nAuthors: Nicoll, A. G.; Szavits-Nossan, J.; Evans, M. R.; Grima, R.\nScore: 4.3, Published: 2023-12-30 DOI: 10.1101/2023.12.30.573521\nWhat features of transcription can be learnt by fitting mathematical models of gene expression to mRNA count data? Given a suite of models, fitting to data selects an optimal one, thus identifying a probable transcriptional mechanism. Whilst attractive, the utility of this methodology remains unclear. Here, we sample steady-state, single-cell mRNA count distributions from parameters in the physiological range, and show they cannot be used to confidently estimate the number of inactive gene states, i.e. the number of rate-limiting steps in transcriptional initiation. Distributions from over 99% of the parameter space generated using models with 2, 3, or 4 inactive states can be well fit by one with a single inactive state. However, we show that if the mRNA lifetime is hours long, then for many minutes following induction, the increase in the mean mRNA count obeys a power law whose exponent equals the sum of the number of states visited from the initial inactive state to the active state and the number of rate-limiting post-transcriptional processing steps. Our study shows that non-linear regression estimation of the exponent from eukaryotic data is sufficient to estimate the total number of regulatory steps in transcription initiation, splicing, and nuclear export.\nPhenylketonuria: modelling cerebral amino acid and neurotransmitter metabolism\nAuthors: Wegrzyn, A. B.; van Vliet, D.; van Eunen, K.; Heiner-Fokkema, M. R.; van der Zee, E. A.; van Spronsen, F.; Bakker, B. M.\nScore: 2.2, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574352\nAO_SCPLOWBSTRACTC_SCPLOWO_ST_ABSObjectiveC_ST_ABSPhenylketonuria (PKU) is caused by a deficiency of the hepatic enzyme phenylalanine hydroxylase, which primarily converts phenylalanine into tyrosine. Despite a phenylalanine-deprived diet, many adult PKU patients display deficits of executive functions. These are hypothesised to be caused by high cerebral phenylalanine and a shortage of monoaminergic neurotransmitters. MethodTo better understand the relationship between plasma and brain amino acid levels and monoaminergic neurotransmitter biochemistry, we constructed a computational model. The model comprises the transport of large neutral amino acids (LNAA) across the blood-brain barrier as well as cerebral amino acid and monoaminergic neurotransmitter metabolism and was validated by direct measurements of brain amino acid concentrations in PKU mice on various diets. ResultsThe model predicts that brain amino acids are positively controlled by the concentrations of the corresponding amino acids in the blood, and to a lesser extent negatively by other amino acids competing for the transport systems. The model suggests that brain levels of monoaminergic neurotransmitters are controlled more by phenylalanine, probably through non-competitive inhibition of the hydroxylases rather than by their precursor amino acids. Therefore, the model predicts that the decrease of neurotransmitters in PKU cannot be fully rescued by the addition of tyrosine and tryptophan alone, but also benefits from a reduction of the phenylalanine level. Thereby, the model substantiates the experimental data of the synergistic effect of both brain phenylalanine reduction and increased precursor amino acid availability to improve brain neurotransmitters in PKU mice. ConclusionWe present the first complete model of the LNAA transport through the blood-brain barrier and subsequent brain neurotransmitter metabolism in PKU. The model leads to a better understanding of the pathophysiological mechanisms and the influence of individual amino acids in the diet on the underlying brain dysfunction in PKU. Moreover, it identifies gaps in our current knowledge about the LNAA transport across the blood-brain barrier. Furthermore, the model can be readily applied in studies of other neurological disorders in which the relation between diet, gene activities, brain amino acids, and neurotransmitters is important.\nOncogenic PIK3CA corrupts growth factor signaling specificity\nAuthors: Madsen, R. R.; Le Marois, A.; Mruk, O.; Voliotis, M.; Yin, S.; Sufi, J.; Qin, X.; Zhao, S. J.; Gorczynska, J.; Morelli, D.; Davidson, L.; Sahai, E.; Korolchuk, V. I.; Tape, C. J.; Vanhaesebroeck, B.\nScore: 28.6, Published: 2023-12-26 DOI: 10.1101/2023.12.23.573207\nPathological activation of the PI3K/AKT pathway is among the most frequent defects in human cancer and is also the cause of rare overgrowth disorders. Yet, there is currently no systematic understanding of the quantitative flow of information within PI3K/AKT signaling and how it is perturbed by disease-causing mutations. Here, we develop scalable, single-cell approaches for systematic analyses of signal processing within the PI3K pathway, enabling precise calculations of its information transfer for different growth factors. Using genetically-engineered human cell models with allele dose-dependent expression of PIK3CAH1047R, we show that this oncogene is not a simple, constitutive pathway activator but a context-dependent modulator of extracellular signal transfer. PIK3CAH1047Rreduces information transmission downstream of IGF1 while selectively enhancing EGF-induced signaling and transcriptional responses. This leads to a gross reduction in signaling specificity, akin to \"blurred\" signal perception. The associated increase in signaling heterogeneity promotes phenotypic diversity in a human cervical cancer cell line model and in human induced pluripotent stem cells. Collectively, these findings and the accompanying methodological advances lay the foundations for a systematic mapping of the quantitative mechanisms of PI3K/AKT-dependent signal processing and phenotypic control in health and disease. One-sentence summarySingle-cell signaling and information theoretic analyses reveal that oncogenic PI3K/AKT activation leads to a gross reduction in signaling specificity, context-dependent EGF response amplification as well as increased phenotypic heterogeneity.\nImpaired biogenesis of basic proteins impacts multiple hallmarks of the aging brain\nAuthors: Di Fraia, D.; Marino, A.; Lee, J. H.; Kelmer Sacramento, E.; Baumgart, M.; Bagnoli, S.; Tomaz da Silva, P.; Kumar Sahu, A.; Siano, G.; Tiessen, M.; Gagneur, J.; Frydman, J.; Cellerino, A.; Ori, A.\nScore: 34.3, Published: 2024-01-09 DOI: 10.1101/2023.07.20.549210\nAging and neurodegeneration entail diverse cellular and molecular hallmarks. Here, we studied the effects of aging on the transcriptome, translatome, and multiple layers of the proteome in the brain of a short-lived killifish. We reveal that aging causes widespread reduction of proteins enriched in basic amino acids that is independent of mRNA regulation, and it is not due to impaired proteasome activity. Instead, we identify a cascade of events where aberrant translation pausing leads to reduced ribosome availability resulting in proteome remodeling independently of transcriptional regulation. Our research uncovers a vulnerable point in the aging brain's biology; the biogenesis of basic DNA/RNA binding proteins. This vulnerability may represent a unifying principle that connects various aging hallmarks, encompassing genome integrity and the biosynthesis of macromolecules.\nMetalinksDB: a flexible and contextualizable resource of metabolite-protein interactions\nAuthors: Farr, E. B.; Dimitrov, D.; Turei, D.; Schmidt, C.; Lobentanzer, S.; Dugourd, A.; Saez-Rodriguez, J.\nScore: 2.5, Published: 2023-12-30 DOI: 10.1101/2023.12.30.573715\nInteractions between proteins and metabolites are key for cellular function, from the catalytic breakdown of nutrients to signaling. An important case is cell-cell communication, where cellular metabolites are secreted into the microenvironment and initiate a signaling cascade by binding to an intra- or extracellular receptor of another cell. While protein-protein mediated cell-cell communication is routinely inferred from transcriptomic data, for metabolite-protein interactions this is challenging due to the limitations of high-throughput single-cell and spatial metabolomics technologies, together with the absence of comprehensive prior knowledge resources that include metabolites. Here we report MetalinksDB, a comprehensive and flexible database of intercellular metabolite-protein interactions that is a magnitude larger than existing ones. MetalinksDB can be tailored to specific biological contexts such as diseases, pathways, or tissue/cellular locations by querying subsets of interactions using the web interface (https://metalinks.omnipathdb.org/) or the knowledge graph adapters. We showcase the use of MetalinksDB by identifying deregulated processes in renal cancer patients from multi-omics data as well as inferring metabolite-mediated cell-cell communication events driving acute kidney injury from spatial transcriptomic data. We anticipate that MetalinksDB will facilitate the study of metabolite-mediated communication processes. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=113 SRC=\"FIGDIR/small/573715v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (34K): org.highwire.dtl.DTLVardef@177f9bforg.highwire.dtl.DTLVardef@1ad8605org.highwire.dtl.DTLVardef@139df3borg.highwire.dtl.DTLVardef@6ebbf1_HPS_FORMAT_FIGEXP M_FIG C_FIG\nOptimization-Based Synthesis with Directed Cell Migration\nAuthors: Havenhill, E.; Ghosh, S.\nScore: 2.7, Published: 2023-12-23 DOI: 10.1101/2023.12.22.573173\nCollective behavior of biological agents such as herds of organisms and cells is a fundamental feature in the systems biology and in the emergence of new phenomena in the biological environ-ment. Collective cell migration under a physical or chemical cue is an example of this fundamental phenomenon where individual cell migration is driven by the collective behavior of the neighboring cells and vice versa. The goal of this research is to discover the mathematical rules of collective cell migration using experimental data and testing the predictive nature of the models in independent experimental data. Such insight is made possible in this work with the hybrid use of dynamic mode decomposition (DMD) [1] and optimal control theory. Both single and multi-cellular systems are simulated, including obstacle courses, using this framework. The results of this work show how cells collectively behave during their migration and also, opens the possibility of designing robotic cells for possible therapeutic purpose where the cell trajectory can be controlled.\n",
  "wordCount" : "2282",
  "inLanguage": "en",
  "datePublished": "2024-01-10T10:40:05Z",
  "dateModified": "2024-01-10T10:40:05Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 10, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.572554">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.572554" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.572554">
        <p class="paperTitle">Open-ST: High-resolution spatial transcriptomics in 3D</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.572554" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.572554" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.</p>
        <p class="info">Score: 78.9, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.572554' target='https://doi.org/10.1101/2023.12.22.572554'> 10.1101/2023.12.22.572554</a></p>
        <p class="abstract">Spatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods. Here, we report Open-ST, a sequencing-based, open-source experimental and computational resource to address these challenges and to study the molecular organization of tissues in 3D. In mouse brain, Open-ST captured transcripts at subcellular resolution and reconstructed cell types. In primary tumor and patient-matched healthy/metastatic lymph nodes, Open-ST captured the diversity of immune, stromal and tumor populations in space. Distinct cell states were organized around cell-cell communication hotspots in the tumor, but not the metastasis. Strikingly, the 3D reconstruction and multimodal analysis of the metastatic lymph node revealed spatially contiguous structures not visible in 2D and potential biomarkers precisely at the 3D tumor/lymph node boundary. We anticipate Open-ST to accelerate the identification of spatial molecular mechanisms in 2D and 3D.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.11.570452">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.11.570452" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.11.570452">
        <p class="paperTitle">Broad H3K4me3 Domain Is Associated with Spatial Coherence during Mammalian Embryonic Development</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.11.570452" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.11.570452" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cao, X.; Ma, T.; Fan, R.; Yuan, G.-C.</p>
        <p class="info">Score: 17.0, Published: 2023-12-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.11.570452' target='https://doi.org/10.1101/2023.12.11.570452'> 10.1101/2023.12.11.570452</a></p>
        <p class="abstract">It is well known that the chromatin states play a major role in cell-fate decision and cell-identity maintenance; however, the spatial variation of chromatin states in situ remains poorly characterized. Here, by leveraging recently available spatial-CUT&amp;Tag data, we systematically characterized the global spatial organization of the H3K4me3 profiles in a mouse embryo. Our analysis identified a subset of genes with spatially coherent H3K4me3 patterns, which together delineate the tissue boundaries. The spatially coherent genes are strongly enriched with tissue-specific transcriptional regulators. Remarkably, their corresponding genomic loci are marked by broad H3K4me3 domains, which is distinct from the typical H3K4me3 signature. Spatial transition across tissue boundaries is associated with continuous shortening of the broad H3K4me3 domains as well as expansion of H3K27me3 domains. Our analysis reveals a strong connection between the genomic and spatial variation of chromatin states, which may play an important role in embryonic development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.573108">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.573108" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.573108">
        <p class="paperTitle">Autism genes converge on microtubule biology and RNA-binding proteins during excitatory neurogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.573108" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.573108" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sun, N.; Teyssier, N.; Wang, B.; Drake, S.; Seyler, M.; Zaltsman, Y.; Everitt, A.; Teerikorpi, N.; Willsey, H. R.; Goodarzi, H.; Tian, R.; Kampmann, M.; Willsey, J.</p>
        <p class="info">Score: 16.8, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.573108' target='https://doi.org/10.1101/2023.12.22.573108'> 10.1101/2023.12.22.573108</a></p>
        <p class="abstract">Recent studies have identified over one hundred high-confidence (hc) autism spectrum disorder (ASD) genes. Systems biological and functional analyses on smaller subsets of these genes have consistently implicated excitatory neurogenesis. However, the extent to which the broader set of hcASD genes are involved in this process has not been explored systematically nor have the biological pathways underlying this convergence been identified. Here, we leveraged CROP-Seq to repress 87 hcASD genes in a human in vitro model of cortical neurogenesis. We identified 17 hcASD genes whose repression significantly alters developmental trajectory and results in a common cellular state characterized by disruptions in proliferation, differentiation, cell cycle, microtubule biology, and RNA-binding proteins (RBPs). We also characterized over 3,000 differentially expressed genes, 286 of which had expression profiles correlated with changes in developmental trajectory. Overall, we uncovered transcriptional disruptions downstream of hcASD gene perturbations, correlated these disruptions with distinct differentiation phenotypes, and reinforced neurogenesis, microtubule biology, and RBPs as convergent points of disruption in ASD.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.26.573143">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.26.573143" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.26.573143">
        <p class="paperTitle">Highly multiplexed, image-based pooled screens in primary cells and tissues with PerturbView</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.26.573143" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.26.573143" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kudo, T.; Meireles, A. M.; Moncada, R.; Chen, Y.; Wu, P.; Gould, J.; Hu, X.; Kornfeld, O.; Jesudason, R.; Foo, C.; Hockendorf, B.; Bravo, H. C.; Town, J. P.; Wei, R.; Rios, A.; Chandrasekar, V.; Heinlein, M.; Cai, S.; Lu, C. S.; Celen, C.; Kljavin, N.; Jiang, J.; Hleap, J. S.; Kayagaki, N.; Melo, F. d. S. e.; McGinnis, L.; Li, B.; Singh, A.; Garraway, L.; Rozenblatt-Rosen, O.; Regev, A.; Lubeck, E.</p>
        <p class="info">Score: 49.3, Published: 2023-12-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.26.573143' target='https://doi.org/10.1101/2023.12.26.573143'> 10.1101/2023.12.26.573143</a></p>
        <p class="abstract">Optical pooled screening (OPS) is a highly scalable method for linking image-based phenotypes with cellular perturbations. However, it has thus far been restricted to relatively low-plex phenotypic readouts in cancer cell lines in culture, due to limitations associated with in situ sequencing (ISS) of perturbation barcodes. Here, we developed PerturbView, an OPS technology that leverages in vitro transcription (IVT) to amplify barcodes prior to ISS, enabling screens with highly multiplexed phenotypic readouts across diverse systems, including primary cells and tissues. We demonstrate PerturbView in iPSC-derived neurons, primary immune cells, and tumor tissue sections from animal models. In a screen of immune signaling pathways in primary bone marrow-derived macrophages, PerturbView uncovered both known and novel regulators of NF{kappa}B signaling. Furthermore, we combined PerturbView with spatial transcriptomics in tissue sections from a mouse xenograft model, paving the way to in vivo screens with rich optical and transcriptomic phenotypes. PerturbView broadens the scope of OPS to a wide range of models and applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.30.573521">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.30.573521" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.30.573521">
        <p class="paperTitle">Transient power-law behaviour following induction distinguishes between competing models of stochastic gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.30.573521" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.30.573521" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nicoll, A. G.; Szavits-Nossan, J.; Evans, M. R.; Grima, R.</p>
        <p class="info">Score: 4.3, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.30.573521' target='https://doi.org/10.1101/2023.12.30.573521'> 10.1101/2023.12.30.573521</a></p>
        <p class="abstract">What features of transcription can be learnt by fitting mathematical models of gene expression to mRNA count data? Given a suite of models, fitting to data selects an optimal one, thus identifying a probable transcriptional mechanism. Whilst attractive, the utility of this methodology remains unclear. Here, we sample steady-state, single-cell mRNA count distributions from parameters in the physiological range, and show they cannot be used to confidently estimate the number of inactive gene states, i.e. the number of rate-limiting steps in transcriptional initiation. Distributions from over 99% of the parameter space generated using models with 2, 3, or 4 inactive states can be well fit by one with a single inactive state. However, we show that if the mRNA lifetime is hours long, then for many minutes following induction, the increase in the mean mRNA count obeys a power law whose exponent equals the sum of the number of states visited from the initial inactive state to the active state and the number of rate-limiting post-transcriptional processing steps. Our study shows that non-linear regression estimation of the exponent from eukaryotic data is sufficient to estimate the total number of regulatory steps in transcription initiation, splicing, and nuclear export.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574352">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574352" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574352">
        <p class="paperTitle">Phenylketonuria: modelling cerebral amino acid and neurotransmitter metabolism</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574352" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574352" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wegrzyn, A. B.; van Vliet, D.; van Eunen, K.; Heiner-Fokkema, M. R.; van der Zee, E. A.; van Spronsen, F.; Bakker, B. M.</p>
        <p class="info">Score: 2.2, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574352' target='https://doi.org/10.1101/2024.01.05.574352'> 10.1101/2024.01.05.574352</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWO_ST_ABSObjectiveC_ST_ABSPhenylketonuria (PKU) is caused by a deficiency of the hepatic enzyme phenylalanine hydroxylase, which primarily converts phenylalanine into tyrosine. Despite a phenylalanine-deprived diet, many adult PKU patients display deficits of executive functions. These are hypothesised to be caused by high cerebral phenylalanine and a shortage of monoaminergic neurotransmitters.

MethodTo better understand the relationship between plasma and brain amino acid levels and monoaminergic neurotransmitter biochemistry, we constructed a computational model. The model comprises the transport of large neutral amino acids (LNAA) across the blood-brain barrier as well as cerebral amino acid and monoaminergic neurotransmitter metabolism and was validated by direct measurements of brain amino acid concentrations in PKU mice on various diets.

ResultsThe model predicts that brain amino acids are positively controlled by the concentrations of the corresponding amino acids in the blood, and to a lesser extent negatively by other amino acids competing for the transport systems. The model suggests that brain levels of monoaminergic neurotransmitters are controlled more by phenylalanine, probably through non-competitive inhibition of the hydroxylases rather than by their precursor amino acids. Therefore, the model predicts that the decrease of neurotransmitters in PKU cannot be fully rescued by the addition of tyrosine and tryptophan alone, but also benefits from a reduction of the phenylalanine level. Thereby, the model substantiates the experimental data of the synergistic effect of both brain phenylalanine reduction and increased precursor amino acid availability to improve brain neurotransmitters in PKU mice.

ConclusionWe present the first complete model of the LNAA transport through the blood-brain barrier and subsequent brain neurotransmitter metabolism in PKU. The model leads to a better understanding of the pathophysiological mechanisms and the influence of individual amino acids in the diet on the underlying brain dysfunction in PKU. Moreover, it identifies gaps in our current knowledge about the LNAA transport across the blood-brain barrier. Furthermore, the model can be readily applied in studies of other neurological disorders in which the relation between diet, gene activities, brain amino acids, and neurotransmitters is important.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.573207">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.573207" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.573207">
        <p class="paperTitle">Oncogenic PIK3CA corrupts growth factor signaling specificity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.573207" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.573207" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Madsen, R. R.; Le Marois, A.; Mruk, O.; Voliotis, M.; Yin, S.; Sufi, J.; Qin, X.; Zhao, S. J.; Gorczynska, J.; Morelli, D.; Davidson, L.; Sahai, E.; Korolchuk, V. I.; Tape, C. J.; Vanhaesebroeck, B.</p>
        <p class="info">Score: 28.6, Published: 2023-12-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.573207' target='https://doi.org/10.1101/2023.12.23.573207'> 10.1101/2023.12.23.573207</a></p>
        <p class="abstract">Pathological activation of the PI3K/AKT pathway is among the most frequent defects in human cancer and is also the cause of rare overgrowth disorders. Yet, there is currently no systematic understanding of the quantitative flow of information within PI3K/AKT signaling and how it is perturbed by disease-causing mutations. Here, we develop scalable, single-cell approaches for systematic analyses of signal processing within the PI3K pathway, enabling precise calculations of its information transfer for different growth factors. Using genetically-engineered human cell models with allele dose-dependent expression of PIK3CAH1047R, we show that this oncogene is not a simple, constitutive pathway activator but a context-dependent modulator of extracellular signal transfer. PIK3CAH1047Rreduces information transmission downstream of IGF1 while selectively enhancing EGF-induced signaling and transcriptional responses. This leads to a gross reduction in signaling specificity, akin to &#34;blurred&#34; signal perception. The associated increase in signaling heterogeneity promotes phenotypic diversity in a human cervical cancer cell line model and in human induced pluripotent stem cells. Collectively, these findings and the accompanying methodological advances lay the foundations for a systematic mapping of the quantitative mechanisms of PI3K/AKT-dependent signal processing and phenotypic control in health and disease.

One-sentence summarySingle-cell signaling and information theoretic analyses reveal that oncogenic PI3K/AKT activation leads to a gross reduction in signaling specificity, context-dependent EGF response amplification as well as increased phenotypic heterogeneity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549210">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549210" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549210">
        <p class="paperTitle">Impaired biogenesis of basic proteins impacts multiple hallmarks of the aging brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549210" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549210" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Di Fraia, D.; Marino, A.; Lee, J. H.; Kelmer Sacramento, E.; Baumgart, M.; Bagnoli, S.; Tomaz da Silva, P.; Kumar Sahu, A.; Siano, G.; Tiessen, M.; Gagneur, J.; Frydman, J.; Cellerino, A.; Ori, A.</p>
        <p class="info">Score: 34.3, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549210' target='https://doi.org/10.1101/2023.07.20.549210'> 10.1101/2023.07.20.549210</a></p>
        <p class="abstract">Aging and neurodegeneration entail diverse cellular and molecular hallmarks. Here, we studied the effects of aging on the transcriptome, translatome, and multiple layers of the proteome in the brain of a short-lived killifish. We reveal that aging causes widespread reduction of proteins enriched in basic amino acids that is independent of mRNA regulation, and it is not due to impaired proteasome activity. Instead, we identify a cascade of events where aberrant translation pausing leads to reduced ribosome availability resulting in proteome remodeling independently of transcriptional regulation. Our research uncovers a vulnerable point in the aging brain&#39;s biology; the biogenesis of basic DNA/RNA binding proteins. This vulnerability may represent a unifying principle that connects various aging hallmarks, encompassing genome integrity and the biosynthesis of macromolecules.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.30.573715">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.30.573715" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.30.573715">
        <p class="paperTitle">MetalinksDB: a flexible and contextualizable resource of metabolite-protein interactions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.30.573715" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.30.573715" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Farr, E. B.; Dimitrov, D.; Turei, D.; Schmidt, C.; Lobentanzer, S.; Dugourd, A.; Saez-Rodriguez, J.</p>
        <p class="info">Score: 2.5, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.30.573715' target='https://doi.org/10.1101/2023.12.30.573715'> 10.1101/2023.12.30.573715</a></p>
        <p class="abstract">Interactions between proteins and metabolites are key for cellular function, from the catalytic breakdown of nutrients to signaling. An important case is cell-cell communication, where cellular metabolites are secreted into the microenvironment and initiate a signaling cascade by binding to an intra- or extracellular receptor of another cell. While protein-protein mediated cell-cell communication is routinely inferred from transcriptomic data, for metabolite-protein interactions this is challenging due to the limitations of high-throughput single-cell and spatial metabolomics technologies, together with the absence of comprehensive prior knowledge resources that include metabolites. Here we report MetalinksDB, a comprehensive and flexible database of intercellular metabolite-protein interactions that is a magnitude larger than existing ones. MetalinksDB can be tailored to specific biological contexts such as diseases, pathways, or tissue/cellular locations by querying subsets of interactions using the web interface (https://metalinks.omnipathdb.org/) or the knowledge graph adapters. We showcase the use of MetalinksDB by identifying deregulated processes in renal cancer patients from multi-omics data as well as inferring metabolite-mediated cell-cell communication events driving acute kidney injury from spatial transcriptomic data. We anticipate that MetalinksDB will facilitate the study of metabolite-mediated communication processes.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=113 SRC=&#34;FIGDIR/small/573715v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (34K):
org.highwire.dtl.DTLVardef@177f9bforg.highwire.dtl.DTLVardef@1ad8605org.highwire.dtl.DTLVardef@139df3borg.highwire.dtl.DTLVardef@6ebbf1_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.573173">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.573173" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.573173">
        <p class="paperTitle">Optimization-Based Synthesis with Directed Cell Migration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.573173" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.573173" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Havenhill, E.; Ghosh, S.</p>
        <p class="info">Score: 2.7, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.573173' target='https://doi.org/10.1101/2023.12.22.573173'> 10.1101/2023.12.22.573173</a></p>
        <p class="abstract">Collective behavior of biological agents such as herds of organisms and cells is a fundamental feature in the systems biology and in the emergence of new phenomena in the biological environ-ment. Collective cell migration under a physical or chemical cue is an example of this fundamental phenomenon where individual cell migration is driven by the collective behavior of the neighboring cells and vice versa. The goal of this research is to discover the mathematical rules of collective cell migration using experimental data and testing the predictive nature of the models in independent experimental data. Such insight is made possible in this work with the hybrid use of dynamic mode decomposition (DMD) [1] and optimal control theory. Both single and multi-cellular systems are simulated, including obstacle courses, using this framework. The results of this work show how cells collectively behave during their migration and also, opens the possibility of designing robotic cells for possible therapeutic purpose where the cell trajectory can be controlled.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
