<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Simultaneous Multi-Omics Analysis by Direct Infusion Mass Spectrometry (SMAD-MS)
Authors: Jiang, Y.; Salladay-Perez, I.; Momenzadeh, A.; Covarrubias, A. J.; Meyer, J. G.
Score: 22.3, Published: 2023-06-29 DOI: 10.1101/2023.06.26.546628
Combined multi-omics analysis of proteomics, polar metabolomics, and lipidomics requires separate liquid chromatography-mass spectrometry (LC-MS) platforms for each omics layer. This requirement for different platforms limits throughput and increases costs, preventing the application of mass spectrometry-based multi-omics to large scale drug discovery or clinical cohorts.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Simultaneous Multi-Omics Analysis by Direct Infusion Mass Spectrometry (SMAD-MS)
Authors: Jiang, Y.; Salladay-Perez, I.; Momenzadeh, A.; Covarrubias, A. J.; Meyer, J. G.
Score: 22.3, Published: 2023-06-29 DOI: 10.1101/2023.06.26.546628
Combined multi-omics analysis of proteomics, polar metabolomics, and lipidomics requires separate liquid chromatography-mass spectrometry (LC-MS) platforms for each omics layer. This requirement for different platforms limits throughput and increases costs, preventing the application of mass spectrometry-based multi-omics to large scale drug discovery or clinical cohorts." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-02T10:39:32+00:00" />
<meta property="article:modified_time" content="2023-07-02T10:39:32+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Simultaneous Multi-Omics Analysis by Direct Infusion Mass Spectrometry (SMAD-MS)
Authors: Jiang, Y.; Salladay-Perez, I.; Momenzadeh, A.; Covarrubias, A. J.; Meyer, J. G.
Score: 22.3, Published: 2023-06-29 DOI: 10.1101/2023.06.26.546628
Combined multi-omics analysis of proteomics, polar metabolomics, and lipidomics requires separate liquid chromatography-mass spectrometry (LC-MS) platforms for each omics layer. This requirement for different platforms limits throughput and increases costs, preventing the application of mass spectrometry-based multi-omics to large scale drug discovery or clinical cohorts."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Simultaneous Multi-Omics Analysis by Direct Infusion Mass Spectrometry (SMAD-MS)\nAuthors: Jiang, Y.; Salladay-Perez, I.; Momenzadeh, A.; Covarrubias, A. J.; Meyer, J. G.\nScore: 22.3, Published: 2023-06-29 DOI: 10.1101/2023.06.26.546628\nCombined multi-omics analysis of proteomics, polar metabolomics, and lipidomics requires separate liquid chromatography-mass spectrometry (LC-MS) platforms for each omics layer. This requirement for different platforms limits throughput and increases costs, preventing the application of mass spectrometry-based multi-omics to large scale drug discovery or clinical cohorts.",
  "keywords": [
    
  ],
  "articleBody": " Simultaneous Multi-Omics Analysis by Direct Infusion Mass Spectrometry (SMAD-MS)\nAuthors: Jiang, Y.; Salladay-Perez, I.; Momenzadeh, A.; Covarrubias, A. J.; Meyer, J. G.\nScore: 22.3, Published: 2023-06-29 DOI: 10.1101/2023.06.26.546628\nCombined multi-omics analysis of proteomics, polar metabolomics, and lipidomics requires separate liquid chromatography-mass spectrometry (LC-MS) platforms for each omics layer. This requirement for different platforms limits throughput and increases costs, preventing the application of mass spectrometry-based multi-omics to large scale drug discovery or clinical cohorts. Here, we present an innovative strategy for simultaneous multi-omics analysis by direct infusion (SMAD) using one single injection without liquid chromatography. SMAD allows quantification of over 9,000 metabolite m/z features and over 1,300 proteins from the same sample in less than five minutes. We validated the efficiency and reliability of this method and then present two practical applications: mouse macrophage M1/M2 polarization and high throughput drug screening in human 293T cells. Finally, we demonstrate relationships between proteomic and metabolomic data are discovered by machine learning.\nMassively integrated coexpression analysis reveals transcriptional regulation, evolution and cellular implications of the noncanonical translatome\nAuthors: Rich, A.; Acar, O.; Carvunis, A.-R.\nScore: 17.3, Published: 2023-06-22 DOI: 10.1101/2023.03.16.533058\nBackgroundRecent studies uncovered pervasive transcription and translation of thousands of noncanonical open reading frames (nORFs) outside of annotated genes. The contribution of nORFs to cellular phenotypes is difficult to infer using conventional approaches because nORFs tend to be short, of recent de novo origins, and lowly expressed. Here we develop a dedicated coexpression analysis framework that accounts for low expression to investigate the transcriptional regulation, evolution, and potential cellular roles of nORFs in Saccharomyces cerevisiae. ResultsOur results reveal that nORFs tend to be preferentially coexpressed with genes involved in cellular transport or homeostasis but rarely with genes involved in RNA processing. Mechanistically, we discover that young de novo nORFs located downstream of conserved genes tend to leverage their neighbors promoters through transcription readthrough, resulting in high coexpression and high expression levels. Transcriptional piggybacking also influences the coexpression profiles of young de novo nORFs located upstream of genes, but to a lesser extent and without detectable impact on expression levels. Transcriptional piggybacking influences, but does not determine, the transcription profiles of de novo nORFs emerging nearby genes. About 40% of nORFs are not strongly coexpressed with any gene but are transcriptionally regulated nonetheless and tend to form entirely new transcription modules. We offer a web browser interface (https://carvunislab.csb.pitt.edu/shiny/coexpression/) to efficiently query, visualize and download our coexpression inferences. ConclusionsOur results suggest that nORF transcription is highly regulated. Our coexpression dataset serves as an unprecedented resource for unraveling how nORFs integrate into cellular networks, contribute to cellular phenotypes, and evolve.\nOne thousand samples per day capillary-flow LC/MS/MS for high-speed, high-sensitivity and in-depth proteomics\nAuthors: Tomioka, A.; Tomioka, R.; Mori, I.; Isobe, Y.; Arita, M.; Imami, K.; Kanao, E.; Ogata, K.; Ishihama, Y.\nScore: 22.0, Published: 2023-06-07 DOI: 10.1101/2023.06.05.543682\nWe developed a capillary-flow LC/MS/MS system with ultrahigh speed, enabling a throughput of 1,000 samples per day while maintaining high sensitivity and depth of analysis. In targeted LC/MS mode, 36 endogenous phosphopeptides in HeLa cells, including EphA2- derived phosphopeptide isomers, were successfully quantified with high selectivity and linearity by combining ion mobility separation. When 500 ng of HeLa cell digest was measured 100 times repeatedly in data-dependent acquisition mode, the coefficient of variation of retention time, peak intensity and number of identified peptides were on average 3.4%, 19.8%, and 6.0%, respectively. In data-independent acquisition mode, this system achieved the identification and quantification of 3,139 protein groups from a 100 ng HeLa cell digest and 2,145 protein groups from a sample of only 10 ng. The coefficient of variation of protein commonly quantified in the triplicate analysis ranged from 12 to 24% for HeLa digest samples ranging from 10 to 1000 ng. Finally, we applied this high-speed system to the spatial proteomics of the mouse brain, and succeeded in capturing the proteome distribution along a 96-sectioned brain structure in 135 minutes. This is the first LC/MS/MS system to achieve both more than 500 samples per day and more than 3000 identified protein groups ID with less than 100 ng human cultured cells simultaneously.\nQuantifying and correcting estimation bias in transcriptional parameter inference from single-cell data\nAuthors: Grima, R.; Esmenjaud, P.-M.\nScore: 10.5, Published: 2023-06-28 DOI: 10.1101/2023.06.19.545536\nThe snapshot distribution of mRNA counts per cell can be readily measured using single molecule FISH or single-cell RNA sequencing. These distributions are often fit to the steady-state distribution of the two-state telegraph model to estimate the three transcriptional parameters for a gene of interest: the rate of transcription, the rate of switching to the active transcriptional state and the rate of switching to the inactive state. These parameters are often understood as being reflective of the average cell in a population. However it is currently unclear to what extent does the cell-to-cell variation of parameters (extrinsic noise) affect the accuracy of parameter estimation. Here we develop an analytical approach to understand the factors determining the size and sign of estimation bias. We find bias signatures that depend on the source of extrinsic noise and the mode of transcriptional activity. We estimate the size of extrinsic noise from the covariance matrix of scRNA-seq data which we use to correct published estimates of parameters for mammalian genes.\nThe genetic and dietary landscape of the muscle insulin signalling network\nAuthors: van Gerwen, J.; Masson, S. W. C.; Cutler, H. B.; Diaz-Vegas, A.; Potter, M.; Stockli, J.; Madsen, S.; Nelson, M. E.; Humphrey, S. J.; James, D. E.\nScore: 6.6, Published: 2023-06-22 DOI: 10.1101/2023.06.21.546001\nMetabolic disease is caused by a combination of genetic and environmental factors, yet few studies have examined how these factors influence signal transduction, a key mediator of metabolism. Using mass spectrometry-based phosphoproteomics, we quantified 23,126 phosphosites in skeletal muscle of five genetically distinct mouse strains in two dietary environments, with and without acute in vivo insulin stimulation. Almost half of the insulin-regulated phosphoproteome was modified by genetic background on an ordinary diet, and high-fat high-sugar feeding affected insulin signalling in a strain-dependent manner. Our data revealed coregulated subnetworks within the insulin signalling pathway, expanding our understanding of the pathways organisation. Furthermore, associating diverse signalling responses with insulin-stimulated glucose uptake uncovered regulators of muscle insulin responsiveness, including the regulatory phosphosite S469 on Pfkfb2, a key activator of glycolysis. Finally, we confirmed the role of glycolysis in modulating insulin action in insulin resistance. Our results underscore the significance of genetics in shaping global signalling responses and their adaptability to environmental changes, emphasizing the utility of studying biological diversity with phosphoproteomics to discover key regulatory mechanisms of complex traits.\nStorm: Incorporating transient dynamics to infer the RNA velocity with metabolic labeling information\nAuthors: Peng, Q.; Qiu, X.; Li, T.\nScore: 14.8, Published: 2023-06-23 DOI: 10.1101/2023.06.21.545990\nAO_SCPLOWBSTRACTC_SCPLOWThe time-resolved scRNA-seq (tscRNA-seq) provides the possibility to infer physically meaningful kinetic parameters, e.g., the transcription, splicing or RNA degradation rate constants with correct magnitudes, and RNA velocities by incorporating temporal information. Previous approaches utilizing the deterministic dynamics and steady-state assumption on gene expression states are insufficient to achieve favorable results for the data involving transient process. We present a dynamical approach, Storm (Stochastic models of RNA metabolic-labeling), to overcome these limitations by solving stochastic differential equations of gene expression dynamics. The derivation reveals that the new mRNA sequencing data obeys different types of cell-specific Poisson distributions when jointly considering both biological and cell-specific technical noise. Storm deals with measured counts data directly and extends the RNA velocity methodology based on metabolic labeling scRNA-seq data to transient stochastic systems. Furthermore, we relax the constant parameter assumption over genes/cells to obtain gene-cell-specific transcription/splicing rates and gene-specific degradation rates, thus revealing time-dependent and cell-state specific transcriptional regulations. Storm will facilitate the study of the statistical properties of tscRNA-seq data, eventually advancing our understanding of the dynamic transcription regulation during development and disease.\nDual proteomic signature of immune cells and Yersinia pestis upon blood infection\nAuthors: Le-Bury, P.; Douche, T.; Giai Gianetto, Q.; Matondo, M.; Pizarro-Cerda, J.; Dussurget, O.\nScore: 7.0, Published: 2023-06-22 DOI: 10.1101/2023.06.19.545537\nEmerging and reemerging infectious diseases represent major public health concerns. The urgent need for infection control measures requires deep understanding of molecular pathogenesis. Global approaches to study biological systems such as mass-spectrometry based proteomics benefited from groundbreaking physical and bioinformatical technological developments over recent years. However, dual proteomic study of highly pathogenic microorganisms and their hosts in complex matrices encountered during infection remains challenging due to high protein dynamic range of samples and requirements imposed in biosafety level 3 or 4 laboratories. Here, we constructed a dual proteomic pipeline of Yersinia pestis in human blood and plasma, mirroring bacteremic phase of plague. We provide the most complete Y. pestis proteome revealing a major reshaping of important bacterial path-ways such as methionine biosynthesis and iron acquisition in human plasma. Remarkably, proteomic profiling in human blood highlights a greater Yersinia outer proteins intoxication of monocytes than neutrophils. Our study unravels global expression changes and points to a specific pathogenic signature during infection, paving the way for future exploration of proteomes in the complex context of host-pathogen interactions. Subject CategoriesMicrobiology, Virology and Host Pathogen Interaction, Proteomics\nHeritable epigenetic changes are constrained by the dynamics of regulatory architectures\nAuthors: Jose, A. M.\nScore: 4.1, Published: 2023-06-09 DOI: 10.1101/2023.06.07.544138\nInteracting molecules create regulatory architectures that can persist despite turnover of molecules. Although epigenetic changes occur within the context of such architectures, there is limited understanding of how they can influence the heritability of changes. Here I develop criteria for the heritability of regulatory architectures and use quantitative simulations of interacting regulators parsed as entities, their sensors and the sensed properties to analyze how architectures influence heritable epigenetic changes. Information contained in regulatory architectures grows rapidly with the number of interacting molecules and its transmission requires positive feedback loops. While these architectures can recover after many epigenetic perturbations, some resulting changes can become permanently heritable. Such stable changes can (1) alter steady-state levels while preserving the architecture, (2) induce different architectures that persist for many generations, or (3) collapse the entire architecture. Architectures that are otherwise unstable can become heritable through periodic interactions with external regulators, which suggests that the evolution of mortal somatic lineages with cells that reproducibly interact with the immortal germ lineage could make a wider variety of regulatory architectures heritable. Differential inhibition of the positive feedback loops that transmit regulatory architectures across generations can explain the gene-specific differences in heritable RNA silencing observed in the nematode C. elegans, which range from permanent silencing, to recovery from silencing within a few generations and subsequent resistance to silencing. More broadly, these results provide a foundation for analyzing the inheritance of epigenetic changes within the context of the regulatory architectures implemented using diverse molecules in different living systems. SIGNIFICANCERegulatory interactions in living systems are recreated in successive generations. Practical ways of analyzing how the information required for this recreation is transmitted across generations and how it can be changed are lacking. Simulating all heritable information by parsing regulatory interactions in terms of entities, their sensors, and the sensed properties reveals the minimal requirements for the heritability of regulatory interactions and how they influence the inheritance of epigenetic changes. Application of this approach can explain recent experimental results on the inheritance of RNA silencing across generations in the nematode C. elegans. Since all interactors can be abstracted as entity-sensor-property systems, similar analyses can be widely used to understand heritable epigenetic changes.\nRegulation of bacterial stringent response by an evolutionary conserved ribosomal protein L11 methylation\nAuthors: Walukiewicz, H. E.; Farris, Y.; Burnet, M. C.; Feid, S. C.; You, Y.; Christensen, D.; Payne, S. H.; Wolfe, A.; Rao, C.; Nakayasu, E. S.\nScore: 2.5, Published: 2023-06-30 DOI: 10.1101/2023.06.29.546327\nLysine and arginine methylation is an important regulator of enzyme activity and transcription in eukaryotes. However, little is known about this covalent modification in bacteria. In this work, we investigated the role of methylation in bacteria. By reanalyzing a large phyloproteomics dataset from 48 bacterial strains representing 6 phyla, we found that almost a quarter of the bacterial proteome is methylated. Many of these methylated proteins are conserved across diverse bacterial lineages, including those involved in central carbon metabolism and translation. Among the proteins with the most conserved methylation sites is ribosomal protein L11 (bL11). bL11 methylation has been a mystery for five decades, as the deletion of its methyltransferase PrmA causes no cell growth defects. A comparative proteomics analysis combined with a guanosine polyphosphate assay of the {Delta}prmA mutant in Escherichia coli revealed that bL11 methylation is important for stringent response signaling. Moreover, we show that the {Delta}prmA mutant has an abnormal polysome profile, suggesting a role in ribosomal homeostasis during stationary growth phase. Overall, our investigation demonstrates that the evolutionary conserved bL11 methylation is important for stringent response signaling and ribosomal homeostasis.\nThe alternative sigma factor RpoD4 pulses at division and links the circadian clock and the cell cycle in Synechococcus elongatus\nAuthors: Ye, C.; Saez, T.; Das, A. K.; Martins, B. M. C.; Locke, J. C. W.\nScore: 2.1, Published: 2023-06-28 DOI: 10.1101/2023.06.28.546855\nThe cyanobacterial circadian clock is an extensive regulator of gene expression, generating a 24-hour rhythm in the majority of genes in Synechococcus elongatus PCC 7942. This raises the question of how the cyanobacterial clock regulates, and receives input from, the diverse cellular processes it controls. Alternative sigma factors, which bind to the core RNA polymerase enzyme and direct the synthesis of specific sets of transcripts, are one mechanism for the clock to set the time of its outputs. In this work, we use single-cell time-lapse microscopy to reveal the transcriptional dynamics of RpoD4, an alternative sigma factor that was previously reported as either arrhythmic or circadian. We find instead that RpoD4 pulses at cell division, dynamics missed by previous bulk averaging. The circadian clock modulates the amplitude of RpoD4 expression pulses, as well as the timing of pulses through its control of division timing. In turn, a rpoD4 mutation causes a reduction in clock period. Further, a rpoD4 mutant results in smaller cell size, and increasing expression of RpoD4 results in larger cell sizes in a dose-response manner, allowing tunable control of cell size. Thus, our single-cell analysis has revealed pulsing gene expression dynamics in cyanobacteria, linking the clock, RpoD4 and the cell cycle.\n",
  "wordCount" : "2371",
  "inLanguage": "en",
  "datePublished": "2023-07-02T10:39:32Z",
  "dateModified": "2023-07-02T10:39:32Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta"><span>updated on July 2, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546628">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546628" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546628">
        <p class="paperTitle">Simultaneous Multi-Omics Analysis by Direct Infusion Mass Spectrometry (SMAD-MS)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546628" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546628" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jiang, Y.; Salladay-Perez, I.; Momenzadeh, A.; Covarrubias, A. J.; Meyer, J. G.</p>
        <p class="info">Score: 22.3, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546628' target='https://doi.org/10.1101/2023.06.26.546628'> 10.1101/2023.06.26.546628</a></p>
        <p class="abstract">Combined multi-omics analysis of proteomics, polar metabolomics, and lipidomics requires separate liquid chromatography-mass spectrometry (LC-MS) platforms for each omics layer. This requirement for different platforms limits throughput and increases costs, preventing the application of mass spectrometry-based multi-omics to large scale drug discovery or clinical cohorts. Here, we present an innovative strategy for simultaneous multi-omics analysis by direct infusion (SMAD) using one single injection without liquid chromatography. SMAD allows quantification of over 9,000 metabolite m/z features and over 1,300 proteins from the same sample in less than five minutes. We validated the efficiency and reliability of this method and then present two practical applications: mouse macrophage M1/M2 polarization and high throughput drug screening in human 293T cells. Finally, we demonstrate relationships between proteomic and metabolomic data are discovered by machine learning.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.03.16.533058">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.03.16.533058" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.03.16.533058">
        <p class="paperTitle">Massively integrated coexpression analysis reveals transcriptional regulation, evolution and cellular implications of the noncanonical translatome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.03.16.533058" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.03.16.533058" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rich, A.; Acar, O.; Carvunis, A.-R.</p>
        <p class="info">Score: 17.3, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.03.16.533058' target='https://doi.org/10.1101/2023.03.16.533058'> 10.1101/2023.03.16.533058</a></p>
        <p class="abstract">BackgroundRecent studies uncovered pervasive transcription and translation of thousands of noncanonical open reading frames (nORFs) outside of annotated genes. The contribution of nORFs to cellular phenotypes is difficult to infer using conventional approaches because nORFs tend to be short, of recent de novo origins, and lowly expressed. Here we develop a dedicated coexpression analysis framework that accounts for low expression to investigate the transcriptional regulation, evolution, and potential cellular roles of nORFs in Saccharomyces cerevisiae.

ResultsOur results reveal that nORFs tend to be preferentially coexpressed with genes involved in cellular transport or homeostasis but rarely with genes involved in RNA processing. Mechanistically, we discover that young de novo nORFs located downstream of conserved genes tend to leverage their neighbors promoters through transcription readthrough, resulting in high coexpression and high expression levels. Transcriptional piggybacking also influences the coexpression profiles of young de novo nORFs located upstream of genes, but to a lesser extent and without detectable impact on expression levels. Transcriptional piggybacking influences, but does not determine, the transcription profiles of de novo nORFs emerging nearby genes. About 40% of nORFs are not strongly coexpressed with any gene but are transcriptionally regulated nonetheless and tend to form entirely new transcription modules. We offer a web browser interface (https://carvunislab.csb.pitt.edu/shiny/coexpression/) to efficiently query, visualize and download our coexpression inferences.

ConclusionsOur results suggest that nORF transcription is highly regulated. Our coexpression dataset serves as an unprecedented resource for unraveling how nORFs integrate into cellular networks, contribute to cellular phenotypes, and evolve.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.05.543682">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.05.543682" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.05.543682">
        <p class="paperTitle">One thousand samples per day capillary-flow LC/MS/MS for high-speed, high-sensitivity and in-depth proteomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.05.543682" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.05.543682" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tomioka, A.; Tomioka, R.; Mori, I.; Isobe, Y.; Arita, M.; Imami, K.; Kanao, E.; Ogata, K.; Ishihama, Y.</p>
        <p class="info">Score: 22.0, Published: 2023-06-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.05.543682' target='https://doi.org/10.1101/2023.06.05.543682'> 10.1101/2023.06.05.543682</a></p>
        <p class="abstract">We developed a capillary-flow LC/MS/MS system with ultrahigh speed, enabling a throughput of 1,000 samples per day while maintaining high sensitivity and depth of analysis. In targeted LC/MS mode, 36 endogenous phosphopeptides in HeLa cells, including EphA2- derived phosphopeptide isomers, were successfully quantified with high selectivity and linearity by combining ion mobility separation. When 500 ng of HeLa cell digest was measured 100 times repeatedly in data-dependent acquisition mode, the coefficient of variation of retention time, peak intensity and number of identified peptides were on average 3.4%, 19.8%, and 6.0%, respectively. In data-independent acquisition mode, this system achieved the identification and quantification of 3,139 protein groups from a 100 ng HeLa cell digest and 2,145 protein groups from a sample of only 10 ng. The coefficient of variation of protein commonly quantified in the triplicate analysis ranged from 12 to 24% for HeLa digest samples ranging from 10 to 1000 ng. Finally, we applied this high-speed system to the spatial proteomics of the mouse brain, and succeeded in capturing the proteome distribution along a 96-sectioned brain structure in 135 minutes. This is the first LC/MS/MS system to achieve both more than 500 samples per day and more than 3000 identified protein groups ID with less than 100 ng human cultured cells simultaneously.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.545536">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.545536" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.545536">
        <p class="paperTitle">Quantifying and correcting estimation bias in transcriptional parameter inference from single-cell data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.545536" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.545536" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Grima, R.; Esmenjaud, P.-M.</p>
        <p class="info">Score: 10.5, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.545536' target='https://doi.org/10.1101/2023.06.19.545536'> 10.1101/2023.06.19.545536</a></p>
        <p class="abstract">The snapshot distribution of mRNA counts per cell can be readily measured using single molecule FISH or single-cell RNA sequencing. These distributions are often fit to the steady-state distribution of the two-state telegraph model to estimate the three transcriptional parameters for a gene of interest: the rate of transcription, the rate of switching to the active transcriptional state and the rate of switching to the inactive state. These parameters are often understood as being reflective of the average cell in a population. However it is currently unclear to what extent does the cell-to-cell variation of parameters (extrinsic noise) affect the accuracy of parameter estimation. Here we develop an analytical approach to understand the factors determining the size and sign of estimation bias. We find bias signatures that depend on the source of extrinsic noise and the mode of transcriptional activity. We estimate the size of extrinsic noise from the covariance matrix of scRNA-seq data which we use to correct published estimates of parameters for mammalian genes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.546001">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.546001" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.546001">
        <p class="paperTitle">The genetic and dietary landscape of the muscle insulin signalling network</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.546001" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.546001" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: van Gerwen, J.; Masson, S. W. C.; Cutler, H. B.; Diaz-Vegas, A.; Potter, M.; Stockli, J.; Madsen, S.; Nelson, M. E.; Humphrey, S. J.; James, D. E.</p>
        <p class="info">Score: 6.6, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.546001' target='https://doi.org/10.1101/2023.06.21.546001'> 10.1101/2023.06.21.546001</a></p>
        <p class="abstract">Metabolic disease is caused by a combination of genetic and environmental factors, yet few studies have examined how these factors influence signal transduction, a key mediator of metabolism. Using mass spectrometry-based phosphoproteomics, we quantified 23,126 phosphosites in skeletal muscle of five genetically distinct mouse strains in two dietary environments, with and without acute in vivo insulin stimulation. Almost half of the insulin-regulated phosphoproteome was modified by genetic background on an ordinary diet, and high-fat high-sugar feeding affected insulin signalling in a strain-dependent manner. Our data revealed coregulated subnetworks within the insulin signalling pathway, expanding our understanding of the pathways organisation. Furthermore, associating diverse signalling responses with insulin-stimulated glucose uptake uncovered regulators of muscle insulin responsiveness, including the regulatory phosphosite S469 on Pfkfb2, a key activator of glycolysis. Finally, we confirmed the role of glycolysis in modulating insulin action in insulin resistance. Our results underscore the significance of genetics in shaping global signalling responses and their adaptability to environmental changes, emphasizing the utility of studying biological diversity with phosphoproteomics to discover key regulatory mechanisms of complex traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545990">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545990" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545990">
        <p class="paperTitle">Storm: Incorporating transient dynamics to infer the RNA velocity with metabolic labeling information</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545990" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545990" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peng, Q.; Qiu, X.; Li, T.</p>
        <p class="info">Score: 14.8, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545990' target='https://doi.org/10.1101/2023.06.21.545990'> 10.1101/2023.06.21.545990</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWThe time-resolved scRNA-seq (tscRNA-seq) provides the possibility to infer physically meaningful kinetic parameters, e.g., the transcription, splicing or RNA degradation rate constants with correct magnitudes, and RNA velocities by incorporating temporal information. Previous approaches utilizing the deterministic dynamics and steady-state assumption on gene expression states are insufficient to achieve favorable results for the data involving transient process. We present a dynamical approach, Storm (Stochastic models of RNA metabolic-labeling), to overcome these limitations by solving stochastic differential equations of gene expression dynamics. The derivation reveals that the new mRNA sequencing data obeys different types of cell-specific Poisson distributions when jointly considering both biological and cell-specific technical noise. Storm deals with measured counts data directly and extends the RNA velocity methodology based on metabolic labeling scRNA-seq data to transient stochastic systems. Furthermore, we relax the constant parameter assumption over genes/cells to obtain gene-cell-specific transcription/splicing rates and gene-specific degradation rates, thus revealing time-dependent and cell-state specific transcriptional regulations. Storm will facilitate the study of the statistical properties of tscRNA-seq data, eventually advancing our understanding of the dynamic transcription regulation during development and disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.545537">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.545537" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.545537">
        <p class="paperTitle">Dual proteomic signature of immune cells and Yersinia pestis upon blood infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.545537" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.545537" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Le-Bury, P.; Douche, T.; Giai Gianetto, Q.; Matondo, M.; Pizarro-Cerda, J.; Dussurget, O.</p>
        <p class="info">Score: 7.0, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.545537' target='https://doi.org/10.1101/2023.06.19.545537'> 10.1101/2023.06.19.545537</a></p>
        <p class="abstract">Emerging and reemerging infectious diseases represent major public health concerns. The urgent need for infection control measures requires deep understanding of molecular pathogenesis. Global approaches to study biological systems such as mass-spectrometry based proteomics benefited from groundbreaking physical and bioinformatical technological developments over recent years. However, dual proteomic study of highly pathogenic microorganisms and their hosts in complex matrices encountered during infection remains challenging due to high protein dynamic range of samples and requirements imposed in biosafety level 3 or 4 laboratories. Here, we constructed a dual proteomic pipeline of Yersinia pestis in human blood and plasma, mirroring bacteremic phase of plague. We provide the most complete Y. pestis proteome revealing a major reshaping of important bacterial path-ways such as methionine biosynthesis and iron acquisition in human plasma. Remarkably, proteomic profiling in human blood highlights a greater Yersinia outer proteins intoxication of monocytes than neutrophils. Our study unravels global expression changes and points to a specific pathogenic signature during infection, paving the way for future exploration of proteomes in the complex context of host-pathogen interactions.

Subject CategoriesMicrobiology, Virology and Host Pathogen Interaction, Proteomics</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.07.544138">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.07.544138" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.07.544138">
        <p class="paperTitle">Heritable epigenetic changes are constrained by the dynamics of regulatory architectures</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.07.544138" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.07.544138" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jose, A. M.</p>
        <p class="info">Score: 4.1, Published: 2023-06-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.07.544138' target='https://doi.org/10.1101/2023.06.07.544138'> 10.1101/2023.06.07.544138</a></p>
        <p class="abstract">Interacting molecules create regulatory architectures that can persist despite turnover of molecules. Although epigenetic changes occur within the context of such architectures, there is limited understanding of how they can influence the heritability of changes. Here I develop criteria for the heritability of regulatory architectures and use quantitative simulations of interacting regulators parsed as entities, their sensors and the sensed properties to analyze how architectures influence heritable epigenetic changes. Information contained in regulatory architectures grows rapidly with the number of interacting molecules and its transmission requires positive feedback loops. While these architectures can recover after many epigenetic perturbations, some resulting changes can become permanently heritable. Such stable changes can (1) alter steady-state levels while preserving the architecture, (2) induce different architectures that persist for many generations, or (3) collapse the entire architecture. Architectures that are otherwise unstable can become heritable through periodic interactions with external regulators, which suggests that the evolution of mortal somatic lineages with cells that reproducibly interact with the immortal germ lineage could make a wider variety of regulatory architectures heritable. Differential inhibition of the positive feedback loops that transmit regulatory architectures across generations can explain the gene-specific differences in heritable RNA silencing observed in the nematode C. elegans, which range from permanent silencing, to recovery from silencing within a few generations and subsequent resistance to silencing. More broadly, these results provide a foundation for analyzing the inheritance of epigenetic changes within the context of the regulatory architectures implemented using diverse molecules in different living systems.

SIGNIFICANCERegulatory interactions in living systems are recreated in successive generations. Practical ways of analyzing how the information required for this recreation is transmitted across generations and how it can be changed are lacking. Simulating all heritable information by parsing regulatory interactions in terms of entities, their sensors, and the sensed properties reveals the minimal requirements for the heritability of regulatory interactions and how they influence the inheritance of epigenetic changes. Application of this approach can explain recent experimental results on the inheritance of RNA silencing across generations in the nematode C. elegans. Since all interactors can be abstracted as entity-sensor-property systems, similar analyses can be widely used to understand heritable epigenetic changes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.546327">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.546327" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.546327">
        <p class="paperTitle">Regulation of bacterial stringent response by an evolutionary conserved ribosomal protein L11 methylation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.546327" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.546327" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Walukiewicz, H. E.; Farris, Y.; Burnet, M. C.; Feid, S. C.; You, Y.; Christensen, D.; Payne, S. H.; Wolfe, A.; Rao, C.; Nakayasu, E. S.</p>
        <p class="info">Score: 2.5, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.546327' target='https://doi.org/10.1101/2023.06.29.546327'> 10.1101/2023.06.29.546327</a></p>
        <p class="abstract">Lysine and arginine methylation is an important regulator of enzyme activity and transcription in eukaryotes. However, little is known about this covalent modification in bacteria. In this work, we investigated the role of methylation in bacteria. By reanalyzing a large phyloproteomics dataset from 48 bacterial strains representing 6 phyla, we found that almost a quarter of the bacterial proteome is methylated. Many of these methylated proteins are conserved across diverse bacterial lineages, including those involved in central carbon metabolism and translation. Among the proteins with the most conserved methylation sites is ribosomal protein L11 (bL11). bL11 methylation has been a mystery for five decades, as the deletion of its methyltransferase PrmA causes no cell growth defects. A comparative proteomics analysis combined with a guanosine polyphosphate assay of the {Delta}prmA mutant in Escherichia coli revealed that bL11 methylation is important for stringent response signaling. Moreover, we show that the {Delta}prmA mutant has an abnormal polysome profile, suggesting a role in ribosomal homeostasis during stationary growth phase. Overall, our investigation demonstrates that the evolutionary conserved bL11 methylation is important for stringent response signaling and ribosomal homeostasis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546855">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546855" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546855">
        <p class="paperTitle">The alternative sigma factor RpoD4 pulses at division and links the circadian clock and the cell cycle in Synechococcus elongatus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546855" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546855" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ye, C.; Saez, T.; Das, A. K.; Martins, B. M. C.; Locke, J. C. W.</p>
        <p class="info">Score: 2.1, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546855' target='https://doi.org/10.1101/2023.06.28.546855'> 10.1101/2023.06.28.546855</a></p>
        <p class="abstract">The cyanobacterial circadian clock is an extensive regulator of gene expression, generating a 24-hour rhythm in the majority of genes in Synechococcus elongatus PCC 7942. This raises the question of how the cyanobacterial clock regulates, and receives input from, the diverse cellular processes it controls. Alternative sigma factors, which bind to the core RNA polymerase enzyme and direct the synthesis of specific sets of transcripts, are one mechanism for the clock to set the time of its outputs. In this work, we use single-cell time-lapse microscopy to reveal the transcriptional dynamics of RpoD4, an alternative sigma factor that was previously reported as either arrhythmic or circadian. We find instead that RpoD4 pulses at cell division, dynamics missed by previous bulk averaging. The circadian clock modulates the amplitude of RpoD4 expression pulses, as well as the timing of pulses through its control of division timing. In turn, a rpoD4 mutation causes a reduction in clock period. Further, a rpoD4 mutant results in smaller cell size, and increasing expression of RpoD4 results in larger cell sizes in a dose-response manner, allowing tunable control of cell size. Thus, our single-cell analysis has revealed pulsing gene expression dynamics in cyanobacteria, linking the clock, RpoD4 and the cell cycle.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
